<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Alberta_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Alberta/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Alberta/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Alberta/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Alberta/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Alberta/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Alberta/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TABLE cellpadding="0" cellspacing="0" style="width: 100%;"><TBODY><TR><TD style="width: 150px;"><DIV class="sideBar"><UL id="menu"><LI class="headerMenu" onmouseover="showSubMenu(null, this);"><A href="/Team:Alberta"><SPAN>Home</SPAN></A></LI><LI class="headerMenu" onmouseover="showSubMenu(null, this);"><A href="https://2012.igem.org/Team:Alberta/Team"><SPAN>Team</SPAN></A></LI><LI class="headerMenu" onmouseover="showSubMenu(null, this);"><A href="/Team:Alberta/Project"><SPAN>Project</SPAN></A></LI><DIV class="subMenu" id="contactSub"><LI><A style="padding:8px 20px 0 35px;" href="/Team:Alberta/Project"><SPAN>Overview</SPAN></A></LI><LI><A style="padding:8px 20px 0 35px;" href="/Team:Alberta/Project/colours"><SPAN>The Colours</SPAN></A></LI><LI><A style="padding:8px 20px 0 35px;" href="/Team:Alberta/Project/gradient"><SPAN>Spatial gradients</SPAN></A></LI><LI><A style="padding:8px 20px 0 35px;" href="/Team:Alberta/Project/controlelement"><SPAN>Control elements</SPAN></A></LI><LI><A style="padding:8px 20px 0 35px;" href="/Team:Alberta/Project/plasmidcontrol"><SPAN>Controling copy number</SPAN></A></LI><LI><A style="padding:8px 20px 0 35px;" href="/Team:Alberta/Project/integrateddesign"><SPAN>Integrated design</SPAN></A></LI><LI><A style="padding:8px 20px 0 35px;" href="/Team:Alberta/Project/achievements"><SPAN>Achievements</SPAN></A></LI></DIV><LI class="headerMenu" onmouseover="showSubMenu(null, this);"><A href="/Team:Alberta/Parts"><SPAN>Parts</SPAN></A></LI><DIV class="subMenu" id="partsSub"><LI><A style="padding:8px 20px 0 35px;" href="/Team:Alberta/Parts"><SPAN>Parts Submitted</SPAN></A></LI><LI><A style="padding:8px 20px 0 35px;" href="/Team:Alberta/Attributions"><SPAN>Attributions</SPAN></A></LI></DIV><LI class="headerMenu" onmouseover="showSubMenu(null, this);"><A href="/Team:Alberta/Humanpractices"><SPAN>Human Practice</SPAN></A></LI><DIV class="subMenu" id="outreachSub"><LI><A style="padding:8px 20px 0 35px;" href="/Team:Alberta/socialexperiment"><SPAN>A social experiment</SPAN></A></LI><LI><A style="padding:8px 20px 0 35px;" href="/Team:Alberta/Outreach"><SPAN>Outreach</SPAN></A></LI><LI><A style="padding:8px 20px 0 35px;" href="/Team:Alberta/collaborations"><SPAN>Collaborations</SPAN></A></LI><LI><A style="padding:8px 20px 0 35px;" href="/Team:Alberta/Safety"><SPAN>Safety</SPAN></A></LI><LI><A style="padding:8px 20px 0 35px;" href="/Team:Alberta/Attributions"><SPAN>Attributions</SPAN></A></LI></DIV><LI class="headerMenu" onmouseover="showSubMenu(null, this);"><A href="/Team:Alberta/Safety"><SPAN>Safety</SPAN></A></LI><LI class="headerMenu" onmouseover="showSubMenu(null, this);"><A href="/Team:Alberta/Sponsors"><SPAN>Sponsors</SPAN></A></LI><DIV class="subMenu" id="sponsorsSub"><LI><A style="padding:8px 20px 0 35px;" href=""><SPAN>About Sponsors</SPAN></A></LI></DIV></UL></DIV></TD></TR></TBODY></TABLE></P><P class="title">   Team Alberta
</P></DIV><DIV class="mainBody"><FONT size="5"><DIV class="underline">
Project outline 
</DIV></FONT><P><TABLE border="0"><TBODY><TR><TD><A href="#summary">Summary of the Project</A></TD><TD><A href="#rainbow">You can’t make a rainbow without primary colours</A></TD><TD><A href="#switchingcolour">Our first attempt at switching colours on and off</A></TD><TD><A href="#testingcolour">Testing colour control spatially</A></TD></TR><TR><TD><A href="#plasmidcontrol">Time for a new approach: plasmid copy # control</A></TD><TD><A href="#firsttest">Our first test of copy # control</A></TD><TD><A href="#circuits">Circuits for programming spatial colour expression</A></TD><TD><A href="#future">Where are we now and where are we going?</A></TD></TR><TR><TD><A href="#achievements">Achievements</A></TD></TR></TBODY></TABLE></P><DIV class="roundBox"><FONT size="5">
Concept
</FONT></DIV><DIV class="underline"><FONT size="5">Summary of the Project
</FONT></DIV><P><FONT size="2">
The end goal of our project is to create spatial color patterns using bacteria, such as a color wheel and a rainbow. Not only would this give an attractive end result, it would also demonstrate a fine level of control over gene expression, as it requires spatial control over several colors simultaneously. The project entails designing self-contained circuits that respond to the changing concentration ratios of two chemical inducers (IPTG and ATC)  through spatial gradients where a given ratio at a given position on the plate is interpreted as a colour. To gain experience, we began with a conventional approach using established red, yellow, and blue proteins as outputs, that were regulated by the LacI and TetR repressors.  The dynamic colour range produced in these attempts were disappointing so we developed a means of adjusting gene expression through plasmid copy number control, achieved by placing the RNA II gene in the pMB1 replication origin under TetR and LacI inducible control. At low levels of inducer, copy number increases with inducer, as greater RNA II expression leads to increased plasmid replication. We show that this method can produce a spatially striking distinction between ON and OFF. Note that this mechanism can make plasmid replication dependent on the presence of inducer effectively creating a new safety switch for genetically engineered organisms which might enter the environment. It also has potentially useful applications related to molecular biology.
</FONT></P><P>                                  </P><DIV class="underline"><FONT size="5">You can’t make a rainbow without primary colours
</FONT></DIV><P><FONT size="2">
A full colour spectrum may be made by mixing red, yellow, and blue in different amounts. From part sequences in the Registry, we selected the the blue chromoprotein amiCFP (K592010), the yellow chromoprotein amilGFP (K592009), and the classic red fluorescent protein (RFP, E1010). These parts were selected due to their excellent presentation by the 2011 Uppsala iGEM team. we designed custom ribosomal binding sites for each open reading frame using the Salis RBS calculator (<A href="https://salis.psu.edu/software/" class="external free" rel="nofollow">https://salis.psu.edu/software/</A>) to give consistent medium-high expression levels (designed transcription initiation rate 50k), ordered the sequences synthesized as gBlocks (IDT), and assembled them. Promoters were selected from the Anderson collection of constitutive sigma 70 promoters of various strengths. These promoters were trimmed outside the -10 and -35 boxes, and the bases in the interior region have been varied to limit recombination when multiple promoters are used in the same system (Pr-1-5, Pr-Lac, Pr-Tet). For details on the promoters, see our <A href="https://2012.igem.org/Team:Alberta/Parts" class="external text" rel="nofollow">Parts</A> page.

The calculated rbs elements were approximately 4x the calculated value of rbs B0034 (used universally by the Uppsala team). We therefore chose the relatively weak promoter Pr-2 to drive the constructs (Parts: Figure 4). While the amilCP plasmid produced intensely blue colonies after 24 hours of growth, the amilGFP and RFP plasmids only developed slight colouration over several days. We therefore modified our parts as follows: for RFP and amilGFP, the engineered RBSes were replaced with B0034. 

We realized that taking the time to choose the  right promoters from the beginning would be important for our later work. At the early stages of our lab training we noticed that plating cells transformed with the RFP expressing plasmid J04450 (also using B0034) developed strong colour but were noticeably smaller than colonies not expressing RFP. We also noticed that a few serial passages of  cells containing J04450, resulted in a fraction of the plated cells that had lost RFP expression altogether. This made us worry that too strong colour expression might be toxic leading to slow growth and plasmid instability. In contrast, we also worried that too weak promoters would not produce the intensity of colour that we would need for convincing visual effects. 

The approach we eventually decided on was to combine all of our synthesized promoters into a single ligation mix containing our three cut parental colour plasmids and let nature sort out the problem by providing us with a range of possibilities. Following transformation and plating we selected several colonies of each colour that showed good colour intensity and reasonable size. These were then sequenced to determine which promoter they were using. Promoters meeting the above criteria were: Pr-3 (amilGFP, RFP, not AmilCP), Pr-Lac and Pr-Tet (all 3 colours). Since Pr-3 was not found among the blue isolates we Chose are original construct under Pr-2 and the calculated RBS. The resulting colour cassettes have been submitted as BBa_K879311 (red), using promoter 3 and the B0034 RBS, BBa_K879313 (yellow), also using promoter 3 and the B0034 RBS,  and BBa_K879222 (blue), using promoter 2 and the RBS engineered for amilCP.
</FONT></P><P>                       </P><BLOCKQUOTE><BLOCKQUOTE><FONT size="2"><B>Fig 1. Colour quality.</B> Shown is the range of colour development  for our submitted parts, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K879222" class="external text" rel="nofollow">BBa_K879222</A> containing amilCP (blue), <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K879313" class="external text" rel="nofollow">BBa_K879313</A> containing amilGFP (yellow under visible light/green under UV light) and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K879311" class="external text" rel="nofollow">BBa_K879311</A> RFP (red) under different growth and light conditions. Each pigment gene is driven from a constitutive promoter. Complete descriptions can be found under <A href="https://2012.igem.org/Team:Alberta/Parts" class="external text" rel="nofollow">Parts</A>. The pellet colours shown in the bottom-right panel are derived from the overnight cultures shown above. Note that amilCP is not a fluorecent chromophore- too bad!
</FONT></BLOCKQUOTE></BLOCKQUOTE><DIV class="underline"><FONT size="5">Our first attempt at switching colours on and off
</FONT></DIV><P><FONT size="2">
For our first try at colour regulation we placed the RFP gene onto pS1C3 under the control of the synthetic LacI inducible promoter previously described by Sidney et. al (2007). Here the LacI operator has been trimmed to 17 bp that sits between the -10 and -35 region of the promoter. The gene for LacI was positioned downstream of RFP using the standard biobrick assembly method. LacI and was expressed constitutively from a moderate strength promoter (Pr-3 see Figure 2 in <A href="https://2012.igem.org/Team:Alberta/Parts" class="external text" rel="nofollow">Parts</A>). When we examined colour expression of this construct in either liquid culture or as colonies on plates, the results were very disappointing. In the presence of the recommended concentrations of IPTG (0.1mM) only pale pink was observed. The colour was only slightly improved in the presence of 1.0 mM IPTG! A closer look at the literature showed that most people interested in repressor-type regulation, place the repressor as a single copy on the E. coli chromosome [Lutz and Bujard (1997), Sidney et. al (2007)]. Since pSB1C3 is a high-copy plasmid we hypothesized that the concentration of LacI in the cell was too high for efficient induction. We therefore devised an easy strategy to weaken the LacI promoter. The Anderson promoters have two restriction sites between the promoter regions (AvrII and NheI) which we preserved in our promoter designs. Since NheI is unique in our construct, we filled the site in with DNA poymerase. The 4-bp addition that results, increases the separation between the -10 and -35 regions from the preferred 17-bp to the highly unfavoured 21bp separation [Brunner and Bujard (1987), Hawley and McClure (1983)]. In short, the promoter would be flying on only one wing. When we examined colour expression in the modified construct the results in the presence and absence of IPTG were much more noticable. We further characterized our constructs, which we call weak lacI and strong LacI, by measuring the fluorescence of RFP in overnight cultures as a function of IPTG concentration. The results are presented below.

One potential problem that we became aware of through these experiments is that the range of colour intensity observed in the presence and absence of the inducer is much different between liquid cultures and plates. When cells containing the weak lacI construct are streaked onto plates, the contrast is significantly reduced. In the absence of IPTG, pink develops over longer times of incubation so that there is only a narrow window for observing white and red. This was a little worrisome since our plan of controlling colours spatially would be based on the colour development of bacterial “lawns” on plates.
</FONT></P><P>                       </P><BLOCKQUOTE><BLOCKQUOTE><FONT size="2"><B>Fig 2. Dynamics of colour expression.</B> Panel A. Schematic of the controllable colour cassette used in these experiments. RFP and LacI are contained on the same plasmid as shown. LacI is constitutively expressed from a very weak promoter (described below) or an intermediate strength promoter (Pr-3). It controls RFP by binding to its operator positioned between the RFP promoter elements. The expression of RFP is induced by the inactivation of LacI in the presence of IPTG. The weak promoter was constructed by filling in the 4-bp overhangs  of the NheI site to the 5’ side of the -10 region (see Fig. 8) This displaces the -10 region and transcriptional start 4 bp to the right. Panel B.  RFP fluorescence from cultures grown to stationary phase in the presence of different IPTG concentrations . The response of cells expressing the weak LacI cassette is indicated in black. The response of cells expressing the strong LacI cassette is indicated in red.
 
</FONT></BLOCKQUOTE></BLOCKQUOTE><P><FONT size="2"><B>Methods:</B> Fresh colonies containing either the strong lacI plasmid or the weak LacI plasmid were inoculated into 5 mL cultures containing chloramphenicol and IPTG at the following exponential series of mM concentrations: 0, 0.001, 0.002, 0.004, 0.008, 0.016, 0.032, 0.064, 0.128. Cultures were grown to stationary phase in a shaking incubator for 20 hours at 37 C. Colour development resulting from RFP expression was measured by fluorescence as follows: Cells were first diluted 10-fold with TE buffer (10mM Tris, 1mM EDTA pH 8.0) in a final volume of 3 mL (the cuvette volume). Measurements were made using a Perkin Elmer luminescence spectrophotometer (LS 50 B), using parameters that were optimized for the highest value (Weak LacI Plasmid; 0.128 mM IPTG) with the help of Justin Fedor (Alberta iGEM alumnist 2009) These parameters are: 1. excitation wavelength (582 nm), 2. emission wavelength (603 nm), 3. filter cutoff (515 nm), 4. Slit widths lex= 10nm, lem= 10nm)
</FONT></P><DIV class="underline"><FONT size="5">Testing colour control spatially
</FONT></DIV><P><FONT size="2">
For the longer-range goal of producing a spectrum on an agar plate, one colour would have to fade out as another colour intensifies. We therefore chose an old and popular method for producing spatial gradients of small molecules that entailed placing a high concentration in a well positioned at the centre of the plate and letting it diffuse over time. If the chemical were IPTG we would expect strong colour at the plate’s centre that faded gradually toward the circumference as the IPTG concentration dropped. Below we have described a fast and reliable way for casting the centre well using cylindrical magnets! Figures 3 and 4 show the detailed casting procedure, and a rough calibration method  using a visible dye.
</FONT></P><P>                       </P><BLOCKQUOTE><BLOCKQUOTE><FONT size="2"><B>Fig 3. After several false starts using food colouring, we developed the following protocol for making gradient plates: LB agar plates were made using 25 mL of LB agar.</B> A well in each plate was made using two cylindrical neodymium magnets (dimensions 12 mm long x 4mm dia) per well that sandwich the lid at the preferred location, as illustrated. When solidified the magnet is removed by gently lifting off the lid. Gradients were formed by adding 40uL of either 100x IPTG (10mM) or 100x ATC (100mM) to each well and allowing 8 hours at room temperature for diffusion prior to plating. Overnight cultures are diluted to an approximate cell concentration of between 0.5-1.0x10<SUP>5</SUP> cells/mL. 1-2x10<SUP>4</SUP> cells in 200 uL are plated by swirling 6 ball-bearings (6 mm diameter) on the plate surface until completely wetted. The plates are then dried under a tissue culture cabinet and incubated at 37C overnight. <B>Important!</B> Cell densities that exceed those specified above produce thin transparent lawns with poor colour development. 
</FONT></BLOCKQUOTE></BLOCKQUOTE><BLOCKQUOTE><BLOCKQUOTE><FONT size="2"><B>Figure 4. Diffusion estimation using food colour.</B> 40 uL of undiluted food colouring was placed in the well of a radial gradient plate as described above and allowed to diffuse at room temperature for the indicated times. Based on these results were selected an 8-hour diffusion period for inducer gradient plates.
</FONT></BLOCKQUOTE></BLOCKQUOTE><P><FONT size="2">
With a method to make inducer gradients, and strains responding to inducer levels, we were able to make our first colour gradients, shown in Figure 5 below. A lawn of cells from strains inducible with IPTG were plated on gradient plates, and imaged after growth under visible and UV light. Far from the plate center, IPTG levels are too low to generate noticeable RFP expression. Closer in, a region of high RFP expression is visible. As with our previous  induction experiments, peak RFP levels are higher with the weak promoter than the strong promoter. Close to the center of the plate, RFP expression is reduced nearly to zero; a consequence of a lack of colonies near the center. This may be due to toxicity of high levels of RFP expression under strong induction leading to a lack of colony formation, or perhaps simply a mechanical effect due to lower cell plating densities on the meniscus formed during well casting. A spatially controlled colour gradient? Yes, but lacking the vibrancy we were hoping for.  
</FONT></P><P>                       </P><BLOCKQUOTE><BLOCKQUOTE><FONT size="2"><B>Fig 5. IPTG gradient plates were prepared as described and plated with ~10<SUP>6</SUP> cells carrying plasmids expressing RFP under weak LacI or strong LacI control, grown overnight, and imaged under white light or UV light.</B></FONT></BLOCKQUOTE></BLOCKQUOTE><P><FONT size="2">
Despite some success in generating colour, switching it &quot;on&quot; and &quot;off&quot;, and controlling it spatially, we were not satisfied with the absolute level of control of expression. In particular, when LacI was expressed under a weak promoter, expression levels were good in the presence of IPTG, but remained high enough in the absence of IPTG that colour developed in the colonies over a period of a couple of days. Raising LacI levels by placing it under control of a strong promoter resulted in better repression in the absence of IPTG, however repression remained substantial even in the presence of IPTG. Just like in our streaking experiment (Fig. 6), the dynamic range of our LacI switch, defined as high over low, was not very good. Ideally, we wanted a system which was able to express RFP at a high level when turned on, but which could be switched totally off when desired. Figure 6 below summarizes our goal, and what we observed with our standard repressor-operator-inducer system. Clearly we needed a way to increase the range of control.
</FONT></P><P>                       </P><BLOCKQUOTE><BLOCKQUOTE><FONT size="2"><B>Figure 6. Cells grown on plates show a limited range of colour development.</B> Plate images are taken from Fig 2D. Upper plates: RFP expression under strong LacI promoter (right -- IPTG+, left --  IPTG-). Lower plates: RFP expression under weak LacI (right -- IPTG+, left -- IPTG-)
</FONT></BLOCKQUOTE></BLOCKQUOTE><DIV class="underline"><FONT size="5">Time for a new approach: plasmid copy # control
</FONT></DIV><P><FONT size="2">
One way to increase the dynamic range of control is by controlling copy number. If we could increase the plasmid count in high expression regime, and decrease or eliminate plasmid levels when in a low expression regime, the range of control would be increased through gene dosing. While a number of copy number control systems have been in use, such as the traditional addition of chloramphenicol to increase copy number before miniprepping, or the multiple origins of plasmid <A href="http://partsregistry.org/wiki/index.php?title=Part:pSB2K3" class="external text" rel="nofollow">pSB2K3</A> from the Parts Registry, these systems have largely been aimed at increasing plasmid yields rather than as useful means of controlling biological systems.

Copy number is not the only possible means of increasing the dynamic range of control. Translational tuning is possible, for instance by riboswitches which prevent or allow protein synthesis depending on the presence of an inducer through the usage of RNA aptamers. However, we did not want to have to introduce additional inducers into our  system. Additional tuning of transcription is possible as well, such as by including an additional repressor. For example, the WT lac promoter includes a CRP operator, leading to additional strong repression in the presence of glucose. Coupled with minimal media lacking any trace of lactose (unlike LB), a greater dynamic range is achievable (eg Kulhman et. al PNAS 2007). We were more interested in a simple, self-contained system. Finally, it might be possible to control cell growth, eg with nutrient gradients. However, copy number is additional to these mechanisms, and has the huge advantage of adding a large level of control: two and a half orders of magnitude between single copy and high copy, with the prospect of total and permanent repression if the plasmid is lost completely. The perfect off-switch! Zero copies! 

Figure 7 below outlines the mechanism for controlling copy number with a repressor. Under antibiotic selection without an artificial means of control, plasmids that have the pMB1 origin are maintained at between 200-300 copies/cell. In the absence of antibiotic selection, the plasmid is lost passively over many generations [Velappan et al (2007)].  Others have shown however, that replication can be controlled transcriptionally (see Herman-Antosiewicz, Mol Biotech 2001 and Lutz NAR 1997). Replication from the pMB1 origin is initiated by the synthesis of an RNA molecule (RNA II) that uses a standard RNA polymerase promoter. Therefore by using the same kind of repressible promoter design already described, it should be possible to control copy number, colour gene copy number and colour intensity. In the previous examples cited above, copy control was used to amplify plasmid beyond normal limits for plasmid purification or as a method of protein over-expression for purification. Our goal is spatial colour control. 
</FONT></P><P>                  </P><BLOCKQUOTE><BLOCKQUOTE><FONT size="2"><B>Fig 7. We developed and tested a series of copy controlled origins, with the promoter for RNA II containing operators for LacI and TetR, inducible by IPTG and ATC respectively.</B> In one case we maintained the natural -10/-35 region. In another, we increased the promoter strength of these regions. In another we made a mistake with PCR primer design that actually proved to be useful. The ones we constructed are listed in Figure 8 below. With the exception of the weak (wild-type) TetO origin, all have been shown to be functional, as detailed next.
</FONT></BLOCKQUOTE></BLOCKQUOTE><P>                 </P><BLOCKQUOTE><BLOCKQUOTE><FONT size="2"><B>Fig 8. Origin promoter sequences. Shown is our design for controlling plasmid copy number through either the TetR or LacI repressors.</B> Synthetic promoters that were used to replace the native RNA II origin promoter (upper sequence) of the pSB1C3-based plasmids that constitutively express either [Pr-3]LacI (in the case of  LacI-regulated RNA II) or [Pr-3]TetR (in the case of TetR-regulated RNA II). The table shows, for each promoter, the sequence, relative strength of the -35/-10 regions, and status, where “+/+/+” indicates the promoter has been constructed/tested/ is functional, respectively. Promoters and their accompanying NsiI sites were incorporated into each repressor plasmid by appending each sequence to the 5’ end of a pcr primer whose annealing region immediately followed the -10 region of the native promoter and which synthesized towards the suffix.  The reverse primer incorporated the NsiI site at its 5’ end and synthesized towards the prefix. The resulting product was then cut with NsiI, ligated and transformed into competent cells. Since the inducer concentration required for optimal growth was unknown, each transformation mix was equally divided between culture tubes with different concentration of the appropriate inducer (IPTG: 0 mM, 0.002 mM, 0.004 mM, 0.008 mM, 0.016 mM and 0.032, 0.64 mM; ATC: same numerical series but at mg/L). Notably no growth was observed in the absence of inducer. LacI-Pr* and TetR-Pr* were our first attempt at this strategy. It was discovered after sequence analysis that the promoter primer contained a 2-bp addition that shifted the transcriptional start site 2-bp to the right. These errors were corrected with LacI-Pr and TetR-Pr.
</FONT></BLOCKQUOTE></BLOCKQUOTE><DIV class="underline"><FONT size="5">Our first test of copy # control
</FONT></DIV><P><FONT size="2">
After constructing our plasmids we had little idea what inducer concentration would be required to transform and propagate them. Too little inducer would mean plasmid loss and colonies that might be false positive. Too much inducer might result in “runaway” replication leading to the same results. We therefore divided each transformation recovery mix between culture tubes ranging in inducer concentration from zero to higher concentrations as described below for Fig 9. Growth was seen in all tubes but only non-zero concentrations resulted in plasmid of the predicted size. We selected minipreps from overnights with IPTG concentrations two-times the lower threshold, retransformed and selected individual colonies. Minipreps that contained the NsiI site that comes with the regulated promoter showed that we did not have a false-positives. 

We then tried to examine plasmid loss semi-quantitatively, by examining miniprep yields of the plasmid grown with and without the inducer, see Fig 9 below. For both the LacO and TetO plasmids, inducer allows greater plasmid replication, especially for the middle strength promoter. Also, the amount of plasmid increases with the strength of the promoter. While this is almost enough to quantitatively measure copy number, a loading control would reduce variation from miniprep procedure.
</FONT></P><P>            </P><BLOCKQUOTE><BLOCKQUOTE><FONT size="2"><B>Fig 9. Shown is the plasmid content of cells containing various repressor controlled plasmid origins.</B> 5 mLs of LB broth was inoculated with a single freshly plated colony  and grown at 37<SUP>o</SUP>C overnight under selective conditions that favoured plasmid maintenance [chloramphenicol (0.3 uL/mL) and either IPTG (0.05 mM) or ATC (1.07 nM)]. Fresh cultures were made next day with ~10<SUP>4</SUP> cells and grown under selective and non-selective conditions (+/- IPTG) and in the absence of chloramphenicol. Note that this experiment could have been improved by adding an internal DNA standard to each sample before miniprepping.
</FONT></BLOCKQUOTE></BLOCKQUOTE><P><FONT size="2">
While the reduction in plasmid level above shows that control of copy number is possible, more exciting is the possibility of total plasmid loss. It is in fact the possibility of reducing copy number to zero which gives copy number control its appeal: it forms a permanently settable switch, which will always be off even if the inducer levels later change, as well as reducing background expression of genes on the plasmid to zero. In Figure 10 below, we have looked at plasmid loss by measuring the loss of antibiotic resistant cells after a period of nonselective growth. The results are consistent with the copy number results above: the number of cells containing the Lac-Ori plasmids is reduced, with a greater reduction for the medium and weak promoter versions; and similarly for the Tet-Ori plasmids. The largest difference is for the medium-strength TetO origin; which is reduced by about 10<SUP>4</SUP>. This is consistent with the only plasmids remaining after overnight growth being the plasmids which were introduced in the initial innoculum- a near-total repression of plasmid replication.
</FONT></P><P>                        </P><BLOCKQUOTE><BLOCKQUOTE><FONT size="2"><B>Fig 10. Bacterial cultures were grown at 37<SUP>o</SUP>C in LB broth overnight under selective conditions that favoured plasmid maintenance [chloramphenicol (0.3 uL/mL) and either IPTG (0.05 mM) or ATC (1.07 nM)].</B> As a starting point, the number of viable cells in each culture was determined by spotting 5 uL of 10-fold serial dilutions ranging from undiluted to 10<SUP>-6</SUP> (7 spots/culture) onto LB-agar plates containing chloramphenicol and either IPTG or ATC at the concentrations cited above. Cell count/mL of culture was determined by counting (or estimating) the number of colonies at the highest resolvable dilution and multiplying by the dilution factor. Fresh cultures were then made by inoculating one uL of the original cultures into 5 mL of LB broth under conditions that favoured plasmid loss (no antibiotic or inducer, -/-) or plasmid maintenance (with inducer but no antibiotic, +/-; or with both, +/+) and grown as above. The number of viable cells remaining under non-selective conditions were determined as described above.
</FONT></BLOCKQUOTE></BLOCKQUOTE><P><FONT size="2">
Finally, we wanted to demonstrate that we could control plasmid copy number and cell survival in a spatially controlled manner. For this we plated our copy control plasmid strains on inducer gradient plates with antibiotic. The results are shown in Figure 11 below. In all cases we see a zone of replication surrounding the center. Cells far from the center are unable to survive and reproduce, since they cannot make the plasmid which gives antibiotic resistance. For the strongest promoter, even at the outer edges of the plate some cells survive, but the cell density is reduced. This is consistent with the relatively irrepressible nature of the strong promoter as seen in the copy number and plasmid loss experiments (Fig 9 and 10). While this result looks similar to the ordinary colour repressor gradients shown earlier (see Fig 11 (IPTG gradients) above), note that the dark regionsof the plate that are closer to its circumference are completely free of cells, as opposed to having cells which are simply expressing the gene at a somewhat lower level. Therefore, with copy number control alone, we should to be able to achieve a very high dynamic range of spatial colour control.
</FONT></P><P>                        </P><BLOCKQUOTE><BLOCKQUOTE><FONT size="2"><B>Fig 11. IPTG and ATC gradient plates spread with cells containing plasmids that
express the repressors LacI (IPTG plates) or TetR (ATC plates)
constitutively driven from Pr-3.</B> Each plasmid contains one of several
repressor-regulated origin promoters (see Fig. 8) indicated by the plate
label. 
</FONT></BLOCKQUOTE></BLOCKQUOTE><DIV class="underline"><FONT size="5">Circuits for programming spatial colour expression
</FONT></DIV><P><FONT size="2">
With these new copy-controlled plasmids as a starting point, we turned to the question of how we could use them to control two (or more!) separate colours at once. Some interesting issues are raised by controlling genes with plasmid loss. In Figure 12 below, we show two designs using only copy number control. They are both two-plasmid systems; with the similar origins maintained by the use of two antibiotics. Both plasmids use repressible origins, so that maintenance requires inducer in the presence of repressor. The two designs differ in the location of the repressor. In one system, which we call auto-repressive, the repressor is expressed by the same plasmid which it is repressing. This is similar to the copy-controlled plasmids we constructed and tested above. In the other design, which we call cross-repressive, each plasmid is repressing the other, but not itself. In both cases removal of an inducer will result in plasmid loss and selection of one of the plasmids and its colour; removal of both inducers should remove both colours. Thus we’d expect that both systems would have the same input-output table. But, there are differences. The auto-repressive circuit maintains the repressor against the surviving plasmid, and so the inducer must be kept in order to maintain plasmid survival. The cross-repressive circuit, on the other hand, loses the repressor against one plasmid as the other plasmid is eliminated. Thus after selection, the switch is permanently set, and all inducers can be withdrawn. We hope that new circuits such as these have new, interesting, and useful behaviours, above and beyond what can be done with classical transcriptional networks. As one example, replication of an auto-repressive plasmid such as the ones we have built is effectively dependent on the presence of inducer, ie the plasmid is now auxotrophic for the inducer. Were such a plasmid to be accidentally introduced into the environment, it would not be able to replicate unless the environment happened to contain sufficiently high levels of the inducer (which is unlikely for artificial molecules) forming a “kill switch” as an additional barrier to environmental contamination of synthetic organisms. Other uses for copy control include plasmid curing, positive selection and plasmid shuffling. See <A href="https://2012.igem.org/Team:Alberta/Safety" class="external text" rel="nofollow">Safety</A> page.
</FONT></P><P>                                  </P><BLOCKQUOTE><BLOCKQUOTE><FONT size="2"><B>Fig 12. Alternative repression control.</B> Auto repression: Two plasmids are initially maintained in one cell under selective conditions (Chlr+ /Kan+ /IPTG+ / ATC+). Each plasmid carrying either red or blue regulates its copy number through its own repressor. The absence of each repressor’s inducer results in the loss of the plasmid on which it resides. Cross repression: Plasmid maintenance as above. In this case however plasmids work in opposition. The survival of one demands the loss of the other. 
Inputs/outputs: In either case, cells grown in liquid culture would be expected to produce 1 of 4 colours (white included) depending on the inducer input combinations. Varying the ratio of inducers would likely vary both the hue and value in ways that we would have to be determined experimentally (probably not this year). 
</FONT></BLOCKQUOTE></BLOCKQUOTE><DIV class="underline"><FONT size="5">Where are we now and where are we going?
</FONT></DIV><P><FONT size="2">
We now have in hand, all of the parts necessary to build each circuit. We are literally one cloning step away. We also have acquired sufficient experience with colour development and plasmid loss that we think we have a reasonable chance of making it work. The spatial demonstration that we are aiming for is shown below. Here three wells sit on the vertices of an equilateral triangle and are loaded with the indicated combinations of inducers. Cells expressing either the plasmids required for the auto-repressible circuit or the cross-repressible circuit are plated as a lawn under nonselective conditions (no antibiotics) and then we shall see what we see, hopefully before the competition! Eventually we would like to expand the circuit to a full ROYGB spectrum in a way that converts the white readout to yellow. WE’RE PUMPED!!
</FONT></P><P>                                   </P><DIV class="underline"><FONT size="5">Achievements
</FONT></DIV><UL><LI>Introduced  three great protein colors into our lab: RFP (red), amilGFP (yellow in visible light), and blue chromoprotein amilCP with useful modifications such as new RBS sequences that were calculated (Salis RBS calculator) and tested. For amilCP, the new RBS is substantially stronger than the existing Registry part (K592009); for RFP, color expression is substantially worse (E1010) Those calculated for LacI and TetR are functional. Different RBSes that work, increase the sequence diversity available which is good for minimizing recombination when used in combination.
</LI><LI>We currently have these colors expressed from multiple constitutive and inducible promoters that were selected for stability and growth while still maintaining vibrance under different types of culture conditions. 
</LI><LI>Designed and tested new versions of the common repressor proteins, LacI, and TetR which lack the LVA degradation tag.
</LI><LI>Tuned control system for inducible gene expression with repressor being generated on same high copy plasmid as the controlled gene as self-contained system that works in complete media.
</LI><LI>Developed two complementary versions of system for expanding the dynamic range of gene expression plasmid copy number control. Our controllable copy number plasmids form a generally useful kill switch for genetically modified organisms at risk of environmental release
and as a positive selection cloning strategy not based on toxicity ie ccB.
</LI><LI>The colour genes have been sequenced (both strands), validated and submitted to the registry.
</LI><LI>The copy-control backbones have been validated, submitted  and will have been sequenced before the competition. 
</LI><LI>Experience in general: SynBio is harder than it looks and not always as preached. RBSes are not modular and tools designed to side-step this problem (RBS calculator) are useful but don’t always work.
</LI><LI>We have prepared and supplied plasmids and strains from supplied strains to the UBC, Calgary, and CINVESTAV-IPN-UNAM_MX iGEM Teams.
</LI><LI>Presented concept of iGEM and our project to high school teachers as part of the ”Women in Scholarship, Engineering,Science and Technology” (wisest) Teachers Appreciation Day.
</LI><LI>Formed a critical nucleus of high school leaders with a commitment to further the cause of the iGEM high school program throughout Edmonton.
</LI><LI>Oh...and did we mention?.... a really cool way of making gradient plates using magnets!!
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Alberta/Project">http://2012.igem.org/Team:Alberta/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Alberta/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Alberta/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Alberta/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Alberta/Project&amp;oldid=267470" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>