<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_METU_HS_Ankara_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:METU HS Ankara/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>METU HS IGEM</TITLE></P><DIV id="nedim" class="mila cssAnimate ct-navbar--fixedTop ct-js-navbarMakeSmaller ct-js-enableWayPoints"><DIV class="ct-navbarMobile ct-navbarMobile--inverse"><BUTTON type="button" class="navbar-toggle"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="ct-menuMobile"><UL class="ct-menuMobile-navbar"><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara">Home </A></LI><LI class="dropdown"><A href="#">Team </A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Team"> Team Members</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Collaborations"> Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Attributions"> Attributions</A></LI></UL></DIV></DIV></DIV><LI class="dropdown"><A href="#">Project </A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Description"> Description</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Design"> Design</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Experiments"> Experiments</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Results"> Results</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Model"> Model</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Notebook"> Notebook</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/InterLab"> InterLab</A></LI></DIV></DIV></DIV><LI class="dropdown"><A href="#">Parts </A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Basic_Part"> Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Composite_Part"> Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Part_Collection"> Parts Collection</A></LI></DIV><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Demonstrate">Demonstrate </A></LI><LI class="dropdown"><A href="#">HP </A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Human_Practices"> Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Public_Engagement"> Education &amp; Engagement</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Safety">Safety </A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Achievements">Achievements </A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Gallery">Gallery </A></LI><DIV id="ct-js-wrapper" class="ct-pageWrapper"><NAV class="navbar is-small navbar-inverse ct-navbar--logoright" role="navigation"><DIV class="container"><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav ct-navbarNav--defaultActive ct-navbar--fadeIn yamm"><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara">Home </A></LI><LI class="dropdown"><A href="#">Team </A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Team"> Team Members</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Collaborations"> Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Attributions"> Attributions</A></LI></UL></DIV></DIV></NAV></DIV><LI class="dropdown"><A href="#">Project </A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Description"> Description</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Design"> Design</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Experiments"> Experiments</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Results"> Results</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Model"> Model</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Notebook"> Notebook</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/InterLab"> InterLab</A></LI><LI class="dropdown"><A href="#">Parts </A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Basic_Part"> Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Composite_Part"> Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Part_Collection"> Parts Collection</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Demonstrate">Demonstrate </A></LI><LI class="dropdown"><A href="#">HP </A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Human_Practices"> Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Public_Engagement"> Education &amp; Engagement</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Safety">Safety </A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Achievements">Achievements </A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Gallery">Gallery </A></LI><HEADER class="ct-pageHeader ct-pageHeader--type2 ct-u-shadowBottom--type2 ct-pageHeader--motive ct-pageHeader--hasDescription ct-u-paddingBoth10"><DIV class="container ct-u-triangleBottomLeft"><DIV class="row"><DIV class="col-md-12"><H1 class="text-capitalize ct-fw-600 ct-u-colorWhite">
                        Results
                    </H1></DIV></DIV></DIV></HEADER><SECTION class="ct-u-paddingBoth50"><DIV class="container"><H3>Overview</H3><P>
                For our lab experiences, we’ve encountered an enormous number of failures which brought us up to the point we are 
                now at; having an extreme feeling of joy, content and tiredness.
            </P><H4>Summary of Achievements in our experiments</H4><P><UL><LI>We have successfully cloned 5 of our parts to pSB1C3 backbone in order to submit to the registry; and 3 of them to pSB1A3 backbone for our biochemical assays.</LI><LI>We have confirmed that our parts were correctly inserted by means of PCR.</LI><LI>We’ve conducted biochemical assays in which we verified our hypothesis and proved the improving effects of our genes.</LI></UL></P><H3> Abbreviations: </H3><UL><LI>
                    NC: Negative Control
                </LI><LI>
                    Comp.: Composite
                </LI></UL><H3>Clonings:</H3><P>
                In our project, our aim was to increase the tolerance and resistance of KO11, ethanologenic strain of E.coli, to byproducts and inhibitors that occur before bioethanol production, 
                specifically during the pretreatment of lignocellulosic biomass. In order to achieve that, we focused on FucO and GSH as our targets.

            </P><P>
                We’ve designed 5 parts, 2 of which are basic and 3 are composite. 
            </P><P>
                One of our basic parts included only FucO and the other only GSH as the protein coding region. On the other hand, our composite parts were designed to include 
                GSH and FucO separately and simultaneously. 

            </P><P>
                As a part of our lab journey, we’ve done 8 clonings in total. 5 of them were to be submitted in the registry and 3 of them were used for our biochemical assays.

            </P><P>
               In order to be able to submit our parts to the registry, we inserted all of our parts to pSB1C3 backbone and cloned them to DH5α.
            </P><H3>DH5α Clonings:</H3><H4>Clonings of basic parts: FucO only and GSH only.</H4><P>
                In order to be able to grow colonies of DH5α with our first and second basic parts, we’ve ligated them to pSB1C3 backbone with a ratio of 1:3. Then, we’ve transformed the 
                ligations to DH5α competent cells and grew colonies on LB agar plates containing chloramphenicol at a final concentration of 40 µg/ml. After obtaining DH5α colonies, we’ve 
            done colony PCR and considered the results which led us to the conclusion that colony no: 4 from FucO (figure 1 and figure 3) and no: 8 from GSH (figure 2) have given the best results. 

            </P><P>
                Thus, we’ve done plasmid isolation for basic 1 from the 4th colony and for basic 2 from the 8th colony, followed by PCR to confirm our results.
            </P><DIV class="col-md-6" style="text-align:center"><I class="parts-info">
                    Figure 1:  Plate image showing FucO basic transformed into DH5α.The medium was LB including Chl 40. 
                </I></DIV><DIV class="col-md-6" style="text-align:center"><I class="parts-info">
                    Figure 2: Plate image showing GSH basic transformed into DH5α colonies. The medium was LB with Chl 40.
                </I></DIV><DIV class="col-md-12" style="text-align:center"><I class="parts-info">
                    Figure 3: BBa_K2571000 (FucO basic) colony PCR check with primers written in parenthesis. For spe. (specific), FucO specific primers are used and expected band length is 194bp. For ori. (orientation), FucO left and VR primers are used and expected band length is 625bp. For NC: dH2O is used.
                </I></DIV><H3>Basic part 1 (FucO) in pSB1C3 (<A href="http://parts.igem.org/Part:BBa_K2571000">BBa_K2571000</A>) PCR Confirmation:</H3><P>
               After colony PCR, we’ve conducted PCR with isolated plasmid from the 4th colony of FucO basic part in DH5 alpha (Figure 4). In the gel image below, the band on the 5th well (starting with ladder) demonstrates our correctly inserted gene in pSB1C3 backbone. In this PCR, we’ve used FucO left and VR primers to check the orientation of our part in backbone. Expected band length to see was 625 bp and the results were obtained.

            </P><DIV class="col-md-12" style="text-align:center"><I class="parts-info">
                    Figure 4: BBa_K2571000 PCR check with FucO left and VR primers. Expected band length:625bp. NC (negative control is dH2O)

                </I></DIV><H3>Basic part 2 (GSH)  in pSB1C3 (<A href="http://parts.igem.org/Part:BBa_K2571001">BBa_K2571001</A>) PCR Confirmation:</H3><P>
                10th well in the gel image below, which belongs to the 8th colony from basic 2 (GSH) cloned DH5α, proves our insertion and transformation right. We’ve used GSH specific primers in PCR and expected to see a band of 225 bp.

            </P><DIV class="col-md-12" style="text-align:center"><I class="parts-info">
                   Figure 5: BBa_K2571001 PCR check with GSH specific primers. Expected band length: 225 bp. Colony no:8 has given the best result. NC is dH2O.
 
                </I></DIV><H3> Clonings of Composite Parts; Only FucO, Only GSH, FucO and GSH Together:  </H3><P>
                In order to be able to grow colonies of DH5α with our first, second and third composite parts; we’ve initially ligated the parts to pSB1C3 backbone. Composite part 1’s 
                insert to vector ratio was 1:3, composite 2’s was 1:2 and composite 3’s was 1:1,5. After that, we’ve transformed the ligations to DH5α competent cells and grew the colonies 
                on LB agar plates containing chloramphenicol at a final concentration of 40 µg/ml. After obtaining DH5α colonies, we’ve done colony PCR and considering the results that we 
                obtained, we’ve decided the best results were from the 3rd colony from composite 1 (Figure 6 and 9), 6th colony from composite 2 (Figure 7) and 3rd from composite 3 (Figure 8).
            </P><DIV class="col-md-4"><I class="parts-info">
                    Figure 6: Plate image showing composite part 1 (FucO) transformed DH5α. The medium was LB with Chl 40.
                </I></DIV><DIV class="col-md-4"><I class="parts-info">
                    Figure 7: Plate image showing composite part 2 (GSH) transformed DH5α. The medium was LB with Chl 40.
                </I></DIV><DIV class="col-md-4"><I class="parts-info">
                    Figure 8: Plate image showing composite part 3 (GSH &amp; FucO) transformed DH5α. The medium was LB with Chl 40.
                </I></DIV><DIV class="col-md-12"><I class="parts-info">
                   Figure 9: BBa_K2571003 (FucO composite) colony PCR with FucO specific primers (FucO left and FucO right). Expected band length is 194bp.
                    C1C3: Composite part 1 (FucO) in pSB1C3 backbone.
                    NC is dH2O.
                </I></DIV><H3>Composite part 1 (FucO) in pSB1C3 (<A href="http://parts.igem.org/Part:BBa_K2571003">BBa_K2571003</A>) PCR Confirmation:</H3><P>
               The gel image below belongs to the PCR confirmation of FucO composite part insertion by using isolated plasmid from colony 3. The PCR we’ve conducted with FucO left and VR primers, using the plasmid as template, was to check orientation of our part in backbone. The expected band length was 754 bp, we decided that 6th and 7th wells (starting with ladder) confirm our transformation. 

            </P><DIV class="col-md-12" style="text-align:center"><I class="parts-info">
                     Figure 10: BBa_K2571003 plasmid PCR check with FucO left and VR primers. Expected band length: 754 bp. NC is dH2O.
                </I></DIV><H3>Composite part 2 (GSH) in pSB1C3 (<A href="http://parts.igem.org/Part:BBa_K2571005">BBa_K2571005</A>) PCR Confirmation:</H3><P>
               We’ve conducted a colony PCR with GSH specific primers to test our transformations of composite part 2 in DH5α. We wanted to see a band of 225 bp on gel and all GSH colonies have given the band as expected. Below, wells from 11 to 17 confirm our transformations were successful and the 6th colony has given the best result.

            </P><DIV class="col-md-12" style="text-align:center"><I class="parts-info">
                    Figure 11: BBa_K2571005 PCR check with GSH specific primers. Expected band length: 225bp.
                </I></DIV><H3>Composite part 3: Dual Insertion of FucO and GSH in pSB1C3 (<A href="http://parts.igem.org/Part:BBa_K2571006">BBa_K2571006</A>) PCR Confirmation:</H3><P>
                
                Our composite part 3 has the genes both FucO and GSH as coding regions and was successfully inserted in the pSB1C3 backbone. After transforming the plasmid to DH5α competent cells, we’ve conducted the colony PCR with FucO specific primers and our expected band length for confirmation was 194 bp (figure 12). All the results seen on the gel (wells 3-11) were positive, proving that our clonings were successful and we have come to the conclusion that the best band was shown by the 3rd colony.

            </P><DIV class="col-md-12" style="text-align:center"><I class="parts-info">
                    Figure 12: BBa_2571006 PCR check with FucO specific primers. Expected band length: 194bp. All of the colonies have given the band.
                    C3C3:Composite part 3 in pSB1C3.
                </I></DIV><H3>E. coli ethanologenic strain KO11 clonings:</H3><P>
                In our biochemical assays, our aim was to see the effects of our parts on ethanol production, cell growth, and lifespan by using KO11. Since KO11 itself had 
                chloramphenicol resistance in its genome, we needed to use another antibiotic resistance. Thus, we inserted our composite parts to pSB1A3 backbone (carrying 
                Ampicillin resistance) as well and did the transformations to KO11.
            </P><P>
               For ligation, first composite part’s (only FucO) insert to vector ratio was 1:3, second part’s only GSH) was 1:2, and the third part’s (Both FucO and GSH) was 1:1,5. 
               We’ve transformed the ligations to KO11 competent cells and grew the colonies on LB agar plates containing chloramphenicol at a final concentration of 150 µg/ml and 
               ampicillin at a final concentration of 100 µg/ml. After obtaining KO11 colonies, we’ve done colony PCR and in the end, we recultured the best result giving colonies 
               for plasmid isolation. Chosen colonies were the 3rd colony for both composite part 1 (figure 13 and 15) and 2 (figure 14); and the 2nd colony for composite part 3.
            </P><DIV class="col-md-6" style="text-align:center"><I class="parts-info">
                   Figure 13: Plate image showing composite part 1 (FucO) and composite part 2 (GSH) transformed 
                </I></DIV><DIV class="col-md-6" style="text-align:center"><I class="parts-info">
                    Figure 14: Plate image showing composite part 3 (FucO and GSH together) transformed KO11. The medium was LB with Chl 150 and Amp 100.
                </I></DIV><DIV class="col-md-12" style="text-align:center"><I class="parts-info">
                    Figure 15: Colony PCR with C1A3: Composite part 1 (FucO) insertion in pSB1A3 backbone. FucO specific primers are used. Expected band length is 194 bp.
                </I></DIV><H3>Composite part 1 (FucO) in pSB1A3 PCR Confirmation with plasmid:</H3><P>
                After plasmid isolation, PCR with FucO left and VR primers was conducted to check the orientation of our composite part 1 to pSB1A3 backbone, and the 
                expected band length for that confirmation was 754 bp (figure 16). 8th well in the image below (obtained from the plasmid isolation of the 3rd colony) 
                confirms the orientation.
            </P><DIV class="col-md-12" style="text-align:center"><I class="parts-info">
                    Figure 16: PCR for composite part 1 in pSB1A3 backbone with FucO left and VR primers. Expected band length is 754bp. 
                </I></DIV><H3>Composite part 2 (GSH) in pSB1A3 PCR Confirmation:</H3><P>
               We’ve conducted colony PCR with GSH specific primers for our composite part 2 in pSB1A3 backbone. We wanted to see a band of 225 bp on gel (figure 17) and GSH colonies 3, 4 &amp; 5 have given the bands as expected. Below; wells 5, 6 and 7 confirm our transformation and we choose to proceed to the plasmid isolation with the colony number 3.

            </P><DIV class="col-md-12" style="text-align:center"><I class="parts-info">
                   Figure 17: PCR check with Composite 2 (GSH) part insertion in pSB1A3 backbone. GSH specific primers are used. Expected band length: 225bp.
                </I></DIV><H3>Composite part 3 (FucO &amp; GSH) in pSB1A3 PCR Confirmation:</H3><P>
                After successfully inserting our composite part 3 (FucO and GSH together) into the pSB1A3 backbone, we’ve done colony PCR with FucO specific primers (Figure 18). Our expected band length for confirmation was 194 bp and all the results came out positive (seen in wells 12-20), confirming our transformation. Since the best result was seen on colony number 2 (well 13), we’ve done the plasmid isolation from that colony.

            </P><DIV class="col-md-12" style="text-align:center"><I class="parts-info">
                   Figure 18: Composite part 3 in pSB1A3 colony PCR check with FucO specific primers. Expected band length: 194 bp. All of the colonies have given the band. (In this image, composite part 3’s insertion in pSB1C3 PCR results are also seen).

                    C3C3:Composite part 3 in pSB1C3.
                    C3A3:Composite part 3 in pS1A3.

                </I></DIV><H3>Biochemical Experiments</H3><P>
               We designed our biochemical characterization experiments in order to evaluate the effects of our circuits on life span, cell mass, and ultimately the bioethanol yield of ethanologenic E. coli strain KO11. We carried out two experimental assays simultaneously. 

            </P><P>
               In both of our biochemical assays, we had four cultured groups of KO11 ethanologenic strains of E.coli to test.

            </P><P>
                               
                #1 KO11 un-engineered
                #2 KO11 with only FucO    
                #3 KO11 with only GSH
                #4 KO11 with Bio-E (FucO and GSH)
            </P><H3>First Assay:</H3><DIV class="col-md-12" style="text-align:center"><I class="parts-info">
                   Figure 19: Representation of our biochemical assay #1
                </I></DIV><P>
               Throughout our first assay, each group was grown in LB broth mediums containing 2% glucose and antibiotics.

            </P><P>
               To culture group #1 (KO11 un-engineered), we only added Chloramphenicol at a final concentration of 40 µg/mL since KO11 un-engineered only had resistance to Chloramphenicol in its genome; and to the mediums of the groups numbered 2, 3 and 4; we added Chloramphenicol at a final concentration of 40 µg/ml and 100 µg/ml Ampicillin. The reason was; groups 2, 3 and 4 had plasmids which carried Ampicillin resistance due to their backbone (pSB1A3). Thus, with the addition of antibiotics to the mediums, selectivity was assured.

            </P><P>
                Cultures were grown overnight, and refreshed in the morning as two sets (First set: 10 mM furfural, 2nd set: 20 mM furfural). After approximately two hours of incubation for both of the sets’ falcon groups (when they reached OD 0,6), furfural was added to their mediums. 

 
            </P><H3> First Set (10 mM furfural): </H3><DIV class="col-md-12" style="text-align:center"><I class="parts-info">
                   Figure 20: Falcon tubes containing the groups for our first assay, set #1.
                </I></DIV><P>
               We added furfural at a final concentration of 10 mM to the first four test groups’ mediums and took OD measurements at Abs 600 nm with 1/10 dilution in 24 hour time intervals.



            </P><P>
                10 mM furfural OD measurements: Abs 600 nm (1/10 dilution):

            </P><TABLE><TBODY><TR><TD>24hrs</TD><TD>48hrs</TD><TD>72hrs</TD></TR><TR><TD>KO11 un-engineered</TD><TD>0.52</TD><TD>0.46</TD><TD>0.40</TD></TR><TR><TD>KO11 with only FucO</TD><TD>0.5</TD><TD>0.5</TD><TD>0.43</TD></TR><TR><TD>KO11 with only GSH</TD><TD>0.6</TD><TD>0.53</TD><TD>0.44</TD></TR><TR><TD>KO11 with Bio-E</TD><TD>0.26</TD><TD>0.45</TD><TD>0.46</TD></TR></TBODY></TABLE><DIV class="col-md-12" style="text-align:center"><I class="parts-info">
                   Figure 21: Graph showing changes in cell mass of assay groups by percentage with respect to the time.

                </I></DIV><H3>Analysis of data:</H3><P>
               Our data demonstrated that group #1 (un-engineered) had a decrease in cell mass throughout the time verifying the inhibition of cell growth in the 
               presence of furfural in the field. Group #2 (KO11 with only FucO) obviously gave better results with respect to un-engineered KO11. However, although 
               the cell mass of the KO11 group with only FucO was stable in the first 48 hours, it was decreased after the 48th hour. This proves that only the presence 
               of the gene FucO in the bacteria wasn’t enough to avoid cell mass decrease in the long term and is in need of another gene for increased tolerance. Also, 
               only GSH’s presence isn’t enough since the group of KO11 with only GSH experienced decrease in cell mass. Group #4 (KO11 with both FucO and GSH) gave 
               measurement results as we hypothesized by continuing cellular growth in the first 48 hours and maintaining it even after the 48th hour, though at a lower rate. 
            </P><H3> Second Set (20 mM furfural):</H3><DIV class="col-md-12" style="text-align:center"><I class="parts-info">
                   Figure 22: Falcon tubes containing the groups for our first assay, set #2.


                </I></DIV><P>   To gather more information to prove our hypothesis, we designed our second experimental set and added furfural at a final concentration of 20 mM to the four test 
              groups’ mediums followed by OD measurements at Abs 600 nm with 1/10 dilution in 24 hour time intervals. 
            </P><P>
                20 mM furfural OD measurements: Abs 600 (1/10 dilution)

            </P><TABLE><TBODY><TR><TD>24hrs</TD><TD>48hrs</TD></TR><TR><TD>Un-engineered KO11</TD><TD>0.67</TD><TD>0.54</TD></TR><TR><TD>KO11 with FucO</TD><TD>0.54</TD><TD>0.43</TD></TR><TR><TD>KO11 with GSH</TD><TD>0.61</TD><TD>0.60</TD></TR><TR><TD>KO11 with Bio-E</TD><TD>0.29</TD><TD>0.55</TD></TR></TBODY></TABLE><P>
                For our second set, we could only obtain the measurements of the first 48 hours since we faced contamination in the last day of wet-lab and we had no more time. 
                Thus, we modelled our second experimental set’s results by demonstrating the comparison of OD results at absorbance 600.
            </P><DIV class="col-md-12" style="text-align:center"><I class="parts-info">
                  Figure 23: Graph showing change in OD of assay groups with respect to the time.
                </I></DIV><H3>Analysis of Data</H3><P>
                    Our data demonstrated that group #1 (KO11 un-engineered) had a decrease in cell mass as time passed. Group #2 (KO11 with only FucO) also experienced a decrease 
                    in cell mass. Group #3 (KO11 with only GSH) gave better results with respect to both of the groups #1 and #2 by maintaining its cell mass stable. Group #4 (KO11 
                    with both FucO and GSH) gave the most promising measurement data as we hypothesized by continuing cellular growth (almost doubling cell mass) in the first 48 hours. 
                </P><H3>Second Assay:</H3><P>
                    For our second characterization, we followed more of qualitative evaluation to see the inhibitory zone of furfural. Firstly, we prepared a solution of furfural at 
                    a final concentration of 20 mM by diluting the stock solution with distilled H2O. Then, we soaked filter paper discs in that solution and placed them on LB agar 
                    plates hosting the four groups of our assay. 
                </P><DIV class="col-md-12" style="text-align:center"><I class="parts-info">
                  Figure 24: Image of the plates right after filter papers soaked in furfural (20 mM) were placed.
                </I></DIV><P>
                After 48 hours, we’ve observed a clear zone around the filter paper of the group containing KO11 un-engineered while others weren’t inhibited as much.
            </P><DIV class="col-md-3" style="text-align:center"><I class="parts-info">
                  Figure 25: Plate having group KO11 un-engineered.
                </I></DIV><DIV class="col-md-3" style="text-align:center"><I class="parts-info">
                  Figure 26 Plate having group KO11 with only FucO.
                </I></DIV><DIV class="col-md-3" style="text-align:center"><I class="parts-info">
                  Figure 27: Plate having group KO11 with only GSH.
                </I></DIV><DIV class="col-md-3" style="text-align:center"><I class="parts-info">
                  Figure 28: Plate having group KO11 with Bio-E.
                </I></DIV><H3> Conclusion: </H3><P>
                When the quantitative measurement data and qualitative phenotypic evaluation for all of our biochemical assays are considered, we can conclude that the groups containing KO11 un-engineered are the weakest ones against furfural toxicity; and neither the group of KO11 with only FucO nor the group with only GSH is resistant enough to continue cellular growth when furfural is present in the medium. Out of four groups, only the group containing KO11 with Bio-E (both FucO and GSH) can sustain its cellular growth and survive. Overall, we can infer that our best part design (Bio-E) was successful enough to combat the inhibitive effects of furfural, indicating trueness of our hypothesis.

            </P></DIV></SECTION><DIV class="container ct-u-paddingBoth50"><DIV class="row ct-u-borderLeft"><DIV class="col-sm-6 col-md-4 ct-u-paddingBoth30"><H5 class="text-uppercase ct-fw-600"> contact Us</H5><SPAN class="text-uppercase ct-fw-600">email</SPAN><A href="mailto:metuhsigemteam@gmail.com" class="text-lowercase ct-u-underline ct-fw-600">metuhsigemteam@gmail.com</A><SPAN class="text-uppercase ct-fw-600">instagram</SPAN><A href="#" class="text-lowercase ct-u-underline ct-fw-600">@metuhsigem</A></DIV></DIV></DIV><DIV class="ct-footerBottom"><DIV class="container"><DIV class="row"><DIV class="col-sm-6"><P>COPYRIGHT © <A href="https://2018.igem.org/Team:METU_HS_Ankara">METU HS IGEM</A> 2018</P></DIV></DIV></DIV></DIV></BODY></HTML>