<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_METU_HS_Ankara_Composite_Part skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:METU HS Ankara/Composite Part</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>METU HS IGEM</TITLE></P><DIV id="nedim" class="mila cssAnimate ct-navbar--fixedTop ct-js-navbarMakeSmaller ct-js-enableWayPoints"><DIV class="ct-navbarMobile ct-navbarMobile--inverse"><BUTTON type="button" class="navbar-toggle"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="ct-menuMobile"><UL class="ct-menuMobile-navbar"><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara">Home </A></LI><LI class="dropdown"><A href="#">Team </A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Team"> Team Members</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Collaborations"> Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Attributions"> Attributions</A></LI></UL></DIV></DIV></DIV><LI class="dropdown"><A href="#">Project </A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Description"> Description</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Design"> Design</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Experiments"> Experiments</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Results"> Results</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Model"> Model</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Notebook"> Notebook</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/InterLab"> InterLab</A></LI></DIV></DIV></DIV><LI class="dropdown"><A href="#">Parts </A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Basic_Part"> Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Composite_Part"> Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Part_Collection"> Parts Collection</A></LI></DIV><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Demonstrate">Demonstrate </A></LI><LI class="dropdown"><A href="#">HP </A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Human_Practices"> Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Public_Engagement"> Education &amp; Engagement</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Safety">Safety </A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Achievements">Achievements </A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Gallery">Gallery </A></LI><DIV id="ct-js-wrapper" class="ct-pageWrapper"><NAV class="navbar is-small navbar-inverse ct-navbar--logoright" role="navigation"><DIV class="container"><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav ct-navbarNav--defaultActive ct-navbar--fadeIn yamm"><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara">Home </A></LI><LI class="dropdown"><A href="#">Team </A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Team"> Team Members</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Collaborations"> Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Attributions"> Attributions</A></LI></UL></DIV></DIV></NAV></DIV><LI class="dropdown"><A href="#">Project </A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Description"> Description</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Design"> Design</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Experiments"> Experiments</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Results"> Results</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Model"> Model</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Notebook"> Notebook</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/InterLab"> InterLab</A></LI><LI class="dropdown"><A href="#">Parts </A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Basic_Part"> Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Composite_Part"> Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Part_Collection"> Parts Collection</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Demonstrate">Demonstrate </A></LI><LI class="dropdown"><A href="#">HP </A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Human_Practices"> Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Public_Engagement"> Education &amp; Engagement</A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Safety">Safety </A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Achievements">Achievements </A></LI><LI><A href="https://2018.igem.org/Team:METU_HS_Ankara/Gallery">Gallery </A></LI><HEADER class="ct-pageHeader ct-pageHeader--type2 ct-u-shadowBottom--type2 ct-pageHeader--motive ct-pageHeader--hasDescription ct-u-paddingBoth10"><DIV class="container ct-u-triangleBottomLeft"><DIV class="row"><DIV class="col-md-12"><H1 class="text-capitalize ct-fw-600 ct-u-colorWhite">
                        Composite Parts
                    </H1></DIV></DIV></DIV></HEADER><SECTION class="ct-u-paddingBoth50"><DIV class="container"><TABLE class="table" style="font-size: 17px"><THEAD><TR><TH>Name</TH><TH>Type</TH><TH>Description</TH><TH>Designer</TH><TH>Length</TH></TR></THEAD><TBODY><TR class="danger"><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2571003">BBa_K2571003</A></TD><TD>FucO / L-1,2-propanediol oxidoreductase</TD><TD>Tugba Inanc &amp; Ceyhun Kayihan</TD><TD>1350 bp</TD></TR><TR class="warning" style="font-size: 17px; color: #000000"><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2571005">BBa_K2571005</A></TD><TD>GSH/ Bifunctional gamma-glutamate-cysteine ligase/Glutathione synthetase</TD><TD>Tugba Inanc &amp; Ceyhun Kayihan</TD><TD>2466 bp</TD></TR><TR class="info"><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2571006">BBa_K2571006</A></TD><TD>Dual Expression of FucO and GSH</TD><TD>Tugba Inanc &amp; Ceyhun Kayihan</TD><TD>3644 bp</TD></TR></TBODY></TABLE><H3>Composite Part 1:</H3><H4>FucO/ L-1,2-Propanediol Oxidoreductase</H4><P>
                FucO is the gene that codes for L-1,2-propanediol oxidoreductase which is a NADH-linked, homodimer enzyme having the role of 
                acting on furfural which is a toxic inhibitor of microbial fermentations causing cell wall and membrane damage, DNA breakdowns 
                and reduced enzymatic activities (Zheng, 2013; Liu, Ma &amp; Song, 2009).
            </P><P>
                The enzyme catalyzes L-lactaldehyde and L-1,2- propanediol while dissimilating fucose in which acetaldehyde, ethylene glycerol, 
                L-lactaldehyde and some more substances are used as substrates. Despite these, it takes an important role in furan reduction 
                to its alcohol derivative (Wang <I>et al.</I>, 2011).
            </P><H5>Our circuit design for FucO gene</H5><P>
                Our circuit consists of prefix, a strong promoter (J23100), RBS (B0034), FucO as protein coding region, double terminator (B0015) 
                and suffix. This part enables our <I>E. coli</I> KO11 strain to convert toxic furfural into furfuryl alcohol. Our construct was inserted 
                into pSB1C3 and delivered to the Registry.
            </P><I class="parts-info">
                Figure 1: Circuit design of Composite part 1 with FucO gene. <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2571003">BBa_K2571003.</A> 
                Our construct includes a strong promoter,  RBS, FucO and double terminator. 
            </I><P>
                    In order to make our gene compatible with RFC 10, 25 and 1000, we reconstructed the nucleotides to get rid of the restriction sites while protecting 
                    the amino acid sequence. We looked through the codon bias property of <I>E. coli</I> and made the nucleotide changes accordingly. 
            </P><P>
                FucO has NADH-dependent furan reductase activity. When furfural is present in the field, the metabolism of furfural by NADPH-dependent oxidoreductases 
                goes active in order to reduce it to its less toxic alcohol derivative-furfuryl alcohol (Zheng, 2013; Wang <I>et al.</I>, 2013; Allen <I>et al.</I>, 2010). 
            </P><I class="parts-info">
                Figure 2: Effect of FucO overexpression in LY180 (Wang <I>et al.</I>, 2011). The cell mass was observed in furfural containing medium. The FucO gene expressing 
                L-1,2-propanediol oxidoreductase reduces the effect of furfural. The specific death rate of normal bacteria is observed to be higher than the specific 
                death rate of bacteria with FucO gene. Thus, FucO is shown to increase the tolerance and lifespan of bacteria.
            </I><P>
                In this metabolism, the expression of oxidoreductases that are NADPH-dependent, such as YqhD, are shown to inhibit the growth and fermentation in <I>E. coli</I> 
                by competing for biosynthesis with NADPH (Zheng, 2013).
            </P><I class="parts-info">
                Figure 3: The overexpression of FucO and YqhD and relationships with furfural resistance traits, metabolism, and reducing cofactors (Wang <I>et al.</I>, 2013).
            </I><P>
                Because the native conversion of NADH to NADPH in <I>E. coli</I> is insufficient to revitalize the NADPH pool during fermentation, the actions shouldn’t be 
                interfering with NADPH metabolism (Wang <I>et al.</I>, 2011). Thus, the overexpression of plasmid-based NADH-dependent propanediol oxidoreductase (FucO) gene 
                reduces furfural to ultimately improve furfural resistance without detrimentally affecting the biosynthesis of NADPH (Wang <I>et al.</I>, 2011).
            </P><I class="parts-info" style="margin-bottom: 20px">
                Figure 4: 3D protein structure of L-1,2-propanediol oxidoreductase.
            </I><P>
                The protein structure of L-1,2-propanediol oxidoreductase was constructed by using Amber 14. It is demonstrated in ribbon diagram which is done by interpolating a smooth curve through the polypeptide backbone. The colors indicate the amino acids in the protein structure.
            </P><DIV class="col-md-6" style="margin-bottom: 30px"><I class="parts-info" style="line-height: 0px !important">
                    Figure 5: BBa_K2571003 check with FucO left and VR primers. Expected band length: 754 bp. Last three wells show positive results. 
                </I></DIV><DIV class="col-md-6"><P>
                    We’ve inserted the FucO composite part to pSB1C3 and pSB1A3 backbones. Then, we’ve transformed the construct for submission, 
                    <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2571003">BBa_K2571003</A>, (in pSB1C3) 
                    to DH5⍺; and the other construct, for our biochemical assay, (in pSB1A3) to KO11. As we isolated the plasmids, we’ve done PCR with FucO left and VR 
                    primers to test orientation of our parts to the backbone. We expected a band of 754 bp between the FucO left and VR primers and the PCR results confirmed 
                    our expectations and showed that our parts were correctly inserted and transformed.
                </P></DIV><P>
                FucO Left and VR primers are as below:
                
                FucO Left: GTGATAAGGATGCCGGAGAA
                
                VR: ATTACCGCCTTTGAGTGAGC
                    
            </P><H3>Composite 2:</H3><H4>GSH:Bifunctional gamma-glutamate-cysteine ligase/Glutathione synthetase</H4><P>
                Reactive Oxygen Species (ROS) are dangerous substances that distort protein based matters by taking electrons (Lu, 2013). The chemical structure of the protein-based 
                substances are altered and become dysfunctional because of ROS (Lu, 2013; Burton &amp; Jauniaux, 2011).
            </P><P>
                Furthermore; one of the most significant protein-based substance, DNA, gets attacked by OH radicals (Burton &amp; Jauniaux, 2011). However, the reduced form GSH can protect 
                the chemical structure of the proteins by donating extra electrons to ROS and free radicals (Lu, 2013). This is accomplished by GSH peroxidase-catalyzed reactions 
                (Lu, 2013).
            </P><I class="parts-info">
                Figure 6: 3D protein structure of Bifunctional gamma-glutamate-cysteine ligase.
            </I><P>
                The protein structure of Bifunctional gamma-glutamate-cysteine ligase was constructed by using Amber 14. It is demonstrated in ribbon diagram which is done by interpolating a smooth curve through the polypeptide backbone. The colors indicate the amino acids in the protein structure.
            </P><H5>Our circuit design for GSH gene</H5><P>
                Our circuit consists of prefix, a strong promoter (J23100), RBS (B0034), GSH as protein coding region, double terminator (B0015) and suffix. This part enables our <I>E. coli</I> KO11 strain to overexpress oxidised Glutathione to reduce oxidative stress, increasing its lifespan (Lu, 2013). Our construct was inserted into pSB1C3 and 
                delivered to the Registry.
            </P><I class="parts-info">
                Figure 7: Circuit design of Composite part 2 with GSH gene. <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2571005">BBa_K2571005</A>. Our construct 
                includes a strong promoter,  RBS, GSH and double terminator. 
            </I><P>
                In order to make our gene compatible with RFC 10, 25 and 1000, we reconstructed the nucleotides to get rid of the restriction sites while protecting the amino acid 
                sequence. We looked through the codon bias property of <I>E. coli</I> and made the nucleotide changes accordingly. 
            </P><I class="parts-info">
                Figure 8: Because Glutathione prevents ROS from harming the bacteria, increase in cell mas was observed in high concentrations of Glutathione. In brief, when 
                Glutathione concentration increases, the specific cell growth rate also increases and we observe an increase in the number of bacteria compared to the bacteria without 
                the GSH gene (Kim &amp; Hahn , 2013).  
            </I><DIV class="col-md-6"><I class="parts-info">
                    Figure 9: <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2571005">BBa_K2571005</A> check with GSH specific  primers. Expected band length: 225 bp. 
                    Last six wells show positive results.  
                </I></DIV><DIV class="col-md-6"><P>
                    We’ve inserted the GSH composite part to pSB1C3 backbone. Then, we’ve transformed the construct for submission, 
                    <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2571005">BBa_K2571005</A>, (in pSB1C3) 
                    to Dh4 alpha and conducted colony PCR. We’ve prepared the PCR with GSH specific primers and expected to see a result of 225 bp. By showing the 
                    band we expected, 225 bp, PCR confirmation for our insertion proved right.
                </P></DIV><P>
                GSH left and right primers are shown as below:
                
                GSH left: TCGGAGGCTAAAACTCAGGA
                
                GSH right: GTGGGCAGTCCAGTCGTAAT
            </P><H3>Composite 3:</H3><H4>Dual Expression of FucO and GSH</H4><P>
                The first protein coding region we have, placed after the RBS, FucO, will code for L-1,2-propanediol oxidoreductase (a homodimer enzyme) 
                in order to act upon furfural presence in the field (Zheng, 2013). The metabolism of furfural by NAD(P)H-dependent oxidoreductases will 
                reduce the toxicity of the chemical by turning it into a derivative, furfuryl alcohol, and increase the furfural tolerance (Zheng, 2013; 
                Wang <I>et al.</I>, 2013; Allen <I>et al.</I>, 2010). Our second protein coding region, bifunctional gamma-glutamate-cysteine ligase/Glutathione 
                synthetase (GSH), is a non-protein thiol group and a tripeptide composed of cysteine, glycine and glutamic acid (Lu, 2013). It is crucial 
                for the detoxification of reactive oxygen species and free radicals (Ask <I>et al.</I> 2013). Reactive oxygen species (ROS) are harmful substances 
                that alter protein based matters by taking electrons (Lu, 2013; Burton &amp; Jauniaux, 2011). Because many benefits of GSH include scavenging 
                of ROS, protection against endogenous toxic metabolites and detoxification of xenobiotics, we choose this gene to integrate with FucO 
                (Höck <I>et al.</I>, 2013). Thus we constructed a multi-functional gene providing long life span and resistance.
            </P><H4>Design Notes of Dual Expression of FucO and GSH <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2571006">(BBa_K2571006)</A></H4><P>
                Our construct for composite part 3 is composed of two stages, first the reduction of furans (specifically furfural and 5-HMF)  and second the 
                detoxification of reactive oxygen species (ROS). To achieve this effect, we designed our composite 3 part as with a prefix, a strong promoter 
                (J23100), RBS (B0034), FucO as the first protein coding region <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2571003">(BBa_K2571003)</A>, 
                RBS (B0034), GSH as the second protein coding region <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2571005">(BBa_K2571005)</A>, 
                double terminator (B0015) and suffix.
            </P><I class="parts-info">
                Figure 10: Circuit design of Composite part 3 with FucO and GSH genes. <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2571006">BBa_K2571006</A>.
                Our construct includes a strong promoter, RBS, FucO, RBS, GSH and double terminator. 
            </I><P>
                Our construct was inserted into pSB1C3 and delivered to the Registry. Our construct was also inserted into pSB1A3 and transferred into KO11 to 
                conduct further biochemical assays. 
            </P><P>
                Given that FucO is NADH-dependent, it outperforms other oxidoreductases by not interfering with the NADPH metabolism of the organism while converting highly 
                toxic substances, furfural and 5-HMF to non-harmful alcohols. This characteristic of FucO is crucial because the expression of oxidoreductases like Yqhd are 
                NADPH-dependent, hence they compete with the biosynthesis for NADPH, which results in inhibiting the growth of the organism.
            </P><P>
                Glutathione, on the other hand, is recycled using NAD(P)H pathways and since its over-expression with NADH metabolism not being altered thanks 
                to FucO, antioxidant capacity of the cell will be increased dramatically; resulting in amplified immunity to both furans and ROS, habilitated cell growth and 
                increased ethanol yield by the virtue of increasing cell mass and reproduction, and improved metabolism.
            </P><H3>Gel Results</H3><I class="parts-info">
                Figure 11: BBa_K2571006 check with GSH and FucO specific  primers. Expected band length: 194 bp. Green boxes show positive results. 
            </I><P>
                We’ve inserted our composite part 3(BBa_K2571006) in both pSB1C3 and pSB1A3 backbones. The construct in pSB1C3 is for submission to registry and is  cultivated 
                in DH5 alpha. The plasmid having pSB1A3 as backbone, thus carrying ampicillin resistance is for our biochemical assays since we’ve chosen the chassis organism 
                for assays as E.coli strain KO11 which already has Chloramphenicol resistance in its genome.  After cloning our genes, we’ve made colony PCR to verify our insertions. 
                We chose the primers as FucO specific since the composite 3 contains FucO coding region. Expected band length was 194 bp, and as expected, the bands were given by all 
                of the DH5 alpha and KO11 colonies we chose, confirming our transformations.
            </P><P>
                FucO specific primers were used:
                FucO left: GTGATAAGGATGCCGGAGAA
                FucO right: CTTCTCGCCGGTAAAGTCAG</P><DIV class="container"><DIV class="row"><DIV class="col-md-12"><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseOne">
                                                References
                                            </A></H4></DIV><DIV id="collapseOne" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI><STRONG>Allen, S. A., Clark, W., McCaffery, J. M., Cai, Z., Lanctot, A., Slininger, P. J., Gorsich, S. W.</STRONG>
                                                    (2010). Furfural induces reactive oxygen species accumulation and cellular damage in Saccharomyces cerevisiae. Biotechnology for Biofuels, 3, 2.
                                                    <A href="http://doi.org/10.1186/1754-6834-3-2">http://doi.org/10.1186/1754-6834-3-2</A></LI><LI><STRONG>Burton, G. J., &amp; Jauniaux, E.</STRONG>
                                                    (2011). Oxidative stress. Best Practice &amp; Research. Clinical Obstetrics &amp; Gynaecology, 25(3), 287–299.
                                                    <A href="http://doi.org/10.1016/j.bpobgyn.2010.10.016">http://doi.org/10.1016/j.bpobgyn.2010.10.016</A></LI><LI><STRONG>Chou, H.-H., Marx, C. J., &amp; Sauer, U.</STRONG>
                                                    (2015). Transhydrogenase Promotes the Robustness and Evolvability of E. coli Deficient in NADPH Production. PLoS Genetics, 11(2), e1005007.
                                                    <A href="http://doi.org/10.1371/journal.pgen.1005007">http://doi.org/10.1371/journal.pgen.1005007</A></LI><LI><STRONG>Lu, S. C.</STRONG>
                                                    (2013). Glutathione Synthesis. Biochemica et Biophysica Acta, 1830(5), 3143–3153.
                                                    <A href="http://doi.org/10.1016/j.bbagen.2012.09.008">http://doi.org/10.1016/j.bbagen.2012.09.008</A></LI><LI><STRONG>Liu, Z.L., Ma M., Song, M.</STRONG>
                                                    (2009). Evolutionarily engineered ethanologenic yeast detoxifies lignocellulosic biomass conversion inhibitors by reprogrammed pathways. Mol Genet Genomics 282, 233-244.
                                                    <A href="http://doi.org/10.1007/s00438-009-0461-7">http://doi.org/10.1007/s00438-009-0461-7</A></LI><LI><STRONG>National Center for Biotechnology Information.</STRONG>
                                                    PubChem Compound Database; CID=124886,
                                                    <A href="https://pubchem.ncbi.nlm.nih.gov/compound/124886">https://pubchem.ncbi.nlm.nih.gov/compound/124886</A>
                                                    (accessed July 18, 2018).
                                                    <A href="https://pubchem.ncbi.nlm.nih.gov/compound/124886#section=Top">https://pubchem.ncbi.nlm.nih.gov/compound/124886#section=Top</A></LI><LI><STRONG>Pizzorno, J.</STRONG>
                                                    (2014). Glutathione! Integrative Medicine: A Clinician’s Journal, 13(1), 8–12.
                                                    <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4684116/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4684116/</A></LI><LI><STRONG>Wang, X., Miller, E. N., Yomano, L. P., Zhang, X., Shanmugam, K. T., &amp; Ingram, L. O.</STRONG>
                                                    (2011). Increased Furfural Tolerance Due to Overexpression of NADH-Dependent Oxidoreductase FucO in Escherichia coli Strains Engineered 
                                                    for the Production of Ethanol and Lactate. Applied and Environmental Microbiology, 77(15), 5132–5140.
                                                    <A href="http://doi.org/10.1128/AEM.05008-11">http://doi.org/10.1128/AEM.05008-11</A></LI><LI><STRONG>Wang, X &amp; N Miller, E &amp; Yomano, Lorraine &amp; Zhang, Xueli &amp; T Shanmugam, K &amp; Ingram, Lonnie.</STRONG>
                                                    ((2011). Increased Furfural Tolerance Due to Overexpression of NADH-Dependent Oxidoreductase FucO in Escherichia coli Strains Engineered 
                                                    for the Production of Ethanol and Lactate. Applied and environmental microbiology.
                                                </LI><LI><STRONG>Xuan Wang, Lorraine P. Yomano, James Y. Lee, Sean W. York, Huabao Zheng,Michael T. Mullinnix, K. T. Shanmugam, and Lonnie O. Ingram,</STRONG>
                                                    (2013), Engineering furfural tolerance in Escherichia coli improves the fermentation of lignocellulosic sugars into renewable chemicals.
                                                    <A href="https://doi.org/10.1073/pnas.1217958110">https://doi.org/10.1073/pnas.1217958110</A></LI><LI><STRONG>Zheng, H., Wang, X., Yomano, L.P., Geddes, R. D, Shanmugan, K. T., Ingram, L.O.</STRONG>
                                                    (2013). Improving Escherichia coli FucO for Furfural Tolerance by Saturation Mutagenesis of Individual Amino Acid Positions. 
                                                    Applied and Environmental Microbiology Vol 79, no 10.3202–3208.
                                                    <A href="http://aem.asm.org/content/79/10/3202.full.pdf+html">http://aem.asm.org/content/79/10/3202.full.pdf+html</A></LI><LI><STRONG>Kim, D., &amp; Hahn, J.-S.</STRONG>
                                                    (2013). Roles of the Yap1 Transcription Factor and Antioxidants in Saccharomyces cerevisiae’s Tolerance to Furfural and 5-Hydroxymethylfurfural, 
                                                    Which Function as Thiol-Reactive Electrophiles Generating Oxidative Stress. Applied and Environmental Microbiology, 79(16), 5069–5077.
          <A href="http://doi.org/10.1128/AEM.00643-13">http://doi.org/10.1128/AEM.00643-13</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></SECTION><DIV class="container ct-u-paddingBoth50"><DIV class="row ct-u-borderLeft"><DIV class="col-sm-6 col-md-4 ct-u-paddingBoth30"><H5 class="text-uppercase ct-fw-600"> contact Us</H5><SPAN class="text-uppercase ct-fw-600">email</SPAN><A href="mailto:metuhsigemteam@gmail.com" class="text-lowercase ct-u-underline ct-fw-600">metuhsigemteam@gmail.com</A><SPAN class="text-uppercase ct-fw-600">instagram</SPAN><A href="#" class="text-lowercase ct-u-underline ct-fw-600">@metuhsigem</A></DIV></DIV></DIV><DIV class="ct-footerBottom"><DIV class="container"><DIV class="row"><DIV class="col-sm-6"><P>COPYRIGHT © <A href="https://2018.igem.org/Team:METU_HS_Ankara">METU HS IGEM</A> 2018</P></DIV></DIV></DIV></DIV></BODY></HTML>