"
Page
Discussion
View source
History
teams
Log in
 
Team:Gaston Day School/biofuels
From 2014.igem.org
Welcome to the 2014 GDS iGEM Wiki
>Projects>Biofuels
Home
Members
Project
Parts
Models
Notebook
Safety
Attributions
Biofuels Project Overview
The rising cost of petroleum, coupled with the environmental concerns that go with its use, have led to a recent increase in biofuel research.  With this research has come developments in using bacteria to produce biofuels such as isobutanol and ethanol. These alcohols are favored because they can easily be swapped into our current infrastructure of car and truck engines. At Gaston Day School, we have decided to launch a biofuel-focused project. To create the alcohols, we are developing alcohol-resistant strains of E. coli through artificial selection. Also, we are using PCR to amplify and clone the genes GlmZ, GlmY, and IlmV, which are used in native alcohol production or increased isobutanol resistance. The combination of these genes and the alcohol resistant strains are the first steps in our new biofuels project.
Special Issues with an Isobutanol Resistant Strain of E. coli
Any bacterial strain that is resistant to an alcohol will have to be handled carefully. While isobutanol is not currently used as a cleaner, several other alcohols are. If there is cross-reactivity between the resistance pathways for isobutanol resistance and resistance to ethanol or to isopropanol, we could have a strain that is impervious to several standard cleaning methods. Atsugi, et al. (2010) stated that they did not see any decrease in sensitivity to ethanol in the isobutanol resistant strain developed in their lab. Studies are currently underway to determine if our isobutanol resistant strain shows any change in resistance to ethanol or to isopropanol. Cleaning procedures in the lab have been changed so that surfaces are decontaminated with 10% bleach since there is a possibility that the strain we are working with may be resistant to other alcohols.
If this strain becomes commercially useful, several options exist for decreasing the risk if the strain is accidentally released. Ideally, a kill switch can be incorporated in the strain that requires the presence of a suppressor molecule in the medium. If the strain is released, the suppressor will not be present which will activate the kill switch and prevent the released bacterium from surviving. Several variations of kill switches can be found in the Biobrick registry. K176036 is a Tetracycline repressible construct that can kill cells, for example. As in our lab, cleaning protocols will have to be adjusted to account for a possibly ethanol resistant bacterial strain. Rather than wiping hands and surfaces with hand sanitizer and/or alcohol, hands should be washed with soap and water and surfaces decontaminated with 10% bleach. Studies done by the Gaston Day School 2009 team showed that the kill rate of 10% bleach is very rapid and almost 100% effective.
Tests are underway in the lab to determine the sensitivity of our isobutanol strain to ethanol and isopropanol. Plans are also being made to incorporate some form of repressible kill switch into the strain.
Biofuels PCR Protocol
Use X PCR tubes according to the amount of samples
Use 1 MCF tube for a master mix
Using the master mix recipe, create enough mix for the amount of samples you plan to run. Make a little more just to be safe
(if you plan to run 3 samples, multiply the master mix volumes by 4). Also, we added taq at a later step to prevent it from
being exposed to heat.
To make the master mix, per PCR tube, use:
5ul Mg-free PCR buffer supplied with enzyme
1.5ul MgCl2
1ul dNTPs
0.5ul taq*
5ul Genomic DNA
32ul dH2O
*We used taq from New England Biolabs to achieve the best results.
Place the PCR tubes and the master mix tube in an ice rack
Label the tubes with 1 being the control and all subsequent numbers being the test samples
For the control, add 5ul dH2O and no primers to the first PCR tube
For the test samples, add 4ul dH2O, 0.5ul of the appropriate forward primer, and 0.5ul of the appropriate reverse primer to
the remaining PCR tubes, with one tube per set of primers.
Add the taq to the master mix.
Add 45ul of the master mix to each PCR tube
Run the tubes in a PCR machine with the following inputs
                                    Degrees Celsius                    Time (seconds)
Initial Denaturation                           95                    900
Denaturation                                    95                    30
Annealing                                         58                    30
Extension                                         68                    120
Final Extension                                68                    1200
Run for 40 cycles
Reference:
Atsugi, S., Wu, T., Machado, I. M., Huang, W., Chen, P., Pellegrini, M., and Liao, J.C., (2010). Evolution, Genomic Analysis, and Reconstruction of Isobutanol Tolerance in E. coli. Molec. Says. Bio 6:449.
Retrieved from " http://2014.igem.org/Team:Gaston_Day_School/biofuels "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
