"
Page
Discussion
View source
History
teams
Log in
 
Team:Lethbridge/project
From 2013.igem.org
No Frameshifting with Continued Translation                                             Frameshifting into -1 Frame
Project
FRAMEchanger – A new tool for regulating gene expression
Goal – To develop a new tool for regulation of gene expression for the parts registry
As the Registry of Standard Biological Parts expands, there are more options becoming available for tight regulation of gene expression. For example, there are a number of well characterized promoters and ribosomal binding sites, each with a distinctive induction pattern or strength. However, biological systems use more than just these two types of elements to control gene expression. The goal of the 2013 Lethbridge iGEM team was to create a new type of regulatory part for the iGEM community. This new class of part could allow synthetic biologists to:
Encode twice the amount of protein sequence in DNA
Tag the same protein in multiple different ways at a predetermined frequency
Express operons at their appropriate ratios in non-native organisms
And more!
You may be wondering, “What kind of part can do all of these things, and how does it work?” Let us introduce you to the RNA secondary structural element called a pseudoknot. This small but highly structured RNA element is capable of inducing ribosomal frameshifts during translation that change the reading frame co-translationally. We have constructed and characterized the first BioBrick part  ( BBa_K1210000 ) that uses an RNA pseudoknot to cause a programmed –1 ribosomal frameshift. While this part has been optimized for use in prokaryotic systems, a small adjustment can be made (see below, "The Spacer") to allow for this element to be used in eukaryotic systems as well. In this way, we have created a new tool for gene regulation in essentially any cell chassis of your choice.
Pseudoknots
Pseudoknots are one of the most common RNA secondary structural elements and are sometimes used in nature to induce ribosomal frameshifts. These elements are generally comprised of two helical regions connected by and interacting with adjacent single-stranded regions or loops. Besides the knot itself, frameshifting also requires a slippery sequence of repeating A’s or U’s and a spacer sequence between the slippery sequence and the pseudoknot (Fig. 1).
Figure 1. Pseudoknot frameshifting element. To induce frameshifting, three elements are needed: a slippery sequence (underlined in blue); a spacer region (underlined in yellow); and a secondary structural element, typically a pseudoknot structure (boxed in green). The sequence shown is adapted from the avian coronavirus infectious bronchitis virus pseudoknot (Hansen et al., Proc Natl Acad Sci, 2007).
The Slippery Sequence
Ribosomal frameshifting occurs at the slippery sequence, which is a heptanucleotide region with the sequence 5’-X XXY YYZ-3’, where X, Y, and Z refer to different nucleotides and spaces denote the codons in the initial reading frame. In the original 0 frame, the P-site and A-site tRNAs will be paired with the XXY and YYZ codons, respectively. When the –1 frameshift occurs, the same tRNAs will now be paired with the XXX and YYY codons of the mRNA by forming base pairs with the first two nucleotides of each codon. The slippery sequence used in our construct has the sequence 5’-U UUA AAG-3’, which has been optimized from the native IBV sequence (U UUA AAC) to be a more efficient slippery sequence in Escherichia coli (Hansen et al., Proc Natl Acad Sci, 2007).
The Spacer
In order to correctly position the ribosome at the slippery site to induce frameshifting, a spacer region of 6-9 nucleotides is required just downstream of the slippery sequence. The length of the spacer can be adjusted to accommodate either the 70S or 80S ribosome, depending on the host organism. We are using a 6-nucleotide spacer region with the sequence 5’-CAGAAA-3’.
The FRAMEchanger pseudoknot
The final element needed to cause a ribosomal frameshift is an RNA secondary structure directly downstream of the slippery sequence and the spacer. This structure is most often a pseudoknot, which is a collection of loops and stems that have specific hydrogen bonding and base pairing interactions. The type of pseudoknot used in our project is an H-type pseudoknot derived from the avian coronavirus infectious bronchitis virus (IBV). The fold of H-type pseudoknots is such that there are two stems that are stacked on top of each other connected by two loop regions. This results in a quasi-continuous helix that has one continuous strand and one discontinuous strand (Fig. 1). The intramolecular strength of the stems of the pseudoknot can influence the frequency of frameshifting, and therefore changes to the pseudoknot sequence can increase or decrease the amount of frameshifting that occurs on a particular mRNA transcript.
Applications for Pseudoknot-Induced Frameshifting
The exact mechanism of frameshifting is not fully understood. It is thought that, while decoding the slippery sequence, the ribosome encounters the pseudoknot and pauses. This pause can cause the ribosome to “slip” backwards by one nucleotide in the slippery sequence (while maintaining correct anticodon-codon pairing with the A- and P-site tRNAs) and then continue translating in the –1 reading frame (Staple and Butcher, PLoS Biol, 2005). Alternatively, the pseudoknot could be acting as a roadblock, becoming wedged in the entrance of the ribosome and building up tension on the mRNA during tRNA accommodation. This tension could then be relieved by melting of the pseudoknot structure, slippage of the ribosome backwards by one nucleotide, or by a combination of both methods (Hansen et al., Proc Natl Acad Sci, 2007). The fraction of ribosomes that change reading frame after pausing at the pseudoknot correlates to the frameshifting frequency of that particular pseudoknot. By using pseudoknots of different stability (i.e. with different frameshift frequencies), a variety of applications can be envisioned that make use of this regulatory element.
Dual-coding
Viruses often use programmed ribosomal frameshifting to reduce their genome size by dual-coding some of their genes. Many iGEM projects require the use of more than one coding sequence, and cloning strategies can quickly become inefficient, time consuming, or unfeasible for these multi-coding sequence constructs. By utilizing duel-coding sequences downstream of a pseudoknot, the ORFs of two proteins can be overlapped and placed in different reading frames, thereby reducing the coding space required to produce a construct.
Variable Tagging
A frameshift can also be used to add different tags to a protein during translation. Placing a pseudoknot downstream of a protein coding sequence and upstream of different cellular localization signals coded in different reading frames can allow for selective localization of a certain percent of the protein population. This can be useful in a variety of situations, such as the selective localization of a protein into a microcompartment while keeping some of the translated protein in the cytosol to maintain normal metabolism levels.
Operon regulation
In nature, there are a number of operons that require specific ratios of protein expression. One example of this is the carboxysome, which is a cellular microcompartment that facilitates carbon-fixation in photosynthetic organisms (Bonacci et al., Proc Natl Acad Sci, 2012). This microcompartment is assembled from repeating units of seven different proteins that are needed in different ratios. A pseudoknot with a characteristic frameshift frequency could be used upstream of two genes from the carboxysome operon to ensure that appropriate levels of each protein are expressed. This would be an alternative method to expressing each protein under the control of different promoters or ribosomal binding sites.
Software for Pseudoknots
In order to make pseudoknots available for use by the synthetic biology community, we are developing a software program to facilitate the overlapping of coding sequences. This program implements two strategies for finding overlapping sequences. The program will take two amino acid sequences, convert them into a DNA sequences, and attempt to align them in all reading frames. Codon redundancy is used to facilitate overlapping of the two sequences. If the sequences can be aligned in a particular region, the program will output a DNA sequence that will have the two original sequences overlapped. This sequence can then be synthesized and inserted into a construct downstream of a pseudoknot to give expression of both input sequences.We see this program as an additional tool that can be used to facilitate use of this new class of BioBrick parts.
This program has a newly added function that allows users to specify a single amino acid sequence of interest. The program will translate that amino acid sequence into all corresponding DNA sequences and generate a list of 10,000,000 DNA sequences that would successfully overlap with the specified coding seqeunce. This list of sequences can be produced as a FASTA file and blasted against the NCBI nucleotide database. This will allow for identification of all proteins that could overlap with the protein of interest. Using this, the correct codon usage and pseudoknot type can be selected for dual coding.
Be advised, this new functionality is computationally intensive. In order to run the new single protein overlap search it is recommended that your system has 32 GB of RAM.
Figure 2. Screenshot of the Program. The upper section demonstrates a comparison between two amino acid strings. The lower section is a print out of the Zipper.pl manual.
The program currently consists of the sequence comparison algorithm as a perl script. If you would like to try out this program, install the perl environment on your computer and download our program from the file below.
Click here to access file
References
Bonacci, W., Teng, P. K., Afonso, B., Niederholtmeyer, H., Grob, P., Silver, P. A., Savage, D. F. (2012) Modularity of a carbon-fixing protein organelle. Proc Natl Acad Sci, 109(2), 478-483.
Hansen, T. M., Reihani, S. M., Oddershede, L. B., Sørensen, M.A. (2007) Correlation between mechanical strength of messenger RNA pseudoknots and ribosomal frameshifting. Proc Natl Acad Sci, 104(14), 5830-5.
Staple, D. W. and Butcher S. E. (2005) Pseudoknots: RNA Structures with Diverse Functions. PLoS Biol, 3(6), e213. Doi:10.1371/journal.pbio.0030213.
Retrieved from " http://2013.igem.org/Team:Lethbridge/project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
