<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NUS_Singapore_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NUS Singapore/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="topmenu-container"><DIV class="dropdown"><BUTTON class="dropbtn">Awards Check </BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:NUS_Singapore/Measurement">Measurement</A><A href="https://2017.igem.org/Team:NUS_Singapore/Model">Modelling</A><A href="https://2017.igem.org/Team:NUS_Singapore/Medal Check List">Medal Check List</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Team</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:NUS_Singapore/Team">Team Member</A><A href="https://2017.igem.org/Team:NUS_Singapore/Attributions">Attributions</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Collaboration</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:NUS_Singapore/Collaborations">Collaboration</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Human Practice</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:NUS_Singapore/HP/Silver">Silver Up</A><A href="https://2017.igem.org/Team:NUS_Singapore/HP/Gold_Integrated">Gold Integrated</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Modelling</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:NUS_Singapore/Overview">Overview</A><A href="https://2017.igem.org/Team:NUS_Singapore/Methodology">Methodology</A><A href="https://2017.igem.org/Team:NUS_Singapore/Model_P">Kill Switch for Probiotics</A><A href="https://2017.igem.org/Team:NUS_Singapore/Model_B">Kill Switch for BeeT</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Projects</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:NUS_Singapore/Description">Description</A><A href="https://2017.igem.org/Team:NUS_Singapore/InterLab">Interlab Study</A><A href="https://2017.igem.org/Team:NUS_Singapore/Parts">Parts</A><A href="https://2017.igem.org/Team:NUS_Singapore/Experiments">Experiment</A><A href="https://2017.igem.org/Team:NUS_Singapore/Notebook">Documentation</A><A href="https://2017.igem.org/Team:NUS_Singapore/Improvement">Improvement</A><A href="https://2017.igem.org/Team:NUS_Singapore/Demonstrate">Demonstrate</A><A href="https://2017.igem.org/Team:NUS_Singapore/Safety">Safety</A><A href="https://2017.igem.org/Team:NUS_Singapore/Future Work">Future Works</A></DIV></DIV><DIV class="dropdown"><A href="https://2017.igem.org/Team:NUS_Singapore">Home</A></DIV></DIV><BUTTON onclick="topFunction()" id="BacktoTopBtn" title="Go to top">Back to Top</BUTTON><DIV style="position:center;"><H1>Lab Journal</H1></DIV><DIV class=".row_container"><BUTTON class="accordion" style=" background-color: #FFEBEE;">May</BUTTON><DIV class="text"><H2>First three weeks</H2><P>Brainstorming for potential ideas. The team have considered a few different topics including detection of sound wave with microbes and design of urinary catheter antimicrobial microbes. After intensive reading and research, the topic was crystalized to the design of a kill-switch for engineered probiotics application. </P><P>As part of our safety culture and practices in NUS, we went on to attend an online safety course and successfully obtained lab safety certificates: A series of online modules to fulfil safety requirements by Office of Safety, Health and Environment (OSHE, http://www.nus.edu.sg/osh/), the department in charge of Laboratory and Work Safety at the National University of Singapore. </P><P>Our team also attended the safety training events before being able to conduct lab work in Centre for Life Science (CeLS), NUS.</P><H2>Last week</H2><P>Basic hands on experiments with the entire construction process. Grasp essential lab skills needed for iGEM such as PCR, Gibson assembly and transformation.</P><P>We began Interlab Study: Cell transformation was performed. Unfortunately, very low transformation efficiency was observed.</P></DIV></DIV><DIV class=".row_container"><BUTTON class="accordion" style=" background-color:#F3E5F5;">June</BUTTON><DIV class="text"><H2>First Week</H2><P>Continuation of Interlab Study: Identifying possible causes of failure (e.g. cells were perhaps left in the ice for too long or cell lysis may occur during prolonged mixing) The transformation process was repeated. The team also completed the third part of the Interlab Study where we conducted cell density measurement at 0, 2, 4, 6, hours, with a micro plate reader.</P><H2>Second Week</H2><P>Design of the plasmids that will be used in the project, Subsequently required gblocks were ordered from IDT. </P><H2>Third Week</H2><P>The whole team had participated in the Seventh International Meeting on Synthetic Biology (SB7.0). We had gained insights and met researchers from different parts of the world.</P><P>For the experiments, we had started construction of the phosphate-sensor with GFP reporter (our improved part, pGEMB), the temperature-sensitive GFP reporter system (our new part, pGEME) and inducible E2 killing protein expression system (pGEMF). </P><P>Here is a brief description of the steps:</P><UL class="nus"><LI>PCR: </LI><OL><LI>Prepare the master mixture. 3) PCR for 1:30 hours in the PCR machine (30 cycles).</LI><LI>Add forward and reverse primers for the constructs. </LI><LI>PCR for 1:30 hours in the PCR machine (30 cycles).</LI></OL><LI>Gibson Assembly: </LI><OL><LI>Prepare the mixture of DNA backbone and respective gblocks with Gibson Master Mix.</LI><LI>Incubate in 50 degrees for a total of 0.5 hours in the thermocycler. </LI></OL><LI>Cell transformation:  </LI><OL><LI>Mix competent cells with assembled plasmid mixture. </LI><LI>Heat shock for fifty seconds and cool down for three minutes.  </LI><LI>Add LB broth.</LI><LI> Incubate in the shaking incubator for one hour.</LI><LI>Place the agar plate in the 37-degrees incubator for one hour. </LI><LI>Plate the cells.</LI></OL><LI>Cell inoculation: </LI><UL class="nus"><LI>Two tubes (6-7 ml of LB broth). Inoculate each tube with one colony.</LI></UL><LI>Preparing glycerol stock:</LI><UL class="nus"><LI>850 ÂµL of solution from cell inoculation into glycerol solution and keep at -80-degrees fridge for long term storage. </LI></UL><LI>Plasmid extraction - Follow the protocol in the manual.</LI><LI>Sequencing: </LI><OL><LI>Prepare sequencing primer. </LI><LI>Primer and 2 samples of constructed plasmid were sent for sequencing.</LI></OL></UL><H2>Last Week</H2><P>We had received sequencing results and the results were not satisfactory. Construction of phosphate sensor with GFP reporter was done again. </P></DIV></DIV><DIV class=".row_container"><BUTTON class="accordion" style=" background-color: #E8EAF6;">July</BUTTON><DIV class="text"><H2>First Week</H2><P>Sequencing result for the phosphate sensor with GFP reporter was good. MOPS medium was purchased for future characterisation of our phosphate sensor-GFP reporter.</P><P>We have began second try of reconstruction for temperature sensitive system - with GFP reporter, and reconstruction of E2 expression system. </P><H2>Second Week</H2><P>Sequencing result for temperature sensitive system was good. We could begin characterisation of this system in the next few weeks.</P><P>MOPS minimal medium and phosphate stock solution were prepared. Plasmid containing the phosphate sensor was retransformed into MG 1655 for characterisation with a microplate reader next week. </P><H2>Third Week</H2><P>8 hours (10 mins sampling of OD and GFP) microplate reading was conducted for the phosphate sensor-GFP reporter. From the characterisation results, the phosphate sensor looked promising, exhibiting greater GFP production at lower concentrations of phosphate, while repressing GFP production at high phosphate concentrations. </P><P>Construction of E2 killing system was repeated this week and the plasmid was sent for sequencing.</P><H2>Last Week</H2><P>The plasmid with temperature sensitive system (with GFP reporter) was re-transformed into MG 1655. 8 hours of continuous micro plate reading was conducted at 37 degrees to gauge the GFP production of the wild type cells under varying IPTG levels.</P><P>Construction of the killing system was still not successful, primers for this system were redesigned. Nevertheless, we had managed to complete two constructs (temperature system and phosphate sensor) in 2 months.  </P></DIV></DIV><DIV class=".row_container"><BUTTON class="accordion" style=" background-color: #E1F5FE;">August</BUTTON><DIV class="text"><H2>First Week</H2><P>Re-characterisation of phosphate system (in MG1655) was done to verify the GFP results. </P><P>Start of school semester. </P><P>Characterisation of temperature sensitive system (MG 1655) was conducted at 30 degrees. Results did not look promising, GFP expression at lower temperature is much higher than GFP expression at higher temperature.</P><P>Concurrently, initial and final 24th hour plate reading of the overnight cultures at 30 and 37 degrees (with the temperature system) was conducted. Surprisingly and Interestingly, the GFP results obtained were as expected, where there is higher GFP production at higher temperature.  </P><P>Construction of the killing system is still not successful. Construction was again repeated, with an optimised gblock to backbone ratio. </P><H2>Second Week</H2><P>Began construction of phosphate-temperature cascaded system with GFP reporter. Characterisation of this construct was conducted with varying phosphate concentrations under two different temperatures, 30 and 37 degrees. Again, the GFP results does not look promising, showing greater GFP expression in lower temperatures. </P><P>Sequencing results for killing system was not good. Began characterisation for this system, and as expected, results were not promising, as the MG 1655 cells had died too quickly, perhaps due to high leakiness of the system to produce E2 killing protein even in repressed states. As an alternative, we had redesigned the killing plasmid with a tighter TetR system. Construction of this system had begun this week. </P><H2>Third Week</H2><P>Characterisation of new TetR based killing system. However, the transformed MG 1655 cells still die too quickly.</P><H2>Last Week</H2><P>Decided to stop experimenting on the killing system and start focusing on âde-buggingâ the temperature sensitive system which performs in opposite fashion to what was expected. </P><P>In response, we had designed the thermal gradient assay by using a conventional thermal cycler to set varying temperature gradient from 30 to 37 degrees across the rows. GFP reading and OD600 were obtained at the 0 hour and 24th hour.</P></DIV></DIV><DIV class=".row_container"><BUTTON class="accordion" style=" background-color: #E8F5E9;">September</BUTTON><DIV class="text"><H2>First Week</H2><P>Repeated the same thermal gradient protocol as last week. To reduce liquid loss impact during the culture transfer,we had loaded 100ul of cells instead of 50ul into each wells and an initial and final plate reading (at the 24th hour) would only be taken.</P><P>This is the detailed plan: Culture the transformed cells ( the temperature sensitive system construct, is transformed into MG 1655) from glycerol and incubate for 36 h in 30 degrees in LB medium;  take 100x dilution of the original sample and regrow in fresh LB medium for 4 hour; load 100ÂµL of cell into each well of the transparent plate and do initial microplate read (0 hr reading) before transferring to PCR plate for incubation; after incubating for 24 hours, transfer from PCR plate to transparent 96 wells plate for plate reading (at 24hours time point ).</P><H2>Second Week</H2><P>Construction of nitrate sensor with GFP reporter and phosphate sensor with GFP reporter (weaker RBS BBa_B0032) had began.</P><P>Characterisation of temperature system (MG 1655) in MOPS medium with2mM of phosphate ions. Characterisation of the phosphate-temperature cascaded system (MG 1655) was also conducted in MOPS medium, with varying amounts of phosphate ions. .</P><H2>Third Week</H2><P>The sequencing results for the modified nitrate and phosphate sensor were good, so we begun transformation and characterization this week (the nitrate protocols can be found here: http://parts.igem.org/Part:BBa_K381001:Experience). We re-characterised the temperature-sensitive system and also the phosphate-temperature system at higher starting OD600 of 0.3 or 0.4a plasmid containing constitutive GFP production  was used as a control. </P><H2>Last Week</H2><P>Characterisation results of the temperature system and phosphate-temperature system Were not in line with expectations.</P><P>We re-transformed the plasmid (temperature sensitive system) into 10Î² cells to compare its GFP expression with earlier results from MG 1655. Characterisation of temperature sensitive system with the thermal gradient assay showed the expected results, which yielded higher GFP production at higher temperatures. </P><P>Characterisation of both phosphate sensors (one is with RBS BBa_B0034 and one is with RBS BBa_B0032) in MOPS medium under varying phosphate concentrations. Characterisation results were satisfactory and suggested the phosphate sensor with stronger RBS  had showed 40 folds increase in GFP expression when compared to the other phosphate sensor construct.</P></DIV></DIV><DIV class=".row_container"><BUTTON class="accordion" style=" background-color: #F9FBE7;">October</BUTTON><DIV class="text"><H2>First Week</H2><P>To prepare submission of standard parts in pSB1c3 to iGEM HQ, we have constructed rrnB-EL222 with this backbone and transformed into 10Î² cells. This construct will constitutively express EL222 proteins, which is a blue-light sensitive transcription factor. </P><H2>Second Week</H2><P>Casamino acids assay was conducted to characterize the effects in affecting the phosphate sensor in phosphate-temperature system (10Î²). </P><P>We characterised the construct with temperature sensitive system (10Î²) and the construct with phosphate-temperature system (10Î²) under respective inducers of IPTG and phosphate in MOPS with 0.2% casamino acids at 37 degrees as well as in 30 degrees.</P><P>Phosphate-temperature cascaded system with controlled expression of IM2 (E2 immunity protein) was constructed and transformed into 10Î².</P><P>Blue light inducible system with RFP expression was transformed into MG 1655 and 10Î². Characterisation of RFP over 8 hours under blue light and control conditions (no blue light) was conducted.</P><H2>Third Week</H2><P>Construction of pGEMBECF (phosphate-temperature inducible Im2 expression and constitutive E2 production) had begun. </P><P>CF-Ind (blue light activating RFP production) in LB media for 10Î² and MG 1655 (with orbital shaking at 120rpm) was conducted. Under blue light illumination, EL222 photosensitve protein (constitutively expressed) is binded upstream of the luxI promoter, enabling RNA polymerase to bind and trigger RFP expression. Without blue light illumination, EL222 protein cannot bind. </P><P>pGEMBE (phosphate-temperature cascaded system) construct characterisation (at higher starting OD600 = 0.2) in MOPS and varying phosphate levels was conducted.</P><P>PGEMBECF OD600 â cell death characterisation was conducted. </P><P>Construction of TlpA36-psb1c3 standard backbone was done. </P><P>Construction of EL222-psb1c3 standard backbone was done. </P><H2>Last Week</H2><P>Characterisation of pGEMBE(phosphate-temperature cascaded system) in MOPS medium under two different temperature conditions ( 37 and 30 degrees), a 8 hour continuous plate reading is conducted where GFP and OD readings are obtained every 10mins. </P><P>In addition, for this construct pGEMBE, under incubation of various temperatures (30 to 37 degrees), GFP production is obtained for every cell by using the flow cytometer.</P><P>Characterisation of cell death profile of pGEMBECF (phosphate-temperature inducible Im2 expression and constitutive E2 production) in MOPS medium under two different temperature conditions ( 37 and 30 degrees), a 8 hour continuous plate reading is conducted where OD readings are obtained every 10mins.</P><P>Miniprep and dry down of the 3 plasmids [(A) rrnB-EL222-PSB1C3, (B) Tlpa36 standard-PSB1C3 &amp; (C) Phosphate-rbs34-PSB1C3 ] to be shipped over to iGEM headquarters. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>