<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NUS_Singapore_Model_B skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NUS Singapore/Model B</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="topmenu-container"><DIV class="dropdown"><BUTTON class="dropbtn">Awards Check </BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:NUS_Singapore/Measurement">Measurement</A><A href="https://2017.igem.org/Team:NUS_Singapore/Model">Modelling</A><A href="https://2017.igem.org/Team:NUS_Singapore/Medal Check List">Medal Check List</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Team</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:NUS_Singapore/Team">Team Member</A><A href="https://2017.igem.org/Team:NUS_Singapore/Attributions">Attributions</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Collaboration</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:NUS_Singapore/Collaborations">Collaboration</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Human Practice</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:NUS_Singapore/HP/Silver">Silver Up</A><A href="https://2017.igem.org/Team:NUS_Singapore/HP/Gold_Integrated">Gold Integrated</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Modelling</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:NUS_Singapore/Overview">Overview</A><A href="https://2017.igem.org/Team:NUS_Singapore/Methodology">Methodology</A><A href="https://2017.igem.org/Team:NUS_Singapore/Model_P">Kill Switch for Probiotics</A><A href="https://2017.igem.org/Team:NUS_Singapore/Model_B">Kill Switch for BeeT</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Projects</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:NUS_Singapore/Description">Description</A><A href="https://2017.igem.org/Team:NUS_Singapore/InterLab">Interlab Study</A><A href="https://2017.igem.org/Team:NUS_Singapore/Parts">Parts</A><A href="https://2017.igem.org/Team:NUS_Singapore/Experiments">Experiment</A><A href="https://2017.igem.org/Team:NUS_Singapore/Notebook">Documentation</A><A href="https://2017.igem.org/Team:NUS_Singapore/Improvement">Improvement</A><A href="https://2017.igem.org/Team:NUS_Singapore/Demonstrate">Demonstrate</A><A href="https://2017.igem.org/Team:NUS_Singapore/Safety">Safety</A><A href="https://2017.igem.org/Team:NUS_Singapore/Future Work">Future Works</A></DIV></DIV><DIV class="dropdown"><A href="https://2017.igem.org/Team:NUS_Singapore">Home</A></DIV></DIV><BUTTON onclick="topFunction()" id="BacktoTopBtn" title="Go to top">Back to Top</BUTTON><H1>Kill Switch for BeeT (Wageningen_UR iGEM 2016)</H1><H2>Introduction</H2><DIV class="columnfull"><P>NUSgem Kill switch Toolkit can also be applied to environmental applications such as the engineered bacteria BeeT from Wageningen_UR iGEM team. The engineered bacteria BeeT is programmed to release toxin that kill the parasitic mites Varroa destructor in beehive. Upon their release into the external environment, the two kill switches will be activated to kill bacteria, thereby preventing the spread of genetically modified genes into the ecosystem [1]. Their original kill switch comprises: </P><OL class="nus"><LI>Optogenetic kill switch based on the pDusk/pDawn blue-light sensing system and the mazEF toxin-antitoxin system. When blue light is present, more toxin is produced. The accumulation of toxin in BeeT will degrade mRNA and cause cell death [2].</LI><LI>Cas9 kill switch which is auxotrophic for a synthetic amino acid, para-L-biphenylalanine (BipA). In the absence of BipA, the activated kill switch degrades DNA [2].</LI></OL></DIV><H2>Analysis and modification of BeeT Kill Switches</H2><DIV class="columnfull"><P>The proposed kill switches is indeed robust, yet it still has some limitations. <B>Firstly</B>, it requires the production of many proteins in the cascaded pDusk/pDawn system for the optogentetic kill switch to function. This may increase metabolic burden on cells. <B>Secondly</B>, the cascaded pDusk/pDawn and Cas9 systems are large. Hence, it may be difficult to integrate both kill switches. <B>Thirdly</B>, it requires additional resources and effort to produce and apply synthetic amino acid, BipA to beehive for the Cas9 kill switch to work.  </P><P>In light of the abovementioned limitations, we propose three modifications for the original kill switch. <B>Firstly</B>, we replace the pDusk/pDawn system with the BLind-v1 blue-light sensor promoter, which is smaller and has similar effectiveness in detecting blue light [3]. <B>Secondly</B>, E2-IM2 killing mechanism is adopted since it can degrade both mRNA and DNA [4]. This eliminates the need for the two kill switch systems, hence alleviating the metabolic burden on cells.<B>Lastly</B>, we choose pH (e.g. Pasr) as the second sensor since it best characterizes the environmental condition inside and outside beehive. The use of the second sensor also make the new kill switch more specific. In addition, for E2-IM2 killing system to function properly, we develop a NAND gate, with pH and blue light as the two inputs. Similar to the first case study, E2 toxin is constitutively produced in all environments and its toxicity is neutralized when IM2 antitoxin is produced. A key would be to control the expression of IM2. </P></DIV><H2>Circuit Design</H2><DIV class="columnfull"><P>Due to time constraint, we were unable to characterize the blue light sensor. Assuming that the BLind-v1 and pH sensor promoter are of medium strength, the following NAND gate, which controls the production of IM2, was generated using CELLO. The design was subsequently modelled using AdvanceSyn Studio software. </P><P id="imgdescription">Figure 1: NAND gate design generated by CELLO</P><P id="imgdescription">Table 1: Truth table</P></DIV><H2>Result</H2><DIV class="columnfull"><P id="imgdescription">Figure 2: BeeT Kill switches simulated performance using the model developed</P></DIV><H2>Conclusion</H2><DIV class="columnfull"><P>Simulation results suggests that E2-IM2 kill switch functions properly. BeeT bacteria are likely to die within 1 – 2 hours after being released into the external environment. The rate of cell death can be optimized by controlling the toxin-antitoxin production. In addition, it is necessary to simplify the current NAND gate design to reduce the high metabolic burden on cell. </P></DIV><H3>Reference</H3><UL style="margin-left:10%;margin-right:10%;" id="Ref"><LI>iGEM Wagenigen UR 2016. Project description. Retrieved October 20, from https://2016.igem.org/Team:Wageningen_UR/Description</LI><LI>iGEM Wagenigen UR 2016. Biocontainment. Retrieved October 20, from https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment</LI><LI>Jayaraman, P., Devarajan, K., Chua, T. K., Zhang, H., Gunawan, E., &amp; Poh, C. L. (2016). Blue light-mediated transcriptional activation and repression of gene expression in bacteria. Nucleic Acid Research, 44 (14), 6994 – 7005. doi: 10.1093/nar/gkw548</LI><LI>Schaller, K., &amp; Nomura, M. (1976). Colicin E2 is a DNA endonuclease. Biochemistry, 73 (11), 3989 – 3993. </LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>