Team:NUS Singapore/Model B
Awards Check
Measurement Modelling Medal Check List
Team
Team Member Attributions
Collaboration
Collaboration
Human Practice
Silver Up Gold Integrated
Modelling
Overview Methodology Kill Switch for Probiotics Kill Switch for BeeT
Projects
Description Interlab Study Parts Experiment Documentation Improvement Demonstrate Safety Future Works
Home
Back to Top
Kill Switch for BeeT (Wageningen_UR iGEM 2016)
Introduction
NUSgem Kill switch Toolkit can also be applied to environmental applications such as the engineered bacteria BeeT from Wageningen_UR iGEM team. The engineered bacteria BeeT is programmed to release toxin that kill the parasitic mites Varroa destructor in beehive. Upon their release into the external environment, the two kill switches will be activated to kill bacteria, thereby preventing the spread of genetically modified genes into the ecosystem [1]. Their original kill switch comprises:
Optogenetic kill switch based on the pDusk/pDawn blue-light sensing system and the mazEF toxin-antitoxin system. When blue light is present, more toxin is produced. The accumulation of toxin in BeeT will degrade mRNA and cause cell death [2].
Cas9 kill switch which is auxotrophic for a synthetic amino acid, para-L-biphenylalanine (BipA). In the absence of BipA, the activated kill switch degrades DNA [2].
Analysis and modification of BeeT Kill Switches
The proposed kill switches is indeed robust, yet it still has some limitations. Firstly, it requires the production of many proteins in the cascaded pDusk/pDawn system for the optogentetic kill switch to function. This may increase metabolic burden on cells. Secondly, the cascaded pDusk/pDawn and Cas9 systems are large. Hence, it may be difficult to integrate both kill switches. Thirdly, it requires additional resources and effort to produce and apply synthetic amino acid, BipA to beehive for the Cas9 kill switch to work.
In light of the abovementioned limitations, we propose three modifications for the original kill switch. Firstly, we replace the pDusk/pDawn system with the BLind-v1 blue-light sensor promoter, which is smaller and has similar effectiveness in detecting blue light [3]. Secondly, E2-IM2 killing mechanism is adopted since it can degrade both mRNA and DNA [4]. This eliminates the need for the two kill switch systems, hence alleviating the metabolic burden on cells.Lastly, we choose pH (e.g. Pasr) as the second sensor since it best characterizes the environmental condition inside and outside beehive. The use of the second sensor also make the new kill switch more specific. In addition, for E2-IM2 killing system to function properly, we develop a NAND gate, with pH and blue light as the two inputs. Similar to the first case study, E2 toxin is constitutively produced in all environments and its toxicity is neutralized when IM2 antitoxin is produced. A key would be to control the expression of IM2.
Circuit Design
Due to time constraint, we were unable to characterize the blue light sensor. Assuming that the BLind-v1 and pH sensor promoter are of medium strength, the following NAND gate, which controls the production of IM2, was generated using CELLO. The design was subsequently modelled using AdvanceSyn Studio software.
Figure 1: NAND gate design generated by CELLO
Table 1: Truth table
Result
Figure 2: BeeT Kill switches simulated performance using the model developed
Conclusion
Simulation results suggests that E2-IM2 kill switch functions properly. BeeT bacteria are likely to die within 1 – 2 hours after being released into the external environment. The rate of cell death can be optimized by controlling the toxin-antitoxin production. In addition, it is necessary to simplify the current NAND gate design to reduce the high metabolic burden on cell.
Reference
iGEM Wagenigen UR 2016. Project description. Retrieved October 20, from https://2016.igem.org/Team:Wageningen_UR/Description
iGEM Wagenigen UR 2016. Biocontainment. Retrieved October 20, from https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment
Jayaraman, P., Devarajan, K., Chua, T. K., Zhang, H., Gunawan, E., & Poh, C. L. (2016). Blue light-mediated transcriptional activation and repression of gene expression in bacteria. Nucleic Acid Research, 44 (14), 6994 – 7005. doi: 10.1093/nar/gkw548
Schaller, K., & Nomura, M. (1976). Colicin E2 is a DNA endonuclease. Biochemistry, 73 (11), 3989 – 3993.
