<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NUS_Singapore_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NUS Singapore/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="topmenu-container"><DIV class="dropdown"><BUTTON class="dropbtn">Awards Check </BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:NUS_Singapore/Measurement">Measurement</A><A href="https://2017.igem.org/Team:NUS_Singapore/Model">Modelling</A><A href="https://2017.igem.org/Team:NUS_Singapore/Medal Check List">Medal Check List</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Team</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:NUS_Singapore/Team">Team Member</A><A href="https://2017.igem.org/Team:NUS_Singapore/Attributions">Attributions</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Collaboration</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:NUS_Singapore/Collaborations">Collaboration</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Human Practice</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:NUS_Singapore/HP/Silver">Silver Up</A><A href="https://2017.igem.org/Team:NUS_Singapore/HP/Gold_Integrated">Gold Integrated</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Modelling</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:NUS_Singapore/Overview">Overview</A><A href="https://2017.igem.org/Team:NUS_Singapore/Methodology">Methodology</A><A href="https://2017.igem.org/Team:NUS_Singapore/Model_P">Kill Switch for Probiotics</A><A href="https://2017.igem.org/Team:NUS_Singapore/Model_B">Kill Switch for BeeT</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Projects</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:NUS_Singapore/Description">Description</A><A href="https://2017.igem.org/Team:NUS_Singapore/InterLab">Interlab Study</A><A href="https://2017.igem.org/Team:NUS_Singapore/Parts">Parts</A><A href="https://2017.igem.org/Team:NUS_Singapore/Experiments">Experiment</A><A href="https://2017.igem.org/Team:NUS_Singapore/Notebook">Documentation</A><A href="https://2017.igem.org/Team:NUS_Singapore/Improvement">Improvement</A><A href="https://2017.igem.org/Team:NUS_Singapore/Demonstrate">Demonstrate</A><A href="https://2017.igem.org/Team:NUS_Singapore/Safety">Safety</A><A href="https://2017.igem.org/Team:NUS_Singapore/Future Work">Future Works</A></DIV></DIV><DIV class="dropdown"><A href="https://2017.igem.org/Team:NUS_Singapore">Home</A></DIV></DIV><BUTTON onclick="topFunction()" id="BacktoTopBtn" title="Go to top">Back to Top</BUTTON><H1>Experiment</H1><H2>Overview</H2><DIV class="columnfull"><P>As part of our effort to create a platform for easier design of kill switches, our experimental team have characterised new iGEM parts and improved parts to suit potential users who are seeking customised kill switches for containment of engineered probiotics.</P></DIV><H2> New Part 1: Temperature Sensitive System <FONT color="limegreen"><STRONG>  ✔Fully Functional</STRONG></FONT></H2><DIV class="columnfull"><P>Our human core temperature lies in the range of 36 – 37 °C. This makes temperature a good physical stimulus for us to leverage upon when designing a kill switch to work inside a human body. We have utilised the thermal sensitive protein’s TlpA36 capability of de-dimerising at temperatures higher than 36 °C, which makes it unable to bind to its intrinsic promoter pTlpA. This system can form a temperature-sensitive switch to sense the temperature drop when the engineered probiotic had escaped from the human host.</P><P>As an illustration, by coupling the temperature system to a GFP reporter <A href="http://parts.igem.org/Part:BBa_K2447014"> BBa_K2447014</A>, under temperature of 36 °C and higher (‘ON’ switch condition), GFP production will not be significantly repressed. However, when temperature is below 36 °C (‘OFF’ switch condition), GFP production will be repressed.</P><P>We have put the TlpA36 protein under control of pLac promoter to check the TlpA36 concentration effects on the GFP expression. For 37 °C (Figure 1), the GFP expression falls with the increasing concentration of IPTG. On the other hand, in 30 °C the increasing IPTG concentration seems not to have much impact on the GFP repression. It is a notable fact, that sensor behaves in both cases best without any IPTG, which would suggest that minimum TlpA36 is required for sensor activation.</P><P><FONT color="red"><STRONG>Characterisation of the temperature construct had proven to be challenging for the experimental team </STRONG></FONT>as the temperature control behaves differently in each individual <I>E.coli</I> strain. Parallel studies involving both transformed 10β and MG1655 cells were conducted to understand whether the temperature construct would be functional in different strains. In this study, (Figure 2 and 3) GFP readings were obtained after 24 hours of incubation. This temperature sensitive construct was working as expected in 10β cells, elucidating higher GFP expression at higher temperatures. However, it was not functioning properly in MG1655, where the GFP expressions were much higher at lower temperatures. </P></DIV><H3>Characterisation Protocol: </H3><DIV class="columnfull"><P>Transformed 10β cells were incubated in LB broth with kanamycin (50 ng/µL) at 30 °C for 48 hours before being diluted 100x and then incubated for another 2-3 hours to reach an OD<SUB>600</SUB> of 0.1. Next, cells were loaded into 96 well plate preloaded with various concentrations of IPTG. 10 mins interval reading of OD<SUB>600</SUB> and GFP absorbance was conducted over a continuous 8 hours run of the microplate reader at temperatures of 30 °C and 37 °C (separately).</P><P id="imgdescription">Figure 1: BBa_K2447014 Temperature-system coupled to GFP reporter. At 37 degree Celsius, the system is switched ‘ON’, generating a 1-fold increase in GFP production.</P><P id="imgdescription">Figure 2: BBa_K2447014 construct which was characterised in 10β cells (incubated overnight). At higher temperatures, the system is switched ‘ON’ as seen by the increase in GFP expressions. </P><P id="imgdescription">Figure 3: BBa_K2447014 construct which was characterised in MG1655 cells (incubated overnight). Unlike the construct in 10β cells, these wild type cells do not exhibit the proper ‘ON’ switch state at higher temperatures; where GFP production is expected not to be repressed.   </P></DIV><H2>New Part 2: Phosphate-temperature Cascaded System <FONT color="goldenrod"><STRONG>  ☒partially working, needs improvement</STRONG></FONT></H2><DIV class="columnfull"><P>Phosphate ions contain phosphorus which is an important component for healthy teeth and bone. Hence, our body readily absorbs these vital ions in our gut. The low phosphate concentration in the intestines following absorption earlier in the tract is thus an important chemical stimulus for us to leverage upon in the design of a kill switch, which is able to detect changes in the host’s environment as the engineered probiotics travel out of the human body. Having guided by in silico modelling, we have come up with a phosphate-temperature cascaded system. </P><P>As a first step, we have layered the phosphate-sensitive promoter over the temperature sensitive system and coupled it to a GFP reporter <A href="http://parts.igem.org/Part:BBa_K2447015"> BBa_K2447015</A>. This part was designed in such a way that the temperature sensitive protein TlpA36 is only produced when the phosphate concentration is minimal (to simulate that the engineered probiotic has reached the gut region). However, before exiting the human environment, the temperature is still high enough (at 36 °C and higher), for TlpA36 proteins to exist as monomers and thus GFP production is not repressed. However, once the engineered probiotic leaves the human GI tract, the temperature drops below 36 °C; thus, TlpA36 proteins can dimerise and bind onto its pTlpA promoter, repressing downstream GFP expression.</P><P>From our experiment (Figure 4), we had successfully elucidated 1-fold higher GFP expressions at the higher temperature (37 °C) when compared to the GFP expressions at 30 °C. This confirmed the fact that our temperature sensitive system (consisting of TlpA36 and pTlpA) was fully functioning, creating a repressible effect on downstream gene expression, only when temperature was below 36 °C. However, this layered construct was not working as expected in terms of sensing for the presence of extracellular phosphate ions. Regardless of the temperature conditions, at high phosphate concentrations, GFP expressions were expected to be similar, but we had obtained differing GFP productions at two different temperatures (37 °C and 30 °C) even at the same phosphate concentration. This may be due to the fact of cell metabolism being significantly different in 30 °C and 37 °C. More importantly, the phosphate promoter might not be tight enough to prevent TlpA36 from leaking. In short, our phosphate-temperature construct was working to a large extent, where the temperature control was functionally properly. But due to time constraint, we did not have the chance to rectify the issue of leakiness in the phosphate promoter, possibly by optimising the binding affinity of this promoter. </P></DIV><H3>Characterisation Protocol:</H3><DIV class="columnfull"><P>Transformed 10β cells were incubated in LB broth with kanamycin (50 ng/µL) at 30 °C for 48 hours before being diluted 100x and then incubated for another 2-3 hours to reach an OD<SUB>600</SUB> of 0.1. Cells were washed in MOPS medium (with 0.2% casamino acid and 0.2% glucose) and subsequently re-suspended in MOPS medium (with 0.2% casamino acid and 0.2% glucose). Next, cells were loaded into 96 wells plate preloaded with various concentrations of phosphate. To characterise cell death, readings of OD<SUB>600</SUB> with 10 min interval were conducted over a continuous 8 hours run in the microplate reader at temperatures of 30 °C and 37 °C (separately). </P><P id="imgdescription">Figure 4: Bba_K2447015 Phosphate-temperature sensitive cascaded system with GFP reporter, experimentation with a microplate reader for continuous 8 hours incubations at both 37 °C and 30 °C. At 37 °C, the engineered probiotic remains in the human body and hence GFP production is turned ‘ON’. At 30 °C, the probiotic is assumed to be out of the human system, and hence GFP production is turned ‘OFF’.</P></DIV><H2>New Part 3: Phosphate-temperature Cascaded Kill Switch <FONT color="goldenrod"><STRONG>☒needs further experimentation</STRONG></FONT></H2><DIV class="columnfull"><P>Following the feasibility of the previous construct (<A href="http://parts.igem.org/Part:BBa_K2447015">BBa_K2447015</A>) in showing GFP production at the appropriate phosphate and temperature stimuli, we have taken the final step into <FONT color="red"><STRONG>the design of the two-inputs kill switch, which is consisted of a double plasmid system</STRONG></FONT>. <STRONG><FONT color="limegreen">In the first plasmid, the double-inputs cascaded system (<A href="http://parts.igem.org/Part:BBa_K2447016">BBa_K2447016</A>) would control IM2 anti-toxin production (a type of E2 Immunity protein, to prevent cell death).</FONT></STRONG>This anti-toxin plasmid will work in tandem with <STRONG><FONT color="blue">the second plasmid (<A href="http://parts.igem.org/Part:BBa_K2447017">BBa_K2447017</A>), a standardised plasmid that is constitutively producing E2 killing protein (Colicin, a type of endonuclease). </FONT></STRONG> However, due to time constraint, we did not construct the standardized plasmid.</P><P>Following the same circuit logic as the phosphate-temperature cascaded system with GFP reporter (<A href="http://parts.igem.org/Part:BBa_K2447015">BBa_K2447015</A>), IM2 proteins will be constantly produced when the engineered probiotics is still inside the human body; thereby able to sequester E2 killing protein and keeping the engineered probiotics alive. However, once it is out of the human body, IM2 protein production will be repressed, causing E2 killing protein to accumulate in the bacterial cell thereby degrading its DNA and breaking down the cell membrane. </P><P>Earlier experimentation with E2 killing protein had proven to be highly challenging due to, most probably, E2 proteins high efficacy in killing the cells, even if the IM2 protein was expressed at the same time. This can be attributed to overproduction of E2 from the medium strength constitutive promoter that we tried to put the protein under. In consequence, our transformation plates were usually empty due to all the transformed cells being killed upon receiving the plasmid.</P><P>Nevertheless, even without the actual construct, our team had modelled this kill switch in AdvanceSyn platform to provide insights of IM2 protein production at various phases with different phosphate levels and temperature conditions (to simulate the bacteria moving down the digestive tract and eventually escaping the body). By manipulating the relative promoter strength of the phosphate promoter via changes in its Vmax (Figure 6), we have observed an uncalled decline in IM2 production, which would prematurely kill the cells due to accumulation of E2 killing protein. This had coincided with the results of our previous experiments with the phosphate and temperature cascaded system with GFP reporter (BBa_k2447015). This highlighted that the phosphate promoter was leaky in its repressed state (at high phosphate concentration) and it may be possibly due to the relatively high binding strength of the phosphate promoter, as pointed out by the model (Figure 6); leading to over production of unwanted TlpA36 proteins. Subsequently, we had modelled (Figure 7) the effects of the pTlpA promoter on IM2 production, while constitutively expressing the E2 killing protein to reveal the IM2-E2 protein interaction. This model (Figure 7) had shown that the promoter pTlpA must be of higher binding strength (higher Vmax) to show sufficient downstream IM2 expression to sequester the constitutively expressed E2 killing protein. </P><P>Based on the modelling insights, future attempts of improving the kill switch could include decreasing the binding affinity of the phosphate promoter and increasing the binding affinity of the downstream temperature sensitive promoter (preferably both). This change in the affinities could be done with use of directed mutagenesis. However, due to the difficulties in engineering the promoter, changing the RBS strength used for the given protein expression could be preliminary solution to optimise the balance between the IM2 and E2 proteins.</P></DIV><H3>Characterisation Protocol:</H3><DIV class="columnfull"><P>Co-Transformed 10β cells (with double plasmid kill switch system) were incubated in LB broth with kanamycin (50 ng/µL) at 30 °C for 48 hours before being diluted 100x and then incubated for another 2-3 hours to reach an OD<SUB>600</SUB> of 0.1. Cells were washed in MOPS medium (with 0.2% casamino acid and 0.2% glucose) and subsequently re-suspended in MOPS medium (with 0.2% casamino acid and 0.2% glucose). Next, cells were loaded into 96 wells plate preloaded with various concentrations of phosphate. To characterise cell death, readings of OD<SUB>600</SUB> with 10 min interval were conducted over a continuous 8 hours run in the microplate reader at temperatures of 30 °C and 37 °C (separately). </P><P id="imgdescription">Figure 5: Double plasmids kill switch system consisting of Bba_K2447016 (controlled anti-killing IM2 protein) and Bba_k2447017 (constitutive E2 killing protein production).High phosphate concentration would allow constant IM2 protein production (kill switch not activated). Only at low phosphate concentration and at lower temperature (less than 36 °C), IM2 production would be repressed and unable to sequester E2 killing protein. Hence the kill switch would be activated.</P><P id="imgdescription">Figure 6: Modelling of the IM2 production by the phosphate promoter in our proposed kill switch. The strength of the phosphate promoter were altered (by changing the Vmax). Higher Vmax for this promoter had elucidated issues of leakiness in the system.</P><P id="imgdescription">Figure 7: Modelling of the temperature sensitive promoter, pTlpA which controls IM2 productions in our proposed kill switch. To find an ideal balance between IM2 and E2 levels (assuming E2 protein interact with IM2 protein in 1 to 1 ratio), the strength of the temperature promoter was altered (by changing the Vmax). Higher Vmax for this promoter had been thought to be ideal to produce sufficient IM2 to sequester the E2 killing proteins. </P></DIV><H2>New Part 4: Blue-light Inducible System with RFP Reporter <FONT color="limegreen"><STRONG>  ✔Fully Functional</STRONG></FONT></H2><DIV class="columnfull"><P>Apart from using temperature and phosphate levels, our team had thought of using blue light as a stimulus for our kill switch application. Since the engineered probiotics would be eventually excreted out of the human body after consumption, modern toilet seats could be slightly altered to shine blue light on the faeces, to switch the kill switch ‘ON’.</P><P id="imgdescription">Figure 8: Modern toilet seat could be modified to include blue light LEDs to activate the kill switch in the engineered probiotics. Source: https://www.aliexpress.com/item/Sensor-Toilet-Light-8-Colors-LED-Battery-operated-Lamp-lamparas-Human-Motion-Activated-PIR-Automatic-RGB/32779033584.html</P><P>To illustrate the feasibility of such design, our team had constructed the blue light inducible system with RFP reporter <A href="http://parts.igem.org/Part:BBa_K2447501"> BBa_K2447501</A>. In this circuit, EL222, a bacterial photosensitive protein is constitutively produced. When the cells are shone with blue light, EL222 proteins dimerise and bind upstream of the LuxI promoter, allowing the recruitment of RNA polymerase, followed by downstream expression of RFP. When light is absent, EL222 proteins are not activated, and hence there is minimal RFP expression. Using this part as an example, the kill switch would be turned ‘ON’ when blue light is shone. Under blue light illumination, both wild type cell (MG 1655) and 10 cells exhibited 3 times increase in RFP expression when compared to no light condition, by the 6th hour.  </P></DIV><H3>Characterisation Protocol:</H3><DIV class="columnfull"><P>Transformed MG 1655 and 10β cells are incubated in LB broth with kanamycin (50 ng/µL) at 37 °C for 24 hours before being diluted 100x and then incubated for another 2-3 hours to reach an OD<SUB>600</SUB> of 0.1. Next, cells are loaded onto 12 well plate. Hourly readings of RFP and OD<SUB>600</SUB> were taken and the cells were illuminated with blue light (control group: covered with black cloth) for 7 hours consecutively under orbital shake (120 rpm) and 37 °C.  </P><P id="imgdescription">Figure 9: BBa_K2447501 Blue light inducible RFP expression. Source: https://www.ncbi.nlm.nih.gov/pubmed/27353329</P><P id="imgdescription">Figure 10: BBa_K2447501 Blue light inducible system with RFP reporter. Transformed cells are strongly switched ‘ON’ when blue light is shone as observed by the 3 fold increase in RFP expression when compared to no light condition. Both MG 1655 and 10β cells have reported similar RFP expressions.  </P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>