<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Missouri_Rolla_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Missouri Rolla/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2017_menu_wrapper"><DIV class="menu_button" id="display_menu_control">
		MENU  
	</DIV><DIV id="menu_content"><A href="https://2017.igem.org/Team:Missouri_Rolla"><DIV class="menu_button direct_to_page">
				HOME
			</DIV></A><DIV class="menu_button"> TEAM 
		</DIV><DIV class="submenu_wrapper" id="team_submenu"><A href="https://2017.igem.org/Team:Missouri_Rolla/Team"><DIV class="submenu_button" id="Team_page">
					Team 
				</DIV></A><A href="https://2017.igem.org/Team:Missouri_Rolla/Collaborations"><DIV class="submenu_button" id="Collaborations_page">
					Collaborations
				</DIV></A></DIV><DIV class="menu_button"> PROJECT
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Missouri_Rolla/Description"><DIV class="submenu_button" id="Description_page">
					Project
				</DIV></A><A href="https://2017.igem.org/Team:Missouri_Rolla/Notebook"><DIV class="submenu_button" id="Notebook_page">
					Notebook
				</DIV></A><A href="https://2017.igem.org/Team:Missouri_Rolla/InterLab"><DIV class="submenu_button" id="InterLab_page">
					InterLab
				</DIV></A><A href="https://2017.igem.org/Team:Missouri_Rolla/Results"><DIV class="submenu_button" id="Results_page">
					Results
				</DIV></A><A href="https://2017.igem.org/Team:Missouri_Rolla/Attributions"><DIV class="submenu_button" id="Attributions_page">
					Attributions 
				</DIV></A></DIV><A href="https://2017.igem.org/Team:Missouri_Rolla/Parts"><DIV class="menu_button direct_to_page">
				PARTS
			</DIV></A><A href="https://2017.igem.org/Team:Missouri_Rolla/Safety"><DIV class="menu_button direct_to_page">
				SAFETY
			</DIV></A><DIV class="menu_button"> HUMAN PRACTICES 
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Missouri_Rolla/Engagement"><DIV class="submenu_button" id="Engagement_page">
					Public Engagement 
				</DIV></A></DIV><A href="https://igem.org/2017_Judging_Form?team=Missouri_Rolla"><DIV class="menu_button direct_to_page">
				JUDGING FORM
			</DIV></A></DIV></DIV><DIV class="igem_2017_content_wrapper"><H1>Missouri_Rolla</H1><DIV class="column full_size"><DIV class="notebook"><H2>30 October 2017</H2><H4 class="h4left"><B>Ryan Baumann</B></H4><H4 class="h4right"><B>Start: 5:00PM</B></H4><H4><B>Cloning efficiency testing of new Universal Acceptor Plasmid</B></H4><B>Purpose:</B> To test the cloning efficiency of the Universal Acceptor Plasmid after deletion of the unexpected 10 bp region via site directed mutagenesis PCR<H4><B>Protocol:</B></H4>Golden gate assemblies were performed to insert TNTR3 gblock into our universal acceptor plasmid. 
Various molar ratios of TNTR3: Universal acceptor plasmid were tested. 
Molar ratios of 1:2, 1:1, 2:1 (recommended ratio), 4:1, and 8:1 were tested for the new plasmid. A 2:1 ratio assembly was performed using our original UAP. A negative control assembly was performed with no inserts added. 

Golden gate assemblies contained the following:
1:2 Ratio
<TABLE><TBODY><TR><TD>Component</TD><TD>Volume</TD></TR><TR><TD>TNTR3 (17.64 ng)</TD></TR><TR><TD>New UAP (100ng)</TD><TD>1 μL</TD></TR><TR><TD>10X Ligase Buffer</TD><TD>2 μL</TD></TR><TR><TD>Ligase</TD><TD>1 μL</TD></TR><TR><TD>BsmBI</TD><TD>1 μL</TD></TR><TR><TD>MilliQ H2O</TD><TD>up to 20μL</TD></TR></TBODY></TABLE>

1:1 ratio
<TABLE><TBODY><TR><TD>Component</TD><TD>Volume</TD></TR><TR><TD>TNTR3 (35.28 ng)</TD></TR><TR><TD>New UAP (100ng)</TD><TD>1 μL</TD></TR><TR><TD>10X Ligase Buffer</TD><TD>2 μL</TD></TR><TR><TD>Ligase</TD><TD>1 μL</TD></TR><TR><TD>BsmBI</TD><TD>1 μL</TD></TR><TR><TD>MilliQ H2O</TD><TD>up to 20μL</TD></TR></TBODY></TABLE>

2:1 Ratio
<TABLE><TBODY><TR><TD>Component</TD><TD>Volume</TD></TR><TR><TD>TNTR3 (70.56 ng)</TD></TR><TR><TD>New UAP (100ng)</TD><TD>1 μL</TD></TR><TR><TD>10X Ligase Buffer</TD><TD>2 μL</TD></TR><TR><TD>Ligase</TD><TD>1 μL</TD></TR><TR><TD>BsmBI</TD><TD>1 μL</TD></TR><TR><TD>MilliQ H2O</TD><TD>up to 20μL</TD></TR></TBODY></TABLE>

4:1 Ratio
<TABLE><TBODY><TR><TD>Component</TD><TD>Volume</TD></TR><TR><TD>TNTR3 (141.12 ng)</TD></TR><TR><TD>New UAP (100ng)</TD><TD>1 μL</TD></TR><TR><TD>10X Ligase Buffer</TD><TD>2 μL</TD></TR><TR><TD>Ligase</TD><TD>1 μL</TD></TR><TR><TD>BsmBI</TD><TD>1 μL</TD></TR><TR><TD>MilliQ H2O</TD><TD>up to 20μL</TD></TR></TBODY></TABLE>

8:1 Ratio
<TABLE><TBODY><TR><TD>Component</TD><TD>Volume</TD></TR><TR><TD>TNTR3 (282.24 ng)</TD></TR><TR><TD>New UAP (100ng)</TD><TD>1 μL</TD></TR><TR><TD>10X Ligase Buffer</TD><TD>2 μL</TD></TR><TR><TD>Ligase</TD><TD>1 μL</TD></TR><TR><TD>BsmBI</TD><TD>1 μL</TD></TR><TR><TD>MilliQ H2O</TD><TD>up to 20μL</TD></TR></TBODY></TABLE>

2:1 Ratio with original UAP
<TABLE><TBODY><TR><TD>Component</TD><TD>Volume</TD></TR><TR><TD>TNTR3 (70.56 ng)</TD></TR><TR><TD>Original UAP (100ng)</TD><TD>1 μL</TD></TR><TR><TD>10X Ligase Buffer</TD><TD>2 μL</TD></TR><TR><TD>Ligase</TD><TD>1 μL</TD></TR><TR><TD>BsmBI</TD><TD>1 μL</TD></TR><TR><TD>MilliQ H2O</TD><TD>up to 20μL</TD></TR></TBODY></TABLE>

Negative Control
<TABLE><TBODY><TR><TD>Component</TD><TD>Volume</TD></TR><TR><TD>TNTR3</TD><TD>None</TD></TR><TR><TD>UAP </TD><TD>None</TD></TR><TR><TD>10X Ligase Buffer</TD><TD>2 μL</TD></TR><TR><TD>Ligase</TD><TD>1 μL</TD></TR><TR><TD>BsmBI</TD><TD>1 μL</TD></TR><TR><TD>MilliQ H2O</TD><TD>up to 20μL</TD></TR></TBODY></TABLE>

All samples were run with the following thermocycler conditions:
– 20 seconds @ 37°C,
– (3 minutes @ 37°C, 4 minutes @ 16°C) X26
– 5 minutes @ 50°C
– 5 minutes @ 80°C
– 5 minutes @ 16°C

Following assembly, 1 uL of each sample was transformed on to Chloramphenicol Blue-White Screening plates following Intact Genomics ig 5-Alpha chemically competent cell High efficiency Transformation protocol. Dillutions of 20% and 5% were plated and spread using sterile glass beads. A positive control transformation was performed using 1 uL of original UAP. 

Colonies were incubated at 37C for 30 hours. 

Blue and white colonies were counted on all plates. <H4><B>Stop: 1:45AM</B></H4><H4><B>Results:</B></H4>See project page<H4><B>Next:</B></H4>Boston!</DIV><DIV class="notebook"><H2>25 October 2017</H2><H4 class="h4left"><B>Ryan Baumann</B></H4><H4 class="h4right"><B>Start: 3</B></H4><H4><B>Blunt end Ligation of 10/22/17 PCR1 and transformation   </B></H4><B>Purpose:</B> Ligation of UAP back to itself after removal of 10bp sequence.<H4><B>Protocol:</B></H4>Blunt end ligation was performed uisng NEB T4 DNA ligase protocol. 2 uL of 10/22/17 OJM PCR 1 was ligated in 20 uL reaction. 

Reaction was conducted at room temperature for 2 hours.

After Ligation, 2 uL of 10/25/17 L1 was transformed using Intact Genomics ig 5-Alpha chemically competent cell High efficiency Transformation protocol. 50% and 5% dilutions were plated onto Chloramphenicol plate. <H4><B>Notes:</B></H4>Check colony growth <H4><B>Stop: 5:00</B></H4><H4><B>Products:</B></H4><TABLE><TBODY><TR><TD>Label</TD><TD>Source </TD><TD>Description</TD></TR><TR><TD>10/25/17 L1</TD><TD>10/22/17 PCR1</TD><TD>UAP Ligated back to itself following removal of 10bp fragment</TD></TR></TBODY></TABLE><H4><B>Next:</B></H4></DIV><DIV class="notebook"><H2>23 October 2017</H2><H4 class="h4left"><B>Ryan Baumann</B></H4><H4 class="h4right"><B>Start: 8:45PM</B></H4><H4><B>Colony PCR's of 10/18/2017 Transformations of PhoB and TNTr3, gels of Golden Gate Round two PCR's and 10/23/17 PCR 1 and 2</B></H4><B>Purpose:</B> To check assembly of golden gate round two assemblies and test OJM1 and OJM2 primers. <H4><B>Protocol:</B></H4>18 Colony PCRs were conducted using Taq 2x Master Mix protocol. 9 Colonies from PhoB and TNTR3 transformation plates were used as template. 

PCR 1-9 - TNTR3
PCR 10-18 PhoR

Three 1% agarose gels were prepared with 2 uL of ethidium bromide
Gel 1:
<TABLE><TBODY><TR><TD>Lane(s)</TD><TD>Component</TD></TR><TR><TD>1 </TD><TD>2 Log purple DNA ladder</TD></TR><TR><TD>2</TD><TD>TNTR3 PCR1</TD></TR><TR><TD>3</TD><TD>2 Log purple DNA ladder</TD></TR><TR><TD>4-10</TD><TD>TNTR3 PCR 2-3 and 5-9</TD></TR><TR><TD>11</TD><TD>2 Log purple DNA ladder</TD></TR><TR><TD>12-20</TD><TD>PhoR PCR 1-9</TD></TR></TBODY></TABLE>

Gel 2: 
<TABLE><TBODY><TR><TD>Lane(s)</TD><TD>Component</TD></TR><TR><TD>1 </TD><TD>2 Log purple DNA ladder</TD></TR><TR><TD>2-8</TD><TD>10/23/17 cPCR's 1-7 (Q92X31)</TD></TR><TR><TD>13</TD><TD>2 Log purple DNA ladder</TD></TR><TR><TD>14-20</TD><TD>10/23/17 cPCR's 8-14 (PhoB)</TD></TR><TR><TD>23</TD><TD>10/23/17 OJM PCR1</TD></TR></TBODY></TABLE>

Gel 3:

<TABLE><TBODY><TR><TD>Lane(s)</TD><TD>Component</TD></TR><TR><TD>1</TD><TD>2 Log Purple DNA Ladder</TD></TR><TR><TD>2-8</TD><TD>10/23/17 cPCR 15-21 (O22527)</TD></TR><TR><TD>11</TD><TD>2 Log Purple DNA Ladder</TD></TR><TR><TD>12-18</TD><TD>10/23/17 cPCR 22-28 (Q8LDU4)</TD></TR></TBODY></TABLE><H4><B>Notes:</B></H4><H4><B>Stop: 10:30</B></H4><H4><B>Next:</B></H4></DIV><DIV class="notebook"><H2>23 October 2017</H2><H4 class="h4left"><B>Jeremy Mesa</B></H4><H4 class="h4right"><B>Start: 5:00 pm</B></H4><B>Purpose:</B> To perform a PCR with mutagenesis primers in order to delete a short base pair sequence from plasmid.<H4><B>Protocol:</B></H4>Contents of PCR tubes:
<TABLE><TBODY><TR><TD>Q5</TD><TD>oJM1</TD><TD>oJM2</TD><TD>UAP</TD><TD>Milli-Q</TD></TR><TR><TD>PCR 1</TD><TD>12.5 μL</TD><TD>1.25 μL</TD><TD>1.25 μL</TD><TD>1 μL</TD><TD>4 μL</TD></TR><TR><TD>PCR 2 </TD><TD>12.5 μL</TD><TD>1.25 μL</TD><TD>1.25 μL</TD><TD>0 μL</TD><TD>5 μL</TD></TR></TBODY></TABLE>
PCR 2 was a negative control.

Thermal Cycler Program:
<TABLE><TBODY><TR><TD>Temp (°C)</TD><TD>Time</TD><TD>Cycles</TD></TR><TR><TD>Step 1</TD><TD>98</TD><TD>30 sec</TD><TD>0</TD></TR><TR><TD>Step 2</TD><TD>98</TD><TD>5 sec</TD><TD>24</TD></TR><TR><TD>Step 3</TD><TD>65</TD><TD>20 sec</TD><TD>24</TD></TR><TR><TD>Step 4</TD><TD>72</TD><TD>75 sec</TD><TD>24</TD></TR><TR><TD>Step 5</TD><TD>72</TD><TD>2 min</TD><TD>0</TD></TR><TR><TD>Step 6</TD><TD>4</TD><TD>Infinite</TD></TR></TBODY></TABLE><H4><B>Stop: 5:45</B></H4><H4><B>Next:</B></H4></DIV><DIV class="notebook"><H2>23 October 2017</H2><H4 class="h4left"><B>Ryan Baumann, Jessica Brooks, Tiffany Kuhnert, Lucas Dyer</B></H4><H4 class="h4right"><B>Start: 4:15pm</B></H4><H4><B>Colony PCR of 10/23/2017 Q92x31 5%, Q22527 5% , PhoB 50% , and  Q8LD44 5% , golden gate round 2 transformation plates</B></H4><B>Purpose:</B> To  confirm assembly of title parts with their promoters and terminators<H4><B>Protocol:</B></H4>28 Colony PCRs were conducted using Taq 2x Master Mix protocol. 7 colonies from each plate were run along with a RFP Construct postitive control and milliQ water negative control.

1-7 Q92X31
8-14 PhoB
15-21 O22527
22-28 Q8LDU4

Reaction Set Up:
<TABLE><TBODY><TR><TD>Component</TD><TD>Volume (μ)</TD></TR><TR><TD>Taq 2x Master Mix</TD><TD>7.5</TD></TR><TR><TD>VR</TD><TD>0.3</TD></TR><TR><TD>VF2</TD><TD>0.3</TD></TR><TR><TD>MilliQ</TD><TD>6.9</TD></TR><TR><TD>Colony1-34</TD></TR></TBODY></TABLE>

Thermocycler Conditions:
<TABLE><TBODY><TR><TD>Temp. (C)</TD><TD>Time</TD><TD>Repeats</TD></TR><TR><TD>Step 1</TD><TD>95</TD><TD>5 min</TD><TD>0</TD></TR><TR><TD>Step 2</TD><TD>95</TD><TD>30 sec</TD><TD>35</TD></TR><TR><TD>Step 3</TD><TD>51</TD><TD>30 sec</TD><TD>35</TD></TR><TR><TD>Step 4</TD><TD>68</TD><TD>3 min</TD><TD>35</TD></TR><TR><TD>Step 5</TD><TD>68</TD><TD>5 min</TD><TD>0</TD></TR><TR><TD>Step 6</TD><TD>4</TD></TR></TBODY></TABLE><H4><B>Stop: </B></H4><H4><B>Next:</B></H4>Run Gels of all the colony PCRs</DIV><DIV class="notebook"><H2>18 October 2017</H2><H4 class="h4left"><B>Ryan Baumann</B></H4><H4 class="h4right"><B>Start: 2:30PM</B></H4><H4><B>Plasmid mini preps of Q9LX31, Q8LDU4, and O22527 transformations. Golden gate assemblies of PhoB, PhoR, TNTR3, Q9LX31, Q8LDU4, and O22527 and transformation on to Kanamysin Blue white screening plates </B></H4><B>Purpose:</B> Purify degreening plasmids and assemble signal transduction pathway, periplasmic binding protein, and degreening parts into transcriptional units.<H4><B>Protocol:</B></H4>Minipreps were conducted using iGEM lab manual 2.5 kitless miniprep protocol. Samples were resuspended in 50 uL of TE buffer. 

DNA Concentration were calculated using Thermoscientic Nanodrop 1000. 
Nanodrop Results:
<TABLE><TBODY><TR><TD>Sample</TD><TD>DNA Conc. (ng/μL)</TD><TD>260/280</TD></TR><TR><TD>10/17 MP3 (O22527)</TD><TD>122.1</TD><TD>2.11</TD></TR><TR><TD>10/17 MP4 (O22527)</TD><TD>364.9</TD><TD>1.86</TD></TR><TR><TD>10/17 MP5 (Q9LX31)</TD><TD>326.9</TD><TD>1.88</TD></TR><TR><TD>10/17 MP6 (Q9LX31)</TD><TD>280.2</TD><TD>1.91</TD></TR><TR><TD>10/18 MP1 (Q8LDU4)</TD><TD>76.7</TD><TD>1.93</TD></TR><TR><TD>10/18 MP2 (Q8LDU4)</TD><TD>59.3</TD><TD>1.90</TD></TR><TR><TD>10/18 MP3 (K1467101)</TD><TD>47.4</TD><TD>1.93</TD></TR><TR><TD>10/18 MP4 (K1467101)</TD><TD>100.3</TD><TD>1.93</TD></TR></TBODY></TABLE>

Golden Gate Assemblies of PhoB (9/8/17 MP3), PhoR (9/13/17 MP1), TNTR3 (9/8/17 MP1), O22527 (10/17/17 MP4), Q9LX31 (10/17/17 MP5), Q8LDU4 (10/18/17 MP2) were performed following the NEB Golden Gate assembly kit protocol into golden braid alpha 1 acceptor plasmid (BBa_P10501). 

The plant pho promoter (9/8/17 MP2) was added to the degreening parts. 

BBa_K1467101 added to TNTR3, PhoR, and PhoB.

BBa_K1618037 (9/13/17 MP3) was added to all parts. 

2 uL of each golden gate assemblies were transformed using Intact Genomics ig 5-Alpha chemically competent cell protocol. 50% and 5% dilutions were plated onto Kanamysin Blue white screening plates. 








<H4><B>Stop: 8:30PM</B></H4><H4><B>Results:</B></H4>Check for colonies <H4><B>Products:</B></H4><TABLE><TBODY><TR><TD>Label </TD><TD>Source</TD><TD>Description</TD></TR><TR><TD>10/17 MP3</TD><TD>O22527</TD><TD>O22527 in the UAP</TD></TR><TR><TD>10/17 MP4</TD><TD>O22527</TD><TD>O22527 in the UAP</TD></TR><TR><TD>10/17 MP5</TD><TD>Q9LX31</TD><TD>Q9LX31 in the UAP</TD></TR><TR><TD>10/17 MP6</TD><TD>Q9LX31</TD><TD>Q9LX31 in the UAP</TD></TR><TR><TD>10/18 MP1</TD><TD>Q8LDU4</TD><TD>Q8LDU4 in the UAP</TD></TR><TR><TD>10/18 MP2</TD><TD>Q8LDU4</TD><TD>Q8LDU4 in the UAP</TD></TR><TR><TD>10/18 MP3</TD><TD>K1467101</TD><TD>K1467101 in the UAP</TD></TR><TR><TD>10/18 MP4</TD><TD>K1467101</TD><TD>K1467101 in the UAP</TD></TR><TR><TD>10/18 GG1</TD><TD>9/8/17 MP3</TD><TD>PhoB with BBa_K1467101 Promoter and BBa_K1618037 Terminator</TD></TR><TR><TD>10/18 GG2</TD><TD>9/13/17 MP1</TD><TD>PhoR with BBa_K1467101 Promoter and BBa_K1618037 Terminator</TD></TR><TR><TD>10/18 GG3</TD><TD>9/8/17 MP1</TD><TD>TNTR3 with BBa_K1467101 Promoter and BBa_K1618037 Terminator</TD></TR><TR><TD>10/18 GG4</TD><TD>10/17 MP4</TD><TD>O22527 with Plant Pho Promoter and BBa_K1618037 Terminator</TD></TR><TR><TD>10/18 GG5</TD><TD>10/17 MP5</TD><TD>Q9LX31 with Plant Pho Promoter and BBa_K1618037 Terminator</TD></TR><TR><TD>10/18 GG6</TD><TD>10/18/17 MP2 </TD><TD>Q8LDU4 with Plant Pho Promoter and BBa_K1618037 Terminator</TD></TR></TBODY></TABLE><H4><B>Next:</B></H4></DIV><DIV class="notebook"><H2>17 Oct. 2017</H2><H4 class="h4left"><B>Jeremy Mesa</B></H4><H4 class="h4right"><B>Start: 2:30</B></H4><B>Purpose:</B> To run PCR with mutagenesis primers and a gel following to visualize the results of the reaction.<H4><B>Protocol:</B></H4>PCR:
Contents of the tube, 20 μL total
<TABLE><TBODY><TR><TD>Tube</TD><TD>Q5</TD><TD>oJM1</TD><TD>oJM2</TD><TD>UAP</TD><TD>Milli-Q</TD></TR><TR><TD>PCR 1</TD><TD>10 μL</TD><TD>1 μL</TD><TD>1 μL</TD><TD>1 μL</TD><TD>7 μL</TD></TR></TBODY></TABLE>

Thermal Cycler Program:
<TABLE><TBODY><TR><TD>Temp. (°C)</TD><TD>Time</TD><TD>Repeats</TD></TR><TR><TD>Step 1</TD><TD>98</TD><TD>30 sec</TD><TD>0</TD></TR><TR><TD>Step 2</TD><TD>98</TD><TD>10 sec</TD><TD>30</TD></TR><TR><TD>Step 3</TD><TD>65</TD><TD>20 sec</TD><TD>30</TD></TR><TR><TD>Step 4</TD><TD>72</TD><TD>2 min</TD><TD>30</TD></TR><TR><TD>Step 5</TD><TD>72</TD><TD>2 min</TD><TD>0</TD></TR><TR><TD>Step 6</TD><TD>4</TD><TD>Infinite</TD><TD>0</TD></TR></TBODY></TABLE>

1% Agarose gel was prepared with 3 μL Ethidium Bromide.
<TABLE><TBODY><TR><TD>Lane 1</TD><TD>Ladder</TD><TD>20 μL</TD></TR><TR><TD>Lane 4</TD><TD>PCR 1</TD><TD>15 μL</TD></TR></TBODY></TABLE><H4><B>Notes:</B></H4>Program name on thermal cycler is saved as PCR10-6. <H4><B>Stop: 4:00</B></H4><H4><B>Results:</B></H4>PCR lane of gel was found to have 4 bands at about 2500bp, 1500bp, 600bp, and less than 100bp. The lane was streaked behind the largest base pair band. <H4><B>Next:</B></H4></DIV><DIV class="notebook"><H2>10 15 2017</H2><H4 class="h4left"><B>Lynell Cunningham, Luke Dyer, Tiffany Kuhnert</B></H4><H4 class="h4right"><B>Start: 3:00</B></H4><H4><B>Colony PCR and Gel of 9/30/17 Golden gate assemblies of degreening parts</B></H4><B>Purpose:</B> To confirm assembly of Q9LX31, Q8LDU4, and O22527 into the universal acceptor plasmid.


<H4><B>Protocol:</B></H4> 17 Colony Pcr's were conducted using Taq 2x Master mix protocol. 5 Colonies from each 9/30/17 golden gate reaction were run along with a RFP Construct postitive control and milliQ water negative control.

Reaction setup: 
<TABLE><TBODY><TR><TD>Component</TD><TD>Volume</TD></TR><TR><TD>Taq 2X Master Mix</TD><TD>10 μL</TD></TR><TR><TD>VR</TD><TD>.4 uL</TD></TR><TR><TD>VF2</TD><TD>.4 uL</TD></TR><TR><TD>MilliQ</TD><TD>9.2 uL</TD></TR><TR><TD>Colony 1-15</TD></TR></TBODY></TABLE>

Thermocycler Conditions:
95C for 5 minutes, 35 Cycles of: (95C for 20 seconds, 66C for 30 sec, 68C for 1min), 68C for 5 minutes, and 4C hold

All samples were run on a 1% agarose gel with the following setup:
Lane 1 and 11: 2 log purple DNA ladder
Lane 2-6 are Colonies 1-5 from Q8LDU4
Lanes 7,8,9,10,and 12 are Colonies 6-10 from O22527
Lanes 13-17 are Colonies 11-15 for Q9LX31
Lane 18 is a water negative control
Lane 19 is the RFP Construct positive control

After imaging, one colony from each golden gate transformation was grown in broth culture containing Chloramphenicol<H4><B>Notes:</B></H4><H4><B>Stop: 10:00pm</B></H4><H4><B>Next:</B></H4>Grow successfully assembled cultures, miniprep, and round two assemblies</DIV><DIV class="notebook"><H2>10 October 2017</H2><H4 class="h4left"><B>Ryan Baumann</B></H4><H4 class="h4right"><B>Start: 4:30PM</B></H4><H4><B>Gel of 9 October 2017 Golden Gate and Colony PCR's</B></H4><B>Purpose:</B> To confirm proper assembly of previous golden gate assemblies <H4><B>Protocol:</B></H4>A 1% agarose gel with 1.5 μL Ethidium Bromide and 2 Log Purple DNA Ladder.10 μL of samples were loaded into wells. 

<TABLE><TBODY><TR><TD>Well</TD><TD>Sample</TD></TR><TR><TD>1</TD><TD>2 Log purple DNA Ladder</TD></TR><TR><TD>2</TD><TD>1</TD></TR><TR><TD>3</TD><TD>2</TD></TR><TR><TD>4</TD><TD>3</TD></TR><TR><TD>5</TD><TD>4</TD></TR><TR><TD>6</TD><TD>5</TD></TR><TR><TD>7</TD><TD>6</TD></TR><TR><TD>8</TD><TD>7</TD></TR><TR><TD>9</TD><TD>8</TD></TR><TR><TD>10</TD><TD>9</TD></TR><TR><TD>11</TD><TD>2 Log Purple DNA Ladder</TD></TR><TR><TD>12</TD><TD>10</TD></TR><TR><TD>13</TD><TD>11</TD></TR><TR><TD>14</TD><TD>12</TD></TR><TR><TD>15</TD><TD>13</TD></TR><TR><TD>16</TD><TD>14</TD></TR><TR><TD>17</TD><TD>15</TD></TR><TR><TD>18</TD><TD>16</TD></TR><TR><TD>19</TD><TD>17</TD></TR><TR><TD>20</TD><TD>18</TD></TR><TR><TD>21</TD><TD>10/9/17 A1</TD></TR><TR><TD>22</TD><TD>10/9/17 A2</TD></TR><TR><TD>23</TD><TD>10/9/7 A3</TD></TR></TBODY></TABLE><H4><B>Notes:</B></H4><H4><B>Stop: 6:00:00PM</B></H4><H4><B>Results:</B></H4>Analyze Gel Results <H4><B>Next:</B></H4></DIV><DIV class="notebook"><H2>30 September 2017</H2><H4 class="h4left"><B>Kent Gorday, Lynell Cunningham, Lucas Dyer, Jeremy Mesa</B></H4><H4 class="h4right"><B>Start: 16:35</B></H4><H4><B>Diagnostic PCRs of round 2 assemblies and assembly of degreening gblocks into UAP</B></H4><B>Purpose:</B> To check addition of constitutive promoter and terminator to coding sequences, and assemble degreening coding sequences into Universal Acceptor Plasmid for submission.<H4><B>Protocol:</B></H4>Six PCRs were performed from round 2 assemblies:

<H5>9/30 A1-6</H5><TABLE><TBODY><TR><TD>10 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>7 μL</TD><TD>MilliQ Water</TD></TR><TR><TD>1 μL</TD><TD>Template Plasmid</TD></TR><TR><TD>1 μL</TD><TD>VF2</TD></TR><TR><TD>1 μL</TD><TD>VR</TD></TR><TR><TD>20 μL</TD><TD>Total Volume</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD>Temperature (°C)</TD><TD>Time (sec)</TD></TR><TR><TD>98</TD><TD>30</TD></TR><TR><TD>98c</TD><TD>8</TD></TR><TR><TD>65c</TD><TD>22</TD></TR><TR><TD>72c</TD><TD>93</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>ꝏ</TD></TR></TBODY></TABLE><P class="centerp">cycle X32</P>

Three Golden Gate reactions into the Universal Acceptor Plasmid were performed:

<H5>9/30 GG1-3</H5><TABLE><TBODY><TR><TD>4.9 μL</TD><TD>100 ng/μL Q9LX31 gblock</TD></TR><TR><TD>2.5 μL</TD><TD>UAP</TD></TR><TR><TD>2 μL</TD><TD>10X T4 DNA Ligase Buffer</TD></TR><TR><TD>1 μL</TD><TD>T4 DNA Ligase</TD></TR><TR><TD>1 μL</TD><TD>BsmBI</TD></TR><TR><TD>8.6 μL</TD><TD>MilliQ Water</TD></TR><TR><TD>15.1 μL</TD><TD>Total Volume</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD>5.9 μL</TD><TD>100 ng/μL O22527 gblock</TD></TR><TR><TD>2.5 μL</TD><TD>UAP</TD></TR><TR><TD>2 μL</TD><TD>10X T4 DNA Ligase Buffer</TD></TR><TR><TD>1 μL</TD><TD>T4 DNA Ligase</TD></TR><TR><TD>1 μL</TD><TD>BsmBI</TD></TR><TR><TD>7.6 μL</TD><TD>MilliQ Water</TD></TR><TR><TD>14.1 μL</TD><TD>Total Volume</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD>5.8 μL</TD><TD>100 ng/μL Q8LDU4 gblock</TD></TR><TR><TD>2.5 μL</TD><TD>UAP</TD></TR><TR><TD>2 μL</TD><TD>10X T4 DNA Ligase Buffer</TD></TR><TR><TD>1 μL</TD><TD>T4 DNA Ligase</TD></TR><TR><TD>1 μL</TD><TD>BsmBI</TD></TR><TR><TD>7.7 μL</TD><TD>MilliQ Water</TD></TR><TR><TD>14.2 μL</TD><TD>Total Volume</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD>Temperature (°C)</TD><TD>Time (sec)</TD></TR><TR><TD>37</TD><TD>20</TD></TR><TR><TD>37c</TD><TD>180</TD></TR><TR><TD>16c</TD><TD>240</TD></TR><TR><TD>50</TD><TD>300</TD></TR><TR><TD>80</TD><TD>300</TD></TR><TR><TD>16</TD><TD>300</TD></TR></TBODY></TABLE><P class="centerp">cycle X32</P>

A 1.2% agarose gel was run with PCR products:
<TABLE><TBODY><TR><TD>Well No.</TD></TR><TR><TD>3</TD><TD>Ladder</TD></TR><TR><TD>4</TD><TD>A1</TD></TR><TR><TD>5</TD><TD>Ladder</TD></TR><TR><TD>6</TD><TD>A2; Bad Load</TD></TR><TR><TD>7</TD><TD>A4</TD></TR><TR><TD>8</TD><TD>A3; Bad Load</TD></TR><TR><TD>9</TD><TD>A5</TD></TR><TR><TD>10</TD><TD>A6</TD></TR></TBODY></TABLE><H4><B>Stop: 20:40</B></H4><H4><B>Results:</B></H4><H4><B>Products:</B></H4><TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>9/30 A1</TD><TD>9/18 MP1</TD><TD> + pro/ter amplified</TD></TR><TR><TD>9/30 A2</TD><TD>9/18 MP2</TD><TD> + pro/ter amplified</TD></TR><TR><TD>9/30 A3</TD><TD>9/18 MP3</TD><TD> + pro/ter amplified</TD></TR><TR><TD>9/30 A4</TD><TD>9/18 MP4</TD><TD> + pro/ter amplified</TD></TR><TR><TD>9/30 A5</TD><TD>9/18 MP5</TD><TD> + pro/ter amplified</TD></TR><TR><TD>9/30 A6</TD><TD>9/18 MP6</TD><TD> + pro/ter amplified</TD></TR><TR><TD>9/30 GG1</TD><TD>Q9LX31 gblock</TD><TD>GUN4 CDS assembled into Universal Acceptor Plasmid</TD></TR><TR><TD>9/30 GG2</TD><TD>O22527 gblock</TD><TD>Chlorophyllase-1 CDS assembled into Universal Acceptor Plasmid</TD></TR><TR><TD>9/30 GG3</TD><TD>Q8LDU4 gblock</TD><TD>RCCR CDS assembled into Universal Acceptor Plasmid</TD></TR></TBODY></TABLE><H4><B>Next:</B></H4>Colony PCR and patch to a new plate a few white colonies from each round 2 assembly transformation.</DIV><DIV class="notebook"><H2>9-21-17</H2><H4 class="h4left"><B>Ryan Baumann</B></H4><H4 class="h4right"><B>Start: 1:30PM</B></H4><H4><B>PCR and Gel of 18 September 2017 minipreps (again)</B></H4><B>Purpose:</B> To identify if phoR, phoB, and TNTR3are properly assembled in transcriptional units with BBa_K1618037 and BBa_K1467101<H4><B>Protocol:</B></H4>Six 20 μL PCR's of 9/18/17 MP 1-6  were run using containing: Q5 HIgh Fidelity 2X Master Mix, 0.4 μL VR and VF2 (10 μM solutions), The following template concentrations, and MilliQ Water
up to 20 μL.

<TABLE><TBODY><TR><TD>Template</TD><TD>Volume</TD></TR><TR><TD>9/18/17 MP1</TD><TD>1 μL</TD></TR><TR><TD>9/18/17 MP2</TD><TD>1 μL</TD></TR><TR><TD>9/18/17 MP3</TD><TD>1 μL</TD></TR><TR><TD>9/18/17 MP4</TD><TD>1 μL</TD></TR><TR><TD>9/18/17 MP5</TD><TD>4 μL</TD></TR><TR><TD>9/18/17 MP6</TD><TD>5 μL</TD></TR></TBODY></TABLE>

A positive control was run using a 1μL of RFP construct in psb1C3.
A negative control was run using MilliQ water.

Samples were run in the thermocycler in the following conditions:

98C for 30 seconds
32 cycles of: 98C for 8 seconds, 65C for 22 seconds, and 72C for 85 seconds
72C for 120 seconds
4C Hold

All PCR's were run on a 1% agarose gel containing 1.5 μL of ethidium bromide

<TABLE><TBODY><TR><TD>Well</TD><TD>Sample</TD></TR><TR><TD>1</TD><TD>2 Log Purple DNA Ladder</TD></TR><TR><TD>3</TD><TD>PCR1(MP1)</TD></TR><TR><TD>4</TD><TD>PCR2(MP2)</TD></TR><TR><TD>5</TD><TD>PCR3(MP3)</TD></TR><TR><TD>6</TD><TD>PCR4(MP4)</TD></TR><TR><TD>7</TD><TD>PCR5(MP5)</TD></TR><TR><TD>8</TD><TD>PCR6(MP6)</TD></TR><TR><TD>9</TD><TD>Positive Control</TD></TR><TR><TD>10</TD><TD>Negative Control</TD></TR></TBODY></TABLE><H4><B>Notes:</B></H4><H4><B>Stop: </B></H4><H4><B>Results:</B></H4>Continue to troubleshoot PCR conditions. No conclusive results from gel. <H4><B>Next:</B></H4>Redo PCR's again</DIV><DIV class="notebook"><H2>21 September 2017 </H2><H4 class="h4left"><B>Ryan Baumann</B></H4><H4 class="h4right"><B>Start: 2:00 PM</B></H4><H4><B>PCR and Gel of 18 September 2017 minipreps</B></H4><B>Purpose:</B> To identify if phoR, phoB, and TNTR3are properly assembled in transcriptional units with BBa_K1618037 and BBa_K1467101<H4><B>Protocol:</B></H4>Six 20 μL PCR's of 9/18/17 MP 1-6  were run using containing:
300ng of template DNA
Q5 High Fidelity 2X Master Mix, 
0.2 μM VR and VF2 
and water up to 20 μL

A positive control was run using an RFP construct in psb1C3.
A negative control was run using MilliQ water.

Samples were run in the thermocycler in the following conditions:

98C for 30 seconds
32 cycles of: 98C for 8 seconds, 65C for 22 seconds, and 72C for 85 seconds
72C for 120 seconds
4C Hold

All PCR's were run on a 1% agarose gel containing 1.5 μL of ethidium bromide

<TABLE><TBODY><TR><TD>Well</TD><TD>Sample</TD></TR><TR><TD>1</TD><TD>2 Log Purple DNA Ladder</TD></TR><TR><TD>3</TD><TD>PCR1(MP1)</TD></TR><TR><TD>4</TD><TD>PCR2(MP2)</TD></TR><TR><TD>5</TD><TD>PCR3(MP3)</TD></TR><TR><TD>6</TD><TD>PCR4(MP4)</TD></TR><TR><TD>7</TD><TD>PCR5(MP5)</TD></TR><TR><TD>8</TD><TD>PCR6(MP6)</TD></TR><TR><TD>9</TD><TD>Positive Control</TD></TR><TR><TD>10</TD><TD>Negative Control</TD></TR></TBODY></TABLE><H4><B>Notes:</B></H4><H4><B>Stop: 5:30PM</B></H4><H4><B>Next:</B></H4>TBD</DIV><DIV class="notebook"><H2>14 September 2017</H2><H4 class="h4left"><B>Ryan Baumann</B></H4><H4 class="h4right"><B>Start: 2:00PM</B></H4><H4><B>PCR and Gel of 13 September 2017 minipreps</B></H4><B>Purpose:</B> To identify if Trg_PhoR and BBa_K1618037 Terminator are properly assembled in the UAP<H4><B>Protocol:</B></H4>PCR's  of 9/13/17 MP1 (9/14/17 PCR1), MP2 (9/14/17 PCR2), and MP3 (9/14/17 PCR3), and an RFP positive control (9/14/17 PCR4) were run with the following conditions:

<TABLE><TBODY><TR><TD>Component</TD><TD>Volume</TD></TR><TR><TD>Taq 2X Master Mix</TD><TD>10 μL</TD></TR><TR><TD>VR (10μM)</TD><TD>0.4 μL</TD></TR><TR><TD>VF2 (10μL)</TD><TD>0.4 μL</TD></TR><TR><TD>Samples</TD><TD>0.7 μL</TD></TR><TR><TD>MilliQ</TD><TD>8.5 μL</TD></TR><TR><TD>Total</TD><TD>20 μL</TD></TR></TBODY></TABLE>

PCR’s were run using taq2kbp program following taq 2x master mix protocol. 
 
Annealing temperatures calculated using IDT’s annealing tempature protocoL

All Colony PCR’s were run on a 1% agarose gel containing 2.0 μL ethidium bromide.

Gel Layout:
<TABLE><TBODY><TR><TD>Lane</TD><TD>Component</TD></TR><TR><TD>1</TD><TD>2 Log Purple DNA Ladder</TD></TR><TR><TD>3</TD><TD>9/14/17 PCR1</TD></TR><TR><TD>5</TD><TD>9/14/17 PCR2</TD></TR><TR><TD>7</TD><TD>9/14/17 PCR3</TD></TR><TR><TD>9</TD><TD>9/14/17 PCR4</TD></TR></TBODY></TABLE><H4><B>Notes:</B></H4><H4><B>Stop: 6:00PM </B></H4><H4><B>Next:</B></H4>Round two of golden gate assemblies.</DIV><DIV class="notebook"><H2>6 September 2017</H2><H4 class="h4left"><B>Ryan Baumann</B></H4><H4 class="h4right"><B>Start: 5:00PM</B></H4><H4><B>Golden Gate Assembly of BBa_K1618037 Terminator into UAP and Transformations</B></H4><B>Purpose:</B> To assemble plant terminator into the universale acceptor plasmid using BsmBI Type IIS endonuclease and transform on to chloramphenicol blue white screening plates.

To transform amilCP chromoprotein (BBa_K592009) and eforRed Chromoprotein (BBa_K592012) on to chloramphenicol plates.
<H4><B>Protocol:</B></H4>Reaction setup:
<TABLE><TBODY><TR><TD>UAP</TD><TD>0.20 μL (~60 ng)</TD></TR><TR><TD>BBa_K1618037</TD><TD>5 μL (~50 ng)</TD></TR><TR><TD>10X T4 DNA Ligase Buffer</TD><TD>2 μL</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>1 μL</TD></TR><TR><TD>BsmBI</TD><TD>1 μL</TD></TR><TR><TD>MilliQ Water</TD><TD>10.8 μL</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE>

Samples Ran:
– 20 seconds @ 37°C,
– (3 minutes @ 37°C, 4 minutes @ 16°C) X26
– 5 minutes @ 50°C
– 5 minutes @ 80°C
– 5 minutes @ 16°C

2 μL of the Terminator assembly, 1 μL of AmilCP, and 1 μL of eforRed were transformed using NEB® 5-alpha Competent E. coli (High Efficiency) protocol.

The 1% and 10% dilutions were plated for each. AmilCP and eforRed were transformed on to chloramphenicol plates. The terminator assembly was plated on Chlor blue-white screening plates.
<H4><B>Stop: 10:30pm</B></H4><H4><B>Products:</B></H4><TABLE><TBODY><TR><TD>Label</TD><TD>Source</TD><TD>Description</TD></TR><TR><TD>9/6/17 AmilCP</TD><TD>2017 Kit Plate 1 Well 19E</TD><TD>amilCP resuspended from kit plate</TD></TR><TR><TD>9/6/17 eforRed</TD><TD>2017 Kit Plate 7 Well 15I</TD><TD>eforRed resuspended from kit plate</TD></TR></TBODY></TABLE><H4><B>Next:</B></H4>Check for colony growth. Plasmid mini prep desired colonies. </DIV><DIV class="notebook"><H2>2 September 2017</H2><H4 class="h4left"><B>Ryan Baumann</B></H4><H4 class="h4right"><B>Start: Colony PCR and Gel of 8/31/17 Transformants </B></H4><H4><B>Confirm colonies with properly assembled plant parts with promoter and terminator</B></H4><B>Purpose:</B> To find properly assembled plasmids of PhoB, PhoR, and TNTr3 with promoter and terminator within alpha 1 acceptor plasmid. <H4><B>Protocol:</B></H4>20uL Colony PCR's were run for 7 PhoB colonies, 7 TNTr3 colonies, and 6 PhoR colonies. 

Sample colony PCR
<TABLE><TBODY><TR><TD>Component</TD><TD>Volume</TD></TR><TR><TD>Taq 2x Master Mix</TD><TD>10 μL</TD></TR><TR><TD>VR</TD><TD>0.8 μL</TD></TR><TR><TD>VF2</TD><TD>0.8 μL</TD></TR><TR><TD>MilliQ</TD><TD>8.4 μL</TD></TR><TR><TD>Colonies 1-20</TD></TR></TBODY></TABLE>

PCR's were run using taq2kbp program following taq 2x master mix protocol with annealing temperatures calculated using IDT's annealing tempature protocol. 

All Colony PCR's were run on a 1% agarose gel containing 1.5 μL ethidium bromide. 

Gel Layout

<TABLE><TBODY><TR><TD>Lane(s)</TD><TD>Sample</TD></TR><TR><TD>1</TD><TD>2 Log Purple Ladder</TD></TR><TR><TD>3-9</TD><TD>PhoB colonies 1-7</TD></TR><TR><TD>10-12</TD><TD>TNTr3 COlonies 1-3</TD></TR><TR><TD>13</TD><TD>2 Log Purple Ladder</TD></TR><TR><TD>15-18</TD><TD>TNTR3 4-7</TD></TR><TR><TD>19-24</TD><TD>PhoR colonies 1-6</TD></TR></TBODY></TABLE><H4><B>Notes:</B></H4><H4><B>Stop: </B></H4><H4><B>Next:</B></H4>Troubleshoot gel results. </DIV><DIV class="notebook"><H2>1 September 2017</H2><H4 class="h4left"><B>Benjamin Bleitz</B></H4><H4 class="h4right"><B>Start: 1:00 pm</B></H4><H4><B>Preparation of Arabidopsis growth medium </B></H4><B>Purpose:</B> to prepare a proper growth medium for Arabidopsis in a laboratory setting<H4><B>Protocol:</B></H4><TABLE><TBODY><TR><TD>Component</TD><TD>Amount</TD></TR><TR><TD>Murashige and Skoog salts</TD><TD>2.15 g</TD></TR><TR><TD>1% Sucrose</TD><TD>5 ml</TD></TR><TR><TD>0.05% MES</TD><TD>0.25 ml</TD></TR><TR><TD>MilliQ H2O</TD><TD>total of 500 ml</TD></TR><TR><TD>KOH </TD><TD>Until pH adjusted to 5.7</TD></TR></TBODY></TABLE>
Sucrose, MES and final solution all Autoclaved for 30 minutes on liquid cycle
Plates poured<H4><B>Stop: 3:30</B></H4><H4><B>Results:</B></H4>18 plates prepared for plant growth<H4><B>Products:</B></H4>18 MS Mediums<H4><B>Next:</B></H4>Test growth mediums</DIV><DIV class="notebook"><H2>31 August 2017</H2><H4 class="h4left"><B>Ryan Baumann</B></H4><H4 class="h4right"><B>Start: 3:00 PM</B></H4><H4><B>Addition of Promoter and Terminator to PhoR, PhoB, and TNTR3 </B></H4><B>Purpose:</B> Addition of BBa_K1467101 Promoter and BBa_K1618037 Terminator to PhoR, PhoB, and TNTR3 into BBa_P10501 Alpha 1 acceptor plasmid via golden gate assembly using NEB golden gate assembly kit. <H4><B>Protocol:</B></H4>20 uL Reactions were set up following NEB golden gate assembly mix using 75ng of BBa_P10501 as the destination plasmid and 100ng of each insert. 

All samples were run at 37C for 1 hour followed by 55C for 5 minutes. 

1 uL of each assembly were transformed using NEB® 5-alpha Competent E. coli (High Efficiency) protocol. 

20% and 2% dilutions were then plated on Kanamysin selective blue white screening plates. <H4><B>Stop: 7:00PM</B></H4><H4><B>Results:</B></H4>Growth of white colonies was seen from assembly transformation.<H4><B>Next:</B></H4>Colony PCR from transformation plates to confirm proper assembly </DIV><DIV class="notebook"><H2>23 August 2017</H2><H4 class="h4left"><B>Ryan Baumann</B></H4><H4 class="h4right"><B>Start: 3:00 PM</B></H4><H4><B>PCR of August 18th plasmid minipreps with VR and VF2 and gel extraction  </B></H4><B>Purpose:</B> To verify correct assembly of fragments in the Universal Acceptor Plasmid and extract correct bands from gel
<H4><B>Protocol:</B></H4>~100 ng of purified template DNA was amplified following the iGEM Lab Manual 2.5 PCR Protocol using Taq 2x MM, VR, VF2, and MilliQ H2O.

A positive control was run using linearized Psb1T3 backbone. 


A negative control as run with a water template.

Sample Reaction:
<TABLE><TBODY><TR><TD>Sample</TD><TD>Volume</TD></TR><TR><TD>Taq 2X Master Mix</TD><TD> 12.5 μL</TD></TR><TR><TD>VR</TD><TD>1 μL</TD></TR><TR><TD>VF2</TD><TD>1 uL</TD></TR><TR><TD>Plant Pho (100ng/1uL)</TD><TD>1 uL</TD></TR><TR><TD>Water </TD><TD>9.5 μL</TD></TR><TR><TD>Total</TD><TD>25 μL</TD></TR></TBODY></TABLE>
 
Samples were run in thermocycler using Taq 2X master mix recommend protocol with appropriate annealing temperature for VR and VF2.

All PCR's were run in a 1% agarose gel containing 2 μL ethidium bromide.

Gel Layout:
<TABLE><TBODY><TR><TD>Lane</TD><TD>Sample</TD></TR><TR><TD>1</TD><TD>2 Log Purple DNA ladder</TD></TR><TR><TD>3</TD><TD>Negative Control</TD></TR><TR><TD>4</TD><TD>TNTR3</TD></TR><TR><TD>5</TD><TD>Plant Pho</TD></TR><TR><TD>6</TD><TD>Positive Control</TD></TR><TR><TD>7</TD><TD>PhoB</TD></TR><TR><TD>8</TD><TD>PhoR</TD></TR></TBODY></TABLE>

Bands containing Assembled TNTR3 (~1100BP), Plant Pho (~450BP), PhoB (~1100BP), and PhoR (~1800 BP) were cut from the gel and DNA was purified using Amicon Ultra-DA Kit for DNA extraction from agarose gel. <H4><B>Notes:</B></H4><H4><B>Stop: 5:06 PM</B></H4><H4><B>Products:</B></H4><TABLE><TBODY><TR><TD>Label </TD><TD>Source </TD><TD>Description</TD></TR><TR><TD>8/23/17 Plant Pho</TD><TD>8/18/17 plant Pho Miniprep</TD><TD>Extracted plant pho DNA from gel</TD></TR><TR><TD>8/23/17 Pho R</TD><TD>8/18/17 PhoR miniprep</TD><TD>extracted PhoR from gel</TD></TR><TR><TD>8/23/17 Pho B</TD><TD>8/18/17 PhoB miniprep</TD><TD>Extracted PhoB from Gel</TD></TR><TR><TD>8/23/17 TNTR3</TD><TD>8/18/17 tntr3 miniprep</TD><TD>Extracted TNTr3 from gel</TD></TR></TBODY></TABLE><H4><B>Next:</B></H4></DIV><DIV class="notebook"><H2>18 August 2017</H2><H4 class="h4left"><B>Ryan Baumann</B></H4><H4 class="h4right"><B>Start: 11:00am</B></H4><H4><B>Plasmid Miniprep of colonies from 8/9/17 COlony PCR's</B></H4><B>Purpose:</B> To isolate plasmids from colonies with correctly assembled PhoB, OhoR, Plant pho, and TNTR3<H4><B>Protocol:</B></H4>Colonies that showed positive bands from PCR were innoculated into broth cultures with chloramphenicol and grown up overnight.

Plasmid DNA was purified using the MST iGEM Lab Manual 2.5 Kitless miniprep protocol and suspended in 40uL of TE buffer.

2uL of sample of was run on Thermoscientific Nanodrop 1000.

Nanodrop results:

<TABLE><TBODY><TR><TD>Sample</TD><TD>Concentration (ng/uL)</TD><TD>260nm/280nm</TD></TR><TR><TD>PhoB</TD><TD>263.2</TD><TD>1.84</TD></TR><TR><TD>PhoR</TD><TD>98.6 </TD><TD>1.80</TD></TR><TR><TD>Plant Pho</TD><TD>99.8</TD><TD>1.86</TD></TR><TR><TD>TNTr3</TD><TD>236.0</TD><TD>1.82</TD></TR></TBODY></TABLE><H4><B>Stop: 1:40PM</B></H4><H4><B>Products:</B></H4><TABLE><TBODY><TR><TD>Label</TD><TD>Source</TD><TD>Description</TD></TR><TR><TD>8/18/17 MP1</TD><TD>PhoB Colonies from 6/26/17 Transformation</TD><TD>PhoB in the UAP</TD></TR><TR><TD>8/18/17 MP2</TD><TD>PhoR from 6/26/17 Transformation</TD><TD>PhoR in the UAP</TD></TR><TR><TD>8/18/17 MP3</TD><TD>Plant Pho from 6/26/17 Transformation</TD><TD>Plant Pho in the UAP</TD></TR><TR><TD>8/18/17 MP4</TD><TD>TNTR3 from 6/26/17 Transformation</TD><TD>TNTr3 in the UAP</TD></TR></TBODY></TABLE><H4><B>Next:</B></H4>PCR with VR and VF2 and extract band containing correctly assembled plasmid for sequencing. If correctly assembled, perform golden gate assembly to add Promoter and terminator to each plant part. </DIV><DIV class="notebook"><H2>9 August 2017 to 10 August 2017</H2><H4 class="h4left"><B>Ben Bleitz</B></H4><H4 class="h4right"><B>Start: 2:00 PM of 9 August 2017</B></H4><H4><B>PCR and Gels of 18 Colonies from the PhoR Plate </B></H4><B>Purpose:</B> Correctly identify assembled of plasmid containing PhoR in universal acceptor   <H4><B>Protocol:</B></H4>PCRs were conducted for marked colonies 1-18 on PhoR
<TABLE><TBODY><TR><TD>Component</TD><TD>Volume</TD></TR><TR><TD>Taq 2x Master Mix</TD><TD>10μl</TD></TR><TR><TD>VR </TD><TD>0.8μl</TD></TR><TR><TD>VF2</TD><TD>0.8μl</TD></TR><TR><TD>MilliQ</TD><TD>8.4μl</TD></TR><TR><TD>Colonies 1-18</TD></TR></TBODY></TABLE>
Samples were run through the Taq2bkp program and then frozen for the next day
All Samples were run on a 1% agarose gel with .75 uL Ethidium Bromide with 2 Log Purple DNA ladder, the following day.<H4><B>Stop: 1:00 PM of 10 August 2017</B></H4><H4><B>Results:</B></H4><H4><B>Next:</B></H4>Inoculate culture from appropriate culture</DIV><DIV class="notebook"><H2>8 August 2017</H2><H4 class="h4left"><B>Ryan Baumann</B></H4><H4 class="h4right"><B>Start: 4:00PM</B></H4><H4><B>Colony PCR of 6/26/17 TNTR3 transformation</B></H4><B>Purpose:</B> To identify correctly assembled plasmid containing TNTr3 in the universal acceptor plasmid

20 μL Sample reaction
<TABLE><TBODY><TR><TD>Component</TD><TD>volume</TD></TR><TR><TD>Taq 2X Master Mix</TD><TD>10 μL</TD></TR><TR><TD>VR</TD><TD>0.8 μL</TD></TR><TR><TD>VF2</TD><TD>0.8 μL</TD></TR><TR><TD>MilliQ</TD><TD>8.4 μL</TD></TR><TR><TD>Colonies 1-16</TD></TR></TBODY></TABLE>

A positive Control was conducted and run

<TABLE><TBODY><TR><TD>Component</TD><TD>volume</TD></TR><TR><TD>Taq 2X Master Mix</TD><TD>10 μL</TD></TR><TR><TD>VR</TD><TD>0.8 μL</TD></TR><TR><TD>VF2</TD><TD>0.8 μL</TD></TR><TR><TD>MilliQ</TD><TD>7.4 μL</TD></TR><TR><TD>7/24/17 K608002</TD><TD>1 uL </TD></TR></TBODY></TABLE>

A negative control PCR was conducted and run

<TABLE><TBODY><TR><TD>Component</TD><TD>volume</TD></TR><TR><TD>Taq 2X Master Mix</TD><TD>10 μL</TD></TR><TR><TD>VR</TD><TD>0.8 μL</TD></TR><TR><TD>VF2</TD><TD>0.8 μL</TD></TR><TR><TD>MilliQ</TD><TD>8.4 μL</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE>

All Samples were run on a 1% agarose gel with .75 uL Ethidium Bromide wit 2 Log Purple DNA ladder.
<H4><B>Protocol:</B></H4>PCR's of 16 colonies were run with VR and VF2 primers, Taq 2x Master mix, and MilliQ Water

<H4><B>Notes:</B></H4><H4><B>Stop: 9:00 PM</B></H4><H4><B>Results:</B></H4>TBD<H4><B>Next:</B></H4>Examine Gel and find if colonies match correctly assembled plasmid</DIV><DIV class="notebook"><H2>3 August 2017</H2><H4 class="h4left"><B>Ben Bleitz</B></H4><H4 class="h4right"><B>Start: 2:00</B></H4><H4><B>Running Gel of Plant Pho, PhoB, PhoR, TNTR3</B></H4><B>Purpose:</B> Running Gels of repeat PCR<H4><B>Protocol:</B></H4><TABLE><TBODY><TR><TD>Lane</TD><TD>Component</TD></TR><TR><TD>1</TD><TD>10 uL NEB 2 Log Purple DNA ladder</TD></TR><TR><TD>3</TD><TD>10 uL 7/27/17 PCR 5</TD></TR><TR><TD>5</TD><TD>10 uL 7/27/17 PCR 1</TD></TR><TR><TD>6</TD><TD>10 uL 7/27/17 PCR 2</TD></TR><TR><TD>7</TD><TD>10 uL 7/27/17 PCR 3</TD></TR><TR><TD>8</TD></TR><TR><TD>9</TD><TD>10 uL 7/24/17 K608002 (+ Control)</TD></TR><TR><TD>10</TD><TD>10 uL MilliQ Water (- Control)</TD></TR></TBODY></TABLE><H4><B>Stop: 5:00</B></H4><H4><B>Results:</B></H4>Somehow managed to repeat mistakes from last week. Will repeat, for a third and hopefully final time for last week. <H4><B>Next:</B></H4>DONT MESS UP</DIV><DIV class="notebook"><H2>2 August 2017</H2><H4 class="h4left"><B>Ben Bleitz</B></H4><H4 class="h4right"><B>Start: 1:00 pm</B></H4><H4><B>PCR of 7/25 L1, 7/24 Plant Pho and 6/29 MP1 (PhoB), MP2 (PhoR), MP6 (TNTR3)</B></H4><B>Purpose:</B> Repeat of previous PCR (and eventual gel) from 27 June 2017. Gel scheduled to be completed following day (3 August 2017) <H4><B>Protocol:</B></H4>Plant Pho PCR:

<TABLE><TBODY><TR><TD>Component</TD><TD>Volume</TD></TR><TR><TD>Q5 2x Master Mix</TD><TD>12.5 μL</TD></TR><TR><TD>VR</TD><TD>1μL</TD></TR><TR><TD>VF2</TD><TD>1 μL</TD></TR><TR><TD>7/24/17 Plant Pho</TD><TD>3 μL</TD></TR><TR><TD>MilliQ </TD><TD>7.5 μL</TD></TR><TR><TD>Total</TD><TD>25 μL</TD></TR></TBODY></TABLE>

Pho B Pcr:

<TABLE><TBODY><TR><TD>Component</TD><TD>Volume</TD></TR><TR><TD>Q5 2x Master Mix</TD><TD>12.5 μL</TD></TR><TR><TD>VR</TD><TD>1μL</TD></TR><TR><TD>VF2</TD><TD>1 μL</TD></TR><TR><TD>6/29/17 MP1</TD><TD>1.5 μL</TD></TR><TR><TD>MilliQ </TD><TD>9 μL</TD></TR><TR><TD>Total</TD><TD>25 μL</TD></TR></TBODY></TABLE>

TNTR3 Pcr:

<TABLE><TBODY><TR><TD>Component</TD><TD>Volume</TD></TR><TR><TD>Q5 2x Master Mix</TD><TD>12.5 μL</TD></TR><TR><TD>VR</TD><TD>1μL</TD></TR><TR><TD>VF2</TD><TD>1 μL</TD></TR><TR><TD>6/29/17 MP6</TD><TD>2.0 μL</TD></TR><TR><TD>MilliQ </TD><TD>8.5 μL</TD></TR><TR><TD>Total</TD><TD>25 μL</TD></TR></TBODY></TABLE>

PhoR Pcr:

<TABLE><TBODY><TR><TD>Component</TD><TD>Volume</TD></TR><TR><TD>Q5 2x Master Mix</TD><TD>12.5 μL</TD></TR><TR><TD>VR</TD><TD>1μL</TD></TR><TR><TD>VF2</TD><TD>1 μL</TD></TR><TR><TD>6/29/17 MP2</TD><TD>1.5 μL</TD></TR><TR><TD>MilliQ </TD><TD>9 μL</TD></TR><TR><TD>Total</TD><TD>25 μL</TD></TR></TBODY></TABLE>

7/25/17 L1 

<TABLE><TBODY><TR><TD>Component</TD><TD>Volume</TD></TR><TR><TD>Q5 2x Master Mix</TD><TD>12.5 μL</TD></TR><TR><TD>VR</TD><TD>1μL</TD></TR><TR><TD>VF2</TD><TD>1 μL</TD></TR><TR><TD>7/25/17 L1</TD><TD>5 μL</TD></TR><TR><TD>MilliQ </TD><TD>5.5 μL</TD></TR><TR><TD>Total</TD><TD>25 μL</TD></TR></TBODY></TABLE><H4><B>Notes:</B></H4>PCR was completed. Gel will be ran following day<H4><B>Stop: 4:20 pm (#Blazeit)</B></H4><H4><B>Results:</B></H4>PCRs frozen after thermocycler completed. Will be determined upon gel electrophoresis completion on the following day. <H4><B>Products:</B></H4><TABLE><TBODY><TR><TD>Label </TD><TD>Source </TD><TD>Description</TD></TR><TR><TD>PCR 1</TD><TD> 7/24/17 Plant Pho</TD><TD>7/24/17 Plant Pho amplified with VR and VF2</TD></TR><TR><TD>PCR 2</TD><TD>6/29/17 MPI</TD><TD>6/29/17 MPI amplified with VR and VF2</TD></TR><TR><TD>PCR3</TD><TD>6/29/17 MP6</TD><TD>6/29/17 amplified with VR and VF2</TD></TR><TR><TD>PCR4</TD><TD>6/29/17 MP2</TD><TD>6/29/17 amplified with VR and VF2</TD></TR><TR><TD>PCR5</TD><TD>7/25/17 L1</TD><TD>7/25/17 L1 amplified with VR and VF2</TD></TR></TBODY></TABLE><H4><B>Next:</B></H4>Run gel</DIV><DIV class="notebook"><H2>27 June 2017</H2><H4 class="h4left"><B>Erin Nischwitz, Ben Bleitz</B></H4><H4 class="h4right"><B>Start: 11:00 am</B></H4><H4><B>PCR and gels of 7/25 L1, 7/24 Plant Pho and 6/29 MP1 (PhoB), MP2 (PhoR), MP6 (TNTR3)</B></H4><B>Purpose:</B> To verify assembly of these constructs <H4><B>Protocol:</B></H4>Five PCR's were ran with VR and VF2 using Q5 2x Master mix Protocol

Plant Pho PCR:

<TABLE><TBODY><TR><TD>Component</TD><TD>Volume</TD></TR><TR><TD>Q5 2x Master Mix</TD><TD>12.5 μL</TD></TR><TR><TD>VR</TD><TD>1μL</TD></TR><TR><TD>VF2</TD><TD>1 μL</TD></TR><TR><TD>7/24/17 Plant Pho</TD><TD>3 μL</TD></TR><TR><TD>MilliQ </TD><TD>7.5 μL</TD></TR><TR><TD>Total</TD><TD>25 μL</TD></TR></TBODY></TABLE>

Pho B Pcr:

<TABLE><TBODY><TR><TD>Component</TD><TD>Volume</TD></TR><TR><TD>Q5 2x Master Mix</TD><TD>12.5 μL</TD></TR><TR><TD>VR</TD><TD>1μL</TD></TR><TR><TD>VF2</TD><TD>1 μL</TD></TR><TR><TD>6/29/17 MP1</TD><TD>1.5 μL</TD></TR><TR><TD>MilliQ </TD><TD>9 μL</TD></TR><TR><TD>Total</TD><TD>25 μL</TD></TR></TBODY></TABLE>

TNTR3 Pcr:

<TABLE><TBODY><TR><TD>Component</TD><TD>Volume</TD></TR><TR><TD>Q5 2x Master Mix</TD><TD>12.5 μL</TD></TR><TR><TD>VR</TD><TD>1μL</TD></TR><TR><TD>VF2</TD><TD>1 μL</TD></TR><TR><TD>6/29/17 MP6</TD><TD>2.0 μL</TD></TR><TR><TD>MilliQ </TD><TD>8.5 μL</TD></TR><TR><TD>Total</TD><TD>25 μL</TD></TR></TBODY></TABLE>

PhoR Pcr:

<TABLE><TBODY><TR><TD>Component</TD><TD>Volume</TD></TR><TR><TD>Q5 2x Master Mix</TD><TD>12.5 μL</TD></TR><TR><TD>VR</TD><TD>1μL</TD></TR><TR><TD>VF2</TD><TD>1 μL</TD></TR><TR><TD>6/29/17 MP2</TD><TD>1.5 μL</TD></TR><TR><TD>MilliQ </TD><TD>9 μL</TD></TR><TR><TD>Total</TD><TD>25 μL</TD></TR></TBODY></TABLE>

7/25/17 L1 

<TABLE><TBODY><TR><TD>Component</TD><TD>Volume</TD></TR><TR><TD>Q5 2x Master Mix</TD><TD>12.5 μL</TD></TR><TR><TD>VR</TD><TD>1μL</TD></TR><TR><TD>VF2</TD><TD>1 μL</TD></TR><TR><TD>7/25/17 L1</TD><TD>5 μL</TD></TR><TR><TD>MilliQ </TD><TD>5.5 μL</TD></TR><TR><TD>Total</TD><TD>25 μL</TD></TR></TBODY></TABLE>

A 1% agarose gel was ran containing all five PCRs, a positive control, and a Negative control

<TABLE><TBODY><TR><TD>Lane</TD><TD>Component</TD></TR><TR><TD>1</TD><TD>10 uL NEB 2 Log Purple DNA ladder</TD></TR><TR><TD>3</TD><TD>10 uL 7/27/17 PCR 5</TD></TR><TR><TD>5</TD><TD>10 uL 7/27/17 PCR 1</TD></TR><TR><TD>6</TD><TD>10 uL 7/27/17 PCR 2</TD></TR><TR><TD>7</TD><TD>10 uL 7/27/17 PCR 3</TD></TR><TR><TD>8</TD><TD>10 uL 7/27/17 PCR 4</TD></TR><TR><TD>9</TD><TD>10 uL 7/24/17 K608002 (+ Control)</TD></TR><TR><TD>10</TD><TD>10 uL MilliQ Water (- Control)</TD></TR></TBODY></TABLE><H4><B>Notes:</B></H4>

-Accidentally used circular DNA as a positive control, so do not expect a valid + Control<H4><B>Stop: 3:30</B></H4><H4><B>Results:</B></H4>TBD<H4><B>Products:</B></H4><TABLE><TBODY><TR><TD>Label </TD><TD>Source </TD><TD>Description</TD></TR><TR><TD>PCR 1</TD><TD> 7/24/17 Plant Pho</TD><TD>7/24/17 Plant Pho amplified with VR and VF2</TD></TR><TR><TD>PCR 2</TD><TD>6/29/17 MPI</TD><TD>6/29/17 MPI amplified with VR and VF2</TD></TR><TR><TD>PCR3</TD><TD>6/29/17 MP6</TD><TD>6/29/17 amplified with VR and VF2</TD></TR><TR><TD>PCR4</TD><TD>6/29/17 MP2</TD><TD>6/29/17 amplified with VR and VF2</TD></TR><TR><TD>PCR5</TD><TD>7/25/17 L1</TD><TD>7/25/17 L1 amplified with VR and VF2</TD></TR></TBODY></TABLE><H4><B>Next:</B></H4>Trouble Shoot</DIV><DIV class="notebook"><H2>24 July 2017</H2><H4 class="h4left"><B>Ryan Baumann</B></H4><H4 class="h4right"><B>Start: 11:00 AM</B></H4><H4><B>Plasmid Miniprep of 7/21/17 Transformations</B></H4><B>Purpose:</B> Prepare plasmids for future use<H4><B>Protocol:</B></H4>Kitless miniprep protocol from igem lab manual version 2.5
Plasmids were suspended in 35 μl of 1x TE buffer<H4><B>Stop: 12:40PM</B></H4><H4><B>Results:</B></H4>Samples tested on Thermoscientific Nanodrop 1000

<TABLE><TBODY><TR><TD>Label</TD><TD>DNA Conc.</TD><TD>260/280</TD></TR><TR><TD>BBa_B0015</TD><TD>893.5 ng/uL</TD><TD>1.95</TD></TR><TR><TD>BBa_K608002</TD><TD>573.5 ng/uL</TD><TD>1.88</TD></TR><TR><TD>Plant Pho</TD><TD>165.7 ng/uL</TD><TD>1.93</TD></TR></TBODY></TABLE>
Skype Backpack
<H4><B>Products:</B></H4><TABLE><TBODY><TR><TD>Label</TD><TD>Source</TD><TD>Description</TD></TR><TR><TD>7/24/17 B0015</TD><TD>2017 Kit Plate 3 well 3F</TD><TD>B0015 Terminator in psb1c3</TD></TR><TR><TD>7/24/17 Plant Pho</TD><TD>Plant Pho top and bottom strand</TD><TD>Plant pho in UA plasmid</TD></TR><TR><TD>7/24/17 K608002</TD><TD>2017 Kit Plate 1 well 3O</TD><TD>K608002 Promoter and RBS in psb1c3</TD></TR></TBODY></TABLE><H4><B>Next:</B></H4></DIV><DIV class="notebook"><H2>25/June/2017</H2><H4 class="h4left"><B>Ben Bleitz</B></H4><H4 class="h4right"><B>Start: 1:30</B></H4><H4><B>Digestion and Ligation of BBa_K608002 and TNTR3_E.Coli</B></H4><B>Purpose:</B> To prepare DNA Samples for Transformations<H4><B>Protocol:</B></H4>Two double digestions were performed 
BBa_K608002 Double digestion
<TABLE><TBODY><TR><TD>Component </TD><TD>Volume</TD></TR><TR><TD>ECORI Enzyme</TD><TD>1 μl </TD></TR><TR><TD>SpeI Enzyme</TD><TD>1 μl</TD></TR><TR><TD>K608002 Insert</TD><TD>3.5 μl</TD></TR><TR><TD>Tango Buffer </TD><TD>2.5 μl</TD></TR><TR><TD>MilliQ H2O</TD><TD>17 μl</TD></TR></TBODY></TABLE>
TNT_E.Coli Double Digestion
<TABLE><TBODY><TR><TD>Component </TD><TD>Volume</TD></TR><TR><TD>ECORI Enzyme</TD><TD>1 μl</TD></TR><TR><TD>Xbal Enzyme </TD><TD>1 μl</TD></TR><TR><TD>TNTR3_E.Coli Vector</TD><TD>5 μl</TD></TR><TR><TD>Tango Buffer</TD><TD>2.5 μl</TD></TR><TR><TD>MilliQ H2O</TD><TD>15.5 μl</TD></TR></TBODY></TABLE>
Products were 25/7/2017 d1 and 25/7/2017 d2
Ligation was then performed
<TABLE><TBODY><TR><TD>Component</TD><TD>Volume</TD></TR><TR><TD>25/7/2017 d1 (insert)</TD><TD>1.5 μl</TD></TR><TR><TD>25/7/2017 d2 (vector)</TD><TD>2 μl</TD></TR><TR><TD>T4 Ligase Buffer</TD><TD>2 μl</TD></TR><TR><TD>T4 Ligase</TD><TD>1 μl</TD></TR><TR><TD>13.5 MilliQ H2O</TD><TD>13.5</TD></TR></TBODY></TABLE><H4><B>Stop: 5:30</B></H4><H4><B>Products:</B></H4><TABLE><TBODY><TR><TD>Label </TD><TD>Source</TD><TD>Description</TD></TR><TR><TD>25/7/2017 d1</TD><TD>BBa_K608002</TD><TD>K608002 cut with ECORI &amp; SPeI</TD></TR><TR><TD>25/7/2017 d2</TD><TD>TNTR3_E.Coli</TD><TD>TNTR3_E.Coli cut with ECORI &amp; Xbal</TD></TR><TR><TD>25/7/2017 L1</TD><TD>d1 and d2</TD><TD>DNA formed from d1 insert and d2 vector</TD></TR></TBODY></TABLE><H4><B>Next:</B></H4>PCR with VR and VF 2 to confirm products</DIV><DIV class="notebook"><H2>20/7/2017</H2><H4 class="h4left"><B>Ben Bleitz</B></H4><H4 class="h4right"><B>Start: 2:30</B></H4><H4><B>Bacterial  promoter/ terminator and plant pho Transformations</B></H4><B>Purpose:</B> Transform components for later prep and kit plate <H4><B>Protocol:</B></H4>Transformations were performed using NEB® 5-alpha Competent E. coli (High Efficiency) protocol.
<TABLE><TBODY><TR><TD>Transformant</TD><TD>DNA Volume</TD><TD>Plate Dilutions</TD></TR><TR><TD>Plant Pho</TD><TD>1μl</TD><TD>2 and 20%</TD></TR><TR><TD>BBa_B0015</TD><TD>1μl</TD><TD>2 and 20%</TD></TR><TR><TD>BBa_K608002</TD><TD>1μl</TD><TD>2 and 20%</TD></TR></TBODY></TABLE><H4><B>Stop: 5:30</B></H4><H4><B>Results:</B></H4>All samples in incubation. Results will be noted at the end of 24 hour incubation period<H4><B>Products:</B></H4><TABLE><TBODY><TR><TD>Label</TD><TD>Source</TD><TD>Description</TD></TR><TR><TD>BBa_B0015</TD><TD>2017 Kit Plate 3 well 3F</TD><TD>Transformed on chlorophenicol plate</TD></TR><TR><TD>BBa_K608002 </TD><TD>2017 Kit Plate 1 well 3O</TD><TD>Transformed on chlorophenicol plate</TD></TR><TR><TD>Plant Pho </TD><TD>Plant Pho top and bottom strand</TD><TD>Transformed on Blue/White chlorophenicol plate</TD></TR></TBODY></TABLE><H4><B>Next:</B></H4>Inoculate in broth culture with appropriate antibiotic for future miniprep.  </DIV><DIV class="notebook"><H2>10 July 2017</H2><H4 class="h4left"><B>Ryan Baumann</B></H4><H4 class="h4right"><B>Start: 12:30</B></H4><H4><B>Golden Gate assembly of Plant Pho Top and Bottom strand into UA plasmid</B></H4><B>Purpose:</B> To assemble PlantPho Oligos into UAP to prepare for assembly of Transcriptional Unit<H4><B>Protocol:</B></H4><TABLE><TBODY><TR><TD>UA MP1 (100ng)</TD><TD>0.3 μL</TD></TR><TR><TD>PlantPho Top Strand (100 ng)</TD><TD>1 μL</TD></TR><TR><TD>PlantPho Bottom Strand (100 ng)</TD><TD>1 μL</TD></TR><TR><TD>10X T4 DNA Ligase Buffer</TD><TD>2 μL</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>1 μL</TD></TR><TR><TD>BsmBI</TD><TD>1 μL</TD></TR><TR><TD>MilliQ H2O</TD><TD>13.7 μL</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE>

Ran at:
– 20 seconds @ 37°C,
– (3 minutes @ 37°C, 4 minutes @ 16°C) X26
– 5 minutes @ 50°C
– 5 minutes @ 80°C
– 5 minutes @ 16°C<H4><B>Stop: 5:00</B></H4><H4><B>Products:</B></H4><TABLE><TBODY><TR><TD>Label</TD><TD>Source</TD><TD>Description</TD></TR><TR><TD>7/10/17</TD><TD>Plant Pho Top &amp; Bottom Strand Oligos</TD><TD>Plant Pho Oligos cut with BsmBI and Ligated into UAP</TD></TR></TBODY></TABLE><H4><B>Next:</B></H4>Transform on to Chlor Blue white Screening Plates</DIV><DIV class="notebook"><H2>29 June 2017</H2><H4 class="h4left"><B>Ryan Baumann, Ben Bleitz</B></H4><H4 class="h4right"><B>Start: 1:30</B></H4><H4><B>Miniprep of 6/26/17 transformations </B></H4><B>Purpose:</B> Prepare plasmids for future use<H4><B>Protocol:</B></H4>Kitless miniprep protocol from igem lab manual version 2.5
Plasmids were suspended in 35 μl of 1x TE buffer<H4><B>Stop: 3:30</B></H4><H4><B>Results:</B></H4>Samples tested on Thermoscientific Nanodrop 1000
<TABLE><TBODY><TR><TD>Label</TD><TD>DNA Concentration (ng/μl)</TD><TD>260/280</TD></TR><TR><TD>MP1</TD><TD>412.7</TD><TD>1.84</TD></TR><TR><TD>MP2</TD><TD>399.6</TD><TD>1.83</TD></TR><TR><TD>MP3</TD><TD>290.2</TD><TD>1.82</TD></TR><TR><TD>MP4</TD><TD>722.4</TD><TD>1.80</TD></TR><TR><TD>MP5</TD><TD>406.4</TD><TD>1.81</TD></TR><TR><TD>MP6</TD><TD>307.4</TD><TD>1.84</TD></TR></TBODY></TABLE><H4><B>Products:</B></H4><TABLE><TBODY><TR><TD>Label </TD><TD>Source</TD><TD>Description</TD></TR><TR><TD>6/29/17 MP1</TD><TD>PhoB-VP64 Transformation plate</TD><TD>PhoB-VP64 transformed into UA Plasmid</TD></TR><TR><TD>6/29/17 MP2</TD><TD>Trg-PhoR Transformation plate</TD><TD>Trg-PhoR transformed into UA Plasmid</TD></TR><TR><TD>6/29/17 MP3</TD><TD>Golden Braid Omega 1 Plasmid Transformation plate</TD><TD>Golden Braid Omega 1 plasmid</TD></TR><TR><TD>6/29/17 MP4</TD><TD>Golden Braid Omega 2 Plasmid Transformation plate</TD><TD>Golden Braid Omega 2 plasmid</TD></TR><TR><TD>6/29/17 MP5</TD><TD>TNTR3 E. Coli Transformation plate</TD><TD>TNTR3 E. Coli in PSB1C3 backbone </TD></TR><TR><TD>6/29/17 MP6</TD><TD>TNTR3 Transformation plate</TD><TD>TNTR3 transformed into UA plasmid</TD></TR></TBODY></TABLE><H4><B>Next:</B></H4>TBD</DIV><DIV class="notebook"><H2>6/26/2017</H2><H4 class="h4left"><B>Ryan Baumann, Ben Bleitz, Erin Nischwitz</B></H4><H4 class="h4right"><B>Start: 5:00 pm</B></H4><H4><B>Chemical Transformations</B></H4><B>Purpose:</B> To transform PhoB-VP64, Trg_PhoR, Plant Pho, and TNTR3 on to chloramphenicol blue white screening plates.

To transform TNTR3_E Coli (6/22/17 L1) on to chloramphenicol plate

To transform Golden Braid Omega 1 and Omega =1 plasmids on to Streptomycin plates. 


<H4><B>Protocol:</B></H4>Transformations were performed using NEB® 5-alpha Competent E. coli (High Efficiency) protocol.

<TABLE><TBODY><TR><TD>Transformant</TD><TD>DNA volume</TD><TD>Plate Dillutions</TD></TR><TR><TD>TNTR3</TD><TD>1 μL</TD><TD>2% and 20%</TD></TR><TR><TD>Trg_PhoR</TD><TD>1 μL</TD><TD>2% and 20%</TD></TR><TR><TD>PhoB_VP64</TD><TD>1 μL</TD><TD>2% and 20%</TD></TR><TR><TD>Plant Pho</TD><TD>2 μL</TD><TD>2% and 20%</TD></TR><TR><TD>6/22/17 L1 (TNTR3 E coli)</TD><TD>3 μL</TD><TD>5% and 50%</TD></TR><TR><TD>Omega 1</TD><TD>1 μL</TD><TD>2% and 20%</TD></TR><TR><TD>Omega Two</TD><TD>1 μL</TD><TD>2% and 20%</TD></TR></TBODY></TABLE><H4><B>Stop: 7:45</B></H4><H4><B>Next:</B></H4>Check for colonies and inoculate cultures</DIV><DIV class="notebook"><H2>6/26/2017</H2><H4 class="h4left"><B>Golden Gate Assembly into Universal Acceptor Plasmid </B></H4><H4 class="h4right"><B>Start: 12:00pm</B></H4><H4><B>Assembly of PhoB_VP64, Trg_PhoR, TNTR3, and Plant Pho into BBa_P10500 plasmid. </B></H4><B>Purpose:</B> To assemble gblocks into UAP to prepare for assembly of Transcriptional Unit
<H4><B>Protocol:</B></H4>Trg_PhoR-Gblock:

<TABLE><TBODY><TR><TD>UA MP1 (100ng) </TD><TD>0.30 μL</TD></TR><TR><TD>Trg_PhoR (100ng)</TD><TD>1 μL</TD></TR><TR><TD>10X T4 DNA Ligase Buffer</TD><TD>2 μL</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>1 μL</TD></TR><TR><TD>BsmBI</TD><TD>1 μL</TD></TR><TR><TD>MilliQ H2O</TD><TD>14.7 μL</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE>

PhoB_VP64-gblock

<TABLE><TBODY><TR><TD>UA MP1 (100ng) </TD><TD>0.30 μL</TD></TR><TR><TD>PhoB_VP64 (100ng)</TD><TD>1 μL</TD></TR><TR><TD>10X T4 DNA Ligase Buffer</TD><TD>2 μL</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>1 μL</TD></TR><TR><TD>BsmBI</TD><TD>1 μL</TD></TR><TR><TD>MilliQ H2O</TD><TD>14.7 μL</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE>

TNT_R3-gblock

<TABLE><TBODY><TR><TD>UA MP1 (100ng) </TD><TD>0.30 μL</TD></TR><TR><TD>TNT_R3 (100ng)</TD><TD>1 μL</TD></TR><TR><TD>10X T4 DNA Ligase Buffer</TD><TD>2 μL</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>1 μL</TD></TR><TR><TD>BsmBI</TD><TD>1 μL</TD></TR><TR><TD>MilliQ H2O</TD><TD>14.7 μL</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE>

PlantPho Oligos Top and Bottom Strands

<TABLE><TBODY><TR><TD>UA MP1 (100ng)</TD><TD>0.30 μL</TD></TR><TR><TD>PlantPho Top Strand (100 ug)</TD><TD>1μL</TD></TR><TR><TD>PlantPho Bottom Strand (100 ug)</TD><TD>1μL</TD></TR><TR><TD>10X T4 DNA Ligase Buffer</TD><TD>2 μL</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>1μL</TD></TR><TR><TD>BsmBI</TD><TD>1μL</TD></TR><TR><TD>MilliQ H2O</TD><TD>13.7 μL</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE>

All Samples Ran: 

- 20 seconds @ 37°C,
- (3 minutes @ 37°C, 4 minutes @ 16°C) X26
- 5 minutes @ 50°C
- 5 minutes @ 80°C
- 5 minutes @ 16°C





<H4><B>Notes:</B></H4>100 μg of top and bottom strand oligos were accidentally used instead of 100ng.<H4><B>Stop: 4:45 PM</B></H4><H4><B>Products:</B></H4><TABLE><TBODY><TR><TD>Label</TD><TD>Source </TD><TD>Description</TD></TR><TR><TD>6/26 TNTR3</TD><TD>TNTR3-GBlock</TD><TD>TNTR3-GBlock cut with BsmBI and Ligated into UAP</TD></TR><TR><TD>6/26 Plant Pho</TD><TD>Plant Pho Top &amp; Bottom Strand Oligos</TD><TD>Plant Pho Oligos cut with BsmBI and Ligated into UAP</TD></TR><TR><TD>6/26 PhoR</TD><TD>Trg_PhoR-Gblock</TD><TD>Trg_PhoR-Gblock cut with BsmBI and Ligated into UAP</TD></TR><TR><TD>6/26 PhoB</TD><TD>PhoB_VP64 Gblock</TD><TD>PhoB_VP64 Gblock cut with BsmBI and Ligated into UAP</TD></TR></TBODY></TABLE><H4><B>Next:</B></H4>Transform assembled plasmids on to Blue White Screening Plates with appropriate antibiotic resistance</DIV><DIV class="notebook"><H2>6/26/2017</H2><H4 class="h4left"><B>Golden Gate Assembly into Universal Acceptor Plasmid </B></H4><H4 class="h4right"><B>Start: 12:00pm</B></H4><H4><B>Assembly of PhoB_VP64, Trg_PhoR, TNTR3, and Plant Pho into BBa_P10500 plasmid. </B></H4><B>Purpose:</B> To assemble gblocks into UAP to prepare for assembly of Transcriptional Unit
<H4><B>Protocol:</B></H4>Trg_PhoR-Gblock:

<TABLE><TBODY><TR><TD>UA MP1 (100ng) </TD><TD>0.30 μL</TD></TR><TR><TD>Trg_PhoR (100ng)</TD><TD>1 μL</TD></TR><TR><TD>10X T4 DNA Ligase Buffer</TD><TD>2 μL</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>1 μL</TD></TR><TR><TD>BsmBI</TD><TD>1 μL</TD></TR><TR><TD>MilliQ H2O</TD><TD>14.7 μL</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE>

PhoB_VP64-gblock

<TABLE><TBODY><TR><TD>UA MP1 (100ng) </TD><TD>0.30 μL</TD></TR><TR><TD>PhoB_VP64 (100ng)</TD><TD>1 μL</TD></TR><TR><TD>10X T4 DNA Ligase Buffer</TD><TD>2 μL</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>1 μL</TD></TR><TR><TD>BsmBI</TD><TD>1 μL</TD></TR><TR><TD>MilliQ H2O</TD><TD>14.7 μL</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE>

TNT_R3-gblock

<TABLE><TBODY><TR><TD>UA MP1 (100ng) </TD><TD>0.30 μL</TD></TR><TR><TD>TNT_R3 (100ng)</TD><TD>1 μL</TD></TR><TR><TD>10X T4 DNA Ligase Buffer</TD><TD>2 μL</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>1 μL</TD></TR><TR><TD>BsmBI</TD><TD>1 μL</TD></TR><TR><TD>MilliQ H2O</TD><TD>14.7 μL</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE>

PlantPho Oligos Top and Bottom Strands

<TABLE><TBODY><TR><TD>UA MP1 (100ng)</TD><TD>0.30 μL</TD></TR><TR><TD>PlantPho Top Strand (100ng)</TD><TD>1μL</TD></TR><TR><TD>PlantPho Bottom Strand (100ng)</TD><TD>1μL</TD></TR><TR><TD>10X T4 DNA Ligase Buffer</TD><TD>2 μL</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>1μL</TD></TR><TR><TD>BsmBI</TD><TD>1μL</TD></TR><TR><TD>MilliQ H2O</TD><TD>13.7 μL</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE>

All Samples Ran: 

- 20 seconds @ 37°C,
- (3 minutes @ 37°C, 4 minutes @ 16°C) X26
- 5 minutes @ 50°C
- 5 minutes @ 80°C
- 5 minutes @ 16°C





<H4><B>Stop: 4:45 PM</B></H4><H4><B>Products:</B></H4><TABLE><TBODY><TR><TD>Label</TD><TD>Source </TD><TD>Description</TD></TR><TR><TD>6/26 TNTR3</TD><TD>TNTR3-GBlock</TD><TD>TNTR3-GBlock cut with BsmBI and Ligated into UAP</TD></TR><TR><TD>6/26 Plant Pho</TD><TD>Plant Pho Top &amp; Bottom Strand Oligos</TD><TD>Plant Pho Oligos cut with BsmBI and Ligated into UAP</TD></TR><TR><TD>6/26 PhoR</TD><TD>Trg_PhoR-Gblock</TD><TD>Trg_PhoR-Gblock cut with BsmBI and Ligated into UAP</TD></TR><TR><TD>6/26 PhoB</TD><TD>PhoB_VP64 Gblock</TD><TD>PhoB_VP64 Gblock cut with BsmBI and Ligated into UAP</TD></TR></TBODY></TABLE><H4><B>Next:</B></H4>Transform assembled plasmids on to Blue White Screening Plates with appropriate antibiotic resistance</DIV><DIV class="notebook"><H2>23/6/17</H2><H4 class="h4left"><B>Ben Bleitz</B></H4><H4 class="h4right"><B>Start: 2:50pm</B></H4><H4><B>Ligation of PsbIC3 (Vector) and 6/21/17 DI (insert)</B></H4><B>Purpose:</B> To create recombinant DNA for transformation <H4><B>Protocol:</B></H4><TABLE><TBODY><TR><TD>T4 Ligase Buffer</TD><TD>2μL</TD></TR><TR><TD>PsbIC3 (Vector)</TD><TD>4μL</TD></TR><TR><TD>6/21/17 DI (Insert)</TD><TD>3μL</TD></TR><TR><TD>MilliQ H2O</TD><TD>10μL</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>1μL</TD></TR></TBODY></TABLE>
Thermal Cycler
16°C for 30 minutes
80°C for 20 minutes<H4><B>Stop: 4:00pm</B></H4><H4><B>Results:</B></H4>6/23/17 L1 <H4><B>Next:</B></H4>E. Coli Transformation </DIV><DIV class="notebook"><H2>6/21/2017</H2><H4 class="h4left"><B>Ben Bleitz</B></H4><H4 class="h4right"><B>Start: 2:00</B></H4><H4><B>Double Digestion of TNT R3_E.Coli</B></H4><B>Purpose:</B> To prepare TNT R3_E.Coli for submission into the registry <H4><B>Protocol:</B></H4><TABLE><TBODY><TR><TD>Volume </TD><TD>Component </TD></TR><TR><TD>2.5 μL</TD><TD>10X Tango Buffer</TD></TR><TR><TD>1 μL</TD><TD>ECORI</TD></TR><TR><TD>1 μL</TD><TD>PSTI</TD></TR><TR><TD>15 μL</TD><TD>TNTR3_E.Coli (10 ng/μL)</TD></TR><TR><TD>5.5 μL</TD><TD>MilliQ H2O</TD></TR><TR><TD>25 μL</TD><TD>in Total </TD></TR></TBODY></TABLE>

Ran @ 37° C for 1 hour 
Heat kill @ 80°C for 30 minutes<H4><B>Stop: 4:00 PM</B></H4><H4><B>Products:</B></H4><TABLE><TBODY><TR><TD>Label</TD><TD>Source</TD><TD>Description </TD></TR><TR><TD>6/21/17 D1</TD><TD>TNTR3_E.Coli </TD><TD>TNTR3_E.Coli cut with ECORI and PSTI</TD></TR></TBODY></TABLE><H4><B>Next:</B></H4>Ligate into PsbIC3 </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>