<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Nanjing-China_Basic_Part skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Nanjing-China/Basic Part</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Home</TITLE></P><DIV class="header"><DIV class="menu_sub"><NAV class="navbar navbar-default "><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2016.igem.org/Team:Nanjing-China" class="active wow fadeInDownBig" data-wow-delay=".1s"><H2 style="padding-bottom:2px;">Home</H2></A></LI><LI><A href="https://2016.igem.org/Team:Nanjing-China/project" class="wow fadeInDownBig" data-wow-delay=".3s"><H2 style="padding-bottom:2px;">Project</H2></A></LI><LI><A href="https://2016.igem.org/Team:Nanjing-China/practices" class="wow fadeInDownBig" data-wow-delay=".5s"><H2 style="padding-bottom:2px;">Human Practices</H2></A></LI><LI><A href="https://2016.igem.org/Team:Nanjing-China/judging" class="wow fadeInDownBig" data-wow-delay=".7s"><H2 style="padding-bottom:2px;">Judging</H2></A></LI><LI><A href="https://2016.igem.org/Team:Nanjing-China/team" class="wow fadeInDownBig" data-wow-delay=".9s"><H2 style="padding-bottom:2px;">Team</H2></A></LI></UL></DIV><DIV class="header-content_sub"><H4 style="padding-bottom:50px; padding-top:50px;">Basic Part</H4></DIV></DIV></NAV></DIV></DIV><DIV class="description"><DIV class="container_1"><DIV class="container_1_title"><H3 style="padding-bottom:20px; padding-top:20px;"><STRONG>Part description</STRONG></H3></DIV><DIV class="container_1_content"><H6>This year we successfully constructed seven parts which are listed as follows:</H6><P> </P><H6>HyaA <A href="http://parts.igem.org/Part:BBa_K1958001">http://parts.igem.org/Part:BBa_K1958001</A></H6><P> </P><H6>HyaB <A href="http://parts.igem.org/Part:BBa_K1958002">http://parts.igem.org/Part:BBa_K1958002</A></H6><P> </P><H6>HyaC <A href="http://parts.igem.org/Part:BBa_K1958003">http://parts.igem.org/Part:BBa_K1958003</A></H6><P> </P><H6>HyaD <A href="http://parts.igem.org/Part:BBa_K1958004">http://parts.igem.org/Part:BBa_K1958004</A></H6><P> </P><H6>HyaE <A href="http://parts.igem.org/Part:BBa_K1958000">http://parts.igem.org/Part:BBa_K1958000</A></H6><P> </P><H6>HyaF <A href="http://parts.igem.org/Part:BBa_K1958006">http://parts.igem.org/Part:BBa_K1958006</A></H6><P> </P><H6>PbrR <A href="http://parts.igem.org/Part:BBa_K1958007">http://parts.igem.org/Part:BBa_K1958007</A></H6></DIV></DIV></DIV><DIV class="description"><DIV class="container_1"><DIV class="container_1_title"><H3 style="padding-bottom:20px; padding-top:20px;"><STRONG>Characterization</STRONG></H3></DIV><DIV class="container_1_content"><P> </P><H3>1. Part Name: BBa_K1958007 </H3><P> </P><H6>Part Type: Coding</H6><P> </P><H6>Short Description: PbrR -&gt; C. <EM>metallidurans</EM></H6><P> </P><H6>Long Description:</H6><P> </P><H6>PbrR is amplified from <EM>Cupriavidus metallidurans</EM> CH34 (formerly <EM>Ralstonia metallidurans</EM>) genome. </H6><P> </P><H6>PbrR, together with its homologues in the same bacterium, are the only known lead(II)-specific binding protein found in nature. PbrR binds lead(II) 1000-fold more selectively over other metal ions such as mercury(II), zinc(II), copper(II), nickel(II), and silver(I). Besides lead(II), PbrR also shows the binding capacity of cadmium(II), but slightly lower than that of lead(II). The binding capacity may be concerned with three conserved Cys residues: Cys78, Cys113 and Cys122.</H6><P> </P><H6>Because the gene encoding PbrR contains a digest site of Pst I (-CTGCAG-), to fit the construction of standard biobrick, we change those site from -CTGCAG- to -CTGCAA- by point mutation.
The source of this part: <EM>Cupriavidus metallidurans</EM> CH34
</H6><P> </P><H6>Our project is to construct an artificial solar-driven system in model organisms. There are two main problems to be solved when we artificially construct solar-driven systems: media for electron transportation and enzymes to harvest electrons to chemical bonds. We successfully solved the two problems in E.coli and testes the efficiency of our system using hydrogen production.</H6><P> </P><H6>To solve the problem of electron transduction, we fused PbrR (BBa_K1958007), which is another basic part of our project this year, with a previously existing basic part OmpA (BBa_J36836). PbrR is a kind of metal binding protein which can adsorb cadmium ions. OmpA is one of the outer membrane protein of <EM>E.coli</EM>. By fusing these two proteins together, we can construct a Biobrick device that can display PbrR onto the surface of <EM>E.coli</EM> so that cadmium ions can be fixed! When we add S2- into the culture medium, CdS can be fixed firmly onto the surface of <EM>E.coli</EM> and conduct electron transportation into the bacteria. The detailed experiment results are listed as follows.</H6><P> </P><H6>The PbrR (BBa_K1958007) based artificial PS system is constructed according to our design. The first step to verify an artificial PS system is to test its photocatalytic capability. The redox dye methyl viologen (MV) is a well-establish indicator. In a photocatalytic reaction, photons spark excited electron on the semiconductor which then reduce MV from its +2 oxidized state to +1 reduced state (figure 1). Increased concentration of reduce MV leads to an increase of absorbance at 605nm. </H6><P> </P><H4>Figure 1 photocatalytic reduction of MV by TiO<SUB>2</SUB> and induced CdS nanoparticles</H4><P> </P><H6>MV reduction was confirmed with TiO2 nanoparticles as the positive control. The reaction mixture consist of 100mM Tris-HCl (pH 7), 150mM NaCl, 5% glycerol, 100 mM ascorbic acid and 5mM MV<SUB>2+</SUB> with or without TiO<SUB>2</SUB> particles. Light irradiation resulted in an absorbance peak around 2 minute after initiation (Figure 1A). The reaction was fast and completed within ten minutes as absorbance drops to horizontal because of the exhaustion of sacrificial electron donor ascorbic acid and the oxidation of MV+ by air oxygen.</H6><P> </P><H6>To confirm the capability of our CdS system based on PbrR we conducted the same photocatalytic assay. Bacteria were divided into three groups. Bacteria were induced to express OmpA-PbrR protein and cultured with both Cd<SUB>2+</SUB> and S<SUB>2-</SUB> in the experiment group. Groups that either lacked induced expression or necessary ions to build semiconductors were the supposed negative control. We found that illumination resulted in a same increasing trend in experiment group (Figure 2B). This confirmed the photocatalytic capability of our PbrR based precipitation of semiconductors.</H6><P> </P><H4>Figure 2 gas chromatography test on hydrogen evolution of hydrogenase with CdS particles</H4><P> </P><H6>To determine the CdS system is as well capable with hydrogenase, we measured H<SUB>2</SUB> evolution amount with CdS and hydrogenase. The formation of <EM>in situ</EM> CdS nanoparticles lead to improved H<SUB>2</SUB> production marking 0.93% in the head space against 0.73% in the control group (Figure 2).</H6><P> </P><H4>Figure 3 TEM image of native <EM>E.coli</EM> cells (right) and cells with <EM>in situ</EM> formed CdS nanoparticles (left)</H4><P> </P><H6>We also revealed clusters of nanoparticles on the surface of <EM>E.coli</EM> using transmission electron microscopy. Black dots revealed under TEM indicate nanoparticle of size smaller than 10nm (Figure 3 left), compared to the smooth outer membrane of native <EM>E.coli</EM> cells (Figure 3 right).</H6><P> </P><P> </P><H3>2. Part Name: BBa_K1958001 </H3><P> </P><H6>Part Type: Coding</H6><P> </P><H6>Short Description: HyaA -&gt; <EM>E. coli</EM></H6><P> </P><H6>Long Description: </H6><P> </P><H6>HyaA is the small subunit of hydrogenase 1 (Hyd-1, from MBH family) from <EM>Escherichia coli</EM> genome. The hya operon, which contains gene for the two HYD1 structural subunits and four additional genes (HyaA-F) was mapped at 22min on the <EM>E.coli</EM> chromosome. And hyaA is the first gene of hya operon, sharing the last few base pairs with hyaB. Under carbon starvation upon entering stationary phase, expression of the hya operon, which encodes Hyd-1, is regulated by sigma-38. The small subunit exhibits sequence homologies to subunits of NADH: ubiquinone oxidoreductase (Complex I). This small subunit of this [Ni-Fe] hydrogenase contains 4 Fe-S clusters ([Fe3S4], [Fe4S4]d, [Fe4S3]) that play a role in ET to and from the large subunit for H2 reduction/oxidation. The supernumerary cysteine in the proximal cluster of the small subunit also plays a role in O2 tolerance.</H6><P> </P><H6>The source of this part: <EM>Escherichia coli</EM> BL21.</H6><P> </P><H4>Figure 1 <EM>hya</EM> gene cluster on the genome of <EM>E.coli</EM></H4><P> </P><H6>This part HyaA encodes for the first subunit of hydrogenase 1 in <EM>E.coli</EM>. </H6><P> </P><H6>First the recombinant plasmid with genes encoding E.coli hydrogenase 1 was constructed. This enzyme is encoded by <EM>hya</EM> operon where lie in all six genes named <EM>hyaABCDEF</EM> (BBa_K1958001, BBa_K1958002, BBa_K1958003, BBa_K1958004, BBa_K1958000, BBa_K1958006) on the genome of <EM>E.coli.</EM> The sequence was PCR amplified according to the sequence from our vessel <EM>E.coli</EM> strain BL21(DE3). We have completed the plasmid pET28a with <EM>hya</EM> cluster promoted by a T7 promoter. The following figure shows the result of enzyme digestion assay of recombinant plasmid in which our target gene cluster displayed a band around 5.5kb, which indicated successful hydrogenase expression.</H6><P> </P><H4>Figure 2 (A) Enzyme digestion assay of recombinant plasmid, showing the band <EM>hyaA-F</EM>. </H4><H4>(B) SDS-PAGE analysis of recombinant <EM>E.coli</EM>, showing the band of purified HyaA and overexpressed hydrogenase in recombinant strain.</H4><P> </P><H6>To detect induced expression of Ec-Hyd1 we performed SDS-PAGE assay. Note that only the small subunit HyaA had a His-tag for purification on this plasmid. The purified subunit ran as the reference for hydrogenase. The assay showed that recombinant strain overexpressed hydrogenase compared to <EM>E.coli</EM> with empty pET28a plasmids.</H6><P> </P><H4>Figure 3 qualitative test of hydrogenase under anaerobic conditions</H4><P> </P><H6>We determined that our enzyme is effective using both qualitative and quantitative tests under anaerobic condition. Solutions for reaction were flushed with nitrogen and then reaction cells were vacuumed and sealed tight. The qualitative test was done after 20h of anaerobic culture (Figure 3A). We found that the recombinant strain overexpresses hydrogenase produces more bubbles than control group. We assumed that this was because more hydrogen evolves in experiment group. WO3 is a redox dye which determines the existence of reduction force in the environment with a color change to blue, compared to its original green. Again in 20h of anaerobic culture (Figure 3B), the WO3 powders displayed a darker color in the recombinant strain than that of native E.coli strain. We assumed that recombinant strain has created more reduction force than native E.coli strain.</H6><P> </P><H4>Figure 4 gas chromatography quantitative test of hydrogenase under anaerobic condition</H4><P> </P><H6>To measure the exact quantity of hydrogen produced in 20h of anaerobic culture, a gas chromatography (GC) test on samples taken from reaction flask headspace was done accordingly later. Nitrogen was the gas carrier and bulk H2 was run as the reference, which showed a peak at approximately 1.4 minute. According to our result, the control group produced 0.58% hydrogen in headspace under 1.96% oxygen proportion which marked the level of native fermentation of E.coli. The recombinant group obviously produced more hydrogen which made up 1.25% in the headspace under 2.06% oxygen level. This is double the production of native fermentation which thus proves that our enzyme is effective under anaerobic conditions.</H6><P> </P><P> </P><H3>3. Part Name: BBa_K1958002</H3><P> </P><H6>Part Type: Coding</H6><P> </P><H6>Short Description: HyaB -&gt; E. coli</H6><P> </P><H6>Long Description:</H6><P> </P><H6>HyaB is the large subunit of hydrogenase 1 (Hyd-1) from Escherichia coli genome. The hya operon, which contains gene for the two HYD1 structual subunits and four additional genes (HyaA-F) was mapped at 22min on the E.coli chromosome. And hyaB is the second gene of hya operon, sharing the first few base pairs with hyaA and the last few base pairs with hyaC. This large subunit of this [Ni-Fe] hydrogenase contains the active site. The large subunit exhibit sequence homologies to subunits of NADH: ubiquinone oxidoreductase (Complex I).[1]</H6><P> </P><H6>The source of this part: Escherichia coli BL21</H6><P> </P><H6>This part HyaB encodes for the second subunit of hydrogenase 1 in E.coli whose expression and function have been tested. See the results and figures listed in HyaA (BBa_K1958001) above for more information.</H6><P> </P><P> </P><H3>4. Part Name: BBa_K1958003</H3><P> </P><H6>Part Type: Coding</H6><P> </P><H6>Short Description: HyaC -&gt; E. coli</H6><P> </P><H6>Long Description:</H6><P> </P><H6>HyaC is a subunit of hydrogenase 1 (Hyd-1) from Escherichia coli genome. The hya operon, which contains gene for the two HYD1 structual subunits and four additional genes (HyaA-F) was mapped at 22min on the E.coli chromosome. And hyaC is the third gene of hya operon, sharing the first few base pairs with hyaB and the last few base pairs with hyaD. HyaC is a cytochrome that anchors the [NiFe]-hydrogenase to the membrane.</H6><P> </P><H6>The source of this part: Escherichia coli BL21</H6><P> </P><H6>This part HyaC encodes for the third subunit of hydrogenase 1 in E.coli whose expression and function have been tested. See the results and figures listed in HyaA (BBa_K1958001) above for more information.</H6><P> </P><P> </P><H3>5. Part Name: BBa_K1958004</H3><P> </P><H6>Part Type: Coding</H6><P> </P><H6>Short Description: HyaD -&gt; E. coli</H6><P> </P><H6>Long Description:</H6><P> </P><H6>HyaD is a subunit of hydrogenase 1 (Hyd-1) from Escherichia coli genome. The hya operon, which contains gene for the two HYD1 structual subunits and four additional genes (HyaA-F) was mapped at 22min on the E.coli chromosome. And hyaD is the fourth gene of hya operon, sharing the first few base pairs with hyaC and the last few base pairs with hyaE. HyaD is the specific protease required for large subunit maturation-terminal processing. HyaD is a hydrogenase maturase.</H6><P> </P><H6>The source of this part: Escherichia coli BL21</H6><P> </P><H6>This part HyaD encodes for the fourth subunit of hydrogenase 1 in E.coli whose expression and function have been tested. See the results and figures listed in HyaA (BBa_K1958001) above for more information.</H6><P> </P><P> </P><H3>6. Part Name: BBa_K19580000</H3><P> </P><H6>Part Type: Coding</H6><P> </P><H6>Short Description: HyaE -&gt; E. coli</H6><P> </P><H6>Long Description:</H6><P> </P><H6>HyaE is a subunit of hydrogenase 1 (Hyd-1) from Escherichia coli genome. The hya operon, which contains gene for the two HYD1 structual subunits and four additional genes (HyaA-F) was mapped at 22min on the E.coli chromosome. And hyaE is the fifth gene of hya operon, sharing the first few base pairs with hyaD and the last few base pairs with hyaF. HyaE is a hydrogenase maturase. It is important to note that hyaE is not required for the assembly of standard O2-sensitive hydrogenases (Hyd-2 and others) and are apparently only involved in the assembly of O2-tolerant respiratory enzymes.</H6><P> </P><H6>The source of this part: Escherichia coli BL21</H6><P> </P><H6>This part HyaE encodes for the fifth subunit of hydrogenase 1 in E.coli whose expression and function have been tested. See the results and figures listed in HyaA (BBa_K1958001) above for more information.</H6><P> </P><P> </P><H3>7. Part Name: BBa_K1958006</H3><P> </P><H6>Part Type: Coding</H6><P> </P><H6>Short Description: HyaF -&gt; E. coli</H6><P> </P><H6>Long Description:</H6><P> </P><H6>HyaF is a subunit of hydrogenase 1 (Hyd-1) from Escherichia coli genome. The hya operon, which contains gene for the two HYD1 structual subunits and four additional genes (HyaA-F) was mapped at 22min on the E.coli chromosome. And hyaE is the sixth gene of hya operon, sharing the first few base pairs with hyaD and the last few base pairs with hyaF. HyaF is a hydrogenase maturase. Similar to hyaE, hyaF is not required for the assembly of standard O2-sensitive hydrogenases (Hyd-2 and others) and are apparently only involved in the assembly of O2-tolerant respiratory enzymes. HyaF interacts with HyaE to form a complex together with the small subunit during assembly. </H6><P> </P><H6>The source of this part: Escherichia coli BL2</H6><P> </P><H6>This part HyaF encodes for the sixth subunit of hydrogenase 1 in E.coli whose expression and function have been tested. See the results and figures listed in HyaA (BBa_K1958001) above for more information.</H6></DIV></DIV></DIV><DIV class="menu_on-top_1" id="menu_on-top_js"><DIV class="menu_on-top_2"><A href="https://2016.igem.org/Team:Nanjing-China/Engagement" style="display:block; margin-right:10%" class="top_menu_1_1"><H6>Engagement</H6></A><A href="https://2016.igem.org/Team:Nanjing-China/Basic_Part" style="display:block;" class="top_menu_1_1"><H6>Basic Parts</H6></A><A href="https://2016.igem.org/Team:Nanjing-China/Safety" style="display:block;" class="top_menu_1_1"><H6>Biosafety</H6></A><A href="https://2016.igem.org/Team:Nanjing-China/judging" style="display:block;" class="top_menu_1_1"><H5>Judging</H5></A></DIV></DIV><DIV class="description" id="e"><DIV class="container_1"><DIV class="container_1_title"><H3 style="padding-bottom:20px; padding-top:20px;"><STRONG>Quick Link</STRONG></H3></DIV><DIV class="container_1_content"><H5>* Put your mouse on those cards to see what will happen</H5></DIV></DIV></DIV><DIV class="bottom_main"><DIV class="bottom_contact"><H4 style="padding-top:50px;padding-bottom:20px;">CONTACT</H4><H5>Address</H5><H6><UL><LI>Life Science Department</LI><LI>#163 Xianlin Blvd, Qixia District</LI><LI>Nanjing University</LI><LI>Nanjing, Jiangsu Province</LI><LI>P.R. of China</LI><LI>Zip: 210046</LI></UL></H6><H5>Email</H5><H6><UL><LI>nanjing_china@163.com</LI></UL></H6><H5>Social media</H5><H6><UL><LI><A href="https://www.facebook.com/IGEM-Nanjing-China-392481144280345/">Facebook</A></LI><LI><A href="https://twitter.com/nanjing__china">Twitter</A></LI></UL></H6></DIV><DIV class="bottom_sponsors"><H4 style="padding-top:50px;padding-bottom:20px;">SPONSORS</H4><DIV class="bottom_sponsors_1"><DIV class="bottom_sponsors_1_content"><H6>南京大学
          生命科学学院</H6><H6>SCHOOL OF LIFE SCIENCES
        NANJING UNIVERSITY</H6></DIV></DIV><DIV class="bottom_sponsors_2"><DIV class="bottom_sponsors_2_content"><H6>南京大学</H6><H6>NANJING UNIVERSITY</H6></DIV></DIV><DIV class="bottom_sponsors_3"><DIV class="bottom_sponsors_3_content"><H6>南京大学
          化学化工学院</H6><H6>SCHOOL OF CHEMISTRY AND CHEMICAL ENGINEERING
          NANJING UNIVERSITY
        </H6></DIV></DIV><DIV class="bottom_sponsors_4"><DIV class="bottom_sponsors_4_content"><H6>南京大学
        化学与生物医学科学研究所</H6><H6>INSTITUTE OF CHEMISTRY AND BIOMEDICAL SCIENCES
          NANJING UNIVERSITY
        </H6></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>