<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bilkent-UNAMBG_Project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bilkent-UNAMBG/Project</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="header" class="container"><DIV id="menux"><UL><LI><A href="https://2017.igem.org/Team:Bilkent-UNAMBG" title="">Home</A></LI><LI class="dropdown"><A href="#">Team</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Bilkent-UNAMBG/Team">Team Members</A><A href="https://2017.igem.org/Team:Bilkent-UNAMBG/Attributions">Attributions</A><A href="https://2017.igem.org/Team:Bilkent-UNAMBG/Collaborations">Collaborations</A></DIV></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Bilkent-UNAMBG/Project">Project</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Bilkent-UNAMBG/Project#desc">Description</A><A href="https://2017.igem.org/Team:Bilkent-UNAMBG/Project#const">Constructs</A><A href="https://2017.igem.org/Team:Bilkent-UNAMBG/Project#notebooks">Notebooks</A><A href="https://2017.igem.org/Team:Bilkent-UNAMBG/Project#modeling">Modeling</A><A href="https://2017.igem.org/Team:Bilkent-UNAMBG/Project#protocols">Protocols</A><A href="https://2017.igem.org/Team:Bilkent-UNAMBG/InterLab">InterLab</A></DIV></LI><LI><A href="https://2017.igem.org/Team:Bilkent-UNAMBG/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Bilkent-UNAMBG/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:Bilkent-UNAMBG/Results">Results</A></LI><LI class="dropdown"><A href="#">Human Practices</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Bilkent-UNAMBG/HP/Silver">Human Practice</A><A href="https://2017.igem.org/Team:Bilkent-UNAMBG/HP/Gold_Integrated">Integrated Human Practice</A></DIV></LI></UL></DIV></DIV><DIV class="projectbg"><H1>Project</H1></DIV><DIV id="page" class="container"><DIV id="desc" class="title"><H2>Description</H2></DIV><H4>Introduction</H4><P>Cancer is one of the most common and lethal disease in the world and it is always highlighted that early diagnosis is crucial in terms of cancer treatment. It is often invasive and painful to diagnose cancer with the existing methods. At this point, breath test appears as a promising non-invasive and real-time technique which allows the monitoring of metabolic status.</P><H4>Cancer and VOCs</H4><P>Metabolic changes due to diseases can be monitored also on volatile organic compound (VOC) levels in patient’s breath. These VOCs have different profile for each disease. The motives of VOCs for each disease have been observed via different spectrometry methods, such as mass and gas spectrometry. The collected data have been narrowed to select the very specific biomarkers for this project. The aim is to produce bacteria that have circuits and logic gates at DNA level which is going to be able to either diagnose or detect the types of cancer by sensing specific VOCs in the exhaled breath of the patients mentioned before in literature.</P><H4>DiagNOSE</H4><P>For our sensors pMimA/MimR, pFor/Hxlr, pAlkM/AlkR, Pu/XylR and pChnB/ChnR promoter/transcription factor duplets have been selected, which recognize acetone, formaldehyde, octane, cyclohexanone and o-xylene presence, respectively. </P><P>
	Transcription factor MimR binds to pMimA in the presence of acetone and isopropanol, HxlR binds to pFor in the presence of formaldehyde and AlkR binds to pAlkM in the presence of octane which activates the transcription of the down stream genes. For cyclohexanone sensor, pChnB (promoter of ChnB gene) has been selected since it binds transcription factor ChnR in the presence of cyclohexanone and for o-xylene sensor Pu (promoter of XylR gene) has been selected since it binds to the transcription factor XylR in the presence of o-xylene, m-xylene, p-xylene and toluene; so that transcription on this promoter is activated.

</P><P>After achieving getting the signal for a particular VOC, we aim to construct logical gates and circuits by using CRISPRi system on plasmids to create well accoutred bacteria which will be able to differentiate these four cancer types.
CRISPRi technology is based on interference of sequence-specific repression of gene. dCas9 and sgRNA are two components of this technique. Integration of dCas9 and sgRNA binds to RNA polymerase binding site and blocks the transcription.
</P><P>First step of AND gate is based on inhibition of  llac promoter. pllac/ara1 is inhibited, unless arabinose is present in the environment. When arabinose is present, promoter  pllac/ara1  is cleared and transcription factor 1 [TF 1]  and transcription factor 2 [TF 2] can be expressed.
Secondly, if first volatile organic compound [VOC1] is present in the environment, TF 1 and VOC1 activates  target promoter [P1] to produce sgRNA. The produced TF 2 can activate the other target promoter [P2] if and only if second volatile organic compound  [VOC2]   is present in the environment. When present, they activate P2  and dCas9 production. 
CRISPRi technology is used in the final step. Since lacI is produced under the constitutively active promoter, promoter pllaco is always inhibited, unless IPTG is added into the environment. To inhibit the expression of lacI, produced sgRNA and dCas9 now come together and block  lacI gene expression  that activates the sequence of pllaco, thus enable sfGFP production, which is the output.
There are 3 different VOC types in lung, breast and colorectal cancer. Two of them are going to be chosen by comparing their efficiency. Best two VOCs are going to be integrated into AND gate, according to  VOC induxtion experiment results. Since prostate cancer is indicated by 2 VOC types, toluene and formaldehyde, transcription factors  which recognize these VOCs are going to be directly integrated AND gate.
</P></DIV><DIV id="page" class="container"><DIV id="protocols" class="title"><H2 id="const">Constructs</H2></DIV><H3>AlkR</H3><P>Acinetobacter sp. strain ADP1 is bacterium that can utilize alkane chains from dodecane to octadecane as a carbon source. This degradation depends at least in five genes. It has been found that AlkR, a AraC-XylS like transcription factor regulator regulates the transcription of AlkM, gene coding for alkane hydroxylase. AlkR, when coupled with an appropriate alkane, activates transcription from pAlkM. Experiments in Acinetobacter sp. strain ADP1 suggest AlkR is responsive to alkanes from heptane to octadecane. [1]</P><P>AlkR/pAlkM system has been shown to be capable of creating a biosensor in Acinetobacter sp. strain ADP1.[2] There are, however conflicting results for utilizing this system in E.Coli. Wu et. Al. reports that this system is unusable is E.Coli, even when AlkR is successfully produced. [3] Tianjin 2013 iGEM reports an opposite result. Their AlkR-PAlkM based alkane sensor assembled in pSB1C3 output a nearly 2-fold signal when induced with 10mM Octane. So we decided to try this system ourselves and if functional, characterize it. </P><H3>HxlR</H3><P>In previous researches, it is shown that the patients having prostate cancer have higher formaldehyde concentration in their urine samples compared to healthy people. [4] The designed construct aims to get a correlated GFP signal with formaldehyde concentration. 
</P><P>
In this construct, HxlR transcription factor from Bacillus subtilis which belongs to the DUF24 protein family and the promoter pFor are utilized. HxlAB operon is characterized from Bacillus subtilis and it regulates formaldehyde metabolism. In this construct, HxlR is the DNA binding protein which acts as a transcription factor and it regulates the formaldehyde operon.. Hence, it is known that in the presence of formaldehyde, the operon is active. Although the mechanism of the activation is not well known, it is suspected that formaldehyde leads conformational change in HxlR protein which promotes its activation. Thus, active HxlR protein can bind the upstream  of pFor promoter and activate the HxlAB operon. [5]
</P><P>
HxlR/pFor system is produced in order to create a formaldehyde biosensor in E.coli. This biosensor aims to detect elevated formaldehyde concentration in prostate cancer patients.
</P><H3>mimR</H3><P>Mycobacterium smegmatis strain MC2 155 and Mycobacterium goodii strain 12523 use propane and acetone as carbon sources. These molecules are used in the metabolism due to mimABCD gene clusters playing role in the oxidation of propane and acetone. [6]</P><P>  
Expression of mimABCD in response to acetone and relative compounds is regulated by mimR gene product whose gene location is upstream of the gene cluster and is divergently transcribed.[6] Furuya et al had tried to determine the specificity of transcriptional activation by mimR with exposure to certain molecules indicated in the figure 1.  As it has been seen from the figure, acetone is the compound increasing the transcriptional activation by mimR better than other molecules. Therefore we had decided to use mimR-pmimA transcription factor and promoter couples as sensor of acetone. 
</P><H3>ChnR</H3><P>Acinetobacter johnsonii  strain NCIMB 9871is bacterium that can convert cyclohexanol  and its derivatives(cyclohexanone and  ε-caprolactone) into adicipic acid. This degradation mainly depends on five genes. ChnA, alcohol dehydrogenase,  degrades cyclohexanol into cyclohexanone, which is degraded by ChnB into  ε-caprolactone. Presence of ChnC, hydrolase, contributes to hydrolization of ε-caprolactone, then ChnD and ChnE, alcohol dehydrogenase and aldehyde dehydrogenase respectively, take an active role in formation of adicipic acid at the end. It has been found that ChnR protein which is a Xyls/AraC-type transcriptional activator from Acinetobacter sp. responds to the presence of cyclohexanone by inducing expression of ChnB protein. pChnB  is the region starting from 537bp upstream of ChnB to its start codon from Acinetobacter sp. cyclohenol gene cluster. pChnB is activated by ChnR in the presence of cyclohexanone. [7]</P><P>

ChnR/pChnB system has been shown to be capable of creating a biosensor in Acinetobacter sp. strain NCIMB 9871.[8]  It is found that breast cancer patients release different amount of cyclohexanone to the environment, therefore we decided to use ChnR/pChnB system to develop a biosensor which is avble to diagnose breast cancer.</P><P>All constructs were put together by PCR using overhang primers followed by Gibson Assembly with respect to the Lab Notebook. Backbone used is pZa from pZ vector family. It has chloramphenicol resistance and a p15A origin which results in 30 to 50 copies in E.coli. Promoter used to produce all the transcription factors are pLTetO-1 and it is induced by anhydrous tetracycline (ATC). [9] Reporter of choice is superfolder GFP.[10] </P><H3>AlkR-pAlkB Based Mid-long Chain Alkane Sensor</H3><P>pZA-AlkR-pAlkM_sfGFP plasmid has pLTetO-1 promoter and BBa_B0034(RBS) with down stream AlkR coding sequence. sfGFP is under the control of AlkR regualted promoter pAlkB, coupled with BBa_B0030. Both coding sequences are shielded with T1 terminator.[11]</P><P>
Aside from this, a secondary plasmid aimed to produce HisTagged AlkR protein was also constructed. This plasmid was used to confirm expression of AlkR via Western Blot analysis.
</P><H3>HxlR-pFor Based Formaldehyde sensor</H3><P>pZA-HxlR-pFor_sfGFP plasmid has pLTetO-1 promoter and BBa_B0034(RBS) with down stream HxlR coding sequence. sfGFP is under the control of HxlR regualted promoter pFor, coupled with BBa_B0030. Both coding sequences are shielded with T1 terminator.[11]</P><P>
Aside from this, a secondary plasmid aimed to produce HisTagged HxlR protein was also constructed. This plasmid was used to confirm expression of HxlR via Western Blot analysis.
</P><H3>MimR-pMimA Based Isopropanol/Acetone Sensor</H3><P>pZA-mimR-pMimA_sfGFP plasmid has pLTetO-1 promoter and BBa_B0034(RBS) with down stream MimR  coding sequence. sfGFP is under the control of MimR regualted promoter pMimA, coupled with BBa_B0034. MimR coding sequence is shielded with His terminator[12] and sfGFP is shielded by T1 terminator.[11]</P><P>
Aside from this, a secondary plasmid aimed to produce HisTagged MimR protein was also constructed. This plasmid was used to confirm expression of MimR via Western Blot analysis.
</P><H3 id="notebooks">Notebooks</H3><UL class="actions notebooks"><LI class="noteb"><A class="buttonx" href="https://static.igem.org/mediawiki/2017/8/82/Bilkent-alkr.pdf" target="_blank">AlkR</A></LI><LI class="noteb"><A class="buttonx" href="https://static.igem.org/mediawiki/2017/b/bb/Bilkent-hxlr.docx.pdf" target="_blank">HxlR</A></LI><LI class="noteb"><A class="buttonx" href="https://static.igem.org/mediawiki/2017/2/21/Bilkent-mimr.pdf" target="_blank">mimR</A></LI><LI class="noteb"><A class="buttonx" href="https://static.igem.org/mediawiki/2017/5/56/Bilkent-chnr.pdf" target="_blank">ChnR</A></LI><LI class="noteb"><A class="buttonx" href="https://static.igem.org/mediawiki/2017/a/ad/Bilkent-lux.pdf" target="_blank">Lux</A></LI><LI class="noteb"><A class="buttonx" href="https://static.igem.org/mediawiki/2017/9/9a/Bilkent-chromo.pdf" target="_blank">Chromoproteins</A></LI></UL></DIV><DIV id="page" class="container"><DIV id="modeling" class="title"><H2>Modeling</H2></DIV><P>TinkerCell is one of the most popular CAD (Computer-Aided-Design) that is used to design and simulate biological systems.1 Here, we built our theoretical final construct and investigated the effect of promoters, substrate concentrations and other factors that can affect the output of our system. Due to time limitations, we were not able to optimize the parameters that are used in the simulation, but we will be working on it to make our sensor more efficient and convenient.</P></DIV><DIV id="page" class="container"><DIV id="protocols" class="title"><H2>Protocols</H2></DIV><UL><LI><EM>Preparation of chemically competent cells (E.coli)</EM></LI></UL><OL><LI>We grow a 5mL overnight culture of cells in LB medium in the shaker at 37C.</LI><LI>Next day we dilute the culture 1:100 and grow them until it reaches an OD600 of 0.2-0.5. </LI><LI>The eppendorfs and falcons as well as the TSS buffer are pre-cooled before use.</LI><LI>We split the culture into two 50mL falcons and incubate on ice for 10 minutes.</LI><LI>They are centrifuged 3000g for 10 minutes at 4C.</LI><LI>We remove the supernatant and resuspend the cells in the chilled TSS buffer 10% volume of the amount of cultured centrifuged.</LI><LI>100uL aliquots are taken.</LI><LI>These are stored in -80C. </LI></OL><UL><LI><EM>Transformation</EM></LI></UL><OL><LI>We take the competent cells out of -80C and incubate on ice for 15-20 minutes.</LI><LI>We add 100ng of DNA into the samples and nothing to the controls. </LI><LI>We incubate the cells on ice for another 30 minutes.</LI><LI>We heat-shock them at 42C for 30 seconds and incubate on ice for 2 more minutes.</LI><LI>We add 500mL LB to the cells and let them recover in the shaker at 37C for 45 minutes.</LI><LI>They are then centrifuged at 5800g for 6 minutes.</LI><LI>We pour off most of the supernatant and resuspend the pellet in the remainder. </LI><LI>We spread the remainder onto agar plates with the appropriate antibiotics and put them into the incubator at 37C overnight. </LI></OL><UL><LI><EM>MiniPrep of Plasmid DNA (MN Plasmid DNA Purification kit was used during these experiments) </EM></LI></UL><OL><LI>We grow bacteria overnight in 10mL LB medium with 1:1000 appropriate antibiotics in 50mL falcons.</LI><LI>Next day we centrifuge them at 8000g for 5 minutes and pour off the supernatant.</LI><LI>We add A1 buffer and resuspend the cell pellet by pipetting up and down then transfer the suspension to eppendorfs.</LI><LI>We add 250uL A2 buffer and mix by inverting the tubes a few times. Samples are incubated at room temperature for 5 minutes and no longer. Be careful of the time!</LI><LI>We add 300uL A3 buffer and mix by inverting the tubes a few times until the blue color of A2 disappears and the sample has a consistency similar to cottage cheese. </LI><LI>We centrifuge the cells at 13000rpm for 10 minutes. </LI><LI>We transfer 700uL from the supernatant to NucleoSpin Columns (as the recommended 750uL usually caused a little overspill when the cap was closed) and centrifuge at 13000rpm for 1 minute. The flow through is discarded.</LI><LI>If there is supernatant remaining, we repeat this step.</LI><LI>We wash the columns with 500uL AW buffer and centrifuge at 13000rpm for 1 minute. The flow through is discarded. </LI><LI>We put 600uL A4 buffer on the columns and centrifuge again at 13000rpm for 1 minute. The flow through is discarded.</LI><LI>We centrifuge them for an additional 2 minutes at 13000rpm to dry the silica membranes. </LI><LI>To elute the DNA, we add 20mL ddH2O -warmed to 65C beforehand- and centrifuge them at 13000rpm for 3 minutes after waiting for 3 minutes at room temperature. </LI><LI>We usually take a second dilution sample by adding another 20uL of ddH2O to the columns and once again centrifuging at 13000rpm for 3 minutes after a 3-minute wait. </LI><LI>The concentrations are measured after the process with NanoDrop with 2uL from the sample. </LI><LI>The samples are then stored in -20C. </LI></OL><UL><LI><EM>Restriction Enzyme Digestion and Ligation</EM></LI></UL><OL><LI>We follow the NEB protocols for our digestions. We put the recommended amount of restriction enzymes (0.4 uL each), CutSmart Buffer (2 uL) and approximately 1000ng of DNA to be cut (if the cut DNA isn’t going to be cloned and only to be used for visualization 300-400ng should be enough), and complete the reaction volume to 20uL with water. </LI><LI>We incubate the samples at 37C for a minimum of two hours.</LI><LI>The restriction results are checked by running the samples on agarose gel. If the samples are needed for a following step, we perform gel extraction. </LI></OL><UL><LI><EM>Gel Extraction (MN Plasmid NucleoSpin Gel and PCR Clean-up kit was used during these experiments)</EM></LI></UL><OL><LI>After cutting the needed bands out of the agarose gel we weigh them. </LI><LI>We add NTI buffer twice the amount of micrograms measured in uL and incubate the samples at 50C 450rpm for 5-10 minutes.</LI><LI>We take 700uL form those samples and add on top of NucleoSpin columns. They are centrifuged at 14000rpm for 1 minute then we discard the flow through. If there is any remaining solution from the samples we repeat this step. </LI><LI>We add 700uL NT3 (with EtOH) and centrifuge at 14000rpm for 1 minute. The flow through is discarded.</LI><LI>Step 4 is repeated. </LI><LI>We centrifuge the columns at 14000rpm again for 3 minutes to dry. </LI><LI>We put the column into new eppendorfs at 65C and incubate approximately 3 minutes. </LI><LI>We add 20mL ddH2O -warmed to 65C beforehand- to the columns and centrifuge them at 14000rpm for 3 minutes after waiting for 3 minutes at room temperature. </LI><LI>Step 8 is repeated.</LI><LI>The concentrations are measured after the process with NanoDrop with 2 uL from the sample. </LI></OL><UL><LI><EM>Induction </EM></LI></UL><OL><LI>We pick colonies from our plates and put them in 3mL (or 5 or 10mL depending on necessity) LB medium with 1:1000 appropriate antibiotics. </LI><LI>The diluted samples are incubated at 37C until OD600 value reaches approximately 0.6.</LI><LI>The samples are then induced with 0.02% arabinose (from %10 stock) and 1X 1M IPTG (from 1000X stock). Uninduced and competent cells are used as controls.</LI><LI>We place them back into the incubator either at 37C for 4hours, 30C for 6h or 18C overnight.         </LI><LI>OD600 values are measured again and 50/OD600  (equaling 5x107 cells) are taken from each sample.</LI><LI>If the cells are to be run on SDS gel, after centrifugation the pellets remaining are resuspended in 18uL 1X Sample Buffer and incubated at 95C for 5 minutes then loaded to the SDS gel. </LI></OL><UL><LI><EM>SDS Gel Electrophoresis </EM></LI></UL><OL><LI>Preparation of SDS gelFor a 5 ml stacking gel:
</LI><LI>H20 2.975 ml</LI><LI>0.5 M Tris-HCl, pH:6.8 1.25 ml</LI><LI>10%(w/v) SDS 0.05 ml</LI><LI>Acrylamide/Bis-acrylamide (30%/0.8%) 0.67 ml</LI><LI>10% (w/v) ammonium persulfate (APS) 0.05 ml</LI><LI>TEMED 0.005 ml </LI>

For a 10 ml separating gel:<TABLE><TBODY><TR><TD><P>Acrylamide percentage</P></TD><TD><P>6%</P></TD><TD><P>8%</P></TD><TD><P>10%</P></TD><TD><P>%12</P></TD><TD><P>%15</P></TD></TR><TR><TD><P>H2O</P></TD><TD><P>5.2 ml</P></TD><TD><P>4.6 ml</P></TD><TD><P>3.8 ml</P></TD><TD><P>3.2 ml</P></TD><TD><P>2.2 ml</P></TD></TR><TR><TD><P>1.5 M Tris-HCl, pH:8.8</P></TD><TD><P>2.6 ml</P></TD><TD><P>2.6 ml</P></TD><TD><P>2.6 ml</P></TD><TD><P>2.6 ml</P></TD><TD><P>2.6 ml</P></TD></TR><TR><TD><P>10%(w/v) SDS</P></TD><TD><P>0.1 ml</P></TD><TD><P>0.1 ml</P></TD><TD><P>0.1 ml</P></TD><TD><P>0.1 ml</P></TD><TD><P>0.1 ml</P></TD></TR><TR><TD><P>Acrylamide/Bis-acrylamide (30%/0.8% w/v)</P></TD><TD><P>2 ml</P></TD><TD><P>2.6 ml</P></TD><TD><P>3.4 ml</P></TD><TD><P>4 ml</P></TD><TD><P>5 ml</P></TD></TR><TR><TD><P>10% (w/v) ammonium persulfate (APS)</P></TD><TD><P>0.1 ml</P></TD><TD><P>0.1 ml</P></TD><TD><P>0.1 ml</P></TD><TD><P>0.1 ml</P></TD><TD><P>0.1 ml</P></TD></TR><TR><TD><P>TEMED</P></TD><TD><P>0.01 ml</P></TD><TD><P>0.01 ml</P></TD><TD><P>0.01 ml</P></TD><TD><P>0.01 ml</P></TD><TD><P>0.01 ml</P></TD></TR></TBODY></TABLE>
Note: APS and TEMED must be added right before each use.






 <LI>We load 20 ul of the samples mixed with loading dye  and run at 120 V.</LI><LI>After the run we place the gel in a container with Coomassie Blue dye for staining and microwave for 15-40seconds. The gel should be checked in between to be safe. </LI><LI>The container is put on shaker for 10 minutes the washed under water.</LI><LI>We then place the gel into another container with Destaining Buffer and it can be put on shaker overnight for the destaining. The bands are usually clearly visible the next morning.  </LI></OL><UL><LI><EM>Western Blot</EM></LI></UL><OL><LI>Before setting up the cassettes we place Whatman filter papers into Transfer Buffer, and the nitrocellulose membrane into methanol first for a few minutes then into Transfer Buffer as well. </LI><LI>One of the Whatman papers is placed in the cassette and flattened with the rolling apparatus gently so there are no bubbles trapped. </LI><LI>The membrane is placed on to the first paper and the gel is placed onto the membrane. The second layer of Whatman paper goes onto the gel. After placing each layer we roll over them with the apparatus again gently to smooth out the bubbles.</LI><LI>We place this cassette inside the tank and it is set to Turbo -&gt; Mini Gel -&gt; Run for a 7-minute transfer. </LI><LI>While the transfer is occurring we prepare the Blocking Buffer with 5% dry milk. This solution can be used up to a few times so we do not discard it.</LI><LI>When the transfer is over we take the membrane out with tweezers gently and the top corner is marked to know which side is up.</LI><LI>The membrane is placed inside the milk solution and incubated inside the shaker for 2 hours.</LI><LI>Then we place the membrane carefully into the 1st Antibody solution and keep it rocking on the shaker for another 2 hours.</LI><LI>The membrane is placed into another container after, and we put enough TBS-T to cover it, leaving it rocking for another 5 minutes. Afterwards the membrane is washed twice, first for 5 then for 10 minutes.</LI><LI>Then we expose the membrane to the 2nd Antibody.</LI><LI>The membrane is placed into another container after, and we put enough TBS-T to cover it, leaving it rocking for another 5 minutes. Afterwards the membrane is washed twice, first for 5 then for 10 minutes.</LI><LI>Then we take the image of the membrane.</LI></OL><UL><LI><EM>PCR</EM></LI></UL><OL><LI>We first prepare the Master Mix with (it's easier to add starting from the larger volumes):
</LI><LI>66 uL H2O</LI><LI>20 uL Q% Reaction Buffer</LI><LI>5 uL Forward primer</LI><LI>5 uL Reverse primer</LI><LI>2 uL dNTP mix</LI><LI>1 uL Q5 Polymerase (be careful to keep the enzymes on ice and work fast)</LI><LI>25 uL from the mix is put into PCR tubes and we only add the template (0.3 uL if plasmid 2uL if genomic DNA) to our samples, while the control has everything but the template. </LI><LI>The reaction is carried out by the PCR machine according to NEB Q5 DNA polymerase PCR reaction protocol.</LI><LI>If the product is going to go under restriction enzyme digestion, the procedure for it is detailed above. </LI></OL><UL><LI><EM>Gibson Reaction</EM></LI></UL><OL><LI>We add 50ng of our vector to 15 uL of Gibson Assembly Mix (already prepared in PCR tubes 7.5uL each stored at -20C). </LI><LI>We add DNA at 1:1 ratio but this can change depending on the length of the DNA and the length of the vector as well as number of different DNA pieces.</LI><LI>The reaction is carried out at 50C in the PCR machine in 1 hour. </LI><LI>The product can be directly taken and transformed into competent cells. </LI></OL><P><B>References</B></P><OL><LI>[1] Ratajczak, Andreas, Walter Geißdörfer, and Wolfgang Hillen. &quot;Expression of Alkane Hydroxylase fromAcinetobacter sp. Strain ADP1 Is Induced by a Broad Range of n-Alkanes and Requires the Transcriptional Activator AlkR.&quot; Journal of bacteriology 180.22 (1998): 5822-5827.</LI><LI>[2] Zhang, Dayi, et al. &quot;Whole‐cell bacterial bioreporter for actively searching and sensing of alkanes and oil spills.&quot; Microbial biotechnology 5.1 (2012): 87-97.</LI><LI>[3] Wu, Wei, et al. &quot;Genetically assembled fluorescent biosensor for in situ detection of bio-synthesized alkanes.&quot; Scientific reports 5 (2015).</LI><LI>[4] Peng, Gang, et al. &quot;Detection of lung, breast, colorectal, and prostate cancers from exhaled breath using a single array of nanosensors.&quot; British journal of cancer 103.4 (2010): 542-551.</LI><LI>[5] Yurimoto, H., Hirai, R., Matsuno, N., Yasueda, H., Kato, N., &amp; Sakai, Y. (2005, June 09). HxlR, a member of the DUF24 protein family, is a DNA‐binding protein that acts as a positive regulator of the formaldehyde‐inducible hxlAB operon in Bacillus subtilis.</LI><LI>[6] Furuya, Toshiki, et al. &quot;Identification of the regulator gene responsible for the acetone-responsive expression of the binuclear iron monooxygenase gene cluster in mycobacteria.&quot; Journal of bacteriology 193.20 (2011): 5817-58237</LI><LI>[7] Brautaset, Trygve, Rahmi Lale, and Svein Valla. &quot;Positively regulated bacterial expression systems.&quot; Microbial biotechnology 2.1 (2009): 15-30.</LI><LI>[8] Benedetti, Ilaria, Pablo I. Nikel, and Víctor de Lorenzo. &quot;Data on the standardization of a cyclohexanone-responsive expression system for Gram-negative bacteria.&quot; Data in brief 6 (2016): 738-744.</LI><LI>[9] https://www.ncbi.nlm.nih.gov/pmc/articles/PMC146584/pdf/251203.pdf</LI><LI>[10] https://www.ncbi.nlm.nih.gov/pubmed/16369541</LI><LI>[11] https://www.ncbi.nlm.nih.gov/pmc/articles/PMC209807/</LI><LI>[12] https://www.ncbi.nlm.nih.gov/pubmed/3007936</LI></OL></DIV><DIV id="copyright" class="container"><P>Design by <A href="http://templated.co" rel="nofollow">TEMPLATED</A>.</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>