<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UCSF_Project_Conjugation_Promoter"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UCSF/Project/Conjugation/Promoter">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UCSF/Project/Conjugation/Promoter&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UCSF/Project/Conjugation/Promoter&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UCSF/Project/Conjugation/Promoter&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UCSF/Project/Conjugation/Promoter" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UCSF/Project/Conjugation/Promoter</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="nav-wrap"><UL id="nav"><LI><A href="https://2013.igem.org/Team:UCSF" id="homelink">Home</A></LI><LI><A class="drop" id="projectlink">Project</A></LI><LI><A class="dropheader" href="https://2013.igem.org/Team:UCSF/Project/Background">Background</A></LI><LI><A class="drop" id="projectlink">CRISPRi Conjugation</A></LI><LI><A href="https://2013.igem.org/Team:UCSF/Project/Conjugation/Design">Project Design</A></LI><LI><A href="https://2013.igem.org/Team:UCSF/Project/Conjugation/Data">Data</A></LI></UL><LI><A class="drop" id="projectlink">CRISPRi Circuit</A></LI><LI><A href="https://2013.igem.org/Team:UCSF/Project/Circuit/Design">Circuit Design</A></LI><LI><A href="https://2013.igem.org/Team:UCSF/Project/Conjugation/Promoter">Promoter Engineering</A></LI><LI><A class="dropheader" href="https://2013.igem.org/Team:UCSF/Project/Circuit/Data">Data</A></LI><LI><A class="drop" id="projectlink">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:UCSF/Modeling2">Conjugation Modeling</A></LI><LI><A href="https://2013.igem.org/Team:UCSF/Modeling">Circuit Modeling</A></LI><LI><A class="drop" id="outreachlink">Human Practices</A></LI><LI><A href="https://2013.igem.org/Team:UCSF/Exploratorium">Exploratorium</A></LI><LI><A href="https://2013.igem.org/Team:UCSF/ALHS Project">Lincoln High</A></LI><LI><A href="https://2013.igem.org/Team:UCSF/Materials">Materials</A></LI><LI><A href="https://2013.igem.org/Team:UCSF/Collaborations">Collaborations</A></LI><LI><A class="drop" id="notebooklink">Protocols</A></LI><LI><A href="https://2013.igem.org/Team:UCSF/Protocols">General Protocols</A></LI><LI><A href="https://2013.igem.org/Team:UCSF/Protocols2">Project Protocols</A></LI><UL><LI><A href="https://2012.igem.org/Team:UCSF/Protocols">General Protocols</A></LI><LI><A href="https://2012.igem.org/Team:UCSF/Protocols2">Project Protocols</A></LI></UL><LI><A class="drop" id="outreachlink">Judging</A></LI><LI><A href="https://2013.igem.org/Team:UCSF/Project/Attribute">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:UCSF/Project/Accomplish">Accomplishments</A></LI><LI><A href="https://2013.igem.org/Team:UCSF/Project/Implications">Implications</A></LI><LI><A href="https://2013.igem.org/Team:UCSF/Parts">Parts</A></LI><LI><A href="https://2013.igem.org/Team:UCSF/Safety">Safety</A></LI><LI><A class="drop" id="team">Team</A></LI><LI><A class="dropheader" href="https://2013.igem.org/Team:UCSF/About Us">About Us</A></LI><LI><A href="https://2013.igem.org/Team:UCSF/Team">Members</A></LI><LI><A href="https://2013.igem.org/Team:UCSF/Advisors">Advisors</A></LI><LI><A class="dropheader" href="https://2013.igem.org/Team:UCSF/Mentors&amp;Instructors">Mentors</A></LI><LI><A class="dropheader" href="https://igem.org/Team.cgi?year=2013&amp;team_name=UCSF">Profile</A></LI><LI><A class="dropheader" href="https://2013.igem.org/Team:UCSF/ContactUs">Contact Us</A></LI><LI><A href="https://2013.igem.org" target="_blank">iGEM</A></LI></DIV><DIV id="description" style="width:950px; height:355px" align="justify"><FONT face="calibri" size="5"><CENTER><B>CRISPRi Circuit: Promoter Design and Engineering</B></CENTER></FONT><P><FONT face="calibri" size="4">
After our initial promoter assays our goal was to create engineered versions of our promoters responsive to high and low levels of inducer, and here we’ve chosen pLAC promoter as proof of concept. 

The lactose inducible promoter pLAC, first discovered in Escherichia coli and serving as a core component of lac operon responsible for lactose metabolism, is one of the best-studied and engineered prokaryotic transcriptional regulatory system. The switch like behavior of this promoter in response to lactose level is achieved through interaction of a regulatory protein LacI and small fragments of DNA sequence, named lac operators, within the promoter region. 

The LacI protein monomers self-associate into an unusual tetramer that appears roughly as a V-shaped dimer of dimers (See Figure 1A for detailed information), and could be further divided into four discrete functional units: a N-terminal headpiece with a helix-turn-helix motif capable of binding to the DNA, a hinge region connecting headpiece with core, and the protein core with a N-terminal lactose binding domain and a C-terminal helix responsible for dimerization. Each dimeric repressor is capable of binding to a 21-base-pair duplex deoxyoligonucleotides, the lac operator site. (Wilson et al., 2007; Lewis, 2011) Meanwhile, the lac operator sequence is a pseudo-symmetric DNA sequence that was first identified to be a 27-base-pair section (lacO1) (Gilbert et al., 1973) and further narrowed down to around 17 base pairs for minimal specific binding requirement (Bahl et al., 1977) (Figure 1B).  
 </FONT></P></DIV><DIV id="description" style="width:700px; height:100px; margin-left:140px" align="justify"><FONT face="calibri" size="2"> Figure 1. Scheme of lac repressor and operator site. (A) A ribbon diagram of the quaternary structure of the lac repressor complexed to DNA. Lac repressor is a tetrameric structure where each monomer is drawn in color (blue/grey for left dimer and green/pink for right dimer). (From Wilson et al., 2007) (B) The 27-base-pair deoxyoligonucleotide first identified as the operator region of the lactose operon (LacO1). Bold characters showed the 17 base pairs required for specific binding. 
</FONT></DIV><DIV id="description" style="width:950px; height:85px" align="justify"><FONT face="calibri" size="4">
In the absence of lactose, the repressor protein binds to lac operator with a high affinity, and therefore compromises RNA polymerase binding, elongation and/or initiation. While in the presence of lactose and a variety of other gratuitous inducers (e.g. isopropyl-β,D-thiogalactoside, IPTG), theses “inducers” binds with high affinity to lac repressors and lower its affinity for the operator sequence, thereby allows the transcription of downstream mRNA (Figure 2). </FONT></DIV><DIV id="description" style="width:700px; height:60px; margin-left:140px" align="justify"><FONT face="calibri" size="2"> Figure 2. Scheme of the lactose operon and its switch-like behavior. The lacZ, lacY, lacA genes are lactose metabolic genes that located downstream of the pLAC promoter. 
</FONT></DIV><DIV id="description" style="width:950px; height:185px" align="justify"><FONT face="calibri" size="4">
Apart from the original pLAC promoter derived from nature, various engineering effort have been made for the optimization of its expression and induction properties and therefore facilitates its usage in synthetic biological systems (i.e. pTAC (de Boer et al., 1983), pLlacO_1 (Lutz and Bujard, 1997), etc., See Figure 3A for detailed information). Meanwhile, in addition to the first identified lacO1 operator section, two auxiliary pseudo-operators, lacO2 (Reznicoff et al., 1974) and lacO3 (Pfahl et al., 1979) sites, were identified downstream (lacO2) or upstream (lacO3) of the promoter region with similar sequences (Figure 3B). These operators were later shown to be required for maximal repression, for the fact that tetrameric lac repressor was ideally suited to bind two operators simultaneously, and therefore caused so-called “repression loops” (Figure 4A) (Mossing et al., 1986; Oehler et al., 1990). Studies have also shown that this DNA looping would also change the kinetic behavior (i.e. activation coefficient) of the promoter (Figure 4B) (Oehler et al., 2006). </FONT></DIV><DIV id="description" style="width:700px; height:60px; margin-left:140px" align="justify"><FONT face="calibri" size="2"> Figure 3. (A) Sequence of different lactose inducible promoters. Yellow rectangle indicates -35 and -10 core elements of the promoter, red character indicates transcription start site, and lac operators are shown in blue boxes. (B) Sequences of three lac operators found in E. coli genome. Bases listed with capital letters are identical to bases in the primary LacO1. 
</FONT></DIV><DIV id="description" style="width:700px; height:70px; margin-left:140px" align="justify"><FONT face="calibri" size="2"> Figure 4. Impact of DNA looping on lac repressor binding to DNA. (A) Minimum energy configurations of DNA fragments complexed with the LacR tetramer (From Swigon et al., 2006). (B) DNA looping would change the kinetic behavior of pLAC promoter. Scheme of promoters (up) and induction results (down) of pLAC with (right) and without (left) DNA looping (From Oehler et al., 2006). 
</FONT></DIV><DIV id="description" style="width:950px; height:185px" align="justify"><FONT face="calibri" size="4">
Based on these previous studies and the modeling result that our desired “low” and “high” sensor would need two promoters that response to the same inducer with different kinetic behaviors (i.e. activation coefficient) and maximum expression level, we would like to build our “low” and “high” lactose sensor by altering the operator numbers and orientations of the pLAC promoter. 

We first constructed an engineered pLAC promoter (pLAC_engineered) by adding an additional lacO3 binding site to the upstream of the original pLAC promoter we previously characterized (pLAC_original), which was identical to the pLlacO_1 promoter discussed before (See Figure 5A for detailed information). However, the characterization of the two promoters showed only difference in maximal expression level but no kinetic behavior change (activation coefficient for pLAC_original was 1.672 and for pLAC_engineered was 1.685) (Figure 5B), probably due to the competition of binding between the three operator sites. 
 </FONT></DIV><DIV id="description" style="width:700px; height:170px; margin-left:140px" align="justify"><FONT face="calibri" size="2"> Figure 5. Schemes and characterization results of pLAC_original and pLAC_engineered. (A) Sequences of the pLAC_original (up) and pLAC_engineered (down). Yellow rectangle indicates -35 and -10 core elements of the promoter, red character indicates transcription start site, and lac operators are shown in blue boxes. (B) Dose-response curve for pLAC_original and pLAC_engineered. Promoter were cloned upstream of a GFP reporter gene, and induced with different amount of inducers (0, 0.1, 1, 5, 10, 30, 60, 100, 400 uM IPTG). Cells were grow to mid-log phase and then start induction. OD600 value and GFP fluorescence level of each sample were measured by plate reader after saturation. GFP fluorescence were corrected for OD600 value. Lines indicated Hill function fit of the dose-response curve and error bars indicated standard deviation calculated on the basis of technical replicates.
</FONT></DIV><DIV id="description" style="width:950px; height:105px" align="justify"><FONT face="calibri" size="4">
Then we sought to use the natural pLAC promoter with lacO3 site upstream (here we called it pLAC_high) instead, and engineered it by removing the lacO3 site to relieve repression (pLAC_low) (See Figure 6A for detailed information). The characterization results of the two promoters (Figure 6B) indicated that they in deed processed different kinetic behaviors (The activation coefficient for pLAC_high was 1.615 and for pLAC_low was 1.066), but the maximum expression level of pLAC_low was much too higher than that of pLAC_high, which was not ideal for our circuit performance. 
 </FONT></DIV><DIV id="description" style="width:700px; height:150px; margin-left:140px" align="justify"><FONT face="calibri" size="2"> Figure 6. Schemes and characterization results of pLAC_low and pLAC_high. (A) Sequences of the pLAC_low (up) and pLAC_high (down). Yellow rectangle indicates -35 and -10 core elements of the promoter, red character indicates transcription start site, and lac operators are shown in blue boxes. (B) Dose-response curve for pLAC_low and pLAC_high. Promoter were cloned upstream of a GFP reporter gene, and induced with different amount of inducers (0, 0.1, 1, 5, 10, 30, 60, 100, 400 uM IPTG). Cells were grow to mid-log phase and then start induction. OD600 value and GFP fluorescence level of each sample were measured by plate reader after saturation. GFP fluorescence were corrected for OD600 value. Lines indicated Hill function fit of the dose-response curve and error bars indicated standard deviation calculated on the basis of technical replicates.
</FONT></DIV><DIV id="description" style="width:950px; height:145px" align="justify"><FONT face="calibri" size="4">
Future engineering of the two promoters (pLAC_high and pLAC_low) could be made to optimize its performance. Previous works found on literatures have shown that by altering sequence of -10 and -35 core elements or the spacer region in between might result in modified prokaryotic promoter strength for over hundred folds (Jensen and Hammer, 1998; Alper et al., 2005), therefore similar approach might be taken into pLAC_low to reduce its maximum expression level or pLAC_high for the increase of the maximum expression level. In addition, alteration of the copy number of the vector bearing the promoters might also be useful for generating desired output for the synthetic circuit, as plasmid copy numbers contribute a lot for the protein expression level of synthetic circuits. 

 </FONT></DIV><DIV id="description" style="width:900px; height:370px; margin-left:0px" align="justify"><FONT face="calibri" size="2"><B>References</B>
Alper, H., Fischer, C., Nevoigt, E. and Stephanopoulos, G. (2005). Tuning genetic control through promoter engineering. Proc. Natl. Acad. Sci. USA 102, 12678-12683. 
Bahl, C.P., Wu, R., Stawinsky, J. and Narang, S.A. Studies on the lactose operon, Minimal length of the lactose operator sequence for the specific recognition by the lactose repressor. (1977). Proc. Natl Acad. Sci. USA 74, 966–970.
de Boer, H. A., Comstock, L. J., and Vasser, M. (1983). The Tac promoter: a functional hybrid derived from the Trp and Lac promoters. Proc Natl Acad Sci U S A. 80, 21-25.
Gilbert, W., and Maxam, A. (1973). The nucleotide sequence of the lac operator, Proc. Natl Acad. Sci. USA 70, 3581–3584.
Jensen, P. R. and Hammer, K. (1998). The Sequence of Spacers between the Consensus Sequences Modulates the Strength of Prokaryotic Promoters. Appl. Environ. Microbiol. 64, 82-87.
Lewis, M. (2011). A Tale of Two Repressors. J Mol Biol. 409, 14-27. 
Mossing M.C. and Record M. T. Jr. (1986). Upstream Operators Enhance Repression of the lac Promoter. Science 223, 889-892.
Oehler. S., Eismann, E. R., Krämer, H. and Müller-Hill, B. (1990). The three operators of the lac operon cooperate in repression. EMBO J. 9, 973-979.
Oehler, S., Alberti, S. and Müller-Hill, B. (2006). Induction of the lac promoter in the absence of DNA loops and the stoichiometry of induction. Nucleic Acids Res. 34, 606-612.
Pfahl, M., Gulde, V. and Bourgeois, S. (1979) ‘Second’ and ’third operator’ of the lac operon: an investigation of their role in the regulatory mechanism. J. Mol. Biol. 127, 339–344.
Reznikoff, W. S., Winter, R. B. and Hurley, C. K. (1974). The location of the repressor binding sites in the lac operon. Proc. Natl. Acad. Sci. USA 71, 2314–2318.
Swigon, D., Coleman, B. D. and Olson, W. K. (2006). Modeling the Lac repressor-operator assembly: The influence of DNA looping on Lac repressor conformation. Proc. Natl. Acad. Sci. USA 103, 9879-9884.
Wilson, C. J., Zhan , H., Swint-Kruse , L., and Matthews , K. S. (2007). The lactose repressor system: paradigms for regulation, allosteric behavior and protein folding. Cell Mol Life Sci. 64, 3-16.</FONT></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:UCSF/Project/Conjugation/Promoter">http://2013.igem.org/Team:UCSF/Project/Conjugation/Promoter</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UCSF/Project/Conjugation/Promoter" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UCSF/Project/Conjugation/Promoter" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UCSF/Project/Conjugation/Promoter&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UCSF/Project/Conjugation/Promoter&amp;oldid=350954" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>