<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_USP_UNIFESP-Brazil_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:USP UNIFESP-Brazil/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="row topo"><DIV class="small-10 columns"><DIV class="row"><DIV class="small-6 columns"><P>AlgAranha Team USP_UNIFESP-Brazil</P></DIV><DIV class="small-6 columns"><P style="text-align: right; padding-right: 10.5%;"><A href="https://2016.igem.org/">iGEM 2016</A></P></DIV></DIV><DIV class="row"><DIV class="small-12 columns"><UL class="menu"><LI><A href="https://2016.igem.org/wiki/index.php?title=Team:USP_UNIFESP-Brazil">Home</A></LI><LI><A href="https://2016.igem.org/Team:USP_UNIFESP-Brazil/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:USP_UNIFESP-Brazil/Project" class="active">Project</A></LI><LI><A href="https://2016.igem.org/Team:USP_UNIFESP-Brazil/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:USP_UNIFESP-Brazil/Interlab">Interlab</A></LI><LI><A href="https://2016.igem.org/Team:USP_UNIFESP-Brazil/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:USP_UNIFESP-Brazil/Awards">Awards</A></LI><LI><A href="https://2016.igem.org/Team:USP_UNIFESP-Brazil/Attributions">Attributions</A></LI></UL></DIV></DIV></DIV></DIV></DIV><UL id="menu-lateral" style="absolute; width:20%"><LI><A href="#gen">General sequences assembly</A></LI><LI><A href="#vect">Gathering of expression vector units</A></LI><LI><A href="#impro">Improvement of mCherry</A></LI><LI><A href="#ass">Assembly of spider silk gene sequence: Getting ready for expression!</A></LI><LI><A href="#enzy">Enzybiotic expression</A></LI></UL></DIV><DIV class="row meio"><DIV class="small-10 columns small-offset-2 titulo-verde"><DIV class="small-11 small-offset-1 columns"><H2> Experiments</H2></DIV></DIV><DIV class="row meio"><DIV class="small-10 columns small-offset-2 titulo-verde"><DIV class="small-11 small-offset-1 columns"><H2> General sequences assembly</H2></DIV></DIV><DIV class="small-10 columns small-offset-2"><DIV class="small-10 small-offset-1 columns"><P class="black"><SPAN style="font-weight: 400;">iGEM requests all teams submit their sequences using pSB1C3 as plasmid backbone. So, we have tried to bind our project sequences to it.</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Initially, all our sequences were designed taking into account the codon usage of </SPAN><EM><SPAN style="font-weight: 400;">C. reinhardtii</SPAN></EM><SPAN style="font-weight: 400;">. Afterwards, we submitted these to IDT offer of gene synthesis (Table 1).</SPAN></P><P class="black"><STRONG>Table 1. </STRONG><SPAN style="font-weight: 400;">Synthetic constructs designed by our team</SPAN></P><TABLE><TBODY><TR><TD><P class="black"><SPAN style="font-weight: 400;">On registry</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">Description</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">Length</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">Function</SPAN></P></TD></TR><TR><TD> </TD><TD><P class="black"><SPAN style="font-weight: 400;">LysK</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">1550</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">Enzybiotic</SPAN></P></TD></TR><TR><TD> </TD><TD><P class="black"><SPAN style="font-weight: 400;">MV-L</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">1508</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">Enzybiotic</SPAN></P></TD></TR><TR><TD><P class="black"><SPAN style="font-weight: 400;">BBa_K2136002</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">Lysostaphin</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">803</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">Enzybiotic</SPAN></P></TD></TR><TR><TD> </TD><TD><P class="black"><SPAN style="font-weight: 400;">b-galacto</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">1547</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">Gene reporter</SPAN></P></TD></TR><TR><TD> </TD><TD><P class="black"><SPAN style="font-weight: 400;">Lip_Thela</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">872</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">Lipase</SPAN></P></TD></TR><TR><TD> </TD><TD><P class="black"><SPAN style="font-weight: 400;">gLuc</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">569</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">Gene reporter</SPAN></P></TD></TR><TR><TD> </TD><TD><P class="black"><SPAN style="font-weight: 400;">Ea MaSp1</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">252</SPAN></P></TD><TD rowspan="4"><P class="black"><SPAN style="font-weight: 400;">Spider silk proteins</SPAN></P></TD></TR><TR><TD> </TD><TD><P class="black"><SPAN style="font-weight: 400;">Lh MaSp1 Type 2</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">267</SPAN></P></TD></TR><TR><TD> </TD><TD><P class="black"><SPAN style="font-weight: 400;">Lh MaSp1 Silwa 1</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">231</SPAN></P></TD></TR><TR><TD> </TD><TD><P class="black"><SPAN style="font-weight: 400;">Lh MaSp1 Silwa 2</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">444</SPAN></P></TD></TR><TR><TD> </TD><TD><P class="black"><SPAN style="font-weight: 400;">USER cassette</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">141</SPAN></P></TD><TD> </TD></TR><TR><TD><P class="black"><SPAN style="font-weight: 400;">BBa_K2136010</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">5' cassete for <I>Chlamydomonas</I> transgenic expression</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">1593</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">Upper part of  our synthetic cassette</SPAN></P></TD></TR><TR><TD> </TD><TD><P class="black"><SPAN style="font-weight: 400;">3' cassete for <I>Chlamydomonas</I> transgenic expression</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">1478</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">Lower part of our synthetic cassette</SPAN></P></TD></TR></TBODY></TABLE><P class="black"><SPAN style="font-weight: 400;">Resource setbacks in life force people to such an extreme to come up with clever solutions. In this regard, a bacterial extract bearing a X7 Pfu polymerase was purified to make affordable further molecular biology operations. USER cloning (see below) takes advantage of this polymerase also because of its ability on dealing with uracil residues.</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Since the IDT sequences arrival on early-July and for the following months, our team dedicated huge efforts on cloning, with tons of unsuccessful transformations, each part. </SPAN></P><P class="black"><SPAN style="font-weight: 400;">At the end of the project, we were able to clone into pSB1C3, both parts <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2136002"> BBa_K2136002 </A> and <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2136010"> BBa_K2136010 </A>.  </SPAN></P><P class="black"><SPAN style="font-weight: 400;">Last but not least, throughout the project, we defy the traditional electrophoresis running buffer, named TAE or TBE, against sodium borate buffer which demonstrated to heat less, and therefore, running on high-voltage conditions without melting the gel.</SPAN></P></DIV></DIV><DIV class="small-10 columns small-offset-2 titulo-verde"><DIV class="small-11 small-offset-1 columns"><H2> Gathering of expression vector units</H2></DIV></DIV><DIV class="small-10 columns small-offset-2"><DIV class="small-10 small-offset-1 columns"><P class="black"><SPAN style="font-weight: 400;">As we got fascinated by <I>Chlamydomonas reinhardtii</I> novelty and eye-catching features </SPAN><SPAN style="font-weight: 400;">, we cogitated that sending each unit of our expression vector system to Registry parts would expand current systems for protein expression, specially, the complex ones. In this way, we felt glad that we could share this with iGEMers, and even, any of each unit could be combined with other parts of the bank to enhance, test, prank in clever ways.. </SPAN></P><P class="black"><STRONG>Characterization of expression vector</STRONG></P><P class="black"><SPAN style="font-weight: 400;">In order to study if our construct worked as we expected, a fluorescent protein mCherry was inserted in it and some analysis about its expression were made. </SPAN></P><P class="black"><STRONG>Screening of mCherry expressing colonies</STRONG></P><P class="black"><SPAN style="font-weight: 400;">Since expression of proteins from nuclear transformation in </SPAN><EM><SPAN style="font-weight: 400;"><I>Chlamydomonas</I></SPAN></EM><SPAN style="font-weight: 400;"> may vary due to insertion location, we decided to screen the colonies upfront in a 96 well layout (Schematic in </SPAN><SPAN style="font-weight: 400;">Proof of concept page</SPAN><SPAN style="font-weight: 400;">). Basically, we picked the colonies from selection plates (TAP media supplemented with 5-10 μg/mL of Zeocin) and incubated it in 200 μL TAP media in an individual well. We performed this screening in two different set ups, due to availability of equipment. The basic setup schematic can be found in </SPAN><SPAN style="font-weight: 400;">Proof of concept page</SPAN><SPAN style="font-weight: 400;">.</SPAN></P><P class="black"><STRONG>SETUP 1  </STRONG></P><P class="black"><SPAN style="font-weight: 400;">Microplate Shaker model :Agitador de Micro Placas Analógico AM 2.4 AN - INBRAS, Jardinopolis, SP, Brazil</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Agitation: 800 RPM</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Temperature: ~25</SPAN><SPAN style="font-weight: 400;">o</SPAN><SPAN style="font-weight: 400;">C</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Light Intensity: 60 μE/cm</SPAN><SPAN style="font-weight: 400;">2</SPAN></P><P class="black"><SPAN style="font-weight: 400;">mCherry measurement: 1 every 12 hours</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Absorbance 750nm: 1 every 12 hours</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Chlorophyl: 1 every 12 hours</SPAN></P><P class="black"><STRONG>SETUP 2</STRONG><SPAN style="font-weight: 400;"> </SPAN></P><P class="black"><SPAN style="font-weight: 400;">Microplate Shaker model: VWR INCUBATOR SHAKER-508 - Radnor, Pennsylvania, US</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Agitation: 800 RPM</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Temperature: ~25</SPAN><SPAN style="font-weight: 400;">o</SPAN><SPAN style="font-weight: 400;">C</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Light Intensity: 60 μE/cm</SPAN><SPAN style="font-weight: 400;">2</SPAN></P><P class="black"><SPAN style="font-weight: 400;">mCherry measurement: 3 every 12 hours</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Absorbance 750nm: 4 every 12 hours</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Chlorophyl: 1 every 12 hours</SPAN></P><P class="black"><SPAN style="font-weight: 400;">We follow mCherry production measuring mCherry fluorescence, cell growth by optical density in 750 nm absorbance and chlorophyll content by its fluorescence every 12 hours in both screenings (See results in </SPAN><SPAN style="font-weight: 400;">Proof of Concept</SPAN><SPAN style="font-weight: 400;">). Data acquisition were different because we tried to have a reduced measurement time in the first screening (~aprox. 3 min per total measurement). Nevertheless, more reliable data were needed and more measurements per well per time was performed (~aprox. 3 min per total measuremnt).</SPAN></P><P class="black"><STRONG>Plate Reading SETUP</STRONG></P><P class="black"><STRONG>Fluorescence Measurement - mCherry</STRONG></P><P class="black"><SPAN style="font-weight: 400;">Mode: Fluorescence Top Reading</SPAN><SPAN style="font-weight: 400;">Excitation Wavelength: 575 nm</SPAN><SPAN style="font-weight: 400;">Emission Wavelength: 608 nm</SPAN><SPAN style="font-weight: 400;">Excitation Bandwidth: 9 nm</SPAN><SPAN style="font-weight: 400;">Emission Bandwidth: 20 nm</SPAN><SPAN style="font-weight: 400;">Gain: 200 Manual</SPAN><SPAN style="font-weight: 400;">Number of Flashes: 10</SPAN><SPAN style="font-weight: 400;">Integration Time: 20 µs</SPAN><SPAN style="font-weight: 400;">Lag Time: 0 µs</SPAN><SPAN style="font-weight: 400;">Settle Time: 0</SPAN><SPAN style="font-weight: 400;">ms</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Z-Position (Manual): 18141 µm</SPAN></P><P class="black"><STRONG>Optical Density - </STRONG><STRONG><EM><I>Chlamydomonas reinhardtii</I></EM></STRONG></P><P class="black"><SPAN style="font-weight: 400;">Mode: Absorbance</SPAN><SPAN style="font-weight: 400;">Multiple Reads per Well (Circle (filled)): 2 x 2</SPAN><SPAN style="font-weight: 400;">Multiple Reads per Well (Border): 750 µm</SPAN><SPAN style="font-weight: 400;">Wavelength: 750 nm</SPAN><SPAN style="font-weight: 400;">Bandwidth: 9 nm</SPAN><SPAN style="font-weight: 400;">Number of Flashes: 25</SPAN><SPAN style="font-weight: 400;">Settle Time: 0</SPAN></P><P class="black"><STRONG>Fluorescence Measurement - Chlorophyll</STRONG></P><P class="black"><SPAN style="font-weight: 400;">Mode: Fluorescence Top Reading</SPAN><SPAN style="font-weight: 400;">Excitation Wavelength: 440 nm</SPAN><SPAN style="font-weight: 400;">Emission Wavelength: 680 nm</SPAN><SPAN style="font-weight: 400;">Excitation Bandwidth: 9 nm</SPAN><SPAN style="font-weight: 400;">Emission Bandwidth: 20 nm</SPAN><SPAN style="font-weight: 400;">Gain: 100 nm</SPAN><SPAN style="font-weight: 400;">Number of Flashes: 10</SPAN><SPAN style="font-weight: 400;">Integration Time: 20 µs</SPAN><SPAN style="font-weight: 400;">Lag Time: 0 µs</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Settle Time: 0 ms</SPAN><SPAN style="font-weight: 400;">Z-Position (Manual): 18141 µm</SPAN></P><P class="black"><STRONG>“Homemade” mCherry detection</STRONG></P><P class="black"><SPAN style="font-weight: 400;">In order to verify mCherry secretion we developed a method to observe its fluorescence in the supernatant (Our target). We had a laser pointer (532nm +- 10nm)  lay around and some filters we bought to construct a transilluminator for gels with GelRed™. We set up a quick experiment with the supernatant of the Wild type <I>Chlamydomonas</I>(cc1690) and the supernatant of our mCherry producing strain. We basically centrifuged 50 mL of culture media with our cells at 2500 x g for 15 min, and fixed the tubes in a support with the red filter in front of part of it. A green laser and a cellphone camera (With fixed iso and exposure) was all we needed to get the awesome photos better described with schematic in the </SPAN><SPAN style="font-weight: 400;">Proof of Concept page.</SPAN><SPAN style="font-weight: 400;"> </SPAN></P><P class="black"><STRONG>Confocal Fluorescence Microscopy</STRONG></P><P class="black"><SPAN style="font-weight: 400;">In order to check if our strain was indeed secreting mCherry we performed confocal fluorescence microscopy to observe mCherry in the secretion vacuoles. This experiment was conducted in a Confocal Zeiss LSM 780-NLO, using the laser 543 nm to excite mCherry and spectral detectors set approximately to 610-650 nm range. For chlorophyll excitation, we used a laser at 405 nm and spectral detector set at 680 nm region. All pictures were taken with the same system configuration.    </SPAN><SPAN style="font-weight: 400;"> </SPAN></P></DIV></DIV><DIV class="small-10 columns small-offset-2 titulo-verde"><DIV class="small-11 small-offset-1 columns"><H2> Improvement of mCherry </H2></DIV></DIV><DIV class="small-10 columns small-offset-2"><DIV class="small-10 small-offset-1 columns"><P class="black"><STRONG>Codon optimization</STRONG></P><P class="black"><SPAN style="font-weight: 400;">In order to express mCherry in algae, we used a codon optimized mCherry. Its comparison with the available sequence at the registry (BBa_J06504), in regard to CAI (Codon Adaptation Index) is summarized below at Table 1.</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Table1: CAI comparison of BBa_J06504 with optimized mCherry to <I>Chlamydomonas reinhardtii</I>.</SPAN></P><TABLE><TBODY><TR><TD><P class="black"><SPAN style="font-weight: 400;">Parameters</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">Available mCherry in Registry BBa_J06504</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">Codon Optimize mCherry  to </SPAN><EM><SPAN style="font-weight: 400;"><I>Chlamydomonas reinhardtii</I></SPAN></EM></P></TD></TR><TR><TD><P class="black"><SPAN style="font-weight: 400;">G+C content (%) - %GC:</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">62.2 </SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">63.9</SPAN></P></TD></TR><TR><TD><P class="black"><SPAN style="font-weight: 400;">%GC1s:</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">59.2</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">61.0</SPAN></P></TD></TR><TR><TD><P class="black"><SPAN style="font-weight: 400;">%GC2s</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">35.9</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">35.9</SPAN></P></TD></TR><TR><TD><P class="black"><SPAN style="font-weight: 400;">%GC3s</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">96.9</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">99.6</SPAN></P></TD></TR><TR><TD><P class="black"><SPAN style="font-weight: 400;">Kolmogorov-Smirnov test for the expected CAI (alpha = 0.05):</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">0.054</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">0.035</SPAN></P></TD></TR><TR><TD><P class="black"><SPAN style="font-weight: 400;">Chi-Square Goodness-of-Fit test for AA (alpha = 0.05)</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">the 100.0% of sequences fit the AA distribution</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">the 100.0% of sequences fit the AA distribution</SPAN></P></TD></TR><TR><TD><P class="black"><SPAN style="font-weight: 400;">Chi-Square Goodness-of-Fit test for G+C (alpha = 0.05): </SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">the 100.0% of sequences fit the G+C distribution</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">the 100.0% of sequences fit the G+C distribution</SPAN></P></TD></TR><TR><TD><P class="black"><SPAN style="font-weight: 400;">Average CAI</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">0.761</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">0.807</SPAN></P></TD></TR><TR><TD><P class="black"><SPAN style="font-weight: 400;">eCAI (p&lt;0.05)</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">0.850</SPAN></P></TD><TD><P class="black"><SPAN style="font-weight: 400;">0.885</SPAN></P></TD></TR></TBODY></TABLE><P class="black"><SPAN style="font-weight: 400;">Ref1: Puigbo P, Bravo IG and Garcia-Vallve S. (2008) E-CAI: a novel server to estimate an expected value of Codon Adaptation Index (eCAI). BMC Bioinformatics, 9:65.</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Ref2: Codon usage table from </SPAN><A href="http://www.kazusa.or.jp/codon/"><SPAN style="font-weight: 400;">Kazusa</SPAN></A><SPAN style="font-weight: 400;">.</SPAN></P><P class="black"><STRONG>FPLC - Fast protein liquid chromatography of mCherry in </STRONG><STRONG><EM><I>Chlamydomonas</I></EM></STRONG><STRONG> supernatant</STRONG></P><P class="black"><SPAN style="font-weight: 400;">Ion exchange purification exploit the net electrostatic charges of proteins, in pH values different from their pI (Isoelectric point). We developed a purification protocol for mCherry, using an anionic resin. First, we performed a gradient purification protocol to establish the best salt concentration to elute mCherry. </SPAN></P><P class="black"><STRONG>Gradient Set_UP:</STRONG></P><P class="black"><SPAN style="font-weight: 400;">Column: Resource Q (6 mL) - GE Healthcare </SPAN></P><P class="black"><SPAN style="font-weight: 400;">Buffer A: Sodium Phosphate 50 mM, pH7.5</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Buffer B: Sodium Phosphate 50 mM, pH7.5 + 1M NaCl</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Equilibration: 2 column volume (CV)</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Injection: 0.5mL 40X Concentrate supernatant sample</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Gradient length: 20 CV</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Flow rate: 5mL/min</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Fractionation: 5mL to unbound and 3 mL to the rest of the method</SPAN></P><P class="black"><SPAN style="font-weight: 400;">We obtained the following result (Figure 1).</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Figure 1: Chromatogram of gradient mCherry purification. Green line (</SPAN><STRONG>-</STRONG><SPAN style="font-weight: 400;">) is the UV sensor reading. Red line (</SPAN><STRONG>-</STRONG><SPAN style="font-weight: 400;">) is buffer B percentage in the mixture. Black line (</SPAN><STRONG>-</STRONG><SPAN style="font-weight: 400;">) is the conductivity measurement. Blue line (</SPAN><STRONG>-</STRONG><SPAN style="font-weight: 400;">) is the fluorescence measurement of fractionated samples. </SPAN></P><P class="black"><SPAN style="font-weight: 400;">We were surprised with the result. Most of <I>Chlamydomonas</I> natural proteins got bound to the resin, and got eluted with an increased salt concentration. This method was an efficient way to purify mCherry from the supernatant. UV absorbance curve integration allow us to estimate the amount of protein separated from mCherry. 99% of all protein detected by the sensor was  from mCherry fractions.</SPAN></P><P class="black"><SPAN style="font-weight: 400;">To further develop our method and reduce processing time, we developed a step based purification method (Figure 2). We kept 0% of B after injection for 3 CV, increase it a little bit to 0.7% of B to try to remove mCherry in this step, followed by a 100% of B step. This strategy was performed in a slower flow rate (3mL/min), and allow us to separate mCherry from 2 peaks in the beginning of the method. mCherry still left in the 0% step, but this method proved to be efficient, 99,7% of detected proteins were separated from mCherry. </SPAN></P><P class="black"><STRONG>Step based purification Set_UP:</STRONG></P><P class="black"><SPAN style="font-weight: 400;">Column: Resource Q (6 mL) - GE Healthcare </SPAN></P><P class="black"><SPAN style="font-weight: 400;">Buffer A: Sodium Phosphate 50 mM, pH7.5</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Buffer B: Sodium Phosphate 50 mM, pH7.5 + 1M NaCl</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Equilibration: 2 column volume (CV)</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Injection: 0.5mL 40X Concentrate supernatant sample</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Step1: 3 CV</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Step2: 2 CV</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Step3: 5 CV</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Flow rate: 3mL/min</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Fractionation: 5mL to unbound and 1 mL to Step 1, 3 mL to Step 2 and 5 mL to step 3.</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Figure 2: Chromatogram of step based mCherry purification. Green line (</SPAN><STRONG>-</STRONG><SPAN style="font-weight: 400;">) is the UV sensor reading. Red line (</SPAN><STRONG>-</STRONG><SPAN style="font-weight: 400;">) is buffer B percentage in the mixture. Black line (</SPAN><STRONG>-</STRONG><SPAN style="font-weight: 400;">) is the conductivity measurement. Blue line (</SPAN><STRONG>-</STRONG><SPAN style="font-weight: 400;">) is the fluorescence measurement of fractionated samples.</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Fluorescence measurements in all fractions were obtained in a plate reader (TECAN Infinite® 200 PRO), following the fluorescence measurement method described above.</SPAN></P><P class="black"><STRONG>mCherry Fluorescence Spectrum (Ex/Em)</STRONG></P><P class="black"><SPAN style="font-weight: 400;">The samples purified by FPLC were used to further characterize our mCherry produced by </SPAN><EM><SPAN style="font-weight: 400;"><I>Chlamydomonas reinhardtii</I></SPAN></EM><SPAN style="font-weight: 400;">. We used aliquots from fractions 5 and 6 (Step purification) to construct and further characterize codon optimized mCherry (</SPAN><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2136016"><SPAN style="font-weight: 400;">BBa_K2136016</SPAN></A><SPAN style="font-weight: 400;">). We set up a plate reader (TECAN Infinite® 200 PRO) to obtain fluorescence spectrum of our mCherry.</SPAN></P><P class="black"><STRONG>Experiment SETUP:</STRONG></P><P class="black"><SPAN style="font-weight: 400;">Mode: Fluorescence Top Reading</SPAN><SPAN style="font-weight: 400;">Excitation Wavelength Start: 300 nm</SPAN><SPAN style="font-weight: 400;">Excitation Wavelength End: 600 nm</SPAN><SPAN style="font-weight: 400;">Excitation Wavelength Step Size: 1nm</SPAN><SPAN style="font-weight: 400;">Excitation Scan Number: 301</SPAN><SPAN style="font-weight: 400;">Emission Wavelength: 640 nm</SPAN><SPAN style="font-weight: 400;">Bandwidth (Em): 280...850: 20 nm</SPAN><SPAN style="font-weight: 400;">Bandwidth (Ex) (Range 1): 230...315: 5 nm</SPAN><SPAN style="font-weight: 400;">Bandwidth (Ex) (Range 2): 316...850: 10 nm</SPAN><SPAN style="font-weight: 400;">Gain: 200 Manual</SPAN><SPAN style="font-weight: 400;">Number of Flashes: 10</SPAN><SPAN style="font-weight: 400;">Integration Time: 20 µs</SPAN><SPAN style="font-weight: 400;">Lag Time: 0 µs</SPAN><SPAN style="font-weight: 400;">Settle Time:0 ms</SPAN><SPAN style="font-weight: 400;">Z-Position (Manual): 18141 µm</SPAN></P><P class="black"><SPAN style="font-weight: 400;">We were happy to see that our Excitation/Emission spectrum obtained was similar to the ones available to mCherry. We extended the available spectrum from 300 nm to 850 nm, allowing us to observe a small excitable region in 360 nm to mCherry. This experiment allowed us, simultaneously, further characterize this important biobrick (</SPAN><A href="http://parts.igem.org/Part:BBa_J06504"><SPAN style="font-weight: 400;">BBa_J06504</SPAN></A><SPAN style="font-weight: 400;">), by adding an extended fluorescence spectrum and demonstrate the presence of mCherry in the supernatant of our algae strain (See Proof of Concept page for mCherry Fluorescence spectrum). </SPAN></P><P class="black"><SPAN style="font-weight: 400;">Figure 3: Excitation/Emission spectrum of mCherry produced and purified from the supernatant of a transformed <I>Chlamydomonas</I> culture. </SPAN></P></DIV></DIV><DIV class="small-10 columns small-offset-2 titulo-verde"><DIV class="small-11 small-offset-1 columns"><H2> Assembly of spider silk gene sequence: Getting ready for expression!</H2></DIV></DIV><DIV class="small-10 columns small-offset-2"><DIV class="small-10 small-offset-1 columns"><H4 class="black"><B>Assembly by USER</B></H4><P class="black"><SPAN style="font-weight: 400;">The size of silk genes is one of the most important problems to be solved in a project about silk production. Chemical synthesis presented some limitations to this sequence due to their high CG content and their repetitives sequences. Many cloning approaches has been used to polymerase them. We decided to explore a new technique to achieve this.</SPAN></P><P class="black"><SPAN style="font-weight: 400;">USER is a seamless assembly method to recombinant molecules from multiples components. USER reaction performs a excision of a Uracil in a nucleotide sequence, and this is exploited to generate complementary overhangs between sequences. Primers are designed to contain a Uracil at nucleotide 8-10 of its sequence. In the reaction a complementary overhang of 8-10 is created and allowed to align. The reaction mixture containing linearized plasmid with a USER adaptor sequence, and desired sequences are transformed into </SPAN><EM><SPAN style="font-weight: 400;">E.coli</SPAN></EM><SPAN style="font-weight: 400;"> where the plasmid and sequence are completely ligated. We designed 4 primers to this experiment. </SPAN></P><H4 class="black"><B>Assembly by cut&amp;paste</B></H4><P class="black">As we had trouble with our USER-specific primers request and could not start with the USER method, we started an alternative way to polymerize the MaSp1 monomers. Based on the Nature protocol of Teulé <I>et al</I> &quot;A protocol for the production of recombinant spider silk-like proteins for artificial fiber spinning&quot; we designed a cut and paste technique, to duplicate the MaSp1 sequence at each round.</P><P class="black">Telué and collaborators use a cloning vector with a restriction site in its resistance gene and two complementary but non regenerable restriction sites, one at each side of the insert, which is the sequence that is going to be duplicated. The vector is cut in two parallel separated digestions, at one side of the insert and at the resistance gene, and on the other side of the insert and also at the resistance gene. In both digestions, two fragments will be generated: one that contains the insert and a part of the resistance gene, and one that does not contain the insert. Ligating the insert-containing fragments of the two different digestions together results in the duplication of the insert in a vector with a complete resistance gene. </P><P class="black">In our case we used the same vector, that would be transformed into our production organism, <I>Chlamydomonas reinhardtii</I>. As you can see in figure 4, the pJP22 has a ScaI restriction site in the ampicillin resistance gene and two compatible but non regenerable restriction sites at the MCS: BglII and BamHI, so we went on and cloned the spider silk monomer MaSp1 Type2. Then, following the scheme of Teulé and collaborators, we cut in parallel digestions the pJP22-MaSp1t2 with ScaI and either BamHI or BglII. The insert-containing fragments were then ligated together and transformed into E. coli DH5α cells  </P><P class="black"><SPAN style="font-weight: 400;">Figure 4: pJP22 containing MaSp1. </SPAN></P></DIV></DIV><DIV class="row meio"><DIV class="small-10 columns small-offset-2 titulo-verde"><DIV class="small-11 small-offset-1 columns"><H2> Enzybiotic expression</H2></DIV></DIV><DIV class="small-10 columns small-offset-2"><DIV class="small-10 small-offset-1 columns"><P class="black"><SPAN style="font-weight: 400;">To analyse the activity of an </SPAN><EM><SPAN style="font-weight: 400;">enzybiotic</SPAN></EM><SPAN style="font-weight: 400;"> that could be applied on silk functionalization in future works, we tried to express Lysostaphin. The lysostaphin sequence that was synthesized by IDT was successfully cloned into pSB1C3 and from there, we tried to clone it into pETDuet to be expressed in BL21 DE3. Unfortunately we did not get positive clones so far.</SPAN></P><P class="black"><SPAN style="font-weight: 400;">Figure 5: Expression vector pETDuet containing the enzybiotic lysostaphin. </SPAN></P><P class="black"> </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>