<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_USP_UNIFESP-Brazil_Proof skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:USP UNIFESP-Brazil/Proof</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="row topo"><DIV class="small-10 columns"><DIV class="row"><DIV class="small-6 columns"><P>AlgAranha Team USP_UNIFESP-Brazil</P></DIV><DIV class="small-6 columns"><P style="text-align: right; padding-right: 10.5%;"><A href="https://2016.igem.org/">iGEM 2016</A></P></DIV></DIV><DIV class="row"><DIV class="small-12 columns"><UL class="menu"><LI><A href="https://2016.igem.org/wiki/index.php?title=Team:USP_UNIFESP-Brazil">Home</A></LI><LI><A href="https://2016.igem.org/Team:USP_UNIFESP-Brazil/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:USP_UNIFESP-Brazil/Project" class="active">Project</A></LI><LI><A href="https://2016.igem.org/Team:USP_UNIFESP-Brazil/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:USP_UNIFESP-Brazil/Interlab">Interlab</A></LI><LI><A href="https://2016.igem.org/Team:USP_UNIFESP-Brazil/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:USP_UNIFESP-Brazil/Awards">Awards</A></LI><LI><A href="https://2016.igem.org/Team:USP_UNIFESP-Brazil/Attributions">Attributions</A></LI></UL></DIV></DIV></DIV></DIV><DIV class="row meio"><DIV class="small-10 columns small-offset-2 titulo-verde"><DIV class="small-11 small-offset-1 columns"><H2>Proof of concept</H2></DIV></DIV><DIV class="small-12 columns small-offset-2"><DIV class="small-10 small-offset-0 columns"><P class="black">The core iGEM idea was not only to produce spider silk proteins and antibiotics. But doing all this in microalgae! Thus, we also aimed to expand the tools and improve <I>Chlamydomonas reinhardtii</I> as a chassis for synthetic biology, characterizing a system for expression and secretion of recombinant proteins!
We were able to genetically transform through electroporation and successfully express a reporter protein, mCherry, in <I> Chlamydomonas</I> and detect its fluorescence. Our composite part, <A href="http://parts.igem.org/Part:BBa_K2136010">BBa_K2136010</A> will be of great value for any other teams working with microalgae in iGEM’s future!
</P><P class="black">
Even though this organism offers exciting possibilities, as well as many other photosynthetic chassis, a limited number of parts is available for synthetic biology work in the registry. Our motivation was to change this lack of parts and contribute to their improvement for the synbio community!
</P><P class="black">
One of our goals within iGEM was to develop a “Chlamydomonas molecular toolkit”, with basic parts for everyone’s use. We had a construct, <A href="http://parts.igem.org/Part:BBa_K2136010">BBa_K2136010</A>, synthesized by IDT gBlocks technology which included:
</P><UL><LI class="black"> An enhancer/promoter construct for transcription, <A href="http://parts.igem.org/Part:BBa_K2136013">BBa_K2136013</A></LI><LI class="black"> A resistance gene, selection marker, against the antibiotic zeocin, <A href="http://parts.igem.org/Part:BBa_K2136014">BBa_K2136014</A></LI><LI class="black"> A self cleaving peptide from foot-and-mouth disease virus, <A href="http://parts.igem.org/Part:BBa_K2136017">BBa_K2136017</A></LI><LI class="black"> A signal peptide which enable secretion of the protein of interest, <A href="http://parts.igem.org/Part:BBa_K2136018">BBa_K2136018</A></LI></UL><P class="black">
This construct was already assembled through an scarless method with mCherry codon optimized coding sequence  <A href="http://parts.igem.org/Part:BBa_K2136016">BBa_K2136016</A> and a <I>Chlamydomonas</I> terminator,  <A href="http://parts.igem.org/Part:BBa_K2136015">BBa_K2136015</A></P>. 
<P class="black">The clones were then selected in Petri dishes containing zeocyn and screened in 96 well plates for mCherry fluorescence (excitation at 575nm, emission at 608nm) in a setup like the one schematized in Figure 1</P><P class="fig-label">Figure 1 - Setup of transformant <I>Chlamydomonas</I> screening</P><P class="black">Even though the vast majority of the cells were productive, there was a variation in great part, due to the random assembly in <I>C. reinhardtii</I>’s genome. This, hopefully, will be surpassed with the development of new expression methods in the future (perhaps even  by iGEM teams!). Here we show the expression profile during growth of the best 5 clones in our first (Figure 2) and second (Figure 3) screening:

</P><DIV class="row"><DIV class="small-6 columns"><P class="fig-label">Figure 2 - mCherry emission/excitation in different cells in our first screen</P></DIV><DIV class="small-6 columns"><P class="fig-label">Figure 3 - mCherry emission/excitation in different cells in our second screen</P></DIV></DIV><P class="black">In order to visually know if our cassette was expressing mCherry, we used a fluorescence microscope (Figures 4 and 5). Chloroplasts occupies about 70% of the cell and it forms a “U” shape with the nucleus in the middle. It is possible to observe on Figure 4D that mCherry is accumulated in the form of vacuoles, and it is located at the region surrounding nucleus, which suggests that this protein is starting its secretion process.</P><P class="black">Figure 4: Fluorescence microscopy of C. reinhardtii. A - Measuring mCherry fluorescence. B- Measuring chlorophyll fluorescence. C - Open field image. D - Superposition of A,B and C. </P><P class="black">Figure 5: 3D Fluorescence microscopy of C. reinhardtii. Chlorophyll is fluorescing in green and mCherry is fluorescing in red.</P><P class="black"> Finally, we checked the ability of our system to secrete proteins through the signal peptide (BBa_K2136018)  using a home made filter, accordingly to our Human Practices principles. We developed a system using a 532nm green laser pointer and a red filter to probe directly our centrifuged cells. This system is schematized in Figure 6.</P><P class="fig-label">Figure 6 - Laser and filter setup </P><P class="black"> In order to check the reliability of our systems, we compared the  emission spectra for the 532nm green laser, the absorption spectra of the Rosco E-Colour #125: Deep golden Amber filter and the excitation/emission spectra for the mCherry protein. The spectra are in Figure 7</P><P class="fig-label">Figure 7 - Laser, filter and mCherry spectra superposition </P><P class="black"> Thus, the expected results were the following: </P><P class="fig-label">Figure 8 - Theoretical results of &quot;homemade&quot; fluorimeter </P><P class="black"> And we achieved it in practice! Corroborating, one more time, our proof of concept, implementing an efficient protein expression and secretion system for <I>Chlamydomonas</I> for the <STRONG>first time in iGEM!!</STRONG></P><P class="fig-label">Figure 9 - Practical results of &quot;homemade&quot; fluorimeter </P><P class="black">The excitation/emission fluorescence spectrum of the mCherry supernatant (without cell lysis, indicating secretion) was also characterized, not only confirming the detection, but improving characterization of this previous protein (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_J06505">BBa_J06505</A>) in the context of iGEM (<A href="http://parts.igem.org/Part:BBa_K2136016">BBa_K2136016</A>).
</P><P class="fig-label">Figure 10 - Codon optimized mCherry spectrum of excitation and emission</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>