Team:USP UNIFESP-Brazil/Experiments
AlgAranha Team USP_UNIFESP-Brazil
iGEM 2016
Home
Team
Project
Parts
Interlab
Human Practices
Awards
Attributions
General sequences assembly
Gathering of expression vector units
Improvement of mCherry
Assembly of spider silk gene sequence: Getting ready for expression!
Enzybiotic expression
Experiments
General sequences assembly
iGEM requests all teams submit their sequences using pSB1C3 as plasmid backbone. So, we have tried to bind our project sequences to it.
Initially, all our sequences were designed taking into account the codon usage of C. reinhardtii. Afterwards, we submitted these to IDT offer of gene synthesis (Table 1).
Table 1. Synthetic constructs designed by our team
On registry
Description
Length
Function
 
LysK
1550
Enzybiotic
 
MV-L
1508
Enzybiotic
BBa_K2136002
Lysostaphin
803
Enzybiotic
 
b-galacto
1547
Gene reporter
 
Lip_Thela
872
Lipase
 
gLuc
569
Gene reporter
 
Ea MaSp1
252
Spider silk proteins
 
Lh MaSp1 Type 2
267
 
Lh MaSp1 Silwa 1
231
 
Lh MaSp1 Silwa 2
444
 
USER cassette
141
 
BBa_K2136010
5' cassete for Chlamydomonas transgenic expression
1593
Upper part of  our synthetic cassette
 
3' cassete for Chlamydomonas transgenic expression
1478
Lower part of our synthetic cassette
Resource setbacks in life force people to such an extreme to come up with clever solutions. In this regard, a bacterial extract bearing a X7 Pfu polymerase was purified to make affordable further molecular biology operations. USER cloning (see below) takes advantage of this polymerase also because of its ability on dealing with uracil residues.
Since the IDT sequences arrival on early-July and for the following months, our team dedicated huge efforts on cloning, with tons of unsuccessful transformations, each part.
At the end of the project, we were able to clone into pSB1C3, both parts BBa_K2136002 and BBa_K2136010 .
Last but not least, throughout the project, we defy the traditional electrophoresis running buffer, named TAE or TBE, against sodium borate buffer which demonstrated to heat less, and therefore, running on high-voltage conditions without melting the gel.
Gathering of expression vector units
As we got fascinated by Chlamydomonas reinhardtii novelty and eye-catching features , we cogitated that sending each unit of our expression vector system to Registry parts would expand current systems for protein expression, specially, the complex ones. In this way, we felt glad that we could share this with iGEMers, and even, any of each unit could be combined with other parts of the bank to enhance, test, prank in clever ways..
Characterization of expression vector
In order to study if our construct worked as we expected, a fluorescent protein mCherry was inserted in it and some analysis about its expression were made.
Screening of mCherry expressing colonies
Since expression of proteins from nuclear transformation in Chlamydomonas may vary due to insertion location, we decided to screen the colonies upfront in a 96 well layout (Schematic in Proof of concept page). Basically, we picked the colonies from selection plates (TAP media supplemented with 5-10 μg/mL of Zeocin) and incubated it in 200 μL TAP media in an individual well. We performed this screening in two different set ups, due to availability of equipment. The basic setup schematic can be found in Proof of concept page.
SETUP 1  
Microplate Shaker model :Agitador de Micro Placas Analógico AM 2.4 AN - INBRAS, Jardinopolis, SP, Brazil
Agitation: 800 RPM
Temperature: ~25oC
Light Intensity: 60 μE/cm2
mCherry measurement: 1 every 12 hours
Absorbance 750nm: 1 every 12 hours
Chlorophyl: 1 every 12 hours
SETUP 2  
Microplate Shaker model: VWR INCUBATOR SHAKER-508 - Radnor, Pennsylvania, US
Agitation: 800 RPM
Temperature: ~25oC
Light Intensity: 60 μE/cm2
mCherry measurement: 3 every 12 hours
Absorbance 750nm: 4 every 12 hours
Chlorophyl: 1 every 12 hours
We follow mCherry production measuring mCherry fluorescence, cell growth by optical density in 750 nm absorbance and chlorophyll content by its fluorescence every 12 hours in both screenings (See results in Proof of Concept). Data acquisition were different because we tried to have a reduced measurement time in the first screening (~aprox. 3 min per total measurement). Nevertheless, more reliable data were needed and more measurements per well per time was performed (~aprox. 3 min per total measuremnt).
Plate Reading SETUP
Fluorescence Measurement - mCherry
Mode: Fluorescence Top Reading
Excitation Wavelength: 575 nm
Emission Wavelength: 608 nm
Excitation Bandwidth: 9 nm
Emission Bandwidth: 20 nm
Gain: 200 Manual
Number of Flashes: 10
Integration Time: 20 µs
Lag Time: 0 µs
Settle Time: 0 ms
Z-Position (Manual): 18141 µm
Optical Density - Chlamydomonas reinhardtii
Mode: Absorbance
Multiple Reads per Well (Circle (filled)): 2 x 2
Multiple Reads per Well (Border): 750 µm
Wavelength: 750 nm
Bandwidth: 9 nm
Number of Flashes: 25
Settle Time: 0
Fluorescence Measurement - Chlorophyll
Mode: Fluorescence Top Reading
Excitation Wavelength: 440 nm
Emission Wavelength: 680 nm
Excitation Bandwidth: 9 nm
Emission Bandwidth: 20 nm
Gain: 100 nm
Number of Flashes: 10
Integration Time: 20 µs
Lag Time: 0 µs
Settle Time: 0 ms
Z-Position (Manual): 18141 µm
“Homemade” mCherry detection
In order to verify mCherry secretion we developed a method to observe its fluorescence in the supernatant (Our target). We had a laser pointer (532nm +- 10nm)  lay around and some filters we bought to construct a transilluminator for gels with GelRed™. We set up a quick experiment with the supernatant of the Wild type Chlamydomonas(cc1690) and the supernatant of our mCherry producing strain. We basically centrifuged 50 mL of culture media with our cells at 2500 x g for 15 min, and fixed the tubes in a support with the red filter in front of part of it. A green laser and a cellphone camera (With fixed iso and exposure) was all we needed to get the awesome photos better described with schematic in the Proof of Concept page.  
Confocal Fluorescence Microscopy
In order to check if our strain was indeed secreting mCherry we performed confocal fluorescence microscopy to observe mCherry in the secretion vacuoles. This experiment was conducted in a Confocal Zeiss LSM 780-NLO, using the laser 543 nm to excite mCherry and spectral detectors set approximately to 610-650 nm range. For chlorophyll excitation, we used a laser at 405 nm and spectral detector set at 680 nm region. All pictures were taken with the same system configuration.  
Improvement of mCherry
Codon optimization
In order to express mCherry in algae, we used a codon optimized mCherry. Its comparison with the available sequence at the registry (BBa_J06504), in regard to CAI (Codon Adaptation Index) is summarized below at Table 1.
Table1: CAI comparison of BBa_J06504 with optimized mCherry to Chlamydomonas reinhardtii.
Parameters
Available mCherry in Registry BBa_J06504
Codon Optimize mCherry  to Chlamydomonas reinhardtii
G+C content (%) - %GC:
62.2
63.9
%GC1s:
59.2
61.0
%GC2s
35.9
35.9
%GC3s
96.9
99.6
Kolmogorov-Smirnov test for the expected CAI (alpha = 0.05):
0.054
0.035
Chi-Square Goodness-of-Fit test for AA (alpha = 0.05)
the 100.0% of sequences fit the AA distribution
the 100.0% of sequences fit the AA distribution
Chi-Square Goodness-of-Fit test for G+C (alpha = 0.05):
the 100.0% of sequences fit the G+C distribution
the 100.0% of sequences fit the G+C distribution
Average CAI
0.761
0.807
eCAI (p<0.05)
0.850
0.885
Ref1: Puigbo P, Bravo IG and Garcia-Vallve S. (2008) E-CAI: a novel server to estimate an expected value of Codon Adaptation Index (eCAI). BMC Bioinformatics, 9:65.
Ref2: Codon usage table from Kazusa .
FPLC - Fast protein liquid chromatography of mCherry in Chlamydomonas supernatant
Ion exchange purification exploit the net electrostatic charges of proteins, in pH values different from their pI (Isoelectric point). We developed a purification protocol for mCherry, using an anionic resin. First, we performed a gradient purification protocol to establish the best salt concentration to elute mCherry.
Gradient Set_UP:
Column: Resource Q (6 mL) - GE Healthcare
Buffer A: Sodium Phosphate 50 mM, pH7.5
Buffer B: Sodium Phosphate 50 mM, pH7.5 + 1M NaCl
Equilibration: 2 column volume (CV)
Injection: 0.5mL 40X Concentrate supernatant sample
Gradient length: 20 CV
Flow rate: 5mL/min
Fractionation: 5mL to unbound and 3 mL to the rest of the method
We obtained the following result (Figure 1).
Figure 1: Chromatogram of gradient mCherry purification. Green line (-) is the UV sensor reading. Red line (-) is buffer B percentage in the mixture. Black line (-) is the conductivity measurement. Blue line (-) is the fluorescence measurement of fractionated samples.
We were surprised with the result. Most of Chlamydomonas natural proteins got bound to the resin, and got eluted with an increased salt concentration. This method was an efficient way to purify mCherry from the supernatant. UV absorbance curve integration allow us to estimate the amount of protein separated from mCherry. 99% of all protein detected by the sensor was  from mCherry fractions.
To further develop our method and reduce processing time, we developed a step based purification method (Figure 2). We kept 0% of B after injection for 3 CV, increase it a little bit to 0.7% of B to try to remove mCherry in this step, followed by a 100% of B step. This strategy was performed in a slower flow rate (3mL/min), and allow us to separate mCherry from 2 peaks in the beginning of the method. mCherry still left in the 0% step, but this method proved to be efficient, 99,7% of detected proteins were separated from mCherry.
Step based purification Set_UP:
Column: Resource Q (6 mL) - GE Healthcare
Buffer A: Sodium Phosphate 50 mM, pH7.5
Buffer B: Sodium Phosphate 50 mM, pH7.5 + 1M NaCl
Equilibration: 2 column volume (CV)
Injection: 0.5mL 40X Concentrate supernatant sample
Step1: 3 CV
Step2: 2 CV
Step3: 5 CV
Flow rate: 3mL/min
Fractionation: 5mL to unbound and 1 mL to Step 1, 3 mL to Step 2 and 5 mL to step 3.
Figure 2: Chromatogram of step based mCherry purification. Green line (-) is the UV sensor reading. Red line (-) is buffer B percentage in the mixture. Black line (-) is the conductivity measurement. Blue line (-) is the fluorescence measurement of fractionated samples.
Fluorescence measurements in all fractions were obtained in a plate reader (TECAN Infinite® 200 PRO), following the fluorescence measurement method described above.
mCherry Fluorescence Spectrum (Ex/Em)
The samples purified by FPLC were used to further characterize our mCherry produced by Chlamydomonas reinhardtii. We used aliquots from fractions 5 and 6 (Step purification) to construct and further characterize codon optimized mCherry ( BBa_K2136016 ). We set up a plate reader (TECAN Infinite® 200 PRO) to obtain fluorescence spectrum of our mCherry.
Experiment SETUP:
Mode: Fluorescence Top Reading
Excitation Wavelength Start: 300 nm
Excitation Wavelength End: 600 nm
Excitation Wavelength Step Size: 1nm
Excitation Scan Number: 301
Emission Wavelength: 640 nm
Bandwidth (Em): 280...850: 20 nm
Bandwidth (Ex) (Range 1): 230...315: 5 nm
Bandwidth (Ex) (Range 2): 316...850: 10 nm
Gain: 200 Manual
Number of Flashes: 10
Integration Time: 20 µs
Lag Time: 0 µs
Settle Time:0 ms
Z-Position (Manual): 18141 µm
We were happy to see that our Excitation/Emission spectrum obtained was similar to the ones available to mCherry. We extended the available spectrum from 300 nm to 850 nm, allowing us to observe a small excitable region in 360 nm to mCherry. This experiment allowed us, simultaneously, further characterize this important biobrick ( BBa_J06504 ), by adding an extended fluorescence spectrum and demonstrate the presence of mCherry in the supernatant of our algae strain (See Proof of Concept page for mCherry Fluorescence spectrum).
Figure 3: Excitation/Emission spectrum of mCherry produced and purified from the supernatant of a transformed Chlamydomonas culture.
Assembly of spider silk gene sequence: Getting ready for expression!
Assembly by USER
The size of silk genes is one of the most important problems to be solved in a project about silk production. Chemical synthesis presented some limitations to this sequence due to their high CG content and their repetitives sequences. Many cloning approaches has been used to polymerase them. We decided to explore a new technique to achieve this.
USER is a seamless assembly method to recombinant molecules from multiples components. USER reaction performs a excision of a Uracil in a nucleotide sequence, and this is exploited to generate complementary overhangs between sequences. Primers are designed to contain a Uracil at nucleotide 8-10 of its sequence. In the reaction a complementary overhang of 8-10 is created and allowed to align. The reaction mixture containing linearized plasmid with a USER adaptor sequence, and desired sequences are transformed into E.coli where the plasmid and sequence are completely ligated. We designed 4 primers to this experiment.
Assembly by cut&paste
As we had trouble with our USER-specific primers request and could not start with the USER method, we started an alternative way to polymerize the MaSp1 monomers. Based on the Nature protocol of Teulé et al "A protocol for the production of recombinant spider silk-like proteins for artificial fiber spinning" we designed a cut and paste technique, to duplicate the MaSp1 sequence at each round.
Telué and collaborators use a cloning vector with a restriction site in its resistance gene and two complementary but non regenerable restriction sites, one at each side of the insert, which is the sequence that is going to be duplicated. The vector is cut in two parallel separated digestions, at one side of the insert and at the resistance gene, and on the other side of the insert and also at the resistance gene. In both digestions, two fragments will be generated: one that contains the insert and a part of the resistance gene, and one that does not contain the insert. Ligating the insert-containing fragments of the two different digestions together results in the duplication of the insert in a vector with a complete resistance gene.
In our case we used the same vector, that would be transformed into our production organism, Chlamydomonas reinhardtii. As you can see in figure 4, the pJP22 has a ScaI restriction site in the ampicillin resistance gene and two compatible but non regenerable restriction sites at the MCS: BglII and BamHI, so we went on and cloned the spider silk monomer MaSp1 Type2. Then, following the scheme of Teulé and collaborators, we cut in parallel digestions the pJP22-MaSp1t2 with ScaI and either BamHI or BglII. The insert-containing fragments were then ligated together and transformed into E. coli DH5α cells
Figure 4: pJP22 containing MaSp1.
Enzybiotic expression
To analyse the activity of an enzybiotic that could be applied on silk functionalization in future works, we tried to express Lysostaphin. The lysostaphin sequence that was synthesized by IDT was successfully cloned into pSB1C3 and from there, we tried to clone it into pETDuet to be expressed in BL21 DE3. Unfortunately we did not get positive clones so far.
Figure 5: Expression vector pETDuet containing the enzybiotic lysostaphin.
 
