<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_USP_UNIFESP-Brazil_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:USP UNIFESP-Brazil/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="row topo"><DIV class="small-10 columns"><DIV class="row"><DIV class="small-6 columns"><P>AlgAranha Team USP_UNIFESP-Brazil</P></DIV><DIV class="small-6 columns"><P style="text-align: right; padding-right: 10.5%;"><A href="https://2016.igem.org/">iGEM 2016</A></P></DIV></DIV><DIV class="row"><DIV class="small-12 columns"><UL class="menu"><LI><A href="https://2016.igem.org/wiki/index.php?title=Team:USP_UNIFESP-Brazil">Home</A></LI><LI><A href="https://2016.igem.org/Team:USP_UNIFESP-Brazil/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:USP_UNIFESP-Brazil/Project" class="active">Project</A></LI><LI><A href="https://2016.igem.org/Team:USP_UNIFESP-Brazil/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:USP_UNIFESP-Brazil/Interlab">Interlab</A></LI><LI><A href="https://2016.igem.org/Team:USP_UNIFESP-Brazil/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:USP_UNIFESP-Brazil/Awards">Awards</A></LI><LI><A href="https://2016.igem.org/Team:USP_UNIFESP-Brazil/Attributions">Attributions</A></LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="row meio"><DIV class="small-10 columns small-offset-2 titulo-verde"><DIV class="small-11 small-offset-1 columns"><H2> Description</H2></DIV></DIV><DIV class="small-10 columns small-offset-2"><DIV class="small-10 small-offset-1 columns"><P class="black">Spider silk is an amazing bionanomaterial due to its exceptional features. It exhibits high tensile strength, high fracture resistance, high elasticity and it has biocompatibility and low degradability. 
Some studies show silk can be compared to a synthetic polymer called Kevlar, that is made of aramid and is used to make body armor (Lewis). Silks are polymers composed of proteins. These proteins and their encoding genes present specific repetitive sequences that are associated with the already mentioned silk properties. Moreover, some studies suggest that the terminal domains of silk proteins play important roles in the polymerization process. In this way, it can be possible to add N- and C- terminal domains in several interesting proteins or motifs and spun it with native proteins to obtain a customized silk.</P><P class="black">The size of genes, their repetitive sequences and high content of Cytosine and Guanine establish a challenge to many expression systems. However, <I>Chlamydomonas reinhardtii</I> has naturally high GC content in its genome to deal with this difficulty, it can perform post translational modifications, and it can also secrete complex proteins, making the purification process easier. Furthermore, it exhibits low cost production, rapid growth, scalability and stable transgenic line generation, all desired characteristics to an expression system (Wijffels, Kruse, and Hellingwerf; Rosenberg et al.). It is important to highlight the low cost of microalgae cultivation, when compared to antibodies production in mammalian cells, which presents an average cost about US$150.00/per gram of raw materials and in plants US$0.05/per gram (Dove). Maylfield has estimated that microalgae can reach US$ 0.002/liter (Rosenberg et al.; Mayfield, Franklin, and Lerner). <I>Chlamydomonas reinhardtii</I> also is Generally Recognized As Safe (GRAS), that is, it has low risk of being contaminated by virus, prions and bacterial endotoxins (Mayfield et al.). The purification ease and low risk of contamination are significant issues when making products for biomedical applications.</P><P class="fig-label">Figure 1. Cassette construction to be inserted in <I>C. renhardtii</I> nuclear genome. Promoter hsp70A/rbcs2: fusion of the promoters hsp70A and rbcs2 (Eichler-Stahlberg et al.; Schroda, Blöcker, and Beck). Sh-Ble: resistance gene to Zeomycin. 2A: self-cleavage peptide from Foot and Mouth Disease Virus (FMDV) (Rasala et al.). SP: secretion signal peptide of the gene Ars1. GOI: gene of interest. His: histidine tag. RbcS2 3’ UTR: terminal sequence of the gene RbcS2 (Fuhrmann et al.). Introns were added as the figure shows (Eichler-Stahlberg et al.; Lumbreras et al.).</P><P class="black">Our project was inspired by the UCLA’s projects presented at iGEM in 2014 and 2015 about synthetic silks. As already mentioned above, silks are a very interesting material but one of the main problems is dealing with its high content of Cytosine and Guanine. Most of the expression systems are not able to deal with that, however we have known <I>Chlamydomonas reinhardtii</I> has naturally high GC content which makes it interesting as an expression chassis. One of UCLA’s projects was called Silk Functionalization: Developing the Next Generation of Performance Fibers. The team has created a genetic construct using the Green Fluorescent Protein (GFP) inserted between N- and C- terminal domains of Bombyx mori’s silk to express proteins able to bind to native silk proteins. When this customized GFPs and native proteins were mixed they were able to recognize each other by the terminal domains, resulting in a fluorescent silk.</P><P class="black">From this idea our team realized: “Silk is a very interesting material and it could be expressed in <I>Chlamydomonas reinhardtii</I>, a model organism that has many advantages and that remains under explored. Which gene could replace GFP to be expressed in microalgae? Which biomolecule could be produced with this amazing silk feature? It had to work while immobilized... Why not enzymes? There are many studies about enzymes immobilization, enzymes are very appropriate and useful in many applications around the world!”. But we knew it could not be any enzyme linked to the silk, we knew it needed to address technical, social and economic issues. So we read that spider silk has been studied for many biomedical applications, like burn victims treatments. Then we studied about burns in order to find an interesting enzyme that could solve some problem and we learned that one of the most important problems affecting burn victims is sepsis. In this way, we sought for antimicrobial enzymes and we found the endolysins. Endolysins are very interesting enzymes because they are able to combat multi-resistant bacteria and they are specific, avoiding problems like the antibiotic resistance.</P><P class="black">For these reasons our project goal was the spider silk production in microalgae, polymerize silk proteins and endolysins in order to make an antimicrobial silk that would be applied on burn victims to fight sepsis.</P></DIV></DIV><DIV class="small-10 columns small-offset-2 titulo-verde"><DIV class="small-11 small-offset-1 columns"><H2>Improvement of a previous part</H2></DIV></DIV><DIV class="small-10 columns small-offset-2"><DIV class="small-10 small-offset-1 columns"><P class="black"> The challenge when working with a new chassis is the development of new tools and parts. In order to access the protein expression efficiency, reporter genes are the state-of-the-art. With that in mind, we decided to improve mCherry characterization (<A href="http://parts.igem.org/Part:BBa_J06504">BBa_J06504</A>), a robust fluorescent protein, perfect for accessing the expression systems efficiency (such as ours!) as well as characterizing promoters, terminators and signal peptides. We further characterize this reporter by expressing and codon optimizing it in a new Chassis <I>Chlamydomonas reinihardtii</I>. The full data about our improvements in characterization and function can be found <A href="http://parts.igem.org/Part:BBa_K2136016"> here</A>.</P><P align="justify" class="black">The first improvement in characterization we made was the mCherry analysis and purification by Fast Protein Liquid Chromatography (FPLC) . FPLC is an Ion exchange purification that exploit the protein net electrostatic charges, in pH values different from their pI (Isoelectric point). We developed a purification protocol for mCherry. First, we performed a gradient purification to establish the best salt concentration to elute this fluorescent protein. </P><P><B>Gradient Set Up:</B></P>
Column: Resource Q (6 mL) - GE Healthcare 
Buffer A: Sodium Phosphate 50 mM, pH7.5
Buffer B: Sodium Phosphate 50 mM, pH7.5 + 1M NaCl
Equilibration: 2 column volume (CV)
Injection: 0.5mL 40X Concentrate supernatant sample
Gradient length: 20 CV
Flow rate: 5mL/min
Fractionation: 5mL to unbound and 3 mL to the rest of the method

<PCLASS>We obtained the following result (Figure 7).<P class="fig-label">Figure 2. Chromatogram of gradient mCherry purification. Green line (-) is the UV sensor reading. Red line (-) is buffer B percentage in the mixture. Black line (-) is the conductivity measurement. Blue line (-) is the fluorescence measurement of fractionated samples.</P><P align="justify" class="black">UV absorbance curve integration allow us to estimate the amount of protein separated from mCherry, 99% of all protein detected by the sensor was separated from mCherry fractions.</P><P align="justify" class="black">The samples purified from this method were used to further characterize our mCherry produced by <I>Chlamydomonas reinhardtii</I>. The Excitation/Emission spectrum (Figure 9) obtained are similar to the ones available to mCherry.</P><P class="fig-label">Figure 3.  Spectra of excitation and emission for <I> C. reinhardtii</I> expressed, codon optimized mCherry </P></PCLASS></DIV></DIV><DIV class="small-10 columns small-offset-2 titulo-verde"><DIV class="small-11 small-offset-1 columns"><H2>References</H2></DIV></DIV><DIV class="small-10 columns small-offset-2"><DIV class="small-10 small-offset-1 columns"><P class="black">Eichler-Stahlberg, Alke et al. “Strategies to Facilitate Transgene Expression in <I>Chlamydomonas reinhardtii</I>.” Planta 229.4 (2009): 873–883. Print.</P><P class="black">Fuhrmann, Markus et al. “A Synthetic Gene Coding for the Green Fluorescent Protein (GFP) Is a Versatile Reporter in <I>Chlamydomonas reinhardtii</I>.” The Plant journal: for cell and molecular biology 19.3 (1999): 353–361. Print.</P><P class="black">Lewis, Randolph V. “Spider Silk: Ancient Ideas for New Biomaterials.” Chemical reviews 106.9 (2006): 3762–3774. Print.</P><P class="black">Lumbreras, Victoria et al. “Efficient Foreign Gene Expression in <I>Chlamydomonas reinhardtii</I> Mediated by an Endogenous Intron.” The Plant journal: for cell and molecular biology 14.4 (1998): 441–447. Print.</P><P class="black">Mayfield, S. P., S. E. Franklin, and R. A. Lerner. “Expression and Assembly of a Fully Active Antibody in Algae.” Proceedings of the National Academy of Sciences 100.2 (2003): 438–442. Print.</P><P class="black">Mayfield, Stephen P. et al. “<I>Chlamydomonas reinhardtii</I> Chloroplasts as Protein Factories.” Current opinion in biotechnology 18.2 (2007): 126–133. Print.</P><P class="black">Rasala, Beth A. et al. “Robust Expression and Secretion of Xylanase1 in <I>Chlamydomonas reinhardtii</I> by Fusion to a Selection Gene and Processing with the FMDV 2A Peptide.” PloS one 7.8 (2012): e43349. Print.</P><P class="black">Rosenberg, Julian N. et al. “A Green Light for Engineered Algae: Redirecting Metabolism to Fuel a Biotechnology Revolution.” Current opinion in biotechnology 19.5 (2008): 430–436. Print.</P><P class="black">Schroda, M., D. Blöcker, and C. F. Beck. “The HSP70A Promoter as a Tool for the Improved Expression of Transgenes in <I>Chlamydomonas</I> .” The Plant journal: for cell and molecular biology 21.2 (2000): 121–131. Print.</P><P class="black">Wijffels, René H., Olaf Kruse, and Klaas J. Hellingwerf. “Potential of Industrial Biotechnology with Cyanobacteria and Eukaryotic Microalgae.” Current opinion in biotechnology 24.3 (2013): 405–413. Print.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>