<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tufts_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tufts/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer"><UL><LI><A href="https://2016.igem.org/Team:Tufts">HOME</A></LI><LI><A href="#">TEAM
            					</A></LI><LI><A href="https://2016.igem.org/Team:Tufts/Team">Team Bios</A></LI></UL><LI><A href="#">PROJECT
            					</A></LI><LI><A href="https://2016.igem.org/Team:Tufts/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Tufts/Experiments">Experiments &amp; Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Tufts/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Tufts/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Tufts/Parts">PARTS</A></LI><LI><A href="https://2016.igem.org/Team:Tufts/Notebook">NOTEBOOK</A></LI><LI><A href="https://2016.igem.org/Team:Tufts/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2016.igem.org/Team:Tufts/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2016.igem.org/Team:Tufts/Practices">HUMAN PRACTICES
          
					</A></LI><LI><A href="https://2016.igem.org/Team:Tufts/Safety">SAFETY</A></LI><LI><A href="https://2016.igem.org/Team:Tufts/Sponsors"> SPONSORS </A></LI></DIV><DIV id="contentContainer"><DIV class="column full_size"><H2> Results </H2><P>In order to have the ability to test the aTcdB-Cas9 delivery system, we needed to develop a pipeline for the production of sgRNAs as needed, beginning with transcription and assaying of multiple sgRNAs in vitro. Initially, we used the pSB1C3 plasmid containing a T7 promoter and sgRNA scaffold sequence , separated by a series of base pairs containing BbsI recognition sites. We first chose two different sequences in the pUC19 plasmid to serve as the spacer sequence to be inserted before the sgRNA scaffold sequence. Phosphorylated spacer sequences were generated from annealed oligonucleotides, which were ligated with the BbsI-digested plasmid. In vitro transcription with T7 RNA Polymerase was used to produce the sgRNAs, and these were used with Cas9 and both linearized and circular pUC19 in cleavage assays, confirming that both sgRNAs were functional.
</P><FIGURE><CENTER><FIGCAPTION> Figure 1: Example of in vitro Cas9 cleavage assay using linear pUC19, with the first lane on the left containing ladder. The third and fourth lanes both contained template, Cas9, and the sgRNA being tested in this set of experiments, but used different buffers to compare their performance (Cas9 buffer in the third lane and NEB 3.1 buffer in the fourth lane). </FIGCAPTION></CENTER></FIGURE><P>After the confirmation that our workflow could produce viable sgRNAs, we aimed to choose a human gene with clinical relevance as a target for the goal of in vivo tests with the aTcdB-Cas9 system. The PDZK1IP1 (PDZK1 Interacting Protein 1, also designated DD96) gene was chosen due to its upregulation in human cancers, ulcerative colitis, and Crohn’s disease. Three sequences were chosen to serve as sgRNA spacers, and initially developed for the same pSB1C3-T7-sgRNA scaffold system as the pUC19 sgRNAs. We found that the transformation efficiency of ligated plasmid was serviceably increased by linearizing the pSB1C3 plasmid via PCR before the BbsI digest, with a BbsI cut site positioned at each end of the linear DNA, indicating a poor double-digest efficiency with BbsI. The performance of these sgRNAs was examined with a cleavage assay by operating on PCR product that included DD96.
</P><FIGURE><CENTER><FIGCAPTION> Figure 2: Results of in vitro cleavage assay with Cas9. Each lane contains the results of a reaction consisting of S. pyogenes Cas9, NEB 3.1 buffer, and either no sgRNA or an sgRNA with a specific target in the DD96 gene. After 10 minutes of incubating the sgRNA and Cas9 together in each reaction tube, a 1839 bp PCR product from the DD96 gene was added to each tube and the reactions were incubated for 1 hour at 37oC. The first lane right of the ladder contains the PCR product without any added sgRNA as a control; the remaining lanes contain the results of in vitro reactions using three different target sgRNAs. The bands expected from successfully cleaved target 401 are 288 bp and 1551 bp, the bands expected from successfully cleaved target 558 are 445 bp and 1394 bp, and the bands expected from successfully cleaved target 618 are 505 bp and 1334 bp. sgRNAs against targets 401 and 618 appear to function, as their larger expected bands are visible alongside the uncut PCR product, which is present in all tubes near the 2.0kb mark, while the expected bands are not visible for 558. </FIGCAPTION></CENTER></FIGURE><P>To produce the Cas9-aTcdB construct, Gibson assembly was used to combine DNA for aTcdB-pHis1522 and Cas9 with a glycine-serine linker. Successful assembly was screened for by Pst1 digest of DNA purified from the colonies (Figure 3 Left). Junction PCR was then used to confirm the successful digest of colony 5. The first three lanes after the ladder are of junction 1, which is between the promoter region of pHis1522 and the beginning of Cas9, and the next three lanes are of junction 2, which is between the end of Cas9 and the beginning of aTcdB (Figure 3 Right). All of the junctional PCRs produced the expected band for the assembled plasmid.
</P><FIGURE><CENTER><FIGCAPTION> Figure 3: Left - From left to right the lanes are ladder, then DNA from seven colonies subjected to Pst1 digest; the fifth colony was successfully digested. Right - From left to right, the lanes are ladder, then three gradient PCRs of junction 1, then three gradient PCRs of junction 2. </FIGCAPTION></CENTER></FIGURE><P>In order to control for the ability of Cas9 to cleave the target site inside the cell even without the aTcdB delivery portion, Cas9-aTcdB (CP) was assembled. Another junctional PCR was run to confirm CAP, AP, and CP. Junction 1, going from the promoter region of pHis into Cas9, should be present in CAP and CP. Junction 2, going from the end of Cas9 to the beginning of aTcdB, should be present in CAP only. AP should not have come up with any product for junction 1, but there are two bands which could be caused by competitive binding of the primers. This result is extra odd, because AP is the plasmid that we know with 100% certainty. </P><FIGURE><CENTER><FIGCAPTION>Figure 4: Junction PCR of CAP, AP, and CP. From left to right, ladder, CAP junction 1, AP junction 1, CP junction 1, CAP junction 2, AP junction 2, and CP junction 2. </FIGCAPTION></CENTER></FIGURE><P>Induction of CAP expression in B. megaterium was attempted using xylose to confirm that it was possible to control production of the large protein. The B. megaterium cells were grown up to OD600, from which a 20 mL portion was removed as an uninduced reference before inducing the remaining bacteria. Very faint bands of the expected size were seen after SDS-PAGE analysis of the lysate and not present in the uninduced time point, which indicates possible successful induction. We attempted to induce again, but the SDS-PAGE analysis came out blank at the high molecular weight we would expect to see our protein. </P><FIGURE><CENTER><FIGCAPTION>Figure 5: SDS-PAGE gel. From left to right, the four visible lanes are uninduced and 1, 2, and 3 hours post-induction with 0.5% w/v xylose. All samples were 20 mL of bacteria and medium which were pelleted and frozen at -80oC before thawing, weighing, and lysing for 15 minutes with room temperature B-PER lysis reagent. Faint bands are visible in the three post-induction lanes above the 185 kD mark of the ladder (far left, mostly not visible), which are absent in the 
induced fraction.</FIGCAPTION></CENTER></FIGURE><P>
Due to the inconsistent results in induction, we wanted to confirm that we can actually induce in B. meg through a construction of NanoLuc with and without aTcdB under the xylose expression system of pHis. Gibson assembly of NanoLuc-aTcdB-pHis (NAP) and NanoLuc-pHis (NP) was attempted and two clones of each construct was checked for correct assembly using a junctional PCR(Figure 6). The gel on the left shows the the amplification of NAP with a forward primer binding upstream of Nanoluc and a primer binding at its 3’ end . If the NanoLuc insert is present, the product should be around 1.2 kB, and without insert, the product should be around 0.7 kB. Colony 1 has the insert and colony 2 does not. The gel on the right shows the amplification around NanoLuc in pHis, but the product is much larger (around 3 kB) than it should be around 1.2 kB.  
</P><FIGURE><CENTER><FIGCAPTION>Figure 6: Junction PCRs to confirm NAP and NP. The left gel is the amplification of NAP, binding to AP and amplifying over NanoLuc. The first lane is the ladder, lanes 2-4 are of colony 1, and lanes 5-7 are of colony 2. Colony 1 has the insert, and colony 2 does not. The gel on the right is for NP, with the primers binding. Again, the first lane is the ladder, lanes 2-4 are of colony 1, and lanes 5-7 are of colony 2. Results for NP are inconclusive.  </FIGCAPTION></CENTER></FIGURE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>