Team:Tufts/Notebook
HOME
TEAM
Team Bios
PROJECT
Description
Experiments & Protocols
Results
Design
PARTS
NOTEBOOK
ATTRIBUTIONS
COLLABORATIONS
HUMAN PRACTICES
SAFETY
SPONSORS
Notebook
This project is a continuation of the Tufts 2015 iGEM project. Lab work from last year included various attempts at assembling Cas9 into the aTcdB-pHis (AP) vector obtained from Dr. Xingmin Sun, as well as attempts at protein expression and purification. The 2016 project lab notebook begins with a new assembly of Cas9-aTcdB-pHis (CAP) using Gibson assembly.
Work from the previous year was carried over into the initial design of the project, mainly the design of the initial pSB1C3 vector which contained a T7 promoter, the gRNA scaffold component, and BbsI cut sites to be used to insert annealed oligonucleotides that contain the gRNA target component. As a proof of concept for gRNA production using this vector, target sites on the pUC19 vector had been selected.
Week 1 : May 23-29
Cas9-aTcdB: Gibson assembly of Cas9 and aTcdB was performed.
Week 2 : May 30-June5
Cas9-aTcdB: Colonies were screened for the proper assembly with Pst1 digest. Colony 5 came up positive on the Pst1 digest (left), then junction PCR was performed at various temperatures over both junctions to confirm (right). Miniprepped and made stocks of this assembly.
Week 3 : June 6-12
Cas9-aTcdB: Prepped all supplies for Bacillus megaterium work. Made competent B. meg cells following Dr. Sun’s protocol. Performed a gram stain on original B. meg stock from Dr. Sun, the new B. meg competent cells, and JM109 E. coli. Microscope was not powerful enough to correctly interpret gram stain with confidence, but the new B. meg competent cells appeared smaller than the original B. meg stock, but not as small as E. coli. Transformed CAP into the B. meg competent cells made in house, as well as the competent cells made by Dr. Sun and commercial competent cells from MoBiTech. The in house competent cells did not produce a pellet, suspected that no/few cells were actually present, so this sample was abandoned. The CAP transformed into MoBiTec’s protoplasts seemed to have worked.
Week 4 : June 13-19
Cas9-aTcdB: Ran a Gibson assembly of Cas9 into pHis1522 (CP), which can be used as a control for any Cas9 activity without the delivery system, due to its lack in the aTcdB domain. This was transformed into NEB 10 beta cells, where many colonies were transformed. A few colonies were grown up and mini prepped.
gRNA: Designed gRNA oligos to target DD96, which is upregulated in Ulcerative Colitis, Crohn’s disease, and human cancers.
Week 5 : June 20-26
Cas9-aTcdB: Induced expression of CAP in B. meg with xylose. The protein induction protocol was followed for the most part, but various time points were used for induction. A culture of CAP B. meg was seeded and grown to an OD600 of about 0.3. 20 mL was withdrawn to serve as uninduced reference. The remaining was induced with 0.5% w/v xylose. 20 mL samples were withdrawn at 1, 2, and 3 hr timepoints, then pelleted and frozen at -80C. Later, all pellets were thawed, weighed, and lysed with room temp B-PER lysis reagent for 15 min on rocker. Clarified lysate by centrifugation for 20 min at around 3200 rpm, then the supernatant was aspirated. An SDS-PAGE gel was then run for all samples. After overnight staining and overnight destaining, faint bands above the 185 kD mark were very faintly seen in the induced timepoints, but not the uninduced.
Week 6 : June 27-July 3
Cas9-aTcdB: Attempted to make B. meg protoplasts and attempted to transform CAP into them again.
gRNA: Digested the T7-pSB1C3 template with BbsI and annealed the gRNA target oligonucleotides for insertion into the vector. Attempted ligation with DD96 target 401, 558, and 618. After miniprep, we determined that the three tested colonies of each ligation did not contain insert, with the relatively high colony count on the control plate indicating poor efficiency.
Week 7 : July 4-10
Cas9-aTcdB: The faintness of the bands where we expect the Cas9-aTcdB protein could have been caused by cleavage of aTcdB. aTcdB naturally undergoes autoproteolytic activity as it enters the cell through endocytic vesicles. In order to reduce this cleavage, we induced expression in B. meg again, this time adding protease inhibitor cocktail (PIC) to some of the samples.
gRNA: Designed primers intended to linearize T7-pSB1C3 with BbsI cut sites on either end, which would be adjacent to the T7 promoter and gRNA scaffold respectively. Reattempted ligation protocol, using a newer tube of BbsI to see if it would sufficiently improve digest efficiency. Control plate still had colonies, though the three insert plates for 401, 558, and 618 appeared to be above background.
Week 8 : July 11-17
Cas9-aTcdB: An SDS-PAGE gel was run of the B. meg CAP induction with PIC. There were no visible bands at a high molecular weight level. Another attempt at making in-house protoplasts failed.
gRNA: Previous week’s ligation determined to be unsuccessful by PCR. Confirmed that the BbsI stock used was active through transformation efficiency of digested vs. undigested T7-pSB1C3 into competent E. coli, indicating that the problem may lie with efficiency of digesting at two sites. PCR of T7-pSB1C3 after the arrival of the primers designed during the previous week to linearize, which were then digested.
Week 9 : July 18-24
gRNA: Ran the ligations of linearized, digested T7-pSB1C3 with inserts 401, 558, and 618. Colony counts indicate success, so the DNA was purified.
Week 10 : July 25-31
gRNA: Prepared and submitted 4 ligation samples of each insert for sequencing, all of them except for anomalous 401 sequence that had two inserts were successfully ligated. PCR to amplify linear section for IVT (in vitro transcription).
Week 11 : August 1-7
Cas9-aTcdB: Designed Gibson assembly primers for Cas9-aTcdB into the control plasmid for IVTT (CAP-IVTT). IVTT stands for In Vitro Transcription and Translation which can be used to make large proteins without the need for a bacterial strain like B. meg. The IVTT control plasmid adds all necessary promoters and binding regions for the IVTT system to produce the protein.
gRNA: Ran IVT reactions on our three samples to obtain gRNAs. These gRNAs were then used with purified Cas9 from NEB in an in vitro assay for cleavage of genomic DD96. The cleavage results can be seen on the gel below. This gel showed clear cuts with target 401 and 618. Target 558 may have worked as well, but we cannot say this with the same degree of certainty because the bands were much fainter for this target.
Left to Right: Ladder, Control, 401, 558, 618
Weeks 12-14 : August 2-22
Multiple attempts to make B. meg protoplasts. For the final time during this period, the sample was checked often under the microscope to see how the B. meg cells changed over time. At the beginning, the B. meg looked normal as can be seen in the image on the left. After a little time in lysozyme, the cells began to break up into very small bits, as seen in the image on the right.
Weeks 15-16 : August 23 - September 4
End of summer break before returning for classes.
Week 17 : September 5-11
Cas9-aTcdB: Transformed CAP and AP into protoplasts made 8/22. It is unclear if colonies grew overnight. Some potential colonies were picked and grown overnight, but there was no growth, so we determined this round of protoplasts did not work. Ran junction PCRs to confirm CAP, AP, and CP. The two junctions tested were at the beginning and end of Cas9 in CAP. Junction 1, going from the promoter region of pHis into Cas9, should be present in CAP and CP. Junction 2, going from the end of Cas9 to the beginning of aTcdB, should be present in CAP only. In the gel image, lane 1 is the ladder, lane 2 is CAP junction 1, lane 3 is AP junction 1, lane 4 is CP junction 1, and lane 5 is CAP junction 2. Nothing came up for AP and CP junction 2. AP should not have come up with any product for junction 1, but there are two bands which could be caused by competitive binding of the primers. It also could’ve been caused by contamination between the lanes of the gel.
Week 18 : September 12-18
Cas9-aTcdB: Assembled CAP-IVTT with the Gibson primers designed in August. Very low concentration and poor DNA quality, and no results on a junction PCR that binds to Cas9-aTcdB and amplifies the IVTT plasmid.
Week 19 : September 19-25
Cas9-aTcdB: Because it appears that CAP has been successfully cloned, but there is trouble with transformation and expression in B. meg, Gibson assembly primers were designed for NanoLuc-aTcdB-pHis (NAP). This will serve as further validation that aTcdB can be used as a delivery system, because NanoLuc is a much smaller protein that is easier for B. meg to produce, and will be easier to validate in an assay. NanoLuc-pHis (NP) was also designed in order to validate the xylose expression system of the pHis vector that is used for protein production in B. meg.
Week 20 : September 26-October 3
Cas9-aTcdB: Attempted to construct NAP and NP. PCR of most pieces worked, but AP did not. Decided to amplify AP in two segments, which would then be put together in a three fragment Gibson assembly to construct NAP. A BioBrick assembly plasmid of NanoLuc-aTcdB was designed.
Week 21 : October 4-10
Cas9-aTcdB: Re-ran the CAP-IVTT Gibson assembly, and transformed NAP, NP, and CAP-IVTT into NEB 10 beta cells. CAP-IVTT control came out very clean with many colonies from the transformation. NAP had four colonies and NP had many colonies, but the control also had a few colonies. Miniprepped a few colonies from each, and all came out with high concentrations and good DNA quality. Ran a restriction digest to confirm the CAP-IVTT plasmid, but that came out negative, so we decided to scratch the CAP-IVTT route of the project. Attempted to transform NAP, NP, and AP into MoBiTec B. meg, but no cells were present when spinning the cells down or after overnight growth of the plates.
Week 22 : October 11-17
Cas9-aTcdB: Submitted NAP, NP, and CAP-IVTT for sequencing. We used power read, but the sequencing results never came back in time for the wiki freeze. We also attempted to induce NP in E. coli to confirm by functionality, but the NanoLuc assay results came out negative. A junction PCR was run to confirm NAP and NP, results below. NanoLuc-aTcdB-pSB1C3 (NA-BioBrick) was also assembled.
