<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tufts_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tufts/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer"><UL><LI><A href="https://2016.igem.org/Team:Tufts">HOME</A></LI><LI><A href="#">TEAM
            					</A></LI><LI><A href="https://2016.igem.org/Team:Tufts/Team">Team Bios</A></LI></UL><LI><A href="#">PROJECT
            					</A></LI><LI><A href="https://2016.igem.org/Team:Tufts/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Tufts/Experiments">Experiments &amp; Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Tufts/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Tufts/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Tufts/Parts">PARTS</A></LI><LI><A href="https://2016.igem.org/Team:Tufts/Notebook">NOTEBOOK</A></LI><LI><A href="https://2016.igem.org/Team:Tufts/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2016.igem.org/Team:Tufts/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2016.igem.org/Team:Tufts/Practices">HUMAN PRACTICES
          
					</A></LI><LI><A href="https://2016.igem.org/Team:Tufts/Safety">SAFETY</A></LI><LI><A href="https://2016.igem.org/Team:Tufts/Sponsors"> SPONSORS </A></LI></DIV><DIV id="contentContainer"><DIV class="column full_size"><H2> Project Description </H2><P>The field of synthetic biology is expanding, and speculating about future technology has become easier as new applications are discovered and directions are pursued. Perhaps biological engineering will provide nutritious, pest-resistant crops that feed the world. Or animals that constantly produce medications, bacteria that power cities with biofuels. A cure to HIV that cuts its DNA straight from out of the genome. With an easily usable CRISPR/Cas9 system, these innovations may all become possible to the global community.
</P><P>C. diff toxin B (tcdB) is a marvelously evolved protein, dedicated to crossing cell membranes. Though massive in size, it binds to carbohydrate receptors on the cell membrane and forces it to undergo endocytosis, enclosing the toxin inside a vesicle. Typically such endosomes have lowered pH to degrade their contents, tcdB however undergoes a conformational change at low pH. The toxin forms a pore in the membrane of the endosome, then releases its payload into the cytosol. In nature, this consists of a lethal glucosyltransferase domain (6). By rendering this region atoxic by mutating catalytic residues, tcdB is rendered nontoxic and becomes a potential vehicle for crossing the cell membrane. Atoxic C. diff toxin B (atcdB) has demonstrated the ability to transport a small alkyltransferase protein across the cell membrane, which was connected to the former position of the glucosyltransferase domain with a glycine-serine linker and successfully released into the cell (7).
</P><FIGURE><FIGCAPTION>C Diff Toxin image from (7)</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION> C Diff Toxin B chain structure image from (7)</FIGCAPTION></FIGURE><P>Clustered Regularly Interspaced Short Palindromic Repeats (or CRISPRs) are naturally-occurring portions of the genetic code found in many bacteria, comprising the smallest known adaptive immune system (1). The DNA itself codes for guide RNAs (gRNA), and there are a set of Cas genes or CRISPR Associated genes, that code for proteins involved in the creation, maintenance, and use of the CRISPR (2). Within the CRISPR, repeats are spaced by a short sequence of about twenty base pairs in length, which targets a specific sequence of invasive DNA, such as a viral genome. When transcribed, self-annealed, and spliced correctly, these sequences give rise to the guide RNAs, capable of directing Cas9. Cas9 in conjunction with the proper gRNA will cut the specified viral DNA with site-specific double-stranded breaks (3). </P><FIGURE><FIGCAPTION> Structure of Cas9 with guide RNA - Crystal image rendered from PDB4CMP Jinek et al. 2014)</FIGCAPTION></FIGURE><P>  The unique efficacy of the CRISPR/Cas9 system lies in the presence of this gRNA. By simply knowing a suitable target sequence, the gRNA can be synthesized with a 5’ region that is complementary to a specific gene, allowing the Cas9/gRNA complex to bind to the target sequence. Compared to other customizable genome-targeting tools, such as zinc-finger nucleases, CRISPR/Cas9’s gRNA is far easier to synthesize and more likely to bind to the correct target. With the addition of a donor DNA construct, it is even possible to incorporate new DNA into the genome where the double-stranded breaks were present by homologous recombination. Even in the present moment, the CRISPR/Cas9 system provides us with the tools needed for programmable, precise, and nearly limitless engineering of the genome (4). </P><P>Although off-target cutting poses significant risks, therapeutic usage CRISPR/Cas9 will not be possible until more effective delivery strategies are developed. Efficient delivery of Cas9 and potential DNA donor sequences has not been achieved with techniques suitable for therapeutic purposes. As operating the CRISPR/Cas9 system directly on an organism requires that the system successfully enter and operate on somatic cells, this is a difficult obstacle to any in vivo applications (5). An effective delivery platform is needed, and this project aims to leverage the Clostridium difficile toxin B’s (tcdB) cellular penetration capability. </P><P>Our project’s aim is to test the efficacy of C. diff toxin B as a platform for transporting a functional Cas9-gRNA complex into a cell for genome editing. The same glycine-serine linker is used to connect the payload of Cas9, but it is much larger and potentially more challenging to deliver than the previously tested alkyltransferase. Furthermore, the protein must be expressed in Bacillus megaterium cells, as the extreme size and complexity of Cas9-aTcdB cannot easily be folded in E. coli, and the purified protein must be incubated with the gRNA prior to treatment of cells.</P><P> The complete atcdB-Cas9 protein will be tested on HeLa cells. The gRNA to be used in this test targets the Tet Repressor Protein (TetR) gene, which in turn prevents the expression of GFP. Cas9 delivered inside the cell membrane should remove the TetR gene, and once the existing TetR degrades, GFP should become expressed at easily measurable levels. If there is a significant increase in fluorescence following exposure of the cell culture to the Cas9-aTcdB construct, then the function of our fusion protein will be verified. As another potential target for our delivered Cas9, we made guide RNAs to target DD96. The PDZK1IP1 (PDZK1 Interacting Protein 1, also designated DD96) gene was chosen due to its upregulation in human cancers, ulcerative colitis, and Crohn’s disease. </P><P> NanoLuc is a luciferase derivative which is fairly small (19kDa) and also produces a much bigger signal (2.5 millions times that of luciferase) (8).  A NanoLuc-aTcdB protein will be developed in parallel as a form of a control experiment, if aTcdB-Cas9 proves to have difficulty folding or otherwise being produced.  The mechanism of delivery would be the same as the Cas9-aTcdB fusion protein, substituting in NanoLuc at the glucosyltransferase domain.  A successful NanoLuc-aTcdB fusion protein would not only provide an easy assay for testing in HeLa cells, but also would demonstrate the versatility of the aTcdB delivery system as an orthogonal tool for wider drug delivery. </P><FIGURE><FIGCAPTION> NanoLuc Size and Luminescence Comparison(8)</FIGCAPTION></FIGURE><H2>Citations</H2><P2>1) Mojica F.M., Díez-Villaseñor C, Garcí­a-Martí­nez J, &amp; Soria, E. Intervening sequences of regularly spaced prokaryotic repeats derive from foreign genetic elements. Journal of Molecular Evolution 60(2), 174-182 (2005) 
2) Jansen R, Embden J. D.A.v., Gaastra W, &amp; Schouls L.M. Identification of genes that are associated with DNA repeats in prokaryotes. Molecular Microbiology, 43(6), 1565-1575 (2002).
3) Jinek M, Chylinski K, Fonfara I, Hauer M, Doudna J.A.,Charpentier E. A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity. Science337, 816–821 (2012).
4) Mali P, Yang L, Esvelt K.M., Aach J, Guell M, DiCarlo J.E., Norville J.E., Church G.M. RNA-Guided Human Genome Engineering via Cas9. Science339(6121), 823-826 (2013).
5) Ebina H., Misawa N., Kanemura Y., Koyanagi Y. Harnessing the CRISPR/Cas9 system to disrupt latent HIV-1 provirus. Sci Rep 3, 2510 (2013)
6) Pruitt RN, Lacy DB (2012) Toward a structural understanding of Clostridium difficile toxins A and B. Front Cell Infect Microbiol 2:28 (2012)
7) Krautz-Peterson G, Zhang Y, Chen K, Oyler GA, Feng H, Shoemaker CB. Retargeting Clostridium difficile Toxin B to neuronal cells as a potential vehicle for cytosolic delivery of therapeutic biomolecules to treat botulism. J. Toxicol. (2012) 
8) Hall MP, Unch J, Binkowski BF, et al. Engineered Luciferase Reporter from a Deep Sea Shrimp Utilizing a Novel Imidazopyrazinone Substrate. ACS Chemical Biology. 2012;7(11):1848-1857. doi:10.1021/cb3002478.</P2></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>