<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NYU_Shanghai_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NYU Shanghai/Protocols</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="nav" class="scrollable"><UL class="noselect"><LI id="project"><H1 id="project-title" class="noselect" onmouseenter="selectProject()" onmouseleave="deselectProject()" onclick="reset();revealProject()"><FONT color="white">Project</FONT></H1></LI><LI><A href="/Team:NYU_Shanghai/Design">Design</A></LI><LI><A href="/Team:NYU_Shanghai/Parts">Parts</A></LI><LI><A href="/Team:NYU_Shanghai/Code">Code</A></LI><LI><A href="https://igem.org/2015_Judging_Form?id=1756">Judging</A></LI></UL><LI id="team"><H1 id="team-title" class="noselect" onmouseenter="selectTeam()" onmouseleave="deselectTeam()" onclick="reset();revealTeam()"><FONT color="white">Team</FONT></H1></LI><LI><A href="/Team:NYU_Shanghai/Members">Members</A></LI><LI><A href="/Team:NYU_Shanghai/Partners">Partners</A></LI><LI><A href="/Team:NYU_Shanghai/Attributions">Attributions</A></LI><LI id="outreach"><H1 id="outreach-title" class="noselect" onmouseenter="selectOutreach()" onmouseleave="deselectOutreach()" onclick="reset();revealOutreach()"><FONT color="white">Outreach</FONT></H1></LI><LI><A href="/Team:NYU_Shanghai/Collaborations">Collaborations</A></LI><LI><A href="/Team:NYU_Shanghai/Practices">Practices</A></LI><LI><A href="/Team:NYU_Shanghai/VirtualReality">Virtual Reality</A></LI><LI id="notebook"><H1 id="notebook-title" class="noselect" onmouseenter="selectNotebook()" onmouseleave="deselectNotebook()" onclick="reset();revealNotebook()"><FONT color="white">Notebook</FONT></H1></LI><LI><A href="/Team:NYU_Shanghai/DailyNotes">Daily Notes</A></LI><LI><A href="/Team:NYU_Shanghai/Protocols">Protocols</A></LI><LI><A href="/Team:NYU_Shanghai/Bibliography">Bibliography</A></LI></DIV><DIV id="overview"><H3>Protocols</H3><P style="font-size: 17px">We built our constructs from pre-made biobrick parts. Our overall conclusion is that 3A assembly is generally inefficient, and an insufficient method for adding small parts (such as a terminator) to a larger construction within pSB1C3. We learned that ratios were extremely important in the process of 3A Assembly, and we made a <A href="https://static.igem.org/mediawiki/2015/c/c0/NYU_Shanghai_3AAssemblyNumbers.pdf">summary sheet</A> of the equations we used in pre-digest and pre-ligation that accounts for digest dilution and amount needed to ensure results are seen on a gel, not just ligation ratios. We wished we used gibson assembly. </P></DIV><DIV id="makingColor"><H4>Making Color</H4><DIV id="luc" class="collapsed"><H5 style="display:inline-block" onclick="expandluc()"><SPAN class="noselect">Luciferase</SPAN></H5><DIV id="lucText" style="display:none"><P>Overview
    <OL><LI>Luciferin substrate must be added.</LI><LI>D-Luciferin is too large of a chemical to cross the plasma membrane of E. Coli so cell lysis is required to extract luciferase.</LI><LI>After cell lysis, the reagent solution can be added to the lysis buffer. Light should be emitted within 5 to 10 seconds of adding the reagent solution.</LI><LI>The luciferase/luciferin reaction at 22.5 ºC theoretically offers the greatest light intensity.</LI><LI>Solutions of D-Luciferin should be aliquotted and stored in darkness at -80 ºC</LI><LI>We were only able to see the color in a very dark room.</LI></OL></P><P>Materials
      <LI>D-Luciferin free acid</LI><LI>ATP</LI><LI>MgSO4 · 7H2O</LI><LI>1M HEPES Buffer</LI><LI>Lysozyme</LI><LI>10 mM Tris-HCl</LI></P><P>Lysis Buffer
    For E. coli cell lysis, use a freshly prepared lysozyme solution (10 mg/ml) in 10 mM Tris-HCl, pH 8.0.
    </P><P>Reagent Solution
    Prepare using ATP free water. Combine 1 mM luciferin or luciferin salt, 3 mM ATP and 15 mM MgSO4 in 30mM HEPES buffer, pH 7.8.  Store substrate solution at -20ºC in polypropylene or glass.
    </P><P>Preparing 1mM D-Luciferin
    Directions for a 5 mg sample: Dissolve 0.034 gr dithiothreitol in 22 mL of QH2O. Add 0.2 mL of this DTT solution to the 5 mg d-luciferin. Add 4μl of 10M NaOH to dissolve the luciferin. Dilute this into the remaining 21.8 mL of DTT solution and store as aliquots at - 80 ºC in darkness until use.
    </P><P>Procedure
    Bacterial lysis:
    <OL><LI>After 12-18 hours of inoculation of bacteria expressing luciferase plasmid, pipette 2 mL of cell culture into a clean 2 mL tube. Centrifuge at 10,000 rpm for 1 minute. Pour out liquid into a collection beaker. Continue this process until all of the cell culture (in the inoculation tube) is gone.</LI><LI>Resuspend the pellets in 350 ml of STET buffer (10 mM Tris-HCl with 1 mM EDTA)</LI><LI>Add 25μl - 30μl of lysozyme buffer to the resuspended pellet.</LI><LI>Mix by vortexing for 3 seconds.</LI><LI>Incubate for 2 hours at room temperature.</LI><LI>If the reagent is not added immediately, store the lysed bacteria in the -20 ºC freezer until use.</LI></OL></P><P>
    Addition of Reagent Solution:
    <OL><LI>Following the above instructions, prepare a 1mM sample of D-Luciferin.</LI><LI>Following the above recipe, prepare the reagent solution.</LI><LI>In a dark room, add about 250-350μl of reagent solution to each sample of lysis product.</LI><LI>Light should be emitted within two-three seconds.</LI></OL></P><P>Example Calculations
    Lysis Buffer (Desired Total Volume: 15mL)<TABLE border="1"><TBODY><TR><TD><FONT color="#d66">Chemical Name</FONT></TD><TD>Tris-HCl</TD><TD>EDTA</TD><TD>NaCl</TD></TR><TR><TD><FONT color="#d66">Molecular Weight</FONT></TD><TD>N/A</TD><TD>292.23 g/mol</TD><TD>58.44 g/mol</TD></TR><TR><TD><FONT color="#d66">Molarity Desired</FONT></TD><TD>10 mM</TD><TD>1mM</TD><TD>0.1M</TD></TR><TR><TD><FONT color="#d66">Calculation</FONT></TD><TD>Dilute 1M Tris-HCl:</TD></TR><TR><TD><FONT color="#d66">Final Amount</FONT></TD><TD>150μl (+14.85 mL ddH2O)</TD><TD>0.00438 g</TD><TD>0.08766 g</TD></TR></TBODY></TABLE></P><P>
    Lysozyme Solution (Desired Total Volume: 15mL)<TABLE border="1"><TBODY><TR><TD><FONT color="#d66">Lysozyme Solubility</FONT></TD><TD>10 mg lysozyme in 1 mL of 10 mM Tris-HCl</TD></TR><TR><TD><FONT color="#d66">Desired Amount of Lysozyme Solution to Make</FONT></TD><TD>15 mL</TD></TR><TR><TD><FONT color="#d66">Amount of Lysozyme Needed</FONT></TD><TD>10 mg x 15 = <B>150 mg</B></TD></TR><TR><TD><FONT color="#d66">Amount of 10 mM Tris-HCl Needed</FONT></TD><TD><B>15 mL</B></TD></TR></TBODY></TABLE></P><P>
    Reagent Solution (Desired total volume: 22 mL)<TABLE border="1"><TBODY><TR><TD><FONT color="#d66">Chemical Name</FONT></TD><TD>ATP disodium salt trihydrate</TD><TD>MgSO4•7H2O</TD><TD>HEPES Buffer</TD><TD>D-Luciferin free acid</TD></TR><TR><TD><FONT color="#d66">Molecular Weight</FONT></TD><TD>605.24 g/mol</TD><TD>246.5 g/mol</TD><TD>238.3 g/mol</TD><TD>280.33 g/mol</TD></TR><TR><TD><FONT color="#d66">Molarity Desired</FONT></TD><TD>3 mM</TD><TD>15 mM</TD><TD>30 mM</TD><TD>1 mM</TD></TR><TR><TD><FONT color="#d66">Calculation</FONT></TD><TD>Use the 1 mM stock solution created earlier</TD></TR><TR><TD><FONT color="#d66">Final Amount</FONT></TD><TD>0.039945 g</TD><TD>0.081345 g</TD><TD>0.1573 g</TD><TD>22 mL</TD></TR></TBODY></TABLE></P><P>
    Controls
    <LI>No arabinose added during inoculation</LI><LI>Use bacteria without luciferase plasmid and go through steps to induce color</LI></P></DIV></DIV><DIV id="chromo" class="collapsed"><H5 style="display:inline-block" onclick="expandchromo()"><SPAN class="noselect">Chromoproteins</SPAN></H5><DIV id="chromoText" style="display:none"><H6><FONT color="#d66">Building our Construct: from biobrick parts in the kit</FONT></H6><P>Note: If using construct with pBAD promoter, DO NOT USE SOC MEDIA. Glucose inhibits the uptake of arabinose, and will inhibit promoter induction.
    Note: We should have used PCR to amplify linearized backbone.
    Note: Always use gel electrophoresis to check digest results.</P><H6><FONT color="#d66">Building our Construct: from IDT gBlocks</FONT></H6><P>Note: We recommend adding a reporter gene to the construct.</P><H6><FONT color="#d66">Expressing XJTLU's Construct</FONT></H6></DIV></DIV></DIV><DIV id="recipes"><H4>Recipes</H4><DIV id="Plate" class="collapsed"><H5 style="display:inline-block" onclick="expandPlate()"><SPAN class="noselect">LB-Agar Plates</SPAN></H5><DIV id="PlateText" style="display:none"><P><OL><LI>Calculate total amount of volume of LB agar you would need to use. You need about 25mL for every 10cm plate and 10mL for every 6cm plate.</LI><LI>For every 100mL of LB agar needed, measure out 4g of LB Agar Powder. 
      </LI><LI>Put the agar powder in a flask. Use a much bigger flask to prevent overflow when autoclaiving. </LI><LI>Put deionized water into the flask up to the desired volume. <STRONG>Do not mix.</STRONG> LB Agar is not supposed to dissolve in room temperature. If you mix the powder it will get stuck on the walls of the flask</LI><LI>Autoclave. </LI><LI>After autoclave, move to the hood. Let the LB agar cool until it’s less than 60ºC or until you can touch the flask. While waiting, you can start labeling the plates. <STRONG>DO NOT LET THE AGAR SOLIDIFY</STRONG></LI><LI>Insert any antibiotics/sugars needed. If you are not sure about the ratio, you can follow this simple example:
      </LI></OL></P><P style="text-align: center;"><B>50mL LB + 500µL Arabinose* + 50µL Ampicillin**</B><SPAN style="font-size: 10;"><I>
      *2mg/mL of Arabinose Stock Solution
      **100mg/mL of Ampicillin Stock Solution</I></SPAN></P><LI>Swirl the antibiotics/sugars that you just added for a minute.</LI><LI>Plate the LB agar into the petri dishes until it is about ⅓ of the level. Do not forget to keep the lid on before and after you plate.</LI><LI> Let it cool for about 5 minutes.</LI><LI>Invert each plate so that it is upside down</LI><LI>Store in the 4ºC freezer</LI></DIV></DIV><DIV id="LB" class="collapsed"><H5 style="display:inline-block" onclick="expandLB()"><SPAN class="noselect">LB Broth</SPAN></H5><DIV id="LBText" style="display:none"><TABLE><TBODY><TR style="text-align: center; font-weight: bold;"><TD>Total Amount of Reagent</TD><TD>100mL</TD><TD>250mL</TD></TR><TR><TD>Deionized Water</TD><TD>100mL</TD><TD>250mL</TD></TR><TR><TD>Yeast</TD><TD>1g</TD><TD>2.5g</TD></TR><TR><TD>Tryptone</TD><TD>0.5g</TD><TD>1.25g</TD></TR><TR><TD>NaCl</TD><TD>1g</TD><TD>2.5g</TD></TR></TBODY></TABLE></DIV></DIV><DIV id="SOC" class="collapsed"><H5 style="display:inline-block" onclick="expandSOC()"><SPAN class="noselect">SOC Media</SPAN></H5><DIV id="SOCText" style="display:none"><P>
    Materials
    <LI>0.5% (w/v) yeast extract</LI><LI>2% (w/v) tryptone</LI><LI>10 mM NaCl</LI><LI>2.5 mM KCl</LI><LI>20 mM MgSO4</LI>Per liter:
    <LI>5 g yeast extract</LI><LI>20 g tryptone</LI><LI>0.584 g NaCl</LI><LI>0.186 g KCl</LI><LI>2.4 g MgSO4</LI></P><P><EM>Note:</EM> Some formulations of SOB use 10 mM MgCl2 and 10 mM MgSO4 instead of 20 mM MgSO4.
SOB medium is also available dry premixed from Difco, 0443-17.Adjust to pH 7.5 prior to use. This requires approximately 25 ml of 1M NaOH per liter.<B>15/10 medium</B>Growth of competent TOP10 cells in Example 2 of the Bloom05 patent is performed in 15/10 broth, which is similar to SOB:</P><LI>1.5% yeast extract</LI><LI>1% Bacto-Tryptone</LI><LI>10mM NaCl</LI><LI>2mM KCl</LI><LI>10 mM MgCl2</LI><LI>10 mM MgSO4</LI><TABLE border="1"><TBODY><TR style="text-align: center; font-weight: bold;"><TD>Reagent</TD><TD>50 mL (in Mols)</TD><TD>100 mL (in M)</TD><TD>100 mL (in Grams)</TD></TR><TR><TD>Tryptone</TD><TD>(2% w/v) 1 g</TD><TD>2 g</TD><TD>2 g</TD></TR><TR><TD>Yeast Extract</TD><TD>(0.5% w/v) 0.25 g</TD><TD>0.5 g</TD><TD>0.5 g</TD></TR><TR><TD>NaCl</TD><TD>0.0005 M</TD><TD>0.01 mol</TD><TD>0.0584g</TD></TR><TR><TD>KCl</TD><TD>0.000125 M</TD><TD>0.0025 M</TD><TD>0.0186 g</TD></TR><TR><TD>MgCl2</TD><TD>0.0005 M</TD><TD>0.01 M</TD><TD>0.09521 g</TD></TR><TR><TD>Glucose</TD><TD>0.001 M</TD><TD>0.02 M</TD><TD>0.36032 g</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV id="3a"><H4>3A Assembly</H4><DIV id="calculations"><H5><SPAN><A href="https://static.igem.org/mediawiki/2015/c/c0/NYU_Shanghai_3AAssemblyNumbers.pdf">Calculations (pdf)</A></SPAN></H5><DIV id="transform" class="collapsed"><H5 style="display:inline-block" onclick="expandtransform()"><SPAN class="noselect">Transformation</SPAN></H5><DIV id="transformText" style="display:none"><P>
  Estimated time: 3 hours (plus 14-18 hour incubation)
  Materials
    <LI>2µl resuspended DNA</LI><LI>Competent cells (50μl per transformation)</LI><LI>Ice</LI><LI>2ml tube</LI><LI>42ºC water bath</LI><LI>SOC or LB media</LI><LI>1 LB plate and 1 LB+antibiotic per transformation</LI><LI>Spreader</LI><LI>37ºC incubator</LI><LI>Shaker</LI></P><P>Procedure
  <OL><LI>Thaw competent cells on ice.</LI><LI>Add 50µL of thawed competent cells into pre-chilled 2ml tube.</LI><LI>Add 2µL of resuspended DNA to the 2ml tube. Pipet up and down a few times, gently. Keep the competent cells on ice.</LI><LI>Close tubes and incubate the cells on ice for 30 minutes.</LI><LI>Heat shock the cells by immersion in a pre-heated water bath at 42ºC for 60 seconds.</LI><LI>Incubate the cells on ice for 5 minutes.</LI><LI>Add 200 μl of SOC or LB media to each transformation</LI><LI>Incubate the cells at 37ºC for 2 hours at 260rpm in the shaker.</LI><LI>Label petri dishes with the appropriate antibiotic with the part number, plasmid backbone, antibiotic resistance, and date. Plate 200 µl of the transformation onto the labeled antibiotic dishes, and spread. This helps ensure that you will be able to pick out a single colony. </LI><LI>Spread remaining transformation onto labeled LB agar plate and spread, approx 50µl.</LI><LI>Incubate the plates at 37ºC for 12-18 hours, making sure the agar side of the plate is up. If incubated for too long the antibiotics start to break down and un-transformed cells will begin to grow. This is especially true for ampicillin - because the resistance enzyme is excreted by the bacteria, and inactivates the antibiotic outside of the bacteria.</LI><LI>You can pick a single colony, make a glycerol stock, grow up a cell culture and miniprep.</LI></OL></P><P>Controls
    <LI>Competent cells on LB plate</LI><LI>Competent cells on LB + antibiotic plate</LI><LI>Competent cells + plasmid on LB plate</LI></P></DIV></DIV><DIV id="mp" class="collapsed"><H5 style="display:inline-block" onclick="expandmp()"><SPAN class="noselect">Miniprep</SPAN></H5><DIV id="mpText" style="display:none"><P>
      Estimated time: 40 minutes
      Materials
      <LI>Biomiga Miniprep Kit</LI></P><P>Procedure<OL><LI>Harvest the bacterial culture by centrifugation for 1 min at 10,000 rpm. Pour off the supernatant and blot the inverted tube on a paper towel to remove residue medium. Remove the residue medium completely. </LI><LI>Add 250 µL Buffer A1 (Add RNase A to Buffer A1 before use) and completely resuspend bacterial pellet by vortexing or pipetting </LI><LI>Add 250 µL Buffer B1, mix gently by inverting the tube 10 times (do not vortex), and incubate at room temperature for 5 minutes. </LI><LI>Add 350 µL Buffer N1, mix completely by inverting/shaking the vial for 5 times and sharp hand shaking for 2 times. Note: Incubating the lysate in ice for 1 min will improve the yield. </LI><LI>Centrifuge the lysate at 13,000 rpm for 10 minutes at room temperature. If the lysate doesn’t appear clean, reverse the tube angle, centrifuge for 5 more minutes and then transfer the clear lysate to DNA column. </LI><LI>Carefully transfer the clear lysate into a DNA column with a collection tube, avoid the precipitations, spin at 13,000 rpm for 1 minute, discard the flow-through and put the column back to the collection tube. </LI><LI>Add 500 µL Buffer KB into the spin column, centrifuge at 13,000 rpm for 1 minute. Remove the spin column from the tube and discard the flow-through. Put the column back to the collection tube. </LI><LI>Add 650 µL DNA Wash Buffer (Add ethanol to DNA wash buffer before use) into the spin column, centrifuge at 13,000 rpm for 1 minute at room temperature. Remove the spin column from the tube and discard the flowthrough.</LI><LI>Reinsert the spin column, with the lid open, into the collection tube and centrifuge for 2 minutes at 13,000 rpm. </LI><LI>Carefully transfer the spin column into a sterile 1.5 mL tube and add 50-100 µL (&gt; 50 µL) Sterile ddH20 or Elution Buffer into the center of the column and let it stand for 2 minutes. Elute the DNA by centrifugation at 13,000 rpm for 1 minute. Reload the eluate into the column and elute again.</LI><LI>Check results on Nanodrop.</LI></OL></P></DIV></DIV><DIV id="rd" class="collapsed"><H5 style="display:inline-block" onclick="expandrd()"><SPAN class="noselect">Restriction Digest</SPAN></H5><DIV id="rdText" style="display:none"><P>Link to <A href="https://www.neb.com/tools-and-resources/usage-guidelines/optimizing-restriction-endonuclease-reactions"> NEB Protocol </A>To determine buffer for <A href="https://www.neb.com/tools-and-resources/interactive-tools/double-digest-finder"> double digests</A>Guide to <A href="https://www.neb.com/tools-and-resources/usage-guidelines/heat-inactivation">heat inactivation</A>General reaction set-up:
      <TABLE border="1"><TBODY><TR><TD>Restriction enzyme</TD><TD>10units, generally 1uL</TD></TR><TR><TD>DNA</TD><TD>1ug</TD></TR><TR><TD>10X NEB Buffer</TD><TD>5μl (1X)</TD></TR><TR><TD>Total Reaction Volume</TD><TD>50uL</TD></TR><TR><TD>Incubation Time</TD><TD>1hr</TD></TR><TR><TD>Incubation Temperature</TD><TD>Enzyme Dependent</TD></TR></TBODY></TABLE></P><P>Controls
  <LI>DNA with known sites for the enzyme</LI><LI>If control DNA cleaved and experimental DNA resists cleavage, the two DNAs can be mixed to determine if an inhibitor is present in the experimental sample</LI></P></DIV></DIV><DIV id="ligate" class="collapsed"><H5 style="display:inline-block" onclick="expandligate()"><SPAN class="noselect">Ligation</SPAN></H5><DIV id="ligateText" style="display:none"><P>Link to <A href="https://www.neb.com/protocols/1/01/01/dna-ligation-with-t4-dna-ligase-m0202"> NEB Protocol </A></P><OL><LI>Make Reaction mixture</LI><TABLE border="1"><TBODY><TR style="font-weight: bold;"><TD><FONT color="#d66">Components</FONT></TD><TD><FONT color="#d66">20μl Reaction</FONT></TD></TR><TR><TD>10X T4 DNA Ligase Buffer*</TD><TD>2μl</TD></TR><TR><TD>Vector DNA (4 kb)</TD><TD>50ng (0.020 pmol)</TD></TR><TR><TD>Insert DNA (1 kb)</TD><TD>37.5ng (0.060 pmol)</TD></TR><TR><TD>Nuclease-free water</TD><TD>to 20μl</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>1μl</TD></TR></TBODY></TABLE><LI>Ligation temperature and times vary</LI><TABLE border="1" width="600"><TBODY><TR><TD>For inserting a part into backbone (no 3A assembly), the suggested NEB protocol worked</TD><TD>16C overnight or room temperature 10min</TD></TR><TR><TD>For 3A Assembly</TD><TD>Room temperature for an hour, then overnight in 4degree</TD></TR></TBODY></TABLE><LI>Heat inactivate at 65°C for 10 minutes.</LI><LI>Chill on ice and transform 1-5 μl of the reaction into 50 μl competent cells.</LI></OL><P>Controls
      <LI>Reaction mixture with no insert DNA</LI><LI>Reaction mixture with no insert DNA and no ligase</LI></P></DIV></DIV><DIV id="gel" class="collapsed"><H5 style="display:inline-block" onclick="expandgel()"><SPAN class="noselect">Gel Electrophoresis</SPAN></H5><DIV id="gelText" style="display:none"><P>Reagents and Materials:</P><LI>1X TAE buffer</LI><LI>Graduated cylinder</LI><LI>125 mL flask</LI><LI>Agarose</LI><LI>Gel pouring tray</LI><LI>Tape</LI><LI>Gel rig</LI><LI>Red Safe</LI><LI>MW ruler</LI><P>Procedures</P><OL><LI>To prepare 0.4% agarose gel for electrophoresis, add 0.4 g of agarose powder into a suitable container with plenty of room to allow the liquid to boil and be swirled.</LI><LI>Add 100 ml of 1x TAE electrophoresis buffer and swirl to suspend the powder in the buffer.</LI><LI>Place the flask or the bottle into the microwave and place on a medium setting for 3 mins. Stop the microwave every 30 seconds and swirl the flask or bottle to suspend any undissolved agarose. Boil and swirl until all of the agarose gel particles are dissolved.</LI><LI>Cool the agarose solution to 55-60ºC. Add 5 µl of 10,000x DuRed and swirl to mix.</LI><LI>Prepare the gel casting apparatus and pour the molten agarose into the gel casting tray containing the comb. Allow the agarose to solidify at room temperature for 15-20 minutes.</LI><LI>Carefully remove the comb from the solidified gel.</LI><LI>Label a microcentrifuge tube for each miniprep sample.</LI><LI>DO NOT add loading dye directly to DNA minipreps collection tube. In a separate tube add 1 µl of 6x loading dye and add 5 µl of DNA minipreps sample and pipet up and down to mix.</LI><LI>Run the prepared agarose gel under water to saturate the wells. Then place the gel in the electrophoresis chamber and pour electrophoresis buffer into the chamber until it completely covers the gel by 5mm.</LI><LI>Load 5 µl of the 1 kb molecular weight ruler into lane one of the gel.</LI><LI>Load 6 µl of miniprep sample with loading dye into the wells of the gel.</LI><LI>Connect the electrophoresis chamber to the power supply and turn on the power. Make sure that the wells are closest to the black, or negative side.</LI><LI>Run the gel at 100V for 30 min, or until the molecular weight ruler is clearly separated. If you need to leave the gel for a longer time (i.e. if you need to go for lunch), you can decrease the voltage to 80V or 90V and run for an hour or 45 minutes, respectively.
      </LI><LI>Note: The voltage and times apply for a 1% agarose gel. For a lower concentration of gel (e.g. 0.7%) it will take less time for the gel to run to completion. We suggest to run the gel at 80V for 30 minutes. For longer times, run the gel at 60V for an hour or at 70V for 45 minutes.</LI><LI>Make sure the dye does not run off the gel.</LI><LI>Visualize the gel and record the results.</LI></OL><P>Controls
      <LI>Uncut plasmid</LI><LI>Uncut insert DNA</LI><LI>Ladder DNA</LI></P></DIV></DIV><DIV id="gelex" class="collapsed"><H5 style="display:inline-block" onclick="expandgelex()"><SPAN class="noselect">Gel Extraction</SPAN></H5><DIV id="gelexText" style="display:none"><P>
 Estimated time: 45 minutes
  Materials
  MinElute Midi Gel Extraction Kit
  Procedure
    <OL><LI>Pre-weigh a 2.0mL microcentrifuge tube</LI><LI>Excise DNA fragment from agarose gel with clean, sharp scalpel. Place in the pre-weighed microcentrifuge tube. Minimize the exposure of the DNA samples under UV and amount of gel cut with DNA.</LI><LI>Weigh the tube and calculate the weight of the gel. For every 1 volume of gel (100mg = 100µl) add 3 times the volume of Buffer QG. The maximum amount of gel slice per spin column is 400mg.For &gt;2% agarose gels, add 6 volumes of Buffer QG.</LI><LI>Incubate at 50ºC for 10 minutes (or until the gel has completely dissolved). Vortex the tube every 2-3 minutes during incubation to help dissolve the gel. If color is orange or violet after gel slice has dissolved, add 10µl 3M sodium acetate, pH 5.0. The color of the mixture will turn to yellow. </LI><LI>Add 1 gel volume of isopropanol to the sample and mix by inverting.</LI><LI>Place a MinElute spin column in a provided 2mL collection tube. Apply the sample in the 2.0mL microcentrifuge tube to the MinElute column and centrifuge for 1 min at 13,000 rpm. Discard the flow-through and place the MinElute column back into the same collection tube. If the sample volume exceeds 800µl, simply reload and spin again.</LI><LI>Add 500µl Buffer QG to the MinElute column and centrifuge for 1 minute at 13,000 rpm. Discard flow-through and place the MinElute column back into the same collection tube.</LI><LI>Add 750µl Buffer PE to MinElute column and centrifuge for 1 minute at 13,000 rpm. Discard flow-through and place the MinElute column back into the same collection tube. If the DNA will be used for salt-sensitive applications, such as direct sequencing and blunt-ended ligation, let the column stand 2-5 minute after addition of Buffer PE.</LI><LI>Centrifuge the column in the 2mL collection tube for 1 minute at 13,000 rpm again to remove residual ethanol.</LI><LI>Place each MinElute column into a clean 1.5mL microcentrifuge tube. To elute DNA, add 10µl Buffer EB or water to the center of the MinElute membrane. Let the column stand for 1 minute, and centrifuge the column for 1 minute at 13,000rpm.</LI><LI>Repeat step 12 but this time load the flow-through back to the membrane. Let the column stand for 1 minute, and centrifuge the column for 1 minute at 13,000rpm.</LI><LI>Check Gel extraction product with Nanodrop</LI></OL></P><P>
  Controls
  <LI>&quot;Digest empty vector cut with a single enzyme, perform the gel extraction, and re-ligate it. A vector cut with one enzyme should re-ligate very easily and provide plenty of colonies on the plate. If it does, then the inability to clone the DNA may be related to some other factor, such as secondary structure of the DNA, repeat sequences causing instability in E.coli, or the DNA cloned codes for a protein that may be toxic in bacteria.&quot; <A href="http://bitesizebio.com/13506/10-tips-for-better-dna-gel-extraction-results/">Bitesize Bio</A></LI></P></DIV></DIV><DIV id="PCR" class="collapsed"><H5 style="display:inline-block" onclick="expandPCR()"><SPAN class="noselect">PCR</SPAN></H5><DIV id="PCRText" style="display:none"><P>These are the conditions we used to PCR the gBlocks received from IDT. We used <A href="http://www.snapgene.com"> Snapgene </A> to design the primers. We used <A href="https://www.neb.com/protocols/2013/12/13/pcr-using-q5-high-fidelity-dna-polymerase-m0491">Q5 High-Fideltiy Polymerase</A>from New England Biolabs.
     <TABLE border="1"><TBODY><TR><TD><FONT color="#d66">Component</FONT></TD><TD><FONT color="#d66">50μl Reaction</FONT></TD><TD><FONT color="#d66">Concentration</FONT></TD></TR><TR><TD>5X Q5 Reaction Mix</TD><TD>25uL</TD><TD>1X</TD></TR><TR><TD>10X NEB Buffer</TD><TD>5uL</TD><TD>1X</TD></TR><TR><TD>10uM Forward primer</TD><TD>2.5uL</TD><TD>0.5uM</TD></TR><TR><TD>10uM Forward primer</TD><TD>2.5uL</TD><TD>0.5uM</TD></TR><TR><TD>Template DNA</TD><TD>10uL</TD><TD>100ng</TD></TR><TR><TD>Nuclease-free water</TD><TD>10μl or none</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><FONT color="#d66">Initial Denaturation</FONT></TD><TD>98C</TD><TD>30s</TD></TR><TR><TD><FONT color="#d66">25 cycles</FONT></TD><TD>98C</TD><TD>15s</TD></TR><TR><TD>Annealing temp 1</TD><TD>59.5C</TD><TD>30s</TD></TR><TR><TD>Annealing temp 2</TD><TD>56.3C</TD><TD>30s</TD></TR><TR><TD>Annealing temp 3</TD><TD>53.7C</TD><TD>30s</TD></TR><TR><TD>Extension</TD><TD>72C</TD><TD>60s</TD></TR><TR><TD><FONT color="#d66">Final Extension</FONT></TD><TD>72C</TD><TD>2m</TD></TR><TR><TD><FONT color="#d66">Hold</FONT></TD><TD>4C</TD></TR></TBODY></TABLE></P><P>Controls
  <LI>No template control</LI><LI>No enzyme control</LI></P></DIV></DIV><DIV id="PCRclean" class="collapsed"><H5 style="display:inline-block" onclick="expandPCRclean()"><SPAN class="noselect">PCR Cleanup</SPAN></H5><DIV id="PCRcleanText" style="display:none"><P>We used <A href="https://static.igem.org/mediawiki/2015/d/d1/NYU_Shanghai_Tianquick.pdf">TIANquick Mini Purification Kit</A>.
    <OL><LI>Add ethanol (96-100%) to Buffer PW before use (see bottle label 
for volume).</LI><LI>Column equilibration: add 500μl Buffer BL to the Spin Column CB1 (put Spin Column CB1 into a collection tube). Centrifuge for 1min at 12,000 rpm. Discard the flow-through, and then place Spin Column CB1 back into the collection tube.</LI><LI>Add 5 volumes of Buffer PB to 1 volume of the PCR reaction or enzymatic reaction and mix.</LI><LI>Transfer the mixture to the Spin Column CB1, incubate at room temperature for 2min. Centrifuge for 30-60s at 12,000rpm. Discard the flow-through, and then place Spin Column CB1 back into the same collection tube.
    The maximum loading volume of the column is 800μl. For sample volumes greater than 800 μl simply load again.</LI><LI>Add 600 μl Buffer PW (ensure that ethanol has been added) to the Spin Column CB1 and centrifuge for 30-60s at 12,000 rpm. Discard the flow-through, and place Spin Column CB1 back in the same collection tube.
    Note: If the purified DNA is used for the subsequent salt sensitive experiments, such as ligation or sequencing experiment, it is suggested to stand for 2-5min after adding Buffer PW, and then centrifuge.</LI><LI>Repeat step 4.</LI><LI>Centrifuge at 12,000 rpm for 2min to remove residual Buffer PW. Discard the flow-through, and allow the column to air dry with the cap open for several minutes to dry the membrane.</LI><LI>Place the Spin Column CB1 in a clean 1.5ml microcentrifuge tube. Add 20μl Buffer EB to the center of membrane, incubate for 2min, and centrifuge for 2min at 12,000 rpm</LI></OL></P></DIV></DIV></DIV><DIV id="sponsors"><P><A href="http://igemnyushanghai.tumblr.com/">tumblr</A> | <A href="http://igemnyushanghai.strikingly.com/">strikingly</A> | <A href="https://www.youtube.com/watch?v=hKKjqzkGo3o">igem.nyushanghai@gmail.com</A></P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>