"
Page
Discussion
View source
History
teams
Log in
 
Team:GeorgiaTech/July
From 2011.igem.org
Lab training sessions with our graduate TA, Sun Young Goo. The majority of the students on the team were new and had little to no lab experience. We covered all of the basics-- from simply preparing media and using an autoclave to performing genomic DNA extraction and restriction digests. We covered PCR amplification and learned the basics of primer design. Basic tutorials on bioinformatics software were provided by Kettner and Mitesh, and everyone learned how to use NEBCutter, blastn, and the art of finding the DNA sequences you are looking for on GenBank.
Order Streptococcus thermophilus strain LMD-9 from ATCC, which is predicted to have an active CRISPR locus based on comparative genome sequencing done by Barrangou, et al (See citations).
Learned that LMD-9 is backordered until August or September, and thus we will not be able to use it. We blasted sequences for the other two sequenced S. thermophilus strains, and chose strain LMG 13811 due to its similarity to strain LMD-9 (in respect to the repeat region). Delivery estimate for LMG is three business days.  We worked on designing primers to amplify the CRISPR1/Cas region from the genomic DNA. We also discussed that while S. thermophilus LMG 18311 was predicted to have a CRISPR system, no papers had documented its functionality.
We mini-prepped DNA from E. coli BL21 cells that were provided by Eric Gaucher’s lab. These cells contained a pUC18 plasmid that we would be using as our shuttling vector. The concentration of DNA from the plasmid was 200ng/uL. Using the vector map, we finalized the design of our primers with built in restriction sites to ensure that our DNA could be put in the vector and cloned in another form of E. coli (to avoid possible autoimmune problems).
Still no S. thermophilus LMG 18311. We anticipate it will arrive tomorrow, and prepared 1L of LM17 media (M17 media with a lactose supplement) and 20 LM17 agar plates. We ordered primers today to amplify our CRISPR/cas locus.
We have finally received our bacteria in the mail! They were sent as a lyophilized sample, which we resusitated, streaked on a agar plate, and also inoculated in liquid media (all with the appropriate antibiotics). Primers arrived in the mail and were stored in the -20C freezer.
Advisor meeting today. We discussed the team project description and the safety proposal, which are due next Friday. The bacteria grown on agar were streaked on a plate, and then we performed a genomic DNA extraction on the ones that had grown in liquid media. The DNA was stored in the -20C freezer overnight.
We performed a PCR of the DNA extraction product, using the primers we received on Thursday. We discussed the issue of the unknown functionality of LMG CRISPR, but hoped that we could make a major contribution to science. However, this is a huge risk to take. We decided to press on and test our CRISPR in E. coli.
Today we prepared electrocompetent E. coli for transformation with our DNA + vector. We also performed a PCR clean-up of the DNA.
Performed a digestion of our plasmid backbone and PCR product. We ligated these two components and transformed our E. coli. We should observe growth tomorrow.
Our colonies had no growth. We had a talk with Dr. Gaucher and Dr. Hammer to figure out what to do. We are suspicious of cloning the vector in E. coli, and discussed using other organisms for our model.
The team project description and the safety proposal were posted on our website.
Retrieved from " http://2011.igem.org/Team:GeorgiaTech/July "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
