"
Page
Discussion
View source
History
teams
Log in
 
Team:GeorgiaTech/Citations
From 2011.igem.org
Title
(2011). "CRISPR." Retrieved 5 May 2011, 2011, from http://en.wikipedia.org/wiki/CRISPR.
11, U. P.-S. (2011). "CRISPRs Database." Retrieved 15 August 2011, 2011, from http://crispr.u-psud.fr/crispr/.
Banfield, A. F. A. a. J. F. (May 23, 2008). "Virus Population Dynamics and Acquired Virus Resistance in Natural Microbial Communities." Science 320(5879): 4.
Viruses shape microbial community structure and function by altering the fitness of their hosts and by promoting genetic exchange. The complexity of most natural ecosystems has precluded detailed studies of virus-host interactions. We reconstructed virus and host bacterial and archaeal genome sequences from community genomic data from two natural acidophilic biofilms. Viruses were matched to their hosts by analyzing spacer sequences that occur among clustered regularly interspaced short palindromic repeats (CRISPRs) that are a hallmark of virus resistance. Virus population genomic analyses provided evidence that extensive recombination shuffles sequence motifs sufficiently to evade CRISPR spacers. Only the most recently acquired spacers match coexisting viruses, which suggests that community stability is achieved by rapid but compensatory shifts in host resistance levels and virus population structure.
Barrangou, P. H. a. R. (8 January 2010). "CRISPR/Cas, the Immune System of Bacteria and Archaea." Science 327(5962): 4.
Microbes rely on diverse defense mechanisms that allow them to withstand viral predation and exposure to invading nucleic acid. In many Bacteria and most Archaea, clustered regularly interspaced short palindromic repeats (CRISPR) form peculiar genetic loci, which provide acquired immunity against viruses and plasmids by targeting nucleic acid in a sequence-specific manner. These hypervariable loci take up genetic material from invasive elements and build up inheritable DNA-encoded immunity over time. Conversely, viruses have devised mutational escape strategies that allow them to circumvent the CRISPR/Cas system, albeit at a cost. CRISPR features may be exploited for typing purposes, epidemiological studies, host-virus ecological surveys, building specific immunity against undesirable genetic elements, and enhancing viral resistance in domesticated microbes.
Blake Wiedenheft, G. C. L., Kaihong Zhou, Matthijs M. Jore, Stan J. J. Brouns, John van der Oost, Jennifer A. Doudna & Eva Nogales (September 22, 2011). "Structures of the RNA-guided surveillance complex from a bacterial immune system." Nature 477: 4.
Bacteria and archaea acquire resistance to viruses and plasmids by integrating short fragments of foreign DNA into clustered regularly interspaced short palindromic repeats (CRISPRs). These repetitive loci maintain a genetic record of all prior encounters with foreign transgressors1, 2, 3, 4, 5, 6. CRISPRs are transcribed and the long primary transcript is processed into a library of short CRISPR-derived RNAs (crRNAs) that contain a unique sequence complementary to a foreign nucleic-acid challenger7, 8, 9, 10, 11, 12. In Escherichia coli, crRNAs are incorporated into a multisubunit surveillance complex called Cascade (CRISPR-associated complex for antiviral defence), which is required for protection against bacteriophages13, 14. Here we use cryo-electron microscopy to determine the subnanometre structures of Cascade before and after binding to a target sequence. These structures reveal a sea-horse-shaped architecture in which the crRNA is displayed along a helical arrangement of protein subunits that protect the crRNA from degradation while maintaining its availability for base pairing. Cascade engages invading nucleic acids through high-affinity base-pairing interactions near the 5′ end of the crRNA. Base pairing extends along the crRNA, resulting in a series of short helical segments that trigger a concerted conformational change. This conformational rearrangement may serve as a signal that recruits a trans-acting nuclease (Cas3) for destruction of invading nucleic-acid sequences.
Bolotin A, Q. B., Renault P, Sorokin A, Ehrlich SD, Kulakauskas S,, G. E. Lapidus A, Mazur M, Pusch GD, Fonstein M, Overbeek R, Kyprides N,, et al. (December 2004). "Complete sequence and comparative genome analysis of the dairy
bacterium Streptococcus thermophilus." Nature Biotechnology 22(12): 5.
Brouns SJ, J. M., Lundgren M, Westra ER, Slijkhuis RJ, Snijders AP, Dickman MJ, Makarova KS, Koonin EV, van der Oost J (15 August 2008). "Small CRISPR RNAs guide antiviral defense in prokaryotes." Science 321(5891): 5.
Prokaryotes acquire virus resistance by integrating short fragments of viral nucleic acid into clusters of regularly interspaced short palindromic repeats (CRISPRs). Here we show how virus-derived sequences contained in CRISPRs are used by CRISPR-associated (Cas) proteins from the host to mediate an antiviral response that counteracts infection. After transcription of the CRISPR, a complex of Cas proteins termed Cascade cleaves a CRISPR RNA precursor in each repeat and retains the cleavage products containing the virus-derived sequence. Assisted by the helicase Cas3, these mature CRISPR RNAs then serve as small guide RNAs that enable Cascade to interfere with virus proliferation. Our results demonstrate that the formation of mature guide RNAs by the CRISPR RNA endonuclease subunit of Cascade is a mechanistic requirement for antiviral defense.
Ekaterina Semenovaa, M. M. J., Kirill A. Datsenkoc, Anna Semenovaa, Edze R. Westrab, Barry Wannerc, John van der Oostb, Stan J. J. Brounsb, and Konstantin Severinova (May 16, 2011). "Interference by clustered regularly interspaced short palindromic repeat (CRISPR) RNA is governed by a seed sequence." Proceedings of the National Academy of Science of the United States 108(25): 6.
Elisabeth Grohmann, G. M., and Manuel Espinosa (June 2003). "Conjugative Plasmid Transfer in Gram-Positive Bacteria." MICROBIOLOGY AND MOLECULAR BIOLOGY REVIEWS 67(2): 24.
Ksenia Pougach, E. S., Ekaterina Bogdanova, Kirill A. Datsenko, Marko Djordjevic, Barry L. Wanner, Konstantin Severinov (2010 July 9). "Transcription, processing and function of CRISPR cassettes in Escherichia coli." Molecular Microbiology 77(6): 13.
Philippe Horvath, D. A. R., Anne-Claire Coûté-Monvoisin, Melissa Richards, Hélène Deveau, Sylvain Moineau, Patrick Boyaval, Christophe Fremaux, and Rodolphe Barrangou (2008). "Diversity, Activity, and Evolution of CRISPR Loci in Streptococcus thermophilus." Journal of Bacteriology 190(4): 12.
Clustered regularly interspaced short palindromic repeats (CRISPR) are hypervariable loci widely distributed in prokaryotes that provide acquired immunity against foreign genetic elements. Here, we characterize a novel Streptococcus thermophilus locus, CRISPR3, and experimentally demonstrate its ability to integrate novel spacers in response to bacteriophage. Also, we analyze CRISPR diversity and activity across three distinct CRISPR loci in several S. thermophilus strains. We show that both CRISPR repeats and cas genes are locus specific and functionally coupled. A total of 124 strains were studied, and 109 unique spacer arrangements were observed across the three CRISPR loci. Overall, 3,626 spacers were analyzed, including 2,829 for CRISPR1 (782 unique), 173 for CRISPR2 (16 unique), and 624 for CRISPR3 (154 unique). Sequence analysis of the spacers revealed homology and identity to phage sequences (77%), plasmid sequences (16%), and S. thermophilus chromosomal sequences (7%). Polymorphisms were observed for the CRISPR repeats, CRISPR spacers, cas genes, CRISPR motif, locus architecture, and specific sequence content. Interestingly, CRISPR loci evolved both via polarized addition of novel spacers after exposure to foreign genetic elements and via internal deletion of spacers. We hypothesize that the level of diversity is correlated with relative CRISPR activity and propose that the activity is highest for CRISPR1, followed by CRISPR3, while CRISPR2 may be degenerate. Globally, the dynamic nature of CRISPR loci might prove valuable for typing and comparative analyses of strains and microbial populations. Also, CRISPRs provide critical insights into the relationships between prokaryotes and their environments, notably the coevolution of host and viral genomes.
Philippe Horvatha, A.-C. C.-M., Dennis A. Romerob, Patrick Boyavala, Christophe Fremauxa, Rodolphe Barrangou (30 May 2008). "Comparative analysis of CRISPR loci in lactic acid bacteria genomes." International Journal of Food Microbiology 131(1).
Clustered regularly interspaced short palindromic repeats (CRISPR) are hypervariable loci widely distributed in bacteria and archaea, that provide acquired immunity against foreign genetic elements. Here, we investigate the occurrence of CRISPR loci in the genomes of lactic acid bacteria (LAB), including members of the Firmicutes and Actinobacteria phyla. A total of 102 complete and draft genomes across 11 genera were studied and 66 CRISPR loci were identified in 26 species. We provide a comparative analysis of the CRISPR/cas content and diversity across LAB genera and species for 37 sets of CRISPR loci. We analyzed CRISPR repeats, CRISPR spacers, leader sequences, and cas gene content, sequences and architecture. Interestingly, multiple CRISPR families were identified within Bifidobacterium, Lactobacillus and Streptococcus, and similar CRISPR loci were found in distant organisms. Overall, eight distinct CRISPR families were identified consistently across CRISPR repeats, cas gene content and architecture, and sequences of the universal cas1 gene. Since the clustering of the CRISPR families does not correlate with the classical phylogenetic tree, we hypothesize that CRISPR loci have been subjected to horizontal gene transfer and further evolved independently in select lineages, in part due to selective pressure resulting from phage predation. Globally, we provide additional insights into the origin and evolution of CRISPR loci and discuss their contribution to microbial adaptation.
Rimantas Sapranauskas, G. G., Christophe Fremaux, Rodolphe Barrangou, Philippe Horvath and Virginijus Siksnys (July 8, 2011). "The Streptococcus thermophilus CRISPR/Cas system provides immunity in Escherichia coli." Nucleic Acids Research http://www.ncbi.nlm.nih.gov/pubmed/21813460.
Rodolphe Barrangou, C. F., Hélène Deveau, Melissa Richards, Patrick Boyaval, Sylvain Moineau, Dennis A. Romero and Philippe Horvath (23 March 2007). "CRISPR Provides Acquired Resistance Against Viruses in Prokaryotes." Science 315(5819): 4.
Clustered regularly interspaced short palindromic repeats (CRISPR) are a distinctive feature of the genomes of most Bacteria and Archaea and are thought to be involved in resistance to bacteriophages. We found that, after viral challenge, bacteria integrated new spacers derived from phage genomic sequences. Removal or addition of particular spacers modified the phage-resistance phenotype of the cell. Thus, CRISPR, together with associated cas genes, provided resistance against phages, and resistance specificity is determined by spacer-phage sequence similarity.
Rotem Sorek, V. K. P. H. (March 2008). "CRISPR — a widespread system that provides acquired resistance against phages in bacteria and archaea." Nature Reviews Microbiology: 6.
Arrays of clustered, regularly interspaced short palindromic repeats (CRISPRs) are widespread in the genomes of many bacteria and almost all archaea. These arrays are composed of direct repeats that are separated by similarly sized non-repetitive spacers. CRISPR arrays, together with a group of associated proteins, confer resistance to phages, possibly by an RNA-interference-like mechanism. This Progress discusses the structure and function of this newly recognized antiviral mechanism.
Retrieved from " http://2011.igem.org/Team:GeorgiaTech/Citations "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
