<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SJTU-BioX-Shanghai_Project_Subproject1_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SJTU-BioX-Shanghai/Project/Subproject1/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SJTU-BioX-Shanghai/Project/Subproject1/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/Project/Subproject1/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/Project/Subproject1/Modeling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SJTU-BioX-Shanghai/Project/Subproject1/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SJTU-BioX-Shanghai/Project/Subproject1/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><TABLE border="0" style="margin-left:-3px;"><TBODY><TR><TD width="200px" style="padding: 0 15px 15px 20px; background-color:#eee;vertical-align:top;"><DIV id="navi"><UL id="nav"><LI><A href="/Team:SJTU-BioX-Shanghai"><B>Home</B></A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Project"><B>Project</B></A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Project">Overview</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Project/Subproject1">Rare-Codon Switch</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Project/Subproject1">Overview</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Project/Subproject1-1">Rare tRNA amount</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Project/Subproject1-2">aaRS</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Project/Subproject1-3">Rare codon</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Project/Subproject1/Modeling">Modeling</A></LI></UL></DIV><LI><A href="/Team:SJTU-BioX-Shanghai/Project/Subproject2">Stop-Codon Switch</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Project/Subproject3">Initial-Codon Switch</A></LI><LI><A>Application</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Project/Application">Regulating tools</A></LI><LI style="height:36px"><A href="/Team:SJTU-BioX-Shanghai/Project/Application-3">Modified aaRS Library
</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Parts/Data_Page"><B>Data Page</B></A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Parts"><B>Parts</B></A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Team"><B>Team</B></A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Team">Overview</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Team/Members">Members</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Team/Gallery">Gallery</A></LI><LI><B>Notebook</B></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog">Lablog</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Notebook/Protocol">Protocol</A></LI><LI><B>Consideration</B></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Human_practice">Human Practice </A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Safety">Safety</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Attributions">Attributions</A></LI></TD><TD width="690px" style="padding:0 15px 15px 20px; vertical-align:top;"><H2><SPAN class="mw-headline" id="Modeling:_Dynamic_Modeling_Based_on_Biochemical_Mechanism">Modeling: Dynamic Modeling Based on Biochemical Mechanism</SPAN></H2><H3><SPAN class="mw-headline" id="1._Motivation">1. Motivation</SPAN></H3><P>Mathematical modeling is a useful tool for understanding how a bio-system works and how to improve it. We build a mathematical model mainly based on the mechanism of the involved biochemical reaction. We focus on the dynamic process of &quot;Rare codon Switch&quot; in order to reveal how the system works over time. The model for other Switches can be gained by altering this existing model
</P><H3><SPAN class="mw-headline" id="2._Overview_Description">2. Overview Description</SPAN></H3><P>Four different networks are involved in our project (figure 1). According to the mechanism of biochemical reaction, each of the networks can be divided into two modules: <B>signal sensing module</B> and <B>translation module</B>.
</P><P>Network 1 represents <I>sulA</I> promoter-tRNA<SUP>Arg</SUP> and P<I>bla</I>-Luc-nAGG (n=2, 4, 6, 8) system. This summarizes the modulating stimulated by ultra-violet ray
</P><P>Network 2 represents <I>lacI</I>-Ptrc-tRNA<SUP>Arg</SUP>) and P<I>bla</I>-Luc-nAGG (n=2, 4, 6, 8) system. This summarizes the modulating stimulated by IPTG 
</P><P>Network 3 represents <I>lacI</I>-Ptrc-tRNA<SUP>Arg</SUP> and PT7-Luc-nAGG (n=2, 4, 6, 8) system. This summarizes the modulating stimulated by IPTG. Note: the reporter gene is under different promoter
</P><P>Network 4 represents PT7-TDRS and P<I>bl</I>a-Luc-TAG system.
</P><P>The above are the 4 networks involved in our project. The modules highlighted by yellow box are signal sensing modules; the ones highlighted by purple box are translation modules
</P><H3><SPAN class="mw-headline" id="3._Modeling">3. Modeling </SPAN></H3><H4><SPAN class="mw-headline" id="3.1_The_signal_sensing_module">3.1 The signal sensing module</SPAN></H4><P>The signal sensing module can be easily abstracted as the two following sub-module (figure 2):
</P><P>Figure 2: the sub-modules of signal sensing module
</P><P>The presence of signal (ultraviolet ray exposure or IPTG in our project) causes the repressor protein's releasing from the promoter of intermediate product gene. Then the transcription of intermediate product starts. Intermediate product can be tRNA, aaRS or mRNA. 
</P><P><I>R</I> represents the effective concentration of repressor protein and varies from 0 to 1. <I>I</I> represents the concentration of intermediate product. 
</P><P>In the absence of signal, repressor protein regains.
</P><P>Besides, the degradation and other factors will cause the loss of Intermediate product
</P><P>Considering the mechanism of tRNA aminoacylation, this process can be described by double-substrate Michaelis-Menten equation as formula 5. 
</P><P>Because of the complicated mechanism of amino acid concentration regulation, we assume that the concentration of amino acid is constant.
</P><H4><SPAN class="mw-headline" id="3.2_Translation_module">3.2 Translation module</SPAN></H4><P>The translation part is relatively invariable (figure 3). 
</P><P>Figure 3: the scheme of translation part
</P><P>When the mRNA of product protein and charged tRNA are both available, our product can be synthesized in the ribosome. The process of a single protein synthesis is composed of the sequential addition of amino acid to the peptide and the pre- and post- translation process. These temporally sequential events can be summarized as the following:
</P><P><I>n</I> represents the number of total amino acid in the open reading frame in a single mRNA. <I>t<SUB>aa</SUB></I> represents the time of the addition of one amino acid.
</P><P>The addition of amino acid can be divided to two parts: the addition of amino acid according to engineered codons, the amount of which is variable, and the addition of amino acid according to native codons, the amount of which is constant.
</P><P><I>n<SUB>en</SUB></I> represents the number of engineered codons, <I>n<SUB>native</SUB></I> that of the native codons in a single mRNA.
</P><P>As to a given protein, the pre- and post- translation process is relatively constant. Therefore only the number of engineered codons is variable in the whole process
</P><P>Reciprocal of time is rate. Therefore, the rate of product protein synthesis from a single mRNA is the reciprocal of formula 7. Similarly, the time of amino acid’s addition to the peptide <I>t</I> is the reciprocal of its rate <I>v</I>:</P><P>The rate of amino acid’s addition to the peptide according to a particular codon can be viewed as a first order reaction.
</P><P>Therefore, the rate of protein synthesis from a single mRNA can be expressed as the following formula:
</P><P>Taken the concentration of mRNA into consideration, the overall rate of protein synthesis is:
</P><P>During the synthesis of protein, charged tRNA are converted into uncharged tRNA, which causes the decrease of charged tRNA 
</P><P>Formula 5 and formula 13 together determine the concentration of charged tRNA.
</P><P>Due to the accumulation of metabolic waste and the consumption of resources in the cell, the synthesis of protein will slow down as the bacteria culture reaches stationary phase.
</P><P>Besides, the tandem of rare codons demand that charged tRNA consecutively particulates in the protein synthesis. The more charged tRNA required at one time, the more obvious the resource shortage is: Therefore, the slowing down of protein synthesis is more obvious. So it is reasonable to assume the following relation 
</P><H3><SPAN class="mw-headline" id="4._Simulation_and_Discussion">4. Simulation and Discussion</SPAN></H3><H4><SPAN class="mw-headline" id="4.1_Typical_Dynamic_Process_of_Modulating">4.1 Typical Dynamic Process of Modulating</SPAN></H4><P>The model summarizes a typical dynamic process of Modulating: external signal releasing intermediate product: <I>sulA</I> promoter-tRNA<SUP>ArgW</SUP> ; the net result of synthesis and consumption of charged tRNA corresponding to tandem rare codons; yield of product protein. 
</P><P>The induction begins when t=0 min. 
</P><P>Fig 4: external signal releasing intermediate product. This figure shows the intermediate product of Modulating stimulated by IPTG induction: tRNA<SUP>ArgW</SUP></P><P>Fig 5: the net result of synthesis and consumption of charged rare codon tRNA. This figure shows the amount of charged tRNA over time in <I>lacI</I>-Ptrc-tRNA<SUP>ArgW</SUP> and PT7-luc-4AGG system.
</P><H4><SPAN class="mw-headline" id="4.2_Comparison_of_Different_Promoters_and_Different_Numbers_of_Rare_Codons">4.2 Comparison of Different Promoters and Different Numbers of Rare Codons</SPAN></H4><P>Fig 7: the yield of product protein in LacI-Ptrc-tRNA<SUP>ArgW</SUP> and Pbla-Luc-nAGG(n=2, 4, 6, 8) system
</P><P>Fig 8: the yield of product protein in LacI-Ptrc-tRNA<SUP>ArgW</SUP> and PT7-Luc-nAGG(n=2, 4, 6, 8) system
</P><P>Fig 9: Comparison of final yield of reporter gene with different numbers of AGG codons and under different promoters
</P><H3><SPAN class="mw-headline" id="5._Conclusion">5. Conclusion</SPAN></H3><P>Our model correctly predicts the role that promotor and number of rare codons play in the 5' end of reporter gene: Strong promoter (eg. T7 promoter) leads to higher yield of product. Besides that, the concentration of charged tRNA is lower in the presence of strong promoter due to fast consumption. Furthermore, according to formula 14, low charged tRNA concentration promotes the sensitiveness to the number of rare codons. In short, the reporter gene with strong promoter can achieve a more stable performance, which accords with our experiment data.
</P><P>The number of rare codons can further modulate the synthesis of reporter gene. The more rare codons inserted, the less product yielded. 
</P><P>To see how the experiment data accord with our model, please visit <A href="https://2011.igem.org/Team:SJTU-BioX-Shanghai/Project/Subproject1/Results_2" class="external text" rel="nofollow"><B>Rare-Codon Switch: control by rare codons</B></A>.
</P></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:SJTU-BioX-Shanghai/Project/Subproject1/Modeling">http://2011.igem.org/Team:SJTU-BioX-Shanghai/Project/Subproject1/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SJTU-BioX-Shanghai/Project/Subproject1/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SJTU-BioX-Shanghai/Project/Subproject1/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/Project/Subproject1/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/Project/Subproject1/Modeling&amp;oldid=254701" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>