<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SJTU-BioX-Shanghai_Notebook_Lablog1"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SJTU-BioX-Shanghai/Notebook/Lablog1&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/Notebook/Lablog1&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/Notebook/Lablog1&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SJTU-BioX-Shanghai/Notebook/Lablog1" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SJTU-BioX-Shanghai/Notebook/Lablog1</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><TABLE border="0" style="margin-left:-3px;"><TBODY><TR><TD width="200px" style="padding: 0 15px 15px 20px; background-color:#eee;vertical-align:top;"><DIV id="navi"><UL id="nav"><LI><A href="/Team:SJTU-BioX-Shanghai"><B>Home</B></A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Project"><B>Project</B></A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Project">Overview</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Project/Subproject1">Rare-Codon Switch</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Project/Subproject1">Overview</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Project/Subproject1-1">Rare tRNA amount</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Project/Subproject1-2">aaRS</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Project/Subproject1-3">Rare codon</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Project/Subproject1/Modeling">Modeling</A></LI></UL></DIV><LI><A href="/Team:SJTU-BioX-Shanghai/Project/Subproject2">Stop-Codon Switch</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Project/Subproject3">Initial-Codon Switch</A></LI><LI><A>Application</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Project/Application">Regulating tools</A></LI><LI style="height:36px"><A href="/Team:SJTU-BioX-Shanghai/Project/Application-3">Modified aaRS Library
</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Parts/Data_Page"><B>Data Page</B></A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Parts"><B>Parts</B></A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Team"><B>Team</B></A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Team">Overview</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Team/Members">Members</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Team/Gallery">Gallery</A></LI><LI><B>Notebook</B></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog">Lablog</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Notebook/Protocol">Protocol</A></LI><LI><B>Consideration</B></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Human_practice">Human Practice </A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Safety">Safety</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Attributions">Attributions</A></LI></TD><TD width="690px" style="padding:0 15px 15px 20px;vertical-align:top;"><H3><SPAN class="mw-headline" id="Calendar">Calendar</SPAN></H3><DIV id="tab"><DIV id="cld"><H5>July</H5></DIV></DIV></TD></TR><TR><TH>Sun</TH><TH>Mon</TH><TH>Tue</TH><TH>Wed</TH><TH>Thu</TH><TH>Fri</TH><TH>Sat</TH></TR><TR><TD>1</TD><TD>2</TD></TR><TR><TD>3</TD><TD>4</TD><TD>5</TD><TD>6</TD><TD>7</TD><TD>8</TD><TD>9</TD></TR><TR><TD>10</TD><TD>11</TD><TD>12</TD><TD>13</TD><TD>14</TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog#7.2F15.2F2011">15</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog#7.2F16.2F2011">16</A></TD></TR><TR><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog#7.2F17.2F2011">17</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog#7.2F18.2F2011">18</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog#7.2F19.2F2011">19</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog#7.2F20.2F2011">20</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog#7.2F21.2F2011">21</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog#7.2F22.2F2011">22</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog#7.2F23.2F2011">23</A></TD></TR><TR><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog#7.2F24.2F2011">24</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog#7.2F25.2F2011">25</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog#7.2F26.2F2011">26</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog#7.2F27.2F2011">27</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog#7.2F28.2F2011">28</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog#7.2F29.2F2011">29</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog#7.2F30.2F2011">30</A></TD></TR><TR><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog#7.2F31.2F2011">31</A></TD></TR></TBODY></TABLE></DIV><DIV id="cld"><H5>August</H5><TABLE><TBODY><TR><TH>Sun</TH><TH>Mon</TH><TH>Tue</TH><TH>Wed</TH><TH>Thu</TH><TH>Fri</TH><TH>Sat</TH></TR><TR><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F1.2F2011">1</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F2.2F2011">2</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F3.2F2011">3</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F4.2F2011">4</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F5.2F2011">5</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F6.2F2011">6</A></TD></TR><TR><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F7.2F2011">7</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F8.2F2011">8</A></TD><TD>9</TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F10.2F2011">10</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F11.2F2011">11</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F12.2F2011">12</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F13.2F2011">13</A></TD></TR><TR><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F14.2F2011">14</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F15.2F2011">15</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F16.2F2011">16</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F17.2F2011">17</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F18.2F2011">18</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F19.2F2011">19</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F20.2F2011">20</A></TD></TR><TR><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F21.2F2011">21</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F22.2F2011">22</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F23.2F2011">23</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F24.2F2011">24</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F25.2F2011">25</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F26.2F2011">26</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F27.2F2011">27</A></TD></TR><TR><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F28.2F2011">28</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F29.2F2011">29</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F30.2F2011">30</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog1#8.2F31.2F2011">31</A></TD></TR></TBODY></TABLE></DIV><DIV id="cld"><H5>September</H5><TABLE><TBODY><TR><TH>Sun</TH><TH>Mon</TH><TH>Tue</TH><TH>Wed</TH><TH>Thu</TH><TH>Fri</TH><TH>Sat</TH></TR><TR><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog2#9.2F1.2F2011">1</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog2#9.2F2.2F2011">2</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog2#9.2F3.2F2011">3</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog2#9.2F4.2F2011">4</A></TD></TR><TR><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog2#9.2F5.2F2011">5</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog2#9.2F6.2F2011">6</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog2#9.2F7.2F2011">7</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog2#9.2F8.2F2011">8</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog2#9.2F9.2F2011">9</A></TD><TD>10</TD><TD>11</TD></TR><TR><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog2#9.2F12.2F2011">12</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog2#9.2F13.2F2011">13</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog2#9.2F14.2F2011">14</A></TD><TD>15</TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog2#9.2F16.2F2011">16</A></TD><TD>17</TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog2#9.2F18.2F2011">18</A></TD></TR><TR><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog2#9.2F19.2F2011">19</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog2#9.2F20.2F2011">20</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog2#9.2F21.2F2011">21</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog2#9.2F22.2F2011">22</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog2#9.2F23.2F2011">23</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog2#9.2F24.2F2011">24</A></TD><TD>25</TD></TR><TR><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog2#9.2F26.2F2011">26</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog2#9.2F27.2F2011">27</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog2#9.2F28.2F2011">28</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog2#9.2F29.2F2011">29</A></TD><TD><A href="/Team:SJTU-BioX-Shanghai/Notebook/Lablog2#9.2F30.2F2011">30</A></TD></TR></TBODY></TABLE></DIV></DIV><H2><SPAN class="mw-headline" id="8.2F1.2F2011">8/1/2011</SPAN></H2><P>Phoenie: re-test the strains. I find that the PCR with bacteria culture and plasmid gives different result. PCR with bacteria culture shows that the AGG4 primers can also give positive result and PCR with plasmid doesn't
</P><P>Pony: Extracted pTrc99B from 4 tubes. Only one is succeed. Then, Bear amplified lacI gene+Trc promoter from pTrc99B and inserted the DNA fregment into plasmid pACYC184 by overlapping PCR.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:446px;"><DIV class="thumbcaption"><I>Fig1</I> Amplified (<I>lacI</I>+<I>tac</I> promoter) from pTrc99B</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="8.2F2.2F2011">8/2/2011</SPAN></H2><P>Phoenie: The seller called me and told me that he had vacation in July and today is his first working day. The enzyme will arrive TOMORROW. 
</P><P>next time we must order the enzyme when there is at least 10 mul left. 
</P><P>太坑爹了！
</P><H2><SPAN class="mw-headline" id="8.2F3.2F2011">8/3/2011</SPAN></H2><P>Chunying：Use the optimized primers to amplify the wild luc gene,failed. Try different proportion of the template and the primer(template 2ul,primer 10p,20p,30p) to amplify the gene, the result will come out tomorrow.
</P><P>Zhangbo:
We have changed the reporter gene into RFP and design the new primer for the PCR.
I have a attempt towards the tRNA mutation
</P><P>Nachte: Our problem is that a right fragment with right extending primers just can not be insert into that danmed plasmid.
</P><H2><SPAN class="mw-headline" id="8.2F4.2F2011">8/4/2011</SPAN></H2><P>Chunying:Different proportion of the template and primers did not result in successful amplification of the wild luc gene. The third amplification got the luc gene.
Insert the wild luc gene into the pReporter(replace the amp gene).
Transform the new pReporter plasmid into the cell and spread bacteria on the plate.(total 4 group, PCR at four different annealing temperature-50,53,57,60)
</P><H2><SPAN class="mw-headline" id="8.2F5.2F2011">8/5/2011</SPAN></H2><P>Zhangbo: 
I have done the PCR of the tRNA mutaion and transfer the plasmid into DH5a.
And utilizing the restriction enzyme to cleave the pET28a-amp and the trpRS product.
</P><P>Chunying:The four transformations failed.Amply the wild luc gene again.
</P><H2><SPAN class="mw-headline" id="8.2F6.2F2011">8/6/2011</SPAN></H2><P>Zhangbo
construction of the tRNA with the promoter and the terminator, I use the PCR to get this parts.
</P><PRE>PCR KOD buf             5ul
dNTP                         5ul
Mg2+                        3ul
F/R(WtRNA-1)            2/2ul
Plasmid(pBRIVTC3.4)  1ul
KOD                          1ul
ddH2O                      31ul 
</PRE><P>Program:
</P><PRE>94 5min 
94 30s 55 30s 68 30s 31cycle
68 7min 
</PRE><P>Purify the DNA by TianGen Solution. 
</P><P>Overlapping PCR
</P><PRE>DNA(above)      10ul
pACYC184         5ul
PCR KOD buf     3ul
dNTP                 3ul
Mg2+             1.8ul
KOD                 1ul
ddH2O            6.2ul
</PRE><P>Program:
</P><PRE>94 5min 
94 30s 55 30s 68 4min30s 31 cycle
68 7min
</PRE><P>Instruction of the trpRS by Nco1 and Nde1
</P><PRE>PCR KOD buf        5ul
dNTP                    5ul
Mg2+                   3ul
F/R(trpRS)          2/2ul
Plasmid(pRS1.1)     1ul
KOD                      1ul
ddH2O                 31ul 
</PRE><P>Purified the PCR product by Tiangen PCR purification Kit
</P><P>Restriction enzyme
</P><PRE>Buf    2ul
DNA   16ul(plasmid)
Nde1   1ul
Nco1   1ul
Total    20ul
</PRE><P>Deal with the PCR result above with the DpnI
</P><PRE>1ul enzyme(Fast Digest) into 50ul PCR result for 2 hours
Transfer the plasmid into the DH5a
10ul into 100ul DH5a
30min on the ice
37℃ 2min and 5min on the ice
220rpm 1 hour 
Tet solid medium for 16 hours to 24 hours
</PRE><P>Bind the pET28a and PCR product
</P><PRE>PEG4000       1.5ul
Buf               1.5ul
Plasmid           3ul
Pcr product     6ul
Lingase            1ul
ddH2O            2ul
Total              15ul
</PRE><P>Program   22℃ 2 hours 72℃ 10min 
</P><P>Transfer the product into the Dh5a 
</P><PRE>10ul into 100ul DH5a
30min on the ice
37℃ 2min and 5min on the ice
220rpm 1 hour 
Tet solid medium for 16 hours to 24 hours
</PRE><P>Bind the pET28a and PCR product
</P><H2><SPAN class="mw-headline" id="8.2F7.2F2011">8/7/2011</SPAN></H2><P>Phoenie: Last night the plate which we spreat bacteria on have colonies on it but this morning the postive test showed that none of them contains the pET28a-luc plasmid. 
</P><P>Parrot suggests that we can change the amount of plasmid and PCR product in each insertion reaction. And we re-cloned the luciferase with both sets of homogenous arms and performed a series of reaction with gradient plasmid and PCR product concertration and purified, transformed the insertion product.
</P><P>We also cultured the strain which was co-transformed with the pSwitch-arg and pReporter-AGG6 last night and the positive strains with pReporter-TAG
</P><P>BTW, the internet is so poor in the office today, I can hardly open any website except the homepage of Baidu.
</P><PRE>             __....__           
        .-~~/  \__/  \~~-.     If only the web can be a little faster~   
       /_/``\__/  \__/``\_\.--.  /           /
      /  \__/  \__/  \__/  \   o`.          /   
  `==/\__/__\__/__\__/__\__/\`'--' _______ /   
     ~/__/__/^^^^^^^^\__\__\~     |_______|
</PRE><P>8.7
Zhangbo:
</P><PRE>PCR testing on the Plasmid(WtRNA)(purified by tiangen Kit)
PCR buf       2ul
dNTP         2ul
Mg2+        1.2ul
F/R(WtRNA-1) 0.8/0.8ul
Plasmid(result)   0.8ul
KOD          0.4ul
ddH2O        12ul 
</PRE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:327px;"><DIV class="thumbcaption">figure1:PCR testing on the Plasmid(WtRNA)</DIV></DIV></DIV></DIV><PRE>PCR testing on the Plasmid(trpRS-lingase)(purified by tiangen Kit)
PCR buf       2ul
dNTP         2ul
Mg2+        1.2ul
F/R(WtRNA-1) 0.8/0.8ul
Plasmid(result)   0.8ul
KOD          0.4ul
ddH2O        12ul
</PRE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:261px;"><DIV class="thumbcaption">figure1:PCR testing on the trpRS</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="8.2F8.2F2011">8/8/2011</SPAN></H2><P>Phoenie: the socond co-transform is success and begin to insert by two step method
8.8 
Zhangbo
</P><P>I choose the WTM-2 and the WRS-1,2 for the sequencing by JieLi corp.
</P><P>Nachte: Another cycle of cloning,we transformed ArgW-pTrc99b.
</P><H2><SPAN class="mw-headline" id="8.2F9.2F2011">8/9/2011</SPAN></H2><H2><SPAN class="mw-headline" id="8.2F10.2F2011">8/10/2011</SPAN></H2><P>Chunying：Divide the wild luciferase into two parts then amplify and insert them respectively. Today, inserted the latter part of the luc gene into the plasmid(pET28a-Amp) and transformed the new plasmid into Dh5a.
</P><P>In case that the luciferase will not be the best choice for our reporter gene, we use beta-lactamase as the other candidate for the reporter gene( because of its easier conduction and cheaper detection). Today we use the primer(R&amp;L 004/005) to introduce mutation into the beta-lactamase gene (in its signal peptide)on the pET28a-Amp and transformed the new plasmid into Dh5a and spread bacterial on the Kana plate.
</P><P>Zhangbo:
In this morning, the result of the sequencing of the samples has arrived by E-mail.
</P><P>The result of the tRNA lose a base in a very important part which may lead to the  failure of the expression of  the tRNA I want. So i check my primer and try for another time in this week.
</P><P>Nachte: Vaccinated several clonies of ArgW-pTrc99b.
</P><H2><SPAN class="mw-headline" id="8.2F11.2F2011">8/11/2011</SPAN></H2><P>Chunying: The bacterial that has half of the wild luciferase gene has grown on the plate. Negative screening for the stains.
</P><P>ZHangbo:
</P><PRE>8.11 
</PRE><P>Instruction of the tRNA with the promoter and the terminator, I use the PCR to get this parts.
</P><PRE>PCR KOD buf            5ul
dNTP                        5ul
Mg2+                      3ul
F/R(WtRNA-1)          2/2ul
Plasmid(pBRIVTC3.4) 1ul
KOD                         1ul
ddH2O                     31ul 
Program:
94 5min 
94 30s 55 30s 68 30s 31cycle
68 7min
</PRE><P>Purify the DNA by TianGen Solution. 
</P><PRE>Overlapping PCR
DNA(above) 10ul
pACYC184  5ul
PCR KOD buf  3ul
dNTP        3ul
Mg2+       1.8ul
KOD         1ul
ddH2O      6.2ul
Program:
94 5min 
94 30s 56 30s 68 4min30s 31 cycle
68 7min
</PRE><P>Nachte:Extracted plasmid ,verificating by PCR,and sending for sequencing.
</P><P>Alfred:Design the primers needed:
</P><P>For T7 promoter plus terminate containing MCS
</P><PRE>T7-F:5'-GCGGTGCGGACTGTTGTATTAATACGACTCACTATAGGGG-3'
T7-R:5'-GATCGTGCTCCTGTCGTTGCCGGAATTCCAAAAAACCCCTCAAGAC-3'
</PRE><P>For truncated AspRS (without N-terminal anti-codon binding domain)
</P><PRE>TDRS-F: 5’-CATGCCATGGATGTTCTGCCGCTTGACTCT-3’
</PRE><P>For AspV gene(tRNAAsp)
</P><PRE>AspV-F: 5’-CCACCATACCCACGCCGAAACCTAAAAATAGCGACTTGGGC-3’ 
AspV-R: 5’-TTGCCGTTACGCACCACCCTTGAACAGAGAAATGGTGGAA-3’
</PRE><P>For mutation of AspV gene (GTC→CCT)
</P><PRE>AspV-M-F: 5’- AGTCGGTTAGAATACCTGCCTCCTACGCAGGGGGTCGC -3’ 
AspV-M-R: 5’- GCGACCCCCTGCGTAGGAGGCAGGTATTCTAACCGACT -3’
</PRE><H2><SPAN class="mw-headline" id="8.2F12.2F2011">8/12/2011</SPAN></H2><P>Phoenie: The colony PCR shows that we may get the plasmid with half of the luciferase gene on it. we culture the colony in liquid medium.
</P><P>However other method turned out unsuccessful.
</P><P>Zhangbo:8.12
</P><P>The result of the WRS by T7 promoter is not correct compared with the trpRS in my database. 
Instruction of the trpRS by Nco1 and Nde1
</P><PRE>PCR KOD buf  5ul
dNTP        5ul
Mg2+        3ul
F/R(trpRS)    2/2ul
Plasmid(pRS1.1) 1ul
KOD           1ul
ddH2O       31ul 
</PRE><P>Purified the PCR product by Tiangen PCR purification Kit
</P><PRE>Restriction enzyme
Buf    2ul
DNA   16ul(plasmid)
Nde1   1ul
Nco1   1ul
Total    20ul
</PRE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:304px;"><DIV class="thumbcaption">figure1: The plasmid and the enzyme</DIV></DIV></DIV></DIV><P>Deal with the PCR result above with the DpnI
</P><PRE>1ul enzyme(Fast Digest) into 50ul PCR result for 2 hours
Transfer the plasmid into the DH5a
10ul into 100ul DH5a
30min on the ice
37℃ 2min and 5min on the ice
220rpm 1 hour 
Tet solid medium for 16 hours to 24 hours
</PRE><P>Nachte:I went to the Kendo training,so it's midnight when I heard that Pony didn't gain GFP and RFP fragments with my primers.According to 8.13 is my birthday , burning midnight oil in the lab should be a awfully good idea of celebrating.
</P><H2><SPAN class="mw-headline" id="8.2F13.2F2011">8/13/2011</SPAN></H2><P>Phoenie: We extract the pET28a-halfluc and insert the front part of wt luc and 4AGG luc into the plasmid.
</P><P>Zhangbo:
8.13
PCR testing on the Plasmid(WtRNA)(purified by tiangen Kit)
</P><PRE>PCR buf       2ul
dNTP         2ul
Mg2+        1.2ul
F/R(WtRNA-1) 0.8/0.8ul
Plasmid(result)   0.8ul
KOD          0.4ul
ddH2O        12ul 
</PRE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:346px;"><DIV class="thumbcaption">figure1: result of the test of the WTM plasmid</DIV></DIV></DIV></DIV><P>I choose both for sequencing.
</P><P>Bind the pET28a and PCR product
</P><PRE>PEG4000 1.5ul
Buf       1.5ul
Plasmid   3ul
Pcr product 6ul
Lingase    1ul
ddH2O     2ul
Total       15ul
Program   22℃ 2 hours 72℃ 10min 
Transfer the product into the Dh5a 
10ul into 100ul DH5a
30min on the ice
37℃ 2min and 5min on the ice
220rpm 1 hour 
Tet solid medium for 16 hours to 24 hours
Bind the pET28a and PCR product
</PRE><P>Nachte：My spectulation is right~
</P><P>Only a little change of the PCR program brought our target to me= 3=
</P><P>Well,frankly,my dear,it's my fault to create such a pair of abnormal temperature requiring primers.
</P><P>Alfred:Amplify T7 promoter plus terminate containing MCS from pET-28a plasmid.
</P><PRE>Reaction mixture:
10× PCR buffer for taq	 5μl
2.5mM dNTP Mixture	 5μl
25mM MgCl2	         3μl
TaKaRa Taq		 0.5μl
T7-F+T7-R	         2+2μl
pET-28a plasmid	 2μl
ddH2O	                 up to 50μl
</PRE><P>After extract the PCR product, I introduced the T7 promoter plus terminate containing MCS into pACYC-184 plasmid (overlapping PCR method).
</P><PRE>Reaction mixture:
10× PCR buffer for KOD	 3μl
2mM dNTP Mixture	 3μl
25mM MgSO4	         1.8μl
KOD - Plus	         0.3μl
T7 segment	         5μl
pACYC-184 plasmid	 5μl
ddH2O	                 up to 30μl
</PRE><P>After overlapping PCR, digest the PCR product with <I>DpnI</I> enzyme for eliminate the template plasmid. Subsequently, transform it into DH5α competent cell.
</P><H2><SPAN class="mw-headline" id="8.2F14.2F2011">8/14/2011</SPAN></H2><P>Phoenie: the result of positive test PCR is quite pulsing. It seems that the front part of the luciferase gene has not been inserted into the plasmid pET28a-halfluc right. So several other colonies are picked and cultured for further test in hope of getting the right strains
</P><P>Nachte:God-Wang send me a pair of ultra homologous and he guaranteed that they work well.I hope so@(ω)@
</P><P>Do a new cycle of construction.
</P><P>Alfred:Choose eight colonies from yesterday’s plant and incubate them in LB liquid medium.
3 hours later, conduct a PCR test for validating whether the T7 promoter plus terminate containing MCS is introduced into pACYC-184 plasmid.
</P><PRE>Reaction mixture:
10× PCR buffer for taq	 2μl
2.5mM dNTP Mixture	 2μl
25mM MgCl	         1.2μl
TaKaRa Taq	         0.3μl
T7-F+T7-R	         1+1μl
colony	                 1μl
ddH2O	                 up to 30μl
</PRE><P>I found that all the colony is correct, so I choose one colony to incubate amplifiedly and extract plasmid, named as pACYC-T7.
</P><P>Amplify AspV gene from E.coli genome.
</P><PRE>Reaction mixture:
10× PCR buffer for taq	 5μl
2.5mM dNTP Mixture	 5μl
25mM MgCl2	         3μl
TaKaRa Taq	         0.5μl
AspV-F+ AspV-R	         2+2μl
E.coli genome	         2μl
ddH2O	                 up to 50μl
</PRE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:72px;"><DIV class="thumbcaption">wall1:AspV wall2:marker</DIV></DIV></DIV></DIV><P>Introduce AspV gene into pACYC-T7 through overlapping PCR. 
</P><P>After overlapping PCR, digest the PCR product with DpnI enzyme for eliminate the template plasmid. Subsequently, transform it into DH5α competent cell.
</P><H2><SPAN class="mw-headline" id="8.2F15.2F2011">8/15/2011</SPAN></H2><P>Phoenie: Send pET28a-halfluc and several other stains to the company for sequencing. extract the plasmid form the strain that may contain pReporter-wtluc pReporter-4AGG wtluc.
</P><P>The following figure shows the PCR test result.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:438px;"><DIV class="thumbcaption">fig1: pReporter-wtluc pReporter-4AGG 's plasmids</DIV></DIV></DIV></DIV><P>Zhangbo:
8.14 / 8.15
</P><PRE>PCR testing on the Plasmid(trpRS-lingase)(purified by tiangen Kit)
PCR buf       2ul
dNTP         2ul
Mg2+        1.2ul
F/R(WtRNA-1) 0.8/0.8ul
Plasmid(result)   0.8ul
KOD          0.4ul
ddH2O        12ul
</PRE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:309px;"><DIV class="thumbcaption">figure2:The result of the test</DIV></DIV></DIV></DIV><P>I choose the WRS-3,4 for sequencing.
</P><P>Nachte:YOOOOOO!The god's creator doesn't work as well!
</P><P>Alfred:There were several colonies on the yesterday’s palnt. I incubated them in LB liquid medium and extract plasmid named as pACYC-T7-AspV. Then I did the PCR to validate its correctness.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:204px;"><DIV class="thumbcaption">wall1:marker    wall2:colony 1 wall3:colony 2</DIV></DIV></DIV></DIV><P>Amplify the truncated AspRS for E.coli genome with the primer TDRS-F+ TDRS-R. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:289px;"><DIV class="thumbcaption">wall1:PCR product 1     wall2:PCR product 1 wall3:marker</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="8.2F16.2F2011">8/16/2011</SPAN></H2><P>Phoenie: Today we test the luciferase activity of E.coli strain ER2566 which contains pSwitch and pReporter to test the argW system and suppressor system. 
</P><P>When the OD600 of the bacteria culture reaches 0.8, we expose the experiment groups under UV light in the sterilized petri dish for 20s.(the power of the UV lamp is 20W; distance between the lamp and petri dish is around 35cm). Then the exposed culture liquid is collected and cultured for another 2 hours. Then culture is centrifuged by 8000 r/min for 3min and resuspended with 2ml 0.85% sterilized pre-cooled NaCl solution. The OD600 of each suspension is measured 
</P><P>The bacteria is ultrasonic disintegrated for 2 min, with 3s pulse for every 3s work. The sonicated bacterial cells is centrifuged by 10000 r/min for 1min to get rid of the cell debris. 
Then we measure the luciferase activity in each sample (mix 100 mul supernatant with 100 mul luciferin substrates).
</P><P>The result is somehow confusing 
</P><PRE>group                     luciferase activity/OD066
AGG6 experiment group     ~11,000,000/3
AGG6 control group           ~11,000,000/3
TAG2 experiment group             ~2,000/3
TAG2 control group                    ~1,000/3
</PRE><P>Phoenie: the sequencing result shows that the AGG4 mutation has been constructed correctly.
</P><P>ZHangbo:
8.16 
In the result of the WTM-1234, i find that the base is still missing and i check the sequence of the tRNA in my database which lead to a sad truth that the the missing is caused by myself so the construction of the tRNA successes.
</P><P>Nachte:Start to build pReporter~No one has experience with linker design，so we have to begin with a terrible high annealing temperature-72℃.
</P><P>The PCR result is horrorable,there are so many mispriming bands that Gel Extraction should be the best choice.
</P><H2><SPAN class="mw-headline" id="8.2F17.2F2011">8/17/2011</SPAN></H2><P>Phoenie: Today we test the luciferase activity of the DH5a stains which contains only pReporter-6AGG and pReporter-4AGG respectively. We culture the bacteria until the OD600 is around 0.8 and directly measure its luciferase activity, without UV exposure and additional 2-hour culture. The result shows that the relative enzyme activity of stains which contains only pReporter-6AGG or pReporter-4AGG is lower than those that contains both pSwitch and pReporter. The strain containing 4AGG has higher activity than that containing 6AGG.
</P><P>ZHangbo:
8.17 
From the work result and the discussion of 8.17, the program of mine changed and find out several mistakes in my program into the instruction of the T7 promoter from the pET28a to the pACYC184 plasmid by the overlapping PCR. 
Redesign the primer WRS-2 for the mistake on the Nco1 restriction site which has a ATG within. 
Design the T7-1 primer.
</P><P>Nachte:Holly shit, the part of fusion protein can't be inserted into the vector too,regarding which our plan of overlapping PCR may be not a good way.
</P><P>Alfred:The mutation of pACYC-T7-AspV was failed. Try again with yesterday’s protocol.
</P><P>In the contrast, the linkage of TDRS segment and pET-28a was successful. Do the colony PCRs using T7-F and T7-R.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:649px;"><DIV class="thumbcaption">wall1: marker wall2-17:Colony PCR</DIV></DIV></DIV></DIV><P>Finally, extract the plasmid named pET-TDRS.
</P><H2><SPAN class="mw-headline" id="8.2F18.2F2011">8/18/2011</SPAN></H2><P>Phoenie: We test the co-transfered colonies by colony PCR. We use the strains without pSwitch-ArgW as negative control. All the five colony we pick give positive result. (The pSwitch-ArgW negative control shows a band that is much weaker than the tested samples.)
</P><P>Zhangbo:
8.18
Overlapping 
</P><PRE>PCR KOD buf  5ul
dNTP        5ul
Mg2+        3ul
F/R(T7-1)    2/2ul
Plasmid(pET28a) 1ul
KOD           1ul
ddH2O       31ul 
Program:
94 5min 
94 30s 55 30s 68 30s 31cycle
68 7min
</PRE><PRE>DNA(above) 10ul
Plasmid(WTM-9)  5ul
PCR KOD buf  3ul
dNTP        3ul
Mg2+       1.8ul
KOD         1ul
ddH2O      6.2ul
Program:
94 5min 
94 30s 55 30s 68 4min30s 31 cycle
68 7min
</PRE><P>Nachte:In this week's gruop meeting ,we made a difficut decision-change to digestion,every step!I spend the whole afternoon to create the new primers,as the sites of EcoRI and XholI would be added to the 5'end.At the midnight，thanX to my bored feeling,I made the last attempt to get pTEnd by overlapping PCR,with a new program:
</P><P>65 30s 68 5min*10cycles
</P><P>60 30s 68 5min*10cycles
</P><P>55 30s 68 5min*10cycles
</P><P>Finally,I got it~
</P><P>Alfred:There were so many colonies of pACYC-T7-AspV mutation on the plant, so that I suspected the <I>DpnI</I> did not work. So I conducted 16 colony PCRs, and finally extracted eight tubes of plasmid to sequence.
</P><P>The align result was followed.
</P><P>So I confirmed the mutation of pACYC-T7-AspV was done, which named pACYC-T7-AspV(CCT).
</P><H2><SPAN class="mw-headline" id="8.2F19.2F2011">8/19/2011</SPAN></H2><P>We test different strains under the following conditions:
</P><PRE>co-transformed 4 AGG codon exposed by UV; 
co-transformed 4 AGG codon non-UV;
only 4 AGG codon pReporter exposed by UV;
only 4 AGG codon pReporter non-UV;
co-transformed 6 AGG codon exposed by UV;
co-transformed 6 AGG codon non-UV;
only 6 AGG codon pReporter exposed by UV; 
only 6 AGG codon pReporter non-UV;
</PRE><P>We take samples after UV exposure every half an hour and measure the OD600 of the 5ml bacteria culture, OD600 of the 2ml suspension NaCl solution, DNA concentration of lysis solution, RLU after mix 100 mul solution and 100mul luciferin substrate.
</P><P>Zhangbo:
8.19
Transfer the product into the Dh5a 
</P><PRE>10ul into 100ul DH5a
30min on the ice
37℃ 2min and 5min on the ice
220rpm 1 hour 
Tet solid medium for 16 hours to 24 hours
Bind the pET28a and PCR product
</PRE><P>Nachte:We are always too simple to be a qualified reseacher!
I
t's easy to know what is digestion from the text book but setting a digestion primer is not a piece of cake.Nearly every new primer should be rewrited%&gt;&lt;%
</P><P>Alfred:Design primers:
</P><PRE>RFP - 1F: 5’- GGAATTCCATATGGCTTCCTCCGAAGACG-3’
RFP - 1R: 5’- CCGCTCGAGTTAAGCACCGGTGGAGTGAC-3’
RFP-2F:
 5’- GGAATTCCATATGAGGAGGAGGAGGAGGAGGGCTTCCTCCGAAGACG-3’
</PRE><P>Amplify the RFP gene and RFP mutation(AGG) from the plasmid on the 2011 Kit Plant (use RFP-1F+ RFP-1R and RFP-2F+ RFP-1R). Then do the PCR product purification.
</P><P>Digest the RFP segment and RFP mutation(AGG) with NdeI and XhoI at the 37℃ for 5 hours. So did the pACYC-T7-AspV(CCT), then linked the RFP segment and RFP mutation(AGG) and pET-28a with T4 ligase. Subsequently, transform it into DH5α competent cell.
</P><H2><SPAN class="mw-headline" id="8.2F20.2F2011">8/20/2011</SPAN></H2><P>We discuss the result of the experiment and prepare the medium and other equipments.
</P><P>Zhangbo:
</P><P>8.20
8.19 failed.
The result of the WRS by T7 promoter is not correct compared with the trpRS in my database. 
Instruction of the trpRS by Nco1 and Nde1
</P><PRE>PCR KOD buf  5ul
dNTP        5ul
Mg2+        3ul
F/R(WRS-2)    2/2ul
Plasmid(pRS1.1) 1ul
KOD           1ul
ddH2O       31ul 
</PRE><P>Purified the PCR product by Tiangen PCR purification Kit
Restriction enzyme
</P><PRE>Buf    2ul
DNA   16ul(plasmid)
Nde1   1ul
Nco1   1ul
Total    20ul
</PRE><P>Nachte:The sequencing result of pTEnd produced(using primers of our own),not untill then I recognized that I had left the RBS before the ArgW sequence.It's hard to say whether that bug will influence the tRNA-Arg(AGG) expression...BTW,all depend on the contransformation result of pTEnd and team-1's reporter.
</P><P>Alfred:Do the colony PCRs for what I transformed yesterday (RFP:6 colonies, RFP-mutation:6 colonies).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:445px;"><DIV class="thumbcaption">wall1:marker wall2-7:Colony PCR: RFP wall8-13:Colony PCR: RFP-mutation</DIV></DIV></DIV></DIV><P>Subsequently, I extracted the plasmids for the positive result. These two plasmid were titled pACYC-RFP-AspV(CCT) and pACYC-RFP(AGG)-AspV(CCT).
</P><P>All the plamids needed, including pET-TDRS, pACYC-RFP-AspV(CCT) and pACYC-RFP(AGG)-AspV(CCT), were sequenced by invotrogen.
</P><H2><SPAN class="mw-headline" id="8.2F21.2F2011">8/21/2011</SPAN></H2><P>We optimize the experiment protocal and test the background RLU by measure the strain with only 4AGG pReporter and co-transformed 4AGG UV system.
</P><P>Zhangbo
8.21 binding the product together
Bind the pET28a and PCR product
</P><PRE>PEG4000 1.5ul
Buf       1.5ul
Plasmid   3ul
Pcr product 6ul
Lingase    1ul
ddH2O     2ul
Total       15ul
Program   22℃ 2 hours 72℃ 10min 
</PRE><P>Transfer the product into the Dh5a 
</P><PRE>10ul into 100ul DH5a
30min on the ice
37℃ 2min and 5min on the ice
220rpm 1 hour 
Tet solid medium for 16 hours to 24 hours
Bind the pET28a and PCR product
</PRE><P>Alfred:Transform pACYC-RFP-AspV(CCT) into ER2566 competent cell.
</P><P>Transform pACYC-RFP(AGG)-AspV(CCT) into ER2566 competent cell.
</P><P>Co-transform pACYC-RFP(AGG)-AspV(CCT) and pET-TDRS  into ER2566 competent cell.
</P><H2><SPAN class="mw-headline" id="8.2F22.2F2011">8/22/2011</SPAN></H2><P>Zhangbo:
</P><P>8.22
8.21Failed.
WtRNA introduction to pACYC184-T7
</P><P>Overlapping 
</P><PRE>PCR KOD buf  5ul
dNTP        5ul
Mg2+        3ul
F/R(WtRNA)    2/2ul
Plasmid(WTM-9) 1ul
KOD           1ul
ddH2O       31ul 
Program:
94 5min 
94 30s 55 30s 68 30s 31cycle
68 7min
</PRE><PRE>DNA(above)            10ul
Plasmid(pACYC184-T7)  5ul
PCR KOD buf  3ul
dNTP        3ul
Mg2+       1.8ul
KOD         1ul
ddH2O      6.2ul
Program:
94 5min 
94 30s 55 30s 68 4min30s 31 cycle
68 7min
Deal with the PCR result above with the DpnI
1ul enzyme(Fast Digest) into 50ul PCR result for 2 hours
Transfer the plasmid into the DH5a
10ul into 100ul DH5a
30min on the ice
37℃ 2min and 5min on the ice
220rpm 1 hour 
Tet solid medium for 16 hours to 24 hours
</PRE><P>Nachte:New program!Referencing to the team-1's findings,besides changing of pReporter(AGG)'s promoter,we made another linker include TAG,which must be very strict as an obstacle.
</P><P>Alfred:Choose three colonies for each to incubate in 5ml LB liquid medium and induce expression with IPTG (1mM).
24 hours later, centrifugate the bacterial suspension and get the deposition. Observe under nature light and UV.
</P><H2><SPAN class="mw-headline" id="8.2F23.2F2011">8/23/2011</SPAN></H2><P>Zhangbo:
8.23 
</P><PRE>PCR testing on the Plasmid(WtRNA)(purified by tiangen Kit)
PCR buf       2ul
dNTP         2ul
Mg2+        1.2ul
F/R(WtRNA-1) 0.8/0.8ul
Plasmid(result)   0.8ul
KOD          0.4ul
ddH2O        12ul 
</PRE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:413px;"><DIV class="thumbcaption">figure:The result of the test of the plasmid WtRNA mutation</DIV></DIV></DIV></DIV><P>Sequencing
</P><P>Nachte:Ordering primers for TAG and AGG-digestion.
</P><H2><SPAN class="mw-headline" id="8.2F24.2F2011">8/24/2011</SPAN></H2><P>We test the the lac system (with 4 AGG codon and 6 AGG codon respectively)for the fist time by compare the differences between bacteria with only pReporter and both pSwitch and pReporter under stimulated condition. The result is that the OD600 of the culture stop rising very early when it is around o.6 for bacteria with pReporter and around 0.9 for co-tranformed bacteria
</P><H2><SPAN class="mw-headline" id="8.2F25.2F2011">8/25/2011</SPAN></H2><P>We test the UV system by testing:
</P><PRE>the co-transformed 4AGG UV system   UV+
only 4AGG pReporter                 UV+
the co-transformed 4AGG UV system   UV-
</PRE><P>Taking sample:
immediately before UV exposure,0.5h 1h 1h20min 1h45min 2h15min 3h 4h 5h after UV exposure
</P><P>Nachte:The new primers for TAG have arrived,so we did a new cycle of linking GFP with RFP.From the process of gel extraction we found that linker-TAG works much better under a fixed temperature between the two Tm of primer-GFP-F and primer-RFP-R in which there is a big gap.
</P><H2><SPAN class="mw-headline" id="8.2F26.2F2011">8/26/2011</SPAN></H2><P>Zhangbo:
</P><P>8.26 
I find out that the DH5a has been polluted by a plasmid which caused the poor result of the link. Try another   Dh5a and try again.
</P><PRE>Instruction of the trpRS by Nco1 and Nde1
PCR KOD buf  5ul
dNTP        5ul
Mg2+        3ul
F/R(WRS-2)    2/2ul
Plasmid(pRS1.1) 1ul
KOD           1ul
ddH2O       31ul 
</PRE><PRE>Purified the PCR product by Tiangen PCR purification Kit
Restriction enzyme
Buf    2ul
DNA   16ul(plasmid)
Nde1   1ul
Nco1   1ul
Total    20ul
</PRE><P>Nachte：A big wave of primers arrived!As it's hard to insert such a long fragment into the vector，both overlapping extension and digestion would be taking in acount,and there are the primer couples：
</P><P>GFPoverR(3*AGG）+GFPoverF
</P><P>GFPdigR（TAG）+GFPoverF
</P><P>GFPoverR(3*AGG）+GFPdigF
</P><P>GFPdigR（TAG）+GFPdigF
</P><P>RFP(3*AGG)+RFPdigR
</P><P>RFP(TAG)+RFPdigR
</P><P>
All these GFP and RFP parts for reporter must be ready tonight,after which we can do a new kind of connection：
</P><P>GFP part           3ul
</P><P>RFP part           3ul
</P><P>primers             3/3ul
</P><P>dNTPs               5ul
</P><P>10*buffer          5ul
</P><P>Mg                   3ul
</P><P>Taq                   1ul
</P><P>ddH2O              24ul
</P><P>
Gel extraction
</P><H2><SPAN class="mw-headline" id="8.2F27.2F2011">8/27/2011</SPAN></H2><P>Phoenie: We test the co-transformed bacteria with UV system and lac system by comparing the different luciferase activity under the stimulated and non-stimulated condition. The result is confusing as usual.
</P><P>Zhangbo；
8.27
</P><PRE>Binding the product together
Transfer the plasmid into the DH5a
10ul into 100ul DH5a
30min on the ice
37℃ 2min and 5min on the ice
220rpm 1 hour 
Tet solid medium for 16 hours to 24 hours
</PRE><P>Nachte:Using the GFP and RFP made yesterday we got two different reporter parts which have digestion sites and homologous sequence,but it seem that primers didn't work well.False priming lead to a lot of bands in various length.
To solve this problem,firstly we choose the overlapping parts who performed better as the template,and then use GFPdig-F and RFPdig-R to amplify part with digestion sites.
</P><P>fragment                  1ul
</P><P>primers                     1/1ul
</P><P>dNTPs                       5ul
</P><P>10*buffer                  5ul
</P><P>Mg                           3ul
</P><P>Taq                           1ul
</P><P>ddH2O                     33ul
</P><P>Finally,we saw clear bands in electrophoresis.
</P><H2><SPAN class="mw-headline" id="8.2F28.2F2011">8/28/2011</SPAN></H2><P>Phoenie:We discuss the possibilities that caused the abnormal data and verify our surmise by a group of preliminary trials to exame the following facts: different batches of reagents, the use of re-suspension solution and freezing. the result shows that reagents that once store at 4 ℃ can greatly effect the read-out but other facts make little different. Besides that we accidentally find out that the DNA concentration read-out can not give efficient information about the lysis of bacteria because the re-suspension solution and lysis solution give similar read-out
</P><P>Zhangbo8.28 
Testing 
</P><PRE>PCR buf       2ul
dNTP         2ul
Mg2+        1.2ul
F/R(WtRNA-1) 0.8/0.8ul
Plasmid(result)   0.8ul
KOD          0.4ul
ddH2O        12ul 
</PRE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:316px;"><DIV class="thumbcaption">figure1:Result of the Plasmid test</DIV></DIV></DIV></DIV><P>Sequencing
</P><H2><SPAN class="mw-headline" id="8.2F29.2F2011">8/29/2011</SPAN></H2><P>test the 4AGG codon UV system again, with new batch of reagent:
</P><PRE>co-transformed 4AGG codon UV  UV system, UV+;
co-transformed 4AGG codon UV system, UV-;
only 4AGG codon UV pReporter, UV+;
only 4AGG codon UV pReporter, UV-;
</PRE><P>Zhangbo:
8.29
</P><PRE>Use the plasmid from Zhaobing to construct the pACYC184-T7-RFP/RFPM
</PRE><H2><SPAN class="mw-headline" id="8.2F30.2F2011">8/30/2011</SPAN></H2><P>Zhangbo:
8.30
</P><PRE>RFP PCR
PCR KOD buf  5ul
dNTP        5ul
Mg2+        3ul
F/R(RFP)    2/2ul
Plasmid(13M RFP) 1ul
KOD           1ul
ddH2O       31ul 
</PRE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:371px;"><DIV class="thumbcaption">figure1:Test of the RFP PCR</DIV></DIV></DIV></DIV><PRE>Wild left mutate right
Purified 
Buf    2ul
DNA   16ul(plasmid)
Nde1   1ul
Xho1   1ul
Total    20ul
37℃ 5hours
</PRE><P>Nachte:EcoRI and XholI condigestion
</P><P>pET28a-Amp                       16ul
</P><P>FD buffer                            2ul
</P><P>EcoRI                                  1ul
</P><P>XholI                                   1ul
</P><P>programm:37℃*6h
</P><P>connection process
</P><P>T4 buffer                             1.5ul
</P><P>PEG                                     1.5ul
</P><P>plasmid                               2 ul
</P><P>'G-L-R'fragment                  4ul
</P><P>T4 lingase                           1ul
</P><P>ddH2O                                5ul
</P><P>22℃*2h，70℃*5min
</P><H2><SPAN class="mw-headline" id="8.2F31.2F2011">8/31/2011</SPAN></H2><P>Zhangbo:
</P><PRE>Binding the product together
Transfer the plasmid into the DH5a
10ul into 100ul DH5a
30min on the ice
37℃ 2min and 5min on the ice
220rpm 1 hour 
Tet solid medium for 16 hours to 24 hours
</PRE><P>Nachte：Pony did transformation last night,and I pick colonies this morning.Using colony PCR to test whether the digestion
is successful.
</P><P>Samples A-1 A-3 T-2 T-3 T-4 T-6 T-8 are positive.After 6h's shaking,we extracted plasmid.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:SJTU-BioX-Shanghai/Notebook/Lablog1">http://2011.igem.org/Team:SJTU-BioX-Shanghai/Notebook/Lablog1</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SJTU-BioX-Shanghai/Notebook/Lablog1" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SJTU-BioX-Shanghai/Notebook/Lablog1" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/Notebook/Lablog1&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/Notebook/Lablog1&amp;oldid=226756" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>