<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Marburg_SYNMIKRO_Results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Marburg_SYNMIKRO/Results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Marburg_SYNMIKRO/Results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Marburg_SYNMIKRO/Results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Marburg_SYNMIKRO/Results&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Marburg_SYNMIKRO/Results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Marburg SYNMIKRO/Results</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><UL id="vertical-navigation"><LI class="home"><A href="/Team:Marburg_SYNMIKRO">Home</A></LI><LI class="team"><A href="/Team:Marburg_SYNMIKRO/Team">Team</A></LI><LI class="child"><A href="/Team:Marburg_SYNMIKRO/TeamProfile">Team members</A></LI><LI class="child"><A href="https://igem.org/Team.cgi?year=2012&amp;team_name=Marburg_SYNMIKRO">Official Team Profile</A></LI></UL><LI class="project"><A href="/Team:Marburg_SYNMIKRO/Project">Project</A></LI><LI class="child"><A href="/Team:Marburg_SYNMIKRO/Results">Results</A></LI><LI class="notebook"><A href="/Team:Marburg_SYNMIKRO/Notebook">Notebook</A></LI><LI class="child"><A href="/Team:Marburg_SYNMIKRO/Parts">Submitted Parts</A></LI><LI class="child"><A href="/Team:Marburg_SYNMIKRO/Primer">Primer List</A></LI><LI class="safety"><A href="/Team:Marburg_SYNMIKRO/Happening">Human Practice</A></LI><LI class="child"><A href="/Team:Marburg_SYNMIKRO/Safety">Safety</A></LI><LI class="attributions"><A href="/Team:Marburg_SYNMIKRO/Attributions">Attributions</A></LI></P><DIV class="conboxb"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Biobricks:"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Biobricks:</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Experiments:"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Experiments:</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Generation_of_test_constructs:"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Generation of test constructs:</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Fluorescence_microscopy:"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Fluorescence microscopy:</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Biobricks:"><B>Biobricks:</B></SPAN></H3><P><B><A href="http://partsregistry.org/Part:BBa_K866000" class="external text" rel="nofollow">BBa_K866000</A> (CFP-Fusion Brick):</B></P><P>In order to evaluate putative recombination events in <I>E. coli</I>, we decided to generate fluorescent fusion proteins, which localize at different compartments within a cell. 
<A href="http://partsregistry.org/Part:BBa_E0020" class="external text" rel="nofollow">BBa_E0020</A> is a biobrick coding for the Cyan Fluorescent Protein (CFP). To make CFP suitable for fusion proteins, we used this biobrick as a template to create a CFP coding sequence (CDS) which can be cloned in-frame to other protein coding sequences via <A href="http://partsregistry.org/Assembly:Standard_assembly" class="external text" rel="nofollow">Standard Assembly</A>. 
</P><P>Our novel CFP-fusion brick was constructed by site directed mutagenesis through polymerase chain reaction (PCR) using primer <A href="https://2012.igem.org/Team:Marburg_SYNMIKRO/Primer" class="external text" rel="nofollow">MS_16</A> and <A href="https://2012.igem.org/Team:Marburg_SYNMIKRO/Primer" class="external text" rel="nofollow">MS_17</A>. PCR products were separated by agarose gel electrophoresis, extracted from the gel and cloned into <A href="http://partsregistry.org/Part:pSB1C3" class="external text" rel="nofollow">pSB1C3</A> via <A href="http://partsregistry.org/Help:Standards/Assembly/RFC10" class="external text" rel="nofollow">Standard Assembly</A>. Positive clones were verified by <A href="/Team:Marburg_SYNMIKRO/Notebook#27.08.2012" title="Team:Marburg SYNMIKRO/Notebook">colony PCR</A> and <A href="http://partsregistry.org/cgi/partsdb/dna.cgi?part_name=BBa_K866000" class="external text" rel="nofollow">sequencing</A>. The CFP fusion brick was verified by expression and localization in <I>E. coli</I> as fusion protein with different intracellular localization domains (A-modules) <A href="/Team:Marburg_SYNMIKRO/Project#Localization_domains" title="Team:Marburg SYNMIKRO/Project">GroES and AmiC</A> using <A href="/Team:Marburg_SYNMIKRO/Results#Fluorescence_microscopy:" title="Team:Marburg SYNMIKRO/Results">fluorescence microscopy</A>.
</P><P><B><A href="http://partsregistry.org/Part:BBa_K866001" class="external text" rel="nofollow">BBa_K866001</A> (mRFP-Fusion Brick):</B></P><P>In addition to the CFP-fusion protein, fusion-competent monomeric Red Fluorescent Protein (mRFP) was contructed from <A href="http://partsregistry.org/Part:BBa_E1010" class="external text" rel="nofollow">BBa_E1010</A> as template. Site directed mutagenesis was performed via PCR using primer <A href="https://2012.igem.org/Team:Marburg_SYNMIKRO/Primer" class="external text" rel="nofollow">MS14</A> and <A href="https://2012.igem.org/Team:Marburg_SYNMIKRO/Primer" class="external text" rel="nofollow">MS15</A>. PCR products were separated by agarose gel electrophoresis, extracted from the gel and cloned into <A href="http://partsregistry.org/Part:pSB1C3" class="external text" rel="nofollow">pSB1C3</A> via <A href="http://partsregistry.org/Help:Standards/Assembly/RFC10" class="external text" rel="nofollow">Standard Assembly</A>. Positive clones have been verified by <A href="/Team:Marburg_SYNMIKRO/Notebook#27.08.2012" title="Team:Marburg SYNMIKRO/Notebook">colony PCR</A> and <A href="http://partsregistry.org/cgi/partsdb/dna.cgi?part_name=BBa%20K866001" class="external text" rel="nofollow">DNA sequencing</A>.
</P><P><B><A href="http://partsregistry.org/Part:BBa_K866002" class="external text" rel="nofollow">BBa_K866002</A>  (Gin DNA Invertase):</B></P><P>The G-segment Invertase (Gin) from <A href="/Team:Marburg_SYNMIKRO/Project#Phage_Mu" title="Team:Marburg SYNMIKRO/Project">phage Mu</A> is a site-specific DNA recombinase, which causes inversion of gene segments flanked by <A href="/Team:Marburg_SYNMIKRO/Project#Gin_recombination" title="Team:Marburg SYNMIKRO/Project">gix sites</A> as inverted repeats. If the gene segment is flanked by gix-sites as direct repeats Gin recombinase causes <A href="/Team:Marburg_SYNMIKRO/Project#Site-specific_recombination" title="Team:Marburg SYNMIKRO/Project">deletions</A> of the gene segment. Efficient recombination depends on a short DNA-segment which acts as recombinational <A href="/Team:Marburg_SYNMIKRO/Project#Enhancer" title="Team:Marburg SYNMIKRO/Project">Enhancer</A>.
To facilitate expression of Gin the biobrick contains a ribosomal binding site (RBS) attached upstream of the Gin start codon.
</P><P>We designed primer <A href="https://2012.igem.org/Team:Marburg_SYNMIKRO/Primer" class="external text" rel="nofollow">MS11</A> and <A href="https://2012.igem.org/Team:Marburg_SYNMIKRO/Primer" class="external text" rel="nofollow">MS12</A> to fuse the well characterized and strong RBS <A href="http://partsregistry.org/Part:BBa_J61100" class="external text" rel="nofollow">BBa_J61100</A> from the Anderson RBS-family during amplification of the Gin coding sequence. As template for PCR, chromosomal DNA from <I>E. coli</I> strain C600 Mucts62 (lysogenic for temperature sensitive phage Mu cts62) was used. PCR amplicons were separated via agarose gel electrophoresis and cloned into <A href="http://partsregistry.org/Part:pSB1C3" class="external text" rel="nofollow">pSB1C3</A> through <A href="http://partsregistry.org/Help:Standards/Assembly/RFC10" class="external text" rel="nofollow">Standard Assembly</A>. Positive clones were evaluated by <A href="/Team:Marburg_SYNMIKRO/Notebook#20.09.2012" title="Team:Marburg SYNMIKRO/Notebook">restriction enzyme digest</A> and <A href="http://partsregistry.org/cgi/partsdb/dna.cgi?part_name=BBa%20K866002" class="external text" rel="nofollow">DNA sequencing</A>.
The <A href="http://partsregistry.org/Part:BBa_K866002" class="external text" rel="nofollow">Gin DNA Invertase Biobrick</A> can be used with promoters of different strength in order to evaluate the optimal level of Gin-expression to stimulate recombination events in <I>E. coli</I>.
</P><P><B><A href="http://partsregistry.org/Part:BBa_K866003" class="external text" rel="nofollow">BBa_K866003</A> (SacB Suicide Gene):</B></P><P>To allow counter-selection against non-recombined plasmids, the suicide gene <I>sacB</I> was chosen. In the presence of sucrose in the medium, expression of the <I>Bacillus subtilis sacB</I> gene is toxic for gram negative <I>E. coli</I> cells. <I>sacB</I> codes for the enzyme levansucrase, which uses sucrose to produce the polymeric levan (2,6-β-D-fructosyl)<SUB>n</SUB> in the periplasm, which disturbs cell metabolism and growth, leading to cell death. In our final <A href="/Team:Marburg_SYNMIKRO/Project#Construct" title="Team:Marburg SYNMIKRO/Project">recombination construct</A>, <I>sacB</I> will be located between the A and B-modules. Successful recombination will cause deletion of the suicide gene and thus cells that contain plasmids with positive recombination events can be selected on sucrose containing media.
The <I>sacB</I> gene was amplified using primer <A href="https://2012.igem.org/Team:Marburg_SYNMIKRO/Primer" class="external text" rel="nofollow">MS26</A> and <A href="https://2012.igem.org/Team:Marburg_SYNMIKRO/Primer" class="external text" rel="nofollow">MS27</A> from the suicide plasmid pNTPS (kindly provided by K. Thormann, MPI Marburg) via PCR. PCR products were separated by agarose gel electrophoresis, extracted from the gel and cloned into <A href="http://partsregistry.org/Part:pSB1C3" class="external text" rel="nofollow">pSB1C3</A> via <A href="http://partsregistry.org/Help:Standards/Assembly/RFC10" class="external text" rel="nofollow">Standard Assembly</A>. Positive clones were verified by <A href="/Team:Marburg_SYNMIKRO/Notebook#17.09.2012" title="Team:Marburg SYNMIKRO/Notebook">restriction enzyme digest</A> and <A href="http://partsregistry.org/cgi/partsdb/dna.cgi?part_name=BBa%20K866003" class="external text" rel="nofollow">DNA sequencing</A>.
<A href="http://partsregistry.org/Part:BBa_K866003" class="external text" rel="nofollow">BBa_K866003</A> can be fused to promoters of different strength and tested for optimal counter-selection through cell death on sucrose containing media.
</P></DIV><DIV class="conboxb"><H3><SPAN class="mw-headline" id="Experiments:"><B>Experiments:</B></SPAN></H3><H4><SPAN class="mw-headline" id="Generation_of_test_constructs:"> Generation of test constructs: </SPAN></H4><P>As a first step towards our complete <A href="/Team:Marburg_SYNMIKRO/Project#Construct" title="Team:Marburg SYNMIKRO/Project">recombination construct</A>, we generated the final recombination gene products i.e. the fusion of a complete A-module with a complete B-module connected by a <I>gix</I> recombination site. These putative recombination constructs would be the outcome if a recombination event would have occurred in the complete <A href="/Team:Marburg_SYNMIKRO/Project#Construct" title="Team:Marburg SYNMIKRO/Project">recombination construct</A>.
For this purpose, all three cellular localization domains (A-Modules), <A href="/Team:Marburg_SYNMIKRO/Project#Localization_domains" title="Team:Marburg SYNMIKRO/Project">HU, GroES and AmiC</A> have been synthesized. The A modules each consist of the localization domain (AmiC, GroES or HU) preceded by a standard ribosome binding site (RBS) <A href="http://partsregistry.org/Part:BBa_J61100" class="external text" rel="nofollow">BBa_J61100</A> and the strong constitutive promotor <A href="http://partsregistry.org/Part:BBa_J23108" class="external text" rel="nofollow">BBa_J23108</A>. Downstream of the localization domains, a <A href="/Team:Marburg_SYNMIKRO/Project#Gin_recombination" title="Team:Marburg SYNMIKRO/Project"><I>gix</I> site</A> is located. Between promotor, RBS, localization domain and the gix site, the <A href="http://partsregistry.org/Help:Standards/Assembly/RFC10" class="external text" rel="nofollow">scar sequence</A> from standard assembly was included to test for the correct expression of the reading frame. The complete A-modules are flanked by the biobrick standard assembly <A href="http://partsregistry.org/Help:Standards/Assembly/RFC10" class="external text" rel="nofollow">RFC10</A> prefix and suffix.
</P><P>The complete A-modules coding for AmiC and GroES were cloned N-terminally to
our fluorescent fusion protein sequences <A href="http://partsregistry.org/Part:BBa_K866000" class="external text" rel="nofollow">BBa_K866000</A> (CFP-Fusion Brick),  <A href="http://partsregistry.org/Part:BBa_K866001" class="external text" rel="nofollow">BBa_K866001</A> (mRFP-Fusion brick as well as the biobrick <A href="http://partsregistry.org/Part:BBa_K125500" class="external text" rel="nofollow">BBa_K125500</A> (GFP-Fusion Brick) via standard assembly. Verification of fluorescence was performed via fluorescence microscopy.
</P><H4><SPAN class="mw-headline" id="Fluorescence_microscopy:"> Fluorescence microscopy: </SPAN></H4><P>In order to screen for fluorescent <I>E. coli</I> harboring our putative test constructs, positively selected colonies through antibiotic resistance have been subjected to fluorescence microscopy.
For this purpose, colonies growing on LB-Agar supplemented with antibiotics for <A href="http://partsregistry.org/Part:BBa_K866000" class="external text" rel="nofollow">BBa_K866000</A> (CFP-Fusion Brick),  <A href="http://partsregistry.org/Part:BBa_K866001" class="external text" rel="nofollow">BBa_K866001</A> (mRFP-Fusion brick) and <A href="http://partsregistry.org/Part:BBa_K125500" class="external text" rel="nofollow">BBa_K125500</A> (GFP-Fusion Brick), respectively, were cultured over-night in LB-medium with appropriate antibiotics. 
Small aliquots (10 µL) of overnight cultures were added onto  2% agarose-pads casted on microscope slides, covered by cover-slips and used for fluorescence microscopy.
</P><P>Altogether, 6 different constructs have been analyzed for fluorescence, being AmiC-GFP, AmiC-CFP, AmiC-mRFP, GroES-GFP, GroES-CFP and GroES-mRFP.
As can be seen in Figure 1, fluorescence was observed in all cells carrying a fusion construct. Especially in cells expressing GroES-GFP and AmiC-GFP (Fig. 1 E and B), intense fluorescence was detectable at cell poles for GroES and in the periplasm for AmiC. Similar localizations were also observed for GroES-mRFP and AmiC-mRFP (Fig. 1 D and A). For GroES-CFP and AmiC-CFP (Fig. 1 F and C), fluorescence is detectable, yet for AmiC-CFP no precise localization could be determined.
</P><P>As a result, our test constructs representing a putative recombination event of our planned <A href="/Team:Marburg_SYNMIKRO/Project#Construct" title="Team:Marburg SYNMIKRO/Project">recombination construct</A> were successfully expressed in <I>E.coli</I>. Furthermore, GFP- and mRFP fused A-modules show enhanced fluorescence at cell poles for GroES and in the periplasm for AmiC.
</P><P><B>Fig. 1</B>: Fluorescent <I>E. coli</I> cells harbouring the fusion constructs of <A href="/Team:Marburg_SYNMIKRO/Project#Construct" title="Team:Marburg SYNMIKRO/Project">A- and B-modules</A>. <B>A</B>: AmiC-mRFP. <B>B</B>: AmiC-GFP. <B>C</B>: AmiC-CFP. <B>D</B>: GroES-mRFP. <B>E</B>: GroES-GFP. <B>F</B>: GroES-CFP. <B>G</B>: <I>E. coli</I> harboring a Promotor-RBS construct which serves as a non-fluorescent control. All images were taken by fluorescence microscopy at a magnification of x100.
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Marburg_SYNMIKRO/Results">http://2012.igem.org/Team:Marburg_SYNMIKRO/Results</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Marburg_SYNMIKRO/Results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Marburg_SYNMIKRO/Results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Marburg_SYNMIKRO/Results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Marburg_SYNMIKRO/Results&amp;oldid=226665" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>