<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CSU_Fort_Collins_Notebook_Quorum_Sensing skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CSU Fort Collins/Notebook/Quorum Sensing</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><A href="/wiki/index.php?title=Template:CSS/Main&amp;action=edit&amp;redlink=1" class="new" title="Template:CSS/Main (page does not exist)">Template:CSS/Main</A></P><P><TITLE>Quorum Sensing Notes</TITLE></P><DIV id="cssmenu"><UL><LI><A href="/Team:CSU_Fort_Collins"><SPAN>Home</SPAN></A></LI><LI class="has-sub"><A href="/Team:CSU_Fort_Collins/Project/"><SPAN>Project</SPAN></A></LI><LI><A href="/Team:CSU_Fort_Collins/Phage/"><SPAN>Phage</SPAN></A></LI><LI><A href="/Team:CSU_Fort_Collins/QuorumSensing/"><SPAN>Quorum Sensing</SPAN></A></LI><LI><A href="/Team:CSU_Fort_Collins/Modeling/"><SPAN>Modeling</SPAN></A></LI></UL><LI class="has-sub"><A href="/Team:CSU_Fort_Collins/Members/"><SPAN>Team</SPAN></A></LI><LI><A href="/Team:CSU_Fort_Collins/Members/"><SPAN>Members</SPAN></A></LI><LI><A href="/Team:CSU_Fort_Collins/Sponsors/"><SPAN>Sponsors</SPAN></A></LI><LI class="last"><A href="/Team:CSU_Fort_Collins/Acknowledgements/"><SPAN>Acknowledgements</SPAN></A></LI><LI class="has-sub" class="active"><A href="/Team:CSU_Fort_Collins/Notebook/"><SPAN>Notebook</SPAN></A></LI><LI class="has-sub"><A href="/Team:CSU_Fort_Collins/Notebook/Protocols"><SPAN>Protocols</SPAN></A></LI><LI class="has-sub"><A href="/Team:CSU_Fort_Collins/Notebook/DailyNotes"><SPAN>Daily Notes</SPAN></A></LI><LI class="has-sub"><A href="/Team:CSU_Fort_Collins/Notebook/Phage"><SPAN>Phage</SPAN></A></LI><LI class="has-sub"><A href="/Team:CSU_Fort_Collins/Notebook/Quorum Sensing"><SPAN>Quorum Sensing</SPAN></A></LI><LI class="last"><A href="/Team:CSU_Fort_Collins/InterLab"><SPAN>Interlab Study</SPAN></A></LI><LI class="has-sub"><A href="/Team:CSU_Fort_Collins/Human Practices/"><SPAN>Human Practices</SPAN></A></LI><LI><A href="/Team:CSU_Fort_Collins/Collab/"><SPAN>Collaboration</SPAN></A></LI><LI class="last"><A href="/Team:CSU_Fort_Collins/Outreach/"><SPAN>Outreach</SPAN></A></LI><LI class="last"><A href="/Team:CSU_Fort_Collins/Integrated Human Practices/"><SPAN>Integrated Human Practices</SPAN></A></LI><LI class="has-sub"><A href="/Team:CSU_Fort_Collins/Parts/"><SPAN>Achievements</SPAN></A></LI><LI><A href="/Team:CSU_Fort_Collins/Parts/"><SPAN>Parts</SPAN></A></LI><LI class="last"><A href="/Team:CSU_Fort_Collins/Judging/"><SPAN>Judging</SPAN></A></LI><LI class="last"><A href="/Team:CSU_Fort_Collins/Safety/"><SPAN>Safety</SPAN></A></LI></DIV><DIV class="wrapper"><CENTER><DIV class="container"><CENTER><SPAN id="ftoc"><A href="/Team:CSU_Fort_Collins/Notebook/" id="ftoc">← Full Table of Contents</A></SPAN></CENTER><H2>Quorum Sensing Notebook</H2><H4>May 21 </H4>
Started discussing quorum sensing (QS) as a tunable mechanism after looking at S. aureus’s QS network. 

<H4>May 28 </H4>
Answered questions like whether a transmembrane protein from the agr locus can be expressed and useful in e. coli because e. coli is gram negative whereas S. aureus is gram positive. Began to brainstorm how to model gene circuit mainly as a switch- like behavior. 

<H4>Jun 4</H4>
Determined that we wanted to model QS system stochastically which requires a matrix for. Used “Modelling a Synthetic Biological Chaotic System: Relaxation Oscillators Coupled by Quorum Sensing” by Chen et. al to begin to understand a more mathematical approach to QS and how others have modelled it before. 

<H4>Jun 11</H4>
Dissected the QS system in S. aureus: agrABCD translate into the auto inducing peptide (AIP)  complex (agrD), a transmembrane protein responsible for the transport of AIP (agrB), a transmembrane protein responsible for binding to intercellular AIP (agrC), and finally a protein that is phosphorylated by agrC in order to upregulate P2 and promote additional translation of the agr complexes or P3 which promotes translation of virulence factors (agrA). From here we proposed that if we were able to implement the only agrAC and have a reporter protein such as GFP then we could sense when staph was present in our system. The one drawback to S. aureus as target is that there are variations of AIP and while is highly specific to different strains of S. aureus can inhibit QS of different strains. (Type 1 AIP – Type 4 AIP)

<H4>Jun 18</H4>
Finally found a paper, “Peptide- Based Communication Systems Enables Escherichia Coli to Bacillus Megaterium Interspecies Signaling” by Marchand et al. Which details the ability to implement S. aureus’s QS system in both E. coli and B. megaterium. This article showed that we can split up the agr complexes and express AIP production in E. coli and signal recognition of the B. megaterium. This was the first paper we found that detailed some gram positive and negative bacteria signaling via QS. 
Modelling ran into issue of how to express a hill function in a stochastic model. 
Found iGEM parts BBa_I746001 and BBa_I746107 which produced agrBD in the former and agrAC in the latter. Figured out promoters to try to attach to parts.


<H4>Jun 25 </H4>
Trying to solve the hill function by possibly applying a finite element analysis

<H4>July 2 </H4>
Made competent cells. Resurrected parts BBa_J23100 (constitutive promoter) BBa_I0500 (Arabinose promoter) BBa_ K180000 (IPTG Promoter) BBa_ I746001 and BBa_I746107 (agr complexes). Failed using SOB media.

<H4>July 9 </H4>
July 9	Created first matrix for modelling using generic rate constants as place holders.

<H4>July 16</H4>
July 16	Nano-dropped resurrected parts, ran a PCR to confirm strain of S. carnosus streaked out parts from SOC resurrection attempt J23100 was streaked out onto wrong antibiotic plates need ampicillin instead of chloramphenicol. Started overnight culture for miniprep the following day ran miniprep for parts K1800000, I0500, and  I746001. Chose fluorescent tags to measure protein expression made sure proteins were greater 50nm emission from each protein.

<H4>July 23</H4>
July 23	Miniprepped all parts. Sent parts out for sequencing to confirm sequences from iGEM kit. 

<H4>July 30</H4>
Created digestion/ligation protocols for each combination of promoter (backbone) and part (insert)

<H4>August 6</H4>
Designed Golden Gate primers for fluorescent protein reactions if ligations work. Failed to PCR confirm first round of ligations

<H4>August 13</H4>
Moved from microbiology lab to Dr. Peebles lab in Scott bioengineering

<H4>August 20</H4>
Rescheduled everyone for academic semester, created tentative plan for iGEM until jamboree.

<H4>August 27</H4>
Re-streaked parts to make sure that bacteria are alive and made ampicillin plates

<H4>September 3</H4>
Ran colony PCR and gel~88 wells 5 gels

<H4>September 10</H4>
Re-Ligated colonies with a “No ligation” control. PCR indicating non-specific binding of primers.  Adjusted ampicillin for 100uM we were operating at 35uM same as chloramphenicol.

<H4>September 17</H4>
Re- Ligated with more people doing the same ligations to try to increase chance of success. Reset to try to plan out better ligation
<H4>September 24</H4>
Prepped parts for miniprep and ran digest, confirm part digestion via electrophoresis. 9 total ligations. Failed to confirm, bought new restriction enzymes.
<H4>October 1</H4>
Ran new digest, used NEB buffer 2.1 and 3.1 to avoid extra restriction enzyme activity and let the digest reaction run longer. 
<H4> October 8</H4>
 Confirmed if digest worked, ligations were confirmed via part digest using EcoR1 and Pst1 Restriction. Submitted part I746107+K180000 as BBa_ K272001.

</DIV></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>