<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CSU_Fort_Collins_QuorumSensing skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CSU Fort Collins/QuorumSensing/</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><A href="/wiki/index.php?title=Template:CSS/Main&amp;action=edit&amp;redlink=1" class="new" title="Template:CSS/Main (page does not exist)">Template:CSS/Main</A></P><P><TITLE>Quorum Sensing </TITLE></P><DIV id="cssmenu"><UL><LI><A href="/Team:CSU_Fort_Collins"><SPAN>Home</SPAN></A></LI><LI class="has-sub"><A href="/Team:CSU_Fort_Collins/Project/"><SPAN>Project</SPAN></A></LI><LI><A href="/Team:CSU_Fort_Collins/Phage/"><SPAN>Phage</SPAN></A></LI><LI><A href="/Team:CSU_Fort_Collins/QuorumSensing/"><SPAN>Quorum Sensing</SPAN></A></LI><LI><A href="/Team:CSU_Fort_Collins/Modeling/"><SPAN>Modeling</SPAN></A></LI></UL><LI class="has-sub"><A href="/Team:CSU_Fort_Collins/Members/"><SPAN>Team</SPAN></A></LI><LI><A href="/Team:CSU_Fort_Collins/Members/"><SPAN>Members</SPAN></A></LI><LI><A href="/Team:CSU_Fort_Collins/Sponsors/"><SPAN>Sponsors</SPAN></A></LI><LI class="last"><A href="/Team:CSU_Fort_Collins/Acknowledgements/"><SPAN>Acknowledgements</SPAN></A></LI><LI class="has-sub" class="active"><A href="/Team:CSU_Fort_Collins/Notebook/"><SPAN>Notebook</SPAN></A></LI><LI class="has-sub"><A href="/Team:CSU_Fort_Collins/Notebook/Protocols"><SPAN>Protocols</SPAN></A></LI><LI class="has-sub"><A href="/Team:CSU_Fort_Collins/Notebook/DailyNotes"><SPAN>Daily Notes</SPAN></A></LI><LI class="has-sub"><A href="/Team:CSU_Fort_Collins/Notebook/Phage"><SPAN>Phage</SPAN></A></LI><LI class="has-sub"><A href="/Team:CSU_Fort_Collins/Notebook/Quorum Sensing"><SPAN>Quorum Sensing</SPAN></A></LI><LI class="last"><A href="/Team:CSU_Fort_Collins/InterLab"><SPAN>Interlab Study</SPAN></A></LI><LI class="has-sub"><A href="/Team:CSU_Fort_Collins/Human Practices/"><SPAN>Human Practices</SPAN></A></LI><LI><A href="/Team:CSU_Fort_Collins/Collab/"><SPAN>Collaboration</SPAN></A></LI><LI class="last"><A href="/Team:CSU_Fort_Collins/Outreach/"><SPAN>Outreach</SPAN></A></LI><LI class="last"><A href="/Team:CSU_Fort_Collins/Integrated Human Practices/"><SPAN>Integrated Human Practices</SPAN></A></LI><LI class="has-sub"><A href="/Team:CSU_Fort_Collins/Parts/"><SPAN>Achievements</SPAN></A></LI><LI><A href="/Team:CSU_Fort_Collins/Parts/"><SPAN>Parts</SPAN></A></LI><LI class="last"><A href="/Team:CSU_Fort_Collins/Judging/"><SPAN>Judging</SPAN></A></LI><LI class="last"><A href="/Team:CSU_Fort_Collins/Safety/"><SPAN>Safety</SPAN></A></LI></DIV><DIV class="wrapper"><CENTER><DIV class="container"><H2> Quorum Sensing Background:</H2>
	Quorum sensing is a very fascinating method of communication in bacteria. While unable to “talk” to each other bacteria can excrete chemicals that relay information from one cell to another. In bacteria that utilize biofilms this process is used to determine the population of the biofilm. As the concentration of this chemical approaches some threshold value a bacterium will begin to utilize different transcriptions pathways as a response to this chemical gradient. For our project we focus on the quorum sensing mechanism expressed in Staphylococcus aureus which uses the concentration gradient of an autoinducing peptides (AIP) to become virulent and infect the host organism. The reason that staph doesn’t begin to infect the host immediately is that a single cell is ineffective against a multicellular organism but the biofilm they create becomes a loading dose as the population of staph increases. This allows the bacteria to survive much longer and affect the host organism.<H2>How we are using this differently:</H2>
	A lot of research has used these mechanisms to inhibit this bacterium. The main idea being that if the staph cannot communicate then they become inert. This is a perfectly logical approach, but we want to try to use this process as backbone for a killing mechanism instead of being the mechanism. S. aureus contains a relatively simple protein network for its immediate quorum sensing processes. There are four proteins that make up the base network for staph AgrD becomes AIP when maturated and excreted by AgrB; which is a transmembrane protein. Then the AIP binds to a transmembrane receptor AgrC which phosphorylates AgrA allowing AgrA to either upregulate the translation of these Agr complexes (in low concentrations of AIP) or begin translating virulence factors. Knowing this we attempted to create a system that can sense AIP and produce a kill mechanism that we desire i.e. a phage. This allows us to guarantee that staff isn’t present when therapy is applied, and the expression of the phage occurs not too early or late. 

We wanted to understand the kinetics of protein expression so that we could compare experimental results to a model developed in MATLAB. To attempt to do this we came up with a combination of different promoters to understand different rate-limiting steps at different time scales. We used parts:<P>BBa_I746107- Gene that translates agrAC and a GFP in opposite directions to simulate virulence choice.</P><P>BBa_I74001- Gene that translates agrBD to produce AIP</P><P>BBa_J23100- A constitutive promoter.</P><P>BBa_I0500- Arabinose inducible promoter</P><P>BBa_K1800000- IPTG inducible promoter</P><P>BBa_E0020- Cyan fluorescent protein</P><P>BBa_E0040- Yellow fluorescent protein</P><P>The fluorescent proteins are used to develop quantitative results via optical density. We approximate the expression of the fluorescent proteins to be proportional to the expression of the Agr complexes.</P><H2>Results: </H2>
	Unfortunately, we were unable to complete the parts to test if this proposed mechanism works. For future work we would need to test the generic primers from iGEM to see if the primers denatured over the summer due to freezing and thawing over and over so we have more accurate PCR amplification. Also, we need to optimize our ligation protocol because that’s where we struggled the most. We also hope to change the fluorescent proteins to actual protein tags to get a more direct measurement of the of translation.<H2> Works Consulted/Cited</H2><SPAN id="source1"><P><CITE>Stochastic Switching in Biology: From Genotype to Phenotype On the Hamiltonian Structure of Large Deviations in Stochastic Hybrid Systems Paul C Bressloff and Olivier Faugeras - A Mathematical Model and Quantitative Comparison of the Small RNA Circuit In.” http://iopscience.iop.org/article/10.1088/1751-8121/aa5db4/pdf </CITE> by Banik, Suman K, Andrew T Fenley, and Rahul V Kulkarni. http://iopscience.iop.org/article/10.1088/1751-8121/aa5db4/pdf (June 20, 2018).</P></SPAN><P><CITE>Modeling a Synthetic Biological Chaotic System: Relaxation Oscillators Coupled by Quorum Sensing.,</CITE> by Chen, Aimin. 2011. Nonlinear Dynamics 63(4): 711–18. https://link-springer-com.ezproxy2.library.colostate.edu/content/pdf/10.1007%2Fs11071-010-9832-1.pdf (May 24, 2018).</P><P><CITE>. Quorum-Sensing Regulation in Staphylococci—an Overview.</CITE> by Huillet, Eugenie, Yves Le Loir, Michael Otto, and Katherine Y Le. 2015. Painted in 1893.https://com-mendeley-prod-publicsharing-pdfstore.s3.eu-west-1.amazonaws.com/8305-PUBMED/10.3389/fmicb.2015.01174/fmicb_06_01174_pdf.pdf?X-Amz-Security-Token=FQoDYXdzEGYaDIAo4dlhMO84j2DZJiK2A30WsCLi0%252FUvpibI4MAebVYAu5ZZWUuQ0bnnpO%252FkYfEqAokyf8ICmNdKSL (June 13, 2018).</P><P><CITE>Peptide-Based Communication System Enables Escherichia Coli to Bacillus Megaterium Interspecies Signaling</CITE> by Marchand, Nicholas, and Cynthia H. Collins. 2013.Biotechnology and Bioengineering 110(11): 3003–12.</P><P><CITE>Influence of the AgrC-AgrA Complex on the Response Time of Staphylococcus Aureus Quorum Sensing</CITE> by Srivastava, Sandeep K et al. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4135676/pdf/zjb2876.pdf (May 24, 2018).</P><P><CITE>Regulation of Virulence in Staphylococcus Aureus: Molecular Mechanisms and Remaining Puzzles</CITE> by Wang, Boyuan, and Tom W Muir. 2016. Cell Chemical Biology 23(2): 214–24. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4847544/pdf/nihms-756132.pdf (June 13, 2018).</P></DIV></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>