Team:CSU Fort Collins/Notebook/Phage
Template:CSS/Main
Phage Notes
Home
Project
Phage
Quorum Sensing
Modeling
Team
Members
Sponsors
Acknowledgements
Notebook
Protocols
Daily Notes
Phage
Quorum Sensing
Interlab Study
Human Practices
Collaboration
Outreach
Integrated Human Practices
Achievements
Parts
Judging
Safety
← Full Table of Contents
Phage Notebook
May 23
Started discussing a trigger to make phage expression work. How would this mechanism be incorporated to relate to concentration gradient (QS). Some triggers include mitomycin C and phleomycin. These triggers are linked to the cellular SOS response (DNA damage). We concluded that an arabinose promoter could be used to regulate the control of a recA gene which would cleave a repressor and cause phage replication. The next step would be to find a phage that we would be able to use.
May 31
Developed a team plan. The team will be taking a whole phage genome (or at least the genes that are initially ‘turned on’ when a phage begins to replicate) and cloning it into a plasmid vector. The plasmid-phage combo will be transformed into E. coli (just like you all have been doing for practice). A promoter will be placed in the phage genome (on the plasmid) that will respond or be ‘turned on’ to a signal. The signal to turn on the promoter will be the quorum sensing molecule. The E. coli will ‘sense’ the quorum sensing molecule and send a signal to the promoter OR the promoter will directly respond to the quorum sensing molecule. Once this occurs, the phage will be induced to replicate. When the phage replicates, it will hopefully lyse E. coli, and phage will be released that will lyse nearby Staphylococcus. In this scenario, the phage will need to be probably a Staphylococcus phage, since once it lyses E. coli it will need to lyse Staphylococcus, preferably a Staphylococcus phage. For proof of concept, it could be a Staphylococcus phage, but this might be harder to find.
June 4
Most phages are specific for the genus (Staphylococcus vs. Escherichia) because they have a receptor to bind only to one genus (Staphylococcus) and not to other genera (like Escherichia). However, there may be other reasons that the phage will only replicate in a certain bacteria such as sigma factors, restriction enzymes present in one bacteria but not another, abortive phage proteins, and other mechanisms. However, we are not going to worry about that right now. By using the plasmid to carry the phage genome, it will be easier to manipulate the phage.
June 12
A promoter that is inducible by arabinose has been found. This part is BBa_K731201. This E. coli promoter is positively regulated by arabinose. AraC is the regulator protein: in absence of arabinose the protein binds to araI1 and araO2 and the DNA forms a loop that prevents RNA polymerase from binding the promoter. When arabinose is added to the culture it binds to araC, disabling the interaction with araI1 and araO2 sites, and also allowing it to bind araI site, promoting transcription. Promoter activity is also influenced by glucose levels: in its absence cAMP levels are high and the the complex cAMP+CAP (catabolite activator protein) binds to lacI promoting transcription. In presence of glucose, cAMP levels are low and promoter activity is inhibited.
June 29
The team continues to research phages that may work. A paper has been found that lists possible phages that may be used. Here is a link to the paper https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3417521/. The most promising choice for usable phage is S aureus BTN1260. This strain of S aureus's DNA encodes a phage that does not encode any virulence factors or toxins. Continued research into this possibility will be done.
July 10
Discussion of how to create primers was made at the meeting. A usable phage has been found. S.aureus ATCC25923 has 5 phages in its genome, 3 are not intact so not functional, 1 is intact but has no recognizable phage genes so is probably not functional, and 1 that is probably functional - called Region 3 on the bacterial chromosome. Dr. Gentry-Weeks has this strain in and we will streak it out on an agar plate. The Region 3 phage genome is 54.1 kb. From here we will streak out the bacteria from the freezer with help from Dr. Gentry-Weeks. Note that this isolate is NOT MRSA, which is great. This is a common lab strain that is used in antibiotic susceptibility testing, so it should be good (safer) for us to use. Then we will induce the prophage to excise and replicate using mitomycin C induction – this will confirm that the phage is functional. It would be a good idea to confirm that the phage that replicates corresponds to the phage encoded in region 3. The phage DNA could be isolated and identity confirmed by PCR using primers that are specific for the region 3 prophage. We will also run tests on a different phage (from S aureus strain BTN1260) once it is received from Dr. Robinson.
July 21
Developed a plan to start phage expression. Here are the parts we are going to use:
arabinose promoter BBa_I0500 --->   recA730  -- stop  lambda c1 regulated promotor BBa_R0051  ---> rBBa_E0020  -- stop  Constitutive Promoter BBa_J23100 --->RBS BBa_B0030  ---> cI repressor  BBa_C0051 -- stop
July 31
Grew B0030, J0445, and R0051 in 5mL LB broth tubes at 37C overnight
August 1
Phage induction conducted.
August 2
Phage induction did not go as planned due to complications from running the spectrophotometer incorrectly. The induction will be conducted again. Miniprep of parts BBa_R0051, BBa_J0445, BBa_B0030. Digested parts E0020 with xbaI and pstI. R0051 with SpeI and PstI. C0051 with xbaI and pstI. And B0030 with SpeI and PstI.
August 8
Conducted phage induction once again. This time when plating the phage plaques a total of 8 hosts were used.
August 9
Phage induction was a success. Plaques were found on multiple of the host plates.
September 6
recA Blunt end Cloning
Mixed dry recA730 with 40uL ddH2O to achieve 25ng/uL. Prepared ligation reaction mixture by combining the following in order: 3uL StrataClone Blunt Cloning Buffer, 2uL recA geneblock, and 1uL Strataclone Blunt Vector Mix. Mix by pippetting, then incubate at room temperature for 5 minutes. Place on ice until competent cells are thawed. Add 1uL of Cloning reaction mix to the thawed competent cells. Mix, then incubate on ice for 20 minutes. Heat shock mixture at 42 degrees C for 45 seconds. Incubate on ice for 2 minutes. Added 250uL of LB, and allowed cells to recover for 1 hour. Prepare LB+ampicillin plates for blue white colony screening by spreading 40uL of 2% X-gal on each plate. Plated 5uL, 100uL and 200uL of culture on separate plates, and incubated overnight.
September 25
Performed recA mutagenesis in protocols.
September 21
Mass digestion of parts conducted
October 7
Gels run on some digested parts
