<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_iHKU_protocol"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:iHKU/protocol">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:iHKU/protocol&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:iHKU/protocol&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:iHKU/protocol&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:iHKU/protocol" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:iHKU/protocol</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV id="main"><DIV id="up1"><TABLE width="100%" border="0"><TBODY><TR><TH width="61%" scope="row"> </TH><TD width="39%"><SPAN class="headline"><A href="http://www.hku.hk">The University of Hong Kong</A> | <A href="http://www.hku.hk/facmed/">Li Ka Shing Faculty of Medicine</A></SPAN></TD></TR></TBODY></TABLE></DIV><DIV id="down1"><DIV id="down1left"><DIV id="menu"><TABLE width="100%" border="0"><TBODY><TR><TH width="7%" height="335" scope="row"> </TH></TR><TR><TH width="5%" scope="row"> </TH><TD width="5%"> </TD></TR><TR><TH scope="row"> </TH><TD> </TD></TR><TR><TH scope="row"> </TH><TD> </TD></TR><TR><TH scope="row"> </TH><TD> </TD></TR><TR><TH scope="row"> </TH><TD> </TD></TR><TR><TH scope="row"> </TH><TD> </TD></TR><TR><TH scope="row"> </TH><TD> </TD></TR><TR><TH scope="row"> </TH><TD> </TD></TR><TR><TH scope="row"> </TH><TD> </TD></TR><TR><TH scope="row"> </TH><TD> </TD><TD> </TD></TR></TBODY></TABLE><P class="style15"> </P>    </DIV></DIV><DIV id="down1right"><DIV id="rightmain"><DIV id="content1"><TABLE width="100%" border="0"><TBODY><TR><TH width="10%"> </TH><TD width="80%" align="left" background="#p1"><H1>Protocols</H1><H3>Content:</H3><H2>Novel Protocols</H2><UL><LI>                              PROTOCOL-1 <A href="#p1">Pattern formation</A></LI><LI>                              PROTOCOL-2 <A href="#p2">Picture capture</A></LI><LI>                              PROTOCOL-3 <A href="#p4">Brightness cell concentration relationship Measurement</A></LI><LI>                              PROTOCOL-4 <A href="#p5">Migration rate measurement</A></LI><LI>                              PROTOCOL-5 <A href="#p6">Harvest sample from agar plate</A></LI></UL><H2 class="style17">Regular Protocols</H2><UL class="style18"><LI>PROTOCOL-6 <A href="#p7">β-galactosidase assay</A></LI><LI>PROTOCOL-7 <A href="#p8">Western blot</A></LI><LI>PROTOCOL-8 <A href="#p9">Recombineering</A></LI><LI>PROTOCOL-9 <A href="#p11">Most Probable Number  method (MPN)</A></LI><LI>PROTOCOL-10 <A href="#p12">Bradford assay</A></LI></UL><H2 class="style17">Other Protocols</H2><UL><LI class="style18"><A href="#p13">Casting  agarose gels</A></LI><LI class="style18"><A href="#p14">Running  agarose gels</A></LI><LI class="style18"><A href="#p15">Quick  gel purifcation</A></LI><LI class="style18"><A href="#p16">PCR  product purifcation</A></LI><LI class="style18"><A href="#p17">Count:  Preparation of Competent Cells for electro transformatio</A></LI><LI class="style18"><A href="#p18">Electro tranformation</A></LI><LI class="style18"><A href="#p19">Ligation</A></LI><LI class="style18"><A href="#p20">DNA extraction</A></LI><LI class="style18"><A href="#p21">Polymerase Chain Reaction</A></LI><LI class="style18"><A href="#p22">Plasmid digestion</A></LI></UL><P> </P><P> </P><P> </P><P class="style7"> </P><H3>Novel Protocols</H3><P><STRONG>PROTOCOL-1 <EM><U>Pattern formation</U></EM></STRONG></P><P>Day1--Inoculation</P><OL><LI><P>Inoculate  single colony into a 5ml LB with antibiotics into a Sterilin 20cm3  tube. Incubate in 37oC rotor at medium or maximum rotational speed.</P></LI></OL><P>Day2--Soft  Agar Plate Preparation</P><OL><LI><P>Weigh  LB broth with accuracy to 3 decimal places. Prepare a  master 2% LB of desired volume in a Pyrex bottle. </P></LI><LI><P>Weigh  BactoAgar with accuracy to 3 decimal places. If BactoTMAgar is used,  add 0.3g per 100ml of LB. If Difco Agar is  used, add 0.45g per 100ml of LB.</P></LI><LI><P>Add  to a Pyrex bottle (autoclavable) the agar and the appropriate LB solution to  dissolve the agar.</P></LI><LI><P>Autoclave the LB agar, then place  the LB agar into a 55oC before it solidifies.</P></LI><LI><P>Leave the LB agar to cool to 55oC  for at least half an hour.</P></LI><LI><P>Add glucose, IPTG (v/v) 1:1000, antibiotics (v/v) 1:1000 (if  applicable) into LB agar. Mix  well.</P></LI><LI><P>Aspirate 10cm3 of LB agar for each Petri dish, shake the  plate to flatter the solution across the whole plate</P></LI><LI><P>Wait for at least one and a half hour before adding cells to the  plate.</P></LI></OL><P> preculture for pattern development on  Soft Agar Plate</P><OL><LI><P>Add 2ml LB with appropriate  antibiotics to a Falcon 10ml preculture tube. </P></LI><LI><P>Add 40ul of the inoculation sample (dilution rate  1:50) to the tube.</P></LI><LI><P>Place  tubes in 37oC shaking incubator until OD600 reaches 0.8 -  1.0.</P></LI><LI><P>Add 2ul of the cell culture to  the centre of a soft agar plate. </P></LI><LI><P>Gently place the plates in 37oC  and incubate overnight (12 – 19 hrs). </P></LI></OL><P>&lt;Note&gt;</P><OL><LI><P>Use fresh plate, do not use plates prepared the day before since  water evaporates out from the plate and will cause the cells to swarm on the  surface. </P></LI><LI><P>Glucose &amp; Arabinose can be  stored in room temperature but best is to store it in 4oC.</P></LI></OL><P align="right"><A href="#top">[Back to Top]</A></P><P><STRONG>PROTOCOL-2 <EM><U>Picture capture</U></EM></STRONG></P><P>Preparation:</P><OL><LI><P>Light source: lamp</P></LI><LI><P>Frame: card-board box (e.g.with size37cm*28cm*22cm)</P></LI><LI><P>Background curtain:  light-absorb cloth</P></LI><LI><P>Picture capture device: web  camera</P></LI><LI><P>Computer: desktop all laptop</P></LI></OL><P>Location: warm room (37 ℃) 
                            Device setting: (Fig.1.) 
                            The lamp should be fixed on  the wall with its light shooting directed to the wall. The reflected light will  then enter the dark box with the light-absorb cloth wrapped inside. Drill  hole(s) on the top of the box whose size is fit for the camera lens. A box of  our size can hold up to 4 web cameras to take picture simultaneously.
                            The light-absorb cloth is simply stapled to the inside wall of the  card-board box. It will largely reduce the light refraction inside the dark box  as well as give a high contrast background to the bacteria pattern which is  white in observation.<EM>(If  the light intensity is still too high, a piece of normal A4 paper or paper  tissue can be used as a mask on the open side of the box.)</EM></P><P>Picture  capturing:
                            Software used for monitoring  and capturing pictures may be differed from various web cameras made from  different companies. Since the web camera we purchased is a product of <EM>Vimicro</EM>, we basically used their  software <EM>Vimcam </EM>(which can be found <A href="http://www.vimicro.com/english/">here</A>) for the analysis.</P><P>Since we may want to take picture for  the same plate repetitively in a fixed time interval and a long time duration,  we used a automatic approach with the help of a simple Java program. (You can  find the Java code and a simple  description here) The Java  program is will simulate a robot who will click and release the left mouse  button for a given time interval and this program will stop after a certain  time duration.</P><P align="right"><A href="#top">[Back to Top]</A></P><P><STRONG>PROTOCOL-3 <EM><U>Brightness cell concentration relationship Measurement</U></EM></STRONG></P><P>Day 1 –Streak the strand of cells to be tested on agarose plate to  form single colony</P><P>Day 2 – Start seed culture</P><OL><LI><P>Morning: inoculate a single  colony of cells (or from a patch amplified from a single colony) into X ml LB.  Incubate on shaker at 250 rpm at 37oC</P></LI></OL><P>Day 2 – Start pre-culture before leaving
                        2)  Afternoon or evening: centrifuge the LB seed culture at RT; drain the  medium as clean as possible; resuspend the cells with the culture medium using  conical flusk with as large volume as possible, I liter or 500 ml flusk is  recommeded (it’s better to centrifuge and resuspend for another time); measure  OD; inoculate the seed cells into 1/4 the flusk volume’s (exp, 250ml for 1  liter flusk) minimal medium (2% LB medium)</P><P>Day 3 – Start experimental culture after  arriving</P><OL><LI><P>Morning: Prepare 200 ml LB-Agar with 2% LB and 0.8% Agar and  autoclave, put on 50oC water bath to keep it in liquid form.</P></LI><LI><P>Centrifuge the pre-culture at RT; drain the medium as clean as  possible; resuspend the cells into 100 ml culture medium (2% LB medium), shake  it on votex to make sure cells are separated thoroughly and unified throughout  the medium, measure OD(if OD is much higher than 1, then dilute with power of 2  and measure).</P></LI><LI><P>Prepare several clean 20 ml tubes. Pull 10 ml bacteria LB-Agar  mixture with cell concentration gradient onto plates                            </P></LI><LI><P>Add different volume of  bacteria into 20 ml tubes. (exp: 1ml, 2ml, 3ml, 4ml, and 5ml)</P></LI><LI><P>Add 2% LB medium to the 20 ml  tube to fill the solution to 5 ml</P></LI><LI><P>Add 5 ml of LB-Agar and pull  plate sequentially.</P></LI></OL></TD></TR><TR><TD width="58" valign="top"><P>Tube</P></TD><TD width="66" valign="top"><P align="center">1</P></TD><TD width="66" valign="top"><P align="center">2</P></TD><TD width="66" valign="top"><P align="center">3</P></TD><TD width="66" valign="top"><P align="center">4</P></TD><TD width="66" valign="top"><P align="center">5</P></TD><TD width="66" valign="top"><P align="center">6</P></TD></TR><TR><TD width="58" valign="top"><P>Bacteria(ml)</P></TD><TD width="66" valign="top"><P align="center">0</P></TD><TD width="66" valign="top"><P align="center">1</P></TD><TD width="66" valign="top"><P align="center">2</P></TD><TD width="66" valign="top"><P align="center">3</P></TD><TD width="66" valign="top"><P align="center">4</P></TD><TD width="66" valign="top"><P align="center">5</P></TD></TR><TR><TD width="58" valign="top"><P>Cell(ml)</P></TD><TD width="66" valign="top"><P align="center">5</P></TD><TD width="66" valign="top"><P align="center">4</P></TD><TD width="66" valign="top"><P align="center">3</P></TD><TD width="66" valign="top"><P align="center">2</P></TD><TD width="66" valign="top"><P align="center">1</P></TD><TD width="66" valign="top"><P align="center">0</P></TD></TR><TR><TD width="58" valign="top"><P>LB-Agar(ml)</P></TD><TD width="66" valign="top"><P align="center">5</P></TD><TD width="66" valign="top"><P align="center">5</P></TD><TD width="66" valign="top"><P align="center">5</P></TD><TD width="66" valign="top"><P align="center">5</P></TD><TD width="66" valign="top"><P align="center">5</P></TD><TD width="66" valign="top"><P align="center">5</P></TD></TR></TBODY></TABLE><BLOCKQUOTE><P> After the  mixture, the Agar Concentration will become 0.4%, which equals to the agar  concentration where cells can swim into the gel, and contain different  concentration of cells.</P></BLOCKQUOTE><UL><UL><LI><P>Fix a plate cover below the  camera, then place the plates upside down on the cover in order to keep  position fixed, and take photos one by one.</P></LI><LI><P>Measure the average brightness  of each plate.</P></LI></UL><P align="right"><A href="#top">[Back to Top]</A></P></UL><P><STRONG>PROTOCOL-4 <EM><U>Migration rate measurement</U></EM></STRONG></P><P>Day 1 –Streak the strand of cells to be tested on agarose  plate to form single colony</P><P>Day 2 – Start seed culture</P><OL><LI><P>Morning: inoculate a single  colony of cells (or from a patch amplified from a single colony) into X ml LB.  Incubate on shaker at 250 rpm at 37oC</P></LI><LI><P>Prepare 100 ml LB-Agar with 2%  LB and 0.4% Agar and autoclave, put on 50oC water bath to cool down,  and pull plate.</P></LI></OL><P>Day 2 – Start experimental culture</P><OL><LI><P>Afternoon or evening:  centrifuge the pre-culture ar RT; drain the medium as clean as possible;  resuspend the cells with the culture medium (it’s better to centrifuge and  resuspend for another time); measure OD; When it is below 1.000 inoculate the  pre-culture cells into 5 ml minimal medium</P></LI><LI><P>When OD reaches 1.000, use  pipette to drop 2 μl bacteria solution onto the centre of the prepared plate.</P></LI><LI><P>Rotate the plate upside down  and fix it below the camera, use automatic Photo taking to capture Cell  migration status over-night.</P></LI></OL><P>Day 3 – Collect  data</P><OL><LI><P> Afternoon: 24 hours after photo  taking, retrieve the image data and measure the brightness of whole plate along  time course to analyze the migration rate.</P></LI></OL><P align="right"><A href="#top">[Back to Top]</A></P><P><STRONG>PROTOCOL-5 <EM><U>Harvest sample from agar plate</U></EM></STRONG></P><OL><LI><P>Culture of the <EM>E.coli</EM> on  the 35mm petradish</P></LI><OL><LI><P>Measure the OD600 of the cells</P></LI><LI><P>Autocleave the LB–agar and let it cool down to 42’C in water bath</P></LI><LI><P>Mix the cells with the agar to a final concentration of OD600 0.01</P></LI><LI><P>Pipet 1ml mixture into each 35mm petradish</P></LI><LI><P>Let grow at 37’C </P></LI></OL><LI><P>Harvest the agar from the petradish </P></LI><OL><LI><P>Remove the agar from the plate completely with the spoon made of  soft plastic(refer to the photo attached)</P></LI><LI><P>Wash the plate with 0.5ml PBS? </P></LI><LI><P>The agar and the washing PBS is to be collected into the 1.5mL  epindoff tubes? </P></LI></OL></OL><BLOCKQUOTE><P>&lt;Note&gt;</P></BLOCKQUOTE><UL><LI><P>Cool the agar to 42’C before mix it with cell otherwise the cell  will be killed.</P></LI><LI><P>Cool the plated agar to the room temperature before put into warm  room; otherwise the solidification of the agar will be prolonged.</P></LI><LI><P>While culturing, the plates should not be piled up vertically,  otherwise the temperature will be lower for the plates in the middle, which  will result in slow growth.</P></LI><LI><P>All agars should be transferred to the tubes for the accuracy of  later measurement.</P></LI></UL><P align="right"><A href="#top">[Back to Top]</A></P><P> </P><P>Regular Protocols</P><P><STRONG>PROTOCOL-6 <EM><U>β-galactosidase assay</U></EM></STRONG></P><P>
                        1. Sample  treatment
                        a)  Harvest the culture by centrifuge (13krpm*1.5min) followed by washing  with 1ml PBS twice
                        b)  Resuspend the pellet with PBS and protease inhibitor cocktail. 
                        c)  Sonication for (5.5seconds+1second pulse)*10 minutes 
                        d)  Centrifuge (13krpm * 1min)
                        2. ONPG  enzymatic reaction
                        a) Aliquot 200 microliter of treated sample  into the 96 wells plate
                        b) Add 40 microliter of ONPG(4mg/ml in ddw)  into each well
                        c)Read the OD420 every 10minutes for 5 minutes  at 37’C on the spectrometer 
                        3.  Trouble shooting
                        a) Raw reading beyond most accurate region of  o.2~0.4
                        b) Loss of activity due to high temperature or  malfunction of the protease inhibitor
                        c) Incomplete cell lysis because too short  sonication time or incomplete resuspension of the pellet
                        d) The temperature for the enzymatic reaction  should be 37’C, so the spectrometer should be prewarmed.&lt;Note&gt;
                        Must keep  the sample at low temperature to avoid the loss of enzymatic activity.</P><P align="right"><A href="#top">[Back to Top]</A></P><P><STRONG>PROTOCOL-7 <EM><U>Western blot</U></EM></STRONG></P><UL><LI><P>Buffer preparation                            </P></LI><LI><P>Rususpension buffer: 1ml stock  + 0.1ml protease inhibitor (10x)</P></LI><LI><P>Loading buffer: 1ml stock + 0.2  ml DTT + Phenol Blue </P></LI><LI><P>Sample treatment                            </P></LI><LI><P>centrifuge the culture into pellet (13k rpm 10min)</P></LI><LI><P>Resuspend the pellet with 5xVolume(pellet) Resuspension buffer</P></LI><LI><P>Add 5xVolume(pellet) loading buffer </P></LI><LI><P>Boil the resuspend for 10 minutes at 100’C</P></LI><LI><P>Centrifuge for 1 min at 13krpm</P></LI><LI><P>SDS-PAGE                            </P></LI><LI><P>15% separation gel</P></LI><LI><P>5% stacking gel</P></LI><LI><P>Load the sample and run under the constant voltage of 100V</P></LI><LI><P>Western blotting                            </P></LI><LI><P>Transfer from gel to membrane</P></LI><LI><P>Assemble &quot;sandwich&quot;  Transblot. </P></LI><LI><P>Prewet the sponges, filter  papers (slightly bigger than gel) in 1x Blotting buffer. </P></LI><LI><P>Transfer for 1 hr at 1 amp at  4°C on a stir plate. Bigger proteins might take longer to transfer. For the  Mini-Transblot, it's 100 V for 1 hr with the cold pack and prechilled buffer.  When finished, immerse membrane in Blocking buffer and block overnight.</P></LI><LI><P>Hybridization  with antibodies                            </P></LI><LI><P>Incubate with primary  antibody diluted in Blocking buffer for 60 min at room temp. </P></LI><LI><P>Wash 3 x 10 min with 0.05% Tween 20 in PBS. </P></LI><LI><P>Incubate with secondary antibody diluted in PBS for 45  min at room temp. </P></LI><LI><P>Wash 3 x 10 min with 0.05% Tween 20 in PBS. </P></LI><LI><P>Detect with SUPER SIGNAL WEST  PICO Kit (1ml luminol solution + 1ml stable peroxide solution</P></LI><LI><P>Result  analysis                            </P></LI><LI><P>The  size of the tagged protein can be determined by the marker (protein ladder)</P></LI><LI><P>The  amount of the protein can be estimated by the brightness of the band, or  accurately analysed by the software.</P></LI></UL><LI><P>Trouble  shooting                            </P></LI><LI><P>Unspecific  binding : 1st antibody-membrane and irrelevant protein to 1st  antibody</P></LI><LI><P>Film  over-exposure or lack of exposure</P></LI><P align="right"><A href="#top">[Back to Top]</A></P><P><STRONG>PROTOCOL-8 <EM><U>Recombineering</U></EM></STRONG></P><P>
                        The procedure  used was similar to that described by Watt <EM>et  al </EM><U>[<A href="#ref">1</A>]</U>. About 500                                                                      μl from overnight cultures [5     ml     L medium (containing antibiotic where applicable) inoculated  from single colonies, grown at 32°C for 18     h] was expanded into 50     ml of L medium in a 250     ml Erlenmeyer flask, and incubated at 32°C for 2     h (until OD600 of <EM>ca.</EM> 0.4–0.6). Flasks were transferred to a  shaking water bath at 42°C and incubated for 14–15     min, before cooling to 0°C as rapidly as possible in iced  water. After 15–20     min, cells were harvested by centrifugation at 0°C (4000     g, 9     min). Cell pellets were carefully washed three times with  sterilized ice-cold water (2     ×     50     ml, then 1     ×     1.5     ml) then re-suspended in 100–200     μl of ice-cold water. Competent cells (50     μl) were transformed with 50–200     ng of (gel purified) linear dsDNA targeting cassette using a  BioRad electroporator (1.8     kV, 25     mF, 200     W). The L medium (1     ml) was added to the transformed cell mixture, which was incubated  at 32°C, for 2–2?     h. Cells were collected by centrifugation, <EM>ca</EM>. 900     μl of supernatant media was discarded, and then the  resuspended cells were plated onto LB agar containing the appropriate  antibiotic to select for resistant colonies.</P><P align="right"><A href="#top">[Back to Top]</A></P><P><STRONG>PROTOCOL-9 <EM><U>Most Probable Number  method (MPN)</U></EM></STRONG>
                        Preparation:</P><OL><LI><P>A original undiluted cell culture sample (&gt; 1ml)</P></LI><LI><P>Distilled water</P></LI><LI><P>Vortex**</P></LI><LI><P>OD600 reader </P></LI><LI><P>Fresh medium (liquid or solid)</P></LI></OL><P align="left">Steps:</P><UL><LI><P>Measure and record the OD600 value of the original undiluted sample.</P></LI><LI><P>Shake the sample thoroughly and draw 100μl  into a new tube which contains 900μl water. Mark the new tube  with 1.</P></LI><LI><P>Repeat step 2 for several times according to the OD600 value which have  been recorded. Every time dilute the previous sample 10 times and mark the new  sample in an ascending order, i.e. 2, 3, 4… (Since it is commonly believed that  there are around 109 cells in 1ml sample with OD600 measured 1.00,  if this sample is used, loop for 9 or 10 times is needed)</P></LI><LI><P>Up to this step, a series of diluted samples should be obtained. Every  time draw 100μl from the sample and drop  it on a plate. For every sample, repeat this procedure for 5 separate plates.</P></LI><LI><P>Incubate the plates over night, at 37℃. </P></LI><LI><P>Record the outcome in the following format, in which 1 represents a  bacteria growth is observed while 0 means not.</P></LI></UL><UL><LI><P>Check up the <U><A href="http://www.jlindquist.net/generalmicro/102dil3a.html">MPN table</A></U> for the final result.</P></LI></UL><P>To see our result, please refer to the <U>RESULT</U> part of our wiki.</P><P align="right"><A href="#top">[Back to Top]</A></P><P><STRONG>PROTOCOL-10 <EM><U>Bradford assay</U></EM></STRONG></P><P>
                        1. Bacteria culturing
                          A 50 mL culture of EColi. was grown in 2%  LB media overnight (16 hours) in 37’C shaker. A fresh 50 mL LB culture was  inoculated with 0.2 mL of these cells and grown for 5 hours to an OD600 between  0.100 and 3.000. The cells were collected with centrifugation (Beckman,  13,000rpm, 2min, rt’C), washed twice with chilled 200 λ PBS Buffer, and  re-suspended in 1mL of chilled PBS Buffer.</P><P>
                          2. Protein quantization
                          Sonication Settings:
                          1,000 μl aliquots of re-suspended bacteria  were transferred to 1.4 mL polyethylene tubes tubes were subjugated to  sonication times of (10”+5”)*10cycles seconds at 40% power (≈4.8 watts/pin). The samples were then centrifuged at 13,000 rpm for  2 minutes to pellet debris. The supernatant containing solubilized proteins was  collected for analysis.
                          Or B-Per protein extraction  kit(recommended):
                          1000 μL aliquots of re-suspended bacteria  were centrifuged at 13k rpm for 10 min. to pellet cells. The cells were  resuspended in 200 μL of B-Per Reagent (Pierce, Rockford, IL) and vortexed for  1 minute corresponding to the Pierce protocol. 800 μl of PBS buffer was added  to the solution and the solution centrifuged 10min 13k rpm to pellet debris and  the supernatant collected for analysis
                          Protein Assay:
                          The amount of protein released after each  sonication time was qualitatively determined by use of Bradford Reagent. An  aliquot of 10 μl of each sample was mixed with 200 μl of 1:5Diluted Bradford  Reagent and the absorption at 595 nm recorded (CARY300 Bio UV-Visible  spectrometer) after 10~30 minutes of mixing time.</P><P align="right"><A href="#top">[Back to Top]</A></P><P> </P><P>Other Protocols</P><P><STRONG><U>Casting  agarose gels</U></STRONG></P><OL start="1" type="1"><LI><P>Assemble casting trays, gel box and gel comb, choose the comb of       appropriate width, size, and number into niche at end of the tray. </P></LI><LI><P>agarose solution(1% gel)：0.4g agarose + 40ml 1X TAE for small gels or       1g agarose + 100ml 1X TAE for large gels. </P></LI><LI><P>add agarose solution in a 500ml beaker. </P></LI><LI><P>Microwave bottle for 5min </P></LI><LI><P>Remove from microwave and let cool </P></LI><LI><P>Once gel is cooled so that it can be touched       comfortably with your gloved hand, add 4μL(for small gel) or 10 μL(for       large gels) EB dye. </P></LI><LI><P>Pour gel into casting trays. </P></LI></OL><P align="right"><A href="#top">[Back to Top]</A></P><P><STRONG><U>Running  agarose gels</U></STRONG></P><OL start="1" type="1"><LI><P>Place the gel in gel box. </P></LI><LI><P>Add enough 1X TAE to fill the reservoirs at both       ends of the gel box and cover the surface of the gel--the gel should be       immersed. </P></LI><LI><P>Mix6Х loading dye with samples: Typically, use 1 μL       loading dye per 5 μL of sample. </P></LI><LI><P>Load samples left to right. </P></LI><LI><P>Place gel box cover on gel box such that your       samples will run towards the positive, red electrode. Make sure that the       cables from the cover are connected to the power supply correctly. </P></LI><LI><P>Turn on the power supply and run your gel at ~120 V       for 30 mins. </P></LI><LI><P>Verify that bubbles are rising from the electrodes       once you start your gel to ensure your gel is running properly. </P></LI></OL><P align="right"><A href="#top">[Back to Top]</A></P><P><STRONG>Marker standard</STRONG>
                        We use 1kb plus DNA ladder.</P><P><U><STRONG>Quick  gel purifcation</STRONG></U>
                          according to <EM>QIAprep </EM>kit  protocol </P><OL start="1" type="1"><LI><P>Agarose gel electrophoresis, cut the DNA band as       thin as possible.Then put it into a clear EP tube. weighing,if the weight       is 100mg,the volume can be considered as 100uL,add QG beffer,put into 55℃ water bath until       complently merge, it takes about 10 minutes </P></LI><LI><P>cool the tube to room temperature </P></LI><LI><P>Add the solution into the collection tube, standing       for 1 minute,13,000rpm for 1 min, discard the flow-through </P></LI><LI><P>Add 650 μl WB,13,000rpm for 1 min, discard the       flow-through </P></LI><LI><P>Again</P></LI><LI><P>13,000rpm for 2 min, discard the flow-through </P></LI><LI><P>Put the collection tube into a clean EP tube, Add 30       μl ddH2O in the center of the collection tube, standing for 5min </P></LI><LI><P>13,000rpm for 2 min. </P></LI><LI><P>store the DNA solution at -20℃ </P></LI></OL><P align="right"><A href="#top">[Back to Top]</A></P><P><U>PCR  product purifcation</U>
                          according  to <EM>QIAprep</EM> kit protocol </P><OL start="1" type="1"><LI><P>Take 100uL PCR product solution, add 5 times volume Binding       Buffer,mix, add into spin column, standing for 2 min, 13000rpm for 1       min,discard the flow-through. </P></LI><LI><P>Add 650 mL buffer WB ,13000 rpm for 1min,discard the       supernatant </P></LI><LI><P>Again</P></LI><LI><P>13000rpm for 2 min to wipe out the remaining WB </P></LI><LI><P>Put the collection tube into a clean EP tube, Add 50uL 60 ℃ddH2O       in the center of the collection tube, standing for 15min </P></LI><LI><P>13000rpm for 12min. </P></LI><LI><P>store the DNA solution at -20℃</P></LI></OL><P align="right"><A href="#top">[Back to Top]</A></P><P> </P><P><U><STRONG>Count:  Preparation of Competent Cells for electro transformatio</STRONG></U></P><P><STRONG>Materials</STRONG></P><OL start="1" type="1"><LI><P>Media </P></LI><OL start="1" type="1"><LI><P>LB(Both liquid media and media containing agar. Add certain        antibiotic if it is necessary.); </P></LI></OL><LI><P>Buffers and Solutions </P></LI></OL><P>10% Glycerin. </P><OL start="3" type="1"><LI><P>Special Equipments </P></LI><OL start="1" type="1"><LI><P>EP tubes(1.5mL), micropipette tips, centrifugation        bottles(polypropylene tubes, 50mL), graduated flask(250mL*2, 5mL*1),        plates and test tubes. </P></LI></OL></OL><P><STRONG>Protocol</STRONG></P><OL start="1" type="1"><LI><P>Sterilization </P></LI><OL start="1" type="1"><LI><P>Including all materials in part2. </P></LI><LI><P>Caution (Remind): use some special marks to distinguish the        sterilized materials from the unsterilized ones. </P></LI></OL><LI><P>Preparation after sterilization </P></LI><OL start="1" type="1"><LI><P>Chill the 10%Glycerin to 4 centigrade degree </P></LI><LI><P>Decant LB containing agar into the plates.(If antibiotics are        necessary, be sure that they are added when the media temperature is below        60 centigrade degree.) </P></LI></OL><LI><P>Streak the prepared strains onto the agar plates. Then incubate       it at 37℃ for 10-16 hours. </P></LI><LI><P>overnight preculture</P></LI><LI><P>take? 0.5mL overnight       culture to 50mL LB bottle. </P></LI><LI><P>37℃ shaking 100~120 min to O.D. 600=0.45~0.6 </P></LI><LI><P>on ice for 30min </P></LI><LI><P>4000rpm 7min at 4℃ </P></LI><LI><P>Add origin volume 10% glycerol,suspend softly. </P></LI><LI><P>4000rpm7 min at 4℃</P></LI><LI><P>Add origin volume 10% glycerol,suspend softly. </P></LI><LI><P>Add origin volume 1/10 10% glycerol, suspend softly. </P></LI><LI><P>4000rpm7 min at 4℃</P></LI><LI><P>Add origin volume 1/100 10% glycerol, suspend softly, store at       -80℃.</P></LI></OL><P align="right"><A href="#top">[Back to Top]</A></P><P> </P><P><U>Electro tranformation</U></P><OL start="1" type="1"><LI><P>Hold       competent cells (from -80℃ refrigerator) on ice. </P></LI><LI><P>Gently mix       ligation product (1-5 μL) with cells. </P></LI><LI><P>transfer       the cell/DNA mix into an electroporation cuvette 
                          Note: the gene pulser should already be set properly
                          - time constant = 4.5 - 5.0 ms
                          - resistance = 200 W
                          - capacitance = 25 mFD
                          for 0.1 cm gap cuvettes, set the volts to 1.7 kV</P></LI><LI><P>pulse the       cells once; the voltage display blinks, and the gene pulser beeps</P></LI><LI><P>quickly       transfer 37 ℃ SOC to cuvette, mix by gently pipetting up and down, and       transfer SOC/cells back to culture tube</P></LI><LI><P>Bath in 37℃       for 60~90 min.? </P></LI><LI><P>Separate       cells on petri-dishes, and cultivate them in 37℃       for 12 hour.</P></LI></OL><P align="right"><A href="#top">[Back to Top]</A></P><P> </P><P align="left"><U><STRONG>Ligation</STRONG></U>
                        We use new England Biolabs T4 DNA  ligase </P><OL start="1" type="1"><LI><P>Choose       reaction volume: 5-10 μL </P></LI><LI><P>Mix proper       proportion (usually 3~5:1) of DNA fragment and vector. </P></LI><LI><P>Add       10×(meaning 1/10 of final volume) ligase buffer. </P></LI><LI><P>Add 0.5 μL       ligase per 10 μL final volume. </P></LI><LI><P>Bath in 16℃       water for 12 hour, </P></LI><LI><P>Begin <A href="http://www.openwetware.org/wiki/IGEM:Peking/2007/Transformation" title="IGEM:Peking/2007/Transformation">transformation</A>. </P></LI></OL><P align="right"><A href="#top">[Back to Top]</A></P><P> </P><P align="left"><U>DNA extraction</U>
                        Use QIAGEN kit</P><P align="left"><U><STRONG>Polymerase Chain Reaction</STRONG></U><STRONG>Choose enzymes</STRONG>
                        1.We use rTaq in testing but not in cloning genes. 
                        2.Ex Taq and LA Taq are modified Taq by Takara, they have 3’-&gt;5’exonuclease  activity and a relatively high fidelity. 
                        3.Generally speaking, genes shorter than 1000bp can be cloned by Ex  Taq. Genes shorter than 1.5kb can be cloned by LA Taq. 
                        4.Taq can add A at the end of each fragment, so their PCR product  can direcly link to T plasmid. <STRONG>PCR system</STRONG>
                        Normal Taq polymerase reaction component has a template of 1ng~1ug,  dNTP 200uM each，primer 50pmol each，polymerase 1U，buffer and  water。Template can be plasmid，colony and genomic DNA. <STRONG>Taq  series polymerase reaction components</STRONG><STRONG>Colony PCR</STRONG>
                        Template:Bacterial Colony 
                        10×PCR rTaqbuffer 2uL 
                        10×dNTP 1.6uL 
                        Primer-F 0.1uL 
                        Primer-R 0.1uL 
                        rTaq 0.2uL 
                        ddH2O 16uL 
                        20uL system <STRONG>Cloning</STRONG>
                        Template 1uL (above 10ngDNA)
                        10×PCR buffer（Ex/LA） 5uL 
                        10×dNTP 4uL 
                        Primer-F 1uL 
                        Primer-R 1uL 
                        Ex/LA Taq 0.25uL 
                        ddH2O 37.75uL 
                        50uL system <STRONG>reaction condition</STRONG>
                        1．95℃ 5min  Taq enzyme activation by heat 
                        2．95℃ 30s  DNA denaturing 
                        3．Tm-5~10℃ 30s Tm  is annealing temperature, with a range of 45～60℃ 
                        4. 72℃ ETs ET is elongation time，1kb/min 
                        28cycles 
                        5. 72℃ 10min add A at the end of each fragment </P><P align="right"><A href="#top">[Back to Top]</A></P><P align="left"> </P><P align="left"><U><STRONG>Plasmid digestion</STRONG></U><STRONG>Digestion temperature</STRONG>
Most endonuclease has the optimal activity temperature of 37℃. <STRONG>enzymatic system</STRONG>
Includs plasmid/PCR product, enzyme, buffer and water, sometimes  with BSAin it. The system for enzymatic test should be no more than 20uL. The  system for recruiting vectors and fragments should be 50-100uL. <STRONG>DNA  concentration in digestion system</STRONG>
The vector concentration should be no more than 1ug, PCR fragment  concentration should be no less than 1ug. <STRONG>Star activity</STRONG>
Some endonuclease will have a less specific substrate selection  under certain conditions. It will cut different sequences from original  recognitions. This is called star activity. 
In order to reduce the star activity, glycero concentration should  not be too high and DNA concentration should not be too low. In the storage of  enzyme there is glycerol. So the maximal amount of enzyme added in a system  should be 10%, especially for enzymes with star activities. <STRONG>The influence of  Methylation</STRONG>
Some endonuclease activity will be affected by methylation of DNA. 
There is a methylation form we can refer to. <STRONG>Terminal  digestion of PCR product</STRONG>
The terminal digestion always has a low efficiency. Some additional  base pairs are always added at the end of PCR product during primer design in  order to improve the digestion efficiency. 
There is a form about PCR terminal digestion. <STRONG>Inactivation</STRONG>
After the digestion, endonuclease must be inactivated before applied  to ligation system. Different enzymes have different inactivation condition. 
There is a form about inactivation conditions of all usual  endonuclease. <STRONG>Digestion Timing</STRONG>
1 hour enables a test digestion. If digestion is used for cutting  vectors and PCR fragments, in order to digest thoroughly, about 16 hours is a  relatively good choice. However, some endonuclease used in the digestion system  inactivates quickly. So we need add new enzymes at intervals during digestion. 
Refer to the form of different endonuclease about their remaining  activity after different periods of time. </P><P align="right"><A href="#top">[Back to Top]</A></P><P align="left"> </P><P align="left"> </P><P align="left"> </P><P align="left"> </P><P align="left"> </P><P align="left"> </P><P align="left"> </P><H3 align="left">Reference</H3><OL><LI><P>Watt RM, Wang J, Leong M,  Kung HF, Cheah KS, Liu D, Danchin A, Huang JD. Visualizing the proteome of  Escherichia coli: an efficient and versatile method for labeling chromosomal  coding DNA sequences (CDSs) with fluorescent protein genes. Nucleic Acids Res. 2007,  35(6):e37.                             </P></LI></OL><P align="left"> </P><P> </P> </DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:iHKU/protocol">http://2008.igem.org/Team:iHKU/protocol</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:iHKU/protocol" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:iHKU/protocol" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:iHKU/protocol&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:iHKU/protocol&amp;oldid=94131" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>