"
Page
Discussion
View source
History
teams
Log in
 
Team:iHKU/biobrick
From 2008.igem.org
 
The University of Hong Kong | Li Ka Shing Faculty of Medicine
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
       
 
Biobrick
CONTENTS:
List of Biobricks:
BBa_K094100 , BBa_K094101 , BBa_K094102 , BBa_K094103 , BBa_K094104 , BBa_K094105 , BBa_K094106 , BBa_K094110 , BBa_K094111 , BBa_K094112 , BBa_K094113 , BBa_K094120 , BBa_K094130 , BBa_K094141 , BBa_K094150 , BBa_K094151
Characterization of BBa_pSB2K3
 
 
 
 
 
List of Biobricks:
BBa_K094100
cheZ: This part contains the cheZ operon  coding region without an rbs nor terminator. cheZ protein is responsible for  the dephosphorylation of cheY protein in bacteria flagella movement.
[Back to top]
 
BBa_K094101
plambda(R-O12)-rbs-cheZ: This part contains  a lambda promoter that is constitutively on and can be repressed by CI protein.  cheZ protein is responsible for the dephosphorylation of cheY protein in  bacteria flagella movement. This part lacks a terminator.
[Back to top]
 
BBa_K094102
pTetR-rbs-cheZ: This part uses BBa_J61003  as a backbone and cheZ as the inserted gene. cheZ protein is responsible for  the dephosphorylation of cheY protein in bacteria flagella movement. The cheZ  operon lacks a terminator.
[Back to top]
 
BBa_K094103
plux-rbs-cheZ-terminator: This is a cheZ gene  that can be induced by luxR protein under high AHL concentration. cheZ protein  is responsible for the dephosphorylation of cheY protein in bacteria flagella  movement.
[Back to top]
 
BBa_K094104
pLacI/ara-1-rbs-mRFP-double terminator:  This part is adapted from part BBa_J04450 by changing the promoter pLacI to  promoter pLacI/ara-1. This part can test the leaky of promoter plac/ara-1.
[Back to top]
 
BBa_K094105
plambda(R-O12)-rbs-cheZ:-rbs-mRFP1-double  terminator: This part can be used to measure the gene expression of cheZ under  the control of promoter plambda(R-O12).
[Back to top]
 
BBa_K094106
Plux-rbs-CI-terminator-plambda-rbs-cheZ:  This part encodes a CI protein that is used to repress lambda promoter. The  part is controlled by the lux promoter that can be induced by luxR protein  under high AHL concentration. In this part, the expression of cheZ is repressed  by increasing AHL concentration in the environment in the presence of luxR  protein. In other circumstances, the cheZ gene express constitutively.
[Back to top]
 
BBa_K094110
cheY*: This is the gene coding for cheY  protein which controls flagella movement. This sequence, however, is mutated to  give a mutant cheY protein which will not respond to the presence of cheZ  protein. This part contains the coding region only.
[Back to top]
 
BBa_K094111
pTetR-rbs-cheY*: This part uses BBa_J61003  as a backbone and cheY* as the inserted gene. cheY is the gene coding for cheY  protein which controls flagella movement. This sequence, however, is mutated to  give a mutant cheY protein which will not respond to the presence of cheZ  protein. This part lacks a terminator.
[Back to top]
 
BBa_K094112
luxR-rbs-luxI-terminator: : The bacterial luciferin - luciferase system is encoded by a set of genes labelled the Lux operon . In V. fischeri five such genes (LuxCDABE) have been identified as active in the  emission of visible light, and two genes (LuxR and LuxI) are  involved in regulating the operon . This  part can be utilized as a part of quorum sensing system.
[Back to top]
 
BBa_K094113
lacZ: This is a reporter gene that codes  for beta galactosidase.
[Back to top]
 
BBa_K094120
pLacI/ara-1: This promoter can be repressed  by lacI protein and induced by either IPTG or arabinose.
[Back to top]
 
BBa_K094130
rbs-mRFP1-terminator: This part is adapted  from part BBa_J04450, with the pLacI promoter deleted. This part can serve as  promoter efficiency test kit.
[Back to top]
 
BBa_K094141 <
pLacIq-rbs-LacIq-terminator: LacIQ can  repress promoter lac. This part contains a mutated pLacIq promoter which  enables high level of transcription. This ensures effective repression of pLacI  promoter that might be present.
[Back to top]
 
BBa_K094150
pLacI/ara-1-rbs-luxI-luxR-terminator: This  is a quorum sensing composition. Induced by IPTG or arabinose, this part can  establish a quorum sensing system, with the luxI protein producing AHL and luxR  protein as the AHL density sensor.
[Back to top]
 
BBa_K094151
pLacI/ara-1-rbs-luxI-terminator: The luxI  operon is controlled by promoter LacI/ara-1.
[Back to top]
 
 
 
 
Characterization: Effect of IPTG inducement on  BBa_pSB2K3 copy number
Plasmid pSB2K3 is a variable copy number  plasmid that carries one F’ replication origin and a P1 lytic replication  origin. The P1 lytic origin must be activated by proteins encoded by a gene  carried by the plasmid itself. The gene, however, is controlled by a pLacI promoter  and is thus inducible by IPTG. When the plasmid is inserted into cells  producing lacI protein, the P1 lytic origin is non-functional and its only  active origin is the F’ origin that keeps plasmid copy number at a very low  level, possibly one or two (which is determined by the nature of F’ origin). When  induced by IPTG, the P1 lytic origin is activated and thus may keep copy number  at a very high level.
To find out the relationship between the concentration  of inducing IPTG and the copy number of the plasmid, we performed a series of  experiments.
Method:
The common method for plasmid copy number  determination is using real-time PCR. However, real-time PCR is not available for  our experiment, so the method we used was plasmid extraction,  gel-electrophoresis followed by brightness analysis.
Procedure:
1. Pre-culture of pSB2K3 containing cells in 2% LB medium with Kanamycin concentration of 50mg/L. (not induced with IPTG)
2. When O.D. has reached 1, begin the experimental culture with a 1:100 dilution of the pre-culture into 2% LB medium containing Kanamycin and IPTG. 10 different IPTG concentrations were selected and they are 0, 10, 20, 40, 60, 80, 100, 200, 500, 1000 (unit: μmol/L).
3. Wait until the experimental culture has reached an O.D. close to 1. Measure the O.D. and calculate the volume of sample to be collected to make it contain the same cell mass as 5ml of O.D.=1.000 culture. Then collect the 10 samples and put them on ice.
4. Centrifuge down the cells and extract plasmid using a QIAGEN extraction kit. The extraction is done with extreme carefulness and strict protocol to ensure a high recovery and high repeatability.
5. The extracted plasmid is run with 1.0% agarose gel. 1kb plus DNA ladder is used as a control of DNA amount and plasmid size. The plasmid is digested with XbaI before gel electrophoresis.
6. Photo of the gel result is analyzed by brightness measuring software.
Rationale and  result interpretation:
The experiment is repeated twice (altogether three  times) to ensure the repeatability of this method. The values shown in the  result chart (figure 1) are the mean values of the three experiments with error  bar. Figure 2 is an enlarged part of figure 1 showing the first two data.
Figure 1
Figure 2
Six data are shown in the chart, because for IPTG  concentration larger than 100 μmol/L the effect of inducement seems to have no  difference. Attached in the chart is one of the original gel photos, with C  being the control. This data is got from exponentially growing culture, in  which the mean copy number is generally smaller than the mean copy number of  cells from stationary phase, for the simple reason that in newly divided cells  there is not enough time for plasmids to replicate to the limit.
The copy number is calculated from the following  statistics: cell number at O.D.=1 from our experimental result; plasmid molar  weight calculated from mean nucleotide weight and the size of the plasmid;  volume of cell culture sample and final volume of plasmid solution eluted (50 μl); the DNA amount  in 1kb plus ladder (control) and the light intensity got from the gel photo by  brightness analysis software. The calculation is very complicated and is  omitted here.
To make sure that the brightness of DNA bands  corresponds with the amount of DNA in it, control experiments with 1kb plus  ladder of different amount to see if there amounts are proportional to their brightness  intensity. The result is shown in figure 3 and we can see that the result is  quite linear. The experiment is also repeated twice and the data is the mean  value with error bar.
Figure 3
The result shows that for IPTG concentration less than  20 μmol/L,  the effect of inducement is scarcely perceivable. It also shows that from 20 μmol/L to 100 μmol/L the copy  number is induced by IPTG in a way quite proportional to its concentration. It  seems that over 100 μmol/L IPTG concentration is “saturated” and a further  increase in IPTG concentration cannot raise the copy number (the data of which  is not shown in the chart).
Attention*: This result is from exponentially growing  culture. For cell culture in stationary phase the copy number possibly  increases (due to time limit, only one experiment is done with stationary phase  culture and the result seems to suggest that this raise in plasmid copy number  is more significant with lower IPTG concentration. Because there are no  repeats, the data is not included).
According to the experiment, we suggest that 100 μmol/L IPTG is  enough for pSB2K3 to reach its peak copy number, and if moderate copy number is  needed then 40-60 μmol/L is a good choice. This result is also valuable  when using pSB2K3 with other IPTG induced genetic devices.
[Back to top]
 
Retrieved from " http://2008.igem.org/Team:iHKU/biobrick "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
