"
Page
Discussion
View source
History
teams
Log in
 
Team:iHKU/home
From 2008.igem.org
 
The University of Hong Kong | Li Ka Shing Faculty of Medicine
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
       
 
 
 
Formation of new patterns
by
programming  cell motility
Abstract
The ability of living organisms to form patterns is an untapped resource for synthetic biology. The HKU iGEM2008 team aims to generate unique patterns by rewiring the genetic circuitry controlling cell motility. Specifically, E. coli cells are programmed to autonomously regulate their movement by sensing local cell density. Interesting patterns are formed by two types of newly engineered cells. The low-density mover cells spread outwards and spontaneously form a distinctive ring of low cell density surrounded by rings of high cell density whilst the high-density mover cells form a Mt.Fuji-like structure. Moreover, we build a theoretical model that satisfactorily fits our current experimental data, and also predicts some parameters which may significantly affect the ring formation. The study of this self-organized spatial distribution of cells helps us to understand principles underlying the formation of natural biological patterns, and synthetic non-natural patterns have various potential applied uses.
[Back to top]
 
Overview
The iGEM2008 iHKU team aims to  deliver a brilliant project this year. We major in multiple disciplines  including Biochemistry, Bioinformatics, Physics, and Chemistry. Using our  different backgrounds and modalities of thought, we complement each other in  developing new ideas, and in carrying out wet/dry lab work ( Team ).
Pattern formation is one of the  most common yet fascinating biological phenomena happening in our daily lives,  though for centuries, biologists, physicists and mathematicians have struggled  to understand its nature. How do highly ordered patterns arise from a few  living cells? How can our hands, our eyes, our bones form their shape with such  low error rates? These questions are fascinating and crucial. The fundamental  elements in biological pattern formation are cell growth, cell movement,  cell-cell communication, and differential gene expression. In this project, we  aim to form new patterns by controlling cell movement, using a single strain of engineered bacteria. Bacterium E. coli was chosen as our  model system. E. coli cells use their flagella to move around. To  generate a recognizable and stable pattern, bacterial motility must be  controlled and coordinated. This can be accomplished by designing genetic  circuits coupling bacterial quorum sensing system and genes controlling  mobility. There are several key genes responsible for the movement of flagella,  two of them are cheY and cheZ. CheY protein has two forms: its  phosphorylated form makes flagella rotate clockwise and the cell will tumble;  its dephosphorylated form makes flagella rotate counterclockwise and the cell  will be driven straight in one direction (run). The CheZ protein is involved in  dephosphorylation of protein CheY. It is known in the literature that cells are immobile in the absence of CheZ.
By rewiring the genetic circuitry  that controls cell motility, we aim to generate unique patterns ( Design ). First, we applied the method of  Recombineering to delete the cheZ gene in chromosome of wild type E.  coli strain, MG1655 ( Protocols ).  Then, a series of biobricks and strains were successfully constructed ( Plasmids and strains ), to (i) import the quorum sensing system of Vibrio fischeri and (2) control the  expression of cheZ by quorum sensing, either positively or negatively. As expected,  interesting patterns were observed ( Results ),  such as Mt.Fuji-like and ring of void patterns, when cells are seeded on low density agar plates. Since the ring-like patterns are so counterintuitive -- they formed despite cell growth and diffusion, our remaining work mainly focused on the characterization  and modeling of these patterns.
Considering E.  coli movement as a random walk, a coarse-grained model was used to describe the basic cell motility in response to AHL (the quorum sensor) synthesized by the cells. Our model is  based on time dependent partial differential equations including the effect of  cell random walk, cell growth, AHL diffusion, AHL synthesis and degradation,  and nutrient diffusion and consumption. ( Modeling ). Analysis of the model suggests a basic mechanism underlying the formation of the intriguing trough-like patterns: regions of low cell density are formed by cells being sucked to  neighboring high density regions as the engineered cells move randomly at low density and become immobile  at high density. Our simulation indicated some factors might significantly affect the  development of ring-like patterns, such as the growth rate of the cell which  was also observed in the experiments. By measuring these factors, we provide  not only solid data to support our hypothesis for our model, but also the  values of the parameters involved ( Results ).  As a result, we were able to achieve a double-ring pattern by modifying the cooperativity of the response to AHL upon adding positive feedback to the genetic circuit. ( Results ).
During the experiments, we encountered numerous challenges and difficulties. To overcome them, we created several NOVEL  protocols, software, and devices with the help of our knowledge from different  fields, such as “growth curve on agar plate” ( Protocols ), “movie taker”, and “reflection spectrophotometer” ( Novel  devices ). We believe more researchers will benefit from our  inventions.
Last but not least, in this  project, we created 15 biobricks and characterized one existing biobrick ( Characterization ), which are considered to  be helpful to future iGEM competitions and the study of synthetic biology.
 
[Back to top]
 
 
Acknowledgement
We thank Dr LingChong You, California Institute of Technology, for providing the plasmid pluxRI2, and thank Dr Ron Weiss, Princeton University, for providing the plasmid pLD.
Sponsors
 
 
 
Retrieved from " http://2008.igem.org/Team:iHKU/home "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
