<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_iHKU_result"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:iHKU/result">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:iHKU/result&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:iHKU/result&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:iHKU/result&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:iHKU/result" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:iHKU/result</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV id="main"><DIV id="up1"><TABLE width="100%" border="0"><TBODY><TR><TH width="61%" scope="row">&lt; </TH><TD width="39%"><SPAN class="headline"><A href="http://www.hku.hk">The University of Hong Kong</A> | <A href="http://www.hku.hk/facmed/">Li Ka Shing Faculty of Medicine</A></SPAN></TD></TR></TBODY></TABLE></DIV><DIV id="down1"><DIV id="down1left"><DIV id="menu"><TABLE width="100%" border="0"><TBODY><TR><TH width="7%" height="335" scope="row"> </TH></TR><TR><TH width="5%" scope="row"> </TH><TD width="5%"> </TD></TR><TR><TH scope="row"> </TH><TD> </TD></TR><TR><TH scope="row"> </TH><TD> </TD></TR><TR><TH scope="row"> </TH><TD> </TD></TR><TR><TH scope="row"> </TH><TD> </TD></TR><TR><TH scope="row"> </TH><TD> </TD></TR><TR><TH scope="row"> </TH><TD> </TD></TR><TR><TH scope="row"> </TH><TD> </TD></TR><TR><TH scope="row"> </TH><TD> </TD></TR><TR><TH scope="row"> </TH><TD> </TD><TD> </TD></TR></TBODY></TABLE><P class="style15"> </P>    </DIV></DIV><DIV id="down1right"><DIV id="rightmain"><DIV id="content1"><TABLE width="100%" border="0"><TBODY><TR><TH width="10%"> </TH><TD width="80%" align="left" background="#1"><H1 class="style7">Results</H1><H2>CONTENTS:</H2><UL><LI class="style18"><A href="#1">Generation of <EM>cheZ</EM> knock-out strain by using λ-Red  mediated deletion strategy</A></LI><LI class="style18"><A href="#2">Introduction of <EM>cheZ</EM> restored the motility of  strain iHKU101</A></LI><LI><SPAN class="style18"><A href="#3">Pattern formation</A></SPAN></LI><LI class="style18"><A href="#31">Circle pattern from MG1655  (wild type Control)</A></LI><LI class="style18"><A href="#32">Mt.Fuji  pattern from iHKU105</A></LI><LI class="style18"><A href="#33">Ring-like  pattern</A></LI><LI class="style18"><A href="#4">Effect of genetic circuit modification on pattern formation</A></LI><LI class="style18"><A href="#fun">Multiple initial spot</A></LI></UL><LI class="style18"><A href="#5">Importance of genetic circuits in pattern formation</A></LI><LI><SPAN class="style18"><A href="#char">Characterization of factors controlling ring-like pattern  formation</A></SPAN></LI><LI class="style18"><A href="#61">Effects of  temperature</A></LI><LI class="style18"><A href="#62">Effects of humidity</A></LI><LI class="style18"><A href="#63">Effects of agar hardness</A></LI><LI class="style18"><A href="https://2008.igem.org/Team:iHKU/modeling">Parameters  controlling ring-like pattern formation in modeling</A></LI><LI><SPAN class="style18"><A href="#7">Quantitative measurements</A></SPAN></LI><LI class="style18"><A href="#71">Growth  rate in batch culture</A></LI><LI class="style18"><A href="#72">Growth  rate on agar plate</A></LI><LI class="style18"><A href="#73">Front propagation speed on agar plate</A></LI><LI class="style18"><A href="#74">Levels of LacZ and  corresponding growth curves</A></LI><LI class="style18"><A href="#movie">Movies</A></LI><H1> </H1><P class="special"><STRONG>Generation of <EM>cheZ</EM> knock-out strain by using </STRONG><STRONG>λ-Red mediated deletion  strategy</STRONG></P><P>We first constructed a plasmid  (PBSKcl) that contained a chloramphenicol (Cm) resistance module as a  selectable marker. These were used as templates for the PCR amplification of  DNA ‘targeting cassettes’, to delete the CDS of interest. PCR primers each  included 45–50 bases homologous to the chromosome immediately upstream (forward  primer, sense strand) and downstream (reverse primer, antisense strand) of <EM>cheZ</EM> gene, as well as 17 bases  homologous to the plasmid template. 
We introduced the λ-Red  recombination plasmid (pSim6)[<A href="#ref1">1</A>] into MG1655, namely MA6 for our chromosomal deletion.  In MA6, exogenous DNA was efficiently introduced by electroporation, and  integrated into the chromosome via a homologous recombination-based process  mediated by the Beta and Exo proteins, which are located in a defective cI857  λ-prophage. A schematic illustrating the procedure is shown in Fig.1A. After the cheZ gene was  replaced by selectable marker, cells was then introduced a temperature-inducible  copy of the <EM>cre</EM> site-specific recombinases, which enables efficient  removal of Cm resistance cassette from chromosome, generating the strain  iHKU101. The replacement and removal was confirmed by PCR (Fig.1B).</P><P><STRONG>Fig. 1 Diagram  illustrating the strategy used to delete <EM>cheZ</EM> (A) and agarose gel electrophoresis of confirmation PCR products (B).</STRONG> B, 1-4, MA6 (<EM>cheZ</EM> replaced with<EM> cm</EM>); 5, iHKU101; 6,  MG1655. </P><P>Thereafter,  we applied cells at log phase onto soft agar plate to test its motility. Cells lacking <EM>cheZ</EM> tumble incessantly and are essentially nonmotile, the spot expanded by only 0.3 cm (Fig2.B), while the parental strain, MG1655, has propagated across the plate (Fig2.A). </P><P align="center"><STRONG>Fig.2 Migration of strains MG1655 and iHKU101 18h after seeding. </STRONG>
A, MG1655; B, iHKU101</P><P align="right"><A href="#p1">[Back to top]</A></P><P align="center"> </P><P class="special"><STRONG>Introduction  of <EM>cheZ</EM> restored the motility of strain  iHKU101</STRONG>
                            To answer the  question whether or not introduction of<EM> cheZ </EM>gene back to iHKU101 could restore its motility, we introduced <EM>cheZ</EM> gene under the control of <EM>plac/ara-1</EM>, whose transcription is  induced by either IPTG or arabinose. SDS-PAGE and western blotting analysis  showed the high expression level of <EM>cheZ</EM> in the presence of IPTG or arabinose (Fig.3A). And arabinose exerted a higher  inductivity. We also observed a moderate amount of CheZ expression in absence  of any inducers, indicating the leaky of <EM>plac/ara-1</EM>.  This might due to the lack of <EM>lacIq</EM> in the chromosome of iHKU106. 
                            We then plated  the iHKU106 cells onto soft agar plate containing arabinose to test its  motility. As shown in Fig.3B, iHKU106 migrated about 4 cm (diameter) from the  center of the plate after 10 h incubation at 37C. </P><P align="center"><STRONG>Fig.3 SDS-PAGE and western blotting analysis of CheZ expression (A)  and migration of strains iHKU106 (B).</STRONG></P><P align="right"><A href="#p1">[Back to top]</A></P><P align="center"> </P><P class="special"><STRONG>Pattern formation </STRONG></P><P>We  confirmed motility is restorable (or inducible) in <EM>E. coli</EM>. We proceeded  to investigate whether a unique pattern can be obtained by our  density-dependent motility design. We constructed the 2 strains, the high-density mover and the low-density mover, as described in  the <STRONG><U><A href="design.html">Design</A></U></STRONG> Section and let  them, together with controls, to grow on soft-agar-plates following the<A href="https://2008.igem.org/Team:iHKU/protocol#p1">  “pattern development on soft-agar-plate” protocol</A>. Specifically, the strains  were pre-culture from overnight inoculums, when reach OD600 ~0.8, 2  μl of the culture was applied to the center of a soft-agar-plate and the plate  was subsequently incubated overnight. We obtained quite different patterns  amongst different strains.</P><P class="special"><STRONG>   Circle pattern from  MG1655 (wild type Control)</STRONG></P><P>In the case of the wild type, the obtained pattern was a circle (Fig. 4A). As can be seen in the density profile (Fig. 4B), cells are uniformly distributed across the area covered. Similar patterns were generated by the <A href="https://2008.igem.org/Team:iHKU/modeling"><STRONG><U>model</U></STRONG></A>(Fig. 4CD) and known as Fisher wave; see the movie below Fig.4 for the dynamics of the expanding bacteria population. </P><P align="center"><STRONG>Fig.4 Pattern of MG1655.</STRONG>
A, bacteria population on soft agar plate 12 hours after seeding at center; B, 3-D profile of A; C,  pattern of wild type cell by model; D,3-D profile of C.</P><H2 align="center"><A href="#mgvideo"><STRONG>Supporting videos</STRONG></A></H2><P align="left"> </P><P class="special"><STRONG>   Mt.Fuji pattern  from iHKU105</STRONG></P><P>But in the case of the high-density mover iHKU105(<U><A href="https://2008.igem.org/Team:iHKU/design">design</A></U>), a different pattern was observed  compared to the wild type strain. In particular, the center of the circle is  high in density (Fig. 5). Similarly, we got a pattern by our model, choosing an increasing function Dρ(h) (<STRONG><U><A href="https://2008.igem.org/Team:iHKU/modeling">modeling</A></U></STRONG>). </P><P align="center"><STRONG>Fig.5 Pattern of iHKU105. </STRONG>
A, soft-agar-plate; B, 3-D  profile of A<STRONG>.</STRONG></P><P class="special"><STRONG>   Ring-like pattern</STRONG>And in the case of iHKU102 (<U>design</U>), more distinctive  pattern was observed compared to the wild type strain. In particular, a region  of low cell density was observed (Fig. 6). We have termed the inner ring with  the lowest density the <STRONG>ring of void</STRONG> and its diameter is roughly 1.3cm. To note that the diameter of this ring of  void would not change once it formed.</P><P> In our model, if the Dρ(h)  decreased smoothly near the threshold, an amazing identical ring-like pattern  was observed (<STRONG><U><A href="https://2008.igem.org/Team:iHKU/modeling">modeling</A></U></STRONG>). </P><P align="center"><STRONG>Fig.6 Pattern of iHKU102. </STRONG>
                          A, soft-agar-plate; B, 3-D profile of A; C, modeling  pattern; D,3-D profile of C</P><H2 align="center"><A href="#102video"><STRONG>Supporting videos</STRONG></A></H2><P> </P><P align="right"><A href="#p1">[Back to top]</A></P><P> </P><P class="special"><STRONG>   Effect of genetic circuit modification on pattern formation</STRONG></P><P>As predicted by <U><A href="modelling.html">modeling</A></U>, if the graded change of <EM>cheZ</EM> expression to changes in cell density in iHKU102 is  turned into an abrupt response, the ring-like pattern formed by iHKU102 will be different, i.e. multiple rings are expected. This can be done by placing the expression of LuxI, LuxR, and the target proteins all under the control of pluxI as demonstrated by Haseltine <EM>et al.</EM>[<A href="#ref2">2</A>]. So in iHKU117, pluxRI2 was replaced with  pluxRI3, by which <EM>luxR </EM>and <EM>luxI </EM>are controlled by positive  auto-feed back. Interestingly, a different pattern really appeared. It’s  apparent that the inner area shrank, and there’s a circle of cells with high  density surrounding a ring of void. Outside this circle of cells, there’s the  other weaker ring of void (Fig. 7). </P><P align="center"><STRONG>Fig.7 Patterns of iHKU102 and  iHKU117. </STRONG>
                            A, soft-agar-plate;  B, 3-D profile of A; C, modeling  pattern of iHKU117; D,3-D profile of C.</P><P align="right"><A href="#p1">[Back to top]</A></P><P class="special"><STRONG>   Multiple initial spot</STRONG>
                            We tested patterns resulting from a variety of different initial seeding, e.g., two spots at different distances and 3 spots. As shown in Fig. 8, the patterns obtained (left panel) are in good agreement with the model prediction (right panel) in all cases.</P><P align="center"> </P><P align="center"><STRONG> Fig8. Multiple initial spot with two or more initial droplets in modelling(left)and in experiment(right)</STRONG></P><P class="special"><STRONG>Importance of genetic circuits in pattern formation</STRONG></P><P> To determine the importance of  introduced genetic circuit in this ring-like pattern, a series of control  strains were generated (<U><A href="https://2008.igem.org/Team:iHKU/design">Plasmids and strains</A></U>).  As shown in Fig.9, loss of any of the following elements -- <EM>luxR</EM> (iHKU104),<EM>luxI</EM> (iHKU103), <EM>CI</EM> (iHKU109), <EM>luxR</EM> and <EM>luxI</EM> (iHKU115), or CI and <EM>cheZ </EM>(iHKU118)-- resulted in the failure of development of ring-like pattern (iHKU102), suggesting that a complete genetic circuitry is required. </P><P align="center"><STRONG>Fig. 9 photograph of  control strains</STRONG></P><P align="right"><A href="#p1">[Back to top]</A></P><P class="special" align="left"><STRONG>Characterization of  factors controlling ring-like pattern formation</STRONG></P><P class="special"><STRONG>   Effects of  temperature</STRONG></P><P>We cultured  the strains at room temperature (23oC), 32oC, and 37oC  to investigate the effects of temperature on pattern development. Except for  the time to form the ring-like pattern, the overall pattern appeared to be  unaffected by temperature (data not shown).</P><P class="special" align="left"><STRONG>   Effects of humidity</STRONG></P><P>During our  experiments, we sought to determine the effects of humidity of the environments  on the pattern formation. When the humidity is too low, the water in the agar  plate will be evaporated into the air rapidly, the agar concentration will  rise, which will then cause the change of the<A href="modelling.html"><U>diffusion coefficient</U></A> of cells. 
In order to  control the humidity in a pattern formation favored range, we did some  experiments to try to control it. We used a big file box with 14 agar plates in  it in the 37 centigrade warm room, which is the normal experiment condition. At  the same time, plates with water (100 ml per plate) were put into the box too.  By changing the number of them, we attempted to control the humidity in the box  at different level. Each time, there were a fixed number of plates of water in  the box, after sufficient time, the humidity should be in equilibrium. We  expected to have different levels of humidity for different number of plates of  water.
We found that  the change of number of water plates do change the humidity in the box, however, there is only 10% increase in humidity (from 73% to 83%) in the presence of 4 water plates. At the  same time, to our surprise, we found that if the box was sealed tightly, even  without water plates in it, the humidity will reach ~80 %. We argue that this  may be due to water evaporation of the agar plates.</P><P align="center"><STRONG>Fig.10 effects of water amount on humidity</STRONG></P><P class="special"><STRONG>   Effects of agar hardness</STRONG></P><P>During our  experiments, we found that the agar concentration of the soft-agar plates is  an influential factor to the development of the pattern. We investigated this  factor by growing our strain on plates over a range of agar concentration. As  shown in figure 11, the ring of void becomes clearer and clearer as agar  concentration increases. This sheds light on<U><A href="https://2008.igem.org/Team:iHKU/modeling">cell diffusion</A></U>, since the denser the agar, the slower the cell diffusion through the matrix. </P><P align="center"><STRONG>Fig.11 effects of agar concentration on ring-like  pattern</STRONG></P><P align="right"><A href="#p1">[Back to top]</A></P><P align="center"> </P><P class="special" align="left"><STRONG>Quantitative  measurements</STRONG></P><P class="special"><STRONG>   Grwoth rate in batch  culture</STRONG></P><P class="special"><STRONG>   Growth curve of iHKU101</STRONG></P><P>iHKU101 is measured as a control for  the experimental strain. After two times of precultures, the initial cell  concentration was diluted to be OD600~0.05, and it entered exponential phase after two doublings and started to go into stationary phase after 100 minutes at OD600~1.6.
The doubling time is 20.03min  (fig.12), which is shorter than that of iHKU102 (<U>below</U>). </P><P align="center"><STRONG>Fig. 12 Growth curve of iHKU101 with the doubling time ~ 20.0 min</STRONG></P><P class="special"><STRONG>   Growth curve of iHKU102</STRONG></P><P>The doubling time of iHKU102 is  generally longer than iHKU101 (more than 30min) even without antibiotics (Fig.13).  The initial OD is also ~0.05 and it enters exponential phase immediately in the  experimental culture.</P><P align="center"><STRONG>Fig. 13 Growth curve of iHKU102. </STRONG></P><DIV align="center"><SPAN class="style17">A, The  doubling time ~ 33.6 min in absence of antibiotics, and B, ~36.5 min  in the presence of antibiotics.</SPAN></DIV><P class="special"><STRONG>   Growth rate on agar  plate </STRONG></P><P class="special"><STRONG>   Measured by total protein assay</STRONG></P><P>In the  measurement of the growth on plates, the total protein within the agar, which  is an index of the <EM>E.coli</EM> population,  was measured by (<U><A href="Protocol.html">Bradford  method</A></U>) for a time range of 12 hours with an interval of one hour.
We want to find out how the cell mass develops on the agar plate  over the time. Unlike the LB batch culture, which can be measured by OD600 directly,  to know the cell density on  soft agar  plate requires special measurements. One is total  protein and the other is the optical measurement. Since it is difficult to harvest sample of a chosen mass/volume, we harvest the whole agar from the 35mm petri-dish, which is of the exact volume of 1 ml. 
Since the cells are mixed evenly in the agar, it doesn’t resemble  the situation on plate for pattern formation. However, the similarity of the  trend of the curves from the optical methods and this total protein method  reassures the validity of the optical methods in the determination of cell  population on plate. What’s more, it provides information about the growth curve of specific strains/conditions, which is important in the modeling.</P><P align="center"><STRONG>Fig.14 Growth curve of iHKU102 on soft agar plate (total  protein).</STRONG></P><P>The total  protein at different time points was measured, which were converted to OD600 values (Fig.14), according to a formula generated before (data not shown). The first time point is before  the cells were added to the agar, and put on ice. However the reading is 0.8944, rather than zero, which may be due to the protein in the LB agar.  Therefore it is needed to subtract the blank from the following readings when estimating the cell population on the agar. Generally the curve increases over time, with one outliner of the third one, which also appears in the brightness  result. The total protein level reaches the maximal at 10 h, with a value about twice as much as the initial protein level, and then begins to drop. </P><P class="special"><STRONG>   Measured by brightness</STRONG></P><P>Another method  to measure cell growth on plate is by analysis of the <U>brightness</U>. This method  can measure the cell density at any specific location of the plate without the  need to harvest sample. However there is a drawback for this method -- variance  may be due to the uneven distribution of the light reflected from the plate.
To investigate  the relation between brightness and cell density, we captured the images of  plates with given ?cell density by using  our novel device (<U>movie  taker</U>). By calculating the brightness of each plate, we created a  graph with OD600 as X-axis and brightness intensity as Y-axis, and  obtained a formula of the trend line: y = 24.229ln(x) + 48.962 (Fig.15). Thus we can convert the brightness intensity value into the OD600 value.</P><P align="center"><STRONG>Fig.15 Relation between  brightness intensity and OD value.</STRONG></P><P align="left">A, photograph of plates containing inceasing cell  density; B, trend line and formula indicating the relation between brightness  intensity reflected from plates in panel A and OD600 value.</P><P>We attempted to test if these two  methods were comparable or not. So the optical analysis was conducted  simultaneously with the total protein analysis at each time point. For each  time point, we took a picture of the plate, and then analyzed the brightness at each pixel and added up for the total light intensity of the whole plate.  Finally the brightness intensity was converted into an OD600 value  and a growth curve was generated (Fig.16). </P><P align="center"><STRONG>Fig.16 Growth curve of  iHKU102 on soft agar plate (brightness).</STRONG></P><P>We next sought to measure the growth rate  of iHKU102 on ring-like pattern plate. Since the total protein method is  difficult to conduct in this case, we measure the brightness intensity of  different region on ring-like pattern plate. In figure 17B, the brightness  intensity of inner area increased in the first 15 h and then stayed at the peak  in following hours. The outer area appeared at 9 h and increased gradually over  time (Fig. 17C). </P><P align="center"><STRONG>Fig.17 Growth curve of iHKU102 on ring-like pattern  plate.</STRONG>
A,  photograph of the ring-like pattern, the inner and outer area are indicated; B,  growth curve of cells within the inner area; C, the outer area; D, both areas. </P><P class="special"><STRONG>   Front propagation speed on agar plate</STRONG></P><P align="left">By using automatic brightness measurement, Front propagation speed of iHKU102  was measured in the first 15 h. The Front propagation speed can be well deducted by  measuring the diameters of the inner and outer area edge. The Front propagation speed of iHKU102 on 0.45% agar plate was 1.33  mm/h (Fig.18A). We next measured the Front propagation speed of iHKU102 cells in agar  with lower concentration (<STRONG><U><A href="#63">Effects of  agar hardness</A></U></STRONG>), and found that the Front propagation speed in 0.45 % agar (<U>Difco</U>)  plate is 1.33 mm/h. It is much lower than that of 0.35 % agar (<U>Difco</U>)  plate (1.53 mm/h), in which no clear ring was observed (Fig.18B; Fig.11). This  data supports our hypothesis that Front propagation speed significantly affects the  development of ring-like pattern.</P><P align="center"><STRONG>Fig.18 Migration  rate of iHKU102 on ring-like pattern plate </STRONG></P><P class="special"><STRONG>   Levels of LacZ and corresponding growth curves. </STRONG></P><P>The expression  of <EM>cheZ</EM> was measured by LacZ assay. By  replacing pRg in iHKU105 with pRglacZ, we generated the strain iHKU114. In this  strain, the regulatory genes<EM> luxI</EM> (autoinducer  synthase gene) and <EM>luxR</EM> (autoinducer  receptor protein gene) are controlled by plac/area-1 and supposed to constitutively  express in the presence of arabinose or IPTG; the expression of <EM>lacZ</EM> gene are driven by pluxI. Consequently,  iHKU114 cells were expected to induce β-galactosidase  synthesis in a cell density-dependent fashion. This was found to be the case  (Fig.19A). <EM>lacZ</EM> expression was  studies along with the growth curve of strains (Fig.19B). The measured LacZ  activity was corrected by normalized to protein concentration and expressed in  fluorescence per mg. When iHKU114 cells were inoculated into fresh medium, β-galactosidase activity decreased at the early 40 min, rose to a  peak. It returned to the initial level at 70 min, and remained constant for 1.5~2  h during growth and was then induced to a level approximately 3-fold higher  than preinduction levels in following 5 h. These results are consistent with the  data reported previously [<A href="#ref3">3</A>], indicating a dependence on autoinducer  for pluxI transcription. 
Since the endogenous <EM>lacZ</EM> gene was not deleted from the  chromosome of iHKU114, to rule out its influence, we included the parental  strain iHKU101 as negative control and used arabinose as inducer. The result  showed that no apparent β-galactosidase  activity was observed in iHKU101 (Fig.19A).</P><P align="center"><STRONG>Fig. 19 Levels of LacZ (panel A) and corresponding growth  curves (panel B) of iHKU114 and iHKU101.</STRONG></P><P>To study <EM>cheZ</EM> expression in iHKU102, we replaced <EM>cheZ</EM> gene with <EM>lacZ</EM> gene  and obtained the strain iHKU108. In this strain, <EM>lacZ</EM> expression is under the control of <EM>λpR-O12</EM>, which is repressed by a strong λ repressor, CI. And CI is driven  by <EM>pluxI</EM>. Thus it was anticipated  that <EM>λpR-O12</EM> would be repressed in a  cell density-dependent manner, and thereby lead to a decrease in the levels of β-Galactosidase activity. 
                            β-Galactosidase activity in this strain was substantially lower than  in iHKU114. We reasoned this as the effect of the basal level expression of CI.  When iHKU108 cells were inoculated into fresh medium, the β-galactosidase activity was at a low level, and rose to a constant  level in 1.5 h. β-Galactosidase remained constant for 1.5  h, and then, as expected, decreased to near basal level in 0.5 h (Fig.20A). We  observed that the β-Galactosidase activity levels of both  iHKU114 and iHKU108 were induced/repressed when the cell densities reach OD600~2  (Fig.19A and 20A). This confirmed the cell density-dependent manner of our  design. </P><P align="center"><STRONG>Fig. 20 Level of LacZ (panel A) and corresponding growth  curve (panel B) of iHKU108.</STRONG></P><P align="right"><A href="#p1">[Back to top]</A></P><P align="left"> </P><H1 align="left">movies</H1><P>To  investigate the development of the obtained patterns, a movie documenting the  duration of the pattern development would bring us invaluable information to  look into the mechanisms. We therefore constructed a set of photo capturing devices  to record snapshots of the plates during the course of their development (<U><A href="device.html">movie taker</A></U>).</P><H3>mg1655</H3></TD></TR></TBODY></TABLE><P align="center" class="style2">(Upper panel, experimental videos of MG1655, lower panel modeling videos)</P><P>The original length of this movie is 10 hours 30 minutes,  and it is compressed in to 14 seconds. The compression ratio is 2700:1.  Similarly, This movie starts when cells can be clearly seen on the plate, i.e.  cells growth after our dropping cells on the plate until the start of this  movie is omitted, and this omitted part corresponded to 1 hour. Compared to the  previous movie, no such clear pattern can be observed during the whole process.  Bacteria migrates outwards normally till the end of the movie. A migration  speed of the bacteria can also be measured. Please refer to <U>table.1</U> for detailed information. </P><P>[<A href="#fig4">Back to Fig.4</A>]</P><P> </P><H3>Ihku102</H3><P align="center" class="style2">(Upper panel, experimental videos of iHKU102, lower panel, modeling videos)</P><P>The original length of this movie is 11 hours, and it is  compressed in to 16 seconds. The compression ratio is 2475:1. This movie starts  when cells can be clearly seen on the plate, i.e. cells grows after  dropping cells on the plate until the start of this movie is omitted, and this  omitted part corresponded to 1 hour. According to the movie, it tooks 5  hours for the bacteria to form the inner ring and the ring ablized at the very location. Another 2.5 hours was needed for a vague outer field to be seen. Bacteria in the outer part will continue to grow as well as migrate outwards normally after the end of this movie. By applying the<A href="https://2008.igem.org/Team:iHKU/protocol"> light intensity measurement</A> the cell number inside the void ring is evaluated. Amigration speed of the bacteria can also be measured. The cell concentration inside the void ring is measured  by our brightness method. Please refer to the <U>table below</U> for detailed information. </P><P>[<A href="#fig6">Back to Fig.6</A>]</P><P> </P><P> </P><P> </P><P> </P><P> </P><P> </P><H1 align="left">References</H1><OL><LI class="style1">Datta S, Costantino N, Court DL. A set of  recombineering plasmids for gram-negative bacteria. Gene. 2006, 379: 109-15.</LI><LI class="style1">Haseltine EL, Arnold FH. Implications of  rewiring bacterial quorum sensing. Appl Environ Microbiol. 2008, 74: 437-45.</LI><LI class="style1">Dunlap PV, Kuo A. Cell density-dependent  modulation of the Vibrio fischeri luminescence system in the absence of  autoinducer and LuxR protein. J Bacteriol. 1992, 174: 2440-8.</LI></OL><P> </P> </DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:iHKU/result">http://2008.igem.org/Team:iHKU/result</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:iHKU/result" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:iHKU/result" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:iHKU/result&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:iHKU/result&amp;oldid=105718" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>