<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Project_Measurement"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Heidelberg/Project_Measurement">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Project_Measurement&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project_Measurement&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project_Measurement&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Project_Measurement" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Project Measurement</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="main_wrapper"><DIV id="header"><DIV id="navigation"><DIV id="menu" style="position:static"><UL><LI id="mainHome"><A class="aMain" href="https://2009.igem.org/Team:Heidelberg">Home</A></LI><LI id="mainTeam"><A class="aTeam" href="https://2009.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Members#Students">Students</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Members#Advisors">Advisors</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Scientific_Environment">Scientific Environment</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Gallery">Gallery</A></LI></UL><LI id="mainProject"><A class="aProject" href="https://2009.igem.org/Team:Heidelberg/Project">Project</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project">Abstract</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Introduction">Introduction</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Measurement">Measurement</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Synthetic_promoters">Synthetic Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Further">Further Subprojects</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_SaO">Summary &amp; Outlook</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_References">References</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Acknowledgement">Acknowledgements</A></LI><LI id="mainParts"><A class="aParts" href="https://2009.igem.org/Team:Heidelberg/Parts">Parts</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Parts">Parts</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Parts_Characterization">Characterization</A></LI><LI id="mainModeling"><A class="aModeling" href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">What is HEARTBEAT?</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_database">HEARTBEAT Database</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_GUI">HEARTBEAT GUI</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_gui">GUI Documentation</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_network">HEARTBEAT Fuzzy Network Modeling</A></LI><LI id="mainNotebook"><A class="aNotebook" href="https://2009.igem.org/Team:Heidelberg/Notebook">Notebook</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_natural_promoters_overview">Natural Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_synthetic_promoters">Synthetic Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_promoters_cells">Cell Culture, Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_cell_line">Stable Cell Line</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_color_output">Multi-Color Output</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_modeling">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_measure">Measurements</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_MaM">Material &amp; Methods</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_meetings">Team Meetings</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Biosafety">Biosafety</A></LI><LI id="mainEucaryopedia"><A class="aEucaryopedia" href="https://2009.igem.org/Team:Heidelberg/Eukaryopedia">Eukaryopedia</A></LI><LI id="mainSponsors"><A class="aSponsors" href="https://2009.igem.org/Team:Heidelberg/Sponsors">Sponsors</A></LI></DIV></DIV></DIV></DIV><TABLE><TBODY><TR valign="top" style="margin-left: 2px;"><TD width="650px" style="padding: 0 15px 15px 20px; background-color:#ede8e2"><H1><SPAN class="mw-headline" id="Measurement"> Measurement </SPAN></H1><H2><SPAN class="mw-headline" id="Abstract"> Abstract </SPAN></H2><P>In order to achieve defined protein levels in a cell, promoters of defined strength are an obvious requirement.  <A href="/Team:Heidelberg/Project_Synthetic_promoters" title="Team:Heidelberg/Project Synthetic promoters">Such promoters</A> can only be valuable to synthetic biology if they are well characterized. For future eukaryotic devices that require <A href="http://www.partsregistry.org/PoPS" class="external text" rel="nofollow">PoPS (Polymerase per second)</A> as an input, our promoters will be very suitable since they deliver PoPS as an output. PoPS is <I>the</I> standard unit of synthetic biology, but it is very difficult to measure directly. For bacteria, relative measurements (relative promoter units, RPU) are most commonly used and it has been shown how to convert them to PoPS <A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[1]</A>. In this project, we identify and deal with several challenges to achieve the same in mammalian cells, suggest solutions and provide easy-to-use relative measures for application in mammalian cells - one based on RNA levels (Relative Mammalian Promoter Units, RMPU), the other based on folded protein levels (Relative Expression Units, REU). We apply those measurements on the characterization of CMV, an existing promoter from the registry. Finally, we discuss how to transform these units into PoPS.
</P><H2><SPAN class="mw-headline" id="Introduction"> Introduction </SPAN></H2><P>The need for standardized measurements of promoter activity <I>in vivo</I> has been widely accepted across the synthetic biology community <A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[1]</A>. Only if a part is well characterized initially, function of an engineered device or system can be predicted reliably. Most work of synthetic biology has focused on bacteria, especially <I>Escherichia coli</I>, as a model system. Novel tasks in synthetic biology, especially for medical applications, will require synthetic biology of mammalian cells. Mammalian systems are the most complex biological systems, and therefore, little work has been done in the field of mammalian synthetic biology, leaving a huge potential for future research.</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:296px;"><DIV class="thumbcaption"><DIV style="text-align:justify;"><B>Box 1: Calculation of PoPS</B>, where <I>γM</I> is the mRNA degradation rate, <I>a</I> is the GFP maturation rate, <I>γI</I> is the degradation rate of immature GFP, ρ is the translation rate of immature GFP from mRNA and <I>n</I> is the number of copies of promoter per cell. <A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[1]</A></DIV></DIV></DIV></DIV><DIV style="text-align:justify;">The holy grail of synthetic biology measurements is to express device input and output in &quot;Polymerase per Second&quot; (PoPS), an absolute unit which describes the rate of RNA polymerase molecules passing by the final base pair of the promoter <A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[2]</A>. If PoPS is to be measured from protein expression (e.g. GFP expression), in bacteria PoPS can be calculated from GFP synthesis rate, mRNA degradation, GFP maturation rate, GFP translation rate, plasmid copy number and degradation rate of immature GFP (see Box 1). Only GFP levels are easy to measure by fluorescence readouts, the other factors require complex experiments. Therefore, it has become a standard procedure to characterize promoters in RPUs. One RPU is the activity equivalent to BBa_J23101. <A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[1]</A> This  is possible because by cloning both the promoter and the reference separately but into the same plasmid backbone, promoters will create the same mRNA (as we always used GFP as a readout), thus having the same mRNA degradation and translation rate. In the same way, plasmid copy number, GFP maturation and degradation rates should be equal when using the one cell line at same conditions.</DIV><P>For mammalian cells, no such unit has been reported, nor is there a kit for measuring promoter activity. Mammalian cells have higher levels of complexity - the process from a gene to a functional protein involves transcription (regulated by the promoter itself and chromatin structure), RNA modification, RNA splicing, RNA export, RNA degradation, protein folding and protein degradation <A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[3]</A>. All of these processes are highly regulated, both generally and specifically for the individual gene.  </P><P>A frequently used system is the dual luciferase assay. We dismissed the idea of using luciferase as it is an invasive technique - it requires killing the cells before measuring  <A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[4]</A>. This makes time-course experiments impossible. Moreover, GFP expression can be detected non-invasively with a wide variety of methods. As outlined <A href="/Team:Heidelberg/Project_Measurement#Identifiaction_of_challenges_unique_to_higher_eukaryotes" title="Team:Heidelberg/Project Measurement">below</A>, simple fluorescence measurements by fluorimeters cannot be the method of choice for measuring promoter activity in mammalian systems.</P><P>For the quantification of reporter gene expression in each transfected cell, we used a dual assay with GFP and mCherry. Our cells are co-transfected with a reference plasmid containing our reference promoter in front of mCherry. The fluorescence intensities were measured by flow cytometry and fluorescence microscopy followed by ImageJ processing. 
</P><P>On the other hand, we attempted to measure the promoter strength through quantification of gene expression on mRNA level. Hereby, we applied the novel technique real-time RT-PCR.  The method consists of 2 steps, a reverse transcription, which synthesizes cDNA from extracted mRNA, and a real-time PCR, which is based on normal PCR, but collects the data throughout the PCR process, thus in real-time. Signal detection is achieved using fluorescent dye, which can  be attached to a probe  . The accumulation of PCR products over cycles results in the increase of fluorescence.  By comparing of cycles required to achieve a certain amount of fluorescence  the difference in mRNA expression profile can be illustrated.  Through quantification of GFP mRNA expression driven by the test promoters, the difference in promoter strength can be easily illustrated. This approach measures directly on mRNA level, which means, translation and maturation of the GFP molecule do not have to be taken into account as in the flow cytometry approach.
</P><P>However, a common problem when measuring any construct or device in mammalian cells is the fact that the transfection rate is not always constant. Thus, one never knows how many copies of a construct are actually in the cell. This is why we attempted to create a system that allows a controlled integration of our constructs into the genome. To accomplish this we used the FRT/Flp system which is based on homologous recombination by the enzyme flippase at a specific sequence – the FRT site <A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[20]</A>. <A href="/Team:Heidelberg/stables" title="Team:Heidelberg/stables">Read more about our stable cell lines</A></P><P><A href="/Team:Heidelberg/Project_Measurement#Measurement" title="Team:Heidelberg/Project Measurement">[TOP]</A></P><H2><SPAN class="mw-headline" id="Results"> Results </SPAN></H2><H3><SPAN class="mw-headline" id="Identification_of_challenges_unique_to_higher_eukaryotes"> Identification of challenges unique to higher eukaryotes </SPAN></H3><P>In this chapter, we start by identifying critical points of promoter characterization in eukaryotes and specify how we proceeded to take on these challenges:</P><H4><SPAN class="mw-headline" id="Variance_of_transfection_efficiencies"> Variance of transfection efficiencies </SPAN></H4><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><DIV style="text-align:justify;"><B>Fig. 1: Transfection efficiency for vectors containing different inserts varies</B>. By flow cytometry, we identified transfected cells and found transfection efficiencies to vary. Here, the transfection efficiency is shown for HeLa and e standard error of the mean (SEM) is also displayed.</DIV></DIV></DIV></DIV><DIV style="text-align:justify;">In prokaryotes and yeast, plasmid copy number is a property of the plasmid as determined by the origin of replication. Higher eukaryotes are unable to stably propagate plasmids, unless stable integration occurs (which does not happen spontaneously). Even plasmids with minor differences show entirely different transfection efficiencies and therefore will result in large variations of GFP expression per well. By flow cytometry, we analyzed transfection efficiency for HeLa cells and calculated plasmid number to vary greatly. We transfected cells with plasmids containing the same backbone, but different (promoter) inserts and found transfection efficiency to vary as much as 20% (Fig. 1).  Also, as expected, a significant percentage of cells remains untransfected in a  experiment (Fig. 1).</DIV><P>Dual assay measurements using co-transfection as a control for transfection efficiency are commonly used<A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[4]</A>. Such assays can be utilized for relative characterizations, but do not provide accurate normalization. In order to resolve this issue, we suggest a method which not only allows relative characterizations but is also able to distinguish between transfected and untransfected cells (flow cytometry or image analysis). For experiments aiming at absolute numbers, stable integration of the measurement plasmid into the mammalian genome containing exactly one integration site is required. In this case the copy number per cell is one.</P><H4><SPAN class="mw-headline" id="Regulation_of_promoter_activity_by_chromatin_structure"> Regulation of promoter activity by chromatin structure </SPAN></H4><P>Considering stable integration of a promoter into the mammalian genome, another level of complexity is added: Chromatin structure affects transcription, though mostly not affected by the promoter<A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[5]</A>. Therefore, integration at different sites of the genome will not result in the same expression strength.</P><P>A cell line having a FRT site stably integrated into the genome is required. Such a cell line involves the recombination of sequences between short Flippase Recognition Target (FRT) sites by the Flippase recombination enzyme (FLP or Flp) derived from the 2µ plasmid of the baker's yeast Saccharomyces cerevisiae.<A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[6]</A>. We started the development of <A href="/Team:Heidelberg/Project_Measurement#A_stable_cell_line_for_promoter_measurement" title="Team:Heidelberg/Project Measurement">such a cell line</A>.
</P><P><A href="/Team:Heidelberg/Project_Measurement#Measurement" title="Team:Heidelberg/Project Measurement">[TOP]</A></P><H4><SPAN class="mw-headline" id="RNA_processing"> RNA processing </SPAN></H4><P>Many RNAs are spliced after transcription. Also, RNAs are processed before being exported from the nucleus, and can be retained in the nucleus both dependent and independently of processing <A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[3]</A>, <A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[7]</A>. Therefore, several mRNA populations exist: Unprocessed, processed and retained in the nucleus, and functional mRNA outside of the nucleus. These effects are a current research focus of molecular cell biologists and barely understood even in model organisms as simple as yeast.</P><P>Splicing does not present a problem for reporter gene constructs, as these are cDNAs which are not spliced. Nevertheless, this means that the ratios between RNA levels and protein levels will change for a single promoter if a different output gene is linked to the promoter. We therefore suggest to assume a black box between RNA and protein, and the introduction of two independent units.
</P><P><A href="/Team:Heidelberg/Project_Measurement#Measurement" title="Team:Heidelberg/Project Measurement">[TOP]</A></P><H4><SPAN class="mw-headline" id="Variety_of_cell_lines"> Variety of cell lines </SPAN></H4><DIV style="text-align:justify;"> TOP10 or DH5α cells have been widely accepted as chassis systems by synthetic biologists working in bacteria. For the work with mammalian cells, no such consensus exists; also, it would not be sensible to limit synthetic mammalian biology to a small number of cell lines, as every cell line is suited for a special application. Scientists working on breast cancer virotherapy by synthetic promoters would choose a breast cancer cell line, whereas scientists working on <A href="https://2008.igem.org/Team:Bay_Area_RSI" class="external text" rel="nofollow">stem cell therapies to myocardial infarction</A> would choose a cardiomyocyte cell line. Cell lines differ greatly, even in expression strength of constitutive promoters (shown <A href="/Team:Heidelberg/Project_Measurement#Measuring_REU_by_flow_cytometry_and_image_analysis" title="Team:Heidelberg/Project Measurement">below</A>).
 <DIV style="text-align:justify;">Each part must be characterized in every cell line. We worked with three cancer cell lines, <A href="/Team:Heidelberg/Eukaryopedia#HeLa" title="Team:Heidelberg/Eukaryopedia">HeLa</A> (cervical cancer), <A href="/Team:Heidelberg/Eukaryopedia#MCF-7" title="Team:Heidelberg/Eukaryopedia">MCF-7</A> (breast cancer) and <A href="/Team:Heidelberg/Eukaryopedia#U2-OS" title="Team:Heidelberg/Eukaryopedia">U2-OS</A> (osteosarcoma). We suggest that the synthetic biology community should pick a small number of cell lines as model systems. We argue that HeLa (well known, widely used, easy to transfect) and/or MCF-7 (very robust to apoptosis) should be part of these cell lines. </DIV><H4><SPAN class="mw-headline" id="Lack_of_truly_constitutive_promoters_in_mammalian_cells"> Lack of truly constitutive promoters in mammalian cells </SPAN></H4><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><DIV style="text-align:justify;"><B>Fig. 2: CMV and JeT strength changes depending on conditions</B> We characterize GFP expression from <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K203112" class="external text" rel="nofollow">JeT</A> and <A href="http://partsregistry.org/Part:BBa_I712004" class="external text" rel="nofollow">CMV</A> fluctuate dependent on condition (Everolimus induces extreme starvation). Measured by flow cytometry 20 hours after transfection (unless specified otherwise) in MCF-7. The standard deviation is represented by the error bars. </DIV></DIV></DIV></DIV><P>Figure 2 shows measurements of GFP expression from <A href="http://partsregistry.org/Part:BBa_I712004" class="external text" rel="nofollow">CMV</A> and  <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K203112" class="external text" rel="nofollow">JeT</A> under varying conditions. This result demonstrates that every promoter in mammalian cells underlies regulation, and therefore, is not truly constitutive. We analyzed the sequence of CMV by <A href="http://www.gene-regulation.com/pub/databases.html" class="external text" rel="nofollow">TRANSFAC Professional</A> and found it to contain two <A href="/Team:Heidelberg/Eukaryopedia#Transcription_factors" title="Team:Heidelberg/Eukaryopedia">NF-κB</A> binding sites, two <A href="/Team:Heidelberg/Eukaryopedia#CREB" title="Team:Heidelberg/Eukaryopedia">CREB</A>-binding sites, and single <A href="/Team:Heidelberg/Eukaryopedia#Ap1" title="Team:Heidelberg/Eukaryopedia">Ap1</A>, RFX1 and SRF binding sites. Of NF-κB, CREB and Ap1, we know that they have a high constitutive activity (<A href="/Team:Heidelberg/Project_Synthetic_promoters#Generation_of_a_library_of_constitutive_promoters" title="Team:Heidelberg/Project Synthetic promoters">compare to Synthetic promoters</A>), but nevertheless, they underlie regulation. For example, NF-κB is induced by inflammation conditions, whereas CREB is activated by the second messenger cAMP<A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[8]</A> and thus responds to many hormones, starvation conditions etc. This impedes comparison of promoters in different conditions. We next <A href="/Team:Heidelberg/Project_Measurement#Characterization_of_promoters_under_different_conditions" title="Team:Heidelberg/Project Measurement">discuss</A> how this can be achieved.
<A href="/Team:Heidelberg/Project_Measurement#Measurement" title="Team:Heidelberg/Project Measurement">[TOP]</A></P><H3><SPAN class="mw-headline" id="A_promoter_measurement_kit_for_use_in_mammalian_systems"> A promoter measurement kit for use in mammalian systems </SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:356px;"><DIV class="thumbcaption"><DIV style="text-align:justify;"><B>Fig. 3: Plasmid map of pSMB_MEASURE.</B> BBb sites are shown in red, core promoter in light blue and proximal promoter in dark blue. The length of this plasmid is 5162 bp.</DIV></DIV></DIV></DIV><DIV style="text-align:justify;">We created a plasmid , pSMB_MEASURE (SMB is for Synthetic Mammalian Biology; <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K203100" class="external text" rel="nofollow">Part:BBa_K203100</A>), which should be used for promoter characterization in mammalian cells. pSMB_MEASURE (see Fig. 3) contains a reference promoter, JeT<A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[9]</A> (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K203112" class="external text" rel="nofollow">Part:BBa_K203112</A>), which is flanked by BBb_2 (Tom Knight) sites and can therefore be replaced by the promoter to be measured. JeT is ideal as a reference promoter for a variety of reasons. First, it has an intermediate expression strength; second, it is regulated by a wide variety of transcription factors and low levels of change in fluorescence among different conditions (compare Fig. 2)<A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[9]</A>. Third, we want to pay tribute to its creators as pioneers in synthetic promoter research.</DIV><P>We separated JeT's core promoter from its proximal promoter by a HindIII site; it can therefore be used for <A href="/Team:Heidelberg/Project_Synthetic_promoters#Results" title="Team:Heidelberg/Project Synthetic promoters">the <I>de novo</I> creation of synthetic promoters</A> containing the JeT core promoter or for modifying the strength of a promoter by <A href="/Team:Heidelberg/Project_Measurement#Different_core_promoters_result_in_different_expression_strength" title="Team:Heidelberg/Project Measurement">core promoter swapping</A>. In addition, it contains a FRT site which will allow for stable integration into mammalian cells also containing a FRT site. Thus, it provides the possibility to characterize the promoter in a defined genome and in this way helps to avoid some of the <A href="/Team:Heidelberg/Project_Measurement#Identification_of_challenges_unique_to_higher_eukaryotes" title="Team:Heidelberg/Project Measurement">challenges outlined above</A>. For the same reason, it also contains a mammalian selection marker (hygromycine). For the generation of the plasmid, please see  <A href="http://partsregistry.org/Part:BBa_K203100:Design" class="external text" rel="nofollow">part design</A>. As a reporter gene, it contains GFP, which is followed by a SV40 mammalian terminator. We generated another plasmid pSMB_REFERENCE, which contains mCherry instead of GFP. It can be used for normalizations of transfection efficiency in flow cytometry and image analysis.
</P><P><A href="/Team:Heidelberg/Project_Measurement#Measurement" title="Team:Heidelberg/Project Measurement">[TOP]</A></P><H3><SPAN class="mw-headline" id="Two_units_for_promoter_activity_in_mammalian_cells"> Two units for promoter activity in mammalian cells </SPAN></H3><P>We define two units: 1 relative mammalian promoter unit (RMPU) is defined in analogy to<A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[1]</A>, but mRNA-based. It is  the amount of total mRNA (that is, processed and unprocessed mRNA inside and outside of the nucleus) generated by a promoter x, relative to the amount of total mRNA generated by the JeT promoter j (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K203112" class="external text" rel="nofollow">BBa_K203112</A>)<A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[9]</A> in steady state. It is directly proportional to PoPs, as we show. In steady state, change of mRNA levels is 0, thus:
</P><P>Where M' is the change in total mRNA level, γM is the mRNA degradation rate and n is the number of promoters per cell (adapted from <A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[1]</A>. Therefore:
</P><P>Since both promoters are cloned seperatly into the same backbone, they generate identical mRNA. As the most important determinant for mRNA stability is the 3' untranslated region, and mRNA stability is generally not affected by promoter structure <A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[10]</A>, we are able to assume the same mRNA degradation rate γM for both promoters. 
</P><P>We also expect the same n for both promoters as this is normalized for during the measurements (as shown below).
We then define RMPU, and achieve a cancelation of terms:
</P><P>On the other hand, we define Relative Expression Units (REU). 1 relative expression unit (REU) is the amount of total folded protein generated by a promoter, relative to the amount of folded protein generated by the JeT promoter (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K203112" class="external text" rel="nofollow">BBa_K203112</A>) under the same cellular condition. It is not directly proportional to PoPS as too many levels of regulation, such as post-transcriptional modifications, enhanced splicing and nuclear shuttling/transport lie between transcription and protein <A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[3]</A>, and REUs would strongly depend on conditions which affect RNA. Still, it is a very useful measure, as for most applications such as metabolic pathway engineering, protein level, not mRNA levels, are of importance. Also, it can be measured with a much easier experimental setup than RMPU. See <A href="/Team:Heidelberg/Project_Measurement#Characterization_of_promoters_under_different_condtions" title="Team:Heidelberg/Project Measurement">discussion</A> for ideas on how to compare REU measurements taken at different conditions.
</P><P><A href="/Team:Heidelberg/Project_Measurement#Measurement" title="Team:Heidelberg/Project Measurement">[TOP]</A></P><H3><SPAN class="mw-headline" id="Measuring_RMPU_by_real-time_RT-PCR"> Measuring RMPU by real-time RT-PCR </SPAN></H3><P>To measure the RMPU (Relative Mammalian Promoter Units), HeLa cells were transfected with plasmids containing the promoter of interest. HeLa cells transfected with plasmid containing JeT promoter were used as reference. At two different time-points (20 h and 50 h after transfection), total RNA (&gt; 200 bp) was isolated. This step was followed by  real-time RT-PCR, where the mRNA amount of GFP was quantified. For each promoter at each time point, 12 replicates were taken to obtain reliable results and reduce the variance. To get comparable results of different samples, we used multiple housekeeping genes as internal controls, while using non-transfected HeLa mRNA as plate-to-plate correction.  The CMV promoter was calculated about 2.89 times stronger as JeT after 20 h and 2.04 times stronger after 50 h (Fig. 4 and 5).
</P></DIV></TD></TR><TR valign="top"><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><DIV style="text-align:justify;"><B>Figure 4: Real-time RT-PCR data of CMV and JeT promoters.</B> One group of HeLa cells were transfected with plasmid containing CMV promoter coupled to GFP. Another group with JeT promoter coupled to GFP was used as reference. RNA was extracted after 20 h and 50 h, followed by real-time RT-PCR. The Ct values were collected with a threshold of 0.05. The CMV activity compared to JeT at the same time point was calculated in MatLab as &quot;arbitrary units&quot; which correspond to amount of mRNA.</DIV></DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><DIV style="text-align:justify;"><B>Figure 5: Real-time RT-PCR data of CMV promoter</B>. Arbitrary units of CMV divided by that of JeT is the <A href="https://2009.igem.org/Team:Heidelberg/Project_Measurement" class="external text" rel="nofollow">RMPU</A>. </DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P><A href="/Team:Heidelberg/Project_Measurement#Measurement" title="Team:Heidelberg/Project Measurement">[TOP]</A></P><H3><SPAN class="mw-headline" id="Measuring_REU_by_flow_cytometry_and_image_analysis"> Measuring REU by flow cytometry and image analysis</SPAN></H3><P>The GFP reporter expression from <A href="http://partsregistry.org/Part:BBa_I712004" class="external text" rel="nofollow">Part:BBa_I712004  (CMV)</A> is examined by two-color flow cytometry<A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement"> [11]</A> as well as fluorescence microscopy with subsequent image analysis. To account for potential differences in transfection efficiency, we have used the fluorescent protein mCherry coupled to the reference promoter JeT, which is co-transfected together with GFP coupled to the promoter of interest. By comparing transfection/expression levels of mCherry, we could exclude samples with low cell numbers or low transfection efficiencies. The relative fluorescence of the CMV promoter coupled to GFP was measured in three different cell lines: <A href="/Team:Heidelberg/Eukaryopedia#HeLa" title="Team:Heidelberg/Eukaryopedia">HeLa</A>, <A href="/Team:Heidelberg/Eukaryopedia#MCF-7" title="Team:Heidelberg/Eukaryopedia">MCF-7</A>, and <A href="/Team:Heidelberg/Eukaryopedia#U2-OS" title="Team:Heidelberg/Eukaryopedia">U2-OS</A>. The HeLa cell line was measured five times, MCF-7 four times and U2-OS also four times by flow cytometry. The microscopy measurement was performed once for the different cell lines. As one would expect, the Relative Expression Units (REU) varied slightly between the different cell lines. Overall, the CMV promoter is very strong relative to our reference promoter JeT with values of 5.52, 6.76 and 9.73 in HeLa, MCF-7 and U2-OS respectively (Fig. 6).
</P><P><A href="/Team:Heidelberg/Project_Measurement#Measurement" title="Team:Heidelberg/Project Measurement">[TOP]</A></P><H3><SPAN class="mw-headline" id="Different_core_promoters_result_in_different_expression_strength"> Different core promoters result in different expression strength </SPAN></H3><P>We cloned the CMV core promoter in front of the JeT proximal promoter to obtain JeT_CMV. We characterized this construct in three different cell lines, HeLa, MCF-7, and U2-OS and found it to have 50% - 60%  percent of JeT's activtiy depending on the cell line (Fig. 7). Thereby, the HeLa cell line was measured five times and the MCF-7 and U2-OS cell line measurements were performed four times. The fact that variations in the core promoter can be used to vary expression strength of a certain promoter of interest comes in useful if the transfer function of an existing promoter is to be altered, and it can be used to further diversify the synthetic promoters we created. We characterized these promoters by the same methods and accuracy as CMV and JeT/CMV (see <A href="/Team:Heidelberg/Project_Synthetic_promoters" title="Team:Heidelberg/Project Synthetic promoters">Synthetic Promoter project</A>)
</P><TABLE><TBODY><TR valign="top"><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><DIV style="text-align:justify;"><B>Figure 6: Flow cytometry and microscopy measurement data of CMV (REU) in different cell lines.</B> The three cell lines MCF-7, U2-OS and HeLa were cotransfected with the CMV promoter coupled to GFP and a reference plasmid including the promoter JeT coupled to mCherry. The relative fluorescence (REU) of GFP was measured 20 hours after transfection. All cell lines were measured once for microscopy. The HeLa cell line was measured five times by flow cytometry and MCF-7 and U2-OS were measured four times. In the flow cytometry measurement the standard error of the mean (SEM) is indicated by the error bars.</DIV></DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><DIV style="text-align:justify;"><B>Figure 7: Flow cytometry measurement data of JeT_CMV (REU) in different cell lines.</B> The three cell lines MCF-7, U2-OS and HeLa were cotransfected with the JeT_CMV promoter coupled to GFP and a reference plasmid including the promoter JeT coupled to mCherry. The relative fluorescence (REU) of GFP was measured 20 hours after transfection. The MCF-7 and the U2-OS cell line were measured four times and the HeLa cell line five times. The standard error of the mean (SEM) is indicated by the error bars. </DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P><A href="/Team:Heidelberg/Project_Measurement#Measurement" title="Team:Heidelberg/Project Measurement">[TOP]</A></P><H3><SPAN class="mw-headline" id="A_stable_cell_line_for_promoter_measurement"> A stable cell line for promoter measurement </SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><DIV style="text-align:justify;"><B>Fig. 8:  Results of second exponential PCR.</B> In all lanes are  several bands visible, indicating that there is more than one FRT-site integrated in each cell line. Negative controls are: 1. untransfected genomic DNA 2. H<SUB>2</SUB>O negative control LAM-PCR, 3. H<SUB>2</SUB>O negative control first exponential PCR, 4. H<SUB>2</SUB>O negative control second exponential PCR. Electrophoresis was carried out on 2% agarose; <A href="http://tools.invitrogen.com/content/sfs/manuals/15628019.pdf" class="external text" rel="nofollow">100 bp DNA ladder</A> was used.</DIV></DIV></DIV></DIV><P><B><A href="/Team:Heidelberg/stables" title="Team:Heidelberg/stables">Main Article: Stable cell line</A></B></P><P>We were able to generate HeLa, MCF-7 and U2-OS cells that stably integrated the FRT-site into their genome. Fig. 8 shows the PCR products of the second exponential PCR.  For each cell line there are several bands visible. Since the number of bands correlates with the number of unique integration sites <A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement"> [2]</A> there must have been more than one integration of the FRT-vector into the genome of the cells.
</P><P><A href="/Team:Heidelberg/Project_Measurement#Measurement" title="Team:Heidelberg/Project Measurement">[TOP]</A></P><H2><SPAN class="mw-headline" id="Discussion"> Discussion </SPAN></H2><H3><SPAN class="mw-headline" id="Flow_Cytometry.2FFluorescence_Microscopy"> Flow Cytometry/Fluorescence Microscopy </SPAN></H3><P>Our flow cytometer measurements of CMV and JeT-CMV in different cell lines (Fig.3 and 4) show that the strength of these promoters varies in different cell lines. Furthermore, the ratio between the two promoters varies slightly amongst different cell lines, but overall CMV is considerably stronger than JeT, while JeT-CMV is consistently lower than JeT. Our measurements were reproducible considering the strength ratios between our standard promoters. 
Between different measurements we detected minimal fluctuations (represented by the standard error of the mean). However, as transfection efficiencies also slightly vary between experiments, this is one source of error when comparing experiments. Additionally, due to the nature of the iGEM competition, experiments have been carried out by several persons, introducing another source for errors.
We have applied microscopy measurements to support the data obtained by flow cytometry. The initial microscopy measurements so far confirm the observations made by flow cytometry, although some inconsistencies appeared. This might partially be due to the different cell preparation for flow cytometry and microscopy. While living cells have been used for the flow cytometric measurmentes, cells were fixed prior to analysis by microscopy. So far we could only carry out one set of microscopic measurements, therefore it will be necessary to repeat the microscopic measurements to obtain reliable data.
Apart from supporting the flow cytometry results, fluorescence microscopy will also gain importance in the future as we are working on a stable cell line which simultaneous expresses differently coloured fluorescent proteins. These fluorescent proteins will be directed to various compartments of the cell and thereby, allow to visualize the activation of several promoters at the same time in one single cell. Read more about our future plans in the <A href="/Team:Heidelberg/Project_SaO" title="Team:Heidelberg/Project SaO">outlook section</A>.
</P><P><A href="/Team:Heidelberg/Project_Measurement#Measurement" title="Team:Heidelberg/Project Measurement">[TOP]</A></P><H3><SPAN class="mw-headline" id="Real-time_RT-PCR"> Real-time RT-PCR </SPAN></H3><P>The <A href="/Team:Heidelberg/Project_Measurement#Measuring_RMPU_by_real-time_RT-PCR" title="Team:Heidelberg/Project Measurement">results of the real-time RT-PCR</A> measurement show that the promoter strength varied significantly over time. Reasons for the variations could be due to the fact, that the activity was measured under transient transfection where the plasmid containing the promoter is not replicated when the cells proliferate. This results in a decrease of amount of plasmids per cell, thus reduced copy of promoter present in the cells. This effect will be increase within each replication. Therefore, long time intervals between measurement and transfection will be more error-prone than shorter intervals. That is why the proportion of error to data is much higher 50 h after transfection compared to the 20 h measurement, although the standard deviations of the two time points are in the same range. It can also be possible that the cells containing the plasmid with promoter construct die faster due to changes in cell physiology. Furthermore, the fluctuation of the results could be from both systematic and random error where all the following reasons could be responsible: pipetting errors during plate preparation, influence of the freeze-thaw process on enzymes, differing contents of mastermix and random error caused by the machines. This could be caused by physiological changes the plasmids and consequently, the promoters put on the cells. Over the course of 20 h the effects those changes evoked were not that distinctiv but over 50 h of interaction enlarged the difference  greatly.
</P><P><A href="/Team:Heidelberg/Project_Measurement#Measurement" title="Team:Heidelberg/Project Measurement">[TOP]</A></P><H3><SPAN class="mw-headline" id="Characterization_of_promoters_under_different_condtions"> Characterization of promoters under different condtions </SPAN></H3><P>Considering what we learned about <A href="/Team:Heidelberg/Project_Measurement#Identification_of_challenges_unique_to_higher_eukaryotes" title="Team:Heidelberg/Project Measurement">the lack of absolute, non-changing reference standards</A>, characterizing promoters under different conditions becomes inherently difficult. All parameters affecting REU and RPMU are altered by a change in conditions. For the assumptions on which REU and RMPU are based to remain valid, promoters must only be compared in one set of conditions. In most scientific work to this date, transcriptional activity is assessed by comparing protein levels between unrelated conditions (see<A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement"> [12]</A> for a typical example). Although giving sufficient information as to whether a promoter is up- or downregulated, this standard cannot be used for an absolute characterization, as if a condition induces, for example, a system-wide increase in translation, it will result in a ratio between the promoters which is not proportional to the change in transcription. If, on the other hand, measurements are related to a reference promoter at each condition individually, this does not represent true levels of change if the reference promoter itself is up- or downregulated by that condition. Providing absolute measurements to compare between different conditions will be a major challenge for synthetic mammalian biology in the next years. We suggest, for now, always to give promoter strength relative to the strength of JeT under the condition regarded.
</P><P><A href="/Team:Heidelberg/Project_Measurement#Measurement" title="Team:Heidelberg/Project Measurement">[TOP]</A></P><H3><SPAN class="mw-headline" id="Towards_PoPs"> Towards PoPs </SPAN></H3><P>The concept of RMPU can easily be converted into PoPs, at least for constitutive promoters. RNA degradation equals RNA synthesis in steady state. Therefore, we suggest to block RNA synthesis by applying Actinomycin D which works in minutes<A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement"> [13]</A>, and then measuring RNA degradation. According to <A href="/Team:Heidelberg/Project_Measurement#Two_units_for_promoter_activity_in_mammalian_cells" title="Team:Heidelberg/Project Measurement">equation 2</A>, PoPS will then equal mRNA degradation, divided by copy number (which is 1, if the experiment is conducted in a stable cell line). Of course, Actinomycin is toxic to cells and might affect mRNA turnover rates system-wide. Accounting for this effect will present a major challenge.
</P><P><A href="/Team:Heidelberg/Project_Measurement#Measurement" title="Team:Heidelberg/Project Measurement">[TOP]</A></P><H3><SPAN class="mw-headline" id="Concluding_remarks"> Concluding remarks </SPAN></H3><P>We reported two new units for promoter characterization in mammalian cells, one of which being directly proportional to PoPS. We showed the advantages and limitations of the concept. Future work will have to focus on improving comparisons between different conditions.
</P><P><A href="/Team:Heidelberg/Project_Measurement#Measurement" title="Team:Heidelberg/Project Measurement">[TOP]</A></P><H2><SPAN class="mw-headline" id="Method_details"> Method details </SPAN></H2><H3><SPAN class="mw-headline" id="Flow_Cytometry"> Flow Cytometry </SPAN></H3><P>Flow cytometry is a commonly used method for the measurement of fluorescence intensity levels. It uses the principles of light scattering, light excitation, and emission of fluorochrome molecules to generate specific multi-parameter data from particles and cells in the size range of 0.5 µm to 40 µm diameter . One unique feature of flow cytometry is that it measures fluorescence per cell or particle. In contrast to high-resolution microscopy, flow cytometric measurements do not reveal intracellular distribution of the measured signal, but give an integrated signal of each cell. However, flow cytometry allows fast, multi-parametric measurements of high numbers of cells (&gt;1000 cells/sample) in a quantitative manner <A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[14]</A>, <A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[19]</A>.
A flow cytometer consists of the fluidics system where cells are hydro-dynamically focused in a sheath of fluid; one or several laser which is the source of focused light for fluorescence and scatter; the optics which gather and direct the light emitted from the cells; up to 8 detectors which receive the light and convert it into an electrical signal, which is finally delivered to a computer system to analyze the signals of each cell. 
The first detector (Forward Scatter; FSC) is in line with the light beam; its signal is proportional to the size of the cell. The larger the cell, the more light is scattered and the higher the electric signal in the detector. Perpendicular to the light beam are several other detectors, including the Side Scatter (SSC). The Side Scatter represents the granularity of the cell and this can be used to further distinguish different cell populations and exclude apoptotic cells. A two-dimensional scatter plot of Forward versus Side Scatter can give a more detailed overview over the nature of the cells in the sample. This plot is used to separate dead cells and cell debris from healthy cells. <A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[14]</A>, <A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[19]</A> 
Following the Side Scatter, the light is directed through a series of mirrors and filters, so that particular wavelengths are delivered to the correct detector. The fluorescence signal is then converted to a voltage signal and can be visualized as a histogram, showing frequencies of cells with different fluorescence levels (usually distributed over 1024 channels) <A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[14]</A>, <A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[19]</A>. 
We have used a Beckman Coulter FC500 MPL, equipped with a 488 nm and 561 nm laser, thereby allowing simultaneous and quantitative measurement of GFP and mCherry fluorescence. 
Cells were prepared in 96-well format with 10<SUP>4</SUP> cells/well and transfected with the promoter of interest and when necessary, induction drug was added for inducible promoters. Before measurement, the medium was removed, the cells were washed with 1xPBS and trypsinized with 60 µl of trypsin per well. After 10 minutes incubation at 37°C, 1xPBS + 1% BSA were added up to a volume of 200 μl per well. When starting the flow cytometry measurement we first adjusted the gates and the background signal to the negative control (Fig. 9). Fig. 10 shows an example of a gated positive control (JeT), where we can see a second normal distribution peak indicating the gated GFP positive cells.
</P><TABLE><TBODY><TR valign="top"><TD width="300px" style="padding: 0 20px 0 0;"><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><DIV style="text-align:justify;"><B>Figure 9 : Example of a negative control in Hela cells by flow cytometry.</B> The number of events is plotted against the fluorescence (log) of GFP. Background signal is set under 10 and the area under the curve is colored in green.</DIV></DIV></DIV></DIV></DIV></TD><TD width="30px"><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><DIV style="text-align:justify;"><B>Figure 10 : Example of a positive control (JeT) in Hela cells by flow cytometry.</B> The number of events is plotted against the fluorescence (log) of GFP. The area under the curve is colored in green.</DIV></DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P><A href="/Team:Heidelberg/Project_Measurement#Measurement" title="Team:Heidelberg/Project Measurement">[TOP]</A></P><H3><SPAN class="mw-headline" id="Microscopy_and_Image_Analysis"> Microscopy and Image Analysis </SPAN></H3><P>For further characterization of our promoters we decided to use fluorescence microscopy. Our cells are co-transfected (1:2) with a reference plasmid, which contains JeT in front of mCherry. Analysis of reporter gene expression can be restricted to cells that express the reference plasmid and are therefore likely to have been successfully co-transfected with the reporter plasmid of interest. This is particularly useful when transfection efficiency is low. We applied this approach to microscopy measurement to minimize the experimental error. The promoter strength was evaluated in relative expression units (REU) relative to the JeT promoter which was set to 1. Microscopy images of our samples fixed with 4% formaldehyde were taken using the Nikon Eclipse 90i upright automated widefield microscope in the Nikon Imaging Center at the University of Heidelberg. Each image was taken in the GFP and mCherry channel and the exposure time had to be adjusted for over- or underexposed images. The conversion factor was experimentally determined by imaging a fluorescent plate with exposure time varying from 10 to 70 ms (the estimated range of our promoters) and plotting the mean grey values of these images versus time. A linear relationship between the exposure times and grey values was observed for this time frame and the conversion factor was determined to be 0.998. For image analysis we used ImageJ (Image Processing and Analysis in Java) software tools for obtaining the mean grey values of each cell containing the promoter of interest. ImageJ is an open source software developed at the National Institutes of Health (NIH (<A href="http://rsbweb.nih.gov/ij/" class="external free" rel="nofollow">http://rsbweb.nih.gov/ij/</A>). 
The measurement is performed by setting the boundaries around the cells using the threshold in the mCherry channel image and generating a mask that is then applied to the image of the GFP channel. For a step by step description on ImageJ analysis see: <A href="/Team:Heidelberg/Notebook_MaM" title="Team:Heidelberg/Notebook MaM"> Materials and Methods</A>.
</P><P><A href="/Team:Heidelberg/Project_Measurement#Measurement" title="Team:Heidelberg/Project Measurement">[TOP]</A></P><H3><SPAN class="mw-headline" id="Real-time_RT-PCR_2"> Real-time RT-PCR </SPAN></H3><P>Real-time RT-PCR, which has been developed tremendously during the last 20 years, is a specific and sensitive technique for detecting and quantification of gene expression on RNA level. In order to obtain reliable results and reduce the variance during the extraction, reverse transcription and PCR, we did the measurement for each promoter with 12 replicates from an identical sample. Prior to PCR, we extracted the total RNA using Qiagen RNeasy Plant Mini Kit following the protocol from the handbook <A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[15]</A>. The extraction was performed by the QIAcube<SUP>TM</SUP><A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[16]</A>, the automated spin-column kits preparation robot, to avoid possible operator error. In order to perform the real-time PCR, primers and probes were designed according to QuantiFast Probe RT-PCR Handbook <A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[17]</A>: The target PCR product length is set between 70-200 bp. The target spans over exon-exon boundary to exclude amplification of genome DNA which may be presented in the RNA extraction as contamination. The probes have reporter fluorophore and quencher fluorophore at 5' and 3' end respectively. During amplification, probes bind specifically to the target gene. The polymerase hydrolyzes the probe during elongation due to its exonuclease activity. The signal of the released fluorophore can then be detected and correlates therefore to the amount of PCR product. The real-time RT-PCR was performed on StepOnePlus<SUP>TM</SUP> Real-time PCR System from Applied Biosystems <A href="/Team:Heidelberg/Project_Measurement#References" title="Team:Heidelberg/Project Measurement">[18]</A>. We quantified the expression of GFP, the reporter gene, together with other 5 housekeeping genes (ß-actin, glycerol-aldehyde-3-phosphate de-hydrogenase, glucose-6-phosphate dehydrogenase, gamma-tubulin, 18S rRNA) for normalization. At the end of each cycle, the fluorophore of each well is read and recorded. Ct (threshold cycle) is defined as the cycle number at which the fluorescent strength (the curve) crosses a certain threshold (Fig. 11). The more amount of starting mRNA, the sooner will the machine detect the fluorescent signals from the PCR reaction, which means a smaller Ct under the same threshold. The threshold should be a careful decision. With a too small threshold, the signal may be too weak for reliable detection. If it is too big, the reaction might be restricted by the enzymes and other materials. Usually, we set the threshold at 0.05 for all plates (Fig. 12.1 and Fig. 12.2). Calculating the difference between Ct values, with normalization of the multiple housekeeping genes, we get the ratio of the activity of the promoter of interest and our reference.
</P><TABLE><TBODY><TR valign="top"><TD width="300px" style="padding: 0 20px 0 0;"><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:212px;"><DIV class="thumbcaption"><DIV style="text-align:justify;"><B>Figure 11 : Ct values of 18s rRNA with CMV promoter 20h after transfection</B> Threshold is set at 0.05, the cycle number at when the amplification plot crosses the threshold is the Ct value.</DIV></DIV></DIV></DIV></DIV></TD><TD width="30px"><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:212px;"><DIV class="thumbcaption"><DIV style="text-align:justify;"><B>Figure 12.1 : Ct values of eGFP with CMV promoter 20h after transfection</B> Threshold is set at 0.05. 12 dots represent 12 RNA extractions from identical sample.</DIV></DIV></DIV></DIV></DIV></TD><TD width="30px"><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:212px;"><DIV class="thumbcaption"><DIV style="text-align:justify;"><B>Figure 12.2 : Ct values of eGFP with JeT promoter 20h after transfection</B> Threshold is set at 0.05. 12 dots represent 12 RNA extractions from identical sample.</DIV></DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P><A href="/Team:Heidelberg/Project_Measurement#Measurement" title="Team:Heidelberg/Project Measurement">[TOP]</A></P><H2><SPAN class="mw-headline" id="References"> References </SPAN></H2><DIV style="text-align:justify;"><P>[1] Kelly J. R., Rubin A. J., Davis J. H., Ajo-Franklin C. M., Cumbers J., Czar M. J., de Mora K., Glieberman A. L., Monie D. D. &amp; Endy D. Measuring the activity of BioBrick promoters using an in vivo reference standard. <I>Journal of Biological Engineering</I> 3 (2009).</P><P>[2] Endy D. &amp; Deese I. Adventures in synthetic biology (comic) <I>Nature</I> 438: 449-453 (2005). Available online at <A href="http://www.nature.com/nature/comics/syntheticbiologycomic/index.html" class="external text" rel="nofollow">nature.com</A></P><P>[3] Alberts B., Johnson A., Walter P. &amp; Lewis J. <I>Molecular Biology of the Cell.</I> 5th edition, 2008. Garland Science) Chapter 6</P><P>[4] <A href="http://www.promega.com/tbs/tm058/tm058.html" class="external free" rel="nofollow">http://www.promega.com/tbs/tm058/tm058.html</A></P><P>[5]  Alberts B., Johnson A., Walter P. &amp; Lewis J. <I>Molecular Biology of the Cell</I>. 5th edition, 2008. Garland Science) Chapter 7, p. 467-476</P><P>[6] Zhu X.D. &amp; Sadowski P. D. Cleavage-dependent Ligation by the FLP Recombinase. <I>J Biol Chem</I> 270: 23044-23054 (1995).</P><P>[7] York J. D., Odom A. R., Murphy R., Ives E. B. &amp; Wente S. R. A phospholipase C-dependent inositol polyphosphate kinase pathway required for efficient messenger RNA export. <I>Science</I> 285: 96-100 (1999).</P><P>[8]  Alberts B., Johnson A., Walter P. &amp; Lewis J. <I>Molecular Biology of the Cell.</I> 5th edition, 2008. Garland Science) Chapter 7, p. 905-908</P><P>[9] Tornoe J. Generation of a synthetic mammalian promoter library by modification of sequences spacing transcription factor binding sites. <I>Gene</I> 297: 21-32 (2002) </P><P>[10] Alberts B., Johnson A., Walter P. &amp; Lewis J. <I>Molecular Biology of the Cell.</I> 5th edition, 2008. Garland Science) Chapter 7, p. 492</P><P>[11] Ducrest A. L., Amacker M., Lingner J. &amp; Nabholz M. Detection of promoter activity by flow cytometric analysis of GFP reporter expression. <I>Nucleic Acids Res.</I> 30: e65 (2002).  </P><P>[12] Rushton P. J., Reinstädler A., Lipka V., Lippok B. &amp; Somssich I. E. Synthetic plant promoters containing defined regulatory elements provide novel insights into pathogen- and wound-induced signalling. <I>Plant Cell</I> 14: 749-762 (2002).</P><P>[13] Degenhardt T., Rybakova K. N., Tomaszewska A., Moné M. J., Westerhoff H. V., Bruggeman F. J. &amp; Carlberg C. Population-Level Transcription Cycles Derive from Stochastic Timing of Single-Cell Transcription. <I>Cell</I> 138: 489-501 (2009).</P><P>[14] Lottspeich F. &amp; Engels J. W.: <I>Bioanalytik</I> (Book. 2nd edition, 2006. Spektrum Akademischer Verlag) Chapter 5, p. 95-96.
</P><P>[15] <A href="http://www1.qiagen.com/Products/RnaStabilizationPurification/RNeasySystem/RNeasyPlantMini.aspx" class="external free" rel="nofollow">http://www1.qiagen.com/Products/RnaStabilizationPurification/RNeasySystem/RNeasyPlantMini.aspx</A></P><P>[16] <A href="http://www1.qiagen.com/products/automation/qiacube.aspx" class="external free" rel="nofollow">http://www1.qiagen.com/products/automation/qiacube.aspx</A></P><P>[17] <A href="http://www1.qiagen.com/literature/render.aspx?id=398" class="external free" rel="nofollow">http://www1.qiagen.com/literature/render.aspx?id=398</A></P><P>[18] <A href="https://products.appliedbiosystems.com/ab/en/US/adirect/ab?cmd=catNavigate2&amp;catID=604109&amp;tab" class="external free" rel="nofollow">https://products.appliedbiosystems.com/ab/en/US/adirect/ab?cmd=catNavigate2&amp;catID=604109&amp;tab</A>=DetailInfo&lt;</P><P>[19] Ormerod M.G.: <I>Flow cytometry: a practical approach</I> (Book. 3rd edition, 2000. Oxford University Press) Chapter 1, p. 7-21.
</P><P>[20] Zhou H. Development of site-specific integration system to high-level expression recombinant proteins in CHO cells. <I>Chinese journal of biotechnology</I> 23: 756-62 (2007).
</P><P><A href="/Team:Heidelberg/Project_Measurement#Measurement" title="Team:Heidelberg/Project Measurement">[TOP]</A></P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Heidelberg/Project_Measurement">http://2009.igem.org/Team:Heidelberg/Project_Measurement</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Project_Measurement" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Project_Measurement" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Project_Measurement&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Project_Measurement&amp;oldid=159425" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>