"
Page
Discussion
View source
History
teams
Log in
 
Team:Heidelberg/Notebook natural promoters
From 2009.igem.org
Home
Team
Overview
Students
Advisors
Scientific Environment
Gallery
Project
Abstract
Introduction
Measurement
Synthetic Promoters
HEARTBEAT
Further Subprojects
Summary & Outlook
References
Acknowledgements
Parts
Parts
Characterization
HEARTBEAT
What is HEARTBEAT?
HEARTBEAT Database
HEARTBEAT GUI
GUI Documentation
HEARTBEAT Fuzzy Network Modeling
Notebook
Natural Promoters
Synthetic Promoters
Cell Culture, Promoters
Stable Cell Line
Multi-Color Output
HEARTBEAT
Measurements
Material & Methods
Team Meetings
Biosafety
Eukaryopedia
Sponsors
Natural Promoters
Contents
Week
Days
33
-
8-11-2009
-
-
-
-
-
34
-
8-18-2009
8-19-2009
8-20-2009
8-21-2009
-
-
35
8-24-2009
8-25-2009
8-26-2009
8-27-2009
8-28-2009
8-29-2009
-
36
8-31-2009
9-01-2009
9-02-2009
9-03-2009
9-04-2009
-
-
37
9-07-2009
9-08-2009
9-09-2009
9-10-2009
9-11-2009
-
-
38
9-14-2009
9-15-2009
9-16-2009
9-17-2009
9-18-2009
-
9-20-2009
39
9-21-2009
9-22-2009
9-23-2009
9-24-2009
9-25-2009
9-26-2009
-
40
9-28-2009
9-29-2009
-
-
-
-
-
plasmid p31 = plasmid pSMB_MEASURE
8-11-2009
Plasmids and primers with the specicfic natural promoters had been ordered
Plasmids
Source
Plasmid with NFAT responsive promoter
Dr. C. Oakes, Heidelberg (DKFZ)
Plasmid with NFkB responsive promoter
Dr. C. Oakes, Heidelberg (DKFZ)
pJC6-GL3 with c-Jun promoter
Addgene, Cambridge
SBE4-Luc with Smad binding element promoter
Addgene,Cambridge
pGL3-RARE-Luciferase with Retinoic Acid Receptor Response Element
Addgene, Cambridge
pGL3-NFAT-Luciferase with NFAT responsive promoter
Addgene, Cambridge
PUMA-Frag1-Luc eith PUMA promoter
Addgene, Cambridge
pLDLR-Luc with LDL receptor promoter
Addgene, Cambridge
pHMGCS-Luc with HMG CoA synthase promoter
Addgene, Cambridge
p52 plasmid with HSP70 promoter
M. Schmidt and F. Hoppe, Würzburg
8-18-2009
Start of natural promoter project
The two promoters, Apolipoprotein A-IV (APOAIV) and Cytochrome P450 1A1 (CYP1A1) promoter, were amplified by PCR from human genomic DNA using primers with BBB_standard.
8-19-2009
APOAIV promoter PCR product was digested with NheI and SpeI at 37°C for one hour.
The backbone plasmid (p31; ~5000 bp) was also digested with the same restriction enzymes (at 37°C, 1 hour) and it was subsequent digested with SAP (at 37°C, 1 hour). The insert of this plasmid p31 is the promoter JeT, which will be cut out after this digest.
8-20-2009
CYP1A1 promoter has the internal restriction site, NheI,...waiting for another CYP1A1 reverse primer with the restriction site PstI instead of NheI.
The plasmid (p31) and the insert (APOAIV promotor) were purified and ligated at 22 °C for 3 hours.
The ligated product was transformed into DH5alpha cells.
8-21-2009
Picked eight colonies of p31/APOAIV plates and did a mini-preparation (miniprep)
Test digest of the eight miniprep's using the restriction enzymes SpeI and NheI (at 37°C, 1 hour)
There is an appropriate band for the APOAIV promoter (~4000bp length).
Miniprep of colony p31/ApoAIV #1, p31/ApoAIV #2 and p31/ApoAIV #3
8-24-2009
The plasmids p31/APOAIV #3 and p31/APOAIV #1 will be sequenced.
8-25-2009
APOAIV sequence did not match the published sequence (NCBI database). --> We have to repeat PCR from genomic DNA.
8-26-2009
Troubleshooting PCR for CYP1A1 and APOAIV using Taq mastermix and Phu with freshly mixed ingredients. We also used various MgCl2 concentrations (1, 2, 3, 4 ul of 25 mM MgCl2 per 50ul reaction) and performed the reaction with and without the addition of DMSO.
Analytic gel below shows CYP1A1 with Taq -/+ DMSO in the first two lanes with bands at the expected height of around 1200 bp. Lanes 1-10 are different CYP1A1 reactions, lanes 11-20 are APOAIV reactions (Fig. 1). APOAIV shows only nonspecific bands, has to be further optimized using gradient PCR or new primers.
Figure 1: 1% agarose gel electrophoresis of the PCR products of APOAIV and CYP1A1 promoter.The APOAIV and CYP1A1 promoter PCR products were analysed by agarose gel electrophoresis. Two reaction mixtures were used; one with Taq polymerase and the other one with Phu polymerase and with and without DMSO adding. Besides, the reaction mixtures of Phu have a varying concentration of MGCl2 (1-4 µl in a 50 µl reaction mixture). Additionally, a DNA ladder (100 - 10000 bp; Fermentas DNA Ladder Mix) is added into the gel for the validation of the DNA product length. The resulting CYP1A1 inserts should be 1229 bp long and APOAIV ~4200 bp. The "CYP1A1 Taq" and "CYP1A1 Taq + DMSO" are the only products, which have the right length. All the other lanes show an unspecific PCR.
8-27-2009
Test digest of CYP1A1 insert (1.2 kb) with NheI (digest at position 614 bp) at 37°C for one hour.
PCR of NFkB responsive and NFAT responsive promoter plasmids to amplify promoters from Oakes plasmids p46 and p47. Amplified using PCR high annealing protocol and primers with BBB_standard:
Step
Time
Temperature
1.
4 min
95°C
2.
30s
95°C
3.
45s
59°C
4.
2 min
72°C
5.
Go to step 2
24 times
6.
30s
95°C
7.
45s
66°C
8.
2 min
72°C
9.
for ever
4°C
8-28-2009
Digest of CYP1A1 insert with PstI and SpeI and NFkB and NFAT inserts with PstI and EcoRI at 37°C for one hour.
Digest of plasmid p31 with PstI/SpeI and PstI/EcoRI with SAP digest afterward at 37°C for one hour each.
PCR purification of digested inserts, gel purification of plasmid digests.
8-29-2009
Ligation of p31 and promoters (NFAT, NFkB) at 22°C for 3 hours.
Transformation of ligated plasmids (p31) with the promoters NFAT and NFkB in DH5alpha cells.
8-31-2009
Miniprep of p31 plasmid + NFAT/NFkB promoter.
Ligation of CYP1A1 promoter and p31 plasmid at 22°C for 3 hours.
Test digest (EcoRI and PstI) of NFAT/NFkB promoter within the p31 plasmid (Fig. 2)at 37°C for one hour.
NFAT#2 and NFAT#5 plasmids sent to sequencing.
Figure 2: 1% agarose gel electrophoresis for a test digest of the NFkB/p31 plasmid and the NFAT/p31 plasmid. The NFkB/p31 and NFAT/p31 plasmids were digested with EcoRI and PstI at 37°C for one hour. The numbers (1,3,4,5 etc.) describe the plasmids of different bacterial colonies. Additionally, a DNA ladder (100 - 10000 bp; Fermentas DNA Ladder Mix) is added into the gel for the validation of the DNA product length. The resulting NFAT insert should be 134 bp long and NFkB 179 bp. NFAT #2 and NFAT #5 seem to be successfully ligated plasmids, because there are small bands at ~ 150 bp. The lanes of NFkB #4 and NFkB #5 show a p31 plasmid band, which is 4919 bp long. But the NFkB inserts could not be recognized. Nevertheless, we will try to sequence them. All other lanes display definetly the wrong products.
9-01-2009
Transformation of CYP1A1/p31 plasmid into DH5alpha cells.
9-02-2009
Minipreparation of CYP1A1 colonies (six colonies)
Test digest of the ligated CYP1A1 construct at 37°C for one hour -> no success
APOAIV promoter: PCR of genomic DNA with old conditions
9-03-2009
Analysis with agarose gelelectrophoresis of the APOAIV-PCR product -> no success
Digest of CYP1A1 promoter and plasmid p31 with the restriction enzymes, SpeI and PstI (at 37°C, 1 hour)
PCR Purification of the CYP1A1 promoter and gel extraction of p31.
Ligation of digested CYP1A1 promoter and p31 (over night) at 16°C.
9-04-2009
PCR with the plasmid p52 to isolate the HSP70 promoter. Design primer with BBB standard, which were adjusted to the plasmid because of the lack of the p52 sequence.
PCR with the plasmid p46 to isolate the promoter with NFkB binding sites
Digest of the plasmid p31 with the restriction enzymes, EcoRI and NheI (at 37°C, 1 hour)
9-07-2009
Digest of promoter with NFkB binding sites (NFkB_prom) with NheI and EcoRI (at 37°C, 1 hour)
Digest of half of the volume of the HSP70 promoter with EcoRI and NheI (at 37°C, 1 hour)
Digest of half of the volume of the HSP70 promoter with PstI and SpeI (at 37°C, 1 hour)
Ligation of the digested NFkB_prom and the HSP70 promoter (EcoRI, NheI) with p31 (EcoRI, NheI; 09/04/09) at 22°C for three hour.
Ligation of the HSP70 promoter (EcoRI, NheI) with p31 (EcoRI, NheI; 09/04/09) at 22°C for three hour.
Ligation of the HSP70 promoter (SpeI, PstI) with p31 (SpeI, PstI; 09/03/09) at 22°C for three hour.
Transformation of the ligated products (above) and the ligated product of CYP1A1 (09/03/09)
9-08-2009
Pick some colonies of transformed bacteria (CYP1A1/p31 construct)
Miniprep of the CYP1A1/p31 construct.
Test digest of the CYP1A1/p31 construct at 37°C for one hour.
9-09-2009
Gel electrophoresis image of the test digest:
CYP1A1_promoter/p31 construct was included in three positive colonies (Fig. 3). Therefore, this construct is ready for mutagenesis PCR to remove the NheI restriction site at 614 bp.
The others have not turned out satisfactory.
Figure 3: 1% agarose gel electrophoresis for a test digest of the CYP1A1/p31 plasmid. The CYP1A1/p31 plasmids were digested with EcoRI and PstI at 37°C for one hour. The numbers (1,3,4,5 etc.) describe the plasmids of different bacterial colonies. Additionally, a DNA ladder (75 -20000 bp); Fermentas DNA 1kb Plus) is added into the gel for the validation of the DNA product length. The resulting CYP1A1 insert should be 1229 bp long. CYP1A1 #2, CYP1A1 #5 and CYP1A1 #8 seem to be successfully ligated plasmids, because there are small bands at ~ 1200 bp. All lanes show a p31 plasmid band, which is 4919 bp long. But other CYP1A1 inserts could not be recognized. All other lanes display definetly the wrong product (JeT, ~180 bp).
9-10-09
The sequence of CYP1A1 #2, #8 and #5 are right.
Plate out the Addgene plasmids: pJC6-GL3, SBE4-Luc, pGL3-RARE-Luc, pGL3-NFAT-Luc, PUMA-Frag1-Luc, pLDLR-Luc, pHMGCS-Luc
9-11-09
Pick some colonies of the plates (10.09.09) and miniprep.
9-14-2009
PCR of HSP70, NFAT responsive and NFkB responsive promoters.
Digest of the PCR products by NheI and EcoRI at 37°C for one hour.
Digest of p31_BBB by NheI and EcoRI at 37°C for one hour.
Send to sequencing: pJC6-GL3, SBE4-Luc, pGL3-RARE-Luc, pGL3-NFAT-Luc, PUMA-Frag1-Luc, pLDLR-Luc, pHMGCS-Luc
Mutagenesis PCR of CYP1A1/p31 #5.
9-15-2009
PCR-Purification with HSP70, NFkB and NFAT
Gel-extraction of p31_BBB, which was digested by NheI and EcoRI at 37°C for one hour.
Ligation: Hsp70 with p31_BBB, NFAT with p31_BBB and NFkB with p31_BBB (at 22°C, 3 hours)
PCR troubleshooting with NFAT responsive and NFkB responsive promoters.
Amplification of the p52 plasmid (with the HSP70 insert).
CYP1A1-muta/p31 PCR product was digested by DpnI (at 37°C, 1 hour).
Transformation of the three ligations (above: Hsp70/p31, NFAT/p31 and NFkB/p31) and CYP1A1-muta/p31
9-16-2009
Arrival of the sequencing results of the ordered Addgene plasmids.
Design of primers for c-Jun promoter, LDL receptor promoter (LDLR), Retinoic Acid Receptor Response Element (RARE) and HMG CoA synthase promoter (HMG CoA)
Unfortunaltely, PUMA promoter is not used because of a PstI restrcition site and there is no time for mutagenesis PCR.
p52 (with HSP70 insert) is amplified in bacteria (DH5alpha).
Miniprep of the following ligation : Hsp70 with p31_BBB, NFAT with p31_BBB and NFkB with p31_BBB
The PCR troubleshooting with NFAT and NFkB were analysed by a agarose gel and there were bright bands at ~300 bp, which are a little bit too long. Nevertheless, the inserts could be used for cloning.
9-17-2009
Test digest of the ligations p52 (with HSP70 insert), NFAT and NFkB with EcoRI and PstI (37°C, 1 hour -> no right bands!
Ligation of NFAT and NFkB (of troubleshooting PCR) with p31 plasmid at 22°C for one hour.
No colonies of CYP1A1-muta -> try again the mutagenesis PCR (with special mutagenesis Kit)
9-18-2009
Design of primers with BBB standard for the HSP70 insert by analysis of the p52 sequence.
Second mutagenesis PCR of CYP1A1
Transformation of NFAT and NFkB ligations
9-20-2009
pick colonies of NFAT and NFkB
9-21-2009
PCR of RARE, LDLR, HMG CoA synthase and c-Jun promoter using primer with BBB_standard. There were two reaction mixture for each promoter. One with Taq polymerase and one with Phu polymerase.
Miniprep of NFAT responsive and NFkB responsive promoters and test digest at 37°C for one hour -> no right bands on the agarose gel, so that we skipped this part of the project.
9-22-2009
Tranformation of CYP1A1-muta.
Analysis of the the following PCR products by agarose gel electrophoresis: RARE, LDLR, HMG CoA synthase and c-Jun promoter (Fig. 4-5).
Digest of these PCR products by the restriction enzymes, NheI and EcoRI, at 37°C for one hour.
Figure 4: 1% agarose gel electrophoresis of the PCR products of LDLR, RARE and c-Jun promoter. The LDLR, RARE and c-Jun promoter PCR products were analysed by agarose gel electrophoresis. Two reaction mixtures were used; one with Taq polymerase and the other one with Phu polymerase. Additionally, a DNA ladder (100 - 10000 bp; Fermentas DNA Ladder Mix) is added into the gel for the validation of the DNA product length. The resulting c-Jun inserts should be 341 bp long, LDLR 363 bp and RARE 212 bp. All bands have the right length and could be used for cloning.
Figure 5: 1% agarose gel electrophoresis of the PCR products of HMG CoA synthase promoter. The HMG CoA synthase promoter PCR products were analysed by agarose gel electrophoresis. Two reaction mixtures were used; one with Taq polymerase and the other one with Phu polymerase. Additionally, a DNA ladder (100 - 10000 bp; Fermentas DNA Ladder Mix) is added into the gel for the validation of the DNA product length. The resulting HMG CoA synthase inserts should be 363 bp long. All bands have the right length and could be used for cloning.
9-23-2009
Gel extraction of RARE, LDLR, HMG CoA synthase and c-Jun insert and combination of the two reaction mixtures (Taq and Phu).
Ligation of these extracted inserts with the digested (NheI and EcoRI) plasmid p31 at 22°C for four hours.
PCR of the p52 (with HSP70 promoter insert) and analysis of the PCR product by agarose gel electrophoresis.
Digest of the HSP70 promoter by NheI and EcoRI at 37°C for one hour
Transformation of the RARE/p31, LDLR/p31, c-Jun/p31 and HMG CoA promoter/p31 construct.
9-24-2009
Ligation of the HSP70 promoter with the digested (NheI and EcoRI) p31 plasmid at 22°C for three hours.
Transformation of the HSP70 promoter/p31 construct.
Colonies of the RARE/p31, LDLR/p31, c-Jun/p31 and HMG CoA promoter/p31 construct are picked.
9-25-2009
Miniprep of RARE/p31, LDLR/p31, c-Jun/p31 and HMG CoA promoter/p31 construct.
Colonies of the HSP70 promoter/p31 construct are picked.
9-26-2009
Miniprep of the HSP70 promoter/p31 construct.
Test digest (NheI and EcoRI) of RARE, c-Jun, LDLR and HMG CoA construct at 37°C for one hour (Fig. 6-10).
Following constructs were send for sequencing:
C-Jun #19, c-Jun #20, c-Jun #21, c-Jun #22, RARE #8, RARE #9, RARE #15, RARE #23, HMG CoA #24, HMG CoA #26, LDLR #3, LDLR #5, LDLR #11, LDLR #12, LDLR #19, LDLR #20, LDLR #21, LDLR #22, LDLR #24, LDLR #25, LDLR #26 and HMG CoA #35 .
Figure 6: 1% agarose gel electrophoresis of a test digest of the c-Jun/p31 plasmid constructs.The c-Jun/p31 plasmids were digested with EcoRI and PstI at 37°C for one hour. The numbers (1,3,4,5 etc.) describe the plasmids of different bacterial colonies. Additionally, a DNA ladder (100 - 10000 bp; Fermentas DNA Ladder Mix) is added into the gel for the validation of the DNA product length. The resulting c-Jun insert should be 341 bp long. C-Jun #19, c-Jun #20, c-Jun #21 and c-Jun #22 seem to be successfully ligated plasmids, because there are small bands at ~ 300 bp. All lanes show a p31 plasmid band, which is 4919 bp long. But other c-Jun inserts could not be recognized and so all other lanes display definetly the wrong product (JeT, ~180 bp).
Figure 7: 1% agarose gel electrophoresis of a test digest of the RARE/p31 plasmid constructs.The RARE/p31 plasmids were digested with EcoRI and PstI at 37°C for one hour. The numbers (1,3,4,5 etc.) describe the plasmids of different bacterial colonies. Additionally, a DNA ladder (100 - 10000 bp; Fermentas DNA Ladder Mix) is added into the gel for the validation of the DNA product length. The resulting RARE insert should be 212 bp long. The RARE inserts are difficult to recognize, because the JeT insert of the original plasmid is ~ 180 bp. Therefore, some good constructs are chosen: RARE #8, RARE #9, RARE #15 and RARE #23.
Figure 8: 1% agarose gel electrophoresis of a test digest of the HMG CoA synthase/p31 plasmid constructs.The HMG CoA synthase/p31 plasmids were digested with EcoRI and PstI at 37°C for one hour. The numbers (1,3,4,5 etc.) describe the plasmids of different bacterial colonies. Additionally, a DNA ladder (100 - 10000 bp; Fermentas DNA Ladder Mix) is added into the gel for the validation of the DNA product length. The resulting HMG CoA synthase insert should be 363 bp long. All lanes show a p31 plasmid band, which is 4919 bp long. But the HMG CoA inserts could not be recognized. All lanes display definetly the wrong product (JeT, ~180 bp).
Figure 9: 1% agarose gel electrophoresis of a test digest of the LDLR/p31 and HMG CoA synthase/p31 plasmid constructs.The LDLR/p31 and HMG CoA synthase/p31 plasmids were digested with EcoRI and PstI at 37°C for one hour. The numbers (1,3,4,5 etc.) describe the plasmids of different bacterial colonies. Additionally, a DNA ladder (100 - 10000 bp; Fermentas DNA Ladder Mix) is added into the gel for the validation of the DNA product length. The resulting LDLR insert should be 363 bp long and HMG CoA 363 bp. HMG CoA #24, HMG CoA #26, LDLR #3, LDLR #5, LDLR #11 and LDLR #12 seem to be successfully ligated plasmids, because there are small bands at ~ 350 bp. All lanes show a p31 plasmid band, which is 4919 bp long. But other HMG CoA and LDLR inserts could not be recognized. All other lanes display definetly the wrong product (JeT, ~180 bp).
Figure 10: 1% agarose gel electrophoresis of a test digest of the LDLR/p31 and HMG CoA synthase/p31 plasmid constructs.The NFkB/p31 and NFAT/p31 plasmids were digested with EcoRI and PstI at 37°C for one hour. The numbers (1,3,4,5 etc.) describe the plasmids of different bacterial colonies. Additionally, a DNA ladder (100 - 10000 bp; Fermentas DNA Ladder Mix) is added into the gel for the validation of the DNA product length. The resulting LDLR insert should be 363 bp long and HMG CoA 363 bp. LDLR #19, LDLR #20, LDLR #21, LDLR #22, LDLR #24, LDLR #25, LDLR #26 and HMG CoA #35 seem to be successfully ligated plasmids, because there are small bands at ~ 350 bp. All lanes show a p31 plasmid band, which is 4919 bp long. But other HMG CoA and LDLR inserts could not be recognized. All other lanes display definetly the wrong product (JeT, ~180 bp).
9-28-2009
Right constructs of RARE, c-Jun, LDLR and HMG CoA are: c-Jun #19, c-Jun #20, c-Jun #21, c-Jun #22, RARE #8, RARE #23, LDLR #19, LDLR #20, LDLR #21 and HMG CoA #35.
Test digest (NheI and EcoRI) of HSP70 at 37°C for one hour (Fig. 11).
Following construct is send for sequencing: HSP70 #9.
Figure 11: 1% agarose gel electrophoresis for a test digest of the HSP70/p31 plasmid construct.The HSP70/p31 plasmids were digested with EcoRI and PstI at 37°C for one hour. The numbers (1,3,4,5 etc.) describe the plasmids of different bacterial colonies. Additionally, a DNA ladder (100 - 10000 bp; Fermentas DNA Ladder Mix) is added into the gel for the validation of the DNA product length. The resulting HSP70 insert should be 402 bp long. HSP70 #9 seem to be a successfully ligated plasmid, because there is a small bands at ~ 400 bp. All lanes show a p31 plasmid band, which is 4919 bp long. But other HSP70 inserts could not be recognized and so all other lanes display definetly the wrong product (JeT, ~180 bp).
9-29-2009
Right constructs of HSP70 is: HSP70 #9.
10-12-2009
TECAN (plate reader) measurement of the natural LDL receptor promoter and the HMG CoA synthase promoter. These promoters, which are coupled to GFP, were cotransfected with a reference plasmid including JeT coupled to mCherry and were induced by Lipoprotein Deficient Serum (See: Cell Culture ).
Figure 12: The HMG CoA synthase promoter and the LDL receptor promoter induced approximately 25-35% by Lipoprotein Deficient Serum. The LDL receptor promoter and HMG CoA synthase promoter  is coupled to the fluorecent protein GFP. This GFP fluorescence was measured by TECAN once (à three replicates). Background fluorescence was substracted, and fluorescence levels are plotted relative to JeT. The standard deviations are indicated by the error bars.
Notebook information about the measurement of natural promoters is found in the Notebook Measurement and Cell Culture .
Retrieved from " http://2009.igem.org/Team:Heidelberg/Notebook_natural_promoters "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
