"
Page
Discussion
View source
History
teams
Log in
 
Team:Heidelberg/Notebook color output nb
From 2009.igem.org
Home
Team
Overview
Students
Advisors
Scientific Environment
Gallery
Project
Abstract
Introduction
Measurement
Synthetic Promoters
HEARTBEAT
Further Subprojects
Summary & Outlook
References
Acknowledgements
Parts
Parts
Characterization
HEARTBEAT
What is HEARTBEAT?
HEARTBEAT Database
HEARTBEAT GUI
GUI Documentation
HEARTBEAT Fuzzy Network Modeling
Notebook
Natural Promoters
Synthetic Promoters
Cell Culture, Promoters
Stable Cell Line
Multi-Color Output
HEARTBEAT
Measurements
Material & Methods
Team Meetings
Biosafety
Eukaryopedia
Sponsors
Contents
Week
Days
32
8-03-2009
8-04-2009
8-05-2009
8-06-2009
8-07-2009
-
-
33
-
-
8-12-2009
-
8-14-2009
-
-
34
8-17-2009
8-18-2009
8-19-2009
8-20-2009
8-21-2009
8-22-2009
-
35
8-24-2009
-
8-26-2009
8-27-2009
8-28-2009
8-29-2009
-
36
8-31-2009
9-01-2009
9-02-2009
9-03-2009
9-04-2009
-
-
37
9-07-2009
9-08-2009
9-09-2009
9-10-2009
9-11-2009
9-12-2009
-
38
9-14-2009
9-15-2009
9-16-2009
9-17-2009
9-18-2009
-
-
39
9-21-2009
9-22-2009
9-23-2009
9-24-2009
9-25-2009
9-26-2009
9-27-2009
40
9-28-2009
9-29-2009
9-30-2009
10-01-2009
10-02-2009
10-03-2009
-
41
10-05-2009
10-06-2009
10-07-2009
-
-
-
10-11-2009
42
10-12-2009
-
-
-
-
-
-
8-3-2009
Colour acquisition
got YFP/CFP-plasmids of Robert (Cellmembrane, ER, Cytosole)
Joel pledged to give us further color contructs (Cellmembrane, Mitochondria, Nucleus)
8-4-2009
Colour acquisition
transformation of all fluorescent proteins from upstairs (P.32-P.44 --> culture and miniprep tbd tomorrow)
8-5-2009
Colour acquisition
transformation of all plasmids with targeted fluorescent proteins (P.32-P.43, P.44 still missing)
made bacterial stocks of p32-43 and placed in -80°C freezer
8-6-2009
Colour acquisition
Miniprep of p32, p33, p34, p35, p36, p37, p38, p39, p40, p41, p42, p43
8-7-2009
Colour acquisition
Ordered primers for: GPI-attachement signal, Ras1b, YFP, CFP, GFP, mi
8-12-2009
Colour acquisition
ordered primers for the mutagenesis PCR of the fluorescent proteins
Plasmid
restriction site
mutagenesis position
p41
PstI
1257
p43
PstI
1001
p37-Ras
PstI
840
p37
NheI/EcoRI/PstI
18/789/840
p33
NheI/PstI/PstI/EcoRI
34/259/799/805
p35
NheI/NheI/PstI
28/67/148
8-14-2009
Colour acquisition
two mutagenesis PCR's were done (p33g259a and p37a789g)
8-17-2009
Colour acquisition
the other mutagenesis PCR's were done
and all mutagenesis PCR products were transformed into DH5alpha bacterial cells
8-18-2009
Colour acquisition
no colony because of problems with the kanamycin plates
all mutagenesis PCR's were repeated
started to test the ligation orientation of inserts into BBb plasmids:
- Digested p31 and a Jet insert with NheI and SpeI. -Ligated both together for 3 hrs at RT. (insert:vector= 1:1, 3:1, 1:3) -Transformed bacteria with ligate and plated them on Amp plates. -Repeated mutagenesis PCR again for (p33, p35, p37, p37-Ras, p41, p43)
8-19-2009
Colour acquisition
Ligation worked for all 3 plates. Picked 6 colonies for minipreping from each plate for analysis.
PCR amplification of each of GPI-attachement signal, EBFP alone, NLS alone and EBFP together with NLS.
Ran EBFP/NLS amplicons on gel and the run seemed to work.
eBFP/NLS PCR amplification
GPI-attachement signal was amplified overnight.
8-20-2009
Restriction digestion of ligation products after minipreping.
Ran PCR products for GPI on gel--> nothing!:(
Restriction digestion showed only efficiency of 5% :( (only 1/18 digests showed insertion in the opposing direction)
8-21-2009
Colour acquisition
new mutagenesis PCR (without kit)
8-22-2009
Colour acquisition
Insert Amplification of mitoneet-eGFP by PCR
8-24-2009
Colour acquisition
Restriction digest of mutagenized Plasmids (PstI) and analysis on gel
Amplified inserts were gel-purificated
What worked: eBFP, eBFP+NLS, eBFP_kozak, eBFP+NLS_kozak, eGFP, eGFP_kozak
What didn't: NLS, NLS_kozak, eGFP+mitomeet, mitomeet, eGFP+mitomeet_kozak, mitomeet_kozak,
8-26-2009
BBBing of Insertsequences
PCR of cherry, cherry_myrpalm, myrpalm, NLS with kozak Primers to amplify cherry_kozak, cherry_myrpalm_kozak, myrpalm_kozak, NLS_kozak
Restriction with NheI and SpeI of localisationsequences and Flourophores, Restricted Plasmid was provided by Synthetic Promoter Group and digested with SAP
Ligation with p31
Transformation in DH5alpha with ligated Plasmids
Outplating of Transformed cells on Amp-plates
8-27-2009
Ligation and Transformation did not work (no colonies, except of two on the NLS )
New PCR with flourophores and localisationsequences, to get higher amounts
GEl purification of: eGFP, eGFP_kozak, eBFP, eBFP_NLS, eBFP_kozak, eBFP_NLS_kozak, NLS_kozak, cherry, cherry_myrpalm, myrpalm, cherry_kozak, cherry_myrpalm_kozak, myrpalm_kozak
8-28-2009
BBBing of Insertsequences2.0
Restrictiondigest of flourophores and localisationsequences with SpeI and NheI (1 h, Buffer 2, BSA)
Restrictiondigest of p49 with SpeI and NheI (1 h, Buffer 2, BSA) and SAP (30 min), purification
Nanodrop of digest shows no DNA inside of the samples -- purification was maybe unsuccessful
8-29-2009
BBBing of Insertsequences2.1
Restrictiondigest of flourophores and localisationsequences with SpeI and NheI (1 h, Buffer 2, BSA)
Restrictiondigest of p49 with SpeI and NheI (1 h, Buffer 2, BSA) and SAP (30 min), purification
8-31-2009
BBBing of Insertsequences2.1 (part 2)
Ligation of Insertsequences with restricted p49
Transformation
Outplating -> Wrong resistance
9-01-2009
BBBing of Insertsequences2.1 (part 3)
Transformation
Plated out on Ampicillin
9-02-2009
Mutagenesis PCR
sequences for mutagenesis PCR arrived: P37-Ras, P33, P43 (worked with a possible insertion, so must be tested by transformation to ensure that no alteration occured to the construct that could interfere with the designated function).
Second mutagenesiscycle of the mutagenized p37-Ras
BBBing of Insertsequences2.1 (part 3)
Picked colonies from the plated out ligations.
Plated out more bacteria to enhance the number because of the recorded low efficiency of ligation in the correct orientation.
9-03-2009
BBBing of Insertsequences2.1
cloning strategy failed, NheI and SpeI cloning system is not effective enough, gel purification of p49 went wrong (Jet was still in)
New Primers with PstI or EcoRI are neccessary.
9-04-2009
Mutagenesis PCR of FP/LS
Ran Mutagenesis PCR for:
P41 (PstI, primers: 3,4)                          P37 (PstI, primers: 40,41)                          P35 (PstI, primers: 53,54)                          P33 (2nd site:primers 48/48')
Transformed bacteria to make more P37, p35, p41--> didnt work when analyzed on gel!
BBBing of polyA sequence
PCR amplification of PolyA out of p6 (PRimers have complete BBB overhang)
PCR purification
restriction digest with EcoRI and PstI of backbone (p49) and PolyA
gel purification of restricted backbone, PCR purification of PolyA
Ligation
Transformation
9-07-2009
Mutagenesis PCR of FP/LS
Ran Mutagenesis PCR for:
P41 (PstI, primers: 3,4)                          P37 (PstI, primers: 40,41)                          P35 (PstI, primers: 53,54)                          P33 (2nd site:primers 48/48')
--> Seemed to have worked on the gel :) so bacteria were transformed.
9-08-09
Mutagenesis PCR of FP/LS
Ran Mutagenesis PCR for:
P41 (PstI, primers: 3,4)                          P37 (PstI, primers: 40,41)                          P35 (PstI, primers: 53,54)                          P33 (2nd site:primers 48/48')
-This time, ran it with a 1:10 dilution of primers, and in each with 3 different plasmid DNA concentrations (10, 20, 50 ng). Also, control plasmid was run with them and a volume of 15 microlitres was used for transformation, although in protocol provided it is mentioned that only 1micro is needed with supercompetent cells. --> 24hrs later, no colonies are yet present on any of the plates, including the control!
*Picked a colony from each of p41, p49, p37 and p35 and grew in corresponding resistance LB.
9-09-2009
BBBing 2nd gen. of GFP
PCR with new Primers
Mutagenesis PCR
-Plates from Monday's trials were picked. very few colonies for p33-mutax2, p37, p35 but none for p41.
-Ran PCR for amplification of Ras and GPI-anchoring signal.
9-10-2009
BBBing 2nd gen. of GFP
restriction with SpeI and PstI
Ligation
BBBing 2nd gen. mCherry
PCR with new Primers to get:mCherry, mCherry+myrpalm, myrpalm, mCherry_kozak, mCherry+myrpalm_kozak, myrpalm_kozak, NLS
BBBing of PolyA
PCR with Primers 197 and 198
9-11-2009
BBBing 2nd gen. of GFP
Transformation and outplating
BBBing 2nd gen. mCherry
Gelpurification
Restriction with SpeI and PstI
Gelpurification
BBBing of PolyA
Restriction of PolyA PCR-product with EcoRI and NheI
Restriction of both p31new and p55 with EcoRI and SpeI
Gelpurification
Mutagenesis PCR
Transformation of bacteria with just 5 microlitres of mutagenesis product because it proved that it was optimal for the control.
--> colonies grew and were picked!
9-12-2009
BBBing 2nd gen. of GFP
Picked colonies
9-14-2009
BBBing 2nd gen. of GFP
Miniprep and restriction digest (PstI and EcoRI)
Gelanalysis shows a positive BBBing of eGFP and eGFP_kzk
BBBing 2nd gen. mCherry
Ligation of Inserts in p49
Transformation and plating out
BBBing of PolyA
Ligation in p31new and p55
Transformation and plating out
BBbing of inserts (into p49)
Test digested the ligation products of Ras and GPI-targetting sequence PCRed from from the mutated plasmids.
--> positive!!!
Mutagenesis PCR
Sequencing results for 10.09.09's sequences show that:
- p37 mutants worked but together with other mutations that seem to have arised. need for a test transfection!
-p35 mutants worked but with a 2 base deletion. need for a test transfection!
-p33 mutants worked but with funny point mutations. need for a test transfection!
Ran a mutagenesis PCR with a normal Taq polymerase, because mutagenesis Turbo mix was finished.
-p37 (42/43)
-p35 (51/52)
-p33 (44/45)
Sequencing primer addition to p31:
-Ran a PCR using Highannealing program of p31 with 201, 202. -->PCR worked
9-15-2009
BBBing 2nd gen. mCherry
Picked colonies
BBBing of PolyA
Picked colonies
Sequencing primer addition to p31
- Digested p31 with EcoRI and PstI. -Digest Purified PCR product with NsiI and Mhe-HF--> too little on gel for a second digest. Must redo with more amounts of PCR product tomorrow.
Mutagenesis PCR
-Transformed bacteria with 5 micro litre of mutagenesis mix-->nothing grew
9-16-2009
Sequencing primer addition to p31
-Did a digest with Mfe-1.
-Digest of P31-new looked funny. purified the plasmid and redigested it, because DNA seemed to be a lot and needed to make sure all was digested.
9-17-2009
-Digested PCR amplicon with Nsi1.
-Redigested BB.
--> Gel purified amplicon and BB and ligated at 16 degrees overnight.
-Prepared Maxiprep cultures of submission plasmid 29-new.
9-18-2009
-Digested P31-new with EcoRI and PstI to isolate jet insert.
-Digested submission plasmid with EcoRI and PstI.
-->ligate intron into p29new and jet into p29new.
CMV from p48 kept for later use.
9-21-2009
Biobricking
CMV - cherry
CMV - myrcherry
CMV - kzkcherry
CMV - eGFP
CMV - kzkeGFP
subm(er)izing:
All transformations worked and test digests show that we have:
-Intron
-Jet (S1)
-PolyA tail
-->problem: No sequencing primers available for verification yet, so nothing is assured, but cloning will be resumed with available components.
-Nevertheless, used one of the Jet submission plasmids (Jet1) to do the subcloning of a CMV core promoter with the Jet proximal.
-->digested S1 with hindIII and NheI
Cloning of sequencing primers into p31:
-Ligation into p31 didn't work. No more PCR product is left, so ran PCR again.-->worked.
-In the meantime, ligation was tried again with different molar ratios (1:1, 1:3, 3:1) --> no colonies grew.
Mutagenesis PCR:
-Didn't work :(((
9-22-2009
Biobricking Preps of the Ligations
Subm(er)izing:
-Ligated CMV-core (from p48) into digested S1.
-->Problem: used the undigested plasmid instead of digested plasmid, so this failed.
Cloning of sequencing primers into P31:
-Transformed ligations from day before --> didn't work
- Digested the new PCR product with NsiI and MfeI using two protocols for the sequential digest.
I:
Digest with Mfe-I HF in buffer 3 (low salt) then add NsiI in buffer 4.
II:
Digest with Mfe-I, PCR purify and then redigest in NsiI.
-Ligated new PCR digest into the old p31.
9-23-2009
Biobricking Restriction, Purification, Ligation, Transformation of:
cherry - PolyA
myrcherry - PolyA
kzkcherry - PolyA
eGFP - PolyA
kzkeGFP - PolyA
CMV - myrpalm
Subm(er)izing
-Digest SI with HindIII and NheI.
-Ligated CMV into SI.
-Transformed and Plated BBBed Sar and GPI on Kana plates. (wrong selection, failed!!!)
Mutagenesis PCR
-One colony grew.
Cloning sequencing primers into P31
-Ligated the new PCR product digest into the new p31 digest.
9-24-2009
Mutagenesis PCR
Colonies grew on the 10ng plate of p41
Cloning of sequencing primers into p31
Transformed bacteria with the ligation from the day before.
Subm(er)izing
- Transformed the CMV/S1 ligation
-Replated GPI and Sar BBBed plasmids on Amp.
BioBricking Ligation
cmv-cherry + PolyA
cmv-myrcherry + PolyA
cmv-kzkcherry + PolyA
cmv-eGFP + PolyA
cmv-kzkeGFP + PolyA
cmv-NLS
9-25-2009
BioBricking Transformation of yesterdays Ligation
Ligation of:
cmv-myrpalm + cherry-PolyA
cmv-myrpalm + kzkcherry-PolyA
cmv-myrpalm + eGFP-PolyA
cmv-myrpalm + kzkeGFP-PolyA
cmv-myrpalm + myrcherry-PolyA
Mutagenesis PCR
Minipreped picked colonies for p41 (2x). Only one of which grew in LB. Sent for sequencing.
Sequencing primers in p31: -minipreped picked colonies.
-transformed new ligations.
Subm(er)izing:
-Minipreped picked clones for S21(Jet core and proximal)
-Transformed S2 ligations (CMV core and Jet proximal)
!!!Problem: Labels on samples got wiped off during minipreping.
Digested all samples with 3 different protocols:
1) EcoRI and PstI: S1 should give an insert of ~200 bps. P41 muta shouldn't be cut.
2) Hind III and NotI: S1 should be digested and so should p41 muta (difference in lenght of backbone and p41 should have a ~500 bp insert)
3)Hind III (same for p31-neu): same as Hind III and NotI but no insert should be detectable. (only linearised fragments of different lenghts).
--> Gel showed abnormal mixture of multiple fragments for all digests. Unassociatable with any of the expected plasmids. possible that plates were left too long in incubator and other bacteria grew on plates.
9-26-2009
Miniprep and testdigest of yesterdays Transformations
Transformation of yesterdays Ligations
Subm(er)izing:
-Made minipreps of: -p37and p34 amplicons in p49 for minipreps of enough DNA to be digested and used for cloning into submission.
--> test digest with EcoRI and PstI shows that the plasmids have not the expected insert. Possible that the plates were too overcrowded and the picked colonies were ones that were negative.
-Made minipreps of: S2 bacterial suspensions.
--> test digest with Hind and PstI shows that the colonies lack the correct insert.
Sequencing primers for p31:
-Made minipreps of picked colonies.
--> Test digest with EcoRI and PstI shows that they have the correct insert. 4 samples were picked for sequencing.
9-27-2009
Miniprep and testdigest of yesterdays Transformations
9-28-2009
Transfektion of:
* cmv-cherry + PolyA    * cmv-myrcherry + PolyA    * cmv-kzkcherry + PolyA    * cmv-eGFP + PolyA    * cmv-kzkeGFP + PolyA        * cmv-myrpalm + cherry-PolyA    * cmv-myrpalm + kzkcherry-PolyA    * cmv-myrpalm + eGFP-PolyA    * cmv-myrpalm + kzkeGFP-PolyA    * cmv-myrpalm + myrcherry-PolyA
Lokalisation of eGFP with myrpalm worked well, unfortunately cherry and kzkcherry did not work
Biobricking of :
* CMV-NLS-cherry-PolyA   * CMV-NLS-kzkcherry-PolyA   * CMV-NLS-myrcherry-PolyA   * CMV-NLS-eGFP-PolyA   * CMV-NLS-kzkeGFPPolyA
9-29-2009
Transformation of:
* CMV-NLS-cherry-PolyA   * CMV-NLS-kzkcherry-PolyA   * CMV-NLS-myrcherry-PolyA   * CMV-NLS-eGFP-PolyA   * CMV-NLS-kzkeGFP-PolyA
9-30-2009
Subm(er)izing:
-Test digested S2 colonies minipreped before with NheI and Hind III to make sure that the digest was negative for reasons got to do with the unfamiliar digestion protocol used.
--> Colonies were still negative.
-Found BBBed inserts of P37 and P34 amplicons and ligated them into submission plasmid digested with SpeI and PstI.
-Re-did the CMV/SI ligation.
-Restreaked p29-neu.
-Picked more S2 colonies.
10-01-2009
Transfection of
* CMV-NLS-cherry-PolyA   * CMV-NLS-kzkcherry-PolyA   * CMV-NLS-myrcherry-PolyA   * CMV-NLS-eGFP-PolyA   * CMV-NLS-kzkeGFPPolyA
Transfection of:
* CMV-NLS-cherry-PolyA   * CMV-NLS-kzkcherry-PolyA   * CMV-NLS-myrcherry-PolyA   * CMV-NLS-eGFP-PolyA   * CMV-NLS-kzkeGFP-PolyA
Subm(er)izing
-Transformed ligations of p37 and p34 amplicons and CMV/SI.
-Minipreped the picked colonies of S2 and test digested them with HindIII and PstI and EcoRI and PstI. (negative for insert)
-Digested Lars's L and S series of promoters with NheI and PstI.
!!!Problem These were not BBBed parts, and needed to be cut with SpeI and HindIII. Needed to gel extract the uncut plasmids since they were the last available without need to plate out glycerol stocks.
-Digested Corri's natural promoters with EcoRI and PstI.
--> Gel extracted the parts.
10-02-2009
Transfection result:
NLS does not work, as well as  mCherry. plasma membrane localized mCherry works.
NLS restart
New PCR to get BBB sites.
Followed by the assembly with cmv-flourophores to get
*cmv-flourophore-NLS
Subm(er)izing:
-Digested Lars's plasmids with SpeI and HindIII.
-Picked colonies for P34 and P37 and CMV ligations in SI.
10-03-2009
Subemerizing
-Minipreped:
p37 and p34 in SI.
p29 neu
S2
-Ran digest of Lars's samples on gel for extraction. --> Gel broke with some of the S series some missing!
10-05-2009
NLS
Miniprep of cmv-flourophore-NLS
Biobricking to get cmv.flourophore-NLS-PolyA
PolyA
Subm(er)izing
10-06-2009
Restart PolyA-submerizing
Transformation and picking of NLS-Test-constructs
10-07-2009
Time lapse fluorescence movie
Plan is to make a time lapse movie of induction of NFKb-promoter NIIL10
Cloning of Videobrickdevices
*Jet-myrcherry-PolyA   *NIIL10-eGFP-PolyA
restriction and ligation
10-11-2009
Time lapse fluorescence movie
Transfection of Videobrickdevices
10-12-2009
Time lapse fluorescence movie starting acquisition
BioBricking of GPI
Restriction, Extraction and Ligation to get the following constructs
*cmv-GPI-eGFP-PolyA   *cmv-eGFP-GPI-PolyA
Time lapse fluorescence movie cells disattached after 4 h so there was no usable filmmaterial
restart acquisition this evening --> less cells, therefore no acquisition
Retrieved from " http://2009.igem.org/Team:Heidelberg/Notebook_color_output_nb "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
