<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Notebook_MaM"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Heidelberg/Notebook_MaM">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Notebook_MaM&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook_MaM&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook_MaM&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Notebook_MaM" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Notebook MaM</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="main_wrapper"><DIV id="header"><DIV id="navigation"><DIV id="menu" style="position:static"><UL><LI id="mainHome"><A class="aMain" href="https://2009.igem.org/Team:Heidelberg">Home</A></LI><LI id="mainTeam"><A class="aTeam" href="https://2009.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Members#Students">Students</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Members#Advisors">Advisors</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Scientific_Environment">Scientific Environment</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Gallery">Gallery</A></LI></UL><LI id="mainProject"><A class="aProject" href="https://2009.igem.org/Team:Heidelberg/Project">Project</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project">Abstract</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Introduction">Introduction</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Measurement">Measurement</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Synthetic_promoters">Synthetic Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Further">Further Subprojects</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_SaO">Summary &amp; Outlook</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_References">References</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Acknowledgement">Acknowledgements</A></LI><LI id="mainParts"><A class="aParts" href="https://2009.igem.org/Team:Heidelberg/Parts">Parts</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Parts">Parts</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Parts_Characterization">Characterization</A></LI><LI id="mainModeling"><A class="aModeling" href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">What is HEARTBEAT?</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_database">HEARTBEAT Database</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_GUI">HEARTBEAT GUI</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_gui">GUI Documentation</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_network">HEARTBEAT Fuzzy Network Modeling</A></LI><LI id="mainNotebook"><A class="aNotebook" href="https://2009.igem.org/Team:Heidelberg/Notebook">Notebook</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_natural_promoters_overview">Natural Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_synthetic_promoters">Synthetic Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_promoters_cells">Cell Culture, Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_cell_line">Stable Cell Line</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_color_output">Multi-Color Output</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_modeling">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_measure">Measurements</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_MaM">Material &amp; Methods</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_meetings">Team Meetings</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Biosafety">Biosafety</A></LI><LI id="mainEucaryopedia"><A class="aEucaryopedia" href="https://2009.igem.org/Team:Heidelberg/Eukaryopedia">Eukaryopedia</A></LI><LI id="mainSponsors"><A class="aSponsors" href="https://2009.igem.org/Team:Heidelberg/Sponsors">Sponsors</A></LI></DIV></DIV></DIV></DIV><TABLE><TBODY><TR valign="top" style="margin-left: 2px;"><TD width="650px" style="padding: 0 15px 15px 20px; background-color:#ede8e2"><H1><SPAN class="mw-headline" id="Materials_.26_Methods">Materials &amp; Methods</SPAN></H1><H2><SPAN class="mw-headline" id="Kits">Kits</SPAN></H2></TD></TR><TR style="background-color:#99cccc;"><TH height="20px," width="350px"> Kit </TH><TH width="350px"> supplier
</TH></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">CompactPrep Plasmid Maxi Kit </TD><TD> QIAGEN
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">High Pure PCR Template Preparation Kit </TD><TD> Roche
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">HISpeed Plasmid Maxi Kit </TD><TD> QIAGEN
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">QIAprep Spin Mini Kit  </TD><TD> QIAGEN
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">QIAquick Gel Extraction Kit </TD><TD> QIAGEN
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">QIAquick PCR Purification Kit </TD><TD> QIAGEN
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">RNeasy Plant Mini Kit </TD><TD> QIAGEN
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">QuantiTect Probe RT-PCR Kit </TD><TD> QIAGEN
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">SensiMix One-Step Kit </TD><TD> Quantace
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">QuikChange XL Site-Directed Mutagenesis Kit </TD><TD> Strategene
</TD></TR></TBODY></TABLE><P><A href="/Team:Heidelberg/Notebook_MaM#Material" title="Team:Heidelberg/Notebook MaM">[Back]</A></P><H2><SPAN class="mw-headline" id="Marker">Marker</SPAN></H2><TABLE class="wikitable centered" style="margin: 1em auto 1em auto"><TBODY><TR style="background-color:#99cccc;"><TH height="20px," width="350px"> Marker </TH><TH width="350px"> supplier
</TH></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">GeneRuler™ High Range DNA Ladder </TD><TD> MBI Fermentas
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">GeneRuler™ 1kb DNA Ladder Mix </TD><TD> MBI Fermentas
</TD></TR></TBODY></TABLE><P><A href="/Team:Heidelberg/Notebook_MaM#Material" title="Team:Heidelberg/Notebook MaM">[Back]</A></P><H2><SPAN class="mw-headline" id="Enzymes">Enzymes</SPAN></H2><TABLE class="wikitable centered" style="margin: 1em auto 1em auto"><TBODY><TR style="background-color:#99cccc;"><TH height="20px," width="350px"> Enzym </TH><TH width="350px"> supplier
</TH></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">Pfu DNA polymerase </TD><TD> Stratagene
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">Phusion DNA polymerase, Mastermix 2x </TD><TD> Finnzymes
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">Taq DNA polymerase </TD><TD> MBI Fermentas
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">T4 DNA ligase </TD><TD> MBI Fermentas / New England Biolabs
</TD></TR></TBODY></TABLE><P><A href="/Team:Heidelberg/Notebook_MaM#Material" title="Team:Heidelberg/Notebook MaM">[Back]</A></P><H2><SPAN class="mw-headline" id="Restriction_Enzymes">Restriction Enzymes</SPAN></H2><TABLE class="wikitable centered" style="margin: 1em auto 1em auto"><TBODY><TR style="background-color:#99cccc;"><TH height="20px," width="350px"> Enzym </TH><TH width="350px"> supplier
</TH></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">BclI </TD><TD> MBI Fermentas
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">EcoRI </TD><TD> New England Biolabs
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">HindIII </TD><TD> New England Biolabs
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">MfeI </TD><TD> New England Biolabs
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">NheI </TD><TD> New England Biolabs
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">NotI </TD><TD> New England Biolabs
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">PstI </TD><TD> New England Biolabs
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">ScaI </TD><TD> New England Biolabs
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">SpeI </TD><TD> New England Biolabs
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">SspI </TD><TD> New England Biolabs
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">SphI </TD><TD> New England Biolabs
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">XbaI </TD><TD> New England Biolabs
</TD></TR></TBODY></TABLE><P><A href="/Team:Heidelberg/Notebook_MaM#Material" title="Team:Heidelberg/Notebook MaM">[Back]</A></P><H2><SPAN class="mw-headline" id="Bacteria">Bacteria</SPAN></H2><TABLE class="wikitable centered" style="margin: 1em auto 1em auto"><TBODY><TR style="background-color:#99cccc;"><TH height="20px," width="200px"> E.coli strain </TH><TH width="250px"> usage </TH><TH width="250px"> reference
</TH></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">DH5a </TD><TD> amplification of plasmids </TD><TD> Invitrogen
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">SCS110 </TD><TD> amplification of unmethylated plasmids </TD><TD> Stratagene
</TD></TR></TBODY></TABLE><P><A href="/Team:Heidelberg/Notebook_MaM#Material" title="Team:Heidelberg/Notebook MaM">[Back]</A></P><H2><SPAN class="mw-headline" id="Bacteria_Growth_Media">Bacteria Growth Media</SPAN></H2><P><B>Luria Broth (LB)</B></P><UL><LI> 10 g/L tryptone
</LI><LI> 5 g/L yeast extract
</LI><LI> 10 g/L  NaCl 
</LI></UL><TABLE class="wikitable centered" style="margin: 1em auto 1em auto"><TBODY><TR style="background-color:#99cccc;"><TH height="20px," width="200px"> Antibiotic </TH><TH width="250px"> concentration
</TH></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">Ampicillin </TD><TD> 100 mg/ml
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">Kanamycin </TD><TD> 50 mg/ml
</TD></TR></TBODY></TABLE><P><A href="/Team:Heidelberg/Notebook_MaM#Material" title="Team:Heidelberg/Notebook MaM">[Back]</A></P><H2><SPAN class="mw-headline" id="Eucaryotic_cell_lines">Eucaryotic cell lines</SPAN></H2><UL><LI><B><A href="/Team:Heidelberg/Eukaryopedia#MCF-7" title="Team:Heidelberg/Eukaryopedia"> MCF-7</A></B></LI><LI><B><A href="/Team:Heidelberg/Eukaryopedia#HeLa" title="Team:Heidelberg/Eukaryopedia"> HeLa</A></B></LI><LI><B><A href="/Team:Heidelberg/Eukaryopedia#U2-OS" title="Team:Heidelberg/Eukaryopedia"> U2-OS</A></B></LI></UL><P><A href="/Team:Heidelberg/Notebook_MaM#Material" title="Team:Heidelberg/Notebook MaM">[Back]</A></P><H2><SPAN class="mw-headline" id="Cell_Culture_Media">Cell Culture Media</SPAN></H2><H3><SPAN class="mw-headline" id="DMEM.2B.2B.2B">DMEM+++</SPAN></H3><UL><LI> DMEM
</LI><LI> 1% L-Glutamine
</LI><LI> 1% Pen/Strep
</LI><LI> 10% FCS
</LI></UL><H3><SPAN class="mw-headline" id="DMEM.2B.2B">DMEM++</SPAN></H3><UL><LI> DMEM
</LI><LI> 1% L-Glutamin
</LI><LI> 1% Pen/Strep
</LI><LI> 1% non-essential aminoacids
</LI></UL><H3><SPAN class="mw-headline" id="Zeocin-Medium">Zeocin-Medium</SPAN></H3><UL><LI> DMEM+++
</LI><LI> 100 µl/ml Zeocin
</LI></UL><H3><SPAN class="mw-headline" id="Hygromycin-Medium">Hygromycin-Medium</SPAN></H3><UL><LI> DMEM+++
</LI><LI> 200 µl/ml Hygromycin
</LI></UL><H3><SPAN class="mw-headline" id="Minimal_Medium">Minimal Medium</SPAN></H3><UL><LI> 2,1 g/l NaHCO<SUB>3</SUB></LI><LI> 1µM CaCl<SUB>2</SUB></LI><LI> 10 µM Hepes-Buffer <A href="https://www.sigmaaldrich.com/catalog/ProductDetail.do?D7=0&amp;N5=SEARCH_CONCAT_PNO%7CBRAND_KEY&amp;N4=70216%7CFLUKA&amp;N25=0&amp;QS=ON&amp;F=SPEC" class="external autonumber" rel="nofollow">[1]</A></LI><LI> 1 % Pen/Strep
</LI><LI> Krebs-Henseleit-Buffer (Sigma-Aldrich <A href="http://www.sigmaaldrich.com/etc/medialib/docs/Sigma/Product_Information_Sheet/1/k3753pis.Par.0001.File.tmp/k3753pis.pdf" class="external autonumber" rel="nofollow">[2]</A>)</LI></UL><P><A href="/Team:Heidelberg/Notebook_MaM#Material" title="Team:Heidelberg/Notebook MaM">[Back]</A></P><H2><SPAN class="mw-headline" id="Oligos"> Oligos </SPAN></H2><P>We used <A href="http://www.lablife.org" class="external text" rel="nofollow">LabLife</A>, a great, non-commercial lab management resource for lab management and especially for keeping track of Oligos we used. All Oligos therefore are numbered, and all Notebook entries describing PCR reactions refer to a certain number of Oligo. A complete list of all Oligos we had in our lab can be downloaded <A href="/wiki/images/e/e1/HD09_complete_oligo_list.txt" class="internal" title="HD09 complete oligo list.txt">as comma-separated values</A>; find below a list of more interesting oligos we used for RA-PCR and LAM-PCR.
</P><H3><SPAN class="mw-headline" id="Oligos_used_for_RA-PCR">Oligos used for <A href="/Team:Heidelberg/Project_Synthetic_promoters" title="Team:Heidelberg/Project Synthetic promoters">RA-PCR</A></SPAN></H3><TABLE class="wikitable centered" style="margin: 1em auto 1em auto"><TBODY><TR style="background-color:#99cccc;"><TH> Name
</TH><TH> Sequence
</TH></TR><TR style="background-color:#9BDDFF;"><TD width="300px">Stop 5'
</TD><TD width="500px">CAGTACTAGTGGGTGACGGGTTCA
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">Stop 3'
</TD><TD width="500px">TGACAAGCTTAGTGAACCCGTCACCC
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">Sp1 responsive (general transactivator)
</TD><TD width="500px">GCGATCGGCAGATCAGGGGCGGGGCGGGTGACGGGTTCA
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">Ap1 responsive (general transactivator)
</TD><TD width="500px">GCGATCGGCAGATCATGACTCAGGGTGACGGGTTCA
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">CREB responsive (general transactivator)
</TD><TD width="500px">GCGATCGGCAGATCABNBVNTGACGTCAGGGTGACGGGTTCA
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">NFY responsive (general transactivator)
</TD><TD width="500px">GCGATCGGCAGATCANNCCAATNNGGGTGACGGGTTCA
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">Spacer forward
</TD><TD width="500px">GCGATCGGCAGATCANNNNNNNNNNGGGTGACGGGTTCA
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">Spacer reverse
</TD><TD width="500px">TGATCTGCCGATCGCNNNNNNNNNNTGAACCCGTCACCC
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">AHR responsive, forwards
</TD><TD width="500px">GCGATCGGCAGATCACCYCNRRSTNGCGTGASAGGGTGACGGGTTCA
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">AHR responsive, reverse
</TD><TD width="500px">TGATCTGCCGATCGCASAGTGCGNTSRRNCYCCTGAACCCGTCACCC
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">Estrogen Receptor responsive, forward v 1
</TD><TD width="500px">GCGATCGGCAGATCAGGTCACTGTGACCGGGTGACGGGTTCA
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">Estrogen Receptor responsive, forward v 2
</TD><TD width="500px">GCGATCGGCAGATCARGNCANNNTGACCYGGGTGACGGGTTCA
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">Estrogen Receptor responsive, reverse
</TD><TD width="500px">TGATCTGCCGATCGCCCAGTGTCACTGGTGAACCCGTCACCC
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">HIF responsive, forward v 1
</TD><TD width="500px">GCGATCGGCAGATCATCTGTACGTGACCACAGGGTGACGGGTTCA
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">HIF responsive, forward v 2
</TD><TD width="500px">GCGATCGGCAGATCAGTCTACGTGCGGACGGGTGACGGGTTCA
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">NFκB responsive, forward v 1
</TD><TD width="500px">GCGATCGGCAGATCAGGGGACTTTGCCGGGTGACGGGTTCA
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">NFκB responsive, forward v 2
</TD><TD width="500px">GCGATCGGCAGATCAGGGGRWYYCCCGGGTGACGGGTTCA
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">NFκB responsive, reverse
</TD><TD width="500px">TGATCTGCCGATCGCCCGTTTCAGGGGTGAACCCGTCACCC
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">p53 responsive, forward
</TD><TD width="500px">GCGATCGGCAGATCAGAACATGTCCAAGCATGCTGGGGTGACGGGTTCA
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">PPARγ responsive, reverse
</TD><TD width="500px">TGATCTGCCGATCGCHACTGGAAACTGGRRTGAACCCGTCACCC
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">PPARγ responsive, forward
</TD><TD width="500px">GCGATCGGCAGATCARRGGTCAAAGGTCAHGGGTGACGGGTTCA
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">SREBP responsive, forward
</TD><TD width="500px">GCGATCGGCAGATCAKATCACCCCACGGGTGACGGGTTCA
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">SREBP responsive, reverse
</TD><TD width="500px">TGATCTGCCGATCGCCACCCCACTAKTGAACCCGTCACCC
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">p53 reverse
</TD><TD width="500px">TGATCTGCCGATCGCGTCGTACGAACCTGTACAAGTGAACCCGTCACCC
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">HIF reverse
</TD><TD width="500px">TGATCTGCCGATCGCGATCCGTCGTGCATCAGTGTGAACCCGTCACC
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Oligos_for_LAM-PCR">Oligos for LAM-PCR</SPAN></H3><TABLE class="wikitable centered" style="margin: 1em auto 1em auto"><TBODY><TR style="background-color:#99cccc;"><TH> Name
</TH><TH> Sequence
</TH></TR><TR style="background-color:#9BDDFF;"><TD width="300px">LAMPCR9_1 (biotinylated)
</TD><TD width="500px">AATACCGCGCCACATAGCAG
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">LAMPCR9_2 (biotinylated)
</TD><TD width="500px">TTCTTCGGGGCGAAAACTCT</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">LAMPCR9_3 (biotinylated)
</TD><TD width="500px">CACTCGTGCACCCAACTGAT
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">LAMPCR9_4
</TD><TD width="500px">TCTGGGTGAGCAAAAACAGGA
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Oligos_for_real-time_RT-PCR">Oligos for real-time RT-PCR</SPAN></H3><TABLE class="wikitable centered" style="margin: 1em auto 1em auto"><TBODY><TR style="background-color:#99cccc;"><TH> Name
</TH><TH> Sequence
</TH></TR><TR style="background-color:#9BDDFF;"><TD width="300px">GAPDH_forward_primer
</TD><TD width="500px">GGTATCGTGGAAGGACTCATGACC
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">GAPDH_reverse_primer
</TD><TD width="500px">GATGACCTTGCCCACAGCCTTG
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">GAPDH_probe
</TD><TD width="500px">CTCTCCAGAACATCATCCCTGCCTCTAC
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">G6PD_forward_primer
</TD><TD width="500px">GAGTCCTGCATGAGCCAGATAGG
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">G6PD_reverse_primer
</TD><TD width="500px">GCACCATGAGGTTCTGCACCATC
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">G6PD_probe
</TD><TD width="500px">ACCAGATCTACCGCATCGACCACTACCT
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">18s_rRNA_forward_primer
</TD><TD width="500px">GGCCCTGTAATTGGAATGAGTCCAC
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">18s_rRNA_reverse_primer
</TD><TD width="500px">GCTCCCAAGATCCAACTACGAGCTT
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">18s_rRNA_probe
</TD><TD width="500px">CAGCAGCCGCGGTAATTCCAGCTCC
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">G2TUB_forward_primer
</TD><TD width="500px">TTCTACCAGGCAGACGATGAGCAC
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">G2TUB_reverse_primer
</TD><TD width="500px">CCCAGTTGTTGCCAGCTCCTC
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">G2TUB_probe
</TD><TD width="500px">CTGCTGGACTTGGAACCCCGGGTG
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">bACT_forward_primer
</TD><TD width="500px">CAGGCACCAGGGCGTGATGG
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">bACT_reverse_primer
</TD><TD width="500px">CTCCATGTCGTCCCAGTTGGTG
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">bACT_probe
</TD><TD width="500px">AGGATTCCTATGTGGGCGACGAGGC
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">eGFP_forward_primer
</TD><TD width="500px">CGACCACATGAAGCAGCACGAC
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">eGFP_reverse_primer
</TD><TD width="500px">CGATGCCCTTCAGCTCGATGC
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">eGFP_probe
</TD><TD width="500px">AGGCTACGTCCAGGAGCGCACCATC
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">mCherry_forward_primer
</TD><TD width="500px">CAAGTGGGAGCGCGTGATGAAC
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">mCherry_reverse_primer
</TD><TD width="500px">CCAGCCCATGGTCTTCTTCTGC
</TD></TR><TR style="background-color:#9BDDFF;"><TD width="300px">mCherry_probe
</TD><TD width="500px">TGACCGTGACCCAGGACTCCTCCC
</TD></TR></TBODY></TABLE><P><A href="/Team:Heidelberg/Notebook_MaM#Material" title="Team:Heidelberg/Notebook MaM">[Back]</A></P><H2><SPAN class="mw-headline" id="Plasmids"> Plasmids </SPAN></H2><P>The following is a list of all used plasmids. Some of them are just working plasmids, others are of greater value for our project. The latter are bold in the table.
</P><TABLE class="wikitable centered" style="margin: 1em auto 1em auto"><TBODY><TR style="background-color:#99cccc;"><TH height="20px," width="100px"> Plasmid Abbreviation </TH><TH width="200px"> plasmid name </TH><TH width="200px"> insert </TH><TH width="300px"> remarks </TH><TH width="100px"> resistance
</TH></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P1 </TD><TD> pcDNA5/FRT </TD><TD> Ampicillin/Hygromycin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P2 </TD><TD> pEGFP template </TD><TD> Ampicillin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P3 </TD><TD> mcherry template </TD><TD> Kanamycin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P4 </TD><TD> mcherry ΔPstI </TD><TD> Kanamycin (?)
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P5 </TD><TD> pcDNA5/FRT ΔPstI </TD><TD> Ampicillin/Hygromycin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"><B>P6</B></TD><TD><B>pcDNA5/FRT ΔPstI/ΔEcoRI</B></TD><TD><B>negative control for <A href="/Team:Heidelberg/Project_Measurement#Measuring_REU_by_flow_cytometry_and_image_analysis" title="Team:Heidelberg/Project Measurement">measurements</A></B></TD><TD><B>Ampicillin/Hygromycin</B></TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P7 </TD><TD> pcDNA5/FRT ΔPstI/ΔEcoRI *unmethylated* </TD><TD> - </TD><TD> transformed in <A href="/Team:Heidelberg/Notebook_MaM#Bacteria" title="Team:Heidelberg/Notebook MaM">SC110</A></TD><TD> Ampicillin/Hygromycin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P8 </TD><TD> Part:BBa_E0040 </TD><TD> E0040 (GFPmut3B) </TD><TD> does not seem to work in eukaryotes </TD><TD> Ampicillin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P9 </TD><TD> pFRT/<I>lacZeo</I></TD><TD> - </TD><TD> plasmid was stably integrated in cell lines and used for <A href="/Team:Heidelberg/stables" title="Team:Heidelberg/stables">stable integration</A> of constructs </TD><TD> Ampicillin/Zeocin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P10 </TD><TD> pcDNA5/FRT ΔPstI/ΔEcoRI EGFP </TD><TD> EGFP (from pEGFP-N1) </TD><TD> Ampicillin/Hygromycin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P11 </TD><TD> pcDNA5/FRT ΔPstI/ΔEcoRI mcherry JeT </TD><TD> mCherry, Jet promoter </TD><TD> Ampicillin/Hygromycin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P12 </TD><TD> pcDNA5/FRT ΔPstI/ΔEcoRI mcherry Min </TD><TD> mCherry, Min promoter </TD><TD> Ampicillin/Hygromycin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P13 </TD><TD> pcDNA5/FRT ΔPstI/ΔEcoRI GFPb3mut </TD><TD> E0040 (GFPmut3B) </TD><TD> Ampicillin/Hygromycin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"><B>P23</B></TD><TD><B>psB1A3 - Registry</B></TD><TD><B>J04450 (RFP device)</B></TD><TD><B>backbone used for creation of BBb submission plasmid</B></TD><TD><B>Ampicillin</B></TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P27 </TD><TD> pEGFP-N1 </TD><TD> EGFP </TD><TD> preliminary (+) control for transfection </TD><TD> Kanamycin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P29 </TD><TD> pSB1A3 - Registry </TD><TD> P1010 </TD><TD> Ampicillin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P30 </TD><TD> pcDNA5/FRT ΔPstI/ΔEcoRI GFP Min </TD><TD> GFP, Min promoter </TD><TD> Ampicillin/Hygromycin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"><B>P31</B></TD><TD><B>pcDNA5/FRT ΔPstI/ΔEcoRI GFP BBbJeT</B></TD><TD><B>GFP, standardized JeT promoter</B></TD><TD><B>Ampicillin/Hygromycin</B></TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P32 </TD><TD> CFP_emp_cytosole </TD><TD> CFP, cytosole localization signal </TD><TD> template </TD><TD> Kanamycin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P33 </TD><TD> YFP_emp_cytosole </TD><TD> YFP, cytosole localization signal </TD><TD> template </TD><TD> Kanamycin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P34 </TD><TD> YFP_GPI_PM </TD><TD> YFP, PM localization signal </TD><TD> template </TD><TD> Kanamycin/Neomycine
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P35 </TD><TD> CFP_GPI_PM </TD><TD> CFP, PM localization signal </TD><TD> template </TD><TD> Kanamycin/Neomycine
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P36 </TD><TD> YFP_Sar1_ER </TD><TD> YFP, ER localization signal </TD><TD> template </TD><TD> Kanamycin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P37 </TD><TD> CFP_Sar1_ER </TD><TD> CFP, ER localization signal </TD><TD> template </TD><TD> Kanamycin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P38 </TD><TD> EBFP_NLS_nucleus </TD><TD> EBFP, nuclear localization signal </TD><TD> template </TD><TD> Kanamycin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P39 </TD><TD> eYFP_NLS-cas8_nucleus </TD><TD> EYFP, nuclear localization signal </TD><TD> template </TD><TD> Kanamycin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P40 </TD><TD> mCherry_H2B_nucleus </TD><TD> mCherry, nuclear localization signal </TD><TD> template </TD><TD> Kanamycin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P41 </TD><TD> eYFP_H2B_nucleus </TD><TD> EYFP, nuclear localization signal </TD><TD> template </TD><TD> Kanamycin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P42 </TD><TD> GFP_myrpalm_PM </TD><TD> GFP, PM localization signal </TD><TD> template </TD><TD> Kanamycin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P43 </TD><TD> mCherry_myrpalm_PM </TD><TD> mCherry, PM localization signal </TD><TD> template </TD><TD> Kanamycin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P44 </TD><TD> GFP_mitomeet_Mitochondria </TD><TD> GFP, mitochondrial localization signal </TD><TD> template </TD><TD> Kanamycin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P45 </TD><TD> pOG44 </TD><TD> FLP </TD><TD> contains FLP enzymes that is used for <A href="/Team:Heidelberg/stables" title="Team:Heidelberg/stables">stable integration</A> of pcDNA5/FRT derived plasmids at FRT sites in the genome </TD><TD> Ampicillin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P46 </TD><TD> Oakes NFkB </TD><TD> 5x NFκB responsive element </TD><TD> backbone: pGL3 </TD><TD> Ampicillin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P47 </TD><TD> Oakes NFAT </TD><TD> NFAT RE </TD><TD> backbone: pGL4 </TD><TD> Ampicillin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"><B>P48</B></TD><TD><B>p31_cmv_core_Jet_prox</B></TD><TD><B>CMV core promoter, JeT proximal promoter</B></TD><TD><B>standardized (BBb) fused promoter</B></TD><TD><B>Ampicillin/Hygromycin</B></TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"><B>P49</B></TD><TD><B>BBb_JeT pSB1A3</B></TD><TD><B>standardized JeT promoter</B></TD><TD><B>backbone: pSB1A3; our very first plasmid to be send to the registry!</B></TD><TD><B>Ampicillin</B></TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"><B>P51</B></TD><TD><B>pcDNA5/FRT ΔPstI/ΔEcoRI BBbJeT</B></TD><TD><B>standardized JeT promoter</B></TD><TD><B>Ampicillin/Hygromycin</B></TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P52 </TD><TD> hsp70-EGFP </TD><TD> EGFP, hsp70 promoter </TD><TD><A href="/Team:Heidelberg/Project_Natural_promoters#Methods" title="Team:Heidelberg/Project Natural promoters">natural heat sensitive promoter</A></TD><TD> Kanamycin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"><B>P53</B></TD><TD><B>pcDNA/FRT ΔPstI/ΔEcoRI BBbCMV</B></TD><TD> standardized CMV promoter </TD><TD> Ampicillin/Hygromycin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"><B>P55</B></TD><TD><B>pcDNA5/FRT ΔPstI/ΔEcoRI GFP_BBbCMV</B></TD><TD><B>GFP with standardized CMV promoter</B></TD><TD><B>exactly the same plasmid as P31 except for the promoter</B></TD><TD><B>Ampicillin/Hygromycin</B></TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P56 </TD><TD> p31 CYP1A1 </TD><TD> GFP, CYP1A1 promoter </TD><TD><A href="/Team:Heidelberg/Project_Natural_promoters#Methods" title="Team:Heidelberg/Project Natural promoters">natural promoter</A></TD><TD> Ampicillin/Hygromycin
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"><B>P57</B></TD><TD><B>pcDNA/FRT ΔPstI/ΔEcoRI mCherry_BBbJeT</B></TD><TD><B>mCherry with standardized JeT promoter</B></TD><TD><B>same plasmid as P31 except for fluorescent protein; used for cotransfection for <A href="/Team:Heidelberg/Project_Measurement#Measuring_REU_by_flow_cytometry_and_image_analysis" title="Team:Heidelberg/Project Measurement">measurements</A></B></TD><TD><B>Ampicillin/Hygromycin</B></TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> P58 </TD><TD> intron in submission plasmid </TD><TD> Ampicillin
</TD></TR></TBODY></TABLE><P><A href="/Team:Heidelberg/Notebook_MaM#Material" title="Team:Heidelberg/Notebook MaM">[Back]</A></P><H2><SPAN class="mw-headline" id="Instruments"> Instruments </SPAN></H2><P>The following is a list of all used Instruments.
</P><TABLE class="wikitable centered" style="margin: 1em auto 1em auto"><TBODY><TR style="background-color:#99cccc;"><TH height="20px," width="300px">Instrument</TH><TH width="300px"> Name </TH><TH width="300px"> Manufactor
</TH></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Table Centrifuge </TD><TD> Microfuge<B>®</B> 18 Centrifuge </TD><TD> Beckman Coulter<SUP>TM</SUP></TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Table Centrifuge  </TD><TD> MicrofugeR 22<B>®</B> Centrifuge  </TD><TD> Beckman Coulter<SUP>TM</SUP></TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Centrifuge  </TD><TD>  Allegra X-12 </TD><TD> Beckman Coulter<SUP>TM</SUP></TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">  Shaker </TD><TD> VORTEXGENIER 2  </TD><TD> Scientific Industries, SI<SUP>TM</SUP></TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">  Heating plate (magnet stirrer) </TD><TD> MR Hei-Standard  </TD><TD> Heidolph
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">  Heatblock </TD><TD> QBD4  </TD><TD> Grant
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">  Heatblock (shakeing function) </TD><TD> Thermomixer comfort  </TD><TD> Eppendorf
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> PCR-machine  </TD><TD> MyCycler<SUP>TM</SUP> thermo cycler  </TD><TD> BioRad
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">  UV-chamber </TD><TD>  Transluminator </TD><TD> Vilber Lourmat
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Scale (fine)  </TD><TD> Pioneer<SUP>TM</SUP> PA114C  </TD><TD> OHAUS
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">  Scale </TD><TD>  Pioneer<SUP>TM</SUP> PA4101C </TD><TD> OHAUS
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Fridge  </TD><TD> KTP 1750 Premium </TD><TD> Liebherr
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Freezer  </TD><TD> GP 1366 Premium  </TD><TD> Liebherr
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> -86 Freezer  </TD><TD>  -86°C Ultralow Freezer </TD><TD> NUAIRE<SUP>TM</SUP></TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;">  Hood </TD><TD> Tischabzug  </TD><TD> Wesemann<B>®</B> Laboreinrichtung
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Draw-off pump  </TD><TD> Vacuhand control  </TD><TD> Vacubrand
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Incubator  </TD><TD> HT Multitron Version 2  </TD><TD> INFORS
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Incubator (cell culture)  </TD><TD> HERAcell 150  </TD><TD> Thermo electron coorperation
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Hood (cell culture)  </TD><TD> HERAsafe  </TD><TD> Thermo electron coorperation
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Plate reader  </TD><TD> Infinite M200  </TD><TD> TECAN
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> FACS  </TD><TD> Cytomics FC 500 MPL  </TD><TD> Beckman Coulter<SUP>TM</SUP></TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Microscope  </TD><TD> CKX41  </TD><TD> Olympus
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Computer  </TD><TD> Sun Microsystems
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> UV/Visivle Spectrometer  </TD><TD> Ultrospec 3300 pro  </TD><TD> Amersham Biosciences
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Photometer  </TD><TD> NanoDrop<B>®</B> ND-1000 Spectrophotometer  </TD><TD> peQLab Biotechnologie GmbH
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Photometer  </TD><TD> NanoVue<SUP>TM</SUP></TD><TD> General Electric
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Ice Machine  </TD><TD> MF22  </TD><TD> SCOTSMAN<B>®</B></TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Gel electrophoresis chamber  </TD><TD> Mupid<B>®</B>-One  </TD><TD> Advance
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> pipetting robot  </TD><TD> QIAcube<SUP>TM</SUP></TD><TD> QIAGEN
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Wide field microscope  </TD><TD> Nikon Eclipse 90i  </TD><TD> Nikon
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> PCR machine  </TD><TD> MyCycler thermal cycler  </TD><TD> BIO-RAD
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Real-time PCR machine  </TD><TD> StepOnePlus<SUP>TM</SUP> Real-time PCR System  </TD><TD> Applied Biosystems<SUP>TM</SUP></TD></TR></TBODY></TABLE><P><A href="/Team:Heidelberg/Notebook_MaM#Material" title="Team:Heidelberg/Notebook MaM">[Back]</A></P><H2><SPAN class="mw-headline" id="Lab_material"> Lab material </SPAN></H2><P>The following is a list of the used Lab material.
</P><TABLE class="wikitable centered" style="margin: 1em auto 1em auto"><TBODY><TR style="background-color:#99cccc;"><TH height="20px," width="300px">Material</TH><TH width="300px"> Name </TH><TH width="300px"> Manufactor
</TH></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Bottle 50 ml, 100 ml, 200 ml, 500 ml, 1000 ml  </TD><TD> Schott Duran  </TD><TD> Schott
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Beaker 50 ml, 100 ml, 200 ml, 500 ml 1000 ml </TD><TD> - </TD><TD> Simax
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Conical flask 500 ml, 1000 ml, 100 ml </TD><TD> FB33141, FB33142, FB33139 </TD><TD> Fischerbrand
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Conical flask 300 ml </TD><TD> - </TD><TD> Simax
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Single channel pipet </TD><TD> Pipetman<B>®</B> P2, P20, P200, P1000 </TD><TD> Gilson
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Multichannel pipet </TD><TD> Research<B>®</B> Multikanal </TD><TD> Eppendorf
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Multistep pipet </TD><TD> e10, e120, e1000, e5000 </TD><TD> Biohit
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Pipet </TD><TD> accu-jet<B>®</B> pro </TD><TD> Brand
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> PCR-tubes </TD><TD> PCR softtubes </TD><TD> Biozym
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Microcentrifuge tubes </TD><TD> safe-lock-tubes 2 ml/1,5 ml </TD><TD> Eppendorf
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Centrifugetubes </TD><TD> BD Falcon<SUP>TM</SUP> 15 ml /50 ml </TD><TD> BD Biosciences
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> PCR 8-strips </TD><TD> Tube Strips, CLR </TD><TD> Biorad
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Serological Pipet </TD><TD> 5 ml, 10 ml, 25 ml </TD><TD> BD Labware
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Gloves </TD><TD> Latex Medical Examination Gloves (S,M,L) </TD><TD> Blossom
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Nitrile Gloves </TD><TD> Freeform<SUP>TM</SUP> SE Examination Gloves (S,M,L) </TD><TD> MICROFLEX<B>®</B></TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Filtertips </TD><TD>  Extended length Filtertips</TD><TD> Starlab
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Pipettips  </TD><TD> TipOne<B>®</B></TD><TD> Starlab
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Counting Chamber </TD><TD> Neubauer </TD><TD> Brand
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Inoculating loop </TD><TD> - </TD><TD> nunc<SUP>TM</SUP></TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Disposal bags </TD><TD> - </TD><TD> Plastibrand<B>®</B></TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Flasks </TD><TD> Tissue Culture Flasks </TD><TD> TPP<B>®</B></TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Plate </TD><TD> Microplates for life science Research </TD><TD> PerkinElmer<SUP>TM</SUP></TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> IBD </TD><TD> Lab-Tek<B>®</B></TD><TD> nunc<SUP>TM</SUP></TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> 6/16-Well plates </TD><TD> Nunclon<SUP>TM</SUP> Surface </TD><TD> nunc<SUP>TM</SUP></TD></TR></TBODY></TABLE><P><A href="/Team:Heidelberg/Notebook_MaM#Material" title="Team:Heidelberg/Notebook MaM">[Back]</A></P><H1><SPAN class="mw-headline" id="Methods"> Methods </SPAN></H1><H2><SPAN class="mw-headline" id="General_Methods">General Methods</SPAN></H2><H3><SPAN class="mw-headline" id="Transformation_of_Bacteria"> Transformation of Bacteria </SPAN></H3><P>For enrichment of vectors, <I>E. coli</I> DH5α were used. For the transformation 100 µl of the competent cells were thawed on ice and 50 – 400 ng DNA solution added (depending on the concentration of the DNA solution). After a 30-60 minute incubation on ice, cells were made permeable for the DNA by heat shocking for 45 seconds at 42 °C and a further 3 minute incubation on ice. The samples were than rescued by adding 500 µl preheated antibiotic free LB-medium and incubated for one hour at 37 °C while shaking for induction of the antibiotic resistance. The selection for plasmid containing and therefore antibiotic resistant bacteria was conducted by plating them on antibiotic containing LB-agar plates.
</P><H3><SPAN class="mw-headline" id="Glycerol_stock"> Glycerol stock </SPAN></H3><P>To store bacteria for long term glycerol stocks were used. Therefore 1 ml of an over night culture were added to 150 µl of 80 % Glycerol into a cryo tube, vortexed and incubated at room temperature for 30 min. Afterwards the glycerole stock was stored at -80 °C.
</P><H3><SPAN class="mw-headline" id="Isolation_of_plasmid_DNA_by_alkaline_lysis_.28mini_and_maxiprep.29"> Isolation of plasmid DNA by alkaline lysis (mini and maxiprep) </SPAN></H3><P>For analysis of ligations and transformations QIAprep Spin Kits (Qiagen, Hilden) were used following the manufacturer instructions. For miniprep a single colony was picked from a LB-agar plate or glycerol stock was used to inoculate 5 ml LB-medium with appropriate antibiotic for selection (100 µg/µl ampicillin, 50 µg/µl kanamycin, 35 µg/µl chloramphenicol). Bacteria were grown over night at 37 °C while shaking (200 rpm). By using 4 ml over night culture with this kit the yield was around 6-10 µg. For maxipreps the Qiagen CompactPrep Plasmid Maxi Kit was used following the instructions given by the instruction manual. In this case 250 ml LB-medium were inoculated and used for preparation of plasmid DNA. The routinely yield was 300-400 µg plasmid DNA. Purity and amount of DNA was analysed using a NanoDrop.
</P><H3><SPAN class="mw-headline" id="Preparing_chemically_competent_cells"> Preparing chemically competent cells </SPAN></H3><P>First, a 20 ml over night culture was inoculated in antibiotic free LB medium from a fresh single colony and transferred into 400 ml antibiotic free LB medium the next day. This culture was incubated at 37 °C while shacking until an OD600 of 0.5 – 0.6 was achieved. The culture was than cooled down on ice, centrifuged (8 min, 4 °C, 3500 rpm), the supernatant discarded and the pellet resuspended in 10 ml 100 mM CaCl<SUB>2</SUB>. After addition of further 190 ml 100 mM CaCl<SUB>2</SUB> the suspension was incubated on ice for 30 min. The suspension was than again centrifuged (8 min, 4 °C, 3500 rpm), the supernatant discarded, the pellet resuspended in 20 ml 82.5 mM CaCl<SUB>2</SUB> with 17.5 % glycerol and aliquoted. The aliquots were flash frozen in liquid nitrogen and than stored at -80 °C until usage.
</P><H3><SPAN class="mw-headline" id="Ligation"> Ligation </SPAN></H3><P><B>Standard Ligation Protocol</B></P><P>T4 ligase joins the 5' phosphate and the 3'-hydroxyl groups of DOUBLE stranded DNA molecules. First step for ligation is to estimate the vector and insert concentrations:if insert is from PCR, assume that 50% is recovered, typically 5 mg, and resuspended in 10 ml of H<SUB>2</SUB>0; same with vector, assume half is recovered in purification/precipitation and resuspend in 10-20 ml of H<SUB>2</SUB>0 if it is not re-suspended already. Second step is to plan control reactions: One reaction with no insert, one reaction with no vector (if enough insert is available). Then Molar Ratio of Insert to Vector is determined where a ratio of Insert:Vector 3:1 (100-150ng Vector DNA) is tried to be achieved. After this,  ligation mixure is setup in the following way:
</P><UL><LI> 10X Ligase buffer (Promega #M1801)
</LI><LI> 1 ml T4 Ligase (Promega #M1801, 3 U/ml)
</LI><LI> Bring volume to 10 (or 20 see note below) ml with nuclease-free water.
</LI><LI> If  doing a blunt-end ligation, it may be needed to add PEG (up to 15%) to increase the efficiency. 
</LI></UL><P>Samples are incubated according to the guidelines below
</P><UL><LI> Incubate sticky end ligation reactions at room temperature for 3 hours (or at 4 to 8 °C, overnight).
</LI><LI> Incubate blunt-end ligation reactions at 17 °C for 4 to 18 hours. 
</LI></UL><P>After incubation  the ligase is heat inactivated by placing tube in 65 °C water bath for 10 minutes. 2 ml of the ligation mixture is used for transformation.
Transformed cells are plated on selective media and incubated overnight.
</P><H3><SPAN class="mw-headline" id="Site-directed_mutagenesis"> Site-directed mutagenesis </SPAN></H3><P>For removal of unwanted Restriction Sites, a PCR-based site direct mutagenesis protocol was adapted from <A href="http://www.stratagene.com/manuals/200518.pdf" class="external text" rel="nofollow">Stratgene</A>. Oligos were designed to have a high (&gt;78°C) T<SUB>m</SUB> by applying the formula

The following scheme was used for pipetting:
</P><TABLE class="wikitable centered" style="margin: 1em auto 1em auto"><TBODY><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> 1 µl   template (10 ng)
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> 1 µl   primer 1 (25µM)
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> 1 µl   primer 2 (25µM)
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> 25 µl Mastermix 2x (Phusion)
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> 22 µl   dH2O
</TD></TR></TBODY></TABLE><P>The PCR procedure was as follows:
</P><UL><LI> Initial denaturation, 1 cycle
</LI><LI> 95 °C, 30 sec
</LI></UL><LI> Amplification, 16 cycles
</LI><LI> denaturation 95 °C, 50 sec
</LI><LI>annealing, 60 °C, 50 sec
</LI><LI>extension, 72 °C, 40sec/kb + 30sec 
</LI><LI> Final Extension,1 cycle 
</LI><LI> 72 °C, 5 – 10 min 
</LI><LI> 4 °C hold, forever
</LI><P>After completion, 1 µL of DpnI (NEB) was added and the mix was incubated for 1h at 37°C. Then, DH5alpha cells were transformed as described above.
</P><H3><SPAN class="mw-headline" id="DNA_synthesis"> DNA synthesis </SPAN></H3><P>Oligos were designed using <A href="http://baderlab.bme.jhu.edu/gd/" class="external text" rel="nofollow">GeneDesign</A>. 15Bp were chosen as an overhang and 56 °C as an annealing temperature. Oligos were ordered at a concentration of 100 µM. 1:10 dilutions of the first and last oligo were prepared. Then, all oligos (including first and last) were pooled at 1 µl each. Water was added to 10x the original volume. (So if a gene is synthesized from x = 14 oligos, water was added to 10*x = 140 µl). This pool was then diluted 1:10. x µl of the dilution were put into a PCR reaction with 25 µl Phusion master mix and 25-x µl water. PCR was conducted as following:
</P><UL><LI> Initial denaturation, 1 cycle
</LI><LI> 95 °C, 5 min
</LI></UL><LI> Amplification, 7 cycles
</LI><LI> denaturation 95 °C, 30 sec
</LI><LI>annealing, 58 °C, 30 sec
</LI><LI>extension, 72 °C, 1 minute 
</LI><LI> 4 °C hold, forever
</LI><P>Afterwards, 1 µl of the 1:10 dilutions of the innermost and outermost primer were added. The same PCR protocol was then repeated, but with 25 instead of 7 cycles. PCR products were run on a 2% agarose gel and gel purified.
</P><H3><SPAN class="mw-headline" id="Purification_of_DNA_from_PCR_reactions">Purification of DNA from PCR reactions</SPAN></H3><P>PCR products were purified by the QIAquick PCR Purification Kit from Qiagen following the instructions of the Qiagen Handbook. To check the purity and amount of extracted DNA an aliquot was analysed using a NanoDrop.
</P><H3><SPAN class="mw-headline" id="Enzymatic_hydrolysis_of_DNA_by_restriction_enzymes">Enzymatic hydrolysis of DNA by restriction enzymes</SPAN></H3><P><B>Restriction digest</B>
Restriction digest of DNA was used for analysis of purified DNA form mini or maxiprep or for isolation of specific DNA fragments for further cloning. Analytical digestions were routinely conducted in 20 µl volume,preparative digestions were routinely conducted in 50 µl volume. In all digestions a minimum of 2 Units restriction enzyme(s) was used per microgram DNA. Optimised buffer conditions were secured by using NEB buffer system. The final reaction volume was achieved by adding H<SUB>2</SUB>O<SUB>dest</SUB>. The sample was incubated at optimal temperature for the restriction enzyme(s) (usually 37 °C). Analysis and preparation of DNA were either done by gelelectrophoreses (loading dye was added to the samples and they were loaded on a agarose gel) or alternatively PCR Purification (PCR Purification QIAquick  Kit from Qiagen)
</P><P><B>Phosphatasing with Shrimp Alkaline Phosphatase</B>
SAP is used for Dephosphorylation of 5'-ends of DNA in Restriction Enzyme Reaction. After restriction digest  1 unit of SAP is added for every 1 pmol of DNA ends (about 1 μg of a 3 kb plasmid) and incubated at 37°C for 30-60 min. The reaction is stopped by heating at 65°C for 15 min (this completely inactivates SAP)
The minimum effective amount of SAP for dephosphorylation of 1 pmol of DNA termini in 1 hr at 37°C is:
</P><UL><LI> 0.05 units for 5'-protruding termini
</LI><LI> 0.05 units for blunt termini
</LI><LI> 0.1 units for 5'-recessed termini
</LI></UL><H3><SPAN class="mw-headline" id="Agarose_gel_electrophoresis_for_separating_DNA">Agarose gel electrophoresis for separating DNA</SPAN></H3><P>In the agarose gel electrophoresis a mixture of DNA fragments with different sizes are separated in an electrical field by their size. This is achieved by moving the negatively charged DNA through an agarose matrix while shorter fragments will run faster. The size of the pores can be controlled by agarose concentration. The higher the agarose concentration the smaller the pores are and the smaller fragments can be separated. Agarose concentrations between 0.7 and 1.5 % agarose in 0.5x TE buffer were used. 
The agarose was dissolved completely by heating up and 0.1 µg/ml ethidium bromide was added. 
The DNA fragments were separated using a constant voltage between 80 and 130 V. Under UV light (λ = 254 nm) DNA is visible through the unspecific intercalated ethidium bromide and can be documented or cut out and extracted from the gel.
</P><H3><SPAN class="mw-headline" id="Isolation_of_DNA_fragments_from_an_agarose_gel">Isolation of DNA fragments from an agarose gel</SPAN></H3><P>Plasmid DNA and DNA fragments were extracted using the Gel Extraction Kit from Qiagen following the manufacture instructions. To check the purity and amount of extracted DNA an aliquot was analysed using a NanoDrop.
</P><P><A href="/Team:Heidelberg/Notebook_MaM#Methods" title="Team:Heidelberg/Notebook MaM">[Back]</A></P><H2><SPAN class="mw-headline" id="Cell_Culture">Cell Culture</SPAN></H2><H3><SPAN class="mw-headline" id="Splitting_cells"> Splitting cells</SPAN></H3><P>Cultures of HeLa, MCF7 and U2OS cells are splitted 1:10 every 3-4 days or 1:3 every day: DMEM+++ (see Material) covering the cells is removed and the cells are washed with 3 ml PBS. The PBS is removed after the washing step and 3 ml of trypsin are added to the flask; almost all of the Trypsin is removed again until the cells are only slightly covered with Trypsin. The trypsinized cells are then incubated for 5 min at 37 °C and 5% CO<SUB>2</SUB>. After this incubation time,  the cells are detached from the bottom and the flask is filled with 5 ml of DMEM+++. Cells are resuspended in medium  by pipetting up and down. To achieve a diluton of 1:10, 0,5 ml of the suspension is transferred to another flask and 5 ml of DMEM+++ are added. This dilution is then incubated for 3-4 days at 37°C and 5% CO<SUB>2</SUB>. The rest of the cell suspension (about 4 ml) is transferred to a 50 ml falcon tube and the same volume of DMEM+++ is added to the falcon. 9,6 µl of this dilution are transferred to a counting chamber and the number of cells is determined (x*10<SUP>4</SUP>).
</P><UL><LI> Note 1: HeLa and U2-OS seem to grow faster than MCF-7; check notebook for information on splitting ratios
</LI></UL><H3><SPAN class="mw-headline" id="Transfection_of_Mammalian_cells"> Transfection of Mammalian cells </SPAN></H3><P><B>1.Fugene (Roche)</B> 10<SUP>5</SUP> cells were moved to 6-well plates. 2,5 ml of DMEM medium plus addiditives were added. The following day, 97 µl pure DMEM was incubated for 5 minutes with 3 µl Fugene6 reagent at room temperature. Afterwards, the mixture was added dropwise to 500 ng of plasmid and allowed to incubate for 15 minutes at RT. The mixture was then added dropwise to the cells. Cells were incubated for 24 hours without selection pressure, afterwards, medium was changed and Zeocin was added to a concentration of 350 ng/ml.
</P><P><B>2. Effectene Transfection (Qiagen)</B> Move x (for apropriate amounts see table below) cells to 6-well plates/ LabTek 8 chamber and add 2 ml/0,5 ml DMEM+++. Incubate over night at 37 °C and 5 % CO<SUB>2</SUB>.The following day, dilute x µg of DNA in Buffer EC to a total volume of x µl (note: Yara just adds 100 µl/50 µl). Add x µl of Enhancer and mix by tapping the tube (or vortex for 1 sec). Incubate for 2-5 min at RT. Add x µl of Effectene Transfection reagent to the DNA-Enhancer Mixture. Mix by tapping the tube (or vortexing for 10 s or pipetting up and down 5 times). Incubate samples for 5- 10 min at RT to allow for transfection-complex formation. While complex formation takes place, aspire medium and add x ml of fresh growth medium. After incubation time add the transfection
-complexes drop-wise to the cells on the well. Gently swirl the plate to ensure uniform distribution of the complexes. 
</P><UL><LI> transient transfection: remove Effectene-DNA complexes after 6-18 hours, wash cells 1x with sterile PBS, add fresh medium. Incubate cells with the complexes under normal growth conditions for an appropriate time for expression.
</LI><LI> stable transfection: change medium after 24 hours (wash with PBS); check confluency after 48 hours - (if required) split to 25 % confluency, add fresh medium and incubate 2-3 hours until cells have attached to culture dish. Then, remove medium and add fresh DMEM+++/Zeocin.
</LI></UL><TABLE class="wikitable centered" style="margin: 1em auto 1em auto"><TBODY><TR style="background-color:#99cccc;"><TH>  Culture format
</TH><TH>  Number of cells
</TH><TH>  DNA [µg]
</TH><TH>  Enhancer [µl]
</TH><TH>  Final Volume of DNA in Buffer EC [µl]
</TH><TH>  Effectene reagent [µl]
</TH><TH>  Volume of Medium to add to cells [µl]
</TH></TR><TR style="background-color:#9BDDFF;"><TD>  96-well
</TD><TD>  10<SUP>4</SUP></TD><TD>  0,1
</TD><TD>  0,8
</TD><TD>  30
</TD><TD>  2,5
</TD><TD>  100
</TD></TR><TR style="background-color:#9BDDFF;"><TD>  Ibidi
</TD><TD>  1,5*10<SUP>4</SUP></TD><TD>  0,15
</TD><TD>  1,2
</TD><TD>  50
</TD><TD>  4
</TD><TD>  200-250
</TD></TR><TR style="background-color:#9BDDFF;"><TD>  24-well
</TD><TD>  4*10<SUP>4</SUP></TD><TD>  0,2
</TD><TD>  1,6
</TD><TD>  60
</TD><TD>  5,0
</TD><TD>  700
</TD></TR><TR style="background-color:#9BDDFF;"><TD>  6-well
</TD><TD>  10<SUP>5</SUP></TD><TD>  0,4
</TD><TD>  3,2
</TD><TD>  100
</TD><TD>  10
</TD><TD>  2000
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Freeze_cells"> Freeze cells </SPAN></H3><P>Cells to be frozen should be confluent, but not too dense. Medium covering the cells is removed  and cells are washed once with PBS. Afterwards cells are rinsed with  3 ml Trypsin; Trypsin is removed until the cells are only slightly  covered (about 1 ml) and cells are incubated for about 5 min. After incubation, cells are resuspended in medium (at least about 4-5 times as much as Trypsin to inactivate that;  used 5 ml). The suspension is transferred to 15 ml falcon and centrifuged at 2000 rpm for 3-5 minutes. The supernatant is discarded and the remaining pellet is resuspended in DMEM+++ (here: 7,5 ml for 5 aliquots, 1,5 ml each). 1,5 ml of the suspension is distributed into freezer vials. After adding of 150 µl DMSO to each vial, cells are put on ice and stored at -80 °C.
</P><H3><SPAN class="mw-headline" id="Picking_cells_with_cloning_disks"> Picking cells with cloning disks </SPAN></H3><P>Foci of cells, which were under selection pressure (zeocin) are marked on the bottom of the petri dish (use pen from underneath the microscope). As many cloning disks as foci available are prepared by incubating them 3-5 minutes in trypsin (in a petri dish). Medium covering the cells is removed and cells are washed with PBS. PBS is removed till the cells are only slightly covered with it. The cloning disks are placed on the previousely marked spots and incubated for 3 minutes at 37 °C. During incubation a 96-well plate is prepared with 0,7 ml DMEM+++/Zeocin per well. The clonig disks are picked up with pressure (or swipe a little) to remove the entire foci off the plate and  the cloning disk is transferred to the 96-well plate (one disk per well). Medium is pipetted up and down to loosen the cells from the cloning disks.
</P><H3><SPAN class="mw-headline" id="Fix_cells_for_microscopy"> Fix cells for microscopy </SPAN></H3><P>The medium is removed from the cells on cover slips. 500 µl of 4% Paraformaldehyde (in PBS) is added to the cells and incubated for 10 minutes. PFA is taken off and cells are washed with 500 µl of PBS. Slides are labeled and  a small drop of fluoromount is added with a pipet tip. Cover slips are washed in dH<SUB>2</SUB>0, by picking them up with tweezers and dipping them into the water. Afterwards cover slips are dried with filter paper without the paper touching the cover slip side containing the cells. Cover slips are layed down on the drop of fluoromount with the cells facing downwards. Fixed cells are dried overnight and can then be used for fluorescence microscopy.
</P><H3><SPAN class="mw-headline" id="Induction"> Induction </SPAN></H3><P>To induce the respective pathways specifically, we searched literature and used knowledge of groups working with the drugs. The following table is an overview of the pathways, the used drugs and the conditions for activation.
</P><TABLE cellpadding="5" class="wikitable"><TBODY><TR style="background-color:#99cccc;"><TH height="20px," width="70px"> Pathway </TH><TH width="70px"> mode </TH><TH width="80px"> cell type </TH><TH width="80px"> drug </TH><TH width="80px"> stock concentration </TH><TH width="100px"> dilution </TH><TH width="80px"> final concentration </TH><TH width="80px"> medium </TH><TH width="80px"> time to leave on cells </TH><TH width="180px"> remarks </TH></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> HIF <A href="/Team:Heidelberg/Notebook_MaM#References" title="Team:Heidelberg/Notebook MaM">[1]</A></TD><TD> activator </TD><TD> MCF-7 </TD><TD><A href="http://www.sigmaaldrich.com/catalog/ProductDetail.do?D7=0&amp;N5=Product%20No.%7CBRAND_KEY&amp;N4=D9533%7CSIGMA&amp;N25=0&amp;QS=ON&amp;F=SPEC" class="external text" rel="nofollow">DFO</A></TD><TD> 50 mM </TD><TD> 1:500 </TD><TD> 100 μM </TD><TD> minimal medium </TD><TD> 24 h </TD><TD> together with HIF-bag </TD></TR><TR style="background-color:#9BDDFF;"><TD> inhibitor </TD><TD> MCF-7 </TD><TD><A href="https://www.sigmaaldrich.com/catalog/ProductDetail.do?D7=0&amp;N5=SEARCH_CONCAT_PNO%7CBRAND_KEY&amp;N4=07741%7CFLUKA&amp;N25=0&amp;QS=ON&amp;F=SPEC" class="external text" rel="nofollow">Everolimus</A></TD><TD> 1:500 000 </TD><TD> minimal medium </TD><TD> 24 h </TD><TD> without HIF-bag </TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> SREBP <A href="/Team:Heidelberg/Notebook_MaM#References" title="Team:Heidelberg/Notebook MaM">[2]</A>, <A href="/Team:Heidelberg/Notebook_MaM#References" title="Team:Heidelberg/Notebook MaM">[8]</A><A href="/Team:Heidelberg/Notebook_MaM#References" title="Team:Heidelberg/Notebook MaM">[9]</A></TD><TD> activator </TD><TD> HeLa </TD><TD> Lipid-depleted-Serum (LDS) </TD><TD> - </TD><TD> 0,5% </TD><TD> - </TD><TD> DMEM++ </TD><TD> 2-3 days </TD><TD> put on cells after transfection and put back on after 3 h with HPCD </TD></TR><TR style="background-color:#9BDDFF;"><TD> activator </TD><TD> HeLa </TD><TD><A href="http://www.sigmaaldrich.com/catalog/ProductDetail.do?D7=0&amp;N5=SEARCH_CONCAT_PNO%7CBRAND_KEY&amp;N4=H107%7CSIGMA&amp;N25=0&amp;QS=ON&amp;F=SPEC" class="external text" rel="nofollow">HPCD</A></TD><TD> 500 mg/ml </TD><TD> 1:100 </TD><TD> 500 μg/ml </TD><TD> DMEM++ with LDS</TD><TD> 3 h </TD><TD> add after cells were 2-3 days in DMEM++ +LDS </TD></TR><TR style="background-color:#9BDDFF;"><TD> inhibitor </TD><TD> HeLa </TD><TD><A href="http://www.sigmaaldrich.com/catalog/ProductDetail.do?D7=0&amp;N5=SEARCH_CONCAT_PNO%7CBRAND_KEY&amp;N4=C8667%7CSIGMA&amp;N25=0&amp;QS=ON&amp;F=SPEC" class="external text" rel="nofollow">Cholesterol</A></TD><TD> 10 mg/ml </TD><TD> 1:1000 </TD><TD> 10 μg/ml </TD><TD> DMEM+++ </TD><TD> 2-3 days </TD></TR><TR style="background-color:#9BDDFF;"><TD> inhibitor </TD><TD> HeLa </TD><TD><A href="http://www.sigmaaldrich.com/catalog/ProductDetail.do?D7=0&amp;N5=SEARCH_CONCAT_PNO%7CBRAND_KEY&amp;N4=L7914%7CSIGMA&amp;N25=0&amp;QS=ON&amp;F=SPEC" class="external text" rel="nofollow">LDL</A></TD><TD> 0,5 mg/ml </TD><TD> 1:50 </TD><TD> 10 μg/ml </TD><TD> DMEM+++ </TD><TD> 2-3 days </TD><TD> got LDL from Jessica Schilde -&gt; self-made in Runz' lab </TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> NFκB<A href="/Team:Heidelberg/Notebook_MaM#References" title="Team:Heidelberg/Notebook MaM">[1]</A><A href="/Team:Heidelberg/Notebook_MaM#References" title="Team:Heidelberg/Notebook MaM">[10]</A></TD><TD> activator </TD><TD> U2-OS </TD><TD><A href="http://www.sigmaaldrich.com/catalog/ProductDetail.do?N4=T0157%7CSIGMA&amp;N5=SEARCH_CONCAT_PNO%7CBRAND_KEY&amp;F=SPEC&amp;lang=null%3E" class="external text" rel="nofollow">TNFα</A></TD><TD> 2,5 mM </TD><TD> 1:1000 </TD><TD> 2,5 μM </TD><TD> DMEM++ </TD><TD> 10 h </TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> p53 <A href="/Team:Heidelberg/Notebook_MaM#References" title="Team:Heidelberg/Notebook MaM">[1]</A>,<A href="/Team:Heidelberg/Notebook_MaM#References" title="Team:Heidelberg/Notebook MaM">[5]</A></TD><TD> activator </TD><TD> MCF-7 </TD><TD><A href="http://www.sigmaaldrich.com/catalog/ProductDetail.do?D7=0&amp;N5=SEARCH_CONCAT_PNO%7CBRAND_KEY&amp;N4=C9911%7CSIGMA&amp;N25=0&amp;QS=ON&amp;F=SPEC" class="external text" rel="nofollow">Camptothecin</A></TD><TD> 2 mM </TD><TD> 1:1000 </TD><TD> 2 μM </TD><TD> minimal medium </TD><TD> 12 h </TD></TR><TR style="background-color:#9BDDFF;"><TD> inhibitor </TD><TD> MCF-7 </TD><TD><A href="http://www.sigmaaldrich.com/catalog/ProductDetail.do?D7=0&amp;N5=SEARCH_CONCAT_PNO%7CBRAND_KEY&amp;N4=P4359%7CSIGMA&amp;N25=0&amp;QS=ON&amp;F=SPEC" class="external text" rel="nofollow">Pifithrin-α</A></TD><TD> 2 mM </TD><TD> 1:1000 </TD><TD> 2 μM </TD><TD> minimal medium </TD><TD> 12 h </TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> AHR <A href="/Team:Heidelberg/Notebook_MaM#References" title="Team:Heidelberg/Notebook MaM">[6]</A></TD><TD> activator </TD><TD> HeLa </TD><TD><A href="http://www.sigmaaldrich.com/catalog/ProductDetail.do?N4=B1760%7CSIGMA&amp;N5=Product%20No.%7CBRAND_KEY&amp;F=SPEC" class="external text" rel="nofollow">Benzo-a-pyrene</A></TD><TD> 100 mM </TD><TD> 1:10 000 </TD><TD> 10 μM </TD><TD> DMEM+++ </TD><TD> 18 h </TD></TR><TR style="background-color:#9BDDFF;"><TD> activator </TD><TD> MCF-7 </TD><TD><A href="http://www.sigmaaldrich.com/catalog/ProductDetail.do?lang=en&amp;N4=48599%7CSUPELCO&amp;N5=SEARCH_CONCAT_PNO%6CBRAND_KEY&amp;F=SPEC" class="external text" rel="nofollow">TCDD</A></TD><TD> 1:3000 </TD><TD> DMEM++ </TD><TD> 48 h </TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> PPARγ<A href="/Team:Heidelberg/Notebook_MaM#References" title="Team:Heidelberg/Notebook MaM">[3]</A>,<A href="/Team:Heidelberg/Notebook_MaM#References" title="Team:Heidelberg/Notebook MaM">[4]</A></TD><TD> activator </TD><TD> U2-OS </TD><TD><A href="http://www.sigmaaldrich.com/catalog/ProductDetail.do?N4=88405X%7CFLUKA&amp;N5=Product%20No.%7CBRAND_KEY&amp;F=SPEC" class="external text" rel="nofollow">2,4-Thiazolidinedione</A></TD><TD> 500 mM </TD><TD> 1:100 000 </TD><TD> 5 μM </TD><TD> DMEM+++ </TD><TD>60 h </TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> Estrogen receptor </TD><TD> activator </TD><TD> MCF-7 </TD><TD><A href="http://www.sigmaaldrich.com/catalog/ProductDetail.do?N4=E8875%7CSIGMA&amp;N5=SEARCH_CONCAT_PNO%7CBRAND_KEY&amp;F=SPEC" class="external text" rel="nofollow">β-Estradiol</A></TD><TD> 100 mM </TD><TD> 1:10 000 000 </TD><TD> 10 μM </TD><TD> DMEM++ </TD><TD> 24 h </TD><TD> keep cells with DMEM++ for 24 h after transfections, then add drug </TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> VDR <A href="/Team:Heidelberg/Notebook_MaM#References" title="Team:Heidelberg/Notebook MaM">[7]</A></TD><TD> activator </TD><TD> MCF-7 </TD><TD><A href="http://www.sigmaaldrich.com/catalog/ProductDetail.do?D7=0&amp;N5=SEARCH_CONCAT_PNO%7CBRAND_KEY&amp;N4=D1530%7CSIGMA&amp;N25=0&amp;QS=ON&amp;F=SPEC" class="external text" rel="nofollow">Vitamine D3</A></TD><TD> 100 μM </TD><TD> 1:1000/ 1:10000 </TD><TD> 100 nM/ 1 nM </TD><TD> DMEM+++ </TD><TD> 24 h, 48 h, 72 h, 96 h </TD><TD> TECAN read every 24 hours -&gt; see induction: FACS </TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> c-Jun <A href="/Team:Heidelberg/Notebook_MaM#References" title="Team:Heidelberg/Notebook MaM">[1]</A></TD><TD> activator </TD><TD> HeLa </TD><TD><A href="https://www.sigmaaldrich.com/catalog/ProductDetail.do?N4=E4269%7CSIGMA&amp;N5=SEARCH_CONCAT_PNO%7CBRAND_KEY&amp;F=SPEC" class="external text" rel="nofollow">EGF</A></TD><TD> 100μg/mL </TD><TD> 1:1000 </TD><TD> 100ng/mL </TD><TD> DMEM+++ </TD><TD> 3h </TD><TD> after 3 h, EGF was removed and FACS measurement was carried out another 4 hours later </TD></TR></TBODY></TABLE><P><A href="/Team:Heidelberg/Notebook_MaM#Methods" title="Team:Heidelberg/Notebook MaM">[Back]</A></P><H2><SPAN class="mw-headline" id="RA-PCR"> RA-PCR </SPAN></H2><P>For a RA-PCR protocol, please refer to <A href="/Team:Heidelberg/Project_Synthetic_promoters#RA-PCR_protocol" title="Team:Heidelberg/Project Synthetic promoters">Project page</A></P><P>We used the following Oligo concentrations:
</P><TABLE class="wikitable centered" style="margin: 1em auto 1em auto"><TBODY><TR><TH> p53
</TH><TH> NFκB II
</TH><TH> HIF
</TH><TH> Activator Mix
</TH></TR><TR style="background-color:#9BDDFF;"><TD width="150px">6µL  p53 (O.91)<P>5µL p53 reverse (O.188)
1µL random (O.56)
1µL Activator Mix
0.8µL Stop 5 new (O.187)
0.8µL Stop 3 (O.58)</P></TD><TD width="150px">3µL NFkB-1 (O.93)<P>3µL NFκB-2 (O.94)
4µL NFκB-rev (O.194)
3µL Random (O.56)
2µL Activator Mix
0.8µL Stop 5 new (O.187)
0.8µL Stop 3 (O.58)</P></TD><TD width="150px">2,5µL  HIF-1 (O.53)<P>2,5µL HIF-2 (O.54)
1µL CREB(O.89)
3µL HIF-rev (O.189)
3µL Random (O.56)
1µL Stop 5 new (O.187)
1µL Stop 3 (O.58)
.2µL each Ap1, Sp1 (O.55, O.57)
</P></TD><TD width="150px">2µL each Ap1, Sp1, CREB (O.55, O.57, O.89)<P>1µL each NFY, Empty (O.90, O.95)
Water to 30µL
</P></TD></TR><TR><TH> SREBP
</TH><TH> AHR
</TH><TH> pPARγ</TH><TH> Estrogen receptor
</TH></TR><TR style="background-color:#9BDDFF;"><TD width="150px">5µL  SREBP (O.208)<P>4µL SREBP reverse (O.209)
1µL Sp1 (O.57)
2µL random (O.56)
1µL Activator Mix
0.8µL Stop 5 new (O.187)
0.8µL Stop 3 (O.58)</P></TD><TD width="150px">5µL  AHR (O.212)<P>4µL AHR reverse (O.213)
2µL random (O.56)
1,5µL Activator Mix
0.8µL Stop 5 new (O.187)
0.8µL Stop 3 (O.58)</P></TD><TD width="150px">5µL  pPARγ (O.210)<P>4µL pPARγ reverse (O.211)
2µL random (O.56)
1,5µL Activator Mix
0.8µL Stop 5 new (O.187)
0.8µL Stop 3 (O.58)</P></TD><TD width="150px">5µL Estrogen receptor (O.210)<P>4µL Estrogen receptor reverse (O.211)
2µL random (O.56)
1,5µL Activator Mix
0.8µL Stop 5 new (O.187)
0.8µL Stop 3 (O.58)</P></TD></TR></TBODY></TABLE><P><A href="/Team:Heidelberg/Notebook_MaM#Methods" title="Team:Heidelberg/Notebook MaM">[Back]</A></P><H2><SPAN class="mw-headline" id="Measurement_and_Screening"> Measurement and Screening </SPAN></H2><H3><SPAN class="mw-headline" id="Screening_by_TECAN"> Screening by TECAN </SPAN></H3><P>Cells were grown in black 96 well plates (PerkinElmer) and induced as <A href="/wiki/index.php?title=Notebook/methods&amp;action=edit&amp;redlink=1" class="new" title="Notebook/methods (page does not exist)">described</A> for the various pathways. Outer wells were left free. After induction, media was removed and replaced by PBS. Fluorescence was then measured using a TECAN infiniteM200 automated plate reader (excitation = 488 nm, emission=518 nm). In order to account for variations in cell number, cells were stained by <A href="http://www.sigmaaldrich.com/etc/medialib/docs/Sigma/Product_Information_Sheet/b2261pis.Par.0001.File.tmp/b2261pis.pdf" class="external text" rel="nofollow">Hoechst 33342 dye</A> . PBS was removed and replaced by PBS containing Hoechst 33342 dye at a concentration of 1µg/mL. Cells were allowed to incubate at room temperature for 30 minutes. Afterwards, medium was replaced by PBS again and fluorescence (excitation = 355nm, emission=455nm) was measured again by TECAN. Fluoresence values from the first reading were divided by fluorescence values of the second reading and multiplied by the lowest value of the second reading. Clones showing induction were considered further.
</P><H3><SPAN class="mw-headline" id="Flow_Cytometry_.28FC.29"> Flow Cytometry (FC) </SPAN></H3><P>Before preparing samples for flow cytometry the flow cytometer has to be turned on at least 20 min before the run. Samples are then prepared in the following way (this is for a 96 well plate): As much DMEM+++ as possible is removed from the cells. 60 µl Trypsin are added and the cells are incubated at 37 C for 10-15 min. Then, 160 µl of 1xPBS + 1% BSA are added. Samples are now ready for FC. At the FC the correct plate format is chosen. The plate is placed in the machine and the lid is taken off. When starting the flow cytometry measurement we first adjusted the gates and the background signal to the negative control. Measurement parameters such as plateformat, wells, and the desired protocol are chosen from the list. Volume and Mix iterations are are also chosen and saved as... .The discriminator is set to increase the threshold(2-10). The measurement is started with forward vs. side scatter in order to find cells. When cells are found, a gate (line) is added around the cells or cell candidates. Selected cells are then measured. 
Cells were prepared in 96-well format with 10^4 cells/well and transfected with the promoter of interest and if necessary induction drug was added for inducible promoters.
</P><H3><SPAN class="mw-headline" id="Microscopy.2F_Image_Analysis"> Microscopy/ Image Analysis </SPAN></H3><P>Microscopy images of our samples fixed with 4% formaldehyde were taken using the Nikon Eclipse 90i upright automated widefield microscope in the Nikon Imaging Center at the University of Heidelberg. Each image was taken in the GFP and mCherry channel and the exposure time had to be adjusted for over- or underexposed images. The conversion factor was experimentally determined by imaging a fluorescent plate with exposure time varying from 10 to 70 ms (the estimated range of our promoters) and plotting the mean grey values of these images versus time. A linear relationship between the exposure times and grey values was observed for this time frame and the conversion factor was determined to be 0.998. For image analysis we used the ImageJ (Image Processing and Analysis in Java) software measuring the mean grey values of each cell containing the promoter of interest. ImageJ is an open source software developed at the National Institutes of Health (<A href="http://rsbweb.nih.gov/ij/" class="external free" rel="nofollow">http://rsbweb.nih.gov/ij/</A>). The measurement is performed by setting the boundaries around the cells using the threshold in the mCherry image and generating a mask that is then applied to the image of the GFP channel in the following steps:
</P><UL><LI> Open image and select split channels
</LI><LI> Select mCherry image (all images should be 8-bit)
</LI><LI> Go to Image&gt; Adjust&gt; Threshold
</LI><LI> Set lower bar to maximum and adjust upper bar so that the cells are still visible
</LI><LI> Remove unwanted spots such as dead cell using the drawing tools
</LI><LI> Apply the threshold (image should appear in all white now with black spots indicating the cells)
</LI><LI> Go to Process&gt; Math&gt; Divide and divide the image by 255 (image should be all white now)
</LI><LI> Next go to Process&gt; Image Calculator and multiply the GFP channel with the mCherry channel pictures.
</LI><LI> Now select the result image and adjust the threshold so that the lower bar is set to the maximum (255) and the upper bar is set to 1 (only GFP containing cells should be visible through the mask)
</LI><LI> Go to Analyze&gt; Set Measurement and make sure the measurement is limited to the threshold and the result image of the multiplication is selected.
</LI><LI> Go to Analyze&gt; Measure.
</LI><LI> Mean grey values should show up in the 'Results' window 
</LI></UL><P>Alternatively the images can be analyzed using the ROI manager where the cell have to be selected by hand using the drawing tools and added to the ROI list. In comparison the threshold method is much quicker and the results are comparable. We preselect the mCherry positive cells which are very likely to contain GFP. The mean grey values of 5-10 images per sample were averaged and adjusted to the same exposure time. The results were given in REU relative to JeT, which was always prepared as a reference sample for each measurement.
</P><H3><SPAN class="mw-headline" id="Real-time_RT-PCR"> Real-time RT-PCR </SPAN></H3><P>Cells were collected as a pellet. Cell number must excede 1*10<SUP>5</SUP>cells/tube and no more than 6*10<SUP>6</SUP>/tube due to column capacity. In order to obtain reliable results and reduce the variance during the extraction, reverse transcription and PCR, we did the measurement for each promoter with 12 replicates from an identical sample. Resuspend the pellet with 600µl RLT buffer containing ß-ME (1:100). QIAcube follows protocol &quot;RNeasy Mini Animal Cell QIAshredder Version 3&quot; using RNeasy Plant Mini Kit. mRNA from untransfected HeLa cells was extracted to be &quot;Standard&quot; to perform plate-to-plate correction.
After RNA is extracted, the extracts were put on ice immediately. 2 tubes of 5 µl aliquots were taken, One for measuring RNA concentration and one for gel electrophoresis. The RNA concentration was measured using NanoDrop under 230 nm. Each sample was measured 3 times. The gel electrophoresis was performed to confirm the RNA integrity because RNA degrade under many conditions. We used 1% agarose gel and 0.5x TAE buffer. 
Next step is the Real-time RT-PCR. We had different setup for different kits.
</P><UL><LI>For SensiMix One-Step Kit (Quantace):
</LI><LI>96-well reaction setup (25 µl/reaction):
</LI><LI>2X MasterMix:                12.5 µl
</LI><LI>forward primer (5 µM):     1.0 µl
</LI><LI>reverse primer (5 µM):     1.0 µl
</LI><LI>probe (1.67 µM):              1.0 µl
</LI><LI>RNase inhibitor:                0.5 µl
</LI><LI>MgCl:                                0.5 µl
</LI><LI>RNase-free water:            3.5 µl
</LI><LI>RNA template (20 ng/µl): 5.0 µl
</LI></UL><LI>Program setup:
</LI><LI>Reverse transcription: 49°C, 30 min
</LI><LI>Enzyme activation: 95°C, 10 min
</LI><LI>Amplification: 35 cycles:
</LI><LI>95°C, 10 s
</LI><LI>60°C, 1 min
</LI><LI>For QuantiTect Probe RT-PCR Kit (QIAGEN):
</LI><LI>96-well reaction setup (25 µl/reaction):
</LI><LI>2X MasterMix: 12.5 µl
</LI><LI>forward primer (5 µM): 2 µl
</LI><LI>reverse primer (5 µM): 2 µl
</LI><LI>probe (1.67 µM):  2 µl
</LI><LI>RNase-free water: 1.25µl
</LI><LI>RT Mix: 0.25µl
</LI><LI>RNA template (20 ng/µl): 5.0 µl
</LI><LI>Program setup:
</LI><LI>Reverse transcription: 50°C, 30 min
</LI><LI>Enzyme activation: 95°C, 15 min
</LI><LI>Amplification: 45 cycles:
</LI><LI>94°C, 15 s
</LI><LI>60°C, 1 min
</LI><P>In each plate, the 5 housekeeping genes were crossed with 12 replicates as well as Standard. eGFP was mixed with 12 replicates and RNase-free water as negative control. After the program finished, data was collected from OneStep software 2.0 associated to the StepOnePlus system. With a certain threshold (0.05), a series of Ct values were determined, then analyzed with MatLab.
</P><P><A href="/Team:Heidelberg/Notebook_MaM#Methods" title="Team:Heidelberg/Notebook MaM">[Back]</A></P><H2><SPAN class="mw-headline" id="References"> References </SPAN></H2><P>[1] Brady N. Bioquant Heidelberg, Eils group; Personal communication. 
[2] Schilde J. Institute for human genetics at the University of Heidelberg; Personal communication.
[3] Panigrahy D., Singer S., Shen L. Q., Butterfield C. E., Freedman D. A., Chen E. J., Moses M. A., Kilroy S., Duensing S., Fletcher C., Fletcher J. A., Hlatky L., Hahnfeldt P., Folkman J. &amp; Kaipainen A. PPARγ ligands inhibit primary tumor growth and metastasis by inhibiting angiogenesis. <I>Journal of clinical cancer</I> 110: 923-932 (2002). 
[4] Lucarelli E., Sangiorgi L., Maini V., Lattanzi G., Marmiroli S., Reggian M., Mordenti M., Gobbi G. A., Scrimieri F., Bertoja A. Z. &amp; Picci P. Troglitazione affects survival of human osteosarcoma cells. <I>International Journal of Cancer</I> 98(3): 344-351 (2002). 
[5] Komarov P. G., Komarova E. A., Kondratov R. V., Christov-Tselkov K., Coon J. S., Chernov M. V. &amp; Gudkov A. V. A chemical inhibitor of p53 that protects mice from the side effects of cancer therapy. <I>Science</I> 285(5434): 1733-7 (1999). 
[6] Wang S. H., Liang C. T., Liu Y. W., Huang M. C., Huang S. C., Hong W. F. &amp; Su J. G. Crosstalk between activated forms of the aryl hydrocarbon receptor and glucocorticoid receptor. <I>Toxicology</I> 262(2): 87-97 (2009). 
[7] Diesing D., Cordes T., Fischer D., Diedrich K. &amp; Friedrich M. Vitamin D--metabolism in the human breast cancer cell line MCF-7. <I>Anticancer Res.</I> 26(4A): 2755-9 (2006). 
[8] Bartz F., Kern L., Erz D., Zhu M., Gilbert D., Meinhof T., Wirkner U., Erfle H., Muckenthaler M., Pepperkok R. &amp; Runz H. Identification of Cholesterol-Regulating Genes by Targeted RNAi Screening. <I>Cell Metabolism</I> 10: 63-75 (2009). 
[9] Runz H., Miura K., Weiss M. &amp; Pepperkok R. Sterols regulate ER-export dynamics of secretory cargo protein ts-O45-G. <I>The EMBO Journal</I> 25: 2953 - 2965 (2006). 
[10] Treede I., Braun A., Jeliaskova P., Giese T., Füllekrug J., Griffiths G., Stremmel W. &amp; Ehehalt R. TNF-α-induced up-regulation of pro-inflammatory cytokines is reduced by phosphatidylcholine in intestinal epithelial cells. <I>BMC Gastroenterology</I> 9:53  (2009)
</P><P><A href="/Team:Heidelberg/Notebook_MaM#Methods" title="Team:Heidelberg/Notebook MaM">[Back]</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Heidelberg/Notebook_MaM">http://2009.igem.org/Team:Heidelberg/Notebook_MaM</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Notebook_MaM" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Notebook_MaM" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook_MaM&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook_MaM&amp;oldid=169540" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>