"
Page
Discussion
View source
History
teams
Log in
 
Team:Heidelberg/Project References
From 2009.igem.org
Home
Team
Overview
Students
Advisors
Scientific Environment
Gallery
Project
Abstract
Introduction
Measurement
Synthetic Promoters
HEARTBEAT
Further Subprojects
Summary & Outlook
References
Acknowledgements
Parts
Parts
Characterization
HEARTBEAT
What is HEARTBEAT?
HEARTBEAT Database
HEARTBEAT GUI
GUI Documentation
HEARTBEAT Fuzzy Network Modeling
Notebook
Natural Promoters
Synthetic Promoters
Cell Culture, Promoters
Stable Cell Line
Multi-Color Output
HEARTBEAT
Measurements
Material & Methods
Team Meetings
Biosafety
Eukaryopedia
Sponsors
References
Introduction
"Synthetic Biology. Applying Engineering to Biology". Report of a NEST High-Level Expert Group. European Commission, Directorate-General for Research. Available online at http://www.synbiosafe.eu/uploads///pdf/EU-highlevel-syntheticbiology.pdf
Endy, D. Foundations for engineering biology. Nature 438: 449-453 (2005).
Endy, D. & Deese I. Adventures in synthetic biology (comic). Nature 438: 449-453 (2005) Available online at nature.com
http://www.partsregistry.org/Assembly:Standard_assembly (Website, accessed 10/15/2009)
Kelly J. R., Rubin A. J., Davis J. H., Ajo-Franklin C. M., Cumbers J., Czar M. J., de Mora K., Glieberman A. L., Monie D. D. & Endy D. Measuring the activity of BioBrick promoters using an in vivo reference standard. Journal of Biological Engineering 3:4 (2009).
Voigt, C. Genetic parts to program bacteria. Current Opinion in Biotechnology 17 (5): 548–557 (2006).
Andrianantoandro E., Basu S., Karig D. K. & Weiss R. Synthetic biology: new engineering rules for an emerging discipline. Molecular Systems Biology 2: article number 0028 (2006).
https://2009.igem.org/About (Website, accessed 10/15/2009)
Goodman, C. Engineering ingenuity at iGEM. Nat Chem Biology 4:13 (2008).
Basu, S. A synthetic multicellular system for programmed pattern formation. Nature 434: 1130-1133 (2005).
Ellis T., Wang X. & Collins J.J. Diversity-based, model-guided construction of synthetic gene networks with predicted functions. Nature Biotechnology 27: 465-471 (2009).
Canton, B., Anna Labno A. & Endy D. Refinement and standardization of synthetic biological parts and devices. Nature Biotechnology 26: 787-793 (2009).
Weber, W. & Fussenegger, M. Engineering of Synthetic Mammalian Gene Networks. Chemistry and Biology 16: 287-297 (2009).
http://openwetware.org/wiki/The_BioBricks_Foundation:Standards/Technical/Formats (Website, accessed 10/19/2009)
Ducrest A-L., Amacker M., Lingner J. & Nabholz M. Detection of promoter activity by flow cytometric analysis of GFP reporter expression. Nucleic Acids Res. 30: e65 (2002).
Venter M. Synthetic promoters: genetic control through cis engineering. Trends in Plant Science 12: 118-124 (2007). (and the references cited therein)
Ogawa R. Construction of strong mammalian promoters by random cis-acting element elongation. Biotechniques 42: 628-632 (2007).
Tornoe J. Generation of a synthetic mammalian promoter library by modification of sequences spacing transcription factor binding sites. Gene 297: 21-32 (2002).
Dorer D. E. & Nettelbeck D. Targeting cancer by transcriptional control in cancer gene therapy and viral oncolysis. Advanced Drug Delivery Reviews 61: 554-557 (2009).
Weber W. & Fussenegger M. The impact of synthetic biology on drug discovery. Drug Discovery Today 14: 956-963 (2009).
Natural Promoters
Alberts B., Johnson A., Walter P. & Lewis J. Molecular Biology of the Cell. 4th edition, 2008. Garland Science, Chapter 6, p. 357-359.
Clarke N., Arenzana N., Hai T., Minden A. & Prywes R. Epidermal Growth Factor Induction of the c-jun Promoter by a Rac Pathway. Mol Cell Biol 18: 1065–1073 (1998).
Hoffman L. M., Garcha K., Karamboulas K., Cowan M. F., Drysdale L. M., Horton W. A. & Underhill T. M. BMP action in skeletogenesis involves attenuation of retinoid signaling. J Cell Biol 174: 101–113 (2006).
Castoreno A. B., Wang Y., Stockinger W., Jarzylo L. A., Du H., Pagnon J. C., Shieh E. C. & Nohturfft A. Transcriptional regulation of phagocytosis-induced membrane biogenesis by sterol regulatory element binding proteins. Proc Natl Acad Sci U S A 102: 13129–13134 (2005).
Schmidt M., Heimberger T., Gruensfelder P., Schler G. & Hoppe F. Inducible promoters for gene therapy of head and neck cancer: an in vitro study. Eur Arch Otorhinolaryngol 261: 208–215 (2004).
Jorgensen E. C. B. & Autrup H. Autoregulation of human CYP1A1 gene promoter activity in HepG2 and MCF-7 cells. Carcinogenesis 17: 435-441 (1996).
Schreiber T. D., Köhle C., Buckler F., Schmohl S., Braeuning A., Schmiechen A., Schwarz M. & Münzel P.A. Regulation of CYP1A1 expression by the antioxidant Tert butylhydroquinone. Drug Metabolism And Disposition 43: 1096–1101 (2006).
Synthetic Promoters
Alberts B., Johnson A., Walter P. & Lewis J. Molecular Biology of the Cell. 4th edition, 2008. Garland Science, p. 432-453
Fussenegger M. & Weber, W. Engineering of Synthetic Mammalian Gene Networks. Chemistry and Biology 16: 287-297 (2009).
Gossen M. & Bujard, L. Tight control of gene expression in mammalian cells by tetracycline-responsive promoters. PNAS 89: 5547-5551 (1992).
Dorer D. E. & Nettelbeck, D. Targeting cancer by transcriptional control in cancer gene therapy and viral oncolysis. Advanced Drug Delivery Reviews 61: 554-557 (2009).
Venter M. Synthetic promoters: genetic control through cis engineering. Trends in Plant Science 12: 118-124 (2007). (and the references cited therein)
Rushton P. J., Reinstädler A., Lipka V., Lippok B. & Somssich I. E. Synthetic plant promoters containing defined regulatory elements provide novel insights into pathogen- and wound-induced signalling. Plant Cell 14: 749–762 (2002).
Ogawa R. Construction of strong mammalian promoters by random cis-acting element elongation. Biotechniques 42: 628-632 (2007).
Edelman G. M., Meech R., Owens G. C. & Jones F. S. Synthetic promoter elements obtained by nucleotide sequence variation and selection for activity. PNAS 97: 3038-43 (2000).
Ellis T., Wang X. & Collins J. J. Diversity-based, model-guided construction of synthetic gene networks with predicted functions. Nature Biotechnology 27: 465-471 (2009).
Tornoe J. Generation of a synthetic mammalian promoter library by modification of sequences spacing transcription factor binding sites. Gene 297: 21-32 (2002).
Heintzman N. D. & Ren B. The gateway to transcription: identifying, characterizing and understanding promoters in the eukaryotic genome. Cellular and Molecular Life Science 64: 386-400 (2007).
Stemmer W. P., Crameri A., Ha K. D., Brennan T. M. & Heyneker H. L. Single-step assembly of a gene and entire plasmid from large numbers of oligodeoxyribonucleotides. Gene 164: 49-53 (1995).
Rattner A. NF-κB activates the HIV promoter in neurons. EMBO 12: 4261–4267 (1993).
Yokoyama D. K., Ohler U. & Wray G. A. Measuring spatial preferences at fine-scale resolution identifies known and novel cis-regulatory element candidates and functional motif-pair relationships. Nuc Acids Res: 1-21 (2009).
del Val C., Pelz O., Glatting K-H, Barta E. & Hotz-Wagenblatt A. PromoterSweep: a tool for identification of transcription factor binding site. Theor Chem Acc (in press)
Ghosh S., May M. J. & Kopp E. B. Rel/NF-κB and IKB proteins: an overview. Seminars in Cancer Biology 8: 63-73 (1997).
Moyano P. & Rotwein P. Mini-review: estrogen action in the uterus and insulin-like growth factor-I. Growth Hormone & IGF Research 14: 431-435 (2009).
Stable cell line
Zhou H. Development of site-specific integration system to high-level expression recombinant proteins in CHO cells. Chinese journal of biotechnology 23: 756-62 (2007).
Andrews B. J., Proteau G. A., Beatty L. G. & Sadowski P. D. The FLP Recombinase of the 2μ Circle DNA of Yeast: Interaction with Its Target Sequences. Cell 40: 795-803 (1985).
Senecoff J. F., Bruckner R. C. & Cox M. M. The FLP recombinase of the yeast 2-μ plasmid: Characterization of its recombination site. Proc. Nati. Acad. Sci. USA 82: 7270-7274 (1985).
Holmberg A., Blomstergren A., Nord O., Lukacs M., Lundeberg J. & Uhlén M. The biotin-streptavidin interaction can be eversibly broken using water at elevated temperatures. Electrophoresis 26(3): 501-10 (2005).
Schmidt M., Schwarzwaelder K., Bartholomae C., Zaoui K., Ball B., Pilz I., Braun S., Glimm H., von Kalle C., High-resolution insertion-site analysis by linear amplification–mediated PCR (LAM-PCR). Nature Methods 4 (12): 1051-1057 (2005).
Outlook and Summary
Venter M. Synthetic promoters: genetic control through cis engineering. Trends in Plant Science 12: 118-24 (2007).
Carey M., Smale S. T. & Hughes H., Transcriptional Regulation in Eukaryotes: Concepts, Strategies and Techniques. New York: CSHL: p. 18-25 (2000).
HEARTBEAT Database & Fuzzy Modeling
References
Harbison C. T., Gordon D. B., Lee T. I., Rinaldi N. J., Macisaac K. D., Danford T. W., Hannett N. M., Tagne J. B., Reynolds D. B., Yoo J., Jennings E. G., Zeitlinger J., Pokholok D. K., Kellis M., Rolfe P. A., Takusagawa K. T., Lander E. S., Gifford D. K., Fraenkel E., Young R. A. Transcriptional regulatory code of a eukaryotic genome. Nature 431: 99-104 (2004).
Hu Z., Killion P. J., Iyer, V. R. Genetic reconstruction of a functional transcriptional regulatory network. Nature Genet. 39: 683-687 (2007).
Gertz J., Siggia E. D., Cohen B. A. Analysis of combinatorial cis-regulation in synthetic and genomic promoters. Nature 457: 215-218 (2009)
Roider H. G., Kanhere A., Manke T., Vingron M. Predicting transcription factor affinities to DNA from a biophysical model. Bioinformatics 23: 134-141 (2006)
Carrera J., Rodrigo G., Jaramillo A. Towards the automated engineering of a synthetic genome. Mol. Biosyst. 5: 733-43 (2009).
Agapakis C. M., Silver P. A. Synthetic biology: exploring and exploiting genetic modularity through the design of novel biological networks. Mol. Biosyst. 5: 704-13 (2009)
Purnick P. E., Weiss R. The second wave of synthetic biology: from modules to systems. Nat Rev Mol Cell Biol. 10: 410-22 (2009).
Bhalerao K. D. Synthetic gene networks: the next wave in biotechnology? Trends Biotechnol. 27: 368-74 (2009).
Andrianantoandro E., Basu S., Karig D. K., Weiss R. Synthetic biology: new engineering rules for an emerging discipline. Mol Sys Biol (2006)
Alberts B., Johnson A., Walter P., Lewis J. Molecular Biology of the Cell, 5th edition, 2008. Garland Science, Chapter 6
Vardhanabhuti S., Wang J., Hannenhalli S. Position and distance specificity are important determinants of cis-regulatory motifs in addition to evolutionary conservation. Nucl Acid Res 35: 3203-3213 (2007).
Yokoyama K. D., Ohler U., Wray G. A. Measuring spatial preferences at fine-scale resolution identifies known and novel cis-regulatory element candidates and functional motif-pair relationships. Nucl Acid Res 37: e92 (2009)
Nelles O. Nonlinear System Identification Springer Verlag GmbH & Co., Berlin, 2000.
Bosl W. J. Systems biology by the rules: hybrid intelligent systems for pathway modeling and discovery.
BMC Systems Biology1:13 (2007).
Laschov D., Margaliot M. Mathematical modeling of the lambda switch:a fuzzy logic approach. J Theor Biol. 21:475-89 (2009)
Aldridge B. B., Saez-Rodriguez J., Muhlich  J. L., Sorger P. K., Lauffenburger D. A. Fuzzy logic analysis of kinase pathway crosstalk in TNF/EGF/insulin-induced signaling PLoS Comput Biol.5:e1000340 (2009).
Roider H. G., Kanhere A., Manke T., Vingron M. Predicting transcription factor affinities to DNA from a biophysical model. Bioinformatics 23: 134-141 (2007).
BCCS-Bristol: Best Model 2008
Heintzman N. D. & Ren B. The gateway to transcription: identifying, characterizing and understanding promoters in the eukaryotic genome. Cellular and Molecular Life Science 64: 386-400 (2007).
Vardhanabhuti, S., Wang, J. & Hannenhalli, S. Position and distance specificity are important determinants of cis-regulatory motifs in addition to evolutionary conservation. Nucl Acid Res 35(10): 3203-3213 (2007).
Yokoyama, K. D., Ohler, U. & Wray, G. A. Measuring spatial preferences at fine-scale resolution identifies known and novel cis-regulatory element candidates and functional motif-pair relationships. Nucl Acid Res 37(13): e92 (2009)
Ellis T., Wang X. & Collins J. J. Diversity-based, model-guided construction of synthetic gene networks with predicted functions. Nature Biotechnology 27: 465-471 (2009).
Kent W. J., Sugnet C. W., Furey T. S., Roskin K. M., Pringle T. H., Zahler A. M. & Haussler D. The human genome browser at UCSC. Genome Res. 12(6): 996-1006 (2002).
Kuhn R. M., Karolchik D., Zweig A. S., Wang T., Smith K. E., Rosenbloom K. R., Rhead B., Raney B. J., Pohl A., Pheasant M., Meyer L., Hsu F., Hinrichs A. S., Harte R. A., Giardine B., Fujita P., Diekhans M., Dreszer T., Clawson H., Barber G. P., Haussler D. & Kent W. J. The UCSC Genome Browser Database: update 2009. Nucleic Acids Res. 37(Database issue): D755-61 (2009).
Kanehisa M., Araki M., Goto S., Hattori M., Hirakawa M., Itoh M., Katayama T., Kawashima S., Okuda S., Tokimatsu T. & Yamanishi, Y. KEGG for linking genomes to life and the environment. Nucleic Acids Res. 36 (Database issue): D480-4 (2008).
Kanehisa M., Goto S., Hattori M., Aoki-Kinoshita K.F., Itoh M., Kawashima S., Katayama T., Araki M. & Hirakawa M. From genomics to chemical genomics: new developments in KEGG. Nucleic Acids Res. 34 (Database issue): D354-7 (2006).
Kanehisa M. & Goto S. KEGG: Kyoto Encyclopedia of Genes and Genomes. Nucleic Acids Res. 28(1): 27-30 (2000).
del Val C., Pelz O., Glatting K.-H., Barta E. & Hotz-Wagenblatt, A. PromoterSweep: a tool for identification of transcription factor binding sites. Theor. Chem. Acc. DOI 10.1007: s00214-009-0643-8 (2009).
Ensembl Compara (database)
Hubbard T., Andrews D., Caccamo M., Cameron G., Chen Y., Clamp M., Clarke L., Coates G., Cox T., Cunningham F., Curwen V., Cutts T., Down T., Durbin R., Fernandez-Suarez X. M., Gilbert J., Hammond M., Herrero J., Hotz H., Howe K., Iyer V., Jekosch K., Kahari A., Kasprzyk A., Keefe D., Keenan S., Kokocinsci F., London D., Longden I., McVicker G., Melsopp C., Meidl P., Potter S., Proctor G., Rae M., Rios D., Schuster M., Searle S., Severin J., Slater G., Smedley D., Smith J., Spooner W., Stabenau A., Stalker J., Storey R., Trevanion S., Ureta-Vidal A., Vogel J., White S., Woodwark C. & Birney E. Ensembl. Nucleic Acids Res. 33 (Database issue): D447-D453 (2005).
NCBI HomoloGene (database)
DoOP (database)
Barta E., Sebestyén E., Pálfy T. B., Tóth G., Ortutay C. P. & Patthy L. DoOP: Databases of Orthologous Promoters, collections of clusters of orthologous upstream sequences from chordates and plants. Nucleic Acids Res. 33 (Database issue): D86-D90 (2004).
EPD (database)
Schmid C. D., Praz V., Delorenzi M., Périer R. & Bucher P. The Eukaryotic Promoter Database EPD: the impact of in silico primer extension. Nucleic Acids Res. 32 (Database issue): D82-5 (2004).
DBTSS (database) Wakaguri H., Yamashita R., Suzuki Y., Sugano S., Nakai K.
DBTSS: database of transcription start sites, progress report 2008. Nucleic Acids Res.36(Database issue): D97-101 (2008)
Gibbs MotifSampler Lawrence (database) C. E., Altschul S. F., Boguski M. S., Liu J. S., Neuwald A.F., Wootton J. C.  Detecting subtle sequence signals: a Gibbs sampling strategy for multiple alignment. Science 262:208-214(1993)
Jaspar Core Library (database) Sandelin A., Alkema W., Engstrom P., Wasserman W. W., Lenhard B. JASPAR: an open-access database for eukaryotic transcription factor binding profiles.
Nucleic Acids Res. 32(Database issue): D91-4 (2004).
Matys V., Kel-Margoulis O. V., Fricke E., Liebich I., Land S., Barre-Dirrie A.,
Reuter I., Chekmenev D., Krull M., Hornischer K., Voss N., Stegmaier P., Lewicki-Potapov B., Saxel H., Kel A. E. & Wingender E. TRANSFAC® and its module TRANSCompel®: transcriptional gene regulation in eukaryotes. Nucleic Acids Res. 34 (Database issue): D108-110 (2006).
Mysql
Kel A. E., Gössling E., Reuter I., Cheremushkin E., Kel-Margoulis O. V. & Wingender E. MATCH: A tool for searching transcription factor binding sites in DNA sequences. Nucleic Acids Res. 31: 3576-3579 (2003).
Hannenhalli S. & Levy S. Predicting transcription factor synergism. Nucleic Acids Res. 30: 4278-84 (2002).
FitzGerald P. C., Shlyakhtenko A., Mir A. A. & Vinson C. Clustering of DNA sequences in human promoters. Genome Res. 14: 1562-74 (2004).
Alberts B., Johnson A., Walter P. & Lewis J. Molecular Biology of the Cell. 5th edition, 2008. Garland Science, Chapter 6
Measurement
Kelly J. R., Rubin A. J., Davis J. H., Ajo-Franklin C. M., Cumbers J., Czar M. J., de Mora K., Glieberman A. L., Monie D. D. & Endy D. Measuring the activity of BioBrick promoters using an in vivo reference standard. Journal of Biological Engineering 3 (2009).
Endy, D. & Deese I. Adventures in synthetic biology (comic) Nature 438: 449-453 (2005). Available online at nature.com
Alberts B., Johnson A., Walter P. & Lewis J. Molecular Biology of the Cell. 5th edition, 2008. Garland Science, Chapter 6
http://www.promega.com/tbs/tm058/tm058.html
Zhou H. Development of site-specific integration system to high-level expression recombinant proteins in CHO cells. Chinese journal of biotechnology 23: 756-62 (2007).
Alberts B., Johnson A., Walter P. & Lewis J. Molecular Biology of the Cell. 5th edition, 2008. Garland Science, Chapter 7, p. 467-476
Zhu X. D. & Sadowski P. D. Cleavage-dependent Ligation by the FLP Recombinase. J Biol Chem 270: 23044-23054 (1995).
York J. D., Odom A. R., Murphy R., Ives E. B. & Wente S. R. A phospholipase C-dependent inositol polyphosphate kinase pathway required for efficient messenger RNA export. Science 285: 96-100 (1999).
Alberts B., Johnson A., Walter P. & Lewis J. Molecular Biology of the Cell. 5th edition, 2008. Garland Science) Chapter 7, p. 905-908
Tornoe, J. Generation of a synthetic mammalian promoter library by modification of sequences spacing transcription factor binding sites. Gene 297: 21-32 (2002).
Alberts B., Johnson A., Walter P. & Lewis J. Molecular Biology of the Cell. 5th edition, 2008. Garland Science, Chapter 7, p. 492
Ducrest, A. L., Amacker M., Lingner J. & Nabholz M. Detection of promoter activity by flow cytometric analysis of GFP reporter expression. Nucleic Acids Res. 30: e65 (2002).
Rushton P. J., Reinstädler A., Lipka V., Lippok B. & Somssich I. E. Synthetic plant promoters containing defined regulatory elements provide novel insights into pathogen- and wound-induced signalling. Plant Cell 14: 749-762 (2002).
Degenhardt T., Rybakova K. N., Tomaszewska A., Moné M. J., Westerhoff H. V., Bruggeman F. J. & Carlberg C. Population-Level Transcription Cycles Derive from Stochastic Timing of Single-Cell Transcription. Cell 138: 489-501 (2009).
Lottspeich F. & Engels J. W.: Bioanalytik (Book. 2nd edition, 2006. Spektrum Akademischer Verlag) Chapter 5, p. 95-96.
http://www1.qiagen.com/Products/RnaStabilizationPurification/RNeasySystem/RNeasyPlantMini.aspx
http://www1.qiagen.com/products/automation/qiacube.aspx
http://www1.qiagen.com/literature/render.aspx?id=398
https://products.appliedbiosystems.com/ab/en/US/adirect/ab?cmd=catNavigate2&catID=604109&tab=DetailInfo
Ormerod M.G.: Flow cytometry: a practical approach (Book. 3rd edition, 2000. Oxford University Press) Chapter 1, p. 7-21.
Material and Methods
Brady N. Bioquant Heidelberg, Eils group; Personal communication.
Schilde J. Institute for human genetics at the University of Heidelberg; Personal communication.
Panigrahy D., Singer S., Shen L. Q., Butterfield C. E., Freedman D. A., Chen E. J., Moses M. A., Kilroy S., Duensing S., Fletcher C., Fletcher J. A., Hlatky L., Hahnfeldt P., Folkman J., Kaipainen A. PPARγ ligands inhibit primary tumor growth and metastasis by inhibiting angiogenesis. Journal of clinical cancer 110: 923-932 (2002)
Lucarelli E., Sangiorgi L., Maini V., Lattanzi G., Marmiroli S., Reggian M., Mordenti M., Gobbi G. A., Scrimieri F., Bertoja A. Z. & Picci P. Troglitazione affects survival of human osteosarcoma cells. International Journal of Cancer 98 (3): 344-351 (2002)
Komarov P. G., Komarova E. A., Kondratov R. V., Christov-Tselkov K., Coon J. S., Chernov M. V., Gudkov A. V. A chemical inhibitor of p53 that protects mice from the side effects of cancer therapy. Science 285(5434): 1733-7 (1999)
Wang S. H., Liang C. T., Liu Y. W., Huang M. C., Huang S. C., Hong W. F., Su J. G. Crosstalk between activated forms of the aryl hydrocarbon receptor and glucocorticoid receptor. Toxicology 262(2): 87-97 (2009)
Diesing D., Cordes T., Fischer D., Diedrich K. & Friedrich M. Vitamin D--metabolism in the human breast cancer cell line MCF-7. Anticancer Res. 26(4A): 2755-9 (2006)
Bartz F., Kern L., Erz D., Zhu M., Gilbert D., Meinhof T., Wirkner U., Erfle H., Muckenthaler M., Pepperkok R. & Runz H. Identification of Cholesterol-Regulating Genes by Targeted RNAi Screening. Cell Metabolism 10: 63-75 (2009)
Runz H., Miura K., Weiss M. & Pepperkok R. Sterols regulate ER-export dynamics of secretory cargo protein ts-O45-G. The EMBO Journal 25: 2953-2965 (2006)
Treede I., Braun A., Jeliaskova P., Giese T., Füllekrug J., Griffiths G., Stremmel W. & Ehehalt R. TNF-α-induced up-regulation of pro-inflammatory cytokines is reduced by phosphatidylcholine in intestinal epithelial cells. BMC Gastroenterology 9:53  (2009)
Eukaryopedia
Clark R. The process of malignant progression in human breast cancer. Annals of oncology: official journal of the European Society for Medical Oncology/ESMO 1: 401-407 (1990).
Osborne C. K., Hobbs K. & Trent J. M. Biological differences among, MCF-7 human breast cancer cell lines from different laboratories. Breast Cancer Research and Treatment 9: 111-121 (1987).
Dickson R. B., Bates S. E., McManaway M. E. & Lippman M. E. Characterization of Estrogen Responsive Transforming Activity in Human Breast Cancer Cell Lines. Cancer Research 46: 1707-1713 (1986).
Booth B. W. & Smith G. H. Roles of transforming growth factor-α in mammary development and disease. Growth Factors 925: 227-235 (2007).
Gey G. O., Coffman W. D. & Kubicek M. T. Tissue culture studies of the proliferative capacity of cervical carcinoma and norml epithelium. Cancer Research 12: 264-265 (1952).
Macville M., Schroeck E., Padilla-Nash H., Keck C., Ghadimi M. B., Zimonjic D., Pospecu N. & Ried T. Comprehensive and definitive moleculare cytogenic characterization of HeLa cells by spectral karyotyping. Cancer Research 59: 141-150 (1999).
Masters J. R. HeLa cells 50 years on: the good, the bad and the ugly. Nature Reviews 2: 315-319 (2002).
Ek E. T. H., Dass C. R. & Choong P. F. M. Commonly used mouse models of osteosarcoma. Critical Reviews in Oncology/Hematology 60: 1-8 (2006).
Ponten J. & Saksela E. Two established in vitro cell lines from human mesenchymal tumours. International Journal of Cancer 2: 434-447 (1967).
Nelissen J. M. D. T., Torensma R., Pluyter M., Adema G. J., Raymakers R. A. P., van Kooyk Y. & Figdor, C. G. Molecular analysis of the hematopoiesis supporting osteoblastic cell line U2-OS. Experimental Hematology 28: 422-432 (2000).
May, M. J. & Ghosh, S. Rel/NF-κB and IKB proteins: an overview. Seminars in Cancer Biology 8: 63-73 (1997).
Courtois G. The NF-κB signaling pathway in human genetic diseases. Cell. Mol. Life Sci. 62: 1682-1691 (2005).
Vazquez A., Bond E. E., Levine A. J. & Bond G. L. The genetics of the p53 pathway, apoptosis and cancer therapy. Nat Rev Drug Discov 7(12): 979-87 (2008).
Mangelsdorf D. J. & Evans R. M. The RXR heterodimers and orphan receptors. Cell 83: 841–850 (1995).
Koeffler H. P. Peroxisome Proliferator-activated Receptor and Cancers. Clinical Cancer Research 9: 1-9 (2003).
Suh N., Wang Y., Honda T., Gribble G. W., Dmitrovsky E., Hickey W. F., Maue R. A., Place A. E., Porter D. M., Spinella M. J., Williams C. R., Wu G., Dannenberg A. J., Flanders K. C., Letterio J. J., Mangelsdorf D. J., Nathan C. F., Nguyen L., Porter W. W., Ren R. F., Roberts A. B., Roche N. S., Subbaramaiah K. & Sporn M. B. A novel synthetic oleanane triterpenoid, 2-cyano-3,12-dioxoolean-1,9- dien-28-oic acid, with potent differentiating, antiproliferative, and antiinflammatory activity. Cancer Res. 59: 336–341 (1999).
Brown M. S. & Goldstein J. L . The SREBP pathway: regulation of cholesterol metabolism by proteolysis of a membrane-bound transcription factor. Cell 89 (3): 331–40 (1997).
Briggs M. R., Kadonaga J. T., Bell S. P. & Tjian R. Purification and biochemical characterization of the promoter-specific transcription factor, Sp1. Science 234: 47-52 (1986).
Stielow B., Sapetschnig A., Wink C., Kruger I. & Suske G. SUMO-modified Sp3 represses transcription by provoking local heterochromatic gene silencing. EMBO Rep. 9: 899-906 (2008).
Zhang Y.,  Liao M. & Dufau M. L. Unlocking repression of the human luteinizing hormone receptor gene by trichostatin A-induced cell-specific phosphatase release. J. Biol. Chem. 283: 24039-24046 (2008).
Nicole Y., Khachigian T. & Khachigian L. M. Sp1 Phosphorylation and Its Regulation of Gene Transcription. Molecular and Cellular Biology 29: 2483-2488 (2009).
Zhong C-Y, Zhou y-M, Douglas G. C., Witschi H-P. & Pinkerton K. E. MAPK/AP-1 signal pathway in tobacco smoke-induced cell proliferation and quamous metaplasia in the lungs of rats. Carcinogenesis 26 (12): 2187–2195 (2005).
Hess J., Angel P. & Schorpp-Kistner M. AP-1 subunits: quarrel and harmony among siblings. J. Cell. Sci. 117: 5965–73 (2004).
Pellikainen J. M. & Kosma V-M.  Activator protein-2 in carcinogenesis with a special reference to breast cancer—A mini review. Int. J. Cancer 120: 2061–2067 (2007).
Hilger-Eversheim K. , Moser M., Schorle H. & Buettner R. Regulatory roles of AP-2 transcription factors in vertebrate development, apoptosis and cell-cycle control. Gene 260(1-2): 1-12 (2000).
Rani C. S. S.,  Qiang M. & Ticku M. K. Potential Role of cAMP Response Element-Binding Protein in Ethanol-Induced N-Methyl-D-aspartate Receptor 2B Subunit Gene Transcription in Fetal Mouse Cortical Cells. Molecular Pharmacology Fast Forward 67: 2126-2136 (2005).
Yin J. C. & Tully T. CREB and the formation of long-term memory. Curr Opin Neurobiol. 6(2): 264-8 (1996).
Pandey S. C., Chartoff E. H., Carlezon W. A., Zou J., Zhang H., Kreibich A. S., Blendy J. A. & Crews F. T. CREB gene transcription factors: Role in molecular mechanisms of alcohol and drug addiction. Alcohol Clin Exp Res. 29(2): 176-184 (2005).
Rany I., Megyesi J. K., Reusch J. E. B. & Safirstein R. L. CREB mediates ERK-induced survival of mouse renal tubular cells after oxidant stress. Kidney Int. 68(4): 1573-82 (2005).
Matuoka K. & Chen K. Y. Transcriptional regulation of cellular ageing by the CCAAT box-binding factor CBF/NF-Y. Ageing Res Rev. 1(4): 639-51 (2002).
Ronchi A., Bellorini M.,  Mongelli N. & Mantovani R. CCAAT-box binding protein NF-Y (CBF, CP1) recognizes the minor groove and distorts DNA. Nucleic Acids Res. 23(22), 4565–4572 (1995).
Carlberg C. & Seuter S. A genomic perspective on vitamin D signaling. Anticancer Res. 9: 3485-93 (2009).
Bertone-Johnson E. R. Vitamin D and breast cancer. Ann Epidemiol. 7: 462-7 (2009).
Obata T., Yanagidani A., Yokoro K., Numoto M. & Yamamoto S. Analysis of the consensus binding sequence and the DNA-binding domain of ZF5. Biochem Biophys Res Commun. 255(2): 528-34 (1999).
Numoto M., Yokoro K. & Koshi J. ZF5, which is a Kruppel-type transcriptional repressor, requires the zinc finger domain for self-association. Biochem Biophys Res Commun. 256(3): 573-8 (1999).
Scholz H., Wagner K. D. & Wagner N. Role of the Wilms' tumour transcription factor, Wt1, in blood vessel formation. Pflugers Arch. 458(2): 315-23 (2009).
Sakamoto Y., Mariya Y., Sasaki S., Teshiromori R., Oshikiri T., Segawa M., Ogura K., Akagi T., Kubo K., Kaimori M. & Funato T. WT1 mRNA level in peripheral blood is a sensitive biomarker for monitoring minimal residual disease in acute myeloid leukemia. Tohoku J Exp Med. 219(2): 169-76 (2009).
Dollé P. Developmental expression of retinoic acid receptors (RARs). Nucl Recept Signal. 7: e006 (2009).
Oh-hora M. & Rao A. The calcium/NFAT pathway: role in development and function of regulatory T cells. Microbes Infect. 11(5): 612-9. (2009)
Goldstein J. L. & Brown M. S. The LDL receptor. Arterioscler Thromb Vasc Biol. 4: 431-8 (2009).
Hegardt F. G. Transcriptional regulation of mitochondrial HMG-CoA synthase in the control of ketogenesis. Biochimie 80(10): 803-6 (1998).
Yu J. & Zhang L. PUMA, a potent killer with or without p53. Oncogene 27: S71-S83 (2008).
Li Z. & Srivastava P. Heat-shock proteins. Curr Protoc Immunol. Appendix 1: Appendix 1T, (2004).
Tso P., Liu M., Kalogeris T. J. & Thomson A. B. R. The role of apolipoprotein A-IV in the regulation of food intake. Annu. Rev. Nutr. 21: 231–54 (2001).
Androutsopoulos V. P., Tsatsakis A. M. & Spandidos D. A. Cytochrome P450 CYP1A1: wider roles in cancer progression and prevention. BMC Cancer 16:9 187, (2009).
Carpenter G. & Cohen S. Epidermal growth factor. The Journal of Biological Chemistry 265(14): 7709–12 (1990).
Schnidar H., Eberl M., Klingler S., Mangelberger D., Kasper M., Hauser-Kronberger C., Regl G., Kroismayr R., Moriggl R., Sibilia M. & Aberger F. Epidermal growth factor receptor signaling synergizes with Hedgehog/GLI in oncogenic transformation via activation of the MEK/ERK/JUN pathway. Cancer Res. 69(4): 1284-92 (2009).
Moore M. J. & Proudfoot N. J. Pre-mRNA processing reaches back to transcription and ahead to translation. Cell 136(4): 688-700 (2009).
Berg J. M., Tymoczko J. L. & Stryer L. Biochemistry. 6th edition, 2007. New York: WH Freeman & Co
Day D. A. & Tuite M. F. Post-transcriptional gene regulatory mechanisms in eukaryotes: an overview, J. Endocrinol. 157: 361-371 (1998).
Litt M., Qiu Y. & Huang S. Histone arginine methylations: their roles in chromatin dynamics and transcriptional regulation. Biosci Rep. 29(2): 131-41 (2009).
Wall M. E., Wani M. C., Cook C. E., Palmer K. H., McPhail A. I., Sim G. A. Plant antitumor agents. I. The isolation and structure of camptothecin, a novel alkaloidal leukemia and tumor inhibitor from camptotheca acuminate. In: J. Am. Chem. Soc. 88, 3888–3890(1966).
Redinbo M. R., Stewart L., Kuhn P., Champoux J. & Hol W. G. J. Crystal structure of human topoisomerase I in covalent and noncovalent complexes with DNA. Science 279: 1504–1513 (1999).
Gao L., Sun C., Qiu H., Liu H., Shao H., Wang J., Li W. Cloning and characterization of a novel human zinc ﬁnger gene, hKid3, from a C2H2-ZNF enriched human embryonic cDNA library, Biochemical and Biophysical Research Communications 325:1145–1152 (2004)
Rahman S., Thomas P. Molecular cloning, characterization and expression of two hypoxia-inducible factor alpha subunits, HIF-1α and HIF-2α, in a hypoxia-tolerant marine teleost, Atlantic croaker (Micropogonias undulatus), Gene 396:273 – 282 (2007)
Fricker M., Runions J. & Moore I. Quantitative fluorescence microscopy: From Art to science. Annual Review of Plant Biology 57: 79-107 (2006).
Heim R. & Tsien R. Y. Engineering green fluorescent protein for improved brightness, longer wavelengths and fluorescence resonance energy transfer. Current Biology 6(2): 178-182 (1996).
Alberts B., Johnson A., Walter P. & Lewis J. Molecular Biology of the Cell. Fifth edition. Garland Science London/New York, p. 742
d'Enfert C., Gensse M. & Gaillardin C. Fission yeast and a plant have functional homologues of the Sari and Sec12 proteins involved in ER to Golgi traffic in budding yeast. The EMBO Journal 11: 4205-4211 (1992).
Zacharias D. A., Violin J. D., Newton A. C. & Tsien R. Y. Partitioning of Lipid-Modified Monomeric GFPs into Membrane Microdomains of Live Cells. Science296(5569): 913-6 (2002).
Alberts B., Johnson A., Walter P. & Lewis J. Molecular Biology of the Cell. Fifth edition. Garland Science London/New York, pp. 706-707
Locksley R. M., Killeen N. & Lenardo M. J. The TNF and TNF receptor superfamilies: integrating mammalian biology. Cell 104(4): 487–501 (2001).
Komarova E. A. & Gudkov A. V. Suppression of p53: a new approach to overcome side effects of antitumor therapy. Biochemistry (Mosc). 65(1): 41-48 (2000).
Murphy P. J., Galigniana M. D., Morishima Y., Harrell J. M., Kwok R. P., Ljungman M. & Pratt W. B. Pifithrin-alpha inhibits p53 signaling after interaction of the tumor suppressor protein with hsp90 and its nuclear translocation. J Biol Chem. 279(29): 30195-201 (2004).
Tsien R. Y. The green fluorescent protein. Annu. Rev.
Biochem. 67: 509–44 (1998).
Tsien R. & Prasher D. Green Fluorescent Protein Properties, Applications, and Protocols. New York: Wiley-Liss, 1998, p. 67-118.
Heim R. & Tsien R. Engineering Green Fluorescent Protein for improved brightness, longer wavelength and fluorescence resonance energy transfer. Current Biology 6: 178-182 (1996).
Gao L., Sun C., Qiu H., Liu H., Shao H., Wang J., Li W. Cloning and characterization of a novel human zinc ﬁnger gene, hKid3, from a C2H2-ZNF enriched human embryonic cDNA library. Biochemical and Biophysical Research Communications 325:1145–1152 (2004)
Rahman S. & Thomas P. Molecular cloning, characterization and expression of two hypoxia-inducible factor alpha subunits, HIF-1α and HIF-2α, in a hypoxia-tolerant marine teleost, Atlantic croaker (Micropogonias undulatus). Gene 396: 273–282 (2007).
Retrieved from " http://2009.igem.org/Team:Heidelberg/Project_References "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
