<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Modeling_network"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Heidelberg/Modeling_network">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Modeling_network&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Modeling_network&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Modeling_network&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Modeling_network" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Modeling network</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="main_wrapper"><DIV id="header"><DIV id="navigation"><DIV id="menu" style="position:static"><UL><LI id="mainHome"><A class="aMain" href="https://2009.igem.org/Team:Heidelberg">Home</A></LI><LI id="mainTeam"><A class="aTeam" href="https://2009.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Members#Students">Students</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Members#Advisors">Advisors</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Scientific_Environment">Scientific Environment</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Gallery">Gallery</A></LI></UL><LI id="mainProject"><A class="aProject" href="https://2009.igem.org/Team:Heidelberg/Project">Project</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project">Abstract</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Introduction">Introduction</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Measurement">Measurement</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Synthetic_promoters">Synthetic Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Further">Further Subprojects</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_SaO">Summary &amp; Outlook</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_References">References</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Acknowledgement">Acknowledgements</A></LI><LI id="mainParts"><A class="aParts" href="https://2009.igem.org/Team:Heidelberg/Parts">Parts</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Parts">Parts</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Parts_Characterization">Characterization</A></LI><LI id="mainModeling"><A class="aModeling" href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">What is HEARTBEAT?</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_database">HEARTBEAT Database</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_GUI">HEARTBEAT GUI</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_gui">GUI Documentation</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_network">HEARTBEAT Fuzzy Network Modeling</A></LI><LI id="mainNotebook"><A class="aNotebook" href="https://2009.igem.org/Team:Heidelberg/Notebook">Notebook</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_natural_promoters_overview">Natural Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_synthetic_promoters">Synthetic Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_promoters_cells">Cell Culture, Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_cell_line">Stable Cell Line</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_color_output">Multi-Color Output</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_modeling">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_measure">Measurements</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_MaM">Material &amp; Methods</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_meetings">Team Meetings</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Biosafety">Biosafety</A></LI><LI id="mainEucaryopedia"><A class="aEucaryopedia" href="https://2009.igem.org/Team:Heidelberg/Eukaryopedia">Eukaryopedia</A></LI><LI id="mainSponsors"><A class="aSponsors" href="https://2009.igem.org/Team:Heidelberg/Sponsors">Sponsors</A></LI></DIV></DIV></DIV></DIV><TABLE><TBODY><TR valign="top" style="margin-left: 2px;"><TD width="650px" style="padding: 0 15px 15px 20px; background-color:#ede8e2"><H1><SPAN class="mw-headline" id="HEARTBEAT_Fuzzy_Modeling"> HEARTBEAT Fuzzy Modeling </SPAN></H1><H2><SPAN class="mw-headline" id="Introduction"> Introduction </SPAN></H2><P>The research field of <A href="/Team:Heidelberg/Eukaryopedia#Regulation_of_transcription_in_eukaryotic_organisms" title="Team:Heidelberg/Eukaryopedia">regulation of gene expression in eukaryotes</A> is a field of biological research growing rapidly <A href="/Team:Heidelberg/Modeling_network#References" title="Team:Heidelberg/Modeling network">[1,2]</A>. Hereby the interaction of DNA with certain proteins known as transcription factors (TFs) plays an essential role for the complex mechanism of transcriptional activation <A href="/Team:Heidelberg/Modeling_network#References" title="Team:Heidelberg/Modeling network">[3,4]</A>. One strong focus of synthetic biology aims at the reconstruction of such gene regulatory networks <A href="/Team:Heidelberg/Modeling_network#References" title="Team:Heidelberg/Modeling network">[5]</A>. To act within the scope of synthetic biology’s duties, the iGEM Team Heidelberg 09 claims that any synthetic promoter can be constructed by using our <A href="/Team:Heidelberg/Project_Synthetic_promoters" title="Team:Heidelberg/Project Synthetic promoters">two methods for the construction of synthetic promoters</A>. However, the only efficient way to construct systems of high complexity (such as computers or airplanes) is simulating these systems on the computer prior to construction <A href="/Team:Heidelberg/Modeling_network#References" title="Team:Heidelberg/Modeling network">[6]</A>. In our case, this strongly emphasizes the necessity of 
<A href="/Team:Heidelberg" title="Team:Heidelberg">HEARTBEAT</A> (Heidelberg Artificial Transcription Factor Binding Site Engineering and Assembly Tool) which comprises data analysis (data analysis (<A href="/Team:Heidelberg" title="Team:Heidelberg">HEARTBEAT DB</A>), a graphical user interface (<A href="/Team:Heidelberg" title="Team:Heidelberg">HEARTBEAT GUI</A>) and network modeling (<A href="/Team:Heidelberg" title="Team:Heidelberg">HEARTBEAT fuzzy network (FN) modeling</A>).
</P><P>Contributing to the HEARTBEAT project, HEARTBEAT FN focuses on simulating the promoter activity by integrating a variety of signals and sequence characteristics as well as on predicting distinct pathway functionalities. This, especially in eukaryotic systems, is a tough challenge since transcriptional activity of a gene is not directly correlated to protein expression <A href="/Team:Heidelberg/Modeling_network#References" title="Team:Heidelberg/Modeling network">[7]</A>.
</P><P>For this purpose we propose fuzzy logic (FL) modeling as an approach to logic-based modeling which is capable of incorporating qualitative data but producing quantitative predictions. New insights will be provided about the operation of gene regulatory networks and relationships between promoter sequence composition and TF-DNA interaction will be unraveled that is understood only marginally so far <A href="/Team:Heidelberg/Modeling_network#References" title="Team:Heidelberg/Modeling network">[8,9]</A>.
</P><H2><SPAN class="mw-headline" id="Background_.2F_Motivation"> Background / Motivation </SPAN></H2><P>We present two different approaches for promoter design resulting in three different types of synthetic promoters: <B>randomly assembled</B><I>constitutive</I> and <I>inducible</I> promoters as well as <B>rationally designed</B> promoters. As an additional type of promoters those <B>occurring in nature</B> can be integrated into vector systems.
These heterogeneous cocktail of promoters can be combined for a precise regulation of pathways. This represents the power of our entire HEARTBEAT project. 
Synthesized promoters can be then used e.g. as a combinatorial gene therapy, i.e. several promoters that are of different types and/or have different strength will be applied as treatment agents. Therefore, a model that not only simulates single promoter activity and following gene expression but also accurately predicts gene expression from combined promoter sequences is indispensable.
</P><P>We constructed a Fuzzy Logic model to provide a formal mathematical framework for prediction of combined activity of multiple promoters upon several stimuli and to gain insight into the mechanisms that generate diverse expression levels.
</P><H2><SPAN class="mw-headline" id="Achievements"> Achievements </SPAN></H2><H3><SPAN class="mw-headline" id="A_Short_Introduction_into_Fuzzy_Logic_Modeling"> A Short Introduction into Fuzzy Logic Modeling </SPAN></H3><P>Fuzzy Logic is a rule-based approximate artificial reasoning method developed by <A href="http://en.wikipedia.org/wiki/Lofti_Zadeh|" class="external text" rel="nofollow">Lotfi Zadeh</A> in 1965. Its motivation is 
the observation that humans often think and communicate in a vague way, and yet can make precise decisions <A href="/Team:Heidelberg/Modeling_network#References" title="Team:Heidelberg/Modeling network">[10]</A>. It has been widely used in engineering and Artificial Intelligence approaches such as Fuzzy Controllers and Fuzzy Expert Systems. Fuzzy Logic has also been used for the modeling of biological pathways <A href="/Team:Heidelberg/Modeling_network#References" title="Team:Heidelberg/Modeling network">[11]</A> and very recently to analyze gene regulatory networks <A href="/Team:Heidelberg/Modeling_network#References" title="Team:Heidelberg/Modeling network">[12]</A>. Key advantages of Fuzzy logic-based approaches are <B>(i)</B> the ability to construct models based on prior knowledge of the system and experimental data and <B>(ii)</B> encode intermediate states for inputs and outputs, thus improving other logic-approaches that can only deal with <I>ON/OFF</I> states such
as Boolean models <A href="/Team:Heidelberg/Modeling_network#References" title="Team:Heidelberg/Modeling network">[13]</A> and <B>(iii)</B> simulations can be derived from both qualitative and quantitative data, both of which can be cast into the form of <I>IF-THEN</I> rules. Thus, FL constitutes a powerful approach for the understanding of heterogeneous datasets.
</P><P>In our project, the complete set of rules will capture the behavior of each promoter in a Multiple-Input Single-Output (MISO) Fuzzy Logic model. Combining the MISO models in a network of all promoters will constitute the final Multiple-Input Multiple-Output (MIMO) model allowing for the simulation and prediction of combined activation of pahways regulated by our promoters. A key advantage of this methodology towards understanding the exclusive pathway activation of our promoters of interest is the possibility to study not only the individual activity of each promoter but also the combined activity, as the signal progresses from one MISO to another.
</P><H2><SPAN class="mw-headline" id="References"> References </SPAN></H2><P>[1] Harbison, C. T. et al. Transcriptional regulatory code of a eukaryotic genome. Nature <B>431</B>, 99-104 (2004).
[2] Hu, Z., Killion, P. J. &amp; Iyer, V. R. Genetic reconstruction of a functional transcriptional regulatory network. Nature Genet. <B>39</B>, 683-687 (2007).
[3] Gertz, J., Siggia E. D. &amp; Cohen, B. A. Analysis of combinatorial <I>cis</I>-regulation in synthetic and genomic promoters. Nature <B>457</B>. 215-218 (2009)
[4] Roider, H. G. et al. Predicting transcription factor affinities to DNA from a biophysical model. Bioinformatics <B>23</B>, 134-141 (2006)
[5] ref to come
[6] Andianantoandro, E. et al. Synthetic biology: new engineering rules for an emerging discipline. Mol Sys Biol (2006)
[7] Alberts, B. et al. Molecular Biology of the Cell, 5th edition. Garland Science, 2008, Chapter 6
[8] Vardhanabhuti, S., Wang, J. &amp; Hannenhalli, S. Position and distance specificity are important determinants of <I>cis</I>-regulatory motifs in addition to evolutionary conservation. Nucl Acid Res <B>35</B>, 3203-3213 (2007).
[9] Yokoyama, K. D., Ohler, U. &amp; Wray, G. A. Measuring spatial preferences at fine-scale resolution identifies known and novel <I>cis</I>-regulatory element candidates and functional motif-pair relationships. Nucl Acid Res, 1-21 (2009)
[10] Nelles, O. Nonlinear System Identification. Springer, 2000. 
[11] Bosl, W. J. BMC systems biology 1, 13 (2007).
[12] Mathematical modeling of the lambda switch:a fuzzy logic approach.
[13] B. B. Aldridge, J. Saez-Rodriguez, J. L. Muhlich et al., PLoS 
computational biology 5 (4), e1000340 (2009).
</P></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Heidelberg/Modeling_network">http://2009.igem.org/Team:Heidelberg/Modeling_network</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Modeling_network" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Modeling_network" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Modeling_network&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Modeling_network&amp;oldid=141025" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>