<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Notebook_modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Heidelberg/Notebook_modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Notebook_modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook_modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook_modeling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Notebook_modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Notebook modeling</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="main_wrapper"><DIV id="header"><DIV id="navigation"><DIV id="menu" style="position:static"><UL><LI id="mainHome"><A class="aMain" href="https://2009.igem.org/Team:Heidelberg">Home</A></LI><LI id="mainTeam"><A class="aTeam" href="https://2009.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Members#Students">Students</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Members#Advisors">Advisors</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Scientific_Environment">Scientific Environment</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Gallery">Gallery</A></LI></UL><LI id="mainProject"><A class="aProject" href="https://2009.igem.org/Team:Heidelberg/Project">Project</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project">Abstract</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Introduction">Introduction</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Measurement">Measurement</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Synthetic_promoters">Synthetic Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Further">Further Subprojects</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_SaO">Summary &amp; Outlook</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_References">References</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Acknowledgement">Acknowledgements</A></LI><LI id="mainParts"><A class="aParts" href="https://2009.igem.org/Team:Heidelberg/Parts">Parts</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Parts">Parts</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Parts_Characterization">Characterization</A></LI><LI id="mainModeling"><A class="aModeling" href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">What is HEARTBEAT?</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_database">HEARTBEAT Database</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_GUI">HEARTBEAT GUI</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_gui">GUI Documentation</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_network">HEARTBEAT Fuzzy Network Modeling</A></LI><LI id="mainNotebook"><A class="aNotebook" href="https://2009.igem.org/Team:Heidelberg/Notebook">Notebook</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_natural_promoters_overview">Natural Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_synthetic_promoters">Synthetic Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_promoters_cells">Cell Culture, Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_cell_line">Stable Cell Line</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_color_output">Multi-Color Output</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_modeling">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_measure">Measurements</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_MaM">Material &amp; Methods</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_meetings">Team Meetings</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Biosafety">Biosafety</A></LI><LI id="mainEucaryopedia"><A class="aEucaryopedia" href="https://2009.igem.org/Team:Heidelberg/Eukaryopedia">Eukaryopedia</A></LI><LI id="mainSponsors"><A class="aSponsors" href="https://2009.igem.org/Team:Heidelberg/Sponsors">Sponsors</A></LI></DIV></DIV></DIV></DIV><TABLE><TBODY><TR valign="top" style="margin-left: 2px;"><TD width="650px" style="padding: 0 15px 15px 20px; background-color:#ede8e2"><H1><SPAN class="mw-headline" id="Notebook_HEARTBEAT"> Notebook HEARTBEAT </SPAN></H1><P>Welcome to the notebook of the HEARTBEAT (Heidelberg Artificial Transcription Factor Binding Sites Assembly and Engineering Tool) project. This notebook comprises the work on three sublanes: HEARTBEAT database (DB), HEARTBEAT graphical user interface (GUI) and HEARTBEAT fuzzy modeling (FN) as well as some additional work on logo as well as wiki design. Have fun!
</P><H2><SPAN class="mw-headline" id="Contents"><B>Contents</B></SPAN></H2><UL><LI><SPAN style="font-size:5mm;"><A href="/Team:Heidelberg/Notebook_modeling#July" title="Team:Heidelberg/Notebook modeling"> July</A></SPAN></LI></UL><UL><LI><SPAN style="font-size:5mm;"><A href="/Team:Heidelberg/Notebook_modeling#August" title="Team:Heidelberg/Notebook modeling"> August</A></SPAN></LI></UL><UL><LI><SPAN style="font-size:5mm;"><A href="/Team:Heidelberg/Notebook_modeling#September" title="Team:Heidelberg/Notebook modeling"> September</A></SPAN></LI></UL><UL><LI><SPAN style="font-size:5mm;"><A href="/Team:Heidelberg/Notebook_modeling#October" title="Team:Heidelberg/Notebook modeling"> October</A></SPAN></LI></UL><H2><SPAN class="mw-headline" id="July"> July </SPAN></H2><H3><SPAN class="mw-headline" id="7-27-2009"> 7-27-2009 </SPAN></H3><UL><LI> Meeting with Oliver Pelz
</LI><LI> Discuss general ideas of our Database Structure and Content
</LI><LI> An introduction into PromoterSweep (LINK). PromoterSweep screens a given sequence for conserved regions giving us consensus sequences and moreover screens them for TFBS by using database search (TRANSFAC, Jasper) (LINK)
</LI><LI> Our new database should contain following informations: promoter sequence, TFs, TFBS, position of TFBS, number of binding TFBS, &quot;host organism&quot;
</LI><LI> We decide to choose MySQL as a appropiate language solving this challenge which allows us also a graphical representation of the database on the web later.
</LI><LI> GUI on wiki: which language? php? javascript?
</LI><LI> Problems: access to PromoterSweep (Husar Bioinformatics Group, DKFZ), choice of Promoter Database (DoOP, UCSC, EnsEMBL) (LINK)
</LI></UL><UL><LI> aim: create database until end of August
</LI></UL><P>[<A href="https://2009.igem.org/Team:Heidelberg/Notebook_modeling" class="external text" rel="nofollow">TOP</A>]
</P><H2><SPAN class="mw-headline" id="August"> August </SPAN></H2></TD></TR><TR><TH> Week </TH><TH colspan="7">Days
</TH></TR><TR><TD style="text-align:center"> Mon
</TD><TD style="text-align:center"> Tue
</TD><TD style="text-align:center"> Wed
</TD><TD style="text-align:center"> Thu
</TD><TD style="text-align:center"> Fri
</TD><TD style="text-align:center"> Sat
</TD><TD style="text-align:center"> Sun
</TD></TR><TR><TD style="text-align:center"> 31
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-1-2009" title="Team:Heidelberg/Notebook modeling">1</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-2-2009" title="Team:Heidelberg/Notebook modeling">2</A></TD></TR><TR><TD style="text-align:center"> 32
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-3-2009" title="Team:Heidelberg/Notebook modeling">3</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-4-2009" title="Team:Heidelberg/Notebook modeling">4</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-5-2009" title="Team:Heidelberg/Notebook modeling">5</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-6-2009" title="Team:Heidelberg/Notebook modeling">6</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-7-2009" title="Team:Heidelberg/Notebook modeling">7</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-8-2009" title="Team:Heidelberg/Notebook modeling">8</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-9-2009" title="Team:Heidelberg/Notebook modeling">9</A></TD></TR><TR><TD style="text-align:center"> 33
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-10-2009" title="Team:Heidelberg/Notebook modeling">10</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-11-2009" title="Team:Heidelberg/Notebook modeling">11</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-12-2009" title="Team:Heidelberg/Notebook modeling">12</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-13-2009" title="Team:Heidelberg/Notebook modeling">13</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-14-2009" title="Team:Heidelberg/Notebook modeling">14</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-15-2009" title="Team:Heidelberg/Notebook modeling">15</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-16-2009" title="Team:Heidelberg/Notebook modeling">16</A></TD></TR><TR><TD style="text-align:center"> 34
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-17-2009" title="Team:Heidelberg/Notebook modeling">17</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-18-2009" title="Team:Heidelberg/Notebook modeling">18</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-19-2009" title="Team:Heidelberg/Notebook modeling">19</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-20-2009" title="Team:Heidelberg/Notebook modeling">20</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-21-2009" title="Team:Heidelberg/Notebook modeling">21</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-22-2009" title="Team:Heidelberg/Notebook modeling">22</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-23-2009" title="Team:Heidelberg/Notebook modeling">23</A></TD></TR><TR><TD style="text-align:center"> 35
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-24-2009" title="Team:Heidelberg/Notebook modeling">24</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-25-2009" title="Team:Heidelberg/Notebook modeling">25</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-26-2009" title="Team:Heidelberg/Notebook modeling">26</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-27-2009" title="Team:Heidelberg/Notebook modeling">27</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-28-2009" title="Team:Heidelberg/Notebook modeling">28</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-29-2009" title="Team:Heidelberg/Notebook modeling">29</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-30-2009" title="Team:Heidelberg/Notebook modeling">30</A></TD></TR><TR><TD style="text-align:center"> 36
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#8-31-2009" title="Team:Heidelberg/Notebook modeling">31</A></TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD></TR></TBODY></TABLE><P>[<A href="https://2009.igem.org/Team:Heidelberg/Notebook_modeling" class="external text" rel="nofollow">TOP</A>]
</P><H3><SPAN class="mw-headline" id="8-3-2009"> 8-3-2009 </SPAN></H3><UL><LI> First contact with MySQL
</LI><LI> Start making an overview of other team's projects
</LI><LI> Configuring our Virtual Server
</LI></UL><H3><SPAN class="mw-headline" id="8-4-2009"> 8-4-2009 </SPAN></H3><UL><LI> Official Team Meeting (LINK) @ BQ seminar room 43: preparaing presentation &amp; writing meeting report
</LI><LI> Start installing developing environment on our internal server
</LI><LI> GNOME
</LI><LI> Mediawiki
</LI></UL><H3><SPAN class="mw-headline" id="8-5-2009"> 8-5-2009 </SPAN></H3><UL><LI> Meeting with Tobias Bauer &amp; Anna-Lena Kranz (Theoretical Bioinformatics, DKFZ) @ TP3, DKFZ
</LI><LI> Integrating ideas of PromoterSweep, Transfac as well as DoOP/CisRED
</LI><LI> select &quot;interesting&quot; TFs (e.g. HIF, NFkB, c-myc, p53) for Wetlab
</LI><LI> select &quot;interesting&quot; pathways (e.g. cell cycle, inflammation, metabolism etc)
</LI><LI> future experimental validation: ChIP-on-Chip
</LI><LI> for this we need a TFBS-free sequence
</LI></UL><LI> idea: plot histogram of TFBS relative to TSS
</LI><LI> problem: choice of sequence: upstream only? inculde downstream?
</LI><LI> new programming language: <B>R</B> and <B>perl</B></LI><LI> next meeting: Friday after team meeting
</LI><UL><LI> Meeting with Karl-Heinz Glatting (HUSAR, DKFZ) @ TP3, DKFZ
</LI><LI> An introduction into PromoterSweep
</LI><LI> Structure and analysis principles of PromoterSweep
</LI><LI> Output is stored in an XML file. This means we have to parse the xml code.
</LI><LI> Oliver Pelz will give help for us in programming
</LI></UL><UL><LI> Protocol of the meeting can be downloaded <A href="https://www.bioquant.uni-heidelberg.de/fileadmin/igem/wiki_docs/Notebook-Modelling/husar_treffen.pdf|" class="external text" rel="nofollow">from here</A>.
</LI></UL><UL><LI> Start working with MySQL
</LI><LI> request UNIX/HUSAR/HPC access at DKFZ (Nao)
</LI><LI> first contact with several databases: EmsEMBL, Compara, cisRED, DoOP, TiProD, contra
</LI></UL><H3><SPAN class="mw-headline" id="8-6-2009"> 8-6-2009 </SPAN></H3><UL><LI> Meeting with Oliver Pelz
</LI><LI> defining workflow with PromoterSweep, Matrix Profile Search and introduction into different Motif Discovery Algorithms
</LI></UL><UL><LI> installation of NX server for access onto internal server from Windows
</LI><LI> configure developing environment (printing from Linux, configure Mediawiki)
</LI><LI> defining basic concept of database construction
</LI><LI> we select annotated promoter sequences in DoOP
</LI><LI> we make a selection of pathway of interest using KEGG
</LI><LI> narrow down number of target promoter sequences &lt;10000.
</LI></UL><H3><SPAN class="mw-headline" id="8-7-2009"> 8-7-2009 </SPAN></H3><UL><LI> Official Team Meeting on Scheduling
</LI><LI> Meeting with Anna-Lena and Tobias
</LI><LI> Introduction into R
</LI><LI> Tobias will give us access to their computing cluster (Group Roland Eils)
</LI><LI> Promoter Selection: DoOP, EnsEMBL, or UCSC?
</LI></UL><UL><LI> HUSAR account arrived
</LI><LI> installation of R, R editor and perl editor
</LI><LI> further configuration of our internal server / mediawiki
</LI><LI> writing first perl program - &quot;Hi there&quot;
</LI></UL><P>[<A href="https://2009.igem.org/Team:Heidelberg/Notebook_modeling" class="external text" rel="nofollow">TOP</A>]
</P><H3><SPAN class="mw-headline" id="8-10-2009"> 8-10-2009 </SPAN></H3><UL><LI> first contact with R and perl
</LI><LI> playing around with R and perl
</LI><LI> playing around with R library: Biobase
</LI><LI> check working on DKFZ cluster
</LI></UL><H3><SPAN class="mw-headline" id="8-11-2009"> 8-11-2009 </SPAN></H3><UL><LI> defining programming languages: perl, R, MySQL
</LI><LI> retrieving first Promotersweep output files
</LI></UL><UL><LI> Meeting with Marti
</LI><LI> ideas for modeling
</LI><LI> we will have at least three colors which overlap in their spectra.
</LI><LI> a very nice approach will be Fuzzy Logic Modeling.
</LI><LI><B>idea 1</B>: <I>error checking</I> of affinity: compare expectation to experimental results and figure out where the error is hiding
</LI><LI><B>idea 2</B>: <I>create&amp;visualize fancy and fuzzy data from </I>in silico<I> simulation</I></LI></UL><LI> combine: promoter, output and graphic representation
</LI><LI> next meeting with Marti: end of next week.
</LI><UL><LI> extract NCBI Entrez Gene IDs with R and perl
</LI><LI> MAC adresses registered for bioquant network
</LI></UL><H3><SPAN class="mw-headline" id="8-12-2009"> 8-12-2009 </SPAN></H3><UL><LI> configure perl working environment
</LI><LI> study structure of DoOP database
</LI><LI> download DoOP and load DoOP database into MySQL
</LI></UL><H3><SPAN class="mw-headline" id="8-13-2009"> 8-13-2009 </SPAN></H3><UL><LI> trying out some DoOP queries
</LI><LI> download fasta sequences from UCSC gene browser
</LI><LI> mapping of NCBI Entrez Gene IDs with RefSeq IDs
</LI><LI> configure perl working environment on Windows XP
</LI><LI> contact Endre Sebestyen concerning the perl module Bio-DoOP-DoOP
</LI></UL><H3><SPAN class="mw-headline" id="8-14-2009"> 8-14-2009 </SPAN></H3><UL><LI> parse UCSC fasta sequences according to our selection
</LI><LI> write parsed sequences into multifasta format
</LI><LI> start PromoterSweep Analysis over Weekend
</LI></UL><P>[<A href="https://2009.igem.org/Team:Heidelberg/Notebook_modeling" class="external text" rel="nofollow">TOP</A>]
</P><H3><SPAN class="mw-headline" id="8-18-2009"> 8-18-2009 </SPAN></H3><P>Tim, Stephen, ab hier müsst ihr eure Sachen selber eintragen!
</P><UL><LI> study outputfile of PromoterSweep. check out general structure and pick up useful information.
</LI><LI> result is grouped in: <I>General Info</I>, <I>Best Genomic Mapping</I>, <I> Promoter DB Search Result</I>, <I>Graphical Overview</I>, <I>Combined Binding Sites</I>, <I>TSS and Exon Info</I>, <I>Profile Matrices</I> and <I>Generated Output Files</I>.
</LI><LI> upon selection, sections of interest will be collected and made ready for entry into MySQL DB
</LI><LI> discuss table structure of our database
</LI></UL><UL><LI> How should our database be called? - Brainstorming -
</LI><LI> SHOULD contain: iGEM, Transcription Factor, Binding Site, Promoter, synthetic biology, Heidelberg
</LI><LI> MAY contain: position, heartbeat, prediction, assembly, eukaryotes
</LI><LI> and still more keywords to come
</LI></UL><LI>establishing local@host access to mysql
</LI><H3><SPAN class="mw-headline" id="8-19-2009"> 8-19-2009 </SPAN></H3><UL><LI> parse Promotersweep xml file into tab-separated text file
</LI><LI> the text file should contain: RefSeq ID, TF name, TFBS position, TF motif sequence, TFBS Quality, TSS, Entrez ID, EnsEMBL ID, further gene description.
</LI><LI> this provided us with several programming problems concerning working with multiple arrays, hashes and their combinations (arrays of hashes, hashes of hashes, etc.) thus
</LI></UL><LI> studying structure and basic concepts of hash &amp; key
</LI><LI>including parsed data into mysql database
</LI><H3><SPAN class="mw-headline" id="8-20-2009"> 8-20-2009 </SPAN></H3><UL><LI> pre-decision for our table-structure
</LI><LI> Table: Main_Info
</LI><LI> RefSeq ID, TF, TF motif start &amp; end position, TFBS motif score, TFBS quality, TSS database info
</LI></UL><LI> Table: Gene_Info
</LI><LI> Ensembl_ID, Gene Symbol, Gene Description.
</LI><LI> we go for the RefSeq ID to be the <B>key</B> connecting these two tables.
</LI><H3><SPAN class="mw-headline" id="8-21-2009"> 8-21-2009 </SPAN></H3><UL><LI> update script for parsing the Promotersweep output files due to unexpected errors
</LI><LI> we forgot to include &quot;weak&quot; as a category for the TFBS quality - added!
</LI><LI> PromoterSweep result contains information about TSS derived from different promoter databases. On which should we rely, if they differ from each other?</LI><LI> We set our highest priority to DoOP database since they show a good accordance within the RefseqID results when compared to other databases (e.g. DBTSS).
</LI></UL><UL><LI> order <A href="http://www.mathworks.com/|" class="external text" rel="nofollow">Matlab</A> iGEM licence
</LI></UL><UL><LI> search for a tool to use MySQL in R programming environment
</LI><LI> wiki: write an short article about the German Cancer Research Center (<A href="/Team:Heidelberg/Team_Scientific_Environment" title="Team:Heidelberg/Team Scientific Environment">DKFZ</A>)
</LI></UL><UL><LI> Meeting with Anna-Lena: once we established our database... then
</LI><LI> two strategies:
</LI><LI> manually select interesting transcription factors and analyse them using database queries
</LI><LI> plot histograms of TFBS occurance within the target promoter sequence (TSS - 1000bp upstream) for each TF and make systematic analysis
</LI></UL><LI> we go for both!
</LI><LI> idea for the future: we can analyze combinatorial appearance of distinct TF pairs
</LI><UL><LI> We have a name for our database - we call it -
</LI></UL><P>
- wait for it -
</P><P><B>HEARTBEAT</B> database (<I><B>He</B>idelberg <B>Ar</B>tificial <B>T</B>ranscription Factor <B>B</B>inding Site <B>E</B>ngineering and <B>A</B>ssembly <B>T</B>ool)</I>
[<A href="https://2009.igem.org/Team:Heidelberg/Notebook_modeling" class="external text" rel="nofollow">TOP</A>]
</P><H3><SPAN class="mw-headline" id="8-24-2009"> 8-24-2009 </SPAN></H3><UL><LI> Meeting with Marti: defining output modeling strategies
</LI><LI> &quot;exclusive promoters&quot;
</LI><LI> a model for predicting the behaviour of activation of one, two, three... promoters at the same time.
</LI><LI> the potential of this model lies in the possibility to model single as well as many pathways in combination and even check for synergistic effects
</LI><LI> modeling logic: quantitative <B>ODE</B> VS. quantitative &amp; qualitative <B>fuzzy logic</B></LI></UL><LI> &quot;error checking&quot;
</LI><LI> what to capture/measure: affinity of transcription factor binding to DNA
</LI><LI> calculate score / reliabilty
</LI><LI> phenotypic measurement
</LI><LI> if we have time in the end: model/experiment optimization by wetlab-drylab-rounds (GRAFIK) 
</LI><LI> if we do not have much time: figure out where is catch
</LI><LI> modeling layers &amp; final visualization
</LI><LI> (i) capture affinity - (ii) model gene expression - (iii) pathway activity - (iv) fancy visualization (Mathworks Simulink?)
</LI><LI> plot: time course, dynamic affinity
</LI><LI> keep in mind the possible high amount of False Positives using promoter search/analysis
</LI><H3><SPAN class="mw-headline" id="8-25-2009"> 8-25-2009 </SPAN></H3><UL><LI> official Team Meeting also with Mr. Kai Ludwig (LANGE + PFLANZ) as guest for Logo / Title Claim discussion
</LI></UL><UL><LI> so far we have 1753 promoter sequences analyzed by PromoterSweep!
</LI></UL><UL><LI> Meeting with Daniela (Nao): <I>Cell Profiler</I> for capturing biological images &amp; data analysis based on MATLAB
</LI></UL><UL><LI> working with R module <B>RMySQL</B> for using the pipeline between R and MySQL
</LI><LI> create a list of useful RMySQL commands
</LI></UL><H3><SPAN class="mw-headline" id="8-26-2009"> 8-26-2009 </SPAN></H3><UL><LI><B>Workflow for plotting histogram</B> - workflow (SOURCE CODE/S?)
</LI><LI> make MySQL query using R
</LI><LI> make list of TFs, avoid duplicates using perl
</LI><LI> pick up each TF (perl/R) and plot histogram (R)
</LI></UL><UL><LI> create MySQL command list including combinatorial queries
</LI></UL><H3><SPAN class="mw-headline" id="8-27-2009"> 8-27-2009 </SPAN></H3><UL><LI> check HEARTBEAT DB for duplicate entries
</LI><LI> how should we plot the histogram?
</LI><LI> (a) histogram - how &quot;wide&quot; should be each bin? 100bp? 50bp? 20bp?
</LI><LI> (b) plot probability density
</LI></UL><LI> study Transfac PWM (position weight matrices) for
</LI><LI> difference in consensus sequences (also ask Anna-Lena)
</LI><LI> different PWM types (vertebrates, plant, insect, fungi, bacteria, nematodes...)
</LI><LI> positive control: when histograms are generated and plotted, check distribution of Sp1
</LI><UL><LI> so far we have 3640 promoter sequences &quot;sweeped&quot;!
</LI></UL><UL><LI>access from R to mysql at the local@host server established
</LI></UL><H3><SPAN class="mw-headline" id="8-28-2009"> 8-28-2009 </SPAN></H3><UL><LI> dealing with perl - introduce transition of variables between perl and R  
</LI></UL><P>[<A href="https://2009.igem.org/Team:Heidelberg/Notebook_modeling" class="external text" rel="nofollow">TOP</A>]
</P><H3><SPAN class="mw-headline" id="8-31-2009"> 8-31-2009 </SPAN></H3><P>[<A href="https://2009.igem.org/Team:Heidelberg/Notebook_modeling" class="external text" rel="nofollow">TOP</A>]
</P><H2><SPAN class="mw-headline" id="September"> September </SPAN></H2><TABLE class="wikitable centered" border="2" rules="rows" width="650px" style="border-color:white;"><TBODY><TR><TH> Week </TH><TH colspan="7">Days
</TH></TR><TR><TD style="text-align:center"> Mon
</TD><TD style="text-align:center"> Tue
</TD><TD style="text-align:center"> Wed
</TD><TD style="text-align:center"> Thu
</TD><TD style="text-align:center"> Fri
</TD><TD style="text-align:center"> Sat
</TD><TD style="text-align:center"> Sun
</TD></TR><TR><TD style="text-align:center"> 36
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-1-2009" title="Team:Heidelberg/Notebook modeling">1</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-2-2009" title="Team:Heidelberg/Notebook modeling">2</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-3-2009" title="Team:Heidelberg/Notebook modeling">3</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-4-2009" title="Team:Heidelberg/Notebook modeling">4</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-5-2009" title="Team:Heidelberg/Notebook modeling">5</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-6-2009" title="Team:Heidelberg/Notebook modeling">6</A></TD></TR><TR><TD style="text-align:center"> 37
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-7-2009" title="Team:Heidelberg/Notebook modeling">7</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-8-2009" title="Team:Heidelberg/Notebook modeling">8</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-9-2009" title="Team:Heidelberg/Notebook modeling">9</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-10-2009" title="Team:Heidelberg/Notebook modeling">10</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-11-2009" title="Team:Heidelberg/Notebook modeling">11</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-12-2009" title="Team:Heidelberg/Notebook modeling">12</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-13-2009" title="Team:Heidelberg/Notebook modeling">13</A></TD></TR><TR><TD style="text-align:center"> 38
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-14-2009" title="Team:Heidelberg/Notebook modeling">14</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-15-2009" title="Team:Heidelberg/Notebook modeling">15</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-16-2009" title="Team:Heidelberg/Notebook modeling">16</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-17-2009" title="Team:Heidelberg/Notebook modeling">17</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-18-2009" title="Team:Heidelberg/Notebook modeling">18</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-19-2009" title="Team:Heidelberg/Notebook modeling">19</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-20-2009" title="Team:Heidelberg/Notebook modeling">20</A></TD></TR><TR><TD style="text-align:center"> 39
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-21-2009" title="Team:Heidelberg/Notebook modeling">21</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-22-2009" title="Team:Heidelberg/Notebook modeling">22</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-23-2009" title="Team:Heidelberg/Notebook modeling">23</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-24-2009" title="Team:Heidelberg/Notebook modeling">24</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-25-2009" title="Team:Heidelberg/Notebook modeling">25</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-26-2009" title="Team:Heidelberg/Notebook modeling">26</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-27-2009" title="Team:Heidelberg/Notebook modeling">27</A></TD></TR><TR><TD style="text-align:center"> 40
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-28-2009" title="Team:Heidelberg/Notebook modeling">28</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-29-2009" title="Team:Heidelberg/Notebook modeling">29</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#9-30-2009" title="Team:Heidelberg/Notebook modeling">30</A></TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD></TR></TBODY></TABLE><P>[<A href="https://2009.igem.org/Team:Heidelberg/Notebook_modeling" class="external text" rel="nofollow">TOP</A>]
</P><H3><SPAN class="mw-headline" id="9-1-2009"> 9-1-2009 </SPAN></H3><UL><LI> derive transcription factor data using R and MySQL
</LI><LI> plot HEARTBEAT TF hit distribution as histograms &amp; density functions for different PWM subsets (all, vertebrates only, single matrices and joined TFs)
</LI><LI>further completion of the database
</LI></UL><H3><SPAN class="mw-headline" id="9-2-2009"> 9-2-2009 </SPAN></H3><UL><LI> discussion on how to make statistical studies on our gained distributions
</LI><LI> ideas: define maximum and variance -&gt; Nao
</LI></UL><LI> look for motif sequences -&gt; Tim
</LI><UL><LI> we have 4476 sequences analysed by Promotersweep so far!
</LI><LI> but we are expecting 4700 sequences - check missing ones!
</LI></UL><H3><SPAN class="mw-headline" id="9-3-2009"> 9-3-2009 </SPAN></H3><UL><LI> internal team meeting: Tim, Lars, Stephen, Nao
</LI><LI> select especially interesting TFs
</LI><LI> criteria: (a) good hits in our distributions; (b) easy experimental handling
</LI><LI> we go for <B>HIF</B>, <B>SREBP</B> and <B>VDR</B> to analyse and make synthetic promoter design
</LI></UL><LI> Transfac PWM: there are some annotaion inconveniences of some matrices
</LI><LI> which &quot;spacer&quot; sequences should we use in order to generate TFBS free sequece parts
</LI><UL><LI> rational design of synthetic promoters
</LI><LI> Tim: SREBP, Nao: VDR
</LI><LI> both go for a total number of 10 sequences
</LI><LI> strategies:
</LI><LI> single TFs: search for density maxima
</LI><LI> check combinatorial appearance and design promoter sequences with multiple binding TFs
</LI></UL><LI> use spacer sequences generated by Lars and check for TFBS using Transfac
</LI><LI> sequence length: max. 1000bp
</LI><UL><LI> back-up idea: if synthesis does not work for a long (~1000bp) sequence then try to work out a protocol for a two-step promoter synthesis combining one empty (TFBS free) sequence with another which consists of many TF and activator binding sites.
</LI></UL><H3><SPAN class="mw-headline" id="9-4-2009"> 9-4-2009 </SPAN></H3><UL><LI> work with Transfac PWM: structure, description, and using consensus sequence
</LI><LI> write script to get the ID's and frequencies for all co-occuring TFBS of VDR and SREBP 
</LI><LI> write script for generating consensus sequence based on Transfac PWM and replacing ambiguity code with A, C, G or T <PRE>Getconsensus.pl, MakeConsensus.pl</PRE></LI></UL><UL><LI> Wiki Meeting (Nao)
</LI><LI> Logo choice &amp; modification
</LI><LI> choose header pics
</LI><LI> navigation layout
</LI><LI> develop a catchy, cool homepage
</LI></UL><H3><SPAN class="mw-headline" id="9-5-2009"> 9-5-2009 </SPAN></H3><UL><LI> Meeting with Tim, design synthetic promoter sequences 
</LI><LI> check spacer sequence (200bp) for TFBS: one TFBS found; remove it by cutting and shortening the sequence to 190bp)
</LI><LI> Kid3 is a repressor!
</LI></UL><H3><SPAN class="mw-headline" id="9-6-2009"> 9-6-2009 </SPAN></H3><UL><LI> design more synthetic promoter sequences by manual iteration process which consists of (i) TFBS check and (ii) TFBS removal &amp; filling up random sequence
</LI></UL><UL><LI> aim: creation of an automatic designing tool for synthetic promoters which include sequence design, transfac search as well as filling the sequence up with spacer sequences.
</LI></UL><P>
[<A href="https://2009.igem.org/Team:Heidelberg/Notebook_modeling" class="external text" rel="nofollow">TOP</A>]
</P><H3><SPAN class="mw-headline" id="9-7-2009"> 9-7-2009 </SPAN></H3><UL><LI> check designed sequences for restriction sites <PRE>CheckRestrictionsites.pl</PRE></LI><LI> finish creating sequences
</LI><LI> consider CMV core promoter into the calculation of the relative position of TFBS to the TSS
</LI><LI> create sequences for negative control
</LI><LI> pure TFBS free sequence
</LI><LI> sequences with TFBS at minima of the density function
</LI></UL><LI> checking for all sequences for further binding sites with the Transfac match tool
</LI><H3><SPAN class="mw-headline" id="9-8-2009"> 9-8-2009 </SPAN></H3><UL><LI> check restriction sites for reverse complementary strand
</LI><LI> add flanking sites with restriction sites and spacer nucleotides to our designed sequences
</LI><LI> is there any possibility to automatize Transfac queries?
</LI><LI> work with combined / joined MySQL query structures
</LI><LI> or solve this process by simply writing new temporary tables?
</LI></UL><UL><LI> workflow summary (short) for manual designing of a synthetic promoter:
</LI><LI> (A) use random sequence
</LI><LI> (B) check TF-matrices
</LI><LI> (C) validate TFs (mouse? human? repressor?)
</LI><LI> (D) check Transfac and restriction sites
</LI></UL><UL><LI> Phone conference with Kai Ludwig, Logo &amp; Web Design (Nao)
</LI></UL><UL><LI> official Team Meeting
</LI></UL><UL><LI> wiki closure on Oct 21st!
</LI></UL><H3><SPAN class="mw-headline" id="9-9-2009"> 9-9-2009 </SPAN></H3><UL><LI> modify synthetic promoter sequences to be ready for ordering
</LI><LI> Sweep more promoter sequences using Promotersweep
</LI><LI> start Modeling
</LI><LI> revise and improve HEARTBEAT
</LI><LI> discuss differences between PWMs
</LI></UL><H3><SPAN class="mw-headline" id="9-10-2009"> 9-10-2009 </SPAN></H3><UL><LI> still modifying synthetic sequences to be ready for shipping
</LI><LI> we have altogether 25 designed promoter sequences!
</LI></UL><H3><SPAN class="mw-headline" id="9-11-2009"> 9-11-2009 </SPAN></H3><UL><LI> Software Meeting (Stephen, Tim, Nao)
</LI><LI> compartibility with mediawiki: HTML, perl, php, R, java?
</LI><LI> GUI design
</LI><LI> simple interface: single TF, auxiliary TFs, #TFBS, sequence length
</LI><LI> &quot;interactive&quot;: multiple TF, choosing auxiliary TFs, additional information (see <A href="/Team:Heidelberg/Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">Eukaryopedia</A>), density function plot &amp; histogram
</LI><LI> &quot;hyper-interactive&quot; step-by-step design &amp; creation
</LI></UL><UL><LI> Modeling Meeting with Marti and Anna-Lena (Tim, Nao)
</LI><LI> aim: fancy visualization to show expectation &amp; prediction providing pathway insights
</LI><LI> TODO/QUESTIONS
</LI><LI> what is the stimulus? collect possible inputs!
</LI><LI> measurable outcome: experiments &amp; pathways
</LI><LI> quality of synthetic sequence: error checking
</LI><LI> we need to define the <B>quality</B> of our sequences
</LI></UL><LI> LEVELS of modeling
</LI><LI> (1) DNA (2) expression/transcriptional activity (3) output
</LI><LI> each with corresponding measurement
</LI><UL><LI> general modeling scheme: input - &quot;What we are affecting&quot; - possible outcomes
</LI><LI> how? We use fuzzy logic
</LI></UL><P>[<A href="https://2009.igem.org/Team:Heidelberg/Notebook_modeling" class="external text" rel="nofollow">TOP</A>]
</P><H3><SPAN class="mw-headline" id="9-14-2009"> 9-14-2009 </SPAN></H3><UL><LI> collect input for inducing the system (e.g. p53: CPT, Pifithrin-alpha; NFkB: TNF-alpha etc.)
</LI><LI> phone conference with Kai Ludwig
</LI><LI> learn how to include Perl code into html code
</LI><LI> learn how to use embperl 
</LI><LI> configure apache2 server such that embperl can be interpreted
</LI><LI> try to make offline use of embperl working
</LI></UL><LI>try to find nice html editor for ubuntu - (seamonkey, Amaya)
</LI><H3><SPAN class="mw-headline" id="9-15-2009"> 9-15-2009 </SPAN></H3><UL><LI> create network picture for meeting tomorrow
</LI><LI> Logo discussion
</LI><LI> Read paper: <I>Fuzzy Logic Modeling of Signaling Networks</I> (Aldridge 2009)
</LI><LI> learn data management of virtual server
</LI><LI> get an overview about the apache2 file and security system
</LI></UL><H3><SPAN class="mw-headline" id="9-16-2009"> 9-16-2009 </SPAN></H3><UL><LI> Modeling Meeting with Marti (Douaa, Tim, Nao)
</LI><LI> update on available drugs/sequences
</LI><LI> decide what to model: (A) error checking, and (B) differential expression?
</LI><LI> use natural promoters to build up model for prediction of activity of synthetic promoters
</LI><LI> Discussion of TF score
</LI><LI> Transfac sequence alignment score
</LI><LI> promotersweep binding site quality
</LI><LI> relative position to TSS: How?
</LI><LI> (A) peak width &amp; amplitude, (B) distance to maximal peak &amp; position, (C) number of PEAK, (D) &quot;sliding window&quot; and calculate area under curve, (E) #TFBS (also for comparison of different synthetic promoters)
</LI></UL><LI> biophysical affinity using TRAP
</LI><LI> first model: build up either on CMV or on JeT
</LI><LI> potential: integrate many stimuli -&gt; find out crosstalks of pathways?
</LI><UL><LI> TODO (meeting)
</LI><LI> collect data
</LI><LI> define WHAT we want to model
</LI><LI> summarize available sequences
</LI><LI> try to formulate IF ... THEN &quot;sentences&quot;
</LI><LI> check MATLAB &amp; MATLAB Fuzzy Logic Toolbox availability
</LI></UL><H3><SPAN class="mw-headline" id="9-17-2009"> 9-17-2009 </SPAN></H3><UL><LI> internal Team Meeting
</LI><LI> find error.log files on the server and learn how to use it
</LI></UL><H3><SPAN class="mw-headline" id="9-18-2009"> 9-18-2009 </SPAN></H3><P>[<A href="https://2009.igem.org/Team:Heidelberg/Notebook_modeling" class="external text" rel="nofollow">TOP</A>]
</P><H3><SPAN class="mw-headline" id="9-20-2009"> 9-20-2009 </SPAN></H3><UL><LI> learn how to use tag language of embperl
</LI><LI> learn how to write loops with embperl
</LI><LI> access of input variables in embperl -- using the %fdat hash
</LI></UL><H3><SPAN class="mw-headline" id="9-21-2009"> 9-21-2009 </SPAN></H3><UL><LI> struggling with how to use R from embperl
</LI></UL><H3><SPAN class="mw-headline" id="9-22-2009"> 9-22-2009 </SPAN></H3><UL><LI> Wiki Meeting (Dani, Cori, Nao)
</LI><LI> install image processing tool
</LI><LI> design wiki, brainstorming for possible navigation bars
</LI></UL><LI> Wiki Phone Meeting with Kai Ludwig (Nao)
</LI><LI> design header &amp;presentation-master as well as team shirts
</LI><UL><LI> Seminar: <B>Martijn Luijsterburg</B> (Karolinska Institute) - <I>Heterochromatin Protein 1 is involved in the DNA damage response</I>. Host: Thomas Höfer, Bioquant
</LI></UL><H3><SPAN class="mw-headline" id="9-23-2009"> 9-23-2009 </SPAN></H3><UL><LI> Modeling Meeting with Marti, Anna-Lena (Tim, Nao)
</LI><LI> contact database group (TP3)
</LI><LI> statistics: characterizing peaks
</LI><LI> we go for area under the curve and affinity. optionally we can choose Transfac sequence score and peak height &amp; width
</LI></UL><LI> strategy to convince the wetlab people from the importance of modeling during the meeting on upcoming friday.
</LI><LI> MATLAB license?
</LI><LI> logical gates: try to start creating model topology after Friday
</LI><UL><LI> Presentation: <B>Marti Bernado Faura</B> (Bioquant, University of Heidelberg): <I>Data-driven Fuzzy Logic modeling of Programmed Cell Death</I></LI><LI> intro into fuzzy logic
</LI><LI> system development &amp; work flow of fuzzy logic
</LI><LI> fuzzy inference &amp; model prediction
</LI><LI> model types: MISO / MIMO
</LI></UL><UL><LI> Wrap-up meeting: Team HEARTBEAT (Tim, Nao)
</LI><LI> split up computational work into three tracks: HEARTBEAT DB, HEARTBEAT GUI and modeling
</LI><LI> database: documentation (until Oct 18), peak characterization, calculate absolute density function
</LI><LI> GUI: based on <I>embperl</I>, design according to our new wiki
</LI><LI> modeling: MATLAB license, collect sequences &amp; input data, develop network model, include pathways
</LI></UL><UL><LI> literature work
</LI></UL><H3><SPAN class="mw-headline" id="9-24-2009"> 9-24-2009 </SPAN></H3><UL><LI> prepare slides for meeting tomorrow
</LI><LI> pathway search: TNF-alpha/NFkB, VDR, SREBP and crosstalks. NFkB has a lot of pathway crosstalks, while SREBP and VDR show a interesting connection. Upon induction, SREBP activates VDR.
</LI></UL><H3><SPAN class="mw-headline" id="9-25-2009"> 9-25-2009 </SPAN></H3><UL><LI> Team Meeting (Wetlab, Nao)
</LI><LI> short progress report of all of us
</LI><LI> modeling: discussing scheme, modeling elements and strategies
</LI></UL><P>
[<A href="https://2009.igem.org/Team:Heidelberg/Notebook_modeling" class="external text" rel="nofollow">TOP</A>]
</P><H3><SPAN class="mw-headline" id="9-28-2009"> 9-28-2009 </SPAN></H3><UL><LI> Wiki Phone Meeting with Kai Ludwig (Nao)
</LI></UL><H3><SPAN class="mw-headline" id="9-29-2009"> 9-29-2009 </SPAN></H3><UL><LI> designed synthetic promoters (HB_0001 - HB_0025) will be joined to CMV core promoter since JeT core promoter contains a Sp1 site in it. All other sequences (random synthesized, e.g.) are coupled with JeT core promoter.
</LI><LI> literature studies on combinatorial <I>cis</I>-regulation as well as on modelig of the lambda-switch
</LI><LI> prepare slides for the next modeling meeting
</LI></UL><H3><SPAN class="mw-headline" id="9-30-2009"> 9-30-2009 </SPAN></H3><UL><LI> Wiki Meeting (Dani, Nao)
</LI><LI> MATLAB license order (Jens)
</LI><LI> postpone Yara meeting (Wetlab, Tim)
</LI></UL><UL><LI> got sequences from Lars
</LI><LI> got qRT-PCR setup from Chenchen
</LI></UL><UL><LI> Modeling Meeting with Marti &amp; Anna-Lena (Tim, Nao)
</LI><LI> still need to collect FACS and microscopy results
</LI><LI> discuss our network prediction model using TNF-alpha as aｎ example
</LI><LI> maybe we can use the lambda switch paper as a good starting point for our modeling
</LI></UL><P>[<A href="https://2009.igem.org/Team:Heidelberg/Notebook_modeling" class="external text" rel="nofollow">TOP</A>]
</P><H2><SPAN class="mw-headline" id="October"> October </SPAN></H2><TABLE class="wikitable centered" border="2" rules="rows" width="650px" style="border-color:white;"><TBODY><TR><TH> Week </TH><TH colspan="7">Days
</TH></TR><TR><TD style="text-align:center"> Mon
</TD><TD style="text-align:center"> Tue
</TD><TD style="text-align:center"> Wed
</TD><TD style="text-align:center"> Thu
</TD><TD style="text-align:center"> Fri
</TD><TD style="text-align:center"> Sat
</TD><TD style="text-align:center"> Sun
</TD></TR><TR><TD style="text-align:center"> 40
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-1-2009" title="Team:Heidelberg/Notebook modeling">1</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-2-2009" title="Team:Heidelberg/Notebook modeling">2</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-3-2009" title="Team:Heidelberg/Notebook modeling">3</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-4-2009" title="Team:Heidelberg/Notebook modeling">4</A></TD></TR><TR><TD style="text-align:center"> 41
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-5-2009" title="Team:Heidelberg/Notebook modeling">5</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-6-2009" title="Team:Heidelberg/Notebook modeling">6</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-7-2009" title="Team:Heidelberg/Notebook modeling">7</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-8-2009" title="Team:Heidelberg/Notebook modeling">8</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-9-2009" title="Team:Heidelberg/Notebook modeling">9</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-10-2009" title="Team:Heidelberg/Notebook modeling">10</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-11-2009" title="Team:Heidelberg/Notebook modeling">11</A></TD></TR><TR><TD style="text-align:center"> 42
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-12-2009" title="Team:Heidelberg/Notebook modeling">12</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-13-2009" title="Team:Heidelberg/Notebook modeling">13</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-14-2009" title="Team:Heidelberg/Notebook modeling">14</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-15-2009" title="Team:Heidelberg/Notebook modeling">15</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-16-2009" title="Team:Heidelberg/Notebook modeling">16</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-17-2009" title="Team:Heidelberg/Notebook modeling">17</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-18-2009" title="Team:Heidelberg/Notebook modeling">18</A></TD></TR><TR><TD style="text-align:center"> 43
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-19-2009" title="Team:Heidelberg/Notebook modeling">19</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-20-2009" title="Team:Heidelberg/Notebook modeling">20</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-21-2009" title="Team:Heidelberg/Notebook modeling">21</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-22-2009" title="Team:Heidelberg/Notebook modeling">22</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-23-2009" title="Team:Heidelberg/Notebook modeling">23</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-24-2009" title="Team:Heidelberg/Notebook modeling">24</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-25-2009" title="Team:Heidelberg/Notebook modeling">25</A></TD></TR><TR><TD style="text-align:center"> 44
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-26-2009" title="Team:Heidelberg/Notebook modeling">26</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-27-2009" title="Team:Heidelberg/Notebook modeling">27</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-28-2009" title="Team:Heidelberg/Notebook modeling">28</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-29-2009" title="Team:Heidelberg/Notebook modeling">29</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-30-2009" title="Team:Heidelberg/Notebook modeling">30</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_modeling#10-31-2009" title="Team:Heidelberg/Notebook modeling">31</A></TD><TD style="text-align:center"> -
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="10-1-2009"> 10-1-2009 </SPAN></H3><UL><LI> Wiki &amp; Presentation Meeting with Dani (Nao)
</LI></UL><H3><SPAN class="mw-headline" id="10-2-2009"> 10-2-2009 </SPAN></H3><UL><LI> some wiki work
</LI></UL><P>[<A href="https://2009.igem.org/Team:Heidelberg/Notebook_modeling" class="external text" rel="nofollow">TOP</A>]
</P><H3><SPAN class="mw-headline" id="10-5-2009"> 10-5-2009 </SPAN></H3><H3><SPAN class="mw-headline" id="10-6-2009"> 10-6-2009 </SPAN></H3><UL><LI> Internal Team Meeting
</LI><LI> check out number and measurement plans of randomly assembled synthetic promoters (5x NFkB, 5x p53, 2x pPARg, 2x SREBP)
</LI></UL><UL><LI> Wiki Meeting (Corinna, Daniela, Nao)
</LI><LI> discuss design of the top page and possible features
</LI><LI> try out CSS design
</LI></UL><H3><SPAN class="mw-headline" id="10-7-2009"> 10-7-2009 </SPAN></H3><UL><LI> Wiki Design (Nao)
</LI><LI> Wiki Phone Meeting with Kai Ludwig (Nao)
</LI></UL><UL><LI> MATLAB has arrived!
</LI><LI> literature work
</LI></UL><UL><LI> Wetlab Meeting: progress report on measurement of random assembled synthetic promoters
</LI><LI> make thoughts about the whole storyboard of our presentation at the jamboree
</LI></UL><H3><SPAN class="mw-headline" id="10-8-2009"> 10-8-2009 </SPAN></H3><UL><LI> Short Meeting with Roland
</LI><LI> image processing work for wiki
</LI></UL><H3><SPAN class="mw-headline" id="10-9-2009"> 10-9-2009 </SPAN></H3><P>[<A href="https://2009.igem.org/Team:Heidelberg/Notebook_modeling" class="external text" rel="nofollow">TOP</A>]
</P><H3><SPAN class="mw-headline" id="10-12-2009"> 10-12-2009 </SPAN></H3><H3><SPAN class="mw-headline" id="10-13-2009"> 10-13-2009 </SPAN></H3><UL><LI> Measurement discussion with Lars: REU/RMPU, defining equations for mammalian systems
</LI><LI> literature work on PoPS paper (Kelly JR et al.) and apply their equations
</LI></UL><UL><LI> Marti Modeling Meeting (Anna-Lena, Tim, Nao)
</LI><LI> Journal Club (Tim, Nao)
</LI><LI> summary of meeting from Team Meeting from last thursday
</LI></UL><UL><LI> Marti: start modeling using MATLAB and Fuzzy Logic Toolbox (FLT), playing around with FLT and tutorial
</LI></UL><H3><SPAN class="mw-headline" id="10-14-2009"> 10-14-2009 </SPAN></H3><P>Nao
</P><UL><LI> develop first test fuzzy inference system (FIS) for testing
</LI><LI> Marti Modeling Meeting, specify model topology
</LI><LI> collect data: FACS (Cori), Microscopy (Hannah), Sequence &amp; TECAN (Lars)
</LI><LI> start calculating position score using R
</LI><LI> translating project abstract
</LI></UL><H3><SPAN class="mw-headline" id="10-15-2009"> 10-15-2009 </SPAN></H3><P>Nao
</P><UL><LI> calculate affinity score using TRAP (Anna-Lena)
</LI><LI> collect ideas for integrating TFwise scores in order to calculate final position/affinity score for one sequence: median, mean, maximum, weighted mean?
</LI><LI> all data analysis is stored in three sheets (SequenceAnalysis, ResultSummary and CalculateTRAP)
</LI><LI> from now on we concentrate on FACS measurements because they are the most reliable ones (TECAN used only for scanning)
</LI><LI> fill up TRAP data with missing transcription factors
</LI></UL><H3><SPAN class="mw-headline" id="10-16-2009"> 10-16-2009 </SPAN></H3><P>Nao
</P><UL><LI> Anna-Lena Meeting: discuss how to integrate sequence scores
</LI><LI> get &amp; check p53, pPARg and random SREBP sequences
</LI><LI> go through FACS results
</LI><LI> add HEARTBEAT sequences for data analysis
</LI><LI> modeling documentation
</LI><LI> parsing experimental setups for modeling use
</LI><LI> Chenchen qRT-PCR results
</LI></UL><UL><LI> define possible modeling layers
</LI><LI> first layer: input
</LI><LI> drug type, pathway, drug mode of action, drug concentration, targeted cells, incubation time
</LI><LI> sequence type, position score, affinity score
</LI><LI> we choose position &amp; affinity score, sequence type and the presence of stimulation. Time as well as different concentration (unfortunately no data available) can be added in future
</LI></UL><LI> second layer: promoters
</LI><LI> 6 constitutives, 3 standards, 6 inducible available
</LI><LI> data analysis narrows this to 5 constitutives, 3 standards and 4 inducible
</LI><LI> HEARTBEAT sequences have to be measured a.s.a.p.
</LI><UL><LI> Marti Modeling Meeting
</LI><LI> try to define some fuzzy rules
</LI><LI> we assume better binding -&gt; better expression
</LI><LI> define membership functions
</LI><LI> start modeling with NFkB results
</LI></UL><P>All
</P><UL><LI> Internal Team Meeting
</LI><LI> reminder: wiki task, wiki to do
</LI></UL><LI> Official Team Meeting
</LI><H3><SPAN class="mw-headline" id="10-17-2009"> 10-17-2009 </SPAN></H3><UL><LI> final decision: we go for maximum of position and affinity score
</LI><LI> added HB sequences for data analysis table; as soon as results are there we can model designed synthetic promoters
</LI><LI> define shape of membership functions
</LI><LI> literature search for missing activity values?
</LI><LI> still TODO: check out p53 results since the p53-NFkB crosstalk is really interesting!
</LI></UL><H3><SPAN class="mw-headline" id="10-18-2009"> 10-18-2009 </SPAN></H3><P>Nao
</P><UL><LI> SREBP/VDR paper arrived
</LI><LI> finish data analysis
</LI><LI> study &amp; playing around with MATLAB FLT, programming from both FLT GUI and MATLAB command line
</LI><LI> define our work to be (i) error checking and (ii) exclusive pathway modeling
</LI><LI> high potential of this model lies in its plug'n'play structrue, with a high capacity of integrating more inputs, outputs and also the middle layer (promoter diversity)
</LI></UL><P>[<A href="https://2009.igem.org/Team:Heidelberg/Notebook_modeling" class="external text" rel="nofollow">TOP</A>]
</P><H3><SPAN class="mw-headline" id="10-19-2009"> 10-19-2009 </SPAN></H3><P>Nao
</P><UL><LI> define final network structure
</LI><LI> wiki work
</LI><LI> reading RFC documentation and correction
</LI><LI> we call this project HEARTBEAT fuzzy network (FN)
</LI><LI> HB FN documentation and first results!
</LI><LI> creating two fuzzy controllers: inducible NFkB and constitutive
</LI></UL><LI> how do we integrate the data? combine via Simulink!
</LI><H3><SPAN class="mw-headline" id="10-20-2009"> 10-20-2009 </SPAN></H3><UL><LI> Creating, developing, integrating and combining fuzzy network modeling (MATLAB, Simulink)
</LI><LI> first analysis of HB sequences
</LI><LI> HEARTBEAT FN documentation
</LI></UL><H3><SPAN class="mw-headline" id="10-21-2009"> 10-21-2009 </SPAN></H3><H3><SPAN class="mw-headline" id="10-22-2009"> 10-22-2009 </SPAN></H3><UL><LI> FROZEN WIKI!!!!
</LI></UL><P>[<A href="https://2009.igem.org/Team:Heidelberg/Notebook_modeling" class="external text" rel="nofollow">TOP</A>]
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Heidelberg/Notebook_modeling">http://2009.igem.org/Team:Heidelberg/Notebook_modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Notebook_modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Notebook_modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook_modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook_modeling&amp;oldid=163954" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>