<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Eukaryopedia"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Heidelberg/Eukaryopedia">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Eukaryopedia&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Eukaryopedia&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Eukaryopedia&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Eukaryopedia" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Eukaryopedia</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="main_wrapper"><DIV id="header"><DIV id="navigation"><DIV id="menu" style="position:static"><UL><LI id="mainHome"><A class="aMain" href="https://2009.igem.org/Team:Heidelberg">Home</A></LI><LI id="mainTeam"><A class="aTeam" href="https://2009.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Members#Students">Students</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Members#Advisors">Advisors</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Scientific_Environment">Scientific Environment</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Gallery">Gallery</A></LI></UL><LI id="mainProject"><A class="aProject" href="https://2009.igem.org/Team:Heidelberg/Project">Project</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project">Abstract</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Introduction">Introduction</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Measurement">Measurement</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Synthetic_promoters">Synthetic Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Further">Further Subprojects</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_SaO">Summary &amp; Outlook</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_References">References</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Acknowledgement">Acknowledgements</A></LI><LI id="mainParts"><A class="aParts" href="https://2009.igem.org/Team:Heidelberg/Parts">Parts</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Parts">Parts</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Parts_Characterization">Characterization</A></LI><LI id="mainModeling"><A class="aModeling" href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">What is HEARTBEAT?</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_database">HEARTBEAT Database</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_GUI">HEARTBEAT GUI</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_gui">GUI Documentation</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_network">HEARTBEAT Fuzzy Network Modeling</A></LI><LI id="mainNotebook"><A class="aNotebook" href="https://2009.igem.org/Team:Heidelberg/Notebook">Notebook</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_natural_promoters_overview">Natural Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_synthetic_promoters">Synthetic Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_promoters_cells">Cell Culture, Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_cell_line">Stable Cell Line</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_color_output">Multi-Color Output</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_modeling">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_measure">Measurements</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_MaM">Material &amp; Methods</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_meetings">Team Meetings</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Biosafety">Biosafety</A></LI><LI id="mainEucaryopedia"><A class="aEucaryopedia" href="https://2009.igem.org/Team:Heidelberg/Eukaryopedia">Eukaryopedia</A></LI><LI id="mainSponsors"><A class="aSponsors" href="https://2009.igem.org/Team:Heidelberg/Sponsors">Sponsors</A></LI></DIV></DIV></DIV></DIV><TABLE><TBODY><TR valign="top" style="margin-left: 2px;"><TD width="650px" style="padding: 0 15px 15px 20px; background-color:#ede8e2"><H1><SPAN class="mw-headline" id="Eukaryopedia">Eukaryopedia</SPAN></H1><P>As most synthetic biologists and iGEM teams work with <I>Escherichia Coli</I>, the use of other model systems can cause confusion. We hope to ease the legibility of our project descriptions by creating <I>eukaryopedia</I>, an overview about <A href="/Team:Heidelberg/Eukaryopedia#Transcription_factors" title="Team:Heidelberg/Eukaryopedia">transcription factors</A> and <A href="/Team:Heidelberg/Eukaryopedia#Cell_lines" title="Team:Heidelberg/Eukaryopedia">cell lines</A> we used in our studies, as well as <A href="/Team:Heidelberg/Eukaryopedia#RNA-processing_and_transcriptional_regulation" title="Team:Heidelberg/Eukaryopedia">general molecular biology issues</A> that affect our work. We hope it can help you find guidance in the jungle that mammalian molecular biology is at the moment.
</P><H2><SPAN class="mw-headline" id="Contents"><B>Contents</B></SPAN></H2><P><B>Cell lines</B></P><P><A href="/Team:Heidelberg/Eukaryopedia#HeLa" title="Team:Heidelberg/Eukaryopedia">HeLa</A> - <A href="/Team:Heidelberg/Eukaryopedia#MCF-7" title="Team:Heidelberg/Eukaryopedia">MCF-7</A> -  <A href="/Team:Heidelberg/Eukaryopedia#U2-OS" title="Team:Heidelberg/Eukaryopedia">U2-OS</A></P><P><B>Transcription factors</B></P><P><A href="/Team:Heidelberg/Eukaryopedia#AP-1" title="Team:Heidelberg/Eukaryopedia">AP-1</A> - <A href="/Team:Heidelberg/Eukaryopedia#AP-2" title="Team:Heidelberg/Eukaryopedia">AP-2</A> - <A href="/Team:Heidelberg/Eukaryopedia#CREB" title="Team:Heidelberg/Eukaryopedia">CREB</A> - <A href="/Team:Heidelberg/Eukaryopedia#HIF-1" title="Team:Heidelberg/Eukaryopedia">HIF-1</A> - <A href="/Team:Heidelberg/Eukaryopedia#NFAT" title="Team:Heidelberg/Eukaryopedia">NFAT</A> - <A href="/Team:Heidelberg/Eukaryopedia#NF-.CE.BAB" title="Team:Heidelberg/Eukaryopedia">NF-κB</A>- <A href="/Team:Heidelberg/Eukaryopedia#pPAR.CE.B3" title="Team:Heidelberg/Eukaryopedia">pPARγ</A> - <A href="/Team:Heidelberg/Eukaryopedia#p53" title="Team:Heidelberg/Eukaryopedia">p53</A> - <A href="/Team:Heidelberg/Eukaryopedia#RAR" title="Team:Heidelberg/Eukaryopedia">RAR</A> - <A href="/Team:Heidelberg/Eukaryopedia#Sp1" title="Team:Heidelberg/Eukaryopedia">Sp1</A> - <A href="/Team:Heidelberg/Eukaryopedia#SREBP" title="Team:Heidelberg/Eukaryopedia">SREBP</A> - <A href="/Team:Heidelberg/Eukaryopedia#Vitamin_D_receptor" title="Team:Heidelberg/Eukaryopedia">Vitamin D receptor</A> - <A href="/Team:Heidelberg/Eukaryopedia#WT1" title="Team:Heidelberg/Eukaryopedia">WT1</A> - <A href="/Team:Heidelberg/Eukaryopedia#ZF5" title="Team:Heidelberg/Eukaryopedia">ZF5</A> - <A href="/Team:Heidelberg/Eukaryopedia#Kid3" title="Team:Heidelberg/Eukaryopedia">Kid3</A></P><P><B>Proteins</B></P><P><A href="/Team:Heidelberg/Eukaryopedia#Apo_A-IV" title="Team:Heidelberg/Eukaryopedia">Apo A IV</A> - <A href="/Team:Heidelberg/Eukaryopedia#CYP1A1" title="Team:Heidelberg/Eukaryopedia">CYP1A1</A> - <A href="/Team:Heidelberg/Eukaryopedia#EGF" title="Team:Heidelberg/Eukaryopedia">EGF</A> - <A href="/Team:Heidelberg/Eukaryopedia#HMG_CoA_synthase" title="Team:Heidelberg/Eukaryopedia">HMG CoA synthase</A> - <A href="/Team:Heidelberg/Eukaryopedia#Hsp70" title="Team:Heidelberg/Eukaryopedia">Hsp70</A> - <A href="/Team:Heidelberg/Eukaryopedia#LDL_receptor" title="Team:Heidelberg/Eukaryopedia">LDL receptor</A> - <A href="/Team:Heidelberg/Eukaryopedia#PUMA" title="Team:Heidelberg/Eukaryopedia">PUMA</A> - <A href="/Team:Heidelberg/Eukaryopedia#TNF-alpha" title="Team:Heidelberg/Eukaryopedia">TNF-alpha</A></P><P><B>Molecular and Cellular Biology</B></P><P><A href="/Team:Heidelberg/Eukaryopedia#Post-transcriptional_modification_.2F_mRNA_processing_in_eukaryotes" title="Team:Heidelberg/Eukaryopedia">Post-transcriptional modification / mRNA processing in eukaryotes</A> - <A href="/Team:Heidelberg/Eukaryopedia#Regulation_of_transcription_in_eukaryotic_organisms" title="Team:Heidelberg/Eukaryopedia">Regulation of transcription in eukaryotic organisms</A></P><P><B>Drugs</B></P><P><A href="/Team:Heidelberg/Eukaryopedia#CPT" title="Team:Heidelberg/Eukaryopedia">Camptothecin</A> - <A href="/Team:Heidelberg/Eukaryopedia#Hygromycin" title="Team:Heidelberg/Eukaryopedia">Hygromycin</A> -  <A href="/Team:Heidelberg/Eukaryopedia#Zeocin" title="Team:Heidelberg/Eukaryopedia">Zeocin</A> - <A href="/Team:Heidelberg/Eukaryopedia#Neomycin" title="Team:Heidelberg/Eukaryopedia">Neomycin</A></P><P><B>Cellular components as tools</B></P><P><A href="/Team:Heidelberg/Eukaryopedia#GPI" title="Team:Heidelberg/Eukaryopedia">GPI</A> - <A href="/Team:Heidelberg/Eukaryopedia#Sar-1" title="Team:Heidelberg/Eukaryopedia">Sar-1</A> - <A href="/Team:Heidelberg/Eukaryopedia#Myrpalm" title="Team:Heidelberg/Eukaryopedia">Myrpalm</A> - <A href="/Team:Heidelberg/Eukaryopedia#NLS" title="Team:Heidelberg/Eukaryopedia">NLS</A> - <A href="/Team:Heidelberg/Eukaryopedia#GFP" title="Team:Heidelberg/Eukaryopedia">GFP</A></P><H2><SPAN class="mw-headline" id="Cell_lines"> Cell lines </SPAN></H2><H3><SPAN class="mw-headline" id="MCF-7"> MCF-7 </SPAN></H3><P>MCF-7 is a hormone-dependent, poorly invasive human breast cancer cell line <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[1]</A>. Originally, the cell line was derived from a postmenopausal woman with metastatic breast cancer at the Michigan Cancer Foundation. It was observed, however, that cell lines used in different laboratories vary greatly in their biological characteristics, so that it is suggested that they were derived from different patients <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[2]</A>. MCF-7 cells are estrogen-receptor positive and require estrogen for tumorigenesis in vivo. 17β-estradiol induces an TGFα-like activity <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[3]</A>, which promotes tumor growth and progression <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[4]</A>. Furthermore the cells express receptors for and respond to several other hormones including androgen, progesterone, glucocorticoids, insulin, epidermal growth factor, insulin-like growth factor, prolactin and thyroid hormone <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[2]</A>.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="HeLa"> HeLa </SPAN></H3><P>The cells were originally derived in 1952 from <B>He</B>nrietta <B>La</B>cks, who suffered from an adenocarcinoma of the cervix. The HeLa cells were the first human epithelial cells established in long-term culture <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[5]</A>. There are three main characteristics of the genome of HeLa by which they can be recognized: hypertriploid chromosome number (3n+), 20 clonally abnormal chromosomes and the integration of multiple copies of HPV18 (Human Papilloma Virus) at various sites <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[6]</A>. It has been shown, that the HeLa genome has been remarkably stable after years of subcultivation <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[6]</A>, but it is also possible to select strains of HeLa cells with certain properties by putting them under selection pressure <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[7]</A>.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="U2-OS"> U2-OS </SPAN></H3><P>U2-OS, formerly known as 2T cell line <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[8]</A>, were derived from a 15-year-old girl with a moderately differentiated osteogenic sarcoma of the shinbone. Cell culture of U2-OS started at the time of amputation of the left leg on September 3, 1964 <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[9]</A>. U2-OS cells express adhesion molecules such as integrins, Ig-CAMs and chemokine receptors as well as growth factors which are either constitutively expressed (such as IL-7) or inducible (such as TNF) by PMA (phorbol ester) or ionomycine. The adhesion molecules and growth factors support the growth of CD34 progenitor cells <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[10]</A>.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H2><SPAN class="mw-headline" id="Transcription_factors"> Transcription factors </SPAN></H2><H3><SPAN class="mw-headline" id="NF-.CE.BAB"> NF-κB </SPAN></H3><P>NF-κB (nuclear factor kappa-light-chain-enhancer of activated B cells) is a transcription factor (TF) which regulates many different target genes resulting in the expression of various proteins. In most cell types (with the exception of B cells and Dentritic cells) NF-κB is bound to the Inhibitor of κB (IκB) withholding NF-κB from entering the nucleus. When the cell becomes activated by an extra cellular stimluli, IκB is degraded and NF-κB can enter the nucleus. Within the nucleus NF-κB is able to enhance transcription of genes which are involved in immune response, cell proliferation or cell survival, depending on cell type and extra cellular stimuli <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[11]</A>. In many cells NF-κB regulates anti-apoptotic proteins (e.g. TRAF1/2) and therby preventing cell death. Therefore mutations of NF-κB resulting in a constitutively active form are often associated with unregulated cell proliferation and cancer <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[12]</A>. In macrophages the NF-κB signalling pathway could be activated by binding of bacterial lipopolysacchride (LPS). There NF-κB activation leads to secretion of cytokines which influence other lymphocytes.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="p53"> p53 </SPAN></H3><P>P53 is a transcription factor (TF) which is involved in several physiological processes. One major function of P53 is cell cycle regulation. P53 is often activated through DNA damage or other cellular stresses like cell cycle abnormalities, hypoxia and oxidative stress. In normal cells the P53 level is kept low by the protein HDM2, which attaches ubiqutin to P53 (acts as ubiquitin ligase). The ubiquitylation of P53 leads to its degradation by the proteasome. In response to cellular stress P53 is phosphorylated and changes its conformation in a way preventing HDM2(mdm2 is the analog protein in mouse) binding. Conformational changes also result in the exposion of the DNA binding domain. This activation of P53 leads to farreaching alteration of gene expression. The cell cycle is stopped between G1 and S phase and DNA repair systems are switched on. If the cell damage is intense P53 accumaltion can also lead to apoptosis of the cell. Because of its roles in DNA protection and cell cycle regulation P53 mutation is often correlated with cancer <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[13]</A>.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="HIF-1"> HIF-1 </SPAN></H3><P>HIF-1 (hypoxia inducible factor-1) is a transcription factor (TF) which is exclusivly active during hypoxia (low oxygen level). HIF-1 is a heterodimer consisting of a α- and a β-subunit. During normal oxygen conditions the α-subunit is hydroxylated by the HIF prolyl-hydroxylase. The hydroxylated α-subunit is a target for an ubiquitin ligase. Ubiquitylation of the α-subunit leads to its degradation by the proteasome. During hypoxia the degradation of the α-subunit does not occur, since the HIF prolyl-hydroxylase uses oxygen as a cosubstrate. The active TF enhances expression of different genes as for instance genes associated with blood vessel formation <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[67]</A>.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="pPAR.CE.B3"> pPARγ</SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><DIV style="text-align:justify;"><B>pPARγ activation</B>. During fatty diet, fatty acids are converted to prostaglandins by oxygenases, which activate pPARγ. Image published under <A href="http://en.wikipedia.org/wiki/File:PPAR-diagram.png" class="external text" rel="nofollow">GNU Free documentation license.</A></DIV></DIV></DIV></DIV> Peroxisome proliferator-activated receptorγ (PPARγ) is a transcription factor belonging to the family of nuclear receptors. PPAR γ plays an important role in glucose metabolism and fatty acid storage. PPARγ is basically activated by ligands like the prostaglandin PGJ2 and through dimerization with retinoid X receptor (RXR) <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[14]</A>. The activated heterodimer binds to the DNA consensus sequence AGGTCANAGGTCA resulting in an increased or decreased transcription of the appropriate gene. The genes activated by PPARγ initiate the uptake of fatty acids and differentiation of cells to adipocytes. Besides its function in metabolism, PPARγ was also shown to be correlated with several diseases such as cancer and diabetes. Activation of PPARγ by synthetic PPARγ ligands result in an increased glucose uptake. These syntethtic ligands are therefore promising agents in diabetes II treatment <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[15]</A>.  Another synthetic ligand of PPARγ is able to inhibit the proliferation of different cancer types <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[16]</A>.
<P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="SREBP"> SREBP </SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:272px;"><DIV class="thumbcaption"><DIV style="text-align:justify;"><B>SREBP induction.</B> Upon Sterol depletion, the interaction of SCAP and Insig in the ER membrane is inhibited, resulting in cleavage of SREBP, and migration to the nucleus. <A href="http://en.wikipedia.org/wiki/File:WikF1.png" class="external text" rel="nofollow">Image is public domain. </A></DIV></DIV></DIV></DIV> Sterol regulatory element-binding protein (SREBP) is a transcription factor involved in the regulation of sterol metabolism. In cells with high concentration of cholestrol SREBP is present in an inactive form anchored to the endoplasmatic reticulum or the nuclear envelop. If the cholesterol concentration decreases SREBP is cleaved by the proteases site-1 protease and site-2 protease resulting in a release of the aminoterminal domain of SREBP. Two additional proteins (Scap and Insig) are needed to regulate this process in a way that the cleavage occurs exclusively during lack of sterol <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[17]</A>. The aminoterminal domain of SREBP is translocated into the nucleus and binds to the DNA consensus sequence TCACNCCAC. The binding causes an up regulation of the genes needed for cholesterol synthesis.
<P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="Sp1"> Sp1 </SPAN></H3><P>Specificity protein 1 (Sp1) is a transcription factor which belongs to the zinc-finger protein family. It binds to promoter elements containing a central CpG motive with the following consensus sequence; 5'-G/TGGGCGGG/AG/AC/T-3'  <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[18]</A>.  Sp1 is involved in chromatin-remodelling processes <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[19]</A> as well as in derecruiting repressor proteins from the promoter <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[20]</A>. For these and for other reasons Sp1 is often considered as a universal transcription supporting protein.  Sp1 was shown to regulate various genes responsible for cellular processes like apoptosis, cell growth and differentiation and immune response <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[21]</A>. It interacts with several well known proteins, such as c-myc, c-Jun and Stat1 <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[21]</A>.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="AP-1"> AP-1 </SPAN></H3><P>Activating Protein1 (AP-1) is a transcription factor which is activated by several pathways and extracellular stimuli like UV radiation, growth factors and bacterial and viral infections. AP-1 is a heterodimer consisting of one member of the Fos and one member of the Jun family.  AP-1 composition and activation is mainly controlled by MAP kinase cascades by up regulating the expression of both Fos and Jun proteins. Besides the heterodimerisation process, phosphorylation of the complex is needed to achieve an efficient transcription of the target genes <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[22]</A>. Activated AP-1 binds to DNA sequences with the consensus sequence 5'-TGAG/CTCA-3' <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[23]</A>. The genes regulated by AP-1 are involved in cellular processes like apoptosis, cell differentiation, cell proliferation and oncogenic transformation <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[22]</A>.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="AP-2"> AP-2 </SPAN></H3><P>Activating protein 2 (AP-2) is a family of transcription factors being closely related (AP-2alpha, -beta and –gamma).  AP-2 proteins are activated through homo or heterodimerisation and bind to GC-rich motives in their target genes <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[24]</A>. The genes both positively and negatively regulated by AP-2 are manifold.  These genes are mainly involved in developmental processes. Mutation of AP-2-beta can lead to the Char syndrome <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[25]</A>. AP-2 transcription factors regulate also genes playing an important role in cell proliferation and apoptosis <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[24]</A>. In this context AP-2alpha is thought to act in a tumor suppressive manner in breast tissues <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[24]</A>. 
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="CREB"> CREB </SPAN></H3><P>CAMP responsive element binding protein (CREB) is a transcription factor which binds to cAMP response elements (consensus sequence 5'- TGACGTCA -3' <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[26]</A>) occurring in many promoter sequences. CREB is activated by MAP kinase cascade but also through the cAMP signalling pathway. Homodimerisation leads finally to an activated complex and binding to DNA occurs via a leucine zipper domain (s. Picture).  Many genes are regulated by CREB including the neurotrophin Brain-derived neurotrophic factor, c-fos and some neuropeptides. CREB is thought to be involved in processes like long-term memory <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[27]</A> and drug addiction <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[28]</A>.  CREB plays also an important role in cell survival <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[29]</A>.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="NF-Y"> NF-Y </SPAN></H3><P>Nuclear factor Y (NF-Y), also called core binding factor (CBF) is a transcription factor which binds to the consensus sequence CCAAT <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[31]</A> occurring in about 25% of eukaryotic genes <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[30]</A>. NF-Y is involved in the transcription regulation of several genes including HSP70, albumin, FGF-4, α-collagen, β-actin and several others <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[31]</A>. NF-Y is a heterotrimeric complex and is evolutionary extremely conserved. It was also shown that NF-Y plays a major role in cellular senescence <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[30]</A>.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="Vitamin_D_receptor"> Vitamin D receptor </SPAN></H3><P>Vitamin D receptor (VDR), also known as calcitriol receptor, is a transcription factor belonging to the family of steroid receptors and to the super family of nuclear receptors. The VDR has a very high affinity towards calcitriol (1α,25(OH)2-cholecalciferol) which is the prohormone of vitamin D3. Binding of calcitriol results in a heterodimerisation of VDR with retinoid-X receptor, the complex is transferred into the nucleus and binds to several promoters (vitamin D response elements) thereby increasing or decreasing the transcription of the appropriate genes. The genes modulated by the VDR-calcitriol complex are involved in activating the immune system, bone formation and protection of cancer <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[32]</A>. Many studies for example indicate a relationship between vitamin D signalling and reduced breast cancer occurance <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[33]</A>. Furthermore the transcription of the VDR gene itself is positively regulated by the presence of calcitriol. 
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="ZF5"> ZF5 </SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><DIV style="text-align:justify;">Zink fingers interacting with DNA. <A href="http://de.wikipedia.org/w/index.php?title=Datei:Zinc_finger_DNA_complex.png" class="external text" rel="nofollow">Image under GNU license</A></DIV></DIV></DIV></DIV><DIV style="text-align:justify;">Zink finger protein 5 (ZF5) is transcription factor playing an important role as a transcriptional repressor. ZF5 is composed of 5 zink finger motives enabling the multimerisation process and the binding of ZF5 to GC- rich DNA elements <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[34]</A>.  Both the HSV thymidine kinase (TK) promoter <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[35]</A> and the c-myc promoter <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[34]</A> are targets of the ZF5 transcription factor. The appropriate genes are repressed as a result of the binding process. However, ZF5 is not only a transcription repressor, binding to human immunodeficiency virus (HIV) promoter leads to an enhanced transcription of the appropriate gene.</DIV><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="WT1"> WT1 </SPAN></H3><P>Wilms' tumor protein 1 (WT1) is a transcription factor containing three zink finger motives. The WT1 transcription factor regulates genes being involved in developmental processes (for example development of the urogenital system, kidney, blood vessel formation and heart <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[36]</A>) and cell survival <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[37]</A>. The many isoforms of WT1 in different tissues are the reason for the multitude of functions.  Mutation of the appropriate gene can result in the formation of Wilms' tumor (nephroblastoma).
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="RAR"> RAR </SPAN></H3><P>Retinoic acid receptor (RAR) is a transcription factor belonging to the family of nuclear receptors. RAR builds a hetereodimer with Retinoid X receptor (RXR), this complex is able to bind to specific response elements. Transcriptional activation of the appropriate gene occurs when the ligands all-trans retinoic acid or 9-cis retinoic acid bind to the complex. Genes regulated by RAR are thought to be involved in developmental processes <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[38]</A>. 
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="NFAT"> NFAT </SPAN></H3><P>Nuclear factor of activated T-cells (NFAT) is a transcription factor family consisting of 5 members (NFATc1, NFATc2, NFATc3, NFATc4, and NFAT5). NFATc1 and NFATc4 are sensitive to calcium signalling. A high calcium level leads to the exposure of the nuclear localization signal and the transcription factor is transported into the nucleus.  NFAT proteins play important roles in developmental processes and in the immune system <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[39]</A>.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="Kid3"> Kid3 </SPAN></H3><P>Kid3 is one out of the huge amount of C2H2 zink finger proteins, that are known in eukaryotic organisms. Because of their DNA binding zink finger domain they are involved in gene expression in the role of transcription factors, especially in the early embryonal development, cell growth, diﬀerentiation and tumorigenesis. Kid3 has a Krüppel-associated box domain (KRAB) at the N-terminus, that performs  a transcription repressing function, and a C-terminal C2H2 zink finger domain. The consensus binding sequence of Kid3 is 5'-CCAC(C/G)-3' <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[66]</A>.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H2><SPAN class="mw-headline" id="Proteins"> Proteins </SPAN></H2><H3><SPAN class="mw-headline" id="LDL_receptor"> LDL receptor </SPAN></H3><P>Low-density lipoprotein receptor (LDL receptor) is a cell surface protein which is responsible for the cholesterol supply of the cell.  The receptor recognizes the protein B100 which is part of the LDL particles. Binding of the B100 protein to the LDL receptor leads to endocytosis via clathrin coated pits. The vesicle fuses with an endosome, the resulting shift in the pH value leads to the detachment of the LDL and the receptor can be transported back to the plasma membrane (receptor recycling). Through this process the cell takes up the cholesterol which is associated with LDL.  LDL accumulation in the blood is responsible for atherosclerosis and many cardiovascular diseases. The LDL gene is regulated by the intracellular level of cholesterol. A low level of cholesterol leads to the activation whereas a high level results in a decrease of transcription <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[40]</A>.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="HMG_CoA_synthase"> HMG CoA synthase </SPAN></H3><P>3-hydroxy-3-methylglutaryl-CoA synthase (HMG CoA synthase) is an enzyme catalyzing the condensation of Acetyl-CoA and acetoacetyl-CoA to form 3-hydroxy-3-methylglutaryl-CoA (HMG-CoA). There are two distinct HMG CoA synthases, the cytosolic and the mitochondrial form, encoded by two different genes. The reaction catalyzed by the cytosolic enzyme is a part of the biosynthesis of cholesterol. In mitochondria the same reaction is responsible for keton body formation. Sterol regulatory elements in the promoter region of the gene are responsible for transcriptional regulation <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[41]</A>.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="PUMA"> PUMA </SPAN></H3><P>P53 upregulated modulator of apoptosis (PUMA) is protein belonging to the BH3-only family of pro-apoptotic proteins. P53 plays an important role both in p53 dependent and p53 independent apoptosis. The activation of PUMA leads to mitochondrial dysfunction and caspase activation [1]. PUMA is regulated by many transcription factors (TF), these TFs in turn are regulated by extra and intra cellular stimuli like genotoxic stress, toxins, oncogene expression, redox status and growth factors <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[42]</A>.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="Hsp70"> Hsp70 </SPAN></H3><P>Heat shock protein 70 (Hsp70) is an enzyme helping other proteins to fold. Hsp70 proteins are found not only in the cytosol but also in mitochondria and in the endoplasmatic reticulum. The protein binds, together with the cochaperone Hsp40, to newly synthesised (hydrophobic) amino acid residues and prevents the aggregation of those. During cellular stress e.g. oxidative or thermal stress proteins may unfold, Hsp70 binds to the hydrophobic regions of the proteins and prevents further unfolding, aggregation and apoptosis <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[43]</A>.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="Apo_A-IV"> Apo A-IV </SPAN></H3><P>Apolipoprotein A-IV (Apo A-IV) is a glycoprotein secreted by the small intestine in humans. The production of Apo A-IV is activated through lipid absorption (Chylomicrons). Several studies indicate that Apo A-IV protects against atherosclerosis. It is also thought to be involved in regulation of food intake <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[44]</A>.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="CYP1A1"> CYP1A1 </SPAN></H3><P>Cytochrome P450 1A1 (CYP1A1) is an enzyme which is regulated by the aryl hydrocarbon receptor (AhR) signalling pathway.  The transcription is also influenced by metal ions and oxidative stress. CYP1A1 catalyzes two of three critical steps in transformation of benz[a]pyren to the carcinogen BP-7,8-dihydrodiol-9,10-epoxide.  Furthermore CYP1A1 is involved in processes like xenobiotic metabolism and drug degradation <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[45]</A>.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="EGF"> EGF </SPAN></H3><P>Epidermal Growth Factor (EGF) is a 6045 Da protein discovered by Stanley Cohen in 1986, which won him a Nobel Prize in Physiology and Medicine. EGF regulates cell proliferation by binding to the epidermal growth factor receptors (EGFRs) which are located on the cell surface. Upon binding of EGF to its receptor intrinsic tyrosine kinase activity is stimulated inducing a signaling cascade inside the cell which leads to increased calcium levels, glycolyisis and protein synthesis in the cell. This process ultimately leads to the proliferation of the cell. It was recently shown that c-Jun is one of the targets of EGF action <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[46]</A>, <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[46]</A>.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="TNF-alpha"> TNF-alpha </SPAN></H3><P>Tumor Necrosis Factor-alpha (TNF-alpha) is a cytokine involved the cells inflammatory response. TNF-alpha is a homotrimer that binds to one of its to receptors (TNF-R1/ TNF-R2) which then form a trimer themselves. Trimerization of the receptor induces a conformational change and the dissociation of the inhibitory protein SODD (Silencer of Death Domain protein) from the intracellular death domain of the receptor. The adaptor protein TRADD (TNF Receptor-associated Death Domain protein) can bind now to the death domain and allow other protein factors to bind aswell. The three main signaling pathways initiated in this way are: the NF-kB pathway, the MAPK opathway and the death signaling cascade <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[60]</A>.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="Pifithrin-.CE.B1"> Pifithrin-α </SPAN></H3><P>Pifithrin-α (PFTα) is proposed to be a specific inhibitor of p53 signaling. It is not yet clear how exactly PFTα inhibits p53, but it seems to act at a stage after p53 translocation to the nucleus. Temporary suppression in vitro of p53 inhibits apoptosis induced by the damage to DNA and thus increases the fraction of cells surviving the stress
<A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[61]</A>, <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[61]</A>.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H2><SPAN class="mw-headline" id="RNA-processing_and_transcriptional_regulation"> RNA-processing and transcriptional regulation </SPAN></H2><H3><SPAN class="mw-headline" id="Post-transcriptional_modification_.2F_mRNA_processing_in_eukaryotes"> Post-transcriptional modification / mRNA processing in eukaryotes </SPAN></H3><P>To express a gene and successfully synthesize the appropriate protein the gene must firstly transcript into mRNA. Unlike in bacteria, this mRNA molecule is not directly ready for translation; the primary transcript is therefore called precursor-mRNA (pre-mRNA). One of the first modifications is a process referred to as 5’-capping. By means of several biochemical steps a 7-methylguanosine molecule is bound to the 5’ end of the pre-mRNA, via a 5’ to 5’ triphoshpate linkage.  This 5’ cap has various functions including prevention of 5’ degradation, export from the nucleus and initiation of translation.  Not only the 5’ end but also the 3’ end is modified, this process is called polyadenylation. Therefore a Polyadenylation signal is needed (consensus sequence 5'- AAUAAA-3'), further in the 3’ direction occurs a 5’-CA-3’ element, these both sequences are recognized by the enzymes cleavage and polyadenylation specificity factor and cleavage stimulation factor. Together they are attracting many other proteins including Polyadenylate Polymerase (PAP). The protein complex cuts the pre-mRNA at the CA element and the PAP adds about 200 adenine residues to the 3’ end.  The function of the poly-A tail is protection against degradation, marking of the end of the transcript and aid in translation initiation.  The pre-mRNA contains not only these sequences coding for the protein, so called exons, but also many sequences which are non-coding. These introns have to be removed, that occurs in a process known as splicing. A protein complex called spliceosom connects all the exons thereby cutting out the introns. Responsible for the recognition of the exon-intron borders are small nuclear RNA within the spliceosom. Many genes can be spliced in several ways, an incident termed alternative splicing <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[48]</A>, <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[49]</A>.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="Regulation_of_transcription_in_eukaryotic_organisms"> Regulation of transcription in eukaryotic organisms </SPAN></H3><P>Cells have to adapt to changes in their environment and must be able to receive and react to extra cellular signals; cells accomplish these requirements by the up and down regulation of certain proteins. The protein expression in eukaryotic cells can be regulated on many different levels, this article concentrate on the regulation of transcriptions.  Only a small percentage of the human genomic DNA is transcribed into mRNA. On the opposite, a huge part of the human genome is involved in regulating the transcription of coding sequences.  To initiate transcription of a gene eukaryotic RNA-polymerases have to bind to several general transcription factors to establish the so called initiation complex (IC), which is able to bind to the DNA. The binding occurs upstream of the transcriptional start site (TSS) in a region called core promoter, this part of the promoter often contains specific elements like the TATA-Box (consensus sequence, TATAA/TAA/T, about 30 bp upstream of TSS <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[50]</A>) and the GC-Box (consensus sequence TGTGGCTNNNAGCCAA)  app. 80 bp upstream of the TSS <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[51]</A> to which the IC can bind. Further upstream is a part of the promoter which is referred to as proximal promoter. Containing specific sequence elements, this part of the promoter is highly important for the transcriptional regulation. Transcription factors can bind to these response elements thereby up regulating or down regulating the gene transcription. Proteins methylating or acetylating the DNA are also involved in gene transcription regulation by remodelling of the chromatin structure.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H2><SPAN class="mw-headline" id="Drugs">Drugs</SPAN></H2><H3><SPAN class="mw-headline" id="CPT"> CPT </SPAN></H3><P>Camptothecin (CPT) is a cytotoxic quinoline alkaloid and a  topoisomerase I inhibitor isolated from the Camptotheca acuminata (Camptotheca or the Happy tree). It was discovered during a screen for natural anti-cancer drugs in 1966 but it is not not used in cancer therapy due to its severe side effects, but there were various derivatives developed to increase the benefits of this drug while decreasing its negative effects <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[52]</A>. The two CPT analogues have been approved for cancer chemotherapy today are topotecan and irinotecan. CPT acts by binding to the topoisomerase I-DNA complex using hydrogen bonds and thereby preventing DNA-religation, inducing DNA damage and ultimately causing the cell to die <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[53]</A>.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="Hygromycin"> Hygromycin </SPAN></H3><P>The aminocyclitol antibiotic hygromycin B, that is produced in <I>Streptomyces hygoscopicus</I>, inhibits protein synthesis by interfering into aminoacyl-tRNA recognition and ribosomal translocation. It shows effects in prokaryotes and eukaryotes alike. Hygromycine can be used as a selection marker. The resistance gene encodes for a hygromycin B phosphotransferase, which inactivates the antibiotic by phosphorylation <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[68]</A>. In the iGEM 2009 project of Heidelberg hygromycin B was used for selection of cells which performed a stable integration of the transfected Plasmid.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="Zeocin"> Zeocin </SPAN></H3><P>Zeocin shows a high effectiveness in a wide range of organisms. Mammalian, insect and yeast cells are effected as well as prokaryotic cells. It damages DNA by intercalating and causing breaks and therefore cell death. The zeocin resistance gene encodes for protein which binds zeocin and prohibits DNA destruction[<A href="http://www.invitrogen.com/site/us/en/home/Products-and-Services/Applications/Cell-Culture/Transfection/Selection/Zeocin.html" class="external text" rel="nofollow">71</A>].
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="Neomycin"> Neomycin </SPAN></H3><P>The aminoglycoside antibiotic neomycin is produced by <I>Streptomyces fradiae</I><A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[69]</A>. Neomycin is a selectionmarker for many different cell types. The resistance gene encodes for a phosphotransferase which inactivates neomycine by phosphorylating it <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[70]</A>.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H2><SPAN class="mw-headline" id="Cellular_components_as_tools"> Cellular components as tools </SPAN></H2><H3><SPAN class="mw-headline" id="GPI"> GPI </SPAN></H3><P>Glycosylphosphatidylinositol (GPI) is a glycolipid. During
the posttranslational modification in eukaryotic cells, it becomes
attached to hydrophobic C-termini of proteins that have a special
singnal peptide on them. This signalpeptide leads their translation into
the ER, where the hydrophobic C-terminus will be replaced by a GPI
anchor. Because of its hydrophobic nature it attaches the bound protein
to the cell membrane <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[54]</A>.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="Sar-1"> Sar-1 </SPAN></H3><P>Sar-1 GTP-binding proteins direct the transport of molecules inside of veiscles from the ER to the golgi and the other way round. Being an
anchor for COPII molecules that cause the budding of vesicles off the membranes, it needs a domain to attach to the ER membrane <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[55]</A>. The
C terminus of the Sar-1 protein fullfills this task. Therefore one can
use the C terminus as an ER targeting sequence for other proteins.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="Myrpalm"> Myrpalm </SPAN></H3><P>This localization signal is located at the N-terminal end of the amino
acid chain. The myrpalm signaling sequence causes a myristilation and
palmitolyation of the targeted protein. Both modifications lead to a
binding to the cell membrane <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[56]</A>.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="NLS"> NLS </SPAN></H3><P>Nuclear localisation signals are peptidesequences that are able to bind to nuclear import receptors. These cause an import of newly synthesized protein through nuclear pores. This feature is caused by several positively charged amino acids. Nuclear localization signals can be located almost anywhere in the peptide chain <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[57]</A>. We used a nuclear localization signal at the C-terminal end of the protein.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H3><SPAN class="mw-headline" id="GFP"> GFP </SPAN></H3><P>Green Fluorescent Protein (GFP) was first discovered by Shimomura et al. in
the Aequorea jellyﬁsh. They described a slightly green colour of a GFP-containing solution that in the sunlight <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[63]</A>. The same group of scientists
investigated the protein in more detail, and have since discovered many characteristics, including the excitation and emission wavelengths . The most important accomplishment was the cloning of the GFP gene into other organisms to make them fluorescent <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[64]</A>, <A href="/Team:Heidelberg/Eukaryopedia#References" title="Team:Heidelberg/Eukaryopedia">[65]</A>. Many scientists have since worked on GFP and introduced mutations to enhance fluorscence levels or change the spectra. Nowadays flourescent proteins exist in different colours exist increasing their range of application even more.
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><DIV style="text-align:justify;"><P>[1] Clark R. The process of malignant progression in human breast cancer. <I>Annals of oncology: official journal of the European Society for Medical Oncology/ESMO</I> 1: 401-407 (1990).
</P><P>[2] Osborne C. K., Hobbs K. &amp; Trent J. M. Biological differences among, MCF-7 human breast cancer cell lines from different laboratories. <I>Breast Cancer Research and Treatment</I> 9: 111-121 (1987).
</P><P>[3] Dickson R. B., Bates S. E., McManaway M. E. &amp; Lippman M. E. Characterization of Estrogen Responsive Transforming Activity in Human Breast Cancer Cell Lines. <I>Cancer Research</I> 46: 1707-1713 (1986).
</P><P>[4] Booth B. W. &amp; Smith G. H. Roles of transforming growth factor-α in mammary development and disease. <I>Growth Factors</I> 25: 227-235 (2007).
</P><P>[5] Gey G. O., Coffman W. D. &amp; Kubicek M. T. Tissue culture studies of the proliferative capacity of cervical carcinoma and norml epithelium. <I>Cancer Research</I> 12: 264-265 (1952). 
</P><P>[6] Macville M., Schroeck E., Padilla-Nash H., Keck C., Ghadimi M. B., Zimonjic D., Pospecu N. &amp; Ried T. Comprehensive and definitive moleculare cytogenic characterization of HeLa cells by spectral karyotyping. <I>Cancer Research</I> 59: 141-150 (1999). 
</P><P>[7] Masters J. R. HeLa cells 50 years on: the good, the bad and the ugly. <I>Nature Reviews</I> 2: 315-319 (2002).
</P><P>[8] Ek E. T. H., Dass C. R. &amp; Choong P. F. M. Commonly used mouse models of osteosarcoma. Critical <I>Reviews in Oncology/Hematology</I> 60: 1-8 (2006).
</P><P>[9] Ponten J. &amp; Saksela E. Two established in vitro cell lines from human mesenchymal tumours. <I>International Journal of Cancer</I> 2: 434-447 (1967).
</P><P>[10] Nelissen J. M. D. T., Torensma R., Pluyter M., Adema G. J., Raymakers R. A. P., van Kooyk Y. &amp; Figdor C. G. Molecular analysis of the hematopoiesis supporting osteoblastic cell line U2-OS. <I>Experimental Hematology</I> 28: 422-432 (2000).
</P><P>[11] May, M. J. &amp; Ghosh, S. Rel/NF-κB and IKB proteins: an overview. <I>Seminars in Cancer Biology</I> 8: 63-73 (1997). 
</P><P>[12] Courtois G. The NF-κB signaling pathway in human genetic diseases. <I>Cell. Mol. Life Sci.</I> 62: 1682-1691 (2005).
</P><P>[13] Vazquez A., Bond E. E., Levine A. J. &amp; Bond G. L. The genetics of the p53 pathway, apoptosis and cancer therapy. <I>Nat Rev Drug Discov</I> 7(12): 979-87 (2008).
</P><P>[14] Mangelsdorf D. J. &amp; Evans R. M. The RXR heterodimers and orphan receptors. <I>Cell</I> 83: 841–850 (1995).
</P><P>[15] Koeffler H. P. Peroxisome Proliferator-activated Receptor and Cancers. <I>Clinical Cancer Research</I> 9: 1-9 (2003).
</P><P>[16] Suh N., Wang Y., Honda T., Gribble G. W., Dmitrovsky E., Hickey W. F., Maue R. A., Place A. E., Porter D. M., Spinella M. J., Williams C. R., Wu G., Dannenberg A. J., Flanders K. C., Letterio J. J., Mangelsdorf D. J., Nathan C. F., Nguyen L., Porter W. W., Ren R. F., Roberts A. B., Roche N. S., Subbaramaiah K. &amp; Sporn M. B. A novel synthetic oleanane triterpenoid, 2-cyano-3,12-dioxoolean-1,9- dien-28-oic acid, with potent differentiating, antiproliferative, and antiinflammatory activity. <I>Cancer Res.</I> 59: 336–341 (1999). 
</P><P>[17] Brown M. S. &amp; Goldstein J. L . The SREBP pathway: regulation of cholesterol metabolism by proteolysis of a membrane-bound transcription factor.<I> Cell</I> 89(3): 331–40 (1997).
</P><P>[18] Briggs M. R., Kadonaga J. T., Bell S. P. &amp; R. Tjian. Purification and biochemical characterization of the promoter-specific transcription factor, Sp1. <I>Science</I> 234: 47-52 (1986).
</P><P>[19] Stielow B., Sapetschnig A., Wink C., Kruger I. &amp; Suske G. SUMO-modified Sp3 represses transcription by provoking local heterochromatic gene silencing. <I>EMBO Rep.</I> 9: 899-906 (2008).
</P><P>[20] Zhang Y.,  Liao M. &amp; Dufau M. L. Unlocking repression of the human luteinizing hormone receptor gene by trichostatin A-induced cell-specific phosphatase release. <I>J. Biol. Chem.</I> 283: 24039-24046 (2008).
</P><P>[21] Nicole Y., Khachigian T. &amp; Khachigian L. M. Sp1 Phosphorylation and Its Regulation of Gene Transcription. <I>Molecular and Cellular Biology</I> 29: 2483-2488 (2009).
</P><P>[22] Zhong C-Y, Zhou y-M, Douglas G. C., Witschi H-P. &amp; Pinkerton K. E. MAPK/AP-1 signal pathway in tobacco smoke-induced cell proliferation and quamous metaplasia in the lungs of rats. <I>Carcinogenesis</I> 26(12): 2187–2195 (2005).
</P><P>[23] Hess J., Angel P. &amp; Schorpp-Kistner M. AP-1 subunits: quarrel and harmony among siblings. <I>J. Cell. Sci.</I> 117: 5965–73 (2004).
</P><P>[24] Pellikainen J. M. &amp; Kosma V-M.  Activator protein-2 in carcinogenesis with a special reference to breast cancer—A mini review. <I>Int. J. Cancer</I> 120: 2061–2067 (2007).
</P><P>[25] Hilger-Eversheim K., Moser M., Schorle H. &amp; Buettner R. Regulatory roles of AP-2 transcription factors in vertebrate development, apoptosis and cell-cycle control. <I>Gene</I> 260(1-2): 1-12 (2000).
</P><P>[26] Rani C. S. S., Qiang M. &amp; Ticku M. K. Potential Role of cAMP Response Element-Binding Protein in Ethanol-Induced N-Methyl-D-aspartate Receptor 2B Subunit Gene Transcription in Fetal Mouse Cortical Cells. <I>Molecular Pharmacology Fast Forward</I> 67: 2126-2136 (2005).
</P><P>[27] Yin J. C. &amp; Tully T. CREB and the formation of long-term memory. <I>Curr Opin Neurobiol.</I> 6(2): 264-8 (1996).
</P><P>[28] Pandey S. C., Chartoff E. H., Carlezon W. A., Zou J., Zhang H., Kreibich A. S., Blendy J. A. &amp; Crews F. T. CREB gene transcription factors: Role in molecular mechanisms of alcohol and drug addiction. <I>Alcohol Clin Exp Res.</I> 29(2): 176-184 (2005).
</P><P>[29] Rany I., Megyesi J. K., Reusch J. E. B. &amp; Safirstein R. L. CREB mediates ERK-induced survival of mouse renal tubular cells after oxidant stress. <I>Kidney Int.</I> 68(4): 1573-82 (2005).
</P><P>[30] Matuoka K. &amp; Chen K. Y. Transcriptional regulation of cellular ageing by the CCAAT box-binding factor CBF/NF-Y. <I>Ageing Res Rev.</I> 1(4): 639-51 (2002).
</P><P>[31] Ronchi A., Bellorini M.,  Mongelli N. &amp; Mantovani R.. CCAAT-box binding protein NF-Y (CBF, CP1) recognizes the minor groove and distorts DNA. <I>Nucleic Acids Res.</I> 23(22): 4565–4572 (1995). 
</P><P>[32] Carlberg C. &amp; Seuter S. A genomic perspective on vitamin D signaling. <I>Anticancer Res.</I> 9: 3485-93 (2009).
</P><P>[33] Bertone-Johnson E. R. Vitamin D and breast cancer. <I>Ann Epidemiol.</I> 7: 462-7 (2009).
</P><P>[34] Obata T., Yanagidani A., Yokoro K., Numoto M. &amp; Yamamoto S. Analysis of the consensus binding sequence and the DNA-binding domain of ZF5. <I>Biochem Biophys Res Commun.</I> 255(2): 528-34 (1999).
</P><P>[35] Numoto M., Yokoro K. &amp; Koshi J. ZF5, which is a Kruppel-type transcriptional repressor, requires the zinc finger domain for self-association. <I>Biochem Biophys Res Commun.</I> 256(3): 573-8 (1999).
</P><P>[36] Scholz H., Wagner K. D. &amp; Wagner N. Role of the Wilms' tumour transcription factor, Wt1, in blood vessel formation. <I>Pflugers Arch.</I> 458(2): 315-23 (2009).
</P><P>[37] Sakamoto Y., Mariya Y., Sasaki S., Teshiromori R., Oshikiri T., Segawa M., Ogura K., Akagi T., Kubo K., Kaimori M. &amp; Funato T. WT1 mRNA level in peripheral blood is a sensitive biomarker for monitoring minimal residual disease in acute myeloid leukemia. <I>Tohoku J Exp Med.</I> 219(2): 169-76 (2009).
</P><P>[38] Dollé P. Developmental expression of retinoic acid receptors (RARs). <I>Nucl Recept Signal.</I> 7: e006 (2009).
</P><P>[39] Oh-hora M. &amp; Rao A. The calcium/NFAT pathway: role in development and function of regulatory T cells. <I>Microbes Infect. </I> 11(5): 612-9 (2009).
</P><P>[40] Goldstein J. L. &amp; Brown M. S. The LDL receptor. <I>Arterioscler Thromb Vasc Biol.</I> 4: 431-8 (2009). 
</P><P>[41] Hegardt F. G. Transcriptional regulation of mitochondrial HMG-CoA synthase in the control of ketogenesis. <I>Biochimie</I> 80(10): 803-6 (1998).
</P><P>[42] Yu J. &amp; Zhang L. PUMA, a potent killer with or without p53. <I>Oncogene</I> 27: S71-S83 (2008).
</P><P>[43] Li Z. &amp; Srivastava P. Heat-shock proteins. <I>Curr Protoc Immunol.</I> Appendix 1: Appendix 1T, (2004).
</P><P>[44] Tso P., Liu M., Kalogeris T. J. &amp; Thomson A. B. R. The role of apolipoprotein A-IV in the regulation of food intake. <I>Annu. Rev. Nutr.</I> 21: 231–54 (2001).
</P><P>[45] Androutsopoulos V. P., Tsatsakis A. M., Spandidos D. A. Cytochrome P450 CYP1A1: wider roles in cancer progression and prevention. <I>BMC Cancer</I> 9: 187 (2009).
</P><P>[46] Carpenter G. &amp; Cohen S. Epidermal growth factor. <I>The Journal of Biological Chemistry</I> 265(14): 7709–12 (1990).
</P><P>[47] Schnidar H., Eberl M., Klingler S., Mangelberger D., Kasper M., Hauser-Kronberger C., Regl G., Kroismayr R., Moriggl R., Sibilia M. &amp; Aberger F. Epidermal growth factor receptor signaling synergizes with Hedgehog/GLI in oncogenic transformation via activation of the MEK/ERK/JUN pathway. <I>Cancer Res.</I> 69(4): 1284-92 (2009).
</P><P>[48] Moore M. J. &amp; Proudfoot N. J. Pre-mRNA processing reaches back to transcription and ahead to translation. <I>Cell</I> 136(4): 688-700 (2009).
</P><P>[49] Berg J. M., Tymoczko J. L. &amp;  Stryer L. <I>Biochemistry</I> (6 ed.) New York: WH Freeman &amp; Co, 2007. 
</P><P>[50] Day D. A. &amp; Tuite M. F. Post-transcriptional gene regulatory mechanisms in eukaryotes: an overview, <I>J. Endocrinol.</I> 157: 361-371 (1998).
</P><P>[51] Litt M., Qiu Y. &amp; Huang S. Histone arginine methylations: their roles in chromatin dynamics and transcriptional regulation. <I>Biosci Rep.</I> 29(2): 131-41 (2009).
</P><P>[52] Wall M. E., Wani M. C., Cook C. E., Palmer K. H., McPhail A. I. &amp; Sim G. A. Plant antitumor agents. I. The isolation and structure of camptothecin, a novel alkaloidal leukemia and tumor inhibitor from camptotheca acuminate. <I>J. Am. Chem. Soc.</I> 88: 3888–3890 (1966).
</P><P>[53] Redinbo M. R., Stewart L., Kuhn P., Champoux J. &amp; Hol W. G. J. Crystal structure of human topoisomerase I in covalent and noncovalent complexes with DNA. <I>Science</I> 279: 1504–1513 (1999).
</P><P>[54] Fricker M., Runions J. &amp; Moore I. Quantitative fluorescence microscopy: From Art to science. <I>Annual Review of Plant Biology</I> 57: 79-107 (2006).
</P><P>[55] Heim R. &amp; Tsien R. Y. Engineering green fluorescent protein for improved brightness, longer wavelengths and fluorescence resonance energy transfer. <I>Current Biology</I> 6(2): 178-182 (1996).
</P><P>[56] Alberts B., et al. (2008): The Cell, Fifth edition, Garland Science London/New York, p. 742
</P><P>[57] d'Enfert C., Gensse M. &amp; Gaillardin C. Fission yeast and a plant have functional homologues of the Sari and Sec12 proteins involved in ER to Golgi traffic in budding yeast. <I>The EMBO Journal</I> 11: 4205-4211 (1992).
</P><P>[58] Zacharias D. A., Violin J. D., Newton A. C. &amp; Tsien R. Y. Partitioning of Lipid-Modified Monomeric GFPs into Membrane Microdomains of Live Cells. <I>Science</I>296(5569): 913-6 (2002).
</P><P>[59] Alberts B., et al. (2008): The Cell, Fifth edition, Garland Science London/New York, pp. 706-707
</P><P>[60] Locksley R. M., Killeen N. &amp; Lenardo M. J. The TNF and TNF receptor superfamilies: integrating mammalian biology. <I>Cell</I> 104(4): 487–501 (2001).
</P><P>[61] Komarova E. A. &amp; Gudkov A. V. Suppression of p53: a new approach to overcome side effects of antitumor therapy. <I>Biochemistry (Mosc).</I> 65(1): 41-48 (2000).
</P><P>[62] Murphy P. J., Galigniana M. D., Morishima Y., Harrell J. M., Kwok R. P., Ljungman M. &amp; Pratt W. B. Pifithrin-alpha inhibits p53 signaling after interaction of the tumor suppressor protein with hsp90 and its nuclear translocation. <I>J Biol Chem.</I> 279(29): 30195-201 (2004). 
</P><P>[63] Tsien R. Y. The green fluorescent protein. <I>Annu. Rev.</I>
Biochem.<I> 67: 509–44 (1998).</I></P><P>[64] Tsien R. &amp; Prasher D. <I>Green Fluorescent Protein Properties, Applications, and Protocols.</I> New York: Wiley-Liss, 1998, p. 67-118.
</P><P>[65] Heim R. &amp; Tsien R. Engineering Green Fluorescent Protein for improved brightness, longer wavelength and fluorescence resonance energy transfer. <I>Current Biology</I> 6: 178-182 (1996).
</P><P>[66] Gao L., Sun C., Qiu H., Liu H., Shao H., Wang J., Li W. Cloning and characterization of a novel human zinc ﬁnger gene, hKid3, from a C2H2-ZNF enriched human embryonic cDNA library. <I>Biochemical and Biophysical Research Communications</I> 325:1145–1152 (2004)
</P><P>[67] Rahman S. &amp; Thomas P. Molecular cloning, characterization and expression of two hypoxia-inducible factor alpha subunits, HIF-1α and HIF-2α, in a hypoxia-tolerant marine teleost, Atlantic croaker (Micropogonias undulatus). <I>Gene</I> 396: 273–282 (2007).
</P><P>[68] Blochinger K. &amp; Diggelmann H. Hygromycin B Phosphotransferase as a Selectable Marker for DNA Transfer Experiments with Higher Eucaryotic Cells. <I>MOLECULAR AND CELLULAR BIOLOGY</I> 12: 2929-2931 (1984).
</P><P>[69] Majumdar M. K. &amp; Majumdar S. K. Effects of Minerals on Neomycin Production by <I>Streptomyces fradiae</I>.  <I>Appl. Environ. Microbiol.</I> 13(2): 190-193 (1965).
</P><P>[70] Yenofsky R. L., Fine M. &amp;  Pellow J. P. A mutant neomycin phosphotransferase II gene reduces the resistance of transformants to antibiotic selection pressure. <I>Proc. Nat. Acad. Sci.</I> 87: 3435-3439 (1990).
</P><P>[71]Invitrogen Corporation. 
<A href="http://www.invitrogen.com/site/us/en/home/Products-and-Services/Applications/Cell-Culture/Transfection/Selection/Zeocin.html" class="external free" rel="nofollow">http://www.invitrogen.com/site/us/en/home/Products-and-Services/Applications/Cell-Culture/Transfection/Selection/Zeocin.html</A>  21.10.2009
</P><P><A href="/Team:Heidelberg/Eukaryopedia#Eukaryopedia" title="Team:Heidelberg/Eukaryopedia">[TOP]</A></P></DIV></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Heidelberg/Eukaryopedia">http://2009.igem.org/Team:Heidelberg/Eukaryopedia</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Eukaryopedia" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Eukaryopedia" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Eukaryopedia&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Eukaryopedia&amp;oldid=159621" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>