<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_stables"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Heidelberg/stables">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/stables&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/stables&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/stables&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/stables" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/stables</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="main_wrapper"><DIV id="header"><DIV id="navigation"><DIV id="menu" style="position:static"><UL><LI id="mainHome"><A class="aMain" href="https://2009.igem.org/Team:Heidelberg">Home</A></LI><LI id="mainTeam"><A class="aTeam" href="https://2009.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Members#Students">Students</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Members#Advisors">Advisors</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Scientific_Environment">Scientific Environment</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Gallery">Gallery</A></LI></UL><LI id="mainProject"><A class="aProject" href="https://2009.igem.org/Team:Heidelberg/Project">Project</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project">Abstract</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Introduction">Introduction</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Measurement">Measurement</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Synthetic_promoters">Synthetic Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Further">Further Subprojects</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_SaO">Summary &amp; Outlook</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_References">References</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Acknowledgement">Acknowledgements</A></LI><LI id="mainParts"><A class="aParts" href="https://2009.igem.org/Team:Heidelberg/Parts">Parts</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Parts">Parts</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Parts_Characterization">Characterization</A></LI><LI id="mainModeling"><A class="aModeling" href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">What is HEARTBEAT?</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_database">HEARTBEAT Database</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_GUI">HEARTBEAT GUI</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_gui">GUI Documentation</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_network">HEARTBEAT Fuzzy Network Modeling</A></LI><LI id="mainNotebook"><A class="aNotebook" href="https://2009.igem.org/Team:Heidelberg/Notebook">Notebook</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_natural_promoters_overview">Natural Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_synthetic_promoters">Synthetic Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_promoters_cells">Cell Culture, Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_cell_line">Stable Cell Line</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_color_output">Multi-Color Output</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_modeling">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_measure">Measurements</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_MaM">Material &amp; Methods</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_meetings">Team Meetings</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Biosafety">Biosafety</A></LI><LI id="mainEucaryopedia"><A class="aEucaryopedia" href="https://2009.igem.org/Team:Heidelberg/Eukaryopedia">Eukaryopedia</A></LI><LI id="mainSponsors"><A class="aSponsors" href="https://2009.igem.org/Team:Heidelberg/Sponsors">Sponsors</A></LI></DIV></DIV></DIV></DIV><TABLE><TBODY><TR valign="top" style="margin-left: 2px;"><TD width="650px" style="padding: 0 15px 15px 20px; background-color:#ede8e2"><H1><SPAN class="mw-headline" id="Stable_Cell_Line">Stable Cell Line</SPAN></H1><H2><SPAN class="mw-headline" id="Abstract"> Abstract </SPAN></H2><P>The one problem that is always faced when measuring any construct or device in mammalian cells is the fact that the transfection rate is not always constant. Thus, one never knows how many copies of a construct are actually in the cell. This is why we attempted to create a system that allows a controled integration of our constructs into the genome. To accomplish this we used the FRT/Flp system which is based on homologous recombination by the enzyme flippase at a specific sequence – the FRT site <A href="/Team:Heidelberg/stables#References" title="Team:Heidelberg/stables">[1]</A>. One of those sequences is on a plasmid that is integrated randomly, but stably into the genome by antibiotic (Zeocin) selection. The other sequence is on the plasmid that contains the construct to be tested. Thus it would be possible to integrate at a specific site on the genome and select cells with the construct via a second antibiotic (Hygromycin). Additionally, the stable integration of an FRT site into the genome allows the characterization of the inserted constructs in the same genomic context thereby providing a normalization of the expression. Lastly, this integration site can also be located by <A href="/Team:Heidelberg/stables#LAM-PCR" title="Team:Heidelberg/stables">LAM-PCR</A>.
</P><H2><SPAN class="mw-headline" id="Introduction"> Introduction</SPAN></H2><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><DIV style="text-align:justify;"><B>Figure 1: Characteristics of FRT site.</B> Cleavage and recombination by FLP takes place within the 8 bp spacer in between repeat 2 and 3. These two repeats are necessary for the reaction. Repeat 1 is present in most of the cases, but not strictly needed for the integration process.</DIV></DIV></DIV></DIV><DIV style="text-align:justify;">The FRT/Flp system originates from yeast. The  eukaryotic site-specific recombinase FLP is originally encoded by the multicopy yeast plasmid 2µ circle DNA that is common to many yeast strains <A href="/Team:Heidelberg/stables#References" title="Team:Heidelberg/stables">[2]</A><A href="/Team:Heidelberg/stables#References" title="Team:Heidelberg/stables">[3]</A>. Naturally, the cleavage and recombination takes place at an 8 bp spacer that is on either side enclosed by two inverted repeats of 599 bp in length. For FLP-mediated recombination only 13 bp of each 599 bp repeat adjacent to the spacer and the spacer itself are necessary (see Fig. 1 sequences 2 and 3). Additionaly, most of the time another 13 bp repeat occurs immediately </DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><DIV style="text-align:justify;"><B>Figure 2: Principle of FRT/Flp recombination.</B> The integration of the GOI is realized by transfecting the cells with both the FRT plasmid containing the GOI and a plasmid which codes for the enzyme flippase. The integration process starting at the 8 bp spacer sequence of the FRT-sites occurs via Flip mediated homologous recombination. The process includes the destruction of the genomic Zeocin resistance and the integration of the Hygromycin resistance originally encoded on the plasmid, thus leading to  new selection conditions for stable clones.</DIV></DIV></DIV></DIV><DIV style="text-align:justify;">upstream of the 5’ repeat (see Fig. 1 sequence 1). The spacer contains the cleavage site and is thus very sensitive to alterations either in length or in sequence. Moreover, its asymmetry alone determines the orientation of the FLP recombination site <A href="/Team:Heidelberg/stables#References" title="Team:Heidelberg/stables">[2]</A>. Integrating one of those FLP recognition target sites (FRT site) stably into the genome and having another one on a plasmid which also contains the construct of interest will allow a controlled integration at that specific site by cotransfecting with an FLP encoding plasmid. Fig. 2 illustrates the nature of this integration. </DIV><H2><SPAN class="mw-headline" id="Results"> Results </SPAN></H2><P>We attempted to make a cell line with an FRT site by using the <A href="/Team:Heidelberg/Notebook_MaM#Plasmids" title="Team:Heidelberg/Notebook MaM">pFRT/<I>lacZeo</I> plasmid</A> (Invitrogen). After transfecting HeLa, U2-OS and MCF-7 cells with the mentioned pFRT/<I>lacZeo</I> plasmid by using the <A href="/Team:Heidelberg/Notebook_MaM#Transfection_of_Mammalian_cells" title="Team:Heidelberg/Notebook MaM">Effectene kit</A> (Qiagen), we selected cells that stably integrated the plasmid by adding Zeocin (100 μg/ml) to the culture medium (<A href="/Team:Heidelberg/Notebook_MaM" title="Team:Heidelberg/Notebook MaM">DMEM+++</A>). After several days, cells with a Zeocin resistance survive and grow in foci originated from one single cell. Those foci were then <A href="/Team:Heidelberg/Notebook_MaM#Picking_cells_with_cloning_disks" title="Team:Heidelberg/Notebook MaM">picked</A> as described in the cell culture methods. They were expanded into culture flasks and aliquots were frozen. After DNA extraction by use of the <A href="/Team:Heidelberg/Notebook_MaM#Kits" title="Team:Heidelberg/Notebook MaM">High Pure PCR Template Preparation Kit</A> (Roche), <A href="/Team:Heidelberg/stables#LAM-PCR" title="Team:Heidelberg/stables">LAM-PCR</A> was carried out to localize the integration sites on the genome. Cotransfection of the plasmids <A href="/Team:Heidelberg/Notebook_MaM#Plasmids" title="Team:Heidelberg/Notebook MaM">pOG44</A> (encodes for FLP) and <A href="/Team:Heidelberg/Notebook_MaM#Plasmids" title="Team:Heidelberg/Notebook MaM">p55</A> (contains CMV promoter and GFP) at a ratio of 9:1 led to the integration of a GFP with a CMV promoter in some of the cells. After this cotransfection the cells were kept under Hygromycin selection (200 µg/ml). There is no promoter in front of the resistance gene for Hygromycin so that cells that randomly and stably integrate the plasmid (as it was done for the pFRT/<I>lacZeo</I>) do not express the resistance. Once again, foci were visible after several days( see Fig. 3). 
</P><P>To properly characterize the integration sites of the pFRT/<I>lacZeo</I> plasmid in the genome we performed <A href="/Team:Heidelberg/stables#LAM-PCR" title="Team:Heidelberg/stables">LAM-PCR</A> at M. Schmidt's lab (see <A href="/Team:Heidelberg/stables#References" title="Team:Heidelberg/stables">reference 5</A> for details) at the nct (national center for tumor diseases) in Heidelberg. Thereby we confirmed that we were able to generate HeLa, MCF-7 and U2-OS cells that stably integrated the FRT-site into their genome. For each cell line there are several bands visible (see Fig. 4). Since the number of bands correlates with the number of unique integration sites <A href="/Team:Heidelberg/stables#References" title="Team:Heidelberg/stables">[5]</A> there must have been more than one integration of the FRT-vector into the genome of the cells.
</P></TD></TR><TR valign="top"><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><DIV style="text-align:justify;"><B> Figure 3: One focus of cells</B> that stably integrated a plasmid containing CMV and GFP into their genome. Cells in the focus should all have evolved from a single cell, that successfully passed through homologous recombination at the FRT- site mediated by flippase </DIV></DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><DIV style="text-align:justify;"><B>Figure 4:  Results of second exponential PCR.</B> In all lanes there are  several bands visible, indicating that there is more than one FRT-site integrated in each cell line. Negative controls are: 1. untransfected genomic DNA 2. H<SUB>2</SUB>O negative control LAM-PCR, 3. H<SUB>2</SUB>O negative control first exponential PCR, 4. H<SUB>2</SUB>O negative control second exponential PCR. Electrophoresis was carried out on 2% agarose; <A href="http://tools.invitrogen.com/content/sfs/manuals/15628019.pdf" class="external text" rel="nofollow">100 bp DNA ladder</A> was used.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Discussion"> Discussion </SPAN></H2><P>Our experiment showed that it is possible to generate cell lines with stable FRT sites. We are aware that our cell lines with multiple FRT sites are not ideal for the characterization of promoters. Nevertheless, we are convinced that a cell line that contains a single integration site can be generated and that it would be very valuable for standardized characterization of BioBrick parts in eukaryotes. Even though those characterizations are more time consuming due to the Hygromycin selection, they are of advantage once established, because the construct cannot be &quot;washed out&quot; as it is observed when dealing with transiently transfected cells. 
Measurements with our cell lines with multiple integration sites are also possible and favorable over transiently transfected cell lines. For one, it is expected that the number of construct integrations is statistically distributed. Furthermore, one can determine the number of constructs integrated. Since the genomic surroundings of the integration sites can be characterized by <A href="/Team:Heidelberg/stables#LAM-PCR" title="Team:Heidelberg/stables">LAM-PCR</A>, another (regular) PCR with one primer that is located on the genome and one that is on the other side of the gene of interest (GOI) can be carried out. 
To generate a cell line with only one integration site, the pFRT/<I>lacZeo</I> should have been linearized before transfection. That way, the transfection efficiency and thus the chance to stably integrate more than one copy would have been lower. Also, linearization of the plasmid is favorable for LAM-PCR, because the sequences directly adjacent to the genomic DNA would be known. Although being very time-consuming, another attempt to create a cell line that stably integrated only one FRT-site would have a great impact for the synthetic biology community.
</P><H2><SPAN class="mw-headline" id="Detailed_Methods"> Detailed Methods </SPAN></H2><H3><SPAN class="mw-headline" id="LAM-PCR"> LAM-PCR </SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><DIV style="text-align:justify;"><B>Figure 5: Schematic outline of LAM-PCR used to amplify the genomic DNA flanking the FRT-site.</B>The solid phase consists of Streptavidin-coupled magnetic beads which allow for the binding of DNA via biotinylated Primers.</DIV></DIV></DIV></DIV><DIV style="text-align:justify;"> In order to determine where the plasmid containing the FRT-site is integrated into the genome, LAM-PCR is applied.  This method established by Schmidt <I>et al</I>. can be used for identifying all unknown DNA sequences adjacent to known DNA-sequences.Based on linear amplification-mediated PCR, it allows for amplifying the sequences flanking the integrated vector thus supplying the information which is needed for alignment with the host genome. 
<P>The procedure starts with linear PCR (Fig 5.a), which is carried out twice with 50 - cycles for each reaction. Primers used in PCR are labeled with biotin, which has a high affinity to Streptavidin <A href="/Team:Heidelberg/stables#References" title="Team:Heidelberg/stables">[4]</A>. By coupling Streptavidin to magnetic beats, removal of non-target DNA can be achieved via magnetic selection: newly synthesized single strands will bind via primer to the magnetic beads and can then be captured with MPC (Magnetic particle concentrator). The same kind of selection is used for the subsequent steps.
After linear amplification, complementary double strands are synthesized by hexanucleotide priming: a hexanucleotide mixture, dNTPS and Polymerase are added to the single stranded PCR-products. Hexanucleotides will prime to the single stranded DNA and polymerase will fill the gaps in between (Fig. 5.c) 
Those double strands are cut with restriction enzymes and a linker cassette with known sequence is ligated to the genomic end of the fragments which sequences are unknown (Fig. 5.d,e). 
This step is followed by denaturation of the double stranded DNA composed of linker cassette, genomic DNA and vector DNA by sodium hydroxid. The resulting single stranded DNA is then amplified by exponential PCR, using primers that prime to linker cassette and vector DNA. Another denaturation and a second exponential PCR is carried out (Fig. 5.f,g). 
PCR-products are visualized by separation on a 2% agarose gel.
</P>
For a detailed description see <A href="/Team:Heidelberg/stables#References" title="Team:Heidelberg/stables">[5]</A></DIV><H2><SPAN class="mw-headline" id="References"> References </SPAN></H2><P>[1] Zhou H. Development of site-specific integration system to high-level expression recombinant proteins in CHO cells. <I>Chinese journal of biotechnology</I> 23: 756-62 (2007).
</P><P>[2] Andrews B. J., Proteau G. A., Beatty L. G. &amp; Sadowski P. D. The FLP Recombinase of the 2μ Circle DNA
of Yeast: Interaction with Its Target Sequences. <I>Cell</I> 40: 795-803 (1985). 
</P><P>[3] Senecoff J. F., Bruckner R. C. &amp; Cox M. M. The FLP recombinase of the yeast 2-μ plasmid: Characterization
of its recombination site. <I>Proc. Nati. Acad. Sci. USA</I> 82: 7270-7274 (1985).
</P><P>[4] Holmberg A., Blomstergren A., Nord O., Lukacs M., Lundeberg J. &amp; Uhlén M. The biotin-streptavidin interaction can be reversibly broken using water at elevated temperatures. <I>Electrophoresis</I> 26(3): 501-10 (2005). 
</P><P>[5] Schmidt M., Schwarzwaelder K., Bartholomae C., Zaoui K., Ball B.,Pilz I.,Braun S.,Glimm H.,von Kalle C., High-resolution insertion-site analysis by linear amplification–mediated PCR (LAM-PCR). <I>Nature Methods</I> 4(12): 1051-1057 (2005).
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Heidelberg/stables">http://2009.igem.org/Team:Heidelberg/stables</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/stables" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/stables" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/stables&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/stables&amp;oldid=156515" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>