<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_HEARTBEAT_network"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Heidelberg/HEARTBEAT_network">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/HEARTBEAT_network&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/HEARTBEAT_network&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/HEARTBEAT_network&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/HEARTBEAT_network" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/HEARTBEAT network</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="main_wrapper"><DIV id="header"><DIV id="navigation"><DIV id="menu" style="position:static"><UL><LI id="mainHome"><A class="aMain" href="https://2009.igem.org/Team:Heidelberg">Home</A></LI><LI id="mainTeam"><A class="aTeam" href="https://2009.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Members#Students">Students</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Members#Advisors">Advisors</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Scientific_Environment">Scientific Environment</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Gallery">Gallery</A></LI></UL><LI id="mainProject"><A class="aProject" href="https://2009.igem.org/Team:Heidelberg/Project">Project</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project">Abstract</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Introduction">Introduction</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Measurement">Measurement</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Synthetic_promoters">Synthetic Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Further">Further Subprojects</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_SaO">Summary &amp; Outlook</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_References">References</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Acknowledgement">Acknowledgements</A></LI><LI id="mainParts"><A class="aParts" href="https://2009.igem.org/Team:Heidelberg/Parts">Parts</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Parts">Parts</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Parts_Characterization">Characterization</A></LI><LI id="mainModeling"><A class="aModeling" href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">What is HEARTBEAT?</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_database">HEARTBEAT Database</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_GUI">HEARTBEAT GUI</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_gui">GUI Documentation</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_network">HEARTBEAT Fuzzy Network Modeling</A></LI><LI id="mainNotebook"><A class="aNotebook" href="https://2009.igem.org/Team:Heidelberg/Notebook">Notebook</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_natural_promoters_overview">Natural Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_synthetic_promoters">Synthetic Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_promoters_cells">Cell Culture, Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_cell_line">Stable Cell Line</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_color_output">Multi-Color Output</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_modeling">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_measure">Measurements</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_MaM">Material &amp; Methods</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_meetings">Team Meetings</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Biosafety">Biosafety</A></LI><LI id="mainEucaryopedia"><A class="aEucaryopedia" href="https://2009.igem.org/Team:Heidelberg/Eukaryopedia">Eukaryopedia</A></LI><LI id="mainSponsors"><A class="aSponsors" href="https://2009.igem.org/Team:Heidelberg/Sponsors">Sponsors</A></LI></DIV></DIV></DIV></DIV><TABLE><TBODY><TR valign="top" style="margin-left: 2px;"><TD style="padding: 0 15px 15px 20px; background-color:#ede8e2"><H1><SPAN class="mw-headline" id="HEARTBEAT_Fuzzy_Modeling"> HEARTBEAT Fuzzy Modeling </SPAN></H1><H2><SPAN class="mw-headline" id="Contents"> Contents </SPAN></H2><UL><LI><A href="/Team:Heidelberg/HEARTBEAT_network#Introduction" title="Team:Heidelberg/HEARTBEAT network">Introduction</A></LI><LI><A href="/Team:Heidelberg/HEARTBEAT_network#Background_.2F_Motivation" title="Team:Heidelberg/HEARTBEAT network">Background / Motivation</A></LI><LI><A href="/Team:Heidelberg/HEARTBEAT_network#Achievements" title="Team:Heidelberg/HEARTBEAT network">Achievements</A></LI><LI><A href="/Team:Heidelberg/HEARTBEAT_network#Results" title="Team:Heidelberg/HEARTBEAT network">Results</A></LI><LI><A href="/Team:Heidelberg/HEARTBEAT_network#Discussion" title="Team:Heidelberg/HEARTBEAT network">Discussion</A></LI><LI><A href="/Team:Heidelberg/HEARTBEAT_network#References" title="Team:Heidelberg/HEARTBEAT network">References</A></LI></UL><UL><LI> If you want directly visit our Results page please <A href="/Team:Heidelberg/HEARTBEAT_network/Results" title="Team:Heidelberg/HEARTBEAT network/Results">click HERE!</A></LI></UL><H2><SPAN class="mw-headline" id="Introduction"> Introduction </SPAN></H2><P>The research field of <A href="/Team:Heidelberg/Eukaryopedia#Regulation_of_transcription_in_eukaryotic_organisms" title="Team:Heidelberg/Eukaryopedia">regulation of gene expression in eukaryotes</A> is a field of biological research growing rapidly <A href="/Team:Heidelberg/HEARTBEAT_network#References" title="Team:Heidelberg/HEARTBEAT network">[1,2]</A>. Hereby the interaction of DNA with certain proteins known as transcription factors (TFs) plays an essential role for the complex mechanism of transcriptional activation <A href="/Team:Heidelberg/HEARTBEAT_network#References" title="Team:Heidelberg/HEARTBEAT network">[3,4]</A>. One strong focus of synthetic biology aims at the reconstruction of such gene regulatory networks <A href="/Team:Heidelberg/HEARTBEAT_network#References" title="Team:Heidelberg/HEARTBEAT network">[5-8]</A>. To act within the scope of synthetic biology’s duties, the iGEM Team Heidelberg 09 claims that any synthetic promoter can be constructed by using our <A href="/Team:Heidelberg/Project_Synthetic_promoters" title="Team:Heidelberg/Project Synthetic promoters">two methods for the construction of synthetic promoters</A>. However, the only efficient way to construct systems of high complexity (such as computers or airplanes) is simulating these systems on the computer prior to construction <A href="/Team:Heidelberg/HEARTBEAT_network#References" title="Team:Heidelberg/HEARTBEAT network">[9]</A>. In our case, this strongly emphasizes the necessity of 
<A href="/Team:Heidelberg/HEARTBEAT" title="Team:Heidelberg/HEARTBEAT">HEARTBEAT</A> (Heidelberg Artificial Transcription Factor Binding Site Engineering and Assembly Tool) which comprises data analysis (<A href="/Team:Heidelberg/HEARTBEAT" title="Team:Heidelberg/HEARTBEAT">HEARTBEAT_database</A>), a graphical user interface (<A href="/Team:Heidelberg/HEARTBEAT_gui" title="Team:Heidelberg/HEARTBEAT gui">HEARTBEAT GUI</A>) and network modeling (<STRONG class="selflink">HEARTBEAT fuzzy network (FN) modeling</STRONG>).
</P><P>Contributing to the HEARTBEAT project, HEARTBEAT FN focuses on simulating the promoter activity by integrating a variety of signals and sequence characteristics as well as on predicting distinct pathway functionalities. This, especially in eukaryotic systems, is a tough challenge since transcriptional activity of a gene is not directly correlated to protein expression <A href="/Team:Heidelberg/HEARTBEAT_network#References" title="Team:Heidelberg/HEARTBEAT network">[10]</A>.
</P><P>For this purpose we propose fuzzy logic (FL) modeling as an approach to logic-based modeling which is capable of incorporating qualitative data but producing quantitative predictions. New insights will be provided about the operation of gene regulatory networks and relationships between promoter sequence composition and TF-DNA interaction will be unraveled that is understood only marginally so far <A href="/Team:Heidelberg/HEARTBEAT_network#References" title="Team:Heidelberg/HEARTBEAT network">[11-12]</A>. 
</P><P>[<A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_network" class="external text" rel="nofollow">TOP</A>]
</P><H2><SPAN class="mw-headline" id="Background_.2F_Motivation"> Background / Motivation </SPAN></H2><P>We present two different approaches for promoter design resulting in three different types of synthetic promoters: <B>randomly assembled</B><I>constitutive</I> and <I>inducible</I> promoters as well as <B>rationally designed</B> promoters. As an additional type of promoters those <B>occurring in nature</B> can be integrated into vector systems.
These heterogeneous cocktail of promoters can be combined for a precise regulation of pathways. This represents the power of our entire HEARTBEAT project. 
Synthesized promoters can be then used e.g. as a combinatorial gene therapy, i.e. several promoters that are of different types and/or have different strength will be applied as treatment agents. Therefore, a model that not only simulates single promoter activity and following gene expression but also accurately predicts gene expression from combined promoter sequences is indispensable.
</P><P>We constructed a Fuzzy Logic model to provide a formal mathematical framework for prediction of combined activity of multiple promoters upon several stimuli and to gain insight into the mechanisms that generate diverse expression levels.
</P><H3><SPAN class="mw-headline" id="A_Short_Introduction_into_Fuzzy_Logic_Modeling"> A Short Introduction into Fuzzy Logic Modeling </SPAN></H3><P>Fuzzy Logic is a rule-based approximate artificial reasoning method developed by <A href="http://en.wikipedia.org/wiki/Lofti_Zadeh|" class="external text" rel="nofollow">Lotfi Zadeh</A> in 1965. Its motivation is 
the observation that humans often think and communicate in a vague way, and yet can make precise decisions <A href="/Team:Heidelberg/HEARTBEAT_network#References" title="Team:Heidelberg/HEARTBEAT network">[13]</A>. It has been widely used in engineering and Artificial Intelligence approaches such as Fuzzy Controllers and Fuzzy Expert Systems. Fuzzy Logic has also been used for the modeling of biological pathways <A href="/Team:Heidelberg/HEARTBEAT_network#References" title="Team:Heidelberg/HEARTBEAT network">[14]</A> and very recently to analyze gene regulatory networks <A href="/Team:Heidelberg/HEARTBEAT_network#References" title="Team:Heidelberg/HEARTBEAT network">[15]</A>. Key advantages of Fuzzy logic-based approaches are <B>(i)</B> the ability to construct models based on prior knowledge of the system and experimental data and <B>(ii)</B> encode intermediate states for inputs and outputs, thus improving other logic-approaches that can only deal with <I>ON/OFF</I> states such
as Boolean models <A href="/Team:Heidelberg/HEARTBEAT_network#References" title="Team:Heidelberg/HEARTBEAT network">[16]</A> and <B>(iii)</B> simulations can be derived from both qualitative and quantitative data, both of which can be cast into the form of <I>IF-THEN</I> rules. Thus, FL constitutes a powerful approach for the understanding of heterogeneous datasets.
</P><H3><SPAN class="mw-headline" id="A_Model_combining_in_silico_prediction_with_experimental_data"> A Model combining <I>in silico</I> prediction with experimental data </SPAN></H3><P>In our project, the complete set of rules will capture the behavior of each promoter in a Multiple-Input Single-Output (MISO) Fuzzy Logic model. Combining the MISO models in a network of all promoters will constitute the final Multiple-Input Multiple-Output (MIMO) model allowing for the simulation and prediction of combined activation of pahways regulated by our promoters. A key advantage of this methodology towards understanding the exclusive pathway activation of our promoters of interest is the possibility to study not only the individual activity of each promoter but also the combined activity, as the signal progresses from one MISO to another. A general idea of our modeling network which is already adapted to our whole <A href="/Team:Heidelberg/Project" title="Team:Heidelberg/Project">project</A> is shown in Fig. 1.
</P></TD></TR><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 1:</B> General idea of our project. Each part illustrated in the network scheme is already adapted to our overall project. The model consists of three layers: Input contains all possible parameters such as sequence information or stimulants, the second layer contains all cellular equipments which can be affected by these inputs allowing crosstalk between each other and the third layer illustrates all possible experimental outcomes including the corresponding measurement methods. Rosy measurement boxes: measurements we are focusing on during the summer of iGEM 2009. Gray measurement boxes: additionally possible experimental techniques to capture cellular behaviour.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>
[<A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_network" class="external text" rel="nofollow">TOP</A>]
</P><H2><SPAN class="mw-headline" id="Achievements"> Achievements </SPAN></H2><H3><SPAN class="mw-headline" id="Model_description"> Model description </SPAN></H3><H4><SPAN class="mw-headline" id="Our_HEARTBEAT_fuzzy_network_model"> Our HEARTBEAT fuzzy network model </SPAN></H4><TABLE><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 2</B>: Interaction of the four layers or our HEARTBEAT fuzzy network model. The Heartbeat (horizontal layer) calculates sequences with a postition score and affinity score. The control of the promoter activity by these scores and every other regulatory event is captured by the fuzzy logic (blue layer). Based on this complex regulated behavior, the fuzzy model predicts the activity of each promoter on the network, capturing as well feedback loops (brown layer). The final model (green layer) combines predicted activity of all promoters testing all our hypothesis</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>The HEARTBEAT fuzzy network model we have developed has four layers with different functionalities (see Fig.2). 
Its global aim is of a particular importance! The Heartbeat tells you what sequence allows for regulation of the expression of your gene of interest, but how do you know how to combine more than one promoter, i.e. what combined activity to expect? What if these promoters crosstalk (see Fig.2)? What if you want to use several exclusive promoters to regulate several pathways? 
Our model predicts the final activity of all promoters by using their Affinity and Position Scores and regulatory events such as inducible/constitutive activity, fixed stimulus, or increasing stimuli. A key advantage of the fuzzy logic core of the modeling approach is the use of both quantitative and qualitative experimental data as well as literature-prior knowledge.
</P><H4><SPAN class="mw-headline" id="Two_Measures_for_scoring_promoter_sequences"> Two Measures for scoring promoter sequences </SPAN></H4><P>To capture the quality of the promoter sequence of interest we introduced two measures:
the affinity of the designated TF to the sequence as well as the impact of the position of the
binding site in the sequence.
The affinity of a TF to the sequence of interest was calculated with TRAP (TRanscription factor
Affinity Prediction) which predicts a relative binding affinity to a DNA sequence using a physical binding model <A href="/Team:Heidelberg/HEARTBEAT_network#References" title="Team:Heidelberg/HEARTBEAT network">[17]</A>.
The position of the binding site of a TF was assessed using its probability density function which was derived from the HEARTBEAT database as <A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">described previously</A>. 
A sliding window of 20bp was shifted over the binding distribution of the transcription factor to determine its preferential binding region. The area under the curve (AUC) at the determined position was then scaled according to the number of total hits of the particular transcription factor. 
In order to further evaluate the significance of a certain TF binding site, we first calculated the difference between the AUC of interest and the mean over all values (ΔAUC(x)).
We then calculated the difference between the maximal AUC value and the mean over all values (ΔAUC(max)) and finally estimated the quotient (Box.1).
By this, we were able to characterize a promoter of interest from two different aspects, biophysical affinity and spatial distance to the TSS, respectively.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tnone"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Box 1:</B> Calculating the position score (PosScore)</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Model_assumptions"> Model assumptions </SPAN></H4><P>While constructing the model, we made several assumptions which our model is based on.
First of all, we choose JeT as a established methodological reference for our data analysis 
prior to model development.
We furthermore assume that a high TF affinity corresponds to a high promoter activity. At the same time, we assume that a high TFBS position score also correlates with a high promoter activity.
A system describing an inducible promoter cannot be invoked if either affinity or position score is low. Upon stimulation, promoter having such characteristics should not gain a high activity. Obviously their basal activity is considered not to be high as well. We also consider that the optimal transcriptional activation of a promoter of interest cannot be reached by a single high score. For this purpose, both scores have to be at a high range.
</P><P>Regarding the scoring of the promoter sequences, the main problem which had to be overcome was the integration of single scores to one overall promoter score. TRAP calculates one score which describes the affinity of one TF to a given sequence. The position score, moreover, is calculated for each single TFBS of each TF on a given sequence. We decided to choose the maximum of the scores (&quot;<I>pos.max.score</I>&quot; or &quot;<I>af.max.score</I>&quot;) because:
</P><UL><LI> it reflects our observations that, when compared to JeT, CMV:JeT has a lower transcriptional activity,
</LI><LI> it is consistent with the assumption that constitutive sequences should show a good correlation between affinity/position score and their promoter activity,
</LI><LI> it corresponds with the observation that inducible promoters reach their overall maximum activity upon (strong) induction and
</LI><LI> the &quot;<I>max.scores</I>&quot; apply for all of our observations regardless of the type of the transcription factor.
</LI></UL><P>At this point it is indispensable to mention that we assume that a better binding of a TF to a promoter sequence leads to a enhanced transcrptional activity.
Since the promoter activity was captured by fluorescence, we assume
that the increase in transcription of target genes (which in our case were coding for the fluorescent proteins) should lead to a higher expression of the respective protein as well.
</P><P>For the last part of our modeling, the combination of distinct pathways, the different systems are integrated by either applying a fuzzy inference system or simply calculating cumulative activity.
</P><H4><SPAN class="mw-headline" id="Defining_Fuzzy_Membership_Functions_.26_Fuzzy_Rules:_capturing_the_behavior_of_our_experimental_data"> Defining Fuzzy Membership Functions &amp; Fuzzy Rules: capturing the behavior of our experimental data </SPAN></H4><P>As a first step to create a model using fuzzy logic, experimental data had to be studied in order to define several so-called membership functions. Based on data analysis techniques we defined 13 classes in which our data could be clustered due to differentail activity for a total of 5 variables. 
In Fuzzy Logic, each class requires a membership function that accurately defines its behavior. Therefore we established as many membership functions (MF) as classes for all inputs. These classes were characterized as follows:
</P><UL><LI> Variable #1: position score (&quot;<I>PosScore</I>&quot;). We caracterized 3 clusters ,i.e. low, medium and high
</LI><LI> MFs for the following classes: Low = 0.0-0.8; Medium = 0.8-0.9; High = 0.9-1.0. 
</LI></UL><LI> Variable #2: affinity score (&quot;<I>AfScore</I>&quot;). We caracterized 3 clusters,i.e. low, medium and high
</LI><LI> MFs for the following classes: Low = 0.0-0.1; Medium = 0.1-0.5; High = 0.5-2.0
</LI><LI>Variable #3: promoter type. In our approach, we establish two types of promoters, i.e. constitutively active and inducible.
</LI><LI>  MFs for the following classes: Constitutive = 0-0.5; Inducible = 0.5-1.0
</LI><LI> Variable #4: stimulus.  In our approach, we establish two systems, i.e. stimulus present or absent.
</LI><LI> MFs for the following classes: No stimulation = 0.0-0.5; Stimulation present = 0.5-1.0
</LI><LI> Variable #5: stimulus.  In our approach, we establish three concentrations of stimuli, i.e. stimulus absent, low concentration or high.
</LI><LI> MFs for the following classes: No stimulation = 0.0-0.; Low = 0.-0.; High = 0.-1.0 <B>(only for p53)</B></LI><P>As described above, we considered JeT as a well established medium active promoter (taken as basal activity here). Based on this assumption, our system should be able to predict decreased level of activity when compared to CMV:JeT as well as an increased activity when compared to CMV (see <A href="/Team:Heidelberg/Project_Measurement" title="Team:Heidelberg/Project Measurement"> Measurement</A>).
</P><P>Our model should be able to reflect the behaviour of at least two inducible NFκB-responsive sequences (I L 12, II L 10). Having different position an affinity scores, these can be considered as &quot;differential&quot; promoters since they are of the same promoter type and induced by the same stimulus (TNFα), thus corresponding to the same pathway.
</P><P>Our model should also be able to describe the behaviour of at least six constitutive (NFκB responsive) promoters (L1, L4, L5, S4, S5, S10). These can be also taken as &quot;differential&quot; because the only differ in their sequence scores.
</P><P>Also, the system should be capable of reproducing the activity characteristics of eight p53 responsive promoters (S6, S8, S9, S12, S24, L8, L19, L22). They do not only differ in their physical sequence scores like in the two case above, but also differ in the concentration of the stimulating agent.
</P><P>Taking all these assumptions into account, we defined our rules which is described in Tab. <B>1</B>. As an example for each single system the MATLAB fuzzy logic toolbox (FLT) rule viewer is displayed (Figs.<B>3-5</B>). The viewer shows the combination of the rules which are defined as an array which has the structure as follows:  [position score; affinity score; stimulation; inducibilty; NFkB-activity (only for p53)].
All rules are combined with an <B>AND</B> gate.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tnone"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption"><B>Table 1: Summary of the applied in this fuzzy network modeling. L = Low, M = Medium, H = High, VH = Very high</B></DIV></DIV></DIV></TD><TD><DIV class="thumb tnone"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption"><B>Figure 3: Rule viewer for the NFkB fuzzy inference system. For input parameters see text.</B></DIV></DIV></DIV></TD><TD><DIV class="thumb tnone"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption"><B>Figure 4: Rule viewer for the constitutive fuzzy inference system. Input parameters are  [0.9 1 1 1]</B></DIV></DIV></DIV></TD><TD><DIV class="thumb tnone"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption"><B>Figure 5: Rule viewer for the p53 fuzzy inference system. Input parameters are  [0.9643 0.3254 0.75 0.8 2.5]</B></DIV></DIV></DIV></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Combining_Exclusive_Pathways"> Combining Exclusive Pathways </SPAN></H4><P>To see how we combine several exclusive pathways see <A href="/Team:Heidelberg/HEARTBEAT_network/Results" title="Team:Heidelberg/HEARTBEAT network/Results">results</A>.
</P><P>[<A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_network" class="external text" rel="nofollow">TOP</A>]
</P><H2><SPAN class="mw-headline" id="Results"> Results </SPAN></H2><P><B>for details please visit our <A href="/Team:Heidelberg/HEARTBEAT_network/Results" title="Team:Heidelberg/HEARTBEAT network/Results">Results page</A>!</B></P><P>Based on our experimental measurements, subsequent data analysis as well as on our assumption we were able to design a network model which is based on fuzzy logic.
This mathematical model reflected the characteristics of all investigated promoter sequences. This model was able to describe the behaviour of three standard promoters (JeT, CMV, CMV:JeT), six constitutively active promoters as well as at least ten inducible promoters (two NFκB- and eight p53-responsive) (data not shown) and it was also able to characterize the joined effect of distinct promoter combinations. This section briefly summarizes our results. <A href="/Team:Heidelberg/HEARTBEAT_network/Results" title="Team:Heidelberg/HEARTBEAT network/Results">Please click here to read more about our results!</A></P><P>All in all, we successfully developed three individual fuzzy inference systems which captured the behaviour of different promoter types. Additionally, we were able to create a dynamic model by joining our three single fuzzy logic submodules together to one whole global system. We showed several interesting characteristics of the system including a pathway crosstalk and a positive feedback. We thus conclude that our model is capable of reflecting not only basic gene regulatory modules but also complex behaviours which integrate a variety of gene regulatory elements. Here are some nice pictures picked up from our results. Have fun!
</P><P>[<A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_network" class="external text" rel="nofollow">TOP</A>]
</P><H2><SPAN class="mw-headline" id="Discussion"> Discussion </SPAN></H2><P>Contributing to the overall <B>HEARTBEAT</B> project, <B>HEARTBEAT fuzzy network</B> (FN) modeling focuses on simulating the promoter activity by integrating a variety of signals and sequence characteristics as well as on predicting distinct pathway functionalities. For this purpose we propose fuzzy logic (FL) modeling as an approach to logic-based modeling which is capable of incorporating qualitative data but producing quantitative predictions. We successfully developed three individual fuzzy inference systems which captured the behaviour of different promoter types. Furthermore, we designed a dynamic network model by joining our three single fuzzy inference systems together to one global model. We were able to show several interesting characteristics of the system including a pathway crosstalk and a positive feedback. We thus conclude that our model is capable of reflecting not only basic gene regulatory modules but also complex behaviours which integrate a variety of gene regulatory elements.
</P><P><B>Biological theories</B> are often written in natural language <A href="/Team:Heidelberg/HEARTBEAT_network#References" title="Team:Heidelberg/HEARTBEAT network">[15]</A>. The rules are of a descriptive nature. Science greatly benefits from these verbal explanations since computing with words exploits the tolerance for imprecision and thereby lowers the cost of solution. The main task of applying Fuzzy Logic (FL) is the definition of input parameters, fuzzy membership functions (MFs) and Fuzzy rules. Obviously our model is not perfect as it shows a slight decrease after reaching its peak for a fully stimulated stimulus (<A href="/wiki/index.php?title=Team:Heidelberg/HEARTBEAT_modeling/Results&amp;action=edit&amp;redlink=1" class="new" title="Team:Heidelberg/HEARTBEAT modeling/Results (page does not exist)">Results, Fig. 15</A>) whereas we assumed a saturation instead. This can be traced back to the lack of a big pool of valuable experimental results. Thus, we could not describe every very single behaviour of the system according to our rules derived from data analysis prior to model definition. In addition, we had to assume some rules which describe the basal behaviour of analyzed promoter modules. Nonetheless, we succeeded in creating a dynamic model using FL and this is a major strength of FL-based modeling. By generating more experimental results in foreseeable future, we will be able to improve and optimize our model, thus being able to predict the behaviour of gene regulatory systems with a enhanced accuracy.
</P><P>Another obstacle was the scoring of the sequences. Our choice of the way of scoring, even though it was able to be integrated into the model functionally, has to be reconsidered and supported by biological reasons besides the spatial preference as well as the biophysical affinity.
</P><P>Examining HEARTBEAT FN from the biological point of view, we could successfully show a pathway interaction.  A interlink in between the single invoked system was incorporated terms of a feedback which considered NFκB activity to be an additional input for the p53 system. Based upon intense literature research we were able to define rules which were of vague nature. However, our results captured the available biological data and reproduced our assumptions as well.
</P><P>A high potential is seen if we want to improve our model from the biological point of view. Regarding the case discussed above, we could define more accurate rules in order to capture distinct system behaviours. Experimental results will help characterizing the mode of action, thus revealing mechanistic insights into gene regulatory networks or into the world of signal transduction by detecting any thresholds for a given excitation. Further molecular mechanisms can be included, e.g. the effect of stimulants on constitutive promoters or the relationship between transcriptional activity and functional protein expression which is by now regarded as a black box. While single activity correlates with our experimental observation, our prediction of combined activity needs to be validated in the future. This can be e.g. done by showing the accuracy of the model by calculating the error via the least square method.
</P><P>HEARTBEAT FN, for our future, is a very promising tool to capture not only different promoter designs which is already captured in our model design, but also different experimental measurements what we aimed at the beginning of the project (include western blotting, real time PCR,  co-immunoprecipitation, etc.). This model should also be able to reflect different experimental setups (an easy example would be including different incubation time and treatments), dynamic activity of the promoter, posttranscriptional modifications or even gene silencing effects.
</P><P>A manifold possibility for visualize the integrated system, as well as a huge potential of further application of HEARBEAT FN as well as HEARTBEAT in gene therapy as well as virotherapy and makes it tremendously interesting for future progress in the research field of systems as well as synthetic biology . HEARTBEAT uses differential scores and stimulation levels which can be used as characterization of new biobrick parts. By this, our modeling project will have a high potential for contribution to synthetic biology by providing a integrative module including data analysis (<A href="/Team:Heidelberg/HEARTBEAT_database" title="Team:Heidelberg/HEARTBEAT database">HEARTBEAT DB</A>), a graphical user interface (<A href="/Team:Heidelberg/HEARTBEAT_GUI" title="Team:Heidelberg/HEARTBEAT GUI">HEARTBEAT GUI</A>) for designing your own promoters of your interest, and this <A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_network" class="external text" rel="nofollow">HEARTBEAT fuzzy network (FN)</A> modeling. To try out our project on our own we have already designed <A href="/Team:Heidelberg/Project_Synthetic_promoters" title="Team:Heidelberg/Project Synthetic promoters">synthetic promoter</A> sequences, of which the first experimental results have just arrived at the timepoint of iGEM wiki closure due to complication during synthesis as well as shipping of the promoter sequences. The <A href="/Team:Heidelberg/Team" title="Team:Heidelberg/Team">iGEM Team Heidelberg 09</A> is looking forward to validate and verify our modeling concept so we can finally estimate the impact of our HEARTBEAT project.
</P><P>[<A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_network" class="external text" rel="nofollow">TOP</A>]
</P><H2><SPAN class="mw-headline" id="References"> References </SPAN></H2><P>[1] Harbison C. T., Gordon D. B., Lee T. I., Rinaldi N. J., Macisaac K. D., Danford T. W., Hannett N. M., Tagne J. B., Reynolds D. B., Yoo J., Jennings E. G., Zeitlinger J., Pokholok D. K., Kellis M., Rolfe P. A., Takusagawa K. T., Lander E. S., Gifford D. K., Fraenkel E., Young R. A. Transcriptional regulatory code of a eukaryotic genome. <I>Nature</I> 431: 99-104 (2004).</P><P>[2] Hu Z., Killion P. J., Iyer, V. R. Genetic reconstruction of a functional transcriptional regulatory network. <I>Nature Genet.</I> 39: 683-687 (2007).</P><P>[3] Gertz J., Siggia E. D., Cohen B. A. Analysis of combinatorial <I>cis</I>-regulation in synthetic and genomic promoters. <I>Nature</I> 457: 215-218 (2009)</P><P>[4] Roider H. G., Kanhere A., Manke T., Vingron M. Predicting transcription factor affinities to DNA from a biophysical model.<I> Bioinformatics</I> 23: 134-141 (2006)</P><P>[5] Carrera J., Rodrigo G., Jaramillo A. Towards the automated engineering of a synthetic genome. <I>Mol. Biosyst.</I> 5: 733-43 (2009).</P><P>[6] Agapakis C. M., Silver P. A. Synthetic biology: exploring and exploiting genetic modularity through the design of novel biological networks. <I>Mol. Biosyst.</I> 5: 704-13 (2009)</P><P>[7] Purnick P. E., Weiss R. The second wave of synthetic biology: from modules to systems. <I>Nat Rev Mol Cell Biol.</I> 10: 410-22 (2009).</P><P>[8] Bhalerao K. D. Synthetic gene networks: the next wave in biotechnology? <I>Trends Biotechnol</I>. 27: 368-74 (2009).</P><P>[9] Andrianantoandro E., Basu S., Karig D. K., Weiss R. Synthetic biology: new engineering rules for an emerging discipline. <I>Mol Sys Biol</I> (2006)</P><P>[10] Alberts B., Johnson A., Walter P., Lewis J. Molecular Biology of the Cell, 5th edition, 2008. Garland Science, Chapter 6</P><P>[11] Vardhanabhuti S., Wang J., Hannenhalli S. Position and distance specificity are important determinants of <I>cis</I>-regulatory motifs in addition to evolutionary conservation. <I>Nucl Acid Res</I> 35: 3203-3213 (2007).</P><P>[12] Yokoyama K. D., Ohler U., Wray G. A. Measuring spatial preferences at fine-scale resolution identifies known and novel <I>cis</I>-regulatory element candidates and functional motif-pair relationships. <I>Nucl Acid Res</I> 37: e92 (2009)</P><P>[13] Nelles O. <I>Nonlinear System Identification</I> Springer Verlag GmbH &amp; Co., Berlin, 2000. </P><P>[14] Bosl W. J. Systems biology by the rules: hybrid intelligent systems for pathway modeling and discovery. 
<I>BMC Systems Biology</I>1:13 (2007).</P><P>[15] Laschov D., Margaliot M. Mathematical modeling of the lambda switch:a fuzzy logic approach. <I>J Theor Biol.</I>   21:475-89 (2009) </P><P>[16] Aldridge B. B., Saez-Rodriguez J., Muhlich  J. L., Sorger P. K., Lauffenburger D. A. Fuzzy logic analysis of kinase pathway crosstalk in TNF/EGF/insulin-induced signaling <I>PLoS Comput Biol.</I>5:e1000340 (2009).
</P><P>[17] Roider H. G., Kanhere A., Manke T., Vingron M. Predicting transcription factor affinities to DNA from a biophysical model. <I>Bioinformatics</I> 23: 134-141 (2007).
</P><P>[<A href="https://2009.igem.org/Team:Heidelberg" class="external text" rel="nofollow">TOP</A>]
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Heidelberg/HEARTBEAT_network">http://2009.igem.org/Team:Heidelberg/HEARTBEAT_network</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/HEARTBEAT_network" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/HEARTBEAT_network" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/HEARTBEAT_network&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/HEARTBEAT_network&amp;oldid=168975" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>