<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Project_References"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Heidelberg/Project_References">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Project_References&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project_References&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project_References&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Project_References" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Project References</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="main_wrapper"><DIV id="header"><DIV id="navigation"><DIV id="menu" style="position:static"><UL><LI id="mainHome"><A class="aMain" href="https://2009.igem.org/Team:Heidelberg">Home</A></LI><LI id="mainTeam"><A class="aTeam" href="https://2009.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Members#Students">Students</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Members#Advisors">Advisors</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Scientific_Environment">Scientific Environment</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Gallery">Gallery</A></LI></UL><LI id="mainProject"><A class="aProject" href="https://2009.igem.org/Team:Heidelberg/Project">Project</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project">Abstract</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Introduction">Introduction</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Measurement">Measurement</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Synthetic_promoters">Synthetic Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Further">Further Subprojects</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_SaO">Summary &amp; Outlook</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_References">References</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Acknowledgement">Acknowledgements</A></LI><LI id="mainParts"><A class="aParts" href="https://2009.igem.org/Team:Heidelberg/Parts">Parts</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Parts">Parts</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Parts_Characterization">Characterization</A></LI><LI id="mainModeling"><A class="aModeling" href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">What is HEARTBEAT?</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_database">HEARTBEAT Database</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_GUI">HEARTBEAT GUI</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_gui">GUI Documentation</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_network">HEARTBEAT Fuzzy Network Modeling</A></LI><LI id="mainNotebook"><A class="aNotebook" href="https://2009.igem.org/Team:Heidelberg/Notebook">Notebook</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_natural_promoters_overview">Natural Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_synthetic_promoters">Synthetic Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_promoters_cells">Cell Culture, Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_cell_line">Stable Cell Line</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_color_output">Multi-Color Output</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_modeling">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_measure">Measurements</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_MaM">Material &amp; Methods</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_meetings">Team Meetings</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Biosafety">Biosafety</A></LI><LI id="mainEucaryopedia"><A class="aEucaryopedia" href="https://2009.igem.org/Team:Heidelberg/Eukaryopedia">Eukaryopedia</A></LI><LI id="mainSponsors"><A class="aSponsors" href="https://2009.igem.org/Team:Heidelberg/Sponsors">Sponsors</A></LI></DIV></DIV></DIV></DIV><TABLE border="0" style="margin-left: 2px;"><TBODY><TR><TD width="930px" style="padding: 0 15px 15px 20px; background-color:#ede8e2"><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><H2><SPAN class="mw-headline" id="Introduction"> Introduction </SPAN></H2><DIV style="text-align:justify;"><UL><LI> &quot;Synthetic Biology. Applying Engineering to Biology&quot;. Report of a NEST High-Level Expert Group. European Commission, Directorate-General for Research. Available online at <A href="http://www.synbiosafe.eu/uploads///pdf/EU-highlevel-syntheticbiology.pdf" class="external free" rel="nofollow">http://www.synbiosafe.eu/uploads///pdf/EU-highlevel-syntheticbiology.pdf</A></LI></UL><UL><LI> Endy, D. Foundations for engineering biology. <I>Nature</I> 438: 449-453 (2005).
</LI></UL><UL><LI> Endy, D. &amp; Deese I. Adventures in synthetic biology (comic). <I>Nature</I> 438: 449-453 (2005) Available online at <A href="http://www.nature.com/nature/comics/syntheticbiologycomic/index.html" class="external text" rel="nofollow">nature.com</A></LI></UL><UL><LI><A href="http://www.partsregistry.org/Assembly:Standard_assembly" class="external free" rel="nofollow">http://www.partsregistry.org/Assembly:Standard_assembly</A> (Website, accessed 10/15/2009)
</LI></UL><UL><LI> Kelly J. R., Rubin A. J., Davis J. H., Ajo-Franklin C. M., Cumbers J., Czar M. J., de Mora K., Glieberman A. L., Monie D. D. &amp; Endy D. Measuring the activity of BioBrick promoters using an <I>in vivo</I> reference standard. <I>Journal of Biological Engineering</I> 3:4 (2009).
</LI></UL><UL><LI> Voigt, C. Genetic parts to program bacteria. <I>Current Opinion in Biotechnology</I> 17 (5): 548–557 (2006).
</LI></UL><UL><LI> Andrianantoandro E., Basu S., Karig D. K. &amp; Weiss R. Synthetic biology: new engineering rules for an emerging discipline. <I>Molecular Systems Biology</I> 2: article number 0028 (2006).
</LI></UL><UL><LI><A href="https://2009.igem.org/About" class="external free" rel="nofollow">https://2009.igem.org/About</A> (Website, accessed 10/15/2009)
</LI></UL><UL><LI> Goodman, C. Engineering ingenuity at iGEM. <I>Nat Chem Biology</I> 4:13 (2008).
</LI></UL><UL><LI> Basu, S. A synthetic multicellular system for programmed pattern formation. <I>Nature</I> 434: 1130-1133 (2005).
</LI></UL><UL><LI> Ellis T., Wang X. &amp; Collins J.J. Diversity-based, model-guided construction of synthetic gene networks with predicted functions. <I>Nature Biotechnology</I> 27: 465-471 (2009).
</LI></UL><UL><LI> Canton, B., Anna Labno A. &amp; Endy D. Refinement and standardization of synthetic biological parts and devices. <I>Nature Biotechnology</I> 26: 787-793 (2009).
</LI></UL><UL><LI> Weber, W. &amp; Fussenegger, M. Engineering of Synthetic Mammalian Gene Networks. <I>Chemistry and Biology</I> 16: 287-297 (2009).
</LI></UL><UL><LI><A href="http://openwetware.org/wiki/The_BioBricks_Foundation:Standards/Technical/Formats" class="external free" rel="nofollow">http://openwetware.org/wiki/The_BioBricks_Foundation:Standards/Technical/Formats</A> (Website, accessed 10/19/2009)
</LI></UL><UL><LI> Ducrest A-L., Amacker M., Lingner J. &amp; Nabholz M. Detection of promoter activity by flow cytometric analysis of GFP reporter expression. <I>Nucleic Acids Res.</I> 30: e65 (2002). 
</LI></UL><UL><LI> Venter M. Synthetic promoters: genetic control through cis engineering. <I>Trends in Plant Science</I> 12: 118-124 (2007). (and the references cited therein) 
</LI></UL><UL><LI> Ogawa R. Construction of strong mammalian promoters by random cis-acting element elongation. <I>Biotechniques</I> 42: 628-632 (2007). 
</LI></UL><UL><LI> Tornoe J. Generation of a synthetic mammalian promoter library by modification of sequences spacing transcription factor binding sites. <I>Gene</I> 297: 21-32 (2002). 
</LI></UL><UL><LI> Dorer D. E. &amp; Nettelbeck D. Targeting cancer by transcriptional control in cancer gene therapy and viral oncolysis. <I>Advanced Drug Delivery Reviews</I> 61: 554-557 (2009). 
</LI></UL><UL><LI> Weber W. &amp; Fussenegger M. The impact of synthetic biology on drug discovery. <I>Drug Discovery Today</I> 14: 956-963 (2009).
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Natural_Promoters">Natural Promoters</SPAN></H2><DIV style="text-align:justify;"><UL><LI> Alberts B., Johnson A., Walter P. &amp; Lewis J. <I>Molecular Biology of the Cell</I>. 4th edition, 2008. Garland Science, Chapter 6, p. 357-359. </LI></UL><UL><LI> Clarke N., Arenzana N., Hai T., Minden A. &amp; Prywes R. Epidermal Growth Factor Induction of the c-jun Promoter by a Rac Pathway. <I>Mol Cell Biol</I> 18: 1065–1073 (1998).</LI></UL><UL><LI> Hoffman L. M., Garcha K., Karamboulas K., Cowan M. F., Drysdale L. M., Horton W. A. &amp; Underhill T. M. BMP action in skeletogenesis involves attenuation of retinoid signaling. <I>J Cell Biol</I> 174: 101–113 (2006). </LI></UL><UL><LI> Castoreno A. B., Wang Y., Stockinger W., Jarzylo L. A., Du H., Pagnon J. C., Shieh E. C. &amp; Nohturfft A. Transcriptional regulation of phagocytosis-induced membrane biogenesis by sterol regulatory element binding proteins. <I>Proc Natl Acad Sci U S A</I> 102: 13129–13134 (2005). </LI></UL><UL><LI> Schmidt M., Heimberger T., Gruensfelder P., Schler G. &amp; Hoppe F. Inducible promoters for gene therapy of head and neck cancer: an in vitro study. <I>Eur Arch Otorhinolaryngol</I> 261: 208–215 (2004).</LI></UL><UL><LI> Jorgensen E. C. B. &amp; Autrup H. Autoregulation of human CYP1A1 gene promoter activity in HepG2 and MCF-7 cells. <I>Carcinogenesis</I> 17: 435-441 (1996). </LI></UL><UL><LI> Schreiber T. D., Köhle C., Buckler F., Schmohl S., Braeuning A., Schmiechen A., Schwarz M. &amp; Münzel P.A. Regulation of CYP1A1 expression by the antioxidant Tert butylhydroquinone. <I>Drug Metabolism And Disposition</I> 43: 1096–1101 (2006).
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Synthetic_Promoters">Synthetic Promoters</SPAN></H2><DIV style="text-align:justify;"><UL><LI> Alberts B., Johnson A., Walter P. &amp; Lewis J. <I>Molecular Biology of the Cell</I>. 4th edition, 2008. Garland Science, p. 432-453 
</LI></UL><UL><LI> Fussenegger M. &amp; Weber, W. Engineering of Synthetic Mammalian Gene Networks. <I>Chemistry and Biology</I> 16: 287-297 (2009).
</LI></UL><UL><LI> Gossen M. &amp; Bujard, L. Tight control of gene expression in mammalian cells by tetracycline-responsive promoters. <I>PNAS</I> 89: 5547-5551 (1992).
</LI></UL><UL><LI> Dorer D. E. &amp; Nettelbeck, D. Targeting cancer by transcriptional control in cancer gene therapy and viral oncolysis. <I>Advanced Drug Delivery Reviews</I> 61: 554-557 (2009).
</LI></UL><UL><LI> Venter M. Synthetic promoters: genetic control through cis engineering. <I>Trends in Plant Science</I> 12: 118-124 (2007). (and the references cited therein)
</LI></UL><UL><LI> Rushton P. J., Reinstädler A., Lipka V., Lippok B. &amp; Somssich I. E. Synthetic plant promoters containing defined regulatory elements provide novel insights into pathogen- and wound-induced signalling. <I>Plant Cell</I> 14: 749–762 (2002).
</LI></UL><UL><LI> Ogawa R. Construction of strong mammalian promoters by random cis-acting element elongation. <I>Biotechniques</I> 42: 628-632 (2007). 
</LI></UL><UL><LI> Edelman G. M., Meech R., Owens G. C. &amp; Jones F. S. Synthetic promoter elements obtained by nucleotide sequence variation and selection for activity. <I>PNAS</I> 97: 3038-43 (2000).
</LI></UL><UL><LI> Ellis T., Wang X. &amp; Collins J. J. Diversity-based, model-guided construction of synthetic gene networks with predicted functions. <I>Nature Biotechnology</I> 27: 465-471 (2009).
</LI></UL><UL><LI> Tornoe J. Generation of a synthetic mammalian promoter library by modification of sequences spacing transcription factor binding sites. <I>Gene</I> 297: 21-32 (2002).
</LI></UL><UL><LI> Heintzman N. D. &amp; Ren B. The gateway to transcription: identifying, characterizing and understanding promoters in the eukaryotic genome. <I>Cellular and Molecular Life Science</I> 64: 386-400 (2007).
</LI></UL><UL><LI> Stemmer W. P., Crameri A., Ha K. D., Brennan T. M. &amp; Heyneker H. L. Single-step assembly of a gene and entire plasmid from large numbers of oligodeoxyribonucleotides. <I>Gene</I> 164: 49-53 (1995).
</LI></UL><UL><LI> Rattner A. NF-κB activates the HIV promoter in neurons. <I>EMBO</I> 12: 4261–4267 (1993).
</LI></UL><UL><LI> Yokoyama D. K., Ohler U. &amp; Wray G. A. Measuring spatial preferences at fine-scale resolution identifies known and novel cis-regulatory element candidates and functional motif-pair relationships. <I>Nuc Acids Res</I>: 1-21 (2009).
</LI></UL><UL><LI> del Val C., Pelz O., Glatting K-H, Barta E. &amp; Hotz-Wagenblatt A. PromoterSweep: a tool for identification of transcription factor binding site. <I>Theor Chem Acc</I> (in press)
</LI></UL><UL><LI> Ghosh S., May M. J. &amp; Kopp E. B. Rel/NF-κB and IKB proteins: an overview. <I>Seminars in Cancer Biology</I> 8: 63-73 (1997). 
</LI></UL><UL><LI> Moyano P. &amp; Rotwein P. Mini-review: estrogen action in the uterus and insulin-like growth factor-I. <I>Growth Hormone &amp; IGF Research</I> 14: 431-435 (2009).
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Stable_cell_line">Stable cell line</SPAN></H2><DIV style="text-align:justify;"><UL><LI> Zhou H. Development of site-specific integration system to high-level expression recombinant proteins in CHO cells. <I>Chinese journal of biotechnology</I> 23: 756-62 (2007). 
</LI></UL><UL><LI> Andrews B. J., Proteau G. A., Beatty L. G. &amp; Sadowski P. D. The FLP Recombinase of the 2μ Circle DNA of Yeast: Interaction with Its Target Sequences. <I>Cell</I> 40: 795-803 (1985). 
</LI></UL><UL><LI> Senecoff J. F., Bruckner R. C. &amp; Cox M. M. The FLP recombinase of the yeast 2-μ plasmid: Characterization of its recombination site. <I>Proc. Nati. Acad. Sci. USA</I> 82: 7270-7274 (1985). 
</LI></UL><UL><LI> Holmberg A., Blomstergren A., Nord O., Lukacs M., Lundeberg J. &amp; Uhlén M. The biotin-streptavidin interaction can be eversibly broken using water at elevated temperatures. <I>Electrophoresis</I> 26(3): 501-10 (2005). 
</LI></UL><UL><LI> Schmidt M., Schwarzwaelder K., Bartholomae C., Zaoui K., Ball B., Pilz I., Braun S., Glimm H., von Kalle C., High-resolution insertion-site analysis by linear amplification–mediated PCR (LAM-PCR). <I>Nature Methods</I> 4 (12): 1051-1057 (2005).
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Outlook_and_Summary">Outlook and Summary</SPAN></H2><DIV style="text-align:justify;"><UL><LI> Venter M. Synthetic promoters: genetic control through cis engineering. <I>Trends in Plant Science</I> 12: 118-24 (2007).
</LI><LI> Carey M., Smale S. T. &amp; Hughes H., Transcriptional Regulation in Eukaryotes: Concepts, Strategies and Techniques. <I>New York: CSHL</I>: p. 18-25 (2000).
</LI></UL></DIV><H2><SPAN class="mw-headline" id="HEARTBEAT_Database_.26_Fuzzy_Modeling">HEARTBEAT Database &amp; Fuzzy Modeling</SPAN></H2><DIV style="text-align:justify;"><H2><SPAN class="mw-headline" id="References_2"> References </SPAN></H2><UL><LI> Harbison C. T., Gordon D. B., Lee T. I., Rinaldi N. J., Macisaac K. D., Danford T. W., Hannett N. M., Tagne J. B., Reynolds D. B., Yoo J., Jennings E. G., Zeitlinger J., Pokholok D. K., Kellis M., Rolfe P. A., Takusagawa K. T., Lander E. S., Gifford D. K., Fraenkel E., Young R. A. Transcriptional regulatory code of a eukaryotic genome. <I>Nature</I> 431: 99-104 (2004).</LI></UL><UL><LI> Hu Z., Killion P. J., Iyer, V. R. Genetic reconstruction of a functional transcriptional regulatory network. <I>Nature Genet.</I> 39: 683-687 (2007).</LI></UL><UL><LI> Gertz J., Siggia E. D., Cohen B. A. Analysis of combinatorial <I>cis</I>-regulation in synthetic and genomic promoters. <I>Nature</I> 457: 215-218 (2009)</LI></UL><UL><LI> Roider H. G., Kanhere A., Manke T., Vingron M. Predicting transcription factor affinities to DNA from a biophysical model.<I> Bioinformatics</I> 23: 134-141 (2006)</LI></UL><UL><LI> Carrera J., Rodrigo G., Jaramillo A. Towards the automated engineering of a synthetic genome. <I>Mol. Biosyst.</I> 5: 733-43 (2009).</LI></UL><UL><LI> Agapakis C. M., Silver P. A. Synthetic biology: exploring and exploiting genetic modularity through the design of novel biological networks. <I>Mol. Biosyst.</I> 5: 704-13 (2009)</LI></UL><UL><LI> Purnick P. E., Weiss R. The second wave of synthetic biology: from modules to systems. <I>Nat Rev Mol Cell Biol.</I> 10: 410-22 (2009).</LI></UL><UL><LI> Bhalerao K. D. Synthetic gene networks: the next wave in biotechnology? <I>Trends Biotechnol</I>. 27: 368-74 (2009).</LI></UL><UL><LI> Andrianantoandro E., Basu S., Karig D. K., Weiss R. Synthetic biology: new engineering rules for an emerging discipline. <I>Mol Sys Biol</I> (2006)</LI></UL><UL><LI> Alberts B., Johnson A., Walter P., Lewis J. Molecular Biology of the Cell, 5th edition, 2008. Garland Science, Chapter 6</LI></UL><UL><LI> Vardhanabhuti S., Wang J., Hannenhalli S. Position and distance specificity are important determinants of <I>cis</I>-regulatory motifs in addition to evolutionary conservation. <I>Nucl Acid Res</I> 35: 3203-3213 (2007).</LI></UL><UL><LI> Yokoyama K. D., Ohler U., Wray G. A. Measuring spatial preferences at fine-scale resolution identifies known and novel <I>cis</I>-regulatory element candidates and functional motif-pair relationships. <I>Nucl Acid Res</I> 37: e92 (2009)</LI></UL><UL><LI> Nelles O. <I>Nonlinear System Identification</I> Springer Verlag GmbH &amp; Co., Berlin, 2000. </LI></UL><UL><LI> Bosl W. J. Systems biology by the rules: hybrid intelligent systems for pathway modeling and discovery. 
</LI></UL><P><I>BMC Systems Biology</I>1:13 (2007).</P><UL><LI> Laschov D., Margaliot M. Mathematical modeling of the lambda switch:a fuzzy logic approach. <I>J Theor Biol.</I>   21:475-89 (2009) </LI></UL><UL><LI> Aldridge B. B., Saez-Rodriguez J., Muhlich  J. L., Sorger P. K., Lauffenburger D. A. Fuzzy logic analysis of kinase pathway crosstalk in TNF/EGF/insulin-induced signaling <I>PLoS Comput Biol.</I>5:e1000340 (2009).
</LI></UL><UL><LI> Roider H. G., Kanhere A., Manke T., Vingron M. Predicting transcription factor affinities to DNA from a biophysical model. <I>Bioinformatics</I> 23: 134-141 (2007).
</LI></UL><UL><LI><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling" class="external text" rel="nofollow">BCCS-Bristol: Best Model 2008</A></LI></UL><UL><LI> Heintzman N. D. &amp; Ren B. The gateway to transcription: identifying, characterizing and understanding promoters in the eukaryotic genome. <I>Cellular and Molecular Life Science</I> 64: 386-400 (2007).
</LI></UL><UL><LI> Vardhanabhuti, S., Wang, J. &amp; Hannenhalli, S. Position and distance specificity are important determinants of <I>cis</I>-regulatory motifs in addition to evolutionary conservation. <I>Nucl Acid Res</I> 35(10): 3203-3213 (2007).</LI></UL><UL><LI> Yokoyama, K. D., Ohler, U. &amp; Wray, G. A. Measuring spatial preferences at fine-scale resolution identifies known and novel <I>cis</I>-regulatory element candidates and functional motif-pair relationships. <I>Nucl Acid Res</I> 37(13): e92 (2009)</LI></UL><UL><LI> Ellis T., Wang X. &amp; Collins J. J. Diversity-based, model-guided construction of synthetic gene networks with predicted functions. <I>Nature Biotechnology</I> 27: 465-471 (2009).
</LI></UL><UL><LI> Kent W. J., Sugnet C. W., Furey T. S., Roskin K. M., Pringle T. H., Zahler A. M. &amp; Haussler D. The human genome browser at UCSC. <I>Genome Res.</I> 12(6): 996-1006 (2002).
</LI></UL><UL><LI> Kuhn R. M., Karolchik D., Zweig A. S., Wang T., Smith K. E., Rosenbloom K. R., Rhead B., Raney B. J., Pohl A., Pheasant M., Meyer L., Hsu F., Hinrichs A. S., Harte R. A., Giardine B., Fujita P., Diekhans M., Dreszer T., Clawson H., Barber G. P., Haussler D. &amp; Kent W. J. The UCSC Genome Browser Database: update 2009. <I>Nucleic Acids Res.</I> 37(Database issue): D755-61 (2009).
</LI></UL><UL><LI> Kanehisa M., Araki M., Goto S., Hattori M., Hirakawa M., Itoh M., Katayama T., Kawashima S., Okuda S., Tokimatsu T. &amp; Yamanishi, Y. KEGG for linking genomes to life and the environment. <I>Nucleic Acids Res.</I> 36 (Database issue): D480-4 (2008). 
</LI></UL><UL><LI> Kanehisa M., Goto S., Hattori M., Aoki-Kinoshita K.F., Itoh M., Kawashima S., Katayama T., Araki M. &amp; Hirakawa M. From genomics to chemical genomics: new developments in KEGG. <I>Nucleic Acids Res.</I> 34 (Database issue): D354-7 (2006).
</LI></UL><UL><LI> Kanehisa M. &amp; Goto S. KEGG: Kyoto Encyclopedia of Genes and Genomes. <I>Nucleic Acids Res.</I> 28(1): 27-30 (2000). 
</LI></UL><UL><LI> del Val C., Pelz O., Glatting K.-H., Barta E. &amp; Hotz-Wagenblatt, A. PromoterSweep: a tool for identification of transcription factor binding sites. Theor. Chem. Acc. DOI 10.1007: s00214-009-0643-8 (2009).
</LI></UL><UL><LI><A href="http://apr2006.archive.ensembl.org/info/software/compara/index.html" class="external text" rel="nofollow">Ensembl Compara (database)</A></LI></UL><P>Hubbard T., Andrews D., Caccamo M., Cameron G., Chen Y., Clamp M., Clarke L., Coates G., Cox T., Cunningham F., Curwen V., Cutts T., Down T., Durbin R., Fernandez-Suarez X. M., Gilbert J., Hammond M., Herrero J., Hotz H., Howe K., Iyer V., Jekosch K., Kahari A., Kasprzyk A., Keefe D., Keenan S., Kokocinsci F., London D., Longden I., McVicker G., Melsopp C., Meidl P., Potter S., Proctor G., Rae M., Rios D., Schuster M., Searle S., Severin J., Slater G., Smedley D., Smith J., Spooner W., Stabenau A., Stalker J., Storey R., Trevanion S., Ureta-Vidal A., Vogel J., White S., Woodwark C. &amp; Birney E. Ensembl. <I>Nucleic Acids Res.</I> 33 (Database issue): D447-D453 (2005). 
</P><UL><LI><A href="http://www.ncbi.nlm.nih.gov/homologene" class="external text" rel="nofollow">NCBI HomoloGene (database)</A></LI></UL><UL><LI><A href="http://doop.abc.hu/" class="external text" rel="nofollow">DoOP (database)</A></LI></UL><P>Barta E., Sebestyén E., Pálfy T. B., Tóth G., Ortutay C. P. &amp; Patthy L. DoOP: Databases of Orthologous Promoters, collections of clusters of orthologous upstream sequences from chordates and plants. <I>Nucleic Acids Res.</I> 33 (Database issue): D86-D90 (2004).
</P><UL><LI><A href="http://www.epd.isb-sib.ch/" class="external text" rel="nofollow">EPD (database)</A></LI></UL><P>Schmid C. D., Praz V., Delorenzi M., Périer R. &amp; Bucher P. The Eukaryotic Promoter Database EPD: the impact of in silico primer extension. <I>Nucleic Acids Res.</I> 32 (Database issue): D82-5 (2004).
</P><UL><LI><A href="http://dbtss.hgc.jp/" class="external text" rel="nofollow">DBTSS (database)</A> Wakaguri H., Yamashita R., Suzuki Y., Sugano S., Nakai K.
</LI></UL><P>DBTSS: database of transcription start sites, progress report 2008. <I>Nucleic Acids Res</I>.36(Database issue): D97-101 (2008)
</P><UL><LI><A href="http://bayesweb.wadsworth.org/gibbs/gibbs.html" class="external text" rel="nofollow">Gibbs MotifSampler Lawrence (database)</A> C. E., Altschul S. F., Boguski M. S., Liu J. S., Neuwald A.F., Wootton J. C.  Detecting subtle sequence signals: a Gibbs sampling strategy for multiple alignment. <I>Science</I> 262:208-214(1993)
</LI></UL><UL><LI><A href="http://jaspar.genereg.net/" class="external text" rel="nofollow">Jaspar Core Library (database)</A> Sandelin A., Alkema W., Engstrom P., Wasserman W. W., Lenhard B. JASPAR: an open-access database for eukaryotic transcription factor binding profiles.
</LI></UL><P><I>Nucleic Acids Res.</I> 32(Database issue): D91-4 (2004).
</P><UL><LI> Matys V., Kel-Margoulis O. V., Fricke E., Liebich I., Land S., Barre-Dirrie A.,
</LI></UL><P>Reuter I., Chekmenev D., Krull M., Hornischer K., Voss N., Stegmaier P.,
Lewicki-Potapov B., Saxel H., Kel A. E. &amp; Wingender E. TRANSFAC<B>®</B> and its module TRANSCompel<B>®</B>: transcriptional gene regulation in eukaryotes. <I>Nucleic Acids Res.</I> 34 (Database issue): D108-110 (2006).</P><UL><LI><A href="http://www.mysql.com/?bydis_dis_index=1" class="external text" rel="nofollow">Mysql</A></LI></UL><UL><LI> Kel A. E., Gössling E., Reuter I., Cheremushkin E., Kel-Margoulis O. V. &amp; Wingender E. MATCH: A tool for searching transcription factor binding sites in DNA sequences.<I> Nucleic Acids Res.</I> 31: 3576-3579 (2003).
</LI></UL><UL><LI> Hannenhalli S. &amp; Levy S. Predicting transcription factor synergism. <I>Nucleic Acids Res.</I> 30: 4278-84 (2002).
</LI></UL><UL><LI> FitzGerald P. C., Shlyakhtenko A., Mir A. A. &amp; Vinson C. Clustering of DNA sequences in human promoters.<I> Genome Res.</I> 14: 1562-74 (2004).
</LI></UL><UL><LI> Alberts B., Johnson A., Walter P. &amp; Lewis J. <I>Molecular Biology of the Cell.</I> 5th edition, 2008. Garland Science, Chapter 6
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Measurement"> Measurement</SPAN></H2><DIV style="text-align:justify;"><UL><LI> Kelly J. R., Rubin A. J., Davis J. H., Ajo-Franklin C. M., Cumbers J., Czar M. J., de Mora K., Glieberman A. L., Monie D. D. &amp; Endy D. Measuring the activity of BioBrick promoters using an <I>in vivo</I> reference standard. <I>Journal of Biological Engineering</I> 3 (2009).</LI></UL><UL><LI> Endy, D. &amp; Deese I. Adventures in synthetic biology (comic) <I>Nature</I> 438: 449-453 (2005). Available online at <A href="http://www.nature.com/nature/comics/syntheticbiologycomic/index.html" class="external text" rel="nofollow">nature.com</A></LI></UL><UL><LI> Alberts B., Johnson A., Walter P. &amp; Lewis J. <I>Molecular Biology of the Cell</I>. 5th edition, 2008. Garland Science, Chapter 6</LI></UL><UL><LI><A href="http://www.promega.com/tbs/tm058/tm058.html" class="external free" rel="nofollow">http://www.promega.com/tbs/tm058/tm058.html</A></LI></UL><UL><LI> Zhou H. Development of site-specific integration system to high-level expression recombinant proteins in CHO cells. <I>Chinese journal of biotechnology</I> 23: 756-62 (2007).
</LI></UL><UL><LI> Alberts B., Johnson A., Walter P. &amp; Lewis J. <I>Molecular Biology of the Cell</I>. 5th edition, 2008. Garland Science, Chapter 7, p. 467-476</LI></UL><UL><LI> Zhu X. D. &amp; Sadowski P. D. Cleavage-dependent Ligation by the FLP Recombinase. <I>J Biol Chem</I> 270: 23044-23054 (1995).</LI></UL><UL><LI> York J. D., Odom A. R., Murphy R., Ives E. B. &amp; Wente S. R. A phospholipase C-dependent inositol polyphosphate kinase pathway required for efficient messenger RNA export. <I>Science</I> 285: 96-100 (1999).</LI></UL><UL><LI> Alberts B., Johnson A., Walter P. &amp; Lewis J. <I>Molecular Biology of the Cell</I>. 5th edition, 2008. Garland Science) Chapter 7, p. 905-908</LI></UL><UL><LI> Tornoe, J. Generation of a synthetic mammalian promoter library by modification of sequences spacing transcription factor binding sites. <I>Gene</I> 297: 21-32 (2002). </LI></UL><UL><LI> Alberts B., Johnson A., Walter P. &amp; Lewis J. <I>Molecular Biology of the Cell</I>. 5th edition, 2008. Garland Science, Chapter 7, p. 492</LI></UL><UL><LI> Ducrest, A. L., Amacker M., Lingner J. &amp; Nabholz M. Detection of promoter activity by flow cytometric analysis of GFP reporter expression. <I>Nucleic Acids Res.</I> 30: e65 (2002).  </LI></UL><UL><LI> Rushton P. J., Reinstädler A., Lipka V., Lippok B. &amp; Somssich I. E. Synthetic plant promoters containing defined regulatory elements provide novel insights into pathogen- and wound-induced signalling. <I>Plant Cell</I> 14: 749-762 (2002).</LI></UL><UL><LI> Degenhardt T., Rybakova K. N., Tomaszewska A., Moné M. J., Westerhoff H. V., Bruggeman F. J. &amp; Carlberg C. Population-Level Transcription Cycles Derive from Stochastic Timing of Single-Cell Transcription. <I>Cell</I> 138: 489-501 (2009).</LI></UL><UL><LI> Lottspeich F. &amp; Engels J. W.: <I>Bioanalytik</I> (Book. 2nd edition, 2006. Spektrum Akademischer Verlag) Chapter 5, p. 95-96.
</LI></UL><UL><LI><A href="http://www1.qiagen.com/Products/RnaStabilizationPurification/RNeasySystem/RNeasyPlantMini.aspx" class="external free" rel="nofollow">http://www1.qiagen.com/Products/RnaStabilizationPurification/RNeasySystem/RNeasyPlantMini.aspx</A></LI></UL><UL><LI><A href="http://www1.qiagen.com/products/automation/qiacube.aspx" class="external free" rel="nofollow">http://www1.qiagen.com/products/automation/qiacube.aspx</A></LI></UL><UL><LI><A href="http://www1.qiagen.com/literature/render.aspx?id=398" class="external free" rel="nofollow">http://www1.qiagen.com/literature/render.aspx?id=398</A></LI></UL><UL><LI><A href="https://products.appliedbiosystems.com/ab/en/US/adirect/ab?cmd=catNavigate2&amp;catID=604109&amp;tab=DetailInfo" class="external free" rel="nofollow">https://products.appliedbiosystems.com/ab/en/US/adirect/ab?cmd=catNavigate2&amp;catID=604109&amp;tab=DetailInfo</A></LI></UL><UL><LI> Ormerod M.G.: <I>Flow cytometry: a practical approach</I> (Book. 3rd edition, 2000. Oxford University Press) Chapter 1, p. 7-21.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Material_and_Methods">Material and Methods</SPAN></H2><DIV style="text-align:justify;"><UL><LI> Brady N. Bioquant Heidelberg, Eils group; Personal communication. </LI><LI> Schilde J. Institute for human genetics at the University of Heidelberg; Personal communication.</LI><LI> Panigrahy D., Singer S., Shen L. Q., Butterfield C. E., Freedman D. A., Chen E. J., Moses M. A., Kilroy S., Duensing S., Fletcher C., Fletcher J. A., Hlatky L., Hahnfeldt P., Folkman J., Kaipainen A. PPARγ ligands inhibit primary tumor growth and metastasis by inhibiting angiogenesis. <I>Journal of clinical cancer</I> 110: 923-932 (2002) </LI><LI> Lucarelli E., Sangiorgi L., Maini V., Lattanzi G., Marmiroli S., Reggian M., Mordenti M., Gobbi G. A., Scrimieri F., Bertoja A. Z. &amp; Picci P. Troglitazione affects survival of human osteosarcoma cells. <I>International Journal of Cancer</I> 98 (3): 344-351 (2002) </LI><LI> Komarov P. G., Komarova E. A., Kondratov R. V., Christov-Tselkov K., Coon J. S., Chernov M. V., Gudkov A. V. A chemical inhibitor of p53 that protects mice from the side effects of cancer therapy. <I>Science</I> 285(5434): 1733-7 (1999) </LI><LI> Wang S. H., Liang C. T., Liu Y. W., Huang M. C., Huang S. C., Hong W. F., Su J. G. Crosstalk between activated forms of the aryl hydrocarbon receptor and glucocorticoid receptor. <I>Toxicology</I> 262(2): 87-97 (2009) </LI><LI> Diesing D., Cordes T., Fischer D., Diedrich K. &amp; Friedrich M. Vitamin D--metabolism in the human breast cancer cell line MCF-7. <I>Anticancer Res.</I> 26(4A): 2755-9 (2006) </LI><LI> Bartz F., Kern L., Erz D., Zhu M., Gilbert D., Meinhof T., Wirkner U., Erfle H., Muckenthaler M., Pepperkok R. &amp; Runz H. Identification of Cholesterol-Regulating Genes by Targeted RNAi Screening. <I>Cell Metabolism</I> 10: 63-75 (2009) </LI><LI> Runz H., Miura K., Weiss M. &amp; Pepperkok R. Sterols regulate ER-export dynamics of secretory cargo protein ts-O45-G. <I>The EMBO Journal</I> 25: 2953-2965 (2006) </LI><LI>  Treede I., Braun A., Jeliaskova P., Giese T., Füllekrug J., Griffiths G., Stremmel W. &amp; Ehehalt R. TNF-α-induced up-regulation of pro-inflammatory cytokines is reduced by phosphatidylcholine in intestinal epithelial cells. <I>BMC Gastroenterology</I> 9:53  (2009)
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Eukaryopedia">Eukaryopedia</SPAN></H2><DIV style="text-align:justify;"><UL><LI> Clark R. The process of malignant progression in human breast cancer. <I>Annals of oncology: official journal of the European Society for Medical Oncology/ESMO</I> 1: 401-407 (1990).
</LI></UL><UL><LI> Osborne C. K., Hobbs K. &amp; Trent J. M. Biological differences among, MCF-7 human breast cancer cell lines from different laboratories. <I>Breast Cancer Research and Treatment</I> 9: 111-121 (1987).
</LI></UL><UL><LI> Dickson R. B., Bates S. E., McManaway M. E. &amp; Lippman M. E. Characterization of Estrogen Responsive Transforming Activity in Human Breast Cancer Cell Lines. <I>Cancer Research</I> 46: 1707-1713 (1986).
</LI></UL><UL><LI> Booth B. W. &amp; Smith G. H. Roles of transforming growth factor-α in mammary development and disease. <I>Growth Factors</I> 925: 227-235 (2007).
</LI></UL><UL><LI> Gey G. O., Coffman W. D. &amp; Kubicek M. T. Tissue culture studies of the proliferative capacity of cervical carcinoma and norml epithelium. <I>Cancer Research</I> 12: 264-265 (1952). 
</LI></UL><UL><LI> Macville M., Schroeck E., Padilla-Nash H., Keck C., Ghadimi M. B., Zimonjic D., Pospecu N. &amp; Ried T. Comprehensive and definitive moleculare cytogenic characterization of HeLa cells by spectral karyotyping. <I>Cancer Research</I> 59: 141-150 (1999). 
</LI></UL><UL><LI> Masters J. R. HeLa cells 50 years on: the good, the bad and the ugly. <I>Nature Reviews</I> 2: 315-319 (2002).
</LI></UL><UL><LI> Ek E. T. H., Dass C. R. &amp; Choong P. F. M. Commonly used mouse models of osteosarcoma. <I>Critical Reviews in Oncology/Hematology</I> 60: 1-8 (2006).
</LI></UL><UL><LI> Ponten J. &amp; Saksela E. Two established <I>in vitro</I> cell lines from human mesenchymal tumours. <I>International Journal of Cancer</I> 2: 434-447 (1967).
</LI></UL><UL><LI> Nelissen J. M. D. T., Torensma R., Pluyter M., Adema G. J., Raymakers R. A. P., van Kooyk Y. &amp; Figdor, C. G. Molecular analysis of the hematopoiesis supporting osteoblastic cell line U2-OS. <I>Experimental Hematology</I> 28: 422-432 (2000).
</LI></UL><UL><LI> May, M. J. &amp; Ghosh, S. Rel/NF-κB and IKB proteins: an overview. <I>Seminars in Cancer Biology</I> 8: 63-73 (1997).
</LI></UL><UL><LI> Courtois G. The NF-κB signaling pathway in human genetic diseases. <I>Cell. Mol. Life Sci.</I> 62: 1682-1691 (2005).
</LI></UL><UL><LI> Vazquez A., Bond E. E., Levine A. J. &amp; Bond G. L. The genetics of the p53 pathway, apoptosis and cancer therapy. <I>Nat Rev Drug Discov</I> 7(12): 979-87 (2008).
</LI></UL><UL><LI> Mangelsdorf D. J. &amp; Evans R. M. The RXR heterodimers and orphan receptors. <I>Cell</I> 83: 841–850 (1995).
</LI></UL><UL><LI> Koeffler H. P. Peroxisome Proliferator-activated Receptor and Cancers. <I>Clinical Cancer Research</I> 9: 1-9 (2003).
</LI></UL><UL><LI> Suh N., Wang Y., Honda T., Gribble G. W., Dmitrovsky E., Hickey W. F., Maue R. A., Place A. E., Porter D. M., Spinella M. J., Williams C. R., Wu G., Dannenberg A. J., Flanders K. C., Letterio J. J., Mangelsdorf D. J., Nathan C. F., Nguyen L., Porter W. W., Ren R. F., Roberts A. B., Roche N. S., Subbaramaiah K. &amp; Sporn M. B. A novel synthetic oleanane triterpenoid, 2-cyano-3,12-dioxoolean-1,9- dien-28-oic acid, with potent differentiating, antiproliferative, and antiinflammatory activity. <I>Cancer Res.</I> 59: 336–341 (1999). 
</LI></UL><UL><LI> Brown M. S. &amp; Goldstein J. L . The SREBP pathway: regulation of cholesterol metabolism by proteolysis of a membrane-bound transcription factor. <I>Cell</I> 89 (3): 331–40 (1997).
</LI></UL><UL><LI> Briggs M. R., Kadonaga J. T., Bell S. P. &amp; Tjian R. Purification and biochemical characterization of the promoter-specific transcription factor, Sp1. <I>Science</I> 234: 47-52 (1986).
</LI></UL><UL><LI> Stielow B., Sapetschnig A., Wink C., Kruger I. &amp; Suske G. SUMO-modified Sp3 represses transcription by provoking local heterochromatic gene silencing. <I>EMBO Rep.</I> 9: 899-906 (2008).
</LI></UL><UL><LI> Zhang Y.,  Liao M. &amp; Dufau M. L. Unlocking repression of the human luteinizing hormone receptor gene by trichostatin A-induced cell-specific phosphatase release. <I>J. Biol. Chem.</I> 283: 24039-24046 (2008).
</LI></UL><UL><LI> Nicole Y., Khachigian T. &amp; Khachigian L. M. Sp1 Phosphorylation and Its Regulation of Gene Transcription. <I>Molecular and Cellular Biology</I> 29: 2483-2488 (2009).
</LI></UL><UL><LI> Zhong C-Y, Zhou y-M, Douglas G. C., Witschi H-P. &amp; Pinkerton K. E. MAPK/AP-1 signal pathway in tobacco smoke-induced cell proliferation and quamous metaplasia in the lungs of rats. <I>Carcinogenesis</I> 26 (12): 2187–2195 (2005).
</LI></UL><UL><LI> Hess J., Angel P. &amp; Schorpp-Kistner M. AP-1 subunits: quarrel and harmony among siblings. <I>J. Cell. Sci.</I> 117: 5965–73 (2004).
</LI></UL><UL><LI> Pellikainen J. M. &amp; Kosma V-M.  Activator protein-2 in carcinogenesis with a special reference to breast cancer—A mini review. <I>Int. J. Cancer</I> 120: 2061–2067 (2007).
</LI></UL><UL><LI> Hilger-Eversheim K. , Moser M., Schorle H. &amp; Buettner R. Regulatory roles of AP-2 transcription factors in vertebrate development, apoptosis and cell-cycle control. <I>Gene</I> 260(1-2): 1-12 (2000).
</LI></UL><UL><LI> Rani C. S. S.,  Qiang M. &amp; Ticku M. K. Potential Role of cAMP Response Element-Binding Protein in Ethanol-Induced N-Methyl-D-aspartate Receptor 2B Subunit Gene Transcription in Fetal Mouse Cortical Cells. <I>Molecular Pharmacology Fast Forward</I> 67: 2126-2136 (2005).
</LI></UL><UL><LI> Yin J. C. &amp; Tully T. CREB and the formation of long-term memory. <I>Curr Opin Neurobiol.</I> 6(2): 264-8 (1996).
</LI></UL><UL><LI> Pandey S. C., Chartoff E. H., Carlezon W. A., Zou J., Zhang H., Kreibich A. S., Blendy J. A. &amp; Crews F. T. CREB gene transcription factors: Role in molecular mechanisms of alcohol and drug addiction. <I>Alcohol Clin Exp Res.</I> 29(2): 176-184 (2005).
</LI></UL><UL><LI> Rany I., Megyesi J. K., Reusch J. E. B. &amp; Safirstein R. L. CREB mediates ERK-induced survival of mouse renal tubular cells after oxidant stress. <I>Kidney Int.</I> 68(4): 1573-82 (2005).
</LI></UL><UL><LI> Matuoka K. &amp; Chen K. Y. Transcriptional regulation of cellular ageing by the CCAAT box-binding factor CBF/NF-Y. <I>Ageing Res Rev.</I> 1(4): 639-51 (2002).
</LI></UL><UL><LI> Ronchi A., Bellorini M.,  Mongelli N. &amp; Mantovani R. CCAAT-box binding protein NF-Y (CBF, CP1) recognizes the minor groove and distorts DNA. <I>Nucleic Acids Res.</I> 23(22), 4565–4572 (1995). 
</LI></UL><UL><LI> Carlberg C. &amp; Seuter S. A genomic perspective on vitamin D signaling. <I>Anticancer Res.</I> 9: 3485-93 (2009).
</LI></UL><UL><LI> Bertone-Johnson E. R. Vitamin D and breast cancer. <I>Ann Epidemiol.</I> 7: 462-7 (2009).
</LI></UL><UL><LI> Obata T., Yanagidani A., Yokoro K., Numoto M. &amp; Yamamoto S. Analysis of the consensus binding sequence and the DNA-binding domain of ZF5. <I>Biochem Biophys Res Commun</I>. 255(2): 528-34 (1999).
</LI></UL><UL><LI> Numoto M., Yokoro K. &amp; Koshi J. ZF5, which is a Kruppel-type transcriptional repressor, requires the zinc finger domain for self-association. <I>Biochem Biophys Res Commun.</I> 256(3): 573-8 (1999).
</LI></UL><UL><LI> Scholz H., Wagner K. D. &amp; Wagner N. Role of the Wilms' tumour transcription factor, Wt1, in blood vessel formation. <I>Pflugers Arch.</I> 458(2): 315-23 (2009).
</LI></UL><UL><LI> Sakamoto Y., Mariya Y., Sasaki S., Teshiromori R., Oshikiri T., Segawa M., Ogura K., Akagi T., Kubo K., Kaimori M. &amp; Funato T. WT1 mRNA level in peripheral blood is a sensitive biomarker for monitoring minimal residual disease in acute myeloid leukemia. <I>Tohoku J Exp Med.</I> 219(2): 169-76 (2009).
</LI></UL><UL><LI> Dollé P. Developmental expression of retinoic acid receptors (RARs). <I>Nucl Recept Signal.</I> 7: e006 (2009).
</LI></UL><UL><LI> Oh-hora M. &amp; Rao A. The calcium/NFAT pathway: role in development and function of regulatory T cells. <I>Microbes Infect.</I> 11(5): 612-9. (2009)
</LI></UL><UL><LI> Goldstein J. L. &amp; Brown M. S. The LDL receptor. <I>Arterioscler Thromb Vasc Biol.</I> 4: 431-8 (2009). 
</LI></UL><UL><LI> Hegardt F. G. Transcriptional regulation of mitochondrial HMG-CoA synthase in the control of ketogenesis. <I>Biochimie</I> 80(10): 803-6 (1998).
</LI></UL><UL><LI> Yu J. &amp; Zhang L. PUMA, a potent killer with or without p53. <I>Oncogene</I> 27: S71-S83 (2008).
</LI></UL><UL><LI> Li Z. &amp; Srivastava P. Heat-shock proteins. <I>Curr Protoc Immunol.</I> Appendix 1: Appendix 1T, (2004).
</LI></UL><UL><LI> Tso P., Liu M., Kalogeris T. J. &amp; Thomson A. B. R. The role of apolipoprotein A-IV in the regulation of food intake. <I>Annu. Rev. Nutr.</I> 21: 231–54 (2001).
</LI></UL><UL><LI> Androutsopoulos V. P., Tsatsakis A. M. &amp; Spandidos D. A. Cytochrome P450 CYP1A1: wider roles in cancer progression and prevention. <I>BMC Cancer</I> 16:9 187, (2009).
</LI></UL><UL><LI> Carpenter G. &amp; Cohen S. Epidermal growth factor. <I>The Journal of Biological Chemistry</I> 265(14): 7709–12 (1990).
</LI></UL><UL><LI> Schnidar H., Eberl M., Klingler S., Mangelberger D., Kasper M., Hauser-Kronberger C., Regl G., Kroismayr R., Moriggl R., Sibilia M. &amp; Aberger F. Epidermal growth factor receptor signaling synergizes with Hedgehog/GLI in oncogenic transformation via activation of the MEK/ERK/JUN pathway. <I>Cancer Res.</I> 69(4): 1284-92 (2009).
</LI></UL><UL><LI> Moore M. J. &amp; Proudfoot N. J. Pre-mRNA processing reaches back to transcription and ahead to translation. <I>Cell</I>  136(4): 688-700 (2009).
</LI></UL><UL><LI> Berg J. M., Tymoczko J. L. &amp; Stryer L. <I>Biochemistry</I>. 6th edition, 2007. New York: WH Freeman &amp; Co
</LI></UL><UL><LI> Day D. A. &amp; Tuite M. F. Post-transcriptional gene regulatory mechanisms in eukaryotes: an overview, <I>J. Endocrinol.</I> 157: 361-371 (1998).
</LI></UL><UL><LI> Litt M., Qiu Y. &amp; Huang S. Histone arginine methylations: their roles in chromatin dynamics and transcriptional regulation. <I>Biosci Rep.</I> 29(2): 131-41 (2009).
</LI></UL><UL><LI> Wall M. E., Wani M. C., Cook C. E., Palmer K. H., McPhail A. I., Sim G. A. Plant antitumor agents. I. The isolation and structure of camptothecin, a novel alkaloidal leukemia and tumor inhibitor from camptotheca acuminate. In: J. Am. Chem. Soc. 88, 3888–3890(1966).
</LI></UL><UL><LI> Redinbo M. R., Stewart L., Kuhn P., Champoux J. &amp; Hol W. G. J. Crystal structure of human topoisomerase I in covalent and noncovalent complexes with DNA. <I>Science</I> 279: 1504–1513 (1999).
</LI></UL><UL><LI> Gao L., Sun C., Qiu H., Liu H., Shao H., Wang J., Li W. Cloning and characterization of a novel human zinc ﬁnger gene, hKid3, from a C2H2-ZNF enriched human embryonic cDNA library, <I>Biochemical and Biophysical Research Communications</I> 325:1145–1152 (2004)
</LI></UL><UL><LI>Rahman S., Thomas P. Molecular cloning, characterization and expression of two hypoxia-inducible factor alpha subunits, HIF-1α and HIF-2α, in a hypoxia-tolerant marine teleost, Atlantic croaker (Micropogonias undulatus), <I>Gene</I> 396:273 – 282 (2007)
</LI></UL><UL><LI>Fricker M., Runions J. &amp; Moore I. Quantitative fluorescence microscopy: From Art to science. <I>Annual Review of Plant Biology</I> 57: 79-107 (2006).
</LI></UL><UL><LI>Heim R. &amp; Tsien R. Y. Engineering green fluorescent protein for improved brightness, longer wavelengths and fluorescence resonance energy transfer. <I>Current Biology</I> 6(2): 178-182 (1996).
</LI></UL><UL><LI>Alberts B., Johnson A., Walter P. &amp; Lewis J. <I>Molecular Biology of the Cell.</I> Fifth edition. Garland Science London/New York, p. 742
</LI></UL><UL><LI>d'Enfert C., Gensse M. &amp; Gaillardin C. Fission yeast and a plant have functional homologues of the Sari and Sec12 proteins involved in ER to Golgi traffic in budding yeast. <I>The EMBO Journal</I> 11: 4205-4211 (1992).
</LI></UL><UL><LI>Zacharias D. A., Violin J. D., Newton A. C. &amp; Tsien R. Y. Partitioning of Lipid-Modified Monomeric GFPs into Membrane Microdomains of Live Cells. <I>Science</I>296(5569): 913-6 (2002).
</LI></UL><UL><LI>Alberts B., Johnson A., Walter P. &amp; Lewis J. <I>Molecular Biology of the Cell.</I> Fifth edition. Garland Science London/New York, pp. 706-707
</LI></UL><UL><LI>Locksley R. M., Killeen N. &amp; Lenardo M. J. The TNF and TNF receptor superfamilies: integrating mammalian biology. <I>Cell</I> 104(4): 487–501 (2001).
</LI></UL><UL><LI>Komarova E. A. &amp; Gudkov A. V. Suppression of p53: a new approach to overcome side effects of antitumor therapy. <I>Biochemistry (Mosc).</I> 65(1): 41-48 (2000).
</LI></UL><UL><LI>Murphy P. J., Galigniana M. D., Morishima Y., Harrell J. M., Kwok R. P., Ljungman M. &amp; Pratt W. B. Pifithrin-alpha inhibits p53 signaling after interaction of the tumor suppressor protein with hsp90 and its nuclear translocation. <I>J Biol Chem.</I> 279(29): 30195-201 (2004). 
</LI></UL><UL><LI>Tsien R. Y. The green fluorescent protein. <I>Annu. Rev.</I></LI></UL><P>Biochem.<I> 67: 509–44 (1998).</I></P><UL><LI>Tsien R. &amp; Prasher D. <I>Green Fluorescent Protein Properties, Applications, and Protocols.</I> New York: Wiley-Liss, 1998, p. 67-118.
</LI></UL><UL><LI>Heim R. &amp; Tsien R. Engineering Green Fluorescent Protein for improved brightness, longer wavelength and fluorescence resonance energy transfer. <I>Current Biology</I> 6: 178-182 (1996).
</LI></UL><UL><LI>Gao L., Sun C., Qiu H., Liu H., Shao H., Wang J., Li W. Cloning and characterization of a novel human zinc ﬁnger gene, hKid3, from a C2H2-ZNF enriched human embryonic cDNA library. <I>Biochemical and Biophysical Research Communications</I> 325:1145–1152 (2004)
</LI></UL><UL><LI>Rahman S. &amp; Thomas P. Molecular cloning, characterization and expression of two hypoxia-inducible factor alpha subunits, HIF-1α and HIF-2α, in a hypoxia-tolerant marine teleost, Atlantic croaker (Micropogonias undulatus). <I>Gene</I> 396: 273–282 (2007).
</LI></UL></DIV></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Heidelberg/Project_References">http://2009.igem.org/Team:Heidelberg/Project_References</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Project_References" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Project_References" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Project_References&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Project_References&amp;oldid=160264" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>