<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_HEARTBEAT_database"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Heidelberg/HEARTBEAT_database">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/HEARTBEAT_database&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/HEARTBEAT_database&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/HEARTBEAT_database&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/HEARTBEAT_database" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/HEARTBEAT database</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="main_wrapper"><DIV id="header"><DIV id="navigation"><DIV id="menu" style="position:static"><UL><LI id="mainHome"><A class="aMain" href="https://2009.igem.org/Team:Heidelberg">Home</A></LI><LI id="mainTeam"><A class="aTeam" href="https://2009.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Members#Students">Students</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Members#Advisors">Advisors</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Scientific_Environment">Scientific Environment</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Gallery">Gallery</A></LI></UL><LI id="mainProject"><A class="aProject" href="https://2009.igem.org/Team:Heidelberg/Project">Project</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project">Abstract</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Introduction">Introduction</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Measurement">Measurement</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Synthetic_promoters">Synthetic Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Further">Further Subprojects</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_SaO">Summary &amp; Outlook</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_References">References</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Acknowledgement">Acknowledgements</A></LI><LI id="mainParts"><A class="aParts" href="https://2009.igem.org/Team:Heidelberg/Parts">Parts</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Parts">Parts</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Parts_Characterization">Characterization</A></LI><LI id="mainModeling"><A class="aModeling" href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">What is HEARTBEAT?</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_database">HEARTBEAT Database</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_GUI">HEARTBEAT GUI</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_gui">GUI Documentation</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_network">HEARTBEAT Fuzzy Network Modeling</A></LI><LI id="mainNotebook"><A class="aNotebook" href="https://2009.igem.org/Team:Heidelberg/Notebook">Notebook</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_natural_promoters_overview">Natural Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_synthetic_promoters">Synthetic Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_promoters_cells">Cell Culture, Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_cell_line">Stable Cell Line</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_color_output">Multi-Color Output</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_modeling">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_measure">Measurements</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_MaM">Material &amp; Methods</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_meetings">Team Meetings</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Biosafety">Biosafety</A></LI><LI id="mainEucaryopedia"><A class="aEucaryopedia" href="https://2009.igem.org/Team:Heidelberg/Eukaryopedia">Eukaryopedia</A></LI><LI id="mainSponsors"><A class="aSponsors" href="https://2009.igem.org/Team:Heidelberg/Sponsors">Sponsors</A></LI></DIV></DIV></DIV></DIV><TABLE><TBODY><TR valign="top" style="margin-left: 2px;"><TD width="750px" style="padding: 0 15px 15px 20px; background-color:#ede8e2"><H1><SPAN class="mw-headline" id="HEARTBEAT:_Rational_design_of_promoter_sequences"> HEARTBEAT: Rational design of promoter sequences </SPAN></H1><H2><SPAN class="mw-headline" id="Contents"> Contents </SPAN></H2><UL><LI><A href="/Team:Heidelberg/HEARTBEAT_database#Introduction" title="Team:Heidelberg/HEARTBEAT database">Introduction</A></LI><LI><A href="/Team:Heidelberg/HEARTBEAT_database#Methods" title="Team:Heidelberg/HEARTBEAT database">Methods</A></LI><LI><A href="/Team:Heidelberg/HEARTBEAT_database#Results" title="Team:Heidelberg/HEARTBEAT database">Results</A></LI><LI><A href="/Team:Heidelberg/HEARTBEAT_database#Discussion" title="Team:Heidelberg/HEARTBEAT database">Discussion</A></LI><LI><A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">References</A></LI></UL><H2><SPAN class="mw-headline" id="Introduction"> Introduction </SPAN></H2><P>In the field of Synthetic Biology mathematical models based on bioinformatic methods are mainly used in order to describe the systems behavior of the involved synthetic constructs. To conduct the orchestra of these parts systems biologists try out different modeling approaches for example analytical or stochastic modeling depending on the underlying system. The introduced parameters  in these models are naturally fitted to experimental data acquired <I>in vivo</I><A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">[1]</A>. But the interests in synthetic biology will not only be concerned about systems of existing parts, the focus of research in systems biology will rather  be in the design of totally rational assembled constructs not described in nature so far. For that purpose we need models which predict and optimize the properties of these <I>in silico</I> designed constructs.  
</P><P>In order to enable this modeling approach it is useful to define a part as a basic biological function encoded as genetic material (<A href="http://blip.tv/file/548184" class="external text" rel="nofollow">listen to Drew Endy, President of the BioBricks Foundation</A>). Starting from this abstract concept we propose to devide up the &quot;parental&quot; part into smaller functional units which we define  “subparts” that exhibit their own biological function. However, this function can only be exerted if all subparts are exactly assembled together in the “parental” part. If all system related parameters for every subpart are known, one will be able to model the characteristics of the &quot;parental&quot; part.
</P><P>In this work we propose a model which describes the rational design of a certain promoter of interest. Furthermore, the model is able to predict the functional outcome of the created sequence and it can be used to improve the input sequence, both based on fuzzy logic modeling (<A href="/Team:Heidelberg/HEARTBEAT_network" title="Team:Heidelberg/HEARTBEAT network">HEARTBEAT fuzzy network (FN)</A>). For the development of the model understanding of the functionality and composition of a promoter was essential: it consists of a core-promoter comprising one or more subparts which can work as the binding site for the basal transcription machinery, i.e. RNA polymerase, TATA-box and the Inr-element. Transcription factors regulating the transcriptional activity interact preferentially with the proximal promoter  <A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">[2]</A>. The intensity of the regulation is further defined by the position of the transcription factor binding site (TFBS) relative to the transcriptional start site (TSS), the binding affinity of the TF to a particular binding motive and also by synergistic effects evolving from interactions between the simultaneously binding of multiple Tfs. <A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">[3]</A>,<A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">[4]</A> In addition the spacer sequences are directly influencing  the promoter strength <A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">[5]</A>. 
</P><P>Based on these assumptions the optimal synthetic promoter sequence can be designed using the following parameters (Fig. 1):
</P><UL><LI> all TFBS selected to be in the sequence
</LI><LI> auxiliary TFBS, which are supposed to have a synergistic effect
</LI><LI> optimal distance of all TFBS to the TSS
</LI><LI> maximal spacer sequence quality between the TFBS 
</LI><LI> optimal distance between TFBSs
</LI></UL><P>In the following articles we propose a standardized strategy to specify these parameters with computerized methods based on HEARTBEAT (<B>He</B>idelberg <B>Ar</B>tificial <B>T</B>ranscription Factor <B>B</B>inding Site <B>E</B>ngineering and <B>A</B>ssembly <B>T</B>ool). We will further explain the manual design of promoter sequences using these parameters  for <A href="/Team:Heidelberg/Eukaryopedia#SREBP" title="Team:Heidelberg/Eukaryopedia">SREBP</A> and <A href="/Team:Heidelberg/Eukaryopedia#VDR" title="Team:Heidelberg/Eukaryopedia">VDR</A>. Initital measurements show the potential of HEARTBEAT-designed sequences to function <I>in vivo</I> (see <A href="/Team:Heidelberg/HEARTBEAT_database#Results" title="Team:Heidelberg/HEARTBEAT database">results</A>). On the basis of our observations, we claim that our way of combining rational generation and experimental validation of synthetic constructs provides a novel and effective strategy in synthetic biology.
</P><P>[<A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_database" class="external text" rel="nofollow">TOP</A>]
</P><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 1: The generation of promoter sequences- from genome scanning to sequence design</B></DIV></DIV></DIV><P><A href="/Team:Heidelberg/HEARTBEAT_database#Introduction" title="Team:Heidelberg/HEARTBEAT database">[TOP]</A></P><H2><SPAN class="mw-headline" id="Methods">Methods</SPAN></H2><P>Our promoter sequence
was defined to be 1000 bp upstream of the TSS. The 
<A href="http://genome.ucsc.edu/cite.html|" class="external text" rel="nofollow">UCSC Genome Browser</A><A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">[6]</A> provides reference sequence and working draft assemblies
for a large collection of genomes including the human genome. We derived the
provided sequences containing the 1000 bases upstream of annotated TSS for each RefSeq gene
<A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">[7]</A>.
Upon this pre-selection, we further narrowed our choice of promoter set by selection of distinct
pathways. KEGG (<A href="http://www.genome.jp/kegg/Kyoto|" class="external text" rel="nofollow">Kyoto Encyclopedia of Genes and Genomes</A>)
<A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">[8]</A>, <A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">[9]</A>, <A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">[10]</A> provides a database of biological systems, consisting of several building blocks
including for instance genes and proteins (KEGG GENES) or hierarchies and relationships of various
biological objects (KEGG BRITE). Here, KEGG PATHWAY which comprises molecular
interaction and reaction networks for metabolism, various cellular processes and human diseases
was of particular interest. From this collection of molecular wiring maps, we chose all
physiologically relevant pathways for our project, thereby ruling out tissue or other highly specific pathways,
like olfactory / taste transduction as well as several pathways related to human diseases.
</P><P>[<A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_database" class="external text" rel="nofollow">TOP</A>]
</P><H3><SPAN class="mw-headline" id="Promotersweep"> Promotersweep </SPAN></H3><P>One of the most challenging problems in bioinformatics remains the computerized localization of
TFBS and the transcriptional start sites (TSS) as well as the determination of the core promoter.
Many of the available motive discovery tools exhibit the problem of extended false positive
predictions. The Promotersweep web-tool improves the accuracy of this analysis by combining a
vaste number of different algorithms and methods simultaneously <A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">[11]</A>. It integrates information from three homology databases (EnsEMBL Compara <A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">[12]</A>, NCBI HomoloGene <A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">[13]</A>,DoOP database <A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">[14]</A>), five
promoter databases(EPD <A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">[15]</A>, DBTSS <A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">[16]</A>), six sequence motive identification tools (e.g. Gibbs
MotifSampler <A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">[17]</A>) and two matrix profile databases (Jaspar Core Library <A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">[18]</A>, Transfac
ProfessionalLibrary <A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">[19]</A>) to identify and annotate TFBS. The Promotersweep pipeline is started by entering a sequence, chosen between human or mouse as origin. Initially a homology search is performed by using different BLAST algorithms. As a result orthologous promoter regions are
deduced from EnsEMBL, Homologene or DoOP, respectively. Subsequently, several motive discovery tools determine shared motives of orthologous or co-regulated sequences. In the last step each TFBS is identified and evaluated with the help of the Transfac - and the Jaspar Core library. Every identified TFBS is classified as weak, conserved or reliable - according to the similarity of the predictions of the different algorithms. In Fig. 2 the result for the Heat shock
cognate 71 kDa protein (NM_153201) promoter is shown. For this promoter four different binding
sites were discovered. Three of them were classified as reliable and one as conserved. For each hit,
the output of Promotersweep contains the position of the motive relative to the TSS. So far we were
able to analyse 4395 different promoter sequences, which hold 29966 TFBS in total.
</P></TD></TR><TR><TD><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 2: Promotersweep output for the Heat shock</B> cognate 71 kDa protein (NM_153201).<B> Four different binding sites were discovered.</B></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>[<A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_database" class="external text" rel="nofollow">TOP</A>]
</P><H3><SPAN class="mw-headline" id="The_HEARTBEAT-database"> The HEARTBEAT-database </SPAN></H3><P>In order to retrieve the information computed by promotersweep as fast as possible we decided to
develop a database structure based on MySQL (My systems query language) <A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">[20]</A>. MySQL is one of the
most popular relational database management systems. It offers not only a language to set up a
hierarchical database but also an interface for easy manipulation of data. The advantage of MySQL
is its very intuitive command language and the table structure which helps to minimize redundant
data. Simple queries are written in a “SELECT - FROM - WHERE” format. With SELECT value
all requested columns are specified. The FROM value calls the corresponding table and WHERE
allows a more accurate selection. For our database the average query duration is below 200 ms. This
enabled us to provide a fluent online access of HEARTBEAT through the <A href="/Team:Heidelberg/HEARTBEAT_gui" title="Team:Heidelberg/HEARTBEAT gui">HEARTBEAT GUI</A>.
Our data is stored in the tables “Main_Info” and “Gene_Info”. Main_Info contains all necessary
data to define the location, binding motive and quality of a TFBS, whereas Gene_Info offers
additional information for the gene as well as several gene annotations, where the TFBS is located
on. In table 1 and 2 the table structure is shown for Main_Info and Gene_Info.
</P><P><B>Table 1: Database structure: Main Info</B></P><TABLE class="wikitable centered" style="margin: 1em auto 1em auto"><TBODY><TR style="background-color:#99cccc;"><TH height="20px," width="100px"> RefseqID </TH><TH width="100px"> TF Name </TH><TH width="100px">TF position start </TH><TH width="100px"> TF position end </TH><TH width="100px"> TF motive </TH><TH width="100px"> TF score </TH><TH width="100px"> BS quality </TH><TH width="100px"> TF matrix
</TH></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> NM_000201    </TD><TD> VDR(V$VDR_Q3)     </TD><TD>               568 </TD><TD>             573 </TD><TD> aagcga          </TD><TD> 0.906    </TD><TD> conserved  </TD><TD> VDR
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> NM_000393    </TD><TD> VDR(V$VDR_Q3)     </TD><TD>               825 </TD><TD>             832 </TD><TD> tagggagg        </TD><TD> 0.955    </TD><TD> conserved  </TD><TD> VDR
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> NM_000564    </TD><TD> VDR(V$VDR_Q3)     </TD><TD>               235 </TD><TD>             243 </TD><TD> tgggaaccc       </TD><TD> 0.908    </TD><TD> conserved  </TD><TD> VDR
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> NM_000684    </TD><TD> VDR(V$VDR_Q3)     </TD><TD>               660 </TD><TD>             665 </TD><TD> ggggtg          </TD><TD> 0.900    </TD><TD> reliable   </TD><TD> VDR
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> NM_000725    </TD><TD> VDR(V$VDR_Q3)     </TD><TD>               911 </TD><TD>             916 </TD><TD> gggtca          </TD><TD> 0.920    </TD><TD> conserved  </TD><TD> VDR
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> NM_000525    </TD><TD> SREBP(V$SREBP_Q6) </TD><TD>               469 </TD><TD>             473 </TD><TD> cgtga           </TD><TD> 0.991    </TD><TD> conserved  </TD><TD> SREBP
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> NM_000817    </TD><TD> SREBP(V$SREBP_Q3) </TD><TD>               909 </TD><TD>             913 </TD><TD> cccga           </TD><TD> 0.962    </TD><TD> conserved  </TD><TD> SREBP
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> NM_000872    </TD><TD> SREBP(V$SREBP_Q3) </TD><TD>               352 </TD><TD>             357 </TD><TD> acccca          </TD><TD> 0.989    </TD><TD> conserved  </TD><TD> SREBP
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> NM_000905    </TD><TD> SREBP(V$SREBP_Q3) </TD><TD>               917 </TD><TD>             926 </TD><TD> gagtcaccca      </TD><TD> 0.960    </TD><TD> reliable   </TD><TD> SREBP
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> NM_000909    </TD><TD> SREBP(V$SREBP_Q6) </TD><TD>               526 </TD><TD>             532 </TD><TD> gcgtgag         </TD><TD> 0.982    </TD><TD> conserved  </TD><TD> SREBP
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> NM_001011551 </TD><TD>SREBP(V$SREBP_Q3) </TD><TD>               320 </TD><TD>             324 </TD><TD> gaata           </TD><TD> 0.967    </TD><TD> conserved  </TD><TD> SREBP
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> NM_001013620 </TD><TD> SREBP(V$SREBP_Q6) </TD><TD>               951 </TD><TD>             960 </TD><TD> cactccagga      </TD><TD> 0.989    </TD><TD> conserved  </TD><TD> SREBP
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> NM_001024    </TD><TD> SREBP(V$SREBP_Q6) </TD><TD>               974 </TD><TD>             978 </TD><TD> acccg           </TD><TD> 0.987    </TD><TD> reliable   </TD><TD> SREBP
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> NM_001025366 </TD><TD> SREBP(V$SREBP_Q6) </TD><TD>               556 </TD><TD>             561 </TD><TD> ggggtc          </TD><TD> 0.983    </TD><TD> reliable   </TD><TD> SREBP
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> NM_001025367 </TD><TD> SREBP(V$SREBP_Q6) </TD><TD>               556 </TD><TD>             561 </TD><TD> ggggtc          </TD><TD> 0.983    </TD><TD> reliable   </TD><TD> SREBP
</TD></TR></TBODY></TABLE><P><B>Table 1: Database structure: Gene Info</B></P><TABLE class="wikitable centered" style="margin: 1em auto 1em auto"><TBODY><TR style="background-color:#99cccc;"><TH height="20px," width="100px"> RefseqID </TH><TH width="100px"> EntrezID </TH><TH width="100px"> Gene symbol</TH><TH width="100px"> EnsembleID </TH><TH width="100px"> TSS doop </TH><TH width="100px">TSS DBTSS  </TH><TH width="100px"> TSS EPD </TH><TH width="100px"> TSS MPromDB
</TH></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> NM_181537 </TD><TD> 342574   </TD><TD> KRT27       </TD><TD> ENSG00000171446 </TD><TD> 986      </TD><TD> 984       </TD><TD> NA      </TD><TD> 984
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> NM_006522 </TD><TD> 7475     </TD><TD> WNT6        </TD><TD> ENSG00000115596 </TD><TD> 1118     </TD><TD> 1116      </TD><TD> NA      </TD><TD> 1099
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="font-weight:bold;"> NM_013445 </TD><TD> 2571     </TD><TD> GAD1        </TD><TD> ENSG00000128683 </TD><TD> NA       </TD><TD> 1199      </TD><TD> NA      </TD><TD> 87
</TD></TR></TBODY></TABLE><P>[<A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_database" class="external text" rel="nofollow">TOP</A>]
</P><H3><SPAN class="mw-headline" id="Promoter_design"> Promoter design </SPAN></H3><P>For the rational design of a responsive promoter construct several preliminary considerations have
to be done. The first question which needs to be addressed concerns the inducibility of the pathway
of interest. Preceding experiments revealed that for <A href="/Team:Heidelberg/Eukaryopedia#VDR" title="Team:Heidelberg/Eukaryopedia">VDR</A> (vitamin D receptor) as well as for <A href="/Team:Heidelberg/Eukaryopedia#SREBP" title="Team:Heidelberg/Eukaryopedia">SREBP</A> (sterol regulatory element binding protein), convenient conditions and treatments exist
under which each pathway can be exclusively activated without killing the chassis, that is in our
studies the transfected cells. For further reference about induction conditions refer to <A href="/Team:Heidelberg/Notebook_MaM#Induction" title="Team:Heidelberg/Notebook MaM">Material and Methods</A>. After deciding what kind of TFBS is to be included appropriate consensus motives where TFs would presumably bind on had to be selected. Reliable consensus motives can be deduced from matrices
provided in the TRANSFAC database. In case of several different binding matrices, we chose the
longest motive which contains the most definite bases <A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">[19]</A>.
Focusing from now on only on VDR and SREBP we created the frequency distributions of the TFBS
occurrence for both TFs based on HEARTBEAT (see Fig. 3 and 4). As mentioned above the basic
assumption of our model is that most transcription factors exhibit a spatial preference for binding
to the DNA which includes not only the binding sequence and the distance to the TSS but also the
mutual distance between potential TF pairs. Based on this concept we specified distance of the pdf-maxima
to the TSS for both VDR and SREBP. Subsequently the binding motive is embedded into
the artificial promoter around the position where the majority of binding sites are located in natural
promoters. With this idea we created first a series of synthetic promoters in which we differed only
the number of binding motives positioned around the pdf-maxima. For all sequences designed for VDR, see <A href="/wiki/images/f/f0/HEARTBEAT_VDR.pdf" class="internal" title="HEARTBEAT VDR.pdf">HEARTBEAT_VDR.pdf</A>. For all sequences designed for SREBP, see <A href="/wiki/images/d/da/HEARTBEAT_SREBP.pdf" class="internal" title="HEARTBEAT SREBP.pdf">HEARTBEAT_SREBP.pdf</A>.
</P><P>With a second series of artificially designed promoter sequences we tried to answer to what extend
further auxiliary TFBS affect the binding activity of VDR and SREBP. Therefore we plotted the
frequencies of all TFBS which are co-occuring when VDR or SREBP is present in a natural
promoter sequence as well (see Fig. 5 and 6). For SREBP  <A href="/Team:Heidelberg/Eukaryopedia#ZF5" title="Team:Heidelberg/Eukaryopedia">ZF5</A> was also present with a relative frequency
of 60%. In case of VDR, <A href="/Team:Heidelberg/Eukaryopedia#AP-2" title="Team:Heidelberg/Eukaryopedia">AP-2</A> co-exists in 48% and <A href="/Team:Heidelberg/Eukaryopedia#WT1" title="Team:Heidelberg/Eukaryopedia">WT1</A> in 54% of all VDR-promoters. In the
following we proceeded in analogy to series one. We created a variety of sequences where we
included TFBS in proximity to the pdf-maximum of their frequency distribution besides the VDR
and SREBP binding sites. Depending on the number of species of TFBS we distinguish between a
blue (1 TFs), green (2 TFs) and orange (3 different TFs) series. Finally all spacer sequences were
filled with a random sequence with A:T and C:G content ratios being equal.
To make sure that our sequences are as specific as possible we iteratively checked and modified our
sequence with the Transfac match tool <A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">[21]</A> until no other TFBS expect for our chosen ones were
detected. Additionally we tested the sequence for every restriction site used in any Biobrick
standard. Finally we added a HindIII at the 5' end and a SpeI restriction site at the 3' end to allow
for cloning the construct into the reporter plasmid.
</P><P><A href="/Team:Heidelberg/HEARTBEAT_database#Introduction" title="Team:Heidelberg/HEARTBEAT database">[TOP]</A></P><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><H3><SPAN class="mw-headline" id="HEARTBEAT_analyzes_transcription_factor_binding_preferences"> HEARTBEAT analyzes transcription factor binding preferences </SPAN></H3><P>For the statistical analysis we plotted the absolute frequency of occurrence for each TF-binding site
in a histogram against the position relative to the TSS where the TSS is located at base 1001 -1003.
Each bin comprises 20 bases analogous to different low resolution approaches which analysed the
spatial distribution of TFBS with a sliding window of 20-25 bp <A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">[22]</A>, <A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">[3]</A>, <A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">[23]</A>. From 356 different TFBS for which Transfac contains at least one binding matrix 144 TFBS occurred at least within 50
from 4390 natural promoters. TFBS with less than 50 counts were removed from the selection and
not considered for further analysis. In Fig. 7-10 the spatial distributions of <A href="/Team:Heidelberg/Eukaryopedia#Sp1" title="Team:Heidelberg/Eukaryopedia">Sp1</A>, <A href="/Team:Heidelberg/Eukaryopedia#AP-2" title="Team:Heidelberg/Eukaryopedia">AP-2</A>, <A href="/Team:Heidelberg/Eukaryopedia#IPF1" title="Team:Heidelberg/Eukaryopedia">IPF1</A> (Insulin
promoter factor 1) and <A href="/Team:Heidelberg/Eukaryopedia#Kid3" title="Team:Heidelberg/Eukaryopedia">Kid3</A> binding sites are shown. The red solid line represents the re-scaled probability density function (pdf). We introduced this function for two reasons. On the one hand the pdf is more robust with respect to outliers than a normal histogram. On the other hand we used the
rescaled area under the curve between a shifting frame of 20 bases as a measurement for the significance of a particular TFBS occurrence. The vertical red line in each plot defines the maximum of the pdf. Around the respective base position the majority of binding motives are located within the natural promoters. The maximum of the pdf will serve in the following as the position where binding sites are introduced into our rational designed promoter sequences.
</P><P>[<A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_database" class="external text" rel="nofollow">TOP</A>]
</P><H3><SPAN class="mw-headline" id="An_in_vivo_Test_of_HEARTBEAT_predicted_sequences"> An <I>in vivo</I> Test of HEARTBEAT predicted sequences </SPAN></H3><DIV style="text-align:justify;"> SREBP-responsive sequences designed as <A href="/Team:Heidelberg/HEARTBEAT_database#Promoter_design" title="Team:Heidelberg/HEARTBEAT database">outlined above</A> were transfected in HeLa cells and SREBP was induced as <A href="/Team:Heidelberg/Notebook_MaM#Induction" title="Team:Heidelberg/Notebook MaM">described</A>. Promoter activity was then preliminarily analyzed by TECAN (automated fluorescence plate reader). Sequence HB9 showed significant induction (Fig. 11)</DIV>.
<DIV class="center"><TABLE class="wikitable centered" border="2" rules="rows" style="border-color:white;"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 3: VDR binding site distribution</B> X axis corresponds to base pair distance from the TSS, where 1000 is the TSS.</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 4: SREBP binding site distribution</B></DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 5: Transcription factors co-occurring with VDR.</B> X axis corresponds to transcription factor ID, vertical axis corresponds to number of hits (transcription factor x occuring together with VDR)</DIV></DIV></DIV></DIV></TD></TR><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 6:  Transcription factors co-occurring with SREBP.</B> X axis corresponds to transcription factor ID, vertical axis corresponds to number of hits (transcription factor x occurring together with SREBP)</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 7: Kid3 binding site distribution</B>X axis corresponds to base pair distance from the TSS, where 1000 is the TSS.</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 8: IPF1 binding site distribution</B> X axis corresponds to base pair distance from the TSS, where 1000 is the TSS.</DIV></DIV></DIV></DIV></TD></TR><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 9: Ap-2 binding site distribution</B>X axis corresponds to base pair distance from the TSS, where 1000 is the TSS.</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 10: Sp1 binding site distribution</B>X axis corresponds to base pair distance from the TSS, where 1000 is the TSS.</DIV></DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><DIV style="text-align:justify;"><B>Figure 11: HEARTBEAT-predicted SREBP responsive promoter is induced by SREBP.</B> Characterization by TECAN plate fluorescence reader. For Induction and inhibition conditions, refer to  <A href="/Team:Heidelberg/Notebook_MaM#Induction" title="Team:Heidelberg/Notebook MaM">Material and Methods</A></DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE></DIV><P><A href="/Team:Heidelberg/HEARTBEAT_database#Introduction" title="Team:Heidelberg/HEARTBEAT database">[TOP]</A></P><H2><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H2><P>In predicting a functional promoter sequence, we followed one of the basic principles of synthetic biology: Studying nature quantitatively and applying the knowledge thus gained to the <I>de novo</I> creation of biological entities. Four points require further discussion: 
</P><OL><LI> What is the scientific significance of the analysis created by HEARTBEAT?
</LI><LI> How can the HEARTBEAT database be improved?
</LI><LI> What does the outcome of our <I>in vivo</I> test tell us?
</LI><LI> Can promoter design become fully automated?
</LI></OL><P>[<A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_database" class="external text" rel="nofollow">TOP</A>]
</P><H3><SPAN class="mw-headline" id="Transcription_factors_have_a_spatial_binding_preference"> Transcription factors have a spatial binding preference </SPAN></H3><P>Upon statistical analysis of over 4000 promoter sequences, we discovered 90 out of 356 TFBS distributions with one significant peak reflecting the spatial occurrence of a particular binding motive. 54 distributions contain two equally high local maxima. For the remaining 212 transcription factors less than 50 TFBS could be detected by Promotersweep and hence were not in the scope of our analysis. Transcription factors have been shown before to have certain spatial binding preferences <A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">[3]</A>, <A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">[4]</A>, but no analysis based upon multiple databases and gene orthologies was ever conducted. Our method therefore minimizes false positives, and still, our results confirm existing findings.
</P><P>[<A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_database" class="external text" rel="nofollow">TOP</A>]
</P><H3><SPAN class="mw-headline" id="Improving_HEARTBEAT_database"> Improving HEARTBEAT database </SPAN></H3><P>As noted above, for 212 TFs, less than 50 TFBSs could be detected. In order to overcome this problem in the future we plan to accomplish the screening of the entire genome and to systematically expand our promoter screening by including genomes from different mammalian species. With the increased variety of input sequences we hope to decrease the influence of false positive hits in our statistics. Furthermore it could also help to understand the 54 distributions with multiple maxima and answer the question about the occurance of multiple potential binding sites of one single TF within a given promoter sequence. To make HEARTBEAT even more universal we plan to screen a broader range of the natural promoters which will comprise the sequence downstream of the TSS as well, as it is a well known fact that TF bind to introns and 5' UTR also <A href="/Team:Heidelberg/HEARTBEAT_database#References" title="Team:Heidelberg/HEARTBEAT database">[24]</A>. In parallel we want to focus on promoters involved in disease related pathways which promises deeper insight in the differences in their molecular regulation.
</P><P>[<A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_database" class="external text" rel="nofollow">TOP</A>]
</P><H3><SPAN class="mw-headline" id="In_vivo_test_of_predicted_sequences_shows_that_functional_promoter_can_be_predicted_by_HEARTBEAT"><I>In vivo</I> test of predicted sequences shows that functional promoter can be predicted by HEARTBEAT </SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:382px;"><DIV class="thumbcaption"><B>Figure 12: HEARTBEAT generates functional promoter sequences</B>. Sequence HB 8 and 9 are inducible under  <A href="/wiki/index.php?title=Team:_Heidelberg/Notebook_MaM&amp;action=edit&amp;redlink=1" class="new" title="Team: Heidelberg/Notebook MaM (page does not exist)">appropriate conditions</A> . Sequences marked red were introduced as a negative control</DIV></DIV></DIV><DIV style="text-align:justify;">Of all the designed sequences, HB8 and HB9 showed activity under <A href="/wiki/index.php?title=Team:_Heidelberg/Notebook_MaM&amp;action=edit&amp;redlink=1" class="new" title="Team: Heidelberg/Notebook MaM (page does not exist)">appropriate conditions</A>(compare Fig. 11, Results). These sequences are highly similar (see Fig. 12 for how SREs were placed) since they have Transcription Factor Binding Sites under <I>both</I> peaks of the distribution with at least one auxiliary Sp1 element. The other sequences either lack the second SREBP binding site or the Sp1 binding site. Sequences having another type of putative SREBP response element are not functional. <B>We thus showed that the sequences which most closely reflect the expected distribution also work best!</B> These results point out that promoters can be predicted by HEARTBEAT. It also becomes clear that from the study of synthetic promoters, very useful information about gene regulation can be deduced. In order to obtain the information it requires <A href="/Team:Heidelberg/HEARTBEAT_network" title="Team:Heidelberg/HEARTBEAT network">elaborate models</A>, which we started to develop. In the combination with <A href="/Team:Heidelberg/Project_Synthetic_promoters#Results" title="Team:Heidelberg/Project Synthetic promoters">RA-PCR</A>, we provide two very powerful tools for the generation of synthetic promoters. We believe that especially by combining the two methods (see <A href="/Team:Heidelberg/Project_Synthetic_promoters#M-RA-PCR" title="Team:Heidelberg/Project Synthetic promoters">discussion M-RA-PCR</A>), we are able to create <I>any</I> promoter desired in the foreseeable future, thereby accelerating the progress of virotherapy development and fundamental research alike.</DIV><H3><SPAN class="mw-headline" id="Automatization_of_promoter_sequence_generation"> Automatization of promoter sequence generation </SPAN></H3><P>In this work, the design of promoter sequences was an <I>ad-hoc</I> process which took an entire day. We therefore developed a <A href="/Team:Heidelberg/HEARTBEAT_gui" title="Team:Heidelberg/HEARTBEAT gui">GUI</A> which assists users  with sequence design and automates certain step. Tools such as this will make promoter design a very easy task in the future.
</P><P><A href="/Team:Heidelberg/HEARTBEAT_database#Introduction" title="Team:Heidelberg/HEARTBEAT database">[TOP]</A></P><H2><SPAN class="mw-headline" id="References"> References </SPAN></H2><P>[1] <A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling" class="external text" rel="nofollow">BCCS-Bristol: Best Model 2008</A></P><P>[2] Heintzman N. D. &amp; Ren B. The gateway to transcription: identifying, characterizing and understanding promoters in the eukaryotic genome. <I>Cellular and Molecular Life Science</I> 64: 386-400 (2007).
</P><P>[3] Vardhanabhuti, S., Wang, J. &amp; Hannenhalli, S. Position and distance specificity are important determinants of <I>cis</I>-regulatory motifs in addition to evolutionary conservation. <I>Nucl Acid Res</I> 35(10): 3203-3213 (2007).</P><P>[4] Yokoyama, K. D., Ohler, U. &amp; Wray, G. A. Measuring spatial preferences at fine-scale resolution identifies known and novel <I>cis</I>-regulatory element candidates and functional motif-pair relationships. <I>Nucl Acid Res</I> 37(13): e92 (2009)</P><P>[5] Ellis T., Wang X. &amp; Collins J. J. Diversity-based, model-guided construction of synthetic gene networks with predicted functions. <I>Nature Biotechnology</I> 27: 465-471 (2009).
</P><P>[6] Kent W. J., Sugnet C. W., Furey T. S., Roskin K. M., Pringle T. H., Zahler A. M. &amp; Haussler D. The human genome browser at UCSC. <I>Genome Res.</I> 12(6): 996-1006 (2002).
</P><P>[7] Kuhn R. M., Karolchik D., Zweig A. S., Wang T., Smith K. E., Rosenbloom K. R., Rhead B., Raney B. J., Pohl A., Pheasant M., Meyer L., Hsu F., Hinrichs A. S., Harte R. A., Giardine B., Fujita P., Diekhans M., Dreszer T., Clawson H., Barber G. P., Haussler D. &amp; Kent W. J. The UCSC Genome Browser Database: update 2009. <I>Nucleic Acids Res.</I> 37(Database issue): D755-61 (2009).
</P><P>[8] Kanehisa M., Araki M., Goto S., Hattori M., Hirakawa M., Itoh M., Katayama T., Kawashima S., Okuda S., Tokimatsu T. &amp; Yamanishi, Y. KEGG for linking genomes to life and the environment. <I>Nucleic Acids Res.</I> 36 (Database issue): D480-4 (2008). 
</P><P>[9] Kanehisa M., Goto S., Hattori M., Aoki-Kinoshita K.F., Itoh M., Kawashima S., Katayama T., Araki M. &amp; Hirakawa M. From genomics to chemical genomics: new developments in KEGG. <I>Nucleic Acids Res.</I> 34 (Database issue): D354-7 (2006).
</P><P>[10] Kanehisa M. &amp; Goto S. KEGG: Kyoto Encyclopedia of Genes and Genomes. <I>Nucleic Acids Res.</I> 28(1): 27-30 (2000). 
</P><P>[11] del Val C., Pelz O., Glatting K.-H., Barta E. &amp; Hotz-Wagenblatt, A. PromoterSweep: a tool for identification of transcription factor binding sites. Theor. Chem. Acc. DOI 10.1007: s00214-009-0643-8 (2009).
</P><P>[12] <A href="http://apr2006.archive.ensembl.org/info/software/compara/index.html" class="external text" rel="nofollow">Ensembl Compara (database)</A> 
Hubbard T., Andrews D., Caccamo M., Cameron G., Chen Y., Clamp M., Clarke L., Coates G., Cox T., Cunningham F., Curwen V., Cutts T., Down T., Durbin R., Fernandez-Suarez X. M., Gilbert J., Hammond M., Herrero J., Hotz H., Howe K., Iyer V., Jekosch K., Kahari A., Kasprzyk A., Keefe D., Keenan S., Kokocinsci F., London D., Longden I., McVicker G., Melsopp C., Meidl P., Potter S., Proctor G., Rae M., Rios D., Schuster M., Searle S., Severin J., Slater G., Smedley D., Smith J., Spooner W., Stabenau A., Stalker J., Storey R., Trevanion S., Ureta-Vidal A., Vogel J., White S., Woodwark C. &amp; Birney E. Ensembl. <I>Nucleic Acids Res.</I> 33 (Database issue): D447-D453 (2005). 
</P><P>[13] <A href="http://www.ncbi.nlm.nih.gov/homologene" class="external text" rel="nofollow">NCBI HomoloGene (database)</A></P><P>[14] <A href="http://doop.abc.hu/" class="external text" rel="nofollow">DoOP (database)</A>
Barta E., Sebestyén E., Pálfy T. B., Tóth G., Ortutay C. P. &amp; Patthy L. DoOP: Databases of Orthologous Promoters, collections of clusters of orthologous upstream sequences from chordates and plants. <I>Nucleic Acids Res.</I> 33 (Database issue): D86-D90 (2004).
</P><P>[15] <A href="http://www.epd.isb-sib.ch/" class="external text" rel="nofollow">EPD (database)</A> 
Schmid C. D., Praz V., Delorenzi M., Périer R. &amp; Bucher P. The Eukaryotic Promoter Database EPD: the impact of in silico primer extension. <I>Nucleic Acids Res.</I> 32 (Database issue): D82-5 (2004).
</P><P>[16] <A href="http://dbtss.hgc.jp/" class="external text" rel="nofollow">DBTSS (database)</A> Wakaguri H., Yamashita R., Suzuki Y., Sugano S., Nakai K.
DBTSS: database of transcription start sites, progress report 2008. <I>Nucleic Acids Res</I>.36(Database issue): D97-101 (2008)
</P><P>[17] <A href="http://bayesweb.wadsworth.org/gibbs/gibbs.html" class="external text" rel="nofollow">Gibbs MotifSampler Lawrence (database)</A> C. E., Altschul S. F., Boguski M. S., Liu J. S., Neuwald A.F., Wootton J. C.  Detecting subtle sequence signals: a Gibbs sampling strategy for multiple alignment. <I>Science</I> 262:208-214(1993)
</P><P>[18] <A href="http://jaspar.genereg.net/" class="external text" rel="nofollow">Jaspar Core Library (database)</A> Sandelin A., Alkema W., Engstrom P., Wasserman W. W., Lenhard B. JASPAR: an open-access database for eukaryotic transcription factor binding profiles.
<I>Nucleic Acids Res.</I> 32(Database issue): D91-4 (2004).
</P><P>[19] Matys V., Kel-Margoulis O. V., Fricke E., Liebich I., Land S., Barre-Dirrie A.,
Reuter I., Chekmenev D., Krull M., Hornischer K., Voss N., Stegmaier P.,
Lewicki-Potapov B., Saxel H., Kel A. E. &amp; Wingender E. TRANSFAC<B>®</B> and its module TRANSCompel<B>®</B>: transcriptional gene regulation in eukaryotes. <I>Nucleic Acids Res.</I> 34 (Database issue): D108-110 (2006). 
</P><P>[20] <A href="http://www.mysql.com/?bydis_dis_index=1" class="external text" rel="nofollow">Mysql</A></P><P>[21] Kel A. E., Gössling E., Reuter I., Cheremushkin E., Kel-Margoulis O. V. &amp; Wingender E. MATCH: A tool for searching transcription factor binding sites in DNA sequences.<I> Nucleic Acids Res.</I> 31: 3576-3579 (2003).
</P><P>[22] Hannenhalli S. &amp; Levy S. Predicting transcription factor synergism. <I>Nucleic Acids Res.</I> 30: 4278-84 (2002).
</P><P>[23] FitzGerald P. C., Shlyakhtenko A., Mir A. A. &amp; Vinson C. Clustering of DNA sequences in human promoters.<I> Genome Res.</I> 14: 1562-74 (2004).
</P><P>[24] Alberts B., Johnson A., Walter P. &amp; Lewis J. <I>Molecular Biology of the Cell.</I> 5th edition, 2008. Garland Science, Chapter 6
</P><P><A href="/Team:Heidelberg/HEARTBEAT_database#Introduction" title="Team:Heidelberg/HEARTBEAT database">[TOP]</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Heidelberg/HEARTBEAT_database">http://2009.igem.org/Team:Heidelberg/HEARTBEAT_database</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/HEARTBEAT_database" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/HEARTBEAT_database" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/HEARTBEAT_database&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/HEARTBEAT_database&amp;oldid=167576" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>