<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Notebook_color_output_nb"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Heidelberg/Notebook_color_output_nb">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Notebook_color_output_nb&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook_color_output_nb&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook_color_output_nb&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Notebook_color_output_nb" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Notebook color output nb</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="main_wrapper"><DIV id="header"><DIV id="navigation"><DIV id="menu" style="position:static"><UL><LI id="mainHome"><A class="aMain" href="https://2009.igem.org/Team:Heidelberg">Home</A></LI><LI id="mainTeam"><A class="aTeam" href="https://2009.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Members#Students">Students</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Members#Advisors">Advisors</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Scientific_Environment">Scientific Environment</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Gallery">Gallery</A></LI></UL><LI id="mainProject"><A class="aProject" href="https://2009.igem.org/Team:Heidelberg/Project">Project</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project">Abstract</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Introduction">Introduction</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Measurement">Measurement</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Synthetic_promoters">Synthetic Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Further">Further Subprojects</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_SaO">Summary &amp; Outlook</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_References">References</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Acknowledgement">Acknowledgements</A></LI><LI id="mainParts"><A class="aParts" href="https://2009.igem.org/Team:Heidelberg/Parts">Parts</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Parts">Parts</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Parts_Characterization">Characterization</A></LI><LI id="mainModeling"><A class="aModeling" href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">What is HEARTBEAT?</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_database">HEARTBEAT Database</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_GUI">HEARTBEAT GUI</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_gui">GUI Documentation</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_network">HEARTBEAT Fuzzy Network Modeling</A></LI><LI id="mainNotebook"><A class="aNotebook" href="https://2009.igem.org/Team:Heidelberg/Notebook">Notebook</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_natural_promoters_overview">Natural Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_synthetic_promoters">Synthetic Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_promoters_cells">Cell Culture, Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_cell_line">Stable Cell Line</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_color_output">Multi-Color Output</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_modeling">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_measure">Measurements</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_MaM">Material &amp; Methods</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_meetings">Team Meetings</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Biosafety">Biosafety</A></LI><LI id="mainEucaryopedia"><A class="aEucaryopedia" href="https://2009.igem.org/Team:Heidelberg/Eukaryopedia">Eukaryopedia</A></LI><LI id="mainSponsors"><A class="aSponsors" href="https://2009.igem.org/Team:Heidelberg/Sponsors">Sponsors</A></LI></DIV></DIV></DIV></DIV><TABLE><TBODY><TR valign="top" style="margin-left: 2px;"><TD width="900px" style="padding: 0 15px 15px 20px; background-color:#ede8e2"><H2><SPAN class="mw-headline" id="Contents"><B>Contents</B></SPAN></H2></TD></TR><TR><TH> Week                       </TH><TH colspan="7">Days
</TH></TR><TR><TD style="text-align:center"> 32
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#8-03-2009" title="Team:Heidelberg/Notebook color output nb">8-03-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#8-04-2009" title="Team:Heidelberg/Notebook color output nb">8-04-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#8-05-2009" title="Team:Heidelberg/Notebook color output nb">8-05-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#8-06-2009" title="Team:Heidelberg/Notebook color output nb">8-06-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#8-07-2009" title="Team:Heidelberg/Notebook color output nb">8-07-2009</A></TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD></TR><TR><TD style="text-align:center"> 33
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#8-12-2009" title="Team:Heidelberg/Notebook color output nb">8-12-2009</A></TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#8-14-2009" title="Team:Heidelberg/Notebook color output nb">8-14-2009</A></TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD></TR><TR><TD style="text-align:center"> 34
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#8-17-2009" title="Team:Heidelberg/Notebook color output nb">8-17-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#8-18-2009" title="Team:Heidelberg/Notebook color output nb">8-18-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#8-19-2009" title="Team:Heidelberg/Notebook color output nb">8-19-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#8-20-2009" title="Team:Heidelberg/Notebook color output nb">8-20-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#8-21-2009" title="Team:Heidelberg/Notebook color output nb">8-21-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#8-22-2009" title="Team:Heidelberg/Notebook color output nb">8-22-2009</A></TD><TD style="text-align:center"> -
</TD></TR><TR><TD style="text-align:center"> 35
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#8-24-2009" title="Team:Heidelberg/Notebook color output nb">8-24-2009</A></TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#8-26-2009" title="Team:Heidelberg/Notebook color output nb">8-26-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#8-27-2009" title="Team:Heidelberg/Notebook color output nb">8-27-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#8-28-2009" title="Team:Heidelberg/Notebook color output nb">8-28-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#8-29-2009" title="Team:Heidelberg/Notebook color output nb">8-29-2009</A></TD><TD style="text-align:center"> -
</TD></TR><TR><TD style="text-align:center"> 36
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#8-31-2009" title="Team:Heidelberg/Notebook color output nb">8-31-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#9-01-2009" title="Team:Heidelberg/Notebook color output nb">9-01-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#9-02-2009" title="Team:Heidelberg/Notebook color output nb">9-02-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#9-03-2009" title="Team:Heidelberg/Notebook color output nb">9-03-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#9-04-2009" title="Team:Heidelberg/Notebook color output nb">9-04-2009</A></TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD></TR><TR><TD style="text-align:center"> 37
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#9-07-2009" title="Team:Heidelberg/Notebook color output nb">9-07-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#9-08-2009" title="Team:Heidelberg/Notebook color output nb">9-08-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#9-09-2009" title="Team:Heidelberg/Notebook color output nb">9-09-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#9-10-2009" title="Team:Heidelberg/Notebook color output nb">9-10-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#9-11-2009" title="Team:Heidelberg/Notebook color output nb">9-11-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#9-12-2009" title="Team:Heidelberg/Notebook color output nb">9-12-2009</A></TD><TD style="text-align:center"> -
</TD></TR><TR><TD style="text-align:center"> 38
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#9-14-2009" title="Team:Heidelberg/Notebook color output nb">9-14-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#9-15-2009" title="Team:Heidelberg/Notebook color output nb">9-15-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#9-16-2009" title="Team:Heidelberg/Notebook color output nb">9-16-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#9-17-2009" title="Team:Heidelberg/Notebook color output nb">9-17-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#9-18-2009" title="Team:Heidelberg/Notebook color output nb">9-18-2009</A></TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD></TR><TR><TD style="text-align:center"> 39
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#9-21-2009" title="Team:Heidelberg/Notebook color output nb">9-21-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#9-22-2009" title="Team:Heidelberg/Notebook color output nb">9-22-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#9-23-2009" title="Team:Heidelberg/Notebook color output nb">9-23-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#9-24-2009" title="Team:Heidelberg/Notebook color output nb">9-24-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#9-25-2009" title="Team:Heidelberg/Notebook color output nb">9-25-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#9-26-2009" title="Team:Heidelberg/Notebook color output nb">9-26-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#9-27-2009" title="Team:Heidelberg/Notebook color output nb">9-27-2009</A></TD></TR><TR><TD style="text-align:center"> 40
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#9-28-2009" title="Team:Heidelberg/Notebook color output nb">9-28-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#9-29-2009" title="Team:Heidelberg/Notebook color output nb">9-29-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#9-30-2009" title="Team:Heidelberg/Notebook color output nb">9-30-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#10-01-2009" title="Team:Heidelberg/Notebook color output nb">10-01-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#10-02-2009" title="Team:Heidelberg/Notebook color output nb">10-02-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#10-03-2009" title="Team:Heidelberg/Notebook color output nb">10-03-2009</A></TD><TD style="text-align:center"> -
</TD></TR><TR><TD style="text-align:center"> 41
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#10-05-2009" title="Team:Heidelberg/Notebook color output nb">10-05-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#10-06-2009" title="Team:Heidelberg/Notebook color output nb">10-06-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#10-07-2009" title="Team:Heidelberg/Notebook color output nb">10-07-2009</A></TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#10-11-2009" title="Team:Heidelberg/Notebook color output nb">10-11-2009</A></TD></TR><TR><TD style="text-align:center"> 42
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_color_output_nb#10-12-2009" title="Team:Heidelberg/Notebook color output nb">10-12-2009</A></TD><TD style="text-align:center"> -</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="8-3-2009"> 8-3-2009 </SPAN></H2><P><B>Colour acquisition</B></P><UL><LI> got YFP/CFP-plasmids of Robert (Cellmembrane, ER, Cytosole)
</LI><LI> Joel pledged to give us further color contructs (Cellmembrane, Mitochondria, Nucleus)
</LI></UL><H2><SPAN class="mw-headline" id="8-4-2009"> 8-4-2009 </SPAN></H2><P><B>Colour acquisition</B></P><UL><LI> transformation of all fluorescent proteins from upstairs (P.32-P.44 --&gt; culture and miniprep tbd tomorrow)
</LI></UL><H2><SPAN class="mw-headline" id="8-5-2009"> 8-5-2009 </SPAN></H2><P><B>Colour acquisition</B></P><UL><LI> transformation of all plasmids with targeted fluorescent proteins (P.32-P.43, P.44 still missing)
</LI><LI> made bacterial stocks of p32-43 and placed in -80Â°C freezer
</LI></UL><H2><SPAN class="mw-headline" id="8-6-2009"> 8-6-2009 </SPAN></H2><P><B>Colour acquisition</B></P><UL><LI>Miniprep of p32, p33, p34, p35, p36, p37, p38, p39, p40, p41, p42, p43
</LI></UL><H2><SPAN class="mw-headline" id="8-7-2009"> 8-7-2009 </SPAN></H2><P><B>Colour acquisition</B></P><UL><LI> Ordered primers for: GPI-attachement signal, Ras1b, YFP, CFP, GFP, mi
</LI></UL><H2><SPAN class="mw-headline" id="8-12-2009"> 8-12-2009 </SPAN></H2><P><B>Colour acquisition</B></P><UL><LI> ordered primers for the mutagenesis PCR of the fluorescent proteins
</LI></UL><TABLE class="wikitable centered" border="2" rules="rows" width="450px" style="border-color:white;"><TBODY><TR style="background-color:#009ACD;"><TD style="text-align:center">Plasmid </TD><TD style="text-align:center"> restriction site </TD><TD style="text-align:center"> mutagenesis position
</TD></TR><TR style="background-color:#63B8FF;"><TD style="text-align:center"> p41 </TD><TD style="text-align:center"> PstI </TD><TD style="text-align:center"> 1257
</TD></TR><TR style="background-color:#63B8FF;"><TD style="text-align:center"> p43 </TD><TD style="text-align:center">PstI </TD><TD style="text-align:center"> 1001
</TD></TR><TR style="background-color:#63B8FF;"><TD style="text-align:center"> p37-Ras </TD><TD style="text-align:center"> PstI </TD><TD style="text-align:center"> 840
</TD></TR><TR style="background-color:#63B8FF;"><TD style="text-align:center"> p37 </TD><TD style="text-align:center"> NheI/EcoRI/PstI </TD><TD style="text-align:center">18/789/840
</TD></TR><TR style="background-color:#63B8FF;"><TD style="text-align:center"> p33 </TD><TD style="text-align:center">NheI/PstI/PstI/EcoRI </TD><TD style="text-align:center">34/259/799/805
</TD></TR><TR style="background-color:#63B8FF;"><TD style="text-align:center"> p35 </TD><TD style="text-align:center">NheI/NheI/PstI </TD><TD style="text-align:center">28/67/148
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="8-14-2009"> 8-14-2009 </SPAN></H2><P><B>Colour acquisition</B></P><UL><LI> two mutagenesis PCR's were done (p33g259a and p37a789g)
</LI></UL><H2><SPAN class="mw-headline" id="8-17-2009"> 8-17-2009 </SPAN></H2><P><B>Colour acquisition</B></P><UL><LI> the other mutagenesis PCR's were done
</LI><LI> and all mutagenesis PCR products were transformed into DH5alpha bacterial cells
</LI></UL><H2><SPAN class="mw-headline" id="8-18-2009"> 8-18-2009 </SPAN></H2><P><B>Colour acquisition</B></P><UL><LI> no colony because of problems with the kanamycin plates
</LI><LI> all mutagenesis PCR's were repeated
</LI><LI>started to test the ligation orientation of inserts into BBb plasmids:
</LI></UL><P>- Digested p31 and a Jet insert with NheI and SpeI.
-Ligated both together for 3 hrs at RT. (insert:vector= 1:1, 3:1, 1:3)
-Transformed bacteria with ligate and plated them on Amp plates.
-Repeated mutagenesis PCR again for (p33, p35, p37, p37-Ras, p41, p43)
</P><H2><SPAN class="mw-headline" id="8-19-2009"> 8-19-2009 </SPAN></H2><P><B>Colour acquisition</B></P><UL><LI> Ligation worked for all 3 plates. Picked 6 colonies for minipreping from each plate for analysis.
</LI><LI> PCR amplification of each of GPI-attachement signal, EBFP alone, NLS alone and EBFP together with NLS.
</LI><LI> Ran EBFP/NLS amplicons on gel and the run seemed to work.
</LI></UL><P><I><B>eBFP/NLS PCR amplification</B></I></P><UL><LI> GPI-attachement signal was amplified overnight.
</LI></UL><H2><SPAN class="mw-headline" id="8-20-2009"> 8-20-2009 </SPAN></H2><UL><LI> Restriction digestion of ligation products after minipreping.
</LI><LI> Ran PCR products for GPI on gel--&gt; nothing!:(
</LI><LI> Restriction digestion showed only efficiency of 5%Â :( (only 1/18 digests showed insertion in the opposing direction)
</LI></UL><H2><SPAN class="mw-headline" id="8-21-2009"> 8-21-2009 </SPAN></H2><P><B>Colour acquisition</B></P><UL><LI> new mutagenesis PCR (without kit)
</LI></UL><H2><SPAN class="mw-headline" id="8-22-2009"> 8-22-2009 </SPAN></H2><P><B>Colour acquisition</B></P><UL><LI>Insert Amplification of mitoneet-eGFP by PCR
</LI></UL><H2><SPAN class="mw-headline" id="8-24-2009"> 8-24-2009 </SPAN></H2><P><B>Colour acquisition</B></P><UL><LI>Restriction digest of mutagenized Plasmids (PstI) and analysis on gel
</LI></UL><UL><LI>Amplified inserts were gel-purificated
</LI></UL><UL><LI>What worked: eBFP, eBFP+NLS, eBFP_kozak, eBFP+NLS_kozak, eGFP, eGFP_kozak
</LI><LI>What didn't: NLS, NLS_kozak, eGFP+mitomeet, mitomeet, eGFP+mitomeet_kozak, mitomeet_kozak,
</LI></UL><H2><SPAN class="mw-headline" id="8-26-2009"> 8-26-2009 </SPAN></H2><P><B>BBBing of Insertsequences</B></P><UL><LI>PCR of cherry, cherry_myrpalm, myrpalm, NLS with kozak Primers to amplify cherry_kozak, cherry_myrpalm_kozak, myrpalm_kozak, NLS_kozak
</LI></UL><UL><LI>Restriction with NheI and SpeI of localisationsequences and Flourophores, Restricted Plasmid was provided by Synthetic Promoter Group and digested with SAP
</LI><LI>Ligation with p31
</LI><LI>Transformation in DH5alpha with ligated Plasmids
</LI><LI>Outplating of Transformed cells on Amp-plates
</LI></UL><H2><SPAN class="mw-headline" id="8-27-2009"> 8-27-2009 </SPAN></H2><UL><LI>Ligation and Transformation did not work (no colonies, except of two on the NLS )
</LI><LI>New PCR with flourophores and localisationsequences, to get higher amounts
</LI></UL><UL><LI>GEl purification of: eGFP, eGFP_kozak, eBFP, eBFP_NLS, eBFP_kozak, eBFP_NLS_kozak, NLS_kozak, cherry, cherry_myrpalm, myrpalm, cherry_kozak, cherry_myrpalm_kozak, myrpalm_kozak
</LI></UL><H2><SPAN class="mw-headline" id="8-28-2009"> 8-28-2009 </SPAN></H2><P><B>BBBing of Insertsequences2.0</B></P><UL><LI>Restrictiondigest of flourophores and localisationsequences with SpeI and NheI (1 h, Buffer 2, BSA)
</LI><LI>Restrictiondigest of p49 with SpeI and NheI (1 h, Buffer 2, BSA) and SAP (30 min), purification
</LI><LI>Nanodrop of digest shows no DNA inside of the samples -- purification was maybe unsuccessful
</LI></UL><H2><SPAN class="mw-headline" id="8-29-2009"> 8-29-2009 </SPAN></H2><P><B>BBBing of Insertsequences2.1</B></P><UL><LI>Restrictiondigest of flourophores and localisationsequences with SpeI and NheI (1 h, Buffer 2, BSA)
</LI><LI>Restrictiondigest of p49 with SpeI and NheI (1 h, Buffer 2, BSA) and SAP (30 min), purification
</LI></UL><H2><SPAN class="mw-headline" id="8-31-2009"> 8-31-2009 </SPAN></H2><P><B>BBBing of Insertsequences2.1 (part 2)</B></P><UL><LI>Ligation of Insertsequences with restricted p49
</LI><LI>Transformation
</LI><LI>Outplating -&gt; Wrong resistance
</LI></UL><H2><SPAN class="mw-headline" id="9-01-2009"> 9-01-2009 </SPAN></H2><P><B>BBBing of Insertsequences2.1 (part 3)</B></P><UL><LI>Transformation
</LI><LI>Plated out on Ampicillin
</LI></UL><H2><SPAN class="mw-headline" id="9-02-2009"> 9-02-2009 </SPAN></H2><P><B>Mutagenesis PCR</B></P><UL><LI>sequences for mutagenesis PCR arrived: P37-Ras, P33, P43 (worked with a possible insertion, so must be tested by transformation to ensure that no alteration occured to the construct that could interfere with the designated function).
</LI><LI>Second mutagenesiscycle of the mutagenized p37-Ras
</LI></UL><P><B>BBBing of Insertsequences2.1 (part 3)</B></P><UL><LI> Picked colonies from the plated out ligations.
</LI><LI> Plated out more bacteria to enhance the number because of the recorded low efficiency of ligation in the correct orientation.
</LI></UL><H2><SPAN class="mw-headline" id="9-03-2009"> 9-03-2009 </SPAN></H2><P><B>BBBing of Insertsequences2.1</B></P><UL><LI>cloning strategy failed, NheI and SpeI cloning system is not effective enough, gel purification of p49 went wrong (Jet was still in)
</LI><LI>New Primers with PstI or EcoRI are neccessary.
</LI></UL><H2><SPAN class="mw-headline" id="9-04-2009"> 9-04-2009 </SPAN></H2><P><B>Mutagenesis PCR of FP/LS</B></P><UL><LI>Ran Mutagenesis PCR for:
</LI></UL><PRE>                         P41 (PstI, primers: 3,4)
                         P37 (PstI, primers: 40,41)
                         P35 (PstI, primers: 53,54)
                         P33 (2nd site:primers 48/48')
</PRE><UL><LI> Transformed bacteria to make more P37, p35, p41--&gt; didnt work when analyzed on gel!
</LI></UL><P><B>BBBing of polyA sequence</B></P><UL><LI>PCR amplification of PolyA out of p6 (PRimers have complete BBB overhang)
</LI><LI>PCR purification
</LI><LI>restriction digest with EcoRI and PstI of backbone (p49) and PolyA
</LI><LI>gel purification of restricted backbone, PCR purification of PolyA
</LI><LI>Ligation
</LI><LI>Transformation
</LI></UL><H2><SPAN class="mw-headline" id="9-07-2009"> 9-07-2009 </SPAN></H2><P><B>Mutagenesis PCR of FP/LS</B></P><UL><LI>Ran Mutagenesis PCR for:
</LI></UL><PRE>                         P41 (PstI, primers: 3,4)
                         P37 (PstI, primers: 40,41)
                         P35 (PstI, primers: 53,54)
                         P33 (2nd site:primers 48/48')
</PRE><P>--&gt; Seemed to have worked on the gelÂ :) so bacteria were transformed.
</P><H2><SPAN class="mw-headline" id="9-08-09">9-08-09 </SPAN></H2><P><B>Mutagenesis PCR of FP/LS</B></P><UL><LI>Ran Mutagenesis PCR for:
</LI></UL><PRE>                         P41 (PstI, primers: 3,4)
                         P37 (PstI, primers: 40,41)
                         P35 (PstI, primers: 53,54)
                         P33 (2nd site:primers 48/48')
</PRE><P>-This time, ran it with a 1:10 dilution of primers, and in each with 3 different plasmid DNA concentrations (10, 20, 50 ng). Also, control plasmid was run with them and a volume of 15 microlitres was used for transformation, although in protocol provided it is mentioned that only 1micro is needed with supercompetent cells.
--&gt; 24hrs later, no colonies are yet present on any of the plates, including the control!
</P><PRE>*Picked a colony from each of p41, p49, p37 and p35 and grew in corresponding resistance LB.
</PRE><H2><SPAN class="mw-headline" id="9-09-2009"> 9-09-2009 </SPAN></H2><P><B>BBBing 2nd gen. of GFP</B></P><UL><LI>PCR with new Primers
</LI></UL><P><B>Mutagenesis PCR</B></P><P>-Plates from Monday's trials were picked. very few colonies for p33-mutax2, p37, p35 but none for p41.
</P><P>-Ran PCR for amplification of Ras and GPI-anchoring signal.
</P><H2><SPAN class="mw-headline" id="9-10-2009"> 9-10-2009 </SPAN></H2><P><B>BBBing 2nd gen. of GFP</B></P><UL><LI>restriction with SpeI and PstI
</LI><LI>Ligation
</LI></UL><P><B>BBBing 2nd gen. mCherry</B></P><UL><LI>PCR with new Primers to get:mCherry, mCherry+myrpalm, myrpalm, mCherry_kozak, mCherry+myrpalm_kozak, myrpalm_kozak, NLS
</LI></UL><P><B>BBBing of PolyA</B></P><UL><LI>PCR with Primers 197 and 198
</LI></UL><H2><SPAN class="mw-headline" id="9-11-2009"> 9-11-2009 </SPAN></H2><P><B>BBBing 2nd gen. of GFP</B></P><UL><LI>Transformation and outplating
</LI></UL><P><B>BBBing 2nd gen. mCherry</B></P><UL><LI>Gelpurification
</LI><LI>Restriction with SpeI and PstI
</LI><LI>Gelpurification
</LI></UL><P><B>BBBing of PolyA</B></P><UL><LI>Restriction of PolyA PCR-product with EcoRI and NheI
</LI><LI>Restriction of both p31new and p55 with EcoRI and SpeI
</LI><LI>Gelpurification
</LI></UL><P><B>Mutagenesis PCR</B></P><UL><LI>Transformation of bacteria with just 5 microlitres of mutagenesis product because it proved that it was optimal for the control.
</LI></UL><P>--&gt; colonies grew and were picked!
</P><H2><SPAN class="mw-headline" id="9-12-2009"> 9-12-2009 </SPAN></H2><P><B>BBBing 2nd gen. of GFP</B></P><UL><LI>Picked colonies
</LI></UL><H2><SPAN class="mw-headline" id="9-14-2009"> 9-14-2009 </SPAN></H2><P><B>BBBing 2nd gen. of GFP</B></P><UL><LI>Miniprep and restriction digest (PstI and EcoRI)
</LI><LI>Gelanalysis shows a positive BBBing of eGFP and eGFP_kzk
</LI></UL><P><B>BBBing 2nd gen. mCherry</B></P><UL><LI>Ligation of Inserts in p49
</LI><LI>Transformation and plating out
</LI></UL><P><B>BBBing of PolyA</B></P><UL><LI>Ligation in p31new and p55
</LI><LI>Transformation and plating out
</LI></UL><P><B>BBbing of inserts</B> (into p49)
</P><UL><LI> Test digested the ligation products of Ras and GPI-targetting sequence PCRed from from the mutated plasmids.
</LI></UL><P>--&gt; positive!!!
</P><P><B>Mutagenesis PCR</B></P><UL><LI>Sequencing results for 10.09.09's sequences show that:
</LI></UL><P>- p37 mutants worked but together with other mutations that seem to have arised. need for a test transfection!
</P><P>-p35 mutants worked but with a 2 base deletion. need for a test transfection!
</P><P>-p33 mutants worked but with funny point mutations. need for a test transfection!
</P><UL><LI> Ran a mutagenesis PCR with a normal Taq polymerase, because mutagenesis Turbo mix was finished.
</LI></UL><P>-p37 (42/43)
</P><P>-p35 (51/52)
</P><P>-p33 (44/45)
</P><P><B>Sequencing primer addition to p31:</B></P><P>-Ran a PCR using Highannealing program of p31 with 201, 202.
--&gt;PCR worked
</P><H2><SPAN class="mw-headline" id="9-15-2009"> 9-15-2009 </SPAN></H2><P><B>BBBing 2nd gen. mCherry</B></P><UL><LI>Picked colonies
</LI></UL><P><B>BBBing of PolyA</B></P><UL><LI>Picked colonies
</LI></UL><P><B>Sequencing primer addition to p31</B></P><P>- Digested p31 with EcoRI and PstI.
-Digest Purified PCR product with NsiI and Mhe-HF--&gt; too little on gel for a second digest. Must redo with more amounts of PCR product tomorrow.
</P><P><B>Mutagenesis PCR</B></P><P>-Transformed bacteria with 5 micro litre of mutagenesis mix--&gt;nothing grew
</P><H2><SPAN class="mw-headline" id="9-16-2009"> 9-16-2009 </SPAN></H2><P><B>Sequencing primer addition to p31</B></P><P>-Did a digest with Mfe-1.
</P><P>-Digest of P31-new looked funny. purified the plasmid and redigested it, because DNA seemed to be a lot and needed to make sure all was digested.
</P><H2><SPAN class="mw-headline" id="9-17-2009"> 9-17-2009 </SPAN></H2><P>-Digested PCR amplicon with Nsi1.
</P><P>-Redigested BB.
</P><P>--&gt; Gel purified amplicon and BB and ligated at 16 degrees overnight.
</P><P>-Prepared Maxiprep cultures of submission plasmid 29-new.
</P><H2><SPAN class="mw-headline" id="9-18-2009"> 9-18-2009 </SPAN></H2><P>-Digested P31-new with EcoRI and PstI to isolate jet insert.
</P><P>-Digested submission plasmid with EcoRI and PstI.
</P><P>--&gt;ligate intron into p29new and jet into p29new.
</P><P>CMV from p48 kept for later use.
</P><H2><SPAN class="mw-headline" id="9-21-2009"> 9-21-2009 </SPAN></H2><P><B>Biobricking</B></P><UL><LI>CMV - cherry
</LI><LI>CMV - myrcherry
</LI><LI>CMV - kzkcherry
</LI><LI>CMV - eGFP
</LI><LI>CMV - kzkeGFP
</LI></UL><P><B>subm(er)izing:</B></P><UL><LI> All transformations worked and test digests show that we have:
</LI></UL><P>-Intron
</P><P>-Jet (S1)
</P><P>-PolyA tail
</P><P>--&gt;problem: No sequencing primers available for verification yet, so nothing is assured, but cloning will be resumed with available components.
</P><P>-Nevertheless, used one of the Jet submission plasmids (Jet1) to do the subcloning of a CMV core promoter with the Jet proximal.
</P><P>--&gt;digested S1 with hindIII and NheI
</P><P><B>Cloning of sequencing primers into p31:</B></P><P>-Ligation into p31 didn't work. No more PCR product is left, so ran PCR again.--&gt;worked.
</P><P>-In the meantime, ligation was tried again with different molar ratios (1:1, 1:3, 3:1) --&gt; no colonies grew.
</P><P><B>Mutagenesis PCR:</B></P><P>-Didn't workÂ :(((
</P><H2><SPAN class="mw-headline" id="9-22-2009"> 9-22-2009 </SPAN></H2><P><B>Biobricking</B>
Preps of the Ligations
</P><P><B>Subm(er)izing:</B></P><P>-Ligated CMV-core (from p48) into digested S1.
</P><P>--&gt;<I>Problem:</I> used the undigested plasmid instead of digested plasmid, so this failed.
</P><P><B>Cloning of sequencing primers into P31:</B></P><P>-Transformed ligations from day before --&gt; didn't work
</P><P>- Digested the new PCR product with NsiI and MfeI using two protocols for the sequential digest.
</P><P><I><B>I:</B></I></P><P>Digest with Mfe-I HF in buffer 3 (low salt) then add NsiI in buffer 4.
</P><P><I><B>II:</B></I></P><P>Digest with Mfe-I, PCR purify and then redigest in NsiI.
</P><P>-Ligated new PCR digest into the old p31.
</P><H2><SPAN class="mw-headline" id="9-23-2009"> 9-23-2009 </SPAN></H2><P><B>Biobricking</B>
Restriction, Purification, Ligation, Transformation of:
</P><UL><LI>cherry - PolyA
</LI><LI>myrcherry - PolyA
</LI><LI>kzkcherry - PolyA
</LI><LI>eGFP - PolyA
</LI><LI>kzkeGFP - PolyA
</LI><LI>CMV - myrpalm
</LI></UL><P><B>Subm(er)izing</B></P><P>-Digest SI with HindIII and NheI.
</P><P>-Ligated CMV into SI.
</P><P>-Transformed and Plated BBBed Sar and GPI on Kana plates. (wrong selection, failed!!!)
</P><P><B>Mutagenesis PCR</B></P><P>-One colony grew.
</P><P><B>Cloning sequencing primers into P31</B></P><P>-Ligated the new PCR product digest into the new p31 digest.
</P><H2><SPAN class="mw-headline" id="9-24-2009"> 9-24-2009 </SPAN></H2><P><B>Mutagenesis PCR</B></P><P>Colonies grew on the 10ng plate of p41
</P><P><B>Cloning of sequencing primers into p31</B></P><P>Transformed bacteria with the ligation from the day before.
</P><P><B> Subm(er)izing</B></P><P>- Transformed the CMV/S1 ligation
</P><P>-Replated GPI and Sar BBBed plasmids on Amp.
</P><P><B>BioBricking</B>
Ligation
</P><UL><LI>cmv-cherry + PolyA
</LI><LI>cmv-myrcherry + PolyA
</LI><LI>cmv-kzkcherry + PolyA
</LI><LI>cmv-eGFP + PolyA
</LI><LI>cmv-kzkeGFP + PolyA
</LI><LI>cmv-NLS
</LI></UL><H2><SPAN class="mw-headline" id="9-25-2009"> 9-25-2009 </SPAN></H2><P><B>BioBricking</B>
Transformation of yesterdays Ligation
</P><P>Ligation of:
</P><UL><LI>cmv-myrpalm + cherry-PolyA
</LI><LI>cmv-myrpalm + kzkcherry-PolyA
</LI><LI>cmv-myrpalm + eGFP-PolyA
</LI><LI>cmv-myrpalm + kzkeGFP-PolyA
</LI><LI>cmv-myrpalm + myrcherry-PolyA
</LI></UL><P><B>Mutagenesis PCR</B></P><P>Minipreped picked colonies for p41 (2x). Only one of which grew in LB.
Sent for sequencing.
</P><P><B>Sequencing primers in p31:</B>

-minipreped picked colonies.
</P><P>-transformed new ligations.
</P><P><B>Subm(er)izing:</B></P><P>-Minipreped picked clones for S21(Jet core and proximal)
</P><P>-Transformed S2 ligations (CMV core and Jet proximal)
</P><P><SPAN style="color:#ff0000"><B>!!!Problem:</B></SPAN> Labels on samples got wiped off during minipreping.
</P><P>Digested all samples with 3 different protocols:
</P><P>1) EcoRI and PstI: S1 should give an insert of ~200 bps. P41 muta shouldn't be cut.
</P><P>2) Hind III and NotI: S1 should be digested and so should p41 muta (difference in lenght of backbone and p41 should have a ~500 bp insert)
</P><P>3)Hind III (same for p31-neu): same as Hind III and NotI but no insert should be detectable. (only linearised fragments of different lenghts).
</P><P>--&gt; Gel showed abnormal mixture of multiple fragments for all digests. Unassociatable with any of the expected plasmids.
possible that plates were left too long in incubator and other bacteria grew on plates.
</P><H2><SPAN class="mw-headline" id="9-26-2009"> 9-26-2009 </SPAN></H2><P>Miniprep and testdigest of yesterdays Transformations
</P><P>Transformation of yesterdays Ligations
</P><P><B>Subm(er)izing:</B></P><P>-Made minipreps of: -p37and p34 amplicons in p49 for minipreps of enough DNA to be digested and used for cloning into submission.
</P><P>--&gt; test digest with EcoRI and PstI shows that the plasmids have not the expected insert. Possible that the plates were too overcrowded and the picked colonies were ones that were negative.
</P><P>-Made minipreps of: S2 bacterial suspensions.
</P><P>--&gt; test digest with Hind and PstI shows that the colonies lack the correct insert.
</P><P><B>Sequencing primers for p31:</B></P><P>-Made minipreps of picked colonies.
</P><P>--&gt; Test digest with EcoRI and PstI shows that they have the correct insert. 4 samples were picked for sequencing.
</P><H2><SPAN class="mw-headline" id="9-27-2009"> 9-27-2009 </SPAN></H2><P>Miniprep and testdigest of yesterdays Transformations
</P><H2><SPAN class="mw-headline" id="9-28-2009"> 9-28-2009 </SPAN></H2><P>Transfektion of:
</P><PRE>   * cmv-cherry + PolyA
   * cmv-myrcherry + PolyA
   * cmv-kzkcherry + PolyA
   * cmv-eGFP + PolyA
   * cmv-kzkeGFP + PolyA
   
   * cmv-myrpalm + cherry-PolyA
   * cmv-myrpalm + kzkcherry-PolyA
   * cmv-myrpalm + eGFP-PolyA
   * cmv-myrpalm + kzkeGFP-PolyA
   * cmv-myrpalm + myrcherry-PolyA 
</PRE><P>Lokalisation of eGFP with myrpalm worked well, unfortunately cherry and kzkcherry did not work
</P><P>Biobricking ofÂ :
</P><PRE>  * CMV-NLS-cherry-PolyA
  * CMV-NLS-kzkcherry-PolyA
  * CMV-NLS-myrcherry-PolyA
  * CMV-NLS-eGFP-PolyA
  * CMV-NLS-kzkeGFPPolyA
</PRE><H2><SPAN class="mw-headline" id="9-29-2009"> 9-29-2009 </SPAN></H2><P>Transformation of:
</P><PRE>  * CMV-NLS-cherry-PolyA
  * CMV-NLS-kzkcherry-PolyA
  * CMV-NLS-myrcherry-PolyA
  * CMV-NLS-eGFP-PolyA
  * CMV-NLS-kzkeGFP-PolyA
</PRE><H2><SPAN class="mw-headline" id="9-30-2009"> 9-30-2009 </SPAN></H2><P><B>Subm(er)izing:</B></P><P>-Test digested S2 colonies minipreped before with NheI and Hind III to make sure that the digest was negative for reasons got to do with the unfamiliar digestion protocol used.
</P><P>--&gt; Colonies were still negative.
</P><P>-Found BBBed inserts of P37 and P34 amplicons and ligated them into submission plasmid digested with SpeI and PstI.
</P><P>-Re-did the CMV/SI ligation.
</P><P>-Restreaked p29-neu.
</P><P>-Picked more S2 colonies.
</P><H2><SPAN class="mw-headline" id="10-01-2009"> 10-01-2009 </SPAN></H2><P>Transfection of 
</P><PRE>  * CMV-NLS-cherry-PolyA
  * CMV-NLS-kzkcherry-PolyA
  * CMV-NLS-myrcherry-PolyA
  * CMV-NLS-eGFP-PolyA
  * CMV-NLS-kzkeGFPPolyA
</PRE><P>Transfection of:
</P><PRE>  * CMV-NLS-cherry-PolyA
  * CMV-NLS-kzkcherry-PolyA
  * CMV-NLS-myrcherry-PolyA
  * CMV-NLS-eGFP-PolyA
  * CMV-NLS-kzkeGFP-PolyA
</PRE><P><B>Subm(er)izing</B></P><P>-Transformed ligations of p37 and p34 amplicons and CMV/SI.
</P><P>-Minipreped the picked colonies of S2 and test digested them with HindIII and PstI and EcoRI and PstI. (negative for insert)
</P><P>-Digested Lars's L and S series of promoters with NheI and PstI.
</P><P><SPAN style="color:#ff0000"><B>!!!Problem</B></SPAN> These were not BBBed parts, and needed to be cut with SpeI and HindIII.
Needed to gel extract the uncut plasmids since they were the last available without need to plate out glycerol stocks.
</P><P>-Digested Corri's natural promoters with EcoRI and PstI.
</P><P>--&gt; Gel extracted the parts.
</P><H2><SPAN class="mw-headline" id="10-02-2009"> 10-02-2009</SPAN></H2><P><B>Transfection result:</B></P><P>NLS does not work, as well as  mCherry. plasma membrane localized mCherry works. 
</P><P><B>NLS restart</B></P><UL><LI>New PCR to get BBB sites. 
</LI><LI>Followed by the assembly with cmv-flourophores to get
</LI></UL><PRE>  *cmv-flourophore-NLS
</PRE><P><B>Subm(er)izing:</B></P><P>-Digested Lars's plasmids with SpeI and HindIII.
</P><P>-Picked colonies for P34 and P37 and CMV ligations in SI.
</P><H2><SPAN class="mw-headline" id="10-03-2009"> 10-03-2009 </SPAN></H2><P><B>Subemerizing</B></P><P>-Minipreped:
</P><UL><LI>p37 and p34 in SI.
</LI><LI>p29 neu
</LI><LI>S2
</LI></UL><P>-Ran digest of Lars's samples on gel for extraction.
--&gt; Gel broke with some of the S series some missing!
</P><H2><SPAN class="mw-headline" id="10-05-2009"> 10-05-2009 </SPAN></H2><P><B>NLS</B></P><UL><LI>Miniprep of cmv-flourophore-NLS
</LI><LI>Biobricking to get cmv.flourophore-NLS-PolyA
</LI></UL><P><B>PolyA</B></P><UL><LI>Subm(er)izing
</LI></UL><H2><SPAN class="mw-headline" id="10-06-2009"> 10-06-2009 </SPAN></H2><UL><LI>Restart PolyA-submerizing
</LI></UL><UL><LI>Transformation and picking of NLS-Test-constructs
</LI></UL><H2><SPAN class="mw-headline" id="10-07-2009"> 10-07-2009 </SPAN></H2><P><B>Time lapse fluorescence movie</B></P><UL><LI>Plan is to make a time lapse movie of induction of NFKb-promoter NIIL10
</LI><LI>Cloning of Videobrickdevices
</LI></UL><PRE>  *Jet-myrcherry-PolyA
  *NIIL10-eGFP-PolyA
</PRE><UL><LI>restriction and ligation
</LI></UL><H2><SPAN class="mw-headline" id="10-11-2009"> 10-11-2009 </SPAN></H2><P><B>Time lapse fluorescence movie</B></P><UL><LI>Transfection of Videobrickdevices
</LI></UL><H2><SPAN class="mw-headline" id="10-12-2009"> 10-12-2009 </SPAN></H2><P><B>Time lapse fluorescence movie</B>
starting acquisition
</P><P><B>BioBricking of GPI</B></P><P>Restriction, Extraction and Ligation to get the following constructs
</P><PRE>  *cmv-GPI-eGFP-PolyA
  *cmv-eGFP-GPI-PolyA
</PRE><P><B>Time lapse fluorescence movie</B>
cells disattached after 4 h so there was no usable filmmaterial
</P><P>restart acquisition this evening
--&gt; less cells, therefore no acquisition 
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Heidelberg/Notebook_color_output_nb">http://2009.igem.org/Team:Heidelberg/Notebook_color_output_nb</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Notebook_color_output_nb" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Notebook_color_output_nb" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook_color_output_nb&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook_color_output_nb&amp;oldid=130370" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>