<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Notebook_natural_promoters"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Heidelberg/Notebook_natural_promoters">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Notebook_natural_promoters&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook_natural_promoters&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook_natural_promoters&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Notebook_natural_promoters" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Notebook natural promoters</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="main_wrapper"><DIV id="header"><DIV id="navigation"><DIV id="menu" style="position:static"><UL><LI id="mainHome"><A class="aMain" href="https://2009.igem.org/Team:Heidelberg">Home</A></LI><LI id="mainTeam"><A class="aTeam" href="https://2009.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Members#Students">Students</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Members#Advisors">Advisors</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Team_Scientific_Environment">Scientific Environment</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Gallery">Gallery</A></LI></UL><LI id="mainProject"><A class="aProject" href="https://2009.igem.org/Team:Heidelberg/Project">Project</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project">Abstract</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Introduction">Introduction</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Measurement">Measurement</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Synthetic_promoters">Synthetic Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_Further">Further Subprojects</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_SaO">Summary &amp; Outlook</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Project_References">References</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Acknowledgement">Acknowledgements</A></LI><LI id="mainParts"><A class="aParts" href="https://2009.igem.org/Team:Heidelberg/Parts">Parts</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Parts">Parts</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Parts_Characterization">Characterization</A></LI><LI id="mainModeling"><A class="aModeling" href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT">What is HEARTBEAT?</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_database">HEARTBEAT Database</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_GUI">HEARTBEAT GUI</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_gui">GUI Documentation</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/HEARTBEAT_network">HEARTBEAT Fuzzy Network Modeling</A></LI><LI id="mainNotebook"><A class="aNotebook" href="https://2009.igem.org/Team:Heidelberg/Notebook">Notebook</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_natural_promoters_overview">Natural Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_synthetic_promoters">Synthetic Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_promoters_cells">Cell Culture, Promoters</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_cell_line">Stable Cell Line</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_color_output">Multi-Color Output</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_modeling">HEARTBEAT</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_measure">Measurements</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_MaM">Material &amp; Methods</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Notebook_meetings">Team Meetings</A></LI><LI><A href="https://2009.igem.org/Team:Heidelberg/Biosafety">Biosafety</A></LI><LI id="mainEucaryopedia"><A class="aEucaryopedia" href="https://2009.igem.org/Team:Heidelberg/Eukaryopedia">Eukaryopedia</A></LI><LI id="mainSponsors"><A class="aSponsors" href="https://2009.igem.org/Team:Heidelberg/Sponsors">Sponsors</A></LI></DIV></DIV></DIV></DIV><TABLE border="0"><TBODY><TR><TD width="900px" style="padding: 0 15px 15px 20px; background-color:#ede8e2"><H2><SPAN class="mw-headline" id="Natural_Promoters"><B>Natural Promoters</B></SPAN></H2><H2><SPAN class="mw-headline" id="Contents"><B>Contents</B></SPAN></H2></TD></TR><TR><TH> Week                       </TH><TH colspan="7">Days
</TH></TR><TR><TD style="text-align:center"> 33
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#8-11-2009" title="Team:Heidelberg/Notebook natural promoters">8-11-2009</A></TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD></TR><TR><TD style="text-align:center"> 34
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#8-18-2009" title="Team:Heidelberg/Notebook natural promoters">8-18-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#8-19-2009" title="Team:Heidelberg/Notebook natural promoters">8-19-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#8-20-2009" title="Team:Heidelberg/Notebook natural promoters">8-20-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#8-21-2009" title="Team:Heidelberg/Notebook natural promoters">8-21-2009</A></TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD></TR><TR><TD style="text-align:center"> 35
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#8-24-2009" title="Team:Heidelberg/Notebook natural promoters">8-24-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#8-25-2009" title="Team:Heidelberg/Notebook natural promoters">8-25-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#8-26-2009" title="Team:Heidelberg/Notebook natural promoters">8-26-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#8-27-2009" title="Team:Heidelberg/Notebook natural promoters">8-27-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#8-28-2009" title="Team:Heidelberg/Notebook natural promoters">8-28-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#8-29-2009" title="Team:Heidelberg/Notebook natural promoters">8-29-2009</A></TD><TD style="text-align:center"> -
</TD></TR><TR><TD style="text-align:center"> 36
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#8-31-2009" title="Team:Heidelberg/Notebook natural promoters">8-31-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#9-01-2009" title="Team:Heidelberg/Notebook natural promoters">9-01-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#9-02-2009" title="Team:Heidelberg/Notebook natural promoters">9-02-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#9-03-2009" title="Team:Heidelberg/Notebook natural promoters">9-03-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#9-04-2009" title="Team:Heidelberg/Notebook natural promoters">9-04-2009</A></TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD></TR><TR><TD style="text-align:center"> 37
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#9-07-2009" title="Team:Heidelberg/Notebook natural promoters">9-07-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#9-08-2009" title="Team:Heidelberg/Notebook natural promoters">9-08-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#9-09-2009" title="Team:Heidelberg/Notebook natural promoters">9-09-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#9-10-2009" title="Team:Heidelberg/Notebook natural promoters">9-10-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#9-11-2009" title="Team:Heidelberg/Notebook natural promoters">9-11-2009</A></TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD></TR><TR><TD style="text-align:center"> 38
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#9-14-2009" title="Team:Heidelberg/Notebook natural promoters">9-14-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#9-15-2009" title="Team:Heidelberg/Notebook natural promoters">9-15-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#9-16-2009" title="Team:Heidelberg/Notebook natural promoters">9-16-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#9-17-2009" title="Team:Heidelberg/Notebook natural promoters">9-17-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#9-18-2009" title="Team:Heidelberg/Notebook natural promoters">9-18-2009</A></TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#9-20-2009" title="Team:Heidelberg/Notebook natural promoters">9-20-2009</A></TD></TR><TR><TD style="text-align:center"> 39
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#9-21-2009" title="Team:Heidelberg/Notebook natural promoters">9-21-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#9-22-2009" title="Team:Heidelberg/Notebook natural promoters">9-22-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#9-23-2009" title="Team:Heidelberg/Notebook natural promoters">9-23-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#9-24-2009" title="Team:Heidelberg/Notebook natural promoters">9-24-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#9-25-2009" title="Team:Heidelberg/Notebook natural promoters">9-25-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#9-26-2009" title="Team:Heidelberg/Notebook natural promoters">9-26-2009</A></TD><TD style="text-align:center"> -
</TD></TR><TR><TD style="text-align:center"> 40
</TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#9-28-2009" title="Team:Heidelberg/Notebook natural promoters">9-28-2009</A></TD><TD style="text-align:center"><A href="/Team:Heidelberg/Notebook_natural_promoters#9-29-2009" title="Team:Heidelberg/Notebook natural promoters">9-29-2009</A></TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD><TD style="text-align:center"> -
</TD></TR></TBODY></TABLE><UL><LI><B> plasmid p31 = plasmid pSMB_MEASURE </B></LI></UL><H2><SPAN class="mw-headline" id="8-11-2009"> 8-11-2009 </SPAN></H2><UL><LI> Plasmids and primers with the specicfic natural promoters had been ordered
</LI></UL><TABLE class="wikitable centered" style="margin: 1em auto 1em auto"><TBODY><TR style="background-color:#99cccc;"><TD style="text-align:center">Plasmids </TD><TD style="text-align:center"> Source
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="text-align:left"> Plasmid with NFAT responsive promoter</TD><TD> Dr. C. Oakes, Heidelberg (DKFZ)
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="text-align:left"> Plasmid with NFkB responsive promoter </TD><TD> Dr. C. Oakes, Heidelberg (DKFZ)
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="text-align:left"> pJC6-GL3 with c-Jun promoter </TD><TD> Addgene, Cambridge
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="text-align:left"> SBE4-Luc with Smad binding element promoter </TD><TD> Addgene,Cambridge
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="text-align:left"> pGL3-RARE-Luciferase with Retinoic Acid Receptor Response Element </TD><TD> Addgene, Cambridge
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="text-align:left"> pGL3-NFAT-Luciferase with NFAT responsive promoter </TD><TD> Addgene, Cambridge
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="text-align:left"> PUMA-Frag1-Luc eith PUMA promoter</TD><TD> Addgene, Cambridge
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="text-align:left"> pLDLR-Luc with LDL receptor promoter </TD><TD> Addgene, Cambridge
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="text-align:left"> pHMGCS-Luc with HMG CoA synthase promoter</TD><TD> Addgene, Cambridge
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="text-align:left"> p52 plasmid with HSP70 promoter </TD><TD> M. Schmidt and F. Hoppe, WÃ¼rzburg
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="8-18-2009"> 8-18-2009 </SPAN></H2><UL><LI> Start of natural promoter project
</LI><LI> The two promoters, Apolipoprotein A-IV (APOAIV) and Cytochrome P450 1A1 (CYP1A1) promoter, were amplified by PCR from human genomic DNA using primers with BBB_standard.
</LI></UL><H2><SPAN class="mw-headline" id="8-19-2009"> 8-19-2009 </SPAN></H2><UL><LI> APOAIV promoter PCR product was digested with NheI and SpeI at 37Â°C for one hour.
</LI><LI> The backbone plasmid (p31; ~5000 bp) was also digested with the same restriction enzymes (at 37Â°C, 1 hour) and it was subsequent digested with SAP (at 37Â°C, 1 hour). The insert of this plasmid p31 is the promoter JeT, which will be cut out after this digest.
</LI></UL><H2><SPAN class="mw-headline" id="8-20-2009"> 8-20-2009 </SPAN></H2><UL><LI> CYP1A1 promoter has the internal restriction site, NheI,...waiting for another CYP1A1 reverse primer with the restriction site PstI instead of NheI.
</LI><LI> The plasmid (p31) and the insert (APOAIV promotor) were purified and ligated at 22 Â°C for 3 hours.
</LI><LI> The ligated product was transformed into DH5alpha cells.
</LI></UL><H2><SPAN class="mw-headline" id="8-21-2009"> 8-21-2009 </SPAN></H2><UL><LI> Picked eight colonies of p31/APOAIV plates and did a mini-preparation (miniprep)
</LI><LI> Test digest of the eight miniprep's using the restriction enzymes SpeI and NheI (at 37Â°C, 1 hour)
</LI><LI> There is an appropriate band for the APOAIV promoter (~4000bp length).
</LI><LI> Miniprep of colony p31/ApoAIV #1, p31/ApoAIV #2 and p31/ApoAIV #3
</LI></UL><H2><SPAN class="mw-headline" id="8-24-2009"> 8-24-2009 </SPAN></H2><UL><LI> The plasmids p31/APOAIV #3 and p31/APOAIV #1 will be sequenced.
</LI></UL><H2><SPAN class="mw-headline" id="8-25-2009"> 8-25-2009 </SPAN></H2><UL><LI> APOAIV sequence did not match the published sequence (NCBI database). --&gt; We have to repeat PCR from genomic DNA.
</LI></UL><H2><SPAN class="mw-headline" id="8-26-2009"> 8-26-2009 </SPAN></H2><UL><LI> Troubleshooting PCR for CYP1A1 and APOAIV using Taq mastermix and Phu with freshly mixed ingredients. We also used various MgCl2 concentrations (1, 2, 3, 4 ul of 25 mM MgCl2 per 50ul reaction) and performed the reaction with and without the addition of DMSO.
</LI><LI> Analytic gel below shows CYP1A1 with Taq -/+ DMSO in the first two lanes with bands at the expected height of around 1200 bp. Lanes 1-10 are different CYP1A1 reactions, lanes 11-20 are APOAIV reactions (Fig. 1). APOAIV shows only nonspecific bands, has to be further optimized using gradient PCR or new primers.
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:573px;"><DIV class="thumbcaption"><B>Figure 1: 1% agarose gel electrophoresis of the PCR products of APOAIV and CYP1A1 promoter.</B>The APOAIV and CYP1A1 promoter PCR products were analysed by agarose gel electrophoresis. Two reaction mixtures were used; one with Taq polymerase and the other one with Phu polymerase and with and without DMSO adding. Besides, the reaction mixtures of <I>Phu</I> have a varying concentration of MGCl2 (1-4 Âµl in a 50 Âµl reaction mixture). Additionally, a DNA ladder (100 - 10000 bp; Fermentas DNA Ladder Mix) is added into the gel for the validation of the DNA product length. The resulting CYP1A1 inserts should be 1229 bp long and APOAIV ~4200 bp. The &quot;CYP1A1 <I>Taq</I>&quot; and &quot;CYP1A1 <I>Taq</I> + DMSO&quot; are the only products, which have the right length. All the other lanes show an unspecific PCR.</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="8-27-2009"> 8-27-2009 </SPAN></H2><UL><LI> Test digest of CYP1A1 insert (1.2 kb) with NheI (digest at position 614 bp) at 37Â°C for one hour.
</LI><LI> PCR of NFkB responsive and NFAT responsive promoter plasmids to amplify promoters from Oakes plasmids p46 and p47. Amplified using PCR high annealing protocol and primers with BBB_standard:
</LI></UL><TABLE class="wikitable centered" style="margin: 1em auto 1em auto"><TBODY><TR style="background-color:#99cccc;"><TD style="text-align:center">Step</TD><TD style="text-align:center"> Time </TD><TD style="text-align:center"> Temperature
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="text-align:center"> 1. </TD><TD style="text-align:center"> 4 min </TD><TD style="text-align:center"> 95Â°C
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="text-align:center"> 2. </TD><TD style="text-align:center">30s </TD><TD style="text-align:center"> 95Â°C
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="text-align:center"> 3. </TD><TD style="text-align:center"> 45s </TD><TD style="text-align:center"> 59Â°C
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="text-align:center"> 4. </TD><TD style="text-align:center"> 2 min </TD><TD style="text-align:center"> 72Â°C
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="text-align:center"> 5. </TD><TD style="text-align:center">Go to step 2 </TD><TD style="text-align:center">24 times
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="text-align:center"> 6. </TD><TD style="text-align:center">30s </TD><TD style="text-align:center">95Â°C
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="text-align:center"> 7. </TD><TD style="text-align:center">45s </TD><TD style="text-align:center">66Â°C
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="text-align:center"> 8. </TD><TD style="text-align:center">2 min </TD><TD style="text-align:center">72Â°C
</TD></TR><TR style="background-color:#9BDDFF;"><TD style="text-align:center"> 9. </TD><TD style="text-align:center">for ever</TD><TD style="text-align:center">4Â°C
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="8-28-2009"> 8-28-2009 </SPAN></H2><UL><LI> Digest of CYP1A1 insert with PstI and SpeI and NFkB and NFAT inserts with PstI and EcoRI at 37Â°C for one hour.
</LI><LI> Digest of plasmid p31 with PstI/SpeI and PstI/EcoRI with SAP digest afterward at 37Â°C for one hour each.
</LI><LI> PCR purification of digested inserts, gel purification of plasmid digests.
</LI></UL><H2><SPAN class="mw-headline" id="8-29-2009"> 8-29-2009 </SPAN></H2><UL><LI> Ligation of p31 and promoters (NFAT, NFkB) at 22Â°C for 3 hours.
</LI><LI> Transformation of ligated plasmids (p31) with the promoters NFAT and NFkB in DH5alpha cells.
</LI></UL><H2><SPAN class="mw-headline" id="8-31-2009"> 8-31-2009 </SPAN></H2><UL><LI> Miniprep of p31 plasmid + NFAT/NFkB promoter.
</LI><LI> Ligation of CYP1A1 promoter and p31 plasmid at 22Â°C for 3 hours.
</LI><LI> Test digest (EcoRI and PstI) of NFAT/NFkB promoter within the p31 plasmid (Fig. 2)at 37Â°C for one hour.
</LI><LI> NFAT#2 and NFAT#5 plasmids sent to sequencing.
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 2: 1% agarose gel electrophoresis for a test digest of the NFkB/p31 plasmid and the NFAT/p31 plasmid.</B> The NFkB/p31 and NFAT/p31 plasmids were digested with EcoRI and PstI at 37Â°C for one hour. The numbers (1,3,4,5 etc.) describe the plasmids of different bacterial colonies. Additionally, a DNA ladder (100 - 10000 bp; Fermentas DNA Ladder Mix) is added into the gel for the validation of the DNA product length. The resulting NFAT insert should be 134 bp long and NFkB 179 bp. NFAT #2 and NFAT #5 seem to be successfully ligated plasmids, because there are small bands at ~ 150 bp. The lanes of NFkB #4 and NFkB #5 show a p31 plasmid band, which is 4919 bp long. But the NFkB inserts could not be recognized. Nevertheless, we will try to sequence them. All other lanes display definetly the wrong products.</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="9-01-2009"> 9-01-2009 </SPAN></H2><UL><LI> Transformation of CYP1A1/p31 plasmid into DH5alpha cells.
</LI></UL><H2><SPAN class="mw-headline" id="9-02-2009"> 9-02-2009 </SPAN></H2><UL><LI> Minipreparation of CYP1A1 colonies (six colonies)
</LI><LI> Test digest of the ligated CYP1A1 construct at 37Â°C for one hour -&gt; no success
</LI><LI> APOAIV promoter: PCR of genomic DNA with old conditions
</LI></UL><H2><SPAN class="mw-headline" id="9-03-2009"> 9-03-2009 </SPAN></H2><UL><LI> Analysis with agarose gelelectrophoresis of the APOAIV-PCR product -&gt; no success
</LI><LI> Digest of CYP1A1 promoter and plasmid p31 with the restriction enzymes, SpeI and PstI (at 37Â°C, 1 hour)
</LI><LI> PCR Purification of the CYP1A1 promoter and gel extraction of p31.
</LI><LI> Ligation of digested CYP1A1 promoter and p31 (over night) at 16Â°C.
</LI></UL><H2><SPAN class="mw-headline" id="9-04-2009"> 9-04-2009 </SPAN></H2><UL><LI> PCR with the plasmid p52 to isolate the HSP70 promoter. Design primer with BBB standard, which were adjusted to the plasmid because of the lack of the p52 sequence.
</LI><LI> PCR with the plasmid p46 to isolate the promoter with NFkB binding sites
</LI><LI> Digest of the plasmid p31 with the restriction enzymes, EcoRI and NheI (at 37Â°C, 1 hour)
</LI></UL><H2><SPAN class="mw-headline" id="9-07-2009"> 9-07-2009 </SPAN></H2><UL><LI> Digest of promoter with NFkB binding sites (NFkB_prom) with NheI and EcoRI (at 37Â°C, 1 hour)
</LI><LI> Digest of half of the volume of the HSP70 promoter with EcoRI and NheI (at 37Â°C, 1 hour)
</LI><LI> Digest of half of the volume of the HSP70 promoter with PstI and SpeI (at 37Â°C, 1 hour)
</LI><LI> Ligation of the digested NFkB_prom and the HSP70 promoter (EcoRI, NheI) with p31 (EcoRI, NheI; 09/04/09) at 22Â°C for three hour. 
</LI><LI> Ligation of the HSP70 promoter (EcoRI, NheI) with p31 (EcoRI, NheI; 09/04/09) at 22Â°C for three hour.
</LI><LI> Ligation of the HSP70 promoter (SpeI, PstI) with p31 (SpeI, PstI; 09/03/09) at 22Â°C for three hour.
</LI><LI> Transformation of the ligated products (above) and the ligated product of CYP1A1 (09/03/09)
</LI></UL><H2><SPAN class="mw-headline" id="9-08-2009"> 9-08-2009 </SPAN></H2><UL><LI> Pick some colonies of transformed bacteria (CYP1A1/p31 construct)
</LI><LI> Miniprep of the CYP1A1/p31 construct.
</LI><LI> Test digest of the CYP1A1/p31 construct at 37Â°C for one hour.
</LI></UL><H2><SPAN class="mw-headline" id="9-09-2009"> 9-09-2009 </SPAN></H2><UL><LI> Gel electrophoresis image of the test digest:
</LI><LI> CYP1A1_promoter/p31 construct was included in three positive colonies (Fig. 3). Therefore, this construct is ready for mutagenesis PCR to remove the NheI restriction site at 614 bp.
</LI><LI> The others have not turned out satisfactory.
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:583px;"><DIV class="thumbcaption"><B>Figure 3: 1% agarose gel electrophoresis for a test digest of the CYP1A1/p31 plasmid.</B> The CYP1A1/p31 plasmids were digested with EcoRI and PstI at 37Â°C for one hour. The numbers (1,3,4,5 etc.) describe the plasmids of different bacterial colonies. Additionally, a DNA ladder (75 -20000 bp); Fermentas DNA 1kb Plus) is added into the gel for the validation of the DNA product length. The resulting CYP1A1 insert should be 1229 bp long. CYP1A1 #2, CYP1A1 #5 and CYP1A1 #8 seem to be successfully ligated plasmids, because there are small bands at ~ 1200 bp. All lanes show a p31 plasmid band, which is 4919 bp long. But other CYP1A1 inserts could not be recognized. All other lanes display definetly the wrong product (JeT, ~180 bp).</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="9-10-09"> 9-10-09 </SPAN></H2><UL><LI> The sequence of CYP1A1 #2, #8 and #5 are right.
</LI><LI> Plate out the Addgene plasmids: pJC6-GL3, SBE4-Luc, pGL3-RARE-Luc, pGL3-NFAT-Luc, PUMA-Frag1-Luc, pLDLR-Luc, pHMGCS-Luc
</LI></UL><H2><SPAN class="mw-headline" id="9-11-09"> 9-11-09 </SPAN></H2><UL><LI> Pick some colonies of the plates (10.09.09) and miniprep.
</LI></UL><H2><SPAN class="mw-headline" id="9-14-2009"> 9-14-2009 </SPAN></H2><UL><LI> PCR of HSP70, NFAT responsive and NFkB responsive promoters.
</LI><LI> Digest of the PCR products by NheI and EcoRI at 37Â°C for one hour.
</LI><LI> Digest of p31_BBB by NheI and EcoRI at 37Â°C for one hour.
</LI><LI> Send to sequencing: pJC6-GL3, SBE4-Luc, pGL3-RARE-Luc, pGL3-NFAT-Luc, PUMA-Frag1-Luc, pLDLR-Luc, pHMGCS-Luc
</LI><LI> Mutagenesis PCR of CYP1A1/p31 #5.
</LI></UL><H2><SPAN class="mw-headline" id="9-15-2009"> 9-15-2009 </SPAN></H2><UL><LI> PCR-Purification with HSP70, NFkB and NFAT
</LI><LI> Gel-extraction of p31_BBB, which was digested by NheI and EcoRI at 37Â°C for one hour.
</LI><LI> Ligation: Hsp70 with p31_BBB, NFAT with p31_BBB and NFkB with p31_BBB (at 22Â°C, 3 hours)
</LI><LI> PCR troubleshooting with NFAT responsive and NFkB responsive promoters.
</LI><LI> Amplification of the p52 plasmid (with the HSP70 insert).
</LI><LI> CYP1A1-muta/p31 PCR product was digested by DpnI (at 37Â°C, 1 hour).
</LI><LI> Transformation of the three ligations (above: Hsp70/p31, NFAT/p31 and NFkB/p31) and CYP1A1-muta/p31
</LI></UL><H2><SPAN class="mw-headline" id="9-16-2009"> 9-16-2009 </SPAN></H2><UL><LI> Arrival of the sequencing results of the ordered Addgene plasmids.
</LI><LI> Design of primers for c-Jun promoter, LDL receptor promoter (LDLR), Retinoic Acid Receptor Response Element (RARE) and HMG CoA synthase promoter (HMG CoA)
</LI><LI> Unfortunaltely, PUMA promoter is not used because of a PstI restrcition site and there is no time for mutagenesis PCR.
</LI><LI> p52 (with HSP70 insert) is amplified in bacteria (DH5alpha).
</LI><LI> Miniprep of the following ligationÂ : Hsp70 with p31_BBB, NFAT with p31_BBB and NFkB with p31_BBB
</LI><LI> The PCR troubleshooting with NFAT and NFkB were analysed by a agarose gel and there were bright bands at ~300 bp, which are a little bit too long. Nevertheless, the inserts could be used for cloning.
</LI></UL><H2><SPAN class="mw-headline" id="9-17-2009"> 9-17-2009 </SPAN></H2><UL><LI> Test digest of the ligations p52 (with HSP70 insert), NFAT and NFkB with EcoRI and PstI (37Â°C, 1 hour -&gt; no right bands!
</LI><LI> Ligation of NFAT and NFkB (of troubleshooting PCR) with p31 plasmid at 22Â°C for one hour.
</LI><LI> No colonies of CYP1A1-muta -&gt; try again the mutagenesis PCR (with special mutagenesis Kit)
</LI></UL><H2><SPAN class="mw-headline" id="9-18-2009"> 9-18-2009 </SPAN></H2><UL><LI> Design of primers with BBB standard for the HSP70 insert by analysis of the p52 sequence.
</LI><LI> Second mutagenesis PCR of CYP1A1
</LI><LI> Transformation of NFAT and NFkB ligations
</LI></UL><H2><SPAN class="mw-headline" id="9-20-2009"> 9-20-2009 </SPAN></H2><UL><LI> pick colonies of NFAT and NFkB
</LI></UL><H2><SPAN class="mw-headline" id="9-21-2009"> 9-21-2009 </SPAN></H2><UL><LI> PCR of RARE, LDLR, HMG CoA synthase and c-Jun promoter using primer with BBB_standard. There were two reaction mixture for each promoter. One with <I>Taq</I> polymerase and one with <I>Phu</I> polymerase.
</LI><LI> Miniprep of NFAT responsive and NFkB responsive promoters and test digest at 37Â°C for one hour -&gt; no right bands on the agarose gel, so that we skipped this part of the project.
</LI></UL><H2><SPAN class="mw-headline" id="9-22-2009"> 9-22-2009 </SPAN></H2><UL><LI> Tranformation of CYP1A1-muta.
</LI><LI> Analysis of the the following PCR products by agarose gel electrophoresis: RARE, LDLR, HMG CoA synthase and c-Jun promoter (Fig. 4-5).
</LI><LI> Digest of these PCR products by the restriction enzymes, NheI and EcoRI, at 37Â°C for one hour.
</LI></UL><TABLE><TBODY><TR valign="top"><TD width="400px" style="background-color:#ede8e5; padding: 0 20px 0 0;"><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:255px;"><DIV class="thumbcaption"><B>Figure 4: 1% agarose gel electrophoresis of the PCR products of LDLR, RARE and c-Jun promoter.</B> The LDLR, RARE and c-Jun promoter PCR products were analysed by agarose gel electrophoresis. Two reaction mixtures were used; one with <I>Taq</I> polymerase and the other one with <I>Phu</I> polymerase. Additionally, a DNA ladder (100 - 10000 bp; Fermentas DNA Ladder Mix) is added into the gel for the validation of the DNA product length. The resulting c-Jun inserts should be 341 bp long, LDLR 363 bp and RARE 212 bp. All bands have the right length and could be used for cloning.</DIV></DIV></DIV></DIV></TD><TD width="450px"><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:147px;"><DIV class="thumbcaption"><B>Figure 5: 1% agarose gel electrophoresis of the PCR products of HMG CoA synthase promoter.</B> The HMG CoA synthase promoter PCR products were analysed by agarose gel electrophoresis. Two reaction mixtures were used; one with <I>Taq</I> polymerase and the other one with <I>Phu</I> polymerase. Additionally, a DNA ladder (100 - 10000 bp; Fermentas DNA Ladder Mix) is added into the gel for the validation of the DNA product length. The resulting HMG CoA synthase inserts should be 363 bp long. All bands have the right length and could be used for cloning.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="9-23-2009"> 9-23-2009 </SPAN></H2><UL><LI> Gel extraction of RARE, LDLR, HMG CoA synthase and c-Jun insert and combination of the two reaction mixtures (<I>Taq</I> and <I>Phu</I>).
</LI><LI> Ligation of these extracted inserts with the digested (NheI and EcoRI) plasmid p31 at 22Â°C for four hours.
</LI><LI> PCR of the p52 (with HSP70 promoter insert) and analysis of the PCR product by agarose gel electrophoresis.
</LI><LI> Digest of the HSP70 promoter by NheI and EcoRI at 37Â°C for one hour 
</LI><LI> Transformation of the RARE/p31, LDLR/p31, c-Jun/p31 and HMG CoA promoter/p31 construct.
</LI></UL><H2><SPAN class="mw-headline" id="9-24-2009"> 9-24-2009 </SPAN></H2><UL><LI> Ligation of the HSP70 promoter with the digested (NheI and EcoRI) p31 plasmid at 22Â°C for three hours.
</LI><LI> Transformation of the HSP70 promoter/p31 construct. 
</LI><LI> Colonies of the RARE/p31, LDLR/p31, c-Jun/p31 and HMG CoA promoter/p31 construct are picked.
</LI></UL><H2><SPAN class="mw-headline" id="9-25-2009"> 9-25-2009 </SPAN></H2><UL><LI> Miniprep of RARE/p31, LDLR/p31, c-Jun/p31 and HMG CoA promoter/p31 construct.
</LI><LI> Colonies of the HSP70 promoter/p31 construct are picked.
</LI></UL><H2><SPAN class="mw-headline" id="9-26-2009"> 9-26-2009 </SPAN></H2><UL><LI> Miniprep of the HSP70 promoter/p31 construct.
</LI><LI> Test digest (NheI and EcoRI) of RARE, c-Jun, LDLR and HMG CoA construct at 37Â°C for one hour (Fig. 6-10).
</LI><LI> Following constructs were send for sequencing:
</LI></UL><P>C-Jun #19, c-Jun #20, c-Jun #21, c-Jun #22, RARE #8, RARE #9, RARE #15, RARE #23, HMG CoA #24, HMG CoA #26, LDLR #3, LDLR #5, LDLR #11, LDLR #12, LDLR #19, LDLR #20, LDLR #21, LDLR #22, LDLR #24, LDLR #25, LDLR #26 and HMG CoA #35 . 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:595px;"><DIV class="thumbcaption"><B>Figure 6: 1% agarose gel electrophoresis of a test digest of the c-Jun/p31 plasmid constructs.</B>The c-Jun/p31 plasmids were digested with EcoRI and PstI at 37Â°C for one hour. The numbers (1,3,4,5 etc.) describe the plasmids of different bacterial colonies. Additionally, a DNA ladder (100 - 10000 bp; Fermentas DNA Ladder Mix) is added into the gel for the validation of the DNA product length. The resulting c-Jun insert should be 341 bp long. C-Jun #19, c-Jun #20, c-Jun #21 and c-Jun #22 seem to be successfully ligated plasmids, because there are small bands at ~ 300 bp. All lanes show a p31 plasmid band, which is 4919 bp long. But other c-Jun inserts could not be recognized and so all other lanes display definetly the wrong product (JeT, ~180 bp).</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:595px;"><DIV class="thumbcaption"><B>Figure 7: 1% agarose gel electrophoresis of a test digest of the RARE/p31 plasmid constructs.</B>The RARE/p31 plasmids were digested with EcoRI and PstI at 37Â°C for one hour. The numbers (1,3,4,5 etc.) describe the plasmids of different bacterial colonies. Additionally, a DNA ladder (100 - 10000 bp; Fermentas DNA Ladder Mix) is added into the gel for the validation of the DNA product length. The resulting RARE insert should be 212 bp long. The RARE inserts are difficult to recognize, because the JeT insert of the original plasmid is ~ 180 bp. Therefore, some good constructs are chosen: RARE #8, RARE #9, RARE #15 and RARE #23.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:589px;"><DIV class="thumbcaption"><B>Figure 8: 1% agarose gel electrophoresis of a test digest of the HMG CoA synthase/p31 plasmid constructs.</B>The HMG CoA synthase/p31 plasmids were digested with EcoRI and PstI at 37Â°C for one hour. The numbers (1,3,4,5 etc.) describe the plasmids of different bacterial colonies. Additionally, a DNA ladder (100 - 10000 bp; Fermentas DNA Ladder Mix) is added into the gel for the validation of the DNA product length. The resulting HMG CoA synthase insert should be 363 bp long. All lanes show a p31 plasmid band, which is 4919 bp long. But the HMG CoA inserts could not be recognized. All lanes display definetly the wrong product (JeT, ~180 bp).</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:594px;"><DIV class="thumbcaption"><B>Figure 9: 1% agarose gel electrophoresis of a test digest of the LDLR/p31 and HMG CoA synthase/p31 plasmid constructs.</B>The LDLR/p31 and HMG CoA synthase/p31 plasmids were digested with EcoRI and PstI at 37Â°C for one hour. The numbers (1,3,4,5 etc.) describe the plasmids of different bacterial colonies. Additionally, a DNA ladder (100 - 10000 bp; Fermentas DNA Ladder Mix) is added into the gel for the validation of the DNA product length. The resulting LDLR insert should be 363 bp long and HMG CoA 363 bp. HMG CoA #24, HMG CoA #26, LDLR #3, LDLR #5, LDLR #11 and LDLR #12 seem to be successfully ligated plasmids, because there are small bands at ~ 350 bp. All lanes show a p31 plasmid band, which is 4919 bp long. But other HMG CoA and LDLR inserts could not be recognized. All other lanes display definetly the wrong product (JeT, ~180 bp).</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:595px;"><DIV class="thumbcaption"><B>Figure 10: 1% agarose gel electrophoresis of a test digest of the LDLR/p31 and HMG CoA synthase/p31 plasmid constructs.</B>The NFkB/p31 and NFAT/p31 plasmids were digested with EcoRI and PstI at 37Â°C for one hour. The numbers (1,3,4,5 etc.) describe the plasmids of different bacterial colonies. Additionally, a DNA ladder (100 - 10000 bp; Fermentas DNA Ladder Mix) is added into the gel for the validation of the DNA product length. The resulting LDLR insert should be 363 bp long and HMG CoA 363 bp. LDLR #19, LDLR #20, LDLR #21, LDLR #22, LDLR #24, LDLR #25, LDLR #26 and HMG CoA #35 seem to be successfully ligated plasmids, because there are small bands at ~ 350 bp. All lanes show a p31 plasmid band, which is 4919 bp long. But other HMG CoA and LDLR inserts could not be recognized. All other lanes display definetly the wrong product (JeT, ~180 bp).</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="9-28-2009"> 9-28-2009 </SPAN></H2><UL><LI> Right constructs of RARE, c-Jun, LDLR and HMG CoA are: c-Jun #19, c-Jun #20, c-Jun #21, c-Jun #22, RARE #8, RARE #23, LDLR #19, LDLR #20, LDLR #21 and HMG CoA #35. 
</LI><LI> Test digest (NheI and EcoRI) of HSP70 at 37Â°C for one hour (Fig. 11).
</LI><LI> Following construct is send for sequencing: HSP70 #9.
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:589px;"><DIV class="thumbcaption"><B>Figure 11: 1% agarose gel electrophoresis for a test digest of the HSP70/p31 plasmid construct.</B>The HSP70/p31 plasmids were digested with EcoRI and PstI at 37Â°C for one hour. The numbers (1,3,4,5 etc.) describe the plasmids of different bacterial colonies. Additionally, a DNA ladder (100 - 10000 bp; Fermentas DNA Ladder Mix) is added into the gel for the validation of the DNA product length. The resulting HSP70 insert should be 402 bp long. HSP70 #9 seem to be a successfully ligated plasmid, because there is a small bands at ~ 400 bp. All lanes show a p31 plasmid band, which is 4919 bp long. But other HSP70 inserts could not be recognized and so all other lanes display definetly the wrong product (JeT, ~180 bp).</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="9-29-2009"> 9-29-2009 </SPAN></H2><UL><LI> Right constructs of HSP70 is: HSP70 #9.
</LI></UL><H2><SPAN class="mw-headline" id="10-12-2009"> 10-12-2009 </SPAN></H2><UL><LI> TECAN (plate reader) measurement of the natural LDL receptor promoter and the HMG CoA synthase promoter. These promoters, which are coupled to GFP, were cotransfected with a reference plasmid including JeT coupled to mCherry and were induced by Lipoprotein Deficient Serum (See: <A href="/Team:Heidelberg/Notebook_promoters_cells" title="Team:Heidelberg/Notebook promoters cells"> Cell Culture</A>). 
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 12: The HMG CoA synthase promoter and the LDL receptor promoter induced approximately 25-35% by Lipoprotein Deficient Serum.</B> The LDL receptor promoter and HMG CoA synthase promoter  is coupled to the fluorecent protein GFP. This GFP fluorescence was measured by TECAN once (Ã  three replicates). Background fluorescence was substracted, and fluorescence levels are plotted relative to JeT. The standard deviations are indicated by the error bars.</DIV></DIV></DIV></DIV><P><B>Notebook information about the measurement of natural promoters is found in the Notebook <A href="/Team:Heidelberg/Notebook_measure/NotebookFC" title="Team:Heidelberg/Notebook measure/NotebookFC"> Measurement</A> and <A href="/Team:Heidelberg/Notebook_promoters_cells" title="Team:Heidelberg/Notebook promoters cells"> Cell Culture</A>.</B></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Heidelberg/Notebook_natural_promoters">http://2009.igem.org/Team:Heidelberg/Notebook_natural_promoters</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Notebook_natural_promoters" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Notebook_natural_promoters" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook_natural_promoters&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook_natural_promoters&amp;oldid=127307" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>