"
Page
Discussion
View source
History
teams
Log in
 
Team:Heidelberg/Modeling network
From 2009.igem.org
Home
Team
Overview
Students
Advisors
Scientific Environment
Gallery
Project
Abstract
Introduction
Measurement
Synthetic Promoters
HEARTBEAT
Further Subprojects
Summary & Outlook
References
Acknowledgements
Parts
Parts
Characterization
HEARTBEAT
What is HEARTBEAT?
HEARTBEAT Database
HEARTBEAT GUI
GUI Documentation
HEARTBEAT Fuzzy Network Modeling
Notebook
Natural Promoters
Synthetic Promoters
Cell Culture, Promoters
Stable Cell Line
Multi-Color Output
HEARTBEAT
Measurements
Material & Methods
Team Meetings
Biosafety
Eukaryopedia
Sponsors
HEARTBEAT Fuzzy Modeling
Introduction
The research field of regulation of gene expression in eukaryotes is a field of biological research growing rapidly [1,2] . Hereby the interaction of DNA with certain proteins known as transcription factors (TFs) plays an essential role for the complex mechanism of transcriptional activation [3,4] . One strong focus of synthetic biology aims at the reconstruction of such gene regulatory networks [5] . To act within the scope of synthetic biology’s duties, the iGEM Team Heidelberg 09 claims that any synthetic promoter can be constructed by using our two methods for the construction of synthetic promoters . However, the only efficient way to construct systems of high complexity (such as computers or airplanes) is simulating these systems on the computer prior to construction [6] . In our case, this strongly emphasizes the necessity of HEARTBEAT (Heidelberg Artificial Transcription Factor Binding Site Engineering and Assembly Tool) which comprises data analysis (data analysis ( HEARTBEAT DB ), a graphical user interface ( HEARTBEAT GUI ) and network modeling ( HEARTBEAT fuzzy network (FN) modeling ).
Contributing to the HEARTBEAT project, HEARTBEAT FN focuses on simulating the promoter activity by integrating a variety of signals and sequence characteristics as well as on predicting distinct pathway functionalities. This, especially in eukaryotic systems, is a tough challenge since transcriptional activity of a gene is not directly correlated to protein expression [7] .
For this purpose we propose fuzzy logic (FL) modeling as an approach to logic-based modeling which is capable of incorporating qualitative data but producing quantitative predictions. New insights will be provided about the operation of gene regulatory networks and relationships between promoter sequence composition and TF-DNA interaction will be unraveled that is understood only marginally so far [8,9] .
Background / Motivation
We present two different approaches for promoter design resulting in three different types of synthetic promoters: randomly assembledconstitutive and inducible promoters as well as rationally designed promoters. As an additional type of promoters those occurring in nature can be integrated into vector systems. These heterogeneous cocktail of promoters can be combined for a precise regulation of pathways. This represents the power of our entire HEARTBEAT project.  Synthesized promoters can be then used e.g. as a combinatorial gene therapy, i.e. several promoters that are of different types and/or have different strength will be applied as treatment agents. Therefore, a model that not only simulates single promoter activity and following gene expression but also accurately predicts gene expression from combined promoter sequences is indispensable.
We constructed a Fuzzy Logic model to provide a formal mathematical framework for prediction of combined activity of multiple promoters upon several stimuli and to gain insight into the mechanisms that generate diverse expression levels.
Achievements
A Short Introduction into Fuzzy Logic Modeling
Fuzzy Logic is a rule-based approximate artificial reasoning method developed by Lotfi Zadeh in 1965. Its motivation is  the observation that humans often think and communicate in a vague way, and yet can make precise decisions [10] . It has been widely used in engineering and Artificial Intelligence approaches such as Fuzzy Controllers and Fuzzy Expert Systems. Fuzzy Logic has also been used for the modeling of biological pathways [11] and very recently to analyze gene regulatory networks [12] . Key advantages of Fuzzy logic-based approaches are (i) the ability to construct models based on prior knowledge of the system and experimental data and (ii) encode intermediate states for inputs and outputs, thus improving other logic-approaches that can only deal with ON/OFF states such as Boolean models [13] and (iii) simulations can be derived from both qualitative and quantitative data, both of which can be cast into the form of IF-THEN rules. Thus, FL constitutes a powerful approach for the understanding of heterogeneous datasets.
In our project, the complete set of rules will capture the behavior of each promoter in a Multiple-Input Single-Output (MISO) Fuzzy Logic model. Combining the MISO models in a network of all promoters will constitute the final Multiple-Input Multiple-Output (MIMO) model allowing for the simulation and prediction of combined activation of pahways regulated by our promoters. A key advantage of this methodology towards understanding the exclusive pathway activation of our promoters of interest is the possibility to study not only the individual activity of each promoter but also the combined activity, as the signal progresses from one MISO to another.
References
[1] Harbison, C. T. et al. Transcriptional regulatory code of a eukaryotic genome. Nature 431, 99-104 (2004).
[2] Hu, Z., Killion, P. J. & Iyer, V. R. Genetic reconstruction of a functional transcriptional regulatory network. Nature Genet. 39, 683-687 (2007).
[3] Gertz, J., Siggia E. D. & Cohen, B. A. Analysis of combinatorial cis-regulation in synthetic and genomic promoters. Nature 457. 215-218 (2009)
[4] Roider, H. G. et al. Predicting transcription factor affinities to DNA from a biophysical model. Bioinformatics 23, 134-141 (2006)
[5] ref to come
[6] Andianantoandro, E. et al. Synthetic biology: new engineering rules for an emerging discipline. Mol Sys Biol (2006)
[7] Alberts, B. et al. Molecular Biology of the Cell, 5th edition. Garland Science, 2008, Chapter 6
[8] Vardhanabhuti, S., Wang, J. & Hannenhalli, S. Position and distance specificity are important determinants of cis-regulatory motifs in addition to evolutionary conservation. Nucl Acid Res 35, 3203-3213 (2007).
[9] Yokoyama, K. D., Ohler, U. & Wray, G. A. Measuring spatial preferences at fine-scale resolution identifies known and novel cis-regulatory element candidates and functional motif-pair relationships. Nucl Acid Res, 1-21 (2009)
[10] Nelles, O. Nonlinear System Identification. Springer, 2000.
[11] Bosl, W. J. BMC systems biology 1, 13 (2007).
[12] Mathematical modeling of the lambda switch:a fuzzy logic approach.
[13] B. B. Aldridge, J. Saez-Rodriguez, J. L. Muhlich et al., PLoS  computational biology 5 (4), e1000340 (2009).
Retrieved from " http://2009.igem.org/Team:Heidelberg/Modeling_network "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
