"
Page
Discussion
View source
History
teams
Log in
 
Team:Heidelberg/Project Natural promoters
From 2009.igem.org
Home
Team
Overview
Students
Advisors
Scientific Environment
Gallery
Project
Abstract
Introduction
Measurement
Synthetic Promoters
HEARTBEAT
Further Subprojects
Summary & Outlook
References
Acknowledgements
Parts
Parts
Characterization
HEARTBEAT
What is HEARTBEAT?
HEARTBEAT Database
HEARTBEAT GUI
GUI Documentation
HEARTBEAT Fuzzy Network Modeling
Notebook
Natural Promoters
Synthetic Promoters
Cell Culture, Promoters
Stable Cell Line
Multi-Color Output
HEARTBEAT
Measurements
Material & Methods
Team Meetings
Biosafety
Eukaryopedia
Sponsors
Natural Promoters
Introduction
Eukaryotic promoters are very complex and difficult to characterize. They usually lie upstream of their target gene, but they can have regulatory elements that are several kilobases away from the promoter. Fig. 1 shows some of the factors involved in the transcriptional regulation of polymerase II [1]. For our assay we want to utilize several natural promoters to sense certain conditions in the cell, such as hypoxia, DNA damage and heat. The selection criteria for these promoters are that they have to be very specific to a transcription factor and they have to be easily induced and inhibited. These natural promoters will give us a chance to start testing and fine-tuning our output system early on which can then be applied to our synthetic promoters.
Figure 1: Diagram of various components involved in regulation of Pol-II transcription. The regulatory elements include a core promoter where Pol II and a variety of general factors bind to specific DNA motifs including the TATA box , which is bound by the "TATA-binding protein" (TBP). The Initiator motif (INR), where Pol II and certain other core factors bind, is present in many Pol II-regulated genes and the Downstream Promoter Element (DPE), which is present in a subset of Pol II genes, and where additional core factors bind. The proximal promoter is upstream (5') of the core promoter, Pol II target genes often have a Proximal Promoter region that spans up to 1000 bp contains a number of functional DNA binding sites for a specific set of transcription activator (TA) and transcription repressor (TR) proteins. Most Pol II regulated genes in higher eukaryotes have one or more distal Enhancer regions which are essential for proper regulation of the gene, often in a cell or tissue-specific pattern. Like the proximal promoter region, each of the distal enhancer regions typically contain a cluster of binding sites for specific TA and/or TR DNA-binding factors, rather than just a single site.
Results
After cloning the natural promoters in our screening plasmid pSMB_MEASURE, they were tested in different cell lines with appropriate induction substances. The transfected cells were measured by flow cytometry and TECAN. Thereby, the c-Jun promoter and the CYP1A1 promoter were tested by flow cytometry and the LDL receptor promoter, HSP70 promoter and HMG CoA synthase promoter were measured by TECAN. The CYP1A1 measurement could not be analyzed, due to the low cell number. In Fig. 2, the flow cytometry measurement data of the c-Jun promoter is displayed, which was induced by Epidermal Growth Factor for three hours. The results present an induction of more than 100%. Fig. 3 presents the TECAN results of the both SREBP responsive promoters, LDL receptor and HMG CoA synthase promoter.
Figure 2: Flow cytometry results of the natural promoter c-Jun (Ap-1). HeLa cells were cotransfected with the natural c-Jun promoter coupled to GFP and a reference plasmid including the promoter JeT coupled to mCherry. The relative fluorescence (REU) of GFP, relative to our reference promoter JeT, is measured 3 hours after induction with EGF (diluted 1:2000 in DMEM++). The standard deviation is indicated by the error bars.
Figure 3: TECAN results of the natural LDL receptor promoter and HMG CoA synthase promoter. The natural LDL receptor promoter and HMG CoA synthase promoter are coupled to GFP and were cotransfected with a reference plasmid, which includes the standard promoter JeT coupled to mCherry, into HeLa cells. The GFP fluorescence relative to JeT is measured after the induction with Lipo Deficient Serum medium. The error bars represent the standard deviation.
Discussion
The evaluation of the c-Jun natural promoter looks very promising. The flow cytometry measurement should be repeated at least three times, which could not be accomplished in the given time frame. Besides, this measurement could be also analyzed by fluorescence microscopy and image analysis for validation. The TECAN measurement of the SREBP responsive promoters, LDL receptor and HMG CoA synthase promoter, show a significant induction. This should also be validated by microscopy and flow cytometry. Unfortunately, the appropriate induction conditions for the other natural promoters could not be established in the remaining time.
Methods
The different natural promoter inserts (see table 1) were integrated in the pSMB_MEASURE . For this purpose, the promoters of the ordered plasmids/genomic DNA were amplified by PCR and primers with BBb standard. These PCR products were digested by restriction enzymes and purified. The purified inserts were ligated into the pSMB_MEASURE plasmid and transformed into DH5alpha bacterial cells. After mini-preparation, the plasmids were sequenced, which revealed the right sequences. The natural promoters were transfected in different cell lines and induced with appropriate substances, which is presented in table 1. The analysis was made by flow cytometry or TECAN measurement. More details about the method are found in the Notebook .
Table 1: Plasmids
Source
Regulated by
Induction
Cell Line
pJC6-GL3 with c-Jun promoter (Plasmid 11979) [2]
Addgene, Cambridge
Epidermal Growth Factor
Epidermal Growth Factor
HeLa
pGL3-RARE-Luciferase with Retinoic Acid Receptor Response Element (Plasmid 13458) [3]
Addgene, Cambridge
Retinoic Acid Receptor
Retinoic Acid
unknown
pLDLR-Luc with LDL receptor promoter (Plasmid 14940) [4]
Addgene, Cambridge
SREBP
Lipoprotein Deficient Serum
HeLa
pHMGCS-Luc with HMG CoA synthase promoter (Plasmid 14947) [4]
Addgene, Cambridge
SREBP
Lipoprotein Deficient Serum
HeLa
p52 plasmid with HSP70 promoter [5]
M. Schmidt and F. Hoppe, Würzburg
Heat
Heat
MCF-7
CYP1A1 [6,7]
genomic DNA
AhR responsive
Benzo-a-pyren
HeLa
References
[1] Alberts, B. et al: „Molekularbiologie der Zelle“ (Book. 4th edition, 2004. WILEY-VCH) Chapter 6, p. 357-359.
[2] Clarke N., Arenzana N., Hai T., Minden A., Prywes R. Epidermal Growth Factor Induction of the c-jun Promoter by a Rac Pathway. Mol Cell Biol 18, 1065–1073 (1998).
[3] Hoffman L. M., Garcha K., Karamboulas K., Cowan M. F., Drysdale L. M., Horton W. A., Underhill T. M. BMP action in skeletogenesis involves attenuation of retinoid signaling. J Cell Biol 174, 101–113 (2006).
[4] Castoreno A. B., Wang Y., Stockinger W., Jarzylo L. A., Du H., Pagnon J. C., Shieh E. C., Nohturfft A. Transcriptional regulation of phagocytosis-induced membrane biogenesis by sterol regulatory element binding proteins. Proc Natl Acad Sci U S A 102, 13129–13134 (2005).
[5] Schmidt M., Heimberger T., Gruensfelder P., Schler G., Hoppe F. Inducible promoters for gene therapy of head and neck cancer: an in vitro study. Eur Arch Otorhinolaryngol 261, 208–215 (2004).
[6] Jorgensen, E. C. B. and Autrup, H. Autoregulation of human CYP1A1 gene promoter activity in HepG2 and MCF-7 cells. Carcinogenesis 17, 435-441 (1996).
[7] Schreiber T. D., Köhle C., Buckler F., Schmohl S., Braeuning A., Schmiechen A., Schwarz M., Münzel P.A. Regulation of CYP1A1 expression by the antioxidant Tert butylhydroquinone. Drug Metabolism And Disposition 43, 1096–1101 (2006).
Retrieved from " http://2009.igem.org/Team:Heidelberg/Project_Natural_promoters "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
