<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Chalmers-Gothenburg_Constructs skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Chalmers-Gothenburg/Constructs</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="main"><DIV id="my_content"><DIV id="section1"><DIV id="section1-menu"><NAV><UL><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg">Home</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Students">Students</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Supervisors">Supervisors</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Gallery">Gallery</A></LI></UL><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Team" class="arrow">Team</A><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Project">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Theory">Theory</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Constructs">Constructs</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/OtherApplications">Other Applications</A></LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Project" class="arrow">Project</A><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/ProjectResults">Project Results</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/BioBricks">Bio Bricks</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Medals">Medals</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Results">Results</A></LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Achievements" class="arrow">Achievements</A><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Detection">Detection</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/SafetySwitch">Safety Switch</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Simulation">Simulation</A></LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Modeling" class="arrow">Modeling</A><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/iGEMCommunity">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/NextGeneration">Next Generation</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Society">Society</A></LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/HumanPractices" class="arrow">Human Practices</A><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Schedule">Lab Journal</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Protocols">Protocols</A></LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Notebook" class="arrow">Notebook</A><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/LabSafety">Lab Safety</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Ethics">Ethics</A></LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Safety" class="arrow">Safety</A><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Sponsors">Sponsors</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Acknowledgements">Acknowledgements</A></LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Thanks" class="arrow">Thanks!</A></NAV></DIV><DIV id="section1-content"><H2> Constructs </H2><P> The products of the project are eight different constructs which in combination creates systems with the functions we aim to develop. To minimize the amount of constructs, two parts were integrated together on some of the vectors.  Every vector contains an origin of replication and a region coding for ampicillin resistance, <I>AmpR</I>, allowing for growth and selection in <I>E. coli</I>. They also have an auxotrophic marker (Leucine, Uracil, or Histidine) which allows for selection in auxotrophic cells. Each vector also integrates the genes in different locations of the chromosomes, e.g. p0255 (pX-2-LoxP-KIURA) implements the genes into chromosome 10 on the 2nd location [1]. </P><UL style="margin-bottom: 15px"><LI>The detection system has a total of four different constructs (1-4), they can be combined in three different ways which alters the amplification of the signal in the system.</LI><LI>The safety switch has a lone construct (5).</LI><LI>The DNA repair system has a total of three different constructs (6-8) which together contains all the parts required for the ESDSA procedure.</LI></UL><P><B>The constructs are as follows:</B></P><H3> Detection system</H3><H4>Construct 1:</H4><P> The first construct of the detection system contains the gene for the fusion receptor coupled to the pTDH3 promoter which is naturally activated in <I>S. cerevisiae</I> resulting in the creation of a pheromone pathway activated by pheromones of <I>S. pombe</I>. The construct also contains the gene for the output signal, <I>mRFP</I>, coupled to the pCYC1m promotor which’s expression is induced by the dCas9-Vp64 transcription factor.</P><P> This construct is intended to be used in conjunction with a construct that “expresses” the transcription factor, and thus has no function on its own. </P><P> The construct is implemented in a plasmid p0255 (pX-2-LoxP-KIURA) which has a gene coding for a uracil forming enzyme. </P><P class="figuretext">Figure 1. Construct 1. The figure depicts the first construct of the detection system which contains the genes for the fusion receptor and for <I>mRFP</I> expression coupled to the pCYC1m promoter. </P><H4>Construct 2:</H4><P> The second construct of the detection system contains the CGC1 gene which expresses the transcription factor dCas9-Vp64. The expression of said transcription factor is coupled to the pFUS1 promoter which occurs in the pheromone response pathway of <I>S. cerevisiae</I>. </P><P> This construct is intended to be used in conjunction with construct one to create a system which expresses the transcription factor as an amplification method but contains no feed-back activation loop. </P><P> The construct is implemented in a plasmid p0257 (pX-3-LoxP-KILEU) which has a gene coding for a leucine forming enzyme. </P><P class="figuretext">Figure 2. Construct 2. The figure depicts the second construct of the detection system which contains the genes for the transcription factor dCas9-Vp64 coupled to the pFUS1 promoter. </P><H4>Construct 3:</H4><P> The third construct of the detection system contains the CGC1 gene coupled to the pCYC1m promoter. If the expression of the transcription factor is initialized in a yeast with this construct implemented a feed-back activation loop is created which creates more transcription factor. </P><P> This construct is intended to be used in conjunction with construct one and two to create a system which expresses the transcription factor as an amplification method and contains a feed-back activation loop further increasing the amount of transcription factor. </P><P> The construct is implemented in a plasmid p0258 (pX-4-LoxP-HIS5) which has a gene coding for a histidine forming enzyme. </P><P class="figuretext">Figure 3. Construct 3. The figure depicts the third construct of the detection system which contains the genes for the transcription factor dCas9-Vp64 coupled to the pCYC1m promoter. </P><H4>Construct 4</H4><P> The fourth construct of the detection system contains the gene for the fusion receptor coupled to the pTDH3 promoter and the gene for the output signal, mRFP, coupled to the pFUS1 promoter. </P><P> This is the only construct in the detection system intended to work on its own, its parts create a system capable of receiving external signals through the fusion receptor and express an output signal in the form of mRFP coupled to signaling cascade of the pheromone response pathway. </P><P> The construct is implemented in a plasmid p0255 (pX-2-LoxP-KIURA) which has a gene coding for a uracil forming enzyme. </P><P class="figuretext">Figure 4. Construct 4. The figure depicts the fourth construct of the detection system which contains the genes for the fusion receptor and for <I>mRFP</I> expression coupled to the pFUS1 promoter. </P><H3>Safety Switch</H3><H4>Construct 5</H4><P> The lone construct of the safety switch contains the gene for the expression of the potentially damaging protein TPK2 coupled to the series of two promoters, pTEF1 and pSUC2. </P><P> This construct is by itself sufficient to create the desired safety switch as the remaining parts of the system is already present in the genome of <I>S. cerevisiae</I>. </P><P> The construct is implemented in a plasmid p0255 (pXII-1-LoxP-KILEU) which has a gene coding for a leucine forming enzyme. </P><P class="figuretext">Figure 5. Construct 5. The figure depicts the lone construct of the safety switch which contains the genes for the potentially damaging protein TPK coupled to the promoters pTEF1 and pSUC2. </P><H3>PUR</H3><H4>Construct 6</H4><P> The first construct of the DNA repair system contains the genes for the expression of the RecA and RecJ proteins. The genes are respectively coupled to the pPGK1 and pTDH3 promoters which are naturally activated in <I>S. cerevisiae</I>. </P><P> This construct is intended to be used in conjunction with the other constructs of the DNA repair system as all parts from these constructs are required for the DNA repair to work. </P><P> The construct is implemented in a plasmid p0389 (pXI-2- LoxP-KIURA3) which has a gene coding for a uracil forming enzyme. </P><P class="figuretext">Figure 6. Construct 6. The figure depicts the first construct of the DNA repair system which contains the genes for the RecA and RecJ proteins coupled to receptors whose expression is induced naturally in <I>S. cerevisiae</I>. </P><H4>Construct 7</H4><P> The second construct of the DNA repair system contains the genes for the expression of the Ssb and RecD2 proteins. The genes are respectively coupled to the pTPI and pPGK1 promoters which are naturally activated in <I>S. cerevisiae</I>. </P><P> This construct is intended to be used in conjunction with the other constructs of the DNA repair system as all parts from these constructs are required for the DNA repair to work. </P><P> The construct is implemented in a plasmid p0388 (pXI-1- LoxP-KILEU2) which has a gene coding for a leucine forming enzyme. </P><P class="figuretext">Figure 7. Construct 7. The figure depicts the second construct of the DNA repair system which contains the genes for the Ssb and RecD2 proteins coupled to receptors whose expression is induced naturally in <I>S. cerevisiae</I>. </P><H4>Construct 8</H4><P> The third construct of the DNA repair system contains the genes for the expression of the PprA and RecQ proteins. The genes are respectively coupled to the pTPI and pTDH3 promoters which are naturally activated in <I>S. cerevisiae</I>. </P><P> This construct is intended to be used in conjunction with the other constructs of the DNA repair system as all parts from these constructs are required for the DNA repair to work. </P><P> The construct is implemented in a plasmid p0391 (pXI-5- LoxP-SpHIS5) which has a gene coding for a histidine forming enzyme. </P><P class="figuretext">Figure 8. Construct 8. The figure depicts the third construct of the DNA repair system which contains the genes for the PprA and RecQ proteins coupled to receptors whose expression is induced naturally in <I>S. cerevisiae</I>. </P><H3>References</H3>
[1] Jensen, N. B., Strucko, T., Kildegaard, K. R., David, F., Maury, J., Mortensen, U. H., . . . Borodina, I. (2014). EasyClone: Method for iterative chromosomal integration of multiple genes in saccharomyces cerevisiae.
</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>