<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Chalmers-Gothenburg_ProjectResults skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Chalmers-Gothenburg/ProjectResults</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="main"><DIV id="my_content"><DIV id="section1"><DIV id="section1-menu"><NAV><UL><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg">Home</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Students">Students</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Supervisors">Supervisors</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Gallery">Gallery</A></LI></UL><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Team" class="arrow">Team</A><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Project">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Theory">Theory</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Constructs">Constructs</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/OtherApplications">Other Applications</A></LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Project" class="arrow">Project</A><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/ProjectResults">Project Results</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/BioBricks">Bio Bricks</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Medals">Medals</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Results">Results</A></LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Achievements" class="arrow">Achievements</A><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Detection">Detection</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/SafetySwitch">Safety Switch</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Simulation">Simulation</A></LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Modeling" class="arrow">Modeling</A><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/iGEMCommunity">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/NextGeneration">Next Generation</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Society">Society</A></LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/HumanPractices" class="arrow">Human Practices</A><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Schedule">Lab Journal</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Protocols">Protocols</A></LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Notebook" class="arrow">Notebook</A><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/LabSafety">Lab Safety</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Ethics">Ethics</A></LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Safety" class="arrow">Safety</A><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Sponsors">Sponsors</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Acknowledgements">Acknowledgements</A></LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Thanks" class="arrow">Thanks!</A></NAV></DIV><DIV id="section1-content"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-2"><A href="#Detection_system"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Detection system</SPAN></A></LI><LI class="toclevel-2"><A href="#Detection_test"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Detection test</SPAN></A></LI><LI class="toclevel-2"><A href="#DNA_Repair_System"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">DNA Repair System</SPAN></A></LI><LI class="toclevel-2"><A href="#Safety_switch"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Safety switch</SPAN></A></LI><LI class="toclevel-3"><A href="#SS_TPK2"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">SS TPK2</SPAN></A></LI><LI class="toclevel-3"><A href="#SS_mRFP"><SPAN class="tocnumber">1.4.2</SPAN><SPAN class="toctext">SS mRFP</SPAN></A></LI></UL></DIV><H2><SPAN class="mw-headline" id="Overview">Overview</SPAN></H2><P>•	Assembled and integrated two different safety switches, based on <I>TPK2</I> overexpression or <I>mRFP</I> expression induced at low ATP levels, into CEN.PK2.</P><P>•	SS TPK2 shows reduced growth rate after OD-measurement but no change in viability compared to wild type.</P><P>•	Fluorescent measurements show that connection of pTEF1 to pSUC2 maintains the ATP repression mechanism of pSUC2, while achieving higher expression at low ATP levels.</P><P>•	Successfully constructed and integrated the system to detect the P-factor from <I>S.pombe</I> into CEN.PK2.</P><P>•	Fluorescent cells in the Detection System</P><P>•	Amplified all parts of the DNA repair system, but obtained vector-only clones after Gibson and transformation into E.coli</P><H3><SPAN class="mw-headline" id="Detection_system">Detection system</SPAN></H3><P>The system without the amplification through dCAS9-VP64 (construct 4) was successfully assembled and integrated into <I>S.cerevisiae</I> CEN.PK2. The genomic integration was verified with colony PCR and sequencing. The detection system was initially intended to be integrated into IMFD-70, but as several transformations failed the strain was changed to CEN.PK2 instead.</P><P>The construction of the application systems (construct 2 and 3) was unsuccessful. The fragments of the constructs was individually amplified with PCR, see figure 1, but only clones with empty vectors were obtained after Gibson assembly and transformation into E.coli.</P><P>Construct 1 was successfully assembled and transformed into E.coli, see figure 2, but the construct is not functional without construct 2 and 3. Because of this limitation, no further steps were taken with construct 1.</P><P><B>Figure 1.Well 1: 1kb ladder. Well 2 and 3: unpurified dCas9-Vp64. Well 4: purified dCas9-Vp64. Expected length: 4335 bp.</B></P><P>Construct 1 was successfully assembled and transformed into E.coli, see figure 2, but the construct is not functional without construct 2 and 3. Because of this limitation, no further steps were taken with construct 1.</P><P><B>Figure 2. Construct 1 digested with MfeI. Expected fragments: 214, 2656 and 5926 bp.</B></P><H3><SPAN class="mw-headline" id="Detection_test">Detection test</SPAN></H3><P>The fluorescent microscopy pictures from the detection test with CEN.PK2 containing construct 4 (C4) are shown in figure 3-6. Samples were made with different amounts of concentrated P-factor and compared with wild type CEN.PK2 (WT).</P><P><B>Figure 3. Positive control. Left: overlay channels (bright field, RFP and GFP). Right: RFP channel.</B></P><P><B>Figure 4. C4 with 230 µl concentrated P-factor. Left: overlay channels (bright field, RFP and GFP). Right: RFP channel.</B></P><P><B>Figure 5. C4 without P-factor. Left: overlay channels (bright field, RFP and GFP). Right: RFP channel.</B></P><P><B>Figure 6. WT with 230 µl concentrated P-factor. Left: overlay channels (bright field, RFP and GFP). Right: RFP channel.</B></P><P><B>Figure 7. WT without P-factor. Left: overlay channels (bright field, RFP and GFP). Right: RFP channel.</B></P><P>Only the C4 samples containing the highest amount of P-factor showed a few fluorescent cells, which is promising for our concept. The weak signal can be explained by the fact that constructs containing the amplification system through dCas9-vp64 could not be completed. This forces the detection system to rely on the presumably weak promoter pFUS1 to express <I>mRFP</I>. This can result in a weak fluorescent signal when the P-factor is detected.</P><H3><SPAN class="mw-headline" id="DNA_Repair_System">DNA Repair System</SPAN></H3><P>The PUR genes were successfully amplified through PCR, see figure 8, but only clones with empty vectors were obtained after Gibson assembly and transformation into E.coli.  One of the problems could have been the size of the constructs, which was around 11-12 kbp, as only chemical transformation into E.coli was used.  Although, it is not certain that electroporation would have solved the problem, as the chemical transformation yielded only very few colonies, all of them containing empty vectors.</P><P><B>Figure 8. 1kb ladder, RecJ, RecA, RecQ, RecD2, SSB, PprA and 1kb ladder.</B></P><P>We cannot tell if this system will work in the same way in a yeast model as it does in <I>D. radiodurans</I>. We did not have enough time to evaluate optimizations, such as additional enzymes needed, as a lot of time was spend on trial and error in the construction of the current DNA repair system. We laid our focus on the other two systems
</P><P><B>Figure 8. 1kb ladder, RecJ, RecA, RecQ, RecD2, SSB, PprA and 1kb ladder.</B></P><H3><SPAN class="mw-headline" id="Safety_switch">Safety switch</SPAN></H3><P>All three variants of the safety switch (pTEF1-pSUC2-TPK2, pTEF1-pSUC2-<I>mRFP</I> and pSUC2-<I>mRFP</I>) was successfully integrated into the genome of <I>S.cerevisiae</I> CEN.PK2 and verified by colony PCR and sequencing.</P><H4><SPAN class="mw-headline" id="SS_TPK2">SS TPK2</SPAN></H4><P>The results for continuous OD-measurement of the wild type <I>S.cerevisiae</I> CEN.PK2 (WT) and CEN.PK2 with integrated safety switch (SS) in YPD are shown in figure 8.</P><P><B>Figure 8. Growth curve of Wild type CEN.PK2 (WT) and recombinant CEN.PK2 with the safety switch (SS).</B></P><P>The measurements indicates that both WT and SS grow initially at the same rate but after ~6 hours, as energy (glucose) levels starts to decline, SS continues to grow at a reduced rate. This is not ideally as the safety switch is supposed to kill the cells, instead of just reduce its growth rate.</P><P>To check if the reduced growth rate was caused by reduced viability of the cells, a drop test was performed after 48 h cultivation in YPD. 10 µL of serially diluted cell suspension was spotted on an YPD plate and incubated overnight. The OD-measurement after 48 hours cultivation is shown in figure 9 and the drop test in figure 10.</P><P><B>Figure 9. OD600 after 48 hours cultivation. Figure 10. Drop test of WT (above) and SS (below). Serially diluted to OD 0.1, 0.05, 0.01 and 0.001</B></P><P>There is no clear difference in viability between WT and SS which indicates that overexpression if TPK2 in this system causes reduced growth rate but no obvious change in viability. A solution for this insufficient safety switch could be to change the TPK2 to a more harmful gene.</P><H4><SPAN class="mw-headline" id="SS_mRFP">SS mRFP</SPAN></H4><P>Expression of <I>mRFP</I> (monomeric Red Fluorescent Protein) was used to analyze expression levels of the safety switch. To evaluate the effect of connecting pTEF1 to pSUC2, two different versions were made: pTEF1-pSUC2-<I>mRFP</I> and pSUC2-<I>mRFP</I>. Both constructs were integrated into the genome of <I>S.cerevisiae</I> CEN.PK2. The fluorescence of the clone with integrated pTEF1-pSUC2-<I>mRFP</I> (TEFSUC) and the clone with pSUC2-<I>mRFP</I> (SUC) was compared with wild type CEN.PK2 (WT).</P><P>The first sample of TEFSUC, SUC and WT was cultivated for 2 hours in YPD. The results from fluorescent microscopy are shown in figure 11-14.</P><P><B>Figure 11. Positive control. Left: overlay channels (bright field, RFP and GFP). Right: RFP channel.</B></P><P><B>Figure 12. TEFSUC sample 1 cultivated for 2 hours. Left: overlay channels (bright field, RFP and GFP). Right: RFP channel.</B></P><P><B>Figure 13. SUC sample 1 cultivated for 2 hours. Left: overlay channels (bright field, RFP and GFP). Right: RFP channel.</B></P><P><B>Figure 14. WT sample 1 cultivated for 2 hours. Left: overlay channels (bright field, RFP and GFP). Right: RFP channel.</B></P><P>3 hours of cultivation in YPD (after overnight preculture) gives no visible RFP fluorescence. A reason for this could be that 2 hours is not enough to give a significant drop in energy levels to relieve the repression of pSUC2.</P><P>A new sample of TEFSUC, SUC and WT was cultivated for 6 hours in YPD. The results from fluorescent microscopy are shown in figure 15-18.</P><P><B>Figure 15. Positive control. Left: overlay channels (bright field, RFP and GFP). Right: RFP channel.</B></P><P><B>Figure 16. TEFSUC sample 2 cultivated for 6 hours. Left: overlay channels (bright field, RFP and GFP). Right: RFP channel.</B></P><P><B>Figure 17. SUC sample 2 cultivated for 6 hours. Left: overlay channels (bright field, RFP and GFP). Right: RFP channel.</B></P><P><B>Figure 18. WT sample 2 cultivated for 6 hours. Left: overlay channels (bright field, RFP and GFP). Right: RFP channel.</B></P><P>Now there is a clear difference between TEFSUC and SUC. TEFSUC gives several highly fluorescent cells while SUC only shows slightly higher fluorescent compared to WT. This indicates that the repression of pSUC2 is reduced which allows expression of <I>mRFP</I> through the high expression promoter pTEF1. </P><P>Another fluorescence measurement was performed on the same sample after 23 hours of cultivation. 
The results from fluorescent microscopy are shown in figure 19-22.</P><P><B>Figure 19. Positive control. Left: overlay channels (bright field, RFP and GFP). Right: RFP channel.</B></P><P><B>Figure 20. TEFSUC sample 2 cultivated for 23 hours. Left: overlay channels (bright field, RFP and GFP). Right: RFP channel.</B></P><P><B>Figure 21. SUC sample 2 cultivated for 23 hours. Left: overlay channels (bright field, RFP and GFP). Right: RFP channel.</B></P><P><B>Figure 22. WT sample 2 cultivated for 23 hours. Left: overlay channels (bright field, RFP and GFP). Right: RFP channel.</B></P><P>This demonstrates that the concept of this safety switch could work, but the lethal gene has to be changed to a more potent one to be classified as a functional safety switch</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>