<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Chalmers-Gothenburg_Theory skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Chalmers-Gothenburg/Theory</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="main"><DIV id="my_content"><DIV id="section1"><DIV id="section1-menu"><NAV><UL><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg">Home</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Students">Students</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Supervisors">Supervisors</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Gallery">Gallery</A></LI></UL><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Team" class="arrow">Team</A><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Project">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Theory">Theory</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Constructs">Constructs</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/OtherApplications">Other Applications</A></LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Project" class="arrow">Project</A><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/ProjectResults">Project Results</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/BioBricks">Bio Bricks</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Medals">Medals</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Results">Results</A></LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Achievements" class="arrow">Achievements</A><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Detection">Detection</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/SafetySwitch">Safety Switch</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Simulation">Simulation</A></LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Modeling" class="arrow">Modeling</A><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/iGEMCommunity">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/NextGeneration">Next Generation</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Society">Society</A></LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/HumanPractices" class="arrow">Human Practices</A><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Schedule">Lab Journal</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Protocols">Protocols</A></LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Notebook" class="arrow">Notebook</A><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/LabSafety">Lab Safety</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Ethics">Ethics</A></LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Safety" class="arrow">Safety</A><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Sponsors">Sponsors</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Acknowledgements">Acknowledgements</A></LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Thanks" class="arrow">Thanks!</A></NAV></DIV><DIV id="section1-content"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Detection_and_amplification_of_signal"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Detection and amplification of signal</SPAN></A></LI><LI class="toclevel-2"><A href="#G_protein_coupled_receptors"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">G protein coupled receptors</SPAN></A></LI><LI class="toclevel-2"><A href="#Phereomone_response_pathway"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Phereomone response pathway</SPAN></A></LI><LI class="toclevel-2"><A href="#Detection_system"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Detection system</SPAN></A></LI><LI class="toclevel-3"><A href="#Fusion_receptor"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Fusion receptor</SPAN></A></LI><LI class="toclevel-3"><A href="#Amplification"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Amplification</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#DNA-extension_dependent_DNA-repair"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">DNA-extension dependent DNA-repair</SPAN></A></LI><LI class="toclevel-2"><A href="#How_can_D._radiodurans_be_so_resistant.3F"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">How can <I>D. radiodurans</I> be so resistant?</SPAN></A></LI><LI class="toclevel-2"><A href="#Repair_in_Our_project"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Repair in Our project</SPAN></A></LI><LI class="toclevel-2"><A href="#The_repair_mechanism"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">The repair mechanism</SPAN></A></LI><LI class="toclevel-1"><A href="#The_repair_proteins"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">The repair proteins</SPAN></A></LI><LI class="toclevel-2"><A href="#RecA"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">RecA</SPAN></A></LI><LI class="toclevel-3"><A href="#DNA_strand_exchange"><SPAN class="tocnumber">3.1.1</SPAN><SPAN class="toctext">DNA strand exchange</SPAN></A></LI><LI class="toclevel-2"><A href="#RecQ"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">RecQ</SPAN></A></LI><LI class="toclevel-3"><A href="#Domains"><SPAN class="tocnumber">3.2.1</SPAN><SPAN class="toctext">Domains</SPAN></A></LI><LI class="toclevel-3"><A href="#Specific_interactions"><SPAN class="tocnumber">3.2.2</SPAN><SPAN class="toctext">Specific interactions</SPAN></A></LI><LI class="toclevel-2"><A href="#SSB"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">SSB</SPAN></A></LI><LI class="toclevel-3"><A href="#Structure"><SPAN class="tocnumber">3.3.1</SPAN><SPAN class="toctext">Structure</SPAN></A></LI><LI class="toclevel-3"><A href="#Binding"><SPAN class="tocnumber">3.3.2</SPAN><SPAN class="toctext">Binding</SPAN></A></LI><LI class="toclevel-3"><A href="#Radiation_tolerance"><SPAN class="tocnumber">3.3.3</SPAN><SPAN class="toctext">Radiation tolerance</SPAN></A></LI><LI class="toclevel-2"><A href="#RecJ"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">RecJ</SPAN></A></LI><LI class="toclevel-3"><A href="#Mutations"><SPAN class="tocnumber">3.4.1</SPAN><SPAN class="toctext">Mutations</SPAN></A></LI><LI class="toclevel-2"><A href="#RecD2"><SPAN class="tocnumber">3.5</SPAN><SPAN class="toctext">RecD2</SPAN></A></LI><LI class="toclevel-3"><A href="#ATPase_activity_and_DNA_translocation"><SPAN class="tocnumber">3.5.1</SPAN><SPAN class="toctext">ATPase activity and DNA translocation</SPAN></A></LI><LI class="toclevel-3"><A href="#Helicase_activity"><SPAN class="tocnumber">3.5.2</SPAN><SPAN class="toctext">Helicase activity</SPAN></A></LI><LI class="toclevel-3"><A href="#DNA_binding"><SPAN class="tocnumber">3.5.3</SPAN><SPAN class="toctext">DNA binding</SPAN></A></LI><LI class="toclevel-3"><A href="#Effects_of_recD_mutations"><SPAN class="tocnumber">3.5.4</SPAN><SPAN class="toctext">Effects of <I>recD</I> mutations</SPAN></A></LI><LI class="toclevel-2"><A href="#PprA"><SPAN class="tocnumber">3.6</SPAN><SPAN class="toctext">PprA</SPAN></A></LI><LI class="toclevel-3"><A href="#DNA_binding_2"><SPAN class="tocnumber">3.6.1</SPAN><SPAN class="toctext">DNA binding</SPAN></A></LI><LI class="toclevel-3"><A href="#Stimulation_of_enzymes"><SPAN class="tocnumber">3.6.2</SPAN><SPAN class="toctext">Stimulation of enzymes</SPAN></A></LI><LI class="toclevel-1"><A href="#Safety_Switch"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Safety Switch</SPAN></A></LI><LI class="toclevel-2"><A href="#The_SUC2_regulation_in_Saccharomyces_cerevisiae"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">The SUC2 regulation in <I>Saccharomyces cerevisiae</I></SPAN></A></LI><LI class="toclevel-2"><A href="#The_Safety_Switch"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">The Safety Switch</SPAN></A></LI></DIV><H2><SPAN class="mw-headline" id="Detection_and_amplification_of_signal">Detection and amplification of signal</SPAN></H2><P>To detect contaminations in a bioreactor we use a system that utilizes extracellular receptors responding to ligands of the unwanted contaminant. The detection system goes by the name DAS, detection and amplification of signal, and it is based on the pheromone response pathway in <I>Saccharomyces cerevisiae</I>. The pheromone pathway uses a G-protein coupled receptor (GPC-receptor) to detect pheromones of another yeast mating type. When the receptor comes in contact with the correct agonist, which is pheromone α for a-type cells and pheromone a for α-type cells, the receptor then activates a pathway which leads to expression of genes that prepare the yeast for mating. In our system we plan to hijack this system and instead make the yeast signal that a contamination is present.</P><H3><SPAN class="mw-headline" id="G_protein_coupled_receptors">G protein coupled receptors</SPAN></H3><P> GPC receptors, figure 1, also known under the name seven-transmembrane receptors, are among the most common and versatile family of receptors found in eukaryotes. The receptors found in this kind of system are signal mediators through many subjects, such as light, olfactory stimulation, and many other signaling substances commonly found in mammals. They act by activating signal transduction pathways inside the cell, resulting in an intracellular response to an external signal.</P><P class="figuretext"> Figure 1. A general GPC receptor. The figure depicts the structure of such a receptor with emphasis on the characteristic transmembrane structure. </P><P> The receptors share a common structure among all eukaryotic cells, and a major characteristic is, as the name suggests, that the receptor has a transmembrane region which passes through the membrane seven times. The N-terminal of the receptor is located extracellularly and it is responsible for transporting and annealing the protein to the cell membrane. In some cases it also serves as a ligand binding site, although it is more common that the ligands associate with one of the three extracellular loops that reside between the transmembrane sites. The C-terminal of the GPC receptor is a cytosolic region and thus has no ligand associating function whatsoever, instead it has an important role as a mediator between the receptor and the cell or more specifically the G protein that associates with the receptor [1, 2, 3].</P><P> The GPC-receptors as the name indicates are bound to G proteins, as seen in figure 2, within the cell. G proteins are trimeric complexes consisting of three different proteins, called Gα, Gβ, and Gγ. G protein is an abbreviation for guanine nucleotide-binding protein and as the name implies is a protein which binds guanine in the form of GDP or GTP. In a natural state the protein prefers to be bound by GDP, while this is the state the trimeric complex is bound together and has no catalytic activity. During ligand association of the GPC receptor the GDP previously bound to the Gα is replaced with a GTP. While the Gα protein is bound by a GTP the Gβ and Gγ separates as a dimer. In this separate phase the Gβγ or the Gα subunit are free to interact with other molecules, this often leads to the activation of a pathway that ultimately causes a cellular response to an external signal [1, 2, 3].</P><P class="figuretext"> Figure 2. A general GPC receptor coupled with a G protein. The figure shows how the activation of the GPC receptor causes the GDP bound to a trimer G protein to be replaced with a GTP, resulting in the splitting of the trimer.</P><H3><SPAN class="mw-headline" id="Phereomone_response_pathway">Phereomone response pathway</SPAN></H3><P> The pheromone response pathway in yeast is one of the most studied pathways of eukarya. Due to yeasts important position as a role model organism for eukaryotic cells much attention has been given to the few GPCR pathways that exist in it, the pheromone pathway is one of them. The pathway induces changes such as a cell cycle arrest and conformational changes due to filamentous response genes when a pheromone of a mating partner is located. The following figure depicts a simplified version of the pheromone response pathway. </P><P class="figuretext">Figure 3. A simplified visualization of the pheromone response pathway in <I>S. cerevisiae</I>. The figure shows how the signaling cascade is activated by an agonist ligand, resulting in the induced expression of certain genes.</P><P> The figure depicts an <B>a</B>-type yeast cell containing the receptor Ste2, this receptor interacts with the pheromone from the α-type yeast cell called α-factor. The receptor Ste2 is in turn intracellularly bound to the three G-protein units α, γ, and β named Gpa1, Ste18, and Ste4 respectively. During ligand association the agonist pheromone produces a conformational change in the receptor structure, exposing parts of the receptor which catalyzes the exchange of the GDP bound to the intracellular α-subunit to a GTP. When the diphosphate is replaced with a triphosphate the conformation of the trimer changes, and the β- and γ-subunits separate as a dimer from the α-subunit.</P><P> This Gβγ dimer interacts with two effectors called Ste20 and Ste5. Ste20 is a PAK (p21-activated protein kinase) and its kinase activity is enabled when it is bound to the G protein Cdc42. Ste5 is a scaffold protein which attracts and binds the three proteins Ste11, Ste7, and Fus3 which are MEKK/MAP3K (Mitogen activated protein kinase kinase kinase), MEK/MAP2K (Mitogen activated protein kinase kinase), and MAPK (Mitogen activated protein kinase) respectively. When both Ste5 and Ste20 bind to the Gβγ dimer it allows for the activated Ste20 kinase to phosphorylate the Ste11 MEKK thus triggering a MAPK cascade. This phosphorylation cascade ends in the release of activated Fus3, which in turn is transported to the nucleus where it activates the transcription factor Ste12 [3].</P><H3><SPAN class="mw-headline" id="Detection_system">Detection system</SPAN></H3><P> In our system, the natural receptor (Ste2 from a-cells) has been replaced with a fusion receptor consisting partially of the natural Ste2 receptor but mostly a pheromone receptor called Mam2 originating from the yeast <I>Schizosaccharomyces pombe</I>. Due to both of those receptors being GPC-receptors with a similar Gα subunit the fusion receptor is capable of interacting with the G-protein of the pheromone response pathway in <I>Saccharomyces cerevisiae</I>. Depicted below is the modified system we use for detection: </P><P class="figuretext">Figure 4. The DAS system as a whole. The figure depicts how the natural GPC receptor of the pheromone pathway has been replaced with a fusion receptor that is activated by an alternative agonist ligand, the resulting signaling cascade results in a signal amplification system that gives an output signal in the form of fluorescent proteins.</P><H4><SPAN class="mw-headline" id="Fusion_receptor">Fusion receptor</SPAN></H4><P> The fusion receptor contains the N-terminus signal peptide of the Ste2 receptor so that the receptor is correctly transported and integrated into the cell membrane of the <I>S. cerevisiae</I> cell wall. The intermediate region and the C-terminus of the receptor is replaced with the equivalent parts of the Mam2 receptor, which makes the receptor capable of interacting with the correct ligand. Since the GPC-receptor is mostly dependent on the extracellular loops in regard of what ligand it associates with it is possible to create a wide array of receptors simply by replacing this region in the receptor [4]. There are many receptors to be found in the eukaryotes, each with its own specificity [5].</P><H4><SPAN class="mw-headline" id="Amplification">Amplification</SPAN></H4><P> In order to produce a strong output signal when a contaminant is detected we have decided to use intracellular amplification to strengthen the signal. To achieve this in an efficient manner, while holding on to the work previously done in the iGEM competition we decided to use the dCas9-Vp64 complex that was used the previous year by Team Gothenburg. By linking an activation domain to a deactivated version of the CRISPR associated protein, dCas9, one obtains a transcription factor guided by the same means as the popular CRISPR system. By creating a promoter with a binding site for a specific RNA sequence and expressing this RNA to act as a guide RNA (gRNA for short) for the dCas9 protein it is possible to make an incredibly precise tool for gene expression.</P><P> Vp64 is a viral protein complex consisting of four Vp16 activation domains, originating from the herpes virus, which when combined form the strong activation domain called Vp64. 
By coupling this activation domain to a DNA binding protein such as dCas9 a strong, activating transcription factor is created [6].</P><P> However, the expression of a new transcription factor in itself, even though it induces a strong expression, is not guaranteed to produce a strong final signal, therefore we decided to amplify the amount of transcription factor through a feed-back activation loop, where we coupled the expression of the transcription factor (dCas9-Vp64) to the promoter that it specifically binds to, pCYC1m. The results of this coupling should be a heightened amount of transcription factor in the cell, allowing for close to the maximum expression of the signal substance, in our case RFP. The dCas9-Vp64 gene is transcribed together with the gRNA as a single mRNA molecule (see figure 5), but thanks to the incorporation of ribozymes these two RNA parts are separated [7].</P><P class="figuretext">Figure 5. A close up of the <I>CGC1</I> gene, the figure depicts the transcription of the <I>CGC1</I> gene and the following ribozyme splicing leading up to the creation of a gRNA and the transcription of the dCas9-Vp64 complex. When these two components are combined they create a transcription factor.</P><P> References: 
[1] Trzaskowski, B., Latek, D., Yuan, S., Ghoshdastider, U., Debinski, A., &amp; Filipek, S. (2012). Action of molecular switches in GPCRs - theoretical and experimental studies. <I>Current Medicinal Chemistry</I>, 19(8), 1090-1109. doi:10.2174/092986712799320556 
[2] Bockaert, J., &amp; Philippe Pin, J. (1999). Molecular tinkering of G protein-coupled receptors: An evolutionary success. <I>The EMBO Journal</I>, 18(7), 1723-1729. doi:10.1093/emboj/18.7.1723  
[3] Bardwell, L. (2004). A walk-through of the yeast mating pheromone response pathway. <I>Peptides</I>, 25(9), 1465-1476. doi:10.1016/j.peptides.2003.10.022 
[4] Fukutani, Y., Nakamura, T., Yorozu, M., Ishii, J., Kondo, A., &amp; Yohda, M. (2012). The N‐terminal replacement of an olfactory receptor for the development of a Yeast‐based biomimetic odor sensor. Biotechnology and Bioengineering, 109(1), 205-212. doi:10.1002/bit.23327 
[5] Hopkins, A. L., Overington, J. P., &amp; Al-Lazikani, B. (2006). How many drug targets are there? <I>Nature Reviews Drug Discovery</I>, 5(12), 993-996. doi:10.1038/nrd2199 
[6] Maeder, M. L., Linder, S. J., Cascio, V. M., Fu, Y., Ho, Q. H., &amp; Joung, J. K. (2013). CRISPR RNA-guided activation of endogenous human genes. <I>Nature Methods</I>, 10(10), 977-979. doi:10.1038/nmeth.2598 
[7] Dower, K., Kuperwasser, N., Merrikh, H., &amp; Rosbash, M. (2004). A synthetic A tail rescues yeast nuclear accumulation of a ribozyme-terminated transcript. <I>RNA</I>, 10(12), 1888-1899. doi:10.1261/rna.7166704 
</P><H2><SPAN class="mw-headline" id="DNA-extension_dependent_DNA-repair">DNA-extension dependent DNA-repair</SPAN></H2><P><I>Deinococcus radiodurans</I> is an extremophilic bacterium with an extraordinary ability to survive vast amounts of DNA-damaging radiation [1, 3]. It is one of the most radiation-resistant organisms known and has even been listed in the Guinness Book of World Records as “the world’s toughest bacterium” [6]. Many DNA damaging agents such as γ-radiation, which is a particularly lethal form of ionizing radiation, induce double-stranded breaks (DSBs) in the genome during exposure. These DSBs are very difficult to repair since there is no template to direct DNA repair as with single-stranded breaks. The amount of ionizing radiation needed to kill 63% of the cells in a population is called the D37 dose. For <I>D. radiodurans</I>, the D37 dose is 6000 Gy. This amount of radiation is estimated to induce 275 DSBs and 3000 single-stranded breaks per genome. For comparison, the introduction of less than 10 DSBs per genome is lethal to <I>Escherichia coli</I> as well as to most other species. [2, 5]</P><H3><SPAN class="mw-headline" id="How_can_D._radiodurans_be_so_resistant.3F">How can <I>D. radiodurans</I> be so resistant?</SPAN></H3><P>This remarkable resistance is due to the extremely proficient DNA repair processes of <I>D. radiodurans</I>. Yet little is known about the precise mechanisms behind this efficient repair. Most, if not all, of the typical bacterial DNA-repair proteins are found in <I>D. radiodurans</I>. So how come that <I>Deinococcus radiodurans</I> is so resistant to ionizing radiation? [4] One possibility is that it utilizes repair mechanisms that are fundamentally different from all other bacteria. Another possibility is that it uses the same repair systems as all the others, but in a more effective way. [3, 4] It appears as if <I>D. radiodurans</I> is able to limit DNA degradation and restrict diffusion of DNA fragments that are produced when irradiated. But what mechanisms underlie these features is still being debated. Both non-homologous end-joining (NHEJ) and extended synthesis-dependent strand annealing (ESDSA) have been suggested. [5] </P><P> NHEJ is claimed to be a useful process for repairing double-stranded breaks in situations where strand ends might not be able to freely diffuse away from each other, such as in condensed chromosomes. NHEJ is present in other bacteria and certain enzymes known to be involved in NHEJ are present in <I>D. radiodurans</I>. [5, 9] However, this mechanism is error-prone which doesn’t consort with the accurate genome repair seen in <I>D. radiodurans</I>. [5, 10]. In ESDSA, single-stranded overhangs with overlapping homologies on shattered chromosomes are used both as primers and templates for synthesis of complementary strands. The elongated complementary strands can then reestablish chromosomal integrity with high precision. This rather extensive DNA repair synthesis is dependent on the activity of the RecA protein. However, it doesn’t seem to be able to explain all the means behind <I>D. radiodurans’s</I> incredible ability to withstand vast amounts of radiation and oxidative stress. [7, 8] The most likely scenario is that several different pathways act alongside each other in <I>D. radiodurans</I> where some are more similar to the pathways used in other organisms, and some are more specific for <I>D. radiodurans</I>.</P><H3><SPAN class="mw-headline" id="Repair_in_Our_project">Repair in Our project</SPAN></H3><P> Our plan is therefore to identify the pathway that is the most specific for <I>D. radiodurans</I> and the specific proteins participating in that pathway. We will then insert these into <I>S. cerevisiae</I> to implement a unique system for post-UV radiation resistance (PUR) in the yeast to increase its resistance to UV irradiation, which will be used in the bioreactor to eliminate contaminants.  The RecA system found in <I>D. radiodurans</I> is unique when compared to other organisms, since it can repair DNA breaks with both sticky and blunt ends. [11] This process involves the proteins RecA, RecQ, RecD2, PprA and DrSSB, and also RecJ, which is often found in the RecF pathway in other organisms. This exonuclease is used both in the RecFOR pathway in <I>D. radiodurans</I> as well as in the RecA pathway to create single-stranded DNA ends to which the other enzymes can associate. Since <I>D. radiodurans</I> is polyploid, the repair system is based on an organism with multiple genome copies, which is why a diploid yeast is going to be used. Also, since the proteins need to be transported into the nucleus to be able to associate with the DNA, an NLS sequence will be added to each protein. </P><H3><SPAN class="mw-headline" id="The_repair_mechanism">The repair mechanism</SPAN></H3><P> At the presence of DNA breaks, the exonuclease RecJ in our system will start the repair process by hydrolyzing the 5’-terminal strand of both fragments, generating two single-stranded fragments. To avoid degradation and formation of hairpin structures in the exposed single DNA strands, DrSSB will attach to the DNA strands. Apart from stabilizing the DNA, SSB can then interact with the RecA enzyme to initiate DNA strand exchange. The RecA enzyme will introduce complementary sequences to the ssDNA, which can be found among the additional genomes in the polyploid organism. RecQ also accompanies RecA, this 3’-5’ helicase will unwind the two strands in the template brought by RecA to allow for a DNA polymerase to associate with the DNA strands. DrSSB will bind to these single DNA strands as well, to avoid self-annealing and protect the strands from degradation. The DNA polymerase already present in yeast will attach to the template and start elongating the sequence of the damaged gene. Following the elongation, the 5’-3’ helicase RecD2 will unwind and release the template from the gene fragment, which hybridizes to the complementary sequence created by the DNA polymerase. After the helicase has released the template, the DNA polymerase can associate once more to the gene fragment, to fully elongate the remaining sequence. The polymerase is accompanied by PprA, a ligase-inducing enzyme, to fully repair the gaps in the sequence.</P><P class="figuretext"> Figure 6. The PUR system as a whole. The figure depicts how the incorporated enzymes from the RecA system in <I>D. radiodurans</I> collaborate with DNA repair enzymes naturally present in yeast, such as the DNA polymerase, to repair a double-stranded break.</P><P>References:
[1] Minton, K. W. (1996). Repair of ionizing-radiation damage in the radiation resistant bacterium <I>Deinococcus radiodurans</I>. <I>Mutation Research/DNA Repair</I>, 363(1), 1-7.
[2] Killoran, M. P., &amp; Keck, J. L. (2006). Three HRDC domains differentially modulate <I>Deinococcus radiodurans</I> RecQ DNA helicase biochemical activity. <I>Journal of Biological Chemistry</I>, 281(18), 12849-12857.
[3] Battista, J. R. (1997). Against all odds: the survival strategies of <I>Deinococcus radiodurans</I>. <I>Annual Reviews in Microbiology</I>, 51(1), 203-224.
[4] Battista, J. R., Earl, A. M., &amp; Park, M. J. (1999). Why is <I>Deinococcus radiodurans</I> so resistant to ionizing radiation?. <I>Trends in microbiology</I>, 7(9), 362-365.
[5] Cox, M. M., &amp; Battista, J. R. (2005). <I>Deinococcus radiodurans</I>—the consummate survivor. <I>Nature Reviews Microbiology</I>, 3(11), 882-892.
[6] Sarah E. DeWeerdt, SDW. (2002) The World’s Toughest Bacterium – <I>Deinococcus radiodurans</I> may be a tool for cleaning up toxic waste and more. <I>Genome News Network</I>. July 5th 2002. <A rel="nofollow" class="external free" href="http://www.genomenewsnetwork.org/articles/07_02/deinococcus.shtml">http://www.genomenewsnetwork.org/articles/07_02/deinococcus.shtml</A> (September 12th 2015).
[7] Zahradka, K., Slade, D., Bailone, A., Sommer, S., Averbeck, D., Petranovic, M., ... &amp; Radman, M. (2006). Reassembly of shattered chromosomes in <I>Deinococcus radiodurans</I>. <I>Nature</I>, 443(7111), 569-573.
[8] Slade, D., Lindner, A. B., Paul, G., &amp; Radman, M. (2009). Recombination and replication in DNA repair of heavily irradiated <I>Deinococcus radiodurans</I>. <I>Cell</I>, 136(6), 1044-1055.
[9] Englander, J., Klein, E., Brumfeld, V., Sharma, A. K., Doherty, A. J., &amp; Minsky, A. (2004). DNA toroids: framework for DNA repair in <I>Deinococcus radiodurans</I> and in germinating bacterial spores. <I>Journal of bacteriology</I>, 186(18), 5973-5977.
[10] Daly, M. J., &amp; Minton, K. W. (1996). An alternative pathway of recombination of chromosomal fragments precedes recA-dependent recombination in the radioresistant bacterium <I>Deinococcus radiodurans</I>. <I>Journal of bacteriology</I>, 178(15), 4461-4471.
[11] Cox, M. M., Keck, J. L., &amp; Battista, J. R. (2010). Rising from the Ashes: DNA Repair in <I>Deinococcus radiodurans</I>. <I>PLoS Genet</I>, 6(1), e1000815.</P><H2><SPAN class="mw-headline" id="The_repair_proteins">The repair proteins</SPAN></H2><H3><SPAN class="mw-headline" id="RecA">RecA</SPAN></H3><P>RecA and RecA-like proteins have been found throughout the phylogenetic tree and their structures are often conserved between species. They play a critical role in the repair processes that require homologous DNA pairing and recombination. The RecA protein from <I>D. radiodurans</I> is essential for the remarkable radiation resistance of the organism and is in many aspects a functional homolog to the protein found in <I>E. coli.</I> [6] The protein from <I>D. radiodurans</I> has been shown to be able to, at least partially, complement the mutated or deleted homolog in <I>E. coli</I> and vice versa [4, 7]. RecA is the protein responsible for promoting DNA strand exchange upon DNA damage in <I>D. radiodurans</I>.</P><P>RecA was first discovered in a DNA damage-sensitive strain of <I>D. radiodurans</I> which was found to be defective in a gene homologous to the RecA gene in <I>E. coli</I>. When transformed with a chromosomal DNA fragment that contained the deinococcal RecA gene from wild-type, the DNA damage resistance in the mutated strain was fully restored. [1] Even though the complete DSB rejoining in <I>D. radiodurans</I> is RecA-dependent, the initial repair is not. <I>D. radiodurans</I> is able to rejoin many DSBs in a sequence-specific manner for the first 1.5 hours, as seen in RecA mutants, before the RecA-dependent homologous recombination is initiated. [3] RecA is thus not detectable in the organism until DNA damage occurs and DNA damage repair has begun [2].</P><H4><SPAN class="mw-headline" id="DNA_strand_exchange">DNA strand exchange</SPAN></H4><P>The common pathway for DNA strand exchange, which is promoted by EcRecA and many of its homologs, is initiated by the formation of a filament on ssDNA, followed by uptake of the duplex substrate. These proteins thus target single-stranded gaps and tails. The RecA protein from <I>D. radiodurans</I> performs the exact inverse of this pathway. [5] There are three reasons for this. (i) DrRecA binds preferentially to dsDNA rather than ssDNA and rather initiates DNA strand exchange with a filament bound to dsDNA. [5, 6] (ii) Heterologous DNA strongly inhibits the DNA strand exchange by DrRecA, unlike EcRecA. (iii) In a DNA strand exchange reaction between a short linear dsDNA and a long circular ssDNA, the amount of DrRecA protein required for optimal reaction is stoichiometric with the duplex DNA and is unaffected by changes in ssDNA concentration, where the opposite is true for EcRecA. [5] One common aspect though is that DNA strand exchange is greatly facilitated by SSB proteins for both DrRecA and EcRecA [6]. </P><P> The significance of using the inverse pathway may be related to the absence of the <I>recB</I> and <I>recC</I> genes in the deinococcal genome. By operating the inverse pathway, RecBC is not needed to prepare single-stranded tails since DrRecA binds directly to dsDNA to promote repair of DSBs. [5]</P><P>References:
[1] Gutman, P. D., Carroll, J. D., lan Masters, C., &amp; Minton, K. W. (1994). Sequencing, targeted mutagenesis and expression of a recA gene required for the extreme radioresistance of <I>Deinococcus radiodurans</I>. <I>Gene</I>, 141(1), 31-37.
[2] Carroll, J. D., Daly, M. J., &amp; Minton, K. W. (1996). Expression of recA in <I>Deinococcus radiodurans</I>. <I>Journal of bacteriology</I>, 178(1), 130-135.
[3] Daly, M. J., &amp; Minton, K. W. (1996). An alternative pathway of recombination of chromosomal fragments precedes recA-dependent recombination in the radioresistant bacterium <I>Deinococcus radiodurans</I>. <I>Journal of bacteriology</I>, 178(15), 4461-4471.
[4] Narumi, I., Satoh, K., Kikuchi, M., Funayama, T., Kitayama, S., Yanagisawa, T., ... &amp; Yamamoto, K. (1999). Molecular analysis of the <I>Deinococcus radiodurans</I> recA locus and identification of a mutation site in a DNA repair-deficient mutant, rec30. <I>Mutation Research/DNA Repair</I>, 435(3), 233-243.
[5] Kim, J. I., &amp; Cox, M. M. (2002). The RecA proteins of <I>Deinococcus radiodurans</I> and Escherichia coli promote DNA strand exchange via inverse pathways. <I>Proceedings of the National Academy of Sciences</I>, 99(12), 7917-7921.
[6] Kim, J. I., Sharma, A. K., Abbott, S. N., Wood, E. A., Dwyer, D. W., Jambura, A., ... &amp; Cox, M. M. (2002). RecA Protein from the extremely radioresistant bacterium <I>Deinococcus radiodurans</I>: expression, purification, and characterization. <I>Journal of bacteriology</I>, 184(6), 1649-1660.
[7] Schlesinger, D. J. (2007). Role of RecA in DNA damage repair in <I>Deinococcus radiodurans</I>. <I>FEMS microbiology letters</I>, 274(2), 342-347.</P><H3><SPAN class="mw-headline" id="RecQ">RecQ</SPAN></H3><P>RecQ helicases are key enzymes in genome maintenance that participate in DNA replication, recombination, and repair. This multifunctional enzyme initiates homologous recombination and suppresses illegitimate recombination. RecQ is a member of the SF1 family of DNA helicases, which are highly conserved from bacteria to eukaryotes.[1,2] In vitro, RecQ helicases are able to exchange and unwind a wide variety of DNA substrates, including G4 DNA and three- or four-stranded DNA structures such as D-loops and Holliday junctions. It has also been suggested that RecQ generates an initiating signal that can recruit RecA for SOS induction and recombination at stalled replication forks.[3]</P><P>In <I>E.coli</I>, null mutations at the <I>recQ</I> locus result in a significant reduction of the efficiency of homologous recombination as well as an increase in sensitivity to UV radiation [6]. Similar deletions of RecQ in <I>D. radiodurans</I> resulted in a low doubling rate and sensitivity to hydrogen peroxide, γ-radiation, UV-light and mitomycin [1,5]. In addition to this, the expression of certain genes involved in iron homeostasis, antioxidant system, electron transport, and energy metabolism were significantly altered, indicating that RecQ has pleiotropic functions in the organism and thereby contributes to <I>D. radiodurans’s</I> astonishing ability to withstand stresses. A study conducted by Chen et al. discovered that upon deletion of DrRecQ, ROS accumulation increased and Fe and Mn metabolisms were altered.[5] Since RecQ helicases have such diverse cellular functions, they require specialization and coordination of multiple protein domains that integrate catalytic functions between both DNA-protein and protein-protein interactions [4].</P><H4><SPAN class="mw-headline" id="Domains">Domains</SPAN></H4><P>Most RecQ helicases have three conserved domains. These are designated as the helicase, RecQ-conserved (RQC), and Helicase and RNase D C-terminal (HRDC) domains in order from N- to C-terminus. The helicase region in the RecQ enzyme unwinds DNA with 3’→5’ polarity with respect to the DNA strand to which the enzyme is bound by using the energy from ATP hydrolysis. The RQC domain, which is unique to RecQ helicases, might have a role in regulating specific protein–protein interactions and adjusting the enzyme’s binding affinity to DNA. [1] HRDC domains are involved in structure-specific nucleic acid binding and assist in targeting RecQ proteins to particular DNA structures.[1,2] In <I>Deinococcus radiodurans</I>, DR1289 is a special member of RecQ family [1]. The structure of the domains in this enzyme is similar to RecQ helicases from other organisms, except that it encodes three HRDC domains whereas most only encode one.[1,2,5]</P><P>If the HRDC domains have been inactivated, the RecQ enzyme can usually function as a DNA-dependent ATPase and helicase, but it often has altered DNA binding activities. Neither the helicase domain nor the three HRDC domains alone are capable of fully restoring the DNA damage resistance compared to a wild type strain. Nevertheless, the helicase and all three HRDC domains are crucial for complete DNA damage resistance in <I>D. radiodurans</I>. As each HRDC domain is sequentially removed from the C-terminus, <I>D. radiodurans</I> displays a gradual decrease in DNA damage resistance. The unique tandem C-terminal arrangement of the HRDC domains is important for DNA damage resistance.[1,2] </P><P>The three HRDC domains in <I>D. radiodurans</I> have evolved to function in DNA binding and regulation. Together they increase the efficiency of the unwinding and ATPase activities of the helicase domain and are important for optimal helicase activity in vitro – removal of all three domains produces and enzyme with a considerably weakened ability to bind DNA.[1,2] However, each domain has a distinct effect on these activities as well as the structure-specific DNA binding of the enzyme. The N-terminal-most HRDC domain is critical for high affinity DNA binding and for efficient unwinding of DNA, whereas the more C-terminal HRDC domains lessen the DNA binding affinity and the ATPase rate. These two domains also possess more complex roles in structure-specific DNA unwinding. [2] The C-terminal-most HRDC domain has an unusual electrostatic surface that regulate structure-specific DNA binding and help direct DrRecQ to specific recombination/repair sites. This domain features a concentrated patch of acidic residues that provides a surface for inter-domain interactions which direct the enzyme to bind to certain DNA structures.[4]</P><P>Although the HRDC domains do not possess any helicase activity, expression of only the C-terminal portion of Dr1289 in a deletion mutant has been shown to be able to partially restore resistance to DNA damaging agents [1]. Another study showed that, unlike single HRDC RecQ helicases, DrRecQ does not simply use its HRDC domains as supplementary DNA binding elements [2]. These activities suggest that the three tandem HRDC domains have evolved to perform an essential function in DNA repair with more complex regulatory roles. The HRDC domains may increase the number of substrates to the enzyme or aid in substrate selection, aside from regulating the level of activity of the helicase.[1,2] Furthermore, because the C-terminal-most HRDC domain alone can partially restore resistance, the recognition of substrate by the tandem HRDCs may recruit other repair factors. For instance, RecQ and RecA interact in <I>E. coli</I> recombination, and DrRecQ may function analogously in DNA repair.[1] Additional potential roles of DrRecQ HRDC domains include acting as sites for protein-protein interactions that could target the enzyme to damaged DNA structures and/or provide a cellular mechanism for regulating the actions of the HRDC domains on DrRecQ in different contexts.[2]</P><H4><SPAN class="mw-headline" id="Specific_interactions">Specific interactions</SPAN></H4><P>SSB proteins prevent the formation of hairpins in single-stranded DNA and thereby stimulate RecQ helicase-promoted unwinding of single-stranded DNA substrates. The unwinding rate of Dr1289 has been shown to increase in the presence of DrSSB, which has a species-specific effect on the enzyme.[1] </P><P>References: 
[1] Huang, L., Hua, X., Lu, H., Gao, G., Tian, B., Shen, B., &amp; Hua, Y. (2007). Three tandem HRDC domains have synergistic effect on the RecQ functions in <I>Deinococcus radiodurans</I>. <I>DNA repair</I>, 6(2), 167-176.
[2] Killoran, M. P., &amp; Keck, J. L. (2006). Three HRDC domains differentially modulate <I>Deinococcus radiodurans</I> RecQ DNA helicase biochemical activity. <I>Journal of Biological Chemistry</I>, 281(18), 12849-12857.
[3] Huang, L. F., Hua, X. T., Lu, H. M., Gao, G. J., Tian, B., Shen, B. H., &amp; Hua, Y. J. (2006). Functional analysis of helicase and three tandem HRDC domains of RecQ in <I>Deinococcus radiodurans</I>. <I>Journal of Zhejiang University Science B</I>, 7(5), 373-376.
[4] Killoran, M. P., &amp; Keck, J. L. (2008). Structure and function of the regulatory C-terminal HRDC domain from <I>Deinococcus radiodurans</I> RecQ. <I>Nucleic acids research</I>, 36(9), 3139-3149.
[5] Chen, H., Huang, L., Hua, X., Yin, L., Hu, Y., Wang, C., ... &amp; Hua, Y. (2009). Pleiotropic effects of RecQ in <I>Deinococcus radiodurans</I>. <I>Genomics</I>, 94(5), 333-340.
[6] Nakayama, K., Irino, N., &amp; Nakayama, H. (1985). The recQ gene of Escherichia coli K12: molecular cloning and isolation of insertion mutants. <I>Molecular and General Genetics MGG</I>, 200(2), 266-271.
</P><H3><SPAN class="mw-headline" id="SSB">SSB</SPAN></H3><P>For DNA replication, recombination and repair to function, e.g. when DNA damage occurs, the dsDNA must be unwound for the necessary enzymes to be able to access the DNA strands. This process is not without risk, ssDNA is prone to chemical and nucelolytic attacks that can produce breaks that are difficult to repair. ssDNA can also self-associate, which inhibits many genome maintenance enzymes. Cells have therefore evolved a specialized class of proteins (ssDNA-binding proteins, or SSBs) that associate with ssDNA, independently of their sequences, to help preserve the ssDNA intermediates and protect them from degradation. These proteins are also known for associating with many different types of genome maintenance enzymes. [1, 4, 6]</P><H4><SPAN class="mw-headline" id="Structure">Structure</SPAN></H4><P>One thing that all SSBs have in common is that they use a conserved domain called an oligonucleotide/oligosaccharide-binding (OB) fold to bind ssDNA. The majority of all bacterial SSBs function as homotetramers, where each monomer contributes a single OB fold. [3] In contrast, SSB from <I>D. radiodurans</I> contains two such folds per monomer. It also functions as a homodimer rather than as a tetramer. [2, 4, 5] The major advantage of linking two OB folds together is that the two OB domains can evolve separately. The asymmetry seen in DrSSB indicates that each domain has developed into possessing specialized roles. [3] </P><P>Another common feature is that the last 10 amino acids at the C-terminus are all highly acidic and very well conserved. These C-terminal tails are suspected to act as binding sites for other proteins involved in genome maintenance. An acidic C-terminus is found in the C-terminal tails in DrSSB as well, but unlike the tetrameric SSBs DrSSB only contains two such tails. [2, 3]</P><H4><SPAN class="mw-headline" id="Binding">Binding</SPAN></H4><P>The OB domains of DrSSB bind ssDNA by using a combination of aromatic residues that stack against nucleotide bases, electrostatic, and polar interactions, such as positively charged residues that form ionic interactions with the DNA backbone [3, 5]. The extended surface to which ssDNA binds on the protein is similar to that used by tetrameric enzymes. Although the general binding path is conserved, there are distinct interaction surfaces present in DrSSB that indicate that species-specific binding to ssDNA occurs. [5]</P><H4><SPAN class="mw-headline" id="Radiation_tolerance">Radiation tolerance</SPAN></H4><P>In a study performed by George et al., upon treatment with heavy ionizing radiation, DrSSB protein levels rose significantly in vivo as early as an hour after the treatment. The accumulation of the protein had increased to maximal levels after 3 hours, and the elevated levels persisted for several hours afterwards. The localization of SSB also changed dramatically during the experiment. Immediately after radiation, most SSBs were found in the nucleoid, but as time passed the proteins started to dissipate through the cytosol. This radiation-dependent accumulation suggests that SSB might be important for the robust repair mechanisms in <I>D. radiodurans</I>. [5]</P><P>Lockheart et al. recently reported that SSB is essential for cell survival. A deletion of the drssb gene is lethal, and a lowered expression of DrSSB affects ionizing radiation tolerance, ultraviolet radiation tolerance, and general growth capabilities. The fact that ultraviolet and ionizing radiation produce different types of DNA damage highlights that DrSSB appears to be central to all DNA repair pathways, regardless of the type of damage involved. [6]</P><P>References:
[1] Shereda, R. D., Kozlov, A. G., Lohman, T. M., Cox, M. M., &amp; Keck, J. L. (2008). SSB as an organizer/mobilizer of genome maintenance complexes. <I>Critical Reviews in Biochemistry and Molecular Biology</I>, 43(5), 289-318.
[2] Eggington, J. M., Haruta, N., Wood, E. A., &amp; Cox, M. M. (2004). The single-stranded DNA-binding protein of <I>Deinococcus radiodurans</I>. <I>BMC microbiology</I>, 4(1), 2.
[3] Bernstein, D. A., Eggington, J. M., Killoran, M. P., Misic, A. M., Cox, M. M., &amp; Keck, J. L. (2004). Crystal structure of the <I>Deinococcus radiodurans</I> single-stranded DNA-binding protein suggests a mechanism for coping with DNA damage. <I>Proceedings of the National Academy of Sciences of the United States of America</I>, 101(23), 8575-8580.
[4] Witte, G., Urbanke, C., &amp; Curth, U. (2005). Single-stranded DNA-binding protein of <I>Deinococcus radiodurans</I>: a biophysical characterization. <I>Nucleic acids research</I>, 33(5), 1662-1670.
[5] George, N. P., Ngo, K. V., Chitteni-Pattu, S., Norais, C. A., Battista, J. R., Cox, M. M., &amp; Keck, J. L. (2012). Structure and cellular dynamics of <I>Deinococcus radiodurans</I> single-stranded DNA (ssDNA)-binding protein (SSB)-DNA complexes. <I>Journal of Biological Chemistry</I>, 287(26), 22123-22132.
[6] Lockhart, J. S., &amp; DeVeaux, L. C. (2013). The essential role of the <I>Deinococcus radiodurans</I> ssb gene in cell survival and radiation tolerance. 
</P><H3><SPAN class="mw-headline" id="RecJ">RecJ</SPAN></H3><P>DSB repair is initiated by degradation of the 5’-terminal strand by an exonuclease in coordination with a helicase. The recombinase enzyme RecA is then able to carry out a strand exchange with a homologous DNA duplex. This process is called extended synthesis dependent strand annealing (ESDSA) in <I>Deinococcus radiodurans</I>. The degradation of the 5’-terminal strand is most often dependent on the RecBCD helicase or on a related enzyme with similar activity. The <I>D. radiodurans</I> genome lacks a homologue for this protein, so an alternative pathway has been suggested. When RecBCD is absent in <I>E. coli</I>, the RecF pathway is an alternative pathway for recombination and DNA repair. <I>D. radiodurans</I> may also rely on enzymes of this pathway for DSB repair. [1, 2]  </P><P>The ssDNA-specific nuclease RecJ is found in most bacteria and is thought to process damaged DNA ends in the RecF (or RecFOR) pathway. It is a 5’→ 3’ ssDNA exonuclease which is unable to process dsDNA. [2] The protein requires a metal ion for activity, the most common regulator of its activity is Mn2+ followed by Mg2+, both of which are present in D. radiodurans. Furthermore, DrSSB has been shown to stimulate DrRecJ exonuclease activity as well. [1, 2]</P><H4><SPAN class="mw-headline" id="Mutations">Mutations</SPAN></H4><P>Mutants with reduced or deleted RecJ gene copy grow more slowly and are more sensitive to UV and γ-radiation and hydrogen peroxide, suggesting an important role for RecJ in DNA repair in <I>D. radiodurans</I>. [1, 2] One possibility as to why these mutations affect <I>D. radiodurans</I> in such a negative way could be because of the lack of the RecBC genes in the deinococcal genome. Only when the RecBCD gene has been inactivated in <I>E. coli</I> is the RecF pathway active, and studies have shown that RecBC-RecJ double mutants are lethal in certain organisms. [1]</P><P>References:
[1] Cao, Z., Mueller, C. W., &amp; Julin, D. A. (2010). Analysis of the recJ gene and protein from <I>Deinococcus radiodurans</I>. <I>DNA repair</I>, 9(1), 66-75.
[2] Jiao, J., Wang, L., Xia, W., Li, M., Sun, H., Xu, G., ... &amp; Hua, Y. (2012). Function and biochemical characterization of RecJ in <I>Deinococcus radiodurans</I>. <I>DNA repair</I>, 11(4), 349-356.</P><H3><SPAN class="mw-headline" id="RecD2">RecD2</SPAN></H3><P>DSBs are thought to be repaired through homologous recombination, a process that depends on the RecA and RecBCD enzymes in most organisms [5]. In general, the RecBCD enzyme processes the broken DNA ends through combined nuclease and helicase activities and loads the RecA protein onto the resulting single DNA strand. The single-stranded filament with the bound RecA enzyme then binds to a homologous DNA duplex, initiating DNA strand exchange and recombination. [6] Since the <I>D. radiodurans genome</I> lacks <I>recB</I> and <I>recC</I> genes, the identities of the helicase(s) and nuclease(s) that process broken DNA ends are not clear. There is, however, a gene encoding a protein called RecD2, which is similar to the RecD protein found in <I>E. coli</I> and other bacteria. [3, 6]</P><P>RecD2 from <I>Deinococcus radiodurans</I> belongs to the SF1 family of DNA helicases and unwinds DNA in the 5’-3’ direction using the energy from ATP hydrolysis [4]. Sequence analysis has shown that seven short amino acids motifs, which are conserved in SF1 helicases and found in <I>E.coli</I> RecD, are present in RecD2 as well. Yet compared to RecD found in <I>E. coli</I>, RecD2 is larger with an N-terminal extension of approximately 360-380 residues. This N-terminal region is not conserved within the RecD family members and do not contain sequences found in other helicases, indicating that RecD2 may have other functions. [1, 3, 6] </P><H4><SPAN class="mw-headline" id="ATPase_activity_and_DNA_translocation">ATPase activity and DNA translocation</SPAN></H4><P>In the absence of DNA, RecD2 has no detectable ATPase activity, but in the presence of single-stranded DNA oligomers, RecD2 exhibits efficient ATP hydrolysis activity [1]. The rate-limiting step appears to be constituted by a structural rearrangement linked to Mg2+ coordination within the enzyme, which occurs before the hydrolysis step, according to a study from Toseland et al. In the absence of magnesium ions, hydrolysis is very slow. This process, which shuttles the enzyme structure between two conformations, appears to induce DNA movement and thus accounts for the tight coupling between translocation and ATPase activity. This mechanism has also been shown for the PcrA helicase and so may be a general mechanism for SF1 helicases. [4]</P><H4><SPAN class="mw-headline" id="Helicase_activity">Helicase activity</SPAN></H4><P>RecD2 is able to unwind short DNA duplexes around 20 base pairs with either a 5’-single-stranded tail or a forked end, but not blunt-ended or 3’-tailed duplexes. Longer substrates are unwound much less efficiently. The enzyme acts catalytically, meaning that a single enzyme unwinds several DNA molecules. Duplexes with tails consisting of 10-12 nucleotides are bound tightly by RecD2, while DNA with shorter tails appear to be poor unwinding substrates since they are bound much less tightly. These results indicate that the enzyme requires very short DNA duplexes with at least a 10 nucleotide long 5’single-stranded tail for tight binding and efficient unwinding. [1]</P><P>One possible explanation for the inefficient unwinding of longer substrates is that RecD2 is a helicase with very low processivity, such that the enzyme is able to unwind shorter duplexes, but falls off before it can completely unwind longer substrates [1].This hypothesis was confirmed in a study by Shadrick et al., which demonstrated that RecD2 does have very low processivity [3]. A second possibility is that the individual DNA strands might rewind behind the enzyme as it travels along the DNA. A way to prevent rewinding in vitro is to include a single-stranded DNA-binding protein (SSB) in the mixture. In a study by Wang et al., the rate and extent of unwinding by RecD2 increased significantly in the presence of SSB from <I>D. radiodurans</I>. However, SSB from <I>E. coli</I> appeared to have little effect on the unwinding rate. [1]</P><H4><SPAN class="mw-headline" id="DNA_binding">DNA binding</SPAN></H4><P>The enzyme binds to 8 nucleotides of ssDNA in a cleft in the protein. This site contains a tyrosine residue supposedly acting as a pin to separate the strands during DNA unwinding. [3] A study conducted by Saikrishnan et al. showed that a pin-less mutant of RecD2 is a fully active ssDNA-dependent ATPase, but completely lacks helicase activity [2].  </P><H4><SPAN class="mw-headline" id="Effects_of_recD_mutations">Effects of <I>recD</I> mutations</SPAN></H4><P>In a study by Servinsky et al., <I>recD</I> mutants were sensitive to ionizing radiation, UV light and hydrogen peroxide, but resistant to mitomycin C (MMC) and methyl methanesulfonate (MMS) [6]. These results indicate that RecD2 serves some function in DNA damage repair and/or recombination. The fact that the <I>recD</I> mutant is sensitive to ionizing radiation and hydrogen peroxide  but  not  to  MMC  suggests  that  RecD2  might  be  involved  in  resistance  to  oxidation  damage, something that has also been suggested by Zhou et al. [5, 6]</P><P>References:
[1] Wang, J., &amp; Julin, D. A. (2004). DNA helicase activity of the RecD protein from <I>Deinococcus radiodurans</I>. <I>Journal of Biological Chemistry</I>, 279(50), 52024-52032.
[2] Saikrishnan, K., Griffiths, S. P., Cook, N., &amp; Wigley, D. B. (2008). DNA binding to RecD: role of the 1B domain in SF1B helicase activity. <I>The EMBO journal</I>, 27(16), 2222-2229.
[3] Shadrick, W. R., &amp; Julin, D. A. (2010). Kinetics of DNA unwinding by the RecD2 helicase from <I>Deinococcus radiodurans</I>. <I>Journal of Biological Chemistry</I>, 285(23), 17292-17300.
[4] Toseland, C. P., &amp; Webb, M. R. (2013). ATPase Mechanism of the 5′-3′ DNA Helicase, RecD2 - Evidence for a Pre-hydrolysis Conformation Change. <I>Journal of Biological Chemistry</I>, 288(35), 25183-25193.
[5] Zhou, Q., Zhang, X., Xu, H., Xu, B., &amp; Hua, Y. (2007). A new role of <I>Deinococcus radiodurans</I> RecD in antioxidant pathway. <I>FEMS microbiology letters</I>, 271(1), 118-125.
[6] Servinsky, M. D., &amp; Julin, D. A. (2007). Effect of a <I>recD</I> mutation on DNA damage resistance and transformation in <I>Deinococcus radiodurans</I>. <I>Journal of bacteriology</I>, 189(14), 5101-5107.
</P><H3><SPAN class="mw-headline" id="PprA">PprA</SPAN></H3><P>It is believed that many different repair processes are active upon DNA damage in <I>D. radiodurans</I>. Non-homologous end-joining is thought to be one of these mechanisms, where PprA plays a critical role [2]. PprA is a pleiotropic protein promoting DNA repair in <I>D. radiodurans</I>. It acts by stimulating other enzymes such as ATP-dependent and NAD-dependent DNA ligases in the DNA repair process. [1, 2] <I>Thermus thermophiles</I> HB27 is the most closely related genus to <I>Deinococcus</I>, but does not contain a <I>pprA</I> homologue, emphasizing the uniqueness of the DNA repair mechanisms of <I>D. radiodurans</I> [2].</P><H4><SPAN class="mw-headline" id="DNA_binding_2">DNA binding</SPAN></H4><P>PprA is able to form complexes with either dsDNA in an open circular form or linear dsDNA, but not dsDNA in a closed circular form or ssDNA, indicating that it binds preferentially to dsDNA containing strand breaks. PprA is also able to inhibit the activity of exonucleases when bound to DNA, thus protecting the strands from degradation. [2]</P><H4><SPAN class="mw-headline" id="Stimulation_of_enzymes">Stimulation of enzymes</SPAN></H4><P>PprA has been shown to be able to stimulate DNA ligation catalyzed by T4 DNA ligase and <I>E. coli</I> DNA ligase, although PprA itself does not possess DNA ligation activity. Narumi et al. showed that the degree of stimulation was dependent on the concentration of PprA. [2] Upon being phosphorylated, PprA increases its DNA binding affinity for dsDNA and is thereby able to stimulate T4 DNA ligase at a much higher rate. [4] In another study by Kota et al., PprA was also able to stimulate the activity of catalase from <I>E. coli</I> both in vitro and in vivo. The transgenic <I>E. coli</I> cells expressing PprA has been shown to have a higher tolerance to hydrogen peroxide as compared to wild type cells due to an increase in antioxidant enzyme activity. [3]</P><P>References:
[1] Tanaka, M., Earl, A. M., Howell, H. A., Park, M. J., Eisen, J. A., Peterson, S. N., &amp; Battista, J. R. (2004). Analysis of <I>Deinococcus radiodurans</I>'s transcriptional response to ionizing radiation and desiccation reveals novel proteins that contribute to extreme radioresistance. <I>Genetics</I>, 168(1), 21-33.
[2] Narumi, I., Satoh, K., Cui, S., Funayama, T., Kitayama, S., &amp; Watanabe, H. (2004). PprA: a novel protein from <I>Deinococcus radiodurans</I> that stimulates DNA ligation. <I>Molecular microbiology</I>, 54(1), 278-285.
[3] Kota, S., &amp; Misra, H. S. (2006). PprA: A protein implicated in radioresistance of <I>Deinococcus radiodurans</I> stimulates catalase activity in Escherichia coli. <I>Applied microbiology and biotechnology</I>, 72(4), 790-796.
[4] Rajpurohit, Y. S., &amp; Misra, H. S. (2013). Structure-function study of deinococcal serine/threonine protein kinase implicates its kinase activity and DNA repair protein phosphorylation roles in radioresistance of <I>Deinococcus radiodurans</I>. <I>The international journal of biochemistry &amp; cell biology</I>, 45(11), 2541-2552.</P><H1><SPAN class="mw-headline" id="Safety_Switch">Safety Switch</SPAN></H1><P>It is our view as a team, that taking responsibility for the work conducted is of highest importance. One of the main aspects of this commitment is the prevention of modified organisms ever escaping the laboratory and accidentally ending up in nature. For this purpose, a genetic construct has been designed called the safety switch, which will serve to address both ethical and safety issues regarding the manufacturing of a more resilient yeast strain.</P><H2><SPAN class="mw-headline" id="The_SUC2_regulation_in_Saccharomyces_cerevisiae">The SUC2 regulation in <I>Saccharomyces cerevisiae</I></SPAN></H2><P>The SUC2 gene in <I>Saccharomyces cerevisiae</I> codes for the enzyme invertase, which catalyses the hydrolysation of the glycosidic bond in the sucrose dimer, breaking it down into its constituent monomers, glucose and fructose.[1] This enzyme is important to the survivability of the yeast cells since it allows them to utilise sucrose as an alternative carbon source should they ever be in a glucose deprived environment. However, this is a less efficient way of obtaining energy than simply metabolising glucose, hence it should be used exclusively when glucose is in short demand. Thus, in order to regulate this, the expression of the SUC2 gene is sensitive to the amount of ATP available to the cell.[2] </P><P>This ATP sensitivity originates from the repression of the SUC2 gene, when enough ATP is present, using a repressor complex which is made up by the repressor protein MIG1 and the general repressor complex SSN6-TUP1. This conjoined repressor binds to the upstream promoter region of the SUC2 gene[3], where it will prevent transcription. However, the assembly of this repressor complex can be hindered by the protein kinase SNF1. A SNF1-dependent phosphorylation of the MIG1 protein makes it unable to interact with the SSN6-TUP1 complex[4], resulting in the SUC2 gene being expressed. Furthermore, the SNF1 kinase will, in turn, be blocked by the presence of glucose, resulting in a glucose dependent mechanism for the expression of the SUC2 gene[2]. </P><P>This means that when glucose is present in high enough concentration, the SUC2 gene will remain repressed due to the formation of the repressor complex. Thus, when the glucose concentration is low, the repressor complex cannot assemble which results in the gene being expressed.</P><H2><SPAN class="mw-headline" id="The_Safety_Switch">The Safety Switch</SPAN></H2><P>Our ambition was to design a genetic construct with the ability to sense whether the cells were still in the lab, or not. For this purpose, we chose the glucose dependence of the SUC2 gene as a way for the cells to sense their environment. This was mainly because we can ensure that the glucose levels are always kept high when working with the cells in the lab, as well as we know that the access to glucose will be much more scarce at any other location. This makes the SUC2 regulation a very appropriate choice for making the cells “understand” when they are no longer in the right place.</P><P>The Safety Switch utilises this system of regulation, except for the gene itself. The SUC2 gene has been replaced with a new gene, which is harmful to the cells, and which will be expressed instead when the glucose concentration is low. For this purpose, we have chosen the gene TPK2 which only becomes dangerous to the cells when it is overexpressed[6]. This choice was motivated by the fact that the repressor of the gene is not perfect, and there will always be small expression of the gene, even when repressed. The fact that a certain amount of TPK2 is required before it becomes dangerous to the cells makes sure that they are not accidentally harmed due to the marginal inefficiency of the repressor. However, this also poses a potential problem when the Safety Switch is activated, since the expression needs to be sufficiently strong for the TPK2 to have the desired effect. Therefore, in addition to the pSUC2 promoter, another promoter called pTEF1 is added upstream of the pSUC2. This is because we want to elevate the expression level of the gene, without altering the way it is regulated. Due to the fact that the pSUC2 region lies in between the new promoter and the gene, the pTEF1 promoter cannot activate the expression of the gene as long as the repressor is bound to the pSUC2. Furthermore, the short length of the pSUC2 promoter allows the pTEF1 promoter to activate expression of <I>TPK2</I>, even though they are not adjacent to each other.</P><P> Expression of <I>mRFP</I> (monomeric Red Fluorescent Protein) was used to analyze expression levels of the safety switch. To evaluate the effect of connecting pTEF1 to pSUC2, two more versions of the safety switch were made: pTEF1-pSUC2-<I>mRFP</I> and pSUC2-<I>mRFP</I>. This allowed comparison of gene expression between these two set ups in order to determine whether or not the addition of pTEF1 actually resulted in a higher level of gene expression.</P><P>  References:
[1] Saccharomyces Genome Database (2010) SUC2 / YIL162W <A rel="nofollow" class="external free" href="http://www.yeastgenome.org/locus/SUC2/overview">http://www.yeastgenome.org/locus/SUC2/overview</A> (September 17:th 2015)
[2] García‐Salcedo, R., Lubitz, T., Beltran, G., Elbing, K., Tian, Y., Frey, S., ... &amp; Hohmann, S. (2014). Glucose de‐repression by yeast AMP‐activated protein kinase SNF1 is controlled via at least two independent steps. Febs Journal, 281(7), 1901-1917.(September 17:th 2015)
[3] Özcan, S., Vallier, L. G., Flick, J. S., Carlson, M., &amp; Johnston, M. (1997). Expression of the SUC2 gene of <I>Saccharomyces cerevisiae</I> is induced by low levels of glucose. Yeast, 13(2), 127-137.(September 17:th 2015)
[4] Papamichos‐Chronakis, M., Gligoris, T., &amp; Tzamarias, D. (2004). The Snf1 kinase controls glucose repression in yeast by modulating interactions between the Mig1 repressor and the Cyc8‐Tup1 co‐repressor. EMBO reports, 5(4), 368-372.(September 17:th 2015)
[5] Saccharomyces Genome Database (2007) TPK2 / YPL203W <A rel="nofollow" class="external free" href="http://www.yeastgenome.org/locus/Tpk2/overview">http://www.yeastgenome.org/locus/Tpk2/overview</A> (September 17:th 2015)</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>