Team:Chalmers-Gothenburg/Description
Home
Students
Supervisors
Attributions
Gallery
Team
Introduction
Theory
Constructs
Other Applications
Project
Project Results
Bio Bricks
Medals
Results
Achievements
Detection
Safety Switch
Simulation
Modeling
Collaborations
Next Generation
Society
Human Practices
Lab Journal
Protocols
Notebook
Lab Safety
Ethics
Safety
Sponsors
Acknowledgements
Thanks!
Project Description
Tell us about your project, describe what moves you and why this is something important for your team.
What should this page contain?
A clear and concise description of your project.
A detailed explanation of why your team chose to work on this particular project.
References and sources to document your research.
Use illustrations and other visual resources to explain your project.
Advice on writing your Project Description
We encourage you to put up a lot of information and content on your wiki, but we also encourage you to include summaries as much as possible. If you think of the sections in your project description as the sections in a publication, you should try to be consist, accurate and unambiguous in your achievements.
Judges like to read your wiki and know exactly what you have achieved. This is how you should think about these sections; from the point of view of the judge evaluating you at the end of the year.
References
iGEM teams are encouraged to record references you use during the course of your research. They should be posted somewhere on your wiki so that judges and other visitors can see how you though about your project and what works inspired you.
Inspiration
See how other teams have described and presented their projects:
Imperial
UC Davis
SYSU Software
Detection of contaminants
To detect contaminations in a bioreactor we use a system that utilizes extracellular receptors responding to ligands of the unwanted contaminant. The detection system goes by the name DAS, detection and amplification of signal, and it is based on the pheromone response pathway in Saccharomyces cerevisiae. The pheromone pathway uses a G-protein coupled receptor (GPC-receptor) to detect pheromones of another yeast mating type. When the receptor comes in contact with the correct agonist, which is pheromone α for a-type cells and pheromone a for α-type cells, the receptor then activates a pathway which expresses genes that prepares the yeast for mating. In our system we plan to hijack this system and instead make the yeast signal that a contamination is present.
G PROTEIN COUPLED RECEPTORS,GPC receptors and their advantages
GPC receptors, also known under the name seven-transmembrane receptors are among the most common and versatile family of receptors found in eukaryotes. The receptors found in this kind of system are signal mediators through many subjects, such as light, olfactory stimulation, and many other signaling substances commonly found in mammals. They act by activating signal transduction pathways inside the cell, resulting in an intracellular response to an external signal.
BILD GPCR receptor!
Figure 1. A common GPCR protein informative text Something something dark side something
The receptors share a common structure among all eukaryotic cells, and a major characteristic is, as the name suggests, that the receptor has a transmembrane region which pass through the membrane seven times. The N-terminal of the receptor is located extracellularly and it is responsible for transporting and annealing the protein to the cell membrane. In some cases it also serves as a ligand binding site, although it is more common that the ligands associate with one of the three extracellular loops that reside between the transmembrane sites. The C-terminal of the GPC receptor is a cytosolic region and thus has no ligand associating function whatsoever, instead it has an important role as a mediator between the receptor and the cell or more specifically the G protein that associates with the receptor.
BILD GPCR P PROTEIN INTERAKTION!
The GPC-receptors as the name indicates are bound to G proteins within the cell.  G proteins are trimeric complexes consisting of three different proteins, called Gα, Gβ, and Gγ. G proteins is an abbreviation for guanine nucleotide-binding proteins and as the name implies is a protein which binds guanine in the form of GDP or GTP. In a natural state the protein prefers to be bound by GDP, while this is the state the trimeric complex is bound together and has no catalytic activity. During ligand association of the GPC receptor the GDP previously bound to the Gα is replaced with a GTP. While the Gα protein is bound by a GTP the Gβ and Gγ separates as a dimer. In this separate phase the Gβγ or the Gα subunit are free to interact with other molecules, this often leads to the activation of a pathway that ultimately causes a cellular response to an external signal [1, 2, 3].
PHEROMONE RESPONSE PATHWAY ,GPC receptor pathway in saccharomyces cerevisiae
The pheromone response pathway in yeast is one of the most studied pathways of eukarya. Due to yeasts important position as a role model organism for eukaryotic cells much attention has been given to the few GPCR pathways that exist in it, the pheromone pathway is one of them. The pathway induces changes such as a cell cycle arrest and conformational changes due to filamentous response genes when a pheromone of a mating partner is located. The following figure depicts a simplified version of the pheromone response pathway.
BILD FEROMONVÄGEN!
The figure depicts an a-type yeast cell containing the receptor Ste2, this receptor interacts with the pheromone from the α-type yeast cell called α-factor. The receptor Ste2 is in turn intracellularly bound to the three G-protein units α, γ, and β named Gpa1, Ste18, and Ste4 respectively. During ligand association the agonist pheromone produces a conformational change in the receptor structure, exposing parts of the receptor which catalyses the exchange of the GDP bound to the intracellular α-subunit to a GTP. When the diphosphate is replaced with a triphosphate the conformation of the trimer changes, and the β- and γ-subunits separate as a dimer from the α-subunit.
This Gβγ dimer interacts with two effectors called Ste20 and Ste5. Ste20 is a PAK (p21-activated protein kinase) and its kinase activity is enabled when it is bound to the G protein Cdc42. Ste5 is a scaffold protein which attracts and binds the three proteins Ste11, Ste7, and Fus3 which are MEKK/MAP3K (Mitogen activated protein kinase kinase kinase), MEK/MAP2K (Mitogen activated protein kinase kinase), and MAPK (Mitogen activated protein kinase) respectively. When both Ste5 and Ste20 binds to the Gβγ dimer it allows for the activated Ste20 kinase to phosphorylate the Ste11 MEKK thus triggering a MAPK cascade. This phosphorylation cascade ends in the release of activated Fus3, which in turn is transported to the nucleus where it activates the transcription factor Ste12. [3]
Detection system
In our system, the natural receptor (Ste2 from a-cells) has been replaced with a fusion receptor consisting partially of the natural Ste2 receptor but mostly a pheromone receptor called Mam2 originating from the yeast Schizosaccharomyces pombe. Due to both of those receptors being GPC-receptors with a similar Gα subunit the fusion receptor is capable of interacting with the G-protein of the pheromone response pathway in Saccharomyces cerevisiae. Depicted below is the modified system we use for detection:
BILD DAS!
Fusion receptor
The fusion receptor contains the N-terminus signal peptide of the Ste2 receptor so that the receptor is correctly transported and integrated into the outer membrane of the S. cerevisiae cell wall. The intermediate region and the C-terminus of the receptor is replaced with the equivalent parts of the Mam2 receptor, which makes the receptor capable of interacting with the correct ligand. Since the GPC-receptor is mostly dependent on the extracellular loops in regard of what ligand it associates with it is possible to create a wide array of receptors simply by replacing this region in the receptor. [4] There are many receptors to be found in the eukaryotic kingdom, each with its own specificity. [5]
Amplification
In order to read a strong signal when a contaminant is detected we have decided to use intracellular amplification to strengthen the signal. To achieve this in an efficient manner, while holding on to the work previously done in the iGEM competition we decided to use the dCas9-Vp64 complex that was used the previous year by Team Gothenburg. By linking an activation domain to a deactivated version of the CRISPR associated protein, dCas9, one obtains a transcription factor guided by the same means as the popular CRISPR system. By creating a promoter with a binding site for a specific RNA sequence and expressing this RNA to act as a guide RNA (gRNA for short) for the dCas9 protein it is possible to make an incredibly precise tool for gene expression.
Vp64 is a viral protein complex consisting of four Vp16 activation domains, originating from the herpes virus, which when combined form the strong activation domain called Vp64.  By coupling this activation domain to a DNA binding protein such as dCas9 a transcription factor with a strong induces expression is created. [6]
However the expression of a new transcription factor in itself, even though it induces a strong expression, is not guaranteed to produce a strong final signal, therefore we decided to amplify the amount of transcription factor through a feed-back activation loop, where we coupled the expression of the transcription factor (dCas9-Vp64) to the promoter that it specifically bind to, pCYC1m. The results of this coupling should be a heightened amount of transcription factor in the cell, allowing for close to the maximum expression of the signal substance, in our case RFP.
BILD AMPLIFICATION!
The dCas9-Vp64 complex is transcribed on the same gene as the gRNA, but thanks to the use of ribozymes these two parts are separated in RNA form. [7] The dCas9-Vp64 comes with more advantages than being a strong and specific transcription factor, due to the system being meant to detect contaminations a false signal would mean that one is alarmed of a contamination and might irradiate the batch in vain. Which mean the cells could be prone false signaling and due to our feed-back activation loop this false expression would lead to a full alarm. However, the dCas9-Vp64 complex requires gRNA as a means of guiding it to the correct promoter. This means that we have a two component system which will not induce expression unless those two components have met. Basically this raises the bar for the initial signal needed to induce signaling as a single expression of the CGC1 gene would create one of each component, and they are unlikely to meet when in such low amounts inside the cell, therefore the activation domain Vp64 would never be guided to the pCYC1m promoter and thus not induce any false expressions.
Referenser: [1] https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3343417/ Allmänt om 7TM [2] http://emboj.embopress.org/content/18/7/1723 Allmänt om 7TM  [3] http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3017506/  walkthrough  [4] http://www.ncbi.nlm.nih.gov/pubmed/21915853 fusionsreceptor [5] http://www.ncbi.nlm.nih.gov/pubmed/17139284 många drug targets  [6] http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3794058/ dCas9-Vp64 [7] http://www.ncbi.nlm.nih.gov/pubmed/15547135 ribozym
