<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Chalmers-Gothenburg_SafetySwitch skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Chalmers-Gothenburg/SafetySwitch</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="main"><DIV id="my_content"><DIV id="section1"><DIV id="section1-menu"><NAV><UL><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg">Home</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Students">Students</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Supervisors">Supervisors</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Gallery">Gallery</A></LI></UL><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Team" class="arrow">Team</A><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Project">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Theory">Theory</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Constructs">Constructs</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/OtherApplications">Other Applications</A></LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Project" class="arrow">Project</A><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/ProjectResults">Project Results</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/BioBricks">Bio Bricks</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Medals">Medals</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Results">Results</A></LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Achievements" class="arrow">Achievements</A><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Detection">Detection</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/SafetySwitch">Safety Switch</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Simulation">Simulation</A></LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Modeling" class="arrow">Modeling</A><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/iGEMCommunity">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/NextGeneration">Next Generation</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Society">Society</A></LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/HumanPractices" class="arrow">Human Practices</A><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Schedule">Lab Journal</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Protocols">Protocols</A></LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Notebook" class="arrow">Notebook</A><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/LabSafety">Lab Safety</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Ethics">Ethics</A></LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Safety" class="arrow">Safety</A><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Sponsors">Sponsors</A></LI><LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Acknowledgements">Acknowledgements</A></LI><A href="https://2015.igem.org/Team:Chalmers-Gothenburg/Thanks" class="arrow">Thanks!</A></NAV></DIV><DIV id="section1-content"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Safety_Swtich_moodel"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Safety Swtich moodel</SPAN></A></LI><LI class="toclevel-2"><A href="#Defining_the_model"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Defining the model</SPAN></A></LI><LI class="toclevel-2"><A href="#Parameters"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Parameters</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#Final_Model"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Final Model</SPAN></A></LI><LI class="toclevel-2"><A href="#High_glucose"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">High glucose</SPAN></A></LI><LI class="toclevel-2"><A href="#Low_glucose"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Low glucose</SPAN></A></LI><LI class="toclevel-2"><A href="#High_to_Low_glucose"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">High to Low glucose</SPAN></A></LI><LI class="toclevel-2"><A href="#References"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">References</SPAN></A></LI></DIV><H2><SPAN class="mw-headline" id="Safety_Swtich_moodel">Safety Swtich moodel</SPAN></H2><P>The safety switch is based on using the regulation of the SUC2 gene to express a potential harmful kinase domain Tpk2. A well know, always on, overexpression promotor pTEF1 was used to amplify the protein expression while the first promotor, pSUC2 is not down regulated. SUC2 is downregulated by the binding if a Zink finger containing protein Mig1 which needs to be in a complex with the Co repressor SSN6/Tup1 to be active as a repressor. </P><H3><SPAN class="mw-headline" id="Defining_the_model">Defining the model</SPAN></H3><P>There have been quite a lot of modeling done one the Mig1 activity based on sugar abundance in the media.  2009, Frey et al managed to create a model with predictable predictability. [1] The model used a heavy side function describe the activity, Mig1 which is a of the Zinc finger containing protein involved in repressing the SUC2 gene. The heavy side functions on or of state was based on the glucose level in the media. Due to the good predictability of the Mig1 model, the first corner stone of the model was set.  Non-associated Ssn+6 and Tup1 was assumed to be relevant for the expression and was therefore included in the model.  The repression mechanism used in the model was a physical mechanism, when the repressor is associated to the promotor, no gene expression is active. And the assumption that a repressor free promoter leads to a linear gene expression was made.</P><P class="figuretext">Figure 1, the modeled pathway in the Safety switch model.</P><P>The reactions were defined as:</P><P class="figuretext">Figure 2.The modelled pathway in the safety switch model  .</P><P>Yielding in the following reaction equations.</P><H3><SPAN class="mw-headline" id="Parameters">Parameters</SPAN></H3><P>Parameters k1, k-1, k2 and k-2 were found in the literature. However, the reaction constant for Reaction 3 were only found as a disassociation constant. An arbitrary number was set as one of the reaction, 2×10^2 /(m*s)and the other parameter was calculated based on the disassociation constant and the arbitrary number.
For reaction 4, two disassociation constants were found due to the Mig1 having zinc fingers and both binding to the DNA strand, the larger value was chosen randomly [5]. We looked into a potential way of modelling the two zinc finger/DNA association, specifically when the first zinc finger is bound to DNA. While in that state the possible volume for the second finger is highly decreased and the local concentration will be increased. However we had no way of determining the possible volume for the second association. The idea was therefore dropped.</P><P>We found no absolute data on the amount of proteins produced from the specific promoters, we therefore lumped all the reactions and gave it an arbitrary magnitude to get a feeling of the relative expression of the different systems and conditions. However due to this fact there is no quantitative predictability of the resulting protein levels, only relative. The literature study also indicated that the promoter had similar relative strength while SUC2 was not down regulated [6] The decay rate of the TPK2 was fitted so that the derivative of the concentration of TPK2 was almost zero in the  Only pSUC2 promoter low abundance of glucose case. .</P><P class="figuretext">Table 1, The rates of the reaction constants and their sources.</P><TABLE style="width:100%; border-collapse: collapse"><TBODY><TR><TD>[Ssn6]</TD><TD>3890 molecules/cell</TD></TR><TR><TD>[Tup1]</TD><TD>5840 molecules/cell</TD></TR><TR><TD>[Mig1]</TD><TD>830 molecules/cell</TD></TR><TR><TD>[pSUC2]</TD><TD>2 molecules/cell</TD></TR><TR><TD>[PTEF1]</TD><TD>2 molecules/cell</TD></TR><TR><TD>[TPK2]</TD><TD>2220 molecules/cell</TD></TR><TR><TD>Rest</TD><TD>0 molecules/cell</TD></TR></TBODY></TABLE><P class="figuretext">Table 2, The boundary conditions for the model [7] .</P><H2><SPAN class="mw-headline" id="Final_Model">Final Model</SPAN></H2><P>The four variants of the model was plotted
</P><H3><SPAN class="mw-headline" id="High_glucose">High glucose</SPAN></H3><P> There is only a small difference between the plots. The output signal is a bit higher on the pTEF1 and pSUC2 model. But the difference is only minor. This prediction was true, by doing empirical testing of the safety switch results.</P><P class="figuretext">Figure 6.Both pTEF1 and pSUC2as promoters, The concentration, in Mol, of all components over time in the absence of glucose.</P><P class="figuretext">Figure 4.Only pSUC2 as promoter, The concentration, in Mol, of all components over time in the absence of glucose.</P><H3><SPAN class="mw-headline" id="Low_glucose">Low glucose</SPAN></H3><P> The difference between the two promoters is bigger compared to the high glucose case.This prediction was true, verified by doing empirical testing of the safety switch.  </P><P class="figuretext">Figure 3.Only pSUC2 as promoter, The concentration, in Mol, of all components over time in the presence of glucose.</P><P class="figuretext">Figure 5. Both pTEF1 and pSUC2 as promoters, The concentration, in Mol, of all components over time in the presence of glucose.</P><H3><SPAN class="mw-headline" id="High_to_Low_glucose">High to Low glucose</SPAN></H3><P> There was a big difference between the tpk2 levels in the prediction.This prediction was also true,it has been verified by doing empirical testing of the safety switch.  </P><P class="figuretext">Figure 5. Both pTEF1 and pSUC2 as promoters, The concentration, in Mol, of all components over time in the presence of glucose.</P><P class="figuretext">Figure 6.Both pTEF1 and pSUC2as promoters, The concentration, in Mol, of all components over time in the absence of glucose.</P><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><P>[1] Frey, S., Sott, K., Smedh, M., Millat, T., Dahl, P., Wolkenhauer, O., &amp; Goksör, M. (2011). A mathematical analysis of nuclear intensity dynamics for Mig1-GFP under consideration of bleaching effects and background noise in Saccharomyces cerevisiae. Molecular BioSystems, 7(1), 215-223</P><P>[2] García‐Salcedo, R., Lubitz, T., Beltran, G., Elbing, K., Tian, Y., Frey, S., ... &amp; Hohmann, S. (2014). Glucose de‐repression by yeast AMP‐activated protein kinase SNF1 is controlled via at least two independent steps. Febs Journal, 281(7), 1901-1917.</P><P>[3] Jabet, C., Sprague, E. R., VanDemark, A. P., &amp; Wolberger, C. (2000). Characterization of the N-terminal Domain of the Yeast Transcriptional Repressor Tup1 PROPOSAL FOR AN ASSOCIATION MODEL OF THE REPRESSOR COMPLEX Tup1· Ssn6. Journal of Biological Chemistry, 275(12), 9011-9018.</P><P>[4] Brinker, A., Scheufler, C., Von der Mülbe, F., Fleckenstein, B., Herrmann, C., Jung, G., ... &amp; Hartl, F. U. (2002). Ligand discrimination by TPR domains relevance and selectivity of eevd-recognition in Hsp70· Hop· Hsp90 complexes. Journal of Biological Chemistry, 277(22), 19265-19275.</P><P>[5]  Needham, P. G., &amp; Trumbly, R. J. (2006). In vitro characterization of the Mig1 repressor from Saccharomyces cerevisiae reveals evidence for monomeric and higher molecular weight forms. Yeast, 23(16), 1151-1166.</P><P>[6] Williams, T. C., Espinosa, M. I., Nielsen, L. K., &amp; Vickers, C. E. (2015). Dynamic regulation of gene expression using sucrose responsive promoters and RNA interference in Saccharomyces cerevisiae. Microbial cell factories, 14(1), 43.</P><P> [7] Chong, Y. T., Koh, J. L., Friesen, H., Duffy, K., Cox, M. J., Moses, A., ... &amp; Andrews, B. J. (2015). Yeast proteome dynamics from single cell imaging and automated analysis. Cell, 161(6), 1413-1424.




</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>