Team:LambertGA/Description
Home
Team
Team Collaborations
Project
Description Design Experiments Proof of Concept Results Notebook
Parts
Parts Basic Parts Composite Parts
Attributions Safety
Human Practices
Human Practices Silver Gold Integrated Practices Engagement
Awards
Hardware Measurement Model
Description
Inspired by the Georgia Tech zinc biosensor, the Lambert High School iGEM Team studied protein degradation to enhance the development of modern biosensors. Today, biosensors are complex biological systems that provide a more reliable and affordable alternative to expensive diagnostic tests. However, in many biosensors, the overexpression of one pigment over another can result in inaccurate readings and subsequently faulty diagnosis.For example, in the hypothetical biosensor X, the system is designed such that it will express purple when the concentration of X is less than 4ùûµM, orange when the concentration of X is between 4ùûµM and 6ùûµM, and red when the concentration of X is between 6ùûµM and 10ùûµM. However, due to leakiness that may exist in the system or fluctuations in the affinities of the biosensing promoters, the purple reporter is overexpressed, as evident by the figure the below.
To address this issue, the Lambert iGEM Team devised a ‚Äúswitch‚Äù - a genetically engineered construct that degrades SsrA-tagged proteins upon induction by IPTG. Ultimately, our main goal was to quantify the relative strength of degradation and subsequently further characterize a known protease system.
In bacterial cells, SsrA is a monocistronic gene that codes for a specialized strand of RNA known as tmRNA; tmRNA is responsible for rescuing ribosomes from abnormally truncated mRNA templates that lack terminator codons.
This process is the SsrA-tagged degradation pathway, whereby tmRNA binds with the ribosome and cotranslationally adds a degradation tag to the defective mRNA strand, hence marking the nascent polypeptide for degradation. (Tao et. al., 2015) In effect, this degradation tag, called the SsrA tag, serves as protein quality control in the cell by preventing the accumulation of aberrant and incomplete proteins. (Keiler et. al., 1996) These tagged proteins are then transported to some protease by a bacterial adaptor. In E. coli cells, SspB is the bacterial adapter that transports tagged proteins to the ClpXP protease. Because this pathway is the most common form of proteolysis in prokaryotes, we decided to exploit this pathway to control the overexpression of pigments in biosensors.
ClpXP is classified as an ATP dependent protease (ATPase) that harnesses energy released from ATP hydrolysis to render protein degradation. Moreover, ClpXP is a protein mechanism that is composed of two separate proteins - ClpX and ClpP. ClpX is the protein that unfolds and translocates the tagged protein into a sequestered proteolytic compartment in ClpP; ClpP is the protein that breaks the individual covalent bonds (polypeptide bonds) that exist between the individual amino acids of the primary structure of the protein.
Accordingly, we devised an inducible genetic construct to study how ClpXP degrades SsrA-tagged GFP and chromoproteins. We grew these constructs in DH10 E. coli strains; however, since the DH10 strains have naturally occurring ClpXP, we decided to grow these constructs in Keio Wild knockout strains to accurately quantify the relative strength of degradation. We were able to successfully grow our constructs in the Keio ClpP Knockout strains and one of the constructs in the Keio Wild strain, but due to the time constraints, we were unable to gain any results with the ClpX Knockout strain. We used the fluorescent plate reader at the Styczynski Lab to obtain these results.  Ultimately, the main goal of our project was to measure the relative strength of degradation of a tagged reporter and consequently further characterize a protease mechanism.
Improving Previous Parts
Our 2016 iGEM team used existing Biobrick parts from the Uppsala team (BBa_K1033906) and also the Endy Lab (BBa_M0050-LAA, M0052-DAS). We furthered Edinburgh‚Äôs work by expanding upon their experiments with GFP and the degradation tags, replacing GFP with the tsPurple chromoprotein and characterizing the ClpXP system.
We first obtained the tsPurple sequence from the 2013 Uppsala iGEM wiki page. The sequence was ordered online from IDT, hydrated, and assembled with 3A assembly to create our constructs. We obtained the tsPurple with no degradation tag, tsPurple with the LAA degradation tag, and tsPurple with the DAS degradation tag. The degradation tags were obtained from the Endy Lab sequencing and correspond to strong degradation for LAA and moderate for DAS.
We sought to improve upon the LAA and DAS degradation tags by demonstrating their relative strength in degrading the tsPurple chromoprotein and to improve upon the characterization of the tsPurple gene so that its relative degradation may be present for anyone who needs it.
References
And, S. A. (2009, February 13). Sarita Ahlawat. ClpXP Degrades SsrA-tagged proteins in S.pneumoniae.Retrieved Summer, 2016, from http://jb.asm.org/content/191/8/2894.full
Andersen , J.B. , Sternberg , C. , Poulsen , L.K. , Bjorn , S.P. , Givskov , M. , and Molin , S. ( 1998 ) New unstable variants of green fluorescent protein for studies of transient gene expression in bacteria . Appl Environ Microbiol 64 : 2240 ‚Äì 2246 .
Baker, T. A., & Sauer, R. T. (2011, June 27). ClpXP, an ATP-powered unfolding and protein-degradation machine. Retrieved Summer, 2016, from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3209554/
Bar-Nun, S., & Glickman, M. H. (2012). Proteasomal AAA-ATPases: Structure and function. Biochimica et Biophysica Acta (BBA) - Molecular Cell Research, 1823(1), 67‚Äì82. doi:10.1016/j.bbamcr.2011.07.009. Retrieved Summer, 2016 from http://www.sciencedirect.com/science/article/pii/S0167488911001984
Bohn , C. , Binet , E. , and Bouloc , P. ( 2002 ) Screening for stabilization of proteins with a trans-translation signature in Escherichia coli selects for inactivation of the ClpXP protease . Mol Genet Genomics 266 : 827 ‚Äì831 .
Burton , R.E. , Siddiqui , S.M. , Kim , Y.I. , Baker , T.A. , and Sauer , R.T. ( 2001 ) Effects of protein stability and structure on substrate processing by the ClpXP unfolding and degradation machine . EMBO J 20 : 3092 ‚Äì3100 .
Ciechanover, A. (2005). Cell death and differentiation - abstract of article: Intracellular protein degradation: From a vague idea thru the lysosome and the ubiquitin-proteasome system and onto human diseases and drug targeting[ast]. Cell Death & Differentiation, 12(9), 1178‚Äì1190. doi:10.1038/sj.cdd.4401692
Cooper, G. M. (2000). Protein degradation. Retrieved Summer, 2016 from http://www.ncbi.nlm.nih.gov/books/NBK9957/
Farrell, C., Grossman, A., & Sauer, R. (2005). Cytoplasmic degradation of ssrA-tagged proteins.Molecular microbiology., 57(6), 1750‚Äì61. Retrieved from http://www.ncbi.nlm.nih.gov/pubmed/16135238
Flynn , J.M. , Levchenko , I. , Seidel , M. , Wickner , S.H. , Sauer , R.T. , and Baker , T.A. ( 2001 ) Overlapping recognition determinants within the ssrA degradation tag allow modulation of proteolysis . Proc Natl Acad Sci USA 11 : 10584 ‚Äì 10589.
Georgia Institute of Technology. (2015, September 1). ‚ÄúBacterial litmus Test‚Äù provides inexpensive measurement of Micronutrients. Retrieved from GT News Center, http://www.news.gatech.edu/2015/09/01/bacterial-litmus-test-provides-inexpensive-measurement-micronutrients
Goldberg, A.L., A.S. Menon, S. Goff and D.T. Chin. 1987. The mechanism and regulation of the ATP-dependent protease La from Escherichia coli. Biochem. Soc. Trans. 15: 809-811. Retrieved October 1, 2016 from http://www.fao.org/wairdocs/ilri/x5550e/x5550e0d.htm
Hwang BJ, Woo KM, Goldberg AL, Chung CH. Protease Ti, a new ATP-dependent protease in Escherichia coli,contains protein-activated ATPase and proteolytic functions in distinct subunits. J Biol Chem. 1988;263:8727‚Äì8734.
Katayama-Fujimura Y, Gottesman S, Maurizi MR. A multiple-component, ATP-dependent protease from Escherichia coli. J Biol Chem. 1987;262:4477‚Äì4485.
Landry, B. P., & St√∂ckel, J. (2013). Use of degradation tags to control protein levels in the Cyanobacterium Synechocystis sp. Strain PCC 6803. Applied and Environmental Microbiology,79(8), 2833‚Äì2835. doi:10.1128/AEM.03741-12
Lee C, Schwartz MP, Prakash S, Iwakura M, Matouschek A. ATP-Dependent Proteases Degrade Their Substrates by Processively Unraveling Them from the Degradation Signal.
McNerney, M. P., Watstein, D. M., & Styczynski, M. P. (2015). Precision metabolic engineering: The design of responsive, selective, and controllable metabolic systems. Metabolic Engineering, 31, 123‚Äì131. doi:10.1016/j.ymben.2015.06.011
Minikel, E. V. (2013, June 11). Basics of protein degradation. Retrieved Summer, 2016, from http://www.cureffi.org/2013/07/11/basics-of-protein-degradation/
Mogk A, Schmidt R, Bukau B. The N-end rule pathway for regulated proteolysis: prokaryotic and eukaryotic strategies. Trends Cell Biol. 2007;17:165‚Äì172.
Purcell, O., Grierson, C. S., Bernardo, M. di, & Savery, N. J. (2012). Temperature dependence of ssrA-tag mediated protein degradation. Journal of Biological Engineering, 6(1), . doi:10.1186/1754-1611-6-10
Schrader, E. K., Harstad, K. G., & Matouschek, A. (n.d.). Targeting proteins for degradation. , 5(11), . Retrieved from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4228941/
Snider, J., Thibault, G., & Houry, W. A. (2008). The AAA+ superfamily of functionally diverse proteins. , 9(4), . Retrieved Summer, 2016 from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2643927/
Tanaka K. The proteasome: overview of structure and functions. Proc Jpn Acad Ser B Phys Biol Sci.2009;85:12‚Äì36.
Tao, L., & Biswas, I. (2015). Degradation of SsrA-tagged proteins in streptococci. , 161(Pt 4),. Retrieved September 9, 2016 from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4857447/
Tu, D., Lee, J., Ozdere, T., Lee, T. J., & You, L. (2007, January ). Engineering Genetic Circuits: Foundations and Applications. Retrieved from http://people.duke.edu/~you/publications/Tu_etal_SyntheticBiology.pdf
Watstein, D. M., McNerney, M. P., & Styczynski, M. P. (2015). Precise metabolic engineering of carotenoid biosynthesis in Escherichia coli towards a low-cost biosensor. Metabolic Engineering,31, 171‚Äì180. doi:10.1016/j.ymben.2015.06.007
