<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_LambertGA_Attributions skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:LambertGA/Attributions</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="headerLinks"><A href="https://2016.igem.org/Team:LambertGA" class="dropbtn">Home</A><LI class="dropdown"><A href="https://2016.igem.org/Team:LambertGA/Team" class="dropbtn">Team</A><DIV class="dropdown-content"><A class="drplink" style="transition: color 0.5s ease-in-out;" href="https://2016.igem.org/Team:LambertGA/Team">Team</A><A class="drplink" style="transition: color 0.5s ease-in-out;" href="https://2016.igem.org/Team:LambertGA/Collaborations">Collaborations</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:LambertGA/Description" class="dropbtn">Project</A><DIV class="dropdown-content" style="transition: display 0.5s ease-in-out;"><A class="drplink" style="transition: color 0.5s ease-in-out;" href="https://2016.igem.org/Team:LambertGA/Description">Description</A><A class="drplink" style="transition: color 0.5s ease-in-out;" href="https://2016.igem.org/Team:LambertGA/Design">Design</A><A class="drplink" style="transition: color 0.5s ease-in-out;" href="https://2016.igem.org/Team:LambertGA/Experiments">Experiments</A><A class="drplink" style="transition: color 0.5s ease-in-out;" href="https://2016.igem.org/Team:LambertGA/Proof">Proof of Concept</A><A class="drplink" style="transition: color 0.5s ease-in-out;" href="https://2016.igem.org/Team:LambertGA/Results">Results</A><A class="drplink" style="transition: color 0.5s ease-in-out;" href="https://2016.igem.org/Team:LambertGA/Notebook">Notebook</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:LambertGA/Parts" class="dropbtn">Parts</A><DIV class="dropdown-content"><A class="drplink" style="transition: color 0.5s ease-in-out;" href="https://2016.igem.org/Team:LambertGA/Parts">Parts</A><A class="drplink" style="transition: color 0.5s ease-in-out;" href="https://2016.igem.org/Team:LambertGA/Basic_Part">Basic Parts</A><A class="drplink" style="transition: color 0.5s ease-in-out;" href="https://2016.igem.org/Team:LambertGA/Composite_Part">Composite Parts</A></DIV></LI><A href="https://2016.igem.org/Team:LambertGA/Attributions" class="dropbtn">Attributions</A><A href="https://2016.igem.org/Team:LambertGA/Safety" class="dropbtn">Safety</A><LI class="dropdown"><A href="https://2016.igem.org/Team:LambertGA/Human_Practices" class="dropbtn">Human Practices</A><DIV class="dropdown-content"><A class="drplink" style="transition: color 0.5s ease-in-out;" href="https://2016.igem.org/Team:LambertGA/Human_Practices">Human Practices</A><A class="drplink" style="transition: color 0.5s ease-in-out;" href="https://2016.igem.org/Team:LambertGA/HP/Silver">Silver</A><A class="drplink" style="transition: color 0.5s ease-in-out;" href="https://2016.igem.org/Team:LambertGA/HP/Gold">Gold</A><A class="drplink" style="transition: color 0.5s ease-in-out;" href="https://2016.igem.org/Team:LambertGA/Integrated_Practices">Integrated Practices</A><A class="drplink" style="transition: color 0.5s ease-in-out;" href="https://2016.igem.org/Team:LambertGA/Engagement">Engagement</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:LambertGA/Hardware" class="dropbtn">Awards</A><DIV class="dropdown-content"><A class="drplink" style="transition: color 0.5s ease-in-out;" href="https://2016.igem.org/Team:LambertGA/Hardware">Hardware</A><A class="drplink" style="transition: color 0.5s ease-in-out;" href="https://2016.igem.org/Team:LambertGA/Measurement">Measurement</A><A class="drplink" style="transition: color 0.5s ease-in-out;" href="https://2016.igem.org/Team:LambertGA/Model">Model</A></DIV></LI></DIV><DIV class="column full_size" style="width:75%; margin:auto;"><CENTER><H1 id="MainTitle"><B> Attributions and Acknowledgements </B></H1></CENTER><H3> Attributions </H3><H2>Research</H2><P><HH3><B>  Constructs: </B> Arjun Bhatt, Jack Kwon </HH3></P><P><HH3><B> Measurement Model: </B> Elynna Chang, Noora Chandasir, David Standeven, Jackson Harris </HH3></P><H2>Labwork </H2><P><HH3> Arjun Bhatt, Jack Kwon, Sara Cleland, Julia Leveille, Alyssa Franklin, Lauren Hong, Jackson Harris, Kevin Tao, David Standeven </HH3></P><P><HH3><B> Proof of Concept: </B> Janet Standeven </HH3></P><H2> Wiki </H2><P><HH3><B> Design: </B> Neha Balachandran </HH3></P><P><HH3><B> Design Support: </B> Sara Cleland, Jackson Harris, Arjun Bhatt </HH3></P><P><HH3><B> Information/Writing: </B> Arjun Bhatt, Jack Kwon, Natalie Shih, Nivi Minjur, Janet Standeven
</HH3></P><P><HH3><B> Pictures/Diagrams: </B> Arjun Bhatt

</HH3></P><H2> Design </H2><P><HH3><B> Spiritwear, Banner, Animations: </B> Elynna Chang  </HH3></P><P><HH3><B> Poster: </B> Alyssa Franklin, Lauren Hong, Julia Leveille, Noora Chandasir</HH3></P><P><HH3><B> Handouts: </B> Sara Cleland, Lubna Orberger </HH3></P><H2> Outreach </H2><P><HH3><B> Atlanta Science Festival: </B> Noor Sohal, Elynna Chang, Neha Balachandran </HH3></P><P><HH3><B> Building with Biology: </B> Nivi Minjur, Noora Chandasir, David Standeven, Natalie Shih </HH3></P><P><HH3><B> RESA: </B> Noor Sohal, Neha Balachandran </HH3></P><P><HH3><B> Next Generation Focus: </B> Neha Balachandran</HH3></P><P><HH3><B>CHD Walk: </B>Noora Chandasir</HH3></P><P><HH3><B>Sharon Science Day: </B> Sara Cleland</HH3></P><H2>Presentation</H2><P><HH3><B>Presenters: </B>Arjun Bhatt, Neha Balachandran, Elynna Chang</HH3></P><P><HH3><B>Script/Other Help: </B>Jack Kwon, Lauren Hong, Jackson Harris</HH3></P><H3> Acknowledgements </H3><H2> General Support </H2><P><HH3><B>Mark Styczynski: </B> Associate Professor at Georgia Institute of Technology </HH3></P><P><HH3><B>Monica McNerney: </B>Doctoral Student at Georgia Institute of Technology</HH3></P><P><HH3><B>Dan Watstein: </B>Doctoral Candidate at Georgia Institute of Technology </HH3></P><P><HH3><B>2015 Lambert iGEM Seniors: </B> Noor Sohal, Lily Ge, Hanna Minot, Lauren Pan, and Sarah Rupert </HH3></P><H2> Parts Support </H2><P><HH3><B>Snapgene: </B>Software for sequence analysis</HH3></P><P><HH3><B>IDT: </B>Synthesis of g blocks</HH3></P><H2> Funding/Materials Support </H2><P><HH3> Lambert High School</HH3></P><P><HH3> National Science Foundation Grant #1254382</HH3></P><P><HH3> Siemens</HH3></P><P><HH3> AGCO</HH3></P><P><HH3> Franklin Technologies</HH3></P><P><HH3> Gofundme</HH3></P><P><HH3> Johns Creek Chick fil A</HH3></P><P><HH3> Rosco: Roscolux </HH3></P><H2> Wiki Support </H2><P><HH3> Joel Joseprabu: Wiki Formatting</HH3></P><P><HH3> Balachandran Srinivasan: Wiki Formatting</HH3></P><H2> Outreach Support </H2><P><HH3> Next Generation Focus/ Grace Chapel Hill Church</HH3></P><P><HH3> Atlanta Science Festival</HH3></P><P><HH3> Atlanta Maker Faire </HH3></P><P><HH3> Sharon Elementary School </HH3></P><H2> Sources </H2><P>And, S. A. (2009, February 13). Sarita Ahlawat. ClpXP Degrades SsrA-tagged proteins in S.pneumoniae.Retrieved Summer, 2016, from http://jb.asm.org/content/191/8/2894.full 

Andersen , J.B. , Sternberg , C. , Poulsen , L.K. , Bjorn , S.P. , Givskov , M. , and Molin , S. ( 1998 ) New unstable variants of green fluorescent protein for studies of transient gene expression in bacteria . Appl Environ Microbiol 64 : 2240 – 2246 .

Baker, T. A., &amp; Sauer, R. T. (2011, June 27). ClpXP, an ATP-powered unfolding and protein-degradation machine. Retrieved Summer, 2016, from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3209554/ 

Bar-Nun, S., &amp; Glickman, M. H. (2012). Proteasomal AAA-ATPases: Structure and function. Biochimica et Biophysica Acta (BBA) - Molecular Cell Research, 1823(1), 67–82. doi:10.1016/j.bbamcr.2011.07.009. Retrieved Summer, 2016 from http://www.sciencedirect.com/science/article/pii/S0167488911001984

Bohn , C. , Binet , E. , and Bouloc , P. ( 2002 ) Screening for stabilization of proteins with a trans-translation signature in Escherichia coli selects for inactivation of the ClpXP protease . Mol Genet Genomics 266 : 827 –831 .

Burton , R.E. , Siddiqui , S.M. , Kim , Y.I. , Baker , T.A. , and Sauer , R.T. ( 2001 ) Effects of protein stability and structure on substrate processing by the ClpXP unfolding and degradation machine . EMBO J 20 : 3092 –3100 .

Ciechanover, A. (2005). Cell death and differentiation - abstract of article: Intracellular protein degradation: From a vague idea thru the lysosome and the ubiquitin-proteasome system and onto human diseases and drug targeting[ast]. Cell Death &amp; Differentiation, 12(9), 1178–1190. doi:10.1038/sj.cdd.4401692

Cooper, G. M. (2000). Protein degradation. Retrieved Summer, 2016 from http://www.ncbi.nlm.nih.gov/books/NBK9957/

Dhakar, L. (n.d.). Image Color Picker (Z. A., Ed.). Retrieved October 10, 2016, from http://www.colorcodepicker.com/

Farrell, C., Grossman, A., &amp; Sauer, R. (2005). Cytoplasmic degradation of ssrA-tagged proteins.Molecular microbiology., 57(6), 1750–61. Retrieved from http://www.ncbi.nlm.nih.gov/pubmed/16135238

Flynn , J.M. , Levchenko , I. , Seidel , M. , Wickner , S.H. , Sauer , R.T. , and Baker , T.A. ( 2001 ) Overlapping recognition determinants within the ssrA degradation tag allow modulation of proteolysis . Proc Natl Acad Sci USA 11 : 10584 – 10589.

Georgia Institute of Technology. (2015, September 1). “Bacterial litmus Test” provides inexpensive measurement of Micronutrients. Retrieved from GT News Center, http://www.news.gatech.edu/2015/09/01/bacterial-litmus-test-provides-inexpensive-measurement-micronutrients

Goldberg, A.L., A.S. Menon, S. Goff and D.T. Chin. 1987. The mechanism and regulation of the ATP-dependent protease La from Escherichia coli. Biochem. Soc. Trans. 15: 809-811. Retrieved October 1, 2016 from http://www.fao.org/wairdocs/ilri/x5550e/x5550e0d.htm

Hwang BJ, Woo KM, Goldberg AL, Chung CH. Protease Ti, a new ATP-dependent protease in Escherichia coli,contains protein-activated ATPase and proteolytic functions in distinct subunits. J Biol Chem. 1988;263:8727–8734.

Katayama-Fujimura Y, Gottesman S, Maurizi MR. A multiple-component, ATP-dependent protease from Escherichia coli. J Biol Chem. 1987;262:4477–4485.

Landry, B. P., &amp;Stöckel, J. (2013). Use of degradation tags to control protein levels in the Cyanobacterium Synechocystis sp. Strain PCC 6803. Applied and Environmental Microbiology,79(8), 2833–2835. doi:10.1128/AEM.03741-12

Léa, D. (2016). Les meilleures banques en ligne - Banque en ligne de l'Université Cefi. Retrieved October 18, 2016, from http://univ-cefi.fr/les-meilleures-banques-en-ligne.html

Lee C, Schwartz MP, Prakash S, Iwakura M, Matouschek A. ATP-Dependent Proteases Degrade Their Substrates by Processively Unraveling Them from the Degradation Signal.

McNerney, M. P., Watstein, D. M., &amp; Styczynski, M. P. (2015). Precision metabolic engineering: The design of responsive, selective, and controllable metabolic systems. Metabolic Engineering, 31, 123–131. doi:10.1016/j.ymben.2015.06.011

Minikel, E. V. (2013, June 11). Basics of protein degradation. Retrieved Summer, 2016, from http://www.cureffi.org/2013/07/11/basics-of-protein-degradation/ 

Mogk A, Schmidt R, Bukau B. The N-end rule pathway for regulated proteolysis: prokaryotic and eukaryotic strategies. Trends Cell Biol. 2007;17:165–172.

Purcell, O., Grierson, C. S., Bernardo, M. di, &amp; Savery, N. J. (2012). Temperature dependence of ssrA-tag mediated protein degradation. Journal of Biological Engineering, 6(1), . doi:10.1186/1754-1611-6-10

Purple color codes. (n.d.). Retrieved October 10, 2016, from http://www.rapidtables.com/web/color/purple-color.htm 

RGB Color Gradient Maker. (n.d.). Retrieved October 10, 2016, from http://www.perbang.dk/rgbgradient/ 

Schrader, E. K., Harstad, K. G., &amp; Matouschek, A. (n.d.). Targeting proteins for degradation. , 5(11), . Retrieved from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4228941/

Snider, J., Thibault, G., &amp; Houry, W. A. (2008). The AAA+ superfamily of functionally diverse proteins. , 9(4), . Retrieved Summer, 2016 from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2643927/

Tamura, K., Shimada, T., Ono, E., Tanaka, Y., Nagatani, A., Higashi, S., . . . Hara-Nishimura, I. (2003, September). Why green fluorescent fusion proteins have not been observed in the vacuoles of higher plants. The Plant Journal, 35(4), 545-555. doi:10.1046/j.1365-313X.2003.01822.x

Tanaka K. The proteasome: overview of structure and functions. Proc Jpn Acad Ser B Phys Biol Sci.2009;85:12–36.

Tao, L., &amp; Biswas, I. (2015). Degradation of SsrA-tagged proteins in streptococci. , 161(Pt 4),. Retrieved September 9, 2016 from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4857447/

Tu, D., Lee, J., Ozdere, T., Lee, T. J., &amp; You, L. (2007, January ). Engineering Genetic Circuits: Foundations and Applications. Retrieved from http://people.duke.edu/~you/publications/Tu_etal_SyntheticBiology.pdf

Watstein, D. M., McNerney, M. P., &amp; Styczynski, M. P. (2015). Precise metabolic engineering of carotenoid biosynthesis in Escherichia coli towards a low-cost biosensor. Metabolic Engineering,31, 171–180. doi:10.1016/j.ymben.2015.06.007
</P><CENTER><H1 id="MainTitle"><B> Sponsors </B></H1></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>