<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tianjin_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tianjin/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;!DOCTYPE html&gt;<TITLE>Team:Tianjin - 2018.igem.org</TITLE></P><DIV class="container-fluid"><DIV class="navbar-collapse navbar-right navbar-main-collapse collapse"><UL class="nav navbar-nav navbar-right" style="margin: 15px;"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
                    PROJECT
                    </A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Description">DESCRIPTION</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Design">DESIGN</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Experiments">EXPERIMENTS</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Notebook">NOTEBOOK</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Model">MODEL</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Demonstrate">DEMONSTRATE</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
                    PARTS
                    </A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Parts">PARTS OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Basic_Part">BASIC PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Composite_Part">COMPOSITE PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Part_Collection">PART COLLECTION</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Improve">IMPROVED PART</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Safety">
                    SAFETY              
                </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
                    HUMAN PRACTICES
                    </A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Human_Practices">INTEGRATED HP</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Human_Practices">COLLABORATION</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Public_Engagement">EDU&amp;PUBLIC ENGAGEMENT</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">              TEAM
                    </A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Team">MEMBERS</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://igem.org/2018_Judging_Form?team=Tianjin">
                    JUDGING FORM               
                </A></LI></DIV></DIV><DIV class="container "><DIV class="row"><DIV class="col-xs-12 titleBox"><DIV class="title title-big"><P>Measurment</P></DIV></DIV></DIV><DIV class="row"><DIV class="panel-group" id="accordion1" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default text-panel"><DIV class="pan-heading text-pan-heading panel-top" id="collapsehead"><DIV class="panel-title"><A href="#collapseOne" role="button" data-toggle="collapse" data-parent="#accordion1" style="text-decoration: none;">
                                How did we choose our reporter gene?
                            </A></DIV></DIV><DIV id="collapseOne" class="panel-collapse collapse" role="tabpanel" aria-labelledby="collapsehead" aria-expanded="false"><DIV class="panel-body"><DIV class="row"><DIV class="col-xs-12 text"><P>           To achieve the best measurement of our oscillating system, we chose two different kinds of reporter gene as our downstream reporter, fluorescent protein gene and luciferase gene,  which are both widely used in the biotechnological labs.
                                    </P></DIV><DIV class="col-xs-12"><DIV class="title title-normal"><P>Fluorescent protein</P></DIV></DIV><DIV class="col-xs-12 text"><P>
                                       In search for the part kit provided by the iGEM headquarter, we found several available fluorescent proteins. Under the instruction of our <A href="https://2018.igem.org/Team:Tianjin/Model">fluorescent protein judging model</A>, we finally chose mCherry and EYFP as our fluorescent protein reporter.
                                    </P></DIV><DIV class="col-xs-12 text"><TABLE class="table table-bordered"><THEAD style="background: #222!important;color: white;"><TR><TH colspan="12">Table 1. Chosen fluorescent proteins</TH></TR></THEAD><TBODY><TR><TD><P>Protein</P></TD><TD>λ<SUB>em</SUB>(nm)</TD><TD>λ<SUB>em</SUB>(nm)</TD><TD><P>part serial number</P></TD><TD><P>site</P></TD></TR><TR><TD><P>mCherry</P></TD><TD><P>587</P></TD><TD><P>610</P></TD><TD><P>Part:BBa_E2060</P></TD><TD><P>3-7L</P></TD></TR><TR><TD><P>EYFP</P></TD><TD><P>513</P></TD><TD><P>527</P></TD><TD><P>Part:BBa_E2030</P></TD><TD><P>3-7J</P></TD></TR></TBODY></TABLE></DIV><DIV class="col-lg-12 picture"><P>Figure 1. Mechanism of fluorescent proteins</P></DIV><DIV class="col-xs-12"><DIV class="title title-normal"><P>Luciferase</P></DIV></DIV><DIV class="col-xs-12 text"><P>
                                       Luciferase is a popular choice as a reporter for gene expression because functional enzyme is created immediately upon translation and the assay is rapid, reliable and easy to perform.<SUP><A href="#re1">[1,2]</A></SUP> The fluorescent output of luciferase gene is due to the chemical reaction between luciferase and corresponding substrate, which can be easily detected by the luminoskan microplate reader.
                                       We used firefly luciferase(Fluc) gene purchased from Promega and Nanoluc luciferase(Nluc)  gene submitted by Team Tuebingen 2015 as another two reporters for our KaiABC system.
                                    </P></DIV><DIV class="col-xs-12 text"><TABLE class="table table-bordered"><THEAD style="background: #222!important;color: white;"><TR><TH colspan="12">Table 2. Chosen luciferase</TH></TR></THEAD><TBODY><TR><TD width="113"><P>luciferase</P></TD><TD width="113">λ<SUB>em</SUB>(nm)</TD><TD width="174"><P>resource</P></TD><TD width="200"><P>Detail</P></TD></TR><TR><TD width="113"><P>Fluc</P></TD><TD width="113"><P>560</P></TD><TD width="174"><P>Bought from Promega</P></TD><TD width="200"><P>pGL4 Luciferase Reporter Vectors</P></TD></TR><TR><TD width="113"><P>Nluc</P></TD><TD width="113"><P>460</P></TD><TD width="174"><P>Part:BBa_K1680009</P></TD><TD width="200"><P>Kit site:7-3N</P></TD></TR></TBODY></TABLE></DIV><DIV class="col-xs-12 picture"><P>Figure 2. Mechanism of luciferase</P></DIV><DIV class="col-xs-12 picture"><P>Figure 3.The bioluminescent reaction catalyzed by Fluc and Nluc</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="panel-group" id="accordion2" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default text-panel"><DIV class="pan-heading text-pan-heading" id="collapsehead"><DIV class="panel-title"><A href="#collapseTwo" role="button" data-toggle="collapse" data-parent="#accordion2" style="text-decoration: none;">
                                How did we design our measurement plan?
                            </A></DIV></DIV><DIV id="collapseTwo" class="panel-collapse collapse" role="tabpanel" aria-labelledby="collapsehead" aria-expanded="false"><DIV class="panel-body"><DIV class="row"><DIV class="col-xs-12 text"><P>
                                       When reconstructed in <EM>Escherichia coli</EM>, KaiABC oscillator showed a periodicity of 24 hours<SUP><A href="#re3">[3]</A></SUP>. To best verify our system can function in <EM>Saccharomyces cerevisiae</EM> , we planned to measure our system using different reporter at <STRONG>3-hour intervals for 1 day, 2 days, 3 days respectively</STRONG><STRONG>.</STRONG>
                                       As for the measurement of fluorescent proteins, we not only used the common detecting instrument, fourescence spectrophotometer, but also make an attempt to record the fluorescent variation via the cutting-edge Leica DMi8 inverted microscope.
                                       Another thing worth mentioning is that, we bought all of the luciferase assay system from company Promega, among them there was a new kind of Nanoluc luciferase system----Nano-Glo<SUP>®</SUP> Live Cell Assay System, which allows experimenters to detect live cell luminescent signals without lytic process, suitable for the 3-day assay. Despite there is no reference indicating that this product, a commonly used assay system in mammalian cells, could be used in<EM> </EM><EM>Saccharomyces cerevisiae</EM>, we decided to take a try and have successfully proved its validity. Consequently, we chose Nluc solely to complete our long time luminescent detection .
                                    </P></DIV><DIV class="col-xs-12 text"><TABLE class="table table-bordered"><THEAD style="background: #222!important;color: white;"><TR><TH colspan="12">Table3.1 day’s measurement ----- Fluorescent protein detection</TH></TR></THEAD><TBODY><TR><TD rowspan="4" width="154"><P><STRONG>Experimental</STRONG></P><P><STRONG>Group</STRONG></P></TD><TD width="468"><P>pABaC +pCiRbS +gal1+mCherry</P></TD></TR><TR><TD width="468"><P>pABaC +pbCiRS +gal1+mCherry</P></TD></TR><TR><TD width="468"><P>pABaC +pCiRbS+gal1+EYFP</P></TD></TR><TR><TD width="468"><P>pABaC +pbCiRS +gal1+EYFP</P></TD></TR><TR><TD rowspan="6" width="154"><P><STRONG>Negative</STRONG></P><P><STRONG>Control Group</STRONG></P></TD><TD width="468"><P>pABaC +gal1+mCherry</P></TD></TR><TR><TD width="468"><P>pCiRbS +gal1+mCherry</P></TD></TR><TR><TD width="468"><P>pbCiRS +gal1+mCherry</P></TD></TR><TR><TD width="468"><P>pABaC +gal1+EYFP</P></TD></TR><TR><TD width="468"><P>pbCiRS +gal1+EYFP</P></TD></TR><TR><TD width="468"><P>pCiRbS +gal1+EYFP</P></TD></TR></TBODY></TABLE></DIV><DIV class="col-xs-12 picture"><P>Figure 4. Procedures for measuring mCherry/EYFP via fluorescent spectrophotometer</P></DIV><DIV class="col-xs-12 picture"><P>Figure 5. Procedures for measuring luciferase via luminaskan microplate reader</P></DIV><DIV class="col-xs-12 text"><P>
                                        Detailed methods are available in our<A href="https://2018.igem.org/Team:Tianjin/Experiments"> Experiment</A>.</P><TABLE class="table table-bordered"><THEAD style="background: #222!important;color: white;"><TR><TH colspan="12">Table 4.2 days’ measurement ----- Fluc&amp;Nluc detection in lytic cells</TH></TR></THEAD><TBODY><TR><TD width="154"><P><STRONG>Experimental</STRONG></P><P><STRONG>Group</STRONG></P></TD><TD width="468"><P>pABaC +pCiRbS +gal1+Fluc</P></TD></TR><TR><TD width="154"><P><STRONG> </STRONG></P></TD><TD width="468"><P>pABaC +pbCiRS +gal1+Fluc</P></TD></TR><TR><TD width="154"><P><STRONG> </STRONG></P></TD><TD width="468"><P>pABaC +pCiRbS +gal2+Nanoluc</P></TD></TR><TR><TD width="154"><P><STRONG> </STRONG></P></TD><TD width="468"><P>pABaC +pbCiRS +gal2+Nanoluc</P></TD></TR><TR><TD width="154"><P><STRONG>Negative</STRONG></P><P><STRONG>Control Group</STRONG></P></TD><TD width="468"><P>pABaC +gal1+Fluc</P></TD></TR><TR><TD width="154"><P><STRONG> </STRONG></P></TD><TD width="468"><P>pbCiRS +gal1+Fluc</P></TD></TR><TR><TD width="154"><P><STRONG> </STRONG></P></TD><TD width="468"><P>pCiRbS +gal1+Fluc</P></TD></TR><TR><TD width="154"><P><STRONG> </STRONG></P></TD><TD width="468"><P>pABaC +gal2+Nanoluc</P></TD></TR><TR><TD width="154"><P><STRONG> </STRONG></P></TD><TD width="468"><P>pbCiRS +gal2+Nanoluc</P></TD></TR><TR><TD width="154"><P><STRONG> </STRONG></P></TD><TD width="468"><P>pCiRbS +gal2+Nanoluc</P></TD></TR></TBODY></TABLE></DIV><DIV class="col-xs-12 text"><TABLE class="table table-bordered"><THEAD style="background: #222!important;color: white;"><TR><TH colspan="12">Table 5.3 days’ measurement ----- Nluc detection in live cells</TH></TR></THEAD><TBODY><TR><TD rowspan="4" width="154"><P><STRONG>Experimental</STRONG></P><P><STRONG>Group</STRONG></P></TD><TD width="468"><P>pABaC +pCiRbS +gal1+Nanoluc</P></TD></TR><TR><TD width="468"><P>pABaC +pbCiRS +gal1+Nanoluc</P></TD></TR><TR><TD width="468"><P>pABaC +pCiRbS +gal2+Nanoluc</P></TD></TR><TR><TD width="468"><P>pABaC +pbCiRS 1+gal2+Nanoluc</P></TD></TR><TR><TD rowspan="4" width="154"><P><STRONG>Control Group</STRONG></P></TD><TD width="468"><P>pABaC +gal2+Nanoluc</P></TD></TR><TR><TD width="468"><P>pbCiRS +gal2+Nanoluc</P></TD></TR><TR><TD width="468"><P>pCiRbS +gal2+Nanoluc</P></TD></TR><TR><TD width="468"><P>TDH3P+NanoLuc</P></TD></TR></TBODY></TABLE></DIV><DIV class="col-xs-12"><DIV class="title title-normal"><P>EYFP measurement using Leica DMi8</P></DIV></DIV><DIV class="col-xs-12 text"><P>
                                        Leica DMi8 is an excellent inverted microscope for life science research, which is utilized extensively in materials inspection and measurement. For live cell research, the DMi8 platform is a complete solution whether precisely follow the development of a single cell in a dish, screen through multiple assays, obtain single molecule resolution, or tease out behaviors of complex processes. 
                                        Combined with easy to use software, high resolution cameras, and brilliant LED illumination, these inverted research systems are flexible to see the expression of the fluorescent protein EYFP in <EM>Saccharomyces cerevisiae</EM>. Through high speed imaging, using the external fluorescent filter wheels, we obtained fast and accurate fluorescent images as following show. It tells that our bacteria that we constructed express EYFP successfully and continuously in oscillating system. Meanwhile, the fluorescence intensity detected was very bright, indicating that the expression of EYFP was at a high level.
                                    </P></DIV><DIV class="col-xs-12 picture"><P>Figure 6. pABaC +pCiRbS+gal1+EYFP photoed by Leica DMi8</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="panel-group" id="accordion3" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default text-panel"><DIV class="pan-heading text-pan-heading" id="collapsehead"><DIV class="panel-title"><A href="#collapseThree" role="button" data-toggle="collapse" data-parent="#accordion3" style="text-decoration: none;">
                                Measuement result
                            </A></DIV></DIV><DIV id="collapseThree" class="panel-collapse collapse" role="tabpanel" aria-labelledby="collapsehead" aria-expanded="false"><DIV class="panel-body"><DIV class="row"><DIV class="col-xs-12 picture"><P>Figure 7. Experimental result reported by pABaC+pbCiRS+gal1+Fluc</P></DIV><DIV class="col-xs-12 picture"><P>Figure 8. Experimental result reported by pABaC+pCiRbS+gal1+Fluc</P></DIV><DIV class="col-xs-12 picture"><P>Figure 9. Experimental result reported by pABaC+pbCiRS+gal1+Nluc(obtained by live cell assays)</P></DIV><DIV class="col-xs-12 picture"><P>Figure 10.Experimental result reported by pABaC+pCiRbS+gal1+Nluc(obtained by live cell assays)</P></DIV><DIV class="col-xs-12 picture"><P>Figure 11.Experimental result reported by pABaC+pbCiRS+gal2+Nluc(obtained by live cell assays)</P></DIV><DIV class="col-xs-12 picture"><P>Figure 12.Experimental result reported by pABaC+pCiRbS+gal2+Nluc(obtained by live cell assays)</P></DIV><DIV class="col-xs-12 text"><P>Some basic information can be gained from the above figures:</P><OL><LI>Compared with the corresponding control groups, all the experimental groups have showed apparent oscillatory signals.</LI><LI>In the measured 3500 min, the oscillations in 700-2000 min are extremely unstable and nearly absent, while the oscillations for the rest of the time are relatively stable.</LI><LI>The cycle is about 500min (about 8.3h).</LI></OL><P><STRONG><STRONG>Note: final measurement results and analysis should be accessed in <A href="https://2018.igem.org/Team:Tianjin/Demonstrate">Result</A> page.</STRONG></STRONG></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="panel-group" id="accordion4" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default text-panel"><DIV class="pan-heading text-pan-heading panel-bottom" id="collapsehead"><DIV class="panel-title"><A href="#collapseFour" role="button" data-toggle="collapse" data-parent="#accordion4" style="text-decoration: none;">
                                Measuement instrument
                            </A></DIV></DIV><DIV id="collapseFour" class="panel-collapse collapse panel-bottom" role="tabpanel" aria-labelledby="collapsehead" aria-expanded="false"><DIV class="panel-body"><DIV class="row"><DIV class="col-xs-12 text"><P>
                                       Fluorescent protein: 
                                       Hitachi F-2500 Flourescence Spectrophotometer;
                                       Leica DMi8 inverted microscope
                                       Luciferase : Thermoscientific Luminoskan Microplate Reader</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV><DIV class="reference"><H1>References</H1><P class="reftext" id="re1"><A>[1]Ow, D.W. et al. (1986) Transient and stable expression of the firefly luciferase gene in plant cells and trans-genic plants. Science 234, 856–9.</A></P><P class="reftext" id="re2"><A>[2]De Wet, J.R. et al. (1987) Firefly luciferase gene: Structure and expression in mammalian cells. Mol. Cell. Biol. 7, 725–37.</A></P><P class="reftext" id="re3"><A>[3]Chen, AH (Chen, Anna H.); Lubkowicz,D (Lubkowicz, David); Yeong, V (Yeong, Vivian); Chang, RL (Chang, Roger L.); Silver, PA(Silver, Pamela A.)  Transplantability of a circadian clock to a noncircadian organism. SCIENCE ADVANCES(2015) DOI: 10.1126/sciadv.1500358</A></P></DIV></DIV></DIV></DIV><DIV id="footer" style="background-color: #79acdd;text-align: center;"><DIV class="container"><DIV class="row"><DIV class="col-sm-3 footer-part"><P>
                         IGEMTIANJIN
                    </P></DIV><DIV class="col-sm-3 footer-part"><P>
                         IGEMTIANJIN2018
                    </P></DIV><DIV class="col-sm-3 footer-part"><P>
                         HEBO@TJU.EDU.CN
                    </P></DIV><DIV class="col-sm-3 footer-part"><P>
                         BEIYANG CAMPUS,TIANJIN UNIVERSITY,TIANJIN
                    </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>