<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tianjin_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tianjin/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;!DOCTYPE html&gt;<TITLE>Team:Tianjin - 2018.igem.org</TITLE></P><DIV class="container-fluid"><DIV class="navbar-collapse navbar-right navbar-main-collapse collapse"><UL class="nav navbar-nav navbar-right" style="margin: 15px;"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
                    PROJECT
                    </A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Description">BACKGROUND</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Design">DESIGN</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Experiments">EXPERIMENTS</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Notebook">NOTEBOOK</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Model">MODEL</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Measurement">MEASUREMENT</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Demonstrate">DEMONSTRATE</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
                    PARTS
                    </A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Parts">PARTS OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Basic_Part">BASIC PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Composite_Part">COMPOSITE PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Part_Collection">PART COLLECTION</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Improve">IMPROVED PART</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Safety">
                    SAFETY              
                </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
                    HUMAN PRACTICES
                    </A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Human_Practices">INTEGRATED</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Collaborations">COLLABORATION</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Public_Engagement">PUBLIC ENGAGEMENT</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
                    TEAM
                    </A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Team">MEMBERS</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Judging">
                    FOR JUDGES               
                </A></LI></DIV></DIV><DIV class="container "><DIV class="row"><DIV class="col-xs-12 titleBox"><DIV class="title title-big"><P>PROJECT DESIGN</P></DIV></DIV></DIV><DIV class="row"><DIV class="col-xs-12 text"><P>
               Circadian clocks, also know as circadian oscillators, are ubiquitous timing systems that induce rhythms of biological activities in synchrony with night and day. Circadian oscillators are post-translationally regulated and affect gene expression in autotrophic circadian cyanobacteria. Most work on the cyanobacterial circadian clock has been performed in <EM>Synechococcus elongatus</EM> PCC 7942 (<EM>S. elongatus</EM>). In <EM>S. elongatus</EM> 7942, timing is generated by a post-translational clock consisting of KaiA, KaiB, and KaiC proteins and a set of output signaling proteins, SasA, RpaA and CikA, which transduce this rhythm to control gene expression and chromosome topology <SUP><A href="#re1">1</A></SUP>.
                </P></DIV></DIV><DIV class="row"><DIV class="col-lg-12 text"><P>
                   The goal of our project is to reconstruct the KaiABC circadian clock system from prokaryotic cyanobacteria (<EM>S.</EM><EM> </EM><EM>elongatus</EM> 7942) in noncircadian eukaryotic <EM>Saccharomyces cerevisiae</EM> (<EM>S</EM><EM>.</EM><EM> cerevisiae</EM>). And we are aim to achieve the controllability of this circadian clock system in Saccharomyces cerevisiae through some practical methods, such as changing the molecular concentration ratio of the core protein KaiA, KaiB and KaiC, which can help us better understand and futher explore the KaiABC system. In addition to the reconstruction of KaiABC, we tried to regulate the metabolic activities of <EM>S</EM><EM>.</EM><EM> cerevisiae</EM> using this prokaryotic circadian clock system. What’s more, Our project got a lot of meaningful inspiration from our modeling work. According to the reaction mechanism, a series of rate equations can be obtained. And our modeling work has successfully influenced and corrected our understanding of the KaiABC rhythm system. Click <A href="https://2018.igem.org/Team:Tianjin/Model">here</A> for more imformation about Modeling.
                </P></DIV></DIV><DIV class="row"><DIV class="col-xs-12 text"><P>
                   Some previous research findings have pointed to a possible global regulatory mechanism of the clock through the systematic alteration of chromosome topology<SUP><A href="#re2">2</A></SUP>. Although the underlying process are not revealed yet, researchers have identified that <EM>S. elongatus</EM> PCC 7942 genes exhibit circadian fluctuations and that these changes are highly correlated with rhythmic changes in the superhelical density of the chromosome<SUP><A href="#re3">3</A></SUP>. Inspired by these studies, we expect that the <EM>S. cerevisiae</EM> chromasome topology will periodically oscillate with the KaiABC circadian clock system.
                </P></DIV></DIV><DIV class="row"><DIV class="col-lg-12 text"><P>
                   In addition to understanding and exploring basic biological science through construction and building, we also explored applications of this circadian clock system in <EM>S</EM><EM>.</EM><EM> cerevisiae</EM>. The novel application we envisioned was that <EM>S</EM><EM>.</EM><EM> cerevisiae</EM> can produce different products alternately under the periodic regulation of the KaiABC circadian clock.
                </P></DIV></DIV><DIV class="row"><DIV class="panel-group" id="accordion1" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default text-panel"><DIV class="pan-heading text-pan-heading panel-top" id="collapsehead"><DIV class="panel-title"><A href="#collapseOne" role="button" data-toggle="collapse" data-parent="#accordion1" style="text-decoration: none;">
                                1. Reconstruction of KaiABC circadian clock with different strategies
                            </A></DIV></DIV><DIV id="collapseOne" class="panel-collapse collapse" role="tabpanel" aria-labelledby="collapsehead" aria-expanded="false"><DIV class="panel-body"><DIV class="row"><DIV class="col-xs-12 text"><P>
                                       In order to enable the reconstitution system of the selected six proteins, KaiA KaiB KaiC RpaA CiKA and SasA, to successfully work in yeast cells, we used three different strategies to assemble the gene expression cassettes of these six proteins onto two plasmids. At the same time, in order to better activate the reporter gene circuits to express the oscillating signal, we explored the interaction of three different couples of proteins in the system with the control of the circadian clock system.
                                    </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="panel-group" id="accordion2" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default text-panel"><DIV class="pan-heading text-pan-heading" id="collapsehead"><DIV class="panel-title"><A href="#collapseTwo" role="button" data-toggle="collapse" data-parent="#accordion2" style="text-decoration: none;">
                                2. Design of gene circuits and output of oscillating signals
                            </A></DIV></DIV><DIV id="collapseTwo" class="panel-collapse collapse" role="tabpanel" aria-labelledby="collapsehead" aria-expanded="false"><DIV class="panel-body"><DIV class="row"><DIV class="col-xs-12 text"><P>
                                       In order to affect cellular physiology, a posttranslational circadian clock needs to be connected to transcriptional output. To do so, we built a synthetic oscillator circuit and introduced a yeast two-hybrid system. 
                                       Firstly, we utilized the synthetic oscillator circuit to control the expression of a selected reporter gene. In order to select a suitable reporter gene, we tested some of the parameters of several commonly used reporter genes by pre-experiment and scored these reporter genes by modeling.
                                    </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="panel-group" id="accordion3" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default text-panel"><DIV class="pan-heading text-pan-heading" id="collapsehead"><DIV class="panel-title"><A href="#collapseThree" role="button" data-toggle="collapse" data-parent="#accordion3" style="text-decoration: none;">
                                3. Regulation of introduced prokaryotic clock systems in eukaryotic cells
   </A></DIV></DIV><DIV id="collapseThree" class="panel-collapse collapse" role="tabpanel" aria-labelledby="collapsehead" aria-expanded="false"><DIV class="panel-body"><DIV class="row"><DIV class="col-xs-12 text"><P>
                                       Based on the information reviewed so far, changing the concentration of the protein or using protein mutants can change the oscillating signal. We control the concentration of expressed protein by using different promoters. And The ability of the promoter to initiate expression was determined by experimental assays.
                                    </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="panel-group" id="accordion4" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default text-panel"><DIV class="pan-heading text-pan-heading" id="collapsehead"><DIV class="panel-title"><A href="#collapseFour" role="button" data-toggle="collapse" data-parent="#accordion4" style="text-decoration: none;">
                                4. Prokaryotic clock system regulates eukaryotic yeast by periodically affecting the topology of eukaryotic chromatin
                            </A></DIV></DIV><DIV id="collapseFour" class="panel-collapse collapse" role="tabpanel" aria-labelledby="collapsehead" aria-expanded="false"><DIV class="panel-body"><DIV class="row"><DIV class="col-xs-12 text"><P>
                                       S. elongatus genes exhibit circadian fluctuations and that these changes are highly correlated with rhythmic changes in the superhelical density of the chromosome . Although the underlying process are not revealed yet, some previous research findings have pointed to a possible global regulatory mechanism of the clock through the systematic alteration of chromosome topology.
                                       Based on this background knowledge, we designed experiments to couple the regulation of the KaiABC rhythm oscillation system with the S. cerevisiae chromatin topology. It is possible to achieve the regulation of the transcription and expression of global endogenous genes by KaiABC rhythm system in yeast. We choosed chromatin remodeling complex for research.<SUP><A href="#re6">6</A></SUP></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="panel-group" id="accordion5" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default text-panel"><DIV class="pan-heading text-pan-heading panel-bottom" id="collapsehead"><DIV class="panel-title"><A href="#collapseFive" role="button" data-toggle="collapse" data-parent="#accordion5" style="text-decoration: none;">
                               5. Novel applications
                            </A></DIV></DIV><DIV id="collapseFive" class="panel-collapse collapse panel-bottom" role="tabpanel" aria-labelledby="collapsehead" aria-expanded="false"><DIV class="panel-body"><DIV class="row"><DIV class="col-xs-12 text"><P>
                                      
                                       Our Tik Tok S. cerevisiae, which has been introduced into the KaiABC biological clock system, can be used as a cell factory that alternates between day and night to produce different products, which contributes to a good work schedule.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV><DIV class="reference"><H1>References</H1><P class="reftext" id="re1"><A>[1]Tseng, R. et al. Structural basis of the day-night transition in a bacterial circadian clock. Science 355, 1174-1180, doi:10.1126/science.aag2516 (2017).</A></P><P class="reftext" id="re2"><A>[2]Shultzaberger, R. K., Boyd, J. S., Diamond, S., Greenspan, R. J. &amp; Golden, S. S. Giving Time Purpose: The Synechococcus elongatus Clock in a Broader Network Context. Annu Rev Genet 49, 485-505, doi:10.1146/annurev-genet-111212-133227 (2015).</A></P><P class="reftext" id="re3"><A>[3]Woelfle, M. A., Xu, Y., Qin, X. &amp; Johnson, C. H. Circadian rhythms of superhelical status of DNA in cyanobacteria. Proc Natl Acad Sci U S A 104, 18819-18824, doi:10.1073/pnas.0706069104 (2007).</A></P><P class="reftext" id="re4"><A>[4]Kageyama, H. et al. Cyanobacterial circadian pacemaker: Kai protein complex dynamics in the KaiC phosphorylation cycle in vitro. Mol Cell 23, 161-171, doi:10.1016/j.molcel.2006.05.039 (2006).</A></P><P class="reftext" id="re5"><A>[5]Terauchi, K. et al. ATPase activity of KaiC determines the basic timing for circadian clock of cyanobacteria. P Natl Acad Sci USA 104, 16377-16381, doi:10.1073/pnas.0706292104 (2007).</A></P><P class="reftext" id="re6"><A>[6]Narlikar, G. J., Fan, H. Y. &amp; Kingston, R. E. Cooperation between complexes that regulate chromatin structure and transcription. Cell 108, 475-487, doi:Doi 10.1016/S0092-8674(02)00654-2 (2002).</A></P><P class="reftext" id="re7"><A>[7]Rust, M. J., Golden, S. S. &amp; O'Shea, E. K. Light-driven changes in energy metabolism directly entrain the cyanobacterial circadian oscillator. Science 331, 220-223, doi:10.1126/science.1197243 (2011).</A></P></DIV></DIV></DIV></DIV><DIV id="footer" style="background-color: #79acdd;text-align: center;"><DIV class="container"><DIV class="row"><DIV class="col-sm-3 footer-part"><P>
                         IGEMTIANJIN
                    </P></DIV><DIV class="col-sm-3 footer-part"><P>
                         IGEMTIANJIN2018
                    </P></DIV><DIV class="col-sm-3 footer-part"><P>
                         HEBO@TJU.EDU.CN
                    </P></DIV><DIV class="col-sm-3 footer-part"><P>
                         BEIYANG CAMPUS,TIANJIN UNIVERSITY,TIANJIN
                    </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>