Team:Tianjin/Experiments
<!DOCTYPE html>
Team:Tianjin - 2018.igem.org
PROJECT
BACKGROUND
DESIGN
EXPERIMENTS
NOTEBOOK
MODEL
MEASUREMENT
DEMONSTRATE
PARTS
PARTS OVERVIEW
BASIC PARTS
COMPOSITE PARTS
PART COLLECTION
IMPROVED PART
SAFETY
HUMAN PRACTICES
INTEGRATED
COLLABORATION
PUBLIC ENGAGEMENT
TEAM
MEMBERS
ATTRIBUTIONS
FOR JUDGES
EXPERIMENTS
Reconstruction of the KaiABC system
Promoter selection
In the cyanobacterium Synechococcus elongatus(PCC 7942), three genes (kaiA, kaiB, and kaiC) are essential components of the circadian clock, whose concentration ratio is also important. According to previous studies,  the concentration ratio of these three core proteins, KaiA, KaiB and KaiC, is  1:1:4 (by weight).
Researchers has incubated KaiC with KaiA and KaiB in vitro at a ratio similar to that measured in vivo in the presence of 1 mM ATP. As a result, KaiC phosphorylation has robustly oscillated for at least three cycles without damping. But the amplitude of this in vitro was smaller than that observed in vivo under continuous light conditions.
More specifically, the change in concentration of these three kind of  proteins could exert a complex impact on the KaiABC circadian clock system, or in other words, the circadian oscillation of KaiC phosphorylation. For example, in cyanobacteria,  KaiC represses its own (kaiBC) expression, whereas KaiA enhances the expression of kaiBC. Therefore, KaiC and KaiA could be, respectively, regarded as negative and positive factors functioning in circadian feedback regulation of kaiBC expression.
Besides, sufficient and appropriate protein concentrations is necessary for Kai protein interactions and KaiC phosphorylation. To take an example, several researchers have already explored the effect of Kai protein concentration on the rhythm of KaiC phosphorylation. In experiment, the standard mixture (×1 solution) was consisted of 1.2 μM KaiA, 3.5 μM KaiB, and 3.5 μM KaiC. When the proteins were at 2.5× or 5× protein concentrations, the KaiC phosphorylation rhythms were nearly the same as those measured under standard conditions. While lowering the protein concentration to 1/10 of the standard mixture, the amplitude of the KaiC phosphorylation rhythm was reduced and the period of the rhythm was prolonged. What’s more, at 1/20 or 1/40 of the standard concentrations, KaiC would be remained dephosphorylated. In addition, in the case of a decrease of the relative concentrations of KaiA or KaiB to KaiC, the rhythm could change accordingly. While lowering either KaiA or KaiB concentrations to one-fourth or one-third of the standard mixture abolished cycling, the rhythm was still maintained in the presence of a two-thirds concentration of either KaiA or KaiB.  However, the period is extended by lowering KaiA concentrations to two-thirds, while lowering the KaiB concentration to two-thirds of the standard couldn’t alter the rhythm.
Owing to a series of reasons like the lack of a complete transcriptional/translational feedback loop(TTFL), the KaiABC system, reconstituted in yeast cells, can not express the correct protein concentrations as in cyanobacteria. Therefore, we intend to get a good command of protein concentrations by selecting promoters of different intensity to construct the corresponding expression cassettes,  thus the KaiABC system works better and more stably. Besides, we measure our promoter strength of six core genes convenient for the analysis of the final results.
Based on the research measuring the promoter strength of 14 constitutive promoters by GFP fluorescence intensity[4], we designed our experiment about the measurement of the promoters by constructing mCherry cassette with different promoters to measure the fluorescence intensity of mCherry, representing the promoter strength. Before our experiment, we selected three common promoters which are TDH3 promoter, PGK1 promoter and TEF1 promoter according the data of the paper as our promoters of six core genes. The detailed information of plasmids we constructed were shown on the table 1 below. We assembled our plasmids by yeast homologous recombination. Taking TEF1P-mCherry-TEF1Tcassette for example, we obtained fragments TEF1P, mCherry and TEF1Twith homologous arms by PCR and then we transformed three fragments and linearized vector pRS415 which had been digested by enzymes HindⅢ and NotΙ  into Saccharomyces cerevisiae to do yeast homologous recombination. Through the screening of nutrition label and the verification of PCR, we got the right strains and did further measurement.
Table 1  Structure of Recombinant Plasmids
Table 1 Plasmids constructed for the measurement of the promoter strength
plasmids
cassette
pTEmCTE
TEF1P-mCherry-TEF1T
pPGmCPG
PGK1P-mCherry-PGK1T
pTDmCAD
TDH3P-mCherry-ADH1T
Notes for table 1:TEF1P, TEF1T, PGK1P, PGK1T, TDH3P and ADH1T represent TEF1 promoter, TEF1 terminator, PGK1 promoter, PGK1 terminator, TDH3 promoter and ADH1 terminator.
In our experiment, we only measured the promoters we have selected before. Our results were as follows.
Figure 1  the promoter strength characterized by mCherry
What’s more, we constructed  three combinations with two kinds of promoter-gene combinations to character different forms of oscillation. As is shown in the Figure 1, we increased the relative concentration of KaiA protein greatly and decreased the relative concentration of KaiB protein slightly in KaiC-SasA and KaiC-CikA combinations, which shortened the period of the circadian oscillation . In our KaiB-KaiC combination, we increased the relative concentration of KaiA protein greatly.
Plasmids assembly
Our aim is to reconstruct the KaiABC circadian clock system of prokaryotic cyanobacteria in nonciracdian eukaryotic Saccharomyces cerevisiae. First of all, we hope to introduce the core proteins of cyanobacterial circadian clock, KaiA, KaiB and KaiC, into yeast to make them oscillate stably. To prevent KaiC from being trapped in phosphorylation state, we select three auxiliary proteins: SasA, CikA and RpaA. The promoter of prokaryotes can not be directly recognized by yeast, so we abandon the way of using the relevant promoters of the RpaA-mediated downstream reaction which are inherent in cyanobacterial, instead use the yeast two-hybrid system to characterize the KaiABC circadian clock system.
We selected three pairs of periodically binding proteins, KaiC-SasA, KaiC-CikA, and KaiB-KaiC, as the "prey" and "bait" of the yeast two-hybrid system, respectively, to construct fusion proteins with activation domain(AD) or DNA binding domain(BD) of Gal4 protein. Taking for example the KaiC-SasA couple, we assembled three gene expression cassettes of KaiA, KaiB, and AD-KaiC onto the pRS413 plasmid named pABaC, and assembled other three gene expression cassettes of CikA, RpaA and BD-SasA onto the pRS415 plasmid named pCiRbS. Other plasmids involved were pbCiRS (the recombinant pRS415 containing cassettes of BD-CikA, RpaA, SasA), pbCRCi (the recombinant pRS413 containing cassettes of BD-KaiC, RpaA, CikA), paBAS (the recombinant pRS415 containing cassettes of AD-KaiB, KaiA, SasA). When the two plasmids are successfully expressed in yeast, that is to say, when the KaiABC circadian clock system successfully operates, KaiA binds to the CII subunit of KaiC during the subjective daytime, stimulating the autokinase activity of the CII subunit, resulting in the phosphorylation of CII subunit. Residues Ser431 and Thr432 are phosphorylated in turn, during which SasA binds to phosphorylated KaiC,  allowing AD and BD to be spatially close enough to activate the promoters Gal1 promoter, Gal2 promoter that regulate downstream genes. When KaiC phosphorylation is complete, the KaiC protein undergoes loop stacking, which causes the binding site of KaiA to be blocked and the binding site of KaiB to be exposed. At the same time, KaiA detaches from KaiC, and KaiB binds to KaiC, stimulating its dephosphorylation. KaiB has a competitive relationship with SasA, which means that SasA falls off from KaiC, causing AD and BD to move away from each other and the related downstream genes not to be started.
In the experiment, we have used the restriction enzyme ligation method, Gibson assembly method and yeast homologous recombination method successively to construct the plasmids mentioned before. Since the experimental results showed that the first two methods were somewhat less efficient than the third one, we finally used the yeast homologous recombination method for plasmids assembly.  (Table 1) Taking KaiC-SasA couple as an example, we first used PCR to add corresponding homology arms to the ends of ten gene fragments including TEF1P (the promoter of TEF1, kaiA, TEF1T（the terminator of TEF1）, PGK1P (the promoter of PGK1), kaiB, PGK1T (the terminator of PGK1), TDH3P (the promoter of TDH3), AD, kaiC and ADH1T(the terminator of ADH1, then we introduced the ten gene fragments into the yeast together with the pRS413 plasmid cut by EcoRI and NotI to construct three gene expression cassettes of KaiA(TEF1P-kaiA-TEF1T), KaiB(PGK1P-kaiB-PGK1T), AD-KaiC(TDH3P-AD-kaiC-ADH1T. The right strains containing the recombinant plasmid were screened by nutrition labeling,  verified by PCR tag and then the recombinant plasmid was amplified in Escherichia Coli to obtain a large amount. Similarly, a sufficient number of recombinant pRS415 plasmids were obtained using the same way. Finally, we transformed the two kinds of plasmids into the final chassis cell,  Saccharomyces cerevisiae BY4741, to complete the construction of the KaiABC system.
Table 2  Structure of Recombinant Plasmids
pABaC
(pRS413)
KaiA   cassette
KaiB  cassette
AD-KaiC  cassette
TEF1P
kaiA
TEFIT
PGK1P
KaiB
PGK1T
TDH3P
AD
kaic
ADH1T
pCiRbS
(pRS415)
CikA   cassette
RpaA   cassette
BD-SasA   cassette
TEF1P
cikA
TEFIT
PGK1P
KaiB
PGK1T
TDH3P
BD
sasA
ADH1T
pbCiRS
(pRS415)
BD-CikA   cassette
RpaA   cassette
SasA   cassette
TEF1P
BD
cikA
TEF1T
PGK1P
rpaA
PGK1T
TDH3p
sasA
ADH1T
pbCRCi
(pRS413)
BD-KaiC   cassette
RpaA   cassette
CikA   cassette
TDH3P
BD
kaiC
ADH1T
PGK1P
rpaA
PGK1T
TEF1P
cikA
TEF1T
paBAS
(pRS415)
AD-KaiB   cassette
KaiA   cassette
SasA   cassette
TDH3P
AD
kaiB
ADH1T
PGK1P
kaiA
PGK1T
TEF1P
sasA
TEF1T
Figure 2 Map of the five plasmids
Construction of the reporter circuits
Reporter genes selection
To characterize the viability of our circadian clock, we required special report genes that functions sensitively in Saccharomyces cerevisiae. There are many report genes available but not everyone is suitable. We looked up papers and websites, screening massively. Our modeling group facilitated us to rate the fluorescent proteins fitted for our project most. They set up an Evaluation Model, which takes issues like lifetime, quantity yield(QY) and strokes into account to select the suitable fluorescent proteins among millions of alternatives.
Besides, Prof. Li recommended luciferase for us. The functional enzyme is created immediately upon translation and the assay is rapid, reliable and easy to perform with ATP, oxygen, and luciferin as substrates. It possesses a number of physical properties that make it an excellent reporter protein: small, monomeric enzyme, high thermal stability and so on.
we primarily chose Fluc, NanoLuc(BBa_K1680009), EYFP(BBa_E2030), mCherry(BBa_E2060), mOrange, and ECFP(BBa_I13602) as alternatives. Then verification experiments were operated by linking report genes above with constitutive promoter TDH3 promoter and measured the fluorescence intensity.
Taking TDH3P-mCherry-ADH1T cassette for example, we obtained fragments TDH3P,mCherry and ADH1T with homologous arms by PCR and then we transformed three fragments and linearized vector pRS413 which had been digested by enzymes BamHΙ and Not Ι into Saccharomyces cerevisiae to do yeast homologous recombination.Through the screening of nutrition label and the verification of PCR, we got the right strains for further measurement.
The detailed information were shown on the table2 below.
Table 3. Plasmids constructed for the measurement of various fluorescent proteins and luciferases
plasmids
cassette
pTDmCAD
TDH3P-mCherry-ADH1T
pTDmOAD
TDH3P-mOrange-ADH1T
pTDEYAD
TDH3P-EYFP-ADH1T
pTDECAD
TDH3P-ECFP-ADH1T
pTDRAD
TDH3P-RFP-ADH1T
pTDFAD
TDH3P-Fluc-ADH1T
pTDNAD
TDH3P-Nanoluc-ADH1T
Part of our results is shown on the figure below.
Figure 3  fluorescence intensity of various fluorescence proteins
We excluded mOrange, ECFP and RFP and finally we chose Nanoluc, Fluc, EYFP and mCherry as our report genes to characterize our system. The reasons were that mOrange showed very low fluorescence intensity and it takes too long for RFP to degrade and we didn’t obtain right genes of ECFP. EYFP and mCherry were finally picked out, which showed higher fluorescence intensity.
Except for the selection of reporter genes, these cassettes play a role in other ways . For example, it was used as a positive control group in measuring. And we measured the degradation curve of mCherry and EYFP, which did a favor for our analysis of the results on the one hand and added new experimental characterization data to part_BBaE2030 and part_BBaE2050 on the other hand.
Y2H report circuits construction
Due to the Y2H system, the promoter needs to have less leakage expression and respond sensitively to the proteins combination. After thorough search, we eventually found a mutant Gal1 promoter( BBa_K2637059 ), which was designed to functionally reduce false positive conditions. Besides, we selected Gal2 promoter( BBa_K2637009 ), which works independently in order to improve the accuracy.
We were successfully cloning four kinds of plasmids carrying the genes of mCherry, EYFP, NanoLuc and Fluc respectively with the Gal1 promoter and the ADH1T terminator by harnessing the principles of yeast homologous recombination. And also other four kinds of plasmids with the Gal2 promoter resembled the above four types, to compare which promoter works better. Moreover, to avoid the appearance of false positive phenomena which are likely to happen in the yeast two-hybrid system, we constructed the plasmids in four genres, and every kind of plasmids contained two cassettes consisting of Gal1 promoter with varying fluorescent proteins and Gal2 promoter with different luciferases. Only when both reporter genes function normally can we ensure that the system succeeds. all circuits are constructed respectively on the plasmid pRS416 and the details can be found in the table4 below. 
Ultimately, fluorescence spectrophotometer and multilabel reader were performed on our detecting process to analyze the expression of the fluorescent proteins and luciferases of the plasmid we constructed in the Saccharomyces cerevisiae respectively.
Figure 4 Example map of the reporter plasmids
Table4 The structure of reporter plasmids
name
number
cassette
p1m
BBa_K2637036
Gal1p-mCherry-ADH1T
p1E
BBa_K2637037
Gal1p-EYFP-ADH1T
p1N
BBa_K2637038
Gal1p-NanoLuc-ADH1T
p1F
\
Gal1p-Fluc-ADH1T
p2m
BBa_K2637039
Gal2p-mCherry-ADH1T
p2E
BBa_K2637040
Gal2p-EYFP-ADH1T
p2N
BBa_K2637041
Gal2p-NanoLuc-ADH1T
p2F
\
Gal2p-Fluc-ADH1T
p1m2N
BBa_K2637042
Gal1p-mCherry-ADH1T- Gal2p-NanoLuc-CYC1
p1E2N
BBa_K2637043
Gal1p-EYFP-ADH1T- Gal2p-NanoLuc-CYC1
p1m2F
\
Gal1p-mCherry-ADH1T- Gal2p-Fluc-CYC1
p1E2F
\
Gal1p-EYFP-ADH1T- Gal2p-Fluc-CYC1
At last, we get a number of different strains respectively containing a pair of selected recombinant plasmids and one reporter plasmid. (Table 5)
Table 5  Final Experimental Strains
(the strains are named after the plasmids that they contain)
Experimental group
Control group
pABaC +pbCiRS +p1F
 
pABaC + p1F
pbCiRS +p1F
pABaC +pbCiRS +p2F
 
pABaC + p2F
pbCiRS +p2F
pABaC +pbCiRS +p1N
 
pABaC + p1N
pbCiRS +p1N
pABaC +pbCiRS +p2N
 
pABaC + p2N
pbCiRS +p2N
pABaC +pbCiRS +p1m
 
pABaC + p1m
pbCiRS +p1m
pABaC +pbCiRS +p2m
 
pABaC + p2m
pbCiRS +p2m
pABaC +pbCiRS +p1E
 
pABaC + p1E
pbCiRS +p1E
pABaC +pbCiRS +p2E
 
pABaC + p2E
pbCiRS +p2E
pABaC +pCiRbS +p1F
 
pABaC + p1F
pCiRbS +p1F
pABaC +pCiRbS +p2F
 
pABaC + p2F
pCiRbS +p2F
pABaC +pCiRbS +p1N
 
pABaC + p1N
pCiRbS +p1N
pABaC +pCiRbS +p2N
 
pABaC + p2N
pCiRbS +p2N
pABaC +pCiRbS +p1m
 
pABaC + p1m
pCiRbS +p1m
pABaC +pCiRbS +p2m
 
pABaC + p2m
pCiRbS +p2m
pABaC +pCiRbS +p1E
 
pABaC + p1E
pCiRbS +p1E
pABaC +pCiRbS +p2E
 
pABaC + p2E
pCiRbS +p2E
pbCRCi + paBAS + p1N
 
pbCRCi +p1N
paBAS + p1N
pbCRCi + paBAS + p1F
 
pbCRCi +p1F
paBAS + p1F
pbCRCi + paBAS + p1E
pbCRCi+ p1E
paBAS + p1E
pbCRCi + paBAS + p1m
pbCRCi +p1m
paBAS + p1m
pbCRCi + paBAS + p2N
pbCRCi +p2N
paBAS + p2N
pbCRCi + paBAS + p2F
pbCRCi +p2F
paBAS + p2F
Novel application
Our S. cerevisiae, which incorporates heterogeneous KaiABC circadian clock system from cyanobacterium Synechococcus elongatus, can be used to construct microbial consortium with two yeasts, making it possible to produce different products in subjective days and nights.
In our human practice , the survey inspired us the idea. We collected the data, sought help from experts, made market analysis to select the proper products. In detail, one of the S. cerevisiae combining KaiC with SasA at subjective dawn is able to initiate the expression of downstream target gene, which can translate into a kind of product with a refreshing effect, such as caffeine. In another S. cerevisiae, KaiC is bound to CikA or KaiB at  subjective dusk to generate the expression of another downstream gene, whose translational products can function as sleep-enhancing supplements or mosquito repellent, such as melatonin (we eliminated it later for unsatisfactory effect) and limonene.  In this way, the KaiABC circadian clock system can be used as a cell factory that alternately and periodically produces two different substances between day and night.
Our cell factory will serve as a platform for a variety of biotechnological applications, by varying the downstream target genes to achieve different functions, such as treatment of circadian rhythm disorders, automatic daily drug delivery, etc. To take an example, as we all know, intestinal microbes have a significant impact on human health such as immunity, emotions, and the body's own biological clock. Therefore, perhaps for the foreseeable future, we can introduce the KaiABC oscillation system or its evolution into a specific kind of intestinal microbes to synthesize and secrete certain chemical substances or biologically active molecules to achieve better regulation of the environment in the human body.
Figure 5  The schematic diagram of our cell factory
Based on this idea, we conducted experimental exploration of caffeine and limonene using the parts provided by iGEM. Caffeine is a xanthine alkaloid compound, a central nervous stimulant that temporarily dispels drowsiness and restores energy, is used clinically to treat neurasthenia and coma. It is also widely used as an additive for a variety of functional refreshing beverages. Limonene is a volatile monoterpenoid compound and produces a lemon-like odor with the effect of repelling mosquitoes. Besides, limonene could inhibits rat mammary gland and other Tumor development, d-limonene has also been used clinically to dissolve cholesterol-containing gallstones. Owing to the benefits mentioned above, inhaling limonene may also have positive influence on our health, with the potentials to prevent cancer and dissolve gallstones. We originally hoped to construct a caffeine-limonenoe cell factory. Unfortunately, the caffeine gene provided in parts showed unclear problems, deterring us from making this idea into reality.
We got the three genes controlling caffeine production (BBa_K801070, BBa_K801071 and BBa_K801072) from the work of Technical University of Munich in 2012 . Unfortunately, the caffeine genes provided in kits showed unclear problems, deterring us from making this idea into reality. By PCR we found it seems not to contain the right sequence. As we can see in the gel photo below, three 1200bp-long sequences are shorter than supposed.
However, we have successfully assembled gene expression cassettes of limonene synthase onto the pRS416 plasmid named pGLA, and transformed pGLA into the S. cerevisiae we have constructed with the KaiABC system to form the limonene cell factory. The newly-built cell factory working with the KaiABC circadian clock systemhas achieved to periodically produce limonene during subjective nights only to help repel mosquitoes. We have constructed a biobrick(BBa_K2637044) encoding limonene sythase expression cassette, which was verifed via SDS-PAGE to tiral its function. You can see the results from here .
Overall, our cell factory proves to be an novel application for Kai ABC circadian clock system and other similar oscillators. We also hope that our work could provide inspirations for both researchers and companies from food, pharmaceutical or other relevant industries. And we would be extremely honored if anyone could transform our design into reality completely in the future.
References
[1] Nakajima M, Imai K, et al. Reconstitution of Circadian Oscillation of Cyanobacterial KaiC Phosphorylation in Vitro. [J]Science, 2005,308(5720):414-415
[2] Kageyama H, Nishiwaki T ,et al. Cyanobacterial Circadian Pacemaker: Kai Protein Complex Dynamics in the KaiC Phosphorylation Cycle In Vitro. [J]Molecular Cell, 2006, 23(2):161-171
[3]Zwicker D, Lubensky DK, ten Wolde. PR Robust circadian clocks from coupled protein-modification and transcription–translation cycles. [J]Proceedings of the national Academy of Sciences of the United States of America , 2010, 107(52):22540-22545
[4] Jie Sun, Zengyi Shao, Hua Zhao, Nikhil Nair, et al. Cloning and Characterization of a Panel of Constitutive Promoters for Applications in Pathway Engineering in Saccharomyces cerevisiae. [J]Biotechnology and Bioengineering, 2012, 109: 2082–2092.
IGEMTIANJIN
IGEMTIANJIN2018
HEBO@TJU.EDU.CN
BEIYANG CAMPUS,TIANJIN UNIVERSITY,TIANJIN
