<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tianjin_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tianjin/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;!DOCTYPE html&gt;<TITLE>Team:Tianjin - 2018.igem.org</TITLE></P><DIV class="container-fluid"><DIV class="navbar-collapse navbar-right navbar-main-collapse collapse"><UL class="nav navbar-nav navbar-right" style="margin: 15px;"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
                    PROJECT
                    </A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Description">BACKGROUND</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Design">DESIGN</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Experiments">EXPERIMENTS</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Notebook">NOTEBOOK</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Model">MODEL</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Measurement">MEASUREMENT</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Demonstrate">DEMONSTRATE</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
                    PARTS
                    </A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Parts">PARTS OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Basic_Part">BASIC PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Composite_Part">COMPOSITE PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Part_Collection">PART COLLECTION</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Improve">IMPROVED PART</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Safety">
                    SAFETY              
                </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
                    HUMAN PRACTICES
                    </A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Human_Practices">INTEGRATED</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Collaborations">COLLABORATION</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Public_Engagement">PUBLIC ENGAGEMENT</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
                    TEAM
                    </A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Team">MEMBERS</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Judging">
                   FOR JUDGES               
                </A></LI></DIV></DIV><DIV class="container "><DIV class="row"><DIV class="col-xs-12 titleBox"><DIV class="title title-big"><P>BACKGROUND</P></DIV></DIV></DIV><DIV class="row"><DIV class="col-xs-12 "><DIV class="title title-normal"><P>The Circadian Clock in <EM>Synechococcus elongatus</EM></P></DIV></DIV><DIV class="col-xs-12 text"><P>
               Many organisms contain a biological rhythm system to adapt to the Earth's circadian rhythm. The biological rhythm system in the cyanobacteria is designed to allow cyanobacterial cells to predict the time of daylighting in order to prepare for light reaction in advance.
                </P></DIV><DIV class="col-xs-12 text"><P>
                    In<EM> Synechococcus elongatus</EM>, the oscillator is composed of only three proteins KaiA, KaiB, and KaiC, which together generate the circadian rhythm of <EM>KaiC</EM> phosphorylation at residues serine 431 (S431) and threonine 432 (T432) in the CII domain. KaiA promotes KaiC (auto)phosphorylation during the subjective day, whereas KaiB provides negative regulation to inhibit KaiA and promotes KaiC (auto)dephosphorylation during the subjective night.<SUP><A href="#re1">[1]</A></SUP>The 24-h KaiC phosphorylation pattern can be reconstituted <EM>in vitro</EM> by merely combining the three Kai proteins and ATP, suggesting that the oscillating system is a post-translational oscillation system. KaiA and KaiB are also involved in regulating two antagonistic clock-output proteins—SasA and CikA, which reciprocally control the final regulator of transcription, RpaA.
                </P></DIV><DIV class="col-xs-12 text"><P>
                   Molecular changes in the KaiABC circadian clock system: Stepwise binding of two KaiA dimers triggers KaiC autophosphorylation at Thr432 and Ser431. These phosphorylation events enable cooperative binding of fold-switched KaiB monomers to the KaiC-CI domain, forming the KaiBC complex. KaiBC provides a scaffold for the successive sequestration of KaiA in ternary KaiABC assemblies, concurring with a rearrangement of the KaiA PsR domains. KaiA sequestration promotes KaiC autodephosphorylation, resulting in the regeneration of free KaiC through release of KaiAB subcomplexes. <SUP><A href="#re2">[2]</A></SUP></P></DIV><DIV class="col-xs-12 text"><P>
                    Temporal information from the oscillator is transmitted to downstream genes via the histidine protein kinase SasA (Synechococcus adaptive sensor A), whose autophosphorylation is stimulated by interaction with KaiC. Phosphorylated SasA in turn transfers a phosphoryl group to RpaA (regulator of phycobilisome association A) , a transcription factor that directly regulates the expression of approximately 100 genes. Moreover, RpaA indirectly regulates the expression of nearly all genes in the genome.<SUP><A href="#re3">[3]</A></SUP>Disruption of sasA also results in severely damped gene expression rhythms. CikA acquires the ability to dephosphorylate RpaA by interacting with the KaiBC complex and acts on downstream genes via RpaA to complete the subjective night biochemical reaction.
                </P></DIV><DIV class="col-xs-12 picture"><P>Figure 1 | The mechanism of Circadian Clock in <EM>Synechococcus elongatus</EM></P></DIV><DIV class="col-xs-12 text"><P>
                    Our team hopes to build a cyanobacteria biological rhythm system in yeast. Since yeast is a eukaryote, prokaryotic promoters cannot be used directly, so we abandoned the strategy of verifying oscillations by controlling downstream genes by RpaA. Finally, we used the yeast two-hybrid system to reporter gene expression.We selected KaiC-SasA, KaiC-CikA, and KaiC-KaiB three pairs of proteins for the yeast two-hybrid system. During the subjectively day, KaiC binds to SasA, causing spatial proximity of AD and BD leading to the initiation of downstream reporter genes. The KaiC and KaiB, CikA and KaiBC complexes are combined on subjective night, and downstream reporter genes are activated by the yeast two-hybrid system respectively.
                </P></DIV></DIV><DIV class="row"><DIV class="col-xs-12 "><DIV class="title title-normal"><P>Yeast Two-Hybrid System</P></DIV></DIV></DIV><DIV class="row"><DIV class="col-xs-12 "><DIV class="title title-small"><P>1.What is Yeast Two-Hybrid System?</P></DIV></DIV><DIV class="col-xs-12 text"><P>
                  Yeast two-hybrid system was put forward by Fields in 1980 according to his research about the transcription factor Gal4.  Gal4 contains two separated but essential domain which is called AD (activation domain) and BD (binding domain). BD is located at 1-147 amino acid at N terminal of Gal4, which can recognize and bind to the upstream activation sequences (UAS) of effector genes of Gal4 and AD is located at 768-881 amino acid at C terminal of Gal4, which can act on some transcription factors such as SAGA ( Spt-Ada-Gcn5-Acetyltransferase complex)、TBP (TATA-box binding protein)、TFIIB (Transcription Factor IIB), thus promoting the transcription of downstream genes. Gal1 promoter and Gal2 promoter, which can be controlled by Gal4, are chosen as the promoters of downstream genes. <SUP><A href="#re4">[4-7]</A></SUP></P></DIV><DIV class="col-xs-12 picture"><P>Figure 2 | transcription factors recruited by AD</P></DIV><DIV class="col-xs-12 text"><P>
                    Except for Gal4 AD and Gal4 BD, there are other types of yeast two-hybrid systems such as LexA BD and VP16 AD (LexA-VP16 system). By looking up related paper such as some applications about yeast two-hybrid system[11-13], we found that Gal4 system is the most common yeast two-hybrid system and it is more suitable for our project. For example, a study found that combinations between KaiC and SasA can’t be characterized by LexA-VP16 system.
                </P></DIV><DIV class="col-xs-12"><DIV class="title title-small"><P>2.How does Yeast Two-Hybrid System Work?</P></DIV></DIV><DIV class="col-xs-12 text">
                This system is often used to explore interactions between proteins and proteins, proteins and RNA, proteins and organic small molecule ligand. Usually, two proteins to be studied are fused with AD and BD respectively. The protein fused with BD is called “bait” and the protein fused with AD is called “prey”. When bait and prey combine with each other, it will make AD and BD spatially close enough. Thus the downstream genes will express. On the contrary, when bait and prey can’t combine, the downstream genes won’t express. <SUP><A href="#re4">[4]</A></SUP></DIV><DIV class="col-lg-6 col-xs-12 picture"><P>Figure 3 | schematic diagram of yeast two-hybrid system</P></DIV><DIV class="col-xs-12"><DIV class="title title-small"><P>3.Interactions between <EM>GAL</EM> genes and Gal proteins.</P></DIV></DIV><DIV class="col-xs-12 text"><P>
                    All the galactose structural genes (<EM>GAL</EM>1, <EM>GAL</EM>10, <EM>GAL</EM>7, <EM>GAL</EM>2) are coordinately regulated at the level of transcription in response to galactose by Gal4, Gal80, and Gal3. In the presence of galactose, Gal3 sequesters the transcriptional repressor Gal80p in the cytoplasm, thereby relieving inhibition of Gal4 and resulting in <EM>GAL </EM> gene expression. In the absence of galactose, Gal80 remains bound as a dimer to Gal4, preventing Gal4 from recruiting other factors of the Pol II transcription machinery. When cells are grown on glucose, <EM>GAL</EM>1 and <EM>GAL</EM>2 is negatively regulated by catabolite repression at both the levels of transcription and protein degradation. Whatever the carbon source is, the Gal4 transcriptional activator is bound as a dimer to UAS(upstream activation sites) found in the promoters of the<EM> GAL </EM> genes. In general, Gal4 and Gal80 can act on Gal1 promoter and Gal2 promoter directly while Gal3 only can act them through Gal80. Besides, Mig1 and Sip1 can downregulate the expression of <EM>GAL</EM>2 and <EM>GAL</EM>1 respectively. <SUP><A href="#re11">[11-23]</A></SUP></P></DIV><DIV class="col-xs-12 picture"><P>Figure 4 | interactions between Gal factors</P></DIV><DIV class="col-xs-12"><DIV class="title title-small"><P>4.Some Experimental Details about Yeast Two-Hybrid System</P></DIV></DIV><DIV class="col-xs-12 text"><P>
                    We choose three periodically combined proteins to construct three sets of combinations. They are KaiC-SasA、KaiC-CikA、KaiC-KaiB respectively. During the subjective day, KaiC will combine with SasA. While during the subjective night, KaiC will combine with KaiB and CikA. Therefore different combinations will express their downstream genes at different times. <SUP><A href="#re25">[25]</A></SUP></P></DIV><DIV class="col-xs-12 text picture"><TABLE class="table table-bordered"><THEAD style="background: #222!important;color: white;"><TR><TH>combination</TH><TH>bait</TH><TH>prey</TH></TR></THEAD><TBODY><TR><TD>KaiC-SasA</TD><TD>SasA</TD><TD>KaiC</TD></TR><TR><TD>KaiC-CikA</TD><TD>CikA</TD><TD>KaiC</TD></TR><TR><TD>KaiC-KaiB</TD><TD>KaiC</TD><TD>KaiB</TD></TR></TBODY></TABLE><P>Table 1 | bait and prey of three combinations</P></DIV><DIV class="col-xs-12 picture"><P>Figure 5 | KaiABC system with yeast two-hybrid system </P></DIV><DIV class="col-xs-12 text"><P>
                    Thanks to the instruction books of <A href="http://www.clontech.com">Clontech</A>, we know many experimental details about yeast two-hybrid system. First of all, we know that the false positive problem is the biggest problems of yeast two-hybrid system to some degree.To decrease the influence of that problem, we take many measures. For example, we set up negative control groups which only includes AD or BD and we choose the mutated Gal1 promoter which is synthesized by GeneScript according to the sequence of <A href="http://parts.igem.org/Part:BBa_K801004">part:BBa_K801004</A>. Since the false negative problem isn’t a big problem, we ignore it and don’t set up the positive control group including two proteins which can combine surely. Besides, we know that the strain to be used for yeast two-hybrid experiment should be knocked out <EM>GAL</EM>4 and <EM>GAL</EM>80 genes. Based on these information above, we knocked out <EM>GAL</EM>4 and <EM>GAL</EM>80 genes of our experimental strains through CRISPR-Cas9. The strains without<EM> GAL</EM>4 and <EM>GAL</EM>80 genes are named after <EM>d-two</EM>.
                </P></DIV></DIV><DIV class="row"><DIV class="col-xs-12"><DIV class="title title-normal"><P>The chromatin remodeling</P></DIV></DIV><DIV class="col-xs-12 text"><P>
                      Chromatin remodeling refers to that the molecular state of chromatin packaging, the histones in nucleosomes, and the corresponding DNA molecules will transform in the process of replication and recombination of gene expression.
                 </P></DIV><DIV class="col-xs-12 text"><P>
                       This remodeling is mainly through two approaches. First, this could be attained by covalent histone modification of specific enzymes, just like histone acetyltransferase (HATs), deacetylase, methyltransferase, and kinase. Specifically, it refers to the addition or removal of various chemical elements on histones under the catalysis of a specific protein complex called a histone modification complex. These enzymatic modifications include acetylation, methylation, phosphorylation, and ubiquitination occurring primarily at the N-terminal tail of histones. Furthermore, these modifications exert effect on the binding affinity between histones and DNA, thereby loosening or tightening the concentrated DNA surrounding the histones. For example, methylation of specific lysine residues in H3 and H4 leads to DNA being further combined with histones which deters transcription factors from binding to genes inhibiting the expression of DNA. In contrast, histone acetylation relaxes chromatin and exposes DNA to bind to transcription factors, giving rise to increasing gene expression.<SUP><A href="#re26">[26][27][28][29]</A></SUP></P></DIV><DIV class="col-xs-12 text"><P>
                       This remodeling is mainly through two approaches. First, this could be attained by covalent histone modification of specific enzymes, just like histone acetyltransferase (HATs), deacetylase, methyltransferase, and kinase. Specifically, it refers to the addition or removal of various chemical elements on histones under the catalysis of a specific protein complex called a histone modification complex. These enzymatic modifications include acetylation, methylation, phosphorylation, and ubiquitination occurring primarily at the N-terminal tail of histones. Furthermore, these modifications exert effect on the binding affinity between histones and DNA, thereby loosening or tightening the concentrated DNA surrounding the histones. For example, methylation of specific lysine residues in H3 and H4 leads to DNA being further combined with histones which deters transcription factors from binding to genes inhibiting the expression of DNA. In contrast, histone acetylation relaxes chromatin and exposes DNA to bind to transcription factors, giving rise to increasing gene expression.<SUP><A href="#re26">[26][27][28][29]</A></SUP></P></DIV><DIV class="col-xs-12 text"><P>
                    Second, ATP-dependent chromatin remodeling complex could move, eject or reconstruct nucleosomes to achieve the aim of remodeling. These protein complexes share a common ATPase domain, which can relocate the position of the nucleosome on the DNA by utilizing the energy of ATP hydrolysis to keep the histone away from DNA or promote the exchange of histone variants, thereby producing nucleosome free the DNA region which will activate the expression of DNA. In addition, some remodeling complexes have DNA translocation activity and can perform specific remodeling tasks. Currently we konw that there are at least five chromatin remodeling families in eukaryotes: <EM>SWI</EM> / <EM>SNF</EM>, <EM>ISWI</EM>, <EM>NuRD</EM> / <EM>Mi-2</EM> / <EM>CHD</EM>, <EM>INO80</EM> and<EM> SWR1</EM>. Although all remodeling complexes share a common ATPase domain, their function is based on several specific biological processes, just like DNA repair, apoptosis, and so on. This is due to the fact that each remodeling complex has a unique protein domain in its catalytic ATPase region and also has different recruitment subunits.<SUP><A href="#re30">[30][31]</A></SUP></P></DIV><DIV class="col-xs-12 picture"><P>Figure 6 | The mechanism of chromatin remodeling</P></DIV><DIV class="col-xs-12 text"><P>
                    Although there still remains some puzzles of the mechanism of the chromatin remodeling, a chunk of researches embodied that these two approaches indeed play a significant role in chromatin remodeling and altering the chromosome topology.
                </P></DIV><DIV class="col-xs-12 text"><P>
                    Therefore, in our project, we selected two types of chromatin remodeling families: <EM>SWI </EM>/ <EM>SNF</EM> and <EM>ISWI</EM> to substitute the reporter as the downstream proteins, since these two families have been well studied, especially in the yeast model.
                </P></DIV><DIV class="col-xs-12 text"><P>
                    Our Saccharomyces cerevisiae incorporated the heterogeneous KaiABC circadian clock from <EM>cynaobacterium Synechococcus elongatus</EM>. Once KaiC combined with SasA, the downstream genes initiate expressing by utilizing the yeast double- hybrid system. And due to the fact that the tightness of these two proteins’ combination will change over time, the amount of genes’ expression will alter accordingly. Since we substituted the genes of <EM>SWI</EM> /<EM> SNF</EM> and <EM>ISWI</EM> for the downstream genes, we could eventually measure the growth curve and observe the shapes of cells which have been modified by use of the Flow cytometry to find out if these modifications had periodic effect on the <EM>Saccharomyces cerevisiae</EM>.
                </P></DIV><DIV class="col-xs-12 text"><P>
                    Unfortunately, due to the limited time and punishing experiments, there leaves us no time and no energy to complete our whole expected experimental tasks, especially the experiments involving the part of chromatin remodeling, for which we pin our hope on the researchers in related fields or who interests in this to continue to study in depth and ultimately make the conclusion one day.  
                </P></DIV></DIV><DIV class="row"><DIV><DIV class="reference"><H1>References</H1><P class="reftext" id="re1"><A>[1] Roger Tseng, Nicolette F. Goularte, Archana Chavan, Jansen Luu, Susan E. Cohen, Yong-Gang Chang, Joel Heisler, Sheng Li, Alicia K. Michael, Sarvind Tripathi, Susan S. Golden, Andy LiWang, Carrie L. Partch, Structural basis of the day-night transition in a bacterial circadian clock. Science, 1174-1180 (2017).</A></P><P class="reftext" id="re2"><A>[2] Joost Snijder, Jan M. Schuller, Anika Wiegard, Philip Lössl, Nicolas Schmelling, Ilka M. Axmann, Jürgen M. Plitzko, Friedrich Förster, Albert J. R. Heck, Structures of the cyanobacterial circadian oscillator frozen in a fully assembled state. Science, 1181-1184 (2017).</A></P><P class="reftext" id="re3"><A>[3] Joseph S.Markson, Joseph R.Piechura, Anna M.Puszynska, Erin K.O’Shea, Circadian Control of Global Gene Expression by the Cyanobacterial Master Regulator RpaA. Cell, 1396-1408 (2013).</A></P><P class="reftext" id="re4"><A>[4]Jun O. Liu,et al.Everything you need to know about the yeast two-hybrid system.DOI：10.1038/788</A></P><P class="reftext" id="re5"><A>[5]Bhaumik S R, Raha T, Aiello D P, et al. In vivo target of a transcriptional activator revealed by fluorescence resonance energy transfer.[J]. DOI:10.1101/gad.1148404</A></P><P class="reftext" id="re6"><A>[6]Melcher K, Johnston S A. GAL4 interacts with TATA-binding protein and coactivators.[J]. DOI:10.1128/MCB.15.5.2839</A></P><P class="reftext" id="re7"><A>[7]Wu Y, Reece R J, Ptashne M. Quantitation of putative activator-target affinities predicts transcriptional activating potentials.DOI:10.1002/j.1460-2075.1996.tb00769.x</A></P><P class="reftext" id="re8"><A>[8]Hideo Iwasaki,* Stanly B. Williams,et al.A KaiC-Interacting Sensory Histidine Kinase, SasA, Necessary to Sustain Robust Circadian Oscillation in Cyanobacteria.Volume 101, Issue 2</A></P><P class="reftext" id="re9"><A>[9]Anang, Saumya，Subramani, Chandru,et al. Identification of critical residues in Hepatitis E virus macro domain involved in its interaction with viral methyltransferase and ORF3 proteins .DOI：10.1038/srep25133</A></P><P class="reftext" id="re10"><A>[10]Xiangui Lin &amp; Xi Huang ,et al.Identification of JAZ-interacting MYC transcription factors involved in latex drainage in Hevea brasiliensis.DOI：10.1038/s41598-018-19206-3</A></P><P class="reftext" id="re11"><A>[11]James R.Broach,et al.Galactose regulation in Saccharomyces cerevisiae. The enzymes encoded by the GAL7, 10, 1 cluster are co-ordinately controlled and separately translated.DOI：10.1016/0022-2836(79)90300-0</A></P><P class="reftext" id="re12"><A>[12]Kristy M. Hawkins and Christina D. Smolke 1,et al.The Regulatory Roles of the Galactose Permease and Kinase in the Induction Response of the GAL Network in Saccharomyces cerevisiae.DOI：10.1074/jbc.M512317200</A></P><P class="reftext" id="re13"><A>[13]M Johnston, J S Flick, T Pexton,et al.Multiple Mechanisms Provide Rapid and Stringent Glucose Repression of GAL Gene Expression in Saccharomyces cerevisiae.DOI：10.1128/MCB.14.6.3834</A></P><P class="reftext" id="re14"><A>[14]Elena Frolova  John Majors,et al.Binding of the glucose-dependent Mig1p repressor to the GAL1 and GAL4 promoters in vivo: regulationby glucose and chromatin structure.DOI：10.1093/nar/27.5.1350</A></P><P class="reftext" id="re15"><A>[15]https://thebiogrid.org/</A></P><P class="reftext" id="re16"><A>[16]DE ROBICHON-SZULMAJSTER H,et al. Induction of enzymes of the galactose pathway in mutants of Saccharomyces cerevisiae. Science 127(3288):28-9</A></P><P class="reftext" id="re17"><A>[17]Platt A and Reece RJ,et al. The yeast galactose genetic switch is mediated by the formation of a Gal4p-Gal80p-Gal3p complex. EMBO J 17(14):4086-91</A></P><P class="reftext" id="re18"><A>[18]Lohr D, et al. Transcriptional regulation in the yeast GAL gene family: a complex genetic network. FASEB J 9(9):777-87</A></P><P class="reftext" id="re19"><A>[19]Peng G and Hopper JE,et al. Gene activation by interaction of an inhibitor with a cytoplasmic signaling protein. Proc Natl Acad Sci U S A 99(13):8548-53</A></P><P class="reftext" id="re20"><A>[20]Mylin LM, et al. SIP1 is a catabolite repression-specific negative regulator of GAL gene expression. Genetics 137(3):689-700</A></P><P class="reftext" id="re21"><A>[21]Horak J and Wolf DH,et al. Catabolite inactivation of the galactose transporter in the yeast Saccharomyces cerevisiae: ubiquitination, endocytosis, and degradation in the vacuole. J Bacteriol 179(5):1541-9</A></P><P class="reftext" id="re22"><A>[22]Horak J and Wolf DH,et al. Glucose-induced monoubiquitination of the Saccharomyces cerevisiae galactose transporter is sufficient to signal its internalization. J Bacteriol 183(10):3083-8</A></P><P class="reftext" id="re23"><A>[23]Horak J and Wolf DH,et al. The ubiquitin ligase SCF(Grr1) is required for Gal2p degradation in the yeast Saccharomyces cerevisiae. Biochem Biophys Res Commun 335(4):1185-90</A></P><P class="reftext" id="re24"><A>[24]https://www.yeastgenome.org/locus/S000004071</A></P><P class="reftext" id="re25"><A>[25]Roger Tseng, Nicolette F. Goularte,et al. Structural basis of the day-night transition in a bacterial circadian clock.Science, 1174-1180(2017)</A></P><P class="reftext" id="re26"><A>[26]Wang GG, Allis CD, Chi P (September 2007). &quot;Chromatin remodeling and cancer, Part I: Covalent histone modifications&quot;. Trends in Molecular Medicine. 13 (9): 363–72. (2007)</A></P><P class="reftext" id="re27"><A>[27]Barski A, Cuddapah S, Cui K, Roh TY, Schones DE, Wang Z, Wei G, Chepelev I, Zhao K (May 2007). &quot;High-resolution profiling of histone methylations in the human genome&quot;. Cell. 129 (4): 823–37.(2007)</A></P><P class="reftext" id="re28"><A>[28]Steger DJ, Lefterova MI, Ying L, Stonestrom AJ, Schupp M, Zhuo D, Vakoc AL, Kim JE, Chen J, Lazar MA, Blobel GA, Vakoc CR (April 2008). &quot;DOT1L/KMT4 recruitment and H3K79 methylation are ubiquitously coupled with gene transcription in mammalian cells&quot;. Molecular and Cellular Biology. 28 (8): 2825–39.(2008)</A></P><P class="reftext" id="re29"><A>[29]Koch CM, Andrews RM, Flicek P, Dillon SC, Karaöz U, Clelland GK, Wilcox S, Beare DM, Fowler JC, Couttet P, James KD, Lefebvre GC, Bruce AW, Dovey OM, Ellis PD, Dhami P, Langford CF, Weng Z, Birney E, Carter NP, Vetrie D, Dunham I (June 2007). &quot;The landscape of histone modifications across 1% of the human genome in five human cell lines&quot;. Genome Research. 17 (6): 691–707.(2007)</A></P><P class="reftext" id="re30"><A>[30]Wang GG, Allis CD, Chi P (September 2007). &quot;Chromatin remodeling and cancer, Part II: ATP-dependent chromatin remodeling&quot;. Trends in Molecular Medicine. 13 (9): 373–80. (2007)</A></P><P class="reftext" id="re31"><A>[31]Saha A, Wittmeyer J, Cairns BR (June 2006). &quot;Chromatin remodelling: the industrial revolution of DNA around histones&quot;. Nature Reviews Molecular Cell Biology. 7 (6): 437–47.(2006)</A></P></DIV></DIV></DIV></DIV><DIV id="footer" style="background-color: #79acdd;text-align: center;"><DIV class="container"><DIV class="row"><DIV class="col-sm-3 footer-part"><P>
                         IGEMTIANJIN
                    </P></DIV><DIV class="col-sm-3 footer-part"><P>
                         IGEMTIANJIN2018
                    </P></DIV><DIV class="col-sm-3 footer-part"><P>
                         HEBO@TJU.EDU.CN
                    </P></DIV><DIV class="col-sm-3 footer-part"><P>
                         BEIYANG CAMPUS,TIANJIN UNIVERSITY,TIANJIN
                    </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>