<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tianjin_Improve skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tianjin/Improve</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;!DOCTYPE html&gt;<TITLE>Team:Tianjin - 2018.igem.org</TITLE></P><DIV class="container-fluid"><DIV class="navbar-collapse navbar-right navbar-main-collapse collapse"><UL class="nav navbar-nav navbar-right" style="margin: 15px;"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
                    PROJECT
                    </A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Description">BACKGROUND</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Design">DESIGN</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Experiments">EXPERIMENTS</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Notebook">NOTEBOOK</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Model">MODEL</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Measurement">MEASUREMENT</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Demonstrate">DEMONSTRATE</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
                    PARTS
                    </A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Parts">PARTS OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Basic_Part">BASIC PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Composite_Part">COMPOSITE PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Part_Collection">PART COLLECTION</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Improve">IMPROVED PART</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Safety">
                    SAFETY              
                </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
                    HUMAN PRACTICES
                    </A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Human_Practices">INTEGRATED</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Collaborations">COLLABORATION</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Public_Engagement">PUBLIC ENGAGEMENT</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
                    TEAM
                    </A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Team">MEMBERS</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2018.igem.org/Team:Tianjin/Judging">
                   FOR JUDGES               
                </A></LI></DIV></DIV><DIV class="container "><DIV class="row"><DIV class="row"><DIV class="col-xs-12 text"><P>
                    We improved a new part (<A href="http://parts.igem.org/Part:BBa_K2637010">BBa_K2637010</A>) from the wide-used luciferase, NanoLuc (<A href="http://parts.igem.org/Part:BBa_K1680009">BBa_K1680009</A>) to speed up its degradation and increase the sensitivity.
                </P></DIV><DIV class="col-xs-12 "><DIV class="title title-normal"><P>Description</P></DIV></DIV><DIV class="col-xs-12 text"><P>
                    Inspired by our time-course measurement for the KaiABC oscillatory system, we realized the importance of the sensitivity of reporter genes when used to characterize the variation in our engineering strains.
                    With the knowledge that the instability of proteins is associated with the existence of the so-called PEST regions, which control the ubiquitination of regulatory short-lived proteins<SUP><A href="#re1">[1]</A></SUP>, we decided to optimize our NanoLuc with the potentially degrading sequence PSET to shorten its intracellular lifetime.
                </P></DIV></DIV><DIV class="row"><DIV class="col-xs-12 "><DIV class="title title-normal"><P>Function of PEST</P></DIV></DIV><DIV class="col-xs-12 text"><P>
                    The covalent linkage of ubiquitin to lysine residues of substrate proteins is a common means used by eucaryotic cells to signal their degradation by the 26S proteasome, a multiprotease complex located in the cytoplasm and the nucleus<SUP><A href="#re2">[2]</A></SUP>. Decades ago it had been demonstrated that the PEST regions, enriched with Pro, Glu, Ser, and Thr, were identified to indeed control the ubiquitination of regulatory short-lived proteins<SUP><A href="#re1">[1]</A></SUP>
, such as transactivator Gcn4<SUP><A href="#re3">[3]</A></SUP> and G1 cyclins in yeast and mammalian cells. Phosphorylation of particular Ser or Thr residues in the PEST regions of these G1 cyclins specifies their recognition and processing by the ubiquitin-proteasome pathway.
                </P></DIV></DIV><DIV class="row"><DIV class="col-xs-12 "><DIV class="title title-normal"><P>Sequence and features</P></DIV></DIV><DIV class="col-xs-12 text"><P>
                    Sequence: 
                    AACTCTCACGGTTTCCCACCAGAAGTTGAAGAACAAGCTGCTGGTACTTTGCCAATGTCTTGTGCTCAAGAATCTGGTATGGACAGACACCCAGCTGCTTGTGCTTCTGCTAGAATCAACGTT
                    Features: shortener NanoLuc lifetime contributes to the higher sensitivity
                </P></DIV></DIV><DIV class="row"><DIV class="col-xs-12 "><DIV class="title title-normal"><P>Origin（Organism）</P></DIV></DIV><DIV class="col-xs-12 text"><P>
                    Sequence: 
                    PEST sequence can be found in variety of eukaryotic cells and the sequence we used was from the literature Engineered Luciferase Reporter from a Deep Sea Shrimp Utilizing a Novel Imidazopyrazinone Substrate<SUP><A href="#re4">[4]</A></SUP>.
                </P></DIV></DIV><DIV class="row"><DIV class="col-xs-12 "><DIV class="title title-normal"><P>Experimental characterization</P></DIV></DIV><DIV class="col-xs-12 text"><P>
                    We compare the degradation of the origin with improved NanoLuc in 3 hours.
                </P></DIV><DIV class="col-xs-12 text"><P>
                    You can download the protocol from the <A href="https://2018.igem.org/Team:Tianjin/Notebook">notebook</A>.
                    In this case, we have proved the newly constructed NanoLucPEST could work effectively in <EM>Saccharomyces cerevisiae</EM> BY4741. The sensitive reporter can be used in ample types of yeast after appropriate codon-optimization<SUP><A href="#re5">[5]</A></SUP></P></DIV></DIV><DIV class="row"><DIV><DIV class="reference"><H1>References</H1><P class="reftext" id="re1"><A>[1]Rogers, S., R. Wells, and M. Rechsteiner. 1986. Amino acid sequences common to rapidly degraded proteins: the PEST hypothesis. Science 234:364–368.</A></P><P class="reftext" id="re2"><A>[2]5. Ciechanover, A. 1994. The ubiquitin-proteasome proteolytic pathway. Cell 79:13–21.</A></P><P class="reftext" id="re3"><A>[3] Kornitzer, D., B. Raboy, R. G. Kulka, and G. R. Fink. 1994. Regulated degradation of the transcription factor Gcn4. EMBO J. 13:6021–6030.</A></P><P class="reftext" id="re4"><A>[4]Hall MP, Unch J, Binkowski BF, et al. 2012. Engineered luciferase reporter from a deep sea shrimp utilizing a novel imidazopyrazinone substrate. ACS Chem Biol 7: 1848–1857.</A></P><P class="reftext" id="re5"><A>[5]Anna E. Masser, Ganapathi Kandasamy, Jayasankar Mohanakrishnan Kaimal and Claes Andréasson.Luciferase NanoLuc as a reporter for gene expression and protein levels in Saccharomyces cerevisiae.YEAST 2016. DOI: 10.1002/yea.3155</A></P></DIV></DIV></DIV></DIV></DIV><DIV id="footer" style="background-color: #79acdd;text-align: center;"><DIV class="container"><DIV class="row"><DIV class="col-sm-3 footer-part"><P>
                         IGEMTIANJIN
                    </P></DIV><DIV class="col-sm-3 footer-part"><P>
                         IGEMTIANJIN2018
                    </P></DIV><DIV class="col-sm-3 footer-part"><P>
                         HEBO@TJU.EDU.CN
                    </P></DIV><DIV class="col-sm-3 footer-part"><P>
                         BEIYANG CAMPUS,TIANJIN UNIVERSITY,TIANJIN
                    </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>