<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NUS_Singapore-Sci_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NUS Singapore-Sci/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV class="main"><UL class="mainnav"><LI class="hassubs"><A href="#" style="text-decoration:none;color:white;">Project</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Overview">Project Overview</A></LI><LI class="subs hassubs"><A href="">Reporter System</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Reporter_Overview">Reporter Overview</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Demonstrate">Dual Reporter - Demonstration</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Improve">Dual Reporter - Improvement</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Reporter_GLB">GLB1 Reporter</A></LI></UL><LI class="subs hassubs"><A href="">Editor System</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Editor_Overview">Editor Overview</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Editor_Results">Editor Results</A></LI><LI class="subs hassubs"><A href="">Modelling</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Model">Modelling Overview</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/enkf">Kalman Filter Ensemble</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/molecular_dynamics">Molecular Dynamics</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/enzyme_kinetics">Enzyme Kinetics</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/grna_design">gRNA Design</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/InterLab">InterLab</A></LI><LI class="hassubs"><A href="#" style="text-decoration:none;color:white;">Human Practice</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Human_Practices">Overview of Human Practice</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Public_Engagement">Public Engagement</A></LI><LI class="subs hassubs"><A href="#">#CasAsks</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Cas_Asks_Survey">#CasAsks: Survey</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Cas_Asks_Interview">#CasAsks: Interview</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Cas_Talks">#CasTalks</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Cas_Teaches">#CasTeaches</A></LI><LI class="subs hassubs"><A href="#">#CasWrites</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Cas_Writes">#CasWrites: Discussion</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Cas_Writes_Collab">#CasWrites: Collaboration</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Cas_Writes_Bioethics">#CasWrites: BioEthics</A></LI><LI><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Collaborations" style="text-decoration:none;color:white;">Collaborations</A></LI><LI class="hassubs"><A href="#" style="text-decoration:none;color:white;">Notebook</A></LI><LI class="subs hassubs"><A href="#">Reporter Logbooks</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Dual_Logbook">Dual Reporter Logbook</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/GLB_Logbook">GLB1 Reporter Logbook</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Editor_Logbook">Editor Logbook</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/General_Docs#">Primers and Protocols</A></LI><LI class="hassubs"><A href="#" style="text-decoration:none;color:white;">People</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Team">Team</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Attributions">Attributions</A></LI><LI class="hassubs"><A href="#" style="text-decoration:none;color:white;">Admin</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Safety">Safety</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Medal">Medals</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Parts">Parts</A></LI></DIV><TITLE> NUS Singapore Science: Dual Reporter </TITLE><H1 id="title"><DIV style="font-size: 2.2em; color: #C0392B"> Demonstration </DIV></H1><DIV class="numberedsection">
1. Mutation of the start codon ATG to ACG in EGFP can significantly block EGFP expression  without affecting mCherry expression
</DIV><DIV class="text">
To demonstrate that a single nucleotide mutation in the start codon could result in ON to OFF change in EGFP expression in cells, we cloned both the EGFP-T2A-mCherry-WT (<A href="http://parts.igem.org/Part:BBa_K2807012" style="font-weight:normal; text-decoration:none;">BBa_K2807012</A>) and the EGFP-T2A-mCherry-ACG mutant (<A href="http://parts.igem.org/Part:BBa_K2807013" style="font-weight:normal; text-decoration:none;">BBa_K2807013</A>) into  C1 mammalian expression vector for expression in HEK293T cells. We transfected HEK293T cells with both wild type (WT) reporter and ACG mutated (ACG) reporter and thereafter, imaged them via microscopy and harvested them for flow cytometry analysis. 

As shown in Figure 1, the cells transfected with EGFP-T2A-mCherry-WT reporter expressed both GFP and mCherry proteins in the same cells, showing that T2A self cleavage peptide is effective. We then further quantified the level of fluorescence by flow cytometry and our results showed that at least 36.6% of the transfected cells are both EGFP+/mCherry+ (Figure 3). There are about 7.9% of the transfected cells with EGFP signal only. This is expected because it is possible for the ribosome to fall off from the mRNA when it encounters the T2A signal peptide and cannot continue with translation. As a result, only EGFP protein is produced but not mCherry. 

On the other hand, HEK293T cells transfected with EGFP-T2A-mCherry-ACG mutant showed red fluorescence expression, with very low or undetectable green fluorescent protein levels as visualised via microscopy (Figure 2). Our experiment showed that the mutation of the start codon in EGFP is indeed effective in abolishing  the expression of EGFP without affecting the expression of the mCherry protein. Compared to the cells transfected with WT reporter, the percentage of EGFP+/mCherry+ positive cells in cells transfected with the EGFP-T2A-mCherry-ACG mutant is almost undetectable  (Figure 3). In addition, the mean fluorescence intensity for mCherry in cells transfected with EGFP-T2A-mCherry-WT and cells transfected with EGFP-T2A-mCherry-ACG mutant are similar, while there is significant reduction in EGFP expression between the two groups of cells.
</DIV><FIGURE class="center"><I><STRONG>Figure 1. Representative fluorescent images of ACG mutant reporter (ACG) wild type reporter (WT) transfected in HEK293T cells.</STRONG> Images taken at 60x magnification, scale bar = 50 µm. </I></FIGURE><FIGURE class="center"><I><STRONG>Figure 2. Representative fluorescent images of ACG mutant reporter (ACG) wild type reporter (WT) transfected in HEK293T cells.</STRONG> Images taken at 10x magnification, scale bar = 400µm.</I></FIGURE><FIGURE class="center"><I><STRONG>Figure 3. Flow cytometry analysis of EGFP and mCherry expression on wild type (WT) and ACG mutant (ACG) reporters transfected in HEK293T cells.</STRONG> (A) Strong correlation between expression level of EGFP and mCherry. (B) No leaky expression of EGFP in mutant ACG reporter. (C) Percentage of EGFP mCherry double positive cells. (D) Quantification of mean fluorescent intensity. Error bar represents SEM, n = 3. </I></FIGURE><DIV class="numberedsection">
2. EGFP-T2A-mCherry Reporter can report DNA editing efficiency of Base Editors
</DIV><DIV class="text">
Although the EGFP-T2A-mCherry Reporter was initially designed for RNA editing, we reasoned that this construct could also be used to report Cas9 editing efficiency in DNA strands. Base editor 3 and 4 (BE3, BE4), fused with the nuclease-deficient dCas9n to the enzyme APOBEC can deaminate C to U in DNA templates, which restores the DNA sequence to T after replication (Komor <I>et al.</I>, 2016; Komor <I>et al.</I>, 2017). 

As such, we tested out our reporter system using BE3 and BE4 plasmids. HEK293T cells were transfected with base editor, ACG reporter and gRNA concurrently and the number of DNA editing events was quantified using flow cytometry. We observed that among the mCherry expressing cells, 60% of them have EGFP fluorescence as a result of base correction from ACG to ATG (Figure 4, 5). The EGFP fluorescence intensity was also restored to a level comparable to the wild type reporter (Table 1), confirming the occurrence of editing events. On the other hand, transfecting cells with only the base editors and ACG reporter showed no EGFP fluorescence, indicating that the editing event is specific and directed by the gRNA sequence. 
</DIV><FIGURE class="center"><I><STRONG>Figure 4. Fluorescent imaging of cells after DNA base editing. </STRONG> HEK293T cells were co-transfected with base editor 3 (BE3), gRNA plasmid, and ACG reporter (A, B), or base editor 4 (BE4), gRNA plasmid, and ACG reporter (C, D), or Base editor 3 (BE3) and ACG reporter (E, F). Images were taken 16 hours post transfection. A-D: 10x magnification, scale bar = 400µm. E-F: 40x magnification, scale bar = 100µm. </I></FIGURE><FIGURE class="center"><I><STRONG>Figure 5. Comparison of editing efficiency using BE3 and BE4.</STRONG> HEK293T cells were transfected with  ACG eGFP-T2A-mCherry reporter with (A) BE3 and gRNA, (B) BE4 and gRNA, (C) BE3 only as control. </I></FIGURE><TABLE style="width:80%;" class="center_table"><CAPTION style="font-size:15px;"><STRONG><I>Table 1. Quantification of base editors (BE3,BE4) editing efficiency using eGFP-T2A-mCherry reporter. Cells are co-transfected with ACG reporter and indicated plasmids.</I></STRONG></CAPTION><TBODY><TR><TH>Sample</TH><TH>EGFP Mean Fluorescent intensity</TH><TH>mCherry Mean Fluorescent Intensity</TH><TH>Percentage of EGFP+ cells among mCherry+ cells</TH></TR><TR><TD>BE3+gRNA</TD><TD>5909</TD><TD>6286</TD><TD>60.6%</TD></TR><TR><TD>BE4+gRNA</TD><TD>5671</TD><TD>6067</TD><TD>62.3%</TD></TR><TR><TD>BE3</TD><TD>816</TD><TD>5456</TD><TD>0.4%</TD></TR></TBODY></TABLE><DIV class="text">
In conclusion, our EGFP-T2A-mCherry reporter system is able to achieve a <STRONG>clear OFF-to-ON switch upon base editing from C to U.</STRONG> It improved the part from a previous iGEM team’s project by adding mCherry (as transfection control) in the same reporter plasmid. In cells expressing our constructs, the expression efficiency, EGFP and mCherry expression showed strong correlation with transfection rates. In addition, our reporter has been shown to work for Cas9-mediated DNA editing, thus providing an alternative tool for real time, <I>in vitro</I> visualization and reporting of base editing efficiency.
</DIV><DIV class="text">
Further results using the Dual-Color Reporter system can be found on our <A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Improve" style="text-decoration:none; font-weight:normal;">Improvements</A> page.
</DIV><DIV class="section">
References
</DIV><DIV class="references">
Komor, A. C., Kim, Y. B., Packer, M. S., Zuris, J. A., &amp; Liu, D. R. (2016). Programmable editing of a target base in genomic DNA without double-stranded DNA cleavage. Nature, 533(7603), 420. 

Komor, A. C., Zhao, K. T., Packer, M. S., Gaudelli, N. M., Waterbury, A. L., Koblan, L. W., Kim, Y.B., Badran, A.H. &amp; Liu, D. R. (2017). Improved base excision repair inhibition and bacteriophage Mu Gam protein yields C: G-to-T: A base editors with higher efficiency and product purity. Science advances, 3(8), eaao4774.
</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>