Team:NUS Singapore-Sci/InterLab
Project
Project Overview
Reporter System
Reporter Overview
Dual Reporter - Demonstration
Dual Reporter - Improvement
GLB1 Reporter
Editor System
Editor Overview
Editor Results
Modelling
Modelling Overview
Kalman Filter Ensemble
Molecular Dynamics
Enzyme Kinetics
gRNA Design
InterLab
Human Practice
Overview of Human Practice
Public Engagement
#CasAsks
#CasAsks: Survey
#CasAsks: Interview
#CasTalks
#CasTeaches
#CasWrites
#CasWrites: Discussion
#CasWrites: Collaboration
#CasWrites: BioEthics
Collaborations
Notebook
Reporter Logbooks
Dual Reporter Logbook
GLB1 Reporter Logbook
Editor Logbook
Primers and Protocols
People
Team
Attributions
Admin
Safety
Medals
Parts
NUS Singapore Science: InterLab
InterLab
Study
This year, Team NUS Singapore-Science took part in the 2018 InterLab Study, which aims to standardise practises for the measurement of gene expression from standard biobrick devices in a standard host organism.
The goal for this year’s InterLab study was to see if the level of green fluorescent protein (GFP) reporter expression correlates with the actual number of transformed bacteria growth (as measured using optical density at 600 nm).
1. Transformation and Inoculation of E. coli strain DH5ɑ cells
We performed this study using the protocol from iGEM HQ and the level of GFP expression was measured using a fluorescence microplate reader (Synergy H1 Hybrid Multi-Mode Reader). Competent cells (Escherichia coli strain DH5ɑ) were transformed with the following plasmids (devices) as shown in Table 1. The transformation protocol was performed as recommended by iGEM HQ. Transformants were grown overnight on Luria Broth (LB) agar plates with chloramphenicol. The next day, two colonies from each plate were picked and inoculated into 5 mL of LB medium with chloramphenicol (25 mg/mL). They were then cultured overnight at 37℃ in a shaking incubator.
Sample Preparation
To prepare the E. coli strain DH5ɑ cells for the cell measurement protocol, a 1:10 dilution was made for each overnight culture in LB medium with Chloramphenicol, and the absorbance at 600 nm (Abs600) of the diluted cultures were measured. The 1:10 diluted cultures were further diluted to a target Abs600 of 0.02 in 12 mL using the formula below:
Volume of 1:10 diluted cultures to add = 0.02 * 12 mL/Abs600
Prior to incubation (t = 0 hours), 0.5 mL of each culture was transferred to a 1.5 mL tube. The tubes were placed on ice. 0.1 mL of each culture was pipetted into a black 96-well plate with clear flat bottom. Four replicates for each culture were made. Eight control wells containing 0.1 mL of LB medium with chloramphenicol were also included. The fluorescence level at 525 nm and Abs600 was measured at t = 0 hours and t = 6 hours.The remaining cultures were incubated for 6 hours at 37℃ in the shaking incubator. At the end of 6-hour incubation, 0.5 mL of each bacterial culture was again transferred to 1.5 mL tubes. Similarly, 0.1 mL of each culture was pipetted into a new black 96-well plate with clear flat bottom. Four replicates per culture were used, and the fluorescence level and absorbance were measured.
The microplate reader used was Synergy H1 Hybrid Multi-Mode Reader (courtesy of National University of Singapore Synthetic Biology for Clinical and Technological Innovation (SynCTI)), which could measure both absorbance and fluorescence. For all the Abs600 and fluorescence measurements, pathlength correction was turned off, and the temperature setting kept to room temperature (20-25°C). The bandpass width is fixed at 16 nm and bottom optics were used.
Table 1. Biobrick Devices as obtained from iGEM HQ. These devices were transformed into E. coli DH5ɑ cells.
Device
Partnumber
Negative control
BBa_R0040
Positive control
BBa_I20270
Test Device 1
BBa_J364000
Test Device 2
BBa_J364001
Test Device 3
BBa_J364002
Test Device 4
BBa_J364007
Test Device 5
BBa_J364008
Test Device 6
BBa_J364009
Calibration procedure
Before the execution of the actual measurement experiment, three different calibration protocols were carried out to obtain: 1) an optical density (OD600) reference point, 2) a particle standard curve for the number of particles at a certain Abs600 and 3) a fluorescein standard curve for fluorescence intensity.
2. OD600 Reference Point - LUDOX Protocol
This calibration protocol was done to obtain a conversion factor for Abs600 as measured by the plate reader to an OD600 measurement. LUDOX CL-X (45% colloidal silica suspension) was used, and the LUDOX solution will give a low absorbance value as it is weakly scattering. Here, pathlength correction was switched off so as to not provide correction for the volume of sample in the 96-well plate.
Table 2. Results of Abs600 of LUDOX CL-X and H2O. Corrected Abs values were used to calculate the conversion factor from Abs600 to OD600.
LUDOX CL-X
H2O
Replicate 1
0.101
0.084
Replicate 2
0.100
0.086
Replicate 3
0.103
0.085
Replicate 4
0.106
0.087
Arith. Mean
0.103
0.086
Corrected ABS
0.017
Reference Abs
0.063
OD600/Abs600
3.706
The conversion factor from an Abs600 value to a OD600 value is obtained by dividing the reference OD600 value measured by a spectrophotometer by the mean correct Abs600 value. As shown in Table 2, the conversion factor obtained is 3.706. Thus, Abs600 measured by the plate reader can be converted to the OD600 reading by multiplying the conversion factor, 3.706. Background correction for the Abs600 value was done by subtracting the mean of four replicates with the mean Abs600 value of H2O.
3. Particle Standard Curve - Microsphere Protocol
This calibration protocol was done to obtain the Abs600 values for known concentrations of microspheres, which have simlar size and optical characteristics compared to E. coli strain DH5ɑ cells. In doing so, for a given Abs600 value, we are able to approximate the number of E. coli strain DH5ɑ cells present in the well. A dilution series of monodisperse silica microspheres were prepared in the 96-well plates and the Abs600 measured. Four replicates for each dilution were prepared.
Table 3. Results of Abs600 for increasing microsphere particle numbers. As particle number increases, Abs600 values increases.
As shown in Table 3, for each microsphere particle number, four replicates were done. The mean of the four replicates were taken, and the standard deviation measured. The mean corrected Abs600 values are obtained by subtracting the mean Abs600 values from that of the blank which is double distilled H2O (ddH2O). From the last row of results as in Table 3 which gives the mean corrected Abs600 values, a particle standard curve (Figure 1) is plotted. A graph of Abs600 against particle count per 0.1 mL is shown in Figure 1. The particle standard curve will provide us with an estimate of the number of particles present in a solution given the Abs600 value.
Figure 1. Standard curve of Abs600 values against particle number. Mean Abs600 data was plotted against particle count per 0.1 mL for serial diluted samples of microspheres.
A log scale of the same particle standard curve in Figure 1 was also plotted, where the Abs600 values and values for particle count per 0.1 mL were applied the logarithmic number to obtain the graph as in Figure 2.
Figure 2. Standard curve of Abs600 values against particle number on a logarithmic scale.
4. Fluorescence Standard Curve - Fluorescein Dye Protocol
This calibration experiment was carried out to compare fluorescence values obtained from the test devices between teams, as different plate readers report different arbitrary units of fluorescence. Here, fluorescein dye is used in place of GFP as it has similar excitation and emission properties to GFP. The excitation wavelength used was 494 nm and the emission maximum used was 525 nm. Serial dilutions of fluorescein in 1x phosphate-buffered saline (PBS) were prepared in the 96-well plates and the fluorescence levels of the dye was measured (Table 4). This will enable us to obtain a standard fluorescence curve allows us to compare  fluorescence values across teams more accurately.
Table 4. Results of fluorescence values against concentration of fluorescein. Fluorescence values increase with increasing concentrations of fluorescein.
Based on the raw fluorescence values obtained in Table 4, a fluorescein standard curve of fluorescence against fluorescein concentration was plotted (Figure 3). The fluorescence standard curve will provide us with an estimate of the amount of fluorescein (or GFP) present in a solution when given the fluorescence value.
Figure 3. Standard curve of fluorescence values at 525 nm against concentration of fluorescein (µM). Mean fluorescence value was plotted against fluorescein concentration for serial diluted samples of fluorescein.
A log scale of the fluorescence standard curve in Figure 3 was also plotted, where the fluorescence values and concentrations of fluorescein were applied the logarithmic number to obtain the graph as in Figure 4.
Figure 4. Standard curve of fluorescence values at 525 nm against fluorescein concentration on a logarithmic scale.
From the Fluorescein µM values, the Molecules of Equivalent Fluorescence (MEFL) can be calculated. MEFL refers to the number of cells present, assuming that each of them give off an equal level of fluorescence. This can be calculated using the following parameters:
Initial Molarity: 1.00E-05
Molecules/mol: 6.02E+23
Well Volume (L): 1.00E-04
Initial molecules: 6.02+14
MEFL/µM: 6.02E+12
Table 5. Calculations to obtain MEFL/a.u.
5. Cell Measurement Protocol for GFP expression over time
As mentioned above, at 0 hours and 6 hours, an aliquot of the culture was pipetted out and the fluorescence at 525 nm and Abs600 values were measured accordingly. Four replicates were done for each of the two colonies per device.
From the calibration protocols in sections 1-3, the unit scaling factors were calculated and shown in Table 6.
Table 6. Unit scaling factors obtained from the calibration protocols.
OD600/Abs600
3.71
µM Fluorescein/a.u.
1.08E-04
Particles/Abs600
2.46E+08
MEFL/a.u.
6.50E+08
Table 7. Net fluorescence in a.u. at t = 0 h (above) and t = 6 h (below).
Table 8. Net Abs600 at t = 0 h (above) and t = 6 h (below).
Table 9. µM Fluorescein/OD at t = 0 h (above) and t = 6 h (below).
Table 10. MEFL/particle at t = 0 h (above) and t = 6 h (below).
6. Protocol: Colony Forming Units per 0.1 OD600 E.coli cultures
This procedure was done to calibrate the OD600 to give colony forming unit (CFU) counts, which can then be related to the bacteria concentration of the culture (CFU/mL). Under this protocol, the assumption is that one bacteria will give rise to one colony. This protocol only involves E. coli cultures that have been transformed with the Positive Control (BBa_I20270) or Negative Control (BBa_R0040) devices. Two colonies from each plate were taken and cultured overnight.
Sample Preparation
The next day, the cell cultures were diluted to the linear detection range of the plate reader by doing a 1:8 dilution. 25 µL of culture was added to 175 µL of LB broth with chloramphenicol in a black 96-well plate. The OD600 of these cultures were measured.
As the starting OD600 should be standardised to 0.1 for all, the E. coli cultures were diluted with LB broth with Chloramphenicol according to the following equation:
C1 * V1 = C2* V2
Where C1=(1:8 OD600 - blank OD600)*8
OD600 values of the diluted cultures (starting samples) were measured again (in triplicates) to ensure that the OD600 values are at approximately 0.1 after subtracting from the blank measurement.
Serial dilutions for the starting samples were prepared as shown in Figure 5. For dilutions 3, 4 and 5, 0.1 mL of the bacterial cultures were spread on LB agar plates with chloramphenicol. The plates were then incubated at 37℃ overnight.
Figure 5. Serial dilutions for the starting samples were done, and dilutions 3,4 and 5 plated and incubated overnight.
The next day, colonies were counted for plates with fewer than 300 colonies (Table 11). The CFU/mL values were calculated by multiplying the number of colonies with the appropriate dilution factor.
Table 11. Number of colonies on each LB agar plate with chloramphenicol after overnight incubation (Too numerous to count, more than 300 colonies = TNTC).
Sample
No. of colonies (Dilution 3: 8*10^4)
No. of colonies (Dilution 4: 8*10^5)
No. of colonies (Dilution 5: 8*10^6)
Calculated CFU/mL of culture based on Dilution 4 (8*10^5)
Positive Control Colony 1 (1)
TNTC
110
2
8.8E+07
Positive Control Colony 1 (2)
TNTC
127
12
10.16E+07
Positive Control Colony 1 (3)
TNTC
88
7
7.04E+07
Positive Control Colony 2 (1)
TNTC
168
8
13.44E+07
Positive Control Colony 2 (2)
TNTC
148
2
11.84E+07
Positive Control Colony 2 (3)
TNTC
136
3
10.88E+07
Negative Control Colony 1 (1)
TNTC
101
1
8.08E+07
Negative Control Colony 1 (2)
TNTC
145
13
11.6E+07
Negative Control Colony 1 (3)
TNTC
169
11
13.52E+07
Negative Control Colony 2 (1)
TNTC
132
15
10.56E+07
Negative Control Colony 2 (2)
TNTC
143
18
11.44E+07
Negative Control Colony 2 (3)
TNTC
62
4
4.96E+07
7. Overall Evaluation
Overall, our team had no issues with the InterLab protocols.
