Team:NUS Singapore-Sci/Demonstrate
Project
Project Overview
Reporter System
Reporter Overview
Dual Reporter - Demonstration
Dual Reporter - Improvement
GLB1 Reporter
Editor System
Editor Overview
Editor Results
Modelling
Modelling Overview
Kalman Filter Ensemble
Molecular Dynamics
Enzyme Kinetics
gRNA Design
InterLab
Human Practice
Overview of Human Practice
Public Engagement
#CasAsks
#CasAsks: Survey
#CasAsks: Interview
#CasTalks
#CasTeaches
#CasWrites
#CasWrites: Discussion
#CasWrites: Collaboration
#CasWrites: BioEthics
Collaborations
Notebook
Reporter Logbooks
Dual Reporter Logbook
GLB1 Reporter Logbook
Editor Logbook
Primers and Protocols
People
Team
Attributions
Admin
Safety
Medals
Parts
NUS Singapore Science: Dual Reporter
Demonstration
1. Mutation of the start codon ATG to ACG in EGFP can significantly block EGFP expression  without affecting mCherry expression
To demonstrate that a single nucleotide mutation in the start codon could result in ON to OFF change in EGFP expression in cells, we cloned both the EGFP-T2A-mCherry-WT ( BBa_K2807012 ) and the EGFP-T2A-mCherry-ACG mutant ( BBa_K2807013 ) into  C1 mammalian expression vector for expression in HEK293T cells. We transfected HEK293T cells with both wild type (WT) reporter and ACG mutated (ACG) reporter and thereafter, imaged them via microscopy and harvested them for flow cytometry analysis.
As shown in Figure 1, the cells transfected with EGFP-T2A-mCherry-WT reporter expressed both GFP and mCherry proteins in the same cells, showing that T2A self cleavage peptide is effective. We then further quantified the level of fluorescence by flow cytometry and our results showed that at least 36.6% of the transfected cells are both EGFP+/mCherry+ (Figure 3). There are about 7.9% of the transfected cells with EGFP signal only. This is expected because it is possible for the ribosome to fall off from the mRNA when it encounters the T2A signal peptide and cannot continue with translation. As a result, only EGFP protein is produced but not mCherry.
On the other hand, HEK293T cells transfected with EGFP-T2A-mCherry-ACG mutant showed red fluorescence expression, with very low or undetectable green fluorescent protein levels as visualised via microscopy (Figure 2). Our experiment showed that the mutation of the start codon in EGFP is indeed effective in abolishing  the expression of EGFP without affecting the expression of the mCherry protein. Compared to the cells transfected with WT reporter, the percentage of EGFP+/mCherry+ positive cells in cells transfected with the EGFP-T2A-mCherry-ACG mutant is almost undetectable  (Figure 3). In addition, the mean fluorescence intensity for mCherry in cells transfected with EGFP-T2A-mCherry-WT and cells transfected with EGFP-T2A-mCherry-ACG mutant are similar, while there is significant reduction in EGFP expression between the two groups of cells.
Figure 1. Representative fluorescent images of ACG mutant reporter (ACG) wild type reporter (WT) transfected in HEK293T cells. Images taken at 60x magnification, scale bar = 50 µm.
Figure 2. Representative fluorescent images of ACG mutant reporter (ACG) wild type reporter (WT) transfected in HEK293T cells. Images taken at 10x magnification, scale bar = 400µm.
Figure 3. Flow cytometry analysis of EGFP and mCherry expression on wild type (WT) and ACG mutant (ACG) reporters transfected in HEK293T cells. (A) Strong correlation between expression level of EGFP and mCherry. (B) No leaky expression of EGFP in mutant ACG reporter. (C) Percentage of EGFP mCherry double positive cells. (D) Quantification of mean fluorescent intensity. Error bar represents SEM, n = 3.
2. EGFP-T2A-mCherry Reporter can report DNA editing efficiency of Base Editors
Although the EGFP-T2A-mCherry Reporter was initially designed for RNA editing, we reasoned that this construct could also be used to report Cas9 editing efficiency in DNA strands. Base editor 3 and 4 (BE3, BE4), fused with the nuclease-deficient dCas9n to the enzyme APOBEC can deaminate C to U in DNA templates, which restores the DNA sequence to T after replication (Komor et al., 2016; Komor et al., 2017).
As such, we tested out our reporter system using BE3 and BE4 plasmids. HEK293T cells were transfected with base editor, ACG reporter and gRNA concurrently and the number of DNA editing events was quantified using flow cytometry. We observed that among the mCherry expressing cells, 60% of them have EGFP fluorescence as a result of base correction from ACG to ATG (Figure 4, 5). The EGFP fluorescence intensity was also restored to a level comparable to the wild type reporter (Table 1), confirming the occurrence of editing events. On the other hand, transfecting cells with only the base editors and ACG reporter showed no EGFP fluorescence, indicating that the editing event is specific and directed by the gRNA sequence.
Figure 4. Fluorescent imaging of cells after DNA base editing. HEK293T cells were co-transfected with base editor 3 (BE3), gRNA plasmid, and ACG reporter (A, B), or base editor 4 (BE4), gRNA plasmid, and ACG reporter (C, D), or Base editor 3 (BE3) and ACG reporter (E, F). Images were taken 16 hours post transfection. A-D: 10x magnification, scale bar = 400µm. E-F: 40x magnification, scale bar = 100µm.
Figure 5. Comparison of editing efficiency using BE3 and BE4. HEK293T cells were transfected with  ACG eGFP-T2A-mCherry reporter with (A) BE3 and gRNA, (B) BE4 and gRNA, (C) BE3 only as control.
Table 1. Quantification of base editors (BE3,BE4) editing efficiency using eGFP-T2A-mCherry reporter. Cells are co-transfected with ACG reporter and indicated plasmids.
Sample
EGFP Mean Fluorescent intensity
mCherry Mean Fluorescent Intensity
Percentage of EGFP+ cells among mCherry+ cells
BE3+gRNA
5909
6286
60.6%
BE4+gRNA
5671
6067
62.3%
BE3
816
5456
0.4%
In conclusion, our EGFP-T2A-mCherry reporter system is able to achieve a clear OFF-to-ON switch upon base editing from C to U. It improved the part from a previous iGEM team’s project by adding mCherry (as transfection control) in the same reporter plasmid. In cells expressing our constructs, the expression efficiency, EGFP and mCherry expression showed strong correlation with transfection rates. In addition, our reporter has been shown to work for Cas9-mediated DNA editing, thus providing an alternative tool for real time, in vitro visualization and reporting of base editing efficiency.
Further results using the Dual-Color Reporter system can be found on our Improvements page.
References
Komor, A. C., Kim, Y. B., Packer, M. S., Zuris, J. A., & Liu, D. R. (2016). Programmable editing of a target base in genomic DNA without double-stranded DNA cleavage. Nature, 533(7603), 420.
Komor, A. C., Zhao, K. T., Packer, M. S., Gaudelli, N. M., Waterbury, A. L., Koblan, L. W., Kim, Y.B., Badran, A.H. & Liu, D. R. (2017). Improved base excision repair inhibition and bacteriophage Mu Gam protein yields C: G-to-T: A base editors with higher efficiency and product purity. Science advances, 3(8), eaao4774.
