Team:NUS Singapore-Sci/grna design
Project
Project Overview
Reporter System
Reporter Overview
Dual Reporter - Demonstration
Dual Reporter - Improvement
GLB1 Reporter
Editor System
Editor Overview
Editor Results
Modelling
Modelling Overview
Kalman Filter Ensemble
Molecular Dynamics
Enzyme Kinetics
gRNA Design
InterLab
Human Practice
Overview of Human Practice
Public Engagement
#CasAsks
#CasAsks: Survey
#CasAsks: Interview
#CasTalks
#CasTeaches
#CasWrites
#CasWrites: Discussion
#CasWrites: Collaboration
#CasWrites: BioEthics
Collaborations
Notebook
Reporter Logbooks
Dual Reporter Logbook
GLB1 Reporter Logbook
Editor Logbook
Primers and Protocols
People
Team
Attributions
Admin
Safety
Medals
Parts
NUS Singapore Science: InterLab
gRNA Design
Algorithm
Design of gRNA sequences
Manual design of the gRNA can be a cumbersome task when large amount of bases have to be  taken into account. To generate double-stranded RNA coding for Cas13b-editing system, we wrote our own programme as a more efficient method to generating suitable gRNA gBlock sequences. Taking the above considerations into account, a Python script was written as a convenient method of generating suitable gRNA gene block (gBlock) sequences. The length of targeted DNA sequences for the spacer region will have to be first defined by the user. The code will perform a search for the Kozak’s sequence within the given strand. If the Kozak sequence is found, the hairpin motif with the U6 terminator sequence is thereafter concatenated to the 3’ end of the spacer sequence. Finally, BbsI restriction sites and junk sequences will be joined to both 5’ and 3’ ends to obtain the final gRNA gBlock construct (Figure 1).
Figure 1. A mock schematics of how the gRNA gBlock sequences were produced using our PythonTM script.The user will have to input a 50 base pair DNA sequence from the reporter gene containing the Kozak sequence. The code generates the complementary and reversed DNA sequence from the targeted binding site, before concatenating the hairpin motif to the 3’ end of the spacer domain. U6 terminator sites were then added to the 3’ end of the hairpin domain. Finally, BbsI recognition and cleavage sites are added to the 5’ and 3’ ends to obtain the final gBlock fragment.
