Team:NUS Singapore-Sci/Editor Overview
Project
Project Overview
Reporter System
Reporter Overview
Dual Reporter - Demonstration
Dual Reporter - Improvement
GLB1 Reporter
Editor System
Editor Overview
Editor Results
Modelling
Modelling Overview
Kalman Filter Ensemble
Molecular Dynamics
Enzyme Kinetics
gRNA Design
InterLab
Human Practice
Overview of Human Practice
Public Engagement
#CasAsks
#CasAsks: Survey
#CasAsks: Interview
#CasTalks
#CasTeaches
#CasWrites
#CasWrites: Discussion
#CasWrites: Collaboration
#CasWrites: BioEthics
Collaborations
Notebook
Reporter Logbooks
Dual Reporter Logbook
GLB1 Reporter Logbook
Editor Logbook
Primers and Protocols
People
Team
Attributions
Admin
Safety
Medals
Parts
NUS Singapore Science: InterLab
Editor System
Overview
Designing a potential RNA-targeting base editor using the CRISPR-Cas System.
This section describes the RNA editing system which we have designed and put together for targeted C-to-U editing. Our system is a fusion protein which consists of the apolipoprotein B mRNA editing enzyme, (APOBEC) and the deactivated Cas13 (dCas13), an RNA-targeting Cas protein.
1. Background
There is yet to exist a system that can make specific Cytidine (C) to Uridine (U) edits on mRNA, which can be accomplished through a deamination process (Figure 1). APOBEC, a cytidine deaminase enzyme, is a well-established protein which is known to perform C to U conversions on RNA strands (Smith, Bennett, Kizilyer, McDougall & Prohaska, 2012). Therefore, we intend to fuse an RNA-targeting Cas13 to the APOBEC enzyme. We hypothesised that the APOBEC enzyme, when fused with a Cas13 protein may perform a C to U conversion for a specific RNA target.
Figure 1. Diagram of the deamination of Cytidine to Uridine, which is accomplished by cytidine deaminases such as APOBEC.
Hence, our project aims to develop and verify an RNA editing system using the enzyme APOBEC fused to dCas13b. Deactivated Cas13b, which differs from the activated Cas13b by two amino acids (H133A and H1058A), is able to still be guided by guide RNAs (gRNAs) to target the specific RNA strand. However, as enzymatic activity is deactivated, no cleavage of RNA strands will occur. dCas13b together with the gRNA will guide the fusion protein to the appropriate site on the target mRNA to be edited. Then, the APOBEC will make the specific C to U edit on the mRNA strand.
2. Cas13-based RNA Editor System Design
Before the fusion protein can be synthesised, a linker sequence is required between rAPOBEC and dCas13 to prevent misfolding of the two proteins. In addition, we attempted to model the expected 3D structure of the fusion protein. In Komor et al. (2016), several different linkers were tested to link rat APOBEC (rAPOBEC) and dCas9, and the XTEN linker sequence (SGSETPGTSESATPES) was shown to be the most efficient. Hence, XTEN was chosen as the linker sequence to link APOBEC and dCas13b for our fusion protein.
rAPOBEC-XTEN was extracted from the Base Editor 3 (BE3) plasmid (rAPOBEC1-XTEN-Cas9n-UGI-NLS, plasmid 73021) via PCR. dCas13b from Prevotella sp. P5-125 was extracted from the pC0049 plasmid as obtained from Addgene plasmid #103865. The gene fragments were cloned into px330A plasmid (Figure 2). We also designed (as elaborated in Section 3) and expressed our gRNA under the U6 promoter.
Figure 2. Plasmid map of px330A which was used in a CRISPR/Cas9 expression system by Zhang lab. The plasmid contains a U6 promoter for gRNA, chicken β-actin promoter for the fusion protein, is suitable for transfection into HEK293T cells and can accommodate large insert sizes. Hence, px330A was chosen as the plasmid to hold the RESCUE editor system.
Figure 3. A schematic illustration of our rAPOBEC-XTEN-dPspCas13b editor system. The rAPOBEC-XTEN and dPspCas13b gene fragments will be assembled into px330A plasmid via NEBuilder HiFi DNA Assembly.
3. Selection and cloning of guide RNA sequences
To design our gRNA sequences, we adopted the method that was described in Cox et al. (2017). According to the paper, the A-to-I base editing efficiency was most optimal when using the hairpin sequence of 5’ -GUUGUGGAAGGUCCAGUUUUGGGGGCUAUUACAACA- 3’ affixed to the 3’ end of the spacer domain. We also used a spacer length of 50 bp, with a mismatch position of around 10-30 bp as suggested in the paper. Such a design criterion has been shown to be successful in optimising successful gRNAs for RNA editing experiments. According to Sakuma et al. (2014), the terminator site is situated at the end of Cas9 hairpin motif site in px330A, thus our gRNA sequence will have to be synthesised with a U6 terminator sequence at the 3’ end to prevent the coding of an elongated hairpin domain.
Taking the above considerations into account, we wrote a PythonTM script to help us design and generate suitable gRNA gene blocks for synthesis. Firstly, the targeted mRNA sequence and its associated mutant site position will be defined by the user. The script will then perform a search for the mismatch base in the defined region and generate a spacer sequence that is reverse-complementary to the original strand. The hairpin motif with the U6 terminator sequence is thereafter concatenated to the 3’ end of the spacer sequence, before being reverse-transcribed to its cDNA form. Finally, BbsI restriction sites and junk sequences will be added to both 5’ and 3’ ends to obtain the gRNA gene block sequence (Figure 4).
Figure 4. A mock schematics of how the gRNA gBlock sequences were produced using our PythonTM script. (A) The script generates the complementary and reversed DNA sequence from the targeted binding site, before concatenating the hairpin motif to the 3’ end of the spacer domain. (B) The gRNA sequence (in RNA abbreviation) is reverse-transcribed to cDNA sequence.  (C) Junk sequences, BbsI recognition sites and cleavage sites are added to the 5’ and 3’ end after the generation of the gRNA cDNA sequence to obtain the final gene block fragment.
4. Testing of the RNA editing system
To test our RNA editing system , we will first analyse the in vitro cleavage activity of our APOBEC enzyme using a deaminase assay. Subsequently, we will analyse our RNA editing system in vivo using cultured cells. To do that, we plan to express our dCas13b-APOBEC device (with gRNA) and our fluorescence reporter device in mammalian cells. We will be checking for the restoration of our green fluorescent protein (GFP) reporter by flow cytometry and fluorescence microscopy.
In addition, we also plan to test our system on its ability to edit mutated bases in our mammalian β-galactosidase enzyme. Point mutations in the β-galactosidase gene have been implicated in lysosomal storage disease and our RNA editing device could act as a potential therapeutic in the treatment of the disease.
References
Cox, D. B. T., Gootenberg, J. S., Abudayyeh, O. O., Franklin, B., Kellner, M. J., Joung, J., & Zhang, F. (2017). RNA Editing with CRISPR-Cas13 HHS Public Access. Science, 358(6366), 1019–1027. http://doi.org/10.1126/science.aaq0180
Komor, A. C., Kim, Y. B., Packer, M. S., Zuris, J. A., & David, R. (2016). HHS Public Access, 533(7603), 420–424. http://doi.org/10.1038/nature17946.Programmable
Sakuma, T., Nishikawa, A., Kume, S., Chayama, K., & Yamamoto, T. (2014). Multiplex genome engineering in human cells using all-in-one CRISPR/Cas9 vector system. Scientific Reports, 4, 4–9. http://doi.org/10.1038/srep05400
Smith, H. C., Bennett, R. P., Kizilyer, A., McDougall, W. M., & Prohaska, K. M. (2012). Functions and Regulation of the APOBEC Family of Proteins. Seminars in Cell & Developmental Biology, 23(3), 258–268. http://doi.org/10.1016/j.semcdb.2011.10.004
