Team:NUS Singapore-Sci/Dual Logbook
Project
Project Overview
Reporter System
Reporter Overview
Dual Reporter - Demonstration
Dual Reporter - Improvement
GLB1 Reporter
Editor System
Editor Overview
Editor Results
Modelling
Modelling Overview
Kalman Filter Ensemble
Molecular Dynamics
Enzyme Kinetics
gRNA Design
InterLab
Human Practice
Overview of Human Practice
Public Engagement
#CasAsks
#CasAsks: Survey
#CasAsks: Interview
#CasTalks
#CasTeaches
#CasWrites
#CasWrites: Discussion
#CasWrites: Collaboration
#CasWrites: BioEthics
Collaborations
Notebook
Reporter Logbooks
Dual Reporter Logbook
GLB1 Reporter Logbook
Editor Logbook
Primers and Protocols
People
Team
Attributions
Admin
Safety
Medals
Parts
NUS Singapore Science: InterLab
Dual Reporter
Logbook
1) Cloning of EGFP-T2A-mCherry reporter into pSB1C3 plasmid
The reporter was constructed with EGFP ligated to mCherry, with a viral cleavage peptide T2A in between the two fluorescent genes, in a pSB1C3 vector backbone. EGFP and mCherry were PCR amplified from the Clontech plasmid (Zhao et al., 2017) and T2A cleavage peptide sequence was synthesised according to a recent publication by Liu et al. (2017). After ligating the EGFP fragment with T2A, the reporter was assembled using the NEBuilder HiFi DNA Assembly kit (NEB).
The ideal concentration of each fragment of interest was calculated based on the recommended proportions for 2-3 fragment ligation from the NEBuilder HiFi DNA Assembly kit (Table 1). The NEBuilder HiFi DNA Assembly 2x master mix was added to the required amount of DNA fragments in a 20ul reaction. The reaction was then incubated at 50℃ for 15 minutes, before transforming into DH5α Competent E. coli cells.
Table 1: The proportion of inserts to vector. The recommended insert(s) to vector mole ratio is 2:2:1. Mass of each gene fragment to use is calculated based on mole ratio and insert size.
Fragment (bp)
Ratio
pmol
Mass (ng)
EGFP-T2A (813 bp)
2
0.08
40.19
mCherry (714 bp)
2
0.08
35.25
pSB1C3 with promoter and terminator (2700 bp)
1
0.04
66.74
We screened for the reporter insert using colony PCR. Colony PCR was carried out for 18 colonies using primers eGFP F1 and mCherry R1, and the expected band size is approximately 1.5 kb. Colonies 8 and 18 were found to contain the desired product (Figure 1).
Figure 1: Colony PCR of 18 colonies picked from transformation plate cultured for 16 hours at 37℃.The two colonies, 8 and 18 were, found to have a faint band at 1.5kb (in the red box), indicative of the desired product.
The two colonies were then subsequently cultured overnight at 37℃ for sequencing reaction. Sequencing results from plasmids extracted from colony 8 showed to have the correct sequence. This properly constructed sequence, together with the constitutive promoter and terminator was then cloned to C1 vector to prepare a mammalian construct for transfection into HEK293T cells.
2) Mutating the ACG base in the WT EGFP gene
Once the WT construct was ready, a mutation was introduced to the start codon such that the T is changed to a C (ATG -> ACG). The mutant was prepared by using VF2 and new ACG R1 primer as well as ACG F1 and VR primers in separate reactions to introduce the mutation. The VF2 ACG R1 mutation should show a band of approximately 700 bp in lanes 1 and 2 while the ACG F1 VR mutation should show a band of approximately 1.7kb as seen in lanes 3 and 4 (Figure 2). An annealing temperature of 72 ℃ is used in all samples.
Figure 2: Fragments of VF2 ACG R1 and ACG F1 VR reactions were run on a 1% gel after PCR.Both VF2 ACG R1 and ACG F1 VR reactions showed bright bands at the expected sizes of 700 bp and 1.7 kb (in the red box), respectively. All reactions made use of the same annealing temperature of 72℃.
The two fragments obtained are then ligated together using PCR and Q5 DNA polymerase at an annealing temperature of 72℃ (Figure 3). Lanes 1 and 2 showed an approximate size of 2.5kb, which is the expected size of the EGFP-T2A-mCherry fragment (lane 1 and 2).
Figure 3: Full fragment of ACG mutant for eGFP-T2A-mCherry. The reaction to stitch both fragments together to obtain eGFP-T2A-mCherry ACG mutant appeared to be successful as aa faint band of expected size 2.5 kb was observed(in red box). The annealing temperature used for the reaction is 72℃.
The mutant construct was then transformed into DH5α cells and the plasmid was isolated for DNA sequencing. DNA Sequencing results showed the correct T to C mutation at the ATG site (ATG to ACG) and this EGFP-T2A-mCherry fragment ACG mutation was ligated to the C1 mammalian expression vector for transfection into HEK293 cells. In addition, the single PstI illegal site found in both WT EGFP-T2A-mCherry fragment (ATG site in place) and the EGFP-T2A-mCherry fragment ACG mutation were removed through PCR mutagenesis (primers used are available here ) . The PstI site was successfully removed for both WT EGFP-T2A-mCherry fragment (ATG site in place) and EGFP-T2A-mCherry fragment ACG mutation constructs. As seen from Figure 4, the EGFP-T2A-mCherry fragment ACG mutant at the expected molecular weight of 2.5kb, was seen on a DNA agarose gel.
Figure 4: Full fragment of EGFP-T2A-mCherry ACG mutation (with PstI site mutation).The EGFP-T2A-mCherry ACG mutation gene showed bands of expected size 2.5kb. The reactions were carried out at annealing temperature 72℃.
The full fragment of EGFP-T2A-mCherry ACG mutation (with PstI site mutation) was ligated into the pSB1C3 vector before transformation into DH5α cells and 10 colonies were picked for analysis using colony PCR (Figure 5). The expected 2.5 kb of the EGFP-T2A-mCherry ACG mutation was observed in all lanes except lane 7 (Fig 5). Then, plasmid isolation was carried out and the plasmids were sent for DNA sequencing to verify the presence of both mutations. Our sequenced results showed successful mutation of both sites.
Figure 5: 10 colonies were picked in total from the transformation plate for eGFP-T2A-mCherry with ACG and PstI site mutations.All colonies showed the correct bands except lane 7. This shows the successful transformation and sequencing was then done to identify the single base mutations.
4) Characterization of EGFP-T2A-mCherry reporter using Flow cytometry
To test the efficiency of EGFP and mCherry expression of the two constructs (EGFP-T2A-mCherry WT and EGFP-T2A-mCherry with ACG mutation) in mammalian cells, transfection was carried out on HEK293T cells as described here . 2 ug of WT EGFP-T2A-mCherry DNA and WT EGFP-T2A-mCherry DNA were used respectively. Triplicates were carried out for each transfection condition. The cells were then harvested for flow cytometry after 24 hours.
The percentage of fluorescent cells were detected by flow cytometry (FACSAriaII flow cytometer, BD Biosciences, CA, USA).  After analysis, we obtained the geometric mean Fluorescence Intensity using the FlowJo software. The raw data is available in the following file.
To further characterize Worcester ATG EGFP ( BBa_K2083009 ) or ACG EGFP ( BBa_K2083010 ) parts, we cloned the biobrick part into a mammalian expression vector using PCR . Mammalian cells were transfected with the following parameters (see below). We used empty mCherry plasmid from Addgene as a transfection control. The cells were then harvested for flow cytometry after 24 hours of transfection. The data is available in the following file.
Sample Name(bp)
GFP
mCherry
Number of replicates
ATG EGFP
1ug
1ug
3
ACG EGFP
1ug
1ug
3
Application of EGFP-T2A-mCherry reporter in DNA editing events
To test the EGFP-T2A-mCherry reporter’s ability to report DNA editing events, we cloned 5’ -CACCACGGAGCTGTTCACC-3’ Cas9 gRNA into pX330A plasmid using BstI sites, and transfected the cells as in the following table. Again, the data is available in the file below.
Sample Name(bp)
ACG Reporter
Base Editor
gRNA
Empty Vector
Number of Replicates
BE3+ACG+gRNA
0.6ug
1ug
0.4ug
0ug
1
BE4+ACG+gRNA
0.6ug
1ug
0.4ug
0ug
1
BE3+ACG
0.6ug
1ug
0ug
0.4ug
1
