<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NUS_Singapore-Sci_Reporter_GLB skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NUS Singapore-Sci/Reporter GLB</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV class="main"><UL class="mainnav"><LI class="hassubs"><A href="#" style="text-decoration:none;color:white;">Project</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Overview">Project Overview</A></LI><LI class="subs hassubs"><A href="">Reporter System</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Reporter_Overview">Reporter Overview</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Demonstrate">Dual Reporter - Demonstration</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Improve">Dual Reporter - Improvement</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Reporter_GLB">GLB1 Reporter</A></LI></UL><LI class="subs hassubs"><A href="">Editor System</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Editor_Overview">Editor Overview</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Editor_Results">Editor Results</A></LI><LI class="subs hassubs"><A href="">Modelling</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Model">Modelling Overview</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/enkf">Kalman Filter Ensemble</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/molecular_dynamics">Molecular Dynamics</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/enzyme_kinetics">Enzyme Kinetics</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/grna_design">gRNA Design</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/InterLab">InterLab</A></LI><LI class="hassubs"><A href="#" style="text-decoration:none;color:white;">Human Practice</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Human_Practices">Overview of Human Practice</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Public_Engagement">Public Engagement</A></LI><LI class="subs hassubs"><A href="#">#CasAsks</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Cas_Asks_Survey">#CasAsks: Survey</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Cas_Asks_Interview">#CasAsks: Interview</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Cas_Talks">#CasTalks</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Cas_Teaches">#CasTeaches</A></LI><LI class="subs hassubs"><A href="#">#CasWrites</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Cas_Writes">#CasWrites: Discussion</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Cas_Writes_Collab">#CasWrites: Collaboration</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Cas_Writes_Bioethics">#CasWrites: BioEthics</A></LI><LI><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Collaborations" style="text-decoration:none;color:white;">Collaborations</A></LI><LI class="hassubs"><A href="#" style="text-decoration:none;color:white;">Notebook</A></LI><LI class="subs hassubs"><A href="#">Reporter Logbooks</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Dual_Logbook">Dual Reporter Logbook</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/GLB_Logbook">GLB1 Reporter Logbook</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Editor_Logbook">Editor Logbook</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/General_Docs#">Primers and Protocols</A></LI><LI class="hassubs"><A href="#" style="text-decoration:none;color:white;">People</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Team">Team</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Attributions">Attributions</A></LI><LI class="hassubs"><A href="#" style="text-decoration:none;color:white;">Admin</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Safety">Safety</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Medal">Medals</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Parts">Parts</A></LI></DIV><TITLE> NUS Singapore Science: InterLab </TITLE><H1 id="title"><DIV style="font-size: 2.2em; color: #C0392B"> Reporter System </DIV><SPAN style="font-size: 2em; color: #000080"> GLB1 </SPAN></H1><DIV class="outline" style="font-size:16px;">
GLB1 encodes a human lysosomal acid β-galactosidase 1 (GLB1), an enzyme that is responsible for the cleavage of terminal β-linked galactose residues from glycoproteins, sphingolipids, keratan sulfate, and other glycoconjugates. The loss of GLB1 in physiological systems will result in autosomal recessive lysosomal storage diseases such as GM1 gangliosidosis and Morquio B disease (Suzuki <I>et al.</I>, 2001). 

Over hundreds of single nucleotide mutations have been implicated in this disease. One of the challenges in building a disease model lies in manipulating GLB1 without affecting normal cellular function since it is an endogenous housekeeping gene. The GLB1-deficient fibroblasts are not easily available for experimental work. Moreover, while the E. coli β-galactosidase gene <I>LacZ</I> is widely used as a reporter in mammalian systems, this bacterial ortholog cannot be reliably used to create accurate representations of human GLB1 related diseases caused by mutations in the gene (Naderian <I>et al.,</I> 2011). Therefore, we want to investigate if overexpression of exogenous GLB1 is possible in mammalian cell lines. Ideally, overexpressed GLB1 activity would be much higher than endogenous activity, leading to a high signal to noise ratio, without affecting the normal function of the cells. This would provide tools for researchers to study the effect of GLB1 mutation without knocking out endogenous GLB1 expression. We also generated T --&gt; C mutants that are reported to be found in GM1 galactosidase patients (Hofer <I>et al.</I>, 2010) to investigate if the loss of enzymatic function is due to enzyme expression and degradation, or enzyme inactivation. We hope this overexpression system can be used to test the ability of Cas13b-APOBEC which is designed to perform C-&gt; U base editing to restore the function of GLB1. This would shed light on the potential of RNA editing to treat diseases such as those involving GLB1 mutations. 
</DIV><DIV class="numberedsection">
1. Construction of EGFP-GLB1 reporter plasmid
</DIV><DIV class="text">
We amplified the flag epitope GLB1 from Genscript GLB1 cDNA ORF clone (<A href=" https://www.ncbi.nlm.nih.gov/nuccore/NM_000404.3" style="text-decoration:none;font-thickenss:normal;">NM_000404.3</A>) using PCR, then cloned it into pSB1C3 (<A href="http://parts.igem.org/Part:BBa_K2807014">BBa_K2807014</A>) and a mammalian expression vector, pEGFP-C1. Subsequently, we carried out point mutations to the wild-type GLB1 gene to generate GLB1 TCT mutant (<A href="http://parts.igem.org/Part:BBa_K2807016">BBa_K2807016</A>) and GLB1 CCG mutant (<A href="http://parts.igem.org/Part:BBa_K2807015">BBa_K2807015</A>) which are point mutations that are reported to reduce GLB1 enzymatic activity by more than 95% in patients suffered from GM1 gangliosidosis (Hofer <I>et al.</I>, 2010). To allow normalization of transfection and expression efficiency of GLB1 in cells, we retained EGFP in C1 vector. To eliminate the possibility of the fusion protein to affect enzymatic activity, we separated EGFP and GLB1 by a stop codon and frameshift linker sequence. We also included a Kozak sequence in front of GLB1 to allow effective ribosome binding to translate GLB1. 
</DIV><DIV class="numberedsection">
2. Full-length precursor GLB1 and its mutant can be overexpressed in mammalian cells
</DIV><DIV class="text">
To verify the expression of GLB1 WT and mutants, we expressed both plasmids in HEK293T cells and then tested for the protein expression levels by Western blot. Cell lysates were separated on an SDS-PAGE gel, and transferred onto a nitrocellulose membrane for blotting. The plasmids with GLB1 had a Flag epitope tag as well as an EGFP tag preceding the GLB1 coding sequence. We probed the membrane with anti-EGFP, anti-Flag and anti-Actin antibodies. Figure 1 showed that GLB1 protein is expressed in both GLB1 WT and the GLB mutants at 84kDa, the expected size of GLB1 precursor, with the non-transfected lane as the negative control. In addition, a 27kDa band corresponding to EGFP was seen for both GLB1 WT and the GLB1 mutants, but not the non-transfected control, confirming our transfection efficiency. Therefore, we can conclude that our WT and mutant EGFP-GLB1 construct are able to express both EGFP and precursor GLB1 at the correct length. Moreover, we confirmed that the reported single nucleotide mutation in GLB1 does not change gene translation level or degradation rate. As such, any change in apparent enzymatic activity of the mutant GLB1 is likely due to a change in the catalytic efficiency of the enzyme but not the amount of enzyme. 
</DIV><FIGURE class="center"><I><STRONG>Figure 1. Western Blot analysis of cell lysates from transfected HEK293T cells expressing GLB1, GLB1 CCG mutant and GLB1 TCT mutant.</STRONG> Cell lysates were run on SDS-PAGE and probed using antibodies against EGFP, Flag epitope tag and Actin. Anti-actin antibodies were used to probe for the housekeeping gene actin in order to ensure equal protein loading across samples.
 </I></FIGURE><DIV class="numberedsection">
3. Precursor GLB1 is not enzymatically active
</DIV><DIV class="text">
Next, we wanted to investigate if the precursor form of GLB1 is enzymatically active by itself. We transfected EGFP-GLB1 plasmid into HEK293T cells and carried out β-galactosidase functional assay using cell lysate. To optimise the method of cell lysis for obtaining the β-galactosidase protein, we tried three methods, namely Promega Passive Lysis Buffer, 1% Triton and freeze-Thaw method using dry ice. Upon running a Western blot with the lysates from the varying methods, we could see that the Promega passive lysis buffer produced the least number of non-specific bands as well as the correct band size for GLB1 at about 84kDa as seen in Figure 2. As such, this lysis buffer was used for subsequent enzymatic assays. 
</DIV><FIGURE class="center"><I><STRONG>Figure 2. Western Blot analysis of cell lysates from transfected HEK293T cells expressing GLB1, GLB1 CCG mutant and GLB1 TCT mutant using Promega Lysis Buffer, 1% Triton and freeze-Thaw method.</STRONG> The cell lysates were run on SDS-PAGE and probed using antibodies against Flag epitope tag to verify the expression levels of GLB1 protein (~84kDa). </I></FIGURE><DIV class="text">
We carried out two β-galactosidase assays, using  galactoside analog o-nitrophenylβ- D-galactopyranoside (ONPG) and 5-bromo-4-chloro-3-indolyl-β-D-galactopyranoside (X-gal) as the substrate respectively. Functional β-galactosidase converts the colourless galactoside analog o-nitrophenylβ-D-galactopyranoside (ONPG) into galactose and chromophore o-nitrophenol, producing a bright yellow product. The final product can then be quantified using a spectrophotometer at 420 nm to determine the amount of substrate converted into the yellow chromophore. On the other hand, GLB1 cleaves X-gal to produce bright blue 5-bromo-4 -chloro-3-hydroxyindole and galactose as products. The appearance of the blue colour indicates that the sample contains a functional β-galactosidase enzyme, while a lack of blue colouration will be expected for the GLB1 mutants. 

As shown in Figure 3, we incubated ONPG with cell lysates from pGEMT expressing DH5α with 6hr or 24hr IPTG induction (Tubes A and B) as positive control, HEK293T transfected with empty vector (Tube C), HEK293 cells transfected with GLB1 WT (Tubes D) and GLB1 TCT and GLB1 CCG mutants (Tubes E and F). The absorbance reading at 420nm is shown in Table 1. Tubes A and B (Figure 3), our positive controls, had significant β-galactosidase activity. However, for Tubes C to F (Figure 3), no colour change was observed. Taking into consideration that GLB1 is a lysosomal enzyme with optimal pH at 4, we repeated the experiment at pH=4.5 and there was no observable colour change.  Similar results were observed in the dot activity assay using X-gal, where both non-transfected cells, wild-type and mutant GLB1 produced very faint blue colouration (Figure 4C-F) as compared to the positive control (Figure 4A-B). 

The above results confirmed that precursor β-galactosidase is not functional, as post-translational modification via proteolytic cleavage at the C-terminal of the GLB1 precursor protein in the acidic environment of the lysosome is required for maturation of beta-galactosidase (Kreutzer <I>et al.</I>, 2008). Therefore, in the future, we will engineer EGFP-GLB1 plasmid to include the cleavage signal to produce 64kDa and 22kDa proteolytic fragment that makes up the mature enzyme. 
</DIV><FIGURE class="center"><I><STRONG>Figure 3. Representative image of reaction tubes containing ONPG with respective cell lysates.</STRONG> (A) IPTG induced DH5α cell lysate (6 hours incubation), (B)  IPTG induced DH5α cell lysate (24 hours incubation), (C) GLB1 CCG mutant transfected HEK293T cell lysate, (D) GLB1 TCT mutant transfected HEK293T cell lysate, (E) GLB1 wild type transfected HEK293T cell lysate (6 hours incubation), (F) GLB1 wild type transfected HEK293T cell lysate (24 hours incubation). </I></FIGURE><TABLE style="width:450px" class="center_table"><CAPTION style="font-size:13px;"><STRONG> Table 1. Absorbance at 420nm for ONPG beta-galactosidase assay. The 6hr and 24hr IPTG induction serves as the positive control, while the non-transfected sample serves as the negative control. </STRONG></CAPTION><TBODY><TR><TH> Sample Name </TH><TH> Abs1 </TH><TH> Abs2 </TH></TR><TR><TD>Non-transfected</TD><TD>0.291</TD><TD>0.202</TD></TR><TR><TD>EGFP-GLB1 WT</TD><TD>0.200</TD><TD>0.202</TD></TR><TR><TD>EGFP-GLB1 CCG</TD><TD>0.130</TD><TD>0.128</TD></TR><TR><TD>EGFP-GLB1-TCT</TD><TD>0.135</TD><TD>0.132</TD></TR><TR><TD>Arith. Mean</TD><TD>0.103</TD><TD>0.086</TD></TR><TR><TD>6hr IPTG induction</TD><TD>1.307</TD><TD>1.304</TD></TR><TR><TD>24hr IPTG induction</TD><TD>1.259</TD><TD>1.252</TD></TR></TBODY></TABLE><FIGURE class="center"><I><STRONG>Figure 4: Enzymatic activity on nitrocellulose membranes using dot-blot method upon incubation with X-Gal.</STRONG> (A) IPTG induced DH5α cell lysate (6 hours incubation), (B)  IPTG induced DH5α cell lysate (24 hours incubation), (C) empty vector transfected HEK293T cell lysate (6 hours incubation), (D) GLB1 wild-type transfected HEK293T cell lysate (24 hours incubation), (E) GLB1 CCG mutant transfected HEK293T cell lysate, (F) GLB1 TCT mutant transfected HEK293T cell lysate. </I></FIGURE><DIV class="section">
References
</DIV><DIV class="references">
Kreutzer, R., Kreutzer, M., Pröpsting, M. J., Sewell, A. C., Leeb, T., Naim, H. Y., &amp; Baumgärtner, W. (2008). Insights into the post-translational processing of β-galactosidase in an animal model resembling late infantile human GM1-gangliosidosis. <I>Journal of Cellular and Molecular Medicine, 12</I>(5a), 1661–1671. http://doi.org/10.1111/j.1582-4934.2007.00204.x
</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>