<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NUS_Singapore-Sci_GLB_Logbook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NUS Singapore-Sci/GLB Logbook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV class="main"><UL class="mainnav"><LI class="hassubs"><A href="#" style="text-decoration:none;color:white;">Project</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Overview">Project Overview</A></LI><LI class="subs hassubs"><A href="">Reporter System</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Reporter_Overview">Reporter Overview</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Demonstrate">Dual Reporter - Demonstration</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Improve">Dual Reporter - Improvement</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Reporter_GLB">GLB1 Reporter</A></LI></UL><LI class="subs hassubs"><A href="">Editor System</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Editor_Overview">Editor Overview</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Editor_Results">Editor Results</A></LI><LI class="subs hassubs"><A href="">Modelling</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Model">Modelling Overview</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/enkf">Kalman Filter Ensemble</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/molecular_dynamics">Molecular Dynamics</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/enzyme_kinetics">Enzyme Kinetics</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/grna_design">gRNA Design</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/InterLab">InterLab</A></LI><LI class="hassubs"><A href="#" style="text-decoration:none;color:white;">Human Practice</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Human_Practices">Overview of Human Practice</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Public_Engagement">Public Engagement</A></LI><LI class="subs hassubs"><A href="#">#CasAsks</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Cas_Asks_Survey">#CasAsks: Survey</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Cas_Asks_Interview">#CasAsks: Interview</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Cas_Talks">#CasTalks</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Cas_Teaches">#CasTeaches</A></LI><LI class="subs hassubs"><A href="#">#CasWrites</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Cas_Writes">#CasWrites: Discussion</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Cas_Writes_Collab">#CasWrites: Collaboration</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Cas_Writes_Bioethics">#CasWrites: BioEthics</A></LI><LI><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Collaborations" style="text-decoration:none;color:white;">Collaborations</A></LI><LI class="hassubs"><A href="#" style="text-decoration:none;color:white;">Notebook</A></LI><LI class="subs hassubs"><A href="#">Reporter Logbooks</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Dual_Logbook">Dual Reporter Logbook</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/GLB_Logbook">GLB1 Reporter Logbook</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Editor_Logbook">Editor Logbook</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/General_Docs#">Primers and Protocols</A></LI><LI class="hassubs"><A href="#" style="text-decoration:none;color:white;">People</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Team">Team</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Attributions">Attributions</A></LI><LI class="hassubs"><A href="#" style="text-decoration:none;color:white;">Admin</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Safety">Safety</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Medal">Medals</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Parts">Parts</A></LI></DIV><TITLE> NUS Singapore Science: InterLab </TITLE><H1 id="title"><DIV style="font-size: 2.2em; color: #C0392B"> GLB1 Reporter </DIV><SPAN style="font-size: 2em; color: #000080"> Logbook </SPAN></H1><DIV class="numberedsection">
1. Cloning of the GLB1 from Genescript Plasmid
</DIV><DIV class="text">
GLB1 is amplified from the template (Genscript plasmid, pCDNA3.1+C(-K)-DYK) using Q5 DNA polymerase (NEB) via PCR (Figure 1). The PCR protocol used is shown below, the primers used are GLB-HindIII-F and GLB-KpnI-R with annealing temperatures of 56℃, 57℃ and 58℃, and extension time of 1 min. The two primers used during PCR amplification also introduced additional HindIII and KpnI restriction sites at the front and back of GLB1 gene for downstream cloning. This GLB1 construct is addressed as GLB1 WT in subsequent experiments and this construct will be cloned into pEGFP-C1 for mammalian expression and pSB1C3 for parts submission.
</DIV><FIGURE class="center"><I><STRONG>Figure 1. PCR amplification of GLB1.</STRONG> GLB1 containing HindIII and KpnI restriction sites (in red box) are amplified during PCR and will be known as GLB1 WT. The size of GLB1 WTI is ~2218 bp and is successfully amplified in all PCR reactions. Lane 1 and 2 are performed under an annealing temperature of 56℃, lane 3 and 4 of 57℃, and lane 5 and 6 of 58℃. The top DNA band (in blue box) represents the original plasmid construct, pCDNA3.1+C(-K)-DYK, from Genscript with a size of ~7400 bp. </I></FIGURE><DIV class="text">
After PCR amplification, GLB1 WT is digested with restriction enzymes HindIII and KpnI, followed by ligation into pEGFP-C1 plasmid for mammalian expression and transformation into DH5α. Twelve colonies were picked for colony PCR using <I>Taq</I> polymerase to ascertain the success of bacterial transformation (Figure 2). GLB-HindIII-F and GLB-KpnI-R are used as primers for colony PCR, with an annealing temperature of 57℃ and extension time of 2 min. Figure 2 shows that colonies 1, 8 and 10 have the right insert size (approximately 2218 bp). Colonies 1, 8 and 10 were then inoculated in overnight LB cultures to isolate the plasmid and the plasmids were verified via restriction digest. 
</DIV><FIGURE class="center"><I><STRONG>Figure 2. Colony PCR for GLB1 in pEGFGP-C1.</STRONG>Twelve colonies were selected for colony PCR after bacterial transformation. Only colonies 1, 8 and 10 contained the GLB1 WT insert that is ~2218 bp (in red box). </I></FIGURE><DIV class="numberedsection">
2. Production of GLB1 mutant plasmid
</DIV><DIV class="text">
Two sets of point mutations were introduced into GLB1 WT for the construction of the pEGFP-GLB1 reporter system. The first point mutant was to change the amino acid (I389-&gt;P389). This (I-&gt; P) mutation results in a 99.5% reduction in GLB gene expression. The second point mutant was created by substituting the amino acid at position 107 from phenylalanine to leucine (F107-&gt;L107), leading to 1.7% of WT ꞵ-galactosidase expression. 

Mutants were created by introducing mutations into GLB1 WT using PCR based mutagenesis, with CCGF1, CCGR1, TCTF, TCTR, GLB-HindIII-F and GLB-KpnI-R primers (Figure 3). Four different annealing temperatures: 69℃, 70℃, 71℃ and 72℃ were used. An extension time of 1 min was used for all four constructs. All bands were extracted to create their respective mutants. All reactions, except lane 12, produced the desired DNA bands. Lanes 1-4 was amplified by CCGF1 and GLB-KpnI-R to produce a band size of ~1034 bp.
</DIV><FIGURE class="center"><I><STRONG>Figure 3.  PCR amplification of GLB1 mutant fragments.</STRONG> Each reaction is performed at four different annealing temperatures: 69℃, 70℃, 71℃ and 72℃. All reactions, except lane 12, produced the desired DNA bands. Lanes 1-4 was amplified by CCGF1 and GLB-KpnI-R to produce a band size of ~1034 bp. Lanes 5-8 was amplified  by CCGR1 and GLB-HindIII-F to produce a band size of ~1211 bp. Lanes 9-12 was amplified by TCTF and GLB-KpnI-R to produce a band size of ~1883 bp. Lanes 12-16 was amplified by TCTR and GLB-HindIII-F to produce a band size of ~362 bp. All bands are extracted for further cloning experiments. </I></FIGURE><DIV class="text">
Subsequently, gel bands extracted from lanes 1-8 were put together to obtain GLB1 CCG-mut (Figure 4). DNA extracted from lanes 9-16 were put together to obtain GLB1 TCT-mut (Figure 5). Four annealing temperatures: 69℃, 70℃, 71℃ and 72℃ were used to create both mutants during PCR. Both mutated genes were cloned into pEGFP-C1 plasmid.
</DIV><FIGURE class="center"><I><STRONG>Figure 4. Production of GLB1 CCG-mut.</STRONG> All lanes showed the correct GLB1 CCG-mut  band size of ~2218 bp (in red box). The annealing temperatures used were 69℃, 70℃, 71℃ and 72℃ (from lanes 1-4). </I></FIGURE><FIGURE class="center"><I><STRONG>Figure 5. Production of GLB1 TCT-mut. </STRONG> All lanes showed the correct GLB1 TCT-mut band sizeof ~2218 bp (in red box). The annealing temperatures used were 69℃, 70℃, 71℃ and 72℃ (from lanes 1-4). </I></FIGURE><DIV class="text">
GLB1 mutants were then transformed into DH5α and isolated for DNA sequencing. Sequencing results showed that both GLB1 mutants have the correct sites mutated. Subsequently, GLB1 mutants were transfected into HEK293T cells for characterization. Concurrently, illegal sites which included 3 PstI sites and an EcoRI site in the GLB1 mutants were also removed by PCR using the respective primers (the list of primers is available <A href="https://2018.igem.org/Team:NUS_Singapore-Sci/General_Docs" style="text-decoration:none;font-weight:normal;">here</A>). 
</DIV><DIV class="numberedsection">
3. Cloning of GLB1 and mutants into pSB1C3
</DIV><DIV class="text">
Both the Biobrick prefix and suffix sites were added into EGFP-GLB1 insert by PCR and then cloned into pSB1C3 for parts submission. The prefix and suffix sites were introduced into all EGFP-GLB1 variants using PCR based mutagenesis with EcoRI Prefix F and SpeI Suffix R primers (Figure 6). An annealing temperature of 69℃ and an extension time of 1 min were used for all three variants. All bands of size 2935 bp were excised and purified for cloning into pSB1C3 vector subsequently. 
</DIV><FIGURE class="center"><I><STRONG>Figure 6. PCR amplification of a) eGFP-GLB1 WT fragments and b) eGFP-GLB1 TCT-mut and CCG-mut fragments.</STRONG>Each reaction was performed at an annealing temperature of 69℃. All reactions produced the desired DNA bands. All lanes were amplified by EcoRI Prefix F and SpeI Suffix R to produce a band size of ~2935 bp. The upper band size of ~6922 bp represents pEGP-C1 plasmid containing GLB1 variants. </I></FIGURE><DIV class="text">
The EGFP-GLB1 containing pSB1C3 vector plasmid was transformed into DH5α competent cells. Colonies were selected and plasmids were isolated by miniprep for both diagnostic restriction enzyme (RE) digestion using EcoRI and SpeI (Figure 7), as well as DNA sequencing. Both RE digestion and sequencing results verified the presence of EGFP-GLB1 in pSB1C3.
</DIV><FIGURE class="center"><I><STRONG>Figure 7. Restriction digestion of a) EGFP-GLB1 WT plasmid and b) EGFP-GLB1 TCT-mut and CCG-mut plasmid.</STRONG>Both EGFP-GLB1 variants were digested with EcoRI-HF and SpeI to give two distinct bands with size  ~2935 bp (in blue box) and ~2070 bp (in red box). This shows the presence of EGFP-GLB1 in pSB1C3. </I></FIGURE><DIV class="numberedsection">
4. Checking expression of Wildtype and mutant GLB1 in HEK293T
</DIV><DIV class="text">
Cultured HEK293T mammalian cells were transfected with 2 ug of GLB1 WT construct, GLB1 with TCT and CCG mutant plasmids in a 6 well plate, and expressed for 24hrs. Cells were harvested in cold 1X PBS using cell lifter, centrifuged and 1) lysed in 1% Triton in PBS or Promega passive lysis buffer, and incubated on ice for 20 minutes, or 2) subjected to 3 freeze-thaw cycles using dry ice. The cell lysate was centrifuged at 14000 rcf for 10 minutes at 4℃  to remove cell debris. Equal amount of cell lysates were loaded onto an SDS-PAGE gel, transferred to a nitrocellulose membrane, and blotted for FLAG-tag, EGFP and beta-actin (detailed protocol can be found <A href="https://2018.igem.org/Team:NUS_Singapore-Sci/General_Docs" style="text-decoration:none;font-weight:normal;">here</A>). The results of the western blot were shown in Figure 8. Since Promega passive lysis buffer gave the least number of unspecific bands, it was used as the lysis buffer of choice for the subsequent experiments. 
</DIV><FIGURE class="center"><I><STRONG>Figure 8. Western Blot analysis of cell lysates from transfected HEK293T cells expressing GLB1, GLB1 CCG mutant and GLB1 TCT mutant using Promega Lysis Buffer, 1% Triton and freeze-Thaw method.</STRONG>he cell lysates were run on SDS-PAGE and probed using antibodies against Flag epitope tag to verify the expression levels of GLB1 protein (~84kDa). </I></FIGURE><DIV class="numberedsection">
5. β-Galactosidase Assay in HEK293T cell lysates
</DIV><DIV class="text">
HEK293T were cultured in 100mm dishes and transfected with plasmid expressing the WT-GLB1 and the TCT and CCG mutant plasmids. Cells were harvested 24hr post transfection and lysed in 100ul Promega Passive Lysis buffer.  25ul of lysate was used in ONPG β-galactosidase assay while 10ul of lysate was used in dot blot activity assay using X-gal as substrate. The detailed protocol can be found <A href="https://2018.igem.org/Team:NUS_Singapore-Sci/General_Docs" style="text-decoration:none;font-weight:normal;">here</A>. For ONPG assay, enzymatic activity was quantified using absorbance at 420 nm (Table 1, Figure 9). For X-gal assay, a picture was taken after 24hr of incubation (Figure 10). In both assays, only the positive control (<I>E. coli</I> cells expressing pGEMT plasmid incubated with IPTG) show β-galactosidase activity. 
</DIV><FIGURE class="center"><I><STRONG>Figure 9. Representative image of reaction tubes containing ONPG with respective cell lysates.</STRONG>(A) IPTG induced DH5α cell lysate (6 hours incubation), (B)  IPTG induced DH5α cell lysate (24 hours incubation), (C) GLB1 CCG mutant transfected HEK293T cell lysate, (D) GLB1 TCT mutant transfected HEK293T cell lysate, (E) GLB1 wild type transfected HEK293T cell lysate (6 hours incubation), (F) GLB1 wild type transfected HEK293T cell lysate (24 hours incubation). </I></FIGURE><TABLE style="width:450px" class="center_table"><CAPTION style="font-size:15px;"><STRONG><I>Table 1. Absorbance at 420 nm for ONPG beta-galactosidase assay. The 6hr and 24hr IPTG induction serves as the positive control, while the non-transfected sample serves as the negative control. 
</I></STRONG></CAPTION><TBODY><TR><TH>Sample Name</TH><TH>Abs1</TH><TH>Abs2</TH></TR><TR><TD>Non-transfected</TD><TD>0.291</TD><TD>0.202</TD></TR><TR><TD>EGFP-GLB1 WT</TD><TD>0.200</TD><TD>0.202</TD></TR><TR><TD>EGFP-GLB1 CCG</TD><TD>0.130</TD><TD>0.128</TD></TR><TR><TD>EGFP-GLB1-TCT</TD><TD>0.135</TD><TD>0.132</TD></TR><TR><TD>6hr IPTG induction</TD><TD>1.307</TD><TD>1.304</TD></TR><TR><TD>24hr IPTG induction</TD><TD>1.259</TD><TD>1.252</TD></TR></TBODY></TABLE><FIGURE class="center"><I><STRONG>Figure 10.  Enzymatic activity on nitrocellulose membranes using dot-blot method upon incubation with X-Gal.</STRONG>(A) IPTG induced DH5α cell lysate (6 hours incubation), (B)  IPTG induced DH5α cell lysate (24 hours incubation), (C) empty vector transfected HEK293T cell lysate (6 hours incubation), (D) GLB1 wild-type transfected HEK293T cell lysate (24 hours incubation), (E) GLB1 CCG mutant transfected HEK293T cell lysate, (F) GLB1 TCT mutant transfected HEK293T cell lysate. </I></FIGURE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>