<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NUS_Singapore-Sci_InterLab skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NUS Singapore-Sci/InterLab</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV class="main"><UL class="mainnav"><LI class="hassubs"><A href="#" style="text-decoration:none;color:white;">Project</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Overview">Project Overview</A></LI><LI class="subs hassubs"><A href="">Reporter System</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Reporter_Overview">Reporter Overview</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Demonstrate">Dual Reporter - Demonstration</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Improve">Dual Reporter - Improvement</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Reporter_GLB">GLB1 Reporter</A></LI></UL><LI class="subs hassubs"><A href="">Editor System</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Editor_Overview">Editor Overview</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Editor_Results">Editor Results</A></LI><LI class="subs hassubs"><A href="">Modelling</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Model">Modelling Overview</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/enkf">Kalman Filter Ensemble</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/molecular_dynamics">Molecular Dynamics</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/enzyme_kinetics">Enzyme Kinetics</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/grna_design">gRNA Design</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/InterLab">InterLab</A></LI><LI class="hassubs"><A href="#" style="text-decoration:none;color:white;">Human Practice</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Human_Practices">Overview of Human Practice</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Public_Engagement">Public Engagement</A></LI><LI class="subs hassubs"><A href="#">#CasAsks</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Cas_Asks_Survey">#CasAsks: Survey</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Cas_Asks_Interview">#CasAsks: Interview</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Cas_Talks">#CasTalks</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Cas_Teaches">#CasTeaches</A></LI><LI class="subs hassubs"><A href="#">#CasWrites</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Cas_Writes">#CasWrites: Discussion</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Cas_Writes_Collab">#CasWrites: Collaboration</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Cas_Writes_Bioethics">#CasWrites: BioEthics</A></LI><LI><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Collaborations" style="text-decoration:none;color:white;">Collaborations</A></LI><LI class="hassubs"><A href="#" style="text-decoration:none;color:white;">Notebook</A></LI><LI class="subs hassubs"><A href="#">Reporter Logbooks</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Dual_Logbook">Dual Reporter Logbook</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/GLB_Logbook">GLB1 Reporter Logbook</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Editor_Logbook">Editor Logbook</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/General_Docs#">Primers and Protocols</A></LI><LI class="hassubs"><A href="#" style="text-decoration:none;color:white;">People</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Team">Team</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Attributions">Attributions</A></LI><LI class="hassubs"><A href="#" style="text-decoration:none;color:white;">Admin</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Safety">Safety</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Medal">Medals</A></LI><LI class="subs"><A href="https://2018.igem.org/Team:NUS_Singapore-Sci/Parts">Parts</A></LI></DIV><TITLE> NUS Singapore Science: InterLab </TITLE><H1 id="title"><DIV style="font-size: 2.2em; color: #C0392B"> InterLab </DIV><SPAN style="font-size: 2em; color: #000080"> Study </SPAN></H1><DIV class="outline">
This year, Team NUS Singapore-Science took part in the 2018 InterLab Study, which aims to standardise practises for the measurement of gene expression from standard biobrick devices in a standard host organism. <STRONG>The goal for this year’s InterLab study was to see if the level of green fluorescent protein (GFP) reporter expression correlates with the actual number of transformed bacteria growth (as measured using optical density at 600 nm).</STRONG></DIV><DIV class="numberedsection"><STRONG> 1. Transformation and Inoculation of <I>E. coli strain DH5ɑ</I> cells</STRONG></DIV><DIV class="text">
We performed this study using the protocol from iGEM HQ and the level of GFP expression was measured using a fluorescence microplate reader (Synergy H1 Hybrid Multi-Mode Reader). Competent cells (<I>Escherichia coli strain DH5ɑ</I>) were transformed with the following plasmids (devices) as shown in Table 1. The transformation protocol was performed as recommended by iGEM HQ. Transformants were grown overnight on Luria Broth (LB) agar plates with chloramphenicol. The next day, two colonies from each plate were picked and inoculated into 5 mL of LB medium with chloramphenicol (25 mg/mL). They were then cultured overnight at 37℃ in a shaking incubator. 
</DIV><DIV class="section">
Sample Preparation
</DIV><DIV class="text">
To prepare the <I>E. coli strain DH5ɑ</I> cells for the cell measurement protocol, a 1:10 dilution was made for each overnight culture in LB medium with Chloramphenicol, and the absorbance at 600 nm (Abs<SUB>600</SUB>) of the diluted cultures were measured. The 1:10 diluted cultures were further diluted to a target Abs<SUB>600</SUB> of 0.02 in 12 mL using the formula below:
</DIV><DIV class="formula">
Volume of 1:10 diluted cultures to add = 0.02 * 12 mL/Abs<SUB>600</SUB></DIV><DIV class="text">
Prior to incubation (t = 0 hours), 0.5 mL of each culture was transferred to a 1.5 mL tube. The tubes were placed on ice. 0.1 mL of each culture was pipetted into a black 96-well plate with clear flat bottom. Four replicates for each culture were made. Eight control wells containing 0.1 mL of LB medium with chloramphenicol were also included. The fluorescence level at 525 nm and Abs<SUB>600</SUB> was measured at t = 0 hours and t = 6 hours.The remaining cultures were incubated for 6 hours at 37℃ in the shaking incubator. At the end of 6-hour incubation, 0.5 mL of each bacterial culture was again transferred to 1.5 mL tubes. Similarly, 0.1 mL of each culture was pipetted into a new black 96-well plate with clear flat bottom. Four replicates per culture were used, and the fluorescence level and absorbance were measured. 

The microplate reader used was Synergy H1 Hybrid Multi-Mode Reader (courtesy of National University of Singapore Synthetic Biology for Clinical and Technological Innovation (SynCTI)), which could measure both absorbance and fluorescence. For all the Abs<SUB>600</SUB> and fluorescence measurements, pathlength correction was turned off, and the temperature setting kept to room temperature (20-25°C). The bandpass width is fixed at 16 nm and bottom optics were used.
</DIV><TABLE style="width:450px" class="center_table"><CAPTION style="font-size:15px;"><STRONG><I>Table 1. Biobrick Devices as obtained from iGEM HQ. These devices were transformed into E. coli DH5ɑ cells.</I></STRONG></CAPTION><TBODY><TR><TH>Device</TH><TH>Partnumber</TH></TR><TR><TD>Negative control</TD><TD>BBa_R0040</TD></TR><TR><TD>Positive control</TD><TD>BBa_I20270</TD></TR><TR><TD>Test Device 1</TD><TD>BBa_J364000</TD></TR><TR><TD>Test Device 2</TD><TD>BBa_J364001</TD></TR><TR><TD>Test Device 3</TD><TD>BBa_J364002</TD></TR><TR><TD>Test Device 4</TD><TD>BBa_J364007</TD></TR><TR><TD>Test Device 5</TD><TD>BBa_J364008</TD></TR><TR><TD>Test Device 6</TD><TD>BBa_J364009</TD></TR></TBODY></TABLE><DIV class="section">
   Calibration procedure
</DIV><DIV class="text">
Before the execution of the actual measurement experiment, three different calibration protocols were carried out to obtain: 1) an optical density (OD<SUB>600</SUB>) reference point, 2) a particle standard curve for the number of particles at a certain Abs<SUB>600</SUB> and 3) a fluorescein standard curve for fluorescence intensity.
</DIV><DIV class="numberedsection"><STRONG> 2. OD<SUB>600</SUB> Reference Point - LUDOX Protocol </STRONG></DIV><DIV class="text">
This calibration protocol was done to obtain a conversion factor for Abs<SUB>600</SUB> as measured by the plate reader to an OD<SUB>600</SUB> measurement. LUDOX CL-X (45% colloidal silica suspension) was used, and the LUDOX solution will give a low absorbance value as it is weakly scattering. Here, pathlength correction was switched off so as to not provide correction for the volume of sample in the 96-well plate.
</DIV><TABLE style="width:450px" class="center_table"><CAPTION style="font-size:13px;"><STRONG><I>Table 2. Results of Abs<SUB>600</SUB> of LUDOX CL-X and H<SUB>2</SUB>O. Corrected Abs values were used to calculate the conversion factor from Abs<SUB>600</SUB> to OD<SUB>600</SUB>.</I></STRONG></CAPTION><TBODY><TR><TH>LUDOX CL-X</TH><TH> H<SUB>2</SUB>O </TH></TR><TR><TD>Replicate 1</TD><TD>0.101</TD><TD>0.084</TD></TR><TR><TD>Replicate 2</TD><TD>0.100</TD><TD>0.086</TD></TR><TR><TD>Replicate 3</TD><TD>0.103</TD><TD>0.085</TD></TR><TR><TD>Replicate 4</TD><TD>0.106</TD><TD>0.087</TD></TR><TR><TD>Arith. Mean</TD><TD>0.103</TD><TD>0.086</TD></TR><TR><TD>Corrected ABS</TD><TD>0.017</TD></TR><TR><TD>Reference Abs</TD><TD>0.063</TD></TR><TR><TD>OD<SUB>600</SUB>/Abs<SUB>600</SUB></TD><TD>3.706</TD></TR></TBODY></TABLE><DIV class="text">
The conversion factor from an Abs<SUB>600</SUB> value to a OD<SUB>600</SUB> value is obtained by dividing the reference OD<SUB>600</SUB> value measured by a spectrophotometer by the mean correct Abs<SUB>600</SUB> value. As shown in Table 2, the conversion factor obtained is 3.706. Thus, Abs<SUB>600</SUB> measured by the plate reader can be converted to the OD<SUB>600</SUB> reading by multiplying the conversion factor, 3.706. Background correction for the Abs<SUB>600</SUB> value was done by subtracting the mean of four replicates with the mean Abs<SUB>600</SUB> value of H<SUB>2</SUB>O.

</DIV><DIV class="numberedsection"><STRONG> 3. Particle Standard Curve - Microsphere Protocol</STRONG></DIV><DIV class="text">
This calibration protocol was done to obtain the Abs<SUB>600</SUB> values for known concentrations of microspheres, which have simlar size and optical characteristics compared to <I>E. coli strain DH5ɑ</I> cells. In doing so, for a given Abs<SUB>600</SUB> value, we are able to approximate the number of <I>E. coli strain DH5ɑ</I> cells present in the well. A dilution series of monodisperse silica microspheres were prepared in the 96-well plates and the Abs<SUB>600</SUB> measured. Four replicates for each dilution were prepared.
</DIV><TABLE class="center_table"><CAPTION style="font-size:13px;"><STRONG><I>Table 3. Results of Abs600 for increasing microsphere particle numbers. As particle number increases, Abs600 values increases.</I></STRONG></CAPTION></TABLE><DIV class="text">
As shown in Table 3, for each microsphere particle number, four replicates were done. The mean of the four replicates were taken, and the standard deviation measured. The mean corrected Abs<SUB>600</SUB> values are obtained by subtracting the mean Abs<SUB>600</SUB> values from that of the blank which is double distilled H<SUB>2</SUB>O (ddH<SUB>2</SUB>O). From the last row of results as in Table 3 which gives the mean corrected Abs<SUB>600</SUB> values, a particle standard curve (Figure 1) is plotted. A graph of Abs<SUB>600</SUB> against particle count per 0.1 mL is shown in Figure 1. The particle standard curve will provide us with an estimate of the number of particles present in a solution given the Abs<SUB>600</SUB> value.
</DIV><FIGURE class="center"><I><STRONG>Figure 1. Standard curve of Abs600 values against particle number.</STRONG> Mean Abs<SUB>600</SUB> data was plotted against particle count per 0.1 mL for serial diluted samples of microspheres. </I></FIGURE><DIV class="text">
A log scale of the same particle standard curve in Figure 1 was also plotted, where the Abs<SUB>600</SUB> values and values for particle count per 0.1 mL were applied the logarithmic number to obtain the graph as in Figure 2. 
</DIV><FIGURE class="center"><I><STRONG>Figure 2. Standard curve of Abs<SUB>600</SUB> values against particle number on a logarithmic scale.</STRONG></I></FIGURE><DIV class="numberedsection"><STRONG>4. Fluorescence Standard Curve - Fluorescein Dye Protocol</STRONG></DIV><DIV class="text">
This calibration experiment was carried out to compare fluorescence values obtained from the test devices between teams, as different plate readers report different arbitrary units of fluorescence. Here, fluorescein dye is used in place of GFP as it has similar excitation and emission properties to GFP. The excitation wavelength used was 494 nm and the emission maximum used was 525 nm. Serial dilutions of fluorescein in 1x phosphate-buffered saline (PBS) were prepared in the 96-well plates and the fluorescence levels of the dye was measured (Table 4). This will enable us to obtain a standard fluorescence curve allows us to compare  fluorescence values across teams more accurately.
</DIV><TABLE class="center_table"><CAPTION style="font-size:13px;"><STRONG><I>Table 4. Results of fluorescence values against concentration of fluorescein. Fluorescence values increase with increasing concentrations of fluorescein.</I></STRONG></CAPTION></TABLE><DIV class="text">
Based on the raw fluorescence values obtained in Table 4, a fluorescein standard curve of fluorescence against fluorescein concentration was plotted (Figure 3). The fluorescence standard curve will provide us with an estimate of the amount of fluorescein (or GFP) present in a solution when given the fluorescence value.
</DIV><FIGURE class="center"><I><STRONG>Figure 3. Standard curve of fluorescence values at 525 nm against concentration of fluorescein (µM).</STRONG> Mean fluorescence value was plotted against fluorescein concentration for serial diluted samples of fluorescein.</I></FIGURE><DIV class="text">
A log scale of the fluorescence standard curve in Figure 3 was also plotted, where the fluorescence values and concentrations of fluorescein were applied the logarithmic number to obtain the graph as in Figure 4. 
</DIV><FIGURE class="center"><I><STRONG>Figure 4. Standard curve of fluorescence values at 525 nm against fluorescein concentration on a logarithmic scale. </STRONG></I></FIGURE><DIV class="text">
From the Fluorescein µM values, the Molecules of Equivalent Fluorescence (MEFL) can be calculated. MEFL refers to the number of cells present, assuming that each of them give off an equal level of fluorescence. This can be calculated using the following parameters: 

Initial Molarity: 1.00E-05 
Molecules/mol: 6.02E+23 
Well Volume (L): 1.00E-04 
Initial molecules: 6.02+14 
MEFL/µM: 6.02E+12
</DIV><TABLE class="center_table"><CAPTION style="font-size:13px;"><STRONG><I>Table 5. Calculations to obtain MEFL/a.u.</I></STRONG></CAPTION></TABLE><DIV class="numberedsection"><STRONG>5. Cell Measurement Protocol for GFP expression over time </STRONG></DIV><DIV class="text">
As mentioned above, at 0 hours and 6 hours, an aliquot of the culture was pipetted out and the fluorescence at 525 nm and Abs<SUB>600</SUB> values were measured accordingly. Four replicates were done for each of the two colonies per device. 
From the calibration protocols in sections 1-3, the unit scaling factors were calculated and shown in Table 6.
</DIV><TABLE style="width:450px" class="center_table"><CAPTION style="font-size:13px;"><STRONG><I>Table 6. Unit scaling factors obtained from the calibration protocols. </I></STRONG></CAPTION><TBODY><TR><TD>OD<SUB>600</SUB>/Abs<SUB>600</SUB></TD><TD>3.71</TD></TR><TR><TD>µM Fluorescein/a.u.</TD><TD>1.08E-04</TD></TR><TR><TD>Particles/Abs600</TD><TD>2.46E+08</TD></TR><TR><TD>MEFL/a.u.</TD><TD>6.50E+08</TD></TR></TBODY></TABLE><TABLE class="center_table"><CAPTION style="font-size:13px;"><STRONG><I>Table 7. Net fluorescence in a.u. at t = 0 h (above) and t = 6 h (below). </I></STRONG></CAPTION></TABLE><TABLE class="center_table"><CAPTION style="font-size:13px;"><STRONG><I>Table 8. Net Abs<SUB>600</SUB> at t = 0 h (above) and t = 6 h (below). </I></STRONG></CAPTION></TABLE><TABLE class="center_table"><CAPTION style="font-size:13px;"><STRONG><I>Table 9. µM Fluorescein/OD at t = 0 h (above) and t = 6 h (below). </I></STRONG></CAPTION></TABLE><TABLE class="center_table"><CAPTION style="font-size:13px;"><STRONG><I>Table 10. MEFL/particle at t = 0 h (above) and t = 6 h (below). </I></STRONG></CAPTION></TABLE><DIV class="numberedsection"><STRONG>6. Protocol: Colony Forming Units per 0.1 OD<SUB>600</SUB><I>E.coli</I> cultures</STRONG></DIV><DIV class="text">
This procedure was done to calibrate the OD<SUB>600</SUB> to give colony forming unit (CFU) counts, which can then be related to the bacteria concentration of the culture (CFU/mL). Under this protocol, the assumption is that one bacteria will give rise to one colony. This protocol only involves <I>E. coli</I> cultures that have been transformed with the Positive Control (BBa_I20270) or Negative Control (BBa_R0040) devices. Two colonies from each plate were taken and cultured overnight.
</DIV><DIV class="section">
Sample Preparation
</DIV><DIV class="text">
The next day, the cell cultures were diluted to the linear detection range of the plate reader by doing a 1:8 dilution. 25 µL of culture was added to 175 µL of LB broth with chloramphenicol in a black 96-well plate. The OD<SUB>600</SUB> of these cultures were measured. 

As the starting OD<SUB>600</SUB> should be standardised to 0.1 for all, the <I>E. coli</I> cultures were diluted with LB broth with Chloramphenicol according to the following equation:
</DIV><DIV class="formula">
C1 * V1 = C2* V2 
Where C1=(1:8 OD<SUB>600</SUB> - blank OD<SUB>600</SUB>)*8
</DIV><DIV class="text">
OD<SUB>600</SUB> values of the diluted cultures (starting samples) were measured again (in triplicates) to ensure that the OD<SUB>600</SUB> values are at approximately 0.1 after subtracting from the blank measurement. 

Serial dilutions for the starting samples were prepared as shown in Figure 5. For dilutions 3, 4 and 5, 0.1 mL of the bacterial cultures were spread on LB agar plates with chloramphenicol. The plates were then incubated at 37℃ overnight.
</DIV><FIGURE class="center"><I><STRONG>Figure 5. Serial dilutions for the starting samples were done, and dilutions 3,4 and 5 plated and incubated overnight. </STRONG></I></FIGURE><DIV class="text">
The next day, colonies were counted for plates with fewer than 300 colonies (Table 11). The CFU/mL values were calculated by multiplying the number of colonies with the appropriate dilution factor.
</DIV><TABLE class="center_table" style="width:450px;"><CAPTION style="font-size:13px;"><STRONG><I>Table 11. Number of colonies on each LB agar plate with chloramphenicol after overnight incubation (Too numerous to count, more than 300 colonies = TNTC). </I></STRONG></CAPTION><TBODY><TR><TH>Sample</TH><TH>No. of colonies (Dilution 3: 8*10^4)</TH><TH>No. of colonies (Dilution 4: 8*10^5)</TH><TH>No. of colonies (Dilution 5: 8*10^6)</TH><TH>Calculated CFU/mL of culture based on Dilution 4 (8*10^5)</TH></TR><TR><TD>Positive Control Colony 1 (1)</TD><TD>TNTC</TD><TD>110</TD><TD>2</TD><TD>8.8E+07</TD></TR><TR><TD>Positive Control Colony 1 (2)</TD><TD>TNTC</TD><TD>127</TD><TD>12</TD><TD>10.16E+07</TD></TR><TR><TD>Positive Control Colony 1 (3)</TD><TD>TNTC</TD><TD>88</TD><TD>7</TD><TD>7.04E+07</TD></TR><TR><TD>Positive Control Colony 2 (1)</TD><TD>TNTC</TD><TD>168</TD><TD>8</TD><TD>13.44E+07</TD></TR><TR><TD>Positive Control Colony 2 (2)</TD><TD>TNTC</TD><TD>148</TD><TD>2</TD><TD>11.84E+07</TD></TR><TR><TD>Positive Control Colony 2 (3)</TD><TD>TNTC</TD><TD>136</TD><TD>3</TD><TD>10.88E+07</TD></TR><TR><TD>Negative Control Colony 1 (1)</TD><TD>TNTC</TD><TD>101</TD><TD>1</TD><TD>8.08E+07</TD></TR><TR><TD>Negative Control Colony 1 (2)</TD><TD>TNTC</TD><TD>145</TD><TD>13</TD><TD>11.6E+07</TD></TR><TR><TD>Negative Control Colony 1 (3)</TD><TD>TNTC</TD><TD>169</TD><TD>11</TD><TD>13.52E+07</TD></TR><TR><TD>Negative Control Colony 2 (1)</TD><TD>TNTC</TD><TD>132</TD><TD>15</TD><TD>10.56E+07</TD></TR><TR><TD>Negative Control Colony 2 (2)</TD><TD>TNTC</TD><TD>143</TD><TD>18</TD><TD>11.44E+07</TD></TR><TR><TD>Negative Control Colony 2 (3)</TD><TD>TNTC</TD><TD>62</TD><TD>4</TD><TD>4.96E+07</TD></TR></TBODY></TABLE><DIV class="numberedsection"><STRONG> 7. Overall Evaluation </STRONG></DIV><DIV class="text">
Overall, our team had no issues with the InterLab protocols.
</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>