Team:NUS Singapore-Sci/Editor Logbook
Project
Project Overview
Reporter System
Reporter Overview
Dual Reporter - Demonstration
Dual Reporter - Improvement
GLB1 Reporter
Editor System
Editor Overview
Editor Results
Modelling
Modelling Overview
Kalman Filter Ensemble
Molecular Dynamics
Enzyme Kinetics
gRNA Design
InterLab
Human Practice
Overview of Human Practice
Public Engagement
#CasAsks
#CasAsks: Survey
#CasAsks: Interview
#CasTalks
#CasTeaches
#CasWrites
#CasWrites: Discussion
#CasWrites: Collaboration
#CasWrites: BioEthics
Collaborations
Notebook
Reporter Logbooks
Dual Reporter Logbook
GLB1 Reporter Logbook
Editor Logbook
Primers and Protocols
People
Team
Attributions
Admin
Safety
Medals
Parts
NUS Singapore Science: Editor System - Logbook
Editor System
Logbook
1.Removal of illegal BioBrick site in rAPOBEC
Gene for Rat APOBEC1 Deaminase Domain
rAPOBEC1 (mammalian) (690 bp)
ATGAGCTCAGAGACTGGCCCAGTGGCTGTGGACCCCACATTGAGACGGCGGATCGAGCCCCATGAGTTTGAGGTATTCTTCGATCCGAGAGAGCTCCGCAAGGAGACCTGCCTGCTTTACGAAATTAATTGGGGGGGCCGGCACTCCATTTGGCGACATACATCACAGAACACTAACAAGCACGTCGAAGTCAACTTCATCGAGAAGTTCACGACAGAAAGATATTTCTGTCCGAACACAAGGTGCAGCATTACCTGGTTTCTCAGCTGGAGCCCATGCGGCGAATGTAGTAGGGCCATCACTGAATTCCTGTCAAGGTATCCCCACGTCACTCTGTTTATTTACATCGCAAGGCTGTACCACCACGCTGACCCCCGCAATCGACAAGGCCTGCGGGATTTGATCTCTTCAGGTGTGACTATCCAAATTATGACTGAGCAGGAGTCAGGATACTGCTGGAGAAACTTTGTGAATTATAGCCCGAGTAATGAAGCCCACTGGCCTAGGTATCCCCATCTGTGGGTACGACTGTACGTTCTTGAACTGTACTGCATCATACTGGGCCTGCCTCCTTGTCTCAACATTCTGAGAAGGAAGCAGCCACAGCTGACATTCTTTACCATCGCTCTTCAGTCTTGTCATTACCAGCGACTGCCCCCACACATTCTCTGGGCCACCGGGTTGAAATGA
We noted that there is the presence of an illegal biobrick site (GAATTC) from bases 504 to 509. As such, mutagenesis was conducted via PCR to mutate away a single base from GAATTC to GAGTTC. The mutation is conducted through multiple PCR reactions as shown in Figure 1 below. A total of five primers shown here were designed to mutate away from the illegal site. The rAPOBEC template used for the PCR reactions was a kind gift from Dr. Alexis Komor and Dr. David Liu’s Laboratory.
Figure 1. A schematic diagram showing how the EcoRI site in rAPOBEC catalytic domain was mutated away. A total of four PCRs were conducted for site-directed mutagenesis of the illegal BioBrick site.
Table 1. Summary of PCR reactions done for site-directed mutagenesis on the illegal EcoRI site.
PCR Reaction
Template
Forward Primer
Reverse Primer
1
rAPOBEC
APOBEC F1
APOBEC R1
2
rAPOBEC
APOBEC F2
APOBEC R2A
3
PCR product of PCR reaction 2
APOBEC F2
APOBEC R2B
4
PCR product of PCR reactions 1 & 3
APOBEC F1
APOBEC R2B
Following agarose gel electrophoresis, a bright band near the 700 bp marker was present in lane 2, which indicates that rAPOBEC without the illegal biobrick site was successfully made (Figure 2).
Figure 2. Gel electrophoresis of rAPOBEC. Gel photo showing a bright band near the 700 bp marker, indicating the presence of APOBEC F1R2B, rAPOBEC catalytic domain.
Subsequently, the sample was then purified using a PCR purification kit and transformed into E. coli DH5α for plasmid isolation. The isolated plasmid was then sent for DNA sequencing (1st Base) using the primers VF2 and VR (refer to the primers document below) The sequencing results as shown in Figure 3 indicate that the mutation via PCR was successful, as the illegal EcoRI site was successfully mutated into GAGTTC.
Figure 3. Sequencing results of rAPOBEC. nBLAST conducted on rAPOBEC against rAPOBEC DNA sequences shows that the illegal BioBrick site (GAATTC) was successfully mutated into GAGTTC.
2. Cloning of the rAPOBEC into pSB1C3 vector
To be able to insert F1R2B (rAPOBEC) into pSB1C3 , both DNA have to be digested with the suitable restriction enzymes. F1R2B (rAPOBEC) and pSB1C3 were digested with XbaI and PstI-HF at 37°C for three hours and an agarose gel electrophoresis was done as shown in the results below (Figure 4). In lane 2 and 3, two bands of approximately 1500 bp and 900 bp were observed respectively, indicating the presence of digested pSB1C3. A single digestion of PstI-HF and Xbal was done on lane 4 and 5 respectively, showing a band between 2000 and 2500 bp and indicating the presence of linearised pSB1C3. Lane 6 shows a undigested pSB1C3 with a band between 2000 bp and 2500 bp.
Figure 4. Gel Electrophoresis of restriction enzyme digested products. Lanes 2 and 3 show two bright bands of digested pSB1C3, while lanes 4 and 5 shows linearized pSB1C3 near approximately 2000 bp. Lane 6 shows an undigested pSB1C3.
Ligation was subsequently conducted using NEB T4 ligase in and incubated at 16°C overnight. Each of the reaction tubes were then transformed into E. coli DH5α and plated on LB agar with chloramphenicol plates and left to incubate at 37°C overnight. Colonies were only present on the agar plate with 1:3 (vector-to-insert) ratio. As such, four random colonies were picked and subjected to colony PCR using VF2 and VR primers to determine whether they contained the recombinant plasmid (Figure 5). Lanes 2, 3, 4 & 5 show a band between 700 and 1000 bp, indicating that the colonies picked all contain the recombinant plasmid.
Figure 5. Gel electrophoresis conducted after colony PCR with VF2 and VR primers to determine whether rAPOBEC was successfully ligated into pSB1C3. The bands are between 700 to 1000 bp, indicating the presence of rAPOBEC in pSB1C3.
The plasmids were isolated and sent for sequencing to confirm the presence of the entire rAPOBEC sequence in pSB1C3 ( BBa_K2807000 ).
3. Cloning of rAPOBEC, BBa_K2083000 & BBa_K2083002 into pGEM-T Easy
rAPOBEC catalytic domain (F1R2B), BBa_K2083000 (APOBEC-XTEN) and BBa_K2083002 (APOBEC- 3XTEN) were cloned into pGEM-T Easy for in vitro expression (Figure 6). The pGEM-T Easy plasmid was chosen as it contained a T7 promoter, ampicillin resistance gene and suitable restriction sites for the restriction enzymes ApaI and PstI-HF (Figure 7).
Figure 6. Cloning of rAPOBEC into pGEM-T Easy plasmid.
Figure 7. Plasmid map of pGEM-T Easy vector from Promega.
As rAPOBEC does not contain the ApaI restriction site in the prefix, a forward primer (ApaI-FP) was designed to include the restriction site via PCR. PCR was conducted using the isolated rAPOBEC as a template with ApaI-FP as the forward primer and APOBEC R2B as the reverse primer. Agarose gel electrophoresis results of the PCR is as shown in Figure 8. The gel photo shows one band between 700 and 1000 bp each at lanes 2 and 3 which indicates a successful PCR reaction.
Figure 8. Gel electrophoresis of the PCR products of F1R2B (rAPOBEC) after incorporation of ApaI restriction site.The photo shows a bright band in both lane 2 and 3, indicating a successful PCR.
Similarly, as BBa_K2083000 and BBa_K2083002 do not contain the ApaI restriction site in the prefix, a forward primer (ApaI-FP BB Part) was designed to include the restriction site via PCR. The PCR was conducted using BBa_K2083000 and BBa_K2083002 from the iGEM Part Registry as a template with ApaI-FP BB Part as the forward primer and APOBEC R2B as the reverse primer. The gel results are as shown in Figure 9. The gel photo shows one band between 700 and 1000 bp each at lanes 2 and 3, indicating a successful PCR reaction.
Figure 9. Gel electrophoresis of the PCR products after incorporation of ApaI restriction site.Lanes 2 and 3 each show a bright band respectively near 1000bp, indicating a successful PCR.
The gene fragments and the pGEM-T Easy vector was then subjected to restriction digest at 16°C overnight with ApaI and PstI-HF enzymes to generate sticky ends. A positive control of pGEM-T Easy vector containing actin was also digested at the same time. Gel electrophoresis was subsequently conducted on the digested products to determine whether the restriction digest was successful (Figures 10 & Figure 11). In Figure 10, lane 2 shows a faint band around 700 bp, while lanes 3 and 4 shows a strong band around 3000 bp. Lane 4 also has another faint band at 1000 bp which indicates that restriction digest with ApaI and PstI-HF is a success, where actin is removed from the pGEM-T Easy vector backbone. In Figure 11, lanes 2 and 3 shows a faint band around 900 bp, while lanes 4 and 5 shows a strong band a faint band around 3000 bp.
Figure 10. Results of gel electrophoresis of restriction digested products containing rAPOBEC gene fragment.Lane 2 shows the digested rAPOBEC gene fragment whereas lane 4 shows a faint band between 500 bp and 700 bp. This indicates that the DNA sequence of actin has been successfully digested by ApaI and PstI-HF.
Figure 11. Results of gel electrophoresis of restriction digested products containing BBa_K2083000 and BBa_K2083002. Lanes 2 and 3 shows a strong band between 900 and 1000 bp whereas lane 5 shows a slight smear with a band between 3000 and 4000 bp. This indicates that the DNA sequence of actin has been successfully digested by ApaI and PstI-HF.
Ligation was subsequently carried out at 16°C overnight with NEB T4 Ligase. The ligated products were then subsequently transformed into E. coli DH5α and plated on LB agar plates with ampicillin. There were colonies on all three plates with differing ligation ratios, albeit the most number of colonies were from 1:5, 1:3 and lastly, 1:1. This shows that 1:5 vector:insert ratio would be the most optimal ligation ratio between pGEMT-Easy vector as well as the 3 insert fragments, rAPOBEC, BBa_K2083000 and BBa_K2083002.
Four colonies were randomly picked from 1:3 and 1:5 rAPOBEC plates each respectively and a colony PCR was conducted to confirm whether the rAPOBEC catalytic domain was successfully ligated into the vector backbone (i.e. recombinant plasmid). Colony PCR was conducted using the same ApaI-FP and APOBEC R2B primers and gel electrophoresis was subsequently conducted on the PCR products with the results shown below (Figure 12). Lanes 4, 5, 6, 7, 8 and 9 all had a single strong band near 800 bp which indicates that the rAPOBEC catalytic domain was successfully cloned into the pGEM-T Easy vector.
Figure 12. Results of gel electrophoresis of colony PCR conducted with ApaI-FP and APOBEC R2B primers. Lanes 4, 5, 6, 7, 8 and 9 all shows bands between 700 bp and 1000 bp. However, it is noted that colony 4 and 7 have a relatively brighter brand than the others. This indicates that the bacterial colony has picked up more copies of the recombinant plasmid.
With regards to BBa_K2083000 and BBa_K2083002, two colonies were randomly picked for plasmid isolation. They were then subjected to restriction digest at 16°C overnight with ApaI and PstI-HF enzymes to generate sticky ends. Gel electrophoresis was then conducted on the digested products (Figure 13).
Figure 13. Results of gel electrophoresis of restriction enzyme digestion using ApaI and PstI-HF. Lanes 2, 6, 7 & 9 have faint bands slightly below 1000 bp which indicates a successful digestion using ApaI and PstI-HF. This also indicates that  BBa_K2083000 and BBa_K2083002 were successfully ligated into the pGEM-T Easy backbone.
Sequencing was conducted on plasmid isolated from colonies 4 and 7 using ApaI-FP as the forward primer and APOBEC R2B as the reverse primer to confirm the presence of the rAPOBEC catalytic domain in pGEM-T Easy. Sequencing of the colonies for BBa_K2083000 and BBa_K2083002  were done using ApaI-FP BB Part and APOBEC R2B. The recombinant pGEM-T Easy plasmids were then subjected to in vitro expression and a deaminase assay was carried out.
4. Cloning of Cas13b and dCas13b into pSB1C3 vector
Both the plasmids, pC0049-EF1a-dPspCas13b-NES-HIV, H133A/H1058A and C0046-EF1a-PspCas13b-NES-HIV were kind gifts from Feng Zhang’s Laboratory (Addgene plasmids #103865 and #103862) and were obtained from Addgene. Cas13b is able to target and cleave messenger RNA strands with the aid of a compatible guide RNA, while the nuclease activity in dead Cas13b (dCas13b) has been inactivated through the mutation of the enzymatic catalytic sites. An inactivated form of Cas13b is important for our RESCUE editor as we aim to make a site-directed change rather than cleave the target strand.
PspCas13b and dPspCas13b were first amplified from Addgene plasmids 103862 and 103865 respectively using PCR. The insert comprises the PspCas13b gene with the inclusion of a nuclear export signal and a haemagglutinin (HA) tag at the C-terminus. The forward CasPSB XbaI FP and reverse CasPSB SpeI RP primers were designed to add XbaI and SpeI restriction sites to the PCR products for cloning into the pSB1C3 backbone. Then, the PCR products and pSB1C3 were restriction digested with the enzymes XbaI and SpeI for 2 hours prior to ligation. The digested PCR products were then ligated with the pSB1C3 backbone overnight and transformed into NEB Stable E. coli.
To confirm the presence of the Cas13b insert in the pSB1C3 plasmid, four colonies were selected from each of the pSB1C3-PspCas13b and pSB1C3-dPspCas13b transformation plates for plasmid isolation and restriction digest. The isolated plasmids were then subjected to XbaI and SpeI restriction digestion to cut the insert from the backbone before being analysed via agarose gel electrophoresis (Figure 14).
Figure 14. Each extracted plasmid sample was digested with XbaI and SpeI. Lanes C3 and D3 contain the desired insert between 3000 to 4000 bp. The size of the pSB1C3 backbone is around 2000 bp.
Running the digested plasmids from these samples yielded a band at around 3400 bp in a DNA agarose gel as seen above in samples C3 and D3 (Figure 14). The approximate size for both PspCas13b and the dPspCas13b insert is 3400 bp in size while the linearized pSB1C3 plasmid is about 2000 bp in size.
As the original PspCas13b and dPspCas13b gene from the Addgene plasmids contained four BioBrick incompatible sites, it was necessary to mutate them. As C3 and D3 contained the desired insert, they were then subjected to Q5 Site-directed Mutagenesis .
For both PspCas13b and dPspCas13b, PstI sites at nucleotide base 237, 633, 1023, and 2445 were mutated from CTGCAG to CTTCAG. Primers for each round of Q5 mutagenesis are listed here. (hyperlink to primers list) Cycling parameters for each round of mutagenesis are listed in Table 2.
Table 2. Q5 Cycling Parameters for each round of Q5 Site-directed Mutagenesis.
Temperature (℃)
Time (min)
Cycles
Initial Denaturation
98
0:30
1
Denaturation
98
0:10
25
Annealing
63-68 (See below)
0:20
Extension
72
3:05
Final Extension
72
2:00
1
Annealing Temperature (℃)
Mutation at site 237
63
Mutation at site 633
60
Mutation at site 1023
63
Mutation at site 2445
68
For each round of Q5 mutagenesis, the mutagenesis reaction products were transformed into NEB Stable E. coli cells prior to plasmid isolation, following which the plasmids were sent for sequencing (Figure 15). Plasmid samples containing the desired mutation at the indicated site were then used for the next round of Q5 mutagenesis until all four BioBrick illegal sites were removed from pSB1C3-PspCas13b ( BBa_K2807002 ) and pSB1C3-dPspCas13b ( BBa_K2807001 ) were generated.
Figure 15. Transformants for the third round of Q5 mutagenesis (mutation at site 1023). For each round of Q5 mutagenesis, a negative control was prepared alongside the samples to ensure only the amplified plasmid was transformed into NEB Stable E. coli.
5. Generation of px330A-PspCas13b and px330A-dPspCas13b constructs for Cas protein expression
We then amplified pSB1C3-PspCas13b ( BBa_K2807002 ) and pSB1C3-dPspCas13b ( BBa_K2807001 ) using the APOBEC3 Cas Assay C FP and IDT Back Overlap RPB primers. The Cas13b and dCas13b genes were then inserted in the mammalian expression vector px330A using the HiFi DNA Assembly Kit (NEB).
The PX330A plasmid was a kind gift from Dr. Polly Chen (NUS). The presence of either PspCas13b or dPspCas13b insert was confirmed through sequencing prior to transfection into HEK293T using Lipofectamine 2000. HEK293T cells containing the plasmids were then cultured and lysed, before the lysates were run on an SDS-PAGE gel prior to western blotting to detect either Cas13b and dCas13b using a monoclonal haemagglutinin antibody.
6. Preparation of linearised PX330A with guide RNA
6.1 Preparation of PX330A with guide RNA
The guide RNAs (gRNAs) #1 to #3 were designed based on our PythonTM script (hyperlink to iGEM Github page) and the resultant sequences are shown in Table 3. These sequences were then synthesised as gene blocks (gBlocks) from Integrated DNA Technologies.
Table 3. Sequences of the guide RNAs designed to target the RESCUE Reporter.
gRNA
Sequence
#1
TATTATGAAGACTACACCGCACCCCGGTGAACAGCTCCATGGTGGCGCTAGCGGATCTGACGGTTCACTGTTGTGGAAGGTCCAGTTTTGGGGGCTATTACAACATTTTTTGTTTCGGTCTTCGCGGCG
#2
TATTATGAAGACTACACCGGGCACCACCCCGGTGAACAGCTCCATGGTGGCGCTAGCGGATCTGACGGTGTTGTGGAAGGTCCAGTTTTGGGGGCTATTACAACATTTTTTGTTTCGGTCTTCGCGGCG
#3
TATTATGAAGACTACACCGGGATGGGCACCACCCCGGTGAACAGCTCCATGGTGGCGCTAGCGGATCTGGTTGTGGAAGGTCCAGTTTTGGGGGCTATTACAACATTTTTTGTTTCGGTCTTCGCGGCG
Both px330A plasmid and the gene block gRNAs #1 to #3 were digested with the BbsI restriction enzyme for 3 hours and the gRNA was cloned into the PX330A plasmid. Subsequently, the restriction enzyme (RE)-digested fragments were cleaned up and ligated together. After an overnight incubation the ligation mixes were transformed into NEB Stable E. coli cells and plated on LB agar with ampicillin. Plasmid miniprep was done to isolate plasmids from colonies that grew. Plasmids were digested with NheI restriction enzyme to check for recombinant plasmids. NheI restriction enzyme was chosen as px330A does not contain a NheI site. Thus, any cleavage seen would be due to the single NheI recognition site in the gRNA sequence. All three gRNAs were successfully cloned into the px330A plasmid, and Figure 16 shows the gel electrophoresis results for RE-digest of gRNA #3.
Figure 16. NheI restriction enzyme digestion of px330A to check for presence of gRNA #3. Results show that all three colonies indeed have gRNA #3 inserted into px330A as linearised plasmids at about 8500 bp were obtained.
Plasmid clones that have been observed to have linearised plasmids after NheI restriction enzyme were then sent for DNA sequencing using the sequencing primers gRNA FP and gRNA RP. Our results show that gRNA #3 has been inserted into px330A and confirm that the sequence remained unchanged (Figure 17).
Figure 17. Sequencing results for gRNA #3 using primers gRNA FP and gRNA RP. Results show that gRNA #3 was correctly inserted into px330A, and the sequence remained unchanged with 100% identity.
6.2 Preparation of linearized PX330A backbone alone
To remove Cas9 from the px330A plasmid, the plasmid px330A was first digested with EcoRI-HF (CutSmart Buffer) for two hours. Subsequently, the linearized plasmid was digested with AarI for another 16 hours. Agarose gel electrophoresis was carried out on the RE-digested samples (Figure 18). Images of the gel reveal two close bands of around 4100 and 4500 bp in length, fitting the theoretical values of 4101 bp for Cas9 gene and 4500 bp for px330A backbone. This confirms that restriction enzyme digestion managed to extract out the Cas9 gene, leaving a linearised plasmid to be used in HiFi DNA Assembly. The fragment corresponding to the px330A backbone was extracted out and purified for further cloning.
Figure 18. Agarose gel electrophoresis of undigested plasmid, EcoRI single digest, double digest with both EcoRI and AarI restriction enzymes. Following both EcoRI and AarI restriction enzyme digestion, the band sizes obtained are about 4100 and 4500 bp, with the larger fragment corresponding to px330A backbone (blue) and the smaller fragment corresponding to the Cas9 gene (red).
7. Preparation of rAPOBEC-XTEN insert
The rAPOBEC-XTEN insert was extracted by PCR from the BE3 plasmid (a kind gift from Dr. Alexis Komor and Dr. David Liu’s Laboratory) and the insert length extended to include overhang regions with that of the linearised px330A and dPspCas13b (Figure 19). This is to facilitate HiFi DNA Assembly after.
Figure 19. Schematics showing the primers used to extract rAPOBEC-XTEN from BE3 and to extend the insert to obtain overhang regions with linearised px330A and dPspCas13b. The primers used are as mentioned, and can be found below.
Following agarose gel electrophoresis, a band size of around 800 bp was observed, which is the expected size of the rAPOBEC-XTEN insert, including the overhang regions (Figure 20). The insert was then gel extracted to obtain a purified sample.
Figure 20. Agarose gel electrophoresis of final rAPOBEC-XTEN insert. Following three rounds of PCR, the rAPOBEC-XTEN insert as shown in Figure 19 was obtained (blue). The band size is about 800 bp.
8. Preparation of dPspCas13b insert
The dPspCas13b insert was extracted from pC0049 plasmid and the insert length extended to include overhang regions with that of the linearised px330A plasmid and the rAPOBEC-XTEN insert (Figure 21). This is to facilitate HiFi DNA Assembly.
Figure 21. Schematics showing the primers used to extract dPspCas13b from pC0049 and to extend the insert to obtain overhang regions with linearised px330A plasmid and rAPOBEC-XTEN insert. The primers used are as mentioned, and can be found below.
Following agarose gel electrophoresis, a single band size of around 3500 bp was observed, which is the expected size of the dPspCas13b insert, including the overhang regions (Figure 22). The insert was then gel extracted to isolate a pure sample.
Figure 22. Agarose gel electrophoresis of final dPspCas13b insert. Following three rounds of PCR, the dPspCas13b insert as shown in Figure 21 was obtained (blue). The band size is about 3500 bp.
9. Generation of rAPOBEC-XTEN-dPspCas13b construct in px330A
rAPOBEC-XTEN and dPspCas13b inserts were combined with PX330A backbone containing the gRNA in a 3:3:1 molar ratio using HiFi DNA Assembly kit (NEB). The HiFi mixes were incubated at 50°C for an hour. Subsequently, reactions were placed on ice before the reaction was transformed into NEB Stable E. coli cells. The transformed bacterial cultures were plated on LB agar with ampicillin and incubated overnight. In total, 12 successful growing colonies integrated with gRNAs 1, 2 or 3 were extracted from the plates and cultured in LB broth with ampicillin overnight. Plasmid miniprep was done to extract the plasmids.
Extracted plasmids were then digested with BamHI restriction enzyme for 1 hour at 37°C and gel electrophoresis was carried out on the digested products. If the fusion protein was successfully cloned, two bands of about 2500 and 7000 bp will be seen. If the fusion protein was not successfully cloned, a linearised plasmid band of 8500 bp will be seen. Gel images taken thereafter suggested that only the digested plasmid from Colony #5 contained the double bands at around 7000 bp and 2500 bp, suggestive of the presence of the dPspCas13b insert within the plasmid (Figure 23).
Figure 23. BamHI restriction enzyme digestion of px330A to check for presence of the fusion protein. Results show that Colony #5 has the two expected bands of 2500 bp (red) and 7000 bp (blue). Hence, plasmid containing the fusion protein was obtained from Colony #5.
To confirm, sequencing was performed using the primer APOBEC3 Cas Assay C FP to check for rAPOBEC, IDT Back RPB to check for the Cas13b and XTEN dCas13 R2 for the XTEN-dPspCas13b junction. Both rAPOBEC-XTEN and dPspCas13b inserts were found to be successfully cloned into the px330A plasmid.
