Team:NUS Singapore-Sci/Reporter Overview
Project
Project Overview
Reporter System
Reporter Overview
Dual Reporter - Demonstration
Dual Reporter - Improvement
GLB1 Reporter
Editor System
Editor Overview
Editor Results
Modelling
Modelling Overview
Kalman Filter Ensemble
Molecular Dynamics
Enzyme Kinetics
gRNA Design
InterLab
Human Practice
Overview of Human Practice
Public Engagement
#CasAsks
#CasAsks: Survey
#CasAsks: Interview
#CasTalks
#CasTeaches
#CasWrites
#CasWrites: Discussion
#CasWrites: Collaboration
#CasWrites: BioEthics
Collaborations
Notebook
Reporter Logbooks
Dual Reporter Logbook
GLB1 Reporter Logbook
Editor Logbook
Primers and Protocols
People
Team
Attributions
Admin
Safety
Medals
Parts
NUS Singapore Science: Reporter - Overview
Reporter System
Overview
Designing a real-time, high throughput and efficient reporter system to analyse RNA base editing in cells.
A significant roadblock in optimising base editing technologies and also their widespread use in different cell types is the lack of an efficient and real-time reporter system to analyse and quantify such editing assays. One potential way of overcoming this is to use a fluorescence protein as a reporter molecule for such assays. Fluorescence molecules are powerful as they can be easily tracked and visualised using fluorescence microscopy or via flow cytometry.
Compared to nucleic acid sequencing, this proposed reporter molecule can provide a real-time read for any editing assay. Therefore, we designed and constructed a real-time fluorescence reporter system for quantification of base editing efficiency in live cells.  Moreover, we aim to incorporate a second reporter in the system that is under the control of the same promoter in order to normalize the effect of differential transfection and expression efficiencies.
1. Design of the dual fluorescence EGFP-T2A-mCherry reporter for quick visualisation of RNA base editing.
Inspired by the work of Martin et al. (2018), as well as WPI Worcester 2016 iGEM project , we designed an EGFP-T2A-mCherry dual fluorescence reporter plasmid system. This plasmid consists of two fluorescence markers, an enhanced green fluorescent protein (EGFP) gene and a mCherry gene (Figure 1). Here, mCherry is expressed constitutively and used as a marker for transfection efficiency and the overall fusion protein expression level. The EGFP gene is modified such that the start codon of EGFP is mutated (ATG → ACG), resulting in a truncated GFP that is non-functional (Figure 1). Since the minimal fluorescence region for EGFP is from amino acid 7-229 (Li et al., 1997), this truncation mutation will cause the production of non-functional EGFP protein, while mCherry expression is not affected. However, when this mutation (ACG → ATG)  is corrected by our RESCUE Editor system , the normal expression of EGFP is restored and can be visualised either on a fluorescence microscope or via flow cytometry.
Both the mCherry and the EGFP genes in the fusion protein are linked by a self-cleavage peptide, T2A. When the ribosome reaches the T2A oligopeptide during translation, it will fall off from the mRNA and translation would be restarted for the next fluorescence gene. We chose T2A among the 2A family peptides as it has the highest cleavage efficiency (Liu et al., 2017). This is to ensure that expression of EGFP and mCherry are under the control of the same promoter, such that the level of mCherry fluorescence is linearly correlated to the level of EGFP fluorescence. In this way, the mCherry expression would be a useful indicator to normalise the expression of the dual reporter system between individual cells in a transient transfection experiment.
The relative number of editing events can be quantified by the percentage of dual-colored cells transfected with mutant ACG reporter as compared to cells transfected with the WT reporter, or relative green fluorescence intensity after normalizing to the red fluorescence intensity. We will be testing our reporter system in both E. coli and transfected HEK293T cells.
A further modification to the original EGFP and mCherry sequence is a six amino acid truncation at the N-terminus of EGFP and a seven amino acid truncation at the C-terminus of mCherry. Since EGFP and mCherry have the same nucleotide sequence for the first 22 and last 23 nucleotides, such truncation makes cloning easier. Moreover, since bacteria like E. coli can utilize alternative start codon GUG up to 13% of the time (Hecht et al., 2017), and the first 18 nucleotides of EGFP sequence contain one GUG, this truncation mutation will eliminate the chance of background translation initiation in bacteria, thus making the reporter suitable for both bacterial and mammalian systems.
Figure 1. A schematic illustration of our EGFP-T2A-mCherry reporter system. (A) The ACG mutated reporter has the start codon mutated from ATG to ACG. As the second ATG codon in-frame is 78 nucleotides away, elimination of the first start codon leads to a 26 amino acid truncation of EGFP at the N-terminus. Since the minimal fluorescence region for EGFP is from amino acid 7-229 (Li et al., 1997), this truncation mutation will cause the production of non-functional EGFP protein, while mCherry is expressed normally. When our RESCUE Editor system successfully edits the C in ACG (on the mRNA) to U in the presence of gRNA, the start codon will be restored and functional EGFP is produced. As such, our dual fluorescence reporter allows for the unambiguous change from OFF (unedited) to ON (edited change), where (B) unedited cells only have mCherry fluorescence while (C) cells with successful editing events will have both green and red fluorescence. Images were taken at 60x magnification, scale bar = 50 µm.
2. Identification of GLB1, a gene mutated in patients with lysosomal storage disease and the designing of a second EGFP-GLB1 dual reporter to test the RESCUE Editor system in cells with mutated GLB1.
The second reporter design involves a GFP fusion protein linked to the human GLB1 gene. GLB1 encodes a human lysosomal acid beta-galactosidase, an enzyme that is responsible for the cleavage of terminal β-linked galactose residues from glycoproteins, sphingolipids, keratan sulfate, and other glycoconjugates. Genetic mutations in the GLB1 gene have been seen in the occurrence of autosomal recessive lysosomal storage diseases such as GM1 gangliosidosis and Morquio B disease (Suzuki et al., 2001). There are more than 130 single base mutations that have been identified in GLB1 and some of these mutations are the (T → C) base change as listed in Table 1. Hence the use of our RESCUE Editor system could potentially restore the function of GLB1 in patients with this mutation, thus acting as potential therapy towards such genetic diseases.
As such, we designed an EGFP-GLB1 fusion protein to act as an alternative base editing reporter for potential testing in a disease model in the future. Based on published work on GLB1 mutation genotyping in patients and its corresponding residual enzymatic activity (Table 1), we carried out site-directed mutagenesis on 319T → C, which leads to phenylalanine at amino acid 107 to be changed to leucine (TCT mutant), and on 1166T → C, which leads to leucine at amino acid 389 to be changed into proline (CCG mutant). These two single nucleotide substitutions are expected to result in total loss or minimal residual beta-galactosidase enzymatic activity.
Using colourimetric (X-gal, ONPG) or fluorescent (DDAOG) beta-galactosidase substrate, we can then check for the restoration of GLB1 enzymatic activity after base editing in transfected cells. A successful base repair by our RESCUE Editor system would allow the conversion of a mutant GLB1 to a functional GLB1, thus leading to restoration of enzymatic function (Figure 2). At the same time, the EGFP reporter can act as transfection/expression control for internal normalisation.
Figure 2. Schematic illustration of EGFP-GLB1 reporter. EGFP and GLB1 gene are separated by the linker which contains a stop codon for EGFP and a Kozak sequence for GLB1 translation initiation. This ensures both proteins are expressed as their native sizes. Our RESCUE Editor system will change UCU to UUU for the TCT mutant and change CCG to CUG for CCG mutant to restore enzymatic activity.
Table 1. A summary table showing the positions of the T to C substitution mutation in patients with lysosomal storage diseases.
Nucleotide change in GLB
Resultant Amino Acid Change
Level of Residual enzymatic activity compared to WT
Disease relevance
Reference
319T>C
F107L
1.4%
hetereoalleic, Juvenile GM1
Hofer et al., 2010
688T>C
C230R
3.5%
homoalleic, Juvenile GM1
Hofer et al., 2010
1166T>C
L389P
<1%
hetereoalleic, Juvenile GM1
Hofer et al., 2010
186T>C
I51T
5-6%
homoalleic, Juvenile GM1
Yoshida et al., 1991
518T>C
L173P
1.8-5%
hetereoalleic, Morquio B
Santamaria et al., 2006
791T>C
L264S
2%
hetereoalleic,Juvenile GM1
Santamaria et al., 2006
485T>C
L162S
1.2%-3.2%
homoalleic, Infantile GM1
Santamaria et al., 2006
References
Suzuki Y, Oshima A, Nanba E. 2001. ß-galactosidase deficiency (ß-galactosidosis) GM1-gangliosidosis and Morquio Bdisease. In: Scriver CR, Beaudet AL, Sly WS, Valle D, editors. The Metabolic and Molecular Bases of Inherited Disease. New York: McGraw-Hill Publishing Co. p 3775-3809.
St Martin, A., Salamango, D., Serebrenik, A., Shaban, N., Brown, W. L., Donati, F., Munagala, U., Conticello, S.G. & Harris, R. S. (2018). A fluorescent reporter for quantification and enrichment of DNA editing by APOBEC–Cas9 or cleavage by Cas9 in living cells. Nucleic acids research.
Li, X., Zhang, G., Ngo, N., Zhao, X., Kain, S. R., & Huang, C. C. (1997). Deletions of the Aequorea victoria green fluorescent protein define the minimal domain required for fluorescence. Journal of Biological Chemistry, 272(45), 28545-28549.
Liu, Z., Chen, O., Wall, J. B. J., Zheng, M., Zhou, Y., Wang, L., Vaseghi, H.R., Li, Q. & Liu, J. (2017). Systematic comparison of 2A peptides for cloning multi-genes in a polycistronic vector. Scientific reports, 7(1), 2193.
Hecht, A., Glasgow, J., Jaschke, P. R., Bawazer, L. A., Munson, M. S., Cochran, J. R., Endy, D. & Salit, M. (2017). Measurements of translation initiation from all 64 codons in E. coli. Nucleic acids research, 45(7), 3615-3626.
Hofer, D., Paul, K., Fantur, K., Beck, M., Roubergue, A., Vellodi, A., Poorthuis, B.J., Michelakakis, H., Plecko, B. & Paschke, E. (2010). Phenotype determining alleles in GM1 gangliosidosis patients bearing novel GLB1 mutations. Clinical genetics, 78(3), 236-246.
Yoshida, K., Oshima, A., Shimmoto, M., Fukuhara, Y., Sakuraba, H., Yanagisawa, N., & Suzuki, Y. (1991). Human beta-galactosidase gene mutations in GM1-gangliosidosis: a common mutation among Japanese adult/chronic cases. American journal of human genetics, 49(2), 435.
Santamaria, R., Chabás, A., Coll, M. J., Miranda, C. S., Vilageliu, L., & Grinberg, D. (2006). Twenty‐one novel mutations in the GLB1 gene identified in a large group of GM1‐gangliosidosis and Morquio B patients: possible common origin for the prevalent p. R59H mutation among gypsies. Human mutation, 27(10), 1060-1060.
Santamaria, R., Blanco, M., Chabas, A., Grinberg, D., & Vilageliu, L. (2007). Identification of 14 novel GLB1 mutations, including five deletions, in 19 patients with GM1 gangliosidosis from South America. Clinical genetics, 71(3), 273-279.
