"
Page
Discussion
View source
History
teams
Log in
 
Team:Carnegie Mellon/Project/Abstract
From 2013.igem.org
Killer Red
Home
Team
Team Profiles
Advisors
Attributions
Official Team Page
Project
Overview
Procedure
Experimental Results
Future Directions
Parts
Killer Red
RFP Characterization
Modeling
KillerRed Synthesis/ Radical Production
Host-Phage Dynamics
Notebook
Protocols
Weekly Updates
Safety
Outreach
Human Practices
Collaboration
Abstract
Due to the widespread misuse and overuse of antibiotics, drug resistant bacteria now pose significant risks to health, agriculture and the environment. An alternative to conventional antibiotics is phage therapy. However, many temperate phage also form prophage. Our approach to antibiotic resistance is to engineer a temperate phage, Lambda (λ), with light-activated production of superoxide. The fluorescent protein KillerRed was cloned into a plasmid vector and λ gt11 with the IPTG inducible lac promoter1. Lysogens were isolated and these strains were characterized and compared to E. coli with KillerRed from high-copy plasmids. Light activation of KillerRed resulted in decreased cell numbers. In addition, we modeled our system at multiple scales, including populations of phage and bacteria, KillerRed gene expression, ROS production, and effects of light. Having two methods of killing, lysis and superoxide, decreases the probability of developing resistance and our system overcomes the prior limitations of using wild-type temperate phages.
Phage life cycles
Two kinds of bacteriophage exist: lytic and temperate. Both are capable of simple lytic reproduction: the viral genome is injected into the host, which then replicates the genome. Viral proteins are synthesized in order to assemble new virions, into which the viral genome is packaged. Lysozymes synthesized from the viral genome enable release of new phage and cause death of the host.  Temperate phages such as $\lambda$ are able, under certain conditions, to integrate their genome into the bacterial chromosome; this state is known as lysogeny, and a phage which has entered lysogeny is known as a prophage. Lysogens (hosts cotaining a prophage) may multiply indefinitely in this state. Stress on a lysogen may cause prophage induction, in which the lytic cycle is resumed, killing the host.
Impact
On September 16, 2013 the CDC released to the public “Antibiotic Resistance Threats in the United States, 2013” . This document is intended to raise public awareness of the problems associated with overuse and misuse of antibiotics and to outline the threats to society caused by these organisms. The organisms have been categorized by hazard level as urgent, serious and concerning. Over 2 million illnesses and 23,000 deaths per year are a direct result of antibiotic resistance. The CDC outlines four core actions that will help fight these deadly infections:
Preventing infections and preventing the spread of resistance
Tracking resistant bacteria
Improving the use of today’s antibiotics
Promoting the development of new antibiotics and developing new diagnostic tests for resistant bacteria
The discovery of antibiotics was quickly followed by the development of antibiotic resistance (Figure 1). New medicines are becoming increasingly scarce (Figure 2). This project addresses this problem.
Rationale
There is a great need for new antimicrobial strategies.
Phage therapy represents a completely different solution.
Incorporating KillerRed phototoxicity provides another level of controlled killing.
Bacteriophages infect only specific hosts. Many phages are temperate, meaning that they can enter the a lysogenic phase and therefore not lyse and kill the host bacteria. The addition of KillerRed to the system offers a second method of killing those bacteria targeted by a lysogenic phage. Thus, our system explores the possibility that temperate phages can also be used for phage therapy and bacteria killing applications.
An additional consideration is the ability to have a kill-switch on the phage and KillerRed can achieve this by also killing the phage upon exposure to light.
Background
Discovery of Antibiotics
Arsphenamine is an arsenic compound  that was discovered to have antisyphilitic properties by Sahachiro Hata and Paul Ehrlich in 1909. In 1929, Sir Alexander Fleming published the results of his study of a substance that he named penicillin. This compound was released by Penicillium fungi and killed bacteria. These antibiotics function by inhibiting an enzyme involved in crosslinking of the peptidoglycan cell wall of gram-positive bacteria. Resistance to β-lactam antibiotics is derived from β-lactamase, an enzyme that cleaves the drug and inactivates it. The gene for this enzyme is commonly used as a selectable marker in recombinant DNA laboratories to confer resistance to ampicillin. Today there are antibiotics of the cephalosporin family that are derived from the compound  isolated in 1945 by Giuseppe Brotzu, from the fungus Cephalosporium acremonium.
The first synthetic antimicrobial compounds were the sulpha drugs, discovered in the early 1930’s. These drugs were developed at Bayer from dyes. They are bacteriostatic and act by inhibiting folate biosynthesis. Since the 1930’s thousands of modifications have been made to improve stability and functionality and decrease side effects. Where sulfonamides and sulfones are analogues of para-aminobenzoic acid, other medicines such as trimethoprim, methotrexate, pyrimethamine bind to dihydrofolate reductase and inhibit formation of tetrahydrofolic acid.
Aminoglycosides, tetracyclines and spectinomycin all inhibit protein synthesis by binding to the 30S ribosomal subunit. Chloramphenicol, lincomycin, clindamycin, and Erythromycin bind to the 50S ribosomal subunit and fusidic acid binds to elongation factor G to inhibit protein synthesis. Inhibitors of nucleic acid synthesis include rifampin, rifamycin, rifampicin and quinolones such as nalidixic acid, ciprofloxacin, oxolinic acid.
Figure 1: Antibiotic resistance timeline since 1940 (Reference 2)
Figure 2: Antibiotic development pipeline since 1980 (Reference 2)
Mechanisms of Antibiotic Resistance
Altered permeability of the antimicrobial agent. Altered permeability may be due to the inability of the antimicrobial agent to enter the bacterial cell or alternatively to the active export of the agent from the cell.
Inactivation of the antimicrobial agent. Resistance is often the result of the production of an enzyme that is capable of inactivating the antimicrobial agent.
Altered target site. Resistance can arise due to alteration of the target site for the antimicrobial agent.
Replacement of a sensitive pathway. Resistance can result from the acquisition of a new enzyme to replace the sensitive one.
Alternative Treatment - Bacteriophage Therapy
As an alternative to small molecule drugs there is bacteriophage therapy. In 1919, Félix d’Herelle, of the Pasteur Institute in Paris, and his colleagues treated a 12-year old boy for dysentery with a phage preparation and he recovered fully in days. Over the next few years, successful trials were conducted around the globe. However, American companies such as Eli Lily had mixed results with trials, and in 1934, the American Medical Association released a strong critique of phage therapy. When small molecule antibiotics were invented, Western medicine adopted these treatments preferentially3. Now that antibiotics are losing their power it is time to consider alternatives.
Having evolved to manipulate the bacterial cell and genome, resistance to bacteriophage is difficult to achieve. Superinfection immunity is where bacteria become resistant because a currently residing prophage does not allow for lytic growth. There are mechanisms that can be engineered into the phage to bypass this limitation. Mutations of the receptor protein that phage bind to prior to DNA injection is another resistance mechanism and lambda mutants can be found that bind to alternative receptors. Bacteriophage are complex and utilize many host pathways so cannot be inactivated or bypassed.
Bacteriophage can kill by lysis however many phage are temperate and form lysogens to harbor the phage indefinitely, we have an idea on how to overcome this limitation of phage therapy.
References
1Young RA and Davis RW. 1983. Efficient isolation of genes by using antibody probes. Proc. Nati Acad. Sci. USA Vol. 80, pp. 1194-1198, March 1983
2 Antibiotic resistance threats in the United States, 2013 . The Center for Disease Control. 9/16/2013.
3 Stone R. 2002. Stalin’s Forgotten Cure. Science 298:728-731
Retrieved from " http://2013.igem.org/Team:Carnegie_Mellon/Project/Abstract "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
