"
Page
Discussion
View source
History
teams
Log in
 
Team:Carnegie Mellon/Week12
From 2013.igem.org
Killer Red
Home
Team
Team Profiles
Advisors
Attributions
Official Team Page
Project
Overview
Procedure
Experimental Results
Future Directions
Parts
Killer Red
RFP Characterization
Modeling
KillerRed Synthesis/ Radical Production
Host-Phage Dynamics
Notebook
Protocols
Weekly Updates
Safety
Outreach
Human Practices
Collaboration
Monday, August 19th
Sequencing results yielded no positive results (with KillerRed as an insert). Cheryl’s attempt at cloning was also unsuccessful. PCR for KillerRed insert needs to be repeated.
Lysogen development and screening:
1089 host was infected with phage containing the KillerRed and RFP ß-gal fusions and plated. 16 colonies were chosen and “patched” (~1cm long streaks) onto LB/amp plates. These plates were kept at 30ºC, 37ºC and 42ºC. 2 colonies on each plate were predicted lysogens because of the colony morphology on the 42ºC plate. These cells were cultured, heat induced, chloroformed and plated to identify the presence of phage. The plates that had phage indicated the true lysogens.
Patching of the new lysogen plates needs to be repeated to determine if colonies are lysogens. Literature require 10mM (1:100 dilution) of IPTG for lysogens. To obtain a lot of RFP (and KillerRed) in the plasmids, grow overnights, dilute 1:100 and wait until OD600 is between .8-1.0 and induce with 10mM IPTG. These promoters are strong but require a lot of inducer.
The backbones that iGEM sent are digested with EcoRI and PstI. We need to design primers that are compatible with this.
( http://parts.igem.org/Help:Constructing_Parts )
iGEM Prefix:
5’ GTTTCTTCGAATTCGCGGCCGCTTCTAGAG [Part]
EcoRI   XbaI
iGEM Suffix
5’ [Part] TACTAGTAGCGGCCGCTGCAGGAAGAAAC
SpeI   PstI
KillerRed DNA sequence:
ATGGGTTCAGAGGGCGGCCCCGCCCTGTTCCAGAGCGACATGACCTTCAAAATCTT CATCGACGGCGAGGTGAACGGCCAGAAGTTCACCATCGTGGCCGACGGCAGCAGCA AGTTCCCCCACGGCGACTTCAACGTGCACGCCGTGTGCGAGACCGGCAAGCTGCCC ATGAGCTGGAAGCCCATCTGCCACCTGATCCAGTACGGCGAGCCCTTCTTCGCCCG CTACCCCGACGGCATCAGCCATTTCGCCCAGGAGTGCTTCCCCGAGGGCCTGAGCA TCGACCGCACCGTGCGCTTCGAGAACGACGGCACCATGACCAGCCACCACACCTAC GAGCTGGACGACACCTGCGTGGTGAGCCGCATCACCGTGAACTGCGACGGCTTCCA GCCCGACGGCCCCATCATGCGCGACCAGCTGGTGGACATCCTGCCCAACGAGACCC ACATGTTCCCCCACGGCCCCAACGCCGTGCGCCAGCTGGCCTTCATCGGCTTCACC ACCGCCGACGGCGGCCTGATGATGGGCCACTTCGACAGCAAGATGACCTTCAACGG CAGCCGCGCCATCGAGATCCCCGGCCCACACTTCGTGACCATCATCACCAAGCAGA TGAGGGACACCAGCGACAAGCGCGACCACGTGTGCCAGCGCGAGGTGGCCTACGCC CACAGCGTGCCCCGCATCACCAGCGCCATCGGTAGCCACGACGAGATCTAG
KillerRed Protein sequence from Evrogen:
MGSEGGPALFQSDMTFKIFIDGEVNGQKFTIVADGSSKFPHGDFNVHAVCETGKLPMSWK PICHLIQYGEPFFARYPDGISHFAQECFPEGLSIDRTVRFENDGTMTSHHTYELDDTCVV SRITVNCDGFQPDGPIMRDQLVDILPNETHMFPHGPNAVRQLAFIGFTTADGGLMMGHFD SKMTFNGSRAIEIPGPHFVTIITKQMRDTSDKRDHVCQREVAYAHSVPRITSAIGSDEDS GLRSRAQASNSAVDGTAGPGSTGSR
KillerRed Protein Sequence From Linstedt Lab:
MGSEGGPALFQSDMTFKIFIDGEVNGQKFTIVADGSSKFPHGDFNVHAVCETGKLPMSWK PICHLIQYGEPFFARYPDGISHFAQECFPEGLSIDRTVRFENDGTMTSHHTYELDDTCVV SRITVNCDGFQPDGPIMRDQLVDILPNETHMFPHGPNAVRQLAFIGFTTADGGLMMGHFD SKMTFNGSRAIEIPGPHFVTIITKQMRDTSDKRDHVCQREVAYAHSVPRITSAIGSHDEI
KillerRed Protein Sequence from PDB:
MEGGPALFQSDMTFKIFIDGEVNGQKFTIVADGSSKFPHGDFNVHAVCETGKLPMSWK PICHLIQYGEPFFARYPDGISHFAQECFPEGLSIDRTVRFENDGTMTSHHTYELDDTCVV SRITVNCDGFQPDGPIMRDQLVDILPNETHMFPHGPNAVRQLAFIGFTTADGGLMMGHFD SKMTFNGSRAIEIPGPHFVTIITKQMRDTSDKRDHVCQREVAYAHSVPRITSAIGSDED
Red highlighted regions indicate disparities. According to the Evrogen website, the DED at the C-terminus is still part of the protein and the sequence from PDB agrees. iGEM likes double stop codons and its worth changing it to a TAA instead of a TAG since its in E. coli. Although it might not mean much, perhaps it would be worthwhile to include it in the submission.
BB_KRF:
5’GTCTTCGAATTCGCGGCCGCTTCTAGAGATGGGTTCAGAGGGCGGCCCCG  50 bp
EcoRI   XbaI start
Tºm=74.1ºC
BB_KRR:
5’GTCTTCCTGCAGGCGGCCGCTACTAGTATTATTAGATCTCGTCGTGGCTACCGATG 56 bp
PstI    SpeI 2xstop
Tºm=70.0ºC
∆T=4.1ºC (<5º)
Tuesday, August 20th
Potential Lysogens were centrifuged in the presence of 100µL of chloroform and diluted to 10-5 and 10-6 to see individual plaques.
XL10 cultures were diluted and induced when OD600 reached 0.6. These cultures are to be used for photobleaching tomorrow
Wednesday, August 21st
According to Coherent FieldMaxII Power Meter
Lamp power at 540nm: .5 W/cm2
Lamp power at 545nm: .8 W/cm2
Lamp power at 565nm: .7 W/cm2
Lamp power at 585nm: .6 W/cm2
Lamp power at 400nm: 1.2 W/cm2
Photobleaching started: 11:50am
2:30pm: KilerRed Induced/Exposed culture shows aggregation and clumps of cells (unclear if viable or dead) where there is no aggregation in any of the other tubes (except the uninduced KillerRed tube but the aggregates are far fewer in number). The aggregation may be from extensive cross-linking due to the presence of superoxide. To a lesser degree, flocculation is also present in the original KillerRed strains before photobleaching, indicating  that the aggregation may be from the presence of the KillerRed since it is oligomeric.
End: 4:50pm
Retrieved from " http://2013.igem.org/Team:Carnegie_Mellon/Week12 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
