"
Page
Discussion
View source
History
teams
Log in
 
Team:UPO-Sevilla/Project/Notebook/MiniTn7
From 2011.igem.org
Home
Project
Foundational Advances
Human Practices
Team
Overview
Data Page
Basic Flip Flop
Improved Flip Flop
Epigenetic Flip Flop
Applications
Parts
Notebook
Judging Criteria
Overview
MiniTn7
BioBrick Creator
Overview
Safety
Tornillos y Genes
BSIES
Randy Rettberg' s Seminar
Workshop
Science Fair
Week of Science
Media
Team Structure
Attributions
Acknowledgements
Home
Project
Notebook
MiniTn7
Notebook
MiniTn7BB
David Caballero's Diary
Week 1 (July, 4-8)
First week in the lab!
During the last few months we have been studying and designing a Tn7-based system to deliver BioBricks into bacterial chromosomes. Now it is time to perform the ideas and experiments we drew on paper.
We ordered the synthesis of a complete basic mini-Tn7 to avoid site-directed mutagenesis, cloning and other intermediate processes. Unfortunately, our order is delayed (22 days by now). So, this week I tried to prepare all the necessary strains and vectors in advance. Anyway, this is my first week in Fernan’s lab at CABD, so I am mostly getting familiar with the location of materials and equipment.
These were my main tasks this week:
Prepare pUC18-SfiI vector for mini-Tn7 insertion. Our mini-Tn7 was designed flanked by SfiI target sites. We plan to use a pUC18 vector, replicative in Enterobacteriaceae to maintain the mini-Tn7. I located the vector in the lab strain collection, set up an inoculum, purified the plasmid DNA, measured the concentration with the Nanodrop and digested with SfiI.
Prepare pUC18R6K-SfiI vector for mini-Tn7 insertion. The goal is to obtain a vector with an R6K replication origin, which is non-replicative in pir- strains to host the mini-Tn7 constructs. We requested a pUC18R6K-mini-Tn7T-Km vector from Herbert Schweizer's lab in Colorado State University. We will use this as a template to amplify vector sequences adding SfiI restriction sites. I had problems to get amplification in the PCR reaction (twice), so I did analytic digestions with different restriction enzymes (four different cuts). The restriction patterns are not as expected. I may have used the wrong template (maybe the tube was mislabeled). I made new mini-preps, double-checking that the strain was the right one. Next week I will try to do this again with the new miniprep DNA.
Prepare large amounts of the vectors necessaries to transpose the mini-Tn7. we will need fairly large amounts of pTNS2 (containing the Tn7 transposase complex), pUC18R6K-mini-Tn7T-Km (positive control), pBBR1MCS5 (negative control) to electroporate into the recipient strains for the transposons. I streaked the strains on selective plates, set up inocula, purified plasmid DNA and measured the concentration of the DNA, which was generally lower than expected. Also I did some checking of the pTNS2 plasmids with restriction enzymes.
Finally I started a protocol to make electro-competent cells of Escherichia coli and Salmonella typhimurium, two of the four strains we will use to characterize the behavior of mini-Tn7.
Week 2 (July, 11-15)
Our order of DNA synthesis has not arrived yet. It is a month behind-schedule by now and we are getting anxious about this delay of the DNA synthesis service. I know I must use my time in the lab despite not having all the required materials, so I made a list of all the experiments I can run without the mini-Tn7. Actually It is a long list with a fair amount of work to do. I set priorities and start from the top!
First of all, I need a suicide vector I can use as the mini-Tn7 delivery system. Similarly to last week, the new minipreps of pUC18R6K-mini-Tn7T-Km yielded no PCR product and the restriction pattern with five different enzyme combinations was again not what we expected. Despite our efforts, it is evident that the plasmid we received was not pUC18R6K-mini-Tn7T-Km as published. After contacting the Schweizer lab about this problem, they informed us that they had sent us a different, although similar plasmid, but unfortunately it is not suitable for our purposes. Instead, restriction analysis confirmed that the transposase-delivery plasmid pTNS2 is based on pUC18R6KT and contains an oriT for conjugative mobilization, so we decided to use this as a template. Unfortunately, our SfiI-bearing primers are not suitable for this plasmid, so we designed new primers to amplify the pUC18R6KT backbone from pTNS2.
The last set of mini-preps I did to purify pTNS2, pUC18R6KT-mini-Tn7T-Km (now discarded) and pBBR1-MCS5 had very low yield. I repeated the process and obtained somewhat more concentrated, but still low-yield preparations.
This week I attempted to prepare several batches of electro-competent cells that I will need for the project. The strains are E. coli DH5α-λpir, E. coli MC4100, and S. typhimurium LT2. The former will be used for cloning into the R6K plasmids, and the other two will be used as hosts for insertion of the miniTn7 transposons. The protocol requires growth of large (500 ml) cultures OD600=0.5, followed by repeated washes with sterile, cold water. Cells are concentrated, allocated in 40 l samples and stored at -80°C. It sounds easy, but I had to repeat the process three times due to beginner's mistakes.
I did mini-preps of two vectors expressing the Flp recombinase: pFLP2, a replicative vector which is lost spontaneously at high frecuency in Pseudomonas strains; and pCP20, which has a termosensitive replication origin that allows replication at 30°C but not at 37°C. This last vector will be used in E. coli and S. typhimurium.
Week 3 (July, 18-22)
This was a less stressful week. We asked the DNA synthesis company about the 1-month delay and they claimed they had problems with the synthesis and still need two more weeks to have it ready. We are getting desperate about this! Still, I keep on preparing material in advance of the arrival of the long-awaited mini-Tn7.
Checking the electro-competent cells I made the week before. I electroporated my electro-competent cells with a pUC19 vector to measure their transformation frequency but I hardly obtained any colonies. Being suspicious that the pUC19 preparation may be bad, I repeated the transformation with a more recent preparation of pBluescript SK(+). I obtained approximately 107 transformants/g for the three strains. This is an acceptable frequency for MC4100 and LT2, but we need a really high transformation frequency (over 109 transformants/g) in order to perform cloning experiments with DH5α-λpir. We decided to prepare a new batch of DH5α-λpir electro-competent cells.
The mini-Tn7 was designed with a Gm resistance cassette flanked by selected restriction sites to facilitate exchange for other resistance cassettes. To construct mini-Tn7 variants with different markers, I designed primers to amplify kanamycin and chloramphenicol resistance cassettes from the vectors pSB4K5 and pSB1C3. Also, I did mini-preps of these vectors.
Another thing we would like to do with the mini-Tn7 is to generate a handful of functionally useful mini-Tn7 vectors. The idea is to insert into the BioBrick cloning site of the mini-Tn7 useful BioBricks, like reporters for gene expression measurement, inducible promoters and constitutively expressed fluorescent and bioluminiscent labels. We selected the following BioBricks:
Promoter measurement cassettes: BBa_K093005 (RBS+mRFP1), BBa_I732094 (LacZα+GFP-AAV).
Inducible promoters: BBa_I0500 (pBad/araC), BBa_I732083 (pLacIQ+RBS+tetR+STOP+ptetR).
Strain labels: BBa_I13521 (ptet+mRFP), BBa_K325909 (pBad+Lux).
This week I transformed these BioBricks in DH5α and prepared the corresponding plasmids.
In order to test the site-specific insertion of the mini-Tn7 transposons in our strains, I designed primers to amplify from the Tn7R end to the glmS gene. I took this opportunity to train my self in doing sequence comparison and analysis using common tools such as Clustal and Blast. I searched for a conserved motif to make one primer suitable for all four bacterial strains, but the conserved sequences were not  long enough. In consequence, I designed two different glmS of primers, one for E. coli and S. typhimurium and another one for P. aeruginosa and P putida.
attTn7, the insertion site of the Tn7 transposon, is very well conserved in Gram-negative bacteria. However, if you want to use our mini-Tn7 based BioBrick delivery system in an organism without the attTn7, don’t worry, we plan to create a "portable" attTn7 for you. To this end, I designed primers to amplify the attTn7 site in the E. coli genome.
Not having a better task to do, as you can see, this was primer design week for me
Week 4 (July, 25-29)
Back to a hard-working week. As my time in the lab passes I realize that I can perform more experiments in less time, which does not always mean to get better results. Wednesday was an incredible day: I did more of the molecular biology technics I already know, and our mini-Tn7 synthesis order arrived. I exploded with enthusiasm. These were my main tasks this week:
Checking DH5α-λpir electro-competent cells. I electroporated the previously prepared cells with a pBluescript SK(+) vector and I got approximately 2.5 108 transformants/μg. It was not the expected >109 transformants/μg, but it is an acceptable frequency even for cloning experiments.
Not amplifying the pUC18R6KT vector from pTNS2. This week I made a huge effort to obtain a suicide vector to deliver the mini-Tn7, but finally I did not get it. I performed at least one PCR reaction with this aim every day, changing conditions (polymerase, nucleotide concentration, DNA concentration, different primer preparations, even already made PCR mix) and PCR cycle (temperature and time of steps). In the last try I performed a gradient PCR to determine the optimum annealing temperature with a PCR mix kit where there was not much that could do wrong, but even then I did not get any amplification. Then we thought that the SfiI restriction site we added to the primers could be avoiding the annealing, so we designed new primers where some nucleotides of the SfiI restriction site were part of the complementary region of the primers and others without the SfiI tail to have a positive control for the amplification.
Mini-prepping. I repeatedly had problems to get highly concentrated plasmid preparations. I used mini-prep kits, so I did not know what my mistake was. I tried different kits and Fernan supervised my work one day, but the error was not obvious. By the end of the week, when one of the solutions of the kit ran out and I had to change it, my mini-preps started to rise their concentration.
Not amplifying resistance cassettes. I tried to amplify kanamycin and chloramphenicol resistance cassettes from pSB4K5 and pSB1C3 vectors, but after three PCR reactions changing conditions I did not get an useful PCR product.
Preparing BioBricks to insert them into mini-Tn7. I set up inocula, did mini-preps of the plasmids and digested them with EcoRI and PstI restriction enzymes. The next step would be isolate from gel, but the concentration was too low to use the fragments for cloning; so I repeated the digestions with high amounts of DNA, but the results were similar. I need mini-preps with higher DNA concentrations.
Synthetizing a portable attTn7. I performed two PCR reactions changing dNTPs concentration and annealing temperature to amplify the insertion site of the Tn7 transposon from the E. coli chromosome, but I did not get any amplification. I preformed this process as a colony PCR (picking a colony to get the DNA template), so next week I will try to amplify the desire region of the E. coli chromosome by purifying the genomic DNA first.
Working on our own-designed mini-Tn7. I performed the first necessary steps to do with a synthesis order: resuspending, transformation in DH5α, plating, setting up inocula and mini-prepping (this was one of the mini-preps of the last day so I got a highly concentrated preparation).
Week 5 (August, 1-5)
Last week in the lab before vacation!
Amplifying the pUC18R6KT vector from pTNS2. Using new primers I performed a gradient PCR in a Taq polymerase catalyzed reaction. This time I got the expected PCR product independently of the annealing temperature; and I isolated bands from gel. This was possible due to improved primer design, and by switching from Pfu to Taq polymerase. Finally, after innumerable problems with the template, dNTPs, enzymes, primers… I got the pUC18R6KT vector.
Amplifying resistance cassettes. I tried to amplify the kanamicin and chloramphenicol resistance cassettes in a Pfu-polymerase catalyzed PCR reaction without positive results. Again, I obtained the expected results when I switched to Taq polymerase. I isolated and purified these PCR products from an agarose gel.
Preparing BioBricks to insert them into mini-Tn7. I digested the desired BioBricks with EcoRI and PstI trying to use the highest amount of DNA possible, but the mini-preps were not concentrated enough and I could only isolate from gel a couple of BioBricks properly digested. I used the traditional alkaline lysis-ethanol precipitation plasmid DNA purification method instead of using a commercial column kit and this time I obtained really high concentration mini-preps, although they were probably less clean. I repeated the digestions with the new samples, but gel electrophoresis revealed that, even though the DNA concentration was good, a large amount of RNA was also present. I isolated and purified the BioBrick fragments that there were not masked by the RNA smear at the bottom part of the lanes.
Synthetizing a portable attTn7. I purified genomic DNA from the MC4100 E. coli strain to use it as a template to amplify attTn7, the insertion site of Tn7 transposons. The PCR reaction resulted in a PCR product of the expected size, which was isolated and purified from an agarose gel. Because the primers added the standard prefix and suffix, the attTn7 could be digested with EcoRI and PstI. This fragment was ligated with EcoRI- and PstI-digested pSB1C3 and pSB4K5, and transformed into DH5α competent cells.
Construction of plasmids containing mini-Tn7. I digested a previously prepared mini-prep of the commercial plasmid containing the synthetic mini-Tn7 construct with SfiI. The SfiI fragment containing the transposon was ligated with SfiI-digested pUC18-SfiI or PCR-amplified and SfiI-digested pUC18R6KT-SfiI, and transformed in DH5α and DH5α-λpir, respectively.
This was the most successful week by now. I obtained most of the results I had been trying since I started and I went on vacation with a smile in my face.
Week 6 (September, 1-2)
Back to work! After three weeks on vacation it was hard to return to the lab, but I was overexcited to face the home stretch to the Regionals. In those two first days of September I mostly checked some results from August and prepared new materials.
Checking the clones containing the portable attTn7 by colony PCR. I had transformed pSB1C3 + attTn7 and pSB4K5 + attTn7 ligations in DH5α but I only obtained colonies for the former construct. The control plates looked OK. So I tested eleven candidates by colony PCR using the pSB1C3-attTn7 plates and the standard primers for prefix and suffix. The PCR products were analyzed in an electrophoretic gel and most of them showed the expected 208 bp attTn7 band. I restreaked five positive candidates onto fresh plates.
Cleaning chloramphenicol and kanamycin resistance cassettes. The gel-purified PCR products bearing resistance cassettes were digested at the flanking NcoI sites. However, we observed that these preparations did not freeze at -20, and we suspected ethanol contamination. Because of that, I discarded the digestion reactions and cleaned the DNA, analyzing the cleaning results in an agarose gel and measuring the DNA concentration.
Cleaning BioBrick mini-preps and digestions. Previous minipreps of the miniTn7 BioBrick inserts contamined with RNA were treated with RNAse and then cut with EcoRI and PstI. The digested BioBricks were isolated from gel and purified, but the resulting preparations had low DNA concentration.
Checking plasmids containing synthetic mini-Tn7 transposon. The pUC18-SfiI-miniTn7-Gm plasmid was checked by SfiI digestion with positive results. On the other hand, the selective plates from the pUC18R6KT-miniTn7-Gm ligation did not show any colonies. I checked the DNA samples used in gel and they showed the expected pattern so the problem must be in the competent cells. We planned to prepare new electro-competent and chemically competent DH5α-λpir cells.
Streaking out frozen strains. Before I went on vacation, I froze away most of the strains I had used, so I had to streak them out to work from now on.
Week 7 (September, 5-9)
The wiki freeze deadline is coming closer! So I have been working all the time I could, nine-ten hours a day in average, and thankfully it was a productive week.
Finishing the preparation of the resistance cassettes. The previously purified Cm and Km resistance cassettes were digested with NcoI. Those digested fragments are perfect to create miniTn7 variants of the basic transposon construct.
Finishing the preparation of BioBrick inserts. The BioBrick mini-preps were digested with EcoRI and PstI. As in previous digestions, all the BioBricks showed the expected size, except for BBa_I732094. We skipped this part because it was not essential for us. The DNA bands were isolated from gel and purified. These digested parts are intended for the construction of mini-Tn7 variants with specialized functions.
Checking the identity of the portable attTn7 by digestion. I performed EcoRI-PstI digestions with mini-preps from previous colony PCR positive clones. Unfortunately, the amount of DNA was too low to reach definitive conclusions; so I repeated that digestion with more DNA and an additional analytical digestion with PvuII. The restriction patterns obtained were as expected, thus confirming the presence of attTn7 in pSB1C3.
Constructing the pUC18R6KT-miniTn7-Gm. This is the central construct of the miniTn7 BioBrick tool kit, so I performed all the experiments I could in order to obtain it. I did new mini-preps of miniTn7-Gm, performed new digestions and ligations (pUC18R6KT + miniTn7-Gm) and did three different electroporations: one with previously prepared DH5α-λpir electro-competent cells, again without positive results; a second one with freshly prepared prepared DH5α-λpir electro-competent cells, obtaining only one colony; and a third electroporation with DH5α-λλpir electro-competent cells prepared with a different protocol. In the latter experiment I obtained a reasonable number of colonies on the selective plates. I set up inocula, mini-prepped and performed two different digestions (SfiI and BglI) to analyze the results. The amount of DNA was too low and because of that I could not reach any conclusions.
Characterizing pUC18-SfiI-miniTn7-Gm. The aim of this characterization is determine the orientation of the miniTn7-Gm inside pUC18-SfiI, measure the frequency of transposition in Pseudomonas putida KT2440, and check the site-specific insertion of the transposon at the end of the glmS gen. The orientation of the miniTn7 was determined by three digestion reactions (EcoRI, PstI and ScaI) and electrophoretic gel analysis. The transposition frequency was tested by conjugation and electroporation, using Pseudomonas putida KT2440 as the host. These assays demonstrated a high transposition frequency particularly by conjugation (10-4 insertions/viable cell).
Week 8 (September, 12-16)
Penultimate week before wiki freeze. I kept on focus in obtain the pUC18R6KT-miniTn7BB-Gm plasmid (main construction of the Tn7 kit) and characterize the pUC18SfiI-miniTn7BB-Gm plasmid.
Preparing attTn7 for sequencing. I set up a couple of inocula for freezing and sequencing, mini-prepping previously.
Starting the characterization of attTn7. Considering that we only have achieved by then the minTn7 in pUC18SfiI vector (replicative in Enterobacteriaceae) we need to insert the attTn7 in a P. putida replicative vector to check the site-specific insertion of the miniTn7 in the attTn7. To this way the miniTn7 will be in a suicide vector and the attTn7 not. We are going to insert the attTn7 in the pBBR1-MCS4. That week I prepared mini-preps of pSB1C3-attTn7 and pBBR1-MCS4 and digested them with EcoRI and SpeI.
Constructing the pUC18R6KT-miniTn7-Gm. The previous week I could not reach any conclusions of the analytic digestions of a putative pUC18R6KT-miniTn7-Gm. I repeated the digestions with higher amounts of DNA and I realized that there were not positive results. Because of the lack of re-ligation of the vector we think it could be not well-cut. I repeated the digestions of pUC18R6KT PCR product and ligation with miniTn7-Gm. Then I performed three transformations: two with two different preparations of electro-competent DH5α-λpir cells and another with chemically competent DH5α-λpir cells. Only the plates of the chemical transformation had a reasonable number of colonies. I set up 24 inocula of the chemical transformation plates, doing replica in a Gm and Ap plate. The next day I did mini-preps only of the inocula whose replica grew (18). The mini-preps were digested with SfiI and BglI and four candidates showed the expected restriction patter. Finally I had got the main construction of the miniTn7 BioBrick tool kit.
Characterizing pUC18-SfiI-miniTn7-Gm. I preformed colony PCR to detect the site-specific insertion of the miniTn7 in the P. putida genome with 11/11 positives in conjugation product strains and 10/12 positives in electroporation product strains. Repetitions of the electroporation and conjugation to measure transposition frequency were made, but I made mistakes diluting before plating and the results were not usable.
Characterizing the resistance excision by Flp recombinase. We were to use the pFLP2 plasmid, which carried the Flp recombinase or flipase gen, to recombine the FRT cassettes who flank the Gm resistance cassette of pUC18SfiI-miniTn7BB-Gm. I followed the protocol for recombination by flipase until patching in different selective plates because there was growth in every patch, what means that the Gm cassette was not excised. Thus, I started again.
Creating variants of pUC18SfiI-miniTn7BB-Gm. The objective was to include BioBricks into the BCS of the miniTn7 and change its resistance cassette. I digested pUC18SfiI-miniTn7BB-Gm with EcoRI/PstI and NcoI; ligated the plasmid with previously digested and isolated BioBricks and resistance cassettes; and transformed the ligations in DH5α competent cells. The next day the number of colonies was low. Colony PCR was performed and there was just one positive: pUC18SfiI-miniTn7BB-Km.
Week 9 (September 19-21)
Characterization of pUC18R6KT-miniTn7BB-Gm. In order to determine the orientation of insertion of the miniTn7-Gm in the pUC18R6KT vector I performed two analytic digestions (EcoRI+HindIII and ScaI). All four strains showed the same orientation of the miniTn7. Thankfully it is the same orientation I entered the plasmid in the Part’s Registry. In order to determine the transposition efficiency of this plasmid I co-transformed DH5α competent cells with pTNS2 and pUC18R6KT-miniTn7BB-Gm. Plate Counting revealed a transposition efficiency of 4x102 transpositions/μg of plasmidic DNA. The transformation frequency was 1x108 cfu/μg of plasmidic DNA. The site-specific insertion of the miniTn7BB-Gm was checked by colony PCR, resulting in 11 out of 12.
Characterizing the resistance excision by Flp recombinase. I followed the protocol for recombination by flipase, making dilutions in plasmid selection plates and then patching in Gm plates and LB plates. This assay showed six strains unable to grow in Gm plates, what means that the Gm resistance cassette had been excised from the miniTn7BB-Gm previously inserted in the genome of the bacterium.
Checking pUC18Sfi-miniTn7BB-Km. I set up inocula, mini-prepped and digested (with NcoI) the colony PCR positive candidates of pUC18Sfi-miniTn7BB-Km. Electrophoretic gel analysis showed the expected patter. So the construction of this new plasmid is verified.
Characterizarion of the portable attTn7. I prepared DH5α/pSB1C3-attTn7 electro-competent cells and then co-transformed the cells with pTNS2 and pUC18R6KT-miniTn7BB-Gm. Plating was made in Cm and Gm plates. The next day the colonies had not grown enough to be able to perform colony PCR and check the insertion of the miniTn7 in the portable attTn7.
Not more information about my lab work until Regional Jamboree passes. It is time to prepare the poster and talk to show to all the European and African iGEM team our great work. Thanks for reading!
Week 10 (October 3-7)
We advance to World Championship!! The European Jamboree in Amsterdam has been incredible, full of experiences. After returning from Amsterdam we had a meeting to organize the work until the World Championship Jamboree. Now I am going to focus mainly on integrating the basic and improved flip-flops into Escherichia coli chromosome using the miniTn7 BioBrick tool kit.
Working with flip-flops. I prepared mini-preps of the basic and improved flip-flops, digested them with EcoRI and PstI and ligated with pUC18R6KT-miniTn7BB-Gm, also digested with the same enzymes. The ligation products were electroporated in DH5α-λpir electro-competent cells. I did this process twice without results, probably because of the low electroporation ratio of the competent cells.
Characterizing attTn7 BioBrick. In order to characterize this BioBrick I prepared electro-competent cells harboring pSB1C3-attTn7 plasmids and electroporated them with pTNS2 and pUC18R6KT-miniTn7BB-Gm. The results would be analyzed the next week.
Continuing characterizing pUC18R6KT-miniTn7BB-Gm. I already measured the transposition frequency of this miniTn7 module by chemical transformation. Then I attended to determine the transposition frequency by mating because this plasmid has an oriT. That week I performed the first steps to cause mating with two receptor strains: E. coli MC4100 and S. typhimurium LT2.
Week 11 (October 10-14)
Cloning the improved flip-flop into the miniTn7 module. Having not results electroporating ligation products, I used heat-shock transformation to try to clone the improved flip-flop into pUC18R6KT-miniTn7BB-Gm and pUC18Sfi-miniTn7-Gm. I got red colonies, mainly when ligating with the second vector, which were checked by analytic digestions. I got one positive candidate for each construction.
Characterizing attTn7. I tried to electroporate a DH5α strain carrying pSB1C3-attTn7 plasmid with pTNS2 and pUC18R6KT-miniTn7BB-Gm twice without achieving transformants in plate. Then I tried to do it by mating.
Characterizing pUC18R6KT-miniTn7BB-Gm. Mating assays with E. coli MC4100 and S. typhimurium LT2 as receptor strains was performed, obtaining frequencies of 4x10-5 transposition/viable cell and 3x10-6 transposition/viable cell respectively. Then the site-specific insertion at the chromosomal attTn7 was checked by PCR, resulting in 12/12 positive results in each strain.
Constructing new plasmid with specialize functions based on pUC18Sfi-miniTn7BB-Gm. A EcoRI/PstI digested pUC18Sfi-miniTn7BB-Gm plasmid was ligated with the follow BioBricks: BBa_K093005, BBa_I732094, BBa_I732083, BBa_I13521, BBa_K325909. After chemically transformation, there were colonies in most of the plates, some of them even red because of the presence of a cloned RFP protein. Colony PCR was performed using the standard prefix and suffix primers, obtaining positive results for BBa_I732094 and BBa_K093005. BBa_I13521 (constitutive RFP) was not confirmed by PCR, by due to the presence of red colonies its verification must be achieved soon.
Week 12 (October 17-21)
Integrating miniTn7BB-Gm-improved_flipflop into E. coli chromosome. The improved flip-flop was inserted into an E. coli strain with double deletion (required to control properly the function of the improved flip-flop) by co-electroporation with pTNS2. Site-specific insertion was checked by colony PCR. Then a positive strain was electroporated with pSB4A5-asRNA (improved flip-flop, module II).
Cloning the basic flip-flop in to the miniTn7BB module. The basic flip-flop was inserted into the miniTn7BB module of pUC18Sfi-miniTn7BB-Gm and pUC18R6KT-miniTn7-Gm by heat-shock transformation of competent E. coli cells.
Characterization of attTn7. I performed a mating assay using as donor pUC18R6KT-miniTn7BB-Gm, as transposition helper pTNS2, as mating helper pRK2013 and as a receptor DH5α/pSB1C3-attTn7. The biomass of the resulting LB-Cm-Gm plate was collected in order to purify the plasmid. This purified plasmid was transformed by heat-shock transformation into DH5α competent cells. Selection was performed in LB-Cm and LB-Cm-Gm plates, and the next day all plates had thousands of colonies. Using colonies from the LB-Cm plate, I performed patches into LB-Gm and LB-Cm plates. 97 out of 100 colonies were able to grow in Gm and Cm, and only 3 out of 100 could not grow selectively in the LB-Cm plate. The function of the portable attTn7 has been successfully characterized. Also, I performed analytic digestions to verified that the insertion occurred into the attTn7.
Checking the construction of new plasmids with specialize functions based on pUC18Sfi-miniTn7BB-Gm. pUC18Sfi-miniTn7BB-Gm-RBS+RFP, pUC18Sfi-miniTn7BB-Gm-LacZ+GFP and pUC18Sfi-miniTn7BB-Gm-RFP were verified by analytic digestions.
Week 13 (October 24-27)
Any new construction? That week I tried to construct more plasmids with specialized function for the miniTn7BB tool kit, but finally there was not time enough to finish them.
Integrating the basic flip-flop into E. coli chromosome. The basic flip-flop was integrated into the E. coli chromosome by mating, obtaining a frequency of 1,4 x 10-4 recipient cfu bearing a miniTn7BB-Gm insertion x viable recipient cfu.
24h assay. All the wetlab subteam worked in the performance of a 24h assay to completely characterize the function of the basic and improved flip-flop, taking data of optical density and fluorescence every 30 minutes.
Basic Flip Flop
Improved Flip Flop
Epigenetic Flip Flop
MiniTn7
Mathematical Modeling
Multiagent Modeling
Bioinformatics
Biobrick Creator
Wiki design
Follow @iGEMupoSevilla
Sponsors
Retrieved from " http://2011.igem.org/Team:UPO-Sevilla/Project/Notebook/MiniTn7 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
