"
Page
Discussion
View source
History
teams
Log in
 
Team:UPO-Sevilla/Project/Improving Flip Flop/Results/Strain Construct
From 2011.igem.org
Home
Project
Foundational Advances
Human Practices
Team
Overview
Data Page
Basic Flip Flop
Improved Flip Flop
Epigenetic Flip Flop
Applications
Parts
Notebook
Judging Criteria
Overview
MiniTn7
BioBrick Creator
Overview
Safety
Tornillos y Genes
BSIES
Randy Rettberg' s Seminar
Workshop
Science Fair
Week of Science
Media
Team Structure
Attributions
Acknowledgements
Home
Project
Improved Flip Flop
Experimental Results
Strain Construct
Strain Construct
In order to test the improved flip flop properly, we constructed a new strain of Escherichia coli based on E. coli X90 ΔSspB and other E. coli X90 ΔClpX (F’ lacIq lac’ pro’/ara Δ[lac-pro] nalA argE [am] rifr thi-1). E. coli X90 strain has two different problems which we should solve for the accurate control of the improved flip-flop: on the one hand, RybB deletion in the chromosome was performed to achieve the expression of the RybB gene only from the improved flip-flop device; on the other hand, due to the presence of lacIq within F’plasmid, whose expression could interfere in the bistability, we caused its lost.
We carried out the mentioned deletion to our Sspb- and Clpx- strains by using the lambda red protocol ( Datsenko & Wanner, 2000 ). We have disrupted the chromosomal gene RybB by using PCR products in which PCR primers provide the homology to the targeted gene. In this procedure, recombination requires the phage lambda Red recombinase, which is synthesized under the control of an arabinose inducible promoter, in a temperature sensitive and low copy number plasmid. We generated PCR products by using primers with 50-nt extension that are homologous to regions adjacent to the RybB gene. The template consisted on an antibiotic resistance cassette ﬂanked by FRT (FLP recognition target) sites. The antibiotic resistance cassette was then eliminated using a helper plasmid encoding the FLP recombinase, which is also temperature sensitive. Figure 1 shows a electrophoresis gel which demonstrates that we achieved the delection in our ΔSspB strain.
Figure 1: Agarose gel results of different steps of the lambda red protocol using the same primers, which flanked the chromosomal RybB gene. (A) Amplification of chromosomal RybB gene. (B) Amplification of the kanamicine resistance cassette that replace the RybB gene after lambda red recombination. (C) Amplification of the disrupted RybB gene after removing the resistance by FRT recombinase expression.
Besides, we achieved a strain without F’ plasmid by setting up a continuous culture without F’ plasmid selection and checking if any bacteria had eventually lost that plasmid. Finally the metabolic burden made its work and strains without this plasmid arose in the culture. The plasmid lost was checked by the lost of beta-galactosidase activity (X-gal plating) and proline auxotrophy (growth in minimal medium plates) whose characteristics were codified by the F' plasmid.
Overview
Data Page
Proteolysis
Inhibition system
Proteolysis regulation
Conclusions
asRNA
Inhibition system
Fusion proteins
Conclusions
Modeling
Introduction
Equations
Simulations
Bioinformatics
Docking SspB-ClpX
Tags Finder
RybB Structure
The Final Construct
Experimental Results
Strain Construct
Plasmids and controls
Change Speed
Stability
Follow @iGEMupoSevilla
Sponsors
Retrieved from " http://2011.igem.org/Team:UPO-Sevilla/Project/Improving_Flip_Flop/Results/Strain_Construct "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
