<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UPO-Sevilla_Project_Epigenetic_Flip_Flop_Bioinformatics"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UPO-Sevilla/Project/Epigenetic_Flip_Flop/Bioinformatics">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UPO-Sevilla/Project/Epigenetic_Flip_Flop/Bioinformatics&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UPO-Sevilla/Project/Epigenetic_Flip_Flop/Bioinformatics&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UPO-Sevilla/Project/Epigenetic_Flip_Flop/Bioinformatics&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UPO-Sevilla/Project/Epigenetic_Flip_Flop/Bioinformatics" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UPO-Sevilla/Project/Epigenetic Flip Flop/Bioinformatics</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="page"><DIV id="contentUPO"><DIV id="header"><DIV class="navegacion"><UL id="jsddm"><LI><A href="/Team:UPO-Sevilla">Home</A></LI><LI><A href="/Team:UPO-Sevilla/Project/Overview">Project</A></LI><LI><A href="/Team:UPO-Sevilla/Foundational_Advances">Foundational Advances</A></LI><LI><A href="/Team:UPO-Sevilla/Human_Practice">Human Practices</A></LI><LI><A href="/Team:UPO-Sevilla/Team">Team</A></LI></UL></DIV><DIV id="topHeader"><UL><LI><A id="menuPOverview" href="/Team:UPO-Sevilla/Project/Overview">Overview</A></LI><LI><A id="menuPDataPage" href="/Team:UPO-Sevilla/Project/Data_Page">Data Page</A></LI><LI><A id="menuPBasic" href="/Team:UPO-Sevilla/Project/Basic_Flip_Flop/Overview">Basic Flip Flop</A></LI><LI><A id="menuPImproved" href="/Team:UPO-Sevilla/Project/Improving_Flip_Flop/Overview">Improved Flip Flop</A></LI><LI><A id="menuPEpigenetic" href="/Team:UPO-Sevilla/Project/Epigenetic_Flip_Flop/Overview">Epigenetic Flip Flop</A></LI><LI><A id="menuPApplicattions" href="/Team:UPO-Sevilla/Project/Applications">Applications</A></LI><LI><A id="menuPParts" href="/Team:UPO-Sevilla/Project/Parts">Parts</A></LI><LI><A id="menuPNotebook" id="menuP" href="/Team:UPO-Sevilla/Project/Notebook">Notebook</A></LI><LI><A id="menuPJudging" href="/Team:UPO-Sevilla/Project/Judging_Criteria">Judging Criteria</A></LI></UL><UL><LI><A id="menuFAOverview" href="/Team:UPO-Sevilla/Foundational_Advances">Overview</A></LI><LI><A id="menuFAMiniTn7" href="/Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Overview">MiniTn7</A></LI><LI><A id="menuFACreator" href="/Team:UPO-Sevilla/Foundational_Advances/Biobrick_Creator/Overview">BioBrick Creator</A></LI></UL><UL><LI><A id="menuHPOverview" href="/Team:UPO-Sevilla/Human_Practice">Overview</A></LI><LI><A id="menuHPSafety" href="/Team:UPO-Sevilla/Human_Practice/Safety">Safety</A></LI><LI><A id="menuHPBlog" href="/Team:UPO-Sevilla/Human_Practice/Tornillos_y_genes">Tornillos y Genes</A></LI><LI><A id="menuHPBSIES" href="/Team:UPO-Sevilla/Human_Practice/BSIES">BSIES</A></LI><LI><A id="menuHPSeminar" href="/Team:UPO-Sevilla/Human_Practice/Randy_Rettberg_Seminar">Randy Rettberg' s Seminar</A></LI><LI><A id="menuHPWorkshop" href="/Team:UPO-Sevilla/Human_Practice/Workshop">Workshop</A></LI><LI><A id="menuHPScienceFair" href="/Team:UPO-Sevilla/Human_Practice/Science_Fair">Science Fair</A></LI><LI><A id="menuHPWeek" href="/Team:UPO-Sevilla/Human_Practice/Week_of_Science">Week of Science</A></LI><LI><A id="menuHPMedia" href="/Team:UPO-Sevilla/Human_Practice/Media">Media</A></LI></UL><UL><LI><A id="menuTStructure" href="/Team:UPO-Sevilla/Team">Team Structure</A></LI><LI><A id="menuTAttributions" href="/Team:UPO-Sevilla/Team/Attributions">Attributions</A></LI><LI><A id="menuTAcknowledgements" href="/Team:UPO-Sevilla/Team/Acknowledgements">Acknowledgements</A></LI></UL></DIV></DIV><DIV id="principal"><DIV class="main"><UL id="breadcrumbs" class="xbreadcrumbs"><LI class=""><A class="home" href="/Team:UPO-Sevilla" style="white-space: nowrap; float: left; overflow: hidden; width: 40px; display: block;" init-width="29">Home</A></LI><LI><A href="/Team:UPO-Sevilla/Project/Overview" style="white-space: nowrap; float: left;">Project</A></LI><LI><A href="/Team:UPO-Sevilla/Project/Epigenetic_Flip_Flop/Overview" style="white-space: nowrap; float: left;">Epigenetic Flip Flop</A></LI><LI class="current"><A href="/Team:UPO-Sevilla/Project/Epigenetic_Flip_Flop/Bioinformatics" style="white-space: nowrap; float: left;">Bioinformatics</A></LI></UL><H1>Bioinformatics</H1><H2>Goal</H2><P>To elucidate if the recombinant proteins generated by this sublab are going to be <STRONG>functional</STRONG>.</P><H2>Background</H2><P>When performing a protein structure prediction by homology modelling strategies, we usually obtain several <STRONG>independent</STRONG> models. Each of these models, corresponds to one of the proteins which have been linked together to form the recombinant one. Thus, <STRONG>it is not possible</STRONG> to obtain a complete single structure for a recombinant protein only by using a protein homology modelling server.</P><H2>Strategy</H2><P>In order to study the possible structure and functionality of the fusion proteins described in this flip-flop system, an <STRONG>indirect docking modelling approach</STRONG> has been carried out. Our approach consists in performing a docking experiment in which we introduce the pdb model of every protein fused in the recombinant main protein. Once the docking method has been performed, the final peptide range of the first protein fused (about the last 30 amino acids) and the initial amino acids of the second protein fused are located in the resulting pdb file, to study any possible interactions. Interactions between these parts of the proteins would mean that the recombinant protein could acquire a native folding structure which would have negative influences to its functionality.</P><H2>Procedure</H2><OL><LI><P><STRONG>Get a pdb model for each protein which is going to be fused.</STRONG> To pursue this, the protein homology modeling <A href="http://swissmodel.expasy.org/">server Swiss Model</A> and <A href="http://www.cbs.dtu.dk/services/CPHmodels/">CPHmodels 3.0 server</A> were used to obtain one putative structure for each protein. The results were compared and contrasted, and the best models were chosen. So, model structures were obtained for the TetR protein and for the Swi6 CDS and the Sir3 BAM protein domain.</P></LI><LI><P><STRONG>Docking experiment with the composing proteins of each recombinant one.</STRONG> In this case, the <A href="http://hex.loria.fr/">Hex protein docking server</A> was used, and the three best results were downloaded (the best result is that one with the lowest level of free energy). The server was run twice, a first time considering only shapes and a second time considering shapes and electrostatics.</P></LI><LI><P><STRONG>Analyse the docking results and locate the final and initial fragment of the fused proteins in order to predict possible interactions.</STRONG> To visualize and locate these peptidic ranges, the <A href="http://rasmol.org/">RasMol 2.7.5.</A> viewer was used.</P></LI></OL><H2>Results and Discussion</H2><OL><LI><P><STRONG>Protein Docking 1 (TetR-Sir3BAM domain)</STRONG></P><P>Figure 1 displays the three best shape and shape+electrostatics docking results. In this image, TetR protein is shown in blue with its last amino acids coloured in yellow. The Sir3 BAM protein domain is shown in red with its first 30 amino acids coloured in green. The free energy of the bindings are:</P><TABLE class="parameters" cellpadding="0" cellspacing="0"><TBODY><TR><TH>Docking Free Energy</TH><TH>Docking shape only</TH><TH>Docking shape+electrostatics</TH></TR><TR><TD>Best Result 1</TD><TD>-627.19 Kcal/mol</TD><TD>-834.17 Kcal/mol</TD></TR><TR><TD>Best Result 2</TD><TD>-605.44 Kcal/mol</TD><TD>-750.62 Kcal/mol</TD></TR><TR><TD>Best Result 3</TD><TD>-598.25 Kcal/mol</TD><TD>-741.00 Kcal/mol</TD></TR></TBODY></TABLE><P><STRONG>Figure 1.</STRONG> Docking models for the TetR-BAMSir3 interaction. The best three shape docking results are displayed in the top row while the best shape+electrostatics docking results are shown in the row below.</P><P>Taking everything into consideration, it seems that there is a <STRONG>high possibility of interaction</STRONG> between the TetR protein and the functional Sir3 BAM protein domain. This is due to the lowest three free energy results (for both shape and shape+electrostatics) can also acquire a <STRONG>stable structure</STRONG> in which the linked parts in the recombinant protein are quite close. These possible interactions <STRONG>would reduce the functionality</STRONG> of our fusion protein TetR::Sir3.</P></LI><LI><P><STRONG>Protein Docking 2 (TetR-Swi6CDS domain)</STRONG></P><P>
Figure 2 displays the three best shape and shape+electrostatics docking results. In this image, TetR protein is shown in blue with its last amino acids coloured in yellow. The Swi6 chromo shadow domain (CDS) is shown in red with its first 30 amino acids coloured in green. The free ebergy of the bindings are:</P><TABLE class="parameters" cellpadding="0" cellspacing="0"><TBODY><TR><TH>Docking Free Energy</TH><TH>Docking shape only</TH><TH>Docking shape+electrostatics</TH></TR><TR><TD>Best Result 1</TD><TD>-627.36 Kcal/mol</TD><TD>-653.73 Kcal/mol</TD></TR><TR><TD>Best Result 2</TD><TD>-612.57 Kcal/mol</TD><TD>-630.66 Kcal/mol</TD></TR><TR><TD>Best Result 3</TD><TD>-604.63 Kcal/mol</TD><TD>-607.76 Kcal/mol</TD></TR></TBODY></TABLE><P><STRONG>Figure 2.</STRONG> Docking models for the TetR-Sir3CDS interaction. The best three shape docking results are displayed in the top row while the best shape+electrostatics docking results are shown in the row below.</P><P>In this case, the free interaction energy resulted in <STRONG>closer</STRONG> figures, which means that all the showed models must be taken into consideration. From all of this information as well as the obtained interaction structure models, it seems that there is a <STRONG>lower possibility of interaction</STRONG> in the case of TetR protein and the Swi6 chromo shadow domain than in the case of the TetR protein and the BAM Sir3 protein domain. As a consequence, the recombinant protein TetR::Swi6CDS <STRONG>should fold itself in an appropriate way</STRONG> to allow individual domain functionality.</P></LI></OL></DIV><DIV class="left"><UL id="menu"><LI><A id="menuVEOverview" class="menuLi" href="/Team:UPO-Sevilla/Project/Epigenetic_Flip_Flop/Overview">Overview</A></LI><LI><A id="menuVEDataPage" class="menuLi" href="/Team:UPO-Sevilla/Project/Epigenetic_Flip_Flop/Data_Page">Data Page</A></LI><LI><A id="menuVEBackground" class="menuLi" href="/Team:UPO-Sevilla/Project/Epigenetic_Flip_Flop/Background">Background</A></LI><LI><A id="menuVEStrategy" class="menuLi" href="/Team:UPO-Sevilla/Project/Epigenetic_Flip_Flop/Strategy">Strategy</A></LI><LI><A id="menuVEDesigning" class="menuLi" href="/Team:UPO-Sevilla/Project/Epigenetic_Flip_Flop/Designing">Designing</A></LI><LI><A id="menuVEBioinf" class="menuLi" href="/Team:UPO-Sevilla/Project/Epigenetic_Flip_Flop/Bioinformatics">Bioinformatics</A></LI><LI><A id="menuVEPandR" class="menuLi" href="/Team:UPO-Sevilla/Project/Epigenetic_Flip_Flop/Procedure_and_Results">Procedure and Results</A></LI></UL></DIV><DIV class="right"><DIV class="follows"><A href="http://twitter.com/iGEMupoSevilla" class="twitter-follow-button">Follow @iGEMupoSevilla</A></DIV></DIV></DIV></DIV><DIV id="footerUPO"><H3>Sponsors</H3></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:UPO-Sevilla/Project/Epigenetic_Flip_Flop/Bioinformatics">http://2011.igem.org/Team:UPO-Sevilla/Project/Epigenetic_Flip_Flop/Bioinformatics</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UPO-Sevilla/Project/Epigenetic_Flip_Flop/Bioinformatics" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UPO-Sevilla/Project/Epigenetic_Flip_Flop/Bioinformatics" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UPO-Sevilla/Project/Epigenetic_Flip_Flop/Bioinformatics&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UPO-Sevilla/Project/Epigenetic_Flip_Flop/Bioinformatics&amp;oldid=258008" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>