"
Page
Discussion
View source
History
teams
Log in
 
Team:UPO-Sevilla/Project/Epigenetic Flip Flop/Designing
From 2011.igem.org
Home
Project
Foundational Advances
Human Practices
Team
Overview
Data Page
Basic Flip Flop
Improved Flip Flop
Epigenetic Flip Flop
Applications
Parts
Notebook
Judging Criteria
Overview
MiniTn7
BioBrick Creator
Overview
Safety
Tornillos y Genes
BSIES
Randy Rettberg' s Seminar
Workshop
Science Fair
Week of Science
Media
Team Structure
Attributions
Acknowledgements
Home
Project
Epigenetic Flip Flop
Designing
Designing
The epigenetic bistable has two main modules: reporter module and compaction module.
Reporter module
Reporter Module contains a GFP inducible expression system, flanked by tetR operator sequences. This module will be integrated and expressed from various regions of the S. pombe genome with different heterochromatin contexts, to test it effect on silencing. We expect that once the protein Sir3, Swi6 or CSD binds the targeting tetR operator sequences they will induce the compaction of the promoter, or at least, prevent its leaky activation, because of the spreading of the engineered silencing proteins and it cooperativity. Additionally, there have been introduce two boundaries or chromatin insulators each site of our system. They are transcriptional terminator that  act as neutral transcriptionally elements that prevent negative or positive influence from extending across chromatin to a promoter ( Fourel et al, 2001 ; Bushey et al, 2008 ; Bähler et al, 1998 ; Watt et al, 2008 ).
Figure 1. Reporter module: Kan-MX6 (selectable cassette for integration in leu1), Adh1 terminator (upstream insulator), tetO2 (upstream operator sites), urg1 promoter, GFP marker gene, Adh1 terminator, tetO4 (downstream operator sites), Act1 terminator (downstream insulator).
Compaction module
It is composed by nmt1 (no message in thiamine) repressible promoter, three alternative proteins (Sir3p, Swi6 and the Chromo-Shadow Domain of Swi6 “CSD” ( Cowieson et al, 2000 ; Haldar et al, 2011 ), fused to the Tetracycline Repressor protein (tetR), and nmt Terminator. These modules will be expressed from a plasmid containing an independent inducible promoter, pREP41X ( Forsburg, 1993 ; Moreno et al, 2000 ).  These fusion proteins or kymeras will bind to specific tetR operator sequences placed upstream of an inducible promoter and downstream of the marker GFP gene, controlled by that promoter.
Figure 2. Alternative compaction modules, composed by Pnmt1 promoter, engineered silencing proteins (tetR+Swi6/CSD/Sir3) and nmt1 terminator. Swi6 belongs to S. pombe and Sir3 belongs to Saccharomyces cerevisiae.
Bioinformatics notes: Key proteins
1. TetR ( Uniprot: D8E9H0 )
Table 1. Relevant information of TetR protein from Uniprot protein database entry D8E9H0 and TetR Pfam database.
Figure 3. TetR structure predicted by Swiss-Model homology modelling server. Pfam domain annotation performed by using the Jmol viewer.
2. Swi6 ( Uniprot: P40381 )
Table 2. Relevant information of Swi6 protein from Uniprot protein database entry P40381 and Swi6 Pfam database entry Swi6 Pfam .
Figure 4. Swi6 CSD domain structure. Protein Data Bank (PDB) entry 1e0b .
3. Sir3 ( Uniprot: P0670 )
Table 3. Relevant information of Sir3 protein from Uniprot protein database entry P0670 and Sir3 Pfam database entry Sir3 Pfam .
Figure 5. Sir3 BAM domain structure predicted by Swiss-Model homology modelling server.
Overview
Data Page
Background
Strategy
Designing
Bioinformatics
Procedure and Results
Follow @iGEMupoSevilla
Sponsors
Retrieved from " http://2011.igem.org/Team:UPO-Sevilla/Project/Epigenetic_Flip_Flop/Designing "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
