<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UPO-Sevilla_Foundational_Advances_MiniTn7_Bioinformatics_attTn7_Insertion_Site"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Bioinformatics/attTn7_Insertion_Site">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Bioinformatics/attTn7_Insertion_Site&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Bioinformatics/attTn7_Insertion_Site&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Bioinformatics/attTn7_Insertion_Site&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Bioinformatics/attTn7_Insertion_Site" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UPO-Sevilla/Foundational Advances/MiniTn7/Bioinformatics/attTn7 Insertion Site</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="page"><DIV id="contentUPO"><DIV id="header"><DIV class="navegacion"><UL id="jsddm"><LI><A href="/Team:UPO-Sevilla">Home</A></LI><LI><A href="/Team:UPO-Sevilla/Project/Overview">Project</A></LI><LI><A href="/Team:UPO-Sevilla/Foundational_Advances">Foundational Advances</A></LI><LI><A href="/Team:UPO-Sevilla/Human_Practice">Human Practices</A></LI><LI><A href="/Team:UPO-Sevilla/Team">Team</A></LI></UL></DIV><DIV id="topHeader"><UL><LI><A id="menuPOverview" href="/Team:UPO-Sevilla/Project/Overview">Overview</A></LI><LI><A id="menuPDataPage" href="/Team:UPO-Sevilla/Project/Data_Page">Data Page</A></LI><LI><A id="menuPBasic" href="/Team:UPO-Sevilla/Project/Basic_Flip_Flop/Overview">Basic Flip Flop</A></LI><LI><A id="menuPImproved" href="/Team:UPO-Sevilla/Project/Improving_Flip_Flop/Overview">Improved Flip Flop</A></LI><LI><A id="menuPEpigenetic" href="/Team:UPO-Sevilla/Project/Epigenetic_Flip_Flop/Overview">Epigenetic Flip Flop</A></LI><LI><A id="menuPApplicattions" href="/Team:UPO-Sevilla/Project/Applications">Applications</A></LI><LI><A id="menuPParts" href="/Team:UPO-Sevilla/Project/Parts">Parts</A></LI><LI><A id="menuPNotebook" id="menuP" href="/Team:UPO-Sevilla/Project/Notebook">Notebook</A></LI><LI><A id="menuPJudging" href="/Team:UPO-Sevilla/Project/Judging_Criteria">Judging Criteria</A></LI></UL><UL><LI><A id="menuFAOverview" href="/Team:UPO-Sevilla/Foundational_Advances">Overview</A></LI><LI><A id="menuFAMiniTn7" href="/Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Overview">MiniTn7</A></LI><LI><A id="menuFACreator" href="/Team:UPO-Sevilla/Foundational_Advances/Biobrick_Creator/Overview">BioBrick Creator</A></LI></UL><UL><LI><A id="menuHPOverview" href="/Team:UPO-Sevilla/Human_Practice">Overview</A></LI><LI><A id="menuHPSafety" href="/Team:UPO-Sevilla/Human_Practice/Safety">Safety</A></LI><LI><A id="menuHPBlog" href="/Team:UPO-Sevilla/Human_Practice/Tornillos_y_genes">Tornillos y Genes</A></LI><LI><A id="menuHPBSIES" href="/Team:UPO-Sevilla/Human_Practice/BSIES">BSIES</A></LI><LI><A id="menuHPSeminar" href="/Team:UPO-Sevilla/Human_Practice/Randy_Rettberg_Seminar">Randy Rettberg' s Seminar</A></LI><LI><A id="menuHPWorkshop" href="/Team:UPO-Sevilla/Human_Practice/Workshop">Workshop</A></LI><LI><A id="menuHPScienceFair" href="/Team:UPO-Sevilla/Human_Practice/Science_Fair">Science Fair</A></LI><LI><A id="menuHPWeek" href="/Team:UPO-Sevilla/Human_Practice/Week_of_Science">Week of Science</A></LI><LI><A id="menuHPMedia" href="/Team:UPO-Sevilla/Human_Practice/Media">Media</A></LI></UL><UL><LI><A id="menuTStructure" href="/Team:UPO-Sevilla/Team">Team Structure</A></LI><LI><A id="menuTAttributions" href="/Team:UPO-Sevilla/Team/Attributions">Attributions</A></LI><LI><A id="menuTAcknowledgements" href="/Team:UPO-Sevilla/Team/Acknowledgements">Acknowledgements</A></LI></UL></DIV></DIV><DIV id="principal"><DIV class="main"><UL id="breadcrumbs" class="xbreadcrumbs"><LI class=""><A class="home" href="/Team:UPO-Sevilla" style="white-space: nowrap; float: left; overflow: hidden; width: 40px; display: block;" init-width="29">Home</A></LI><LI><A href="/Team:UPO-Sevilla/Foundational_Advances" style="white-space: nowrap; float: left;">Foundational Advances</A></LI><LI><A href="/Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Overview" style="white-space: nowrap; float: left;">MiniTn7</A></LI><LI><A href="/Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Bioinformatics/attTn7_Insertion_Site" style="white-space: nowrap; float: left;">Bioinformatics</A></LI><LI class="current"><A href="/Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Bioinformatics/attTn7_Insertion_Site" style="white-space: nowrap; float: left;">attTn7 Insertion Site</A></LI></UL><H1>Is attTn7 insertion site higly conserved?</H1><H2>Goal</H2><P>To study the level of conservation of the Tn7 transposon recognition site (attTn7) over the phylogeny. </P><H2>Background</H2><P><STRONG>Tn7</STRONG> is quite a specific bacterial transposon which encodes five transposition proteins (TnS, A, B, C, D and E) (<A href="http://genesdev.cshlp.org/content/15/6/737.long" target="_blank">Peters et al, 2001</A>). TnsD is one of the targets selector which inserts Tn7. TnsD recognizes a specific chromosomal site, known as an <STRONG>attachment site</STRONG> (attTn7), and modifies the DNA structure allowing the binding of TnsC, TnsA and TnsB (<A href="http://genesdev.cshlp.org/content/2/2/137.long" target="_blank">Waddel and Craig, 1988</A>). These two last proteins insert the transposon. Therefore, insertion takes place preserving expression downstream the essential glmS gene (<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1146532/?tool=pubmed" target="_blank">Gay et al, 1986</A>). It has been reported that the TnsD binding site is found in E.coli in the last 36bp of the glmS ORF (<A href="http://www.nature.com/emboj/journal/v20/n4/full/7593597a.html" target="_blank">Kuduvalli et al, 2001</A>). This site encodes the active site region of <STRONG>GlmS</STRONG> whose amino acid sequence is nearly completely <STRONG>100% conserved</STRONG> in all organisms (<A href="http://www.sciencedirect.com/science/article/pii/S0167483802003187" target="_blank">Milewski, 2002</A>). However, when considering nucleotide sequences, this attTn7 site has not been totally characterized in a wide range of organisms. In 2010, Rupak et al. reported a schematic representation of the nucleotide attTn7 recognition site obtained from the alignment of 25 bacterial glmS genes which have the Tn7 family transposon and the homologous Drosophila, zebrafish and human genes (<A href="http://www.mobilednajournal.com/content/1/1/18" target="_blank">Rupak et al, 2010</A>). 
</P><P class="" caption=""><STRONG>Figure 1</STRONG>. (from <A href="http://genesdev.cshlp.org/content/15/6/737.long" target="_blank">Prasad et al, 2001</A>). Working model for the TnsABC+D pathway. attTn7 is indicated by the rectangle, the relative positions of TnsD, TnsC and the transposase TnsAB are marked by ovals, and the donor DNA is also shown at the insertion site, which is denoted 0. The top portion of the rectangle indicates the major groove side of attTn7 and the bottom side represents the minor groove face of attTn7. Also illustrated in the box is the 5 bp duplication (underlined) that occurs on Tn7 insertion and the arrows show the positions of joining of the transposon ends, executed by the transposase (TnsAB) on attTn7.</P><P class="" caption=""><STRONG>Figure 2</STRONG>. The organization of attTn7 (from <A href="http://www.mobilednajournal.com/content/1/1/18" target="_blank">Rupak et al, 2010</A>). A schematic representation of the attTn7 at the C-terminus of the glmS gene with the TnsD binding site is shown. The sequence of the E. coli GlmS protein, and the consensus attTn7 sequence were derived as described in the text (note that the least conserved nucleotides correspond to the third position of each codon).
</P><H2>Strategy</H2><P>To improve and extend the study of attTn7 over phylogeny. Also, we will analyze its level of conservation with the final idea of making our <STRONG>miniTn7 BioBrick toolkit</STRONG> extensible to other organisms. Moreover, in the development of a complete synthetic Tn7 transposon toolkit, obtaining a <STRONG>consensus nucleotide attTn7 sequence</STRONG> might be also interesting (it would be included in our system). To achieve this, several database searches, BLASTs, multi sequence alignments, phylogenetic trees and sequence logos were performed. 
</P><H2>Procedure</H2><OL><LI><P>Obtaining a consensus nucleotide sequence from organisms which have the Tn7 family transposon.</P></LI><OL><LI><P><STRONG>Searching</STRONG> for organisms with Tn7 family transposon. Several BLASTs were run in the <A href="http://www.uniprot.org/" target="_blank">Uniprot database</A> BLAST server using as input E.coli Tn7 transposon proteins (for example TnsB). From the results, several organisms were chosen and it was checked on bibliography if Tn7 transposon had been found in the chosen organisms.</P></LI><LI><P>Glms protein sequences were <STRONG>downloaded</STRONG> from <A href="http://www.uniprot.org/" target="_blank">Uniprot database</A> and the corresponding codifying sequences were downloaded from (from <A href="http://www.ncbi.nlm.nih.gov/genbank/" target="_blank">GeneBank database</A>).</P></LI><LI><P><STRONG>Multialignment</STRONG> of all the protein and codifying sequences (see figure 3). In the case of the Glms protein sequences this multialignment was performed to verify the high level of conservation reported by Milewski (<A href="http://www.sciencedirect.com/science/article/pii/S0167483802003187" target="_blank">Milewski, 2002</A>). This sequences alignment was performed by using the <A href="http://www.clustal.org/" target="_blank">software ClustalX2</A>.</P></LI><LI><P><A href="http://pbil.univ-lyon1.fr/software/seaview.html" target="_blank">SeaVieW 4.3.0 sofware </A> was used to obtain both a circular and a squared <STRONG>phylogenetic tree</STRONG> (see figure 4). Phylogenetic trees are an interesting tool which enables a better visualization of the genetic distances among the genetic sequences under study.</P></LI><LI><P>The last 38 nucleotide bases of the glmS gene (the conserved part which corresponds to attTn7 recognition site) were introduced in the <A href="http://weblogo.threeplusone.com/" target="_blank">WebLogo3 server</A> to create a <STRONG>logo</STRONG> (figure 3). </P></LI><LI><P>The obtained consensus logo sequence was <STRONG>translated</STRONG> by using the <A href="http://web.expasy.org/translate/" target="_blank">ExPaSy Translate Tool </A> to create a <STRONG>logo</STRONG>. </P></LI></OL><LI><P>Same process but with <STRONG>species from all kingdoms</STRONG>, model organisms preferentially. It should be mentioned that in most of these organisms, the Tn7 family transposon has not been found.</P></LI><OL><LI><P><STRONG>Searching</STRONG> for organisms with Glms protein. BLASTs were run by introducing the E.coli Glms protein. As most of the results were fungus and bacteria, another BLAST was run with the Glms human protein as input. In this second case, most of the results were eukaryotic organisms. Organisms representing the main phylogenetic groups and model organisms were chosen.</P></LI><LI><P>Glms protein sequences were <STRONG>downloaded</STRONG> from <A href="http://www.uniprot.org/" target="_blank">Uniprot database</A> and the corresponding codifying sequences were downloaded from <A href="http://www.ncbi.nlm.nih.gov/genbank/" target="_blank">GeneBank</A>.</P></LI><LI><P><STRONG>Multialignment</STRONG> of all the protein and codifying sequences. Sequences alignments were performed by using the <A href="http://www.clustal.org/" target="_blank">software ClustalX2</A></P></LI><LI><P><A href="http://pbil.univ-lyon1.fr/software/seaview.html" target="_blank">SeaVieW 4.3.0 sofware </A> was used to obtain both a circular and a squared <STRONG>phylogenetic tree</STRONG>.</P></LI><LI><P>The last 50 nucleotide bases of the glmS gene were introduced to create a <STRONG>logo</STRONG> in the <A href="http://weblogo.threeplusone.com/" target="_blank">WebLogo3 server</A>.</P></LI><LI><P>The obtained consensus logo sequence was <STRONG>translate</STRONG>d by using the <A href="http://web.expasy.org/translate/" target="_blank">ExPaSy Translate Tool </A> to create a <STRONG>logo</STRONG>.</P></LI></OL><LI><P><STRONG>Comparison</STRONG> of the obtained sequences in both cases.</P></LI></OL></DIV><DIV class="left"><UL id="menu"><LI><A id="menuTN7Overview" class="menuLi" href="/Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Overview">Overview</A></LI><LI><A id="menuTN7DataPage" class="menuLi" href="/Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Data_Page">Data Page</A></LI><LI><A id="menuTN7Bio" class="menuLi" href="/Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Bioinformatics/attTn7_Insertion_Site">Bioinformatics</A></LI><LI><A id="menuTN7BInsertion" href="/Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Bioinformatics/attTn7_Insertion_Site">attTn7 Insertion Site</A></LI><LI><A id="menuTN7BOrgnism" href="/Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Bioinformatics/Tn7_organisms">Results. Tn7 organisms</A></LI><LI><A id="menuTN7BWholeP" href="/Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Bioinformatics/Whole_phylogeny">Results. Whole phylogeny</A></LI><LI><A id="menuTN7BConclusions" href="/Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Bioinformatics/Conclusions">Conclusions</A></LI></UL><LI><A id="menuTN7ER" class="menuLi" href="/Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Experimental_Results/Design_and_Synthesis">Experimental Results</A></LI><LI><A id="menuTN7ERDesign" href="/Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Experimental_Results/Design_and_Synthesis">Design &amp; Synthesis </A></LI><LI><A id="menuTN7ERVectors" href="/Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Experimental_Results/Constructions_of_Vectors">Constructions of Vectors</A></LI><LI><A id="menuTN7ERCharacterization" href="/Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Experimental_Results/Characterization">Characterization</A></LI><LI><A id="menuTN7ERderivatives" href="/Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Experimental_Results/miniTn7BB_derivatives">miniTn7BB derivatives</A></LI><LI><A id="menuTN7ERtarget" href="/Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Experimental_Results/attTn7_target_site">attTn7 target site</A></LI><LI><A id="menuTN7ERflipflop" href="/Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Experimental_Results/MiniTn7_and_flip-flops">miniTn7 &amp; flip-flops</A></LI></DIV><DIV class="right"><DIV class="follows"><A href="http://twitter.com/iGEMupoSevilla" class="twitter-follow-button">Follow @iGEMupoSevilla</A></DIV></DIV></DIV></DIV><DIV id="footerUPO"><H3>Sponsors</H3></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Bioinformatics/attTn7_Insertion_Site">http://2011.igem.org/Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Bioinformatics/attTn7_Insertion_Site</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Bioinformatics/attTn7_Insertion_Site" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Bioinformatics/attTn7_Insertion_Site" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Bioinformatics/attTn7_Insertion_Site&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Bioinformatics/attTn7_Insertion_Site&amp;oldid=250340" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>