"
Page
Discussion
View source
History
teams
Log in
 
Team:UPO-Sevilla/Project/Improving Flip Flop/Proteolysis/Inhibition system
From 2011.igem.org
Home
Project
Foundational Advances
Human Practices
Team
Overview
Data Page
Basic Flip Flop
Improved Flip Flop
Epigenetic Flip Flop
Applications
Parts
Notebook
Judging Criteria
Overview
MiniTn7
BioBrick Creator
Overview
Safety
Tornillos y Genes
BSIES
Randy Rettberg' s Seminar
Workshop
Science Fair
Week of Science
Media
Team Structure
Attributions
Acknowledgements
Home
Project
Improving Flip Flop
Proteolysis
Inhibition system
Inhibition system
Proteolysis is a very common system to regulate gene expression in all organisms. In bacteria, proteins can be targeted for proteolysis by adding a special degradation tag in their coding sequences. That’s how the Sspb-ClpXP system, which we are going to use, works.
In E.coli, ClpX recognize specific degradation tags – ssrA tags -, unfolds the attached protein and translocates this denatured polypeptide into ClpP for its degradation, which is very fast. The action of this ClpXP protease is enhanced by the SspB adaptator protein. However, it’s impossible to control proteolysis by this mechanism by manipulating SspB availability, as ClpXP protease can recognize and degrade these tagged proteins without Sspb’s aid. The degradation model is shown in Figure 1.
Figure 1. ClpXP-Sspb proteolysis model. Taken from McGinness et al. (2006)
Different sequences of the ssrA tag are recognized by the protease and the adaptator. As we can see in Figure 2, de AANDENY portion of the tag binds SspB and the LAA portion binds ClpX. McGinness et al. came to the conclusion that if the interaction of the tag with ClpX was decreased, without affecting that with the SspB, the adaptor-dependence would be increased. With this change, it would be possible to control this proteolysis system by regulating the SspB levels.
Figure 2. Sequence of the ssrA tag and its mutants. Adapted from McGinness et al. (2006) .
This team replaced two residues in the ssrA tag in order to weaken its interaction with ClpX but not with SspB (see Figure 2). They chose to use a Ser (serine) as a C-terminal residue as fewer than 10% of ClpX substrates with this tag had a Serine in this position. Asp was chosen as the antepenultimate residue because it decreased ClpXP degradation slightly. These new ssrA modified tags were called DAS tags. Fusing these DAS tags (Figure 2) to a protein, they found out that with the DAS-tagged substrate it was necessary the presence of Sspb for a high-affinity binding and  degradation by ClpXP.
Furthermore, they demonstrated that inserting residues between the ClpX and the SspB binding sites improved the SspB substrate delivering process to ClpXP. They tried out 3 variants of DAS tag: DAS+2, DAS+4 and DAS+8 (see Figure 2) and concluded that the DAS+4 tag showed the highest degradation level differences with and without SspB (see Figure 3 and Figure 4).
Figure 3. Degradation ratio between presence and absence of SspB. Taken from McGinness et al. (2006) .
Figure 4. Initial rates of ClpXP degradation, with and without Sspb using the DAS+4 tag. Taken from McGinness et al. (2006) .
Therefore, this group has designed a proteolysis system that can be controlled by the presence or absence of the adaptor protein Sspb.
Overview
Data Page
Proteolysis
Inhibition system
Proteolysis regulation
Conclusions
asRNA
Inhibition system
Fusion proteins
Conclusions
Modeling
Introduction
Equations
Simulations
Bioinformatics
Docking SspB-ClpX
Tags Finder
RybB Structure
The Final Construct
Experimental Results
Strain Construct
Plasmids and controls
Change Speed
Stability
Follow @iGEMupoSevilla
Sponsors
Retrieved from " http://2011.igem.org/Team:UPO-Sevilla/Project/Improving_Flip_Flop/Proteolysis/Inhibition_system "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
