"
Page
Discussion
View source
History
teams
Log in
Â 
Team:UPO-Sevilla/Project/Epigenetic Flip Flop/Bioinformatics
From 2011.igem.org
Home
Project
Foundational Advances
Human Practices
Team
Overview
Data Page
Basic Flip Flop
Improved Flip Flop
Epigenetic Flip Flop
Applications
Parts
Notebook
Judging Criteria
Overview
MiniTn7
BioBrick Creator
Overview
Safety
Tornillos y Genes
BSIES
Randy Rettberg' s Seminar
Workshop
Science Fair
Week of Science
Media
Team Structure
Attributions
Acknowledgements
Home
Project
Epigenetic Flip Flop
Bioinformatics
Bioinformatics
Goal
To elucidate if the recombinant proteins generated by this sublab are going to be functional.
Background
When performing a protein structure prediction by homology modelling strategies, we usually obtain several independent models. Each of these models, corresponds to one of the proteins which have been linked together to form the recombinant one. Thus, it is not possible to obtain a complete single structure for a recombinant protein only by using a protein homology modelling server.
Strategy
In order to study the possible structure and functionality of the fusion proteins described in this flip-flop system, an indirect docking modelling approach has been carried out. Our approach consists in performing a docking experiment in which we introduce the pdb model of every protein fused in the recombinant main protein. Once the docking method has been performed, the final peptide range of the first protein fused (about the last 30 amino acids) and the initial amino acids of the second protein fused are located in the resulting pdb file, to study any possible interactions. Interactions between these parts of the proteins would mean that the recombinant protein could acquire a native folding structure which would have negative influences to its functionality.
Procedure
Get a pdb model for each protein which is going to be fused. To pursue this, the protein homology modeling server Swiss Model and CPHmodels 3.0 server were used to obtain one putative structure for each protein. The results were compared and contrasted, and the best models were chosen. So, model structures were obtained for the TetR protein and for the Swi6 CDS and the Sir3 BAM protein domain.
Docking experiment with the composing proteins of each recombinant one. In this case, the Hex protein docking server was used, and the three best results were downloaded (the best result is that one with the lowest level of free energy). The server was run twice, a first time considering only shapes and a second time considering shapes and electrostatics.
Analyse the docking results and locate the final and initial fragment of the fused proteins in order to predict possible interactions. To visualize and locate these peptidic ranges, the RasMol 2.7.5. viewer was used.
Results and Discussion
Protein Docking 1 (TetR-Sir3BAM domain)
Figure 1 displays the three best shape and shape+electrostatics docking results. In this image, TetR protein is shown in blue with its last amino acids coloured in yellow. The Sir3 BAM protein domain is shown in red with its first 30 amino acids coloured in green. The free energy of the bindings are:
Docking Free Energy
Docking shape only
Docking shape+electrostatics
Best Result 1
-627.19 Kcal/mol
-834.17 Kcal/mol
Best Result 2
-605.44 Kcal/mol
-750.62 Kcal/mol
Best Result 3
-598.25 Kcal/mol
-741.00 Kcal/mol
Figure 1. Docking models for the TetR-BAMSir3 interaction. The best three shape docking results are displayed in the top row while the best shape+electrostatics docking results are shown in the row below.
Taking everything into consideration, it seems that there is a high possibility of interaction between the TetR protein and the functional Sir3 BAM protein domain. This is due to the lowest three free energy results (for both shape and shape+electrostatics) can also acquire a stable structure in which the linked parts in the recombinant protein are quite close. These possible interactions would reduce the functionality of our fusion protein TetR::Sir3.
Protein Docking 2 (TetR-Swi6CDS domain)
Figure 2 displays the three best shape and shape+electrostatics docking results. In this image, TetR protein is shown in blue with its last amino acids coloured in yellow. The Swi6 chromo shadow domain (CDS) is shown in red with its first 30 amino acids coloured in green. The free ebergy of the bindings are:
Docking Free Energy
Docking shape only
Docking shape+electrostatics
Best Result 1
-627.36 Kcal/mol
-653.73 Kcal/mol
Best Result 2
-612.57 Kcal/mol
-630.66 Kcal/mol
Best Result 3
-604.63 Kcal/mol
-607.76 Kcal/mol
Figure 2. Docking models for the TetR-Sir3CDS interaction. The best three shape docking results are displayed in the top row while the best shape+electrostatics docking results are shown in the row below.
In this case, the free interaction energy resulted in closer figures, which means that all the showed models must be taken into consideration. From all of this information as well as the obtained interaction structure models, it seems that there is a lower possibility of interaction in the case of TetR protein and the Swi6 chromo shadow domain than in the case of the TetR protein and the BAM Sir3 protein domain. As a consequence, the recombinant protein TetR::Swi6CDS should fold itself in an appropriate way to allow individual domain functionality.
Overview
Data Page
Background
Strategy
Designing
Bioinformatics
Procedure and Results
Follow @iGEMupoSevilla
Sponsors
Retrieved from " http://2011.igem.org/Team:UPO-Sevilla/Project/Epigenetic_Flip_Flop/Bioinformatics "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
