"
Page
Discussion
View source
History
teams
Log in
 
Team:UPO-Sevilla/Project/Notebook/Epigenetic Flip Flop
From 2011.igem.org
Home
Project
Foundational Advances
Human Practices
Team
Overview
Data Page
Basic Flip Flop
Improved Flip Flop
Epigenetic Flip Flop
Applications
Parts
Notebook
Judging Criteria
Overview
MiniTn7
BioBrick Creator
Overview
Safety
Tornillos y Genes
BSIES
Randy Rettberg' s Seminar
Workshop
Science Fair
Week of Science
Media
Team Structure
Attributions
Acknowledgements
Home
Project
Notebook
Epigenetic Flip Flop
Notebook
Epigenetic Flip Flop
Paola Gallardo
New year, new project. This time, let's try to make an epigenetic switch!
Before the end of the classes and during the spring, this sub-team has outlined the project by  organizing weekly meetings to discuss about the design of our project and the strategies to carry it out.
Week 1 (July, 4-8)
Initial contact with the lab. First of all, it is necessary to amplify some vectors that will be used in the construction of the modules, Pnmt1(41X) and Purg1, as well as two constructions that have been gently sent from the Dr. Attila Becskei from the University of Zurich. On the other hand, I have designed some primers that will be used later.
There are still lots of things to do.
Week 2 (July, 11-15)
Short week. The order of synthesized part tetR-CSD has just arrived to the Lab, so I can start to build up the compaction proteins modules. First of all, I have had to transform the plasmids into E. coli. Also, I made a PCR to amplify swi6 from the genome of Schizosaccharomyces pombe, and sir3, from a plasmid. The objective was to get enough DNA of these parts and, by using appropriate restriction sites that were introduced into the designed primes, start assembling the  fusion proteins for this module.  Next week we will see if the experiments have worked out as expected.
Week 3 (July, 18-22)
This week I have started to assemble the DNA fragments that will be used for the compaction protein modules, so firstly, I have had to made sure that all its parts were available. I set up an inocula of the E. coli previously transformed with the tetR-CSD plasmid and made a miniprep. At the same time, I checked the results of the PCR of  sir3 and swi6. I have just been lucky with the last one.
I have repeated the PCR to get sir3 from the plasmid, using different conditions: I modified the initial concentration of the DNA template and reduced the annealing temperature), but the result was again negative. Then, I have launched a new strategy and cut the plasmid containing sir3 gene with  restriction endonucleases to purify the entire gene and use it late as template for a new PCR. This strategy also allows me to check that the plasmid was the right one.
In the meantime, I have made sequential cuts of tetR-CSD, pPnmt1(41X) and swi6, all the components of the compaction module, apart from the slippery sir3. However, we have got some bad news. The rest of the synthesized parts that we had order will be delayed at least four weeks. Desperation!!! Let's do it old-style way, amplifying the parts with primers and assembling them with DNA ligase!
Week 4 (July, 25-29)
I began this week trying to clone into the pREP41X vector the parts of two compaction modules (tetR-CSD, tetR-Swi6). I transformed E. coli DH5α with the ligation products. The following day, I used a new method for screening recombinant clones, colony cracking, but I didn't obtain positive results. So, I tried with minipreps and digestions, but results were negatives too. I followed the same protocol, but digestions didn't work.
I changed again the PCR conditions to get sir3 and made a gradient PCR, but it didn't work either. To make sure that the template was the correct one, I made analytic cuts of it, with different restriction enzymes. After two tries, results were conclusive. What's wrong? New primers.
Apart from that, obtained Tadh1-tetOn (ADH1 terminator with a variable number of tetR operator sites repetitions). For doing that PCR conditions had to be optimized (annealing temperature, template concentration, extension time, …).
Week 5 (August, 1-5)
I digested again tetR-CSD with the appropriate restriction enzymes, XhoI and XmaI, but and electrophoresis with the digestion products demonstrated that there wasn't cuts. Analysis of tetR-CSD DNA sequence showed that restriction sites were lost. A site-directed mutagenesis was order to recover restriction sites.
I made analytic digestions of pPR013 and pPR074, obtaining coherent results.
I also amplified and purified Swi6 using an optimized PCR reaction, in order to use it when needed. Amplification of sir3 with new primers gave no results. So, I need to optimize the PCR conditions.
At last, we received another synthesized part: tetO2. I transformed DH5α E. coli with this part and the following day I inoculated isolated colonies to make minipreps.
In the meantime, I began to construct the Purg-GFP-Tadh1 module, a positive control of the designed module containing the tetO flanking sequences. This construction will be integrated into the genome of S. pombe at the leu1 locus by homologous recombination.
In other hand, I ligated the fragment containing the Tadh1-tetOn obtained by PCR into pGEM-T vector and transformed it in E.coli. Colonies will be screened after the holiday break.
Week 6 (September, 2)
When I came back from holidays, I found the rest of synthesis parts (GFP·tetO4 and tetR·CSD) over my table. I used the standard protocol to transform them into E coli.
I made digestions of tetO2·Tadh1 and Purg with BglII restriction enzyme.
Week 7 (September, 5-9)
Colonies of transformation plates were quite strange. I set up inocula of them and repeated the transformation protocol. Some of the inocula didn't grow up, maybe because of selection of satellite colonies in ampicillin plates. I made minipreps and digestions of them, to get tetR·CSD (XhoI/XmaI), tetR (XhoI/BamHI) and GFP·tetO4 (PacI/AscI). I also digested pREP41X plasmid with XhoI and XmaI, in order to ligate tetR·CSD and tetR+Swi6 between pnmt41X promoter and nmt1 terminator. However, the result of the ligation was not positive possibly because the plasmid was not properly digested.
I made an electrophoresis and I tried to purify TAdh1·tetO2 and Purg from agarose gel, but TAdh1·tetO2 fragment migrates with bromophenol, so I had to repeat this process again. Then, I proceeded to ligate them o/n while keeping my fingers crossed, and transformed ligation product in DH5α the following day. There appeared colonies in spread petri plates. Colonies cracking didn't work, so I set up inocula to do minipreps next day. After an immeasurable colonies screening, I obtained two positive clones (19 and 35), Tadh1·tetO2·Purg is ready.
Digestion of Tadh1·tetO2·Purg with AscI and PacI is made to introduce the next component of this module, GFP·Tadh1·tetO4. I had the same problem: plasmid was not totally digested and control religation plate was plenty of colonies. Lesson learned: if you don't give enough time to a reaction, you'll be wasting your time.
Week 8 (September, 12-16)
Due to last week result, I made a new ligation reaction of Tadh1·tetO2·Purg plus  GFP·Tadh1·tetO4, but I obtained the same results: the same number of colonies in religation plate and ligation plates.
I kept on trying to clone tetR·CSD and tetR+Swi6 into the pREP41X vector, by digestion, precipitation of DNA, ligation and transformation, but the last step always failed.
Finally, I obtained sir3 gene by PCR, and also swi6, but they were lost in their purification.
Week 9 (September, 19-23)
Colony cracking of pREP41x + tetR-CSD and pREP41x + tetR + Swi6 didn't work, so I set up inocula of all the colonies in the plates to make an screening of them. I have made analytical digestions of the minipreps and, after two electrophoresis, it seems that I have a positive clone of thirty.
In the middle of the week, I have received the primers that will let me obtain all the parts in BioBrick standard.
Week 10 (September, 26-28)
I made a screening of the plates of the putative pREP41X + tetR-CSD by colony cracking, but results were quite strange. I started to amplify the parts to be sent to Registry of Standard Biological Parts.
Week 11 (October, 3-8)
After European Jamboree...
Analytical digestions of  the putative construction Tadh1·tetO2·Purg plus GFP·Tadh1·tetO4, showed negative results, so it is necessary to go back to the beginning.
A new try to clone tetR-CSD in pREP41X, adjusting volumes, concentrations, and patience, and again with negative results after a whole Petri plate colonies screening.
I analyzed Tadh1-tetO2-Purg construction, and it seemed that something was wrong. I must return to the beginning and start everything.
I set up inocula of all the parts to begin from the scratch. Having nightmares with electrophoresis gels and Petri plates can't be good. Again, minipreps, sequential digestions, …
Week 12 (October, 10-14)
... purifications, ligations, DH5-a transformation. A new ligation reaction with dephosphorylated plasmid pfa6a-MX6-Purg. Transformation results in empty plates. I tried again with no  dephosphorylated plasmid, and the the ligation plate is plenty of colonies, as well as the control plate. I made a 51 colonies screening and I found some positives clones.
I cloned Purg1, Swi6, tetR-CSD and tetO2 into pSB1C3 and transformed the ligations products into  DH5-alfa E. coli strain. No colonies in plates.
Week 13 (October, 17-21)
Finally, I did cloned  tetR-CSD in pREP41X: three colonies, three positives.
I digested (Tadh1·tetO2)+pfa6a-MX6-Purg  plasmid and  GFP·Tadh1·tetO4 synthetic construct with AscI.
I begin the transformation protocol of S. pombe to stochastically integrate pfa6a-MX6-Purg and (Tadh1·tetO2)+pfa6a-MX6-Purg in its genome as controls.
Week 14 (October, 24-28)
It seems that protocol integration in S. pombe genome didn't work, because there is no growing colonies in the selection replicated plates.
Digestion with  AscI didn't work very well, so I tried a new one.
Anyway, there is no time to finish the experiments. Time is running out. Maybe next time.
Basic Flip Flop
Improved Flip Flop
Epigenetic Flip Flop
MiniTn7
Mathematical Modeling
Multiagent Modeling
Bioinformatics
Biobrick Creator
Wiki design
Follow @iGEMupoSevilla
Sponsors
Retrieved from " http://2011.igem.org/Team:UPO-Sevilla/Project/Notebook/Epigenetic_Flip_Flop "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
