"
Page
Discussion
View source
History
teams
Log in
Â 
Team:UPO-Sevilla/Project/Improving Flip Flop/Bioinformatics/Docking SspB-ClpX
From 2011.igem.org
Home
Project
Foundational Advances
Human Practices
Team
Overview
Data Page
Basic Flip Flop
Improved Flip Flop
Epigenetic Flip Flop
Applications
Parts
Notebook
Judging Criteria
Overview
MiniTn7
BioBrick Creator
Overview
Safety
Tornillos y Genes
BSIES
Randy Rettberg' s Seminar
Workshop
Science Fair
Week of Science
Media
Team Structure
Attributions
Acknowledgements
Home
Project
Improving Flip Flop
Bioinformatics
Docking_SspB-ClpX
Docking SspB-ClpX
Goal
To generate an interaction model for the SspB-ClpX system used in our V sublab.
Background
In the ClpXP proteolysis model reported by McGinness et al (McGinnes et al, 2006) SspB acts as an adaptor protein which helps in the target protein degradation (general information of SspB is presented in table 1). This SspB protein interacts with the protease subunit ClpX. ClpX (information of this protein available in table 2) forms a protease complex with ClpP in which the target protein is degraded.
Table 1. Relevant information of SspB protein from Uniprot protein database entry B1XHK1 and the corresponding SspB Pfam database entry .
Table 2. Relevant information of ClpX protein from Uniprot protein database entry P0A6H1 and the corresponding ClpX Pfam database entry .
The complete structure of ClpX was published by Glynn (Glynn et al, 2009) and it is displayed in figure 1. This molecule acquires a homohexameric ring-like structure which allows the peptidic degradation process when interacting with ClpP, the other protease subunit.
Figure 1. ClpX protein complex (from PDB entry 3hte ). A. Structure coloured by chain. B. Same structure displaying the AAA 2 protein domain (red) and the ClpB D2-small domain (blue). More information of these domains is available in the ClpX protein Pfam entry.
Strategy
A protein-protein docking has been carried out to obtain an interaction model for the SspB adaptor protein and the protease subunit ClpX. This interaction model would be the first step in the development of a complete real structural model for the degradation process of a target protein by this protease system.
Procedure
Obtain a pdb model for the SspB and ClpX proteins. In the case of SspB, no pdb model was available for this protein in the RCSB Protein Data Bank (PDB) , so the protein homology modeling server Swiss Model and CPHmodels 3.0 server were used to obtain a structural model of this protein. However, for ClpX protein several pdb 3D files were available and the largest one was chosen ( PDB entry 3hte ).
Run the Hex protein-protein Docking server. A shape complementarity approach was chosen.
Analyse the docking results. RasMol 2.7.5. and Jmol 12.0 molecule viewers were used.
Results and Discussion
The Hex server output is shown in figure 2. This model is comparable with the model previously reported by McGinness et al (McGinnes et al, 2006) . Thus, it seems that SspB interacts with the ClpX homohexameric ring in two different main points. These interactions are displayed in figures 3 and 4. The interacting amino acids are displayed in a ball and stick way in which the amino acids side chains interactions can be perfectly appreciated.
Figure 2. SspB-ClpX interaction model obtained from Hex protein-protein docking server (shape complementarity approach). SspB protein has been coloured in red and the ClpX is coloured by chain. Structure displayed with Jmol 12.0 molecule viewer.
Figure 3. Obtained docking model of the SspB-ClpX interaction in which the interacting amino acids of the left-side SspB dimer are displayed in CPK colouring and ball and stick standard. Displayed with RasMol 2.7.5. viewer .
Figure 4. Obtained docking model of the SspB-ClpX interaction in which the interacting amino acids of the right-side SspB dimer are displayed in CPKcoloring and ball and stick standard. Displayed with RasMol 2.7.5. viewer .
Overview
Data Page
Proteolysis
Inhibition system
Proteolysis regulation
Conclusions
asRNA
Inhibition system
Fusion proteins
Conclusions
Modeling
Introduction
Equations
Simulations
Bioinformatics
Docking SspB-ClpX
Tags Finder
RybB Structure
The Final Construct
Experimental Results
Strain Construct
Plasmids and controls
Change Speed
Stability
Follow @iGEMupoSevilla
Sponsors
Retrieved from " http://2011.igem.org/Team:UPO-Sevilla/Project/Improving_Flip_Flop/Bioinformatics/Docking_SspB-ClpX "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
