<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UCSF_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UCSF/Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UCSF/Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UCSF/Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UCSF/Protocols&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UCSF/Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UCSF/Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="Main"><DIV id="navigation"><UL id="UL"><LI class="" id="N1"><A href="https://2010.igem.org/Team:UCSF">UCSF Home</A></LI><LI class="" id="N2"><A href="https://2010.igem.org/Team:UCSF/Team">Team</A></LI><LI class="sub-nav" id="N3"><A href="https://2010.igem.org/Team:UCSF/Project">Project</A></LI><LI class="cat-item" id="N3-1"><A href="https://2010.igem.org/Team:UCSF/Project" title="">Introduction</A></LI><LI class="cat-item" id="N3-2"><A href="https://2010.igem.org/Team:UCSF/Project/Precision" title="">Greater Precision</A></LI><LI class="cat-item" id="N3-3"><A href="https://2010.igem.org/Team:UCSF/Project/Signaling" title="">Stronger Signaling</A></LI><LI class="cat-item" id="N3-4"><A href="https://2010.igem.org/Team:UCSF/Project/Arsenal" title="">Better Arsenal</A></LI><LI class="cat-item" id="N3-5"><A href="https://2010.igem.org/Team:UCSF/Project/Future_Implications" title="">Future Implications</A></LI></UL><LI class="sub-nav" id="N4"><A href="https://2010.igem.org/Team:UCSF/Protocols">Protocols</A></LI><LI class="cat-item" id="N4-1"><A href="https://2010.igem.org/Team:UCSF/Protocols" title="">Protocols</A></LI><LI class="cat-item" id="N4-2"><A href="https://2010.igem.org/Team:UCSF/Protocols/Assays" title="">Assays</A></LI><LI class="" id="N5"><A href="https://2010.igem.org/Team:UCSF/Parts">Parts</A></LI><LI class="" id="N6"><A href="https://2010.igem.org/Team:UCSF/Medal">Medal Requisites</A></LI><LI class="sub-nav" id="N7"><A href="https://2010.igem.org/Team:UCSF/Notes">Notebook</A></LI><LI class="cat-item" id="N7-1"><A href="https://2010.igem.org/Team:UCSF/Notes/Timeline">Project Timeline</A></LI><LI class="cat-item" id="N7-2"><A href="https://2010.igem.org/Team:UCSF/Notes/Bootcamp">Summer Bootcamp</A></LI><LI class="cat-item" id="N7-3"><A href="https://2010.igem.org/Team:UCSF/Notes/Attributions">Attributions</A></LI><LI class="" id="N8"><A href="https://2010.igem.org/Team:UCSF/Safety">Safety</A></LI><LI class="" id="N9"><A href="https://2010.igem.org/Team:UCSF/Sponsors">Sponsors</A></LI><LI class="" id="N10"><A href="https://2010.igem.org/Team:UCSF/FunStuff">Just for Fun</A></LI><LI class=""><A href="https://2010.igem.org/Team_Wikis?year=2010">Other Teams</A></LI><LI class=""><A href="https://2010.igem.org">iGEM Home</A></LI></DIV><TABLE border="0" style="margin:0px;padding:0px;margin-top:0px;"><TBODY><TR><TD width="707px" style="padding: 0 0 0 0; background-color:#f2eee9;margin:0;" valign="top"><DIV id="left" style="width:100%;height:100%;"><H3><SPAN class="mw-headline" id="AarI_Digest">AarI Digest</SPAN></H3><P>1. Label PCR tubes.
2. Add the following reagents into the PCR tubes:</P><PRE><B>Aar1 Digest Reagents:</B>
5 ug DNA
2.5ul Aar1 Enzyme
0.9ul Aarl oligo
6 ul 10x Aar1 Buffer
x ul dH2O
60 ul total reaction</PRE><P><I>Note: x ul dH2O may change in volume for different reactions in order to bring total volume up to 60 ul.</I></P><P>3. Briefly vortex and spin down the reaction.
4. Incubate reaction at 37C for 3 hours.
</P><P><I>Note: PCR program can be set to 37C for 3 hours and cooled down to 4C indefinitely.</I></P><H3><SPAN class="mw-headline" id="PCR_with_Phusion_enzyme">PCR with Phusion enzyme</SPAN></H3><P>1. Label PCR tubes.
2. Add the following reagents into the PCR tubes (in order):</P><PRE><B>PCR Reagents</B>
23.7 ul dH2O
10 ul 5x HF Buffer
5 ul forward primer
5 ul reverse primer
5 ul dNTP
0.3 ul template
1 ul Phusion
50 ul total</PRE><P>3. Vortex and spin down the reaction.
4. Set up PCR program.</P></DIV></TD></TR><TR><TD><B>Cycle Step</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD><TD><B># Cycles</B></TD><TD>Initial Denature</TD><TD>98C</TD><TD>3min</TD><TD>1
</TD></TR><TR><TD>DenatureAnnealingExtension</TD><TD>98C55C72C</TD><TD>10s30s30s</TD><TD>30
</TD></TR><TR><TD>Final Extension</TD><TD>72C</TD><TD>5min</TD><TD>1
</TD></TR><TR><TD>Hold</TD><TD>4C</TD></TR></TBODY></TABLE><P><I>Note: Times may differ for different reactions based on the size of the desired PCR product. Annealing temperature may also differ depending on primer properties.</I></P><H3><SPAN class="mw-headline" id="Ligation">Ligation</SPAN></H3><PRE><B> Ligation Reagents:</B>
50 ng vector
DNA insert(s)
Buffer
1 ul Ligase</PRE><P><B>Ligation Reaction:</B>
1. Calculate amount of DNA insert and vector needed for reaction.
150 ng / ( # bp in backbone/ # bp in insert) = ng needed
Concentration of insert or vector divided by ng needed = volume of DNA needed
</P><P>2. Calculate amount of buffer needed depending on concentration for volume needed. A typical ligation reaction volume can be 20 ul.
</P><P>3. Add reagents together, from smallest volumes to largest. Distilled water can be used to bring the volume up so that the reaction has the proper buffer concentration for the reaction. Ligase or enzymes in general should be added at the end.
</P><P>4. Vortex or pipet up and down to mix. Spin down afterwards if vortexing.
</P><P>5. Let reaction sit at room temperature for 5 to 20 minutes depending on ligase used. The ligase may start acting up if left too long.
</P><H3><SPAN class="mw-headline" id="Agarose_Gel_Electrophoresis">Agarose Gel Electrophoresis</SPAN></H3><PRE><B>Gel Electrophoresis Reagents/Materials:</B>
Gel caster
Gel tray
Agar powder
TAE buffer
10, 000X Sybr Safe
50ml conical tube
Gel comb
Microwave
Large flask (microwave safe)</PRE><P><B>Preparing the agarose solution:</B></P><P>1. Pour in flask the amount of TAE buffer desired. This will be the volume of agarose solution made.
</P><P>2. Measure out the amount of agar needed for desired concentration. For example to get a 1% agarose gel, you would add 1 gram of agarose powder to 100 ml of TAE buffer. Most concentrations commonly fall between 0.7- 2%.  Higher concentrations of agarose are better suited  for small fragment separation while lower concentrations are suitable for large fragments.
</P><P>3. Microwave three minutes, may vary depending on volume in flask. Make sure cap is on loosely.
</P><P>4. Using hot gloves, swirl bottle gently to ensure complete dissolving of agar. The solution should be clear when agar is completely dissolved. If it is not, microwave for another minute.
</P><P>5. Place flask of agarose solution in 80C bath to store as to prevent solidifying of agarose solution.
</P><P><B>Casting gel:</B></P><P>1. Secure gel tray in caster
</P><P>2. Pour desired volume of premade agarose solution into 50 ml conical tube. Volume desired will vary depending on gel tray size and desired thickness of gel. Thin gels solidify quicker and can show faint bands clearer, but the wells of thicker gels can hold more DNA.
</P><P>3. Add Sybr safe to agarose solution. For 10 ml of agarose solution, add 1ul Sybr Safe. The Sybr safe is used to stain and visualize the DNA when viewed under a blue light.
</P><P>4. Invert tube several times until Sybr safe is equally distributed. Invert gently to avoid creating air bubbles.
</P><P>5. Pour contents of conical tube into secured gel tray.
</P><P>6. Place gel comb into notches of gel tray. Check that comb is evenly submerged in agarose.  The comb will form wells in the gel to load DNA.
</P><P>7. Dry at room temperature. The gel will appear opaque when done.
</P><H3><SPAN class="mw-headline" id="Colony_PCR">Colony PCR</SPAN></H3><PRE><B>Colony PCR reagents:</B>
12ul dH20
4 ul 5x gotaq green buffer
2 ul 10mM dntp
1 ul forward primer
1 ul reverse primer
0.1 ul GoTaq Polymerase</PRE><P>1. After adding reagents to PCR tube, pick a colony and touch the bottom of the PCR tube with the pipette.
</P><P>2. Place PCR tubes into the PCR machine and set the cycling parameters to be optimal for the piece of DNA being copied.
</P><H3><SPAN class="mw-headline" id="Transformation_of_E._coli">Transformation of E. coli</SPAN></H3><PRE><B>Transformation Reagents</B>
LB Plates
Ice box
Water bath
37C Incubator
Competent cells &amp; Plasmid
Spreading tool (beads)</PRE><P>1. Prewarm LB plates in the 37C incubator
</P><P>2. Remove desired cells from -80C freezer (or other storage location) and melt over ice for at least 10 minutes. (Make sure to aliquot the correct amount of cells for transformation)
</P><P>3. Pipette up 5ul of your plasmid DNA and gently pipette into the suspended cells. Do not mix by triterating the solution (pipetting up and down).
</P><P>4. Let the cells sit on ice for 15-30 minutes.
</P><P>5. Heat shock your cells for 75 seconds in a 42C water bath.
</P><P>6. Quickly return your tubes to the ice bucket for 2-3 minutes.
</P><P>7. When you are ready to plate, remove your prewarmed plate from the incubator. Using sterile technique, carefully pipette your entire tube of cells onto the LB plate.
</P><P>8. Again using sterile technique, spread the cells across your plate using either sterile beads or another spreading tool.
</P><P>9. Put the plate in the 37C incubator for 5-15 hours.
</P><H3><SPAN class="mw-headline" id="Gel_Extraction.2FPurification_of_DNA">Gel Extraction/Purification of DNA</SPAN></H3><PRE><B>Gel Extraction/Purification Materials &amp; Reagents:</B>
Sterile Blade
2ml microcentrifuge tube
1.5ml microcentrifuge tube
QIAGEN Gel Extraction Kit
Centrifuge
Heat Block/Bath</PRE><P>1. Using a sterile blade, carefully excise the desired band from your gel.
</P><P>2. Transfer the agarose chunk into a  new 2ml microcentrifuge tube.
</P><P>3. Weigh the excised agarose chunk and add 3 times its weight in microliters of QG buffer.
</P><P>4. Move the tube to a melting block for 10 minutes at 50C or until the agarose is completely melted.
</P><P>5. Add isopropanol to the tube at a 1:3 ratio with QG, then vortex the solution briefly to help precipitate the DNA.
</P><P>6. Transfer the solution to a Qiagen gel extraction/PCR purification spin column.
</P><P>7. Spin for 1 minute at 13,000rpm and dump the supernatant.
</P><P>8.  Add 500Î¼l of QG buffer. Centrifuge columns at 13,000 rpm for 1 min and dump or vacuum the flowthrough.
</P><P>9.  Add 750Î¼l of PE Buffer. Centrifuge columns at 13,000 rpm for 1 min and dump or vacuum flowthrough.<I>NOTE: Wash the rim with the buffer.</I></P><P>10. Repeat Step 9
</P><P>11. Centrifuge columns at 13,000 rpm for 1 min and dump supernatant. (Skip if you are using a vacuum)
</P><P>12. Add 62Î¼l of EB Buffer to spin column. Place spin column in a 1.5 ml tube. Centrifuge tune at 13,000 rpm for 1 min. Remove spin column.
</P><P>13. Place tubes in a -20oC freezer box.
</P><H3><SPAN class="mw-headline" id="Miniprep">Miniprep</SPAN></H3><PRE><B>Miniprep Materials</B>
Qiagen Miniprep Kit
Centrifuge</PRE><P>1.  Spin down bacterial culture at 3500 rpm for 5 minutes
</P><P>2.  Aspirate supernatant. Change tips for every sample.
</P><P>3.  Resuspend in 250Î¼l of P1 Buffer. Vortex until cell pellet is not visible. Transfer into a 1.5 ml tube.
</P><P>4.  Add 250Î¼l of P2 Buffer. Invert 5-6 times. Wait 3-5 minutes.<I>NOTE: When you open the cap, you should see a string of DNA.</I></P><P>5.  Add 350Î¼l of N3 Buffer. Invert 5-6 times or until colorless
</P><P>6.  Centrifuge at 13,000 rpm for 10 minutes. Label blue spin columns. Transfer supernatant into blur spin columns.
</P><P>7.  Centrifuge columns at 13,000 rpm for 1 min and dump supernatant or vacuum supernatant.
</P><P>8.  Add 500Î¼l of PB buffer. Centrifuge columns at 13,000 rpm for 1 min and dump supernatant or vacuum supernatant.
</P><P>9.  Add 750Î¼l of PE Buffer. Centrifuge columns at 13,000 rpm for 1 min and dump or vacuum supernatant.<I>NOTE: Wash the rim with the buffer.</I></P><P>10. Repeat Step 9
</P><P>11. Centrifuge columns at 13,000 rpm for 1 min and dump supernatant. (Skip if you are using a vacuum)
</P><P>12. Add 62Î¼l of EB Buffer to spin column. Place spin column in a 1.5 ml tube. Centrifuge tune at 13,000 rpm for 1 min. Remove spin column.
</P><P>13. Place tubes in a -20oC freezer box.
</P><H3><SPAN class="mw-headline" id="Maxiprep">Maxiprep</SPAN></H3><PRE><B>Maxiprep Materials &amp; Reagents</B>
250ml centrifuge tubes
Isopropanol
QIAGEN Maxiprep Stand
QIAGEN Maxiprep Kit
Large Centrifuge
50ml conical tubes
100% Ethanol</PRE><P>1. Pick a single colony and inoculate in 2-5 ml LB medium containing the appropriate antibiotic, Incubate for around 8 hrs at 37C with vigorous shaking.
</P><P>2. Dilute 1/500 to 1/1000 into selective LB medium. For high copy plasmids, inoculate 100 ml medium with 100-200Î¼l of starter culture. For low copy plasmids, inoculate 250 ml medium with 250-500Î¼l of starter culture. Grow at 37C for 12-16 hrs with vigorous shaking.
</P><P>3. Spin at 6000 x g for 15 minutes at 4C
</P><P>4. Resuspend pellet in 10 ml P1 Buffer
</P><P>5. Add 10 ml P2 Buffer, mix thoroughly, Incubate at room temperature for 5 mins.
</P><P>6. Add 10 ml chilled P3 Buffer, mix immediately and thoroughly
</P><P>7. Pour into the barrel of the QIAfilter cartridge. Incubate at room temperature for 10 mins.
</P><P>8. Remove the cap from the QIAfilter cartridge outlet nozzle, insert the plunger into the QIAfilter Maxi cartridge and filter the cell lysate into a 50 ml tube.
</P><P>9. Add 2.5 ml ER Buffer, mix by inverting 10 times, Incubate on ice for 30 mins.
</P><P>10. Equilibrate a QIAgen-Tip 500 by applying 10 ml QBT Buffer and allow the column to empty by gravity flow
</P><P>11. Apply the filtered lysate from step 9 to the QIAGEN-Tip and allow it to enter the resin by gravity flow.
</P><P>12. Wash the QIAGEN-Tip with 30 ml QC Buffer, Repeat QC wash
</P><P>13. Elute DNA with 15 ml QN Buffer            	
</P><P>14. Precipitate DNA by adding 10.5 ml room temperature isopropanol to the eluted DNA. Mix and centrifuge immediately at â¥ 15,000 x g for 10 mins. Carefully decant supernatant without disturbing the pellet
</P><P>15. Air-dry the pellet for 5-10 min, redissolve the DNA in a suitable volume of endotoxin-free TE Buffer.
</P><H3><SPAN class="mw-headline" id="Antibody_Staining">Antibody Staining</SPAN></H3><PRE><B>Antibody Staining Reagents:</B>
Transfected and Untransfected cells of the same cell type.
Cells with empty vector or put through electroporation without vector
Dylight 488-conjugated AffiniPure Goat anti-mouse IgG, F(abâ)2, Jackson Immuno Research cat# 115-485-072, Lot# 83241 [should be protected from light].
Pierce Human IgG, whole molecule, ThermoScientific, Cat# 31154, Lot# LE1311903. (concentration=11mg/ml).
D-PBS CMF (Calcium, Magnesium free PBS).
Wash Buffer: 0.5% BSA, 0.1% Sodium Azide in D-PBS CMF.
FC Block: 1ul Human IgG in 1mL Wash Buffer, prepare fresh for each use.
Propidium Iodide(2mg/ml).
PI solution: 1ul Propidium Iodide in 1mL D-PBS CMF.</PRE><P>Work in an ice bucket (ethanol the bucket)
</P><P>1. Pipette 1ml wash buffer into 1.5ml microcentrifuge tubes. 
</P><P>2. Add 1x10^5 cells into wash buffer in each tube.
</P><P>3. Spin for 1 min for NKL and 2 minutes for CD8+ then aspirate. (Donât aspirate the pellet!)
</P><P>4. Go to step 3, repeat one more time. (Meanwhile, prepare FC Block)
</P><P>5. Resuspend in 100ul of FC Block. Icubate in ice for 15 min.
</P><P>6. Add 1ul of Goat anti-F(abâ)2 to each tube. Wrap the tube with foil. Incubate in dark for 30 minutes at 4 degrees. For the samples not being labeled, donât add the Goat antib-F(abâ)2.
</P><P>7. Add 1ml wash buffer to the Ab-labeled samples. Repeat step 3, two more times.
</P><P>8. Resuspend in 250ul of wash buffer.
</P><P>9. Take samples to the FACS machine.
</P><P>10. Add 250ul PI solution to the samples the moment before you run the samples.
</P><H3><SPAN class="mw-headline" id="TOPO_cloning">TOPO cloning</SPAN></H3><P><B>Zero Blunt TOPO</B></P><PRE>- Follow manufacturer's protocol
</PRE><H3><SPAN class="mw-headline" id="Transfection">Transfection</SPAN></H3><P><B>Jurkat Cell Transfection Protocol</B>
1. Cultivate the required number of cells for samples.
2. Prepare DNA for each sample.
3. Pre-warm the supplemented Cell Line NucleofectorÂ® Solution V to room temperature. Either pre-warm an aliquot of culture medium at 37Â°C in a 50 ml tube (500 Î¼l per sample) or add an additional 500 Î¼l in the step below.
4. Prepare 12-well plates by filling appropriate number of wells with 1 ml of culture medium containing supplements and serum. Pre-incubate plates in a humidified37Â°C/5%CO2 incubator.
5. Take an aliquot of cell suspension and count the cells to determine the cell density.
6. Centrifuge the required number of cells (1 x 106 cells per nucleofectionÂ® sample) at 500xg at room temperature for 5 min. Discard supernatant completely so that noresidual medium covers the cell pellet.
7. Resuspend the pellet in room temperature Cell Line NucleofectorÂ® Solution V to a final concentration of 1 x 106 cells/100 Î¼l. Avoid storing the cell suspension longer than 15 min in NucleofectorÂ® Solution as this reduces cell viability and gene transfer efficiency.</P><P><B>Important: Steps 8 - 13 should be performed for each sample separately.</B>
8. Transfer the nucleofectionÂ® sample into an amaxa certified cuvette. Make sure that the sample covers the bottom of the cuvette, avoid air bubbles while pipetting. Close the cuvette with the blue cap.
9. Select the appropriate NucleofectorÂ® program, X-01/X-001 or X-05/X-005 Insert the cuvette into the cuvette holder and press the âXâ button to start the program.
10. After the program has finished (display showing &quot;OK&quot;) take the cuvette out of the holder and incubate the sample in the cuvette for 10 min at room temperature.
11. If in step 3 you pre-warmed culture medium in a 50 ml tube, then after the 10 min incubation step, add 500 Î¼l of this pre-warmed culture medium to the cuvette and transfer the sample into the prepared 12-well plates. If you added an additional 500Î¼l to each well in step 4, then take 500 Î¼l from the well, add to the cuvette, and transfer the sample back into the prepared 12-well plates.To transfer the cells from the cuvettes, we strongly recommend using the plastic pipettes provided in the kit to prevent damage and loss of cells.
12. Press the âXâ button to reset the NucleofectorÂ®.
13. Repeat steps 8 - 13 for the remaining samples. Cultivation after 15.Incubate cells in a humidified 37Â°C/5% CO2 incubator. Following nucleofectionÂ®,nucleofectionÂ® gene expression should be analyzed at different times. Depending on the gene, expression is often detectable after 4 - 8 hours. If this is not the case, the incubation period may be prolonged up to 24 hours.</P><H3><SPAN class="mw-headline" id="FACS">FACS</SPAN></H3><P><B>T-Cell Activation (NFAT-GFP readout by FACS)</B>
Resuspend Jurkat and K562 cells at 1 million live cell/mL in RPMI supplemented with glutamine and 10% FBS (10G RPMI)
Add 100ul of each cell into a 96 well plate
Incubate overnight
Jurkat induction with anti-TCR (anti-CD3 ( clone C305))
Resuspend Jurkat at 1 million live cell/mL in FBS supplemented with glutamine and 10% FBS
Add 100ul into a well in 96 well plate
Add 100ul of 2X induction media (2X C305 = 1:1000 dilution of C305 in 10G RPMI)
Incubate overnight
FACS</P><H3><SPAN class="mw-headline" id="Imaging">Imaging</SPAN></H3><P><B>IMAGING PROTOCOL</B> 
Open up granules and stain with anti-GFP   (fixed cells)
Pre-warm all solutions/buffers that need to be warm (RPMI-1640, Myelocult, PBS)
DiI-labeling
1.     Count NKLs and spin down 2e6 cells at 400g for 5 minutes 
2.     Resuspend in 2mL of pre-warmed RPMI-1640 (at 1e6/mL).
3.     Add 10ul DiI stain and mix by swirling tube, etc
4.     Incubate cells and dye for 10 minutes at 37 C 
5.     Centrifuge cells for 5 minutes at 400g
6.     Aspirate off supernatant
7.     Resuspend in 2ml Myelocult and incubate for ~5 minutes at 37C in TC incubator (ârecoveryâ)</P><P>Lysotracker Blue labeling
LB1.   Spin down cells at 400g for 5 minutes.
LB2.   Resuspend at ~1e6/mL in 5uM LysoTracker Blue (diluted into RPMI-1640; note: 1:200 of 1mM stock). Incubate ~30 minutes at 37C in TC incubator.
LB3.  Centrifuge cells for 5 minutes at 400g
LB4.  Resuspend in (0.5 ml) Myelocult and incubate for ~5 minutes at 37C in TC incubator
Plating cells down on Fibronectin-coated Chambered Coverglass (8-well)					
P1.   Aspirate myelocult out of each well.
P2.   Transfer 0.2mL cells to a fibronectin-coated well incubate at 37C for 15-30 minutes 
P3.   Check to make sure cells appear to be stuck (on scope)
P4.   If not enough cells stuck, add more cells (repeat steps P2-P3).</P><P>Fix the cells / permeabilize / stain with anti-GFP Alexa647
F1. Fix in fixative solution (4% formaldehyde in PBS) for 15 minutes at room temperature with gentle agitation in the dark. Remove the solution.
F2. Wash cells twice in PBS for 1 minute each with gentle agitation. Remove PBS. 
F3. Permeabilize the specimen with Permeabilization solution (0.25% TritonÂ® X-100 in PBS) for 5 minutes at room temperature with gentle agitation in the dark. Remove the solution.
F4. Wash cells twice in PBS for 1 minute each with gentle agitation. Remove PBS. 
F5. Add Blocking solution (5% FBS in PBS pH 7.4). Incubate for 15 min at room temperature with gentle agitation. 
F6. ANTIBODY STAINING
-Add anti-GFP Alexa 647. (final conc: 10ug/mL, 1:20 overall dilution)
-Incubate for 0.5 hour at room temperature with gentle agitation.
F7. Decant antibody solution.
F8. Wash cells twice in PBS for 2 minutes each with gentle agitation. After the final wash, add PBS+BSA to the sample. </P><P>Prepare for microscopy
10/13 - Weâve tried 1:20 dilution and will stick with it.</P></DIV><DIV id="right" style="width:100%;height:100%;"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#AarI_Digest"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">AarI Digest</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#PCR_with_Phusion_enzyme"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">PCR with Phusion enzyme</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Ligation"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Ligation</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Agarose_Gel_Electrophoresis"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Agarose Gel Electrophoresis</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Colony_PCR"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Colony PCR</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Transformation_of_E._coli"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Transformation of E. coli</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Gel_Extraction.2FPurification_of_DNA"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Gel Extraction/Purification of DNA</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Miniprep"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Miniprep</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Maxiprep"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Maxiprep</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Antibody_Staining"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Antibody Staining</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#TOPO_cloning"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">TOPO cloning</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Transfection"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Transfection</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#FACS"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">FACS</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#Imaging"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">Imaging</SPAN></A></LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:UCSF/Protocols">http://2010.igem.org/Team:UCSF/Protocols</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UCSF/Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UCSF/Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UCSF/Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UCSF/Protocols&amp;oldid=204754" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>