<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UCSF_Project_Arsenal"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UCSF/Project/Arsenal">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UCSF/Project/Arsenal&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UCSF/Project/Arsenal&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UCSF/Project/Arsenal&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UCSF/Project/Arsenal" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UCSF/Project/Arsenal</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="Main"><DIV id="navigation"><UL id="UL"><LI class="" id="N1"><A href="https://2010.igem.org/Team:UCSF">UCSF Home</A></LI><LI class="" id="N2"><A href="https://2010.igem.org/Team:UCSF/Team">Team</A></LI><LI class="sub-nav" id="N3"><A href="https://2010.igem.org/Team:UCSF/Project">Project</A></LI><LI class="cat-item" id="N3-1"><A href="https://2010.igem.org/Team:UCSF/Project" title="">Introduction</A></LI><LI class="cat-item" id="N3-2"><A href="https://2010.igem.org/Team:UCSF/Project/Precision" title="">Greater Precision</A></LI><LI class="cat-item" id="N3-3"><A href="https://2010.igem.org/Team:UCSF/Project/Signaling" title="">Stronger Signaling</A></LI><LI class="cat-item" id="N3-4"><A href="https://2010.igem.org/Team:UCSF/Project/Arsenal" title="">Better Arsenal</A></LI><LI class="cat-item" id="N3-5"><A href="https://2010.igem.org/Team:UCSF/Project/Future_Implications" title="">Future Implications</A></LI></UL><LI class="sub-nav" id="N4"><A href="https://2010.igem.org/Team:UCSF/Protocols">Protocols</A></LI><LI class="cat-item" id="N4-1"><A href="https://2010.igem.org/Team:UCSF/Protocols" title="">Protocols</A></LI><LI class="cat-item" id="N4-2"><A href="https://2010.igem.org/Team:UCSF/Protocols/Assays" title="">Assays</A></LI><LI class="" id="N5"><A href="https://2010.igem.org/Team:UCSF/Parts">Parts</A></LI><LI class="" id="N6"><A href="https://2010.igem.org/Team:UCSF/Medal">Medal Requisites</A></LI><LI class="sub-nav" id="N7"><A href="https://2010.igem.org/Team:UCSF/Notes">Notebook</A></LI><LI class="cat-item" id="N7-1"><A href="https://2010.igem.org/Team:UCSF/Notes/Timeline">Project Timeline</A></LI><LI class="cat-item" id="N7-2"><A href="https://2010.igem.org/Team:UCSF/Notes/Bootcamp">Summer Bootcamp</A></LI><LI class="cat-item" id="N7-3"><A href="https://2010.igem.org/Team:UCSF/Notes/Attributions">Attributions</A></LI><LI class="" id="N8"><A href="https://2010.igem.org/Team:UCSF/Safety">Safety</A></LI><LI class="" id="N9"><A href="https://2010.igem.org/Team:UCSF/Sponsors">Sponsors</A></LI><LI class="" id="N10"><A href="https://2010.igem.org/Team:UCSF/FunStuff">Just for Fun</A></LI><LI class=""><A href="https://2010.igem.org/Team_Wikis?year=2010">Other Teams</A></LI><LI class=""><A href="https://2010.igem.org">iGEM Home</A></LI></DIV><TABLE border="0" style="margin:0px;padding:0px;margin-top:0px;"><TBODY><TR><TD width="707px" style="padding: 0 0 0 0; background-color:#f2eee9;margin:0;" valign="top"><DIV id="left" style="width:100%;height:100%;"><H3><SPAN class="mw-headline" id="Better_Arsenal"><B>Better Arsenal</B></SPAN></H3><P><B>Goal:</B></P><P>Direct a model protein, green fluorescent protein (GFP), to killer cells’ granules by identifying and fusing granule localization “address” tags to the protein.
</P><P><B>Approach:</B></P><P>One way that cancer cells can evade the immune system is by evolving resistance to the arsenal of cytotoxic proteins found in killer cells’ granules. Our goal for this summer was to direct a model protein, green fluorescent protein (GFP), to killer cells’ granules by identifying and fusing granule localization “address” tags to GFP. This would be an important first step toward loading granules with novel cytotoxic proteins against which cancer cells would likely be defenseless!
While it is not completely understood how killer cells send proteins to cytotoxic granules, we were able to learn a great deal about the “address” tags of various proteins that are sent to granules by searching and reading the literature. We designed a number of parts using these tags and made devices by fusing them in various combinations to GFP.
</P><H3><SPAN class="mw-headline" id="Brief_Introduction">Brief Introduction</SPAN></H3><P>
Current opinion on secretory granules is that the secreted granules are closely linked to lysosomes, they serve the function of lysosome when not secreted <A href="#References">[1]</A>. Proteins without signal peptides remain in the cytoplasm, while those with a signal peptide can be transported into the Endpplasmic reticulum, proteins that go into the ER are sorted according to its sorting signal, which can be regarded as an address tag. For Example, proteins with KEDL motif is maintained in the Endo reticulum. So “KEDL” motif is the Endo reticulum tag. Proteins with this tag finally stay in the endo reticulum. There are also tags that mark protein for granule, proteins with such tag are finally transported into the granule.</P><P>The address tags of the proteins utilize the cellular transportation system to go to different compartments of the cell. It is like mails, although there are addresses on them, they themselves will not go to the address; it depends on the post system to pick them up and deliver them to the right place.
</P><P>So what’s the transportation system of the cell like? They are made of transmembrane proteins. The rumen sides of the transporter proteins can recognize and bind to the address tags on cargo proteins. The cytoplasmic side of the transporter proteins contains specific motifs. When the golgi bubbles out, different bubbles contains different sets of transporter proteins whose cytoplasmic motifs will decide where the bubble should go (The mechanism will not be discussed here).
</P><P>Now let’s take a look at how granule resident proteins are sorted to the granule. By transporter, the sorting pathways can be divided into Mannose Phosphate Receptor (MPR)-dependent pathway and MPR-independent pathways.
</P><P>MPR-dependent pathways use MPR as the transporter. And the address tag is mannose-6-phosphate. Proteins transported in this pathway are glycosylated with mannose-6-phosphate groups, these groups serve as the address tag. The rumen side of the MPR can bind to the mannose-6-phosphate motif (We can see the from the naming of MPR). The cytoplasmic side of MPR contains “DXXLL” motif, which decides that MPR will be sorted to lysosome/granule.
</P><P>MPR-independent pathways, from the name, use transporters other than MPR. It may or may not depend on the mannose-6-phosphate tag on the cargo. This includes sortilin, LIMPII, LRP;<A href="#References">[2]</A><A href="#References">[3]</A><A href="#References">[4]</A>the addressing tags involved are not very clear.
</P><P>Granzymes are transported through the MPR-dependent pathway<A href="#References">[5]</A>.The signal peptide of the granzymes get cleaved upon entering the Endo reticulum. However, they are inactive in the Endo reticulum in the Endo reticulum and the golgi. Only after they reached granule, the N-terminal dipeptide can be cleaved off by DPPI aka cathepsin C, this leaves behind the active protein which begins with the conserved IIGG sequence. For Example, the N-terminal of Granzyme B is QPILLLLAFLLLPRADAGEIIGG underlined Amino acids are the signal sequence. The GE in italic is the dipeptide which inhibits the activity of granzyme B. In the granule, GE will get cleaved off by cathepsin C, and then granzyme B is activated.
</P><H3><SPAN class="mw-headline" id="Directing_GFP_to_killer_cells.27_granules_by_fusing_.22address.22_tags">Directing GFP to killer cells' granules by fusing &quot;address&quot; tags</SPAN></H3><P><B>Concept and experimental design</B></P><P>We hypothesized that fusing address tags from proteins that are naturally sent to the granule may work to send GFP to the granule as well.
</P><P><B>1. N-terminal signal peptides</B></P><P>Proteins should be translocated into the endoplasmic reticulum (ER) first in order to be further sorted to granule. Since it is not sure whether the signal sequence would play some role in the granule sorting, we decided to use the signal sequences from the granzymes. Each Granzyme has an individual signal peptide sequence that delivers it to the Endo Reticulum. Every signal sequence is distinct in size and amino acid content.
</P><P><B>2. C-terminal &quot;address tag&quot;sequences</B></P><P>We have two strategies to send our cargo from the endoplasmic reticulum (ER) to the granule:
</P><P>1. Directly fuse our cargo to the granule specific transporters.
</P><P>2. Fuse our cargo to some address tag which can bind to the transporter.
</P><P><B>Strategy 1</B></P><P>The proteins listed below all have granule localization motif on their cytoplasmic tails.
</P><P><B>Y-motif:</B></P><P>In a previous work<A href="#References">[6]</A>, scientists delivered TNFR1 to the granule by fusing it to the CD63 trans-membrane and cytoplasmic fragment. The mechanism is that the CD63 protein is a lysosome resident protein, it has signal sequence “SIRSGYEVM” on its cytoplasmic end, which contains the YXXØ motif (X represents any Amino Acid, Ø stands for hydrophobic amino acid). Similarly, other protein cargos can also be fused to the CD63 protein to be transported to the granule.
</P><P><B>MPR:</B></P><P>Many of the normal granule resident proteins in the cell are glycosylated and form a mannose-6-phosphate in the Golgi. MPR(mannose phosphate receptor) are membrane receptors for the mannose-6-phosphate motif. The cytoplasmic motif “DXXLL” of the MPR decided that it will be sorted to the granule together with the cargos bound. We make it another candidate to fuse our cargo to.
</P><P><B>LIMP-II &amp; LAMP1:</B></P><P>Lysosomal membrane proteins have been proposed to reach lysosomes by two pathways referred to as “direct” and “indirect”. In the direct pathway, lysosomal membrane proteins are transported intracellularly from the TGN to either early or late endosomes and then to lysosomes without ever appearing at the cell surface. In contrast, the indirect pathway involves constitutive transport from the TGN to the plasma membrane, followed by internalization into early endosomes and eventual delivery to late endosomes and lysosomes. Members of the LAMP/LIMP class are the preeminent example of proteins thought to traffic via the direct pathway, that’s why we picked LIMP-II and LAMP1. We hope that by employing the direct pathway we can reduce the leakiness of our cargo to extracellular compartment.
</P><P><B>Sortilin:</B></P><P>Sortilin is identified to be an alternative receptor than MPR in transporting lysosomal proteins. Sortilin has been reported to mediate lysosomal trafficking of prosaposin and acid sphingomyelinase. The cytoplasmic tail “GYHDDSDEDLL” contains “DXXLL” motif.
</P><P><B>Strategy 2</B></P><P>
Most of the soluble proteins in the granule are transported through the mannose-6-phosphate route, labeling the cargo protein with mannose-6-phosphate is the most direct and obvious method to use. However, mannose-6-phosphate is not directly genetically coded in the DNA sequence, instead it is the post-translational modification in the Golgi that delivers the glycosyl group to the protein. It is clear that enzymes only selectively create mannose-6-phosphate on those lysosome resident proteins, leaving the other proteins unmodified. So we turn to find the determining motif for glycosylation. Unfortunately, it turns out that this motif depends on the structure of the protein. No such domains that decides a protein should be decorated by mannose-6-phosphate, in other words, it is not modular.
</P><P>However, by looking at the works on the treatment of lysosomal storage diseases, we found a glycosylation independent motif that binds to MPR<A href="#References">[7]</A>.
</P><P><B>GILT:</B></P><P>GILT stands for “glycosylation-independent lysosomal targeting”, it actually uses a truncation of insulin-like grow factor II (IGF II). The Cation Independent mannose phosphate receptor has a IGF II binding motif. Since CI-MPR can also go to the cell membrane, its natural function is to internalize the circulating IGF II proteins and send them to the lysosome for degradation, thus regulating IGF II concentration. Therapy of lysosome storage disease takes advantage of the IGF II, and fuse exogenous lysosomal enzymes to the GILT tag. These genetically modified enzymes can be taken up by patient cells to fulfill some deficient function. We use the GILT tag to deliver endogenous proteins to lysosome, which also looks feasible.
</P><H3><SPAN class="mw-headline" id="Cloning_Strategy">Cloning Strategy</SPAN></H3><P>We used the <A href="http://dspace.mit.edu/bitstream/handle/1721.1/46721/BBFRFC28.pdf?sequence=1">BBF RFC 28</A> strategy to make fusion protein.
</P><P>Signal sequences are made into AB parts
</P><P>the cargo (GFP) is made into BC part
</P><P>and all the granule localization adress tags are made into CD parts.
</P><P>After assembling the AB-BC and CD part, we can get the constructs we want and express them in CTL cells.
</P><P>Since the signal sequences and the granule localization sequences are all short in length, we decided to make the basic AB and BC parts fused together through overlap extension PCR-based methods used in gene synthesis. These methods are robust for assembling oligos of 40-50nt into sequences up to and greater than 3kb. (Protocols from personal communication, Jason Park). We used the web-based software Helix System DNAworks (<A href="http://helixweb.nih.gov/dnaworks/" class="external free" rel="nofollow">http://helixweb.nih.gov/dnaworks/</A>) for human codon-optimization and oligonucleotide parsing.
</P><P>After we get all the AB, BC and CD parts, we were able the assemble them in different combination and make devices for testing.
</P><H3><SPAN class="mw-headline" id="Device_List">Device List</SPAN></H3></DIV></TD></TR><TR><TD><B>#</B></TD><TD><B>[N-terminal Granzyme sequence]-[cargo]-[C-terminal “address” tag]</B></TD><TD><B>Plasmid Name</B></TD></TR><TR><TD>1</TD><TD>GRZM B-EGFP-CDMPR</TD><TD>pCPL_056
</TD></TR><TR><TD>2</TD><TD>GRZM B-EGFP-GILT</TD><TD>pCPL_057
</TD></TR><TR><TD>3</TD><TD>GRZM B-EGFP-LAMP</TD><TD>pCPL_058
</TD></TR><TR><TD>4</TD><TD>GRZM B-EGFP-LIMP</TD><TD>pCPL_059
</TD></TR><TR><TD>5</TD><TD>GRZM B-EGFP-Y TAIL</TD><TD>pCPL_060
</TD></TR><TR><TD>6</TD><TD>GRZM B-EGFP-STOP</TD><TD>pCPL_061
</TD></TR><TR><TD>7</TD><TD>GRZM M-EGFP-CDMPR</TD><TD>pCPL_062
</TD></TR><TR><TD>8</TD><TD>GRZM M-EGFP-GILT</TD><TD>pCPL_063
</TD></TR><TR><TD>9</TD><TD>GRZM M-EGFP-LAMP</TD><TD>pCPL_064
</TD></TR><TR><TD>10</TD><TD>GRZM M-EGFP-LIMP</TD><TD>pCPL_065
</TD></TR><TR><TD>11</TD><TD>GRZM M-EGFP-Y TAIL</TD><TD>pCPL_066
</TD></TR><TR><TD>12</TD><TD>GRZM M-EGFP-STOP</TD><TD>pCPL_067
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Testing_the_Constructs">Testing the Constructs</SPAN></H3><P>In this part of our project we used Amaxa electroporation to transfect NKL cells (an NK cell line) with plasmids containing our devices. Then we grew cells and started to select for cells expressing our construct for our assay by selecting with the selection agent G418. The cells used for the assay were mixed populations as the selection was not complete.
</P><P>Our assay consisted of using fluorescence microscopy to determine how well we were able to target EGFP to the granules. Before microscopy, we first stained our NKL cell membranes with the membrane Vybrant DiI stain and then with the Lysotracker Blue lysosomal stain (killer cell granules are specialized lysosomes). After our initial assay we saw that the EGFP wasn’t fluorescing well, which we attributed to quenching due to the low pH levels of the granules, or degradation due to other lysosomal proteins. To address this issue we decided to fixed and permeabilized our cells, and stained the EGFP inside with a fluorescent anti-GFP antibody. This showed much clearer results.
</P><H3><SPAN class="mw-headline" id="Results">Results</SPAN></H3><P>We were able to test six of our devices in the assay: pCPL_056, pCPL_059, pCPL_060, pCPL_061, pCPL_063, and pCPL_067.
</P><P>We also had two negative controls for comparison:
</P><P><B>NKL (untransfected control):</B></P><P>We analyzed this condition because we wanted to have a negative control for auto-fluorescence in the cell line.  However, we had some problems because the background staining intensity of the fluorescent anti-GFP antibody was different from the experimental samples. We hypothesized that this was because of the different health of these cells, which had not been growing under G418 selection.
</P><P><B>pMAX GFP:</B></P><P>This control is the same cell line (NKL) transfected with the pMAX-GFP plasmid. pMAX-GFP is a powerful expression vector for GFP available from Lonza. Because it only expresses GFP, and doesn’t contain the N-terminal granzyme signals or the C-terminal sorting tags, the GFP will not be sent to the granules. The purpose of this control is to illustrate the difference between GFP left in the cytosol and GFP localized into the granules.
</P><P><B>Complications / troubleshooting in the assay</B></P><P>- At times, staining with the anti-GFP antibody did not appear to be uniform between all of the different samples. This was likely a technical error and results probably would be improved with more practice.
- Staining the granules with the lysosomal stain Lysotracker Blue (Invitrogen Molecular Probes) was often problematic. Most cells appeared to have high levels of non-specific background staining in the cytoplasm, especially after the cells were fixed and permeabilized. In addition, we were told that fluorescent dyes in the blue part of the light spectrum are often more challenging to visualize (more noise). However, we were able to observe reasonable Lysotracker Blue staining in some of our images.
</P><P>In some of our images, we had to independently adjust the LUTs settings for the Lysotracker to visualize the staining in some of our images (noted in the results tables below). We performed the image processing function “Smoothing” (NIS-Elements AR software) on all of our images (identical settings for all images).
</P><P>We were able to observe evidence of granular anti-GFP staining and workable granule staining in samples pCPL_056, pCPL_059, pCPL_060, and pCPL_061. (See summary table below.) In some samples, the granular anti-GFP staining and granule staining appear to be well-colocalized, suggesting that EGFP was loaded into the killer cell granules.
</P><P><B>SUMMARY OF RESULTS</B>
(See below for more details)
				colocalization stats	 images ready for wiki
</P><TABLE cellspacing="0"><TBODY><TR><TH> Construct
</TH><TH> GFP looks granular
</TH><TH> Granule stain worked
</TH><TH> GFP and granule stain appear colocalized
</TH><TH> Location of granules
</TH><TH> Colocalization statistics
</TH></TR><TR><TD>GZMB-eGFP-CDMPR (56)
</TD><TD>yes
</TD><TD>yes
</TD><TD>yes
</TD><TD>Center
</TD><TD>Pearson's Correlation: 0.831 Mander's Overlap: 0.967
</TD></TR><TR><TD>GZMB-eGFP-LIMP (59)
</TD><TD>yes
</TD><TD>Kind of
</TD><TD>Not really
</TD><TD>Edges
</TD><TD>Pearson's Correlation: 0.577 Mander's Overlap: 0.993
</TD></TR><TR><TD>GZMB-eGFP-Ymotif (60)
</TD><TD>Yes
</TD><TD>Yes (with adjustment to visualization settings)
</TD><TD>Yes
</TD><TD>throughout cell, mostly central
</TD><TD>Pearson's Correlation: 0.829 Mander's Overlap: 0.936
</TD></TR><TR><TD>GZMB-eGFP-STP (61)
</TD><TD>Yes
</TD><TD>Yes (with adjustment to visualization settings)
</TD><TD>Yes
</TD><TD>Edges
</TD><TD>Pearson's Correlation: 0.774 Mander's Overlap: 0.976
</TD></TR><TR><TD>pMAX-GFP
</TD><TD>No
</TD><TD>No
</TD><TD>No
</TD><TD>N/A
</TD><TD>N/A
</TD></TR><TR><TD>GZMM-eGFP-GILT (63)
</TD></TR><TR><TD>GZMM-eGFP-STP (67)
</TD></TR><TR><TD>NKL-untransfected
</TD><TD>No
</TD><TD>No
</TD><TD>No
</TD><TD>N/A
</TD><TD>N/A
</TD></TR></TBODY></TABLE><P>
Here are the visual results for each sample:
</P><P>This is our best result. The anti-GFP stain (green) is well colocalized with the Granule stain (red). It also appears to be well centralized in the cell which seems to show that it is stably located in the correct granules, since it is not being sent out of the cell.
</P><P>In this image we can see granular looking anti-GFP staining (green). However, it is located near the cell membrane and does not seem to overlap with the granule stain (red).
</P><P>This image also shows well colocalized anti-GFP (green) and granules (red) (the yellow part represnts the heavily overlapped area) the results here are less clear than in image 56003, but nevertheless show centralized, colocalized anti-GFP and Granule stain.
</P><P>As you can see in this image, the granular anti-GFP is mostly at the edge of the membrane. The granule stain seems to slightly overlap the anti-GFP in some places, but is largely not colocalized with it. Since this construct only has the granzyme signal sequence we hypothesized that it would serve as a control for localization.
</P><P>In this image we have stained the pMAX-GFP with anti-GFP. As you can see, since the GFP is expressed throughout the cell, the stain is also everywhere. Unlike the EGFP constructs, this image has no granular GFP, but rather an entirely lite up cell.
</P><H3><SPAN class="mw-headline" id="Future_application">Future application</SPAN></H3><P>
We are encouraged by our preliminary results suggesting that we can successfully target the model protein EGFP into killer cell granules. With a few more tests we can fully optimize the process and figure out what combination of N- and C- terminal address tags work the best. Then, we can use the best combinations of address tags to route novel cytotoxic cargo into the granules for cancer killing. Some of the various ideas we have explored for cargo that we could load into granules include: proteins important for normal regulation such as p53, and the cytochrome and BL family proteins; Stronger killing agents, such as TNF and Par-4.
We can also use it as a new means for delivering existing cancer therapies, such as growth inhibitors, with more specificity. Once this process has been further investigated and fine tuned, it will offer a new revolutionary means of combating cancer and creating more powerful killer cells.
</P><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><P><B>1. L is for lytic granules: lysosomes that kill</B></P><P>Page LJ, Darmon AJ, Uellner R, Griffiths GM.
</P><P>Biochim Biophys Acta. 1998 Feb 4;1401(2):146-56.
</P><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/9531970" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/9531970</A></P><P><B>2. The lysosomal trafficking of sphingolipid activator proteins (SAPs) is mediated by sortilin.</B></P><P>Lefrancois S, Zeng J, Hassan AJ, Canuel M, Morales CR.
</P><P>EMBO J. 2003 Dec 15;22(24):6430-7.
</P><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/14657016" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/14657016</A></P><P><B>3. LIMP-2 is a receptor for lysosomal mannose-6-phosphate-independent targeting of beta-glucocerebrosidase.</B></P><P>Reczek D, Schwake M, Schr?der J, Hughes H, Blanz J, Jin X, Brondyk W, Van Patten S, Edmunds T, Saftig P.
</P><P>Cell. 2007 Nov 16;131(4):770-83.
</P><P><A href="http://www.cell.com/abstract/S0092-8674%2807%2901290-1" class="external free" rel="nofollow">http://www.cell.com/abstract/S0092-8674%2807%2901290-1</A></P><P><B>4. Cellular uptake of saposin (SAP) precursor and lysosomal delivery by the low density lipoprotein receptor-related protein (LRP).</B></P><P>Hiesberger T, Hüttler S, Rohlmann A, Schneider W, Sandhoff K, Herz J.
</P><P>EMBO J. 1998 Aug 17;17(16):4617-25.
</P><P><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1170791/" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1170791/</A></P><P><B>5. Granzymes A and B Are Targeted to the Lytic Granules of Lymphocytes by the Mannose-6-Phosphate Receptor</B>
Griffiths GM, Isaaz S.
</P><P>J Cell Biol. 1993 Feb;120(4):885-96.
</P><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/8432729" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/8432729</A></P><P><B>6. Hematopoietic secretory granules as vehicles for the local delivery of cytokines and soluble cytokine receptors at sites of inflammation</B></P><P>Hansson M, Gao Y, Rosén H, Tapper H, Olsson I.
</P><P>Eur Cytokine Netw. 2004 Jul-Sep;15(3):167-76.
</P><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/15542440" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/15542440</A></P><P><B>7. Glycosylation-independent targeting enhances enzyme delivery to lysosomes and decreases storage in mucopolysaccharidosis type VII mice.</B></P><P>LeBowitz JH, Grubb JH, Maga JA, Schmiel DH, Vogler C, Sly WS.
</P><P>Proc Natl Acad Sci U S A. 2004 Mar 2;101(9):3083-8. Epub 2004 Feb 19.
</P><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/14976248" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/14976248</A></P><P><B>References for sorting signals:</B></P><P><B>Sorting of lysosomal proteins</B></P><P>Thomas Braulke and Juan S. Bonifacino
</P><P>doi:10.1016/j.bbamcr.2008.10.016
</P><P><A href="http://www.cimr.cam.ac.uk/investigators/robinson/docs/Bonifacino%2009.pdf" class="external free" rel="nofollow">http://www.cimr.cam.ac.uk/investigators/robinson/docs/Bonifacino%2009.pdf</A></P></DIV><DIV id="right" style="width:100%;height:100%;"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Better_Arsenal"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Better Arsenal</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Brief_Introduction"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Brief Introduction</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Directing_GFP_to_killer_cells.27_granules_by_fusing_.22address.22_tags"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Directing GFP to killer cells' granules by fusing &quot;address&quot; tags</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Cloning_Strategy"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Cloning Strategy</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Device_List"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Device List</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Testing_the_Constructs"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Testing the Constructs</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Results"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Future_application"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Future application</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#References"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:UCSF/Project/Arsenal">http://2010.igem.org/Team:UCSF/Project/Arsenal</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UCSF/Project/Arsenal" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UCSF/Project/Arsenal" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UCSF/Project/Arsenal&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UCSF/Project/Arsenal&amp;oldid=207811" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>