<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Uppsala-Sweden_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Uppsala-Sweden/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Uppsala-Sweden/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Uppsala-Sweden/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Uppsala-Sweden/Notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Uppsala-Sweden/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Uppsala-Sweden/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV class="container_16"><DIV class="wrap_3"><DIV><P>Visit our blog:</P><P><A href="http://igem2011uu.blogspot.com" class="external free" rel="nofollow">http://igem2011uu.blogspot.com</A></P></DIV></DIV><DIV class="wrap_12"><DIV class="prefix_4 grid_11 suffix_1 maincontent" id="Uppsalacontent"><DIV class="grid_Orange_thin omega mywhite"><P>Notebook
</P></DIV><H2><SPAN class="mw-headline" id="Week_1"> Week 1 </SPAN></H2><P>This week was dedicated to make buffers such as CCMB80 and SOC, SOB and LB medium (See the protocols <A href="https://2011.igem.org/Team:Uppsala-Sweden/Project/Protocol#SOB-medium" class="external text" rel="nofollow">SOB-medium</A>, <A href="https://2011.igem.org/Team:Uppsala-Sweden/Project/Protocol#LB_.E2.80.93_Medium" class="external text" rel="nofollow">LB medium</A> and <A href="https://2011.igem.org/Team:Uppsala-Sweden/Project/Protocol#Competent_Cell_Preparation_Procedure" class="external text" rel="nofollow">competent cell preparation</A>). We also prepared selective agar plates with ampicillin, chloramphenicol and kanamycin (See agarplate preparation). Finally we prepared competent TOP10 cells and froze them in -80°C. We received the pTJ122 plasmid carrying the ccaS, ccaR and cph8 genes as well as the PcpcG2 promoter from Christopher A Voigt at University of California San Francisco.
</P><H2><SPAN class="mw-headline" id="Week_2"> Week 2 </SPAN></H2><H6><SPAN class="mw-headline" id="2011-06-27">2011-06-27</SPAN></H6><P>This week we started with testing the competence in our cells. For this transformation we used one positive control (plasmid pUC19 from New England Biolabs) as well as a negative control without plasmid. For the procedure we followed the protocol for <A href="https://2011.igem.org/Team:Uppsala-Sweden/Project/Protocol#Transforming_TOP10_competent_cells" class="external text" rel="nofollow">transforming TOP10 competent cells</A>. The result was good, we measured a competence efficiency of 1.7 * 10^8 transformants /ug DNA.
</P><P><SPAN style="color:blue">Blue</SPAN> / <SPAN style="color:green">green</SPAN> output
</P><P>The strains carrying the amilGFP (green/yellow output) and amilCP (blue output) plasmids were provided by J.F Miller, UCLA. They arrived on plates witch were malhandled during the delivery. Both colors looked really nice but since the colonies were all mixed into each other we started by re-plating to obtain single colonies. 
</P><H6><SPAN class="mw-headline" id="2011-06-28">2011-06-28</SPAN></H6><P><SPAN style="color:blue">Blue</SPAN> / <SPAN style="color:green">green</SPAN> output
</P><P>Started overnight cultures of E coli carrying the plasmids pGEM11-amilGFP and pGEM14-amilCP.  We followed the protocol for <A href="https://2011.igem.org/Team:Uppsala-Sweden/Project/Protocol#Overnight_culture_and_glycerol_stock" class="external text" rel="nofollow">overnight culture and glycerol stock</A> for the initial preparations on the output modules.
</P><P>Transformation of BioBrick plasmids (protocol for transforming TOP10 competent cells):
pSB1A3-J04450	(ampR backbone) – Lei &amp; Sibel 
pSB1C3-J04450	(CmR backbone) – Lei &amp; Sibel 
pSB1K3-J04450	(KanR backbone) – Lie &amp; Sibel 
pSB1AK3-B0014	(Double terminator) –Laura &amp; Pikkei 
pSB1AK3-B1001	(synthetic terminator) – Karl &amp;Hamid</P><P>pSB1A2-B0034	(Standard RBS) - Karl &amp;Hamid 
pSB2K3-I15008	(ho1, chromophore synthesis gene)- Mohammed, Erik L &amp; Tomas
pSB2K3-I15009	(pcyA, chomophore synthesis gene) - Mohammed, Erik L &amp;Tomas
pSB1A2-R0011	(PLlacO, lacI repressable promotor) -–Laura &amp; Pekkie
pSB2K3-I15010	(cph8 red sensor) – Lei &amp; Sibel</P><H6><SPAN class="mw-headline" id="2011-06-29">2011-06-29</SPAN></H6><P>Transformation of BioBrick plasmids (protocol for transforming TOP10 competent cells):
pSB2K3-Q03530	(cII inverter) - Tomas, Erik L, Pikkei, Lidaw 
pSB1AK3-B0015	(Double terminator) - Antonio, Lei, Sibel, Cherno 
pSB2K3-Q01511	(cI inverter) – Laura &amp; Hamid 
pSB1A2-R0082	(PompC) - Tomas, Erik L, Pikkei, Lidaw 
pSB1A2-K093005	(RBS + RFP, red pigment) - Laura &amp; Hamid </P><P>Streaked on plate 
pSB3T5-J04450	(low copy vector tet, ori P15A) – From Erik G </P><P>Evaluation: All samples were good. pSB1K3 plates with RFP inserts are less red than pSB1A3 and pSB1C3. Re-streaking of the successful transformations from 2011-07-28. The same people who did the transformation proceeded with re-streaking. 
Cph8 had no colonies at all, neither on the 1x or 10X dilution plate. Apparently there seems to be a problem with cph8. Thus, cph8 (red sensor) had to be redone due to failure in the first attempt. 
</P><H6><SPAN class="mw-headline" id="2011-06-30">2011-06-30</SPAN></H6><P>Re-streaking of the transformants from 11-06-29 and started overnight cultures from the re-streaked plates from the previous day (11-06-28). 
</P><P><B>Evaluation:</B>
Most transformations succeeded. However, cI inverter gave very little amount of colonies and cph8 failed again. It’s time to use another approach. We got one single colony on the transformation plates and we will screen that one, but if that colony is wrong will try to run a PCR using the standard VF2 and VR primers using the DNA in the kit as template. If there is any BioBrick plasmid in that well, we should get a PCR product. If this will not work out, we need to either synthesize the gene or to PCR up the gene from the Voigt plasmid pJT122. We will then need to clone it into a BioBrick plasmid and run site-directed mutagenesis on it to eliminate the illegal PstI restriction site. Since cph8 is an important part of our project we have to solve this dilemma.
</P><H6><SPAN class="mw-headline" id="2011-07-01">2011-07-01</SPAN></H6><P>Started by doing overnight cultures from the re-streaked plates from 11-06-30. 
After that Lei and Sibel sterile filtered 20 % glycerol for making glycerol stocks. All the overnight cultures prepared 11-06-30 were frozen in -80°C.
</P><H6><SPAN class="mw-headline" id="2011-07-02">2011-07-02</SPAN></H6><P>All the overnight cultures prepared 11-07-01 were mixed with glycerol and frozen in -80°C. At this stage content with the work done this far.
</P><H2><SPAN class="mw-headline" id="Week_3"> Week 3 </SPAN></H2><H6><SPAN class="mw-headline" id="2011-07-04">2011-07-04</SPAN></H6><P>Started by doing overnight cultures (3 ml in selective LB) of the strains carrying these plasmids:
pSB1A3-J04450	(vector, ampR)
pSB1C3-J04450	(vector, CmR)
pSB1K3-J04450	(vector, KanR)
pSB1A2-B0034	(Standard RBS)
pSB2K3-I15008	(ho1, chromophore synthesis gene)
pSB2K3-I15009	(pcyA, chomophore synthesis gene)
pSB2K3-Q03530	(cII inverter)
pSB1AK3-B0015	(Double terminator)
pSB2K3-Q01511	(cI inverter)
pSB1A2-R0082	(PompC)</P><P>The purpose was to make plasmid preparation the day after. 
</P><H6><SPAN class="mw-headline" id="2011-07-05">2011-07-05</SPAN></H6><P><SPAN style="color:green">Green sensor</SPAN>
The day started off by running BioBrick overhang PCR on the DNA template of the green light sensor, which includes the two genes ccaR, ccaS and the PcpcG2 promotor (see protocol ccaR BioBrick, protocol ccaS BioBrick &amp; protocol PcpcG2 BioBrick). Voigt(UCSF) provided the green sensor, the red sensor and PcpcG2 on plasmid pJT122 (11,088 bp) [<A href="http://www.ncbi.nlm.nih.gov/pubmed/21035461" class="external text" rel="nofollow">Tabor et al 2011</A>].
</P><P><SPAN style="color:green">Green</SPAN>&amp;<SPAN style="color:blue">Blue</SPAN> output
Site-specific mutagenesis was performed on amilGFP and amilCP in order to remove illegal EcoRI sites within the genes (protocol amilGFP_EcoRI, protocol amilCP_EcoRI).
</P><P>After lunch we did a plasmid preparation of the overnight cultures from 2011-07-04. 
</P><P><SPAN style="color:red">Red sensor</SPAN>
At the end of the day we performed colony PCR to verify the length of the single clone we got of red light sensor (cph8) strain. 
</P><H6><SPAN class="mw-headline" id="2011-07-06">2011-07-06</SPAN></H6><P>Preparation of agarose gel (1%) for the gel electrophoresis. Then we ran our PCR products from 11-07-05 on the gel. 
</P><P>Finally we initiated the first BioBrick assembly. The following entities were assembled:
</P><P><B>Chromophore</B> 
RBS (B0034) + pcyA (I15009) - Lidaw, Pikkei &amp; Johanna
RBS (B0034) +ho1 (I15008 ) - Erik L &amp; Tomas</P><P><SPAN style="color:red">Red</SPAN> output
PompC (R0082) + Inv.cl (Q01511) – Kalle, Hamid, Ismael</P><P>The plasmids were cut, the enzymes heat inactivated. After mixing the upstream parts, the downstream parts and the backbones, T4 ligase was added and the ligation mixes were left over night in room temperature.
</P><P><B>Evalutation:</B> We couldn’t observe any band on the gel corresponding to cph8. Thus the transformation of cph8 has failed completely. We came up with the following conclusions: either there were no or little DNA in the iGEM kit, or the kit just contained something else.
</P><P>All the other gels were successful but the band for pcpcG2 was slightly weak. The gel bands for ccaR and ccaS looked good. Hence, we have successfully PCR amplified the green sensor parts. Also, the PCR products from the site directed mutagenesis of amilCP_EcoRI and amilGFP_EcoRI showed strong and clear bands at the expected size. Furthermore we can now proceed on cloning ccaR, ccaS and the PcpcG2. 
</P><H6><SPAN class="mw-headline" id="2011-07-07">2011-07-07</SPAN></H6><P>Re-circularized of pigment vectors after site directed mutagenesis by Sibel. The PCR product was purified, phosphorylated with PNK, template plasmid was degraded using DpnI and finally the linear plasmid was re-circularized using T4 ligase (PCR purification protocol, Phosphorylation of DNA, DpnI digestion protocol). 
</P><P>Tomas and Antonio performed transformation of assembly RBS-ho1 (Protocol for transforming TOP10 competent cells). Transformation of mutagenized amilCP_EcoRI and amilGFP_EcoRI by Hamid and Lei.
</P><P>Digestion of ccaS and ccaR PCR products and plasmid backbone plasmid pSB1C3 by Tomas and Antonio followed up by cloning and transformation by Erik L (BioBrick Assembly Manual). Lidaw And Johanna transformed the RBS-PcyA assembly. 
</P><H6><SPAN class="mw-headline" id="2011-07-08">2011-07-08</SPAN></H6><P>The transformation from yesterday was successful; hence the same people could proceed with restreaking the transformants. Hamid and Tomas re-did PCR on pcpcG2 using primers with BioBrick overhangs because of the weak band we got last time. This time we got a stronger band. 
</P><P>λ <SPAN style="color:red">Red</SPAN></P><P>After internal discussions concerning progresses this far, we decided to start preparing the λ Red assembly parallel with the other assemblies. We need to knock out the gene envZ in our E coli strain to avoid cross-talk with our red sensor cph8. To do this, we will use Lambda Red recombineering [<A href="http://www.ncbi.nlm.nih.gov/pubmed/10829079" class="external text" rel="nofollow">Datsenko and Wanner, 2000</A>].
</P><P>First step was to perform colony PCR of a strain carrying the plasmid pKD4 (Datsenko) to generate a PCR fragment with a kanamycin resistance cassette flanked by FRT sites and homologies to envZ: “EnvZ- Knock out FRT-Kan” (PCR protocol of envZ knockout FRT-Kan_FRT). We could observe the bands around 1477 bp as we expected. However, we also got a stronger band for the negative control (NC). Even though it seems like we got the right product, the appearance of a band in the NC force us to redo the experiment. Just as a precautious measure. 
</P><H6><SPAN class="mw-headline" id="2011-07-10">2011-07-10</SPAN></H6><P>Today we screened the transformants from 11-07-08 
3A assemblies: 
RBS-pcyA,  RBS-ho1  
PompC-Inv.cl, 
Clonings: 
ccaS and ccaR.  Mutagenesis:  amilCP and amilGFP.  There were 6 clones of each transformants, 42 samples in total. 
</P><P>We selected and suspended one colony from each re-streaked clone in 20-30 µl of PBS. We ran colony PCR from of each suspended colony using taq polymerase using the VF2 and VR primers. We also started overnight cultures in selective medium from the suspensions, 1 ml in each.
</P><P>As a final attempt to get the cph8 gene from the iGEM kit, we tried running a PCR using the VF2 and VR primers and DNA from the kit as template. To be able to get high quality on the PCR product, we used Phusion DNA polymerase.
</P><P>Last thing was to run an agarose gel of the PCR products. We also ran the PCR product of the envZ Knockout cassette and cph8 on agarose gel. The envZ Knockout had a clear band (we lowered the annealing temperature from  60°C to 59°C). Last thing we did was doing PCR purification of envZ Knockout (Purification protocol). The cph8 showed no bands on the gel, indicating that there simply is no BioBrick plasmid DNA in the kit well.
</P><H2><SPAN class="mw-headline" id="Week_4"> Week 4 </SPAN></H2><H6><SPAN class="mw-headline" id="2011-07-11">2011-07-11</SPAN></H6><P>Today we did a frozen stock of all overnight cultures in -80°C freezer (ccaS clone 1,RBS-pcyA, amilGFP, amilCP, ccaR, RBS-ho1, PompC-Inv.cI). We also started 3 ml cultures by inoculating them with 100 µl of the overnight cultures of the two clones of  ccaR, RBS-pcyA and RBS-ho1, the constructs needed for the upcoming assemblies. When the cultures had grown for about 4 hours, we did plasmid preparations of them. We did PCR of amilCP using the pGEM14-amilCP_EcoRI plasmid as template and using primers to add BioBrick prefix and suffix. Mutagensis of amilGFP to remove PstI site and ccaS_EcoRI mutagenesis, round one.
</P><P>After studying the experience section on the Registry’s homepage, we decided to change inverters to parts Q04400 (TetR inverter) and Q04510 (cI inverter). The inverters we originally chose were not so well characterized, and the promoters they used as output might be too weak for our system.
</P><P>Transformation of parts Q04400 (TetR inverter) and Q04510 (cI inverter).
</P><H6><SPAN class="mw-headline" id="2011-07-12">2011-07-12</SPAN></H6><P>The day started by running a gel of the mutagenized ccaS, amilCP-BB10 PCR product, and amilGFP_PstI linear plasmid. At the same time we prepared the samples (ccaR, ccaS, RBS-ho1 and RBS-pcyA) for sequencing. Plating of strains carrying plasmids with terminator B1001, PLlacO, pSB3T5 and pSB4K5. Purification of PCR products of ccaS_EcoRI, amilGFP_PstI, amilCP_BB10, and PcpcG2. Re-ligation of mutagenized ccaS and amilGFP, and cloning of amilCP_BB10. 
</P><P>The transformation of inverters from yesterday didn’t go well. The lambda-cI-inverter had only a few colonies, while the tetR inverter had no colonies at all. So TetR inverter has to be redone. This time the transformation used 2 µl of DNA from the MIT distribution. Re-streak of cI inverter transformants from 2011-07-11.
</P><H6><SPAN class="mw-headline" id="2011-07-13">2011-07-13</SPAN></H6><P>We started the day by running EnvZ-KO, ccaS, EcoRI- digested amilCP and amilGFP on a gel. We obtained good results; there were no distinguishable multiple bands on the gel, although the bands seemed a bit smeared. Then we did transformation of ccaS_EcoRI, amilGFP_PstI, pSB1A3-amilCP, pSB-pcpcG2 and pSB4A5 backbone. Re-streak (There were one colony on the whole plate) and colony PCR of tetR inverter (DNA taken from well distributed from MIT). 
</P><P>Started overnight cultures of B1001 terminator, PlacO, PSB3T5 and PSB4K5-backbone (prepared for frozen stock + plasmid preparation) and finally RFP with RBS and lambda c1.inv (prepared for frozen stock + plasmid preparation).
</P><P>The last thing we did was to make new plates of all the three antibiotics (A, C, K). We also made some plate with tetracycline.
</P><H6><SPAN class="mw-headline" id="2011-07-14">2011-07-14</SPAN></H6><P>Re-streaking of ccaS_EcoRI, pGEM-amilGFP and pSB4A5. We were supposed to re-streak pSB1A3-amilCP and pSB1A3-PcpcG2 as well, but the transformants had no colonies at all.  We suspect wrong plasmid backbone or no ligase added during the cloning attempt. To investigate this the transformation was redone and we re-cloned pSB1A3-amilCP and pSB1A3-PcpcG2 again. O/n of pSB1A3 from the strain collection to test whether we had the right backbone in the frozen stock. We also transformed pSB1A3 backbone, and plated the transformants on A, C and K-plates. We have to find out what went wrong with the transformation of pSB1A3-amilCP and pSB1A3-PcpcG2.
Plasmid prep of pSB1AK3-B1001, pSB1A2-R0011, pSB3T5-backbone, pSB4K5 backbone. Freezing glycerol stocks of pSB4K5-B0034 and λ-c1.inv. PCR amplification of tetR.inv with phusion, ran a gel of the PCR products. 
</P><P>The Chromophore 
The second Biobrick assembly step on the chromophores was performed. Parts assembled: B0034-pcyA + B1001 and PLlacO (R0011) + B0034-ho1.  
</P><P>Output/promoter test plasmid 						                     
Tomas extracted the standard promoter J23101, pSB3K3 and EYFP (E0034) from the MIT kit in order to prepare the test plasmid.
</P><P>Evaluation 
The sequencing result arrived today around lunchtime and they were really good. Both clones of B0034-pcyA that we sent looked fine. B0034-ho1 also looked good. The only clone sent of ccaR looked good. ccaS from Tabor’s plasmid was correctly amplified.   
</P><P>Summary so far
</P><P>Things are going as planned but we have encountered some problem. We have successfully performed the first BioBrick assembly on the chromophore, extracted ccaR, ccaS and PcpcG2 from pJT122, mutagenized amilGFP_EcoRI and amilGFP_EcoRI and prepared a frozen stock of all the constructs generated so far. The sequencing data confirmed that everything was in order. However, we are still working on cph8 and have to incorporate new inverters in the system. We are happy with what we have accomplished so far and excited to tackle the challenges ahead. 
</P><H6><SPAN class="mw-headline" id="2011-07-15">2011-07-15</SPAN></H6><P>Erik L cloned tetR inverter into pSB1K3 and later transformed it. Lidaw and Johanna continued with the transformation of B0034-pcyA-B1001, Laura and Sibel transformed PLlacO-B0034-ho1 and pSB4K5. We did plasmid preparation of pSB1K3-B0032, o/n of pSB1C3-CcaS_EcoRI, pGEM-amilCP_PstI and pBS4A5.
</P><P>mCherry is an alternative red output module that we found in the BioBrick registry. We decided to try it out since its red output is stronger than RFP. Thus we transformed the mCherry (J06702, with RBS and double terminator) from the kit. 
</P><H6><SPAN class="mw-headline" id="2011-07-16">2011-07-16</SPAN></H6><P>Started by doing plasmid preparation and freezing PSB4A5 stock. We also did plasmid preparation of pSB1A2-B0034. Then we proceeded to screen of PcpcG2 and pGEM-amilCP_PstI.
</P><P>Made glycerol stock and &amp; mutagenesis/BB10 PCR of pSB1C3-CcaS_EcoRI (--&gt;ccaS_EcoRI_SpeI), pGEM-amilCP_PstI and pBS4A5 (amilGFP_Pst1_BB10). Only amilGFP_Pst1_BB10 showed a band on the gel. Re-streak of the transformations from yesterday and o/n of pcpcG2 and amilCP. 
</P><H6><SPAN class="mw-headline" id="2011-07-17">2011-07-17</SPAN></H6><P>Frozen stock and plasmid preparation of pcpcG2 and amilCP. Phosphorylation (Phosphorylation protocol), PCR purification, Diegestion, re-ligation, transformation and plating of ccaS_EcoRI_SpeI. PCR of AmilGFP_BB10. Plasmid preparation of pcpcG2 and amilCP. 
</P><P>O/n culture of:
pSB3T5_CA2A (B0034-pcyA-term)
pSB4K5_CA2B (pLlacO-B0034-ho1)
pSB1K3_tetR.inv
pSB1A3_mCherry
pSB3K3-backbone
pSB1K3_YF1
pSB1C3_BFP
J61002-J23101 (test plasmid promotor)
</P><H2><SPAN class="mw-headline" id="Week_5"> Week 5 </SPAN></H2><H6><SPAN class="mw-headline" id="2011-07-18">2011-07-18</SPAN></H6><P>Ran CA2A (B0034-pcyA-term), CA2B (pLlacO-B0034-ho1) and tetR.inv on gel and plasmid prepared of all the o/n cultueres from yesterday. Phusion PCR amplification (using BB10 primers) of non-Biobrick compatible cph8 from Voigt’s plasmid. This attempt was successful. We finally managed to get cph8. Gel ran for TagBFP and amilGFP. TagBFP came from colony PCR, and they turned out to have the right length. 
</P><P>Chromophore 
We finally put together the final step in the chromophore assembly; CA2B + CA2A --&gt; CA3  
</P><P>Red output assembly
PompC + tetR.inv 
</P><P>Green output assembly
PcpcG2 + B0034
</P><P>Blue output assembly
B0034+ amilCP
</P><P>Green sensor assembly
B0034 + ccaR
B0034 + ccaS_EcoRI_SpeI (completely mutated, BioBrick-compatible)
</P><P>We also assembled J23101 + mCherry (J06702)
</P><P>Frozen stock
pSB3T5_CA2A
pSB4K5_CA2B 
pSB1K3_tetR.inv
J61002-J23101– test plasmid
pSB1A3_mCherry
pSB3K3-backbone
pSB1K3_YF1
</P><P>λ Red
Create new plasmid for Lambda Red…………
</P><P>YF1, FixJ and FixK2 promoter (or PfixK) arrived today. Due to the overwhelming amount of work, we decided to wait for a couple of days before transforming them.
</P><H6><SPAN class="mw-headline" id="2011-07-19">2011-07-19</SPAN></H6><P>Transformation of all the Biobrick assemblies from yesterday and performed colony PCR on the plated ccaS_EcoRI_SpeI from 2011-07-17. We also did some O/N cultures of ccaS_EcoRI_SpeI. The gel showed nice and clear bands of the appropriate length.
</P><P>Cloning of amilGFP_BB10 into pSB1K3 and later it was transformed together with YF1, FixJ and PfixK. These are parts of the blue sensor and they were synthesized by GenScript. Cph8 was cloned into pSB1K3.
</P><P>The following parts were sent to sequencing: pcpcG2, CcaS_EcoRI_SpeI, CA2A, CA2B and tetR.inv. TagBFP and amilCP are sequenced by Erik Gullberg via IMBIM.
</P><H6><SPAN class="mw-headline" id="2011-07-20">2011-07-20</SPAN></H6><P>Plasmid preparation of B0034-ccaS_EcoRI_SpeI, Re-streaking of pSB4A4-CA3, pSB1C3-PompC-tetR.inv, pSB1K3-B0034-ccaR, pSB1C3-B0034-amilCP, pSB1C3-PcpcG2-B0034, pSB1A3-J23101-mCherry. Transformation of pSB1K3-cph8 and Re-streak of pSB1K3-amilGFP, YF1, fixJ and PfixK.  This was done late because they were transformed very late. By the end of the day we did new ampecilin plates since there were non left.
</P><P>The lambda red assembly plan must gain higher priority. The first assembly line involving EnvZ-KO could’ve been started earlier, but hasn’t. Thus, we have to hurry up the work on λ Red.
</P><H6><SPAN class="mw-headline" id="2011-07-21">2011-07-21</SPAN></H6><P>Screening and o/n culture of the re-streaked plate from yesterday. Followed by Re-streak of pSB1K3-cph8, pSB1A3-B0032-BFP, pSB1K3-B0034-BFP, pSB1C3-B0032-YF1, pSB1C3-B0034-YF1.  We plasmid prepared pSB1K3 and pSB1C3 backbones.
</P><P>Erik L became responsible for λ-Red in order to keep up with the assembly. He takes guidance from Erik Gullberg (supervisor) and together they lay down a good approach. 
Evaluation 
</P><P>Sequencing result of ccaS showed an unwanted insert. It’s probably because of bad primer alignment during mutagenesis. Our initial conclusion was that the primers amplified the DNA of some contaminants. Theoretically, this shouldn’t have occurred, since we did PCR purification. The small DNA that caused the insert should’ve been removed. 
</P><P>Erik Gullberg thinks we might have taken a bad clone from the plate when doing mutagenesis to remove EcoRI. So he thinks we should take another clone post-EcoRI and tries it out again. The result of the sequencing reflects a very rare accident. Theoretically we should be able to do it again, without having the same problem.
Otherwise, we are considering having Bioneer synthesize ccaS with codon optimization. 
CA2B samples were gave poor results, it seemed like we accidently put two primers into the sequencing sample. This is probably due to stress.
</P><P>Because of the work intensity and the stress people are starting to feel right now. Suggestions came up that we should move our weekly meeting on Mondays to Friday afternoon. Considering the current situation we decided to try it out next week. 
</P><H6><SPAN class="mw-headline" id="2011-07-22">2011-07-22</SPAN></H6><P>Frozen stock of:
amilGFP
YF1
FixJ
PfixK
pcpcG2-B0034
pompc-tetR.inv
</P><P>PCR purification of B0034-ccaR and the above mentioned parts. New Re-streaking of 8 colonies of ccaS_EcoRI. This is because we believe we were unlucky to pick a colony with insert. Thus we try our luck with other colonies. Then we plasmid prepared  amilGFP, B0034-ccaR, pcpcG2-B0034 and pompc-tetR.inv.
</P><P>Cloning of:
pSB1AK3-YF1  pSB1C3
pUC57-fixJ pSB1K3
pUC57-YF1 pSB1K3
pUC57-PfixJ pSB1K3
</P><P>O/N cultures and PCR screening of B0032-BFP, B0034-BFP, B0034-YFP and B0032-BFP.These are the parts of the output/promotor test plasmid assembly.
</P><H6><SPAN class="mw-headline" id="2011-07-23">2011-07-23</SPAN></H6><P>Screening of :
pSB4A5-CA3, pSB1C3-ccaS_EcoRI and pSB1C3-B0034-amilCP. Followed by screening of pSB1C3 – CcaS_EcoRI_SpeI (has insert).
</P><P>We also made screening of Cph8, 4 samples (VF2 and VR, biobrick backbone) and making the glycerol frozen stock of Chp8 (un-mutated, Biobrick Backbone). Screened with gel and all the bands had accurate lengths. 
</P><P>Glycerol frozen stock &amp; plasmid prep of:
pSB1A3-B0032-BFP 
pSB1K3-B0034-BFP
pSB1C3-B0032-YFP 
pSB1C3-B0034-YFP 
Mutagenesis of pSB1C3 – CcaS_EcoRI_SpeI (new 2nd mut). 
</P><P>Output/promotor test plasmid assembly
3A assembly of:
PompC-tetR.inv + B0034-BFP 
PompC-tetR.inv + B0034-RFP 
PompC-tetR.inv + B0034-RFP 
</P><P>Green output assembly
pSB1A3 – PcpcG2 + B0034-amilGFP
</P><P>Last thing was making O/N culture of pSB1C3-B0034-amilCP and of CA3.
</P><P>Evaluation
The new Ccas-EcoRI showed good bands, Ccas_EcoRI_SpeI did not (as expected due to illegal insert of approximately 50 bp shown after sequence data). The reason why we screened it again was to compare the band length with the new mutation of Ccas-EcoRI. Cph8 insert was successfully cloned in the pSB1K3. We had to redo PCR procedure of B0034-amilCP and do the second, new mutagenesis of ccaS_EcoRI to remove illegale SpeI restriction site.  
</P><H6><SPAN class="mw-headline" id="2011-07-24">2011-07-24</SPAN></H6><P>Started by running a gel on PCR-amplification products from yesterday. O/N- culture of pSB1C3-B0034-amilCP from yesterday had to be redone. The negative control was contaminated. CA3 was plasmid prepared.
</P><P>3A assembly followed by transformation of
</P><P>Output/promotor test plasmid assembly
pSB1A3-J23101 + B0032-BFP
pSB1K3-J23101 + B0032-YFP
pSB1A3-PcpcG2 + B0034-BFP
pSB1K3-PcpcG2 + B0034-YFP
</P><P>Inverter test plasmid assembly
pSB2K3-I0500 (Para/BAD)
pSB1A2-E0430 (YFP)
</P><P>Additional transformation of:
pSB1K3-YF1
pSB1K3-FixJ
pSB1K3-PfixK
pSB1K3 -PompC-tetR.inv-B0034-BFP
pSB1K3- PompC-tetR.inv-B0034 RFP
pSB1K3- PompC-tetR.inv-B0034-YFP
</P><P>Evaluation 
ccaS_EcoRI_SpeI (new 2nd mutagenesis) showed no bands which means it need to be redone. B0034 – amilCP has accurate length. Lei tried to improve the result by change the annealing temperature to 57,3 oC. Cph8 showed accurate length. 
</P><H2><SPAN class="mw-headline" id="Week_6"> Week 6 </SPAN></H2><H6><SPAN class="mw-headline" id="2011-07-25">2011-07-25</SPAN></H6><P>Re-streak and screening of all transformants from yesterday. New transformation of pSB2K3-I0500 due to no colonies from yesterdays plating. 
</P><P>Making Lambda Red compatible BioBrick plasmids
Screening of BioBrick plasmid pSB3T5_specR by using designed primers PEG3x5F (containing the whole terminator part and SacI restriction site) and primer, PEG3x5F2 (containing a part of the terminator and without SacI restriction site) 
</P><P>O/N-culture of pSB4A5-CA3, pSB1C3-B0034-amilCP and pSB1C3-YFP. Lei used his new protocol to screen ccaS_EcoRI_SpeI. He proceeded with phosphorylation, re-ligation and transformation. 
</P><P>Evaluation 
specR showed no band for the primer PEG3x5F. We decided to do PCR again by using primer PEG3x5F2 as a template, PEG3x5F and PEG3x5R as primers in order to check the functionality of the primers PEG3x5F2. The new result was to short bands on the gel. The primer PEG3x5F2 doesn't work and we need to design and order new primers. 
The mutagensis product of ccaS_EcoRI_SpeI showed accurate length on the gel and was therefore re-circulized.
</P><H6><SPAN class="mw-headline" id="2011-07-26">2011-07-26</SPAN></H6><P>Re-streak of pSB2K3-I0500 and CcaS_EcoRI_SpeI. Second screening of pSB1K3-CcaSEcoRI_SpeI to ensure we got the cells with correct insert. This was followed by PCR purification.  
</P><P>PCR of pSB1A2-PcpcG2-B0034-amilGFP. 
</P><P>Plasmid preparation and glycerol-stock of:
pSB1C3-E0034 (eYFP)
pSB4A5-CA3
pSB1C3-B0034-amilCP
</P><P>O/N-culture of re-streaked colonies:
pSB1K3-YFI
pSB1K3-FixJ
pSB1K3-PfixK
pSB1K3-cph8
pSB1C3-CcaSEcoRI_SpeI 
pSB1A2-E0430 (YFP)- test plasmid
pSB1A3-pcpcG2-B0034-amilGFP
</P><P>Prepared samples for sequencing (CA3, B0034-amilCP, cph8, ccaS_EcoRI_SpeI, pcpcG2-B0034, PompC-tetR.inv, B0034-CcaR, pcpcG2-B0034-amilGFP). 
</P><P>Evaluation 
The screening of CcaS_EcoRI_SpeI showed accurate length on the gel and was sent for sequencing. PompC-tetR.inv-B0034-RFP showed wrong length. 
</P><H6><SPAN class="mw-headline" id="2011-07-27">2011-07-27</SPAN></H6><P>Yesterdays’ samples were sent for sequencing. Glycerol stock and plasmid prep of the O/N cultures from Yesterdays’. Mohammed and Pikkei made new LB medium. We also screened our yellow output (PcpcG2-B0034-amilGFP) and the band on the gel showed accurate length. 
</P><P>Red output assembly
PompC-tetR.inv + B0034-RFP in pSB1K3-B0032 and pSB1K3-B0034-RFP. We wanted to the test the output since the output can become either red or colorless. 
</P><P>O/N -cultures of pSB2K3-I0500 (inverter test plasmid assembly).  
</P><P>Evaluation 
The screening showed that the assembly of yellow output was done successfully.
</P><H6><SPAN class="mw-headline" id="2011-07-28">2011-07-28</SPAN></H6><P>Today we did plasmid prep and glycerol stock of pSB2K3-I0500. Re-sreak of transformed and plated λ-Red plasmid and envZ amplicon (contains Kan-cassette)
</P><P>Screening of: 
YFI
FixJ
PfixK
PcpcG2-B0034-BFP
PcpcG2-B0034-YFP
J23101-B0032-BFP
J23101-B0032-YFP
PompC-tetR.inv-B0034-BFP
PompC-tetR.inv-B0034-YFP
</P><P>O/N cultures of pSB1A3-PcpcG2-B0034-BFP, pSB3K3-J23101-B0032-BFP
pSB3K3-J23101-B0032-YFP and pSB1K3-PcpcG2-B0034-YFP.
</P><P>Evaluation 
The band on the gel for the screening showed correct length of all samples except for the PompC-tetR.inv-B0034-BFP.
</P><H6><SPAN class="mw-headline" id="2011-07-29">2011-07-29</SPAN></H6><P>Johanna and Laura did glycerol stock and plasmid prep of the O/N - cultures from 2011-07-28. They also re-transformed the red output (PompC-tetR.inv + B0034-RFP) and did the experiment from 2011-07-27 again since we could come to a solid conclusion the first time.
</P><P>Blue output assembly 
PfixK+λC1.inv 
</P><P>Blue sensor assembly 
B0034+FixJ 
B0034+YF1
These assemblies were later transformed. 
</P><P>O/N- cultures of pSB1K3-PcpcG2-B0034-YFP 
</P><P>Evaluation 
There were no colonies on the re-streaked plate from yesterday. The result of the red output-assembly showed a plate with only white colonies (contain backbone pSB1K3-B0032) and one with both red and white (contain backbone pSB1K3-B0034-RFP). We picked the white colonies from the plate where the insert was put in pSB1K3-B0034-RFP- backbone which from now on be called red output with RFP (ROR). Hopefully this contains our insert and does not become induced by the TOP10-strain. 
</P><H6><SPAN class="mw-headline" id="2011-07-30">2011-07-30</SPAN></H6><P>Re-streak of all yesterday’s transformants. Today we also started making new competent TOP10 cells and inoculated them for the night. λ-Red plasmid is still processing slowly. But we ran a PCR to amplify envZ-KAN gene (Kan-cassette), followed by screening. O/N culture and screening of pSB1K3-PompC-tetR.inv-B0034-RFP. Point mutagenisis and O/N on pSB1K3-cph8_PstI since earlier mutagensis (first on) didn't work. Re-pleated, amplified and screening of earlier mutaded pSB1K3-Cph8_PstI. 
</P><P>Evaluation 
The gel picture of amplified pSB1K3-cph8_PstI showed good result. It also seemed like the point mutagenesis of pSB1K3-cph8_PstI had an accurate length. The reason why we did a new mutagenesis is because we had a lot of trouble with our sensor before and we wanted to be on the safe side. The gel picture of envZ-KO and the red output didn't show any band and needs to be redone. At this stage the failure rate has increased leading reforms in the standard procedures so far. However, we are coming closer to the goal and need to keep to going.
</P><H6><SPAN class="mw-headline" id="2011-07-31">2011-07-31</SPAN></H6><P>Testing the new top 10 competent cell by doing a transformation of PUC57 (with PfixK gene) and plating it on Amp plate. Running a gel of PfixK-λC1, B0034-YF1 and B0034-FixJ.  
</P><P>(egentligen 1/8, men jag flytta för att det ska passa med summary)
Colony PCR of B0034 - YF1 and B0034 – FixJ followed by screening. Additional screening of ROR and envZ-KO.
</P><P>PT5lac promotor arrived today
</P><P>Summary so far
</P><P>Blue output: B0034-amilCP (BOA) is assembled and sequenced and is ready for the final step. PfixK-λC1.inv is assembled but not yet sequenced. Blue output will soon be ready for the final assembly.
</P><P>Red output: Final assembly has been done several times with unsuccessful result. The reason for the problem is not clear. There seems to be a problem in the assembly of PompC-tetR.inv + B0034-RFP. Over and over again we can only see the band for tetR.inv on the gel. The sequence comes back with bad result every time. However, sequencing of the two individual components is good. Our screening method is might not be good enough.
</P><P>We decided to resume the attempt with mcherry and use it as a backup in case we would fail totally. 
</P><P>Green output: Final assembly (pSB1K3-pcpcG2 + B0034-amilGFP) is done and the sequencing data showed perfect assembly. However, we are considering putting the final construct in a low copy plasmid (pSB3C5) because the promotor is slightly leaky. 
</P><P>Chromophore assembly: Last assembly pSB4A5-PlacO-B0034-ho1 (CA2B) + B0034-pcyA-Term (CA2A) has been done but the sequencing result showed double peaks. Needs to be sequenced again. We will also check the previous assembly step CA2B by sending it for sequencing. At this stage we are considering an alternative assembly. We believe that ho1 activity decreases the concentration of Hem. Hence we will put the PLlacO promotor last.
</P><P>Red sensor: chp8 has been mutagenized two times to remove the illegal PstI site. The first time showed a large insert. Thus the mutagenezis has to be redone and send for sequencing. 
</P><P>Green sensor: the mutation of ccaS has been a tough one. It has to be mutagenized two times and the first sequencing result showed an insert. Assuming that we were un-fortuned to pick a colony with inserts. We went back and choose other colonies. It was the right assumption because the second sequencing was correct. ccaR is already assembled with B0034. 
</P><P>Blue sensor: Since the arrival of all the components the assembly of the blue sensor took of quit well. B0034-fixJ showed good sequencing results but B0034-YF1 showed no bands. Thus it needs to be assembled again!
</P><P>λ-red: Is lagging behind and has to be taken care of. All the attempts with envZ-KO so fare show modest or no results.   
</P><P>Modelling: remains one of the toughest challenges. So far we haven’t found a model that can be easily implemented and understood. We are still working on it.
</P><P>Bioneer 
We sent an order to one of our sponsors Bioneer to synthesize tree color proteins that we can use as alternative output modules. aeBlue and cjBlue are different tones of blue and eforRed is a red protein.
</P><P>Test plasmid assembly 
Antionio and Thomas are focusing on making test plasmids. The aim to be able to meassure the output, promotor activity as well as inverter activity. 
</P><H2><SPAN class="mw-headline" id="Week_7"> Week 7 </SPAN></H2><H6><SPAN class="mw-headline" id="2011-08-01">2011-08-01</SPAN></H6><P>Glycerol stock and plasmid preparation of PfixK-λC1.inv and red output. The chromophore (CA2A + CA2B) was PCR amplified together with B0034-RFP. Transformation of BBa_I732731.
</P><H6><SPAN class="mw-headline" id="2011-08-02">2011-08-02</SPAN></H6><P>PCR purification of B0034-FixJ followed by plasmid preparation and freezing. Transformation of PT5lac. PCR purification of yesterday and preparation for sequencing. Re-streak of BBa_I732731 and screening of envZ-KO and B0034-YF1. O/N-culture of BBa_I732731.
</P><P>O/N-culture of pSB3C5, pSB4C5 and pSB1C5
</P><P>Green sensor assembly
B0034 + ccaS  pSB1K3
</P><H6><SPAN class="mw-headline" id="2011-08-03">2011-08-03</SPAN></H6><P>Plasmid preparation and glycerol stock of yesterday’s O/N-culture and cloning of pSB1K3-pcpcG2-B0034-amilGFP  pSB3C5. Re-streak of PT5lac and O/N culture of pSB4A5-CA3, pSB3K5, and PompC-tetR.inv. 
</P><P>Output/promotor test plasmid assembly
PompC-tetR.inv+B0034-BFP
PompC-tetR.inv+B0034-YFP
</P><P>Blue output assembly
PfixK+λC1.inv + B0034-amilCP
</P><P>Blue sensor assembly
B0034 + YF1
</P><P>Sent for sequencing:
CA2A, CA2B, PompC-tetR.inv, B0034-RFP, chp8_ PstI (new mut), B0034-FixJ, B0034-YF1.
</P><H6><SPAN class="mw-headline" id="2011-08-04">2011-08-04</SPAN></H6><P>Transformation of chp8_ PstI, pcpcG2-B0034-amilGFP, plasmid preparation and glycerol stock of BBa_I732731, PompC-tetR.inv and CA3. Re-streak of blue sensor and output (B0034-YF1, PfixK- λC1-B0034-amilCP), PompC-tetR.inv-B0034-BFP and of PompC-tetR.inv-B0034-YFP. Screening of B0034-ccaS. Digestion of pSB1C3-B0034-YF1 with XbaI and with pstI- restriction enzyme. PCR amplification of CA3, screening and O/N-culture of PT5lac.
</P><P>Output/promotor test plasmid assembly
PfixK- λC1.inv + B0034-YFP
PfixK- λC1.inv + B0034-BFP
</P><P>Inverter test plasmid assembly
Para/BAD + B0034-BFP
Para/BAD + B0032-BFP
</P><P>Green sensor assembly
B0034+ccaS  pSB1K3
</P><P>Red sensor assembly
B0034+cph8  pSB1C3
</P><P>O/N- culture of PompC-tetR.inv, PompC-tetR.inv-B0034-RFP, B0034-ccaS
</P><P>Evaluation
B0034-ccaS and PT5lac showed accurate length on the gel. Digestion of pSB1C3-B0034-YF1 because we wanted to check the length of the insert. Furthermore the sequencing result were good thus we can proceed with the following steps.
</P><H6><SPAN class="mw-headline" id="2011-08-05">2011-08-05</SPAN></H6><P>Started with transformation of Para/BAD-B0034-BFP, Para/BAD-B0032-BFP, B0034-ccaS, B0034-cph8. We also did Plasmid prep and glycerol stock of PompC-tetR.inv-B0034-RFP, B0034-ccaS. 
</P><P>We ran B0034-YF1, envZ-KO and CA3, on a gel and all the samples showed accurate length this time. PCR purification of B0034-ccaS.
</P><P>Screening of: 
pSB1A3-PompC-tetRinv-B0034-RFP
pSB1A3-PompC-tetRinv-B0034-YFP 
</P><P>Output/promotor test plasmid assembly
PfixK- λC1.inv + B0034-YFP 
PfixK- λC1.inv + B0034-BFP
</P><P>Re-streak of chp8_ PstI. O/N- culture of B0034-YF1, PfixK- λC1-B0034-amilCP. 
</P><P>Lambda red plasmid (Anto. pikkei…..)
</P><P>Evaluation
No band shown after gel screening of pSB1A3-PompC-tetRinv-B0034-RFP and pSB1A3-PompC-tetRinv-B0034-YFP thus no need for O/N- Culture yet. The screening was redone with a lower annealing temperature. 
</P><H6><SPAN class="mw-headline" id="2011-08-06">2011-08-06</SPAN></H6><P>Re-streak of yesterday’s transformats. Successful screening of pSB1A3-PompC-tetRinv-B0034-BFP and pSB1A3-PompC-tetRinv-B0034-RFP. Hence, we proceeded with O/N-culture B0034-YF1 and PT5lac. 
</P><P>Screening of;
PfixK-λC1.inv-B0034-amilCP
B0034-YF1
PUC57-PT5lac
</P><P>Evaluation 
Only B0034-YF1 gave good result. New blue assembly of blue output is required.
</P><P>Green sensor assembly
B0034-ccaS + B0034-ccaR  pSB4C5 
Followed by transformation
</P><H6><SPAN class="mw-headline" id="2011-08-07">2011-08-07</SPAN></H6><P>Amplification of λ-red plasmid and PCR purification of envZ-KO and B0034-YF1. We also did plasmid preparation and glycerol stock of yesterday’s O/N-cultures and re-streak of B0034-ccaS-B0034-ccaR. We made new agar plates, and as usually plates with A, C and K resistance. 
</P><P>Screening of: 
B0034-ccaS
PfixK-λC1.inv-B0034-YFP
PfixK-λC1.inv-B0034-BFP
B0034-cph8
PcpcG2-B0034-amilGFP
PcpcG2-B0034-BFP
J23101-B0032-BFP
PcpcG2-B0034-YFP
J23101-B0032-YFP
</P><P>Evaluation 
Good results for:
PcpcG2-B0034-BFP
J23101-B0032-BFP
J23101-B0032-YFP
PcpcG2-B0034-YFP
PcpcG2-B0034-amilGFP 
B0034-ccaS
</P><P>Bad result for: 
PfixK-λC1.inv-B0034-YFP
PfixK-λC1.inv-B0034-BFP 
B0034-cph8
</P><H2><SPAN class="mw-headline" id="Week_8"> Week 8 </SPAN></H2><H6><SPAN class="mw-headline" id="2011-08-08">2011-08-08</SPAN></H6><P>PCR purification of yesterday’s screens followed by glycerol and plasmid. Cloning of PT5lac  pSB1C3 and O/N-culture of Para/BAD-B0032-BFP and B0034-ccaS-B0034-ccaR. 
</P><P>Output/promotor test plasmid assembly
J23101 + B0034-BFP
J23101 + B0034-YFP
J23101 + B0034-amilCP
J23101 + B0034-amilGFP
</P><H6><SPAN class="mw-headline" id="2011-08-09">2011-08-09</SPAN></H6><P>Transformation of pSB3C5, pSB3K5 and pSB3S5 for later λ-red constructs. Additional transformation J231001-B0034-amilCP and J231001-B0034-amilGFP. The assembly of J23101 + B0034-BFP and J23101 + B0034-YFP didn’t succeed. Plasmid preparation of Para/BAD-B0032-BFP and B0034-ccaS-B0034-ccaR. 
</P><P>Screening of:
PfixK-λC1.inv-B0034-YFP, PfixK-λC1.inv-B0034-BFP and B0034-cph8 again. pSB1A3-PompC-tetRinv-B0034-YFP
pSB1A3-PompC-tetRinv-B0034-RFP
pSB1A3-PompC-tetRinv-B0034-BFP
Para/BAD-B0032-BFP
Para/BAD-B0034-BFP
B0034-ccaS-B0034-ccaR
</P><P>Evaluation 
Good results only for:
Para/BAD-B0032-BFP
B0034-ccaS-B0034-ccaR 
</P><P>New assemblies for the failed screenings
</P><P>Sent for sequencing:
B0034-YF1- B0034-FixJ, PfixK-λc1-inv-B0034-amilCP, PT5lac, PfixK-λc1 inv-B0034-YFP, B0034-ho1-B0034-PcyA-term, RBS-cph8, B0014-PlacO, Para/Bad-RBS-BFP, pSB3K3-K098010 and pSB1A3-PompC-tetRinv-B0034-RFP
</P><H6><SPAN class="mw-headline" id="2011-08-10">2011-08-10</SPAN></H6><P>Re-streak of J231001-B0034-amilCP and J231001-B0034-amilGFP. New assemblies made for failed sequencing followed by transformation. 
</P><P>Blue output assembly
New assembly of PfixK-λC1.inv + B0034-amilCP
Followed by transformation
</P><P>Output/promotor test plasmid assembly
PfixK-λC1.inv + B0034-BFP 
PfixK-λC1.inv + B0034-YFP
Followed by transformation
</P><H6><SPAN class="mw-headline" id="2011-08-11">2011-08-11</SPAN></H6><P>Evaluation
The sequence data for Para/BAD+B0032-BFP was hard to interpret. The sequence data of the new assembly of pSB4K5-PLlacO-B0034-ho1, pSB1K3-PompC-tetRinv-B0034-RFP and the old assembly of pSB1K3-PompC-tetRinv-B0034-RFP didn’t show satisfying result. However, B0034-RFP segment is clear. The rest of the samples that we sent for sequencing showed good result. 
</P><P>Re-streak of yesterday’s transformants and pSB3C5-FRT backbone and pSB3S5-FRT. 
</P><P>O/N-cultures of 
pSB1AK3-B0014  
pSB1A2-R0011
pSB1K3-J23101-B0034-amilCP 
pSB1K3-J23101-B0034-amilGFP 
</P><P>Red sensor assembly
New assembly of B0034+cph8  pSB1C3
Followed by transformation
</P><P>Making Lambda Red compatible BioBrick plasmids
Repeat cloning of SpecR and KanR-amplicon 
</P><H6><SPAN class="mw-headline" id="2011-08-12">2011-08-12</SPAN></H6><P>Re-streak of yesterday’s transformants and of A, C, K-backbone. 
</P><P>Screening of:
PT5lac 
PfixK-λC1.inv-B0034-amilCP
PfixK-λC1.inv-B0034-YFP
PfixK-λC1.inv + B0034-BFP
Plasmid prep and frozen stock of: 
PT5lac
PfixK-λC1.inv-B0034-amilCP
PfixK-λC1.inv-B0034-YFP
pompC-tetR.inv-B0034-RFP
J23101-B0034-amilGFP
J23101-B0034-amilCP
</P><P>Assembly of our coupled solution for chromphore and sensor 
B0014+PLlacO  pSB1C3
Output/promotor test plasmid assembly
New assembly of PfixK-λC1.inv+B0034-BFP 
</P><P>Blue sensor assembly
RBS-YF1+RBS-fixJ
Followed by transformation
</P><P>Chromophore assembly
RBS-ho1+RBS-pcyA-B1001 (term)
Followed by transformation
</P><P>Evaluation
Since the band of PfixK-λC1.inv + B0034-BFP showed wrong length we need to do the assembly of again. The rest of the samples showed the right length. 
</P><P>Making Lambda Red compatible BioBrick plasmids
Ligation of Spec and KanR……..
</P><P>Transformation of:
pSB1A10, the plasmid that are going to be used for the inverter test plasmid.
Transformation of different inverter components taken from the BioBrick-kit. 
Followed by transformation of Lambda red components (SpecR, KanFRT). 
</P><H6><SPAN class="mw-headline" id="2011-08-13">2011-08-13</SPAN></H6><P>Re-streak of pSB1A10 and RBS-YF1+RBS-fixJ. Additional re-streak of pSB3C5-FRT because of the first attempt failed. 
</P><P>Output/promotor test plasmid assembly
J23101 + B0034-BFP (new assembly)
J23101 + B0034-YFP (new assembly)
PT5lac + B0032-YFP  pSB3K3
PT5lac + B0032-BFP  pSB3K3
PT5lac + B0034-BFP  pSB1K3
PT5lac + B0034-YFP  pSB1A3
</P><P>
Red sensor assembly
New assembly of B0034+chph8 
</P><P>Inverter test plasmid assembly
New assembly of PARA/BAD+B0032
PARA/BAD+B0034
Followed by transformation
</P><P>
Amplification of pfixK-λC1.inv-B0034-amilCP with touch down PCR, since older gel picture showed two bands.
</P><P>Transformation of:
PfixK-λC1.inv-B0034-BFP
pSB1C3-B0014-PLlacO
pSB3K3-K098010 (Harvard's chromo mod)
</P><P>O/N-culture of:
pSB1A3-backbone
pSB1C3-backbone
pSB1K3-backbone
</P><P>Evaluation
Still, even though using a touch down approach, we saw one weak second band for the pfixK-λC1.inv-B0034- amilCP amplicon. Next time we will change the protocol with a slightly higher starting and ending temperature. 
</P><H6><SPAN class="mw-headline" id="2011-08-14">2011-08-14</SPAN></H6><P>O/N culture of pSB3C5-FRT (λ-red plasmid) and screening of RBS-YF1+RBS-fixJ. 
</P><P>Re-streak of 
PfixK-λC1.inv-B0034-BFP
B0014-PLlacO
B0034+chph8
</P><P>Antonio
Digest; pSB3C5SpecR
pSB3X5Kan-FRT
</P><P>Plasmid prep and glycerol stock of 
pSB1A3-backbone
pSB1C3-backbone
pSB1K3-backbone
</P><P>
Second amplification attempt of PfixK-λC1.inv-B0034-BFP with an alter touch down PCR protocol. Re-streak of yesterday’s transformants.
</P><P>Transformation of
pSB1A2-J06504 (mcherry, from kit)
pSB1K3-J23101-B0034-YFP
pSB1K3-pT5lac-B0034-BFP
pSB1K3-pT5lac-B0034-YFP
pSB3K3-pT5lac-B0032-YFP
pSB3K3-pT5lac-B0032-BFP
</P><H2><SPAN class="mw-headline" id="Week_9"> Week 9 </SPAN></H2><H6><SPAN class="mw-headline" id="2011-08-15">2011-08-15</SPAN></H6><P>Plasmid prep and glycerol stock of pSB3C5-FRT, RBS-YF1+RBS-fixJ and pSB1A10. Re-streak of yesterday’s transformations. 
</P><P>Screening and O/N of:
B0034+chph8
pSB3K3-K098010
B0014-PLlacO
PT5lac-B0034-YFP
RBS-ho1-RBS-pcyA-B1001
</P><P>Output/promotor test plasmid assembly                                                                                                  pSB1A3-PompC-tetRinv + B0034-RFP 
Followed by transformation
Tomas Transformation and re-streak 
pSB3C5SpecR
pSB3X5Kan-FRT
</P><P>Evaluation
Satisfying screening results. However, pSB3C5-FRT, RBS-YF1+RBS-fixJ failed again. Wrong band on the gel.
</P><H6><SPAN class="mw-headline" id="2011-08-16">2011-08-16</SPAN></H6><P>PCR purification of yesterday’s screens
</P><P>Antonio
Digest; pSB3C5-Kan-FRT with Sal1 &amp; Sac1
</P><P>Ligation of al ready digested components: 
</P><P>Output/promotor test plasmid assembly                                                                                                  
pSB1K3-PT5lac + B0032-YFP
pSB1K3-PT5lac + B0034-BFP
pSB1K3-PT5lac + B0034-BFP
</P><H6><SPAN class="mw-headline" id="2011-08-17">2011-08-17</SPAN></H6><P>Transformation of:
PomPC-tet.R.inv-B0034-amilCP
PT5lac-B0034-BFP
PT5lac-B0032-YFP
PT5lac-B0032-BFP
KanR-FRT
SpecR
</P><P>O/N-cultures of:
PfixK-λ C1-B0034-BFP 
B0014
mCherry
PT5lac-B0034-YFP
J23101-B0034-YFP
J23101-B0034-BFP
</P><P>Red output assembly
PompC-tetRinv + B0034-amilCP --&gt; pSB1A3
A test to see if our assembly method to put our red output together should be done in a diffrent way.
</P><P>Red sensor assembly
B0034-Cph8 + B0014 --&gt; pSB3T5
Digestion of B0034-Cph8+B0014-PlacO --&gt; pSB3T5 
The promotor that we have been using might be toxic to the cell, therefor we are trying out different approaches. 
</P><P>Colony PCR of pSB1A3-PfixK-λ C1-B0034-BFP 
</P><H6><SPAN class="mw-headline" id="2011-08-18">2011-08-18</SPAN></H6><P>Glycerol stock of:
PfixK-λC1.inv-RBS-BFP
mCherry
PT5lac-RBS-YFP
J23101-RBS-BFP
J23101-RBS-YFP
</P><P>Restreak of: 
PompC-tetR.inv-RBS-amilCP
SpecR
KanR-FRT
PT5lac-B0032-YFP
PT5lac-B0034-BFP
PT5lac-B0032-BFP
</P><P>Transformation of:
RBS-Cph8-B0014-PlacO
</P><P>Ligation of pSB3T5-B0034-Cph8+B0014-PlacO over night. 
</P><P>Red output assembly
Assembly of: 
RBS+mCherry --&gt; pSB3C5 (Alternative for our red output.) 
RBS-cph8+B0014 --&gt; pSB3T5 
</P><H6><SPAN class="mw-headline" id="2011-08-19">2011-08-19</SPAN></H6><P>Re-streak of pSB3T5-RBS-cph8-B0014-PlacO
</P><P>O/N-culture of J23113 and pSB4K5-LacIq (Repressor)
</P><P>Screening of:
PfixK-λC1.inv-RBS-BFP
PT5lac-B0032-YFP
PT5lac-B0032-BFP
</P><P>Transformation of :
pSB1A3-PompC-tetRinv-B0034-BFP
pSB1K3-PompC-tetRinv-B0034-YFP
pSB3K5 FRT backbone (for lambda -red)
pSB3T5-RBS-cph8-B0014-PLlacO
</P><H6><SPAN class="mw-headline" id="2011-08-20">2011-08-20</SPAN></H6><P>Re-streak of:
PT5lac-RBS-YF1-RBS-FixJ
RBS-cph8-B0014-PlacO
PompC-tetR.inv-B0034-BFP
PompC-tetR.inv-B0034-YFP
RBS-cph8-B0014
</P><P>Plasmid prep and glycerol stock of:
PT5lac-B0032-YFP
PT5lac-B0032-BFP
laqIq
J23113
</P><P>Red output assembly
New assembly of 
B0034+mCherry --&gt; pSB1C3
since earlier transformation didn't show any colonies. 
</P><P>O/N-cultures of: 
KanR-FRT
SpecR
B0034-cph8-B0014-PLacO
PompC-tetR.inv-B0034-BFP
</P><P>Screening of:
SpecR
KanR-FRT
pSB3C5-FRT
RBS-cph8-B0014-PLacO
PompC-tetR.inv-B0034-BFP
PT5lac-B0034-BFP
</P><P>Evaluation
The screening showed accurate lenght of the following samples:
KanR-FRT
SpecR
B0034-cph8-B0014-PLacO
PompC-tetR.inv-B0034-BFP
</P><H6><SPAN class="mw-headline" id="2011-08-21">2011-08-21</SPAN></H6><P>Transformation of B0034-mCherry
</P><P>Frozen stock and plasmid prep of:
B0034-cph8-B0014-PlacO
SpecR
KanR-FRT
</P><H2><SPAN class="mw-headline" id="Week_10"> Week 10 </SPAN></H2><H6><SPAN class="mw-headline" id="2011-08-22">2011-08-22</SPAN></H6><P>Another, new transformation of B0034-mCherry, since yesterdays transformation didn't show anything on the plate. 
</P><P>Plating of EnvZ-KO (clone one and two)
</P><P>Screening of:
PompC-tetR.inv-B0034-BFP
PompC-tetR.inv-B0034-YFP
RBS-cph8-B0014
</P><P>Evaluation
Screening of samples,see above, showed accurate band length for all assemblies except for PompC-tetR.inv-B0034-BFP. 
</P><H6><SPAN class="mw-headline" id="2011-08-23">2011-08-23</SPAN></H6><P>Samples prepared and sent for sequencing:
PompC-tetR.inv-B0034-YFP
RBS-cph8-B0014-PlacO
RBS-cph8-B0014
</P><P>Blue sensor assembly
J23101+RBS-YF1-RBS-FixJ
</P><P>Green output assembly
J23101+B0034-ccaS-B0034-ccaR --&gt; pSB3K3
</P><P>Red sensor assembly
J23101+B0034-cph8 --&gt; pSB3K3
</P><P>Re-digestion and ligation of:
PfixK-λ C1--&gt; pSB1C3
B0034-BFP --&gt; pSB1K3
</P><P>Re-ligation of B0034-mCherry --&gt; pSB1C3
</P><H6><SPAN class="mw-headline" id="2011-08-24">2011-08-24</SPAN></H6><P>Plasmid prep and glycerol stock of:
B0034-CcaS-B0034-CcaR
J23113
</P><P>Transformation of:
pSB3K3-J23101-RBS-cph8
psB3C5-J23101-RBS-YF1-RBS-fixJ
RBS-mCherry
PfixK-λ C1-B0034-BFP
J23101-RBS-cph8
J23101-RBS-YF1-RBS-FixJ
PT5lac-RBS-YF1-RBS-FixJ
pSB1K3-J23101-B0034-CcaS-B0034-CcaR
</P><P>The Chromopher 
Assembly of J23113-RBS-ho1-RBS-pcyA-B1001, followed by transformation.
</P><P>Screening of: 
B0034-CcaS-B0034-CcaR,
RBS-YF1-RBS-fixJ, used to compare the band length with, PT5lac-RBS-YF1-RBS-FixJ
</P><P>Evaluation
Screening showed right band-length of PT5lac-RBS-YF1-RBS-FixJ and of B0034-CcaS-B0034-CcaR.  
</P><H6><SPAN class="mw-headline" id="2011-08-25">2011-08-25</SPAN></H6><P>Screening of :
J23101-B0032-BFP
J23101-B0032-YFP
PT5lac-B0032-BFP 
PT5lac-B0032-YFP
FRT-lacIq
EnvZ-KO
</P><P>O/n- culture of:
PT5lac-B0032-BFP 
PT5lac-B0032-YFP
mCherry 
PT5lac-RBS-YF1-RBS-FixJ
</P><P>Re-streak of:
J23101-RBS-YF1-RBS-FixJ
J23101-RBS-cph8
J23101-B0034-ccaS-B0034-ccaR
J23113-RBS-ho1-RBS-pcyA
PfixK-λ C1-B0034-BFP
</P><P>Evaluation
Gel picture of screened samples showed good length except for EnvZ-KO. 
</P><H6><SPAN class="mw-headline" id="2011-08-26">2011-08-26</SPAN></H6><P>Red output assembly
Redo assembly of B0034+mCherry --&gt; pSB1C3
</P><P>Screening of:
J23101-RBS-YF1-RBS.FixJ
PfixK-λ C1-B0034-BFP
J23101-RBS-cph8
FRT-lacIq
</P><P>O/n-culture of:
J23101-RBS-YF1-RBS-FixJ
PfixK-λ C1-B0034-BFP
J23101-RBS-cph8 
FRT-lacIq
</P><P>Evaluation
Very bad gel picture due to the agarose.
</P><H6><SPAN class="mw-headline" id="2011-08-27">2011-08-27</SPAN></H6><P>Screening and O/n-culture of:
J23101-B0034-ccaS-B0034-ccaR
J23113-RBS-ho1-RBS-pcyA-B1001
envZ-KO
</P><P>Transformation of:
B0034-mCherry
Pfixk
</P><H6><SPAN class="mw-headline" id="2011-08-28">2011-08-28</SPAN></H6><P>Output/promotor test plasmid assembly                                                                                                  
PcpcG2+B0032-BFP --&gt; pSB3K3
PcpcG2+B0032-YFP --&gt; pSB3K3
PlacO+B0032-BFP --&gt; pSB3K3
PlacO+B0032-YFP --&gt; pSB3K3
J23113+B0032-BFP --&gt; pSB3K3
J23113+B0032-YFP --&gt; pSB3K3
</P><P>Re-streak of: 
J23101-B0034-amilCP
B0034-ccaS-B0034-ccaR-RBS-cph8-B0014
RBS-ccaS-RBS-ccaR-RBS-cph8-B0034-PlacO
PfixK
</P><H2><SPAN class="mw-headline" id="Week_11"> Week 11 </SPAN></H2><H6><SPAN class="mw-headline" id="2011-08-29">2011-08-29</SPAN></H6><P>Screening of:
pSB3K3-B0034-ccaS-B034-ccaR-B0034-cph8-B0014
pSB3K3-B0034-ccaS-B034-ccaR-B0034-cph8-B0014-PLlacO
PfixK
</P><P>O/n-culture of:
J23113-RBS-ho1-RBS-pcyA-B1001
J23101-RBS-cph8
FRT-lacIq
J23101-B0034-amilCP
B0034-ccaS-B0034-ccaR-B0034-cph8-B0014
B0034-ccaS-B0034-ccaR-B0034-cph8-B0014-PlacO
PfixK
</P><P>Blue backbone assembly
J23101-B0034-amilCP+B0034 --&gt; pSB1C3 
J23101-B0034-amilCP --&gt; pSB1C3
</P><P>Red output assembly
B0034+mcherry --&gt; pSB1C3
</P><P>Transformation of:
pSB3K3-PcpcG2-B0032-BFP
pSB3K3-PcpcG2-B0032-YFP
pSB3K3-PLlacO-B0032-BFP
pSB3K3-PLlacO-B0032-YFP
pSB3K3-J23113-B0032-BFP
pSB3K3-J23113-B0032-YFP
</P><P>Evaluation
Couldn't see any band on the gel for PfixK and the bands for samples pSB3K3-B0034-ccaS-B034-ccaR-B0034-cph8-B0014 and
pSB3K3-B0034-ccaS-B034-ccaR-B0034-cph8-B0014-PLlacO had the wrong length. 
</P><H6><SPAN class="mw-headline" id="2011-08-30">2011-08-30</SPAN></H6><P>Another screening of 
pSB3K3-B0034-ccaS-B034-ccaR-B0034-cph8-B0014
pSB3K3-B0034-ccaS-B034-ccaR-B0034-cph8-B0014-PLlacO
but this time using both taq and phusion polymerase. 
</P><P>Transformation of
J23101-B0034-amilCP-B0014
B0034-mcherry
J23101-B0034-amilCP
</P><P>PCR purification of:
RBS-cph8
J23113-RBS-ho1-RBS-pcyA-B1001
</P><P>Re-streak of: 
pSB3K3-PcpcG2-B0032-BFP
pSB3K3-PcpcG2-B0032-YFP
pSB3K3-PLlacO-B0032-BFP
pSB3K3-PLlacO-B0032-YFP
pSB3K3-J23113-B0032-BFP
pSB3K3-J23113-B0032-YFP
</P><P>Output/promotor test plasmid assembly                                                                                                  
PfixK+B0032-BFP --&gt; pSB3K3
PfixK+B0032-YFP --&gt; pSB3K3
</P><H6><SPAN class="mw-headline" id="2011-08-31">2011-08-31</SPAN></H6><P>Screening of:
pSB3K3-PcpcG2-B0032-BFP
pSB3K3-PcpcG2-B0032-YFP
pSB3K3-PLlacO-B0032-BFP
pSB3K3-PLlacO-B0032-YFP
pSB3K3-J23113-B0032-BFP
pSB3K3-J23113-B0032-YFP
</P><H6><SPAN class="mw-headline" id="2011-09-01">2011-09-01</SPAN></H6><P>Transformation of: 
PfixK-B0032-BFP
PfixK+B0032-YFP
</P><P>PCR amplification of FRT-lacIq
</P><H6><SPAN class="mw-headline" id="2011-09-02">2011-09-02</SPAN></H6><P>Re-streak of:
PfixK-B0032-BFP
PfixK-B0032-YFP
</P><H6><SPAN class="mw-headline" id="2011-09-04">2011-09-04</SPAN></H6><P>O/n- culture of:
J23101-B0032-BFP
J23101-B0032-YFP
PT5lac-B0032-BFP
PT5lac-B0032-YFP
PcpcG2-B0032-BFP
PcpcG2-B0032-YFP
PLlacO-B0032-BFP
PLlacO-B0032-YFP
J23113-B0032-BFP
J23113-B0032-YFP
</P><H2><SPAN class="mw-headline" id="Week_12"> Week 12 </SPAN></H2><H6><SPAN class="mw-headline" id="2011-09-05">2011-09-05</SPAN></H6><P>Screening of:
FRT-lacIq
Pt5lac-RBS-YF1-RBS-FixJ
J23101-RBS-YF1-RBS-FixJ
</P><P>Evaluation
The screened samples showed right length for FRT-lacIq but not for the rest. 
</P><H6><SPAN class="mw-headline" id="2011-09-07">2011-09-07</SPAN></H6><P>Plasmid prep and frozen stock of:
J23101-B0034-amilCP
J23113-RBS-ho1-RBS-pcyA-B1001
pGEM-cjBlue
pGEM-eforRed
J23101-RBS-amilCP-B0014
</P><P>Transformation of
pcpcG2
PompC-tetRinv
B0034-CcaS
CA2A
B0032-BFP
amilGFP
B0034-ccaR
YF1
FixJ
RBS-ho1-RBS-pcyA-B1001
RBS-YF1-RBS-fixJ
B0034-CcaS-B0034-CcaR
</P><H6><SPAN class="mw-headline" id="2011-09-08">2011-09-08</SPAN></H6><P>Assembly of: 
B0034 + eforRed 
B0034 + pompC-tetR
B0034 + mcherry
J23101-B0034 + eforRed 
J23101-B0034 + cjBlue 
</P><P>Re-streak of yesterday’s transformants.
</P><P>PCR amplification of:
B0034-cph8-B0014-PLlacO 
J23101-RBS-YF1-RBS-fixJ
</P><H6><SPAN class="mw-headline" id="2011-09-09">2011-09-09</SPAN></H6><P>O/n-culture of
J23101-RBS-amilCP-B0014
</P><P>PCR amplification:
pcpcG2
CA2A
B0032-BFP
B0034-BFP
YF1 
FixJ  
amilGFP
cph8_PstI 
B0034-CcaS 
RBS-ho1-RBS-pcyA-B1001
</P><P>Re-streak of:
PompC-tetR-B0034
B0034-ccaR
B0034-YF1-B0034-fixJ
J23101-B0034-eforRed
J23113-RBS-ho1-RBS-pcyA-B1001 
</P><H6><SPAN class="mw-headline" id="2011-09-10">2011-09-10</SPAN></H6><P>O/N-cultur of pSB1C3-J23101-RBS-amilCP-B0014. Digest of J23101-RBS-YF1-RBS-fixJ, pSB4A5-PT5lac-RBS-YF1-RBS-FixJ.Screening of yesderday’s amplicons
</P><P>Colony PCR of:
J23101-B0034-eforRed
J23113-RBS-ho1-RBS-pcyA-B1001 
B0034-YF1-B0034-fixJ
B0034-ccaR
PompC-tetRinv-B0034
PT5lac-RBS-YF1-RBS-FixJ
J23101-RBS-YF1-RBS-fixJ
</P><P>Evaluation
The gel showed no bands. Suspect that something might be wrong with the gel. Thus, we did a new batch.
</P><H6><SPAN class="mw-headline" id="2011-09-11">2011-09-11</SPAN></H6><P>New screening attempt of yesterday’s PCR. It was difficult to see anything significant from the other bands except from that of the digested blue sensor. PT5lac-B0034-YF1-RBS-FixJ and J23101-B0034-YF1-RBS-fixJ. 
</P><P>Tranformation of PT5lac-B0034-YF1-B0034-FixJ, pSB1K3-PfixK-λ C1-B0034-amilCP and pSB1K3-PfixK-λ C1-B0034-YFP in MG1655 strain.
</P><P>PCR amplification of PT5lac-B0034-YF1-B0034-FixJ for sequencing. 
</P><H2><SPAN class="mw-headline" id="Week_13"> Week 13 </SPAN></H2><H6><SPAN class="mw-headline" id="2011-09-12">2011-09-12</SPAN></H6><P>Cloning of constructs into pSB1C3 to be sent to the registry:
amilCP
PcpcG2-B0034-BFP
J23101-B0032-BFP
J23101-B0032-BFP
J23101-B0032-YFP
PcpcG2-B0034-amilGFP
B0034-CcaS 
PcpcG2-B0034-YFP
PfixK-λ C1-B0034-amilCP
PT5lac-B0032-BFP
PT5lac-B0032-YFP
2011-09-12
Re-streak of:
B0034-cjGreen
B0034-eforRed
PfixK-λ C1-B0034
</P><P>Prepare agar stab BL-21 E.coli strains to be sent to Bilkent Unam Unversity (Turkey, Ankara)
</P><H6><SPAN class="mw-headline" id="2011-09-13">2011-09-13</SPAN></H6><P>Transformation of the parts to be sent to the registry. Re-streak of pSB1K3-J23101-B0034-cjGreen followed by O/N.
</P><P>PCR and gel of;
J23101-B0034-CcaS-B0034-CcaR
PfixK-B0032-BFP
PfixK-B0032-YFP
PT5lac-RBS-YF1-RBS-FixJ
J23101-RBS-YF1-RBS-fixJ
</P><P>Screening showed good results.
</P><H6><SPAN class="mw-headline" id="2011-09-14">2011-09-14</SPAN></H6><P>Re-streak of:
pSB1K3-PfixK-λ C1-B0034-YFP
pSB1K3-PfixK-λ C1-B0034-amilCP
pSB1K3-J23101-B0034-CcaS-B0034-CcaR
pSB1C3-amilCP
pSB1K3-PfixK-λ C1-B0034
pSB1C3-PcpcG2-B0034-BFP
pSB1C3-PcpcG2-B0034-YFP
pSB1C3-PcpcG2-B0034-amilGFP
pSB1C3-B0034-CcaS 
pSB1C3-PfixK-λ C1-B0034-amilCP
pSB1C3-PT5lac-B0032-BFP
pSB1C3-PT5lac-B0032-YFP
pSB1C3-J23101-B0032-BFP
pSB1C3-J23101-B0032-YFP
</P><P>Assembly of:
J23101-B0034 + pGEM-eforRed  pSB1K3
</P><P>Cloning of:
pGEM-eforRed  pSB1C3
pGEM-cjGreen  pSB1C3
</P><H6><SPAN class="mw-headline" id="2011-09-15">2011-09-15</SPAN></H6><P>O/N-culture and glycerol stock of yesterday’s re-streaks.
</P><P>Plasmid prep of: 
RBS-YF1-RBS-fixJ
amilGFP
B0034-ccaR
B0032-BFP
B0032-YFP
J23113-B0032-BFP
PT5lac-B0032-BFP
PT5lac-B0032-YFP
PcpcG2-B0032-BFP
PcpcG2-B0032-YFP
PLlacO-B0032-BFP
PLlacO-B0032-YFP
J23101-B0032-BFP
</P><P>Assembly of:
J23113 + B0032-YFP 
J23101 + B0034-YFP
</P><H6><SPAN class="mw-headline" id="2011-09-16">2011-09-16</SPAN></H6><P>Re-streak of: 
cjGreen
pSB1C3-eforRed
J23101-B0034-cjGreen
J23101-B0034-eforRed
J23113-B0032-YFP
J23101-B0034-YFP (YFP from part registry)
Screening of:
pcpcG2
B0034-BFP
YF1 
FixJ
cph8_PstI 
B0034-CcaS 
PfixK-λ C1-B0034
J23101-B0032-YFP
PcpcG2-B0034-YFP
PcpcG2-B0034-amilGFP
B0034-CcaS 
PfixK-λ C1-B0034-amilCP
cph8_PstI 
PfixK-λ C1-B0034
pSB1C3-amilCP
PcpcG2-B0034-BFP
J23101-B0032-BFP
J23101-B0032-YFP
PcpcG2-B0034-YFP
PcpcG2-B0034-amilGFP
B0034-CcaS 
PfixK-λ C1-B0034-amilCP
PT5lac-B0032-BFP 
PT5lac-B0032-YFP
amilCP
</P><P>plasmid prep and O/n-culture of:
amilCP
PcpcG2-B0034-BFP
J23101-B0032-BFP
PfixK-λ C1-B0034-amilCP
PT5lac-B0032-BFP 
PT5lac-B0032-YFP
</P><P>Evaluation
Everything that we had screened showed the right length except for YF1 and cph8_PstI.
</P><H6><SPAN class="mw-headline" id="2011-09-17">2011-09-17</SPAN></H6><P>Re-streak of:
J23101-B0034-YFP (Erik Gullbergs YFP)
YFP (Erik Gullbergs YFP)
</P><P>Blue sensor assembly
J23101+RBS-YF1-RBS-fixJ   pSB4C5
</P><P>Output/promotor test plasmid assembly                                                                                                  
J23101+B0034-YFP  pSB3K3 (Erik Gullbergs YFP)
</P><P>O/n-culture of:
Pt5lac-RBS-YF1-RBS-fixJ
RFP
amilGFP
amilCP
cjGreen
</P><P>J23101-B0034-cjGreen
J23113-B0032-YFP
</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Uppsala-Sweden/Notebook">http://2011.igem.org/Team:Uppsala-Sweden/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Uppsala-Sweden/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Uppsala-Sweden/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Uppsala-Sweden/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Uppsala-Sweden/Notebook&amp;oldid=243442" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>