<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tsinghua-A skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tsinghua-A</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team:Tsinghua-A - 2015.igem.org</TITLE></P><DIV class="header-container"><DIV class="row"><NAV class="top-bar"><SECTION class="hide-for-small"><UL class="right"><LI><A href="#homepage">Home</A></LI><LI><A href="#project" id="firstclass1">Project</A></LI><LI><A href="#modeling">Modeling</A></LI><LI><A href="#humanpractice">Human Practice</A></LI><LI><A href="#collaboration">Collaboration</A></LI><LI><A href="#team">Team&amp;Attribution</A></LI></UL></SECTION></NAV></DIV></DIV><DIV class="homepage-text"><H1>We Are Dreamers</H1><H2>WE TRY TO INVENT A TEST PAPER FOR CANCER</H2><H3>Want to know how we do it?</H3></DIV><DIV class="content-container"><DIV class="parallax-one"><DIV class="row"><DIV class="twelve columns parallax-container"><H1 class="parallax-title">PROJECT</H1><H2 class="parallax-subtitle">In this column we are going to present the description,experiment and result of our project</H2></DIV></DIV></DIV><DIV class="page-container pattern-2" id="project"><DIV class="row"><DIV class="twelve columns page-content"><H1 class="page-title">Description &amp; Design</H1><H1 class="page-subtitle2"><SPAN>Description about Esophageal Cancer</SPAN></H1><P class="page-main" style="text-indent:1em">Esophageal cancer is cancer arising from the esophagus. It is the eighth most common cancer and sixth leading cause of cancer mortality in the world. Especially, nearly 90% of esophageal cancer patient are found in china. Chinese patients account for more than a half of all deaths caused by esophageal cancer worldwide. Genetic factors and bad eating habits (e.g. addiction to alcohol, addiction to tobacco, eating too fast and drinking hot liquids) can be the cause of esophageal cancer.</P><P class="page-main" style="text-indent:1em">5-year survival rate of esophageal cancer declines sharply as stage of cancer increases. In Stage I, during which cancer cells haven’t transferred, the 5-year survival rate can reach 70%. When it comes to Stage II, this number dramatically drops under 50%. And for Stage III and IV, it is 20% and 10% respectively. Thus early diagnosis plays an important role in the treatment of esophageal cancer.</P><P class="page-main" style="text-indent:1em">However, since the early symptoms are not significant and the methods for clinical diagnosis of esophageal cancer are expensive and inconvenient, which makes the early diagnosis become difficult. Prominent symptoms usually appear only when a patient is already in the advanced stage. Early symptoms like swallowing difficulty, pain when swallowing, reduced appetite and loss of weight are often considered a minor illness and easily ignored. The diagnosing method used in hospitals mainly depends on radiology and endoscopic biopsy, which seems to be invasive and uncomfortable. To help solve this dilemma our team aim to build a convenient and harmless method for early diagnosis of esophageal cancer.</P><H1 class="page-subtitle2"><SPAN>miRNA as Potential Biomarkers</SPAN></H1><P class="page-main" style="text-indent:1em">Studies have confirmed that miRNA expression is highly concordant cross individuals. And aberrant expression of miRNA may relate to diseases. Some studies have reported that specific miRNAs in tissue and plasma can be discriminatory bio-makers for detecting cancers. However, getting access to tissue of plasma means physical harm to the human body. So we turn to the easily accessible saliva. Since saliva is considered to be a terminal product of blood circulation, components like proteins and RNAs which are present in plasma are also present in saliva. In fact, both coding RNAs and non-coding RNAs, including some miRNAs, have been found in human saliva. Although mRNAs are highly degraded in saliva, miRNAs are stably and abundantly present in saliva. Recently, there are many reports on cancer-related miRNA expression in saliva. Featured miRNA expression are reported to be found in oral squamous cell carcinoma, parotid gland tumors and esophageal cancer, indicating potential salivary miRNA to be biomarkers for detecting these diseases. According to the test done by Zijun Xie group, there are three type of miRNAs significantly upregulated in the whole saliva from the esophageal cancer patient group in contract to normal control group – miR-10b, miR-144, and miR-451 (p value 0.001, 0.012 and 0.002, respectively; AUC 0.762, 0.706 and 0.756, respectively). And four miRNAs are significantly upregulated in saliva supernatants from the esophageal cancer patient group – miR-10b, miR-144, miR-21 and miR-451. Among them, miR-21 is the most frequently reported one for its high performance in specific expression level related to esophageal cancer (according to one of the reports, p &lt; 0.05, AUC = 0.8820, sensitivity = 90.20% and specificity 70.69%; different tests may report different results).</P><P class="page-main" style="text-indent:1em">Comprehensively considering the performance of each miRNA, we finally chose <SPAN>miR-144</SPAN> as our biomarkers for esophageal cancer. To make our detection quick and convenient, we designed synthetic gene pathways <SPAN>based on paper</SPAN>. It will only require some saliva to complete the detection, which will do no harm to human body. And this techniques can be expanded to be used in the detection of many other diseases which has specific miRNA expression pattern in saliva. </P><P class="page-main" style="text-indent:1em">To make our detection quick and convenient, we designed synthetic gene pathways. It will only require some saliva to complete the detection, which will do no harm to human body. And this technique can be expanded to be used in the detection of many other diseases which has specific miRNA expression pattern in saliva.</P><H1 class="page-subtitle2"><SPAN>What is Toehold Switch</SPAN></H1><P class="page-main" style="text-indent:1em">We choose toehold switch as our miRNA detector. The structure of toehold switch is similar to hairpin, except it has a loop at the top as ‘toehold’. Toehold switch functions as riboregulator through linear-linear interaction between RNAs. When target RNA appears, it will bind one of the toehold switch stems and open the loop, exposing the RBS.</P><P class="page-main" style="text-indent:1em">Toehold switch systems are composed of two RNA strands referred to as the switch and trigger. The switch RNA contains the coding sequence of the gene being regulated. Upstream of this coding sequence is a hairpin-based processing module containing both a strong RBS and a start codon that is followed by a common 21 nt linker sequence coding for low-molecular-weight amino acids added to the N terminus of the gene of interest. A single-stranded toehold sequence at the 50 end of the hairpin module provides the initial binding site for the trigger RNA strand. This trigger molecule contains an extended single-stranded region that completes a branch migration process with the hairpin to expose the RBS and start codon, thereby initiating translation of the gene of interest.</P><H1 class="page-subtitle2"><SPAN>Our Circuit to Detect miRNA-144</SPAN></H1><P class="page-main" style="text-indent:1em">There are 3 parts in our circuit in total.</P><P class="page-main" style="text-indent:1em">Part 1 includes toehold switch for miRNA-144 and GFP coding sequence. When miRNA-144 exists, the switch is on and mRNA for GFP is transcribed.</P><P class="page-main" style="text-indent:1em">Part 2 contains toehold switch for GFP mRNA and T3 RNA polymerase coding sequence (BBa_K346000). Note that, since the maximum length of trigger RNA for toehold switch is about 25nt, so we analyzed GFP mRNA’s structure and choose a small piece from it which ensures binding specificity and stability.</P><P class="page-main" style="text-indent:1em">Part 3 is simply a GFP generator (BBa_E0840), with T3 promoter. As we know, T3 promoter can only function when bound with T3 RNA polymerase.</P><P class="page-main" style="text-indent:1em">So now it’s clear that, part 2 and part 3 are designed for amplification. They form a positive feedback loop. So the whole process is as follow: when miRNA-144 exists, it triggers part 1 and GFP mRNAs are transcribed. These mRNA on the one hand can be directly translated to GFP and show green fluorescence, on the other hand, can trigger part 2 and T3 RNA polymerase is transcribed and translated, which enables part 3 to work, thus transcribe more GFP mRNAs. And these GFP mRNAs can also be used to active more part 2.</P></DIV><DIV class="twelve columns page-content"><H1 class="page-title">Experiment &amp; Result</H1><P class="page-main" style="text-indent:1em"><SPAN>Firstly, we need to construct three parts of the gene circuit:</SPAN></P><P class="page-main" style="text-indent:1em"><SPAN>Part 1: T7 promoter + toehold triggered by miR144 + GFP + T7 terminator</SPAN></P><P class="page-main" style="text-indent:1em">For part 1, we want to add the toehold triggered by miR-144 to the upstream region of the GFP coding sequence. We used gene2oligo to design many short single-strand DNA and then conducted LCR and second PCR to synthesis our part 1.<A href="http://parts.igem.org/Part:BBa_K1311000:Design"><SPAN>  (http://parts.igem.org/Part:BBa_K1700000:Design)</SPAN></A></P><P class="page-main" style="text-indent:1em"><SPAN>Part 2: T7 promoter + toehold triggered by GFP mRNA + T3 RNA polymerase + T7 terminator</SPAN></P><P class="page-main" style="text-indent:1em">We can obtained T3 RNAP sequence from BBa_K346000 in iGEM distribution. This part was much larger than part 1, so LCR could be inconvenient and expensive. So we used assembly PCR method to add T7 promotor, toehold triggered by GFP mRNA and T7 terminator to the gene.<A href="http://parts.igem.org/Part:BBa_K1311001:Design"><SPAN>  (http://parts.igem.org/Part:BBa_K1700001:Design)</SPAN></A></P><P class="page-main" style="text-indent:1em"><SPAN>Part 3: T3 promoter + RBS + GFP + terminator</SPAN></P><P class="page-main" style="text-indent:1em">Part 3 consisted of a GFP generator and a T3 promoter. GFP together with RBS and terminator was got from BBa_E0840 from iGEM distribution. Then we again used assembly PCR to add T3 promotor to the gene.</P><P class="page-main" style="text-indent:1em"><SPAN>Secondly, we linked the 3 parts in order into one plasmid – pSB1C1.</SPAN> We designed primers which helped add BsaI cutting site to the parts and the backbone of the plasmid. BsaI site in the parts was mutated by overlap PCR. Then we used Golden Gate assembly to link the three parts and the plasmid together.</P><P class="page-main" style="text-indent:1em"><SPAN>Finally, we want to apply our experiment to the cell free system and make the freeze-drying paper and use the paper to test saliva for diagnosing esophagus cancer.</SPAN> We are really sorry that we didn’t finish this part because our time is limited and so we didn’t have time to do it.</P></DIV><DIV class="twelve columns page-content"><H1 class="page-title">Reference</H1><P class="page-main"><SPAN>1. Pardee K, Green A A, Ferrante T, et al. Paper-based synthetic gene networks.[J]. Cell, 2014, 159.</SPAN></P><P class="page-main"><SPAN>2. Green A, Silver P, Collins J, et al. Toehold Switches: De-Novo-Designed Regulators of Gene Expression[J]. Cell, 2014, 159:925–939.</SPAN></P><P class="page-main"><SPAN>3. Rouillard J M, Lee W, Truan G, et al. Gene2Oligo: oligonucleotide design for in vitro gene synthesis[J]. Nucleic acids research, 2004, 32(suppl 2): W176-W180.</SPAN></P><P class="page-main"><SPAN>4. Shimizu Y, Inoue A, Tomari Y, et al. Cell-free translation reconstituted with purified components[J]. Nature biotechnology, 2001, 19(8): 751-755.</SPAN></P><P class="page-main"><SPAN>5. Gibson D G, Young L, Chuang R Y, et al. Enzymatic assembly of DNA molecules up to several hundred kilobases[J]. Nature methods, 2009, 6(5): 343-345.</SPAN></P><P class="page-main"><SPAN>6. http://www.snapgene.com/resources/plasmid_files/your_time_is_valuable/</SPAN></P><P class="page-main"><SPAN>7. http://www.addgene.org/plasmid-protocols/gibson-assembly/</SPAN></P><P class="page-main"><SPAN>8. http://www.snapgene.com/resources/gibson_assembly/</SPAN></P><P class="page-main"><SPAN>9. Lin X, Lo H C, Wong D T, et al. Noncoding RNAs in Human Saliva as Potential Disease Biomarkers[J]. Frontiers in Genetics, 2015, 6.</SPAN></P><P class="page-main"><SPAN>10. Zijun, Xie, Gang, Chen, Xuchao, Zhang, et al. Salivary MicroRNAs as Promising Biomarkers for Detection of Esophageal Cancer[J]. Plos One, 2013, 8(4):e57502.</SPAN></P><P class="page-main"><SPAN>11. Minhua Y E, Penghui Y E, Zhang W, et al. [Diagnostic values of salivary versus and plasma microRNA-21 for early esophageal cancer].[J]. Journal of Southern Medical University, 2014, 34(6):885-889.</SPAN></P><P class="page-main"><SPAN> 12. http://rna.tbi.univie.ac.at/cgi-bin/RNAfold.cgi</SPAN></P><P class="page-main"><SPAN> 13. http://www.nupack.org</SPAN></P><P class="page-main"><SPAN> 14. http://helixweb.nih.gov/dnaworks</SPAN></P><P class="page-main"><SPAN> 15. http://berry.engin.umich.edu/gene2oligo</SPAN></P></DIV><DIV class="twelve columns page-content"><H1 class="page-title">Acknowledgement</H1><P class="page-main"><SPAN>Team Peking provided us pET-28a vector. (Although we didn’t use it at last.)</SPAN></P><P class="page-main"><SPAN>Tsinghua Team once gave us competent cells when we needed to do transformation but there was no competent cells left in our lab. And we used their machine to dry our part for part submission.</SPAN></P><P class="page-main"><SPAN>Xu Yingqi from Art School, Tsinghua University helps us with painting the picture. We appreciate her help sincerelly.</SPAN></P></DIV></DIV></DIV><DIV class="page-container pattern-2" id="modeling"><DIV class="parallax-two"><DIV class="row"><DIV class="twelve columns parallax-container"><H1 class="parallax-title">MODELING</H1><H2 class="parallax-subtitle">We attempt to describe, in a precise way, an understanding of the elements of a system of interest, their states, and their interactions with other elements. </H2></DIV></DIV></DIV><DIV class="row"><DIV class="twelve columns page-content"><H1 class="page-title">Overview</H1><P class="page-main" style="text-indent:1em">Our project hopes to judge whether a man has esophagus cancer or the tendency of having it (set as y∈{0,1}, 0 stands for not having it or no tendency and 1 otherwise) through the expression level of a few kinds of miRNA (set as x∈Rn, n denotes the kinds of the miRNA being selected) in human saliva. This problem can be decomposed into two sub-problems, which is 1) what kind of mathematical relationship does the expression level of these certain kinds of miRNA satisfies so that we can make the prediction of whether or not having the cancer and 2) how to construct a biological circuit to express this kind of mathematical relationship and the conclusion of the judgment.</P><P class="page-main" style="text-indent:1em">Sub-problem 1 can be summarized as a pattern recognition problem. We can use some statistical learning (supervised learning) methods to obtain the pattern of the way that the concentrations of these kinds of miRNAs are combined, concretely, a set of parameters that the classifier possesses, to make prediction of whether or not having cancer.</P><P class="page-main" style="text-indent:1em">If sub-problem 1 is an applied mathematical problem, then sub-problem 2 is more related to biology, because a biological system is needed to physically realize the mathematical model obtained from sub-problem 1. We can analyze this problem from a systematic view. The input of this system is the concentrations of n kinds of miRNA, which is quite trivial. And how to use a biological system to express “ill or not”? The common method is to use some chemical to be the signal molecule, and our project follows the same routine. So the output of the system is the expression level of a certain kind of chemical (such as GFP).</P><P class="page-main" style="text-indent:1em">If sub-problem 1 uses combinatory logic to characterize the relationship between y and x, first the thresholds should be set. Secondly, gene circuits needs to be designed to characterize the relationship of “AND” “OR” “NOT”. Usually this can be realized through the combination of different kinds of promoter or repressor and also cascade reactions and so on. To characterize the threshold, the binding site of the trigger and switch needs to be designed and so does the binding strength of the promoter and repressor. This part involves more analysis of biochemistry and biophysics.</P><P class="page-main" style="text-indent:1em">If sub-problem 1 utilizes linear combination to characterize the relationship between y and x, the analysis of the problem is similar to the above.</P></DIV><DIV class="twelve columns page-content"><H1 class="page-title">Model1</H1><H2 class="page-subtitle2"><SPAN>Aim</SPAN></H2><P class="page-main" style="text-indent:1em">To find out the relationship between whether or not have the cancer with the combinatory concentration of 4 or 3 kinds of miRNAs in the saliva.</P><H2 class="page-subtitle2"><SPAN>Idea</SPAN></H2><P class="page-main" style="text-indent:1em">This problem can be summarized as a classification problem. Concretely, we can use variable y, which only takes the values 1 or 0, to indicate with or without the cancer, and a 4 or 3 dimensional vector x, each element of which takes a real value, to represent the concentration of 4 or 3 kinds of miRNAs in the saliva. Then y can be modeled as a function of x:y=g(x)</P><P class="page-main" style="text-indent:1em">The parameters of the model is unknown, but it can be learned using some machine learning techniques. Considering the scale and nature of this problem, we can use Fisher linear discriminant, logistic regression or SVM to solve this classification problem. Concretely, we can separate the data into a training set and a test set. We will use the training set to obtain the parameters, and the test set to test whether these learned parameters can make good predictions on new examples.</P><H2 class="page-subtitle2"><SPAN>Method and Results</SPAN></H2><P class="page-main" style="text-indent:1em">We use the experimental data from Salivary MicroRNAs as Promising Biomarkers for Detection of Esophageal Cancer, Zijun Xie et. al. Altogether, there are 58 samples, 46 of which are patients and the rest of which are healthy control.</P><H2 class="page-subtitle2" style="font-size:28px">Plotting the ROC curve</H2><P class="page-main" style="text-indent:1em">The ROC curve can be used to test whether a single feature (in our case, the concentration of a particular kind of miRNA in a person’s saliva) is linearly separable. The bigger the area under the ROC curve, the higher the possibility of the single feature being linearly separable is.</P><P class="page-main" style="text-indent:1em">We have the following results:</P><P class="page-main" style="text-indent:1em"><SPAN>Using the original concentration of miRNA:</SPAN></P><P class="page-main" style="text-indent:1em"><SPAN>Using the log2 of the concentration of miRNA:</SPAN></P><P class="page-main" style="text-indent:1em"><SPAN>Using the log10 of the concentration of miRNA:</SPAN></P><P class="page-main" style="text-indent:1em">For the above 3 figures, the first 3 subfigures are the results of 3 kinds of miRNA in the sample’s whole saliva, and the last 4 subfigures are the results of 4 kinds of miRNA in the sample’s saliva supernatant. The results are not satisfying. We assume that it is because the dataset collected is not large enough.</P><H2 class="page-subtitle2" style="font-size:28px"> Application of different kinds of classifiers</H2><P class="page-main"><SPAN>Fisher Linear Discriminant:</SPAN></P><P class="page-main"><SPAN>Logistic Regression and SVM with linear kernel:</SPAN></P><P class="page-main" style="text-indent:1em">Logistic Regression and SVM are 2 kinds of commonly used classifiers which belong to the domain of machine learning. For this problem, considering the scale of the dataset and the number of features (see Machine Learning Lecture 12 on Coursera), it is most appropriate to use SVM with Gaussian kernel. This method was tried and the outcome was good. However, we decide not to choose this method. The reason is that Gaussian kernel or other more complex kernels transform the original features to another set of features, which is difficult to realize in in vivo biological systems, and the parameters obtained are hard to be transformed back to those of miRNA concentration.</P><P class="page-main" style="text-indent:1em"><SPAN>Tuning the Parameters:</SPAN></P><P class="page-main" style="text-indent:1em">Considering randomness, regularization is added to the model and the parameter needs to be chosen. Take logistic regression for whole saliva as an example. The regularization parameter is lambda, and the following routine is applied:</P><P class="page-main" style="text-indent:1em">Step 1: Set up a vector lambdas of the parameter lambda, each element of which is a value to be tested. To start with, the vector was [0.0001; 0.0003; 0.001; 0.003; 0.01; 0.03; 0.1; 0.3; 1; 3; 10].</P><P class="page-main" style="text-indent:1em">Step 2: Set up a k * kk zero matrix called Evaluation, to take down the time that one particular value of lambda has satisfied the criteria which will be discussed later. k is the length of the vector lambdas, while kk is the time that an evaluation for the whole vector is implemented.</P><P class="page-main" style="text-indent:1em">Step 3: The evaluation procedure is a triple loop. The outer loop is from 1 to kk; the middle loop is from 1 to 100 (This number can be modified); the inner loop is from 1 to k. We take one outer loop as an example. For one middle loop, we randomly split the dataset into a training set and a test set. The training set consists of 31 positive examples and 15 negative examples, while the test set 8 and 4. Then we go into the inner loop, which use each of the value in the vector to train the model and learn the model parameters. When the training is done, we would evaluate whether the learned model meets certain criteria. If so, the corresponding entry in the Evaluation matrix will add one. And the loop goes on and on until finish.</P><P class="page-main" style="text-indent:1em">Step 4: By checking out the Evaluation matrix, we can see how many times that a value for lambda has met the criteria for good parameter. By sorting out the first 3 values that has the best performance, we can narrow down the scope of the parameter and set a new vector for lambdas and repeat the above procedure.</P><P class="page-main" style="text-indent:1em"><SPAN>The Final Value for the Regularization Parameter:</SPAN></P><P class="page-main" style="text-indent:1em">1. lambda for whole saliva logistic regression: 0.0016</P><P class="page-main" style="text-indent:2em">Criteria:</P><P class="page-main" style="text-indent:2em">1) 2 F1-scores both ≥ 0.65</P><P class="page-main" style="text-indent:2em">2) 2 accuracies both ≥ 0.8</P><P class="page-main" style="text-indent:2em">3) mean and variance: mean = 0.232, variance = 0.00097</P><P class="page-main" style="text-indent:1em">2. lambda for supernatant logistic regression: 0.03</P><P class="page-main" style="text-indent:2em">Criteria:</P><P class="page-main" style="text-indent:2em">1) 2 F1-scores both ≥ 0.55</P><P class="page-main" style="text-indent:2em">2) 2 accuracies both ≥ 0.7</P><P class="page-main" style="text-indent:2em">3) mean and variance: mean = 0.25, variance = 0.0008</P><P class="page-main" style="text-indent:1em">3. SVM with linear kernel performs badly on the dataset of whole-saliva samples, so we give up using this model to classify the whole-saliva samples.</P><P class="page-main" style="text-indent:1em">4. C for SVM with linear kernel: 10000</P><P class="page-main" style="text-indent:2em">It took about 20 hours to select the parameter. Totally the optimization process was called 1500 times. The larger the C value is, the longer it took to train. Considering performance and efficiency, we chose the value 10000.</P><P class="page-main" style="text-indent:2em">Criteria:</P><P class="page-main" style="text-indent:2em">1) 2 F1-scores both ≥ 0.5</P><P class="page-main" style="text-indent:2em">2) 2 accuracies both ≥ 0.7</P><P class="page-main" style="text-indent:2em">3) mean and variance: mean = 0.145, variance = 1 * 10-4</P><P class="page-main" style="text-indent:1em"><SPAN>Matlab Source Code:</SPAN></P><P class="page-main" style="text-indent:1em"><A href="https://2015.igem.org/File:TsinghuaA_source_code_for_model1.zip">Download link(Click here)</A></P></DIV><DIV class="twelve columns page-content"><H1 class="page-title">Model2</H1><H1 class="page-subtitle2"><SPAN>Using CV Technology to Quantitative Forecasting</SPAN></H1><P class="page-main" style="text-indent:1em">Hypothetically, you use the test paper we design and you can see three lines (Which means there are three kinds of mi-RNA have been found in your body fluid). Don’t worry, it doesn’t necessarily mean you got the cancer. Because the amount of those three mi-RNA has to satisfy this equation：</P><P class="page-main" style="text-indent:1em">So, we have to turn the GFP brightness information into the possibility of patient being afflicted with cancer. And we use the CV technology to help us achieve that goal.</P><P class="page-main" style="text-indent:1em">For example, we now have the photo of the test paper(The color of test paper itself might be light yellow due to the ambient light or the paper quality).</P><P class="page-main" style="text-indent:1em">We use Matlab to read this file and we can get an array of the RGB value of this image.Then,we can use this function to calculate the Brightness of each pixel:Y=0.299*R+0.587*G+0.114*B</P><P class="page-main" style="text-indent:1em">Then we have to normalize these data to eliminate the effect of the ambient light.We pick an pixel in the “white” area and calculate its brightness/255, set this as constant C, and use it to multiply every pixel’s brightness.</P><P class="page-main" style="text-indent:1em">After that, we will get a new array of the normalized brightness information. We can use the relative position to pick 10 pixel in each line, and calculate the mean value of each line’s brightness. We assume that there is a linear relationship between the brightness of the system and the amount of GFP in the system. Now we can get a relative stoichiometric relation of three mi-RNA. We use the function mentioned above to sum the data proportionately. We set B2P(Brightness to Possibility) as the symbol of this value.</P><P class="page-main" style="text-indent:1em">Finally we use this data to calculate the possibility of having cancer.(Of course we have to get the threshold of B2P value. But that can be easily calculated through a machine learning or a basic statistical method if we can get enough samples.)</P></DIV></DIV></DIV><DIV class="page-container pattern-2" id="humanpractice"><DIV class="parallax-three"><DIV class="row"><DIV class="twelve columns parallax-container"><H1 class="parallax-title">HUMAN PRACTICE</H1><H2 class="parallax-subtitle">We try to &quot;go beyond the lab&quot; to imagine our projects in a social context, to better understand issues that might influence the design and use of our technologies. </H2></DIV></DIV></DIV><DIV class="row"><DIV class="twelve columns page-content"><H1 class="page-title">Esophagus meets Tsinghua-A</H1><H2 class="page-subtitle2"><SPAN>The Situation</SPAN></H2><P class="page-main" style="text-indent:1em">Esophageal cancer (squamous cell carcinoma) is occurred in esophageal epithelial tissue malignant tumor, is one of the diseases having serious threat to people's health and life. The early symptoms of Esophageal Cancer is hidden and subtle so that there is rare clinical early esophageal cancer patients. While the patients who found that the symptoms, is late more, and lose the chance of early treatment.</P><H2 class="page-subtitle2"><SPAN>Morbidity</SPAN></H2><P class="page-main" style="text-indent:1em">Esophageal cancer (ICD - 10: C15) is a common cancer in China. According to the international agency for research on cancer in 2008, the world cancer incidence and death report (Globocan 2008) estimated that 481600 cases of esophageal cancer in the world, China's 259200 cases, accounting for 53.82% of the world, global 406500 cases died during this period, the China of 211100 cases (51.92%).</P><P class="page-main" style="text-indent:1em">In China, around the Taihang mountain area of Henan, Hebei and Shanxi provinces border region, especially in Linzhou, Anyang, and Huixian, esophageal cancer incidence and mortality rates are the highest in the world, high and low rates area arranged in concentric circles, only hundreds kilometers apart. This phenomenon is prompt environmental factors play an important role in esophageal cancer occurs. The lack of nutrient elements [vitamins such as riboflavin, etc.) and trace elements (such as zinc, etc.)] , nitrite, virus (HPV) and some toxins (such as mycotoxin) is considered to be important risk factors of the disease.</P><H2 class="page-subtitle2"><SPAN>Diagnosis and Treatment</SPAN></H2><H2 class="page-subtitle2"><SPAN>Preventive Tips</SPAN></H2><P class="page-main" style="text-indent:1em">1.	Eat less hot or irritant food, such as hot pot, pepper, etc. Long-term consumption of such food will result in esophageal mucosa injury.  If the stimulation last for some time, it may cause inflammation of the esophagus and esophageal cancer.</P><P class="page-main" style="text-indent:1em">2.	You must be chewed before swallowing the food and eat slowly so that food will be better digested, the poisonous substance will be neutralized and bacteria will be killed.</P><P class="page-main" style="text-indent:1em">3.	Maintain comprehensive nutrition intake, to prevent malnutrition. Because if we often avoid certain food, our human nutrition won’t reach the standard. Esophageal cancer maybe occur after the lack of protein and vitamin A, B.</P><P class="page-main" style="text-indent:1em">4.	Prevent carcinogens. While carcinogenic substances mainly nitrite, a lot of food contain this kind of material, such as vegetables or meat after curing, leftovers that remain overnight and  salt will generate into nitrite at high temperature. In addition, such as mildew grain, corn, peanuts contain large amounts of yellow aspergillus. Eating many moldy food will be very easy to cause the esophageal cancer.</P><P class="page-main" style="text-indent:1em">In addition to developing healthy eating habits, we should be patient with poor swallowing or foreign body sensation and do gastroscopy as early as possible in order to find early esophageal cancer or precancerous lesions. Remember that early esophageal cancer can be cured completely.</P></DIV><DIV class="twelve columns page-content"><H1 class="page-title">Our HP story</H1><P class="page-question" style="text-indent:0.5em"><SPAN>What has been most interesting and enjoyable for us about Human Practices?</SPAN></P><P class="page-main" style="text-indent:1em">We think Human Practice itself is an interesting thing for us which we enjoy so much. We know a lot about the society. Throughout this summer, we visit the best cancer hospital in China----The National Cancer Center and communicate with the experts who have a deep understandings of Esophagus Cancer in order to ensure that our project is addressing a genuine need as defined by those who know best.</P><P class="page-main" style="text-indent:1em">We learn how doctors work, how they take care of patients, how they do experiments and overcome difficulties. Doctors in China have really hard work to do every day. We have a better understanding of the disease which we deal with. Over 90% esophagus cancer happens in China and the survival rate is lower than 20%. We also visit labs and give questionnaires to the public. What we do makes us acquire the current situation of esophagus cancer and the life of doctors in China. These things really attracts us.</P><P class="page-question" style="text-indent:0.5em"><SPAN>What challenges has your team faced in developing and running our Human Practices activities?</SPAN></P><P class="page-main" style="text-indent:1em">At the beginning, the challenge for us is how we can visit the hospital. We don’t know any doctors and we are not sure whether the doctors want to communicate with us. So we contact with some medical students in The National Cancer Center at first. Thankfully, they’d like to help us and make an appointment with the doctors.</P><P class="page-main" style="text-indent:1em">During our visiting, it comes the biggest difficulty------the way we communicate to the patient. We want to know how they feel and we want to comfort them, but none of us know how to start to talk. We are afraid that we might not be polite to them or disturb them. At first we were a bit embarrassed when facing them. However, they were friendly and we had a good conversation.</P><P class="page-question" style="text-indent:0.5em"><SPAN>What support or resources can iGEM offer to help us improve our Human Practice work?</SPAN></P><P class="page-main" style="text-indent:1em">We hope that the iGEM foundation can propagandize more about iGEM competition and make more people know about it. It’ll help us easier to get access to the resources. Meanwhile, we hope that iGEM can help us propagandize our ideas. We’ll be really thankful!</P></DIV><DIV class="twelve columns page-content"><H1 class="page-title">Synthetic BiologyMeetsClinical Medicine</H1><P class="page-question" style="text-indent:0.5em"><SPAN>What kinds of people are more vulnerable to esophageal cancer?</SPAN></P><P class="page-main" style="text-indent:1em"> Nearly 90% of esophageal cancer patient are found in china. There’s also a featured geographical distribution of this disease. Most cases aggregate in area along the Taihang mountain, which covers provinces like Heibei, Heinan. There’re also some villages in Xinjiang and Sichuan where esophageal cancer is frequent. Through clinical observation, many esophageal cancer patients often drink really hot water or soup, and eat stimulating food. Drinking alcohol and smoking also contribute to high risk of esophageal cancer.</P><P class="page-question" style="text-indent:0.5em"><SPAN>What are the current therapeutic tools of esophageal cancer?</SPAN></P><P class="page-main" style="text-indent:1em"> Like most cancer, treatment of esophageal cancer mainly involves 3 methods, which are surgery, chemotherapy and radiotherapy. According to the patients’ situation, doctor will take one or a combination of these 3 methods. And the curative effect also varies with different patient.</P><P class="page-question" style="text-indent:0.5em"><SPAN>How about chances that esophageal cancer can be completely cured?</SPAN></P><P class="page-main" style="text-indent:1em">That depends. We usually divide cancer into 4 stages according to the extent of cancer cell proliferation. In Stage I, during which cancer cells haven’t transferred, the 5-year survival rate can reach 70%. When it comes to Stage II, this number dramatically drops under 50%. And for Stage III and IV, it is 20% and 10% respectively.</P><P class="page-question" style="text-indent:0.5em"><SPAN>Can esophageal cancer be easily diagnosed in early stage?</SPAN></P><P class="page-main" style="text-indent:1em">Currently there are two major approaches to test esophageal cancer. First one is CT (computed tomography), which can show the thicken area in the esophageal wall. Another one is doing gastroscope, which allows doctor to directly see the abnormity in esophageal and take sample for further examination. Though these two approaches have high accuracy, each of them has their own drawbacks. CT may bring radiation hazard. Doing gastroscope is a rather painful process, and it’s hard and expensive to book a painless gastroscopy. These drawbacks deter people from these checks when they don’t have any symptom. However, it’s usually too late when one can feel the lesion. </P><P class="page-question" style="text-indent:0.5em"><SPAN>According to your observation during work, which stage does most patients in when they realize the disease?</SPAN></P><P class="page-main" style="text-indent:1em">It varies with individual, but according to my observation, most patients are sent here in Stage III or IV. Because the standard test approaches, as I’ve said before, are expensive, time-consuming and painful, most people won’t do these tests unless they feel really uncomfortable. </P><P class="page-question" style="text-indent:0.5em"><SPAN>Are there any biomarkers for esophageal cancer that are currently in clinical application?</SPAN></P><P class="page-main" style="text-indent:1em">No. There are currently many researches focus on miRNA working as potential biomarkers for esophageal cancer. But none of them is fully demonstrated enough to be used clinically. The most popular research method is screening large number of miRNAs, measuring their expression level in subjects, and find possible relations. However, limited by time and expense, the sample capacity of these researches is not able to be large enough to represent the entirety. Thus the results of these researches have limitation. Even so, as statistics of each research accumulating, and human technology developing, biomarkers are a promising trend of disease detection.</P><P class="page-question" style="text-indent:0.5em"><SPAN>What do you think about our project?</SPAN></P><P class="page-main" style="text-indent:1em"> It’s a really interesting one. Your test paper can work as a preliminary screening of esophageal cancer high risk people, because it’s convenient and noninvasive. To do this, you need to ensure the sensitivity, not specificity, of the test paper. Though at present you may not be able to work out a test paper that can be used in clinically, it’s a very good idea, or concept. </P></DIV><DIV class="twelve columns page-content"><H1 class="page-title">The Result of Our Questionnaire</H1><P class="page-main" style="text-indent:1em"><SPAN>Q1: Our country(China) is one of the areas of high incidence of esophageal cancer in the world, there are more than ten million people died of esophageal cancer each year. Do you know the causes of esophageal cancer? (Nitrosamines and other chemical carcinogens, fungi, micronutrient deficiencies, poor diet, genetic factors, etc.)</SPAN></P><P class="page-main" style="text-indent:1em"><SPAN>Q2: Have you ever heard which poor eating habits are related to esophageal cancer? (Tobacco and alcohol addiction, swallowing hot or irritating esophagus excellent food, etc.)</SPAN></P><P class="page-main" style="text-indent:1em"><SPAN>Q3: Do you know how the hospital diagnose esophageal cancer? (Ultrasound, barium meal, endoscopy, etc.)</SPAN></P><P class="page-main" style="text-indent:1em"><SPAN>Q4: Do you know the symptoms of esophageal cancer? (Weight plummeted, vomiting, difficulty swallowing, indigestion, etc.)</SPAN></P><P class="page-main" style="text-indent:1em"><SPAN>Q5: If there is a strange feeling esophagus (such as dysphagia, pain, etc.), will you immediately go to the hospital to check it?</SPAN></P><P class="page-main" style="text-indent:1em"><SPAN>Q6: Which of the following is the main reason that you do not choose to go to the hospital to check?</SPAN></P><P class="page-main" style="text-indent:1em"><SPAN>Q7: How much would you like to pay for the test paper if it’s result is trustful?</SPAN></P><P class="page-main" style="text-indent:1em"><SPAN>Q8: Which of the following choice is the most acceptable one for you to diagnose cancer?</SPAN></P></DIV></DIV></DIV><DIV class="page-container pattern-2" id="collaboration"><DIV class="parallax-five"><DIV class="row"><DIV class="twelve columns parallax-container"><H1 class="parallax-title">COLLABORATION</H1><H2 class="parallax-subtitle">Sharing and collaboration are core values of iGEM.</H2></DIV></DIV></DIV><DIV class="row"><DIV class="twelve columns page-content"><H1 class="page-title">Communication &amp; Collabroation</H1><H2 class="page-subtitle2"><SPAN>Help Tianjin with the Modeling</SPAN></H2><P class="page-main" style="text-indent:1em">From July 24 to 26, team member from the Tianjin visited us in Beijing. During the three-day stay here, we discussed about our ideas, our projects and team and team management.</P><P class="page-main" style="text-indent:1em">Then we helped Tianjin-IGEM in their programming. In their modelling, they want to make the model visible by stimulating their processes using Matlab. So we help them abstract calculation process from real reactions.</P><P class="page-main" style="text-indent:1em">The basic method is Monte Carlo.</P><H2 class="page-subtitle2"><SPAN>Participate in CCIC</SPAN></H2><P class="page-main" style="text-indent:1em">During Aug 14th-15th, our team participated in CCIC(Conference of China iGEMer Committee) in PKU. We gave ‘peer’ review to other iGEM team and gained feedback from others. We enjoyed the process reaching out to other teams, asking for advice and talking about ideas. In the CCIC, the young, ambitious and creative synthetic biologist got together and had fun with each other.</P><H2 class="page-subtitle2"><SPAN>Try out Team Tsinghua's Prototype</SPAN></H2><P class="page-main" style="text-indent:1em">This is our feedback after trying out their product</P><P class="page-main" style="text-indent:1em">Tsinghua Team has made a very interesting device. It looks a mysterious, with two boxes, a 96-well plate and many wires. And it needs a computer when it works. After a short introduction I learned how to use it. Their software has a funny flash at the beginning and a friendly interface. There are two main functions their system can achieve. </P><P class="page-main" style="text-indent:1em">The first function of their device is very imaginative – their device can store information in bacteria. I can choose a small-sized file and then the program will encode the information into the bacteria in the 96-well plate. Since this process would take a relatively long time, I just loaded the file but didn’t wait for the cells to grow up. </P><P class="page-main" style="text-indent:1em">The second function allowed me to encode the information by myself. I can choose the time and the color of the light and change these parameters at different time. And for every well it can receive a different pattern of light input. Then, the 96-well plate will receive a set of light input, which may transfer the information to the bacteria. Besides, I think this light change can be used as a light inducer which may activate gene expression or change the state of a photosensitive protein. That is, we can conduct many different experiments in one 96-well plate. So, their device can be a useful tool not only in information encoding but also in many other fields, like synthetic biology or optogenetics. </P><P class="page-main" style="text-indent:1em">Although their device is an experimental model by now, I believe it is not hard to bring it into real utility. </P></DIV></DIV></DIV><DIV class="page-container pattern-2" id="team"><DIV class="parallax-seven"><DIV class="row"><DIV class="twelve columns parallax-container"><H1 class="parallax-title">Team&amp;Attribution</H1><H2 class="parallax-subtitle">...... </H2></DIV></DIV></DIV><DIV class="row"><DIV class="twelve columns page-content"><H1 class="page-title">Meet Our Amazing Team</H1><H2 class="page-subtitle"><SPAN>Instructors</SPAN></H2></DIV><DIV class="three columns padding-four-columns team-member mobile-two"><H3 class="our-team-title">Zhen Xie</H3><H3 class="our-team-subtitle">Professor</H3><P class="our-team-text">Advice on Wet Lab</P></DIV><DIV class="three columns padding-four-columns team-member mobile-two"><H3 class="our-team-title">Xiaowo Wang</H3><H3 class="our-team-subtitle">Professor</H3><P class="our-team-text">Advice on Dry Lab</P></DIV><DIV class="three columns padding-four-columns team-member mobile-two"><H3 class="our-team-title">Weixi Liao</H3><H3 class="our-team-subtitle">Phd Candidate</H3><P class="our-team-text">Advice on Wet Lab</P></DIV><DIV class="three columns padding-four-columns team-member mobile-two"><H3 class="our-team-title">Lei Wei</H3><H3 class="our-team-subtitle">Phd Candidate</H3><P class="our-team-text">Advice on Dry Lab</P></DIV><DIV class="three columns padding-four-columns team-member mobile-two"><H3 class="our-team-title">Zhanhao Peng</H3><H3 class="our-team-subtitle">Instructor on Dry Lab</H3><P class="our-team-text">Advice on Dry Lab</P></DIV><DIV class="three columns padding-four-columns team-member mobile-two"><H3 class="our-team-title">Dacheng Ma</H3><H3 class="our-team-subtitle">Instructor on Wet Lab</H3><P class="our-team-text">Advice on Wet Lab</P></DIV><H2 class="page-subtitle"><SPAN>Students</SPAN></H2><DIV class="three columns padding-four-columns team-member mobile-two"><H3 class="our-team-title">Yicong Du</H3><H3 class="our-team-subtitle">Biological Science</H3><P class="our-team-text">Team Leader/Wet Lab</P></DIV><DIV class="three columns padding-four-columns team-member mobile-two"><H3 class="our-team-title">Yunxuan Zhang</H3><H3 class="our-team-subtitle">Automation</H3><P class="our-team-text">Team Leader/Human Practice</P></DIV><DIV class="three columns padding-four-columns team-member mobile-two"><H3 class="our-team-title">Ruochi Zhang</H3><H3 class="our-team-subtitle">Automation</H3><P class="our-team-text">Modeling/Wiki Building</P></DIV><DIV class="three columns padding-four-columns team-member mobile-two"><H3 class="our-team-title">Shuya Li</H3><H3 class="our-team-subtitle">Biological Science</H3><P class="our-team-text">Wet Lab</P></DIV><DIV class="three columns padding-four-columns team-member mobile-two"><H3 class="our-team-title">Meixi Liu</H3><H3 class="our-team-subtitle">Biological Science</H3><P class="our-team-text">Wet Lab/Human Practice</P></DIV><DIV class="three columns padding-four-columns team-member mobile-two"><H3 class="our-team-title">Tianyi Sun</H3><H3 class="our-team-subtitle">Automation</H3><P class="our-team-text">Modeling</P></DIV></DIV><DIV class="footer-container"><DIV class="row"><DIV class="four columns first-column hide-for-small"><P class="footer-left">2015 iGEM</P></DIV><DIV class="four columns last-column mobile-four"><P class="footer-right"><A href="#
/user/AVAThemes/portfolio" target="_blank">Tsinghua-A</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>