<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Uppsala_Project_UnaG skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Uppsala/Project/UnaG</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="nav" class="container-fluid"><DIV class="row"><DIV class="col-lg-12"><UL class="nav nav-pills nav-justified"><LI role="presentation"><A href="https://2016.igem.org/Team:Uppsala">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:Uppsala/Description"> ★  Description </A></LI><LI><A href="https://2016.igem.org/Team:Uppsala/Project/Microfluidics">Microfluidics</A></LI><LI><A href="https://2016.igem.org/Team:Uppsala/Project/CRISPR">CRISPR/CPF1</A></LI><LI><A href="https://2016.igem.org/Team:Uppsala/Project/UnaG">UnaG</A></LI><LI><A href="https://2016.igem.org/Team:Uppsala/Design"> ★ Design </A></LI><LI><A href="https://2016.igem.org/Team:Uppsala/Model"> ★ Modeling </A></LI><LI><A href="https://2016.igem.org/Team:Uppsala/Experiments"> Protocols </A></LI><LI><A href="https://2016.igem.org/Team:Uppsala/Proof"> ★ Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:Uppsala/Demonstrate"> ★ Demonstrate </A></LI><LI><A href="https://2016.igem.org/Team:Uppsala/Interlab"> Interlab study </A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2016.igem.org/Team:Uppsala/Team">The Team </A></LI><LI><A href="https://2016.igem.org/Team:Uppsala/Collaborations">★ Collaborations </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Uppsala/Parts">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Uppsala/Basic_Part"> ★ Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:Uppsala/Composite_Part"> ★ Composite Parts </A></LI><LI><A href="https://2016.igem.org/Team:Uppsala/Part_Collection"> ★ Part Collection </A></LI><LI role="presentation"><A href="https://2016.igem.org/Team:Uppsala/Safety">Safety</A></LI><LI role="presentation"><A href="https://2016.igem.org/Team:Uppsala/Attributions"> ★ Attributions</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Uppsala/Human_Practices">Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Uppsala/HP/Silver"> ★ The Price of Transformation</A></LI><LI><A href="https://2016.igem.org/Team:Uppsala/HP/Gold"> ★ The Manual </A></LI><LI><A href="https://2016.igem.org/Team:Uppsala/Integrated_Practices"> ★ Integrated Practices </A></LI><LI><A href="https://2016.igem.org/Team:Uppsala/Engagement"> ★ Engagement </A></LI></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="accordion" id="myAccordion"><H2> UnaG </H2><DIV class="col-lg-6 col-lg-pull-6"><P>
                        UnaG is a fluorescent protein that was recently discovered in eel muscle (<I>Anguilla japonica</I>). Its small size and ability to work under hypoxic conditions make it stand out from other fluorescent proteins like GFP. These special abilities and the fact that it is a new discovered protein inspired us to work with UnaG.
                </P></DIV><DIV class="col-lg-5 col-lg-pull-6"><A data-toggle="collapse" data-target="#collapsible-1" data-parent="#myAccordion"><DIV class="mybutton text-center results panel-default"><DIV class="panel-body"> Introduction </DIV></DIV></A><A data-toggle="collapse" data-target="#collapsible-2" data-parent="#myAccordion"><DIV class="mybutton text-center results panel-default" class="collapsed"><DIV class="panel-body">Dig deeper into UnaG</DIV></DIV></A><A data-toggle="collapse" data-target="#collapsible-3" data-parent="#myAccordion"><DIV class="mybutton text-center results panel-default" class="collapsed"><DIV class="panel-body">Dig deeper into Bilirubin</DIV></DIV></A><A data-toggle="collapse" data-target="#collapsible-4" data-parent="#myAccordion"><DIV class="mybutton text-center results panel-default"><DIV class="panel-body"> Results</DIV></DIV></A></DIV><DIV class="panel"><DIV class="col-lg-8 collapse" id="collapsible-1"><H3> Go Green - Go UnaG! </H3><P> 
As part of this iGEM project, research has been done on the newly discovered green fluorescent protein called UnaG. This protein is interesting firstly because it is about half the size of other fluorescent proteins that are being used today, making it suitable as a fusion protein. Secondly, it fluoresces only when a bilirubin molecule is bound non-covalently to the protein. This feature could make UnaG suitable as an inducible marker by the addition and removal of bilirubin to cells. Moreover, unlike all other fluorescent proteins of today, UnaG does not need molecular oxygen in order to fluoresce, meaning that it can be used as a marker protein in research under hypoxic conditions. These cool features, alongside the fact that it is always exciting to work with new discoveries, made us want to add this protein to the iGEM toolbox! During the summer we created three different UnaG biobricks; one with a hexahistidine tag  suitable for affinity chromatography, one with the hexahistidine tag and a flexible linker of six amino acids suitable for fusing to other proteins, and lastly one biobrick with only the flexible linker attached to UnaG. 
 </P></DIV></DIV><DIV class="panel"><DIV class="col-lg-8 collapse" id="collapsible-2"><H3> UnaG - A Fluorescent Member of the FABP Family</H3><P> 
UnaG was discovered as late as 2013. It originates from the muscle fibers of the Japanese Unagi eel Anguilla japonica. This is the first fluorescent protein to be found in a vertebrate species, giving it a range of biochemical properties that differs from proteins found in the original FP family - one of the most astounding ones being that fluorescence occurs only when a ligand binds non-covalently to the protein, namely bilirubin. (I)
</P><H4>Structure of UnaG</H4><P>
UnaG is a fatty acid-binding protein (FABP). These proteins are cytosolic and exist in most vertebrates and invertebrates. They contribute to the transport of lipophilic compounds, often fatty acids. Although it is still rather evasive in which ways FABPs function in cells, some of these proteins appear to play roles in cellular anabolism. Wild-type UnaG is 139 amino acids long and has a molecular weight of 15.6 kDa. UnaG is a ß barrel structure containing eleven ß strands - a typical quality for FABPs. Instead of binding a fatty acid molecule, the ligand of UnaG is bilirubin, which binds non-covalently to the center of this ß barrel. Bilirubin exists in all vertebrates and is a metabolite of the heme group in hemoglobin. Bilirubin can only bind to UnaG in its unconjugated (4Z,15Z) form, meaning that wild-type UnaG shows high specificity to this particular molecule. In fact, BR is the actual fluorophore of UnaG. When BR binds to the cavity of the protein, it emits green light at 527 nm. Holo-UnaG has maximum absorption at 498 (i.e. blue light) nm. UnaG does not need molecular oxygen in order to fluoresce, making it suitable for research executed under anaerobic conditions where FPs are not functional. According to Akiko Kumagai and his team, the quantum yield of holo-UnaG is 0.51 and it is stable in a pH ranging from 4 to 11 (I). GFPs are usually stable in a pH range from 6-10, but this varies depending on which mutant is used. (II) 
</P><FIGURE><FIGCAPTION> Figure 1: Crystal Structure of UnaG at 1.2 Å. Source: PDB:4I3D. Kumagai et. al  (2013) Cell(Cambridge,Mass.) 153: 1602-1611.
</FIGCAPTION><H4>Using UnaG as a Fluorescent Marker</H4><P>
Apo-UnaG does not fluoresce, but Akiko Kumagai’s research group showed that Holo-UnaG (i.e. UnaG+BR) fluoresces instantaneously. No other fluorescent protein has a ligand that binds non-covalently to it. This could mean that it is possible to create an inducible fluorescent marker that could be used instead of photoswitchable FPs. Bilirubin does not exist naturally in lower classes of organisms such as bacteria and yeast, so it would have to be added exogenously when research is being done on these types of cells. UnaG has shown the ability of being fused to other proteins via a flexible linker both on the C terminus and the N terminus, making it suitable to use as a fusion marker. (I)
</P><P>
Since UnaG was discovered fairly recently, no mutagenic variations of the protein have been developed with improved characteristics than those of wtUnaG. However, several fluorescent FABPs have recently been discovered in other marine organisms. Two of them are called Chlopsid FPI and FPII. These two were also found in eels, although in another species, and both of them emit green fluorescence. Researchers have identified a tripeptide motif – Gly-Pro-Pro – conserved in all fluorescent FABPs. Chlopsid FPI and FPII are also bilirubin inducible. Alignment of the DNA sequences of UnaG and the Chlopsid proteins show 54% homology. There is a slight blue shift in the fluorescence (UnaG is 498/527 nm ex/em and Chlopsid FPI and FPII 489/523 nm ex/em). The researchers suggest that this blue shift stems from the fact that the 57th amino acid in UnaG is asparagine, whereas Chlopsid FPI and FPII contain a histidine at this site. The histidine would contribute to a more π-conjugated system, resulting in a difference in fluorescence. This discovery suggests possibilities of changing the emission wavelength of these proteins by mutagenesis comparableto what has been done on GFP for the past 40 years. This could in that case enable creation of fluorescent FABPs of different colors. (III)
 </P><H4>References</H4><SMALL>
(I) Kumagai, A. Ando, R. Miyatake, H. Greimel, P. Kobayashi, T. Hirabayashi, Y. Shimogori, T. Miyawaki, A. (2013). A Bilirubin-Inducible Fluorescent Protein from Eel Muscle. The Cell, Volume 153, Issue 7, 1602-1611


(II) Campbell, TN. Choy, FYM. (2001). The Effect of pH on Green Fluorescent Protein: a Breef Review. Molecular Biology Today (2001) 2(1): 1-4


(III) Gruber DF, Gaffney JP, Mehr S, DeSalle R, Sparks JS, Platisa J, et al. (2015) Adaptive Evolution of Eel Fluorescent Proteins from Fatty Acid Binding Proteins Produces Bright Fluorescence in the Marine Environment. PLoS ONE 10(11): e0140972. doi:10.1371/journal.pone.0140972 
</SMALL></FIGURE></DIV></DIV><DIV class="panel"><DIV class="col-lg-8 collapse" id="collapsible-3"><H3> Bilirubin </H3><P>
Bilirubin is a yellow pigment (Benioff Children’s Hospital, San Francisco) created by the enzymatic reaction of haemoglobin degradation.  Since the haemoglobin is located inside the red blood cells, the protein is hidden from the catabolism. It is only when the exchange into new red blood cells takes its place that the haemoglobin becomes exposed and decomposed by either fagocytosis or haemolysis. The result of the degradation are two toxic compounds; carbon monoxide (CO) and the bilirubin, were as it is only the carbon monoxide that is possible for the body to excrete easily (Mugnoli et al. 1983). The bilirubin is difficult to excrete since it is a non-polar compound and therefor not soluble in water (Fevery, J et al. 1977). The bilirubin molecules have a tendency to create different complexes with other proteins as well, such as serum albumin. The pigment also has the ability to conjugate with three types of proteins that are involved in the transport and the metabolism of bilirubin in vivo (Bonnett et al. 1978).
</P><DIV class="media"><DIV class="media-body"><P>
Bilirubin is an unstable compound (Le Bas, Allegret et al. 1980) and can occur both as bound and free pigment among the humans. The molecule has a tendency to form several derivate of bilirubin such as biliproteins, several conjugates and glucuronides etc (Harris, Kellermeyer, 1970). The schematic form of bilirubin (Figure 1B.) is almost never present (if so the molecule would have been directly polar and lipophilic) (Mugnoli et al. 1983). The more stable and dominant formation of bilirubin (Figure 1C.) is present when bilirubin is in liquid and solid constellations (Le Bas, Allegret, et al. 1980). In these forms the pigment creates several hydrogen bonds between the NH/O-groups and OH/-groups, which in turn makes the hydrophilic COOH- and NH-groups not available for polar interaction with their environment. Consequently the bilirubin becomes insoluble in water and methanol, but easy to dissolve in chloroform (Mugnoli et al. 1983). Even at the pH-values where the COOH-group is in an ionic state the formation of Figure 1C is predominant. These features also occur in well-known lipophilic compounds such as dioxins and polychlorated biphenyls (PCBs). Common properties for these compounds, among which bilirubin is included, is that they can diffuse through biological membranes in the placenta, blood brain barrier and hepatocytes. 
</P></DIV><DIV class="media-left"><FIGURE><FIGCAPTION> Figure 1B: Linear structure of bilirubin. (Antony F. Mcdonagh et al. 1985).
</FIGCAPTION><FIGURE><FIGCAPTION> Figure 1C: Crystalline and liquid structure of bilirubin. (Antony F. Mcdonagh et al. 1985).
</FIGCAPTION></FIGURE></FIGURE></DIV></DIV><P>
Bilirubin is naturally excreted in the liver by, with the help of enzymes, conjugating with the sugar glucoronic acid to create glucuronides. Since the glucoronic acid contains several COOH- and OH-groups the bilirubin conjugate becomes more hydrophilic than the bilirubin alone. Thereby it becomes possible for the bilirubin to leave the liver and go along with the bile, through the kidneys and then finally get excreted with the urine. 
</P><P>
One chain of glucuronic acid is enough to make the bilirubin excretable, but diglucoronides can also occur in the human bile. These contains two glucuronic acids per bilirubin molecule and the compound seems to simply be an effect of bilirubin’s three dimensional structure. No indications that the diglucuronides are more biological efficient have been noticed so far. The human being´s survival is dependent on the monoglucuronides, but not on the diglucuronides (Gordon et al. 1976).
</P><P>
Hyperbilirubinemia, also called Jaundice disease or icterus (www.mesh.kib.ki.se), occurs when either the formation of bilirubin glucuronides or the excretion of these does not work in the correct manner. Consequently the concentration of bilirubin and glucuronic acid accumulates in the body (Buchan, 1826). High levels of bilirubin are toxic and it can result in neurologic harm that causes deafness, delayed development and cerebral pares (Benioff Children’s Hospital, San Francisco). 
</P><H4>References</H4><H4>Literature</H4><SMALL>
Bonnett, R., Davies, J. E., Hursthouse, M. B., &amp; Sheldrick, G. M. (1978). The structure of bilirubin.Proceedings of the Royal Society of London. Series B, Biological Sciences, 202(1147), 249-268. 

Fevery, J., Van De Vijver, M., Michiels, R., &amp; Heirwegh, K. P. M. (1977). Comparison in different species of biliary bilirubin-IXα conjugates with the activities of hepatic and renal bilirubin-IXα-uridine diphosphate glycosyltransferases. Biochemical Journal, 164(3), 737-746.

Gordon, E. R., Goresky, C. A., Chang, T. H., &amp; Perlin, A. S. (1976). The isolation and characterization of bilirubin diglucuronide, the major bilirubin conjugate in dog and human bile. Biochemical Journal,155(3), 477-486.
Harris, J. W., &amp; Kellermeyer, R. W. (1970). The red cell

Le Bas, G., Allegret, A., Mauguen, Y., de Rango, C., &amp; Bailly, M. (1980). The structure of triclinic bilirubin chloroform–methanol solvate. Acta Crystallographica Section B Structural Crystallography and Crystal Chemistry, 36(12), 3007-3011. doi:10.1107/S0567740880010692

Mugnoli, A., Manitto, P., &amp; Monti, D. (1983). Structure of bilirubin IXα (isopropylammonium salt) chloroform solvate, C33H34N4O62−.2C3H10N+.2CHCl3.Acta Crystallographica Section C Crystal Structure Communications,39(9), 1287-1291. doi:10.1107/S0108270183008252
<H4>Websites</H4>
Karolinska institutet, https://mesh.kib.ki.se/term/D007567/jaundice-neonatal, Retreived 2016-08-11.

Benioff Children’s Hospital, San Francisco. https://www.ucsfbenioffchildrens.org/conditions/jaundice/, Retrieved 2016-08-11
<H4>Figures</H4>
1B, 1C: Antony F. Mcdonagh, PhD, and David A. Lightner, PhD 1985. Pediatrics- ”Like a Shrivelled Blood Orange”- Bilirubin, Jaundice and Phototeraphy.
</SMALL></DIV></DIV><DIV class="panel"><DIV class="col-lg-8 collapse" id="collapsible-4"><H3> Results </H3><H4 id="design">Designing the UnaG BioBricks</H4><P> In the original paper (Kumagai et al., 2013) the authors obtained the UnaG sequence from a cDNA library of unagi eel muscle. Since we lack the budget to go fishing in Japan, we ordered the nucleotide sequence from IDT. This not only gave the possibility to address codon bias, but through synonymous base substitutions to eliminate all restriction sites as per the iGEM BioBrick guidelines. An exception is an NsiI site in the N-terminal 6xHis tag, but a BioBrick without it was also made. In addition to the affinity tag, the original sequence was also modified to include C-terminal flexible linker (GSG)2 in case the protein would be part of a fusion construct. By default a double TAA stop-codon is present right before this flexible linker, which prevents it from expressing and thus potentially affecting protein function. This does not interfere with 3A assembly, but a variant of the BioBrick without the flexible linker also exists.</P><P>IDT sequenced the entire pUCIDT-Amp vector including the ordered insert as part of their quality control workflow, so their results were used as a reference when doing modification to the plasmid.
The IDT plasmid contains the lac-promoter so it is possible to do test expression directly with IPTG. However due to the way the final product is created, the UnaG sequence had a 50% chance to be incorporated in reverse, which was the case. Therefore suites of different BioBrick promoters were extracted from the 2016 distribution to test for expression. Those include a medium constitutive promoter (BBa_K608006), a strong constitutive promoter (BBa_K880005), IPTG inducible promoter (BBa_J04500), and a T7 promoter (BBa_K525998). All of those contain an RBS already assembled in order to speed up work. The UnaG sequence itself was excised from the IDT plasmid using EcoRI and PstI and ligated into pSB1C3 backbone cut with the same set of enzymes. </P><P>After successfully obtaining UnaG assembled with a variety of promoters, mutagenesis was performed in order to create several variants of the BioBrick for future use. The “stock” option contains the UnaG coding sequence. Upstream of it lies a 6xHis affinity tag, separated by an additional Serine amino acid to increase flexibility between the tag and the protein. In the registry it is annotated as BBa_K2003010. Note that this part is designed with RFC25 (Freiburg Standard) prefix and suffix in mind, hence it contains additional restriction sites, that do not affect normal 3A assembly but enable in-frame protein fusion without creating stop codons. Downstream of the part lies a short Glycine-rich flexible linker to minimize the effects of possible protein fusion. However the base part still contains the double TAA codons before this flexible linker.
The stop signal has been removed with PCR in BBa_K2003011. This is part retains all the features, including the RFC25 prefix and suffix, but now the flexible linker is properly expressed and in case of fusing to another CDS would not cause premature termination.</P><P>Finally, to avoid potential interference from the 6xHistidine affinity tag, BBa_K2003012 was created based on BBa_K2003011. In this part, the six amino acids were removed (again through PCR), as well as the Serine linker in between. This part has been designed for studying the properties of the protein in vivo, since the high positive charge of the affinity tag could interfere with its function or localization. At each step of the experiments, sequencing was performed using the iGEM standard VF2 and VR verification primers to ensure CDS integrity and especially to avoid introduced frameshift mutations.</P><H4 id="small expression">Small-scale Expression of UnaG</H4><P>In addition to mutagenesis experiments, small-scale expression tests were performed to verify that UnaG expresses under our laboratory conditions (Kumagai et.al (2013) purified and crystalized the protein from E.coli, so in theory it should be functional in prokaryotes). BioBricks BBa_K880005 and BBa_K2003010 were 3A assembled and grown overnight in LB. Bilirubin is not very soluble in water, so the stock 1mM solution was prepared in DMSO instead. It seems the molecule does not permeate the cell membrane easily, so appropriate amounts were added to crude cell extracts instead, to a final concentration of 100 µM. Those were obtained by replacing LB-medium with room temperature PBS pH 7.4 containing 1mg/ml lysozyme and breaking open the cells through incubation for 30 minutes and occasional mixing. To avoid unspecific fluorescence, cell debris were pelleted before the experiment and small aliquots of the supernatant were added to 200 µL thin-walled PCR tubes.</P><FIGURE><FIGCAPTION>Figure 1. Induction of fluorescence of UnaG by the addition of bilirubin. The results verify that UnaG does not fluoresce in absence of bilirubin. The samples are exposed to UV light instead of blue light, hence the fluorescence does not appear green.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 2. Fluorescence of UnaG under UV light compared to GFP and RFP. Annotations: 2_3F - UnaG under a strong constitutive promoter BBa_K880005  + bilirubin
1_5M - GFP under a medium constitutive promoter BBa_K608011 (control)
RFP - pSB1C3 standard backbone with red fluorescent protein insert (control) 
PBS - phosphate-buffered saline solution with lysozyme (control)
bilirubin - 1mM bilirubin solution in DMSO (control)</FIGCAPTION></FIGURE><P>Results are displayed in the figures:
 Abbreviations:
2_3F - UnaG under a strong constitutive promoter BBa_K880005  + bilirubin
1_5M - GFP under a medium constitutive promoter BBa_K608011 (control)
RFP - pSB1C3 standard backbone with red fluorescent protein insert (control) 
PBS - phosphate-buffered saline solution with lysozyme (control)
bilirubin - 1mM bilirubin solution in DMSO (control)</P><H4 id="large expression">Large-scale Expression of UnaG</H4><P>After observing the small-scale expression results, plans for large-scale purification were designed. Escherichia coli (E.coli) BL21DE3 cells were used in conjunction with T7 promoter-driven UnaG in TB medium to achieve peak levels of expression. T7 expression is induced using IPTG at 18°C for 16 hours. Breaking open the cells was performed using Qsonica Q700 titanium-tip sonicator since lysozyme has nearly the same size as UnaG and would make purification more difficult, as well as just slower and less efficient compared to sonication. It also breaks DNA, reducing overall lysate viscosity. After centrifugation at 4°C and 10 000 g for 1 hour, the supernatant was filtered through 0.2 µM pore filter and loaded to a Ni2+ affinity column, also called IMAC. </P><H4 id="IMAC">IMAC procedure</H4><FIGURE><FIGCAPTION>Figure 3. Setup for IMAC procedure.</FIGCAPTION></FIGURE><P>First the column had to be packed using Nickel Sepharose Fast Flow resin (GE Healthcare). The calculated column volume for this setup was 5 ml. Afterwards the column had to be charged with Ni2+ this is done with a ion binding buffer (50 mM Na+CH3COO-, 300 mM NaCl, pH4). The column was equilibrated by pumping through 5 column volumes (CV) with a flowrate of 0.7ml/min. After the column is equilibrated with ion binding buffer, the buffer it switched out with 0.3 M NiSO4 and a new equilibration step is made with 5 CV’s of ion solution. The final equilibration is done with 5CV ion binding buffer to wash away all of the excess Ni+2 ions. Three other buffers were prepared, having a base composition of 50 mM Na+CH3COO- and 300 mM NaCl. The binding buffer for UnaG contains 10 mM imidazole, the washing buffer (to remove the excess protein from the column) contains 20 mM imidazole and the elution buffer contains 350 mM imidazole. The column is equilibrated with 5 CV of binding buffer, afterward 25 ml of the binding buffer is mixed with the UnaG lysate and loaded onto the column. Afterwards 5 CV of wash buffer is pumped through to remove of all the excess protein still in the column. Finally the elution buffer is used and fractions are collected. </P><FIGURE><FIGCAPTION>Figure 4. Gel analysis from SDS-PAGE with fractions collected from IMAC. UnaG does not seem to be present in any of the fractions 1-7. 
Annotations:
L: PageRuler Prestained Protein Ladder, 10 to 180 kDa. 
1 -7: Fractions collected from IMAC</FIGCAPTION></FIGURE><P>All of the fractions were then tested for the target protein using SDS-PAGE. 
L - PageRuler Prestained Protein Ladder, 10 to 180 kDa;
1-7 Fractions collected from IMAC
Samples were boiled with 2x Laemmli Sample Buffer with DTT and 10 µL loaded on each start. The protein ladder was 5 µL, not boiled. After staining the gel with Coomassie Brilliant Blue and destaining, the band corresponding to UnaG’s expected size (15.6 kDa) was not observed anywhere. In addition, none of the fractions fluoresced when bilirubin was added.
Since sequencing confirmed that there are no mutations, and we previously managed to obtain impure functioning UnaG during the small-scale experiments of UnaG, the conclusion was that the IPTG solution was faulty and did not manage to induce the T7 promoter driven UnaG expression. </P><H4 id="references">References</H4><P>Kumagai, A., Ando, R., Miyatake, H., Greimel, P., Kobayashi, T., Hirabayashi, Y., Shimogori, T., and Miyawaki, A. (2013). A Bilirubin-Inducible Fluorescent Protein from Eel Muscle. Cell 153, 1602–1611.</P></DIV></DIV></DIV></DIV></DIV><FOOTER><DIV class="container-fluid"><DIV class="container"><DIV class="row"><DIV class="col-lg-2"><P><B>Contact</B><UL><LI><A href="http://igemuppsala.se/"> Uppsala iGEM association </A></LI><LI>  info@igemuppsala.se </LI></UL></P><A href="#nav">
                        To top ↑       
                </A></DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>