Team:Uppsala/Parts
Home
Project
★  Description
Microfluidics
CRISPR/CPF1
UnaG
★ Design
★ Modeling
Protocols
★ Proof of Concept
★ Demonstrate
Interlab study
Team
The Team
★ Collaborations
Parts
Parts
★ Basic Parts
★ Composite Parts
★ Part Collection
Safety
★ Attributions
Human Practices
Human Practices
★ The Price of Transformation
★ The Manual
★ Integrated Practices
★ Engagement
Parts
CRISPR Biobricks
During the summer we planned to build several biobricks. In first place we wanted to assemble a composite biobrick made of a pBAD/araC (BBa_I0500), a medium RBS (BBa_0032) our his-tagged CPF1 protein and a double terminator (BBa_0015). The arabinose induced promoter was chosen to avoid expression leakage since expression of endonucleases might lead to cell toxicity (Figure 1a). Secondly, we planned to build a pre-crRNA endowed of a double terminator (BBa_0015) that could be used to target any sequence the user wants to cleave (Figure 1b).
Figure 1: Assembly of the CRISPR biobricks
The spacer of this Biobrick was designed so that it could have been easily replaced with a simple digestion in order for future teams to design and synthesize only their 24 nucleotides long spacer sequence targeting the desired DNA, instead of the full CRISPR array. This pre-crRNA was designed in Full Length Repeat- Spacer- Full Length Repeat, where the repetitive regions were copied from F.tularensis subsp. Novicida 7 strain U112 (Genebank CP000439.1), and a random replaceable spacer sequence of 33 bp was inserted (33 base pairs were used because this was the length of the spacer introduced in between F.tularensis CPF1 CRISPR array. The spacer was endowed of the recognition sites for bsaI restriction enzymes. In this way, straggered cut with different overhangs occurs in the repetitive region, so that by designing the spacer with the correct overhangs, scarless insertion of any spacer into the array is possible (Figure 2). Designing of this sequence was inspired by the design used by Bikard et al. in 2013 for Cas9 CRISPR array plasmid.
Figure 2: Schematic representation of the replaceable spacer. Digestion with BsaI allow insertion of the desired spacer
UnaG Biobricks
As part of this year’s iGEM project, the newly discovered inducible green fluorescent protein UnaG was characterized. At a size of 15.6 kDa, it is about half the size of GFP. Hence, UnaG would be appropriate to use as a fusion protein. With this in mind, we designed three different UnaG BioBricks; UnaG with a 6xHis-tag (BBa_K2003010) for characterization of the protein, UnaG with a 6xHistag and a flexible linker (Gly-Ser-Gly)2 (BBa_K2003011), and lastly UnaG with only the flexible linker (BBa_K2003012).
The default UnaG BioBrick (BBa_K2003010) was designed and ordered from IDT with a hexahistidine tag on the 5’ end of the gene and a flexible linker (GSG)2 on the 3’ end. However, right upstream of the flexible linker, two stop codons were added, meaning that the flexible linker will not be translated from this gene. By PCR mutagenesis, these two stop codons were removed, resulting in BBa_K2003011 which contains both the flexible linker and the histidine tag. Lastly, PCR was performed in order to remove the 6xHis-tag from BBa_K2003011, resulting in a UnaG BioBrick with only the flexible linker.
Figure 3: Schematic view of the three different UnaG BioBricks. From the top left is; BBa_K2003011, BBa_K2003012, and BBa_K2003010, respectively.
Here are all the biobricks that we have submitted.
<groupparts>iGEM2016 Uppsala</groupparts>
Contact
Uppsala iGEM association
info@igemuppsala.se
To top ↑
