"
Page
Discussion
View source
History
teams
Log in
 
Team:EPF-Lausanne/Notebook
From 2008.igem.org
Home
The Team
The Project
Parts
2-step PCR
Microfluidics
Modeling
Notebook
Contents
1 Notebook
2 Protocols
2.1 Media and buffers
2.1.1 LB : Luria Browth medium
2.1.2 SOC
2.1.3 TE
2.1.4 SOB: Super Original Broth
2.1.5 CCMB80 buffer
2.1.6 Antibiotics
2.2 Cloning
2.2.1 Competent cells preparation
2.2.2 Transformation
2.2.3 Plasmid Purification
2.2.4 Ligation
2.3 MITOMI
2.3.1 Target DNA Protocol
2.3.2 2-Step PCR Reaction
2.3.3 In vitro transcription translation (ITT)
2.3.4 On chip experiment
2.4 Parts Characterization
2.5 Microfluidics
2.5.1 SU-8 wafers
2.5.2 AZ9260 wafers
2.5.3 Masks lithography
2.5.4 PDMS 2 layer device fab
3 Safety
4 References
Notebook
July
M
T
W
T
F
S
S
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
August
M
T
W
T
F
S
S
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
October
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
Protocols
Reference : Most of these protocols come from http://openwetware.org
Media and buffers
LB : Luria Browth medium
We used a dehydrated culture medium from Pronadisa containing :
Triptone 10.0 g/L
Yeast Extract 5.0 g/L
Sodium chloride 10.0 g/L
Dissolve 25g of the mix in 1L of distilled water. For plates pouring, add 15g agar for 1 L of solution. Shake and close, then autoclave it to sterilize.
SOC
Super Original Catabolite Repressor
Bactotryptone                         20g.
Bacto-yeast extract                   5g.
NaCl                                  0.5g.
1M KCl                                2.5ml
ddH2O                                 to 1000 ml
Total Volume                          1000ml
Adjust pH to 7, with 10N NaOH. Autoclave to sterilize Add 20 ml of 1M glucose before use.
TE
10xTE for 1 liter from stock solutions 10 ml 1M Tris-HCl pH 8.0 2 ml 0.5M EDTA pH 8.0 988 ml ddH2O
→ 10xTE is 10 mM Tris-HCl and 1 mM EDTA
For the Tris-HCl use Tris base and adjust to desired pH using HCl.
SOB: Super Original Broth
Used in growing bacteria for preparing chemically competent cells Ingredients
0.5% (w/v) yeast extract
2% (w/v) tryptone
10 mM NaCl
2.5 mM KCl
20 mM MgSO4
Per liter:
5 g yeast extract
20 g tryptone
0.584 g NaCl
0.186 g KCl
2.4 g MgSO4
!!! Adjust to pH 7.5 prior to use. This requires approximately 25 ml of 1M NaOH per liter.
CCMB80 buffer
For 1L
10 mM KOAc pH 7.0 (10 ml of a 1M stock/L)
80 mM CaCl2.2H2O (11.8 g/L)
20 mM MnCl2.4H2O (4.0 g/L)
10 mM MgCl2.6H2O (2.0 g/L)
10% glycerol (100 ml/L)
adjust pH DOWN to 6.4 with 0.1N HCl if necessary
adjusting pH up will precipitate manganese dioxide from Mn containing solutions.
sterile filter and store at 4°C
slight dark precipitate appears not to affect its function
Antibiotics
For each antibiotic, solutions of 5 ml have been done and stored at -20°C (09.07.08)
Cloning
Competent cells preparation
We followed the protocol coming from OpenWetWare .
Transformation
Thaw competent cells on ice for 30 minutes.
Mix cells and DNA, put back on ice for 20 minutes.
Heat shock: 45 seconds at 42° C.
Add 200 μl of LB broth (without antibiotics).
Incubate at 37° C, 230 rpm for 1 hour.
Plate the cell culture on agar containing the correct antibiotic.
Incubate at 37° C overnight.
Plasmid Purification
We use the QIAprep Miniprep Spin Kit .
To get a higher DNA concentration, use only 30 ml Elution Buffer.
Ligation
Preparation:
Digest insert and vector.
Example:
700 ng DNA
1 μl NEB Buffer 2 (or depending on enzymes used)
1 μl BSA 10X Buffer
0.1 μl Enzyme 1
0.1 μl Enzyme 2
Complete with water to final volume of 10 μl.
Incubate at 37° C for 1h30 - 2h.
If DNA concentration is low, complete to e.g. 20 μl instead and adjust Buffer volumes accordingly.
After digestion, treat the vector with Antarctic Phosphatase (1 μl AP, 1 μl AP Buffer) to prevent autoligation. Incubate for 20 minutes at 37° C.
Run digested sample on an agarose gel.
Purify the fragments of interest using a Gel Extraction Kit.
Ligation:
For correct reaction calculations, follow DNA Ligation Protocol on OWW
The following simplified protocol worked for inserts up to a couple of hundreds basepairs:
6 μl Insert
1.5 μl Vector
1 μl Ligase Buffer with ATP
0.5 μl T4 DNA Ligase
1 μl Water
Autoligation Control: Prepare the same reaction again but replace Insert with water.
Incubate 30 minutes at room temperature.
MITOMI
Target DNA Protocol
Materials:
5’Comp Cy5 primer
5’Cy5-GTC ATA CCG CCG GA
order from IDT at 1umole, HPLC purified
suspend to 500uM
Library primers:
See spreadsheet for details
5’ Linker - BINDING SITE – 3’ Linker – TCCGGCGGTATGAC
5’ Linker generally: AAC
Binding Site Generally a 12mer but can be longer
3’Linker generally: C
order from IDT in a 96 well v-bottom plate suspended at 150uM in TE
Klenow Fragment 3’-5’ exo-
Order from NEB (Cat #: M0212L)
dNTPs at 10mM each
order from Roche (PCR Nucleotide Mix: 11581295001)
Method:
Synthesis reaction: 3uL dNTP 3uL Buffer 2 2uL Library primer 0.4uL 5’Comp Cy5 primer 1uL Klenow 3’-5’ exo- 20.6uL dH2O 30uL Final Volume
Prepare a master mix without the library primer, and load 28uL into a 96 well plate. Add 2uL of Library primer to the 96 well plate. Cycle as follows:
37°C for 1 hour
75°C for 20 mins
ramp down to 30°C at 0.1°C/sec
hold at 4°C
After the synthesis add 70uL of 0.5% BSA in dH2O to each well and transfer to a 384 well plate (flat bottom). Prepare dilutions as needed using 0.5% BSA dH2O.
To set up the 384 load 30uL 0.5% BSA dH2O into all columns except 1, 7, 13, 19.
Transfer the column 1 of the klenow reaction (100uL total) to column 1 row A (A1) on the 384 well plate. Perform 5 dilutions of 70uL sample into the adjacent wells containing 30uL BSA dH2O. After the 5th dilution (in column 6) discard the remaining 70uL.
Transfer column 2 of the klenow plate into A7 and perform dilutions.
Column3 -> A13, column 4 -> A19, Column 5->B1, col 6-> B7,  col7->B13, col8 -> B19
You should have 30uL of target DNA in each well of the 384 plate now. Add 30uL of 0.5% BSA dH2O to each well.
The samples are now ready to be spotted.
2-Step PCR Reaction
Purpose: generate linear PCR templates for in vitro transcription/translation of cDNA clones using E.coli colonies as starting material. Should also be able to amplify genomic targets.
Template: E.coli colonies picked from agar plates  Primer stock: 5’ + 3’ gene specific primers each at 500uM  Primers pair: 5’+3’ gene specific primers 1/10 in dH2O to 50uM each  Polymerase: HiFi Plus (Roche)
suspend colony in 2.5uL lyse ‘n go buffer (Pierce)
optionally 2.5uL of  an overnight LB culture can also be used (adjust dH2O accordingly)
heat to 95°C for 7 minutes then cool to 4°C
add 1uL primer pair and 46.5 uL PCR Mix
PCR Mix: 1uL dNTP 10uL 5x Buffer + MgCl2 .5uL HiFi DNA polymerase 35uL dH2O 46.5uL Final Volume
Temp. cycles: 1 94°C 4:00 2 94°C 0:30 3 55°C 1:00 4  72°C  2:00 5 Cycle 2-4 30x  6 72°C 7:00 7 4°C forever
run 1uL of product on gel to check for presence and length
optionally: run a Qiagen PCR purification
2nd step of PCR
5’ext1 and 3’ext2 each at 50uM
Primer Mix: 5’ext1 + 3’ext2 primers diluted 1/200 in dH2O to  250nM each
Template = previous PCR mix
1uL dNTP 10uL  5x Buffer + MgCl2 1uL  Primer Mix 0.5uL  template 0.5uL  HiFi Plus DNAp 37uL  dH2O 50uL  Final Volume
Temp. cycles: 1 94°C 4:00 2 94°C 0:30 3 55°C 1:00 4 72°C 2:00 5 Cycle 2-4 10x  6 72°C 7:00 7 4°C forever
5’Final and 3’Final at 500uM
Final Primer Mix: 5’Final and 3’Final diluted 1/10 in dH2O
add 1uL of Final Primer Mix
Temp. cycles: 1 94°C 4:00 2 94°C 0:30 3 50°C 1:00 4 72°C 2:00 5 Cycle 2-4 25-30x  6 72°C 7:00 7 4°C forever
check 1uL product on a 1% agarose gel
no further purification is necessary even though PCR purification is recommended, eluting in 50uL of elution buffer (EB) or pH adjusted dH2O
In vitro transcription translation (ITT)
Linear PCR templates are transcribed and translated using the RTS 100 E.coli HY Kit (ROCHE) Master Mix:
12uL/sample E.coli lysate
10uL/sample Reaction mix
12uL/sample Amino Acid
1uL/sample Methionine
5uL/sample reconstitution buffer
6uL ddH2O
2uL Linear PCR template
2uL GFP labelled tRNA
Incubate for 4-6h at 30°C
On chip experiment
Material:
MITOMI device bonded to epoxy slide with spotted target DNA array
set of pressure valve, tubing, manometer and pressure gauge
Fluorescence scanner
ITT product (transcription factor)
synthetic AHLs solution (stock 1mM)
Method:
Load control lines with dH2O at 5psi
Run 2mg/ml BSA-biotin at 5psi for 20min, chambers closed
Run 500ug/ml NA/PBS for 20min
Run PBS for 5min
Close button, continue PBS for 1-2min
Run 2mg/ml BSA-biotin for 20min
Run PBS for 10min
Run 1/500 penta-His Antibody in PBS for 5min
Open button, continue antibody for 20min
Run PBS for 2min
Run ITT product with corresponding AHL for 20min (50ul ITT, 10ul of 1:100 diluted AHL stock)
Close sandwich, open chamber, incubate at RT for 90min
Close button, close chamber, open sandwich and run PBS for 5min
Scan for red and green fluorescence
Parts Characterization
BBa_K092600 Characterization:
Three 5 ml cultures of LB medium and antibiotic (ampicillin, 20 µg/ml) were inoculated with single colonies from a glycerol stock stored at –80°C.
Cultures were grown in 17 mm test tubes for 15 hrs at 37°C with shaking at 70 rpm.
Cultures were diluted 1:1000 into 5 ml of fresh medium and grown to an OD600 of 0.2 under the same conditions as before.
Twenty-four 100 µl aliquots of each of the cultures were transferred into a flat-bottomed 96 well plate (BD-96).
Tetracycline stock solution was prepared at a concentration of 6 mg/ml.
Varying quantities of tetracycline was added to each well to yield 14 different final concentrations. Three replicate wells were measured for each concentration of tetracycline. Three wells were each filled with 200 µl of culture medium to measure the background.
Plate was left in incubator for 4 hours at 37°C at 60rpm.
The plate was incubated in a multi-well fluorimeter (Perkin Elmer) at 37°C and assayed with a protocol of fluorescene measurements at 530/580 excitation/emission wavelength filters.
The transfer function fluorescence intensity after four hours of expression. Error bars representing the 95% confidence interval in the population for the independent samples.
Flow cytometry was carried out with samples to check for presence of RFP producing cells.
Microfluidics
SU-8 wafers
Material:
100mm silicon wafer
SU-8 negative photoresist - GM1060 from Gersteltech
Method:
Oxygen plasma
dispose GM1060 on wafer and spin coat at 2000 rpm for 40s.
Soft-bake for 5min at 130°C with a slow temperature increase/decrease (ramp of 4C°/min).
Expose using corresponding mask (Control mask) for 16s. at 275mW.
Post-exposure bake for 30min at 100°C with a slow temperature increase/decrease (ramp of 4C°/min).
Develop in PGMEA bath for 2x5min and rinse with 2-propanol
Air dry and inspect section thickness with optical microscope
AZ9260 wafers
Material:
100mm silicon wafer
AZ9260 positive photoresist - MicroChemicals GmbH
Method:
Oxygen plasma
Automatized coating process using EVG150 - resist thickness: 8um
Expose for 2x15s at 274mW using corresponding mask (inverted control mask)
Automatized developing process using EVG150
Hard-bake for 20min at 160°C
Inspect section thickness with optical microscope
Masks lithography
Material:
5" square masks
Chip design implemented in Clewin (.gdsi files)
Method:
Convert Clewin files (positive resist needs to be inverted)
Setup LASER patterning - Use 4mm head
Develop photoresist
Chr etch for 110s, quick rinse and ultra clean rinse
Clean for 15min in 1165 remover hot bath and for 15min in 1165 clean bath, quick rinse and ultra clean rinse
Air dry and inspect with optical microscope
PDMS 2 layer device fab
Materials:
flow and control layer molds
Sylgard 184 part A and part B
TMCS
Method:
Place molds into a TMCS vapor chamber
Control layer mixture: 30g Part A + 6g Part B
Mix for 1 minute, degas for 2 minutes
pour onto control layer mold and place mold in vacuum chamber
Flow layer mixture: 30g Part A + 1.5g Part B
Mix for 1 minute and degas for 2 minutes
Spin coat onto flow layer at 2600-3000rpm for 30secs
Remove control layer mold from vacuum chamber, making sure no bubbles are left on the surface.
Place the control and flow layer in a 80C convection oven and incubate for 30 minutes
Remove casts from oven, cut out control layer, punch holes, and align to flow layer
Put aligned device back into 80C oven and incubate for at least 90 minutes.
Remove devices from oven and punch flow layer holes
Safety
In accordance to the guidelines of the iGEM committee, we address here the question of safety in our project. Since our project relies solely on expression in bacteria of quorum sensing molecules and fluorescent markers, it does not pose any particular threats to humans, animals or the environment, other than the ones which are encountered in standard laboratory work with E.coli. Therefore, we estimate the risk to researchers, the public or environment as very low. However, we would like to underline the particular attachment which our institution, EPFL, holds for the safety of all parties concerned by research conducted in its facilities.
As referenced on this web page , our school has Biosafety Officers, whose charges are to supervise activities linked to pathogens or genetically modified organisms. They also verify that good laboratory pratice and safety measures are respected. We have not contacted our Biosafety officer for the duration of our tasks, as they did not present any risks. If necessary iGEM teams would be able to collaborate with such safety organisations in the institution.
References
Anderson, J. C., Voigt, C. A., and Arkin, A. P., Environmental signal  integration by a modular AND gate. Molecular Systems Biology 3 (2007).
Basu, S. et al., A synthetic multicellular system for programmed pattern  formation. Nature 434 (7037), 1130 (2005).
Brenner, K., Karig, D. K., Weiss, R., and Arnold, F. H., Engineered  bidirectional communication mediates a consensus in a microbial biofilm  consortium. Proceedings of the National Academy of Sciences of the  United States of America 104 (44), 17300 (2007).
Camilli, A. and Bassler, B. L., Bacterial small-molecule signaling pathways.  Science 311 (5764), 1113 (2006).
Fuqua, C., Parsek, M. R., and Greenberg, E. P., Regulation of gene  expression by cell-to-cell communication: acyl-homoserine lactone  quorum sensing. Annu Rev Genet 35, 439 (2001).
Waters, C. M. and Bassler, B. L., Quorum sensing: cell-to-cell  communication in bacteria. Annu Rev Cell Dev Biol 21, 319 (2005).
You, L. C., Cox, R. S., Weiss, R., and Arnold, F. H., Programmed population  control by cell-cell communication and regulated killing. Nature 428  (6985), 868 (2004).
Home
The Team
The Project
Parts
2-step PCR
Microfluidics
Modeling
Notebook
Retrieved from " http://2008.igem.org/Team:EPF-Lausanne/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
