"
Page
Discussion
View source
History
teams
Log in
 
Team:UCL/Science/Proto
From 2014.igem.org
Goodbye Azodye UCL iGEM 2014
iGEM
HOME
PROJECT
SCIENCE
HUMANS
FAQ
UCL iGEM
Project
Science
Humans
FAQ
-.
Competent cells
Digestion
Ligation
Agar Plates
Transformation
Agarose Electrophoresis
Colony Boil
PCR
Growth Inhibition Assay
ispB Growth Assay
ispB Inoculation Assay
BsDyP Degradation Assay
Creating competent cells
Materials
LB Media, 50ml Falcon Tubes, Ice, Chilled centrifuge, Calcium Chloride (CaCl2), Eppendorf tubes (300ul/tube)
Procedure
1. Inoculate a single colony into 5ml Lb in 50ml falcon tube. Grown O/N @ 37oC
2. Use 1ml to inoculate 100ml of LB in 250ml bottle the next morning.
Shake @ 37oC for 1.5-3 hours.
Or
1. Inoculate a single colony into 25ml LB in a 250ml bottle in the morning
2. Shake @ 37oC for 4-6 hours.
Then…
3. Put the cells on ice for 10mins (keep cold from now on).
4. Collect the cells by centrifugation in the big centrifuge for 3 minutes @ 6Krpm.
5. Decant supernatant and gently resuspend on 10ml cold 0.1M CaCl (cells sensitive to mechanical disruption).
6. Incubate on ice x 20 minutes
7. Centrifuge as in 2.
8. Discard supernatant and gently resuspend on 5ml cold 0.1M CaCl/15%Glycerol.
9. Dispense in microtubes (300ųl/tube). Freeze at -80oC.
Note: Home-made competent cells were used to transform registry BioBricks
Digestion
Materials
MilliQ water, NEBuffer 2.1, BSA, Pipettes and autoclaved tips  Enzymes: EcoRI-HF, SpeI, XbaI and PstI,
Notes
The volume of DNA to be digested will depend on the concentration obtained from miniprep of the DNA from culture. This must be made up to 500ng for each 50ul digestion
eg. if concentration BBa_R0010 is available at a concentration of 330ng/ul
we require 500/330 = 1.5ul of the DNA solution for a digestion
All digestions carried out for iGEM purposes i.e. BioBrick creation are double digests. NEBuffer 2.1 is the ideal buffer for all combinations used.
Depending on whether the part being digested is an upstream insert, downstream insert or plasmid backbone, the combinations of enzymes differ.
For Upstream parts, such as promoters digest with EcoRI-HF and SpeI
Digest downstream parts with XbaI and PstI
Digest plasmid backbones with EcoRI-HF and PstI
Procedure
1. Calculate amount of DNA solution that will be needed for 500ng
2. Following this, calculate the amount of water needed to make up to 50ul if both enzymes and BSA are 1ul each and Buffer is 5ul.
3. Add the components to a microcentrifuge tube in the order: Water > Buffer > BSA > DNA > Enzymes
4. Leave to incubate at 37ºC for 1-2 hours
5. Remove and deactivate the enzymes by incubating at 80ºC for 20 minutes
Ligation
Materials
MilliQ water, Ice, Pipettes and autoclaved tips, NEB T4 DNA Ligase Buffer, T4 DNA Ligase
Notes
Try not to repeatedly freeze and thaw the buffer by making 10-20ul aliquots and using them instead of the entire stock solution each time
All ligations should be done with an aim to be transformed.
No-insert controls must be plated onto plates with antibiotics corresponding to the backbone resistance and no-vector controls on antibiotic plates corresponding to resistance of original host vector
Procedure
1. Calculate water needed to total volume of 20ul where enzyme added will be 1 ul and all other components will be 2ul
- Usually this will include buffer, the upstream part digestion, downstream part digestion and backbone digestion.
2. Do no-insert and no-vector control ligations for each backbone and insert
3. Add components in the order: Water > Buffer > DNA upstream part > DNA downstream part > Backbone DNA > Ligase
4. Leave for 10 minutes at room temperature approx. 25ºC
5. Deactivate at 80ºC for 20 minutes
6. Transform products of ligation into competent cells
Making LB Agar Plates
Materials
LB Agar powder, plates, antibiotics, sterile jar, sterile flame or biosafety cabinet.
Procedure
1. Dissolve LB agar in sterile/autoclaved jar of sterile water 40g/L.
2. Swirl until completely dissolved
3. Autoclave LB agar
The following steps should be completed under a sterile flame or biosafety cabinet. Ideally after autoclaving, before LB Agar solidifies:
4. Add antibiotics from their stock solutions in 1:1000 ratio with LB agar, can be added either directly to each individual plate or to the entire solution.
5. Pour agar into plates, roughly half way (30-50mL per plate). Leave plates with lids half on (prevent condensation).
6. Allow plates to solidify under flame
7. Cover and leave on bench or in fridge
Plasmid DNA Transformation
Materials
Competent Cells, Plasmid DNA, Antibiotic Plates
Procedure
1.  Thaw competent cells on ice
2.  50uL cells enough for 1 transformation
3.  Add 1ug of DNA to 50uL competent cells
If BioBrick from distribution, resuspend DNA well in 10uL ddH20
4.  Add 2-5uL DNA i.e. ligation product or BioBrick re-suspension to 50uL competent cells
5.  Flick by hand or pipette up and down gently
6.  Place cells on ice for 30 minutes
7.  Place cells in water bath or heat block at 42oC for 30 seconds
8.  Place cells on ice for 2 minutes
9. Add 950ul of SOC or LB media and place in a shaking incubator for a maximum of 2 hours (37oC/250rpm) 10. Label two petri dishes with LB agar and the appropriate antibiotics(s) with the date, plasmid backbone and ligated genes/parts and antibiotic added.
You may also choose to write down the volume of transformed product plated
11. Spin down the incubated cells for 2mins at 4000rpm to obtain a cell pellet.
12. Pipette 100ul of the supernatant or fresh LB media before discarding the rest of the supernatant and re-suspending the pellet in the 100ul of media
13. Incubate the plates at 37ºC for 12-14 hours, always place plates lid-down.
If incubated for too long the antibiotics start to break down and un-transformed cells will begin to grow. This is especially true for ampicillin - because the resistance enzyme is excreted by the bacteria, and inactivates the antibiotic outside of the bacteria
You can pick a single colony, make a glycerol stock, grow up a cell culture and miniprep.
Count the colonies on the 20 μl control plate and calculate your competent cell efficiency.
Agarose Gel Electrophoresis
Prepare Agarose Gel (1%) for Electrphoresis
Materials:
Agarose powder, spatula, weighing boat, microwave, Ethidium Bromide, Gel dock,
1. Add ingredients to flask and microwave for about 120 seconds. Shake. Repeat until mixture fully dissolved.
2.Leave to cool for 20-30 seconds.
3.Add 20ul of Ethidium Bromide.
Place comb in gel frame. When flask is cool enough to handle, pour the solution into the gel frame and leave to set (10-20 minutes).
Preparing for Loading:
1. Prepare Ladder
a. Add 1ul of DNA ladder to labelled Eppendorf.
b. Add 1ul of DNA loading buffer (dye) to Eppendorfs containing DNA and DNA ladder (Unless ladder is already combined with dye).
c. Add 1ul of milliQwater
d. Spin down.
2. Prepare samples:
a. Add 10ul of digest (250ng of DNA) to 3ul of loading buffer.
Loading the gels
Prepare Ladder
1. Place gel frame in electrophoresis box with black (cathode) part away from you  and red (anode) part closest to you.
2. Carefully remove comb from gel.
3. Fill the top and bottom wells with 1x TAE. Pour over top of gel, continue pouring until gel is covered.
4. Load the wells starting with the DNA ladder in lane 1. Be sure to note which lane each sample is loaded into.
5. Run gel for  approximately 60 minutes at constant voltage of 120V. Check for bubbles at the cathode which shows it has started.
Visualising the gels
1. Remove the frame from the electrophoresis box and pour off any liquid. Wipe the bottom of the frame with tissue.
2. Place in GelDoc 2000, press Epi white to position gel, press Epi UV to visualise.
3. On the computer open program Quantity One. Select scanner, GelDoc XR, select manual acquire, adjust value until gel is visible.
4. Print or save to USB.
Visualising the gels
TAE from the gel run can be re-used, pour back into bottle.
Ensure all ethidium bromide rubbish is disposed of in ethidium bromide bin.
Colony Boil
Colony boil can be used to isolate genes present in an organism. We used a colony boil to isolate AzoR from e.coli using primers originally designed for PCR.
1. Using a sterile pipette tip carefully lift a single colony from any most recently transformed E.coli plate.
Repeat for more colonies from different strains if available.
2. Point the tip into 250ul of distilled/MilliQ water in an eppendorf, MUST BE screw-cap, and shake a bit. Withdraw and discard tip.
3.Place the tube in a 100͒C water bath for 5 minutes.
4. Remove and dry the outside of the tube. Shake the contents of the tube to the bottom. Use 2ul of the boiled cell solution for PCR with genomic DNA primers for any genes of interest that are present in the genome such as IspB
Standard PCR Protocol
Although our PCR protocol evolved throughout the project this protocol represents the core procedure we used throughout the project.
1. Make stock solution primer mix
Make 1/10 dilution of forward and reverse primer mix with TE buffer solution or Milli-Q water (e.g 10uL forward primer, 10uL reverse primer and 180uL TE buffer). Label with primer names, concentration, date and name of user.
2. Plan the PCR
V1 x C1=V2 x C2 where V= volume in uL, C=concentration in uM for each of the primers.
Use this calculation to calculate the volumes of forward and reverse primers to be used in the PCR.
Reagents for 1 x PCR:
10X Buffer - 5.0ul
MgCl2 - 1.0ul
dNTP - 1.0ul
Forward Primer - 2.5ul
Reverse Primer - 2.5ul
DNA Template - 1.0ul
Taq Polymerase - 0.5ul
H2O (Make up to total vol) - 36.5ul
Total = 50.0ul
3. PCR Program:
Stages
NEB Phusion DNA Polymerase
(5X Phusion HF Buffer)
Roche Taq DNA Polymerase
PROMEGA Pfu DNA Polymerase
Hot start (oC) / time
98oC / 30s
94oC / 2min
95oC / 1-2min
Cycles(#)
25-35
25-30
25-35
Denaturation (oC) / time
98oc / 5-10s
94oC / 15-30s
95oC  / 30s-1min
Annealing (oC) / time
45-72oC / 10-30s
55 to 65oC / 30-60s
42-65oC / 30s
Elongation (oC) / time
72oC / 15-30s per kb
72 OR 68oC / 45s - 2min
72-74oC / 2-4min
Final Extension (oC) / time
72oC / 5-10min
72 OR 68oC / 7min
72-74oC / 5min
Hold temp (oC) / time
4 to 10oC / infidenlity
4oC / infidelity
4oC
AZO DYE TOXICITY ASSAY (GROWTH)
Materials
LB Media, 50mL Falcon tubes, E.coli glycerol stock, conical flask, Azo dyes, Eppendorf tubes
Prepare azo dye stocks
The powders used were 55% for Reactive Black 5 and 85% for Acid Orange 7 azo dye and were dissolved in sterile Milli-Q water.
Azo Dye
Low Concentration
Medium Concentration
High Concentration
Reactive Black 5
0.05 mg/mL
0.5 mg/mL
5 mg/mL
Acid Orange 7
0.0155 mg/mL
0.155 mg/mL
1.55 mg/mL
Prepare Bacterial Solution
1.Prepare stock bacterial solution of 90mL LB in 180uL of bacterial glycerol stock
2. Make aliquots of 10mL into 8 falcon tubes
RB5 Zero
RB5 Low (0.05 mg/mL)
RB5 Medium (0.5 mg/mL)
RB5 High (5 mg/mL)
AO7 Zero
AO7 Low (0.0155 mg/mL)
AO7 Medium (0.155 mg/mL)
3. Place tubes in incubator at 37oC overnight at 250rpm. 4. Next morning add dyes to tubes 5. Record OD data at 600nm
Take two samples of 0.5 mL from each tube for the measurements, and place in a cuvette to be analysed in spectrophotometer
13 hours post inoculation– 2x0.5
14 hours post inoculation– 2x0.5
15 hours post inoculation– 2x0.5
16 hours post inoculation– 2x0.5
18 hours post inoculation– 2x0.5
6. Plot OD data
LEC + ispB Growth Assay
Materials:
LB Media, 50mL Falcon tubes, E.coli glycerol stock, conical flask, Azo dyes, Eppendorf tubes
1. Prepare bacterial solutions:
Plasmid-free:
Add 10mL of LB. Inoculate with 20uL of glycerol stock
Add 25uL of sterile water.
LEC-ispB and LEC-ispB + IPTG:
Add 25mL to a Falcon tube.
Add 25uL of 25ng/uL Chloramphenicol.
Inoculate with 50uL of ispB Glycerol stock.
Distribute 10mL into each of the two tubes
2. Grow overnight at 37oC 250rpm.
3. Add add IPTG (1mM) to tubes (ZERO) and (LEC-ispB + IPTG)
Add 10uL of IPTG (1M) to 10 mL of tube for a final concentration of 1mM.
Add 10uL of sterile water to ispB tube.
4. Record OD data at 600nm
Take two samples of 0.5 mL from each tube for the measurements, and place in a cuvette to be analysed in spectrophotometer
13 hours post inoculation– 2x0.5
14 hours post inoculation– 2x0.5
15 hours post inoculation– 2x0.5
16 hours post inoculation– 2x0.5
18 hours post inoculation– 2x0.5
5. Plot OD data
ispB Inoculation Assay
Materials: LB media, M9+thymine, M9+thymine+CASaa, IPTG(1M), Chloramphenicol, 50ml Falcon tubes, E.coli glycerol stocks, ispB glycerol stocks, LEC + ispB glycerol stocks.
1. Make LB and M9 solutions specified below:
ispB
35mL LB
60uL glycerol stock
50uL Chloramphenicol 25ng/uL
LEC + ispB - 1
35mL LB
60uL glycerol stock
35ul Chloramphenicol (25ng/uL)
LEC + ispB - 2
35mL LB
60uL glycerol stock
35ul Chloramphenicol (25ng/uL)
LEC + ispB - 3
35mL LB
60uL glycerol stock
35ul Chloramphenicol (25ng/uL)
2. Distribute 10mL aliquots of bacterial solutions into falcon tubes
3. Inoculate dyes, adding 100uL of corresponding stock dye solutions and 100uL of sterile water for the controls.
4. Add 10uL of 1M IPTG.
5. Record OD data at 680nm
0 hours post inoculation– 2x0.5
2 hours post inoculation– 2x0.5
4 hours post inoculation– 2x0.5
6 hours post inoculation– 2x0.5
8 hours post inoculation– 2x0.5
6. plot OD data.
LEC + BsDyP DEGRADATION ASSAY
Materials:
Autoclaved Media (LB), 50ml Falcon tubes, Pipettes (10ul - 5ml), Pipette Tips (10ul - 5ml), BsDyP glycerol stock (T2_80_5, 9/10/14 gel), Conical flask, Azo Dyes
1. Prepare Azo dye stock solutions:
Azo Dye
V-Low concentration
Low concentration
Medium concentration
Reactive Black 5 (55%)
0.005 mg/mL
0.05 mg/mL
0.5
mg/mL
Acid Orange 7
(85%)
0.00155
mg/mL
0.0155
mg/mL
0.155 mg/mL
* Reactive Black 5
Medium concentration stock, a 1 in 10 dilution of the high concentration stock (50 mg/mL) 20uL of the High concentration stock must be added to 180 uL of water in an Eppendorf tube Add 100 uL of solution to the corresponding Falcon tube to give a final concentration of 0.5mg/mL Low concentration stock, a 1 in 10 dilution of the medium concentration stock (5 mg/mL) 20uL of the Medium concentration stock must be added to 180 uL of water in an Eppendorf tube Add 100 uL of solution to the corresponding Falcon tube to give a final concentration of 0.05 mg/mL Very Low concentration stock, a 1 in 10 dilution of the low concentration stock (0.5 mg/mL) 20uL of the Low concentration stock must be added to 180 uL of water in an Eppendorf tube Add 100 uL of solution to the corresponding Falcon tube to give a final concentration of 0.005 mg/mL
Acid orange 7
Medium concentration stock, a 1 in 10 dilution of the high concentration stock (15.5mg/mL)
20uL of the High concentration stock must be added to 180 uL of water in an Eppendorf tube
Add 100 uL of solution to the corresponding Falcon tube to give a final concentration of 0.155 mg/mL
Low concentration stock, a 1 in 10 dilution of the medium concentration stock (1.55 mg/mL)
20uL of the Medium concentration stock must be added to 180 uL of water in an Eppendorf tube
Add 100 uL of solution to the corresponding Falcon tube to give a final concentration of 0.0155 mg/mL
Very Low concentration stock, a 1 in 10 dilution of the low concentration stock (0.155mg/mL)
20uL of the Low concentration stock must be added to 180 uL of water in an Eppendorf tube
Add 100 uL of solution to the corresponding Falcon tube to give a final concentration of 0.00155 mg/mL
Prepare bacterial stock solution
LB 80 mL
Glycerol stock 160 uL
Chloramphenicol 80 uL
IPTG (1M) 80 uL
Distribute into 7 Falcon tubes, with no chemical, labelled
RB5 Plasmid-free Growth
RB5 V-LOW (0.005 mg/mL) Growth
RB5 LOW (0.05 mg/mL) Growth
* RB5 MEDIUM (0.5 mg/mL) Growth
RB5 ZERO LOW (0.05mg/mL)
AO7 V-LOW (0.00155 mg/mL) Growth
AO7 LOW (0.0155 mg/mL) Growth
AO7 MEDIUM (0.155 mg/mL) Growth
​
Place tubes in incubator at 37 ºC at 250 rpm. (8pm 13/10/14)
Discard remains of conical flask and sterilise
The next morning, at 9 am, add dyes to tubes
Record OD data.
RB5: 600nm
AO7: 480nm
Blank with LB at 600nm, do measurements of ZERO and RB5. Then blank with LB at 480, and do measurements of ZERO (again) and RB5.
With the ZERO control sample, measure at 600nm then measure RB5 samples. Then measure the control again at 480nm, and record samples for AO7.
Record OD data once for no-cell controls (only dyes)
Take two samples of 200uL from each tube for the measurements, place in a cuvette, perform 1 in 10 dilution (1.8mL of LB, that is, two pippetings of 0.9mL) and record value.
* 14 hours post inoculation– 2x0.5
* 15 hours post inoculation– 2x0.5
* 16 hours post inoculation– 2x0.5
* 17 hours post inoculation– 2x0.5
* 18 hours post inoculation– 2x0.5
Plot OD data
Contact Us
University College London
Gower Street - London
WC1E 6BT
Biochemical Engineering Department
Phone: +44 (0)20 7679 2000
Email: ucligem2014@gmail.com
Sponsors
Follow Us
Tweets by @UCLiGEM
Retrieved from " http://2014.igem.org/Team:UCL/Science/Proto "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
