<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UCL_Humans_Collab"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UCL/Humans/Collab">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UCL/Humans/Collab&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UCL/Humans/Collab&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UCL/Humans/Collab&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UCL/Humans/Collab" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UCL/Humans/Collab</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Goodbye Azodye UCL iGEM 2014</TITLE></P><DIV id="nav-container" class="" style="top: 0px; margin-top: 0px; height: 70px;"><NAV id="nav-main" style="overflow: visible; position: fixed;"><DIV id="nav-inner"><UL class="inline-block-list"><LI id="logo"><A href="/" id="iGEM">iGEM
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL" id="HOME">HOME
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL/Project" id="PROJECT">PROJECT
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL/Science" id="SCIENCE">SCIENCE
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL/Humans" id="HUMANS">HUMANS
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL/FAQ" id="FAQ">FAQ
							</A></LI></UL></DIV></NAV></DIV><DIV id="sexyheader"><DIV id="layout"><DIV id="menu"><DIV class="pure-menu pure-menu-open"><A class="pure-menu-heading" href="https://2014.igem.org/Team:UCL">UCL iGEM</A><UL><LI class="rezmenu"><A href="https://2014.igem.org/Team:UCL/Project">Project</A></LI><LI class="rezmenu"><A href="https://2014.igem.org/Team:UCL/Science">Science</A></LI><LI class="rezmenu"><A href="https://2014.igem.org/Team:UCL/Humans">Humans</A></LI><LI class="rezmenu"><A href="https://2014.igem.org/Team:UCL/FAQ">FAQ</A></LI>
-.
							</UL></DIV></DIV></DIV></DIV><DIV id="bodyContent"><DIV class="textArena"><DIV class="collapse-card"><DIV class="title"><STRONG><B>Edinburgh 2014 iGEM Team: RewirED</B></STRONG></DIV><DIV class="body"><H4><CENTER>How it all began: Making sense of antisense together!</CENTER></H4><P1>The <A href="https://2014.igem.org/Team:Edinburgh">RewirED Edinburgh Team</A> focused on the creation of a metabolic wiring system as a novel way of connecting logic gates in different bacterial strains. They developed <A href="https://2014.igem.org/Team:Edinburgh/modelling/software">a software tool to analyze sequences of antisense RNA for gene silencing</A>  which identifies the optimal sequence (~100bp, covering RBS and start codon) and analyses the structure to find the most stable antisense RNA.  
In this collaboration they provided the sequence of the antisense gene which, according to their model, has the fewest secondary structures in the core regions and analysed the behaviour of our design of an antisense gene.


From our side we provided real world data  on the behaviour of the antisense gene silencing  in order to test the accuracy of their model and efficacy of their software. Specifically we analysed the <A href="https://2014.igem.org/Team:UCL/Science/Results/Xeno#Xeno">growth</A> in different media of <I>E. coli</I> engineered with the <A href="http://parts.igem.org/Part:BBa_K1336006"> antisense gene silencing BioBrick </A>. The silenced gene is core for the survival of E. coli and the reduction in growth corresponds to the efficacy of the antisense. We sent them all the data we gathered that they could then compare to their <I>in silico</I> prediction.
The sequence we designed didn't effectively repress growth in <I>E.coli</I> as modelled by the software. We designed new primers  to amplify the sequence suggested by Edinburgh: a smaller fragment with better predicted functionality, and we are now in the stage of cloning and testing it to provide them with more data on their software's effectiveness. 
</P1>
Identification of the optimal antisense RNA for ispB silencing

Analysis of our antisense RNA design for ispB silencing
</DIV></DIV><DIV class="collapse-card"><DIV class="title"><STRONG><B>Instituto de Tecnologia Química e Biológica, Universidade Nova de Lisboa</B></STRONG></DIV><DIV class="body"><P1>The <A href="http://met.itqb.unl.pt/">Microbial &amp; Enzyme Technology Lab</A> led by Dr Lígia O. Martins at the Instituto de Tecnologia Química e Biológica, Universidade Nova de Lisboa, is one of the world leaders in bioremediation with microorganisms and enzymes. Their paper titled 'Synergistic action of azoreductase and laccase leads to maximal decolourization and detoxification of model dye-containing wastewaters' <A href="http://www.ncbi.nlm.nih.gov/pubmed/21890348">[1]</A> was the fundamental inspiration for our Goodbye AzoDye project. We are truly grateful for their initial support and guidance, and for sending us the following plasmids for our experiments:<B>pAzoR (pLP-1)</B> containing the FMN-dependent NADH-azoreductase 1 gene. <A href="http://www.ncbi.nlm.nih.gov/pubmed/21655981">[2]</A><B>pCotA (pLOM10)</B> containing the Spore Coat Protein Laccase gene. <A href="http://www.ncbi.nlm.nih.gov/pubmed/11884407">[3]</A><B>p1B6</B> containing the mutant FMN-dependent NADH-azoreductase 1 gene <A href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Improving+kinetic+PpAzoR">[4]</A><B>pBsDyp (pRC-2)</B> containing the Dye Decolourising Peroxidase BSU38260 gene. <A href="http://www.ncbi.nlm.nih.gov/pubmed/23820555">[5]</A><B>pPpDyp (pRC-1)</B> containing the Dye Decolourising Peroxidase PP_3248 gene.<A href="http://www.ncbi.nlm.nih.gov/pubmed/23820555">[5]</A></P1></DIV></DIV><DIV class="collapse-card"><DIV class="title"><STRONG><B>Central St Martins</B></STRONG></DIV><DIV class="body"><P>We approached the Central St Martins textiles department with our ideas of synthetic biology and science and they asked ‘When does technology like this become accessible?’ This question yielded a set of beautiful visualisation of the way our bacteria could be used to create art if controlled by light.  These pieces by second year Textiles Design BA students Cameo Bondy and Barbara Czepiel exhibit the textiles that could be created if our bacteria contained optogenetic biobricks that switched their dye breakdown capacities on and off via light cues. </P></DIV></DIV><DIV class="collapse-card"><DIV class="title"><STRONG><B>Natsai Audrey</B></STRONG></DIV><DIV class="body"><P>A practicing independent designer and researcher, Natsai Audrey Chieza is a Design Futurist inspired by material innovation and technology. Natsai considers her creative pursuits with a strong interest in how the life sciences can enable new craft processes for a more robust environmental paradigm.</P><P>Natsai contributed a series of pieces to be displayed at the #UncolourMeCurious from her Faber Futures exhibition, exploring the use of bacteria to create pigments and dye fabrics, deviating from the standardisation of a petri dish.</P><P>Natsai has achieved measurable success in design research projects for Microsoft, Nissan, Unilever and EDF Energy. She has also exhibited in numerous design exhibitions and events across Europe including the Victoria &amp; Albert Muesum, London; Audax Textile Museum, Tilburg; Salone Internazionale del Mobile di Milano, Milan; Designersblock LDF, London; EN VIE/ ALIVE, Paris; Science Gallery, Dublin; and Heimtextil, Frankfurt.</P></DIV></DIV><DIV class="collapse-card"><DIV class="title"><STRONG><B>London University Collaborations</B></STRONG></DIV><DIV class="body"><TABLE style="width:100%"><TBODY><TR><TD><B>Westminster:</B> This year the UCL iGEM Bioprocess Team paid a visit to Godfrey Kyazze, a Lecturer in Bioprocess Technology at University of Westminster. He is involved in water science research, using microbial fuel cells to produce electricity upon the degradation of azo dyes by bacterial cells. We greatly appreciated the opportunity to speak to him about his research, go into the lab and see some real examples of fuel cell modules. Through the visit, the team has certainly gained a valuable perspective on the potential application of azo dye degradation, not only for environmental remediation, but also for the production of energy.
<B>King's College London:</B> We took in Bez Karkaria, a KCL student, as a fully integrated member of our team, in the hope that he will spread the skills and knowledge he has acquired alongside us to KCL, and form an iGEM team there next year.
<B>Birkbeck:</B> We met with the nascent Birkbeck iGEM team, who plan to enter the competition next year, and advised them on how to construct their project, how to conduct public engagement and policy work and what to focus on in the wet and dry labs.
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="collapse-card"><DIV class="title"><STRONG><B>Linden Gledhill</B></STRONG></DIV><DIV class="body">
Linden Gledhill is a scientist inspired to art. Like us, he uses the tools of science, in his case advanced microscopy and high speed equipment to break down the physical work at different spatial and temporal scales. We noticed his pictures of azo dye crystals on <A href="http://www.flickr.com/photos/13084997@N03/8330858341/"> flickr </A> and contacted him over a potential collaboration. After contacting him over azo dye photos, discussing possible experiments we could carry out (LCMS, azo dye auxotrophy) and ending our conversation on discussing zombies and a ferrofluidic attack on humanity, we got him on board to help us. He kindly offered us to use his pictures and videos of azo dyes crystallisation both in our website and exhibition. IN addition to the possible ways of using his images  he gave us advice on how to make them ourselves, with the dyes we were decolourizing in the lab. We exposed a video at out UncolourmeCurious event using some footage he sent us of the crystallisation. Unfortunately we weren't able to use the pictures on the website due to the mandatory Creative Common policy of the iGEM website.
It was nevertheless very interesting to have some perspectives from a scientist and artist on the project, hearing how he moved from science to art and was then able to display the beauty of the microscopic work.

</DIV></DIV><DIV class="collapse-card"><DIV class="title"><STRONG><B>The Slade</B></STRONG></DIV><DIV class="body"><P>We wanted to reach out even further into the Arts community, to get people thinking about dyes and how they are used. We collaborated with The Slade School of Fine Art and with them unthreaded the chemical history of the dye industry. We constructed a timeline that peered back through time at the ancient and pre-industrial uses of pigments, to the rise of the azo dye, to help contextualise the Slade's installation in the UCL Wilkins Building, and highlight the terrific importance of azo dyes to the way we use colour. </P></DIV></DIV></DIV></DIV><FOOTER style="margin-bottom: -31px;"><DIV class="wrapper clearfix"><DIV class="FooterContent"><DIV id="FC11" style="font-size:120%"><H3 class="fontfixer" style="margin-top: 5px;">Contact Us</H3><P class="fontfixer">University College London  Gower Street - London  WC1E 6BTBiochemical Engineering Department<STRONG>Phone:</STRONG><A href="tel:phonenum" class="tele">+44 (0)20 7679 2000</A><STRONG>Email:</STRONG><A href="mailto:email@domain.com">ucligem2014@gmail.com</A></P></DIV><DIV id="FC22"><H3 class="fontfixer" style="margin-top: 5px;"><A href="https://2014.igem.org/Team:UCL/Humans/Sponsor" style="color: black;">Sponsors</A></H3></DIV><DIV id="FC33"><H3 class="fontfixer" style="margin-top: 5px;">Follow Us</H3><A class="twitter-timeline" href="https://twitter.com/UCLiGEM" data-widget-id="494172163977392128" height="235px" width="300px">Tweets by @UCLiGEM</A></DIV></DIV></DIV></FOOTER><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:UCL/Humans/Collab">http://2014.igem.org/Team:UCL/Humans/Collab</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UCL/Humans/Collab" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UCL/Humans/Collab" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UCL/Humans/Collab&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UCL/Humans/Collab&amp;oldid=396529" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>