<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UCL_Science_Primers"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UCL/Science/Primers">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UCL/Science/Primers&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UCL/Science/Primers&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UCL/Science/Primers&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UCL/Science/Primers" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UCL/Science/Primers</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Goodbye Azodye UCL iGEM 2014</TITLE></P><DIV id="nav-container" class="" style="top: 0px; margin-top: 0px; height: 70px;"><NAV id="nav-main" style="overflow: visible; position: fixed;"><DIV id="nav-inner"><UL class="inline-block-list"><LI id="logo"><A href="/" id="iGEM">iGEM
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL" id="HOME">HOME
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL/Project" id="PROJECT">PROJECT
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL/Science" id="SCIENCE">SCIENCE
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL/Humans" id="HUMANS">HUMANS
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL/FAQ" id="FAQ">FAQ
							</A></LI></UL></DIV></NAV></DIV><DIV id="sexyheader"><DIV id="layout"><DIV id="menu"><DIV class="pure-menu pure-menu-open"><A class="pure-menu-heading" href="https://2014.igem.org/Team:UCL">UCL iGEM</A><UL><LI class="rezmenu"><A href="https://2014.igem.org/Team:UCL/Project">Project</A></LI><LI class="rezmenu"><A href="https://2014.igem.org/Team:UCL/Science">Science</A></LI><LI class="rezmenu"><A href="https://2014.igem.org/Team:UCL/Humans">Humans</A></LI><LI class="rezmenu"><A href="https://2014.igem.org/Team:UCL/FAQ">FAQ</A></LI>
-.
							</UL></DIV></DIV></DIV></DIV><DIV class="textArena"><DIV class="textTitle"><H3 class="widthCorrect">List of Primer Designs for Creation of BioBrick Parts</H3></DIV><OL><LI><A href="/Team:UCL/Science/Primers#Enzyme01">Enzyme 01: AzoR, Azoreductase</A></LI><LI><A href="/Team:UCL/Science/Primers#Enzyme02">Enzyme 02: 1B6, Azoreductase heat-stable mutant</A></LI><LI><A href="/Team:UCL/Science/Primers#Enzyme03">Enzyme 03: CotA</A></LI><LI><A href="/Team:UCL/Science/Primers#Enzyme04">Enzyme 04: BsDyP, <EM>Bacilus subtilis</EM> dye-decolourising peroxidase</A></LI><LI><A href="/Team:UCL/Science/Primers#Enzyme05">Enzyme 05: PpDyP, <EM>Pseudomonas putida</EM> dye-decolourising peroxidase</A></LI><LI><A href="/Team:UCL/Science/Primers#Enzymet06">Enzyme 06: LiP, Lignin peroxidase / Ligninase</A></LI></OL><H1>UNEDITED, WILL BE BACK</H1><H4>Extraction of <I>Bacillus subtilis</I> genomic DNA</H4><DIV class="byline"> Adam Denyer, Tanel Ozdemir   <ABBR class="published" title="June 13, 2014">June 13, 2014</ABBR><STRONG>  Protocols  </STRONG><A href="/Team:UCL/Science/Protocols"><SPAN class="label label-warning">DNA extraction</SPAN></A></DIV><P>Our literature search identified a number of bacterial species that have been proven to degrade azo dye compounds including <I>B. subtilis</I> and <I>P. aeruginosa</I>.  We were able to obtain a <I>B. subtilis</I> strain for use in our project from ?.  We extracted the genomic DNA from this strain using a Promega Wizard Genomic DNA extraction kit so that we could subsequently amplify the azo-reducatase gene (AzoR1) and create our first azo-reductase BioBrick.  After completing the genomic DNA extracton we ran a gel to show that we had successfully extracted the <I>B. subtilis</I> genomic DNA.</P><H4>Transforming <I>E. coli</I> with Azo-reductase plasmids</H4><DIV class="byline"> Adam Denyer   <ABBR class="published" title="Monday, October 15, 2013,  8:21 PM">October 15, 2013</ABBR><STRONG>  Protocols  </STRONG><A href="/Team:UCL/Science/Protocols"><SPAN class="label label-warning">PCR</SPAN></A><A href="/Team:UCL/Science/Protocols"><SPAN class="label label-warning">competent cells</SPAN></A><A href="/Team:UCL/Science/Protocols"><SPAN class="label label-warning">transformation</SPAN></A><A href="/Team:UCL/Science/Protocols"><SPAN class="label label-warning">miniprep</SPAN></A></DIV><P>We were gratefully provided with a set of five plasmids from a group of researchers working at the University of Lisbon, Portugal who are researching how azo-dye degrading enzymes function and who were keen to collaborate with us.  These plasmids contained a number of genes encoding azo-dye degrading enzymes from both <I>B. subtilis</I> and <I>P. putida</I>  including mutated forms found to exhibit enhanced degradation activity.  As the DNA concentration of the plasmids we were sent was insufficient to perform PCR amplification on we transformed each of these plasmids into our <I>E. coli</I> NEB5alpha competent cells.  After growing the cells overnight we then mini-prepped each of them to obtain plasmids at sufficient concentrations for future experimental work.</P><TABLE class="table table-striped table-bordered"><THEAD><TR><TH> Name </TH><TH> Function </TH><TH> Source </TH><TH> Concentration </TH><TH> Sequence </TH><TH> Initial Plasmid / Vector </TH><TH> Comments </TH></TR></THEAD><TBODY><TR><TD> pAzoR </TD><TD> FMN-dependent NADH-azoreductase 1 </TD><TD><EM>Pseudomonas putida</EM></TD><TD> Miniprep, 
        48 ng/uL, 
      </TD><TD><A href=" http://www.ncbi.nlm.nih.gov/nuccore/26986745?report=fasta&amp;from=3267527&amp;to=3268138">597 bp <STRONG>[Check! Not 612 bp?]</STRONG></A></TD><TD> Expression vector <A href="http://www.addgene.org/browse/sequence_vdb/2549/ ">pET-21a (+) </A><A href=" http://biochem.web.utah.edu/hill/links/pET21.pdf">(ampicillin resistant (ampR))</A>, initially cloned between <EM>NdeI</EM> and <EM>BamHI</EM>.</TD><TD><A href="http://www.ncbi.nlm.nih.gov/pubmed/21655981">Plasmid provided by Lisbon</A></TD></TR><TR><TD> p1B6 (AzoR 1B6) </TD><TD> Mutant: Heat-stable; FMN-dependent NADH-azoreductase 1 </TD><TD><EM>Pseudomonas putida</EM></TD><TD> Miniprep,
        68 ng/uL, 
      </TD><TD><A href="">597 bp <STRONG>[Check! Not 612 bp?]</STRONG></A></TD><TD> Expression vector <A href="http://www.addgene.org/browse/sequence_vdb/2549/ ">pET-21a (+) </A><A href=" http://biochem.web.utah.edu/hill/links/pET21.pdf">(ampR)</A>, initially cloned between <EM>NdeI</EM> and <EM>BamHI</EM>. </TD><TD><A href=" http://www.ncbi.nlm.nih.gov/pubmed/24475252">Plasmid provided by Lisbon</A>.</TD></TR><TR><TD> pCotA </TD><TD> Spore Coat Protein Laccase </TD><TD><EM>Bacillus subtilis</EM></TD><TD> Miniprep, 
        103 ng/uL
      </TD><TD><A href="">1733 bp <STRONG>[Check! Not 1539 bp?]</STRONG></A></TD><TD> Expression vector <A href="http://www.addgene.org/browse/sequence_vdb/2549/ ">pET-21a (+) </A><A href=" http://biochem.web.utah.edu/hill/links/pET21.pdf">(ampR)</A>, initially cloned between <EM>NheI</EM> and <EM>BamHI</EM>. </TD><TD><A href=" http://www.itqb.unl.pt/martins/index_files/JBC2002.pdf">Plasmid provided by Lisbon</A>. </TD></TR><TR><TD> pBsDyP </TD><TD> Dye Decolourising Peroxidase BSU38260 </TD><TD><EM>Bacillus subtilis</EM></TD><TD> Miniprep, 
        51 ng/uL, 
      </TD><TD><A href="">1251 bp</A></TD><TD> Expression vector <A href="http://www.addgene.org/browse/sequence_vdb/2549/ ">pET-21a (+) </A><A href=" http://biochem.web.utah.edu/hill/links/pET21.pdf">(ampR)</A>, initially cloned between <EM>NdeI</EM> and <EM>BamHI</EM>. </TD><TD><A href="http://www.ncbi.nlm.nih.gov/pubmed/23820555">Plasmid provided by Lisbon</A>.</TD></TR><TR><TD> pPpDyP </TD><TD> Dye Decolourising Peroxidase PP_3248 </TD><TD><EM>Pseudomonas putida</EM></TD><TD> Miniprep, 
         55 ng/uL</TD><TD><A href="">861 bp <STRONG>[Check! Not 864 bp?]</STRONG></A></TD><TD> Expression vector <A href="http://www.addgene.org/browse/sequence_vdb/2549/ ">pET-21a (+) </A><A href=" http://biochem.web.utah.edu/hill/links/pET21.pdf">(ampR)</A>, initially cloned between <EM>NdeI</EM> and <EM>BamHI</EM>. </TD><TD><A href="  http://www.ncbi.nlm.nih.gov/pubmed/23820555">Plasmid provided by Lisbon</A>.</TD></TR></TBODY></TABLE><H4>Diagnostic digest of azo-reductase plasmids</H4><DIV class="byline"> Adam Denyer   <ABBR class="published" title="Monday, October 15, 2013,  8:21 PM">October 15, 2013</ABBR><STRONG>  Protocols  </STRONG><A href="/Team:UCL/Science/Protocols"><SPAN class="label label-warning">digest</SPAN></A><A href="/Team:UCL/Science/Protocols"><SPAN class="label label-warning">gel</SPAN></A></DIV><P>After successfully transforming these plasmids into competent <I>E. coli</I> NEB5alpha cells we then performed a diagnostic digest and gel electrophoresis experiment to ascertain that these plasmids contained the gene we expected.  Each plasmid was digested using two restriction enzymes chosen to digest DNA as specific points on the plasmids and create fragments of known length which we could then confirm using gel electrophoresis.</P><H4>Creation of azo-reductase BioBrick parts from plasmids</H4><DIV class="byline"> Adam Denyer   <ABBR class="published" title="Monday, October 15, 2013,  8:21 PM">October 15, 2013</ABBR><STRONG>  Protocols  </STRONG><A href="/Team:UCL/Science/Protocols"><SPAN class="label label-warning">competent cells</SPAN></A><A href="/Team:UCL/Science/Protocols"><SPAN class="label label-warning">transformation</SPAN></A><A href="/Team:UCL/Science/Protocols"><SPAN class="label label-warning">miniprep</SPAN></A></DIV><P>senectus et netus et malesuada</P><H4>Diagnostic digest of azo-reductase BioBrick parts</H4><DIV class="byline"> Adam Denyer   <ABBR class="published" title="Monday, October 15, 2013,  8:21 PM">October 15, 2013</ABBR><STRONG>  Protocols  </STRONG><A href="/Team:UCL/Science/Protocols"><SPAN class="label label-warning">digest</SPAN></A><A href="/Team:UCL/Science/Protocols"><SPAN class="label label-warning">gel</SPAN></A></DIV><P>senectus et netus et malesuada</P><H4>Extraction of useful BioBrick plasmids from iGEM 2014 Distribution Kit</H4><DIV class="byline"> Adam Denyer   <ABBR class="published" title="Monday, October 15, 2013,  8:21 PM">October 15, 2013</ABBR>  <STRONG>Protocols  </STRONG><A href="/Team:UCL/Science/Protocols"><SPAN class="label label-warning">competent cells</SPAN></A><A href="/Team:UCL/Science/Protocols"><SPAN class="label label-warning">transformation</SPAN></A><A href="/Team:UCL/Science/Protocols"><SPAN class="label label-warning">miniprep</SPAN></A><A href="/Team:UCL/Science/Protocols"><SPAN class="label label-warning">digest</SPAN></A><A href="/Team:UCL/Science/Protocols"><SPAN class="label label-warning">gel</SPAN></A></DIV><P>We began our project by identifying a range of BioBrick parts present in the iGEM 2014 distribution kit which we required as part of our cloning strategy.  These parts primarily consisted of both constituitive and inducible promoter systems with ribosome binding sites which we could then use in conjunction with our azo-reductase BioBricks to assemble a functional azo dye degrading gene.  We also decided that we would use the Red Florescent Protein expresing BioBrick as a control for any further transformation experiments.   As the level of DNA present within each plate of the distribution kit is insufficient to perform digest and ligation reactions on it was necessary to transform each of these plasmids into our NEB5alpha competent cells.  After growing our transformed cells overnight we then mini-prepped each of them to obtain BioBrick plasmids at suitable concentrations for future experiments.</P><H4>Assembling azo-reductase BioBrick Device(s)</H4><DIV class="byline"> Adam Denyer   <ABBR class="published" title="Monday, October 15, 2013,  8:21 PM">October 15, 2013</ABBR><STRONG>  Protocols  </STRONG><A href="/Team:UCL/Science/Protocols"><SPAN class="label label-warning">competent cells</SPAN></A><A href="/Team:UCL/Science/Protocols"><SPAN class="label label-warning">transformation</SPAN></A><A href="/Team:UCL/Science/Protocols"><SPAN class="label label-warning">miniprep</SPAN></A><A href="/Team:UCL/Science/Protocols"><SPAN class="label label-warning">digest</SPAN></A><A href="/Team:UCL/Science/Protocols"><SPAN class="label label-warning">gel</SPAN></A></DIV><P>senectus et netus et malesuada</P><H4>Characterisation of azo-reductase BioBrick devices</H4><DIV class="byline"> Adam Denyer   <ABBR class="published" title="Monday, October 15, 2013,  8:21 PM">October 15, 2013</ABBR><STRONG>Protocols  </STRONG><A href="/Team:UCL/Science/Protocols"><SPAN class="label label-warning">competent cells</SPAN></A><A href="/Team:UCL/Science/Protocols"><SPAN class="label label-warning">transformation</SPAN></A><A href="/Team:UCL/Science/Protocols"><SPAN class="label label-warning">miniprep</SPAN></A><A href="/Team:UCL/Science/Protocols"><SPAN class="label label-warning">digest</SPAN></A><A href="/Team:UCL/Science/Protocols"><SPAN class="label label-warning">gel</SPAN></A></DIV><P>senectus et netus et malesuada</P></DIV><DIV id="primerdesignboxo"><P>The format of these primer designs is indicated right below.</P><H3>FMN-dependent NADH-azoreductase</H3><P>From <I>Pseudomonas Putida</I> (CDS = 612bp) <B>Primer Design</B></P><P>From <I>Pseudomonas Putida</I> (CDS = 612bp) <B>Two Site-directed Mutagenesis Designs</B></P><H3>Heat-stable AzoR Mutant (AzoR 1B6)</H3><P>From <I>Pseudomonas Putida</I> (CDS = 612bp) <B>Primer Design</B></P><P>From <I>Pseudomonas Putida</I> (CDS = 612bp) <B>Two site-directed Mutagenesis Designs</B></P><H3>Spore coat protein laccase (cotA)</H3><P>From <I>Bacillus subtilis</I> (CDS = 1539bp) <B>Primer Design</B></P><P>From <I>Bacillus subtilis</I> (CDS = 1539bp) <B>Two Site-Directed Mutagenesis Designs</B></P><H3>Dye decolourising peroxidase BSU38260 (BsDyP)</H3><P>From <I>Bacillus subtilis</I> (CDS = 1251bp) <B>Primer Design</B></P><P>From <I>Bacillus subtilis</I> (CDS = 1251bp) <B>Site-Directed Mutagenesis Designs</B></P></DIV></DIV></DIV><FOOTER style="margin-bottom: -31px;"><DIV class="wrapper clearfix"><DIV class="FooterContent"><DIV id="FC11" style="font-size:120%"><H3 class="fontfixer" style="margin-top: 5px;">Contact Us</H3><P class="fontfixer">University College London  Gower Street - London  WC1E 6BTBiochemical Engineering Department<STRONG>Phone:</STRONG><A href="tel:phonenum" class="tele">+44 (0)20 7679 2000</A><STRONG>Email:</STRONG><A href="mailto:email@domain.com">ucligem2014@gmail.com</A></P></DIV><DIV id="FC22"><H3 class="fontfixer" style="margin-top: 5px;"><A href="https://2014.igem.org/Team:UCL/Humans/Sponsor" style="color: black;">Sponsors</A></H3></DIV><DIV id="FC33"><H3 class="fontfixer" style="margin-top: 5px;">Follow Us</H3><A class="twitter-timeline" href="https://twitter.com/UCLiGEM" data-widget-id="494172163977392128" height="235px" width="300px">Tweets by @UCLiGEM</A></DIV></DIV></DIV></FOOTER><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:UCL/Science/Primers">http://2014.igem.org/Team:UCL/Science/Primers</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UCL/Science/Primers" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UCL/Science/Primers" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UCL/Science/Primers&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UCL/Science/Primers&amp;oldid=386701" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>