"
Page
Discussion
View source
History
teams
Log in
 
Team:UCL/Project/Xenobiology
From 2014.igem.org
Goodbye Azodye UCL iGEM 2014
iGEM
HOME
PROJECT
SCIENCE
HUMANS
FAQ
UCL iGEM
Project
Science
Humans
FAQ
-.
Introduction
Strategies
Cofactors
XenoRank
Xeno-quinones
Silencing
References
Xenobiology - The Ultimate Biosafety Tool
The wide use of genetically modified organisms causes concerns on how they will interact in the natural environment. In particular could the genetically modified microbes escape our constrains, and outcompete the organisms found in the natural ecosystem? Could the DNA we inserted into a specific bacteria be transmitted, with unknown spread of information?
Since the early days of genetic engineering we had to reflect on biosafety strategies to control these risks, and synthetic biology is bringing these concerns to another level: the more we tinker with biology, the more our biosafety needs to be bullet-proof.
Xenobiology implements the term "synthetic" by creating organisms that are unable to survive in the natural environment and necessitate an artificial intervention from man to exist. It aims to create a synthetic "man-made" version of Biology, that respects the definition of life, but is based on entirely different mechanisms to function. The biochemistry of a xeno-organism uses new XNAs, genetic codes and cofactors different from the ones explored by Biology and is therefore incompatible with other forms of life. This allows a much higher level of control: a xeno-organism will not be able to find the xenocompounds in the natural environmentnor will be able to use bacterial communication systems.
We explored this possibility with the longer term vision of creating an X. coli which is metabolically dependent on azo dyes. An alien form of life, different from the one we know, will merge synthetic chemistry with synthetic biology - allowing the remediate the damage that the first one caused and making the remediating agent dependent on the toxic compounds. This system would be completely incompatible and invisible to regular biology, now we can ask: is non-biological life safe enough?
Biological vs. Xenobiological Strategies
Biosafety strategies have so far explored biology to implement clever control mechanisms to control. They investigated various strategies that allow to kill bacteria when needed or that hinder genetic information to spread among different organisms. Our biosafety strategy is exploring the regions outside of Biology, with the ultimate goal of bringing Biology to a parallel domain where it does not interact with our own one. Why tinkering with our same Biology when we can create a new on, at the same time biology and technology, that we can control at a much higher level?
Biological Strategies
The biosafety mechanism is added to the system as additional layers of protection, the most explored are:
Restriction enzyme systems: autodestruction of the transformed plasmid when task ended
Semantic containment: different meaning of stop codon, other bacteria will read as stop e.g. amber codon
Auxotrophy: knock-out of biosynthesis of a key naturally produced compound that needs to be provided in the media
Suicide system: bacteria die when finished its task/changes environment e.g. toxin/antitoxin where the bacteria stops producing antitoxin when triggered hence dies
Xenobiological Strategies
The safety mechanism embedded is into the system on three different levels:
Genetic Firewall: Use of XNAs, incompatible with other organisms and synthetic nucleotides not found in nature
Semantic Firewall: Genetic code has a different meaning than the natural code, all the codons code for a different amino acid from the standard table and could code for non-natural amino acids as well
Metabolic Firewall: A Synthetic auxotrophy that uses a xenobiotic compound as key cofactor/amino acid which the bacteria is unable to produce or find in the natural enviroment
Designing Xeno-Coli
We aim to engineer the bacteria to utilise the synthetic dyes - a completely xenobiotic compound - as the key cofactor in respiration, substituting quinones in the electron transport chain.
Our X. coli will therefore only be able to survive in the presence of azodyes, a particular environment only found in the wastewater of the textile indutry that it is aimed to degrade. The biosafety strategy is embedded into the system, and tighly linked to the survival of the xenobiological organism.
Choosing an Essential Co-Factor
In order to create a xenobiological organism with a metabolic firewall we decided to try to design cofactor that would be essential to E-Coli metabolism that could be derived from our Azo Dye waste products. This cofactor would need to be functionally similar to an existing molecule in the E-Coli metabolism.
Image Credits: EcoCyc Metabolic Database To see the full and interactive tree visit: http://goo.gl/bEK46y
Given the vast number of cofactors in the e-coli we needed a method to select the molecular structures closest to azo dye waste products. To solve this problem we developed a computer program that would do just that: Many different azodyes exist – each of which gives different products after being broken down by cleaving enzymes. But how are these products related? Do they all look very similar? Or are they all very different? We're interested because we would like to take the products of one or more of the azodyes and use it to chemically synthesise a xenobiological compound that our engineered bacteria would absolutely need to continue to survive.
The image above is not a map of stars or galaxies, but a map of the chemical similarity space of the products of azodye breakdown. We call it the 'Azodye Night Sky'. Here the colour denotes the colour of the original azodye (except black = white), and distance is a rough measure of the similarity of two compounds.
This image was included in our exhibition as part of our Uncolour Me Curious event.
How was the Azodye Night Sky generated?
There exist computational chemistry tools that can analyse the similarity between two molecules. These work by first encoding each molecule of interest as a bit string "10010001100101…" where each bit represents the presence (1) or absence (0) of some substructure within the molecule. These bit strings are known as fingerprints.
We can then compare molecules by taking the bitwise AND operation on the two fingerprints. This is a function that is only 1 if both molecules are 1. For example:
A = 0110101...       B = 0011111... A AND B = 0010101…
Then we can get the similarity between the two molecules by the fraction: (Number of 1s in A AND B) / (Total number of bits)   But if we want to visualise this we don't actually want the similarity but instead the dissimilarity (the distance between two molecules in similarity space): dissimilarity = 1 - similarity   Now imagine we have N molecules. Then the NxN dissimilarity matrix gives us the dissimilarity between any two of those molecules. But because similarity space is so complex, if we wanted to draw  the map of these distances we would need to use (in general) N-1 dimensions!   Because we want to draw this information in 2 dimensions, we need to use a method to reduce the number of dimensions while keeping as much of the distance information as we can. Here we have chosen to use Multidimensional Scaling (MDS).   Finally we can plot the map of our molecules – incorporating their fingerprint dissimilarity – our Azodye Night Sky!   This work was performed in the Python programming language using the RDKit package (to generate molecules, fingerprints, and dissimilarity).
XenoRank: A Tool for Prioritising Xenobiological Synthesis
Our Azodye Night Sky is attractive, but really we want to use these techniques to help us find suitable xenobiological compounds.   So we have developed a web application to help us prioritise which azodye breakdown products are most similar to a list of xenobiological cofactor compounds that we are interested in. We've called this tool XenoRank.
We start by entering a list of molecules in the SMILES format. These are compared with a set of default compounds of xenobiological interest. Currently this is a list of cofactor compounds.
The results of the application is a report, where the compounds we are interested in (for us Azodye breakdown products) are ordered with respect to the highest similarity to any of our xenobiological compounds.
We show the above diagram for each compound, showing the similarity to each of the xenobiological compounds.   We have published this tool on Github under an MIT licence. We hope it to be useful for other iGEM teams, and the synthetic biology community in general.
Designing the Xeno-quinones
The pathway chosen from XenoRank was that of the cofactors ubiquinone and menaquinone. They are the electron buffer substituent of coenzyme Q in complex III of the electron transport chain that creates ATP on the membrane of E-Coli.
Ubiquinone is the electron buffer used in aerobic conditions and menaquinone in anaerobic conditions. Both quinols produce protons and electrons to create the electrochemical gradient used later in Complex IV and V to yield ATP.  In order to design Xeno versions of these molecules, it was important to ensure that we understood what each constituent on the the molecule provided to the chemical reaction.
From this understanding we designed the following molecules for our xeno-coli that we believe retains the properties described above
Once we The next step was to generate the organic chemical mechanism to go from the azo dye waste products to the new quinols. They had to be generated via organic chemistry as opposed to synthetic biology because if they could be produced by biological mechanisms they would not work as xeno molecules (please see science of xeno section).
Replacing the Natural Quinones
In order for our organism to become an autotroph to our xenobiological quinones it was necessary for us to remove the natural quinones. The best approach it seemed was to do a gene knock out for a biosynthesis step of ubiquinone and menaquinone. The metabolic pathway provided a number of options
In order for our organism to become an autotroph to our xenobiological quinones it was necessary for us to remove the natural quinones. The best approach it seemed was to do a gene knock out for a biosynthesis step of ubiquinone and menaquinone. The metabolic pathway provided a number of options.  A search of the literature showed that a knockout of ispB successfully killed e-coli colonies as it is [1]essential for E. coli's . ispB codes for the protein that attaches the hydrophobic side chain on the quinones, allowing them to dock in the membrane. As ispB is used to synthesise all electron buffer quinones, it was the perfect knock out as it meant that only one was required.
See paper here: http://goo.gl/MlMY3c
Antisense RNA Gene Silencing
Our first step was to try a proof of concept knockdown technique: antisense RNA gene silencing. We designed primers to obtain the reverse complement strand of a section of the gene we want to silence. When this section is transcribed it interferes with ispB affecting its translation.
The design for the primers that would amplify the gene contained the bbk prefix and suffix. The forward primer contains suffix, reverse primer contains prefix hence sequence inserted as reverse complement into the vector
The gene was then cloned into pSB1C3 under the control of lac inducible promoter to observe effect on growth.
The knockdown showed to be unsuccessful in reducing bacteria growth and therefore cannot be use to create a xenobiological organism. The next step to try was a full knock out using the crispr technique
CRISPR Knock Outs
The CRISPR technique is based on the bacterial ability to cut out or replace viral DNA that has been inserted into it’s plasmids. We now can now utilise the Cas9 protein with a target sequence to do cut out of any gene within a plasmid.
The Cas9 protein is expressed with gRNA that is complimentary for the target sequence within the DNA you wish to cut out with a PAM (Protospacer Adjacent Motif) sequence downstream of the target. The PAM sequence in e-coli must be an NGG sequence (N representing any base)
To design our gRNA we used the target sequence online design tool DNA 2.0: https://www.dna20.com/eCommerce/cas9/input
Our gRNA target sequences were as follows:
Despite our plans there was not sufficient time to complete a knockout using CRISPR and that will be our next step beyond the competition.
References:
Wright, O., Stan, G.-B., and Ellis, T. (2013). Building-in biosafety for synthetic biology. (Review) Microbiology, 159, 1221-1235. http://www.ncbi.nlm.nih.gov/pubmed/23519158
Okada, K., Minehira, M., and Zhu, X. (1997). The ispB gene encoding octaprenyl diphosphate synthase is essential for growth of Escherichia coli. Journal of Bacteriology, 179, 3058–3060. http://www.ncbi.nlm.nih.gov/pubmed/9139929
Søballe, B. , Poole, K. R. (1999). Microbial ubiquinones: multiple roles in respiration, gene regulation and oxidative stress management. (Review) Microbiology, 145, 1817-1830. http://www.ncbi.nlm.nih.gov/pubmed/10463148
Schmidt, M (2010). Xenobiology: A new form of life as the ultimate biosafety tool Bioessays, 32, 322-331. http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2909387/
Malyshev, D.A., Dhami, K., Lavergne, T. et al. (2014). A semi-synthetic organism with an expanded genetic alphabet Nature, 509, 385-388. http://www.nature.com/nature/journal/v509/n7500/full/nature13314.html
Contact Us
University College London
Gower Street - London
WC1E 6BT
Biochemical Engineering Department
Phone: +44 (0)20 7679 2000
Email: ucligem2014@gmail.com
Sponsors
Follow Us
Tweets by @UCLiGEM
Retrieved from " http://2014.igem.org/Team:UCL/Project/Xenobiology "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
