<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UCL_Science_Experiment"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UCL/Science/Experiment">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UCL/Science/Experiment&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UCL/Science/Experiment&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UCL/Science/Experiment&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UCL/Science/Experiment" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UCL/Science/Experiment</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Goodbye Azodye UCL iGEM 2014</TITLE></P><DIV id="nav-container" class="" style="top: 0px; margin-top: 0px; height: 70px;"><NAV id="nav-main" style="overflow: visible; position: fixed;"><DIV id="nav-inner"><UL class="inline-block-list"><LI id="logo"><A href="/" id="iGEM">iGEM
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL" id="HOME">HOME
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL/Project" id="PROJECT">PROJECT
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL/Science" id="SCIENCE">SCIENCE
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL/Humans" id="HUMANS">HUMANS
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL/FAQ" id="FAQ">FAQ
							</A></LI></UL></DIV></NAV></DIV><DIV id="sexyheader"><DIV id="layout"><DIV id="menu"><DIV class="pure-menu pure-menu-open"><A class="pure-menu-heading" href="https://2014.igem.org/Team:UCL">UCL iGEM</A><UL><LI class="rezmenu"><A href="https://2014.igem.org/Team:UCL/Project">Project</A></LI><LI class="rezmenu"><A href="https://2014.igem.org/Team:UCL/Science">Science</A></LI><LI class="rezmenu"><A href="https://2014.igem.org/Team:UCL/Humans">Humans</A></LI><LI class="rezmenu"><A href="https://2014.igem.org/Team:UCL/FAQ">FAQ</A></LI>
-.
							</UL></DIV></DIV></DIV></DIV><TITLE>jQuery UI Accordion - Default functionality</TITLE><DIV id="bodyContent"><DIV class="textArena"><UL class="tabs"><LI><A href="#view1">Stage 1</A></LI><LI><A href="#view2">Stage 2</A></LI><LI><A href="#view3">Stage 3</A></LI><LI><A href="#view4">Stage 4</A></LI><LI><A href="#view5">Stage 5</A></LI><LI><A href="#view6">Stage 6</A></LI></UL><DIV class="tabcontents"><DIV id="view1"><DIV class="textTitle"><H4>Stage 01: Extraction of Useful BioBrick Plasmids from iGEM 2014 Distribution Kit</H4></DIV><DIV><STRONG>Protocols  </STRONG><A href="/Team:UCL/Science/Proto"><SPAN class="label label-warning">competent cells</SPAN></A><A href="/Team:UCL/Science/Proto"><SPAN class="label label-warning">transformation</SPAN></A><A href="/Team:UCL/Science/Proto"><SPAN class="label label-warning">miniprep</SPAN></A><A href="/Team:UCL/Science/Proto"><SPAN class="label label-warning">digest</SPAN></A><A href="/Team:UCL/Science/Proto"><SPAN class="label label-warning">gel</SPAN></A></DIV><P>We began our project by identifying a range of BioBrick parts present in the iGEM 2014 distribution kit which we required as part of our cloning strategy. These parts primarily consisted of both constituitive and inducible promoter systems with ribosome binding sites which we could then use in conjunction with our azo-reductase BioBricks to assemble a functional azo dye degrading gene. We also decided that we would use the Red Florescent Protein expressing BioBrick as a control for any further transformation experiments. As the level of DNA present within each plate of the distribution kit is insufficient to perform digest and ligation reactions on it was necessary to transform each of these plasmids into our NEB5alpha competent cells. After growing our transformed cells overnight we then mini-prepped each of them to obtain BioBrick plasmids at suitable concentrations for future experiments.</P><FONT size="2"><TABLE border="1px" width="100%" height="auto"><THEAD><TR><TH> Registry ID </TH><TH> Name / Function </TH><TH> Antibiotic Resistance </TH><TH> Source </TH><TH> Size </TH></TR></THEAD><TBODY><TR><TD><CENTER>U</CENTER></TD><TD> <A href="http://parts.igem.org/Part:BBa_K314103">BBa_K314103</A></TD><TD> IPTG-inducible LacI Expression Cassette </TD><TD> Chloramphenicol</TD><TD> Spring 2014 BioBrick Distribution. Plate 1, Well 4D.</TD><TD> <A href="/Team:UCL/Science/Sequences#BBa_K314103">1638 bp</A></TD></TR><TR><TD><CENTER>T</CENTER></TD><TD> <A href="http://parts.igem.org/Part:BBa_J04450">BBa_J04450</A></TD><TD> RFP Coding Device </TD><TD> Chloramphenicol</TD><TD> Spring 2014 BioBrick Distribution. Plate 4, Well 4B. </TD><TD> <A href="/Team:UCL/Science/Sequences#BBa_J04450">1069 bp</A></TD></TR><TR><TD><CENTER>T</CENTER></TD><TD> <A href="http://parts.igem.org/Part:BBa_R0010">BBa_R0010</A></TD><TD> IPTG-inducible LacI Promoter</TD><TD> Chloramphenicol</TD><TD> Spring 2014 BioBrick Distribution. Plate 3, Well 4G.</TD><TD> <A href="/Team:UCL/Science/Sequences#BBa_R0010">200 bp</A></TD></TR><TR><TD><CENTER>T</CENTER></TD><TD> <A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A></TD><TD> Ribosomal Binding Site (RBS)</TD><TD> Chloramphenicol</TD><TD> Spring 2014 BioBrick Distribution. Plate 4, Well 1N.</TD><TD> <A href="/Team:UCL/Science/Sequences#BBa_B0034">12 bp</A></TD></TR><TR><TD><CENTER>T</CENTER></TD><TD> <A href="http://parts.igem.org/Part:BBa_K518012">BBa_K518012</A></TD><TD> RBS + RFP + Double Terminator</TD><TD> Chloramphenicol</TD><TD> Spring 2014 BioBrick Distribution. Plate 1, Well 18C.</TD><TD> <A href="/Team:UCL/Science/Sequences#BBa_K518012">828 bp</A></TD></TR><TR><TD><CENTER>N</CENTER></TD><TD> <A href="http://parts.igem.org/Part:BBa_K206000">BBa_K206000</A></TD><TD> pBAD Strong Promoter</TD><TD> Chloramphenicol</TD><TD> Spring 2014 BioBrick Distribution. Plate 3, Well 14A.</TD><TD> <A href="/Team:UCL/Science/Sequences#BBa_K206000">130 bp</A></TD></TR><TR><TD><CENTER>! N</CENTER></TD><TD> <A href="http://parts.igem.org/Part:BBa_R0011">BBa_R0011</A></TD><TD> LacI-Regulated, Lambda pL Hybrid Promoter</TD><TD> Chloramphenicol</TD><TD> Spring 2014 BioBrick Distribution. Plate 2, Well 6D.</TD><TD> <A href="/Team:UCL/Science/Sequences#BBa_R0011">55 bp</A></TD></TR><TR><TD><CENTER>! N</CENTER></TD><TD> <A href="http://parts.igem.org/Part:BBa_B0012">BBa_B0012</A></TD><TD> Transcription Terminator for E. coli RNA Polymerase</TD><TD> Chloramphenicol</TD><TD> Spring 2014 BioBrick Distribution. Plate 2, Well 2B.</TD><TD> <A href="/Team:UCL/Science/Sequences#BBa_B0012">41 bp</A></TD></TR></TBODY></TABLE></FONT><DIV><FONT size="1">Note: U = Used in experiments; T = Used for testing purposes but not for making BioBrick Devices; N = Transformed from Distribution Kits, but not used in experiments; ! = Problematic parts (see Parts Registry), were not used.</FONT></DIV></DIV><DIV id="view2"><DIV class="textTitle"><H4>Stage 02: Identification of Useful Genes for Making New BioBricks</H4></DIV><STRONG>Identifying Azo-Dye Degrading Enzymes</STRONG><P>Searching through the literature, we identified a number of bacterial species (including <EM>Bacillus subtilis</EM> and <EM>Pseudomonas sp.</EM>) that have proven to degrade azo dye compounds <A href="http://www.ncbi.nlm.nih.gov/pubmed/21655981">[1]</A><A href="http://www.ncbi.nlm.nih.gov/pubmed/24475252">[2]</A><A href="http://www.itqb.unl.pt/martins/index_files/JBC2002.pdf">[3]</A><A href="http://www.ncbi.nlm.nih.gov/pubmed/23820555">[4]</A>. 
    We contacted the <A href="http://www.itqb.unl.pt/martins">Microbial &amp; Enzyme Technology Lab</A> led by Dr Lígia O. Martins at the Universidade Nova de Lisboa, who are currently researching how azo dye degrading enzymes function, and they were keen to collaborate with us on our project. They agreed to send us a set of five plasmids, each containing different genes encoding azo dye degrading enzymes from both <EM>B. subtilis</EM> and <EM>P. putida</EM> (including mutated forms found to exhibit enhanced degradation activity), for us to use in our investigations (see Table below). </P><FONT size="2"><TABLE border="1px" width="100%" height="auto"><THEAD><TR><TH> Gene ID</TH><TH> Name / Function </TH><TH> Source </TH><TH> Size </TH><TH> Plasmid </TH></TR></THEAD><TBODY><TR><TD> <A href="http://www.ncbi.nlm.nih.gov/pubmed/21655981">pAzoR</A></TD><TD> FMN-dependent NADH-azoreductase 1 </TD><TD> <EM>Pseudomonas putida</EM></TD><TD> 612 bp </TD><TD> In expression vector: pET-21a (+) (ampicillin resistant) <A href="http://www.addgene.org/browse/sequence_vdb/2549/ ">[2] </A><A href=" http://biochem.web.utah.edu/hill/links/pET21.pdf">[3] </A>, initially cloned between <EM>NdeI</EM> and <EM>BamHI</EM> restriction sites. </TD></TR><TR><TD> <A href="http://www.ncbi.nlm.nih.gov/pubmed/24475252">p1B6</A></TD><TD> AzoR Heat-stable Mutant</TD><TD> <EM>Pseudomonas putida</EM></TD><TD> 612 bp </TD><TD> In expression vector: pET-21a (+) (ampicillin resistant) <A href="http://www.addgene.org/browse/sequence_vdb/2549/ ">[2] </A><A href=" http://biochem.web.utah.edu/hill/links/pET21.pdf">[3] </A>, initially cloned between <EM>NdeI</EM> and <EM>BamHI</EM> restriction sites. </TD></TR><TR><TD> <A href="http://www.itqb.unl.pt/martins/index_files/JBC2002.pdf">pCotA</A></TD><TD> Spore Coat Protein Laccase</TD><TD> <EM>Bacillus subtilis</EM></TD><TD> 1542 bp </TD><TD> In expression vector: pET-21a (+) (ampicillin resistant (ampR)) <A href="http://www.addgene.org/browse/sequence_vdb/2549/ ">[2] </A><A href=" http://biochem.web.utah.edu/hill/links/pET21.pdf">[3] </A>, initially cloned between <STRONG><EM>NheI</EM></STRONG> and <EM>BamHI</EM> restriction sites. </TD></TR><TR><TD> <A href="http://www.ncbi.nlm.nih.gov/pubmed/23820555">pBsDyP</A></TD><TD> Dye Decolourising Peroxidase BSU38260</TD><TD> <EM>Bacillus subtilis</EM></TD><TD> 1251 bp </TD><TD> In expression vector: pET-21a (+) (ampicillin resistant) <A href="http://www.addgene.org/browse/sequence_vdb/2549/ ">[2] </A><A href=" http://biochem.web.utah.edu/hill/links/pET21.pdf">[3] </A>, initially cloned between <EM>NdeI</EM> and <EM>BamHI</EM> restriction sites. </TD></TR><TR><TD> <A href="http://www.ncbi.nlm.nih.gov/pubmed/23820555">pPpDyP</A></TD><TD> Dye Decolourising Peroxidase PP_3248 </TD><TD> <EM>Pseudomonas putida</EM></TD><TD> 864 bp </TD><TD> In expression vector: pET-21a (+) (ampicillin resistant) <A href="http://www.addgene.org/browse/sequence_vdb/2549/ ">[2] </A><A href=" http://biochem.web.utah.edu/hill/links/pET21.pdf">[3] </A>, initially cloned between <EM>NdeI</EM> and <EM>BamHI</EM> restriction sites. </TD></TR></TBODY></TABLE></FONT><STRONG>Extraction of <EM>B. Subtilis</EM> Genomic DNA</STRONG><DIV><STRONG>Protocols  </STRONG><A href="/Team:UCL/Science/Proto"><SPAN class="label label-warning">DNA extraction</SPAN></A></DIV><P>In the meantime, Helina (in our team), was able to obtain <EM>B. subtilis</EM> and <EM>P. aeruginosa</EM> strains for us to test whether we could retrieve azo dye degrading enzymes from their genomes, specifically, the azo-reductase gene (AzoR). This would be the first step for our first azoreductase BioBrick. 
    We extracted the genomic DNA from <EM>B. subtilis</EM> strain using a Promega Wizard Genomic DNA extraction kit so that we could subsequently amplify the azoreducatase gene (AzoR1) and create our first azoreductase BioBrick. After completing the genomic DNA extracton we ran a gel to show that we had successfully extracted the <I>B. subtilis</I> genomic DNA.</P></DIV><DIV id="view3"><DIV class="textTitle"><H4>Stage 03: Transforming E. coli with Azo-Dye Degrading Plasmids from Lisbon</H4></DIV><STRONG>Transforming <EM>E. coli</EM> with Azo-Dye Degrading Plasmids</STRONG><DIV><STRONG>Protocols  </STRONG><A href="/Team:UCL/Science/Proto"><SPAN class="label label-warning">competent cells</SPAN></A><A href="/Team:UCL/Science/Proto"><SPAN class="label label-warning">transformation</SPAN></A><A href="/Team:UCL/Science/Proto"><SPAN class="label label-warning">miniprep</SPAN></A></DIV><P>The five azo dye degrading enzymes from Lisbon arrived as the respective genes in pET-21a (+) ampicillin resistant (ampR) expression vectors/plasmids (size: 5443 bp)<A href="http://www.addgene.org/browse/sequence_vdb/2549/ ">[1]</A><A href=" http://biochem.web.utah.edu/hill/links/pET21.pdf">[2]</A>. The DNA concentrations of these plasmids, however, were insufficient to perform PCR amplification, therefore we transformed each into our own <EM>E. coli</EM> competent cells (grown from NEB DH5α derivatives). After growing the cells overnight, we made bacterial glycerol stocks and miniprepped the cells to obtain plasmids at sufficient concentrations for further work.</P><STRONG>Diagnostic Digest of Azo-Dye Degrading Plasmids</STRONG><DIV><STRONG>Protocols  </STRONG><A href="/Team:UCL/Science/Proto"><SPAN class="label label-warning">digest</SPAN></A><A href="/Team:UCL/Science/Proto"><SPAN class="label label-warning">gel</SPAN></A></DIV><P>A diagnostic digest was performed to ascertain that these pET-21a (+) plasmids contained the gene we expected. As each plasmid possessed <EM>EcoRI</EM> and <EM>XbaI</EM> restriction sites close to the genes of interest, we performed double-digests using these recognition enzymes and predicted the digest fragments. The digestion products were visualised using gel electrophoresis (see image right). </P></DIV><DIV id="view4"><DIV class="textTitle"><H4>Stage 04: Creation of Azo-Reductase BioBrick Parts from Plasmids</H4></DIV><DIV><STRONG>Protocols  </STRONG><A href="/Team:UCL/Science/Proto"><SPAN class="label label-warning">analytical digest</SPAN></A><A href="/Team:UCL/Science/Proto"><SPAN class="label label-warning">gel</SPAN></A></DIV><P>After isolating our genes of interest we attempted to use PCR as a method of prefix and suffix generation to fit the BioBrick standard assembly parts format.</P><P>Achieving a successful PCR proved difficult; this may have been due to poor PCR reagent quality. We repeated the PCR using various polymerases (Taq, Phusion and Pfu) and also different dNTP mixes. Eventually, we succeeded in amplifying AzoR 1B6, BsDyP, and ispB asDNA with the required BioBrick Prefix and Suffix. Given the time-constraints, we did not succeed in also amplifying AzoR, CotA, and PpDyP with the Prefix and Suffix.</P><DIV><STRONG>Protocols  </STRONG><A href="/Team:UCL/Science/Proto"><SPAN class="label label-warning">digestion</SPAN></A><A href="/Team:UCL/Science/Proto"><SPAN class="label label-warning">ligation</SPAN></A><A href="/Team:UCL/Science/Proto"><SPAN class="label label-warning">transformation</SPAN></A><A href="/Team:UCL/Science/Proto"><SPAN class="label label-warning">inoculation</SPAN></A><A href="/Team:UCL/Science/Proto"><SPAN class="label label-warning">analytical digest</SPAN></A><A href="/Team:UCL/Science/Proto"><SPAN class="label label-warning">gel</SPAN></A></DIV><P>Our next step was to ligate these into the required pSB1C3 backbone. For BsDyP and ispB asDNA, this proved to be fairly straightforward, and quickly resulted in the production of our first new BioBricks: <A href="http://parts.igem.org/Part:BBa_K1336003">BBa_K1336003</A> for BsDyP+pSB1C3, and <A href="http://parts.igem.org/Part:BBa_K1336005">BBa_K1336005</A> for ispB+pSB1C3.
    This was trickier for 1B6, as this gene possessed 2 illegal <EM>PstI</EM> restriction sites. <A href="https://2014.igem.org/Team:UCL/Science/Primers">Site-directed mutagenesis primers</A> were designed to remove these sites, however, we could not completed this in time for submission. We did, however, succeed in performing a directionless ligation into pSB1C3. From here, we screened for plasmids with the correct orientation, and started our characterisation assays with this pseudo-BioBrick part.</P></DIV><DIV id="view5"><DIV class="textTitle"><H4>Stage 05: Assembly of BioBrick Devices</H4></DIV><P>
    We decided to assemble both BsDyP and ispB in a LacI cassette, inducible by IPTG. 
Issues with inconclusive antibiotic effectivity led to major delays in construction of these composite parts. We first had to prove our antibiotics were functioning properly before making progress on our project.</P><P>We confirmed the construction of our BsDyP and ispB cassettes using analytical gel digest cutting at sites E and P.</P><P>Ideally, we wanted a Azo-Remediation Chassis (ARC), our BioBrick System, to be assembled as follows:</P><P>A further development on this prototype would be to have Bba_K1336000, the AzoR gene, to be inducibly transcribed by one promoter (say BBa_K314103, the LacI Expression Cassette) such that it is expressed in the reductive step of our azo dye remediation process. This would form a distinct BioBrick Device of &quot;promotor A + AzoR + double terminator&quot;. A secondary BioBrick Device (of &quot;promoter B + gene 2 + double terminator&quot;) would follow this, where gene 2 would be one of our enzymes that function in the oxidative step of the azo dye remediation process, e.g. laccase or one of the dye decolourising peroxidases. A further tertiary BioBrick Device, with another oxidative enzyme would also be ideal. At least 2 oxidative enzymes are proposed as the enzymes are specialised for different substrates (as described in our <A href="/Team:UCL/Project/Biobricks">BioBrick</A> page. 
        As always, time works against us, and we succeeded only in constructing 2 composite parts (BioBrick Devices): <A href="http://parts.igem.org/Part:BBa_K1336007">BBa_K1336007</A> for LacIEC+BsDyP+pSB1C3, and <A href="http://parts.igem.org/Part:BBa_K1336006">BBa_K1336006</A> for LacIEC+ispB+pSB1C3. This latter ispB asDNA device has functions in the <A href="/Team:UCL/Project/Xenobiology">biosafety aspects</A> of our ARC.</P><FONT size="2"><TABLE border="1px" width="100%" height="auto"><THEAD><TR><TH> Registry ID </TH><TH> Gene ID</TH><TH> Name / Function </TH><TH> Source </TH><TH> Size </TH><TH> Status </TH></TR></THEAD><TBODY><TR><TD> <A href="http://parts.igem.org/Part:BBa_K1336000">BBa_K1336000</A></TD><TD> AzoR </TD><TD> FMN-dependent NADH-azoreductase 1 </TD><TD> <EM>Pseudomonas putida</EM></TD><TD> <A href="/Team:UCL/Science/Sequences#BBa_K1336000">612 bp</A></TD><TD> [<A href="/Team:UCL/Science/Primers">In Progress</A>]: primers designed </TD></TR><TR><TD> <A href="http://parts.igem.org/Part:BBa_K1336001">BBa_K1336001</A></TD><TD> 1B6 </TD><TD> AzoR heat-stable mutant</TD><TD> <EM>Pseudomonas putida</EM></TD><TD> <A href="/Team:UCL/Science/Sequences#BBa_K1336001">612 bp</A></TD><TD> [<A href="/Team:UCL/Science/Experiments">In Progress</A>]: to remove 2 illegal PstI sites </TD></TR><TR><TD> <A href="http://parts.igem.org/Part:BBa_K1336002">BBa_K1336002</A></TD><TD> CotA </TD><TD> Spore Coat Protein Laccase</TD><TD> <EM>Bacillus subtilis</EM></TD><TD> <A href="/Team:UCL/Science/Sequences#BBa_K1336002">1542 bp</A></TD><TD> [<A href="/Team:UCL/Science/Primers">In Progress</A>]: primers designed </TD></TR><TR><TD> <A href="http://parts.igem.org/Part:BBa_K1336003">BBa_K1336003</A></TD><TD> BsDyP </TD><TD> Dye Decolourising Peroxidase BSU38260</TD><TD> <EM>Bacillus subtilis</EM></TD><TD> <A href="/Team:UCL/Science/Sequences#BBa_K1336003">1251 bp</A></TD><TD> [<A href="/Team:UCL/Science/Experiments">New BioBrick Part</A>]: submitted </TD></TR><TR><TD> <A href="http://parts.igem.org/Part:BBa_K1336004">BBa_K1336004</A></TD><TD> PpDyP </TD><TD> Dye Decolourising Peroxidase PP_3248 </TD><TD> <EM>Pseudomonas putida</EM></TD><TD> <A href="/Team:UCL/Science/Sequences#BBa_K1336004">864 bp</A></TD><TD> [<A href="/Team:UCL/Science/Primers">In Progress</A>]: primers designed </TD></TR><TR><TD> <A href="http://parts.igem.org/Part:BBa_K1336005">BBa_K1336005</A></TD><TD> ispB RNAi </TD><TD> RNAi of Octaprenyl Diphosphate Synthase fragment </TD><TD> <EM>Escherichia coli, K12 strain</EM></TD><TD> <A href="/Team:UCL/Science/Sequences#BBa_K1336005">562 bp</A></TD><TD> [<A href="/Team:UCL/Science/Experiment">New BioBrick Part</A>]: submitted </TD></TR><TR><TD> <A href="http://parts.igem.org/Part:BBa_K1336006">BBa_K1336006</A></TD><TD> LacIEC+ispB </TD><TD> IPTG inducible ispB RNAi </TD><TD> <EM>Escherichia coli, K12 strain </EM></TD><TD> <A href="/Team:UCL/Science/Sequences#BBa_K1336006">2208 bp</A></TD><TD> [<A href="/Team:UCL/Science/Experiment">New BioBrick Device</A>]: submitted </TD></TR><TR><TD> <A href="http://parts.igem.org/Part:BBa_K1336007">BBa_K1336007</A></TD><TD> LacIEC+BsDyP </TD><TD> IPTG inducible BsDyP </TD><TD> <EM>Bacillus subtilis</EM></TD><TD> <A href="/Team:UCL/Science/Sequences#BBa_K1336007">2895 bp</A></TD><TD> [<A href="/Team:UCL/Science/Experiment">New BioBrick Device</A>]: submitted </TD></TR><TR><TD> <A href="http://parts.igem.org/Part:BBa_K729006">BBa_K729006</A></TD><TD> CueO </TD><TD> Laccase </TD><TD> <EM>Escherichia coli </EM></TD><TD> <A href="/Team:UCL/Science/Sequences#BBa_K729006">1612 bp</A></TD><TD> [<A href="/Team:UCL/Science/Experiment">In Progress</A>]: ascertaining identity </TD></TR><TR><TD><CENTER>()</CENTER></TD><TD> <A href="http://parts.igem.org/Part:BBa_K500000">BBa_K500000</A></TD><TD> LiP </TD><TD> Lignin Peroxidase </TD><TD> <EM>Phanerochaete chrysosporium</EM></TD><TD> <A href="/Team:UCL/Science/Sequences#BBa_K500000">1116 bp</A></TD><TD> [<A href="/Team:UCL/Science/Results">Improved Characterisation</A>]: toxicity issues in gene synthesis.  [<A href="/Team:UCL/Science/Experiment">In Progress</A>]: to subclone into pSB1C3/pSB3C5. </TD></TR><TR><TD> <A href="http://parts.igem.org/Part:BBa_K729004">BBa_K729004</A></TD><TD> nucB </TD><TD> Extracellular nuclease </TD><TD> <EM>Staphylococcus aureus</EM></TD><TD> <A href="/Team:UCL/Science/Sequences#BBa_K729004">561 bp</A></TD><TD> [<A href="/Team:UCL/Science/Results">Improved Function</A>] </TD></TR></TBODY></TABLE></FONT></DIV><DIV id="view6"><DIV class="textTitle"><H4>Stage 06: Characterisation Assays of BioBrick Device(s)</H4></DIV><P>Results and protocols for the characterisation pages can be found <A href="https://2014.igem.org/Team:UCL/Science/Results">here</A></P></DIV></DIV></DIV></DIV><FOOTER style="margin-bottom: -31px;"><DIV class="wrapper clearfix"><DIV class="FooterContent"><DIV id="FC11" style="font-size:120%"><H3 class="fontfixer" style="margin-top: 5px;">Contact Us</H3><P class="fontfixer">University College London  Gower Street - London  WC1E 6BTBiochemical Engineering Department<STRONG>Phone:</STRONG><A href="tel:phonenum" class="tele">+44 (0)20 7679 2000</A><STRONG>Email:</STRONG><A href="mailto:email@domain.com">ucligem2014@gmail.com</A></P></DIV><DIV id="FC22"><H3 class="fontfixer" style="margin-top: 5px;"><A href="https://2014.igem.org/Team:UCL/Humans/Sponsor" style="color: black;">Sponsors</A></H3></DIV><DIV id="FC33"><H3 class="fontfixer" style="margin-top: 5px;">Follow Us</H3><A class="twitter-timeline" href="https://twitter.com/UCLiGEM" data-widget-id="494172163977392128" height="235px" width="300px">Tweets by @UCLiGEM</A></DIV></DIV></DIV></FOOTER><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:UCL/Science/Experiment">http://2014.igem.org/Team:UCL/Science/Experiment</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UCL/Science/Experiment" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UCL/Science/Experiment" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UCL/Science/Experiment&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UCL/Science/Experiment&amp;oldid=401634" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>