<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UCL_Science_Model"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UCL/Science/Model">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UCL/Science/Model&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UCL/Science/Model&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UCL/Science/Model&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UCL/Science/Model" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UCL/Science/Model</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Goodbye Azodye UCL iGEM 2014</TITLE></P><DIV id="nav-container" class="" style="top: 0px; margin-top: 0px; height: 70px;"><NAV id="nav-main" style="overflow: visible; position: fixed;"><DIV id="nav-inner"><UL class="inline-block-list"><LI id="logo"><A href="/" id="iGEM">iGEM
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL" id="HOME">HOME
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL/Project" id="PROJECT">PROJECT
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL/Science" id="SCIENCE">SCIENCE
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL/Humans" id="HUMANS">HUMANS
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL/FAQ" id="FAQ">FAQ
							</A></LI></UL></DIV></NAV></DIV><DIV id="sexyheader"><DIV id="layout"><DIV id="menu"><DIV class="pure-menu pure-menu-open"><A class="pure-menu-heading" href="https://2014.igem.org/Team:UCL">UCL iGEM</A><UL><LI class="rezmenu"><A href="https://2014.igem.org/Team:UCL/Project">Project</A></LI><LI class="rezmenu"><A href="https://2014.igem.org/Team:UCL/Science">Science</A></LI><LI class="rezmenu"><A href="https://2014.igem.org/Team:UCL/Humans">Humans</A></LI><LI class="rezmenu"><A href="https://2014.igem.org/Team:UCL/FAQ">FAQ</A></LI>
-.
							</UL></DIV></DIV></DIV></DIV><DIV class="textArena"><UL class="tabs"><LI><A href="#view1">Modelling Degradation</A></LI><LI><A href="#view2">Parameter Inference</A></LI><LI><A href="#view3">Flux Balance Analysis</A></LI><LI><A href="#view4">Enzyme Kinetics</A></LI><LI><A href="#view5">Chemical Mechanism</A></LI><LI><A href="#view6">References</A></LI></UL><DIV class="tabcontents"><DIV id="view1"><H4>Overview</H4><P> There are three ways we can degrade azodyes: using Azoreductase (AzoR), Laccase (Lac) or BsDyp. Azoreductase breaks down AzoDye (AzoD) into two products Laccase breaks down AzoDye  as well as the products of the reaction of Azoreductase with AzoDye. BsDyP acts on sulfonated AzoDyes (sAzoD):</P>  <P> In order to model this system we used COPASI. We included equations for gene expression and degradation for each gene in our pathway, as well as the intake and excretion of AzoDyes and sulfonated AzoDyes. The equations we included as well as the parameter assigned to each one are shown below: </P><DIV class="collapse-card"><DIV class="title"><STRONG>
Equations for pathway model
</STRONG></DIV></DIV><P>Using reasonable parameter values, the simulation showed that the AzoDye is degraded within two days (48 hours). This timeframe agrees with the experimental results!</P><DIV class="collapse-card"><DIV class="title"><STRONG>
Simulated timecourse data of Acid Orange AzoDye degradation by Azoreductase, Laccase and BsDyP
</STRONG></DIV></DIV></DIV><DIV id="view2"><H4>Parameter Inference</H4><P> We wanted to see which part of the pathway is the bottleneck in degrading the AzoDyes and sulfonated AzoDyes. So we analysed the parameters of our model to see which one is the most constrained, which could give us an insight on which one to tweak experimentally in the future in order to speed up the degradation. To do that we used ABC-SysBio (Liepe, 2014) .</P><P> Approximate Bayesian Computation (ABC) is a method that utilises Bayesian statistics for parameter inference in synthetic biology. Given a model and data form that model, it computes the most likely parameters that could give rise to that data. We used the model and simulated data we had in order to find out which parameters are restricted in the values they can have in order to achieve that behaviour. </P><P> To use ABC-SysBio we had to make an SBML file describing our model and write an xml input file. The input file contains values for initial conditions of each species in our model, as well as prior distributions for each parameter. The prior distributions consist of a range of values for each parameter, from which the algorithm will sample values. The input file also contains the data from the degradation of AzoDyes and sulfonated AzoDyes over two days. </P> <P>ABC-SysBio samples a value for each parameter from the priors and using the initial conditions provided, simulates the model. The resulting time course is compared to the data provided, and if the distance between the two is greater than a threshold, the sampled parameter set is rejected. This is repeated for 100 sets of samples, consisting of one population. The sets that were accepted are then perturbed by a small amount and then a new population is sampled from the perturbed sets. This process is repeated until the distance between the data and the simulations is minimised:
 
 The parameter values that gave rise to this final population are called the 'posterior distribution'.</P><DIV class="collapse-card"><DIV class="title"><STRONG>
Posterior distribution of model parameters
</STRONG></DIV><DIV class="body"><P>The distribution of values for each parameter are shown in the diagonal. All distributions are between 0 and 1. Drawing a straight line from one parameter to the other, at the point where the two meet, the two parameters have been plotted against each other in a density contour plot. Three parameters stand out as very constricted, k3, k8, k17 and k18. These are the parameters of the reactions for intake  (k3) and secretion (k8) of AzoDyes as well as the intake (k17) and secretion (k18) of sulfonated AzoDyes by the cell. This shows that the bottleneck happens at those points in our pathway. So if we increase the rate of intake and secretion of AzoDyes and sulfonated AzoDyes in our synthetic pathway, we could speed up the process of degradation! </P></DIV></DIV></DIV><DIV id="view3"><H4>Flux Balance Analysis</H4><P> In order to see whether our xenobiological approach would work we wanted to check whether lack of Ubiquinone would have an effect on the growth rate of the chassis. The literature (Okada 1997 and Soballe, 1999) suggested that Ubiquinone is essential for E.coli growth so we decided to put that to the test! In order to do that we used Flux Balance Analysis (FBA). FBA is a method that uses the metabolism model of E.coli (see below) and calculates the flow of metabolites through that system that is required to maximise a given objective.</P>
The image below demonstrates the <I>E. coli</I> metabolism plotted in Cytoscape (Cline, 2007):
 <P> In our case we used growth rate as the objective to maximise. We performed FBA for the core E.coli metabolism with and without Ubiquinone present. With Ubiquinone present the growth rate was calculated to be 0.98 h<SUP> -1 </SUP>. Without Ubiquione in the system the growth rate was found to be 0  h<SUP> -1 </SUP>, indicating that E.coli would not grow and survive without ubiquione. This suggested that silencing the essential genes for Ubiquinone production and supplying it externally would give us control over the survival of the chassis and ultimately allow us to contain it.</P><DIV class="collapse-card"><DIV class="title"><STRONG>
Core metabolism map used for FBA
</STRONG></DIV></DIV></DIV><DIV id="view5"><P> Currently Synthetic biology is primarily based on the use of active modules (usually enzymes) from organisms to create one single organism that can successfully execute a goal. However without understanding the enzymatic action on a molecular scale we are unlikely to ever be able to improve them or design our own. Be believe that this will be the future of SynBio and therefore we have made a special effort to further the understanding of the enzymes we are using via chemical mechanism modelling in conjunction with our chemistry department. </P><H4>Azo Reductase</H4><P>The mechanism of reductive cleavage can either be thought of a step wise addition of H+ ions and electrons or hydride and H+ ions in concert as pictured below 
</P><H4>Laccase and Peroxidases</H4><P>Although many papers have touched on these oxidation mechanisms; they tend to skip steps and don’t make entire sense. Examples of this exist in [1]:
</P><P>It’s issues include radicals gaining electrons and remaining radicals. Protons disappearing and more of the like. We have therefore worked hard to create a mechanism that makes sense.
</P><P> Other reactions such as the polymerisation are lacking literature completely and therefore have been modelled as below. The example polymerisation is via the azo reductase product of mordant brown 33. </P><H4> References </H4><P> [1] Phenolic Azo Dye Oxidation by Laccase from Pyricularia oryzae, Appl. Environ. Microbiol.December 1995 vol. 61 no. 124374-4377</P></DIV><DIV id="view4"><P>Enzyme kinetics are used to further understand reaction parameters of the enzyme. Enzyme kinetics are largely based on the Michaelis-Menten kinetic model that allows us to calculate Vmax (The maximum rate of reaction) and Km (Michaelis Constant: the substrate concentration at which the reaction rate is at half-maximum).
</P><P> Where [S]=Substrate concentration and V=Rate of Reaction
</P><P>The lineweaver burke plot is a double reciprocal plot of 1/[S] against 1/[V] that allows 1/Vmax and -1/Km to be understood via y and x intercepts respectively. We used our data for the decolorisation via enzyme BsDyp (see data page) to create a lineweaver burke plot and hence infer the values of Vmax and Km. 
</P><P>From this we can infer that Vmax=0.0305 mg/mol per hour (4 d.p.) and Km=0.0034 mg/mol (4 d.p.)
</P></DIV><DIV id="view6"><H4>References</H4><P> Liepe, J., Kirk, P., Filippi, S., Toni, T., et al. (2014) A framework for parameter estimation and model selection from experimental data in systems biology using approximate Bayesian computation. [Online] 9 (2), 439–456.</P><P> Hoops S., Sahle S., Gauges R., Lee C., Pahle J., Simus N., Singhal M., Xu L., Mendes P. and Kummer U. (2006). COPASI: a COmplex PAthway SImulator. Bioinformatics 22, 3067-74.</P><P> Cline, M.S., Smoot, M., Cerami, E., Kuchinsky, A., et al. (2007) Integration of biological networks and gene expression data using Cytoscape. Nature Protocols. [Online] 2 (10), 2366–2382. </P><P> Orth, J.D., Thiele, I. &amp; Palsson, B.O. (2010) What is flux balance analysis? Nature Biotechnology. [Online] 28 (3), 245–248. </P><P> Okada, K., Minehira, M., Zhu, X., Suzuki, K., et al. (1997) The ispB gene encoding octaprenyl diphosphate synthase is essential for growth of Escherichia coli. Journal of bacteriology. 179 (9), 3058–3060. </P><P> Søballe, B. &amp; Poole, R.K. (1999) Microbial ubiquinones: multiple roles in respiration, gene regulation and oxidative stress management. Microbiology (Reading, England). 145 ( Pt 8)1817–1830 </P></DIV></DIV></DIV><FOOTER style="margin-bottom: -31px;"><DIV class="wrapper clearfix"><DIV class="FooterContent"><DIV id="FC11" style="font-size:120%"><H3 class="fontfixer" style="margin-top: 5px;">Contact Us</H3><P class="fontfixer">University College London  Gower Street - London  WC1E 6BTBiochemical Engineering Department<STRONG>Phone:</STRONG><A href="tel:phonenum" class="tele">+44 (0)20 7679 2000</A><STRONG>Email:</STRONG><A href="mailto:email@domain.com">ucligem2014@gmail.com</A></P></DIV><DIV id="FC22"><H3 class="fontfixer" style="margin-top: 5px;"><A href="https://2014.igem.org/Team:UCL/Humans/Sponsor" style="color: black;">Sponsors</A></H3></DIV><DIV id="FC33"><H3 class="fontfixer" style="margin-top: 5px;">Follow Us</H3><A class="twitter-timeline" href="https://twitter.com/UCLiGEM" data-widget-id="494172163977392128" height="235px" width="300px">Tweets by @UCLiGEM</A></DIV></DIV></DIV></FOOTER><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:UCL/Science/Model">http://2014.igem.org/Team:UCL/Science/Model</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UCL/Science/Model" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UCL/Science/Model" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UCL/Science/Model&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UCL/Science/Model&amp;oldid=382960" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>