"
Page
Discussion
View source
History
teams
Log in
 
Team:UCL/Science/Primers
From 2014.igem.org
Goodbye Azodye UCL iGEM 2014
iGEM
HOME
PROJECT
SCIENCE
HUMANS
FAQ
UCL iGEM
Project
Science
Humans
FAQ
-.
List of Primer Designs for Creation of BioBrick Parts
Enzyme 01: AzoR, Azoreductase
Enzyme 02: 1B6, Azoreductase heat-stable mutant
Enzyme 03: CotA
Enzyme 04: BsDyP, Bacilus subtilis dye-decolourising peroxidase
Enzyme 05: PpDyP, Pseudomonas putida dye-decolourising peroxidase
Enzyme 06: LiP, Lignin peroxidase / Ligninase
UNEDITED, WILL BE BACK
Extraction of Bacillus subtilis genomic DNA
Adam Denyer, Tanel Ozdemir    June 13, 2014   Protocols   DNA extraction
Our literature search identified a number of bacterial species that have been proven to degrade azo dye compounds including B. subtilis and P. aeruginosa.  We were able to obtain a B. subtilis strain for use in our project from ?.  We extracted the genomic DNA from this strain using a Promega Wizard Genomic DNA extraction kit so that we could subsequently amplify the azo-reducatase gene (AzoR1) and create our first azo-reductase BioBrick.  After completing the genomic DNA extracton we ran a gel to show that we had successfully extracted the B. subtilis genomic DNA.
Transforming E. coli with Azo-reductase plasmids
Adam Denyer    October 15, 2013   Protocols   PCR competent cells transformation miniprep
We were gratefully provided with a set of five plasmids from a group of researchers working at the University of Lisbon, Portugal who are researching how azo-dye degrading enzymes function and who were keen to collaborate with us.  These plasmids contained a number of genes encoding azo-dye degrading enzymes from both B. subtilis and P. putida including mutated forms found to exhibit enhanced degradation activity.  As the DNA concentration of the plasmids we were sent was insufficient to perform PCR amplification on we transformed each of these plasmids into our E. coli NEB5alpha competent cells.  After growing the cells overnight we then mini-prepped each of them to obtain plasmids at sufficient concentrations for future experimental work.
Name
Function
Source
Concentration
Sequence
Initial Plasmid / Vector
Comments
pAzoR
FMN-dependent NADH-azoreductase 1
Pseudomonas putida
Miniprep,
48 ng/uL,
597 bp [Check! Not 612 bp?]
Expression vector pET-21a (+) (ampicillin resistant (ampR)) , initially cloned between NdeI and BamHI.
Plasmid provided by Lisbon
p1B6 (AzoR 1B6)
Mutant: Heat-stable; FMN-dependent NADH-azoreductase 1
Pseudomonas putida
Miniprep,
68 ng/uL,
597 bp [Check! Not 612 bp?]
Expression vector pET-21a (+) (ampR) , initially cloned between NdeI and BamHI.
Plasmid provided by Lisbon .
pCotA
Spore Coat Protein Laccase
Bacillus subtilis
Miniprep,
103 ng/uL
1733 bp [Check! Not 1539 bp?]
Expression vector pET-21a (+) (ampR) , initially cloned between NheI and BamHI.
Plasmid provided by Lisbon .
pBsDyP
Dye Decolourising Peroxidase BSU38260
Bacillus subtilis
Miniprep,
51 ng/uL,
1251 bp
Expression vector pET-21a (+) (ampR) , initially cloned between NdeI and BamHI.
Plasmid provided by Lisbon .
pPpDyP
Dye Decolourising Peroxidase PP_3248
Pseudomonas putida
Miniprep,
55 ng/uL
861 bp [Check! Not 864 bp?]
Expression vector pET-21a (+) (ampR) , initially cloned between NdeI and BamHI.
Plasmid provided by Lisbon .
Diagnostic digest of azo-reductase plasmids
Adam Denyer    October 15, 2013   Protocols   digest gel
After successfully transforming these plasmids into competent E. coli NEB5alpha cells we then performed a diagnostic digest and gel electrophoresis experiment to ascertain that these plasmids contained the gene we expected.  Each plasmid was digested using two restriction enzymes chosen to digest DNA as specific points on the plasmids and create fragments of known length which we could then confirm using gel electrophoresis.
Creation of azo-reductase BioBrick parts from plasmids
Adam Denyer    October 15, 2013   Protocols   competent cells transformation miniprep
senectus et netus et malesuada
Diagnostic digest of azo-reductase BioBrick parts
Adam Denyer    October 15, 2013   Protocols   digest gel
senectus et netus et malesuada
Extraction of useful BioBrick plasmids from iGEM 2014 Distribution Kit
Adam Denyer    October 15, 2013   Protocols   competent cells transformation miniprep digest gel
We began our project by identifying a range of BioBrick parts present in the iGEM 2014 distribution kit which we required as part of our cloning strategy.  These parts primarily consisted of both constituitive and inducible promoter systems with ribosome binding sites which we could then use in conjunction with our azo-reductase BioBricks to assemble a functional azo dye degrading gene.  We also decided that we would use the Red Florescent Protein expresing BioBrick as a control for any further transformation experiments.   As the level of DNA present within each plate of the distribution kit is insufficient to perform digest and ligation reactions on it was necessary to transform each of these plasmids into our NEB5alpha competent cells.  After growing our transformed cells overnight we then mini-prepped each of them to obtain BioBrick plasmids at suitable concentrations for future experiments.
Assembling azo-reductase BioBrick Device(s)
Adam Denyer    October 15, 2013   Protocols   competent cells transformation miniprep digest gel
senectus et netus et malesuada
Characterisation of azo-reductase BioBrick devices
Adam Denyer    October 15, 2013 Protocols   competent cells transformation miniprep digest gel
senectus et netus et malesuada
The format of these primer designs is indicated right below.
FMN-dependent NADH-azoreductase
From Pseudomonas Putida (CDS = 612bp) Primer Design
From Pseudomonas Putida (CDS = 612bp) Two Site-directed Mutagenesis Designs
Heat-stable AzoR Mutant (AzoR 1B6)
From Pseudomonas Putida (CDS = 612bp) Primer Design
From Pseudomonas Putida (CDS = 612bp) Two site-directed Mutagenesis Designs
Spore coat protein laccase (cotA)
From Bacillus subtilis (CDS = 1539bp) Primer Design
From Bacillus subtilis (CDS = 1539bp) Two Site-Directed Mutagenesis Designs
Dye decolourising peroxidase BSU38260 (BsDyP)
From Bacillus subtilis (CDS = 1251bp) Primer Design
From Bacillus subtilis (CDS = 1251bp) Site-Directed Mutagenesis Designs
Contact Us
University College London
Gower Street - London
WC1E 6BT
Biochemical Engineering Department
Phone: +44 (0)20 7679 2000
Email: ucligem2014@gmail.com
Sponsors
Follow Us
Tweets by @UCLiGEM
Retrieved from " http://2014.igem.org/Team:UCL/Science/Primers "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
