<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UCL_Science_Proto"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UCL/Science/Proto">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UCL/Science/Proto&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UCL/Science/Proto&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UCL/Science/Proto&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UCL/Science/Proto" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UCL/Science/Proto</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Goodbye Azodye UCL iGEM 2014</TITLE></P><DIV id="nav-container" class="" style="top: 0px; margin-top: 0px; height: 70px;"><NAV id="nav-main" style="overflow: visible; position: fixed;"><DIV id="nav-inner"><UL class="inline-block-list"><LI id="logo"><A href="/" id="iGEM">iGEM
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL" id="HOME">HOME
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL/Project" id="PROJECT">PROJECT
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL/Science" id="SCIENCE">SCIENCE
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL/Humans" id="HUMANS">HUMANS
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL/FAQ" id="FAQ">FAQ
							</A></LI></UL></DIV></NAV></DIV><DIV id="sexyheader"><DIV id="layout"><DIV id="menu"><DIV class="pure-menu pure-menu-open"><A class="pure-menu-heading" href="https://2014.igem.org/Team:UCL">UCL iGEM</A><UL><LI class="rezmenu"><A href="https://2014.igem.org/Team:UCL/Project">Project</A></LI><LI class="rezmenu"><A href="https://2014.igem.org/Team:UCL/Science">Science</A></LI><LI class="rezmenu"><A href="https://2014.igem.org/Team:UCL/Humans">Humans</A></LI><LI class="rezmenu"><A href="https://2014.igem.org/Team:UCL/FAQ">FAQ</A></LI>
-.
							</UL></DIV></DIV></DIV></DIV><DIV id="bodyContent"><DIV class="textArena"><UL class="tabs"><LI><A href="#view1">Competent cells</A></LI><LI><A href="#view2">Digestion</A></LI><LI><A href="#view3">Ligation</A></LI><LI><A href="#view4">Agar Plates</A></LI><LI><A href="#view5">Transformation</A></LI><LI><A href="#view6">Agarose Electrophoresis</A></LI><LI><A href="#view7">Colony Boil</A></LI><LI><A href="#view8">PCR</A></LI><LI><A href="#view9">Growth Inhibition Assay</A></LI><LI><A href="#view10">ispB Growth Assay</A></LI><LI><A href="#view11">ispB Inoculation Assay</A></LI><LI><A href="#view12">BsDyP Degradation Assay</A></LI></UL><DIV class="tabcontents"><DIV id="view1"><DIV class="textTitle"><H4>Creating competent cells</H4></DIV><P1><B>Materials</B>
LB Media, 50ml Falcon Tubes, Ice, Chilled centrifuge, Calcium Chloride (CaCl2), Eppendorf tubes (300ul/tube)<B>Procedure</B>
1. Inoculate a single colony into 5ml Lb in 50ml falcon tube. Grown O/N @ 37oC
2. Use 1ml to inoculate 100ml of LB in 250ml bottle the next morning.
Shake @ 37oC for 1.5-3 hours.

Or

1. Inoculate a single colony into 25ml LB in a 250ml bottle in the morning
2. Shake @ 37oC for 4-6 hours.

Then…

3. Put the cells on ice for 10mins (keep cold from now on).
4. Collect the cells by centrifugation in the big centrifuge for 3 minutes @ 6Krpm.
5. Decant supernatant and gently resuspend on 10ml cold 0.1M CaCl (cells sensitive to mechanical disruption).
6. Incubate on ice x 20 minutes
7. Centrifuge as in 2.
8. Discard supernatant and gently resuspend on 5ml cold 0.1M CaCl/15%Glycerol.
9. Dispense in microtubes (300ųl/tube). Freeze at -80oC.

Note: Home-made competent cells were used to transform registry BioBricks 

</P1></DIV><DIV id="view2"><DIV class="textTitle"><H4>Digestion</H4></DIV><P1><B>Materials</B>
MilliQ water, NEBuffer 2.1, BSA, Pipettes and autoclaved tips  Enzymes: EcoRI-HF, SpeI, XbaI and PstI, <B>Notes</B><LI> The volume of DNA to be digested will depend on the concentration obtained from miniprep of the DNA from culture. This must be made up to 500ng for each 50ul digestion
	eg. if concentration BBa_R0010 is available at a concentration of 330ng/ul
	we require 500/330 = 1.5ul of the DNA solution for a digestion</LI><LI> All digestions carried out for iGEM purposes i.e. BioBrick creation are double digests. NEBuffer 2.1 is the ideal buffer for all combinations used.</LI><LI> Depending on whether the part being digested is an upstream insert, downstream insert or plasmid backbone, the combinations of enzymes differ. 
	For Upstream parts, such as promoters digest with EcoRI-HF and SpeI 
	Digest downstream parts with XbaI and PstI
	Digest plasmid backbones with EcoRI-HF and PstI<B>Procedure</B>
1. Calculate amount of DNA solution that will be needed for 500ng 
2. Following this, calculate the amount of water needed to make up to 50ul if both enzymes and BSA are 1ul each and Buffer is 5ul.
3. Add the components to a microcentrifuge tube in the order: Water &gt; Buffer &gt; BSA &gt; DNA &gt; Enzymes
4. Leave to incubate at 37ºC for 1-2 hours 
5. Remove and deactivate the enzymes by incubating at 80ºC for 20 minutes </LI></P1></DIV><DIV id="view3"><DIV class="textTitle"><H4>Ligation</H4></DIV><P1><B>Materials</B>
MilliQ water, Ice, Pipettes and autoclaved tips, NEB T4 DNA Ligase Buffer, T4 DNA Ligase <B>Notes</B><LI>Try not to repeatedly freeze and thaw the buffer by making 10-20ul aliquots and using them instead of the entire stock solution each time </LI><LI> All ligations should be done with an aim to be transformed. 
</LI><LI> No-insert controls must be plated onto plates with antibiotics corresponding to the backbone resistance and no-vector controls on antibiotic plates corresponding to resistance of original host vector<B>Procedure</B>
1. Calculate water needed to total volume of 20ul where enzyme added will be 1 ul and all other components will be 2ul
 - Usually this will include buffer, the upstream part digestion, downstream part digestion and backbone digestion.
2. Do no-insert and no-vector control ligations for each backbone and insert 
3. Add components in the order: Water &gt; Buffer &gt; DNA upstream part &gt; DNA downstream part &gt; Backbone DNA &gt; Ligase 
4. Leave for 10 minutes at room temperature approx. 25ºC
5. Deactivate at 80ºC for 20 minutes 
6. Transform products of ligation into competent cells </LI></P1></DIV><DIV id="view4"><DIV class="textTitle"><H4>Making LB Agar Plates</H4></DIV><P><B>Materials</B>
LB Agar powder, plates, antibiotics, sterile jar, sterile flame or biosafety cabinet.<B>Procedure</B>
1. Dissolve LB agar in sterile/autoclaved jar of sterile water 40g/L.
2. Swirl until completely dissolved
3. Autoclave LB agar
The following steps should be completed under a sterile flame or biosafety cabinet. Ideally after autoclaving, before LB Agar solidifies:
4. Add antibiotics from their stock solutions in 1:1000 ratio with LB agar, can be added either directly to each individual plate or to the entire solution.
5. Pour agar into plates, roughly half way (30-50mL per plate). Leave plates with lids half on (prevent condensation).
6. Allow plates to solidify under flame
7. Cover and leave on bench or in fridge</P></DIV><DIV id="view5"><DIV class="textTitle"><H4>Plasmid DNA Transformation</H4></DIV><P><B>Materials</B>
Competent Cells, Plasmid DNA, Antibiotic Plates<B>Procedure</B>

1.  Thaw competent cells on ice
2.  50uL cells enough for 1 transformation
3.  Add 1ug of DNA to 50uL competent cells

If BioBrick from distribution, resuspend DNA well in 10uL ddH20

4.  Add 2-5uL DNA i.e. ligation product or BioBrick re-suspension to 50uL competent cells
5.  Flick by hand or pipette up and down gently
6.  Place cells on ice for 30 minutes
7.  Place cells in water bath or heat block at 42oC for 30 seconds
8.  Place cells on ice for 2 minutes
9. Add 950ul of SOC or LB media and place in a shaking incubator for a maximum of 2 hours (37oC/250rpm)
10. Label two petri dishes with LB agar and the appropriate antibiotics(s) with the date, plasmid backbone and ligated genes/parts and antibiotic added.
	You may also choose to write down the volume of transformed product plated 
11. Spin down the incubated cells for 2mins at 4000rpm to obtain a cell pellet.
12. Pipette 100ul of the supernatant or fresh LB media before discarding the rest of the supernatant and re-suspending the pellet in the 100ul of media 
13. Incubate the plates at 37ºC for 12-14 hours, always place plates lid-down.

If incubated for too long the antibiotics start to break down and un-transformed cells will begin to grow. This is especially true for ampicillin - because the resistance enzyme is excreted by the bacteria, and inactivates the antibiotic outside of the bacteria

You can pick a single colony, make a glycerol stock, grow up a cell culture and miniprep.

Count the colonies on the 20 μl control plate and calculate your competent cell efficiency.</P></DIV><DIV id="view6"><DIV class="textTitle"><H4>Agarose Gel Electrophoresis</H4></DIV><B>Prepare Agarose Gel (1%) for Electrphoresis</B><B>Materials:</B>
Agarose powder, spatula, weighing boat, microwave, Ethidium Bromide, Gel dock, <P>1. Add ingredients to flask and microwave for about 120 seconds. Shake. Repeat until mixture fully dissolved.
2.Leave to cool for 20-30 seconds.
3.Add 20ul of Ethidium Bromide.
Place comb in gel frame. When flask is cool enough to handle, pour the solution into the gel frame and leave to set (10-20 minutes).</P><B>Preparing for Loading:</B><P><B>1. Prepare Ladder</B><UL><LI>a. Add 1ul of DNA ladder to labelled Eppendorf.
   </LI><LI>b. Add 1ul of DNA loading buffer (dye) to Eppendorfs containing DNA and DNA ladder (Unless ladder is already combined with dye).
   </LI><LI>c. Add 1ul of milliQwater
   </LI><LI>d. Spin down. 
   </LI></UL></P><P><B>2. Prepare samples:</B><UL><LI>a. Add 10ul of digest (250ng of DNA) to 3ul of loading buffer.
   </LI></UL><B>Loading the gels</B></P><P>Prepare Ladder<UL><LI>1. Place gel frame in electrophoresis box with black (cathode) part away from you  and red (anode) part closest to you.
   </LI><LI>2. Carefully remove comb from gel.
</LI><LI>3. Fill the top and bottom wells with 1x TAE. Pour over top of gel, continue pouring until gel is covered.
</LI><LI>4. Load the wells starting with the DNA ladder in lane 1. Be sure to note which lane each sample is loaded into.
</LI><LI>5. Run gel for  approximately 60 minutes at constant voltage of 120V. Check for bubbles at the cathode which shows it has started.
   </LI></UL><B>Visualising the gels</B><UL><LI>1. Remove the frame from the electrophoresis box and pour off any liquid. Wipe the bottom of the frame with tissue.
   </LI><LI>2. Place in GelDoc 2000, press Epi white to position gel, press Epi UV to visualise.
   </LI><LI>3. On the computer open program Quantity One. Select scanner, GelDoc XR, select manual acquire, adjust value until gel is visible.
   </LI><LI>4. Print or save to USB.
</LI></UL><B>Visualising the gels</B><UL><LI>TAE from the gel run can be re-used, pour back into bottle.
</LI><LI>Ensure all ethidium bromide rubbish is disposed of in ethidium bromide bin.   
  </LI></UL></P></DIV><DIV id="view7"><DIV class="textTitle"><H4>Colony Boil</H4></DIV><P>Colony boil can be used to isolate genes present in an organism. We used a colony boil to isolate AzoR from e.coli using primers originally designed for PCR.</P><P>
1. Using a sterile pipette tip carefully lift a single colony from any most recently transformed E.coli plate.
Repeat for more colonies from different strains if available.
2. Point the tip into 250ul of distilled/MilliQ water in an eppendorf, MUST BE screw-cap, and shake a bit. Withdraw and discard tip.
3.Place the tube in a 100͒C water bath for 5 minutes.
4. Remove and dry the outside of the tube. Shake the contents of the tube to the bottom. Use 2ul of the boiled cell solution for PCR with genomic DNA primers for any genes of interest that are present in the genome such as IspB
</P></DIV><DIV id="view8"><DIV class="textTitle"><H4>Standard PCR Protocol</H4></DIV><P>Although our PCR protocol evolved throughout the project this protocol represents the core procedure we used throughout the project.</P><P><B>1. Make stock solution primer mix</B>
Make 1/10 dilution of forward and reverse primer mix with TE buffer solution or Milli-Q water (e.g 10uL forward primer, 10uL reverse primer and 180uL TE buffer). Label with primer names, concentration, date and name of user.</P><P><B>2. Plan the PCR</B>

V1 x C1=V2 x C2 where V= volume in uL, C=concentration in uM for each of the primers.

Use this calculation to calculate the volumes of forward and reverse primers to be used in the PCR.
Reagents for 1 x PCR:<UL><LI>10X Buffer - 5.0ul
</LI><LI>MgCl<SUB>2</SUB> - 1.0ul
</LI><LI>dNTP - 1.0ul
</LI><LI>Forward Primer - 2.5ul
</LI><LI>Reverse Primer - 2.5ul
</LI><LI>DNA Template - 1.0ul
</LI><LI>Taq Polymerase - 0.5ul
</LI><LI>H<SUB>2</SUB>O (Make up to total vol) - 36.5ul
</LI><LI>Total = 50.0ul

</LI></UL></P><P><B>3. PCR Program:</B><TABLE class="tg"><TBODY><TR><TH class="tg-031e">Stages</TH><TH class="tg-031e">NEB Phusion DNA Polymerase(5X Phusion HF Buffer)</TH><TH class="tg-031e">Roche Taq DNA Polymerase</TH><TH class="tg-031e">PROMEGA Pfu DNA Polymerase</TH></TR><TR><TD class="tg-031e">Hot start (<SUP>o</SUP>C) / time</TD><TD class="tg-031e">98<SUP>o</SUP>C / 30s</TD><TD class="tg-031e">94<SUP>o</SUP>C / 2min</TD><TD class="tg-031e">95<SUP>o</SUP>C / 1-2min</TD></TR><TR><TD class="tg-031e">Cycles(#)</TD><TD class="tg-031e">25-35</TD><TD class="tg-031e">25-30</TD><TD class="tg-031e">25-35</TD></TR><TR><TD class="tg-031e">Denaturation (<SUP>o</SUP>C) / time</TD><TD class="tg-031e">98<SUP>o</SUP>c / 5-10s</TD><TD class="tg-031e">94<SUP>o</SUP>C / 15-30s</TD><TD class="tg-031e">95<SUP>o</SUP>C  / 30s-1min</TD></TR><TR><TD class="tg-031e">Annealing (<SUP>o</SUP>C) / time</TD><TD class="tg-031e">45-72<SUP>o</SUP>C / 10-30s</TD><TD class="tg-031e">55 to 65<SUP>o</SUP>C / 30-60s</TD><TD class="tg-031e">42-65<SUP>o</SUP>C / 30s</TD></TR><TR><TD class="tg-031e">Elongation (<SUP>o</SUP>C) / time</TD><TD class="tg-031e">72<SUP>o</SUP>C / 15-30s per kb</TD><TD class="tg-031e">72 OR 68<SUP>o</SUP>C / 45s - 2min</TD><TD class="tg-031e">72-74<SUP>o</SUP>C / 2-4min</TD></TR><TR><TD class="tg-031e">Final Extension (<SUP>o</SUP>C) / time</TD><TD class="tg-031e">72<SUP>o</SUP>C / 5-10min</TD><TD class="tg-031e">72 OR 68<SUP>o</SUP>C / 7min</TD><TD class="tg-031e">72-74<SUP>o</SUP>C / 5min</TD></TR><TR><TD class="tg-031e">Hold temp (<SUP>o</SUP>C) / time</TD><TD class="tg-031e">4 to 10<SUP>o</SUP>C / infidenlity</TD><TD class="tg-031e">4<SUP>o</SUP>C / infidelity</TD><TD class="tg-031e">4<SUP>o</SUP>C</TD></TR></TBODY></TABLE></P></DIV><DIV id="view9"><DIV class="textTitle"><H4>AZO DYE TOXICITY ASSAY (GROWTH)</H4></DIV><B>Materials</B><UL><P>LB Media, 50mL Falcon tubes, E.coli glycerol stock, conical flask, Azo dyes, Eppendorf tubes</P><B>Prepare azo dye stocks</B><P>The powders used were 55% for Reactive Black 5 and 85% for Acid Orange 7 azo dye and were dissolved in sterile Milli-Q water.
<TABLE class="tg"><TBODY><TR><TH class="tg-031e">Azo Dye</TH><TH class="tg-031e">Low Concentration</TH><TH class="tg-031e">Medium Concentration </TH><TH class="tg-031e">High Concentration</TH></TR><TR><TD class="tg-031e">Reactive Black 5</TD><TD class="tg-031e">0.05 mg/mL</TD><TD class="tg-031e">0.5 mg/mL</TD><TD class="tg-031e">5 mg/mL</TD></TR><TR><TD class="tg-031e">Acid Orange 7</TD><TD class="tg-031e">0.0155 mg/mL</TD><TD class="tg-031e">0.155 mg/mL</TD><TD class="tg-031e">1.55 mg/mL</TD></TR></TBODY></TABLE></P><B>Prepare Bacterial Solution</B><P>1.Prepare stock bacterial solution of 90mL LB in 180uL of bacterial glycerol stock
2. Make aliquots of 10mL into 8 falcon tubes</P><TABLE class="tg"><TBODY><TR><TH class="tg-031e">RB5 Zero</TH><TH class="tg-031e">RB5 Low (0.05 mg/mL)</TH><TH class="tg-031e">RB5 Medium (0.5 mg/mL)</TH><TH class="tg-031e">RB5 High (5 mg/mL)</TH></TR><TR><TD class="tg-031e">AO7 Zero</TD><TD class="tg-031e">AO7 Low (0.0155 mg/mL)</TD><TD class="tg-031e">AO7 Medium (0.155 mg/mL)</TD></TR></TBODY></TABLE><B>3. Place tubes in incubator at 37<SUP>o</SUP>C overnight at 250rpm.</B><B>4. Next morning add dyes to tubes</B><B>5. Record OD data at 600nm</B><P> Take two samples of 0.5 mL from each tube for the measurements, and place in a cuvette to be analysed in spectrophotometer</P><UL><LI>13 hours post inoculation– 2x0.5
</LI><LI>14 hours post inoculation– 2x0.5
</LI><LI>15 hours post inoculation– 2x0.5
</LI><LI>16 hours post inoculation– 2x0.5
</LI><LI>18 hours post inoculation– 2x0.5
</LI></UL><B>6. Plot OD data</B><DIV id="view10"><DIV class="textTitle"><H4>LEC + ispB Growth Assay</H4></DIV><P><B>Materials:</B>
LB Media, 50mL Falcon tubes, E.coli glycerol stock, conical flask, Azo dyes, Eppendorf tubes</P><B>1. Prepare bacterial solutions:</B><P>Plasmid-free:<UL><LI>Add 10mL of LB. Inoculate with 20uL of glycerol stock 
</LI><LI>Add 25uL of sterile water.</LI></UL>
LEC-ispB and LEC-ispB + IPTG:<UL><LI>Add 25mL to a Falcon tube. 
</LI><LI>Add 25uL of 25ng/uL Chloramphenicol.
</LI><LI>Inoculate with 50uL of ispB Glycerol stock.
</LI><LI>Distribute 10mL into each of the two tubes</LI></UL><B>2. Grow overnight at 37<SUP>o</SUP>C 250rpm.</B><B>3. Add add IPTG (1mM) to tubes (ZERO) and (LEC-ispB + IPTG)</B><UL><LI>Add 10uL of IPTG (1M) to 10 mL of tube for a final concentration of 1mM.
</LI><LI>Add 10uL of sterile water to ispB tube.
</LI></UL><B>4. Record OD data at 600nm</B></P><P> Take two samples of 0.5 mL from each tube for the measurements, and place in a cuvette to be analysed in spectrophotometer</P><UL><LI>13 hours post inoculation– 2x0.5
</LI><LI>14 hours post inoculation– 2x0.5
</LI><LI>15 hours post inoculation– 2x0.5
</LI><LI>16 hours post inoculation– 2x0.5
</LI><LI>18 hours post inoculation– 2x0.5
</LI></UL><B>5. Plot OD data</B></DIV><DIV id="view11"><DIV class="textTitle"><H4>ispB Inoculation Assay</H4></DIV><B>Materials:</B>
LB media, M9+thymine, M9+thymine+CASaa, IPTG(1M), Chloramphenicol, 50ml Falcon tubes, E.coli glycerol stocks, ispB glycerol stocks, LEC + ispB glycerol stocks.
<B>1. Make LB and M9 solutions specified below:</B><P>ispB<UL><LI>35mL LB
</LI><LI>60uL glycerol stock
</LI><LI>50uL Chloramphenicol 25ng/uL</LI></UL><B>LEC + ispB - 1</B><UL><LI>35mL LB
</LI><LI>60uL glycerol stock
</LI><LI>35ul Chloramphenicol (25ng/uL)</LI></UL><B>LEC + ispB - 2</B><UL><LI>35mL LB
</LI><LI>60uL glycerol stock
</LI><LI>35ul Chloramphenicol (25ng/uL)</LI></UL><B>LEC + ispB - 3</B><UL><LI>35mL LB
</LI><LI>60uL glycerol stock
</LI><LI>35ul Chloramphenicol (25ng/uL)</LI></UL><B>2. Distribute 10mL aliquots of bacterial solutions into falcon tubes</B><B>3. Inoculate dyes, adding 100uL of corresponding stock dye solutions and 100uL of sterile water for the controls.</B><B>4. Add 10uL of 1M IPTG.</B><B>5. Record OD data at 680nm</B><UL><LI>0 hours post inoculation– 2x0.5
</LI><LI>2 hours post inoculation– 2x0.5
</LI><LI>4 hours post inoculation– 2x0.5
</LI><LI>6 hours post inoculation– 2x0.5
</LI><LI>8 hours post inoculation– 2x0.5</LI></UL><B>6. plot OD data.</B></P></DIV><DIV id="view12" style="display: block;"><DIV class="textTitle"><H4>LEC + BsDyP DEGRADATION ASSAY</H4></DIV><P><B>Materials:</B>
Autoclaved Media (LB), 50ml Falcon tubes, Pipettes (10ul - 5ml), Pipette Tips (10ul - 5ml), BsDyP glycerol stock (T2_80_5, 9/10/14 gel), Conical flask, Azo Dyes
<B>1. Prepare Azo dye stock solutions:</B><TABLE><TBODY><TR><TH>Azo Dye</TH><TH>V-Low concentration</TH><TH>Low concentration</TH><TH>Medium concentration</TH></TR><TR><TD>Reactive Black 5 (55%)</TD><TD>0.005 mg/mL</TD><TD>0.05 mg/mL</TD><TD>0.5</TD> mg/mL</TR><TR><TD>Acid Orange 7 </TD><TD>(85%)</TD><TD>0.00155</TD><TD>mg/mL</TD> 0.0155<TD>mg/mL</TD><TD>0.155 mg/mL</TD></TR></TBODY></TABLE></P><P><B>* Reactive Black 5</B>

Medium concentration stock, a 1 in 10 dilution of the high concentration stock (50 mg/mL)
20uL of the High concentration stock must be added to 180 uL of water in an Eppendorf tube
Add 100 uL of solution to the corresponding Falcon tube to give a final concentration of 0.5mg/mL
Low concentration stock, a 1 in 10 dilution of the medium concentration stock (5 mg/mL)
20uL of the Medium concentration stock must be added to 180 uL of water in an Eppendorf tube
Add 100 uL of solution to the corresponding Falcon tube to give a final concentration of 0.05 mg/mL
Very Low concentration stock, a 1 in 10 dilution of the low concentration stock (0.5 mg/mL)
20uL of the Low concentration stock must be added to 180 uL of water in an Eppendorf tube
Add 100 uL of solution to the corresponding Falcon tube to give a final concentration of 0.005 mg/mL</P><P><B>Acid orange 7</B>
Medium concentration stock, a 1 in 10 dilution of the high concentration stock (15.5mg/mL)
20uL of the High concentration stock must be added to 180 uL of water in an Eppendorf tube
Add 100 uL of solution to the corresponding Falcon tube to give a final concentration of 0.155 mg/mL
Low concentration stock, a 1 in 10 dilution of the medium concentration stock (1.55 mg/mL)
20uL of the Medium concentration stock must be added to 180 uL of water in an Eppendorf tube
Add 100 uL of solution to the corresponding Falcon tube to give a final concentration of 0.0155 mg/mL
Very Low concentration stock, a 1 in 10 dilution of the low concentration stock (0.155mg/mL)
20uL of the Low concentration stock must be added to 180 uL of water in an Eppendorf tube
Add 100 uL of solution to the corresponding Falcon tube to give a final concentration of 0.00155 mg/mL</P><P><B>Prepare bacterial stock solution</B><UL><LI>LB 80 mL</LI><LI>Glycerol stock 160 uL</LI><LI>Chloramphenicol 80 uL</LI><LI>IPTG (1M) 80 uL</LI></UL></P><P><B>Distribute into 7 Falcon tubes, with no chemical, labelled</B></P><TABLE class="tg" style="undefined;table-layout: fixed; width: 982px"><TBODY><TR><TH class="tg-031e">RB5 Plasmid-free Growth</TH><TH class="tg-031e">RB5 V-LOW (0.005 mg/mL) Growth</TH><TH class="tg-031e">RB5 LOW (0.05 mg/mL) Growth</TH><TH class="tg-031e">* RB5 MEDIUM (0.5 mg/mL) Growth</TH><TH class="tg-031e">RB5 ZERO LOW (0.05mg/mL)</TH></TR><TR><TD class="tg-031e">AO7 V-LOW (0.00155 mg/mL) Growth</TD><TD class="tg-031e">AO7 LOW (0.0155 mg/mL) Growth</TD><TD class="tg-031e">AO7 MEDIUM (0.155 mg/mL) Growth</TD></TR></TBODY></TABLE>​
<P>
Place tubes in incubator at 37 ºC at 250 rpm. (8pm 13/10/14)
Discard remains of conical flask and sterilise
The next morning, at 9 am, add dyes to tubes
Record OD data.
RB5: 600nm
AO7: 480nm
Blank with LB at 600nm, do measurements of ZERO and RB5. Then blank with LB at 480, and do measurements of ZERO (again) and RB5.
With the ZERO control sample, measure at 600nm then measure RB5 samples. Then measure the control again at 480nm, and record samples for AO7.
Record OD data once for no-cell controls (only dyes)
Take two samples of 200uL from each tube for the measurements, place in a cuvette, perform 1 in 10 dilution (1.8mL of LB, that is, two pippetings of 0.9mL) and record value.
* 14 hours post inoculation– 2x0.5
* 15 hours post inoculation– 2x0.5
* 16 hours post inoculation– 2x0.5
* 17 hours post inoculation– 2x0.5
* 18 hours post inoculation– 2x0.5
Plot OD data</P></DIV><FOOTER style="margin-bottom: -31px;"><DIV class="wrapper clearfix"><DIV class="FooterContent"><DIV id="FC11" style="font-size:120%"><H3 class="fontfixer" style="margin-top: 5px;">Contact Us</H3><P class="fontfixer">University College London  Gower Street - London  WC1E 6BTBiochemical Engineering Department<STRONG>Phone:</STRONG><A href="tel:phonenum" class="tele">+44 (0)20 7679 2000</A><STRONG>Email:</STRONG><A href="mailto:email@domain.com">ucligem2014@gmail.com</A></P></DIV><DIV id="FC22"><H3 class="fontfixer" style="margin-top: 5px;"><A href="https://2014.igem.org/Team:UCL/Humans/Sponsor" style="color: black;">Sponsors</A></H3></DIV><DIV id="FC33"><H3 class="fontfixer" style="margin-top: 5px;">Follow Us</H3><A class="twitter-timeline" href="https://twitter.com/UCLiGEM" data-widget-id="494172163977392128" height="235px" width="300px">Tweets by @UCLiGEM</A></DIV></DIV></DIV></FOOTER><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:UCL/Science/Proto">http://2014.igem.org/Team:UCL/Science/Proto</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UCL/Science/Proto" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UCL/Science/Proto" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UCL/Science/Proto&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UCL/Science/Proto&amp;oldid=401682" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>