<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UCL_Project_Xenobiology"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UCL/Project/Xenobiology">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UCL/Project/Xenobiology&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UCL/Project/Xenobiology&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UCL/Project/Xenobiology&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UCL/Project/Xenobiology" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UCL/Project/Xenobiology</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Goodbye Azodye UCL iGEM 2014</TITLE></P><DIV id="nav-container" class="" style="top: 0px; margin-top: 0px; height: 70px;"><NAV id="nav-main" style="overflow: visible; position: fixed;"><DIV id="nav-inner"><UL class="inline-block-list"><LI id="logo"><A href="/" id="iGEM">iGEM
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL" id="HOME">HOME
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL/Project" id="PROJECT">PROJECT
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL/Science" id="SCIENCE">SCIENCE
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL/Humans" id="HUMANS">HUMANS
							</A></LI><LI><A href="https://2014.igem.org/Team:UCL/FAQ" id="FAQ">FAQ
							</A></LI></UL></DIV></NAV></DIV><DIV id="sexyheader"><DIV id="layout"><DIV id="menu"><DIV class="pure-menu pure-menu-open"><A class="pure-menu-heading" href="https://2014.igem.org/Team:UCL">UCL iGEM</A><UL><LI class="rezmenu"><A href="https://2014.igem.org/Team:UCL/Project">Project</A></LI><LI class="rezmenu"><A href="https://2014.igem.org/Team:UCL/Science">Science</A></LI><LI class="rezmenu"><A href="https://2014.igem.org/Team:UCL/Humans">Humans</A></LI><LI class="rezmenu"><A href="https://2014.igem.org/Team:UCL/FAQ">FAQ</A></LI>
-.
							</UL></DIV></DIV></DIV></DIV><DIV class="textArena"><UL class="tabs"><LI><A href="#view1">Introduction</A></LI><LI><A href="#view2">Strategies</A></LI><LI><A href="#view3">Cofactors</A></LI><LI><A href="#view4">XenoRank</A></LI><LI><A href="#view5">Xeno-quinones</A></LI><LI><A href="#view6">Silencing</A></LI><LI><A href="#view7">References</A></LI></UL><DIV class="tabcontents"><DIV id="view1"><DIV class="textTitle"><H4>Xenobiology - The Ultimate Biosafety Tool</H4></DIV><P>The wide use of genetically modified organisms causes concerns on how they will interact in the natural environment. In particular could the genetically modified microbes escape our constrains, and outcompete the organisms found in the natural ecosystem? Could the DNA we inserted into a specific bacteria be transmitted, with unknown spread of information?

Since the early days of genetic engineering we had to reflect on biosafety strategies to control these risks, and synthetic biology is bringing these concerns to another level: the more we tinker with biology, the more our biosafety needs to be bullet-proof.

Xenobiology implements the term &quot;synthetic&quot; by creating organisms that are unable to survive in the natural environment and necessitate an artificial intervention from man to exist. It aims to create a synthetic &quot;man-made&quot; version of Biology, that respects the definition of life, but is based on entirely different mechanisms to function. The biochemistry of a xeno-organism uses new XNAs, genetic codes and cofactors different from the ones explored by Biology and is therefore incompatible with other forms of life. This allows a much higher level of control: a xeno-organism will not be able to find the xenocompounds in the natural environmentnor will be able to use bacterial communication systems.
</P><DIV class="SCJBBHIGHLIGHT"><P>We explored this possibility with the longer term vision of creating an <I>X. coli</I> which is metabolically dependent on azo dyes. An alien form of life, different from the one we know, will merge synthetic chemistry with synthetic biology - allowing the remediate the damage that the first one caused and making the remediating agent dependent on the toxic compounds. This system would be completely incompatible and invisible to regular biology, now we can ask: is non-biological life safe enough?
</P></DIV></DIV><DIV id="view2"><DIV class="textTitle"><H4>Biological vs. Xenobiological Strategies</H4></DIV><P>Biosafety strategies have so far explored biology to implement clever control mechanisms to control. They investigated various strategies that allow to kill bacteria when needed or that hinder genetic information to spread among different organisms.
Our biosafety strategy is exploring the regions outside of Biology, with the ultimate goal of bringing Biology to a parallel domain where it does not interact with our own one. Why tinkering with our same Biology when we can create a new on, at the same time biology and technology, that we can control at a much higher level?
</P><H5>Biological Strategies</H5><P>The biosafety mechanism is added to the system as additional layers of protection, the most explored are:
<OL><LI>Restriction enzyme systems: autodestruction of the transformed plasmid when task ended </LI><LI>Semantic containment: different meaning of stop codon, other bacteria will read as stop e.g. amber codon </LI><LI>Auxotrophy: knock-out of biosynthesis of a key naturally produced compound that needs to be provided in the media </LI><LI>Suicide system: bacteria die when finished its task/changes environment e.g. toxin/antitoxin where the bacteria stops producing antitoxin when triggered hence dies</LI></OL></P><H5>Xenobiological Strategies</H5><P>The safety mechanism embedded is into the system on three different levels:
<OL><LI>Genetic Firewall: Use of XNAs, incompatible with other organisms and synthetic nucleotides not found in nature</LI><LI>Semantic Firewall: Genetic code has a different meaning than the natural code, all the codons code for a different amino acid from the standard table and could code for non-natural amino acids as well</LI><LI>Metabolic Firewall: A Synthetic auxotrophy that uses a xenobiotic compound as key cofactor/amino acid which the bacteria is unable to produce or find in the natural enviroment </LI></OL></P><H5>Designing Xeno-Coli</H5><P>We aim to engineer the bacteria to utilise the synthetic dyes - a completely xenobiotic compound - as the key cofactor in respiration, substituting quinones in the electron transport chain.  
Our <I>X. coli </I> will therefore only be able to survive in the presence of azodyes, a particular environment only found in the wastewater of the textile indutry that it is aimed to degrade. The biosafety strategy is embedded into the system, and tighly linked to the survival of the xenobiological organism.
</P></DIV><DIV id="view3"><DIV class="textTitle"><H4>Choosing an Essential Co-Factor</H4></DIV><P>In order to create a xenobiological organism with a metabolic firewall we decided to try to design cofactor that would be essential to E-Coli metabolism that could be derived from our Azo Dye waste products. This cofactor would need to be functionally similar to an existing molecule in the E-Coli metabolism.
</P><DIV style="font-size:0.5em;"><P>Image Credits: EcoCyc Metabolic Database To see the full and interactive tree visit: http://goo.gl/bEK46y</P></DIV><P>Given the vast number of cofactors in the e-coli we needed a method to select the molecular structures closest to azo dye waste products. To solve this problem we developed a computer program that would do just that:
Many different azodyes exist – each of which gives different products after being broken down by cleaving enzymes. But how are these products related? Do they all look very similar? Or are they all very different? We're interested because we would like to take the products of one or more of the azodyes and use it to chemically synthesise a xenobiological compound that our engineered bacteria would absolutely need to continue to survive.</P><P>The image above is not a map of stars or galaxies, but a map of the chemical similarity space of the products of azodye breakdown. We call it the 'Azodye Night Sky'. Here the colour denotes the colour of the original azodye (except black = white), and distance is a rough measure of the similarity of two compounds.</P><P>This image was included in our exhibition as part of our Uncolour Me Curious event.</P><H5>How was the Azodye Night Sky generated?</H5><P>There exist computational chemistry tools that can analyse the similarity between two molecules. These work by first encoding each molecule of interest as a bit string &quot;10010001100101…&quot; where each bit represents the presence (1) or absence (0) of some substructure within the molecule. These bit strings are known as fingerprints.</P><P>We can then compare molecules by taking the bitwise AND operation on the two fingerprints. This is a function that is only 1 if both molecules are 1. For example:
<PRE>
      A = 0110101...
      B = 0011111...
A AND B = 0010101…
</PRE>
 
Then we can get the similarity between the two molecules by the fraction:
(Number of 1s in A AND B) / (Total number of bits)
 
But if we want to visualise this we don't actually want the similarity but instead the dissimilarity (the distance between two molecules in similarity space):
dissimilarity = 1 - similarity
 
Now imagine we have N molecules. Then the NxN dissimilarity matrix gives us the dissimilarity between any two of those molecules. But because similarity space is so complex, if we wanted to draw  the map of these distances we would need to use (in general) N-1 dimensions!
 
Because we want to draw this information in 2 dimensions, we need to use a method to reduce the number of dimensions while keeping as much of the distance information as we can. Here we have chosen to use Multidimensional Scaling (MDS).
 
Finally we can plot the map of our molecules – incorporating their fingerprint dissimilarity – our Azodye Night Sky!
 
This work was performed in the Python programming language using the RDKit package (to generate molecules, fingerprints, and dissimilarity).</P></DIV><DIV id="view4"><DIV class="textTitle"><H4>XenoRank: A Tool for Prioritising Xenobiological Synthesis</H4></DIV><A name="robPRO">
Our Azodye Night Sky is attractive, but really we want to use these techniques to help us find suitable xenobiological compounds.
 
So we have developed a web application to help us prioritise which azodye breakdown products are most similar to a list of xenobiological cofactor compounds that we are interested in. We've called this tool XenoRank.
 
We start by entering a list of molecules in the </A><A href="http://en.wikipedia.org/wiki/Simplified_molecular-input_line-entry_system">SMILES</A> format. These are compared with a set of default compounds of xenobiological interest. Currently this is a list of cofactor compounds.
 

The results of the application is a report, where the compounds we are interested in (for us Azodye breakdown products) are ordered with respect to the highest similarity to any of our xenobiological compounds.
 

We show the above diagram for each compound, showing the similarity to each of the xenobiological compounds.
 
We have published this tool on <A href="https://github.com/robjstan/xenorank">Github</A> under an MIT licence. We hope it to be useful for other iGEM teams, and the synthetic biology community in general.

</DIV><DIV id="view5"><DIV class="textTitle"><H4>Designing the Xeno-quinones</H4></DIV><P> The pathway chosen from XenoRank was that of the cofactors ubiquinone and menaquinone. They are the electron buffer substituent of coenzyme Q in complex III of the electron transport chain that creates ATP on the membrane of E-Coli. </P><P> Ubiquinone is the electron buffer used in aerobic conditions and menaquinone in anaerobic conditions. Both quinols produce protons and electrons to create the electrochemical gradient used later in Complex IV and V to yield ATP.  In order to design Xeno versions of these molecules, it was important to ensure that we understood what each constituent on the the molecule provided to the chemical reaction. 
</P><P>From this understanding we designed the following molecules for our xeno-coli that we believe retains the properties described above 
</P><P>Once we The next step was to generate the organic chemical mechanism to go from the azo dye waste products to the new quinols. They had to be generated via organic chemistry as opposed to synthetic biology because if they could be produced by biological mechanisms they would not work as xeno molecules (please see science of xeno section).
</P><H5>Replacing the Natural Quinones </H5><P>In order for our organism to become an autotroph to our xenobiological quinones it was necessary for us to remove the natural quinones. The best approach it seemed was to do a gene knock out for a biosynthesis step of ubiquinone and menaquinone. The metabolic pathway provided a number of options 
</P><P>In order for our organism to become an autotroph to our xenobiological quinones it was necessary for us to remove the natural quinones. The best approach it seemed was to do a gene knock out for a biosynthesis step of ubiquinone and menaquinone. The metabolic pathway provided a number of options.  A search of the literature showed that a knockout of ispB successfully killed e-coli colonies as it is <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC179075/">[1]essential for <I>E. coli's</I></A> . ispB codes for the protein that attaches the hydrophobic side chain on the quinones, allowing them to dock in the membrane. As ispB is used to synthesise all electron buffer quinones, it was the perfect knock out as it meant that only one was required.</P><DIV style="font-size:0.5em;"><P>See paper here: http://goo.gl/MlMY3c 
</P></DIV></DIV><DIV id="view6"><DIV class="textTitle"><H4>Antisense RNA Gene Silencing</H4></DIV><P> Our first step was to try a proof of concept knockdown technique: antisense RNA gene silencing. We designed primers to obtain the reverse complement strand of a section of the gene we want to silence. When this section is transcribed it interferes with ispB affecting its translation.  
</P><P> The design for the primers that would amplify the gene contained the bbk prefix and suffix. The forward primer contains suffix, reverse primer contains prefix hence sequence inserted as reverse complement into the vector</P><P>The gene was then cloned into pSB1C3 under the control of lac inducible promoter to observe effect on growth.
</P><P>The knockdown  <A href="https://2014.igem.org/Team:UCL/Science/Results/Xeno#Xeno">showed to be unsuccessful in reducing bacteria growth</A> and therefore cannot be use to create a xenobiological organism. The next step to try was a full knock out using the crispr technique </P><H5>CRISPR Knock Outs</H5><P>The CRISPR technique is based on the bacterial ability to cut out or replace viral DNA that has been inserted into it’s plasmids. We now can now utilise the Cas9 protein with a target sequence to do cut out of any gene within a plasmid.
</P><P>The Cas9 protein is expressed with gRNA that is complimentary for the target sequence within the DNA you wish to cut out with a PAM (Protospacer Adjacent Motif) sequence downstream of the target. The PAM sequence in e-coli must be an NGG sequence (N representing any base)
</P><P>To design our gRNA we used the target sequence online design tool DNA 2.0: https://www.dna20.com/eCommerce/cas9/input </P><P>Our gRNA target sequences were as follows:
</P><P> Despite our plans there was not sufficient time to complete a knockout using CRISPR and that will be our next step beyond the competition. </P></DIV><DIV id="view7"><DIV class="textTitle"><H4> References:</H4></DIV><DIV class="referenceBoring"><OL><LI>Wright, O., Stan, G.-B., and Ellis, T. (2013). Building-in biosafety for synthetic biology. (Review) <EM>Microbiology</EM>, <STRONG>159</STRONG>, 1221-1235. <A href="http://www.ncbi.nlm.nih.gov/pubmed/23519158">http://www.ncbi.nlm.nih.gov/pubmed/23519158</A></LI><LI>Okada, K., Minehira, M., and Zhu, X. (1997). The ispB gene encoding octaprenyl diphosphate synthase is essential for growth of Escherichia coli. <EM>Journal of Bacteriology</EM>, <STRONG>179</STRONG>, 3058–3060. <A href="http://www.ncbi.nlm.nih.gov/pubmed/9139929">http://www.ncbi.nlm.nih.gov/pubmed/9139929 </A></LI><LI>Søballe, B. , Poole, K. R. (1999). Microbial ubiquinones: multiple roles in respiration, gene regulation and oxidative stress management. (Review) <EM>Microbiology</EM>, <STRONG>145</STRONG>, 1817-1830. <A href="http://www.ncbi.nlm.nih.gov/pubmed/10463148">http://www.ncbi.nlm.nih.gov/pubmed/10463148 </A></LI><LI>Schmidt, M (2010). Xenobiology: A new form of life as the ultimate biosafety tool <EM>Bioessays</EM>, <STRONG>32</STRONG>, 322-331. <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2909387/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2909387/ </A></LI><LI>Malyshev, D.A., Dhami, K., Lavergne, T. et al. (2014). A semi-synthetic organism with an expanded genetic alphabet <EM>Nature</EM>, <STRONG>509</STRONG>, 385-388. <A href="http://www.nature.com/nature/journal/v509/n7500/full/nature13314.html">http://www.nature.com/nature/journal/v509/n7500/full/nature13314.html </A></LI></OL></DIV></DIV></DIV></DIV></DIV><DIV class="wrapper clearfix"><DIV class="FooterContent"><DIV id="FC11" style="font-size:120%"><H3 class="fontfixer" style="margin-top: 5px;">Contact Us</H3><P class="fontfixer">University College London  Gower Street - London  WC1E 6BTBiochemical Engineering Department<STRONG>Phone:</STRONG><A href="tel:phonenum" class="tele">+44 (0)20 7679 2000</A><STRONG>Email:</STRONG><A href="mailto:email@domain.com">ucligem2014@gmail.com</A></P></DIV><DIV id="FC22"><H3 class="fontfixer" style="margin-top: 5px;"><A href="https://2014.igem.org/Team:UCL/Humans/Sponsor" style="color: black;">Sponsors</A></H3></DIV><DIV id="FC33"><H3 class="fontfixer" style="margin-top: 5px;">Follow Us</H3><A class="twitter-timeline" href="https://twitter.com/UCLiGEM" data-widget-id="494172163977392128" height="235px" width="300px">Tweets by @UCLiGEM</A></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:UCL/Project/Xenobiology">http://2014.igem.org/Team:UCL/Project/Xenobiology</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UCL/Project/Xenobiology" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UCL/Project/Xenobiology" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UCL/Project/Xenobiology&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UCL/Project/Xenobiology&amp;oldid=389937" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>