"
Page
Discussion
View source
History
teams
Log in
 
Team:UCL/Science/Experiment
From 2014.igem.org
Goodbye Azodye UCL iGEM 2014
iGEM
HOME
PROJECT
SCIENCE
HUMANS
FAQ
UCL iGEM
Project
Science
Humans
FAQ
-.
jQuery UI Accordion - Default functionality
Stage 1
Stage 2
Stage 3
Stage 4
Stage 5
Stage 6
Stage 01: Extraction of Useful BioBrick Plasmids from iGEM 2014 Distribution Kit
Protocols   competent cells transformation miniprep digest gel
We began our project by identifying a range of BioBrick parts present in the iGEM 2014 distribution kit which we required as part of our cloning strategy. These parts primarily consisted of both constituitive and inducible promoter systems with ribosome binding sites which we could then use in conjunction with our azo-reductase BioBricks to assemble a functional azo dye degrading gene. We also decided that we would use the Red Florescent Protein expressing BioBrick as a control for any further transformation experiments. As the level of DNA present within each plate of the distribution kit is insufficient to perform digest and ligation reactions on it was necessary to transform each of these plasmids into our NEB5alpha competent cells. After growing our transformed cells overnight we then mini-prepped each of them to obtain BioBrick plasmids at suitable concentrations for future experiments.
Registry ID
Name / Function
Antibiotic Resistance
Source
Size
U
  BBa_K314103
 IPTG-inducible LacI Expression Cassette
 Chloramphenicol
 Spring 2014 BioBrick Distribution. Plate 1, Well 4D.
  1638 bp
T
  BBa_J04450
 RFP Coding Device
 Chloramphenicol
 Spring 2014 BioBrick Distribution. Plate 4, Well 4B.
  1069 bp
T
  BBa_R0010
 IPTG-inducible LacI Promoter
 Chloramphenicol
 Spring 2014 BioBrick Distribution. Plate 3, Well 4G.
  200 bp
T
  BBa_B0034
 Ribosomal Binding Site (RBS)
 Chloramphenicol
 Spring 2014 BioBrick Distribution. Plate 4, Well 1N.
  12 bp
T
  BBa_K518012
 RBS + RFP + Double Terminator
 Chloramphenicol
 Spring 2014 BioBrick Distribution. Plate 1, Well 18C.
  828 bp
N
  BBa_K206000
 pBAD Strong Promoter
 Chloramphenicol
 Spring 2014 BioBrick Distribution. Plate 3, Well 14A.
  130 bp
! N
  BBa_R0011
 LacI-Regulated, Lambda pL Hybrid Promoter
 Chloramphenicol
 Spring 2014 BioBrick Distribution. Plate 2, Well 6D.
  55 bp
! N
  BBa_B0012
 Transcription Terminator for E. coli RNA Polymerase
 Chloramphenicol
 Spring 2014 BioBrick Distribution. Plate 2, Well 2B.
  41 bp
Note: U = Used in experiments; T = Used for testing purposes but not for making BioBrick Devices; N = Transformed from Distribution Kits, but not used in experiments; ! = Problematic parts (see Parts Registry), were not used.
Stage 02: Identification of Useful Genes for Making New BioBricks
Identifying Azo-Dye Degrading Enzymes
Searching through the literature, we identified a number of bacterial species (including Bacillus subtilis and Pseudomonas sp.) that have proven to degrade azo dye compounds [1] [2] [3] [4] .
We contacted the Microbial & Enzyme Technology Lab led by Dr Lígia O. Martins at the Universidade Nova de Lisboa, who are currently researching how azo dye degrading enzymes function, and they were keen to collaborate with us on our project. They agreed to send us a set of five plasmids, each containing different genes encoding azo dye degrading enzymes from both B. subtilis and P. putida (including mutated forms found to exhibit enhanced degradation activity), for us to use in our investigations (see Table below).
Gene ID
Name / Function
Source
Size
Plasmid
  pAzoR
 FMN-dependent NADH-azoreductase 1
 Pseudomonas putida
 612 bp
 In expression vector: pET-21a (+) (ampicillin resistant) [2] [3] ,
initially cloned between NdeI and BamHI restriction sites.
  p1B6
 AzoR Heat-stable Mutant
 Pseudomonas putida
 612 bp
 In expression vector: pET-21a (+) (ampicillin resistant) [2] [3] ,
initially cloned between NdeI and BamHI restriction sites.
  pCotA
 Spore Coat Protein Laccase
 Bacillus subtilis
 1542 bp
 In expression vector: pET-21a (+) (ampicillin resistant (ampR)) [2] [3] ,
initially cloned between NheI and BamHI restriction sites.
  pBsDyP
 Dye Decolourising Peroxidase BSU38260
 Bacillus subtilis
 1251 bp
 In expression vector: pET-21a (+) (ampicillin resistant) [2] [3] ,
initially cloned between NdeI and BamHI restriction sites.
  pPpDyP
 Dye Decolourising Peroxidase PP_3248
 Pseudomonas putida
 864 bp
 In expression vector: pET-21a (+) (ampicillin resistant) [2] [3] ,
initially cloned between NdeI and BamHI restriction sites.
Extraction of B. Subtilis Genomic DNA
Protocols   DNA extraction
In the meantime, Helina (in our team), was able to obtain B. subtilis and P. aeruginosa strains for us to test whether we could retrieve azo dye degrading enzymes from their genomes, specifically, the azo-reductase gene (AzoR). This would be the first step for our first azoreductase BioBrick.
We extracted the genomic DNA from B. subtilis strain using a Promega Wizard Genomic DNA extraction kit so that we could subsequently amplify the azoreducatase gene (AzoR1) and create our first azoreductase BioBrick. After completing the genomic DNA extracton we ran a gel to show that we had successfully extracted the B. subtilis genomic DNA.
Stage 03: Transforming E. coli with Azo-Dye Degrading Plasmids from Lisbon
Transforming E. coli with Azo-Dye Degrading Plasmids
Protocols   competent cells transformation miniprep
The five azo dye degrading enzymes from Lisbon arrived as the respective genes in pET-21a (+) ampicillin resistant (ampR) expression vectors/plasmids (size: 5443 bp) [1] [2] . The DNA concentrations of these plasmids, however, were insufficient to perform PCR amplification, therefore we transformed each into our own E. coli competent cells (grown from NEB DH5α derivatives). After growing the cells overnight, we made bacterial glycerol stocks and miniprepped the cells to obtain plasmids at sufficient concentrations for further work.
Diagnostic Digest of Azo-Dye Degrading Plasmids
Protocols   digest gel
A diagnostic digest was performed to ascertain that these pET-21a (+) plasmids contained the gene we expected. As each plasmid possessed EcoRI and XbaI restriction sites close to the genes of interest, we performed double-digests using these recognition enzymes and predicted the digest fragments. The digestion products were visualised using gel electrophoresis (see image right).
Stage 04: Creation of Azo-Reductase BioBrick Parts from Plasmids
Protocols   analytical digest gel
After isolating our genes of interest we attempted to use PCR as a method of prefix and suffix generation to fit the BioBrick standard assembly parts format.
Achieving a successful PCR proved difficult; this may have been due to poor PCR reagent quality. We repeated the PCR using various polymerases (Taq, Phusion and Pfu) and also different dNTP mixes. Eventually, we succeeded in amplifying AzoR 1B6, BsDyP, and ispB asDNA with the required BioBrick Prefix and Suffix. Given the time-constraints, we did not succeed in also amplifying AzoR, CotA, and PpDyP with the Prefix and Suffix.
Protocols   digestion ligation transformation inoculation analytical digest gel
Our next step was to ligate these into the required pSB1C3 backbone. For BsDyP and ispB asDNA, this proved to be fairly straightforward, and quickly resulted in the production of our first new BioBricks: BBa_K1336003 for BsDyP+pSB1C3, and BBa_K1336005 for ispB+pSB1C3.
This was trickier for 1B6, as this gene possessed 2 illegal PstI restriction sites. Site-directed mutagenesis primers were designed to remove these sites, however, we could not completed this in time for submission. We did, however, succeed in performing a directionless ligation into pSB1C3. From here, we screened for plasmids with the correct orientation, and started our characterisation assays with this pseudo-BioBrick part.
Stage 05: Assembly of BioBrick Devices
We decided to assemble both BsDyP and ispB in a LacI cassette, inducible by IPTG.  Issues with inconclusive antibiotic effectivity led to major delays in construction of these composite parts. We first had to prove our antibiotics were functioning properly before making progress on our project.
We confirmed the construction of our BsDyP and ispB cassettes using analytical gel digest cutting at sites E and P.
Ideally, we wanted a Azo-Remediation Chassis (ARC), our BioBrick System, to be assembled as follows:
A further development on this prototype would be to have Bba_K1336000, the AzoR gene, to be inducibly transcribed by one promoter (say BBa_K314103, the LacI Expression Cassette) such that it is expressed in the reductive step of our azo dye remediation process. This would form a distinct BioBrick Device of "promotor A + AzoR + double terminator".
A secondary BioBrick Device (of "promoter B + gene 2 + double terminator") would follow this, where gene 2 would be one of our enzymes that function in the oxidative step of the azo dye remediation process, e.g. laccase or one of the dye decolourising peroxidases. A further tertiary BioBrick Device, with another oxidative enzyme would also be ideal. At least 2 oxidative enzymes are proposed as the enzymes are specialised for different substrates (as described in our BioBrick page. As always, time works against us, and we succeeded only in constructing 2 composite parts (BioBrick Devices): BBa_K1336007 for LacIEC+BsDyP+pSB1C3, and BBa_K1336006 for LacIEC+ispB+pSB1C3. This latter ispB asDNA device has functions in the biosafety aspects of our ARC.
Registry ID
Gene ID
Name / Function
Source
Size
Status
  BBa_K1336000
 AzoR
 FMN-dependent NADH-azoreductase 1
 Pseudomonas putida
  612 bp
 [ In Progress ]: primers designed
  BBa_K1336001
 1B6
 AzoR heat-stable mutant
 Pseudomonas putida
  612 bp
 [ In Progress ]: to remove 2 illegal PstI sites
  BBa_K1336002
 CotA
 Spore Coat Protein Laccase
 Bacillus subtilis
  1542 bp
 [ In Progress ]: primers designed
  BBa_K1336003
 BsDyP
 Dye Decolourising Peroxidase BSU38260
 Bacillus subtilis
  1251 bp
 [ New BioBrick Part ]: submitted
  BBa_K1336004
 PpDyP
 Dye Decolourising Peroxidase PP_3248
 Pseudomonas putida
  864 bp
 [ In Progress ]: primers designed
  BBa_K1336005
 ispB RNAi
 RNAi of Octaprenyl Diphosphate
Synthase fragment
 Escherichia coli, K12 strain
  562 bp
 [ New BioBrick Part ]: submitted
  BBa_K1336006
 LacIEC+ispB
 IPTG inducible ispB RNAi
 Escherichia coli, K12 strain
  2208 bp
 [ New BioBrick Device ]: submitted
  BBa_K1336007
 LacIEC+BsDyP
 IPTG inducible BsDyP
 Bacillus subtilis
  2895 bp
 [ New BioBrick Device ]: submitted
  BBa_K729006
 CueO
 Laccase
 Escherichia coli
  1612 bp
 [ In Progress ]: ascertaining identity
(
)
  BBa_K500000
 LiP
 Lignin Peroxidase
 Phanerochaete chrysosporium
  1116 bp
 [ Improved Characterisation ]: toxicity issues in gene synthesis.
 [ In Progress ]: to subclone into pSB1C3/pSB3C5.
  BBa_K729004
 nucB
 Extracellular nuclease
 Staphylococcus aureus
  561 bp
 [ Improved Function ]
Stage 06: Characterisation Assays of BioBrick Device(s)
Results and protocols for the characterisation pages can be found here
Contact Us
University College London
Gower Street - London
WC1E 6BT
Biochemical Engineering Department
Phone: +44 (0)20 7679 2000
Email: ucligem2014@gmail.com
Sponsors
Follow Us
Tweets by @UCLiGEM
Retrieved from " http://2014.igem.org/Team:UCL/Science/Experiment "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
