<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SUSTC-Shenzhen_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SUSTC-Shenzhen/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SUSTC-Shenzhen/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SUSTC-Shenzhen/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SUSTC-Shenzhen/Modeling&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SUSTC-Shenzhen/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SUSTC-Shenzhen/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Team SUSTC-Shenzhen</TITLE></P><DIV class="container"><NAV class="navbar navbar-default navbar-fixed-top" role="navigation" id="index-nav-main"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="/Team:SUSTC-Shenzhen">Home</A></LI><LI><A href="/Team:SUSTC-Shenzhen/Project">Project</A></LI><LI><A href="/Team:SUSTC-Shenzhen/Notebook">Notebook</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">More</A></LI><LI><A href="/Team:SUSTC-Shenzhen/JudgingForm">Judging Form</A></LI><LI><A href="/Team:SUSTC-Shenzhen/Attributions">Attributions</A></LI></UL><LI><A href="https://igem.org/Team.cgi?id=1431">Team Profile</A></LI><UL class="nav navbar-nav navbar-right"><LI><FORM action="/Special:Search" id="searchform" class="navbar-form" role="search"><BUTTON type="submit" name="go" class="btn btn-default" id="mw-searchButton" value="Go" title="Search the pages for this text">Search</BUTTON></FORM></LI><LI><A id="nav-login-status">Log in</A></LI></UL></DIV></NAV></DIV><DIV class="bs-docs-header"><DIV class="container"><H1>

Modeling

        </H1><P>

The art of mathematics, blended with practice.

        </P></DIV></DIV><DIV id="main-content" class="container"><DIV class="col-md-3" id="document-sidebar"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Introduction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Derivation_of_the_model"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Derivation of the model</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Theoretical_analysis"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Theoretical analysis</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Symbol_description"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Symbol description</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Formula_derivation"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Formula derivation</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-6"><A href="#Matlab_analysis"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Matlab analysis</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Condition_1"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Condition 1</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Condition_2"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Condition 2</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Condition_3"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Condition 3</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Matlab_code"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">Matlab code</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Results"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Reference"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Reference</SPAN></A></LI></TD></TR></TBODY></TABLE></DIV><DIV class="col-md-9" id="document-body"><H2><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H2><P>The project is intended to treat retrovirus diseases. And we take AIDS as an example.Currently, no effective HIV vaccine exists. In this project, we want to establish a more effective HIV-curing system with less side-effects by integrating CRISPR/Cas9 system into human hemopoietic stem cells, which aims to protect the helper T cells from virus infection.
</P><P>So we want to know how effective the CRISPR/Cas9 system is for HIV. In this model, we discuss about dynamic changes of different cells in a person’s body using Matlab and we can see how the system works. So this model will help us to forecast the future applications of our project.
</P><P>In our model, we can see the influence the CRISPR/Cas system produces by the change of virus and cells. But there are too many parameters and that makes it impossible to get clear relation of these parameters easily. If the environment doesn't change, we can stimulate the situation in Matlab for a short time.
</P><H2><SPAN class="mw-headline" id="Derivation_of_the_model">Derivation of the model</SPAN></H2><H3><SPAN class="mw-headline" id="Theoretical_analysis">Theoretical analysis</SPAN></H3><P>First, stably transfect hematopoietic stem cells with the plasmid which encodes Cas9 protein. To make the question easier, we suppose that the efficiency is constant. As is shown in other part in this project, second, we can construct a plasmid which encodes the gRNA sequence which is a 20bp conserved regions in HIV DNA and has no conservation in human genome. When the human body is infected by AIDS, using non-viral DNA transfecting methods (in our project we choose A-B toxin based shuttle) to deliver the plasmid into human somatic cells (especially CD4+ cells) and the CRISPR/Cas system is activated. The Cas9 protein recognizes viral DNA with the help of gRNA and cut the target sequence, causing double strand break (DSB). The DSB is then repaired by the non-homologous end joining mechanism in the cell but with high error rate (in this model we suppose the rate doesn’t change), causing frame-shift mutation and gene knockout. The viral protein synthesis is stopped and further infection is impossible, but it provides potentiation for eradicating AIDS. 
</P><CENTER>Fig1 the relationship between HIV copies and CD4 counts over the average course of untreated HIV infection </CENTER><P>
Fig 1 is a generalized graph of the relationship between HIV copies (viral load) and CD4 counts over the average course of untreated HIV infection from Wikipedia. Because any particular individual's disease course may vary considerably.
</P><H3><SPAN class="mw-headline" id="Symbol_description">Symbol description</SPAN></H3><P>\(M_{thymus}\)    the maximum number of immunological cells
</P><P>\(N_{thymus}\)    the number of immunological cells
</P><P>\( N_h\) high resistant immune cells number
</P><P>\( N_l\) low resistant immune cells number
</P><P>\( P_{thymus}\)    the enviroment pressure for immune cell growth
</P><P>\(P_{hiv}\)    almost no self-pressure for hiv virus
</P><P>\( f_{ht}\)    the hiv-caused the number of immunological cells decreasing factor
</P><P>\( f_{hlh}\)    the high and low immune cell induced increasing factor
</P><P>\(vg_{tg}\)    the self-generation rate of stem cell
</P><P>\(vg_{sh}\)    the self-generation rate of hiv virus
</P><P>\(f_m\)    the mutation factor
</P><P>\(f_{hr}\)    the high resistance factor
</P><P>\(f_{lr}\)    the low resistance factor
</P><P>\(M_{hiv}\)    the possible total hiv counting, can be set to be infinity
</P><P>\(N_{hiv}\)    the number of hiv virus
</P><P>\(vg_{th}\)    the generation rate by the number of immunological cells
</P><P>\(vd_{td}\)    the decay(death) rate from high resistant T cell to low ones
</P><P>\(vg_{lh}\)    the gain rate from low resistant T cell to high ones
</P><P>\(vg_{tl}\)    generation rate from the number of immunological cells
</P><P>\(vd_{hl}\)    the death rate caused by hiv virus
</P><P>\(vg_{gt}\)    the gain rate by the immune decaying of high resistant T cells
</P><P>\(r\)   random number between 0 and 1
</P><H3><SPAN class="mw-headline" id="Formula_derivation">Formula derivation</SPAN></H3><P>We suppose that a person was infected by HIV for some time. And the environment doesn't change in our model. Because the existence of CRISPR/Cas9 system and other environment effects, we can roughly divide the immunological cells into two sides. One is with high resistance, the other is with low resistance.
</P><P>English letters express variables, while Greek letters express parameters.
</P><P>HIV infects vital cells in the human immune system such as helper T cells (specifically CD4+ T cells), macrophages, and so on. HIV virus will attract T cells, so we can suppose that there is a linear relationship between the decreasing rate and the number of HIV and we will get equation (1) as follow.
</P><P>
\begin{equation}
\frac{{\rm d} N^{(1)}_{thymus}}{{\rm d} t}=-\alpha_1 N_{hiv}
\end{equation}

Some kinds of immunological cells can replicate, so the increasing rate of immunological cells is influenced by the number of immunological cells. So we will get equation (2) as follow.

\begin{equation}
\frac{{\rm d} N^{(2)}_{thymus}}{{\rm d} t}=\alpha_2 N_h +\alpha_3 N_l
\end{equation}

T cells as hosts can boost the increasing of HIV, so we can get equation (3).

\begin{equation}
\frac{{\rm d} N_{hiv}}{{\rm d} t}=\beta_1 f_m(\beta_2 N_l+\beta_3 N_h)
\end{equation}

High resistance ones can decrease the number of HIV virus, which implies equation (4).

\begin{equation}
\frac{{\rm d} N_{hiv}}{{\rm d} t}=\epsilon(1-f_m)N_{hiv}N_h
\end{equation}

Of course, high resistance ones increase due to T cells' increasing.

\begin{equation}
\frac{{\rm d} N^{(1)}_{h}}{{\rm d} t}=\delta N_{thymus}
\end{equation}

HIV virus caused the high ones increasing slower than normal situation.

\begin{equation}
\frac{{\rm d} N^{(2)}_{h}}{{\rm d} t}=\gamma_1(1-\gamma_2 N_{hiv})
\end{equation}

If we combine equation (5) and equation (6), we can get equation (7) as follows. 

\begin{equation}
\frac{{\rm d} N_{h}}{{\rm d} t}=\delta N_{thymus}+\gamma_1(1-\gamma_2 N_{hiv})
\end{equation}

Immunity decay rate for high ones due to mutation, which we can express by equation (9).

\begin{equation}
\frac{{\rm d} vd_{td}}{{\rm d} t}=\eta N_h N_{hiv}e^{\varepsilon f_{hr}}
\end{equation}

The number of high ones increase due to low ones that enhance the immune capacity.

\begin{equation}
\frac{{\rm d} N_h}{{\rm d} t}=\zeta N_l e^{ f_{lr}}
\end{equation}

Because low ones increase due to the number of immunological cells, we can get mathematic relation between the two variables.

\begin{equation}
\frac{{\rm d} N_l}{{\rm d} t}=\iota N_{thymus}
\end{equation}

HIV virus caused the high ones increasing slower than normal.

\begin{equation}
\frac{{\rm d} N_h}{{\rm d} t}=1-\kappa N_{hiv}
\end{equation}

Cells will die after HIV's attack. death formula for low ones due to HIV virus is as follow.

\begin{equation}
\frac{{\rm d} N_l}{{\rm d} t}=\lambda N_l N_{hiv} e^{\mu f_{lr}}
\end{equation}

</P><P>The values of parameters will depend on whether there is CRISPR/Cas9 system in the body. And we can change the values of parameters on different environments.
</P><P>\(N_{thymus}\) changes with the change of time \(t \).

\begin{equation}
N_{thymus}=\int_{0}^{t}[vg_{tg}M_{thymus}(1-\frac{P_{thymus}N_{thymus}}{M_{thymus}})+f_{ht}\alpha_1 N_{hiv}+f_{hlh}\alpha_2 N_h +\alpha_3 N_l]{{\rm d} t}
\end{equation}

\(N_{hiv}\) changes with the change of time \(t \).

\begin{equation}
N_{hiv}=\int_{0}^{t}[vg_{sh}N_{hiv}(1-\frac {N_{hiv}}{M_{hiv}}P_{hiv})\beta_1 f_m(\beta_2 N_l+\beta_3 N_h))-\epsilon(1-f_m)N_{hiv}N_h]{{\rm d} t}
\end{equation}

\(N_h\) changes with the change of time \(t \).

\begin{equation}
N_h=\int_{0}^{t}[vg_{th}\delta N_{thymus}\gamma_1(1-\gamma_2 N_{hiv})-vd_{td}\eta N_h N_{hiv}e^{\varepsilon f_hr}+vg_{lh}\zeta N_l e^{ f_{lr}}] {{\rm d} t}
\end{equation}
\(N_l\) changes with the change of time \(t \).

\begin{equation}
N_l=\int_{0}^{t}[vg_{tl}\iota N_{thymus}(1-\kappa N_{hiv})-vd_{hl}\lambda N_l N_{hiv} e^{\mu f_{lr}}]{{\rm d} t}
\end{equation}

\(f_m\) is also a variable and also has certain randomness.

\begin{equation}
f_m=\xi_1(\frac{N_h}{N_h+N_l}+\xi_2r)
\end{equation}

</P><H2><SPAN class="mw-headline" id="Matlab_analysis">Matlab analysis </SPAN></H2><P>After long period of being infected by HIV, the virus persists at a relatively stable level and the CD4+ T-cell count gradually falls. This period, during which the patient is asymptomatic, can last many years. In the final stages of the disease, the viral load usually increases rapidly and the CD4+ T-cell count decreases to a point at which the immune system fails to provide protection against opportunistic infections. Death usually occurs due to one of these infections.
</P><P>According to derivation of the model above, we can get the approximate results as follows by using Matlab on different conditions.
CRISPR/Cas9 system is introduced into the body, everything will change.
</P><P>For different conditions, we change the values of parameters to suit a given situation. Here are some examples.
</P><H3><SPAN class="mw-headline" id="Condition_1">Condition 1</SPAN></H3><P>In conditon 1, CRISPR/Cas9 system is introduced into the body at time of &quot;200&quot;. As for Fig2.1 to Fig 2.5, the parameters are same.
</P><CENTER>Fig2.1 High-resistance T cells counting versus Time </CENTER><P>As is shown in Fig2.1, after the CRISPR/Cas9 system introduced in the body, high-resistance T cells appear and increase rapidly.
</P><CENTER>Fig2.2 Low-resistance T cells counting versus Time </CENTER><P>Fig2.2 shows the low-resistance T cells counting versus time, we can find that low-resistance T cells also increase after the CRISPR/Cas9 system introduced in the body, but it's much slower than that of high-resistance T cells.
</P><CENTER>Fig2.3 Low-resistance T cell counting versus Time (partial enlarged view) </CENTER><P>Fig2.3 is partial enlarged view of Fig 2.3. It can help us understand the change about the time of &quot;200&quot;. 
</P><CENTER>Fig2.4 HIV counting versus Time </CENTER><P>Fig2.4 can roughly describe of change of HIV virus' number. After the CRISPR/Cas9 system introduced in the body, HIV virus decrease.
</P><CENTER>Fig2.5 HIV counting versus Time (partial enlarged view) </CENTER><P>Fig2.5 is partial enlarged view of Fig 2.4. It can help us understand the change.
</P><H3><SPAN class="mw-headline" id="Condition_2">Condition 2</SPAN></H3><P>Under condition 2, there is no CRISPR/Cas9 system. And the patient didn't take actions to compete with HIV.
</P><CENTER>Fig 3. HIV counting versus Time  </CENTER><P>As is shown by the Fig 3., the HIV will reach a very high value and the patient will be very dangerous without the outside help.
</P><H3><SPAN class="mw-headline" id="Condition_3">Condition 3</SPAN></H3><CENTER>Fig 4.1   </CENTER><CENTER>Fig 4.2  </CENTER><P>All parameters in Fig 4.1 and Fig 4.2 are the same. The only different between at first is the value of low-resistance T cells counting. If the number of low-resistance T cells is too small, it may increase for a while.
</P><H3><SPAN class="mw-headline" id="Matlab_code">Matlab code</SPAN></H3><P>A simple Matlab code example: <A href="https://2014.igem.org/Team:SUSTC-Shenzhen/Modeling/matlabcode" class="external free" rel="nofollow">https://2014.igem.org/Team:SUSTC-Shenzhen/Modeling/matlabcode</A></P><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><P>The relationship between cells and HIV is too complex that we can't make a model to fit out each situations. But the existence of the parameters enable us to handle most of the situations. If the environment doesn't change, we can stimulate the situation in Matlab for a short time. The result is that the CRISPR/Cas9 system will take effect. A small change may cause the butterfly effect.
</P><H2><SPAN class="mw-headline" id="Reference">Reference</SPAN></H2><P>Berezovskaya, F. S., Wolf, Y. I., Koonin, E. V., &amp; Karev, G. P. (2014). Pseudo-chaotic oscillations in CRISPR-virus coevolution predicted by bifurcation analysis. Biology direct, 9(1), 13.
</P><P>Cunningham, A. L., Donaghy, H., Harman, A. N., Kim, M., &amp; Turville, S. G. (2010). Manipulation of dendritic cell function by viruses. Current opinion in microbiology, 13(4), 524-529.
</P><P>Cong, L., Ran, F. A., Cox, D., Lin, S. L., Barretto, R., Habib, N., . . . Zhang, F. (2013). Multiplex Genome Engineering Using CRISPR/Cas Systems. Science, 339(6121), 819-823. 
</P><P>Mittler, J. E., Sulzer, B., Neumann, A. U., &amp; Perelson, A. S. (1998). Influence of delayed viral production on viral dynamics in HIV-1 infected patients. Mathematical biosciences, 152(2), 143-163.
</P><P>&quot;HIV&quot; [Online] Available: <A href="http://en.wikipedia.org/wiki/HIV" class="external free" rel="nofollow">http://en.wikipedia.org/wiki/HIV</A> (Sep 17, 2014)
</P></DIV></DIV><FOOTER class="bs-docs-footer col-md-12" role="contentinfo"><DIV class="container"><P>Maintained by the iGEM team SUSTC-Shenzhen.</P><P>Licensed under <A href="http://creativecommons.org/licenses/by/4.0/">CC BY 4.0</A>.</P><UL class="bs-docs-footer-links muted"><LI>©2014 SUSTC-Shenzhen</LI></UL></DIV></FOOTER><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:SUSTC-Shenzhen/Modeling">http://2014.igem.org/Team:SUSTC-Shenzhen/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SUSTC-Shenzhen/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SUSTC-Shenzhen/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SUSTC-Shenzhen/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SUSTC-Shenzhen/Modeling&amp;oldid=401699" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>