<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SUSTC-Shenzhen_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SUSTC-Shenzhen/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SUSTC-Shenzhen/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SUSTC-Shenzhen/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SUSTC-Shenzhen/Project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SUSTC-Shenzhen/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SUSTC-Shenzhen/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Team SUSTC-Shenzhen</TITLE></P><DIV class="container"><NAV class="navbar navbar-default navbar-fixed-top" role="navigation" id="index-nav-main"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="/Team:SUSTC-Shenzhen">Home</A></LI><LI><A href="/Team:SUSTC-Shenzhen/Project">Project</A></LI><LI><A href="/Team:SUSTC-Shenzhen/Notebook">Notebook</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">More</A></LI><LI><A href="/Team:SUSTC-Shenzhen/JudgingForm">Judging Form</A></LI><LI><A href="/Team:SUSTC-Shenzhen/Attributions">Attributions</A></LI></UL><LI><A href="https://igem.org/Team.cgi?id=1431">Team Profile</A></LI><UL class="nav navbar-nav navbar-right"><LI><FORM action="/Special:Search" id="searchform" class="navbar-form" role="search"><BUTTON type="submit" name="go" class="btn btn-default" id="mw-searchButton" value="Go" title="Search the pages for this text">Search</BUTTON></FORM></LI><LI><A id="nav-login-status">Log in</A></LI></UL></DIV></NAV></DIV><DIV class="bs-docs-header"><DIV class="container"><H1>

Project Description

        </H1><P>

a small overview to the whole big ideas

        </P></DIV></DIV><DIV id="main-content" class="container"><DIV class="col-md-3" id="document-sidebar"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Introduction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Background"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#HIV_.28Human_immunodeficiency_virus.29"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">HIV (Human immunodeficiency virus)</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#CRISPR.2FCas_system"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">CRISPR/Cas system</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#PiggyBac_.28PB.29_transposon_system"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">PiggyBac (PB) transposon system</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#A-B_toxin"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">A-B toxin</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-7"><A href="#Experiment_design"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Experiment design</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Constructing_EGFP-Cas9_expressing_cell_lines"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Constructing EGFP-Cas9 expressing cell lines</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Transfecting_the_cell_line_with_gRNA-encoding_plasmid"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Transfecting the cell line with gRNA-encoding plasmid</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Deliver_gRNA_plasmids_by_A-B_toxin-based_shuttle"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Deliver gRNA plasmids by A-B toxin-based shuttle</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#gRNA_design"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">gRNA design</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Modeling"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Modeling</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Safety"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Safety</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#Future_Perspectives"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Future Perspectives</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#In_vivo_establishment_and_validation_of_the_system"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">In vivo establishment and validation of the system</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Permanent_implanting_Cas9_gene_into_hematopoietic_stem_cells"><SPAN class="tocnumber">7.2</SPAN><SPAN class="toctext">Permanent implanting Cas9 gene into hematopoietic stem cells</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Using_CRISPR.2FCas_system_to_integrate_Cas9_into_human_cells"><SPAN class="tocnumber">7.3</SPAN><SPAN class="toctext">Using CRISPR/Cas system to integrate Cas9 into human cells</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Using_HIV-modified_vector_to_specifically_deliver_Cas9_gene_or_gRNA_plasmid_into_CD4.2B_cells"><SPAN class="tocnumber">7.4</SPAN><SPAN class="toctext">Using HIV-modified vector to specifically deliver Cas9 gene or gRNA plasmid into CD4+ cells</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#Application_in_other_retroviral_diseases"><SPAN class="tocnumber">7.5</SPAN><SPAN class="toctext">Application in other retroviral diseases</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#References"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE></DIV><DIV class="col-md-9" id="document-body"><CENTER><FONT size="6"><B>Human CRISPR/Cas system against HIV</B></FONT></CENTER><CENTER><FONT size="6"><B>with gRNA delivered by </B></FONT></CENTER><CENTER><FONT size="6"><B>A-B toxin-based shuttle</B></FONT></CENTER><CENTER><B>SUSTC-Shenzhen, iGEM 2014</B></CENTER><H1><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H1><P>In this project, we intended to establish a synthetic biology-based effective HIV-curing system with less side effects. The key goal is to integrate CRISPR/Cas system into human blood system to protect CD4+ cells against viral infection. The gRNA is designed to target the relative conserved regions in HIV viral genome and inactivate its biological activity. Since viral vectors seem to be of limited use in gene therapy strategies (e.g., potential pathogenicity), there is in dire need of a simple, efficient system for cell-specific delivery of nucleic acids. By using non-viral DNA delivery system such as A-B-toxin-GAL4 fusion protein, we can deliver plasmids encoding gRNA into the CD4+ cells and readily attack multiple HIV genome sites simultaneously with the most up-to-date knowledge of HIV epidemic.
</P><H1><SPAN class="mw-headline" id="Background">Background</SPAN></H1><H2><SPAN class="mw-headline" id="HIV_.28Human_immunodeficiency_virus.29">HIV (Human immunodeficiency virus)</SPAN></H2><P><B>HIV</B> is the trigger of the lethal disease <B>AIDS</B>. It replicates in and kills <B>T cells</B>, weakens human immune system and allows life-threatening opportunistic infection and cancer to thrive. HIV infects vital cells in the human immune system such as helper T cells (especially CD4+ T cells), macrophages, and dendritic cells. As a retrovirus, HIV reverse transcribes its genome and integrated it into the host cell genome after infection. Figure 1 illustrates the schematic structure of HIV virus (Source: Wikipedia).
</P><CENTER>Figure 1. Schematic demonstration of HIV</CENTER><P>Traditional therapies such as drug treatment can control the development of the disease but not eradicate the virus. The chronic inflammation and immune dysfunction caused by long-term chemotherapy and radiotherapy may also lead to non-AIDS morbidity and mortality (Kiem, Jerome, Deeks, &amp; McCune, 2012). The process of reverse transcription is extremely error-prone, and the resulting mutations may lead to drug resistance or allow the virus to evade the body's immune system.
</P><H2><SPAN class="mw-headline" id="CRISPR.2FCas_system">CRISPR/Cas system</SPAN></H2><P><B>CRISPR/Cas</B> (Clustered regularly interspaced short palindromic repeats/CRISPR-associated system) is originally a self-protecting mechanism in bacteria against external DNA such as virus genome or plasmids. It uses <B>CAS complex</B> to <B>cut the external viral or plasmid DNA</B> and integrating a short DNA segment into the CRISPR loci in the bacteria genome. This short DNA segment will be transcribed into pre-crRNA and bind to another CAS endonuclease called Cas9 with the help of tracrRNA. The Cas9 endonuclease cuts the external DNA in the existence of not only crRNA but also a special recognizing sequence at the 3â end of the target DNA called PAM (Protospacer Adjacent Motif). When the same external DNA invades next time, the Cas9-tracrRNA-crRNA complex will recognize and cut it in order to destroy its biological activity. 
Figure 2 illustrates the mechanism of Type II CRISPR/Cas system in bacteria (Mali, Esvelt, &amp; Church, 2013). The laboratories of Dr. George Church (Mali, Yang, et al., 2013) and Dr. Feng Zhang (Cong et al., 2013) have successfully developed the Type II CRISPR-Cas system for human cells, which lay the foundation for this project.
</P><CENTER>Figure 2. Mechanism of Type II CRISPR/Cas system in bacteria</CENTER><H2><SPAN class="mw-headline" id="PiggyBac_.28PB.29_transposon_system">PiggyBac (PB) transposon system</SPAN></H2><P><B>PiggyBac (PB) transposon</B> is a mobile genetic element that <B>efficiently transposes between vectors and chromosomes</B> via a <B>&quot;cut and paste&quot;</B> mechanism but leave no <B>âfootprintâ</B>. PB transposase in this system recognizes the transposon-specific inverted terminal repeat sequence (PB5 &amp; PB3 in Figure 5) located on both end of transposon vector and efficiently moves the contents from the original sites and efficiently integrates them into TTAA chromosomal sites. Figure 4 illustrates the working mechanism of PB transposon system (Source: Wikipedia; Author: Transposagenbio).
</P><CENTER>Figure 3. Mechanism of PiggyBac transposon system</CENTER><H2><SPAN class="mw-headline" id="A-B_toxin">A-B toxin</SPAN></H2><P>A-B toxinare two-component exotoxin secreted by a number of pathogenic bacteria. The complexes contain two subunits called A and B. A subunit is the active portion that is poisonous to host cells. B subunit has a translocation domain and cell receptor binding domain, which <B>helps the A subunit going into the cell through endosomes</B>. Figure 4 shows how Anthrax toxin (a kind of A-B toxin) enters into human cells (Source: Wikipedia; Author: Y tambe).
</P><P><A href="https://2014.igem.org/Team:SUSTC-Shenzhen/Notebook/HeLaCell/Stably_transfect_cell_with_target-sequence_EGFP" class="external free" rel="nofollow">https://2014.igem.org/Team:SUSTC-Shenzhen/Notebook/HeLaCell/Stably_transfect_cell_with_target-sequence_EGFP</A></P><CENTER>Figure 4. Model of Anthrax toxin entering into human cells</CENTER><H1><SPAN class="mw-headline" id="Experiment_design">Experiment design</SPAN></H1><P><A href="/Team:SUSTC-Shenzhen/Project/Plasmid" class="btn btn-success btn-lg" style="color:#fff">See the full list of plasmids we used.</A></P><H2><SPAN class="mw-headline" id="Constructing_EGFP-Cas9_expressing_cell_lines">Constructing EGFP-Cas9 expressing cell lines</SPAN></H2><P>The program aims to endow the human cell the capacity of resisting the infection of HIV by <B>introducing the CRISPR/Cas9 system into the human cell</B>. To test the efficiency of the Cas9 and explore the optimal conditions of the system, we need a reporter to indicate whether the Cas9 functions and the targeted sites are cut off and destructed. So a cell line integrated with EGFP and Cas9 gene is firstly constructed in a similar way. We use PiggyBac transposon to transfect Hela cell for <B>permanently expressing Cas9 protein</B>. Once the proper gRNA is delivered into these cells, Cas9 binds to the gRNA and start to cut the complementary DNA strand. The vector design is based on Feng Zhang labâs and Wei Huang labâs plasmids.
</P><CENTER>Figure 5. PB-TetON-Cas9 plasmid</CENTER><CENTER>Figure 6. PB-EGFP plasmid</CENTER><P><A href="/Team:SUSTC-Shenzhen/Project/Cell_Line_Construction" class="btn btn-success btn-lg" style="color:#fff">Result Page for Cell Line Construction</A></P><H2><SPAN class="mw-headline" id="Transfecting_the_cell_line_with_gRNA-encoding_plasmid">Transfecting the cell line with gRNA-encoding plasmid</SPAN></H2><P>Cas9 uses gRNA, a 23nt short RNA fragment which binds to its complementary target and 'guide' Cas9 to cut the target DNA. We first check the efficiency of this system. We construct a plasmid based on Zhang Feng Labâs pX330 plasmid that encodes a chimeric guide RNA scaffold and mCherry reporter gene followed by a NLS (nucleus leading sequence). It also contains 0, 2, 5 or 7 repeats of UAS sequences which are necessary for the vehicleâs recognition and binding (See Future applications â A-B toxin based gRNA shuttle for details).
</P><CENTER>Figure 7. Constructed plasmid to express gRNA in host cells</CENTER><P>The chimeric guide RNA scaffold is used to insert gRNA. We acquired the gRNA for EGFP from Wei Huangâs lab and put it into our constructed plasmid. Figure 8 shows the design of chimeric guide RNA scaffold (Cong et al., 2013).
</P><CENTER>Figure 8. Mechanism of gRNA insertion into pX330-derived plasmid backbone</CENTER><P>In this way, with the ratio of reduced green fluorescence and the quantity of red fluorescence we can roughly obtain the efficiency of the CRISPR-Cas system we created. And finally, we reconstruct the plasmid but this time with the gRNA targeting HIV and test its efficacy.
</P><P><A href="/Team:SUSTC-Shenzhen/Project/gRNA-mCherry-UAS_Carried_Plasmids_Design_and_Construction" class="btn btn-success btn-lg" style="color:#fff">See how we design and construct our powerful gRNA-mCherry-UAS plasmids</A></P><H2><SPAN class="mw-headline" id="Deliver_gRNA_plasmids_by_A-B_toxin-based_shuttle">Deliver gRNA plasmids by A-B toxin-based shuttle</SPAN></H2><P>CRISPR/Cas system needs two essential parts for normal function: Cas9 and gRNA. Traditional methods usually transfect the cell with Cas9 and gRNA together (Hu et al., 2014). However, this limits the usage of this system because HIV has high mutated rate, which needs to update gRNA. So we decided to use a flexible nucleic acid delivery system. 
We choose a non-viral DNA delivery system which is based on a modified A-B toxin. The chimeric fusion protein mainly comprises 3 parts: target cell-specific binding domain, a translocation domain and a nucleic acid binding domain. The target cell-specific binding domain recognizes the EGF (epidermal growth factor) receptors on the cell surface. The translocation domain enhances nucleic acid escape from the cellular vesicle system and thus to augment nucleic acid transfer. The nucleic acid binding domain, which derives from the yeast GAL4 transcription factor, can carry plasmids with UAS (Upstream Activation Sequence) sequences into cells in vivo. Several research groups (Gaur, Gupta, Goyal, Wels, &amp; Singh, 2002) (Chen et al., 2000) (Fominaya, Uherek, &amp; Wels, 1998) have achieved this goal. Prof. Wels was kindly enough to provide his TEG vehicle for us to deliver DNA into Hela for this project. Figure 7 shows the schematic representation of the TEG fusion gene in the E.coli expression plasmid pWF47-TEG (Wels, Winfried 63110 Rodgau (DE)). A-B toxin delivery system for various human immune cells will need to be developed in the future. 
</P><CENTER>Figure 9. Schematic representation of the TEG fusion gene</CENTER><CENTER>Figure 10. Schematic representation of the TEG fusion gene</CENTER><P><A href="/Team:SUSTC-Shenzhen/Project/A-B_toxin" class="btn btn-success btn-lg" style="color:#fff">See how we get our A-B toxin-based shuttle</A></P><H1><SPAN class="mw-headline" id="gRNA_design">gRNA design</SPAN></H1><P>gRNA is a 23nt short RNA fragment (not include tracrRNA) which binds to its complementary target and 'guide' Cas9 to cut the target DNA. Since we would like to destroy the HIV virus inside human body, a gRNA sequence that matched part of HIV conserved region but have few off-target matches in the human genome is desired. However, designing a gRNA for a human-infective virus is difficult due to the very large difference in the genome size. For virus like HIV, it is even more difficult because HIV can mutate in a very high rate. So we change our goal to finding a group of specific quasi-conservative sequences (gRNA) which are able to target one or more species of HIV.
</P><P>Our search for the sequence roughly followed the process described in George M. Church's paper (Mali, Yang, et al., 2013). We modified it slightly to fit our own purpose. We first used bioinformatics method to find the quasi-conservative regions in the HIV-1 whole genome reference library by NIH. From the candidates, we selected the region around 720bp from beginning of the genome (aligned), in the less-selected region slightly off the LTR. We then used the online tools of Feng Zhang's Lab at MIT to find our desired gRNA composition. We also did some calculation based on phenomenological energy calculation to estimate the stability and effectiveness of our gRNA sequences (Hsu et al., 2013). The tool shows that almost no off-target binding will occur. BLAST was used to further confirm the results. We also analyzed the structure of the resulted gRNA, which shows an approximate free energy of -1.4kJ (Zuker, 2003). For detailed information, see our gRNA design page.
</P><P><A href="/Team:SUSTC-Shenzhen/gRNA_Design" class="btn btn-success btn-lg" style="color:#fff">See how we design our gRNA.</A></P><H1><SPAN class="mw-headline" id="Modeling">Modeling</SPAN></H1><P>Our project is intended to treat retrovirus diseases.So we want to know how effective the CRISPR/Cas9 system is. In our model, we discuss about dynamic changes of different cells in a personâs body using Matlab and we take HIV for exmple.
After theoretical analysis, symbol description, formula derivation and Matlab analysis, we can see how the system works and the influence the CRISPR/Cas system produces by the change of virus and cells in this model. It will help us to forecast the future applications of our project.If the environment doesn't change, we can stimulate the situation in Matlab for a short time. The result is that the CRISPR/Cas9 system will take effect.
</P><CENTER>Figure 12. The relationship between HIV copies and CD4 counts over the average course of untreated HIV infection </CENTER><P><A href="/Team:SUSTC-Shenzhen/Modeling" class="btn btn-success btn-lg" style="color:#fff">See how we do the modeling.</A></P><H1><SPAN class="mw-headline" id="Safety">Safety</SPAN></H1><P>Our system has several built-in design to reduce the potential safety risks of the system utilization.
</P><P><A href="/Team:SUSTC-Shenzhen/Safety" class="btn btn-success btn-lg" style="color:#fff">See how we concerned about the Safety.</A></P><H1><SPAN class="mw-headline" id="Future_Perspectives">Future Perspectives</SPAN></H1><H2><SPAN class="mw-headline" id="In_vivo_establishment_and_validation_of_the_system">In vivo establishment and validation of the system</SPAN></H2><P>Currently all our experiment design is based on in vitro conditions. However, our ultimate goal is to fight against HIV in vivo. A recent published paper in PNAS, sharing the similar design with our project, presented strong evidence for the effect of this system in other cell lines (Hu et al., 2014). So one of our important further works is to establish an in vivo system to test and improve our design. We plan to transplant the engineered Hela cell to immune deficient mice such as Nude or NOD/SCID mice. We will then optimize the dynamics of inducing of Cas9 of the transplanted cell by feeding the mice with Dox.  The effect can be monitored in live mice in real time is we conjugate a luciferase gene to Cas9 gene and imaging its activate in the live mice. We will then optimize the delivery method and dosage of the A-B toxin-gRNA plasmid complex and monitor the changing in GFP expression of transplant cell using either biopsy or intravital fluorescent imaging.
</P><H2><SPAN class="mw-headline" id="Permanent_implanting_Cas9_gene_into_hematopoietic_stem_cells">Permanent implanting Cas9 gene into hematopoietic stem cells</SPAN></H2><P>Hematopoietic stem cells are the progenitor of all blood cells, including CD4+ T/B cells. Itâs very difficult to extract all CD4+ T/B cells from the human blood system, doing the gene modification and then sent them back. Also, these cells canât replicate, which means regularly gene modification should be done, which is unrealistic. Instead, one of our team members called Lin Le suggested to modify hematopoietic stem cells because all the offspring of the modified hematopoietic stem cells, including CD4+ T/B cells, carries our system, and it wonât loss since the renewal of CD4+ T/B cell will continue if the modified hematopoietic stem cells are still alive. The field of gene therapy has help accumulated a lot of knowledge in this area but it was partially hindered by the viral vectors used. The recent developed high efficient genome editing method including CRISPR/Cas9, Transcription activator-like effector nucleaseï¼TALENï¼ and zinc-finger nucleases (ZFNs) might provide a safer approach.  A recent report (Holt et al., 2010) using engineered ZFNs to eliminate CCR5 in human CD34+ hematopoietic stem /progenitor cells (HSPCs) revealed the possibility to manipulate hematopoietic stem cells. Mouse model used in the report shows positive results after genome editing by ZFNs, indicating promising future for CRISPR/Cas9 mediated genome editing.
</P><H2><SPAN class="mw-headline" id="Using_CRISPR.2FCas_system_to_integrate_Cas9_into_human_cells">Using CRISPR/Cas system to integrate Cas9 into human cells</SPAN></H2><P>Our current project uses Piggybac transposon system to deliver Cas9 into human cells for proof-of-principle studies. However, the integration site of Piggybac is non-specific, and it also has preferences for active transcribing sites, which may be normal or oncogene sites. Thus it is not ideal for clinical usage. However, the genome editing tool we used in this project (CRISPR/Cas system) has strong specificity for its target and the target can be designed to non-coding regions on human genome by carefully designed gRNA. One of the possible solutions is to integrate homologous recombinant sites around Cas9. By transient transfect the cell with the Cas9 encoding plasmid shown in Figure 13 and the gRNA expression plasmid shown in Figure 7 and use doxycycline to induce the expression of Cas9, we expect to insert our system into the locus which does no harm to normal gene functions.
</P><CENTER>Figure 13. Possible design of CRISPR-based Cas9 integration plasmid.</CENTER><H2><SPAN class="mw-headline" id="Using_HIV-modified_vector_to_specifically_deliver_Cas9_gene_or_gRNA_plasmid_into_CD4.2B_cells">Using HIV-modified vector to specifically deliver Cas9 gene or gRNA plasmid into CD4+ cells</SPAN></H2><P>One of the main limitations of our transfection system is non-directional, especially for A-B toxin based gRNA delivery because its target receptor is widely distributed on many cell types. However, the main group our system intends to protect is CD4+ cells because they are most susceptible to HIV. Interestingly, a paper published in 2009 reported a clinical trial that 2 X-linked adrenoleukodystrophy (ALD) patients received very positive treatment effect after gene therapy by using HIV-based lentivirus vector to transfect hematopoietic stem/progenitor cells (HSPCs) ex vivo (Cartier et al., 2009). So we will utilize the targeting capability of lentiviral vector while disarm its genome integration function to enable transient expression of the gRNA in the proper cells in vivo without the side effects of random genome insertion. Thereâs a saying in Chinese which is âLike cures likeâ. We think it explicitly expressed the idea of this approach.
</P><H2><SPAN class="mw-headline" id="Application_in_other_retroviral_diseases">Application in other retroviral diseases</SPAN></H2><P>The system designed by us can also be applied in treating other retrovirus diseases, such as Hepatitis B. Weâve designed gRNA for HBV and constructed target sequence plasmids to test its effectiveness.
</P><P><A href="/Team:SUSTC-Shenzhen/gRNA_Design" class="btn btn-success btn-lg" style="color:#fff">See how we design gRNA for HBV</A></P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><P>Cartier, N., Hacein-Bey-Abina, S., Bartholomae, C. C., Veres, G., Schmidt, M., Kutschera, I., . . . Aubourg, P. (2009). Hematopoietic stem cell gene therapy with a lentiviral vector in X-linked adrenoleukodystrophy. Science, 326(5954), 818-823. doi: 10.1126/science.1171242
</P><P>Chen, T. Y., Hsu, C. T., Chang, K. H., Ting, C. Y., Whang-Peng, J., Hui, C. F., &amp; Hwang, J. (2000). Development of DNA delivery system using Pseudomonas exotoxin A and a DNA binding region of human DNA topoisomerase I. Applied Microbiology and Biotechnology, 53(5), 558-567. 
</P><P>Clontech. Tet-On 3G Inducible Expression Systems User Manual. <A href="http://www.clontech.com/TW/Products/Inducible_Systems/Tetracycline-Inducible_Expression/ibcGetAttachment.jsp?cItemId=17569&amp;fileId=6832390&amp;sitex=10021:22372:US" class="external free" rel="nofollow">http://www.clontech.com/TW/Products/Inducible_Systems/Tetracycline-Inducible_Expression/ibcGetAttachment.jsp?cItemId=17569&amp;fileId=6832390&amp;sitex=10021:22372:US</A>. 
</P><P>Cong, L., Ran, F. A., Cox, D., Lin, S. L., Barretto, R., Habib, N., . . . Zhang, F. (2013). Multiplex Genome Engineering Using CRISPR/Cas Systems. Science, 339(6121), 819-823. doi: DOI 10.1126/science.1231143
</P><P>Fominaya, J., Uherek, C., &amp; Wels, W. (1998). A chimeric fusion protein containing transforming growth factor-alpha mediates gene transfer via binding to the EGF receptor. Gene Ther, 5(4), 521-530. doi: 10.1038/sj.gt.3300614
</P><P>Gaur, R., Gupta, P. K., Goyal, A., Wels, W., &amp; Singh, Y. (2002). Delivery of nucleic acid into mammalian cells by anthrax toxin. Biochemical and Biophysical Research Communications, 297(5), 1121-1127. doi: Pii S0006-291x(02)02299-4 Doi 10.1016/S0006-291x(02)02299-4
</P><P>Holt, N., Wang, J., Kim, K., Friedman, G., Wang, X., Taupin, V., . . . Cannon, P. M. (2010). Human hematopoietic stem/progenitor cells modified by zinc-finger nucleases targeted to CCR5 control HIV-1 in vivo. Nature Biotechnology, 28(8), 839-847. doi: 10.1038/nbt.1663
</P><P>Hsu, P. D., Scott, D. A., Weinstein, J. A., Ran, F. A., Konermann, S., Agarwala, V., . . . Zhang, F. (2013). DNA targeting specificity of RNA-guided Cas9 nucleases. Nature Biotechnology, 31(9), 827-+. doi: Doi 10.1038/Nbt.2647
</P><P>Hu, W., Kaminski, R., Yang, F., Zhang, Y., Cosentino, L., Li, F., . . . Khalili, K. (2014). RNA-directed gene editing specifically eradicates latent and prevents new HIV-1 infection. Proc Natl Acad Sci U S A. doi: 10.1073/pnas.1405186111
</P><P>Kiem, H. P., Jerome, K. R., Deeks, S. G., &amp; McCune, J. M. (2012). Hematopoietic-stem-cell-based gene therapy for HIV disease. Cell Stem Cell, 10(2), 137-147. doi: 10.1016/j.stem.2011.12.015
</P><P>Mali, P., Esvelt, K. M., &amp; Church, G. M. (2013). Cas9 as a versatile tool for engineering biology. Nat Methods, 10(10), 957-963. doi: 10.1038/nmeth.2649
</P><P>Mali, P., Yang, L., Esvelt, K. M., Aach, J., Guell, M., DiCarlo, J. E., . . . Church, G. M. (2013). RNA-guided human genome engineering via 
Cas9. Science, 339(6121), 823-826. doi: 10.1126/science.1232033
</P><P>Zuker, M. (2003). Mfold web server for nucleic acid folding and hybridization prediction. Nucleic Acids Res, 31(13), 3406-3415. doi: Doi 10.1093/Nar/Gkg595
</P></DIV></DIV><FOOTER class="bs-docs-footer col-md-12" role="contentinfo"><DIV class="container"><P>Maintained by the iGEM team SUSTC-Shenzhen.</P><P>Licensed under <A href="http://creativecommons.org/licenses/by/4.0/">CC BY 4.0</A>.</P><UL class="bs-docs-footer-links muted"><LI>Â©2014 SUSTC-Shenzhen</LI></UL></DIV></FOOTER><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:SUSTC-Shenzhen/Project">http://2014.igem.org/Team:SUSTC-Shenzhen/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SUSTC-Shenzhen/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SUSTC-Shenzhen/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SUSTC-Shenzhen/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SUSTC-Shenzhen/Project&amp;oldid=401652" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>