<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SUSTC-Shenzhen_Notebook_Parts_building"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SUSTC-Shenzhen/Notebook/Parts_building">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SUSTC-Shenzhen/Notebook/Parts_building&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SUSTC-Shenzhen/Notebook/Parts_building&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SUSTC-Shenzhen/Notebook/Parts_building&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SUSTC-Shenzhen/Notebook/Parts_building" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SUSTC-Shenzhen/Notebook/Parts building</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Team SUSTC-Shenzhen</TITLE></P><DIV class="container"><NAV class="navbar navbar-default navbar-fixed-top" role="navigation" id="index-nav-main"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="/Team:SUSTC-Shenzhen">Home</A></LI><LI><A href="/Team:SUSTC-Shenzhen/Project">Project</A></LI><LI><A href="/Team:SUSTC-Shenzhen/Notebook">Notebook</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">More</A></LI><LI><A href="/Team:SUSTC-Shenzhen/JudgingForm">Judging Form</A></LI><LI><A href="/Team:SUSTC-Shenzhen/Attributions">Attributions</A></LI></UL><LI><A href="https://igem.org/Team.cgi?id=1431">Team Profile</A></LI><UL class="nav navbar-nav navbar-right"><LI><FORM action="/Special:Search" id="searchform" class="navbar-form" role="search"><BUTTON type="submit" name="go" class="btn btn-default" id="mw-searchButton" value="Go" title="Search the pages for this text">Search</BUTTON></FORM></LI><LI><A id="nav-login-status">Log in</A></LI></UL></DIV></NAV></DIV><DIV class="bs-docs-header"><DIV class="container"><H1>

Parts building

        </H1><P>

design and building

        </P></DIV></DIV><DIV id="main-content" class="container"><DIV class="col-md-3" id="document-sidebar"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_4.2C_Aug"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 4, Aug</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#2014.2F08.2F22"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">2014/08/22</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#2014.2F08.2F23"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">2014/08/23</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-4"><A href="#Week_5.2C_Aug"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Week 5, Aug</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#2014.2F08.2F24"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">2014/08/24</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#2014.2F08.2F28"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">2014/08/28</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Week_2.2C_Sep"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Week 2, Sep</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#2014.2F09.2F07"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">2014/09/07</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#2014.2F09.2F11"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">2014/09/11</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Week_3.2C_Sep"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Week 3, Sep</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#2014.2F09.2F18"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">2014/09/18</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#2014.2F09.2F19"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">2014/09/19</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#2014.2F09.2F20"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">2014/09/20</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#Week_4.2C_Sep"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Week 4, Sep</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#2014.2F09.2F21"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">2014/09/21</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#2014.2F09.2F22"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">2014/09/22</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#2014.2F09.2F24"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">2014/09/24</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#2014.2F09.2F25"><SPAN class="tocnumber">5.4</SPAN><SPAN class="toctext">2014/09/25</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#2014.2F09.2F26"><SPAN class="tocnumber">5.5</SPAN><SPAN class="toctext">2014/09/26</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#2014.2F09.2F27"><SPAN class="tocnumber">5.6</SPAN><SPAN class="toctext">2014/09/27</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#2014.2F09.2F30"><SPAN class="tocnumber">5.7</SPAN><SPAN class="toctext">2014/09/30</SPAN></A></LI><LI class="toclevel-1 tocsection-22"><A href="#Week_1.2C_Oct"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Week 1, Oct</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#2014.2F10.2F01"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">2014/10/01</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#2014.2F10.2F02"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">2014/10/02</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#2014.2F10.2F03"><SPAN class="tocnumber">6.3</SPAN><SPAN class="toctext">2014/10/03</SPAN></A></LI><LI class="toclevel-1 tocsection-26"><A href="#Week_2.2C_Oct"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Week 2, Oct</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#2014.2F10.2F08"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">2014/10/08</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#2014.2F10.2F09"><SPAN class="tocnumber">7.2</SPAN><SPAN class="toctext">2014/10/09</SPAN></A></LI><LI class="toclevel-2 tocsection-29"><A href="#2014.2F10.2F10"><SPAN class="tocnumber">7.3</SPAN><SPAN class="toctext">2014/10/10</SPAN></A></LI><LI class="toclevel-2 tocsection-30"><A href="#2014.2F10.2F11"><SPAN class="tocnumber">7.4</SPAN><SPAN class="toctext">2014/10/11</SPAN></A></LI><LI class="toclevel-2 tocsection-31"><A href="#2014.2F10.2F12"><SPAN class="tocnumber">7.5</SPAN><SPAN class="toctext">2014/10/12</SPAN></A></LI></TD></TR></TBODY></TABLE></DIV><DIV class="col-md-9" id="document-body"><P><I>By Rifei Chen</I></P><P>For the linear backbone were so few and the backbone of BBa_J04450 was pSB1C3, we decide to cut it by XbaI and PstI restriction enzyme and recycle the about 2000bp vector as the backbone for biobricks. Of course, one advantage to make it as backbone is that we can recognize whether the gene we designed have inserted. Because if not, the bacterium will express red chromoprotein which one we unnecessary to pick up.
</P><H2><SPAN class="mw-headline" id="Week_4.2C_Aug">Week 4, Aug</SPAN></H2><H4><SPAN class="mw-headline" id="2014.2F08.2F22"><B>2014/08/22</B></SPAN></H4><P><SPAN style="color: red"><I><U>Transfect the backbone of biobricks</U></I></SPAN></P><OL><LI>Add 10μl ddwater to the plate, wait 10min, and absorb all the liquid into a PCR tube. 
</LI><LI>Took a tube of supercompetent cells from -80℃ fridge, Immediately place the tube on ice, allow cell to thaw on ice, 10 min.
</LI><LI>Check the cell to see if they have thawed, gently flick the cells 1-2 times to evenly resuspend the cells.
</LI><LI>Mark the label with BBa_J04450.
</LI><LI>Add 2μl plasmid to the tube, shaking slightly.
</LI><LI>Incubate on ice for 30min.
</LI><LI>Heat shock, 42℃, 90s.
</LI><LI>Keep on ice immediately, 2 min.
</LI><LI>Add 200μl LB medium to each tube, shake with 37°C, 200rpm, 45min.
</LI><LI>Centrifuge, 4000rmp, 3min at room template.
</LI><LI>Discard most of the medium and reserve 50μl, resuspend by tips.
</LI><LI>Flam the SS-Spreader; put aside to cool; coated plates.
</LI><LI>Incubate the plates at 37°C, 16 hours.
</LI></OL><H4><SPAN class="mw-headline" id="2014.2F08.2F23"><B>2014/08/23</B></SPAN></H4><P><SPAN style="color: red"><I><U>Shake the bacterium in tubes</U></I></SPAN></P><OL><LI>Add 3ml LB broth with ampicillin antibiotics into a tube.
</LI><LI>Flam the tweezers to red; put aside to cool.
</LI><LI>Pick up a tip which was sterilized; pick monoclone and throw the tip with monoclone into the tube directly. 
</LI><LI>Shake with 37°C, 200rpm, and overnight.
</LI></OL><H2><SPAN class="mw-headline" id="Week_5.2C_Aug">Week 5, Aug</SPAN></H2><H4><SPAN class="mw-headline" id="2014.2F08.2F24"><B>2014/08/24</B></SPAN></H4><P><SPAN style="color: red"><I><U>Plasmid purification</U></I></SPAN></P><OL><LI>Pour 1.5ml of broth into a centrifuge tube, 11000 rpm, 2min.
</LI><LI>Repeat step1 again.
</LI><LI>Using TIANprep Mini Plasmid Kit, remove the medium completely, add 150μl P1 buffer, resuspend by tips, votex.
</LI><LI>Add 150μl P2 buffer, up and down gently 6~8 times; till it turn clear.
</LI><LI>Add 350μl P5, immediately invert up and down 10~12 times; till it clear and appear flocks. 11000rpm, 5 min.
</LI><LI>Transfer the supernatant into absorption column CP3, do not carry the precipitate. 11000rpm, 1 min. Discard the liquid waste.
</LI><LI>Add 600μl PW buffer; wait 3min; 11000rpm, 1min. Discard the liquid waste.
</LI><LI>Repeat step7 again.
</LI><LI>Centrifuge again at 11000 rpm, 2 min. Discard the liquid waste.
</LI><LI>Put the absorption column CP3 into a new centrifuge tube, open the cover and wait alcohol to spread completely.
</LI><LI>Add 70μl TB buffer into the absorption column center; wait 2min, 11000rpm, 2min.
</LI><LI>Re-suck up TB buffer which were centrifuged and add into the absorption column center again; wait 2min, 11000rpm, 2min.
</LI><LI>Measure the concentration of plasmid by Nanodrop2000. Mark in the tube.
</LI></OL><H4><SPAN class="mw-headline" id="2014.2F08.2F28"><B>2014/08/28</B></SPAN></H4><P><SPAN style="color: red"><I><U>Design the biobricks</U></I></SPAN></P><H2><SPAN class="mw-headline" id="Week_2.2C_Sep">Week 2, Sep</SPAN></H2><H4><SPAN class="mw-headline" id="2014.2F09.2F07"><B>2014/09/07</B></SPAN></H4><P><SPAN style="color: red"><I><U>Design the primers</U></I></SPAN></P><H4><SPAN class="mw-headline" id="2014.2F09.2F11"><B>2014/09/11</B></SPAN></H4><P><SPAN style="color: red"><I><U>Place an order of primers and completely synthesis</U></I></SPAN></P><UL><LI>Primers show:</LI></UL><P>SV40 promoter For: T TCTAGA G CGAACTGTGGAATGTGTG
SV40 promoter Rev: A GGATCC AT GAAGAC GACGAAAATGGATATACAAG
SV40 PolyA For: G AGATCT AT GAAGAC CCAACTTGTTTATTGCAGCTTA
SV40 PolyA Rev: CTGCAG CGGCCGC T ACTAGT ATCCATGCCGAGAGTGATGAA
pTRE-3G promoter For: T TCTAGA G TTTAAACTTTACTCCCTATC
pTRE-3G promoter Rev: A GGATCC AT GAAGAC GATTTACGAGGGTAGGAAGTG
TetOn For: T TCTAGA TGGGATCAAGACTGGACAAGA
TetOn Rev: CTGCAG CGGCCGC T ACTAGT A CCGAAGCCCAACCTTTCATA
CBh promoter For: T TCTAGA G TCTAGAGGTACCCGTTACA
CBh promoter Rev: A GGATCC AT GAAGAC GTCCAACCTGAAAAAAAGTGA
PB3 For: G AGATCT AT GAAGAC TGCTAGACTATAACAAGAA
PB3 Rev: CTGCAG CGGCCGC T ACTAGT A CGAAATTAACCCTCACTAA
PB5 For: T TCTAGA G ATTAGAAACTATTATTTAAC
PB5 Rev: A GGATCC AT GAAGAC GACACACATTCCACAGTTCG</P><UL><LI>Completely synthesis show: </LI></UL><P>HBV1: 
5'GATCAAGAAGACCTCACCAGGACCCCTGCTCGTGTTACAGGGTTTGGGTCTTCGATCTC 3'</P><P>HBV2: 
5'GATCAAGAAGACCTCACCTCCGCAGTATGGATCGGCAGGTTTGGGTCTTCGATCTC 3'</P><P>HIV: 
5'GATCAAGAAGACCTCACCTCTAGCAGTGGCGCCCGAACAGGGTTTGGGTCTTCGATCGA</P><P>5*UAS: 
5'CGGAATTCGCGGCCGCTTCTAGAGCGGAGTACTGTCCTCCGAGCGGAGTACTGTCCTCCGACTCGAGCGGAGTACTGTCCTCCGATCGGAGTACTGTCCTCCGCGATTTCCGGAGTACTGTCCTCCGTACTAGTAGCGGCCGCTGCAGAACCAATGCATTGG 3'</P><P>7*UAS: 
5'CGGAATTCGCGGCCGCTTCTAGAGCGGAGTACTGTCCTCCGCGGAGTACTGTCCTCCGGAATTGCGATAGGTACCGAGTTACTAGACGGAGTACTGTCCTCCGAGCGGAGTACTGTCCTCCGACTCGAGCGGAGTACTGTCCTCCGATCGGAGTACTGTCCTCCGCGATTTCCGGAGTACTGTCCTCCGTACTAGTAGCGGCCGCTGCAGAACCAATGCATTGG 3'</P><P>PolyA: 
5'GAAGATCTTCGAAGACCTAGAGCTCGCTGATCAGCCTCGACTGTGCCTTCTAGTTGCCAGCCATCTGTTGTTTGCCCCTCCCCCGTGCCTTCCTTGACCCTGGAAGGTGCCACTCCCACTGTCCTTTCCTAATAAAATGAGGAAATTGCATCGCATTGTCTGAGTAGGTGTCATTCTATTCTGGGGGGTGGGGTGGGGCAGGACAGCAAGGGGGAGGATTGGGAAGAGAATAGCAGGCATGCTGGGGATACTAGTAGCGGCCGCTGCAGAACCAATGCATTGG 3'</P><P>NLS+NLS:
5'TCTGGAATTCGCGGCCGCTTCTAGAGCCAGATGGCCCCAAAGAAGAAGCGGAAGGTCGGTATCCACGGAGTCCCAGCAGCCACGTCTTCATGGATCCTATGAAGACTGAAAAGGCCGGCGGCCACGAAAAAGGCCGGCCAGGCAAAAAAGAAAAAGTACTAGTAGCGGCCGCTGCAGTCCGGCAAAAAAGGGC 3'</P><H2><SPAN class="mw-headline" id="Week_3.2C_Sep">Week 3, Sep</SPAN></H2><H4><SPAN class="mw-headline" id="2014.2F09.2F18"><B>2014/09/18</B></SPAN></H4><P><SPAN style="color: red"><I><U>PCR TetOn, PB5, PB3, SV40/CBh/pTRE-3G promoter, PolyA1, 2*UAS</U></I></SPAN></P><P>For TetOn and CBh were more than 700bp, we decided to use Q5<SUP>®</SUP> High-Fidelity DNA Polymerases, but others by OneTaq<SUP>®</SUP> 2X Master Mix. 2*UAS without template because the forward primer and reverse primer coμld combine each other and extend.<B>The PCR systems set by protocol show below, we made 50μl reaction.</B></P><P><B>Thermocycling Conditions for a Routine PCR:</B></P><TABLE class="table"><TBODY><TR><TH> Step
</TH><TH> Temp (℃)
</TH><TH> Time (s)
</TH></TR><TR><TD> Initial denaturation
</TD><TD> 98
</TD><TD> 90
</TD></TR><TR><TD rowspan="3"> 30cycles
</TD><TD> 98
</TD><TD> 30
</TD></TR><TR><TD> 48,52,55,58,60,63
</TD><TD> 30
</TD></TR><TR><TD> 68
</TD><TD> 30
</TD></TR><TR><TD> Final extension
</TD><TD> 68
</TD><TD> 300
</TD></TR><TR><TD> Hold
</TD><TD> 4
</TD><TD> ∞
</TD></TR></TBODY></TABLE><P><B>Make a 1% Agarose gel:</B></P><TABLE class="table"><TBODY><TR><TH> Component
</TH><TH> Volume
</TH></TR><TR><TD> 1×TAE buffer
</TD><TD> 50ml
</TD></TR><TR><TD> Agarose
</TD><TD> 0.50g
</TD></TR><TR><TD> Gene Green
</TD><TD> 0.5ul
</TD></TR></TBODY></TABLE><P><B>Gel electrophoresis result:</B></P><P>PB3 and PB5 were both failed.
Purified others by TIANgel Midi Purification Kit. Use Nanodrop2000 to get each sample’s concentration. Digest TetOn, 2*UAS and backbone by XbaI and PstI overnight; CBh/SV40/pTRE-3G promoter by XbaI and BamHI; PolyA1 by BglII and PstI; purified.
</P><P><B>The digest systems:</B></P><TABLE class="table"><TBODY><TR><TH> Enzyme(μl)
</TH><TH> Buffer(μl)
</TH><TH> DNA(ng)
</TH><TH> ddH<SUB>2</SUB>O(μl)
</TH><TH> Total volume(μl)
</TH></TR><TR><TD> 0.5+0.5
</TD><TD> 2.5
</TD><TD> 500
</TD><TD> up to 25
</TD><TD> 25
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="2014.2F09.2F19"><B>2014/09/19</B></SPAN></H4><P><SPAN style="color: red"><I><U>Ligation and transformation</U></I></SPAN></P><P><B>The ligation systems (vector + TetOn, vector + 2*UAS, vector + CBh/SV40/pTRE-3G promoter + PolyA1) :</B></P><TABLE class="table"><TBODY><TR><TH> T4 ligase(μl)
</TH><TH> T4 DNA ligation buffer(μl)
</TH><TH> Vector and insert mole ratio
</TH><TH> ddH<SUB>2</SUB>O(μl)
</TH><TH> Total volume(μl)
</TH></TR><TR><TD> 1
</TD><TD> 2
</TD><TD> 1:5/1:5:5
</TD><TD> up to 20
</TD><TD> 20
</TD></TR></TBODY></TABLE><P>16℃ for 35min, inactive at 65℃ for 10min.
Transfect with DH5α
Re-PCR PB3 and PB5, still without any bands.
</P><H4><SPAN class="mw-headline" id="2014.2F09.2F20"><B>2014/09/20</B></SPAN></H4><P><SPAN style="color: red"><I><U>Shake the bacterium in tubes</U></I></SPAN></P><P>Pick up backbone + TetOn, backbone + 2*UAS monoclone and shake overnight.
There were no bacteria grown of backbone + CBh/SV40/pTRE-3G promoter + PolyA1.
Analyzed and find the primers of CBh/SV40/pTRE-3G promoter and PolyA1 were all out of protect bases. Re-design and order new primers.
</P><H2><SPAN class="mw-headline" id="Week_4.2C_Sep">Week 4, Sep</SPAN></H2><H4><SPAN class="mw-headline" id="2014.2F09.2F21"><B>2014/09/21</B></SPAN></H4><P><SPAN style="color: red"><I><U>Plasmid purification and digest to check length, sequencing</U></I></SPAN></P><P>Purified plasmids by TIANprep Mini Plasmid Kit. Digest by XbaI and PstI for two hours.
Make a 1% Agarose gel. The results show right length
</P><H4><SPAN class="mw-headline" id="2014.2F09.2F22"><B>2014/09/22</B></SPAN></H4><P><SPAN style="color: red"><I><U>Send TetOn and 2*UAS to sequencing</U></I></SPAN></P><H4><SPAN class="mw-headline" id="2014.2F09.2F24"><B>2014/09/24</B></SPAN></H4><P><SPAN style="color: red"><I><U>Re-PCR CBh/SV40/pTRE-3G promoter, PolyA1, PB3, PB5 by new primers and digest</U></I></SPAN></P><P>The same PCR systems and reaction conditions like 2014/09/18, but CBh promoter and PB5 failed for wrong length. SV40/ pTRE-3G promoter have many bands for the primers’ low specificity. Still purified CBh and PB5.
Purified by TIANgel Midi Purification Kit. Use Nanodrop2000 to get each sample’s concentration.
Digest CBh/SV40/pTRE-3G promoter and PB5 by XbaI and BamHI; PolyA1 and PB3 by BglII and PstI overnight.
</P><P><B>The digest systems:</B></P><TABLE class="table"><TBODY><TR><TH> Enzyme(μl)
</TH><TH> Buffer(μl)
</TH><TH> DNA(ng)
</TH><TH> ddH<SUB>2</SUB>O(μl)
</TH><TH> Total volume(μl)
</TH></TR><TR><TD> 1+1
</TD><TD> 2.5
</TD><TD> 1000
</TD><TD> up to 25
</TD><TD> 25
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="2014.2F09.2F25"><B>2014/09/25</B></SPAN></H4><P><SPAN style="color: red"><I><U>Re-ligase CBh/SV40/pTRE-3G promoter + PolyA1, PB5+PB3 and transformation</U></I></SPAN></P><P><B>The ligation systems (vector + CBh/SV40/pTRE-3G promoter + PolyA1 and vector + PB5 + PB3) :</B></P><TABLE class="table"><TBODY><TR><TH> T4 ligase(μl)
</TH><TH> T4 DNA ligation buffer(μl)
</TH><TH> Vector and insert mole ratio
</TH><TH> ddH<SUB>2</SUB>O(μl)
</TH><TH> Total volume(μl)
</TH></TR><TR><TD> 1
</TD><TD> 2
</TD><TD> 1:5:5
</TD><TD> up to 20
</TD><TD> 20
</TD></TR></TBODY></TABLE><P>16℃ for 35min, inactive at 65℃ for 10min.
Transfect with DH5α
</P><P>Sequencing show TenOn was right but 2*UAS lost one SpeI site.Analyzed and find due to the wrong direction of primer. Re-design and order new primers.
</P><H4><SPAN class="mw-headline" id="2014.2F09.2F26"><B>2014/09/26</B></SPAN></H4><P><SPAN style="color: red"><I><U>Shake the bacterium in tubes</U></I></SPAN></P><P>Pick up monoclone and shake overnight. PB5 + PB3 failed for without any monoclone.
</P><H4><SPAN class="mw-headline" id="2014.2F09.2F27"><B>2014/09/27</B></SPAN></H4><P><SPAN style="color: red"><I><U>Purified CBh/SV40/pTRE-3G promoter + PolyA1 and digest, check by gel electrophoresis</U></I></SPAN></P><P>Purified plasmids by TIANprep Mini Plasmid Kit. Digest by XbaI and PstI for two hours. Make a 1% Agarose gel.<B>Gel electrophoresis result:</B></P><P>All results get wrong.
</P><H4><SPAN class="mw-headline" id="2014.2F09.2F30"><B>2014/09/30</B></SPAN></H4><P><SPAN style="color: red"><I><U>Digest UAS, HIV, HBV, PolyA2, NLS-NLS</U></I></SPAN></P><P>Get completely synthesis sequence plasmid and digest UAS, HIV, HBV, PolyA2, NLS-NLS overnight.<B>The digest systems of 5*UAS/7*UAS:</B></P><TABLE class="table"><TBODY><TR><TH> Enzyme XbaI+PstI(μl)
</TH><TH> Buffer 2.1(μl)
</TH><TH> DNA(ng)
</TH><TH> ddH<SUB>2</SUB>O(μl)
</TH><TH> Total volume(μl)
</TH></TR><TR><TD> 0.5+0.5
</TD><TD> 2.0
</TD><TD> 400
</TD><TD> 7
</TD><TD> 20
</TD></TR></TBODY></TABLE><P><B>The digest systems of HIV/HBV1/HBV2/PolyA2/NLS-NLS:</B></P><TABLE class="table"><TBODY><TR><TH> Enzyme XbaI+PstI(μl)
</TH><TH> Buffer 2.1(μl)
</TH><TH> DNA(ng)
</TH><TH> ddH<SUB>2</SUB>O(μl)
</TH><TH> Total volume(μl)
</TH></TR><TR><TD> 0.5+0.5
</TD><TD> 2.0
</TD><TD> 225
</TD><TD> 7
</TD><TD> 20
</TD></TR></TBODY></TABLE><P><SPAN style="color: red"><I><U>Touchdown PCR for CBh promoter, PolyA1, PB3 and PB5</U></I></SPAN></P><P><B>The PCR system show:</B></P><TABLE class="table"><TBODY><TR><TH> Taq polymerase(μl)
</TH><TD> 0.5
</TD></TR><TR><TH> Buffer(μl)
</TH><TD> 5
</TD></TR><TR><TH> dNTP(μl)
</TH><TD> 1
</TD></TR><TR><TH> Primer For(μl)
</TH><TD> 0.5
</TD></TR><TR><TH> Primer Rev(μl)
</TH><TD> 0.5
</TD></TR><TR><TH> Template(μl)
</TH><TD> 1
</TD></TR><TR><TH> ddH<SUB>2</SUB>O(μl)
</TH><TD> 41.5
</TD></TR><TR><TD> Total(μl)
</TD><TD> 50
</TD></TR></TBODY></TABLE><P><B>Thermocycling Conditions for a Routine PCR:</B></P><TABLE class="table"><TBODY><TR><TH> Step
</TH><TH> Temp (℃)
</TH><TH> Time (s)
</TH></TR><TR><TD> Initial denaturation
</TD><TD> 95
</TD><TD> 180
</TD></TR><TR><TD rowspan="3"> 15 touchdown cycleseach cycle down 1℃ of annealing temperature
</TD><TD> 95
</TD><TD> 15
</TD></TR><TR><TD> 55
</TD><TD> 30
</TD></TR><TR><TD> 72
</TD><TD> 40
</TD></TR><TR><TD rowspan="3"> 35cycles
</TD><TD> 95
</TD><TD> 15
</TD></TR><TR><TD> 45
</TD><TD> 30
</TD></TR><TR><TD> 68
</TD><TD> 40
</TD></TR><TR><TD> Final extension
</TD><TD> 68
</TD><TD> 300
</TD></TR><TR><TD> Hold
</TD><TD> 4
</TD><TD> ∞
</TD></TR></TBODY></TABLE><P><B>Gel electrophoresis result:</B></P><P>Only PolyA1 and PB3 were PCR successfully, CBh and PB5 were wrong length. But still purified and digest overnight.
</P><P>PCR 2*UAS by new primers. Check, purify and digest overnight.
</P><H2><SPAN class="mw-headline" id="Week_1.2C_Oct">Week 1, Oct</SPAN></H2><H4><SPAN class="mw-headline" id="2014.2F10.2F01"><B>2014/10/01</B></SPAN></H4><P><SPAN style="color: red"><I><U>Ligase UAS, HIV, HBV, NLS-NLS, PB3+PB5 to backbone</U></I></SPAN></P><P>Purified the digest plasmids and PCR products by TIANgel Midi Purification Kit. Use Nanodrop2000 to get each sample’s concentration. 
Ligase UAS/gRNA for HIV/gRNA for HBV/NLS-NLS with backbone by mole ratio 7:1 at 16℃ for 35min, and then inactive at 60℃ for 10min.
Ligase PB3+PB5, CBh/SV40/pTRE-3G promoter+PolyA1, CBh/SV40/pTRE-3G promoter+PolyA2 with backbone by mole ratio 7:10:1 at 16℃ for 35min, and then inactive at 60℃ for 10min.
Transfect ligation plasmid but increase the incubation time to 2 hours.
</P><H4><SPAN class="mw-headline" id="2014.2F10.2F02"><B>2014/10/02</B></SPAN></H4><P><SPAN style="color: red"><I><U>Pick up the monoclony to do colonyPCR and shake</U></I></SPAN></P><P>CBh promoter + PolyA do not grow the right bacterium for all grown bacterium show red chromoprotein. So decide give it up.
Pick up 5 monoclonies each sample of SV40/pTRE-3G promoter+PolyA1, SV40/pTRE-3G promoter+PolyA2 and a colonyPCR to check whether the length were rigth.</P><P><B>Gel electrophoresis result:</B></P><P>
Picture shows SV40/pTRE-3G promoter+PolyA1 were all right length but SV40/pTRE-3G promoter+PolyA2 were all wrong.Shake three sample each
Pick up a monoclony each sample of 2*UAS, HIV, HBV
</P><P><B>Gel electrophoresis result:</B></P><P>
Picture shows all sample were right.Shake them in tubes.
</P><H4><SPAN class="mw-headline" id="2014.2F10.2F03"><B>2014/10/03</B></SPAN></H4><P><SPAN style="color: red"><I><U>Purified SV40/pTRE-3G promoter + PolyA1 and digest, check by gel electrophoresis</U></I></SPAN></P><P>Purified plasmids by TIANprep Mini Plasmid Kit. Digest by XbaI+PstI, XbaI+BbssI for two hours.
Make a 1% Agarose gel and wait 1h to be solid. 
</P><P><B>Gel electrophoresis result:</B></P><P>
Picture shows all sample were right. We both chose the sample3 to send to sequencing.
Of course send UAS, gRNA and NLS to sequencing.
PCR the target sequence of HIV, HBV1, HBV2.And the next three days do the same thing like above, purify, digest, ligase, sequencing.
</P><H2><SPAN class="mw-headline" id="Week_2.2C_Oct">Week 2, Oct</SPAN></H2><H4><SPAN class="mw-headline" id="2014.2F10.2F08"><B>2014/10/08</B></SPAN></H4><P><SPAN style="color: red"><I><U>The last sample send to sequencing</U></I></SPAN></P><H4><SPAN class="mw-headline" id="2014.2F10.2F09"><B>2014/10/09</B></SPAN></H4><P><SPAN style="color: red"><I><U>All plasmids make dry and send to the HQ of iGEM</U></I></SPAN></P><H4><SPAN class="mw-headline" id="2014.2F10.2F10"><B>2014/10/10</B></SPAN></H4><P><SPAN style="color: red"><I><U>PCR eGFP for SV40 and pTRE-3G</U></I></SPAN></P><P><B>The PCR system show:</B></P><TABLE class="table"><TBODY><TR><TH> Ex Taq polymerase(μl)
</TH><TD> 0.25
</TD></TR><TR><TH> Buffer(μl)
</TH><TD> 5
</TD></TR><TR><TH> dNTP(μl)
</TH><TD> 4
</TD></TR><TR><TH> Mg<SUP>2+</SUP>(μl)
</TH><TD> 4
</TD></TR><TR><TH> Primer For(μl)
</TH><TD> 0.5
</TD></TR><TR><TH> Primer Rev(μl)
</TH><TD> 0.5
</TD></TR><TR><TH> Template(μl)
</TH><TD> 1
</TD></TR><TR><TH> ddH<SUB>2</SUB>O(μl)
</TH><TD> 34.75
</TD></TR><TR><TD> Total(μl)
</TD><TD> 50
</TD></TR></TBODY></TABLE><P><B>Thermocycling Conditions for a Routine PCR:</B></P><TABLE class="table"><TBODY><TR><TH> Step
</TH><TH> Temp (℃)
</TH><TH> Time (s)
</TH></TR><TR><TD> Initial denaturation
</TD><TD> 98
</TD><TD> 180
</TD></TR><TR><TD rowspan="3"> 35cycles
</TD><TD> 98
</TD><TD> 15
</TD></TR><TR><TD> 52,57,61
</TD><TD> 30
</TD></TR><TR><TD> 68
</TD><TD> 60
</TD></TR><TR><TD> Final extension
</TD><TD> 68
</TD><TD> 300
</TD></TR><TR><TD> Hold
</TD><TD> 4
</TD><TD> ∞
</TD></TR></TBODY></TABLE><P>Took 1μl each sample to make a gel electrophoresis, observed all the aim bands PCR successfully. By TIANgel Midi Purification Kit, we get the purified sequence. 
</P><H4><SPAN class="mw-headline" id="2014.2F10.2F11"><B>2014/10/11</B></SPAN></H4><P><SPAN style="color: red"><I><U>Digest and ligase SV40/pTRE-3G promoter + eGFP </U></I></SPAN></P><P>Digest SV40, pTRE-3G, eGFP for SV40 and pTRE-3G by BbsI restriction enzymes for 3hours.
By TIANgel Midi Purification Kit, get the vector of SV40 and pTRE-3G. Use Nanodrop2000 to get the concentration.
Ligase the vector and insert by mole ratio 1:5 at 16℃ for 35min, and then inactive at 60℃ for 10min.
</P><P>Transfect ligation plasmids but increase the incubation time to 2hours.
</P><H4><SPAN class="mw-headline" id="2014.2F10.2F12"><B>2014/10/12</B></SPAN></H4><P><SPAN style="color: red"><I><U>Colony PCR of SV40/pTRE-3G promoter + eGFP</U></I></SPAN></P><P>Pick up 8 monoclone from SV40 promoter + eGFP plate, 8 monoclone from pTRE-3G promoter + eGFP, and 1 monoclone as active control group. Each monoclone dissolve in 10μl ddH<SUB>2</SUB>O respectively, and suck 1.5μl as the template.
</P><P><B>Gel electrophoresis result:</B></P><P>
There were only two samples PCR successfully, and not right length. So we think that may be the PCR system got wrong. We decide to do that again but change system.
</P><P><SPAN style="color: red"><I><U>Re-Colony PCR of SV40/pTRE-3G promoter + eGFP</U></I></SPAN></P><P>The same thinking as above, but do with another kind of taq polymerase.
Pick up 6 monoclone from SV40 promoter + eGFP plate, 6 monoclone from pTRE-3G promoter + eGFP, and 1 monoclone as active control group. Each monoclone dissolve in 10μl ddH<SUB>2</SUB>O respectively, and suck 1.5μl as the template.
</P><P><B>Gel electrophoresis result:</B></P><P>
All the PCR results were wrong for the wrong length. I doubt the restriction enzymes did not cut the vector or insert.
</P><P>For the limited time, we decide not to do that again.
</P></DIV></DIV><FOOTER class="bs-docs-footer col-md-12" role="contentinfo"><DIV class="container"><P>Maintained by the iGEM team SUSTC-Shenzhen.</P><P>Licensed under <A href="http://creativecommons.org/licenses/by/4.0/">CC BY 4.0</A>.</P><UL class="bs-docs-footer-links muted"><LI>©2014 SUSTC-Shenzhen</LI></UL></DIV></FOOTER><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:SUSTC-Shenzhen/Notebook/Parts_building">http://2014.igem.org/Team:SUSTC-Shenzhen/Notebook/Parts_building</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SUSTC-Shenzhen/Notebook/Parts_building" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SUSTC-Shenzhen/Notebook/Parts_building" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SUSTC-Shenzhen/Notebook/Parts_building&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SUSTC-Shenzhen/Notebook/Parts_building&amp;oldid=327820" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>