<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SUSTC-Shenzhen_Notebook_CRISPR_mCherry-BbsI-pointmutation"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SUSTC-Shenzhen/Notebook/CRISPR/mCherry-BbsI-pointmutation">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SUSTC-Shenzhen/Notebook/CRISPR/mCherry-BbsI-pointmutation&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SUSTC-Shenzhen/Notebook/CRISPR/mCherry-BbsI-pointmutation&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SUSTC-Shenzhen/Notebook/CRISPR/mCherry-BbsI-pointmutation&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SUSTC-Shenzhen/Notebook/CRISPR/mCherry-BbsI-pointmutation" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SUSTC-Shenzhen/Notebook/CRISPR/mCherry-BbsI-pointmutation</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Team SUSTC-Shenzhen</TITLE></P><DIV class="container"><NAV class="navbar navbar-default navbar-fixed-top" role="navigation" id="index-nav-main"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="/Team:SUSTC-Shenzhen">Home</A></LI><LI><A href="/Team:SUSTC-Shenzhen/Project">Project</A></LI><LI><A href="/Team:SUSTC-Shenzhen/Notebook">Notebook</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">More</A></LI><LI><A href="/Team:SUSTC-Shenzhen/JudgingForm">Judging Form</A></LI><LI><A href="/Team:SUSTC-Shenzhen/Attributions">Attributions</A></LI></UL><LI><A href="https://igem.org/Team.cgi?id=1431">Team Profile</A></LI><UL class="nav navbar-nav navbar-right"><LI><FORM action="/Special:Search" id="searchform" class="navbar-form" role="search"><BUTTON type="submit" name="go" class="btn btn-default" id="mw-searchButton" value="Go" title="Search the pages for this text">Search</BUTTON></FORM></LI><LI><A id="nav-login-status">Log in</A></LI></UL></DIV></NAV></DIV><DIV class="bs-docs-header"><DIV class="container"><H1>

mCherry BbsI point mutation

        </H1><P>

one step for constructing a universal usable gRNA-insertion plasmid

        </P></DIV></DIV><DIV id="main-content" class="container"><DIV class="col-md-3" id="document-sidebar"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Agilent.27s_Pointmutation_method"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Agilent's Pointmutation method</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#PCR_reaction"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">PCR reaction</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#DpnI_digestion"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">DpnI digestion</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Nanodrop_test_after_digestion"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Nanodrop test after digestion</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Bacteria_transformation"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Bacteria transformation</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Gel_electrophoresis"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Gel electrophoresis</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-7"><A href="#Agilent.27s_Pointmutation_method_II"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Agilent's Pointmutation method II</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#PCR"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">PCR</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Nanodrop_test"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Nanodrop test</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Gel_electrophoresis_2"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Gel electrophoresis</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#2-pair_primers_PCR"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">2-pair primers PCR</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#PCR_2"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">PCR</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Gel_electrophoresis_3"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Gel electrophoresis</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#PCR_II"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">PCR II</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Gel_electrophoresis:"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">Gel electrophoresis:</SPAN></A></LI><LI class="toclevel-1 tocsection-16"><A href="#2-pari_primers_PCR_II"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">2-pari primers PCR II</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#PCR_3"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">PCR</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Gel_electrophoresis_4"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Gel electrophoresis</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#Gel_electrophoresis_II"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Gel electrophoresis II</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#Half-plasmid_PCR_.26_Double_primer_PCR"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Half-plasmid PCR &amp; Double primer PCR</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#Half-plasmid_PCR"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Half-plasmid PCR</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#Double_primer_PCR"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Double primer PCR</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#Gel_electrophoresis_5"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">Gel electrophoresis</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#Gel_extraction"><SPAN class="tocnumber">5.4</SPAN><SPAN class="toctext">Gel extraction</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Nanodrop_test_2"><SPAN class="tocnumber">5.5</SPAN><SPAN class="toctext">Nanodrop test</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#BbsI_enzyme_digestion"><SPAN class="tocnumber">5.6</SPAN><SPAN class="toctext">BbsI enzyme digestion</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#Gel_electrophoresis_6"><SPAN class="tocnumber">5.7</SPAN><SPAN class="toctext">Gel electrophoresis</SPAN></A></LI><LI class="toclevel-1 tocsection-28"><A href="#Verification_of_the_Point_mutation"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Verification of the Point mutation</SPAN></A></LI><LI class="toclevel-2 tocsection-29"><A href="#Enzyme_Digestion"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Enzyme Digestion</SPAN></A></LI><LI class="toclevel-3 tocsection-30"><A href="#BbsI.2C_BspEI_double_digestion.2810:05.7E21:30.29"><SPAN class="tocnumber">6.1.1</SPAN><SPAN class="toctext">BbsI, BspEI double digestion(10:05~21:30)</SPAN></A></LI><LI class="toclevel-3 tocsection-31"><A href="#BspEI_digestion_.2815:41.7E21:30.29"><SPAN class="tocnumber">6.1.2</SPAN><SPAN class="toctext">BspEI digestion (15:41~21:30)</SPAN></A></LI><LI class="toclevel-2 tocsection-32"><A href="#Gel_electrophoresis_7"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Gel electrophoresis</SPAN></A></LI><LI class="toclevel-2 tocsection-33"><A href="#Gel_electrophoresis_II_2"><SPAN class="tocnumber">6.3</SPAN><SPAN class="toctext">Gel electrophoresis II</SPAN></A></LI><LI class="toclevel-1 tocsection-34"><A href="#References"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE></DIV><DIV class="col-md-9" id="document-body"><P><I>By Yicong Tao</I></P><P>
Plasmid used: pBX-084 PB5-HS4-TRE-mCherrynuc-2A-BlaloxN-GpA-HS4-PB3 (from Wei Huang’s lab). <SPAN style="color: red">Total length is 6843bp.</SPAN></P><P>Point mutation primer design:
Forward(5’-3’): cagcccatggttttcttctgcattacggggccg	33nt
Reverse(5’-3’): cggccccgtaatgcagaagaaaaccatgggctg	33nt
The primer pair is intended to eliminate the BbsI site in the mCherry because we need to use BbsI to insert our gRNA into our plasmid. Designed by Agilent Technologies point-mutation primer online design tools. The primers are complement with each other.
</P><H1><SPAN class="mw-headline" id="Agilent.27s_Pointmutation_method">Agilent's Pointmutation method</SPAN></H1><P>8.13~8.14 
</P><H2><SPAN class="mw-headline" id="PCR_reaction">PCR reaction</SPAN></H2><P>(12:06~14:XX 13th Aug, about 2h)
</P><P><B>300ul, 50ul/reaction:</B></P><TABLE class="table"><TBODY><TR><TH> Component
</TH><TH> Volume
</TH><TH> Final Concentration
</TH></TR><TR><TD> Q5 master mix 2X
</TD><TD> 150ul
</TD><TD> 1X
</TD></TR><TR><TD> DNA template (251.4ng/ul)
</TD><TD> 1.5ul
</TD><TD> 1.257ng/ul
</TD></TR><TR><TD> Primer F (50uM)
</TD><TD> 3ul
</TD><TD> 500nM
</TD></TR><TR><TD> Primer R (50uM)
</TD><TD> 3ul
</TD><TD> 500nM
</TD></TR><TR><TD> ddH20
</TD><TD> 142.5ul
</TD><TD> -
</TD></TR><TR><TD> Total
</TD><TD> 300ul
</TD><TD> -
</TD></TR></TBODY></TABLE><P><B>PCR reaction conditions:</B></P><TABLE class="table"><TBODY><TR><TH> Step
</TH><TH> Temp (℃)
</TH><TH> Time (s)
</TH></TR><TR><TD> Initial denaturation
</TD><TD> 98
</TD><TD> 60
</TD></TR><TR><TD> 18 cycles
</TD><TD> 98
</TD><TD> 10
</TD></TR><TR><TD> 58, 60, 61, 62, 63, 65
</TD><TD> 15
</TD></TR><TR><TD> 72
</TD><TD> 360
</TD></TR><TR><TD> Final extension
</TD><TD> 72
</TD><TD> 120
</TD></TR><TR><TD> Hold
</TD><TD> 4
</TD><TD> ∞
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="DpnI_digestion">DpnI digestion</SPAN></H2><P>(13 Aug 15:05~19:06)
</P><P>Add 1ul DpnI directly to the amplification system
</P><P>Digest 4h 
</P><P>No termination
</P><P>Stored at -20℃
</P><H2><SPAN class="mw-headline" id="Nanodrop_test_after_digestion">Nanodrop test after digestion</SPAN></H2><TABLE class="table"><TBODY><TR><TH> Anneal Temp (℃)
</TH><TH> 58
</TH><TH> 60
</TH><TH> 61
</TH><TH> 62
</TH></TR><TR><TD> Conc (ng/ul)
</TD><TD> 467.4
</TD><TD> 465.9
</TD><TD> 470.4
</TD><TD> 457.9
</TD></TR><TR><TD> 260/280
</TD><TD> 1.8
</TD><TD> 1.8
</TD><TD> 1.8
</TD><TD> 1.81
</TD></TR><TR><TD> 260/230
</TD><TD> 0.73
</TD><TD> 0.73
</TD><TD> 0.74
</TD><TD> 0.73
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Bacteria_transformation">Bacteria transformation</SPAN></H2><P>(2014.8.13 22:08~2014.8.14 14:50)
</P><P><B>Procedures:</B></P><OL><LI>Immediately, place the tubes on ice, allow cell to thaw on ice, 10 min
</LI><LI>Check the cell to see if they have thawed, gently flick the cells 1-2 times to evenly resuspend the cells.
</LI><LI>Divide the competent cells into 6 tubes,50μL each tube, Mark them with 58, 60, 61, 62, 63, 65. Keep the bacteria in ice during the procedure.
</LI><LI>Add 5μL plasmid to each tube, shaking slightly
</LI><LI>Incubate on ice for 30min
</LI><LI>Heat shock, 42°C, 90s. 
</LI><LI>Keep on ice, 2 min
</LI><LI>Add 200μl SOC medium to each tube, incubate 37°C, 200rpm, 45min
</LI><LI>Centrifuge, 4000rmp, 5min, RT. Discard most of the medium and reserve 50μL. Resuspend.
</LI><LI>Add 50μL bacteria to amp LB agar plate, distribute.
</LI><LI>Incubate the plates at 37°C, 16 hours.
</LI></OL><P><B>Results:</B></P><P>For all 6 groups, no colony forms.
</P><H2><SPAN class="mw-headline" id="Gel_electrophoresis">Gel electrophoresis</SPAN></H2><P><B>Making the agarose gel (1%)</B></P><TABLE class="table"><TBODY><TR><TH> Component
</TH><TH> Volume
</TH></TR><TR><TD> TAE
</TD><TD> 25ml
</TD></TR><TR><TD> Agarose
</TD><TD> 0.25g
</TD></TR><TR><TD> Gene Green
</TD><TD> 0.5ul
</TD></TR></TBODY></TABLE><P><B>Adding samples</B></P><TABLE class="table"><TBODY><TR><TH> Component
</TH><TH> Volume
</TH></TR><TR><TD> PCR product
</TD><TD> 1ul
</TD></TR><TR><TD> 6X loading dye
</TD><TD> 2ul
</TD></TR><TR><TD> TAE
</TD><TD> 9ul
</TD></TR><TR><TD> Total
</TD><TD> 12ul
</TD></TR></TBODY></TABLE><P>DNA marker (Takara DL5000, diluted): 10ul
</P><P><B>Results:</B></P><P>The result of electrophoresis is not clear, so I redid the gel electrophoresis with the same sample.
</P><P>No wanted band (6843bp) is observed.
</P><P>Discussion:
</P><OL><LI>PCR failed, so improving PCR conditions is the first choice. Try extending PCR cycles, denaturation time, annealing time and final extension time.
</LI><LI>Do reaction termination after DpnI enzyme digestion (80℃ 20min) next time because reactive enzyme may binding to the DNA substrate and interfere the transformation efficiency.
</LI><LI>The concentration of template DNA is also very essential, so using 2-step concentration ladder next time.
</LI></OL><H1><SPAN class="mw-headline" id="Agilent.27s_Pointmutation_method_II">Agilent's Pointmutation method II</SPAN></H1><P>8.15
</P><H2><SPAN class="mw-headline" id="PCR">PCR</SPAN></H2><P>(23:35 14th Aug-08:30 15th Aug), about 2h. Next morning, check PCR machine was not at 4℃ hold conditions, and the heating stand was not cold<B>200ul, 50ul/reaction:</B></P><TABLE class="table"><TBODY><TR><TH> Component
</TH><TH> Volume
</TH><TH> Final Concentration
</TH></TR><TR><TD> Q5 master mix 2X
</TD><TD> 100ul
</TD><TD> 1X
</TD></TR><TR><TD> DNA template (251.4ng/ul)
</TD><TD> 1.0ul (A,B), 2.0ul (C,D)
</TD><TD> 1.257ng/ul (A,B), 2.514ng/ul (C,D)
</TD></TR><TR><TD> Primer F (50uM)
</TD><TD> 2ul
</TD><TD> 500nM
</TD></TR><TR><TD> Primer R (50uM)
</TD><TD> 2ul
</TD><TD> 500nM
</TD></TR><TR><TD> ddH20
</TD><TD> 95ul
</TD><TD> -
</TD></TR><TR><TD> Total
</TD><TD> 200ul
</TD><TD> -
</TD></TR></TBODY></TABLE><P><B>PCR reaction conditions:</B></P><TABLE class="table"><TBODY><TR><TH> Step
</TH><TH> Temp (℃)
</TH><TH> Time (s)
</TH></TR><TR><TD> Initial denaturation
</TD><TD> 98
</TD><TD> 60
</TD></TR><TR><TD rowspan="3"> 25 cycles
</TD><TD> 98
</TD><TD> 20
</TD></TR><TR><TD> 63
</TD><TD> 30
</TD></TR><TR><TD> 72
</TD><TD> 420
</TD></TR><TR><TD> Final extension
</TD><TD> 72
</TD><TD> 600
</TD></TR><TR><TD> Hold
</TD><TD> 4
</TD><TD> ∞
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Nanodrop_test">Nanodrop test</SPAN></H2><TABLE class="table"><TBODY><TR><TH> Anneal Temp (℃)
</TH><TH> A
</TH><TH> B
</TH><TH> C
</TH><TH> D
</TH></TR><TR><TD> Conc (ng/ul)
</TD><TD> 474.2
</TD><TD> 492.0
</TD><TD> 466.7
</TD><TD> 445.9
</TD></TR><TR><TD> 260/280
</TD><TD> 1.82
</TD><TD> 1.81
</TD><TD> 1.81
</TD><TD> 1.81
</TD></TR><TR><TD> 260/230
</TD><TD> 0.72
</TD><TD> 0.73
</TD><TD> 0.72
</TD><TD> 0.70
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Gel_electrophoresis_2">Gel electrophoresis</SPAN></H2><P><B>Making the agarose gel (1%)</B></P><TABLE class="table"><TBODY><TR><TH> Component
</TH><TH> Volume
</TH></TR><TR><TD> TAE
</TD><TD> 25ml
</TD></TR><TR><TD> Agarose
</TD><TD> 0.25g
</TD></TR><TR><TD> Gene Green
</TD><TD> 0.5ul
</TD></TR></TBODY></TABLE><P><B>Adding samples</B></P><TABLE class="table"><TBODY><TR><TH> Component
</TH><TH> Volume
</TH></TR><TR><TD> PCR product
</TD><TD> 5ul
</TD></TR><TR><TD> 6X loading dye
</TD><TD> 2ul
</TD></TR><TR><TD> TAE
</TD><TD> 5ul
</TD></TR><TR><TD> Total
</TD><TD> 12ul
</TD></TR></TBODY></TABLE><P>DNA marker (Takara DL5000, diluted): 10ul
</P><P><B>Results:</B></P><P><B>Analysis:</B>
The band was in 2000bp location, which is far away from the original plasmid length (6800bp), which meant PCR didn’t got complete plasmid. The possible reason might be:
</P><OL><LI>The primer forms dimers which strongly influence the efficiency of PCR (The smallest band is very light).
</LI><LI>DNA is a double helix, which means the plasmid double strand can also anneal with each other. The elongation time was very long, which left enough time for the plasmid double strand to anneal with each other and prohibit the PCR process.
</LI></OL><P><B>Solution:</B></P><OL><LI>Redesign the primer so that the primer pairs are not totally complementary with each other. Put the point mutation point near the 5’ end.
</LI><LI>Use other PCR methods.
</LI></OL><P><SPAN style="color: red"><B>So I decided to change the PCR method.</B></SPAN></P><H1><SPAN class="mw-headline" id="2-pair_primers_PCR">2-pair primers PCR</SPAN></H1><P>I had two pairs of primers: one is for mCherry (called A), another is for point mutation (called B). I then used the forward primer of A to pair with the reverse primer of B, so did the forward primer of B with the reverse primer of A. Thus, I would get 2 short segments which had a small complementary region. Then, I added the two segments together and let them do PCR using each other as the template and primer in order to get the final whole length PCR product.
</P><P><B>mCherry primer design:</B>
Forward(5’-3’): TCGTTTAGTGAACCGTCAG 19nt
Reverse(5’-3’): cgaggaattcctattaTTCCTCTGCCCTCACTAG 34nt</P><H2><SPAN class="mw-headline" id="PCR_2">PCR</SPAN></H2><P><B>50ul, 25ul/reaction:</B><I>Reaction 1:</I></P><TABLE class="table"><TBODY><TR><TH> Component
</TH><TH> Volume
</TH><TH> Final Concentration
</TH></TR><TR><TD> Q5 master mix 2X
</TD><TD> 25ul
</TD><TD> 1X
</TD></TR><TR><TD> pBX084 diluted (25.14ng/ul)
</TD><TD> 2.0ul
</TD><TD> 1.257ng/ul
</TD></TR><TR><TD> mCherry Primer F (50uM)
</TD><TD> 0.5ul
</TD><TD> 500nM
</TD></TR><TR><TD> Pointmutation Primer R (50uM)
</TD><TD> 0.5ul
</TD><TD> 500nM
</TD></TR><TR><TD> ddH20
</TD><TD> 22ul
</TD><TD> -
</TD></TR><TR><TD> Total
</TD><TD> 50ul
</TD><TD> -
</TD></TR></TBODY></TABLE><P><I>Reaction 2:</I></P><TABLE class="table"><TBODY><TR><TH> Component
</TH><TH> Volume
</TH><TH> Final Concentration
</TH></TR><TR><TD> Q5 master mix 2X
</TD><TD> 25ul
</TD><TD> 1X
</TD></TR><TR><TD> pBX084 diluted (25.14ng/ul)
</TD><TD> 2.0ul
</TD><TD> 1.257ng/ul
</TD></TR><TR><TD> mCherry Primer R (50uM)
</TD><TD> 0.5ul
</TD><TD> 500nM
</TD></TR><TR><TD> Pointmutation Primer F (50uM)
</TD><TD> 0.5ul
</TD><TD> 500nM
</TD></TR><TR><TD> ddH20
</TD><TD> 22ul
</TD><TD> -
</TD></TR><TR><TD> Total
</TD><TD> 50ul
</TD><TD> -
</TD></TR></TBODY></TABLE><P><B>PCR reaction conditions:</B></P><TABLE class="table"><TBODY><TR><TH> Step
</TH><TH> Temp (℃)
</TH><TH> Time (s)
</TH></TR><TR><TD> Initial denaturation
</TD><TD> 98
</TD><TD> 60
</TD></TR><TR><TD rowspan="3"> 25 cycles
</TD><TD> 98
</TD><TD> 20
</TD></TR><TR><TD> 61
</TD><TD> 15
</TD></TR><TR><TD> 72
</TD><TD> 20
</TD></TR><TR><TD> Final extension
</TD><TD> 72
</TD><TD> 300
</TD></TR><TR><TD> Hold
</TD><TD> 4
</TD><TD> ∞
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Gel_electrophoresis_3">Gel electrophoresis</SPAN></H2><P><B>Making the agarose gel (1%)</B></P><TABLE class="table"><TBODY><TR><TH> Component
</TH><TH> Volume
</TH></TR><TR><TD> TAE
</TD><TD> 25ml
</TD></TR><TR><TD> Agarose
</TD><TD> 0.25g
</TD></TR><TR><TD> Gene Green
</TD><TD> 0.5ul
</TD></TR></TBODY></TABLE><P><B>Adding samples</B></P><TABLE class="table"><TBODY><TR><TH> Component
</TH><TH> Volume
</TH></TR><TR><TD> PCR product
</TD><TD> 10ul
</TD></TR><TR><TD> 6X loading dye
</TD><TD> 2ul
</TD></TR><TR><TD> Total
</TD><TD> 12ul
</TD></TR></TBODY></TABLE><P>DNA marker (Takara DL5000, diluted): 30ul</P><P><B>Results:</B>
A2: reaction 1 (A: mCherry primer F; 2: Pointmutation Primer R)
1B: reaction 2 (B: mCherry primer R; 1: Pointmutation Primer F)</P><P><B>Analysis:</B>
The predicting results is 448bp band in A2 and 372bp band in 1B (almost equally separate the mCherry). But the band in A2 was in 800bp and no band is observed in 1B. The possible reasons might be:</P><OL><LI>I added wrong primer pairs: two mCherry primers for A2 and two pointmutation primers for 1B. So that in A2 I got mCherry and in 1B I got primer dimer.
</LI><LI>The elongation time might be too long.
</LI></OL><P><SPAN style="color: red"><B>So I redid the 2-pari primers PCR with modified PCR reaction conditions.</B></SPAN></P><H2><SPAN class="mw-headline" id="PCR_II">PCR II</SPAN></H2><P><B>Reaction conditions:</B></P><TABLE class="table"><TBODY><TR><TH> Step
</TH><TH> Temp (℃)
</TH><TH> Time (s)
</TH></TR><TR><TD> Initial denaturation
</TD><TD> 98
</TD><TD> 60
</TD></TR><TR><TD rowspan="3"> 25 cycles
</TD><TD> 98
</TD><TD> 20
</TD></TR><TR><TD> 61
</TD><TD> 15
</TD></TR><TR><TD> 72
</TD><TD> 15
</TD></TR><TR><TD> Final extension
</TD><TD> 72
</TD><TD> 120
</TD></TR><TR><TD> Hold
</TD><TD> 4
</TD><TD> ∞
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Gel_electrophoresis:">Gel electrophoresis:</SPAN></H2><P><B>Results:</B>
No band is observed.<B>Analysis:</B></P><OL><LI>The elongation time might be too short…
</LI><LI>The annealing time for 1B might be wrong.
</LI></OL><H1><SPAN class="mw-headline" id="2-pari_primers_PCR_II">2-pari primers PCR II</SPAN></H1><P>8.16 10:25am~11.25am
I set a temperature gradient (45, 48, 51, 54, 57, 60℃) without shortening the elongation time.
</P><H2><SPAN class="mw-headline" id="PCR_3">PCR</SPAN></H2><P>150ul, 25ul/reaction:<I>Reaction 1:</I></P><TABLE class="table"><TBODY><TR><TH> Component
</TH><TH> Volume
</TH><TH> Final Concentration
</TH></TR><TR><TD> Q5 master mix 2X
</TD><TD> 25ul
</TD><TD> 1X
</TD></TR><TR><TD> pBX084 diluted (25.14ng/ul)
</TD><TD> 0.5ul
</TD><TD> 12.57ng
</TD></TR><TR><TD> Pointmutation Primer F (50uM)
</TD><TD> 1ul
</TD><TD> 1000nM
</TD></TR><TR><TD> ddH20
</TD><TD> 23ul
</TD><TD> -
</TD></TR><TR><TD> Total
</TD><TD> 49.5ul
</TD><TD> -
</TD></TR></TBODY></TABLE><P><I>Reaction 2:</I></P><TABLE class="table"><TBODY><TR><TH> Component
</TH><TH> Volume
</TH><TH> Final Concentration
</TH></TR><TR><TD> Q5 master mix 2X
</TD><TD> 25ul
</TD><TD> 1X
</TD></TR><TR><TD> pBX084 diluted (25.14ng/ul)
</TD><TD> 0.5ul
</TD><TD> 12.57ng
</TD></TR><TR><TD> Pointmutation Primer R (50uM)
</TD><TD> 1ul
</TD><TD> 1000nM
</TD></TR><TR><TD> ddH20
</TD><TD> 23ul
</TD><TD> -
</TD></TR><TR><TD> Total
</TD><TD> 49.5ul
</TD><TD> -
</TD></TR></TBODY></TABLE><P><B>PCR reaction conditions:</B></P><TABLE class="table"><TBODY><TR><TH> Step
</TH><TH> Temp (℃)
</TH><TH> Time (s)
</TH></TR><TR><TD> Initial denaturation
</TD><TD> 98
</TD><TD> 60
</TD></TR><TR><TD rowspan="3"> 25 cycles
</TD><TD> 98
</TD><TD> 20
</TD></TR><TR><TD> 63
</TD><TD> 15
</TD></TR><TR><TD> 72
</TD><TD> 5min
</TD></TR><TR><TD> Final extension
</TD><TD> 72
</TD><TD> 10min
</TD></TR><TR><TD> Hold
</TD><TD> 4
</TD><TD> ∞
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Gel_electrophoresis_4">Gel electrophoresis</SPAN></H2><P>11:50am ~12:25pm
Unfortunately, only in A2 did I observe the band. I doubt the annealing temperature was not high enough because strong primer dimers forms in 1B group. So I redid the 2-pair primers PCR again with even higher temperature gradient (62,63,64,65,66,67℃).
</P><H2><SPAN class="mw-headline" id="Gel_electrophoresis_II">Gel electrophoresis II</SPAN></H2><P><B>Results:</B>
Only vague bands were observed in A2. Still no distinct bands observed in 1B.<B>Analysis:</B></P><OL><LI>The reverse primer of pointmutation might be very easy to form dimer with the forward primer of mCherry, which led to the failure of PCR.
</LI><LI>The compatibility of the two separate primers might be very bad.
</LI></OL><P><SPAN style="color: red"><B>I decided to try another methods.</B></SPAN></P><H1><SPAN class="mw-headline" id="Half-plasmid_PCR_.26_Double_primer_PCR">Half-plasmid PCR &amp; Double primer PCR</SPAN></H1><H2><SPAN class="mw-headline" id="Half-plasmid_PCR">Half-plasmid PCR</SPAN></H2><P>First, only add one direction primer so I could get half linear plasmid. After that, I mix the two DNA together to get the complete DNA.
50ul, 25ul/reaction:<I>Reaction 1</I></P><TABLE class="table"><TBODY><TR><TH> Component
</TH><TH> Volume
</TH><TH> Final Concentration
</TH></TR><TR><TD> Q5 master mix 2X
</TD><TD> 25ul
</TD><TD> 1X
</TD></TR><TR><TD> pBX084 diluted (25.14ng/ul)
</TD><TD> 0.5ul
</TD><TD> 12.57ng
</TD></TR><TR><TD> Pointmutation Primer F (50uM)
</TD><TD> 1ul
</TD><TD> 1000nM
</TD></TR><TR><TD> ddH20
</TD><TD> 23ul
</TD><TD> -
</TD></TR><TR><TD> Total
</TD><TD> 49.5ul
</TD><TD> -
</TD></TR></TBODY></TABLE><P><I>Reaction 2</I></P><TABLE class="table"><TBODY><TR><TH> Component
</TH><TH> Volume
</TH><TH> Final Concentration
</TH></TR><TR><TD> Q5 master mix 2X
</TD><TD> 25ul
</TD><TD> 1X
</TD></TR><TR><TD> pBX084 diluted (25.14ng/ul)
</TD><TD> 0.5ul
</TD><TD> 12.57ng
</TD></TR><TR><TD> Pointmutation Primer R (50uM)
</TD><TD> 1ul
</TD><TD> 1000nM
</TD></TR><TR><TD> ddH20
</TD><TD> 23ul
</TD><TD> -
</TD></TR><TR><TD> Total
</TD><TD> 49.5ul
</TD><TD> -
</TD></TR></TBODY></TABLE><P><B>PCR reaction conditions:</B></P><TABLE class="table"><TBODY><TR><TH> Step
</TH><TH> Temp (℃)
</TH><TH> Time (s)
</TH></TR><TR><TD> Initial denaturation
</TD><TD> 98
</TD><TD> 60
</TD></TR><TR><TD rowspan="3"> 25 cycles
</TD><TD> 98
</TD><TD> 20
</TD></TR><TR><TD> 63
</TD><TD> 15
</TD></TR><TR><TD> 72
</TD><TD> 5min
</TD></TR><TR><TD> Final extension
</TD><TD> 72
</TD><TD> 10min
</TD></TR><TR><TD> Hold
</TD><TD> 4
</TD><TD> ∞
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Double_primer_PCR">Double primer PCR</SPAN></H2><P>Mix the two pairs of primer together but the density of the pointmutation primer is only 1/10 of the mCherry primer. So the pointmutation primer will be consumed very quickly in the first several rounds.
150ul, 25ul/reaction:</P><TABLE class="table"><TBODY><TR><TH> Component
</TH><TH> Volume
</TH><TH> Final Concentration
</TH></TR><TR><TD> Q5 master mix 2X
</TD><TD> 75ul
</TD><TD> 1X
</TD></TR><TR><TD> pBX084 diluted (25.14ng/ul)
</TD><TD> 1ul
</TD><TD> 25.14ng
</TD></TR><TR><TD> mCherry Primer F&amp;R (50uM)
</TD><TD> 2ul
</TD><TD> 6666.7nM
</TD></TR><TR><TD> Pointmutation Primer R (50uM)
</TD><TD> 0.2ul
</TD><TD> 666.7nM
</TD></TR><TR><TD> ddH20
</TD><TD> 69.6ul
</TD><TD> -
</TD></TR><TR><TD> Total
</TD><TD> 150ul
</TD><TD> -
</TD></TR></TBODY></TABLE><P><B>PCR reaction conditions:</B></P><TABLE class="table"><TBODY><TR><TH> Step
</TH><TH> Temp (℃)
</TH><TH> Time (s)
</TH></TR><TR><TD> Initial denaturation
</TD><TD> 98
</TD><TD> 60
</TD></TR><TR><TD rowspan="3"> 25 cycles
</TD><TD> 98
</TD><TD> 20
</TD></TR><TR><TD> 45,48,52,55,58,61
</TD><TD> 20
</TD></TR><TR><TD> 72
</TD><TD> 20
</TD></TR><TR><TD> Final extension
</TD><TD> 72
</TD><TD> 5min
</TD></TR><TR><TD> Hold
</TD><TD> 4
</TD><TD> ∞
</TD></TR></TBODY></TABLE><P>The two PCR methods proceed simultaneously from 17:00 to 19:30, then reserved in 4℃.
</P><H2><SPAN class="mw-headline" id="Gel_electrophoresis_5">Gel electrophoresis</SPAN></H2><P>8.17
<B>Analysis:</B>
From Double-primer PCR we got 2 (or 3?) bands. The band which had length between 500~750 were very likely to be the successfully mutated (or the original template). The band between 250~500 might be the combination product of the two primer pairs.
Next I would do gel extraction for the 500~750 band and do BbsI enzyme digestion to verify if the BbsI site was successfully mutated or not.
</P><H2><SPAN class="mw-headline" id="Gel_extraction">Gel extraction</SPAN></H2><P>(13:53~15:20 Aug 15th)
Tiangen Miniprep gel extraction kit (50 rxn)
As the Tiangen’s protocol instructed.
</P><H2><SPAN class="mw-headline" id="Nanodrop_test_2">Nanodrop test</SPAN></H2><TABLE class="table"><TBODY><TR><TH> Band
</TH><TH> 700bp
</TH><TH> 400bp
</TH></TR><TR><TD> Conc (ng/ul)
</TD><TD> 54.7
</TD><TD> 73.2
</TD></TR><TR><TD> 260/280
</TD><TD> 1.094
</TD><TD> 1.464
</TD></TR><TR><TD> 260/230
</TD><TD> 0.634
</TD><TD> 0.445
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="BbsI_enzyme_digestion">BbsI enzyme digestion</SPAN></H2><P>(15:50~20:00, 4h10min)
</P><TABLE class="table"><TBODY><TR><TH> Component
</TH><TH> Volume
</TH></TR><TR><TD> NEB BbsI enzyme
</TD><TD> 0.2ul
</TD></TR><TR><TD> DNA (700bp band)
</TD><TD> 3ul
</TD></TR><TR><TD> 10X NEB buffer 2.1
</TD><TD> 1ul
</TD></TR><TR><TD> ddH20
</TD><TD> 5.8ul
</TD></TR><TR><TD> Total
</TD><TD> 10ul
</TD></TR></TBODY></TABLE><P>75℃ 20min inactivation in dry bath
</P><H2><SPAN class="mw-headline" id="Gel_electrophoresis_6">Gel electrophoresis</SPAN></H2><P>(23:00~23:40)：
<B>Analysis:</B>
After enzyme digestion, the band wasn’t cut into two bands (one about 440, another about 370). 
The next day I would do BbsI and BspEI double digestion to verify the results because BspEI would cut out a 99bp band. 
P.S. The expected band length is 820bp, which seems to be inconsistent with the figure. However, comparing with the PCR result of mCherry, the band location seems to be the same, which means it was likely to be the problem of Marker.
</P><H1><SPAN class="mw-headline" id="Verification_of_the_Point_mutation">Verification of the Point mutation</SPAN></H1><P>8.18 
</P><H2><SPAN class="mw-headline" id="Enzyme_Digestion">Enzyme Digestion</SPAN></H2><H3><SPAN class="mw-headline" id="BbsI.2C_BspEI_double_digestion.2810:05.7E21:30.29">BbsI, BspEI double digestion(10:05~21:30)</SPAN></H3><TABLE class="table"><TBODY><TR><TH> Component
</TH><TH> Volume
</TH></TR><TR><TD> BbsI
</TD><TD> 0.8ul
</TD></TR><TR><TD> BspEI
</TD><TD> 0.2ul
</TD></TR><TR><TD> 700bp band
</TD><TD> 3ul
</TD></TR><TR><TD> 10X NEBuffer 3.1
</TD><TD> 1ul
</TD></TR><TR><TD> ddH2O
</TD><TD> 5ul
</TD></TR><TR><TD> Total
</TD><TD> 10ul
</TD></TR></TBODY></TABLE><P><SPAN style="color: red">The activity of BbsI in NEBuffer 3.1 is only about 25%, so I increase the volumn the BbsI to 4X.</SPAN></P><H3><SPAN class="mw-headline" id="BspEI_digestion_.2815:41.7E21:30.29">BspEI digestion (15:41~21:30)</SPAN></H3><TABLE class="table"><TBODY><TR><TH> Component
</TH><TH> Volume
</TH></TR><TR><TD> BbsI
</TD><TD> 0.2ul
</TD></TR><TR><TD> 700bp band
</TD><TD> 3ul
</TD></TR><TR><TD> 10X NEBuffer 3.1
</TD><TD> 1ul
</TD></TR><TR><TD> ddH2O
</TD><TD> 5.8ul
</TD></TR><TR><TD> Total
</TD><TD> 10ul
</TD></TR></TBODY></TABLE><P>80℃ inactivation, then immediately put into ice to avoid self-ligation.(21:40~22:40）</P><H2><SPAN class="mw-headline" id="Gel_electrophoresis_7">Gel electrophoresis</SPAN></H2><P>(22:50~23:20)
<B>Analysis:</B>
The DNA might decompose due to too long inactivation time (1h at 80℃).
However, Le Lin and Zeng Lidan thought high temp wouldn’t make DNA decompose. It was due to putting in ice immediately that makes DNA renaturation difficult.
So, I decided to do enzyme digestion again without heat inactivation and do gel electrophoresis to see the results. I also will put the enzyme digestion system into 4℃ refrigerator overnight and do gel electrophoresis again the next day to verify if Le and Zeng’s idea was correct.</P><H2><SPAN class="mw-headline" id="Gel_electrophoresis_II_2">Gel electrophoresis II</SPAN></H2><P>8.19
<B>Analysis:</B>
From the gel electrophoresis result, Lin and Zeng’s idea was not correct because again no band is observed. BspEI digestion cut off a 99bp short segment and we can see the band moved quicker than BbsI band. The 273bp band was not observed in BspEI &amp; BbsI double digestion, which means no BbsI site is in the PCR product. 
</P><P><SPAN style="color: red">So the pointmutation was finally successful!!!</SPAN></P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><OL><LI><A href="http://www.genomics.agilent.com/primerDesignProgram.jsp|" class="external text" rel="nofollow">QuikChange Primer Design</A></LI><LI><A href="http://www.genomics.agilent.com/article.jsp?pageId=384|" class="external text" rel="nofollow">QuikChange II Site-Directed Mutagenesis Kits - Details &amp; Specifications</A></LI><LI> Mr. Wan Pei's Advice
</LI></OL></DIV></DIV><FOOTER class="bs-docs-footer col-md-12" role="contentinfo"><DIV class="container"><P>Maintained by the iGEM team SUSTC-Shenzhen.</P><P>Licensed under <A href="http://creativecommons.org/licenses/by/4.0/">CC BY 4.0</A>.</P><UL class="bs-docs-footer-links muted"><LI>©2014 SUSTC-Shenzhen</LI></UL></DIV></FOOTER><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:SUSTC-Shenzhen/Notebook/CRISPR/mCherry-BbsI-pointmutation">http://2014.igem.org/Team:SUSTC-Shenzhen/Notebook/CRISPR/mCherry-BbsI-pointmutation</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SUSTC-Shenzhen/Notebook/CRISPR/mCherry-BbsI-pointmutation" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SUSTC-Shenzhen/Notebook/CRISPR/mCherry-BbsI-pointmutation" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SUSTC-Shenzhen/Notebook/CRISPR/mCherry-BbsI-pointmutation&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SUSTC-Shenzhen/Notebook/CRISPR/mCherry-BbsI-pointmutation&amp;oldid=274530" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>