"
Page
Discussion
View source
History
teams
Log in
 
Team:TMU-Tokyo/Notebook experiment/Result
From 2013.igem.org
Home
Team
Members
Attribution
Project
Idea
Breakthrough
Pythagorean Device
Parts
Future Plan
Reference
HumanPractice
Collaboration
Promote Synthetic Biology
Safety
Lab safety
Questions
Achievement
Notebook
Protocol
Experiment
Fig.1
Fig.2
Fig.3
Fig.4
Assay1
We checked whether our device functioned definitely by the following assays. First, we measured activity of β-galactosidase using X-GAL (5-bromo-4-chloro-3-indolyl-β-D-galactopyranoside). X-GAL is broken down by β-galactosidase and produces a pigment of the blue. When “Genomic Pythagorean Device” functions definitely, the expression of β-galactosidase should be induced by addition of arabinose and this reaction would not happen in the wild type of MG 1655. So, we first confirmed the expression of β-galactosidase after the addition of arabinose in both control group and experimental group. However, As a result, the expression of β-galactosidase wasn’t seen in experimental group. And colonies of control group are seen in light blue. We didn’t understand of this reason (Fig.1).
On the other hand, the addition of IPTG will induce the expression of β-galactosidase to both MG 1655 which have our device (experimental group or which haven’t it (control group). So, then we confirmed the expression of β-galactosidase after the addition of IPTG in both control group and experimental group. As a result, expression of β-galactosidase was seen in both groups (Fig.2). From this result, we could check that this reaction is reversible. Also, we could check that control group is not lacz － stock. (Control Experiment 1)
Then, we checked the expression of β‐galactosidase when we added nothing in LB medium (Fig.3). In this results, the expression of β-galactosidase wasn’t seen in both groups. (Control Experiment 2)
Therefore, we understand that some fragments in our device didn’t work as expected from this assay. In our device, plural chain reactions happen and finally β-galactosidase should be expressed. So, it was difficult to find where the cause that this device didn’t work was.
Assay2
In order to pinpoint the problems in our device, we also carried out following experiments and checked whether inversion happens in Fragment1 by PCR (Fig.4). First, we designed primers 1, 2 and 3. And then we did PCR between primer 1 and 3, and primer 2 and 3. In order to examine to which direction pTet fronts, we checked the DNA band of the PCR product by electrophoresis. If the promoter inversion was happened by site specific recombination which mediated by HbiF, DNA band of PCR product which is performed between primer 1 and primer 3 only appear. DNA band of PCR product which is performed between primer 2 and primer3 should notbe appear. But the results don’t match to our prediction. The products of PCR which is performed either primer combinations didn't appear. The reason is not unclear so we will try to perform PCR with other new primer combinations.
Additionally, we performed PCR in order to check the deletion of lacI because of site specific recombination between two FRT sites in Fragment3. If lacI gene was deleted by the recombination, the PCR products which is performed between two outside primer of fragment3 become smaller than before deletion. If deletion didn’t happened, the PCR products has over 3kbm. On the other hand, the PCR products become only 1.4kb after deletion of lacI.
In our result, the size of all PCR products which are performed between two outside primer of fragment3 are over 3kb. So we suspect that our parts are not completely work. We should try other cultivating condition (for example cultivating over 8 hours), or specify which parts don’t work and fix it.
Retrieved from " http://2013.igem.org/Team:TMU-Tokyo/Notebook_experiment/Result "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
