<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BNU-CHINA_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BNU-CHINA/Description</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team:BNU-CHINA - 2015.igem.org</TITLE></P><DIV class="container"><DIV class="navbar-right"><UL class="nav navbar-nav"><LI class=""><A href="https://2015.igem.org/Team:BNU-CHINA">HOME</A></LI><LI class="active dropdown"><A href="https://2015.igem.org/Team:BNU-CHINA/Project" class="dropdown-toggle" role="button" aria-haspopup="true" aria-expanded="false">PROJECT </A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Project">OVERVIEW</A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Module#bait">BAIT</A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Module#killer">KILLER</A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Module#suicide">SUICIDE</A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Circuit_Design">CIRCUIT DESIGN</A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Results">RESULTS</A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Description">IMPROVEMENT</A></LI></UL><LI class=" dropdown"><A href="https://2015.igem.org/Team:BNU-CHINA/Modeling" class="dropdown-toggle" role="button" aria-haspopup="true" aria-expanded="false">MODELING </A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Modeling#design">DESIGN</A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Modeling#simulation">SIMULATION</A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Modeling#database">DATABASE</A></LI><LI class=" dropdown"><A href="https://2015.igem.org/Team:BNU-CHINA/Achievements" class="dropdown-toggle" role="button" aria-haspopup="true" aria-expanded="false">ACHIEVEMENTS </A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Achievements#parts">PARTS</A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Achievements#judging">JUDGING</A></LI><LI class=""><A href="https://2015.igem.org/Team:BNU-CHINA/Practice" class="text-center">PRACTICES</A></LI><LI class=""><A href="https://2015.igem.org/Team:BNU-CHINA/Safety">SAFETY</A></LI><LI class=" dropdown"><A href="https://2015.igem.org/Team:BNU-CHINA/Team" class="dropdown-toggle" role="button" aria-haspopup="true" aria-expanded="false">TEAM </A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Team">MEMBERS</A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Acknowledgement">ACKNOWLEDGEMENT</A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Notebook">NOTEBOOK</A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Protocol">PROTOCOL</A></LI></DIV></DIV><DIV class="content container"><ARTICLE class="col-lg-10 col-lg-offset-1 col-md-12 col-md-offset-0 col-sm-12 col-sm-offset-0"><HEADER class="page-header"><H1> Project <SMALL> Description </SMALL></H1></HEADER><H2 id="improvement">Improvement</H2><H3>Our Project</H3><P>This year, our project uses the photoinduced system to regulate the synthesis of bait and toxic protein. Our photoinduced system consists of the following four parts. They are Cph8 (<A href="http://parts.igem.org/Part:BBa_K592000">BBa_K592000</A>), pcyA (<A href="http://parts.igem.org/Part:BBa_I15009">BBa_I15009</A>), ho1 (<A href="http://parts.igem.org/Part:BBa_I15008">BBa_I15008</A>) and promoter PompC (<A href="http://parts.igem.org/Part:BBa_R0082">BBa_R0082</A>). </P><H4>cph8</H4><P>The red light sensor (Cph8) is a fusion protein which consists of a phytochrome Cph1 and a histidine kinase domain, Envz-OmpR. Cph1 is the first member of the plant photoreceptor family that has been identified in bacteria. EnvZ-OmpR, a dimeric osmosensor, is a multidomain transmembrane protein. Cph1 can be inactivated under red light, Upon changes of extracellular osmolarity, EnvZ specifically phosphorylates its cognate response regulator OmpR, which, in turn, regulates the PompC. Cph8 can serve as a photoreceptor that regulates gene expression through PompC. Without being exposed to red light, Cph1 keeps being activated and it enables EnvZ-OmpR to autophosphorylate which in turn activates PompC. Under the exposure of red light, however, Cph1 is deactivated, inhibiting the autophosphorylation, thus turning off gene expression.</P><P>We inserted RBS(BBa_B0035) and a constitutive promoter(BBa_J23100) to the upstream of Cph8 gene to make sure that Cph8 can be expressed in <EM>E.coli</EM>.</P><H4>pcyA+ho1</H4><P>Moreover, in order to function well, Cph8 has to form chromophore with PCB biosynthetic genes (<A href="http://parts.igem.org/Part:BBa_I15008">BBa_I15008</A> and <A href="http://parts.igem.org/Part:BBa_I15009">BBa_I15009</A>), where the formation of PCB requires the gene pcyA and hol. Ho1 is a sort of heme oxygenase from <EM>Synechocystis</EM> which can oxidize the heme group using a ferredoxin cofactor, generating biliverdin IXalpha. And then, PcyA, a kind of ferredoxin oxidoreductase from <EM>Synechocystis</EM>, functions to turn biliverdin IXalpha (BV) into PCB.
                    </P><P>We connected gene pcyA and ho1 by the way of overlap PCR, together along with the constitutive promoter(BBa_J23100) to ensure a continuous work of the photoinduced system in our <EM>E.coli</EM>.
                    </P><H4>PompC</H4><P>PompC is a OmpR-controlled promoter which can be positively regulated by phosphorylated OmpR. This promoter is taken from the upstream region of ompC gene. Phosphorylated OmpR binds to the three operator sites and activates transcription.
                    </P><P>We insert PompC into the upstream region of the functional gene gp35 by the way of tranditional cleavage and ligation method, which together was connected to vector pSB1C3 afterwards, Therefore we are able to regulate the expression of bait and toxin through the control of light.
                    </P><H3>Improvement Plan</H3><P> According to earlier research, Cph8 is sensitive to red light, and the maximal response is located around the wavelength 650nm. In order to improve the light sensitive characteristic of Cph8, and to satisfy the different needs of this part, such as regulating the efficiency of the promoter by controling the wave length of the light, we designed an experiment to study the sensibility of the Cph8 to wave length of 650nm nearby. This way we can control the output of the downstream product.
         </P><P>There are 20 experimental groups. We use the bandpass filter to control the wave length of the light from 550nm to 750nm. We set up each experimental group at a gradient of 20nm; The light intensity of each group is controlled at 0.08W/m<SUP>2</SUP>.
                </P><H4>Materials</H4><P>bandpass filters (550 /570…650 nm bandpass filters) , light (82 V, 300 W Philips FocusLine quartz bulb).
                    </P><H4><EM>E. coli</EM> Growth, Light Exposure and Harvesting Protocol</H4><FIGURE class="text-center"><FIGCAPTION>Fig.1 Showing the reaction flow (take the 650nm wavelength experimental group as an example, and in our experiment we replace sfGFP with RFP <SUP><A href="#ref-1">[1]</A></SUP>.
                        </FIGCAPTION></FIGURE><P>The protocol is performed over two consecutive days.
                    </P><OL><LI>
                            Late in the day, start a 37°C, shaking overnight culture from a −80 °C stock in a tube containing 3 mL LB medium and the appropriate antibiotics (50 μg/mL kanamycin, 50 μg/mL ampicillin and 34 μg/mL chloramphenicol).
                        </LI><LI>
                            After the overnight culture has grown for 10–12 h, prepare 100 mL LB medium. Add appropriate antibiotics to medium. Shake/stir the container to ensure the antibiotics are mixed well in the medium.
                        </LI><LI>
                            Measure the OD<SUB>600</SUB> of the overnight culture.
                        </LI><LI>
                            Dilute the overnight culture 1mL into the 100mL LB + antibiotics. Shake/stir the container to ensure the cells are mixed well in the medium.
                        </LI><LI>
                            Place triangle bottles in the shaker and grow at 37°C with shaking at 250 rpm for 8 h. Use the narrow bandpass filter to set the wavelength of each bottles respectively. The intensity of light was measured in power units of watts per square meter using a EPP2000 UVN-SR calibrated spectroradiometer.
                        </LI><LI>
                            After 8 h of growth, harvest all test bottles by immediately transferring them into an ice-water bath. Wait 10 min for the cultures to equilibrate to the cold temperature and for gene expression to stop.
                        </LI><LI>
                            Approximately 1.5 h before stopping the experimental cultures, begin preparing a solution of phosphate-buffered saline (PBS; 137 mM NaCl, 2.7 mM KCl, 10 mM Na<SUB>2</SUB>HPO<SUB>4</SUB>, 2 mM KH<SUB>2</SUB>PO<SUB>4</SUB>, pH to 7.4). Prepare at least 1 mL for each culture to be measured via flow cytometry. At this time, begin preparing a 37°C water bath.
                        </LI><LI>
                            Filter the dissolved solution of PBS through a 0.22-μm 20-mL syringe filter.
                        </LI><LI>
                            Transfer 1 mL of the filtered PBS into one 5-mL cytometer tube per culture sample, and chill tubes in a rack in an ice-water bath.
                        </LI><LI>
                            Incubate the rack(s) of PBS + culture tubes in a 37 °C water bath for 1 h. Our measurements data is for the expression of RFP.
                        </LI><LI>
                            Transfer the rack(s) back into ice-water bath.
                        </LI><LI>
                            Wait 15 min, and then begin measuring each tube on a flow cytometer.
                     </LI></OL><H4>Expected Result</H4><FIGURE class="text-center"><FIGCAPTION>Fig.2 The line graph to describe the sensibility towards different wave lengths of Cph8
                        </FIGCAPTION></FIGURE><P>According to the literatures, what we expected is that the output of the protein produced by experimental groups will increased at the early time, and then decreased later with the increasing of the light frequency. The highest point is at around 650nm. According to the results, we can draw a line graph to describe the sensibility towards different wave lengths of Cph8. The following graph shows the tendency.
                    </P><H3>Our Findings</H3><P>We constructed the PompC-RBS-RFP circuit first (<A href="https://2015.igem.org/Team:BNU-CHINA/Circuit_Design">see cuicuit design page</A>), when we just transformed this circuit into the <EM>E.coli</EM> Top 10, we wondrously found some of the colonies became red. It indicated that these colonies had expressed RFP. It indicated that these colonies had expressed RFP. It means without the regulation of OmpR, promoter PompC can start the transcription of the downstream target gene. And then we sequenced these colonies which expressed RFP. The result indicated that the PompC-RBS-RFP circuit did lead the expression of RFP.</P><FIGURE class="text-center"><FIGCAPTION>Fig. 3 <EM><EM>E.coli</EM></EM> TOP10 transformed the PompC-RBS-rfp circuit.
                    </FIGCAPTION></FIGURE><FIGURE class="text-center"><FIGCAPTION>Fig. 4 The result of sequencing.
                    </FIGCAPTION></FIGURE><P>We detected the light sensibility of red colony later. RFP coding device （<A href="http://parts.igem.org/Part:BBa_J04450">BBa_J04450</A>）, RBS-rfp-terminator (<A href="http://parts.igem.org/Part:BBa_k516032">BBa_K516032</A>) and pSB1C3 were set as controls. We plated 100μL the overnight culture on LB medium + chloramphenicol (34μg/mL) and cultivated them at 37℃. And half of them were with light avoidance treatment. After 12 hours we observed the colonies.
                </P><FIGURE class="text-center"><FIGCAPTION>Fig. 5 Detecting the sensibility to light (from left to right: PompC-rfp, pSB1C3, rfp and RBS-rfp-Ter)
                    </FIGCAPTION></FIGURE><FIGURE class="text-center"><FIGCAPTION>Fig. 6 The results of the light sensibility experiment (1, from left to right: pSB1C3, RBS-rfp-Ter, PompC-rfp and rfp; 2, from left to right: PompC-rfp with light，PompC-rfp without light; 3, from left to right: PompC-rfp without light, rfp with light).
                    </FIGCAPTION></FIGURE><P>We found all the plates transformed PompC-rfp and rfp became red. But the PompC-rfp colonies showed faintly red, and the differences with light or not are not obvious. It shows that PompC-rfp biobrick itself was not sensitive to light. And it also indicated that the PompC promoter has basal activities in <EM>E.coli</EM> TOP10.
                </P><P>Because in nature, this promoter PompC is upstream of the ompC porin gene. The regulation of ompC is determined by the EnvZ-OmpR osmosensing machinery. EnvZ phosphorylates OmpR to OmpR-P. At high osmolarity, EnvZ is more active, creating more OmpR-P. OmpR-P then binds to the low-affinity OmpR operator sites upstream of ompC. <SUP><A href="#ref-3">[3]</A></SUP></P><P>The essence is that the EnvZ protein senses the mediun osmolarity and then forces the OmpR protein to take one of two alternative structures, which positively regulate OmpC synthesis. <SUP><A href="#ref-4">[4]</A></SUP></P><P>So we designed an experiment to detect under the normal level of the EnvZ, the trend of <EM>E.coli</EM> PompC activities with the change of osmotic pressure.
                </P><P>Overnight cultures of Top10 strains transformed with PompC-rfp, rfp, pSB1C3 and RBS-rfp-Ter respectively grown at 37 °C in LB medium containing appropriate antibiotics were diluted at least 1:100 in the medium and incubated at 37 °C as fresh cultures. After their OD<SUB>600</SUB> reached 0.2~0.4, the fresh culture was diluted 1 : 3 into 4 ml of LBON medium(1g Tryptone, 1g Yeast Extract in 100mL H2O). <SUP><A href="#ref-4">[4]</A></SUP> For osmolarity conditions, the cultures were diluted with NaCl supplemented medium to the final concentration of 0%, 0.25%, 0.50% and 1% (wt/vol). After 12 hours of induction, the results are as follows.
                </P><P>With the osmotic pressure increasing, the expression of RFP didn’t increase in experimental groups as we expected. That is to say under natural conditions, the expression of EnvZ-OmpR is too low to regulate the activity of PompC promoter. However, from the pictures we can see the colony of experimental groups still became red. It shows that the existence of EnvZ-OmpR makes the PompC promoter become a little bit active under the natural conditions, the basal activity of the PompC is correspondingly higher. So if we want to try to control the expression of the downstream target gene of the PompC by using EnvZ-OmpR-PompC circuit, we’d better knock out the EnvZ-OmpR gene in the engineering bacteria first.
                </P><FIGURE class="text-center"><FIGCAPTION>Fig. 7 0%， 0.25%， 0.50%， 1% NaCl supplemented to the LBON medium.
                    </FIGCAPTION></FIGURE><P><A href="http://parts.igem.org/Part:BBa_R0082:Experience">The work is added to the Experience Page of BBa_R0082</A>, and a biobrick <A href="http://parts.igem.org/Part:BBa_K1660005">BBa_K1660005</A> is constructed to detect the basal activity of PompC.</P><DIV class="reference"><OL><LI id="ref-1"> Tabor J J, Salis H M, Simpson Z B, et al. A synthetic genetic edge detection program[J]. Cell, 2009, 137(7): 1272-1281.</LI><LI id="ref-2"> Mizuno T M S. Isolation and characterization of deletion mutants of ompR and envZ, regulatory genes for expression of the outer membrane proteins OmpC and OmpF in Escherichia coli.[J]. Journal of Biochemistry, 1987, 101(2):387-396.</LI><LI id="ref-3"> Hall,  M.N. &amp; Silhavy, T.J. (1981) J. Mol. Biol. 151, 1-15.</LI><LI id="ref-4"> Tabor, Jeffrey J., Anselm Levskaya, and Christopher A. Voigt. &quot;Multichromatic control of gene expression in Escherichia coli.&quot; Journal of molecular biology 405.2 (2011): 315-324.
</LI></OL></DIV></ARTICLE></DIV></DIV></DIV></DIV></DIV></BODY></HTML>