Team:BNU-CHINA/Collaboration
Team:BNU-CHINA - 2015.igem.org
HOME
PROJECT
OVERVIEW
BAIT
KILLER
SUICIDE
CIRCUIT DESIGN
RESULTS
IMPROVEMENT
MODELING
DESIGN
SIMULATION
DATABASE
ACHIEVEMENTS
PARTS
JUDGING
PRACTICES
SAFETY
TEAM
MEMBERS
ATTRIBUTIONS
COLLABORATIONS
ACKNOWLEDGEMENT
NOTEBOOK
PROTOCOL
Team
Collaborations
Collaborate with BIT
This year we communicated with team BIT-China, helping each other at several aspects.
1.  We assisted BIT-China with their modeling
It's our pleasure to assist team BIT-China with their modeling part. We mainly helped them to establish a model of substance changes during the pH regulation stage. Specifically, we built a model to link six chemical reactions together, all of which were controlled by pH switch. There is only one chemical reaction with a specific pH, and with the pH changing, the chemical reaction will change subsequently, which makes the pH range from 5 to 7 smoothly. They prove us the conditions and equations of each chemical reaction stage, and we are responsible for the realization of the procedure during the entire process.
The difficulty that BIT-China met was how to connect the six groups of chemical reactions together with the pH switch to find out the reaction time and the results of each chemical reaction, in order to draw a image to reflect the pH variation trend with time. Because some of our team members are majoring in physics, math, computer science and chemistry, we have plenty experience in the solution of differential equations, application of MATLAB and design of procedure. With these resources, we have helped them solve their problems successfully.
The differential reaction equations of the chemical reactions are as follows:
Table 1. The meaning of variables used in differential reaction equations
Variables
Meanings
X1
Promoter induced by base
X2
C(OH-)
X3
p-atp2 and hydroxide ion
X4
Cre mRNA
X5
Cre
X6
J23119 that can express acid
X7
mldhA mRNA
X8
mldhA
X9
IdhA
X10
p-asr
X10
C(H+)
X11
p-asr and H+
X12
Flp mRNA
X13
Flp mRNA
X14
Flp mRNA
X15
J23119 that can express base
X16
gadA mRNA
X17
gadA
X18
GABA
Group A：（condition: 7 \(\leq\) pH \(\leq\) 9
\(\begin{equation} \left\{ \begin{array}{l} dx_1 = -0.003 * x_1 * x_2 + 0.002 * x_3\\ dx_2 = -0.003 * x_1 * x_2 + 0.002 * x_3 - 0.03 * x_2 * x_9 \\ dx_3 = 0.003 * x_1 * x_2 - 0.002 * x_3 \\ dx_4 = 0.02 * x_3 - 0.002 * x_4 \\ dx_5 = 0.03 * x_4 - 0.003 * x_5 \\ dx_6 = 0.02 * x_5 \\ dx_7 = 0.002 * x_6 - 0.02 * x_7 \\ dx_8 = 0.003 * x_7 - 0.03 * x_8 \\ dx_9 = 0.07 * 0.2 * x_8 /(((0.07+0.0005)/0.003)+0.2)- 0.03 * x_2 * x_9 \\ dx_{10} = 0 \\ dx_{11} = 0 \\ dx_{12} = 0 \\ dx_{13} = 0 \\ dx_{14} = 0 \\ dx_{15} = 0\\ dx_{16} = 0\\ dx_{17} = 0\\ dx_{18} = 0\\ \end{array} \right. \end{equation} \)
Group B(condition: 5 < pH <7)
\(\begin{equation} \left\{ \begin{array}{l} dx_1 = 0\\ dx_2 = -0.03 * x_2 * x_9 \\ dx_3 = 0 \\ dx_4 = 0 \\ dx_5 = 0 \\ dx_6 = 0 \\ dx_7 = 0.02 * x_6 - 0.003 * x_7 \\ dx_8 = 0.03 * x_7-0.03 * x_8 \\ dx_9 = 0.07 * 0.2 * x_8 /(((0.07+0.0005)/0.003)+0.2)- 0.03 * x_2 * x_9 \\ dx_{10} = 0 \\ dx_{11} = 0 \\ dx_{12} = 0 \\ dx_{13} = 0 \\ dx_{14} = 0 \\ dx_{15} = 0\\ dx_{16} = 0\\ dx_{17} = 0\\ dx_{18} = 0\\ \end{array} \right. \end{equation} \)
Group C(conditon: pH = 5，terminate condition: x7=0)
\(\begin{equation} \left\{ \begin{array}{l} dx_1 = 0\\ dx_2 = 0 \\ dx_3 = 0 \\ dx_4 = 0 \\ dx_5 = 0 \\ dx_6 = -0.02 * x_{14} * x_6 \\ dx_7 = 0.02 * x_6 - 0.003 * x_7 \\ dx_8 = 0.03 * x_7 - 0.03 * x_8 \\ dx_9 = 0.07 * 0.2 * x_8 / (((0.07+0.0005)/0.003)+0.2)- 0.03 * (10^{-14}/x_{11}) * x_9 \\ dx_{10} = -0.003 * x_{10} * x_{11} + 0.002 * x_{12} \\ dx_{11} = -0.003 * x_{10} * x_{11} + 0.002 * x_{12}\\ dx_{12} = 0.003 * x_{10} * x_{11} - 0.002 * x_{12} \\ dx_{13} = 0.02 * x_{12} - 0.002 * x_{13} \\ dx_{14} = 0.03 * x_3 - 0.003 * x_{14}\\ dx_{15} = 0\\ dx_{16} = 0\\ dx_{17} = 0\\ dx_{18} = 0\\ \end{array} \right. \end{equation} \)
Group D（condition: pH < 5）
\(\begin{equation} \left\{ \begin{array}{l} dx_1 = 0\\ dx_2 = 0 \\ dx_3 = 0 \\ dx_4 = 0 \\ dx_5 = 0 \\ dx_6 = 0 \\ dx_7 = 0 \\ dx_8 = 0 \\ dx_9 = 0\\ dx_{10} = -0.003 * x_10 * x_11 + 0.002 * x_{12} \\ dx_{11} = -0.003 * x_{10} * x_{11} + 0.002 * x_{12} - 0.03 * x_{11} * x_{18}\\ dx_{12} = 0.003 * x_{10} * x_{11} - 0.002 * x_{12} \\ dx_{13} = 0.02 * x_{12} - 0.002 * x_{13} \\ dx_{14} =0.003 * x_{13} - 0.003 * x_{14}\\ dx_{15} = 0.02 * x_{14} \\ dx_{16} = 0.02 * x_{15} - 0.003 * x_{16} \\ dx_{17} = 0.03 * x_{16} - 0.03 * x_{17} \\ dx_{18} = 0.07 * 0.2 * x_{17} /(((0.07+0.0005)/0.003)+0.2- 0.03 * x_{11} * x_{18} \\ \end{array} \right. \end{equation} \)
Group E（condition 5 < pH \(\leq\) 7）
\(\begin{equation} \left\{ \begin{array}{l} dx_1 = 0\\ dx_2 = 0 \\ dx_3 = 0 \\ dx_4 = 0 \\ dx_5 = 0 \\ dx_6 = 0 \\ dx_7 = 0 \\ dx_8 = 0 \\ dx_9 = 0\\ dx_{10} = 0 \\ dx_{11} = -0.03 * x_{11 }* x_{18} \\ dx_{12} =0 \\ dx_{13} = 0 \\ dx_{14} = 0\\ dx_{15} = 0 \\ dx_{16} = 0.02 * x_{15} - 0.003 * x_{16} \\ dx_{17} = 0.03 * x_{16} - 0.03 * x_{17} \\ dx_{18} = 0.07 * 0.2 * x_{17} /(((0.07+0.0005)/0.003)+0.2- 0.03 * x_{11} * x_{18} \\ \end{array} \right. \end{equation} \)
Group F(conditon: pH = 7,terminate condition: x15 = 0)
\(\begin{equation} \left\{ \begin{array}{l} dx_1 = -0.003 * x_1 * x_2 + 0.002 * x_3 \\ dx_2 = -0.003 * x_1 * x_2 + 0.002 * x_3 \\ dx_3 = 0.003 * x_1 * x_2 - 0.002 * x_3 \\ dx_4 = (0.02 * x_3) - 0.002 * x_4\\ dx_5 = 0.03 * x_4 - 0.003 * x_5 \\ dx_6 = 0 \\ dx_7 = 0 \\ dx_8 = 0 \\ dx_9 = 0\\ dx_{10} = 0 \\ dx_{11} = -0.03 * x_{11} * x_{10} \\ dx_{12} =0 \\ dx_{13} = 0 \\ dx_{14} = 0\\ dx_{15} = -0.02 * x_5 * x_{15} \\ dx_{16} = 0.02 * x_{15} - 0.003 * x_{16} \\ dx_{17} = 0.03 * x_{16} - 0.03 * x_{17} \\ dx_{18} = 0.07 * 0.2 * x_{17} / (((0.07+0.0005)/0.003)+0.2）- 0.03 * x_{11} * x_{18} \\ \end{array} \right. \end{equation} \)
After many thorough discussions with BIT-China, we fully understood the background of their modeling , and what problems they wanted to settle. Meanwhile, we helped them find some problems. For example, the variable numbers of each equations were inconsistent, and the parameter settings of some equations didn’t accord with the fact.
First of all, by adjusting a series of independent variables and correcting the parameters, we solved the problems above. We gained the numerical solution of the equation by applying the Runge-Kutta Method as well. The method is suitable for differential equation system in different dimensions with high efficiency. According to concentration of hydroxide ion in equation ABCDE group , we got the pH and use the pH to judge the termination condition. In equation C and F, we jumped out the conditions according to concentration of specific substances. The initial value of every equation group is the terminal value of the last equation group. By this way we achieved the whole cycle process and finally got dynamic image of pH.
Fig. 1 The dynamic change of pH during the regulation process
The initial concentration of OH- is 0.2×10-6, and the initial pH is higher than 7, so the whole reaction starts from Group A, and the reactions are, in order, Group A, B, C, D, E, F, A, B, C, D, E, F.
2.  We helped BIT-China to detect protein NhaA and NhaB.
BIT-China’s project this year is about regulating pH, thus they need to express two kinds of membrane proteins, NhaA and NhaB. Taking account of the fact that membrane proteins are especially hard to express and detect, we offered to help them with this part. To this end, we reviewed an amount of relevant literatures and consulted several professors of this field, on which basis we tried various protocols to express, purify and detect NhaA as well as NhaB.
Initially we did SDS-PAGE of the whole protein of bacteria, and the result is showed as below (Fig. 2). To put it in detail, we centrifuged 5 mL bacterium solution under overnight shake culture at 4°C, 6000 rpm for 10 minutes, which was boiled for 10 minutes, then with protein loading buffer added. Afterwards we ran SDS-PAGE of the mixture with the running gel concentration 12%.
Fig. 2 SDS-PAGE of whole-cell protein
As the whole cell layer was too stick, we couldn’t make an accurate estimation from the result, but anyway there was no sign of expression of the objective proteins whose molecular weight were about 46 kDa. Then we tried normal SDS-PAGE of the homogenate and supernatant and the result came out as Fig.3 has shown.
Fig. 3 Normal SDS-PAGE of NhaA and NhaB
Lane 1, molecular weight standards (kDa); Lane 2-4, the homogenate of pSB1C3, NhaA and NhaB; Lane 5, 6 and 8, the supernatant of pSB1C3, NhaA and NhaB, Bacteria transformed with plasmid pSB1C3 as a negative control.
The bands this time were clearer to see, whereas still there was no significant difference between the empty vector plasmid and the transformed ones.
After two times of unsuccessful attempts, we searched the internet and reviewed some literatures of a reliable protocol for membrane extraction and detection. With the protocols collected we processed the experiment the third time.
Fig.4 SDS-PAGE of extracted membrane proteins
Lane 1, molecular weight standards (kDa); lane 2-4, the sediment of pSB1C3, NhaA and NhaB after centrifuging for the second time; Lane 5-7, the supernatant of pSB1C3, NhaA and NhaB after centrifuging for the second time; Lane 8-10, the supernatant of pSB1C3, NhaA and NhaB after centrifuging for the first time. Bacteria transformed with plasmid pSB1C3 as a negative control.
From this result we can conclude that there are some differences between the bands of controlled plasmid and the experimental ones, whereas it seems these differences have nothing to do with our objective product. So it came out that we failed another time.
Relevant agent formula:
1. Protein lysis buffer
Tris-HCl (pH=8.0)
Sucrose
β-mercaptoethanol
85 mL
25.67 g
15 mL
Membrane solution buffer
Protein lysis buffer : urea solution (8M) = 4 : 1 (volume ratio ))
Therefore, it’s a pity that we failed to detect those two proteins, but we were able to give some opinions to BIT-China about this detection through our experience:
Changing the expression strain specially for overexpressing of NhaA and NhaB according to the literature [1] , the details are in the table 2.
Doing western blot if possible.
This experience makes us learn much about membrane proteins which we believe would work sooner or later. More importantly, it helps us two teams communicate frequently and therefore develop a tight relationship eventually.
Table.2 Bacteria strains specially for overexpressing NhaA
BIT_China for BNU-China:
This year, we collaborated with BNU-China team and held a meeting on 11st. Jul. In this meeting, we exchanged our idea and communicated about our project. Finally, we confirmed our collaboration relationship between our two teams.
Fig.5 Meeting with BNU-CHINA
Because most of BNU-China’s members are fresh, they did not have enough experience about iGEM. After the discussion, we decided that we helped them finish part of their gene circuits. Meanwhile, we suffered some difficulties in characterize two basic parts, NhaA and NhaB. We also meet some problems in modeling, so BNU-China helped us with characterization of our parts and modeling part of our project.
We helped them finish the light regulation system (Pcons+B0034+PcyA+B0034+ho1). In this system, Pcons is a constitutive promoter family member (from J23100 to J23119) which can be used to tune the expression level of express part and what we choose is J23100. B0034 is strong ribosome binding sites (RBS). PcyA and ho1 are two requisite genes which are required for the transformation from heme into PCB. Ho1 will oxidizes the heme group and then generate biliverdin IX alpha, and PcyA converts biliverdin IX alpha into phycocyanobilin (PCB).
Fig.2 Gene circuits of light regulation system
The overlap extension PCR (OE PCR) was used to finish their gene circuits. However, there are secondary structure in this part. So we could not get the part at first. Then we changed DNA polymerase Pfu to PrimerSTAR, another kind of DNA polymerase which performs better than Pfu. Finally the construction of light regulation system is successful.
Fig.3 The construction result of light regulation system
In this period, we also had some discussions about experiments and modeling face to face. Through these discussions, we learnt more about each other and made progress together.
Fig.4 Discussions with BNU_China
Collaborate with ZJU-China
Last year, team ZJU-China constructed a kind of reverse promoter which we hoped to add to our biobricks this year. So they generously provided us with these biobricks. We appreciate ZJU-China very much and we are looking forward to further cooperation in the future.
They constructed two main circuits, first one expresses a serine integrase. This intergrase can exclusively catalyze site-specific recombination between attB and attP, the attachment sites on phage chromosome and host chromosome. This recombination results in the reverse of the sequence between attB and attP, changing the two sites to attL and attR at the meantime. This inversion can be reversible by appropriately controlling the conditional expression of integrase and an excisionase in Bxb1 named gp47 at certain ratio. They change start codon ATG to GTG for a appropriate expression quantity.
The second one contains a switch, two reporters, two RBS and terminators. At first, the plasmid expresses GFP, when gp35 is expressed, the switch will turn around and RFP on the other side of the plasmid is going to be expressed.
Map of those two circuits are shown below.
And figures below show us circuit 1 and 2 using Agarose Gel Electroporesis.
They performed experiment to see whether their circuits work. From the figure above, they found out that when only circuit 2 is transformed (A), GFP is expressed, when both of the two circuits are transformed, the switch turns and RFP is expressed.
Map of those two circuits are shown above.
And figures below show us circuit 1 and 2 using Agarose Gel Electroporesis.
We performed experiment to see whether our circuits work. From the figure above, we can find out that when only circuit 2 is transformed (A), GFP is expressed, when both of the two circuits are transformed, the switch turns and RFP is expressed.
Padan E, Dwivedi M. Overexpression, Isolation, Purification, and Crystallization of NhaA[J]. Methods in Enzymology, 2015, 557.
VIEW OUR ACKNOWLEDGEMENT›
