<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BNU-CHINA_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BNU-CHINA/Results</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team:BNU-CHINA - 2015.igem.org</TITLE></P><DIV class="container"><DIV class="navbar-right"><UL class="nav navbar-nav"><LI class=""><A href="https://2015.igem.org/Team:BNU-CHINA">HOME</A></LI><LI class="active dropdown"><A href="https://2015.igem.org/Team:BNU-CHINA/Project" class="dropdown-toggle" role="button" aria-haspopup="true" aria-expanded="false">PROJECT </A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Project">OVERVIEW</A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Module#bait">BAIT</A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Module#killer">KILLER</A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Module#suicide">SUICIDE</A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Circuit_Design">CIRCUIT DESIGN</A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Results">RESULTS</A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Description">IMPROVEMENT</A></LI></UL><LI class=" dropdown"><A href="https://2015.igem.org/Team:BNU-CHINA/Modeling" class="dropdown-toggle" role="button" aria-haspopup="true" aria-expanded="false">MODELING </A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Modeling#design">DESIGN</A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Modeling#simulation">SIMULATION</A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Modeling#database">DATABASE</A></LI><LI class=" dropdown"><A href="https://2015.igem.org/Team:BNU-CHINA/Achievements" class="dropdown-toggle" role="button" aria-haspopup="true" aria-expanded="false">ACHIEVEMENTS </A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Achievements#parts">PARTS</A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Achievements#judging">JUDGING</A></LI><LI class=""><A href="https://2015.igem.org/Team:BNU-CHINA/Practice" class="text-center">PRACTICES</A></LI><LI class=""><A href="https://2015.igem.org/Team:BNU-CHINA/Safety">SAFETY</A></LI><LI class=" dropdown"><A href="https://2015.igem.org/Team:BNU-CHINA/Team" class="dropdown-toggle" role="button" aria-haspopup="true" aria-expanded="false">TEAM </A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Team">MEMBERS</A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Acknowledgement">ACKNOWLEDGEMENT</A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Notebook">NOTEBOOK</A></LI><LI><A href="https://2015.igem.org/Team:BNU-CHINA/Protocol">PROTOCOL</A></LI></DIV></DIV><DIV class="content container"><ARTICLE class="col-lg-10 col-lg-offset-1 col-md-12 col-md-offset-0 col-sm-12 col-sm-offset-0"><HEADER class="page-header"><H1> Project <SMALL> Results </SMALL></H1></HEADER><H2>l-/d-limonene synthase
</H2><H3>Agarose Gel Electrophoresis of l-/d- Limonene Synthase Gene</H3><FIGURE class="text-center"><DIV class="row"><FIGCAPTION>Fig.1 The agarose gel electrophoresis of l-/d- limonene synthase gene, the bands pointed by arrows represent l-limonene gene and d-limonene gene. Lane M, DNA marker DL2000; Fig1A, Lane 1 and 2, l-LS in pSB1C3 plasmid; Fig 1B, Lane1 and 2, d-LS in pSB1C3 plasmid; Plasmid are digested by EcoRI and PstI.
        </FIGCAPTION></DIV></FIGURE><P>In Fig.1, limonene synthase gene is inserted into the backbone pSB1C3. Then the plasmids are digested by PstI and EcoRI. The length of the l-LS gene, d-LS gene and backbone are respectively 1735bp, 1904bp and 2624bp. The target genes are marked with arrows, and the figure shows that limonene synthase gene was transferred into the <EM>E.coli</EM> BL21(DE3) cells successfully.
</P><H3>SDS-PAGE Analysis of l-/d- Limonene Synthase
</H3><FIGURE class="text-center"><FIGCAPTION>Fig.2 SDS-PAGE of l-limonenen synthase, d-limonene synthase and GPPS. Lane M, molecular weight standards (kDa); Lane 2 and 3, bacteria homogenate of d-limonene synthase; Lane 4 and 5, bacteria homogenate of l-limonene synthase; the lower bands pointed by arrows show GPPS and the upper bands pointed by arrows show limonene synthases.
    </FIGCAPTION></FIGURE><P>In Fig.2，we hope to verify whether the limonene synthase and GPPS gene are expressed or not. The molecular weight of limonene and GPP Synthase is 89 kDa (with GST on pGEX-4T-1 plasmid) and 49 kDa respectively. In Fig. 2, the target proteins are marked with arrows, and the figure shows that synthases are expressed in the cell successfully.
</P><H3>GC-MS analysis of l-/d- limonene
</H3><P>Comparing the differences of TIC between standard samples and test samples, we confirm that standard limonene samples' peak appears at 4.934 minute. And in the figure of our d-limonene sample (n-hexane aq) extracted by <EM>E.coli</EM> BL21, there is a peak at 4.934 minute, which means that our test samples may contain limonene. Then we analyzed the test samples at this time point by mass spectrometric detection. The MS was operated in SIM mode using ions of 136, 68, and 93 m/z. Compared with Fig.3, there are similar peaks at 136, 68, and 93 m/z in Fig.4. The result shows that the <EM>E. coli</EM> we designed express d-limonene successfully.
</P><FIGURE class="text-center"><FIGCAPTION>Fig.3 The result of GC-MS: Control group of pure d-limonene compound.
         (The MS is operated in SIM mode using ions of 136, 68, and 93 m/z.)
    </FIGCAPTION></FIGURE><FIGURE class="text-center"><FIGCAPTION>Fig.4 The result of GC-MS : Experimental group of d-limonene extracted by <EM>E. coli</EM> BL21.
         (The MS is operated in SIM mode using ions of 136, 68, and 93 m/z.)
    </FIGCAPTION></FIGURE><H3>Verification of Attraction of Limonene Towards <EM>C. elegans</EM></H3><P>In order to figure out whether limonene attracts <EM>C. elegans</EM>, filter paper is dropped of 5μL 5% limonene (DMSO aq) and put on one side. The contrasted paper is put on the other side with only 5μL DMSO as the control group. To eliminate the effect of DMSO, two other control groups are set.
</P><TABLE class="table table-condensed"><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH></TR><TR><TD>Control guoup 1</TD><TD>DMSO</TD><TD>117</TD><TD>113</TD></TR><TR><TD>DMSO</TD><TD>110</TD><TD>159</TD></TR><TR><TD>Control group 2</TD><TD>DMSO</TD><TD>180</TD><TD>86</TD><TD>30</TD></TR><TR><TD>M9</TD><TD>198</TD><TD>91</TD><TD>26</TD></TR></TBODY></TABLE><P style="font-family: 'Computer Modern Serif', serif; font-size: 16px; color: rgb(39, 138, 117); text-align:center;">Table 1A. The distribution of the <EM>C. elegans</EM> on the plates of control group Control group is used to eliminate the effect of DMSO, and some other experimental factors.
    </P><TABLE class="table table-condensed"><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH><TH>7</TH><TH>8</TH><TH>9</TH><TH>10</TH><TH>11</TH><TH>12</TH><TH>13</TH><TH>14</TH><TH>15</TH><TH>16</TH><TH>17</TH><TH>18</TH><TH>19</TH><TH>20</TH></TR><TR><TD>DMSO</TD><TD>110</TD><TD>8</TD><TD>61</TD><TD>129</TD><TD>74</TD><TD>90</TD><TD>19</TD><TD>51</TD><TD>172</TD><TD>144</TD><TD>89</TD><TD>125</TD><TD>49</TD><TD>114</TD><TD>28</TD><TD>63</TD><TD>109</TD><TD>346</TD><TD>54</TD><TD>70</TD></TR><TR><TD>Limonene+DMSO</TD><TD>149</TD><TD>64</TD><TD>37</TD><TD>185</TD><TD>128</TD><TD>100</TD><TD>94</TD><TD>150</TD><TD>202</TD><TD>190</TD><TD>114</TD><TD>124</TD><TD>43</TD><TD>129</TD><TD>57</TD><TD>101</TD><TD>261</TD><TD>495</TD><TD>59</TD><TD>79</TD></TR></TBODY></TABLE><P style="font-family: 'Computer Modern Serif', serif; font-size: 16px; color: rgb(39, 138, 117); text-align:center;">Table 1B. The distribution of the <EM>C. elegans</EM> on the plates of experimental group.
    This group shows significantly biased movement by compareing l-limonene with DMSO.
</P><P style="font-family: 'Computer Modern Serif', serif; font-size: 16px; color: rgb(39, 138, 117); text-align:center;">Table 1. The results of verification of limonene to attract <EM>C. elegans</EM>.
</P><P>We find that nematodes of experimental group show significantly biased movement when we count the number of nematodes (Fig.5). However, nematodes of control group show no significant difference about the distribution of nematodes on the plate.
</P><FIGURE class="text-center"><FIGCAPTION>Fig.5 The distribution of <EM>C. elegans</EM> on the plates.
        <P>C stands for the control group in which the filter paper is dropped with 5 \(\mu\)L DMSO; T stands for the experimental group in which the filter paper is dropped with 5 \(\mu\)L 5% l-limonene.
        </P></FIGCAPTION></FIGURE><P>According to the data we get, we did simple analysis, of which we used the ratio of the number of nematodes distributed on two sides and drew a histogram(Fig.6). In the histogram, nematodes distribution of experimental group show significant difference that nematodes prefer limonene.
</P><FIGURE class="text-center"><FIGCAPTION>Fig.6 Proportion of nematodes' distribution.
    </FIGCAPTION></FIGURE><P>During our verification of attraction of limonene, we analyzed 20 samples. In order to verify if there is any difference between test group and the control group at statistic level, we used paired t test. The methods are as follows:
</P><P>The confidence of the following is 0.05, α=0.05. Firstly, we used the normality test. We selected the single sample K-S test to verify whether it had normality.
</P><OL><LI>Judging standard of the single sample K-S test.
        If P &gt; 0.05, it has normality, otherwise it doesn’t have normality.
    </LI><LI>We used SPSS to derive the data and output the results.
    </LI></OL><TABLE class="table table-condensed"><TBODY><TR><TH>Control</TH><TH>Test</TH></TR><TR><TD>Sample size</TD><TD>19</TD><TD>19</TD></TR><TR><TD>Mean</TD><TD>97.0526</TD><TD>143.3684</TD></TR><TR><TD>Variance</TD><TD>74.46362</TD><TD>102.14108</TD></TR><TR><TD>K-S statistics</TD><TD>0.767</TD><TD>0.920</TD></TR><TR><TD>P value</TD><TD>0.598</TD><TD>0.366</TD></TR></TBODY></TABLE><P style="font-family: 'Computer Modern Serif', serif; font-size: 16px; color: rgb(39, 138, 117); text-align:center;">Table 2. The output of the normality test.</P><H4>Analysis</H4><P>The p value of the control group is 0.598, while that of the test group is 0.366. Both of these two groups meet normality.
</P><P>After testing the normality, we did a significance test——paired t test to verify if there was any difference between these two groups.
</P><H5>Set Up A Hypothesis</H5><P>$$H_0: \mu1 = \mu2$$</P><P>$$H1: \mu1
    &lt; \mu2$$</P><P>Among them, \(\mu_1\) is the mean of the control group, \(\mu_2\) is the mean of the test group.</P><H5>The Judging Standard of The Paired T Test
</H5><P>If p
            &lt; 0.05, we reject <EM>H<SUB>0</SUB></EM> and accept <EM>H<SUB>1</SUB></EM> and the limonene can attract the nematodes. And if p &gt; = 0.05, we accept <EM>H<SUB>0</SUB></EM> and the limonene has no effect on the nematodes.
        </P><H5>Output The Results</H5><TABLE class="table table-condensed"><TBODY><TR><TH>Paring difference</TH></TR><TR><TD>95 % confidence interval of difference</TD></TR><TR><TD>Mean</TD><TD>Standard deviation</TD><TD>The standard error of mean</TD><TD>Lower limit</TD><TD>Upper limit</TD></TR><TR><TD>paired control - test
                    </TD><TD>-46.31579</TD><TD>45.38484</TD><TD>10.41200</TD><TD>-68.19058</TD><TD>-24.44100</TD></TR></TBODY></TABLE><P style="font-family: 'Computer Modern Serif', serif; font-size: 16px; color: rgb(39, 138, 117); text-align:center;">Table 3. Paring sample test.</P><TABLE class="table table-condensed"><TBODY><TR><TH>T statistics</TH><TH>Freedom</TH><TH>P value</TH></TR><TR><TD>Paired control-test <EM>d</EM></TD><TD>-4.448</TD><TD>18</TD><TD>0.000</TD></TR></TBODY></TABLE><P style="font-family: 'Computer Modern Serif', serif; font-size: 16px; color: rgb(39, 138, 117); text-align:center;">Table 4. Paring sample test.
        </P><H2>Bace 16</H2><H3>Agarose Gel Electrophoresis of Bace16</H3><P>We did restriction endonuclease analysis to test if we had successfully synthesized bace16 with pBAD promoter(BBa_K206000) in its upstream region. We digested the plasmid with restriction enzymes EcoRI and PstI. Then we did agarose gel electrophoresis and found that the plasmid was digested into two fragments, one of which was about 1300bp long(Fig.7). This proves that we have successfully built the biobrick bace16-pSB1C3 as we planned earlier.
        </P><FIGURE class="text-center"><FIGCAPTION>Fig.7 the agarose gel electrophoresis of gene bace16, the bands pointed by arrows represent bace16 genes. Lane M, DNA marker DL2000; Lane 1 and 2, bace16-pSB1C3 plasmids digested by EcoRI and PstI.
            </FIGCAPTION></FIGURE><H3>SDS-PAGE of Bace 16
</H3><P>After we successfully built this biobrick, we transferred the vectors into <EM>E.coli</EM> BW25113 to express Bace16 protein. SDS-PAGE was done to testify the expression of Bace 16 protein. The expression of Bace16 was induced by L-Arabinose at the concentrations ranging from 8\(\mu\)M-12\(\mu\)M, culturing was at 26℃ in shaking incubator for over 5h. As the literature indicated<SUP><B><A href="#ref-1">[1]</A></B></SUP>, Bace16 is an extracellular secretion protein, so we gathered the supernatant to test the exist of it. According to the SDS-PAGE figure, we failed to expressed Bace16 in <EM>E.coli</EM> BW25113(Fig.8, 9)
        </P><FIGURE class="text-center"><FIGCAPTION>Fig.8 SDS-PAGE of Bace16. Lane 1, molecular weight standards (kDa); Lane 2-5, ammonium sulfate precipitation of the supernatant after 0, 8 \(\mu\)M, 10uM and 12 \(\mu\)M L-Ara induction respectively; Lane 6-9, total bacterial lysate after 0, 8 \(\mu\)M, 10 \(\mu\)M and 12 \(\mu\)M L-Ara induction respectively.
          </FIGCAPTION></FIGURE><FIGURE class="text-center"><FIGCAPTION>Fig.9 SDS-PAGE of Bace16. Lane 1, molecular weight standards (kDa); Lane 2-5, the supernatant after 0, 8 \(\mu\)M, 10 \(\mu\)M and 12 \(\mu\)M L-Ara induction respectively, after 11000 rpm for 10 min; Lane 6-9,the sediment.
            </FIGCAPTION></FIGURE><H3>Discussion</H3><P>The molecular mass of Bace16 mature protein is 28kDa, but the SDS-PAGE analysis did not show the 28kDa band. There are mainly two reasons for this result.</P><P>As Bace16 protein is an extracellular secretion protein in nature<SUP><B><A href="#ref-1">[2]</A></B></SUP>, we gathered the supernatant and did ammonium sulfate precipitation, so if the ammonium sulfate precipitation was not successfully conducted, we would not observe the target band in our experiment.</P><P>Bace16 is a serine protease containing a presequence signal peptide of 30 amino acids and a propeptide of 77 amino acids<SUP><B><A href="#ref-2">[2]</A></B></SUP>. We add the presequence and prosequence when we designing the biobrick, thusmay lead the bace16 not be expressed well in <EM>E. coli</EM>. Besides, it might be degrased by the proteins from <EM>E. coli</EM>, so we couldn’t detect the expression of Bace16 in <EM>E. coli</EM>.
        </P><H2>rMpL</H2><H3>Agarose Gel Electrophoresis of rMpL</H3><P>Firstly, we did restriction digestion analysis to test if we had successfully synthesized rMpL gene with pBAD promoter(BBa_K206000) in its upstream region. We digested the plasmid with restriction enzymes EcoRI and PstI. Then we did agarose gel electrophoresis(AGE) and found that the plasimd was digested into two fragments, one of which was about 700bp long, as we expected(Fig.10). It proves that we have successfully built the vector rMpL-pSB1C3 as we planned earlier.
        </P><FIGURE class="text-center"><FIGCAPTION>Fig.10 The agarose gel electrophoresis of gene rMpL, the bands pointed by arrows represent rMpL genes. Lane M,DNA marker; Lane 1-4, rMpL-pSB1C3 plasmids digested by EcoRI and PstI.
            </FIGCAPTION></FIGURE><P>Besides, we have built another vector including rMpL gene. The backbone of this vector was also pSB1C3, while the promoter of rMpL has been changed into a constitutive promoter(BBa_J23100). We also did restriction endonuclease analysis on this vector either. The agarose gel electrophoresis figure shows that we successfully built this part too.
        </P><H3>SDS-PAGE</H3><P>After the vectors being constructed successfully , we transferred them into different <EM>E.coli</EM> strains depending on the kinds of vectors .Vectors with pBAD promoter were transferred into <EM>E.coli BW25113</EM> and those with constitutive promoter into <EM>E.coli</EM> BL21(DE3) to express rMpL protein respectively. After rMpl gene was expressed in the bacteria, we did SDS-PAGE to testify the expression of rMpL protein. And according to the SDS-PAGE figure, we expressed rMpL in both <EM>E.coli</EM> strains successful.(Fig.11, 12)
        </P><FIGURE class="text-center"><FIGCAPTION>Fig.11 SDS-PAGE of rMpL
            </FIGCAPTION></FIGURE><P>Lane 1, molecular weight standards (kDa); lane 2-5, supernatant after 1.2, 1.2, 1.0, 1.0 \(\mu\)M L-Ara induction respectively; Lane 7, supernatant after 0 \(\mu\)M L-Ara induction. Lane 8: supernatant of the pSB1C3; Lane 9,10,12,13,14: homogenate after 1.2, 1.2, 1.0, 1.0, \(\mu\)M L-Ara induction respectively; Lane 15: homogenate of the pSB1C3.
        </P><FIGURE class="text-center"><FIGCAPTION>Fig.12 SDS-PAGE of rMpL (constitutive promoter ) decoloration for five hours.
            </FIGCAPTION></FIGURE><FIGURE class="text-center"><FIGCAPTION>Fig.13 SDS-PAGE of rMpL (constitutive promoter) decoloration for ten hours.
            </FIGCAPTION></FIGURE><FIGURE class="text-center"><FIGCAPTION>Fig.14 SDS-PAGE of rMpL expressed in <EM>E.coli</EM> BL21(DE3)
                 Lane 1, supernatant of pSB1C3-BL21(DE3) with constitutive promoter; Lane 2, supernatant of pSB1C3-BW25113 with pBAD promoter after 0 \(mu\)M L-Ara induction; Lane 3, supernatant of pSB1C3. </FIGCAPTION></FIGURE><H3>Nematoxicity Test
</H3><P>At last, we did nematoxicity test to find if rMpL protein did have the ability to prevent <EM>C.elegans</EM> from developing to adulthood from larva. As we showed in the microscopic figures below, nematodes treated with recombinant bacteria show significant difficulty in developing into adulthood according to several measurement index such as the length of the nematodes and the motion activity of nematodes. While the control group showed the opposite phenomena, which is to say the worms can develop to adulthood normally. And as for the motility of the nematodes, the nematodes which ate the bacteria-expressed rMpL moved slower than control group. The following figures shows the differences between the experimental groups and the control groups.
        </P><FIGURE class="text-center"><FIGCAPTION>Fig.15 After nematodes ate the bacteria only with pSB1C3.

      </FIGCAPTION></FIGURE><FIGURE class="text-center"><FIGCAPTION>Fig.16 After nematodes ate the bacteria with rMpL.
            </FIGCAPTION></FIGURE><H3>Discussion </H3><P>During our experiment, we found an interesting phenomenon. When we changed our pBAD promoter to constitutive promoter, and growth of the <EM>E.coli</EM> became much slower(Fig.17). pSB1C3-BW25113 is the bacteria with the pBAD promoter and the pSB1C3-BL21(DE3) is the bacteria with constitutive promoter(BBa_J23100). We can easily find the difference.
        </P><FIGURE class="text-center"><FIGCAPTION>Fig.18 Cultivating for 5 hours.
            </FIGCAPTION></FIGURE><FIGURE class="text-center"><FIGCAPTION>Fig.19 Cultivating for 10 hours.
            </FIGCAPTION></FIGURE><P>We think it may because rMpL could do harm to the <EM>E.coli</EM> themselves. Hence, it verifies that the pBAD promoter is more suitable than constitutive promoter.
        </P><H2>Circuit Design and Test</H2><H3>pcyA+ho1</H3><P>We connected gene pcyA（<A href="http://parts.igem.org/Part:BBa_I15009">BBa_I15009</A>）and ho1（<A href="http://parts.igem.org/Part:BBa_I15008">BBa_I15008</A>）together along with the constitutive promoter(<A href="http://parts.igem.org/Part:BBa_J23100">BBa_J23100</A>) through overlap PCR. After we transformed the restriction-ligation product into <EM>E.coli</EM> TOP10, we did colony PCR to test if we had successfully synthesized this biobrick. The agarose gel electrophoresis showes that this biobrick was successfully constructed(Fig.20). And this work was helped by BIT-CHINA, for more information, please see the <A href="https://2015.igem.org/Team:BNU-CHINA/Collaborations">collaboration section</A>.
        </P><FIGURE class="text-center"><FIGCAPTION>Fig.20 The agarose gel electrophoresis of pcyA+ho1, the bands pointed by arrows represent the expected result. Lane M, DNA marker DL2000; Lane 1 to 12, colony PCR of pcyA+ho1 biobrick; Lane 11, presents an approximately 1500bp of pcyA+ho1 band which is as expected.
            </FIGCAPTION></FIGURE><H3>PompC+RFP
</H3><P>We constructed the PompC-RBS-RFP circuit first(Fig.26), when we just transformed this circuit into the <EM>E.coli</EM> Top 10, we wondrously found some of the colonies became red. It indicated that these colonies had expressed RFP. It indicated that these colonies had expressed RFP. It means without the regulation of OmpR, promoter PompC can start the transcription of the downstream target gene. And then we sequenced these colonies which expressed RFP. The result indicated that the PompC-RBS-RFP circuit did lead the expression of RFP.</P><FIGURE class="text-center"><FIGCAPTION>Fig.21 <EM><EM>E.coli</EM></EM> TOP10 transformed the PompC-RBS-rfp circuit.
            </FIGCAPTION></FIGURE><FIGURE class="text-center"><FIGCAPTION>Fig.22 The result of sequencing.
            </FIGCAPTION></FIGURE><P>We detected the light sensibility of red colony later. RFP coding device （<A href="http://parts.igem.org/Part:BBa_J04450">BBa_J04450</A>）, RBS-rfp-terminator (<A href="http://parts.igem.org/Part:BBa_k516032">BBa_K516032</A>) and pSB1C3 were set as controls. We plated 100μL the overnight culture on LB medium + chloramphenicol (34μg/mL) and cultivated them at 37℃. And half of them were with light avoidance treatment. After 12 hours we observed the colonies.
        </P><FIGURE class="text-center"><FIGCAPTION>Fig.23 Detecting the sensibility to light (from left to right: PompC-rfp, pSB1C3, rfp and RBS-rfp-Ter)
            </FIGCAPTION></FIGURE><FIGURE class="text-center"><FIGCAPTION>Fig.24 The results of the light sensibility experiment (1, from left to right: pSB1C3, RBS-rfp-Ter, PompC-rfp and rfp; 2, from left to right: PompC-rfp with light，PompC-rfp without light; 3, from left to right: PompC-rfp without light, rfp with light).
            </FIGCAPTION></FIGURE><P>We found all the plates transformed PompC-rfp and rfp became red. But the PompC-rfp colonies showed faintly red, and the differences with light or not are not obvious. It shows that PompC-rfp biobrick itself was not sensitive to light. And it also indicated that the PompC promoter has basal activities in <EM>E.coli</EM> TOP10.
        </P><P>Because in nature, this promoter PompC is upstream of the ompC porin gene. The regulation of ompC is determined by the EnvZ-OmpR osmosensing machinery. EnvZ phosphorylates OmpR to OmpR-P. At high osmolarity, EnvZ is more active, creating more OmpR-P. OmpR-P then binds to the low-affinity OmpR operator sites upstream of ompC. <SUP><A href="#ref-3"><B>[3]</B></A></SUP></P><P>The essence is that the EnvZ protein senses the mediun osmolarity and then forces the OmpR protein to take one of two alternative structures, which positively regulate OmpC synthesis. <SUP><A href="#ref-4">[4]</A></SUP></P><P>So we designed an experiment to detect under the normal level of the EnvZ, the trend of <EM>E.coli</EM> PompC activities with the change of osmotic pressure.
        </P><P>Overnight cultures of Top10 strains transformed with PompC-rfp, rfp, pSB1C3 and RBS-rfp-Ter respectively grown at 37 °C in LB medium containing appropriate antibiotics were diluted at least 1:100 in the medium and incubated at 37 °C as fresh cultures. After their OD<SUB>600</SUB> reached 0.2~0.4, the fresh culture was diluted 1 : 3 into 4 ml of LBON medium(1g Tryptone, 1g Yeast Extract in 100mL H<SUB>2</SUB>O). <SUP><A href="#ref-4"><B>[4]</B></A></SUP> For osmolarity conditions, the cultures were diluted with NaCl supplemented medium to the final concentration of 0%, 0.25%, 0.50% and 1% (wt/vol). After 12 hours of induction, the results are as follows.
        </P><P>With the osmotic pressure increasing, the expression of RFP didn’t increase in experimental groups as we expected. That is to say under natural conditions, the expression of EnvZ-OmpR is too low to regulate the activity of PompC promoter. However, from the pictures we can see the colony of experimental groups still became red. It shows that the existence of EnvZ-OmpR makes the PompC promoter become a little bit active under the natural conditions, the basal activity of the PompC is correspondingly higher. So if we want to try to control the expression of the downstream target gene of the PompC by using EnvZ-OmpR-PompC circuit, we’d better knock out the EnvZ-OmpR gene in the engineering bacteria first.
        </P><FIGURE class="text-center"><FIGCAPTION>Fig.25 0%， 0.25%， 0.50%， 1% NaCl supplemented to the LBON medium.
            </FIGCAPTION></FIGURE><H3>PompC+6N+gp35
</H3><P>We inserted the gp35 gene into the downstream region of the PompC promotor by means of 3A Assembly. After we transformed the restriction-ligation product into E.coli TOP10, we did colony pcr. The agarose gel electrophoresis showed that this biobrick was successfully constructed (Fig.26).
        </P><FIGURE class="text-center"><FIGCAPTION>Fig.26 the agarose gel electrophoresis of PompC+RFP and PompC+6N+gp35, the bands pointed by arrows represent the expected result. Lane 1,DNA marker DL5000; Lane 2, PompC+RFP; Lane3, PompC+6N+gp35</FIGCAPTION></FIGURE><DIV class="reference"><OL><LI id="ref-1">Niu Q, Huang X, Zhang L, et al. From the Cover: A Trojan horse mechanism of bacterial pathogenesis against nematodes[J]. Proceedings of the National Academy of Sciences of the United States of America, 2010, 107(38):16631-16636.
                </LI><LI id="ref-2">Day R M, Thalhauser C J, Sudmeier J L, et al. Tautomerism, acid‐base equilibria, and H‐bonding of the six histidines in subtilisin BPN′ by NMR[J]. Protein Science, 2003, 12(4): 794-810.
                </LI><LI id="ref-3">Mizuno T M S. Isolation and characterization of deletion mutants of ompR and envZ, regulatory genes for expression of the outer membrane proteins OmpC and OmpF in Escherichia coli.[J]. Journal of Biochemistry, 1987, 101(2):387-396.
                </LI><LI id="ref-4">Hall, M.N. &amp; Silhavy, T.J. (1981) J. Mol. Biol. 151, 1-15</LI></OL></DIV></ARTICLE></DIV></DIV></DIV></DIV></DIV></BODY></HTML>