<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_USTC_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:USTC/Results</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar"><NAV class="blue darken-4"><DIV class="nav-wrapper hoverable" style="z-index:2"><DIV id="nav-mobile" class="right hide-on-med-and-down"><H4 class="header white-text lighten-5" style="padding-right:4px;">Nanomachine Detecting Microbiotics </H4></DIV></DIV></NAV></DIV><DIV class="Nav2nd row white z-depth-1"><DIV class="Nav1th col s2 blue darken-3"><A class="dropdown-button btn white-text" data-beloworigin="true" data-hover="true" href="#!" data-activates="Nav-dropdown">RESULTS<I class="material-icons">arrow_drop_down</I></A></DIV><DIV class="col s10" style="z-index:2;"><UL class="tabs tabs-wrapper transparent"><LI class="tab col s1"><A href="#Overview" class="blue-text active waves-effect waves-light">Overview</A></LI><LI class="tab col s2"><A href="#Permeability_Improvement" class="blue-text waves-effect waves-light">Permeability Improvement</A></LI><LI class="tab col s2"><A href="#Chemotaxis_Modification_and_ROSE" class="blue-text waves-effect waves-light">Chemotaxis
                    &amp; ROSE</A></LI><LI class="tab col s2"><A href="#Adhesion_Assay" class="blue-text waves-effect waves-light">Adhesion Assay</A></LI><LI class="tab col s2"><A href="#Film_Determination" class="blue-text waves-effect waves-light">Film Determination</A></LI><LI class="tab col s1"><A href="#Calibration" class="blue-text waves-effect waves-light">Calibration</A></LI></UL></DIV></DIV><UL id="Nav-dropdown" class="dropdown-content"><LI><A href="https://2015.igem.org/Team:USTC" class="waves-effect waves-light">Home</A></LI><LI><A href="https://2015.igem.org/Team:USTC/Description" class="waves-effect waves-light">Project</A></LI><LI><A href="https://2015.igem.org/Team:USTC/Modeling" class="waves-effect waves-light">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:USTC/Achievements" class="waves-effect waves-light">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:USTC/Measurement" class="waves-effect waves-light">Measurement</A></LI><LI><A href="https://2015.igem.org/Team:USTC/Software" class="waves-effect waves-light">Software</A></LI><LI><A href="https://2015.igem.org/Team:USTC/Parts" class="waves-effect waves-light">Parts</A></LI><LI><A href="https://2015.igem.org/Team:USTC/Tutorials" class="waves-effect waves-light">Tutorials</A></LI><LI><A href="https://2015.igem.org/Team:USTC/Notebook" class="waves-effect waves-light">NoteBook</A></LI><LI><A href="https://2015.igem.org/Team:USTC/Safety" class="waves-effect waves-light">Safety</A></LI><LI><A href="https://2015.igem.org/Team:USTC/Practices" class="waves-effect waves-light">Policy&amp;Practices</A></LI><LI><A href="https://2015.igem.org/Team:USTC/Team" class="waves-effect waves-light">Team</A></LI><LI><A href="https://2015.igem.org/Team:USTC/Collaborations" class="waves-effect waves-light">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:USTC/Attributions" class="waves-effect waves-light">Attributions</A></LI></UL><DIV class="row"><DIV class="col offset-m1 offset-l2 s12 m10 l8"><DIV id="Overview" class="row"><DIV class="card hoverable"><DIV class="col s12 m9"><DIV class="card-content"><P>That is so amazing! We finally made NDM this year! The results page will deliver the fresh
                            results from us. Our results include these sections:</P><UL><LI><P>Permeability Improvement: In this section, we will demonstrate our synthetic
                                bioloical base constrcuction, characterization of permeability improvement. This is the
                                fundation of CACCI construction, as well as NDM.</P></LI><LI><P>Chemotaxis Modification: This section introduces results of chemotaxis
                                engineering.</P></LI><LI><P>Adhesion Assay: This section explicitly explain adhesion methods, adhesion dynamics
                                and adhesion protocols recommended for all users, which is an important part of our
                                results.</P></LI><LI><P>Film Determination: This section introduced how we finally determinated our film,
                                from plastic film to our final candidate. See how difficult and interesting it is!</P></LI><LI><P>Calibration: This is our final step demonstrating the feasibility of NDM. See how
                                perfect our project is!</P></LI></UL></DIV></DIV></DIV></DIV><DIV id="Permeability_Improvement" class="row"><DIV class="card hoverable"><DIV class="row"><DIV class="col s12 m9"><DIV class="card-content"><H4 id="Extraction_of_Permeability_Improvement_Fragements" class="scrollspy">Extraction of
                                Permeability Improvement Fragements</H4><P>We successfully get
                                gene fragements that are used for improving bacterial permeability, which including,
                            </P><UL><LI>SCVE, a viroporin originally from SARS virus, is synthesized by Sangon Co. ctd.</LI></UL><FIGURE><FIGCAPTION>
                              Figure 1: Extraction of SCVE fragement
                              </FIGCAPTION></FIGURE><UL><LI>OprF, a bigger porin compared with OmpF in <EM>E. coli</EM>, is extracted from <EM>P.aeruginosaï¼Œ
                                    PAO1</EM> genome by PCR.
                                </LI></UL><FIGURE><FIGCAPTION>
                              Figure 2: Extraction of OprF fragement
                              </FIGCAPTION></FIGURE><UL><LI><P>T7 is a strong promoter which we got form Parts Registry. T7 will be used to
                                    trigger the expression of both OprF and SCVE.<FIGURE><FIGCAPTION>
                                Figure 3: T7 extraction from parts
                                </FIGCAPTION></FIGURE></P></LI><LI><P>Cas9 is got from 2015 Part Distribution.</P></LI></UL><FIGURE><FIGCAPTION>
                              Figure 4: Extraction of Cas9 from parts
                              </FIGCAPTION></FIGURE><UL><LI>gRNA-AcrB and gRNA-EmrE, these two fragments are for silencing bacterial
                                    transmembrane protein AcrB and EmrE to strongly block drug efflux system. These two
                                    parts are synthesized by Sangon Biotech Company.
                                </LI></UL><FIGURE><FIGCAPTION>
                              Figure 5: Extraction of gRNA-AceB and gRNA-EmrE, two fragments are synthesized by Sangon Biotech Company
                              </FIGCAPTION></FIGURE><H4 id="Construction_of_Permeability_Improving_Plasmid" class="scrollspy">Construction of
                                Permeability Improving Plasmid</H4><UL><LI>Construction of plasimid with T7-SCVE (<A href="http://parts.igem.org/Part:BBa_K1593667">BBa_K1593667</A>) . We finally
                                    ligated the T7 with SCVE and got T7-SCVE (<A href="http://parts.igem.org/Part:BBa_K1593667">BBa_K1593667</A>).
                                </LI></UL><FIGURE><FIGCAPTION>
                              Figure 6: Construction of T7-SCVE ([BBa_K1593667](http://parts.igem.org/Part:BBa_K1593667))
                              </FIGCAPTION></FIGURE><UL><LI>Construction of plasimid containing T7-OprF (<A href="http://parts.igem.org/Part:BBa_K1593210">BBa_K1593210</A>). This is the
                                    result of T7 with OprF.
                                </LI></UL><FIGURE><FIGCAPTION>
                              Figure 7: Construction of T7-OprF ([BBa_K1593210](http://parts.igem.org/Part:BBa_K1593210))
                              </FIGCAPTION></FIGURE><H4 id="Growth_Characterization" class="scrollspy">Growth Characterization</H4><P>We firstly characterized the growth rate of genetically modified CACCI along with wild
                                type BL21 using optical density at 600 nm, which is called OD 600, in order to
                                demonstrate that the overexpression of porin(OprF with T7,<A href="http://parts.igem.org/Part:BBa_K1593209">BBa_K1593209</A>)
                                or viroporin(SCVE with T7, (<A href="http://parts.igem.org/Part:BBa_K1593667">BBa_K1593667</A>)
                                won't severely affect the growth of bacteria, at least it won't significantly inhibit
                                bacteria growth and become a kill switch.</P><P>All bacteria are cultured previously in LB medium about 12 h. And then the measurement of
                                bacteria growth through time began at 8 A.M. the other day.</P><FIGURE><FIGCAPTION>
                              Figure 8: Bacteria growth characterization
                              </FIGCAPTION></FIGURE><P>As the images above illustrated, we found <EM>E. coli</EM> BL21 wild type has the best
                                growth characteristics with a maximal OD600=3.427. Whereas <EM>E. coli</EM> BL21 with
                                OprF (<A href="http://parts.igem.org/Part:BBa_K1593210">BBa_K1593210</A>) has a
                                relatively low optical density as its OD600 after 12 h is 2.808 compared to the
                                wildtype. In the same way, characterization of <EM>E. coli</EM> BL21 with SCVE, <A href="http://parts.igem.org/Part:BBa_K1593667">http://parts.igem.org/Part:BBa_K1593667</A>
                                showed a slightly smaller OD600, as its finally turned to 2.835. <EM>E. coli</EM> BL21
                                with T7-OprF (<A href="http://parts.igem.org/Part:BBa_K1593210">BBa_K1593210</A>) and
                                <EM>E. coli</EM> BL21 with T7-SCVE, (<A href="http://parts.igem.org/Part:BBa_K1593667">BBa_K1593667</A>)
                                are showing 19% decreased maximal cell density and 18.5% decreased maximal cell density
                                respectively in comparison with wild type.</P><P>Consequently, the result implies that the genetical modification on bacterial
                                permeability, at least the overexpression of OprF and SCVE with a strong promoter T7,
                                won't significantly influence the bacterial growth. Thus, genetically modified bacteria
                                can be used in the following experiments.</P><P>Later on, we will characterize the basic permeability capability of <EM>E. coli</EM> BL21
                                with T7-OprF (<A href="http://parts.igem.org/Part:BBa_K1593210">BBa_K1593210</A>) and
                                <EM>E. coli</EM> BL21 with T7-SCVE, (<A href="http://parts.igem.org/Part:BBa_K1593667">BBa_K1593667</A>).
                         </P><H4 id="ONPG_Assay" class="scrollspy">ONPG Assay</H4><FIGURE><FIGCAPTION>
                              Figure 9: ONPG reaction
                              </FIGCAPTION></FIGURE><P>ONPG, that is the abbreviation of ortho-Nitrophenyl-Î²-galactoside. ONPG is a colorimetric
                                and spectrophotometric substrate, previously used for detecting Î²-galactosidase
                                activity. ONPG is actually colorless at normal situation, however if it triggers the
                                degradation reaction catalyzed by Î²-galactosidase, we will get galactose and
                                ortho-nitrophenol. Surprisingly, the compound ortho-nitrophenol has a yellow color,
                                which means we are able to detect the chemical reaction through OD detection. Besides,
                                owing to the existence of Î²-galactosidase in bacteria, a more intense optical density we
                                detected, that means there are more ONPG uptaked by bacteria. Therefore, using ONPG we
                                are able to distinguish the permeability capability of different bacteria.</P><P>Here's our result.</P><P>Note: The wavelength for detecting the bacterial concentration and absortion of ONPG are
                                respectively 600nm and 406nm.</P><P>First, we detected the bacterial solutions to control the initial concentrations. </P><FIGURE><FIGCAPTION>
                              Figure 10: ONPG assay schematic show
                              </FIGCAPTION></FIGURE><P>The OD values for <EM>E.coli </EM>BL21, <EM>E.coli </EM>BL21 with T7-OprF (<A href="http://parts.igem.org/Part:BBa_K1593210">BBa_K1593210</A>) , <EM>E.coli</EM>
                                BL21 with T7-SCVE(<A href="http://parts.igem.org/Part:BBa_K1593667">BBa_K1593667</A>)
                                are respectively 2.856,1.806 and 1.889. Based on these, we know that the concentrations
                                of these three bacteria are close.</P><P>Then we add ONPG to the bacterial solutions and detect the OD.</P><FIGURE><FIGCAPTION>
                              Figure 11: ONPG assay, absorption OD406
                              </FIGCAPTION></FIGURE><P>From this figure, we can see a rising trend in all bacterial solutions. The OD values for
                                <EM>E.coli</EM> BL21 with T7-OprF(<A href="http://parts.igem.org/Part:BBa_K1593210">BBa_K1593210</A>)
                                and <EM>E.coli </EM>BL21 with T7-SCVE(<A href="http://parts.igem.org/Part:BBa_K1593667">BBa_K1593667</A>)
                                are respectively 0.2520 and 0.2588,both are higher than <EM>E.coli</EM> BL21,the wild
                                type. We can infer that the absorbtion of modified bacteria is higher than the wild
                                type.</P><P>There is a more visual figure for comparing the absorbtions. We made a zero calibration
                                using the OD value of wild type as the zero level.</P><FIGURE><FIGCAPTION>
                              Figure 12: ONPG assay, absorption OD406 in average
                              </FIGCAPTION></FIGURE><P>In general, the OD values of both two modified bacteria are rising. For<EM> E.coli</EM>
                                BL21 with T7-OprF(<A href="http://parts.igem.org/Part:BBa_K1593210">BBa_K1593210</A>),
                                the rising is relatively smooth. For <EM>E.coli</EM> BL21 with T7-SCVE(<A href="http://parts.igem.org/Part:BBa_K1593667">BBa_K1593667</A>),the value goes
                                down first, then goes up .And the <EM>E.coli</EM> BL21 with T7-SCVE(<A href="http://parts.igem.org/Part:BBa_K1593667">BBa_K1593667</A>) has a higher
                                absorbtion than <EM>E.coli </EM>BL21 with T7-OprF(<A href="http://parts.igem.org/Part:BBa_K1593210">BBa_K1593210</A>).</P><H4 id="NPN_Assay" class="scrollspy">NPN Assay</H4><FIGURE><FIGCAPTION>Figure 13: Schematic Structure of N-Phenyl-1-naphthylamine, NPN</FIGCAPTION></FIGURE><P>N-Phenyl-1-naphthylamine(NPN),C16H13N, CAS:90-30-2, FW: 219.29, is a nonpolar probe,
                                whose fluorescence signal is relatively strong in a phospholipid environment, but weak
                                in aqueous surroundings. Consequently, using this feature, we are able to characterize
                                bacteria permeability by measuring the fluorescence intensity of the bacteria solution.
                                When NPN molecules are absorbed by bacteria, we will observe a significantly rising
                                fluorescence intensity. The more fluorescence intensity we got, the more permeable the
                                bacteria are. Here are what we got from NPN uptake assay and the conclusion on
                                genetically engineered bacteria permeability improvement. </P><P>Here's our results, the first picture illustrate the total fluorescence intensity
                                measurement by <A href="http://www.bmglabtech.com/en/products/clariostar/">BMG Labtech
                                    CLARIOstarÂ®</A>.</P><P><EM>Note</EM>: The fluorescence intensity signal at excitation wavelength range is 305nm
                                to 335nm and emission wavelength range is 370nm to 410nm, which is recommended by
                                previous research. </P><FIGURE><FIGCAPTION>Figure 14: NPN uptake result-1</FIGCAPTION></FIGURE><P>The total fluorescence intensity of <EM>E. coli</EM> BL21 wild type is 126451. On the
                                other hand, the fluorescence intensity in <EM>E. coli</EM> BL21 with T7-OprF (<A href="http://parts.igem.org/Part:BBa_K1593210">BBa_K1593210</A>) and <EM>E.
                                    coli</EM> BL21 with T7-SCVE, (<A href="http://parts.igem.org/Part:BBa_K1593667">BBa_K1593667</A>)
                                are respectively 207322 and 151820, which are significantly larger than the wildtype
                                assay.</P><P>Though we have already got the exact data on NPN uptake, we still need to revise its
                                effectiveness because of the different bacteria concentration of <EM>E. coli</EM> BL21
                                wild type, <EM>E. coli</EM> BL21 with T7-OprF (<A href="http://parts.igem.org/Part:BBa_K1593210">BBa_K1593210</A>) and <EM>E.
                                    coli</EM> BL21 with T7-SCVE, (<A href="http://parts.igem.org/Part:BBa_K1593667">BBa_K1593667</A>).
                                Here is the exact bacteria concentration data, based on OD600nm,</P><TABLE class="striped"><THEAD><TR><TH>Strain</TH><TH>OD600nm</TH></TR></THEAD><TBODY><TR><TD><EM>E. coli</EM> BL21 wild type</TD><TD>3.663</TD></TR><TR><TD><EM>E. coli</EM> BL21 with T7-OprF (<A href="http://parts.igem.org/Part:BBa_K1593210">BBa_K1593210</A>)
                                    </TD><TD>2.431</TD></TR><TR><TD><EM>E. coli</EM> BL21 with T7-SCVE, (<A href="http://parts.igem.org/Part:BBa_K1593667">BBa_K1593667</A>)
                                    </TD><TD>2.441</TD></TR></TBODY></TABLE><P>After correcting these data, we are able to get bacteria absorption uptake capability in
                                average, which is a more accurate value to characterize bacterial permeability
                                property.</P><FIGURE><FIGCAPTION>Figure 15: NPN uptake result in average</FIGCAPTION></FIGURE><P>The fluorescence intensity in average of <EM>E. coli</EM> BL21 wild type is 34521.16. On
                                the contrary, the fluorescence intensity revised by bacterial concentration in <EM>E.
                                    coli</EM> BL21 with T7-OprF (<A href="http://parts.igem.org/Part:BBa_K1593210">BBa_K1593210</A>)
                                and <EM>E. coli</EM> BL21 with T7-SCVE, (<A href="http://parts.igem.org/Part:BBa_K1593667">BBa_K1593667</A>) are
                                respectively 85282.60 and 62195.8, which are relatively 2.5 folds and approximately 2
                                folds to <EM>E. coli</EM> wild type, that is a significantly improvement compared to
                                wild type.</P><P>Consequently, through NPN uptake assay, we are able to conclude that the small molecule
                                uptake capability of bacteria improved after our genetical modification. Thus <EM>E.
                                    coli</EM> BL21 with T7-OprF (<A href="http://parts.igem.org/Part:BBa_K1593210">BBa_K1593210</A>)
                                and <EM>E. coli</EM> BL21 with T7-SCVE, (<A href="http://parts.igem.org/Part:BBa_K1593667">BBa_K1593667</A>) are both able
                                to use as the candidate bacteria strains for NDM measurement.</P><H4 id="bibliography" class="scrollspy">Bibliography</H4><OL><LI><P><EM>T. Mattila-Sandholm et al.</EM> Fluorometric assessment of Gram-negative
                                    bacterial permeabilization. <STRONG>Journal of Applied Microbiology</STRONG> 2000,
                                    88, 213â€“219</P></LI><LI><P><EM>Scott Banta et al.</EM> Genetic Manipulation of Outer Membrane Permeability:
                                    Generating Porous Heterogeneous Catalyst Analogs in <EM>Escherichia coli</EM>
                                    dx.doi.org/10.1021/sb400202s <STRONG>ACS Synth. Biol.</STRONG> 2014, 3, 848âˆ’854</P></LI></OL></DIV></DIV><DIV class="col hide-on-small-only m3"><DIV class="toc-wrapper pinned"><UL class="section table-of-contents"><LI><A href="#Extraction_of_Permeability_Improvement_Fragements">Extraction of
                                        Permeability Improvement Fragement</A></LI><LI><A href="#Construction_of_Permeability_Improving_Plasmid">Construction of
                                        Permeability Improving Plasmid</A></LI><LI><A href="#Growth_Characterization">Growth Characterization</A></LI><LI><A href="#ONPG_Assay">ONPG Assay</A></LI><LI><A href="#NPN_Assay">NPN Assay</A></LI><LI><A href="#bibliography">Bibliography</A></LI></UL></DIV></DIV></DIV></DIV></DIV><DIV id="Chemotaxis_Modification_and_ROSE" class="row"><DIV class="card hoverable"><DIV class="row"><DIV class="col s12 m9"><DIV class="card-content"><H4 id="Genetic_Fragments_Extraction" class="scrollspy">Genetic Fragments Extraction</H4><UL><LI>micF and SoxS, these two antibiotic substance responding promoters were extracted
                                    from E. coli Top 10(K-12 strain) by conducting polymerase chain reaction on its
                                    genome.
                                </LI></UL><FIGURE><FIGCAPTION>Figure 1: fragements extraction of micF and SoxS</FIGCAPTION></FIGURE><UL><LI>cheZ, this is the chemotaxis modified fragment.</LI></UL><FIGURE><FIGCAPTION>Figure 2: fragements extraction of cheZ</FIGCAPTION></FIGURE><UL><LI>BBa_R0010, this is promoter of lac, which triggers the expression of cheZ.</LI></UL><FIGURE>[Figure 3: fragment extraction of OprF and Lac](http://parts.igem.org/Part:BBa_K1593997)</FIGURE><P><STRONG>Construction of plasimid with lac-cheZ <A href="http://parts.igem.org/Part:BBa_K1593997">BBa_K1593997</A></STRONG></P><P>This is our final construction of lac-cheZ <A href="http://parts.igem.org/Part:BBa_K1593997">BBa_K1593997</A></P><FIGURE><FIGCAPTION>Figure 4: plasmid construction of lac-cheZ [BBa_K1593997](http://parts.igem.org/Part:BBa_K1593997)</FIGCAPTION></FIGURE><H4 id="Growth_characteristics" class="scrollspy">Growth characteristics</H4><P>We detected OD600(nm) to identify the effect
                                of cheZ (<A href="http://parts.igem.org/Part:BBa_K1593997">BBa_K1593997</A>) expressed
                                on <EM>E. coli</EM> BL21 with time induced by IPTG or not.</P><FIGURE><FIGCAPTION>Figure 5: Growth characteristics of BL21 WT with lac-cheZ [BBa_K1593997](http://parts.igem.org/Part:BBa_K1593997) induced by IPTG or not</FIGCAPTION></FIGURE><P>As the images above illustrated, we found <EM>E. coli</EM> BL21 wild type has the best
                                growth characteristics with a maximal OD600=2.6. Whereas <EM>E. coli</EM> BL21 with cheZ
                                (<A href="http://parts.igem.org/Part:BBa_K1593997">BBa_K1593997</A>) induced by ITPG has
                                a relatively low optical density as its OD600 after 9 h is 2.3 compared to the wildtype.
                                In the same way, characterization of <EM>E. coli</EM> BL21 with cheZ (<A href="http://parts.igem.org/Part:BBa_K1593997">BBa_K1593997</A>) not induced by
                                IPTG showed a slightly smaller OD600, as its finally turned to 2.1. The exogenous
                                protein expression affects the growth of bacteria, which was indicated by slight slower
                                grow-up in <EM>E. coli</EM> containing cheZ plasmid. In total, cheZ doesn't have obvious
                                negative impact on bacteria grown. </P><H4 id="SDS-PAGE" class="scrollspy">SDS-PAGE</H4><P>After induced by IPTG in 1mM about 4 h, cheZ was found in
                                SDS-PAGE gel, compared to the wild type (Top10) and bacteria without IPTG induction.
                                Through this result, we concluded that cheZ has been successfully expressed with the
                                IPTG induction.</P><FIGURE><FIGCAPTION>Figure 6: characterization of cheZ expression by SDS-PAGE</FIGCAPTION></FIGURE><P>See protocol
                                in <A href="https://2015.igem.org/Team:USTC/Notebook#Protocols">Protocols</A></P></DIV></DIV><DIV class="col hide-on-small-only m3"><DIV class="toc-wrapper pinned"><UL class="section table-of-contents"><LI><A href="#Genetic_Fragments_Extraction">Genetic Fragments Extraction</A></LI><LI><A href="#Growth_characteristics">Growth characteristics</A></LI><LI><A href="#SDS-PAGE">SDS-PAGE</A></LI></UL></DIV></DIV></DIV></DIV></DIV><DIV id="Adhesion_Assay" class="row"><DIV class="card hoverable"><DIV class="col s12 m9"><DIV class="card-content"><H4 id="characterization-of-optimal-conditions-on-polylysine-pll-coated-assay" class="scrollspy">
                            Characterization of Optimal Conditions on Polylysine(PLL) Coated Assay</H4><P>To see the details on the mechanism of polylysine adhesion, please refer to <A href="https://2015.igem.org/Team:USTC/Description#CACCI">Project-CACCI</A>.</P><P>To get the final PLL-coated protocol, check <A href="https://2015.igem.org/Team:USTC/Notebook#Protocols">Protocols</A> for further information.</P><P>Our original characterization of optimal conditions on polylysine(PLL)-coated assay
                            requires several factors to get, including:</P><OL><LI>Best bacterial developmental interval along with recommended dilution conditions.
                            </LI><LI>Best PLL-Coated Concentration and Pre-treatment Time.</LI><LI>Best PLL-Coated Time, given full consideration to the completion of bacterial
                                adhesion time.
                            </LI><LI>Best Measurement Interval, which illustrates the possible time for bacterial
                                response on antibiotics pressure.
                            </LI><LI>Possible Determination Interval, which refers to antibiotics response interval.</LI></OL><P>Here we present all the optimistic conditions for users to get the effective results on
                            antibiotics using our CACCI and SPRING.</P><H4 id="Best_Bacterial_Developmental_Interval_Along_with_Recommended_Dilution_Conditions" class="scrollspy">Best Bacterial Developmental Interval Along with Recommended Dilution
                            Conditions</H4><P>According to experience on <EM>E. coli</EM> development based on OD detector. We
                            concluded that bacteria exploding during <STRONG>Logarithmic Period(OD:535nm about
                                0.4~0.5)</STRONG> are at the most energetic moments with proper bacterial density.
                            Consequently, we highly recommend user to culture bacteria at Logarithmic Period and
                            then <STRONG>dilute bacterial solution about twenty times to fifty times.</STRONG></P><H4 id="Best_PLL-Coated_Concentration_and_Pre-treatment_Time" class="scrollspy">Best PLL-Coated
                            Concentration and Pre-treatment Time.</H4><P>Best PLL-Coated Conditions, which include PLL-Pretreatment time, PLL-Coated concentration
                            and PLL-Coated time are measured for best adhesive condictions for bacterial
                            treatment.</P><P>As for PLL-Pretreatment time, we adopt traditional pretreatment time, incubating about
                            12~16 h, for recommendation, which is originally used for neurons coated on neural
                            science research. And when it comes to PLL-coated concentration, experience on neurons
                            adhesion is also precious. According to previous research, PLL concentration in the
                            interval of 20 ug/mL to 100 ug/mL is quite effective for bacterial adhesion.</P><H4 id="Adhesion_Assay_with_PLL_treatment" class="scrollspy">Adhesion Assay with PLL
                            treatment</H4><P>Here we deliver the PLL treatment
                            results comparing with no PLL treatment assay. During pre-experiment adhesion assay,
                            PAO1, a strain of <EM>Pseudomonas aeruginosa</EM>, is used for adhesion effects. </P><P>The picture below showed the amount of bacteria(PAO1) under microscope without PLL
                            treatment, which is abbreviated as PLL(-). To observe capability of bacterial adhesion,
                            we wash observed place with PBS:</P><FIGURE><FIGCAPTION>Figure 1: Control Assay-Without PLL treatment</FIGCAPTION></FIGURE><P>After elution, a significant decline in the number of bacteria is recorded. As a matter
                            of fact, there is no bacteria in observed field after PBS washing. </P><P><EM>To get the algorithm of bacterial counting, please refer to <A href="https://2015.igem.org/Team:USTC/Modeling#Adhesion-Dynamics">Modeling: Adhesion Dynamics</A></EM>.
                        </P><FIGURE><FIGCAPTION>Figure 2: Control Assay-Without PLL treatment2</FIGCAPTION></FIGURE><P>As for treatment with PLL in 20 ug/mL, which is abbreviated as PLL(+,20), the impressive
                            adhesion effect is observed. To ensure the exact effect, bacteria observing field is
                            washed twice by PBS.</P><FIGURE><FIGCAPTION>Figure 3: Adhesion Assay-With PLL treatment in 20 ug/mL</FIGCAPTION></FIGURE><P>The number of bacteria doesn't decline, and on the contrary, we see slight increase of
                            bacteria number. The possible reason is constant bacterial settlement during adhesion
                            assay. After twice PBS wash, the number of bacteria is relatively stable as well.
                            Consequently, we concluded PLL has significant adhesive ability on bacteria.</P><FIGURE><FIGCAPTION>Figure 4: Adhesion Assay-With PLL treatment in 20 ug/mL</FIGCAPTION></FIGURE><H4 id="Best_PLL-Coated_Time" class="scrollspy">Best PLL-Coated Time</H4><P>Then, we tried to figure out the cohesive effect
                            of polylysine through time to get the best PLL-coated time. We gathered data just after
                            PLL treatment(0s), after 1min and after 5min. And we also recorded bacteria number in
                            the following 20 s. Here we provide the analysis of bacterial number variation after PLL
                            treatment.We respectively use PAO1, a kind of <EM>Pseudomonas aeruginosa</EM> and HCB1,
                            a kind of <EM>E. coli</EM> to handle the assay. PAO1 contains self-adhesive ability and
                            HCB1 has strong mobile ability. Using these genetically natural bacteria, we would
                            conclude with the effect of polylysine treatment.</P><P><EM>As for PAO1</EM>, Without polylysine coated, bacteria have strong swimming ability.
                            Because PAO1 is a kind of self-adhesive bacteria for experiment, thus we could see
                            adhesive bacteria increasing during assay.</P><FIGURE><FIGCAPTION>Figure 5: no antibiotics bacterial number variation through time without PLL</FIGCAPTION></FIGURE><P>With 20 ug/mL polylysine treatement, bacterial adhesive effect due to strong
                            electrostatic adhesion becomes stronger, and we, as well are able to observe the number
                            of adherent bacteria gradually rising until reaching equilibrium. </P><FIGURE>Figure 6: no antibiotics bacterial number variation through time with PLL</FIGURE><P>After mathematical simulation we found that the adhesion rate of PAO1 by polylysine after
                            treatment of PLL is increased, while for E. coli, the adhesion quantity rise, not
                            fall.</P><P><EM>As for HCB1</EM>, a kind of <EM>E. coli</EM> and the number of bacteria inside
                            observed field is stable and relatively declined because of its lack of capability of
                            adhesion.</P><P>After treated with 20ug/mL polylysine, adhesive bacteria number obviously
                            increased:</P><P>Modeling on adhesion process strictly demonstrated the adhesion assay fits Langmuir
                            absorption isoform.</P><P>Simulation result:\(\sigma =K\sigma _{0}\times (1-e^{-\frac{K_{\alpha}CV_{z}}{K\sigma _{0}}(t-t_{0})})\)Constants
                            value and details:</P><P>To know more about our modeling, please check <A href="https://2015.igem.org/Team:USTC/Modeling#Adhesion-Dynamics">Modeling:
                            Adhesion Dynamics</A></P><P>This significant reverse of bacterial adhesion tendency stronly proved that polylysine
                            has effective adhesive ability for SPRING.</P><P>After we confirmed the feasibility of PLL treatment for bacterial adhesion, then the
                            treating time should be taken into consideration because only in the case of stable
                            adhesion is effective for SPRING to gather effective and stable data.</P><P>As the plot illustrates, bacteria number is growing at the beginning of PLL treatment
                            because of opening strong electrostatic adsorption derived from PLL. And after 1min, the
                            number of bacteria become stable, and there is nearly no difference on bacteria number
                            comparing after 1 min treatment to after 5 min treatment.</P><P>Consequently, we recommended that PLL treatment after 5 min would be a promising set for
                            SPRING to output stable data.</P><H4 id="Best_Measurement_Interval" class="scrollspy">Best Measurement Interval</H4><P>According to chemotaxis, bacteria responsing to surrounding pressure or beneficits will
                            be presented as change of bacteria movement. Here we analyzed bacterial movement data
                            treating with different antibiotics concentration, specificly, chloromycetin
                            concentration. And we gathered bacterial movement data just after antibiotics
                            treatment(0s), after 1 min treatment and after 5 min treatment, and we again recorded
                            all pictures in the following 18 s to get the dynamic data. Then, we concluded from the
                            percentage of bacterial movement as an important data to intercept the effect of
                            bacterial response on antibiotics.</P><P>The measurement assay is illustrated as following:</P><P><EM>PLL(-)</EM></P><TABLE class="striped"><THEAD><TR><TH>Antibiotics Concentration(ug/mL)</TH><TH>Incubation Time(s)</TH></TR></THEAD><TBODY><TR><TD>0</TD><TD>0</TD></TR><TR><TD>0</TD><TD>60</TD></TR><TR><TD>0</TD><TD>120</TD></TR><TR><TD>0</TD><TD>180</TD></TR><TR><TD>0</TD><TD>240</TD></TR><TR><TD>0</TD><TD>300</TD></TR><TR><TD>1</TD><TD>0</TD></TR><TR><TD>1</TD><TD>60</TD></TR><TR><TD>1</TD><TD>300</TD></TR></TBODY></TABLE><P><EM>PLL(+,20)</EM></P><TABLE class="striped"><THEAD><TR><TH>Antibiotics Concentration(ug/mL)</TH><TH>Incubation Time(s)</TH></TR></THEAD><TBODY><TR><TD>0</TD><TD>0</TD></TR><TR><TD>0</TD><TD>60</TD></TR><TR><TD>0</TD><TD>120</TD></TR><TR><TD>0</TD><TD>180</TD></TR><TR><TD>0</TD><TD>240</TD></TR><TR><TD>0</TD><TD>300</TD></TR><TR><TD>0.1</TD><TD>0</TD></TR><TR><TD>0.1</TD><TD>60</TD></TR><TR><TD>0.1</TD><TD>300</TD></TR><TR><TD>0.5</TD><TD>0</TD></TR><TR><TD>0.5</TD><TD>60</TD></TR><TR><TD>0.5</TD><TD>300</TD></TR><TR><TD>1</TD><TD>0</TD></TR><TR><TD>1</TD><TD>60</TD></TR><TR><TD>1</TD><TD>300</TD></TR></TBODY></TABLE><P>When bacteria are treated with PLL, the movement of bacteria is been limited for strong
                            adhesion, after 5 min treatment, the proportion of movement decreased through
                            time.</P><P>However, when bacteria are not treated with PLL, incubated in 1ug/mL chloromycetin
                            solution, we got the movement percentage of bacteria as following:</P><P>The amount of moving bacteria is relatively increasing after 5 min antibiotics treatment.
                            Bacause of no PLL treatment, we could see the relative increase of bacteria movement
                            percentage.</P><P>As for bacteria treated with 20 ug/mL PLL, different pattern is observed:</P><P>This is the bacterial movement percentage variation with time, treated with 0.1 ug/mL
                            chloromycetin solution. Antibiotics pressure is not that strong, consequently, the
                            movement of bacteria is not ignited significantly.</P><P>When it comes to 0.5 ug/mL chloromycetin solution, the proportion of the moving bacteria
                            is increased. And as a matter of fact, the increase is quite corresponding through time,
                            reflecting bacteria impressive response on antibiotics substance.</P><P>What if the concentration of chloromycetin solution up to 1 ug/mL?A
                            quite linear increase of the proportion of the moving bacteria with time is observed.
                        </P><P>Consequently, according to our experiment, genetically naturally bacteria could respond
                            to chloromycetin solution from 0.1 ug/mL to more than 1ug/mL, which is quite promising
                            after genetic modification.</P></DIV></DIV><DIV class="col hide-on-small-only m3"><DIV class="toc-wrapper pinned"><UL class="section table-of-contents"><LI><A href="#characterization-of-optimal-conditions-on-polylysine-pll-coated-assay">Characterization
                                    of Optimal Conditions on Polylysine(PLL) Coated Assay</A></LI><LI><A href="#Best_Bacterial_Developmental_Interval_Along_with_Recommended_Dilution_Conditions">Best
                                    Bacterial Developmental Interval Along with Recommended Dilution Conditions</A></LI><LI><A href="#Best_PLL-Coated_Concentration_and_Pre-treatment_Time">Best PLL-Coated
                                    Concentration and Pre-treatment Time.</A></LI><LI><A href="#Adhesion_Assay_with_PLL_treatment">Adhesion Assay with PLL treatment</A></LI><LI><A href="#Best_PLL-Coated_Time">Best PLL-Coated Time</A></LI><LI><A href="#Best_Measurement_Interval">Best Measurement Interval</A></LI></UL></DIV></DIV></DIV></DIV><DIV id="Film_Determination" class="row"><DIV class="card hoverable"><DIV class="col s12 m9"><DIV class="card-content"><P>Let's do some crasy science this time! We'll show you how we test and select the most
                            important part in our project, the special artificial original film!</P><H4 id="modeling-guide-us-choose-the-candidates" class="scrollspy">Modeling guide us choose the
                            candidates</H4><P>In our modeling, we start with single bacteria force, analyse the interaction between
                            bacteria and film, and propose the request of Young modulus (&lt;1GPa) of film
                            eventually.</P><P>And there are various film candidates come into the front.</P><H4 id="processing-film" class="scrollspy">Processing Film</H4><P>The film must be kind of soft and one of the surface should have enough reflectance while
                            another surface should have the ability to adhere to bacteria.In fact, we don't hear any
                            of this type of film. So we need to produce the film by our own.</P><H4 id="film-i" class="scrollspy">Film I</H4><P><STRONG>Low Pressure Polyethylene</STRONG></P><FIGURE><FIGCAPTION>Figure 1: Film I, polyethylene</FIGCAPTION></FIGURE><P>Low pressure polyethylene is soft enough, and is too soft.</P><P>We use aerosol paint cover one of the surface. That can make the other surface of the film
                            will become a reflect surface. And we use 400ul 20ug/ml PLL coating the same surface at the
                            temperature of 4â„ƒ to make this surface has the ability to adhere to bacteria.</P><P>But we fail with <STRONG>film I</STRONG> in the end that we can't get interference fringes.
                            Because the surface of low pressure polyethylene is not as smooth as we want, that will
                            cause a diffuse reflection. Thus we can not get interference inescapably.</P><P>So we need the film becoem more smooth and more elastic, and then come out <STRONG>film
                            II-Rubbers</STRONG></P><H4 id="film-ii" class="scrollspy">Film II</H4><P><STRONG>Rubbers-Condom</STRONG></P><FIGURE><FIGCAPTION>Figure 2: Film II, condom</FIGCAPTION></FIGURE><P>If you want the material smooth enough and thick enough and elastic enough, that is, of
                            cause, <STRONG>condom!</STRONG></P><P>When we plan to use paint cover, the truth give us a hard hit. Because the film shrink too
                            much when we spray paint.</P><P>And we try silver mirror reaction as replacement. But still fail Because the ammonia erosion
                            is too severe.</P><P>Finally we find the final film candidate that we missed-<STRONG>Glass</STRONG></P><H4 id="film-iii" class="scrollspy">Film III</H4><P><STRONG>Cover slip</STRONG></P><FIGURE><FIGCAPTION>Figure 3: Film III, Cover slip</FIGCAPTION></FIGURE><P>The Young modulus of Glass is about 50GPa. But according to our <STRONG>adhesion
                            assay</STRONG> result and modeling, there will several numbers of fringes changes in the
                            experiment.</P><P>So we spraying it, coating it, testing it, and we made it!The glass have a great
                            potential to reflect, so we can get interference very easy.With modeling guidance, we
                            can know the relation between fringes changes and antibiotics concentration.\(A_{0}+\frac{B_{0}}{A}=\frac{1}{\Delta N}\)</P><P>Then we want to develop a calibration with three point.</P><FIGURE><FIGCAPTION>Figure 5: Experimental results of calibration</FIGCAPTION></FIGURE><P>The fitting result indicate that
                            our modeling and normalization operation was exactly correct and effective. <STRONG>OUR
                                CRAZY MIND HIT OUR MODEL, AND OUR MODEL EXACYLY HIT OUR RESULT! WE LOVE
                                SCIENCE!</STRONG></P><P>That means the <STRONG>Processed Glass</STRONG> worth the name of <STRONG>special artificial
                            original film!</STRONG></P></DIV></DIV><DIV class="col hide-on-small-only m3"><DIV class="toc-wrapper pinned"><UL class="section table-of-contents"><LI><A href="#modeling-guide-us-choose-the-candidates">Modeling guide us choose the candidates</A></LI><LI><A href="#processing-film">Processing Film</A></LI><LI><A href="#film-i">Film I</A></LI><LI><A href="#film-ii">Film II</A></LI><LI><A href="#film-iii">Film III</A></LI></UL></DIV></DIV></DIV></DIV><DIV id="Calibration" class="row"><DIV class="card hoverable"><DIV class="col s12 m9"><DIV class="card-content"><P>You can detect various of kinds of material in solution, as long as you follow our standard to make the engineered bacteria. See our future NDM improvement plan at <A href="https://2015.igem.org/Team:USTC/Achievements#Future-NDM">Future NDM</A></P><P>There is one thing to be sure is that whatever bacteria you used, you need to build calibration ruler to measure the concentration quantificationally. According to our modeling in calibration we need to confirm three constants <STRONG>'A0', 'B0', 'n'</STRONG>.</P><P>\(A_{0}+\frac{B_{0}}{A}=\frac{1}{\Delta N}\)</P><P>That means we need to test at least three point to confirm the formula.</P><H4 id="establishement-of-the-calibration-ruler" class="scrollspy">Establishement of the Calibration Ruler</H4><P>To build a calibration ruler between <STRONG>chloramphenicol</STRONG> concentration and</P><P>interference fringes pattern.</P><P>In this case we use our <EM>E. coli</EM> BL21 with T7-OprF (<A href="http://parts.igem.org/Part:BBa_K1593210">BBa_K1593210</A>), a type of bacteria in the best perfomance during genetically modification.(See our modification at <A href="https://2015.igem.org/Team:USTC/Results#Permeability-Improvement">Results-Permeability Improvement</A>). According to our experiment results, we found the <EM>E. coli</EM> BL21 with T7-OprF (<A href="http://parts.igem.org/Part:BBa_K1593210">BBa_K1593210</A>) was more sensitive to chloramphenicol.</P><P>And according to <STRONG>adhesion pre-experiment</STRONG> we chose the concentration range with 0.5ug/ml~5ug/ml in the experiment.</P><P>The changes of fringe number and test concentration shown below(take three pictures in a row with the time interval 20s),</P><TABLE><THEAD><TR><TH>Time(s)</TH><TH>5.0ug/ml(Fringe number)</TH><TH>1.0ug/ml(Fringe number)</TH><TH>0.5ug/ml(Fringe number)</TH></TR></THEAD><TBODY><TR><TD>0</TD><TD>42/43</TD><TD>51/49/52</TD><TD>44/43</TD></TR><TR><TD>20</TD><TD>-</TD><TD>50/51/50</TD><TD>-</TD></TR><TR><TD>40</TD><TD>-</TD><TD>50/46</TD><TD>-</TD></TR><TR><TD>60</TD><TD>-</TD><TD>48/48/50</TD><TD>45</TD></TR><TR><TD>80</TD><TD>-</TD><TD>48/49/49</TD><TD>-</TD></TR><TR><TD>100</TD><TD>-</TD><TD>51/53/49</TD><TD>45</TD></TR><TR><TD>120</TD><TD>42/43</TD><TD>-</TD><TD>-</TD></TR><TR><TD>140</TD><TD>-</TD><TD>51/48/52</TD><TD>-</TD></TR><TR><TD>160</TD><TD>-</TD><TD>-</TD><TD>41</TD></TR><TR><TD>180</TD><TD>-</TD><TD>48/52/54</TD><TD>-</TD></TR><TR><TD>200</TD><TD>48</TD><TD>55/47/49</TD><TD>43/43</TD></TR><TR><TD>220</TD><TD>43/47</TD><TD>51/52</TD><TD>45</TD></TR><TR><TD>240</TD><TD>48/46</TD><TD>53/51</TD><TD>44</TD></TR><TR><TD>starting average</TD><TD>42.5</TD><TD>50.5</TD><TD>43.5</TD></TR><TR><TD>ending average</TD><TD>46.4</TD><TD>51.75</TD><TD>44.2</TD></TR><TR><TD>Î”N</TD><TD>3.9</TD><TD>1.25</TD><TD>0.7</TD></TR></TBODY></TABLE><P>Then we can fit this data with calibration formula, assume <STRONG>n=3</STRONG>(through trying different value of n get the best value of n) then get the simplified formula:</P><P>\(A_{0}+\frac{B_{0}}{A}=\frac{1}{\Delta N}\)</P><P>Then the fitting result illstruates as,</P><P>And we can get the value of unknown constants:</P><TABLE><THEAD><TR><TH>constant</TH><TH>value</TH></TR></THEAD><TBODY><TR><TD>A0</TD><TD>0.1339</TD></TR><TR><TD>B0</TD><TD>0.6513</TD></TR></TBODY></TABLE><P><STRONG>What an amazing result! </STRONG> The fitting result indicated that our modeling and normalization operation was exactly correct and effective. <STRONG>OUR CRAZY MIND HIT OUR MODEL, AND OUR MODEL EXACYLY HIT OUR RESULT!</STRONG><STRONG>WE LOVE SCIENCE!</STRONG></P><H4 id="reverification-of-calibration" class="divider">Reverification of Calibration</H4><P>There may some arguement on our accuracy on calibration. So we need to conduct another experiment to reverficate our calibration ruler.</P><P>In order to test whether the calibration is correct and to confirm the coefficients, we choose two concentration as check point.</P><P>And use SPRING to measure its concentration. The real concentration and experiment results shown in table:</P><TABLE><THEAD><TR><TH>time(s)</TH><TH>1.8ug/ml(Fringe number)</TH><TH>3.2ug/ml(Fringe number)</TH></TR></THEAD><TBODY><TR><TD>0</TD><TD>78/80</TD><TD>77/76/77</TD></TR><TR><TD>20</TD><TD>77/77</TD><TD>79/77</TD></TR><TR><TD>40</TD><TD>78</TD><TD>75/72</TD></TR><TR><TD>60</TD><TD>79</TD><TD>74</TD></TR><TR><TD>80</TD><TD>81</TD><TD>75/75</TD></TR><TR><TD>100</TD><TD>75</TD><TD>72/73</TD></TR><TR><TD>120</TD><TD>80</TD><TD>74/74</TD></TR><TR><TD>140</TD><TD>83/82</TD><TD>71/75</TD></TR><TR><TD>160</TD><TD>81/79</TD><TD>72/76</TD></TR><TR><TD>180</TD><TD>82/82</TD><TD>75/74</TD></TR><TR><TD>200</TD><TD>82/81</TD><TD>75</TD></TR><TR><TD>220</TD><TD>80/82</TD><TD>74/74/75</TD></TR><TR><TD>240</TD><TD>81/80</TD><TD>74/75/74</TD></TR><TR><TD>starting average</TD><TD>79</TD><TD>77.2</TD></TR><TR><TD>ending average</TD><TD>81</TD><TD>74.3</TD></TR><TR><TD>Î”N</TD><TD>2.0</TD><TD>2.9</TD></TR><TR><TD>experiment concentration</TD><TD>1.80ug/ml</TD><TD>3.09ug/ml</TD></TR></TBODY></TABLE><P>The result showed that the experiment concentration is <STRONG>EXTREMELY CLOSE</STRONG> to the real concentration, which means <STRONG>OUR NDM IS ABLE TO TELL ANTIBIOTIC CONCENTRATION IN VERY ACCURATE WAY.</STRONG></P><P>By the way, <STRONG>DO NOT FORGET,  </STRONG> operating our NDM only cost you <STRONG>100 SECONDS</STRONG>.</P><P>Yes, such fast and accurate hardware detecting antibiotics, <STRONG>that is our NDM</STRONG>, <STRONG>ONLY OUR NDM</STRONG>.</P><P><EM>Note:</EM> To get the program of our calibration analysis, please visit <A href="https://2015.igem.org/Team:USTC/Software">Software</A>. To know more information on how we analyzed our results, please visit <A href="https://2015.igem.org/Team:USTC/Modeling#Calibration">Modeling: Calibration</A>. To see our final measurement report with theoretical analysis, please visit <A href="https://2015.igem.org/Team:USTC/Measurement">Measurement</A>.</P></DIV></DIV><DIV class="col hide-on-small-only m3"><DIV class="toc-wrapper pinned"><UL class="section table-of-contents"><LI><A href="#establishement-of-the-calibration-ruler">Establishement of the Calibration Ruler</A></LI><LI><A href="#reverification-of-calibration">Reverification of Calibration</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="fixed-action-btn" style="bottom: 45px; right: 24px;"><A class="btn-floating btn-large amber darken-1 waves-effect waves-light" onclick="$('html, body').animate({scrollTop:0}, 'slow');"><I class="material-icons">publish</I></A><UL><LI><A class="btn-floating green waves-effect waves-light" style="transform: scaleY(0.4) scaleX(0.4) translateY(40px); opacity: 0;" onclick="location.reload();"><I class="material-icons">refresh</I></A></LI></UL></DIV><FOOTER class="page-footer grey darken-1" style="padding-top: 0px;"><DIV class="container"><DIV class="row" style="margin-bottom:0px;"><DIV class="col s8"><H5 class="white-text">Contact Us</H5><P class="grey-text text-lighten-4">University of Science and Technology of China, No.96, JinZhai Road Baohe District,Hefei,Anhui, 230026,P.R.China.</P></DIV><DIV class="col s4"><H5 class="white-text">Links</H5></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></BODY></HTML>