Team:USTC/Notebook
Nanomachine Detecting Microbiotics
NOTEBOOKarrow_drop_down
Protocols
Daily Notes-Stage 1
Daily Notes-Stage 2
Home
Project
Modeling
Results
Achivements
Parts
Tutorials
Safety
Policy&Practices
Collaborations
Team
Attributions
Synthetic Biology
Genome Extraction
Material
Items (Volume)
Buffer GA (200ul)
Proteinase K solution (20ul)
Buffer GB (220ul)
Ethanol (220ul)
Buffer GD (500ul)
Rinse PW (600ul)
Elution Buffer TE (100ul)
Extraction steps:
Absolute ethanol before use in buffer and rinse GD PW, adding volume refer to the label on the bottle.
Take inoculum 1-5 ml, 10,000 rpm (~ 11,500 × g) was centrifuged 1 min, the supernatant net absorption as possible.
The cell pellet is added to 200 μl buffer GA, shaking to complete the cell suspension.
The tube is added to 20 μl Proteinase K solution, mixed evenly.
Add 220 μl buffer GB, oscillation 15 sec, 70 ℃ and placed 10 min, strain the solution clear, brief centrifugation to remove the tube drops on the inner wall .
Add 220 μl ethanol, mix thoroughly shaken 15 sec, flocculent precipitate may occur at this time, a brief centrifugation to remove the tube drops on the inner wall .
Previous resulting solution and the flocculent precipitate are added into a adsorption column CB3(adsorption column into the collection tube), 12,000 rpm centrifugation 30 sec, discard the liquid, set the adsorption column CB3 into the collection tube.
500 μl buffer GD was added to CB3 adsorption column(check whether ethanol has been added before use), 12,000 rpm (~ 13,400 × g) centrifugation 30 sec, discard the liquid, set the adsorption column CB3 into the collection tube.
600 μl rinse PW was added to CB3 adsorption column(check whether ethanol has been added before use), 12,000 rpm(~ 13,400 × g) centrifugation 30 sec, discard the liquid, set the CB3 adsorption column into the collection tube.
Repeat steps 8.
Place the column back into the collection tube CB3, 12,000 rpm (~ 13,400 × g) centrifugation 2 min, discard the waste.Place the adsorption column CB3 at room temperature for several minutes to completely dry absorbent material.
Put the CB3 adsorption column into a clean centrifuge tube and drop 50-200 μl elution buffer TE to the middle of the adsorbed film vacant. Put at room temperature 2-5 min, 12,000 rpm (~ 13,400 × g) centrifugation 2 min, collect the solution in a centrifuge tube.
Detect the concentration of DNA.
Plasmid Extraction
Preparation
Check out whether RNaseA has been added in Buffer P1.
Check out whether ethyl alcohol has been added in Wash Solution
Check out whether sediment exist in Buffer P2 and P2.
Procedure
Absorb 1.5 to 5 mL bacteria solution in to EP tubes, centrifuge them at 8,000xg 2 mins and then discard the culture medium.
Add 250 ul Buffer P1 into sediment, and use spearhead to make bacteria suspended.
Add 250 ul Buffer P2, and overturn the EP tubes 5 to 10 times immediately and tenderly. Stand tubes about 2 to 4 mins.
Add 350 ul Buffer P3 and overturn the EP tubes 5 to 10 times again immediately and tenderly.
Centrifuge tubes at 12,000xg about 5 to 10 mins.
Pour the supernatant liquid into absorption columns ,centrifuge them at 8,000xg about 30 sec and then discard the liquid in the collecting pipe.
optional:Add 500 ul Buffer DW1 and centrifuge them at 9,000xg about 30 sec. Then discard the liquid in the collecting pipe.
Add 500 ul Wash Solution, centrifuge them at 9,000xg about 30 sec and discard the liquid in the collecting pipe.
Do the step 8 again.
Centrifuge the empty columns at 9,000xg about 1 min.
Put the columns in clean 1.5mL EP tubes, add 50 to 100 ul Elution Buffer on the absorption film, stand 1 min and centrifuge about 1 min at 9,000xg.
Keep the DNA solution for further work.
The Protocol is based on SanPrep Column Type DNA Plasmid Extraction Kit.
PCR System Preparation and Conditions Setting
PCR system(set 50ul system as an example):
The amount of substance of premier is written on the EP tubes. Calculating the volume of 1XTE required for 10uM primer solution is required before the system preparation.
Template DNA xul as required.
Forward Primer(10 uM) about 1 to 2 ul. The final concentration will be 0.2 to 0.4 uM.
Reverse Primer(10 uM) about 1 to 2 ul. The final concentration will be 0.2 to 0.4 uM.
5xTransStart FastPfu Fly Buffer 10 ul.
2.5 mM dNTPs 5 uM. The final concentration will be 0.25 mM.
TransStart FastPfu Fly DNA Polymerase 1 ul.
Double distilled water added to 50 ul.
Reaction conditions
Calculate the Tm Value. If the complementary strip is smaller than 20 bp, Tm=4X(A+T)+2X(C+G), and the annealing temperature is Tm-5 degree centigrade. If not, use software to figure out the exact Tm value.(Also Tm value exist in report)
PCR cycle:
1 cycle of 95 degree centigrade about 2 min for pre-degeneration;
30 to 35 cycles of 95 degree centigrade about 10 sec, Tm-5 degree centigrade about 20 sec and 72 degree centigrade 1kb per min;
1 cycle of 75 degree centigrade about 5 mins and stay the device in 4 degree centigrade.
The protocol is based on TransStart FastPfu Fly DNA Polymerase.
PCR Production Extraction
Prepation
Check out whether ethyl alcohol has been added into Wash Solution.
Check out whether sediment exist in Buffer B3.
Check out whether isopropanol has been added into Buffer B3.
Procedure
Add Buffer B3 3 times volume of PCR system solution and incorporated thoroughly.
Centrifuge them at 8,000xg about 30 sec and drain the liquid in the collecting pipe.
Add 500 ul Wash Solution at 9,000xg about 30 sec and drain the liquid in the collecting pipe.
Do The step 3 again.
Centrifuge the empty pipes at 9,000Xg about 1 min.
Put the absorption column in some clean 1.5 mL EP tubes, add 15 to 40 ul Elution Buffer, stand for 1 min and centrifuge them at 9,000xg about 1min.
Keep the DNA solution.
Restriction Enzymes Analysis
Protocol
20ul system as following:
2ul plasmid DNA sample;
1ul restriction enzyme(both 1 ul for double digestion);
2ul 10X green buffer;
double distilled water added to 20 ul.
Timetable:react the whole system at 37 degree centigrade about 2 hours and then inactivate the enzymes at 60 degree centigrade about 20 mins.
DNA Gel Electrophoresis
Standard 1% agarose gel
(Generally, 0.7%~2% agarose gel is widely used in lab)
Measure out 1g agarose in scale, and then put the sample into conical flask.
Add 100mL 1XTAE into conical flask, shaking evenly.
(about 1xTAE: dilute the 50x TAE to 1x TAE: add 10mL 50x TAE and 490 mL double distilled water to form as a system for further use)
Plastic wrap can be used to pack the bottleneck.
Use microwave oven to heat the solution about 5mins until the agarose is all dissolved and the solution is boiling and clear. It would be better if we shake the flask again after heating 2 mins.(CAUTION: HOT! Please use wool gloves when picking and placing flask.)
Let the gel cool down about 2 mins.
Pour the gel into gel tray with the appropriate comb inserted in the side of tray and wait till the gel solified(It may costs about 10 mins). Then remove the comb
Put the gel into gel box with the 1XTAE just immerse the gel exactly. Please let passages stay towards the negative pole(Often black).
Carefully add samples with 6X loading buffer and marker into passages. Dosage is based on the depth and width of passages.
Set electrophoresis conditions: 90V 600mA and 30mins and then turn on the power.
Disconnect the electrodes.
Utilize the devices that contains UV light to observe the gel and analyze the result.
Save the result in the computer for further use.
Gel Extraction
Preparation:
Check out whether ethyl alcohol is added into Wash Solution.
Check out whether Buffer B2 has sediment.
Adjust the thermostat water bath at 50 degree centigrade.
Procedure:
Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.
Measure out the weight of tubes of gel.
Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.
Optional: If the length of the DNA strip is shorter than 500 bp, it would be better add isopropanol one third volume of Buffer B2.
After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s.
Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.
Do the step 5 again.
Centrifuge the empty columns at 9000xg about 1min.
Open the columns and air them about 10 mins.
Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.
Keep the DNA solution and often measure the concentration of DNA and A260/A280 for further study.
The protocol is based on SanPerp Column Type DNA Gel Extraction Kit.
Preparation of Competent Cell
Procedure
Pick out the simple colony(DH5α) and cultivate bacteria in 2 to 3 mL LB media at 37 degree centigrade about 12 to 16 hours with shocking.
Take 0.05mL nutrient solution into 50 mL media and cultivate bacteria at 37 degree centigrade about 2 to 3 hours with shocking till the OD550 reaches 0.2 to 0.4.
Absorb 1.5mL nutrient solution into EP tubes with ice-bath about 10 mins.
Centrifuge the tubes at 4000xg at 4 degree centigrade about 10mins and discard the supernatant liquid.
Utilize 0.5 to 1 mL 0.1M calcium chloride with 15% glycerol to suspend bacteria.(ATTENTION:USE SPEARHEAD TO SUSPEND BACTERIA TENDERLY INSTEAD OF VIBRATOR )
Centrifuge the tubes at 4000xg at 4 degree centigrade about 8 mins and discard the supernatant liquid.
Utilize 100 ul 0.1M calcium chloride with 15% glycerol to suspend bacteria and then ready for transformation quickly or restore at -70 degree centigrade.(WITH ATTENTION AGAIN)
TIPS: EXPERIENCE SAYS MODERATE MAGNESIUM ION HELPS DEVELOPMENT OF COMPETENT CELL.
Transformation
NOTE:Generally, competent bacteria are restrored in -70 degree centrigrade environment.
Procedure
Take the competent bacteria from refrigerator and incubate them into ice about 5 mins until it is dissolved
Absorb 100pg to 10 ng plasmid(normally 1 to 2 uL, DO NOT add more than 5% volumn of bacteria solution) and mix it with bacteria solution thoroughly.
ATTENTION: Please operate this step tenderly!!!
Put the tubes on the ice about 30 mins.(Time SHOULD BE ACCURATE)
Make a heat shock at 42 degree centigrade about 30 sec(TIME SHOULD BE ACCURATE)
Put the tubes on the ice about 2 to 3 mins again.
Add 900 ul LB medium into EP tubes and cultivate the bacteria at 37 degree centrigrade about 40 to 60 min.
Centrifuge them at 12,000xg about 15 sec and we will se sediment in the tubes.
Discard the supernatant liquid and leave about 220 ul medium.
Coat plate: add 200 ul solution in a large plate while add 20 ul solution in a small plate.
Cultivate these bacteria overnight for further use.
Biochemical Analysis
SDS-PAGE
Separation Gel
H2O：0.6ml
80%glycerol:0.6ml
Gel buffer:1.2ml
40%Arc-Bis Mix:1.2ml
10%APS:20ul
TEMED:2ul
Stacking Gel
H2O：1.0ml
Gel buffer:0.40ml
40%Arc-Bis Mix:0.20ml
10%APS:20ul
TEMED:2ul
1×glycine Running Buffer
Coomassie brilliant blue solution
Protocol
Assemble the sheet for adding the gel.
Prepare separating gel, and load into the sheet suitable height, then dry it at room temperature.
Prepare stacking gel into the sheet, and plug in the comb, then dry it at room temperature.
Remove the sheet, and pull out a comb, then transfer the plastic sheet into the electrophoresis tank.
Add the electrophoresis tank glycine running buffer to the appropriate location.
Sample electrophoresis.
After completion of the electrophoresis, the gel was removed, placed in Coomassie brilliant blue solution, stain in a microwave for 1-2min.
Gel elution, to background staining disappear.
Observe gel.
NPN Assay
Material
BL21
BL21-T7-SCVE
BL21-T7-OprF
10mM N-Phenyl-1-naphthylamine(NPN)
Inoculate an overnight culture (30 mL with 1 mL of pre-culture).
Centrifugation (15 min at 3000 g) of the whole overnight culture and discard supernatant.
Resuspend pellet in equal volume of 10 mM PBS buffer.
Repeat steps 2) and 3) twice.
Mix 1 mL of washed cells with a 10 mM stock solution of NPN to a final concentration of 10 µM.
Transfer 200 µL of samples to a flat bottomed black 96 well plate.
Detect fluorescence intensity.
Fluorescence intensity of sample is recorded by BMG Labtech CLARIOstar®
ONPG Assay
BL21
BL21-T7-SCVE
BL21-T7-OprF
ONPG solution.
Inoculate an overnight culture (30 mL with 1 mL of pre-culture).
Centrifugation (8 min at 3000 g) of the whole overnight culture and discard supernatant.
Resuspend pellet in equal volume of 10 mM PBS buffer.
Repeat steps 2) and 3) twice.
Inside a clear, flat bottomed 96 well plate, mix 180 µL ONPG (o-nitrophenyl-b-galactopyranoside) stock solution (3 mM in 10 mM PBS buffer) with 20 µL of washed cells.
Absorbance of sample is recorded by BMG Labtech CLARIOstar®
Special Protocol for NDM
Prepare Bacterial Powder
for 500mL
Ingredients
Mass
Yeast Extract
2.5g
Trtptone
2.5g
NaCl
15g
KH2PO4
0.5g
Na2HPO4·12H2O
6.5g
glycerol
1.5g
NaCl
15g
prepare the culture medium as above.
prepare the drying agent:
skim milk 12%(mass fraction)
NaCl 2%(mass fraction)
culture the bacterial with the culture medium to log phase.
centrifuge the bacterial solution at 5,000r/min.
resuspend bacterial with drying agent in the same volume.
divide the resuspension into EP tube.
freeze the bacterial at -80 degree centigrade for 8 hours.
drying the bacterial with lyophilizer.
store the bacterial powder at -20 degree centigrade.
NDM Operation
This is a brief protocol on NDM operation. For more information, please visit Tutorials
Preparation of antibiotics ruler
To detect antibiotics concentration, users need firstly prepare standard antibiotic concentration, we will set chloromycetin as an example.
Ruler Sample
Concentration ug/mL
1
0.1
2
0.5
3
1.0
4
5.0
Coating film with polylysine
Add approximiately 400 ul 20ug/mL polylysine on the film, and store the film at 4 degree celcius for more than 4 hours. After 4 hours, absorb polylysine and then wash the film by PBS buffer. Note: polylysine can be recycled.
Adhesion Assay
Add 200 ul bacteria solution on film about 100s. Note: bacteria grown should be in steady state, and you should dilate bacteria and let its OD(600) approximiately reach 0.05
Operate Optical Path
Operating optical path within 100s would be highly recommended for users. To get the best images, you should observe and get fringes parallel to y axis of screen as possible.
Observe and Record
The measurement period is about 300s. Consequently, we recommend users to take a series of images each 10s during the beginning of 300s.
Materials
Materials of LB Culture Medium
LB Culture(500ml)
Tryptone:5g
Yeast Extract:2.5g
NaCl:5g
LB Solid Culture(200ml)
Tryptone:2g
Yeast Extract:1g
NaCl:2g
AgerA:3g
PBS Buffer(1L)
KH2PO4:0.74g
Na2HPO4:1.42g
NaCl:8g
KCl:0.2g
Add ddH2O about 800mL fully dissolved, then add concentrated hydrochloric acid pH to 7.4, finally set the capacity to 1L.
ONPG Stock Solution
ONPG:0.4g
ddH2O:75ml
PBS buffer:25ml
37℃dissolve the ONPG, store in refrigerator without night at 2-5℃.
NPN Stock Solution(10mM, volume 100ml)
1N NPN: 1g
Dissolve the NPN with 100ml anhydrous alcohol, store it store in refrigerator without night at 2-5℃.
1.5M Tris-HCl(PH 8.8)
Weigh 181.7g Tris into a 1L beaker
Add about 800ml de ionized water, stir well
To be cooled, with concentrated hydrochloric acid to adjust PH to 8.8
Solution set to 1L
After high temperature and high pressure sterilization, room temperature preservation
1.5M Tris-HCl(PH 6.8)
weigh 181.7g Tris into a 1L beaker
add about 800ml de ionized water, stir well
to be cooled, with concentrated hydrochloric acid to adjust PH to 6.8
solution set to 1L
after high temperature and high pressure sterilization, room temperature preservation
10% SDS
Get 10gSDS adding 80ml ddH2O into a 100ml beaker, 68℃, heated and dissolved.
Drop the concentrated hydrochloric acid to PH 7.2.
The solution is determined to be 100ml and the room temperature is kept.
30% Acrylamide
Get 280g acrylamide and BIS to 1L.
Add 800ml deionized water to the beaker, stir to dissolve.
Add the solution to 1L with deionized water, then use the 0.45um filter to filter the impurities.
Store the solution in brown bottle at 4℃.
5×Tris Glycine Buffer
Ingredients
Mass(g)
Tris
15.1g
Glycine
94g
SDS
5.0g
The following reagents are added in a 1L beaker
Add about 800ml deionized water, stir and dissolve.
With deionized water, the solution is set to 1L, and the room temperature is kept.
5×SDS-PAGE Loading Buffer
Ingredients
Mass or volume
1M Tris-HCl(PH 6.8)
1.25ml
SDS
0.5g
BPB
25mg
Glycerol
2.5ml
The following reagents are added in a 10ml centrifugal tube.
Adding deionized water to dissolve the capacity to 5ml.
Per 500ul packaging, at room temperature.
Before using 25ul 2-ME added to each of the small.
Buffer with 2-ME can be saved for a month.
Bradford R-250 Staining Solution
Get 1g Bradford R-250, placed in a 1L beaker.
Take 250ml isopropanol into the beaker, stir to dissolve.
Adding 100ml to the glacial acetic acid, stirring evenly.
Add 650ml of deionized water, stirring evenly.
filter paper to remove particulate matter, room temperature preservation.
Bradford R-250 Destaining Solution
Ingredients
Mass or volume
Acetic acid
100ml
Ethanol
50ml
dH2O
850ml
Take the solution in a 1L beaker.
Fully mixed for use.
50×TAE Buffer
Ingredients
Mass or volume
Tris
242g
Na2EDTA·2H2O
37.2g
The following reagents, placed in a 1L beaker.
Add 800ml deionized water to the beaker, stir to dissolve.
Add 57.1ml acetic acid, stirring.
Add deionized water into the solution to 1L, then keep at room temperature.
Agarose
Prepare 1×TAE buffer solution.
According to the required amount of agarose and concentration, add into appropriate conical bottle.
Adding a certain amount of 1 x buffer TAE.
Seal conical bottle with fresh-keeping film, then use the microwave to dissolve agarose.
Solution was cooling to 60 degrees Celsius, fully mixed.
Insert a suitable comb in the plastic mold, then pour the solution into the plastic mold.
Solidify the solution at the room temperature, then place the mold in the electrophoresis tank.
6×Loading Buffer
The following reagents, placed in a 500ml beaker.
Add 200ml of deionized water to the beaker, heating and stirring fully dissolved.
After add 180ml glycerol, the use of NaOH 2N to adjust PH to 7.
Add deionized water to 500ml, then keep at room temperature.
Ampicillin
5g ampicillin in 50ml Centrifugal tube.
After adding 40ml to the water, fully dissolve the solution, then keep it at 50ml.
Using 0.22ul filter sterilization.
Packaging (1ml), stored at -20℃.
IPTG
Place 1.2gIPTG in the 50ml centrifuge tube.
Add 40ml dH2O, fully dissolve the solution, then keep it at 50ml.
Using 0.22ul filter sterilization.
Packaging (1ml), stored at 20℃.
TB Culture
Ingredients
Mass(g)
Tryptone
10g
Yeast Extract
5g
NaCl
10g
Preparation of phosphate buffer solution: 2.31gKH2PO4 and 12.54gK2HPO4 in 90ml deionized water, stirring and dissolving, then keep it at 100ml, high temperature and high pressure sterilized.
Take the following reagents, placed in a 1L conical flask.
Add 800ml deionized water, stir and dissolve.
Keep volume at 1L, high temperature and high pressure sterilization.
Cooling to 60℃, add 100ml phosphate buffer.
4℃ for preservation.
SOB Culture
Ingredients
Mass(g)
Tryptone
10g
Yeast Extract
5g
NaCl
10g
Preparation of 250mM KCl solution: 90ml deionized water dissolved in 1.86gKCl, then keep the solution at 100ml.
Preparation of KCl 2M solution: 90ml deionized water dissolved in 19gMgCl2, then keep the solution at 100ml, high temperature and high pressure sterilization.
Take the following reagents, placed in a 1L conical flask.
Add 800ml deionized water, stir well.
Add 10ml KCl solution into the conical flask.
Drop NaOH 5N solution, adjust PH to 7.
Adding de ionized water to 1L.
High temperature and high pressure sterilization, stored at 4℃.
Add 5ml 2M MgCl2 solution before using.
SOC Culture
Prepare 1M glucose solution：add 18g glucose into 90ml dH2O， fully dissolve the glucose， keep the solution at 100ml.
Add 2ml 1M glucose solution into 100ml SOB solution，mix them evenly.
Preserve at 4℃.
Semi-Solid M63 Medium
In order to characterization the movement of TOP10, semi-solid M63 medium preparation is indispensable in this research.
(500mL Protocol)
Ingredients
Mass
Proportion(g/mL) or Mole
Yeast Extract
2.5g
0.5%
Trtptone
5g
1%
NaCl
5g
1%
Agar
1.25g
0.25%
KH2PO4
6.8g
50mM
KOH
2.1g
37.5mM
(NH4)2SO4
0.09g
7.5mM
MgSO4
0.06g
0.5mM
FeSO4
0.2969g
1.95mM
D-Glucose
1.98g
11mM
Calibration
Use 1ug/ml Chloromycetin activating bacteria to test the film.
Film
Film type:glass(cover clip——thickness~0.16mm width~2.5cm longth~3.7cm)
Use the aerosol paint to process one of the surface of the film.
Coating the film with 400ul 20ug/ml PLL(Poly-L-Lysine) at the temperature of 4℃ for over 4 hours.
Bacteria solution
Bacteria type: BL21+OprF
Mix bacteria with PBS and make the OD~0.3.
Sample solution
1ug/ml Chloromycetin PBS solution.
Inoculation
Drop 200ul bacteria solution on the processed film for 100s
After 100s, wash the film with wash solution twice Immediately.
Record images
1.Put the film into the sample box filled with sample solution.
2.Put the sample box in SPRING and push or pull the sample box at the recommended position, and rotate the sample box to get clear interference fringes.
3.From 0s to 240s, take 2~3 pictures in a row with the time interval of 20s.
Results
Succeed in getting the excellent interference fringes.
Record data and process the image.
Use the Matlab process the image.
Get the fitting according to the module thus get the calibration.
Verifying Calibration
Sample solution
（1）1.8ug/ml Chloromycetin PBS solution.
（2）3.2ug/ml Chloromycetin PBS solution.
result
Process the gotten images and get the △N.
Calculate the △N and get the concentration which is 1.80ug/ml and 3.09ug/ml.
Synthetic Biology
Biochemical Analysis
Special Protocol for NDM
Materials
2015 4/7
Medium preparation
Prepare and keep a part of solid medium and broth in the refrigerator as a backup.
Cell culture
Culture the cell,TOP10, with LB liquid solution overnight.
2015 5/7
Design protocol
Take the solid medium, heat it until liquid. Then make two plate of solid medium, evenly. Cool it to the room temperature.
prepare antibiotic solution with concentration gradient
a. prepare antibiotic(neomycin) solution with ddH2O at 50mg/ml as mother solution 1. And dilute it to 10% concertration(5mg/ml) with ddH2O as mother solution 2. Then prepare 8 parts of solution as this:
Number
Solution 1(V/uL)
Solution 2(V/uL)
0
0
0
1
0
0.2
2
0
1
3
0.2
0
4
0.4
0
5
0.6
0
6
0.8
0
7
1
0
Keep them in the 70℃ water bath. Then add 800uL liquid solid medium into each Ep tubes. Transfer them into 50℃ water bath.
Cut 4 hole in each plate of solid medium, whose diameter is at 8mm.
Take 5mL culture into the 2 Ep tubes. Centrifuge it at twice. Then add liquid solid medium at 50℃, each 800uL, dissolve evenly.
Take 200uL top10 solution into 8 Ep tubes each at 50℃. Then add the mixed solution into the hole at once until the hole is added up.
Results: the raw results showed differerent characteristics of chemotaxis based on distinct antibiotic concentration.
2015 12/7
4th Colonial PCR for COMPX
2μl Taq buffer with KCl
0.4μl dNTPs
1μl COMPX-F Primer
1μl COMPX-R Primer
0.25μl Taq DNA Polymerase
2μl MgCl2
1μl TOP10 Solution
12.35μl ddH2O
Renaturation at 55℃, extend for 1 min. 2 tubes without Taq DNA Polymerase as blank control.
4th Electrophoresis for Colonical PCR
Bands are predicted except one. Maybe caused by accident. PCR product keeped for later use.
1st Plasmid Extraction for BBa K1492002 & pSB1C3
1 Centifuge 5 ml bacterium solution, few sediment getted.
2 Add 250 μl Buffer P1, resuspend cells.
3 Add 250 μl Buffer P2, mix well, 4 min's standing.
4 Add 350 μl Buffer P3, mix well.
5 13.4 rpm centifuge 10 min, move all supernatant to adsorption column, 11.6 rpm centifuge 30 s, discard filtrate.
6 Add 500 μl Wash Solution, 11.6 rpm centifuge 30 s, discard filtrate. Repeat once.
7 11.6 rpm centifuge 1 min.
8 Add 50 μl ddH2O, 2 min's standing, 11.6 rpm centifuge 1 min.
1st Electrophoresis for Plasmid Extraction
No result. May due to high concentration of chloramphenicol, bacteria grow very slow. As there is few bacteria, no plasmid getted. Bacterium solution dilute 1:1 with LB medium.
2015 13/7
The steak-plate of Pseudomonas aeruginosa
By using the steak-plate method, we successful separated the single conlony of P. aeruginosa.
\(^o^)/~！
2nd Plasmid Extraction for BBa K1492002 & pSB1C3
Centifuge 4.5 ml bacterium solution.
Add 250 μl Buffer P1, resuspend cells.
Add 250 μl Buffer P2, mix well, 4 min's standing.
Add 350 μl Buffer P3, mix well.
13.4 rpm centifuge 10 min, move all supernatant to adsorption column, 11.6 rpm centifuge 30 s, discard filtrate.
Add 500 μl Wash Solution, 11.6 rpm centifuge 30 s, discard filtrate. Repeat once.
11.6 rpm centifuge 1 min.
Add 30 μl ddH2O, 2 min's standing, 11.6 rpm centifuge 1 min.
2nd Electrophoresis for Plasmid Extraction
Successfully extracted tRNA plasmid. Failed to extract pSB1C3 plasmid, cause culture is contaminated.
Result:
tRNA 114ng/μl
1st PCR for tRNA
2μl Taq buffer with KCl
0.4μl dNTPs
1μl tRNA-F2 Primer
1μl tRNA-R Primer
0.25μl Taq DNA Polymerase
2μl MgCl2
1μl tRNA plasmid
12.35μl ddH2O
Renaturation at 50℃, extend for 1:30.
1st Electrophoresis for tRNA PCR
No result.
Transformation of Four Parts of Plate 4
material:
competent cell TG1 from the lab 319
4 parts of the plate 4(pSB1A2)
No.1:BBa_B0034 No.2:BBa_C0051
No.3:BBa_R0051 No.4:BBa_E0040
the procedure
Take the competent bacteria from refrigerator and incubate them into ice about 10 mins until it is dissolved
Absorb 1 uL or 2 uL plasmid and mix it with bacteria solution thoroughly.
Put the tubes on the ice about 30 mins.
Make a heat shock at 42 degree centigrade about 60 sec
Put the tubes on the ice about 3 mins again.
Add 900 ul LB medium into EP tubes and cultivate the bacteria at 37 degree centrigrade about 60 min.
Centrifuge them at 4,000xg about 15 sec and we will se sediment in the tubes.
Discard the supernatant liquid and leave about 100ul medium.
Coat plate: add 100ul solution in a plate
2015 14/7
The Results of Transformation of Jul 13th
transformations of three parts are successful:
No.1 B0034
No.2 C0051
No.4 E0040
transformation of one part is failing:
No.3 R0051
Plasmid Extraction of pSB1C3
the plasmis:pSB1C3
the procedure:
Absorb 6 mL bacteria solution in to EP tubes, centrifuge them at 12,000xg 1mins and then discard the culture medium.
Add 250 ul Buffer S1 into sediment, and use spearhead to make bacteria suspended.
Add 250 ul Buffer S2, and overturn the EP tubes 8 times immediately and tenderly. Stand tubes about 3 mins.
Add 350 ul Buffer W3 and overturn the EP tubes 10 times again immediately and tenderly.
Centrifuge tubes at 12,000xg about 10 mins.
Pour the supernatant liquid into absorption columns ,centrifuge them at 12,000xg about 30 sec and then discard the liquid in the collecting pipe.
Add 500 ul Buffer W1 and centrifuge them at 12,000xg about 30 sec. Then discard the liquid in the collecting pipe.
Add 700 ul Buffer W2, centrifuge them at 12,000xg about 30 sec and discard the liquid in the collecting pipe.
Do the step 8 again.
Centrifuge the empty columns at 12,000xg about 1 min.
Put the columns in clean 1.5mL EP tubes, add 80 ul Elution Buffer on the absorption film, stand 1 min and centrifuge about 1 min at 12,000xg.
PS:this protocol is from lab 319
the concentration :11.0ng/ul
PS:the concentration of the control plasmid is 136.0 ng/ul,so we guess that maybe the concentration of bacterium solution is low.Also we guess that it's caused by the missing of spreading.
Transformation of Three Parts and One Plasmid
the plasmid going to be transformed:
No.1 BBa_R0062(plate 2;pSB1C3)
No.2 BBa_B0015(plate 3;pSB1C3)
No.3 BBa_R0051(plate 4;pSB1A2)
No.4 pSB1C3
the procedure:
note:add 1 ul R0062 ,1 ul B0015,2 ul R0051,
2 ul pSB1C3 to 100 ul competent cells respectively
Take the competent bacteria from refrigerator and incubate them into ice about 10 mins until it is dissolved
Absorb 1 uL or 2 uL plasmid and mix it with bacteria solution thoroughly.
Put the tubes on the ice about 30 mins.
Make a heat shock at 42 degree centigrade about 60 sec
Put the tubes on the ice about 3 mins again.
Add 900 ul LB medium into EP tubes and cultivate the bacteria at 37 degree centrigrade about 60 min.
Centrifuge them at 4,000xg about 15 sec and we will se sediment in the tubes.
Discard the supernatant liquid and leave about 100ul medium.
Coat plate: add 100ul solution in a plate
Plasmid Extraction of Three Parts of Plate 4
the procedure is the same as before
the concentration :
B0034 157.6ng/ul, 113.7ng/ul;
C0051 171.7ng/ul, 194.6ng/ul;
E0040 278.5ng/ul, 228.2ng/ul
2015 15/7
Results of transformation
R0051 no colony
R0062 have single colony
B0015 have single colony
pSB1C3 have single colony and the colony is red
Plasmid Extraction Results
This is extraction was for previous transformation.
pSB1C3(1): 108ng/uL
pSB1C3(2): 157ng/uL
B0015(1): 19ng/uL
B0015(2): 93ng/uL
Prepared for further use.
2015 16/7
Colonical PCR for micF and SoxS
Primers produced from Sangon Biotech Co.,Ltd.
Primer
Sequence(5'-3')
micF-luxI-R
TTATAGTCATATAAATTTTCTCCTCTTTCCGAATGCGAGGCATCC
micF-F
GCGGCCGCTTCTAGAGCCTCATTAATCAGTCGGC
SoxS-RBS-luxI-R
TATAGTCATATAAATTTTCTCCTCTTTCTAGTGCGTGCGCCGGTACCTT
BB-SoxS-F
CGGCCGCTTCTAGAGCGTGCGGAACATTCG
For micF, primers including micF-luxI-R, micF-F, protocol as following:
2μl Taq buffer with KCl
0.4μl dNTPs
1μl micF-luxI-R Primer
1μl micF-F Primer
0.25μl Taq DNA Polymerase
2μl MgCl2
1μl Top10(K-12) Strain E. coli bacterial solution
12.35μl ddH2O
For SoxS, primers including SoxS-RBS-luxI-R, BB-SoxS-F, protocol illustrates as:
2μl Taq buffer with KCl
0.4μl dNTPs
1μl SoxS-RBS-luxI-R Primer
1μl BB-SoxS-F Primer
0.25μl Taq DNA Polymerase
2μl MgCl2
1μl 1μl Top10(K-12) Strain E. coli bacterial solution
12.35μl ddH2O
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)
Elongation for whole at 72 degree celcius 7 min
4 celcius degree for preserveation.
Characterization of Colonical PCR by Gel Electrophoresis
The result illustrated the success of PCR
2015 17/7
Colonical PCR for OprF from Psudomonus aeruginosa
Primers produced from Sangon Biotech Co.,Ltd.
Primer
Sequence(5'-3')
OprF-F
AGAGGAGAAAATCGGAATGAAACTGAAGAACAC
OprF+Ter-R
TGATGCCTGGCTCTAGTATTACTTGGCTTCAGCTT
PCR protocol:
For OprF, primers including OprF-F, OprF+Ter-R, protocol as following:
2μl Taq buffer with KCl
0.4μl dNTPs
1μl OprF-F Primer
1μl OprF+Ter-R Primer
0.25μl Taq DNA Polymerase
2μl MgCl2
1μl Psudomonus aeruginosa (PAO1) bacterial solution
12.35μl ddH2O
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)
Elongation for whole at 72 degree celcius 7 min
4 celcius degree for preserveation.
Plasmid Extraction of SCVE
SCVE synthesized from Sangon Biotech Co.,Ltd.
Procedure including:
Absorb 6 mL bacteria solution in to EP tubes, centrifuge them at 12,000xg 1mins and then discard the culture medium.
Add 250 ul Buffer S1 into sediment, and use spearhead to make bacteria suspended.
Add 250 ul Buffer S2, and overturn the EP tubes 8 times immediately and tenderly. Stand tubes about 3 mins.
Add 350 ul Buffer W3 and overturn the EP tubes 10 times again immediately and tenderly.
Centrifuge tubes at 12,000xg about 10 mins.
Pour the supernatant liquid into absorption columns ,centrifuge them at 12,000xg about 30 sec and then discard the liquid in the collecting pipe.
Add 500 ul Buffer W1 and centrifuge them at 12,000xg about 30 sec. Then discard the liquid in the collecting pipe.
Add 700 ul Buffer W2, centrifuge them at 12,000xg about 30 sec and discard the liquid in the collecting pipe.
Do the step 8 again.
Centrifuge the empty columns at 12,000xg about 1 min.
Put the columns in clean 1.5mL EP tubes, add 80 ul Elution Buffer on the absorption film, stand 1 min and centrifuge about 1 min at 12,000xg.
Characterization of SCVE and OprF by Gel Electrophoresis
Only 1/4 PCR results successfully duplicated results, plasimid extraction from SCVE showed well.
Second PCR for OprF didn't work well:(
Digestion of E0040(EGFP)
Enzyme from Thermo Scientific FastDigest Enzyme with Green Buffer
Protocol as following:
Nuclease-free water: 15ul
E0040: 1ul
10* FastDigest Green Buffer: 2ul
EcoRI: 1ul
SpeI: 1ul
We are trying to figure out the best conditions of our experiment and whether we need loading buffer
Time
With Loading Buffer or not
5min
Yes(1), No(2)
2h
Yes(3), No(4)
PCR of E0040(EGFP)
Primers produced from Sangon Biotech Co.,Ltd.
Primer
Sequence(5'-3')
E0040-F
ATGCGTAAAGGAGAAGAACTTT
R0051-E0040-R
TACGCATATAAATTTTCTCCTCTTTGCAACCA
For E0040, primers including E0040-F, R0051-E0040-R, protocol as following:
2μl Taq buffer with KCl
0.4μl dNTPs
1μl E0040-F Primer
1μl R0051-E0040-R Primer
0.25μl Taq DNA Polymerase
2μl MgCl2
1μl plasmid(pSB1C3) containing E0040
12.35μl ddH2O
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)
Elongation for whole at 72 degree celcius 7 min
4 celcius degree for preserveation.
Attention: R0051-E0040-R Primer is not appropriate because of only 8 complementary base pairs.
Characterization of E0040 from Digestion and PCR
Result illustrated that We do need loading buffer, and 5 min do work!
PCR showed more concentration of E0040!
Gel Extraction
Preparation:
Check out whether ethyl alcohol is added into Wash Solution.
Check out whether Buffer B2 has sediment.
Adjust the thermostat water bath at 50 degree centigrade.
Procedure:
Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.
Measure out the weight of tubes of gel.
Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.
Optional: If the length of the DNA strip is shorter than 500 bp, it would be better add isopropanol one third volume of Buffer B2.
After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s.
Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.
Do the step 5 again.
Centrifuge the empty columns at 9000xg about 1min.
Open the columns and air them about 10 mins.
Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.
Keep the DNA solution and often measure the concentration of DNA and A260/A280 for further study.
Final OD results:
Code
OD(ng/ul)
PCR(1)
15.3
PCR(2)
4.5
PCR(3)
27.0
Digestion(1) with 5 mins
15.4
Digestion(3) with 2 h
22.7
Perseved for further use.
2015 18/7
Transfofmation of Three Parts
Protocol as following
100μl (three tubes)competent cells
C0062(pSB1C3)
R0010(pSB1C3)
C0061(pSB1C3)
LB without antibiotic
LB medium with Amp/CmR
Plasmid Extraction of E0040
Procedure including:
1.Absorb 6 mL bacteria solution in to EP tubes, centrifuge them at 12,000xg 1mins and then discard the culture medium.
2.Add 250 ul Buffer S1 into sediment, and use spearhead to make bacteria suspended.
3.Add 250 ul Buffer S2, and overturn the EP tubes 8 times immediately and tenderly. Stand tubes about 3 mins.
4.Add 350 ul Buffer W3 and overturn the EP tubes 10 times again immediately and tenderly.
5.Centrifuge tubes at 12,000xg about 10 mins.
6.Pour the supernatant liquid into absorption columns ,centrifuge them at 12,000xg about 30 sec and then discard the liquid in the collecting pipe.
7.Add 500 ul Buffer W1 and centrifuge them at 12,000xg about 30 sec. Then discard the liquid in the collecting pipe.
8.Add 700 ul Buffer W2, centrifuge them at 12,000xg about 30 sec and discard the liquid in the collecting pipe.
9.Do the step 8 again.
10.Centrifuge the empty columns at 12,000xg about 1 min.
11.Put the columns in clean 1.5mL EP tubes, add 80 ul Elution Buffer on the absorption film, stand 10 min and centrifuge about 1 min at 12,000xg.
Results
Code
OD(ng/ul)
E0040(1)
166.7
E0040(2)
169.2
Colonical PCR for OprF from Psudomonus aeruginosa
PCR protocol:
For OprF, primers including OprF-F, OprF+Ter-R, protocol as following:
2μl Taq buffer with KCl
0.4μl dNTPs
1μl OprF-F Primer
1μl OprF+Ter-R Primer
0.5μl Taq DNA Polymerase(add more 0.25ul then previous protocol)
2μl MgCl2
1μl Psudomonus aeruginosa (PAO1) bacterial solution
12.35μl ddH2O
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)
Elongation for whole at 72 degree celcius 7 min
4 celcius degree for preserveation.
Digestion of E0040(EGFP)
Enzyme from Thermo Scientific FastDigest Enzyme with Green Buffer
Protocol as following:
Nuclease-free water: 15ul
E0040: 1ul
10* FastDigest Green Buffer: 2ul
EcoRI: 1ul
SpeI: 1ul
Characterization of OprF and E0040
Plasmid Extraction of Cas9(K1218011)
The bacteria were cultured at 9 am from successfully transformed bacteria.
Procedure including:
1.Absorb 6 mL bacteria solution in to EP tubes, centrifuge them at 12,000xg 1mins and then discard the culture medium.
2.Add 250 ul Buffer S1 into sediment, and use spearhead to make bacteria suspended.
3.Add 250 ul Buffer S2, and overturn the EP tubes 8 times immediately and tenderly. Stand tubes about 3 mins.
4.Add 350 ul Buffer W3 and overturn the EP tubes 10 times again immediately and tenderly.
5.Centrifuge tubes at 12,000xg about 10 mins.
6.Pour the supernatant liquid into absorption columns ,centrifuge them at 12,000xg about 30 sec and then discard the liquid in the collecting pipe.
7.Add 500 ul Buffer W1 and centrifuge them at 12,000xg about 30 sec. Then discard the liquid in the collecting pipe.
8.Add 700 ul Buffer W2, centrifuge them at 12,000xg about 30 sec and discard the liquid in the collecting pipe.
9.Do the step 8 again.
10.Centrifuge the empty columns at 12,000xg about 1 min.
11.Put the columns in clean 1.5mL EP tubes, add 80 ul Elution Buffer on the absorption film, stand 10 min and centrifuge about 1 min at 12,000xg.
Characterization of Cas9
2015 19/7
Digestion of K1218011(Cas9)
Enzyme from Thermo Scientific FastDigest Enzyme with Green Buffer
Protocol as following:
Nuclease-free water: 15ul
K1218011: 1ul
10* FastDigest Green Buffer: 2ul
EcoRI: 1ul
SpeI: 1ul
Incubate into 37 degree celcius, 2h
Digestion of SCVE
Enzyme from Thermo Scientific FastDigest Enzyme with Green Buffer
Protocol as following:
Nuclease-free water: 16ul
SCVE: 1ul
10* FastDigest Green Buffer: 2ul
SmaI: 1ul
Incubate into 37 degree celcius, 2h
Colonical PCR for OprF from *Psudomonus aeruginosa, micF and SoxS from K-12
PCR protocol:
For OprF, primers including OprF-F, OprF+Ter-R, protocol as following:
2μl Taq buffer with KCl
0.4μl dNTPs
1μl OprF-F Primer
1μl OprF+Ter-R Primer
0.25μl Taq DNA Polymerase
2μl MgCl2
1μl Psudomonus aeruginosa (PAO1) bacterial solution
12.35μl ddH2O
For micF, primers including micF-luxI-R, micF-F, protocol as following:
2μl Taq buffer with KCl
0.4μl dNTPs
1μl micF-luxI-R Primer
1μl micF-F Primer
0.25μl Taq DNA Polymerase
2μl MgCl2
1μl Top10(K-12) Strain E. coli bacterial solution
12.35μl ddH2O
For SoxS, primers including SoxS-RBS-luxI-R, BB-SoxS-F, protocol illustrates as:
2μl Taq buffer with KCl
0.4μl dNTPs
1μl SoxS-RBS-luxI-R Primer
1μl BB-SoxS-F Primer
0.25μl Taq DNA Polymerase
2μl MgCl2
1μl 1μl Top10(K-12) Strain E. coli bacterial solution
12.35μl ddH2O
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)
Elongation for whole at 72 degree celcius 7 min
4 celcius degree for preserveation.
Praparation for Competent Cell
protocol as the following:
Pick out two single colonies(TG1) and cultivate bacteria in 5mL LB media at 37 degree centigrade about 12 hours with shocking.
Take 5mL nutrient solution into 100 mL media and cultivate bacteria at 37 degree centigrade about one hour and a half with shocking till the OD660 reaches 0.4-0.5.
Absorb 50 mL nutrient solution into EP tubes with ice-bath about 10 mins.
Centrifuge the tubes at 4100xg at 4 degree centigrade about 10mins and discard the supernatant liquid.
Utilize 30 mL 0.1M calcium chloride to suspend bacteria.
Centrifuge the tubes at 4100xg at 4 degree centigrade about 10 mins and discard the supernatant liquid.
Utilize 2 mL 0.1M calcium chloride with 15% glycerol to suspend bacteria and then ready for transformation quickly or restore at -70 degree centigrade.
Time(min)
OD600
40
0.195
60
0.203
80
0.309
90
0.439
PS:dilute the solution by adding 100 mL LB media after first test.
2015 20/7
Negative Chemotaxis Characterization
term 1, material:
LB solid medium plates x4
1g/ml Chloramphenicol
E.coli Top10
filter paper
Plasmid Extraction of EmrE and ArcB
done by Wang Luyao
Procedure including:
Absorb 6 mL bacteria solution in to EP tubes, centrifuge them at 12,000xg 1mins and then discard the culture medium.
Add 250 ul Buffer S1 into sediment, and use spearhead to make bacteria suspended.
Add 250 ul Buffer S2, and overturn the EP tubes 8 times immediately and tenderly. Stand tubes about 3 mins.
Add 350 ul Buffer W3 and overturn the EP tubes 10 times again immediately and tenderly.
Centrifuge tubes at 12,000xg about 10 mins.
Pour the supernatant liquid into absorption columns ,centrifuge them at 12,000xg about 30 sec and then discard the liquid in the collecting pipe.
Add 500 ul Buffer W1 and centrifuge them at 12,000xg about 30 sec. Then discard the liquid in the collecting pipe.
Add 700 ul Buffer W2, centrifuge them at 12,000xg about 30 sec and discard the liquid in the collecting pipe.
Do the step 8 again.
Centrifuge the empty columns at 12,000xg about 1 min.
Put the columns in clean 1.5mL EP tubes, add 80 ul Elution Buffer on the absorption film, stand 10 min and centrifuge about 1 min at 12,000xg.
the concentration as following
EmrE 70.2 ng/ul 92.3 ng/ul
ArcB 62.6 ng/ul 95.2 ng/ul
Transformation of R0051 and C0061
the source of parts:
No.1 R0051 2014 kit plate 4
No.2 R0051 2014 kit plate 4
No.3 control
No.4 C0061 2015 kit plate 4
No.5 C0061 2015 kit plate 4
No.6 C0061 2014 kit plate 4
No.1 and No.4 the competent cell is from lab319
No.2 and No.5 the competent cell is made by us (firt)
No.3 and No.6 the competent cell is made by us (second)
the protocol as following
Take the competent bacteria from refrigerator and incubate them into ice about 10 mins until it is dissolved
Absorb 1 uL or 2 uL plasmid and mix it with bacteria solution thoroughly.
Put the tubes on the ice about 30 mins.
Make a heat shock at 42 degree centigrade about 60 sec
Put the tubes on the ice about 3 mins again.
Add 900 ul LB medium into EP tubes and cultivate the bacteria at 37 degree centrigrade about 60 min.
Centrifuge them at 4,000xg about 15 sec and we will se sediment in the tubes.
Discard the supernatant liquid and leave about 100ul medium.
Coat plate: add 100ul solution in a plate
PCR of OrpF
material:
13 uL Taq Master Mix
11 uL dd water
1 uL PAO1 bacterial
0.5 uL primer Fd(OrpF-F)
0.5 uL primer Rv(OrpF+Ter-R)
PS:Taq Master Mix is from lab 319 ,and this reagent is used for bacterial PCR.
Renaturation at 50℃, extend for 1:10
2015 21/7
PCR of OprF
material:
No.1 and No.2:
6.5 uL Taq Master Mix
1 uL PAO1 bacterial solution
4.5 uL dd water
0.5 uL OprF-F
0.5 uL OprF+Ter-R
No.3 and No.4:
6.5 uL Taq Master Mix
4 uL OprF PCR product
1.5 uL dd water
0.5 uL OprF-F
0.5 uL OprF+Ter-R
PS:Taq Master Mix is from lab 319 ,and this reagent is used for bacterial PCR.
Renaturation at 55℃, extend for 1:10
OprF for PCR  and gel electrophoresisOprF
analyze: the length of the product is
2015 22/7
Preparation of Semi-Solid LB Medium
In order to characterization the movement of TOP10, semi-solid LB medium preparation is indispensable in this research.
(500mL Protocol)
Ingredients
Mass
Proportion(g/mL)
Yeast Extract
2.5g
0.5%
Trtptone
5g
1%
NaCl
5g
1%
Agar
1.25g
0.25%
Compared with solid LB medium, containing 1.5% agar in there.
Plasmid Extraction C0051,B0015 and R0062
Procedure including:
Absorb 6 mL bacteria solution in to EP tubes, centrifuge them at 12,000xg 1mins and then discard the culture medium.
Add 250 ul Buffer S1 into sediment, and use spearhead to make bacteria suspended.
Add 250 ul Buffer S2, and overturn the EP tubes 8 times immediately and tenderly. Stand tubes about 3 mins.
Add 350 ul Buffer W3 and overturn the EP tubes 10 times again immediately and tenderly.
Centrifuge tubes at 12,000xg about 10 mins.
Pour the supernatant liquid into absorption columns ,centrifuge them at 12,000xg about 30 sec and then discard the liquid in the collecting pipe.
Add 500 ul Buffer W1 and centrifuge them at 12,000xg about 30 sec. Then discard the liquid in the collecting pipe.
Add 700 ul Buffer W2, centrifuge them at 12,000xg about 30 sec and discard the liquid in the collecting pipe.
Do the step 8 again.
Centrifuge the empty columns at 12,000xg about 1 min.
Put the columns in clean 1.5mL EP tubes, add 80 ul Elution Buffer on the absorption film, stand 10 min and centrifuge about 1 min at 12,000xg.
the concentration as following
C0015:171.7 ng/uL
B0015:93 ng/uL
R0062:144.3 ng/uL
PCR of C0051,B0015 and R0062
material:
C0051 plasmid(171.7 ng/uL)
B0015 plasmid(93 ng/uL)
R0062 plasmid(144.3 ng/uL)
all the solutions are diluted 10-folds
Primers produced from Sangon Biotech Co.,Ltd.
Primers
Sequence(5' to 3')
C0051-F(I)
att tat atg agc aaa aag aaa cca tta tca
C0051-B0015-R(I)
tat ttg atg cct ggc tct agt gat cta cac tag cac ta
B0015-R(I)
gcc gct act agt taa acg cag aaa ggc cca cc
B0015-F
cca ggc atc aaa taa aac gaa agg
R0062-F
gcg gcc gct tct aga gac ctg tag gat cg
R0062-C0051-R
tgt gct cat ata aat ttt ctc ctc ttt cta gat ttt att cga c
No.1 and No.2 for C0051(803bp)
Protocol as this:
2.5 μl Taq buffer with KCl
2.5 μl dNTPs
0.75 μl C0051-F Primer
0.75 μl C0051-B0015-R(I) Primer
0.5 μl Taq DNA Polymerase
1.0 μl MgCl2
2 uL C0051 solution
15 μl ddH2O
No.3 and No.4 for B0015(142bp)
Protocol as this:
2.5 μl Taq buffer with KCl
2.5 μl dNTPs
0.75 μl B0015-F Primer
0.75 μl B0015-R(I) Primer
0.5 μl Taq DNA Polymerase
1.0 μl MgCl2
3 uL C0051 solution
14 μl ddH2O
No.5 and No.6 for R0062(104bp)
Protocol as this:
2.5 μl Taq buffer with KCl
2.5 μl dNTPs
0.75 μl R0062-F Primer
0.75 μl R0062-C0051-R Primer
0.5 μl Taq DNA Polymerase
1.0 μl MgCl2
2 uL R0062 solution
15 μl ddH2O
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)
Elongation for whole at 72 degree celcius 2min(C0061)or 1min(B0015 and R0062)
4 celcius degree for preserveation.
Preparation Semi-Solid M63 Medium
In order to characterization the movement of TOP10, semi-solid M63 medium preparation is indispensable in this research.
(500mL Protocol)
Ingredients
Mass
Proportion(g/mL) or Mole
Yeast Extract
2.5g
0.5%
Trtptone
5g
1%
NaCl
5g
1%
Agar
1.25g
0.25%
KH2PO4
6.8g
50mM
KOH
2.1g
37.5mM
(NH4)2SO4
0.09g
7.5mM
MgSO4
0.06g
0.5mM
FeSO4·7H2O
0.2710g
1.95mM
D-Glucose
1.98g
11mM
Colonical PCR for OprF from Psudomonus aeruginosa and SCVE from pUC57
PCR protocol:
For OprF, primers including OprF-F, OprF+Ter-R, protocol as following:
2μl Taq buffer with KCl
0.4μl dNTPs
1μl OprF-F Primer
1μl OprF+Ter-R Primer
0.25μl Taq DNA Polymerase
2μl MgCl2
1μl Psudomonus aeruginosa (PAO1) bacterial solution
12.35μl ddH2O
For SCVE, Primers produced from Sangon Biotech Co.,Ltd.
Primer
Sequence(5'-3')
SCVE-T-R
TTGATGCCTGGCTCTAGTACACCAGCAGATCCGG
T7-RBS-SCVE-F
TTTTTGCATACTAGAGAAAGAGGAGAAAATTTATATGTATAGCTTTGTG
Protocol as this:
2μl Taq buffer with KCl
0.4μl dNTPs
1μl SCVE-T-R Primer
1μl T7-RBS-SCVE-F Primer
0.25μl Taq DNA Polymerase
2μl MgCl2
1μl SCVE solution from plasmid extraction
12.35μl ddH2O
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)
Elongation for whole at 72 degree celcius 7 min
4 celcius degree for preserveation.
Gel Electrophoresis of OprF, SCVE micF and SoxS
micF and SoxS
OprF and SCVE
Gel Extraction
Material including: OprF, SoxS, micF
Preparation:
Check out whether ethyl alcohol is added into Wash Solution.
Check out whether Buffer B2 has sediment.
Adjust the thermostat water bath at 50 degree centigrade.
Procedure:
Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.
Measure out the weight of tubes of gel.
Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.
Optional: If the length of the DNA strip is shorter than 500 bp, it would be better add isopropanol one third volume of Buffer B2.
After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s.
Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.
Do the step 5 again.
Centrifuge the empty columns at 9000xg about 1min.
Open the columns and air them about 10 mins.
Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.
Keep the DNA solution and often measure the concentration of DNA and A260/A280 for further study.
2015 23/7
Plasmid Extraction of R0062
Procedure including:
Absorb 6 mL bacteria solution in to EP tubes, centrifuge them at 12,000xg 1mins and then discard the culture medium.
Add 250 ul Buffer S1 into sediment, and use spearhead to make bacteria suspended.
Add 250 ul Buffer S2, and overturn the EP tubes 8 times immediately and tenderly. Stand tubes about 3 mins.
Add 350 ul Buffer W3 and overturn the EP tubes 10 times again immediately and tenderly.
Centrifuge tubes at 12,000xg about 10 mins.
Pour the supernatant liquid into absorption columns ,centrifuge them at 12,000xg about 30 sec and then discard the liquid in the collecting pipe.
Add 500 ul Buffer W1 and centrifuge them at 12,000xg about 30 sec. Then discard the liquid in the collecting pipe.
Add 700 ul Buffer W2, centrifuge them at 12,000xg about 30 sec and discard the liquid in the collecting pipe.
Do the step 8 again.
Centrifuge the empty columns at 12,000xg about 1 min.
Put the columns in clean 1.5mL EP tubes, add 80 ul Elution Buffer on the absorption film, stand 10 min and centrifuge about 1 min at 12,000xg.
PCR for OprF and C0061
PCR protocol:
For OprF, primers including OprF-F, OprF+Ter-R, protocol as following:
2μl Taq buffer with KCl
0.4μl dNTPs
1μl OprF-F Primer
1μl OprF+Ter-R Primer
0.25μl Taq DNA Polymerase
2μl MgCl2
1μl Psudomonus aeruginosa (PAO1) bacterial solution
12.35μl ddH2O
For SCVE, Primers produced from Sangon Biotech Co.,Ltd.
Primer
Sequence(5'-3')
C0061-F
ATGACTATAATGATAAAAAAATCGGATTTTTTGGCA
C0061-B0015-R
ATTTGATGCCTGGGAGATCTACACTAGC
Protocol as this:
2μl Taq buffer with KCl
0.4μl dNTPs
1μl C0061-F Primer
1μl C0061-B0015-R Primer
0.25μl Taq DNA Polymerase
2μl MgCl2
1μl C0061 solution
12.35μl ddH2O
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)
Elongation for whole at 72 degree celcius 7 min
4 celcius degree for preserveation.
Gel Extraction of C0062,B0015 and R0062
Material including: C0051,B0015 and R0062
Preparation:
Check out whether ethyl alcohol is added into Wash Solution.
Check out whether Buffer B2 has sediment.
Adjust the thermostat water bath at 50 degree centigrade.
Procedure:
Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.
Measure out the weight of tubes of gel.
Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.
Optional: If the length of the DNA strip is shorter than 500 bp, it would be better add isopropanol one third volume of Buffer B2.
After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s.
Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.
Do the step 5 again.
Centrifuge the empty columns at 9000xg about 1min.
Open the columns and air them about 10 mins.
Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.
Keep the DNA solution and often measure the concentration of DNA and A260/A280 for further study.
the concentration as following:
C0051:12.7 ng/uL
B0015:7.3 ng/uL
R0062:7.7 ng/uL
PCR of OprF
material:
13 uL Taq Master Mix
11 uL dd water
1 uL PAO1 bacterial
0.5 uL primer Fd(OrpF-F)
0.5 uL primer Rv(OrpF+Ter-R)
Renaturation at 55℃, extend for 2min
Gel Electrophoresis Characterization
C0051 and B0015
OprF
That was relatively successully extracted OprF from colonical PCR.
R0062
2015 24/7
Preparation for Homologous Recombination: Plasimid Digestion
Enzyme from Thermo Scientific FastDigest Enzyme with 10X Green Buffer
Protocol as following:
Nuclease-free water: 15ul
pSB1C3: 1ul
10* FastDigest Green Buffer: 2ul
EcoRI: 1ul
SpeI: 1ul
Incubate solution at 37 degree celcius about 2h.
PCR of C0061,B0015 and micF
material:
C0061 plasmid: 477.1 ng/uL(diluted 20-folds)
B0015 PCR product:7.3 ng/uL
micF-1 PCR product:13.4 ng/uL
ForB0015 Primers produced from Sangon Biotech Co.,Ltd.
Primer
Sequence(5'-3')
B0015-F
cca ggc atc aaa taa aac gaa agg
B0015-R(I)
gcc gct act agt ata taa acg cag aaa ggc cca cc
protocol:
2μl Taq buffer with KCl
0.4μl dNTPs
1μl Primer-Fd
1μl Primer-Rv
0.25μl Taq DNA Polymerase
2μl MgCl2
1uL C0061 plasmid,4uL B0015 plasmid,2uL plasmid
(13.35-X)μl ddH2O
Renaturation at 55℃, extend for 1min
2015 25/7
PCR of C0061,B0015,R0062,R0010 and C0062
material:
C0061 plasmid:477.1 ng/uL(0.2 uL)
B0015 plasmid:19 ng/uL(3 uL)
R0062 plasmid:143.3 ng/uL(0.5 uL)
C0051 plasmid:171.7 ng/uL(0.5 uL)
R0010 plasmid:357.7 ng/uL(diluted 10-folds,1 uL)
C0062 plasmid:739.4 ng/uL(diluted 10-folds,1uL)
for R0062.R0010 and C0062Primers produced from Sangon Biotech Co.,Ltd.
Primer
Sequence(5'-3')
R0062-F
gcg gcc gct tct aga ctg tag gat cg
R0062-C0051-R
tgt gct cat ata aat ttt ctc ttt cta gat ttt att cga c
backbone-R0010-F
ccgctt cta gag cca tac gca aac cgc ctc tc
C0062-B0034-R
cta gtg tc agt aat aaa ttt tct cct ctt tg tgt gaa att gt
B0034-C0062-F
tat tac tga gca cta gat gaa aaa cat aaa tgc cga c
B0015-C0062-R
atg cct vggc tct agt gat cta cac tag cac t
protocol:
2μl Taq buffer with KCl
0.4μl dNTPs
1μl Primer-Fd
1μl Primer-Rv
0.25μl Taq DNA Polymerase
2μl MgCl2
XuL plasmid
(13.35-X)μl ddH2O
Renaturation at 55℃, extend for 1min
PS:
C0061:656bp B0015:142bp R0062:104bp
C0051:803bp C0062:813bp R0010:243bp
Characterization of micF and SoxS by Gel Electrophoresis
micF were successfully PCRed.
Characterization of pSB1C3 and SoxS by Gel Electrophoresis
2015 26/7
Characterization of R0051, C0061 and C0051 by Gel Electrophoresis
Plasimid Extraction for pSB1C3 and C0061
Procedure including:
1.Absorb 6 mL bacteria solution in to EP tubes, centrifuge them at 12,000xg 1mins and then discard the culture medium.
2.Add 250 ul Buffer S1 into sediment, and use spearhead to make bacteria suspended.
3.Add 250 ul Buffer S2, and overturn the EP tubes 8 times immediately and tenderly. Stand tubes about 3 mins.
4.Add 350 ul Buffer W3 and overturn the EP tubes 10 times again immediately and tenderly.
5.Centrifuge tubes at 12,000xg about 10 mins.
6.Pour the supernatant liquid into absorption columns ,centrifuge them at 12,000xg about 30 sec and then discard the liquid in the collecting pipe.
7.Add 500 ul Buffer W1 and centrifuge them at 12,000xg about 30 sec. Then discard the liquid in the collecting pipe.
8.Add 700 ul Buffer W2, centrifuge them at 12,000xg about 30 sec and discard the liquid in the collecting pipe.
9.Do the step 8 again.
10.Centrifuge the empty columns at 12,000xg about 1 min.
11.Put the columns in clean 1.5mL EP tubes, add 80 ul Elution Buffer on the absorption film, stand 10 min and centrifuge about 1 min at 12,000xg.
Gel Extraction
Material including: R0051, C0051 and C0061
Preparation:
Check out whether ethyl alcohol is added into Wash Solution.
Check out whether Buffer B2 has sediment.
Adjust the thermostat water bath at 50 degree centigrade.
Procedure:
Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.
Measure out the weight of tubes of gel.
Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.
Optional: If the length of the DNA strip is shorter than 500 bp, it would be better add isopropanol one third volume of Buffer B2.
After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s.
Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.
Do the step 5 again.
Centrifuge the empty columns at 9000xg about 1min.
Open the columns and air them about 10 mins.
Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.
Keep the DNA solution and often measure the concentration of DNA and A260/A280 for further study.
Characterization of B0015 and R0051 by Gel Electrophoresis
2015 27/7
Overlap Circuit
This sensing circuit including promoter micF and SoxS.
Overlapping Fragments
In order to ligate the fragements micF or SoxS, C0061 and B0015, we need to make the equal mole proportion.
The mole of DNA molecule is calculated in the following ways:
among which, c is the concentration of DNA, V is the volumn required and n is the length of DNA. Because of the same molar mass of As, Gs, Cs and Ts, computation of same molar mass is omitted.
According to our sample extracted from PCR and calculation, ingradients characterize as below:
Name
Concentration(ng/uL)
Length(bp)
micF
12.6
358
SoxS
27.5
705
C0061(LuxI)
18.5
656
B0015(Terminator)
9.5
143
Total
1161
The theoratical volumn proportion is:
SoxS-C0061-B0015: 5: 7: 3
micF-C0061-B0015: 5.5: 7: 3
Firstly, to ligate these parts, we need 10 cycles without primer:
for circuit SoxS-C0061-B0015:
Name
Volumn(uL)
SoxS
5
C0061
7
B0015
3
Taq Polymerase
0.25
MgCl2
2
dNTP
0.4
10X buffer with KCl
2
ddH2O
0.35
for circuit micF-C0061-B0015:
Name
Volumn(uL)
micF
5.5
C0061
7
B0015
3
Taq Polymerase
0.25
MgCl2
2
dNTP
0.4
10X buffer with KCl
2
PCR cycle including:
Preheat: 94 degree celcius, 5min
10 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)
Elongation for whole at 72 degree celcius 1min
4 celcius degree for preserveation.
Then, we extracted 5 ul as template, and 15 ul adding with primer:
Primers were synthesized from Sangon Biotech. Co.,Ltd.
Primers
Sequence(5' to 3')
micF-F
GCGGCCGCTTCTAGAGCCTCATTAATCAGTCGGC
B0015-R
GCCGCTACTAGTATATAAACGCAG
BB-SoxS-F
CGGCCGCTTCTAGAGCGTGCGGAACATTCG
5ul-template protocol:
Name
Volumn(uL)
Template
5
Taq Polymerase
0.25
MgCl2
2
dNTP
0.4
micF-F(BB-SoxS-F)
1
B0015-R
1
10X buffer with KCl
2
ddH2O
8.35
15ul-protocol:
Name
Volumn(uL)
Solution
15
micF-F(BB-SoxS-F)
1
B0015-R
1
10X buffer with KCl
2
ddH2O
1
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)
Elongation for whole at 72 degree celcius 1min
4 celcius degree for preserveation.
PCR of Five Parts
material:
R0010 plasmid:193.1ng/uL
C0062 plasmid:952.5ng/uL
B0015 plasmid:93ng/uL
R0062 plasmid：143.3ng/uL
For R0051 Primers produced from Sangon Biotech Co.,Ltd.
protocol:
2uL 10X buffer with KCl
0.4uL dNTP
1uL primers-F
1uL primers-R
0.25uL Taq Polymerase
2uL MgCl2
1uL plasmid
12.35 uL ddH2O
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)
Elongation for whole at 72 degree celcius 1min
4 celcius degree for preserveation.
Overlap PCR of bacteria III
material:
R0062 PCR product:21.2 ng/uL
C0051 PCR product:22.8 ng/uL
B0015 PCR product:24.6 ng/uL
protocol:
total volume 20 uL
R0062(104bp) 1 uL
C0051(803bp) 8 uL
B0015(153bp) 1.5uL
ddH2O 4.85 uL
buffer 2uL
dNTP 0.4uL
Taq polymerase 0.25uL
MgCl2 2uL
PCR cycle including:
Preheat: 94 degree celcius, 5min
10 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)
Elongation for whole at 72 degree celcius 1min
4 celcius degree for preserveation.
After the 10 cycles,we divide the 20 ul system into two parts:5ul and 15ul.The 5ul PCR product will be used as template in another new 20ul system.On the other hand, add primers
(R0062-F primer 1uL;C0062-B0015-R(I) primer 1uL)to the remaining 15 uL PCR product.
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)
Elongation for whole at 72 degree celcius 1min
4 celcius degree for preserveation.
Result:
we don't get the product we want.We guess the templates are not the right products,maybe they have mutations .So we decide to do the overlap PCR of bateriaI to test whether the method is suitable.
2015 28/7
Plasmid Extraction of gRNA-AcrB and gRNA-EmrE
Procedure including:
1.Absorb 6 mL bacteria solution in to EP tubes, centrifuge them at 12,000xg 1mins and then discard the culture medium.
2.Add 250 ul Buffer S1 into sediment, and use spearhead to make bacteria suspended.
3.Add 250 ul Buffer S2, and overturn the EP tubes 8 times immediately and tenderly. Stand tubes about 3 mins.
4.Add 350 ul Buffer W3 and overturn the EP tubes 10 times again immediately and tenderly.
5.Centrifuge tubes at 12,000xg about 10 mins.
6.Pour the supernatant liquid into absorption columns ,centrifuge them at 12,000xg about 30 sec and then discard the liquid in the collecting pipe.
7.Add 500 ul Buffer W1 and centrifuge them at 12,000xg about 30 sec. Then discard the liquid in the collecting pipe.
8.Add 700 ul Buffer W2, centrifuge them at 12,000xg about 30 sec and discard the liquid in the collecting pipe.
9.Do the step 8 again.
10.Centrifuge the empty columns at 12,000xg about 1 min.
11.Put the columns in clean 1.5mL EP tubes, add 80 ul Elution Buffer on the absorption film, stand 10 min and centrifuge about 1 min at 12,000xg.
Gel Extraction
Material including: R0061
Case 1: Sangon Biotech.
Preparation:
Check out whether ethyl alcohol is added into Wash Solution.
Check out whether Buffer B2 has sediment.
Adjust the thermostat water bath at 50 degree centigrade.
Procedure:
Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.
Measure out the weight of tubes of gel.
Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.
Optional: If the length of the DNA strip is shorter than 500 bp, it would be better add isopropanol one third volume of Buffer B2.
After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s.
Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.
Do the step 5 again.
Centrifuge the empty columns at 9000xg about 1min.
Open the columns and air them about 10 mins.
Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.
Keep the DNA solution and often measure the concentration of DNA and A260/A280 for further study.
Case 2: Axygen
Procedure:
Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.
Measure out the weight of tubes of gel.
Add 400~600 ul Buffer DE-A and then incubate the EP tubes in thermostat water bath about 75 degree centigrade within 10 mins.
After gel is all dissolved, add 1/3 volumn of Buffer DE-B into EP tubes. The solution will turn yellow, illustrating complete dissolved.
Pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s.
Add 500 ul Buffer W1 in a centrifuge at 9000Xg about 30s and pour out the solution.
Add 700 ul Buffer W2 in a centrifuge at 9000Xg about 30s and pour out the solution.
Centrifuge the empty columns at 9000xg about 1min.
Open the columns and air them about 10 mins.
Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.
Keep the DNA solution and often measure the concentration of DNA and A260/A280 for further study.
Overlapping Fragments
We conduct the overlapping again.
However, the sample has a little difference. The concentration of micF changes to 58.7ng/uL. According to our sample extracted from PCR and calculation, ingradients characterize as below:
Name
Concentration(ng/uL)
Length(bp)
micF
57.8
358
SoxS
27.5
705
C0061(LuxI)
18.5
656
B0015(Terminator)
9.5
143
Total
1161
The theoratical volumn proportion is:
SoxS-C0061-B0015: 5: 7: 3
micF-C0061-B0015: 1.3: 7: 3
Firstly, to ligate these parts, we need 10 cycles without primer:
for circuit SoxS-C0061-B0015:
Name
Volumn(uL)
SoxS
5
C0061
7
B0015
3
Taq Polymerase
0.25
MgCl2
2
dNTP
0.4
10X buffer with KCl
2
ddH2O
0.35
for circuit micF-C0061-B0015:
Name
Volumn(uL)
micF
1.3
C0061
7
B0015
3
Taq Polymerase
0.25
MgCl2
2
dNTP
0.4
10X buffer with KCl
2
ddH2O
4.1
PCR cycle including:
Preheat: 94 degree celcius, 5min
10 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)
Elongation for whole at 72 degree celcius 1min
4 celcius degree for preserveation.
Primers are the same as before.
Name
Volumn(uL)
Template
5
Taq Polymerase
0.25
MgCl2
2
dNTP
0.4
micF-F(BB-SoxS-F)
1
B0015-R
1
10X buffer with KCl
2
ddH2O
8.35
15ul-protocol:
Name
Volumn(uL)
Solution
15
micF-F(BB-SoxS-F)
1
B0015-R
1
10X buffer with KCl
2
ddH2O
1
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)
Elongation for whole at 72 degree celcius 1min
4 celcius degree for preserveation.
PCR of Cas9(80l)
material:
8 uL buffer with KCl
35 uL dd water
8 uL Plasmid with Cas9
4 uL primer R
4 uL primer F
8 uL MgCl2
8 uL dNTPs
5 uL DMSO
1 uL Tap polymerase
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 4.5 min)
Elongation for whole at 72 degree celcius 1min
4 celcius degree for preserveation.
2015 29/7
Negative Chemotaxis Characterization
Done by Wang Luyao
Term 1 results
I put the paper on this plate after the E.coli grows homogeneously， so let's see the result later.
the controlled plate
the last plate was contaminated
Term 2
material:
LB solid medium plates x6
0.8、0.6、0.4、0.2、0.1g/ml Chloramphenicol
E.coli Top10
filter paper
Enzyme Digestion
Performing twice separately
Both as following protocol:
Name
Volumn(uL)
pSB1C3
10
XbaI
1
SpeI
1
green buffer
2
ddH2O
6
Overlap PCR of bacteriaI
material:
micF PCR product :33.6ng/uL
C0061 PCR product: 18.5ng/uL
B0015 PCR product:9.2ng/uL
micF-F primer
B0015-R primer
protocol:
total volume: 25uL
Taq Master Mix 12.5uL
micF 2uL
C0061 6uL
B0015 2.6uL
ddH2O 1.9uL
PCR cycle including:
Preheat: 94 degree celcius, 5min
10 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 90s)
Elongation for whole at 72 degree celcius 10min
4 celcius degree for preserveation.
After the 10 cycles,we divide the 20 ul system into two parts:5ul and 15ul.The 5ul PCR product will be used as template in another new 20ul system.On the other hand, add primers
to the remaining 15 uL PCR product.
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 90s)
Elongation for whole at 72 degree celcius 10min
4 celcius degree for preserveation.
Results:
We don't get the product we want.We make some research and guess that the probable reasons are the following:Taq polymerase can cause 3'd A to PCR product;the annealing temperature is unsuitable;the length of overlap fragment isn't enough .
2015 30/7
Negative Chemotaxis Characterization
Done by Wang Luyao
7.30
Term 2 results
The 0.8、0.6、0.4 plates just look similar, the denser Chloramphenicol the paper hold， the bigger the clear zone is.
but the 0.2 and 0.1g/ml plates are different, Top10 grows denser on the edge of the clear zone, I supposed it can show that E.coli has moved. So next time I'll try under 0.2g/ml.
The controlled plate
PCR of C0062 and R0010
material:
10 ul Taq PCR Master Mix
7 ul ddH2O
1 ul Primer F
1 ul Primer R
1 ul C0062 or R0010
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 4.5 min)
Elongation for whole at 72 degree celcius 1min
4 celcius degree for preserveation.
2015 31/7
Negative Chemotaxis Characterization
Done by Wang Luyao
Term 3, material:
LB solid medium plates x5
0.2、0.1、0.05、0.025g/ml Chloramphenicol
E.coli Top10
filter paper
PCR of micF, SoxS, C0061 and R0015
material(volume 50uL):
25 ul Taq PCR Master Mix
15 ul ddH2O
2.5 ul Primer F
2.5 ul Primer R
2.5 ul Top10 culture, Top10 culture, C0061 or R0015
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 4.5 min)
Elongation for whole at 72 degree celcius 1min
4 celcius degree for preserveation.
PCR of tRNA
Primers
Sequence(5' to 3')
tRNA-F
GAATTCGCGGCCGCTTCTAGATAATACGACTCACTATAGGGAATACAAGCTACTTGTTCTTTTTGCAATGGACGAGTTCGAAATGATT
tRNA-R
CTGCAGCGGCCGCTACTAGTTTACAGACGTTTGCGAATTG
tRNA-F2
TTTAATCCCCGTCCACTGGGTAATACGACTCACTATAGGGAATAC
Protocol (volume 20uL):
2 ul Taq Buffer
0.4 ul dNTP
1 ul tRNA-F
1 ul tRNA-R
0.25 ul Taq DNA Polymerase
2 ul MgCl2
1 ul tRNA
12.35 ul ddH2O
Protocol (volume 20uL):
2 ul Taq Buffer
0.4 ul dNTP
1 ul tRNA-F2
1 ul tRNA-R
0.25 ul Taq DNA Polymerase
2 ul MgCl2
1 ul tRNA
12.35 ul ddH2O
PCR cycle including:
Preheat: 94 degree celcius, 7min
25 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s ) and elongation(72 degree celcius 1.5 min)
Elongation for whole at 72 degree celcius 10min
4 celcius degree for preserveation.
Result
Only one sample get PCR product, concentration 441ng/μL. tRNA-F2 dosen't work for wrong template.
Remember to review gene map before experiments!
PCR of cheZ
material:
Top 10 Bacterial Solution
Primers produced from Sangon Biotech Co.,Ltd.
Primers
Sequence(5' to 3')
cheZ-F
GTGCAATTGATCCAGGAGCTCAGCCAGGC
cheZ-T7-R
GCTCCTGGATCAATTGCACATAAATTTTCTCCTCTTTTGCAAAAAGAA
for cheZ (~600bp)
Protocol as this:
2.5 μl Taq buffer with KCl
2.5 μl dNTPs
0.75 μl cheZ-F Primer
0.75 μl cheZ-T7-R Primer
0.5 μl Kod DNA Polymerase(600bp/min)
1.0 μl MgCl2
2 uL Top10 Solution
15 μl ddH2O
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1 min 30s)
Elongation for whole at 72 degree celcius 2min(C0061)or 1min(B0015 and R0062)
4 celcius degree for preserveation.
It turned out we used wrong primer:(
PCR of CheZ and B0015
material:
TOP 10 baterial solution
PAO1 bacterial solution
B0015 plasmid :93.0 ng/uL
protocol:
total system 25uL
buffer 2.5 uL
dNTPs 2.5uL
MgSO4 2uL
primer-F 0.75 uL
primer-R 0.75 uL
bacterial solution/plasmid 1uL
KOD-Plus enzyme 0.5uL
ddH2O 17uL
PCR cycle including:
Preheat: 94 degree celcius, 2min
35 cycles containing: degradation(94 degree celcius, 15s; annealing(55 degree celcius, 30s and elongation(68 degree celcius 50s)
Elongation for whole at 68 degree celcius 7min.
4 celcius degree for preserveation.
The protocol is from lab319,the enzyme is KOD-Plus.
Result:
We don't get the product we want .It turned out because of wrong primer we implemented.
Transformation of C0062
the protocol as following
Take the competent bacteria from refrigerator and incubate them into ice about 10 mins until it is dissolved
Absorb 1 uL or 2 uL plasmid and mix it with bacteria solution thoroughly.
Put the tubes on the ice about 30 mins.
Make a heat shock at 42 degree centigrade about 60 sec
Put the tubes on the ice about 3 mins again.
Add 900 ul LB medium into EP tubes and cultivate the bacteria at 37 degree centrigrade about 60 min.
Centrifuge them at 4,000xg about 15 sec and we will se sediment in the tubes.
Discard the supernatant liquid and leave about 100ul medium.
Coat plate: add 100ul solution in a plate
2015 1/8
Negative Chemotaxis Characterzation
Done by Wang Luyao
Term 3 results
this time I got the similar results, but I cannot tell whether the bacteria is moved or didn't grow up.
so let's see the term 1 result:
I put the paper on this plate after the E.coli grows homogeneously. Now after I took it out I got this.
It's obviously that the bacteria moved. If E.coli died, the medium cannot be clear, the bodies will stay. In the middle of the area, there was a bubble formed by the medium and the wet paper. There is dilute Chloramphenicol, so the some bacteria moved there.
So we qualitatively proved that E.coli negative chemotaxis !!!
PCR of cheZ
Material:
Top 10 Bacterial Solution
Primers produced from Sangon Biotech Co.,Ltd.
Primers
Sequence(5' to 3')
cheZ-F
GTGCAATTGATCCAGGAGCTCAGCCAGGC
che-Ter-R
TTTATTTGATGCCTGGCTCTAGTATCAGAAACCCAGGCTGGACA
for cheZ (~600bp)
Protocol as this:
5 μl Taq buffer with KCl
1 μl dNTPs
2.5 μl cheZ-F Primer
2.5 μl che-Ter-R Primer
0.65 μl Pfu DNA Polymerase(600bp/min)
5 μl MgCl2
2.5 uL Top10 Solution
31 μl ddH2O
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1 min 30s)
1.Elongation for whole at 72 degree celcius 10 mins
4 celcius degree for preserveation.
No results: May be because of Pfu enzyme has already been out of date.
Protocol again:
Ingredients
Volumn(uL)
2X Taq Master Mix
25
cheZ-F
2.5
che-Ter-R
2.5
Top10 Solution
5
ddH2O
15
Ingredients
Volumn(uL)
2X Taq Master Mix
25
cheZ-F
2.5
che-Ter-R
2.5
Top10 Solution
5
ddH2O
12.5
DMSO
2.5
No results：it turns out cheZ sequence in TOP10 is different from PAO1, we designed after PAO1
Protocol again:
Ingredients
Volumn(uL)
2X Taq Master Mix
25
cheZ-F
2.5
che-Ter-R
2.5
PAO1 Solution
5
ddH2O
15
Ingredients
Volumn(uL)
Taq buffer with KCl
5
dNTPs
1
che-Ter-R
2.5
PAO1 Solution
5
ddH2O
28.35
MgCl2
5
Taq Polymerase
0.65
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1 min)
Elongation for whole at 72 degree celcius 10 mins
4 celcius degree for preserveation.
PCR of tRNA
Protocol (volume 20uL):
2 ul Taq Buffer
0.4 ul dNTP
1 ul tRNA-F2
1 ul tRNA-R
0.25 ul Taq DNA Polymerase
2 ul MgCl2
1 ul tRNA
12.35 ul ddH2O
PCR cycle including:
Preheat: 94 degree celcius, 7min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s ) and elongation(72 degree celcius 1.5 min)
Elongation for whole at 72 degree celcius 10min
4 celcius degree for preserveation.
Gel Extraction of tRNA
Preparation:
Check out whether ethyl alcohol is added into Wash Solution.
Check out whether Buffer B2 has sediment.
Adjust the thermostat water bath at 50 degree centigrade.
Procedure:
Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.
Add 1.5mL Buffer B2 and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.
After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s.
Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.
Do the step 5 again.
Centrifuge the empty columns at 9000xg about 1min.
Open the columns and air them about 10 mins.
Put the columns into clean 1.5 mL EP tubes and add 20uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 10 min and then centrifuge at 9000xg about 1 min.
Keep the DNA solution and often measure the concentration of DNA and A260/A280 for further study.
Result
Concentration 64ng/ul.
PCR of CheZ
material:
Top 10 Bacterial Solution
protocol:
total system 25uL
buffer 2.5 uL
dNTPs 2.5uL
MgSO4 3uL
primer-F 0.75 uL
primer-R 0.75 uL
bacterial solution/plasmid 1uL
KOD-Plus-Neo enzyme 0.5uL
ddH2O 16uL
PCR cycle including:
Preheat: 94 degree celcius, 2min
35 cycles containing: degradation(98 degree celcius, 10s; annealing(55 degree celcius, 30s and elongation(68 degree celcius 30s)
Elongation for whole at 68 degree celcius 7min.
4 celcius degree for preserveation.
The protocol is from lab 319.
PCR of T7 promoter
Primers
Sequence(5' to 3')
T7-F
GAATTCGCGGCCGCTTCTAG
T7-R2
agacatgcaatttttttcattccgattttctcctcttttgcaaaaagaacaagtagct
T7 Promoter1
GAATTCGCGGCCGCTTCTAGAtaatacgactcactatagggaatacaagctacttgttctttttgca
T7 Promoter2
tgcaaaaagaacaagtagcttgtattccctatagtgagtcgtattaTCTAGAAGCGGCCGCGAATTC
Protocol (volume 20uL):
10 ul Mix
1 ul T7-F
1 ul T7-R2
1 ul T7 Promoter1
1 ul T7 promoter2
6 ul ddH2O
PCR cycle including:
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s ) and elongation(72 degree celcius 30s)
Elongation for whole at 72 degree celcius 7min.
4 celcius degree for preserveation.
PCR Purification of T7
Add Buffer B3 in 5 times volumn of PCR product.
Pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s.
Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.
Do the step 5 again.
Centrifuge the empty columns at 9000xg about 1min.
Open the columns and air them about 10 mins.
Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.
Keep the DNA solution and measure the concentration of DNA and A260/A280 for further study.
Result
Concentration 28ng/ul.
PCR of T7 promoter
Conduct the PCR again.
Protocol (volume 20uL):
10 ul Mix
1 ul T7-F
1 ul T7-R2
1 ul T7 Promoter1
1 ul T7 promoter2
6 ul ddH2O
PCR cycle including:
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s ) and elongation(72 degree celcius 30s)
Elongation for whole at 72 degree celcius 7min.
4 celcius degree for preserveation.
Seamless Clone of tRNA Plasmid
Protocol:
8.5 ul Seamless cloning Master Mix
3 ul linear PSB1C3 plasmid
2 ul tRNA
6.5 ul ddH2O
Incubate at 50℃ for 60min.
Seamless Clone of Anchor Plasmid
Protocol:
8.5 ul Seamless cloning Master Mix
3 ul linear PSB1C3 plasmid
1 ul T7 promoter
1 ul COMPX
6.5 ul ddH2O
Incubate at 50℃ for 60min.
Transformation of tRNA Plasmid & Anchor Plasmid
Procedure:
Add 4ul clone product to competent cells, mix well, incubate on ice for 30min.
Heat in 42℃ bath for 90s. Then put on ice for 5min.
Add 500ul SOC culture, incubate at 37℃ for 45min.
4000rpm centrifuge for 3min, discard 500ul supernate.
Spread on plate with chloramphenicol, incubate overnight at 37℃.
2015 2/8
Plasmid Extraction of C0061
Procedure including:
Absorb 6 mL bacteria solution in to EP tubes, centrifuge them at 12,000xg 1mins and then discard the culture medium.
Add 250 ul Buffer S1 into sediment, and use spearhead to make bacteria suspended.
Add 250 ul Buffer S2, and overturn the EP tubes 8 times immediately and tenderly. Stand tubes about 3 mins.
Add 350 ul Buffer W3 and overturn the EP tubes 10 times again immediately and tenderly.
Centrifuge tubes at 12,000xg about 10 mins.
Pour the supernatant liquid into absorption columns ,centrifuge them at 12,000xg about 30 sec and then discard the liquid in the collecting pipe.
Add 500 ul Buffer W1 and centrifuge them at 12,000xg about 30 sec. Then discard the liquid in the collecting pipe.
Add 700 ul Buffer W2, centrifuge them at 12,000xg about 30 sec and discard the liquid in the collecting pipe.
Do the step 8 again.
Centrifuge the empty columns at 12,000xg about 1 min.
Put the columns in clean 1.5mL EP tubes, add 80 ul Elution Buffer on the absorption film, stand 10 min and centrifuge about 1 min at 12,000xg.
PCR of cheZ and OprF from PAO1
Yesterday, we may extract cheZ from PAO 1, though with a lot of primer dimer. Today, DMSO will be implemented trying to eliminate these fragements.
Ingredients
Volumn(uL)
Taq buffer with KCl
5
dNTPs
1
cheZ-F
2.5
cheZ-Ter-R
2.5
PAO1 Solution
5
ddH2O
25.85
MgCl2
5
Pfu Polymerase
0.65
DMSO
2.5
Ingredients
Volumn(uL)
Taq buffer with KCl
5
dNTPs
1
Opr-F
2.5
OprF-Ter-R
2.5
PAO1 Solution
5
ddH2O
25.85
MgCl2
5
Pfu Polymerase
0.65
DMSO
2.5
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 2 min 30 s)
Elongation for whole at 72 degree celcius 10 mins
4 celcius degree for preserveation.
PCR Purification of T7
Add Buffer B3 in 5 times volumn of PCR product.
Pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s.
Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.
Do the step 5 again.
Centrifuge the empty columns at 9000xg about 1min.
Open the columns and air them about 10 mins.
Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.
Keep the DNA solution and often measure the concentration of DNA and A260/A280 for further study.
Gel Extraction
Material including: OprF, cheZ
Preparation:
Check out whether ethyl alcohol is added into Wash Solution.
Check out whether Buffer B2 has sediment.
Adjust the thermostat water bath at 50 degree centigrade.
Procedure:
Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.
Measure out the weight of tubes of gel.
Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.
Optional: If the length of the DNA strip is shorter than 500 bp, it would be better add isopropanol one third volume of Buffer B2.
After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s.
Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.
Do the step 5 again.
Centrifuge the empty columns at 9000xg about 1min.
Open the columns and air them about 10 mins.
Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.
Keep the DNA solution and often measure the concentration of DNA and A260/A280 for further study.
Colonial PCR of PSB1C3-T7-COMPX & PSB1C3-T7-tRNA
Protocol(volume 20ul)
1μl T7-F Primer
1μl COMPX-R or tRNA-R Primer
10μl Mix
1μl Bacteria Clone
7μl ddH2O
PCR cycle including:
Preheat: 94 degree celcius, 7min
35 cycles containing: degradation(94 degree celcius, 30s), annealing(55 degree celcius, 30s) and elongation(72 degree celcius 1.5 min)
Elongation for whole at 72 degree celcius 10min.
4 celcius degree for preserveation.
No Result.
Colonial PCR of PSB1C3-T7-COMPX & PSB1C3-T7-tRNA
Conduct the PCR again.
Protocol(volume 20ul)
1μl T7-F Primer
1μl COMPX-R or tRNA-R Primer
10μl Mix
1μl Bacteria Clone
7μl ddH2O
PCR cycle including:
Preheat: 94 degree celcius, 7min
35 cycles containing: degradation(94 degree celcius, 30s), annealing(55 degree celcius, 30s) and elongation(72 degree celcius 1.5 min)
Elongation for whole at 72 degree celcius 10min.
4 celcius degree for preserveation.
No Result.
PCR Purification of tRNA & COMPX
Add Buffer B3 in 5 times volumn of PCR product.
Pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s.
Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.
Do the step 5 again.
Centrifuge the empty columns at 9000xg about 1min.
Open the columns and air them about 10 mins.
Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.
Keep the DNA solution and measure the concentration of DNA and A260/A280 for further study.
Result
tRNA 26ng/ul, COMPX 24ng/ul.
Transformation of tRNA Plasmid & Anchor Plasmid
Procedure:
Add 4ul clone product to competent cells, mix well, incubate on ice for 30min.
Heat in 42℃ bath for 90s. Then put on ice for 5min.
Add 500ul SOC culture, incubate at 37℃ for 45min.
4000rpm centrifuge for 3min, discard 500ul supernate.
Spread half of the bacteria culture on plate with chloramphenicol, incubate overnight at 37℃.
2015 3/8
Colonial PCR of PSB1C3-T7-COMPX & PSB1C3-T7-tRNA
Protocol(volume 20ul)
1μl T7-F Primer
1μl COMPX-R or tRNA-R Primer
10μl Mix
1μl Bacteria Clone
7μl ddH2O
PCR cycle including:
Preheat: 94 degree celcius, 7min
35 cycles containing: degradation(94 degree celcius, 30s), annealing(55 degree celcius, 30s) and elongation(72 degree celcius 1.5 min)
Elongation for whole at 72 degree celcius 10min.
4 celcius degree for preserveation.
PCR of OprF from PAO1
Yesterday, we may extract cheZ from PAO 1, though with a lot of primer dimer. Today, DMSO will be implemented trying to eliminate these fragements.
Ingredients
Volumn(uL)
Taq buffer with KCl
5
dNTPs
1
Opr-F
2.5
OprF-Ter-R
2.5
PAO1 Solution
5
ddH2O
28.35
MgCl2
5
Pfu Polymerase
0.65
DMSO
2.5
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 2 min)
Elongation for whole at 72 degree celcius 10 mins
4 celcius degree for preserveation.
PCR of C0061
material(volume 100uL):
6 ul MgCl2
71 ul ddH2O
4 ul Primer F
4 ul Primer R
2 ul dNTP
1 ul Taq DNA polymerase
10 ul 10*PCR Buffer
2 ul C0061
PCR cycle including:
1 Preheat: 94 degree celcius, 5min
2 35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 4.5 min)
3 Elongation for whole at 72 degree celcius 1min
4 4 celcius degree for preserveation.
Gel Extraction
Material including: R0061
Preparation:
• Check out whether ethyl alcohol is added into Wash Solution.
• Check out whether Buffer B2 has sediment.
• Adjust the thermostat water bath at 50 degree centigrade.
Procedure:
5 Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.
6 Measure out the weight of tubes of gel.
7 Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.
8 Optional: If the length of the DNA strip is shorter than 500 bp, it would be better add isopropanol one third volume of Buffer B2.
9 After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s.
10 Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.
11 Do the step 5 again.
12 Centrifuge the empty columns at 9000xg about 1min.
13 Open the columns and air them about 10 mins.
14 Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.
PCR for gRNA-AcrB and gRNA-EmrE
material:
gRNA-AcrB plasmid(280.7 ng/uL)
gRNA-EmrE plasmid(222.4 ng/uL)
PCR protocol:
For gRNA-AcrB and gRNA-EmrE, primers including gRNA-F, gRNA-R, protocol as following:
10μl Taq Master Mix
1μl gRNA-F Primer
1μl gRNA-R Primer
0.25μl Taq DNA Polymerase
6.75μl ddH2O
1μl gRNA-AcrB(or gRNA-EmrE) Plasmid
For gRNA, Primers produced from Sangon Biotech Co.,Ltd.
Primer
Sequence(5'-3')
gRNA-F
GCGTCTAGATTGACAGCTAGCTCAGTAGGTATAAT
gRNA-R
TAGTAGGTCGTATTAAAAAAAAGCACCGAC
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)
Elongation for whole at 72 degree celcius 7 min
4 celcius degree for preserveation.
Gel Electrophoresis Characterization
gRNA-AcrB and gRNA-EmrE
Gel Extraction of gRNA-AcrB and gRNA-EmrE
Material including: gRNA-AcrB and gRNA-EmrE
Preparation:
Check out whether ethyl alcohol is added into Wash Solution.
Check out whether Buffer B2 has sediment.
Adjust the thermostat water bath at 50 degree centigrade.
Procedure:
Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.
Measure out the weight of tubes of gel.
Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.
Optional: If the length of the DNA strip is shorter than 500 bp, it would be better add isopropanol one third volume of Buffer B2.
After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s.
Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.
Do the step 5 again.
Centrifuge the empty columns at 9000xg about 1min.
Open the columns and air them about 10 mins.
Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.
Keep the DNA solution and often measure the concentration of DNA and A260/A280 for further study.
the concentration as following:
gRNA-AcrB: 44 ng/uL
gRNA-EmrE: 41 ng/uL
PCR for T7-tet Promoter
PCR protocol:
For T7-tet promoter, primers including T7-tet-F, T7-tet-R, protocol as following:
10μl Taq Master Mix
1μl T7-tet-F Primer
1μl T7-tet-R Primer
1μl template-F Primer
1μl template-R Primer
0.25μl Taq DNA Polymerase
5.75μl ddH2O
Use template-F Primer and template-R Primer as templates
For T7-tet, Primers produced from Sangon Biotech Co.,Ltd.
Primer
Sequence(5'-3')
template-F
taatacgactcactatgatagaaaagaggagaaaatc
template-R
gattttctcctcttttctatcatagtgagtcgtatta
T7-tet-F
Ttaatacgactcactatgatagaaaagaggag
T7-tet-R
gtatttcttatccattccgattttctcctcttttctat
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)
Elongation for whole at 72 degree celcius 7 min
4 celcius degree for preserveation.
Gel Electrophoresis Characterization
T7-tet Promoter
Gel Extraction of T7-tet Promoter
Material including: T7-tet Promoter
Preparation:
Check out whether ethyl alcohol is added into Wash Solution.
Check out whether Buffer B2 has sediment.
Adjust the thermostat water bath at 50 degree centigrade.
Procedure:
Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.
Measure out the weight of tubes of gel.
Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.
Optional: If the length of the DNA strip is shorter than 500 bp, it would be better add isopropanol one third volume of Buffer B2.
After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s.
Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.
Do the step 5 again.
Centrifuge the empty columns at 9000xg about 1min.
Open the columns and air them about 10 mins.
Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.
Keep the DNA solution and often measure the concentration of DNA and A260/A280 for further study.
the concentration as following:
T7-tet: 30 ng/uL
Seamless Clone of cheZ Plasmid and Circuit Containing E0040
Protocol:
Fragements
Length(bp)
Mass Required(ug)
linear PSB1C3 plasmid
2070
41.4
T7 Promoter
46
9.2
cheZ
683
13.66
B0015
143
2.86
Ingredients
Volumn
Seamless cloning Master Mix
8.5 ul
linear PSB1C3 plasmid
3 ul
T7 Promoter
1ul(+3 folds ddH2O)
cheZ
1ul(+1 fold ddH2O)
B0015
1ul(+9 folds ddH2O)
ddH2O
5.5ul
Incubate at 50℃ for 60min.
Fragements
Length(bp)
Mass Required(ug)
linear PSB1C3 plasmid
2070
41.4
R0061
30
0.6
E0040
720
14.4
B0015
143
2.86
Ingredients
Volumn(uL)
Seamless cloning Master Mix
8.5 ul
linear PSB1C3 plasmid
3 ul
R0061
1ul(+20 fold ddH2O)
E0040
1ul(+1 fold ddH2O)
B0015
1ul(+9 folds ddH2O)
ddH2O
5.5ul
PCR for Cas9-PART I and Cas9-PART II
material:
Cas9 plasmid(511.3 ng/uL)
PCR protocol:
For Cas9-PART I and Cas9-PART II, primers including Cas9-1-F, Cas9-1-R,Cas9-2-F, Cas9-2-R, protocol as following:
10μl Taq Master Mix
1μl Cas9-1-F(or Cas9-2-F) Primer
1μl Cas9-1-R(or Cas9-2-R) Primer
0.25μl Taq DNA Polymerase
6.75μl ddH2O
1μl Cas9 Plasmid
For Cas9, Primers produced from Sangon Biotech Co.,Ltd.
Primer
Sequence(5'-3')
Cas9-1-F
ggaatggataagaaatactcaataggcttag
Cas9-1-R
gctgtttcatcaccttatcatcaaaga
Cas9-2-F
gataaggtgatgaaacagcttaaacgtc
Cas9-2-R
ctactagtgggaccattcaaaacagcatagc
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 4.5min)
Elongation for whole at 72 degree celcius 7 min
4 celcius degree for preserveation.
Gel Electrophoresis Characterization
Cas9-PART I and Cas9-PART II
Gel Extraction of Cas9-PART I and Cas9-PART II
Material including: Cas9-PART I and Cas9-PART II
Preparation:
Check out whether ethyl alcohol is added into Wash Solution.
Check out whether Buffer B2 has sediment.
Adjust the thermostat water bath at 50 degree centigrade.
Procedure:
Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.
Measure out the weight of tubes of gel.
Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.
Optional: If the length of the DNA strip is shorter than 500 bp, it would be better add isopropanol one third volume of Buffer B2.
After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s.
Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.
Do the step 5 again.
Centrifuge the empty columns at 9000xg about 1min.
Open the columns and air them about 10 mins.
Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.
Keep the DNA solution and often measure the concentration of DNA and A260/A280 for further study.
the concentration as following:
Cas9-PART I : 31 ng/uL
Cas9-PART II: 16 ng/uL
Seamless Clone of tRNA Plasmid
Protocol:
8.5 ul Seamless cloning Master Mix
3 ul linear PSB1C3 plasmid
1 ul tRNA
7.5 ul ddH2O
Incubate at 50℃ for 60min.
Seamless Clone of Anchor Plasmid
Protocol:
8.5 ul Seamless cloning Master Mix
3 ul linear PSB1C3 plasmid
1 ul T7 promoter
1 ul COMPX
6.5 ul ddH2O
Incubate at 50℃ for 60min.
Transformation of tRNA Plasmid & Anchor Plasmid
Procedure:
Add 4ul clone product to competent cells, 1mix well, incubate on ice for 30min.
Heat in 42℃ bath for 90s. Then put on ice for 5min.
Add 500ul SOC culture, incubate at 37℃ for 45min.
Spread on plate with chloramphenicol, incubate overnight at 37℃.
2015 4/8
PCR for micF and C0061
Protocol:
Ingradients
Volumn(uL)
10X PCR buffer
8
dNTPs
1.6
micF-F
4
micF-luxI-R
4
Top10 Solution
8
Pfu Polymerase
0.5
ddH2O
53.9
Ingradients
Volumn(uL)
10X PCR buffer
8
dNTPs
1.6
C0061-F
4
C0061-B0015-R
4
C0061
8
Pfu Polymerase
0.5
ddH2O
53.9
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min 30s)
Elongation for whole at 72 degree celcius 7 min
4 celcius degree for preserveation.
Seamless Clone of circuit I
material:
pSB1C3(XbaI&SpeI digest) 2062bp 7.1ng/uL
micF 402bp 58.7ng/uL
C0061 656bp 27.4ng/uL
B0015 142bp 7.1ng/uL
seamless clone kit
protocol:
pSB1C3 12.4uL
micF 0.5uL
C0061 1.73uL
B0015 1.46uL
seamless clone mix 8.5uL
Incubate at 50℃ for 60min.
Transformation of circuit I
Procedure:
Add 4ul clone product to competent cells, 1mix well, incubate on ice for 30min.
Heat in 42℃ bath for 90s. Then put on ice for 3min.
Add 500ul LB culture(no antibiotics), shaking at 37℃ for 30min.
Spread on plate with chloramphenicol, incubate overnight at 37℃.
2015 5/8
Seamless Clone of circuit I
material:
pSB1C3(XbaI&SpeI digest) 2062bp 7.1ng/uL
micF 402bp 58.7ng/uL
C0061 656bp 27.4ng/uL
B0015 142bp 7.1ng/uL
seamless clone kit
protocol:
pSB1C3 12.4uL
micF 0.5uL
C0061 1.73uL
B0015 1.46uL
seamless clone mix 8.5uL
Incubate at 50℃ for 60min.
Transformation of circuit I
Protocol:
Add 4ul clone product to competent cells, 1mix well, incubate on ice for 30min.
Heat in 42℃ bath for 90s. Then put on ice for 3min.
Add 500ul LB culture(no antibiotics), shaking at 37℃ for 30min.
Spread on plate with chloramphenicol, incubate overnight at 37℃.
Transformation of pSB1C3
protocol:
Add 2ul clone product to competent cells, 1mix well, incubate on ice for 30min.
Heat in 42℃ bath for 90s. Then put on ice for 3min.
Add 500ul LB culture(no antibiotics), shaking at 37℃ for 30min.
Spread on plate with chloramphenicol, incubate overnight at 37℃.
Double digest of pSB1C3
protocol:
pSB1C3 plasmid solution 10ul
XbaI 1ul
SpeI 1ul
Tango.buffer 2ul
ddH2O 6ul
incubate at 37℃ for 2 hours.
2015 6/8
Colonial PCR of PSB1C3-T7-COMPX & PSB1C3-T7-tRNA
Protocol(volume 20ul)
1μl T7-F Primer
1μl COMPX-R or tRNA-R Primer
10μl Mix
1μl Bacteria Clone
7μl ddH2O
PCR cycle including:
Preheat: 94 degree celcius, 7min
35 cycles containing: degradation(94 degree celcius, 30s), annealing(55 degree celcius, 30s) and elongation(72 degree celcius 1.5 min)
Elongation for whole at 72 degree celcius 10min.
4 celcius degree for preserveation.
Colonial PCR of PSB1C3-T7-cheZ & PSB1C3-R0051-E0040-B0015
Protocol(volume 13.5ul)
0.25μl Primer 1
0.25μl Primer 2
6.5μl Mix
1μl Bacteria Clone
5.5μl ddH2O
PCR cycle including:
Preheat: 94 degree celcius, 7min
35 cycles containing: degradation(94 degree celcius, 30s), annealing(55 degree celcius, 30s) and elongation(72 degree celcius 1 min)
Elongation for whole at 72 degree celcius 10min.
4 celcius degree for preserveation.
Overlap Circuit of gRNA-Cas9
Overlapping Fragments
In order to ligate the fragements gRNA-ArcB or gRNA-EmrE, T7-tet, Cas9-PART I and Cas9-PART II, we need to make the equal mole proportion.
The mole of DNA molecule is calculated in the following ways:
among which, c is the concentration of DNA, V is the volumn required and n is the length of DNA. Because of the same molar mass of As, Gs, Cs and Ts, computation of same molar mass is omitted.
According to our sample extracted from PCR and calculation, ingradients characterize as below:
Name
Concentration(ng/uL)
Length(bp)
gRNA-ArcB
44
162
gRNA-EmrE
41
162
T7-tet
30
56
Cas9-PART I
31
1954
Cas9-PART II
16
2358
Total
4530
The theoratical volumn proportion is:
gRNA-ArcB-T7-tet-Cas9-PART I-Cas9-PART II : 2: 1: 31: 64
gRNA-EmrE-T7-tet-Cas9-PART I-Cas9-PART II: 2: 1: 31: 64
Firstly, to ligate these parts, we need 10 cycles without primer:
for circuit gRNA-ArcB-T7-tet-Cas9-PART I-Cas9-PART II:
Name
Volumn(uL)
gRNA-ArcB
0.2
T7-tet
0.1
Cas9-PART I
3.1
Cas9-PART II
6.4
Taq Polymerase
0.25
dNTP(+MgCl2)
2
10X buffer
2
DMSO
1
ddH2O
4.95
for circuit gRNA-EmrE-T7-tet-Cas9-PART I-Cas9-PART II:
Name
Volumn(uL)
gRNA-EmrE
0.2
T7-tet
0.1
Cas9-PART I
3.1
Cas9-PART II
6.4
Taq Polymerase
0.25
dNTP(+MgCl2)
2
10X buffer
2
DMSO
1.25
ddH2O
4.7
PCR cycle including:
Preheat: 94 degree celcius, 5min
10 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 6.5min)
Elongation for whole at 72 degree celcius 1min
4 celcius degree for preserveation.
Then, we extracted 5 ul as template, and 15 ul adding with primer:
Primers were synthesized from Sangon Biotech. Co.,Ltd.
Primers
Sequence(5' to 3')
gRNA-F
GCGTCTAGATTGACAGCTAGCTCAGTAGGTATAAT
Cas9-2-R
CTACTAATGGGACCATTCAAAACAGCATAGC
5ul-template protocol:
Name
Volumn(uL)
Template
5
Taq Polymerase
0.25
dNTP(+MgCl2)
2
10X buffer
2
DMSO
1.25
ddH2O
7.5
gRNA-F
1
Cas9-2-R
1
15ul-protocol:
Name
Volumn(uL)
Solution
15
gRNA-F
1
Cas9-2-R
1
DMSO
1
10X buffer
2
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 6.5min)
Elongation for whole at 72 degree celcius 1min
4 celcius degree for preserveation.
2015 7/8
Plasmid Extraction of Circuit 1, B0015, T7
Procedure including:
1.Absorb 6 mL bacteria solution in to EP tubes, centrifuge them at 12,000xg 1mins and then discard the culture medium.
2.Add 250 ul Buffer S1 into sediment, and use spearhead to make bacteria suspended.
3.Add 250 ul Buffer S2, and overturn the EP tubes 8 times immediately and tenderly. Stand tubes about 3 mins.
4.Add 350 ul Buffer W3 and overturn the EP tubes 10 times again immediately and tenderly.
5.Centrifuge tubes at 12,000xg about 10 mins.
6.Pour the supernatant liquid into absorption columns ,centrifuge them at 12,000xg about 30 sec and then discard the liquid in the collecting pipe.
7.Add 500 ul Buffer W1 and centrifuge them at 12,000xg about 30 sec. Then discard the liquid in the collecting pipe.
8.Add 700 ul Buffer W2, centrifuge them at 12,000xg about 30 sec and discard the liquid in the collecting pipe.
9.Do the step 8 again.
10.Centrifuge the empty columns at 12,000xg about 1 min.
11.Put the columns in clean 1.5mL EP tubes, add 80 ul Elution Buffer on the absorption film, stand 10 min and centrifuge about 1 min at 12,000xg.
Enzyme Digestion
Protocol:
Name Volumn(uL)
pSB1C3 10
XbaI 1
SpeI 1
10* Buffer Tango 2
ddH2O 15
Colonial PCR of PSB1C3-T7-COMPX
Protocol(volume 20ul)
1μl T7-F Primer
1μl COMPX-R
10μl Mix
1μl Bacteria Clone
7μl ddH2O
PCR cycle including:
Preheat: 94 degree celcius, 7min
35 cycles containing: degradation(94 degree celcius, 30s), annealing(55 degree celcius, 30s) and elongation(72 degree celcius 1.5 min)
Elongation for whole at 72 degree celcius 10min.
4 celcius degree for preserveation.
Double Digest of T7 Plasmid
Protocol (Volume 20ul)
4ul Tango Buffer
2ul BamHI
2ul SalI
10ul T7 Plasmid
2ul ddH2O
Incubate at 37℃ for 3h.
Colonial PCR of PSB1C3-T7-COMPX
Conduct the PCR again.
Protocol(volume 20ul)
1μl T7-F Primer
1μl COMPX-R
10μl Mix
1μl Bacteria Clone
7μl ddH2O
PCR cycle including:
Preheat: 94 degree celcius, 7min
35 cycles containing: degradation(94 degree celcius, 30s), annealing(55 degree celcius, 30s) and elongation(72 degree celcius 1.5 min)
Elongation for whole at 72 degree celcius 10min.
4 celcius degree for preserveation.
Double Digest of T7 Plasmid
Protocol (Volume 20ul)
4ul Tango Buffer
2ul BamHI
2ul SalI
10ul T7 Plasmid
2ul ddH2O
Incubate at 37℃ overnight.
2015 8/8
Seamless Clone of circuit I
material:
pSB1C3(XbaI&SpeI digest) 2062bp 36.7ng/uL
micF 402bp 39ng/uL
SoxS 736bp 44.4ng/uL
C0061 656bp 27.4ng/uL
B0015 142bp 22ng/uL
seamless clone kit
protocol:
pSB1C3 1.12uL
micF 0.2uL or SoxS 0.33uL
C0061 0.48uL
B0015 0.14uL
seamless clone mix 8.5uL
Incubate at 50℃ for 60min.
Transformation of circuit I
Procedure:
Add 4ul clone product to competent cells, 1mix well, incubate on ice for 30min.
Heat in 42℃ bath for 90s. Then put on ice for 3min.
Add 500ul LB culture(no antibiotics), shaking at 37℃ for 30min.
Spread on plate with chloramphenicol, incubate overnight at 37℃.
Colony PCR of Circuit I
pick 16 single colonies and shaking culture for 5 hours.
protocol:
6.5 ul Taq Master Mix
5.5 ul ddH2O
0.25 ul C0061-F primer
0.25 ul C0061-B0015-R primer
0.5 ul bacterial solution
PCR cycle including:
Preheat: 95 degree celcius, 5min
30 cycles containing: degradation(95 degree celcius, 30s), annealing(55 degree celcius, 30s) and elongation(72 degree celcius 50 s)
Elongation for whole at 72 degree celcius 10min.
4 celcius degree for preserveation.
2015 10/8
PCR of C0062
material(volume 100uL):
6 ul MgCl2
71 ul ddH2O
4 ul Primer F
4 ul Primer R
2 ul dNTP
1 ul pfu DNA polymerase
10 ul 10*PCR Buffer
2 ul C0062
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 4.5 min)
Elongation for whole at 72 degree celcius 1min
4 celcius degree for preserveation.
2015 11/8
Seamless Clone of pSB1C3-R0010-C0062-B0015
Ingredients
Volumn
Seamless cloning Master Mix
8.5 ul
linear PSB1C3 plasmid
2 ul
R0010
0.4ul
C0062
0.6ul
B0015
2.4ul
ddH2O
6.1ul
Incubate at 50 degree celcius 1 h.
Seamless Clone of pSB1C3-T7-COMPX
Ingredients
Volumn
Seamless cloning Master Mix
8.5 ul
linear PSB1C3 plasmid
2ul
T7
2ul
COMPX
1ul
ddH2O
6.5ul
Ingredients
Volumn
Seamless cloning Master Mix
8.5 ul
linear PSB1C3 plasmid
2ul
T7
1ul
COMPX
1ul
ddH2O
14.5ul
Incubate at 50 degree celcius 1 h.
Transformation of pSB1C3-T7-COMPX
Procedure:
Add 4ul clone product to competent cells, 1mix well, incubate on ice for 30min.
Heat in 42℃ bath for 90s. Then put on ice for 5min.
Add 500ul SOC culture, incubate at 37℃ for 45min.
Spread on plate with chloramphenicol, incubate overnight at 37℃.
Enzyme Digestion of pSB1C3-T7-COMPX and pSB1C3-tRNA
Ingredients
Volumn
Tango Buffer
2ul
plasmid
10ul
Spe I
1ul
Xba I
1ul
ddH2O
6ul
Incubate at 37 degree celcius for 2 h.
2015 12/8
Transformation of pSB1C3-R0010-C0062-B0015
Procedure:
Add 4ul clone product to competent cells, 1mix well, incubate on ice for 30min.
Heat in 42℃ bath for 90s. Then put on ice for 5min.
Add 500ul SOC culture, incubate at 37℃ for 45min.
Spread on plate with chloramphenicol, incubate overnight at 37℃.
Transformation of pSB3A3
Procedure:
Add 4ul clone product to competent cells, 1mix well, incubate on ice for 30min.
Heat in 42℃ bath for 90s. Then put on ice for 5min.
Add 500ul SOC culture, incubate at 37℃ for 45min.
Spread on plate with Ampcilin, incubate overnight at 37℃.
Plasmid Extraction of T7
Procedure including:
Absorb 4.5 mL bacteria solution in to EP tubes, centrifuge them at 12,000xg 1mins and then discard the culture medium.
Add 250 ul Buffer S1 into sediment, and use spearhead to make bacteria suspended.
Add 250 ul Buffer S2, and overturn the EP tubes 8 times immediately and tenderly. Stand tubes about 3 mins.
Add 350 ul Buffer W3 and overturn the EP tubes 10 times again immediately and tenderly.
Centrifuge tubes at 12,000xg about 10 mins.
Pour the supernatant liquid into absorption columns ,centrifuge them at 12,000xg about 30 sec and then discard the liquid in the collecting pipe.
Add 500 ul Buffer W1 and centrifuge them at 12,000xg about 30 sec. Then discard the liquid in the collecting pipe.
Add 700 ul Buffer W2, centrifuge them at 12,000xg about 30 sec and discard the liquid in the collecting pipe.
Do the step 8 again.
Centrifuge the empty columns at 12,000xg about 1 min.
Put the columns in clean 1.5mL EP tubes, add 80 ul Elution Buffer on the absorption film, stand 10 min and centrifuge about 1 min at 12,000xg.
PCR of OprF from PAO1
Ingredients
Volumn(uL)
2XTaq Mix
25
OprF-F
2.5
OprF-Ter-R
2.5
PAO1
2.5
ddH2O
17.5
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 2 min)
Elongation for whole at 72 degree celcius 10 mins
4 celcius degree for preserveation.
2015 13/8
Plasmid Extraction of R0062, B0015, SCVE, T7
Procedure including:
1.Absorb 4.5 mL bacteria solution in to EP tubes, centrifuge them at 12,000xg 1mins and then discard the culture medium.
2.Add 250 ul Buffer S1 into sediment, and use spearhead to make bacteria suspended.
3.Add 250 ul Buffer S2, and overturn the EP tubes 8 times immediately and tenderly. Stand tubes about 3 mins.
4.Add 350 ul Buffer W3 and overturn the EP tubes 10 times again immediately and tenderly.
5.Centrifuge tubes at 12,000xg about 10 mins.
6.Pour the supernatant liquid into absorption columns ,centrifuge them at 12,000xg about 30 sec and then discard the liquid in the collecting pipe.
7.Add 500 ul Buffer W1 and centrifuge them at 12,000xg about 30 sec. Then discard the liquid in the collecting pipe.
8.Add 700 ul Buffer W2, centrifuge them at 12,000xg about 30 sec and discard the liquid in the collecting pipe.
9.Do the step 8 again.
10.Centrifuge the empty columns at 12,000xg about 1 min.
11.Put the columns in clean 1.5mL EP tubes, add 80 ul Elution Buffer on the absorption film, stand 10 min and centrifuge about 1 min at 12,000xg.
PCR of R0062, C0051, E0040, B0015
Ingredients
Volumn(uL)
PCR Buffer
5
Primer 1
2.5
Primer 2
2.5
Plasimid Solution
2.5
ddH2O
31.9
Pfu Polymerase
0.6
dNTP mix
5
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min 30 s)
Elongation for whole at 72 degree celcius 10 mins
4 celcius degree for preserveation.
Pre-annealing of R0051
Ingredients
Volumn(uL)
R0051-F1
10
R0051-R1
10
Incubate solution in 95 degree celcius 30 min and then used for PCR.
PCR of R0051, T7-OprF, T7-SCVE, SCVE, micF and soxS
Ingredients
Volumn(uL)
PCR Buffer
5
Primer 1
2.5
Primer 2
2.5
Plasimid Solution
2.5
ddH2O
31.9
Pfu Polymerase
0.6
dNTP mix
5
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 2min)
Elongation for whole at 72 degree celcius 10 mins
4 celcius degree for preserveation.
PCR of OprF, micF, SoxS
Ingredients
Volumn(uL)
PCR Buffer
5
Primer 1
2.5
Primer 2
2.5
Bacterial Solution
2.5
ddH2O
31.9
Pfu Polymerase
0.6
dNTP mix
5
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min 30 s)
Elongation for whole at 72 degree celcius 10 mins
4 celcius degree for preserveation.
Gel Extraction
Material including: R0061, E0040, B0015, T7-OprF
Preparation:
• Check out whether ethyl alcohol is added into Wash Solution.
• Check out whether Buffer B2 has sediment.
• Adjust the thermostat water bath at 50 degree centigrade.
Procedure:
5 Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.
6 Measure out the weight of tubes of gel.
7 Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.
8 Optional: If the length of the DNA strip is shorter than 500 bp, it would be better add isopropanol one third volume of Buffer B2.
9 After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s.
10 Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.
11 Do the step 5 again.
12 Centrifuge the empty columns at 9000xg about 1min.
13 Open the columns and air them about 10 mins.
14 Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.
PCR of OprF, SoxS, micF
Ingredients
Volumn(uL)
PCR Buffer
5
Primer 1
2.5
Primer 2
2.5
Bactieral Solution
2.5
ddH2O
31.9
Pfu Polymerase
0.6
dNTP mix
5
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(Temperature gradient: 48 degree celcius, 50 degree celcius, 53 degree celcius and 55 degree celcius 30s) and elongation(72 degree celcius 2min)
Elongation for whole at 72 degree celcius 10 mins
4 celcius degree for preserveation.
Gel Extraction
Material including: T7-SCVE, SCVE
Preparation:
• Check out whether ethyl alcohol is added into Wash Solution.
• Check out whether Buffer B2 has sediment.
• Adjust the thermostat water bath at 50 degree centigrade.
Procedure:
5 Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.
6 Measure out the weight of tubes of gel.
7 Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.
8 Optional: If the length of the DNA strip is shorter than 500 bp, it would be better add isopropanol one third volume of Buffer B2.
9 After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s.
10 Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.
11 Do the step 5 again.
12 Centrifuge the empty columns at 9000xg about 1min.
13 Open the columns and air them about 10 mins.
14 Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min
Enzyme Digestion of pSB1C3
incubate at 37 degree celcius about 2 h
Ingredients
Volumn(uL)
Enzyme 1
1
Enzyme 2
1
pSB1C3
4
Tango Buffer
4
ddH2O
10
strategy
Enzyme 1
Enzyme 2
Strategy 1
EcoRI
PstI
Strategy 2
EcoRI
SpeI
Strategy 1
XbaI
PstI
Strategy 2
XbaI
SpeI
2015 14/8
PCR of C0061, micF/SoxS-binding C0061
for C0061
Ingredients
Volumn(uL)
PCR Buffer
5
C0061-F
2.5
C0061-B0015-RI
2.5
Plasimid Solution
2.5
ddH2O
31.9
Pfu Polymerase
0.6
dNTP mix
5
for micF-C0061
Ingredients
Volumn(uL)
PCR Buffer
5
micF-R-BB
2.5
B0034-C0061-R
2.5
Plasimid Solution
2.5
ddH2O
31.9
Pfu Polymerase
0.6
dNTP mix
5
for SoxS-C0061
Ingredients
Volumn(uL)
PCR Buffer
5
SoxS-R-BB
2.5
B0034-C0061-R
2.5
Plasimid Solution
2.5
ddH2O
31.9
Pfu Polymerase
0.6
dNTP mix
5
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 2min)
Elongation for whole at 72 degree celcius 10 mins
4 celcius degree for preserveation.
Seamless ligation of pSB1C3-T7-SCVE-B0015
Ingradients
Volumn(uL)
pSB1C3
1.2
T7-SCVE
1
SCVE
1
B0015
1
Seamless Cloning mix
8.5
ddH2O
to 20
Incubate at 50 degree celcuis 1 h
2015 15/8
PCR of OprF, SoxS
Ingredients
Volumn(uL)
PCR Buffer
5
Primer 1
2.5
Primer 2
2.5
Bacterial Solution
2.5
ddH2O
31.9
Pfu Polymerase
0.6
dNTP mix
5
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min 30 s)
Elongation for whole at 72 degree celcius 10 mins
4 celcius degree for preserveation.
PCR of micF-C0061
Ingredients
Volumn(uL)
PCR Buffer
5
micF-F-BB
2.5
B0034-C0061-R
2.5
Bacterial Solution
2.5
ddH2O
31.9
Pfu Polymerase
0.6
dNTP mix
5
PCR cycle including:
Preheat: 94 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min 30 s)
Elongation for whole at 72 degree celcius 10 mins
4 celcius degree for preserveation.
Gel Extraction
Material including: micF, pSB1C3
Preparation:
• Check out whether ethyl alcohol is added into Wash Solution.
• Check out whether Buffer B2 has sediment.
• Adjust the thermostat water bath at 50 degree centigrade.
Procedure:
5 Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.
6 Measure out the weight of tubes of gel.
7 Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.
8 Optional: If the length of the DNA strip is shorter than 500 bp, it would be better add isopropanol one third volume of Buffer B2.
9 After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s.
10 Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.
11 Do the step 5 again.
12 Centrifuge the empty columns at 9000xg about 1min.
13 Open the columns and air them about 10 mins.
14 Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.
Ligation of pSB1C3-tRNA
incubate at 16 degree celcius more than 20 h
Ingredients
Volumn(uL)
T4 DNA ligation Buffer
2
T4 DNA ligase
1
pSB1C3 digested by EcoRI and PstI
1
tRNA
6
ddH2O
11
Seamless Cloning of pSB1C3-micF-C0061-B0015
Ingredients
Volumn
Seamless cloning Master Mix
8.5 ul
linear PSB1C3 plasmid
2 ul
micF
1ul
C0061
1ul
B0015
1ul
ddH2O
6.5ul
Incubate at 50℃ for 60min.
Seamless Cloning of pSB1C3-T7-COMPX
Ingredients
Volumn
Seamless cloning Master Mix
8.5 ul
linear PSB1C3 plasmid
2 ul
COMPX
1ul
ddH2O
8.5ul
Incubate at 50℃ for 60min.
Transformation of pSB1C3-micF-C0061-B0015 and pSB1C3-T7-COMPX
Procedure:
Add 4ul clone product to competent cells, 1mix well, incubate on ice for 30min.
Heat in 42℃ bath for 90s. Then put on ice for 5min.
Add 500ul SOC culture, incubate at 37℃ for 45min.
Spread on plate with chloramphenicol, incubate overnight at 37℃.
2015 17/8
Construction of cheZ
PCR of CheZ
CheZ:
Material(volume 50ul)
CheZ-BB-F:1ul
CheZ-BB-R:1ul
Primestar:25ul
ddH2O:21ul
PAO1:2ul
Divide the 50ul into 2 parts. Both are
25ul
PCR cycle including:
Preheat: 98 degree celcius, 5min
35 cycles containing: Degradation(94 degree celcius, 30s; Annealing(55 degree celcius, 30s and elongation(72 degree celcius 1.5 min).
Elongation for whole at 72 degree celcius 1min
4 celcius degree for preserveation.
Gel Electrophoresis Characterization
CheZ(PCR)
Gel Extraction
Material including: CheZ(PCR)
Preparation:
•Check out whether ethyl alcohol is added into Wash Solution.
•Check out whether Buffer B2 has sediment.
•Adjust the thermostat water bath at 50 degree centigrade.
Procedure:
Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.
Measure out the weight of tubes of gel.
Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.
Add isopropanol one third volume of Buffer B2.
After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s, then add the solutions into the absorption columns again and centrifuge columns again.
Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.
Do the step 6 again.
Centrifuge the empty columns at 9000xg about 1min.
Open the columns and air them about 10 mins.
Put the columns into clean 1.5 mL EP tubes and add 20uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.
Double digest of CheZ and PSB1C3
Detect the concentration of CheZ:16.9ng/ul
Volume:20ul
Material:
CheZ:15ul
EcoRI:1ul
PstI:2ul
Tango:2ul
37℃digest 2 hours.
Detect the concentration of PSB1C3:74.1ng/ul
Volume:20ul
Material:
PSB1C3:5ul
EcoR1:1ul
Pst1:2ul
Tango:2ul
ddH2O:10ul
37℃digest 2 hours.
Gel Electrophoresis Characterization
CheZ(ep)
PSB1C3(ep)
Gel Extraction
Material including: CheZ(ep)
Preparation:
•Check out whether ethyl alcohol is added into Wash Solution.
•Check out whether Buffer B2 has sediment.
•Adjust the thermostat water bath at 50 degree centigrade.
Procedure:
Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.
Measure out the weight of tubes of gel.
Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.
Add isopropanol one third volume of Buffer B2.
After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s, then add the solutions into the absorption columns again and centrifuge columns again.
Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.
Do the step 10 again.
Centrifuge the empty columns at 9000xg about 1min.
Open the columns and air them about 10 mins.
Put the columns into clean 1.5 mL EP tubes and add 20uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.
Ligation of PSB1C3 and CheZ
In order to ligate the CheZ and PSB1C3, we need to make the equal mole proportion.
The mole of DNA molecule is calculated in the following ways:
C1V1/n1=C2V2/n2=……=0.1~0.3
Among which, c is the concentration of DNA, V is the volumn required and n is the length of DNA. Because of the same molar mass of As, Gs, Cs and Ts, computation of same molar mass is omitted.
At first, we need the mass of PSB1C3 at 50ng. The concentrate of the plasmid we use is 8.4ng/ul while the one of the CheZ is 16.7ng/ul. So we choose to add 6ul into the reaction. Through the relation of the mole of DNA, we get the volume of the CheZ.
Material:
CheZ:4ul
PSB1C3:6ul
Solution I:10ul
16℃ligate over night, prepare for the Transformation of the next day.
Transformation of CheZ and PSB1C3 Plasmid
Procedure:
Add 4ul clone product to competent cells, 1mix well, incubate on ice for 30min.
Heat in 42℃ bath for 90s. Then put on ice for 5min.
Add 900ul LB culture, incubate at 37℃ for 60min. Centrifuge the culture at 4000×1min, give away 850ul culture.
Spread on plate with chloramphenicol, incubate overnight at 37℃.
Colonial PCR of CheZ
Take one bacterial colony into 10ul H2O for the PCR.
Protocol(volume 13ul)
1μl CheZ-BB-F
1μl CheZ-BB-R
10μl Mix
1μl Bacteria Clone
PCR cycle including:
Preheat: 98 degree celcius, 7min
35 cycles containing: degradation(98 degree celcius, 30s), annealing(55 degree celcius, 30s) and elongation(72 degree celcius 1.5 min)
Elongation for whole at 72 degree celcius 10min.
4 celcius degree for preserveation.
Gel Electrophoresis Characterization
Failed.
However, we incubate two of the bacteria into 50ul centrifuge tube at 37℃, we find it grow well.
Construction of T7-SCVE
PCR of T7 and SCVE
Preparation:
Add 10ul T7-F,10ul T7-R ,heating at 95 degree celcius for 30min and cool to 37 degree celcius.The mix will be used as template for the PCR.
Material(volume 50ul)
T7-exten-F:1ul
T7-exten-R:1ul
Primestar:25ul
ddH2O:18ul
T7 as template:5ul
Divide the 50ul into 2 parts. Both are
25ul
Material(volume 50ul)
SCVE-F:1ul
SCVE-R:1ul
Primestar:25ul
ddH2O:22ul
SCVE plasmid:1ul
Divide the 50ul into 2 parts. Both are
25ul
PCR cycle including:
Preheat: 98 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1 min)
Elongation for whole at 72 degree celcius 10min
4 celcius degree for preserveation.
Gel Electrophoresis Characterization
T7(PCR)
Gel Electrophoresis Characterization
T7(PCR)
SCVE(PCR)
Gel Extraction
Material including: T7,SCVE(PCR)
Preparation:
•Check out whether ethyl alcohol is added into Wash Solution.
•Check out whether Buffer B2 has sediment.
•Adjust the thermostat water bath at 50 degree centigrade.
Procedure:
Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.
Measure out the weight of tubes of gel.
Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.
Add isopropanol one third volume of Buffer B2.
After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s, then add the solutions into the absorption columns again and centrifuge columns again.
Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.
Do the step 10 again.
Centrifuge the empty columns at 9000xg about 1min.
Open the columns and air them about 10 mins.
Put the columns into clean 1.5 mL EP tubes and add 20uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.
Double digest of T7 and SCVE
Detect the concentration of T7:35ng/ul
Volume:20ul
Material:
T7:15ul
XbaI:1ul
HindIII:2ul
Tango:2ul
37℃digest 2 hours.
Detect the concentration of SCVE:7.8ng/ul
volume:20ul
Material:
SCVE:15ul
PstI:1ul
HindIII:1ul
bufferR:2ul
ddH2O:1ul
37℃digest 2 hours.
Detect the concentration of PSB1C3:160.2ng/ul
Volume:20ul
Material
PSB1C3:5ul
XbaI:1ul
Pst1:2ul
Tango:2ul
ddH2O:10ul
37℃digest 2 hours.
Gel Electrophoresis Characterization
T7(ep)
SCVE(ep)
PSB1C3(xp)
Gel Extraction
Material including: T7,SCVE(ep)
Preparation:
•Check out whether ethyl alcohol is added into Wash Solution.
•Check out whether Buffer B2 has sediment.
•Adjust the thermostat water bath at 50 degree centigrade.
Procedure:
Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.
Measure out the weight of tubes of gel.
Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.
Add isopropanol one third volume of Buffer B2.
After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s, then add the solutions into the absorption columns again and centrifuge columns again.
Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.
Do the step 10 again.
Centrifuge the empty columns at 9000xg about 1min.
Open the columns and air them about 10 mins.
Put the columns into clean 1.5 mL EP tubes and add 20uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.
Ligation of PSB1C3 ,T7 and SCVE
In order to ligate the T7,SCVE and PSB1C3, we need to make the equal mole proportion.
The mole of DNA molecule is calculated in the following ways:
C1V1/n1=C2V2/n2=……=0.1~0.3
Among which, c is the concentration of DNA, V is the volumn required and n is the length of DNA. Because of the same molar mass of As, Gs, Cs and Ts, computation of same molar mass is omitted.
Material:
T7:1ul
SCVE:2ul
PSB1C3:6ul
Solution I:10ul
16℃ligate over night, prepare for the Transformation of the next day.
Transformation of T7+SCVE Plasmid
Procedure:
Add 4ul clone product to competent cells, 1mix well, incubate on ice for 30min.
Heat in 42℃ bath for 90s. Then put on ice for 5min.
Add 900ul LB culture, incubate at 37℃ for 60min. Centrifuge the culture at 4000×1min, give away 850ul culture.
Spread on plate with chloramphenicol, incubate overnight at 37℃.
Colonial PCR of T7+SCVE
Take one bacterial colony into 10ul H2O for the PCR.
Protocol(volume 13ul)
• 1μl T7-exten-F
• 1μl T7-exten-R
• 10μl Mix
• 1μl Bacteria Clone
PCR cycle including:
Preheat: 98 degree celcius, 7min
35 cycles containing: degradation(98 degree celcius, 30s), annealing(55 degree celcius, 30s) and elongation(72 degree celcius 1 min)
Elongation for whole at 72 degree celcius 10min.
4 celcius degree for preserveation.
Gel Electrophoresis Characterization
T7+SCVE
T7-SCVE(PCR detection)
Then take one of the positive bacteria into 50ul centrifuge tube, incubate at 37℃.
Construction of T7-OprF
PCR of OprF
Material(volume 50ul)
OprF-F:1ul
OprF-R:1ul
Primestar:25ul
ddH2O:21ul
top10:2ul
Divide the 50ul into 2 parts. Both are
25ul
PCR cycle including:
Preheat: 98 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)
Elongation for whole at 72 degree celcius 10min
4 celcius degree for preserveation.
the protocol of pcr of T7 is the same as the above
Gel Electrophoresis Characterization
OprF(PCR)
Gel Extraction
Material including: OprF(PCR)
Preparation:
•Check out whether ethyl alcohol is added into Wash Solution.
•Check out whether Buffer B2 has sediment.
•Adjust the thermostat water bath at 50 degree centigrade.
Procedure:
Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.
Measure out the weight of tubes of gel.
Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.
Add isopropanol one third volume of Buffer B2.
After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s, then add the solutions into the absorption columns again and centrifuge columns again.
Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.
Do the step 10 again.
Centrifuge the empty columns at 9000xg about 1min.
Open the columns and air them about 10 mins.
Put the columns into clean 1.5 mL EP tubes and add 20uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.
Double digest of T7 and OprF
Detect the concentration of T7:23ng/ul,OprF:40ng/ul
Volume:20ul
Material:
T7:15ul
EcoRI:1ul
NdeI:1ul
bufferO:2ul
ddH2O:1ul
37℃digest 2 hours.
Material:
OprF:15ul
PstI:1ul
NdeI:1ul
bufferO:2ul
ddH2O:1ul
37℃digest 2 hours.
Detect the concentration of PSB1C3:160.2ng/ul
Volume:20ul
Material
PSB1C3:5ul
EcoR1:1ul
Pst1:2ul
Tango:2ul
ddH2O:10ul
37℃digest 2 hours.
Gel Electrophoresis Characterization
OprF(ep)
T7(ep)
Gel Extraction
Material including: OprF(ep)
Preparation:
•Check out whether ethyl alcohol is added into Wash Solution.
•Check out whether Buffer B2 has sediment.
•Adjust the thermostat water bath at 50 degree centigrade.
Procedure:
Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.
Measure out the weight of tubes of gel.
Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.
Add isopropanol one third volume of Buffer B2.
After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s, then add the solutions into the absorption columns again and centrifuge columns again.
Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.
Do the step 10 again.
Centrifuge the empty columns at 9000xg about 1min.
Open the columns and air them about 10 mins.
Put the columns into clean 1.5 mL EP tubes and add 20uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.
Ligation of T7,OprF and pSB1C3
In order to ligate the T7,OprF and PSB1C3, we need to make the equal mole proportion.
The mole of DNA molecule is calculated in the following ways:
C1V1/n1=C2V2/n2=……=0.1~0.3
Among which, c is the concentration of DNA, V is the volumn required and n is the length of DNA. Because of the same molar mass of As, Gs, Cs and Ts, computation of same molar mass is omitted.
Material:
T7:1ul
OprF:2.5ul
PSB1C3:6ul
Solution I:10ul
16℃ligate over night, prepare for the Transformation of the next day.
Transformation of T7+OprF Plasmid
Procedure:
Add 4ul clone product to competent cells, 1mix well, incubate on ice for 30min.
Heat in 42℃ bath for 90s. Then put on ice for 5min.
Add 900ul LB culture, incubate at 37℃ for 60min. Centrifuge the culture at 4000×1min, give away 850ul culture.
Spread on plate with chloramphenicol, incubate overnight at 37℃.
Colonial PCR of T7+OprF
Take one bacterial colony into 10ul H2O for the PCR.
Protocol(volume 13ul)
1μl T7-exten-F
1μl OprF-R
10μl Mix
1μl Bacteria Clone
PCR cycle including:
Preheat: 98 degree celcius, 7min
35 cycles containing: degradation(98 degree celcius, 30s), annealing(55 degree celcius, 30s) and elongation(72 degree celcius 1.5 min)
Elongation for whole at 72 degree celcius 10min.
4 celcius degree for preserveation.
Gel Electrophoresis Characterization
T7-OprF
Then take one of the positive bacteria into 50ul centrifuge tube, incubate at 37℃.
Construction of lac+cheZ
PCR of lac and cheZ
Material(volume 50ul)
EcoRI-lac-F:1ul
NdeI-lac-R:1ul
Primestar:25ul
ddH2O:22ul
template:1ul
Divide the 50ul into 2 parts. Both are
25ul
Material(volume 50ul)
NdeI-cheZ-F:1ul
PstI-his-cheZ-R:1ul
Primestar:25ul
ddH2O:22ul
template:1ul
Divide the 50ul into 2 parts. Both are
25ul
PCR cycle including:
Preheat: 98 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)
Elongation for whole at 72 degree celcius 10min
4 celcius degree for preserveation.
the protocol of pcr of T7 is the same as the above
Gel Electrophoresis Characterization
CheZ(PCR)
R0010(PCR)
Gel Extraction
Material including: lac and cheZ(PCR)
Preparation:
•Check out whether ethyl alcohol is added into Wash Solution.
•Check out whether Buffer B2 has sediment.
•Adjust the thermostat water bath at 50 degree centigrade.
Procedure:
Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.
Measure out the weight of tubes of gel.
Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.
Add isopropanol one third volume of Buffer B2.
After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s, then add the solutions into the absorption columns again and centrifuge columns again.
Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.
Do the step 10 again.
Centrifuge the empty columns at 9000xg about 1min.
Open the columns and air them about 10 mins.
Put the columns into clean 1.5 mL EP tubes and add 20uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.
Double digest of lac and cheZ
Detect the concentration of lac:45ng/ul cheZ:38ng/ul
Volume:20ul
Material:
R0010:15ul
EcoRI:1ul
NdeI:1ul
bufferO:2ul
ddH2O:1ul
37℃digest 2 hours.
Material:
cheZ:15ul
PstI:1ul
NdeI:1ul
bufferO:2ul
ddH2O:1ul
37℃digest 2 hours.
Gel Electrophoresis Characterization
R0010(np)
CheZ(xn)
Gel Extraction
Material including: lac and cheZ(ep)
Preparation:
•Check out whether ethyl alcohol is added into Wash Solution.
•Check out whether Buffer B2 has sediment.
•Adjust the thermostat water bath at 50 degree centigrade.
Procedure:
Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.
Measure out the weight of tubes of gel.
Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.
Add isopropanol one third volume of Buffer B2.
After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s, then add the solutions into the absorption columns again and centrifuge columns again.
Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.
Do the step 10 again.
Centrifuge the empty columns at 9000xg about 1min.
Open the columns and air them about 10 mins.
Put the columns into clean 1.5 mL EP tubes and add 20uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.
Ligation of lac ,cheZ and pSB1C3
In order to ligate the lac,cheZ and PSB1C3, we need to make the equal mole proportion.
The mole of DNA molecule is calculated in the following ways:
C1V1/n1=C2V2/n2=……=0.1~0.3
Among which, c is the concentration of DNA, V is the volumn required and n is the length of DNA. Because of the same molar mass of As, Gs, Cs and Ts, computation of same molar mass is omitted.
Material:
lac:0.5ul
cheZ:3.0ul
PSB1C3:6ul
Solution I:10ul
16℃ligate over night, prepare for the Transformation of the next day.
Transformation of lac+cheZ Plasmid
Procedure:
Add 4ul clone product to competent cells, 1mix well, incubate on ice for 30min.
Heat in 42℃ bath for 90s. Then put on ice for 5min.
Add 900ul LB culture, incubate at 37℃ for 60min. Centrifuge the culture at 4000×1min, give away 850ul culture.
Spread on plate with chloramphenicol, incubate overnight at 37℃.
Colonial PCR of lac+cheZ
Take one bacterial colony into 10ul H2O for the PCR.
Protocol(volume 13ul)
1μl EcoRI-lac-F
1μl NdeI-lac-R
10μl Mix
1μl Bacteria Clone
PCR cycle including:
Preheat: 98 degree celcius, 7min
35 cycles containing: degradation(98 degree celcius, 30s), annealing(55 degree celcius, 30s) and elongation(72 degree celcius 1.5 min)
Elongation for whole at 72 degree celcius 10min.
4 celcius degree for preserveation.
Gel Electrophoresis Characterization
lac+cheZ
Then take one of the positive bacteria into 50ul centrifuge tube, incubate at 37℃.
Construction of micF(SoxS)-GFP
PCR of micF(SoxS) and GFP
Material(volume 50ul)
micF(SoxS)-F-BB:1ul
micF(SoxS)-E0040-R:1ul
Primestar:25ul
ddH2O:22ul
template:1ul
Divide the 50ul into 2 parts. Both are
25ul
Material(volume 50ul)
E0040-F:1ul
E0040-R:1ul
Primestar:25ul
ddH2O:22ul
template:1ul
Divide the 50ul into 2 parts. Both are
25ul
PCR cycle including:
Preheat: 98 degree celcius, 5min
35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)
Elongation for whole at 72 degree celcius 10min
4 celcius degree for preserveation.
Gel Electrophoresis Characterization
micF and SoxS(PCR)
E0040(PCR)
Gel Extraction
Material including: micF(SoxS) and GFP(PCR)
Preparation:
•Check out whether ethyl alcohol is added into Wash Solution.
•Check out whether Buffer B2 has sediment.
•Adjust the thermostat water bath at 50 degree centigrade.
Procedure:
Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.
Measure out the weight of tubes of gel.
Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.
Add isopropanol one third volume of Buffer B2.
After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s, then add the solutions into the absorption columns again and centrifuge columns again.
Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.
Do the step 10 again.
Centrifuge the empty columns at 9000xg about 1min.
Open the columns and air them about 10 mins.
Put the columns into clean 1.5 mL EP tubes and add 20uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.
Double digest of micF(SoxS) and GFP
Detect the concentration of micF:100ng/ul
SoxS:80ng/ul GFP:65ng/ul
Volume:20ul
Material:
micF:15ul
XbaI:1ul
HindIII:1ul
Tango buffer:2ul
ddH2O:1ul
37℃digest 2 hours.
Material:
GFP:15ul
PstI:1ul
HindIII:1ul
bufferR:2ul
ddH2O:1ul
37℃digest 2 hours.
Gel Electrophoresis Characterization
micF and SoxS(xH)
GFP(PH)
Gel Extraction
Material including: micF(SoxS) and GFP(ep)
Preparation:
•Check out whether ethyl alcohol is added into Wash Solution.
•Check out whether Buffer B2 has sediment.
•Adjust the thermostat water bath at 50 degree centigrade.
Procedure:
Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.
Measure out the weight of tubes of gel.
Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.
Add isopropanol one third volume of Buffer B2.
After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s, then add the solutions into the absorption columns again and centrifuge columns again.
Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.
Do the step 10 again.
Centrifuge the empty columns at 9000xg about 1min.
Open the columns and air them about 10 mins.
Put the columns into clean 1.5 mL EP tubes and add 20uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.
Ligation of micF(SoxS), GFP and pSB1C3
In order to ligate the micF(SoxS), GFP and PSB1C3, we need to make the equal mole proportion.
The mole of DNA molecule is calculated in the following ways:
C1V1/n1=C2V2/n2=……=0.1~0.3
Among which, c is the concentration of DNA, V is the volumn required and n is the length of DNA. Because of the same molar mass of As, Gs, Cs and Ts, computation of same molar mass is omitted.
Material:
micF(SoxS):0.8ul
GFP:2.0ul
PSB1C3:6ul
Solution I:10ul
16℃ligate over night, prepare for the Transformation of the next day.
Transformation of micF(SoxS)+GFP Plasmid
Procedure:
Add 4ul clone product to competent cells, 1mix well, incubate on ice for 30min.
Heat in 42℃ bath for 90s. Then put on ice for 5min.
Add 900ul LB culture, incubate at 37℃ for 60min. Centrifuge the culture at 4000×1min, give away 850ul culture.
Spread on plate with chloramphenicol, incubate overnight at 37℃.
Then take two bacteria colony into 50ul centrifuge tube, incubate at 37℃.
Then take one of the positive bacteria into 50ul centrifuge tube, incubate at 37℃.
2015 10/9
Pre-experiment
Use 1ug/ml Chloromycetin activated the bacteria to test the film.
Film test
Film
Film type:low-pressure polyethylene.
Use the aerosol paint to process one of the surface of the film.
Coating the film with 400ul 20ug/ml PLL(Poly-L-Lysine) at the temperature of 4℃ for over 4 hours.
Bacteria solution
Bacteria type: BL21+OprF
Mix bacteria with PBS and make the OD~0.3.
Sample solution
1ug/ml Chloromycetin PBS solution.
Inoculation
Drop 200ul bacteria solution on the processed film for 100s
After 100s, wash the film with wash solution twice Immediately.
Record images
Put the film into the sample box filled with samplw solution.
Put the sample box in SPRING and push or pull the sample box at the recommended position, and rotate the sample box to get clear interference fringes.
From 0s to 240s, take 2~3 pictures in a row with the time interval of 20s.
Results
Fail to get the ideal interference fringes.
Cause reflectance is not good enough and have a small amount of diffuse reflection.
2015 11/9
Pre-experiment
Use 1ug/ml Chloromycetin activated the bacteria to test the film.
Film
Film type: Rubbers（Condom）
1,Use the aerosol paint to process one of the
surface of the film.
Fail.Because the film shrink too much.
2,Try silver mirror reaction.
Fail.Because the ammonia erosion is too severe.
Film
Film type:glass(cover clip——thickness~0.16mm width~2.5cm longth~3.7cm)
Use the aerosol paint to process one of the surface of the film.
Coating the film with 400ul 20ug/ml PLL(Poly-L-Lysine) at the temperature of 4℃ for over 4 hours.
Bacteria solution
Bacteria type: BL21+OprF
Mix bacteria with PBS and make the OD~0.3.
Sample solution
1ug/ml Chloromycetin PBS solution.
Inoculation
Drop 200ul bacteria solution on the processed film for 100s
After 100s, wash the film with wash solution twice Immediately.
Record images
Put the film into the sample box filled with samplw solution.
Put the sample box in SPRING and push or pull the sample box at the recommended position, and rotate the sample box to get clear interference fringes.
From 0s to 240s, take 2~3 pictures in a row with the time interval of 20s.
Results
Succeed in getting the excellent interference fringes.
Record data and process the image.
2015 12/9
Building Calibration
Film test
Sample solution
（1）5ug/ml Chloromycetin PBS solution.
（2）0.5ug/ml Chloromycetin PBS solution.
Bacteria solution
Bacteria type: BL21+OprF
Mix bacteria with PBS and make the OD~0.3.
Inoculation
Drop 200ul bacteria solution on the processed film for 100s
After 100s, wash the film with wash solution twice Immediately.
Record images
Put the film into the sample box filled with samplw solution.
Put the sample box in SPRING and push or pull the sample box at the recommended position, and rotate the sample box to get clear interference fringes.
From 0s to 240s, take 2~3 pictures in a row with the time interval of 20s.
Results
Succeed in getting the excellent interference fringes.
Record data and process the image.
Use the Matlab to process the image.
Get the fitting according to the modeling so get the calibration.
2015 13/9
Verifying Calibration
Film test
Sample solution
（1）1.8ug/ml Chloromycetin PBS solution.
（2）3.2ug/ml Chloromycetin PBS solution.
Bacteria solution
Bacteria type: BL21+OprF
Mix bacteria with PBS and make the OD~0.3.
Inoculation
Drop 200ul bacteria solution on the processed film for 100s
After 100s, wash the film with wash solution twice Immediately.
Record images
Put the film into the sample box filled with samplw solution.
Put the sample box in SPRING and push or pull the sample box at the recommended position, and rotate the sample box to get clear interference fringes.
From 0s to 240s, take 2~3 pictures in a row with the time interval of 20s.
result
Process the gotten images and get the △N.
Calculate the △N and get the concentration is 1.80ug/ml and 3.09ug/ml.
publish
refresh
Contact Us
University of Science and Technology of China, No.96, JinZhai Road Baohe District,Hefei,Anhui, 230026,P.R.China.
Links
