<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_USTC_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:USTC/Notebook</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar"><NAV class="blue darken-4"><DIV class="nav-wrapper hoverable" style="z-index:2"><DIV id="nav-mobile" class="right hide-on-med-and-down"><H4 class="header white-text lighten-5" style="padding-right:4px;">Nanomachine Detecting Microbiotics </H4></DIV></DIV></NAV></DIV><DIV class="Nav2nd row white z-depth-1"><DIV class="Nav1th col l2 m2 hide-on-small-only blue darken-3"><A class="dropdown-button btn white-text" data-beloworigin="true" data-hover="true" href="#!" data-activates="Nav-dropdown">NOTEBOOK<I class="material-icons">arrow_drop_down</I></A></DIV><DIV class="col s10" style="z-index:2;"><UL class="tabs tabs-wrapper transparent"><LI class="tab col s3"><A href="#Protocols" class="blue-text active waves-effect waves-light">Protocols</A></LI><LI class="tab col s3 "><A href="#Daily-Notes-Stage-1" class="blue-text waves-effect waves-light">Daily Notes-Stage 1</A></LI><LI class="tab col s3 "><A href="#Daily-Notes-Stage-2" class="blue-text waves-effect waves-light">Daily Notes-Stage 2</A></LI></UL></DIV></DIV><UL id="Nav-dropdown" class="dropdown-content"><LI><A href="https://2015.igem.org/Team:USTC" class="waves-effect waves-light">Home</A></LI><LI><A href="https://2015.igem.org/Team:USTC/Description" class="waves-effect waves-light">Project</A></LI><LI><A href="https://2015.igem.org/Team:USTC/Modeling" class="waves-effect waves-light">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:USTC/Results" class="waves-effect waves-light">Results</A></LI><LI><A href="https://2015.igem.org/Team:USTC/Achievements" class="waves-effect waves-light">Achivements</A></LI><LI><A href="https://2015.igem.org/Team:USTC/Parts" class="waves-effect waves-light">Parts</A></LI><LI><A href="https://2015.igem.org/Team:USTC/Tutorials" class="waves-effect waves-light">Tutorials</A></LI><LI><A href="https://2015.igem.org/Team:USTC/Safety" class="waves-effect waves-light">Safety</A></LI><LI><A href="https://2015.igem.org/Team:USTC/Practices" class="waves-effect waves-light">Policy&amp;Practices</A></LI><LI><A href="https://2015.igem.org/Team:USTC/Collaborations" class="waves-effect waves-light">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:USTC/Team" class="waves-effect waves-light">Team</A></LI><LI><A href="https://2015.igem.org/Team:USTC/Attributions" class="waves-effect waves-light">Attributions</A></LI></UL><DIV class="row"><DIV class="col offset-m1 offset-l2 s12 m10 l8"><DIV id="Protocols" class="row"><DIV class="card hoverable"><DIV class="col s12 m9"><DIV class="card-content"><H4 id="synthetic-biology" class="scrollspy">Synthetic Biology</H4><P><STRONG>Genome Extraction</STRONG></P><P><EM>Material</EM>Items (Volume)
              Buffer GA (200ul)
              Proteinase K solution (20ul)
              Buffer GB (220ul)
              Ethanol (220ul)
              Buffer GD (500ul)
              Rinse PW (600ul)
              Elution Buffer TE (100ul)</P><P><EM>Extraction steps:</EM></P><OL><LI><P>Absolute ethanol before use in buffer and rinse GD PW, adding volume refer to the label on the bottle.</P></LI><LI><P>Take inoculum 1-5 ml, 10,000 rpm (~ 11,500 × g) was centrifuged 1 min, the supernatant net absorption as possible.</P></LI><LI><P>The cell pellet is added to 200 μl buffer GA, shaking to complete the cell suspension.</P></LI><LI><P>The tube is added to 20 μl Proteinase K solution, mixed evenly.</P></LI><LI><P>Add 220 μl buffer GB, oscillation 15 sec, 70 ℃ and placed 10 min, strain the solution clear, brief centrifugation to remove the tube drops on the inner wall .</P></LI><LI><P>Add 220 μl ethanol, mix thoroughly shaken 15 sec, flocculent precipitate may occur at this time, a brief centrifugation to remove the tube drops on the inner wall .</P></LI><LI><P>Previous resulting solution and the flocculent precipitate are added into a adsorption column CB3(adsorption column into the collection tube), 12,000 rpm centrifugation 30 sec, discard the liquid, set the adsorption column CB3 into the collection tube.</P></LI><LI><P>500 μl buffer GD was added to CB3 adsorption column(check whether ethanol has been added before use), 12,000 rpm (~ 13,400 × g) centrifugation 30 sec, discard the liquid, set the adsorption column CB3 into the collection tube.</P></LI><LI><P>600 μl rinse PW was added to CB3 adsorption column(check whether ethanol has been added before use), 12,000 rpm(~ 13,400 × g) centrifugation 30 sec, discard the liquid, set the CB3 adsorption column into the collection tube.</P></LI><LI><P>Repeat steps 8.</P></LI><LI><P>Place the column back into the collection tube CB3, 12,000 rpm (~ 13,400 × g) centrifugation 2 min, discard the waste.Place the adsorption column CB3 at room temperature for several minutes to completely dry absorbent material.</P></LI></OL><OL><LI><P>Put the CB3 adsorption column into a clean centrifuge tube and drop 50-200 μl elution buffer TE to the middle of the adsorbed film vacant. Put at room temperature 2-5 min, 12,000 rpm (~ 13,400 × g) centrifugation 2 min, collect the solution in a centrifuge tube.</P></LI><LI><P>Detect the concentration of DNA.</P></LI></OL><P><STRONG>Plasmid Extraction</STRONG>Preparation</P><OL><LI>Check out whether RNaseA has been added in Buffer P1. </LI><LI>Check out whether ethyl alcohol has been added in Wash Solution </LI><LI>Check out whether sediment exist in Buffer P2 and P2.
                Procedure</LI><LI>Absorb 1.5 to 5 mL bacteria solution in to EP tubes, centrifuge them at 8,000xg 2 mins and then discard the culture medium. </LI><LI>Add 250 ul Buffer P1 into sediment, and use spearhead to make bacteria suspended. </LI><LI>Add 250 ul Buffer P2, and overturn the EP tubes 5 to 10 times immediately and tenderly. Stand tubes about 2 to 4 mins. </LI><LI>Add 350 ul Buffer P3 and overturn the EP tubes 5 to 10 times again immediately and tenderly. </LI><LI>Centrifuge tubes at 12,000xg about 5 to 10 mins. </LI><LI>Pour the supernatant liquid into absorption columns ,centrifuge them at 8,000xg about 30 sec and then discard the liquid in the collecting pipe. </LI><LI>optional:Add 500 ul Buffer DW1 and centrifuge them at 9,000xg about 30 sec. Then discard the liquid in the collecting pipe. </LI><LI>Add 500 ul Wash Solution, centrifuge them at 9,000xg about 30 sec and discard the liquid in the collecting pipe. </LI><LI>Do the step 8 again. </LI><LI>Centrifuge the empty columns at 9,000xg about 1 min. </LI><LI>Put the columns in clean 1.5mL EP tubes, add 50 to 100 ul Elution Buffer on the absorption film, stand 1 min and centrifuge about 1 min at 9,000xg. </LI><LI>Keep the DNA solution for further work.
                The Protocol is based on SanPrep Column Type DNA Plasmid Extraction Kit.</LI></OL><P><STRONG>PCR System Preparation and Conditions Setting</STRONG>PCR system(set 50ul system as an example):
              The amount of substance of premier is written on the EP tubes. Calculating the volume of 1XTE required for 10uM primer solution is required before the system preparation.</P><OL><LI>Template DNA xul as required. </LI><LI>Forward Primer(10 uM) about 1 to 2 ul. The final concentration will be 0.2 to 0.4 uM. </LI><LI>Reverse Primer(10 uM) about 1 to 2 ul. The final concentration will be 0.2 to 0.4 uM. </LI><LI>5xTransStart FastPfu Fly Buffer 10 ul. </LI><LI>2.5 mM dNTPs 5 uM. The final concentration will be 0.25 mM. </LI><LI>TransStart FastPfu Fly DNA Polymerase 1 ul. </LI><LI>Double distilled water added to 50 ul.
                Reaction conditions </LI><LI>Calculate the Tm Value. If the complementary strip is smaller than 20 bp, Tm=4X(A+T)+2X(C+G), and the annealing temperature is Tm-5 degree centigrade. If not, use software to figure out the exact Tm value.(Also Tm value exist in report) </LI><LI>PCR cycle: </LI><LI>1 cycle of 95 degree centigrade about 2 min for pre-degeneration; </LI><LI>30 to 35 cycles of 95 degree centigrade about 10 sec, Tm-5 degree centigrade about 20 sec and 72 degree centigrade 1kb per min; </LI><LI>1 cycle of 75 degree centigrade about 5 mins and stay the device in 4 degree centigrade.
                The protocol is based on TransStart FastPfu Fly DNA Polymerase.</LI></OL><P><STRONG>PCR Production Extraction</STRONG>Prepation</P><OL><LI>Check out whether ethyl alcohol has been added into Wash Solution. </LI><LI>Check out whether sediment exist in Buffer B3. </LI><LI>Check out whether isopropanol has been added into Buffer B3.
                Procedure</LI><LI>Add Buffer B3 3 times volume of PCR system solution and incorporated thoroughly. </LI><LI>Centrifuge them at 8,000xg about 30 sec and drain the liquid in the collecting pipe. </LI><LI>Add 500 ul Wash Solution at 9,000xg about 30 sec and drain the liquid in the collecting pipe. </LI><LI>Do The step 3 again. </LI><LI>Centrifuge the empty pipes at 9,000Xg about 1 min. </LI><LI>Put the absorption column in some clean 1.5 mL EP tubes, add 15 to 40 ul Elution Buffer, stand for 1 min and centrifuge them at 9,000xg about 1min. </LI><LI>Keep the DNA solution.</LI></OL><P><STRONG>Restriction Enzymes AnalysisProtocol</STRONG>20ul system as following:</P><OL><LI>2ul plasmid DNA sample; </LI><LI>1ul restriction enzyme(both 1 ul for double digestion); </LI><LI>2ul 10X green buffer; </LI><LI>double distilled water added to 20 ul.
                Timetable:react the whole system at 37 degree centigrade about 2 hours and then inactivate the enzymes at 60 degree centigrade about 20 mins.</LI></OL><P><STRONG>DNA Gel Electrophoresis</STRONG>Standard 1% agarose gel
              (Generally, 0.7%~2% agarose gel is widely used in lab)</P><OL><LI>Measure out 1g agarose in scale, and then put the sample into conical flask. </LI><LI>Add 100mL 1XTAE into conical flask, shaking evenly.
                (about 1xTAE: dilute the 50x TAE to 1x TAE: add 10mL 50x TAE and 490 mL double distilled water to form as a system for further use)</LI><LI>Plastic wrap can be used to pack the bottleneck. </LI><LI>Use microwave oven to heat the solution about 5mins until the agarose is all dissolved and the solution is boiling and clear. It would be better if we shake the flask again after heating 2 mins.(CAUTION: HOT! Please use wool gloves when picking and placing flask.) </LI><LI>Let the gel cool down about 2 mins. </LI><LI>Pour the gel into gel tray with the appropriate comb inserted in the side of tray and wait till the gel solified(It may costs about 10 mins). Then remove the comb </LI><LI>Put the gel into gel box with the 1XTAE just immerse the gel exactly. Please let passages stay towards the negative pole(Often black). </LI><LI>Carefully add samples with 6X loading buffer and marker into passages. Dosage is based on the depth and width of passages. </LI><LI>Set electrophoresis conditions: 90V 600mA and 30mins and then turn on the power. </LI><LI>Disconnect the electrodes. </LI><LI>Utilize the devices that contains UV light to observe the gel and analyze the result. </LI><LI>Save the result in the computer for further use.</LI></OL><P><STRONG>Gel Extraction</STRONG>Preparation:</P><OL><LI>Check out whether ethyl alcohol is added into Wash Solution. </LI><LI>Check out whether Buffer B2 has sediment. </LI><LI>Adjust the thermostat water bath at 50 degree centigrade.
                Procedure:</LI><LI>Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers. </LI><LI>Measure out the weight of tubes of gel. </LI><LI>Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins. </LI><LI>Optional: If the length of the DNA strip is shorter than 500 bp, it would be better add isopropanol one third volume of Buffer B2. </LI><LI>After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s. </LI><LI>Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution. </LI><LI>Do the step 5 again. </LI><LI>Centrifuge the empty columns at 9000xg about 1min. </LI><LI>Open the columns and air them about 10 mins. </LI><LI>Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min. </LI><LI>Keep the DNA solution and often measure the concentration of DNA and A260/A280 for further study.
                The protocol is based on SanPerp Column Type DNA Gel Extraction Kit.</LI></OL><P><STRONG>Preparation of Competent CellProcedure</STRONG></P><OL><LI>Pick out the simple colony(DH5α) and cultivate bacteria in 2 to 3 mL LB media at 37 degree centigrade about 12 to 16 hours with shocking. </LI><LI>Take 0.05mL nutrient solution into 50 mL media and cultivate bacteria at 37 degree centigrade about 2 to 3 hours with shocking till the OD550 reaches 0.2 to 0.4. </LI><LI>Absorb 1.5mL nutrient solution into EP tubes with ice-bath about 10 mins. </LI><LI>Centrifuge the tubes at 4000xg at 4 degree centigrade about 10mins and discard the supernatant liquid. </LI><LI>Utilize 0.5 to 1 mL 0.1M calcium chloride with 15% glycerol to suspend bacteria.(<EM>ATTENTION:USE SPEARHEAD TO SUSPEND BACTERIA TENDERLY INSTEAD OF VIBRATOR </EM>) </LI><LI>Centrifuge the tubes at 4000xg at 4 degree centigrade about 8 mins and discard the supernatant liquid. </LI><LI>Utilize 100 ul 0.1M calcium chloride with 15% glycerol to suspend bacteria and then ready for transformation quickly or restore at -70 degree centigrade.(WITH ATTENTION AGAIN)
                TIPS: EXPERIENCE SAYS MODERATE MAGNESIUM ION HELPS DEVELOPMENT OF COMPETENT CELL.</LI></OL><P><STRONG>Transformation</STRONG>NOTE:Generally, competent bacteria are restrored in -70 degree centrigrade environment.
              Procedure</P><OL><LI>Take the competent bacteria from refrigerator and incubate them into ice about 5 mins until it is dissolved </LI><LI>Absorb 100pg to 10 ng plasmid(normally 1 to 2 uL, DO NOT add more than 5% volumn of bacteria solution) and mix it with bacteria solution thoroughly.
                ATTENTION: Please operate this step tenderly!!!</LI><LI>Put the tubes on the ice about 30 mins.(Time SHOULD BE ACCURATE) </LI><LI>Make a heat shock at 42 degree centigrade about 30 sec(TIME SHOULD BE ACCURATE) </LI><LI>Put the tubes on the ice about 2 to 3 mins again. </LI><LI>Add 900 ul LB medium into EP tubes and cultivate the bacteria at 37 degree centrigrade about 40 to 60 min. </LI><LI>Centrifuge them at 12,000xg about 15 sec and we will se sediment in the tubes. </LI><LI>Discard the supernatant liquid and leave about 220 ul medium. </LI><LI>Coat plate: add 200 ul solution in a large plate while add 20 ul solution in a small plate. </LI><LI>Cultivate these bacteria overnight for further use.</LI></OL><H4 id="biochemical-analysis" class="scrollspy">Biochemical Analysis</H4><P><STRONG>SDS-PAGE</STRONG></P><P><EM>Separation Gel</EM></P><P>H2O：0.6ml
              80%glycerol:0.6ml
              Gel buffer:1.2ml
              40%Arc-Bis Mix:1.2ml
              10%APS:20ul
              TEMED:2ul</P><P><EM>Stacking Gel</EM></P><P>H2O：1.0ml
              Gel buffer:0.40ml
              40%Arc-Bis Mix:0.20ml
              10%APS:20ul
              TEMED:2ul</P><P><EM>1×glycine Running Buffer</EM></P><P><EM>Coomassie brilliant blue solution</EM></P><P><EM>Protocol</EM></P><OL><LI>Assemble the sheet for adding the gel.</LI><LI>Prepare separating gel, and load into the sheet suitable height, then dry it at room temperature.</LI><LI>Prepare stacking gel into the sheet, and plug in the comb, then dry it at room temperature.</LI><LI>Remove the sheet, and pull out a comb, then transfer the plastic sheet into the electrophoresis tank.</LI><LI>Add the electrophoresis tank glycine running buffer to the appropriate location.</LI><LI>Sample electrophoresis.</LI><LI>After completion of the electrophoresis, the gel was removed, placed in Coomassie brilliant blue solution, stain in a microwave for 1-2min.</LI><LI>Gel elution, to background staining disappear.</LI><LI>Observe gel.</LI></OL><P><STRONG>NPN Assay</STRONG></P><P><EM>Material</EM></P><P>BL21
              BL21-T7-SCVE
              BL21-T7-OprF
              10mM N-Phenyl-1-naphthylamine(NPN)</P><OL><LI>Inoculate an overnight culture (30 mL with 1 mL of pre-culture). </LI><LI>Centrifugation (15 min at 3000 g) of the whole overnight culture and discard supernatant. </LI><LI>Resuspend pellet in equal volume of 10 mM PBS buffer. </LI><LI>Repeat steps 2) and 3) twice. </LI><LI>Mix 1 mL of washed cells with a 10 mM stock solution of NPN to a final concentration of 10 µM. </LI><LI>Transfer 200 µL of samples to a flat bottomed black 96 well plate. </LI><LI>Detect fluorescence intensity.</LI></OL><P>Fluorescence intensity of sample is recorded by <A href="http://www.bmglabtech.com/en/products/clariostar/">BMG Labtech CLARIOstar®</A></P><P><STRONG>ONPG Assay</STRONG></P><P>BL21
              BL21-T7-SCVE
              BL21-T7-OprF
              ONPG solution.</P><OL><LI>Inoculate an overnight culture (30 mL with 1 mL of pre-culture). </LI><LI>Centrifugation (8 min at 3000 g) of the whole overnight culture and discard supernatant. </LI><LI>Resuspend pellet in equal volume of 10 mM PBS buffer. </LI><LI>Repeat steps 2) and 3) twice. </LI><LI>Inside a clear, flat bottomed 96 well plate, mix 180 µL ONPG (o-nitrophenyl-b-galactopyranoside) stock solution (3 mM in 10 mM PBS buffer) with 20 µL of washed cells. </LI></OL><P>Absorbance of sample is recorded by <A href="http://www.bmglabtech.com/en/products/clariostar/">BMG Labtech CLARIOstar®</A></P><H4 id="special-protocol-for-ndm" class="scrollspy">Special Protocol for NDM</H4><P><STRONG>Prepare Bacterial Powder</STRONG></P><P>for 500mL</P><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Mass</TH></TR></THEAD><TBODY><TR><TD>Yeast Extract</TD><TD>2.5g</TD></TR><TR><TD>Trtptone</TD><TD>2.5g</TD></TR><TR><TD>NaCl</TD><TD>15g</TD></TR><TR><TD>KH2PO4</TD><TD>0.5g</TD></TR><TR><TD>Na2HPO4·12H2O</TD><TD>6.5g</TD></TR><TR><TD>glycerol</TD><TD>1.5g</TD></TR><TR><TD>NaCl</TD><TD>15g</TD></TR></TBODY></TABLE><OL><LI><P>prepare the culture medium as above.</P></LI><LI><P>prepare the drying agent:
                  skim milk 12%(mass fraction)
                  NaCl 2%(mass fraction)</P></LI><LI>culture the bacterial with the culture medium to log phase.</LI><LI>centrifuge the bacterial solution at 5,000r/min.</LI><LI>resuspend bacterial with drying agent in the same volume.</LI><LI>divide the resuspension into EP tube.</LI><LI>freeze the bacterial at -80 degree centigrade for 8 hours.</LI><LI>drying the bacterial with lyophilizer.</LI><LI>store the bacterial powder at -20 degree centigrade.</LI></OL><P><STRONG>NDM Operation</STRONG>This is a brief protocol on NDM operation. For more information, please visit <A href="https://2015.igem.org/Team:USTC/Tutorials">Tutorials</A></P><P><EM>Preparation of antibiotics ruler</EM>To detect antibiotics concentration, users need firstly prepare standard antibiotic concentration, we will set chloromycetin as an example.</P><TABLE><THEAD><TR><TH>Ruler Sample</TH><TH>Concentration ug/mL</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>0.1</TD></TR><TR><TD>2</TD><TD>0.5</TD></TR><TR><TD>3</TD><TD>1.0</TD></TR><TR><TD>4</TD><TD>5.0</TD></TR></TBODY></TABLE><P><EM>Coating film with polylysine</EM>Add approximiately 400 ul 20ug/mL polylysine on the film, and store the film at 4 degree celcius for more than 4 hours. After 4 hours, absorb polylysine and then wash the film by PBS buffer. <EM>Note: polylysine can be recycled.</EM></P><P><EM>Adhesion Assay</EM>Add 200 ul bacteria solution on film about 100s. <EM>Note: bacteria grown should be in steady state, and you should dilate bacteria and let its OD(600) approximiately reach 0.05</EM></P><P><EM>Operate Optical Path</EM>Operating optical path within 100s would be highly recommended for users. To get the best images, you should observe and get fringes parallel to y axis of screen as possible.</P><P><EM>Observe and Record</EM>The measurement period is about 300s. Consequently, we recommend users to take a series of images each 10s during the beginning of 300s. </P><H4 id="materials" class="scrollspy">Materials</H4><P><STRONG>Materials of LB Culture Medium</STRONG><STRONG>LB Culture(500ml)</STRONG></P><P>Tryptone:5g
              Yeast Extract:2.5g
              NaCl:5g</P><P><STRONG>LB Solid Culture(200ml)</STRONG></P><P>Tryptone:2g
              Yeast Extract:1g
              NaCl:2g
              AgerA:3g</P><P><STRONG>PBS Buffer(1L)</STRONG></P><P>KH2PO4:0.74g
              Na2HPO4:1.42g
              NaCl:8g
              KCl:0.2g
              Add ddH2O about 800mL fully dissolved, then add concentrated hydrochloric acid pH to 7.4, finally set the capacity to 1L.</P><P><STRONG>ONPG Stock Solution</STRONG></P><P>ONPG:0.4g
              ddH2O:75ml
              PBS buffer:25ml
              37℃dissolve the ONPG, store in refrigerator without night at 2-5℃.</P><P><STRONG>NPN Stock Solution(10mM, volume 100ml)</STRONG></P><P>1N NPN: 1g
              Dissolve the NPN with 100ml anhydrous alcohol, store it store in refrigerator without night at 2-5℃.</P><P><STRONG>1.5M Tris-HCl(PH 8.8)</STRONG></P><OL><LI>Weigh 181.7g Tris into a 1L beaker</LI><LI>Add about 800ml de ionized water, stir well</LI><LI>To be cooled, with concentrated hydrochloric acid to adjust PH to 8.8</LI><LI>Solution set to 1L</LI><LI>After high temperature and high pressure sterilization, room temperature preservation</LI></OL><P><STRONG>1.5M Tris-HCl(PH 6.8)</STRONG></P><OL><LI>weigh 181.7g Tris into a 1L beaker</LI><LI>add about 800ml de ionized water, stir well</LI><LI>to be cooled, with concentrated hydrochloric acid to adjust PH to 6.8</LI><LI>solution set to 1L</LI><LI>after high temperature and high pressure sterilization, room temperature preservation</LI></OL><P><STRONG>10% SDS</STRONG></P><OL><LI>Get 10gSDS adding 80ml ddH2O into a 100ml beaker, 68℃, heated and dissolved.</LI><LI>Drop the concentrated hydrochloric acid to PH 7.2.</LI><LI>The solution is determined to be 100ml and the room temperature is kept.</LI></OL><P><STRONG>30% Acrylamide</STRONG></P><OL><LI>Get 280g acrylamide and BIS to 1L.</LI><LI>Add 800ml deionized water to the beaker, stir to dissolve.</LI><LI>Add the solution to 1L with deionized water, then use the 0.45um filter to filter the impurities.</LI><LI>Store the solution in brown bottle at 4℃.</LI></OL><P><STRONG>5×Tris Glycine Buffer</STRONG></P><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Mass(g)</TH></TR></THEAD><TBODY><TR><TD>Tris</TD><TD>15.1g</TD></TR><TR><TD>Glycine</TD><TD>94g</TD></TR><TR><TD>SDS</TD><TD>5.0g</TD></TR></TBODY></TABLE><OL><LI><P>The following reagents are added in a 1L beaker</P></LI><LI><P>Add about 800ml deionized water, stir and dissolve.</P></LI><LI>With deionized water, the solution is set to 1L, and the room temperature is kept.</LI></OL><P><STRONG>5×SDS-PAGE Loading Buffer</STRONG></P><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Mass or volume</TH></TR></THEAD><TBODY><TR><TD>1M Tris-HCl(PH 6.8)</TD><TD>1.25ml</TD></TR><TR><TD>SDS</TD><TD>0.5g</TD></TR><TR><TD>BPB</TD><TD>25mg</TD></TR><TR><TD>Glycerol</TD><TD>2.5ml</TD></TR></TBODY></TABLE><OL><LI><P>The following reagents are added in a 10ml centrifugal tube.</P></LI><LI><P>Adding deionized water to dissolve the capacity to 5ml.</P></LI><LI>Per 500ul packaging, at room temperature.</LI><LI>Before using 25ul 2-ME added to each of the small.</LI><LI>Buffer with 2-ME can be saved for a month.</LI></OL><P><STRONG>Bradford R-250 Staining Solution</STRONG></P><OL><LI>Get 1g Bradford R-250, placed in a 1L beaker.</LI><LI>Take 250ml isopropanol into the beaker, stir to dissolve.</LI><LI>Adding 100ml to the glacial acetic acid, stirring evenly.</LI><LI>Add 650ml of deionized water, stirring evenly.</LI><LI>filter paper to remove particulate matter, room temperature preservation.</LI></OL><P><STRONG>Bradford R-250 Destaining Solution</STRONG></P><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Mass or volume</TH></TR></THEAD><TBODY><TR><TD>Acetic acid</TD><TD>100ml</TD></TR><TR><TD>Ethanol</TD><TD>50ml</TD></TR><TR><TD>dH2O</TD><TD>850ml</TD></TR></TBODY></TABLE><OL><LI><P>Take the solution in a 1L beaker.</P></LI><LI><P>Fully mixed for use.</P></LI></OL><P><STRONG>50×TAE Buffer</STRONG></P><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Mass or volume</TH></TR></THEAD><TBODY><TR><TD>Tris</TD><TD>242g</TD></TR><TR><TD>Na2EDTA·2H2O</TD><TD>37.2g</TD></TR></TBODY></TABLE><OL><LI><P>The following reagents, placed in a 1L beaker.</P></LI><LI><P>Add 800ml deionized water to the beaker, stir to dissolve.</P></LI><LI><P>Add 57.1ml acetic acid, stirring.</P></LI><LI><P>Add deionized water into the solution to 1L, then keep at room temperature.</P></LI></OL><P><STRONG>Agarose</STRONG></P><OL><LI><P>Prepare 1×TAE buffer solution.</P></LI><LI><P>According to the required amount of agarose and concentration, add into appropriate conical bottle.</P></LI><LI><P>Adding a certain amount of 1 x buffer TAE.</P></LI><LI><P>Seal conical bottle with fresh-keeping film, then use the microwave to dissolve agarose.</P></LI><LI><P>Solution was cooling to 60 degrees Celsius, fully mixed.</P></LI><LI><P>Insert a suitable comb in the plastic mold, then pour the solution into the plastic mold.</P></LI><LI><P>Solidify the solution at the room temperature, then place the mold in the electrophoresis tank.</P></LI></OL><P><STRONG>6×Loading Buffer</STRONG></P><OL><LI>The following reagents, placed in a 500ml beaker.</LI><LI>Add 200ml of deionized water to the beaker, heating and stirring fully dissolved.</LI><LI>After add 180ml glycerol, the use of NaOH 2N to adjust PH to 7.</LI><LI>Add deionized water to 500ml, then keep at room temperature. </LI></OL><P><STRONG>Ampicillin</STRONG></P><OL><LI>5g ampicillin in 50ml Centrifugal tube.</LI><LI>After adding 40ml to the water, fully dissolve the solution, then keep it at 50ml.</LI><LI>Using 0.22ul filter sterilization.</LI><LI>Packaging (1ml), stored at -20℃.</LI></OL><P><STRONG>IPTG</STRONG></P><OL><LI>Place 1.2gIPTG in the 50ml centrifuge tube.</LI><LI>Add 40ml dH2O, fully dissolve the solution, then keep it at 50ml.</LI><LI>Using 0.22ul filter sterilization.</LI><LI>Packaging (1ml), stored at 20℃.</LI></OL><P><STRONG>TB Culture</STRONG></P><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Mass(g)</TH></TR></THEAD><TBODY><TR><TD>Tryptone</TD><TD>10g</TD></TR><TR><TD>Yeast Extract</TD><TD>5g</TD></TR><TR><TD>NaCl</TD><TD>10g</TD></TR></TBODY></TABLE><OL><LI>Preparation of phosphate buffer solution: 2.31gKH2PO4 and 12.54gK2HPO4 in 90ml deionized water, stirring and dissolving, then keep it at 100ml, high temperature and high pressure sterilized.</LI><LI><P>Take the following reagents, placed in a 1L conical flask.</P></LI><LI><P>Add 800ml deionized water, stir and dissolve.</P></LI><LI>Keep volume at 1L, high temperature and high pressure sterilization.</LI><LI>Cooling to 60℃, add 100ml phosphate buffer.</LI><LI>4℃ for preservation.</LI></OL><P><STRONG>SOB Culture</STRONG></P><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Mass(g)</TH></TR></THEAD><TBODY><TR><TD>Tryptone</TD><TD>10g</TD></TR><TR><TD>Yeast Extract</TD><TD>5g</TD></TR><TR><TD>NaCl</TD><TD>10g</TD></TR></TBODY></TABLE><OL><LI>Preparation of 250mM KCl solution: 90ml deionized water dissolved in 1.86gKCl, then keep the solution at 100ml.</LI><LI>Preparation of KCl 2M solution: 90ml deionized water dissolved in 19gMgCl2, then keep the solution at 100ml, high temperature and high pressure sterilization.</LI><LI><P>Take the following reagents, placed in a 1L conical flask.</P></LI><LI><P>Add 800ml deionized water, stir well.</P></LI><LI>Add 10ml KCl solution into the conical flask.</LI><LI>Drop NaOH 5N solution, adjust PH to 7.</LI><LI>Adding de ionized water to 1L.</LI><LI>High temperature and high pressure sterilization, stored at 4℃.</LI><LI>Add 5ml 2M MgCl2 solution before using.</LI></OL><P><STRONG>SOC Culture</STRONG></P><OL><LI>Prepare 1M glucose solution：add 18g glucose into 90ml dH2O， fully dissolve the glucose， keep the solution at 100ml.</LI><LI>Add 2ml 1M glucose solution into 100ml SOB solution，mix them evenly.</LI><LI>Preserve at 4℃.</LI></OL><P><STRONG>Semi-Solid M63 Medium</STRONG>In order to characterization the movement of TOP10, semi-solid M63 medium preparation is indispensable in this research.</P><P> (500mL Protocol)</P><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Mass</TH><TH>Proportion(g/mL) or Mole</TH></TR></THEAD><TBODY><TR><TD>Yeast Extract</TD><TD>2.5g</TD><TD>0.5%</TD></TR><TR><TD>Trtptone</TD><TD>5g</TD><TD>1%</TD></TR><TR><TD>NaCl</TD><TD>5g</TD><TD>1%</TD></TR><TR><TD>Agar</TD><TD>1.25g</TD><TD>0.25%</TD></TR><TR><TD>KH2PO4</TD><TD>6.8g</TD><TD>50mM</TD></TR><TR><TD>KOH</TD><TD>2.1g</TD><TD>37.5mM</TD></TR><TR><TD>(NH4)2SO4</TD><TD>0.09g</TD><TD>7.5mM</TD></TR><TR><TD>MgSO4</TD><TD>0.06g</TD><TD>0.5mM</TD></TR><TR><TD>FeSO4</TD><TD>0.2969g</TD><TD>1.95mM</TD></TR><TR><TD>D-Glucose</TD><TD>1.98g</TD><TD>11mM</TD></TR></TBODY></TABLE><P><STRONG>Calibration</STRONG></P><P>Use 1ug/ml Chloromycetin activating bacteria to test the film.</P><P><STRONG><EM>Film</EM></STRONG></P><P>Film type:glass(cover clip——thickness~0.16mm width~2.5cm longth~3.7cm)
              Use the aerosol paint to process one of the surface of the film.
              Coating the film with 400ul 20ug/ml PLL(Poly-L-Lysine) at the temperature of 4℃ for over 4 hours.</P><P><EM>Bacteria solution</EM>Bacteria type: BL21+OprF
              Mix bacteria with PBS and make the OD~0.3.</P><P><EM>Sample solution</EM>1ug/ml Chloromycetin PBS solution.</P><P><EM>Inoculation</EM>Drop 200ul bacteria solution on the processed film for 100s
              After 100s, wash the film with wash solution twice Immediately.</P><P><EM>Record images</EM>1.Put the film into the sample box filled with sample solution.
              2.Put the sample box in SPRING and push or pull the sample box at the recommended position, and rotate the sample box to get clear interference fringes.
              3.From 0s to 240s, take 2~3 pictures in a row with the time interval of 20s.</P><P><EM>Results</EM>Succeed in getting the excellent interference fringes.
              Record data and process the image.
              Use the Matlab process the image.
              Get the fitting according to the module thus get the calibration.</P><P><STRONG><EM>Verifying Calibration</EM></STRONG></P><P><EM>Sample solution</EM>（1）1.8ug/ml Chloromycetin PBS solution.
              （2）3.2ug/ml Chloromycetin PBS solution.</P><P><EM>result</EM>Process the gotten images and get the △N.
              Calculate the △N and get the concentration which is 1.80ug/ml and 3.09ug/ml.</P></DIV></DIV><DIV class="col hide-on-small-only m3"><DIV class="toc-wrapper pinned"><UL class="section table-of-contents"><LI><A href="#synthetic-biology">Synthetic Biology</A></LI><LI><A href="#biochemical-analysis">Biochemical Analysis</A></LI><LI><A href="#special-protocol-for-ndm">Special Protocol for NDM</A></LI><LI><A href="#materials">Materials</A></LI></UL></DIV></DIV></DIV></DIV><DIV id="Daily-Notes-Stage-1" class="row"><DIV class="card hoverable"><DIV class="col s12"><DIV class="card-content"><UL class="cbp_tmtimeline"><LI><TIME class="cbp_tmtime" datetime="2015-07-04 00:00"><SPAN>2015</SPAN><SPAN>4/7</SPAN></TIME><DIV class="cbp_tmlabel"><H5>Medium preparation</H5><P>Prepare and keep a part of solid medium and broth in the refrigerator as a backup.</P><H5>Cell culture</H5><P>Culture the cell,TOP10, with LB liquid solution overnight.</P></DIV></LI><LI><TIME class="cbp_tmtime" datetime="2015-07-05 00:00"><SPAN>2015</SPAN><SPAN>5/7</SPAN></TIME><DIV class="cbp_tmlabel"><H5>Design protocol</H5></DIV></LI><LI>Take the solid medium, heat it until liquid. Then make two plate of solid medium, evenly. Cool it to the room temperature.</LI><LI>prepare antibiotic solution with concentration gradient
                      a. prepare antibiotic(neomycin) solution with ddH2O at 50mg/ml as mother solution 1. And dilute it to 10% concertration(5mg/ml) with ddH2O as mother solution 2. Then prepare 8 parts of solution as this:</LI><TABLE><THEAD><TR><TH>Number</TH><TH>Solution 1(V/uL)</TH><TH>Solution 2(V/uL)</TH></TR></THEAD><TBODY><TR><TD>0</TD><TD>0</TD><TD>0</TD></TR><TR><TD>1</TD><TD>0</TD><TD>0.2</TD></TR><TR><TD>2</TD><TD>0</TD><TD>1</TD></TR><TR><TD>3</TD><TD>0.2</TD><TD>0</TD></TR><TR><TD>4</TD><TD>0.4</TD><TD>0</TD></TR><TR><TD>5</TD><TD>0.6</TD><TD>0</TD></TR><TR><TD>6</TD><TD>0.8</TD><TD>0</TD></TR><TR><TD>7</TD><TD>1</TD><TD>0</TD></TR></TBODY></TABLE><P> Keep them in the 70℃ water bath. Then add 800uL liquid solid medium into each Ep tubes. Transfer them into 50℃ water bath. </P><OL><LI>Cut 4 hole in each plate of solid medium, whose diameter is at 8mm. </LI><LI>Take 5mL culture into the 2 Ep tubes. Centrifuge it at twice. Then add liquid solid medium at 50℃, each 800uL, dissolve evenly. </LI><LI>Take 200uL top10 solution into 8 Ep tubes each at 50℃. Then add the mixed solution into the hole at once until the hole is added up.</LI></OL><P>Results: the raw results showed differerent characteristics of chemotaxis based on distinct antibiotic concentration.</P><LI><TIME class="cbp_tmtime" datetime="2015-07-12 00:00"><SPAN>2015</SPAN><SPAN>12/7</SPAN></TIME><DIV class="cbp_tmlabel"><H5>4th Colonial PCR for COMPX</H5></DIV></LI><LI>2μl Taq buffer with KCl</LI><LI>0.4μl dNTPs</LI><LI>1μl COMPX-F Primer</LI><LI>1μl COMPX-R Primer</LI><LI>0.25μl Taq DNA Polymerase</LI><LI>2μl MgCl2</LI><LI>1μl TOP10 Solution</LI><LI>12.35μl ddH2O</LI></UL><P>Renaturation at 55℃, extend for 1 min. 2 tubes without Taq DNA Polymerase as blank control.</P><H5>4th Electrophoresis for Colonical PCR</H5><P>Bands are predicted except one. Maybe caused by accident. PCR product keeped for later use.</P><H5>1st Plasmid Extraction for BBa K1492002 &amp; pSB1C3</H5><P>1 Centifuge 5 ml bacterium solution, few sediment getted.</P><P>2 Add 250 μl Buffer P1, resuspend cells.</P><P>3 Add 250 μl Buffer P2, mix well, 4 min's standing.</P><P>4 Add 350 μl Buffer P3, mix well.</P><P>5 13.4 rpm centifuge 10 min, move all supernatant to adsorption column, 11.6 rpm centifuge 30 s, discard filtrate.</P><P>6 Add 500 μl Wash Solution, 11.6 rpm centifuge 30 s, discard filtrate. Repeat once.</P><P>7 11.6 rpm centifuge 1 min.</P><P>8 Add 50 μl ddH2O, 2 min's standing, 11.6 rpm centifuge 1 min.</P><H5>1st Electrophoresis for Plasmid Extraction</H5><P>No result. May due to high concentration of chloramphenicol, bacteria grow very slow. As there is few bacteria, no plasmid getted. Bacterium solution dilute 1:1 with LB medium.</P></DIV><LI><TIME class="cbp_tmtime" datetime="2015-07-13 00:00"><SPAN>2015</SPAN><SPAN>13/7</SPAN></TIME><DIV class="cbp_tmlabel"><H5>The steak-plate of <EM>Pseudomonas aeruginosa</EM></H5><P>By using the steak-plate method, we successful separated the single conlony of <EM>P. aeruginosa</EM>.</P><P>\(^o^)/~！</P><H5>2nd Plasmid Extraction for BBa K1492002 &amp; pSB1C3</H5></DIV></LI><LI>Centifuge 4.5 ml bacterium solution.</LI><LI>Add 250 μl Buffer P1, resuspend cells.</LI><LI>Add 250 μl Buffer P2, mix well, 4 min's standing.</LI><LI>Add 350 μl Buffer P3, mix well.</LI><LI>13.4 rpm centifuge 10 min, move all supernatant to adsorption column, 11.6 rpm centifuge 30 s, discard filtrate.</LI><LI>Add 500 μl Wash Solution, 11.6 rpm centifuge 30 s, discard filtrate. Repeat once.</LI><LI>11.6 rpm centifuge 1 min.</LI><LI>Add 30 μl ddH2O, 2 min's standing, 11.6 rpm centifuge 1 min.</LI><H5>2nd Electrophoresis for Plasmid Extraction</H5><P>Successfully extracted tRNA plasmid. Failed to extract pSB1C3 plasmid, cause culture is contaminated.</P><P>Result:</P><P>tRNA 114ng/μl</P><H5>1st PCR for tRNA</H5><UL><LI>2μl Taq buffer with KCl</LI><LI>0.4μl dNTPs</LI><LI>1μl tRNA-F2 Primer</LI><LI>1μl tRNA-R Primer</LI><LI>0.25μl Taq DNA Polymerase</LI><LI>2μl MgCl2</LI><LI>1μl tRNA plasmid</LI><LI>12.35μl ddH2O</LI></UL><P>Renaturation at 50℃, extend for 1:30.</P><H5>1st Electrophoresis for tRNA PCR</H5><P>No result. </P><H5>Transformation of Four Parts of Plate 4 </H5><UL>material:
                        <LI>competent cell TG1 from the lab 319</LI><LI>4 parts of the plate 4(pSB1A2)</LI><LI>No.1:BBa_B0034 No.2:BBa_C0051</LI><LI>No.3:BBa_R0051 No.4:BBa_E0040</LI></UL><OL>the procedure
                        <LI>Take the competent bacteria from refrigerator and incubate them into ice about 10 mins until it is dissolved</LI><LI>Absorb 1 uL or 2 uL plasmid and mix it with bacteria solution thoroughly.</LI><LI>Put the tubes on the ice about 30 mins.</LI><LI>Make a heat shock at 42 degree centigrade about 60 sec</LI><LI>Put the tubes on the ice about 3 mins again.</LI><LI>Add 900 ul LB medium into EP tubes and cultivate the bacteria at 37 degree centrigrade about 60 min.</LI><LI>Centrifuge them at 4,000xg about 15 sec and we will se sediment in the tubes.</LI><LI>Discard the supernatant liquid and leave about 100ul medium.</LI><LI>Coat plate: add 100ul solution in a plate</LI></OL></DIV><LI><TIME class="cbp_tmtime" datetime="2015-07-14 00:00"><SPAN>2015</SPAN><SPAN>14/7</SPAN></TIME><DIV class="cbp_tmlabel"><H5>The Results of Transformation of Jul 13th</H5><P>transformations of three parts are successful:</P></DIV></LI><LI>No.1 B0034</LI><LI>No.2 C0051</LI><LI>No.4 E0040</LI><P>transformation of one part is failing:</P><P>No.3 R0051</P><H5>Plasmid Extraction of pSB1C3</H5><P>the plasmis:pSB1C3</P><P>the procedure:</P><OL><LI>Absorb 6 mL bacteria solution in to EP tubes, centrifuge them at 12,000xg 1mins and then discard the culture medium.</LI><LI>Add 250 ul Buffer S1 into sediment, and use spearhead to make bacteria suspended.</LI><LI>Add 250 ul Buffer S2, and overturn the EP tubes 8 times immediately and tenderly. Stand tubes about 3 mins.</LI><LI>Add 350 ul Buffer W3 and overturn the EP tubes 10 times again immediately and tenderly.</LI><LI>Centrifuge tubes at 12,000xg about 10 mins.</LI><LI>Pour the supernatant liquid into absorption columns ,centrifuge them at 12,000xg about 30 sec and then discard the liquid in the collecting pipe.</LI><LI>Add 500 ul Buffer W1 and centrifuge them at 12,000xg about 30 sec. Then discard the liquid in the collecting pipe.</LI><LI>Add 700 ul Buffer W2, centrifuge them at 12,000xg about 30 sec and discard the liquid in the collecting pipe.</LI><LI>Do the step 8 again.</LI><LI>Centrifuge the empty columns at 12,000xg about 1 min.</LI><LI>Put the columns in clean 1.5mL EP tubes, add 80 ul Elution Buffer on the absorption film, stand 1 min and centrifuge about 1 min at 12,000xg.</LI></OL><P>PS:this protocol is from lab 319</P><P>the concentration :11.0ng/ul</P><P>PS:the concentration of the control plasmid is 136.0 ng/ul,so we guess that maybe the concentration of bacterium solution is low.Also we guess that it's caused by the missing of spreading.</P><H5>Transformation of Three Parts and One Plasmid</H5><OL>the plasmid going to be transformed:
                        <LI>No.1 BBa_R0062(plate 2;pSB1C3)</LI><LI>No.2 BBa_B0015(plate 3;pSB1C3)</LI><LI>No.3 BBa_R0051(plate 4;pSB1A2)</LI><LI>No.4 pSB1C3</LI></OL><OL>the procedure:
                      <P>note:add 1 ul R0062 ,1 ul B0015,2 ul R0051,</P><P>2 ul pSB1C3 to 100 ul competent cells respectively</P><LI>Take the competent bacteria from refrigerator and incubate them into ice about 10 mins until it is dissolved</LI><LI>Absorb 1 uL or 2 uL plasmid and mix it with bacteria solution thoroughly.</LI><LI>Put the tubes on the ice about 30 mins.</LI><LI>Make a heat shock at 42 degree centigrade about 60 sec</LI><LI>Put the tubes on the ice about 3 mins again.</LI><LI>Add 900 ul LB medium into EP tubes and cultivate the bacteria at 37 degree centrigrade about 60 min.</LI><LI>Centrifuge them at 4,000xg about 15 sec and we will se sediment in the tubes.</LI><LI>Discard the supernatant liquid and leave about 100ul medium.</LI><LI>Coat plate: add 100ul solution in a plate </LI></OL><H5>Plasmid Extraction of Three Parts of Plate 4</H5><P>the procedure is the same as before</P><P>the concentration :</P><P>B0034 157.6ng/ul, 113.7ng/ul;</P><P>C0051 171.7ng/ul, 194.6ng/ul;</P><P>E0040 278.5ng/ul, 228.2ng/ul</P></DIV><LI><TIME class="cbp_tmtime" datetime="2015-07-15 00:00"><SPAN>2015</SPAN><SPAN>15/7</SPAN></TIME><DIV class="cbp_tmlabel"><H5>Results of transformation </H5><P>R0051 no colony</P><P>R0062 have single colony</P><P>B0015 have single colony</P><P>pSB1C3 have single colony and the colony is red</P><H5>Plasmid Extraction Results</H5><P>This is extraction was for previous transformation.</P></DIV></LI><LI>pSB1C3(1): 108ng/uL</LI><LI>pSB1C3(2): 157ng/uL</LI><LI>B0015(1): 19ng/uL</LI><LI>B0015(2): 93ng/uL</LI><P>Prepared for further use.</P></DIV><LI><TIME class="cbp_tmtime" datetime="2015-07-16 00:00"><SPAN>2015</SPAN><SPAN>16/7</SPAN></TIME><DIV class="cbp_tmlabel"><H5>Colonical PCR for micF and SoxS</H5><P>Primers produced from Sangon Biotech Co.,Ltd.</P><TABLE><THEAD><TR><TH>Primer</TH><TH>Sequence(5'-3')</TH></TR></THEAD><TBODY><TR><TD>micF-luxI-R</TD><TD>TTATAGTCATATAAATTTTCTCCTCTTTCCGAATGCGAGGCATCC</TD></TR><TR><TD>micF-F</TD><TD>GCGGCCGCTTCTAGAGCCTCATTAATCAGTCGGC</TD></TR><TR><TD>SoxS-RBS-luxI-R</TD><TD>TATAGTCATATAAATTTTCTCCTCTTTCTAGTGCGTGCGCCGGTACCTT</TD></TR><TR><TD>BB-SoxS-F</TD><TD>CGGCCGCTTCTAGAGCGTGCGGAACATTCG</TD></TR></TBODY></TABLE><P>For micF, primers including micF-luxI-R, micF-F, protocol as following:</P></DIV></LI><LI>2μl Taq buffer with KCl</LI><LI>0.4μl dNTPs</LI><LI>1μl micF-luxI-R Primer</LI><LI>1μl micF-F Primer</LI><LI>0.25μl Taq DNA Polymerase</LI><LI>2μl MgCl2</LI><LI>1μl Top10(K-12) Strain <EM>E. coli</EM> bacterial solution</LI><LI>12.35μl ddH2O</LI><P>For SoxS, primers including SoxS-RBS-luxI-R, BB-SoxS-F, protocol illustrates as:</P><UL><LI>2μl Taq buffer with KCl</LI><LI>0.4μl dNTPs</LI><LI>1μl SoxS-RBS-luxI-R Primer</LI><LI>1μl BB-SoxS-F Primer</LI><LI>0.25μl Taq DNA Polymerase</LI><LI>2μl MgCl2</LI><LI>1μl 1μl Top10(K-12) Strain <EM>E. coli</EM> bacterial solution</LI><LI>12.35μl ddH2O</LI></UL><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)</LI><LI>Elongation for whole at 72 degree celcius 7 min</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>Characterization of Colonical PCR by Gel Electrophoresis</H5><P>The result illustrated the success of PCR</P></DIV><LI><TIME class="cbp_tmtime" datetime="2015-07-17 00:00"><SPAN>2015</SPAN><SPAN>17/7</SPAN></TIME><DIV class="cbp_tmlabel"><H5>Colonical PCR for OprF from <EM>Psudomonus aeruginosa</EM></H5><P>Primers produced from Sangon Biotech Co.,Ltd.</P><TABLE><THEAD><TR><TH>Primer</TH><TH>Sequence(5'-3')</TH></TR></THEAD><TBODY><TR><TD>OprF-F</TD><TD>AGAGGAGAAAATCGGAATGAAACTGAAGAACAC</TD></TR><TR><TD>OprF+Ter-R</TD><TD>TGATGCCTGGCTCTAGTATTACTTGGCTTCAGCTT</TD></TR></TBODY></TABLE><P>PCR protocol:</P><P>For OprF, primers including OprF-F, OprF+Ter-R, protocol as following:</P></DIV></LI><LI>2μl Taq buffer with KCl</LI><LI>0.4μl dNTPs</LI><LI>1μl OprF-F Primer</LI><LI>1μl OprF+Ter-R Primer</LI><LI>0.25μl Taq DNA Polymerase</LI><LI>2μl MgCl2</LI><LI>1μl <EM>Psudomonus aeruginosa</EM> (PAO1) bacterial solution</LI><LI>12.35μl ddH2O</LI><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)</LI><LI>Elongation for whole at 72 degree celcius 7 min</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>Plasmid Extraction of SCVE</H5><P>SCVE synthesized from Sangon Biotech Co.,Ltd.</P><P>Procedure including:</P><OL><LI>Absorb 6 mL bacteria solution in to EP tubes, centrifuge them at 12,000xg 1mins and then discard the culture medium.</LI><LI>Add 250 ul Buffer S1 into sediment, and use spearhead to make bacteria suspended.</LI><LI>Add 250 ul Buffer S2, and overturn the EP tubes 8 times immediately and tenderly. Stand tubes about 3 mins.</LI><LI>Add 350 ul Buffer W3 and overturn the EP tubes 10 times again immediately and tenderly.</LI><LI>Centrifuge tubes at 12,000xg about 10 mins.</LI><LI>Pour the supernatant liquid into absorption columns ,centrifuge them at 12,000xg about 30 sec and then discard the liquid in the collecting pipe.</LI><LI>Add 500 ul Buffer W1 and centrifuge them at 12,000xg about 30 sec. Then discard the liquid in the collecting pipe.</LI><LI>Add 700 ul Buffer W2, centrifuge them at 12,000xg about 30 sec and discard the liquid in the collecting pipe.</LI><LI>Do the step 8 again.</LI><LI>Centrifuge the empty columns at 12,000xg about 1 min.</LI><LI>Put the columns in clean 1.5mL EP tubes, add 80 ul Elution Buffer on the absorption film, stand 1 min and centrifuge about 1 min at 12,000xg. </LI></OL><H5>Characterization of SCVE and OprF by Gel Electrophoresis</H5><P>Only 1/4 PCR results successfully duplicated results, plasimid extraction from SCVE showed well.</P><P>Second PCR for OprF didn't work well:(</P><H5>Digestion of E0040(EGFP)</H5><P>Enzyme from Thermo Scientific FastDigest Enzyme with Green Buffer</P><P>Protocol as following:</P><UL><LI>Nuclease-free water: 15ul</LI><LI>E0040: 1ul</LI><LI>10* FastDigest Green Buffer: 2ul</LI><LI>EcoRI: 1ul</LI><LI>SpeI: 1ul</LI></UL><P>We are trying to figure out the best conditions of our experiment and whether we need loading buffer</P><TABLE><THEAD><TR><TH>Time</TH><TH>With Loading Buffer or not</TH></TR></THEAD><TBODY><TR><TD>5min</TD><TD>Yes(1), No(2)</TD></TR><TR><TD>2h</TD><TD>Yes(3), No(4)</TD></TR></TBODY></TABLE><H5>PCR of E0040(EGFP)</H5><P>Primers produced from Sangon Biotech Co.,Ltd.</P><TABLE><THEAD><TR><TH>Primer</TH><TH>Sequence(5'-3')</TH></TR></THEAD><TBODY><TR><TD>E0040-F</TD><TD>ATGCGTAAAGGAGAAGAACTTT</TD></TR><TR><TD>R0051-E0040-R</TD><TD>TACGCATATAAATTTTCTCCTCTTTGCAACCA</TD></TR></TBODY></TABLE><P>For E0040, primers including E0040-F, R0051-E0040-R, protocol as following:</P><UL><LI>2μl Taq buffer with KCl</LI><LI>0.4μl dNTPs</LI><LI>1μl E0040-F Primer</LI><LI>1μl R0051-E0040-R Primer</LI><LI>0.25μl Taq DNA Polymerase</LI><LI>2μl MgCl2</LI><LI>1μl plasmid(pSB1C3) containing E0040</LI><LI>12.35μl ddH2O</LI></UL><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)</LI><LI>Elongation for whole at 72 degree celcius 7 min</LI><LI>4 celcius degree for preserveation.</LI></OL><P><EM>Attention</EM>: R0051-E0040-R Primer is not appropriate because of only 8 complementary base pairs.</P><H5>Characterization of E0040 from Digestion and PCR </H5><P>Result illustrated that <EM>We do need loading buffer, and 5 min do work!</EM></P><P>PCR showed more concentration of E0040!</P><H5>Gel Extraction</H5><P>Preparation:</P><OL><LI>Check out whether ethyl alcohol is added into Wash Solution. </LI><LI>Check out whether Buffer B2 has sediment. </LI><LI>Adjust the thermostat water bath at 50 degree centigrade.</LI></OL><P>Procedure:</P><OL><LI>Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers. </LI><LI>Measure out the weight of tubes of gel. </LI><LI>Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins. </LI><LI>Optional: If the length of the DNA strip is shorter than 500 bp, it would be better add isopropanol one third volume of Buffer B2. </LI><LI>After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s. </LI><LI>Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution. </LI><LI>Do the step 5 again. </LI><LI>Centrifuge the empty columns at 9000xg about 1min. </LI><LI>Open the columns and air them about 10 mins. </LI><LI>Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min. </LI><LI>Keep the DNA solution and often measure the concentration of DNA and A260/A280 for further study.</LI></OL><P>Final OD results:</P><TABLE><THEAD><TR><TH>Code</TH><TH>OD(ng/ul)</TH></TR></THEAD><TBODY><TR><TD>PCR(1)</TD><TD>15.3</TD></TR><TR><TD>PCR(2)</TD><TD>4.5</TD></TR><TR><TD>PCR(3)</TD><TD>27.0</TD></TR><TR><TD>Digestion(1) with 5 mins</TD><TD>15.4</TD></TR><TR><TD>Digestion(3) with 2 h</TD><TD>22.7</TD></TR></TBODY></TABLE><P>Perseved for further use.</P></DIV><LI><TIME class="cbp_tmtime" datetime="2015-07-18 00:00"><SPAN>2015</SPAN><SPAN>18/7</SPAN></TIME><DIV class="cbp_tmlabel"><H5>Transfofmation of Three Parts</H5><P>Protocol as following</P></DIV></LI><LI>100μl (three tubes)competent cells</LI><LI>C0062(pSB1C3)</LI><LI>R0010(pSB1C3)</LI><LI>C0061(pSB1C3)</LI><LI>LB without antibiotic</LI><LI>LB medium with Amp/CmR</LI><H5>Plasmid Extraction of E0040</H5><P>Procedure including:</P><P>1.Absorb 6 mL bacteria solution in to EP tubes, centrifuge them at 12,000xg 1mins and then discard the culture medium.</P><P>2.Add 250 ul Buffer S1 into sediment, and use spearhead to make bacteria suspended.</P><P>3.Add 250 ul Buffer S2, and overturn the EP tubes 8 times immediately and tenderly. Stand tubes about 3 mins.</P><P>4.Add 350 ul Buffer W3 and overturn the EP tubes 10 times again immediately and tenderly.</P><P>5.Centrifuge tubes at 12,000xg about 10 mins.</P><P>6.Pour the supernatant liquid into absorption columns ,centrifuge them at 12,000xg about 30 sec and then discard the liquid in the collecting pipe.</P><P>7.Add 500 ul Buffer W1 and centrifuge them at 12,000xg about 30 sec. Then discard the liquid in the collecting pipe.</P><P>8.Add 700 ul Buffer W2, centrifuge them at 12,000xg about 30 sec and discard the liquid in the collecting pipe.</P><P>9.Do the step 8 again.</P><P>10.Centrifuge the empty columns at 12,000xg about 1 min.</P><P>11.Put the columns in clean 1.5mL EP tubes, add 80 ul Elution Buffer on the absorption film, stand 10 min and centrifuge about 1 min at 12,000xg. </P><P><EM>Results</EM></P><TABLE><THEAD><TR><TH>Code</TH><TH>OD(ng/ul)</TH></TR></THEAD><TBODY><TR><TD>E0040(1)</TD><TD>166.7</TD></TR><TR><TD>E0040(2)</TD><TD>169.2</TD></TR></TBODY></TABLE><H5>Colonical PCR for OprF from <EM>Psudomonus aeruginosa</EM></H5><P>PCR protocol:</P><P>For OprF, primers including OprF-F, OprF+Ter-R, protocol as following:</P><UL><LI>2μl Taq buffer with KCl</LI><LI>0.4μl dNTPs</LI><LI>1μl OprF-F Primer</LI><LI>1μl OprF+Ter-R Primer</LI><LI>0.5μl Taq DNA Polymerase(<EM>add more 0.25ul then previous protocol</EM>)</LI><LI>2μl MgCl2</LI><LI>1μl <EM>Psudomonus aeruginosa</EM> (PAO1) bacterial solution</LI><LI>12.35μl ddH2O</LI></UL><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)</LI><LI>Elongation for whole at 72 degree celcius 7 min</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>Digestion of E0040(EGFP)</H5><P>Enzyme from Thermo Scientific FastDigest Enzyme with Green Buffer</P><P>Protocol as following:</P><UL><LI>Nuclease-free water: 15ul</LI><LI>E0040: 1ul</LI><LI>10* FastDigest Green Buffer: 2ul</LI><LI>EcoRI: 1ul</LI><LI>SpeI: 1ul</LI></UL><H5>Characterization of OprF and E0040</H5><H5>Plasmid Extraction of Cas9(K1218011)</H5><P>The bacteria were cultured at 9 am from successfully transformed bacteria. </P><P>Procedure including:</P><P>1.Absorb 6 mL bacteria solution in to EP tubes, centrifuge them at 12,000xg 1mins and then discard the culture medium.</P><P>2.Add 250 ul Buffer S1 into sediment, and use spearhead to make bacteria suspended.</P><P>3.Add 250 ul Buffer S2, and overturn the EP tubes 8 times immediately and tenderly. Stand tubes about 3 mins.</P><P>4.Add 350 ul Buffer W3 and overturn the EP tubes 10 times again immediately and tenderly.</P><P>5.Centrifuge tubes at 12,000xg about 10 mins.</P><P>6.Pour the supernatant liquid into absorption columns ,centrifuge them at 12,000xg about 30 sec and then discard the liquid in the collecting pipe.</P><P>7.Add 500 ul Buffer W1 and centrifuge them at 12,000xg about 30 sec. Then discard the liquid in the collecting pipe.</P><P>8.Add 700 ul Buffer W2, centrifuge them at 12,000xg about 30 sec and discard the liquid in the collecting pipe.</P><P>9.Do the step 8 again.</P><P>10.Centrifuge the empty columns at 12,000xg about 1 min.</P><P>11.Put the columns in clean 1.5mL EP tubes, add 80 ul Elution Buffer on the absorption film, stand 10 min and centrifuge about 1 min at 12,000xg. </P><H5>Characterization of Cas9</H5></DIV><LI><TIME class="cbp_tmtime" datetime="2015-07-19 00:00"><SPAN>2015</SPAN><SPAN>19/7</SPAN></TIME><DIV class="cbp_tmlabel"><H5>Digestion of K1218011(Cas9)</H5><P>Enzyme from Thermo Scientific FastDigest Enzyme with Green Buffer</P><P>Protocol as following:</P></DIV></LI><LI>Nuclease-free water: 15ul</LI><LI>K1218011: 1ul</LI><LI>10* FastDigest Green Buffer: 2ul</LI><LI>EcoRI: 1ul</LI><LI>SpeI: 1ul</LI><P>Incubate into 37 degree celcius, 2h</P><H5>Digestion of SCVE</H5><P>Enzyme from Thermo Scientific FastDigest Enzyme with Green Buffer</P><P>Protocol as following:</P><UL><LI>Nuclease-free water: 16ul</LI><LI>SCVE: 1ul</LI><LI>10* FastDigest Green Buffer: 2ul</LI><LI>SmaI: 1ul</LI></UL><P>Incubate into 37 degree celcius, 2h</P><H5>Colonical PCR for OprF from *Psudomonus aeruginosa, micF and SoxS from K-12 </H5><P>PCR protocol:</P><P>For OprF, primers including OprF-F, OprF+Ter-R, protocol as following:</P><UL><LI>2μl Taq buffer with KCl</LI><LI>0.4μl dNTPs</LI><LI>1μl OprF-F Primer</LI><LI>1μl OprF+Ter-R Primer</LI><LI>0.25μl Taq DNA Polymerase</LI><LI>2μl MgCl2</LI><LI>1μl <EM>Psudomonus aeruginosa</EM> (PAO1) bacterial solution</LI><LI>12.35μl ddH2O</LI></UL><P>For micF, primers including micF-luxI-R, micF-F, protocol as following:</P><UL><LI>2μl Taq buffer with KCl</LI><LI>0.4μl dNTPs</LI><LI>1μl micF-luxI-R Primer</LI><LI>1μl micF-F Primer</LI><LI>0.25μl Taq DNA Polymerase</LI><LI>2μl MgCl2</LI><LI>1μl Top10(K-12) Strain <EM>E. coli</EM> bacterial solution</LI><LI>12.35μl ddH2O</LI></UL><P>For SoxS, primers including SoxS-RBS-luxI-R, BB-SoxS-F, protocol illustrates as:</P><UL><LI>2μl Taq buffer with KCl</LI><LI>0.4μl dNTPs</LI><LI>1μl SoxS-RBS-luxI-R Primer</LI><LI>1μl BB-SoxS-F Primer</LI><LI>0.25μl Taq DNA Polymerase</LI><LI>2μl MgCl2</LI><LI>1μl 1μl Top10(K-12) Strain <EM>E. coli</EM> bacterial solution</LI><LI>12.35μl ddH2O</LI></UL><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)</LI><LI>Elongation for whole at 72 degree celcius 7 min</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>Praparation for Competent Cell</H5><P>protocol as the following:</P><OL><LI>Pick out two single colonies(TG1) and cultivate bacteria in 5mL LB media at 37 degree centigrade about 12 hours with shocking. </LI><LI>Take 5mL nutrient solution into 100 mL media and cultivate bacteria at 37 degree centigrade about one hour and a half with shocking till the OD660 reaches 0.4-0.5. </LI><LI>Absorb 50 mL nutrient solution into EP tubes with ice-bath about 10 mins. </LI><LI>Centrifuge the tubes at 4100xg at 4 degree centigrade about 10mins and discard the supernatant liquid.</LI><LI>Utilize 30 mL 0.1M calcium chloride to suspend bacteria.</LI><LI>Centrifuge the tubes at 4100xg at 4 degree centigrade about 10 mins and discard the supernatant liquid. </LI><LI>Utilize 2 mL 0.1M calcium chloride with 15% glycerol to suspend bacteria and then ready for transformation quickly or restore at -70 degree centigrade.</LI></OL><TABLE><THEAD><TR><TH>Time(min)</TH><TH>OD600</TH></TR></THEAD><TBODY><TR><TD>40</TD><TD>0.195</TD></TR><TR><TD>60</TD><TD>0.203</TD></TR><TR><TD>80</TD><TD>0.309</TD></TR><TR><TD>90</TD><TD>0.439</TD></TR></TBODY></TABLE><P>PS:dilute the solution by adding 100 mL LB media after first test.</P></DIV><LI><TIME class="cbp_tmtime" datetime="2015-07-20 00:00"><SPAN>2015</SPAN><SPAN>20/7</SPAN></TIME><DIV class="cbp_tmlabel"><H5>Negative Chemotaxis Characterization</H5><P>term 1, material:</P></DIV></LI><LI>LB solid medium plates x4</LI><LI>1g/ml Chloramphenicol</LI><LI>E.coli Top10</LI><LI>filter paper</LI><H5>Plasmid Extraction of EmrE and ArcB</H5><P>done by Wang Luyao</P><P>Procedure including:</P><OL><LI>Absorb 6 mL bacteria solution in to EP tubes, centrifuge them at 12,000xg 1mins and then discard the culture medium.</LI><LI>Add 250 ul Buffer S1 into sediment, and use spearhead to make bacteria suspended.</LI><LI>Add 250 ul Buffer S2, and overturn the EP tubes 8 times immediately and tenderly. Stand tubes about 3 mins.</LI><LI>Add 350 ul Buffer W3 and overturn the EP tubes 10 times again immediately and tenderly.</LI><LI>Centrifuge tubes at 12,000xg about 10 mins.</LI><LI>Pour the supernatant liquid into absorption columns ,centrifuge them at 12,000xg about 30 sec and then discard the liquid in the collecting pipe.</LI><LI>Add 500 ul Buffer W1 and centrifuge them at 12,000xg about 30 sec. Then discard the liquid in the collecting pipe.</LI><LI>Add 700 ul Buffer W2, centrifuge them at 12,000xg about 30 sec and discard the liquid in the collecting pipe.</LI><LI>Do the step 8 again.</LI><LI>Centrifuge the empty columns at 12,000xg about 1 min.</LI><LI>Put the columns in clean 1.5mL EP tubes, add 80 ul Elution Buffer on the absorption film, stand 10 min and centrifuge about 1 min at 12,000xg. </LI></OL><P>the concentration as following</P><P>EmrE 70.2 ng/ul 92.3 ng/ul</P><P>ArcB 62.6 ng/ul 95.2 ng/ul</P><H5>Transformation of R0051 and C0061</H5><P>the source of parts:</P><P>No.1 R0051 2014 kit plate 4</P><P>No.2 R0051 2014 kit plate 4</P><P>No.3 control</P><P>No.4 C0061 2015 kit plate 4</P><P>No.5 C0061 2015 kit plate 4</P><P>No.6 C0061 2014 kit plate 4</P><P>No.1 and No.4 the competent cell is from lab319</P><P>No.2 and No.5 the competent cell is made by us (firt)</P><P>No.3 and No.6 the competent cell is made by us (second)</P><P>the protocol as following</P><OL><LI>Take the competent bacteria from refrigerator and incubate them into ice about 10 mins until it is dissolved</LI><LI>Absorb 1 uL or 2 uL plasmid and mix it with bacteria solution thoroughly.</LI><LI>Put the tubes on the ice about 30 mins.</LI><LI>Make a heat shock at 42 degree centigrade about 60 sec</LI><LI>Put the tubes on the ice about 3 mins again.</LI><LI>Add 900 ul LB medium into EP tubes and cultivate the bacteria at 37 degree centrigrade about 60 min.</LI><LI>Centrifuge them at 4,000xg about 15 sec and we will se sediment in the tubes.</LI><LI>Discard the supernatant liquid and leave about 100ul medium.</LI><LI>Coat plate: add 100ul solution in a plate</LI></OL><H5>PCR of OrpF</H5><P>material:</P><UL><LI>13 uL Taq Master Mix </LI><LI>11 uL dd water</LI><LI>1 uL PAO1 bacterial</LI><LI>0.5 uL primer Fd(OrpF-F)</LI><LI>0.5 uL primer Rv(OrpF+Ter-R)
        PS:Taq Master Mix is from lab 319 ,and this reagent is used for bacterial PCR.</LI></UL><P>Renaturation at 50℃, extend for 1:10</P></DIV><LI><TIME class="cbp_tmtime" datetime="2015-07-21 00:00"><SPAN>2015</SPAN><SPAN>21/7</SPAN></TIME><DIV class="cbp_tmlabel"><H5>PCR of OprF</H5><P>material:</P><P>No.1 and No.2:</P></DIV></LI><LI>6.5 uL Taq Master Mix</LI><LI>1 uL PAO1 bacterial solution</LI><LI>4.5 uL dd water</LI><LI>0.5 uL OprF-F</LI><LI>0.5 uL OprF+Ter-R</LI><P>No.3 and No.4:</P><UL><LI>6.5 uL Taq Master Mix</LI><LI>4 uL OprF PCR product</LI><LI>1.5 uL dd water </LI><LI>0.5 uL OprF-F</LI><LI>0.5 uL OprF+Ter-R</LI></UL><P>PS:Taq Master Mix is from lab 319 ,and this reagent is used for bacterial PCR.</P><P>Renaturation at 55℃, extend for 1:10</P><H5>OprF for PCR  and gel electrophoresisOprF</H5><P>analyze: the length of the product is </P></DIV><LI><TIME class="cbp_tmtime" datetime="2015-07-22 00:00"><SPAN>2015</SPAN><SPAN>22/7</SPAN></TIME><DIV class="cbp_tmlabel"><H5>Preparation of Semi-Solid LB Medium</H5><P>In order to characterization the movement of TOP10, semi-solid LB medium preparation is indispensable in this research.</P><P> (500mL Protocol)</P><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Mass</TH><TH>Proportion(g/mL)</TH></TR></THEAD><TBODY><TR><TD>Yeast Extract</TD><TD>2.5g</TD><TD>0.5%</TD></TR><TR><TD>Trtptone</TD><TD>5g</TD><TD>1%</TD></TR><TR><TD>NaCl</TD><TD>5g</TD><TD>1%</TD></TR><TR><TD>Agar</TD><TD>1.25g</TD><TD>0.25%</TD></TR></TBODY></TABLE><P>Compared with solid LB medium, containing 1.5% agar in there.</P><H5>Plasmid Extraction C0051,B0015 and R0062</H5><P>Procedure including:</P></DIV></LI><LI>Absorb 6 mL bacteria solution in to EP tubes, centrifuge them at 12,000xg 1mins and then discard the culture medium.</LI><LI>Add 250 ul Buffer S1 into sediment, and use spearhead to make bacteria suspended.</LI><LI>Add 250 ul Buffer S2, and overturn the EP tubes 8 times immediately and tenderly. Stand tubes about 3 mins.</LI><LI>Add 350 ul Buffer W3 and overturn the EP tubes 10 times again immediately and tenderly.</LI><LI>Centrifuge tubes at 12,000xg about 10 mins.</LI><LI>Pour the supernatant liquid into absorption columns ,centrifuge them at 12,000xg about 30 sec and then discard the liquid in the collecting pipe.</LI><LI>Add 500 ul Buffer W1 and centrifuge them at 12,000xg about 30 sec. Then discard the liquid in the collecting pipe.</LI><LI>Add 700 ul Buffer W2, centrifuge them at 12,000xg about 30 sec and discard the liquid in the collecting pipe.</LI><LI>Do the step 8 again.</LI><LI>Centrifuge the empty columns at 12,000xg about 1 min.</LI><LI>Put the columns in clean 1.5mL EP tubes, add 80 ul Elution Buffer on the absorption film, stand 10 min and centrifuge about 1 min at 12,000xg. </LI><P>the concentration as following</P><P>C0015:171.7 ng/uL</P><P>B0015:93 ng/uL</P><P>R0062:144.3 ng/uL</P><H5>PCR of C0051,B0015 and R0062</H5><P>material:</P><P>C0051 plasmid(171.7 ng/uL)</P><P>B0015 plasmid(93 ng/uL)</P><P>R0062 plasmid(144.3 ng/uL)</P><P>all the solutions are diluted 10-folds </P><P>Primers produced from Sangon Biotech Co.,Ltd.</P><TABLE><THEAD><TR><TH>Primers</TH><TH>Sequence(5' to 3')</TH></TR></THEAD><TBODY><TR><TD>C0051-F(I)</TD><TD>att tat atg agc aaa aag aaa cca tta tca</TD></TR><TR><TD>C0051-B0015-R(I)</TD><TD>tat ttg atg cct ggc tct agt gat cta cac tag cac ta</TD></TR><TR><TD>B0015-R(I)</TD><TD>gcc gct act agt taa acg cag aaa ggc cca cc</TD></TR><TR><TD>B0015-F</TD><TD>cca ggc atc aaa taa aac gaa agg</TD></TR><TR><TD>R0062-F</TD><TD>gcg gcc gct tct aga gac ctg tag gat cg</TD></TR><TR><TD>R0062-C0051-R</TD><TD>tgt gct cat ata aat ttt ctc ctc ttt cta gat ttt att cga c</TD></TR></TBODY></TABLE><P>No.1 and No.2 for C0051(803bp)</P><P>Protocol as this:</P><UL><LI>2.5 μl Taq buffer with KCl</LI><LI>2.5 μl dNTPs</LI><LI>0.75 μl C0051-F Primer</LI><LI>0.75 μl C0051-B0015-R(I) Primer</LI><LI>0.5 μl Taq DNA Polymerase</LI><LI>1.0 μl MgCl2</LI><LI>2 uL C0051 solution</LI><LI>15 μl ddH2O</LI></UL><P>No.3 and No.4 for B0015(142bp)</P><P>Protocol as this:</P><UL><LI>2.5 μl Taq buffer with KCl</LI><LI>2.5 μl dNTPs</LI><LI>0.75 μl B0015-F Primer</LI><LI>0.75 μl B0015-R(I) Primer</LI><LI>0.5 μl Taq DNA Polymerase</LI><LI>1.0 μl MgCl2</LI><LI>3 uL C0051 solution</LI><LI>14 μl ddH2O</LI></UL><P>No.5 and No.6 for R0062(104bp)</P><P>Protocol as this:</P><UL><LI>2.5 μl Taq buffer with KCl</LI><LI>2.5 μl dNTPs</LI><LI>0.75 μl R0062-F Primer</LI><LI>0.75 μl R0062-C0051-R Primer</LI><LI>0.5 μl Taq DNA Polymerase</LI><LI>1.0 μl MgCl2</LI><LI>2 uL R0062 solution</LI><LI>15 μl ddH2O</LI></UL><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)</LI><LI>Elongation for whole at 72 degree celcius 2min(C0061)or 1min(B0015 and R0062)</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>Preparation Semi-Solid M63 Medium</H5><P>In order to characterization the movement of TOP10, semi-solid M63 medium preparation is indispensable in this research.</P><P> (500mL Protocol) </P><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Mass</TH><TH>Proportion(g/mL) or Mole</TH></TR></THEAD><TBODY><TR><TD>Yeast Extract</TD><TD>2.5g</TD><TD>0.5%</TD></TR><TR><TD>Trtptone</TD><TD>5g</TD><TD>1%</TD></TR><TR><TD>NaCl</TD><TD>5g</TD><TD>1%</TD></TR><TR><TD>Agar</TD><TD>1.25g</TD><TD>0.25%</TD></TR><TR><TD>KH2PO4</TD><TD>6.8g</TD><TD>50mM</TD></TR><TR><TD>KOH</TD><TD>2.1g</TD><TD>37.5mM</TD></TR><TR><TD>(NH4)2SO4</TD><TD>0.09g</TD><TD>7.5mM</TD></TR><TR><TD>MgSO4</TD><TD>0.06g</TD><TD>0.5mM</TD></TR><TR><TD>FeSO4·7H2O</TD><TD>0.2710g</TD><TD>1.95mM</TD></TR><TR><TD>D-Glucose</TD><TD>1.98g</TD><TD>11mM</TD></TR></TBODY></TABLE><H5>Colonical PCR for OprF from <EM>Psudomonus aeruginosa</EM> and SCVE from pUC57</H5><P>PCR protocol:</P><P>For OprF, primers including OprF-F, OprF+Ter-R, protocol as following:</P><UL><LI>2μl Taq buffer with KCl</LI><LI>0.4μl dNTPs</LI><LI>1μl OprF-F Primer</LI><LI>1μl OprF+Ter-R Primer</LI><LI>0.25μl Taq DNA Polymerase</LI><LI>2μl MgCl2</LI><LI>1μl <EM>Psudomonus aeruginosa</EM> (PAO1) bacterial solution</LI><LI>12.35μl ddH2O</LI></UL><P>For SCVE, Primers produced from Sangon Biotech Co.,Ltd.</P><TABLE><THEAD><TR><TH>Primer</TH><TH>Sequence(5'-3')</TH></TR></THEAD><TBODY><TR><TD>SCVE-T-R</TD><TD>TTGATGCCTGGCTCTAGTACACCAGCAGATCCGG</TD></TR><TR><TD>T7-RBS-SCVE-F</TD><TD>TTTTTGCATACTAGAGAAAGAGGAGAAAATTTATATGTATAGCTTTGTG</TD></TR></TBODY></TABLE><P>Protocol as this:</P><UL><LI>2μl Taq buffer with KCl</LI><LI>0.4μl dNTPs</LI><LI>1μl SCVE-T-R Primer</LI><LI>1μl T7-RBS-SCVE-F Primer</LI><LI>0.25μl Taq DNA Polymerase</LI><LI>2μl MgCl2</LI><LI>1μl SCVE solution from plasmid extraction</LI><LI>12.35μl ddH2O</LI></UL><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)</LI><LI>Elongation for whole at 72 degree celcius 7 min</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>Gel Electrophoresis of OprF, SCVE micF and SoxS</H5><P>micF and SoxS</P><P>OprF and SCVE</P><H5>Gel Extraction</H5><P>Material including: OprF, SoxS, micF</P><P>Preparation:</P><OL><LI>Check out whether ethyl alcohol is added into Wash Solution. </LI><LI>Check out whether Buffer B2 has sediment. </LI><LI>Adjust the thermostat water bath at 50 degree centigrade.</LI></OL><P>Procedure:</P><OL><LI>Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers. </LI><LI>Measure out the weight of tubes of gel. </LI><LI>Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins. </LI><LI>Optional: If the length of the DNA strip is shorter than 500 bp, it would be better add isopropanol one third volume of Buffer B2. </LI><LI>After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s. </LI><LI>Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution. </LI><LI>Do the step 5 again. </LI><LI>Centrifuge the empty columns at 9000xg about 1min. </LI><LI>Open the columns and air them about 10 mins. </LI><LI>Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min. </LI><LI>Keep the DNA solution and often measure the concentration of DNA and A260/A280 for further study.</LI></OL><LI><TIME class="cbp_tmtime" datetime="2015-07-23 00:00"><SPAN>2015</SPAN><SPAN>23/7</SPAN></TIME><DIV class="cbp_tmlabel"><H5>Plasmid Extraction of R0062</H5><P>Procedure including:</P></DIV></LI><LI>Absorb 6 mL bacteria solution in to EP tubes, centrifuge them at 12,000xg 1mins and then discard the culture medium.</LI><LI>Add 250 ul Buffer S1 into sediment, and use spearhead to make bacteria suspended.</LI><LI>Add 250 ul Buffer S2, and overturn the EP tubes 8 times immediately and tenderly. Stand tubes about 3 mins.</LI><LI>Add 350 ul Buffer W3 and overturn the EP tubes 10 times again immediately and tenderly.</LI><LI>Centrifuge tubes at 12,000xg about 10 mins.</LI><LI>Pour the supernatant liquid into absorption columns ,centrifuge them at 12,000xg about 30 sec and then discard the liquid in the collecting pipe.</LI><LI>Add 500 ul Buffer W1 and centrifuge them at 12,000xg about 30 sec. Then discard the liquid in the collecting pipe.</LI><LI>Add 700 ul Buffer W2, centrifuge them at 12,000xg about 30 sec and discard the liquid in the collecting pipe.</LI><LI>Do the step 8 again.</LI><LI>Centrifuge the empty columns at 12,000xg about 1 min.</LI><LI>Put the columns in clean 1.5mL EP tubes, add 80 ul Elution Buffer on the absorption film, stand 10 min and centrifuge about 1 min at 12,000xg. </LI><H5>PCR for OprF and C0061</H5><P>PCR protocol:</P><P>For OprF, primers including OprF-F, OprF+Ter-R, protocol as following:</P><UL><LI>2μl Taq buffer with KCl</LI><LI>0.4μl dNTPs</LI><LI>1μl OprF-F Primer</LI><LI>1μl OprF+Ter-R Primer</LI><LI>0.25μl Taq DNA Polymerase</LI><LI>2μl MgCl2</LI><LI>1μl <EM>Psudomonus aeruginosa</EM> (PAO1) bacterial solution</LI><LI>12.35μl ddH2O</LI></UL><P>For SCVE, Primers produced from Sangon Biotech Co.,Ltd.</P><TABLE><THEAD><TR><TH>Primer</TH><TH>Sequence(5'-3')</TH></TR></THEAD><TBODY><TR><TD>C0061-F</TD><TD>ATGACTATAATGATAAAAAAATCGGATTTTTTGGCA</TD></TR><TR><TD>C0061-B0015-R</TD><TD>ATTTGATGCCTGGGAGATCTACACTAGC</TD></TR></TBODY></TABLE><P>Protocol as this:</P><UL><LI>2μl Taq buffer with KCl</LI><LI>0.4μl dNTPs</LI><LI>1μl C0061-F Primer</LI><LI>1μl C0061-B0015-R Primer</LI><LI>0.25μl Taq DNA Polymerase</LI><LI>2μl MgCl2</LI><LI>1μl C0061 solution</LI><LI>12.35μl ddH2O</LI></UL><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)</LI><LI>Elongation for whole at 72 degree celcius 7 min</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>Gel Extraction of C0062,B0015 and R0062</H5><P>Material including: C0051,B0015 and R0062</P><P>Preparation:</P><OL><LI>Check out whether ethyl alcohol is added into Wash Solution. </LI><LI>Check out whether Buffer B2 has sediment. </LI><LI>Adjust the thermostat water bath at 50 degree centigrade.</LI></OL><P>Procedure:</P><OL><LI>Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers. </LI><LI>Measure out the weight of tubes of gel. </LI><LI>Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins. </LI><LI>Optional: If the length of the DNA strip is shorter than 500 bp, it would be better add isopropanol one third volume of Buffer B2. </LI><LI>After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s. </LI><LI>Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution. </LI><LI>Do the step 5 again. </LI><LI>Centrifuge the empty columns at 9000xg about 1min. </LI><LI>Open the columns and air them about 10 mins. </LI><LI>Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min. </LI><LI>Keep the DNA solution and often measure the concentration of DNA and A260/A280 for further study.</LI></OL><P>the concentration as following:</P><P>C0051:12.7 ng/uL</P><P>B0015:7.3 ng/uL</P><P>R0062:7.7 ng/uL</P><H5>PCR of OprF</H5><P>material:</P><UL><LI>13 uL Taq Master Mix </LI><LI>11 uL dd water</LI><LI>1 uL PAO1 bacterial</LI><LI>0.5 uL primer Fd(OrpF-F)</LI><LI>0.5 uL primer Rv(OrpF+Ter-R)</LI></UL><P>Renaturation at 55℃, extend for 2min</P><H5>Gel Electrophoresis Characterization</H5><P>C0051 and B0015</P><P>OprF</P><P>That was relatively successully extracted OprF from colonical PCR.</P><P>R0062</P><LI><TIME class="cbp_tmtime" datetime="2015-07-24 00:00"><SPAN>2015</SPAN><SPAN>24/7</SPAN></TIME><DIV class="cbp_tmlabel"><H5>Preparation for Homologous Recombination: Plasimid Digestion</H5><P>Enzyme from <EM>Thermo Scientific FastDigest</EM> Enzyme with 10X Green Buffer</P><P>Protocol as following:</P></DIV></LI><LI>Nuclease-free water: 15ul</LI><LI>pSB1C3: 1ul</LI><LI>10* FastDigest Green Buffer: 2ul</LI><LI>EcoRI: 1ul</LI><LI>SpeI: 1ul</LI><P>Incubate solution at 37 degree celcius about 2h.</P><H5>PCR of C0061,B0015 and micF</H5><P>material:</P><P>C0061 plasmid: 477.1 ng/uL(diluted 20-folds)</P><P>B0015 PCR product:7.3 ng/uL</P><P>micF-1 PCR product:13.4 ng/uL</P><P>ForB0015 Primers produced from Sangon Biotech Co.,Ltd.</P><TABLE><THEAD><TR><TH>Primer</TH><TH>Sequence(5'-3')</TH></TR></THEAD><TBODY><TR><TD>B0015-F</TD><TD>cca ggc atc aaa taa aac gaa agg</TD></TR><TR><TD>B0015-R(I)</TD><TD>gcc gct act agt ata taa acg cag aaa ggc cca cc</TD></TR></TBODY></TABLE><P>protocol:</P><UL><LI>2μl Taq buffer with KCl</LI><LI>0.4μl dNTPs</LI><LI>1μl Primer-Fd</LI><LI>1μl Primer-Rv</LI><LI>0.25μl Taq DNA Polymerase</LI><LI>2μl MgCl2</LI><LI>1uL C0061 plasmid,4uL B0015 plasmid,2uL plasmid</LI><LI>(13.35-X)μl ddH2O</LI></UL><P>Renaturation at 55℃, extend for 1min</P><LI><TIME class="cbp_tmtime" datetime="2015-07-25 00:00"><SPAN>2015</SPAN><SPAN>25/7</SPAN></TIME><DIV class="cbp_tmlabel"><H5>PCR of C0061,B0015,R0062,R0010 and C0062 </H5><P>material:</P><P>C0061 plasmid:477.1 ng/uL(0.2 uL)</P><P>B0015 plasmid:19 ng/uL(3 uL)</P><P>R0062 plasmid:143.3 ng/uL(0.5 uL)</P><P>C0051 plasmid:171.7 ng/uL(0.5 uL)</P><P>R0010 plasmid:357.7 ng/uL(diluted 10-folds,1 uL)</P><P>C0062 plasmid:739.4 ng/uL(diluted 10-folds,1uL)</P><P> for R0062.R0010 and C0062Primers produced from Sangon Biotech Co.,Ltd.</P><TABLE><THEAD><TR><TH>Primer</TH><TH>Sequence(5'-3')</TH></TR></THEAD><TBODY><TR><TD>R0062-F</TD><TD>gcg gcc gct tct aga ctg tag gat cg</TD></TR><TR><TD>R0062-C0051-R</TD><TD>tgt gct cat ata aat ttt ctc ttt cta gat ttt att cga c</TD></TR><TR><TD>backbone-R0010-F</TD><TD>ccgctt cta gag cca tac gca aac cgc ctc tc</TD></TR><TR><TD>C0062-B0034-R</TD><TD>cta gtg tc agt aat aaa ttt tct cct ctt tg tgt gaa att gt</TD></TR><TR><TD>B0034-C0062-F</TD><TD>tat tac tga gca cta gat gaa aaa cat aaa tgc cga c</TD></TR><TR><TD>B0015-C0062-R</TD><TD>atg cct vggc tct agt gat cta cac tag cac t</TD></TR></TBODY></TABLE><P>protocol:</P></DIV></LI><LI>2μl Taq buffer with KCl</LI><LI>0.4μl dNTPs</LI><LI>1μl Primer-Fd</LI><LI>1μl Primer-Rv</LI><LI>0.25μl Taq DNA Polymerase</LI><LI>2μl MgCl2</LI><LI>XuL plasmid</LI><LI>(13.35-X)μl ddH2O</LI><P>Renaturation at 55℃, extend for 1min</P><P>PS:</P><P>C0061:656bp B0015:142bp R0062:104bp</P><P>C0051:803bp C0062:813bp R0010:243bp</P><H5>Characterization of micF and SoxS by Gel Electrophoresis</H5><P>micF were successfully PCRed.</P><H5>Characterization of pSB1C3 and SoxS by Gel Electrophoresis</H5><LI><TIME class="cbp_tmtime" datetime="2015-07-26 00:00"><SPAN>2015</SPAN><SPAN>26/7</SPAN></TIME><DIV class="cbp_tmlabel"><H5>Characterization of R0051, C0061 and C0051 by Gel Electrophoresis</H5><H5>Plasimid Extraction for pSB1C3 and C0061</H5><P>Procedure including:</P><P>1.Absorb 6 mL bacteria solution in to EP tubes, centrifuge them at 12,000xg 1mins and then discard the culture medium.</P><P>2.Add 250 ul Buffer S1 into sediment, and use spearhead to make bacteria suspended.</P><P>3.Add 250 ul Buffer S2, and overturn the EP tubes 8 times immediately and tenderly. Stand tubes about 3 mins.</P><P>4.Add 350 ul Buffer W3 and overturn the EP tubes 10 times again immediately and tenderly.</P><P>5.Centrifuge tubes at 12,000xg about 10 mins.</P><P>6.Pour the supernatant liquid into absorption columns ,centrifuge them at 12,000xg about 30 sec and then discard the liquid in the collecting pipe.</P><P>7.Add 500 ul Buffer W1 and centrifuge them at 12,000xg about 30 sec. Then discard the liquid in the collecting pipe.</P><P>8.Add 700 ul Buffer W2, centrifuge them at 12,000xg about 30 sec and discard the liquid in the collecting pipe.</P><P>9.Do the step 8 again.</P><P>10.Centrifuge the empty columns at 12,000xg about 1 min.</P><P>11.Put the columns in clean 1.5mL EP tubes, add 80 ul Elution Buffer on the absorption film, stand 10 min and centrifuge about 1 min at 12,000xg. </P><H5>Gel Extraction</H5><P>Material including: R0051, C0051 and C0061</P><P>Preparation:</P></DIV></LI><LI>Check out whether ethyl alcohol is added into Wash Solution. </LI><LI>Check out whether Buffer B2 has sediment. </LI><LI>Adjust the thermostat water bath at 50 degree centigrade.</LI><P>Procedure:</P><OL><LI>Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers. </LI><LI>Measure out the weight of tubes of gel. </LI><LI>Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins. </LI><LI>Optional: If the length of the DNA strip is shorter than 500 bp, it would be better add isopropanol one third volume of Buffer B2. </LI><LI>After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s. </LI><LI>Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution. </LI><LI>Do the step 5 again. </LI><LI>Centrifuge the empty columns at 9000xg about 1min. </LI><LI>Open the columns and air them about 10 mins. </LI><LI>Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min. </LI><LI>Keep the DNA solution and often measure the concentration of DNA and A260/A280 for further study.</LI></OL><H5>Characterization of B0015 and R0051 by Gel Electrophoresis</H5><LI><TIME class="cbp_tmtime" datetime="2015-07-27 00:00"><SPAN>2015</SPAN><SPAN>27/7</SPAN></TIME><DIV class="cbp_tmlabel"><H5>Overlap Circuit</H5><P>This sensing circuit including promoter micF and SoxS.</P><H5>Overlapping Fragments</H5><P>In order to ligate the fragements micF or SoxS, C0061 and B0015, we need to make the equal mole proportion.</P><P>The mole of DNA molecule is calculated in the following ways:</P><P>among which, c is the concentration of DNA, V is the volumn required and n is the length of DNA. Because of the same molar mass of As, Gs, Cs and Ts, computation of same molar mass is omitted.</P><P>According to our sample extracted from PCR and calculation, ingradients characterize as below:</P><TABLE><THEAD><TR><TH>Name</TH><TH>Concentration(ng/uL)</TH><TH>Length(bp)</TH></TR></THEAD><TBODY><TR><TD>micF</TD><TD>12.6</TD><TD>358</TD></TR><TR><TD>SoxS</TD><TD>27.5</TD><TD>705</TD></TR><TR><TD>C0061(LuxI)</TD><TD>18.5</TD><TD>656</TD></TR><TR><TD>B0015(Terminator)</TD><TD>9.5</TD><TD>143</TD></TR><TR><TD>Total</TD><TD>1161</TD></TR></TBODY></TABLE><P>The theoratical volumn proportion is: </P></DIV></LI><LI>SoxS-C0061-B0015: 5: 7: 3</LI><LI>micF-C0061-B0015: 5.5: 7: 3</LI><P>Firstly, to ligate these parts, we need 10 cycles without primer:</P><P>for circuit SoxS-C0061-B0015:</P><TABLE><THEAD><TR><TH>Name</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>SoxS</TD><TD>5</TD></TR><TR><TD>C0061</TD><TD>7</TD></TR><TR><TD>B0015</TD><TD>3</TD></TR><TR><TD>Taq Polymerase</TD><TD>0.25</TD></TR><TR><TD>MgCl2</TD><TD>2</TD></TR><TR><TD>dNTP</TD><TD>0.4</TD></TR><TR><TD>10X buffer with KCl</TD><TD>2</TD></TR><TR><TD>ddH2O</TD><TD>0.35</TD></TR></TBODY></TABLE><P>for circuit micF-C0061-B0015:</P><TABLE><THEAD><TR><TH>Name</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>micF</TD><TD>5.5</TD></TR><TR><TD>C0061</TD><TD>7</TD></TR><TR><TD>B0015</TD><TD>3</TD></TR><TR><TD>Taq Polymerase</TD><TD>0.25</TD></TR><TR><TD>MgCl2</TD><TD>2</TD></TR><TR><TD>dNTP</TD><TD>0.4</TD></TR><TR><TD>10X buffer with KCl</TD><TD>2</TD></TR></TBODY></TABLE><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>10 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)</LI><LI>Elongation for whole at 72 degree celcius 1min</LI><LI>4 celcius degree for preserveation.</LI></OL><P>Then, we extracted 5 ul as template, and 15 ul adding with primer:</P><P>Primers were synthesized from <EM>Sangon Biotech. Co.,Ltd.</EM></P><TABLE><THEAD><TR><TH>Primers</TH><TH>Sequence(5' to 3')</TH></TR></THEAD><TBODY><TR><TD>micF-F</TD><TD>GCGGCCGCTTCTAGAGCCTCATTAATCAGTCGGC</TD></TR><TR><TD>B0015-R</TD><TD>GCCGCTACTAGTATATAAACGCAG</TD></TR><TR><TD>BB-SoxS-F</TD><TD>CGGCCGCTTCTAGAGCGTGCGGAACATTCG</TD></TR></TBODY></TABLE><P>5ul-template protocol:</P><TABLE><THEAD><TR><TH>Name</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>Template</TD><TD>5</TD></TR><TR><TD>Taq Polymerase</TD><TD>0.25</TD></TR><TR><TD>MgCl2</TD><TD>2</TD></TR><TR><TD>dNTP</TD><TD>0.4</TD></TR><TR><TD>micF-F(BB-SoxS-F)</TD><TD>1</TD></TR><TR><TD>B0015-R</TD><TD>1</TD></TR><TR><TD>10X buffer with KCl</TD><TD>2</TD></TR><TR><TD>ddH2O</TD><TD>8.35</TD></TR></TBODY></TABLE><P>15ul-protocol:</P><TABLE><THEAD><TR><TH>Name</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>Solution</TD><TD>15</TD></TR><TR><TD>micF-F(BB-SoxS-F)</TD><TD>1</TD></TR><TR><TD>B0015-R</TD><TD>1</TD></TR><TR><TD>10X buffer with KCl</TD><TD>2</TD></TR><TR><TD>ddH2O</TD><TD>1</TD></TR></TBODY></TABLE><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)</LI><LI>Elongation for whole at 72 degree celcius 1min</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>PCR of Five Parts</H5><P>material:</P><P>R0010 plasmid:193.1ng/uL</P><P>C0062 plasmid:952.5ng/uL</P><P>B0015 plasmid:93ng/uL</P><P>R0062 plasmid：143.3ng/uL</P><P>For R0051 Primers produced from Sangon Biotech Co.,Ltd.</P><P>protocol:</P><UL><LI>2uL 10X buffer with KCl</LI><LI>0.4uL dNTP</LI><LI>1uL primers-F</LI><LI>1uL primers-R</LI><LI>0.25uL Taq Polymerase</LI><LI>2uL MgCl2</LI><LI>1uL plasmid</LI><LI>12.35 uL ddH2O</LI></UL><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)</LI><LI>Elongation for whole at 72 degree celcius 1min</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>Overlap PCR of bacteria III</H5><P>material:</P><UL><LI>R0062 PCR product:21.2 ng/uL</LI><LI>C0051 PCR product:22.8 ng/uL</LI><LI>B0015 PCR product:24.6 ng/uL</LI></UL><P>protocol:</P><P>total volume 20 uL </P><UL><LI>R0062(104bp) 1 uL</LI><LI>C0051(803bp) 8 uL</LI><LI>B0015(153bp) 1.5uL</LI><LI>ddH2O 4.85 uL</LI><LI>buffer 2uL</LI><LI>dNTP 0.4uL</LI><LI>Taq polymerase 0.25uL</LI><LI>MgCl2 2uL</LI></UL><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>10 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)</LI><LI>Elongation for whole at 72 degree celcius 1min</LI><LI>4 celcius degree for preserveation.</LI></OL><P>After the 10 cycles,we divide the 20 ul system into two parts:5ul and 15ul.The 5ul PCR product will be used as template in another new 20ul system.On the other hand, add primers</P><P>(R0062-F primer 1uL;C0062-B0015-R(I) primer 1uL)to the remaining 15 uL PCR product.</P><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)</LI><LI>Elongation for whole at 72 degree celcius 1min</LI><LI>4 celcius degree for preserveation.</LI></OL><P>Result:</P><P> we don't get the product we want.We guess the templates are not the right products,maybe they have mutations .So we decide to do the overlap PCR of bateriaI to test whether the method is suitable.</P><LI><TIME class="cbp_tmtime" datetime="2015-07-28 00:00"><SPAN>2015</SPAN><SPAN>28/7</SPAN></TIME><DIV class="cbp_tmlabel"><H5>Plasmid Extraction of gRNA-AcrB and gRNA-EmrE</H5><P>Procedure including:</P></DIV></LI><LI>1.Absorb 6 mL bacteria solution in to EP tubes, centrifuge them at 12,000xg 1mins and then discard the culture medium.</LI><LI>2.Add 250 ul Buffer S1 into sediment, and use spearhead to make bacteria suspended.</LI><LI>3.Add 250 ul Buffer S2, and overturn the EP tubes 8 times immediately and tenderly. Stand tubes about 3 mins.</LI><LI>4.Add 350 ul Buffer W3 and overturn the EP tubes 10 times again immediately and tenderly.</LI><LI>5.Centrifuge tubes at 12,000xg about 10 mins.</LI><LI>6.Pour the supernatant liquid into absorption columns ,centrifuge them at 12,000xg about 30 sec and then discard the liquid in the collecting pipe.</LI><LI>7.Add 500 ul Buffer W1 and centrifuge them at 12,000xg about 30 sec. Then discard the liquid in the collecting pipe.</LI><LI>8.Add 700 ul Buffer W2, centrifuge them at 12,000xg about 30 sec and discard the liquid in the collecting pipe.</LI><LI>9.Do the step 8 again.</LI><LI>10.Centrifuge the empty columns at 12,000xg about 1 min.</LI><LI>11.Put the columns in clean 1.5mL EP tubes, add 80 ul Elution Buffer on the absorption film, stand 10 min and centrifuge about 1 min at 12,000xg. </LI><H5>Gel Extraction</H5><P>Material including: R0061</P><H5><EM>Case 1: Sangon Biotech.</EM></H5><P>Preparation:</P><OL><LI>Check out whether ethyl alcohol is added into Wash Solution. </LI><LI>Check out whether Buffer B2 has sediment. </LI><LI>Adjust the thermostat water bath at 50 degree centigrade.</LI></OL><P>Procedure:</P><OL><LI>Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers. </LI><LI>Measure out the weight of tubes of gel. </LI><LI>Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins. </LI><LI>Optional: If the length of the DNA strip is shorter than 500 bp, it would be better add isopropanol one third volume of Buffer B2. </LI><LI>After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s. </LI><LI>Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution. </LI><LI>Do the step 5 again. </LI><LI>Centrifuge the empty columns at 9000xg about 1min. </LI><LI>Open the columns and air them about 10 mins. </LI><LI>Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min. </LI><LI>Keep the DNA solution and often measure the concentration of DNA and A260/A280 for further study.</LI></OL><H5><EM>Case 2: Axygen</EM></H5><P>Procedure:</P><OL><LI>Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers. </LI><LI>Measure out the weight of tubes of gel. </LI><LI>Add 400~600 ul Buffer DE-A and then incubate the EP tubes in thermostat water bath about 75 degree centigrade within 10 mins. </LI><LI>After gel is all dissolved, add 1/3 volumn of Buffer DE-B into EP tubes. The solution will turn yellow, illustrating complete dissolved.</LI><LI>Pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s. </LI><LI>Add 500 ul Buffer W1 in a centrifuge at 9000Xg about 30s and pour out the solution. </LI><LI>Add 700 ul Buffer W2 in a centrifuge at 9000Xg about 30s and pour out the solution. </LI><LI>Centrifuge the empty columns at 9000xg about 1min. </LI><LI>Open the columns and air them about 10 mins. </LI><LI>Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min. </LI><LI>Keep the DNA solution and often measure the concentration of DNA and A260/A280 for further study.</LI></OL><H5>Overlapping Fragments</H5><P>We conduct the overlapping again.</P><P>However, the sample has a little difference. The concentration of micF changes to 58.7ng/uL. According to our sample extracted from PCR and calculation, ingradients characterize as below:</P><TABLE><THEAD><TR><TH>Name</TH><TH>Concentration(ng/uL)</TH><TH>Length(bp)</TH></TR></THEAD><TBODY><TR><TD>micF</TD><TD>57.8</TD><TD>358</TD></TR><TR><TD>SoxS</TD><TD>27.5</TD><TD>705</TD></TR><TR><TD>C0061(LuxI)</TD><TD>18.5</TD><TD>656</TD></TR><TR><TD>B0015(Terminator)</TD><TD>9.5</TD><TD>143</TD></TR><TR><TD>Total</TD><TD>1161</TD></TR></TBODY></TABLE><P>The theoratical volumn proportion is: </P><UL><LI>SoxS-C0061-B0015: 5: 7: 3</LI><LI>micF-C0061-B0015: 1.3: 7: 3</LI></UL><P>Firstly, to ligate these parts, we need 10 cycles without primer:</P><P>for circuit SoxS-C0061-B0015:</P><TABLE><THEAD><TR><TH>Name</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>SoxS</TD><TD>5</TD></TR><TR><TD>C0061</TD><TD>7</TD></TR><TR><TD>B0015</TD><TD>3</TD></TR><TR><TD>Taq Polymerase</TD><TD>0.25</TD></TR><TR><TD>MgCl2</TD><TD>2</TD></TR><TR><TD>dNTP</TD><TD>0.4</TD></TR><TR><TD>10X buffer with KCl</TD><TD>2</TD></TR><TR><TD>ddH2O</TD><TD>0.35</TD></TR></TBODY></TABLE><P>for circuit micF-C0061-B0015:</P><TABLE><THEAD><TR><TH>Name</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>micF</TD><TD>1.3</TD></TR><TR><TD>C0061</TD><TD>7</TD></TR><TR><TD>B0015</TD><TD>3</TD></TR><TR><TD>Taq Polymerase</TD><TD>0.25</TD></TR><TR><TD>MgCl2</TD><TD>2</TD></TR><TR><TD>dNTP</TD><TD>0.4</TD></TR><TR><TD>10X buffer with KCl</TD><TD>2</TD></TR><TR><TD>ddH2O</TD><TD>4.1</TD></TR></TBODY></TABLE><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>10 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)</LI><LI>Elongation for whole at 72 degree celcius 1min</LI><LI>4 celcius degree for preserveation.</LI></OL><P>Primers are the same as before.</P><TABLE><THEAD><TR><TH>Name</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>Template</TD><TD>5</TD></TR><TR><TD>Taq Polymerase</TD><TD>0.25</TD></TR><TR><TD>MgCl2</TD><TD>2</TD></TR><TR><TD>dNTP</TD><TD>0.4</TD></TR><TR><TD>micF-F(BB-SoxS-F)</TD><TD>1</TD></TR><TR><TD>B0015-R</TD><TD>1</TD></TR><TR><TD>10X buffer with KCl</TD><TD>2</TD></TR><TR><TD>ddH2O</TD><TD>8.35</TD></TR></TBODY></TABLE><P>15ul-protocol:</P><TABLE><THEAD><TR><TH>Name</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>Solution</TD><TD>15</TD></TR><TR><TD>micF-F(BB-SoxS-F)</TD><TD>1</TD></TR><TR><TD>B0015-R</TD><TD>1</TD></TR><TR><TD>10X buffer with KCl</TD><TD>2</TD></TR><TR><TD>ddH2O</TD><TD>1</TD></TR></TBODY></TABLE><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)</LI><LI>Elongation for whole at 72 degree celcius 1min</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>PCR of Cas9(80l)</H5><P>material:</P><P>8 uL buffer with KCl</P><P>35 uL dd water</P><P>8 uL Plasmid with Cas9</P><P>4 uL primer R</P><P>4 uL primer F</P><P>8 uL MgCl2</P><P>8 uL dNTPs</P><P>5 uL DMSO</P><P>1 uL Tap polymerase</P><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 4.5 min)</LI><LI>Elongation for whole at 72 degree celcius 1min</LI><LI>4 celcius degree for preserveation.</LI></OL><LI><TIME class="cbp_tmtime" datetime="2015-07-29 00:00"><SPAN>2015</SPAN><SPAN>29/7</SPAN></TIME><DIV class="cbp_tmlabel"><H5>Negative Chemotaxis Characterization</H5><P>Done by Wang Luyao</P><P><EM>Term 1 results</EM></P><P> I put the paper on this plate after the E.coli grows homogeneously， so let's see the result later.</P><P>the controlled plate</P><P>the last plate was contaminated</P><P><EM>Term 2</EM></P><P>material:</P><P>LB solid medium plates x6</P><P>0.8、0.6、0.4、0.2、0.1g/ml Chloramphenicol</P><P>E.coli Top10</P><P>filter paper</P><H5>Enzyme Digestion</H5><P>Performing twice separately</P><P>Both as following protocol:</P><TABLE><THEAD><TR><TH>Name</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>pSB1C3</TD><TD>10</TD></TR><TR><TD>XbaI</TD><TD>1</TD></TR><TR><TD>SpeI</TD><TD>1</TD></TR><TR><TD>green buffer</TD><TD>2</TD></TR><TR><TD>ddH2O</TD><TD>6</TD></TR></TBODY></TABLE><H5>Overlap PCR of bacteriaI</H5><P>material:</P></DIV></LI><LI>micF PCR product :33.6ng/uL</LI><LI>C0061 PCR product: 18.5ng/uL</LI><LI>B0015 PCR product:9.2ng/uL</LI><LI>micF-F primer</LI><LI>B0015-R primer</LI><P>protocol:</P><P>total volume: 25uL</P><UL><LI>Taq Master Mix 12.5uL</LI><LI>micF 2uL</LI><LI>C0061 6uL</LI><LI>B0015 2.6uL</LI><LI>ddH2O 1.9uL</LI></UL><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>10 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 90s)</LI><LI>Elongation for whole at 72 degree celcius 10min</LI><LI>4 celcius degree for preserveation.</LI></OL><P>After the 10 cycles,we divide the 20 ul system into two parts:5ul and 15ul.The 5ul PCR product will be used as template in another new 20ul system.On the other hand, add primers</P><P>to the remaining 15 uL PCR product.</P><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 90s)</LI><LI>Elongation for whole at 72 degree celcius 10min</LI><LI>4 celcius degree for preserveation.</LI></OL><P>Results:</P><P>We don't get the product we want.We make some research and guess that the probable reasons are the following:Taq polymerase can cause 3'd A to PCR product;the annealing temperature is unsuitable;the length of overlap fragment isn't enough .</P><LI><TIME class="cbp_tmtime" datetime="2015-07-30 00:00"><SPAN>2015</SPAN><SPAN>30/7</SPAN></TIME><DIV class="cbp_tmlabel"><H5>Negative Chemotaxis Characterization</H5><P>Done by Wang Luyao</P><P>7.30</P><P><EM>Term 2 results</EM></P><P>The 0.8、0.6、0.4 plates just look similar, the denser Chloramphenicol the paper hold， the bigger the clear zone is.</P><P>but the 0.2 and 0.1g/ml plates are different, Top10 grows denser on the edge of the clear zone, I supposed it can show that E.coli has moved. So next time I'll try under 0.2g/ml.</P><P>The controlled plate</P><H5>PCR of C0062 and R0010</H5><P>material:</P></DIV></LI><LI>10 ul Taq PCR Master Mix</LI><LI>7 ul ddH2O</LI><LI>1 ul Primer F</LI><LI>1 ul Primer R</LI><LI>1 ul C0062 or R0010</LI><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 4.5 min)</LI><LI>Elongation for whole at 72 degree celcius 1min</LI><LI>4 celcius degree for preserveation.</LI></OL><LI><TIME class="cbp_tmtime" datetime="2015-07-31 00:00"><SPAN>2015</SPAN><SPAN>31/7</SPAN></TIME><DIV class="cbp_tmlabel"><H5>Negative Chemotaxis Characterization</H5><P>Done by Wang Luyao</P><P><EM>Term 3</EM>, material:</P></DIV></LI><LI><P>LB solid medium plates x5</P></LI><LI><P>0.2、0.1、0.05、0.025g/ml Chloramphenicol</P></LI><LI><P>E.coli Top10</P></LI><LI><P>filter paper</P></LI><H5>PCR of micF, SoxS, C0061 and R0015</H5><P>material(volume 50uL):</P><UL><LI>25 ul Taq PCR Master Mix</LI><LI>15 ul ddH2O</LI><LI>2.5 ul Primer F</LI><LI>2.5 ul Primer R</LI><LI>2.5 ul Top10 culture, Top10 culture, C0061 or R0015</LI></UL><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 4.5 min)</LI><LI>Elongation for whole at 72 degree celcius 1min</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>PCR of tRNA</H5><TABLE><THEAD><TR><TH>Primers</TH><TH>Sequence(5' to 3')</TH></TR></THEAD><TBODY><TR><TD>tRNA-F</TD><TD>GAATTCGCGGCCGCTTCTAGATAATACGACTCACTATAGGGAATACAAGCTACTTGTTCTTTTTGCAATGGACGAGTTCGAAATGATT</TD></TR><TR><TD>tRNA-R</TD><TD>CTGCAGCGGCCGCTACTAGTTTACAGACGTTTGCGAATTG</TD></TR><TR><TD>tRNA-F2</TD><TD>TTTAATCCCCGTCCACTGGGTAATACGACTCACTATAGGGAATAC</TD></TR></TBODY></TABLE><P>Protocol (volume 20uL):</P><UL><LI>2 ul Taq Buffer</LI><LI>0.4 ul dNTP</LI><LI>1 ul tRNA-F</LI><LI>1 ul tRNA-R</LI><LI>0.25 ul Taq DNA Polymerase</LI><LI>2 ul MgCl2</LI><LI>1 ul tRNA</LI><LI>12.35 ul ddH2O</LI></UL><P>Protocol (volume 20uL):</P><UL><LI>2 ul Taq Buffer</LI><LI>0.4 ul dNTP</LI><LI>1 ul tRNA-F2</LI><LI>1 ul tRNA-R</LI><LI>0.25 ul Taq DNA Polymerase</LI><LI>2 ul MgCl2</LI><LI>1 ul tRNA</LI><LI>12.35 ul ddH2O</LI></UL><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 7min</LI><LI>25 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s ) and elongation(72 degree celcius 1.5 min)</LI><LI>Elongation for whole at 72 degree celcius 10min</LI><LI>4 celcius degree for preserveation.</LI></OL><P>Result</P><P>Only one sample get PCR product, concentration 441ng/μL. tRNA-F2 dosen't work for wrong template.</P><P>Remember to review gene map before experiments!</P><H5>PCR of cheZ</H5><P>material:</P><P>Top 10 Bacterial Solution</P><P>Primers produced from Sangon Biotech Co.,Ltd.</P><TABLE><THEAD><TR><TH>Primers</TH><TH>Sequence(5' to 3')</TH></TR></THEAD><TBODY><TR><TD>cheZ-F</TD><TD>GTGCAATTGATCCAGGAGCTCAGCCAGGC</TD></TR><TR><TD>cheZ-T7-R</TD><TD>GCTCCTGGATCAATTGCACATAAATTTTCTCCTCTTTTGCAAAAAGAA</TD></TR></TBODY></TABLE><P>for cheZ (~600bp)</P><P>Protocol as this:</P><UL><LI>2.5 μl Taq buffer with KCl</LI><LI>2.5 μl dNTPs</LI><LI>0.75 μl cheZ-F Primer</LI><LI>0.75 μl cheZ-T7-R Primer</LI><LI>0.5 μl Kod DNA Polymerase(600bp/min)</LI><LI>1.0 μl MgCl2</LI><LI>2 uL Top10 Solution</LI><LI>15 μl ddH2O</LI></UL><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1 min 30s)</LI><LI>Elongation for whole at 72 degree celcius 2min(C0061)or 1min(B0015 and R0062)</LI><LI>4 celcius degree for preserveation.</LI></OL><P>It turned out we used wrong primer:(</P><H5>PCR of CheZ and B0015</H5><P>material:</P><UL><LI>TOP 10 baterial solution</LI><LI>PAO1 bacterial solution </LI><LI>B0015 plasmid :93.0 ng/uL</LI></UL><P>protocol:</P><P>total system 25uL</P><UL><LI>buffer 2.5 uL</LI><LI>dNTPs 2.5uL</LI><LI>MgSO4 2uL</LI><LI>primer-F 0.75 uL</LI><LI>primer-R 0.75 uL</LI><LI>bacterial solution/plasmid 1uL</LI><LI>KOD-Plus enzyme 0.5uL</LI><LI>ddH2O 17uL</LI></UL><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 2min</LI><LI>35 cycles containing: degradation(94 degree celcius, 15s; annealing(55 degree celcius, 30s and elongation(68 degree celcius 50s)</LI><LI>Elongation for whole at 68 degree celcius 7min.</LI><LI>4 celcius degree for preserveation.</LI></OL><P>The protocol is from lab319,the enzyme is KOD-Plus.</P><P>Result:</P><P> We don't get the product we want .It turned out because of wrong primer we implemented.</P><H5>Transformation of C0062</H5><P>the protocol as following</P><OL><LI>Take the competent bacteria from refrigerator and incubate them into ice about 10 mins until it is dissolved</LI><LI>Absorb 1 uL or 2 uL plasmid and mix it with bacteria solution thoroughly.</LI><LI>Put the tubes on the ice about 30 mins.</LI><LI>Make a heat shock at 42 degree centigrade about 60 sec</LI><LI>Put the tubes on the ice about 3 mins again.</LI><LI>Add 900 ul LB medium into EP tubes and cultivate the bacteria at 37 degree centrigrade about 60 min.</LI><LI>Centrifuge them at 4,000xg about 15 sec and we will se sediment in the tubes.</LI><LI>Discard the supernatant liquid and leave about 100ul medium.</LI><LI>Coat plate: add 100ul solution in a plate</LI></OL><LI><TIME class="cbp_tmtime" datetime="2015-08-01 00:00"><SPAN>2015</SPAN><SPAN>1/8</SPAN></TIME><DIV class="cbp_tmlabel"><H5>Negative Chemotaxis Characterzation</H5><P>Done by Wang Luyao</P><P><EM>Term 3 results</EM></P><P>this time I got the similar results, but I cannot tell whether the bacteria is moved or didn't grow up.</P><P>so let's see the term 1 result:</P><P> I put the paper on this plate after the E.coli grows homogeneously. Now after I took it out I got this.</P><P>It's obviously that the bacteria moved. If E.coli died, the medium cannot be clear, the bodies will stay. In the middle of the area, there was a bubble formed by the medium and the wet paper. There is dilute Chloramphenicol, so the some bacteria moved there.</P><P>So we qualitatively proved that E.coli negative chemotaxis !!!</P><H5>PCR of cheZ</H5><P>Material:</P><P>Top 10 Bacterial Solution</P><P>Primers produced from Sangon Biotech Co.,Ltd.</P><TABLE><THEAD><TR><TH>Primers</TH><TH>Sequence(5' to 3')</TH></TR></THEAD><TBODY><TR><TD>cheZ-F</TD><TD>GTGCAATTGATCCAGGAGCTCAGCCAGGC</TD></TR><TR><TD>che-Ter-R</TD><TD>TTTATTTGATGCCTGGCTCTAGTATCAGAAACCCAGGCTGGACA</TD></TR></TBODY></TABLE><P>for cheZ (~600bp)</P><P>Protocol as this:</P></DIV></LI><LI>5 μl Taq buffer with KCl</LI><LI>1 μl dNTPs</LI><LI>2.5 μl cheZ-F Primer</LI><LI>2.5 μl che-Ter-R Primer</LI><LI>0.65 μl Pfu DNA Polymerase(600bp/min)</LI><LI>5 μl MgCl2</LI><LI>2.5 uL Top10 Solution</LI><LI>31 μl ddH2O</LI><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1 min 30s)
                      1.Elongation for whole at 72 degree celcius 10 mins</LI><LI>4 celcius degree for preserveation.</LI></OL><P><EM>No results: May be because of Pfu enzyme has already been out of date.</EM></P><P>Protocol again:</P><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>2X Taq Master Mix</TD><TD>25</TD></TR><TR><TD>cheZ-F</TD><TD>2.5</TD></TR><TR><TD>che-Ter-R</TD><TD>2.5</TD></TR><TR><TD>Top10 Solution</TD><TD>5</TD></TR><TR><TD>ddH2O</TD><TD>15</TD></TR></TBODY></TABLE><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>2X Taq Master Mix</TD><TD>25</TD></TR><TR><TD>cheZ-F</TD><TD>2.5</TD></TR><TR><TD>che-Ter-R</TD><TD>2.5</TD></TR><TR><TD>Top10 Solution</TD><TD>5</TD></TR><TR><TD>ddH2O</TD><TD>12.5</TD></TR><TR><TD>DMSO</TD><TD>2.5</TD></TR></TBODY></TABLE><P><EM>No results：it turns out cheZ sequence in TOP10 is different from PAO1, we designed after PAO1</EM></P><P>Protocol again:</P><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>2X Taq Master Mix</TD><TD>25</TD></TR><TR><TD>cheZ-F</TD><TD>2.5</TD></TR><TR><TD>che-Ter-R</TD><TD>2.5</TD></TR><TR><TD>PAO1 Solution</TD><TD>5</TD></TR><TR><TD>ddH2O</TD><TD>15</TD></TR></TBODY></TABLE><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>Taq buffer with KCl</TD><TD>5</TD></TR><TR><TD>dNTPs</TD><TD>1</TD></TR><TR><TD>che-Ter-R</TD><TD>2.5</TD></TR><TR><TD>PAO1 Solution</TD><TD>5</TD></TR><TR><TD>ddH2O</TD><TD>28.35</TD></TR><TR><TD>MgCl2</TD><TD>5</TD></TR><TR><TD>Taq Polymerase</TD><TD>0.65</TD></TR></TBODY></TABLE><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1 min)</LI><LI>Elongation for whole at 72 degree celcius 10 mins</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>PCR of tRNA</H5><P>Protocol (volume 20uL):</P><UL><LI>2 ul Taq Buffer</LI><LI>0.4 ul dNTP</LI><LI>1 ul tRNA-F2</LI><LI>1 ul tRNA-R</LI><LI>0.25 ul Taq DNA Polymerase</LI><LI>2 ul MgCl2</LI><LI>1 ul tRNA</LI><LI>12.35 ul ddH2O</LI></UL><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 7min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s ) and elongation(72 degree celcius 1.5 min)</LI><LI>Elongation for whole at 72 degree celcius 10min</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>Gel Extraction of tRNA</H5><P>Preparation:</P><OL><LI>Check out whether ethyl alcohol is added into Wash Solution. </LI><LI>Check out whether Buffer B2 has sediment. </LI><LI>Adjust the thermostat water bath at 50 degree centigrade.</LI></OL><P>Procedure:</P><OL><LI>Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers. </LI><LI>Add 1.5mL Buffer B2 and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins. </LI><LI>After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s. </LI><LI>Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution. </LI><LI>Do the step 5 again. </LI><LI>Centrifuge the empty columns at 9000xg about 1min. </LI><LI>Open the columns and air them about 10 mins. </LI><LI>Put the columns into clean 1.5 mL EP tubes and add 20uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 10 min and then centrifuge at 9000xg about 1 min. </LI><LI>Keep the DNA solution and often measure the concentration of DNA and A260/A280 for further study.</LI></OL><P>Result</P><P>Concentration 64ng/ul.</P><H5>PCR of CheZ</H5><P>material:</P><P>Top 10 Bacterial Solution</P><P>protocol:</P><P>total system 25uL</P><UL><LI>buffer 2.5 uL</LI><LI>dNTPs 2.5uL</LI><LI>MgSO4 3uL</LI><LI>primer-F 0.75 uL</LI><LI>primer-R 0.75 uL</LI><LI>bacterial solution/plasmid 1uL</LI><LI>KOD-Plus-Neo enzyme 0.5uL</LI><LI>ddH2O 16uL</LI></UL><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 2min</LI><LI>35 cycles containing: degradation(98 degree celcius, 10s; annealing(55 degree celcius, 30s and elongation(68 degree celcius 30s)</LI><LI>Elongation for whole at 68 degree celcius 7min.</LI><LI>4 celcius degree for preserveation.
                      The protocol is from lab 319.</LI></OL><H5>PCR of T7 promoter</H5><TABLE><THEAD><TR><TH>Primers</TH><TH>Sequence(5' to 3')</TH></TR></THEAD><TBODY><TR><TD>T7-F</TD><TD>GAATTCGCGGCCGCTTCTAG</TD></TR><TR><TD>T7-R2</TD><TD>agacatgcaatttttttcattccgattttctcctcttttgcaaaaagaacaagtagct</TD></TR><TR><TD>T7 Promoter1</TD><TD>GAATTCGCGGCCGCTTCTAGAtaatacgactcactatagggaatacaagctacttgttctttttgca</TD></TR><TR><TD>T7 Promoter2</TD><TD>tgcaaaaagaacaagtagcttgtattccctatagtgagtcgtattaTCTAGAAGCGGCCGCGAATTC</TD></TR></TBODY></TABLE><P>Protocol (volume 20uL):</P><UL><LI>10 ul Mix</LI><LI>1 ul T7-F</LI><LI>1 ul T7-R2</LI><LI>1 ul T7 Promoter1</LI><LI>1 ul T7 promoter2</LI><LI>6 ul ddH2O</LI></UL><P>PCR cycle including:</P><OL><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s ) and elongation(72 degree celcius 30s)</LI><LI>Elongation for whole at 72 degree celcius 7min.</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>PCR Purification of T7</H5><OL><LI>Add Buffer B3 in 5 times volumn of PCR product.</LI><LI>Pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s. </LI><LI>Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution. </LI><LI>Do the step 5 again. </LI><LI>Centrifuge the empty columns at 9000xg about 1min. </LI><LI>Open the columns and air them about 10 mins. </LI><LI>Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min. </LI><LI>Keep the DNA solution and measure the concentration of DNA and A260/A280 for further study.</LI></OL><P>Result</P><P>Concentration 28ng/ul.</P><H5>PCR of T7 promoter</H5><P>Conduct the PCR again.</P><P>Protocol (volume 20uL):</P><UL><LI>10 ul Mix</LI><LI>1 ul T7-F</LI><LI>1 ul T7-R2</LI><LI>1 ul T7 Promoter1</LI><LI>1 ul T7 promoter2</LI><LI>6 ul ddH2O</LI></UL><P>PCR cycle including:</P><OL><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s ) and elongation(72 degree celcius 30s)</LI><LI>Elongation for whole at 72 degree celcius 7min.</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>Seamless Clone of tRNA Plasmid</H5><P>Protocol:</P><UL><LI>8.5 ul Seamless cloning Master Mix</LI><LI>3 ul linear PSB1C3 plasmid</LI><LI>2 ul tRNA </LI><LI>6.5 ul ddH2O
                      Incubate at 50℃ for 60min.</LI></UL><H5>Seamless Clone of Anchor Plasmid</H5><P>Protocol:</P><UL><LI>8.5 ul Seamless cloning Master Mix</LI><LI>3 ul linear PSB1C3 plasmid</LI><LI>1 ul T7 promoter</LI><LI>1 ul COMPX</LI><LI>6.5 ul ddH2O
                      Incubate at 50℃ for 60min.</LI></UL><H5>Transformation of tRNA Plasmid &amp; Anchor Plasmid</H5><P>Procedure:</P><OL><LI>Add 4ul clone product to competent cells, mix well, incubate on ice for 30min.</LI><LI>Heat in 42℃ bath for 90s. Then put on ice for 5min.</LI><LI>Add 500ul SOC culture, incubate at 37℃ for 45min.</LI><LI>4000rpm centrifuge for 3min, discard 500ul supernate.</LI><LI>Spread on plate with chloramphenicol, incubate overnight at 37℃.</LI></OL><LI><TIME class="cbp_tmtime" datetime="2015-08-02 00:00"><SPAN>2015</SPAN><SPAN>2/8</SPAN></TIME><DIV class="cbp_tmlabel"><H5>Plasmid Extraction of C0061</H5><P>Procedure including:</P></DIV></LI><LI>Absorb 6 mL bacteria solution in to EP tubes, centrifuge them at 12,000xg 1mins and then discard the culture medium.</LI><LI>Add 250 ul Buffer S1 into sediment, and use spearhead to make bacteria suspended.</LI><LI>Add 250 ul Buffer S2, and overturn the EP tubes 8 times immediately and tenderly. Stand tubes about 3 mins.</LI><LI>Add 350 ul Buffer W3 and overturn the EP tubes 10 times again immediately and tenderly.</LI><LI>Centrifuge tubes at 12,000xg about 10 mins.</LI><LI>Pour the supernatant liquid into absorption columns ,centrifuge them at 12,000xg about 30 sec and then discard the liquid in the collecting pipe.</LI><LI>Add 500 ul Buffer W1 and centrifuge them at 12,000xg about 30 sec. Then discard the liquid in the collecting pipe.</LI><LI>Add 700 ul Buffer W2, centrifuge them at 12,000xg about 30 sec and discard the liquid in the collecting pipe.</LI><LI>Do the step 8 again.</LI><LI>Centrifuge the empty columns at 12,000xg about 1 min.</LI><LI>Put the columns in clean 1.5mL EP tubes, add 80 ul Elution Buffer on the absorption film, stand 10 min and centrifuge about 1 min at 12,000xg. </LI><H5>PCR of cheZ and OprF from PAO1</H5><P>Yesterday, we may extract cheZ from PAO 1, though with a lot of primer dimer. Today, DMSO will be implemented trying to eliminate these fragements.</P><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>Taq buffer with KCl</TD><TD>5</TD></TR><TR><TD>dNTPs</TD><TD>1</TD></TR><TR><TD>cheZ-F</TD><TD>2.5</TD></TR><TR><TD>cheZ-Ter-R</TD><TD>2.5</TD></TR><TR><TD>PAO1 Solution</TD><TD>5</TD></TR><TR><TD>ddH2O</TD><TD>25.85</TD></TR><TR><TD>MgCl2</TD><TD>5</TD></TR><TR><TD>Pfu Polymerase</TD><TD>0.65</TD></TR><TR><TD>DMSO</TD><TD>2.5</TD></TR></TBODY></TABLE><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>Taq buffer with KCl</TD><TD>5</TD></TR><TR><TD>dNTPs</TD><TD>1</TD></TR><TR><TD>Opr-F</TD><TD>2.5</TD></TR><TR><TD>OprF-Ter-R</TD><TD>2.5</TD></TR><TR><TD>PAO1 Solution</TD><TD>5</TD></TR><TR><TD>ddH2O</TD><TD>25.85</TD></TR><TR><TD>MgCl2</TD><TD>5</TD></TR><TR><TD>Pfu Polymerase</TD><TD>0.65</TD></TR><TR><TD>DMSO</TD><TD>2.5</TD></TR></TBODY></TABLE><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 2 min 30 s)</LI><LI>Elongation for whole at 72 degree celcius 10 mins</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>PCR Purification of T7</H5><OL><LI>Add Buffer B3 in 5 times volumn of PCR product.</LI><LI>Pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s. </LI><LI>Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution. </LI><LI>Do the step 5 again. </LI><LI>Centrifuge the empty columns at 9000xg about 1min. </LI><LI>Open the columns and air them about 10 mins. </LI><LI>Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min. </LI><LI>Keep the DNA solution and often measure the concentration of DNA and A260/A280 for further study.</LI></OL><H5>Gel Extraction</H5><P>Material including: OprF, cheZ</P><P>Preparation:</P><OL><LI>Check out whether ethyl alcohol is added into Wash Solution. </LI><LI>Check out whether Buffer B2 has sediment. </LI><LI>Adjust the thermostat water bath at 50 degree centigrade.</LI></OL><P>Procedure:</P><OL><LI>Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers. </LI><LI>Measure out the weight of tubes of gel. </LI><LI>Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins. </LI><LI>Optional: If the length of the DNA strip is shorter than 500 bp, it would be better add isopropanol one third volume of Buffer B2. </LI><LI>After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s. </LI><LI>Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution. </LI><LI>Do the step 5 again. </LI><LI>Centrifuge the empty columns at 9000xg about 1min. </LI><LI>Open the columns and air them about 10 mins. </LI><LI>Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min. </LI><LI>Keep the DNA solution and often measure the concentration of DNA and A260/A280 for further study.</LI></OL><H5>Colonial PCR of PSB1C3-T7-COMPX &amp; PSB1C3-T7-tRNA</H5><P>Protocol(volume 20ul)</P><UL><LI>1μl T7-F Primer</LI><LI>1μl COMPX-R or tRNA-R Primer</LI><LI>10μl Mix</LI><LI>1μl Bacteria Clone</LI><LI>7μl ddH2O</LI></UL><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 7min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s), annealing(55 degree celcius, 30s) and elongation(72 degree celcius 1.5 min)</LI><LI>Elongation for whole at 72 degree celcius 10min.</LI><LI>4 celcius degree for preserveation.</LI></OL><P><EM>No Result.</EM></P><H5>Colonial PCR of PSB1C3-T7-COMPX &amp; PSB1C3-T7-tRNA</H5><P>Conduct the PCR again.</P><P>Protocol(volume 20ul)</P><UL><LI>1μl T7-F Primer</LI><LI>1μl COMPX-R or tRNA-R Primer</LI><LI>10μl Mix</LI><LI>1μl Bacteria Clone</LI><LI>7μl ddH2O</LI></UL><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 7min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s), annealing(55 degree celcius, 30s) and elongation(72 degree celcius 1.5 min)</LI><LI>Elongation for whole at 72 degree celcius 10min.</LI><LI>4 celcius degree for preserveation.</LI></OL><P><EM>No Result.</EM></P><H5>PCR Purification of tRNA &amp; COMPX</H5><OL><LI>Add Buffer B3 in 5 times volumn of PCR product.</LI><LI>Pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s. </LI><LI>Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution. </LI><LI>Do the step 5 again. </LI><LI>Centrifuge the empty columns at 9000xg about 1min. </LI><LI>Open the columns and air them about 10 mins. </LI><LI>Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min. </LI><LI>Keep the DNA solution and measure the concentration of DNA and A260/A280 for further study.</LI></OL><P>Result</P><P>tRNA 26ng/ul, COMPX 24ng/ul.</P><H5>Transformation of tRNA Plasmid &amp; Anchor Plasmid</H5><P>Procedure:</P><OL><LI>Add 4ul clone product to competent cells, mix well, incubate on ice for 30min.</LI><LI>Heat in 42℃ bath for 90s. Then put on ice for 5min.</LI><LI>Add 500ul SOC culture, incubate at 37℃ for 45min.</LI><LI>4000rpm centrifuge for 3min, discard 500ul supernate.</LI><LI>Spread half of the bacteria culture on plate with chloramphenicol, incubate overnight at 37℃.</LI></OL><LI><TIME class="cbp_tmtime" datetime="2015-08-03 00:00"><SPAN>2015</SPAN><SPAN>3/8</SPAN></TIME><DIV class="cbp_tmlabel"><H5>Colonial PCR of PSB1C3-T7-COMPX &amp; PSB1C3-T7-tRNA</H5><P>Protocol(volume 20ul)</P></DIV></LI><LI>1μl T7-F Primer</LI><LI>1μl COMPX-R or tRNA-R Primer</LI><LI>10μl Mix</LI><LI>1μl Bacteria Clone</LI><LI>7μl ddH2O</LI><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 7min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s), annealing(55 degree celcius, 30s) and elongation(72 degree celcius 1.5 min)</LI><LI>Elongation for whole at 72 degree celcius 10min.</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>PCR of OprF from PAO1</H5><P>Yesterday, we may extract cheZ from PAO 1, though with a lot of primer dimer. Today, DMSO will be implemented trying to eliminate these fragements.</P><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>Taq buffer with KCl</TD><TD>5</TD></TR><TR><TD>dNTPs</TD><TD>1</TD></TR><TR><TD>Opr-F</TD><TD>2.5</TD></TR><TR><TD>OprF-Ter-R</TD><TD>2.5</TD></TR><TR><TD>PAO1 Solution</TD><TD>5</TD></TR><TR><TD>ddH2O</TD><TD>28.35</TD></TR><TR><TD>MgCl2</TD><TD>5</TD></TR><TR><TD>Pfu Polymerase</TD><TD>0.65</TD></TR><TR><TD>DMSO</TD><TD>2.5</TD></TR></TBODY></TABLE><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 2 min)</LI><LI>Elongation for whole at 72 degree celcius 10 mins</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>PCR of C0061</H5><P>material(volume 100uL):</P><UL><LI>6 ul MgCl2</LI><LI>71 ul ddH2O</LI><LI>4 ul Primer F</LI><LI>4 ul Primer R</LI><LI>2 ul dNTP</LI><LI>1 ul Taq DNA polymerase</LI><LI>10 ul 10*PCR Buffer</LI><LI>2 ul C0061</LI></UL><P>PCR cycle including:</P><P>1 Preheat: 94 degree celcius, 5min</P><P>2 35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 4.5 min)</P><P>3 Elongation for whole at 72 degree celcius 1min</P><P>4 4 celcius degree for preserveation.</P><H5>Gel Extraction</H5><P>Material including: R0061</P><P>Preparation:</P><P>• Check out whether ethyl alcohol is added into Wash Solution.</P><P>• Check out whether Buffer B2 has sediment.</P><P>• Adjust the thermostat water bath at 50 degree centigrade.</P><P>Procedure:</P><P>5 Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.</P><P>6 Measure out the weight of tubes of gel.</P><P>7 Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.</P><P>8 Optional: If the length of the DNA strip is shorter than 500 bp, it would be better add isopropanol one third volume of Buffer B2.</P><P>9 After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s.</P><P>10 Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.</P><P>11 Do the step 5 again.</P><P>12 Centrifuge the empty columns at 9000xg about 1min.</P><P>13 Open the columns and air them about 10 mins.</P><P>14 Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.</P><H5>PCR for gRNA-AcrB and gRNA-EmrE</H5><P>material:</P><P>gRNA-AcrB plasmid(280.7 ng/uL)</P><P>gRNA-EmrE plasmid(222.4 ng/uL)</P><P>PCR protocol:</P><P>For gRNA-AcrB and gRNA-EmrE, primers including gRNA-F, gRNA-R, protocol as following:</P><UL><LI>10μl Taq Master Mix</LI><LI>1μl gRNA-F Primer</LI><LI>1μl gRNA-R Primer</LI><LI>0.25μl Taq DNA Polymerase</LI><LI>6.75μl ddH2O</LI><LI>1μl gRNA-AcrB(or gRNA-EmrE) Plasmid</LI></UL><P>For gRNA, Primers produced from Sangon Biotech Co.,Ltd.</P><TABLE><THEAD><TR><TH>Primer</TH><TH>Sequence(5'-3')</TH></TR></THEAD><TBODY><TR><TD>gRNA-F</TD><TD>GCGTCTAGATTGACAGCTAGCTCAGTAGGTATAAT</TD></TR><TR><TD>gRNA-R</TD><TD>TAGTAGGTCGTATTAAAAAAAAGCACCGAC</TD></TR></TBODY></TABLE><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)</LI><LI>Elongation for whole at 72 degree celcius 7 min</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>Gel Electrophoresis Characterization</H5><P>gRNA-AcrB and gRNA-EmrE</P><H5>Gel Extraction of gRNA-AcrB and gRNA-EmrE</H5><P>Material including: gRNA-AcrB and gRNA-EmrE</P><P>Preparation:</P><OL><LI>Check out whether ethyl alcohol is added into Wash Solution. </LI><LI>Check out whether Buffer B2 has sediment. </LI><LI>Adjust the thermostat water bath at 50 degree centigrade.</LI></OL><P>Procedure:</P><OL><LI>Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers. </LI><LI>Measure out the weight of tubes of gel. </LI><LI>Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins. </LI><LI>Optional: If the length of the DNA strip is shorter than 500 bp, it would be better add isopropanol one third volume of Buffer B2. </LI><LI>After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s. </LI><LI>Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution. </LI><LI>Do the step 5 again. </LI><LI>Centrifuge the empty columns at 9000xg about 1min. </LI><LI>Open the columns and air them about 10 mins. </LI><LI>Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min. </LI><LI>Keep the DNA solution and often measure the concentration of DNA and A260/A280 for further study.</LI></OL><P>the concentration as following:</P><P>gRNA-AcrB: 44 ng/uL</P><P>gRNA-EmrE: 41 ng/uL</P><H5>PCR for T7-tet Promoter</H5><P>PCR protocol:</P><P>For T7-tet promoter, primers including T7-tet-F, T7-tet-R, protocol as following:</P><UL><LI>10μl Taq Master Mix</LI><LI>1μl T7-tet-F Primer</LI><LI>1μl T7-tet-R Primer</LI><LI>1μl template-F Primer</LI><LI>1μl template-R Primer</LI><LI>0.25μl Taq DNA Polymerase</LI><LI>5.75μl ddH2O</LI></UL><P>Use template-F Primer and template-R Primer as templates</P><P>For T7-tet, Primers produced from Sangon Biotech Co.,Ltd.</P><TABLE><THEAD><TR><TH>Primer</TH><TH>Sequence(5'-3')</TH></TR></THEAD><TBODY><TR><TD>template-F</TD><TD>taatacgactcactatgatagaaaagaggagaaaatc</TD></TR><TR><TD>template-R</TD><TD>gattttctcctcttttctatcatagtgagtcgtatta</TD></TR><TR><TD>T7-tet-F</TD><TD>Ttaatacgactcactatgatagaaaagaggag</TD></TR><TR><TD>T7-tet-R</TD><TD>gtatttcttatccattccgattttctcctcttttctat</TD></TR></TBODY></TABLE><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)</LI><LI>Elongation for whole at 72 degree celcius 7 min</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>Gel Electrophoresis Characterization</H5><P>T7-tet Promoter</P><H5>Gel Extraction of T7-tet Promoter</H5><P>Material including: T7-tet Promoter</P><P>Preparation:</P><OL><LI>Check out whether ethyl alcohol is added into Wash Solution. </LI><LI>Check out whether Buffer B2 has sediment. </LI><LI>Adjust the thermostat water bath at 50 degree centigrade.</LI></OL><P>Procedure:</P><OL><LI>Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers. </LI><LI>Measure out the weight of tubes of gel. </LI><LI>Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins. </LI><LI>Optional: If the length of the DNA strip is shorter than 500 bp, it would be better add isopropanol one third volume of Buffer B2. </LI><LI>After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s. </LI><LI>Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution. </LI><LI>Do the step 5 again. </LI><LI>Centrifuge the empty columns at 9000xg about 1min. </LI><LI>Open the columns and air them about 10 mins. </LI><LI>Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min. </LI><LI>Keep the DNA solution and often measure the concentration of DNA and A260/A280 for further study.</LI></OL><P>the concentration as following:</P><P>T7-tet: 30 ng/uL</P><H5>Seamless Clone of cheZ Plasmid and Circuit Containing E0040</H5><P>Protocol:</P><TABLE><THEAD><TR><TH>Fragements</TH><TH>Length(bp)</TH><TH>Mass Required(ug)</TH></TR></THEAD><TBODY><TR><TD>linear PSB1C3 plasmid</TD><TD>2070</TD><TD>41.4</TD></TR><TR><TD>T7 Promoter</TD><TD>46</TD><TD>9.2</TD></TR><TR><TD>cheZ</TD><TD>683</TD><TD>13.66</TD></TR><TR><TD>B0015</TD><TD>143</TD><TD>2.86</TD></TR></TBODY></TABLE><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Volumn</TH></TR></THEAD><TBODY><TR><TD>Seamless cloning Master Mix</TD><TD>8.5 ul</TD></TR><TR><TD>linear PSB1C3 plasmid</TD><TD>3 ul</TD></TR><TR><TD>T7 Promoter</TD><TD>1ul(+3 folds ddH2O)</TD></TR><TR><TD>cheZ</TD><TD>1ul(+1 fold ddH2O)</TD></TR><TR><TD>B0015</TD><TD>1ul(+9 folds ddH2O)</TD></TR><TR><TD>ddH2O</TD><TD>5.5ul</TD></TR></TBODY></TABLE><P>Incubate at 50℃ for 60min.</P><TABLE><THEAD><TR><TH>Fragements</TH><TH>Length(bp)</TH><TH>Mass Required(ug)</TH></TR></THEAD><TBODY><TR><TD>linear PSB1C3 plasmid</TD><TD>2070</TD><TD>41.4</TD></TR><TR><TD>R0061</TD><TD>30</TD><TD>0.6</TD></TR><TR><TD>E0040</TD><TD>720</TD><TD>14.4</TD></TR><TR><TD>B0015</TD><TD>143</TD><TD>2.86</TD></TR></TBODY></TABLE><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>Seamless cloning Master Mix</TD><TD>8.5 ul</TD></TR><TR><TD>linear PSB1C3 plasmid</TD><TD>3 ul</TD></TR><TR><TD>R0061</TD><TD>1ul(+20 fold ddH2O)</TD></TR><TR><TD>E0040</TD><TD>1ul(+1 fold ddH2O)</TD></TR><TR><TD>B0015</TD><TD>1ul(+9 folds ddH2O)</TD></TR><TR><TD>ddH2O</TD><TD>5.5ul</TD></TR></TBODY></TABLE><H5>PCR for Cas9-PART I and Cas9-PART II</H5><P>material:</P><P>Cas9 plasmid(511.3 ng/uL)</P><P>PCR protocol:</P><P>For Cas9-PART I and Cas9-PART II, primers including Cas9-1-F, Cas9-1-R,Cas9-2-F, Cas9-2-R, protocol as following:</P><UL><LI>10μl Taq Master Mix</LI><LI>1μl Cas9-1-F(or Cas9-2-F) Primer</LI><LI>1μl Cas9-1-R(or Cas9-2-R) Primer</LI><LI>0.25μl Taq DNA Polymerase</LI><LI>6.75μl ddH2O</LI><LI>1μl Cas9 Plasmid</LI></UL><P>For Cas9, Primers produced from Sangon Biotech Co.,Ltd.</P><TABLE><THEAD><TR><TH>Primer</TH><TH>Sequence(5'-3')</TH></TR></THEAD><TBODY><TR><TD>Cas9-1-F</TD><TD>ggaatggataagaaatactcaataggcttag</TD></TR><TR><TD>Cas9-1-R</TD><TD>gctgtttcatcaccttatcatcaaaga</TD></TR><TR><TD>Cas9-2-F</TD><TD>gataaggtgatgaaacagcttaaacgtc</TD></TR><TR><TD>Cas9-2-R</TD><TD>ctactagtgggaccattcaaaacagcatagc</TD></TR></TBODY></TABLE><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 4.5min)</LI><LI>Elongation for whole at 72 degree celcius 7 min</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>Gel Electrophoresis Characterization</H5><P>Cas9-PART I and Cas9-PART II</P><H5>Gel Extraction of Cas9-PART I and Cas9-PART II</H5><P>Material including: Cas9-PART I and Cas9-PART II</P><P>Preparation:</P><OL><LI>Check out whether ethyl alcohol is added into Wash Solution. </LI><LI>Check out whether Buffer B2 has sediment. </LI><LI>Adjust the thermostat water bath at 50 degree centigrade.</LI></OL><P>Procedure:</P><OL><LI>Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers. </LI><LI>Measure out the weight of tubes of gel. </LI><LI>Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins. </LI><LI>Optional: If the length of the DNA strip is shorter than 500 bp, it would be better add isopropanol one third volume of Buffer B2. </LI><LI>After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s. </LI><LI>Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution. </LI><LI>Do the step 5 again. </LI><LI>Centrifuge the empty columns at 9000xg about 1min. </LI><LI>Open the columns and air them about 10 mins. </LI><LI>Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min. </LI><LI>Keep the DNA solution and often measure the concentration of DNA and A260/A280 for further study.</LI></OL><P>the concentration as following:</P><P>Cas9-PART I : 31 ng/uL</P><P>Cas9-PART II: 16 ng/uL</P><H5>Seamless Clone of tRNA Plasmid</H5><P>Protocol:</P><UL><LI>8.5 ul Seamless cloning Master Mix</LI><LI>3 ul linear PSB1C3 plasmid</LI><LI>1 ul tRNA </LI><LI>7.5 ul ddH2O</LI></UL><P>Incubate at 50℃ for 60min.</P><H5>Seamless Clone of Anchor Plasmid</H5><P>Protocol:</P><UL><LI>8.5 ul Seamless cloning Master Mix</LI><LI>3 ul linear PSB1C3 plasmid</LI><LI>1 ul T7 promoter</LI><LI>1 ul COMPX</LI><LI>6.5 ul ddH2O</LI></UL><P>Incubate at 50℃ for 60min.</P><H5>Transformation of tRNA Plasmid &amp; Anchor Plasmid</H5><P>Procedure:</P><OL><LI>Add 4ul clone product to competent cells, 1mix well, incubate on ice for 30min.</LI><LI>Heat in 42℃ bath for 90s. Then put on ice for 5min.</LI><LI>Add 500ul SOC culture, incubate at 37℃ for 45min.</LI><LI>Spread on plate with chloramphenicol, incubate overnight at 37℃.</LI></OL><LI><TIME class="cbp_tmtime" datetime="2015-08-04 00:00"><SPAN>2015</SPAN><SPAN>4/8</SPAN></TIME><DIV class="cbp_tmlabel"><H5>PCR for micF and C0061</H5><P>Protocol:</P><TABLE><THEAD><TR><TH>Ingradients</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>10X PCR buffer</TD><TD>8</TD></TR><TR><TD>dNTPs</TD><TD>1.6</TD></TR><TR><TD>micF-F</TD><TD>4</TD></TR><TR><TD>micF-luxI-R</TD><TD>4</TD></TR><TR><TD>Top10 Solution</TD><TD>8</TD></TR><TR><TD>Pfu Polymerase</TD><TD>0.5</TD></TR><TR><TD>ddH2O</TD><TD>53.9</TD></TR></TBODY></TABLE><TABLE><THEAD><TR><TH>Ingradients</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>10X PCR buffer</TD><TD>8</TD></TR><TR><TD>dNTPs</TD><TD>1.6</TD></TR><TR><TD>C0061-F</TD><TD>4</TD></TR><TR><TD>C0061-B0015-R</TD><TD>4</TD></TR><TR><TD>C0061</TD><TD>8</TD></TR><TR><TD>Pfu Polymerase</TD><TD>0.5</TD></TR><TR><TD>ddH2O</TD><TD>53.9</TD></TR></TBODY></TABLE><P>PCR cycle including:</P></DIV></LI><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min 30s)</LI><LI>Elongation for whole at 72 degree celcius 7 min</LI><LI>4 celcius degree for preserveation.</LI><H5>Seamless Clone of circuit I</H5><P>material:</P><P>pSB1C3(XbaI&amp;SpeI digest) 2062bp 7.1ng/uL</P><P>micF 402bp 58.7ng/uL</P><P>C0061 656bp 27.4ng/uL</P><P>B0015 142bp 7.1ng/uL</P><P>seamless clone kit</P><P>protocol:</P><P>pSB1C3 12.4uL</P><P>micF 0.5uL</P><P>C0061 1.73uL</P><P>B0015 1.46uL</P><P>seamless clone mix 8.5uL</P><P>Incubate at 50℃ for 60min.</P><H5>Transformation of circuit I</H5><P>Procedure:</P><OL><LI>Add 4ul clone product to competent cells, 1mix well, incubate on ice for 30min.</LI><LI>Heat in 42℃ bath for 90s. Then put on ice for 3min.</LI><LI>Add 500ul LB culture(no antibiotics), shaking at 37℃ for 30min.</LI><LI>Spread on plate with chloramphenicol, incubate overnight at 37℃.</LI></OL><LI><TIME class="cbp_tmtime" datetime="2015-08-05 00:00"><SPAN>2015</SPAN><SPAN>5/8</SPAN></TIME><DIV class="cbp_tmlabel"><H5>Seamless Clone of circuit I</H5><P>material:</P><P>pSB1C3(XbaI&amp;SpeI digest) 2062bp 7.1ng/uL</P><P>micF 402bp 58.7ng/uL</P><P>C0061 656bp 27.4ng/uL</P><P>B0015 142bp 7.1ng/uL</P><P>seamless clone kit</P><P>protocol:</P><P>pSB1C3 12.4uL</P><P>micF 0.5uL</P><P>C0061 1.73uL</P><P>B0015 1.46uL</P><P>seamless clone mix 8.5uL</P><P>Incubate at 50℃ for 60min.</P><H5>Transformation of circuit I</H5><P>Protocol:</P></DIV></LI><LI>Add 4ul clone product to competent cells, 1mix well, incubate on ice for 30min.</LI><LI>Heat in 42℃ bath for 90s. Then put on ice for 3min.</LI><LI>Add 500ul LB culture(no antibiotics), shaking at 37℃ for 30min.</LI><LI>Spread on plate with chloramphenicol, incubate overnight at 37℃.</LI><H5>Transformation of pSB1C3</H5><P>protocol:</P><OL><LI>Add 2ul clone product to competent cells, 1mix well, incubate on ice for 30min.</LI><LI>Heat in 42℃ bath for 90s. Then put on ice for 3min.</LI><LI>Add 500ul LB culture(no antibiotics), shaking at 37℃ for 30min.</LI><LI>Spread on plate with chloramphenicol, incubate overnight at 37℃.</LI></OL><H5>Double digest of pSB1C3</H5><P>protocol:</P><P>pSB1C3 plasmid solution 10ul</P><P>XbaI 1ul</P><P>SpeI 1ul</P><P>Tango.buffer 2ul</P><P>ddH2O 6ul</P><P>incubate at 37℃ for 2 hours.</P><LI><TIME class="cbp_tmtime" datetime="2015-08-06 00:00"><SPAN>2015</SPAN><SPAN>6/8</SPAN></TIME><DIV class="cbp_tmlabel"><H5>Colonial PCR of PSB1C3-T7-COMPX &amp; PSB1C3-T7-tRNA</H5><P>Protocol(volume 20ul)</P></DIV></LI><LI>1μl T7-F Primer</LI><LI>1μl COMPX-R or tRNA-R Primer</LI><LI>10μl Mix</LI><LI>1μl Bacteria Clone</LI><LI>7μl ddH2O</LI><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 7min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s), annealing(55 degree celcius, 30s) and elongation(72 degree celcius 1.5 min)</LI><LI>Elongation for whole at 72 degree celcius 10min.</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>Colonial PCR of PSB1C3-T7-cheZ &amp; PSB1C3-R0051-E0040-B0015</H5><P>Protocol(volume 13.5ul)</P><UL><LI>0.25μl Primer 1</LI><LI>0.25μl Primer 2</LI><LI>6.5μl Mix</LI><LI>1μl Bacteria Clone</LI><LI>5.5μl ddH2O</LI></UL><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 7min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s), annealing(55 degree celcius, 30s) and elongation(72 degree celcius 1 min)</LI><LI>Elongation for whole at 72 degree celcius 10min.</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>Overlap Circuit of gRNA-Cas9</H5><H5>Overlapping Fragments</H5><P>In order to ligate the fragements gRNA-ArcB or gRNA-EmrE, T7-tet, Cas9-PART I and Cas9-PART II, we need to make the equal mole proportion.</P><P>The mole of DNA molecule is calculated in the following ways:</P><P>among which, c is the concentration of DNA, V is the volumn required and n is the length of DNA. Because of the same molar mass of As, Gs, Cs and Ts, computation of same molar mass is omitted.</P><P>According to our sample extracted from PCR and calculation, ingradients characterize as below:</P><TABLE><THEAD><TR><TH>Name</TH><TH>Concentration(ng/uL)</TH><TH>Length(bp)</TH></TR></THEAD><TBODY><TR><TD>gRNA-ArcB</TD><TD>44</TD><TD>162</TD></TR><TR><TD>gRNA-EmrE</TD><TD>41</TD><TD>162</TD></TR><TR><TD>T7-tet</TD><TD>30</TD><TD>56</TD></TR><TR><TD>Cas9-PART I</TD><TD>31</TD><TD>1954</TD></TR><TR><TD>Cas9-PART II</TD><TD>16</TD><TD>2358</TD></TR><TR><TD>Total</TD><TD>4530</TD></TR></TBODY></TABLE><P>The theoratical volumn proportion is: </P><UL><LI>gRNA-ArcB-T7-tet-Cas9-PART I-Cas9-PART II : 2: 1: 31: 64</LI><LI>gRNA-EmrE-T7-tet-Cas9-PART I-Cas9-PART II: 2: 1: 31: 64</LI></UL><P>Firstly, to ligate these parts, we need 10 cycles without primer:</P><P>for circuit gRNA-ArcB-T7-tet-Cas9-PART I-Cas9-PART II:</P><TABLE><THEAD><TR><TH>Name</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>gRNA-ArcB</TD><TD>0.2</TD></TR><TR><TD>T7-tet</TD><TD>0.1</TD></TR><TR><TD>Cas9-PART I</TD><TD>3.1</TD></TR><TR><TD>Cas9-PART II</TD><TD>6.4</TD></TR><TR><TD>Taq Polymerase</TD><TD>0.25</TD></TR><TR><TD>dNTP(+MgCl2)</TD><TD>2</TD></TR><TR><TD>10X buffer</TD><TD>2</TD></TR><TR><TD>DMSO</TD><TD>1</TD></TR><TR><TD>ddH2O</TD><TD>4.95</TD></TR></TBODY></TABLE><P>for circuit gRNA-EmrE-T7-tet-Cas9-PART I-Cas9-PART II:</P><TABLE><THEAD><TR><TH>Name</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>gRNA-EmrE</TD><TD>0.2</TD></TR><TR><TD>T7-tet</TD><TD>0.1</TD></TR><TR><TD>Cas9-PART I</TD><TD>3.1</TD></TR><TR><TD>Cas9-PART II</TD><TD>6.4</TD></TR><TR><TD>Taq Polymerase</TD><TD>0.25</TD></TR><TR><TD>dNTP(+MgCl2)</TD><TD>2</TD></TR><TR><TD>10X buffer</TD><TD>2</TD></TR><TR><TD>DMSO</TD><TD>1.25</TD></TR><TR><TD>ddH2O</TD><TD>4.7</TD></TR></TBODY></TABLE><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>10 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 6.5min)</LI><LI>Elongation for whole at 72 degree celcius 1min</LI><LI>4 celcius degree for preserveation.</LI></OL><P>Then, we extracted 5 ul as template, and 15 ul adding with primer:</P><P>Primers were synthesized from <EM>Sangon Biotech. Co.,Ltd.</EM></P><TABLE><THEAD><TR><TH>Primers</TH><TH>Sequence(5' to 3')</TH></TR></THEAD><TBODY><TR><TD>gRNA-F</TD><TD>GCGTCTAGATTGACAGCTAGCTCAGTAGGTATAAT</TD></TR><TR><TD>Cas9-2-R</TD><TD>CTACTAATGGGACCATTCAAAACAGCATAGC</TD></TR></TBODY></TABLE><P>5ul-template protocol:</P><TABLE><THEAD><TR><TH>Name</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>Template</TD><TD>5</TD></TR><TR><TD>Taq Polymerase</TD><TD>0.25</TD></TR><TR><TD>dNTP(+MgCl2)</TD><TD>2</TD></TR><TR><TD>10X buffer</TD><TD>2</TD></TR><TR><TD>DMSO</TD><TD>1.25</TD></TR><TR><TD>ddH2O</TD><TD>7.5</TD></TR><TR><TD>gRNA-F</TD><TD>1</TD></TR><TR><TD>Cas9-2-R</TD><TD>1</TD></TR></TBODY></TABLE><P>15ul-protocol:</P><TABLE><THEAD><TR><TH>Name</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>Solution</TD><TD>15</TD></TR><TR><TD>gRNA-F</TD><TD>1</TD></TR><TR><TD>Cas9-2-R</TD><TD>1</TD></TR><TR><TD>DMSO</TD><TD>1</TD></TR><TR><TD>10X buffer</TD><TD>2</TD></TR></TBODY></TABLE><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 6.5min)</LI><LI>Elongation for whole at 72 degree celcius 1min</LI><LI>4 celcius degree for preserveation.</LI></OL><LI><TIME class="cbp_tmtime" datetime="2015-08-07 00:00"><SPAN>2015</SPAN><SPAN>7/8</SPAN></TIME><DIV class="cbp_tmlabel"><H5>Plasmid Extraction of Circuit 1, B0015, T7</H5><P>Procedure including:</P><P>1.Absorb 6 mL bacteria solution in to EP tubes, centrifuge them at 12,000xg 1mins and then discard the culture medium.</P><P>2.Add 250 ul Buffer S1 into sediment, and use spearhead to make bacteria suspended.</P><P>3.Add 250 ul Buffer S2, and overturn the EP tubes 8 times immediately and tenderly. Stand tubes about 3 mins.</P><P>4.Add 350 ul Buffer W3 and overturn the EP tubes 10 times again immediately and tenderly.</P><P>5.Centrifuge tubes at 12,000xg about 10 mins.</P><P>6.Pour the supernatant liquid into absorption columns ,centrifuge them at 12,000xg about 30 sec and then discard the liquid in the collecting pipe.</P><P>7.Add 500 ul Buffer W1 and centrifuge them at 12,000xg about 30 sec. Then discard the liquid in the collecting pipe.</P><P>8.Add 700 ul Buffer W2, centrifuge them at 12,000xg about 30 sec and discard the liquid in the collecting pipe.</P><P>9.Do the step 8 again.</P><P>10.Centrifuge the empty columns at 12,000xg about 1 min.</P><P>11.Put the columns in clean 1.5mL EP tubes, add 80 ul Elution Buffer on the absorption film, stand 10 min and centrifuge about 1 min at 12,000xg.</P><H5>Enzyme Digestion</H5><P>Protocol:</P><P>Name Volumn(uL)</P><P>pSB1C3 10</P><P>XbaI 1</P><P>SpeI 1</P><P>10* Buffer Tango 2</P><P>ddH2O 15</P><H5>Colonial PCR of PSB1C3-T7-COMPX</H5><P>Protocol(volume 20ul)</P></DIV></LI><LI>1μl T7-F Primer</LI><LI>1μl COMPX-R </LI><LI>10μl Mix</LI><LI>1μl Bacteria Clone</LI><LI>7μl ddH2O</LI><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 7min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s), annealing(55 degree celcius, 30s) and elongation(72 degree celcius 1.5 min)</LI><LI>Elongation for whole at 72 degree celcius 10min.</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>Double Digest of T7 Plasmid</H5><P>Protocol (Volume 20ul)</P><UL><LI>4ul Tango Buffer</LI><LI>2ul BamHI</LI><LI>2ul SalI</LI><LI>10ul T7 Plasmid</LI><LI>2ul ddH2O
                      Incubate at 37℃ for 3h.</LI></UL><H5>Colonial PCR of PSB1C3-T7-COMPX</H5><P>Conduct the PCR again.</P><P>Protocol(volume 20ul)</P><UL><LI>1μl T7-F Primer</LI><LI>1μl COMPX-R </LI><LI>10μl Mix</LI><LI>1μl Bacteria Clone</LI><LI>7μl ddH2O</LI></UL><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 7min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s), annealing(55 degree celcius, 30s) and elongation(72 degree celcius 1.5 min)</LI><LI>Elongation for whole at 72 degree celcius 10min.</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>Double Digest of T7 Plasmid</H5><P>Protocol (Volume 20ul)</P><UL><LI>4ul Tango Buffer</LI><LI>2ul BamHI</LI><LI>2ul SalI</LI><LI>10ul T7 Plasmid</LI><LI>2ul ddH2O</LI><LI>Incubate at 37℃ overnight.</LI></UL><LI><TIME class="cbp_tmtime" datetime="2015-08-08 00:00"><SPAN>2015</SPAN><SPAN>8/8</SPAN></TIME><DIV class="cbp_tmlabel"><H5>Seamless Clone of circuit I</H5><P>material:</P><P>pSB1C3(XbaI&amp;SpeI digest) 2062bp 36.7ng/uL</P><P>micF 402bp 39ng/uL</P><P>SoxS 736bp 44.4ng/uL</P><P>C0061 656bp 27.4ng/uL</P><P>B0015 142bp 22ng/uL</P><P>seamless clone kit</P><P>protocol:</P><P>pSB1C3 1.12uL</P><P>micF 0.2uL or SoxS 0.33uL</P><P>C0061 0.48uL</P><P>B0015 0.14uL</P><P>seamless clone mix 8.5uL</P><P>Incubate at 50℃ for 60min.</P><H5>Transformation of circuit I</H5><P>Procedure:</P></DIV></LI><LI>Add 4ul clone product to competent cells, 1mix well, incubate on ice for 30min.</LI><LI>Heat in 42℃ bath for 90s. Then put on ice for 3min.</LI><LI>Add 500ul LB culture(no antibiotics), shaking at 37℃ for 30min.</LI><LI>Spread on plate with chloramphenicol, incubate overnight at 37℃.</LI><H5>Colony PCR of Circuit I</H5><P>pick 16 single colonies and shaking culture for 5 hours.</P><P>protocol:</P><UL><LI>6.5 ul Taq Master Mix</LI><LI>5.5 ul ddH2O</LI><LI>0.25 ul C0061-F primer</LI><LI>0.25 ul C0061-B0015-R primer</LI><LI>0.5 ul bacterial solution</LI></UL><P>PCR cycle including:</P><OL><LI>Preheat: 95 degree celcius, 5min</LI><LI>30 cycles containing: degradation(95 degree celcius, 30s), annealing(55 degree celcius, 30s) and elongation(72 degree celcius 50 s)</LI><LI>Elongation for whole at 72 degree celcius 10min.</LI><LI>4 celcius degree for preserveation.</LI></OL><LI><TIME class="cbp_tmtime" datetime="2015-08-10 00:00"><SPAN>2015</SPAN><SPAN>10/8</SPAN></TIME><DIV class="cbp_tmlabel"><H5>PCR of C0062</H5><P>material(volume 100uL):</P></DIV></LI><LI>6 ul MgCl2</LI><LI>71 ul ddH2O</LI><LI>4 ul Primer F</LI><LI>4 ul Primer R</LI><LI>2 ul dNTP</LI><LI>1 ul pfu DNA polymerase</LI><LI>10 ul 10*PCR Buffer</LI><LI>2 ul C0062</LI><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 4.5 min)</LI><LI>Elongation for whole at 72 degree celcius 1min</LI><LI>4 celcius degree for preserveation.</LI></OL><LI><TIME class="cbp_tmtime" datetime="2015-08-11 00:00"><SPAN>2015</SPAN><SPAN>11/8</SPAN></TIME><DIV class="cbp_tmlabel"><H5>Seamless Clone of pSB1C3-R0010-C0062-B0015</H5><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Volumn</TH></TR></THEAD><TBODY><TR><TD>Seamless cloning Master Mix</TD><TD>8.5 ul</TD></TR><TR><TD>linear PSB1C3 plasmid</TD><TD>2 ul</TD></TR><TR><TD>R0010</TD><TD>0.4ul</TD></TR><TR><TD>C0062</TD><TD>0.6ul</TD></TR><TR><TD>B0015</TD><TD>2.4ul</TD></TR><TR><TD>ddH2O</TD><TD>6.1ul</TD></TR></TBODY></TABLE><P>Incubate at 50 degree celcius 1 h.</P><H5>Seamless Clone of pSB1C3-T7-COMPX</H5><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Volumn</TH></TR></THEAD><TBODY><TR><TD>Seamless cloning Master Mix</TD><TD>8.5 ul</TD></TR><TR><TD>linear PSB1C3 plasmid</TD><TD>2ul</TD></TR><TR><TD>T7</TD><TD>2ul</TD></TR><TR><TD>COMPX</TD><TD>1ul</TD></TR><TR><TD>ddH2O</TD><TD>6.5ul</TD></TR></TBODY></TABLE><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Volumn</TH></TR></THEAD><TBODY><TR><TD>Seamless cloning Master Mix</TD><TD>8.5 ul</TD></TR><TR><TD>linear PSB1C3 plasmid</TD><TD>2ul</TD></TR><TR><TD>T7</TD><TD>1ul</TD></TR><TR><TD>COMPX</TD><TD>1ul</TD></TR><TR><TD>ddH2O</TD><TD>14.5ul</TD></TR></TBODY></TABLE><P>Incubate at 50 degree celcius 1 h.</P><H5>Transformation of pSB1C3-T7-COMPX</H5><P>Procedure:</P></DIV></LI><LI>Add 4ul clone product to competent cells, 1mix well, incubate on ice for 30min.</LI><LI>Heat in 42℃ bath for 90s. Then put on ice for 5min.</LI><LI>Add 500ul SOC culture, incubate at 37℃ for 45min.</LI><LI>Spread on plate with chloramphenicol, incubate overnight at 37℃.</LI><H5>Enzyme Digestion of pSB1C3-T7-COMPX and pSB1C3-tRNA</H5><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Volumn</TH></TR></THEAD><TBODY><TR><TD>Tango Buffer</TD><TD>2ul</TD></TR><TR><TD>plasmid</TD><TD>10ul</TD></TR><TR><TD>Spe I</TD><TD>1ul</TD></TR><TR><TD>Xba I</TD><TD>1ul</TD></TR><TR><TD>ddH2O</TD><TD>6ul</TD></TR></TBODY></TABLE><P>Incubate at 37 degree celcius for 2 h.</P><LI><TIME class="cbp_tmtime" datetime="2015-08-12 00:00"><SPAN>2015</SPAN><SPAN>12/8</SPAN></TIME><DIV class="cbp_tmlabel"><H5>Transformation of pSB1C3-R0010-C0062-B0015</H5><P>Procedure:</P></DIV></LI><LI>Add 4ul clone product to competent cells, 1mix well, incubate on ice for 30min.</LI><LI>Heat in 42℃ bath for 90s. Then put on ice for 5min.</LI><LI>Add 500ul SOC culture, incubate at 37℃ for 45min.</LI><LI>Spread on plate with chloramphenicol, incubate overnight at 37℃.</LI><H5>Transformation of pSB3A3</H5><P>Procedure:</P><OL><LI>Add 4ul clone product to competent cells, 1mix well, incubate on ice for 30min.</LI><LI>Heat in 42℃ bath for 90s. Then put on ice for 5min.</LI><LI>Add 500ul SOC culture, incubate at 37℃ for 45min.</LI><LI>Spread on plate with Ampcilin, incubate overnight at 37℃.</LI></OL><H5>Plasmid Extraction of T7</H5><P>Procedure including:</P><OL><LI>Absorb 4.5 mL bacteria solution in to EP tubes, centrifuge them at 12,000xg 1mins and then discard the culture medium.</LI><LI>Add 250 ul Buffer S1 into sediment, and use spearhead to make bacteria suspended.</LI><LI>Add 250 ul Buffer S2, and overturn the EP tubes 8 times immediately and tenderly. Stand tubes about 3 mins.</LI><LI>Add 350 ul Buffer W3 and overturn the EP tubes 10 times again immediately and tenderly.</LI><LI>Centrifuge tubes at 12,000xg about 10 mins.</LI><LI>Pour the supernatant liquid into absorption columns ,centrifuge them at 12,000xg about 30 sec and then discard the liquid in the collecting pipe.</LI><LI>Add 500 ul Buffer W1 and centrifuge them at 12,000xg about 30 sec. Then discard the liquid in the collecting pipe.</LI><LI>Add 700 ul Buffer W2, centrifuge them at 12,000xg about 30 sec and discard the liquid in the collecting pipe.</LI><LI>Do the step 8 again.</LI><LI>Centrifuge the empty columns at 12,000xg about 1 min.</LI><LI>Put the columns in clean 1.5mL EP tubes, add 80 ul Elution Buffer on the absorption film, stand 10 min and centrifuge about 1 min at 12,000xg.</LI></OL><H5>PCR of OprF from PAO1</H5><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>2XTaq Mix</TD><TD>25</TD></TR><TR><TD>OprF-F</TD><TD>2.5</TD></TR><TR><TD>OprF-Ter-R</TD><TD>2.5</TD></TR><TR><TD>PAO1</TD><TD>2.5</TD></TR><TR><TD>ddH2O</TD><TD>17.5</TD></TR></TBODY></TABLE><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 2 min)</LI><LI>Elongation for whole at 72 degree celcius 10 mins</LI><LI>4 celcius degree for preserveation.</LI></OL><LI><TIME class="cbp_tmtime" datetime="2015-08-13 00:00"><SPAN>2015</SPAN><SPAN>13/8</SPAN></TIME><DIV class="cbp_tmlabel"><H5>Plasmid Extraction of R0062, B0015, SCVE, T7</H5><P>Procedure including:</P><P>1.Absorb 4.5 mL bacteria solution in to EP tubes, centrifuge them at 12,000xg 1mins and then discard the culture medium.</P><P>2.Add 250 ul Buffer S1 into sediment, and use spearhead to make bacteria suspended.</P><P>3.Add 250 ul Buffer S2, and overturn the EP tubes 8 times immediately and tenderly. Stand tubes about 3 mins.</P><P>4.Add 350 ul Buffer W3 and overturn the EP tubes 10 times again immediately and tenderly.</P><P>5.Centrifuge tubes at 12,000xg about 10 mins.</P><P>6.Pour the supernatant liquid into absorption columns ,centrifuge them at 12,000xg about 30 sec and then discard the liquid in the collecting pipe.</P><P>7.Add 500 ul Buffer W1 and centrifuge them at 12,000xg about 30 sec. Then discard the liquid in the collecting pipe.</P><P>8.Add 700 ul Buffer W2, centrifuge them at 12,000xg about 30 sec and discard the liquid in the collecting pipe.</P><P>9.Do the step 8 again.</P><P>10.Centrifuge the empty columns at 12,000xg about 1 min.</P><P>11.Put the columns in clean 1.5mL EP tubes, add 80 ul Elution Buffer on the absorption film, stand 10 min and centrifuge about 1 min at 12,000xg.</P><H5>PCR of R0062, C0051, E0040, B0015</H5><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>PCR Buffer</TD><TD>5</TD></TR><TR><TD>Primer 1</TD><TD>2.5</TD></TR><TR><TD>Primer 2</TD><TD>2.5</TD></TR><TR><TD>Plasimid Solution</TD><TD>2.5</TD></TR><TR><TD>ddH2O</TD><TD>31.9</TD></TR><TR><TD>Pfu Polymerase</TD><TD>0.6</TD></TR><TR><TD>dNTP mix</TD><TD>5</TD></TR></TBODY></TABLE><P>PCR cycle including:</P></DIV></LI><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min 30 s)</LI><LI>Elongation for whole at 72 degree celcius 10 mins</LI><LI>4 celcius degree for preserveation.</LI><H5>Pre-annealing of R0051</H5><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>R0051-F1</TD><TD>10</TD></TR><TR><TD>R0051-R1</TD><TD>10</TD></TR></TBODY></TABLE><P>Incubate solution in 95 degree celcius 30 min and then used for PCR.</P><H5>PCR of R0051, T7-OprF, T7-SCVE, SCVE, micF and soxS</H5><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>PCR Buffer</TD><TD>5</TD></TR><TR><TD>Primer 1</TD><TD>2.5</TD></TR><TR><TD>Primer 2</TD><TD>2.5</TD></TR><TR><TD>Plasimid Solution</TD><TD>2.5</TD></TR><TR><TD>ddH2O</TD><TD>31.9</TD></TR><TR><TD>Pfu Polymerase</TD><TD>0.6</TD></TR><TR><TD>dNTP mix</TD><TD>5</TD></TR></TBODY></TABLE><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 2min)</LI><LI>Elongation for whole at 72 degree celcius 10 mins</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>PCR of OprF, micF, SoxS</H5><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>PCR Buffer</TD><TD>5</TD></TR><TR><TD>Primer 1</TD><TD>2.5</TD></TR><TR><TD>Primer 2</TD><TD>2.5</TD></TR><TR><TD>Bacterial Solution</TD><TD>2.5</TD></TR><TR><TD>ddH2O</TD><TD>31.9</TD></TR><TR><TD>Pfu Polymerase</TD><TD>0.6</TD></TR><TR><TD>dNTP mix</TD><TD>5</TD></TR></TBODY></TABLE><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min 30 s)</LI><LI>Elongation for whole at 72 degree celcius 10 mins</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>Gel Extraction</H5><P>Material including: R0061, E0040, B0015, T7-OprF</P><P>Preparation:</P><P>• Check out whether ethyl alcohol is added into Wash Solution.</P><P>• Check out whether Buffer B2 has sediment.</P><P>• Adjust the thermostat water bath at 50 degree centigrade.</P><P>Procedure:</P><P>5 Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.</P><P>6 Measure out the weight of tubes of gel.</P><P>7 Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.</P><P>8 Optional: If the length of the DNA strip is shorter than 500 bp, it would be better add isopropanol one third volume of Buffer B2.</P><P>9 After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s.</P><P>10 Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.</P><P>11 Do the step 5 again.</P><P>12 Centrifuge the empty columns at 9000xg about 1min.</P><P>13 Open the columns and air them about 10 mins.</P><P>14 Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.</P><H5>PCR of OprF, SoxS, micF</H5><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>PCR Buffer</TD><TD>5</TD></TR><TR><TD>Primer 1</TD><TD>2.5</TD></TR><TR><TD>Primer 2</TD><TD>2.5</TD></TR><TR><TD>Bactieral Solution</TD><TD>2.5</TD></TR><TR><TD>ddH2O</TD><TD>31.9</TD></TR><TR><TD>Pfu Polymerase</TD><TD>0.6</TD></TR><TR><TD>dNTP mix</TD><TD>5</TD></TR></TBODY></TABLE><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(<EM>Temperature gradient: 48 degree celcius, 50 degree celcius, 53 degree celcius and 55 degree celcius</EM> 30s) and elongation(72 degree celcius 2min)</LI><LI>Elongation for whole at 72 degree celcius 10 mins</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>Gel Extraction</H5><P>Material including: T7-SCVE, SCVE</P><P>Preparation:</P><P>• Check out whether ethyl alcohol is added into Wash Solution.</P><P>• Check out whether Buffer B2 has sediment.</P><P>• Adjust the thermostat water bath at 50 degree centigrade.</P><P>Procedure:</P><P>5 Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.</P><P>6 Measure out the weight of tubes of gel.</P><P>7 Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.</P><P>8 Optional: If the length of the DNA strip is shorter than 500 bp, it would be better add isopropanol one third volume of Buffer B2.</P><P>9 After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s.</P><P>10 Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.</P><P>11 Do the step 5 again.</P><P>12 Centrifuge the empty columns at 9000xg about 1min.</P><P>13 Open the columns and air them about 10 mins.</P><P>14 Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min</P><H5>Enzyme Digestion of pSB1C3</H5><P>incubate at 37 degree celcius about 2 h</P><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>Enzyme 1</TD><TD>1</TD></TR><TR><TD>Enzyme 2</TD><TD>1</TD></TR><TR><TD>pSB1C3</TD><TD>4</TD></TR><TR><TD>Tango Buffer</TD><TD>4</TD></TR><TR><TD>ddH2O</TD><TD>10</TD></TR></TBODY></TABLE><TABLE><THEAD><TR><TH>strategy</TH><TH>Enzyme 1</TH><TH>Enzyme 2</TH></TR></THEAD><TBODY><TR><TD>Strategy 1</TD><TD>EcoRI</TD><TD>PstI</TD></TR><TR><TD>Strategy 2</TD><TD>EcoRI</TD><TD>SpeI</TD></TR><TR><TD>Strategy 1</TD><TD>XbaI</TD><TD>PstI</TD></TR><TR><TD>Strategy 2</TD><TD>XbaI</TD><TD>SpeI</TD></TR></TBODY></TABLE><LI><TIME class="cbp_tmtime" datetime="2015-08-14 00:00"><SPAN>2015</SPAN><SPAN>14/8</SPAN></TIME><DIV class="cbp_tmlabel"><H5>PCR of C0061, micF/SoxS-binding C0061</H5><P>for C0061</P><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>PCR Buffer</TD><TD>5</TD></TR><TR><TD>C0061-F</TD><TD>2.5</TD></TR><TR><TD>C0061-B0015-RI</TD><TD>2.5</TD></TR><TR><TD>Plasimid Solution</TD><TD>2.5</TD></TR><TR><TD>ddH2O</TD><TD>31.9</TD></TR><TR><TD>Pfu Polymerase</TD><TD>0.6</TD></TR><TR><TD>dNTP mix</TD><TD>5</TD></TR></TBODY></TABLE><P>for micF-C0061</P><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>PCR Buffer</TD><TD>5</TD></TR><TR><TD>micF-R-BB</TD><TD>2.5</TD></TR><TR><TD>B0034-C0061-R</TD><TD>2.5</TD></TR><TR><TD>Plasimid Solution</TD><TD>2.5</TD></TR><TR><TD>ddH2O</TD><TD>31.9</TD></TR><TR><TD>Pfu Polymerase</TD><TD>0.6</TD></TR><TR><TD>dNTP mix</TD><TD>5</TD></TR></TBODY></TABLE><P>for SoxS-C0061</P><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>PCR Buffer</TD><TD>5</TD></TR><TR><TD>SoxS-R-BB</TD><TD>2.5</TD></TR><TR><TD>B0034-C0061-R</TD><TD>2.5</TD></TR><TR><TD>Plasimid Solution</TD><TD>2.5</TD></TR><TR><TD>ddH2O</TD><TD>31.9</TD></TR><TR><TD>Pfu Polymerase</TD><TD>0.6</TD></TR><TR><TD>dNTP mix</TD><TD>5</TD></TR></TBODY></TABLE><P>PCR cycle including:</P></DIV></LI><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 2min)</LI><LI>Elongation for whole at 72 degree celcius 10 mins</LI><LI>4 celcius degree for preserveation.</LI><H5>Seamless ligation of pSB1C3-T7-SCVE-B0015</H5><TABLE><THEAD><TR><TH>Ingradients</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>pSB1C3</TD><TD>1.2</TD></TR><TR><TD>T7-SCVE</TD><TD>1</TD></TR><TR><TD>SCVE</TD><TD>1</TD></TR><TR><TD>B0015</TD><TD>1</TD></TR><TR><TD>Seamless Cloning mix</TD><TD>8.5</TD></TR><TR><TD>ddH2O</TD><TD>to 20</TD></TR></TBODY></TABLE><P>Incubate at 50 degree celcuis 1 h</P><LI><TIME class="cbp_tmtime" datetime="2015-08-15 00:00"><SPAN>2015</SPAN><SPAN>15/8</SPAN></TIME><DIV class="cbp_tmlabel"><H5>PCR of OprF, SoxS</H5><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>PCR Buffer</TD><TD>5</TD></TR><TR><TD>Primer 1</TD><TD>2.5</TD></TR><TR><TD>Primer 2</TD><TD>2.5</TD></TR><TR><TD>Bacterial Solution</TD><TD>2.5</TD></TR><TR><TD>ddH2O</TD><TD>31.9</TD></TR><TR><TD>Pfu Polymerase</TD><TD>0.6</TD></TR><TR><TD>dNTP mix</TD><TD>5</TD></TR></TBODY></TABLE><P>PCR cycle including:</P></DIV></LI><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min 30 s)</LI><LI>Elongation for whole at 72 degree celcius 10 mins</LI><LI>4 celcius degree for preserveation.</LI><H5>PCR of micF-C0061</H5><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>PCR Buffer</TD><TD>5</TD></TR><TR><TD>micF-F-BB</TD><TD>2.5</TD></TR><TR><TD>B0034-C0061-R</TD><TD>2.5</TD></TR><TR><TD>Bacterial Solution</TD><TD>2.5</TD></TR><TR><TD>ddH2O</TD><TD>31.9</TD></TR><TR><TD>Pfu Polymerase</TD><TD>0.6</TD></TR><TR><TD>dNTP mix</TD><TD>5</TD></TR></TBODY></TABLE><P>PCR cycle including:</P><OL><LI>Preheat: 94 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min 30 s)</LI><LI>Elongation for whole at 72 degree celcius 10 mins</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>Gel Extraction</H5><P>Material including: micF, pSB1C3</P><P>Preparation:</P><P>• Check out whether ethyl alcohol is added into Wash Solution.</P><P>• Check out whether Buffer B2 has sediment.</P><P>• Adjust the thermostat water bath at 50 degree centigrade.</P><P>Procedure:</P><P>5 Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.</P><P>6 Measure out the weight of tubes of gel.</P><P>7 Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.</P><P>8 Optional: If the length of the DNA strip is shorter than 500 bp, it would be better add isopropanol one third volume of Buffer B2.</P><P>9 After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s.</P><P>10 Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.</P><P>11 Do the step 5 again.</P><P>12 Centrifuge the empty columns at 9000xg about 1min.</P><P>13 Open the columns and air them about 10 mins.</P><P>14 Put the columns into clean 1.5 mL EP tubes and add 15-40uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.</P><H5>Ligation of pSB1C3-tRNA</H5><P>incubate at 16 degree celcius more than 20 h</P><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Volumn(uL)</TH></TR></THEAD><TBODY><TR><TD>T4 DNA ligation Buffer</TD><TD>2</TD></TR><TR><TD>T4 DNA ligase</TD><TD>1</TD></TR><TR><TD>pSB1C3 digested by EcoRI and PstI</TD><TD>1</TD></TR><TR><TD>tRNA</TD><TD>6</TD></TR><TR><TD>ddH2O</TD><TD>11</TD></TR></TBODY></TABLE><H5>Seamless Cloning of pSB1C3-micF-C0061-B0015 </H5><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Volumn</TH></TR></THEAD><TBODY><TR><TD>Seamless cloning Master Mix</TD><TD>8.5 ul</TD></TR><TR><TD>linear PSB1C3 plasmid</TD><TD>2 ul</TD></TR><TR><TD>micF</TD><TD>1ul</TD></TR><TR><TD>C0061</TD><TD>1ul</TD></TR><TR><TD>B0015</TD><TD>1ul</TD></TR><TR><TD>ddH2O</TD><TD>6.5ul</TD></TR></TBODY></TABLE><P>Incubate at 50℃ for 60min.</P><H5>Seamless Cloning of pSB1C3-T7-COMPX </H5><TABLE><THEAD><TR><TH>Ingredients</TH><TH>Volumn</TH></TR></THEAD><TBODY><TR><TD>Seamless cloning Master Mix</TD><TD>8.5 ul</TD></TR><TR><TD>linear PSB1C3 plasmid</TD><TD>2 ul</TD></TR><TR><TD>COMPX</TD><TD>1ul</TD></TR><TR><TD>ddH2O</TD><TD>8.5ul</TD></TR></TBODY></TABLE><P>Incubate at 50℃ for 60min.</P><H5>Transformation of pSB1C3-micF-C0061-B0015 and pSB1C3-T7-COMPX</H5><P>Procedure:</P><OL><LI>Add 4ul clone product to competent cells, 1mix well, incubate on ice for 30min.</LI><LI>Heat in 42℃ bath for 90s. Then put on ice for 5min.</LI><LI>Add 500ul SOC culture, incubate at 37℃ for 45min.</LI><LI>Spread on plate with chloramphenicol, incubate overnight at 37℃.</LI></OL><DIV id="Daily-Notes-Stage-2" class="row"><DIV class="card hoverable"><DIV class="col s12"><DIV class="card-content"><UL class="cbp_tmtimeline"><LI><TIME class="cbp_tmtime" datetime="2015-08-17 00:00"><SPAN>2015</SPAN><SPAN>17/8</SPAN></TIME><DIV class="cbp_tmlabel"><H4>Construction of cheZ</H4><H5>PCR of CheZ</H5><P>CheZ:</P><P>Material(volume 50ul)</P><P>CheZ-BB-F:1ul</P><P>CheZ-BB-R:1ul</P><P>Primestar:25ul</P><P>ddH2O:21ul</P><P>PAO1:2ul</P><P>Divide the 50ul into 2 parts. Both are</P><P>25ul</P><P>PCR cycle including:</P></DIV></LI><LI>Preheat: 98 degree celcius, 5min</LI><LI>35 cycles containing: Degradation(94 degree celcius, 30s; Annealing(55 degree celcius, 30s and elongation(72 degree celcius 1.5 min).</LI><LI>Elongation for whole at 72 degree celcius 1min</LI><LI>4 celcius degree for preserveation.</LI><H5>Gel Electrophoresis Characterization</H5><P>CheZ(PCR)</P><H5>Gel Extraction</H5><P>Material including: CheZ(PCR)</P><P>Preparation:</P><P>•Check out whether ethyl alcohol is added into Wash Solution.</P><P>•Check out whether Buffer B2 has sediment.</P><P>•Adjust the thermostat water bath at 50 degree centigrade.</P><P>Procedure:</P><OL><LI>Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.</LI><LI>Measure out the weight of tubes of gel.</LI><LI>Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.</LI><LI>Add isopropanol one third volume of Buffer B2.</LI><LI>After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s, then add the solutions into the absorption columns again and centrifuge columns again.</LI><LI>Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.</LI><LI>Do the step 6 again.</LI><LI>Centrifuge the empty columns at 9000xg about 1min.</LI><LI>Open the columns and air them about 10 mins.</LI><LI>Put the columns into clean 1.5 mL EP tubes and add 20uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.</LI></OL><H5>Double digest of CheZ and PSB1C3</H5><P>Detect the concentration of CheZ:16.9ng/ul</P><P>Volume:20ul</P><P>Material:</P><P>CheZ:15ul</P><P>EcoRI:1ul</P><P>PstI:2ul</P><P>Tango:2ul</P><P>37℃digest 2 hours.</P><P>Detect the concentration of PSB1C3:74.1ng/ul</P><P>Volume:20ul</P><P>Material:</P><P>PSB1C3:5ul</P><P>EcoR1:1ul</P><P>Pst1:2ul</P><P>Tango:2ul</P><P>ddH2O:10ul</P><P>37℃digest 2 hours.</P><H5>Gel Electrophoresis Characterization</H5><P>CheZ(ep)</P><P>PSB1C3(ep)</P><H5>Gel Extraction</H5><P>Material including: CheZ(ep)</P><P>Preparation:</P><P>•Check out whether ethyl alcohol is added into Wash Solution.</P><P>•Check out whether Buffer B2 has sediment.</P><P>•Adjust the thermostat water bath at 50 degree centigrade.</P><P>Procedure:</P><OL><LI>Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.</LI><LI>Measure out the weight of tubes of gel.</LI><LI>Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.</LI><LI>Add isopropanol one third volume of Buffer B2.</LI><LI>After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s, then add the solutions into the absorption columns again and centrifuge columns again.</LI><LI>Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.</LI><LI>Do the step 10 again.</LI><LI>Centrifuge the empty columns at 9000xg about 1min.</LI><LI>Open the columns and air them about 10 mins.</LI><LI>Put the columns into clean 1.5 mL EP tubes and add 20uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.</LI></OL><H5>Ligation of PSB1C3 and CheZ</H5><P>In order to ligate the CheZ and PSB1C3, we need to make the equal mole proportion.</P><P>The mole of DNA molecule is calculated in the following ways:</P><P>C1V1/n1=C2V2/n2=……=0.1~0.3</P><P>Among which, c is the concentration of DNA, V is the volumn required and n is the length of DNA. Because of the same molar mass of As, Gs, Cs and Ts, computation of same molar mass is omitted.</P><P>At first, we need the mass of PSB1C3 at 50ng. The concentrate of the plasmid we use is 8.4ng/ul while the one of the CheZ is 16.7ng/ul. So we choose to add 6ul into the reaction. Through the relation of the mole of DNA, we get the volume of the CheZ. </P><P>Material:</P><P>CheZ:4ul</P><P>PSB1C3:6ul</P><P>Solution I:10ul</P><P>16℃ligate over night, prepare for the Transformation of the next day.</P><H5>Transformation of CheZ and PSB1C3 Plasmid</H5><P>Procedure:</P><OL><LI>Add 4ul clone product to competent cells, 1mix well, incubate on ice for 30min.</LI><LI>Heat in 42℃ bath for 90s. Then put on ice for 5min.</LI><LI>Add 900ul LB culture, incubate at 37℃ for 60min. Centrifuge the culture at 4000×1min, give away 850ul culture. </LI><LI>Spread on plate with chloramphenicol, incubate overnight at 37℃.</LI></OL><H5>Colonial PCR of CheZ</H5><P>Take one bacterial colony into 10ul H2O for the PCR.</P><P>Protocol(volume 13ul)</P><UL><LI>1μl CheZ-BB-F</LI><LI>1μl CheZ-BB-R</LI><LI>10μl Mix</LI><LI>1μl Bacteria Clone</LI></UL><P>PCR cycle including:</P><OL><LI>Preheat: 98 degree celcius, 7min</LI><LI>35 cycles containing: degradation(98 degree celcius, 30s), annealing(55 degree celcius, 30s) and elongation(72 degree celcius 1.5 min)</LI><LI>Elongation for whole at 72 degree celcius 10min.</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>Gel Electrophoresis Characterization</H5><P>Failed.</P><P>However, we incubate two of the bacteria into 50ul centrifuge tube at 37℃, we find it grow well. </P><H3 id="construction-of-t7-scve">Construction of T7-SCVE</H3><H5>PCR of T7 and SCVE</H5><P>Preparation:</P><P>Add 10ul T7-F,10ul T7-R ,heating at 95 degree celcius for 30min and cool to 37 degree celcius.The mix will be used as template for the PCR.</P><H5>Material(volume 50ul)</H5><P>T7-exten-F:1ul</P><P>T7-exten-R:1ul</P><P>Primestar:25ul</P><P>ddH2O:18ul</P><P>T7 as template:5ul</P><P>Divide the 50ul into 2 parts. Both are</P><P>25ul</P><H5>Material(volume 50ul)</H5><P>SCVE-F:1ul</P><P>SCVE-R:1ul</P><P>Primestar:25ul</P><P>ddH2O:22ul</P><P>SCVE plasmid:1ul</P><P>Divide the 50ul into 2 parts. Both are</P><P>25ul</P><P>PCR cycle including:</P><OL><LI>Preheat: 98 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1 min)</LI><LI>Elongation for whole at 72 degree celcius 10min</LI><LI>4 celcius degree for preserveation.
                      Gel Electrophoresis Characterization
                      T7(PCR)</LI></OL><H5>Gel Electrophoresis Characterization</H5><P>T7(PCR)</P><P>SCVE(PCR)</P><H5>Gel Extraction</H5><P>Material including: T7,SCVE(PCR)</P><P>Preparation:</P><P>•Check out whether ethyl alcohol is added into Wash Solution.</P><P>•Check out whether Buffer B2 has sediment.</P><P>•Adjust the thermostat water bath at 50 degree centigrade.</P><P>Procedure:</P><OL><LI>Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.</LI><LI>Measure out the weight of tubes of gel.</LI><LI>Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.</LI><LI>Add isopropanol one third volume of Buffer B2.</LI><LI>After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s, then add the solutions into the absorption columns again and centrifuge columns again.</LI><LI>Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.</LI><LI>Do the step 10 again.</LI><LI>Centrifuge the empty columns at 9000xg about 1min.</LI><LI>Open the columns and air them about 10 mins.</LI><LI>Put the columns into clean 1.5 mL EP tubes and add 20uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.</LI></OL><H5>Double digest of T7 and SCVE</H5><P>Detect the concentration of T7:35ng/ul</P><P>Volume:20ul</P><P>Material:</P><P>T7:15ul</P><P>XbaI:1ul</P><P>HindIII:2ul</P><P>Tango:2ul</P><P>37℃digest 2 hours.</P><P>Detect the concentration of SCVE:7.8ng/ul</P><P>volume:20ul</P><P>Material:</P><P>SCVE:15ul</P><P>PstI:1ul</P><P>HindIII:1ul</P><P>bufferR:2ul</P><P>ddH2O:1ul</P><P>37℃digest 2 hours.</P><P>Detect the concentration of PSB1C3:160.2ng/ul</P><P>Volume:20ul</P><H5>Material</H5><P>PSB1C3:5ul</P><P>XbaI:1ul</P><P>Pst1:2ul</P><P>Tango:2ul</P><P>ddH2O:10ul</P><P>37℃digest 2 hours.</P><H5>Gel Electrophoresis Characterization</H5><P>T7(ep)</P><P>SCVE(ep)</P><P>PSB1C3(xp)</P><H5>Gel Extraction</H5><P>Material including: T7,SCVE(ep)</P><P>Preparation:</P><P>•Check out whether ethyl alcohol is added into Wash Solution.</P><P>•Check out whether Buffer B2 has sediment.</P><P>•Adjust the thermostat water bath at 50 degree centigrade.</P><P>Procedure:</P><OL><LI>Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.</LI><LI>Measure out the weight of tubes of gel.</LI><LI>Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.</LI><LI>Add isopropanol one third volume of Buffer B2.</LI><LI>After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s, then add the solutions into the absorption columns again and centrifuge columns again.</LI><LI>Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.</LI><LI>Do the step 10 again.</LI><LI>Centrifuge the empty columns at 9000xg about 1min.</LI><LI>Open the columns and air them about 10 mins.</LI><LI>Put the columns into clean 1.5 mL EP tubes and add 20uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.</LI></OL><H5>Ligation of PSB1C3 ,T7 and SCVE</H5><P>In order to ligate the T7,SCVE and PSB1C3, we need to make the equal mole proportion.</P><P>The mole of DNA molecule is calculated in the following ways:</P><P>C1V1/n1=C2V2/n2=……=0.1~0.3</P><P>Among which, c is the concentration of DNA, V is the volumn required and n is the length of DNA. Because of the same molar mass of As, Gs, Cs and Ts, computation of same molar mass is omitted.</P><P>Material:</P><P>T7:1ul</P><P>SCVE:2ul</P><P>PSB1C3:6ul</P><P>Solution I:10ul</P><P>16℃ligate over night, prepare for the Transformation of the next day.</P><H5>Transformation of T7+SCVE Plasmid</H5><P>Procedure:</P><OL><LI>Add 4ul clone product to competent cells, 1mix well, incubate on ice for 30min.</LI><LI>Heat in 42℃ bath for 90s. Then put on ice for 5min.</LI><LI>Add 900ul LB culture, incubate at 37℃ for 60min. Centrifuge the culture at 4000×1min, give away 850ul culture.</LI><LI>Spread on plate with chloramphenicol, incubate overnight at 37℃.</LI></OL><H5>Colonial PCR of T7+SCVE</H5><P>Take one bacterial colony into 10ul H2O for the PCR.</P><P>Protocol(volume 13ul)</P><P>• 1μl T7-exten-F</P><P>• 1μl T7-exten-R</P><P>• 10μl Mix</P><P>• 1μl Bacteria Clone</P><P>PCR cycle including:</P><OL><LI>Preheat: 98 degree celcius, 7min</LI><LI>35 cycles containing: degradation(98 degree celcius, 30s), annealing(55 degree celcius, 30s) and elongation(72 degree celcius 1 min)</LI><LI>Elongation for whole at 72 degree celcius 10min.</LI><LI>4 celcius degree for preserveation. </LI></OL><H5>Gel Electrophoresis CharacterizationT7+SCVE</H5><P>T7-SCVE(PCR detection)</P><P>Then take one of the positive bacteria into 50ul centrifuge tube, incubate at 37℃.</P><H3 id="construction-of-t7-oprf">Construction of T7-OprF</H3><H2 id="pcr-of-oprf">PCR of OprF</H2><P>Material(volume 50ul)</P><P>OprF-F:1ul</P><P>OprF-R:1ul</P><P>Primestar:25ul</P><P>ddH2O:21ul</P><P>top10:2ul</P><P>Divide the 50ul into 2 parts. Both are</P><P>25ul</P><P>PCR cycle including:</P><OL><LI>Preheat: 98 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)</LI><LI>Elongation for whole at 72 degree celcius 10min</LI><LI>4 celcius degree for preserveation.
                      the protocol of pcr of T7 is the same as the above</LI></OL><H5>Gel Electrophoresis Characterization</H5><P>OprF(PCR)</P><H5>Gel Extraction</H5><P>Material including: OprF(PCR)</P><P>Preparation:</P><P>•Check out whether ethyl alcohol is added into Wash Solution.</P><P>•Check out whether Buffer B2 has sediment.</P><P>•Adjust the thermostat water bath at 50 degree centigrade.</P><P>Procedure:</P><OL><LI>Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.</LI><LI>Measure out the weight of tubes of gel.</LI><LI>Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.</LI><LI>Add isopropanol one third volume of Buffer B2.</LI><LI>After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s, then add the solutions into the absorption columns again and centrifuge columns again.</LI><LI>Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.</LI><LI>Do the step 10 again.</LI><LI>Centrifuge the empty columns at 9000xg about 1min.</LI><LI>Open the columns and air them about 10 mins.</LI><LI>Put the columns into clean 1.5 mL EP tubes and add 20uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.</LI></OL><H5>Double digest of T7 and OprF</H5><P>Detect the concentration of T7:23ng/ul,OprF:40ng/ul</P><P>Volume:20ul</P><P>Material:</P><P>T7:15ul</P><P>EcoRI:1ul</P><P>NdeI:1ul</P><P>bufferO:2ul</P><P>ddH2O:1ul</P><P>37℃digest 2 hours.</P><P>Material:</P><P>OprF:15ul</P><P>PstI:1ul</P><P>NdeI:1ul</P><P>bufferO:2ul</P><P>ddH2O:1ul</P><P>37℃digest 2 hours.</P><P>Detect the concentration of PSB1C3:160.2ng/ul</P><P>Volume:20ul</P><H5>Material</H5><P>PSB1C3:5ul</P><P>EcoR1:1ul</P><P>Pst1:2ul</P><P>Tango:2ul</P><P>ddH2O:10ul</P><P>37℃digest 2 hours.</P><H5>Gel Electrophoresis Characterization</H5><P>OprF(ep)</P><P>T7(ep)</P><H5>Gel Extraction</H5><P>Material including: OprF(ep)</P><P>Preparation:</P><P>•Check out whether ethyl alcohol is added into Wash Solution.</P><P>•Check out whether Buffer B2 has sediment.</P><P>•Adjust the thermostat water bath at 50 degree centigrade.</P><P>Procedure:</P><OL><LI>Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.</LI><LI>Measure out the weight of tubes of gel.</LI><LI>Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.</LI><LI>Add isopropanol one third volume of Buffer B2.</LI><LI>After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s, then add the solutions into the absorption columns again and centrifuge columns again.</LI><LI>Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.</LI><LI>Do the step 10 again.</LI><LI>Centrifuge the empty columns at 9000xg about 1min.</LI><LI>Open the columns and air them about 10 mins.</LI><LI>Put the columns into clean 1.5 mL EP tubes and add 20uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.</LI></OL><H5>Ligation of T7,OprF and pSB1C3</H5><P>In order to ligate the T7,OprF and PSB1C3, we need to make the equal mole proportion.</P><P>The mole of DNA molecule is calculated in the following ways:</P><P>C1V1/n1=C2V2/n2=……=0.1~0.3</P><P>Among which, c is the concentration of DNA, V is the volumn required and n is the length of DNA. Because of the same molar mass of As, Gs, Cs and Ts, computation of same molar mass is omitted.</P><P>Material:</P><P>T7:1ul</P><P>OprF:2.5ul</P><P>PSB1C3:6ul</P><P>Solution I:10ul</P><P>16℃ligate over night, prepare for the Transformation of the next day.</P><H5>Transformation of T7+OprF Plasmid</H5><P>Procedure:</P><OL><LI>Add 4ul clone product to competent cells, 1mix well, incubate on ice for 30min.</LI><LI>Heat in 42℃ bath for 90s. Then put on ice for 5min.</LI><LI>Add 900ul LB culture, incubate at 37℃ for 60min. Centrifuge the culture at 4000×1min, give away 850ul culture. </LI><LI>Spread on plate with chloramphenicol, incubate overnight at 37℃.</LI></OL><H5>Colonial PCR of T7+OprF</H5><P>Take one bacterial colony into 10ul H2O for the PCR.</P><P>Protocol(volume 13ul)</P><UL><LI>1μl T7-exten-F</LI><LI>1μl OprF-R</LI><LI>10μl Mix</LI><LI>1μl Bacteria Clone</LI></UL><P>PCR cycle including:</P><OL><LI>Preheat: 98 degree celcius, 7min</LI><LI>35 cycles containing: degradation(98 degree celcius, 30s), annealing(55 degree celcius, 30s) and elongation(72 degree celcius 1.5 min)</LI><LI>Elongation for whole at 72 degree celcius 10min.</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>Gel Electrophoresis Characterization</H5><P>T7-OprF</P><P>Then take one of the positive bacteria into 50ul centrifuge tube, incubate at 37℃.</P><H3 id="construction-of-lac-chez">Construction of lac+cheZ</H3><H5>PCR of lac and cheZ</H5><P>Material(volume 50ul)</P><P>EcoRI-lac-F:1ul</P><P>NdeI-lac-R:1ul</P><P>Primestar:25ul</P><P>ddH2O:22ul</P><P>template:1ul</P><P>Divide the 50ul into 2 parts. Both are</P><P>25ul</P><P>Material(volume 50ul)</P><P>NdeI-cheZ-F:1ul</P><P>PstI-his-cheZ-R:1ul</P><P>Primestar:25ul</P><P>ddH2O:22ul</P><P>template:1ul</P><P>Divide the 50ul into 2 parts. Both are</P><P>25ul</P><P>PCR cycle including:</P><OL><LI>Preheat: 98 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)</LI><LI>Elongation for whole at 72 degree celcius 10min</LI><LI>4 celcius degree for preserveation.
                      the protocol of pcr of T7 is the same as the above</LI></OL><H5>Gel Electrophoresis Characterization</H5><P>CheZ(PCR)</P><P>R0010(PCR)</P><H5>Gel Extraction</H5><P>Material including: lac and cheZ(PCR)</P><P>Preparation:</P><P>•Check out whether ethyl alcohol is added into Wash Solution.</P><P>•Check out whether Buffer B2 has sediment.</P><P>•Adjust the thermostat water bath at 50 degree centigrade.</P><P>Procedure:</P><OL><LI>Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.</LI><LI>Measure out the weight of tubes of gel.</LI><LI>Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.</LI><LI>Add isopropanol one third volume of Buffer B2.</LI><LI>After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s, then add the solutions into the absorption columns again and centrifuge columns again.</LI><LI>Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.</LI><LI>Do the step 10 again.</LI><LI>Centrifuge the empty columns at 9000xg about 1min.</LI><LI>Open the columns and air them about 10 mins.</LI><LI>Put the columns into clean 1.5 mL EP tubes and add 20uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.</LI></OL><H5>Double digest of lac and cheZ</H5><P>Detect the concentration of lac:45ng/ul cheZ:38ng/ul</P><P>Volume:20ul</P><P>Material:</P><P>R0010:15ul</P><P>EcoRI:1ul</P><P>NdeI:1ul</P><P>bufferO:2ul</P><P>ddH2O:1ul</P><P>37℃digest 2 hours.</P><P>Material:</P><P>cheZ:15ul</P><P>PstI:1ul</P><P>NdeI:1ul</P><P>bufferO:2ul</P><P>ddH2O:1ul</P><P>37℃digest 2 hours.</P><H5>Gel Electrophoresis Characterization</H5><P>R0010(np)</P><P>CheZ(xn)</P><H5>Gel Extraction</H5><P>Material including: lac and cheZ(ep)</P><P>Preparation:</P><P>•Check out whether ethyl alcohol is added into Wash Solution.</P><P>•Check out whether Buffer B2 has sediment.</P><P>•Adjust the thermostat water bath at 50 degree centigrade.</P><P>Procedure:</P><OL><LI>Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.</LI><LI>Measure out the weight of tubes of gel.</LI><LI>Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.</LI><LI>Add isopropanol one third volume of Buffer B2.</LI><LI>After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s, then add the solutions into the absorption columns again and centrifuge columns again.</LI><LI>Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.</LI><LI>Do the step 10 again.</LI><LI>Centrifuge the empty columns at 9000xg about 1min.</LI><LI>Open the columns and air them about 10 mins.</LI><LI>Put the columns into clean 1.5 mL EP tubes and add 20uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.</LI></OL><H5>Ligation of lac ,cheZ and pSB1C3</H5><P>In order to ligate the lac,cheZ and PSB1C3, we need to make the equal mole proportion.</P><P>The mole of DNA molecule is calculated in the following ways:</P><P>C1V1/n1=C2V2/n2=……=0.1~0.3</P><P>Among which, c is the concentration of DNA, V is the volumn required and n is the length of DNA. Because of the same molar mass of As, Gs, Cs and Ts, computation of same molar mass is omitted.</P><P>Material:</P><P>lac:0.5ul</P><P>cheZ:3.0ul</P><P>PSB1C3:6ul</P><P>Solution I:10ul</P><P>16℃ligate over night, prepare for the Transformation of the next day.</P><H5>Transformation of lac+cheZ Plasmid</H5><P>Procedure:</P><OL><LI>Add 4ul clone product to competent cells, 1mix well, incubate on ice for 30min.</LI><LI>Heat in 42℃ bath for 90s. Then put on ice for 5min.</LI><LI>Add 900ul LB culture, incubate at 37℃ for 60min. Centrifuge the culture at 4000×1min, give away 850ul culture. </LI><LI>Spread on plate with chloramphenicol, incubate overnight at 37℃.</LI></OL><H5>Colonial PCR of lac+cheZ</H5><P>Take one bacterial colony into 10ul H2O for the PCR.</P><P>Protocol(volume 13ul)</P><UL><LI>1μl EcoRI-lac-F</LI><LI>1μl NdeI-lac-R</LI><LI>10μl Mix</LI><LI>1μl Bacteria Clone</LI></UL><P>PCR cycle including:</P><OL><LI>Preheat: 98 degree celcius, 7min</LI><LI>35 cycles containing: degradation(98 degree celcius, 30s), annealing(55 degree celcius, 30s) and elongation(72 degree celcius 1.5 min)</LI><LI>Elongation for whole at 72 degree celcius 10min.</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>Gel Electrophoresis Characterization</H5><P>lac+cheZ</P><P>Then take one of the positive bacteria into 50ul centrifuge tube, incubate at 37℃.</P><H3 id="construction-of-micf-soxs-gfp">Construction of micF(SoxS)-GFP</H3><H5>PCR of micF(SoxS) and GFP</H5><P>Material(volume 50ul)</P><P>micF(SoxS)-F-BB:1ul</P><P>micF(SoxS)-E0040-R:1ul</P><P>Primestar:25ul</P><P>ddH2O:22ul</P><P>template:1ul</P><P>Divide the 50ul into 2 parts. Both are</P><P>25ul</P><P>Material(volume 50ul)</P><P>E0040-F:1ul</P><P>E0040-R:1ul</P><P>Primestar:25ul</P><P>ddH2O:22ul</P><P>template:1ul</P><P>Divide the 50ul into 2 parts. Both are</P><P>25ul</P><P>PCR cycle including:</P><OL><LI>Preheat: 98 degree celcius, 5min</LI><LI>35 cycles containing: degradation(94 degree celcius, 30s; annealing(55 degree celcius, 30s and elongation(72 degree celcius 1min)</LI><LI>Elongation for whole at 72 degree celcius 10min</LI><LI>4 celcius degree for preserveation.</LI></OL><H5>Gel Electrophoresis Characterization</H5><P>micF and SoxS(PCR)</P><P>E0040(PCR)</P><H5>Gel Extraction</H5><P>Material including: micF(SoxS) and GFP(PCR)</P><P>Preparation:</P><P>•Check out whether ethyl alcohol is added into Wash Solution.</P><P>•Check out whether Buffer B2 has sediment.</P><P>•Adjust the thermostat water bath at 50 degree centigrade.</P><P>Procedure:</P><OL><LI>Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.</LI><LI>Measure out the weight of tubes of gel.</LI><LI>Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.</LI><LI>Add isopropanol one third volume of Buffer B2.</LI><LI>After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s, then add the solutions into the absorption columns again and centrifuge columns again.</LI><LI>Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.</LI><LI>Do the step 10 again.</LI><LI>Centrifuge the empty columns at 9000xg about 1min.</LI><LI>Open the columns and air them about 10 mins.</LI><LI>Put the columns into clean 1.5 mL EP tubes and add 20uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.</LI></OL><H5>Double digest of micF(SoxS) and GFP</H5><P>Detect the concentration of micF:100ng/ul</P><P>SoxS:80ng/ul GFP:65ng/ul</P><P>Volume:20ul</P><P>Material:</P><P>micF:15ul</P><P>XbaI:1ul</P><P>HindIII:1ul</P><P>Tango buffer:2ul</P><P>ddH2O:1ul</P><P>37℃digest 2 hours.</P><P>Material:</P><P>GFP:15ul</P><P>PstI:1ul</P><P>HindIII:1ul</P><P>bufferR:2ul</P><P>ddH2O:1ul</P><P>37℃digest 2 hours.</P><H5>Gel Electrophoresis Characterization</H5><P>micF and SoxS(xH)</P><P>GFP(PH)</P><H5>Gel Extraction</H5><P>Material including: micF(SoxS) and GFP(ep)</P><P>Preparation:</P><P>•Check out whether ethyl alcohol is added into Wash Solution.</P><P>•Check out whether Buffer B2 has sediment.</P><P>•Adjust the thermostat water bath at 50 degree centigrade.</P><P>Procedure:</P><OL><LI>Cut up the effective bands under UV light and put the fragments into EP tubes with series numbers.</LI><LI>Measure out the weight of tubes of gel.</LI><LI>Add Buffer B2 3 to 6 times the volume of gel and then incubate the EP tubes in thermostat water bath about 50 degree centigrade about 10 mins.</LI><LI>Add isopropanol one third volume of Buffer B2.</LI><LI>After gel is all dissolved, pour the solutions into absorption columns and centrifuge columns at 8000xg about 30s, then add the solutions into the absorption columns again and centrifuge columns again.</LI><LI>Add 500 ul wash solution in a centrifuge at 9000Xg about 30s and pour out the solution.</LI><LI>Do the step 10 again.</LI><LI>Centrifuge the empty columns at 9000xg about 1min.</LI><LI>Open the columns and air them about 10 mins.</LI><LI>Put the columns into clean 1.5 mL EP tubes and add 20uL Elution Buffer(2.5mM Tris-HCL,PH 8.5)(also can be substituted by TE or double distilled water). Stand about 1 min and then centrifuge at 9000xg about 1 min.</LI></OL><H5>Ligation of micF(SoxS), GFP and pSB1C3</H5><P>In order to ligate the micF(SoxS), GFP and PSB1C3, we need to make the equal mole proportion.</P><P>The mole of DNA molecule is calculated in the following ways:</P><P>C1V1/n1=C2V2/n2=……=0.1~0.3</P><P>Among which, c is the concentration of DNA, V is the volumn required and n is the length of DNA. Because of the same molar mass of As, Gs, Cs and Ts, computation of same molar mass is omitted.</P><P>Material:</P><P>micF(SoxS):0.8ul</P><P>GFP:2.0ul</P><P>PSB1C3:6ul</P><P>Solution I:10ul</P><P>16℃ligate over night, prepare for the Transformation of the next day.</P><H5>Transformation of micF(SoxS)+GFP Plasmid</H5><P>Procedure:</P><OL><LI>Add 4ul clone product to competent cells, 1mix well, incubate on ice for 30min.</LI><LI>Heat in 42℃ bath for 90s. Then put on ice for 5min.</LI><LI>Add 900ul LB culture, incubate at 37℃ for 60min. Centrifuge the culture at 4000×1min, give away 850ul culture. </LI><LI>Spread on plate with chloramphenicol, incubate overnight at 37℃.</LI></OL><P>Then take two bacteria colony into 50ul centrifuge tube, incubate at 37℃.</P><P>Then take one of the positive bacteria into 50ul centrifuge tube, incubate at 37℃.</P><LI><TIME class="cbp_tmtime" datetime="2015-09-10 00:00"><SPAN>2015</SPAN><SPAN>10/9</SPAN></TIME><DIV class="cbp_tmlabel"><H4>Pre-experiment</H4><P>Use 1ug/ml Chloromycetin activated the bacteria to test the film.</P><H4>Film test</H4><H5>Film</H5><P>Film type:low-pressure polyethylene.</P><P>Use the aerosol paint to process one of the surface of the film.</P><P>Coating the film with 400ul 20ug/ml PLL(Poly-L-Lysine) at the temperature of 4℃ for over 4 hours.</P><H5>Bacteria solution</H5><P>Bacteria type: BL21+OprF</P><P>Mix bacteria with PBS and make the OD~0.3.</P><H5>Sample solution</H5><P>1ug/ml Chloromycetin PBS solution.</P><H5>Inoculation</H5><P>Drop 200ul bacteria solution on the processed film for 100s</P><P>After 100s, wash the film with <STRONG>wash solution</STRONG> twice Immediately.</P><H5>Record images</H5><P>Put the film into the sample box filled with samplw solution.</P><P>Put the sample box in SPRING and push or pull the sample box at the recommended position, and rotate the sample box to get clear interference fringes.</P><P>From 0s to 240s, take 2~3 pictures in a row with the time interval of 20s.</P><H5>Results</H5><P>Fail to get the ideal interference fringes.</P><P>Cause reflectance is not good enough and have a small amount of diffuse reflection.</P></DIV></LI><LI><TIME class="cbp_tmtime" datetime="2015-09-11 00:00"><SPAN>2015</SPAN><SPAN>11/9</SPAN></TIME><DIV class="cbp_tmlabel"><H4>Pre-experiment</H4><P>Use 1ug/ml Chloromycetin activated the bacteria to test the film.</P><H5>Film</H5><P>Film type: Rubbers（Condom）</P><P>1,Use the aerosol paint to process one of the</P><P>surface of the film.</P><P>Fail.Because the film shrink too much.</P><P>2,Try silver mirror reaction.</P><P>Fail.Because the ammonia erosion is too severe.</P><H5>Film</H5><P>Film type:glass(cover clip——thickness~0.16mm width~2.5cm longth~3.7cm)</P><P>Use the aerosol paint to process one of the surface of the film.</P><P>Coating the film with 400ul 20ug/ml PLL(Poly-L-Lysine) at the temperature of 4℃ for over 4 hours.</P><H5>Bacteria solution</H5><P>Bacteria type: BL21+OprF</P><P>Mix bacteria with PBS and make the OD~0.3.</P><H5>Sample solution</H5><P>1ug/ml Chloromycetin PBS solution.</P><H5>Inoculation</H5><P>Drop 200ul bacteria solution on the processed film for 100s</P><P>After 100s, wash the film with <STRONG>wash solution</STRONG> twice Immediately.</P><H5>Record images</H5><P>Put the film into the sample box filled with samplw solution.</P><P>Put the sample box in SPRING and push or pull the sample box at the recommended position, and rotate the sample box to get clear interference fringes.</P><P>From 0s to 240s, take 2~3 pictures in a row with the time interval of 20s.</P><H5>Results</H5><P>Succeed in getting the excellent interference fringes.</P><P>Record data and process the image.</P></DIV></LI><LI><TIME class="cbp_tmtime" datetime="2015-09-12 00:00"><SPAN>2015</SPAN><SPAN>12/9</SPAN></TIME><DIV class="cbp_tmlabel"><H4>Building Calibration</H4><H4>Film test</H4><H5>Sample solution</H5><P>（1）5ug/ml Chloromycetin PBS solution.</P><P>（2）0.5ug/ml Chloromycetin PBS solution.</P><H5>Bacteria solution</H5><P>Bacteria type: BL21+OprF</P><P>Mix bacteria with PBS and make the OD~0.3.</P><H5>Inoculation</H5><P>Drop 200ul bacteria solution on the processed film for 100s</P><P>After 100s, wash the film with <STRONG>wash solution</STRONG> twice Immediately.</P><H5>Record images</H5><P>Put the film into the sample box filled with samplw solution.</P><P>Put the sample box in SPRING and push or pull the sample box at the recommended position, and rotate the sample box to get clear interference fringes.</P><P>From 0s to 240s, take 2~3 pictures in a row with the time interval of 20s.</P><H5>Results</H5><P>Succeed in getting the excellent interference fringes.</P><P>Record data and process the image.</P><P>Use the Matlab to process the image.</P><P>Get the fitting according to the modeling so get the calibration.</P></DIV></LI><LI><TIME class="cbp_tmtime" datetime="2015-09-13 00:00"><SPAN>2015</SPAN><SPAN>13/9</SPAN></TIME><DIV class="cbp_tmlabel"><H4>Verifying Calibration</H4><H4>Film test</H4><H5>Sample solution</H5><P>（1）1.8ug/ml Chloromycetin PBS solution.</P><P>（2）3.2ug/ml Chloromycetin PBS solution.</P><H5>Bacteria solution</H5><P>Bacteria type: BL21+OprF</P><P>Mix bacteria with PBS and make the OD~0.3.</P><H5>Inoculation</H5><P>Drop 200ul bacteria solution on the processed film for 100s</P><P>After 100s, wash the film with <STRONG>wash solution</STRONG> twice Immediately.</P><H5>Record images</H5><P>Put the film into the sample box filled with samplw solution.</P><P>Put the sample box in SPRING and push or pull the sample box at the recommended position, and rotate the sample box to get clear interference fringes.</P><P>From 0s to 240s, take 2~3 pictures in a row with the time interval of 20s.</P><H5>result</H5><P>Process the gotten images and get the △N.</P><P>Calculate the △N and get the concentration is 1.80ug/ml and 3.09ug/ml.</P></DIV></LI></UL></DIV></DIV></DIV></DIV><DIV class="fixed-action-btn" style="bottom: 45px; right: 24px;"><A class="btn-floating btn-large amber darken-1 waves-effect waves-light" onclick="$('html, body').animate({scrollTop:0}, 'slow');"><I class="material-icons">publish</I></A><UL><LI><A class="btn-floating green waves-effect waves-light" style="transform: scaleY(0.4) scaleX(0.4) translateY(40px); opacity: 0;" onclick="location.reload();"><I class="material-icons">refresh</I></A></LI></UL></DIV><FOOTER class="page-footer grey darken-1" style="padding-top: 0px;"><DIV class="container"><DIV class="row" style="margin-bottom:0px;"><DIV class="col s8"><H5 class="white-text">Contact Us</H5><P class="grey-text text-lighten-4">University of Science and Technology of China, No.96, JinZhai Road Baohe District,Hefei,Anhui, 230026,P.R.China.</P></DIV><DIV class="col s4"><H5 class="white-text">Links</H5></DIV></DIV></DIV></FOOTER></BODY></HTML>