"
Page
Discussion
View source
History
teams
Log in
 
Team:UC Davis/Project/Directed Evolution
From 2012.igem.org
Team:UC Davis - 2012.igem.org
teams
Page
Discussion
Edit
History
Move
Watch
Log in
iGEM
Main iGEM
Criteria
Human Practices
Attributions
Data
Cutinase Activity
Ethylene Glycol
Modeling
Parts
Notebook
Notebook
Protocols
Gallery
Safety
Project
Project Overview
Module Engineering
Protein Engineering
Chassis Engineering
Background
Directed Evolution
Rational Engineering
Team
Home
UCDavis iGEM Tweets
Our Sponsors
Directed Evolution
Evolution occurs naturally by selection pressure, but in an overall slow pace. To speed up the process for certain desired traits, it is possible to re-passage cells to grow on a certain type of media, or expose certain mutagens to increase the variation in the cells to select for surviving mutants. Our team has carried out both of these selection pressures in hopes to isolate an ethylene glycol utilizing bacterium.
History
Mutants of E. coli able to grow on propylene glycol were selected for ethylene glycol enzymatic breakdown after ethyl methanesulfonate (EMS) mutagenesis by the University of Barcelona [1]. The culture conditions strongly selected for ethylene glycol utilizing mutants, by having ethylene glycol as the sole carbon source in liquid culture. Through directed evolution, colonies were continuously streaked on ethylene glycol plates for three more generations, in order to isolate ethylene glycol degrading mutants. Through spectrophotometric assays, increased activities of glycolaldehyde reductase and glycolaldehyde dehydrogenase were observed in the ethylene glycol isolates.
Genome Sequencing
We are sequencing the genome of Strain E-15 EG3 so that we can see what changes are present between the MG1655 and Strain E-15 EG3. Once sequenced, we will be able to compare the sequences between each strain which would show single nucleotide polymorphisms (SNPs), as well as any deletions or insertions. From this, we will be able to apply the knowledge to a hybrid approach with the rational engineering.  Using the transposase, Tn5, we have constructed an Illumina compatible library that we plan to submit to our Sequencing Core Facilities. Unfortunately, the sequencing run on the MiSeq takes around one week and we were not able to get sequence data before the competition.
Tecan Experiments
Re-streaks
After receiving Strain E-15 EG3, the stab culture was re-streaked on ethylene glycol (EG) agar for three generations to ensure that the strain retained the two functional enzymes key for ethylene glycol utilization after thirty years of storage. Colonies became defined on the EG plates every three to four days, and were immediately re-streaked onto a new EG plate. With this selection pressure to choose the fastest developing colony, our hope is to isolate an evolved version of Strain E-15 EG3, that is capable of replicating on EG at a faster pace.
We are comparing the most recent re-streak (re-streak #12) with the earliest re-streak (re-streak #3) to test if constantly selecting Strain E-15 EG3 improves the overall growth rate of the strain.
Liquid Cultures
We made liquid cultures of the re-streaked Strain E-15 EG3 and re-passaged the strain in new ethylene glycol media for twenty-five generation cycles each time the tubes reached an OD 600 of 0.2. By repeating this process, we were constantly selecting the fastest growing cells on ethylene glycol. We hoped to create a population whose growth rate was even greater than the original Strain E-15 EG3. (Click here for data relating to the repassaging.) Since the liquid culture contains a mixed population of EG utilizing cells, it is important to first plate the cells and select for the fastest developing colonies. After observing the presence of colonies, the first fifty-five colonies were selected, tested for rate of growth in EG media, and compared to the standard, Strain E-15 EG3 from re-streak #3.
EMS mutagenesis
Ethyl methanesulfonate (EMS) is a liquid mutagen that favors a G/C to A/T transition. We are employing the EMS to both the MG1655 and E-15 EG3 liquid cultures in different concentrations (28 μL, 56 μL, 84 μL to 4 mL of cells) for different amounts of time (0 minute, 15 minutes, 30 minutes, and 45 minutes) [3]. However, we are not focusing on a specific gene with a high G/C content, so the EMS just serves the purpose of adding variation to the population. We wanted to compare the effects of EMS and whether mutagenesis introduced any variation in the utilization of ethylene glycol at different rates. To set up the comparison across plates for relative growth rates, a set of Strain E-15 EG3 standards were included in each plate. We expect that the EMS caused some variation in each of the populations. However, they are probably minute variations, and are likely to be similar in efficiency.
The cultures that were exposed to EMS for 45 minutes still showed colonies, meaning that the EMS was not strong enough to kill off the cells. Therefore, these colonies should theoretically show the most variation. The first fifty-five colonies that developed in both MG1655 and E-15 EG13 were compared at the 0 minute time point and the 45 minute time point using the Tecan (here's our data ).
Given more time, these colonies would be sequenced and compared to the E-15 EG13 to see how any base changes introduced by EMS will affect the efficiency of the enzymes.
References
1. Boronat, Albert, Caballero, Estrella, and Juan Aguilar. “Experimental Evolution of a Metabolic Pathway for Ethylene Glycol Utilization by Escherichia coli.” Journal of Bacteriology, Vol. 153 No. 1, pp. 134-139, January 1983.
2. Andrianantoandro, Ernesto, Subhayu Basu, David K. Karig, and Ron Weiss. "Synthetic biology: new engineering rules for an emerging discipline." Nature - Molecular Systems Biology. (2006): n. page. Web. 29 Aug. 2012.
.
3. Cupples, Claire, and Jeffrey Miller. "A set of lacZ mutations in Escherichia coli that allow rapid detection of each of the six base substitutions." Proceedings of the National Academy of Sciences. 86. (1989): 5345-5349. Print.
Home
Welcome
Tweets
Sponsors
Criteria
Team
Who we are
Students
Advisors
Project
Project Overview
Module Engineering
Protein Engineering
Chassis Engineering
- Directed Evolution
- Rational Engineering
Critera
Safety
Safety
Notebook
Notebook
Protocols
Gallery
Critera
Data
Cutinase Activity
Ethylene Glycol
Modeling
Parts
Attribution
Attribution
iGEM
Main iGEM
Criteria
Human Practices
Retrieved from " https://2012.igem.org/Team:UC_Davis "
Retrieved from " http://2012.igem.org/Team:UC_Davis/Project/Directed_Evolution "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
