"
Page
Discussion
View source
History
teams
Log in
 
Team:UC Davis/Safety
From 2012.igem.org
Team:UC Davis - 2012.igem.org
teams
Page
Discussion
Edit
History
Move
Watch
Log in
iGEM
Main iGEM
Criteria
Human Practices
Attributions
Data
Cutinase Activity
Ethylene Glycol
Modeling
Parts
Notebook
Notebook
Protocols
Gallery
Safety
Project
Project Overview
Module Engineering
Protein Engineering
Chassis Engineering
Background
Directed Evolution
Rational Engineering
Team
Home
UCDavis iGEM Tweets
Our Sponsors
Background
Originally, we wanted to optimize our engineered bacteria to live in landfills. Our goal was to break down PET at the source of the problem itself. As a team, we took a trip to Yolo County Landfill and learned that a diverse group of bacteria already lives amongst the debris. However, we also learned that our engineered bacteria would have next to no chance of survival amid the competitive microbes already present. Since our E. coli could not survive in the landfill, we shifted our focus to thinking about how the bacteria would work on an industrial level, with the controlled atmosphere of bioreactors to help the E. coli survive. If our project were implemented in industry, this would have the added bonus for safety of containment, which would prevent any concerns associated with releasing the bacteria into the environment.
Safety
Our team made sure to perform a thorough analysis of safety for all parts of our project, since safety for researchers, the public, and the environment are all important. We work within a Level 2 Biosafety Lab, as defined by the Laboratory Biosafety Manual (1). This facility is well equipped to work with all the strains of E. coli we used. We used MG1655, DH5α, and K12, all of which are Risk Level 1, meaning these strains pose no pathogenic risk to humans, and were completely contained within the lab.
We always made use of protective gear, such as wearing gloves at all times, face masks when using UV lamps, and lab coats when appropriate. All of our waste was autoclaved on site as well, and all tools and glassware thoroughly cleaned after each use.
We worked with ethylene glycol during our experiments, which is toxic to the kidney, liver and central nervous system, as well as a skin and eye irritant. Obviously, we wanted to be very safe when using this chemical, so we made sure to consult with our advisers on how best to handle it. When using ethylene glycol, we always worked under a fume hood and wore lab coats.  We had a separate waste bottle for all of the byproducts with ethylene glycol, so that the waste could be treated and properly disposed of.
The University of California at Davis has a biosafety group, which heads the disposal and use of hazardous materials. They approve of our project and the procedures that we have followed. Further, they are excited for the potential that this project has, in terms of making the environment safer for inhabitants.
Our project, when properly functioning, should not have any harmful effect on the environment, researchers, or the public. We keep everything contained within the lab, and all of the plates and wastes are autoclaved to prevent all bacteria we use from spreading to the outside environment.
If we were to implement our bacteria in a landfill, it could not survive past a couple generations and therefore would not be very toxic. However, if we were to imagine a more robust version of our strain created in the future, we would implement a kill switch, which has already been created using synthetic biology, to make sure we could stop it (2). We would also create a system similar to Imperial College London’s Gene Guard, which would stop to plastic-degrading plasmid from getting transferred to other bacteria (3). This would contain the plasmid within only our bacteria, so it would not get out of hand.
Our project could potentially be dangerous if our strain received a pathogenic plasmid by lateral gene transfer from other organisms. However, if this were a concern, our strain would be just a susceptible to pathogenicity as any other. If someone with malicious intent were to use our construct, perhaps the most they could do is release it into the water way and have it degrade plastic pipes made of PET, or modify it to degrade other plastics in addition to PET and degrade something useful. Cutinase, the first enzyme in our degradation pathway, functions optimally at 50° C, and the water in our pipes would most likely not be at that temperature. Also, it is unlikely the strain would survive under conditions such as this temperature and being in an aquatic environment (4).
Questions
1. Would any of your project ideas raise safety issues in terms of: researcher safety, public safety, or environmental safety?
Our projects involves ethylene glycol, a moderately toxic substance, which is eventually oxidized to oxalic acid – a toxic substance that affects the central nervous system via the liver. Because ethylene glycol must be ingested to pose a problem, researchers take extra precaution to make sure there are no splashes of ethylene glycol in the laboratory and the wastes will be disposed of in the appropriate hazardous waste receptacles. Ethylene glycol can also be a mild irritant if it comes in contact with the skin or if it is inhaled, so researchers wear eye protection as well as gloves and lab coats, and always work with ethylene glycol in the confine of a fume hood.
In the environment, ethylene glycol is degraded by hydroxyl radicals and in sewage sludge, it is readily biodegradable. However, ethylene glycol can potentially be toxic within waterways so to ensure environmental safety the team has taken extra precautions to dispose of ethylene glycol safely. Additionally, we have considered how to deal with ethylene glycol by designing a circuit in which we produce enzymes that will degrade ethylene glycol into glycoaldehyde and then glycolate. The glycolate has the potential to be turned in to oxaloacetate, a metabolic intermediate.
2. Do any of the new BioBrick parts (or devices) that you made this year raise any safety issues? If yes, did you document these issues in the Registry? How did you manage to handle the safety issue? How could other teams learn from your experience?
Our BioBrick parts this year do not raise any serious safety issues. However one part, the LC-Cutinase gene BBa_K936000 , encodes an enzyme that breaks down PET into ethylene glycol, a moderately toxic substance. This has been noted on its registry page. The safety management procedures have been discussed in the previous question’s answer. The novel part that we are submitting to the Registry is the cutinase gene, in BioBrick format. This is a lipolytic enzyme that does not pose a threat to humans.
3. Is there a local biosafety group, committee, or review board at your institution? If yes, what does your local biosafety group think about your project? If no, which specific biosafety rules or guidelines do you have to consider in your country?
The University of California at Davis has a biosafety group, which heads the disposal and use of hazardous materials. They approve of our project and the procedures that we have followed. Further, they are excited for the potential that this project has, in terms of making the environment safer for inhabitants.
4. Do you have any other ideas how to deal with safety issues that could be useful for future iGEM competitions? How could parts, devices and systems be made even safer through biosafety engineering?
Safety issues could be headed by a designated safety monitor in each iGEM group. This set-up allows all of the teams to have a person who will always make sure that the safest procedures are being followed at all times. Also, the production of potentially hazardous materials should always be controlled by an inducible promoter, so that it may be stopped at any time. The different parts can be made safer by making the parts work only with BioBrick assembly units and nothing else that exists in nature.
References
1. Laboratory Biosafety Manual. The World Health Organization. 2004. 3rd edition.
2. Callura, Jared, Dwyer, Daniel, Isaacs, Farren, Cantor, Charles, Collins, James. “Tracking, tuning, and terminating microbial physiology using synthetic riboregulators”. PNAS vol. 107 no. 36, pp. 15898-15903. Web. 13 July 2010.
3. “Gene Overview”, Imperial College London iGEM Wiki 2011.
4. S. Sulaiman, S. Yamato, E. Kanaya, J. Kim, Y. Koga, K. Takano, S. Kanaya. "Isolation of a Novel Cutinase Homolog with Polyethylene Terephthalate-Degrading Activity from Leaf-Branch Compost by Using a Metagenomic Approach." Applied and Environment Microbiology, vol. 78 no. 5, pp. 1556-1562, March 2012.
Home
Welcome
Tweets
Sponsors
Criteria
Team
Who we are
Students
Advisors
Project
Project Overview
Module Engineering
Protein Engineering
Chassis Engineering
- Directed Evolution
- Rational Engineering
Critera
Safety
Safety
Notebook
Notebook
Protocols
Gallery
Critera
Data
Cutinase Activity
Ethylene Glycol
Modeling
Parts
Attribution
Attribution
iGEM
Main iGEM
Criteria
Human Practices
Retrieved from " https://2012.igem.org/Team:UC_Davis "
Retrieved from " http://2012.igem.org/Team:UC_Davis/Safety "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
