"
Page
Discussion
View source
History
teams
Log in
 
Team:UC Davis/Notebook/Protocols
From 2012.igem.org
Team:UC Davis - 2012.igem.org
teams
Page
Discussion
Edit
History
Move
Watch
Log in
iGEM
Main iGEM
Criteria
Human Practices
Attributions
Data
Cutinase Activity
Ethylene Glycol
Modeling
Parts
Notebook
Notebook
Protocols
Gallery
Safety
Project
Project Overview
Module Engineering
Protein Engineering
Chassis Engineering
Background
Directed Evolution
Rational Engineering
Team
Home
UCDavis iGEM Tweets
Our Sponsors
Protocols
Registry Part Distribution Rehydration
Add 20 µL sterile H2O to desired well in distribution plate.
Incubate at room temperature for ~10 minutes.
Transfer resuspension to microcentrifuge tube.
Transformations
Materials
Competent Cells
DNA template
800 µL LB
LB+Antibiotic Plates
Procedure
Thaw competent cells on ice.
Transfer 50 µL of competent cells to chilled falcon tubes.
Add 1 µL of template to cells (2.5 µL if dilute).
Incubate on ice for 30 minutes.
Heat schock in 42 °C water bath for 90 seconds.
Immediately place back onto ice for 2 minutes.
Add 800 µL of LB to each tube.
Incubate at 37 °C for 1 hour.
Place 200 µL of the transformed cells on plates containing LB and the appropriate antibiotic.
Incubate overnight at 37 °C.
Restriction Enzyme Double Digest
Materials
22 µL dH20
1 µL BSA
5 µLBuffer
20 µL Template
1 µL Enzyme 1
1 µL Enzyme 2
Procedure
Mix reactants, adding enzyme last.
Place at 37 °C for 3-5 hours.
If not purifying or running on a gel immediately, increase to 80 °C for 20 minutes to kill enzymes (some enzymes need only a 65 °C heatkill, check enzyme).
Run on a gel and extract product.
PCR (sequence dependent)
Materials
10 uL Q Solution
5 µL 10x Buffer
1.25 µL 10mM dNTPs
1 µL Template
1 µL Forward Primer
1 µL Reverse Primer
0.3 µL Taq
0.15 µL PFU
30 µL dH20
Procedure
Mix reactants into PCR tubes. A large mix can be made and then aliquoted into PCR tubes if desired.
Run in thermal cycler.
PCR SOEing (Polymerase Chain Reaction Splicing by Overlapping Extension)
Materials
10 µL Q Solution
5 µL 10x Buffer
1.25 µL 10mM dNTPs
2.5 µL Forward Primer
2.5 µL Reverse Primer
0.3 µL Taq
0.15 µL PFU
Template based on concentrations determined by SOEing calculator: “Link”
±18.30 µL dH20 based on concentrations of template (above) by SOEing calculator: “Link again”
Procedure
Mix reactants into PCR tubes.
Run in thermal cycler.
Continue PCR SOEing of parts until completed.
Ligation
Materials
Digested Vector
Digested Insert
Water
T4 DNA Ligase
T4 DNA Ligase Buffer
Procedure
Mix these materials in the amounts determined by the reaction volume calculator here .
Ethylene Glycol Media
Materials
34 mM NaH2PO4
64 mM K2HPO4
20 mM (NH4)2SO4
1 µM FeSO4
0.1 mM MgSO4
10 µM CaCl2
30 mM Ethylene Glycol
30 mM Glycolate
0.5% wt/vol Casein Acid Hydrolysate
1.5% wt/vol Agar (if making solid media)
Procedure
Be sure to wear a lab coat while making this media, since both glycolate and ethylene glycol are irritants.
Mix together NaH2PO4, K2HPO4, (NH4)2SO4, FeSO4, MgSO4, and CaCl2.
Titrate to pH 7 with HCl.
Add the glycolate and casein acid hydrolysate.
Mix in the ethylene glycol in a fume hood.
If making solid media, also mix in agar.
Autoclave on an appropriate cycle.
Ethylene Glycol Toxicity Assay
Materials
Luria Broth Media
Ethylene Glycol
Tecan M200 Pro
E. coli
Procedure
Be sure to wear a lab coat while making dilutions and aliquots, since ethylene glycol is an irritant.
Inoculate bacterial colony in 5 mL of luria broth (LB) media in a 15 mL falcon tube to create a starter culture.
Incubate at 37°C overnight on a shaker at 150 rpm.
From the incubated starter culture, take 20 µL and add it into 3 mL of LB media.
Incubate at 37°C for 2-3 hours on a shaker at 150 rpm.
Measure the optical density (OD) reading every thirty minutes after two hours to reach an ideal range of 0.5 to 0.7.
Use 15 mL conical tubes to make dilutions of ethylene glycol with LB media (see diagram below).
With a 96-well round-bottom plate, fill the outermost wells with LB media to prevent dehydration on the edges.
Add “blank” wells of just LB media to an entire column to serve as a control for the LB.
Include a column for 0 mM ethylene glycol and LB media as a baseline for activity without ethylene glycol.
Aliquot the remaining dilutions as the diagram below depicts.
Add the appropriate amount of cells to the wells for a concentration of 0.01 OD before any of the readings.
Follow the steps of your Tecan machine appropriate to the specific bacterial strain.
EMS (Ethyl methanesulfonate Mutagenesis)
Materials
Buffer A: Minimal medium + 0.2 M Tris (pH 7.5). per liter:
K2HPO4: 10.5g
KH2PO4: 4.5g
(NH4)2SO4: 1g
Sodium Citrate * 2H2O: 0.5g
1M Tris (pH 7.5): 200mL Tris
Obtained components from “Miller,  J. H. (1972) Experiments  in Molecular Genetics  (Cold Spring Harbor Lab., Cold Spring Harbor, NY)” pg 432
EMS (Sigma)
50mL conical Corning tubes
15mL Falcon tubes
LB Broth
Ethylene Glycol Agar Plates
Procedure
1. Prepare a 10mL liquid culture in LB in a 50mL conical tube and grow it overnight until it reaches OD 0.2.
Chill the cells on ice and spin down the 10mL aliquots at max speed in Eppendorf Centrifuge 5810 R for 10 minutes.
Wash twice with 10mL Buffer A.
Pipet up and down to mix, pellet cells, decant supernatant.
Re-suspend the pellet in 5mL of Buffer A and transfer the medium to a Falcon tube.
Add (35μL, 70μL, or 105μL) of EMS into each tube of re-suspended cells.
Close tubes and parafilm the lid 2X.
Vortex to mix and place in a secondary container (50mL conical tube).
Transfer to mixing platform and agitate at ~30 rpm at 37°C.
Withdraw samples at fixed time points.
At each time point, take 1mL aliquots of the sample and place in a 15mL Falcon tube. Place in a secondary container (50mL Conical tube) and spin at max speed in Eppendorf Centrifuge 5810 R for 5 minutes.
Discard supernatant in waste container.
Wash twice in 5mL of Buffer A and discard supernatant in waste container.
Re-suspend in 5mL of Buffer A and titer for viable cells.
Add 0.5mL of mutagenized culture into 10mL of LB broth in a 50mL conical tube.
Grow cultures overnight at 37°C and plate for mutants on EG agar plates.
Look for viable cells
Safety: Health Hazards
Acute toxicity (oral, dermal, inhalation), category 4
Skin irritation, category 2
Eye irritation, category 2
Skin sensitization, category 1
Specific Target Organ Toxicity – Single exposure, category 3
LD50 – 470mg/kg
Recommended Safety & Handling
Always work in fume hood and wear lab coat, goggles, and gloves.
Construction of a Transposase Illumina Sequencing Library
Procedure
Extract genomic DNA using any proprietary gDNA extraction kit. Once DNA is extracted, quantify your DNA and run it on an agarose gel to check DNA quality.
Prepare Illumina Adapters for Transposase Priming, Mix the Following:
10µM No_PCR_Adapter_1: 10µL
10µM No_PCR_Adapter_2: 10µL
10µM Mosaic Element: 20µL
5M NaCl: 1.2µL
Incubate at 98C for 5 mins and let cool to room temperature
Prime Transposase
Transposase: 10µL
Annealed adapter (above): 5µL
Water: 15µL
Incubate at room temperature for 20-30 mins.
Tagmentation Reaction
Primed Transposase: 10µL (depends on the concentration of available Tn5)
200ng of gDNA extration: XµL
5x Tn5 Buffer: 3µL
Water: up to 15µL
Run multiple reactions in parallel to achieve appropriate amount of input DNA for sequencing platform
Incubate at 55C for 10 mins, and immediately pool reactions and clean up using a PCR clean up kit, elute with 15µL water
Gap Filling
Run a typical PCR reaction, without oligos, at 72C for 5 mins
Size Selection
Run gap filling products on 2% agarose gel, size select for the appropriate size required for your sequencing platform
Quality check your library
Run bioanalyzer to check your library quality, and qPCR to quantify your library before submission.
SUBMIT!
Cutinase Expression and Western Blot
Procedure
Prepare a starter culture of MG1655 E. coli with pBad regulated and his-tagged LC-Cutinase along with a negative control.
Inoculate new 5 mL cultures from the starter cultures at an OD 600 of about 0.05.
Let the cultures grow until and OD 600 of 0.5, then inoculate two 100 mL cultures (one for the his-tagged protein and one for the negative control).
Once these cultures are at an OD of 0.8, separate into two 50 mL cultures. Induce one at 10 uM, leave the other uninduced.
Take 1 mL samples at different time points.
Centrifuge samples at 5000g for 5 minutes then take off and save media.
Wash cells with water.
Resuspend with 300 uL of B-PER Protein Extraction Reagent and follow the instructions provided with that kit. After this, there should be samples of culture media, and soluble and insoluble whole cell lysate.
Take 15 uL of each sample, including controls, and run on western blot.
pNPB Assay
Materials
pNPB buffer (should be made and used in fumehood due to acetonitriles toxicity)
10mM pNPB in acetonitrile
1:4:95 ratio of acetonitrile pNPB solution, 100% ethanol and 50mM Tris-HCL (pH 8)
LB with specific resistance
Cell Culture
96 well plate
Protocol
First assign each well of the plate except for outer wells which should be filled with LB to prevent evaporation of medium.
Fill wells with 95 uL of LB
Fill wells with 5 uL of cell culture
In fumehood, add 100 uL of bufer to each well
Run in Tecan taking both ODs and absorbance 405 readings
SDM (Site Directed Mutagenesis)
Materials
Pfu turbo
10X Pfu turbo buffer
dNTPs (10mM)
Forward and reverse primers (0.1ug/uL, see methods section for design tips)
dH20
Dpn1
Competent cells
Methods
Primer Design
Forward primer should be between 25 and 45 bases in length and contain the desired mutation in the center with correct sequences on both sides; the reverse primer is the reverse complement of this.
Primers should have a minimum GC content of 40% and terminate in one or more C's or G's.
Tm should be greater or equal to 78°C and can be calculated as follows:
Tm = 81.5 + 0.41(%GC) - 675/(length in bases) - %mismatch
Reaction
Template DNA: 1 µL
10X Buffer: 5 µL
Forward Primer (100 ng/µL): 1µL
Reverse Primer (100 ng/µL): 1µL
10mM dNTPs: 1µL
Pfu Turbo (Stratagene): 1µL
MilliQ H20: 40µL
PCR Program
95°C for 1 minute
95°C for 50 seconds, 60°C for 50 seconds, 68°C for 1 minute/kb of plasmid length -- repeat this step 17 times, or 18 cycles total
68°C for 7 minutes
4°C hold
Following PCR - save 4µL of PCR reaction.
To remaining 46µLs add 1µL of Dpn1 to PCR reaction (PCR cleanup is not necessary). Incubate at 37°C for 1-2 hours to digest parental DNA. Run 5µL of the digested reaction on a gel and compare to the undigested parental plasmid - there should be some difference in band pattern. Transform into competent cells (PCR cleanup is not necessary).
Protocol Adapted From Here
Home
Welcome
Tweets
Sponsors
Criteria
Team
Who we are
Students
Advisors
Project
Project Overview
Module Engineering
Protein Engineering
Chassis Engineering
- Directed Evolution
- Rational Engineering
Critera
Safety
Safety
Notebook
Notebook
Protocols
Gallery
Critera
Data
Cutinase Activity
Ethylene Glycol
Modeling
Parts
Attribution
Attribution
iGEM
Main iGEM
Criteria
Human Practices
Retrieved from " https://2012.igem.org/Team:UC_Davis "
Retrieved from " http://2012.igem.org/Team:UC_Davis/Notebook/Protocols "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
