<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UC_Davis_Project_Protein_Engineering"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UC_Davis/Project/Protein_Engineering">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UC_Davis/Project/Protein_Engineering&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UC_Davis/Project/Protein_Engineering&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UC_Davis/Project/Protein_Engineering&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UC_Davis/Project/Protein_Engineering" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UC Davis/Project/Protein Engineering</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>Team:UC Davis - 2012.igem.org</TITLE></P><DIV id="globalWrapper"><DIV id="main-content"><DIV id="topmenubar" class="topleft-menu"><UL><LI style="color:#014457; cursor:default"><A>teams</A></LI><LI class="selected"><A href="https://2012.igem.org/Team:UC_Davis/Project/Protein_Engineering">Page</A></LI><LI class="new"><A href="https://2012.igem.org/wiki/index.php?title=Talk:Team:UC_Davis/Project/Protein_Engineering&amp;action=edit&amp;redlink=1">Discussion</A></LI><LI><A href="https://2012.igem.org/wiki/index.php?title=Team:UC_Davis/Project/Protein_Engineering&amp;action=edit">Edit</A></LI><LI><A href="https://2012.igem.org/wiki/index.php?title=Team:UC_Davis/Project/Protein_Engineering&amp;action=history">History</A></LI><LI><A href="https://2012.igem.org/Special:MovePage/Team:UC_Davis/Project/Protein_Engineering">Move</A></LI><LI><A href="https://2012.igem.org/wiki/index.php?title=Team:UC_Davis/Project/Protein_Engineering&amp;action=watch">Watch</A></LI><LI><A href="https://igem.org/Login">Log in</A></LI></UL></DIV><DIV id="newnavi"><UL class="newmenu"><LI><A target="new" href="https://2012.igem.org/" title="Back to iGEM">iGEM</A></LI><LI><A target="new" href="https://2012.igem.org/">Main iGEM</A></LI><LI><A href="https://2012.igem.org/Team:UC_Davis/Criteria">Criteria</A></LI><LI><A href="https://2012.igem.org/Team:UC_Davis/Human_Practices">Human Practices</A></LI></UL><LI><A href="https://2012.igem.org/Team:UC_Davis/Attributions" title="Attributions">Attributions</A></LI><LI><A title="https://2012.igem.org/Team:UC_Davis/Data" title="Data">Data</A></LI><LI><A href="https://2012.igem.org/Team:UC_Davis/Data/Cutinase_Activity" title="Data">Cutinase Activity</A></LI><LI><A href="https://2012.igem.org/Team:UC_Davis/Data/Ethylene_Glycol" title="Data">Ethylene Glycol</A></LI><LI><A href="https://2012.igem.org/Team:UC_Davis/Data/Modeling" title="Data">Modeling</A></LI><LI><A href="https://2012.igem.org/Team:UC_Davis/Parts">Parts</A></LI><LI><A href="https://2012.igem.org/Team:UC_Davis/Notebook" title="Notebook">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:UC_Davis/Notebook">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:UC_Davis/Notebook/Protocols ">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:UC_Davis/Notebook/Gallery">Gallery</A></LI><LI><A href="https://2012.igem.org/Team:UC_Davis/Safety" title="Safety">Safety</A></LI><LI class="selected"><A href="https://2012.igem.org/Team:UC_Davis/Project" title="Project">Project</A></LI><LI><A href="https://2012.igem.org/Team:UC_Davis/Project">Project Overview</A></LI><LI><A href="https://2012.igem.org/Team:UC_Davis/Project/Catalyst">Module Engineering</A></LI><LI><A href="https://2012.igem.org/Team:UC_Davis/Project/Protein_Engineering">Protein Engineering</A></LI><LI><A title="https://2012.igem.org/Team:UC_Davis/Project/Strain">Chassis Engineering </A></LI><LI><A href="https://2012.igem.org/Team:UC_Davis/Project/Strain">Background</A></LI><LI><A href="https://2012.igem.org/Team:UC_Davis/Project/Directed_Evolution">Directed Evolution</A></LI><LI><A href="https://2012.igem.org/Team:UC_Davis/Project/Our_Strain">Rational Engineering </A></LI><LI><A href="https://2012.igem.org/Team:UC_Davis/Team" title="Team">Team</A></LI><LI><A href="https://2012.igem.org/Team:UC_Davis" title="Home">Home</A></LI></DIV><DIV id="bodyContent"><DIV id="sponsorbox"><DIV id="sponsorbox" class="sponsorfloat"><DIV id="tweets"><CENTER><H2>UCDavis iGEM Tweets</H2></CENTER></DIV></DIV><DIV id="sponsorbox" class="sponsorfloat"><CENTER><H2>Our Sponsors</H2></CENTER></DIV></DIV><DIV id="myleftbox"><DIV class="smallbox" id="myleftbox"><H1>Protein Engineering</H1><ARTICLE>

Beyond the general circuit outline for our degradation of PET, there are also a protein engineering project we performed. In the project we constructed a more effective and higher yield form of the LC-Cutinase protein. We hoped to move toward a more realistic and economically feasible rate of degradation. 
</ARTICLE></DIV><DIV class="smallbox" id="myleftbox"><H1>LC-Cutinase Engineering</H1><P>Confirming expectation of common results from previous mutations</P><ARTICLE>
We had two goals with the engineering of cutinase: replicate mutations made in the paper by Silva et. al., which consisted of three residues mutated to alanines (1), as well as to generate our own chosen mutations. Before we decided to replicate Silva’s mutations we initially wanted to validate if we could expect the same results in our protein with the same mutations. In order to do this we wanted to compare best fit models (proteins with known structure who's sequence best matches an input sequence) generated for LC-Cutinase’s sequence and Silva’s Tfu_0883’s sequence. Replicating Silva et al. (1) we used <A target="new" href="http://swissmodel.expasy.org/">Swiss-Model</A> to generate a best fit model for LC-Cutinase, resulting in the protein 3visB with 61% homology. In Silva’s paper they received 1jfr, a model that was second best in LC-Cutinase’s output, with a 55% homology (1). This similarity between generated models and high homology rate gave support to the expectation of similar results from the same mutations.
<TABLE background="transparent"><TBODY><TR><TD><CAPTIONTEXT>Multiple sequence alignment results. Positions 177 (Serine) 225 (Asparagine) 255 (Histidine) on the bottom consensus sequence mark active domains which can be seen to hold constant throughout the multiple sequences. </CAPTIONTEXT></TD><TD>Further validation was found using a multiple sequence alignment (shown to the right) using <A target="new" href="http://tcoffee.crg.cat/apps/tcoffee/do:regular">T-Coffee</A> between: LC-Cutinase, 3visB, Tfu_0883 and 1jfr. The alignment showed strong homology in all active site residues as well as between two of the three residues targeted by Silva et. al. for mutation. The third targeted residue, though significantly different in Tfu_0883 compared to in LC-Cutinase, was expected to result in similar protein activity gains upon mutation to alanine. This reasoning was supported by loading both models for 3visB and 1jfr in Pymol and assessing that the residues held a similar placement above the active site. Their placement, as well as the placement of the other two targeted residues, also showed that mutation to a smaller residue, alanine, would result in better binding ability into the active site by the ligand.</TD></TR></TBODY></TABLE><P>Generating our own theoretical mutations</P><TABLE background="transparent"><TBODY><TR><TD><CAPTIONTEXT>SwissDock results of best fit sites for PET in LC-Cutinase.</CAPTIONTEXT><CAPTIONTEXT>SwissDock results for ultimate best fit position of PET in LC-Cutinase</CAPTIONTEXT></TD><TD>
After validating the previous mutations we wished to make our own. The first step was to generate a theoretical 3D model for LC-Cutinase. This was done by using Swiss-Model again to fit LC-Cutinase’s amino acid sequence into the structure of 3visB. The results, including the stability of each part of the sequence, are shown below. The results show generally high stability along the sequence. Also, upon overlapping this newly generated model of LC-Cutinase with 3visB’s model in Pymol  we received a low RMS value of .78A. This resulting data supported the idea that the generated 3D model for LC-Cutinase was stable and fit 3visB’s structure closely. The next stage was to get our theoretical model for LC-Cutinase and the two ligands it would interact with, PET and pNPB, into the <A target="new" href="http://fold.it/portal/">Foldit program</A>, a protein folding simulator. In Foldit we could perform and assay theoretical point mutations for increased stability and function of the protein. However, because Foldit does not directly measure function but measures stability instead, it was necessary to load LC-Cutinase with each ligand docked correctly into the active site. This would allow changes made at the active site that increased stability due to better ligand/protein interaction to also equate to better ligand/protein binding and thus better function by the protein. We created the ligand files through <A target="new" href="http://cactus.nci.nih.gov/translate/">Cactus</A> and the docking of the protein with each of the two ligands was done through <A target="new" href="http://swissdock.vital-it.ch/">Swissdock</A>. The images to the left show the overall docking results as well as the best fit ligand binding position. From here we loaded each specific ligand/protein combo into foldit and generated the following mutations with each rational.</TD></TR></TBODY></TABLE><P>Mutant List and Rationale</P><UL><LI><B>Silva et al. Mutant Replicants</B></LI><LI>T96A: Increase active site and change a hydrophilic residue for a hydrophobic alanine</LI></UL><UL><LI>Y127A:Increase active site and change a hydrophilic residue for a hydrophobic alanine</LI></UL><UL><LI>V212A:Enlarge active site</LI></UL><UL><LI><B>Team chosen mutants</B></LI><LI>S101A:Increase active site and change a hydrophilic residue for a hydrophobic alanine</LI></UL><UL><LI>Y95A:Greatly open active site by removal of bulky side group and increase active site's hydrophobicity</LI></UL><UL><LI>D98T:Enlarge active site and generated hydrogen bond</LI></UL><UL><LI>F125R:Enlarge active site</LI></UL><UL><LI>F125Y:Enlarge active site</LI></UL><UL><LI>F125A:Increase active site and change a hydrophilic residue for a hydrophobic alanine</LI></UL><UL><LI>T96G:Similar as T96A however amino acid change takes place right before a alpha helix and a replacement with a G instead of A will avoid helix breaking</LI></UL><P> Additional Photos</P>1) Homology model results. Right shows overall stability across amino acid sequence.  
2) PET (Yellow and red) and LC Cutinase model loaded in Foldit
 3) Overlapping of 3VisB and generated LC Cutinase homology models.
</ARTICLE></DIV><DIV class="smallbox" id="myleftbox"><H1> References </H1><ARTICLE>
1. Silva C, et al. 2011. Engineered Thermobifida fusca cutinase with increased activity on polyester substrates. Biotechnol. J. 6:1230–1239.
 
2. S. Sulaiman, S. Yamato, E. Kanaya, J. Kim, Y. Koga, K. Takano, S. Kanaya. &quot;Isolation of a Novel Cutinase Homolog with Polyethylene Terephthalate-Degrading Activity from Leaf-Branch Compost by Using a Metagenomic Approach.&quot; Applied and Environment Microbiology, vol. 78 no. 5, pp. 1556-1562, March 2012.

3. Ö. Faiz et al.
Determination and characterization of thermostable esterolytic activity from a novel thermophilic bacterium Anoxybacillus gonensis
J. Biochem. Mol. Biol., 40 (2007), pp. 588–594</ARTICLE></DIV><DIV id="myleftbox" class="smallboxsite"><UL style="font-size:10px;list-style-image:none;list-style-type:none;float:left;display:inline;color:#000000;"><LI style="float:left;margin:0 10px;"><A href="https://2012.igem.org/Team:UC_Davis"><P>Home</P></A></LI><LI><A style="color:#000000 " href="https://2012.igem.org/Team:UC_Davis">Welcome</A></LI><LI><A style="color:#000000 " href="https://2012.igem.org/Team:UC_Davis">Tweets</A></LI><LI><A style="color:#000000 " href="https://2012.igem.org/Team:UC_Davis">Sponsors</A></LI><LI><A style="color:#000000 " href="https://2012.igem.org/Team:UC_Davis/Criteria">Criteria</A></LI></UL><LI style="float:left;margin:0 10px;"><A href="https://2012.igem.org/Team:UC_Davis/Team"><P>Team</P></A></LI><LI><A style="color:#000000 " href="https://2012.igem.org/Team:UC_Davis/Team">Who we are</A></LI><LI><A style="color:#000000 " href="https://2012.igem.org/Team:UC_Davis/Team">Students</A></LI><LI><A style="color:#000000 " href="https://2012.igem.org/Team:UC_Davis/Team">Advisors</A></LI><LI style="float:left ;margin:0 10px;"><A href="https://2012.igem.org/Team:UC_Davis/Project "><P>Project</P></A></LI><LI><A style="color:#000000 " href="https://2012.igem.org/Team:UC_Davis/Project">Project Overview</A></LI><LI><A style="color:#000000 " href="https://2012.igem.org/Team:UC_Davis/Project/Catalyst ">Module
Engineering</A></LI><LI><A style="color:#000000 " href="https://2012.igem.org/Team:UC_Davis/Project/Protein_Engineering
">Protein Engineering</A></LI><LI><A style="color:#000000 " href="https://2012.igem.org/Team:UC_Davis/Project/Strain ">Chassis
Engineering</A></LI><LI><A style="color:#000000 " href="https://2012.igem.org/Team:UC_Davis/Project/Directed_Evolution ">
- Directed Evolution</A></LI><LI><A style="color:#000000 " href="https://2012.igem.org/Team:UC_Davis/Project/Our_Strain "> -
Rational Engineering</A></LI><LI><A style="color:#000000 " href="https://2012.igem.org/Team:UC_Davis/Criteria">Critera</A></LI><LI style="float:left ;margin:0 10px"><A href="https://2012.igem.org/Team:UC_Davis/Safety "><P>Safety</P></A><A style="color:#000000 " href="https://2012.igem.org/Team:UC_Davis/Safety "> Safety</A></LI><LI style="float:left ;margin:0 10px;"><A href="https://2012.igem.org/Team:UC_Davis/Notebook "><P>Notebook</P></A></LI><LI><A style="color:#000000 " href="https://2012.igem.org/Team:UC_Davis/Notebook">Notebook</A></LI><LI><A style="color:#000000 " href="https://2012.igem.org/Team:UC_Davis/Notebook/Protocols
">Protocols</A></LI><LI><A style="color:#000000 " href="https://2012.igem.org/Team:UC_Davis/Notebook/Gallery">Gallery</A></LI><LI><A style="color:#000000 " href="https://2012.igem.org/Team:UC_Davis/Criteria">Critera</A></LI><LI style="float:left ;margin:0 10px;"><A title="https://2012.igem.org/Team:UC_Davis/Data "><P>Data </P></A></LI><LI><A style="color:#000000 " href="https://2012.igem.org/Team:UC_Davis/Data/Cutinase_Activity ">
Cutinase Activity</A></LI><LI><A style="color:#000000 " href="https://2012.igem.org/Team:UC_Davis/Data/Ethylene_Glycol ">
Ethylene Glycol</A></LI><LI><A style="color:#000000 " href="https://2012.igem.org/Team:UC_Davis/Data/Modeling ">
Modeling</A></LI><LI><A style="color:#000000 " href="https://2012.igem.org/Team:UC_Davis/Parts ">Parts</A></LI><LI style="float:left ;margin:0 10px"><A href="https://2012.igem.org/Team:UC_Davis/Attributions "><P>Attribution </P></A><A style="color:#000000 " href="https://2012.igem.org/Team:UC_Davis/Attributions ">
Attribution</A></LI><LI style="float:left ;margin:0 10px"><A href="https://2012.igem.org/Main_Page "><P>iGEM </P></A></LI><LI><A style="color:#000000 " href="https://2012.igem.org/Main_Page
">Main iGEM</A></LI><LI><A style="color:#000000 " href="https://2012.igem.org/Team:UC_Davis/Criteria "> Criteria</A></LI><LI><A style="color:#000000 " href="https://2012.igem.org/Team:UC_Davis/Human_Practices ">Human
Practices</A></LI></DIV></DIV></DIV></DIV></DIV><DIV id="footpage"><DIV class="printfooter">
Retrieved from &quot;<A href="https://2012.igem.org/Team:UC_Davis">https://2012.igem.org/Team:UC_Davis</A>&quot;</DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:UC_Davis/Project/Protein_Engineering">http://2012.igem.org/Team:UC_Davis/Project/Protein_Engineering</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UC_Davis/Project/Protein_Engineering" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UC_Davis/Project/Protein_Engineering" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UC_Davis/Project/Protein_Engineering&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UC_Davis/Project/Protein_Engineering&amp;oldid=296429" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>