"
Page
Discussion
View source
History
teams
Log in
 
Team:UC Davis/Project/Protein Engineering
From 2012.igem.org
Team:UC Davis - 2012.igem.org
teams
Page
Discussion
Edit
History
Move
Watch
Log in
iGEM
Main iGEM
Criteria
Human Practices
Attributions
Data
Cutinase Activity
Ethylene Glycol
Modeling
Parts
Notebook
Notebook
Protocols
Gallery
Safety
Project
Project Overview
Module Engineering
Protein Engineering
Chassis Engineering
Background
Directed Evolution
Rational Engineering
Team
Home
UCDavis iGEM Tweets
Our Sponsors
Protein Engineering
Beyond the general circuit outline for our degradation of PET, there are also a protein engineering project we performed. In the project we constructed a more effective and higher yield form of the LC-Cutinase protein. We hoped to move toward a more realistic and economically feasible rate of degradation.
LC-Cutinase Engineering
Confirming expectation of common results from previous mutations
We had two goals with the engineering of cutinase: replicate mutations made in the paper by Silva et. al., which consisted of three residues mutated to alanines (1), as well as to generate our own chosen mutations. Before we decided to replicate Silva’s mutations we initially wanted to validate if we could expect the same results in our protein with the same mutations. In order to do this we wanted to compare best fit models (proteins with known structure who's sequence best matches an input sequence) generated for LC-Cutinase’s sequence and Silva’s Tfu_0883’s sequence. Replicating Silva et al. (1) we used Swiss-Model to generate a best fit model for LC-Cutinase, resulting in the protein 3visB with 61% homology. In Silva’s paper they received 1jfr, a model that was second best in LC-Cutinase’s output, with a 55% homology (1). This similarity between generated models and high homology rate gave support to the expectation of similar results from the same mutations.
Multiple sequence alignment results. Positions 177 (Serine) 225 (Asparagine) 255 (Histidine) on the bottom consensus sequence mark active domains which can be seen to hold constant throughout the multiple sequences.
Further validation was found using a multiple sequence alignment (shown to the right) using T-Coffee between: LC-Cutinase, 3visB, Tfu_0883 and 1jfr. The alignment showed strong homology in all active site residues as well as between two of the three residues targeted by Silva et. al. for mutation. The third targeted residue, though significantly different in Tfu_0883 compared to in LC-Cutinase, was expected to result in similar protein activity gains upon mutation to alanine. This reasoning was supported by loading both models for 3visB and 1jfr in Pymol and assessing that the residues held a similar placement above the active site. Their placement, as well as the placement of the other two targeted residues, also showed that mutation to a smaller residue, alanine, would result in better binding ability into the active site by the ligand.
Generating our own theoretical mutations
SwissDock results of best fit sites for PET in LC-Cutinase.
SwissDock results for ultimate best fit position of PET in LC-Cutinase
After validating the previous mutations we wished to make our own. The first step was to generate a theoretical 3D model for LC-Cutinase. This was done by using Swiss-Model again to fit LC-Cutinase’s amino acid sequence into the structure of 3visB. The results, including the stability of each part of the sequence, are shown below. The results show generally high stability along the sequence. Also, upon overlapping this newly generated model of LC-Cutinase with 3visB’s model in Pymol  we received a low RMS value of .78A. This resulting data supported the idea that the generated 3D model for LC-Cutinase was stable and fit 3visB’s structure closely. The next stage was to get our theoretical model for LC-Cutinase and the two ligands it would interact with, PET and pNPB, into the Foldit program , a protein folding simulator. In Foldit we could perform and assay theoretical point mutations for increased stability and function of the protein. However, because Foldit does not directly measure function but measures stability instead, it was necessary to load LC-Cutinase with each ligand docked correctly into the active site. This would allow changes made at the active site that increased stability due to better ligand/protein interaction to also equate to better ligand/protein binding and thus better function by the protein. We created the ligand files through Cactus and the docking of the protein with each of the two ligands was done through Swissdock . The images to the left show the overall docking results as well as the best fit ligand binding position. From here we loaded each specific ligand/protein combo into foldit and generated the following mutations with each rational.
Mutant List and Rationale
Silva et al. Mutant Replicants
T96A: Increase active site and change a hydrophilic residue for a hydrophobic alanine
Y127A:Increase active site and change a hydrophilic residue for a hydrophobic alanine
V212A:Enlarge active site
Team chosen mutants
S101A:Increase active site and change a hydrophilic residue for a hydrophobic alanine
Y95A:Greatly open active site by removal of bulky side group and increase active site's hydrophobicity
D98T:Enlarge active site and generated hydrogen bond
F125R:Enlarge active site
F125Y:Enlarge active site
F125A:Increase active site and change a hydrophilic residue for a hydrophobic alanine
T96G:Similar as T96A however amino acid change takes place right before a alpha helix and a replacement with a G instead of A will avoid helix breaking
Additional Photos
1) Homology model results. Right shows overall stability across amino acid sequence.
2) PET (Yellow and red) and LC Cutinase model loaded in Foldit
3) Overlapping of 3VisB and generated LC Cutinase homology models.
References
1. Silva C, et al. 2011. Engineered Thermobifida fusca cutinase with increased activity on polyester substrates. Biotechnol. J. 6:1230–1239.
2. S. Sulaiman, S. Yamato, E. Kanaya, J. Kim, Y. Koga, K. Takano, S. Kanaya. "Isolation of a Novel Cutinase Homolog with Polyethylene Terephthalate-Degrading Activity from Leaf-Branch Compost by Using a Metagenomic Approach." Applied and Environment Microbiology, vol. 78 no. 5, pp. 1556-1562, March 2012.
3. Ö. Faiz et al. Determination and characterization of thermostable esterolytic activity from a novel thermophilic bacterium Anoxybacillus gonensis J. Biochem. Mol. Biol., 40 (2007), pp. 588–594
Home
Welcome
Tweets
Sponsors
Criteria
Team
Who we are
Students
Advisors
Project
Project Overview
Module Engineering
Protein Engineering
Chassis Engineering
- Directed Evolution
- Rational Engineering
Critera
Safety
Safety
Notebook
Notebook
Protocols
Gallery
Critera
Data
Cutinase Activity
Ethylene Glycol
Modeling
Parts
Attribution
Attribution
iGEM
Main iGEM
Criteria
Human Practices
Retrieved from " https://2012.igem.org/Team:UC_Davis "
Retrieved from " http://2012.igem.org/Team:UC_Davis/Project/Protein_Engineering "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
