"
Page
Discussion
View source
History
teams
Log in
 
Team:UC Davis/Notebook
From 2012.igem.org
Team:UC Davis - 2012.igem.org
teams
Page
Discussion
Edit
History
Move
Watch
Log in
iGEM
Main iGEM
Criteria
Human Practices
Attributions
Data
Cutinase Activity
Ethylene Glycol
Modeling
Parts
Notebook
Notebook
Protocols
Gallery
Safety
Project
Project Overview
Module Engineering
Protein Engineering
Chassis Engineering
Background
Directed Evolution
Rational Engineering
Team
Home
UCDavis iGEM Tweets
Our Sponsors
01
02
03
04
05
06
07
08
09
10
11
12
13
14
15
16
17+
Week 1
Monday June 18th
Today was the first day of out iGEM team officially meeting in the summer. We got a look around the laboratory space we would be using, and since almost everyone on the team is new to iGEM, we went over the things we had to accomplish over summer. We rehydrated some parts from the distribution kits to get practice, and those parts were: J23101 E0240 E0040 R0010 pSB1C3 pSB1A3 pSB1AK3 pSB1K3 B0034 I13602
These parts were later transformed into the E. Coli strain DH5α.  We cultured 4 parts from the glycerol stocks for more practice. The parts were: J45120, which emits a wintergreen scent. This was done in 2mM salicylic acid. J45200, which emits a banana scent, respectively done in 5mM isoamyl alcohol. R0010, as a promoter E0240, or more commonly known as GFP! Our team took time to discuss our project, and comparing our interest with the research that has been conducted thus far, we narrowed our project down to three ideas:
1. Producing spider silk proteins in bacteria.
2. Degrading plastics in bacteria.
3. Creating an Archaea toolkit and using them for bioplastic, biofuel, and isoprenoid production.
We realized that more research had to be done to understand what each project specifically requires to judge how feasible and successful it could be.
Tuesday June 19th
We began the day with some wetlab work to keep on practicing techniques. The E. coli cultured with J45200 smelled like bananas, which was exciting! The R0010 and E0240 liquid cultures were miniprepped, but we messed up on them due to inexperience. More practice was done to learn the procedures of DNA purifications, while other members of the team nanodropped the miniprepped R0010 and E0240 liquid cultures for practice using the machine. Following the nanodrop, a digestion was set up of the R0010 and E0240 parts. R0010 was cut at SpeI and PstI sites; E0240 was cut at XbaI and PstI. We liquid cultured the transformations from yesterday, and made plates and LB media. In terms of our project, more research was done on spider silk and plastic degradation, as we read papers on studies that dealt with both themes and we looked at the methodology used to approach the problems, searching for similar assays and pathways that we could use.
Wednesday June 20th
As soon as we came into lab today, some of our team members began miniprepping the liquid cultures from yesterday. These were nanodropped, and the concentrations looked good! Some of our team members were also taught how to make and use gels, as we ran the digested parts from yesterday on a one percent agarose gel. We also made another stock of 1x TAE buffer from the 50x TAE buffer by dilution. Once extracted, the parts were ligated and transformed, which took some time. Once finished, we stored them for further examination. In terms of research, we spoke with one of our mentors, and narrowed down the possible project ideas to either plastic degradation or a toolkit for archaea. The spider silk seemed to be more and more impractical as we read more papers concerning the spinning and formation of the fibrous materials that make up spider silk.
Thursday June 21st
Today consisted of mainly reading and researching more into pathways for plastic degradation and meeting the gold medal requirements. The Ligation from yesterday turned out to be unsuccessful, but the parts we were trying to ligate were not immediately necessary, but it was rather done more for practice. We also went out as a team to a sushi buffet, continuing a tradition of team bonding on Thursday for lunch.
Friday June 22nd
Friday was assigned as a weekly meetup day with our entire team and our advisors. We explained where we are at in terms of researching for our project, and they gave us some interesting aspects we can look at. We narrowed our project down to just plastic degradation, as the archaea engineering would be a clear cut contribution to the parts registry, but would not be as interesting to the team as a whole and not a sufficient project to pursue for the entire summer. However, it was suggested that we make a toolkit for parts for archaea so future iGEM teams can pursue constructs in archaea.
Week 2
Monday June 25th
Monday signified the beginning of week two for iGEM! The day was mainly composed of reading papers and looking into ways to which we could degrade plastic. We also scheduled a visit to a local landfill, with an intent to see how the landfill deals with plastics and other synthetic polymers. The toolkit idea was also considered a secondary project idea.
Tuesday June 26th
We practiced lab protocols a bit more, and conducted more research after finding out the planned trip to the landfill. We also prepared questions and did research concerning what the landfill does and how we can benefit from it.
Wednesday June 27th
Today we went to the Yolo County Landfill to meet with Ramin Yazdani, the senior civil engineer at the landfill. We were given a tour of the landfill, and discussed procedures and systems Dr. Yazdani has in place regarding the decomposition of trash, specifically in terms of methanogens and methanotrophs. He similarly provided us with guidance concerning the direction of our project, and the possible application and impact our project could potentially have in terms of trash degradation.
Thursday June 28th
We spoke with our mentors regarding the feasibility of our project within the given time span. Ruled out polyurethane due to toxic byproducts.
Friday June 29th
Conducted more research in engineering methanogens and methanotrophs. We also looking into how our project could apply to the landfill, and discussed the feasibility of PET degradation versus methanotroph and methanogen consumption of methane.
Week 3
Monday July 2nd
We further looked into the means by which our project can be outlined and planned.
Our team read a paper titled Isolation of a Novel Cutinase Homolog with Polyethylene Terephthalate-Degrading Activity from Leaf-Branch Compost by Using a Metagenomic Approach to further understand the mechanism by which cutinase assists in plastic degradation.
We looked at the value of surface modification, specifically how it alters PET and whether or not it is valuable.
Research was conducted on PelB and the potential for a scar, and furthermore whether a scar will influence our results.
Tuesday July 3rd
Today we focused on wet-lab procedures -We rehydrated the PelB tag (J32015), T7 constitutive promoter (I719005), Stop (B0015), And LuxR inducible promoter (C0062).
We also transformed the parts mentioned above. ( Transformation Protocol )
We plated the parts.
Wednesday July 4th
Ran liquid cultures on B0015 (stop codon), I719005 (T7), J32015 (pelB), C0062 (luxR).
We took a half day for July 4th!
Thursday 5th
We conducted a double digest of the T7 constitutive promoter (I719005) and Ribosome Binding Site (B0034). ( Double Digest Protocol )
We worked on the safety page of our wiki after concern arose regarding the safety of procedures in our planned project. We continued research regarding how we can plan and carry out our project.
Friday July 6th
We created an in-depth plan for how we were going to assess our project. We divided the project up into manageable sections and assigned particular areas to people on our team. One area we decided on was protein engineering in which we hoped to engineer a more efficient cutinase protein. Minipreped J23101 and digested it.
We took a look at the requirements for achieving the Gold medal, and wrote down and organized how we are going to achieve each criteria.
Week 4
Monday July 9th
We wanted to rehydrate some parts, they are listed below. They were located in the distribution kits.
J23100 J23119 J23100 I13453 I13458 C0012 K206000 Along with these parts, we also looked at a new miniprep kit that was given to us for free, titled Biobasic. We wanted to test it in comparison to the Invitrogen kit we currently use. Following the hydration we transformed the parts and left them overnight in the 37 °C room. Ran the digested J23101 on a gel.
Tuesday July 10th
Today we did a double digest of B0034 at the S and P sites and sequentially digested B0034 first at the E site using Buffer 4. We miniprepped samples using both the Invitrogen and Biobasic kits. Using the nanodrop spectroscopy, we determined that the Invitrogen kit was still better, so we continued to use that. We also began to look into Gibson assembly, and due to past iGEM inability to attain success, we requested the protocol from the iGEM Washington 2011 team. Minipreped part B0034 (RBS) and digested it in two ways, E+X and S+P.
Wednesday July 11th
Took cutinase sequence and put it into swiss model, gave us a model that was similar. Starting off the protein engineering side of the project we looked towards first replicating previously made mutations in the paper titled “Engineered Thermobifida fusca cutinase with increased activity on polyester substrates” referenced in our original paper, “Isolation of a Novel Cutinase Homolog with Polyethylene Terephthalate-Degrading Activity from Leaf-Branch Compost by Using a Metagenomic Approach.” We wished to replicate three mutations, T96A, V212A and Y127A. We wanted to verify that these mutations would produce similar increases in protein activity in our protein, LC Cutinase, as they did in the paper’s protein Tfu_0883. In order to do this we wished to compare best fit homology models generated for our protein and the papers. Using Swissmodel we generated 3VisB as a best fit for LC Cutinase and 1JFR as the best fit for the paper’s Tfu_0083.
Thursday July 12th
We performed digests, double digests, and ligations. LC Cutinase was ordered at Genscript. We further looked over the the results of the homology models generated for the two proteins. We compared them in two ways: 1) Generating a multiple sequence alignment for both proteins and homology models and accessing the similarity of the mutation sites. 2) Loading the homology models in pymol and accessing the three dimensional similarity visually. Both methods gave us confidence in the mutations resulting in similar activity increase. J23100 and J23101 minipreps ran.
Friday July 13th
We performed gel extraction and redid double digest of parts that were mislabeled. We contacted Piedmont Plastics about specific kinds of PET. After looking over the results of the mutation verification with our mentors it was decided that we could expect with high confidence the mutations conferring similar results.
Week 5
Monday July 16th
Our team was finally back together, as a majority of people were absent for the latter part of last week. A few of us went over what was done last week, and how there was an error in running the gel extractions, so we made a decision to reattach the promoters to the B0034 RBS. Some of us began by running digestions of J23100, K206000, J23101, and R0010 at the E and S sites. B0034+C0012 was digested at X and P, J23101 was digested at S and P, and B0034 was digested at E and X. We also wanted to carry on some of the themes from last years project, and began trying to mutate K206000, or the pBAD promoter, to improve its function through random mutagenesis. Some of our team members also began preparing the E. Coli strain, numbered MG1655, by streaking the organisms on plates from the glycerol stock. After, we rehydrated MG1655 strain. We also liquid cultured and plated DH5α cells and tried to make competent cells for MG1655 strain, but put them in the 37 °C room. Some of our team members began to run error prone PCR ( PCR Protocol ) on the K206000 part as well, to make a part family as well. In our discussion of how we were going to assay our samples, we also ordered pNPB, another sample used to determine degradation in comparison to PET. With the previous mutations confirmed we looked towards generating our own mutations. The first step was generating a 3D structure from LC Cutinase’s sequence (LC Cutinase being newly discovered has no known 3D model). We started this by using Swiss-model again to fit LC Cutinase’s sequence into the 3D model of 3VisB. This process took Swiss-model the most of the rest of the day to generate.
Tuesday July 17th
When the team came into lab today, we used the spectrophotometer and the OD600 readings were over 1, which signifies we overgrew them. The error came from placing the cells in the 37 °C room instead of at room temperature. Oops. So we then remade the component cells for the MG1655 strain and extracted genomic DNA from the MG1655 and DH5α E. Coli strain. Luria Broth was made, and we incubated the starter culture and made inoue transformation buffer. We miniprepped the parts that were transformed yesterday, and sequentially digested B0034 and B0034+E0240 at the E site with plans for the X site to be digested on Wednesday. The PCR products from yesterday were ran on a gel to confirm the products, and then a purification was run, and the process was repeated twice. In the area of protein engineering we looked over the Swiss-Model generated 3D structure and confirmed its realism. The energy diagram showed that LC Cutinases sequence fit quite well into the model of 3VisB. We also performed an overlap of the two models, the newly generated LC Cutinase model and 3VisB’s model, in order to check how well they matched. We received an RMS value of .78A, quite good. Minipreps ran B0034, J23100, B0034+E0240, B0034+C0012, R0010, K20600
Wednesday July 18th
Today genomic DNA from DH5α and MG1655 was purified and we restreaked DH5α cells from yesterdays sample. The purified PCR from yesterday was also put on a gel and extracted. Some of our team members also began working on the modeling of the cutinase enzyme using computer software to see how we could optimize degradation. More research was done in regards to making sure we had all the materials and that the assays to be conducted were also looked into and finalized. A gel was also ran of the purified MG1655 and DH5α genome. Going further with our generated model of LC Cutinase we wished to create the two ligands with which it would interact, PET and pNPB. We did this in SMILES (a structural file generator). We then looked to fit these ligands into LC Cutinase’s active site in pymol and load that file in Foldit. Ran gels on minipreps and performed gel extractions and purification.
K23100
J23101
K206000
R0010
B0034
B0034+C0012
J23101
B0034+E0240
Thursday July 19th
The day started off with us receiving the MG1655 strain we ordered from Yale University. Even though we already began to use samples from one of our mentors' labs, it was a good idea for us to make liquid cultures and plates of the cells in case something went wrong. The two enzymes required in the pathway for breaking down ethylene glycol also underwent PCR along with genomic data from the MG1655 and DH5α cells. Luria Broth (LB) was made for kanamycin plates as well. We preformed ligation and transformations of the genomic data into the DH5α cells, and successfully cultured competent cells. At the end of the day, we rehydrated primers for the mutated cutinase as well, with a hope that the mutations will improve the cutinase functioning. On the protein end of things we manually fit the ligands into the active site of LC Cutinase in pymol. However, upon loading these files in Foldit the protein did not wrap around the ligand as hoped. We then seeked out an alternative means in which to better fit or dock the ligands into the active site. Nano-dropped gel purifications.
Friday July 20th
We purified PCR of dehydrogenase. Competent cells failed, so we remade some more while making some modifications to the protocol. In addition we performed some gel DNA clean up, mini preps, digestions, ligations, and transformations. We decided on using Swissdock, a ligand to protein docking program, to fit our two ligands into LC Cutinase. This program ran a majority of the day but we received successful docking files for both ligands by the end of the day. Minipreps of J23100, J23101, K206000, R0010, B0034
Week 6
Monday July 23rd
We ran a gel and did some PCR cleanup. More work was done on SOEing. Did some digests, sent stuff for sequencing, and worked on reductase part construction. For the protein part of the project we moved forward with our successful docking files to load them into the Foldit program. However, we ran into technical difficulties with the requirements Foldit had for file types accepted. Ran gel digests/ purifications on J23100, J23101, K206000, R0010, B0034
Tuesday July 24th
On the protein engineering part we discussed with our professor the issues we had with the files. He gave us a helpful layout of the steps needed to convert files to the correct format as well as generate certain needed files. This took the rest of the day but we were able to load both ligands bound to LC Cutinase into Foldit successfully. Redigested B0034.
Wednesday July 25th
Ethylene glycol was ordered today. We did some more digest, double digest, and ligation. For the protein engineering part we now used Foldit to come up with our own mutations listed in the protein engineering page. We also further tested the previously made mutations in Foldit to helped substantiate our confidence in them further.
Thursday July 26th
We made forward and reverse primers so that the reductase could work in aerobic conditions. We performed ligations of K20600+B0034 (1), K20600+B0034 (3), reducates+psBIA3, vector controls (B0034 [1] and [3] and pSBIA3), and insert control K206000. For the protein part we had our mentors look over our generated mutations and decided upon seven different individual mutations.
Friday July 27th
Today was the day scheduled for the iGEM meetup! After doing some stuff in the lab (find out what everyone did), our team left around 1 P.M. for Genentech Hall at the UC San Francisco campus. There we interacted with teams from UC Berkeley, UCSF, and Stanford-Brown and presented our respective projects. The advice and critiques were helpful to us as it gave us a different viewpoint of how we can improve our project. We also socialized outside following the presentations, and it was an interesting experience for our team as a whole. Today we finalized the generated protein mutations and used a website called PrimeX to generate the SDM primers for all ten mutations.
Week 7
Monday July 30th
More work was done in the soeing of DNA together while removing the restriction sites, however we ran into some errors while trying to correctly follow the procedure. Dehydrogenase and Reductase were digested at X and S
Tuesday July 31st
The team split up again for more work and research. We finally got the strain we requested from Dr. Aguilar in Barcelona (Strain E-15 EG3), and we also received the primers for the glycoaldehyde reductase compatibility with an aerobic environment, so that was exciting. We similarly got a stock of ethylene glycol for which we can test the activity of the enzymes, and began culturing on it. We also continued our work on the SOEing procedure, trying to remove the Pstl restriction sites from the sequences after a brief mix up yesterday. We also got in contact with another professor here at UC Davis that does work in textiles and material sciences, and obtained some samples of PET that we can use in our assays of our enzyme.
Wednesday August 1st
After talking with our advisor we decided to pursue the Entrepreneurial track in the iGEM competition. We ran a PCR on Reaction 7 and Reaction 8. After, we ran the products on a gel and PCR purified them. The screening gel showed Reaction 8 to be successful as well!
Thursday August 2nd
We skyped with the UC Merced team and looked at possible collaborations we could possibly have with them. It was also a chance for us to impart some of our knowledge to them, as this is their first year participating in the iGEM competition. We also spoke with the CEO of Micromidas Inc., a green technology startup based in Sacramento that does work in taking waste water and sludge, and through microbial processes, generating biodegradable plastics for marketing. This gave us some insight into the economic situation of the environmental market and how we could conduct an economic analysis of our project. We got the Cutinase. Transformed that into DH5α and MG1655 and we left that overnight. Ran a gel of many of our cutinase constructs, various backbones and both dehydrogenase and reductase.
Friday August 3rd
We ran a gel and PCR purification of the PCR from yesterday, adding an Eco cut site to the reductase and dehydrogenase parts. After, a gel extraction on the digest of reaction 8 was also done. (pg. 24 of Sahar's notebook)
Week 8
Monday August 6th
We mini prepped R0010, Cutinase MG1655, and Cutinase DH5α. The cutinase and backbone minipreps were double digested at E + P sites. There were growth on E6 plates of DH5α but not from MG1655.
Tuesday August 7th
Ligated and transformed parts, plated them and left them overnight.
Wednesday August 8th
We received vectors to conduct gibson assembly today. Ran a screening gel after doing a PCR to see if the products were actually what we thought they were. Seemed right, attach picture! We also ran a PCR gradient on Kanamycin because it seemed to not be working with the regular PCR and we wanted to test different annealing temperatures to see if it would work. We ligated: Cutinase from DH5α cut at E+P with pSB1A3 cut at E+P, Cutinase from MG1655 cut at E+P with pSB1A3 cut at E+P We also ligated a vector control of just pSB1A3 cut at E+P and an insert control the cutinase from both DH5α and MG1655. After the ligations, the parts were transformed and left overnight.
Thursday August 9th
Today we made tecan liquid cultures and did some PCR screening.
Friday August 10th
Gibson assembly backbones were miniprepped, PCR screen was run on all parts.
Week 9
Monday August 13th
Our project is coming along! Last weeks PCR screen yielded a few parts for our cutinase, dehydrogenase, and reductase genes, and we prepared the best colony from each of the respective plates for sequencing at the UC Davis DNA Sequencing Center. We PCR screened the DH5α and MG1655 Cutinase miniprep, Cutinase plus PelB miniprep, Reductase miniprep, Dehydrogenase miniprep, and the Cutinase in pSB1K3 miniprep.
Tuesday August 14th
Isothermal Buffer Reaction for gibson assembly. Rehydrated VF and VR2. We used a spectrometer to look at the competent cells from yesterday, and on the third check ( Competent Cell Protocol ) the culture with 6mL of starter culture was closest to 0/55 absorbance.
Wednesday August 15th
Fast double digest worked! We ligated and transformed the dehydrogenase, reductase, an insert control and two vector controls. The competent cells did not turn out too successfully, as there were a limited number of cells that grew after a generic transformation to test them. Ran gels on J23101, B0034 and R0010+E0240 using fast digest enzymes.
Thursday August 16th
Today was a busy day in the lab. We began constructing a genomic library for the Strain E-15 EG3, through using transposases. After getting the sequences in from Monday, there ended up be a base pair deletion and a point mutation in reductase, and there was a single base pair mutation in the dehydrogenase. The dehydrogenase was not as bad, as we can use site-directed mutagenesis (SDM) to fix the mutation. However, the reductase will need to be redone. Good news was that the SOEing worked, however! Nanodroped gel purification and made liquid cultures of Full Cutinase, Cutinase without His tag, K206000+B0034, B0015, B0034, R0010+E0240 and KAN.
Friday August 17th
Ran a gel of the PCR screen from yesterday. We began the second Tecan experiment – varying amount of ethylene glycol for the Strain E-15 EG3 to see the optimal concentration for growth. We also fragmented the Strain E-15 EG3 genome for genomic sequencing. Finally, we miniprepped B0034 + reductase, and B0034 + dehydrogenase, along with B0015 to build simple construct later on. Ran gel on Thursdays liquid cultures. First attempt on Gibson PCR and PCR screen, resulting gel showed weak bands and wrong lengths.
Saturday August 18th
Ligations and transformations of Full Cutinase + (K20600+B0034), Cutinase without His tag +(K20600+B0034), Full Cutinase + B0034 and KAN + B0015
Week 10
Sunday August 19th
Made minipreps of Full Cutinase, Cutinase with PelB and J23101+R0010+E0240.
Monday August 20th
We redid the PCR screen for the part LacI part (J23101+B0034+C0012) as well as B0034+Kan, K206000+B0034+K936014 (Cutinase construct with inducible promoter), and B0034+K936014 (Cutinase construct with RBS). Using a fast digest, we ligated B0034+dehydrogenase along with B0034+reductase to build the simple construct. Also, we added a stop to the two enzymes in order to prepare and build the complex construct. The resulting nanodrop concentrations were lower than expected, so we decided to scale up the amount of DNA in the digestion reaction. Second attempt at Gibson PCR and PCR screen, bands at wrong lengths. Gel run and gel purification done on J23101+R0010+E0240 and B0034.
Tuesday August 21st
We began sequencing Strain E-15 EG3 today to see the different mutations that occurred from the University of Barcelona laboratory. It was actually easier to make a library than it sounds. We followed the non-PCR protocol, so it involved a lot of incubating and mixing. Literally, the hardest part was the math involved to find dilutions and concentration. In other news, we found out that the J23101 + B0034 + C0012 ligations worked! We are en route to our complex construction of the ethylene glycol side of things. We also met with our advisor, Dr. Facciotti, about the wiki to make some edits and additions. Sequencing for K206000+B0034+Full Cutinase sent in. Gel ran and gel purification done on B0034, KAN+B0015 and Full Cutinase.
Wednesday August 22nd
We miniprepped our way through the morning. The liquid cultures of J23101 + B0034 + C0012 were pink, meaning that there was RFP present. This was strange to us, but we will continue because the RFP is past the insert area, so it may not have gotten cut out. In case this is not the right one, we are also digesting B0034 + C0012 and J23101 so that we can ligate them once more. The RFP should be digested out of the construct, assuming that the SpeI and PstI are working. We are also beginning digestions from one step back with B0034 + C0012 so that it can make the before mentioned pieces. As it turns out, the genome sequencing library did not turn out well. The genome had not been sheared, so we began our work again. This time the reactions all worked well and we will continue our work with this piece. Finally, we PCR screened the B0034 + Reductase + B0034 + Dehydrogenase, B0034 + Reductase + B0015, and B0034 + Dehydrogenase + B0015. They are all present in the colonies we have!
Thursday August 23rd
We glycerol stocked the constructs that we PCR screened yesterday. We are excited for the end of our construction process! Continuing on with the results from the PCR screen pieces, we are ligating the promoters (J23101 and K206000) with B0034 + Reductase + B0034 + Dehydrogenase, R0010 with B0034 + Dehydrogenase + B0015, and B0034 with C0012. We’ll see how the ligations are when we transform them tomorrow. We got our sequencing data back, and it turns out that we have some mutations that we need to fix in the reductase and dehydrogenase enzymes. There was a mutation and a deletion in the reductase, and there was a mutation in the dehydrogenase. However, these will be fixed with some site-directed mutagenesis (SDM). PCR screen on KAN and Full Cutinase performed.
Friday August 24th
Time to transform! We transformed the ligations from yesterday, and we will hopefully see some colonies tomorrow. Later, we analyzed the sheared genomic sequence concentration with quantitative real-time PCR (qPCR). It is really nitpicky to do qPCR because of the sterilization required because we do not want bacteria to be introduced in and have their DNA be replicated through the thermal cycler. We also devised another experiment for directed evolution of the Strain E-15 EG3 to choose for efficient, rapid growth. We will grow the cells in 30mM of ethylene glycol and when they reach 0.2 OD, place 200µL of the cells into 5 mL of 30mM ethylene glycol.  Currently, the doubling time of Strain E-15 EG3 on ethylene glycol is about 6 hours, so we want to decrease this time to see colonies much more rapidly. Digestion performed on C0012, R0010+B0034+E0240, J23101, B0034+KAN+B0015 and B0034+Full Cutinase. First round of SDM performed for Cutinase mutants.
Saturday August 25th
We PCR screened the transformations from yesterday to see if we actually got the parts. It turns out that we successfully got the J23101 + B0034 + Reductase + B0034 + Dehydrogenase, K20600 + B0034 + Reductase + B0034 + Dehydrogenase, and the R0010 + B0034 + Dehydrogenase + B0015. This is excellent because now we are much closer to the final product. The simple construct is not completed, except for some site-directed mutagenesis (SDM) that is required to fix some mutations that were introduced.
Week 11
Sunday August 26th
We miniprepped a few things this fine Sunday morning. Thirty minipreps makes for a fun day!
Monday August 27th
We analyzed our qPCR data and saw that we have a concentration of 5.617 pM of the sheared genomic DNA. We will be sending out our library ASAP for sequencing. We are also doing our first round of SDM today! We want to mutate the reductase to become more efficient under aerobic conditions by changing out two base pairs. The dilutions are tricky, but we got them down by the end of the day! We performed the SDM protocol on the J23101 + B0034 + Reductase + B0034 + Dehydrogenase, K206000 + B0034 + Reductase + B0034 + Dehydrogenase, and B0034 + Reducase + B0015. We also continued on with our Strain E-15 EG3 directed evolution, and we will almost every day. Digestion and gel extraction/purification run on J23101, B0034+KAN+B0015, B0034+Full Cutinase, C0012 and R0010+B0034+E0240. DPN1 added to first SDM PCR products and gel screen run of results.
Tuesday August 28th
The transformation of the SDM reaction with B0034 + Reductase + B0015 had no colonies so we are redoing the transformation. This time, instead of adding 1 µL of the vector, we will add 4 µL, 5 µL, and 6 µL. Hopefully this increases the amount of cells that take up the vector and therefore the mutated plasmid. The other two transformations were perfectly fine. We continued the SDM experiments on the DNA that came from the SDM of yesterday (pre-transformation) as well as B0034 + Dehydrogenase + B0015 to fix the dehydrogenase mutation. We ran the gels of the SDM and we did not see any bands before or after digestion with DpnI, which digests methylated parental DNA. However, we continued on with the transformations. Gel run on R0010+B0034+E0240 and B0034+Full Cutinase. Second round of SDM done. Transformations of first SDM PCR.
Wednesday August 29th
It turns out that we should not have done the second round of SDM on the DNA straight from the previous SDM reaction. The plasmid DNA must be methylated for the reaction to work, and the only way the DNA gets methylated is in the actual cells. Our new plan is to wait for the sequencing data to come back after each round of SDM to see if we have the correct sequences. Then we will take the correct ones and do a subsequent round of SDM. We redid the SDM with the B0034 + Reductase + B0015 and we finally finished the transformation at 9PM! We transformed with MG1655. Ran screen on second SDM run with and without DPN1.
Thursday August 30th
We started the process for making some DH5 competent cells so that we have a ready stock on hand. We also revised the wiki and added some more information to the pages. We ran a gel on the PCR screen of the SDM reactions to fix the aerobic mutations on J23101 + B0034 + Reductase + B0034 + Dehydrogenase and K206000 + B0034 + Reductase + B0034 + Dehydrogenase and B0034 + Reductase + B0015. We only had bands show up correctly for the J23101 + B0034 + Reductase + B0034 + Dehydrogenase and K206000 + B0034 + Reductase + B0034 + Dehydrogenase, so we prepared liquid cultures for four colonies each in preparation for miniprepping and sequencing. Another PCR Screen was set up for R0010 + B0034 + Dehydrogenase + B0015, and these were the correct band sizes. Ran third round of SDM. Varied amounts of template, 50ng/ul and 20ng/ul. Also ran another segment SDM PCR on Y127A and T96A mutants, 50ng/ul. Conducted a PCR screen on parts J23101+B0034+KAN+B0015 and J23101+B0034+Full Cutinase. Added DPN1 to third SDM and ran screening gel. Third SDM products transformed.
Friday August 31th
We met with our advisers today and learned that we should strategically approach the SDM. Rather than do the previous sequential protocol that we were following, we should run multiple primers with the template plasmid DNA in the same reaction. Even though this is riskier, it would save a lot of time if it worked. In addition to the multiple sets of primers, we are devising a plan to SOE together the different segments. We will PCR out the fragments using the SDM primers. Then the SDM primers will be able to complement each other on different strands and complete the whole strand, incorporating the mutations in. We also made competent cells for DH5α, and we will test exactly how competent they are in the days ahead.
Saturday September 1st
Minipreps on original synthesized cutinase cultures. Screening gel on segment SDM PCR run. Transformations on third SDM checked, failed.
Week 12
Sunday September 2nd
Plates used in third SDM transformation tested.
Monday September 3rd
We did a SDM reaction on J23101 + B0034 + Reductase + B0034 + Dehydrogenase and K206000 + B0034 + Reductase + B0034 + Dehydrogenase, using multiple primer sets, including aerobic mutations (aer), dehydrogenase mutation (fix_deh), and the deletion in reductase (del_red). For B0034 + Reductase + B0015, we did a multiple SDM reaction using primers to fix all of the reductase mutations (aer), (del_red), and (mut_red). Re-did segment SDM on unsuccessful middle piece. Plate test showed plates were ok. DH5 alpha cells tested in transformation to check their ability. Transformed on DH5 alpha and MG6155. Liquid cultured J23101+B0034+KAN+B0015 and J23101+B0034+Full Cutinase.
Tuesday September 4th
Today, we began our SDM reactions by SOEing on the B0034 + Reductase + B0015, J23101 + B0034 + Reductase + B0034 + Dehydrogenase, and K206000 + B0034 + Reductase + B0034 + Dehydrogenase constructs. We are following the same protocol as before -- PCR then gel extraction and purification to select for the correct fragment lengths. Instead of using Taq and Pfu, we used only Pfu in a 1:1 substitution. We performed PCR screens on the SDM (multiple sets of primers) from yesterday of the J23101/K206000 + B0034 + Reductase + B0034 + Dehydrogenase, but we saw no bands. Therefore, we did another set of 12 PCR screens to see if there are any properly sized inserts in any of the colonies. We also did a multiple set SDM of the B0034 + Reductase + B0015 and transformed them. We will see how they turn out tomorrow. PCR screen on few successful transformations of mutants. Mutants F125Y, V212 and Y127A showed successful PCR screens. Performed liquid cultures on them. Ran digests on J23101 and B0034+KAN+B0015. Reattempted gibson with new primers.
Wednesday September 5th
We miniprepped some R0010 + B0034 + Dehydrogenase + B0015 constructs that had the dehydrogenase mutation SDM primers. We will sequence these once our primers are received! Because the previous SOEing PCR did not have bands for every segment, we repeated the reactions in hopes that there are not any pipetting errors. We also used less template DNA -- 50 ng. With the previous SOEing, we tried to SOE together the smaller pieces that have the complementary pieces with. We ran all of the SOEing pieces on a gel, and we saw the correct sized bands for select reactions, but no bands for others. We also PCR screened the multiple set SDMs that we did on the J23101 + B0034 + Reductase + B0034 + Dehydrogenase and the K206000 version of the construct. There were no bands at all and the plates looked strange because of the different morphologies of the colonies. We are also sending some things off for sequencing -- the J23101 + B0034 + Reductase + B0034 + Dehydrogenase to check for the aerobic mutations.
Thursday September 6th
We took the OD reading of the Strain E-15 EG3 and MG1655 and halted the culture at OD 0.2. The cells were taken to Arcadia for mutagenesis (EMS) and grown in LB overnight. A PCR screen was done for B0034+ Reductase+ B15 (aerobic), which showed successful band sizes and liquid cultured overnight. A gel was also run for the SOEing PCR, resulting in only several visible bands. The SOEing fragments were gel extracted but not ligated together because there were no complementary pieces. A new multiple set SDM was set up for J23101 + B0034 + Reductase + B0034 + Dehydrogenase, K206000 + B0034 + Reductase + B0034 + Dehydrogenase, and B0034 + Reductase + B0015 using a longer elongation time (10 minutes) and was transformed. Gibson PCR screen results had sizes expected. Gibson reaction itself was run afterward. Second Gibson PCR run using lower concentrations. Gibson reaction results showed no combination of pieces, unsuccessful. PCR screen on B0034+KAN+B0015+J23101 run and liquid cultures taken from successful screens.
Friday September 7th
We are trying the SOEing reactions again, in hopes that we get more bands. We went back to our stocks to make primer dilutions so that everything was at 10 uM, and again, we used 50 ng of template DNA. We had to re-transform the multiple set SDM reactions that we had from yesterday because they were accidentally vortexed and destroyed the competent cells. From yesterday, we are taking the mutants that grew overnight and plated 200 and 400 uL on ethylene glycol plates. We got our sequencing data back for the J23101 + B0034 + Reductase + B0034 + Dehydrogenase for the aerobic mutations. Minipreps done on B0034+KAN+B0015+J23101 and B0034. PCR screen done on Gibson PCR of all gibson backbones with varying concentrations. No results seen on gel. Fourth round of SDM ran using longer extension time.
Saturday September 7th
A PCR screen of the multi-set SDM of B0034 + Reductase + B0015 and K206000 + B0034 + Reductase + B0034 + Dehydrogenase was done (which the plates looked normal because there were only around 15 - 20 colonies). Liquid cultures were prepared for J23101 + B0034 + Reductase + B0034 + Dehydrogenase (aer) to prepare for a glycerol stock and mini-prep sample. The EMS plates showed no visible colonies and were left in the 37 °C incubator.
Week 13
Sunday September 9th
The liquid cultures of J23101 + B0034 + Reductase + B0034 + Dehydrogenase (aer) were miniprepped and glycerol stocked. The gel of the multi-set SDM showed promising band sizes and therefore more liquid cultures were made for the multi-set SDM for J23101 + B0034 + Reductase + B0034 + Dehydrogenase,  K206000 + B0034 + Reductase + B0034 + Dehydrogenase, and B0034 + Reductase + B0015.
Monday September 10th
We received our sample data back from sequencing for the J23101 + B0034 + Reductase + B0034 + Dehydrogenase on Friday and we did another round of SDM to fix the deletion that we also had. We used 50 ng of template again and followed the same protocol as before. We are also trying out a new method for SOEing, where we take a tiny amount of the gel via a P200 and then adding that to a PCR mix to amplify it. We also sent a million samples for sequencing. The different samples are the J23101 + B0034 + Reductase + B0034 + Dehydrogenase, K206000 + B0034 + Reductase + B0034 + Dehydrogenase, B0034 + Reductase + B0015, and R0010 + B0034 + Dehydrogenase + B0015. Hopefully these turn out well! DPN1 added to fourth SDM run and gel screen performed. Transformations performed. GFP full Lac Construct, J23101+B0034+Full Cutinase, V212A, F125Y, and Y127A sent for sequencing.
Tuesday September 11th
Today was full of analyzing sequences! The results were a mixed bag. Some constructs had one or two fixes, when we actually needed four to be fixed. We glycerol stocked more of the things that we sent to sequencing so that we had a culture that we could be more confident in, rather than on the replication plate. We miniprepped the things that we glycerol stocked. We also PCR screened the J23101 + B0034 + Reductase + B0034 + Dehydrogenase construct that was sequenced for the aerobic mutation and we added the deletion fixes. We will run the gel of it tomorrow. We also looked at all of our plates from the mutagenesis and marked which had growth, lawns, or no growth. Mutant sequences showed no mutations. Performed another transformation on fourth SDM PCRs using varying concentrations, spins some down, using various competent cells and using different resistance plates.
Wednesday September 12th
More sequences were received of the multi-set SDM reactions for J23101 + B0034 + Reductase + B0034 + Dehydrogenase, K206000 + B0034 + Reductase + B0034 + Dehydrogenase , and B0034 + Reductase + B0015. These yielded only one set of mutation changes and therefore another SDM was set up for these samples to introduce one additional change. Our goal changed to mainly fix the reductase deletion and the dehydrogenase mutation. The additional change for the aerobic reductase mutation was set as a secondary priority after building the construct. The  SDM reactions were then digested and transformed, and will be prepared for further sequencing tomorrow. Another genomic library of the Strain E-15 EG3 was prepared in hopes to send off for genomic sequencing. The process was halted after the purification step and will be continued with the gel purification tomorrow. Growth seen on some plates. Screen run on successful transformation results. Tranformations on T96A and Y127A done.
Thursday September 13th
We miniprepped the J23101 + B0034 + Reductase + B0034 + Dehydrogenase with the aerobic mutations that we did a SDM reaction to fix the deletion. We also analyzed the sequences that we just received this morning. We began our replating of the EMS mutations. We scraped the lawns and then resuspended them in 500 uL of the buffer A. Then we took the OD and diluted it down twice to get a solution that would have 150 cells per 200 uL. We also transformed in the J23101 + B0034 + Reductase + B0034 + Dehydrogenase, K206000 + B0034 + Reductase + B0034 + Dehydrogenase, and R0010 + B0034 + Dehydrogenase + B0015. We will use these for testing in the Tecan later on. We sent more things off for sequencing. One of our members, Christine Olson, was so dedicated and stayed until 11 PM to do over one hundred PCR screens. What a dedicated iGEMer. From our genomic sequencing, we ran the qPCR to quantify our concentration. Successful screened mutants at this point were T96A, D98T, S101A, V212A and F125Y. All sent in for sequencing.
Friday September 14th
We analyzed the qPCR results of the Strain E-15 EG3 library preparation for trial #3, and the genomic library yielded 220pM of DNA. This yield was not high enough to continue with genomic sequencing, so another library preparation needs to be set up.  A gel was run on the PCR screens done yesterday, but this did not give any bands. So, another PCR screen was set up directly from the previous rep plate. The EMS dilutions made yesterday were accidentally plated on LB carb plates, and not on ethylene glycol, so more dilutions were prepared today. A digestion was set up to ligate J23101 and K206000 together with one enzyme, dehydrogenase. However, it was digested at the wrong cut sites, and so the parts were redigested. The parts were gel purified afterwards to continue on with ligation on Monday. Gel screens run on T96A and Y127A successful transformed colonies. Succesful screens sent off for sequencing.
Saturday September 15th
We did a PCR screen of the things that we transformed the previous night. From yesterday’s PCR screens, we ran the gel.
Week 14
Monday September 17th
We digested B0034 + Dehydrogenase + B0015 to ligate it together with the J23101 and K206000. We want to do this so that we can see if just the expression of dehydrogenase will increase the success rate of the MG1655 and Strain E-15 EG3 living. We had to re-digest the pieces because the nanodrop of the gel purification was erratic. However, when we went to nanodrop the second round of digestions, the previous gel purifications were at a higher concentration than that of the second digestion. We ligated the pieces overnight at 16C. The concentration of the library for the genomic sequencing was too low, so we had to redo the preparation. We ran the qPCR overnight to see how much there is in the new prep. We also took the liquid cultures from yesterday and put them into glycerol stocks and minipreps. Successful sequence results were seen for T96A, D98T, S101A, F125Y and Y127A. pNPB assay performed on control (cells with no cutinase), J23101 and inducible construct all at varying cell concentrations.
On the cutinase front, we started a new round of cultures from which to take samples for western blots. First, we inoculated two 5mL cultures cultures one with pBad regulated cutinase in MG1655 and the other with only the antibiotic resistance (to be used as a control). When the cultures reached an OD of 0.5, we then used this inoculate two 100 mL cultures. Once these reached an OD 600 of 0.8, we divided them into four 50 mL cultures and induced two of them. At this point we had four 50 mL cultures at an OD of 0.8, an induced and uninduced pBad promoted cutinase and an induced and uninduced negative control.
From these cultures, we took 1 mL samples at several different time points: 0, 0.5, 1, 1.75, and 3.5 hours. Tomorrow we will prepare these samples to run on an sds page gel and western blot.
Tuesday September 18th
The minipreps from yesterday were nanodropped and sent off for sequencing. We also analyzed the sequence data that we sent off last week. We also planned out the different experiments that we will run in the Tecan as soon as tomorrow. Finally, we transformed the ligations that we had from overnight. We also made some solutions for the Davis high biology class. We made some LB media, some plates for halophiles, and some halophile liquid media.
Today, we ran another gel of the PCR screen and liquid cultured a million things for miniprepping and sequencing later on.
With the cutinase and control cultures from yesterday, we first spun down the 1 mL samples, took off, and saved the media. We then washed the cell pellet and resuspended and lysed the cells in BPER. After vortexing this mixture, we centrifuged and separated the soluble and insoluble portions of the cell lysate. We plan on running all of these samples on an sds page gel so that we can then do a western blot to determine whether or not the cutinase protein is making it into the media of the cell. Today we started by running the media samples on an SDS-PAGE gel and transferring onto a pvdf membrane. We have been more careful about washing the equipment and have started using much less ladder to reduce background noise. We have left it overnight in blocking solution and will probe and wash it tomorrow.
Wednesday September 19th
Today, we analyzed more sequences that we sent out yesterday. We also had to redo our digestions and ligations that we did a few days ago because it turns out that the purification was not done as well as it could have. Therefore we redid it in hopes of a better result. Lastly, we began a Tecan experiment where we test the optimal amount of arabinose for the induction of the K206000 + B0034 + Reductase + B0034 + Dehydrogenase in MG1655 and Strain E-15 EG3. We did a round of SDM on B0034 + Reductase + B0015 to get rid of the deletion in reductase. pNPB assay results looked promising. Rerun taking OD, using antibiotic LB, comparing induced/uninduced cells, comparing mutants and taking supernatant.
Today we finished with the western blot for our media samples, but unfortunately, there was a lot of background in the picture. We are attempting to block longer and wash our equipment better to reduce the background.
Thursday September 20th
Today we reattempted to take our media samples through the western blot process. We ran them on an SDS-PAGE gel and plan to complete the process tomorrow.
We finished the DpnI digest of the SDM from yesterday, and also digested fixed parts from previous rounds of SDM. We digested K206000 + B0034 + Reductase + B0034 + Dehydrogenase in both an aerobically-adapted and anaerobically-adapted form, J23101 + B0034 + Reductase + B0034 + Dehydrogenase in an anaerobic form, pelB + Cutinase + His, and K206000 + B0034 + pelB + Cutinase + His at Eco and Pst. We did this specific digestion so that we could put these parts (which are currently in a pSB1A3 backbone into the pSB1C3 backbone which is the requirement for BioBrick parts for this year. We ligated these parts together with the new backbone, and transformed them into DH5α. We transformed K206000 + B0034 + Dehydrogenase + B0015 and J23101 + B0034 + Dehydrogenase + B0015 into DH5α, MG1655, and Strain E-15 EG3. We continued the Tecan experiment from yesterday. We went through all of the EMS replates to see the amounts of colonies and documented how many there were. We also transformed the J23101/K206000 + B0034 + Dehydrogenase + B0015 constructs into MG1655, DH5α, and Strain E-15 EG3. Transformations of mutants into MG6155 done.
Friday September 21st
The western blot had too much background once again. We have started the process again with more samples and have recruited extra help from our faculty advisor to determine where the process could be going wrong.
The transformations were successful from yesterday and they had a good amount of colonies. We PCR screened the transformations. We began the liquid culture for the EMS tecan runs. We are beginning with the Strain E-15 EG3 with 0 minutes with EMS. We also transformed the some ligations that we did with the simple constructs and cutinase (separately) into pSB1C3. Mutant transformations screened and liquid cultures taken of successful screens. Another pNPB assay run using only 5 and 10ul of cells between control, induced, uninduced, J23101 and a second control.
Saturday September 22nd
Once again, there is too much background the take good data from our western blots. pNPB assay run using only blanks (no cells) and J23101 only in order to check for reasons of the strangely high degradation seen in blanks from last assay results. This assay showed normal blanks suggesting a one time screw up in the previous assay. Thus, another assay performed using 5 successful mutants, wild type control,J23101 and J23101 supernantant.
We liquid cultured the SDM of the B0034 + Reductase + B0015 to fix the deletion. We also ran a PCR screen on the pSB1C3 and they looked good. We also documented the colonies from the EMS plates and there were enough for the Tecan experiments. We set them aside in the 4 degree room to be used for the Tecan cultures later.
Week 15
Sunday September 23rd
We took the OD of the Strain E-15 EG3 that we liquid cultured on the 21st. There was a spread, but none had reached 0.2 OD, so we let them grow another night. Digestion, Ligation and Transformation done on mutants to change backbone. Another pNPB assay run using cells and supernatant from controls and J23101.
Monday September 24th
We took the OD of each of the Strain E-15 EG3 cultures again and they ranged from 0.08 to 1.5. We washed them in mineral solution and then took the OD again. Then we set up the Tecan to measure which of them have the best growth. We want to be able to see the variation in the Strain E-15 EG3 without any EMS and how the EMS changes the variation in the population. This plate serves as a control for the comparisons. We are also sequencing the B0034 + Reductase + B0015 to see if the SDM worked for the deletion fix. We also began a crude experiment to see if the K206000 + B0034 + Reductase + B0034 + Dehydrogenase versus the K206000 + B0034 + Dehydrogenase + B0015 versus the strain by itself. We want to be able to see if there is growth with one piece or only if there is both. We also did it with J23101 instead of the K206000. These tests were done in the MG1655, Strain E-15 EG3, and DH5α. We looked at the Tecan results that had just finished its run. We concluded that it was not done for a long enough time because of its growth curves. It had not made significant changes. We put it back in to grow longer in the Tecan.
We made liquid cultures for multiple parts including PSB1C3, PelB+Cutinase+PSB1A3, Cutinase+Histag+PSB1A3, Cutinase +PSB1A3.
Tuesday  September 25th
We did a lot of basic chores (carb plates and chloro plates made and glassware autoclaved). We also set up multiple liquid cultures of the remaining EMS colonies. These amounted to about 180 liquid cultures done in 96 well plates. We also liquid cultured the same constructs as yesterday, except this time, we added in 2 µM and 10 µM of arabinose in the K206000 constructs to see if there is a difference between low and high induction. We will use these in the Tecan experiments as well, later on.
We then minipreped the liquid cultures PSB1C3, PelB+Cutinase+PSB1A3, Cutinase+Histag+PSB1A3, Cutinase +PSB1A3, measured the concentrations to do a digest, and digested them (PSB1C3, (and the inserts) PelB+Cutinase, Cutinase+Histag, and Cutinase, and Bba_K936020). After the digestion, we ran an extraction gel to extract our parts we want to put together. Afterwards we purified, and nanodropped them to get the concentrations. With these concentrations we did a ligation and transformation of each of the inserts with the backbone PSB1C3.
Wednesday  September 26th
We are going to set up more liquid cultures and design experiments for the Tecan for more data! We are also miniprepping the transformations of the pSB1C3 ligations that we had from a few days ago. We will sequence them and hopefully everything is good! We are one step closer to sending them in to the parts registry!
We did a PCR screen on two of Tuesday’s transformations (Pelb+Cut+PSB1C3 and Cut+His+PSB1C3); the other two had not grown colonies. We also made liquid cultures of them as well.
Thursday September 27th
We ran a gel on Wednesday’s PCR, (Pelb+Cut+PSB1C3 and Cut+His+PSB1C3) they turned out well! So then we minipreped the liquid cultures of them and had them sequenced. Hopefully, they will be good to send to the registry! Cut+PSB1C3 and Bba_K936020+PSB1C3 plates had grown colonies and now we PCR screened them, and made a liquid culture as well.
Friday September 28th
We ran a gel on Thursday’s PCR (Cut+PSB1C3 and Bba_K936020+PSB1C3), they also turned out well! Then we took the liquid cultures and minipreped them (or purified them), measured their concentrations with the nanodrop, and then had them sequenced.
Week 16
Monday October 1st
With classes started, it is getting hard for us to come in and keep track of what is going on. We are working on the wiki as well today, as there is much information that must go on the website before the wiki freeze. We also had some of our last constructs sequenced, analyzed and prepared to send to the registry.
Tuesday October 2nd
We spent a large amount of time developing the wiki further, as the wiki freeze is approaching. We also took PET samples that were incubated with our strain, and weighed out the masses. Our data shows generally expected results, although we still need to furthermore fully characterize the results in order to effectively say our enzyme works. We also finalized our shipment and sent in our parts to the registry!
Wednesday October 3rd
Today centered around finalizing the wiki for the wiki freeze at 9:00. Hope everything goes well!
Week 17 and onward
Since the regional jamboree, we have been hard at work to gather more data and better characterize our parts.
We have conducted more experiments to better characterize the expression, secretion, and activity of the cutinase enzyme. There is still more characterization to do and we are still working to purify the expressed gene.
We are also further characterizing the E-15 EG3 strain and have submitted it for sequencing.
The most recent data and results can be found on our data pages.
Home
Welcome
Tweets
Sponsors
Criteria
Team
Who we are
Students
Advisors
Project
Project Overview
Module Engineering
Protein Engineering
Chassis Engineering
- Directed Evolution
- Rational Engineering
Critera
Safety
Safety
Notebook
Notebook
Protocols
Gallery
Critera
Data
Cutinase Activity
Ethylene Glycol
Modeling
Parts
Attribution
Attribution
iGEM
Main iGEM
Criteria
Human Practices
Retrieved from " https://2012.igem.org/Team:UC_Davis "
Retrieved from " http://2012.igem.org/Team:UC_Davis/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
