"
Page
Discussion
View source
History
teams
Log in
 
Team:UC Davis/Project/Catalyst
From 2012.igem.org
Team:UC Davis - 2012.igem.org
teams
Page
Discussion
Edit
History
Move
Watch
Log in
iGEM
Main iGEM
Criteria
Human Practices
Attributions
Data
Cutinase Activity
Ethylene Glycol
Modeling
Parts
Notebook
Notebook
Protocols
Gallery
Safety
Project
Project Overview
Module Engineering
Protein Engineering
Chassis Engineering
Background
Directed Evolution
Rational Engineering
Team
Home
UCDavis iGEM Tweets
Our Sponsors
Modules
We have created several modules and biobrick parts for the degradation and utilization of PET. The PET degradation pathway can be seen below. The parts that we focused on for our project are marked with an asterisk.
LC-Cutinase and Initial PET Degradation
When looking for a catalyst capable of breaking down PET, we came across a paper that conducted a metagenomic analysis of leaf-branch compost, identified a cutinase homolog, and demonstrated its PET-degrading activity [1]. It was found that this catalyst broke PET down into two by-products: ethylene glycol and terepthalic acid (TPA). We chose to use this leaf-branch compost cutinase (LC-Cutinase) in our module as it was reported to have a high activity and showed potential for use in PET degradation and surface modification.
Design
We had the LC-Cutinase gene synthesized with a pelB leader sequence and a 6-his tag (the full construct being labeled as Bba_K936014 ). The pelB leader sequence, when translated, is a protein tag that directs the attached protein to the periplasmic space and is removed by a pelB peptidase once in the periplasm. We used this tag as a method of protein secretion as it had been reported that once LC-cutinase was brought to the membrane, a leakage occurs that helps the catalyst to be secreted into the extracellular matrix [1]. This secretion is desirable for both the purification of the enzyme and for future applications where the bacterial culture would be directly incubated with the PET. The his-tag was included at the end of the sequence for additional purification purposes and to conduct experiments that would determine where the protein goes after production (i.e. extracellular or intracellular space). We have two different constructs expressing this part: one with a constitutive promoter ( Bba_J23101 ) and one with the inducible pBad promoter ( Bba_K206000 ) allowing for induction of the cutinase gene.
Experiments
We have designed and conducted three different experiments to determine (1) if the pelB tag is working to secrete the catalyst, (2) if the LC-cutinase protein is displaying its expected esterase behavior, and (3) if the protein is capable of breaking down PET from various sources.
First, to determine where pelB-cutinase was being expressed, we cultured the pelB-cutinase-6His version of our construct.  Beginning during exponential growth phase, we separated cells from the supernatant media at different time points, and ran samples on a western blot probing for the 6x-His tag.  The details of this experiment can be found on the protocols page under "Cutinase Expression and Western Blot". The results show that pelB is likely working as a secretion mechanism for cutinase. We are currently conducting more westerns to better characterize the expression and secretion of the gene.
Second, we attempted to see if the Cutinase gene was exhibiting any esterase activity. This was done using a p-nitrophenol butyrate (pNPB) assay in hopes of evaluating and quantifying the esterase activity. pNPB is a monomer similar to the plastic PET that is cut by esterases in the same way. The benefit of the pNPB however is that as it is cut, the solution's absorbance at 405nm increases. This can be easily measured using a plate reader. The protocol for this assay was taken from literature and can be found on our protocols page under "pNPB Assay"[4].
Although this assay is usually done with purified esterases, we have had difficulties up to this point purifying cutinase. We did test, however, the activity of whole cells of our induced, uninduced, constitutive, and mutant constructs (our data is posted here ). Though these results gave us a general outline of the activity of each construct and of cutinase itself it’s important to note that these results only show general esterase activity and cannot necessarily be expected to parallel results with PET itself.
Finally, to test our cutinase's activity with PET we set up a series of experiments in which we incubated the enzyme with various samples of PET from different plastic suppliers. Again, as we were unable to purify the enzyme at this time we simply incubated the plastic with cell cultures after washing the PET according to industry standards. We measured the degradation by dry weight but were unable to effectively measure degradation over time. The results from these experiments and the inability to purify this protein suggest to us that there is some biological activity that we do not quite understand at this time. We plan on further investigating it in the upcoming weeks.
Ethylene Glycol Modules
Ceratin strains of E. coli can grow with ethylene glycol as a sole carbon source by expressing two enzymes, glycolaldehyde reductase and glycolaldehyde dehydrogenase (2). Ethylene glycol is toxic for vertebrates because it can cause kidney damage, but E. coli strains such as MG1655 can use it as an energy source (2).
Glycolaldehyde reductase (DL-1,2-propanediol oxidoreductase) is normally an anaerobic protein that reduces L-lactaldehyde into L-1,2-propanediol, an excreted product. In addition, a mutant from the paper aforementioned was able to live on L-1,2-propanediol as a sole carbon source. Not only are there mutants that live on the L-1,2-propanediol, but there are also mutants selected for growth on ethylene glycol.
Unlike reductase, glycolaldehyde dehydrogenase is a protein that already favors aerobic conditions. This dehydrogenase oxidizes glycolaldehyde further to glycolate, and the glycolate will be used further downstream in metabolism to provide the carbon source for the E. coli to live.
Experiments
We mutated one version of reductase to work optimally under aerobic conditions, rather than under anaerobic conditions. We got the idea to mutate the enzyme from the scientific literature, and we mutated reductase by using site directed mutagenesis (the protocol we used can be found here ) [3]. We are currently working on comparing growth rates between these different enzyme forms by running cultures in a Tecan.
We expressed these enzymes polycistronically with the Bba_J23101 (a constitutive promoter) and Bba_K206000 (pBad- an arabinose-inducible promoter) promoters to see the difference in overproduction of the enzymes. We wanted to use an inducible promoter to finely control how much enzyme is expressed, and our goal is to compare this controlled enzymatic production against the constitutive promoter. We are also testing these construct in 96-well Tecan plates as well. Our modular efforts in plasmids will eventually be applied toward a rational strain engineering approach, where we manipulate the MG1655 chromosome to optimize the degradation of ethylene glycol.
References
1. S. Sulaiman, S. Yamato, E. Kanaya, J. Kim, Y. Koga, K. Takano, S. Kanaya. "Isolation of a Novel Cutinase Homolog with Polyethylene Terephthalate-Degrading Activity from Leaf-Branch Compost by Using a Metagenomic Approach." Applied and Environment Microbiology, vol. 78 no. 5, pp. 1556-1562, March 2012.
2. Boronat, Albert, Caballero, Estrella, and Juan Aguilar. “Experimental Evolution of a Metabolic Pathway for Ethylene Glycol Utilization by Escherichia coli.” Journal of Bacteriology, Vol. 153 No. 1, pp. 134-139, January 1983.
3. Lu, Zhe, Elisa Cabiscol, Nuria Obradors, Jordi Tamarit, Joaquim Ros, Juan Aguilar, and E.C.C. Lin. "Evolution of an Escherichia coli Protein with Increased Resistance to Oxidative Stress." Journal of Biological Chemistry. 273.14 (1998): n. page. Print.
4. Ö. Faiz et al. Determination and characterization of thermostable esterolytic activity from a novel thermophilic bacterium Anoxybacillus gonensis J. Biochem. Mol. Biol., 40 (2007), pp. 588–594.
Home
Welcome
Tweets
Sponsors
Criteria
Team
Who we are
Students
Advisors
Project
Project Overview
Module Engineering
Protein Engineering
Chassis Engineering
- Directed Evolution
- Rational Engineering
Critera
Safety
Safety
Notebook
Notebook
Protocols
Gallery
Critera
Data
Cutinase Activity
Ethylene Glycol
Modeling
Parts
Attribution
Attribution
iGEM
Main iGEM
Criteria
Human Practices
Retrieved from " https://2012.igem.org/Team:UC_Davis "
Retrieved from " http://2012.igem.org/Team:UC_Davis/Project/Catalyst "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
