"
Page
Discussion
View source
History
teams
Log in
 
Team:UC Davis/Project/Our Strain
From 2012.igem.org
Team:UC Davis - 2012.igem.org
teams
Page
Discussion
Edit
History
Move
Watch
Log in
iGEM
Main iGEM
Criteria
Human Practices
Attributions
Data
Cutinase Activity
Ethylene Glycol
Modeling
Parts
Notebook
Notebook
Protocols
Gallery
Safety
Project
Project Overview
Module Engineering
Protein Engineering
Chassis Engineering
Background
Directed Evolution
Rational Engineering
Team
Home
UCDavis iGEM Tweets
Our Sponsors
Rational Engineering
Rational engineering targets certain chromosomal regions for manipulation, which enables the organism to express a certain gene without a plasmid. It involves extrapolating knowledge from plasmid-based genetic experiments and applying them with electroporation or similar methods. The chromosomal expression of genes improves the efficiency of the production of the enzymes because there is no longer a cellular agenda and a human agenda for the cell [1].
What we're doing (in plasmids)
Our goal was to reproduce this ability to utilize ethylene glycol with plasmids expressed in two ordinary E. coli strains: DH5α (a strain that cannot degrade ethylene glycol) and MG1655 (degrades ethylene glycol in small quantities). We devised two approaches to achieve this design using pSB1A3 as the backbone. Our first procedure involves a polycistronic system, with two genes (reductase and dehydrogenase) under the control of one promoter. The reductase comes in two forms - aerobic and anaerobic [2], accomplished from site-directed mutagenesis (SDM). We will have two variants of the plasmid, one with an inducible pBAD promoter and one with the constitutive J23101 promoter.
Our second approach separates the genes, allowing us to see if the genes can be expressed more efficiently when they are under the control of one promoter each. The separation also permits us to induce one promoter and therefore express one gene at a time. With the genes expressed independently, we are able to control the production of each enzyme and ensure equal amounts are expressed. The glycolaldehyde reductase enzyme will be under the control of the pBAD promoter; the glycolaldehyde dehydrogenase enzyme will be under the control of the pLAC promoter. Because we are employing the lac promoter, we must have the lacI operon to act as the repressor. The diagrams below depict the cassette orientation within each plasmid.
Tecan Experiments
We wanted to test if ethylene glycol is toxic to E. coli by mixing it in various concentrations of ethylene glycol into LB media. The set up of the Tecan experiment is pictured below.
For growth curve, click here
The Tecan experiments with MG1655 and DH5α show us that the ethylene glycol does not hinder the growth and development of the strains, as long as it is mixed with LB media. The growth curves all had the same shape, independent of the amount of ethylene glycol in solution. We chose a broad, nearly exponential range of ethylene glycol concentrations to allow a broad range to test the toxicity. We attempted to find the lower limit of toxicity due to a saturation of ethylene glycol. However, we had not reached it. In our engineered strain, we will not expect to see a concentration of ethylene glycol above 150mM, so we can expect our strain to be able to live in an environment with a concentration as high as that.
After seeing that ethylene glycol does not pose a threat to MG1655 and DH5α, we subjected the Strain E-15 EG3 to the same broad range of ethylene glycol. We sought out to find the most efficient concentration of ethylene glycol for this strain, as a guideline for the efficient concentration of EG for our engineered strain. While analyzing the data, we realized that we have to define efficiency more clearly. Efficiency can mean faster growth on low amounts of ethylene glycol or it could mean a higher optical density after a certain amount of time, where it reaches the stationary phase. We saw that once the ethylene glycol concentration reaches a certain threshold (49.34 mM), the growth curves are all the same in terms of time when the stationary phase has been reached. We saw that some of the E. coli were efficient at low concentrations, making us focus on the fast growth efficiency at lower concentrations of EG because the LC-cutinase will not degrade quickly enough to produce 49.34 mM in a cell’s solution. Now, we have an ongoing experiment where we re-passage cells between ethylene glycol media at 30 mM. We discuss this in more detail in our directed evolution section.
For the ethylene glycol section of our project, we have devised multiple Tecan experiments to test various aspects of our constructs as well as directed evolution. Our first run was for an assay on the optimal arabinose concentration for the K206000 + B0034 + Reductase + B0034 + Dehydrogenase construct in MG1655 and Strain E-15 EG3. From the graph provided on the parts registry, we ran a concentration range of 0 µM to 12 µM of arabinose to test the entire spectrum of the effect of the inducer on our inducible construct. Our expected results are that the higher the concentration of arabinose, the higher the expression of the enzymes.
We also are currently setting up several Tecan runs with the following constructs:
DH5α:
K206000 + B0034 + Reductase (aerobic)+ B0034 + Dehydrogenase
K206000 + B0034 + Reductase (anaerobic) + B0034 + Dehydrogenase
J23101 + B0034 + Reductase (anaerobic) + B0034 + Dehydrogenase
K206000 + B0034 + Dehydrogenase + B0015
J23101 + B0034 + Dehydrogenase + B0015
MG1655:
K206000 + B0034 + Reductase (anaerobic) + B0034 + Dehydrogenase
J23101 + B0034 + Reductase (anaerobic) + B0034 + Dehydrogenase
K206000 + B0034 + Dehydrogenase + B0015
J23101 + B0034 + Dehydrogenase + B0015
Strain E-15 EG3:
K206000 + B0034 + Reductase (anaerobic) + B0034 + Dehydrogenase
J23101 + B0034 + Reductase (anaerobic) + B0034 + Dehydrogenase
K206000 + B0034 + Dehydrogenase + B0015
J23101 + B0034 + Dehydrogenase + B0015
Our goal in this experiment is to see if the cells will live with just one enzyme (Dehydrogenase) or if they require both enzymes to be expressed. We also want to see the relative efficiency of the K206000 versus the J23101 analogous constructs. We are hoping to be able to present our results from this section at Regionals!
References
1. Danchin, Antoine. "Scaling up synthetic biology: Do not forget the chassis." Elsevier. (2011): December. Print.
2. Lu, Zhe, Elisa Cabiscol, Nuria Obradors, Jordi Tamarit, Joaquim Ros, Juan Aguilar, and E.C.C. Lin. "Evolution of an Escherichia coli Protein with Increased Resistance to Oxidative Stress." Journal of Biological Chemistry. 273.14 (1998): 8308-8316. Print.
Home
Welcome
Tweets
Sponsors
Criteria
Team
Who we are
Students
Advisors
Project
Project Overview
Module Engineering
Protein Engineering
Chassis Engineering
- Directed Evolution
- Rational Engineering
Critera
Safety
Safety
Notebook
Notebook
Protocols
Gallery
Critera
Data
Cutinase Activity
Ethylene Glycol
Modeling
Parts
Attribution
Attribution
iGEM
Main iGEM
Criteria
Human Practices
Retrieved from " https://2012.igem.org/Team:UC_Davis "
Retrieved from " http://2012.igem.org/Team:UC_Davis/Project/Our_Strain "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
