<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Gifu_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Gifu/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Gifu/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Gifu/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Gifu/Modeling&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Gifu/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Gifu/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="tab"><A class="tab-gray" href="https://2014.igem.org/Team:Gifu/Parts">Parts</A><A class="tab-white" href="https://2014.igem.org/Team:Gifu/Modeling">Modeling</A></DIV><DIV id="main"><H1 id="theme">Modeling</H1><H1 class="theme3">Summary</H1><P>
We examined the efficiency of circularization of mRNA in <I>E. coli</I>. The mRNA synthesized by transcription of induced plasmid may be considered that there are mainly 3 shape patterns in the process of circularization (fig.1). </P><P><B>Figure. 1  reaction mechanism of the self-splicing and position of each sequence</B></P><P>In this experiment, we carried out reverse transcription of RNA of <I>Escherichia coli</I>, and calculated the abundance ratio of each shape pattern by detecting specific sequences by using MPN-PCR (Most Probable Number-PCR). 
</P><P>
By examining the abundance ratio of sequence A, B or C, D , and B / A shows the probability that the reaction doesn’t start (step Ⅰ). Also, we can calculate the probability that RNA will cyclize (step Ⅱ,Ⅲ) by C and D.
By examining them, we investigated the rate-limiting step in the process of cyclization or probability to cyclization of the RNA. 
</P><H1 class="theme3">Experiment</H1><P>
We extracted total RNA from <I>E. coli</I> and then carried out the reverse transcription. This time we used the two types of primer; oligo dt primer and random primer. And we carried out reverse transcription with them. We serially diluted obtained cDNA, and calculated abundance rate by using MPN-PCR.
</P><H2>Case 1: Determination of the sequence A and B</H2><P>
We carried out reverse transcription with the oligo dt primer which is complementary to the poly A sequence at the 3 'end of the mRNA (fig.2).
</P><P><B>Figure. 2  the sequence of oligo dt primer</B></P><P>
Oligo dt primer anneals specifically to the 3 'end of the mRNA, so the ratio of reverse transcription of A and B are the same. Thus, B/A calculated by the MPN-PCR shows the abundance ratio of step Ⅰ, that is, the probability that no reaction started. 
</P><H2>Case 2: Determination of the sequence C and D</H2><P>
Since there’s no poly A sequence on the mRNA after starting cyclization, it’s impossible to use the oligo dt primer. Therefore, we use the random primer. However, some problems arise when we determine them with random primer.
</P><H3>Problems</H3><P>
Reverse transcription is carried out in the following steps. 
<OL><LI>Primer is added to RNA. </LI><LI>The cDNA is synthesized by reverse transcriptase.</LI><LI>Inactivation of reverse transcriptase</LI></OL></P><P>
It means, if the primer anneals to C or D, this sequence doesn’t exist in cDNA. And if the primer did not anneal at 3' end, C or D don't exist in synthesized cDNA (linear mRNA). (fig. 3)<B>Figure. 3  difference in amplification with the change of the binding site</B></P><P>
We calculated the ratio made C or D in cDNA each step when we carried out reverse transcription with random primer.
<OL><LI>The ratio of random primer annealing depended on only base pairs of RNA.</LI><LI>When mRNA was transcribed from DNA, transcription finished at the same sequence.</LI><LI>There was no shape pattern of RNA without figure 1.</LI></OL></P><P>
On the step Ⅰ, when random primer annealed the sequence which is cyclized and its 3’ end, the ratio that D is reverse transcribed is the following.
</P><P>Step Ⅰ (fig. 4)</P><P><B>Figure. 4  probability on the stepⅠ</B></P><P>Step Ⅱ (fig. 5)</P><P><B>Figure. 5  probability on the stepⅡ</B></P><P>Step Ⅲ (fig. 6)</P><P><B>Figure. 6  probability on the stepⅢ</B></P><P>The C/D calculated in this experiment shows the following.</P><P>a<SUB>Ⅰ</SUB>and a<SUB>Ⅱ</SUB> show the following.</P><P>From the above, by examining the C/D, we are able to calculate the abundance ratio of stepⅢ ; the abundance ratio of the cyclization of the RNA.</P><H1 class="theme3">Result&amp;Data analysis</H1><H2>About A and B</H2><P>We carried out PCR and agarose gel electrophoresis (fig.7).</P><P><B>Figure. 7  agarose gel shown the result of A and B</B></P><P>
We carried out image analysis. We calculated the intensity of light relatively and then decided whether the objective band detected or not(figure.8 and 9). Judging from the square measure of each peak, we obtained following result (table.1). Table.1 shows a number of the positive fraction.
</P><P><B>Figure. 8  difference in brightness on the agarose gel of A</B><B>Figure. 9  difference in brightness on the agarose gel of B</B></P><B>table. 1  a number of the positive fraction on A and B</B><P>The ratio (B/A) which shows that the reaction doesn’t proceed is expressed the following.</P><H2>About C and D</H2><P>We carried out PCR and agarose gel electrophoresis (fig.10).</P><P><B>Figure. 10  agarose gel shown the result of C and D</B></P><P>
We carried out image analysis. We calculated the intensity of light relatively and then decided whether the objective band detected or not (figure.11 and 12). Judging from the square measure of each peak, we obtained following result (table. 2). Table.2 shows a number of the positive fraction.
</P><P><B>Figure. 11  difference in brightness on the agarose gel of C</B><B>Figure. 12  difference in brightness on the agarose gel of D</B></P><B> table. 2  a number of the positive fraction on C and D</B><P>We show the ratio (C/D) as follows.</P><P>The ratio of RNA cyclization was 2.5%.
The abundance ratio of stepⅠwas 4.7%. In other words, the rate of reaction was 95.3%.
The abundance ratio of stepⅡwas 92.8%. In stepⅡ, the sequence of the 5’intron attack on the 3’intron. It shows that this reaction is the rate-limiting step in the process of circularization. Why this step is the rate-limiting step? It is conceivable that because distance of 5’introns and 3’intron is far. So we think that the shorter protein coding gene is, the more efficiency of RNA cyclization increase. If you can streamline this step, usability of the circular mRNA may widen.</P><H1 class="theme3">Support from other team</H1><P>Because our theme “Circular mRNA” is qualitative experiment, we had a hard time with Modeling. That’s why a member of UT-Tokyo advised us the Modeling about our theme. </P><P><OL type="Ⅰ"><LI><B>Growth curve of <I>E. coli</I> and overexpression of protein by circular mRNA
    →How much does the synthesis of long-chain protein influence growth of <I>E. coli</I>?</B>By comparing our result with control (empty vector or non-plasmid), we decide parameter. We also learnt it   from UT-Tokyo that past studies will be good references for these population dynamics.
<A href="https://2013.igem.org/Team:British_Columbia/Modeling">https://2013.igem.org/Team:British_Columbia/Modeling</A></LI></OL></P><P><LI><B> Does the strength of circular mRNA depend on the length of exon?</B>If we can make a parameter which shows how long the insert is the most stable when we had Circular mRNA, this will be helpful for the correction of the model.</LI></P><P>We greatly appreciate UT-Tokyo taught us Modeling despite their busy.</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Gifu/Modeling">http://2014.igem.org/Team:Gifu/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Gifu/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Gifu/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Gifu/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Gifu/Modeling&amp;oldid=398080" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>