"
Page
Discussion
View source
History
teams
Log in
 
Team:Harvard/Project/Design
From 2011.igem.org
bar
Overview | Design | Synthesize | Test | Zinc Finger Background | Protocols
We emailed the OPEN consortium [2] and Anton Persikov [3] to acquire their respective databases of zinc fingers: Persikov had compiled the results of the past 10 years of zinc finger research. We then decided how to analyze this data and choose targets for new zinc fingers (details below). Then we programmed our ideas into a Python program, and generated 55,000 potential zinc fingers.
Make your own zinc finger sequences using our generator . Written in Python 2.7, this program is what team members created to generate 55,000 zinc finger sequences.
Contents
1 Terminology
2 Past Zinc Finger Designers
3 Our Approach
3.1 Selection of Target Sequences
3.1.1 Zinc Finger Binding Site Finder
3.2 Target Site Background Information
3.3 Data and Analysis
3.3.1 Helix Dependencies
3.4 Programming
3.4.1 Overall Method: Probabilities and Randomization
3.4.2 Refinement: Pseudocounts
4 Results: 55,000 Possible Zinc Fingers
5 References
zif268, with main structures labeled
Terminology
Backbone: contains most of the amino acids of a zinc finger protein: zif268 is the most famous backbone.
Fingers: contain a backbone and a helix, bind to a 3-base DNA triplet
Helix: the alpha helix in a finger. It is responsible for binding to a DNA triplet. Helices are made up of 7 amino acids, and fit into a specified position in a backbone. Amino acid positions are specified by -1,1,2,3,4,5, and 6.
Zinc finger proteins (ZFPs): arrays of three fingers that bind to 9 bases (3 triplets) of DNA.
Past Zinc Finger Designers
CODA's approach
Designing new zinc finger proteins (ZFPs, which are arrays of three fingers) is not an easy task: how they bind and interact with DNA bases is not fully understood, and is an active area of research. Notable past attempts to create novel ZFPs - CODA [1] and OPEN [2] -  tried a two distinct methods: CODA took a modular approach. OPEN took two known fingers from an array, and randomized protein sequences to try to generate a third finger to bind a new triplet.
Both techniques were successful in finding ZFPs to bind to novel DNA sequences, but neither was an efficient way to create new zinc finger proteins to bind novel DNA sequences.
Our Approach
Combining the concepts of OPEN and CODA, we decided to design ZFPs where the first two DNA triplets can be bound, but the third cannot. For example, if the sequence GTG GGA CCA can be bound but GTG GGA TGG cannot, we would use the first two fingers and generate the third (see [4] for the tables of such combinations created by CODA). OPEN simply randomized amino acid sequences to try to create a third finger: we wrote software that uses data from known fingers to "intelligently" generate new fingers.
Selection of Target Sequences
We decided to select six target DNA sequences that would serve as targets for our approach to designing custom zinc fingers.  These six sequences were selected due to their potential as gene therapy targets that can be targeted with zinc finger nucleases (ZFNs), which would be capable of inserting or deleting genes with great precision.  ZFNs are composed of zinc finger arrays fused to FokI type II nucleases, and they induce double strand breaks in DNA by binding to specific DNA sequences flanking the desired cut site.  These sequences are on opposite DNA strands, and are separated by a spacer that is typically 5-7 bp long.
In the following image, the blue zinc finger array represents the "bottom" ZFP while the red represents the "top" ZFP, each binding to their respective 9-bp DNA sequence.  Note the arrangement of the two ZFPs such that the FokI nucleases align for a double stranded DNA break.
We selected six target sequences across three genetic diseases, which include red-green colorblindness, familial hypercholesterolemia, and cancer caused by activation of the Myc oncogene.  These diseases were chosen due to their monogenetic nature in which insertion or deletion of a single gene carries the potential for treatment.  In terms of how we decided specifically which target sites to pursue, we followed the following steps:
Using the CoDA tables [4] , we found 9-bp DNA sequences with a known F2/F3 relationship but with an unknown F1/F2 relationship.  For instance, the helix sequences for GNNGNN are known (corresponding to the F2 and F3 helices, respectively), but the helix sequences for GNNCNN are unknown (corresponding to the F2 and F1 helices).  In total, this 9-bp DNA sequence would be 5'-GNNGNNCNN-3'.
In total, we defined six broad target sequences of varying "risk", based on what currently exists in the CoDA tables.  For instance, an F1 triplet of GNN or TNN is much more characterized, and hence less "risky" than a triplet of ANN or CNN.
Using the UCSD Genome Browser , we searched nucleotide sequences from 3kb - 10kb in size for the six broad target sites as defined above.  For diseases that would be treated with gene insertion (i.e. colorblindness and familial hypercholesterolemia), this nucleotide sequence was located in an "empty" stretch of DNA that contained no known genes.  For diseases that involved gene knockout (i.e. Myc-related cancers), the gene itself was used as a search space for ZFN binding.
The resulting target sequences can be found in the tables below:
Disease
Target Range
Binding Site Location
Bottom Finger
Top Finger
Bottom AA (F3 to F1)
Top AA (F3 to F1)
Colorblindness
chrX:153,403,001-153,407,000
3627
GTG GGA TGG
GAA GGG ACC
RNTALQH.QSAHLKR.#######
QDGNLGR.RREHLVR.#######
Familial Hypercholesterolemia
chr19:11,175,000-11,195,000
14001
GGC TGA GAC
GGA GTC CTG
ESGHLKR.QREHLTT.#######
QTTHLSR.DHSSLKR.#######
Myc-gene Cancer
chr8:128,938,529-128,941,440
198
GGT GCA GGG
GGC TGA CTC
VDHHLRR.QSTTLKR.RRAHLQN
ESGHLKR.QREHLTT.#######
Myc-gene Cancer
chr8:128,938,529-128,941,440
981
GGA GAG GGT
GGC TGG AAA
QANHLSR.RQDNLGR.TRQKLET
EKSHLTR.RREHLTI.#######
Green cells are our target sequences.
The bolded DNA triplets are the targets of our variable F1 regions in our living plasmid library .
The amino acid sequences represent the corresponding binding helix sequences for each finger in the 3-finger array for each ZFP, with a "#" sign representing the unknown specific sequence that we are looking for.
In a different format, including primers:
Disease
Target Sequence
Forward Primer (5'-3' NOT REVERSE COMPLEMENT)
Reverse Primer (5'-3' REVERSE COMPLEMENT)
Colorblindness (bot)
GCT GGA TGG
ATATAGATGCCGTCCTAGCG
TGGGCACAGGAAAGATACTT
Colorblindness (top)
GCG GGG ACC
CCCTTTAATCAGATGCGTCG
GGTCGCCCTTATTACTACCA
Familial Hypercholesterolemia (bot)
GGC TGA GAC
TTGGTCATGTGCTTTTCGTT
TCTGAGTATCCGATACCCCT
Familial Hypercholesterolemia (top)
GGA GTC CTG
GGGTGGGTAAATGGTAATGC
GCTATATCCGGGGAATCGAT
Myc-gene Cancer
GGC TGA CTC
TCCGACGGGGAGTATATACT
TTGGCCTGAAGCAGTTAGTA
Myc-gene Cancer
GGC TGG AAA
CATGTTTAGGAACGCTACCG
GGGAGGGAACGGAGATTATT
Controls
n/a
GTACATGAAACGATGGACGG
CGCTGAGGAGACTATACCAG
Zinc Finger Binding Site Finder
To use the application that we designed to search any DNA sequence for two ZFN flanking sites, please visit our Zinc Finger Binding Site Finder .  This is how we located the eight pairs of flanking sequences in the above table.
Target Site Background Information
Colorblindness (Green Opsin)
Goal: Produce functional green opsin photoreceptor proteins in the eye
Method: Insertion of functional green opsin gene (OPN1LW 1 ) upstream of normal locus in patient lacking the gene
Journal Articles
http://www.nature.com/nature/journal/v461/n7265/abs/nature08401.html
http://www.nejm.org/doi/full/10.1056/NEJMc0903652
In the News
http://www.nature.com/news/2009/090916/full/news.2009.921.html
http://www.scientificamerican.com/podcast/episode.cfm?id=gene-therapy-cures-colorblind-monke-09-09-16
http://www.msnbc.msn.com/id/32879284/ns/health-health_care/t/gene-therapy-fixes-color-blindness-monkeys/
http://www.wired.com/wiredscience/2009/09/colortherapy/
Inherited High Cholesterol (Familial Hypercholesterolemia)
Goal: Produce functional LDLR protein to remove LDL cholesterol from the blood
Method: Insertion of functional LDLR gene upstream of nonfunctional allele
http://www.ncbi.nlm.nih.gov/pubmedhealth/PMH0001429/
http://www.genome.gov/25520184
http://emedicine.medscape.com/article/121298-overview#a0199
Cancer (Myc Oncogene)
Goal: Knock out the oncogenic protein product and stop cancerous proliferation
Method: Targeted disruption (deletion) in mutated oncogene
http://www.ncbi.nlm.nih.gov/gene/4609
http://omim.org/entry/190080
Data and Analysis
By analyzing data from OPEN and Persikov, we created new zinc fingers tailored to bind specfic DNA triples. We took frequencies of each amino acid in each position (-1 to 6) in each of 12 DNA triplet types (ANN, NAN, NNA, etc, where N is any of the four DNA bases) and used that data along with knowledge of zinc finger's DNA binding properties to create over 9000 zinc fingers for each desired triplet. This diagram represents the CTG zinc fingers we created. Size of a letter in each position (-1 to 6) represents the frequency of the amino acid whose abbreviation is that letter. For example, L - leucine - occupies position 4 around 90% of the time.
OPEN provided us with a spreadsheet of ZFPs produced by their research. Anton Persikov, during his own ZFP research, has compiled a database of ZFPs from studies from 1980-2005 which he shared with us. From these two datasets, we distilled over 3000 unique ZFPs which contained approximately 1700 unique fingers. Our database is available as part of a download of our zinc finger sequence generator .
We analyzed this dataset for frequency (how often a given amino acid appears in a given position in the helix) and pairing (if amino acid A is in position 1, how often is amino acid B next to it).
Helix Dependencies
Amino acids do not exist in a vacuum: they must somehow be affected by amino acids around them. Besides pairing data, we realized that other interactions could be taking place, and that we needed a way to see these other relationships.
We know that the DNA bases affect the amino acid sequence, so we started looking for evidence that, for example, changing the third base (going from NNA to NNC, etc) affects position -1.
To do this, we created these graphs of the frequency of amino acids in each position, and then blinked the graphs against each other to see what changes. We looked at the probability graphs to determine which amino acid positions on the finger's helix interact with which bases. For example, if you compare NAN to NCN, you will see a large change in the asparagine content in position 3.
We saw some interactions that are fairly well established, while others have been more recently proposed [5] .
Click on the triplets on the left to compare the frequencies for various DNA triplets:
GNN
TNN
CNN
ANN
NGN
NTN
NCN
NAN
NNG
NNT
NNC
NNA
Probability data for the 783 fingers that bind to GNN triplets. Note the high probability of leucine at position 4 and arginine at position 6.
Probability data for the 128 fingers that bind to TNN triplets. Note the high probability of leucine at position 4.
Probability data for the 16 fingers that bind to CNN triplets. There may not be enough data to consider this information statistically significant
Probability data for the 29 fingers that bind to ANN triplets. There may not be enough data to consider this information statistically significant
Probability data for the 298 fingers that bind to NGN triplets. The position 4 leucine motif remains. There is also a high probability (> 0.5) of a histidine at position 3 and an arginine at position 6.
Probability data for the 177 fingers that bind to NTN triplets. The position 4 leucine motif remains.
Probability data for the 244 fingers that bind to NCN triplets. The position 4 leucine motif remains. There is also a very high probability of an arginine at position 6.
Probability data for the 248 fingers that bind to NAN triplets. The position 4 leucine motif remains. There is also a very high probability (> 0.75) of an asparagine at position 3 and an arginine at position 6.
Probability data for the 234 fingers that bind to NNG triplets. The position 4 leucine motif remains. There is also a very high probability (> 0.75) of an asparagine at position 1 and a high probability (> 0.5) of an aspartic acid at position 2 and an arginine at position 6.
Probability data for the 247 fingers that bind to NNT triplets. The position 4 leucine motif remains. There is also a high (> 0.5) probability of an arginine at position 6.
Probability data for the 262 fingers that bind to NNC triplets. The position 4 leucine motif remains. There is also a very high (> 0.75) probability of an arginine at position 6.
Probability data for the 218 fingers that bind to NNA triplets. The position 4 leucine motif remains. There is also a very high (> 0.75) probability of a glutamine at position -1 and an arginine at position 6.
Interactions between amino acids and DNA: from Persikov [5] and our own work.
(A more rigorous way to calculate this is to calculate the entropy change as you change the amino acids in each position. But that is computationally intensive)
By doing this, we were able to see several patterns.
xNN(Vary base 1): Amino acid 6 changes
NxN(Vary base 2): Amino acid 3 changes
NNx(Vary base 3): Amino acid -1 and 2(?) changes
Our program looks at dependencies between amino acids when generating sequences.
We decided on these amino acid dependencies, using both established data and patterns we saw in the OPEN data:
-1 and 2
2 and 1
6 and 5
How we assign amino acids based on probabilities.
Because there is not much data for 'CNN' and 'ANN' sequences (with 16 and 29 known fingers that bind to each triplet, respectively), we should use pseudocounts for these sequences, so that our frequency generator is not too biased toward probabilities that may not be significant.
Programming
Overall Method: Probabilities and Randomization
Our generation program uses these amino acid frequencies as probabilities that position X contains amino acid X, given what triplet we are trying to bind. Using the dependencies we found, we change which frequency tables are used to generate the new helix. Frequency tables are built using the data from the above graphs.
See the image at right for explanation on how we turn probabilities into amino acids.
To generate one helix (7 amino acids), the program goes through the following steps:
Step
Example for TGG
Generate an amino acid for position -1 (P0), using probabilities only from NNx
R _ _ _ _ _ _
Taking into account the amino acid chosen for P0, generate P2, also using probabilities only from NNx
R _ S _ _ _ _
Taking into account the amino acid chosen for P2, generate P1, using overall probabilities for P1
R L S _ _ _ _
Generate P3, using probabilities only from NxN
R L S H _ _ _
Generate P4, using overall probabilities for P4
R L S H L _ _
Generate P6, using probabilities only from xNN
R L S H L _ M
Taking into account the amino acid chosen for P6, generate P5, using overall probabilities for P5
R L S H L Q M
These steps are based on the relationships we found from reading papers [Persikov] and studying the above frequency graphs, which were created from successful ZFPs.
The generated helix is then placed into a backbone: for example, this helix was placed in the zif268 backbone, giving a finger with a final amino acid sequence of FQCRICMRNFSRLSHLQMHIRTH.
This finger is then reverse-translated into DNA (along with the sequences for the fixed first two fingers of the ZFP) for inclusion in the chip.
Refinement: Pseudocounts
Pseudocounts are necessary for data that has small sample size - we could be missing out on working helices because a letter's frequency is 0 when it shouldn't be. For CNN and ANN, our dataset is tiny compared to GNN and TNN: CNN and ANN have around 20 datapoints while GNN has over 700. Because of this discrepancy in sample size, we must add psuedocounts to CNN and ANN in order to allow for more variation than is shown in our data.
When generating helixes for CTC (because of position 6's reliance on the CNN frequencies) to test psuedocounts, we see in the created sequences the difference pseudocounts make.  A psuedocount of .015 changes the frequency of any amino acid from whose frequency is 0 by bumping it up to the value of the psuedocount: ex. A = 0 becomes A = .015, giving A a 1.5% chance of being selected instead of none at all.
Visualizing our data, we get various pseudocount (psu = ) values for position 7 (which, in reality, is position 6 in the helix). The size of the letter directly corresponds to the percentage of sequences that have that letter in that position. A letter that takes up 1/3 of a column is present in that position in 33% of the helices.
psu = 0.000
psu = 0.010
0.015
0.020
Notice how psu = 0 gives only the four letters found in our CNN dataset, while psu > 0 adds in other letters, each with a small probability.
The question is how much psu to add: less means we weight our (possibly flawed) data of proven zinc fingers more. Higher psu adds more randomness (variation) to our sequences, but some (perhaps large) fraction of those sequences will not work, and take away space from the proven amino acids.
We ultimately chose psu = .015 for our software.
Results: 55,000 Possible Zinc Fingers
We made 55,000 sequences, distributed evenly among 6 DNA target triplets. That's 9150 per target.
Because our program's output changes dramatically based on the input triplet, no two sets of sequences are the same:
AAA
ACC
CTC
CTG
GAC
TGG
References
1. Jeffry D Sander, Elizabeth J Dahlborg, Mathew J Goodwin, Lindsay Cade, Feng Zhang, Daniel Cifuentes, Shaun J Curtin, Jessica S Blackburn, Stacey Thibodeau-Beganny, Yiping Qi, Christopher J Pierick, Ellen Hoffman, Morgan L Maeder, Cyd Khayter, Deepak Reyon, Drena Dobbs, David M Langenau, Robert M Stupar, Antonio J Giraldez, Daniel F Voytas, Randall T Peterson,Jing-Ruey J Yeh, J Keith Joung. Selection-free zinc-finger-nuclease engineering by context-dependent assembly (CoDA)(2011). Nature Methods 8, 67–69. [1]
2. Morgan L. Maeder, Stacey Thibodeau-Beganny, Anna Osiak, David A. Wright, Reshma M. Anthony, Magdalena Eichtinger, Tao Jiang, Jonathan E. Foley, Ronnie J. Winfrey, Jeffrey A. Townsend, Erica Unger-Wallace, Jeffry D. Sander, Felix Müller-Lerch, Fengli Fu, Joseph Pearlberg, Carl Göbel, Justin P. Dassie, Shondra M. Pruett-Miller, Matthew H. Porteus, Dennis C. Sgroi, A. John Iafrate, Drena Dobbs, Paul B. McCray Jr., Toni Cathomen, Daniel F. Voytas, J. Keith Joung. Rapid “Open-Source” Engineering of Customized Zinc-Finger Nucleases for Highly Efficient Gene Modification (2008). Molecular Cell Volume 31, Issue 2, 25 July 2008, Pages 294-301. [2]
3. Anton Persikov, PhD. [3]
4. Page 9-10 of Supplementary Material for: Jeffry D Sander, Elizabeth J Dahlborg, Mathew J Goodwin, Lindsay Cade, Feng Zhang, Daniel Cifuentes, Shaun J Curtin, Jessica S Blackburn, Stacey Thibodeau-Beganny, Yiping Qi, Christopher J Pierick, Ellen Hoffman, Morgan L Maeder, Cyd Khayter, Deepak Reyon, Drena Dobbs, David M Langenau, Robert M Stupar, Antonio J Giraldez, Daniel F Voytas, Randall T Peterson,Jing-Ruey J Yeh, J Keith Joung. Selection-free zinc-finger-nuclease engineering by context-dependent assembly (CoDA)(2011). Nature Methods 8, 67–69. [4]
5. Persikov A.V., Osada R., Singh M. (2009) "Predicting DNA recognition by Cys2His2 zinc finger proteins". Bioinformatics, 25 (1): 22-29. [5]
Retrieved from " http://2011.igem.org/Team:Harvard/Project/Design "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
