"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Harvard/Template:NotebookDataSeptember
From 2011.igem.org
Contents
1 September 5th
1.1 CB Bottom Library Preliminary Sequencing Results
2 September 7
2.1 File Processing for Library Sequencing
3 September 8th
4 September 11th
5 September 12th
5.1 Library Construction
5.2 CB bottom library
6 September 17th
7 September 19th
8 September 21st
9 September 22nd
10 September 23rd
10.1 CB bottom hits:
September 5th
CB Bottom Library Preliminary Sequencing Results
Recap of what has happened:
After the CB bottom library was transformed into (turbocomp or selection strain?) cells, 94 colonies were picked off of the plate after transformation.  These colonies were then sequenced, with the eventual objective of determining several factors about our transformed library:
How much spread does our library cover?  Theoretically, the 94 colonies should be evenly spread throughout the 9,150 chip sequences, and the likelihood of a given sequence being represented twice is very low.
Additionally, are the backbones being represented after transformation in the same ratios as they are on the chip?
What are the chip error rates?  The process of synthesizing the chip is error-prone, but we do not know what percentage of the oligos on the chip contain errors.  These errors can be SNPs or frameshifts from insertions/deletions.
Currently, the sequencing files are all located under sequences/CB bottom insert seq 08292011.
29 sequences were analyzed by hand by importing the sequence files into SeqBuilder and manually identifying the position of the F1 motif based on its translation.
Out of these 29 sequences, 6 showed a frameshift mutation, causing the F1 and F2/F3 to be expressed in different reading frames.  This indicates that based on these results, roughly 20% of the oligos on the chip contain frameshift mutations.  This may be ok, due to the remaining 80% being available.  The mutated oligos will simply be dropped out of the pool during selection due to creation of a nonfunctional protein.
Furthermore, 2 of the 29 sequences contained SNPs.  One of them changed the F in "FQCRICMRN" to an S, while the other SNP introduced an early stop codon into the zinc finger, also making it non-functional.
Out of the 94 colonies that were sequenced, 20 of the traces were sufficiently poor quality that no F1 was identified.
For the future, we will need to create a script that will compare the sequenced results to the actual chip database.  This can be accomplished with the following steps:
Perform restriction cuts on the chip in silico, that is, manually edit out the primer tag/type II binding site, leaving only the part that encodes the F1 finger.  This will be the dictionary that we will use to check all of our sequences against.
Parse the sequencing data by identifying the part that codes for F1.  This can be accomplished with regular expressions.  One would need to identify the end of the omega/F1 linker (the last 30 or so nucleotides) and the beginning of the F1/F2 linker (the first 30 nucleotides).  Everything in between can be assumed to be F1 material.
The program will then check to see if the F1 that is parsed from the sequencing data exists in the dictionary.  If it does, that is great.  If it does not, it will do one of two things:
Perform a modulus 3 operation to see if there was a frameshift.  If the F1 is not a multiple of 3, it can be assumed that a frameshift occured.
Look for the closest entry in the dictionary to the errant sequence (something off by 1 nucleotide, for example)
This will allow us to automate our efforts, and will make it easier when more sequencing is performed.  Vatsan recommends using BioPython to write the script.
September 7
File Processing for Library Sequencing
Preliminary processing of the library sequences consists of agreggating the 94 individual fasta files into one multifasta file (which will be easier to work with) and taking the reverse complements of all the sequences (since we used a reverse primer for sequencing, the 100_bp_junction primer).
A Python script for converting fasta into multifasta can be found here: [1] , and a website for converting a multifasta file into its reverse complement can be found here: [2] .
The reverse complement multifasta file for the 94 sequences thus far can be found in the dropbox under sequences/CB bottom insert seq 08292011 with the name CB bottom seq multifasta RC.fasta.
Remaining efforts include parsing the file to extract the F1 subunits, which will require either removing the newline characters and using regex or by using a regex that can recognize sequences across multiple lines (i.e. regex that recognizes sequences with newline characters randomly in there).  The chip dictionary still needs to be built and the code for comparing our sequences with the dictionary also still needs to be written.
September 8th
Biobricks:
The plates looked good: the plates with the omega-ZF and ZFB-his3-ura3 inserts had many more colonies than the pSB1C3 self-ligation control.
Chose 4 colonies from each biobrick and will grow them overnight. Tomorrow they will be miniprepped and sent out for sequencing using the standard primers on the backbone. (This should cover the entire insert for omega-ZF and most of the insert for ZFB-his3-ura3.)
Selection strains:
Streaked out fresh plates from the glycerol stocks of the EcNR2 selection strain with the various zinc finger binding sites. We will hopefully use these when the other libraries are assembled (and we can use the CB bottom colonies for a fresh transformation with its library).
September 11th
CB bottom ZFB strain:
Repeated the transformation with CB bottom ZF library (100ng)
recovered for 1 hr in LB
plated on LB+spec to check that the transformation worked
reinoculated 1mL of the transformation into 10mL LB+spec and let it grow for another hour
plated 10uL (brought up to 100uL in NM) on LB+spec, NM+his+spec, NM+spec, NM+spec+1mM 3-AT, NM+spec+3mM 3-AT, NM+spec+10mM 3-AT plates
let rest of LB+spec culture grow overnight to use tomorrow for glycerol stocks and more plating
September 12th
Library Construction
Overnight cultures were miniprepped (yields around 30-40ng/uL) and digested following the August 22nd protocol (but with 1ug of template plasmid)
The digested plasmid was then immediately phosphatized by adding 1uL of Antarctic phosphatase and 10uL of the phosphatase buffer and incubating for 30min at 37C followed by 5 min at 65C.
Minelute purified the product: recovered about half of the DNA
CB bottom library
Made glycerol stock of CB bottom strain + CB bottom library
LB+spec plates look great, so the transformation worked! Could be small colonies on the NM-based plates, but they are very small so we will let them grow up for longer
Also want to plate washed cells from the transformation: washed twice in NM, and then plated either the equivalent of 100uL or 10uL of cells onto LB+spec, NM+his+spec, NM+spec, NM+1mM 3AT+spec, NM+3mM 3AT+spec, NM+10mM 3AT+spec plates.
September 17th
ZFB-his3-ura3 biobrick:
Digested insert and pSB1C3 backbone with EcoR1 and Spe1 following NEBiolabs protocol
Minelute: 50ng/uL and 17ng/uL respectively
ligated according to NEBiolabs protocol (both with and without ZFB-hisura insert) and transformed into chem comp cells. Recovered for 1 hr and plated on LB+chloramphenicol
CB bottom library:
Retransformed CB bottom ZFB strain with CB bottom library and Zif268 as a negative control (about 100ng each)
Plated 10uL (unwashed) of both cultures on the full range of plates. The rest of the culture was inoculated with spec and left to grow overnight.
Library assembly:
Ran the digested and phosphotased samples on a gel with the uncut plasmids to verify that they were cut correctly.  The cut samples ran differently from the uncut, showing that the plasmids were successfully linearized.
Digested and undigested zinc finger backbones 9/17/11
September 19th
PCR using VF2 and VR primers to check that biobrick assembly worked
Results: bands were too small--insertion not successful
repeated digestion (NEBiolabs protocol) for ZFB-hisura and pSB1C3 with EcoR1 and Spe1
September 21st
ligated and transformed ZFB-his3-ura3 biobrick into chem comp cells using same procedure as before
CB bottom library plates: some of the plates looked like they may have colonies, so we picked a few to grow up in LB+spec for further confirmation.
September 22nd
Ligation plates still had very few colonies, but when I repeated the PCR with the VR and VF2 primers, it looks like one of them (#18) does have the proper insert! Culture will grow overnight and be miniprepped tomorrow for Quikchange.
ZFB-his3-URA3 biobrick PCR: expected band size about 1.5kb
Miniprepped TolC selection strain KS to get a fresh batch of Zif268 on the spec plasmid
26 of the 30 CB bottom cultures grew overnight in LB+spec, so I did a PCR of the pZE23G plasmid junction (same procedure as July 12th). Zif268 acted as a positive control. It appears that most of the colonies that grew do indeed have the zinc finger expression plasmid in them--they might be novel hits!!!!
CB bottom colony PCR: expected band size about 1.4kb
September 23rd
CB bottom hits:
Plate reader:
Columns 1, 7: LB+spec
Colummns 2, 8: NM+his
Columns 3, 9: NM
Columns 4, 10: 1mM 3-AT
Columns 5, 11: 3mM 3-AT
Columns 6, 12: 10mM 3-AT
Row A: negative control (CB bottom +Zif268)(1-6) and D1 (7-12)
Row B: D2 and C3
Row C: B1 and B2
Row D: B3 and B4
Row E: B5, B6
Row F: B7, B8
Row G: B9, B10
(D1-2 are from 10mM 3-AT plate, C3 from 1mM 3-AT, all Bs from 3mM 3-AT plate)
Sequencing:
Repeat yesterday's PCR to be sent out for sequencing
Biobricks:
Miniprepped chem comp cells with ZFB-hisura biobrick for Quikchange tomorrow.
Retrieved from " http://2011.igem.org/Team:Harvard/Template:NotebookDataSeptember "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
