"
Page
Discussion
View source
History
teams
Log in
 
Team:Harvard/Technology/Chip Synthesis
From 2011.igem.org
bar
Overview | MAGE | Chip-Based Synthesis | Lambda Red | Protocols
To see how we successfully used chip-based synthesis, go to our Synthesize page and results section .
For a step-by-step procedure, see our Chip DNA Extraction Protocols .
Chip-Based DNA Synthesis
We are creating 55,000 zinc fingers using microchip synthesis (Kosuri et al). These fingers will then be tried against the DNA sequences we wish to bind.
Summary (Adapted from Kosuri et al) [1]
The synthesis of DNA encoding regulatory elements, genes, pathways and entire genomes provides powerful ways to both test biological hypotheses and harness biology for our use. For example, from the use of oligonucleotides in deciphering the genetic code to the recent complete synthesis of a viable bacterial genome, DNA synthesis has engendered tremendous progress in biology. Currently, almost all DNA synthesis relies on the use of phosphoramidite chemistry on controlled-pore glass (CPG) substrates. The synthesis of gene-sized fragments (500–5,000 base pairs (bp)) relies on assembling many CPG oligonucleotides together using a variety of gene synthesis techniques. Technologies to assemble verified gene-sized fragments into much larger synthetic constructs are now fairly mature.
The price of gene synthesis has fallen drastically over the last decade. However, the current commercial price of gene synthesis, ~$0.40–1.00/bp, has begun to approach the relatively stable cost of the CPG oligonucleotide precursors (~$0.10–0.20/bp)1, suggesting that oligonucleotide cost is limiting. At these prices, the construction of large gene libraries and synthetic genomes is out of reach to most.
A promising route is to harness existing DNA microchips, which can produce up to a million different oligonucleotides on a single chip, as a source of DNA. Recently, the quality of microchip-synthesized oligonucleotides was improved by controlling depurination during the synthesis process. These arrays produce up to 55,000 200-mer oligonucleotides on a single chip and are sold as a ~1–10 picomole pools of oligonucleotides, termed OLS pools (oligo library synthesis). Estimations of the frequency of transitions, transversions, insertions and deletions in OLS pools found the overall error rate to be ~1/500 bp both before and after PCR amplification, suggesting that OLS pools can be used for accurate large-scale gene synthesis.
The Man Behind the Research: Sriram Kosuri on Chip-Based DNA Synthesis
References
1. Sriram Kosuri, Nikolai Eroshenko, Emily M LeProust, Michael Super, Jeffrey Way, Jin Billy Li, George M Church. (2010). Scalable gene synthesis by selective amplification of DNA pools from high-fidelity microchips. Nature Biotechnology, 28(12):1295-9. [1]
Retrieved from " http://2011.igem.org/Team:Harvard/Technology/Chip_Synthesis "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
