"
Page
Discussion
View source
History
teams
Log in
 
Team:Harvard/Template:NotebookDataAugust4
From 2011.igem.org
Contents
1 August 17th
1.1 Justin
1.1.1 Miniprep of negative control library colonies
1.1.2 Selection plasmid concentrations
1.2 Chip library assembly
1.2.1 PCR of Chip Oligos
2 August 18th
2.1 Justin
2.1.1 Transformation results
2.1.2 Pre-transformation library selection plates
2.1.3 Selection plate seeding protocol
2.1.4 GFP reporter plasmid miniprep
2.2 Chip library assembly
2.2.1 Digestion of library and CB bottom backbone
2.2.2 Phosphatase reaction
2.2.3 Planning for ligation of library and backbone
3 August 19th
3.1 Plate reader
3.2 GFP reporter Zif268 plasmid transformation
3.3 Chip library assembly
3.3.1 Adding phosphatase to CB bottom
3.3.2 Ligation of CB bottom and library inserts
4 August 20th
4.1 Plate reader for ECNR2 Wolfe
4.2 Chip library assembly
4.2.1 Transformation
5 August 21st
5.1 Chip library assembly
5.1.1 Results of library transformation
5.1.2 Minipreps and glycerol stocks
5.1.3 Cross junction PCR of library colonies
6 August 22nd
6.1 Zif268-GFP results
6.2 Wolfe system biobrick
6.3 Chip library assembly
6.3.1 Cross junction PCR: Take 2
6.3.2 Digestion of CB bottom
7 August 23rd
7.1 Selection plates
7.2 Changing ZF binding site
7.3 Plate reader
7.4 Chip library
7.4.1 Phosphatase of newest cut backbone
7.4.2 Ligation of CB bottom and library
7.4.3 Transformation of CB bottom+library and controls
8 August 24th
8.1 Plate reader results
8.2 Chip library
8.2.1 Results of yesterday's transformation
8.2.2 Cutting the library
8.2.3 Ligation and transformation
8.3 Selection plates
9 August 25th
9.1 Selection plates and cultures
9.2 Chip library
9.2.1 Results of transoformation
9.2.2 Cross junction check of library colonies
10 August 26th
10.1 Biobricks
10.2 Selection strain and zinc finger library
10.3 Plate reader
10.4 Chip library
10.4.1 Sequencing results
11 August 27th
11.1 Plate reader results
August 17th
Justin
Miniprep of negative control library colonies
Today I miniprepped the 24 colonies that were picked and grown to saturation last night.  These 24 colonies should contain a good number of the 18 practice plate plasmids within them.  The minipreps were very successful for these strains, yielding concentrations from ~30-60 ng/ul.  Nida ran a junction PCR on the 24 miniprepped products, and we sent them out for sequencing in order to determine which of the 18 practice plate oligos are contained within these 24 colonies.  The yields of the minipreps can be seen below:
Miniprep concentrations and purities
Selection plasmid concentrations
In order to proceed with the experiment #2 outlined yesterday, Nida and I combined the miniprepped products (everything except for sample 6, which had very low miniprep yields for some reason) in equal molar ratios according to their concentrations.  This constitutes our negative control library, containing complete zinc fingers targeted towards the CB bottom target, but not towards the Zif268 binding site.
I then took the negative control library and spiked it with varying amounts of Zif268 whole plasmid, yielding mixes that contained in total 500 ng of plasmid.  Thus, for the positive and negative plasmid controls, they each contained 500 ng of Zif268 and 500 ng of negative control library, respectively.  For the 1:10 dilution, there were 50 ng of Zif268 and 450 ng of negative control, and so on.  The Zif268 plasmid that was used originated from the midiprep that Nida performed last Friday.
The 8 mixtures outlined yesterday were then transformed into the EcNR2 selection strain, and plated on spec plates.  This will simply select for bacteria that contain plasmid, but not specifically for those that contain Zif268 plasmid.
To see the volumes used for creating the plasmid dilutions, see the following file: Practice Library Assembly
Chip library assembly
PCR of Chip Oligos
With Sri and Dan’s guidance, I performed a qPCR on the chip oligo's using the protocol outlined in the supplementary section of Sri's paper. I performed six reactions using the appropriate primer in each case (CB top, CB bottom, FH top, FH bottom, Myc 981, Myc 198). I used the following recipe for the PCR (total volume 200 ul):
100 ul Sybr Green master mix
200 nm primer (4 ul each of 10 um primer)
2 ul chip
90 ul water
I ran the reactions on the thermocycler, while standing by to see the graphs being produced, using the following program:
95* for 1 minute
Initially, the reactions occur at an exponential rate. During this period:
95* for 10 seconds
62* for 30 seconds
Once the reaction began to plateau, I stopped the reaction. Prior to this point all the oligos were being amplified equally. Once the plateau occurs, amplication occurs at varying rates. This took about 12 cycles total.
FH Top samples had slightly less volume, and plateaued earlier. This may have occurred due to too little water or master mix being added.
I performed a PCR purification on the PCR product at got the following yields:
Name        Conc.        260/280   CB Top:     18.2         1.83  CB Bottom:  25.1         1.78  FH Top:     20.3         20.3  FH Bottom:  21.4         21.4  Myc 981     26.1         1.73  Myc 198     19.7         1.84
In order to confirm the nanodrop readings, I ran 1 ul of each sample on a 4% e-gel. We ran a low range quanitative DNA ladder, at varying dilutions.
The results of the qPCR on our library
When comparing the intensity of the bands to the ladder, it does indeed look like there is approximately 20 ng/ul DNA in the products.
August 18th
Justin
Transformation results
The plates of transformed bacteria yesterday were uneven in their efficiencies because some of the plates had a low time constant for the electroporation (around 4.8) when a good time constant was around 5.8.  We got excellent efficiency in the transformation for Zif268 alone, but for the plasmid mixes that included the negative control library, we got poor time constants.  We still got colonies, but instead of getting a lawn of bacteria there were approximately 150 discrete colonies on the plate.
To get around this, Noah picked colonies from the plates and inoculated fresh LB/spec with them, and let them grow to saturation over the course of the day.  We used these resulting saturated cultures in our selection plates later in the day.
Pre-transformation library selection plates
Today, Noah and I prepared selection plates to test our ability to distinguish a positive hit among negative controls in varying proportions.  The pre-transformation aspect refers to the fact that it is the Zif268 plasmid that is diluted into the negative control plasmids before transformation, rather than diluting Zif268 bacteria into the negative control bacteria after transformation (this is analogous to how it will be done with the chip).  Using the negative control library assembled from yesterday, we created a 96-well plate with the following dimensions.
Going horizontally (each column):
1: Positive control (only Zif268)
2: Positive control (replicate of column 1)
3: 10^-1
4: 10^-2
5: 10^-3
6: 10^-4
7: 10^-5
8: 10^-6
9: Negative plasmid control (only negative control library)
10: Negative plasmid control (replicate of column 9)
11: No plasmid control (Wolfe ECNR2 strain without any plasmid)
12: No plasmid control (replicate of column 12)
Going vertically (each row):
A: LB + carb
B: NM + his (0.1%)
C: NM
D: NM (replicate of row C)
E: NM + 1 mM 3-AT
F: NM + 3 mM 3-AT
G: NM + 10 mM 3-AT
H: NM + 25 mM 3-AT
We left this in the plate reader overnight.
Selection plate seeding protocol
Notes: Each well contained 150 ul of liquid, and a 1/100 dilution of bacteria at an OD of 3 (so this would be adding 1.5 ul of bacteria).  The bacteria were grown up in LB to saturation and were washed in NM twice.  This was accomplished by the following steps:
Take the equivalent of 800 ul of 1.5 OD bacteria into a tube.
Centrifuge the bacteria into a pellet (max speed for 1 min)
Pipette off as much LB as possible, using a 1000 ul pipet first and then switching to a 200 ul pipet.
Resuspend the bacteria pellet in 500 ul of NM (1st wash).
Recentrifuge into pellet.
Repeat steps 3 and 4 (2nd wash).
Resuspend in 400 ul of NM (this gets it to an OD of 3).
The bacteria sample is now ready to be used.
GFP reporter plasmid miniprep
Today, Nida and I miniprepped 6 ml of GFP reporter cells, consolidating the equivalent of 2 minipreps into one column.  However, the yields and purities were pretty terrible, with yields of 7 and 11 ng/ul and purities that were much greater than 2.  This is likely to be caused by the fact that the cultures that we miniprepped were prepared by spiking 3 ml of LB/spec with 1 ml of saturated culture in an attempt to grow up a lot of culture in a short period of time.  There was likely a lot of cell lysis and general cell debris that hindered the miniprep, causing it to give such terrible yields.
We inoculated 12 more ml of culture for miniprepping tomorrow morning, since we are low on time.  This should hopefully give us enough culture to work with such that the miniprep will eventually give us the quantity/concentration of DNA required for transformation and/or plasmid library testing.
Chip library assembly
Digestion of library and CB bottom backbone
We digested our CB bottom library pool and CB bottom plasmid with Bsa1 using the following recipes:
Library (40 ul total)
30 ul pool library (600 ng)
0.75 ul enzyme
4 ul buffer 4
1 ul bsa
4.25 ul water
CB bottom backbone (10 ul total)
4.2 ul dna (1000 ng)
1 ul enzyme
1 ul buffer
0.25 ul bsa
3.55 ul water
We ran the digestion for 2 hours at 37*. Then we heat inactivated the enzyme at 65* for 20 minutes.
Phosphatase reaction
We then took the product of the digestion, and added phosphotase to prevent the pieces from re-ligating by removing the 5' phosphate. We used NEB Antarctic phosphotase. We used the following recipes:
CB bottom backbone
10 ul backbone
1.22 ul buffer
1 ul phosphatase
We ran the reaction for 4.75 hours at 37*, and then heat inactivated the enzyme at 65* for 20 minutes.
We then performed a PCR clean-up on the product, with the following yields:
Library: 9.8 ng/ul (total 294 ng)
CB bottom: 11 ng/ul (total 330 ng)
Planning for ligation of library and backbone
We were unable to perform the ligation today. If we want to use 200 ng per ligation reaction we need 1000 ng of DNA, which is the amount we started with. We can cut some more backbone, or phosphatase the backbone I had cut earlier (conc: 69 ng/ul). We can also use 100 ng of backbone instead.
Based on the yields of the PCR clean up, as assuming 100 ng of backbone, we'll need the following amounts of insert for our ligations:
Insert:backbone   Amount of insert (ul)  2:1               1  3:1               1.5  5:1               2.5  8:1               4
This is because our insert is 137 bp and our backbone is 2788 bp. For equal molar ratios, we'll need 4.9 ng (or 0.5 ul) of insert for a 1:1 insert:backbone.
We also plan on running cut backbone (no phosphotase), and phosphotased backbone as controls.
August 19th
Plate reader
Yesterday's plate reader experiment looked great! There is definitely a difference between 10^-6 dilution of Zif268 and the negative controls!
All conditions in NM (2nd duplicate) 8/18/11
Comparison of 10^-6 dilution of Zif268 with negative library control 8/18/11
We will duplicate this experiment in a slightly different way to validate it.  Using the same media conditions, we will dilute the amount of Zif268 spec plasmid post-transformation rather than pre-transformation. (Instead of adding different amounts of plasmid to the cells being transformed, we will add different ratios of saturated EcNR2 selection strain+Zif268 spec and saturated EcNR2 selection strain+negative control plasmids.)
For the protocol we will use in seeding the plate, see yesterday's entry.
GFP reporter Zif268 plasmid transformation
We also want to duplicate the plate reader experiment using our GFP reporter strain, so that we can show definitively that there is more Zif268 (represented by the GFP) in one well versus another.
Miniprep of GFP Zif268 reporter plasmid: 151.5ng/uL, 260/280=1.89
Transformed 1.5mL EcNR2 selection strain with 1uL plasmid (time constant of 4.8)
Recovered 1hr in LB, and then added spec
Grew 1hr in spec and then reinoculated 30 uL into a fresh tube of LB+spec to grow overnight (seeing as even with a poor time constant, 100 ul yielded over 100 colonies on the plate that Noah had tried earlier).
Chip library assembly
Adding phosphatase to CB bottom
Because we did not cut enough DNA yesterday, we decided to add phosphatase to a sample of CB bottom backbone we had cut before. This sample, which had already been PCR purified and eluted in 30 ul of EB, had a concentration of 69 ng/ul according to the nanodrop.
In order to make sure the sample was cut appropriately, we ran it and a sample of the CB bottom we cut yesterday on a 4% e-gel.
A comparison of our old and new CB bottom backbones. Note, new has already been phosphotased.
The band was much more intense for the old sample, and it looked like it had a concentration of between 40 and 100 ng/ul. We decided to perform the reaction on 20 ul of the cut sample (for approximately 1380 ng of DNA) and used the following recipe:
20 ul DNA
1 ul phosphatase
2.33 ul phosphatase reaction buffer.
We incubated the reaction at 37* for 1 hour 10 minutes, and heat deactivated the enzyme by heating it at 65* for 20 minutes.
We then PCR purified the sample, eluting in 30 ul of EB, and got a yield of 41.7 ng/ul, meaning we had a total of 1251 ng of backbone, which was more than enough for our ligation.
Ligation of CB bottom and library inserts
We performed a ligation using 100 ng of backbone and insert to backbone ratios of 2:1, 3:1, 5:1, and 8:1. We also ligated the cut plasmid and the phosphotased plasmid as controls. We used the following recipe for our ligation:
2.4 ul backbone (100 ng)
insert:
2:1 -> 1 ul
3:1 -> 1.5 ul
5:1 -> 2.5 ul
8:1 -> 4 ul
2 ul 10x ligation buffer
1 ul ligase
water to bring the total volume to 20 ul
We ran the reaction at 16* for 12 hours, and then let sit at 4* forever.
August 20th
Plate reader for ECNR2 Wolfe
Yesterday's plate reader results: the trends are ok but the graphs are messy, probably because of condensation on the lid.
All dilutions in NM 8/19/11
All dilutions in NM+3mM 3-AT 8/19/11
Comparison of Zif268 and 10^-4 dilution 8/19/11
This experiment is to figure out the appropriate concentration of IPTG, with varying concentrations of 3-AT.
the concentrations are
IPTG concentrations from 0,10 and 100mM (columns 1-4, 5-8, 9-12)
3-AT concentrations from 1,3,10 and 25mM (rows E,F,G,H)
LB (A), NM+histidine (B), NM (C & D)
The four strains are
EcNR2∆HisB∆PyrF∆rpoZ+kan-ZFB-wp-his3-ura3 + no plasmid (columns 1, 5, 9)
EcNR2∆HisB∆PyrF∆rpoZ+kan-ZFB-wp-his3-ura3 + CB_bottom plasmid (negative control, library plasmid) (columns 2, 6, 10)
EcNR2∆HisB∆PyrF∆rpoZ+kan-ZFB-wp-his3-ura3 + zif268 spec plasmid (selection strain with the matching ZFB and ZFP)(columns 3, 7, 11)
EcNR2∆HisB∆PyrF∆rpoZ+kan-ZFB-wp-his3-ura3 + (zif268+GFP) spec plasmid (selection strain with the matching ZFB and ZFP, should fluoresce)(columns 4, 8, 12)
We will take one fluorescence reading before placing it in the plate reader, and one after we finish it.
In addition to the plate reader, we are curious to see how IPTG affects fluorescence levels. We made LB+spec plates with 0, 6.25uM, 25uM, and 100uM concentrations of IPTG
Each plate was spread with 1uL of EcNR2∆HisB∆PyrF∆rpoZ+kan-ZFB-wp-his3-ura3 + (zif268+GFP) and left overnight at 30C
Tomorrow we can compare fluorescence levels between the different IPTG concentrations.
Chip library assembly
Transformation
We ran the ligation products through a min-elute column, eluting in 10 ul of water, with the following yields:
Cut: 22.4 ng/ul
Phosphotased: 24.2 ng/ul
2 to 1: 25 ng/ul
3 to 1: 25.9 ng/ul
5 to 1: 20.1 ng/ul
8 to 1: 20.9 ng/ul
Later we ran a transformation, using the turbo competent cells protocol. We used 100 ng of ligation product for the transformation:
cut: 4.46 ul
phosphotased: 4.13 ul
2 to 1: 4 ul
3 to 1: 3.86 ul
5 to 1: 4.98 ul
8 to 1: 4.78 ul
We then plated 1 ul and 10 ul of cells on spec plates, and let them incubate at 37* for 12 hours. Only 10 ul plates were made for the controls. We took the rest of the transformed cells (excluding the controls) and let them incubate for 12 hours on the shaker at 37* in 20 mL LB spec.
August 21st
Chip library assembly
Results of library transformation
Colonies grew on all the plates. However there were many colonies on the two control plates as well. There was a small amount of uncut (or nicked) plasmid according to the gel ran earlier, so this may be a result of that. Strangely, there were more colonies on the two control plates, than on the 2:1 and 3:1 10 ul plates. This does not make sense because one would expect that the controls demonstrate the amount of background, and the other plates should have more colonies on top of that in addition to the background noise.
The 5:1 plate had noticably more colonies than the others, including the control plates, making it the ideal insert:backbone for ligation.
Minipreps and glycerol stocks
We performed a 4 2 mL minipreps on the 5:1 culture, eluting in 30 ul of EB, and got the following yields:
1: 86.5 ng/ul
2: 94.0 ng/ul
3: 90.8 ng/ul
4: 94.2 ng/ul
We also made 4 glyercol stocks of the 5:1 culture, using 333 ul of 55% glycerol and 1500 ul culture.
Cross junction PCR of library colonies
In order to see whether our colonies actually had the insert, we decided to run a cross junction PCR on 9 colonies and the CB bottom plasmid (as a control). We diluted each colony in 10 ul of water and used that as our template. We had a 12.9 ul reaction volume.
Attempt at cross junction colony PCR on transformed library‎
Unfortunately, it seems as if the PCR did not work. This may be because too much template was loaded into the reaction. Next time, we plan on dliuting the colony in 50 ul of water and using 0.5 ul as template.
August 22nd
Zif268-GFP results
IPTG spec plates: the plates with IPTG did have more bacterial growth (which might just be a plating error) but none of the plates fluoresced.
Our endpoint fluorescence reads on the plate reader did not show any fluorescence, though this may have been due to an incorrect setup.
Growth curves (see graphs below):
GFP Zif strain grew in NM, showing that selection is working.
In complete media, NM, and NM+1 or 3mM 3-AT, GFP Zif grew as well as the regular Zif268 strain. However, as 3-AT concentrations increased, GFP Zif began to grow considerably less compared to Zif268. The reason for this is unclear.
There was no real difference between 0 and 10uM IPTG, but 100uM IPTG was still detrimental for cells containing zinc finger expression plasmids.
GFP Zif with 0 IPTG growth curves 8/20/11
NM media (1st replicate), all IPTG concentrations 8/20/11
Zif268 and GFP Zif 3-AT gradient, 0 IPTG 8/20/11
Wolfe system biobrick
One of our biobricks will be the Wolfe selection system (ZFB, wp, his3, ura3).  We had started putting this on pSB4K5, but a PCR test of our ligation showed that none of the plasmids had the insert.
picked 23 colonies, 10uL KAPA reactions, Spe1 ZFB_F and Xba1 URA3_R primers, 65 C annealing and 90 sec elongation
used insert containing Spe1 and Xba1 sites as positive control
pSB4K5 ZFB hisura plasmid check 8/22/11
We will not continue putting the system on pSB4K5 and instead will focus on inserting it into the pSB1C3 shipping plasmid.  We ordered the proper primers and quikchange oligos (removing two Pst1 sites) to complete the system.
Chip library assembly
Cross junction PCR: Take 2
We picked 9 more colonies from the 1 ul 5:1 plate, diluted them in 50 ul water, and used 0.5 ul as our template in a cross junction PCR. We also ran the reaction on CB bottom as a control. We followed the usual cross junction protocol with the following changes:
Initial denaturation: 6 minutes at 95* (instead of 5 minutes)
Final extension: 10 minutes at 72* (instead of 5 minutes)
Second attempt at colony PCR on chip library
It looks like none of the colonies contain an insert. In order to try and figure out what went wrong, we ran the uncut CB bottom plasmid, cut plasmid, and cut and phosphatased plasmid on a 1% e-gel.
Attempt to compare uncut, cut, and phosphatased plasmids
The gel looked extremely strange, and the ladder was not present. We tried it again, this time running uncut plasmid, new phosphatased plasmid, and old phosphatased plasmid.
Comparison of uncut plasmid, new cut and phosphotased, old cut and phosphotased
The gel isn't perferct, but it looks like there was indeed uncut plasmid present after digestion in the sample used for ligation.
Digestion of CB bottom
We decided to cut more CB bottom, and try a second ligation, only using a 5 to 1 backbone to insert ratio for the ligation. We also decided to do a double digestion with both Bsa1 and Xba1 to minimize the amount of whole plasmid left over. We used the following recipe:
4000 ng DNA (16.8 ul)
8 ul Bsa1 (40 units)
2 ul Xba1 (40 units)
10 ul 10x buffer
1 ul bsa
62.2 ul water
We let the reaction run for 1 hour at 37* and then heat inactivated the enzyme at 65* for 20 minutes. We then performed a PCR clean up on the product, eluting in 50 ul of EB and had a yield of 45.4 ng/ul. We save 2.5 ul to use as a control in the transformation.
August 23rd
Selection plates
We have shown that selection works in liquid media, but we would like to also be able to use plates with different 3-AT concentrations, since that would make it easier to pick individual colonies that grow successfully.  To check sensitivity, we plated 1uL of saturated culture of cells transformed with either all Zif268 spec, all negative control plasmids, or a mixture (10^-1 Zif, 10^-2...10^-6) on 3mM NM+3-AT plates.
Did not see successful growth, though this may be because plates were old. An attempt with 100uL showed growth on both positive and negative control plates, likely due to the LB the cells were plated with.
We now have fresh plates, so we will try again tonight.
washed 100uL of positive control (100% Zif), 10^-1, 10^-2, 10^-3, 10^-4, 10^-5, 10^-6, negative control (0% Zif)with NM (500uL) two times and plated on NM+spec
washed 100uL of EcNR2 selection strain that has not been transformed with any plasmids and plated on NM.
Changing ZF binding site
We are trying to use MAGE, but the palindromic structure of the weak promoter gave the oligos ridiculous secondary structure, so we don't expect this to work. Instead, we are going to use lambda red to switch out the current kan-ZFB construct in the cells with a new construct containing a tetracycline antibiotic cassette and ZF binding site. By tying the ZFB change to cell survival, we hope to have greater efficiency.
Plate reader
Repeated post-transformation dilutions of Zif with negative control library.
Chip library
Phosphatase of newest cut backbone
We used the following recipe to perform a phosphatase reaction on the backbone we cut yesterday with Bsa1 and Xba1:
45 ul DNA
1 ul phosphatase
5.1 ul buffer
We ran the reaction for an hour and heat inactivated the enzyme at 65* for 20 minutes. We then performed a PCR clean up on the product, with a yield of 52.6 ng/ul.
Ligation of CB bottom and library
We used the following recipe for the ligation with the insert:
100 ng backbone (1.90 ul)
2.5 ul insert (for a 5:1 ratio of backbone to insert)
2 ul 10x ligation buffer
12.6 ul water
1 ul ligase
To test the background we also ligated the cut and phosphotased plasmid, using the following recipe:
100 ng backbone (1.90 ul)
2 ul 10x ligase buffer
1 ul ligase
15.1 ul water
We added everything but the enzyme, and then heated the mixture at 37* for five minutes to separate any DNA fragment that were non specifically annealing. We then let it cool down back down to room temperature before adding the ligase.
We ligated the product for 10 minutes at room temperature. Then we cleaned up the product using min-elute columns, and eluting in 10 ul of water. We had the following yields:
With insert: 27.4 ng/ul
Control: 20.8 ng/ul
Transformation of CB bottom+library and controls
We then ran the following transformations using turbo comp cells, adding 100 ng DNA to our cells:
Cut backbone: 2.2 ul
Cut and phosphatased backbone: 1.9 ul
Cut, phosphatased, and ligated backbone: 4.81 ul
Cut, phosphatased backbone ligated with insert: 3.65 ul
Our old cut and phosphatased backbone (high background): 2.4 ul
We plated 10 and 1 ul and made 20 mL cultures of the backbone plus insert, and plated 10 ul of all others. All samples were grown in LB+spec overnight at 37*.
August 24th
Plate reader results
The cultures did not grow as well as they have in the past. This could be due to experimental error (the plate adapter was accidentally left in the reader overnight) or perhaps the cultures we inoculated with were too old. Another possibility is that the change from LB to NM-based media causes a metabolic shock, so we will try the plate reader experiment again with cultures grown up to saturation in NM+his (with the appropriate antibiotic).
Summary: the dilutions did not grow as well as previous plate reader experiments had shown. See graph below for example.
1mM 3-AT 8/23/11
Chip library
Results of yesterday's transformation
The background has significantly decreased, so it looks like adding the Xba1 helped.
There is the possiblity that the background has artificially decreased because of adding that Xba1--that is, it went down because most of the plasmids have the Xba1 cut, and not the Bsa1 cut.
However, there is not much of a difference between the control plates and the transformed plates, so it looks like our transformation did not work.
It is possible that this occured because our Bsa1 enzyme has gone bad. We used new Bsa1 when cutting the backbone yesterday. We should try re-cutting the library with new Bsa1 and transforming with that to see it helps.
Cutting the library
Ligation and transformation
Selection plates
The plates put after 16 hours in the incubator have a lawn of very small colonies growing very slowly. The selection with no plasmid is not growing on NM, as expected, but we do see colonies on the negative control library plate. However, there are clearly a lot more cells on the Zif268 positive control, and the dilutions seem to have colony amounts in between or similar to the negative control library plate.
We will try several different variations:
plate 1uL of each saturated culture on NM+spec (or plain NM in the case of the selection strain without plasmid)
plate 100uL of each washed saturated culture on NM+10mM 3-AT
===Chip library===
August 25th
Selection plates and cultures
1uL NM plates: not much is growing on even the Zif268 control. 1uL may not be enough to plate in minimal media. We will let them continue growing to see if anything shows up.
100uL 10mM 3-AT plates: looks very promising!  Zif268 has lots of small colonies, and the amount of growth steps down through the dilutions. The negative control library shows practically no growth at all. We will also keep these in the incubator to see if that helps makes the differences even clearer.
NM+his liquid cultures: none of them are as saturated as we would like, so we'll leave them in longer.
Chip library
Results of transoformation
We got colonies! We did not run a second background test yesterday, but based on the previous background plate (which contained the same backbone used in this reaction) most of our colonies should have the insert.
The best plate was the 2:1 insert to backbone ligation. The 8:1 reaction had a similar number of colonies, as well. The 3:1 and 5:1 plates had far fewer.
We performed 4 2 mL minipreps (with yields of approximately 100 ng/ul for all) on the 2:1 culture and made 4 1.5 mL glycerol stocks.
Cross junction check of library colonies
To ensure that our plates actually had the insert, we ran a cross junction PCR on 8 colonies (diluting in 50 ul water). We also performed the cross junction PCR on some whole CB bottom plasmid, to see if there is a approxmiately 80 bp difference in the gel.
We used the following recipe for the reaction:
12.5 ul water
10 ul kapa
0.75 ul forward primer
0.75 ul reverse primer
0.5 ul template
gel
The expected product size if there is an insert is about 1500 bp, while the product without the insert is about 1400 bp. It is difficult to see whether there is a difference in the sizes of the bands.
We also sent the samples out for sequencing (with the reverse sequencing primer), to confirm whether or not our ligation was successful. If it is, then we will send out 96 samples for sequencing.
August 26th
Biobricks
Zinc finger expression plasmid digests
Yesterday's digestions with LacI and LacIq did not work, so we tried again using a slightly different protocol from New England Biolab's Biobrick kit instructions .
Got the same odd doublet again
Digested one part with EcoR1 and one part with Spe1 to see if one of the enzymes would give us the expected ~50 bp fragment, but this also didn't work
We have concluded that there must be something wrong with the LacI and LacIq sequences.  Possibly one of the restriction sites is missing. We will abandon this version of the biobrick for the time being.
pSB1C3 shipping plasmid: 1st PCR cleanup failed, so we redigested using the New England Biolab's Biobrick kit instructions . Minelute produced a yield of 30 ng/uL, so a lot of DNA was lost but the concentration was still workable.
Zinc finger + omega subunit: Mineulte produced 77.8ng/uL
ZFB-hisura biobrick digests
PCR to add restriction sites to insert:
EcoXba ZFB_F and PstNotSpe URA_R primers; 10ng of Spe-ZFB-hisura-Xba as template; 25uL KAPA reactions
60C annealing and 90 sec elongation
Minelute of 4 reactions combined = 616 ng/uL
Digested with EcoR1 and Pst1 following NE Biolabs protocol
Minelute: 65ng/uL
Ligation
Followed New England Biolab's Biobrick ligation protocol for 3 conditions:
pSB1C3 and ZF+omega
pSB1C3 and ZFB-hisura
pSB1C3 self-ligation control
Transformed into chemcomp cells
Plated on LB+chloramphenicol
Selection strain and zinc finger library
Lambda red to change the binding site worked for all the sites! All had colonies growing in tet, and sequencing confirmed that the binding site was changed.
All were grown up in LB+tet and glycerol stocks were made
Since the CB bottom zinc finger library has been sequence verified, we will go ahead and transform it into the corresponding selection strain
transformed with 100ng of library plasmids
recovered for 1hr after electroporation
plated 10uL on LB+spec, 10uL and 100uL on NM+spec, and reinoculated 1mL of transformation culture into 20mL of LB+spec
Plate reader
We repeated the zif268 dilution experiment using cultures that had been grown up to saturation in NM+his
Empty selection strain, selection strain+Zif268, and selection strain+negative control library cultures were spun down, washed twice with NM, and resuspended to make and OD of 3. 1.5uL was used to inoculate each well.
Media the same as the past week's plate reader experiments.
Chip library
Sequencing results
====Sending out a plate for sequencing====
August 27th
Plate reader results
Looks great! Not all of the wells grew perfectly (perhaps a loading error?) but there is a huge difference between 10^-6 dilution and the negative controls.
All cell types in NM (2nd dilution) 8/26/11
Comparison of 10^-6 and negative controls 8/26/11
Retrieved from " http://2011.igem.org/Team:Harvard/Template:NotebookDataAugust4 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
