<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Harvard_Template_NotebookData3"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Harvard/Template:NotebookData3">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Harvard/Template:NotebookData3&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Harvard/Template:NotebookData3&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Harvard/Template:NotebookData3&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Harvard/Template:NotebookData3" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Harvard/Template:NotebookData3</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="621" style="display:none"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#June_21st"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">June 21st</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Wet_Lab"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Wet Lab</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#June_21st_-_Bioinformatics"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">June 21st - Bioinformatics</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Persikov_Statistics_-_Graphs"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Persikov Statistics - Graphs</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Phone_Call_with_Dan"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Phone Call with Dan</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Chip_Design"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Chip Design</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Identifying_dependencies"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">Identifying dependencies</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-8"><A href="#June_22nd_-_Wet_Lab"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">June 22nd - Wet Lab</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#June_22nd_-_Bioinformatics"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">June 22nd - Bioinformatics</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Final_target_sequences"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Final target sequences</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Finalizing_the_non-Zif268_backbones"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Finalizing the non-Zif268 backbones</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Updated_Chip_Design"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">Updated Chip Design</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Finishing_the_generator"><SPAN class="tocnumber">5.4</SPAN><SPAN class="toctext">Finishing the generator</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#June_23rd_-_Wet_Lab"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">June 23rd - Wet Lab</SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#June_23rd_-_Bioinformatics"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">June 23rd - Bioinformatics</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Revising_Target_Sequences"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Revising Target Sequences</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="June_21st">June 21st </SPAN></H2><H2><SPAN class="mw-headline" id="Wet_Lab">Wet Lab</SPAN></H2><P><B>His3 sequencing results:</B></P><P>The sequencing results showed that the His3 (HisB) gene is still present in the strain and without any early stop codons. There is a 2 aa deletion in the middle of the protein, but its purpose is unknown and the gene likely is still fully functional.
</P><UL><LI>Restreak selection strain on plate from glycerol stock--tomorrow we will PCR the His3 locus and sequence again just to be sure.
</LI><LI>Made oligos for MAGE to insert stop codons and make a frame shift in the endogenous His3 gene, so that if necessary we can knockout His3 ourselves.
</LI></UL><P><B>Selection strain with lambda red:</B></P><UL><LI>Reinoculated and made glycerol stock
</LI><LI>prepared for MAGE tomorrow
</LI></UL><H2><SPAN class="mw-headline" id="June_21st_-_Bioinformatics">June 21st - Bioinformatics</SPAN></H2><H3><SPAN class="mw-headline" id="Persikov_Statistics_-_Graphs">Persikov Statistics - Graphs</SPAN></H3><TABLE><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Scatterplot of top/bottom 20 with SVM polynomial</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Sequence by sequence (lin SVM)</DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Top/Bottom 20 ZFs (SVM linear)</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Comparison of polynomial vs. linear distribution (polynomial generally higher values)</DIV></DIV></DIV></TD></TR></TBODY></TABLE><UL><LI>FQCRICMRNFS<SUB>zif268 F2 Backbone</SUB>/<I><B>Helix F1</B></I>/TGEKP<SUB>linker</SUB></LI></UL><UL><LI>The Persikov data shows weak predictive power for OPEN amino acid sequences. Our conclusion is that Persikov's program is not well-suited for incorporation into our helix generator. Testing Persikov's helices in his program yeilded mostly accurate results (approximately 24/25 matched known binding information). This is an important test because it proved that we are using the program correctly and that the program is in fact working properly. However, testing the OPEN sequences in Persikov's program resulted in numerous false negative values which informed our decision not to use Persikov's program to check our own hellix-generating program.
</LI></UL><H3><SPAN class="mw-headline" id="Phone_Call_with_Dan">Phone Call with Dan</SPAN></H3><UL><LI>How conservative/risky should we be in terms of using other backbones?
</LI><LI><U><B>Conservative</B></U></LI><LI><B>Possible Pros:</B></LI><LI>More likely to get something that will work
</LI><LI>Depending on how &quot;smart&quot; our probabilities are (from our ZF generation algorithm), we could cover a lot of novel space without straying too far from zif268
</LI><LI><I>Worst Case</I>:Something we can show for iGEM (we covered the same ground OPEN did, and found many of the same ZFs, but with a targeted approach, a &quot;smarter&quot; method-- not throwing random things at it; Chip is not ours, but the program is &quot;smarter&quot;) 
</LI></UL><LI><B>Possible Cons:</B></LI><LI>Might end up covering the same ground as OPEN, but doing a &quot;worse&quot; job than they did
</LI><LI>Less likely to discover new/groundbreaking things (i.e., TNN triplets)
</LI><LI><U><B>Less Conservative</B></U></LI><LI>Have 3-6 target sequences (we're currently going for 8)
</LI><LI>More backbones from non-zif268 than zif268
</LI><LI><B>Pros:</B></LI><LI>We could get luck and find something no one has ever seen before (TNN, ANN). If we throw enough things at it, we're more likely to get luck.
</LI><LI><B>Cons:</B></LI><LI><I>Risk:</I> Many of these backbones (from entire ZF world)may NOT bind DNA (i.e., may bind proteins)
</LI><LI><I>Risk:</I> May not find anything that binds, then the whole project is a dud
</LI><LI><B>What is the more important variable, helices or backbones?</B></LI><LI>Helices seem to be more important, backbones of secondary importance
</LI><LI>Backbones: ZF's unravel DNA, open the major groove-- backbone is important here, changes the bond angle, etc. (Brandon's paper-??)
</LI><LI><I><B>Balance</B></I> needed between low and high risk
</LI><LI>If we find backbones that we know bind DNA, greatly lowers our risk
</LI><LI>Limited spaces on chip: zero-sum game
</LI><LI>With a middle of the road approach, we diminish both benefits and risk (diminishes the benefits of the high risk approach much more than it diminishes the benefits of the conservative approach; i.e., if you're playing the lottery, you're more likely to win if you buy many more tickets)
</LI><LI>We need to compare probabilities of randomly-generated OPEN sequences vs. probabilities of sequences randomly generated by our program
</LI><LI>OPEN tries to cover all space: smaller probability
</LI><LI>If we have a &quot;smarter&quot; algorithm, we can produce fewer
</LI><LI>However,  the idea is not to repeat OPEN, but to go somewhere else, non-GNN sequences
</LI><LI><I><B>Remember:</B></I> OPEN is a <I>Cell</I> paper; the point of the project is not to compare ourselves to them
</LI><LI>If we find binders for 1-2 of our sequences, that would be awesome
</LI><LI>Probably we'll have some that find none, some have 10, our last one might have 1,000 hits (then, we do bioinformatics to figure out why/what those hits were)
</LI><LI>Point: to learn and do high-level bioinformatics, and high-tech cloning techniques in the lab
</LI><LI>If you do find binders, you can write a paper about it!
</LI><LI>We have all the resources we need right now to build our chip
</LI><LI>We need to pick out targets
</LI><LI><B>Need to decide exactly what we want for:</B></LI><LI>No. of target sequences/which ones
</LI><LI>No. of helices/ which ones
</LI><LI>Ratio of zif268 backbones: non-zif268 backbones
</LI><LI>Avoid switching Leucine out of position 4, then change other positions based on our frequencies
</LI><H3><SPAN class="mw-headline" id="Chip_Design">Chip Design</SPAN></H3><UL><LI>No. of sequences will be more than we can put on the chip
</LI><LI>Helices: essentially unlimited
</LI><LI>Put more-likely-to-bind helices into the risky backbones
</LI><LI>Put less-likely-to-bind helices into a zif268 backbone
</LI></UL><LI>Backbones
</LI><LI>Maybe revert to a more targeted approach: pick backbones that we know are transcription factors (TFs), that we know bind to DNA
</LI><LI><I>OR</I> research the ZFs from the phylogenetic tree
</LI><LI>Pick clades to research, see if one looks better than the other
</LI><LI>Why did OPEN cover so many helices, without changing the backbone, but still yield predominantly GNNs?
</LI><LI>If we have an idea of how the backbone might affect binding, maybe we could look into some sort of low-level modeling, etc. so that we wouldn't be grasping? Could Vatsan help with this?
</LI><LI>See 2000 Wolfe paper <A href="http://www.ncbi.nlm.nih.gov/pubmed/10940247" class="external autonumber" rel="nofollow">[1]</A></LI><LI>Backbones <I>could</I> affect interactions between fingers
</LI><LI>Theory: energy penalty to ZF binding-- unravels DNA when binds to it
</LI><LI>We have 12 target sequences
</LI><LI>2 per 4 diseases, 4 for the 5th disease
</LI><LI>If we want to be more conservative, we could throw out Type III, but it could be something cool
</LI><LI><B>We should have mostly Type I (CoDA argument, if this is an F2)</B></LI><LI>Proposed: 3 diseases, 6 sequences
</LI><LI>4 Type I (F3 and F2 known, F1 novel)
</LI><LI>1 Type II (GNN, ANN, GNN)
</LI><LI>1 Type III (All unknown, e.g., TNN, ANN, TNN;<I><B>max 1</B></I>)
</LI><P>Or, for 3 diseases:
</P><OL><LI> Type I's
</LI><LI> Type I, Type II
</LI><LI> Type I, Type III
</LI></OL><UL><LI><B><U>Clinical Targets</U></B></LI></UL><OL><LI> Colorblindness (<B>Type I's</B>)
</LI><LI> Familial Hypercholesterolemia (FH) (1 in 500)
</LI><LI><DEL>Cystic Fibrosis (CF)</DEL></LI><LI><DEL>Tay Sachs</DEL></LI><LI> KRAS- (oncogene/cancer)
</LI></OL><UL><LI><B>Main goal of project</B>: to build outside of what is already known
</LI><LI>If we wanted to cure a disease only, we could just use existing ZFs (i.e., find GNN binding locations)
</LI><LI>Also, we lend a level of specificity for insertion/deletion
</LI><LI>There is the possibility that there might be some area where specificity might demand ANN codons
</LI></UL><P><U><B>Current decision on chip design:</B></U></P><UL><LI>We will have 6 target sequences, 2 each from colorblindness, FH, and KRAS.  All are &quot;Type I&quot; targets (only F1 is novel) with the middle finger chosen from the CODA paper (either GNN or TNN)
</LI><LI>N.B.: the CB and FH sequences make up full ZF nuclease cut sites. The KRAS sites, due to the small number of GNNTNN F3F2 combos available in CODA, are separate, with the flanking ZF nuclease site added afterwards in parentheses
</LI></UL><OL><LI> GGT<B>G</B>GT<B>A</B>AG (CB)
</LI><LI> GGA<B>G</B>TC<B>C</B>TG (FH)
</LI><LI> GGC<B>T</B>GA<B>T</B>GC (KRAS) (CTGAAAATT)
</LI><LI> GGC<B>T</B>GA<B>C</B>AC (FH)
</LI><LI> GGC<B>T</B>GG<B>A</B>AT (KRAS) (GACAAGAGC)
</LI><LI> GTC<B>G</B>CC<B>T</B>CC (CB)
</LI></OL><UL><LI>Targets 3, 4, and 6 are similar to sequences Zif268 variants successfully bind to, so the backbones will be weighted accordingly:
</LI><LI>Zif268_F2 backbone: 6000 helices (per target)
</LI><LI>10 backbones more closely related to Zif268: 300 helices each
</LI></UL><LI>Targets 1, 2, and 5 will have equal distributions of backbones:
</LI><LI>Zif268_F2: 3000 helices
</LI><LI>10 backbones closely related to Zif268: 300 each
</LI><LI>10 backbones more distantly related to Zif268: 300 each
</LI><H3><SPAN class="mw-headline" id="Identifying_dependencies">Identifying dependencies</SPAN></H3><UL><LI>We looked at the <A href="#Probability_data">probability graphs</A> to determine which amino acid positions on the finger's helix interact with which bases. 
</LI><LI>Some interactions are fairly well estabilished, while others have been more recently proposed (See <A href="#June_17_-_Bioinformatics">interaction map (Persikov 2011)</A>)
</LI><LI>To identify these interactions in our own data we looked at which helix positions varied most when you changed the bases. A more rigorous way to do this is to calculate the entropy change as you change the amino acids in each position.  
</LI><LI><B>xNN</B>(Vary base 1): Amino acid 6 changes 
</LI><LI><B>NxN</B>(Vary base 2): Amino acid 3 changes
</LI><LI><B>NNx</B>(Vary base 3): Amino acid -1 and 2(?) changes 
</LI></UL><LI>Our program looks at dependencies between amino acids when generating sequences. 
</LI><LI>We decided on these amino acid dependencies, using both established data and patterns we saw in the OPEN data:
</LI><LI>-1 and 2
</LI><LI>2 and 1
</LI><LI>6 and 5
</LI><LI>Because there is not much data for 'CNN' and 'ANN' sequences (with 16 and 29 known fingers that bind to each triplet, respectively), we should use pseudocounts for these sequences, so that our frequency generator is not too biased toward probabilities that may not be significant.</LI></DIV><DIV id="622" style="display:none"><H2><SPAN class="mw-headline" id="June_22nd_-_Wet_Lab">June 22nd - Wet Lab</SPAN></H2><P><B>Preparing media/reagents for selection system:</B></P><UL><LI>Made 0.1M zinc chloride solution, M9 salt, and 1M magnesium sulphate solutions for the amino acid mixture
</LI><LI><B>M9 Salt solution (20x)</B></LI><LI>67.8 g of disodium phosphate
</LI><LI>30 g of monopotassium phosphate
</LI><LI>5 g of sodium chloride
</LI><LI>10 g of ammonium chloride
</LI><LI>All in 500 mL of distilled water
</LI><LI>Sterile filtered when done dissolving
</LI></UL><P><B>Overhang PCR for 3-part assembly of ZFs, omega subunit, and backbone vector (pZE21G, spec resistance)</B></P><UL><LI>Clone out omega+Zif268:
</LI><LI>template: original selection construct plasmid (ZFB, his3, etc.)
</LI><LI>primers: omega_F+homolog, Zif268_R+homolog
</LI><LI>Protocol
</LI><LI>98 C for 30 sec
</LI><LI>98 C for 10 sec
</LI><LI>68 C for 30 sec
</LI><LI>72 C for 30 sec
</LI><LI>Repeat steps 2-4, 30 times
</LI><LI>4  C for ever
</LI></UL><LI>Clone out omega only:
</LI><LI>template: original selection construct
</LI><LI>primers: omega_F+homolog, omega_R
</LI><LI>Protocol
</LI><LI>Same as omega + Zif268
</LI><LI>Clone OZ052 with overhang:
</LI><LI>template: OZ052 overhang (overhang currently matches selection construct), 1:10b, 15ng/ÂµL
</LI><LI>primers: OZ052_F+omega homolog, OZ052_R+homolog
</LI><LI>Protocol
</LI><LI>Same as omega + Zif268
</LI><LI>Clone OZ123 with overhang:
</LI><LI>template: OZ123 overhang (overhang currently matches selection construct), 1, 6.3ng/ÂµL
</LI><LI>primers: OZ123_F+omega homolog, OZ123_R+homlog
</LI><LI>Protocol
</LI><LI>Same as omega + Zif268
</LI><LI>clone out pZE21G backbone
</LI><LI>template: pZE21G containing cells from plate, diluted 1:10
</LI><LI>primers: back_F, back_R
</LI><LI>Protocol
</LI><LI>98 C for 30 sec
</LI><LI>98 C for 10 sec
</LI><LI>68 C for 30 sec
</LI><LI>72 C for 1:30
</LI><LI>Repeat steps 2-4, 30 times
</LI><LI>72 C for 5 min
</LI><LI>4  C for ever
</LI><LI>25ÂµL reaction:
</LI><LI>12.5ÂµL Phusion mastermix
</LI><LI>1.25ÂµL each primer
</LI><LI>1ÂµL of 1ng/ÂµL dilution of template
</LI><LI>9ÂµL of ddH2O
</LI><LI>Ran gel of the above PCR products and imaged below: only the omega and omega+Zif268 reactions seemed to work
</LI><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">PCR for 3-part ZF assembly</DIV></DIV></DIV><P><B>pZE21G backbone:</B></P><UL><LI>1 colony of pZE21G grown in 3mL LB, 3ÂµL spectinomycin (1000x) until mid-log. Glycerol stock made.
</LI><LI>Miniprep of pZE21G in order to PCR the backbone: used Qiagen kit
</LI><LI>5ng/ÂµL and didn't seem pure: miniprep (or nanodrop) not working
</LI><LI>another colony used to start an overnight culture for PCR/glycerol stocks
</LI></UL><LI>Ran gradient PCR on the miniprep product in order to obtain backbone
</LI><LI>same protocol as before, with 1ÂµL of template
</LI><LI>6 tubes spaced so that annealing temp=60,62,64,66,68,70
</LI><LI>Parameters: (program on PCR5, IGEM-&gt; DOGGED)
</LI><LI>98Â°C for 30s (initial denaturation)
</LI><LI>98Â°C for 10s (denature)
</LI><LI>60Â°C to 71Â°C for 15s (anneal)
</LI><LI>72Â°C for 90s (extend)
</LI><LI>Repeat steps 2-4 for 30 cycles total (denature, anneal, extend)
</LI><LI>72Â°C for 5 min
</LI><LI>4Â°C forever
</LI><P><B>HisB locus PCR:</B> We repeated the PCR of the selection strain at this locus just to be sure HisB is still present
</P><UL><LI>grew 1 colony from a new selection strain plate Vatsan brought in 0.5mL LB, 0.5ÂµL tet
</LI><LI>used 1 ÂµL of bacterial suspension for PCR following same procedure as 6/20
</LI></UL><P><B>Lambda red to make selection system:</B></P><UL><LI>grewÂ ?His3?PyrF?rpoZ+pKD46 to mid-log (0.4 using OD)
</LI><LI>induced lambda red by shaking culture in 42C water bath for 15 min
</LI><LI>spin down 1mL for 1 min, 18000 rcf at 4C
</LI><LI>wash 2x with cold water, removing as much supernatant as possible
</LI><LI>resuspend with 200 ng kan-ZFB-wp-his-ura template (20ÂµL) and water up to 50ÂµL (30ÂµL)
</LI><LI>electroporate using 1mm gap cuvettes adn 1.80KV. Immediately afterward add 1mL LB to cuvette, mix, and transfer to culture tube containing 2 mL more of LB
</LI><LI>recover for 2hrs, 30C
</LI><LI>spread on kanamycin plates: 100ÂµL, 10ÂµL, or 1ÂµL (the last two dilute with 100ÂµL LB to help spread more easily)
</LI><LI>grow overnight at 30C
</LI></UL><H2><SPAN class="mw-headline" id="June_22nd_-_Bioinformatics">June 22nd - Bioinformatics</SPAN></H2><H3><SPAN class="mw-headline" id="Final_target_sequences">Final target sequences</SPAN></H3><P>Our &quot;tentatively&quot; Final DNA Target Sequences (i.e. barring any major objections, we're going with this):
</P><TABLE class="wikitable" cellpadding="5"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Disease</B></TD><TD align="center" style="background:#f0f0f0;"><B>Target Range</B></TD><TD align="center" style="background:#f0f0f0;"><B>Binding Site Location</B></TD><TD align="center" style="background:#f0f0f0;"><B>Bottom Finger</B></TD><TD align="center" style="background:#f0f0f0;"><B>Top Finger</B></TD><TD align="center" style="background:#f0f0f0;"><B>Bottom AA (F3 to F1)</B></TD><TD align="center" style="background:#f0f0f0;"><B>Top AA (F3 to F1)</B></TD></TR><TR><TD> Colorblindness</TD><TD>chrX:153,402,679-153,408,753</TD><TD>256</TD><TD style="background:#92D050">GGC TGA GGC</TD><TD>GTA GCT GGG</TD><TD>ESGHLKR.QREHLTT.#######</TD><TD>QSGTLTR.QRSDLTR.KKDHLHR
</TD></TR><TR><TD> Colorblindness</TD><TD>chrX:153,402,679-153,408,753</TD><TD>2067</TD><TD>GAA GGG GAC</TD><TD style="background:#92D050">GGG GCT CAC</TD><TD>QDGNLGR.RREHLVR.EEANLRR</TD><TD>RTEHLAR.QRSDLTR.#######
</TD></TR><TR><TD> Familial Hypercholesterolemia</TD><TD>chr19:11,175,000-11,195,000</TD><TD>2707</TD><TD style="background:#92D050"> GGC TGG ATG</TD><TD style="background:#92D050"> GGC TGG CTC*</TD><TD>ESKHLTR.RREHLTI.#######</TD><TD>ESKHLTR.RREHLTI.#######
</TD></TR><TR><TD> Pancreatic Cancer</TD><TD>chr7:117,074,084-117,089,556</TD><TD>4423</TD><TD style="background:#92D050"> GCA GAC TGT</TD><TD style="background:#92D050"> GCA GGA AAA</TD><TD>QGNTLTR.DRGNLTR.#######</TD><TD>QDVSLVR.QSAHLKR.#######
</TD></TR></TBODY></TABLE><UL><LI>Drier was unable to find a ZF that bound specifically to CTC. Instead he found zinc fingers that bound to CTC and other sequences with equal binding affinity.  
</LI><LI>Note: The green cells are the target sequences that we are aiming for on our chip.
</LI></UL><H3><SPAN class="mw-headline" id="Finalizing_the_non-Zif268_backbones">Finalizing the non-Zif268 backbones</SPAN></H3><P>In addition, we locked down the non-Zif268 backbones that we will be using for the chip.  We have 10 backbones that are more closely related to Zif268, and 10 that are more distantly related:
</P><TABLE><TBODY><TR><TD align="center" colspan="2" style="background:#f0f0f0;"><B>More Closely Related Backbones</B></TD><TD style="background:#f0f0f0; color: #f0f0f0;">Â Â Â Â Â </TD><TD align="center" colspan="2" style="background:#f0f0f0;"><B>More Distantly Related Backbones</B></TD></TR><TR><TD align="center" style="background:#f0f0f0;"><B>Name</B></TD><TD align="center" style="background:#f0f0f0;"><B>Sequence (with helix)</B></TD><TD align="center" style="background:#f0f0f0;"><B>Name</B></TD><TD align="center" style="background:#f0f0f0;"><B>Sequence (with helix)</B></TD></TR><TR><TD> 44GLAS_DROME</TD><TD style="font-family: monospace, sans-serif"> FRCPI---CDRRFSQSSSVTTH-MRTH--</TD><TD>56EGR1_HUMAN</TD><TD style="font-family: monospace, sans-serif"> FAC---DICGRKFARSDERKRHTKIH---
</TD></TR><TR><TD> 38KRUP_DROME</TD><TD style="font-family: monospace, sans-serif"> FTCKI---CSRSFGYKHVLQNH-ERTH--</TD><TD>47MZF1_HUMAN</TD><TD style="font-family: monospace, sans-serif"> FVC---GDCGQGFVRSARLEEHRRVH---
</TD></TR><TR><TD> 124EVI1_HUMAN</TD><TD style="font-family: monospace, sans-serif"> YRC---KYCDRSFSISSNLQRHVRNIH--</TD><TD>23CF2_DROME</TD><TD style="font-family: monospace, sans-serif"> YTC---SYCGKSFTQSNTLKQHTRIH---
</TD></TR><TR><TD> 6HUNB_DROME</TD><TD style="font-family: monospace, sans-serif"> YECK---YCDIFFKDAVLYTIHMGY--H-</TD><TD>19ZEP2_RAT</TD><TD style="font-family: monospace, sans-serif"> YICE---ECGIRCKKPSMLKKHIRTH---
</TD></TR><TR><TD> 16SUHW_DROME</TD><TD style="font-family: monospace, sans-serif"> FPCEQ---CDEKFKTEKQLERH-VKTH--</TD><TD>49SDC1_CAEEL</TD><TD style="font-family: monospace, sans-serif"> VVC---FHCG-TRCHYTLLHDHLDYCH--
</TD></TR><TR><TD> 125CF2_DROME</TD><TD style="font-family: monospace, sans-serif"> YTC---PYCDKRFTQRSALTVHTTKLH--</TD><TD>27SDC1_CAEEL</TD><TD style="font-family: monospace, sans-serif"> LTC---AHCDWSFDNVMKLVRH-RGVH--
</TD></TR><TR><TD> 43EVI1_HUMAN</TD><TD style="font-family: monospace, sans-serif"> FKCHL---CDRCFGQQTNLDRH-LKKH--</TD><TD>130TTKB_DROME</TD><TD style="font-family: monospace, sans-serif"> YRC---KVCSRVYTHISNFCRHYVTSH--
</TD></TR><TR><TD> 118ADR1_YEAST</TD><TD style="font-family: monospace, sans-serif"> YPC---GLCNRCFTRRDLLIRHAQKIH--</TD><TD>80ESCA_DROME</TD><TD style="font-family: monospace, sans-serif"> YQC---PDCQKSYSTFSGLTKH-QQFH--
</TD></TR><TR><TD> 24EVI1_HUMAN</TD><TD style="font-family: monospace, sans-serif"> QECK---ECDQVFPDLQSLEKHMLS--H-</TD><TD>20IKZF1_MOUSE</TD><TD style="font-family: monospace, sans-serif"> HKCG---YCGRSYKQRSSLEEHKERCH--
</TD></TR><TR><TD> 25SUHW_DROME</TD><TD style="font-family: monospace, sans-serif"> MSCKV---CDRVFYRLDNLRSH-LKQH--</TD><TD>127SRYD_DROME</TD><TD style="font-family: monospace, sans-serif"> QECTT---CGKVYNSWYQLQKHISEEH--
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Updated_Chip_Design">Updated Chip Design</SPAN></H3><P>The CODA article produced zinc fingers that bound a GNN or TNN F2 with either a ANN, GNN, or TNN F3. These results lead us to the following distribution of three types of zinc finger backbones (Zif268, similar but not equal to Zif268, and dissimilar to Zif268) across our 6 target DNA sequences. With 55,000 spaces on our chip, each of the 6 target DNA sequences is allotted 9,150 spaces with 100 spaces set aside for control zinc fingers from CODA and OPEN. Note that the values in the table below represent the number of helices inserted into each type of backbone.
</P><TABLE class="wikitable" cellpadding="5"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Disease</B></TD><TD align="center" style="background:#f0f0f0;"><B>Target DNA Finger 2</B></TD><TD align="center" style="background:#f0f0f0;"><B>Target DNA Finger 1</B></TD><TD align="center" style="background:#f0f0f0;"><B>Helices in Zif268 Backbone</B></TD><TD align="center" style="background:#f0f0f0;"><B>Helices in Zif268 Closely-Related Backbones</B></TD><TD align="center" style="background:#f0f0f0;"><B>Helices in Zif268 Distantly-Related Backbones</B></TD></TR><TR><TD> Colorblindness</TD><TD>TNN</TD><TD>GNN</TD><TD>5150</TD><TD>3000</TD><TD>1000
</TD></TR><TR><TD> Colorblindness</TD><TD>GNN</TD><TD>CNN</TD><TD>3050</TD><TD>3050</TD><TD>3050
</TD></TR><TR><TD> Familial Hypercholesterolemia</TD><TD>TNN</TD><TD>ANN</TD><TD>3050</TD><TD>3050</TD><TD>3050
</TD></TR><TR><TD> Familial Hypercholesterolemia</TD><TD>TNN</TD><TD>CNN</TD><TD>3050</TD><TD>3050</TD><TD>3050
</TD></TR><TR><TD> Pancreatic Cancer</TD><TD>GNN</TD><TD>TNN</TD><TD>5150</TD><TD>3000</TD><TD>1000
</TD></TR><TR><TD> Pancreatic Cancer</TD><TD>GNN</TD><TD>ANN</TD><TD>3050</TD><TD>3050</TD><TD>3050
</TD></TR></TBODY></TABLE><P>N.B.: The chip will only be holding our F1 zinc fingers- the F2 and F3 will be on a separate plasmid that we must make ourselves
</P><P><I><B>To Do:</B></I> The distribution of helices to each backbone set/target sequence needs to be finalized. For example, the program can generate a set of helices for the Zif268 backbone to be applied to the colorblindness target sequence, but should the same set or a completely different helix set be applied to the Zif268 backbones for the familial hypercholesterolemia target sequences?
</P><UL><LI>If we want to test the effect of the backbone, would need to keep the helices constant-- but we could do this within a single target, to keep all other variables constant
</LI></UL><H3><SPAN class="mw-headline" id="Finishing_the_generator">Finishing the generator</SPAN></H3><P>We finished and finalized the program that generates zinc finger sequences. The following changes were made today:
</P><UL><LI>We incorporated the data from Persikov's database into out generator. 
</LI><LI>We included the ability to remove duplicate sequences in the output file (with a dictionary). 
</LI><LI>We added pseudocounts for fingers that bind to 'ANN' or 'CNN' targets. Because there is not much information for these targets, the data we do have may be biased. Thus, we want to make sure that amino acids that currently have no probability of occurring are bumped up to a minimum (currently 0.01).
</LI><LI>We placed additional weight on non zif-268 backbones. Formerly, the amino acids for positions 1, 4, and 5 were fixed based on zif-268 data, regardless of the original helix sequences on these backbones. Now information from both zif-268 and the original helix sequence is considered when assigning weights to the amino acids. 
</LI><LI>We started working with Noah's reverse translate program. 
</LI></UL><P>
We tested our generator to ensure that the sequences it was producing appeared to be legitimate.
</P><UL><LI>Jamie looked at the multiple sequence alignments of the fingers generated, so see if the frequencies correlated with what we expected them to be. (?)
</LI><LI>We input a known DNA triplet to see if the program generated sequences known to bind, according to OPEN data. When generating 10000 sequences, about half the known binders for the input triplet were found.
</LI></UL><P>
We created <A href="http://weblogo.berkeley.edu/" class="external text" rel="nofollow">WebLogos</A> to more easily visualize how adding the Persikov data affects the sequences we generate. The size of the letters correspond to the frequency of that amino acid in that position. We decided to incorporate the Persikov data so that our generator incorporates more information when generating sequences. Doing so does not drastically change the sequences generated. 
</P><TABLE><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">WebLogo for the OPEN data.</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">WebLogo for the Persikov data.</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">WebLogo for 10000 sequences generated with our program, when it incorporates only OPEN data.</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">WebLogo for 10000 sequences generated with our program, when it incorporates both OPEN and Persikov data.</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">WebLogo for fingers that bind to ANN according to OPEN data.</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">WebLogo for 10000 sequences generated for an ANN triplet with our program, when it incorporates only OPEN data.</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">WebLogo for 10000 sequences generated for an ANN triplet with our program, when it incorporates both OPEN and Persikov data.</DIV></DIV></DIV></TD></TR></TBODY></TABLE></DIV><DIV id="623" style="display:none"><H2><SPAN class="mw-headline" id="June_23rd_-_Wet_Lab">June 23rd - Wet Lab</SPAN></H2><P><B>Ran gel to determine the results of the PCR products</B></P><UL><LI>Determined the HisB presence in selection strain
</LI><LI>Finalized the presence of hisB through the gel image below
</LI></UL><LI>Determined the success of the pZE21G backbone primers through gel on gradient PCR
</LI><LI>PCR for the backbone failed again, even done through gradient PCR
</LI><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">PCR of HisB locus</DIV></DIV></DIV><P><B>PCR more Kan-ZFB-His3-Ura3</B></P><UL><LI>Two 50 ÂµL reactions
</LI><LI>Doubled the protocol used on the 16th and used HisUraKan_F and ZFBwpHisUra_R primers
</LI><LI>PCR produced very low concentration of Kan-ZFG-His3-Ura3 because melting temperature of primer was too high, so primers stuck to annealing DNA and did not dissociate
</LI><LI>In image below the low concentration of desired product can be seen, along with the high concentration of unused primers
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">PCR of kan-ZFB-wp-his3-ura3</DIV></DIV></DIV><UL><LI>Ran gel of initial overlap PCR product (undiluted, starting without primers) from June 16th, using whole sample
</LI><LI>Performed gel extraction in order to have more Kan-ZFB-His3-Ura3 product for the transformation tomorrow: 8.5ng/ÂµL, 260/280=2.03 
</LI></UL><P><B>Determined success of the selection construct transformation</B></P><UL><LI>Checked the plates all day,and finally came to the conclusion that the transformation did not work
</LI><LI>Discovered that lambda red has a promoter induced by arabinose, not temperature (though the strain is still temperature sensitive). That is why it didn't work--we'll get arabinose and hopefully have a successful recombination.
</LI><LI>Preparing all parts for transformation today and will finish it tomorrow
</LI></UL><P><B>Oligo Design for MAGE</B></P><UL><LI>Designed 90bp long oligos for OZ052 and OZ123 insertion in the ZFB sites in place of Zif268. Reverse complement taken.
</LI></UL><P><B>Miniprep pZE21G plasmid for backbone PCR</B></P><UL><LI>Ran miniprep again for pZE21G plasmid: 6ng/ÂµL, 260/280=2
</LI><LI>Worried that the miniprep didn't work: ran gel also on this miniprep and concluded that DNA was present in the sample
</LI><LI>Gel image seen below
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">pZE21G miniprep</DIV></DIV></DIV><P><B>Ran PCR for pZE21G backbone</B></P><UL><LI>Used same protocol from June 22 in PCR today for pZE21G backbone
</LI></UL><H2><SPAN class="mw-headline" id="June_23rd_-_Bioinformatics">June 23rd - Bioinformatics</SPAN></H2><H3><SPAN class="mw-headline" id="Revising_Target_Sequences">Revising Target Sequences</SPAN></H3><TABLE class="wikitable" border="1" cellpadding="5"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Target DNA</B></TD><TD align="center" style="background:#f0f0f0;"><B>Cystic Fibrosis</B></TD><TD align="center" style="background:#f0f0f0;"><B>Familial Hypercholesterolemia</B></TD><TD align="center" style="background:#f0f0f0;"><B>Retinal Blastoma</B></TD><TD align="center" style="background:#f0f0f0;"><B>p53</B></TD><TD align="center" style="background:#f0f0f0;"><B>Myc</B></TD><TD align="center" style="background:#f0f0f0;"><B>Pancreatic Cancer</B></TD></TR><TR><TD> GNN A</TD><TD><B>Flank 1</B></TD><TD>?
</TD></TR><TR><TD> GNN T</TD><TD><B>Flank 1</B></TD></TR><TR><TD> GNN C</TD><TD>?</TD><TD><B>Flank 2</B></TD></TR><TR><TD> TNN G</TD><TD><B>Flank 2</B></TD><TD>X</TD></TR><TR><TD> TNN C</TD><TD>Flank 3</TD><TD><B>?</B></TD></TR><TR><TD> TNN A</TD><TD>Flank 3</TD><TD><B>?</B></TD></TR></TBODY></TABLE><P>
This is the set of final, final target sequences based on the table above:
</P><TABLE class="wikitable" cellpadding="5"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Disease</B></TD><TD align="center" style="background:#f0f0f0;"><B>Target Range</B></TD><TD align="center" style="background:#f0f0f0;"><B>Binding Site Location</B></TD><TD align="center" style="background:#f0f0f0;"><B>Bottom Finger</B></TD><TD align="center" style="background:#f0f0f0;"><B>Top Finger</B></TD><TD align="center" style="background:#f0f0f0;"><B>Bottom AA (F3 to F1)</B></TD><TD align="center" style="background:#f0f0f0;"><B>Top AA (F3 to F1)</B></TD></TR><TR><TD> Colorblindness</TD><TD>chrX:153,403,001-153,407,000</TD><TD>3627</TD><TD style="background:#92D050">GCT GGC TGG </TD><TD style="background:#92D050"> GCG GTA ATG</TD><TD>EGSGLKR.EAHHLSR.#######</TD><TD>RRDDLTR.QRSSLVR.#######
</TD></TR><TR><TD> Familial Hypercholesterolemia</TD><TD>chr19:11,175,000-11,195,000</TD><TD>14001</TD><TD style="background:#92D050"> GGC TGA GAC</TD><TD style="background:#92D050"> GGA GTC CTG</TD><TD>ESGHLKR.QREHLTT.#######</TD><TD>QTTHLSR.DHSSLKR.#######
</TD></TR><TR><TD> Myc-gene Cancer</TD><TD>chr8:128,938,529-128,941,440</TD><TD>198</TD><TD>GGT GCA GGG</TD><TD style="background:#92D050"> GGC TGA CTC</TD><TD>VDHHLRR.QSTTLKR.RRAHLQN</TD><TD>ESGHLKR.QREHLTT.#######
</TD></TR><TR><TD> Myc-gene Cancer</TD><TD>chr8:128,938,529-128,941,440</TD><TD>981</TD><TD>GGA GAG GGT</TD><TD style="background:#92D050"> GGC TGG AAA</TD><TD>QANHLSR.RQDNLGR.TRQKLET</TD><TD>EKSHLTR.RREHLTI.#######
</TD></TR></TBODY></TABLE><UL><LI>Green cells are our target sequences.</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Harvard/Template:NotebookData3">http://2011.igem.org/Team:Harvard/Template:NotebookData3</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Harvard/Template:NotebookData3" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Harvard/Template:NotebookData3" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Harvard/Template:NotebookData3&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Harvard/Template:NotebookData3&amp;oldid=147374" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>