<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Harvard_Brainstorming"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Harvard/Brainstorming">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Harvard/Brainstorming&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Harvard/Brainstorming&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Harvard/Brainstorming&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Harvard/Brainstorming" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Harvard/Brainstorming</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><TITLE>bar</TITLE></P></DIV><DIV class="whitebox"><H1><SPAN class="mw-headline" id="Bioinformatics_Brainstorming_Notes">Bioinformatics Brainstorming Notes</SPAN></H1><H2><SPAN class="mw-headline" id="June_6th"> June 6th </SPAN></H2><UL><LI>While many different zinc fingers exist naturally in organisms, the human transcription factor zinc finger zif268 (Fig.1) has undergone the most extensive characterization and manipulation. Consequently, zif268 has shaped the interaction between synthetic biology and zinc fingers more than any other single zinc finger protein.
</LI><LI>Zif268 displays the Cys<SUB>2</SUB>His<SUB>2</SUB> motif (Fig.2) characteristic of many zinc fingers
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><A href="/File:Zif268_wDNA.png" title="File:Zif268 wDNA.png">File:Zif268 wDNA.png</A><DIV class="thumbcaption"><B>Fig.1</B> Crystal structure of zif268, shown bound to its DNA recognition sequence &lt;ref&gt;<A href="http://en.wikipedia.org/wiki/Zif268" class="external free" rel="nofollow">http://en.wikipedia.org/wiki/Zif268</A>&lt;/ref&gt;</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><A href="/File:C2H2_motif.png" title="File:C2H2 motif.png">File:C2H2 motif.png</A><DIV class="thumbcaption"><B>Fig.2</B> Characteristic Cys<SUB>2</SUB>His<SUB>2</SUB> motif &lt;ref&gt;<A href="http://en.wikipedia.org/wiki/Zinc_finger" class="external free" rel="nofollow">http://en.wikipedia.org/wiki/Zinc_finger</A>&lt;/ref&gt;</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="June_13th">June 13th</SPAN></H2><H3><SPAN class="mw-headline" id="Functions_to_design:">Functions to design:</SPAN></H3><UL><LI> generate(matrix, pseudocounts (lambda), dependency tuples)
</LI></UL><DL><DD> takes a matrix of zinc-finger AA position counts, a list of dependent amino acid pairs, and a pseudocount multiplier and generates a list of potential amino acid sequences weighted by independent and dependent probabilities
</DD></DL><UL><LI> add_pseudo(dependent matrix row,independent matrix row)
</LI></UL><DL><DD> given a matrix row of dependent counts (i.e. how many times 'a' occurs at position n when 'b' is set to some AA at position m) and a row of independent matrix counts (how many times 'a' occurs at n regardless of b's AA) return an adjusted matrix row, based on the dependent matrix row, that has pseudocounts added to the values that are empty in the dependent matrix row but filled in the independent matrix row. 
</DD></DL><UL><LI> generate_indep(matrix)
</LI></UL><DL><DD> randomly pick an amino acid, given a matrix row, from a weighted random distribution based on the values in the row
</DD></DL><UL><LI> generate_dep(indep_row, dep_row, lambda)
</LI></UL><DL><DD> add pseudo counts (call add_pseudo) and generate a dependent random call for a position (using generate_indep on the adjusted matrix)
</DD></DL><H3><SPAN class="mw-headline" id="New_Triplet_List_File">New Triplet List File</SPAN></H3><P>I created a new triplet list from the zfDB table that we got. Its columns are:
</P><PRE>Triplet	Upstream	Downstream	Helix Sequence	F Number
</PRE><P>Upstream and Downstream are the two bases in those directions. One note of possible confusion; because the triplets are in reverse order, downstream is the next two bases in the following open triplet, and upstream is the previous two bases at the end of the previous open triplet. If you were reading 5' to 3', these would be reversed. 
</P><P>For future reference (mostly mine), the regex I used is: 
</P><PRE>find: \d+\t\d+\t\d+\t\d+\t(\w)(\w)(\w)\t(\w)(\w)(\w)\t(\w)(\w)(\w)\t(\w+)\t(\w+)\t(\w+)
</PRE><PRE>replace: $1$2$3\t\t$4$5\t$10\t1\n$4$5$6\t$2$3\t$7$8\t$11\t2\n$7$8$9\t$5$6\t\t$12\t3
</PRE><H3><SPAN class="mw-headline" id="Persikov_2009_SVM">Persikov 2009 SVM</SPAN></H3><P>There's a paper in
particular that I think would be useful for you to read, its <B>2009 Persikov</B> on the dropbox. One of the things we need to do is get the SVM up
and running with SVMlight. Check out the instructions here:
</P><P><A href="http://compbio.cs.princeton.edu/zf/help.html" class="external free" rel="nofollow">http://compbio.cs.princeton.edu/zf/help.html</A></P><P>It goes along with the persikov 2009 paper.
</P><H3><SPAN class="mw-headline" id="Input_example_for_SVM:">Input example for SVM:</SPAN></H3><P>testin.txt
</P><P>0 3:1 59:1 94:1 318:1 # abcdef
</P><P>command line:
</P><P>./svm_classify testin.txt SVMp.mod testout.txt
</P><P>testout.txt
</P><P>2.5055193
</P><P>The first input column states that this is a row to be classified.  The 4 following columns indicate amino acid to base relationships as described in the model's README file:
</P><P>Conversion table from SVM features to amino acid - nucleotide contacts. 
Canonials contacts are listed as (see Fig 1 at Persikov et al., 2008):
</P><P>01 - between amino acid a6 and nucleotide b1
</P><P>02 - between amino acid a3 and nucleotide b2
</P><P>03 - between amino acid a-1 and nucleotide b3
</P><P>04 - between amino acid a2 and nucleotide b4
</P><P>
1 01Aa
</P><P>2 01Ac
</P><P>3 01Ag
</P><P>4 01At
</P><P>5 01Ca
</P><P>
316 04Wt
</P><P>317 04Ya
</P><P>318 04Yc
</P><P>319 04Yg
</P><P>320 04Yt
</P><P>The first line in the models must be changed from:
</P><P>SVM-light Version V6.01 to SVM-light Version V6.02
</P><H4><SPAN class="mw-headline" id="Zinc_Finger_Backbones">Zinc Finger Backbones</SPAN></H4><P><A href="http://compbio.cs.princeton.edu/zf/sources.html" class="external free" rel="nofollow">http://compbio.cs.princeton.edu/zf/sources.html</A></P><P><A href="http://www.ebi.ac.uk/interpro/IEntry?ac=IPR007087" class="external free" rel="nofollow">http://www.ebi.ac.uk/interpro/IEntry?ac=IPR007087</A></P><P>Zif268 Middle Finger with no adjacent linker:
</P><P>FQCRICMRNFS_RSDHLTTHIR_TH
</P><H3><SPAN class="mw-headline" id="Type_IIs_Design">Type IIs Design</SPAN></H3><P>Good points Dan, hopefully there is still time to address. Absolutely we need positive controls on the chip, lets make sure those make it on there.
</P><P>I've done some legwork on using they Type IIs cutters to do the plasmid assembly. Since I'm going to be out of town tomorrow and over the weekend (back sometime Sunday), I thought it was best to share this with you guys now.
</P><P>The strategy would be to use GAAG as the 5 overhang site and CCGT as the 3' overhang site. Note that all overhangs are 5' overhangs on their strand, since there are no type IIs cutters listed on NEB that have short recognition sites and more than 1 base 3' overhang. I had initially wanted to have one end be 5' overhangs, and the other end 3' overhangs to prevent the vector from self-ligating non-specifically during assembly, but this may not be possible. 
</P><P>5' side of insert = gAAG overhang, with the AAG encoding the C-terminal most residue (lysine) of the upstream part. I realize this ends up adding an extra lysine at the omega subunit-F1 junction, but this is unlikely to make the linker unusable, and allows us to use our library sequences as F2 or F3 fingers, if we ever desired to. This is because the C-terminal residue of ZiF268 F2 (which we will use for non-library fingers) is K, and the g in the gAAG can be the last base of omega subunit (threonine) or the glutamic acid upstream of lysine in ZiF268 F2.
</P><P>3' side of insert = CCGt, with CCG encoding the N-terminal proline of the next finger (or adding an extra proline before the stop codon, if used as an F3). 
</P><P>If there are strong objections to adding a K between omega and finger 1, we can also use a 5'-NNN overhang-producing type IIs cutter and put the 5' overlap in the c-terminal threonine of omega subunit, and the 3' overlap in the n-terminal proline of F2. However, this prevents us from using the library members as anything but F1 (unless we add a threonine between fingers, possibly). 
</P><P>Here I've used BsaI enzyme since it has a 6 bp recognition site not found in omega subunit or ZiF268, but there are several dozen other 5'-NNN or 5'-NNNN type IIs cutters on NEB alone.
</P><P>Other options are to use USER cloning, which means we have to do separate PCRs for each backbone, and then mix them in equal ratios before inserting them. This may be more efficient than the ligation based approach, but means more PCR work that can also possibly introduce bias (since USER primers are annealing to different sequences). 
</P><P>If this is all terribly confusing, I apologize, and here are some Seqbuilder files that might be helpful (also in the dropbox under Sequences/chip sequence files).
</P><P>Good luck finalizing the target sequences and the outputting sequences.
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Harvard/Brainstorming">http://2011.igem.org/Team:Harvard/Brainstorming</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Harvard/Brainstorming" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Harvard/Brainstorming" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Harvard/Brainstorming&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Harvard/Brainstorming&amp;oldid=63736" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>