"
Page
Discussion
View source
History
teams
Log in
 
Team:Harvard/Template:NotebookDataAugust
From 2011.igem.org
Contents
1 August 1st
1.1 Team Wolfe
1.2 Team ZF
1.2.1 Restriction enzyme cut test, take 3
1.2.2 Redoing the isothermal assembly/transformation for Pos con 77 and the 3 controls
1.2.3 Practice plate library assembly
1.3 Team TolC
1.3.1 Lambda red recombination of zeocin into ECNR2
1.3.2 Transformation of Zif268 into ECNR2+zeocin without Kan+ZFB+wp
1.3.3 Observation of growth through plate reader
2 August 2nd
2.1 Team TolC
2.2 Team ZF
2.2.1 Checking on the Pos Con 77 and F2/F3 Control Plates
2.2.2 Miniprep of CB Bottom for ligation
2.2.3 Ligation of CB Bottom backbone and F1
2.2.4 PCR of Zif268 to create GFP Reporter Strain
2.3 Team Wolfe
3 August 3rd
3.1 Team ZF
3.1.1 Yesterday's last digestion and PCR results
3.1.2 Innoculation of Zif 268
3.1.3 Sending Pos Con 77 and swap controls for sequencing
3.1.4 Constructing the reporter Zif268 Plasmid
3.1.5 Practice plate ligation
3.2 Team Wolfe
3.3 Team TolC
4 August 4th
4.1 Team ZF
4.1.1 Pos Con 77 and control swap sequencing results
4.1.2 Checking on the reporter strain
4.1.3 Ligation of oligos and backbone
4.1.4 Making more Zif268 (whole) plasmid
4.2 Team Wolfe
4.3 Team TolC
4.3.1 Plate Reader
4.3.2 PCR of plate reader colonies
August 1st
Team Wolfe
Optimizing PCR reaction for pSB4K5:
The gradient PCR did work, but the yield was still very low. We will try a few different approaches to increase our PCR product.  All these reactions were done only with 3:11P (since 2:17H showed no product in the gradient).
1) Gel Stab PCR:
We ran out more of the gradient PCR product (see image below) and used it as the template for a new PCR
Gel stab went into 20µL H2O; 66 C annealing, 3 min elongation; pSB4K5_F+ZFBhl and pSB4K5_R+URA3hl primers
Results: unsuccessful. Perhaps the gel stab did not give an adequate amount of template.
7/29 pSB4K5 gradient samples for gel stab
pSB4K5 gel stab PCR results 8/1/11
2) Reverse Touchdown PCR:
Because the primer initially anneals with 20 bases and then with 60, we will start with a low annealing temperature and then raise it as the overlapping region grows.
58-66 C annealing, changing 1 degree per cycle (9 cycles); the rest of the 25 cycles were done at 66 C
3 min elongation
Results: one of the four reactions worked. This is very strange since the reactions should have been identical.
pSB4K5 reverse touchdown 8/1/11
3) Template concentration gradient PCR:
The primer dimers may have been forming since there wasn't enough template to compete with the primer-primer interactions.
All reagents were kept the same except template: 1µL, 1.5, 2, 2.5, 3, 3.5, or 4µL were used
66 C anneal, 3 min elongation
Results: only one of the reactions worked. We did use the gradient cycler which has not always worked well for us. We also noticed that when we removed the tubes from the cycler, most of the liquid had condensed on the lids.
pSB4K5 template concentration gradient PCR 8/1/11
4) Gradient PCR (overnight):
Since our original gradient seemed to have been one of the more successful ways, we will repeat it but with a tighter temperature range
Normal KAPA protocol, annealing 65-69 C, 3 min elongation
ZFB-wp-his3-ura3:
For isothermal assembly we need a high concentration of each of the DNA components, so we decided to PCR more of our ZFB-wp-his3-ura3 insert and use the Qiagen Minelute kit to purify.
primers: ZFB-wp-f and ZFB+kan_short_R; ZFB-wp-hisura template
because of a substantial difference in primer melting temperatures, we did a 57-61 C annealing gradient; 90 sec elongation
Results: all 8 conditions worked!
ZFB-wp-his3-ura3 gradient PCR 8/1/11
Minelute PCR purification of 4 of the reactions: yield=epic.
BEST YIELD EVER!
pSR01 plate reading:
We shared a 100-well plate with TolC and will take readings overnight with the following conditions:
Wells 51, 61...91: LB+tet/amp
Wells 52, 62...92: NM+his
Wells 53, 63...93: NM
Wells 54, 64...94: NM+1mM 3-AT
Wells 55, 65...95: NM+2.5mM 3-AT
Wells 56, 66...96: NM+5mM 3-AT
Wells 57, 67...97: NM+his+100µM IPTG
Wells 58, 68...98: NM+100µM IPTG
Wells 59, 69...99: NM+1mM 3-AT+100µM IPTG
Wells 60, 70...100: NM+5mM 3-AT+100µM IPTG
51-60: empty
61-70 and 71-80: inoculate with 1µL of selection strain culture
81-90 and 91-100: inoculated with 1µL of selection strain + pSR01 culture
Team ZF
Restriction enzyme cut test, take 3
We repeated the restriction enzyme reaction once more to obtain a better and more definitive gel image, and we used the following formulation:
8 ul of 1x BSA/Buffer 4
2 ul of DNA (used the 79.4 junction PCR product again)
0.5 ul of BsaI-HF
This time it was incubated at 37*C for 15 minutes, and the product was PCR purified.
1.5 ul was run on the gel.
We can clearly see the appropriate sized bands (717 and 578 bps) after cutting our cross junction product with Bsa1
Redoing the isothermal assembly/transformation for Pos con 77 and the 3 controls
Today we redid the isothermal assembly for Pos con 77, Zif268, OZ052, and OZ123 using our previous protocol.  We also performed a chemical transformation using Top 10 ChemComp cells, and plated them with 100 ul each plate.  We will have to wait and see whether colonies grow tomorrow.
We also performed a cross junction PCR on the isothermal assembly product to make sure that it worked, using our usual protocol. The gel (pictured below) shows appropriately sized bands for the cross junction.
All our isothermal assemblies worked, since we can see bands at ~1.4 kb.
Practice plate library assembly
We cut 20 ul of CB bottom plasmid using the following recipe:
2 ul of 10x Buffer 4
16 ul of DNA (at 40 ng/ul)
1 ul of enzyme
1 ul of BSA
This was run out on a gel, and the band at size 2756 bp was extracted.  This will be extracted and used in a ligation reaction tomorrow.
Product after digesting our plasmid with Bsa1
Team TolC
Lambda red recombination of zeocin into ECNR2
Performed electroporation and lambda red on ECNR2 in order to insert zeocin for rpoZ knockout, in order to have for control in plate reader
Performed PCR and ran gel on products
Successful!!
8/1/11 Zeocin insertion into ECNR2 for plate reader
Transformation of Zif268 into ECNR2+zeocin without Kan+ZFB+wp
Performed transformation of Zif268 on spec plasmid into ECNR@ without Kan insertion in order to have a positive control against our selection system
Run PCR on product tomorrow to determine success
Observation of growth through plate reader
Divided plate up in to 45 different cultures
1-15 contained ECNR2+kan-ZFB-wp+zeocin+Zif268 spec plasmid
This is the selection system we created
They were grown in .005% SDS and 2,4,8,12,16 µM of IPTG
16-30 contained ECNR2+Kan-ZFB-wp with out Zif268 spec plasmid
This served as a negative control
Also grow in .005% SDS and 2,4,8,12,16 µM of IPTG
31-45 contained solely ECNR2
This served as a postive control
Grown in .005% SDS and 2,4,8,12,16 µM of IPTG
Plate was taken down to plate reader at 5:25 pm and began a 16 hour program
August 2nd
Team TolC
PCR to check for spec plasmid in ECNR2+zeocin strain successful. The cultures are taking a long time to grow, though, so we might make another transformation with the zif268 plasmid tomorrow, if they do not appear to have grown by tomorrow
IPTG appears to be not working properly. It appears to retard growth. It is possible that the IPTG here may not be working, so an aliquot of IPTG from the Wyss should be brought in tomorrow.
Team ZF
Checking on the Pos Con 77 and F2/F3 Control Plates
All our plates grew bacteria colonies. At the end of the day, we picked 4 colonies from each plate, and grew them in 2 mL of LB+spec.
Miniprep of CB Bottom for ligation
We performed another miniprep on the CB Bottom plasmid, but this time we concurrently performed a miniprep on PZE22G that had been grown overnight in LB+Kan.
Miniprep results. PZE22G had excellent yields, while CB Bottom did not.
Because the yield of the PZE22G miniprep is in the range that we expect (100-200 ng/ul), while the yield of the CB Bottom miniprep is still so low, we concluded that our problem is likely due to the plasmid or LB+spec media. We made new LB+spec. In the future, we could also pick new colonies and grow them up in cultures.
We were planning on using the miniprep product in our ligation reaction. We originally thought that we would need 100 ng of DNA for the reaction, but then we were told that we could have as little as 20 ng of DNA. We decided that in that case, our current yields were sufficient.
Ligation of CB Bottom backbone and F1
We wanted to insert an F1 into a CB backbone today. Yesterday, we cut the CB Bottom plasmid, in preparation for the ligation reaction. Today we gel purified the product. It was difficult to tell exactly which band was the one we were looking for, so we cut out 2 just to be safe. Our gel purification yields were:
Top band: 23 ng/ul
Bottom band: 16 ng/ul
We attempted to cut the F1 oligo from the plate using Bsa1. Our product after cutting was 79 bp, which is technically smaller than the 100 bp minimum for PCR purification. However, we attempted the PCR purification anyway, because this cut-off is unlikely to be strict. We then ran a gel to see the band size of the product that remained. Because our first gel was unsatisfactory, we repeated the digestion a second time.
Attempt One:
Digestion reaction recipe:
2ul DNA
8 ul 1x BSA/Buffer 4 solution
0.5 ul Bsa1
PCR purification yield: 34 ng/ul
Our first digestion did not show any bands below 100 bp.
Attempt Two:
Digestion reaction recipe:
2ul DNA
1 ul Bsa1
50 ul buffer BSA
PCR purification yield: 33.5 ng/ul
We also kept a sample of unpurified PCR product from this digestion to run on a gel, to see if we're losing our desired product in the PCR purification step.
The bands on this 1.2% e-gel were unexpectedly high. It maybe a problem with the gel.
We performed a ligation reaction using the following recipe:
5 ul ligation mix
4 ul backbone
1 ul oligo
But after seeing the gel results we are unsure whether the ligation could be successful.
In order to further troubleshoot our less-than-ideal gels for our digestion product, we ran a third digestion reaction, using the second recipe. This time we simultaneously digested our oligo, and our cross junction (which had been shown to work before). We also ran the unpurified PCR product from recipe 2 on this gel.
See tomorrow's entry for the gel image. Our digestion of the cross junction appeared to have worked, while once again, we got unusually large bands for our oligo.
PCR of Zif268 to create GFP Reporter Strain
We performed a PCR to clone out Zif268 so that we can later assemble Zif and GFP into our reporter plasmid. We used the Zif clone Forward and Zif clone Reverse primers (Primer index: 148 and 149). We had an annealing temperature of 62* and an extension time of 90 seconds. Our expected product was ~3 kb.
See tomorrow's entry for the gel. Our PCR appeared to have worked, and we have a strong band at our expected product size.
Team Wolfe
Plate reader:
We got some beautiful growth curves! A few of the more critical ones are below:
NM+his: both selection strain and pSR01 grew, but the selection strain grew faster. This might just be an artifact from having accidentally added more cells to the selection strain than the pSR01 wells, but it may also show that the pSR01 strain grows slower. Another possibility is that we are expressing enough zinc fingers for it to be slightly toxic for the cells. Next time maybe we should let the plate reader go for longer than 16 hours just in case the cells need more time.
NM: selection strain does not grow, while pSR01 does a perfect rescue!
pSR01 HARV3-AT comparison: the concentration of 3-AT does not seem to make a difference in the growth curves (the differences appear to be due to different wells starting with slightly different amounts of cells). Next time we can try a higher 3-AT concentration, but since we expect Zif268 to bind strongly this might just be what you get.
NM+his+IPTG: IPTG more or less killed the pSR01 cells but not the selection strain. This may be caused by too many zinc fingers being expressed, so we may want to try less IPTG.
EcNR2 Wolfe Selection system: MAGE
Our MAGE oligos are finally here!  We performed 3 rounds of MAGE and designed oligos for allele-specific PCR to do tomorrow to check our colonies.
2µM final concentration HisB VE deletion and PyrF KO oligos; 2.5-3hr recovery between rounds; plated 1µL, 10µL, and 100µL
pSB4K5:
Our pSB4K5 PCR still eludes us. Our overnight gradient did work, but no better than the other days' reactions.
pSB4K5 gradient 8/2/11
Gel extraction and Minelute only gave us a 9.4ng/µL yield (260/280=1.82) which is not enough for isothermal assembly. We will have to miniprep more plasmid for template for PCRs, and we might want to consider using ligation or some other method.
August 3rd
Team ZF
Yesterday's last digestion and PCR results
We ran a gel of our digestion product from yesterday, and our PCR to clone out Zif 268 to construct the reporter plasmid.
Our results for the final digestion we ran yesterday, and the PCR to clone out Zif 268.
Innoculation of Zif 268
We were running low on Zif 268 plasmids, so we decided to make a new culture from the glycerol stock. It has been left in the 37* shaker overnight.
Sending Pos Con 77 and swap controls for sequencing
We performed a miniprep on our cultures of Pos Con 77, and OZ052 swap, OZ123 swap, and Zif 268 swap.
Results of our miniprep.
Once again, we got unusually low yields. We decided to test the quality of our spec, to see if that was the problem. We grew PZE22G(which does not have spec resistance) in spec to see if it grew. By the end of the day, nothing had grown.
Still, we decided to PCR the miniprep product, since any DNA that is in the product should be amplified. We performed our usual protocol for a cross junction PCR, but we repeated the reaction for 30 cycles.
Some of the bands varied in size, but we decided to send it all in for sequencing.
Constructing the reporter Zif268 Plasmid
PCRs of Zif 268 and GFP
We decided to perform more PCRs to clone out Zif 268, since it will have to be gel purified. This reaction had the same conditions as yesterday's.
We also received the GFP plate today, so we decided to perform a colony PCR in order to get out the sequence that encodes GFP. We used our GFP Forward and GFP Reverse (Primer Index: 146 and 147) for the reaction, had an annealing temperature of 58*, and an extension time of 30 seconds. The expected product size was ~700 bp.
The last four lanes show the expected band size of ~825 bp for GFP. However the bands are comparatively faint.
Gel Purification of Zif 268 and GFP
We performed a gel purification of our PCR products.
We made a 1% agarose gel to gel purify our zif clone and GFP. The bands for GFP are much fainter than those for Zif 268.
After gel purification, Zif 268 had a concentration of approximately 110 ng/ul.
While we had a bright band for Zif 268 on our gel, our band for GFP was much fainter, so we decided not to gel purify it, and instead to re-do the PCR (4 times) for that reaction.
We ran another PCR to get out GFP.
Our yield for the gel extraction of GFP was 10.2 ng/ul.
Isothermal assembly and transformation
Our yield for the gel extraction was lower than what we wanted, but we figured we would try the assembly anyway. This meant that we needed 2.9 ul GFP and 0.9 ul backbone. Ideally all the DNA takes up 2.5 ul when performing an isothermal assembly. We decided to perform 2 reactions, so that one of them might work.
First recipe (total volume 10 ul):
1.9 ul GFP
0.6 ul backbone
7.5 ul isothermal assembly mix
Second recipe (total volume 11.3 ul):
2.9 ul GFP
0.9 ul backbone
7.5 ul isothermal assembly mix
We performed a transformation using ChemComp cells, plated 100 ul, and left them in the incubator overnight.
Practice plate ligation
After seeing the results of our third oligo digestion yesterday, we were confused about why the reaction was not working. After talking with our teammates, we realized that we had been performing the reaction on single stranded DNA, instead of our double stranded PCR product. We performed the digestion again today with oligos 1 and 2, using the following recipe:
15 ul DNA
0.5 ul enzyme
85 ul buffer/BSA
After letting the reaction sit at 37* for about 45 minutes, we performed a PCR purification, and then ran the product on a gel:
We could see bands of the appropriate size, after digesting our PCR product. Unpure = cut without PCR purification.
Finally, we had gotten bands at the appropriate size! We noticed that there were multiple bands smaller than our uncut oligo. We figured the smallest band was the one we were looking for. The one that was slightly larger was likely the result of the enzyme only cutting at one end of our oligo. We wondered whether all of our product would have been the appropriate size if we had let the reaction sit at 37* for a longer period of time. Regardless, we decided to use this product in our ligation reaction. If both ends of the oligo aren't cut, then it should not be able to bind with the backbone.
We performed a ligation reaction with our cut CB Bottom and our digested oligos, using the following recipe:
5 ul ligation mix
4 ul backbone (~20 ng/ul)
1 ul insert (~15 ng/ul)
This would leave to a vector to insert ratio of 1 to 8, which is what is normally used for ligation reactions.
This is how we calculated the amounts of inserted needed for our ligation:
PCR product: ~60 ng/ul We want 1 ug of DNA to be cut, so that after the PCR purification, when we elute in 30 ul we will get a concentration of ~30 ng/ul.
In practice, we end up losing some of our product, so we ended up with 15 ng/ul.
After letting the reaction sit at room temperature for 1 hour, we transformed ChemComp bacteria with the ligation products. We plated 50 ul and 400 ul of the transformation and left the plates at 37* overnight.
Team Wolfe
Overnight MAGE round 3 colonies grew - 1µL, 10µL, and 100µL used with (too)many cells on 100µL plate and a workable amount on the 1µL plate.
MAGE round 3 PCR
10µL reactions
Primers: HisB VE KO Check_R, His_F and PyrF KO Check_F, PyrF IR
47 colonies tested for knock out of HisBUra3 and 47 colonies tested for knock out of pyrF
HisB Control - selection strain with same primers
PyrF Control - Colony 1 with PyrF forward and PyrF internal reverse primers
Reverse Touchdown PCR to obtain the kan backbone with homology from pSB4K5 biobrick plasmid:
Used annealing temperature range of 58 to 68 with two rounds elongation at 2 degree intervals and 15 elongation cycles at 68 degrees.
PCR failed
A successful PCR would have resulted in a 3.5-4Kb band.
Miniprep of pSB4K5 strain for future PCR yielded 15ng/µL
Plate reader
Wells 1, 8, 15... 36 LB+tet/amp
Wells 2,9,16...37 1mM 3-AT
Wells 3,10, 17...38 2.5mM 3-AT
Wells 4,11, 18...39 5mM 3-AT
Wells 5,12, 19...40 10mM 3-AT
Wells 6,13,20...41 25mM 3-AT
Wells 7,14,21...42 50mM 3-AT
Wells 1-7 = empty
Wells 8-14 = selection strain
Wells 15-28 = single selection strain + pSR01 colony (duplicates)
Wells 29-42 = second selection strain + pSR01 colony (wells 33-35 were not usable)
MAGE rounds 4&5 completed, colonies plated at 1µL, 10µL, and 100µL volumes.
Team TolC
Set up and ran plate reader for the growth rates of ECNR2+kan-ZFB-wp+zeocin+spec plasmid, ECNR2+kan-ZFB-wp+zeocin Δspec plasmid, ECNR2, and ECNR2+zeocin+spec plasmid Δkan-ZFB-wp in .04% SDS and 25, 12.5, 6.25, 3.125, 1.5625 µM
August 4th
Team ZF
Pos Con 77 and control swap sequencing results
Pos. Con. 77
Notes
1
lots of snps and deletions
✓
2
G->C
3
deletion before type II cut site
4
deletion in terminator
OZ052 swap
Notes
✓
1
G->C
✓
2
G->C
3
deletions and mismatches in ultramer
4
terrible
OZ123 swap
Notes
✓
1
fine, deletion at end
2
deletions and snps
3
long deletion in helix
✓
4
fine
Zif268 swap
Notes
✓
1
fine, TGEKP instead of TGQKP
✓
2
G->C, TGEKP instead of TGQKP
3
terrible
✓
4
G->C, TGEKP instead of TGQKP
We had a good sequence for all of our samples. We made 2 ml cultures of these, to miniprep, make final plates, and glycerol stocks. Note, that for Zif 268, the linker between F1/F2 is 'TGEKP', as that was the sequence in the ultramer we ordered. The actual sequence is 'TGQKP'. If this proves to be a problem, we can always try using quick change to insert the appropriate codon.
Checking on the reporter strain
Our transformed bacteria with the 11.3 ul isothermal assembly product and 100 ng backbone grew. We picked four from the plate and are growing 2 ml cultures from time to miniprep and send for sequencing tomorrow.
Our other bacteria with 10 ul product, but less than 100 ng backbone, did not grow.
Ligation of oligos and backbone
Checking our plates
No colonies grew on any of our plates that contained bacteria transformed with ligation product. We wondered if this was due to our ligase no longer working, so we decided to test our ligase.
Testing our ligase
We wanted to test our ligase by cutting our cross junction with Xba1, and then ligating it, so see if it formed the original cross junction. We ran a gel comparing our whole cross junction, our digested product, and our ligated product.
Comparing our original, cut, and ligated cross junction.
Our uncut cross junction was ~1.4 kb. We saw the appropriate sized bands of 593 and 739 bp after cutting the junction with Xba1. However, after ligation, we saw bands of varying lengths, at the one we expected (at ~1.4 kb) was not noticeably brighter than the other bands. We are unsure what these bands are.
Cutting and gel purifying CB Bottom
We also wondered if our unsuccessful ligation was due to an ineffective CB Bottom segment. We had been unsure exactly which band was the appropriate one, and so we cut both. Just in case we decided to re-do the digestion of the CB Bottom plasmid, with Bsa1. We used the following recipe:
46 ul DNA (to get 1 ug)
1 ul enzyme
5 ul Buffer T4
0.6 ul BSA
We then ran the product on a 0.7% agarose gel:
A gel of the CB Bottom backbone, after digestion. We expected a band at ~2.7 kb
We cut out the appropriate band, and put it in the fridge to gel purify tomorrow.
Making more Zif268 (whole) plasmid
We performed a miniprep of the Zif268 culture we grew yesterday. The miniprep had a yield of ~16 ng/ul.
Team Wolfe
Plate Reader
Looks great!
1mM, 2.5mM, and 5mM 3-AT have slight differences in growth rate, but it is hard to tell how significant they are. However, starting with 10mM 3-AT, the growth rate drops dramatically! Here is an example from 1 culture set:
MAGE 3 HisB and PyrF Knock Out Gel
HisB Check
Control reaction showed a nice strong band.  All the other lanes had bands, but they varied in strength.  It looks like we may have some hits, but we need our WT control primers to come in for us to be sure.
PyrF check
Similar to HisB--all the lanes had bands but some were brighter than others.
HisB
PyrF
ZFB-wp-his3-ura3 restriction enzyme PCR
PCR to add Spe1 and Xba1 sites to join insert with pSB4K5 plasmid
Spe1 ZFB_F and Xba1 URA3-R primers
90 sec elongation, 62˚C annealing
Team TolC
Plate Reader
Plate Reader Graph of KN.
Plate Reader Graph of WN.
Plate Reader Graph of KS.
Plate Reader Graph of WS.
PCR of plate reader colonies
Made glycerol stock of ECNR2+spec plasmid+zeocin, WS
Retrieved from " http://2011.igem.org/Team:Harvard/Template:NotebookDataAugust "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
