<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Harvard_Template_NotebookDataAugust3"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Harvard/Template:NotebookDataAugust3">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Harvard/Template:NotebookDataAugust3&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Harvard/Template:NotebookDataAugust3&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Harvard/Template:NotebookDataAugust3&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Harvard/Template:NotebookDataAugust3" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Harvard/Template:NotebookDataAugust3</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="810" style="display:none"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#August_10"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">August 10</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Team_TolC"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Team TolC</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#MAGE_OZ052_and_OZ123"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">MAGE OZ052 and OZ123</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Recombination_of_Zeocin_into_ECNR2"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">Recombination of Zeocin into ECNR2</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-5"><A href="#Team_Wolfe"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Team Wolfe</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Lambda_Red_RpoZ_Knockout_Plates"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Lambda Red RpoZ Knockout Plates</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Rpoz_Knockout_Check_PCR"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Rpoz Knockout Check PCR</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Digestion_of_pSB4K5_Plasmid"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">Digestion of pSB4K5 Plasmid</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Team_ZF"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Team ZF</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Mini_library_ligation_results"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Mini library ligation results</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Sequencing_results_for_oligos_5_and_6"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Sequencing results for oligos 5 and 6</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Western_blot_to_detect_Zif268"><SPAN class="tocnumber">1.3.3</SPAN><SPAN class="toctext">Western blot to detect Zif268</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Library_ligation_optimization"><SPAN class="tocnumber">1.3.4</SPAN><SPAN class="toctext">Library ligation optimization</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#August_11th"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">August 11th</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Team_Wolfe_2"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Team Wolfe</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Ligation_of_the_HisUra_Insert"><SPAN class="tocnumber">2.1.1</SPAN><SPAN class="toctext">Ligation of the HisUra Insert</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#Transformation_of_pSB4K5_plasmid_with_His3Ura3_Insert_into_Chemically_Competent_Cells"><SPAN class="tocnumber">2.1.2</SPAN><SPAN class="toctext">Transformation of pSB4K5 plasmid with His3Ura3 Insert into Chemically Competent Cells</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#Transformation_of_Zif268_Plasmid"><SPAN class="tocnumber">2.1.3</SPAN><SPAN class="toctext">Transformation of Zif268 Plasmid</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Transformation_of_the_pSB4K5_Plasmid_with_His3Ura3_Insert_into_Electrocompetent_Cells"><SPAN class="tocnumber">2.1.4</SPAN><SPAN class="toctext">Transformation of the pSB4K5 Plasmid with His3Ura3 Insert into Electrocompetent Cells</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Team_ZF_2"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Team ZF</SPAN></A></LI><LI class="toclevel-3 tocsection-21"><A href="#Western_Blot"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">Western Blot</SPAN></A></LI><LI class="toclevel-3 tocsection-22"><A href="#PCR_to_get_out_GFP"><SPAN class="tocnumber">2.2.2</SPAN><SPAN class="toctext">PCR to get out GFP</SPAN></A></LI><LI class="toclevel-3 tocsection-23"><A href="#Check_oligo_pool_sequencing_results"><SPAN class="tocnumber">2.2.3</SPAN><SPAN class="toctext">Check oligo pool sequencing results</SPAN></A></LI><LI class="toclevel-3 tocsection-24"><A href="#Finding_the_optimal_backbone:insert_for_ligation"><SPAN class="tocnumber">2.2.4</SPAN><SPAN class="toctext">Finding the optimal backbone:insert for ligation</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Team_TolC_2"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Team TolC</SPAN></A></LI><LI class="toclevel-3 tocsection-26"><A href="#Search_for_a_better_weak_promoter"><SPAN class="tocnumber">2.3.1</SPAN><SPAN class="toctext">Search for a better weak promoter</SPAN></A></LI><LI class="toclevel-3 tocsection-27"><A href="#Transformation_of_Zif268_Plasmid_2"><SPAN class="tocnumber">2.3.2</SPAN><SPAN class="toctext">Transformation of Zif268 Plasmid</SPAN></A></LI><LI class="toclevel-3 tocsection-28"><A href="#Plater_Reader"><SPAN class="tocnumber">2.3.3</SPAN><SPAN class="toctext">Plater Reader</SPAN></A></LI><LI class="toclevel-3 tocsection-29"><A href="#Gradient_Plates"><SPAN class="tocnumber">2.3.4</SPAN><SPAN class="toctext">Gradient Plates</SPAN></A></LI><LI class="toclevel-1 tocsection-30"><A href="#August_12th"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">August 12th</SPAN></A></LI><LI class="toclevel-2 tocsection-31"><A href="#Team_Wolfe_3"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Team Wolfe</SPAN></A></LI><LI class="toclevel-3 tocsection-32"><A href="#Transformations"><SPAN class="tocnumber">3.1.1</SPAN><SPAN class="toctext">Transformations</SPAN></A></LI><LI class="toclevel-3 tocsection-33"><A href="#PCR_check_for_Zif268_plasmid_insert"><SPAN class="tocnumber">3.1.2</SPAN><SPAN class="toctext">PCR check for Zif268 plasmid insert</SPAN></A></LI><LI class="toclevel-3 tocsection-34"><A href="#Selection_strain_genotype_confirmation"><SPAN class="tocnumber">3.1.3</SPAN><SPAN class="toctext">Selection strain genotype confirmation</SPAN></A></LI><LI class="toclevel-3 tocsection-35"><A href="#EcNR2_selection_strain_plate_reader_experiment"><SPAN class="tocnumber">3.1.4</SPAN><SPAN class="toctext">EcNR2 selection strain plate reader experiment</SPAN></A></LI><LI class="toclevel-2 tocsection-36"><A href="#Team_ZF_3"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Team ZF</SPAN></A></LI><LI class="toclevel-3 tocsection-37"><A href="#Midiprep_of_Zif268_and_CB_Bottom_Plasmids"><SPAN class="tocnumber">3.2.1</SPAN><SPAN class="toctext">Midiprep of Zif268 and CB Bottom Plasmids</SPAN></A></LI><LI class="toclevel-3 tocsection-38"><A href="#Results_of_Cross_Junction_PCR_of_2_random_colonies_from_ligation"><SPAN class="tocnumber">3.2.2</SPAN><SPAN class="toctext">Results of Cross Junction PCR of 2 random colonies from ligation</SPAN></A></LI><LI class="toclevel-3 tocsection-39"><A href="#Send_out_24_colonies_for_sequencing"><SPAN class="tocnumber">3.2.3</SPAN><SPAN class="toctext">Send out 24 colonies for sequencing</SPAN></A></LI><LI class="toclevel-3 tocsection-40"><A href="#Isothermal_assembly_and_transformation_of_GFP_reporter"><SPAN class="tocnumber">3.2.4</SPAN><SPAN class="toctext">Isothermal assembly and transformation of GFP reporter</SPAN></A></LI><LI class="toclevel-3 tocsection-41"><A href="#CB_Bottom_Digestion"><SPAN class="tocnumber">3.2.5</SPAN><SPAN class="toctext">CB Bottom Digestion</SPAN></A></LI><LI class="toclevel-1 tocsection-42"><A href="#August_13th"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">August 13th</SPAN></A></LI><LI class="toclevel-2 tocsection-43"><A href="#Team_ZF_4"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Team ZF</SPAN></A></LI><LI class="toclevel-3 tocsection-44"><A href="#Sequencing_Results"><SPAN class="tocnumber">4.1.1</SPAN><SPAN class="toctext">Sequencing Results</SPAN></A></LI><LI class="toclevel-3 tocsection-45"><A href="#Results_of_GFP_transformation"><SPAN class="tocnumber">4.1.2</SPAN><SPAN class="toctext">Results of GFP transformation</SPAN></A></LI><LI class="toclevel-3 tocsection-46"><A href="#PCR_purification_of_CB_Bottom_digestion_product"><SPAN class="tocnumber">4.1.3</SPAN><SPAN class="toctext">PCR purification of CB Bottom digestion product</SPAN></A></LI><LI class="toclevel-1 tocsection-47"><A href="#August_15"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">August 15</SPAN></A></LI><LI class="toclevel-2 tocsection-48"><A href="#TolC"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">TolC</SPAN></A></LI><LI class="toclevel-2 tocsection-49"><A href="#Wolfe"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Wolfe</SPAN></A></LI><LI class="toclevel-1 tocsection-50"><A href="#August_16"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">August 16</SPAN></A></LI><LI class="toclevel-2 tocsection-51"><A href="#Sarah"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Sarah</SPAN></A></LI><LI class="toclevel-3 tocsection-52"><A href="#TolC_2"><SPAN class="tocnumber">6.1.1</SPAN><SPAN class="toctext">TolC</SPAN></A></LI><LI class="toclevel-3 tocsection-53"><A href="#Wolfe_2"><SPAN class="tocnumber">6.1.2</SPAN><SPAN class="toctext">Wolfe</SPAN></A></LI><LI class="toclevel-2 tocsection-54"><A href="#Team_ZF_5"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Team ZF</SPAN></A></LI><LI class="toclevel-3 tocsection-55"><A href="#GFP_reporter_strain"><SPAN class="tocnumber">6.2.1</SPAN><SPAN class="toctext">GFP reporter strain</SPAN></A></LI><LI class="toclevel-3 tocsection-56"><A href="#Control_library_assembly"><SPAN class="tocnumber">6.2.2</SPAN><SPAN class="toctext">Control library assembly</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="August_10">August 10</SPAN></H2><H3><SPAN class="mw-headline" id="Team_TolC">Team TolC</SPAN></H3><H5><SPAN class="mw-headline" id="MAGE_OZ052_and_OZ123">MAGE OZ052 and OZ123</SPAN></H5><UL><LI>MAGE round 4a, time constants of 5.6s (OZ052) and 1.2s and 5.4s (OZ123). There appeared to be a spark for the OZ123 electroporation (Sarah)
</LI></UL><H5><SPAN class="mw-headline" id="Recombination_of_Zeocin_into_ECNR2">Recombination of Zeocin into ECNR2</SPAN></H5><UL><LI>Zeocin insert colonies appear to be growing extremely slowly, only 3 colonies after 14 hours, but may be due to rpoZ knockout
</LI><LI>Growing up culture in case we need to do lambda red of zeocin again
</LI><LI>PCR of the TolC, rpoZ and zeocin bands for colonies 1 and 2 off the zeocin plate.
</LI><LI>numbered, from 1-6, 1-3 is colony 1, 4-6 is colony 2. 1,4=TT 2,5=RR, 3,6=ZR
</LI><LI>PCR successful!! We have the real WN strain now. It should be ready for transformation tomorrow with the zif268 spec plasmid
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Zeocin/rpoZ and TolC PCR of lambda red colonies 8/10/11</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Team_Wolfe">Team Wolfe</SPAN></H3><H4><SPAN class="mw-headline" id="Lambda_Red_RpoZ_Knockout_Plates">Lambda Red RpoZ Knockout Plates</SPAN></H4><UL><LI>Small colonies grew after about 14 hours on zeocin plates
</LI></UL><H4><SPAN class="mw-headline" id="Rpoz_Knockout_Check_PCR">Rpoz Knockout Check PCR</SPAN></H4><UL><LI>Picked 24 colonies and used zeocin binding primers as a control and rpoZ binding primers to check for a knockout.
</LI><LI>48 10ÂµL reactions in total (24 colonies X 2 different primer sets mentioned above)
</LI><LI>Primers: rpoZ_R with Zeocin_R and rpoZ_R with rpoZ_F
</LI><LI>64 degree annealing temperature
</LI><LI>90 seconds elongation
</LI></UL><P><B>Results</B></P><UL><LI>rpoZ was knocked out!
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Zeocin/rpoZ PCR of lambda red colonies 8/10/11</DIV></DIV></DIV><UL><LI>Reinoculated LB+zeo with culture 9 to use tomorrow for glycerol stocks and transformations
</LI></UL><H4><SPAN class="mw-headline" id="Digestion_of_pSB4K5_Plasmid">Digestion of pSB4K5 Plasmid</SPAN></H4><UL><LI>Gel with yesterday's digestion product suggests that neither of the two restriction enzymes (Xba1 or Spe1) cut our plasmid. This was determined by comparing the band sizes to those obtained by midiporep of the pSB4K5 (whole) plasmid.
</LI><LI>Plan of action: We will use smaller reactions to increase the relative concentration of enzymes and we will use slightly more BSA buffer.
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:137px;"><DIV class="thumbcaption">pSB4K5 digestion with Spe1  and Xba1</DIV></DIV></DIV><P><B>Today's Digestion of pSB4K5 Plasmid</B></P><UL><LI>Xba1
</LI><LI>16ÂµL DNA (~1Âµg)
</LI><LI>2ÂµL buffer4
</LI><LI>.5ÂµL BSA
</LI><LI>1.5ÂµL Xba1
</LI></UL><LI>Spe1
</LI><LI>16ÂµL DNA (~1Âµg)
</LI><LI>2ÂµL buffer4
</LI><LI>.5ÂµL BSA
</LI><LI>1.5ÂµL Spe1
</LI><LI>Xba1+Spe1
</LI><LI>14.5ÂµL DNA (~1Âµg)
</LI><LI>2ÂµL buffer4
</LI><LI>.5ÂµL BSA
</LI><LI>1.5ÂµL Xba1
</LI><LI>1.5ÂµL Spe1
</LI><LI>Results: the size of the bands isn't what we expected (backbone should be ~3.4kb, insert ~1.2), but the comparison between the different digestions and the uncut plasmid does imply that the digestion we worked. We will go ahead and gel extract from the Spe+Xba digest and use both ligation and sequencing to be sure that this is indeed the plasmid we want.
</LI><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">pSB4K5 digest with Spe1, Xba1, and both 8/10/11</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Team_ZF">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Mini_library_ligation_results">Mini library ligation results</SPAN></H4><P>We checked our plates from our practice plate library assembly yesterday, and we got colonies!!  Strangely, our 450 ul plate didn't have a huge number more colonies than the 50 ul plate.  Dan suggested that this was because we plated all the cells in a 450 ul volume, which may have sloshed around and taken cells with it.  Next time we should pellet the cells and resuspend in 100 ul and plate that.  Based on our colony numbers with 50 ul, we should probably plate with 100 ul in the future.
</P><P>Due to the proliferation of colonies, we picked 90 colonies with the assistance of Will.  These 90 colonies should hopefully represent all 18 oligos in equal amounts.  We did colony PCRs on all 90 with the junction PCR protocol, and sent them out for sequencing today.  We saved these 90 colonies by adding LB/spec to each of the wells after suspending them in the 10 ul of water for the colony PCR.  We will eventually harvest and gather all 18 assembled plasmids from this step.
</P><H4><SPAN class="mw-headline" id="Sequencing_results_for_oligos_5_and_6">Sequencing results for oligos 5 and 6</SPAN></H4><P>Sequencing indicated that most of our colonies worked, and that our F1s were inserted correctly!  This is a huge step forward, and gives us good hopes that our 90 colonies also have their F1s inserted.  Out of the 8 colonies each that we tested for oligo 5 and 6, only 5.8 (missing chunk on promoter), 5.3 (deletion on linker), and 6.8 (bad trace file) were bad.  However, all the rest were good!
</P><P>We also now have an idea of our combined ligation/transformation efficiency.  NUMBERS HERE PLZ
</P><H4><SPAN class="mw-headline" id="Western_blot_to_detect_Zif268">Western blot to detect Zif268</SPAN></H4><P>Today we began a western blot in an attempt to detect Zif268 expression from IPTG induction.  Due to the fact that the WB gels only have 10 lanes, we dropped the 2 uM IPTG sample for now, keeping the negative control (pze22g + gfp), 0 uM IPTG, 10 uM IPTG, and 100 uM IPTG.
</P><P>Lysis buffer had the following composition:
</P><UL><LI>150 mM NaCl
</LI><LI>50 mM Tris-HCl
</LI><LI>5 mM EDTA
</LI><LI>1% Triton X-100
</LI><LI>1x proteinase inhibitor
</LI></UL><P>We used the following protocol for lysis:
</P><OL><LI>Spin down 3 ml of cells into a pellet, discarding the LB supernatant.
</LI><LI>Resuspend cells in 1.5 ml of ice cold lysis buffer, take 200 ul of this into a new tube and add 800 ul of cold lysis buffer.
</LI><LI>Sonicate at power level 2 in two sets of five bursts (each about one second long).  Be sure to wait 10 seconds in between sets and keep tube on ice to prevent overheating.
</LI><LI>Spin down at maximum speed for 5 minutes at 4 degrees.
</LI><LI>Take supernatant into a new tube.  This is the final protein lysis mixture.
</LI></OL><P>Note!  In the future, we should only spin down 200 ul of cells for lysis.  According to Alain, most of the time they even just use 100 ul and directly boil it with SDS.  May try this protocol tomorrow if current protocol doesn't work (the downside is that we have to start again from the culture stage and not the protein stage).
</P><P>Prepping samples for the gel:
</P><OL><LI>For loading duplicates, load 9 ul of 5x sample buffer with 36 ul of protein for a total of 45 ul.
</LI><LI>Parafilm tubes to prevent popping, and boil for 5 minutes at 100 degrees.
</LI><LI>Load 10 ul of protein ladder into gel, and load 15 ul of sample into each well.
</LI></OL><P>Note!  For this particular gel, the wells go in this order from left to right: Marker, neg con, 0 uM, 10 uM, 100 uM, neg con, 0 uM, 10 uM, 100 uM, Marker.
</P><P>Running samples on gel:
</P><UL><LI>Gel was run at 150 V until the marker band reached approximately 80% of the way to the bottom of the gel.
</LI></UL><P>Transfer:
</P><UL><LI>Semi-dry transfer was accomplished by wetting blotting paper and nitrocellulose membrane in transfer buffer and making the sandwich on the transfer apparatus.  From bottom to top (red to black electrode), the loading order was as follows:
</LI><LI>2 sheets of blotting paper
</LI><LI>Nitrocellulose membrane
</LI><LI>Gel
</LI><LI>2 sheets of blotting paper
</LI><LI>Transfer was run at 20 V for 1 hour.
</LI></UL><P>Note!  The side containing the protein is face up when the notch in the membrane is situated on the top right hand corner.  However, due to troubles with the gel, the lanes are now in the opposite order of loading.  So, with the protein right side up, the lanes go: marker, 100 uM, 10 uM, 0 uM, neg con, etc.  Also, we forgot to roll bubbles out from the sandwich with the pipette fragment before transferring (my bad!)  Hopefully this doesn't impact anything.  One last note on the transfer: the current cap was 400 mA, and when we started the transfer, it initially began at 17 V and 400 mA.  Over a few minutes, the voltage increased to 20 V and the current began to decrease, hitting 110 mA after a half hour of gradual decay, which is where we expected the current to be.
</P><P>The transfer seemed to have worked, transferring the marker bands from the gel onto the membrane.
</P><P>Blocking:
</P><UL><LI>Membrane was blocked at 4 degrees overnight.
</LI></UL><P>Tomorrow we will proceed with primary and secondary antibody staining and band visualization.
</P><H4><SPAN class="mw-headline" id="Library_ligation_optimization">Library ligation optimization</SPAN></H4><P>Dan suggests that we optimize the vector:insert ratio for our library assembly.  We will try using the following insert ratios (assuming a vector unit of 1): 1, 2, 3, 4, 8, 10, 16.  Unfortunately, today we did not have enough cut CB bottom plasmid to immediately begin the ligation and transformation, so we attempted to cut more CB bottom.  After performing a cut with Bsa-HF using the previous protocol, we performed a PCR purification on the product, only to find that there were only 3.4 ng/ul of cut plasmid present.  This is not enough to perform the ligation, and so we will have to try the CB bottom plasmid cut again tomorrow morning when the cultures have grown enough.  We can then do a ligation and transformation tomorrow and count colonies on Friday.
</P>
As stated in the earlier section, we should probably plate 100 ul of transformed bacteria to standardize everything and to keep colony counting fairly manageable.</DIV><DIV id="811" style="display:none"><H2><SPAN class="mw-headline" id="August_11th">August 11th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_Wolfe_2">Team Wolfe</SPAN></H3><H4><SPAN class="mw-headline" id="Ligation_of_the_HisUra_Insert">Ligation of the HisUra Insert</SPAN></H4><UL><LI>follow <A href="http://www.neb.com/nebecomm/products/protocol2.asp" class="external text" rel="nofollow">New England Biolabs protocol</A>:
</LI><LI>1.5ÂµL of cut Spe ZFB-wp-hisura Xba insert (75ng,a 3-fold molar excess to backbone)
</LI><LI>6ÂµL cut pSB4K5 backbone (50ng)
</LI><LI>1.5ÂµL Water (bring volume up to 10ÂµL)
</LI><LI>1ÂµL Quick T4 DNA Ligase
</LI><LI>10ÂµL 2X quick ligation buffer and mix
</LI><LI>spin briefly and incubate for 5 min at room temperature
</LI><LI>keep at -20ËC or on ice until transformation
</LI></UL><LI>Transformation: ligation mixture cannot be directly electroporated, so we will start with top10 chem comp cells (see protocols page)
</LI><LI>Following <A href="http://www.neb.com/nebecomm/products/protocol3.asp" class="external text" rel="nofollow">New England Biolab's transformation recommendations</A>, we used 2ÂµL of ligation mixture in one tube of 50ÂµL cells, and 1ÂµL of unligated cut backbone (about an equal mass) in another tube of cells as a control.
</LI><H4><SPAN class="mw-headline" id="Transformation_of_pSB4K5_plasmid_with_His3Ura3_Insert_into_Chemically_Competent_Cells">Transformation of pSB4K5 plasmid with His3Ura3 Insert into Chemically Competent Cells</SPAN></H4><UL><LI>Transformed into delta HisB delta pyrF delta rpoZ Wolfe strain 
</LI><LI>Cells plated in 1mL and 2mL quantities on Kan plates after 3 hours recovery
</LI></UL><H4><SPAN class="mw-headline" id="Transformation_of_Zif268_Plasmid">Transformation of Zif268 Plasmid</SPAN></H4><UL><LI>Transformed into HisB delta pyrF delta rpoZ EcNR2 strain
</LI><LI>Cells were plated after 3.5 hours recovery on Spec plates at 10ÂµL and 100ÂµL
</LI></UL><H4><SPAN class="mw-headline" id="Transformation_of_the_pSB4K5_Plasmid_with_His3Ura3_Insert_into_Electrocompetent_Cells">Transformation of the pSB4K5 Plasmid with His3Ura3 Insert into Electrocompetent Cells</SPAN></H4><UL><LI>Transformed into delta HisB delta pyrF delta rpoZ Wolfe strain 
</LI><LI>3ÂµL DNA from ligation reaction (unpurified)
</LI></UL><H3><SPAN class="mw-headline" id="Team_ZF_2">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Western_Blot">Western Blot</SPAN></H4><H4><SPAN class="mw-headline" id="PCR_to_get_out_GFP">PCR to get out GFP</SPAN></H4><P>Our new GFP forward primer arrived today, so we performed a PCR to get out the GFP region. Because the product will be used for gel extraction, we ran 4 reactions. We had an annealing temperature of ... and an extenstion time of 30 seconds.
</P><H4><SPAN class="mw-headline" id="Check_oligo_pool_sequencing_results">Check oligo pool sequencing results</SPAN></H4><P>Unfortunately, most of the samples we sent in for sequencing had problems. For many, the primers did not anneal; for others there were poor quality reads. 13 samples in total had good quality reads. Among those we got a pretty good distribution among our 18 oligos. This is promising. We will perform a PCR on 2 random samples to ensure that the cross junction is present. If we get positive results, we will send in 24 more samples for sequencing. 
</P><P>Summary of sequencing results:
</P><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Oligo</B></TD><TD align="center" style="background:#f0f0f0;"><B>Frequency</B></TD></TR><TR><TD> 5</TD><TD>3
</TD></TR><TR><TD> 6</TD><TD>1
</TD></TR><TR><TD> 8</TD><TD>1
</TD></TR><TR><TD> 9</TD><TD>1
</TD></TR><TR><TD> 10</TD><TD>1
</TD></TR><TR><TD> 11</TD><TD>2
</TD></TR><TR><TD> 12</TD><TD>1
</TD></TR><TR><TD> 17</TD><TD>1
</TD></TR><TR><TD> 18</TD><TD>2
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Finding_the_optimal_backbone:insert_for_ligation">Finding the optimal backbone:insert for ligation</SPAN></H4><P>We wanted to find the optimal backbone to insert ratio for ligation. Previously we had successfully used 100 ng of backbone, and a ratio of 1 to 8. We decided to try the following ratios: 
</P><UL><LI>1:1, 1:2, 1:3, 1:4, 1:8, 1:10, 1:16
</LI></UL><P>We ran out of cut CB Bottom backbone, so we needed to digest some more. We performed a miniprep on cultures that grew up overnight, using min-elute columns. Our final concentration of CB Bottom Plasmid was 50 ng/ul and 54 ng/ul.  
</P><P>We followed our usual protocol for digestion, and then performed a PCR purification, once again using a min-elute column to maximize our yield. However, the resulting product only had a concentration of ~5 ng/ul and a purity of ~1.6. We got similarly low yields yesterday, and are unsure where things are going wrong, since we have performed the same procedure in the past with success. Even if the plasmids weren't cutting, they should be present in the PCR purification product, so the low yields are puzzling. We decided to try a midi-prep tomorrow, so see if having more CB Bottom plasmids would help. 
</P><P>We grew up the following 100 mL cultures for midi-prep tomorrow: CB Bottom, Zif268.
</P><H3><SPAN class="mw-headline" id="Team_TolC_2">Team TolC</SPAN></H3><H4><SPAN class="mw-headline" id="Search_for_a_better_weak_promoter">Search for a better weak promoter</SPAN></H4><UL><LI>We have searched for a better weak promoter than the one we have now and compared their sequences to the present weak promoter in the kan-ZFB-wp construct, sequences were not similar enough to change the weak promoter through MAGE
</LI><LI>To change the promoter would be a big process, so we are thinking of using a negative selection of TolC using colicin and then creating a higher TolC production standard to begin with which would allow the KS and KN growth curves separate more
</LI></UL><H4><SPAN class="mw-headline" id="Transformation_of_Zif268_Plasmid_2">Transformation of Zif268 Plasmid</SPAN></H4><UL><LI>Performed transformation of the Zif268 plasmid into the good ECNR2+zeocin that definitely does have TolC
</LI><LI>Plated 1 mL of culture onto spec plate
</LI></UL><H4><SPAN class="mw-headline" id="Plater_Reader">Plater Reader</SPAN></H4><UL><LI>Kept the IPTG concentration constant at 100 ÂµM while changing the SDS concentration from 2% to 2.5%, 3%, 3.5%, and 4%
</LI><LI>Making two wells per different culture
</LI><LI>Cultures are changing by SDS and IPTG concentration, while also changing from KS, KN, WS, WN colonies
</LI></UL><LI>Since colonies are not near mid-log we may have to fill plate reader tomorrow and run then as well
</LI><H4><SPAN class="mw-headline" id="Gradient_Plates">Gradient Plates</SPAN></H4><UL><LI>Created 5 50 mL total gradient plates with 2x Kan in the wedge layer, 20ul of Bluejuice in each 25ml of LB+agar+Kan to mark it with a dye.</LI></UL><DIV id="812" style="display:none"><H2><SPAN class="mw-headline" id="August_12th">August 12th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_Wolfe_3">Team Wolfe</SPAN></H3><H4><SPAN class="mw-headline" id="Transformations">Transformations</SPAN></H4><UL><LI>EcNR2 strain with transformed zif268 plasmid grew on spec plates: After 15 hours, 1mL and 2mL plates have ~20 colonies each
</LI><LI>Wolfe strain with transformed ligated pSB4K5 plasmid had no growth on kan plate at 1mL and unusual looking growth on a tet plate at 1mL after 12 hours
</LI><LI>The pSB4K5 plasmid was successfully transformed into the chemically coMpetent cells - will midiprep these to obtain a clean sample of ligated pSB4K5 for another transformation in electrocompetent Wolfe strain cells.
</LI></UL><H4><SPAN class="mw-headline" id="PCR_check_for_Zif268_plasmid_insert">PCR check for Zif268 plasmid insert</SPAN></H4><UL><LI>Looks like the plasmid's presence is confirmed by PCR in 8 colonies
</LI><LI>Team ZF's &quot;CJUN&quot; protocol: pZE23G 2123R and pZE23G 3581F primers
</LI><LI>Note well 5 is fainter but is probably due to a loading error where a small volume of sample was left out.
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">EcNR2 selection strain colonies transformed with zif268 8/12/11</DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Selection_strain_genotype_confirmation">Selection strain genotype confirmation</SPAN></H4><UL><LI>We will do one last sequencing round of our EcNR2 selection strain to confirm that HisB and PyrF are indeed knocked out.
</LI><LI>Same parameters as previously described; His_R and PyrF_F used for sequencing
</LI><LI>PCR was successful, as shown by an E gel of several of the reactions.
</LI></UL><H4><SPAN class="mw-headline" id="EcNR2_selection_strain_plate_reader_experiment">EcNR2 selection strain plate reader experiment</SPAN></H4><UL><LI>We want to see how the EcNR2 selection strain works with and without plasmid, especially in the presence of various concentrations of 3-AT and 5-FOA.
</LI><LI>Wells 1, 14, 27, 40, 53: LB+kan/spec
</LI><LI>Wells 2, 15, 28, 41, 54: NM+his
</LI><LI>Wells 3, 16, 29, 42, 55: NM
</LI><LI>Wells 4, 17, 30, 43, 56: NM+1mM 3-AT
</LI><LI>Wells 5, 18, 31, 44, 57: NM+5mM 3-AT
</LI><LI>Wells 6, 19, 32, 45, 58: NM+10mM 3-AT
</LI><LI>Wells 7, 20, 33, 46, 59: NM+25mM 3-AT
</LI><LI>Wells 8, 21, 34, 47, 60: NM+50mM 3-AT
</LI><LI>Wells 9, 22, 35, 48, 61: NM+0.1mM 5-FOA
</LI><LI>Wells 10, 23, 36, 49, 62: NM+0.25mM 5-FOA
</LI><LI>Wells 11, 24, 37, 50, 63: NM+0.5mM 5-FOA
</LI><LI>Wells 12, 26, 38, 51, 64: NM+1mM 5-FOA
</LI><LI>Wells 13, 27, 39, 52, 65: NM+2.5mM 5-FOA
</LI><LI>Wells 1-13: empty
</LI><LI>Wells 14-26: EcNR2âHisBâPyrFârpoZ+kan-ZFB-wp-his3-ura3 (6)
</LI><LI>Wells 27-39: EcNR2âHisBâPyrFârpoZ+kan-ZFB-wp-his3-ura3 (9)
</LI><LI>Wells 40-52: EcNR2âHisBâPyrFârpoZ+kan-ZFB-wp-his3-ura3 +Zif268 spec (1)
</LI><LI>Wells 53-65: EcNR2âHisBâPyrFârpoZ+kan-ZFB-wp-his3-ura3 +Zif268 spec (2)
</LI></UL><H3><SPAN class="mw-headline" id="Team_ZF_3">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Midiprep_of_Zif268_and_CB_Bottom_Plasmids">Midiprep of Zif268 and CB Bottom Plasmids</SPAN></H4><P>We performed a midi-prep on the Zif268 (whole) and CB Bottom plasmids. Zif268 was eluted in 100 ul of EB, while CB Bottom was eluted in 75 ul. 
</P><UL><LI>Zif had a final concentration of 261.45 ng/ul and a 260/280 of 1.90.
</LI><LI>CB Bottom had a final concentration of 237.93 ng/ul and a 260/280 1.91.
</LI></UL><H4><SPAN class="mw-headline" id="Results_of_Cross_Junction_PCR_of_2_random_colonies_from_ligation">Results of Cross Junction PCR of 2 random colonies from ligation</SPAN></H4><P>After running an e-gel we saw the expected 1.4 kb band. (The gel image is not available, because we were unable to get the software running at the time). We decided to proceed with sending 24 colonies out for sequencing.
</P><H4><SPAN class="mw-headline" id="Send_out_24_colonies_for_sequencing">Send out 24 colonies for sequencing</SPAN></H4><P>We performed a cross-junction PCR on the following colonies to send them out for sequencing: 
</P><UL><LI>1B, 1C, 1D, 1E, 1F, 1G, 1H, 2A, 3C, 3D, 3E, 3F, 3G, 3H, 4A, 4B, 10A, 10B, 10C, 10D, 10E, 10F, 10G, 10H
</LI></UL><P>We picked cultures that did not have sequencing results from our previous attempt. A 1 ul of a 1:10 dilution of each culture was used as template in the PCR. We used the Junction Reverse primer.
</P><H4><SPAN class="mw-headline" id="Isothermal_assembly_and_transformation_of_GFP_reporter">Isothermal assembly and transformation of GFP reporter</SPAN></H4><P>We ran our PCR product from yesterday on a 1% agarose gel. We got an appropriate band at approximately 700 bp. (Unfortunately, we could not get the imaging software to work at the time, so no image is available)
</P><P>We performed a gel extraction on our GFP PCR product, with the following yields:
</P><UL><LI>114.90 ng/ul; 260/280 of 1.89
</LI><LI>92.02 ng/ul; 260/280 of 1.94
</LI></UL><P>We then performed an isothermal assembly, using the following recipe:
</P><UL><LI>0.91 ul Zif 268 (backbone)
</LI><LI>0.27 ul GFP
</LI><LI>1.32 ul water
</LI><LI>7.5 ul isothermal assembly mix
</LI></UL><P>We transformed using ChemComp cells, plating 10 and 150 ul.
</P><H4><SPAN class="mw-headline" id="CB_Bottom_Digestion">CB Bottom Digestion</SPAN></H4>
We digested our CB Bottom plasmid, using our usual recipe. We ran the digestion for 2 hours at 37*, 20 minutes at 65*, and then let the reaction sit at 4* overnight.</DIV><DIV id="813" style="display:none"><H2><SPAN class="mw-headline" id="August_13th">August 13th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_4">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Sequencing_Results">Sequencing Results</SPAN></H4><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Oligo</B></TD><TD align="center" style="background:#f0f0f0;"><B>ID</B></TD><TD align="center" style="background:#f0f0f0;"><B>Notes</B></TD></TR><TR><TD> 2</TD><TD>1B</TD><TD>â
</TD></TR><TR><TD> 4</TD><TD>3H</TD><TD>â
</TD></TR><TR><TD> 5</TD><TD>10A</TD><TD>â
</TD></TR><TR><TD> 5</TD><TD>1D</TD><TD>â
</TD></TR><TR><TD> 5</TD><TD>1E</TD><TD>lots of snps at beginning
</TD></TR><TR><TD> 5</TD><TD>3G</TD><TD>â
</TD></TR><TR><TD> 6</TD><TD>10C</TD><TD>â
</TD></TR><TR><TD> 6</TD><TD>10G</TD><TD>â
</TD></TR><TR><TD> 6</TD><TD>3C</TD><TD>â
</TD></TR><TR><TD> 6</TD><TD>4A</TD><TD>2 deletions at F1
</TD></TR><TR><TD> 7</TD><TD>10B</TD><TD>â
</TD></TR><TR><TD> 7</TD><TD>10D</TD><TD>â
</TD></TR><TR><TD> 7</TD><TD>10E</TD><TD>â
</TD></TR><TR><TD> 7</TD><TD>1F</TD><TD>â
</TD></TR><TR><TD> 10</TD><TD>3E</TD><TD>â
</TD></TR><TR><TD> 12</TD><TD>1C</TD><TD>deletion right before F1
</TD></TR><TR><TD> 12</TD><TD>1G</TD><TD>2 deletions at end of F1
</TD></TR><TR><TD> 14</TD><TD>1H</TD><TD>insert 'T' on F1
</TD></TR><TR><TD> 15</TD><TD>2A</TD><TD>â
</TD></TR><TR><TD> 17</TD><TD>10F</TD><TD>â
</TD></TR><TR><TD> 17</TD><TD>10H</TD><TD>â
</TD></TR><TR><TD> 18</TD><TD>3D</TD><TD>â
</TD></TR><TR><TD> 18</TD><TD>3F</TD><TD>â
</TD></TR><TR><TD> 18</TD><TD>4B</TD><TD>â
</TD></TR></TBODY></TABLE><P>Combined with the sequencing results from before, we are missing oligos 3, 13, 14, and 16.
</P><H4><SPAN class="mw-headline" id="Results_of_GFP_transformation">Results of GFP transformation</SPAN></H4><P>We have colonies on both of our plates! We still need to check if they actually express GFP. 
</P><H4><SPAN class="mw-headline" id="PCR_purification_of_CB_Bottom_digestion_product">PCR purification of CB Bottom digestion product</SPAN></H4>
We performed a PCR purification on the CB Bottom digestion, with a yield of 68.96 ng/ul and a 260/280 of 1.85. This will be used in a ligation in the future.</DIV><DIV id="815" style="display:none"><H2><SPAN class="mw-headline" id="August_15">August 15</SPAN></H2><H3><SPAN class="mw-headline" id="TolC">TolC</SPAN></H3><P>There appears to be very low selection (only on the order of 1:10) for the KS:KN strain.
</P><H3><SPAN class="mw-headline" id="Wolfe">Wolfe</SPAN></H3><P>Yay!!!
</P><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Selection strain with and without plasmid, and 3-AT 8/12/11</DIV></DIV></DIV><P>Interestingly, our 5-FOA concentrations did not seem very potent: 3-AT was more effective at retarding growth. We might drop the URA3 part of our selection and instead use higher amounts of 3-AT to distinguish between true hits and false positives.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Selection strain without Zif268 with 3-AT and 5-FOA 8/12/11</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Selection strain with Zif268 with 3-AT and 5-FOA 8/12/11</DIV></DIV></DIV></DIV><DIV id="816" style="display:none"><H2><SPAN class="mw-headline" id="August_16">August 16</SPAN></H2><P>In prioritizing our work for the next two weeks, we decided to focus on three main experiments:
</P><OL><LI>Preparing the oligo pool from the chip using Sri's protocol (using qPCR to amplify the pools equally)
</LI><LI>Continuing the experiment of picking out a positive hit amongst negative signal
</LI><LI>Biobricking our expression plasmids
</LI></OL><P>Eventually, the first two experiments will come together in terms of actually using the chip and figuring out how to pull out and select the positive binders.
</P><P>On average we expect about three students working at a time, so we'll have these three experiments running in parallel by each student.  So this means our previous teams will have to be dissolved, also since TolC will no longer be usedÂ :'(.
</P><P>Each experiment in more detail:
</P><OL><LI>We need to work out the protocol with Sri in order to prepare the oligo pools for ligation into our target plasmid backbones.  This will involve running a qPCR to monitor the amount of amplification so that the oligos in each pool will be amplified in equal amounts (so that we get a representative pool of oligos that is not biased to one side or the other).  We will need to do this with each subpool.  Futhermore, we will have to confirm that the primers that Sri provided are indeed specific enough to pull out their target subpools only and not other subpools.  This is important for the tweaking of our selection system, since we will need to know whether our selection system needs to be specific enough to pick out signal from 1:10,000 or 1:100,000.
</LI><LI>We will need to assemble a control practice library from the 18 practice plate oligos.  We will basically formulate a negative control pool with equal amounts of each of the 18 oligos, and spike this pool with varying amounts of Zif268.  We will then transform our selection strain with these pools with varying amounts of Zif268, and measure how specifically we can pick a winner out amongst non-winners (so instead of a bacteria dilution, it will be a plasmid dilution).  We will use 8 different concentrations:
</LI><LI>Positive control (all Zif268)
</LI><LI>1:10
</LI><LI>1:100
</LI><LI>1:1,000
</LI><LI>1:10,000 (this is approximately the ratio we expect for a winner amongst a pool of 9,149 non-winners for the chip subpools)
</LI><LI>1:100,000
</LI><LI>1:1,000,000
</LI><LI>Negative control (only contains the 18 practice plate oligos)
</LI><LI>We will start figuring out the procedure for converting our expression plasmids into Biobricks for submission.  This entails locating and eliminating all of the five specified restriction sites from our biobrick part (XbaI, SpeI, NotI, PstI, EcoRI), and assembling the part into a Biobrick submission plasmid (<A href="http://partsregistry.org/wiki/index.php/Part:pSB1C3" class="external text" rel="nofollow">pSB1C3</A>).  This may be accomplished through the use of isothermal assembly to combine the Biobrick shipping plasmid backbone with our Biobrick part (the omega subunit + zinc finger array).
</LI></OL><H3><SPAN class="mw-headline" id="Sarah">Sarah</SPAN></H3><H4><SPAN class="mw-headline" id="TolC_2">TolC</SPAN></H4><P>TolC selection abandoned.
</P><H4><SPAN class="mw-headline" id="Wolfe_2">Wolfe</SPAN></H4><P>There appears to be low selection for the Wolfe strain as well. i.e. 1:10. One possibility is that the wrong strain was picked, the other. In the absence of the 96 well plate, we will use the culture tube, and possibly the 100 well, plate reader plate.
</P><H3><SPAN class="mw-headline" id="Team_ZF_5">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="GFP_reporter_strain">GFP reporter strain</SPAN></H4><P>We looked at the plate of GFP reporter that grew colonies and had the correct Shine-Dalgarno RBS with the correct spacer, putting it over the blue light of an e-gel box.  We have glowing colonies!  Only about seven of the ~250 colonies glowed, which may be due to an extra leaky promoter.  We then picked a glowing colony and put it in the warm room to grow overnight in preparation for miniprep.
</P><H4><SPAN class="mw-headline" id="Control_library_assembly">Control library assembly</SPAN></H4>
We picked 24 colonies from our plate which was transformed with the library mix of the 18 practice plate oligos and grew them up in LB/spec for miniprep tomorrow.</DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Harvard/Template:NotebookDataAugust3">http://2011.igem.org/Team:Harvard/Template:NotebookDataAugust3</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Harvard/Template:NotebookDataAugust3" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Harvard/Template:NotebookDataAugust3" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Harvard/Template:NotebookDataAugust3&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Harvard/Template:NotebookDataAugust3&amp;oldid=139129" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>