"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Harvard/ZF Binding Site Finder
From 2011.igem.org
bar
Overview | MAGE | Chip-Based Synthesis | Lambda Red | Protocols
Zinc Finger Binding Site Finder
Zinc Finger Binding Site Finder
by Justin Chew
Insert target DNA sequence here
Bottom Zinc Finger Array:
5'-
-3'
Top Zinc Finger Array:
5'-
-3'
Nucleotide Gap:
How do I use this tool? (Click to toggle help)
Background
This tool is designed specifically to find binding sites for zinc finger nucleases (ZFNs) in order to create double stranded breaks in DNA.  Ultimately, through these double stranded breaks, ZFNs allow genome editing with the insertion or deletion of genes at very specific target DNA sites.  Each ZFN recognizes and binds to a specific 9-bp DNA sequence that is unique to each ZFN, and the binding sites are arranged such that two ZFNs flank the cut site on opposite strands of DNA, as pictured below:
Given two 9-bp DNA sequences, this program will search a string of DNA to find a configuration such that these two 9-bp sequences are located on opposite strands with a short gap in between them (5-7 bp long), where the double stranded break will occur.  This is useful due to the fact that we currently do not have a library of ZFNs that spans the space of all 9-bp recognition sites.  Therefore, if the 9-bp binding sites for two zinc fingers are known and well-characterized, this program will search for a site in which these two known zinc fingers can be used to make a cut for gene insertion or deletion.
Tool Usage
The "bottom" ZFN binds to the bottom strand of DNA, while the "top" ZFN binds to the top strand.  To use the tool, input a 9-bp DNA sequence into the text boxes for both bottom and top ZFNs, and then click "Find binding sites".  Input must be a 9-bp DNA sequence (you may use "N" for an unspecified nucleotide).  The results will be listed in a table which shows:
The relative position of the binding site within the input DNA string
The binding sequence for the bottom ZFN, 5' to 3'
The binding sequence for the top ZFN, 5' to 3'
The entire binding sequence, including the 5-7 bp nucleotide gap, from 5' to 3' on the top strand
Tool Demo
If you would like to try out an example, you can test the tool out with a DNA sequence that we used to search for a suitable target site for our colorblindness target, in which we tried to search for a site upstream of the red opsin gene to insert a copy of the green opsin gene.  This DNA sequence is a stretch of the human X chromosome just upstream of red opsin, and the two zinc fingers we would like to search for have the form of GNNGNNTNN for the bottom finger and GNNGNNANN for the top finger.
Click here to test the tool using these sequences (this is actually how we located our colorblindness targets on the genome).
Retrieved from " http://2011.igem.org/Team:Harvard/ZF_Binding_Site_Finder "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
