<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Harvard_Template_NotebookData"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Harvard/Template:NotebookData">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Harvard/Template:NotebookData&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Harvard/Template:NotebookData&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Harvard/Template:NotebookData&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Harvard/Template:NotebookData" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Harvard/Template:NotebookData</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="606" style="display:none"><H2><SPAN class="mw-headline" id="June_6th">June 6th</SPAN></H2>
First day of iGEM!</DIV><DIV id="607" style="display:none"><H2><SPAN class="mw-headline" id="June_7th"> June 7th </SPAN></H2><P><B>Miniprep of pKD42 (lambda red)</B></P><P>The lambda red plasmid is needed to enable the recombination used to insert the selection/expression systems into our E. coli cultures.
</P><P>Procedure: followed Qiagen Kit instructions, each student (8) using 1 mL cell suspension 
</P>
Results: DNA reasonably pure (260/280 between 1.8 and 2) and between 25 and 50 ng/µL</DIV><DIV id="608" style="display:none"><H2><SPAN class="mw-headline" id="June_8th"> June 8th </SPAN></H2><P><B>PCR to connect ultramers into OZ052 (Zif268 F2 triplicate, GCCGATGTC)and OZ123 (Zif268 F2 triplicate, GAGTGGTTA):</B></P><P>OZ052:
</P><UL><LI>3µL OZ052_F (10µM stock)
</LI><LI>3µL OZ052_R (10µM stock)
</LI><LI>5µL 10x Pfx amplification buffer
</LI><LI>1.5µL dNTPs
</LI><LI>1µL MgSO4
</LI><LI>0.4µL DNA polymerase
</LI><LI>36.1µL ddH2O
</LI></UL><P>OZ123:
</P><UL><LI>3µL OZ123_F (10µM stock)
</LI><LI>3µL OZ123_R (10µM stock)
</LI><LI>5µL 10x Pfx amplification buffer
</LI><LI>1.5µL dNTPs
</LI><LI>1µL MgSO4
</LI><LI>0.4µL DNA polymerase
</LI><LI>36.1µL ddH2O
</LI></UL><P>Parameters:
</P><UL><LI>1) 94⁰C for 5 min
</LI><LI>2) 94⁰C for 15 sec
</LI><LI>3) 60⁰C for 30 sec
</LI><LI>4) 68⁰C for 1 min
</LI><LI>5) Repeat 2-4 for 25 cycles
</LI><LI>6) 68⁰C for 5 min
</LI><LI>7) 4⁰C forever</LI></UL><DIV id="609" style="display:none"><H2><SPAN class="mw-headline" id="June_9th_-_Wet_Lab"> June 9th - Wet Lab </SPAN></H2><UL><LI>Created cell culture with selection construct (contains ZFB, His3, pyrF on plasmid) and reporter RFP (this will be used to test positive control ZFs, cells fluoresce green when ZF binds)
</LI><LI>Picked colonies, grew in LB/amp liquid media until mid-log
</LI><LI>3 mL of LB, 1.5 µL of 2000x amp
</LI></UL><LI>Once mid-log reached, created glycerol stock, stored stock at -80⁰C.
</LI><LI><DEL>300 µL bacteria, 1200 µL 80% glycerol </DEL><B>This should have been 1200 µL bacteria media, 300 µL 80% glycerol <I>(Corrected 6/14/2011)</I></B> (80% pure glycerol, 20% molecular grade water)
</LI><LI>Spiked new tubes of media with 25 µL bacteria from the mid-log tube to leave overnight
</LI><P>NOTE: reporter RFP did not grow to mid-log by end of day, will let grow overnight to saturation and continue creating glycerol stock tomorrow.
</P><UL><LI>Plated selection strain from gel stab onto tet plate.
</LI></UL><UL><LI>Began primer design for creating the kan/selection construct fusion.
</LI></UL><H2><SPAN class="mw-headline" id="June_9th_-_Bioinformatics">June 9th - Bioinformatics</SPAN></H2><P>Today we focused on reacquainting and familiarizing ourselves with Python. We completed the parsing (reading in) of the sequence and amino acid data so that it is easy to work with: by substituting each amino acid abbreviation (ex. A, N) with its numeric equivalent (ex. 1, 14), we can use a lot of nice math comparisons instead of messy letter/&quot;string&quot; comparisons.
</P><P>After that, we worked on counting the number of times each amino acid appears in each of the 7 positions (unfortunately given by -1,1,2,3,5,6,7), and counting the number of times amino acids are next to each other (ex. ACTQRNF has AC, CT, TQ, etc pairings). Taken overall, we found that L is overwhelmingly in position 5. 
</P><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Acid</B></TD><TD align="center" style="background:#f0f0f0;"><B>-1</B></TD><TD align="center" style="background:#f0f0f0;"><B>1</B></TD><TD align="center" style="background:#f0f0f0;"><B>2</B></TD><TD align="center" style="background:#f0f0f0;"><B>3</B></TD><TD align="center" style="background:#f0f0f0;"><B>5</B></TD><TD align="center" style="background:#f0f0f0;"><B>6</B></TD><TD align="center" style="background:#f0f0f0;"><B>7</B></TD></TR><TR><TD> A</TD><TD>77</TD><TD>140</TD><TD>210</TD><TD>197</TD><TD>0</TD><TD>312</TD><TD>85
</TD></TR><TR><TD> C</TD><TD>12</TD><TD>24</TD><TD>1</TD><TD>6</TD><TD>14</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> D</TD><TD>413</TD><TD>16</TD><TD>694</TD><TD>258</TD><TD>0</TD><TD>142</TD><TD>14
</TD></TR><TR><TD> E</TD><TD>125</TD><TD>74</TD><TD>152</TD><TD>107</TD><TD>0</TD><TD>58</TD><TD>132
</TD></TR><TR><TD> F</TD><TD>0</TD><TD>0</TD><TD>22</TD><TD>0</TD><TD>10</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> G</TD><TD>12</TD><TD>201</TD><TD>328</TD><TD>125</TD><TD>0</TD><TD>177</TD><TD>62
</TD></TR><TR><TD> H</TD><TD>93</TD><TD>144</TD><TD>232</TD><TD>652</TD><TD>0</TD><TD>51</TD><TD>17
</TD></TR><TR><TD> I</TD><TD>70</TD><TD>21</TD><TD>3</TD><TD>26</TD><TD>0</TD><TD>94</TD><TD>73
</TD></TR><TR><TD> K</TD><TD>108</TD><TD>372</TD><TD>46</TD><TD>169</TD><TD>6</TD><TD>321</TD><TD>52
</TD></TR><TR><TD> L</TD><TD>176</TD><TD>37</TD><TD>20</TD><TD>22</TD><TD>3325</TD><TD>75</TD><TD>55
</TD></TR><TR><TD> M</TD><TD>36</TD><TD>54</TD><TD>5</TD><TD>28</TD><TD>0</TD><TD>31</TD><TD>10
</TD></TR><TR><TD> N</TD><TD>23</TD><TD>150</TD><TD>129</TD><TD>940</TD><TD>0</TD><TD>182</TD><TD>61
</TD></TR><TR><TD> P</TD><TD>3</TD><TD>298</TD><TD>77</TD><TD>7</TD><TD>0</TD><TD>36</TD><TD>8
</TD></TR><TR><TD> Q</TD><TD>813</TD><TD>158</TD><TD>180</TD><TD>13</TD><TD>0</TD><TD>136</TD><TD>30
</TD></TR><TR><TD> R</TD><TD>870</TD><TD>539</TD><TD>137</TD><TD>55</TD><TD>3</TD><TD>428</TD><TD>2517
</TD></TR><TR><TD> S</TD><TD>99</TD><TD>970</TD><TD>859</TD><TD>278</TD><TD>0</TD><TD>140</TD><TD>12
</TD></TR><TR><TD> T</TD><TD>243</TD><TD>134</TD><TD>223</TD><TD>350</TD><TD>0</TD><TD>834</TD><TD>83
</TD></TR><TR><TD> V</TD><TD>166</TD><TD>26</TD><TD>27</TD><TD>115</TD><TD>0</TD><TD>341</TD><TD>146
</TD></TR><TR><TD> W</TD><TD>19</TD><TD>0</TD><TD>13</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> Y</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>10</TD><TD>0</TD><TD>0</TD><TD>1
</TD></TR></TBODY></TABLE><P>For pairings, we found patterns, but none as obvious as the L-in-position-5. Read this like a multiplication table: the intersection of L row and M column is how often that pairing was observed.
</P><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;">'
</TD><TD align="center" style="background:#f0f0f0;"><B>A</B></TD><TD align="center" style="background:#f0f0f0;"><B>C</B></TD><TD align="center" style="background:#f0f0f0;"><B>D</B></TD><TD align="center" style="background:#f0f0f0;"><B>E</B></TD><TD align="center" style="background:#f0f0f0;"><B>F</B></TD><TD align="center" style="background:#f0f0f0;"><B>G</B></TD><TD align="center" style="background:#f0f0f0;"><B>H</B></TD><TD align="center" style="background:#f0f0f0;"><B>I</B></TD><TD align="center" style="background:#f0f0f0;"><B>K</B></TD><TD align="center" style="background:#f0f0f0;"><B>L</B></TD><TD align="center" style="background:#f0f0f0;"><B>M</B></TD><TD align="center" style="background:#f0f0f0;"><B>N</B></TD><TD align="center" style="background:#f0f0f0;"><B>P</B></TD><TD align="center" style="background:#f0f0f0;"><B>Q</B></TD><TD align="center" style="background:#f0f0f0;"><B>R</B></TD><TD align="center" style="background:#f0f0f0;"><B>S</B></TD><TD align="center" style="background:#f0f0f0;"><B>T</B></TD><TD align="center" style="background:#f0f0f0;"><B>V</B></TD><TD align="center" style="background:#f0f0f0;"><B>W</B></TD><TD align="center" style="background:#f0f0f0;"><B>Y</B></TD></TR><TR><TD> A</TD><TD>10</TD><TD>0</TD><TD>99</TD><TD>55</TD><TD>0</TD><TD>29</TD><TD>122</TD><TD>20</TD><TD>32</TD><TD>332</TD><TD>2</TD><TD>59</TD><TD>55</TD><TD>63</TD><TD>255</TD><TD>87</TD><TD>24</TD><TD>43</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> C</TD><TD>0</TD><TD>0</TD><TD>15</TD><TD>0</TD><TD>0</TD><TD>3</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>5</TD><TD>0</TD><TD>0</TD><TD>6</TD><TD>0</TD><TD>31</TD><TD>6</TD><TD>14</TD><TD>0</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> D</TD><TD>99</TD><TD>15</TD><TD>94</TD><TD>92</TD><TD>0</TD><TD>39</TD><TD>62</TD><TD>6</TD><TD>84</TD><TD>342</TD><TD>15</TD><TD>120</TD><TD>55</TD><TD>42</TD><TD>277</TD><TD>290</TD><TD>87</TD><TD>21</TD><TD>0</TD><TD>8
</TD></TR><TR><TD> E</TD><TD>55</TD><TD>0</TD><TD>92</TD><TD>42</TD><TD>0</TD><TD>34</TD><TD>77</TD><TD>1</TD><TD>38</TD><TD>141</TD><TD>2</TD><TD>39</TD><TD>4</TD><TD>29</TD><TD>134</TD><TD>28</TD><TD>90</TD><TD>26</TD><TD>0</TD><TD>1
</TD></TR><TR><TD> F</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>10</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>22</TD><TD>4</TD><TD>0</TD><TD>2</TD><TD>4</TD><TD>6</TD><TD>0</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> G</TD><TD>29</TD><TD>3</TD><TD>39</TD><TD>34</TD><TD>0</TD><TD>38</TD><TD>56</TD><TD>0</TD><TD>14</TD><TD>126</TD><TD>1</TD><TD>95</TD><TD>28</TD><TD>47</TD><TD>119</TD><TD>125</TD><TD>38</TD><TD>7</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> H</TD><TD>122</TD><TD>0</TD><TD>62</TD><TD>77</TD><TD>0</TD><TD>56</TD><TD>118</TD><TD>9</TD><TD>103</TD><TD>498</TD><TD>4</TD><TD>88</TD><TD>24</TD><TD>26</TD><TD>87</TD><TD>159</TD><TD>70</TD><TD>2</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> I</TD><TD>20</TD><TD>0</TD><TD>6</TD><TD>1</TD><TD>10</TD><TD>0</TD><TD>9</TD><TD>6</TD><TD>8</TD><TD>95</TD><TD>3</TD><TD>5</TD><TD>17</TD><TD>3</TD><TD>62</TD><TD>16</TD><TD>17</TD><TD>4</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> K</TD><TD>32</TD><TD>0</TD><TD>84</TD><TD>38</TD><TD>0</TD><TD>14</TD><TD>103</TD><TD>8</TD><TD>84</TD><TD>386</TD><TD>24</TD><TD>44</TD><TD>19</TD><TD>102</TD><TD>269</TD><TD>163</TD><TD>113</TD><TD>22</TD><TD>1</TD><TD>0
</TD></TR><TR><TD> L</TD><TD>332</TD><TD>5</TD><TD>342</TD><TD>141</TD><TD>0</TD><TD>126</TD><TD>498</TD><TD>95</TD><TD>386</TD><TD>174</TD><TD>32</TD><TD>686</TD><TD>16</TD><TD>112</TD><TD>362</TD><TD>276</TD><TD>875</TD><TD>360</TD><TD>0</TD><TD>8
</TD></TR><TR><TD> M</TD><TD>2</TD><TD>0</TD><TD>15</TD><TD>2</TD><TD>0</TD><TD>1</TD><TD>4</TD><TD>3</TD><TD>24</TD><TD>32</TD><TD>0</TD><TD>7</TD><TD>2</TD><TD>11</TD><TD>39</TD><TD>14</TD><TD>3</TD><TD>1</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> N</TD><TD>59</TD><TD>0</TD><TD>120</TD><TD>39</TD><TD>22</TD><TD>95</TD><TD>88</TD><TD>5</TD><TD>44</TD><TD>686</TD><TD>7</TD><TD>8</TD><TD>36</TD><TD>28</TD><TD>120</TD><TD>254</TD><TD>84</TD><TD>34</TD><TD>1</TD><TD>0
</TD></TR><TR><TD> P</TD><TD>55</TD><TD>6</TD><TD>55</TD><TD>4</TD><TD>4</TD><TD>28</TD><TD>24</TD><TD>17</TD><TD>19</TD><TD>16</TD><TD>2</TD><TD>36</TD><TD>0</TD><TD>3</TD><TD>29</TD><TD>150</TD><TD>21</TD><TD>13</TD><TD>11</TD><TD>0
</TD></TR><TR><TD> Q</TD><TD>63</TD><TD>0</TD><TD>42</TD><TD>29</TD><TD>0</TD><TD>47</TD><TD>26</TD><TD>3</TD><TD>102</TD><TD>112</TD><TD>11</TD><TD>28</TD><TD>3</TD><TD>100</TD><TD>261</TD><TD>314</TD><TD>125</TD><TD>19</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> R</TD><TD>255</TD><TD>31</TD><TD>277</TD><TD>134</TD><TD>2</TD><TD>119</TD><TD>87</TD><TD>62</TD><TD>269</TD><TD>362</TD><TD>39</TD><TD>120</TD><TD>29</TD><TD>261</TD><TD>618</TD><TD>343</TD><TD>504</TD><TD>281</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> S</TD><TD>87</TD><TD>6</TD><TD>290</TD><TD>28</TD><TD>4</TD><TD>125</TD><TD>159</TD><TD>16</TD><TD>163</TD><TD>276</TD><TD>14</TD><TD>254</TD><TD>150</TD><TD>314</TD><TD>343</TD><TD>592</TD><TD>173</TD><TD>91</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> T</TD><TD>24</TD><TD>14</TD><TD>87</TD><TD>90</TD><TD>6</TD><TD>38</TD><TD>70</TD><TD>17</TD><TD>113</TD><TD>875</TD><TD>3</TD><TD>84</TD><TD>21</TD><TD>125</TD><TD>504</TD><TD>173</TD><TD>154</TD><TD>28</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> V</TD><TD>43</TD><TD>0</TD><TD>21</TD><TD>26</TD><TD>0</TD><TD>7</TD><TD>2</TD><TD>4</TD><TD>22</TD><TD>360</TD><TD>1</TD><TD>34</TD><TD>13</TD><TD>19</TD><TD>281</TD><TD>91</TD><TD>28</TD><TD>12</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> W</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>1</TD><TD>0</TD><TD>0</TD><TD>1</TD><TD>11</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> Y</TD><TD>0</TD><TD>0</TD><TD>8</TD><TD>1</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>8</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0
</TD></TR></TBODY></TABLE>
Follow up work on this will be to convert this table to frequencies instead of values: values are less meaningful. </DIV><DIV id="610" style="display:none"><H2><SPAN class="mw-headline" id="June_10th_-_Wet_Lab"> June 10th - Wet Lab </SPAN></H2><UL><LI>What we learned today: don't put E. coli plates in the -20C freezer!
</LI></UL><UL><LI>Observed a well populated selection strain plate and placed it in the 4C refrigerator
</LI></UL><UL><LI>Took the selection construct culture and extracted the plasmid through miniprep
</LI><LI>Observed 260/280 ratio of 1.90 and 1.88 through Nanodrop
</LI><LI>Observed concentrations of 87.7 and 100.6 ng/µL through Nanodrop
</LI></UL><UL><LI>Made 10 new agar plates with LB and amp
</LI></UL><H2><SPAN class="mw-headline" id="June_10th_-_Bioinformatics">June 10th - Bioinformatics </SPAN></H2><H3><SPAN class="mw-headline" id="Visualizations">Visualizations</SPAN></H3><P>We spent the first few hours today making cool visualizations and graphs of the data we found on the 9th: heatmaps turned out to be an annoying limitation of Python, so a Python/R hybrid was used, and bar charts were made exclusively in Python. See the dropbox for our pretty (and hopefully informative compared to spreadsheets) charts/graphs.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">A heatmap of the pairing data. The darker the blues indicate that the pairing occurs more often.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>We then started work on TNN and GNN properties specifically (essentially repeating the June 9th work, but confined to smaller data sets). There are some differences between TNN and GNN: see graphs in dropbox. We decided that there was not enough data for fingers that bind to ANN and CNN triplets to perform significant analysis on it.  
</P><TABLE><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">A heatmap of the GNN pairing data.</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">A heatmap of the TNN pairing data.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><UL><LI>Overall, similar color clusters are found in the heatmaps. In all cases, L and N are often placed consecutively on the helix. There are fewer clusters of high frequency when looking at TNN binders. 
</LI></UL><P>
We then, using the theorized framework from a paper (2011 Persikov <A href="http://iopscience.iop.org.ezp-prod1.hul.harvard.edu/1478-3975/8/3/035010/" class="external autonumber" rel="nofollow">[1]</A>), tried to match amino acid binding to each base pair to see if there was a pattern. See dropbox document .../bioinformatics/Binding Frequency for that data. There's a lot of it.
</P><H3><SPAN class="mw-headline" id="Properties_of_amino_acids">Properties of amino acids</SPAN></H3><P>We then worked on finding properties of the each position (hydrophobic/phillic, non/polar):
</P><P><B>Hydrophilic vs Hydrophobic</B></P><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Position</B></TD><TD align="center" style="background:#f0f0f0;"><B>Very Phobic</B></TD><TD align="center" style="background:#f0f0f0;"><B>Hydrophobic</B></TD><TD align="center" style="background:#f0f0f0;"><B>Neutral</B></TD><TD align="center" style="background:#f0f0f0;"><B>Hydrophillic</B></TD></TR><TR><TD> 6</TD><TD>285</TD><TD>85</TD><TD>204</TD><TD>2782
</TD></TR><TR><TD> 5</TD><TD>542</TD><TD>312</TD><TD>1334</TD><TD>1169
</TD></TR><TR><TD> 4</TD><TD>3334</TD><TD>14</TD><TD>0</TD><TD>9
</TD></TR><TR><TD> 3</TD><TD>191</TD><TD>203</TD><TD>1417</TD><TD>1536
</TD></TR><TR><TD> 2</TD><TD>91</TD><TD>211</TD><TD>1819</TD><TD>1236
</TD></TR><TR><TD> 1</TD><TD>138</TD><TD>164</TD><TD>1604</TD><TD>1451
</TD></TR><TR><TD> -1</TD><TD>468</TD><TD>90</TD><TD>1257</TD><TD>1542
</TD></TR></TBODY></TABLE><P><B>Polar vs Nonpolar</B></P><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Position</B></TD><TD align="center" style="background:#f0f0f0;"><B>Polar</B></TD><TD align="center" style="background:#f0f0f0;"><B>Nonpolar</B></TD></TR><TR><TD> 6</TD><TD>2917</TD><TD>440
</TD></TR><TR><TD> 5</TD><TD>2290</TD><TD>1067
</TD></TR><TR><TD> 4</TD><TD>9</TD><TD>3348
</TD></TR><TR><TD> 3</TD><TD>2830</TD><TD>527
</TD></TR><TR><TD> 2</TD><TD>2652</TD><TD>705
</TD></TR><TR><TD> 1</TD><TD>2555</TD><TD>802
</TD></TR><TR><TD> -1</TD><TD>2784</TD><TD>573
</TD></TR></TBODY></TABLE>
Follow up work here is to check more properties, and maybe try individual pairings (ex. phobic-philic, polar-phillic).</DIV><DIV id="613" style="display:none"><H2><SPAN class="mw-headline" id="June_13th_-_Wet_Lab"> June 13th - Wet Lab </SPAN></H2><P>The control zinc fingers OZ052 and OZ123 were amplified with overhanging primers to allow its insertion into the Wolfe plasmid:
</P><P><B>Overhang PCR for ultramers:</B> the template was the product of the ultramer PCR (see 6/8/11), and several concentrations were used
</P><P>In all the tubes:
</P><UL><LI>5 µL Pfx amplification buffer
</LI><LI>1.5 µL dNTPs
</LI><LI>1 µL MgSO4
</LI><LI>0.4 µL polymerase
</LI><LI>38.1 µL ddH2O
</LI><LI>1.5 µL OZ052_up and 1.5 µL OZ052_down OR 1.5 µL OZ123_up and 1.5 µL OZ123_down
</LI></UL><P>In OZ052 (1) and OZ123 (1):
</P><UL><LI>1 µL of ultramer PCR product
</LI></UL><P>In OZ052 (1:10) and OZ123 (1:10):
</P><UL><LI>1 µL of a 1 in 10 dilution of ultramer PCR product
</LI></UL><P>In OZ052 (1:100) and OZ123 (1:100):
</P><UL><LI>1 µL of a 1 in 100 dilution of ultramer PCR product
</LI></UL><P>Parameters:
</P><UL><LI>94⁰C for 5 min
</LI><LI>94⁰C for 15 sec
</LI><LI>55⁰C for 30 sec
</LI><LI>68⁰C for 30 sec
</LI><LI>Repeat steps 2-4 for 25 cycles
</LI><LI>68⁰C for 5 min
</LI><LI>4⁰C forever
</LI></UL><P>Gel to verify proper amplification (1% agarose gel, 10 µL 1 kb ladder, 120 V):
</P><P>The OZ052 lanes (1-3) had bands at the proper length (328 bp) at all three concentrations, although there were several fainter bands likely from side products. Only the undiluted OZ123 lane showed any bands, and from the faint band at 328 and the stronger band around 250 it appears that the PCR did not work well, and the majority of the product was the ultramer from the first PCR. 
</P><P><B>PCR around vector:</B> the template used was the Wolfe selection construct plasmid miniprepped 6/10/11 (100.6 ng/µL stock)
Reagents the same as above except:
</P><UL><LI>1.5 µL of Wolfe_F and 1.5 µL of Wolfe_R primers to each tube
</LI><LI>plasmid tube (1 ng) given 1 ng of template (1 µL of a 1 in 100 dilution)
</LI><LI>plasmid tube (10 ng) given 10 ng of template (1 µL of a 1 in 10 dilution)
</LI></UL><P>Parameters same as above except:
</P><UL><LI>elongation (step 4) 5 minutes (vector approximately 5 kb)
</LI></UL><P>Gel to verify proper amplification (1% agarose, 10 µL 1 kb ladder, 170V)
</P><P>There were no bands of the correct size in the lanes.  The only band that appeared was a faint, short band in one lane that likely was a primer.  Since the DNA ladder worked, the problem likely was not with the electrophoresis but with the PCR reaction, perhaps due to issues with the primers.
</P><H3><SPAN class="mw-headline" id="Gel_images">Gel images</SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Ultramer Overhang 6/13/11</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Backbone plasmid 6/13/11</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="June_13th_-_Bioinformatics"> June 13th - Bioinformatics </SPAN></H2><P>Today we started work on a program to statistically generate possible sequences.
</P><P>The four functions needed to do this are:
</P><UL><LI> generate(matrix, pseudocounts (lambda), dependency tuples)
</LI></UL><DL><DD> takes a matrix of zinc-finger AA position counts, a list of dependent amino acid pairs, and a pseudocount multiplier and generates a list of potential amino acid sequences weighted by independent and dependent probabilities
</DD></DL><UL><LI> add_pseudo(dependent matrix row,independent matrix row)
</LI></UL><DL><DD> given a matrix row of dependent counts (i.e. how many times 'a' occurs at position n when 'b' is set to some AA at position m) and a row of independent matrix counts (how many times 'a' occurs at n regardless of b's AA) return an adjusted matrix row, based on the dependent matrix row, that has pseudocounts added to the values that are empty in the dependent matrix row but filled in the independent matrix row. 
</DD></DL><UL><LI> generate_indep(matrix)
</LI></UL><DL><DD> randomly pick an amino acid, given a matrix row, from a weighted random distribution based on the values in the row
</DD></DL><UL><LI> generate_dep(indep_row, dep_row, lambda)
</LI></UL><DL><DD> add pseudo counts (call add_pseudo) and generate a dependent random call for a position (using generate_indep on the adjusted matrix)
</DD></DL>
We finished generate_indep, generate_dep, and add_pseudo today, along with creating a 140x140 matrix of needed values.</DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Harvard/Template:NotebookData">http://2011.igem.org/Team:Harvard/Template:NotebookData</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Harvard/Template:NotebookData" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Harvard/Template:NotebookData" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Harvard/Template:NotebookData&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Harvard/Template:NotebookData&amp;oldid=73805" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>