"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Harvard/Results/Design
From 2011.igem.org
bar
Overview | Biobricks | Source Code | Acknowledgments | Accomplishments
Contents
1 Bioinformatics
1.1 55,000 Potential Zinc Fingers
2 Zinc Finger Binding Site Finder
2.1 Background
Bioinformatics
55,000 Potential Zinc Fingers
We made 55,000 sequences, distributed evenly among 6 DNA target triplets. That's 9150 per target.
Because our program's output changes dramatically based on the input triplet, no two sets of sequences are the same:
AAA
ACC
CTC
CTG
GAC
TGG
Zinc Finger Binding Site Finder
Check out our Zinc Finger Binding Site Finder Tool .
Background
This tool is designed specifically to find binding sites for zinc finger nucleases (ZFNs) in order to create double stranded breaks in DNA.  Ultimately, through these double stranded breaks, ZFNs allow genome editing with the insertion or deletion of genes at very specific target DNA sites.  Each ZFN recognizes and binds to a specific 9-bp DNA sequence that is unique to each ZFN, and the binding sites are arranged such that two ZFNs flank the cut site on opposite strands of DNA:
Given two 9-bp DNA sequences, this program will search a string of DNA to find a configuration such that these two 9-bp sequences are located on opposite strands with a short gap in between them (5-7 bp long), where the double stranded break will occur.  This is useful due to the fact that we currently do not have a library of ZFNs that spans the space of all 9-bp recognition sites.  Therefore, if the 9-bp binding sites for two zinc fingers are known and well-characterized, this program will search for a site in which these two known zinc fingers can be used to make a cut for gene insertion or deletion.
Retrieved from " http://2011.igem.org/Team:Harvard/Results/Design "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
