"
Page
Discussion
View source
History
teams
Log in
 
Team:Harvard/Template:NotebookDataJuly4
From 2011.igem.org
Contents
1 July 23rd
1.1 Team Wolfe
2 July 24th
2.1 Team ZF
2.1.1 Sequencing Results
3 July 25th
3.1 Team TolC
3.1.1 Sequencing of Jun's rpoZ MAGE knockout
3.1.2 Lambda Red Recombination of zeocine into TolC(assistance from Naomi, Cub team member)
3.1.3 rpoZ MAGE knockout attempt...yet again
3.2 Team Wolfe Cub
3.3 Team ZF
3.3.1 Result of OZ052 and OZ123 Transformation
3.3.2 Glycerol stockmaking
3.3.3 Restriction enzyme test
3.3.4 Ultramer Construction of Pos Con 77 and Other F2/F3 positive controls
4 July 26
4.1 Team ZF
4.1.1 Isothermal Assembly of Pos Con 77, Zif 268, and OZs
4.1.2 Prepare control OZs for sequencing
4.1.3 Practice Plate resuspension
4.1.4 Colony finalization
4.2 Team TolC
4.2.1 Zeocin insertion into ECNR2
4.2.2 MAGE rpoZ knockout
4.3 Team Wolfe
5 July 27th
5.1 Team TolC
5.1.1 Culture of TolC+Kan-ZFB-wp+Zeocin
5.2 Team ZF
5.2.1 Checking the OZ sequencing results
5.2.2 Transformation of Pos Con 77, OZs, Zif 268
5.3 Team Wolfe
6 July 28th
6.1 Team Wolfe
6.2 Team TolC
6.2.1 cultures of ECNR2+Kan-ZFB-wp+zeocin from F9,F11, for glycerol stocks and transformation
6.2.2 Sequencing
6.2.3 Plasmid transformation
6.3 Team ZF
6.3.1 Checking on Pos Con 77, OZs, and Zif 268 plates
6.3.2 Designing the reporter strain
6.3.3 PCR with Practice Plate
6.3.4 Glycerol stocks of successful plasmids
7 July 29th
7.1 Team Wolfe
7.2 Team TolC
7.2.1 Sequencing of Kan+ZFB+wp and Zeocin insertions
7.2.2 Zif268 Plasmid Insertion into ECNR2+Kan-ZFB-wp+zeocin
7.3 Team ZF
7.3.1 Transformation plating results
7.3.2 Amplification and purification of practice plate oligos
7.3.3 Practice plate primer tag test
8 July 30th
8.1 Team TolC
8.2 Team ZF
8.2.1 CB bottom miniprep
9 July 31st
9.1 Team ZF
9.1.1 Primer tag test two
9.1.2 Preparing for another CB bottom miniprep
July 23rd
Team Wolfe
Lambda red results: no real colonies, only cheaters.
July 24th
Team ZF
Sequencing Results
We got back our sequencing results from the samples we sent out on Friday, and the results are summarized in the table below.  The checkmarks indicate whether the sample is adequate for usage or not.
FH top (77)
Notes
✓
77.1
Only G -> C in ala
✓
77.3
Only G -> C in ala
77.4
Single deletion in F3
FH bottom (78)
78.1
Many, many SNPs throughout, oddly F and R differ
78.2
Many, many SNPSs, R starts in a strange place
78.3
Four deletions throughout on both F and R
78.4
Many, many SNPs throughout
Original
Only one SNP in F3, but unacceptable
CB top (79)
79.1
Many, many SNPs and deletions throughout, F and R the same
✓
79.2
No SNPs
✓
79.3
No SNPs
79.4
G -> C in ala, deletion in the Q of the F3 \"FSQ\"
CB bottom (80)
✓
80.1
No SNPs
80.2
G -> C in ala, deletion in the homology region shortly after F3 ends
80.4
G -> C in ala, five deletions in F3/homology region
Myc 198 (81)
✓
81.1
No SNPs
✓
81.4
Only G -> C in ala
Myc 981 (82)
✓
82.1
Only G -> C in ala
Pos con 16 (83)
✓
83.4
Only G -> C in ala
Pos con 55 (84)
✓
84.1
Only G -> C in ala
84.4
G -> C in ala, two deletions
Zif268
✓
Zif268
No SNPs
July 25th
Team TolC
Sequencing of Jun's rpoZ MAGE knockout
Prepared six products and primers for sequencing
Sending Jun colony 4, PCR's 1-mutant_R, 2-mutant_R, 3-mutant_R, 4-mutant_R, 2-mutant_Int_R, and 4-mutant_Int_R
delayed until tomorrow (July 25, 2011) because pickup for the day already done.
Lambda Red Recombination of zeocine into TolC(assistance from Naomi, Cub team member)
PCR out zeocin cassette (replacing the rpoZ, for selection) from Wolfe Strain
Used primers rpoZ_F and rpoz_R, expecting a band about 1500bp? Total length about 939bp, including the promoters, according to [ Roth lab @ UC Davis, Zeocin ]
Ran 8, 25 µL solutions with KAPA
PCR protocol(PCR 7 machine, "ZEOC")
95˚ C, for 5 min
98˚ C, for 20 sec
62˚ C, for 15 sec
72˚ C, for 45 sec
Repeat 29 times
72˚ C, for 5 min
4˚ C, forever
Gel extracted the PCR product NOTE. Since the gel, was so clean, we may do a PCR clean up next time, instead of running the PCR product through a gel for gel extraction. Just make sure to use these exact same parameters.
Nanodrop
156.2 ng/µL and 107.6 ng/µL, each in 30ul of water (distilled, deionised- molecular grade)
Performed lambda red recombination
Time constant of 5.6 ms
rpoZ MAGE knockout attempt...yet again
Used new oligo to perform a MAGE knockout of rpoZ
Oligo named rpoZ_MAGE_Subs
Electroporator read 5.2 ms
Allowed for 3 hours of growth and spread on zeocine plate
Team Wolfe Cub
Plate reader is still being repaired--we will have to wait to check the growth curves
Since plans A and B are not working well at the moment, we will go ahead and start preparing for plan C (using a plasmid). Ordered primers that will clone out the pSB4K5 (a biobrick vector with a kan cassette, http://partsregistry.org/Part:pSB4K5 ) backbone with homology to the ZFB-wp-his3-ura3 construct. We will then connect the two with Gibson assembly.
To this end, we need to use the biobrick DNA sent to us to transform chemically competent E coli.  Unfortunately the 384-well plates that they are stored in could not be found because Andrew is out of town--hopefully they will turn up tomorrow.
For the rest of the day, we joined team TolC for gel extraction, MAGE, and lambda red!
Team ZF
Result of OZ052 and OZ123 Transformation
Last Friday we plated out the transformed bacteria for the two OZ controls in 10 ul and 50 ul volumes.  They both yielded colonies, but the 10 ul plates only had a handful while the 50 ul plates had around 15-20 colonies each.  We picked four colonies from each plate and grew them in LB/spec so that tomorrow we can perform a miniprep and send them off for sequencing.  We should know the final verdict by Wednesday regarding which ones are good or bad.
Glycerol stockmaking
We got the sequencing results back from last Friday, and took the following colonies to make into glycerol stock solutions: 77.1, 79.2, 80.1, 81.1 (x2 for both FH bottom and Myc 198), 82.1, 83.4, 84.1, Zif268.  These colonies are a subset of the "good" colonies from our sequencing results from Sunday.  We mixed 120 ul of bacteria in mid-log with 300 ul of 80% glycerol to make the final solution, which was stored at -80 degrees C.
Restriction enzyme test
We tested both XbaI and BsaI on the junction PCR product using colony 79.4 as a template (which was used because it was a "bad" sequence overall but had intact BsaI and XbaI cut sites).  The junction PCR product is 1326 bp, and we expected to see bands of 593 and 739 bp with XbaI, and 717 and 578 bp with BsaI.  Our restriction digest recipe consisted of the following per reaction:
1 ul 10X buffer (NE biolabs buffer 4)
7.9 ul H2O
1 ul DNA (~100 ng)
1 ul enzyme (~2 units)
1 ul BSA (100x)
The two gels we ran for this digest showed that XbaI did NOT work, which may be due to XbaI being too old.  However, we did get double bands both times for BsaI.  Our egel was strange because of uneven loading, and so we were unable to determine the true size of both bands, but the agarose gel we ran later showed the two bands in the right positions.  The two gels can be seen below:
E-gel of our first digestion. The gel looks unusual (notice the bubble-like formation), so we performed a second digestion.
1% agarose gel of our second digestion. Although the bands are faint, we can see the appropriately sized bands for Bsa1.
Ultramer Construction of Pos Con 77 and Other F2/F3 positive controls
We redid the ultramer overlap PCR for Pos Con 77, this time with the correct reverse sequence.  We also assembled ultramers for Zif268, OZ052 and OZ123 (these are the plasmids that will have the type II cut sites in them along with F2/F3 only, as opposed to having the entire finger already assembled).  For the Zif268 reaction, an annealing temperature of 65*C was used, and for the rest, the annealing temperature was 58*C.  For all reactions, the extension time was 15 seconds.
The following primers were used for each reaction (index taken from the Dropbox primer index):
Pos con 77 - 76, 122
Zif268 - 123, 124
OZ052 - 76, 125
OZ123 - 76, 126
July 26
Team ZF
Isothermal Assembly of Pos Con 77, Zif 268, and OZs
Today we checked the PCR products of the ultramer reaction from yesterday.  They did not look good.  We repeated the reaction using a touchdown PCR beginning with an annealing temperature of 70 degrees and decreasing 2 degrees every other cycle until hitting a minimum of 58 degrees, and we brought the total number of cycles to 25.  Extension time was 30 seconds.  This looked great.  The pictures for the two reactions can be seen below:
Our first PCR was extremely streaky, so we decided to redo it.
We did a touchdown PCR, and our product looks better, although it's still streaky.
Before the isothermal assembly, we did a PCR purification and measured the concentrations of the ultramer reaction products:
Concentrations of our PCR product after PCR purification. This was used in our isothermal assembly.
We then performed an isothermal assembly, using the following mix for Pos con 77, OZ052, and OZ123:
0.63 ul of spec backbone at ~157 ng/ul
0.3 ul of ultramer (~50 ng/ul)
0.13 ul of omega subunit/linker
1.44 ul of H2O
7.5 ul of isothermal mix
For Zif268:
0.63 ul of spec backbone
0.19 ul of Zif (~90 ng/ul)
0.13 ul of omega/linker
1.55 of H2O
7.5 ul of isothermal mix
This was left at 50 degrees for an hour, then stored at 4 degrees.
Prepare control OZs for sequencing
We performed a miniprep on the eight samples for sequencing, obtaining the following concentrations:
Our OZ concentrations after miniprep.
The miniprep lysis step looked good, and the samples turned blue after lysis.
We then performed a junction PCR using the previous protocol, and checked the products on a gel:
Gel of our PCR product that we sent out for sequencing.
They looked good, so we sent them out for sequencing.  Hopefully we will have our final OZ control plasmids for tomorrow.
Practice Plate resuspension
We resuspended the practice plate oligos in 40 ul of water to produce a 100 uM solution, taking 10 ul of the final solutions into 90 ul of water in separate tubes to create 10 uM working concentrations.  When we receive our tag primers, we should be able to test the presence of the oligos in each tube as well as amplify them in preparation for integration into the expression plasmids.
Colony finalization
For the colonies that we plated yesterday, we saw that the plates all had good growth on them and so we wrapped the plates in parafilm and put them at 4 degrees for storage.  1/1000 ul from saturated culture seems to be a good amount to plate.
Team TolC
Zeocin insertion into ECNR2
Plated 3 mL of our recombination culture on zeocin plate, and also plated 2 mL of MAGE results on zeocin plate as a control
The recombination plate had about 15 colonies around 9 30am, and the MAGE plate had none
Continued by PCR of 4 colonies on recombination plate
Each template was analysed with presence of different sized bands for rpoZ (using rpoZ_F and rpoZ_R) and presence of Zeocin cassette (using rpoZ_R and zeocin_R)
Gel resulted with the following:
Zeocin insert check 7/26/11
SUCCESS!!! Yippee!!!
NOTE. Since there was a bit of non specific annealing, we might want to use a higher annealing temperature of 64C instead of 62C
Decided to run PCR for the Kan insertion just to make sure it was still present in the 4 colonies we chose, and it is!!!
Kan-ZFB-wp insert check 7/26/11
WERE READY TO ROOOCCKKK!!!!!
MAGE rpoZ knockout
Plated 1:10 and 1:100 dilution of MAGE results, and this morning at 10 am, they both had cells and in appropriate proportions
If above PCR and gel of zeocin insertion does not appear to be successful, then we will run 50 PCR on 50 different colonies on the 1:100 dilution plate of MAGE rpoZ knockout
Team Wolfe
Plate reader: trying to get in contact with Christian Daly who runs the Spectramax plate reader in the Bauer facility (Northwest basement). Hopefully we will get access soon.
Selection system plasmid: still trying to get hold of the biobricks. Designed Quikchange primers to change the zinc finger binding sites.
Lambda red recombination: gave lambda red one last chance, this time with a positive control
negative control (water): plated 1mL
positive control (kan cassette from Wyss): 50ng DNA, plated 1mL and 100uL
thio kan-ZFB-wp-his3-ura3: 180ng DNA, plated 1mL
July 27th
Impromptu meeting for all teams, TFs, and advisors in NW labs basement, to discuss "how we will move forward to test the selection, now that we have a TolC strain that should work for selection and a His3Ura3 plasmid-based selection that is almost complete," and to lay out and assign a lab to-do for iGEM parts, selection systems, and what to do in preparation for The Chip to arrive/backup plan in case it arrives too late (see notes in the dropbox under "Selection Meeting 7-27-11.docx")
Team TolC
Culture of TolC+Kan-ZFB-wp+Zeocin
Did not realize that colonies needed to be grown up without antibiotic before being grown in Kanamycin, so all the three colonies from yesterday passed away and didn't make it through the night :(, so we grew up 4 more colonies and ran PCR and gel to solidify the presence of the necessary insertions
Team ZF
Checking the OZ sequencing results
OZ052
Notes
✓
1
No snps
✓
2
No snps
✓
3
No snps
4
Deletion at F1
OZ123
Notes
✓
1
No snps
2
deletion in omega linker
✓
3
No snps
✓
4
No snps
We made glycerol stocks and final plates from 0Z052.1 and OZ123.1.
Transformation of Pos Con 77, OZs, Zif 268
We transformed Top 10 One Shot Electrocomp bacteria with our isothermal assembly product from yesterday. We let the cells recover for an hour, and plated 50 ul, leaving them at 37* overnight.
Team Wolfe
Morning meeting: Since TolC has gotten their selection system onto the genome, our priorities are now going to be making the plasmid-based Wolfe selection system and getting data on the plate reader. We also will eventually try to use a genome-based selection system but using the EcNR2 strain (oligos have been ordered)
Plate Reader:
We took a trip downstairs to B2 and were trained to use the Bioscreen plate reader in the Bauer Core facility.  We now have access to a working plate reader!
Bioscreen uses a 100-well honeycomb plate, so the organization was as follows:
Well 1, 11, 21, etc: LB+tet
Well 2, 12, 22, etc: NM+his
Well 3, 13, 23, etc: NM
Well 4, 14, 24, etc: NM+1mM 3-AT
Well 5, 15, 25, etc: NM+2.5mM 3-AT
Well 6, 16, 26, etc: NM+5mM 3-AT
Well 7, 17, 27, etc: LB+tet+100µM IPTG
Well 8, 18, 28, etc: NM+IPTG
Well 9, 19, 29, etc: NM+1mM 3-AT+IPTG
Well 10, 20, 30, etc: NM+5mM 3-AT+IPTG
Column 1: no cells
Column 2-3: ∆HisB∆pyrF∆rpoZ
Columns 4-10: ∆HisB∆pyrF∆rpoZ+pSR01
Electrocomp Transformation
Used Team ZF's protocol to transform electrocompetent cells with two of the pSB4K5 plasmids found in our biobricks plates (2:17H and 3:11P)
Used 10µL ddH2O to resuspend DNA in biobrick plate wells and transferred to eppendorf.
2µL DNA (which, interestingly, was red...)
plated 10µL and 100µL of each on kan plates
Quikchange primers:
Realized that we may need to consider the bases on either side of the zinc finger binding site, so we will check this and put off ordering another day.
July 28th
Team Wolfe
Plate reader and pSR01:
the Bauer plate reader seems to have worked very well, bu† unfortunately none of our pSR01-transformed colonies grew.  This is probably because the plates were a couple weeks old.  Unfortunately our glycerol stock of the strain also is missing, so we will try growing up some of the colonies in case on of them works.  Otherwise, we will grow up ∆HisB∆pyrF∆rpoZ and transform again.
Transformation: usual protocol, about 100µL DNA, recovery 2hrs, plate 1µL, 10µL, 100µL.
pSB4K5:
The kan plates from the transformation all had colonies! We chose one from 2:17H and one from 3:11P to grow up in LB+kan for miniprep and glycerol stocks.
Ordered quikchange primers: for OZ052 and OZ123 we added the context bases 5' and 3' of the binding sequence (found in OPEN supplementary materials).  Since CODA did not include the extra bases, we left them the same as the original plasmid (i.e. Zif268's).
Team TolC
cultures of ECNR2+Kan-ZFB-wp+zeocin from F9,F11, for glycerol stocks and transformation
Sequencing
Ran PCR and sent out sequencing to solidify the presence of the Kan+ZFB+wp and zeocin within the ECNR2 genome
Used rpoZ_R, rpoZ_F for PCR of the zeocin cassette (and rpoZ_F for sequencing) and TolC_Seq_R, TolC_Seq_F for PCR of the Kan-ZFB-wp insert (and tolC_seq_F for sequencing)
Plasmid transformation
Performed transformation on the Zif268 plasmid into the ECNR2 culture
Used 3 µL in a 50 µL dilution (49+1) of 1 ul of 55 ng/µL concentration of plasmids
1.82 V for 5.00 ms according to the electroporator
Team ZF
Checking on Pos Con 77, OZs, and Zif 268 plates
Unfortunately, nothing grew on our plates from yesterday. Our time constants from the electroporation were rather low (~3.8), so 50 ul may not be enough to see growth. We decided to spin down the cells, resuspend them in 50 ul of LB, and re-plated this solution. We also re-did the transformation using ChemComp cells, and made 100 ul plates. Both sets of plates were left overnight at 37*.
Designing the reporter strain
Instead of placing the GFP reporter on a separate plasmid, we decided to insert it on our expression plasmid, immediately after our zinc fingers, and before the transcription terminator. We placed a ribosome binding site (Shine Dalgarno sequence: AGGAGGTT before the GFP. We will only add the GFP to Zif 268 (whole) because if we can show through GFP that the selection system works for Zif 268, we can assume it also works for the other zinc fingers. This is easier and more cost effective than ordering 12 different primers for each of our plasmids.
We will construct the reporter strain by performing a PCR to get the plasmid in linear form. We will also perform a second PCR to get out GFP and add homology regions to our Zif 268 plasmid, as well as adding the Shine-Dalgarno sequence. The primers for these PCRs were ordered today. We'll then assemble these pieces to construct a whole plasmid, and transform our bacteria.
PCR with Practice Plate
We performed a PCR to amplify our F1s from the practice plate. We used our primers for CB bottom (primer index: 66 and 67) for the 18 targets and our primers for the controls (primer index: 72 and 73) for the 6 controls. We had an annealing temperature of 52 degrees and an extension time of 15 seconds.
Practice plate primer test
We got products for all of the oligos at our expected product size of 131 bp.
Glycerol stocks of successful plasmids
We happened to check our glycerol stocks, and noticed that the glycerol and bacteria were not mixed thoroughly. We remade all of our glycerol stocks and put them into the -80*.
July 29th
Team Wolfe
pSR01:
The colonies in the 96 well plate did grow overnight, but they were not saturated, implying that they were very unhealthy. We will PCR a few of them (1,2,3,6) just to see if they actually have the plasmid still
Transformation plates look great--we will PCR 12 colonies to verify the plasmid is present
PCR: colonies/culture diluted 1:20 as template (and colonies grown up simultaneously); M13_F and M13_R primers; 50˚C annealing and 3min elongation; 25 cycles
Results: bands are similar to the ones seen the last time we did this PCR (and those samples had also been sequence-verified). These colonies should be good to go!
pSR01 check 7/29/11
pSB4K5:
Miniprepped both cultures. Yields weren't great (in the case of 2:17H, there actually was a spill) but they are still usable.
2:17H: 6.4 ng/µL 260/280=2.2
3:11P: 9.6ng/µL 260/280=2.4
PCR to add ZFB-wp-his3-ura3 homology:
1µL of plasmid as template
pSB4K5_F+ZFBhl and pSB4K5_R+URA3hl primers
58˚C anneal, 3 min elongation, 25 cycles
Results: the product was really really really faint--primer dimers were by far the bulk of the product.  We realized this is probably because one of the primers includes a Not1 restriction site, which is palindromic, 8 bases long, and all Cs and Gs.
pSB4K5 PCR to add homology 7/29/11
In order to counteract the primer dimers, we will try a gradient PCR with a range of higher annealing temperatures:
same protocol as before except the annealing temp went from 58-65 with 8 samples (1=58, 8=65)
Results: 3:11P did have visible product bands, but the primer dimers are still stronger
pSB4K5 gradient PCR 7/29/11
Team TolC
Sequencing of Kan+ZFB+wp and Zeocin insertions
Sequencing results were successful and solidified presence of both insertions
Zif268 Plasmid Insertion into ECNR2+Kan-ZFB-wp+zeocin
The plates had growth on them!!!
So we chose 5 colonies and ran PCR on them to gel them out
The gel shows success for colonies 2,3,4, and 5 that we chose!!!
Zif268 plasmid insertion check 7/29/11
READY FOR SELECTION!!!
Team ZF
Transformation plating results
We got no growth on any of the plated transformed bacteria for either the ElectroComp or ChemComp bacteria.  To check whether the isothermal assembly itself was the problem, we ran a junction PCR using our previous protocol on the isothermal assembly mix, along with a positive control (Myc981):
No bands visible for the isothermal assembly
As observed, there were no bands in the isothermal mix lanes while the positive control worked.  Thus, we can conclude that our isothermal assembly did not work, and we will need to repeat it again (currently waiting for more isothermal mix to arrive from Wyss).
Amplification and purification of practice plate oligos
In preparation for our swap reactions (switching the oligo F1s into our expression plasmid), we mapped out the following two experiments that we will need to perform:
Testing the selection system by preparing our expression plasmids in parallel (i.e. creating 18 plasmids in separate reactions), and then combining them all at the end and transforming them into the selection strains.  This will allow us to cut and ligate the plasmids without much concern, but test whether we will be able to pick out the binder amongst the rest of the non-binders.
Testing the cutting/ligation by combining all of the oligos first, and then carrying out the construction of the different expression plasmids all in the same tube.
Note that the results for both will ultimately yield DNA libraries if they work (except that the first experiment is much more foolproof in terms of actually yielding a library than the second experiment).
To carry out the first experiment, the following steps need to be undertaken:
Perform PCRs on each of the practice plate oligos such that we end up with a relatively high concentration of DNA after PCR and PCR purification (over 40 ng/ul).
Cut both plasmid backbone and oligos with BsaI type II restriction enzyme.
PCR purify restriction products to get rid of enzyme and junk DNA.
Run phosphatase reaction to prevent autoligation.
Mix and ligate to get full plasmid.
Some tips:
Each ligation reaction should only have 10-15 ul volume.
You should have 100 ng of backbone per reaction
Since we are planning 18 reactions, we will need 1800 ng of backbone, which is several minipreps worth
The backbone:oligo ratio should be appoximately 1:4
In our case, the backbone is 2.2 kb and the insert is 131 bp, so for 100 ng of backbone there should be ~20 ng of insert
Gel extract plasmid to get final library.
Today, we did PCRs and PCR purifications for step one as outlined above.  We tested the concentrations of the results, displayed below:
DNA concentrations
In order to have enough backbone to work with, we seeded some LB/spec with CB bottom bacteria in preparation for 4 minipreps of 3 mL each tomorrow.  This should definitely yield the 1800 ng of backbone we will need for the ligation.
Practice plate primer tag test
We tested the primers to see if their selection of subpools was exclusive enough or not (i.e. to make sure that our control primers did not amplify the CB bottom sequences, etc.).  We ran a PCR crossing primers with DNA with the following results:
Strange results
The first and third lanes are CB primer/CB DNA and control primer/control DNA respectively, which should yield bands.  Lanes two and four were control primer/CB DNA and CB primer/control DNA respectively, which should yield no bands.  The fact that we got bands for these lanes is a little bit worrying, and so we are repeating this PCR again to see if we get the same results when we run the gel tomorrow.
July 30th
Team TolC
Checked the plate with ECNR2 with Zeocin knocked into the place of rpoZ. There are appear to be some 15-20 colonies on the plate. This strain is ready for transformation with the zif268 plasmid
Team ZF
CB bottom miniprep
Today we performed a miniprep on the 4 tubes of CB bottom bacteria with 4 ml of culture in each tube.  6 ml of culture were eventually consolidated into a single spin tube during the miniprep, so this would be the equivalent of 4 minipreps in one tube.  Of note, after lysis (during addition of buffer P2), the tubes turned extremely blue, darker blue than we had ever seen before.  This would seem to indicate that the lysis step occurred very successfully.  However, even after consolidating four minipreps into one tube, we still got terrible concentration results for our samples, displayed below:
DNA concentrations after miniprep
We will thus seed more LB/spec culture with CB bottom bacteria tomorrow, and perform the miniprep on Monday with TF supervision to hopefully troubleshoot some of the problems with low yield.
Also of note is that the bacteria were left on the shaker for approximately 23 hours at the time of miniprep, which is longer than the ideal 12-14 hours.
July 31st
Team ZF
Primer tag test two
On Friday, we repeated the PCR to test for sub-pool specificity of our primer oligo tags for the chip, using the same PCR protocol.  Today, we ran a gel to see if the results were any different, but alas, they were the same.  It seems that the primers for CB bottom also amplify out the control sequences, which means that our primer tags are not as specific as we thought they were.  The gel can be seen below, with the lanes arranged as follows: CB primer + CB DNA, control primer + CB DNA, control primer + control DNA, and control primer + CB DNA.  We would ideally expect bands in lanes 1 and 3 and no bands in lanes 2 and 4.  However, this was not the case:
PCR repeat results
We might try doing touchdown PCR or raising the annealing temperature to get more specificity in the future.  It may also be worth ordering oligos to represent the other 5 subpools of targets and cross test the primers with each DNA subpool to make sure that the primers are specific enough to pull out only one subpool.  In the end though, this specificity is not critical to our success, and so it is ok if it doesn't eventually work.
Preparing for another CB bottom miniprep
Today we seeded 4 tubes of culture with 100 ul of CB bottom bacteria at saturation.  Each tube carried 4 ml of culture.  The tubes were put on the shaker at approximately 6:30 pm, so they will be ready for miniprep tomorrow morning after the lab meeting.
Retrieved from " http://2011.igem.org/Team:Harvard/Template:NotebookDataJuly4 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
