"
Page
Discussion
View source
History
teams
Log in
 
Team:Harvard/Template:NotebookData2
From 2011.igem.org
Contents
1 June 14
1.1 Wet Lab
1.2 Today's Gel Images
2 June 14 - Bioinformatics
3 June 15th - Wet Lab
4 June 15th - Bioinformatics
5 June 16th - Wet Lab
6 June 16 - Bioinformatics
7 June 17th - Wet Lab
8 June 17 - Bioinformatics
8.1 Goals
8.1.1 Options for Target DNA Sequences / ZF Helices
8.1.2 References
9 June 20th - Wet Lab
10 June 20th - Bioinformatics
10.1 Goals for the week
10.2 Today
10.3 Probability data
June 14
Wet Lab
Made four LB-based media solutions, and later created glycerol stocks from these and placed in -80⁰C freezer
Selection strain (ΔHis3ΔPyrFΔrpoZ) in 3 mL LB and 3µL of 1000x Tet solution stock
Selection strain (ΔHis3ΔPyrFΔrpoZ) in 3 mL of LB only solution stock (control)
Kan cassette (pZE22G) in 3 mL of LB solution stock and 3 µL of kanamycin solution stock
Lambda Red (pKD42) in 3 mL of LB and 3 µL Amp solution stock
For all of these stocks, we tried to grow all to mid-log and then place them in 1,200 µL of culture and 300 µL of 80 % glycerol solution (This is the correct protocol for creating a glycerol stock; refer to June 9th)
We were only able to get the kan cassette to mid-log and created glycerol stock of the kan cassette
Observations included contamination of a pKD46 liquid culture, and we are leaving Lambda Red and both solutions with the selection strain for overnight growth
Ran 1% gel (150V) with the rest of the OZ123 and OZ052 overhang PCR samples
Used better ladder today, less the 1 kb ladder
Bands followed the same pattern as the gel run on 6/13/11
Used gel extraction to obtain the correct OZ123 and OZ052 PCR product from the gel
Used Qiagen quick gel extraction kit
OZ052 (from undiluted lane): 7.0 ng/µL, 260/280=2.42 (Note: this sample had a strange yellow substance in the column, may have been contaminated)
OZ052 (from undiluted lane): 12.0 ng/µL, 260/280=2.02
OZ052 (from 1:10 dilution): 10.8 ng/µL, 260/280=2.04
OZ052 (from 1:10 dilution): 15.2 ng/µL, 260/280=1.82
OZ052 (from 1:100 dilution): 20.6 ng/µL, 260/280=2.17
OZ123 (from undiluted lane): 6.3 ng/µL, 260/280=2.17
PCR the backbone fragment of the plasmid using Wolfe_R and Wolfe_L primers and a lower annealing temperature than before due to the lower melting point of Wolfe_L
Reagents:
22µL Invitrogen Platinum PCR supermix
1µL template from a 1 in 100 dilution (1 ng)
1µL Wolfe_F and 1µL Wolfe_R
94°C for 30 s
94°C for 30 s
53°C for 30 s
70°C for 5 min
Previous three steps repeat 30 times
70°C for 5 min
4°C forever
Performed a restriction enzyme digestion on the selection construct plasmid using EcoRI to test for presence/absence of inserted selection construct
1µL EcoR1
1µL buffer 4
2µL backbone plasmid
6µL ddH2O
Incubate at 37 degrees for 90 min
There is only one EcoR1 site (GAATTC) in the plasmid, so we should see 1 band at about 5kb
Ran a gel (1%, 170 V) on the backbone fragment (Wolfe primers) PCR product and restriction digestion result
Observations: the EcoR1 digest produced the expected band of around 5kb. The backbone did produce a 5kb band but also had a secondary smaller product, perhaps due to one of the primers annealing to a sequence that is a close match to its target.
Began gradient PCR on the selection strain backbone with Wolfe_R and Wolfe_F primers because of the large difference in melting temperatures between the two
Set the annealing temperatures within the PCR to go from 50-57 C and ran with 5 minute extension phases at 70 C
Ran overnight
Today's Gel Images
Ultramer Overhang 6/14/11
Digestion and PCR Backbone on Selection Construct Plasmid 6/14/11
June 14 - Bioinformatics
We finished writing the generate function, and now have a working sequence generator. We also began more in-depth research into the 2011 work by Persikov which deals with how the zinc finger binds to DNA. He predicts several relations which we should be able to test.
Proposed novel interactions between helical zinc finger residues and base pairs of the target DNA sequence (based on Persikov 2011 )
Persikov sent us his SVM code (used to calculate the probability of a sequence binding to given DNA), so we also worked on adapting this to use when narrowing our sequences to those most likely to work.
There are four canonical amino acid-base interactions involved in zinc finger interactions.
Amino acids in positions -1, 3, and 6 on the helix are known to interact with the 3 bases of the triplet. In addition, the amino acid in position 2 interacts with the upstream base of the complementary strand (Klug 2010).
In addition, Persikov (2011) has proposed novel interactions between these four amino acids based on his analysis of zinc finger binding data up to 2005.
Persikov uses the information between these four amino acid-base interactions in his SVM, to determine whether a finger would be a good binder to a particular DNA sequence.
In order to use his program, we need to convert the helices that are created by the generator into a format the SVM accepts.
The SVM only considers the four canonical interactions. It assigns a numerical value to each possible amino acid-base combination. The program that converts our data into a format the SVM accepts creates a string with these numerical values based on Persikov's key. (See this page for more details on this program.)
Brandon learned basic Python today. Justin created a JavaScript program that recognizes potential binding sites from a given sequence.
June 15th - Wet Lab
Gradient PCR: 5µL of each PCR product were run on a 1% gel. No bands appeared: the PCR appears to not have worked.
Selection construct: bacteria containing the selection construct (plasmid containing ZF, omega subunit, ZFB, His3, URA3, etc.) was made into a glycerol stock (see 6/14 and 6/9), miniprepped, and used for PCR:
Miniprep: used Qiagen kit
82.5ng/µL, 260/280=1.99
91.1ng/µL, 260/280=2.01
77.9ng/µL, 260/280=1.98
PCR:
PCR used to amplify section of plasmid containing zinc finger binding site, weak promoter, His3, and URA3 (with homology to join it to kan cassette)
Reagents
zinc finger binding site and weak promoter, selection construct plasmid:
1µL ZFB-wp-f (5µM)  (made by 1:20 dilution of 100µM stock)
1µL ZFB-wp-hisura-r (5µM)  (made by 1:20 dilution of 100µM stock)
1µL selection construct (1:100 dilution of overnight culture)
22.5µL of invitrogen's Platinum PCR SuperMix
PCR used to amplify the kan cassette
Reagents
KAN cassette, pZE22g plasmid:
1µL hisura-kan-f (5µM)  (made by 1:20 dilution of 100µM stock)
1µL kan-r (5µM)  (made by 1:20 dilution of 100µM stock)
1µL pZE22g (1:100 dilution of glycerol stock culture)
22.5µL of invitrogen's Platinum PCR SuperMix
Parameters:
1) 94°C for 2 min (denature template, activate enzyme)
2) 94°C for 30 sec (denature)
3) 53°C for 30 sec (anneal)
4) 72°C for 2 min (extend)
5) Repeat 2-4 for 25 cycles total
6) 72⁰C for 5 min
7) 4°C forever
PCR Purification:
Used the Qiagen PCR purification kit and instructions in order to purify the Kan cassette and selections construct PCR products
Nanodrop the purification results and observed 3 ng/µL for Kan cassette and 29.8 ng/µL for ZFB-wp-His3: purification did not work well, especially for the kan
Selection strain (ΔHis3ΔPyrFΔrpoZ):
saturated overnight culture was inoculated again: 3mL LB, 3µL tet, 30µL of overnight culture, at 37C until mid-log
glycerol stock
For transformation tomorrow we grew up pKD42 in 3 mL of LB, 1.5 µL of ampicillin(2000x) and one colony at 30 C
Also grew up more of the selection strain so it will be ready for electroporation transformation
Gel
Ran gel with Kan cassette and selection construct (Binding site, His3, and URA3)
Observations successful and image below
Used 1 kb plus ladder
Kan and ZFB-wp-his constructs 6/15/11
PCR Overlap
since the purification was not very successful, we used 3µL saved from the original PCR product
Procedure
25µL of 2x Phusion Master Mix
1 µL of ZFB-wp-HisURA-R (100µM)
1µL of HisURA-Kan-F (100µM)
21 µL of water
1 µL Kan template and 1 µL of ZFB-wp-His3-URA3
4 tubes
Both undiluted
Both 1:10 dilution
Both 1:100 dilution
Both 1:1000 dilution
Protocol
98 C for 30 s
98 C for 10 s
53 C for 30 s
72 C for 3 min
Repeat steps 2-4 for 24 more cycles
72 C for 5 min
4 C 4EVA!!!
June 15th - Bioinformatics
We continued research into Persikov's and others' work on binding.
We worked on using the OPEN data to test Persikov's binding-predicting program: SVM
Justin continued work on a sequence-finding program, the most up to date version can be found in the Dropbox under code/zfsitefinder.html.
Justin and Will found 10 candidate sequences across 4 diseases that hopefully should encompass a good amount of diversity in terms of expanding the ZF library.  These sequences can be found in the table below, with more details here .
Disease
Target Range
Binding Site Location
Bottom Finger
Top Finger
Bottom AA (F3 to F1)
Top AA (F3 to F1)
Colorblindness
chrX:153,403,001-153,407,000
370
GTATTTGTT
GGGCCTGCT
N/A
N/A
Colorblindness
chrX:153,403,001-153,407,000
3627
GCTGGCTGG
GCGGTAATG
EGSGLKR.EAHHLSR.#######
RRDDLTR.QRSSLVR.#######
Cystic Fibrosis
chr7:117,074,084-117,089,556
14767
GCAGGTGAT
AAAGAGCCC
QNGTLGR.EAHHLSR.#######
N/A
Familial Hypercholesterolemia
chr19:11,175,000-11,195,000
14001
GGCTGAGAC
GGAGTCCTG
ESGHLKR.QREHLTT.#######
QTTHLSR.DHSSLKR.#######
Tay-Sachs
chr15:72,674,944-72,688,031
5888
GTCTGGTCA
TCAAACTCC
DRSSLRR.RREHLTI.#######
N/A
Pancreatic Cancer
chr7:117,074,084-117,089,556
1739
GATCAAGCT
GTTTCAGTG
N/A
N/A
We collected 15 alternative zinc finger backbones (different from zif268 backbone) and their corresponding base sequences. Many of these were from Persikov 2011 and all binding sequences were confirmed on the Protein Data Bank website . The zinc finger PDB ID's and related links are:
PDB ID
Binding Sequence
Link
1F2I
ATGGGCGCGCCCAT
[1]
1G2D
GACGCTATAAAAGGAG
[2]
1G2F
TCCTTTTATAGCGTCC
[3]
1MEY
ATGAGGCAGAACT
[4]
1TF6
ACGGGCCTGGTTAGTACCTGGATGGGAGACC
[5]
1UBD
AGGGTCTCCATTTTGAAGCG
[6]
1TF6
ACGGGCCTGGTTAGTACCTGGATGGGAGACC
[7]
1YUI
GCCGAGAGTAC
[8]
2DRP
CTAATAAGGATAACGTCCG
[9]
2GLI
TTTCGTCTTGGGTGGTCCACG
[10]
2I13
CAGATGTAGGGAAAAGCCCGGG
[11]
2KMK
CATAAATCACTGCCTA
[12]
2PRT
CGCGGGGGCGTCTG
[13]
2WBS
GAGGCGC
[14]
2WBU
GAGGCGTGGC
[15]
June 16th - Wet Lab
There was totally a crazy bee hive outside today!!
Glycerol Stock pKD42
Grew up pKD42 in 30 C and once reached mid-log created glycerol stock and placed in -80 refrigerator
Overlap PCR gel
Ran gel to test if overlap PCR that ran through the night worked, and it did not
Used PCR product without purification which gives good explanation for why it didn't work
PCR: Since the PCR done the previous day (6/15) we made a back up PCR using phusion mastermix (Finnzyme)
PCR used to amplify section of plasmid containing zinc finger binding site, weak promoter, His3, and URA3 (with homology to join it to kan cassette)
Reagents
zinc finger binding site and weak promoter, selection construct plasmid:
1µL ZFB-wp-f (100µM)  (taken directly from the primer tube)
1µL ZFB-wp-hisura-r (100µM)  (taken directly from the primer tube)
2µL selection construct (1:100 dilution of overnight culture)
25µL Phusion High-Fidelity PCR Master Mix
21µL distilled water (for total volume of 50µL)
PCR used to amplify the kan cassette
Reagents
KAN cassette, pZE22g plasmid:
1µL hisura-kan-f (100µM)  (taken directly from the primer tube)
1µL kan-r (100µM)  (taken directly from the primer tube)
2µL pZE22g (1:100 dilution of glycerol stock culture)
25µL of Phusion High-Fidelity PCR Master Mix
21µL distilled water (for total volume of 50µL)
Parameters:
1) 94°C for 2 min (denature template, activate enzyme)
2) 94°C for 30 sec (denature)
3) 53°C for 30 sec (anneal)
4) 72°C for 2 min (extend)
5) Repeat 2-4 for 25 cycles total
6) 72⁰C for 5 min
7) 4°C forever
Ran PCR product on a gel: bands of the correct size were observed, though the kan band was much fainter than the ZFB-wp-his3
Kan cassette and ZFB-wp-his3 constructs 6/16/11
Repeat PCR (to get a higher concentration of the Kan cassette and ZFB-wp-his3 constructs)
same as the above backup PCR (since it was successful), but to a 4x total volume of 200µL, compared to 50µL
PCR products were run on a gel: the correct bands were observed--see image below ("second gel")
PCR product purification: followed Qiagen kit instructions. Strangely, the conc. was 64.6 ng/µL for Kan, purity 2.09 (260/280) and for ZFB-wp-his3, the conc. was 23.7ng/µL and the purity was 1.92 (260/280).
Kan cassette and ZFB-wp-his3 constructs second gel 6/16/11
Overlap PCR: used the kan cassette (64.6 ng/µL) and ZFB-wp-his3-ura3 (23.7 ng/µL) purified above
1 µL of kan and 1 µL of ZFB-wp-his3-ura in each tube according to the following conditions:
two tubes: undiluted
two tubes: both diluted 1 in 10
two tubes: both diluted 1 in 100
12.5µL Phusion master mix
8 µL ddH2O
primers: 1.25 µL ZFB-wp-hisura_r (10 µM) and 1.25 µL hisura-kan_f (10 µM)
we tried two different reaction types: one added the primers as usual before starting the PCR reaction, the other added the primers after 10 PCR cycles (allowing the polymerase to first use the overlapping kan and ZFB to elongate, and then the primers)
Parameters for PCR starting with primers:
(PCR machine 5, program name  EXT3KB in IGEM folder)
1) 98°C for 1 min (denature template, activate enzyme)
2) 98°C for 15 sec (denature)
3) 65°C for 15 sec (anneal)
4) 72°C for 2 min (extend)
5) Repeat 2-4 for 30 cycles total
6) 72⁰C for 5 min
7) 4°C forever
Parameters for PCR starting without primers:
1) 98C for 1 min
2) 98C for 15 sec
3) 65C for 15 sec
4) 72C for 1 min
5) back to step 2 for 10 cycles (PCR paused after 10 and primers added)
6) 98C for 15 sec
7) 65C for 15 sec
8) 72C for 2 min
9) back to step 6 for 20 cycles
10) 72C for 5 min
11) 4C forever
Transformation
Used the selection strain (ΔHis3ΔPyrFΔrpoZ) cells at mid-log and attempted to place lambda red (pKD42) plasmid into the cell
Procedure
Keep on ice through out whole procedure before use of the electroporation machine
Spin 1.5 mL of mid-log cells at 4 C for 1 minute at 18000 rcf (we created two tubes through the following steps)
Discard supernatant and resuspend with 1 mL of cold water
Spin again and repeat for a second water wash
With each wash, try to get as much supernatant out as possible(even use pipette) because don't want salts to interfere with the electrical pulse
Resuspend pellet with 50 µL of cold water
Add 1 ng of pKD42 to one of the tubes and 45 ng of pKD 42 to the other
Take all of the liquid in each tube and place in two separate cuvettes for electroporation
Make sure the electroporation machine is on the right setting (for the cuvettes we used today it was "Ec2")
Wipe off all water on the side of the cuvette
Have 1 mL of LB in hand and after pulsing, immediately put LB in cuvette
Transfer to culture tube and place in 30 C for 2 hours
Make 4 LB/amp plates and spread E. coli using glass beads:
Plate 1: 10 µL of 1 ng culture
Plate 2: take 700 µL of 1 ng culture, spin down and remove supernatant, resuspend in about 30 µL of LB and plate
Plate 3: 10 µL of 45 ng culture
Plate 4: take 700 µL of 45 ng culture, spin down and remove supernatant, resuspend in about 30 µL of LB and plate
Grow overnight at 30 C
PCR to confirm knockouts of selection strain
this PCR was to confirm that the ΔHis3ΔPyrFΔrpoZ was indeed a knockout for the His3, PyrF, and rpoZ genes
each primer set was used for two conditions: wild-type (we used a pKD42 culture) and knockout (ΔHis3ΔPyrFΔrpoZ culture, left over from transformation)
1 µL of either wt or ko template, diluted 1:20
12.5 µL Phusion master mix
1.25 µL of each 10µM primer:
test for His3:
1)His3_F, His3_R
2)His3_F, His3_internalR
test for PyrF:
3)PyrF_F, PyrF_R
4)PyrF_F, PyrF_internalR
test for rpoZ:
5)rpoZ_F, rpoZ_R
6)rpoZ_F, rpoZ_internalR
test for Zeocin (there are two primer sets because we don't know what orientation the Zeocin gene is in)
7)Zeocin_R, rpoZ_F
8)Zeocin_R, rpoZ_R
ddH2O up to 25 µL
Parameters:
98 C for 5 min
98 C for 10 sec
65 C for 25 sec
72 C for 45 sec
cycle 30 times
72 C for 5 min
4 C forever
Results: ran PCR products out on 1% gel (see below). There were some nonspecific bands, but the PyrF and rpoZ genes do appear to be knocked out in the selection strain. His3, however, looks like it's still present--we'll test again to confirm.
Gel Confirmation of Knockouts in Selection Strain 6/16/11
June 16 - Bioinformatics
Research Targets
Clinically relevant targets
Existing ZFs that bind under-represented triplets
Updating our programs
Many of our current programs currently look at overall data or data based on specific DNA triplets (for example: 'GAT' or 'AAA'). However, in order to more easily understand some of the patterns that occur in the datasets, we want to examine broader subsets of data. For example, do different patterns appear when looking at fingers that bind to 'GNN' triplets versus 'NGN' triplets (where 'N' represents any of the 4 bases)?
We added the capability for our programs to accept inputs with the variable 'N' by using regular expressions.
We can now create lists of the zinc fingers that bind to any triplet, and create interaction matrices and frequency tables for any triplet input.
June 17th - Wet Lab
Update on selection strain knockout status: We are trying to reach Addgene to check how His3 was knocked out---instead of deleting the gene, they may have simply introduced an early stop codon. If that's the case, our gel would have the correct bands because the primers we designed can only show whether a deletion or insertion was in that locus.
Transformation results: successful!!
The only plate with colonies was the one plated with 700 µL of cells transformed with 45 ng of pKD42
Chose a colony to grow in 3mL LB, 1.5µL amp, 30C; make glycerol stock with mid-log cells
Plate with colonies at 4C
Miniprep of pZE22G: (to have the plasmid containing the kan cassette on hand)
used 2 tubes of 1.5mL overnight culture, followed Qiagen kit instructions
38.0 ng/µL, 260/280=1.99
27.8ng/µL, 260/280=2.02
Overlap PCR gel and extraction: 1%, 150V
Results: adding the primers in after 10 cycles was much more successful than adding them at the beginning, and all three dilutions showed the expected product band (about 2.5kb). The rest of the 1:10 dilution will be run on a gel and extracted.
11.2ng/µL, 260/280=2.10
Successful Overlap of Kan Cassette and ZFB Gel 6/17/11
June 17 - Bioinformatics
Goals
Make BB Database in program-readable format ✓
Edit out BB with incomplete helices ✓
GNN, TNN, CNN, ANN frequencies
Targets (5-10; 8) x Backbones (???) x Helices (≥500)=55,000
Backbones: similar, but not too similar to zif268; more than 1-2 aa changes, but <10
Helices fixed based on our program-- eventually saturates and levels out
Graph: # of var (# of tries by the computer) vs. % space covered
Proposed interactions between helical zinc finger residues and base pairs of the target DNA sequence (based on Persikov 2011 )
Options for Target DNA Sequences / ZF Helices
F3(known) / F2(known) / F1(novel)
F3(known) / F2(SNP in b1 position) / F1(known)
F3(unknown) / F2(unknown) / F2(unknown)
Excluded Rare Codons (for E. coli)
CodonUsage OpenWetWareCodonUsage NIHRareCodonCalculator
:
CTA
ATA
CCC
CGA
CGG
AGA
AGG
GGA
GGG
References
Persikov2011 pmid=21572177
CodonUsage http://www.sci.sdsu.edu/~smaloy/MicrobialGenetics/topics/in-vitro-genetics/codon-usage.html
OpenWetWareCodonUsage http://openwetware.org/wiki/Escherichia_coli/Codon_usage
NIHRareCodonCalculator http://nihserver.mbi.ucla.edu/RACC/
June 20th - Wet Lab
Grew up colony of the selection strain with pKD46 in an attempt to reach mid-log and create glycerol stock
Unable to reach mid-log, so going to leave growing over night and use saturated culture tomorrow
Determined primers in order to piece together the omega subunit and ZFP genes into the pZE21G plasmid (spec cassette)
Ran PCR on His3 locus and sent to GENEWIZ to be sequenced
used the same procedure as the earler WT/KO PCR, but with 1µL undiluted template and only His_F and His_R primers
ran 3 reactions and sent in three primers (His_F, His_R, His_internalR)
June 20th - Bioinformatics
Goals for the week
Finish designing the chip, by Wednesday hopefully
Need chip order out, takes 4 weeks
Need all sequences by Friday!!!
FIRST PRIORITY: If we can get Persikov to work, good!
Step one: get results he’s published, get the web app to "work" with his data, then OPEN data, and finally our data
Brainstorming session (tomorrow?) to decide how many targets/sequences
Determine the importance of the first/second/third nucleotide positions
Look at NGN, NTN, NAN, NCN (Not just GNN, etc.)
Pick a particular GNN, plot vs. TNN- is there a pronounced difference in position 1, or -1?
Today
Testing Persikov's Data for validation
Persikov v. himself  ✓
Persikov v. OPEN
Persikov v. our sequences
Probability data
The following are graphs of the probability of finding each amino acid at each position on the alpha helix.
Probability data for the 783 fingers that bind to GNN triplets. Note the high probability of leucine at position 4 and arginine at position 6.
Probability data for the 128 fingers that bind to TNN triplets. Note the high probability of leucine at position 4.
Probability data for the 16 fingers that bind to CNN triplets. There may not be enough data to consider this information statistically significant
Probability data for the 29 fingers that bind to ANN triplets. There may not be enough data to consider this information statistically significant
Probability data for the 298 fingers that bind to NGN triplets. The position 4 leucine motif remains. There is also a high probability (> 0.5) of a histidine at position 3 and an arginine at position 6.
Probability data for the 177 fingers that bind to NTN triplets. The position 4 leucine motif remains.
Probability data for the 244 fingers that bind to NCN triplets. The position 4 leucine motif remains. There is also a very high probability of an arginine at position 6.
Probability data for the 248 fingers that bind to NAN triplets. The position 4 leucine motif remains. There is also a very high probability (> 0.75) of an asparagine at position 3 and an arginine at position 6.
Probability data for the 234 fingers that bind to NNG triplets. The position 4 leucine motif remains. There is also a very high probability (> 0.75) of an asparagine at position 1 and a high probability (> 0.5) of an aspartic acid at position 2 and an arginine at position 6.
Probability data for the 247 fingers that bind to NNT triplets. The position 4 leucine motif remains. There is also a high (> 0.5) probability of an arginine at position 6.
Probability data for the 262 fingers that bind to NNC triplets. The position 4 leucine motif remains. There is also a very high (> 0.75) probability of an arginine at position 6.
Probability data for the 218 fingers that bind to NNA triplets. The position 4 leucine motif remains. There is also a very high (> 0.75) probability of a glutamine at position -1 and an arginine at position 6.
Retrieved from " http://2011.igem.org/Team:Harvard/Template:NotebookData2 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
