<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Harvard_Template_NotebookDataJuly3"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Harvard/Template:NotebookDataJuly3">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Harvard/Template:NotebookDataJuly3&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Harvard/Template:NotebookDataJuly3&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Harvard/Template:NotebookDataJuly3&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Harvard/Template:NotebookDataJuly3" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Harvard/Template:NotebookDataJuly3</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="715" style="display:none"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#July_15th"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">July 15th</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Team_Wolfe"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Team Wolfe</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Team_ZF"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Team ZF</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#PCR_of_electro_comp_expression_plasmids"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">PCR of electro comp expression plasmids</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Checking_the_expression_plasmid_plates"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Checking the expression plasmid plates</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#PCR_to_check_for_omega.2Bultramers"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">PCR to check for omega+ultramers</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-7"><A href="#July_16th"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">July 16th</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Team_ZF_2"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Team ZF</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Colony_minipreps"><SPAN class="tocnumber">2.1.1</SPAN><SPAN class="toctext">Colony minipreps</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#PCR_on_minipreps_to_check_cross_junction"><SPAN class="tocnumber">2.1.2</SPAN><SPAN class="toctext">PCR on minipreps to check cross junction</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Replating_final_samples_for_78.2C_79.2C_82"><SPAN class="tocnumber">2.1.3</SPAN><SPAN class="toctext">Replating final samples for 78, 79, 82</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#July_17th"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">July 17th</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Team_ZF_3"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Team ZF</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Replating_final_samples.2C_take_2"><SPAN class="tocnumber">3.1.1</SPAN><SPAN class="toctext">Replating final samples, take 2</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Gel_analysis_of_miniprep_junction_PCR"><SPAN class="tocnumber">3.1.2</SPAN><SPAN class="toctext">Gel analysis of miniprep junction PCR</SPAN></A></LI><LI class="toclevel-1 tocsection-16"><A href="#July_18th"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">July 18th</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Team_ZF_4"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Team ZF</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#Sending_expression_plasmids_out_for_sequencing"><SPAN class="tocnumber">4.1.1</SPAN><SPAN class="toctext">Sending expression plasmids out for sequencing</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Making_more_cultures_and_plates_of_our_expression_plasmids"><SPAN class="tocnumber">4.1.2</SPAN><SPAN class="toctext">Making more cultures and plates of our expression plasmids</SPAN></A></LI><LI class="toclevel-3 tocsection-20"><A href="#Creating_the_96_well_test_plate"><SPAN class="tocnumber">4.1.3</SPAN><SPAN class="toctext">Creating the 96 well test plate</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#Team_Wolfe_2"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Team Wolfe</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#Team_TolC"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Team TolC</SPAN></A></LI><LI class="toclevel-3 tocsection-23"><A href="#Primers"><SPAN class="tocnumber">4.3.1</SPAN><SPAN class="toctext">Primers</SPAN></A></LI><LI class="toclevel-3 tocsection-24"><A href="#Kan_Plates"><SPAN class="tocnumber">4.3.2</SPAN><SPAN class="toctext">Kan Plates</SPAN></A></LI><LI class="toclevel-3 tocsection-25"><A href="#Kan-ZFB-wp_insertion"><SPAN class="tocnumber">4.3.3</SPAN><SPAN class="toctext">Kan-ZFB-wp insertion</SPAN></A></LI><LI class="toclevel-1 tocsection-26"><A href="#July_19th"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">July 19th</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#Team_ZF_5"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Team ZF</SPAN></A></LI><LI class="toclevel-3 tocsection-28"><A href="#Checking_on_plates_78.2C_79.2C_82"><SPAN class="tocnumber">5.1.1</SPAN><SPAN class="toctext">Checking on plates 78, 79, 82</SPAN></A></LI><LI class="toclevel-3 tocsection-29"><A href="#Minipreps_of_colonies_and_controls"><SPAN class="tocnumber">5.1.2</SPAN><SPAN class="toctext">Minipreps of colonies and controls</SPAN></A></LI><LI class="toclevel-3 tocsection-30"><A href="#PCR_of_miniprep_products"><SPAN class="tocnumber">5.1.3</SPAN><SPAN class="toctext">PCR of miniprep products</SPAN></A></LI><LI class="toclevel-3 tocsection-31"><A href="#Starting_a_new_plate_for_sample_85"><SPAN class="tocnumber">5.1.4</SPAN><SPAN class="toctext">Starting a new plate for sample 85</SPAN></A></LI><LI class="toclevel-3 tocsection-32"><A href="#Designing_ultramers_for_Zif268.2C_OZ052.2C_and_OZ123"><SPAN class="tocnumber">5.1.5</SPAN><SPAN class="toctext">Designing ultramers for Zif268, OZ052, and OZ123</SPAN></A></LI><LI class="toclevel-2 tocsection-33"><A href="#Team_TolC_2"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Team TolC</SPAN></A></LI><LI class="toclevel-3 tocsection-34"><A href="#PCR_to_test_insertion_of_KAN-ZFB-wp_into_ECNR2"><SPAN class="tocnumber">5.2.1</SPAN><SPAN class="toctext">PCR to test insertion of KAN-ZFB-wp into ECNR2</SPAN></A></LI><LI class="toclevel-3 tocsection-35"><A href="#Sequencing_of_Kan-ZFB-wp_insertion"><SPAN class="tocnumber">5.2.2</SPAN><SPAN class="toctext">Sequencing of Kan-ZFB-wp insertion</SPAN></A></LI><LI class="toclevel-3 tocsection-36"><A href="#New_MAGE_oligo"><SPAN class="tocnumber">5.2.3</SPAN><SPAN class="toctext">New MAGE oligo</SPAN></A></LI><LI class="toclevel-2 tocsection-37"><A href="#Team_Wolfe_3"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">Team Wolfe</SPAN></A></LI><LI class="toclevel-1 tocsection-38"><A href="#July_20th"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">July 20th</SPAN></A></LI><LI class="toclevel-2 tocsection-39"><A href="#Team_ZF_6"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Team ZF</SPAN></A></LI><LI class="toclevel-3 tocsection-40"><A href="#Checking_on_1.2F100_ul_and_1.2F10000_ul_78.2C_79.2C_and_82_plates"><SPAN class="tocnumber">6.1.1</SPAN><SPAN class="toctext">Checking on 1/100 ul and 1/10000 ul 78, 79, and 82 plates</SPAN></A></LI><LI class="toclevel-3 tocsection-41"><A href="#Check_on_85_culture"><SPAN class="tocnumber">6.1.2</SPAN><SPAN class="toctext">Check on 85 culture</SPAN></A></LI><LI class="toclevel-3 tocsection-42"><A href="#Finalizing_the_practice_plate"><SPAN class="tocnumber">6.1.3</SPAN><SPAN class="toctext">Finalizing the practice plate</SPAN></A></LI><LI class="toclevel-3 tocsection-43"><A href="#Expression_Plasmid_Sequencing_Results"><SPAN class="tocnumber">6.1.4</SPAN><SPAN class="toctext">Expression Plasmid Sequencing Results</SPAN></A></LI><LI class="toclevel-3 tocsection-44"><A href="#Construction_of_OZ052_and_OZ123:_Joining_the_Finger_with_Homology_regions"><SPAN class="tocnumber">6.1.5</SPAN><SPAN class="toctext">Construction of OZ052 and OZ123: Joining the Finger with Homology regions</SPAN></A></LI><LI class="toclevel-2 tocsection-45"><A href="#Team_TolC_3"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Team TolC</SPAN></A></LI><LI class="toclevel-3 tocsection-46"><A href="#rpoZ_MAGE_knockout"><SPAN class="tocnumber">6.2.1</SPAN><SPAN class="toctext">rpoZ MAGE knockout</SPAN></A></LI><LI class="toclevel-3 tocsection-47"><A href="#Glycerol_Stocks"><SPAN class="tocnumber">6.2.2</SPAN><SPAN class="toctext">Glycerol Stocks</SPAN></A></LI><LI class="toclevel-3 tocsection-48"><A href="#convinced_that_the_gradient_PCR_machine_is_out_of_order.2C_ought_to_inform_Andrew"><SPAN class="tocnumber">6.2.3</SPAN><SPAN class="toctext">convinced that the gradient PCR machine is out of order, ought to inform Andrew</SPAN></A></LI><LI class="toclevel-2 tocsection-49"><A href="#Team_Wolfe_4"><SPAN class="tocnumber">6.3</SPAN><SPAN class="toctext">Team Wolfe</SPAN></A></LI><LI class="toclevel-1 tocsection-50"><A href="#July_21st"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">July 21st</SPAN></A></LI><LI class="toclevel-2 tocsection-51"><A href="#Team_ZF_7"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Team ZF</SPAN></A></LI><LI class="toclevel-3 tocsection-52"><A href="#Checking_colonies_78_and_79"><SPAN class="tocnumber">7.1.1</SPAN><SPAN class="toctext">Checking colonies 78 and 79</SPAN></A></LI><LI class="toclevel-3 tocsection-53"><A href="#Sending_out_our_expression_colonies_for_sequencing"><SPAN class="tocnumber">7.1.2</SPAN><SPAN class="toctext">Sending out our expression colonies for sequencing</SPAN></A></LI><LI class="toclevel-3 tocsection-54"><A href="#Gel_Extraction_and_Isothermal_Assembly_of_OZ_fingers.2Bhomology"><SPAN class="tocnumber">7.1.3</SPAN><SPAN class="toctext">Gel Extraction and Isothermal Assembly of OZ fingers+homology</SPAN></A></LI><LI class="toclevel-2 tocsection-55"><A href="#Team_Tolc_4"><SPAN class="tocnumber">7.2</SPAN><SPAN class="toctext">Team Tolc</SPAN></A></LI><LI class="toclevel-2 tocsection-56"><A href="#Team_Wolfe_5"><SPAN class="tocnumber">7.3</SPAN><SPAN class="toctext">Team Wolfe</SPAN></A></LI><LI class="toclevel-1 tocsection-57"><A href="#July_22nd"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">July 22nd</SPAN></A></LI><LI class="toclevel-2 tocsection-58"><A href="#Team_ZF_8"><SPAN class="tocnumber">8.1</SPAN><SPAN class="toctext">Team ZF</SPAN></A></LI><LI class="toclevel-3 tocsection-59"><A href="#Sequencing_Results_of_Expression_Plasmids"><SPAN class="tocnumber">8.1.1</SPAN><SPAN class="toctext">Sequencing Results of Expression Plasmids</SPAN></A></LI><LI class="toclevel-3 tocsection-60"><A href="#Transformation_for_OZ_plasmids"><SPAN class="tocnumber">8.1.2</SPAN><SPAN class="toctext">Transformation for OZ plasmids</SPAN></A></LI><LI class="toclevel-2 tocsection-61"><A href="#Team_Wolfe_6"><SPAN class="tocnumber">8.2</SPAN><SPAN class="toctext">Team Wolfe</SPAN></A></LI><LI class="toclevel-2 tocsection-62"><A href="#Team_TolC_5"><SPAN class="tocnumber">8.3</SPAN><SPAN class="toctext">Team TolC</SPAN></A></LI><LI class="toclevel-3 tocsection-63"><A href="#Gel_for_Jun.27s_MAGE_rpoZ.2C_rowD"><SPAN class="tocnumber">8.3.1</SPAN><SPAN class="toctext">Gel for Jun's MAGE rpoZ, rowD</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="July_15th">July 15th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_Wolfe">Team Wolfe</SPAN></H3><P><B>Thio kan-ZFB-wp-hisura optimizing PCR results:</B></P><UL><LI>The gel stab PCR produced a product of the right size, but the 2kb side product was still brighter. The 2-PCR protocol also still had substantial side products but at least they did not completely overshadow the real product. We will accordingly go ahead and do a gel extraction on the 2-PCR samples so that we can try lambda red with them next week.
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Different attempts and thio kan-ZFB-hisura PCR 7/15/11</DIV></DIV></DIV><P><B>pSR01 growth phenotype results:</B></P><UL><LI>Unfortunately the computer hooked up to the plate reader spontaneously restarted during the night to download updates, so we lost all the data. Just by looking at the plate, it still seems like IPTG is inhibiting growth. The ∆HisB∆pyrF∆rpoZ controls also did not grow, probably because the plate the colonies were taken from was too old, so we will restreak from the glycerol stock and set up the plate reader again this weekend.
</LI></UL><H3><SPAN class="mw-headline" id="Team_ZF">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="PCR_of_electro_comp_expression_plasmids">PCR of electro comp expression plasmids</SPAN></H4><P>We performed a mini-prep of the cultures that we grew overnight of the electrocompetent cells. We then performed a PCR on the product of the mini-prep, checking for the 1.4 kb cross junction to confirm the presence of our omega+ultramers+spec plasmid. We used the same recipe as the PCR we performed <A href="#PCR_of_expression_plasmid_cross-junction">on Tuesday</A>.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Description of gel below.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>No bands appeared for PCR peformed on a 1 to 10 dilution of the cultures. Miniprep product from colonies 78, 79, and 82 had the expected band at 1.4 kb, while 80 and 81 had a band that was slightly smaller than the the expected product size. We saw a similar band in our <A href="#PCR_of_product_from_isothermal_assembly_of_expression_plasmid">PCR of the isothermal assembly reaction product</A>. We are unsure exactly what is causing this band. If either the omega subunit or ultramers were missing from the plasmid, a similar sized product would result since both are ~350 bp. We are curious if a plasmid can still be formed if one of these components were missing. 
</P><P>We checked the concentrations of our miniprep product to see if that would explain our problem. 
</P><PRE>ID     Conc. (ng/ul)     260/280     1.4 kb band?
78      16.12             2.10            Y
79      41.64             1.64            Y
80       5.84             2.01            N
81      61.24             1.62            N
82      20.25             1.72            Y
</PRE><P>We can't tell whether the concentration of the miniprep product contributed to the strange band we saw. 81 had the highest concentration but still didn't work. We decided to try plating 4 more colonies from 80 and 81, to see if any of them have the appropriate band. 
</P><P>We also decided to make final plates of cultures 78, 79, and 82, using 2 ul of culture and 50 ul of LB+spec. They were plated on new spec plates. We will incubate them at 37* overnight.
</P><H4><SPAN class="mw-headline" id="Checking_the_expression_plasmid_plates">Checking the expression plasmid plates</SPAN></H4><P>All the plates had colonies on them, although there were still areas of unspecific growth, presumably because of uneven dilution of the spec into the LB plates.
</P><P>We made cultures of all the chem comp cells (77, 83, 84, 85) and the electro comp cell that did not have the correct insertion (80, 81). We picked 4 colonies from each plate, resulting in a total of 24 cultures. We have left them in the 37* shaker overnight, and tomorrow plan on performing a miniprep and PCR to check for the cross-junction. 
</P><H4><SPAN class="mw-headline" id="PCR_to_check_for_omega.2Bultramers">PCR to check for omega+ultramers</SPAN></H4><P>We performed a PCR on the five electro comp miniprep products, to find out what may be missing. We used the omega_F+pLacO (index: 73) as our forward primer, and each of the respective reverse ultramers as our reverse primers (index: 78-82). We had an annealing temperature of 58* and an extension time of 30 seconds, and the expected product size is 646 bp. 
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Team ZF results in lanes 8-12</DIV></DIV></DIV></TD></TR></TBODY></TABLE>
Strangely, we did not get results for sample 78 (FH bottom), even though we got a band there in a PCR earlier today that used the junction primers.  We got positive results for samples 79 and 82, and everything else seems to check out with these two samples (bands present in the correct size with junction primers from both isothermal assembly and miniprep templates).  However, we are still having problems with samples 80 and 81, which may be a problem with the colonies that we picked.  For now, we will assume that the isothermal assembly worked, due to colonies growing on the spec plates for samples 80 and 81, and that we had simply picked bad colonies.  We will find out tomorrow if other colonies from the same plate give us the same results.</DIV><DIV id="716" style="display:none"><H2><SPAN class="mw-headline" id="July_16th">July 16th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_2">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Colony_minipreps">Colony minipreps</SPAN></H4><P>Minipreps were performed on all of the samples except for a few where nothing grew in the LB/spec media.  These exceptions included 85.3, 85.4, 84.2, 81.2, and 81.3.  Possible reasons for no growth for these samples may include: (1) The colony was somehow not a transformed colony and was a cheater (possibly picked from a spot where there was a smaller amount of spec in the unevenly distributed dish), and (2) the colony did not get picked up properly with the pipet tip.
</P><P>The concentrations of the DNA after miniprep were somewhat low, and a full table of these concentrations and DNA purities can be found below:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Plasmid concentrations</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="PCR_on_minipreps_to_check_cross_junction">PCR on minipreps to check cross junction</SPAN></H4><P>PCR was done on the miniprep products to check for the cross junction length, expected band size of ~1.4 kb.  The protocol used was the same as the previous time.  In addition to PCR-ing the miniprep products, a positive control was also included, which was the finished Zif268 + omega + spec backbone plasmid (this was the first successful junction PCR we obtained, and so it should work again).
</P><H4><SPAN class="mw-headline" id="Replating_final_samples_for_78.2C_79.2C_82">Replating final samples for 78, 79, 82</SPAN></H4>
Strangely, nothing grew on the plates that we seeded yesterday with 2ul of bacteria.  We seeded plates again for samples 78, 79, and 82 with 15ul of transformed bacteria instead, and hopefully these should grow by tomorrow.</DIV><DIV id="717" style="display:none"><H2><SPAN class="mw-headline" id="July_17th">July 17th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_3">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Replating_final_samples.2C_take_2">Replating final samples, take 2</SPAN></H4><P>We realized that the plates we seeded yesterday (July 16) were seeded from the transformation bacteria tubes, which has a mix of bacteria that may/may not have taken up the plasmid.  These plates therefore cannot serve as the final stock plates.  We took the E. coli from our initial colony picking and put them in LB + spec to grow to saturation tomorrow, where we will replate again with those bacteria (for which the good PCR results should correlate).
</P><H4><SPAN class="mw-headline" id="Gel_analysis_of_miniprep_junction_PCR">Gel analysis of miniprep junction PCR</SPAN></H4><P>The miniprep junction PCR products from yesterday were ran on an agarose gel, with the results below:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">PCR Results, expected band at ~1.4 kb</DIV></DIV></DIV></TD></TR></TBODY></TABLE>
While all the bands are between 1000 and 1650 kb, there are slight variations in the size for which we are unable to account. We will send these out for sequencing, to try to figure out what is going on.</DIV><DIV id="718" style="display:none"><H2><SPAN class="mw-headline" id="July_18th">July 18th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_4">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Sending_expression_plasmids_out_for_sequencing">Sending expression plasmids out for sequencing</SPAN></H4><P>We decided to send out the expression plasmids that had bands at approximately the correct size out for sequencing. We originally wanted to sequence the plasmids, but our miniprep concentrations were not high enough. Apparently, we should be getting high yields from minipreps, but thus far our yields have been around 50 ng/ul at the highest.
</P><P>We decided to sequence the unpurified PCR products instead. In order to do this we had to re-do the PCRs for Zif268, FH bottom (78), CB top (79), Myc 981 (82). Below are our 1% e-gel results, verifying that we had appropriately sized bands:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gel of cross junction of Zif268, FH bottom, CB top, Myc 981</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>Our final samples sent out for sequencing are:
</P><PRE>Index     Sample ID             Index     Sample ID
 1         77_1                  11        80_3
 2         77_2                  12        80_4
 3         77_3                  13        -
 4         77_4                  14        83_2
 5         84_1                  15        83_3
 6         84_4                  16        83_4
 7         81_1                  17        Zif268
 8         81_4                  18        FH_bottom
 9         80_1                  19        CB_top
 10        80_2                  20        Myc_981
</PRE><H4><SPAN class="mw-headline" id="Making_more_cultures_and_plates_of_our_expression_plasmids">Making more cultures and plates of our expression plasmids</SPAN></H4><P>We picked two more colonies from plate 85 today and made 5 ml cultures from them. We also made 2 fresh cultures from the old 85 cultures, by taking 50 ul of old culture and 5 ml LB+spec. We will leave them overnight in the 37* shaker and perform minipreps and PCRs on them tomorrow. 
</P><P>We made positive control cultures from 81.1, 81.4, 80.1, and 80.2, in order to test our minipreps tomorrow. They reactions have a wide range of concentrations post-miniprep, and we are curious to see if the same will happen after performing a miniprep a second time.
</P><P>Another way to figure out if our expression plasmids contain the appropriate insert is by using a restriction enzyme to cut it and then performing a PCR on the linear DNA. We had previously inserted 2 cutsites on our plasmids, Bsa1 (for insertion of finger 1) and Xba1. The lab already has Xba1, and we ordered Bsa1. Xba1 can be affected by methylation if it is preceded by 'GA' or followed by 'TC' <A href="http://www.neb.com/nebecomm/products/faqproductR0145.asp#874" class="external autonumber" rel="nofollow">[1]</A>; we checked and our sequences should not be affected.
</P><P>
We also re-plated 78, 79, and 82 today, one set of plates with 1 ul of culture, and the other set with 10 ul.
</P><H4><SPAN class="mw-headline" id="Creating_the_96_well_test_plate">Creating the 96 well test plate</SPAN></H4><H3><SPAN class="mw-headline" id="Team_Wolfe_2">Team Wolfe</SPAN></H3><P><B>SNOWMAN!</B></P><P><B>EcNR2 lambda red (Noah's prep):</B></P><UL><LI>Noah remade the kan-ZFB-wp-hisura insert for us with the kan cassette in the opposite orientation from the his3 and ura3 genes. (The kan cassette is under a strong promoter and does not have a transcriptional terminator, so it's possible that RNA polymerase might keep transcribing through the end of kan and into his3.)  He also used it for lambda red in EcNR2, and a couple of plates look like they may have a couple colonies.
</LI><LI>picked 4 colonies, diluted in 20µL H20 and used 1µL as template for 1529620 locus PCR. The rest was grown in LB+kan.
</LI></UL><P><B>Lambda red with Noah's kanZFBhisura:</B></P><UL><LI>We recently discovered that the kan plates we had been using were very old, and that might be why we got that weird crystalline lawn of colony-like things.  We will retry lambda red in the ∆hisB∆pyrF∆rpoZ +pKD46 strain and see if this solves the problem
</LI><LI>1.5mL cells divided in two: half for kan-his-ura (300ng) and half as a negative control (water)
</LI></UL><P><B>Thio insert:</B></P><UL><LI>In case lambda red also does not work, we will try to add phosphothioate bases to the insert, but only to the lagging strand 5' end (so only the Ura3 side). Since our primers will have very different melting temps, we will try 2 different reactions:
</LI><LI>Touchdown PCR:
</LI><LI>template: Noah's kan-ZFB-hisura (163ng/µL) diluted to ~10ng per reaction (4 reactions)
</LI><LI>primers: kan_f+homology, Ura3 R phosphothioate
</LI><LI>2 cycles at each annealing temperature (67, 66...57) for 11 cycles followed by 15 more cycles at 57; 3 min elongation
</LI></UL><LI>Gradient PCR:
</LI><LI>same as above but with 12 reactions, annealing temperatures from 57-67˚C
</LI><LI>Results: most of the gradient reactions produced a faint but clean band at 3kb.  The touch down PCR had a stronger 3kb band but also lots of side products.
</LI><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Thio kan-ZFB-hisura (Noah's) gradient part 1 7/18/11</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Team_TolC">Team TolC</SPAN></H3><H4><SPAN class="mw-headline" id="Primers">Primers</SPAN></H4><UL><LI>Created and ordered primers in order to test the presence of MAGE rpoZ knockout oligo
</LI><LI>Primers made are named 
</LI><LI>rpoZ_MAGE_KO plus 4, rpoZ_MAGE_KO minus 4, which have all 4 of the stop codons
</LI><LI>rpoZ_MAGE_KO plus 2, and rpoZ_MAGE_KO minus 2, which have only the first two stop codons.
</LI></UL><H4><SPAN class="mw-headline" id="Kan_Plates">Kan Plates</SPAN></H4><UL><LI>Performed control experiment on Kan plates by growing up liquid culture from glycerol stock and then spreading on Kan plate in order to determine the speed of growth and intensity of the presence of Kan on the plates
</LI><LI>Will check on these tomorrow.
</LI></UL><H4><SPAN class="mw-headline" id="Kan-ZFB-wp_insertion">Kan-ZFB-wp insertion</SPAN></H4><UL><LI>Picked 5 colonies from ZF090 and ZF091 plates for a total of 10, ZF092 had too few colonies to pick from
</LI><LI>Placed colonies in 20 μL of water and used 1 μl for a 20 μL PCR reaction
</LI><LI>Then took 10 μL from 20 μL dilution and added to 100 μL of LB in 96 well plate and put in 30°C fridge, at 3:19 pm
</LI><LI>Same protocol as <A href="https://2011.igem.org/Team:Harvard/Protocols#PCR" class="external text" rel="nofollow">KAPA PCR</A> with the following changes made
</LI><LI>Step 3, gradient for annealing, 55°C - 77°C, for a range of 55,57,59,61 and 63°C for the two rows of 5 PCR tubes each (placed leftmost)
</LI><LI>Step 4, Extension, 1 min
</LI><LI>Step 5, 25 cycles total
</LI></UL><LI>Name: TOLCGRAD, PCR5 machine.</LI></DIV><DIV id="719" style="display:none"><H2><SPAN class="mw-headline" id="July_19th">July 19th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_5">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Checking_on_plates_78.2C_79.2C_82">Checking on plates 78, 79, 82</SPAN></H4><P>Both the 10 ul and 1 ul essentially had a lawn of bacteria growing on them. We re-plated these samples, using 1/100 ul and 1/10000 ul of bacteria in 50 ul of LB. We will leave these plates to incubate overnight at 37*.
</P><P>We also made a culture from the old 85 culture, which we will plate tomorrow.
</P><H4><SPAN class="mw-headline" id="Minipreps_of_colonies_and_controls">Minipreps of colonies and controls</SPAN></H4><P>Our minipreps from Saturday had low yields, especially for culture 85. We suspect this is why the PCR [[#Gel analysis of miniprep junction PCR on Sunday did not work for this sample. Unfortunately, 85.3 and 85.4 did not grow even after being left in the shaker overnight. These were two cultures we made by picking new colonies from plate 85. This may have occurred because we did not actually pick the colony, or the colony was a cheater. Thus, we performed minipreps on 80.1, 80.2, 81.1, 81.4 (the controls, chosen for their wide range of concentrations after our first miniprep), and 85.1 and 85.2. 
</P><P>In order to figure our why our minipreps were resulting in unexpectedly low yields and increase our yields:
</P><UL><LI>We tested the pH of our water and Buffer EB. 
</LI><LI>The pH of our aliquot of water was ~4-5 (too acidic...we are unsure why)
</LI><LI>The pH of our Buffer EB was ~7. 
</LI><LI>Ideally, the DNA should be eluted by a solution of pH 7, so we decided to use Buffer EB during the elution step.
</LI></UL><LI>We also checked the copy number of the origin of replication used in our plasmids. ColE1 apparently has a low copy number, which might also explain our low yields.
</LI><LI>We had made 5 ml of culture in an attempt to increase our yields. 
</LI><P>Our yields were larger than our first round of minipreps, although the purities were rather low. We decided to proceed anyway. 
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Concentration and purity of our miniprep product</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="PCR_of_miniprep_products">PCR of miniprep products</SPAN></H4><P>We then ran a PCR for the cross-junction on all 6 (80.1, 80.2, 81.1, 81.4, 85.1, 85.2) samples, using our usual cross junction protocol. 
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gel results, expected product size of 1.4 kb</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>We saw bands for samples 80.1, 81.1, 81.4, 85.1, and a very faint band for 85.2.  The strange thing about these bands is that they appeared to be right around 1650 bp, larger than our 1.4 kb target size.  We are unsure of the identity of these bands and will send them out for sequencing.
</P><H4><SPAN class="mw-headline" id="Starting_a_new_plate_for_sample_85">Starting a new plate for sample 85</SPAN></H4><P>Since we have been having such bad luck with sample 85, we seeded 2 ml of LB+spec with 100 ul of the transformed bacteria for sample 85.  This is in preparation for making a new plate of sample 85 (the old plate was not a good spec plate and had lots of unspecific growth as well as very few colonies, so if we used this plate for more colony picking our choices would be very limited).  This new plate should also give us more luck with picking colonies and having them actually grow in the LB+spec.
</P><H4><SPAN class="mw-headline" id="Designing_ultramers_for_Zif268.2C_OZ052.2C_and_OZ123">Designing ultramers for Zif268, OZ052, and OZ123</SPAN></H4><P>In preparation for designing our 96-well practice plate, we designed ultramers for our three positive controls Zif268, OZ052, and OZ123.  These ultramers contain the Type II binding sites as well as the F2/F3 fingers.  Since the practice plate will contain oligos that encode for F1, we can use these three positive controls to practice and ensure that we secure the technique of using the Type II binding sites to swap in F1 into our expression plasmids.  The ultramers have not yet been ordered.
</P><H3><SPAN class="mw-headline" id="Team_TolC_2">Team TolC</SPAN></H3><H4><SPAN class="mw-headline" id="PCR_to_test_insertion_of_KAN-ZFB-wp_into_ECNR2">PCR to test insertion of KAN-ZFB-wp into ECNR2</SPAN></H4><UL><LI>Made 26 dilutions of culture from three different recombineering plates
</LI><LI>10 cultures from Kan_tolC_locus_F1 (ZF090 plate) primer recombination
</LI><LI>10 cultures from Kan_tolC_locus_F2 (ZF091 plate) primer recombination
</LI><LI>5 cultures from Kan_tolC_locus_F3 (ZF092 plate) primer reecombination
</LI><LI>1 culture from an old Kan plate with recombination from last week
</LI></UL><LI>5 of ZF090, 6 of ZF091, and 5 ZF092 were placed in a PCR with 58˚C annealing temperature
</LI><LI>5 of ZF090, 4 of ZF091, and 1 from old plate were run in a PCR with 62˚C annealing temperature
</LI><LI>primers used were TolC_seq_F and TolC_seq_R
</LI><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">EcNR2 lambda red 1529620 PCR 7/19/11</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">EcNR2 lambda red 1529620 PCR 7/19/11</DIV></DIV></DIV><UL><LI>Chose colonies based on the intensity of the bands and grew up cultures of ZF091-2d, ZF091-4d, ZF090-1d, ZF091-5d, ZF090-1s, ZF090-2s, ZF090-3s, and ZF091-3s
</LI></UL><H4><SPAN class="mw-headline" id="Sequencing_of_Kan-ZFB-wp_insertion">Sequencing of Kan-ZFB-wp insertion</SPAN></H4><UL><LI>Prepared 16 PCR products of the 8 cultures we chose to grow up in order to send out for sequencing
</LI><LI>Used TolC_Seq_F and ZFB-wp-F primers
</LI><LI>Bands looked a little small when ran out gel for longer
</LI></UL><H4><SPAN class="mw-headline" id="New_MAGE_oligo">New MAGE oligo</SPAN></H4><UL><LI>Constructed a new oligo for MAGE knockout of rpoZ, with oligos more than 4 mutations and substitutions the efficiency drops from 15% down to below 5%, so we created a rpoZ knockout oligo with only 2 substitutions
</LI><LI>The new MAGE rpoZ knockout is rpoZ_MAGE_Subs
</LI></UL><LI>Also ordered primers to test the presence of the knockout rpoZ_MAGE_Subs1+, rpoZ_MAGE_Subs1-, rpoZ_MAGE_Subs2+, rpoZ_MAGE_Subs2-
</LI><H3><SPAN class="mw-headline" id="Team_Wolfe_3">Team Wolfe</SPAN></H3><P><B>EcNR2 lambda red colony PCR results:</B></P><UL><LI>PCR showed that colonies 1-3 had the insert while 4 did not. Yay! (Note that in the gel image, we accidentally cut off the 300bp band in lane 4)
</LI><LI>We will grow up a reinoculated culture of colony 1 for a glycerol stock. If the Wolfe strain keeps not working, we at least have the insert in an EcNR2 line and can hopefully MAGE out the other genes without too much difficulty.
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">EcNR2 lambda red 1529620 PCR 7/19/11</DIV></DIV></DIV><P><B>pSR01 growth phenotype:</B></P><UL><LI>We will try to used the plate reader again to compare the growth phenotypes of ∆HisB∆pyrF∆rpoZ and ∆HisB∆pyrF∆rpoZ+pSR01. Used the same media  as last time but since we're using a 48 well plate, each well has 250µL media. We used 5 pSR01 colonies and one selection strain colony as a control.
</LI><LI>N.B.: this plate we are using is not necessarily meant for OD tests like this and may give some variable readings.
</LI></UL><P><B>Lambda red results:</B></P><UL><LI>Yesterday's lambda red, which used Noah's kan-ZFB-wp-hisura, has yet to produce colonies. We will leave the plates in longer in case any come up later.
</LI></UL><P><B>Thio kanZFB:</B></P><UL><LI>Since yesterday's lambda red may not have worked, we need to get the thio kan up and running. We ran out the touchdown PCR samples (since they have more DNA) and will gel extract.
</LI><LI>57.7ng/µL 260/280=1.98
</LI><LI>69.3ng/µL 260/280=1.98</LI></UL><DIV id="720" style="display:none"><H2><SPAN class="mw-headline" id="July_20th">July 20th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_6">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Checking_on_1.2F100_ul_and_1.2F10000_ul_78.2C_79.2C_and_82_plates">Checking on 1/100 ul and 1/10000 ul 78, 79, and 82 plates</SPAN></H4><P>We got colonies on all the plates,  but the 1/100 ul plates look a bit crowded. On the other hand, the 1/10000 plates only have a few colonies. We decided to save both plates, and decide which to use later. 
</P><UL><LI>Based on these results, a 1/1000 ul dilution might be best. 
</LI></UL><H4><SPAN class="mw-headline" id="Check_on_85_culture">Check on 85 culture</SPAN></H4><P>There was no growth in our culture.
</P><P>Later: It ends up that our ultramer from 85 (Pos Con 77) had an error in it. Instead of coding for fingers 2 and 3, it codes for fingers 1 and 2. We ordered a new ultramer, and we no longer need to work on 85. 
</P><H4><SPAN class="mw-headline" id="Finalizing_the_practice_plate">Finalizing the practice plate</SPAN></H4><P>We decided on using a practice plate with 24 oligos, consisting of 18 target sequences, and 6 controls. Our 6 controls are Zif268, OZ052, OZ123, and 3 sequences from the CODA table. We will need to construct the expression plasmid for Zif268, OZ052, and OZ123. Their ultramers were ordered today. 
</P><P>Our 18 targets were GNN/TNN sequences, which are more likely to have a binder, and were distributed among the backbones. The target and control groups each have their own set of primers, to facilitate picking them from the pool.
</P><H4><SPAN class="mw-headline" id="Expression_Plasmid_Sequencing_Results">Expression Plasmid Sequencing Results</SPAN></H4><P>We got back our sequencing results:
</P><TABLE border="1"><TBODY><TR><TD> Name</TD><TD>ID</TD><TD>Final?</TD><TD>Notes
</TD></TR><TR><TD style="background: green"><B>FH Top (77)</B></TD><TD>1</TD><TD>N</TD><TD>1 snp
</TD></TR><TR><TD>2</TD><TD>N</TD><TD>5 snps
</TD></TR><TR><TD><B>3</B></TD><TD><B>Y</B></TD><TD><B>G -&gt; C. Still Ala</B></TD></TR><TR><TD>4</TD><TD>N</TD><TD>1 snp
</TD></TR><TR><TD style="background: red"><B>FH Bottom (78)</B></TD><TD>1</TD><TD>N</TD><TD>1 Deletion at 1746, last nucleotide of F of FQRICMRN
</TD></TR><TR><TD style="background: red"><B>CB Top (79)</B></TD><TD>1</TD><TD>N</TD><TD>A lot of deletions/mistakes in f2/f3 region
</TD></TR><TR><TD style="background: green"><B>CB Bottom (80)</B></TD><TD><B>1</B></TD><TD><B>Y</B></TD></TR><TR><TD><B>2</B></TD><TD><B>Y</B></TD><TD><B>G-&gt;C. Stiil Ala</B></TD></TR><TR><TD> 3</TD><TD>N</TD><TD>deletion 1613-1631
</TD></TR><TR><TD>4</TD><TD>N</TD><TD>many mistakes at end, may be ok
</TD></TR><TR><TD style="background: green"><B>Myc 198 (81)</B></TD><TD><B>1</B></TD><TD><B>Y</B></TD></TR><TR><TD><B>4</B></TD><TD><B>Y</B></TD><TD><B>G-&gt;C. Still Ala.</B></TD></TR><TR><TD style="background: green"><B>Myc 981 (82)</B></TD><TD><B>1</B></TD><TD><B>Y</B></TD></TR><TR><TD style="background: green"><B>Pos Con 16 (83)</B></TD><TD><B> 4</B></TD><TD><B>Y</B></TD><TD><B>G-&gt;C. Still Ala.</B></TD></TR><TR><TD>2</TD><TD>N</TD><TD>A lot of deletions/mistakes in f2/f3 region
</TD></TR><TR><TD>3</TD><TD>N</TD><TD>A lot of deletions/mistakes in f2/f3 region
</TD></TR><TR><TD style="background: green"><B>Pos Con 55 (84)</B></TD><TD><B>1</B></TD><TD><B>Y (?)</B></TD><TD><B>Right sequence, but messy trace</B></TD></TR><TR><TD> 4</TD><TD>N</TD><TD>at 547: deletion at type II binding site
<P>at 665: deletion at beginning of F3
</P></TD></TR><TR><TD style="background: green"><B>Zif 268</B></TD><TD><B>1</B></TD><TD><B>Y</B></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Summary:</B></TD></TR><TR><TD><B>Good:</B> 77.3, 80.1, 80.2, 81.1, 81.4, 82, 83, 84.1 (?), Zif268
</TD></TR><TR><TD><B>Bad (Redo):</B> 78, 79
</TD></TR></TBODY></TABLE><P>We picked 4 new colonies for 78, and 79 since their sequences had many errors. We left these cultures overnight at 37* in the shaker.
</P><H4><SPAN class="mw-headline" id="Construction_of_OZ052_and_OZ123:_Joining_the_Finger_with_Homology_regions">Construction of OZ052 and OZ123: Joining the Finger with Homology regions</SPAN></H4><P>Previously, we did a PCR to reassemble the ultramers for OZ052 and OZ123, but they only coded strictly for F1/F2/F3 and did not have any overhang homology with any surrounding code.  Thus, we performed PCR to add these homology overhangs so that we can integrate them with the omega subunit as well as the spec backbone to create a final expression plasmid.
</P><P>The program we used was 55* annealing temperature, and a 15 second extension time.  Gel results pictured below.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Overlap PCR to extend ultramers with homologies</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>We had an expected product size of 400 bp., which we can see in the gel. Bands are a little weird since there are bands heavier than the heaviest theoretical band (which is the strong product).  We will do a gel extraction tomorrow to get rid of these bands to obtain pure product for use in the isothermal assembly.
</P><H3><SPAN class="mw-headline" id="Team_TolC_3">Team TolC</SPAN></H3><H4><SPAN class="mw-headline" id="rpoZ_MAGE_knockout">rpoZ MAGE knockout</SPAN></H4><UL><LI>Ran two rounds of MAGE on 4 ECNR2 cultures with the kan-ZFB-wp insertion using Noah's rpoZ knockout oligo
</LI><LI>Grew cultures up to mid-log in kan+amp and LB
</LI><LI>Four cultures are ZF091-2d, ZF091-4d, ZF090-1d, and ZF090-5d
</LI><LI>Electroporator read 5.4 ms for all four cultures in first round
</LI><LI>Could not do a third round of MAGE since the E.Coli do not appear to have recovered, and are nowhere near midlog (MAGE round 2 was done by 5:15pm and the cultures were checked at 8:30pm). Will let the culture go on overnight.
</LI><LI>Put 50ul of a 1:100 dilution of each culture on Amp plates. Used a spreader for plates 1&amp;2 and glass beads for plates 3 &amp; 4.
</LI></UL><H4><SPAN class="mw-headline" id="Glycerol_Stocks">Glycerol Stocks</SPAN></H4><UL><LI>Made 4 glycerol stocks of the cultures sent to be sequenced
</LI><LI>ZF090-1s, ZF090-2s, ZF090-3s, ZF091-3s
</LI></UL><H4><SPAN class="mw-headline" id="convinced_that_the_gradient_PCR_machine_is_out_of_order.2C_ought_to_inform_Andrew">convinced that the gradient PCR machine is out of order, ought to inform Andrew</SPAN></H4><H3><SPAN class="mw-headline" id="Team_Wolfe_4">Team Wolfe</SPAN></H3><P><B>IPTG Plate and the Plate Reader</B></P><UL><LI>Technical delays have allowed us only a few hours worth of plate reading for a 46-well plate with selection strain + pSR01 in LB+Amp +/- IPTG, NM+His +/- IPTG, NM +/- IPTG, and NM+3-AT +/- IPTG. Strains used: ∆HisB∆pyrF∆rpoz+pSR01 (5 colonies) and ∆HisB∆pyrF∆rpoz (1 colony).
</LI><LI>The plate had a noticeable white residue under the lid around around the edges. We believe this was due to flaking of the plastic cover from vibration in the plate reader.
</LI><LI> Cell growth in the plate suggested that IPTG is killing our cells since few cultures survived with IPTG and those that did (LB+Amp) grew less than the equivalent cultures without IPTG.  
</LI></UL><P><B>New Plate Without IPTG</B></P><UL><LI> On plate reader at 11:30AM
</LI><LI>Strains used: Water as control, Selection (Wolfe) Strain, Selection Strain + pSR01
</LI><LI>Media Used: LB+Tet (LB+Amp for SS+pSR01),NM+His, NM, NM+3-AT (1µM), NM+3-AT (2.5µM), NM+3-AT (5µM).
</LI></UL><P><B>Lambda Red</B>
Same procedure as usual
</P><UL><LI>Added ~350ng of Thio-Kan-ZFB-wp-His3-Ura3
</LI><LI>Used &quot;new cuvettes&quot;
</LI><LI>Negative water control
</LI></UL><P><B>Plate Reader Update</B>
TEMPORARILY OUT OF SERVICE
</P><P><B>Your worst nightmare</B></P><A href="http://www.youtube.com/watch?v=v4o8MDjuu-A" class="external free" rel="nofollow">http://www.youtube.com/watch?v=v4o8MDjuu-A</A></DIV><DIV id="721" style="display:none"><H2><SPAN class="mw-headline" id="July_21st">July 21st</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_7">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Checking_colonies_78_and_79">Checking colonies 78 and 79</SPAN></H4><P>All of the cultures grew overnight, so we performed a miniprep on them to obtain the plasmids. 
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Plasmid concentrations after miniprep</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>Our concentrations were better than they have been in the past; notably, after the addition of buffer P2, our solution actually turned a bright blue (indicating lysis of the cells), which we had not seen before.
</P><P>We then ran a PCR on our miniprep product, to check for our desired insert. We used our usual protocol for a PCR of the expression plasmid cross-junction.  
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">PCR results, note that some lanes do not show strong bands</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>Some of the products did not show up very strongly, but we sent them out for sequencing in case it was a PCR/gel error.
</P><H4><SPAN class="mw-headline" id="Sending_out_our_expression_colonies_for_sequencing">Sending out our expression colonies for sequencing</SPAN></H4><P>We sent out the samples from yesterday that had no more than 1 SNP for reverse sequencing. In addition, we also sent out 78 and 79 for forward and reverse sequencing. 
</P><UL><LI>Reverse sequencing: 77.1, 77.3, 77.4, 78.1, 80.1, 80.2, 80.4, 81.1, 81.4, 82.1, 83.4, 84.1, 84.4, Zif268
</LI><LI>Forward and reverse sequencing: 781, 78.2, 78.3, 78.4, 79.1, 79.2, 79.3, 79.4
</LI></UL><P>For sequencing, we used the forward primer pZE23G_3581_F CCAACCTTACCAGAGGGCGC (Junction forward) and the reverse primer pZE23G_2123_R CGAACGACCTACACCGAACT (Junction reverse)
</P><P>We did a PCR of the sequences from yesterday that we want sequenced again. 
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">PCR results, looking good</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>The bands were clear, and we will send these products to be sequenced.
</P><H4><SPAN class="mw-headline" id="Gel_Extraction_and_Isothermal_Assembly_of_OZ_fingers.2Bhomology">Gel Extraction and Isothermal Assembly of OZ fingers+homology</SPAN></H4><P>We ran a gel extraction of our PCR product from yesterday, so that we could use it for isothermal assembly later.  Strangely, we got bands that were larger than our largest theoretical product.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Prepare yourself to be gel-extracted</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>We had the following yields from our gel extraction:
</P><PRE> ID         Conc. (ng/ul)            260/280
 OZ052      96.3                     1.93
 OZ123      125.5                    1.91
</PRE><P>Based on these concentrations, we used the following recipes for our isothermal reaction:
</P><UL><LI>OZ052:
</LI><LI>Spec: 0.63 ul
</LI><LI>OZ052: 1.56 ul
</LI><LI>Omega: 0.13 ul
</LI><LI>Water: 1.56 ul
</LI><LI>Isothermal assembly solution: 7.5 ul 
</LI></UL><LI>OZ123: 
</LI><LI>Spec: 0.63 ul
</LI><LI>OZ123: 0.14 ul
</LI><LI>Omega: 0.13 ul
</LI><LI>Water: 1.60 ul
</LI><LI>Isothermal assembly solution: 7.5 ul
</LI><P>We left the isothermal assembly product in the -20* freezer, and will perform the transformation tomorrow.
</P><H3><SPAN class="mw-headline" id="Team_Tolc_4">Team Tolc</SPAN></H3><H3><SPAN class="mw-headline" id="Team_Wolfe_5">Team Wolfe</SPAN></H3><UL><LI>Lambda red culture did not grow colonies overnight on a kanamycin plate. This may be due to a low time (4.8ms) during the electroporation.
</LI><LI>Repeated lambda red using the same procedure. Got a similar time constant and noticed that the water negative control had more cells than the culture containing the insert, but we went ahead and plated 1mL of each on kan plates.
</LI><LI>Also replated the 7/20 lambda red culture to see if we can get any good cells (1.5mL).</LI></UL><DIV id="722" style="display:none"><H2><SPAN class="mw-headline" id="July_22nd">July 22nd</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_8">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Sequencing_Results_of_Expression_Plasmids">Sequencing Results of Expression Plasmids</SPAN></H4><P>The good news is that we managed to get a perfect sequence for each of our plasmids.  The bad news is that we do not know which sequences are the matches, since the names are all scrambled for some reason (i.e. sample 78.x does not align with FH bottom like it should, but instead aligns perfectly with another plasmid sequence).  We will thus need to resend our samples for sequencing to be able to choose the correct plasmids that match.  Below is a summary of the scrambled data from the sequencing we sent in yesterday:
</P><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>ID</B></TD><TD align="center" style="background:#f0f0f0;"><B>Is a perfect match with</B></TD><TD align="center" style="background:#f0f0f0;"><B>Notes</B></TD></TR><TR><TD> Myc 198 (81)</TD><TD>78.2, 78.3</TD></TR><TR><TD> FH Top (77)</TD><TD>81.4, Myc 981</TD><TD>83.4 has 1 deletion
</TD></TR><TR><TD> CB Bottom (80)</TD><TD>84.4</TD><TD>78.1 has 5 deletions; Zif has 1 deletion
</TD></TR><TR><TD> FH bottom (78)</TD><TD>n/a</TD><TD>84.1 deletion at beginning of F3
</TD></TR><TR><TD> CB Top (79)</TD><TD>79.3</TD></TR><TR><TD> Myc 981 (82)</TD><TD>78.4</TD></TR><TR><TD> Zif 268</TD><TD>79.4</TD></TR><TR><TD> Pos Con 55 (83)</TD><TD>79.2</TD><TD>79.3 has deletion at beginning of F3
</TD></TR><TR><TD> Pos Con 16</TD><TD>79.1</TD></TR></TBODY></TABLE><P>
To this end, we redid the PCRs for everything in preparation for the resequencing, using the junction primers and the previous protocol.  The resulting gels can be observed below:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">E-gel showing bands for samples 78.1-78.4 and 79.1-79.4, expected size of 1.4 kb</DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Remaining samples, expected size of 1.4 kb</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>From these PCRs, all the reactions look like they produced bands of the right size, a good indicator that the isothermal assemblies worked.  Only sequencing will tell whether they are truly the products that we are looking for.
</P><P>We sent these samples out for sequencing, making sure that all the samples were in the appropriate tubes.
</P><H4><SPAN class="mw-headline" id="Transformation_for_OZ_plasmids">Transformation for OZ plasmids</SPAN></H4><P>Today, we took the results of the isothermal assembly from yesterday and did chemical transformation to introduce the plasmids into the Top 10 ChemComp bacteria, using <A href="https://2011.igem.org/Team:Harvard/Protocols#Cultures" class="external text" rel="nofollow">our protocol for chemical transformations</A>. They were left to recover for approximately 1.5 hours. We plated 2 dilutions, 10 ul and 50 ul, and left them overnight at 37* in the incubator.
</P><H3><SPAN class="mw-headline" id="Team_Wolfe_6">Team Wolfe</SPAN></H3><UL><LI>Yesterday's lambda red plate had several large colonies--but so did the negative control plate. We will go ahead and do a PCR of the 1529620 locus (as previously described) on 12 of the colonies in case one of them happens to have the insert. The colonies were also grown up simultaneously in LB+kan.
</LI><LI>Results: all of the colonies were cheaters
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">1529620 locus PCR 7/22/11</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">1529620 locus PCR part 2 7/22/11</DIV></DIV></DIV><UL><LI>In case these colonies do not have the insert, as we suspect, we will repeat lambda red with the thio kan-ZFB-wp-his3-ura3 insert
</LI><LI>1mL culture for each condition, 210 ng DNA
</LI><LI>time constant for thio insert sample was 5.0
</LI></UL><H3><SPAN class="mw-headline" id="Team_TolC_5">Team TolC</SPAN></H3><H4><SPAN class="mw-headline" id="Gel_for_Jun.27s_MAGE_rpoZ.2C_rowD">Gel for Jun's MAGE rpoZ, rowD</SPAN></H4><UL><LI>The gel appeared to have bands, so we are running a second PCR to confirm it</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Harvard/Template:NotebookDataJuly3">http://2011.igem.org/Team:Harvard/Template:NotebookDataJuly3</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Harvard/Template:NotebookDataJuly3" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Harvard/Template:NotebookDataJuly3" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Harvard/Template:NotebookDataJuly3&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Harvard/Template:NotebookDataJuly3&amp;oldid=65931" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>