<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Harvard_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Harvard/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Harvard/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Harvard/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Harvard/Notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Harvard/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Harvard/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><TITLE>bar</TITLE></P></DIV><DIV style="font-smooth:always;" id="vsebina"><DIV id="vsebina_mid"><DIV id="vsebina_mid_center"><DIV id="vse_students"><DIV id="desno_students"><DIV id="levo_students"><DIV class="notebook_box"><TABLE class="monthtable"><CAPTION><B>June</B></CAPTION><TBODY><TR><TH>Sun</TH><TH>Mon</TH><TH>Tue</TH><TH>Wed</TH><TH>Thu</TH><TH>Fri</TH><TH>Sat</TH></TR><TR><TD>1</TD><TD> 2</TD><TD> 3</TD><TD> 4</TD></TR><TR><TD>5</TD><TD><SPAN id="student" onclick="show(1)">
6
</SPAN></TD><TD><SPAN id="student" onclick="show(2)">
7
</SPAN></TD><TD><SPAN id="student" onclick="show(3)">
8
</SPAN></TD><TD><SPAN id="student" onclick="show(4)">
9
</SPAN></TD><TD><SPAN id="student" onclick="show(5)">
10
</SPAN></TD><TD>11</TD></TR><TR><TD>12</TD><TD><SPAN id="student" onclick="show(6)">
13
</SPAN></TD><TD><SPAN id="student" onclick="show(7)">
14
</SPAN></TD><TD><SPAN id="student" onclick="show(8)">
15
</SPAN></TD><TD><SPAN id="student" onclick="show(9)">
16
</SPAN></TD><TD><SPAN id="student" onclick="show(10)">
17
</SPAN></TD><TD>18</TD></TR><TR><TD>19</TD><TD><SPAN id="student" onclick="show(11)">
20
</SPAN></TD><TD><SPAN id="student" onclick="show(12)">
21
</SPAN></TD><TD><SPAN id="student" onclick="show(13)">
22
</SPAN></TD><TD><SPAN id="student" onclick="show(14)">
23
</SPAN></TD><TD><SPAN id="student" onclick="show(15)">
24
</SPAN></TD><TD><SPAN id="student" onclick="show(16)">
25
</SPAN></TD></TR><TR><TD>26</TD><TD><SPAN id="student" onclick="show(17)">
27
</SPAN></TD><TD><SPAN id="student" onclick="show(18)">
28
</SPAN></TD><TD><SPAN id="student" onclick="show(19)">
29
</SPAN></TD><TD><SPAN id="student" onclick="show(20)">
30
</SPAN></TD></TR></TBODY></TABLE></DIV><DIV class="notebook_box"><TABLE class="monthtable"><CAPTION><B>July</B></CAPTION><TBODY><TR><TH>Sun</TH><TH>Mon</TH><TH>Tue</TH><TH>Wed</TH><TH>Thu</TH><TH>Fri</TH><TH>Sat</TH></TR><TR><TD><SPAN id="student" onclick="show(64)">
1
</SPAN></TD><TD> 2</TD></TR><TR><TD> 3</TD><TD><SPAN id="student" onclick="show(21)">
4
</SPAN></TD><TD><SPAN id="student" onclick="show(22)">
5
</SPAN></TD><TD><SPAN id="student" onclick="show(23)">
6
</SPAN></TD><TD><SPAN id="student" onclick="show(24)">
7
</SPAN></TD><TD><SPAN id="student" onclick="show(25)">
8
</SPAN></TD><TD><SPAN id="student" onclick="show(65)">
9
</SPAN></TD></TR><TR><TD><SPAN id="student" onclick="show(66)">
10
</SPAN></TD><TD><SPAN id="student" onclick="show(26)">
11
</SPAN></TD><TD><SPAN id="student" onclick="show(27)">
12
</SPAN></TD><TD><SPAN id="student" onclick="show(28)">
13
</SPAN></TD><TD><SPAN id="student" onclick="show(29)">
14
</SPAN></TD><TD><SPAN id="student" onclick="show(30)">
15
</SPAN></TD><TD><SPAN id="student" onclick="show(67)">
16
</SPAN></TD></TR><TR><TD><SPAN id="student" onclick="show(68)">
17
</SPAN></TD><TD><SPAN id="student" onclick="show(31)">
18
</SPAN></TD><TD><SPAN id="student" onclick="show(32)">
19
</SPAN></TD><TD><SPAN id="student" onclick="show(33)">
20
</SPAN></TD><TD><SPAN id="student" onclick="show(34)">
21
</SPAN></TD><TD><SPAN id="student" onclick="show(35)">
22
</SPAN></TD><TD><SPAN id="student" onclick="show(69)">
23
</SPAN></TD></TR><TR><TD><SPAN id="student" onclick="show(70)">
24
</SPAN></TD><TD><SPAN id="student" onclick="show(36)">
25
</SPAN></TD><TD><SPAN id="student" onclick="show(37)">
26
</SPAN></TD><TD><SPAN id="student" onclick="show(38)">
27
</SPAN></TD><TD><SPAN id="student" onclick="show(39)">
28
</SPAN></TD><TD><SPAN id="student" onclick="show(40)">
29
</SPAN></TD><TD><SPAN id="student" onclick="show(71)">
30
</SPAN></TD></TR><TR><TD><SPAN id="student" onclick="show(72)">
31
</SPAN></TD></TR></TBODY></TABLE></DIV><DIV class="notebook_box"><TABLE class="monthtable"><CAPTION><B>August</B></CAPTION><TBODY><TR><TH>Sun</TH><TH>Mon</TH><TH>Tue</TH><TH>Wed</TH><TH>Thu</TH><TH>Fri</TH><TH>Sat</TH></TR><TR><TD><SPAN id="student" onclick="show(41)">
1
</SPAN></TD><TD><SPAN id="student" onclick="show(42)">
2
</SPAN></TD><TD><SPAN id="student" onclick="show(43)">
3
</SPAN></TD><TD><SPAN id="student" onclick="show(44)">
4
</SPAN></TD><TD><SPAN id="student" onclick="show(45)">
5
</SPAN></TD><TD><SPAN id="student" onclick="show(100)">
6
</SPAN></TD></TR><TR><TD><SPAN id="student" onclick="show(101)">
7
</SPAN></TD><TD><SPAN id="student" onclick="show(46)">
8
</SPAN></TD><TD><SPAN id="student" onclick="show(47)">
9
</SPAN></TD><TD><SPAN id="student" onclick="show(48)">
10
</SPAN></TD><TD><SPAN id="student" onclick="show(49)">
11
</SPAN></TD><TD><SPAN id="student" onclick="show(50)">
12
</SPAN></TD><TD><SPAN id="student" onclick="show(203)">
13
</SPAN></TD></TR><TR><TD>14</TD><TD><SPAN id="student" onclick="show(51)">
15
</SPAN></TD><TD><SPAN id="student" onclick="show(52)">
16
</SPAN></TD><TD><SPAN id="student" onclick="show(53)">
17
</SPAN></TD><TD><SPAN id="student" onclick="show(54)">
18
</SPAN></TD><TD><SPAN id="student" onclick="show(55)">
19
</SPAN></TD><TD><SPAN id="student" onclick="show(200)">
20
</SPAN></TD></TR><TR><TD><SPAN id="student" onclick="show(201)">
21
</SPAN></TD><TD><SPAN id="student" onclick="show(56)">
22
</SPAN></TD><TD><SPAN id="student" onclick="show(57)">
23
</SPAN></TD><TD><SPAN id="student" onclick="show(58)">
24
</SPAN></TD><TD><SPAN id="student" onclick="show(59)">
25
</SPAN></TD><TD><SPAN id="student" onclick="show(60)">
26
</SPAN></TD><TD>27</TD></TR><TR><TD>28</TD><TD>29</TD><TD>30</TD><TD>31</TD></TR></TBODY></TABLE></DIV><DIV class="notebook_box"><TABLE class="monthtable"><CAPTION><B>September</B></CAPTION><TBODY><TR><TH>Sun</TH><TH>Mon</TH><TH>Tue</TH><TH>Wed</TH><TH>Thu</TH><TH>Fri</TH><TH>Sat</TH></TR><TR><TD>1</TD><TD> 2</TD><TD> 3</TD></TR><TR><TD>4</TD><TD><SPAN id="student" onclick="show(400)">
5
</SPAN></TD><TD>6</TD><TD><SPAN id="student" onclick="show(401)">
7
</SPAN></TD><TD><SPAN id="student" onclick="show(402)">
8
</SPAN></TD><TD>9</TD><TD>10</TD></TR><TR><TD><SPAN id="student" onclick="show(403)">
11
</SPAN></TD><TD><SPAN id="student" onclick="show(404)">
12
</SPAN></TD><TD>13</TD><TD>14</TD><TD>15</TD><TD>16</TD><TD><SPAN id="student" onclick="show(405)">
17
</SPAN></TD></TR><TR><TD>18</TD><TD><SPAN id="student" onclick="show(406)">
19
</SPAN></TD><TD>20</TD><TD><SPAN id="student" onclick="show(407)">
21
</SPAN></TD><TD><SPAN id="student" onclick="show(408)">
22
</SPAN></TD><TD><SPAN id="student" onclick="show(409)">
23
</SPAN></TD><TD>24</TD></TR><TR><TD>25</TD><TD>26</TD><TD>27</TD><TD>28</TD><TD>29</TD><TD>30</TD><TD>31</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV id="606" style="display:none"><H2><SPAN class="mw-headline" id="June_6th">June 6th</SPAN></H2>
First day of iGEM!</DIV><DIV id="607" style="display:none"><H2><SPAN class="mw-headline" id="June_7th"> June 7th </SPAN></H2><P><B>Miniprep of pKD42 (lambda red)</B></P><P>The lambda red plasmid is needed to enable the recombination used to insert the selection/expression systems into our E. coli cultures.
</P><P>Procedure: followed Qiagen Kit instructions, each student (8) using 1 mL cell suspension 
</P>
Results: DNA reasonably pure (260/280 between 1.8 and 2) and between 25 and 50 ng/µL</DIV><DIV id="608" style="display:none"><H2><SPAN class="mw-headline" id="June_8th"> June 8th </SPAN></H2><P><B>PCR to connect ultramers into OZ052 (Zif268 F2 triplicate, GCCGATGTC)and OZ123 (Zif268 F2 triplicate, GAGTGGTTA):</B></P><P>OZ052:
</P><UL><LI>3µL OZ052_F (10µM stock)
</LI><LI>3µL OZ052_R (10µM stock)
</LI><LI>5µL 10x Pfx amplification buffer
</LI><LI>1.5µL dNTPs
</LI><LI>1µL MgSO4
</LI><LI>0.4µL DNA polymerase
</LI><LI>36.1µL ddH2O
</LI></UL><P>OZ123:
</P><UL><LI>3µL OZ123_F (10µM stock)
</LI><LI>3µL OZ123_R (10µM stock)
</LI><LI>5µL 10x Pfx amplification buffer
</LI><LI>1.5µL dNTPs
</LI><LI>1µL MgSO4
</LI><LI>0.4µL DNA polymerase
</LI><LI>36.1µL ddH2O
</LI></UL><P>Parameters:
</P><UL><LI>1) 94⁰C for 5 min
</LI><LI>2) 94⁰C for 15 sec
</LI><LI>3) 60⁰C for 30 sec
</LI><LI>4) 68⁰C for 1 min
</LI><LI>5) Repeat 2-4 for 25 cycles
</LI><LI>6) 68⁰C for 5 min
</LI><LI>7) 4⁰C forever</LI></UL><DIV id="609" style="display:none"><H2><SPAN class="mw-headline" id="June_9th_-_Wet_Lab"> June 9th - Wet Lab </SPAN></H2><UL><LI>Created cell culture with selection construct (contains ZFB, His3, pyrF on plasmid) and reporter RFP (this will be used to test positive control ZFs, cells fluoresce green when ZF binds)
</LI><LI>Picked colonies, grew in LB/amp liquid media until mid-log
</LI><LI>3 mL of LB, 1.5 µL of 2000x amp
</LI></UL><LI>Once mid-log reached, created glycerol stock, stored stock at -80⁰C.
</LI><LI><DEL>300 µL bacteria, 1200 µL 80% glycerol </DEL><B>This should have been 1200 µL bacteria media, 300 µL 80% glycerol <I>(Corrected 6/14/2011)</I></B> (80% pure glycerol, 20% molecular grade water)
</LI><LI>Spiked new tubes of media with 25 µL bacteria from the mid-log tube to leave overnight
</LI><P>NOTE: reporter RFP did not grow to mid-log by end of day, will let grow overnight to saturation and continue creating glycerol stock tomorrow.
</P><UL><LI>Plated selection strain from gel stab onto tet plate.
</LI></UL><UL><LI>Began primer design for creating the kan/selection construct fusion.
</LI></UL><H2><SPAN class="mw-headline" id="June_9th_-_Bioinformatics">June 9th - Bioinformatics</SPAN></H2><P>Today we focused on reacquainting and familiarizing ourselves with Python. We completed the parsing (reading in) of the sequence and amino acid data so that it is easy to work with: by substituting each amino acid abbreviation (ex. A, N) with its numeric equivalent (ex. 1, 14), we can use a lot of nice math comparisons instead of messy letter/&quot;string&quot; comparisons.
</P><P>After that, we worked on counting the number of times each amino acid appears in each of the 7 positions (unfortunately given by -1,1,2,3,5,6,7), and counting the number of times amino acids are next to each other (ex. ACTQRNF has AC, CT, TQ, etc pairings). Taken overall, we found that L is overwhelmingly in position 5. 
</P><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Acid</B></TD><TD align="center" style="background:#f0f0f0;"><B>-1</B></TD><TD align="center" style="background:#f0f0f0;"><B>1</B></TD><TD align="center" style="background:#f0f0f0;"><B>2</B></TD><TD align="center" style="background:#f0f0f0;"><B>3</B></TD><TD align="center" style="background:#f0f0f0;"><B>5</B></TD><TD align="center" style="background:#f0f0f0;"><B>6</B></TD><TD align="center" style="background:#f0f0f0;"><B>7</B></TD></TR><TR><TD> A</TD><TD>77</TD><TD>140</TD><TD>210</TD><TD>197</TD><TD>0</TD><TD>312</TD><TD>85
</TD></TR><TR><TD> C</TD><TD>12</TD><TD>24</TD><TD>1</TD><TD>6</TD><TD>14</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> D</TD><TD>413</TD><TD>16</TD><TD>694</TD><TD>258</TD><TD>0</TD><TD>142</TD><TD>14
</TD></TR><TR><TD> E</TD><TD>125</TD><TD>74</TD><TD>152</TD><TD>107</TD><TD>0</TD><TD>58</TD><TD>132
</TD></TR><TR><TD> F</TD><TD>0</TD><TD>0</TD><TD>22</TD><TD>0</TD><TD>10</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> G</TD><TD>12</TD><TD>201</TD><TD>328</TD><TD>125</TD><TD>0</TD><TD>177</TD><TD>62
</TD></TR><TR><TD> H</TD><TD>93</TD><TD>144</TD><TD>232</TD><TD>652</TD><TD>0</TD><TD>51</TD><TD>17
</TD></TR><TR><TD> I</TD><TD>70</TD><TD>21</TD><TD>3</TD><TD>26</TD><TD>0</TD><TD>94</TD><TD>73
</TD></TR><TR><TD> K</TD><TD>108</TD><TD>372</TD><TD>46</TD><TD>169</TD><TD>6</TD><TD>321</TD><TD>52
</TD></TR><TR><TD> L</TD><TD>176</TD><TD>37</TD><TD>20</TD><TD>22</TD><TD>3325</TD><TD>75</TD><TD>55
</TD></TR><TR><TD> M</TD><TD>36</TD><TD>54</TD><TD>5</TD><TD>28</TD><TD>0</TD><TD>31</TD><TD>10
</TD></TR><TR><TD> N</TD><TD>23</TD><TD>150</TD><TD>129</TD><TD>940</TD><TD>0</TD><TD>182</TD><TD>61
</TD></TR><TR><TD> P</TD><TD>3</TD><TD>298</TD><TD>77</TD><TD>7</TD><TD>0</TD><TD>36</TD><TD>8
</TD></TR><TR><TD> Q</TD><TD>813</TD><TD>158</TD><TD>180</TD><TD>13</TD><TD>0</TD><TD>136</TD><TD>30
</TD></TR><TR><TD> R</TD><TD>870</TD><TD>539</TD><TD>137</TD><TD>55</TD><TD>3</TD><TD>428</TD><TD>2517
</TD></TR><TR><TD> S</TD><TD>99</TD><TD>970</TD><TD>859</TD><TD>278</TD><TD>0</TD><TD>140</TD><TD>12
</TD></TR><TR><TD> T</TD><TD>243</TD><TD>134</TD><TD>223</TD><TD>350</TD><TD>0</TD><TD>834</TD><TD>83
</TD></TR><TR><TD> V</TD><TD>166</TD><TD>26</TD><TD>27</TD><TD>115</TD><TD>0</TD><TD>341</TD><TD>146
</TD></TR><TR><TD> W</TD><TD>19</TD><TD>0</TD><TD>13</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> Y</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>10</TD><TD>0</TD><TD>0</TD><TD>1
</TD></TR></TBODY></TABLE><P>For pairings, we found patterns, but none as obvious as the L-in-position-5. Read this like a multiplication table: the intersection of L row and M column is how often that pairing was observed.
</P><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;">'
</TD><TD align="center" style="background:#f0f0f0;"><B>A</B></TD><TD align="center" style="background:#f0f0f0;"><B>C</B></TD><TD align="center" style="background:#f0f0f0;"><B>D</B></TD><TD align="center" style="background:#f0f0f0;"><B>E</B></TD><TD align="center" style="background:#f0f0f0;"><B>F</B></TD><TD align="center" style="background:#f0f0f0;"><B>G</B></TD><TD align="center" style="background:#f0f0f0;"><B>H</B></TD><TD align="center" style="background:#f0f0f0;"><B>I</B></TD><TD align="center" style="background:#f0f0f0;"><B>K</B></TD><TD align="center" style="background:#f0f0f0;"><B>L</B></TD><TD align="center" style="background:#f0f0f0;"><B>M</B></TD><TD align="center" style="background:#f0f0f0;"><B>N</B></TD><TD align="center" style="background:#f0f0f0;"><B>P</B></TD><TD align="center" style="background:#f0f0f0;"><B>Q</B></TD><TD align="center" style="background:#f0f0f0;"><B>R</B></TD><TD align="center" style="background:#f0f0f0;"><B>S</B></TD><TD align="center" style="background:#f0f0f0;"><B>T</B></TD><TD align="center" style="background:#f0f0f0;"><B>V</B></TD><TD align="center" style="background:#f0f0f0;"><B>W</B></TD><TD align="center" style="background:#f0f0f0;"><B>Y</B></TD></TR><TR><TD> A</TD><TD>10</TD><TD>0</TD><TD>99</TD><TD>55</TD><TD>0</TD><TD>29</TD><TD>122</TD><TD>20</TD><TD>32</TD><TD>332</TD><TD>2</TD><TD>59</TD><TD>55</TD><TD>63</TD><TD>255</TD><TD>87</TD><TD>24</TD><TD>43</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> C</TD><TD>0</TD><TD>0</TD><TD>15</TD><TD>0</TD><TD>0</TD><TD>3</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>5</TD><TD>0</TD><TD>0</TD><TD>6</TD><TD>0</TD><TD>31</TD><TD>6</TD><TD>14</TD><TD>0</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> D</TD><TD>99</TD><TD>15</TD><TD>94</TD><TD>92</TD><TD>0</TD><TD>39</TD><TD>62</TD><TD>6</TD><TD>84</TD><TD>342</TD><TD>15</TD><TD>120</TD><TD>55</TD><TD>42</TD><TD>277</TD><TD>290</TD><TD>87</TD><TD>21</TD><TD>0</TD><TD>8
</TD></TR><TR><TD> E</TD><TD>55</TD><TD>0</TD><TD>92</TD><TD>42</TD><TD>0</TD><TD>34</TD><TD>77</TD><TD>1</TD><TD>38</TD><TD>141</TD><TD>2</TD><TD>39</TD><TD>4</TD><TD>29</TD><TD>134</TD><TD>28</TD><TD>90</TD><TD>26</TD><TD>0</TD><TD>1
</TD></TR><TR><TD> F</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>10</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>22</TD><TD>4</TD><TD>0</TD><TD>2</TD><TD>4</TD><TD>6</TD><TD>0</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> G</TD><TD>29</TD><TD>3</TD><TD>39</TD><TD>34</TD><TD>0</TD><TD>38</TD><TD>56</TD><TD>0</TD><TD>14</TD><TD>126</TD><TD>1</TD><TD>95</TD><TD>28</TD><TD>47</TD><TD>119</TD><TD>125</TD><TD>38</TD><TD>7</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> H</TD><TD>122</TD><TD>0</TD><TD>62</TD><TD>77</TD><TD>0</TD><TD>56</TD><TD>118</TD><TD>9</TD><TD>103</TD><TD>498</TD><TD>4</TD><TD>88</TD><TD>24</TD><TD>26</TD><TD>87</TD><TD>159</TD><TD>70</TD><TD>2</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> I</TD><TD>20</TD><TD>0</TD><TD>6</TD><TD>1</TD><TD>10</TD><TD>0</TD><TD>9</TD><TD>6</TD><TD>8</TD><TD>95</TD><TD>3</TD><TD>5</TD><TD>17</TD><TD>3</TD><TD>62</TD><TD>16</TD><TD>17</TD><TD>4</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> K</TD><TD>32</TD><TD>0</TD><TD>84</TD><TD>38</TD><TD>0</TD><TD>14</TD><TD>103</TD><TD>8</TD><TD>84</TD><TD>386</TD><TD>24</TD><TD>44</TD><TD>19</TD><TD>102</TD><TD>269</TD><TD>163</TD><TD>113</TD><TD>22</TD><TD>1</TD><TD>0
</TD></TR><TR><TD> L</TD><TD>332</TD><TD>5</TD><TD>342</TD><TD>141</TD><TD>0</TD><TD>126</TD><TD>498</TD><TD>95</TD><TD>386</TD><TD>174</TD><TD>32</TD><TD>686</TD><TD>16</TD><TD>112</TD><TD>362</TD><TD>276</TD><TD>875</TD><TD>360</TD><TD>0</TD><TD>8
</TD></TR><TR><TD> M</TD><TD>2</TD><TD>0</TD><TD>15</TD><TD>2</TD><TD>0</TD><TD>1</TD><TD>4</TD><TD>3</TD><TD>24</TD><TD>32</TD><TD>0</TD><TD>7</TD><TD>2</TD><TD>11</TD><TD>39</TD><TD>14</TD><TD>3</TD><TD>1</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> N</TD><TD>59</TD><TD>0</TD><TD>120</TD><TD>39</TD><TD>22</TD><TD>95</TD><TD>88</TD><TD>5</TD><TD>44</TD><TD>686</TD><TD>7</TD><TD>8</TD><TD>36</TD><TD>28</TD><TD>120</TD><TD>254</TD><TD>84</TD><TD>34</TD><TD>1</TD><TD>0
</TD></TR><TR><TD> P</TD><TD>55</TD><TD>6</TD><TD>55</TD><TD>4</TD><TD>4</TD><TD>28</TD><TD>24</TD><TD>17</TD><TD>19</TD><TD>16</TD><TD>2</TD><TD>36</TD><TD>0</TD><TD>3</TD><TD>29</TD><TD>150</TD><TD>21</TD><TD>13</TD><TD>11</TD><TD>0
</TD></TR><TR><TD> Q</TD><TD>63</TD><TD>0</TD><TD>42</TD><TD>29</TD><TD>0</TD><TD>47</TD><TD>26</TD><TD>3</TD><TD>102</TD><TD>112</TD><TD>11</TD><TD>28</TD><TD>3</TD><TD>100</TD><TD>261</TD><TD>314</TD><TD>125</TD><TD>19</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> R</TD><TD>255</TD><TD>31</TD><TD>277</TD><TD>134</TD><TD>2</TD><TD>119</TD><TD>87</TD><TD>62</TD><TD>269</TD><TD>362</TD><TD>39</TD><TD>120</TD><TD>29</TD><TD>261</TD><TD>618</TD><TD>343</TD><TD>504</TD><TD>281</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> S</TD><TD>87</TD><TD>6</TD><TD>290</TD><TD>28</TD><TD>4</TD><TD>125</TD><TD>159</TD><TD>16</TD><TD>163</TD><TD>276</TD><TD>14</TD><TD>254</TD><TD>150</TD><TD>314</TD><TD>343</TD><TD>592</TD><TD>173</TD><TD>91</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> T</TD><TD>24</TD><TD>14</TD><TD>87</TD><TD>90</TD><TD>6</TD><TD>38</TD><TD>70</TD><TD>17</TD><TD>113</TD><TD>875</TD><TD>3</TD><TD>84</TD><TD>21</TD><TD>125</TD><TD>504</TD><TD>173</TD><TD>154</TD><TD>28</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> V</TD><TD>43</TD><TD>0</TD><TD>21</TD><TD>26</TD><TD>0</TD><TD>7</TD><TD>2</TD><TD>4</TD><TD>22</TD><TD>360</TD><TD>1</TD><TD>34</TD><TD>13</TD><TD>19</TD><TD>281</TD><TD>91</TD><TD>28</TD><TD>12</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> W</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>1</TD><TD>0</TD><TD>0</TD><TD>1</TD><TD>11</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0
</TD></TR><TR><TD> Y</TD><TD>0</TD><TD>0</TD><TD>8</TD><TD>1</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>8</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0
</TD></TR></TBODY></TABLE>
Follow up work on this will be to convert this table to frequencies instead of values: values are less meaningful. </DIV><DIV id="610" style="display:none"><H2><SPAN class="mw-headline" id="June_10th_-_Wet_Lab"> June 10th - Wet Lab </SPAN></H2><UL><LI>What we learned today: don't put E. coli plates in the -20C freezer!
</LI></UL><UL><LI>Observed a well populated selection strain plate and placed it in the 4C refrigerator
</LI></UL><UL><LI>Took the selection construct culture and extracted the plasmid through miniprep
</LI><LI>Observed 260/280 ratio of 1.90 and 1.88 through Nanodrop
</LI><LI>Observed concentrations of 87.7 and 100.6 ng/µL through Nanodrop
</LI></UL><UL><LI>Made 10 new agar plates with LB and amp
</LI></UL><H2><SPAN class="mw-headline" id="June_10th_-_Bioinformatics">June 10th - Bioinformatics </SPAN></H2><H3><SPAN class="mw-headline" id="Visualizations">Visualizations</SPAN></H3><P>We spent the first few hours today making cool visualizations and graphs of the data we found on the 9th: heatmaps turned out to be an annoying limitation of Python, so a Python/R hybrid was used, and bar charts were made exclusively in Python. See the dropbox for our pretty (and hopefully informative compared to spreadsheets) charts/graphs.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">A heatmap of the pairing data. The darker the blues indicate that the pairing occurs more often.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>We then started work on TNN and GNN properties specifically (essentially repeating the June 9th work, but confined to smaller data sets). There are some differences between TNN and GNN: see graphs in dropbox. We decided that there was not enough data for fingers that bind to ANN and CNN triplets to perform significant analysis on it.  
</P><TABLE><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">A heatmap of the GNN pairing data.</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">A heatmap of the TNN pairing data.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><UL><LI>Overall, similar color clusters are found in the heatmaps. In all cases, L and N are often placed consecutively on the helix. There are fewer clusters of high frequency when looking at TNN binders. 
</LI></UL><P>
We then, using the theorized framework from a paper (2011 Persikov <A href="http://iopscience.iop.org.ezp-prod1.hul.harvard.edu/1478-3975/8/3/035010/" class="external autonumber" rel="nofollow">[1]</A>), tried to match amino acid binding to each base pair to see if there was a pattern. See dropbox document .../bioinformatics/Binding Frequency for that data. There's a lot of it.
</P><H3><SPAN class="mw-headline" id="Properties_of_amino_acids">Properties of amino acids</SPAN></H3><P>We then worked on finding properties of the each position (hydrophobic/phillic, non/polar):
</P><P><B>Hydrophilic vs Hydrophobic</B></P><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Position</B></TD><TD align="center" style="background:#f0f0f0;"><B>Very Phobic</B></TD><TD align="center" style="background:#f0f0f0;"><B>Hydrophobic</B></TD><TD align="center" style="background:#f0f0f0;"><B>Neutral</B></TD><TD align="center" style="background:#f0f0f0;"><B>Hydrophillic</B></TD></TR><TR><TD> 6</TD><TD>285</TD><TD>85</TD><TD>204</TD><TD>2782
</TD></TR><TR><TD> 5</TD><TD>542</TD><TD>312</TD><TD>1334</TD><TD>1169
</TD></TR><TR><TD> 4</TD><TD>3334</TD><TD>14</TD><TD>0</TD><TD>9
</TD></TR><TR><TD> 3</TD><TD>191</TD><TD>203</TD><TD>1417</TD><TD>1536
</TD></TR><TR><TD> 2</TD><TD>91</TD><TD>211</TD><TD>1819</TD><TD>1236
</TD></TR><TR><TD> 1</TD><TD>138</TD><TD>164</TD><TD>1604</TD><TD>1451
</TD></TR><TR><TD> -1</TD><TD>468</TD><TD>90</TD><TD>1257</TD><TD>1542
</TD></TR></TBODY></TABLE><P><B>Polar vs Nonpolar</B></P><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Position</B></TD><TD align="center" style="background:#f0f0f0;"><B>Polar</B></TD><TD align="center" style="background:#f0f0f0;"><B>Nonpolar</B></TD></TR><TR><TD> 6</TD><TD>2917</TD><TD>440
</TD></TR><TR><TD> 5</TD><TD>2290</TD><TD>1067
</TD></TR><TR><TD> 4</TD><TD>9</TD><TD>3348
</TD></TR><TR><TD> 3</TD><TD>2830</TD><TD>527
</TD></TR><TR><TD> 2</TD><TD>2652</TD><TD>705
</TD></TR><TR><TD> 1</TD><TD>2555</TD><TD>802
</TD></TR><TR><TD> -1</TD><TD>2784</TD><TD>573
</TD></TR></TBODY></TABLE>
Follow up work here is to check more properties, and maybe try individual pairings (ex. phobic-philic, polar-phillic).</DIV><DIV id="613" style="display:none"><H2><SPAN class="mw-headline" id="June_13th_-_Wet_Lab"> June 13th - Wet Lab </SPAN></H2><P>The control zinc fingers OZ052 and OZ123 were amplified with overhanging primers to allow its insertion into the Wolfe plasmid:
</P><P><B>Overhang PCR for ultramers:</B> the template was the product of the ultramer PCR (see 6/8/11), and several concentrations were used
</P><P>In all the tubes:
</P><UL><LI>5 µL Pfx amplification buffer
</LI><LI>1.5 µL dNTPs
</LI><LI>1 µL MgSO4
</LI><LI>0.4 µL polymerase
</LI><LI>38.1 µL ddH2O
</LI><LI>1.5 µL OZ052_up and 1.5 µL OZ052_down OR 1.5 µL OZ123_up and 1.5 µL OZ123_down
</LI></UL><P>In OZ052 (1) and OZ123 (1):
</P><UL><LI>1 µL of ultramer PCR product
</LI></UL><P>In OZ052 (1:10) and OZ123 (1:10):
</P><UL><LI>1 µL of a 1 in 10 dilution of ultramer PCR product
</LI></UL><P>In OZ052 (1:100) and OZ123 (1:100):
</P><UL><LI>1 µL of a 1 in 100 dilution of ultramer PCR product
</LI></UL><P>Parameters:
</P><UL><LI>94⁰C for 5 min
</LI><LI>94⁰C for 15 sec
</LI><LI>55⁰C for 30 sec
</LI><LI>68⁰C for 30 sec
</LI><LI>Repeat steps 2-4 for 25 cycles
</LI><LI>68⁰C for 5 min
</LI><LI>4⁰C forever
</LI></UL><P>Gel to verify proper amplification (1% agarose gel, 10 µL 1 kb ladder, 120 V):
</P><P>The OZ052 lanes (1-3) had bands at the proper length (328 bp) at all three concentrations, although there were several fainter bands likely from side products. Only the undiluted OZ123 lane showed any bands, and from the faint band at 328 and the stronger band around 250 it appears that the PCR did not work well, and the majority of the product was the ultramer from the first PCR. 
</P><P><B>PCR around vector:</B> the template used was the Wolfe selection construct plasmid miniprepped 6/10/11 (100.6 ng/µL stock)
Reagents the same as above except:
</P><UL><LI>1.5 µL of Wolfe_F and 1.5 µL of Wolfe_R primers to each tube
</LI><LI>plasmid tube (1 ng) given 1 ng of template (1 µL of a 1 in 100 dilution)
</LI><LI>plasmid tube (10 ng) given 10 ng of template (1 µL of a 1 in 10 dilution)
</LI></UL><P>Parameters same as above except:
</P><UL><LI>elongation (step 4) 5 minutes (vector approximately 5 kb)
</LI></UL><P>Gel to verify proper amplification (1% agarose, 10 µL 1 kb ladder, 170V)
</P><P>There were no bands of the correct size in the lanes.  The only band that appeared was a faint, short band in one lane that likely was a primer.  Since the DNA ladder worked, the problem likely was not with the electrophoresis but with the PCR reaction, perhaps due to issues with the primers.
</P><H3><SPAN class="mw-headline" id="Gel_images">Gel images</SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Ultramer Overhang 6/13/11</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Backbone plasmid 6/13/11</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="June_13th_-_Bioinformatics"> June 13th - Bioinformatics </SPAN></H2><P>Today we started work on a program to statistically generate possible sequences.
</P><P>The four functions needed to do this are:
</P><UL><LI> generate(matrix, pseudocounts (lambda), dependency tuples)
</LI></UL><DL><DD> takes a matrix of zinc-finger AA position counts, a list of dependent amino acid pairs, and a pseudocount multiplier and generates a list of potential amino acid sequences weighted by independent and dependent probabilities
</DD></DL><UL><LI> add_pseudo(dependent matrix row,independent matrix row)
</LI></UL><DL><DD> given a matrix row of dependent counts (i.e. how many times 'a' occurs at position n when 'b' is set to some AA at position m) and a row of independent matrix counts (how many times 'a' occurs at n regardless of b's AA) return an adjusted matrix row, based on the dependent matrix row, that has pseudocounts added to the values that are empty in the dependent matrix row but filled in the independent matrix row. 
</DD></DL><UL><LI> generate_indep(matrix)
</LI></UL><DL><DD> randomly pick an amino acid, given a matrix row, from a weighted random distribution based on the values in the row
</DD></DL><UL><LI> generate_dep(indep_row, dep_row, lambda)
</LI></UL><DL><DD> add pseudo counts (call add_pseudo) and generate a dependent random call for a position (using generate_indep on the adjusted matrix)
</DD></DL>
We finished generate_indep, generate_dep, and add_pseudo today, along with creating a 140x140 matrix of needed values.</DIV><DIV id="614" style="display:none"><H2><SPAN class="mw-headline" id="June_14"> June 14 </SPAN></H2><H3><SPAN class="mw-headline" id="Wet_Lab">Wet Lab</SPAN></H3><UL><LI>Made four LB-based media solutions, and later created glycerol stocks from these and placed in -80⁰C freezer
</LI><LI>Selection strain (ΔHis3ΔPyrFΔrpoZ) in 3 mL LB and 3µL of 1000x Tet solution stock
</LI><LI>Selection strain (ΔHis3ΔPyrFΔrpoZ) in 3 mL of LB only solution stock (control)
</LI><LI>Kan cassette (pZE22G) in 3 mL of LB solution stock and 3 µL of kanamycin solution stock
</LI><LI>Lambda Red (pKD42) in 3 mL of LB and 3 µL Amp solution stock
</LI><LI>For all of these stocks, we tried to grow all to mid-log and then place them in 1,200 µL of culture and 300 µL of 80 % glycerol solution (<I>This is the correct protocol for creating a glycerol stock; refer to June 9th</I>)
</LI><LI>We were only able to get the kan cassette to mid-log and created glycerol stock of the kan cassette
</LI><LI>Observations included contamination of a pKD46 liquid culture, and we are leaving Lambda Red and both solutions with the selection strain for overnight growth
</LI></UL><LI>Ran 1% gel (150V) with the rest of the OZ123 and OZ052 overhang PCR samples
</LI><LI>Used better ladder today, less the 1 kb ladder
</LI><LI>Bands followed the same pattern as the gel run on 6/13/11
</LI><LI>Used gel extraction to obtain the correct OZ123 and OZ052 PCR product from the gel
</LI><LI>Used Qiagen quick gel extraction kit
</LI><LI>OZ052 (from undiluted lane): 7.0 ng/µL, 260/280=2.42 (Note: this sample had a strange yellow substance in the column, may have been contaminated)
</LI><LI>OZ052 (from undiluted lane): 12.0 ng/µL, 260/280=2.02
</LI><LI>OZ052 (from 1:10 dilution): 10.8 ng/µL, 260/280=2.04
</LI><LI>OZ052 (from 1:10 dilution): 15.2 ng/µL, 260/280=1.82
</LI><LI>OZ052 (from 1:100 dilution): 20.6 ng/µL, 260/280=2.17
</LI><LI>OZ123 (from undiluted lane): 6.3 ng/µL, 260/280=2.17
</LI><LI>PCR the backbone fragment of the plasmid using Wolfe_R and Wolfe_L primers and a lower annealing temperature than before due to the lower melting point of Wolfe_L
</LI><LI>Reagents:
</LI><LI>22µL Invitrogen Platinum PCR supermix
</LI><LI>1µL template from a 1 in 100 dilution (1 ng)
</LI><LI>1µL Wolfe_F and 1µL Wolfe_R 
</LI><LI>94°C for 30 s
</LI><LI>94°C for 30 s
</LI><LI>53°C for 30 s
</LI><LI>70°C for 5 min
</LI><LI>Previous three steps repeat 30 times
</LI><LI>70°C for 5 min
</LI><LI>4°C forever
</LI><LI>Performed a restriction enzyme digestion on the selection construct plasmid using EcoRI to test for presence/absence of inserted selection construct
</LI><LI>1µL EcoR1
</LI><LI>1µL buffer 4
</LI><LI>2µL backbone plasmid
</LI><LI>6µL ddH2O
</LI><LI>Incubate at 37 degrees for 90 min
</LI><LI>There is only one EcoR1 site (GAATTC) in the plasmid, so we should see 1 band at about 5kb
</LI><LI>Ran a gel (1%, 170 V) on the backbone fragment (Wolfe primers) PCR product and restriction digestion result
</LI><LI>Observations: the EcoR1 digest produced the expected band of around 5kb. The backbone did produce a 5kb band but also had a secondary smaller product, perhaps due to one of the primers annealing to a sequence that is a close match to its target.
</LI><LI>Began gradient PCR on the selection strain backbone with Wolfe_R and Wolfe_F primers because of the large difference in melting temperatures between the two
</LI><LI>Set the annealing temperatures within the PCR to go from 50-57 C and ran with 5 minute extension phases at 70 C
</LI><LI>Ran overnight
</LI><H3><SPAN class="mw-headline" id="Today.27s_Gel_Images">Today's Gel Images</SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Ultramer Overhang 6/14/11</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Digestion and PCR Backbone on Selection Construct Plasmid 6/14/11</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="June_14_-_Bioinformatics"> June 14 - Bioinformatics </SPAN></H2><P>We finished writing the generate function, and now have a working sequence generator. We also began more in-depth research into the 2011 work by Persikov which deals with how the zinc finger binds to DNA. He predicts several relations which we should be able to test.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:339px;"><DIV class="thumbcaption">Proposed novel interactions between helical zinc finger residues and base pairs of the target DNA sequence (based on <A href="http://www.ncbi.nlm.nih.gov/pubmed/21572177" class="external text" rel="nofollow">Persikov 2011</A>)</DIV></DIV></DIV><P>Persikov sent us his SVM code (used to calculate the probability of a sequence binding to given DNA), so we also worked on adapting this to use when narrowing our sequences to those most likely to work.
</P><UL><LI>There are four canonical amino acid-base interactions involved in zinc finger interactions. 
</LI><LI>Amino acids in positions -1, 3, and 6 on the helix are known to interact with the 3 bases of the triplet. In addition, the amino acid in position 2 interacts with the upstream base of the complementary strand (Klug 2010).
</LI><LI>In addition, Persikov (2011) has proposed novel interactions between these four amino acids based on his analysis of zinc finger binding data up to 2005. 
</LI><LI>Persikov uses the information between these four amino acid-base interactions in his SVM, to determine whether a finger would be a good binder to a particular DNA sequence. 
</LI></UL><LI>In order to use his program, we need to convert the helices that are created by the generator into a format the SVM accepts. 
</LI><LI>The SVM only considers the four canonical interactions. It assigns a numerical value to each possible amino acid-base combination. The program that converts our data into a format the SVM accepts creates a string with these numerical values based on Persikov's key. (See <A href="https://2011.igem.org/Team:Harvard/Brainstorming#Persikov_2009_SVM" class="external text" rel="nofollow">this page</A> for more details on this program.) 
</LI>
Brandon learned basic Python today. Justin created a JavaScript program that recognizes potential binding sites from a given sequence.</DIV><DIV id="615" style="display:none"><H2><SPAN class="mw-headline" id="June_15th_-_Wet_Lab">June 15th - Wet Lab</SPAN></H2><P><B>Gradient PCR:</B> 5µL of each PCR product were run on a 1% gel. No bands appeared: the PCR appears to not have worked.
</P><P><B>Selection construct:</B> bacteria containing the selection construct (plasmid containing ZF, omega subunit, ZFB, His3, URA3, etc.) was made into a glycerol stock (see 6/14 and 6/9), miniprepped, and used for PCR:
</P><UL><LI>Miniprep: used Qiagen kit
</LI><LI>82.5ng/µL, 260/280=1.99
</LI><LI>91.1ng/µL, 260/280=2.01
</LI><LI>77.9ng/µL, 260/280=1.98
</LI></UL><LI><B>PCR:</B></LI><LI>PCR used to amplify section of plasmid containing zinc finger binding site, weak promoter, His3, and URA3 (with homology to join it to kan cassette)
</LI><LI><B>Reagents</B></LI><LI>zinc finger binding site and weak promoter, selection construct plasmid:
</LI><LI>1µL ZFB-wp-f (5µM)  (made by 1:20 dilution of 100µM stock)
</LI><LI>1µL ZFB-wp-hisura-r (5µM)  (made by 1:20 dilution of 100µM stock)
</LI><LI>1µL selection construct (1:100 dilution of overnight culture)
</LI><LI>22.5µL of invitrogen's Platinum PCR SuperMix 
</LI><LI>PCR used to amplify the kan cassette
</LI><LI><B>Reagents</B></LI><LI>KAN cassette, pZE22g plasmid:
</LI><LI>1µL hisura-kan-f (5µM)  (made by 1:20 dilution of 100µM stock)
</LI><LI>1µL kan-r (5µM)  (made by 1:20 dilution of 100µM stock)
</LI><LI>1µL pZE22g (1:100 dilution of glycerol stock culture)
</LI><LI>22.5µL of invitrogen's Platinum PCR SuperMix
</LI><LI><B>Parameters:</B></LI><LI>1) 94°C for 2 min (denature template, activate enzyme)
</LI><LI>2) 94°C for 30 sec (denature)
</LI><LI>3) 53°C for 30 sec (anneal)
</LI><LI>4) 72°C for 2 min (extend)
</LI><LI>5) Repeat 2-4 for 25 cycles total
</LI><LI>6) 72⁰C for 5 min
</LI><LI>7) 4°C forever
</LI><P><B>PCR Purification:</B></P><UL><LI>Used the Qiagen PCR purification kit and instructions in order to purify the Kan cassette and selections construct PCR products
</LI><LI>Nanodrop the purification results and observed 3 ng/µL for Kan cassette and 29.8 ng/µL for ZFB-wp-His3: purification did not work well, especially for the kan
</LI></UL><P><B>Selection strain (ΔHis3ΔPyrFΔrpoZ):</B></P><UL><LI>saturated overnight culture was inoculated again: 3mL LB, 3µL tet, 30µL of overnight culture, at 37C until mid-log
</LI><LI>glycerol stock
</LI><LI>For transformation tomorrow we grew up pKD42 in 3 mL of LB, 1.5 µL of ampicillin(2000x) and one colony at <B>30 C</B></LI><LI>Also grew up more of the selection strain so it will be ready for electroporation transformation
</LI></UL><P><B>Gel</B></P><UL><LI>Ran gel with Kan cassette and selection construct (Binding site, His3, and URA3)
</LI><LI>Observations successful and image below
</LI><LI>Used 1 kb plus ladder
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Kan and ZFB-wp-his constructs 6/15/11</DIV></DIV></DIV><P><B>PCR Overlap</B></P><UL><LI>since the purification was not very successful, we used 3µL saved from the original PCR product
</LI><LI>Procedure
</LI><LI>25µL of 2x Phusion Master Mix
</LI><LI>1 µL of ZFB-wp-HisURA-R (100µM)
</LI><LI>1µL of HisURA-Kan-F (100µM)
</LI><LI>21 µL of water
</LI><LI>1 µL Kan template and 1 µL of ZFB-wp-His3-URA3
</LI><LI>4 tubes
</LI><LI>Both undiluted
</LI><LI>Both 1:10 dilution
</LI><LI>Both 1:100 dilution
</LI><LI>Both 1:1000 dilution
</LI></UL><LI>Protocol
</LI><LI>98 C for 30 s
</LI><LI>98 C for 10 s
</LI><LI>53 C for 30 s
</LI><LI>72 C for 3 min
</LI><LI>Repeat steps 2-4 for 24 more cycles
</LI><LI>72 C for 5 min
</LI><LI>4 C 4EVA!!!
</LI><H2><SPAN class="mw-headline" id="June_15th_-_Bioinformatics">June 15th - Bioinformatics </SPAN></H2><UL><LI>We continued research into Persikov's and others' work on binding.
</LI></UL><UL><LI>We worked on using the OPEN data to test Persikov's binding-predicting program: SVM 
</LI></UL><UL><LI>Justin continued work on a sequence-finding program, the most up to date version can be found in the Dropbox under code/zfsitefinder.html.  
</LI><LI>Justin and Will found 10 candidate sequences across 4 diseases that hopefully should encompass a good amount of diversity in terms of expanding the ZF library.  These sequences can be found in the table below, with more details <A href="https://2011.igem.org/File:HARVZF_Binding_Sequence_Candidates.xls" class="external text" rel="nofollow">here</A>.
</LI></UL><TABLE class="wikitable" cellpadding="5"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Disease</B></TD><TD align="center" style="background:#f0f0f0;"><B>Target Range</B></TD><TD align="center" style="background:#f0f0f0;"><B>Binding Site Location</B></TD><TD align="center" style="background:#f0f0f0;"><B>Bottom Finger</B></TD><TD align="center" style="background:#f0f0f0;"><B>Top Finger</B></TD><TD align="center" style="background:#f0f0f0;"><B>Bottom AA (F3 to F1)</B></TD><TD align="center" style="background:#f0f0f0;"><B>Top AA (F3 to F1)</B></TD></TR><TR><TD> Colorblindness</TD><TD>chrX:153,403,001-153,407,000</TD><TD>370</TD><TD>GTATTTGTT</TD><TD>GGGCCTGCT</TD><TD>N/A</TD><TD>N/A
</TD></TR><TR><TD> Colorblindness</TD><TD>chrX:153,403,001-153,407,000</TD><TD>3627</TD><TD>GCTGGCTGG</TD><TD>GCGGTAATG</TD><TD>EGSGLKR.EAHHLSR.#######</TD><TD>RRDDLTR.QRSSLVR.#######
</TD></TR><TR><TD> Cystic Fibrosis</TD><TD>chr7:117,074,084-117,089,556</TD><TD>14767</TD><TD>GCAGGTGAT</TD><TD>AAAGAGCCC</TD><TD>QNGTLGR.EAHHLSR.#######</TD><TD>N/A
</TD></TR><TR><TD> Familial Hypercholesterolemia</TD><TD>chr19:11,175,000-11,195,000</TD><TD>14001</TD><TD>GGCTGAGAC</TD><TD>GGAGTCCTG</TD><TD>ESGHLKR.QREHLTT.#######</TD><TD>QTTHLSR.DHSSLKR.#######
</TD></TR><TR><TD> Tay-Sachs</TD><TD>chr15:72,674,944-72,688,031</TD><TD>5888</TD><TD>GTCTGGTCA</TD><TD>TCAAACTCC</TD><TD>DRSSLRR.RREHLTI.#######</TD><TD>N/A
</TD></TR><TR><TD> Pancreatic Cancer</TD><TD>chr7:117,074,084-117,089,556</TD><TD>1739</TD><TD>GATCAAGCT</TD><TD>GTTTCAGTG</TD><TD>N/A</TD><TD>N/A
</TD></TR></TBODY></TABLE><UL><LI>We collected 15 alternative zinc finger backbones (different from zif268 backbone) and their corresponding base sequences. Many of these were from Persikov 2011 and all binding sequences were confirmed on the <A href="http://www.pdb.org/pdb/home/home.do" class="external text" rel="nofollow">Protein Data Bank website</A>. The zinc finger PDB ID's and related links are: 
</LI></UL><TABLE border="1" cellpadding="5"><TBODY><TR><TH scope="col"> PDB ID
</TH><TH scope="col"> Binding Sequence
</TH><TH scope="col"> Link
</TH></TR><TR><TD>1F2I
</TD><TD>ATGGGCGCGCCCAT
</TD><TD><A href="http://www.pdb.org/pdb/explore/explore.do?structureId=1F2I" class="external autonumber" rel="nofollow">[2]</A></TD></TR><TR><TD>1G2D
</TD><TD>GACGCTATAAAAGGAG
</TD><TD><A href="http://www.pdb.org/pdb/explore/explore.do?structureId=1G2D" class="external autonumber" rel="nofollow">[3]</A></TD></TR><TR><TD>1G2F
</TD><TD>TCCTTTTATAGCGTCC
</TD><TD><A href="http://www.pdb.org/pdb/explore/explore.do?structureId=1G2F" class="external autonumber" rel="nofollow">[4]</A></TD></TR><TR><TD>1MEY
</TD><TD>ATGAGGCAGAACT
</TD><TD><A href="http://www.pdb.org/pdb/explore/explore.do?structureId=1MEY" class="external autonumber" rel="nofollow">[5]</A></TD></TR><TR><TD>1TF6
</TD><TD>ACGGGCCTGGTTAGTACCTGGATGGGAGACC
</TD><TD><A href="http://www.pdb.org/pdb/explore/explore.do?structureId=1TF6" class="external autonumber" rel="nofollow">[6]</A></TD></TR><TR><TD>1UBD
</TD><TD>AGGGTCTCCATTTTGAAGCG
</TD><TD><A href="http://www.pdb.org/pdb/explore/explore.do?structureId=1UBD" class="external autonumber" rel="nofollow">[7]</A></TD></TR><TR><TD>1TF6
</TD><TD>ACGGGCCTGGTTAGTACCTGGATGGGAGACC
</TD><TD><A href="http://www.pdb.org/pdb/explore/explore.do?structureId=1TF6" class="external autonumber" rel="nofollow">[8]</A></TD></TR><TR><TD>1YUI
</TD><TD>GCCGAGAGTAC
</TD><TD><A href="http://www.pdb.org/pdb/explore/explore.do?structureId=1YUI" class="external autonumber" rel="nofollow">[9]</A></TD></TR><TR><TD>2DRP
</TD><TD>CTAATAAGGATAACGTCCG
</TD><TD><A href="http://www.pdb.org/pdb/explore/explore.do?structureId=2DRP" class="external autonumber" rel="nofollow">[10]</A></TD></TR><TR><TD>2GLI
</TD><TD>TTTCGTCTTGGGTGGTCCACG
</TD><TD><A href="http://www.pdb.org/pdb/explore/explore.do?structureId=2GLI" class="external autonumber" rel="nofollow">[11]</A></TD></TR><TR><TD>2I13
</TD><TD>CAGATGTAGGGAAAAGCCCGGG
</TD><TD><A href="http://www.pdb.org/pdb/explore/explore.do?structureId=2I13" class="external autonumber" rel="nofollow">[12]</A></TD></TR><TR><TD>2KMK
</TD><TD>CATAAATCACTGCCTA
</TD><TD><A href="http://www.pdb.org/pdb/explore/explore.do?structureId=2KMK" class="external autonumber" rel="nofollow">[13]</A></TD></TR><TR><TD>2PRT
</TD><TD>CGCGGGGGCGTCTG
</TD><TD><A href="http://www.pdb.org/pdb/explore/explore.do?structureId=2PRT" class="external autonumber" rel="nofollow">[14]</A></TD></TR><TR><TD>2WBS
</TD><TD>GAGGCGC
</TD><TD><A href="http://www.pdb.org/pdb/explore/explore.do?structureId=2WBS" class="external autonumber" rel="nofollow">[15]</A></TD></TR><TR><TD>2WBU
</TD><TD>GAGGCGTGGC
</TD><TD><A href="http://www.pdb.org/pdb/explore/explore.do?structureId=2WBU" class="external autonumber" rel="nofollow">[16]</A></TD></TR></TBODY></TABLE></DIV><DIV id="616" style="display:none"><H2><SPAN class="mw-headline" id="June_16th_-_Wet_Lab">June 16th - Wet Lab</SPAN></H2><UL><LI>There was totally a crazy bee hive outside today!!
</LI></UL><P><B>Glycerol Stock pKD42</B></P><UL><LI>Grew up pKD42 in 30 C and once reached mid-log created glycerol stock and placed in -80 refrigerator
</LI></UL><P><B>Overlap PCR gel</B></P><UL><LI>Ran gel to test if overlap PCR that ran through the night worked, and it did not
</LI><LI>Used PCR product without purification which gives good explanation for why it didn't work 
</LI></UL><UL><LI><B>PCR:</B> Since the PCR done the previous day (6/15) we made a back up PCR using phusion mastermix (Finnzyme)
</LI><LI>PCR used to amplify section of plasmid containing zinc finger binding site, weak promoter, His3, and URA3 (with homology to join it to kan cassette)
</LI><LI><B>Reagents</B></LI><LI>zinc finger binding site and weak promoter, selection construct plasmid:
</LI><LI>1µL ZFB-wp-f (100µM)  (taken directly from the primer tube)
</LI><LI>1µL ZFB-wp-hisura-r (100µM)  (taken directly from the primer tube)
</LI><LI>2µL selection construct (1:100 dilution of overnight culture)
</LI><LI>25µL Phusion High-Fidelity PCR Master Mix 
</LI><LI>21µL distilled water (for total volume of 50µL)
</LI></UL><LI>PCR used to amplify the kan cassette
</LI><LI><B>Reagents</B></LI><LI>KAN cassette, pZE22g plasmid:
</LI><LI>1µL hisura-kan-f (100µM)  (taken directly from the primer tube)
</LI><LI>1µL kan-r (100µM)  (taken directly from the primer tube)
</LI><LI>2µL pZE22g (1:100 dilution of glycerol stock culture)
</LI><LI>25µL of Phusion High-Fidelity PCR Master Mix 
</LI><LI>21µL distilled water (for total volume of 50µL)
</LI><LI><B>Parameters:</B></LI><LI>1) 94°C for 2 min (denature template, activate enzyme)
</LI><LI>2) 94°C for 30 sec (denature)
</LI><LI>3) 53°C for 30 sec (anneal)
</LI><LI>4) 72°C for 2 min (extend)
</LI><LI>5) Repeat 2-4 for 25 cycles total
</LI><LI>6) 72⁰C for 5 min
</LI><LI>7) 4°C forever
</LI><LI>Ran PCR product on a gel: bands of the correct size were observed, though the kan band was much fainter than the ZFB-wp-his3
</LI><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Kan cassette and ZFB-wp-his3 constructs 6/16/11</DIV></DIV></DIV><UL><LI><B>Repeat PCR</B> (to get a higher concentration of the Kan cassette and ZFB-wp-his3 constructs)
</LI><LI>same as the above backup PCR (since it was successful), but to a 4x total volume of 200µL, compared to 50µL
</LI><LI>PCR products were run on a gel: the correct bands were observed--see image below (&quot;second gel&quot;)
</LI><LI>PCR product purification: followed Qiagen kit instructions. Strangely, the conc. was 64.6 ng/µL for Kan, purity 2.09 (260/280) and for ZFB-wp-his3, the conc. was 23.7ng/µL and the purity was 1.92 (260/280).
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Kan cassette and ZFB-wp-his3 constructs second gel 6/16/11</DIV></DIV></DIV><P><B>Overlap PCR:</B> used the kan cassette (64.6 ng/µL) and ZFB-wp-his3-ura3 (23.7 ng/µL) purified above
</P><UL><LI>1 µL of kan and 1 µL of ZFB-wp-his3-ura in each tube according to the following conditions:
</LI><LI>two tubes: undiluted
</LI><LI>two tubes: both diluted 1 in 10
</LI><LI>two tubes: both diluted 1 in 100
</LI></UL><LI>12.5µL Phusion master mix
</LI><LI>8 µL ddH2O
</LI><LI>primers: 1.25 µL ZFB-wp-hisura_r (10 µM) and 1.25 µL hisura-kan_f (10 µM)
</LI><LI>we tried two different reaction types: one added the primers as usual before starting the PCR reaction, the other added the primers after 10 PCR cycles (allowing the polymerase to first use the overlapping kan and ZFB to elongate, and then the primers)
</LI><LI>Parameters for PCR starting with primers: 
</LI><LI>(PCR machine 5, program name  EXT3KB in IGEM folder)
</LI><LI>1) 98°C for 1 min (denature template, activate enzyme)
</LI><LI>2) 98°C for 15 sec (denature)
</LI><LI>3) 65°C for 15 sec (anneal)
</LI><LI>4) 72°C for 2 min (extend)
</LI><LI>5) Repeat 2-4 for 30 cycles total
</LI><LI>6) 72⁰C for 5 min
</LI><LI>7) 4°C forever
</LI><LI>Parameters for PCR starting without primers: 
</LI><LI>1) 98C for 1 min
</LI><LI>2) 98C for 15 sec
</LI><LI>3) 65C for 15 sec
</LI><LI>4) 72C for 1 min
</LI><LI>5) back to step 2 for 10 cycles (PCR paused after 10 and primers added)
</LI><LI>6) 98C for 15 sec
</LI><LI>7) 65C for 15 sec
</LI><LI>8) 72C for 2 min
</LI><LI>9) back to step 6 for 20 cycles
</LI><LI>10) 72C for 5 min
</LI><LI>11) 4C forever
</LI><P><B>Transformation</B></P><UL><LI>Used the selection strain (ΔHis3ΔPyrFΔrpoZ) cells at mid-log and attempted to place lambda red (pKD42) plasmid into the cell
</LI><LI>Procedure
</LI><LI>Keep on ice through out whole procedure before use of the electroporation machine
</LI><LI>Spin 1.5 mL of mid-log cells <B>at 4 C</B> for 1 minute at 18000 rcf (we created two tubes through the following steps)
</LI><LI>Discard supernatant and resuspend with 1 mL of <B>cold</B> water
</LI><LI>Spin again and repeat for a second water wash
</LI><LI>With each wash, try to get as much supernatant out as possible(even use pipette) because don't want salts to interfere with the electrical pulse
</LI></UL><LI>Resuspend pellet with 50 µL of cold water
</LI><LI>Add 1 ng of pKD42 to one of the tubes and 45 ng of pKD 42 to the other
</LI><LI>Take all of the liquid in each tube and place in two separate cuvettes for electroporation
</LI><LI>Make sure the electroporation machine is on the right setting (for the cuvettes we used today it was &quot;Ec2&quot;)
</LI><LI>Wipe off all water on the side of the cuvette
</LI><LI>Have 1 mL of LB in hand and after pulsing, immediately put LB in cuvette
</LI><LI>Transfer to culture tube and place in 30 C for 2 hours
</LI><LI>Make 4 LB/amp plates and spread E. coli using glass beads:
</LI><LI>Plate 1: 10 µL of 1 ng culture
</LI><LI>Plate 2: take 700 µL of 1 ng culture, spin down and remove supernatant, resuspend in about 30 µL of LB and plate
</LI><LI>Plate 3: 10 µL of 45 ng culture
</LI><LI>Plate 4: take 700 µL of 45 ng culture, spin down and remove supernatant, resuspend in about 30 µL of LB and plate
</LI><LI>Grow overnight at 30 C
</LI><P><B>PCR to confirm knockouts of selection strain</B></P><UL><LI>this PCR was to confirm that the ΔHis3ΔPyrFΔrpoZ was indeed a knockout for the His3, PyrF, and rpoZ genes
</LI><LI>each primer set was used for two conditions: wild-type (we used a pKD42 culture) and knockout (ΔHis3ΔPyrFΔrpoZ culture, left over from transformation)
</LI><LI>1 µL of either wt or ko template, diluted 1:20
</LI><LI>12.5 µL Phusion master mix
</LI><LI>1.25 µL of each 10µM primer:
</LI><LI>test for His3:
</LI><LI>1)His3_F, His3_R
</LI><LI>2)His3_F, His3_internalR
</LI></UL><LI>test for PyrF:
</LI><LI>3)PyrF_F, PyrF_R
</LI><LI>4)PyrF_F, PyrF_internalR
</LI><LI>test for rpoZ:
</LI><LI>5)rpoZ_F, rpoZ_R
</LI><LI>6)rpoZ_F, rpoZ_internalR
</LI><LI>test for Zeocin (there are two primer sets because we don't know what orientation the Zeocin gene is in)
</LI><LI>7)Zeocin_R, rpoZ_F
</LI><LI>8)Zeocin_R, rpoZ_R
</LI><LI>ddH2O up to 25 µL
</LI><LI>Parameters:
</LI><LI>98 C for 5 min
</LI><LI>98 C for 10 sec
</LI><LI>65 C for 25 sec
</LI><LI>72 C for 45 sec
</LI><LI>cycle 30 times
</LI><LI>72 C for 5 min
</LI><LI>4 C forever
</LI><LI>Results: ran PCR products out on 1% gel (see below). There were some nonspecific bands, but the PyrF and rpoZ genes do appear to be knocked out in the selection strain. His3, however, looks like it's still present--we'll test again to confirm. 
</LI><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gel Confirmation of Knockouts in Selection Strain 6/16/11</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="June_16_-_Bioinformatics">June 16 - Bioinformatics</SPAN></H2><UL><LI><B>Research Targets</B></LI></UL><OL><LI>Clinically relevant targets
</LI><LI>Existing ZFs that bind under-represented triplets
</LI></OL><P><B>Updating our programs</B></P><UL><LI>Many of our current programs currently look at overall data or data based on specific DNA triplets (for example: 'GAT' or 'AAA'). However, in order to more easily understand some of the patterns that occur in the datasets, we want to examine broader subsets of data. For example, do different patterns appear when looking at fingers that bind to 'GNN' triplets versus 'NGN' triplets (where 'N' represents any of the 4 bases)?
</LI><LI>We added the capability for our programs to accept inputs with the variable 'N' by using regular expressions. 
</LI><LI>We can now create lists of the zinc fingers that bind to any triplet, and create interaction matrices and frequency tables for any triplet input.</LI></UL><DIV id="617" style="display:none"><H2><SPAN class="mw-headline" id="June_17th_-_Wet_Lab">June 17th - Wet Lab</SPAN></H2><P><B>Update on selection strain knockout status:</B> We are trying to reach Addgene to check how His3 was knocked out---instead of deleting the gene, they may have simply introduced an early stop codon. If that's the case, our gel would have the correct bands because the primers we designed can only show whether a deletion or insertion was in that locus.
</P><P><B>Transformation results</B>: successful!!
</P><UL><LI>The only plate with colonies was the one plated with 700 µL of cells transformed with 45 ng of pKD42
</LI><LI>Chose a colony to grow in 3mL LB, 1.5µL amp, 30C; make glycerol stock with mid-log cells
</LI><LI>Plate with colonies at 4C
</LI></UL><P><B>Miniprep of pZE22G:</B> (to have the plasmid containing the kan cassette on hand)
</P><UL><LI>used 2 tubes of 1.5mL overnight culture, followed Qiagen kit instructions
</LI><LI>38.0 ng/µL, 260/280=1.99
</LI><LI>27.8ng/µL, 260/280=2.02
</LI></UL><P><B>Overlap PCR gel and extraction</B>: 1%, 150V
</P><UL><LI>Results: adding the primers in after 10 cycles was much more successful than adding them at the beginning, and all three dilutions showed the expected product band (about 2.5kb). The rest of the 1:10 dilution will be run on a gel and extracted.
</LI><LI>11.2ng/µL, 260/280=2.10
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Successful Overlap of Kan Cassette and ZFB Gel 6/17/11</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="June_17_-_Bioinformatics">June 17 - Bioinformatics</SPAN></H2><H3><SPAN class="mw-headline" id="Goals">Goals</SPAN></H3><OL><LI>Make BB Database in program-readable format ✓
</LI><LI>Edit out BB with incomplete helices ✓
</LI><LI>GNN, TNN, CNN, ANN frequencies
</LI></OL><UL><LI>Targets (5-10; <B>8</B>) <B>x</B> Backbones (???) <B>x</B> Helices (≥500)=55,000
</LI><LI>Backbones: similar, but not <I>too</I> similar to zif268; more than 1-2 aa changes, but &lt;10
</LI><LI>Helices fixed based on our program-- eventually saturates and levels out
</LI><LI>Graph: # of var (# of tries by the computer) vs. % space covered
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:339px;"><DIV class="thumbcaption">Proposed interactions between helical zinc finger residues and base pairs of the target DNA sequence (based on <A href="http://www.ncbi.nlm.nih.gov/pubmed/21572177" class="external text" rel="nofollow">Persikov 2011</A>)</DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Options_for_Target_DNA_Sequences_.2F_ZF_Helices">Options for Target DNA Sequences / ZF Helices</SPAN></H4><OL><LI>F3(known) / F2(known) / <B>F1(novel)</B></LI><LI>F3(known) / <B>F2(SNP in b<SUB>1</SUB> position)</B> / F1(known)
</LI><LI><B>F3(unknown) / F2(unknown) / F2(unknown)</B></LI></OL><UL><LI>Excluded Rare Codons (for <I>E. coli</I>)<CITE>CodonUsage OpenWetWareCodonUsage NIHRareCodonCalculator</CITE>:
</LI><LI>CTA
</LI><LI>ATA
</LI><LI>CCC
</LI><LI>CGA
</LI><LI>CGG
</LI><LI>AGA
</LI><LI>AGG
</LI><LI>GGA
</LI><LI>GGG
</LI></UL><H4><SPAN class="mw-headline" id="References">References</SPAN></H4><OL><LI>Persikov2011 pmid=21572177
</LI><LI>CodonUsage <A href="http://www.sci.sdsu.edu/~smaloy/MicrobialGenetics/topics/in-vitro-genetics/codon-usage.html" class="external free" rel="nofollow">http://www.sci.sdsu.edu/~smaloy/MicrobialGenetics/topics/in-vitro-genetics/codon-usage.html</A></LI><LI>OpenWetWareCodonUsage <A href="http://openwetware.org/wiki/Escherichia_coli/Codon_usage" class="external free" rel="nofollow">http://openwetware.org/wiki/Escherichia_coli/Codon_usage</A></LI><LI>NIHRareCodonCalculator <A href="http://nihserver.mbi.ucla.edu/RACC/" class="external free" rel="nofollow">http://nihserver.mbi.ucla.edu/RACC/</A></LI></OL></DIV><DIV id="620" style="display:none"><H2><SPAN class="mw-headline" id="June_20th_-_Wet_Lab">June 20th - Wet Lab</SPAN></H2><UL><LI>Grew up colony of the selection strain with pKD46 in an attempt to reach mid-log and create glycerol stock
</LI><LI>Unable to reach mid-log, so going to leave growing over night and use saturated culture tomorrow
</LI></UL><LI>Determined primers in order to piece together the omega subunit and ZFP genes into the pZE21G plasmid (spec cassette)
</LI><LI>Ran PCR on His3 locus and sent to GENEWIZ to be sequenced
</LI><LI>used the same procedure as the earler WT/KO PCR, but with 1µL undiluted template and only His_F and His_R primers
</LI><LI>ran 3 reactions and sent in three primers (His_F, His_R, His_internalR)
</LI><H2><SPAN class="mw-headline" id="June_20th_-_Bioinformatics">June 20th - Bioinformatics</SPAN></H2><H3><SPAN class="mw-headline" id="Goals_for_the_week"><B>Goals for the week</B></SPAN></H3><UL><LI>Finish designing the chip, by Wednesday hopefully
</LI><LI>Need chip order out, takes 4 weeks
</LI><LI>Need all sequences by <I><B>Friday</B></I>!!!
</LI></UL><LI>FIRST PRIORITY: If we can get Persikov to work, good!
</LI><LI>Step one: get results he’s published, get the web app to &quot;work&quot; with his data, then OPEN data, and finally our data
</LI><LI>Brainstorming session (tomorrow?) to decide how many targets/sequences
</LI><LI>Determine the importance of the first/second/third nucleotide positions
</LI><LI>Look at NGN, NTN, NAN, NCN (<I>Not just GNN, etc.</I>)
</LI><LI>Pick a particular GNN, plot vs. TNN- is there a pronounced difference in position 1, or -1?
</LI><H3><SPAN class="mw-headline" id="Today">Today</SPAN></H3><UL><LI>Testing Persikov's Data for validation
</LI><LI>Persikov v. himself  ✓
</LI><LI>Persikov v. OPEN
</LI><LI>Persikov v. our sequences
</LI></UL><H3><SPAN class="mw-headline" id="Probability_data">Probability data</SPAN></H3><UL><LI>The following are graphs of the probability of finding each amino acid at each position on the alpha helix. 
</LI></UL><TABLE><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Probability data for the 783 fingers that bind to <B>GNN</B> triplets. Note the high probability of leucine at position 4 and arginine at position 6.</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Probability data for the 128 fingers that bind to <B>TNN</B> triplets. Note the high probability of leucine at position 4.</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Probability data for the 16 fingers that bind to <B>CNN</B> triplets. There may not be enough data to consider this information statistically significant</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Probability data for the 29 fingers that bind to <B>ANN</B> triplets. There may not be enough data to consider this information statistically significant</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Probability data for the 298 fingers that bind to <B>NGN</B> triplets. The position 4 leucine motif remains. There is also a high probability (&gt; 0.5) of a histidine at position 3 and an arginine at position 6.</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Probability data for the 177 fingers that bind to <B>NTN</B> triplets. The position 4 leucine motif remains.</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Probability data for the 244 fingers that bind to <B>NCN</B> triplets. The position 4 leucine motif remains. There is also a very high probability of an arginine at position 6.</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Probability data for the 248 fingers that bind to <B>NAN</B> triplets. The position 4 leucine motif remains. There is also a very high probability (&gt; 0.75) of an asparagine at position 3 and an arginine at position 6.</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Probability data for the 234 fingers that bind to <B>NNG</B> triplets. The position 4 leucine motif remains. There is also a very high probability (&gt; 0.75) of an asparagine at position 1 and a high probability (&gt; 0.5) of an aspartic acid at position 2 and an arginine at position 6.</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Probability data for the 247 fingers that bind to <B>NNT</B> triplets. The position 4 leucine motif remains. There is also a high (&gt; 0.5) probability of an arginine at position 6.</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Probability data for the 262 fingers that bind to <B>NNC</B> triplets. The position 4 leucine motif remains. There is also a very high (&gt; 0.75) probability of an arginine at position 6.</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Probability data for the 218 fingers that bind to <B>NNA</B> triplets. The position 4 leucine motif remains. There is also a very high (&gt; 0.75) probability of a glutamine at position -1 and an arginine at position 6.</DIV></DIV></DIV></TD></TR></TBODY></TABLE></DIV><DIV id="621" style="display:none"><H2><SPAN class="mw-headline" id="June_21st">June 21st </SPAN></H2><H2><SPAN class="mw-headline" id="Wet_Lab_2">Wet Lab</SPAN></H2><P><B>His3 sequencing results:</B></P><P>The sequencing results showed that the His3 (HisB) gene is still present in the strain and without any early stop codons. There is a 2 aa deletion in the middle of the protein, but its purpose is unknown and the gene likely is still fully functional.
</P><UL><LI>Restreak selection strain on plate from glycerol stock--tomorrow we will PCR the His3 locus and sequence again just to be sure.
</LI><LI>Made oligos for MAGE to insert stop codons and make a frame shift in the endogenous His3 gene, so that if necessary we can knockout His3 ourselves.
</LI></UL><P><B>Selection strain with lambda red:</B></P><UL><LI>Reinoculated and made glycerol stock
</LI><LI>prepared for MAGE tomorrow
</LI></UL><H2><SPAN class="mw-headline" id="June_21st_-_Bioinformatics">June 21st - Bioinformatics</SPAN></H2><H3><SPAN class="mw-headline" id="Persikov_Statistics_-_Graphs">Persikov Statistics - Graphs</SPAN></H3><TABLE><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Scatterplot of top/bottom 20 with SVM polynomial</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Sequence by sequence (lin SVM)</DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Top/Bottom 20 ZFs (SVM linear)</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Comparison of polynomial vs. linear distribution (polynomial generally higher values)</DIV></DIV></DIV></TD></TR></TBODY></TABLE><UL><LI>FQCRICMRNFS<SUB>zif268 F2 Backbone</SUB>/<I><B>Helix F1</B></I>/TGEKP<SUB>linker</SUB></LI></UL><UL><LI>The Persikov data shows weak predictive power for OPEN amino acid sequences. Our conclusion is that Persikov's program is not well-suited for incorporation into our helix generator. Testing Persikov's helices in his program yeilded mostly accurate results (approximately 24/25 matched known binding information). This is an important test because it proved that we are using the program correctly and that the program is in fact working properly. However, testing the OPEN sequences in Persikov's program resulted in numerous false negative values which informed our decision not to use Persikov's program to check our own hellix-generating program.
</LI></UL><H3><SPAN class="mw-headline" id="Phone_Call_with_Dan">Phone Call with Dan</SPAN></H3><UL><LI>How conservative/risky should we be in terms of using other backbones?
</LI><LI><U><B>Conservative</B></U></LI><LI><B>Possible Pros:</B></LI><LI>More likely to get something that will work
</LI><LI>Depending on how &quot;smart&quot; our probabilities are (from our ZF generation algorithm), we could cover a lot of novel space without straying too far from zif268
</LI><LI><I>Worst Case</I>:Something we can show for iGEM (we covered the same ground OPEN did, and found many of the same ZFs, but with a targeted approach, a &quot;smarter&quot; method-- not throwing random things at it; Chip is not ours, but the program is &quot;smarter&quot;) 
</LI></UL><LI><B>Possible Cons:</B></LI><LI>Might end up covering the same ground as OPEN, but doing a &quot;worse&quot; job than they did
</LI><LI>Less likely to discover new/groundbreaking things (i.e., TNN triplets)
</LI><LI><U><B>Less Conservative</B></U></LI><LI>Have 3-6 target sequences (we're currently going for 8)
</LI><LI>More backbones from non-zif268 than zif268
</LI><LI><B>Pros:</B></LI><LI>We could get luck and find something no one has ever seen before (TNN, ANN). If we throw enough things at it, we're more likely to get luck.
</LI><LI><B>Cons:</B></LI><LI><I>Risk:</I> Many of these backbones (from entire ZF world)may NOT bind DNA (i.e., may bind proteins)
</LI><LI><I>Risk:</I> May not find anything that binds, then the whole project is a dud
</LI><LI><B>What is the more important variable, helices or backbones?</B></LI><LI>Helices seem to be more important, backbones of secondary importance
</LI><LI>Backbones: ZF's unravel DNA, open the major groove-- backbone is important here, changes the bond angle, etc. (Brandon's paper-??)
</LI><LI><I><B>Balance</B></I> needed between low and high risk
</LI><LI>If we find backbones that we know bind DNA, greatly lowers our risk
</LI><LI>Limited spaces on chip: zero-sum game
</LI><LI>With a middle of the road approach, we diminish both benefits and risk (diminishes the benefits of the high risk approach much more than it diminishes the benefits of the conservative approach; i.e., if you're playing the lottery, you're more likely to win if you buy many more tickets)
</LI><LI>We need to compare probabilities of randomly-generated OPEN sequences vs. probabilities of sequences randomly generated by our program
</LI><LI>OPEN tries to cover all space: smaller probability
</LI><LI>If we have a &quot;smarter&quot; algorithm, we can produce fewer
</LI><LI>However,  the idea is not to repeat OPEN, but to go somewhere else, non-GNN sequences
</LI><LI><I><B>Remember:</B></I> OPEN is a <I>Cell</I> paper; the point of the project is not to compare ourselves to them
</LI><LI>If we find binders for 1-2 of our sequences, that would be awesome
</LI><LI>Probably we'll have some that find none, some have 10, our last one might have 1,000 hits (then, we do bioinformatics to figure out why/what those hits were)
</LI><LI>Point: to learn and do high-level bioinformatics, and high-tech cloning techniques in the lab
</LI><LI>If you do find binders, you can write a paper about it!
</LI><LI>We have all the resources we need right now to build our chip
</LI><LI>We need to pick out targets
</LI><LI><B>Need to decide exactly what we want for:</B></LI><LI>No. of target sequences/which ones
</LI><LI>No. of helices/ which ones
</LI><LI>Ratio of zif268 backbones: non-zif268 backbones
</LI><LI>Avoid switching Leucine out of position 4, then change other positions based on our frequencies
</LI><H3><SPAN class="mw-headline" id="Chip_Design">Chip Design</SPAN></H3><UL><LI>No. of sequences will be more than we can put on the chip
</LI><LI>Helices: essentially unlimited
</LI><LI>Put more-likely-to-bind helices into the risky backbones
</LI><LI>Put less-likely-to-bind helices into a zif268 backbone
</LI></UL><LI>Backbones
</LI><LI>Maybe revert to a more targeted approach: pick backbones that we know are transcription factors (TFs), that we know bind to DNA
</LI><LI><I>OR</I> research the ZFs from the phylogenetic tree
</LI><LI>Pick clades to research, see if one looks better than the other
</LI><LI>Why did OPEN cover so many helices, without changing the backbone, but still yield predominantly GNNs?
</LI><LI>If we have an idea of how the backbone might affect binding, maybe we could look into some sort of low-level modeling, etc. so that we wouldn't be grasping? Could Vatsan help with this?
</LI><LI>See 2000 Wolfe paper <A href="http://www.ncbi.nlm.nih.gov/pubmed/10940247" class="external autonumber" rel="nofollow">[17]</A></LI><LI>Backbones <I>could</I> affect interactions between fingers
</LI><LI>Theory: energy penalty to ZF binding-- unravels DNA when binds to it
</LI><LI>We have 12 target sequences
</LI><LI>2 per 4 diseases, 4 for the 5th disease
</LI><LI>If we want to be more conservative, we could throw out Type III, but it could be something cool
</LI><LI><B>We should have mostly Type I (CoDA argument, if this is an F2)</B></LI><LI>Proposed: 3 diseases, 6 sequences
</LI><LI>4 Type I (F3 and F2 known, F1 novel)
</LI><LI>1 Type II (GNN, ANN, GNN)
</LI><LI>1 Type III (All unknown, e.g., TNN, ANN, TNN;<I><B>max 1</B></I>)
</LI><P>Or, for 3 diseases:
</P><OL><LI> Type I's
</LI><LI> Type I, Type II
</LI><LI> Type I, Type III
</LI></OL><UL><LI><B><U>Clinical Targets</U></B></LI></UL><OL><LI> Colorblindness (<B>Type I's</B>)
</LI><LI> Familial Hypercholesterolemia (FH) (1 in 500)
</LI><LI><DEL>Cystic Fibrosis (CF)</DEL></LI><LI><DEL>Tay Sachs</DEL></LI><LI> KRAS- (oncogene/cancer)
</LI></OL><UL><LI><B>Main goal of project</B>: to build outside of what is already known
</LI><LI>If we wanted to cure a disease only, we could just use existing ZFs (i.e., find GNN binding locations)
</LI><LI>Also, we lend a level of specificity for insertion/deletion
</LI><LI>There is the possibility that there might be some area where specificity might demand ANN codons
</LI></UL><P><U><B>Current decision on chip design:</B></U></P><UL><LI>We will have 6 target sequences, 2 each from colorblindness, FH, and KRAS.  All are &quot;Type I&quot; targets (only F1 is novel) with the middle finger chosen from the CODA paper (either GNN or TNN)
</LI><LI>N.B.: the CB and FH sequences make up full ZF nuclease cut sites. The KRAS sites, due to the small number of GNNTNN F3F2 combos available in CODA, are separate, with the flanking ZF nuclease site added afterwards in parentheses
</LI></UL><OL><LI> GGT<B>G</B>GT<B>A</B>AG (CB)
</LI><LI> GGA<B>G</B>TC<B>C</B>TG (FH)
</LI><LI> GGC<B>T</B>GA<B>T</B>GC (KRAS) (CTGAAAATT)
</LI><LI> GGC<B>T</B>GA<B>C</B>AC (FH)
</LI><LI> GGC<B>T</B>GG<B>A</B>AT (KRAS) (GACAAGAGC)
</LI><LI> GTC<B>G</B>CC<B>T</B>CC (CB)
</LI></OL><UL><LI>Targets 3, 4, and 6 are similar to sequences Zif268 variants successfully bind to, so the backbones will be weighted accordingly:
</LI><LI>Zif268_F2 backbone: 6000 helices (per target)
</LI><LI>10 backbones more closely related to Zif268: 300 helices each
</LI></UL><LI>Targets 1, 2, and 5 will have equal distributions of backbones:
</LI><LI>Zif268_F2: 3000 helices
</LI><LI>10 backbones closely related to Zif268: 300 each
</LI><LI>10 backbones more distantly related to Zif268: 300 each
</LI><H3><SPAN class="mw-headline" id="Identifying_dependencies">Identifying dependencies</SPAN></H3><UL><LI>We looked at the <A href="#Probability_data">probability graphs</A> to determine which amino acid positions on the finger's helix interact with which bases. 
</LI><LI>Some interactions are fairly well estabilished, while others have been more recently proposed (See <A href="#June_17_-_Bioinformatics">interaction map (Persikov 2011)</A>)
</LI><LI>To identify these interactions in our own data we looked at which helix positions varied most when you changed the bases. A more rigorous way to do this is to calculate the entropy change as you change the amino acids in each position.  
</LI><LI><B>xNN</B>(Vary base 1): Amino acid 6 changes 
</LI><LI><B>NxN</B>(Vary base 2): Amino acid 3 changes
</LI><LI><B>NNx</B>(Vary base 3): Amino acid -1 and 2(?) changes 
</LI></UL><LI>Our program looks at dependencies between amino acids when generating sequences. 
</LI><LI>We decided on these amino acid dependencies, using both established data and patterns we saw in the OPEN data:
</LI><LI>-1 and 2
</LI><LI>2 and 1
</LI><LI>6 and 5
</LI><LI>Because there is not much data for 'CNN' and 'ANN' sequences (with 16 and 29 known fingers that bind to each triplet, respectively), we should use pseudocounts for these sequences, so that our frequency generator is not too biased toward probabilities that may not be significant.</LI></DIV><DIV id="622" style="display:none"><H2><SPAN class="mw-headline" id="June_22nd_-_Wet_Lab">June 22nd - Wet Lab</SPAN></H2><P><B>Preparing media/reagents for selection system:</B></P><UL><LI>Made 0.1M zinc chloride solution, M9 salt, and 1M magnesium sulphate solutions for the amino acid mixture
</LI><LI><B>M9 Salt solution (20x)</B></LI><LI>67.8 g of disodium phosphate
</LI><LI>30 g of monopotassium phosphate
</LI><LI>5 g of sodium chloride
</LI><LI>10 g of ammonium chloride
</LI><LI>All in 500 mL of distilled water
</LI><LI>Sterile filtered when done dissolving
</LI></UL><P><B>Overhang PCR for 3-part assembly of ZFs, omega subunit, and backbone vector (pZE21G, spec resistance)</B></P><UL><LI>Clone out omega+Zif268:
</LI><LI>template: original selection construct plasmid (ZFB, his3, etc.)
</LI><LI>primers: omega_F+homolog, Zif268_R+homolog
</LI><LI>Protocol
</LI><LI>98 C for 30 sec
</LI><LI>98 C for 10 sec
</LI><LI>68 C for 30 sec
</LI><LI>72 C for 30 sec
</LI><LI>Repeat steps 2-4, 30 times
</LI><LI>4  C for ever
</LI></UL><LI>Clone out omega only:
</LI><LI>template: original selection construct
</LI><LI>primers: omega_F+homolog, omega_R
</LI><LI>Protocol
</LI><LI>Same as omega + Zif268
</LI><LI>Clone OZ052 with overhang:
</LI><LI>template: OZ052 overhang (overhang currently matches selection construct), 1:10b, 15ng/µL
</LI><LI>primers: OZ052_F+omega homolog, OZ052_R+homolog
</LI><LI>Protocol
</LI><LI>Same as omega + Zif268
</LI><LI>Clone OZ123 with overhang:
</LI><LI>template: OZ123 overhang (overhang currently matches selection construct), 1, 6.3ng/µL
</LI><LI>primers: OZ123_F+omega homolog, OZ123_R+homlog
</LI><LI>Protocol
</LI><LI>Same as omega + Zif268
</LI><LI>clone out pZE21G backbone
</LI><LI>template: pZE21G containing cells from plate, diluted 1:10
</LI><LI>primers: back_F, back_R
</LI><LI>Protocol
</LI><LI>98 C for 30 sec
</LI><LI>98 C for 10 sec
</LI><LI>68 C for 30 sec
</LI><LI>72 C for 1:30
</LI><LI>Repeat steps 2-4, 30 times
</LI><LI>72 C for 5 min
</LI><LI>4  C for ever
</LI><LI>25µL reaction:
</LI><LI>12.5µL Phusion mastermix
</LI><LI>1.25µL each primer
</LI><LI>1µL of 1ng/µL dilution of template
</LI><LI>9µL of ddH2O
</LI><LI>Ran gel of the above PCR products and imaged below: only the omega and omega+Zif268 reactions seemed to work
</LI><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">PCR for 3-part ZF assembly</DIV></DIV></DIV><P><B>pZE21G backbone:</B></P><UL><LI>1 colony of pZE21G grown in 3mL LB, 3µL spectinomycin (1000x) until mid-log. Glycerol stock made.
</LI><LI>Miniprep of pZE21G in order to PCR the backbone: used Qiagen kit
</LI><LI>5ng/µL and didn't seem pure: miniprep (or nanodrop) not working
</LI><LI>another colony used to start an overnight culture for PCR/glycerol stocks
</LI></UL><LI>Ran gradient PCR on the miniprep product in order to obtain backbone
</LI><LI>same protocol as before, with 1µL of template
</LI><LI>6 tubes spaced so that annealing temp=60,62,64,66,68,70
</LI><LI>Parameters: (program on PCR5, IGEM-&gt; DOGGED)
</LI><LI>98°C for 30s (initial denaturation)
</LI><LI>98°C for 10s (denature)
</LI><LI>60°C to 71°C for 15s (anneal)
</LI><LI>72°C for 90s (extend)
</LI><LI>Repeat steps 2-4 for 30 cycles total (denature, anneal, extend)
</LI><LI>72°C for 5 min
</LI><LI>4°C forever
</LI><P><B>HisB locus PCR:</B> We repeated the PCR of the selection strain at this locus just to be sure HisB is still present
</P><UL><LI>grew 1 colony from a new selection strain plate Vatsan brought in 0.5mL LB, 0.5µL tet
</LI><LI>used 1 µL of bacterial suspension for PCR following same procedure as 6/20
</LI></UL><P><B>Lambda red to make selection system:</B></P><UL><LI>grew ?His3?PyrF?rpoZ+pKD46 to mid-log (0.4 using OD)
</LI><LI>induced lambda red by shaking culture in 42C water bath for 15 min
</LI><LI>spin down 1mL for 1 min, 18000 rcf at 4C
</LI><LI>wash 2x with cold water, removing as much supernatant as possible
</LI><LI>resuspend with 200 ng kan-ZFB-wp-his-ura template (20µL) and water up to 50µL (30µL)
</LI><LI>electroporate using 1mm gap cuvettes adn 1.80KV. Immediately afterward add 1mL LB to cuvette, mix, and transfer to culture tube containing 2 mL more of LB
</LI><LI>recover for 2hrs, 30C
</LI><LI>spread on kanamycin plates: 100µL, 10µL, or 1µL (the last two dilute with 100µL LB to help spread more easily)
</LI><LI>grow overnight at 30C
</LI></UL><H2><SPAN class="mw-headline" id="June_22nd_-_Bioinformatics">June 22nd - Bioinformatics</SPAN></H2><H3><SPAN class="mw-headline" id="Final_target_sequences">Final target sequences</SPAN></H3><P>Our &quot;tentatively&quot; Final DNA Target Sequences (i.e. barring any major objections, we're going with this):
</P><TABLE class="wikitable" cellpadding="5"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Disease</B></TD><TD align="center" style="background:#f0f0f0;"><B>Target Range</B></TD><TD align="center" style="background:#f0f0f0;"><B>Binding Site Location</B></TD><TD align="center" style="background:#f0f0f0;"><B>Bottom Finger</B></TD><TD align="center" style="background:#f0f0f0;"><B>Top Finger</B></TD><TD align="center" style="background:#f0f0f0;"><B>Bottom AA (F3 to F1)</B></TD><TD align="center" style="background:#f0f0f0;"><B>Top AA (F3 to F1)</B></TD></TR><TR><TD> Colorblindness</TD><TD>chrX:153,402,679-153,408,753</TD><TD>256</TD><TD style="background:#92D050">GGC TGA GGC</TD><TD>GTA GCT GGG</TD><TD>ESGHLKR.QREHLTT.#######</TD><TD>QSGTLTR.QRSDLTR.KKDHLHR
</TD></TR><TR><TD> Colorblindness</TD><TD>chrX:153,402,679-153,408,753</TD><TD>2067</TD><TD>GAA GGG GAC</TD><TD style="background:#92D050">GGG GCT CAC</TD><TD>QDGNLGR.RREHLVR.EEANLRR</TD><TD>RTEHLAR.QRSDLTR.#######
</TD></TR><TR><TD> Familial Hypercholesterolemia</TD><TD>chr19:11,175,000-11,195,000</TD><TD>2707</TD><TD style="background:#92D050"> GGC TGG ATG</TD><TD style="background:#92D050"> GGC TGG CTC*</TD><TD>ESKHLTR.RREHLTI.#######</TD><TD>ESKHLTR.RREHLTI.#######
</TD></TR><TR><TD> Pancreatic Cancer</TD><TD>chr7:117,074,084-117,089,556</TD><TD>4423</TD><TD style="background:#92D050"> GCA GAC TGT</TD><TD style="background:#92D050"> GCA GGA AAA</TD><TD>QGNTLTR.DRGNLTR.#######</TD><TD>QDVSLVR.QSAHLKR.#######</TD></TR></TBODY></TABLE><UL><LI>Drier was unable to find a ZF that bound specifically to CTC. Instead he found zinc fingers that bound to CTC and other sequences with equal binding affinity.  
</LI><LI>Note: The green cells are the target sequences that we are aiming for on our chip.
</LI></UL><H3><SPAN class="mw-headline" id="Finalizing_the_non-Zif268_backbones">Finalizing the non-Zif268 backbones</SPAN></H3><P>In addition, we locked down the non-Zif268 backbones that we will be using for the chip.  We have 10 backbones that are more closely related to Zif268, and 10 that are more distantly related:
</P><TABLE><TBODY><TR><TD align="center" colspan="2" style="background:#f0f0f0;"><B>More Closely Related Backbones</B></TD><TD style="background:#f0f0f0; color: #f0f0f0;">     </TD><TD align="center" colspan="2" style="background:#f0f0f0;"><B>More Distantly Related Backbones</B></TD></TR><TR><TD align="center" style="background:#f0f0f0;"><B>Name</B></TD><TD align="center" style="background:#f0f0f0;"><B>Sequence (with helix)</B></TD><TD align="center" style="background:#f0f0f0;"><B>Name</B></TD><TD align="center" style="background:#f0f0f0;"><B>Sequence (with helix)</B></TD></TR><TR><TD> 44GLAS_DROME</TD><TD style="font-family: monospace, sans-serif"> FRCPI---CDRRFSQSSSVTTH-MRTH--</TD><TD>56EGR1_HUMAN</TD><TD style="font-family: monospace, sans-serif"> FAC---DICGRKFARSDERKRHTKIH---
</TD></TR><TR><TD> 38KRUP_DROME</TD><TD style="font-family: monospace, sans-serif"> FTCKI---CSRSFGYKHVLQNH-ERTH--</TD><TD>47MZF1_HUMAN</TD><TD style="font-family: monospace, sans-serif"> FVC---GDCGQGFVRSARLEEHRRVH---
</TD></TR><TR><TD> 124EVI1_HUMAN</TD><TD style="font-family: monospace, sans-serif"> YRC---KYCDRSFSISSNLQRHVRNIH--</TD><TD>23CF2_DROME</TD><TD style="font-family: monospace, sans-serif"> YTC---SYCGKSFTQSNTLKQHTRIH---
</TD></TR><TR><TD> 6HUNB_DROME</TD><TD style="font-family: monospace, sans-serif"> YECK---YCDIFFKDAVLYTIHMGY--H-</TD><TD>19ZEP2_RAT</TD><TD style="font-family: monospace, sans-serif"> YICE---ECGIRCKKPSMLKKHIRTH---
</TD></TR><TR><TD> 16SUHW_DROME</TD><TD style="font-family: monospace, sans-serif"> FPCEQ---CDEKFKTEKQLERH-VKTH--</TD><TD>49SDC1_CAEEL</TD><TD style="font-family: monospace, sans-serif"> VVC---FHCG-TRCHYTLLHDHLDYCH--
</TD></TR><TR><TD> 125CF2_DROME</TD><TD style="font-family: monospace, sans-serif"> YTC---PYCDKRFTQRSALTVHTTKLH--</TD><TD>27SDC1_CAEEL</TD><TD style="font-family: monospace, sans-serif"> LTC---AHCDWSFDNVMKLVRH-RGVH--
</TD></TR><TR><TD> 43EVI1_HUMAN</TD><TD style="font-family: monospace, sans-serif"> FKCHL---CDRCFGQQTNLDRH-LKKH--</TD><TD>130TTKB_DROME</TD><TD style="font-family: monospace, sans-serif"> YRC---KVCSRVYTHISNFCRHYVTSH--
</TD></TR><TR><TD> 118ADR1_YEAST</TD><TD style="font-family: monospace, sans-serif"> YPC---GLCNRCFTRRDLLIRHAQKIH--</TD><TD>80ESCA_DROME</TD><TD style="font-family: monospace, sans-serif"> YQC---PDCQKSYSTFSGLTKH-QQFH--
</TD></TR><TR><TD> 24EVI1_HUMAN</TD><TD style="font-family: monospace, sans-serif"> QECK---ECDQVFPDLQSLEKHMLS--H-</TD><TD>20IKZF1_MOUSE</TD><TD style="font-family: monospace, sans-serif"> HKCG---YCGRSYKQRSSLEEHKERCH--
</TD></TR><TR><TD> 25SUHW_DROME</TD><TD style="font-family: monospace, sans-serif"> MSCKV---CDRVFYRLDNLRSH-LKQH--</TD><TD>127SRYD_DROME</TD><TD style="font-family: monospace, sans-serif"> QECTT---CGKVYNSWYQLQKHISEEH--
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Updated_Chip_Design">Updated Chip Design</SPAN></H3><P>The CODA article produced zinc fingers that bound a GNN or TNN F2 with either a ANN, GNN, or TNN F3. These results lead us to the following distribution of three types of zinc finger backbones (Zif268, similar but not equal to Zif268, and dissimilar to Zif268) across our 6 target DNA sequences. With 55,000 spaces on our chip, each of the 6 target DNA sequences is allotted 9,150 spaces with 100 spaces set aside for control zinc fingers from CODA and OPEN. Note that the values in the table below represent the number of helices inserted into each type of backbone.
</P><TABLE class="wikitable" cellpadding="5"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Disease</B></TD><TD align="center" style="background:#f0f0f0;"><B>Target DNA Finger 2</B></TD><TD align="center" style="background:#f0f0f0;"><B>Target DNA Finger 1</B></TD><TD align="center" style="background:#f0f0f0;"><B>Helices in Zif268 Backbone</B></TD><TD align="center" style="background:#f0f0f0;"><B>Helices in Zif268 Closely-Related Backbones</B></TD><TD align="center" style="background:#f0f0f0;"><B>Helices in Zif268 Distantly-Related Backbones</B></TD></TR><TR><TD> Colorblindness</TD><TD>TNN</TD><TD>GNN</TD><TD>5150</TD><TD>3000</TD><TD>1000
</TD></TR><TR><TD> Colorblindness</TD><TD>GNN</TD><TD>CNN</TD><TD>3050</TD><TD>3050</TD><TD>3050
</TD></TR><TR><TD> Familial Hypercholesterolemia</TD><TD>TNN</TD><TD>ANN</TD><TD>3050</TD><TD>3050</TD><TD>3050
</TD></TR><TR><TD> Familial Hypercholesterolemia</TD><TD>TNN</TD><TD>CNN</TD><TD>3050</TD><TD>3050</TD><TD>3050
</TD></TR><TR><TD> Pancreatic Cancer</TD><TD>GNN</TD><TD>TNN</TD><TD>5150</TD><TD>3000</TD><TD>1000
</TD></TR><TR><TD> Pancreatic Cancer</TD><TD>GNN</TD><TD>ANN</TD><TD>3050</TD><TD>3050</TD><TD>3050
</TD></TR></TBODY></TABLE><P>N.B.: The chip will only be holding our F1 zinc fingers- the F2 and F3 will be on a separate plasmid that we must make ourselves
</P><P><I><B>To Do:</B></I> The distribution of helices to each backbone set/target sequence needs to be finalized. For example, the program can generate a set of helices for the Zif268 backbone to be applied to the colorblindness target sequence, but should the same set or a completely different helix set be applied to the Zif268 backbones for the familial hypercholesterolemia target sequences?
</P><UL><LI>If we want to test the effect of the backbone, would need to keep the helices constant-- but we could do this within a single target, to keep all other variables constant
</LI></UL><H3><SPAN class="mw-headline" id="Finishing_the_generator">Finishing the generator</SPAN></H3><P>We finished and finalized the program that generates zinc finger sequences. The following changes were made today:
</P><UL><LI>We incorporated the data from Persikov's database into out generator. 
</LI><LI>We included the ability to remove duplicate sequences in the output file (with a dictionary). 
</LI><LI>We added pseudocounts for fingers that bind to 'ANN' or 'CNN' targets. Because there is not much information for these targets, the data we do have may be biased. Thus, we want to make sure that amino acids that currently have no probability of occurring are bumped up to a minimum (currently 0.01).
</LI><LI>We placed additional weight on non zif-268 backbones. Formerly, the amino acids for positions 1, 4, and 5 were fixed based on zif-268 data, regardless of the original helix sequences on these backbones. Now information from both zif-268 and the original helix sequence is considered when assigning weights to the amino acids. 
</LI><LI>We started working with Noah's reverse translate program. 
</LI></UL><P>
We tested our generator to ensure that the sequences it was producing appeared to be legitimate.
</P><UL><LI>Jamie looked at the multiple sequence alignments of the fingers generated, so see if the frequencies correlated with what we expected them to be. (?)
</LI><LI>We input a known DNA triplet to see if the program generated sequences known to bind, according to OPEN data. When generating 10000 sequences, about half the known binders for the input triplet were found.
</LI></UL><P>
We created <A href="http://weblogo.berkeley.edu/" class="external text" rel="nofollow">WebLogos</A> to more easily visualize how adding the Persikov data affects the sequences we generate. The size of the letters correspond to the frequency of that amino acid in that position. We decided to incorporate the Persikov data so that our generator incorporates more information when generating sequences. Doing so does not drastically change the sequences generated. 
</P><TABLE><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">WebLogo for the OPEN data.</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">WebLogo for the Persikov data.</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">WebLogo for 10000 sequences generated with our program, when it incorporates only OPEN data.</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">WebLogo for 10000 sequences generated with our program, when it incorporates both OPEN and Persikov data.</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">WebLogo for fingers that bind to ANN according to OPEN data.</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">WebLogo for 10000 sequences generated for an ANN triplet with our program, when it incorporates only OPEN data.</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">WebLogo for 10000 sequences generated for an ANN triplet with our program, when it incorporates both OPEN and Persikov data.</DIV></DIV></DIV></TD></TR></TBODY></TABLE></DIV><DIV id="623" style="display:none"><H2><SPAN class="mw-headline" id="June_23rd_-_Wet_Lab">June 23rd - Wet Lab</SPAN></H2><P><B>Ran gel to determine the results of the PCR products</B></P><UL><LI>Determined the HisB presence in selection strain
</LI><LI>Finalized the presence of hisB through the gel image below
</LI></UL><LI>Determined the success of the pZE21G backbone primers through gel on gradient PCR
</LI><LI>PCR for the backbone failed again, even done through gradient PCR
</LI><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">PCR of HisB locus</DIV></DIV></DIV><P><B>PCR more Kan-ZFB-His3-Ura3</B></P><UL><LI>Two 50 µL reactions
</LI><LI>Doubled the protocol used on the 16th and used HisUraKan_F and ZFBwpHisUra_R primers
</LI><LI>PCR produced very low concentration of Kan-ZFG-His3-Ura3 because melting temperature of primer was too high, so primers stuck to annealing DNA and did not dissociate
</LI><LI>In image below the low concentration of desired product can be seen, along with the high concentration of unused primers
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">PCR of kan-ZFB-wp-his3-ura3</DIV></DIV></DIV><UL><LI>Ran gel of initial overlap PCR product (undiluted, starting without primers) from June 16th, using whole sample
</LI><LI>Performed gel extraction in order to have more Kan-ZFB-His3-Ura3 product for the transformation tomorrow: 8.5ng/µL, 260/280=2.03 
</LI></UL><P><B>Determined success of the selection construct transformation</B></P><UL><LI>Checked the plates all day,and finally came to the conclusion that the transformation did not work
</LI><LI>Discovered that lambda red has a promoter induced by arabinose, not temperature (though the strain is still temperature sensitive). That is why it didn't work--we'll get arabinose and hopefully have a successful recombination.
</LI><LI>Preparing all parts for transformation today and will finish it tomorrow
</LI></UL><P><B>Oligo Design for MAGE</B></P><UL><LI>Designed 90bp long oligos for OZ052 and OZ123 insertion in the ZFB sites in place of Zif268. Reverse complement taken.
</LI></UL><P><B>Miniprep pZE21G plasmid for backbone PCR</B></P><UL><LI>Ran miniprep again for pZE21G plasmid: 6ng/µL, 260/280=2
</LI><LI>Worried that the miniprep didn't work: ran gel also on this miniprep and concluded that DNA was present in the sample
</LI><LI>Gel image seen below
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">pZE21G miniprep</DIV></DIV></DIV><P><B>Ran PCR for pZE21G backbone</B></P><UL><LI>Used same protocol from June 22 in PCR today for pZE21G backbone
</LI></UL><H2><SPAN class="mw-headline" id="June_23rd_-_Bioinformatics">June 23rd - Bioinformatics</SPAN></H2><H3><SPAN class="mw-headline" id="Revising_Target_Sequences">Revising Target Sequences</SPAN></H3><TABLE class="wikitable" border="1" cellpadding="5"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Target DNA</B></TD><TD align="center" style="background:#f0f0f0;"><B>Cystic Fibrosis</B></TD><TD align="center" style="background:#f0f0f0;"><B>Familial Hypercholesterolemia</B></TD><TD align="center" style="background:#f0f0f0;"><B>Retinal Blastoma</B></TD><TD align="center" style="background:#f0f0f0;"><B>p53</B></TD><TD align="center" style="background:#f0f0f0;"><B>Myc</B></TD><TD align="center" style="background:#f0f0f0;"><B>Pancreatic Cancer</B></TD></TR><TR><TD> GNN A</TD><TD><B>Flank 1</B></TD><TD>?
</TD></TR><TR><TD> GNN T</TD><TD><B>Flank 1</B></TD></TR><TR><TD> GNN C</TD><TD>?</TD><TD><B>Flank 2</B></TD></TR><TR><TD> TNN G</TD><TD><B>Flank 2</B></TD><TD>X</TD></TR><TR><TD> TNN C</TD><TD>Flank 3</TD><TD><B>?</B></TD></TR><TR><TD> TNN A</TD><TD>Flank 3</TD><TD><B>?</B></TD></TR></TBODY></TABLE><P>
This is the set of final, final target sequences based on the table above:
</P><TABLE class="wikitable" cellpadding="5"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Disease</B></TD><TD align="center" style="background:#f0f0f0;"><B>Target Range</B></TD><TD align="center" style="background:#f0f0f0;"><B>Binding Site Location</B></TD><TD align="center" style="background:#f0f0f0;"><B>Bottom Finger</B></TD><TD align="center" style="background:#f0f0f0;"><B>Top Finger</B></TD><TD align="center" style="background:#f0f0f0;"><B>Bottom AA (F3 to F1)</B></TD><TD align="center" style="background:#f0f0f0;"><B>Top AA (F3 to F1)</B></TD></TR><TR><TD> Colorblindness</TD><TD>chrX:153,403,001-153,407,000</TD><TD>3627</TD><TD style="background:#92D050">GCT GGC TGG </TD><TD style="background:#92D050"> GCG GTA ATG</TD><TD>EGSGLKR.EAHHLSR.#######</TD><TD>RRDDLTR.QRSSLVR.#######
</TD></TR><TR><TD> Familial Hypercholesterolemia</TD><TD>chr19:11,175,000-11,195,000</TD><TD>14001</TD><TD style="background:#92D050"> GGC TGA GAC</TD><TD style="background:#92D050"> GGA GTC CTG</TD><TD>ESGHLKR.QREHLTT.#######</TD><TD>QTTHLSR.DHSSLKR.#######
</TD></TR><TR><TD> Myc-gene Cancer</TD><TD>chr8:128,938,529-128,941,440</TD><TD>198</TD><TD>GGT GCA GGG</TD><TD style="background:#92D050"> GGC TGA CTC</TD><TD>VDHHLRR.QSTTLKR.RRAHLQN</TD><TD>ESGHLKR.QREHLTT.#######
</TD></TR><TR><TD> Myc-gene Cancer</TD><TD>chr8:128,938,529-128,941,440</TD><TD>981</TD><TD>GGA GAG GGT</TD><TD style="background:#92D050"> GGC TGG AAA</TD><TD>QANHLSR.RQDNLGR.TRQKLET</TD><TD>EKSHLTR.RREHLTI.#######
</TD></TR></TBODY></TABLE><UL><LI>Green cells are our target sequences.</LI></UL><DIV id="624" style="display:none"><H2><SPAN class="mw-headline" id="June_24">June 24</SPAN></H2><UL><LI>Designed primer for testing HisB deletion, reuse His_Internal_R to test the band
</LI></UL><P><B>pZE21G:</B></P><UL><LI>reinoculated culture with 100µL of saturated solution, grew to mid-log, and made glycerol stock
</LI><LI>backbone PCR: ran E gel but no bands--PCR unsuccessful. We may need to use a different backbone for the zinc fingers.
</LI></UL><P><B>Omega and Omega+Zif268:</B></P><UL><LI>these were the only two PCR reactions from 6/22/11 to work
</LI><LI>PCR purified using Qiagen kit: 
</LI><LI>omega: 6.1ng/µL, 260/280=1.83
</LI><LI>omega+Zif268: 11.3 ng/µL, 260/280=1.67
</LI></UL><P><B>Lambda red recombination of selection system:</B></P><UL><LI>reinoculated selection strain+pKD46 with 100µL of saturated solution
</LI><LI>just before mid-log (about 4 hours after inoculation) divided culture in half (1.5mL) and added either 37.5µL or 3.75µL of 20% arabinose solution (to try two different induction levels). Cultures grew for another hour.
</LI><LI>The rest of the procedure was the same as the 6/22/11 attempt but without the 42C water bath.
</LI></UL><H2><SPAN class="mw-headline" id="June_24th_-_Bioinformatics">June 24th - Bioinformatics</SPAN></H2><H3><SPAN class="mw-headline" id="Playing_with_Pseudocounts">Playing with Pseudocounts</SPAN></H3><P>Using CTC because of position 6's reliance on the CNN frequencies, we see what difference values of pseudocounts (if in the frequency table, the frequency of an amino acid is 0, bump it up to the psuedocount: ex. A = 0 becomes A = .015 with a psuedocount of .015) make. Pseudocounts are necessary for data that has small sample size - we could be missing out on working helices because a letter's frequency is 0 when it shouldn't be.
</P><P>Various pseudocount (psu = ) values. Look at the 7th column, which is position 6 in the helix:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">psu = 0</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">psu = .005</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">psu = .008</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">psu = .01</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">psu = .015.</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">psu = .020.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>The variation from E being the top letter to A being top back to E is from a slight adjustment in how we add on psuedocounts: the 'new' way is a more proportional approach. 
</P><P>Notice how psu = 0 gives only the four letters found in our dataset, while psu &gt; 0 adds in other letters, each with a small probability ranging from .5% to 2%. 
</P><P>The question is how much psu to add: less means we weight our (possibly flawed) data of proven zinc fingers more. Higher psu adds more randomness (variation) to our sequences, but some fraction of those sequences will not work.
</P><H3><SPAN class="mw-headline" id="Updated_Closest_Zif268_Fingers">Updated Closest Zif268 Fingers</SPAN></H3><P>We realized that some of our &quot;close non-zif268 fingers&quot; were actually not all that close to Zif268, and so we went into the 88,000 zinc finger database and pulled out zinc fingers surrounding zif268.  In fact, there were many, many, many zinc fingers that had identical sequences to the Zif268 F2 finger, and so we looked at sequences around it.  The tree below shows the new non-zif268 backbones that are actually close to zif268 compared to our old set.  The new set is in gray, the old set is in black.  This gives us a potential seven more backbones to work with.
</P><H3><SPAN class="mw-headline" id="Sequence_Generation">Sequence Generation</SPAN></H3><P>We made some small updates to the sequence generator, based on the frequencies we noticed in the outputs of the tests we ran. 
</P><UL><LI>We decided to only include pseudocounts for position 6 for 'CNN' and 'ANN.' Originally, 'CNN' and 'ANN' were using pseudocounts for all seven positions. However, this introduced a noticeable increase in amino acids, such as tyrosine (Y), that have been shown to occur rarely in zinc fingers (according to our data from OPEN and Persikov). Additionally, because tryosines occured so rarely in the data (11 times total in the open data set), we decided not to give tyrosine a pseudocount. 
</LI><LI>We added the capability to prevent repeat backbone-helix combinations on the chip. That is, we wanted to make sure that the same exact zinc finger was not generated for different triplet inputs. 
</LI></UL><P>
To test the sequence generator, we made two sets of 2000 sequences for GAA, then infographic-d the results. Comparing these with the images for OPEN and OPEN+Persikov shows that our generation follows the major themes of those datasets, but also introduces variation. The two generated sets also vary slightly from each other, which shows the influence of randomness on the generation. 
</P><TABLE><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Round 1 of generating sequences for GAA with the program.</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Round 2 of generating sequences for GAA with the program.</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">GAA sequences from the OPEN dataset.</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">GAA sequences from Persikov and OPEN datasets.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE class="wikitable" border="3" cellpadding="5"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Disease</B></TD><TD align="center" style="background:#f0f0f0;"><B>Target DNA Finger 1</B></TD><TD align="center" style="background:#f0f0f0;"><B>Helices in Zif268 Backbone</B></TD><TD align="center" style="background:#f0f0f0;"><B>Helices in Zif268 Closely-Related Backbones</B></TD><TD align="center" style="background:#f0f0f0;"><B>Helices in Zif268 Distantly-Related Backbones</B></TD></TR><TR><TD> Colorblindness <I>(Bottom)</I></TD><TD>TGG</TD><TD>5150</TD><TD>3000</TD><TD>1000
</TD></TR><TR><TD> Colorblindness <I>(Top)</I></TD><TD>ATG</TD><TD>3050</TD><TD>3050</TD><TD>3050
</TD></TR><TR><TD> Familial Hypercholesterolemia <I>(Bottom)</I></TD><TD>GAC</TD><TD>5150</TD><TD>3000</TD><TD>1000
</TD></TR><TR><TD> Familial Hypercholesterolemia <I>(Top)</I></TD><TD>CTG</TD><TD>3050</TD><TD>3050</TD><TD>3050
</TD></TR><TR><TD> Myc <I>(Top<SUB>198</SUB>)</I></TD><TD>CTC</TD><TD>3050</TD><TD>3050</TD><TD>3050
</TD></TR><TR><TD> Myc <I>(Top<SUB>981</SUB>)</I></TD><TD>AAA</TD><TD>3050</TD><TD>3050</TD><TD>3050
</TD></TR></TBODY></TABLE><P>Table of target sequences and helix distribution across backbones
</P><UL><LI>Distribution: Zif268 : Zif268 similar : Zif 268 dissimilar
</LI><LI>Conservative distribution 56.3 : 32.8 : 10.9
</LI><LI>Riskier distribution 33.3 : 33.3 : 33.3
</LI></UL><P><B>List of Remaining Goals:</B></P><UL><LI>Sort fingers by target
</LI><LI>Pick and assign primer sets
</LI><LI>Reverse translate fingers avoiding type II restriction enzymes and primers
</LI><LI>Append type II restriction enzyme and primer sequences to each finger
</LI><LI>Yay</LI></UL><DIV id="625" style="display:none"><H2><SPAN class="mw-headline" id="June_25th-26th_-_Bioinformatics">June 25th-26th - Bioinformatics</SPAN></H2><P>This is the set of final target sequences with assigned forward and reverse primers (tags for PCR):
</P><TABLE class="wikitable" cellpadding="5"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Disease</B></TD><TD align="center" style="background:#f0f0f0;"><B>Target Sequence</B></TD><TD align="center" style="background:#f0f0f0;"><B>Forward Primer (5'-3' NOT REVERSE COMPLEMENT)</B></TD><TD align="center" style="background:#f0f0f0;"><B>Reverse Primer (5'-3' NOT REVERSE COMPLEMENT)</B></TD></TR><TR><TD> Colorblindness</TD><TD>GCT GGC TGG</TD><TD>ATATAGATGCCGTCCTAGCG</TD><TD>AAGTATCTTTCCTGTGCCCA
</TD></TR><TR><TD> Colorblindness</TD><TD>GCG GTA ATG</TD><TD>CCCTTTAATCAGATGCGTCG</TD><TD>TGGTAGTAATAAGGGCGACC
</TD></TR><TR><TD> Familial Hypercholesterolemia</TD><TD>GGC TGA GAC</TD><TD>TTGGTCATGTGCTTTTCGTT</TD><TD>AGGGGTATCGGATACTCAGA
</TD></TR><TR><TD> Familial Hypercholesterolemia</TD><TD>GGA GTC CTG</TD><TD>GGGTGGGTAAATGGTAATGC</TD><TD>ATCGATTCCCCGGATATAGC
</TD></TR><TR><TD> Myc-gene Cancer</TD><TD>GGC TGA CTC</TD><TD>TCCGACGGGGAGTATATACT</TD><TD>TACTAACTGCTTCAGGCCAA
</TD></TR><TR><TD> Myc-gene Cancer</TD><TD>GGC TGG AAA</TD><TD>CATGTTTAGGAACGCTACCG</TD><TD>AATAATCTCCGTTCCCTCCC
</TD></TR></TBODY></TABLE>
Additionally, primer tags <B>(forward: GTACATGAAACGATGGACGG, reverse:CTGGTATAGTCTCCTCAGCG)</B> will be assigned to the 100 control sequences.</DIV><DIV id="627" style="display:none"><H2><SPAN class="mw-headline" id="June_27">June 27 </SPAN></H2><H3><SPAN class="mw-headline" id="Wet_lab_3">Wet lab</SPAN></H3><P><B>Sequencing PyrF, rpoZ loci:</B></P><UL><LI>We will sequence these genes in the selection strain just to make sure they are knocked out, especially since it appears HisB is not.
</LI><LI>Picked a colony off ∆HisB∆PyrF∆rpoZ plate (6/21) and grew in 150µL LB plus tet in a 96 well plate for about 2 hrs at 37˚C
</LI><LI>diluted 1 in 20 and used 1µL as template in PCR with KAPA mastermix (see protocols for reagent amounts and parameters)
</LI><LI>annealing temp 65˚C, elongation time 1:15
</LI></UL><OL><LI>PyrF_F, PyrF_R primers
</LI><LI>PyrF_F, PyrF_internalR
</LI><LI>rpoZ_F, rpoZ_R
</LI><LI>rpoZ_F, rpoZ_internalR
</LI><LI>rpoZ_R, zeocin_R
</LI></OL><UL><LI>Run on E Gel to check PCR worked: bands are at the same sizes as the original genotyping gel.
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">PCR of kan-ZFB-wp-his3-ura3</DIV></DIV></DIV><UL><LI>Tomorrow we will send samples to Genewiz for sequencing
</LI></UL><P><B>Lambda Red recombination:</B></P><UL><LI>The plates made from the recombination (6/24) did have colonies, but they were very small and took a long time to grow, and so they may not actually have the kan-ZFB insert. We will have to PCR the locus to see.
</LI><LI>Chose 8 colonies from each plate and grew at 30˚C in 150µL LB plus kan in a 96 well plate
</LI><LI>When our primers arrive, we will PCR the locus to check for the insert.
</LI></UL><P><B>Selection system media:</B></P><H2><SPAN class="mw-headline" id="June_27th_-_Bioinformatics">June 27th - Bioinformatics</SPAN></H2><H3><SPAN class="mw-headline" id="To_Do_for_Today">To Do for Today</SPAN></H3><OL><LI> 100 sequences (and control), 2 each with the same F3 and F2, but different F1, from our test sequences [zif268, OZ123, OZ052, CoDA]✓ 
</LI><LI>Type II nuclease cut site sequences- put the binding sites into our oligos ✓ 
</LI><LI>Final backbones with helices ✓ 
</LI><LI>Programming stuff- Check to make sure there are no cut sites or primers in any of our backbone/helices combinations; check translation order (translates F1→F3)✓
</LI></OL><H3><SPAN class="mw-headline" id="100_Control_Sequences">100 Control Sequences</SPAN></H3><UL><LI> See our <A href="https://static.igem.org/mediawiki/2011/5/5d/HARVPositive_Control_Sequences_PostMacro.pdf" class="external text" rel="nofollow">Positive Control Sequences</A>, updated June 28th
</LI><LI> Selected known binding zinc fingers from the CODA table that bind sequences similar to our target sequences 
</LI><LI> All control helices from CODA were inserted into Zif268 F2 backbones and have been assigned a seventh primer tag separate from the tags given to the 6 target sequences.
</LI></UL><H3><SPAN class="mw-headline" id="Updated_Target_Sequences">Updated Target Sequences</SPAN></H3><P>One of our sequences from before was bad because the F3/F2 combo did not appear in the CODA table... faulty checking, my bad :(
</P><P>Here is the newest table of target sequences:
</P><TABLE class="wikitable" cellpadding="5"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Disease</B></TD><TD align="center" style="background:#f0f0f0;"><B>Target Range</B></TD><TD align="center" style="background:#f0f0f0;"><B>Binding Site Location</B></TD><TD align="center" style="background:#f0f0f0;"><B>Bottom Finger</B></TD><TD align="center" style="background:#f0f0f0;"><B>Top Finger</B></TD><TD align="center" style="background:#f0f0f0;"><B>Bottom AA (F3 to F1)</B></TD><TD align="center" style="background:#f0f0f0;"><B>Top AA (F3 to F1)</B></TD></TR><TR><TD> Colorblindness</TD><TD>chrX:153,403,001-153,407,000</TD><TD>3666</TD><TD style="background:#92D050">GTG GGA TGG </TD><TD style="background:#92D050"> GAA GGG ACC</TD><TD>RNTALQH.QSAHLKR.#######</TD><TD>QDGNLGR.RREHLVR.#######
</TD></TR><TR><TD> Familial Hypercholesterolemia</TD><TD>chr19:11,175,000-11,195,000</TD><TD>14001</TD><TD style="background:#92D050"> GGC TGA GAC</TD><TD style="background:#92D050"> GGA GTC CTG</TD><TD>ESGHLKR.QREHLTT.#######</TD><TD>QTTHLSR.DHSSLKR.#######
</TD></TR><TR><TD> Myc-gene Cancer</TD><TD>chr8:128,938,529-128,941,440</TD><TD>198</TD><TD>GGT GCA GGG</TD><TD style="background:#92D050"> GGC TGA CTC</TD><TD>VDHHLRR.QSTTLKR.RRAHLQN</TD><TD>ESGHLKR.QREHLTT.#######
</TD></TR><TR><TD> Myc-gene Cancer</TD><TD>chr8:128,938,529-128,941,440</TD><TD>981</TD><TD>GGA GAG GGT</TD><TD style="background:#92D050"> GGC TGG AAA</TD><TD>QANHLSR.RQDNLGR.TRQKLET</TD><TD>EKSHLTR.RREHLTI.#######
</TD></TR></TBODY></TABLE><UL><LI>Green cells are our target sequences.
</LI></UL><H3><SPAN class="mw-headline" id="Cut_Site_Design">Cut Site Design</SPAN></H3><UL><LI>See our <A href="https://2011.igem.org/Team:Harvard/Cut_Site_Design" class="external text" rel="nofollow">Cut Site Design</A> page
</LI><LI>We left in one proline (P) between the linker and the starting FCQ... of finger 2, but as this proline is the last AA of the OPEN linker (TGEKP) and occurs before the beta sheet in every zinc finger in Zif268 (see zif268's sequence on its <A href="http://www.pdb.org/pdb/explore/remediatedSequence.do?structureId=1AAY" class="external text" rel="nofollow">PDB page</A>)
</LI><LI>This configuration also allows the library to be used at any finger position because proline ends the OPEN linker.
</LI></UL><H3><SPAN class="mw-headline" id="Updates_on_the_program">Updates on the program</SPAN></H3><P>The program appears to run extremely slowly because of the computationally intensive step of checking the reverse translated sequences 
</P><UL><LI>In addition to checking for the primers and cutsites, we also have to check for 'GGGGGG' because it can lead to undesirable structures forming. In addition, we have to check for the reverse complements for all these undesirable sequences.
</LI><LI>We decided on a similarity of 0.8 as the maximum acceptable similarity between the sequence the primer bind to and any other part of the generated sequence. If the sequences are too similar, the primer might mishybridize. We originally had a similarity threshold of 0.6 but that made the program run too slowly, so we decided on a <B>threshold of 0.8</B>.</LI></UL><H2><SPAN class="mw-headline" id="June_28th">June 28th</SPAN></H2><P><B>Sequencing:</B></P><UL><LI>the following samples from 6/27 were sent to Genewiz for sequencing:
</LI><LI>PyrF F, R (one sample with PyrF_F, one with PyrF_R)
</LI><LI>rpoz F, R (one sample with rpoz_F, one with rpoz_R)
</LI></UL><P><B>Lambda red results:</B></P><UL><LI>the colonies on the plates did not look promising, and the ones we chose and grew up in LB+kan did not actually grow.  Just to be certain, we choose 18 more colonies: 6 from 37.5µL arabinose 100µL plated, 6 from 3.75µL arabinose 100µL plated, and 6 from 37.5µL arabinose 1.5mL plated.  Three from each plate were grown in plain LB and three with kan. We will let it grow in 30˚C, overnight if necessary, and hopefully see bacteria for PCR.
</LI><LI>Assuming this does not work, we prepared more ∆HisB∆PyrF∆rpoZ+pKD46 in two ways: we put 3 colonies in LB+amp from the 6/16 transformation plate, and we streaked a new amp plate from the glycerol stock
</LI><LI>Another possibility is that something is wrong with our lambda red. We designed primers to verify that the pKD46 plasmid is really in the cells.
</LI></UL><P><B>Kan-ZFB-wp-his3-ura3 construct:</B></P><UL><LI>Our last few PCR purifications have given us very low yields, and consequently we have had to use large amounts of our DNA (and the large amounts of buffer salts may also be why our lambda red recombinations have failed). When we tried to amplify our current DNA using the hisura-kan_F and ZFB-wp-hisura_R primers and the Phusion mastermix, it did not work (see 6/23).  We will try to gain more product in two ways:
</LI></UL><P>1) Repeat 6/23 PCR but use KAPA mastermix
</P><UL><LI>the KAPA mix may work better than the Phusion.
</LI><LI>Used KAPA protocol with 1µL of kan-ZFB overlap as template, hisura-kan_F and ZFB-wp-hisura_R primers, 65˚C annealing temp, 90 sec elongation time
</LI><LI>made 2 reactions
</LI></UL><P>2) Repeat overlap extension PCR (see 6/16) with KAPA mastermix
</P><UL><LI>used KAPA protocol with 1µL kan cassette and 1µL of ZFB-wp-hisura, 65˚C annealing temp, 90 sec elongation
</LI><LI>10 cycles without primers; hisura-kan_F and ZFB-wp-hisura_R primers added; 15 more cycles
</LI><LI>made 6 reactions
</LI></UL><UL><LI>E gel to check reactions worked: all 6 overlap PCRs successful, but not the other two reactions.
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">kan-ZFB-wp-hisura construct 6/28/11</DIV></DIV></DIV><UL><LI>combined samples 1-3 and 4-6 and ran on 1% agarose gel for extraction
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">kan-ZFB-wp-hisura construct for gel extraction 6/28/11</DIV></DIV></DIV><UL><LI>used Qiagen gel extraction kit and instructions with the following modifications:
</LI><LI>gel bands were dissolved in 500µL of buffer QG regardless of the gel volume
</LI><LI>gel heated at 50C for 20 min (to make up for reduced amount of buffer QG)
</LI><LI>after melting, 10µL of NaOAC (3M) were added to adjust the pH
</LI><LI>DNA from samples 1-3 were eluted in 20µL of ddH2O; DNA from samples 4-6 were eluted in 20µL of buffer EB
</LI><LI>water sample: 273.4 ng/µL, 260/280=1.92
</LI><LI>EB sample: 136.9 ng/µL, 260/280=2.38
</LI></UL><H2><SPAN class="mw-headline" id="June_28th_-_Bioinformatics">June 28th - Bioinformatics</SPAN></H2><P><FONT color="red"><I><B>Attention all Harvard iGEM-ers!!!</B></I></FONT><FONT color="blue"> According to the <A href="https://2011.igem.org/Main_Page" class="external text" rel="nofollow">iGEM Main Page</A>, our preliminary project descriptions and safety proposals are due on</FONT><FONT color="red"><I><B>July 15</B></I></FONT>. <FONT color="blue"> Please see the aforementioned link so we can get this done ASAP- we don't want to miss any deadlines and have all our hard work wasted!</FONT></P><UL><LI>Finalized our <A href="https://static.igem.org/mediawiki/2011/5/5d/HARVPositive_Control_Sequences_PostMacro.pdf" class="external text" rel="nofollow">Positive Control Sequences</A>, using Justin's macro to insert the F1 helices into the appropriate zif268 F2 backbone
</LI></UL><UL><LI>Length of chip oligos: 131-140bp (based on <A href="/Cut_Site_Design" title="Cut Site Design">Cut Site Design</A>)
</LI><LI>Primers: 20bp (x2= 40bp)
</LI><LI>zif268 F2 backbone + helix= 23aa (x3=69bp; some fingers ~3aa longer)
</LI><LI>Some alternate backbones are longer than zif268 F2 backbone
</LI><LI>Type II binding/cut sites= 11bp on each side (22bp total)
</LI><LI>Standard legnth: 40 + 69 + 22 = 131bp
</LI></UL><UL><LI>Use WebLogos as a final visual check of our final generated sequences
</LI></UL><H3><SPAN class="mw-headline" id="Plasmid_and_Oligo_Design_Schematics">Plasmid and Oligo Design Schematics</SPAN></H3><TABLE><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Oligo Design</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Expression Plasmid Design</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Chip-Based_Sequence_Design_Schematic">Chip-Based Sequence Design Schematic</SPAN></H3><TABLE><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Chip-based process for sequence design, taken from Kosuri, et al. 2010 model of scalable gene synthesis <CITE>Kosuri2010</CITE></DIV></DIV></DIV></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="References_2">References</SPAN></H4><P>&lt;biblio&gt;</P><OL><LI>Kosuri2010 pmid=21113165
</LI></OL><P>&lt;/biblio&gt;</P><H3><SPAN class="mw-headline" id="Harvard_Logo">Harvard Logo</SPAN></H3><H3><SPAN class="mw-headline" id="Running_the_Generator.21">Running the Generator!</SPAN></H3><P><A href="/File:HARVFasta_total.csv" title="File:HARVFasta total.csv">File:HARVFasta total.csv</A> NOTE: LATER GENERATED NEW SEQUENCES. NOT UP TO DATE.
</P><H4><SPAN class="mw-headline" id="Generated_Final_Chip_Sequences">Generated Final Chip Sequences</SPAN></H4><UL><LI>We ran the generator once earlier this afternoon, but had to re-run it again due to a typo in the cut sites and the number of sequences we desired for each backbone. Luckily, we caught these errors, and after checking the program once again, we ran it a final time this afternoon.
</LI><LI>It took about 45 minutes for the program to generate and reverse translate the 54900 sequences. 
</LI><LI>During this time, we created a function that will re-translate the sequences that the generator output. It compares the original helix with the re-translated helix to make sure that our reverse-translate works properly.
</LI><LI>This step went smoothly, and we verified that the sequences were reverse-translated properly.
</LI></UL><LI>To make sure that the distributions generated were as expected, we made WebLogos of the helices generated(see below).
</LI><LI>The output file (in the Dropbox: iGem &gt; chip &gt; final chip.csv) originally had the following headers: 'Target', 'Backbone #', 'Helix Sequence', 'Backbone Sequence', 'Nucleotide Sequence of Zinc Finger'
</LI><LI>We wanted to convert this information into FASTA format.
</LI><LI>We wrote a function that converted our original file into fasta format (in the Dropbox: iGem &gt; chip &gt; fasta.csv)
</LI><LI>The file FASTA_total (also linked above) contains the FASTA for all 50000 sequences (including the 100 controls).
</LI><LI>For those curious, the FASTA format just a format that looks like this:
</LI><PRE>&gt;Header (For us the header is: Target, Backbone #, Helix Sequence, Backbone Sequence)(The header for the controls are: Index Number, 'control')
 Sequence (In our case, the nucleotide sequence of the zinc fingers)
</PRE><H4><SPAN class="mw-headline" id="Generated_WebLogos_for_Final_Chip">Generated WebLogos for Final Chip</SPAN></H4><TABLE><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">AAA</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">ACC</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">CTC</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">CTG</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">GAC</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">TGG</DIV></DIV></DIV></TD></TR></TBODY></TABLE><UL><LI>FASTA-Formatted Chip Data:
</LI></UL><DL><DD>&gt;NNN(Target Triplet)  BB#  Helix Seq.
</DD><DD>Nucleotide seq. of ZF
</DD></DL><H3><SPAN class="mw-headline" id="Bioinformatics_Candids">Bioinformatics Candids</SPAN></H3><TABLE><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">zif268 sequence by memory. You know you've stared at too many zif268 sequences when...</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P><A href="/File:HARVPrimer_Index_iGEM_2011.xls" title="File:HARVPrimer Index iGEM 2011.xls">File:HARVPrimer Index iGEM 2011.xls</A></P><H3><SPAN class="mw-headline" id="Design_of_Plate_Practice_Sequences">Design of Plate Practice Sequences</SPAN></H3><P>While we wait for the chip to come in, we have a number of techniques and protocols that we can practice on beforehand, so that when the chip comes we'll be ready to go to use what they give us.  We will be practicing the following techniques:
</P><UL><LI>Cutting ZF1 out of our oligos
</LI><LI>Inserting ZF1 into the expression plasmid in between the omega subunit and the linker before F2
</LI><LI>Verifying that combination of our F1 from the oligo with the plasmid produces a viable, functional ZF
</LI><LI>Amplifying subpools of oligos for testing
</LI><LI>Inserting the expression plasmids into the E. coli containing our selection genome
</LI><LI>Verifying that our ZF-binding site/GFP expression paradigm works
</LI></UL><P>To this end, we will be ordering a 96-well plate from IDT containing oligos that will simulate the entire tube of oligos that we will receive from Agilent in four weeks.  These oligos will consist of the following:
</P><UL><LI>6 positive controls (we know which DNA sequences these bind to)
</LI><LI>3 of them being the F1 fingers of Zif268, OZ052, and OZ123
</LI><LI>3 of them being ZF F1s derived from CODA.
</LI></UL><LI>90 generated sequences, picked from a subset of the chip
</LI><LI>These are picked evenly across the 9,150 sequences generated on the cihp for the TGG triplet F1 target from the colorblindness &quot;bottom finger&quot; target, GTG GGA TGG.  This particular target was chosen because the F2/F1 is a GNNTNN combo, which might be more likely to get hits from our chip generation sequences.
</LI><P>
The primer tag sequences for the 90 generated sequence subset will be the same as they are on the chip (for the sake of explanation, we will refer to them now as P1F and P1R in this paragraph).  The positive controls will be flanked immediately by the same primers as the generated subset so that we can amplify everything as one pool altogether should we need to (so this will be P1F and P1R).  However, we will also put an additional set of primers outside of the P1F/P1R primers for the positive controls so that we can specifically amplify the positive control subpool, should we want to.  These primers will be the same as the primers for the positive control on the chip (which will be called P2F and P2R here).
</P><P>To recap, on the chip we will have the following oligos :
</P><PRE>
45750 other oligos for the 5 other target sequences
Oligos (TGG set, 9150 total):   | P1F | type II binding site | generated F1 | type II binding site | P1R |
Oligos (+ control, 100 total):  | P2F | type II binding site |  control F1  | type II binding site | P2R |
</PRE><P>In our test pool of 96 sequences, we will have two types of oligos (note the two pairs of primers around the positive controls):
</P><PRE>
Oligo (TGG set, 90 total):          | P1F | type II binding site | generated F1 | type II binding site | P1R |
Oligo (+ control, 6 total):   | P2F | P1F | type II binding site | generated F1 | type II binding site | P1R | P2R |
</PRE><P>Once we get our test sequences back from IDT, they will come in a 96-well plate with one oligo in each plate.  We should make a mixture using some of each well in order to create a tube that contains all 96 sequences.  This will simulate the tube that we will receive from Agilent, except instead of 55,000 sequences we will have 96 sequences only in this tube.  From here, we can practice using this as a library.
</P><P>We can pretend that this tube is just 96 generated sequences on the chip, treating the positive controls as if they were also generated sequences (we only include them in the 96 to ensure that we will indeed get a &quot;hit&quot; from this practice screening).  Thus, we can just use the P1F/P1R primer set to amplify all of them in order to use them for the subsequent steps.
</P><P>These subsequent steps will be those that were outlined above, namely cutting out the F1 sequence from each oligo, ligating this F1 into our expression plasmid, putting the expression plasmid into our selection strain, observing colonies which get infused with ZFs that bind to our target site (the &quot;hits&quot;), and sequencing the colonies that get hits to determine which ZF they are expressing.
</P>
We will be repeating these exact same steps once we get the chip, so if we can perfect our protocols with these practice sequences, we should be golden when the chip comes in.</DIV><DIV id="629" style="display:none"><H2><SPAN class="mw-headline" id="June_29th">June 29th</SPAN></H2><P>Our first day with everyone in the wet lab!
</P><P><B>PyrF and rpoZ sequencing:</B></P><UL><LI>For some reason, Genewiz said that sequencing failed due to &quot;no priming.&quot; We will redo the PCR and send the products in again today.
</LI></UL><P><B>Lambda red recombination and MAGE:</B></P><UL><LI>The cultures made from the kan plates from our earlier attempt at lambda red did (in some cases) grow, including one colony in kan from 3.75µL arabinose, 100µL plated.
</LI><LI>PCR of liquid culture: used 1µL of culture diluted 1:20 as well as saturated culture of ∆hisB∆pyrF∆rpoZ+pKD46 (diluted 1:20) as a 1529620 locus wild-type control
</LI><LI>primers (from Vatsan): 1529481-f, 1529806-r
</LI></UL><LI>KAPA mastermix and procedure with 56˚C annealing and 90 seconds elongation
</LI><LI>E Gel of product: both wild-type and the sample hopefully containing the insert had the same short band of around 350bp--the recombination was unsuccessful
</LI><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">kan-ZFB insertion into the 1529620 locus 6/29/11</DIV></DIV></DIV><UL><LI>Used overnight saturated culture to reinoculate; once close to mid-log, culture split into 2 1.5mL amounts and 37.5µL arabinose added to each
</LI><LI>same procedure as previously described, but with about 300ng kan-ZFB construct and a 2.5µM final concentration of HisBNuke3 (12.5µL of 10µM stock)
</LI><LI>electroporate with 1.8 kV, about 5
</LI><LI>recover 3 hrs
</LI><LI>kan-ZFB insertion colonies plated on kan, MAGE on amp
</LI></UL><H3><SPAN class="mw-headline" id="PCR_Preparation">PCR Preparation</SPAN></H3><UL><LI>Lambda Red- Selection strain glycerol stock: 1/100 dilution, 2 uL stock with 198 uL ddH<SUB>2</SUB>O
</LI><LI>Spec (Colony)- Touch 1 colony with pipet tip, add and mix with pipet in 20 uL ddH<SUB>2</SUB>O, then vortex
</LI></UL><H3><SPAN class="mw-headline" id="PCRs">PCRs</SPAN></H3><OL><LI>PKD46 (Lambda Red)
</LI></OL><UL><LI>Kapa Mix 2x- 12.5 uL
</LI><LI>Primer_F- 0.75 uL
</LI><LI>Primer_R- 0.75 uL
</LI><LI>Template 1 uL
</LI><LI>ddH<SUB>2</SUB>O- 10 uL
</LI><LI><I>(25 uL total)</I></LI></UL><OL><LI>Spec (Colony)
</LI></OL><UL><LI>Kapa Mix 2x- 12.5 uL
</LI><LI>Primer_F- 0.75 uL
</LI><LI>Primer_R- 0.75 uL
</LI><LI>Template 1 uL
</LI><LI>ddH<SUB>2</SUB>O- 10 uL
</LI><LI><I>(25 uL total)</I></LI></UL><OL><LI>Spec (Miniprep)
</LI></OL><UL><LI>Kapa Mix 2x- 12.5 uL
</LI><LI>Primer_F- 0.75 uL
</LI><LI>Primer_R- 0.75 uL
</LI><LI>Template 2 uL
</LI><LI>ddH<SUB>2</SUB>O- 9 uL
</LI><LI><I>(25 uL total)</I></LI></UL><H3><SPAN class="mw-headline" id="Expression_Plasmid_Design_in_silico">Expression Plasmid Design in silico</SPAN></H3><P>Today, we designed our expression plasmids in SeqBuilder.  This included plasmids for our 6 target sequences, and 3 positive controls (9 in total).  These positive controls were the following:
</P><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Index</B></TD><TD align="center" style="background:#f0f0f0;"><B>Nucleotide Sequence (5\'-3\')</B></TD><TD align="center" style="background:#f0f0f0;"><B>Helices (F3 to F1)</B></TD><TD align="center" style="background:#f0f0f0;"><B>Notes</B></TD></TR><TR><TD> 16</TD><TD>GAA GGG AAC</TD><TD>QDGNLGR RREHLVR HRTNLIA</TD><TD>Very similar to one of our target sequences (CB top), which is GAA GGG ACC
</TD></TR><TR><TD> 55</TD><TD>GGA GTG GTG</TD><TD>QTTHLSR DHSSLKR RNFILQR</TD><TD>Very similar to a target sequence (FH top), which is GGA GTG CTG
</TD></TR><TR><TD> 77</TD><TD>TGT GAA TAG</TD><TD>RRRNLQI QQTNLTR QPHGLTA</TD><TD>Out of ze air
</TD></TR></TBODY></TABLE><P>Each of our expression plasmids contained:
</P><UL><LI>Omega subunit
</LI><LI>Omega/F1 linker (taken from paper that Dan and Noah emailed us), [<A href="http://nar.oxfordjournals.org/content/36/8/2547.short" class="external autonumber" rel="nofollow">[18]</A>]
</LI><LI>type II binding sites
</LI><LI>gap between type II binding sites that contains XbaI restriction enzyme site (which is not present anywhere else in the entire expression plasmid)
</LI><LI>F1/F2 TGEKP linker
</LI><LI>F2 for a specific target sequence
</LI><LI>F2/F3 TGEKP linker
</LI><LI>F3 for a specific target sequence
</LI><LI>TAA stop codon immediately after F3
</LI></UL><P>All of this is on our spec-resistance-containing plasmid.  The above construct replaced the GFP which was present previously on this plasmid.
</P>
Tomorrow we will begin design of our primers from these SeqBuilder files.</DIV><DIV id="630" style="display:none"><H2><SPAN class="mw-headline" id="June_30th">June 30th</SPAN></H2><P><B>Lambda Red, Backbone, and Sequencing PCR</B></P><UL><LI>Gel run on the presence of a Lambda Red protein in the pKD46 plasmid showed that it is indeed present, so our recombination failures have not been due to an incorrect plasmid.
</LI><LI>Gel run on the backbone of pZE21G plasmid was success and took us one step closer to obtaining all parts necessary for the three part assembly
</LI><LI>Gel run on the pyrF and rpoZ was success
</LI><LI>Therefore we sent the PCR products and primers to GENEWIZ for sequencing again
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">pKD46, pZE21G, and PyrF and rpoZ loci 6/30/11</DIV></DIV></DIV><P><B>pZE21G backbone:</B></P><UL><LI>Since last night's PCR was successful, we will redo it with a few protocol adaptations to get a cleaner product and to increase our yield when we purify
</LI><LI>KAPA mastermix and protocol: primers HindIII-F and KpnI-R
</LI><LI>template: pZE21G miniprepped plasmid, 1µL
</LI><LI>2 min elongation time, 30 cycles
</LI><LI>2 samples at 55˚C annealing, 2 samples at 60˚C
</LI></UL><P><B>Lambda red and MAGE:</B></P><UL><LI>Yesterday's prep produced tiny colonies on the MAGE plates and (so far) none on the kan-ZFB plates. Just in case it didn't work, we will redo the lambda red using even more DNA and perform a second round of MAGE using culture from yesterday that was not plated.
</LI><LI>same procedure, but with the following changes:
</LI><LI>5µL kan-ZFB (about 1 mg)
</LI><LI>recover 3 hrs
</LI><LI>kan-ZFB: plate 100µL and 2 mL on kan plates
</LI><LI>MAGE: plate 1µL and 10µL on amp plates
</LI></UL><LI>To see if the colonies on the MAGE plate knocked out HisB, we chose 24 colonies, resuspended them in water, and put half the cells in LB (complete media) and half in NM media (does not have histidine). 96 well plate, 150µL media, grown overnight at 30˚C.
</LI><H3><SPAN class="mw-headline" id="ZF_Expression_Plasmid_Ultramer_and_Primer_Design">ZF Expression Plasmid Ultramer and Primer Design</SPAN></H3><P>Today, we designed primers ZF_073 through ZF_085 as listed in the iGEM Primer Index spreadsheet.  These were basically two sets of primers: the primers to clone out the omega subunit and linker, and the ultramers that would construct the last part of the linker along with the type II binding sites and F2/F3 fingers.  One should refer to the primer list for the sequences.
</P><P>Note: the annealing sequence for the ultramer overlap contained a 72 degree melting temp hairpin.  To get around this, I changed one of the codons in the F2 backbone.  The F2 backbone begins with &quot;FQCRIC&quot;, and so I changed the codon for the arginine (R) from CGC to CGT, which resolved the hairpin problem.
</P><H2><SPAN class="mw-headline" id="June_30th_-_Bioinformatics">June 30th - Bioinformatics</SPAN></H2><H3><SPAN class="mw-headline" id="Updated_Primer_list_and_FASTA_formatting">Updated Primer list and FASTA formatting</SPAN></H3><P>We ran into a small hiccup, when we were informed that we had forgotten to reverse translate the reverse primer sequences that were being appended to the generated sequence. This is because the primer sequences we were given were the sequences for the actual primers, rather than sequences to which the primers would bind. Luckily, we caught this error! We did have the re-run the generator because we had to make sure that our generated sequences did not contain the new primers. 
</P><UL><LI>Here is the updated primer list:
</LI></UL><P>This is the set of final target sequences with assigned forward and reverse primers (tags for PCR):
</P><TABLE class="wikitable" cellpadding="5"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Disease</B></TD><TD align="center" style="background:#f0f0f0;"><B>Target Sequence</B></TD><TD align="center" style="background:#f0f0f0;"><B>Forward Primer (5'-3' NOT REVERSE COMPLEMENT)</B></TD><TD align="center" style="background:#f0f0f0;"><B>Reverse Primer (5'-3' REVERSE COMPLEMENT)</B></TD></TR><TR><TD> Colorblindness</TD><TD>GCT GGC TGG</TD><TD>ATATAGATGCCGTCCTAGCG</TD><TD>TGGGCACAGGAAAGATACTT
</TD></TR><TR><TD> Colorblindness</TD><TD>GCG GTA ACC</TD><TD>CCCTTTAATCAGATGCGTCG</TD><TD>GGTCGCCCTTATTACTACCA
</TD></TR><TR><TD> Familial Hypercholesterolemia</TD><TD>GGC TGA GAC</TD><TD>TTGGTCATGTGCTTTTCGTT</TD><TD>TCTGAGTATCCGATACCCCT
</TD></TR><TR><TD> Familial Hypercholesterolemia</TD><TD>GGA GTC CTG</TD><TD>GGGTGGGTAAATGGTAATGC</TD><TD>GCTATATCCGGGGAATCGAT
</TD></TR><TR><TD> Myc-gene Cancer</TD><TD>GGC TGA CTC</TD><TD>TCCGACGGGGAGTATATACT</TD><TD>TTGGCCTGAAGCAGTTAGTA
</TD></TR><TR><TD> Myc-gene Cancer</TD><TD>GGC TGG AAA</TD><TD>CATGTTTAGGAACGCTACCG</TD><TD>GGGAGGGAACGGAGATTATT
</TD></TR><TR><TD> Controls </TD><TD> n/a </TD><TD>GTACATGAAACGATGGACGG</TD><TD>CGCTGAGGAGACTATACCAG
</TD></TR></TBODY></TABLE><P>There was also a small error in the FASTA formatting. There are not supposed to be any spaces in the header, so the spaces were replaced with underscores. 
</P><UL><LI>Example:
</LI></UL><PRE>&gt;1_control
</PRE>
 GTACATGAAACGATGGACGGGGTCTCAGCCATTCCAATGTCGTATCTGTATGCGTAATTTTTCACGCAAACACCATTTGGGTCGTCATATCCGTACGCACACGGTGAGACCCGCTGAGGAGACTATACCAG</DIV><DIV id="701" style="display:none"><H2><SPAN class="mw-headline" id="July_1st">July 1st</SPAN></H2><P><B>HisBNuke3 MAGE results:</B></P><UL><LI>All the MAGE round 1 colonies grew overnight in the LB but not the NM medium: either they all had the HisB gene knocked out (unlikely) or there is something wrong with the NM medium.  As a control we grew (∆HisB)∆pyrF∆rpoZ+pKD46 in NM, but nothing grew, implying that the problem is with the NM medium.
</LI><LI>There were small colonies on the MAGE round 2 amp plates, so we chose 12 from each plate and grew them in 100µL of SOC medium.
</LI><LI>PCR of the colonies for sequencing:
</LI><LI>KAPA mastermix and protocols, with His3_F and His3_internalR primers, 2µL culture as template, 65˚C annealing, 15 sec elongation, 25 cycles
</LI><LI>ran on gel: band around 500-600, but very faint (probably because we did not let the culture grow up enough)
</LI><LI>sent to Genewiz to be sequenced with His3_F primer
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">HisBNuke3 MAGE round 2 samples for sequencing 7/1/11</DIV></DIV></DIV><P><B>Lambda Red recombination results:</B></P><UL><LI>The plates from 6/30 had an almost crystalline formation on them, which may or may not be bacteria. The 6/29 plates, which we had left at 30˚C, did have colonies but they may be contaminants.  We grew up 12 colonies from each 6/29 plate and 6 &quot;colonies&quot; from the 2mL 6/30 plate in SOC medium with kanamycin, and next week we will PCR the 1529620 locus to check.
</LI></UL><P><B>pZE21G spec backbone:</B></P><UL><LI>PCR of backbone was successful (see E gel below). Gel extracted with Qiagen kit with the following modifications to the kit instructions:
</LI><LI>2 gel volumes of buffer QG, heated 10-15 min
</LI><LI>after heating added 10µL 3M NaOAc 
</LI><LI>eluted in 30µL ddH2O
</LI><LI>157.4ng/µL 260/280=1.9; 129.2ng/µL 260/280=1.88
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">pZE21G backbone 7/1/11</DIV></DIV></DIV><P><B>Selection system plates:</B></P><UL><LI>Made 3-AT and 5-FOA plates following 2006 Meng Nature Protocol (see <A href="https://2011.igem.org/Team:Harvard/Protocols" class="external text" rel="nofollow">protocols</A> for details). 5-FOA plates have a blue stripe, 3-AT plates (0.5 mL 3-AT) have a green stripe.</LI></UL><DIV id="704" style="display:none"><H2><SPAN class="mw-headline" id="July_4th">July 4th</SPAN></H2>
FIREWORKS</DIV><DIV id="705" style="display:none"><H2><SPAN class="mw-headline" id="July_5th">July 5th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_Assembly">Team ZF Assembly</SPAN></H3><UL><LI>Assemble and confirm zif268 plasmid
</LI><LI>Lac-ω-zif268 with spec → Lac-zif-ω with cpic
</LI><LI>ZF: 2-3 ultramers
</LI><LI>PCR out ω subunit
</LI></UL><H3><SPAN class="mw-headline" id="Team_Wolfe_Selection">Team Wolfe Selection</SPAN></H3><P><B>Goals for the week:</B></P><UL><LI>MAGE out HisB
</LI><LI>What we've been trying/why it's not working — keep trying!
</LI></UL><LI>PCR out Kan Cassette (colony PCR)- analysis/check that it worked
</LI><LI>We did the insertion and have things growing in Kan that look like our cells — we probably have it!
</LI><LI>As long as it's the right size and in the right place, it probably worked, meaning we have 2 HIS genes → MAGE is to remove the endogenous genome copy in vivo
</LI><LI><B>Construct</B> (PSR01): ~ - KAN - ZFbs - weak promoter - URA3 - HIS3- ~
</LI><LI>Fix/Test NM media
</LI><LI>Made last week, nothing worked in it — try more glucose
</LI><LI>Wolfe strain seems to grow really slowly (~36h); try glycerol stock of PKD46, grows faster
</LI><LI>Make other 3-AT plates (1.5)
</LI><LI>3-AT is a competitive inhibitor of His; the stronger His is expressed, the more 3-AT the cell can tolerate and still grow
</LI><LI>Test 3-AT plates
</LI><P><B>Today's results:</B></P><UL><LI>PCR to check whether kan-ZFB-wp-hisura was successfully inserted into the genome
</LI><LI>Used KAPA master mix and protocols, same procedure as 6/29/11
</LI><LI>Because of KAPA shortage, only ran samples 6-8 from 100µL 6/29 plate and 6-7 from 2mL 6/29 plate plus (ΔHisB)ΔpyrFΔrpoZ+pKD46 from 6/29 culture as a control (should produce a band around 300bp, while the insert would make the band 2-3kb)
</LI><LI>Ran on E-Gel with team TolC's samples, but there was an issue loading the gel, so also ran separately on another gel. Unfortunately most of the PCR reactions strangely did not work, but two of the samples appeared not to have the insert (see below).
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">2011.07.05.kanZFBwp_tolC+kanZFBinsert_check.ai</DIV></DIV></DIV><UL><LI>Redo PCR:
</LI><LI>out of KAPA so had to use Invitrogen Platinum Supermix
</LI><LI>each reaction had 22.5µL supermix, 0.5 µL each 10µM primer, 1µL template, 0.5µL H2O
</LI><LI>20 reactions, including one from the same control used above.
</LI><LI>followed Invitrogen instructions, with 3 min elongation, 56 C annealing, 30 cycles
</LI></UL><LI>Assuming the PCR is unsuccessful, we will grow up (ΔHisB)ΔpyrFΔrpoZ+pKD46 (from 6/28 plate) overnight in LB+amp
</LI><LI>NM medium:
</LI><LI>To test Sarah's NM (made 6/30) we will grow up pKD46 strain in 3mL of it overnight at 30 C.  If no bacteria grew, we will try remaking it ourselves.
</LI><H3><SPAN class="mw-headline" id="Team_TolC">Team TolC</SPAN></H3><UL><LI>PCR out and insert Kan construct in front of TolC (similar to Wolfe selection)
</LI><LI><B>Construct 1</B>: ~ - KAN - ZFbs - weak promoter - ~ (done, PCR, gel extraction)
</LI><LI>Used primers ZFB-wp-tolC-r and tolC-kan-f
</LI></UL><LI><B>Construct 2</B>: ~ TolC ~ (do not have primers yet)
</LI><LI><I>Note that ~ indicates a homology region</I></LI><LI>Need the EcNR2 strain to come in before we can work with it (came in Thursday 7/7/2011, the one from Wednesday 7/6/2011 was not growing well.)
</LI><LI>Inactivate rpoZ (similar to inactivating His3, but should be easier to use MAGE with- EcNR2 is a strain designed for MAGE)
</LI><LI>MAGE oligo rpoZ deletion will arrive on Thursday 7/7/2011
</LI><LI>Make SDS (how we test TolC +vely)(done, stored at room temperature)
</LI><LI>Only cells with active TolC can survive SDS treatment
</LI><H3><SPAN class="mw-headline" id="Web_Design">Web Design</SPAN></H3><H4><SPAN class="mw-headline" id="To_Do">To Do</SPAN></H4><UL><LI>Safety Description
</LI><LI>Email Alain/Harvard biosafety group  <FONT color="red">✔ (Meeting pending)</FONT></LI><LI>Needed information:
</LI><LI>Strain names
</LI><LI>Antibiotic Resistance
</LI><LI>DNA inserts/removals
</LI></UL><LI>Draft Project Description <FONT color="red">✔</FONT></LI><LI>Finalize
</LI><LI>Update the Facebook page/create a new 2011 page
</LI><LI>Get the Public Wiki up and running
</LI><LI>Figure out templates
</LI><LI>Update Twitter
</LI><H3><SPAN class="mw-headline" id="Notes_to_All_Teams">Notes to All Teams</SPAN></H3><UL><LI>Teams: Make sure everything is organized in your dropbox!
</LI><LI><I>*Announcing a New (and Final) Plasmid Name*</I><B>PSR01</B>: formerly known as Wolfe Plasmid/Vatsan's Plasmid/Selection 
</LI><LI>Contains ~ - KAN - ZFbs - weak promoter - URA3 - HIS3- ~</LI></UL><DIV id="706" style="display:none"><H2><SPAN class="mw-headline" id="July_6th">July 6th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_Assembly_2">Team ZF Assembly</SPAN></H3><P>Because our primers still haven't arrived, the members of Team ZF Assembly are working with the Web Design Team.
</P><H3><SPAN class="mw-headline" id="Team_Wolfe_Selection_2">Team Wolfe Selection</SPAN></H3><UL><LI>Redo PCR Results:
</LI><LI>Samples 1,3,4,5,6,7,8,9,11 from the 100µL 6/29 plate all showed unsuccessful introduction of kan-ZFB-wp-hisura
</LI><LI>Samples 2,3,4,6,7,8,9,10,11,12 from the 2mL 6/29 plate all showed unsuccessful introduction of kan-ZFB-wp-hisura
</LI><LI>(ΔHisB)ΔpyrFΔrpoZ+pKD46 from 6/29 culture used as a control
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">1529620 locus PCR 1 7/6/11</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">1529620 locus PCR 2 7/6/11</DIV></DIV></DIV><UL><LI>NM media has 1% of the glucose concentration written in the protocol, it is 0.4mg/ml instead of 40mg/ml. Nevertheless, our overnight culture of pKD46 grew successfully so we will continue to use the media unless other problems arise.
</LI><LI>One possible explanation for our failure to PCR out a kan-ZFB insert from the recombined colonies is that the insert was integrated into the genome somewhere else besides the 1529620 locus. This might be why the PCR shows no insert while the colony still grows in kanamycin. We will try to PCR the insert using primers that anneal to the kan and his-ura sections (since these primers are about 70 bases long, we will do a 2-step cycle)
</LI><LI>Invitrogen Platinum Supermix reagents and protocols, with 3:30 at 72˚C instead of separate annealing and elongation, 25 cycles
</LI><LI>used same cultures as for PCR listed above
</LI><LI>kan-ZFB-wp-hisura insert as control
</LI><LI>results from 2 E Gels: PCR not very successful. Colonies showed no bands while the control showed only a side product too small to be the full insert.
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Kan-ZFB check using kan and hisura primers 7/6/11</DIV></DIV></DIV><UL><LI>Grew up 24 more kan-ZFB lambda red recombined colonies (12 from 100µL 6/29 plate, 12 from 2mL 6/29 plate) in LB+kan in 96 well plate for PCR tomorrow
</LI><LI>Diluted 24 colonies (1 ss+pKD46 control, 11 from MAGE round 2 1µL plate, 12 from MAGE round 2 10µL plate) in 30µL ddH2O and split between NM,NN+Histidine, and LB+amp to check his phenotype
</LI><LI>NM+his media: 49mL NM, 1 mL 5% histidine solution
</LI></UL><LI>Made 1.5mL 3-AT plates
</LI><H3><SPAN class="mw-headline" id="Team_TolC_2">Team TolC</SPAN></H3><UL><LI>Gel extraction of KAN-ZFBs-wp as there is a side product.
</LI><LI>To increase yield, spin down the contents of both eppendorf tubes into a single column, have a 2:1 volume ratio of QG Buffer to gel volume, e.g. 1ml QG buffer to 0.5ml gel in step 2, and elute in 30µl water in step 13)
</LI></UL><LI>checked the ECNR2 strains, both appear to have not grown
</LI><LI>note from 7/7/2011, new ECNR2
</LI><LI>running another PCR since there was errors in gel extraction, and the yield was less than 20ng/µl, and purity was very bad.
</LI><P>
TOLCINV PCR-Machine 7
</P><UL><LI>94°C for 2 min
</LI><LI>94°C for 30 s
</LI><LI>53°C for 30 s
</LI><LI>72°C for 1.5 min
</LI><LI>Previous three steps repeat 30 times
</LI></UL><LI>72°C for 5 min
</LI><LI>4°C forever
</LI><DIV class="thumb tnone"><DIV class="thumbinner" style="width:184px;"><DIV class="thumbcaption">KAN-ZFB-WP</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Web_Design_2">Web Design</SPAN></H3><UL><LI>Project description
</LI><LI>Web design brainstorming;exploring past designs and templates</LI></UL><DIV id="707" style="display:none"><H2><SPAN class="mw-headline" id="July_7th">July 7th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_Assembly_3">Team ZF Assembly</SPAN></H3><P>Our primers still haven't arrived. We are working with the Web Design Team.
</P><H3><SPAN class="mw-headline" id="Team_Wolfe_Selection_3">Team Wolfe Selection</SPAN></H3><UL><LI>PCR for Round 2 MAGE cultures in LB media
</LI><LI>The samples came from the overnight culture of MAGE/control strains grown in LB+amp, NM+his, and NM. Oddly, the (∆HisB)∆pyrF∆rpoZ+pKD46 control grew only in NM+His and LB, and not in plain NM.
</LI><LI> Samples
</LI><LI> Control= 1 ss+pKD46
</LI><LI> 11 cultures from 1μL plate in LB media 
</LI><LI> 12 cultures from 10μL plate in LB media
</LI></UL><LI>Program
</LI><LI>Anneal at 56°
</LI><LI>Elongation for 90 seconds
</LI><LI>All else according to standard KAPA protocol
</LI><LI>E gel of control and 1µL plate colonies 1-8 to check PCR worked (it was successful)
</LI><LI>samples prepared for sequencing tomorrow
</LI><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">MAGE 2 samples for sequencing 7/7/11</DIV></DIV></DIV><UL><LI>Kan-ZFB insert check PCR results:
</LI><LI>the overnight cultures grown yesterday were used in a PCR (same KAPA protocol as before) around the 1529620 locus. Not all the cultures grew; 100µL plate cultures 1-5 and 2mL plate cultures 1-12 were used.
</LI><LI>Results: some of the reactions, strangely, did not work, but the ones that did showed a short band and thus did not have the insert (see below)
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">1529620 locus PCR 7/7/11</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">1529620 locus PCR 7/7/11</DIV></DIV></DIV><UL><LI>HisBNuke3 MAGE round 3:
</LI><LI>used same protocol as previously described, using 1.5mL mid-log culture from MAGE round 2
</LI><LI>will recover overnight so that we can reinoculate and continue MAGE rounds tomorrow
</LI></UL><UL><LI>NM medium:
</LI><LI>added 3.9g glucose to 100mL NM medium to bring the glucose concentration up to the correct level.
</LI><LI>Because (∆HisB)∆pyrF∆rpoZ+pKD46 did not grow in plain NM, we are curious to see if that was due to low glucose levels. One colony from a plate was put in 3mL NM(gluc) and will be grown overnight.
</LI></UL><UL><LI>HisB MAGE2:
</LI><LI>In case the sequencing does not show successful MAGE, we will grow overnight 48 more colonies (24 from each plate) in 100µL of NM(gluc) and LB+amp
</LI></UL><H3><SPAN class="mw-headline" id="Team_TolC_3">Team TolC</SPAN></H3><UL><LI>making culture tubes of ECNR2 and ECNR2delTolC for glycerol stock (no Amp)
</LI><LI>making culture tubes of ECNR2 for MAGE to delete rpoZ (one with amp, one without)
</LI><LI>dilutions of rpoZ deletion MAGE oligo, 20µM
</LI><LI>dilution of rpoZ deletion MAGE oligo, 1µM, for MAGE.
</LI><LI>serial dilution of ECNR2 after MAGE round 1, and round 2, with 20µl of 100x dilution of concentrated culture. For plating onto amp plates.
</LI><LI>During Gel extraction, ignored 3x volume in step 2 and instead used 500 µl of QG buffer. Additionally, used 18 µl of EB buffer and not water in step 13
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Kan-ZFB-WP</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Web_Design_3">Web Design</SPAN></H3><H4><SPAN class="mw-headline" id="Brainstorming">Brainstorming</SPAN></H4><UL><LI>Minimalist toolbar, metallic crimson color
</LI><LI>&quot;Popout&quot; menu below, <I>not</I> dropdown
</LI><LI>Slideshow on front page
</LI><LI>First slide: Brief text project description, brief youtube video description (in layman's terms)
</LI></UL></DIV><DIV id="708" style="display:none"><H2><SPAN class="mw-headline" id="July_8th">July 8th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_Assembly_4">Team ZF Assembly</SPAN></H3><H4><SPAN class="mw-headline" id="PCR_of_omega.2Bzif268_with_overhangs">PCR of omega+zif268 with overhangs</SPAN></H4><P>We performed a PCR to take out our omega subunit plus zif268. We used the following recipe:
</P><UL><LI>10 ul KAPA readymix
</LI><LI>0.75 ul of hindIII (primer)
</LI><LI>0.75 ul of wF+plLacO (primer)
</LI><LI>1.2 ul template
</LI><LI>12.3 ul water
</LI></UL><P>The first steps of the PCR involved heating at 95* (for 5 minutes) and 98* (for 20 seconds). Annealing temperature was 56*. Extension time was 30 seconds. 
</P><P>The product of the PCR should be 709 bps. It includes the omega subunit, zif268, as well as overlap sequences homologous to the selection plasmid. 
We ran an e-gel to ensure that our desired product was present:
</P><H4><SPAN class="mw-headline" id="Gel_purification_of_PCR_product">Gel purification of PCR product</SPAN></H4><P>We had to gel purify our PCR product in preparation for isothermal assembly, using <A href="/Protocols#Gel_purification" title="Protocols"> our protocol for gel purification</A>.  The result from running the gel can be seen below - we cut out the bands at 709 bp, the expected size of our product.
</P><P>In the final step of the purification process, we eluted our DNA in buffer EB. We used the nanodrop spectophotometer to determine the concentration of DNA present. 
</P><UL><LI>The concentration was 54.3 ng/ul.
</LI></UL><H3><SPAN class="mw-headline" id="Team_Wolfe_Selection_4">Team Wolfe Selection</SPAN></H3><P><B>MAGE round 3:</B></P><UL><LI>MAGE cultures that recovered overnight were saturated, so we made 1:1000, 1:10000, and 1:100000 dilutions and plated 20µL of each onto amp plates and left at 30˚C overnight.
</LI></UL><P><B>MAGE round 2 cultures from yesterday:</B></P><UL><LI>We did a PCR (His_F and and His_R primers) on 24 of the 48 cultures we grew and sent them off for sequencing.
</LI></UL><P><B>Results of NM culture:</B></P><UL><LI>The selection strain (∆HisB∆pyrF∆rpoZ+pKD46) did not grow in NM, which is worrisome.  None of the MAGE samples grew in NM either, so either they all have HisB knocked out or there is something wrong already with the strain's His pathway.
</LI><LI>We will grow up selection strain (without pKD46) from glycerol stock in LB+tet and transform with Vatsan's plasmid tomorrow to see if that has any effect on the his-deficiency phenotype.
</LI></UL><P><B>Kan-ZFB insertion: modified nucleotides:</B></P><UL><LI>Primers finally arrived that would put modified nucleotides on the 5' ends of the insert, which should improve lambda red efficiency.
</LI><LI>PCR: KAPA protocol with Kan forward phosphorthioate and URA2 reverse phosphorthioate, 56˚C annealing, 90 sec elongation, 25 cycles, 1µL kan-ZFB-hisura DNA as template
</LI><LI>Results: PCR only produced a smaller side product. This is strange since we also ran a gel of all our different kan-ZFB-hisura purifications and they all run at the correct size (see images below).
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Thio kan-ZFB 7/8/11</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">kan-ZFB insert purifications</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Team_TolC_4">Team TolC</SPAN></H3><UL><LI>Performed re-inoculation of ECNR2-Placed into tube labeled ECNR 2 MAGE 3 in 30 degree shaker.
</LI><LI>grew up 24 colonies total for sequencing ECNR2 for rpoZ deletion (the colonies appear to have grown much faster than expected).
</LI><LI>rows A (MAGE round 1) and B (MAGE round 2), for PCR and sequencing. to be placed in the 4C refrigerator after use as template.
</LI></UL><LI>RPOZ PCR-Machine 6
</LI><LI>Primer Annealing = 56 degrees
</LI><LI>Extension 48 = seconds
</LI><H3><SPAN class="mw-headline" id="Team_Web_Design">Team Web Design</SPAN></H3><UL><LI>Edited/Re-drafted our project description, see Dropbox (It's a bit long as is now, but the excess informaiton that we cut out can still probably be used somewhere on our website
</LI><LI>Sketched draft pages of the public wiki 
</LI><LI>Homepage- first draft sketch done based on the characteristics and requirements we'd discussed. However, there are a few issues (i.e., how to include some of the iGEM wiki reuqirements).</LI></UL><DIV id="709" style="display:none"><H2><SPAN class="mw-headline" id="July_9">July 9</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Isothermal_Assembly">Isothermal Assembly</SPAN></H4><P>We performed an isothermal assembly in order to insert omega+zif268 into the plasmid containing the spec backbone. 
</P><UL><LI>The concentration of the plasmid (previously prepared) was 157.4 ng/ul.
</LI><LI>The concentration of the w+zif268 was 54.3 ng/ul.
</LI></UL><P>Using <A href="https://2011.igem.org/Team:Harvard/Protocols#Isothermal_assembly" class="external text" rel="nofollow">our isothermal assembly protocol</A> we determined we needed:
</P><UL><LI>0.63 ul backbone (length: ~2.2 kb)
</LI><LI>0.59 ul w+zif268 (length: ~0.7 kb)
</LI><LI>2.78 ul water 
</LI><LI>15.0 ul isothermal assembly solution
</LI></UL><H4><SPAN class="mw-headline" id="Transformation_of_electro-competent_E._coli">Transformation of electro-competent <I>E. coli</I></SPAN></H4><P>We electroporated our cells so they would take up the plasmids, using:
</P><UL><LI>1.0 ul DNA 
</LI><LI>40.0 ul cells
</LI><LI>500 ul LB
</LI></UL><P>We plated the cells, <A href="https://2011.igem.org/Team:Harvard/Protocols#Cultures" class="external text" rel="nofollow">using this protocol</A>, on spec plates, one of which contained 50 ul of cells and the other 150 ul of cells. The cells have been incubated at 37*, and will be left there overnight. Cells that have taken up the plasmid have spec-resistance and should be able to survive. Tomorrow, we will be picking colonies and performing PCR on them to determine whether they contain w+zif268.
</P><H3><SPAN class="mw-headline" id="Team_Wolfe">Team Wolfe</SPAN></H3><UL><LI>MAGE3 plates look good--we'll choose colonies for sequencing on Monday.
</LI><LI>Transformation of selection strain with Vatsan's plasmid:
</LI><LI>We're curious to see whether Vatsan's plasmid (Zif268 omega, ZFB, wp, hisura) will have an effect on the (∆HisB)∆pyrF∆rpoZ strain's growth despite the seemingly intact HisB.
</LI><LI>Transformed mid-log selection strain (reinoculated from overnight culture) with about 100ng of Vatsan's plasmid following standard procedure.
</LI><LI>After 2 hr recovery plated on amp plates with 100µL or 1mL</LI></UL><DIV id="710" style="display:none"><H2><SPAN class="mw-headline" id="July_10th">July 10th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_2">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="PCR_to_check_for_omega.2Bzif268">PCR to check for omega+zif268</SPAN></H4><P>All colonies that grew on the plates we left overnight should have spec-resistance, but they may not have the omega and zif268 unit that we inserted yesterday with isothermal assembly. We performed a PCR on 13 colonies picked from the 50 ul plate to find a colony that contains the w+zif268 insertion. We used the same recipe at our PCR reaction [[#PCR of omega+zif268 with overhangs on Friday. If the insert is present, then we should see a band at 709 bp. We also included 2 positive controls: our isothermal assembly product and our selection strain from which the w+zif268 was originally copied. 
</P><P>We ran an agarose gel to check the bands:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Lanes 1-8</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Lanes 9-16</DIV></DIV></DIV></TD></TR></TBODY></TABLE>
None of the colonies displayed any bands, except for primer residues. Lane 16 contained the selection plasmid, on which we performed <A href="https://2011.igem.org/Team:Harvard/Protocols#PCR_purification" class="external text" rel="nofollow">a successful PCR</A> on Friday, so we definitely expected a band at 709 bp. Because the positive controls did not have a band at 709 bp, we suspect that something went wrong with the PCR. We plan on re-running the PCR tomorrow.</DIV><DIV id="711" style="display:none"><H2><SPAN class="mw-headline" id="July_11th">July 11th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_3">Team ZF</SPAN></H3><P>We did a miniprep of the 12 colonies we picked Sunday and left overnight in LB. Here are the concentrations of our products:
</P><TABLE><TBODY><TR><TH>ID
</TH><TH>Conc. (ng/ul)
</TH><TH>260/280
</TH></TR><TR><TD>1
</TD><TD>6.63
</TD><TD>1.27
</TD></TR><TR><TD>2
</TD><TD>12.96
</TD><TD>1.53
</TD></TR><TR><TD>3
</TD><TD>9.28
</TD><TD>1.41
</TD></TR><TR><TD>4
</TD><TD>7.23
</TD><TD>1.31
</TD></TR><TR><TD>5
</TD><TD>12.64
</TD><TD>1.55
</TD></TR><TR><TD>6
</TD><TD>10.77
</TD><TD>1.48
</TD></TR><TR><TD>7
</TD><TD>17.19
</TD><TD>1.73
</TD></TR><TR><TD>8
</TD><TD>39.70
</TD><TD>1.78
</TD></TR><TR><TD>9
</TD><TD>55.73
</TD><TD>1.81
</TD></TR><TR><TD>10
</TD><TD>24.34
</TD><TD>1.72
</TD></TR><TR><TD>11
</TD><TD>25.19
</TD><TD>1.68
</TD></TR><TR><TD>12
</TD><TD>38.57
</TD><TD>1.74
</TD></TR><TR><TD>13
</TD><TD>27.26
</TD><TD>1.75
</TD></TR></TBODY></TABLE><P>Then we performed a PCR on the 12 miniprep products, the 12 colonies in LB, as well as 3 controls (negative control, isothermal assembly product, selection strain) to find a colony that has the omega+zif26 insertion:
</P><UL><LI>10 ul KAPA readymix
</LI><LI>0.75 ul of hindIII (primer)
</LI><LI>0.75 ul of wF+plLacO (primer)
</LI><LI>1.2 ul template
</LI><LI>12.3 ul water
</LI></UL><P>The first steps of the PCR involved heating at 95* (for 5 minutes) and 98* (for 20 seconds). Annealing temperature was 56*. Extension time was 30 seconds. 
</P><P>
We ran a 1% 100 ml. agarose gel at 120V to check the success of the PCR.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gel of our colonies, miniprep-ed colonies, and controls to check for w+zif268 insertion</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>The bands were rather thick, so it was difficult to estimate the exact band size. However, the bands appeared a bit below 650 bp, although we expected a size of 709 bp, so we ran an e-gel on some of the colonies to ensure that that bands were indeed the right size.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gel to ensure our bands were the correct size (709 bp)</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>The bands were closer to the size that we expected, so we decided that we successfully inserted w+zif268 into the plasmid. We decided to pick colony 9 (which had the highest concentration post-miniprep) and work with that from now on. We made spec plates with 50 ul of colony 9.
</P><H3><SPAN class="mw-headline" id="Team_Wolfe_2">Team Wolfe</SPAN></H3><P><B>Selection strain transformed with pSR01:</B></P><UL><LI>both plates had lots of colonies--almost too many.  We need to make sure the plasmid is, indeed, in these cells.
</LI><LI>Colony PCR:
</LI><LI>picked 6 colonies and diluted in 7µL H2O: 3 for PCR template and 4 to be grown up in LB+amp
</LI><LI>KAPA protocol using M13_F and M13_R primers (anneal to the vector), 50˚C annealing, 2min elongation
</LI><LI>E gel results: lots of bands but rather messy and not exactly what we'd expect. We'll try to verify the presence of the plasmid in a few other ways.
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">pSR01 colony PCR 7/11/11</DIV></DIV></DIV><UL><LI>Sequencing:
</LI><LI>We sent out the PCR samples with M13_R to Genewiz
</LI></UL><LI>Culture PCR:
</LI><LI>We used the cultures made with the colonies used for PCR above as template to see if this gives us cleaner results.
</LI><LI>same protocol as above except with 1µL of 1:20 dilution of culture as template.
</LI><LI>Assuming that the plasmid is indeed in the colonies, we will grow an overnight culture in LB+amp as well as the Wolfe selection strain and tomorrow we will use a growth reader to check their phenotypes in NM
</LI><P><B>MAGE round 3 colonies:</B></P><UL><LI>In case pSR01 does not rescue the selection strain's growth phenotype, we will PCR 96 colonies from the MAGE3 plates and send them out for sequencing tomorrow.
</LI><LI>Used 9µL KAPA, His_F and His_R primers, 3µL suspended colony (out of 7--the remaining 4 grown in LB+amp)
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">MAGE3 colonies for sequencing 7/11/11</DIV></DIV></DIV><P><B>Thio kanZFB:</B></P><UL><LI>We retried the PCR to put modified nucleotides on the ends of the kan-ZFB-wp-hisura insert using the same procedure as 7/8/11
</LI><LI>E gel: reactions didn't work
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">thio kan-ZFB PCR 7/11/11</DIV></DIV></DIV><UL><LI>Tried again with a longer elongation time (3 min) and let run overnight.
</LI></UL><H3><SPAN class="mw-headline" id="Team_TolC_5">Team TolC</SPAN></H3><H4><SPAN class="mw-headline" id="Sequencing_of_rpoZ_delete">Sequencing of rpoZ delete</SPAN></H4><UL><LI>Since the sequencing was sent after 3pm on Friday, it wasn't picked up before today (Monday), so we are going ahead with lambda red recombineering of the KAN-ZFB-wp into the ECNR2 genome
</LI><LI>glycerol stocks of ECNR2 and ECNR2ΔTolC were made.
</LI></UL><H3><SPAN class="mw-headline" id="Team_Web">Team Web</SPAN></H3><UL><LI>Looked at good wikis from last year. </LI></UL><DIV id="712" style="display:none"><H2><SPAN class="mw-headline" id="July_12th">July 12th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_4">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="PCRs_of_finger_ultramers">PCRs of finger ultramers</SPAN></H4><P>We received the ultramers for fingers two and three of our 6 experimental expression plasmids. We performed an overlap PCR in order to join the two ultramer pieces and make a whole unit. Ultimately this unit with be assembled with the omega subunit and the spec backbone to make our final expression plasmids. We used the following recipe:
</P><UL><LI>10 ul KAPA
</LI><LI>13.5 ul water
</LI><LI>0.75 ul forward ultramer
</LI><LI>0.75 ul reverse ultramer 
</LI></UL><P>We had an annealing temperature of 58* and an extension time of 15 seconds. 
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gel of our product from the ultramer PCR. Note that the first lane's PCR was done in a single tube, while all the others were done in a strip.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>However, in addition to the expected band at approximately 300 bp, the bands for our ultramers was extremely streaky, indicating that there may be other contaminants. We decided to perform a PCR purification of these samples, using <A href="/Protocols#PCR_purification" title="Protocols">our protocols</A>, and see if there was an improvement in the gel.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gel of our product after PCR purification</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>Even after the gel purification, our bands were streaky. We decided to redo the PCR, placing all the reactions in single tubes, performing a touchdown PCR instead. Our annealing temperature started at 70*, repeated for 2 cycles, and then decreased 2*, every two cycles, until it reached 58* (the optimal annealing temperature), where it remained until the 25 cycles were complete. We left this PCR going overnight. (See July 13th for the gel image.)
</P><H4><SPAN class="mw-headline" id="PCR_of_expression_plasmid_cross-junction">PCR of expression plasmid cross-junction</SPAN></H4><P>We also performed a second PCR on colony nine (the colony we decided with which we decided to proceed) to confirm that our omega+zif268 in the spec backbone was successfully inserted into the <I>E. coli</I>. The expected product was the cross-junction on the plasmid, approximately 1.4 kb. We used the same recipe as that used on July 9th, except with a forward primer of PZE23G-3581 F and a reverse primer of PZE23G-2133 R, and had a 55* annealing temperature and an extension time of 45 seconds. 
</P><P>After running an e-gel, we confirmed that our w+zif268 was inserted into the plasmid.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption"><B>Lane 3</B>: Gel of our PCR to check for the cross-junction in colony 9.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="PCR_and_gel_extraction_of_omega_subunit">PCR and gel extraction of omega subunit</SPAN></H4><P>We performed a PCR to get out the omega subunit+linker, with an annealing temperature of 56* and an extension time of 15 seconds. The miniprep from colony 8 was the template DNA. We will use the product in assembling our final expression plasmids. 
</P><P>We determined that the PCR was successful after running the sample on an agarose gel, and performed a gel extraction so that we could use the product in our isothermal assembly.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption"><B>Lanes 1 and 2</B>: Gel of our PCR to get omega subunit.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>The gel extraction was successful, and we had a final concentration of 120.5 ng/ul.
</P><H4><SPAN class="mw-headline" id="Overlap_PCR_of_OZ052_and_OZ123">Overlap PCR of OZ052 and OZ123</SPAN></H4><P>We performed an overlap PCR using the ultramer pairs that solely encoded the F1, F2, and F3 fingers of OZ052 and OZ123.  The annealing temperature was 62* and the extension time was 15 seconds.  The results of the PCR can be found below (the products are only in the last two lanes) and the products were also PCR purified before being put into storage at -20*C.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption"><B>Last two lanes</B>: OZ052 and OZ123 products, respectively</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Team_Wolfe_3">Team Wolfe</SPAN></H3><P><B>MAGE round 3 samples:</B></P><UL><LI>94 sent out for sequencing (numbers go A1-H1, etc.)
</LI></UL><P><B>Thio kan-ZFB results:</B></P><UL><LI>PCR run out on E Gel: not successful.  We do not know what is still causing our PCR to fail.
</LI></UL><P><B>pSR01-transformed selection strain:</B></P><UL><LI>We discovered that hisB is a two-function protein, with the N-terminal region acting as an HPase and the C-terminal region as an IDPG. His3 only complements the IDPG activity, so even if our MAGE worked, it might destroy our selection system.  The two aa deleted in hisB are in the C terminal region, so maybe they do inactivate it.  We emailed Keith Joung whose lab made the selection strain to see how HisB was deleted.
</LI><LI>In case this mutation is correct but lambda red in the selection strain still eludes us, we designed MAGE oligos for use in EcNR2 to either delete the two aa or put 3 stop codons after the HPase region.
</LI></UL><LI>culture PCR results: cultures 3-4 had a strong band (and 2 and 5 weak ones) around 850-1000bp.  It is interesting how different these results are from yesterday's colony PCR gel.
</LI><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">pSR01 culture and thio kan-ZFB PCRs 7/12/11</DIV></DIV></DIV><UL><LI>Sequencing results: The sequences sent from Genewiz aligned well with the pSR01 plasmid sequence and covered pretty much all of URA3. This implies that the cells did indeed take up our plasmid.
</LI><LI>Growth phenotype:
</LI><LI>12 colonies were chosen and split between NM (gluc), NM+IPTG (to induce zif268 expression), NM+his (gluc), NM+his+IPTG, LB+amp, LB+amp+IPTG and grown overnight. (Final concentration IPTG=500µM)
</LI></UL><H3><SPAN class="mw-headline" id="Team_TolC_6">Team TolC</SPAN></H3><UL><LI><B>MAGE knockout sequencing</B></LI><LI>Received sequencing from the MAGE knockout of rpoZ within the TolC genome
</LI><LI>Observed all 24 sequenced attempts and it appears as none of them have the knockout and stop codons placed into genome
</LI><LI>Possible reason behind failure is that one, we may not have been reading the results correctly or two, the MAGE oligo had all three stop codons right next to each other in the oligo and may have needed to be separated by three nucleotides each in order to maximal annealing
</LI></UL><LI><B>Lambda Red Recombination of kan-ZFB-wp insertion into MAGE knockouts</B></LI><LI>Plates from the recombination showed small colonies
</LI><LI>With this we picked 6 colonies from the 4 different plated recombination cultures and grew them to saturation
</LI><LI>Began PCR with TolC_Seq_F and TolC_Seq_R primers in order to determine whether recombination was successful
</LI><LI>Procedure (on gradient PCR machine, name: ABC, folder: igem)
</LI><LI>KAPA mixture
</LI><LI>Then 95 C, 5 min
</LI><LI>98˚ C, 20 sec
</LI><LI>60˚ C, 15 sec
</LI><LI>75˚ C, 45 sec
</LI><LI>Repeated cycle for 30 times
</LI><LI>72˚ C, 5 min
</LI><LI>4˚ C, forever
</LI><LI>Tomorrow we plan to run gel
</LI><LI>Length will be:
</LI><LI>450 bp if insertion failed
</LI><LI>1200 bp if insertion of Kan cassette only
</LI><LI>1650 bp if insertion of Kan-ZFB-wp
</LI><LI><B>Insertion Construct for TolC (homology-kan-ZFB-wp-homolgy)</B></LI><LI>Began 90 µL total of PCR for the insertion construct since we ran out
</LI><LI>If gels show failed PCR tomorrow we will start from normal ENR2 without any MAGE and begin with lambda red recombination and possibly make new MAGE primer and then knockout rpoZ
</LI><H3><SPAN class="mw-headline" id="Team_Web_Design_2">Team Web Design</SPAN></H3><UL><LI>Met with Harvard biosafety officer, Sidney
</LI><LI>Send to him: list of strains, plasmids that we are using
</LI><LI><FONT color="red"><I><B>Note to all iGEM-ers:</B></I> We will all be attending a biosafety seminar on Friday, July 15th from 10am to 12n in Biolabs 1058. </FONT></LI></UL><DL><DD>In case you haven't already, please see Alain's message below:
</DD></DL><DL><DD>Every member of the Harvard iGEM team (unless you are absent on Friday) has to attend the next lab Safey/Biosafety Blood born pathogens training this coming Friday 7/15/2011 from 10 am to 12 noon.  The training takes place in the Biological laboratories (16 Divinity Ave) room 1058. If you cannot go and have a very good reason let me (<I>Alain</I>) know. If you attended a similar training session on the Cambridge campus or at the Medical school you don't have to attend. Otherwise you have to go.</DD></DL><H2><SPAN class="mw-headline" id="July_13th">July 13th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_5">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Results_of_ultramer_touchdown_PCR">Results of ultramer touchdown PCR</SPAN></H4><P>We ran an e-gel on the product of our touchdown PCR. Once again we have our band at 344 bp. It is still slighly streaky, but much less than our <A href="#PCRs_of_finger_ultramers">earlier PCR reaction</A>. There are some faint bands smaller than our expected product, which are likely the ultramers we used in the reaction. We'll use the product for our isothermal assembly later today.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gel of our ultramer touchdown PCR product</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Isothermal_assembly_and_transformation_of_expression_plasmids">Isothermal assembly and transformation of expression plasmids</SPAN></H4><P>We followed <A href="/Protocols#Isothermal_assembly" title="Protocols">our protocol</A> for isothermal assembly, using the following recipe:
</P><UL><LI>0.63 ul backbone (at 157.4 ng/ul to get 100 ng)
</LI><LI>0.13 ul omega+linker (at 120.5 ng/ul)
</LI><LI>0.50 ul ultramer (~30 ng/ul)
</LI><LI>1.24 ul water
</LI><LI>7.5 ul isothermal assembly solution
</LI></UL><P>The sizes of the pieces we put together were the following:
</P><UL><LI>Backbone: 2.2 kb
</LI><LI>omega+linker: 352 bp
</LI><LI>ultramer: 344 bp
</LI></UL><P>Note: We also made a negative control of the isothermal assembly, only putting in the spec backbone without omega+linker or ultramer.
</P><P>We then performed transformations to insert our isothermal assembly products into <I>E. coli</I>, following our <A href="/Protocols#Cultures" title="Protocols">usual protocol</A>. However, we ran out of electrocompetent cells, so only samples 78 - 82 were done with electroporation while 77, 83-85, and the negative control were done with chemical transformation, using <A href="/Protocols#Chemical_transformation" title="Protocols">these protocols</A> (the samples are named after the reverse ultramer ID in the primer index that were used to generate the ultramer sequences).
</P><P>We then plated 1 ul of the cells on spec plates and left them overnight at 37*. Because we ran out of our own spec plates, we also had to use LB plates that had spec added to it, and spec plates that were made in April.
</P><H3><SPAN class="mw-headline" id="Team_Wolfe_4">Team Wolfe</SPAN></H3><P><B>∆HisB∆pyrF∆rpoZ+pSR01 growth results:</B></P><UL><LI>When no IPTG was present, the bacteria grew in LB+amp, NM+his, and NM, showing that the plasmid rescues the selection stain's growth phenotype! This implies that our strain had the right genotype all along. However, when IPTG was added to the media, the strain only grew well in LB.  This is very odd, since IPTG was supposed to induce zif268 and thus His3 expression. We will try a couple things to make sure nothing went wrong:
</LI><LI>redo cultures in the same pattern as yesterday--perhaps IPTG+/- were mislabeled
</LI><LI>inoculate cultures of ∆HisB∆pyrF∆rpoZ +/- pSR01 so that Vatsan can use a plate reader to compare their growths
</LI></UL><LI>Joung wrote back and said that the HisB deletion is in fact 6 bases in frame. He also mentioned something about a lac deletion which we are looking into.
</LI><P><B>MAGE3 sequencing results:</B></P><UL><LI>none of the colonies had the stop codons.  This may be because mutS is not knocked out, and it can correct strands that have up to 5 bases mismatched.
</LI></UL><P><B>Kan-ZFB for lambda red:</B></P><UL><LI>Noah did lambda red in EcNR2 and the plate was covered in crystalline patterns that may be cells. We will test this by:
</LI><LI>Restreaking some of these &quot;colonies&quot; onto a fresh kan plate to get the bacteria more spread out
</LI><LI>Streaking a kan plate with pKD46 (from glycerol stock) to see if the kan is effective
</LI><LI>Taken a streak off the plate, diluted it in 200µL H20, and used 1µL as template for PCR
</LI><LI>KAPA, 1529620-flanking primers, 90 sec elongation and 56 anneal, 25 cycles
</LI><LI>Results: 300bp band--the streaks on the plate are bacteria, but they do not have the insert.
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">1529620 locus PCR of EcNR2 lambda red 7/13/11</DIV></DIV></DIV><UL><LI>We also retried the thio kan-ZFB PCR using some new primer aliquots. Once again it failed.
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Thio kan-ZFB PCR 7/13/11</DIV></DIV></DIV><UL><LI>Because this PCR keeps failing, we're going to try a different way and do two separate and shorter PCRs to be joined later in an overlap PCR:
</LI><LI>PCR of kan cassette: kan cassette as template (diluted 6/16 aliquot), kan F phosphothio and kan R primers, KAPA, 56˚C anneal and 90 sec elongation
</LI><LI>PCR of ZFB-wp-hisura: ZFB-wp-hisura as template (diluted 6/16 aliquot), ZFB-wp-f and URA3 R phosphothio primers, KAPA, 60˚C anneal and 90 sec elongation
</LI></UL><H3><SPAN class="mw-headline" id="Team_TolC_7">Team TolC</SPAN></H3><UL><LI><B>PCR ECNR2</B></LI><LI>Ran PCR on four colonies from lambda red recombination results, ECNR2 liquid culture from Church lab, and colony from original ECNR2 plate
</LI><LI>Grew each up in LB Amp liquid culture
</LI><LI>Tomorrow the gel will tell us whether the insertion was successful and also if the ECNR2 cultures are pure
</LI></UL><LI><B>PCR Noah's MAGE on rpoZ knockout</B></LI><LI>Picked 15 colonies from each plate with ECNR2 without TolC and ECNR2 with TolC
</LI><LI>Created liquid cultures of all 30 colonies picked and growing up over night
</LI><LI>PCR will be ready to send out tomorrow for sequencing
</LI><LI><B>More kan-ZFB-wp insertion construct needed</B></LI><LI>Ran PCR in order to obtain more kan-ZFB-wp insertion construct from the kan-ZFB-wp-His3-PyrF construct
</LI><LI>Placed on gel and used gel extraction in order to purify
</LI><LI>10.7 ng/
</LI><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">KAN-ZFB-wp construct with tolC homology 7/13/11</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Team_Web_Design_3">Team Web Design</SPAN></H3><UL><LI>Safety, safety, safety!
</LI><LI>Received safety confirmation letter
</LI><LI>Finalized project description
</LI><LI>Updated our iGEM wiki page for the upcoming deadlines (Project description and safety, due July 15th)</LI></UL><DIV id="714" style="display:none"><H2><SPAN class="mw-headline" id="July_14th">July 14th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_6">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Results_of_the_expression_plasmid_transformation">Results of the expression plasmid transformation</SPAN></H4><P>Only samples 78 to 82 had colonies after overnight incubation--these were the same colonies that were electroporated, instead of chemically transformed. Because we only added 1 ul of our cells to the plates, it is possible that this was not enough sample for us to see colonies, especially since chemical transformation is not as efficient as electroporation. The samples on the LB+spec plates had what appeared to be unspecific growth around the edges. This may be because the spec did not diffuse evenly. We will be re-plating all of our samples later today, on new LB+spec plates.
</P><H4><SPAN class="mw-headline" id="PCR_of_product_from_isothermal_assembly_of_expression_plasmid">PCR of product from isothermal assembly of expression plasmid</SPAN></H4><P>We also ran a PCR on the product of the three-part isothermal assembly (omega+linker, ultramers, spec backbone). We used the cross-junction primers and the same recipe as our PCR <A href="#PCR_of_expression_plasmid_cross-junction">on Tuesday</A>. Once again we expected a product of 1.4 kb.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">1.2% E-gel of our ultramer isothermal assembly reaction product</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>There were fairly sharp bands underneath the band we expected. Additionally, the expected 1.4 kb band was not visible for samples 80 and 81, although they were successfully grown on plates. Sample 85 had no bands, not even primers. 
</P><P>Since 80 and 81 grew on our plates yesterday, we continued with our plan of re-plating our original transformations, especially since the chemically transformed cells do have the appropriate isothermal assembly product to insert. We re-plated all the transformations on the LB+spec plates made today, and will leave them overnight in the 37* incubator. We also made cultures from the colonies on plates 78-82 to mini-prep tomorrow.
</P><H4><SPAN class="mw-headline" id="Rerunning_of_PCR_products_.28ultramers_and_isothermal_assembly.29">Rerunning of PCR products (ultramers and isothermal assembly)</SPAN></H4><P>In addition, we decided to see if the streaking was possibly caused by the gel instead of a PCR problem, and so we ran three of our products in a 1% agarose gel instead of an E-gel, with the results below.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Products ran in an agarose gel.  Note that wells 78-80 on the left side of the gel are empty due to sample mysteriously floating out of the well.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Team_Web_Design_4">Team Web Design</SPAN></H3><UL><LI>Intro video: brief (~1:30) Flash video explaining our project (Our project in 30 sec? 1 min?)
</LI><LI>Interview Joung/Persikov-- videotape (documentary style) for Human Practices page
</LI><LI>Human Practices
</LI><LI>Broader societal impacts
</LI><LI>Gene therapy, personalized medicine
</LI></UL><H3><SPAN class="mw-headline" id="Team_Wolfe_5">Team Wolfe</SPAN></H3><P><B>thio kan and thio ZFB-wp-hisura PCR:</B></P><UL><LI>Ran overnight PCR of kan cassette and ZFB-wp-hisura on E gel: looks great!
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">thio kan and thio ZFB-wp-hisura PCR 7/14/11</DIV></DIV></DIV><UL><LI>PCR purification with Qiagen kit:
</LI><LI>added an additional 300µL buffer PB to adjust the pH
</LI><LI>thio kan cassette: 59.9ng/µL 260/280=1.90
</LI><LI>thio ZFB-wp-hisura: 100.9ng/µL 260/280=1.89
</LI></UL><LI>Overlap PCR:
</LI><LI>4 rxns: two used undiluted template (60ng thio kan cassette and 101ng thio ZFB-wp-hisura), two used template diluted 1:10
</LI><LI>KAPA, 56˚C anneal, 90 sec elongation, 25 cycles
</LI><LI>primers (kan f phosphothio and ura3 r phosphothio) added after 10 cycles
</LI><LI>Results: PCR worked but with tons of side products so we will try to optimize the procedure. Expected product = 3Kb, notable side products = 2Kb, 4Kb. N.B. The samples were accidentally loaded while the gel was running so they are staggered on the gel.
</LI><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">thio kan-ZFB-wp-hisura overlap PCR (with loading error)</DIV></DIV></DIV><UL><LI>Gel stab PCR
</LI><LI> DNA from the 3Kb e-gel band were removed using the stabbing technique and placed in 4 respective reaction tubes (diluted in 20µL water)
</LI><LI> The 4 samples were separated into template dilutions of 20X, 100X, 200X, and 400X and phosphothio primers for PCR to amplify the desired 3Kb kan-ZFB-wp-hisura sequence. PCR results will be reported tomorrow.
</LI></UL><LI>Template PCR followed by PCR with primers:
</LI><OL><LI> PCR just the template (1µL diluted thio kan cassette and 1µL diluted thio ZFB-wp-hisura) with no primers, KAPA, 12 cycles, 60˚C anneal, 90 sec elongation
</LI><LI> take PCR product and use it as template for new PCR either undiluted, diluted 1:4, 1:16, 1:64, 1:256; kan f phosphothio and ura3 r phosphothio primers, 25 cycles, 59˚C anneal, 90 sec elongation
</LI></OL><P><B>pSR01 phenotype:</B></P><UL><LI>Just to be sure, we will assemble cultures in a flat-bottomed 96 well plate and use a plate reader to track the growth curves.
</LI><LI>2 colonies ∆HisB∆pyrF∆rpoZ, 10 colonies transformed with pSR01
</LI><LI>media: LB+amp, NM+his, NM, NM+3-AT (1mM), LB+amp+IPTG(100µM), NM+his+IPTG, NM+IPTG, NM+3-AT+IPTG
</LI><LI>plate reader: absorbency of 600nm, measurements every 10 min, shaking 560 sec before each measurement (settings saved in iGEM folder on computer in tissue culture room)
</LI></UL><H3><SPAN class="mw-headline" id="Team_TolC_8">Team TolC</SPAN></H3><UL><LI>PCR done of 4 colonies picked from KAN plate (with ECNR2 after recombineering with the KAN-ZFB-wp construct. Running an E-gel showed that the insert was not present. Since the KAN plates were old, it might be speculated that the plates were old, and so, the colonies were not selected for KAN resistance.
</LI><LI>Primers were TolC_seq_F and TolC_seq_R.
</LI><LI>Without insertion these two primers will create 449 bp sequence, and with insertion length will be 1650 bp
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">TolC insert check 1 7/14/11</DIV></DIV></DIV><UL><LI>Some of the original culture after recombineering, 1ml solution spun down, and resuspended in 100μl of LB, and plated on new KAN plates. To check, one of the old KAN plates was plated the same way and put in at 2:30pm. (These plates will be checked tomorrow morning.)
</LI></UL><LI>PCR done of a few of the rpoZ MAGE with stop codon insertions. Oligo name: rpoZ_MAGE_KO (Noah's oligo) (Noah's plates, after two cycles of MAGE, on both ECNR2 and ECNR2ΔTolC). The side product in the gel could be due to a low annealing temperature of 53°C, so we should do a gradient PCR (e.g.55-66°C) to get the most product the next time. </LI></DIV><DIV id="715" style="display:none"><H2><SPAN class="mw-headline" id="July_15th">July 15th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_Wolfe_6">Team Wolfe</SPAN></H3><P><B>Thio kan-ZFB-wp-hisura optimizing PCR results:</B></P><UL><LI>The gel stab PCR produced a product of the right size, but the 2kb side product was still brighter. The 2-PCR protocol also still had substantial side products but at least they did not completely overshadow the real product. We will accordingly go ahead and do a gel extraction on the 2-PCR samples so that we can try lambda red with them next week.
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Different attempts and thio kan-ZFB-hisura PCR 7/15/11</DIV></DIV></DIV><P><B>pSR01 growth phenotype results:</B></P><UL><LI>Unfortunately the computer hooked up to the plate reader spontaneously restarted during the night to download updates, so we lost all the data. Just by looking at the plate, it still seems like IPTG is inhibiting growth. The ∆HisB∆pyrF∆rpoZ controls also did not grow, probably because the plate the colonies were taken from was too old, so we will restreak from the glycerol stock and set up the plate reader again this weekend.
</LI></UL><H3><SPAN class="mw-headline" id="Team_ZF_7">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="PCR_of_electro_comp_expression_plasmids">PCR of electro comp expression plasmids</SPAN></H4><P>We performed a mini-prep of the cultures that we grew overnight of the electrocompetent cells. We then performed a PCR on the product of the mini-prep, checking for the 1.4 kb cross junction to confirm the presence of our omega+ultramers+spec plasmid. We used the same recipe as the PCR we performed <A href="#PCR_of_expression_plasmid_cross-junction">on Tuesday</A>.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Description of gel below.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>No bands appeared for PCR peformed on a 1 to 10 dilution of the cultures. Miniprep product from colonies 78, 79, and 82 had the expected band at 1.4 kb, while 80 and 81 had a band that was slightly smaller than the the expected product size. We saw a similar band in our <A href="#PCR_of_product_from_isothermal_assembly_of_expression_plasmid">PCR of the isothermal assembly reaction product</A>. We are unsure exactly what is causing this band. If either the omega subunit or ultramers were missing from the plasmid, a similar sized product would result since both are ~350 bp. We are curious if a plasmid can still be formed if one of these components were missing. 
</P><P>We checked the concentrations of our miniprep product to see if that would explain our problem. 
</P><PRE>ID     Conc. (ng/ul)     260/280     1.4 kb band?
78      16.12             2.10            Y
79      41.64             1.64            Y
80       5.84             2.01            N
81      61.24             1.62            N
82      20.25             1.72            Y
</PRE><P>We can't tell whether the concentration of the miniprep product contributed to the strange band we saw. 81 had the highest concentration but still didn't work. We decided to try plating 4 more colonies from 80 and 81, to see if any of them have the appropriate band. 
</P><P>We also decided to make final plates of cultures 78, 79, and 82, using 2 ul of culture and 50 ul of LB+spec. They were plated on new spec plates. We will incubate them at 37* overnight.
</P><H4><SPAN class="mw-headline" id="Checking_the_expression_plasmid_plates">Checking the expression plasmid plates</SPAN></H4><P>All the plates had colonies on them, although there were still areas of unspecific growth, presumably because of uneven dilution of the spec into the LB plates.
</P><P>We made cultures of all the chem comp cells (77, 83, 84, 85) and the electro comp cell that did not have the correct insertion (80, 81). We picked 4 colonies from each plate, resulting in a total of 24 cultures. We have left them in the 37* shaker overnight, and tomorrow plan on performing a miniprep and PCR to check for the cross-junction. 
</P><H4><SPAN class="mw-headline" id="PCR_to_check_for_omega.2Bultramers">PCR to check for omega+ultramers</SPAN></H4><P>We performed a PCR on the five electro comp miniprep products, to find out what may be missing. We used the omega_F+pLacO (index: 73) as our forward primer, and each of the respective reverse ultramers as our reverse primers (index: 78-82). We had an annealing temperature of 58* and an extension time of 30 seconds, and the expected product size is 646 bp. 
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Team ZF results in lanes 8-12</DIV></DIV></DIV></TD></TR></TBODY></TABLE>
Strangely, we did not get results for sample 78 (FH bottom), even though we got a band there in a PCR earlier today that used the junction primers.  We got positive results for samples 79 and 82, and everything else seems to check out with these two samples (bands present in the correct size with junction primers from both isothermal assembly and miniprep templates).  However, we are still having problems with samples 80 and 81, which may be a problem with the colonies that we picked.  For now, we will assume that the isothermal assembly worked, due to colonies growing on the spec plates for samples 80 and 81, and that we had simply picked bad colonies.  We will find out tomorrow if other colonies from the same plate give us the same results.</DIV><DIV id="716" style="display:none"><H2><SPAN class="mw-headline" id="July_16th">July 16th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_8">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Colony_minipreps">Colony minipreps</SPAN></H4><P>Minipreps were performed on all of the samples except for a few where nothing grew in the LB/spec media.  These exceptions included 85.3, 85.4, 84.2, 81.2, and 81.3.  Possible reasons for no growth for these samples may include: (1) The colony was somehow not a transformed colony and was a cheater (possibly picked from a spot where there was a smaller amount of spec in the unevenly distributed dish), and (2) the colony did not get picked up properly with the pipet tip.
</P><P>The concentrations of the DNA after miniprep were somewhat low, and a full table of these concentrations and DNA purities can be found below:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Plasmid concentrations</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="PCR_on_minipreps_to_check_cross_junction">PCR on minipreps to check cross junction</SPAN></H4><P>PCR was done on the miniprep products to check for the cross junction length, expected band size of ~1.4 kb.  The protocol used was the same as the previous time.  In addition to PCR-ing the miniprep products, a positive control was also included, which was the finished Zif268 + omega + spec backbone plasmid (this was the first successful junction PCR we obtained, and so it should work again).
</P><H4><SPAN class="mw-headline" id="Replating_final_samples_for_78.2C_79.2C_82">Replating final samples for 78, 79, 82</SPAN></H4>
Strangely, nothing grew on the plates that we seeded yesterday with 2ul of bacteria.  We seeded plates again for samples 78, 79, and 82 with 15ul of transformed bacteria instead, and hopefully these should grow by tomorrow.</DIV><DIV id="717" style="display:none"><H2><SPAN class="mw-headline" id="July_17th">July 17th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_9">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Replating_final_samples.2C_take_2">Replating final samples, take 2</SPAN></H4><P>We realized that the plates we seeded yesterday (July 16) were seeded from the transformation bacteria tubes, which has a mix of bacteria that may/may not have taken up the plasmid.  These plates therefore cannot serve as the final stock plates.  We took the E. coli from our initial colony picking and put them in LB + spec to grow to saturation tomorrow, where we will replate again with those bacteria (for which the good PCR results should correlate).
</P><H4><SPAN class="mw-headline" id="Gel_analysis_of_miniprep_junction_PCR">Gel analysis of miniprep junction PCR</SPAN></H4><P>The miniprep junction PCR products from yesterday were ran on an agarose gel, with the results below:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">PCR Results, expected band at ~1.4 kb</DIV></DIV></DIV></TD></TR></TBODY></TABLE>
While all the bands are between 1000 and 1650 kb, there are slight variations in the size for which we are unable to account. We will send these out for sequencing, to try to figure out what is going on.</DIV><DIV id="718" style="display:none"><H2><SPAN class="mw-headline" id="July_18th">July 18th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_10">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Sending_expression_plasmids_out_for_sequencing">Sending expression plasmids out for sequencing</SPAN></H4><P>We decided to send out the expression plasmids that had bands at approximately the correct size out for sequencing. We originally wanted to sequence the plasmids, but our miniprep concentrations were not high enough. Apparently, we should be getting high yields from minipreps, but thus far our yields have been around 50 ng/ul at the highest.
</P><P>We decided to sequence the unpurified PCR products instead. In order to do this we had to re-do the PCRs for Zif268, FH bottom (78), CB top (79), Myc 981 (82). Below are our 1% e-gel results, verifying that we had appropriately sized bands:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gel of cross junction of Zif268, FH bottom, CB top, Myc 981</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>Our final samples sent out for sequencing are:
</P><PRE>Index     Sample ID             Index     Sample ID
 1         77_1                  11        80_3
 2         77_2                  12        80_4
 3         77_3                  13        -
 4         77_4                  14        83_2
 5         84_1                  15        83_3
 6         84_4                  16        83_4
 7         81_1                  17        Zif268
 8         81_4                  18        FH_bottom
 9         80_1                  19        CB_top
 10        80_2                  20        Myc_981
</PRE><H4><SPAN class="mw-headline" id="Making_more_cultures_and_plates_of_our_expression_plasmids">Making more cultures and plates of our expression plasmids</SPAN></H4><P>We picked two more colonies from plate 85 today and made 5 ml cultures from them. We also made 2 fresh cultures from the old 85 cultures, by taking 50 ul of old culture and 5 ml LB+spec. We will leave them overnight in the 37* shaker and perform minipreps and PCRs on them tomorrow. 
</P><P>We made positive control cultures from 81.1, 81.4, 80.1, and 80.2, in order to test our minipreps tomorrow. They reactions have a wide range of concentrations post-miniprep, and we are curious to see if the same will happen after performing a miniprep a second time.
</P><P>Another way to figure out if our expression plasmids contain the appropriate insert is by using a restriction enzyme to cut it and then performing a PCR on the linear DNA. We had previously inserted 2 cutsites on our plasmids, Bsa1 (for insertion of finger 1) and Xba1. The lab already has Xba1, and we ordered Bsa1. Xba1 can be affected by methylation if it is preceded by 'GA' or followed by 'TC' <A href="http://www.neb.com/nebecomm/products/faqproductR0145.asp#874" class="external autonumber" rel="nofollow">[19]</A>; we checked and our sequences should not be affected.
</P><P>
We also re-plated 78, 79, and 82 today, one set of plates with 1 ul of culture, and the other set with 10 ul.
</P><H4><SPAN class="mw-headline" id="Creating_the_96_well_test_plate">Creating the 96 well test plate</SPAN></H4><H3><SPAN class="mw-headline" id="Team_Wolfe_7">Team Wolfe</SPAN></H3><P><B>SNOWMAN!</B></P><P><B>EcNR2 lambda red (Noah's prep):</B></P><UL><LI>Noah remade the kan-ZFB-wp-hisura insert for us with the kan cassette in the opposite orientation from the his3 and ura3 genes. (The kan cassette is under a strong promoter and does not have a transcriptional terminator, so it's possible that RNA polymerase might keep transcribing through the end of kan and into his3.)  He also used it for lambda red in EcNR2, and a couple of plates look like they may have a couple colonies.
</LI><LI>picked 4 colonies, diluted in 20µL H20 and used 1µL as template for 1529620 locus PCR. The rest was grown in LB+kan.
</LI></UL><P><B>Lambda red with Noah's kanZFBhisura:</B></P><UL><LI>We recently discovered that the kan plates we had been using were very old, and that might be why we got that weird crystalline lawn of colony-like things.  We will retry lambda red in the ∆hisB∆pyrF∆rpoZ +pKD46 strain and see if this solves the problem
</LI><LI>1.5mL cells divided in two: half for kan-his-ura (300ng) and half as a negative control (water)
</LI></UL><P><B>Thio insert:</B></P><UL><LI>In case lambda red also does not work, we will try to add phosphothioate bases to the insert, but only to the lagging strand 5' end (so only the Ura3 side). Since our primers will have very different melting temps, we will try 2 different reactions:
</LI><LI>Touchdown PCR:
</LI><LI>template: Noah's kan-ZFB-hisura (163ng/µL) diluted to ~10ng per reaction (4 reactions)
</LI><LI>primers: kan_f+homology, Ura3 R phosphothioate
</LI><LI>2 cycles at each annealing temperature (67, 66...57) for 11 cycles followed by 15 more cycles at 57; 3 min elongation
</LI></UL><LI>Gradient PCR:
</LI><LI>same as above but with 12 reactions, annealing temperatures from 57-67˚C
</LI><LI>Results: most of the gradient reactions produced a faint but clean band at 3kb.  The touch down PCR had a stronger 3kb band but also lots of side products.
</LI><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Thio kan-ZFB-hisura (Noah's) gradient part 1 7/18/11</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Team_TolC_9">Team TolC</SPAN></H3><H4><SPAN class="mw-headline" id="Primers">Primers</SPAN></H4><UL><LI>Created and ordered primers in order to test the presence of MAGE rpoZ knockout oligo
</LI><LI>Primers made are named 
</LI><LI>rpoZ_MAGE_KO plus 4, rpoZ_MAGE_KO minus 4, which have all 4 of the stop codons
</LI><LI>rpoZ_MAGE_KO plus 2, and rpoZ_MAGE_KO minus 2, which have only the first two stop codons.
</LI></UL><H4><SPAN class="mw-headline" id="Kan_Plates">Kan Plates</SPAN></H4><UL><LI>Performed control experiment on Kan plates by growing up liquid culture from glycerol stock and then spreading on Kan plate in order to determine the speed of growth and intensity of the presence of Kan on the plates
</LI><LI>Will check on these tomorrow.
</LI></UL><H4><SPAN class="mw-headline" id="Kan-ZFB-wp_insertion">Kan-ZFB-wp insertion</SPAN></H4><UL><LI>Picked 5 colonies from ZF090 and ZF091 plates for a total of 10, ZF092 had too few colonies to pick from
</LI><LI>Placed colonies in 20 μL of water and used 1 μl for a 20 μL PCR reaction
</LI><LI>Then took 10 μL from 20 μL dilution and added to 100 μL of LB in 96 well plate and put in 30°C fridge, at 3:19 pm
</LI><LI>Same protocol as <A href="https://2011.igem.org/Team:Harvard/Protocols#PCR" class="external text" rel="nofollow">KAPA PCR</A> with the following changes made
</LI><LI>Step 3, gradient for annealing, 55°C - 77°C, for a range of 55,57,59,61 and 63°C for the two rows of 5 PCR tubes each (placed leftmost)
</LI><LI>Step 4, Extension, 1 min
</LI><LI>Step 5, 25 cycles total
</LI></UL><LI>Name: TOLCGRAD, PCR5 machine.</LI></DIV><DIV id="719" style="display:none"><H2><SPAN class="mw-headline" id="July_19th">July 19th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_11">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Checking_on_plates_78.2C_79.2C_82">Checking on plates 78, 79, 82</SPAN></H4><P>Both the 10 ul and 1 ul essentially had a lawn of bacteria growing on them. We re-plated these samples, using 1/100 ul and 1/10000 ul of bacteria in 50 ul of LB. We will leave these plates to incubate overnight at 37*.
</P><P>We also made a culture from the old 85 culture, which we will plate tomorrow.
</P><H4><SPAN class="mw-headline" id="Minipreps_of_colonies_and_controls">Minipreps of colonies and controls</SPAN></H4><P>Our minipreps from Saturday had low yields, especially for culture 85. We suspect this is why the PCR [[#Gel analysis of miniprep junction PCR on Sunday did not work for this sample. Unfortunately, 85.3 and 85.4 did not grow even after being left in the shaker overnight. These were two cultures we made by picking new colonies from plate 85. This may have occurred because we did not actually pick the colony, or the colony was a cheater. Thus, we performed minipreps on 80.1, 80.2, 81.1, 81.4 (the controls, chosen for their wide range of concentrations after our first miniprep), and 85.1 and 85.2. 
</P><P>In order to figure our why our minipreps were resulting in unexpectedly low yields and increase our yields:
</P><UL><LI>We tested the pH of our water and Buffer EB. 
</LI><LI>The pH of our aliquot of water was ~4-5 (too acidic...we are unsure why)
</LI><LI>The pH of our Buffer EB was ~7. 
</LI><LI>Ideally, the DNA should be eluted by a solution of pH 7, so we decided to use Buffer EB during the elution step.
</LI></UL><LI>We also checked the copy number of the origin of replication used in our plasmids. ColE1 apparently has a low copy number, which might also explain our low yields.
</LI><LI>We had made 5 ml of culture in an attempt to increase our yields. 
</LI><P>Our yields were larger than our first round of minipreps, although the purities were rather low. We decided to proceed anyway. 
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Concentration and purity of our miniprep product</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="PCR_of_miniprep_products">PCR of miniprep products</SPAN></H4><P>We then ran a PCR for the cross-junction on all 6 (80.1, 80.2, 81.1, 81.4, 85.1, 85.2) samples, using our usual cross junction protocol. 
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gel results, expected product size of 1.4 kb</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>We saw bands for samples 80.1, 81.1, 81.4, 85.1, and a very faint band for 85.2.  The strange thing about these bands is that they appeared to be right around 1650 bp, larger than our 1.4 kb target size.  We are unsure of the identity of these bands and will send them out for sequencing.
</P><H4><SPAN class="mw-headline" id="Starting_a_new_plate_for_sample_85">Starting a new plate for sample 85</SPAN></H4><P>Since we have been having such bad luck with sample 85, we seeded 2 ml of LB+spec with 100 ul of the transformed bacteria for sample 85.  This is in preparation for making a new plate of sample 85 (the old plate was not a good spec plate and had lots of unspecific growth as well as very few colonies, so if we used this plate for more colony picking our choices would be very limited).  This new plate should also give us more luck with picking colonies and having them actually grow in the LB+spec.
</P><H4><SPAN class="mw-headline" id="Designing_ultramers_for_Zif268.2C_OZ052.2C_and_OZ123">Designing ultramers for Zif268, OZ052, and OZ123</SPAN></H4><P>In preparation for designing our 96-well practice plate, we designed ultramers for our three positive controls Zif268, OZ052, and OZ123.  These ultramers contain the Type II binding sites as well as the F2/F3 fingers.  Since the practice plate will contain oligos that encode for F1, we can use these three positive controls to practice and ensure that we secure the technique of using the Type II binding sites to swap in F1 into our expression plasmids.  The ultramers have not yet been ordered.
</P><H3><SPAN class="mw-headline" id="Team_TolC_10">Team TolC</SPAN></H3><H4><SPAN class="mw-headline" id="PCR_to_test_insertion_of_KAN-ZFB-wp_into_ECNR2">PCR to test insertion of KAN-ZFB-wp into ECNR2</SPAN></H4><UL><LI>Made 26 dilutions of culture from three different recombineering plates
</LI><LI>10 cultures from Kan_tolC_locus_F1 (ZF090 plate) primer recombination
</LI><LI>10 cultures from Kan_tolC_locus_F2 (ZF091 plate) primer recombination
</LI><LI>5 cultures from Kan_tolC_locus_F3 (ZF092 plate) primer reecombination
</LI><LI>1 culture from an old Kan plate with recombination from last week
</LI></UL><LI>5 of ZF090, 6 of ZF091, and 5 ZF092 were placed in a PCR with 58˚C annealing temperature
</LI><LI>5 of ZF090, 4 of ZF091, and 1 from old plate were run in a PCR with 62˚C annealing temperature
</LI><LI>primers used were TolC_seq_F and TolC_seq_R
</LI><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">EcNR2 lambda red 1529620 PCR 7/19/11</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">EcNR2 lambda red 1529620 PCR 7/19/11</DIV></DIV></DIV><UL><LI>Chose colonies based on the intensity of the bands and grew up cultures of ZF091-2d, ZF091-4d, ZF090-1d, ZF091-5d, ZF090-1s, ZF090-2s, ZF090-3s, and ZF091-3s
</LI></UL><H4><SPAN class="mw-headline" id="Sequencing_of_Kan-ZFB-wp_insertion">Sequencing of Kan-ZFB-wp insertion</SPAN></H4><UL><LI>Prepared 16 PCR products of the 8 cultures we chose to grow up in order to send out for sequencing
</LI><LI>Used TolC_Seq_F and ZFB-wp-F primers
</LI><LI>Bands looked a little small when ran out gel for longer
</LI></UL><H4><SPAN class="mw-headline" id="New_MAGE_oligo">New MAGE oligo</SPAN></H4><UL><LI>Constructed a new oligo for MAGE knockout of rpoZ, with oligos more than 4 mutations and substitutions the efficiency drops from 15% down to below 5%, so we created a rpoZ knockout oligo with only 2 substitutions
</LI><LI>The new MAGE rpoZ knockout is rpoZ_MAGE_Subs
</LI></UL><LI>Also ordered primers to test the presence of the knockout rpoZ_MAGE_Subs1+, rpoZ_MAGE_Subs1-, rpoZ_MAGE_Subs2+, rpoZ_MAGE_Subs2-
</LI><H3><SPAN class="mw-headline" id="Team_Wolfe_8">Team Wolfe</SPAN></H3><P><B>EcNR2 lambda red colony PCR results:</B></P><UL><LI>PCR showed that colonies 1-3 had the insert while 4 did not. Yay! (Note that in the gel image, we accidentally cut off the 300bp band in lane 4)
</LI><LI>We will grow up a reinoculated culture of colony 1 for a glycerol stock. If the Wolfe strain keeps not working, we at least have the insert in an EcNR2 line and can hopefully MAGE out the other genes without too much difficulty.
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">EcNR2 lambda red 1529620 PCR 7/19/11</DIV></DIV></DIV><P><B>pSR01 growth phenotype:</B></P><UL><LI>We will try to used the plate reader again to compare the growth phenotypes of ∆HisB∆pyrF∆rpoZ and ∆HisB∆pyrF∆rpoZ+pSR01. Used the same media  as last time but since we're using a 48 well plate, each well has 250µL media. We used 5 pSR01 colonies and one selection strain colony as a control.
</LI><LI>N.B.: this plate we are using is not necessarily meant for OD tests like this and may give some variable readings.
</LI></UL><P><B>Lambda red results:</B></P><UL><LI>Yesterday's lambda red, which used Noah's kan-ZFB-wp-hisura, has yet to produce colonies. We will leave the plates in longer in case any come up later.
</LI></UL><P><B>Thio kanZFB:</B></P><UL><LI>Since yesterday's lambda red may not have worked, we need to get the thio kan up and running. We ran out the touchdown PCR samples (since they have more DNA) and will gel extract.
</LI><LI>57.7ng/µL 260/280=1.98
</LI><LI>69.3ng/µL 260/280=1.98</LI></UL><DIV id="720" style="display:none"><H2><SPAN class="mw-headline" id="July_20th">July 20th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_12">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Checking_on_1.2F100_ul_and_1.2F10000_ul_78.2C_79.2C_and_82_plates">Checking on 1/100 ul and 1/10000 ul 78, 79, and 82 plates</SPAN></H4><P>We got colonies on all the plates,  but the 1/100 ul plates look a bit crowded. On the other hand, the 1/10000 plates only have a few colonies. We decided to save both plates, and decide which to use later. 
</P><UL><LI>Based on these results, a 1/1000 ul dilution might be best. 
</LI></UL><H4><SPAN class="mw-headline" id="Check_on_85_culture">Check on 85 culture</SPAN></H4><P>There was no growth in our culture.
</P><P>Later: It ends up that our ultramer from 85 (Pos Con 77) had an error in it. Instead of coding for fingers 2 and 3, it codes for fingers 1 and 2. We ordered a new ultramer, and we no longer need to work on 85. 
</P><H4><SPAN class="mw-headline" id="Finalizing_the_practice_plate">Finalizing the practice plate</SPAN></H4><P>We decided on using a practice plate with 24 oligos, consisting of 18 target sequences, and 6 controls. Our 6 controls are Zif268, OZ052, OZ123, and 3 sequences from the CODA table. We will need to construct the expression plasmid for Zif268, OZ052, and OZ123. Their ultramers were ordered today. 
</P><P>Our 18 targets were GNN/TNN sequences, which are more likely to have a binder, and were distributed among the backbones. The target and control groups each have their own set of primers, to facilitate picking them from the pool.
</P><H4><SPAN class="mw-headline" id="Expression_Plasmid_Sequencing_Results">Expression Plasmid Sequencing Results</SPAN></H4><P>We got back our sequencing results:
</P><TABLE border="1"><TBODY><TR><TD> Name</TD><TD>ID</TD><TD>Final?</TD><TD>Notes
</TD></TR><TR><TD style="background: green"><B>FH Top (77)</B></TD><TD>1</TD><TD>N</TD><TD>1 snp
</TD></TR><TR><TD>2</TD><TD>N</TD><TD>5 snps
</TD></TR><TR><TD><B>3</B></TD><TD><B>Y</B></TD><TD><B>G -&gt; C. Still Ala</B></TD></TR><TR><TD>4</TD><TD>N</TD><TD>1 snp
</TD></TR><TR><TD style="background: red"><B>FH Bottom (78)</B></TD><TD>1</TD><TD>N</TD><TD>1 Deletion at 1746, last nucleotide of F of FQRICMRN
</TD></TR><TR><TD style="background: red"><B>CB Top (79)</B></TD><TD>1</TD><TD>N</TD><TD>A lot of deletions/mistakes in f2/f3 region
</TD></TR><TR><TD style="background: green"><B>CB Bottom (80)</B></TD><TD><B>1</B></TD><TD><B>Y</B></TD></TR><TR><TD><B>2</B></TD><TD><B>Y</B></TD><TD><B>G-&gt;C. Stiil Ala</B></TD></TR><TR><TD> 3</TD><TD>N</TD><TD>deletion 1613-1631
</TD></TR><TR><TD>4</TD><TD>N</TD><TD>many mistakes at end, may be ok
</TD></TR><TR><TD style="background: green"><B>Myc 198 (81)</B></TD><TD><B>1</B></TD><TD><B>Y</B></TD></TR><TR><TD><B>4</B></TD><TD><B>Y</B></TD><TD><B>G-&gt;C. Still Ala.</B></TD></TR><TR><TD style="background: green"><B>Myc 981 (82)</B></TD><TD><B>1</B></TD><TD><B>Y</B></TD></TR><TR><TD style="background: green"><B>Pos Con 16 (83)</B></TD><TD><B> 4</B></TD><TD><B>Y</B></TD><TD><B>G-&gt;C. Still Ala.</B></TD></TR><TR><TD>2</TD><TD>N</TD><TD>A lot of deletions/mistakes in f2/f3 region
</TD></TR><TR><TD>3</TD><TD>N</TD><TD>A lot of deletions/mistakes in f2/f3 region
</TD></TR><TR><TD style="background: green"><B>Pos Con 55 (84)</B></TD><TD><B>1</B></TD><TD><B>Y (?)</B></TD><TD><B>Right sequence, but messy trace</B></TD></TR><TR><TD> 4</TD><TD>N</TD><TD>at 547: deletion at type II binding site
<P>at 665: deletion at beginning of F3
</P></TD></TR><TR><TD style="background: green"><B>Zif 268</B></TD><TD><B>1</B></TD><TD><B>Y</B></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Summary:</B></TD></TR><TR><TD><B>Good:</B> 77.3, 80.1, 80.2, 81.1, 81.4, 82, 83, 84.1 (?), Zif268
</TD></TR><TR><TD><B>Bad (Redo):</B> 78, 79
</TD></TR></TBODY></TABLE><P>We picked 4 new colonies for 78, and 79 since their sequences had many errors. We left these cultures overnight at 37* in the shaker.
</P><H4><SPAN class="mw-headline" id="Construction_of_OZ052_and_OZ123:_Joining_the_Finger_with_Homology_regions">Construction of OZ052 and OZ123: Joining the Finger with Homology regions</SPAN></H4><P>Previously, we did a PCR to reassemble the ultramers for OZ052 and OZ123, but they only coded strictly for F1/F2/F3 and did not have any overhang homology with any surrounding code.  Thus, we performed PCR to add these homology overhangs so that we can integrate them with the omega subunit as well as the spec backbone to create a final expression plasmid.
</P><P>The program we used was 55* annealing temperature, and a 15 second extension time.  Gel results pictured below.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Overlap PCR to extend ultramers with homologies</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>We had an expected product size of 400 bp., which we can see in the gel. Bands are a little weird since there are bands heavier than the heaviest theoretical band (which is the strong product).  We will do a gel extraction tomorrow to get rid of these bands to obtain pure product for use in the isothermal assembly.
</P><H3><SPAN class="mw-headline" id="Team_TolC_11">Team TolC</SPAN></H3><H4><SPAN class="mw-headline" id="rpoZ_MAGE_knockout">rpoZ MAGE knockout</SPAN></H4><UL><LI>Ran two rounds of MAGE on 4 ECNR2 cultures with the kan-ZFB-wp insertion using Noah's rpoZ knockout oligo
</LI><LI>Grew cultures up to mid-log in kan+amp and LB
</LI><LI>Four cultures are ZF091-2d, ZF091-4d, ZF090-1d, and ZF090-5d
</LI><LI>Electroporator read 5.4 ms for all four cultures in first round
</LI><LI>Could not do a third round of MAGE since the E.Coli do not appear to have recovered, and are nowhere near midlog (MAGE round 2 was done by 5:15pm and the cultures were checked at 8:30pm). Will let the culture go on overnight.
</LI><LI>Put 50ul of a 1:100 dilution of each culture on Amp plates. Used a spreader for plates 1&amp;2 and glass beads for plates 3 &amp; 4.
</LI></UL><H4><SPAN class="mw-headline" id="Glycerol_Stocks">Glycerol Stocks</SPAN></H4><UL><LI>Made 4 glycerol stocks of the cultures sent to be sequenced
</LI><LI>ZF090-1s, ZF090-2s, ZF090-3s, ZF091-3s
</LI></UL><H4><SPAN class="mw-headline" id="convinced_that_the_gradient_PCR_machine_is_out_of_order.2C_ought_to_inform_Andrew">convinced that the gradient PCR machine is out of order, ought to inform Andrew</SPAN></H4><H3><SPAN class="mw-headline" id="Team_Wolfe_9">Team Wolfe</SPAN></H3><P><B>IPTG Plate and the Plate Reader</B></P><UL><LI>Technical delays have allowed us only a few hours worth of plate reading for a 46-well plate with selection strain + pSR01 in LB+Amp +/- IPTG, NM+His +/- IPTG, NM +/- IPTG, and NM+3-AT +/- IPTG. Strains used: ∆HisB∆pyrF∆rpoz+pSR01 (5 colonies) and ∆HisB∆pyrF∆rpoz (1 colony).
</LI><LI>The plate had a noticeable white residue under the lid around around the edges. We believe this was due to flaking of the plastic cover from vibration in the plate reader.
</LI><LI> Cell growth in the plate suggested that IPTG is killing our cells since few cultures survived with IPTG and those that did (LB+Amp) grew less than the equivalent cultures without IPTG.  
</LI></UL><P><B>New Plate Without IPTG</B></P><UL><LI> On plate reader at 11:30AM
</LI><LI>Strains used: Water as control, Selection (Wolfe) Strain, Selection Strain + pSR01
</LI><LI>Media Used: LB+Tet (LB+Amp for SS+pSR01),NM+His, NM, NM+3-AT (1µM), NM+3-AT (2.5µM), NM+3-AT (5µM).
</LI></UL><P><B>Lambda Red</B>
Same procedure as usual
</P><UL><LI>Added ~350ng of Thio-Kan-ZFB-wp-His3-Ura3
</LI><LI>Used &quot;new cuvettes&quot;
</LI><LI>Negative water control
</LI></UL><P><B>Plate Reader Update</B>
TEMPORARILY OUT OF SERVICE
</P><P><B>Your worst nightmare</B></P><A href="http://www.youtube.com/watch?v=v4o8MDjuu-A" class="external free" rel="nofollow">http://www.youtube.com/watch?v=v4o8MDjuu-A</A></DIV><DIV id="721" style="display:none"><H2><SPAN class="mw-headline" id="July_21st">July 21st</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_13">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Checking_colonies_78_and_79">Checking colonies 78 and 79</SPAN></H4><P>All of the cultures grew overnight, so we performed a miniprep on them to obtain the plasmids. 
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Plasmid concentrations after miniprep</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>Our concentrations were better than they have been in the past; notably, after the addition of buffer P2, our solution actually turned a bright blue (indicating lysis of the cells), which we had not seen before.
</P><P>We then ran a PCR on our miniprep product, to check for our desired insert. We used our usual protocol for a PCR of the expression plasmid cross-junction.  
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">PCR results, note that some lanes do not show strong bands</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>Some of the products did not show up very strongly, but we sent them out for sequencing in case it was a PCR/gel error.
</P><H4><SPAN class="mw-headline" id="Sending_out_our_expression_colonies_for_sequencing">Sending out our expression colonies for sequencing</SPAN></H4><P>We sent out the samples from yesterday that had no more than 1 SNP for reverse sequencing. In addition, we also sent out 78 and 79 for forward and reverse sequencing. 
</P><UL><LI>Reverse sequencing: 77.1, 77.3, 77.4, 78.1, 80.1, 80.2, 80.4, 81.1, 81.4, 82.1, 83.4, 84.1, 84.4, Zif268
</LI><LI>Forward and reverse sequencing: 781, 78.2, 78.3, 78.4, 79.1, 79.2, 79.3, 79.4
</LI></UL><P>For sequencing, we used the forward primer pZE23G_3581_F CCAACCTTACCAGAGGGCGC (Junction forward) and the reverse primer pZE23G_2123_R CGAACGACCTACACCGAACT (Junction reverse)
</P><P>We did a PCR of the sequences from yesterday that we want sequenced again. 
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">PCR results, looking good</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>The bands were clear, and we will send these products to be sequenced.
</P><H4><SPAN class="mw-headline" id="Gel_Extraction_and_Isothermal_Assembly_of_OZ_fingers.2Bhomology">Gel Extraction and Isothermal Assembly of OZ fingers+homology</SPAN></H4><P>We ran a gel extraction of our PCR product from yesterday, so that we could use it for isothermal assembly later.  Strangely, we got bands that were larger than our largest theoretical product.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Prepare yourself to be gel-extracted</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>We had the following yields from our gel extraction:
</P><PRE> ID         Conc. (ng/ul)            260/280
 OZ052      96.3                     1.93
 OZ123      125.5                    1.91
</PRE><P>Based on these concentrations, we used the following recipes for our isothermal reaction:
</P><UL><LI>OZ052:
</LI><LI>Spec: 0.63 ul
</LI><LI>OZ052: 1.56 ul
</LI><LI>Omega: 0.13 ul
</LI><LI>Water: 1.56 ul
</LI><LI>Isothermal assembly solution: 7.5 ul 
</LI></UL><LI>OZ123: 
</LI><LI>Spec: 0.63 ul
</LI><LI>OZ123: 0.14 ul
</LI><LI>Omega: 0.13 ul
</LI><LI>Water: 1.60 ul
</LI><LI>Isothermal assembly solution: 7.5 ul
</LI><P>We left the isothermal assembly product in the -20* freezer, and will perform the transformation tomorrow.
</P><H3><SPAN class="mw-headline" id="Team_Tolc_12">Team Tolc</SPAN></H3><H3><SPAN class="mw-headline" id="Team_Wolfe_10">Team Wolfe</SPAN></H3><UL><LI>Lambda red culture did not grow colonies overnight on a kanamycin plate. This may be due to a low time (4.8ms) during the electroporation.
</LI><LI>Repeated lambda red using the same procedure. Got a similar time constant and noticed that the water negative control had more cells than the culture containing the insert, but we went ahead and plated 1mL of each on kan plates.
</LI><LI>Also replated the 7/20 lambda red culture to see if we can get any good cells (1.5mL).</LI></UL><DIV id="722" style="display:none"><H2><SPAN class="mw-headline" id="July_22nd">July 22nd</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_14">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Sequencing_Results_of_Expression_Plasmids">Sequencing Results of Expression Plasmids</SPAN></H4><P>The good news is that we managed to get a perfect sequence for each of our plasmids.  The bad news is that we do not know which sequences are the matches, since the names are all scrambled for some reason (i.e. sample 78.x does not align with FH bottom like it should, but instead aligns perfectly with another plasmid sequence).  We will thus need to resend our samples for sequencing to be able to choose the correct plasmids that match.  Below is a summary of the scrambled data from the sequencing we sent in yesterday:
</P><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>ID</B></TD><TD align="center" style="background:#f0f0f0;"><B>Is a perfect match with</B></TD><TD align="center" style="background:#f0f0f0;"><B>Notes</B></TD></TR><TR><TD> Myc 198 (81)</TD><TD>78.2, 78.3</TD></TR><TR><TD> FH Top (77)</TD><TD>81.4, Myc 981</TD><TD>83.4 has 1 deletion
</TD></TR><TR><TD> CB Bottom (80)</TD><TD>84.4</TD><TD>78.1 has 5 deletions; Zif has 1 deletion
</TD></TR><TR><TD> FH bottom (78)</TD><TD>n/a</TD><TD>84.1 deletion at beginning of F3
</TD></TR><TR><TD> CB Top (79)</TD><TD>79.3</TD></TR><TR><TD> Myc 981 (82)</TD><TD>78.4</TD></TR><TR><TD> Zif 268</TD><TD>79.4</TD></TR><TR><TD> Pos Con 55 (83)</TD><TD>79.2</TD><TD>79.3 has deletion at beginning of F3
</TD></TR><TR><TD> Pos Con 16</TD><TD>79.1</TD></TR></TBODY></TABLE><P>
To this end, we redid the PCRs for everything in preparation for the resequencing, using the junction primers and the previous protocol.  The resulting gels can be observed below:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">E-gel showing bands for samples 78.1-78.4 and 79.1-79.4, expected size of 1.4 kb</DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Remaining samples, expected size of 1.4 kb</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>From these PCRs, all the reactions look like they produced bands of the right size, a good indicator that the isothermal assemblies worked.  Only sequencing will tell whether they are truly the products that we are looking for.
</P><P>We sent these samples out for sequencing, making sure that all the samples were in the appropriate tubes.
</P><H4><SPAN class="mw-headline" id="Transformation_for_OZ_plasmids">Transformation for OZ plasmids</SPAN></H4><P>Today, we took the results of the isothermal assembly from yesterday and did chemical transformation to introduce the plasmids into the Top 10 ChemComp bacteria, using <A href="https://2011.igem.org/Team:Harvard/Protocols#Cultures" class="external text" rel="nofollow">our protocol for chemical transformations</A>. They were left to recover for approximately 1.5 hours. We plated 2 dilutions, 10 ul and 50 ul, and left them overnight at 37* in the incubator.
</P><H3><SPAN class="mw-headline" id="Team_Wolfe_11">Team Wolfe</SPAN></H3><UL><LI>Yesterday's lambda red plate had several large colonies--but so did the negative control plate. We will go ahead and do a PCR of the 1529620 locus (as previously described) on 12 of the colonies in case one of them happens to have the insert. The colonies were also grown up simultaneously in LB+kan.
</LI><LI>Results: all of the colonies were cheaters
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">1529620 locus PCR 7/22/11</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">1529620 locus PCR part 2 7/22/11</DIV></DIV></DIV><UL><LI>In case these colonies do not have the insert, as we suspect, we will repeat lambda red with the thio kan-ZFB-wp-his3-ura3 insert
</LI><LI>1mL culture for each condition, 210 ng DNA
</LI><LI>time constant for thio insert sample was 5.0
</LI></UL><H3><SPAN class="mw-headline" id="Team_TolC_13">Team TolC</SPAN></H3><H4><SPAN class="mw-headline" id="Gel_for_Jun.27s_MAGE_rpoZ.2C_rowD">Gel for Jun's MAGE rpoZ, rowD</SPAN></H4><UL><LI>The gel appeared to have bands, so we are running a second PCR to confirm it</LI></UL><DIV id="723" style="display:none"><H2><SPAN class="mw-headline" id="July_23rd">July 23rd</SPAN></H2><H3><SPAN class="mw-headline" id="Team_Wolfe_12">Team Wolfe</SPAN></H3><UL><LI>Lambda red results: no real colonies, only cheaters.</LI></UL><DIV id="724" style="display:none"><H2><SPAN class="mw-headline" id="July_24th">July 24th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_15">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Sequencing_Results">Sequencing Results</SPAN></H4><P>We got back our sequencing results from the samples we sent out on Friday, and the results are summarized in the table below.  The checkmarks indicate whether the sample is adequate for usage or not.
</P><TABLE><TBODY><TR><TD style="background:#f0f0f0;"><B>FH top (77)</B></TD><TD style="background:#f0f0f0;"><B>Notes</B></TD></TR><TR><TD> ✓</TD><TD>77.1</TD><TD>Only G -&gt; C in ala
</TD></TR><TR><TD> ✓</TD><TD>77.3</TD><TD>Only G -&gt; C in ala
</TD></TR><TR><TD>77.4</TD><TD>Single deletion in F3
</TD></TR><TR><TD style="background:#f0f0f0;"><B>FH bottom (78)</B></TD></TR><TR><TD>78.1</TD><TD>Many, many SNPs throughout, oddly F and R differ
</TD></TR><TR><TD>78.2</TD><TD>Many, many SNPSs, R starts in a strange place
</TD></TR><TR><TD>78.3</TD><TD>Four deletions throughout on both F and R
</TD></TR><TR><TD>78.4</TD><TD>Many, many SNPs throughout
</TD></TR><TR><TD>Original</TD><TD>Only one SNP in F3, but unacceptable
</TD></TR><TR><TD style="background:#f0f0f0;"><B>CB top (79)</B></TD></TR><TR><TD>79.1</TD><TD>Many, many SNPs and deletions throughout, F and R the same
</TD></TR><TR><TD> ✓</TD><TD>79.2</TD><TD>No SNPs
</TD></TR><TR><TD> ✓</TD><TD>79.3</TD><TD>No SNPs
</TD></TR><TR><TD>79.4</TD><TD>G -&gt; C in ala, deletion in the Q of the F3 \&quot;FSQ\&quot;
</TD></TR><TR><TD style="background:#f0f0f0;"><B>CB bottom (80)</B></TD></TR><TR><TD> ✓</TD><TD>80.1</TD><TD>No SNPs
</TD></TR><TR><TD>80.2</TD><TD>G -&gt; C in ala, deletion in the homology region shortly after F3 ends
</TD></TR><TR><TD>80.4</TD><TD>G -&gt; C in ala, five deletions in F3/homology region
</TD></TR><TR><TD style="background:#f0f0f0;"><B>Myc 198 (81)</B></TD></TR><TR><TD> ✓</TD><TD>81.1</TD><TD>No SNPs
</TD></TR><TR><TD> ✓</TD><TD>81.4</TD><TD>Only G -&gt; C in ala
</TD></TR><TR><TD style="background:#f0f0f0;"><B>Myc 981 (82)</B></TD></TR><TR><TD> ✓</TD><TD>82.1</TD><TD>Only G -&gt; C in ala
</TD></TR><TR><TD style="background:#f0f0f0;"><B>Pos con 16 (83)</B></TD></TR><TR><TD> ✓</TD><TD>83.4</TD><TD>Only G -&gt; C in ala
</TD></TR><TR><TD style="background:#f0f0f0;"><B>Pos con 55 (84)</B></TD></TR><TR><TD> ✓</TD><TD>84.1</TD><TD>Only G -&gt; C in ala
</TD></TR><TR><TD>84.4</TD><TD>G -&gt; C in ala, two deletions
</TD></TR><TR><TD style="background:#f0f0f0;"><B>Zif268</B></TD></TR><TR><TD> ✓</TD><TD>Zif268</TD><TD>No SNPs
</TD></TR></TBODY></TABLE></DIV><DIV id="725" style="display:none"><H2><SPAN class="mw-headline" id="July_25th">July 25th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_TolC_14">Team TolC</SPAN></H3><H4><SPAN class="mw-headline" id="Sequencing_of_Jun.27s_rpoZ_MAGE_knockout">Sequencing of Jun's rpoZ MAGE knockout</SPAN></H4><UL><LI>Prepared six products and primers for sequencing
</LI><LI>Sending Jun colony 4, PCR's 1-mutant_R, 2-mutant_R, 3-mutant_R, 4-mutant_R, 2-mutant_Int_R, and 4-mutant_Int_R
</LI><LI>delayed until tomorrow (July 25, 2011) because pickup for the day already done.
</LI></UL><H4><SPAN class="mw-headline" id="Lambda_Red_Recombination_of_zeocine_into_TolC.28assistance_from_Naomi.2C_Cub_team_member.29">Lambda Red Recombination of zeocine into TolC(assistance from Naomi, Cub team member)</SPAN></H4><UL><LI>PCR out zeocin cassette (replacing the rpoZ, for selection) from Wolfe Strain
</LI><LI>Used primers rpoZ_F and rpoz_R, expecting a band about 1500bp? Total length about 939bp, including the promoters, according to [<A href="http://rothlab.ucdavis.edu/drugs/zeo.html" class="external text" rel="nofollow">Roth lab @ UC Davis, Zeocin</A>]
</LI><LI>Ran 8, 25 µL solutions with KAPA
</LI><LI>PCR protocol(PCR 7 machine, &quot;ZEOC&quot;)
</LI><LI>95˚ C, for 5 min
</LI><LI>98˚ C, for 20 sec
</LI><LI>62˚ C, for 15 sec
</LI><LI>72˚ C, for 45 sec
</LI><LI>Repeat 29 times
</LI><LI>72˚ C, for 5 min
</LI><LI>4˚ C, forever
</LI></UL><LI>Gel extracted the PCR product <B>NOTE. Since the gel, was so clean, we may do a PCR clean up next time, instead of running the PCR product through a gel for gel extraction. Just make sure to use these exact same parameters.</B></LI><LI>Nanodrop
</LI><LI>156.2 ng/µL and 107.6 ng/µL, each in 30ul of water (distilled, deionised- molecular grade)
</LI><LI>Performed lambda red recombination
</LI><LI>Time constant of 5.6 ms
</LI><H4><SPAN class="mw-headline" id="rpoZ_MAGE_knockout_attempt...yet_again">rpoZ MAGE knockout attempt...yet again</SPAN></H4><UL><LI>Used new oligo to perform a MAGE knockout of rpoZ
</LI><LI>Oligo named rpoZ_MAGE_Subs
</LI></UL><LI>Electroporator read 5.2 ms
</LI><LI>Allowed for 3 hours of growth and spread on zeocine plate
</LI><H3><SPAN class="mw-headline" id="Team_Wolfe_Cub">Team Wolfe Cub</SPAN></H3><UL><LI>Plate reader is still being repaired--we will have to wait to check the growth curves
</LI><LI>Since plans A and B are not working well at the moment, we will go ahead and start preparing for plan C (using a plasmid). Ordered primers that will clone out the pSB4K5 (a biobrick vector with a kan cassette, <A href="http://partsregistry.org/Part:pSB4K5" class="external free" rel="nofollow">http://partsregistry.org/Part:pSB4K5</A>) backbone with homology to the ZFB-wp-his3-ura3 construct. We will then connect the two with Gibson assembly.
</LI><LI>To this end, we need to use the biobrick DNA sent to us to transform chemically competent E coli.  Unfortunately the 384-well plates that they are stored in could not be found because Andrew is out of town--hopefully they will turn up tomorrow.
</LI></UL><LI>For the rest of the day, we joined team TolC for gel extraction, MAGE, and lambda red!
</LI><H3><SPAN class="mw-headline" id="Team_ZF_16">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Result_of_OZ052_and_OZ123_Transformation">Result of OZ052 and OZ123 Transformation</SPAN></H4><P>Last Friday we plated out the transformed bacteria for the two OZ controls in 10 ul and 50 ul volumes.  They both yielded colonies, but the 10 ul plates only had a handful while the 50 ul plates had around 15-20 colonies each.  We picked four colonies from each plate and grew them in LB/spec so that tomorrow we can perform a miniprep and send them off for sequencing.  We should know the final verdict by Wednesday regarding which ones are good or bad.
</P><H4><SPAN class="mw-headline" id="Glycerol_stockmaking">Glycerol stockmaking</SPAN></H4><P>We got the sequencing results back from last Friday, and took the following colonies to make into glycerol stock solutions: 77.1, 79.2, 80.1, 81.1 (x2 for both FH bottom and Myc 198), 82.1, 83.4, 84.1, Zif268.  These colonies are a subset of the &quot;good&quot; colonies from our sequencing results from Sunday.  We mixed 120 ul of bacteria in mid-log with 300 ul of 80% glycerol to make the final solution, which was stored at -80 degrees C.
</P><H4><SPAN class="mw-headline" id="Restriction_enzyme_test">Restriction enzyme test</SPAN></H4><P>We tested both XbaI and BsaI on the junction PCR product using colony 79.4 as a template (which was used because it was a &quot;bad&quot; sequence overall but had intact BsaI and XbaI cut sites).  The junction PCR product is 1326 bp, and we expected to see bands of 593 and 739 bp with XbaI, and 717 and 578 bp with BsaI.  Our restriction digest recipe consisted of the following per reaction:
</P><UL><LI>1 ul 10X buffer (NE biolabs buffer 4)
</LI><LI>7.9 ul H2O
</LI><LI>1 ul DNA (~100 ng)
</LI><LI>1 ul enzyme (~2 units)
</LI><LI>1 ul BSA (100x)
</LI></UL><P>The two gels we ran for this digest showed that XbaI did NOT work, which may be due to XbaI being too old.  However, we did get double bands both times for BsaI.  Our egel was strange because of uneven loading, and so we were unable to determine the true size of both bands, but the agarose gel we ran later showed the two bands in the right positions.  The two gels can be seen below:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">E-gel of our first digestion. The gel looks unusual (notice the bubble-like formation), so we performed a second digestion.</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">1% agarose gel of our second digestion. Although the bands are faint, we can see the appropriately sized bands for Bsa1.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Ultramer_Construction_of_Pos_Con_77_and_Other_F2.2FF3_positive_controls">Ultramer Construction of Pos Con 77 and Other F2/F3 positive controls</SPAN></H4><P>We redid the ultramer overlap PCR for Pos Con 77, this time with the correct reverse sequence.  We also assembled ultramers for Zif268, OZ052 and OZ123 (these are the plasmids that will have the type II cut sites in them along with F2/F3 only, as opposed to having the entire finger already assembled).  For the Zif268 reaction, an annealing temperature of 65*C was used, and for the rest, the annealing temperature was 58*C.  For all reactions, the extension time was 15 seconds.
</P><P>The following primers were used for each reaction (index taken from the Dropbox primer index):
</P><UL><LI>Pos con 77 - 76, 122
</LI><LI>Zif268 - 123, 124
</LI><LI>OZ052 - 76, 125
</LI><LI>OZ123 - 76, 126</LI></UL><DIV id="726" style="display:none"><H2><SPAN class="mw-headline" id="July_26">July 26</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_17">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Isothermal_Assembly_of_Pos_Con_77.2C_Zif_268.2C_and_OZs">Isothermal Assembly of Pos Con 77, Zif 268, and OZs</SPAN></H4><P>Today we checked the PCR products of the ultramer reaction from yesterday.  They did not look good.  We repeated the reaction using a touchdown PCR beginning with an annealing temperature of 70 degrees and decreasing 2 degrees every other cycle until hitting a minimum of 58 degrees, and we brought the total number of cycles to 25.  Extension time was 30 seconds.  This looked great.  The pictures for the two reactions can be seen below:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Our first PCR was extremely streaky, so we decided to redo it.</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">We did a touchdown PCR, and our product looks better, although it's still streaky.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>Before the isothermal assembly, we did a PCR purification and measured the concentrations of the ultramer reaction products:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Concentrations of our PCR product after PCR purification. This was used in our isothermal assembly.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>We then performed an isothermal assembly, using the following mix for Pos con 77, OZ052, and OZ123:
</P><UL><LI>0.63 ul of spec backbone at ~157 ng/ul
</LI><LI>0.3 ul of ultramer (~50 ng/ul)
</LI><LI>0.13 ul of omega subunit/linker
</LI><LI>1.44 ul of H2O
</LI><LI>7.5 ul of isothermal mix
</LI></UL><P>For Zif268:
</P><UL><LI>0.63 ul of spec backbone
</LI><LI>0.19 ul of Zif (~90 ng/ul)
</LI><LI>0.13 ul of omega/linker
</LI><LI>1.55 of H2O
</LI><LI>7.5 ul of isothermal mix
</LI></UL><P>This was left at 50 degrees for an hour, then stored at 4 degrees.
</P><H4><SPAN class="mw-headline" id="Prepare_control_OZs_for_sequencing">Prepare control OZs for sequencing</SPAN></H4><P>We performed a miniprep on the eight samples for sequencing, obtaining the following concentrations:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Our OZ concentrations after miniprep.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>The miniprep lysis step looked good, and the samples turned blue after lysis.
</P><P>We then performed a junction PCR using the previous protocol, and checked the products on a gel:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gel of our PCR product that we sent out for sequencing.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>They looked good, so we sent them out for sequencing.  Hopefully we will have our final OZ control plasmids for tomorrow.
</P><H4><SPAN class="mw-headline" id="Practice_Plate_resuspension">Practice Plate resuspension</SPAN></H4><P>We resuspended the practice plate oligos in 40 ul of water to produce a 100 uM solution, taking 10 ul of the final solutions into 90 ul of water in separate tubes to create 10 uM working concentrations.  When we receive our tag primers, we should be able to test the presence of the oligos in each tube as well as amplify them in preparation for integration into the expression plasmids.
</P><H4><SPAN class="mw-headline" id="Colony_finalization">Colony finalization</SPAN></H4><P>For the colonies that we plated yesterday, we saw that the plates all had good growth on them and so we wrapped the plates in parafilm and put them at 4 degrees for storage.  1/1000 ul from saturated culture seems to be a good amount to plate.
</P><H3><SPAN class="mw-headline" id="Team_TolC_15">Team TolC</SPAN></H3><H4><SPAN class="mw-headline" id="Zeocin_insertion_into_ECNR2">Zeocin insertion into ECNR2</SPAN></H4><UL><LI>Plated 3 mL of our recombination culture on zeocin plate, and also plated 2 mL of MAGE results on zeocin plate as a control
</LI><LI>The recombination plate had about 15 colonies around 9 30am, and the MAGE plate had none
</LI></UL><LI>Continued by PCR of 4 colonies on recombination plate
</LI><LI>Each template was analysed with presence of different sized bands for rpoZ (using rpoZ_F and rpoZ_R) and presence of Zeocin cassette (using rpoZ_R and zeocin_R)
</LI><LI>Gel resulted with the following:
</LI><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Zeocin insert check 7/26/11</DIV></DIV></DIV><UL><LI><B>SUCCESS!!! Yippee!!!</B></LI><LI>NOTE. Since there was a bit of non specific annealing, we might want to use a higher annealing temperature of 64C instead of 62C
</LI></UL><LI>Decided to run PCR for the Kan insertion just to make sure it was still present in the 4 colonies we chose, and it is!!!
</LI><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Kan-ZFB-wp insert check 7/26/11</DIV></DIV></DIV><P>WERE READY TO ROOOCCKKK!!!!!
</P><H4><SPAN class="mw-headline" id="MAGE_rpoZ_knockout">MAGE rpoZ knockout</SPAN></H4><UL><LI>Plated 1:10 and 1:100 dilution of MAGE results, and this morning at 10 am, they both had cells and in appropriate proportions
</LI><LI>If above PCR and gel of zeocin insertion does not appear to be successful, then we will run 50 PCR on 50 different colonies on the 1:100 dilution plate of MAGE rpoZ knockout
</LI></UL><H3><SPAN class="mw-headline" id="Team_Wolfe_13">Team Wolfe</SPAN></H3><UL><LI>Plate reader: trying to get in contact with Christian Daly who runs the Spectramax plate reader in the Bauer facility (Northwest basement). Hopefully we will get access soon.
</LI><LI>Selection system plasmid: still trying to get hold of the biobricks. Designed Quikchange primers to change the zinc finger binding sites.
</LI><LI>Lambda red recombination: gave lambda red one last chance, this time with a positive control
</LI><LI>negative control (water): plated 1mL
</LI><LI>positive control (kan cassette from Wyss): 50ng DNA, plated 1mL and 100uL
</LI><LI>thio kan-ZFB-wp-his3-ura3: 180ng DNA, plated 1mL</LI></UL><DIV id="727" style="display:none"><H2><SPAN class="mw-headline" id="July_27th">July 27th</SPAN></H2><UL><LI>Impromptu meeting for all teams, TFs, and advisors in NW labs basement, to discuss &quot;how we will move forward to test the selection, now that we have a TolC strain that should work for selection and a His3Ura3 plasmid-based selection that is almost complete,&quot; and to lay out and assign a lab to-do for iGEM parts, selection systems, and what to do in preparation for The Chip to arrive/backup plan in case it arrives too late (see notes in the dropbox under &quot;Selection Meeting 7-27-11.docx&quot;)
</LI></UL><H3><SPAN class="mw-headline" id="Team_TolC_16">Team TolC</SPAN></H3><H4><SPAN class="mw-headline" id="Culture_of_TolC.2BKan-ZFB-wp.2BZeocin">Culture of TolC+Kan-ZFB-wp+Zeocin</SPAN></H4><UL><LI>Did not realize that colonies needed to be grown up without antibiotic before being grown in Kanamycin, so all the three colonies from yesterday passed away and didn't make it through the night :(, so we grew up 4 more colonies and ran PCR and gel to solidify the presence of the necessary insertions
</LI></UL><H3><SPAN class="mw-headline" id="Team_ZF_18">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Checking_the_OZ_sequencing_results">Checking the OZ sequencing results</SPAN></H4><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>OZ052</B></TD><TD align="center" style="background:#f0f0f0;"><B>Notes</B></TD></TR><TR><TD> ✓</TD><TD>1</TD><TD>No snps
</TD></TR><TR><TD> ✓</TD><TD>2</TD><TD>No snps
</TD></TR><TR><TD> ✓</TD><TD>3</TD><TD>No snps
</TD></TR><TR><TD>4</TD><TD>Deletion at F1
</TD></TR><TR><TD align="center" style="background:#f0f0f0;"><B>OZ123</B></TD><TD align="center" style="background:#f0f0f0;"><B>Notes</B></TD></TR><TR><TD> ✓</TD><TD>1</TD><TD>No snps
</TD></TR><TR><TD>2</TD><TD>deletion in omega linker
</TD></TR><TR><TD> ✓</TD><TD>3</TD><TD>No snps
</TD></TR><TR><TD> ✓</TD><TD>4</TD><TD>No snps
</TD></TR></TBODY></TABLE><P>We made glycerol stocks and final plates from 0Z052.1 and OZ123.1.
</P><H4><SPAN class="mw-headline" id="Transformation_of_Pos_Con_77.2C_OZs.2C_Zif_268">Transformation of Pos Con 77, OZs, Zif 268</SPAN></H4><P>We transformed Top 10 One Shot Electrocomp bacteria with our isothermal assembly product from yesterday. We let the cells recover for an hour, and plated 50 ul, leaving them at 37* overnight.
</P><H3><SPAN class="mw-headline" id="Team_Wolfe_14">Team Wolfe</SPAN></H3><UL><LI>Morning meeting: Since TolC has gotten their selection system onto the genome, our priorities are now going to be making the plasmid-based Wolfe selection system and getting data on the plate reader. We also will eventually try to use a genome-based selection system but using the EcNR2 strain (oligos have been ordered)
</LI></UL><P><B>Plate Reader:</B></P><UL><LI>We took a trip downstairs to B2 and were trained to use the Bioscreen plate reader in the Bauer Core facility.  We now have access to a working plate reader!
</LI><LI>Bioscreen uses a 100-well honeycomb plate, so the organization was as follows:
</LI><LI>Well 1, 11, 21, etc: LB+tet
</LI><LI>Well 2, 12, 22, etc: NM+his
</LI><LI>Well 3, 13, 23, etc: NM
</LI><LI>Well 4, 14, 24, etc: NM+1mM 3-AT
</LI><LI>Well 5, 15, 25, etc: NM+2.5mM 3-AT
</LI><LI>Well 6, 16, 26, etc: NM+5mM 3-AT
</LI><LI>Well 7, 17, 27, etc: LB+tet+100µM IPTG
</LI><LI>Well 8, 18, 28, etc: NM+IPTG
</LI><LI>Well 9, 19, 29, etc: NM+1mM 3-AT+IPTG
</LI><LI>Well 10, 20, 30, etc: NM+5mM 3-AT+IPTG
</LI><LI>Column 1: no cells
</LI><LI>Column 2-3: ∆HisB∆pyrF∆rpoZ
</LI><LI>Columns 4-10: ∆HisB∆pyrF∆rpoZ+pSR01
</LI></UL><P><B>Electrocomp Transformation</B></P><UL><LI>Used Team ZF's protocol to transform electrocompetent cells with two of the pSB4K5 plasmids found in our biobricks plates (2:17H and 3:11P)
</LI><LI>Used 10µL ddH2O to resuspend DNA in biobrick plate wells and transferred to eppendorf.
</LI><LI>2µL DNA (which, interestingly, was red...)
</LI><LI>plated 10µL and 100µL of each on kan plates
</LI></UL><P><B>Quikchange primers:</B></P><UL><LI>Realized that we may need to consider the bases on either side of the zinc finger binding site, so we will check this and put off ordering another day.</LI></UL><DIV id="728" style="display:none"><H2><SPAN class="mw-headline" id="July_28th">July 28th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_Wolfe_15">Team Wolfe</SPAN></H3><P><B>Plate reader and pSR01:</B></P><UL><LI>the Bauer plate reader seems to have worked very well, bu† unfortunately none of our pSR01-transformed colonies grew.  This is probably because the plates were a couple weeks old.  Unfortunately our glycerol stock of the strain also is missing, so we will try growing up some of the colonies in case on of them works.  Otherwise, we will grow up ∆HisB∆pyrF∆rpoZ and transform again.
</LI><LI>Transformation: usual protocol, about 100µL DNA, recovery 2hrs, plate 1µL, 10µL, 100µL.
</LI></UL><P><B>pSB4K5:</B></P><UL><LI>The kan plates from the transformation all had colonies! We chose one from 2:17H and one from 3:11P to grow up in LB+kan for miniprep and glycerol stocks.
</LI><LI>Ordered quikchange primers: for OZ052 and OZ123 we added the context bases 5' and 3' of the binding sequence (found in OPEN supplementary materials).  Since CODA did not include the extra bases, we left them the same as the original plasmid (i.e. Zif268's).
</LI></UL><H3><SPAN class="mw-headline" id="Team_TolC_17">Team TolC</SPAN></H3><H4><SPAN class="mw-headline" id="cultures_of_ECNR2.2BKan-ZFB-wp.2Bzeocin_from_F9.2CF11.2C_for_glycerol_stocks_and_transformation">cultures of ECNR2+Kan-ZFB-wp+zeocin from F9,F11, for glycerol stocks and transformation</SPAN></H4><H4><SPAN class="mw-headline" id="Sequencing">Sequencing</SPAN></H4><UL><LI>Ran PCR and sent out sequencing to solidify the presence of the Kan+ZFB+wp and zeocin within the ECNR2 genome
</LI><LI>Used rpoZ_R, rpoZ_F for PCR of the zeocin cassette (and rpoZ_F for sequencing) and TolC_Seq_R, TolC_Seq_F for PCR of the Kan-ZFB-wp insert (and tolC_seq_F for sequencing)
</LI></UL><H4><SPAN class="mw-headline" id="Plasmid_transformation">Plasmid transformation</SPAN></H4><UL><LI>Performed transformation on the Zif268 plasmid into the ECNR2 culture
</LI><LI>Used 3 µL in a 50 µL dilution (49+1) of 1 ul of 55 ng/µL concentration of plasmids
</LI><LI>1.82 V for 5.00 ms according to the electroporator
</LI></UL><H3><SPAN class="mw-headline" id="Team_ZF_19">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Checking_on_Pos_Con_77.2C_OZs.2C_and_Zif_268_plates">Checking on Pos Con 77, OZs, and Zif 268 plates</SPAN></H4><P>Unfortunately, nothing grew on our plates from yesterday. Our time constants from the electroporation were rather low (~3.8), so 50 ul may not be enough to see growth. We decided to spin down the cells, resuspend them in 50 ul of LB, and re-plated this solution. We also re-did the transformation using ChemComp cells, and made 100 ul plates. Both sets of plates were left overnight at 37*.
</P><H4><SPAN class="mw-headline" id="Designing_the_reporter_strain">Designing the reporter strain</SPAN></H4><P>Instead of placing the GFP reporter on a separate plasmid, we decided to insert it on our expression plasmid, immediately after our zinc fingers, and before the transcription terminator. We placed a ribosome binding site (Shine Dalgarno sequence: AGGAGGTT before the GFP. We will only add the GFP to Zif 268 (whole) because if we can show through GFP that the selection system works for Zif 268, we can assume it also works for the other zinc fingers. This is easier and more cost effective than ordering 12 different primers for each of our plasmids. 
</P><P>We will construct the reporter strain by performing a PCR to get the plasmid in linear form. We will also perform a second PCR to get out GFP and add homology regions to our Zif 268 plasmid, as well as adding the Shine-Dalgarno sequence. The primers for these PCRs were ordered today. We'll then assemble these pieces to construct a whole plasmid, and transform our bacteria.
</P><H4><SPAN class="mw-headline" id="PCR_with_Practice_Plate">PCR with Practice Plate</SPAN></H4><P>We performed a PCR to amplify our F1s from the practice plate. We used our primers for CB bottom (primer index: 66 and 67) for the 18 targets and our primers for the controls (primer index: 72 and 73) for the 6 controls. We had an annealing temperature of 52 degrees and an extension time of 15 seconds.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Practice plate primer test</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>We got products for all of the oligos at our expected product size of 131 bp.
</P><H4><SPAN class="mw-headline" id="Glycerol_stocks_of_successful_plasmids">Glycerol stocks of successful plasmids</SPAN></H4>
We happened to check our glycerol stocks, and noticed that the glycerol and bacteria were not mixed thoroughly. We remade all of our glycerol stocks and put them into the -80*.</DIV><DIV id="729" style="display:none"><H2><SPAN class="mw-headline" id="July_29th">July 29th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_Wolfe_16">Team Wolfe</SPAN></H3><P><B>pSR01:</B></P><UL><LI>The colonies in the 96 well plate did grow overnight, but they were not saturated, implying that they were very unhealthy. We will PCR a few of them (1,2,3,6) just to see if they actually have the plasmid still
</LI><LI>Transformation plates look great--we will PCR 12 colonies to verify the plasmid is present
</LI><LI>PCR: colonies/culture diluted 1:20 as template (and colonies grown up simultaneously); M13_F and M13_R primers; 50˚C annealing and 3min elongation; 25 cycles
</LI><LI>Results: bands are similar to the ones seen the last time we did this PCR (and those samples had also been sequence-verified). These colonies should be good to go!
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">pSR01 check 7/29/11</DIV></DIV></DIV><P><B>pSB4K5:</B></P><UL><LI>Miniprepped both cultures. Yields weren't great (in the case of 2:17H, there actually was a spill) but they are still usable.
</LI><LI>2:17H: 6.4 ng/µL 260/280=2.2
</LI><LI>3:11P: 9.6ng/µL 260/280=2.4
</LI></UL><LI>PCR to add ZFB-wp-his3-ura3 homology:
</LI><LI>1µL of plasmid as template
</LI><LI>pSB4K5_F+ZFBhl and pSB4K5_R+URA3hl primers
</LI><LI>58˚C anneal, 3 min elongation, 25 cycles
</LI><LI>Results: the product was really really really faint--primer dimers were by far the bulk of the product.  We realized this is probably because one of the primers includes a Not1 restriction site, which is palindromic, 8 bases long, and all Cs and Gs.
</LI><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">pSB4K5 PCR to add homology 7/29/11</DIV></DIV></DIV><UL><LI>In order to counteract the primer dimers, we will try a gradient PCR with a range of higher annealing temperatures:
</LI><LI>same protocol as before except the annealing temp went from 58-65 with 8 samples (1=58, 8=65)
</LI><LI>Results: 3:11P did have visible product bands, but the primer dimers are still stronger
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">pSB4K5 gradient PCR 7/29/11</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Team_TolC_18">Team TolC</SPAN></H3><H4><SPAN class="mw-headline" id="Sequencing_of_Kan.2BZFB.2Bwp_and_Zeocin_insertions">Sequencing of Kan+ZFB+wp and Zeocin insertions</SPAN></H4><UL><LI>Sequencing results were successful and solidified presence of both insertions
</LI></UL><H4><SPAN class="mw-headline" id="Zif268_Plasmid_Insertion_into_ECNR2.2BKan-ZFB-wp.2Bzeocin">Zif268 Plasmid Insertion into ECNR2+Kan-ZFB-wp+zeocin</SPAN></H4><UL><LI>The plates had growth on them!!!
</LI><LI>So we chose 5 colonies and ran PCR on them to gel them out
</LI><LI>The gel shows success for colonies 2,3,4, and 5 that we chose!!!
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Zif268 plasmid insertion check 7/29/11</DIV></DIV></DIV><P><B>READY FOR SELECTION!!!</B></P><H3><SPAN class="mw-headline" id="Team_ZF_20">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Transformation_plating_results">Transformation plating results</SPAN></H4><P>We got no growth on any of the plated transformed bacteria for either the ElectroComp or ChemComp bacteria.  To check whether the isothermal assembly itself was the problem, we ran a junction PCR using our previous protocol on the isothermal assembly mix, along with a positive control (Myc981):
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">No bands visible for the isothermal assembly</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>As observed, there were no bands in the isothermal mix lanes while the positive control worked.  Thus, we can conclude that our isothermal assembly did not work, and we will need to repeat it again (currently waiting for more isothermal mix to arrive from Wyss).
</P><H4><SPAN class="mw-headline" id="Amplification_and_purification_of_practice_plate_oligos">Amplification and purification of practice plate oligos</SPAN></H4><P>In preparation for our swap reactions (switching the oligo F1s into our expression plasmid), we mapped out the following two experiments that we will need to perform:
</P><UL><LI>Testing the selection system by preparing our expression plasmids in parallel (i.e. creating 18 plasmids in separate reactions), and then combining them all at the end and transforming them into the selection strains.  This will allow us to cut and ligate the plasmids without much concern, but test whether we will be able to pick out the binder amongst the rest of the non-binders.
</LI><LI>Testing the cutting/ligation by combining all of the oligos first, and then carrying out the construction of the different expression plasmids all in the same tube.
</LI></UL><P>Note that the results for both will ultimately yield DNA libraries if they work (except that the first experiment is much more foolproof in terms of actually yielding a library than the second experiment).
</P><P>To carry out the first experiment, the following steps need to be undertaken:
</P><OL><LI>Perform PCRs on each of the practice plate oligos such that we end up with a relatively high concentration of DNA after PCR and PCR purification (over 40 ng/ul).
</LI><LI>Cut both plasmid backbone and oligos with BsaI type II restriction enzyme.
</LI><LI>PCR purify restriction products to get rid of enzyme and junk DNA.
</LI><LI>Run phosphatase reaction to prevent autoligation.
</LI><LI>Mix and ligate to get full plasmid.
</LI><LI>Some tips:
</LI><LI>Each ligation reaction should only have 10-15 ul volume.
</LI><LI>You should have 100 ng of backbone per reaction
</LI><LI>Since we are planning 18 reactions, we will need 1800 ng of backbone, which is several minipreps worth
</LI><LI>The backbone:oligo ratio should be appoximately 1:4
</LI><LI>In our case, the backbone is 2.2 kb and the insert is 131 bp, so for 100 ng of backbone there should be ~20 ng of insert
</LI><LI>Gel extract plasmid to get final library.
</LI></OL><P>Today, we did PCRs and PCR purifications for step one as outlined above.  We tested the concentrations of the results, displayed below:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">DNA concentrations</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>In order to have enough backbone to work with, we seeded some LB/spec with CB bottom bacteria in preparation for 4 minipreps of 3 mL each tomorrow.  This should definitely yield the 1800 ng of backbone we will need for the ligation.
</P><H4><SPAN class="mw-headline" id="Practice_plate_primer_tag_test">Practice plate primer tag test</SPAN></H4><P>We tested the primers to see if their selection of subpools was exclusive enough or not (i.e. to make sure that our control primers did not amplify the CB bottom sequences, etc.).  We ran a PCR crossing primers with DNA with the following results:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Strange results</DIV></DIV></DIV></TD></TR></TBODY></TABLE>
The first and third lanes are CB primer/CB DNA and control primer/control DNA respectively, which should yield bands.  Lanes two and four were control primer/CB DNA and CB primer/control DNA respectively, which should yield no bands.  The fact that we got bands for these lanes is a little bit worrying, and so we are repeating this PCR again to see if we get the same results when we run the gel tomorrow.</DIV><DIV id="730" style="display:none"><H2><SPAN class="mw-headline" id="July_30th">July 30th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_TolC_19">Team TolC</SPAN></H3><UL><LI>Checked the plate with ECNR2 with Zeocin knocked into the place of rpoZ. There are appear to be some 15-20 colonies on the plate. This strain is ready for transformation with the zif268 plasmid
</LI></UL><H3><SPAN class="mw-headline" id="Team_ZF_21">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="CB_bottom_miniprep">CB bottom miniprep</SPAN></H4><P>Today we performed a miniprep on the 4 tubes of CB bottom bacteria with 4 ml of culture in each tube.  6 ml of culture were eventually consolidated into a single spin tube during the miniprep, so this would be the equivalent of 4 minipreps in one tube.  Of note, after lysis (during addition of buffer P2), the tubes turned extremely blue, darker blue than we had ever seen before.  This would seem to indicate that the lysis step occurred very successfully.  However, even after consolidating four minipreps into one tube, we still got terrible concentration results for our samples, displayed below:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">DNA concentrations after miniprep</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>We will thus seed more LB/spec culture with CB bottom bacteria tomorrow, and perform the miniprep on Monday with TF supervision to hopefully troubleshoot some of the problems with low yield.
</P>
Also of note is that the bacteria were left on the shaker for approximately 23 hours at the time of miniprep, which is longer than the ideal 12-14 hours.</DIV><DIV id="731" style="display:none"><H2><SPAN class="mw-headline" id="July_31st">July 31st</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_22">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Primer_tag_test_two">Primer tag test two</SPAN></H4><P>On Friday, we repeated the PCR to test for sub-pool specificity of our primer oligo tags for the chip, using the same PCR protocol.  Today, we ran a gel to see if the results were any different, but alas, they were the same.  It seems that the primers for CB bottom also amplify out the control sequences, which means that our primer tags are not as specific as we thought they were.  The gel can be seen below, with the lanes arranged as follows: CB primer + CB DNA, control primer + CB DNA, control primer + control DNA, and control primer + CB DNA.  We would ideally expect bands in lanes 1 and 3 and no bands in lanes 2 and 4.  However, this was not the case:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">PCR repeat results</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>We might try doing touchdown PCR or raising the annealing temperature to get more specificity in the future.  It may also be worth ordering oligos to represent the other 5 subpools of targets and cross test the primers with each DNA subpool to make sure that the primers are specific enough to pull out only one subpool.  In the end though, this specificity is not critical to our success, and so it is ok if it doesn't eventually work.
</P><H4><SPAN class="mw-headline" id="Preparing_for_another_CB_bottom_miniprep">Preparing for another CB bottom miniprep</SPAN></H4>
Today we seeded 4 tubes of culture with 100 ul of CB bottom bacteria at saturation.  Each tube carried 4 ml of culture.  The tubes were put on the shaker at approximately 6:30 pm, so they will be ready for miniprep tomorrow morning after the lab meeting.</DIV><DIV id="801" style="display:none"><H2><SPAN class="mw-headline" id="August_1st">August 1st</SPAN></H2><H3><SPAN class="mw-headline" id="Team_Wolfe_17">Team Wolfe</SPAN></H3><P><B>Optimizing PCR reaction for pSB4K5:</B></P><P>The gradient PCR did work, but the yield was still very low. We will try a few different approaches to increase our PCR product.  All these reactions were done only with 3:11P (since 2:17H showed no product in the gradient).
</P><P>1) Gel Stab PCR:
</P><UL><LI>We ran out more of the gradient PCR product (see image below) and used it as the template for a new PCR
</LI><LI>Gel stab went into 20µL H2O; 66 C annealing, 3 min elongation; pSB4K5_F+ZFBhl and pSB4K5_R+URA3hl primers
</LI><LI>Results: unsuccessful. Perhaps the gel stab did not give an adequate amount of template.
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">7/29 pSB4K5 gradient samples for gel stab</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">pSB4K5 gel stab PCR results 8/1/11</DIV></DIV></DIV><P>2) Reverse Touchdown PCR:
</P><UL><LI>Because the primer initially anneals with 20 bases and then with 60, we will start with a low annealing temperature and then raise it as the overlapping region grows.
</LI><LI>58-66 C annealing, changing 1 degree per cycle (9 cycles); the rest of the 25 cycles were done at 66 C
</LI><LI>3 min elongation
</LI><LI>Results: one of the four reactions worked. This is very strange since the reactions should have been identical.
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">pSB4K5 reverse touchdown 8/1/11</DIV></DIV></DIV><P>3) Template concentration gradient PCR:
</P><UL><LI>The primer dimers may have been forming since there wasn't enough template to compete with the primer-primer interactions.
</LI><LI>All reagents were kept the same except template: 1µL, 1.5, 2, 2.5, 3, 3.5, or 4µL were used
</LI><LI>66 C anneal, 3 min elongation
</LI><LI>Results: only one of the reactions worked. We did use the gradient cycler which has not always worked well for us. We also noticed that when we removed the tubes from the cycler, most of the liquid had condensed on the lids.
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">pSB4K5 template concentration gradient PCR 8/1/11</DIV></DIV></DIV><P>4) Gradient PCR (overnight):
</P><UL><LI>Since our original gradient seemed to have been one of the more successful ways, we will repeat it but with a tighter temperature range
</LI><LI>Normal KAPA protocol, annealing 65-69 C, 3 min elongation
</LI></UL><P><B>ZFB-wp-his3-ura3:</B></P><UL><LI>For isothermal assembly we need a high concentration of each of the DNA components, so we decided to PCR more of our ZFB-wp-his3-ura3 insert and use the Qiagen Minelute kit to purify.
</LI><LI>primers: ZFB-wp-f and ZFB+kan_short_R; ZFB-wp-hisura template
</LI><LI>because of a substantial difference in primer melting temperatures, we did a 57-61 C annealing gradient; 90 sec elongation
</LI><LI>Results: all 8 conditions worked!
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">ZFB-wp-his3-ura3 gradient PCR 8/1/11</DIV></DIV></DIV><UL><LI>Minelute PCR purification of 4 of the reactions: yield=epic.
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">BEST YIELD EVER!</DIV></DIV></DIV><P><B>pSR01 plate reading:</B></P><UL><LI>We shared a 100-well plate with TolC and will take readings overnight with the following conditions:
</LI><LI>Wells 51, 61...91: LB+tet/amp
</LI><LI>Wells 52, 62...92: NM+his
</LI><LI>Wells 53, 63...93: NM
</LI><LI>Wells 54, 64...94: NM+1mM 3-AT
</LI><LI>Wells 55, 65...95: NM+2.5mM 3-AT
</LI><LI>Wells 56, 66...96: NM+5mM 3-AT
</LI><LI>Wells 57, 67...97: NM+his+100µM IPTG
</LI><LI>Wells 58, 68...98: NM+100µM IPTG
</LI><LI>Wells 59, 69...99: NM+1mM 3-AT+100µM IPTG
</LI><LI>Wells 60, 70...100: NM+5mM 3-AT+100µM IPTG
</LI><LI>51-60: empty
</LI><LI>61-70 and 71-80: inoculate with 1µL of selection strain culture
</LI><LI>81-90 and 91-100: inoculated with 1µL of selection strain + pSR01 culture
</LI></UL><H3><SPAN class="mw-headline" id="Team_ZF_23">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Restriction_enzyme_cut_test.2C_take_3">Restriction enzyme cut test, take 3</SPAN></H4><P>We repeated the restriction enzyme reaction once more to obtain a better and more definitive gel image, and we used the following formulation:
</P><UL><LI>8 ul of 1x BSA/Buffer 4
</LI><LI>2 ul of DNA (used the 79.4 junction PCR product again)
</LI><LI>0.5 ul of BsaI-HF
</LI></UL><P>This time it was incubated at 37*C for 15 minutes, and the product was PCR purified.
</P><P>1.5 ul was run on the gel.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">We can clearly see the appropriate sized bands (717 and 578 bps) after cutting our cross junction product with Bsa1</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Redoing_the_isothermal_assembly.2Ftransformation_for_Pos_con_77_and_the_3_controls">Redoing the isothermal assembly/transformation for Pos con 77 and the 3 controls</SPAN></H4><P>Today we redid the isothermal assembly for Pos con 77, Zif268, OZ052, and OZ123 using our previous protocol.  We also performed a chemical transformation using Top 10 ChemComp cells, and plated them with 100 ul each plate.  We will have to wait and see whether colonies grow tomorrow.
</P><P>We also performed a cross junction PCR on the isothermal assembly product to make sure that it worked, using our usual protocol. The gel (pictured below) shows appropriately sized bands for the cross junction.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">All our isothermal assemblies worked, since we can see bands at ~1.4 kb.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Practice_plate_library_assembly">Practice plate library assembly</SPAN></H4><P>We cut 20 ul of CB bottom plasmid using the following recipe:
</P><UL><LI>2 ul of 10x Buffer 4
</LI><LI>16 ul of DNA (at 40 ng/ul)
</LI><LI>1 ul of enzyme
</LI><LI>1 ul of BSA
</LI></UL><P>This was run out on a gel, and the band at size 2756 bp was extracted.  This will be extracted and used in a ligation reaction tomorrow.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Product after digesting our plasmid with Bsa1</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Team_TolC_20">Team TolC</SPAN></H3><H4><SPAN class="mw-headline" id="Lambda_red_recombination_of_zeocin_into_ECNR2">Lambda red recombination of zeocin into ECNR2</SPAN></H4><UL><LI>Performed electroporation and lambda red on ECNR2 in order to insert zeocin for rpoZ knockout, in order to have for control in plate reader
</LI><LI>Performed PCR and ran gel on products
</LI><LI>Successful!!
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">8/1/11 Zeocin insertion into ECNR2 for plate reader</DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Transformation_of_Zif268_into_ECNR2.2Bzeocin_without_Kan.2BZFB.2Bwp">Transformation of Zif268 into ECNR2+zeocin without Kan+ZFB+wp</SPAN></H4><UL><LI>Performed transformation of Zif268 on spec plasmid into ECNR@ without Kan insertion in order to have a positive control against our selection system
</LI><LI>Run PCR on product tomorrow to determine success
</LI></UL><H4><SPAN class="mw-headline" id="Observation_of_growth_through_plate_reader">Observation of growth through plate reader</SPAN></H4><UL><LI>Divided plate up in to 45 different cultures
</LI><LI>1-15 contained ECNR2+kan-ZFB-wp+zeocin+Zif268 spec plasmid
</LI><LI>This is the selection system we created
</LI><LI>They were grown in .005% SDS and 2,4,8,12,16 µM of IPTG
</LI></UL><LI>16-30 contained ECNR2+Kan-ZFB-wp with out Zif268 spec plasmid
</LI><LI>This served as a negative control
</LI><LI>Also grow in .005% SDS and 2,4,8,12,16 µM of IPTG
</LI><LI>31-45 contained solely ECNR2
</LI><LI>This served as a postive control
</LI><LI>Grown in .005% SDS and 2,4,8,12,16 µM of IPTG
</LI><LI>Plate was taken down to plate reader at 5:25 pm and began a 16 hour program</LI></DIV><DIV id="802" style="display:none"><H2><SPAN class="mw-headline" id="August_2nd">August 2nd</SPAN></H2><H3><SPAN class="mw-headline" id="Team_TolC_21">Team TolC</SPAN></H3><UL><LI>PCR to check for spec plasmid in ECNR2+zeocin strain successful. The cultures are taking a long time to grow, though, so we might make another transformation with the zif268 plasmid tomorrow, if they do not appear to have grown by tomorrow
</LI><LI>IPTG appears to be not working properly. It appears to retard growth. It is possible that the IPTG here may not be working, so an aliquot of IPTG from the Wyss should be brought in tomorrow.
</LI></UL><H3><SPAN class="mw-headline" id="Team_ZF_24">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Checking_on_the_Pos_Con_77_and_F2.2FF3_Control_Plates">Checking on the Pos Con 77 and F2/F3 Control Plates</SPAN></H4><P>All our plates grew bacteria colonies. At the end of the day, we picked 4 colonies from each plate, and grew them in 2 mL of LB+spec. 
</P><H4><SPAN class="mw-headline" id="Miniprep_of_CB_Bottom_for_ligation">Miniprep of CB Bottom for ligation</SPAN></H4><P>We performed another miniprep on the CB Bottom plasmid, but this time we concurrently performed a miniprep on PZE22G that had been grown overnight in LB+Kan. 
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Miniprep results. PZE22G had excellent yields, while CB Bottom did not.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>Because the yield of the PZE22G miniprep is in the range that we expect (100-200 ng/ul), while the yield of the CB Bottom miniprep is still so low, we concluded that our problem is likely due to the plasmid or LB+spec media. We made new LB+spec. In the future, we could also pick new colonies and grow them up in cultures. 
</P><P>We were planning on using the miniprep product in our ligation reaction. We originally thought that we would need 100 ng of DNA for the reaction, but then we were told that we could have as little as 20 ng of DNA. We decided that in that case, our current yields were sufficient.
</P><H4><SPAN class="mw-headline" id="Ligation_of_CB_Bottom_backbone_and_F1">Ligation of CB Bottom backbone and F1</SPAN></H4><P>We wanted to insert an F1 into a CB backbone today. Yesterday, we cut the CB Bottom plasmid, in preparation for the ligation reaction. Today we gel purified the product. It was difficult to tell exactly which band was the one we were looking for, so we cut out 2 just to be safe. Our gel purification yields were:
</P><UL><LI>Top band: 23 ng/ul
</LI><LI>Bottom band: 16 ng/ul
</LI></UL><P>We attempted to cut the F1 oligo from the plate using Bsa1. Our product after cutting was 79 bp, which is technically smaller than the 100 bp minimum for PCR purification. However, we attempted the PCR purification anyway, because this cut-off is unlikely to be strict. We then ran a gel to see the band size of the product that remained. Because our first gel was unsatisfactory, we repeated the digestion a second time. 
</P><P>Attempt One:
</P><UL><LI>Digestion reaction recipe:
</LI><LI>2ul DNA
</LI><LI>8 ul 1x BSA/Buffer 4 solution
</LI><LI>0.5 ul Bsa1
</LI></UL><LI>PCR purification yield: 34 ng/ul
</LI><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Our first digestion did not show any bands below 100 bp.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>Attempt Two:
</P><UL><LI>Digestion reaction recipe:
</LI><LI>2ul DNA
</LI><LI>1 ul Bsa1
</LI><LI>50 ul buffer BSA
</LI></UL><LI>PCR purification yield: 33.5 ng/ul
</LI><LI>We also kept a sample of unpurified PCR product from this digestion to run on a gel, to see if we're losing our desired product in the PCR purification step.
</LI><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:140px;"><DIV class="thumbcaption">The bands on this 1.2% e-gel were unexpectedly high. It maybe a problem with the gel.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>We performed a ligation reaction using the following recipe:
</P><UL><LI>5 ul ligation mix
</LI><LI>4 ul backbone
</LI><LI>1 ul oligo
</LI></UL><P>But after seeing the gel results we are unsure whether the ligation could be successful.
</P><P>In order to further troubleshoot our less-than-ideal gels for our digestion product, we ran a third digestion reaction, using the second recipe. This time we simultaneously digested our oligo, and our cross junction (which had been shown to work before). We also ran the unpurified PCR product from recipe 2 on this gel. 
</P><P>See tomorrow's entry for the gel image.
Our digestion of the cross junction appeared to have worked, while once again, we got unusually large bands for our oligo.
</P><H4><SPAN class="mw-headline" id="PCR_of_Zif268_to_create_GFP_Reporter_Strain">PCR of Zif268 to create GFP Reporter Strain</SPAN></H4><P>We performed a PCR to clone out Zif268 so that we can later assemble Zif and GFP into our reporter plasmid. We used the Zif clone Forward and Zif clone Reverse primers (Primer index: 148 and 149). We had an annealing temperature of 62* and an extension time of 90 seconds. Our expected product was ~3 kb.
</P><P>See tomorrow's entry for the gel. Our PCR appeared to have worked, and we have a strong band at our expected product size.
</P><H3><SPAN class="mw-headline" id="Team_Wolfe_18">Team Wolfe</SPAN></H3><P><B>Plate reader:</B></P><UL><LI>We got some beautiful growth curves! A few of the more critical ones are below:
</LI><LI>NM+his: both selection strain and pSR01 grew, but the selection strain grew faster. This might just be an artifact from having accidentally added more cells to the selection strain than the pSR01 wells, but it may also show that the pSR01 strain grows slower. Another possibility is that we are expressing enough zinc fingers for it to be slightly toxic for the cells. Next time maybe we should let the plate reader go for longer than 16 hours just in case the cells need more time.
</LI></UL><UL><LI>NM: selection strain does not grow, while pSR01 does a perfect rescue!
</LI></UL><UL><LI>pSR01 HARV3-AT comparison: the concentration of 3-AT does not seem to make a difference in the growth curves (the differences appear to be due to different wells starting with slightly different amounts of cells). Next time we can try a higher 3-AT concentration, but since we expect Zif268 to bind strongly this might just be what you get.
</LI></UL><UL><LI>NM+his+IPTG: IPTG more or less killed the pSR01 cells but not the selection strain. This may be caused by too many zinc fingers being expressed, so we may want to try less IPTG.
</LI></UL><P><B>EcNR2 Wolfe Selection system: MAGE</B></P><UL><LI>Our MAGE oligos are finally here!  We performed 3 rounds of MAGE and designed oligos for allele-specific PCR to do tomorrow to check our colonies.
</LI><LI>2µM final concentration HisB VE deletion and PyrF KO oligos; 2.5-3hr recovery between rounds; plated 1µL, 10µL, and 100µL
</LI></UL><P><B>pSB4K5:</B></P><UL><LI>Our pSB4K5 PCR still eludes us. Our overnight gradient did work, but no better than the other days' reactions.
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">pSB4K5 gradient 8/2/11</DIV></DIV></DIV><UL><LI>Gel extraction and Minelute only gave us a 9.4ng/µL yield (260/280=1.82) which is not enough for isothermal assembly. We will have to miniprep more plasmid for template for PCRs, and we might want to consider using ligation or some other method.</LI></UL><DIV id="803" style="display:none"><H2><SPAN class="mw-headline" id="August_3rd">August 3rd</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_25">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Yesterday.27s_last_digestion_and_PCR_results">Yesterday's last digestion and PCR results</SPAN></H4><P>We ran a gel of our digestion product from yesterday, and our PCR to clone out Zif 268 to construct the reporter plasmid.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Our results for the final digestion we ran yesterday, and the PCR to clone out Zif 268.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Innoculation_of_Zif_268">Innoculation of Zif 268</SPAN></H4><P>We were running low on Zif 268 plasmids, so we decided to make a new culture from the glycerol stock. It has been left in the 37* shaker overnight.
</P><H4><SPAN class="mw-headline" id="Sending_Pos_Con_77_and_swap_controls_for_sequencing">Sending Pos Con 77 and swap controls for sequencing</SPAN></H4><P>We performed a miniprep on our cultures of Pos Con 77, and OZ052 swap, OZ123 swap, and Zif 268 swap.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Results of our miniprep.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>Once again, we got unusually low yields. We decided to test the quality of our spec, to see if that was the problem. We grew PZE22G(which does not have spec resistance) in spec to see if it grew. By the end of the day, nothing had grown. 
</P><P>Still, we decided to PCR the miniprep product, since any DNA that is in the product should be amplified. We performed our usual protocol for a cross junction PCR, but we repeated the reaction for 30 cycles. 
</P><P>Some of the bands varied in size, but we decided to send it all in for sequencing.
</P><H4><SPAN class="mw-headline" id="Constructing_the_reporter_Zif268_Plasmid">Constructing the reporter Zif268 Plasmid</SPAN></H4><P><B>PCRs of Zif 268 and GFP</B></P><P>We decided to perform more PCRs to clone out Zif 268, since it will have to be gel purified. This reaction had the same conditions as yesterday's.
</P><P>We also received the GFP plate today, so we decided to perform a colony PCR in order to get out the sequence that encodes GFP. We used our GFP Forward and GFP Reverse (Primer Index: 146 and 147) for the reaction, had an annealing temperature of 58*, and an extension time of 30 seconds. The expected product size was ~700 bp. 
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">The last four lanes show the expected band size of ~825 bp for GFP. However the bands are comparatively faint.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P><B>Gel Purification of Zif 268 and GFP</B></P><P>We performed a gel purification of our PCR products. 
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">We made a 1% agarose gel to gel purify our zif clone and GFP. The bands for GFP are much fainter than those for Zif 268.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>After gel purification, Zif 268 had a concentration of approximately 110 ng/ul. 
</P><P>While we had a bright band for Zif 268 on our gel, our band for GFP was much fainter, so we decided not to gel purify it, and instead to re-do the PCR (4 times) for that reaction.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:169px;"><DIV class="thumbcaption">We ran another PCR to get out GFP.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>Our yield for the gel extraction of GFP was 10.2 ng/ul. 
</P><P><B>Isothermal assembly and transformation</B></P><P>Our yield for the gel extraction was lower than what we wanted, but we figured we would try the assembly anyway. This meant that we needed 2.9 ul GFP and 0.9 ul backbone. Ideally all the DNA takes up 2.5 ul when performing an isothermal assembly. We decided to perform 2 reactions, so that one of them might work. 
</P><UL><LI>First recipe (total volume 10 ul):
</LI><LI>1.9 ul GFP
</LI><LI>0.6 ul backbone
</LI><LI>7.5 ul isothermal assembly mix 
</LI></UL><LI>Second recipe (total volume 11.3 ul):
</LI><LI>2.9 ul GFP
</LI><LI>0.9 ul backbone
</LI><LI>7.5 ul isothermal assembly mix
</LI><P>We performed a transformation using ChemComp cells, plated 100 ul, and left them in the incubator overnight.
</P><H4><SPAN class="mw-headline" id="Practice_plate_ligation">Practice plate ligation</SPAN></H4><P>After seeing the results of our third oligo digestion yesterday, we were confused about why the reaction was not working. After talking with our teammates, we realized that we had been performing the reaction on single stranded DNA, instead of our double stranded PCR product. We performed the digestion again today with oligos 1 and 2, using the following recipe:
</P><UL><LI>15 ul DNA
</LI><LI>0.5 ul enzyme
</LI><LI>85 ul buffer/BSA 
</LI></UL><P>After letting the reaction sit at 37* for about 45 minutes, we performed a PCR purification, and then ran the product on a gel:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">We could see bands of the appropriate size, after digesting our PCR product. Unpure = cut without PCR purification.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>Finally, we had gotten bands at the appropriate size! We noticed that there were multiple bands smaller than our uncut oligo. We figured the smallest band was the one we were looking for. The one that was slightly larger was likely the result of the enzyme only cutting at one end of our oligo. We wondered whether all of our product would have been the appropriate size if we had let the reaction sit at 37* for a longer period of time. Regardless, we decided to use this product in our ligation reaction. If both ends of the oligo aren't cut, then it should not be able to bind with the backbone.
</P><P>We performed a ligation reaction with our cut CB Bottom and our digested oligos, using the following recipe:
</P><UL><LI>5 ul ligation mix
</LI><LI>4 ul backbone (~20 ng/ul)
</LI><LI>1 ul insert (~15 ng/ul)
</LI></UL><P>This would leave to a vector to insert ratio of 1 to 8, which is what is normally used for ligation reactions. 
</P><P>This is how we calculated the amounts of inserted needed for our ligation:
</P><PRE>PCR product: ~60 ng/ul
We want 1 ug of DNA to be cut, so that after the PCR purification, when we elute in 30 ul we will get a concentration of ~30 ng/ul.
</PRE><P>In practice, we end up losing some of our product, so we ended up with 15 ng/ul.</P><P>After letting the reaction sit at room temperature for 1 hour, we transformed ChemComp bacteria with the ligation products. We plated 50 ul and 400 ul of the transformation and left the plates at 37* overnight.
</P><H3><SPAN class="mw-headline" id="Team_Wolfe_19">Team Wolfe</SPAN></H3><UL><LI>Overnight MAGE round 3 colonies grew - 1µL, 10µL, and 100µL used with (too)many cells on 100µL plate and a workable amount on the 1µL plate.  
</LI><LI>MAGE round 3 PCR 
</LI><LI> 10µL reactions
</LI><LI> Primers: HisB VE KO Check_R, His_F and PyrF KO Check_F, PyrF IR 
</LI><LI>47 colonies tested for knock out of HisBUra3 and 47 colonies tested for knock out of pyrF
</LI><LI> HisB Control - selection strain with same primers  
</LI><LI>PyrF Control - Colony 1 with PyrF forward and PyrF internal reverse primers
</LI></UL><LI>Reverse Touchdown PCR to obtain the kan backbone with homology from pSB4K5 biobrick plasmid:
</LI><LI>Used annealing temperature range of 58 to 68 with two rounds elongation at 2 degree intervals and 15 elongation cycles at 68 degrees.
</LI><LI>PCR failed
</LI><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">A successful PCR would have resulted in a 3.5-4Kb band.</DIV></DIV></DIV><UL><LI>Miniprep of pSB4K5 strain for future PCR yielded 15ng/µL
</LI><LI> Plate reader
</LI><LI> Wells 1, 8, 15... 36 LB+tet/amp
</LI><LI>Wells 2,9,16...37 1mM 3-AT
</LI><LI>Wells 3,10, 17...38 2.5mM 3-AT
</LI><LI>Wells 4,11, 18...39 5mM 3-AT
</LI><LI>Wells 5,12, 19...40 10mM 3-AT
</LI><LI>Wells 6,13,20...41 25mM 3-AT
</LI><LI>Wells 7,14,21...42 50mM 3-AT
</LI><LI>Wells 1-7 = empty
</LI><LI>Wells 8-14 = selection strain
</LI><LI>Wells 15-28 = single selection strain + pSR01 colony (duplicates)
</LI><LI>Wells 29-42 = second selection strain + pSR01 colony (wells 33-35 were not usable)
</LI></UL><LI>MAGE rounds 4&amp;5 completed, colonies plated at 1µL, 10µL, and 100µL volumes.
</LI><H3><SPAN class="mw-headline" id="Team_TolC_22">Team TolC</SPAN></H3><UL><LI>Set up and ran plate reader for the growth rates of ECNR2+kan-ZFB-wp+zeocin+spec plasmid, ECNR2+kan-ZFB-wp+zeocin Δspec plasmid, ECNR2, and ECNR2+zeocin+spec plasmid Δkan-ZFB-wp in .04% SDS and 25, 12.5, 6.25, 3.125, 1.5625 µM</LI></UL><DIV id="804" style="display:none"><H2><SPAN class="mw-headline" id="August_4th">August 4th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_26">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Pos_Con_77_and_control_swap_sequencing_results">Pos Con 77 and control swap sequencing results</SPAN></H4><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Pos. Con. 77</B></TD><TD align="center" style="background:#f0f0f0;"><B>Notes</B></TD></TR><TR><TD>1</TD><TD>lots of snps and deletions
</TD></TR><TR><TD> ✓</TD><TD>2</TD><TD>G-&gt;C
</TD></TR><TR><TD>3</TD><TD>deletion before type II cut site
</TD></TR><TR><TD>4</TD><TD>deletion in terminator
</TD></TR><TR><TD align="center" style="background:#f0f0f0;"><B>OZ052 swap</B></TD><TD align="center" style="background:#f0f0f0;"><B>Notes</B></TD></TR><TR><TD> ✓</TD><TD>1</TD><TD>G-&gt;C
</TD></TR><TR><TD> ✓</TD><TD>2</TD><TD>G-&gt;C
</TD></TR><TR><TD>3</TD><TD>deletions and mismatches in ultramer
</TD></TR><TR><TD>4</TD><TD>terrible
</TD></TR><TR><TD align="center" style="background:#f0f0f0;"><B>OZ123 swap</B></TD><TD align="center" style="background:#f0f0f0;"><B>Notes</B></TD></TR><TR><TD> ✓</TD><TD>1</TD><TD>fine, deletion at end
</TD></TR><TR><TD>2</TD><TD>deletions and snps
</TD></TR><TR><TD>3</TD><TD>long deletion in helix
</TD></TR><TR><TD> ✓</TD><TD>4</TD><TD>fine
</TD></TR><TR><TD align="center" style="background:#f0f0f0;"><B>Zif268 swap</B></TD><TD align="center" style="background:#f0f0f0;"><B>Notes</B></TD></TR><TR><TD> ✓</TD><TD>1</TD><TD>fine, TGEKP instead of TGQKP
</TD></TR><TR><TD> ✓</TD><TD>2</TD><TD>G-&gt;C, TGEKP instead of TGQKP
</TD></TR><TR><TD>3</TD><TD>terrible
</TD></TR><TR><TD> ✓</TD><TD>4</TD><TD>G-&gt;C, TGEKP instead of TGQKP
</TD></TR></TBODY></TABLE><P>We had a good sequence for all of our samples. We made 2 ml cultures of these, to miniprep, make final plates, and glycerol stocks. Note, that for Zif 268, the linker between F1/F2 is 'TGEKP', as that was the sequence in the ultramer we ordered. The actual sequence is 'TGQKP'. If this proves to be a problem, we can always try using quick change to insert the appropriate codon.
</P><H4><SPAN class="mw-headline" id="Checking_on_the_reporter_strain">Checking on the reporter strain</SPAN></H4><P>Our transformed bacteria with the 11.3 ul isothermal assembly product and 100 ng backbone grew. We picked four from the plate and are growing 2 ml cultures from time to miniprep and send for sequencing tomorrow. 
</P><P>Our other bacteria with 10 ul product, but less than 100 ng backbone, did not grow.
</P><H4><SPAN class="mw-headline" id="Ligation_of_oligos_and_backbone">Ligation of oligos and backbone</SPAN></H4><P><B>Checking our plates</B></P><P>No colonies grew on any of our plates that contained bacteria transformed with ligation product. We wondered if this was due to our ligase no longer working, so we decided to test our ligase. 
</P><P><B>Testing our ligase</B></P><P>We wanted to test our ligase by cutting our cross junction with Xba1, and then ligating it, so see if it formed the original cross junction. We ran a gel comparing our whole cross junction, our digested product, and our ligated product. 
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Comparing our original, cut, and ligated cross junction.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>Our uncut cross junction was ~1.4 kb. We saw the appropriate sized bands of 593 and 739 bp after cutting the junction with Xba1. However, after ligation, we saw bands of varying lengths, at the one we expected (at ~1.4 kb) was not noticeably brighter than the other bands. We are unsure what these bands are.
</P><P><B>Cutting and gel purifying CB Bottom</B></P><P>We also wondered if our unsuccessful ligation was due to an ineffective CB Bottom segment. We had been unsure exactly which band was the appropriate one, and so we cut both. Just in case we decided to re-do the digestion of the CB Bottom plasmid, with Bsa1. We used the following recipe:
</P><UL><LI>46 ul DNA (to get 1 ug)
</LI><LI>1 ul enzyme
</LI><LI>5 ul Buffer T4
</LI><LI>0.6 ul BSA
</LI></UL><P>We then ran the product on a 0.7% agarose gel:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">A gel of the CB Bottom backbone, after digestion. We expected a band at ~2.7 kb</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>We cut out the appropriate band, and put it in the fridge to gel purify tomorrow.
</P><H4><SPAN class="mw-headline" id="Making_more_Zif268_.28whole.29_plasmid">Making more Zif268 (whole) plasmid</SPAN></H4><P>We performed a miniprep of the Zif268 culture we grew yesterday. The miniprep had a yield of ~16 ng/ul.
</P><H3><SPAN class="mw-headline" id="Team_Wolfe_20">Team Wolfe</SPAN></H3><P><B>Plate Reader</B></P><UL><LI>Looks great!
</LI><LI>1mM, 2.5mM, and 5mM 3-AT have slight differences in growth rate, but it is hard to tell how significant they are. However, starting with 10mM 3-AT, the growth rate drops dramatically! Here is an example from 1 culture set:
</LI></UL><P><B>MAGE 3 HisB and PyrF Knock Out Gel</B></P><UL><LI>HisB Check
</LI><LI> Control reaction showed a nice strong band.  All the other lanes had bands, but they varied in strength.  It looks like we may have some hits, but we need our WT control primers to come in for us to be sure.
</LI></UL><LI>PyrF check
</LI><LI>Similar to HisB--all the lanes had bands but some were brighter than others.
</LI><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">HisB</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">PyrF</DIV></DIV></DIV><P><B>ZFB-wp-his3-ura3 restriction enzyme PCR</B></P><UL><LI>PCR to add Spe1 and Xba1 sites to join insert with pSB4K5 plasmid
</LI><LI>Spe1 ZFB_F and Xba1 URA3-R primers
</LI><LI>90 sec elongation, 62˚C annealing
</LI></UL><H3><SPAN class="mw-headline" id="Team_TolC_23">Team TolC</SPAN></H3><H4><SPAN class="mw-headline" id="Plate_Reader">Plate Reader</SPAN></H4><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Plate Reader Graph of KN.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Plate Reader Graph of WN.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Plate Reader Graph of KS.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Plate Reader Graph of WS.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="PCR_of_plate_reader_colonies">PCR of plate reader colonies</SPAN></H4><UL><LI>Made glycerol stock of ECNR2+spec plasmid+zeocin, WS</LI></UL><DIV id="805" style="display:none"><H2><SPAN class="mw-headline" id="August_5th">August 5th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_Wolfe_21">Team Wolfe</SPAN></H3><P><B>Spe1 ZFB-wp-his3-ura3 Xba1 PCR purification:</B></P><UL><LI>Minelute: 595 ng/uL, 260/280=1.87
</LI></UL><P><B>pSB4K5 miniprep:</B></P><UL><LI>To get more pSB4K5 3:11P we did a miniprep of a 6 mL culture
</LI><LI>Yield: 8ng/uL
</LI><LI>This yield is still very low, but a gel showed that DNA is present in the sample.
</LI></UL><P>File
</P><P><B>ZFB hisura digest:</B></P><UL><LI>40uL total volume:
</LI><LI>4uL buffer 4(10x)
</LI><LI>0.4uL BSA (100x)
</LI><LI>2uL Spe1
</LI><LI>2uL Xba1
</LI><LI>2uL Spe ZFB-his3-ura3 Xba1 (about 1ug)
</LI><LI>29.6uL H2O 
</LI></UL><LI>incubate overnight at 37 C
</LI><P><B>Prep for tomorrow:</B></P><UL><LI>pSB4K5 3:11P culture: repeat miniprep tomorrow and hope for a better yield
</LI><LI>selection strain + pKD46: to transform with finished ligation product
</LI><LI>selection strain: streak new plate
</LI></UL><H3><SPAN class="mw-headline" id="Team_ZF_27">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Gel_extraction_of_cut_CB_bottom_plasmid">Gel extraction of cut CB bottom plasmid</SPAN></H4><P>We performed a gel extraction on the CB bottom plasmid that we cut with Bsa-HF and ran through a gel yesterday.  However, we got an extremely low yield of 3.4 ng/ul.
</P><P>After talking with Dan about the not-so-ideal experiences with Bsa-HF, we decided to make a few changes to our protocols, and that we would try the following to see how they would work:
</P><P><I>To troubleshoot the poor oligo cutting:</I>
1.5 ul of enzyme (20 units/ul)
2 ul of NEB buffer 4
0.5 ul of BsaI-HF
16 ul of DNA</P><P>20 ul total
</P><P>Incubute for 1.5 hour in a thermal cycler at 37 degrees.
Denature at 65 degrees for 20 minutes and then pcr purify.
</P><P><I>For troubleshooting the poor plasmid cutting:</I>
Do the same thing but do it for longer (2 hours) and also run an uncut plasmid to see which bands come up.
</P><P>We checked to see if dcm methylation happens for bsaI at our cut sites, but there were no signs of the dcm recognition sequences (CCAGG or CCTGG) anywhere near them.
</P><H4><SPAN class="mw-headline" id="GFP_reporter_plasmid_sequencing">GFP reporter plasmid sequencing</SPAN></H4><P>Today we miniprepped the four GFP colonies that we picked yesterday in order to run a junction PCR on them and send them out for sequencing.  Our junction PCR used the previous protocol's annealing temperature of 55 degrees, but the extension time was lengthened to 1:15 to account for the extra length of the GFP insert (some additional 700-800 bp).  After the PCR, the samples were sent out for sequencing with both the forward junction primer and the reverse 100 bp primer (ZF116 by the primer index).
</P><H4><SPAN class="mw-headline" id="Glycerol_stocks_and_final_platings">Glycerol stocks and final platings</SPAN></H4>
We made glycerol stocks of the final four plasmids in Top 10 ChemComp bacteria, which were Pos Con 77 (85.2), OZ052 (52.1), OZ123 (123.4), and Zif268 (Zif268.1).  We also made final plates of these bacteria, using 1/1000 ul on spec plates.</DIV><DIV id="806" style="display:none"><H2><SPAN class="mw-headline" id="August_6th">August 6th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_28">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="New_miniprep_protocol">New miniprep protocol</SPAN></H4><P>Today, we tried a new miniprep protocol in which the DNA is precipitated out of solution with EtOH, and no spin columns are used.  This is because we suspect that our low yields may be due in some part to DNA being washed out of the column prematurely, or DNA not being washed completely out of the column during the elution step.  In getting rid of the column, there is no step during which we may lose DNA.
</P><P>We are extremely happy to report that after weeks of getting dismal miniprep yields, we have found a new protocol that gives us insanely high yields (compared to before)!!!  The protocol is the following:
</P><OL><LI>Centrifuge 1.5 ml of saturated cells at maximum speed for 1 minute, discard supernatant.
</LI><LI>Resuspend cells in 200 ul of Buffer P1.
</LI><LI>Add 200 ul of Buffer P2 to lyse the cells.  Mix by inverting several times, and let sit for 3 minutes.
</LI><LI>Add 200 ul of Buffer N3 to precipitate SDS, mix by inverting several times.
</LI><LI>Centrifuge at 4 degrees for 15 minutes.
</LI><LI>Transfer supernatant to new tube, add 1 ml of 100% EtOH to precipitate DNA.
</LI><LI>Centrifuge for 5 minutes at RT.  At this point a DNA pellet should be visible at the bottom of the tube.
</LI><LI>Discard EtOH, wash DNA pellet with 1.5 ml of 70% EtOH (this gets rid of salts).
</LI><LI>Centrifuge for 5 minutes at RT.
</LI><LI>Discard EtOH, let tubes dry for approximately 6-8 minutes by inverting them on a piece of Kimwipe tissue.
</LI><LI>Resuspend DNA pellet in 20 ul of heated EB buffer.
</LI></OL><P>Our resulting yield was overall <B>463.4 ng/ul</B>!!  The purity also looked extremely good, with a 260/280 value of 1.98.  The nanodrop results can be viewed below:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Over 20x times normal yield compared with the Qiagen miniprep kit.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>We will have to repeat this again tomorrow to see if we get the same yields with this procedure.  Also, we should run some of these out on a gel to verify what it looks like.
</P><H4><SPAN class="mw-headline" id="New_restriction_enzyme_protocol">New restriction enzyme protocol</SPAN></H4><P>The protocol used for the restriction enzyme digest will be as described yesterday:
</P><UL><LI>16 ul DNA (any concentration)
</LI><LI>2 ul 10x buffer 4
</LI><LI>0.5 ul 100x BSA
</LI><LI>1.5 ul enzyme (30 units)
</LI></UL><P>This gives a 20 ul total reaction volume.  The reaction is incubated at 37* for 2 hours, denatured at 65* for 20 minutes, and then left at 4* for storage overnight.
</P><P>A restriction digest using this protocol was performed on practice plate oligos 5 and 6 (reactions for oligos 3 and 4 were no good due to a pipetting mishap).
</P><H4><SPAN class="mw-headline" id="Sequencing_results_for_the_GFP_reporter_plasmid">Sequencing results for the GFP reporter plasmid</SPAN></H4><P>Due to Seqman crashing and the Harvard VPN not being active, we will have to wait until Monday for these results.
</P><H3><SPAN class="mw-headline" id="Team_Wolfe_22">Team Wolfe</SPAN></H3><P><B>MAGE round 3 allele-specific PRC:</B></P><UL><LI>did 4 types of PCR: HisB KO (HisB VE KO check_R and His_F), HisB WT (HisB VE WT check_R and His_F), PyrF KO (PyrF KO check_F and PyrF_internal_R), PyrF WT (PyrF WT check_F and PyrF_internal_R)
</LI><LI>24 colonies: A10, A11, C2, C3, C6, C8, C9, D1, D5-12, E3, E7, F2-4, G2, G3, G9
</LI><LI>10uL total reaction volume
</LI><LI>same protocol as earlier reaction (64C anneal, 1min elongation, 25 cycles)
</LI><LI>HisB bands at about 900bp; PyrF bands at about 400bp.
</LI><LI>Results: none of the colonies have both genes knocked out but some do look like they may have the HisB deletion.  We may try again using MAGE round 5 colonies.
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">HisB and PyrF allele specific PCRs part 1. Yellow-labeled colonies may have a knock out. 8/6/11</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Separate E gel of the pyrF PCRs cut off in gel 1. 8/6/11</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">HisB and PyrF allele specific PCRs part 2 8/6/11</DIV></DIV></DIV><P><B>pSB4K5 digestion:</B></P><UL><LI>Since our minipreps have been giving such low yields, we tried an <A href="http://openwetware.org/wiki/Knight:Miniprep_low_copy_plasmids" class="external text" rel="nofollow">altered protocol</A>.  The result, however was the same: 6mL of culture produced 19ng/uL (in 30uL)
</LI><LI>PCR purified Spe1 Xba1 ZFB-wp-his3-ura3 digestion (Minelute): 53.8ng/uL 260/280=1.82.  This is interesting, since it appears that half our DNA has been lost.
</LI><LI>pSB4K5 Spe1 and Xba1 digestion: 72uL total
</LI><LI>about 1mg DNA=60uL
</LI><LI>2uL each enzyme
</LI><LI>7.2uL buffer 4
</LI><LI>.72uL BSA
</LI><LI>3 hour incubation at 37C
</LI></UL><LI>gel extraction: the gel was empty. No clue why. Obviously this is a setback--we will have to redo the miniprep and digestion before we can form the whole plasmid.</LI></DIV><DIV id="807" style="display:none"><H2><SPAN class="mw-headline" id="August_7th">August 7th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_29">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Miniprep_repeat_with_new_protocol">Miniprep repeat with new protocol</SPAN></H4><P>Another miniprep was repeated on 3 ml of CB bottom plasmid culture using the protocol from yesterday, with a yield of 1 ug/ul.  This seemed very high considering our previous results, and we are suspicious that there may be some genomic DNA contamination.  Contamination may occur either from leaving cultures on the shaker for too long (with cells lysing and releasing their genomic DNA) or from shaking too hard during the lysis process (shearing chromosomal DNA and allowing it to escape the cell).  The lysis process was performed by mixing through tube inversion, so this is likely not the cause.  The tubes that were miniprepped yesterday were left on the shaker for approximately 22 hours, while the tubes from today were on the shaker for 17 hours.  Strangely, although the tubes were on the shaker for a shorter amount of time, there was still higher yield.
</P><P>The concentration and absorbance curve data can be seen below:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Hopefully this is actually real, it seems almost too good to be true.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>As a note, the tubes were unusually dark blue this round after addition of the lysis buffer (buffer P2).
</P><P>The uncut plasmid was also run out on a gel alongside the cut oligos (loaded 0.5 ul).  This should theoretically have been around 500 ng of DNA, but for some reason it only shows up very faintly.  This is not good news.
</P><H4><SPAN class="mw-headline" id="Results_for_new_restriction_enzyme_protocol">Results for new restriction enzyme protocol</SPAN></H4><P>The resulting reaction from the oligo cut with Bsa-HF yesterday was run out on a gel with the following results:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">blah</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>There appears to be a band of the correct size.  However, there is still incomplete cutting as indicated by the second band at around 100 bp, but there is no longer a band at the uncut position, indicating that all of the oligos underwent a single or double cut.
</P><P>The plasmid DNA from the miniprep today was also put through a restriction digest using the same 2 hour protocol as the oligo digest.  The result was then PCR purified, but the yield was extremely low compared to the input.  Using 16 ul of DNA, there were 16 ug of plasmid that went into the restriction digest.  After PCR purification and elution in 30 ul of EB, the yield was only 28.5 ng/ul, meaning only 0.855 ug of DNA in the purified product.  This translate to over <B>95%</B> of the DNA somehow being lost in the PCR purification.  We are at a complete loss as to why this has happened...
</P><P>The cut plasmid DNA was run alongside uncut plasmid in a gel, with the results below:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:124px;"><DIV class="thumbcaption">blah</DIV></DIV></DIV></TD></TR></TBODY></TABLE>
Although the bands are faint, the cut band is somewhere around the correct size (around 2.8 kb).  Although the band appears to be anywhere between 3 kb and 4 kb in the gel, the ladder ran strangely for some reason.  However, there is still a fairly well-defined band instead of a smear in the cut lane.  This in itself is a huge relief - if there were significant genomic DNA contamination, we might expect to get a smear from many differently sized cuts.  Furthermore, the uncut band looks standard for a plasmid, similar to the picture on the website here: [<A href="http://faculty.plattsburgh.edu/donald.slish/Electrophoresis.html" class="external autonumber" rel="nofollow">[20]</A>].  There are three bands in our gel as they appear on the website.  However, our linearized plasmid is actually fairly close to the bottom band for the uncut plasmid, which is presumably the supercoiled form.</DIV><DIV id="808" style="display:none"><H2><SPAN class="mw-headline" id="August_8th">August 8th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_Wolfe_23">Team Wolfe</SPAN></H3><H4><SPAN class="mw-headline" id="MAGE_sequencing">MAGE sequencing</SPAN></H4><UL><LI>Colonies
</LI><LI>A10, C3, and D9 from round 3 MAGE 
</LI><LI>A1-D8 round 5 MAGE (colonies picked today).
</LI></UL><LI>Primers for PCR
</LI><LI>His_F, His_R
</LI><LI>PyrF_F, PyrF_R
</LI><LI>Primers for sequencing
</LI><LI>His_R, PyrF_F
</LI><LI>64 degrees annealing and 90 seconds elongation
</LI><LI>Gel shows PCR was successful
</LI><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">MAGE sequencing PCR</DIV></DIV></DIV><H4><SPAN class="mw-headline" id="5-FOA_Concentration_Gradient">5-FOA Concentration Gradient</SPAN></H4><UL><LI>Protocol recommended 2.5mM conc. FOA but we tested lower concentrations because FOA = $$$:  ($FOA&gt;$Gold)
</LI><LI>61,68...89 NM+His
</LI><LI>62,69...90 NM
</LI><LI>63,70...91 .1mM 5-FOA
</LI><LI>64,71...92 .25mM 5-FOA
</LI><LI>65,72...93 .5mM 5-FOA
</LI><LI>66,73...94 1mM 5-FOA
</LI><LI>67,74...95 2.5mM 5-FOA
</LI><LI>61-67 = selection strain colony 1
</LI><LI>68-74 = selection strain colony 2
</LI><LI> 75-81 = pSR01 colony 1
</LI><LI> 82-88 = pSR01 colony 2
</LI><LI> 89-95 = empty
</LI></UL><H4><SPAN class="mw-headline" id="pSB4K5">pSB4K5</SPAN></H4><UL><LI>Since minipreps are not working we will try a midiprep
</LI><LI>Innoculated 100mL LB+Kan with a pSB4K5 3:11P colony
</LI></UL><H4><SPAN class="mw-headline" id="Streaked_Plates">Streaked Plates</SPAN></H4><UL><LI>Selection strain 
</LI><LI>Selection strain + pSR01 plates because they were getting old
</LI></UL><H3><SPAN class="mw-headline" id="Team_ZF_30">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="GFP_Sequencing_Results">GFP Sequencing Results</SPAN></H4><P>We were able to access the results of our GFP sequencing.
</P><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>GFP Forward</B></TD><TD align="center" style="background:#f0f0f0;"><B>Notes</B></TD></TR><TR><TD>1</TD><TD>1 deletion at end
</TD></TR><TR><TD> ✓</TD><TD>2</TD><TD>perfect
</TD></TR><TR><TD>3</TD><TD>Insert of G
</TD></TR><TR><TD>4</TD><TD>perfect
</TD></TR><TR><TD align="center" style="background:#f0f0f0;"><B>GFP Reverse</B></TD><TD align="center" style="background:#f0f0f0;"><B>Notes</B></TD></TR><TR><TD>1</TD><TD>4 snps
</TD></TR><TR><TD> ✓</TD><TD>2</TD><TD>1 deletion at end
</TD></TR><TR><TD>3</TD><TD>0 snps
</TD></TR><TR><TD>4</TD><TD> snps
</TD></TR></TBODY></TABLE><P>However our sequencing failed to cover the the Shine-Dalgarno region. We decided to test our strains by growing them on a 96 well plate. Each well had 150 ul LB+spec. Colonies designated 1-4 are 5 ul of the ones we sent out for sequencing. Those designated 5-8 were 2 ul colonies which were picked from our reporter plate then and diluted in 10 ul LB+spec. The plate had the following configuration:
</P><UL><LI>Rows A and B, 1 through 8: New colonies picked from the plate of our reporter strain. 
</LI><LI>Row E, 1 through 8: Colonies 1-8 in 0 ul IPTG
</LI><LI>Row F, 1 through 8: Colonies 1-8 in 2 ul IPTG
</LI><LI>Row G, 1 through 8: Colonies 1-8 in 10 ul IPTG
</LI><LI>Row H, 1 through 8: Colonies 1-8 in 100 ul IPTG
</LI></UL><P>After shining blue light on the plate, nothing glowed. We realized that this was because we did not have a spacer between the Shine-Dalgarno sequence and the start codon, meaning the ribosome would not be able to reach both sites. After doing some reading, we realized that a spacer of 6-10 bps is ideal. We will design this tomorrow.
</P><H4><SPAN class="mw-headline" id="Re-do_minipreps_for_control_backbones_with_new_protocol">Re-do minipreps for control backbones with new protocol</SPAN></H4><P>We used the new protocol to do a miniprep on the control backbones. 
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">We got (unusually?) high yields using the new technique.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>Our yields were extremely high with the new technique--it was difficult to believe. We were a bit wary of the nanodrop readings so we decided to run the product on a gel and  compare the brightness of the band to the ladder in order to determine the actual concentration of DNA present in the miniprep product. 
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">We ran a gel of our new miniprep product.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>The first six lanes are our miniprep product. The next four contain serial dilutions of the Zif 268 plasmid. We determined that it looks like our miniprep product actually contains ~80 ng/ul of DNA, after comparing it with the 1650 bp band.
</P><H4><SPAN class="mw-headline" id="Ligation_and_transformation_of_Oligo_5_and_6_and_CB_Bottom">Ligation and transformation of Oligo 5 and 6 and CB Bottom</SPAN></H4><P>We decided to try the ligation of our oligos and the backbone, using our previous protocol. We then transformed the product using ChemComp cells, plating 50 and 450 ul.
</P><H4><SPAN class="mw-headline" id="Cutting_the_Oligo_Pool">Cutting the Oligo Pool</SPAN></H4><P>We decided to try cutting a pool of our 18 oligos. We used our new recipe, adding 0.78 ul of each oligo so that the DNA takes up a total of 16 ul. We followed the new protocol for one reaction, and the other digestion was left to run overnight. We will compare the results tomorrow.
</P><H3><SPAN class="mw-headline" id="Team_TolC_24">Team TolC</SPAN></H3><H4><SPAN class="mw-headline" id="Plate_Reader_2">Plate Reader</SPAN></H4><UL><LI>Set up and began running 160 well plate reader
</LI><LI>Plate was set up as follows:
</LI></UL><H4><SPAN class="mw-headline" id="PCR_on_TolC">PCR on TolC</SPAN></H4><UL><LI>Have failed twice before in trying to PCR out the TolC sequence from the WS and WN strains, which are ECNR2 without the Kan-ZFB-wp insert, worried that TolC may have somehow been deleted or removed which seems very unlikely
</LI><LI>Ran third PCR and gel and failed again
</LI><LI><B>NOTE. It's possible that the ECNR2delTolC strain was used, but that still doesn't explain the 800bp band using the colonies from the zeocin plate used for selecting the zeocin cassette insertions that led to the WN strain (Noah's PCR 8/9/2011). It's strange given that the colonies off that plate give different results for the TolC region flanked by the TolC_seq primers, given that the zeocin PCR shows that all colonies have the zeocin insert (primers are Zeocin_R and rpoZ_R)</B></LI></UL><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">blah</DIV></DIV></DIV></TD></TR></TBODY></TABLE></DIV><DIV id="809" style="display:none"><H2><SPAN class="mw-headline" id="August_9th">August 9th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_Wolfe_24">Team Wolfe</SPAN></H3><H4><SPAN class="mw-headline" id="Sequencing_check_for_HisB_and_PyrF_deletions">Sequencing check for HisB and PyrF deletions</SPAN></H4><UL><LI>One colony definitely has both deletions! It is now being grown for lambda red and glycerol stock
</LI></UL><H4><SPAN class="mw-headline" id="Zeocin_Lambda_Red">Zeocin Lambda Red</SPAN></H4><UL><LI>Made 10 zeocin plates (zeocin concentration=50µg/mL)
</LI><LI>Grew up cells from D2 MAGE#5 to midlog and used 1mL for lambda red
</LI><LI>2µL zeocin cassette (about 300ng)
</LI><LI>recovered for 2 hours before plating
</LI></UL><H4><SPAN class="mw-headline" id="pSB4K5_Digestion">pSB4K5 Digestion</SPAN></H4><UL><LI>Enzymes
</LI><LI>2µL spe1
</LI><LI>2µL xba1
</LI></UL><LI>DNA = pSB4K5 plasmid (1µg, 17.2µL)
</LI><LI>4µL buffer 4
</LI><LI>.4µL BSA (100x)
</LI><LI>14.4µL H2O
</LI><LI>incubated for 2hrs at 37C
</LI><H4><SPAN class="mw-headline" id="5-FOA_concentration_gradient_results">5-FOA concentration gradient results</SPAN></H4><UL><LI>Since 5-FOA is more than worth its weight in gold, we were hoping that lower concentrations than recommended in the protocol would work for inhibiting the growth of cells expressing pyrF/URA3. According to last night's plate reader experiment, concentrations as low as .1mM completely stop cell growth!
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">NM+his, NM, and 5-FOA concentration gradient in selection strain+pSR01</DIV></DIV></DIV><UL><LI>Also, another nice growth curve showing pSR01's rescue of the selection strain in incomplete media:
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">selection strain and pSR01 in NM media</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Team_Wolfe-ZF_Collaboration">Team Wolfe-ZF Collaboration</SPAN></H3><H4><SPAN class="mw-headline" id="Midiprep">Midiprep</SPAN></H4><UL><LI>Note: Instead of second lysate clearing spin, supernatant was transferred to falcon tube and poured into column
</LI><LI>Yields
</LI><LI>pSB4K5 = 58ng/µL, 260/280=1.84
</LI><LI>Zif268 = 168.52 ng/ul, 260/280 = 1.84
</LI></UL><P>More data on the concentrations can be seen below:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Nanodrop data for midiprep.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>The bands for the midiprep look similar to those from the miniprep using the new ethanol precipitation protocol, except they are much brighter.  In the following gel image, there were 8 ul of plasmid loaded for both the miniprep and midiprep plasmids.  It appears that there is much, much less miniprep plasmid than midiprep.  Considering that the midiprep plasmid is at approximately 168.52 ng/ul, the miniprep plasmid is probably around our old yield.  This means that the Nanodrop reader LIED TO US!! :'( :'( :'(
</P><P>Here is the gel:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:178px;"><DIV class="thumbcaption">Miniprep vs. midiprep plasmid with equal loading volumes.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Team_ZF_31">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Checking_our_5_and_6_Ligation_Plate">Checking our 5 and 6 Ligation Plate</SPAN></H4><P>Colonies grew on all our plates! We ran our cross-junction PCR on 8 colonies each from the 450 ul plates and sent it out for sequencing. 
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Bands expected at around 1.5 kb instead of the usual 1.4</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Making_more_practice_plate_oligos">Making more practice plate oligos</SPAN></H4><P>We were running low on oligos 1 through 6 from our practice plate, so we did a PCR to make more, using the same protocol as before. We then PCR purified the product.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Results of our PCR purification</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Design_a_new_primer_for_GFP">Design a new primer for GFP</SPAN></H4><P>We designed a new primer for GFP, with a 7 bp spacer between the Shine-Dalgarno sequence and the start codon for GFP.  According to what we discovered online, it seems that the spacing between the ribosome binding site and the ATG start codon is very important for transcription efficiency.  We had previously thought that there were 8 bp from the <I>beginning</I> of the RBS to the ATG codon, but instead it is really 5-9 bp in between the <I>end</I> of the RBS and the ATG codon that is correct.  We thus added the spacer that comes directly from the Ambion website, which is AGGAGG<B>TATATAC</B>ATG (spacer bolded).  More info can be found here: [<A href="http://www.ambion.com/techlib/basics/translation/index.html" class="external autonumber" rel="nofollow">[21]</A>].
</P><H4><SPAN class="mw-headline" id="Practice_plate_oligo_pool">Practice plate oligo pool</SPAN></H4><P>This morning, we took out our digestion reactions from the thermocycler, containing all 18 practice plate target oligos mixed together. The tube containing the overnight reaction was a bit crushed by the machine, and it looks like we lost a bit of volume.
</P><P>Regardless, we performed a PCR purification on the product, and got a concentration of ~17 ng/ul.
</P><P>Then, we performed a ligation with the Oligo Pool and the CB Bottom backbone using the typical protocol. We performed a chemical transformation with the ligation product, plating at two densities using 50 ul of transformed bacteria and 450 ul.  We shall see tomorrow whether any colonies grew.
</P><H4><SPAN class="mw-headline" id="Yogurt_Plate">Yogurt Plate</SPAN></H4><P>Yesterday we plated 10 ul of <A href="http://www.stonyfield.com/products/stonyfield/smoothies/10oz-smoothies/peach-0" class="external text" rel="nofollow">Stonyfield Super Smoothie</A> from lunch, which contains <A href="http://www.stonyfield.com/products/stonyfield/what-makes-our-products-so-special/six-live-cultures" class="external text" rel="nofollow">six varieties of active yogurt bacteria</A>.  Today it grew colonies.  It smelled like yogurt this morning.  This evening it smelled like rotting yogurt.  Yuck.
</P><P>(UPDATE: According to Dan it smells like Swiss cheese.  Now it smells slightly better).
</P><H4><SPAN class="mw-headline" id="Prepping_for_western_blot_tomorrow">Prepping for western blot tomorrow</SPAN></H4><P>We inoculated cultures (4 ml each) in preparation for western blotting tomorrow to verify that the zinc fingers are indeed being expressed by IPTG.  Here is the setup:
</P><UL><LI>Kan - pze22g with GFP as a complete negative control (there should be no zinc fingers in there... hoping there's also no proteins with flag tags in there)
</LI><LI>Spec - Zif268 whole with 0 uM IPTG
</LI><LI>Spec - Zif268 whole with 2 uM IPTG
</LI><LI>Spec - Zif268 whole with 10 uM IPTG
</LI><LI>Spec - Zif268 whole with 100 uM IPTG
</LI></UL><P>We expect to see zinc finger expression to varying degrees in the cultures with IPTG, depending on the level of IPTG.  Hopefully we'll have enough protein and FLAG tag antibody to work with tomorrow.
</P><H3><SPAN class="mw-headline" id="Team_TolC_25">Team TolC</SPAN></H3><H4><SPAN class="mw-headline" id="Plate_Reader_3">Plate Reader</SPAN></H4><UL><LI>There appears to be a significant difference between the KN and KS strands, but only at high SDS levels i.e. 2% (lag of about 5 hours). At lower levels, there appears to be no significant lag, but a difference in the slope of the log phase.
</LI><LI>Even at 0 IPTG, there is expression of the ZF plasmid, suggesting leakiness (constitutively 'ON' promoter for the ZFP)
</LI><LI>WS and WN strands show lack of growth in the presence of SDS at any concentration (except 0) and PCR shows odd results (no band to 800bp bands for the product using tolC_seq_F and tolC_seq_R) add images here
</LI><LI>need to discard the WS and WN glycerol stocks into a 'bad glycerol stocks' box
</LI><LI>need to move the rpoZ MAGE oligos into the bad primers box
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Plate Reader of KN and KS at 2% SDS and varying IPTG conc. 8/9/11</DIV></DIV></DIV><H4><SPAN class="mw-headline" id="OZ052_and_OZ123_MAGE_into_ECNR2.2BKan-ZFB-wp.2Bzeocin_without_plasmid">OZ052 and OZ123 MAGE into ECNR2+Kan-ZFB-wp+zeocin without plasmid</SPAN></H4><UL><LI>Ran two rounds of MAGE for both OZ052 and OZ123 in the KN strain order to create the other two positive controls.
</LI><LI>time constants were 5.2s for the first round, and 5.6s for the second round. After the second round of MAGE, the cells appear to have been growing rather slowly. MAGE 2 was done at 2pm, and the cultures are barely cloudy at 6:28pm
</LI><LI>3rd round at 10:10pm, time constants are 5.4s and 4.6s
</LI></UL><H4><SPAN class="mw-headline" id="Lambda_Red_Recombination_of_zeocin_into_new_ECNR2_culture_for_rpoZ_knockout">Lambda Red Recombination of zeocin into new ECNR2 culture for rpoZ knockout</SPAN></H4><UL><LI>put the colony in 2ml of LB+amp to grow at 3pm. It should be ready for lambda red by 7pm, hopefully?</LI></UL><DIV id="810" style="display:none"><H2><SPAN class="mw-headline" id="August_10">August 10</SPAN></H2><H3><SPAN class="mw-headline" id="Team_TolC_26">Team TolC</SPAN></H3><H5><SPAN class="mw-headline" id="MAGE_OZ052_and_OZ123">MAGE OZ052 and OZ123</SPAN></H5><UL><LI>MAGE round 4a, time constants of 5.6s (OZ052) and 1.2s and 5.4s (OZ123). There appeared to be a spark for the OZ123 electroporation (Sarah)
</LI></UL><H5><SPAN class="mw-headline" id="Recombination_of_Zeocin_into_ECNR2">Recombination of Zeocin into ECNR2</SPAN></H5><UL><LI>Zeocin insert colonies appear to be growing extremely slowly, only 3 colonies after 14 hours, but may be due to rpoZ knockout
</LI><LI>Growing up culture in case we need to do lambda red of zeocin again
</LI><LI>PCR of the TolC, rpoZ and zeocin bands for colonies 1 and 2 off the zeocin plate.
</LI><LI>numbered, from 1-6, 1-3 is colony 1, 4-6 is colony 2. 1,4=TT 2,5=RR, 3,6=ZR
</LI><LI>PCR successful!! We have the real WN strain now. It should be ready for transformation tomorrow with the zif268 spec plasmid
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Zeocin/rpoZ and TolC PCR of lambda red colonies 8/10/11</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Team_Wolfe_25">Team Wolfe</SPAN></H3><H4><SPAN class="mw-headline" id="Lambda_Red_RpoZ_Knockout_Plates">Lambda Red RpoZ Knockout Plates</SPAN></H4><UL><LI>Small colonies grew after about 14 hours on zeocin plates
</LI></UL><H4><SPAN class="mw-headline" id="Rpoz_Knockout_Check_PCR">Rpoz Knockout Check PCR</SPAN></H4><UL><LI>Picked 24 colonies and used zeocin binding primers as a control and rpoZ binding primers to check for a knockout.
</LI><LI>48 10µL reactions in total (24 colonies X 2 different primer sets mentioned above)
</LI><LI>Primers: rpoZ_R with Zeocin_R and rpoZ_R with rpoZ_F
</LI><LI>64 degree annealing temperature
</LI><LI>90 seconds elongation
</LI></UL><P><B>Results</B></P><UL><LI>rpoZ was knocked out!
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Zeocin/rpoZ PCR of lambda red colonies 8/10/11</DIV></DIV></DIV><UL><LI>Reinoculated LB+zeo with culture 9 to use tomorrow for glycerol stocks and transformations
</LI></UL><H4><SPAN class="mw-headline" id="Digestion_of_pSB4K5_Plasmid">Digestion of pSB4K5 Plasmid</SPAN></H4><UL><LI>Gel with yesterday's digestion product suggests that neither of the two restriction enzymes (Xba1 or Spe1) cut our plasmid. This was determined by comparing the band sizes to those obtained by midiporep of the pSB4K5 (whole) plasmid.
</LI><LI>Plan of action: We will use smaller reactions to increase the relative concentration of enzymes and we will use slightly more BSA buffer.
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:137px;"><DIV class="thumbcaption">pSB4K5 digestion with Spe1  and Xba1</DIV></DIV></DIV><P><B>Today's Digestion of pSB4K5 Plasmid</B></P><UL><LI>Xba1
</LI><LI>16µL DNA (~1µg)
</LI><LI>2µL buffer4
</LI><LI>.5µL BSA
</LI><LI>1.5µL Xba1
</LI></UL><LI>Spe1
</LI><LI>16µL DNA (~1µg)
</LI><LI>2µL buffer4
</LI><LI>.5µL BSA
</LI><LI>1.5µL Spe1
</LI><LI>Xba1+Spe1
</LI><LI>14.5µL DNA (~1µg)
</LI><LI>2µL buffer4
</LI><LI>.5µL BSA
</LI><LI>1.5µL Xba1
</LI><LI>1.5µL Spe1
</LI><LI>Results: the size of the bands isn't what we expected (backbone should be ~3.4kb, insert ~1.2), but the comparison between the different digestions and the uncut plasmid does imply that the digestion we worked. We will go ahead and gel extract from the Spe+Xba digest and use both ligation and sequencing to be sure that this is indeed the plasmid we want.
</LI><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">pSB4K5 digest with Spe1, Xba1, and both 8/10/11</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Team_ZF_32">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Mini_library_ligation_results">Mini library ligation results</SPAN></H4><P>We checked our plates from our practice plate library assembly yesterday, and we got colonies!!  Strangely, our 450 ul plate didn't have a huge number more colonies than the 50 ul plate.  Dan suggested that this was because we plated all the cells in a 450 ul volume, which may have sloshed around and taken cells with it.  Next time we should pellet the cells and resuspend in 100 ul and plate that.  Based on our colony numbers with 50 ul, we should probably plate with 100 ul in the future.
</P><P>Due to the proliferation of colonies, we picked 90 colonies with the assistance of Will.  These 90 colonies should hopefully represent all 18 oligos in equal amounts.  We did colony PCRs on all 90 with the junction PCR protocol, and sent them out for sequencing today.  We saved these 90 colonies by adding LB/spec to each of the wells after suspending them in the 10 ul of water for the colony PCR.  We will eventually harvest and gather all 18 assembled plasmids from this step.
</P><H4><SPAN class="mw-headline" id="Sequencing_results_for_oligos_5_and_6">Sequencing results for oligos 5 and 6</SPAN></H4><P>Sequencing indicated that most of our colonies worked, and that our F1s were inserted correctly!  This is a huge step forward, and gives us good hopes that our 90 colonies also have their F1s inserted.  Out of the 8 colonies each that we tested for oligo 5 and 6, only 5.8 (missing chunk on promoter), 5.3 (deletion on linker), and 6.8 (bad trace file) were bad.  However, all the rest were good!
</P><P>We also now have an idea of our combined ligation/transformation efficiency.  NUMBERS HERE PLZ
</P><H4><SPAN class="mw-headline" id="Western_blot_to_detect_Zif268">Western blot to detect Zif268</SPAN></H4><P>Today we began a western blot in an attempt to detect Zif268 expression from IPTG induction.  Due to the fact that the WB gels only have 10 lanes, we dropped the 2 uM IPTG sample for now, keeping the negative control (pze22g + gfp), 0 uM IPTG, 10 uM IPTG, and 100 uM IPTG.
</P><P>Lysis buffer had the following composition:
</P><UL><LI>150 mM NaCl
</LI><LI>50 mM Tris-HCl
</LI><LI>5 mM EDTA
</LI><LI>1% Triton X-100
</LI><LI>1x proteinase inhibitor
</LI></UL><P>We used the following protocol for lysis:
</P><OL><LI>Spin down 3 ml of cells into a pellet, discarding the LB supernatant.
</LI><LI>Resuspend cells in 1.5 ml of ice cold lysis buffer, take 200 ul of this into a new tube and add 800 ul of cold lysis buffer.
</LI><LI>Sonicate at power level 2 in two sets of five bursts (each about one second long).  Be sure to wait 10 seconds in between sets and keep tube on ice to prevent overheating.
</LI><LI>Spin down at maximum speed for 5 minutes at 4 degrees.
</LI><LI>Take supernatant into a new tube.  This is the final protein lysis mixture.
</LI></OL><P>Note!  In the future, we should only spin down 200 ul of cells for lysis.  According to Alain, most of the time they even just use 100 ul and directly boil it with SDS.  May try this protocol tomorrow if current protocol doesn't work (the downside is that we have to start again from the culture stage and not the protein stage).
</P><P>Prepping samples for the gel:
</P><OL><LI>For loading duplicates, load 9 ul of 5x sample buffer with 36 ul of protein for a total of 45 ul.
</LI><LI>Parafilm tubes to prevent popping, and boil for 5 minutes at 100 degrees.
</LI><LI>Load 10 ul of protein ladder into gel, and load 15 ul of sample into each well.
</LI></OL><P>Note!  For this particular gel, the wells go in this order from left to right: Marker, neg con, 0 uM, 10 uM, 100 uM, neg con, 0 uM, 10 uM, 100 uM, Marker.
</P><P>Running samples on gel:
</P><UL><LI>Gel was run at 150 V until the marker band reached approximately 80% of the way to the bottom of the gel.
</LI></UL><P>Transfer:
</P><UL><LI>Semi-dry transfer was accomplished by wetting blotting paper and nitrocellulose membrane in transfer buffer and making the sandwich on the transfer apparatus.  From bottom to top (red to black electrode), the loading order was as follows:
</LI><LI>2 sheets of blotting paper
</LI><LI>Nitrocellulose membrane
</LI><LI>Gel
</LI><LI>2 sheets of blotting paper
</LI><LI>Transfer was run at 20 V for 1 hour.
</LI></UL><P>Note!  The side containing the protein is face up when the notch in the membrane is situated on the top right hand corner.  However, due to troubles with the gel, the lanes are now in the opposite order of loading.  So, with the protein right side up, the lanes go: marker, 100 uM, 10 uM, 0 uM, neg con, etc.  Also, we forgot to roll bubbles out from the sandwich with the pipette fragment before transferring (my bad!)  Hopefully this doesn't impact anything.  One last note on the transfer: the current cap was 400 mA, and when we started the transfer, it initially began at 17 V and 400 mA.  Over a few minutes, the voltage increased to 20 V and the current began to decrease, hitting 110 mA after a half hour of gradual decay, which is where we expected the current to be.
</P><P>The transfer seemed to have worked, transferring the marker bands from the gel onto the membrane.
</P><P>Blocking:
</P><UL><LI>Membrane was blocked at 4 degrees overnight.
</LI></UL><P>Tomorrow we will proceed with primary and secondary antibody staining and band visualization.
</P><H4><SPAN class="mw-headline" id="Library_ligation_optimization">Library ligation optimization</SPAN></H4><P>Dan suggests that we optimize the vector:insert ratio for our library assembly.  We will try using the following insert ratios (assuming a vector unit of 1): 1, 2, 3, 4, 8, 10, 16.  Unfortunately, today we did not have enough cut CB bottom plasmid to immediately begin the ligation and transformation, so we attempted to cut more CB bottom.  After performing a cut with Bsa-HF using the previous protocol, we performed a PCR purification on the product, only to find that there were only 3.4 ng/ul of cut plasmid present.  This is not enough to perform the ligation, and so we will have to try the CB bottom plasmid cut again tomorrow morning when the cultures have grown enough.  We can then do a ligation and transformation tomorrow and count colonies on Friday.
</P>
As stated in the earlier section, we should probably plate 100 ul of transformed bacteria to standardize everything and to keep colony counting fairly manageable.</DIV><DIV id="811" style="display:none"><H2><SPAN class="mw-headline" id="August_11th">August 11th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_Wolfe_26">Team Wolfe</SPAN></H3><H4><SPAN class="mw-headline" id="Ligation_of_the_HisUra_Insert">Ligation of the HisUra Insert</SPAN></H4><UL><LI>follow <A href="http://www.neb.com/nebecomm/products/protocol2.asp" class="external text" rel="nofollow">New England Biolabs protocol</A>:
</LI><LI>1.5µL of cut Spe ZFB-wp-hisura Xba insert (75ng,a 3-fold molar excess to backbone)
</LI><LI>6µL cut pSB4K5 backbone (50ng)
</LI><LI>1.5µL Water (bring volume up to 10µL)
</LI><LI>1µL Quick T4 DNA Ligase
</LI><LI>10µL 2X quick ligation buffer and mix
</LI><LI>spin briefly and incubate for 5 min at room temperature
</LI><LI>keep at -20˚C or on ice until transformation
</LI></UL><LI>Transformation: ligation mixture cannot be directly electroporated, so we will start with top10 chem comp cells (see protocols page)
</LI><LI>Following <A href="http://www.neb.com/nebecomm/products/protocol3.asp" class="external text" rel="nofollow">New England Biolab's transformation recommendations</A>, we used 2µL of ligation mixture in one tube of 50µL cells, and 1µL of unligated cut backbone (about an equal mass) in another tube of cells as a control.
</LI><H4><SPAN class="mw-headline" id="Transformation_of_pSB4K5_plasmid_with_His3Ura3_Insert_into_Chemically_Competent_Cells">Transformation of pSB4K5 plasmid with His3Ura3 Insert into Chemically Competent Cells</SPAN></H4><UL><LI>Transformed into delta HisB delta pyrF delta rpoZ Wolfe strain 
</LI><LI>Cells plated in 1mL and 2mL quantities on Kan plates after 3 hours recovery
</LI></UL><H4><SPAN class="mw-headline" id="Transformation_of_Zif268_Plasmid">Transformation of Zif268 Plasmid</SPAN></H4><UL><LI>Transformed into HisB delta pyrF delta rpoZ EcNR2 strain
</LI><LI>Cells were plated after 3.5 hours recovery on Spec plates at 10µL and 100µL
</LI></UL><H4><SPAN class="mw-headline" id="Transformation_of_the_pSB4K5_Plasmid_with_His3Ura3_Insert_into_Electrocompetent_Cells">Transformation of the pSB4K5 Plasmid with His3Ura3 Insert into Electrocompetent Cells</SPAN></H4><UL><LI>Transformed into delta HisB delta pyrF delta rpoZ Wolfe strain 
</LI><LI>3µL DNA from ligation reaction (unpurified)
</LI></UL><H3><SPAN class="mw-headline" id="Team_ZF_33">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Western_Blot">Western Blot</SPAN></H4><H4><SPAN class="mw-headline" id="PCR_to_get_out_GFP">PCR to get out GFP</SPAN></H4><P>Our new GFP forward primer arrived today, so we performed a PCR to get out the GFP region. Because the product will be used for gel extraction, we ran 4 reactions. We had an annealing temperature of ... and an extenstion time of 30 seconds.
</P><H4><SPAN class="mw-headline" id="Check_oligo_pool_sequencing_results">Check oligo pool sequencing results</SPAN></H4><P>Unfortunately, most of the samples we sent in for sequencing had problems. For many, the primers did not anneal; for others there were poor quality reads. 13 samples in total had good quality reads. Among those we got a pretty good distribution among our 18 oligos. This is promising. We will perform a PCR on 2 random samples to ensure that the cross junction is present. If we get positive results, we will send in 24 more samples for sequencing. 
</P><P>Summary of sequencing results:
</P><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Oligo</B></TD><TD align="center" style="background:#f0f0f0;"><B>Frequency</B></TD></TR><TR><TD> 5</TD><TD>3
</TD></TR><TR><TD> 6</TD><TD>1
</TD></TR><TR><TD> 8</TD><TD>1
</TD></TR><TR><TD> 9</TD><TD>1
</TD></TR><TR><TD> 10</TD><TD>1
</TD></TR><TR><TD> 11</TD><TD>2
</TD></TR><TR><TD> 12</TD><TD>1
</TD></TR><TR><TD> 17</TD><TD>1
</TD></TR><TR><TD> 18</TD><TD>2
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Finding_the_optimal_backbone:insert_for_ligation">Finding the optimal backbone:insert for ligation</SPAN></H4><P>We wanted to find the optimal backbone to insert ratio for ligation. Previously we had successfully used 100 ng of backbone, and a ratio of 1 to 8. We decided to try the following ratios: 
</P><UL><LI>1:1, 1:2, 1:3, 1:4, 1:8, 1:10, 1:16
</LI></UL><P>We ran out of cut CB Bottom backbone, so we needed to digest some more. We performed a miniprep on cultures that grew up overnight, using min-elute columns. Our final concentration of CB Bottom Plasmid was 50 ng/ul and 54 ng/ul.  
</P><P>We followed our usual protocol for digestion, and then performed a PCR purification, once again using a min-elute column to maximize our yield. However, the resulting product only had a concentration of ~5 ng/ul and a purity of ~1.6. We got similarly low yields yesterday, and are unsure where things are going wrong, since we have performed the same procedure in the past with success. Even if the plasmids weren't cutting, they should be present in the PCR purification product, so the low yields are puzzling. We decided to try a midi-prep tomorrow, so see if having more CB Bottom plasmids would help. 
</P><P>We grew up the following 100 mL cultures for midi-prep tomorrow: CB Bottom, Zif268.
</P><H3><SPAN class="mw-headline" id="Team_TolC_27">Team TolC</SPAN></H3><H4><SPAN class="mw-headline" id="Search_for_a_better_weak_promoter">Search for a better weak promoter</SPAN></H4><UL><LI>We have searched for a better weak promoter than the one we have now and compared their sequences to the present weak promoter in the kan-ZFB-wp construct, sequences were not similar enough to change the weak promoter through MAGE
</LI><LI>To change the promoter would be a big process, so we are thinking of using a negative selection of TolC using colicin and then creating a higher TolC production standard to begin with which would allow the KS and KN growth curves separate more
</LI></UL><H4><SPAN class="mw-headline" id="Transformation_of_Zif268_Plasmid_2">Transformation of Zif268 Plasmid</SPAN></H4><UL><LI>Performed transformation of the Zif268 plasmid into the good ECNR2+zeocin that definitely does have TolC
</LI><LI>Plated 1 mL of culture onto spec plate
</LI></UL><H4><SPAN class="mw-headline" id="Plater_Reader">Plater Reader</SPAN></H4><UL><LI>Kept the IPTG concentration constant at 100 µM while changing the SDS concentration from 2% to 2.5%, 3%, 3.5%, and 4%
</LI><LI>Making two wells per different culture
</LI><LI>Cultures are changing by SDS and IPTG concentration, while also changing from KS, KN, WS, WN colonies
</LI></UL><LI>Since colonies are not near mid-log we may have to fill plate reader tomorrow and run then as well
</LI><H4><SPAN class="mw-headline" id="Gradient_Plates">Gradient Plates</SPAN></H4><UL><LI>Created 5 50 mL total gradient plates with 2x Kan in the wedge layer, 20ul of Bluejuice in each 25ml of LB+agar+Kan to mark it with a dye.</LI></UL><DIV id="812" style="display:none"><H2><SPAN class="mw-headline" id="August_12th">August 12th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_Wolfe_27">Team Wolfe</SPAN></H3><H4><SPAN class="mw-headline" id="Transformations">Transformations</SPAN></H4><UL><LI>EcNR2 strain with transformed zif268 plasmid grew on spec plates: After 15 hours, 1mL and 2mL plates have ~20 colonies each
</LI><LI>Wolfe strain with transformed ligated pSB4K5 plasmid had no growth on kan plate at 1mL and unusual looking growth on a tet plate at 1mL after 12 hours
</LI><LI>The pSB4K5 plasmid was successfully transformed into the chemically coMpetent cells - will midiprep these to obtain a clean sample of ligated pSB4K5 for another transformation in electrocompetent Wolfe strain cells.
</LI></UL><H4><SPAN class="mw-headline" id="PCR_check_for_Zif268_plasmid_insert">PCR check for Zif268 plasmid insert</SPAN></H4><UL><LI>Looks like the plasmid's presence is confirmed by PCR in 8 colonies
</LI><LI>Team ZF's &quot;CJUN&quot; protocol: pZE23G 2123R and pZE23G 3581F primers
</LI><LI>Note well 5 is fainter but is probably due to a loading error where a small volume of sample was left out.
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">EcNR2 selection strain colonies transformed with zif268 8/12/11</DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Selection_strain_genotype_confirmation">Selection strain genotype confirmation</SPAN></H4><UL><LI>We will do one last sequencing round of our EcNR2 selection strain to confirm that HisB and PyrF are indeed knocked out.
</LI><LI>Same parameters as previously described; His_R and PyrF_F used for sequencing
</LI><LI>PCR was successful, as shown by an E gel of several of the reactions.
</LI></UL><H4><SPAN class="mw-headline" id="EcNR2_selection_strain_plate_reader_experiment">EcNR2 selection strain plate reader experiment</SPAN></H4><UL><LI>We want to see how the EcNR2 selection strain works with and without plasmid, especially in the presence of various concentrations of 3-AT and 5-FOA.
</LI><LI>Wells 1, 14, 27, 40, 53: LB+kan/spec
</LI><LI>Wells 2, 15, 28, 41, 54: NM+his
</LI><LI>Wells 3, 16, 29, 42, 55: NM
</LI><LI>Wells 4, 17, 30, 43, 56: NM+1mM 3-AT
</LI><LI>Wells 5, 18, 31, 44, 57: NM+5mM 3-AT
</LI><LI>Wells 6, 19, 32, 45, 58: NM+10mM 3-AT
</LI><LI>Wells 7, 20, 33, 46, 59: NM+25mM 3-AT
</LI><LI>Wells 8, 21, 34, 47, 60: NM+50mM 3-AT
</LI><LI>Wells 9, 22, 35, 48, 61: NM+0.1mM 5-FOA
</LI><LI>Wells 10, 23, 36, 49, 62: NM+0.25mM 5-FOA
</LI><LI>Wells 11, 24, 37, 50, 63: NM+0.5mM 5-FOA
</LI><LI>Wells 12, 26, 38, 51, 64: NM+1mM 5-FOA
</LI><LI>Wells 13, 27, 39, 52, 65: NM+2.5mM 5-FOA
</LI><LI>Wells 1-13: empty
</LI><LI>Wells 14-26: EcNR2∆HisB∆PyrF∆rpoZ+kan-ZFB-wp-his3-ura3 (6)
</LI><LI>Wells 27-39: EcNR2∆HisB∆PyrF∆rpoZ+kan-ZFB-wp-his3-ura3 (9)
</LI><LI>Wells 40-52: EcNR2∆HisB∆PyrF∆rpoZ+kan-ZFB-wp-his3-ura3 +Zif268 spec (1)
</LI><LI>Wells 53-65: EcNR2∆HisB∆PyrF∆rpoZ+kan-ZFB-wp-his3-ura3 +Zif268 spec (2)
</LI></UL><H3><SPAN class="mw-headline" id="Team_ZF_34">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Midiprep_of_Zif268_and_CB_Bottom_Plasmids">Midiprep of Zif268 and CB Bottom Plasmids</SPAN></H4><P>We performed a midi-prep on the Zif268 (whole) and CB Bottom plasmids. Zif268 was eluted in 100 ul of EB, while CB Bottom was eluted in 75 ul. 
</P><UL><LI>Zif had a final concentration of 261.45 ng/ul and a 260/280 of 1.90.
</LI><LI>CB Bottom had a final concentration of 237.93 ng/ul and a 260/280 1.91.
</LI></UL><H4><SPAN class="mw-headline" id="Results_of_Cross_Junction_PCR_of_2_random_colonies_from_ligation">Results of Cross Junction PCR of 2 random colonies from ligation</SPAN></H4><P>After running an e-gel we saw the expected 1.4 kb band. (The gel image is not available, because we were unable to get the software running at the time). We decided to proceed with sending 24 colonies out for sequencing.
</P><H4><SPAN class="mw-headline" id="Send_out_24_colonies_for_sequencing">Send out 24 colonies for sequencing</SPAN></H4><P>We performed a cross-junction PCR on the following colonies to send them out for sequencing: 
</P><UL><LI>1B, 1C, 1D, 1E, 1F, 1G, 1H, 2A, 3C, 3D, 3E, 3F, 3G, 3H, 4A, 4B, 10A, 10B, 10C, 10D, 10E, 10F, 10G, 10H
</LI></UL><P>We picked cultures that did not have sequencing results from our previous attempt. A 1 ul of a 1:10 dilution of each culture was used as template in the PCR. We used the Junction Reverse primer.
</P><H4><SPAN class="mw-headline" id="Isothermal_assembly_and_transformation_of_GFP_reporter">Isothermal assembly and transformation of GFP reporter</SPAN></H4><P>We ran our PCR product from yesterday on a 1% agarose gel. We got an appropriate band at approximately 700 bp. (Unfortunately, we could not get the imaging software to work at the time, so no image is available)
</P><P>We performed a gel extraction on our GFP PCR product, with the following yields:
</P><UL><LI>114.90 ng/ul; 260/280 of 1.89
</LI><LI>92.02 ng/ul; 260/280 of 1.94
</LI></UL><P>We then performed an isothermal assembly, using the following recipe:
</P><UL><LI>0.91 ul Zif 268 (backbone)
</LI><LI>0.27 ul GFP
</LI><LI>1.32 ul water
</LI><LI>7.5 ul isothermal assembly mix
</LI></UL><P>We transformed using ChemComp cells, plating 10 and 150 ul.
</P><H4><SPAN class="mw-headline" id="CB_Bottom_Digestion">CB Bottom Digestion</SPAN></H4>
We digested our CB Bottom plasmid, using our usual recipe. We ran the digestion for 2 hours at 37*, 20 minutes at 65*, and then let the reaction sit at 4* overnight.</DIV><DIV id="813" style="display:none"><H2><SPAN class="mw-headline" id="August_13th">August 13th</SPAN></H2><H3><SPAN class="mw-headline" id="Team_ZF_35">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="Sequencing_Results_2">Sequencing Results</SPAN></H4><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Oligo</B></TD><TD align="center" style="background:#f0f0f0;"><B>ID</B></TD><TD align="center" style="background:#f0f0f0;"><B>Notes</B></TD></TR><TR><TD> 2</TD><TD>1B</TD><TD>✓
</TD></TR><TR><TD> 4</TD><TD>3H</TD><TD>✓
</TD></TR><TR><TD> 5</TD><TD>10A</TD><TD>✓
</TD></TR><TR><TD> 5</TD><TD>1D</TD><TD>✓
</TD></TR><TR><TD> 5</TD><TD>1E</TD><TD>lots of snps at beginning
</TD></TR><TR><TD> 5</TD><TD>3G</TD><TD>✓
</TD></TR><TR><TD> 6</TD><TD>10C</TD><TD>✓
</TD></TR><TR><TD> 6</TD><TD>10G</TD><TD>✓
</TD></TR><TR><TD> 6</TD><TD>3C</TD><TD>✓
</TD></TR><TR><TD> 6</TD><TD>4A</TD><TD>2 deletions at F1
</TD></TR><TR><TD> 7</TD><TD>10B</TD><TD>✓
</TD></TR><TR><TD> 7</TD><TD>10D</TD><TD>✓
</TD></TR><TR><TD> 7</TD><TD>10E</TD><TD>✓
</TD></TR><TR><TD> 7</TD><TD>1F</TD><TD>✓
</TD></TR><TR><TD> 10</TD><TD>3E</TD><TD>✓
</TD></TR><TR><TD> 12</TD><TD>1C</TD><TD>deletion right before F1
</TD></TR><TR><TD> 12</TD><TD>1G</TD><TD>2 deletions at end of F1
</TD></TR><TR><TD> 14</TD><TD>1H</TD><TD>insert 'T' on F1
</TD></TR><TR><TD> 15</TD><TD>2A</TD><TD>✓
</TD></TR><TR><TD> 17</TD><TD>10F</TD><TD>✓
</TD></TR><TR><TD> 17</TD><TD>10H</TD><TD>✓
</TD></TR><TR><TD> 18</TD><TD>3D</TD><TD>✓
</TD></TR><TR><TD> 18</TD><TD>3F</TD><TD>✓
</TD></TR><TR><TD> 18</TD><TD>4B</TD><TD>✓
</TD></TR></TBODY></TABLE><P>Combined with the sequencing results from before, we are missing oligos 3, 13, 14, and 16.
</P><H4><SPAN class="mw-headline" id="Results_of_GFP_transformation">Results of GFP transformation</SPAN></H4><P>We have colonies on both of our plates! We still need to check if they actually express GFP. 
</P><H4><SPAN class="mw-headline" id="PCR_purification_of_CB_Bottom_digestion_product">PCR purification of CB Bottom digestion product</SPAN></H4>
We performed a PCR purification on the CB Bottom digestion, with a yield of 68.96 ng/ul and a 260/280 of 1.85. This will be used in a ligation in the future.</DIV><DIV id="815" style="display:none"><H2><SPAN class="mw-headline" id="August_15">August 15</SPAN></H2><H3><SPAN class="mw-headline" id="TolC">TolC</SPAN></H3><P>There appears to be very low selection (only on the order of 1:10) for the KS:KN strain.
</P><H3><SPAN class="mw-headline" id="Wolfe">Wolfe</SPAN></H3><P>Yay!!!
</P><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Selection strain with and without plasmid, and 3-AT 8/12/11</DIV></DIV></DIV><P>Interestingly, our 5-FOA concentrations did not seem very potent: 3-AT was more effective at retarding growth. We might drop the URA3 part of our selection and instead use higher amounts of 3-AT to distinguish between true hits and false positives.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Selection strain without Zif268 with 3-AT and 5-FOA 8/12/11</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Selection strain with Zif268 with 3-AT and 5-FOA 8/12/11</DIV></DIV></DIV></DIV><DIV id="816" style="display:none"><H2><SPAN class="mw-headline" id="August_16">August 16</SPAN></H2><P>In prioritizing our work for the next two weeks, we decided to focus on three main experiments:
</P><OL><LI>Preparing the oligo pool from the chip using Sri's protocol (using qPCR to amplify the pools equally)
</LI><LI>Continuing the experiment of picking out a positive hit amongst negative signal
</LI><LI>Biobricking our expression plasmids
</LI></OL><P>Eventually, the first two experiments will come together in terms of actually using the chip and figuring out how to pull out and select the positive binders.
</P><P>On average we expect about three students working at a time, so we'll have these three experiments running in parallel by each student.  So this means our previous teams will have to be dissolved, also since TolC will no longer be used :'(.
</P><P>Each experiment in more detail:
</P><OL><LI>We need to work out the protocol with Sri in order to prepare the oligo pools for ligation into our target plasmid backbones.  This will involve running a qPCR to monitor the amount of amplification so that the oligos in each pool will be amplified in equal amounts (so that we get a representative pool of oligos that is not biased to one side or the other).  We will need to do this with each subpool.  Futhermore, we will have to confirm that the primers that Sri provided are indeed specific enough to pull out their target subpools only and not other subpools.  This is important for the tweaking of our selection system, since we will need to know whether our selection system needs to be specific enough to pick out signal from 1:10,000 or 1:100,000.
</LI><LI>We will need to assemble a control practice library from the 18 practice plate oligos.  We will basically formulate a negative control pool with equal amounts of each of the 18 oligos, and spike this pool with varying amounts of Zif268.  We will then transform our selection strain with these pools with varying amounts of Zif268, and measure how specifically we can pick a winner out amongst non-winners (so instead of a bacteria dilution, it will be a plasmid dilution).  We will use 8 different concentrations:
</LI><LI>Positive control (all Zif268)
</LI><LI>1:10
</LI><LI>1:100
</LI><LI>1:1,000
</LI><LI>1:10,000 (this is approximately the ratio we expect for a winner amongst a pool of 9,149 non-winners for the chip subpools)
</LI><LI>1:100,000
</LI><LI>1:1,000,000
</LI><LI>Negative control (only contains the 18 practice plate oligos)
</LI><LI>We will start figuring out the procedure for converting our expression plasmids into Biobricks for submission.  This entails locating and eliminating all of the five specified restriction sites from our biobrick part (XbaI, SpeI, NotI, PstI, EcoRI), and assembling the part into a Biobrick submission plasmid (<A href="http://partsregistry.org/wiki/index.php/Part:pSB1C3" class="external text" rel="nofollow">pSB1C3</A>).  This may be accomplished through the use of isothermal assembly to combine the Biobrick shipping plasmid backbone with our Biobrick part (the omega subunit + zinc finger array).
</LI></OL><H3><SPAN class="mw-headline" id="Sarah">Sarah</SPAN></H3><H4><SPAN class="mw-headline" id="TolC_2">TolC</SPAN></H4><P>TolC selection abandoned.
</P><H4><SPAN class="mw-headline" id="Wolfe_2">Wolfe</SPAN></H4><P>There appears to be low selection for the Wolfe strain as well. i.e. 1:10. One possibility is that the wrong strain was picked, the other. In the absence of the 96 well plate, we will use the culture tube, and possibly the 100 well, plate reader plate.
</P><H3><SPAN class="mw-headline" id="Team_ZF_36">Team ZF</SPAN></H3><H4><SPAN class="mw-headline" id="GFP_reporter_strain">GFP reporter strain</SPAN></H4><P>We looked at the plate of GFP reporter that grew colonies and had the correct Shine-Dalgarno RBS with the correct spacer, putting it over the blue light of an e-gel box.  We have glowing colonies!  Only about seven of the ~250 colonies glowed, which may be due to an extra leaky promoter.  We then picked a glowing colony and put it in the warm room to grow overnight in preparation for miniprep.
</P><H4><SPAN class="mw-headline" id="Control_library_assembly">Control library assembly</SPAN></H4>
We picked 24 colonies from our plate which was transformed with the library mix of the 18 practice plate oligos and grew them up in LB/spec for miniprep tomorrow.</DIV><DIV id="817" style="display:none"><H2><SPAN class="mw-headline" id="August_17th">August 17th</SPAN></H2><H3><SPAN class="mw-headline" id="Justin">Justin</SPAN></H3><H4><SPAN class="mw-headline" id="Miniprep_of_negative_control_library_colonies">Miniprep of negative control library colonies</SPAN></H4><P>Today I miniprepped the 24 colonies that were picked and grown to saturation last night.  These 24 colonies should contain a good number of the 18 practice plate plasmids within them.  The minipreps were very successful for these strains, yielding concentrations from ~30-60 ng/ul.  Nida ran a junction PCR on the 24 miniprepped products, and we sent them out for sequencing in order to determine which of the 18 practice plate oligos are contained within these 24 colonies.  The yields of the minipreps can be seen below:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Miniprep concentrations and purities</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Selection_plasmid_concentrations">Selection plasmid concentrations</SPAN></H4><P>In order to proceed with the experiment #2 outlined yesterday, Nida and I combined the miniprepped products (everything except for sample 6, which had very low miniprep yields for some reason) in equal molar ratios according to their concentrations.  This constitutes our negative control library, containing complete zinc fingers targeted towards the CB bottom target, but not towards the Zif268 binding site.
</P><P>I then took the negative control library and spiked it with varying amounts of Zif268 whole plasmid, yielding mixes that contained in total 500 ng of plasmid.  Thus, for the positive and negative plasmid controls, they each contained 500 ng of Zif268 and 500 ng of negative control library, respectively.  For the 1:10 dilution, there were 50 ng of Zif268 and 450 ng of negative control, and so on.  The Zif268 plasmid that was used originated from the midiprep that Nida performed last Friday.
</P><P>The 8 mixtures outlined yesterday were then transformed into the EcNR2 selection strain, and plated on spec plates.  This will simply select for bacteria that contain plasmid, but not specifically for those that contain Zif268 plasmid.
</P><P>To see the volumes used for creating the plasmid dilutions, see the following file: <A href="/File:HARVPractice_Library_Assembly.xlsx" title="File:HARVPractice Library Assembly.xlsx">Practice Library Assembly</A></P><H3><SPAN class="mw-headline" id="Chip_library_assembly">Chip library assembly</SPAN></H3><H4><SPAN class="mw-headline" id="PCR_of_Chip_Oligos">PCR of Chip Oligos</SPAN></H4><P>With Sri and Dan’s guidance, I performed a qPCR on the chip oligo's using the protocol outlined in the supplementary section of Sri's paper. I performed six reactions using the appropriate primer in each case (CB top, CB bottom, FH top, FH bottom, Myc 981, Myc 198). I used the following recipe for the PCR (total volume 200 ul): 
</P><UL><LI>100 ul Sybr Green master mix
</LI><LI>200 nm primer (4 ul each of 10 um primer) 
</LI><LI>2 ul chip
</LI><LI>90 ul water 
</LI></UL><P>I ran the reactions on the thermocycler, while standing by to see the graphs being produced, using the following program:
</P><UL><LI>95* for 1 minute
</LI><LI>Initially, the reactions occur at an exponential rate. During this period: 
</LI><LI>95* for 10 seconds
</LI><LI>62* for 30 seconds
</LI></UL><LI>Once the reaction began to plateau, I stopped the reaction. Prior to this point all the oligos were being amplified equally. Once the plateau occurs, amplication occurs at varying rates. This took about 12 cycles total. 
</LI><P>FH Top samples had slightly less volume, and plateaued earlier. This may have occurred due to too little water or master mix being added. 
</P><P>I performed a PCR purification on the PCR product at got the following yields:
</P><PRE>Name        Conc.        260/280 
 CB Top:     18.2         1.83
 CB Bottom:  25.1         1.78
 FH Top:     20.3         20.3
 FH Bottom:  21.4         21.4
 Myc 981     26.1         1.73
 Myc 198     19.7         1.84
</PRE><P>In order to confirm the nanodrop readings, I ran 1 ul of each sample on a 4% e-gel. We ran a low range quanitative DNA ladder, at varying dilutions. 
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">The results of the qPCR on our library</DIV></DIV></DIV></TD></TR></TBODY></TABLE>
When comparing the intensity of the bands to the ladder, it does indeed look like there is approximately 20 ng/ul DNA in the products.</DIV><DIV id="818" style="display:none"><H2><SPAN class="mw-headline" id="August_18th">August 18th</SPAN></H2><H3><SPAN class="mw-headline" id="Justin_2">Justin</SPAN></H3><H4><SPAN class="mw-headline" id="Transformation_results">Transformation results</SPAN></H4><P>The plates of transformed bacteria yesterday were uneven in their efficiencies because some of the plates had a low time constant for the electroporation (around 4.8) when a good time constant was around 5.8.  We got excellent efficiency in the transformation for Zif268 alone, but for the plasmid mixes that included the negative control library, we got poor time constants.  We still got colonies, but instead of getting a lawn of bacteria there were approximately 150 discrete colonies on the plate.
</P><P>To get around this, Noah picked colonies from the plates and inoculated fresh LB/spec with them, and let them grow to saturation over the course of the day.  We used these resulting saturated cultures in our selection plates later in the day.
</P><H4><SPAN class="mw-headline" id="Pre-transformation_library_selection_plates">Pre-transformation library selection plates</SPAN></H4><P>Today, Noah and I prepared selection plates to test our ability to distinguish a positive hit among negative controls in varying proportions.  The pre-transformation aspect refers to the fact that it is the Zif268 plasmid that is diluted into the negative control plasmids before transformation, rather than diluting Zif268 bacteria into the negative control bacteria after transformation (this is analogous to how it will be done with the chip).  Using the negative control library assembled from yesterday, we created a 96-well plate with the following dimensions.
</P><P>Going horizontally (each column):
</P><UL><LI>1: Positive control (only Zif268)
</LI><LI>2: Positive control (replicate of column 1)
</LI><LI>3: 10^-1
</LI><LI>4: 10^-2
</LI><LI>5: 10^-3
</LI><LI>6: 10^-4
</LI><LI>7: 10^-5
</LI><LI>8: 10^-6
</LI><LI>9: Negative plasmid control (only negative control library)
</LI><LI>10: Negative plasmid control (replicate of column 9)
</LI><LI>11: No plasmid control (Wolfe ECNR2 strain without any plasmid)
</LI><LI>12: No plasmid control (replicate of column 12)
</LI></UL><P>Going vertically (each row):
</P><UL><LI>A: LB + carb
</LI><LI>B: NM + his (0.1%)
</LI><LI>C: NM
</LI><LI>D: NM (replicate of row C)
</LI><LI>E: NM + 1 mM 3-AT
</LI><LI>F: NM + 3 mM 3-AT
</LI><LI>G: NM + 10 mM 3-AT
</LI><LI>H: NM + 25 mM 3-AT
</LI></UL><P>We left this in the plate reader overnight.
</P><H4><SPAN class="mw-headline" id="Selection_plate_seeding_protocol">Selection plate seeding protocol</SPAN></H4><P>Notes: Each well contained 150 ul of liquid, and a 1/100 dilution of bacteria at an OD of 3 (so this would be adding 1.5 ul of bacteria).  The bacteria were grown up in LB to saturation and were washed in NM twice.  This was accomplished by the following steps:
</P><OL><LI>Take the equivalent of 800 ul of 1.5 OD bacteria into a tube.
</LI><LI>Centrifuge the bacteria into a pellet (max speed for 1 min)
</LI><LI>Pipette off as much LB as possible, using a 1000 ul pipet first and then switching to a 200 ul pipet.
</LI><LI>Resuspend the bacteria pellet in 500 ul of NM (1st wash).
</LI><LI>Recentrifuge into pellet.
</LI><LI>Repeat steps 3 and 4 (2nd wash). 
</LI><LI>Resuspend in 400 ul of NM (this gets it to an OD of 3).
</LI><LI>The bacteria sample is now ready to be used.
</LI></OL><H4><SPAN class="mw-headline" id="GFP_reporter_plasmid_miniprep">GFP reporter plasmid miniprep</SPAN></H4><P>Today, Nida and I miniprepped 6 ml of GFP reporter cells, consolidating the equivalent of 2 minipreps into one column.  However, the yields and purities were pretty terrible, with yields of 7 and 11 ng/ul and purities that were much greater than 2.  This is likely to be caused by the fact that the cultures that we miniprepped were prepared by spiking 3 ml of LB/spec with 1 ml of saturated culture in an attempt to grow up a lot of culture in a short period of time.  There was likely a lot of cell lysis and general cell debris that hindered the miniprep, causing it to give such terrible yields.
</P><P>We inoculated 12 more ml of culture for miniprepping tomorrow morning, since we are low on time.  This should hopefully give us enough culture to work with such that the miniprep will eventually give us the quantity/concentration of DNA required for transformation and/or plasmid library testing.
</P><H3><SPAN class="mw-headline" id="Chip_library_assembly_2">Chip library assembly</SPAN></H3><H4><SPAN class="mw-headline" id="Digestion_of_library_and_CB_bottom_backbone">Digestion of library and CB bottom backbone</SPAN></H4><P>We digested our CB bottom library pool and CB bottom plasmid with Bsa1 using the following recipes:
</P><UL><LI>Library (40 ul total)
</LI><LI>30 ul pool library (600 ng)
</LI><LI>0.75 ul enzyme
</LI><LI>4 ul buffer 4
</LI><LI>1 ul bsa
</LI><LI>4.25 ul water
</LI></UL><UL><LI>CB bottom backbone (10 ul total)
</LI><LI>4.2 ul dna (1000 ng)
</LI><LI>1 ul enzyme
</LI><LI>1 ul buffer 
</LI><LI>0.25 ul bsa
</LI><LI>3.55 ul water
</LI></UL><P>We ran the digestion for 2 hours at 37*. Then we heat inactivated the enzyme at 65* for 20 minutes.
</P><H4><SPAN class="mw-headline" id="Phosphatase_reaction">Phosphatase reaction</SPAN></H4><P>We then took the product of the digestion, and added phosphotase to prevent the pieces from re-ligating by removing the 5' phosphate. We used NEB Antarctic phosphotase. We used the following recipes:
</P><UL><LI>CB bottom backbone
</LI><LI>10 ul backbone
</LI><LI>1.22 ul buffer
</LI><LI>1 ul phosphatase
</LI></UL><P>We ran the reaction for 4.75 hours at 37*, and then heat inactivated the enzyme at 65* for 20 minutes. 
</P><P>We then performed a PCR clean-up on the product, with the following yields:
</P><UL><LI>Library: 9.8 ng/ul (total 294 ng)
</LI><LI>CB bottom: 11 ng/ul (total 330 ng)
</LI></UL><H4><SPAN class="mw-headline" id="Planning_for_ligation_of_library_and_backbone">Planning for ligation of library and backbone</SPAN></H4><P>We were unable to perform the ligation today. If we want to use 200 ng per ligation reaction we need 1000 ng of DNA, which is the amount we started with. We can cut some more backbone, or phosphatase the backbone I had cut earlier (conc: 69 ng/ul). We can also use 100 ng of backbone instead.
</P><P>Based on the yields of the PCR clean up, as assuming 100 ng of backbone, we'll need the following amounts of insert for our ligations:
</P><PRE>Insert:backbone   Amount of insert (ul)
 2:1               1
 3:1               1.5
 5:1               2.5
 8:1               4
</PRE><P>This is because our insert is 137 bp and our backbone is 2788 bp. For equal molar ratios, we'll need 4.9 ng (or 0.5 ul) of insert for a 1:1 insert:backbone.
</P>
We also plan on running cut backbone (no phosphotase), and phosphotased backbone as controls.</DIV><DIV id="819" style="display:none"><H2><SPAN class="mw-headline" id="August_19th">August 19th</SPAN></H2><H3><SPAN class="mw-headline" id="Plate_reader_4">Plate reader</SPAN></H3><UL><LI>Yesterday's plate reader experiment looked great! There is definitely a difference between 10^-6 dilution of Zif268 and the negative controls!
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">All conditions in NM (2nd duplicate) 8/18/11</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Comparison of 10^-6 dilution of Zif268 with negative library control 8/18/11</DIV></DIV></DIV><UL><LI>We will duplicate this experiment in a slightly different way to validate it.  Using the same media conditions, we will dilute the amount of Zif268 spec plasmid post-transformation rather than pre-transformation. (Instead of adding different amounts of plasmid to the cells being transformed, we will add different ratios of saturated EcNR2 selection strain+Zif268 spec and saturated EcNR2 selection strain+negative control plasmids.)
</LI><LI>For the protocol we will use in seeding the plate, see yesterday's entry.
</LI></UL><H3><SPAN class="mw-headline" id="GFP_reporter_Zif268_plasmid_transformation">GFP reporter Zif268 plasmid transformation</SPAN></H3><UL><LI>We also want to duplicate the plate reader experiment using our GFP reporter strain, so that we can show definitively that there is more Zif268 (represented by the GFP) in one well versus another.
</LI><LI>Miniprep of GFP Zif268 reporter plasmid: 151.5ng/uL, 260/280=1.89
</LI><LI>Transformed 1.5mL EcNR2 selection strain with 1uL plasmid (time constant of 4.8)
</LI><LI>Recovered 1hr in LB, and then added spec
</LI><LI>Grew 1hr in spec and then reinoculated 30 uL into a fresh tube of LB+spec to grow overnight (seeing as even with a poor time constant, 100 ul yielded over 100 colonies on the plate that Noah had tried earlier).
</LI></UL><H3><SPAN class="mw-headline" id="Chip_library_assembly_3">Chip library assembly</SPAN></H3><H4><SPAN class="mw-headline" id="Adding_phosphatase_to_CB_bottom">Adding phosphatase to CB bottom</SPAN></H4><P>Because we did not cut enough DNA yesterday, we decided to add phosphatase to a sample of CB bottom backbone we had cut before. This sample, which had already been PCR purified and eluted in 30 ul of EB, had a concentration of 69 ng/ul according to the nanodrop. 
</P><P>In order to make sure the sample was cut appropriately, we ran it and a sample of the CB bottom we cut yesterday on a 4% e-gel. 
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">A comparison of our old and new CB bottom backbones. Note, new has already been phosphotased.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>The band was much more intense for the old sample, and it looked like it had a concentration of between 40 and 100 ng/ul. We decided to perform the reaction on 20 ul of the cut sample (for approximately 1380 ng of DNA) and used the following recipe:
</P><UL><LI>20 ul DNA
</LI><LI>1 ul phosphatase
</LI><LI>2.33 ul phosphatase reaction buffer. 
</LI></UL><P>We incubated the reaction at 37* for 1 hour 10 minutes, and heat deactivated the enzyme by heating it at 65* for 20 minutes.
</P><P>We then PCR purified the sample, eluting in 30 ul of EB, and got a yield of 41.7 ng/ul, meaning we had a total of 1251 ng of backbone, which was more than enough for our ligation.
</P><H4><SPAN class="mw-headline" id="Ligation_of_CB_bottom_and_library_inserts">Ligation of CB bottom and library inserts</SPAN></H4><P>We performed a ligation using 100 ng of backbone and insert to backbone ratios of 2:1, 3:1, 5:1, and 8:1. We also ligated the cut plasmid and the phosphotased plasmid as controls. We used the following recipe for our ligation:
</P><UL><LI>2.4 ul backbone (100 ng)
</LI><LI>insert:
</LI><LI>2:1 -&gt; 1 ul
</LI><LI>3:1 -&gt; 1.5 ul
</LI><LI>5:1 -&gt; 2.5 ul
</LI><LI>8:1 -&gt; 4 ul
</LI></UL><LI>2 ul 10x ligation buffer
</LI><LI>1 ul ligase
</LI><LI>water to bring the total volume to 20 ul
</LI>
We ran the reaction at 16* for 12 hours, and then let sit at 4* forever.</DIV><DIV id="820" style="display:none"><H2><SPAN class="mw-headline" id="August_20th">August 20th</SPAN></H2><H3><SPAN class="mw-headline" id="Plate_reader_for_ECNR2_Wolfe">Plate reader for ECNR2 Wolfe</SPAN></H3><UL><LI>Yesterday's plate reader results: the trends are ok but the graphs are messy, probably because of condensation on the lid.
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">All dilutions in NM 8/19/11</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">All dilutions in NM+3mM 3-AT 8/19/11</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Comparison of Zif268 and 10^-4 dilution 8/19/11</DIV></DIV></DIV><UL><LI>This experiment is to figure out the appropriate concentration of IPTG, with varying concentrations of 3-AT.
</LI><LI>the concentrations are
</LI><LI>IPTG concentrations from 0,10 and 100mM (columns 1-4, 5-8, 9-12)
</LI><LI>3-AT concentrations from 1,3,10 and 25mM (rows E,F,G,H)
</LI><LI>LB (A), NM+histidine (B), NM (C &amp; D) 
</LI></UL><LI>The four strains are 
</LI><LI> EcNR2∆HisB∆PyrF∆rpoZ+kan-ZFB-wp-his3-ura3 + no plasmid (columns 1, 5, 9)
</LI><LI> EcNR2∆HisB∆PyrF∆rpoZ+kan-ZFB-wp-his3-ura3 + CB_bottom plasmid (negative control, library plasmid) (columns 2, 6, 10)
</LI><LI> EcNR2∆HisB∆PyrF∆rpoZ+kan-ZFB-wp-his3-ura3 + zif268 spec plasmid (selection strain with the matching ZFB and ZFP)(columns 3, 7, 11)
</LI><LI> EcNR2∆HisB∆PyrF∆rpoZ+kan-ZFB-wp-his3-ura3 + (zif268+GFP) spec plasmid (selection strain with the matching ZFB and ZFP, should fluoresce)(columns 4, 8, 12)
</LI><LI>We will take one fluorescence reading before placing it in the plate reader, and one after we finish it.
</LI><LI>In addition to the plate reader, we are curious to see how IPTG affects fluorescence levels. We made LB+spec plates with 0, 6.25uM, 25uM, and 100uM concentrations of IPTG
</LI><LI>Each plate was spread with 1uL of EcNR2∆HisB∆PyrF∆rpoZ+kan-ZFB-wp-his3-ura3 + (zif268+GFP) and left overnight at 30C
</LI><LI>Tomorrow we can compare fluorescence levels between the different IPTG concentrations.
</LI><H3><SPAN class="mw-headline" id="Chip_library_assembly_4">Chip library assembly</SPAN></H3><H4><SPAN class="mw-headline" id="Transformation">Transformation</SPAN></H4><P>We ran the ligation products through a min-elute column, eluting in 10 ul of water, with the following yields:
</P><UL><LI>Cut: 22.4 ng/ul
</LI><LI>Phosphotased: 24.2 ng/ul
</LI><LI>2 to 1: 25 ng/ul
</LI><LI>3 to 1: 25.9 ng/ul
</LI><LI>5 to 1: 20.1 ng/ul
</LI><LI>8 to 1: 20.9 ng/ul
</LI></UL><P>
Later we ran a transformation, using the turbo competent cells protocol. We used 100 ng of ligation product for the transformation:
</P><UL><LI>cut: 4.46 ul
</LI><LI>phosphotased: 4.13 ul
</LI><LI>2 to 1: 4 ul
</LI><LI>3 to 1: 3.86 ul
</LI><LI>5 to 1: 4.98 ul
</LI><LI>8 to 1: 4.78 ul 
</LI></UL>
We then plated 1 ul and 10 ul of cells on spec plates, and let them incubate at 37* for 12 hours. Only 10 ul plates were made for the controls. We took the rest of the transformed cells (excluding the controls) and let them incubate for 12 hours on the shaker at 37* in 20 mL LB spec.</DIV><DIV id="821" style="display:none"><H2><SPAN class="mw-headline" id="August_21st">August 21st</SPAN></H2><H3><SPAN class="mw-headline" id="Chip_library_assembly_5">Chip library assembly</SPAN></H3><H4><SPAN class="mw-headline" id="Results_of_library_transformation">Results of library transformation</SPAN></H4><P>Colonies grew on all the plates. However there were many colonies on the two control plates as well. There was a small amount of uncut (or nicked) plasmid according to the gel ran earlier, so this may be a result of that. Strangely, there were more colonies on the two control plates, than on the 2:1 and 3:1 10 ul plates. This does not make sense because one would expect that the controls demonstrate the amount of background, and the other plates should have more colonies on top of that in addition to the background noise. 
</P><P>The 5:1 plate had noticably more colonies than the others, including the control plates, making it the ideal insert:backbone for ligation.
</P><H4><SPAN class="mw-headline" id="Minipreps_and_glycerol_stocks">Minipreps and glycerol stocks</SPAN></H4><P>We performed a 4 2 mL minipreps on the 5:1 culture, eluting in 30 ul of EB, and got the following yields:
</P><UL><LI>1: 86.5 ng/ul
</LI><LI>2: 94.0 ng/ul
</LI><LI>3: 90.8 ng/ul
</LI><LI>4: 94.2 ng/ul
</LI></UL><P>We also made 4 glyercol stocks of the 5:1 culture, using 333 ul of 55% glycerol and 1500 ul culture.
</P><H4><SPAN class="mw-headline" id="Cross_junction_PCR_of_library_colonies">Cross junction PCR of library colonies</SPAN></H4><P>In order to see whether our colonies actually had the insert, we decided to run a cross junction PCR on 9 colonies and the CB bottom plasmid (as a control). We diluted each colony in 10 ul of water and used that as our template. We had a 12.9 ul reaction volume. 
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Attempt at cross junction colony PCR on transformed library‎</DIV></DIV></DIV></TD></TR></TBODY></TABLE>
Unfortunately, it seems as if the PCR did not work. This may be because too much template was loaded into the reaction. Next time, we plan on dliuting the colony in 50 ul of water and using 0.5 ul as template.</DIV><DIV id="822" style="display:none"><H2><SPAN class="mw-headline" id="August_22nd">August 22nd</SPAN></H2><H3><SPAN class="mw-headline" id="Zif268-GFP_results">Zif268-GFP results</SPAN></H3><UL><LI>IPTG spec plates: the plates with IPTG did have more bacterial growth (which might just be a plating error) but none of the plates fluoresced.
</LI><LI>Our endpoint fluorescence reads on the plate reader did not show any fluorescence, though this may have been due to an incorrect setup.
</LI><LI>Growth curves (see graphs below):
</LI><LI>GFP Zif strain grew in NM, showing that selection is working.
</LI><LI>In complete media, NM, and NM+1 or 3mM 3-AT, GFP Zif grew as well as the regular Zif268 strain. However, as 3-AT concentrations increased, GFP Zif began to grow considerably less compared to Zif268. The reason for this is unclear.
</LI><LI>There was no real difference between 0 and 10uM IPTG, but 100uM IPTG was still detrimental for cells containing zinc finger expression plasmids.
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">GFP Zif with 0 IPTG growth curves 8/20/11</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">NM media (1st replicate), all IPTG concentrations 8/20/11</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Zif268 and GFP Zif 3-AT gradient, 0 IPTG 8/20/11</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Wolfe_system_biobrick">Wolfe system biobrick</SPAN></H3><UL><LI>One of our biobricks will be the Wolfe selection system (ZFB, wp, his3, ura3).  We had started putting this on pSB4K5, but a PCR test of our ligation showed that none of the plasmids had the insert.
</LI><LI>picked 23 colonies, 10uL KAPA reactions, Spe1 ZFB_F and Xba1 URA3_R primers, 65 C annealing and 90 sec elongation
</LI><LI>used insert containing Spe1 and Xba1 sites as positive control
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">pSB4K5 ZFB hisura plasmid check 8/22/11</DIV></DIV></DIV><UL><LI>We will not continue putting the system on pSB4K5 and instead will focus on inserting it into the pSB1C3 shipping plasmid.  We ordered the proper primers and quikchange oligos (removing two Pst1 sites) to complete the system.
</LI></UL><H3><SPAN class="mw-headline" id="Chip_library_assembly_6">Chip library assembly</SPAN></H3><H4><SPAN class="mw-headline" id="Cross_junction_PCR:_Take_2">Cross junction PCR: Take 2</SPAN></H4><P>We picked 9 more colonies from the 1 ul 5:1 plate, diluted them in 50 ul water, and used 0.5 ul as our template in a cross junction PCR. We also ran the reaction on CB bottom as a control. We followed the usual cross junction protocol with the following changes:
</P><UL><LI>Initial denaturation: 6 minutes at 95* (instead of 5 minutes)
</LI><LI>Final extension: 10 minutes at 72* (instead of 5 minutes)
</LI></UL><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Second attempt at colony PCR on chip library</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>It looks like none of the colonies contain an insert. In order to try and figure out what went wrong, we ran the uncut CB bottom plasmid, cut plasmid, and cut and phosphatased plasmid on a 1% e-gel.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:178px;"><DIV class="thumbcaption">Attempt to compare uncut, cut, and phosphatased plasmids</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>The gel looked extremely strange, and the ladder was not present. We tried it again, this time running uncut plasmid, new phosphatased plasmid, and old phosphatased plasmid. 
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Comparison of uncut plasmid, new cut and phosphotased, old cut and phosphotased</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>The gel isn't perferct, but it looks like there was indeed uncut plasmid present after digestion in the sample used for ligation.
</P><H4><SPAN class="mw-headline" id="Digestion_of_CB_bottom">Digestion of CB bottom</SPAN></H4><P>We decided to cut more CB bottom, and try a second ligation, only using a 5 to 1 backbone to insert ratio for the ligation. We also decided to do a double digestion with both Bsa1 and Xba1 to minimize the amount of whole plasmid left over. We used the following recipe:
</P><UL><LI>4000 ng DNA (16.8 ul)
</LI><LI>8 ul Bsa1 (40 units)
</LI><LI>2 ul Xba1 (40 units)
</LI><LI>10 ul 10x buffer
</LI><LI>1 ul bsa
</LI><LI>62.2 ul water
</LI></UL>
We let the reaction run for 1 hour at 37* and then heat inactivated the enzyme at 65* for 20 minutes. We then performed a PCR clean up on the product, eluting in 50 ul of EB and had a yield of 45.4 ng/ul. We save 2.5 ul to use as a control in the transformation.</DIV><DIV id="823" style="display:none"><H2><SPAN class="mw-headline" id="August_23rd">August 23rd</SPAN></H2><H3><SPAN class="mw-headline" id="Selection_plates">Selection plates</SPAN></H3><UL><LI>We have shown that selection works in liquid media, but we would like to also be able to use plates with different 3-AT concentrations, since that would make it easier to pick individual colonies that grow successfully.  To check sensitivity, we plated 1uL of saturated culture of cells transformed with either all Zif268 spec, all negative control plasmids, or a mixture (10^-1 Zif, 10^-2...10^-6) on 3mM NM+3-AT plates.
</LI><LI>Did not see successful growth, though this may be because plates were old. An attempt with 100uL showed growth on both positive and negative control plates, likely due to the LB the cells were plated with.
</LI><LI>We now have fresh plates, so we will try again tonight.
</LI><LI>washed 100uL of positive control (100% Zif), 10^-1, 10^-2, 10^-3, 10^-4, 10^-5, 10^-6, negative control (0% Zif)with NM (500uL) two times and plated on NM+spec
</LI><LI>washed 100uL of EcNR2 selection strain that has not been transformed with any plasmids and plated on NM.
</LI></UL><H3><SPAN class="mw-headline" id="Changing_ZF_binding_site">Changing ZF binding site</SPAN></H3><UL><LI>We are trying to use MAGE, but the palindromic structure of the weak promoter gave the oligos ridiculous secondary structure, so we don't expect this to work. Instead, we are going to use lambda red to switch out the current kan-ZFB construct in the cells with a new construct containing a tetracycline antibiotic cassette and ZF binding site. By tying the ZFB change to cell survival, we hope to have greater efficiency.
</LI></UL><H3><SPAN class="mw-headline" id="Plate_reader_5">Plate reader</SPAN></H3><UL><LI>Repeated post-transformation dilutions of Zif with negative control library.
</LI></UL><H3><SPAN class="mw-headline" id="Chip_library">Chip library</SPAN></H3><H4><SPAN class="mw-headline" id="Phosphatase_of_newest_cut_backbone">Phosphatase of newest cut backbone</SPAN></H4><P>We used the following recipe to perform a phosphatase reaction on the backbone we cut yesterday with Bsa1 and Xba1:
</P><UL><LI>45 ul DNA
</LI><LI>1 ul phosphatase
</LI><LI>5.1 ul buffer
</LI></UL><P>We ran the reaction for an hour and heat inactivated the enzyme at 65* for 20 minutes. We then performed a PCR clean up on the product, with a yield of 52.6 ng/ul.
</P><H4><SPAN class="mw-headline" id="Ligation_of_CB_bottom_and_library">Ligation of CB bottom and library</SPAN></H4><P>We used the following recipe for the ligation with the insert:
</P><UL><LI>100 ng backbone (1.90 ul)
</LI><LI>2.5 ul insert (for a 5:1 ratio of backbone to insert)
</LI><LI>2 ul 10x ligation buffer
</LI><LI>12.6 ul water
</LI><LI>1 ul ligase
</LI></UL><P>To test the background we also ligated the cut and phosphotased plasmid, using the following recipe:
</P><UL><LI>100 ng backbone (1.90 ul)
</LI><LI>2 ul 10x ligase buffer
</LI><LI>1 ul ligase
</LI><LI>15.1 ul water
</LI></UL><P>
We added everything but the enzyme, and then heated the mixture at 37* for five minutes to separate any DNA fragment that were non specifically annealing. We then let it cool down back down to room temperature before adding the ligase. 
</P><P>We ligated the product for 10 minutes at room temperature. Then we cleaned up the product using min-elute columns, and eluting in 10 ul of water. We had the following yields:
</P><UL><LI>With insert: 27.4 ng/ul
</LI><LI>Control: 20.8 ng/ul
</LI></UL><H4><SPAN class="mw-headline" id="Transformation_of_CB_bottom.2Blibrary_and_controls">Transformation of CB bottom+library and controls</SPAN></H4><P>We then ran the following transformations using turbo comp cells, adding 100 ng DNA to our cells:
</P><UL><LI>Cut backbone: 2.2 ul 
</LI><LI>Cut and phosphatased backbone: 1.9 ul
</LI><LI>Cut, phosphatased, and ligated backbone: 4.81 ul
</LI><LI>Cut, phosphatased backbone ligated with insert: 3.65 ul
</LI><LI>Our old cut and phosphatased backbone (high background): 2.4 ul
</LI></UL>
We plated 10 and 1 ul and made 20 mL cultures of the backbone plus insert, and plated 10 ul of all others. All samples were grown in LB+spec overnight at 37*.</DIV><DIV id="824" style="display:none"><H2><SPAN class="mw-headline" id="August_24th">August 24th</SPAN></H2><H3><SPAN class="mw-headline" id="Plate_reader_results">Plate reader results</SPAN></H3><UL><LI>The cultures did not grow as well as they have in the past. This could be due to experimental error (the plate adapter was accidentally left in the reader overnight) or perhaps the cultures we inoculated with were too old. Another possibility is that the change from LB to NM-based media causes a metabolic shock, so we will try the plate reader experiment again with cultures grown up to saturation in NM+his (with the appropriate antibiotic).
</LI><LI>Summary: the dilutions did not grow as well as previous plate reader experiments had shown. See graph below for example.
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">1mM 3-AT 8/23/11</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Chip_library_2">Chip library</SPAN></H3><H4><SPAN class="mw-headline" id="Results_of_yesterday.27s_transformation">Results of yesterday's transformation</SPAN></H4><P>The background has significantly decreased, so it looks like adding the Xba1 helped.
</P><UL><LI>There is the possiblity that the background has artificially decreased because of adding that Xba1--that is, it went down because most of the plasmids have the Xba1 cut, and not the Bsa1 cut.
</LI></UL><P>However, there is not much of a difference between the control plates and the transformed plates, so it looks like our transformation did not work.
</P><UL><LI>It is possible that this occured because our Bsa1 enzyme has gone bad. We used new Bsa1 when cutting the backbone yesterday. We should try re-cutting the library with new Bsa1 and transforming with that to see it helps.
</LI></UL><H4><SPAN class="mw-headline" id="Cutting_the_library">Cutting the library</SPAN></H4><H4><SPAN class="mw-headline" id="Ligation_and_transformation">Ligation and transformation</SPAN></H4><H3><SPAN class="mw-headline" id="Selection_plates_2">Selection plates</SPAN></H3><UL><LI>The plates put after 16 hours in the incubator have a lawn of very small colonies growing very slowly. The selection with no plasmid is not growing on NM, as expected, but we do see colonies on the negative control library plate. However, there are clearly a lot more cells on the Zif268 positive control, and the dilutions seem to have colony amounts in between or similar to the negative control library plate.
</LI><LI>We will try several different variations:
</LI><LI>plate 1uL of each saturated culture on NM+spec (or plain NM in the case of the selection strain without plasmid)
</LI><LI>plate 100uL of each washed saturated culture on NM+10mM 3-AT
</LI></UL>
===Chip library===</DIV><DIV id="825" style="display:none"><H2><SPAN class="mw-headline" id="August_25th">August 25th</SPAN></H2><H3><SPAN class="mw-headline" id="Selection_plates_and_cultures">Selection plates and cultures</SPAN></H3><UL><LI>1uL NM plates: not much is growing on even the Zif268 control. 1uL may not be enough to plate in minimal media. We will let them continue growing to see if anything shows up.
</LI><LI>100uL 10mM 3-AT plates: looks very promising!  Zif268 has lots of small colonies, and the amount of growth steps down through the dilutions. The negative control library shows practically no growth at all. We will also keep these in the incubator to see if that helps makes the differences even clearer.
</LI><LI>NM+his liquid cultures: none of them are as saturated as we would like, so we'll leave them in longer.
</LI></UL><H3><SPAN class="mw-headline" id="Chip_library_3">Chip library</SPAN></H3><H4><SPAN class="mw-headline" id="Results_of_transoformation">Results of transoformation</SPAN></H4><UL><LI>We got colonies! We did not run a second background test yesterday, but based on the previous background plate (which contained the same backbone used in this reaction) most of our colonies should have the insert. 
</LI><LI>The best plate was the 2:1 insert to backbone ligation. The 8:1 reaction had a similar number of colonies, as well. The 3:1 and 5:1 plates had far fewer.
</LI></UL><P>We performed 4 2 mL minipreps (with yields of approximately 100 ng/ul for all) on the 2:1 culture and made 4 1.5 mL glycerol stocks. 
</P><H4><SPAN class="mw-headline" id="Cross_junction_check_of_library_colonies">Cross junction check of library colonies</SPAN></H4><P>To ensure that our plates actually had the insert, we ran a cross junction PCR on 8 colonies (diluting in 50 ul water). We also performed the cross junction PCR on some whole CB bottom plasmid, to see if there is a approxmiately 80 bp difference in the gel.
</P><P>We used the following recipe for the reaction:
</P><UL><LI>12.5 ul water
</LI><LI>10 ul kapa
</LI><LI>0.75 ul forward primer
</LI><LI>0.75 ul reverse primer
</LI><LI>0.5 ul template
</LI></UL><P><B>gel</B></P><P>The expected product size if there is an insert is about 1500 bp, while the product without the insert is about 1400 bp. It is difficult to see whether there is a difference in the sizes of the bands. 
</P>
We also sent the samples out for sequencing (with the reverse sequencing primer), to confirm whether or not our ligation was successful. If it is, then we will send out 96 samples for sequencing.</DIV><DIV id="826" style="display:none"><H2><SPAN class="mw-headline" id="August_26th">August 26th</SPAN></H2><H3><SPAN class="mw-headline" id="Biobricks">Biobricks</SPAN></H3><P><B>Zinc finger expression plasmid digests</B></P><UL><LI>Yesterday's digestions with LacI and LacIq did not work, so we tried again using a slightly different protocol from <A href="http://www.neb.com/nebecomm/products/protocol445.asp" class="external text" rel="nofollow">New England Biolab's Biobrick kit instructions</A>.  
</LI><LI>Got the same odd doublet again
</LI><LI>Digested one part with EcoR1 and one part with Spe1 to see if one of the enzymes would give us the expected ~50 bp fragment, but this also didn't work
</LI><LI>We have concluded that there must be something wrong with the LacI and LacIq sequences.  Possibly one of the restriction sites is missing. We will abandon this version of the biobrick for the time being.
</LI></UL><LI>pSB1C3 shipping plasmid: 1st PCR cleanup failed, so we redigested using the <A href="http://www.neb.com/nebecomm/products/protocol445.asp" class="external text" rel="nofollow">New England Biolab's Biobrick kit instructions</A>. Minelute produced a yield of 30 ng/uL, so a lot of DNA was lost but the concentration was still workable.
</LI><LI>Zinc finger + omega subunit: Mineulte produced 77.8ng/uL
</LI><P><B>ZFB-hisura biobrick digests</B></P><UL><LI>PCR to add restriction sites to insert:
</LI><LI>EcoXba ZFB_F and PstNotSpe URA_R primers; 10ng of Spe-ZFB-hisura-Xba as template; 25uL KAPA reactions
</LI><LI>60C annealing and 90 sec elongation
</LI></UL><LI>Minelute of 4 reactions combined = 616 ng/uL
</LI><LI>Digested with EcoR1 and Pst1 following NE Biolabs protocol
</LI><LI>Minelute: 65ng/uL
</LI><P><B>Ligation</B></P><UL><LI>Followed <A href="http://www.neb.com/nebecomm/products/protocol446.asp" class="external text" rel="nofollow">New England Biolab's Biobrick ligation protocol</A> for 3 conditions:
</LI><LI>pSB1C3 and ZF+omega
</LI><LI>pSB1C3 and ZFB-hisura
</LI><LI>pSB1C3 self-ligation control
</LI></UL><LI>Transformed into chemcomp cells
</LI><LI>Plated on LB+chloramphenicol
</LI><H3><SPAN class="mw-headline" id="Selection_strain_and_zinc_finger_library">Selection strain and zinc finger library</SPAN></H3><UL><LI>Lambda red to change the binding site worked for all the sites! All had colonies growing in tet, and sequencing confirmed that the binding site was changed.
</LI><LI>All were grown up in LB+tet and glycerol stocks were made
</LI></UL><LI>Since the CB bottom zinc finger library has been sequence verified, we will go ahead and transform it into the corresponding selection strain
</LI><LI>transformed with 100ng of library plasmids
</LI><LI>recovered for 1hr after electroporation
</LI><LI>plated 10uL on LB+spec, 10uL and 100uL on NM+spec, and reinoculated 1mL of transformation culture into 20mL of LB+spec
</LI><H3><SPAN class="mw-headline" id="Plate_reader_6">Plate reader</SPAN></H3><UL><LI>We repeated the zif268 dilution experiment using cultures that had been grown up to saturation in NM+his
</LI><LI>Empty selection strain, selection strain+Zif268, and selection strain+negative control library cultures were spun down, washed twice with NM, and resuspended to make and OD of 3. 1.5uL was used to inoculate each well.
</LI><LI>Media the same as the past week's plate reader experiments.
</LI></UL><H3><SPAN class="mw-headline" id="Chip_library_4">Chip library</SPAN></H3><H4><SPAN class="mw-headline" id="Sequencing_results_3">Sequencing results</SPAN></H4>
====Sending out a plate for sequencing====</DIV><DIV id="827" style="display:none"><H2><SPAN class="mw-headline" id="August_27th">August 27th</SPAN></H2><H3><SPAN class="mw-headline" id="Plate_reader_results_2">Plate reader results</SPAN></H3><UL><LI>Looks great! Not all of the wells grew perfectly (perhaps a loading error?) but there is a huge difference between 10^-6 dilution and the negative controls.
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">All cell types in NM (2nd dilution) 8/26/11</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Comparison of 10^-6 and negative controls 8/26/11</DIV></DIV></DIV></DIV><DIV id="905" style="display:none"><H2><SPAN class="mw-headline" id="September_5th">September 5th</SPAN></H2><H3><SPAN class="mw-headline" id="CB_Bottom_Library_Preliminary_Sequencing_Results">CB Bottom Library Preliminary Sequencing Results</SPAN></H3><P>Recap of what has happened:
</P><P>After the CB bottom library was transformed into <B>(turbocomp or selection strain?)</B> cells, 94 colonies were picked off of the plate after transformation.  These colonies were then sequenced, with the eventual objective of determining several factors about our transformed library:
</P><UL><LI>How much spread does our library cover?  Theoretically, the 94 colonies should be evenly spread throughout the 9,150 chip sequences, and the likelihood of a given sequence being represented twice is very low.
</LI><LI>Additionally, are the backbones being represented after transformation in the same ratios as they are on the chip?
</LI></UL><LI>What are the chip error rates?  The process of synthesizing the chip is error-prone, but we do not know what percentage of the oligos on the chip contain errors.  These errors can be SNPs or frameshifts from insertions/deletions.
</LI><P>Currently, the sequencing files are all located under sequences/CB bottom insert seq 08292011.
</P><P>29 sequences were analyzed by hand by importing the sequence files into SeqBuilder and manually identifying the position of the F1 motif based on its translation.
</P><P>Out of these 29 sequences, 6 showed a frameshift mutation, causing the F1 and F2/F3 to be expressed in different reading frames.  This indicates that based on these results, roughly 20% of the oligos on the chip contain frameshift mutations.  This may be ok, due to the remaining 80% being available.  The mutated oligos will simply be dropped out of the pool during selection due to creation of a nonfunctional protein.
</P><P>Furthermore, 2 of the 29 sequences contained SNPs.  One of them changed the F in &quot;FQCRICMRN&quot; to an S, while the other SNP introduced an early stop codon into the zinc finger, also making it non-functional.
</P><P>Out of the 94 colonies that were sequenced, 20 of the traces were sufficiently poor quality that no F1 was identified.
</P><P>For the future, we will need to create a script that will compare the sequenced results to the actual chip database.  This can be accomplished with the following steps:
</P><OL><LI>Perform restriction cuts on the chip in silico, that is, manually edit out the primer tag/type II binding site, leaving only the part that encodes the F1 finger.  This will be the dictionary that we will use to check all of our sequences against.
</LI><LI>Parse the sequencing data by identifying the part that codes for F1.  This can be accomplished with regular expressions.  One would need to identify the end of the omega/F1 linker (the last 30 or so nucleotides) and the beginning of the F1/F2 linker (the first 30 nucleotides).  Everything in between can be assumed to be F1 material.
</LI><LI>The program will then check to see if the F1 that is parsed from the sequencing data exists in the dictionary.  If it does, that is great.  If it does not, it will do one of two things:
</LI><LI>Perform a modulus 3 operation to see if there was a frameshift.  If the F1 is not a multiple of 3, it can be assumed that a frameshift occured.
</LI><LI>Look for the closest entry in the dictionary to the errant sequence (something off by 1 nucleotide, for example)
</LI></OL>
This will allow us to automate our efforts, and will make it easier when more sequencing is performed.  Vatsan recommends using BioPython to write the script.</DIV><DIV id="907" style="display:none"><H2><SPAN class="mw-headline" id="September_7">September 7</SPAN></H2><H3><SPAN class="mw-headline" id="File_Processing_for_Library_Sequencing">File Processing for Library Sequencing</SPAN></H3><P>Preliminary processing of the library sequences consists of agreggating the 94 individual fasta files into one multifasta file (which will be easier to work with) and taking the reverse complements of all the sequences (since we used a reverse primer for sequencing, the 100_bp_junction primer).
</P><P>A Python script for converting fasta into multifasta can be found here: <A href="http://sourceforge.net/projects/f2m2f/" class="external autonumber" rel="nofollow">[22]</A>, and a website for converting a multifasta file into its reverse complement can be found here: <A href="http://www.ualberta.ca/~stothard/javascript/rev_comp.html" class="external autonumber" rel="nofollow">[23]</A>.
</P><P>The reverse complement multifasta file for the 94 sequences thus far can be found in the dropbox under sequences/CB bottom insert seq 08292011 with the name CB bottom seq multifasta RC.fasta.
</P>
Remaining efforts include parsing the file to extract the F1 subunits, which will require either removing the newline characters and using regex or by using a regex that can recognize sequences across multiple lines (i.e. regex that recognizes sequences with newline characters randomly in there).  The chip dictionary still needs to be built and the code for comparing our sequences with the dictionary also still needs to be written.</DIV><DIV id="908" style="display:none"><H2><SPAN class="mw-headline" id="September_8th">September 8th</SPAN></H2><UL><LI>Biobricks:
</LI><LI>The plates looked good: the plates with the omega-ZF and ZFB-his3-ura3 inserts had many more colonies than the pSB1C3 self-ligation control.
</LI><LI>Chose 4 colonies from each biobrick and will grow them overnight. Tomorrow they will be miniprepped and sent out for sequencing using the standard primers on the backbone. (This should cover the entire insert for omega-ZF and most of the insert for ZFB-his3-ura3.)
</LI></UL><LI>Selection strains:
</LI><LI>Streaked out fresh plates from the glycerol stocks of the EcNR2 selection strain with the various zinc finger binding sites. We will hopefully use these when the other libraries are assembled (and we can use the CB bottom colonies for a fresh transformation with its library).</LI></DIV><DIV id="911" style="display:none"><H2><SPAN class="mw-headline" id="September_11th">September 11th</SPAN></H2><UL><LI>CB bottom ZFB strain:
</LI><LI>Repeated the transformation with CB bottom ZF library (100ng)
</LI><LI>recovered for 1 hr in LB
</LI><LI>plated on LB+spec to check that the transformation worked
</LI><LI>reinoculated 1mL of the transformation into 10mL LB+spec and let it grow for another hour
</LI><LI>plated 10uL (brought up to 100uL in NM) on LB+spec, NM+his+spec, NM+spec, NM+spec+1mM 3-AT, NM+spec+3mM 3-AT, NM+spec+10mM 3-AT plates
</LI><LI>let rest of LB+spec culture grow overnight to use tomorrow for glycerol stocks and more plating</LI></UL><DIV id="912" style="display:none"><H2><SPAN class="mw-headline" id="September_12th">September 12th</SPAN></H2><H3><SPAN class="mw-headline" id="Library_Construction">Library Construction</SPAN></H3><UL><LI>Overnight cultures were miniprepped (yields around 30-40ng/uL) and digested following the August 22nd protocol (but with 1ug of template plasmid)
</LI><LI>The digested plasmid was then immediately phosphatized by adding 1uL of Antarctic phosphatase and 10uL of the phosphatase buffer and incubating for 30min at 37C followed by 5 min at 65C.
</LI><LI>Minelute purified the product: recovered about half of the DNA
</LI></UL><H3><SPAN class="mw-headline" id="CB_bottom_library">CB bottom library</SPAN></H3><UL><LI>Made glycerol stock of CB bottom strain + CB bottom library
</LI><LI>LB+spec plates look great, so the transformation worked! Could be small colonies on the NM-based plates, but they are very small so we will let them grow up for longer
</LI><LI>Also want to plate washed cells from the transformation: washed twice in NM, and then plated either the equivalent of 100uL or 10uL of cells onto LB+spec, NM+his+spec, NM+spec, NM+1mM 3AT+spec, NM+3mM 3AT+spec, NM+10mM 3AT+spec plates. </LI></UL><DIV id="917" style="display:none"><H2><SPAN class="mw-headline" id="September_17th">September 17th</SPAN></H2><UL><LI>ZFB-his3-ura3 biobrick:
</LI><LI>Digested insert and pSB1C3 backbone with EcoR1 and Spe1 following NEBiolabs protocol
</LI><LI>Minelute: 50ng/uL and 17ng/uL respectively
</LI><LI>ligated according to NEBiolabs protocol (both with and without ZFB-hisura insert) and transformed into chem comp cells. Recovered for 1 hr and plated on LB+chloramphenicol
</LI></UL><LI>CB bottom library:
</LI><LI>Retransformed CB bottom ZFB strain with CB bottom library and Zif268 as a negative control (about 100ng each)
</LI><LI>Plated 10uL (unwashed) of both cultures on the full range of plates. The rest of the culture was inoculated with spec and left to grow overnight.
</LI><LI>Library assembly:
</LI><LI>Ran the digested and phosphotased samples on a gel with the uncut plasmids to verify that they were cut correctly.  The cut samples ran differently from the uncut, showing that the plasmids were successfully linearized.
</LI><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Digested and undigested zinc finger backbones 9/17/11</DIV></DIV></DIV></DIV><DIV id="919" style="display:none"><H2><SPAN class="mw-headline" id="September_19th">September 19th</SPAN></H2><UL><LI>PCR using VF2 and VR primers to check that biobrick assembly worked
</LI><LI>Results: bands were too small--insertion not successful
</LI></UL><LI>repeated digestion (NEBiolabs protocol) for ZFB-hisura and pSB1C3 with EcoR1 and Spe1</LI></DIV><DIV id="921" style="display:none"><H2><SPAN class="mw-headline" id="September_21st">September 21st</SPAN></H2><UL><LI>ligated and transformed ZFB-his3-ura3 biobrick into chem comp cells using same procedure as before
</LI><LI>CB bottom library plates: some of the plates looked like they may have colonies, so we picked a few to grow up in LB+spec for further confirmation.</LI></UL><DIV id="922" style="display:none"><H2><SPAN class="mw-headline" id="September_22nd">September 22nd</SPAN></H2><UL><LI>Ligation plates still had very few colonies, but when I repeated the PCR with the VR and VF2 primers, it looks like one of them (#18) does have the proper insert! Culture will grow overnight and be miniprepped tomorrow for Quikchange.
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">ZFB-his3-URA3 biobrick PCR: expected band size about 1.5kb</DIV></DIV></DIV><UL><LI>Miniprepped TolC selection strain KS to get a fresh batch of Zif268 on the spec plasmid
</LI><LI>26 of the 30 CB bottom cultures grew overnight in LB+spec, so I did a PCR of the pZE23G plasmid junction (same procedure as July 12th). Zif268 acted as a positive control. It appears that most of the colonies that grew do indeed have the zinc finger expression plasmid in them--they might be novel hits!!!!
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">CB bottom colony PCR: expected band size about 1.4kb</DIV></DIV></DIV></DIV><DIV id="923" style="display:none"><H2><SPAN class="mw-headline" id="September_23rd">September 23rd</SPAN></H2><H3><SPAN class="mw-headline" id="CB_bottom_hits:">CB bottom hits:</SPAN></H3><P><B>Plate reader:</B></P><UL><LI>Columns 1, 7: LB+spec
</LI><LI>Colummns 2, 8: NM+his
</LI><LI>Columns 3, 9: NM
</LI><LI>Columns 4, 10: 1mM 3-AT
</LI><LI>Columns 5, 11: 3mM 3-AT
</LI><LI>Columns 6, 12: 10mM 3-AT
</LI><LI>Row A: negative control (CB bottom +Zif268)(1-6) and D1 (7-12)
</LI><LI>Row B: D2 and C3
</LI><LI>Row C: B1 and B2
</LI><LI>Row D: B3 and B4
</LI><LI>Row E: B5, B6
</LI><LI>Row F: B7, B8
</LI><LI>Row G: B9, B10
</LI><LI>(D1-2 are from 10mM 3-AT plate, C3 from 1mM 3-AT, all Bs from 3mM 3-AT plate)
</LI></UL><P><B>Sequencing:</B></P><UL><LI>Repeat yesterday's PCR to be sent out for sequencing
</LI></UL><P><B>Biobricks:</B></P><UL><LI>Miniprepped chem comp cells with ZFB-hisura biobrick for Quikchange tomorrow. </LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Harvard/Notebook">http://2011.igem.org/Team:Harvard/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Harvard/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Harvard/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Harvard/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Harvard/Notebook&amp;oldid=244127" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>