"
Page
Discussion
View source
History
teams
Log in
 
Team:Harvard/Technology
From 2011.igem.org
bar
Overview | MAGE | Chip-Based Synthesis | Lambda Red | Protocols
Our project integrates 4 main technologies:
chip-based synthesis of DNA [1]
multiplex automated genome engineering (MAGE) [2] [3]
lamba red recombination [4] [5]
bioinformatics
Team members combined these technologies - 3 recently developed, one already widely used - in ways that no one else has tried: we successfully utilized all four over the course of our 10 week project. Considering that two of these methods were originally published less than 2 years ago (MAGE and chip synthesis), Harvard iGEM has reduced new ideas into practice: see our Protocols page and use our methods for other applications.
Although the project is the first to utilize several key technologies in novel ways, these technologies were developed outside of Harvard iGEM.  The multiplex automated genome engineering (MAGE) method was developed by Harris Wang et al [2] .  Chip-based DNA synthesis method was developed by Sri Kosuri et al [1] , and the actual oligo synthesis was generously provided by Agilent Technologies, a sponsor of iGEM. Lambda red was originally developed by Yu et al [4] .
By submitting the necessary strains and parts to the registry , and publishing these easy-to-follow protocols along with our source code , it is our hope that future iGEM teams will also use these techniques in their own synthetic biology projects.
Bioinformatics
See our Design page for details on the computational aspects of our project technology.
Our bioinformatics source code for our zinc finger sequence generator and zinc finger binding site finder is available here .
Chip-Based Synthesis
See Chip Synthesis for details about the technology, and read our Synthesize page for details on how we applied chip synthesis to zinc finger proteins.
Using microchip synthesis (provided by Agilent Technologies), we have 55,000 potential zinc fingers (whose sequences were generated by Team Harvard's bioinformatics ) to test. These fingers will then be tried against the DNA sequences we wish to bind.
Original Paper: http://www.nature.com/nbt/journal/v28/n12/full/nbt.1716.html
Our protocols for getting DNA pools off of the chip: https://2011.igem.org/Team:Harvard/Protocols#Chip_DNA_Extraction
MAGE
See MAGE for details about the technology, and read our Test page for details on how we used MAGE to build our selection strain.
Multiplex automated genome engineering (MAGE) is a new method for large-scale programming and evolution of cells. MAGE simultaneously targets many locations on the chromosome, thus producing combinatorial genomic diversity.
Original Paper: http://www.nature.com/nature/journal/v460/n7257/full/nature08187.html
Our Protocols: https://2011.igem.org/Team:Harvard/Protocols#MAGE
Lambda Red Mediated Recombineering
See Lambda Red for details about the technology, and read our Test page for details on how we used lambda red to build our selection strain.
Genes can be altered by recombination with linear DNA molecules. This requires a high internal DNA concentration, achievable by electroporation. The lambda red system allows efficient recombination between homologous sequences as short as 40 bp, which frees us of the need to provide long tracts of homology for recombination into the chromosome.
Gene Knockouts and Exchanges by Linear Transformation: http://rothlab.ucdavis.edu/protocols/Lin.Transform.html
Open Wet Ware Protocol: http://openwetware.org/wiki/Recombineering/Lambda_red-mediated_gene_replacement
Our Protocol: https://2011.igem.org/Team:Harvard/Protocols#Lambda_Red
Gibson (Isothermal) Assembly
See our Synthesize page for how we used Gibson Assemble to create our zinc finger expression plasmids.
An isothermal, single-reaction method for assembling multiple overlapping DNA molecules by the concerted action of a 5′ exonuclease, a DNA polymerase and a DNA ligase.
Original Paper: http://www.nature.com/nmeth/journal/v6/n5/full/nmeth.1318.html
Protocol: http://www.nature.com/protocolexchange/protocols/554#/main
Our Protocol: https://2011.igem.org/Team:Harvard/Protocols#Isothermal_assembly
References
1. Sriram Kosuri, Nikolai Eroshenko, Emily M LeProust, Michael Super, Jeffrey Way, Jin Billy Li, George M Church. (2010). Scalable gene synthesis by selective amplification of DNA pools from high-fidelity microchips. Nature Biotechnology, 28(12):1295-9. [1]
2. Harris H. Wang, Farren J. Isaacs, Peter A. Carr, Zachary Z. Sun, George Xu, Craig R. Forest, George M. Church. Programming cells by multiplex genome engineering and accelerated evolution. (2009). Nature, 460(7257):894-8. [2]
3. Isaacs FJ, Carr PA, Wang HH, Lajoie MJ, Sterling B, Kraal L, Tolonen AC, Gianoulis TA, Goodman DB, Reppas NB, Emig CJ, Bang D, Hwang SJ, Jewett MC, Jacobson JM, Church GM. (2011). Precise manipulation of chromosomes in vivo enables genome-wide codon replacement. Science, 333(6040):348-53. [3]
4. Yu D., H. M. Ellis, et al. (2000). An efficient recombination system for chromosome engineering in Escherichia coli. Proceedings of the National Academy of Sciences of the United States of America 97(11): 5978-5983. [4]
5. Mosberg JA, Lajoie MJ, Church GM. Lambda red recombineering in Escherichia coli occurs through a fully single-stranded intermediate. Genetics 2010;186:791-799. [5]
Retrieved from " http://2011.igem.org/Team:Harvard/Technology "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
