"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Harvard/Results/Acknowledgements
From 2011.igem.org
bar
Overview | Biobricks | Source Code | Acknowledgments | Accomplishments
Team Acknowledgments
Our project was selected from a pool of many possible project ideas submitted by both team members and advisers. Team members decided to focus on zinc finger proteins because of the technologies (chip DNA synthesis and MAGE) available to us: we believed that developing new zinc finger proteins was a way to use both technologies to their greatest possible extent. This project was not an extension or a continuation of any existing work on zinc fingers, in the Church lab or otherwise. The original idea of zinc finger proteins was contributed by the Harvard iGEM 2011 advisors and mentors, including but not limited to, Jagesh Shah, Srivatsan Raman, Daniel Goodman, Noah Taylor, Jameson Rogers, Joyce Yang, and Jun Li.
This project was made possible by support from Dr. George Church's lab at the Wyss Institute for Biologically Inspired Engineering as well as Alain Viel at the Harvard Department of Molecular and Cellular Biology.  Without their valuable guidance and great patience, we never would have been able to achieve what have done so far. While our teaching fellows - members of the Church Lab - gave us many opportunities, they also made sure that we achieved our goals and results entirely via our own work: our teaching fellows provided inspiration, team members achieved what we did through our own perspiration.
Although the project is the first to combine several key technologies in novel ways, these technologies were developed outside of Harvard iGEM.  The multiplex automated genome engineering (MAGE) method was developed by Harris Wang et al [2] .  Chip-based DNA synthesis method was developed by Sri Kosuri et al [1] , and the actual oligo synthesis was generously provided by Agilent Technologies, a sponsor of iGEM. Lambda red was originally developed by Yu et al [3] . We would like to make it clear that we do not claim any credit whatsoever for the creation of the technologies used in this project. Our project and Foundational Advance proposal rest on our successful use of and novel integration of these multiple technologies in concert, and subsequent application of this system to generate novel zinc finger proteins as a proof of concept.
We owe great thanks to Alain Viel for his flawless management of the logistics of iGEM, including but not limited to providing lab space, ordering lab supplies when we needed them, arranging our trip to the Americas Regional and World Jamborees, and for his excellent taste in footwear. Furthermore, we would also like to thank our lab manager Andrew Cumming for all of his practical lab advice and for helping to make everything run smoothly in the day to day logistics of lab.
Jagesh Shah provided us with excellent advice on both our project and the iGEM competition as a whole: what to expect, how to improve our presentation, and not to take ourselves too seriously. We also owe Jagesh's children Harvard iGEM t-shirts for depriving them of Jagesh's company when several of our meetings ran late into the night.
Team Member Acknowledgments
All project work was done by team members, divided according to their skills and expertise, with primary areas of work including:
Kristin Barclay
Bioinformatics , chip sequences design and programming , majority of wiki coding and design, human practices
Justin Chew
Bioinformatics , ZF target selection , ZF expression plasmid construction , Wolfe selection system testing , wiki coding and design, poster design
William Clerx
ZF target selection , wiki design, video editing, freelance wet lab, human practices
Sarah Choudhury
TolC selection system construction and testing and Wolfe selection system testing, biobrick construction
Naomi Genuth
Majority of one-hybrid selection system construction and testing , biobrick construction and registration
Brandon Gerberich
ZF target selection , Wolfe selection system testing , interview cameraman and video editor, human practices
Mark Kopelman
Wet lab
Matt Lunati
TolC selection system construction and testing
Nida Naushad
Bioinformatics , chip sequences design and programming , ZF expression plasmid construction , chip library assembly , biobrick construction
Adviser Acknowledgments
Finally, we would like to specially acknowledge all of our teaching fellows (TFs) who have always been there for us, no matter how small or big the problem.  Their advice has been of immeasurable value to our project's success.  We would also like to also acknowledge the following TFs for their special contributions:
Daniel Goodman
Harvard iGEM logo design , wiki design (contributer to CSS and design of the navigation bar), and being an all around awesome team leader
Noah Taylor
Used lambda-red recombination to successfully change our zinc finger binding sites when MAGE was impractical, contributed a reverse translating function from his own research to bioinformatics
Srivatsan Raman
Collaborator in our project abstract and human practices page
References
1. Sriram Kosuri, Nikolai Eroshenko, Emily M LeProust, Michael Super, Jeffrey Way, Jin Billy Li, George M Church. (2010). Scalable gene synthesis by selective amplification of DNA pools from high-fidelity microchips. Nature Biotechnology, 28(12):1295-9. [1]
2. Harris H. Wang, Farren J. Isaacs, Peter A. Carr, Zachary Z. Sun, George Xu, Craig R. Forest, George M. Church. Programming cells by multiplex genome engineering and accelerated evolution. (2009). Nature, 460(7257):894-8. [2]
3. Yu D., H. M. Ellis, et al. (2000). An efficient recombination system for chromosome engineering in Escherichia coli. Proceedings of the National Academy of Sciences of the United States of America 97(11): 5978-5983. [3]
Retrieved from " http://2011.igem.org/Team:Harvard/Results/Acknowledgements "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
