<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Ehime-Japan_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Ehime-Japan/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Ehime-Japan/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Ehime-Japan/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Ehime-Japan/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Ehime-Japan/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Ehime-Japan/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>Ehime-Japan iGEM Team: Welcome</TITLE></P><DIV id="Ehime-Japan_wiki_content"><UL id="nav"><LI><A href="https://2012.igem.org/Team:Ehime-Japan">Home</A></LI><LI><A href="https://2012.igem.org/Team:Ehime-Japan/Project">Project</A></LI><LI><A href="https://2012.igem.org/Team:Ehime-Japan/Project#one">E.co-mail</A></LI><LI><A href="https://2012.igem.org/Team:Ehime-Japan/Project#two">E.co-Domino</A></LI><LI><A href="https://2012.igem.org/Team:Ehime-Japan/Project#three">E.cold-heat sensing system</A></LI><LI><A href="https://2012.igem.org/Team:Ehime-Japan/Project#four">E.colingual</A></LI></UL><LI><A href="https://2012.igem.org/Team:Ehime-Japan/Modeling">Modeling</A></LI><LI><A href="#">Data</A></LI><LI><A href="https://2012.igem.org/Team:Ehime-Japan/Parts">BioBrick parts</A></LI><LI><A href="https://2012.igem.org/Team:Ehime-Japan/PlasmidsAndStrain">Plasmids And Strain</A></LI><LI><A href="https://2012.igem.org/Team:Ehime-Japan/PrimerList">Primer List</A></LI><LI><A href="#">Human Practice</A></LI><LI><A href="https://2012.igem.org/Team:Ehime-Japan/HumanPractice-E.create">E.create</A></LI><LI><A href="https://2012.igem.org/Team:Ehime-Japan/HumanPractice-Questionnaire">Questionnaire</A></LI><LI><A href="#">Other</A></LI><LI><A href="https://2012.igem.org/Team:Ehime-Japan/Team">team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2012&amp;team_name=Ehime-Japan">team profile</A></LI><LI><A href="https://2012.igem.org/Team:Ehime-Japan/Notebook">note book</A></LI><LI><A href="https://2012.igem.org/Team:Ehime-Japan/Safety">safety</A></LI><LI><A href="https://2012.igem.org/Team:Ehime-Japan/Attributions">attribution</A></LI><P><FONT size="6">
Principles
</FONT></P><P>Light sensor genes</P><P><FONT size="3"><FONT color="green">
・Green light sensor :pJT118, pPLPCB(S) [1]
　</FONT></FONT></P><P>Ho1 and PcyA synthesize 3z-phycocyanobirin (PCB) depending on ferredoxin (Fd).
When PCB binds to Ccas, it catches light and affects the function of CcaS depending on the wavelength of the light. When the light is green,  autophosphorylation of CcaS and phosphate transfer to CcaR increase. When CcaR is phosphorylated, it promotes the transcription from the P<SUB>cpcG2</SUB> promoter. On the other hand, when the received light is red,  CcaS responds reversely, to cause depression of the P<SUB>cpcG2</SUB> promoter.
 In the original system, pPLPCB(S) provide Ho1 and PcyA, and pJT118 provides Ccas and CcaR in addition to the reporter lacZ gene under the control of PcpcG2.We replaced the LacZ gene in pJT118 with the gene coding for a fluorescent protein.</P><P><FONT size="3"><FONT color="red">・Red light sensor :pCph8, pJT106b, pPLPCB(S)</FONT></FONT></P><P>
Cph8 (a Cph1-EnvZ chimaeras) is expressed from P<SUB>LTetO-1</SUB> promoter in the phosphorylated ground state.
The Cph1 part of Cph8 is synthesized as the apo from, and holo Cph1 is formed when apo Cph1 ligates PCB.
Holo Cph1 can absorb light. When far red light (705 nm) is absorbed, the rate of phosphotransfer from the EnvZ
domain of Cph8 to OmpR and the affinity of the phosphorylated OmpR (OmpR-P) to the OmpC promoter is increased.
This raises transcription of the cI gene downstream an OmpC promoter. The cI repressor represses transcription of
a lacZ ORF under the control of the P<SUB>&amp;lamda;</SUB> promoter. On the contrary, when red light (650 nm) is absorbed, the rate
of phosphotransfer of the EnvZ part of Cph8 is decreased and less cI is expressed. As a result, the amount of transcription from P<SUB>&amp;lamda;</SUB> promoter is increased.
<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3053042/"> 
[1] Tabor, J. J. et al. (2011) Multichromatic control of gene expression in Escherichia coli. J. Mol. Biol., 405, 315-324.
<FONT color="black">
[2] Levskaya, A. et al. (2005) Engineering Escherichia coli to see light. Nature, 438, 441-442.

[3] Gambetta, G. A. and Lagarias, J. C. (2001) Genetic engineering of phytochrome biosynthesis in bacteria.
Proc. Natl. Acad. Sci. USA, 98, 10566-10571


[4] Mattison, K. and Kenney, L.J. (2002) Phosphorylation Alters the Interaction of the Response Regulator OmpR with Its Sensor Kinase EnvZ. J. Biol. Chem., 277
</FONT></A></P><P><FONT size="5">
Degradation system
</FONT></P><P>
In most bacteria, ssrA-tagged proteins are degraded by protease. In E. coli, the ssrA  tag has 11-amino acid sequence that is recognized mainly by the ClpXP protease. The tag has been minimized into the three amino acid LVA tag, and this minimum tag is utilized widely in synthetic biology and in iGEM for temporal expression and degradation of stable protein, such as GFP and repressors.
Mycoplasamas, however, do not have the ClpXP protease. The ssrA tag sequences are quite different from those of the other bacteria. It was found that the tag is degraded mainly by the Lon protease in Mesoplasma florum, a Mycoplasma species [1].So, we call the tag “Lon-tag”.
The Lon-tagged proteins of M. florum are efficiently recognized and degraded by the M. florum Lon protease. M. florum Lon does not degrade proteins bearing the E. coli-ssrA tag, and E. coli Lon does not efficiently degrade proteins bearing the M.  florum ssrA tag . 
In principle, Lon-tagged proteins in E. coli cells should be degraded more rapidly upon induction of the M.florum Lon protease.
</P><P>
[1] Gur, E. and Suer, R. T. (2008) Proc. Natl. Acad. Sci. USA, 105, 16113-16118.
Evolution of the ssrA degradation tag in Mycoplasma: Specificity switch to a different protease.
</P><P><FONT size="6">
Projects
</FONT></P><P><FONT size="5">
E.co-mail
</FONT></P><P>
In order to make a circulatable communication system, we used the green and red light sensor
plasmids: pJT122, pJT106b, and pPLPCB(S). The E.coli having these plasmids expresses lacZ
when the E. coli absorbs the green or red light. First, we constructed the plasmids below.
<FONT size="5">Fig. 1 PcpcG2-GFP-mf-ssrA(pJT122)
<FONT size="5">
Fig. 2 Pλ-mf.Lon-LVA(pJT106b)</FONT></FONT></P><P>
We replaced the lacZ of pJT122 with GFP＋ssrA tag (mf. lon) and the lacZ of pJT106b with 
mf.Lon+LVA. As a result, GFP＋ssrA tag is synthesized when the E.coli is exposed green light
and mf.Lon+LVA is expressed by red light. This mechanism is shown next. </P><P>

1. Expose one of the two test tubes with E. coli (pJT122, pJT106b, and pPLPCB(S)) to green light.

↓
</P>
The E. coli exposed to green light expresses GFP＋ssrA tag.
<P>
2. GFP emits its green light byactivation with UV (364 nm) and the light is transferred to the other test tube through an
optical fiber. 

↓
</P>
The other E. coli receives the green light and expresses GFP too.
<P>
3. Expose the E. coli to red light.
</P>
↓
<P>
mf.Lon+LVA tag is expressed and GFP is degraded by the Lon protease.
</P><P>
4. mf.Lon+LVA tag is degraded by ClpX preexisting in E. coli
</P>
↓
<P>
Return to No.1
</P><P>
Finally, we could construct an E-mail system by making this system smaller !!

Furthermore, the circulatable system could work well owing to the two degradation systems which
are independent each other. The details are on &quot;Degradation system&quot;.
</P><P>
Because it was difficult to regulate the quantitative control, we did not
carry out the experiment to check if mf-ssrA tag is recognized by mf-Lon more speedily than
the Lon and LVA tag from E. coli.  
</P><FONT size="5">E.co-mail AssayⅠ
</FONT><P>
We made 6 E. coli samples harvoring a combination of 3 plasmids including pPLPCB(S) in common for 
studying &quot;The effect of mf-Lon&quot;,&quot;The effect of LVA tag &quot;, JW4092 (Lon-deficient)
The combinations of the plasmids are as follows.
</P>
 
 
1. The effect of mf-Lon <P> pJT122 (GFP+mf-ssrA tag), pPLPCB(S)and either.
</P>

(1) pJT106b with the mf-Lon gene replacing the lacZ gene. 

(2) pJT106b without lacZ. 


2. The effect of mf-ssrA <P>pJT106b (mf-Lon) and pPLPCB(S), and either.</P>(3) pJT122 with the GFP+mf-ssrA tag gene replacing lacZ gene. 

(4) pJT122 with the GFP gene (no mf-ssrA tag) replacing the lacZ gene. 


3. The effect of LVA tag <P>pJT122 (GFP+mf-ssrA tag), pPLPCB(S), and either.</P>

(5) pJT106b with the mf-Lon+LVA tag gene replacing the lacZ gene or. 

(6) pJT106b with the mf-Lon gene (no LVA tag) replacing the lacZ gene. 
<FONT size="5">E.co-mail ResultⅠ
</FONT>
1. The effect of mf-Lon

(1)

(2)


2. The effect of mf-ssrA 
(3)
(4)



3. The effect of LVA tag
(5)
(6)<FONT size="5">E.co-mail DiscussionⅠ
</FONT><P>We could not tell based on Figures I and II whether the pair of mf-Lon and mf-ssrA tag worked. However, Figure III by itself could be interpreted as a result of degradation by mf-Lon: mf-Lon with the LVA tag should be less stable that without LVA, and the GFP with the mf-ssrA tag should be more slowly degraded by the LVA-tagged mf-Lon. It is possible that the expression of mf-Lon was lower in the cases of (1), (3), and (4).  </P><FONT size="5">E.co-mail AssayⅡ
</FONT><P>Next, we constructed the simple device to confirm whether GFP is expressed by the green light of GFP through an optical fiber. </P><P>With an optical fiber, one test tube containing GFP protein emitting by UV (364 nm) was connected to the other containing the E.coli and shaded. </P><P>Because it was difficult to regulate the quantitative control, we did not carry out the experiment to check if mf-ssrA tag is recognized by mf-Lon more speedy than those from E.coli. </P><FONT size="5">E.co-mail ResultⅡ
</FONT><TABLE border="3"><TBODY><TR><TH>Light intensity
2</TH></TR><TR><TD>Dark</TD><TD>40.201</TD></TR><TR><TD>Green light</TD><TD>46.427</TD></TR></TBODY></TABLE><FONT size="5">E.co-mail Discussion Ⅱ
</FONT><P>Above picture shows that E.,coli lit GFP light develops more than that inhibited proteosynthesis and set a darkroom.</P><FONT size="5"><P>E.co-mail Refecence
</P></FONT><P>Evolution of the ssrA degradation tag in Mycoplasma:
Specificity switch to a different protease.</P><P><FONT size="5">
E.co-Domino
</FONT></P><P><FONT size="3">Let's make toppling dominoes with E.coli !

</FONT></P><P>This project is based on an idea that, if E.coli transformants that emit green light when they catch green light are placed along a line on an agar plate, and if the end is activated with UV or blue light so that it emit green light, it would look like domino toppling with the green light moving along the line toward the other end. We planned to construct this light transfer system by using the red and green light sensor systems, the light receptor PCB catches light and activate transcription of a specific promoter upstream the lacZ reporter gene. So, we replaced the lacZ gene with the sequences coding for GFP and RFP.</P><FONT size="5">Firework mechanism</FONT> (using domino system)
<P>We are trying to draw a movie of a firework that goes upward and bursts scattering light-emitting pieces.
The basic green light sensor needs two plasmids.pPLPCB provides a light-sensing molecule that binds to CcaS molecule. pJT118 harbors the genes for CcaS and CcaR and the reporter lacZ gene under the control of the PcpcG2 promoter. CcaS activates　PcpcG2,when the light-sensing molecule catches green light, by activating phosphorylation of CcaR. We will substitute the lacZ gene with the GFP gene. Therefore, our green sensor should produce GFP in reaponse to green light.</P><P>We will draw, with E. coli harboring pPLPCB and the GFP-substituted pJT118,a picture of a trajectory of a rising firework on an ager plate, and will “fire”with a pilot light at the bottom. Under the blue LED light(or weak UV light),the bottom part of the picture should begin to emit green light that should activate the neighboring bacteria.
Because the GFP has a degradation tag, the emission will stop soon. It is exposed that, as a result, the green light will move upward along the trajectory until it reaches the top.</P><P>At the top, we put E.coli with pPLPCB and an RFP-substituted pJT118.This will sense the green light coming from the neighboring bacteria and emit red light.</P><P>At the last, we put E.coli with pPLPCB and an RFP-substituted pJT106b. This will sense the red light coming from the neighboring bacteria and emit red light. </P><P>


=== Part 2 ===





</P><H1><A name="two">=== The Experiments ===</A></H1><FONT size="5"><FONT color="pink">Eco-Domino</FONT></FONT><P>pJT118 harbors the lacZ reporter gene in addition to the green sensor components. So, we replaced the lacZ sequence with the DNA coding for GFP.pJT106b contains the sequence for the lacZ reporter for the red light sensor. So, we replaced it with the gene  for RFP. We also construct a pJT118 derivative containing the RFP gene.</P><P>With these plasmids, we believe that we could draw movie pictures on an ager plate canvas. Our first one would be that of a firework in which a ball of green light would go straight upward and burst, scattering lines of red light.
 The data for the plasmid construction experiments are shown below.</P>
 
Plasmid construction
<P>
1.We amplified a sequence in pJT118 spanning almost all of the plasmid except the region of the lacZ ORF, by PCR with KOD Fx Neo. (using the primers GCGGCCGCTCGAGTCTAATTTTTTTG and ATCTATCATAGATAAAGTTAGTAATTAAAC). The 5780bp fragment was obtained and gel-purified. </P><P>2.We amplified the sequence of the GFP ORF from BioBrick (BBa-E0040) using the primers CTTTATCTATGATAGATATGCGTAAAGGAGAAGAACTT and GACCTGAGCGGCCGCTTTGTATAGTTCATCCATGCCAT.The 750bp fragment was gel-purified.</P><P>3.We put the fragment from 1 and 2 together by the InFusion svstem (Clontech). The miniprep plasmid sample from acolony was checked by agarose gel electrophoresis (Figure1,the middlelabeled as118+GFP). The other plasmids were constructed in almost the same way, and checked on agarose gels (Figures 1and 2).</P>figure1  figur2 figure3figure4figure5<P>figur1  figur2  figur3 
figur4  
figur5  
fig. 1: An agarose gel separating the plasmids. center, 118+GFP; right, 118+RFP.  fig. 2: An agarose gel separating plasmids. 106b+RFP is on the third lane. fig, 3: Schematic representation of plasmid 118+GFP.  fig. 4: An illustration of 118+RFP. fig. 5: An illustration of 106b+RFP

</P><P><FONT size="5">

Assay

</FONT></P><P><FONT size="3">
The function of light sensor 

・In liquid medium
</FONT></P><P>  
In order to confirm if the light sensor system normally works, a control experiment (shown on
Table1) was carried out.
</P><P><FONT size="3"><TABLE border="3"><CAPTION>Table1. The assay of light sensor</CAPTION><TBODY><TR><TD>Under white light (42 hours)</TD><TD>Dark (42 hours)</TD></TR><TR><TD>pJT118 (GFP), pPLPCB(S)</TD><TD>No change</TD><TD>No change</TD></TR><TR><TD>pJT118 (RFP), pPLPCB(S)</TD><TD>No change</TD><TD>No change</TD></TR><TR><TD>pJT106b (RFP), pCph8, pPLPCB(S)</TD><TD>RFP was expressed</TD><TD>RFP was expressed</TD></TR></TBODY></TABLE></FONT></P><P><FONT size="3">

We transformed JT2 (ΔEnvZ strain) with the plasmids and picked up an individual colony to put into liquid medium. Incubation with shaking at 37℃ was continued for 42 hours. We expected that the E. coli under white light will express GFP or RFP, and on the other hand, the E. coli under Dark will not express them. Unfortunately, we were not able to see that the E. coli having pJT118 (GFP and RFP) worked. 
</FONT></P><P>
We wanted to try to find the best condition for GFP or RFP expression (green sensor), but time was running out, so we decided to use the red light sensor only for our project. 

</P><P><FONT size="3">
・On plate
To investigate the condition for expression of RFP, We incubated JT2 transformed with three plasmids (106b+RFP, cph8, and pPLPCB) with shaking at 37℃ for 16 hours, exposing white light to them. After centrifugation, the bacteria were plated on M9 agar and were incubated for 24 hours under three different conditions (at room temperature). 

1. Red light
2. White light
3. Dark
Result → RFP was expressed under the all condition.　 
</FONT></P><FONT size="">
・Make E.co-Domino
<P>Purpose: To measure time required for activation of the red light sensor by RFP light.
We useed JT2 transformed with the plasmids(106b+RFP,cph8, and pPLPCB). It was first incubated with shaking incubate on 37℃ was constituted for 12 hours in LB medium. After centrifugation, the medium was replaced with  100μL of buffer(MQ). The samples were put into the wells of a microtiter plate. They were observed under a UV light. We tried this experiment under two different conditions.1.Exposing UV 2.Dark<FONT size="4">Result</FONT>RFP was expressed in 9 hours.
</P><P>0hour→9hour</P>
Result and Discussion

・Compare the time for expressing RFP from the different cells  ( cell (1)-(3) ).

The cells are named (1)-(3) on above the picture.
<P>
(1) The E.coli on the top of E.coli expressing RFP (RFP-E.coli)

(2)The E.coli on the right of (1)

(3) The E.coli on the right of RFP-E.coli


RFP was expressed on the center of the cell.
</P><TABLE border="3"><CAPTION>Table2. The assay of E.co-Domino</CAPTION><TBODY><TR><TH>(1)</TH><TH>(2)</TH><TH>(3)</TH></TR><TR><TD>7 hours</TD><TD>9 hours</TD><TD>10 hours</TD></TR></TBODY></TABLE><P>
In comparison with the cell (2), the amount of RFP in the cell (3) was more. Also, the cell (1) and cell (2) expressed RFP.
From the result above, we found that the red sensor genes are induced by UV (364 nm) and UV+RFP. Furthermore, the condition of RFP+UV is more effective for red sensor genes than UV only.
</P><P>
Therefore, E.co-Domino will be realized by the red light sensor system. 
</P><H1><A name="three">=== The Experiments ===</A></H1><FONT size="5"><FONT color="orange">E.cold-heat Sensing System</FONT></FONT><P><FONT size="3"><FONT color="red">・Heat Shock Sensor</FONT></FONT></P><P>  This is a sensor that promotes expression of RFP at a particular temperature. We made two plasmids. One produces a signaling molecule, and the other receives the signaling molecule. The plasmids that produces the signaling molecule has a heat shock promoter(HSP). Transcription of luxI gene is regulated by a HSP. LuxI produces OHHL(3OC6HSL). The other plasmid has Ptet promoter and transcription of the luxR gene is constitutively regulated by the promoter. The luxR protein binds to OHHL upregurates and upregulated the luxpR promoter, causing transcription of the mCherry reporter.</P><P><FONT size="3"><FONT color="red">・Cold Shock Sensor</FONT></FONT></P><P>This is a sensor that promotes expression of GFP a particular temperature. The sensor also consists of two plasmids. The plasmid that produces signaling molecule has a cold shock promoter(CSP). Transcription of the lasI gene is regulated by a CSP. LasR produces 3OC12-HSL. The other plasmid has a constitutive promoter upstream, transcription of the lasR gene is regulated by the promoter. The lasR protein binds to 3OC12-HSL and upregulates the laspR promoter, causing transcription of the GFP.</P>
Figure1
Figure2　　　　　PlasmidⅠ　　　　　　　　　　　　　　　     　 　　PlasmidⅡ
Figure3
Figure4　　　　　PlasmidⅢ　　　　　　　　　　　　　　　     　 　　PlasmidⅣ
'''Heat Shock Sensor'''
Plasmid 1 (Fig.1） was made for BBa_K338081 and BBa_C0261 by ligation.
'''Cold Shock Sensor'''
Plasmid 2 (Fig.2 BBa_K794000) was made for BBa_S03878 and BBa_K328001 by ligation.
And we are planning to replace the origin(rep(pMB1)) of Plasmid 2 with the origin(p15A) of pPLPCB(S) by the InFusion system (Clontech).
Plasmid 4 (Fig.1) was made from BBa_S03878 and BBa_K328001 by repetition of the InFusion recombination
 (primer list 1～5).<DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Ehime-Japan/Project">http://2012.igem.org/Team:Ehime-Japan/Project</A>&quot;</DIV></FONT></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Ehime-Japan/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Ehime-Japan/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Ehime-Japan/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Ehime-Japan/Project&amp;oldid=238999" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>