Team:SCU-WestChina/Description
index
Home
Background
Project
Description
Proof
Parts
Measurement
Hardware
Modeling
Human Practice
Silver
Gold
Education and Public Engagement
Collaborations
Notebook
Notebook
Protocols
Medals
Team
Team
Attributions
Acknowledgement
InterLab
Description
Hyperuricemia and refractory gout are caused by the high urate concentration in the blood. Considering there                 are still debates on whether the drug should be used in the asymptomatic hyperuricemia patients and the drug                 resistance of the refractory gout patients, new approaches are eagerly needed. We constructed the urate metabolic                 pathway in the probiotic E. coli Nissle 1917 and applied it in the gut to reduce the urate concentration                 in the blood indirectly. In addition, we built a dialysis-like device combined with modified bacteria to                 utilize the urate directly in the blood. Our solutions provide a suitable, long-term and non-drug treatment                 for the asymptomatic hyperuricemia patients and an ultimate treatment for the refractory gout patients.
Reducing the Urate in the Gut
The Pathway Promoters Measurement Protein Expression Seperated Crude Extraction Mix Experiment Crude Extraction for Pathway Experiment YgfU, the Transporter The Parameters for Urate Consuming Enzymes in the Supernatant? Animal Experiment Future Work
Improvement of Gifu 2016 Project
The Improvement
Dialysis-like Device
The Device The Bacteria Biosafety of the Device Function of the Device
Reducing the urate concentration in the gut for HUA patients.
There’s a pathway to metabolite urate to allatonin existing in bacteria and plant. The process is shown in the Figure 1.                     Urate is catalyzed into hydroxyisourate by uricase (1.7.3.3), then hydroxyisourate can react spontaneously                     and form allantoin. Allantoin is highly soluble in water and can be easily excreted from the body. The                     abovementioned new uricase drug is based on this mechanism. There are another two enzymes (3.5.2.17,                     4.1.1.97) that can speed up the process from hydroxyisourate to allantoin.
Figure 1 The metabolic pathway design of the project. The dotted lines represent the enzymes which can accelerate the spontaneous                     reaction processes. Urate oxidase (uricase, EC 1.7.3.4) is an enzyme with copper bonds that catalyzes                     the oxidative opening of the purine ring of uric acid to form 5-hydroxyisourate. HIU hydrolase (3.5.2.17)                     catalyzes the second reaction of the pathway, hydrolyzing 5-hydroxyisourate (HIU), a product of uricase                     in the first step to form 2-oxo-4-hydroxy-4-carboxy-5-ureidoimidazoline (OHCU). OHCU is catalyzed by                     OHCU decarboxylase (4.1.1.97).
YgfU is a high-capacity transporter for uric acid in Escherichia coli, which is homologous to nucleobase transporters of                     the ubiquitous family NCS2. We plan to overexpress the protein to achieve greater efficiency of uric                     acid utilization.
We found detailed information about these metabolic enzymes and the transporter from the BRENDA database (Table 1)[1,2,3,4], and we                     plan to use the constitutive promoter family Anderson Promoter in iGEM database to express them. As for                     the non-regulated expression pathway, if we want to promote the speed of the reaction to the maximum,                     the relative expression quantity of the enzymes must be considered. The rate-limiting step determines                     the whole speed of the pathway. Based on that, the extra protein expression of other enzymes may cause                     systematic burden in our bacteria. To design a better pathway, we did the modeling of our urate metabolic                     pathway according to the known Km and Kcat parameters published before, so as to determine the proper                     promoters upstream of different genes in order to optimize the expression with lowest energy consumed.                     We got the best ratio of PucL and 4.1.1.97. However, the detailed parameters of PucM are still unknown                     because they are hard to measure according to previous studies. As a result, we decide to try different                     promoters on PucM to construct the optimum pathway. (For 3.5.2.17, we used the sequence from soybean                     at the first time because its kinetic parameters have been reported. But the protein can’t be expressed                     in our constitutive promoter gene expression structure.
Table 1 Characteristics of the three enzymes and transporter.
Our quantitative gene expression is based on the Anderson‘s Promoters. Anderson's promoters are the most widely used constitutive                     promoters family. Those promoters are well characterized by RFP fluorescence. However, a key problem                     of generalizing the results of characterizations is that the dynamics of gene expression are influenced                     by the protein coded for. The RFP expression result might not be compatible with our gene expression.                     In 2016 iGEM competition, team William_and_Mary tried to use insulator RiboJ to make relative expression                     levels similar between different proteins. They have verified this phenomenon as a paper published before.                     We adopted the same RiboJ insulator design to measure the 10 promoters from Anderson’s Promoters family                     by expressing eGFP to avoid the influence brought by different coding sequences for our pathway construction[5,6].
The reason why we start to do the measurement is that we need to calculate to evaluate if our pathway is efficient enough                     to reduce the urate in the gut based on the gut urate concentration, to theoretically avoid our pathway                     expressing unnecessary extra protein to reduce the cell burden as much as possible to maintain colonization. In this condition, absolute protein concentration expressed by different promoters should be known. We                     have to find out the relationship between eGFP quantitative protein concentration and the promoters.                     And that's the first time in iGEM to evaluate the absolute protein expression of a promoter with the                     help of insulator to predict the metabolic ability of a pathway. Our measurement provided a prospective                     for future teams to evaluate the efficiency of a pathway.
To achieve this, we decide to draw a standard curve of eGFP fluorescence-eGFP absolute protein quantity. We expressed the                     eGFP-6x his-tag in pET28a driven by T7 promoter. We purified the protein and drew the standard curve                     successfully (See measurement, protocols for purification and measurement experiment details). The final                     results are as follows.
Table 2 The relative expression capacity of the Anderson's promoters measured by SCU-WestChina 2017.
Figure 2 The relative expression capacity of the Anderson's promoters measured by SCU-WestChina 2017. (Bar table). J00 is                     short for J23100, J04 is short for J23104, etc.
Based on the eGFP measurement and modeling, we designed our molecular biology experiment. Considering the urate concentration                     in the gut (about 100uM) and E. coli Nissle 1917 expression ability, we chose J23100 for YgfU and pucL                     expression, J23113 for 4.1.1.97 expression. (See modeling and measurement for details)
We constructed 16 parts for our pathway evaluation.
Table 3 The parts related to urate metabolic pathway submitted.
BBa_K2334001-BBa_K2334004 are generators of the enzymes and transporter we planned to use. The strongest Anderson’s Promoter                     J23100 is used in these parts to test if the proteins are expressed. Parts were transformed into E. coli BL21 for expression. We detected our target protein by SDS-PAGE.
It’s obvious that PucL, PucM, 4.1.1.97 were expressed successfully. However, the membrane protein YgfU wasn’t detected in                     the lysate precipitation after centrifugation. As an application-targeting project, we didn’t continue                     to verify the expression of the protein by Western Blotting or qPCR detection. We tested its function                     by adding this gene directly into the pathway to see if it can help the pathway function, which will                     be discussed later.
Figure 3 pucL, pucM, 4.1.1.97 expressed successfully driven by J23100+RiboJ+B0034 gene strucutre. The proteins expressed                     are noted in the figure. However, the expression of YgfU was not detected.
To test the function of the metabolic enzymes we expressed (pucL, pucM, 4.1.1.97), we used the crude bacteria extraction                     (expressed in
E. coli BL21) to test directly. Before we started to react, the total protein quantity                     of each sample was made the same. 100ul reaction extraction was added into 900ul PBS with urate (PH=8.0).                     HPLC was performed after reaction for 2h and heat for 10min. The components of reaction extraction are                     described in the table with brackets. (See protocols for more details). The result shows that, pucM and                     4.1.1.97 didn’t help much in the pathway in this experiment, which is hard to explain because it’s not                     consistent with the experiment we would describe later (Table 4) . Theoretically, after adding pucM and                     4.1.1.97, the reactions can attain equilibrium more quickly and perform better in urate consuming when                     measured at the same time.
Table 4 Results of separated crude extraction mix experiment. Only with pucL the urate concentration can decrease. The numbers                     refer to the HPLC original peak area.
We still ligated two (pucL + pucM) enzymes system and three enzymes (pucL + pucM + 4.1,1,97) in pathway in one plasmid (transformed                     in E. coli BL21) to test the function (Table 5). Promoters for pucM are chosen based on the range of                     their expression ability. We wanted to elevate the expression level of pucM as much as possible. So,                     J23100, J23106, J23107, J23108, J23117 were chosen. Those pathways with different promoters were submitted                     as parts and details are in the following table.
Table 5 The pathway constructed. PucM is driven by promoters with different strengths.
We used the crude bacteria extraction to test the whole pathway function directly. Before we started to react, the total                     protein quantity of each sample was made the same. 100ul crude extraction was added into 900ul PBS with                     urate (PH=8.0). HPLC was performed after reaction for 2h and 100 ℃ heat for 10min.The results show that,                     pucM did work (Figure 4, Figure 5). The tendency can be explained as follows: when the promoter is too                     strong, it causes excessive consuming of energy in the bacteria, and the expression of the main enzyme                     pucL is thus limited; when the promoter is too weak, the reaction can’t attain equilibrium as quickly                     as with a stronger promoter. So, when measured at the time, if the equilibrium was not attained, the                     performance of the pathway with weaker promoter would be worse. Our modeling result is consistent with                     our experiment result here. However, the result of separated crude extraction mix experiment can't be                     explained with the same theory, because when the continuous measurement was performed, performances of                     different reaction systems with PucL protein were always the same when measured at a time.
Figure 4 The result of crude extraction experiment of BBa_K2334007, BBa_K2334008, BBa_K2334009, BBa_K2334010. The numbers                     in the figure refer to HPLC original peak area urate decrease. The result shows that the expression ability                     of pucM does influence the performance of urate consuming. J23106 perform best in the group.
Figure 5 The result of crude extraction experiment of BBa_K2334012, BBa_K2334013, BBa_K2334014, BBa_K2334015, BBa_K2334016.                     The numbers in the figure refer to HPLC original peak area urate decrease. The result shows that the                     expression ability of pucM does influence the performance of urate consuming. J23106 perform best in                     the group. We compared the effect of pucL together in this group.
As for YgfU, we have mentioned that we couldn’t detect the protein expression via SDS-PAGE. We constructed the part BBa_K2334006,                     which consists of K2334004 (J23100 + RiboJ + B0034 + YgfU, Urate Transporter Generator) and K2334001                     (J23100 + RiboJ + B0034 + pucL, Urate Oxidase Generator).
We tested the function of YgfU by detecting the urate concentration in the overnight cultured LB medium. In this experiment,                     we used pucL(K2334001), pucL + YgfU(K2334004), eGFP(J23100 + RiboJ + B0034 + eGFP, not submitted) in                     pSB1C3 vector (transformed E. coli BL21). The result shows that, those constructions can’t reduce the                     urate concentration in the LB medium (Figure 6).
Figure 6 pucL, YgfU + pucL & eGFP were cultivated in urate LB medium overnight (Urate concentration was about 500uM). The                     initial OD600 of the LB medium was controlled at 0.02. Urate concentration in the LB medium was detected                     by HPLC. The values we showed are the raw peak area data. The figure shows that pucL, YgfU + pucL were                     not able to reduce the urate concentration in the LB medium.
We doubted that it was because the bacteria cell couldn’t intake urate that our system couldn’t work. To verify it, we measured                     the urate concentration in the cell cytoplasm when two different genes [eGFP(J23100 + RiboJ + B0034 +                     eGFP, not submitted), pucL(BBa_K2334001)] were expressed in pSB1C3 vector.
To our surprise, the result shows that urate did enter the cytoplasm, but at one point, it’s excluded from the cell and won’t                     be taken in again anymore (Figure 7). If uricase exists in the cytoplasm, it can still work to reduce                     the urate concentration in the cytoplasm, but even so, urate still can’t be taken in continuously. There’s                     a possible unknown mechanism to prevent the urate from entering the cell when the mechanism is activated.                     In previous research, YgfU was expressed driven by T7 promoter, which was a furious expression process                     that the mechanism couldn’t influence in a short time. When YgfU is constitutively over-expressed, the                     mechanism can actually act successfully.
Figure 7 pucL and eGFP were cultivated in urate LB medium overnight (Urate concentration is about 1mM). The initial OD600                     of the LB medium was controlled at 0.02. Urate concentration in the LB medium was detected by HPLC. Cytoplasm                     urate concentration was detected by HPLC as well. Diluted cell lysate was used for detection.
We tried to figure out the parameters that influence the unknown mechanism. In our experiment, we tried two parameters: 1)                     Oxygen. 2) Nitrogen Source.
The reason why we suspected these two parameters is that it’s reported that E. coli can utilize allantoin as nitrogen resource                     under anaerobic conditions. Based on that, we designed our experiment to verify if those two parameters                     are important for urate utilization. We cultivated eGFP, pucL, YgfU+pucL, LM4 Full, LM4 Optimum (transformed                     in E. coli Nissle 1917) in M9/LB medium and aerobic & anaerobic environments (see protocol). Considering                     that the number of bacteria can influence the ability of urate consuming, we measured the OD600 of the                     bacteria when the sample was ready for HPLC test.
The values in different independent repeated experiments are not always the same. But the tendency is the same in every group:                     For urate utilization ability, M9(Anaerobic)＞M9(Aerobic) ＞LB(Anaerobic)＞LB(Aerobic). It means that bad                     nutrition environment and anaerobic environment can better reduce the urate level. The urate concentration                     in gut is about 100uM-400uM. According to our experiment results, our gene constructions (pucL, YgfU                     + pucL, LM4 Full, LM4 Optimum) all have the ability to reduce the urate concentration to the ideal value.                     However, we cannot determine which one is the best choice. LM4 Full and LM4 Optimum showed no significant                     difference in this urate consuming experiment. Moreover, the LM4 optimum grew better than LM4 Full, which                     means that our modeling works well (Table 6).
Furthermore, some LB medium samples, even cultivated in an aerobic condition, showed an decrease in urate concentration.                     However, we never found urate concentration reducing in LB medium before, which may be related to the                     expression host we chose (BL21 and Nissle 1917).
Meanwhile, We measured the OD600 after cultivation for another 24h, and we found that if the absolute value of the difference                     between OD600 at 24h & 48h is below 0.05 (which means that bacteria growth is in the platform phase at                     24h), the possibility that the urate concentration decrease is more than 100uM(about 1000 if shown in                     HPLC peak area) is 85% on average. We doubted that the decrease of urate was due to the outflow of the                     uricase after cell death and lysis. The hypothesis can also interpret why anaerobic & worse nutrition                     environment, which is with lower environment capacity, is better for urate reduction.
Table 6 Urate consuming parameters test. eGFP refers to gene "J23100 + RiboJ + B0034 + eGFP", pucL refers to part BBa_K2334001, pucL + YgfU refers to part BBa_K2334006, LM4 Full refers to part BBa_K2334017, and LM4 Optimum refers                     to part BBa_K2334013. We use HPLC peak area to present rate consuming directly.
Figure 8 The experiment results of Group 1-3 original HPLC peak area result shown in bar. Green bar is eGFP as control. It's                     clear that M9 and anaerobic cultivation performed better for urate consuming.
Here's the original experiment result.
To verify the theory whether the consuming of the urate is due to the cell lysis after cell death, we centrifuged the M9                     anaerobic cultivated medium and added its 100ul supernatant and 10min-heated 100ul supernatant respectively                     into 900ul PBS with 3M Urate(PH=8.0) to react in 37 ℃ for 2h. To our surprise, result shows that there’s                     no urate consuming detected for each experiment (Figure 9). We suppose that the enzymes might not be leaked to the outside, and the bacteria can utilize urate when the environment is fully loaded, which                     leaves a pressure on them. More experiment and repeats should be conducted to find the mechanism in the                     future.
Figure 9 The result shows that the supernatant cannot utilize urate, which is not consistent with the hypothesis we raised                     before. It may indicate that the bacteria can utilize the urate in a special stage without leakage.
No matter what, the urate concentration outside is lowered anyway, even in LB medium it can meet our requirement to reduce the urate about 100uM. But the real condition in the human gut is so complicated: a. Nutrition environment is hard to simulate because of the circadian rhythm of food intake, the food eaten and the enzymes released by intestine. b. The effect of gut microbiome. c. the peristalsis process in the intestine. d. the oxygen environment. To evaluate whether our project can work in the real condition, we decided to do the animal experiment.
Our animal experiment is conducted with the permission from the Institutional Review Board of West China Medical Center,                     Sichuan University.
Firstly, we measured the ability of bacteria colonization. E .coli Nissle 1917 transformed with eGFP-pSB1C3 was given by                     oral gavage[7]. After gavage, we collected the feces every two days and cultivated it in 3x chloramphenicol                     LB medium overnight. (See protocols for details). The results showed that, we can still detect Nissle                     1917 left in feces at Day 5 after gavage. However, we can’t detect Nissle 1917 at Day 7. We can conclude                     that Nissle 1917 was decolonized or lost its plasmid between Day 5 and Day 7 (Figure 10).
Figure 9 The result shows that the supernatant cannot utilize urate, which is not consistent with the hypothesis we raised                     before. It may indicate that the bacteria can utilize the urate in a special stage without leakage.
We tried to establish the HUA mouse model in the first stage in our animal experiment. We divided 10 male 5-week-aged balb/c mice into two groups randomly: Group 1 (n=5) and Group 2 (n=5). Set one week’s time without special handling to let them adapt to the new environment. Feed Group 2 with special food containing 0.1% adenine since week 2 while feeding Group 1 with normal food. All the feeding condition is the same expect for food.The blood of all the mice from caudal vein was taken after 3 weeks’ feeding. Isolate the serum and demonstrate the concentration of blood uric acid with ELISA.  The result showed that, we can use the method to establish the HUA model successfully (Figure 11a.).
Then, the engineered bacteria was used in the animal experiment. We divided 25 male 5-week-aged balb/c mice into five groups randomly: Group Control (Group A, n=5), Group HUA (Group B, n=5), Group Allopurinol (Group C, n=5), Group engineered E. coli Nissle 1917 (transformed with BBa_K2334001) (Group D,n=5) and Group E. coli Nissle 1917 (Group E, n=5) [Figure 11b]. Set one week’s time without special handling to let them adapt to the new environment. Establish hyperuricemia animal models with all the groups except Group Control.Performe oval gavage as protocol from week 2, last for 3 weeks Isolate the serum and demonstrate the concentration of blood uric acid with ELISA.
Allopurinol is used in clinics to reduce the serum urate concentration, whose group is the negative control. Group B and C indicated that we established the HUA model successfully. Group D is the experiment group, which is of distinctive lower urate concentration compared to Group A. However, the group E has similar performance as Group D, which suggested E. coli Nissle 1917 itself has a ability to reduce the serum urate concentration, which is so surprising. We can't determine if our engineered pathway is working in vivo. But it's clear our engineered bacteria itself works. It may be related to the remodeling of the gut microbiome if a lot of bacteria were given by oral gavage (Figure 11c). Though no group diarrhea was found.
Figure 11 (a) The urate concentration in serum in experiment group is higher than that in control group obviously, which indicated the successful establishment of HUA mouse model. (b) The arrangement of engineered bacteria in vivo experiment. (c) Group Control (Group A, n=5), Group HUA (Group B, n=5), Group Allopurinol (Group C, n=5), Group engineered E. coli Nissle 1917 (Nissle 1917 transformed with BBa_K2334001, Group D,n=5) and Group E. coli Nissle 1917 (Group E, n=5). The urate concentration in serum in engineerd bacteria experiment. Group B and C indicated we established the HUA model successfully. The experiment group has obviously lower urate concentration than that of the control group A. However, there's no significant difference between group D and E.
Future Work
To elongate the existence of engineered bacteria in the gut to make a better colonization for human beings and sustain the                     maximum and stable gene expression as much as possible, we decided to use the plasmid to express the                     gene instead of genome integrated expression to promote the protein production. And we planned to knock                     out the genes alr, dadX in E. coli Nissle 1917, based on the paper published by In Young Hwang et al.                     from National Univeristy of Singapore. Alr and dadX are responsible for turning D-type amino acids into                     L-type amino acids to form cell walls. The knockout of those two genes leads to the death of the bacteria.                     Then we planned to add alr gene into the plasmid backbone, to form a complementary plasmid and to prevent                     plasmid loss. In the condition, antibiotic resistance marker can be deleted for biosafety, and the selection                     can be completed by auxotrophic selection[7].
Figure 12 the knockout of genes responsible for D-type amino acids synthesis of E. coli cell wall and formation of a complementary                     plasmid, according to National University of Singapore[7].
However, after 2 months of knockout experiment, we failed to knockout the genes. Fortunately, we obtained the strain from                     National Univeristy of Singapore after looking for their help and confirmed the character of this strain                     of E. coli Nissle 1917 (Δalr, ΔdadX) in our laboratory (Figure 12).
Figure 13 The growth ability test with (a) and without (b) D-alanine. Only when the medium is added with D-alanine, the E. coli Nissle 1917 (Δalr, ΔdadX) can grow.
We’ve successfully ligated alr with the pSB1C3 backbone.
Figure 14 alr gene with original promoter and RBS from genome is ligated into pSB1C3 backbone by Gibson Assembly. The map                     the plasmid is in the figure.
After several times of attempt, we found it hard to transform the engineered plasmid into this kind of strain in the same                     way we applied in E. coli Nissle 1917.
We are going to continue our experiment in the future.
Improvement of Gifu 2016 Project
At the same time, we improved the project of 2016 Gifu iGEM team. We used the same pathway in our project. The difference                     is that they used this pathway to clean the bird dropping. Here’s their abstract.
“In Japan, environmental pollution caused by excrement of birds is a problem that should be solved. For example, dieback                     of trees and spoiling the cityscape are major problems. Birds’ dropping consists mainly of uric acid                     and that is really insoluble. Uric acid can be degraded to soluble material, urea. So, our goal is to                     catalyze uric acid to urea and make it possible to wash away dropping by rainwater. We’d like to lead                     our project to the solution of the pollution. Considering purine metabolism pathway, three enzymes, urate                     oxidase, allantoinase, allantoicase, must be synthesized to degrade uric acid to urea.”
In their project, they intended to reduce the urate concentration outside the cell as well. However, they didn't succeed                     in the end due to the lack of the exploration of parameters may influence the result and they only used                     uricase in their project. In our project, we constructed the complete pathway and verified the parameters                     to reduce the urate concentration outside the cell successfully. For their project, according to our                     experiment, we suggested them to use M9 medium to culture the bacteria then spread it to the surface                     of the bird dropping.
Dialysis-like device
The key to the treatment of refractory gout is to lower the blood uric acid concentration, but direct protein contact may                     be blocked immediately by the IgG antibodies (of a long half-life) of the uricase-resistant patient,                     so it is necessary to build a relatively independent immunologically privileged sites. Through the establishment                     of a dialysis system, we try to solve the problem of refractory gout. At the same time, if the efficiency                     of the device is acceptable, it can be used as an inexpensive treatment regimen to accelerate the dissolution                     of gout and may be able to change the principles of treatment for chronic tophi. Or even more,
We plan to overexpress the protein regulating the uric acid metabolic pathway in Bacillus subtilis to activate the pathway,                     and add an in vitro dialysis device, trying to treat patients with refractory gout who are resistant                     to intravenous treatment of uricase drugs.
We designed a dialysis device which injects the needles into the human vein in both directions and allow blood to flow through                     it. We used commercialized dialyzer in our design. In this dialyzer, the blood flows through the dialyzer                     which only allows small molecules to pass through and the bacteria remains in the another side of the                     fiber tube, unable to enter the bloodstream. The blood flow inside the tube is mainly driven by the peristaltic                     pump (Figure 14).
Figure 15 The figure of the dialysis-like system and the real device system we made.
At the point of blood exchange, blood flows into the dialysis tube; outside the dialysis tube is artificial serum medium.                     Small molecules in the blood (such as urate) can be exchanged outside through the dialysis tube.
For more details, please visit Hardware page .
The strain of the bacteria we use for this device is extremely important due to biosafety reasons. Bacillus subtilis is a                     common gram-positive bacterium, which is widely used in engineering field. Human infection of Bacillus                     subtilis is rarely reported, so it is not considered as a pathogenic bacterium. Also, in the cause-of-death                     statistics of the World Health Organization, no data on B. subtilis infections are present since, even                     if reported, they would be “invisible” at the international comparative level due to the coding used                     for classification of death causes[9]. In the literature, only a few cases of infections due to B. subtilis are reported and only one retrospective study describes the isolation of antibiotic-resistant strains                     of B. subtilis. Because Bacillus subtilis itself has a uric acid metabolic pathway, we plan to use an                     engineered Bacillus subtilis strain which hasn’t been used in the blood related clinic therapy before.
Bacillus subtilis itself contains a urate metabolic pathway. The pucJ/K helps uric acid to be transported into the cell,                     and pucL/M converts uric acid into allantoin. pucJKLM can be regulated by PucR protein to initiate the                     transcription. Therefore, we decide to overexpress pucR gene in Bacillus subtilis to activate the downstream                     uric acid metabolic pathway[10,11].
Figure 16 The metabolic pathway for catabolism of purine nucleobases in B. subtilis. The large oval represents the inner                     cell membrane of B. subtilis containing transport proteins for purine nucleobases and their catabolites                     (black circles) encoded by the given genes[10].
Figure 17 PucR can activate the expression of pucJKLM operon to promote the expression of uric acid metabolism pathway genes.[11]
We chose Bacillus subtilis R179 as our host because it’s used in clinics now to treat gut environment disorder cause by gut                     microbiome in China, which has passed through safety evaluation at least in intestine.
We ligated the pucR to Bacillus Subtilis PHT43 plasmid successfully. PucR can be expressed with IPTG induced. However, after                     many attempts, we didn’t transform it into Bacillus subtilis R179 successfully before the wiki freeze.                     As a result, we didn’t test the pucR function successfully before the wiki freeze.
Figure 18 The plasmid map of pucR-PHT43.
Some Bacillus subtilis strains can induce hemolysis. We tested the hemolysis ability of the Bacillus subtilis strain itself, Bacillus subtilis WB800 and Bacillus subtilis R179. The result showed that R179 is with little hemolysis                     ability. In our experiment, we decided to use R179 as our chassis. Though there was a Bacillus subtilis strain with no hemolysis ability reported, which can be our future chassis[9].
Figure 19 The hemolysis ability test of Bacillus Subtilis with blood agar plate. The zone of hemolysis of R179 is obviously                     smaller than that of WB800, which is widely used in bioengineering field.
The details about the hardware can be seen in our Hardware page . In description, we’d like to only show the experiment result.
Though the microporous aperture of the fiber tube in the dialyzer is about 7-9nm, we still did the experiment to ensure the                     bacteria cannot pass through the membrane. After adding the bacteria culture and sterilized water to                     two sides of the fiber tube for more than 12h, we cultivated the liquid sampled from both sides of the                     fiber tube. The result showed that no bacteria were detected in the sterilized water side, which indicated                     that the membrane can prevent the leakage of the bacteria (Figure 17).
Figure 20 The bacteria leakage experiment. The sample from bacteria culture side grew bacteria. The sample from sterilized                     water side is with no bacteria grown, which indicated that the membrane can prevent the leakage of the                     bacteria.
After ensuring that the bacteria cannot pass through the dialyzer, we considered that bacteria might secret protein when                     grown or after death, which might pass through the fiber tube and get into the blood, leading to the                     immune response. So we did the leakage experiment to find out how much protein can be leaked. M9 with Bacillus subtilis (Bacillus subtilis was grown in LB medium overnight, bacteria was centrifuged and added                     into M9 medium) and M9 were added into the bacteria culture flask and simulated blood flask separately.                     Samples were taken for BCA protein assay at the time. The device ran for 4 hours to simulate the application                     in real condition. After that, samples from bacteria culture flask and stimulated blood flask were taken                     for BCA protein assay. To our surprise, the result showed that after the running of the device, protein                     concentration was the same or even decreased (Table 7). We suspected that it’s due to that the dialyzer                     can absorb some protein in it (the surface of the fiber tube, eg.), which means that, at least in our                     experiment, the biosafety of our device is highly promising.
Table 7 The experiment of protein leakage experiment.
To simulate the possible veins that might be used in the clinics, we took out one experiment. For simulated femoral vein, we set the flow rate in the simulated blood extracorporeal circulation part and the bacterial circulation part to be 60ml/min; for simulated cephalic vein, we set the flow rate in the simulated blood extracorporeal circulation part and the bacterial circulation part to be 14ml/min. The experiment is carried out as protocol and the data we get is shown below in table 8.
Table 8 The original HPLC data of uric acid permeating experiment. Because in the two experiment instruments we used to get the HPLC data is different, there is a big different the two sets of data.
We transferred the data into the concentration of uric acid and plotted them. Data-fitting is done to verify that the data conforms to the model, and the result is shown below:
Figure 21 fitting curve and plots of cephalic vein simulation
Table 9 parameters and results of cephalic vein simulation
Figure 22 fitting curve and plots of femoral vein simulation
Table 10 parameters and results of femoral vein simulation
According to figure 6 and figure 7 and parameters (in table 9 and table 10) got from data-fitting, we can find that for femoral vein simulation, the fitting curve matches the plotted data very well. But the match between experiment data and fitting curve of cephalic vein simulation is not so good. While the values of k in the two results are not the same. The reason for those conditions could be that when the flow rate is low, the assumption of uniform uric acid concentration is not valid. Which can be justified in the future work. The results show that efficiency of uric permeating in femoral vein simulation is much better than that in cephalic vein simulation, because for the former one approximately 30 minutes are needed to reach concentration equilibrium but for the later one the time consumed is about 1 hour. It suggests that femoral vein could be more feasible for our device to work with. But there is also an application for cephalic vein. The traditional dialysis demands abundant dialysate fluid to keep the concentration gradient but our bacteria can metabolize uric acid permeated into the bacterial circulation which means we do not need vast dialysate fluid. This working characteristic together with the microminiaturization of sensors make it possible for our hardware to become a wearable device.
Reference
[1] Papakostas K, Frillingos S. Substrate selectivity of YgfU, a uric acid transporter from Escherichia coli.[J]. Journal of Biological Chemistry, 2012, 287(19):15684-15695.
[2] Nishiya Y, Hibi T, Oda J I. A purification method of the diagnostic enzyme Bacillus, uricase using magnetic beads and non-specific protease[J]. Protein Expression & Purification, 2002, 25(3):426.
[3] Jung, Du Kyo, et al. "Structural and Functional Analysis of PucM, a Hydrolase in the Ureide Pathway and a Member of the Transthyretin-Related Protein Family." Proceedings of the National Academy of Sciences of the United States of America 103.26(2006):9790-5.
[4] French J B, Ealick S E. Structural and mechanistic studies on Klebsiella pneumoniae 2-Oxo-4-hydroxy-4-carboxy-5-ureidoimidazoline decarboxylase.[J]. Journal of Biological Chemistry, 2010, 285(46):35446-35454.
[5] C. Lou, B. Stanton, Y.-J. Chen, B. Munsky, C. A. Voigt, Ribozyme-based insu lator parts buffer synthetic circuits from genetic context. Nat. Biotechnol. 30, 1137 (2012). doi:10.1038/nbt.2401 pmid:23034349
[6] 2016 William & Mary iGEM Team, https://2016.igem.org/Team:William_and_Mary/RiboJ, 2017/10/01.
[7] Hwang I Y, Koh E, Wong A, et al. Engineered probiotic Escherichia coli can eliminate and prevent Pseudomonas aeruginosa gut infection in animal models.[J]. Nature Communications, 2017, 8:15028.
[8] Lefevre, M, et al. "Safety assessment of Bacillus subtilis CU1 for use as a probiotic in humans. " Regulatory Toxicology & Pharmacology 83(2017):54.
[9] Oggioni M R, Pozzi G, Valensin P E, et al. Recurrent septicemia in an immunocompromised patient due to probiotic strains of Bacillus subtilis.[J]. Journal of Clinical Microbiology, 1998, 36(1):325-6.
[10] Ma P, Patching S G, Ivanova E, et al. The allantoin transport protein, PucI, from Bacillus subtilis: evolutionary relationships, amplified expression, activity and specificity[J]. Microbiology, 2016.
[11] Beier L, Nygaard P, Jarmer H, et al. Transcription analysis of the Bacillus subtilis PucR regulon and identification of a cis-acting sequence required for PucR-regulated expression of genes involved in purine catabolism.[J]. Journal of Bacteriology, 2002, 184(12):3232-3241.
