Team:Bilkent-UNAMBG/Notebook
Home
Project
Parts
Protocols
Human Practices
Team
Collaborations
Attributions
Demostrate
Results
Notebook
Safety
June
1st week:  Constructs for IDT order and test constructs with the IDT sequences had planned. Biobricks that we need had ordered. Primers for test constructs had been designed.   2nd and 3rd weeks:  Biobricks that we had in part plates had transformed and minipreped.  4th week:  Biobricks ordered had arrived. Except for Q04400, every plate that had been streaked were able to select single colony. For two of the test constructs that we could clone with the DNA we had we had ordered primers for cloning. I0500 transformed into E. coli dh5a
July
1st week: Primers for cloning of two test constructs arrived and were started cloning.  PCR was performed on 5A with S3_T2_2F (57) and K86_T1_R (62) primers in order to create 5B. Annealing temperatures: 57, 60, 62 celcius  K1834847 O/N culture samples were miniprepped.  Gibson assembly was performed with pSB1C3 and 5B and the product (T5) was transformed into E. coli dh5a.  12 colonies from T5 plate were selected and grown O/N.  O/N cultures were miniprepped, digested with HindIII. Unexpected (>10kb) bands were observed. Samples were digested again with NotI and HindIII.   pSB1C3 linearized plasmid was digested with EcoRI and PstI. Gibson assembly at 1.7.16 was repeated using digested pSB1C3. T5 was transformed into E. coli dh5a.  PCR was performed on K814102 with K86-2_T2_F and K86_T1_R primers in order to produce 2C.  Colony PCR was performed on T5 colonies using S3_T2_SF and K86_T1_R primers. Colonies with positive results were grown O/N. 2nd week: Test constructs were started clonning 1, 3, 5 and 8. Cloning of test construct 5 was done. In addition, PCR reactions of test constructs 1, 2, 3, 4, 6, and 8 were repeated since they did not work.  O/N cultures were miniprepped and digested with HindIII. Same as before.  EcoRI-PstI digested and not digested pSB1C3 samples were run on gel. Nothing unexpected.  PCR was performed on B0015, S1,R0010, S4A, S2, S7 and S6 in order to produce 1B, 1C, 3C, 8C, 8D, 1A, 3A and 4A. Fail.  PCR was repeated, failed.
3rd week: PCR reactions of test constructs 2, 3, 4, 6, and 8 were performed for cloning purposes. PCR reactions did not work in spite of optimization for elongation period and TM, even though gradient PCR reactions we couldn’t get any required yield when we ran them on the gel. Induction reactions of decanal sensor was initiated, which is the test construct 5.   PCR repeat of 1A, 1B,1D,2A3A and 4A with gradient annealing temperature.
PCR repeat of 1A, 3A and 4A with two step PCR
Overlap extension PCR of 1A, 1B, 1C and 1D in order to produce T1. 3kb band was extracted
T5 containing bacteria were induced with decanal. O/N culture was diluted into OD600 as 1. Half centrifuged and resuspended at PBS. Decanal was diluted 1:10, 1:100 and 1:1000 in water. Decanal was added in 99uL cells so that 1%, 0.1%, 0.01% and 0.001% decanal concentration can be obtained. 490nm emission was measured after 2, 10,20 and 30 mins of decanal addition. No difference was observed. Possible problem was using clear 96 well plate instead of black.   OEPCR was performed on 2B and 2C for T2, on 3B, 3C and 2C for T3 and 6B and 2C for T6.
Decanal induction was repeated using black bottom plate. No change.  T1, T2, T3 and T6 were extracted from gel and amplified with PCR.   T1, T2, T3 and T6 EcoRI and PstI digested, extracted from gel and cloned into pSB1C3 using ligation. T1 was not present in gel.  Colony PCR was performed on 6 colonies with YFP signal each from T2, T3 and T6. Gel lanes: T2;1,2,3,4,5,6 T3; 1,2,3,4,5,6 and T6; 1,2,3,4,5,6. T2.6, T3.4, T3.6 and T6.6 had expected bands so they cultured O/N.  OEPCR of T1 was repeated
Band extracted from OEPCR of T1 amplified with PCR.  Decanal induction of T5 repeated. All samples centrifuged and resuspended in PBS. Decanal diluted in 50% EtOH before addition on cells. No emission at 490 nm. Miniprep of T2.6, T3.4, T3.6 and T6.6 failed so another O/N culture prepared. 4th week: SDS - PAGE experiment for test construct 5 (decanal sensor) was performed. No luciferase protein was detected. PCR reactions for test construct 4 were repeated with different set of primers since the previous reactions were all smear on the gel. Re-ordered Q04400 arrived  in stabbed agar, was inoculated in LB and LB with Kanamycin. T2.6, T3.4, T3.6  and T6.6 samples were sent for sequencing. T5 failed because people could see LuxAB activity with naked eyes on colonies in plated when decanal fume is present in the environment. There was no light emission in any of the colonies.
August
1st week: Protocol preparation and optimization for 2-propanol and octane inductions initiated. We sent selected colony plasmids of test constructs  2, 3 and 5 for sequencing. 6th colony of test construct 2 was verified. The selected two colony plasmids of test construct 3 had stop  codons in them. Plasmids of test construct 5 did not have any insert in them, so we repeated PCR reactions with gradient PCR. Test construct 6 for formaldehyde sensor worked.  Colony PCR on T7 colonies was performed. 2nd week: PCR reactions for test constructs 1, 4 and 5 were repeated, but it was observed that the primers were  being amplified around 200 bps sequences. 2-propanol and octane inductions had results. These results indicated that there was a significant increase between empty c ells and cells containing plasmids, but no significant increase was observed between uninduced and induced samples. 3rd week: PCR reactions of test constructs 1, 3 and 4 was repeated and we couldn’t get any results. The verified test construct 2 colony was  induced by 2-propanol and test construct 7 by Atc. The same insignificant results were obtained. Test construct 1 PCR products were  used for Gibson Assembly to produce test construct 1. On plate, we saw colonies, so we digested test construct 1 colonies for verification before sequencing. They did not give the bands we expected.  PCR of 3A, 4A and 5A was repeated using excessive amount of primers. Failed again.  T8.1 was transformed into dh5a.  T8.1 was induced with Arabinose for 2, 4 and 6 hours. 107 cells were collected.  SDS page performed on cells in order to observe dCas9 expression around 150 kDa. No dCas9 expression observed in any of the samples. 4th week: PCR reactions for test construct 5 was repeated and used for Gibson Assembly. Since test construct 1 colonies was problematic, we repeated Gibson Assembly. Since we had so little time and no good results, we decided to narrow the project down to only breast cancer sensors. sfGFP with TetO promoter was obtained and used instead of T7. TetO-sfGFP was transformed into dh5a pro. TetO-sfGFP induced with ATC and expected results observed.
September
1st week: Maps of the inserts to be ordered from Genewiz was prepared. The cloning strategies was planned. 2nd , 3rd and 4th week: We were sperated into couples. Each couple planned a flow-chart for the experiments until the Genewiz orders arrive. Vectors was prepared for cloning. They were digested with the relative enzymes. Ordered sgRNA targeting LacI and pGEX-6P-1-sfGFP were amplified with PCR for gibson assembly.
October
1st week: Primers for sfGFP amplification was designed and ordered. sfGFP was amplified with these primers. Genewiz orders was and transformed.   dCas9 and sgRNA  were co-transformed sequentially by generating dCas9 plasmid containing competent cells and transforming them with  sgRNA. ATC induction was repeated. Each day a set of sample was centrifuged and their medium was renewed while other set samples were diluted  1:100 into new media. Each day fluorescence  was measured. After 5 days of induction no fluorescence difference was observed. pet22b containing cells and pza containing cells were grown O/N. Miniprep was done on pza and pet22b. pza and pet22b were digested with EcoRI/MluI; NotI respectively. They were run on the gel. Cut pza and pet22b were extracted from the gel. puc57 with XylR sequence containing(construct#1) cells were grown O/N.  2nd week: Transformed Genewiz orders was inoculated in LB and miniprepd. Plasmids was digested and the inserts was cloned into pZA and pET22b  vectors by cut and ligate. The amplified sfGFP was used in Gibson Assembly with pZA and pET22b vectors in order them to had the reporter gene. Although Gibson Assembly reaction plates had colonies, the respective plasmids weren’t successful.  Six bands in the middle of the gel represent the cut puc57 Cut puc57 was extracted from gel. Gibson was done for clonning sfGFP into puc57 construct#1. Transformatin was done and cells were spreaded onto plate. puc57 construct#1 with sfGFP was restricted with NotI and XhoI in order to verify if sfGFP was clonned into puc57 construct#1. Restricted DNAs were run on the gel. It was seen that 3 of 4 colonies from tha plate of puc57 construct#1 with sfGFP have sfGFP!! puc57 with chnR sequence (construct#2) was restricted with BglII.  Gibson Assembly was done for clonning sfGFP into puc57 construct#2. Transformation was done and cells were spreaded onto plate.   T4 ligation was with puc57 construct#1 with sfGFP(previously cut with XhoI and NotI) and pet22b(previously cut with XhoI and NotI). Transformation was done and cells were spreaded onto plate. 6 colonies were picked from the plate and grown O/N.  Plasmid isolation was done on 4 falcons that contain previously grown puc57 construct#2 with sfGFP cells.  puc57 construct#2 with sfGFP  was restricted with BglII and NotI to verify if sfGFP was inserted successfully. We could not see any sign to sfGFP insert.   For the last four bands, if puc57 construct#2 could have been inserted with sfGFP, we should have seen three bands but there were two.   Pu cloned in front of dCas9 while pChnB cloned in front of sgRNA by cut ligate.  Constructs verified by digestion.  3rd week: Plasmid isolation was done on remaining 2 falcons that contain previously grown puc57 construct#2 with sfGFP cells. Isolated plasmids  were restricted with BglII and NotI. For 1 of the colonies we could see a band that belongs to sfGFP.  We saw puc57 with construct#1 ligation with pet22b was unsuccessful.  First three belong to unsuccessful gibson of puc57 construct#2 and sfGFP. Next three belong to successful gibson of them. Last four bands belong to unsuccessful ligation of puc57 construct#1 with pet22b. In order to amplify XylR and ChnR genes, PCR was done. Run on the gel. We saw that we could amplify XylR but not ChnR.  First four bands belong to amplified XylR. Last four bands belong to ChnR, failed amplification. We tried to amplify CnhR by using different vector. This time we were successful, there was not any problem with primers. pet22b containing XylR and sfGFP genes was digested in order to verify if the clonning was successful. However, we could not see any  band, so we performed T4 ligation again. Transformation was done and spreaded onto plate.  First four bands belong to unsuccessul ligation of pet22b with puc57 construct#1. 5th band is control of ChnR pcr. Last four bands are amplified ChnR. We did another T4 ligation for puc57 construct#2 with sfGFP(previously cut with BglII and NotI) and pet22b(previously cut with NotI). Transformation was done and spreaded onto plate. We saw colonies on our control plate, therefore we performed all the steps done yesterday. Additionally, Gibson was done with newly amplified XylR, ChnR and previously cut pzs. Transformation was done and spreaded onto plate.  Constructs will be verified.
