"
Page
Discussion
View source
History
teams
Log in
 
Team:Berkeley Wetlab/Cell Surface Display Parts
From 2009.igem.org
(Redirected from Team:Berkeley Wetlab/Assay Protocols )
Cell Surface Display Parts
Passengers
Follow any of the links below to see assay information for each of the passengers we made.
MGFP-5
The Sticky Protein
Displaying this sticky peptide would allow you to make an underwater bio-glue!
Cellulase
The Enzyme
Displaying enzymes that degrade cellulose would allow consolidated bioprocessing for biofuel production!
Type III Needle scFv
The Antibody Fragment
Displaying antibody fragments would allow easy pathogen detection!
Streptavidin Binding Tag
The Tag
A tag on the surface of E.coli would allow you to purify bacteria preferentially!
Leucine Zippers
The Heterodimeric Binding Peptide
Displaying heterodimeric binding peptides would allow you to make heterogeneous bacteria systems!
Ag4
The Silver Binding Peptide
The metal head of our set! Displaying a heavy metal binding peptide would promote bioremediation of heavy metals and aid in synthesis of biomaterials.
Spacers
Spacer elements are present in most natural display systems, which suggests that they are important components of cell surface display. However, the precise role of spacers in the functionality of outer membrane proteins has not been extensively characterized and most people engineer cell surface display devices without structural spacers.  We decided to build a few cell surface display systems with spacer elements in order to begin characterizing their role within engineered cell surface display devices. We introduced five different spacer domains to our surface display system: INP-repeats, beta roll, beta helix, Gly-Ser repeats and GFP-LVA. These elements are further described below.
INP-Repeats
Synthetic spacer region composed of repeated portions of ice nucleation protein (INP) sequence.
Beta Roll
Protein tertiary structure that is natively found in autotransporters and other outermembrane proteins.
Beta Helix
Protein helical structures stabilized by hydrogen bonds and protein-protein interactions. The resulting structure contains two to three faces formed by the association of parallel beta strands.
Gly-Ser Repeats
Synthetic spacer made from fourteen repeated glycine and serine residues. This spacer is essentially devoid of secondary structure, it is simply a long flexible linker.  We made in order to differentiate between the importance of a natural spacer with secondary/tertiary structure and a long linker between the displayer and its passenger.
GFP-LVA
Modified green fluorescent protein
Displayers
A displayer is defined as an outmembrane protein that carries another protein to the extracellular space of the cell.
For successful cell surface display of proteins, there must be an effective protein localization mechanism. Gram-negative bacteria such as E. Coli have two membranes, which present a problem for transporting proteins synthesized in the cytoplasm to the outside of the cell. Various transport schemes exist in gram-negative bacteria to effectively localize proteins to the outermembrane. The most common schemes are TypeI, TypeIII, and TypeV secretion.
In our display systems, we chose a class of outermembrane proteins called autotransporters that localizes proteins via the TypeV secretion mechanism. Over 700 autotransporters have been sequenced, many of which are used to export virulence factors to the outside of the cell. We decided to harvest this localization system for cell surface display because the outermembrane protein (aka displayer) spontaneously inserts into the outermembrane and pulls the protein it is covalently linked to (aka passenger) into the extracellular space. Moreover, autotransporters are capable of pulling through large proteins, such as enzymes and single-chain variable fragments. We have removed the native passengers in our autotransporter constructs and fused heterologous passengers of interest to the N-terminus of these autotransporters.
As depicted in the diagram above, autotransporter transport begins with localization to the periplasm via the Sec secretion pathway. The translocated protein remains unfolded in the periplasm until it inserts into the outermembrane by forming a beta barrel with its C-terminal 250-300 amino acyl residues. The N-terminus of the protein (containing our passenger of interest) is then pulled through the barrel to the outside of the cell. Passengers of displayers are often cleaved for extracellular secretion. In our systems, however, we removed the signal sequence that signals for peptide cleavage so our passengers remain attached to the transmembrane displayer protein.
In constructing our parts, we looked into a broad range of autotransporters, some well characterized and others putative, to explore the spectrum of display machinery and to establish the functionality of novel autotransporters for cell surface display.
Azo1653 AtD (putative)
Organism: Azoarcus sp. (strain BH72)
Autotransporter type: AT-1 family
OprF AtD
Organism: Pseudomonas fluorescens
Structure: an 8-stranded beta barrel in the outermembrane
Cl02365 AtD (putative)
Organism: Neisseria meningitidis
Autotransporter type: AT-1 family
VtaA11 AtD
Organism: Haemophilus parasuis
Autotransporter type: AT-2 family
Hag AtD
Organism: Moraxella catarrhalis
Autotransporter type: dimeric family
Structure: 200kDa protein with 10-stranded beta barrel
Pcryo_1225 AtD (putative)
Organism: Psychrobacter cryohalolentis
Hia AtD
Organism: Haemophilus influenzae
Autotransporter domain: trimeric family
Structure: modular segments containing repeats of structurally distinct domains
upaG_short
Organism: Escherichia Coli
Autotransporter type: trimeric family
espP(beta)
Organism: Escherichia coli
Structure: 12-stranded beta barrel
ehaB
Organism: Escherichia coli
Features: primary sequence alone is sufficient for crossing the bacterial membrane
TshA
Organism: Escherichia coli
Autotransporter type: serine protease subfamily (because of the 7AA serine protease motif)
VirG(IcsA)
Organism: Shigella flexneri
YuaQ AtD (putative)
Organism: Escherichia coli
Features: bears sequence similarity to the confirmed autotransporters AIDA and Ag43
AIDA-I
Organism: Escherichia Coli
Features: identified to be similar to IgA1, the first autotransporter used for surface display. Occurs naturally in the host organism, E. coli, and is a robust tool for surface display
Ag43_short
Organism: Escherichia Coli MG1655
Features: expression of Ag43 is evenly distributed around the bacterial cell
Structure: 14 antiparallel beta strands each composed of about 12AA residues
eCPX (circularly permuted OmpX)
Organism: Escherichia Coli
Features: protein is an enhanced CPX variant located in the outermembrane that joins the N- and C-termini of OmpX.
CPG_L2 (circularly permuted OmpG)
Organism: Escherichia Coli
Features: protein is circularly permuted with its backbone opening in loop 2, allowing both the N- and C- termini to be present in the extracellular space.
CPG_L6 (circularly permuted OmpG)
Organism: Escherichia Coli
Features: protein is circularly permuted with its backbone opening in loop 6, allowing both the N- and C- termini to be present in the extracellular space.
Some of these proteins are putative autotransporters that have sequence homology to confirmed autotransporters. We chose these proteins because we wanted to test their functionality and expand the range of displayers available for surface display.
References
Pina, S et al. Trimeric Autotransporters of Haemophilus parasuis: Generation of an Extensive Passenger Domain Repertoire Specific for Pathogenic Strains. J Bacteriol. January 2009; 191(2): 576–587. Available Online: http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2620822/ (Accessed: 20 October 2009).
Kostakioti, M et al. Functional analysis of the Tsh autotransporter from an avian pathogenic Escherichia coli strain. Infect Immun. October 2004;72(10):5548-54. Available Online: http://www.ncbi.nlm.nih.gov/pubmed/15385451?ordinalpos=5&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_DefaultReportPanel.Pubmed_RVDocSum (Accessed: 20 October 2009).
http://www.uniprot.org/uniprot/?query=H81868&sort=score
http://pfam.sanger.ac.uk/family?acc=PF05736
http://www.genome.jp/dbget-bin/www_bget?azo:azo1653
http://www.genome.jp/dbget-bin/www_bget?pcr:Pcryo_1225
http://www.uniprot.org/uniprot/Q9JMS3
Retrieved from " http://2009.igem.org/Team:Berkeley_Wetlab/Cell_Surface_Display_Parts "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
