"
Page
Discussion
View source
History
teams
Log in
 
Team:Wisconsin-Madison/protocol
From 2012.igem.org
Team
Team Members
Advisors
Sponsors
Projects
Engineering Limonene Production in E.coli
Translational Coupling Cassette (TCC)
Parts
Translational Coupling Cassette
Negative TCC Control
Positive TCC Control
Codon-optomized LIMS1
Modeling
Protocol
Safety
Human practice
The labbager's great scroll of protocol
Alkaline lysis plasmid extraction
Solutions
Alkaline Lysis Solution 1
50 mM glucose
25 mM Tris-Cl (pH 8.0)
10 mM EDTA (pH 8.0)
Prepare Solution I from standard stocks in batches of -100 ml, autoclave for 15 minutes at 15 psi (l.05 kg/cm2 ) on liquid cycle, and store at 4 degrees C.
Alkaline Lysis Solution 2
0.2 N NaOH (freshly diluted from a 10 N stock)
1% (w/v ) SDS
Prepare Solution II fresh and use at room temperature.
Alkaline Lysis Solution 3
5 M potassium acetate - 60.0 ml
Glacial acetic acid - 11.5 mL
H2O - 28.5 ml
The resulting solution is 3 M with respect to potassium and 5 M with respect to acetate. Store the solution at 4 degrees C and transfer it to an ice bucket just before use.
Protocol
1. Pellet the overnight cultures in a 1.5ml eppendorf tube for 1 min. Spin down 4 to 4.5 ml of culture. (For low copy spin down about 6 ml.
2. Resuspend each pellet in 200ul of Alkaline Lysis Sol I, RnaseA added (final RNaseA concentration should be 100ug/ml) Make sure there are no lumps, homogenized.
3. Add 400ul Alkaline Lysis Sol II. Invert 4-6 times to mix. Do not allow reaction to lyse for more than 5min. Sample should clarify.
4. Add 300ul Alkaline Lysis Sol III. Invert 4-6 times to mix. Sample should have a white precipitate.
5. Add 100ul of chloroform. Do this in a fume hood. Invert 4-6 times to mix.
6. Rest on ice for 5-10min. This step is so that the chloroform does not get too hot in the centrifuges and leak out of the tubes. If you want to skip this step you might consider using less cholorform. I put the tubes at -20 for a couple of minutes.
7. Centrifuge at max. speed (14,000rpm) for 10min.
8. Pipet 750ul of supernatant/aqueous layer into a fresh tube. I do up to 800ul.
9. Add 1/10 volume (75ul) 3M NaOAc, pH 5.2. Flick to mix. 80ul if have 800ul of supernatant.
10. Add 0.7-1.0 volume COLD isopronanol (in freezer). Vortex/Flick to mix. If in a hurry go straight to step 11, otherwise rest on ice for 10-30minutes. I have even let it precipitate overnight at 4C if convenient. 600ul isopropanol. Then I put it at -20C for 5minuetes up to over the weekend if needed.
11. Centrifudge at max. speed for 25min. Most miniprep protocols say to do this at 4C, but I have not noticed decreased yield by centrifuging at room temp.
12. Remove and discard the supernatant. Don’t disturb the pellet. Sometimes I can’t see a pellet and more often than not I still have DNA.
13. Add 1ml of 70% EtOH (at room temp.) Invert 4-6 times to rinse the tube.
14. Centrifuge at max speed for ~5 minutes. Room temp. is fine. Remove and discard the EtOH.
15. Repeat steps 14 and 15 to remove all traces of isopropanol. Pulse spin after removing bulk of final EtOH wash and pipet off remaining EtOH.
16. Air dry the pellet for ~15min (pellet will change from white to clear as it dries). Resuspend desired volume (~30ul) of H2O or EB or T10E1 depending on downstream applications. If you pipet off the EtOH well, then I have done this for as little as 2 min. For fosmids, I usually resuspend the pellet in 20ul water.
Colony PCR
Colony PCR is an efficient way to screen for the construct with the desired insert. Colony PCR uses primers that attaches on the outside of the cloning site and amplify only the cloning region.
Master Mix
This is for 10ul reactions. The volume below is for one 10ul reaction. If doing more than one, multiply the amount by how many colonies you will be screening, and probably add one extra reaction worth to account for pipetting error.
Reagent Volume
Sterile H20 4ul
GoTaq 5.0ul
FWD primer 0.5ul
REV primer 0.5ul
In each PCR tube, place 1 colony DNA and 10ul of Master Mix.
Thermal Cycler Protocol
Step Temperature Time
1. 95C 3 min
2. 95C 30sec
3. 55C 30sec
4. 72C 3min (depends on length 1min/kb)
5. Go to step 2 24x
6. 72C 5 min
7. 4C Forever
After the thermal cycler is done, run on a 1% gel at 100V for 30min. Use colonies that display the correct band to make overnight cultures with antibiotic to miniprep the next day for sequencing.
Digestion for cloning
Sterile Milli-Q Water (20 - x) uL x=amount of reagents used Total volume should be 20uL
DNA (2000ng/[DNA])
_ uL
BSA* [10X]
2 uL
B4 Buffer
2 uL
Enzyme #1
1 uL
Enzyme #2
1 uL
ddH2O
_ uL
Total
20uL
*BSA should only be added if needed. Check the NEB poster on the -80 to see if it is needed.
Incubate for 1 hour in the 37 degree water bath
Notes:
Keep everything on ice when thawing and mixing reagents. Do Not vortex the master mix; you can gently bump it with your fingers and then spin it down for less than 10 seconds.
If your DNA [ ] is lower and your volume will exceed 20uL, scale up the amounts of B4 Buffer and BSA to 3uL each and make a 30uL total volume reaction.
IMPORTANT: Enzymes must be kept in the blue cold box.
Dishwashing protocol
1. Bleach / wait / remove stickers
2. Pour down drain
3. Rinse 2-3 times with tap water
4. Add alconox and brush if necessary
5. Rinse 3x with DI, put upside down on a rack
Electroporation
Use the electro-competent cells you just made for transformation. ALL OF THIS IS ON ICE
1. 100uL electrocompetent cell aliquots and UV sterilized cuvettes should already be in an ice bucket. (UV means 3 mins in the gel doc on full)
2. Label 13mm plastic test tubes beforehand, have 900uL of LB ready to pipette into the cuvette immediately after shocking
3. Set electroporator machine to bacteria
4.  If plasmid DNA is ~150 ng/uL use 2.5uL. If 200 ng/uL, use 2uL (~400ng) into 100uL aliquots of electocomp cells
5. Use a kimwipe to wipe off any condensation from the exterior of the cuvette - important!!
6. Place the cuvette in the holder in the right orientation, slide in until you hear the click. The two electrodes should touch the sides of the cuvette
7. Take up 900uL of LB into the pipette tip
8. Press the SHOCK BUTTON
9. Immediately, pull out the cuvette and add the 900uL LB directly to the slit inside
10. Pipette up and down twice very gently. Transfer to the labelled 13mm tube (Don't make bubbles)
11. Repeat for all samples, INCLUDING YOUR NEGATIVE AND POSITIVE CONTROLS
12. The time constant reading should be between 3.8 and 5 ms (If it arcs, it is not usable)
13. Incubate in the gentle shaker (37 degrees and 100rpm) for 1 hour
14. Plate 100uL aliquots onto appropriate media, leave the remaining 800uL in the tubes to sit on the bench top overnight
15. For more difficult transformations, pellet 700uL of the culture, pour off supernatant and re-suspend pellet in 100uL of LB. Plate the concentrated 100uL of transformation.
16. To the remaining 200uL of transformation, add 1mL of LB and let sit on bench top overnight
17. Incubate all plates in 37oC incubator overnight, inspect for colonies in morning.
Ethanol Precipitation
1. 1/10 volume of Sodium Acetate (3 M, pH 5.2).
2. Add 2.5-3.0 X volume (calculated after addition of sodium acetate) of at least 95% (ice-cold) ethanol.
3. Incubate on ice for 15-30 minutes. In case of small DNA fragments (<100bp) or high dilutions overnight incubation gives best results, incubation below 0 °C does not significantly improve efficiency.
4. Centrifuge at > 14,000 x g (max speed) for 10 minutes at room temperature or 0 °C.
5. Remove supernatant being careful not to disturb the DNA pellet, which may be invisible. Remove any drops of liquid that adhere to the walls of the tube. (It is best to save the supernatant from valuable DNA samples until recovery of the precipitated DNA has been verified).
6. Fill tube 1/2 way with 70% Ethanol to rinse and recentrifuge at maximum speed for 2 minutes.
7. Repeat step 5.
8. If necessary, store the open tube on the bench at room temperature just until the last traces of fluid have evaporated.
9. Dissolve pellet (which is often invisible) in desired volume of buffer. (Usually TE [pH between 7.6 and 8.0], or water)
NOTE: Make sure to rinse the wall of the tube well with the buffer since up to 50% of the DNA will have been smeared on the wall. This can be done by pushing a bead of fluid over the surface of the wall using a disposable pipette tip.
Freezer stock preparation
Freezer Stocks
750ul of culture 250ul of 60% Glycerol Filter Sterilized using equipment in the Pfleger Lab Store them in -80C  Combine in labelled (strain, date, initials) cryotube (lids & tubes in separate blue boxes on center of benches)
Gel extraction
If you are planning on extracting more than 2 fragments, make sure you add a ladder to the left of each well in order to split up the gel when you excise the band. This will lower the time for UV exposure.
1. Make a 1% gel, except increase the weight to 0.3g of agarose and 30mL of 1X TAE, and the 8 welled comb.
2. Pre-Weigh a micro-centrifuge tube for later use
3. To load a 30uL digestion, add 6uL of 6X Loading Dye to the digestion tube
4. Also load 1uL of dyed uncut DNA into another row (as the control), and 2Log Ladder into a third row to compare band sizes and to confirm the DNA has been digested correctly.
5. Run the gel at 100V for 30min (unless the gel bands are extremely close in size
6. Put up the UV shield on the gel imager, and find some gloves
7. During extraction, use the Prep UV setting on the gel imager so the UV isn’t as strong
8. Also you need to cut out (excise) the band very quickly (~5 sec)
9. Then take a picture of the rest of the gel to record the excised band
Note: The picture is taken after excision to reduce the amount of exposure time to the DNA. Too much exposure (~1min) will potentially cause thiamine dimmers and other mutations in your DNA.  After this proceed using the direction found in the gel purification/extraction kit. Finally, elute in 25 uL of water.
Gibson Protocol
Written by Jackie in the Pfleger Lab
1. Generate forward and reverse primers to amplify insert and vector, refer to figure 1, can also use gibthon.org to design primers
Figure 1. Primer Design for 1 insert into a vector. Include at least 20 base pair homologous region over hang (pictured as the orange section on the gene of interest primers). For the insertion of more genes, just be sure each gene has an overlap region for the genes around it
2. PCR amplify insert and vector using the Tm of section that is actually annealing to the template and not the overall Tm that includes the overhang region
3. Gel extract the products to ensure correct amplification and to remove residual template plasmid. Can possibly Dpn1 the reaction then PCR purify to reduce background noise, but I have not tried yet
4. Add the following amount of Gibson Master Mix and vector and insert and mix together, then incubate at 50°C for 1 hour
Components
Suggested Amount
Gibson Master Mix
15uL
Plasmid
10-100ng
Insert (each)
2-3x of plasmid
5. Add 2 μL of reaction mix to competent cells (chemically competent DH5α), and transform according to standard protocol.
Matt's gibson protocol
1) Run PCR rxn
- take small aliquot (3 PCR µl + 7 µl dH20 + 2 µl 6x dye) and run on a gel
- take the rest and PCR clean-up all of it
2) Dpn1 digest the PCR cleaned-up DNA
- (1 hr, 37º C, make sure you add the NEB buffer 4)
3) Run out a gel (extraction wells) of both insert PCR, and vector PCR
4) finish gel extraction protocol
5) Nanodrop time
6) Setup Gibson rxn - run for 1 hr at 50º C
Plasmid
10-100 ng (shoot for 50-75)
Insert
2-3 times whatever you put in for plasmid
dH20
20-(plasmid + insert)
Total
20µl
7) Transformation time, take 3µl from gibson rxn and transform directly using that handy protocol
8) Stay classy
Ligation
Calculations
For the ligation, you should follow the following calculations to determine the ideal amount to achieve the typical ratio of 3:1, insert:vector. Also the Ligation buffer vial should be made into 5uL aliquots to prevent constant freeze thaw and keep it active. The freeze thawing will eventually cause the sulfur ends to degrade. As a quick check, smell the vial to see if it still smells like rotten eggs (sulfur smell). A sulfur smell means the buffer is still good to use.
(Length of insert/ Length of vector) X 100ng of vector X 3 = ng of insert for a 3:1 ratio
On top of the standard ligation, run a control ligation by replacing the insert with water
Reaction
For 20 uL reaction (in this order):
2 uL ligase buffer (tubes marked with a green dash, in a box marked with a green dash in the -20)
17 uL of the above-calculated mixture of insert & vector & water
1 uL T4 DNA ligase (in enzyme box)
Run benchtop ligation 15 minutes. Take 3uL from this, and tranform it into competent cells. Take the remaining 17uL and place in strip tubes, then thermocycler for an overnight ligation at 16 degrees.
Limonene Production Assay w/ GC-MS Chromatography
This protocol is work in progress.
Day 1 :
1. Freshly prepare transformed strains which are capable of : (Cultures to be produced in triplicates)
a. LIMS1 synthesis in LB medium. (pBba5c + LIMS1)
b. ADS synthesis in LB medium. (pBba5c + ADS )
c. Untransformed cells in LB medium. (DH10B)
Make 5 ml liquid cultures of each in triplicates and incubate overnight @ 37 C.
Day 2
2. Overnight cultures to be diluted 1:100 in LB medium to 80 ml.
Induce with 500mM IPTG
Overlay the solution with (20%) 20 ml dodecane
Also, prepare a 80ml flask (blank) containing LB and overlay with 20% dodecane (20ml).
Add Limonene in the medium of the following volumes (to the three flasks) :
1. 0.15 ml
2. 0.075 ml
3. 0.0 ml
Incubated at 37*C for 72 h.
Day 5
At the end of the production period, dodecane layers are to be sampled for limonene content by gas chromatography/mass spectrometry on a DB-5 column using the internal standard caryophyllene to normalize between samples.
Amendment for amorphadiene:
GC-MS analysis of amorphadiene. Amorphadiene production by the various strains was measured by GC-MS as previously described21 by scanning only for two ions, the molecular ion (204 m/z) and the 189 m/z ion. Cells were grown in LB medium at 37 °C for 2 h and induced to express the ADS and the mevalonate pathway by the simultaneous addition of 0.5 mM IPTG and varying concentrations of mevalonate. Amorphadiene concentrations were converted to caryophyllene equivalents using a caryophyllene standard curve and the relative abundance of ions 189 and 204 m/z to their total ions. The sesquiterpene caryophyllene was purchased from Sigma-Aldrich.
Making Chemically Competent DH5α Cells
Solutions:
Transformation Buffer "TB" (1L)
10 mM PIPES = 3 g
15 mM CaCl22H2O = 2.2 g
250 mM KCl = 18.6 g
Dissolve in 1L ddH20, adjust pH to 6.7-6.8 by adding 5N KOH
Add 10.9 g MnCl2 for final concentration of 55 mM
Filter with 0.22 filter
SOB (1L)
bacto tryptone = 20 g
bacto yeast extract = 5 g
5M NaCl = 2 mL
2M KCl = 1.25 mL
ddH2O = 990 mL
Autoclave to sterilize
Sterile Mg solution
1M MgSO47H2O
1M MgCl26H2O
Filter with 0.22 µm filter
Add 10 ml of sterile Mg solution to 1L of autoclaved SOB
Protocol:
Day 0
Streak out DH5 alpha E. coli on LB plate without selection
Day 1
Pick a single colony for a 5 ml of SOB for overnight culture at 37ºC
Day 2
Add 5 ml of starter culture to 200 ml of SOB
Grow at 18ºC until OD600 = 0.4-0.6 (approximately 24 hrs)
Day 3
Spin down at 2000 rpm for 20 min
Gently resuspend on ice with 66 ml of ice cold TB per 200 mL (1/3rd total volume) (avoid bubbles)
Spin down at 2000 rpm for 20 min
Gently resuspend on ice with 16 ml of ice cold TB per 200 mL (2/25th total volume) (avoid bubbles)
Slowly add 1.12 ml of DMSO/200 mL to 7% final
Incubate on ice for 10 min
Aliquot 100 µl per tube
Flash freeze in liquid nitrogen
Store in -80 ºC
Media recipes
Media Recipes
C-Media, undefined AA content
for 1L: Ingredients
10x C-media buffer [recipe below] 100mL
1000x MgSo4 7H2O 1mL
1000x Solution J 1mL
20% w/v Casamino Acids (fridge) 50mL
60% Glycerol (bench) 56.67mL
Water 791.33mL
C media buffer
KH2PO4 15g
Na2HPO4 (or NaHPO4 7H2O) 30g (or 56.6g)
NaCl 15g
NH4Cl 10g
Water Fill to 500mL
Here’s the rundown:
Dissolve all the chemicals in a graduated cylinder with a stir bar for C-media buffer and fill to 500 ml. Autoclave it.
Make the C-media (sans buffer) and autoclave it. The 1000x MgSO4·7 H2O stock should be 60 mg/ml. Solution J is upstairs between Dan’s and Rebecca’s benches. Casamino acids are a powder at the right end of the chemical storage upstairs, use 20 g into 100 ml water. Autoclave the 900 ml solution.
With sterile technique, add 100 ml of the autoclaved buffer solution to the 900 ml autoclaved media solution. You did it. You made C-media. We’re so proud.
P1vir Phage Transduction
Protocol pulled from Open Wet Ware and was originally created by the Sauer Lab
Background
Phage transduction is used to move selectable genetic markers from one "donor" strain to another "recipient" strain. Nat Sternberg, among others, pioneered the use of phage P1 to move genetic elements in E. coli and the use of the Cre/Lox system from P1 for controlled recombination. Today, phage P1 is commonly used as a transducing agent because it is a generalized tranducer (it can package random sections of the host chromosome instead of its own genome) giving rise to "transducing particles". P1vir is a mutant phage that enters the lytic cycle upon infection (ensuring replication and lysis). During the replication and lysis of the phage in a culture of bacteria, a small percentage of the phage particles will contain a genome segment that contains your gene of interest. P1 packages approximately 90 kb of DNA, so you can transduce genes that are linked to a selectable marker.
Once a phage population has been generated from a donor host, the phage are used to infect a recipient host. Most of the bacteria are lysed by phage that packaged P1 genomes, but a fraction of the phage inject a genome segment derived from the donor host. Homologous recombination then allows the incoming genomic segment to replace the existing homolgous segment. The infected recipient bacteria are plated on a medium that selects for the genome segment of the donor bacteria (antibiotic resistance, prototrophy, etc.
All of this would not work if the infectivity of the phage could not be controlled. Otherwise, phage released from neighboring cells would infect and lyse the bacteria that had been infected with transducing particles. Someone really smart discovered that phage P1 requires calcium for infectivity. Therefore, you can control P1 infectivity by growing in the presence and absence of calcium. The calcium chelator citrate is usually used because it lowers the concentration of free calcium (by forming Ca-citrate) low enough to prevent P1 infection, but not so low as to starve the cells for calcium.
Protocol
Reagents
1. LB-S1 (LB supplement 1)
* LB of any amount
* bring up to;
* 10-25 mM MgCl2
* 5mM CaCl2
* 0.1-0.2% glucose (1mL per lysate)
Recipe
 
Assuming 500mL of reagent
 
500mL
LB
2.033g
MgCl2 (assuming 20mM)
0.368g
CaCl2
?g
glucose
2. LB-S2 (LB supplement 2)
* LB of any amount
* bring up to;
* 100mM MgSO4
* 5mM CaCl2
* note: 10 mM MgSO4 works fine, too, so you can use the 0.1 M MgSO4 the kitchen
Recipe
 
Assuming 1L of reagent
 
1L
LB
6.019g
MgSO4
0.368g
CaCl2
?g
glucose
3. 1 M Na-Citrate
4. Normal LB
Lysate preparation
1. Dilute an overnight culture of donor strain grown with selection for the marker to be transduced 1:100 in fresh LB supplemented with 10-25 mM MgCl2, 5 mM CaCl2, and 0.1-0.2% glucose (1 mL per lysate)
DO NOT ADD ANTIBIOTIC TO THIS CULTURE
Set up at least 5 of these cultures
2. Grow with aeration at 37 ˚C for 1-2 h
3. When the cells are in early log phase (slightly turbid, but noticeable growth) Add a gradient of µL of P1 phage lysate to the culture ranging from 5uL to 100ul (5uL, 20uL, 50uL, 75uL, 100uL), continue growing at 37 ˚C. Monitor for 1–3 hr until the culture has lysed
You'll see cellular debris in the tube and the culture will have significantly lessened in its turbidity
4. Add several drops (50-100 uL) of chloroform to the lysate and vortex
5. Centrifuge away the debris (14,000 rpm, 1–2 min) and transfer the supernatant to a fresh tube
6. Add a few drops of chloroform and store at 4 ˚C
Transduction
1. Grow recipient strain overnight in LB medium
Site reccomends that 2mL culture is plenty
2. On the next day, harvest the cells by centrifugation (6000 rpm, 2 min) and resuspend in 1/5-1/3 the harvested culture volume in fresh LB + 100 mM MgSO4 + 5 mM CaCl2
3. Transfer 100 uL of transducing P1 lysate into a 1.5 mL microfuge tube for each transduction and incubate them with the caps opened at 37 ˚C for ~30 minutes
This step allows excess chloroform to evaporate from the phage stock
You can place your resuspended recipient strain in the incubator as well during this time to help them wake up from their nap
4. Set up four "reactions" for each of the 5 phage concentrations, by adding recipient bacteria to the tubes with phage, mix rapidly after addition, close the caps
100 µL undiluted P1 lysate + 100 µL recipient cells
100 µL 1:10 diluted P1 lysate + 100 µL recipient cells
100 µL LB + 100 µL recipient cells
100 µL undiluted P1 lysate + 100 µL LB
LB = LB + 100 mM MgSO4 + 5 mM CaCl2
dilute your P1 lysate in this as well
5. Incubate tubes at 37 ˚C for 30 min
6. Add 200 µL 1 M Na-Citrate (pH 5.5), then add 1 mL LB (the real thing this time) and incubate at 37 ˚C for 1 hr to allow expression of the antibiotic resistance marker.
If you are working with a marker or recipient that needs to grow at 30 ˚C, double the recovery time.
7. Spin cells at 6000 rpm for 5 min
8. Resuspend each in 100 µL LB supplemented with 100 mM Na-Citrate (pH 5.5), vortex well to disperse cells, and plate all of it on an appropriate antibiotic-containing plate
9. You should get anywhere from ~ 10 to 2000 colonies. These colonies are growing on a plate that is covered with P1 phage. If you simply pick a colony from this plate and prepare a freezer stock, you will most likely have phage contamination that will manifest when a culture is grown up in the absence of a calcium chelator. Therefore, prepare a plate spread with the selection antibiotic mixed in 100 µL of 1 M citrate (pH 5.5). Then, use a toothpick to touch the top of a few colonies and re-streak on the new plate for isolated colonies
10. Test a colony from each re-streak for the presence of the mutant gene you intended to transduce using diagnostic PCR or Southern blotting
PCR amplification with PFU
Rxn. Mixture
1 uL 10uM F primer
1 uL 10uM R primer
0.5 uL dNTPs
0.5 uL Pfu
2.5 uL 10x Pfu buffer
1 uL template
18.5 uL ddH2O MilliQ
Total: 25 uL
Program
95C 3min
95C 30sec
Tm-5C 30sec
72C X time 1min/1kb
back to #2 35 times
72C 3min
4C infinity
PCR Amplification with Phusion
Protocol taken from NEB manual.
PCR Amplification
For 50uL reactions:
Component
Volume
H20
fill to 50uL
5x Phusion Buffer*
10uL
DNTPs
1uL
Primer 1
2.5uL
Primer 2
2.5uL
Template DNA
1uL
DMSO*
1.5uL*
Phusion
0.5uL
*If the sequence has an extremely high GC content, use the Phusion GC buffer instead of the HF buffer. You must also add the DMSO
Cycle Step
Temperature
Time
Denaturation
98
30sec
Denaturation
98
10sec
Annealing
Varies
30sec
Extension
72
15-30sec/kb**
GoTo 2
-
24-34 times
Extension
72
5mins
Fridge
4
forever
**15 sec for amplification off of plasmid DNA or PCR Product, 30seconds for genomic DNA
PIPE Protocol
Length limit (approximate) ~8kb
PIPE Cloning Protocol
1) Linearize Vector (regular primers)
2) Amplify Insert (primers w/ overhangs)
a. For Optimal annealing temp of primers run a previous reaction w/ temp gradient and run gel to see which temp gives cleanest product. (I added protocol after this protocol)
3) EtOH precipitate Both PCR reactions separately
4) Cloning Procedure
a. Add 2uL of NEB buffer 4 + 0.5uL of DpnI to lin vector
b. Incubate @ 37C for min 1hr
c. Heat inactivate DpnI @ 80C for 20minutes
d. Combine 10uL of Insert and Vector each
e. Heat @ 94C for 1 minute and let cool to Room Temp
f. Transform into chemo or electro comp cells
Optimal Annealing Temp
1) Make a master mix (eg. 25uL) of PCR reaction
2) Alliquot into 5uL amounts into PCR tubes
3) Set up Thermo cycler so that Annealing Temp has a gradient 50C-70C
4) Arrange tubes in cycler to be in appropriate temp slots (eg. 50, 55, 60, 65, 70) (good idead to check what is closes to predicted Tm)
5) Run 5uL on Gel (remember which lane corresponds to which temp)
6) Lane with cleanest/most crisp band is appropriate temp to use.
Primer making
Making primers
Primers are single stranded DNA sequences that attach to the template on either side of the targeted gene. Primers are normally about 21 basepairs long. The easiest way to make primers is to use the Lasergene program or Ape program.
IMPORTANT: Before starting, check to see if there are any cut sites you will be using for cloning within the gene. If there are, you may need to find other compatible cut sites with compatible overhangs or mutate the cut site out. A program you can use to check is ApE and is a free download from the internet. Forward primer is the same as sequence and reverse primer is reverse complement of same sequence on other side of gene. For adding restriction cut sites, make sure they are in the 5’-3’ orientation for both the forward and the reverse. For example, in the reverse primer the cut sites should be (PstI) 5’-CTGCAG-3’ and not 3’-GACGTC- 5’. Below is an example of a desired gene. For the Forward primer, try and have the primer sequence be the sequence including the end part of the gene, so you don’t have too much of the junk DNA after the gene included as your insert part.
5’-TTTTCCTGGAATA[Beginning of gene] TTATCAGGGGGAGCCGTTGACCGTCGACGCGCGCAGCGTCGGACACAAT….….ATGAACAAAAACAGAGGGTTAACGCCTCTGGCGGTCGTTCTGATG -3’ For example, the dark grey is the highlighted sequence used as the primers and includes the beginning of the gene. So the sequence of the forward primer is 5’-TTATCAGGGGGAGCCGTTGAC-3’
Then add the cut site (EcoRI is in blue) in the 5’ to 3’ direction…
5’- GAATTC TTATCAGGGGGAGCCGTTGAC-3’
Then add junk DNA (i.e. about 3C or T) before the last cut site, so the enzyme has room to grab the DNA to cut. So the finished forward primer, should look like…
5’- CCC GAATTC TTATCAGGGGGAGCCGTTGAC-3’
The reverse primer is the reverse compliment of the highlighted sequence. The easiest way is to convert the highlighted sequence is to use an oligonucleotide analyzer, such as Integrated DNA Technologies (just google IDT). So the primer should look like …
5’- CGACCGCCAGAGGCGTTAAC -3’
Then add the cut sites (PstI is in red) in the 5’ to 3’ direction…
5’- CTGCAG CGACCGCCAGAGGCGTTAAC -3’
Then add junk DNA (i.e. about 6 C or T) before the last cut site, so the enzyme has room to grab the DNA to cut. So the finished reverse primer, should look like…
5’- CCCCTGCAG CGACCGCCAGAGGCGTTAAC -3’
Once you have your primers, use IDT to analyze them for hairpin structures.
Sequencing through Functional Biosciences
Note: In order to get sequence data by noon the next day, it must be submitted by 4. Only submit once a day.
1. Go to order.functionalbio.com on the right lab computer
2. Login with '???????' and '???????'
3. Go to 'Submit sequencing requests'
4. Leave the default information and click continue
5. Select what you're sending (probably the default purified plasmid)
6. Click 'Go' then fill out the information for the sample (making use of 'Length in Plasmids…' under 'Tools' on the part page)
7. Print the invoice, and call Badger Cab 608 256 5566 and ask them to deliver the sample to:
Functional Biosciences, 505 South Rosa Street Suite 17 Madison, WI 53719
8. Include the printed out invoice in the container with the plasmid (and possibly primers)Details on samples to send:
Plasmids should be provided at ~100ng/ul in total volume of 12ul. PCR products            should be provided at ~20-50ng/ul in a total volume of 12ul. If you are doing more than two reactions per            sample please increase the volume provided by 3ul for each additional reaction.
Transforming into Chemically Competent Cells
1. Thaw 100uL chemically competent cell aliquots on ice
2. Put 3uL (or 200-400ng) of ligation product into each Remember a control for media AND your part (if possible)
3. Wait 5 min
4. Heat Shock 45 sec. @ 42ºC while keeping the tubes completely still
5. Recover 2 min on ice
6. add 1ml LB
7. Recover 1 hr. 37ºC in the shaker at 100rpm
8. Pipette out 100µL and plate out on selective media
9. If there is reason to doubt the number of colonies from the 100uL aliquot, spin down the remaining cells, re-suspend in 100µL LB, and plate onto appropriate selective media
Using primers from IDT
Rehydrating primers
Each blue-capped tube from NEB contains a dry DNA film at the bottom of the tube. The label (and paperwork which comes with the primer) reports the amount of DNA in the tube in nanomoles at the bottom right, and is usually in the range of 20 to 30. To this blue-capped tube, add 10 uL of (ideally nuclease-free) water for every nanomole of DNA (for example, 22.7 nm of DNA would get 227 uL water). Put the tube in the 50 degree water bath for 30 minutes to dissolve the DNA in the water. This is now your primer stock.
Aliquoting primers for use
Before using the primers in a PCR reaction, you need to make 10x dilutions. The volume's not terribly important, but in the past aliquots of 50 uL usually start causing problems (non-specific amplification, etc.) before they run out. 20 uL aliquots seem to be good (2 uL of your primer stock, 18 uL water).
What's the concentration?
Following the above protocol, the primer stock is 100 uM and the aliquots are 10 uM.
Phusion reactions are supposed to be run at 500 nM (.5 uM) primer concentration, so 1/20 of the reaction volume should be each primer (2.5 uL of the aliquot per 50 uL Phusion reaction)
GoTaq reactions can be run with primers between 0.1-1.0 uM, so anywhere from 1 to 10% of the reaction volume should be each primer.
Viability Test for Toxicity Analysis of Bacterial Cultures
Toxicity Testing Protocol
The night before, start 5 mL cultures of the strains to be tested (MG1655, DH1, etc.). The following morning, aliquot 25 mL of medium (whichever type you like, I normally just use LB) into an appropriate number of 250 mL baffled shake flasks.
Growing the Cultures
1. Take the OD600 of the overnights.
2. Inoculate cultures to an OD600 of .01 using the following formula: You're probably familiar with this formula from general chemistry. Because the OD600 is proportional to the concentration (as long as we are within the linear range of the spectrophotometer) of cells in a culture, we may treat OD600 as a concentration. The reason for doing this is to inoculate each culture with approximately the same number of cells so that no culture has a growth advantage from having a larger inoculum. For example, say we took the OD600 of an overnight of E. coli and found it to be 5.11. If we let V2 in the above equation be the volume of overnight we want to add to 25 mL of LB to achieve an OD600 of .01 in 25 mL of medium, then solving for V2 we get: So to inoculate the 25 mL culture we would need to add 49 μL of the overnight. Note the volume of the new culture is not exactly 25 mL, it's actually closer to 25.049 mL. In the above calculation we have neglected the volume added (this is a very good approximation).
3. Add the appropriate amount of solvent to each shake flask. It is normally advisable to have a no-solvent control. When dealing with solvents that are sparingly soluble in water, I like to measure "concentration" in terms of volume of solvent added per volume of culture (for example, 10 μL of R-Limonene per mL of culture, which just means that I added 10 μL of solvent for every 1 mL of culture that was originally present). This is more correct because using volume % and molarity assumes you have a single-phase system - you frequently will not (it is preferable to use molarity if you have a single phase system)
4. Incubate for 24 hours at 37°C with shaking (250 RPM). This works for E. coli - if you ever use another microorganism you will likely have to change how you incubate your cultures. 5. After 24 hours, remove the cultures from the incubator. If there is an organic layer present (i.e. if you are working above the solubility limit of your solvent), allow the cultures to settle for approximately 30 minutes (depending on the solvent more or less time may be needed).
Viability Testing
1. Set up Eppendorf tubes with 900 μL of PBS (phosphate buffered saline) in each. You will need 6 tubes per culture. Position each set of six tubes in a line.
2. Pipette 100 μL from the aqueous phase of each culture into the first tube in the corresponding series of 6 tubes. Be sure to label the first tube in each series appropriately!
3. The first tube in each series is diluted 10 times from the original culture. Vortex this tube and add 100 μL of this suspension to the next tube in the line (this will be diluted 100 times from the original culture). Continue until you have diluted the culture 106 times, vortexing the suspension before diluting it.
4. Repeat step 3 for each set of 6 tubes.
5. Spot 3 μL of each 104, 105, and 106 dilution onto an LB agar plate.
6. Incubate the plates overnight at 37°C (~14 hours is sufficient). Because the plates are likely still wet, make sure not to turn them up-side down.
7. Remove and image the plates the following morning.
If you do not have an emulsion of solvent you may just take the OD600 like you normally would. Withdraw the appropriate volume from the aqueous phase and dilute it (if necessary), then take the OD600. If you do have an emulsion, you will need to wash your cells before taking an OD600 measurement, as the presence of solvent bubbles will greatly inflate your OD600 measurement.
1. Aliquot 1.5 mL of the aqueous phase of each culture into Eppendorf tubes.
2. Centrifuge tubes for 10 minutes at 5000xg. Some samples may have essentially no cells (no cell pellet). You can just ignore those samples and not bother washing them.
3. Decant the liquid from each sample.
4. Resuspend in 1.5 mL of PBS.
5. Repeat steps 2-4 two more times.
6. Using PBS as a blank, take the OD600 of each sample. It may be necessary to dilute some samples 1:10.
If you want you can also take the OD600 directly from the aqueous phase of each culture. Some samples will show that they have a significant OD600 when the only thing present is bubbles of solvent. It is normally a good idea to do this for cultures that give no cell pellet while washing the cells. If this is the case you will know from your viability data, as nothing will have grown.
Washing Beads
1. Pour beads into a flask.
2. Add warm water, Alconox and bleach to flask, cap it.
3. Place in shaker overnight.
4. The next morning, dump out the wash solution and rinse repeatedly with deionized H20 until water is completely clear and no soapy bubbles remain.
5. Dump out all remaining water.
6. Ideally - you would place the beads in an oven to dry at this step, but we don't have an oven so don't worry about it.
7. Dispense the beads into desired jar/ container.
8. Autoclave - run the longest possible gravity dry cycle.
Retrieved from " http://2012.igem.org/Team:Wisconsin-Madison/protocol "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
