<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Wisconsin-Madison_modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Wisconsin-Madison/modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Wisconsin-Madison/modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Wisconsin-Madison/modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Wisconsin-Madison/modeling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Wisconsin-Madison/modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Wisconsin-Madison/modeling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="divscontainer"><CENTER><DIV id="divthebuttons"><UL class="main-menu"><LI id="n_team"><H2><STRONG>Team</STRONG></H2></LI><LI><A href="/Team:Wisconsin-Madison/teammembers">Team Members</A></LI><LI><A href="/Team:Wisconsin-Madison/advisors">Advisors</A></LI><LI><A href="/Team:Wisconsin-Madison/thanksponsors">Sponsors</A></LI></UL><LI id="n_project"><H2><STRONG>Projects</STRONG></H2></LI><LI><A href="/Team:Wisconsin-Madison/lemon">Engineering Limonene Production in E.coli</A></LI><LI><A href="/Team:Wisconsin-Madison/SDF">Translational Coupling Cassette (TCC)</A></LI><LI id="n_part"><H2><A href="/Team:Wisconsin-Madison/parts"><STRONG>Parts</STRONG></A></H2></LI><LI><A href="/Team:Wisconsin-Madison/parts_1">Translational Coupling Cassette</A></LI><LI><A href="/Team:Wisconsin-Madison/parts_2">Negative TCC Control</A></LI><LI><A href="/Team:Wisconsin-Madison/parts_3">Positive TCC Control</A></LI><LI><A href="/Team:Wisconsin-Madison/parts_4">Codon-optomized LIMS1</A></LI><LI id="n_modeling"><H2><A href="/Team:Wisconsin-Madison/modeling"><STRONG>Modeling</STRONG></A></H2></LI><LI id="n_note"><H2><A href="/Team:Wisconsin-Madison/protocol"><STRONG>Protocol</STRONG></A></H2></LI><LI id="n_safety"><H2><A href="/Team:Wisconsin-Madison/safety"><STRONG>Safety</STRONG></A></H2></LI><LI id="n_humanPractice"><H2><A href="/Team:Wisconsin-Madison/hpractice"><STRONG>Human practice</STRONG></A></H2></LI></DIV><DIV id="divthecontent"><P align="left" class="classtheinlinecontent"><STRONG style="font-size:25px; color: rgb(183, 1, 1);">Kinetic modeling for the limonene production pathway</STRONG></P><P align="left" class="classtheinlinecontent"><STRONG style="text-decoration: underline;">Objective</STRONG>: To model the limonene production pathway and find the effect of different enzyme activities on maximum limonene production</P><P align="left" class="classtheinlinecontent"><STRONG>Author      : </STRONG>Shashank Shekhar</P><P align="left" class="classtheinlinecontent"><STRONG>Advisors : </STRONG>Jennifer Reed, PhD Brian Pfleger, PhD Matthew Copeland, PhD</P><P align="left" class="classtheinlinecontent" style="text-decoration:underline; font-size:20px;"><STRONG>Introduction</STRONG></P><BLOCKQUOTE><P align="left" class="classtheinlinecontent">An enzyme kinetic modelling of the Limonene production pathway is done in this study. The work reported uses Michaelis-Menten assumptions to model the enzyme activity and rate of production of substrates</P></BLOCKQUOTE><BLOCKQUOTE><P align="left" class="classtheinlinecontent">A set of ordinary differential equations were used to represent change in concentration of different substrates in the pathway using the Michaelis-Menten equations and solved using the ode45 solver available in <STRONG style="text-decoration:underline;"> MATLAB</STRONG>. The given differential equations is the rate of production of limonene, V<SUB>9M</SUB> and K<SUB>9</SUB> are the V<SUB>Max</SUB> and K<SUB>m</SUB> for the Limonene synthase enzyme.</P></BLOCKQUOTE><BLOCKQUOTE><P align="left" class="classtheinlinecontent">A steady state analysis was targeted and concentration of all enzymes was kept at <STRONG>10<SUP>-6</SUP>mM</STRONG>. The initial concentration of all substrates was set to <STRONG> 0mM</STRONG>.The equations solved at (<STRONG>T = 10<SUP>13</SUP> seconds</STRONG> equivalent to <STRONG>27x10<SUP>8</SUP>hours</STRONG>) did not achieve steady state. The substrates <STRONG>acetoacetyl-CoA</STRONG>, GPP and Limonene did not achieve a steady concentration. The concentration of mevalonate phosphate (Mev-P) showed stable oscillations. It was concluded that a steady state analysis cannot be used.</P><P align="left" class="classtheinlinecontent">The aim of the project is to find the optimum range of enzyme concentrations for all enzymes in the limonene synthesis pathway used in our engineered organisms after maximizing Limonene production in a in-silico model of the same pathway. The control parameters for the model simulation are only the enzyme concentrations. This input parameter (Ei, Enzyme concentration) was in real-time optimized using a direct evolution algorithm. The range of enzyme concentrations in which maximum limonene (with a tolerance of 1%) in time T was produced in the in-silico model is the result. 
  </P></BLOCKQUOTE><P align="left" class="classtheinlinecontent" style="text-decoration:underline; font-size:20px;"><STRONG>Enzyme Kinetic Data: </STRONG></P><BLOCKQUOTE><P align="left" class="classtheinlinecontent">
    The enzyme kinetics data was collected through research in various journals and the online database available on <A href="www.brenda-enzymes.info">Brenda-enzyme</A>.
  </P></BLOCKQUOTE><BLOCKQUOTE><P align="left" class="classtheinlinecontent">Km values in the original organism were available for all enzymes except phosphomevalonate Kinase (ERG8), in this case the protein sequence for ERG8 was blasted (NCBI) and the closest homologous sequence from a different organism human protein ERG8 was used in the model<SUP>3</SUP>. Similarly for K<SUB>cat</SUB> values where data was not available fitting approximations from a different source was made.
  </P><P align="left" class="classtheinlinecontent">The mevalonate pathway begins with acetyl-CoA, an abundant cellular metabolite. Studies on recombinant poly(3-hydroxybutyrate) (PHB) production demonstrated that acetyl-CoA is available in <I> E. coli </I> to produce up to194.1 g/L of PHB from the native source of E.Coli. <STRONG style="text-decoration:underline;"> (Choi et al., 1998; Choi and Lee,1999)</STRONG>. Hence, the input substrate acetyl – CoA and other key substrates in the model (ATP, NADPH) were assumed at a constant level,<STRONG>A-CoA<SUP>4</SUP> = 0.96 mM ATP = 0.12 mM NADPH<SUP>4</SUP> = 0.61 mM</STRONG>. These substrates are part of pathways in the cell metabolism. It is essential that the cell keep them at a steady state concentration. Mevalonate is not known to be metabolize inside the cell and same is true for subsequent intermediate substrates in the pathway. Hence, any regulation in the organism has also been excluded in the model. The minor substrates such as CoA is not considered in the rate equation since it is recycled in the pathway and it is an important co-factor maintained in a steady concentration in the cell. The reported Km values of the genes reported in <STRONG>Table 1</STRONG> were used. The tHMGR is substituted with a more stable enzyme HMG2. The kinetic data of other enzymes is from in-vitro assays reported in other publications.
  </P></BLOCKQUOTE><P align="left" class="classtheinlinecontent" style="text-decoration:underline; font-size:20px;"><STRONG>Model specifications: </STRONG></P><BLOCKQUOTE><P align="left" class="classtheinlinecontent">The maximum enzyme concentration was limited to 10mM, the limit was guessed using data available on protein concentration in E.Coli cytosol reported in<SUP>5</SUP>. The 10mM maximum enzyme concentration value is corroborated with data available on high copy number of oriCs in available plasmids and transcription rates [Dennis and Bremer 1996], so that the cell can achieve the necessary enzymes concentration in the growth phase. 
  </P><P align="center" class="classtheinlinecontent">Enzyme concentration = Transcription rate x Copy number of plasmid x Time in growth phase
  </P><P align="left" class="classtheinlinecontent">The time of T = 100Hrs was used to solve the set of ordinary differential equations to capture sufficient time of cell in stationary phase. The optimization model shown in <STRONG style="text-decoration:underline;">figure 3</STRONG>, takes given Time, initial concentration of substrates and enzyme kinetic model. The algorithm sets the concentration of all enzymes at initially 1nM. Through a directed evolutionary approach, the concentration of the enzymes in sequence beginning from E1, is modified by a multiplication factor and the new Limonene produced by the model solved at time (T) is calculated by the algorithm. 
  </P><P align="left" class="classtheinlinecontent">The best performing enzyme concentration of E1 is fed back to the model. In the subsequent turn the following enzyme E2 is multiplied from a set of factors and sent as input. Similarly, the best performing enzyme concentration is fed back to the model for the next enzymes (E3)‘s run. 
    The algorithm is run several times until the maximum limonene produced reaches a stable concentration with the enzyme concentrations optimized at after each run. 
  </P></BLOCKQUOTE><P align="left" class="classtheinlinecontent" style="text-decoration:underline; font-size:20px;"><STRONG>Results: </STRONG></P><BLOCKQUOTE><P align="left" class="classtheinlinecontent">The concentration of limonene produced , using the kinetic model solved at <STRONG>T=100</STRONG> Hours (Figure below) at different concentration of each enzyme, keeping the concentration of other enzymes at their best optimum, shows that HMGS, GPPS and LIMS are the rate limiting enzymes . The production is also sensitive to the concentration of IDI.  A comparison of rate of change in slope of the curves of HMGR, GPPS and LIMS showed that the limonene production was more sensitive to HMGS and LIMS compared to GPPS, at high concentration near 10mM.</P></BLOCKQUOTE><P align="left" class="classtheinlinecontent" style="text-decoration:underline; font-size:20px;"><STRONG>Criticism: </STRONG></P><BLOCKQUOTE><P align="left" class="classtheinlinecontent">•	Substrates in the pathway such as limonene and HMG-CoA have been shown to be toxic<SUP>1</SUP> to cell growth. Hence the over-expression of HMGS for large production will have toxic effect on cell growth. In which case, HMGS may not play the role of the rate limiting enzyme.</P><P align="left" class="classtheinlinecontent">•	Many enzymatic reactions are reversible in this pathway, however in this model all reactions have been considered to be irreversible. This fallback is probably responsible for the sensitivity of the IDI enzyme reported, since IPP -&gt; DMAPP maintains an equilibrium balance using only a forward reaction.</P></BLOCKQUOTE><P align="left" class="classtheinlinecontent" font-size:20px=""><STRONG>Reference: </STRONG></P><DIV align="left" class="classtheinlinecontent"><UL><B>Piterra</B><B><I>et. al,</I></B> 2007. Balancing a heterologous mevalonate pathway for improved isoprenoid
production in <I>Escherichia coli</I>. Metabolic Engineering. 2:193-207
</UL><UL><B>Dunlop</B><B><I>et. al,</I></B> 2011. Engineering microbial biofuel tolerance and export using efflux pumps. Molecular Systems Biology. 7:487:487
</UL><UL><B>Primak</B><B><I>et. al,</I></B> 2011. Characterization of a Feedback-Resistant Mevalonate Kinase from the Archaeon <I>Methanosarcina mazei</I>. Applied Environmental Microbiology. 77:7772-7778
</UL><UL><B>Bennett</B><B><I>et. al,</I></B> 2009. Absolute metabolite concentrations and implied enzyme active site occupancy in Escherichia coli. Nat Chem Bio. 5:593-599
</UL><UL><B>Ishihama</B><B><I>et. al,</I></B> 2009. Protein abundance profiling of the <I>Escherichia coli</I> cytosol. BMC Genomics. 9:102
</UL><UL><B>Martin</B><B><I>et. al,</I></B> 2003. Engineering a mevalonate pathway in <I>Escherichia coli</I> for production of terpenoids. Nature Biotechnology. 21:7:796-802
</UL></DIV></DIV></CENTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Wisconsin-Madison/modeling">http://2012.igem.org/Team:Wisconsin-Madison/modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Wisconsin-Madison/modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Wisconsin-Madison/modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Wisconsin-Madison/modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Wisconsin-Madison/modeling&amp;oldid=263196" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>