"
Page
Discussion
View source
History
teams
Log in
 
Team:Wisconsin-Madison/lemon
From 2012.igem.org
Team
Team Members
Advisors
Sponsors
Projects
Engineering Limonene Production in E.coli
Translational Coupling Cassette (TCC)
Parts
Translational Coupling Cassette
Negative TCC Control
Positive TCC Control
Codon-optomized LIMS1
Modeling
Protocol
Safety
Human practice
Engineering a limonene production pathway of E.coli
Why Produce Limonene?
Limonene is a 10-carbon monoterpene found in the oils of citrus fruits.  It is used as a cleaning agent, solvent, and food additive, and recent research has shown it has potential anti-cancer applications. Limonene also possesses the chemical properties of an ideal biofuel; its low freezing point, combustibility, and high energy density make it a potential jet fuel replacement. Currently, industrial production is restricted to direct extraction from citrus fruits, preventing collection in quantities large enough to be useful as a biofuel. Other organisms, such as Escherichia coli , could be used to produce limonene more effectively and efficiently.
The Mevalonate Pathway
The mevalonate pathway, found in plants and fungi, produces 3-isopentenyl pyrophosphate (IPP) and dimethylallyl pyrophosphate (DMAPP) from acetyl-CoA. IPP and DMAPP are the building blocks of the isoprenoids, a group of organic molecules with a wide array of functions. These compounds are normally synthesized in small amounts, preventing large-scale production; by moving the mevalonate pathway into E. coli this limitation can be circumvented. The Keasling lab at UC-Berkeley used this approach to produce the isoprenoid amorphadiene, which can be converted into the antimalarial drug artemisinin. This breakthrough brought the price of artemisinin down dramatically for treatment in the developing world.
We have constructed a strain containing the mevalonate pathway, a codon-optimized geranyl diphosphate synthase (GPPS), and a codon-optimized limonene synthase gene. The plasmid pBba5c contains the genes for the mevalonate pathway under a lactose-inducible promoter. The plasmid originally contained the unnecessary ispA gene; we used the Gibson cloning method to replace ispA with the GPPS gene. The resulting plasmid, pBba5c-GPPS, should theoretically allow higher production of limonene.
The Production Assay
Five milliliter cultures of each strain (listed below) were grown overnight at 37°C in LB, then normalized to an OD600 of 1 and diluted 1:100 into 40 mL cultures of LB.  These were then grown to an OD600 of 0.2 and the promoters in the pBba5C vector induced with 1 mM IPTG. A 10 mL overlay of dodecane was used to trap any limonene produced.  These cultures were then grown for 18 hours, centrifuged, and 1mL of the dodecane overlay diluted in ethyl acetate and analyzed by GC/MS.
Strain
Purpose
pBba5c + J23102-RFP
J23102 promoter only as a negative control
pBba5c + J23102-LIMS1
Production strain
pBba5c + J23102-CO_LIMS1
Production Strain
pBba5c + pTRC-ADS
pTRC-ADS is the amorphadiene synthase
pBba5c-GPPS + J23102-RFP
Negative control
pBba5c-GPPS + J23102-LIMS1
Production strain
pBba5c-GPPS + J23102-CO_LIMS1
Production strain
pBba5c-GPPS + J23102-pTRC-ADS
Testing the GPPS-ispA swap with gibson
In figures A, C, and D, the large black limonene peak is a doped limonene standard included as a positive control for the GC/MS assay. The blue and purple lines cannot be seen, because they are along the baseline; i.e. no limonene is being produced. J23102 (empty) in figures A, B, and D is the promoter run as a negative control. Figure B demonstrates the decrease in amorphadiene between the regular pBbA5c and the pBbA5c with the GPPS swapped in for the ispA gene.
As seen in this GC/MS data, limonene was not produced. There was a small peak located where limonene was expected, but this peak also appeared in the empty promoter strain as well. In order to the strain's production capacity, we replaced the limonene synthase with amorphadiene synthase. The ispA gene produces farnesyl pyrophosphate (FPP), the precursor to amorphadiene. Both the pBbA5c with ispA and GPPS were used to troubleshoot the mevalonate pathway by attempting to generate amorphadiene. In the GC/MS data, both strains created amorphadiene when amorphadiene synthase was included in the strain. Thus, we surmise that the mevalonate pathway is functioning correctly. The pBba5C-GPPS strain with amorphadiene synthase should not create amorphadiene, as ispA has been replaced with GPPS; however, because the E. coli genome naturally contains
, some amount may still be produced. The amorphadiene peak associated with the pBbA5c-GPPS was much smaller than the peak generated by the pBbA5c-ispA construct, as expected. However, this does not prove that the GPPS is working as anticipated.
The Conclusion
The data shows that the mevalonate pathway is functioning correctly. This indicates that the problem is either limonene synthase or GPPS. To further assess the production strains, a growth curve was run to determine if cell viability was being affected by the synthetic pathway.
Growth curves are organized either by base mevalonate vector or by version of limonene synthase. pBAD33 mimics pBbA5c, acting as an empty vector. Four replicates of each strain were grown on a 96-well plate; error bars indicate standard deviation.
The growth curves show two important things. The naturally-occurring limonene synthase shows a growth defect in comparison to the synthesized codon-optimized limonene synthase. More notably, the pBbA5c with ispA greatly inhibits cell growth. A possible explanation for this is the toxicity of FPP, as the pBbA5c-GPPS did not inhibit cell growth in comparison to the pBbA5c-ispA. Since ispA is native to E. coli , it should be translated more efficiently than the other enzymes in the mevalonate pathway, including GPPS. To further pursue this hypothesis it would be beneficial to insert GPPS and ispA into the Translation Coupling Cassette (TCC) and compare their translation efficiency. We are focusing on GPPS as the issue as our TCC research shows that the codon-optimized limonene synthase is being translated (See TCC page for further explanation). However, successful translation does not show that it is functional in the cell. This will be be assessed by in vitro assay, as described on the TCC project page. With this additional data we can work to further improve the production capacity of our strain.
References:
Dunlop et. al, 2011.Engineering microbial biofuel tolerance and export using efflux pumps. Molecular Systems Biology. 7:487:487
Kirby et. al, 2009. Biosynthesis of plant isoprenoids: perspectives for microbial engineering. Annual Review of Plant Biology. 60:335-355
Lücker et. al, 2002. Monoterpene biosynthesis in lemon (citrus lemon). European Journal of Biochemistry. 269:13:3160-3171
Martin et. al, 2003. Engineering a mevalonate pathway in Escherichia coli for production of terpenoids. Nature Biotechnology. 21:7:796-802
Retrieved from " http://2012.igem.org/Team:Wisconsin-Madison/lemon "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
