"
Page
Discussion
View source
History
teams
Log in
 
Team:Wisconsin-Madison/SDF
From 2012.igem.org
Team
Team Members
Advisors
Sponsors
Projects
Engineering Limonene Production in E.coli
Translational Coupling Cassette (TCC)
Parts
Translational Coupling Cassette
Negative TCC Control
Positive TCC Control
Codon-optomized LIMS1
Modeling
Protocol
Safety
Human practice
How the translational coupling cassette works
The translational coupling cassette (TCC) is used to assess translation efficiency of heterologous proteins in E. coli. The gene of interest is fused to a downstream reporter gene, linked by a region encoding a histidine tag and a ribosome binding site (Fig. 1). When the construct is transcribed, the His-tag and RBS form a hairpin structure in the mRNA. If the gene of interest is not fully translated, the hairpin remains intact and prevents translation of the reporter gene by occluding the RBS. However, if the gene of interest is properly translated, the ribosomal helicase activity is sufficient to break the hairpin, exposing the RBS and enabling translation of the reporter gene (Figure 2). In our system, the reporter gene is a red fluorescent protein (RFP) from the Parts Registry (BBa_E1010). This allows the user to visually determine the translation efficiency of their gene of interest.
Assays for the translational coupling cassette
The fluorescence produced by the translational coupling cassette was assessed using a Typhoon Imager (GE Healthcare, provided by the UW-Madison Biotech Center). This instrument excites a sample with a laser (532 nm) and detects the emitted fluorescent signal via a photomultiplier tube. We spotted 10 uL droplets of our various strains (listed below) on selective media plates containing exactly 20mL of LB agar. The plates were incubated overnight before being imaged on the Typhoon Imager and analyzed.
Construct
Purpose
TCC
Negative control for fluorescence
J23102 - E0040 - TCC
Positive control for testing translation of the target gene without the use of the TCC
J23102 - Lims1 - TCC
Testing translation
J23102 - Lims1 -Stop- TCC
Negative control with a stop codon for testing translation
J23102 - COLims1 - TCC
Testing translation
J23102 - COLims1 -MidStop- TCC
Negative control for testing translation
J23102 - E0840
Testing the amount of RFP signal and ensuring that we account for GFP fluorescence bleed-through into the RFP channel in our positive control construct
The J23102-E0040-TCC construct was used as a positive control to test the functionality of the translational coupling cassette; E0040 is well-characterized and routinely used by many iGEM teams, suggesting it should not have any issues being fully translated. When scanning samples in the Typhoon Imager, the GFP-expressing (J23100-E0040) construct that was included in the assay produced low levels of fluorescence in the red fluorescent channel. This is not an uncommon problem when using fluorophores, particularly fluorescent proteins, and is due to bleed-through of the GFP fluorescent signal into the RFP emission channel. Thus, some of the red fluorescence signal produced by our positive control construct (J23102-E0040-TCC) was due to the GFP. This required us to account for the bleed-through to determine if the RFP signal from our GFP-TCC-RFP construct was actually from the RFP reporter gene or just bleed-through from the GFP. Figure 3c. (below) demonstrates that the signal intensity from the GFP-TCC-RFP construct is well above the signal produced from the GFP-only strain, showing that our translational coupling cassette is working as intended.
Figure 3A: Quantification of fluorescence from Fig.3C by pixel saturation. From liquid cultures of each strain, three 10uL droplets were spotted onto 20mL LB agar plates with proper selection. Using Adobe Photoshop, the average pixel saturation of the three droplets from the Typhoon Image (Fig.3C) was calculated, normalizing for background fluorescence.Figure 3B: Plate as imaged by Nikon camera Figure 3C: Typhoon Image of plate in Fig.3B, edited using Adobe Photoshop
The codon optimized limonene synthase, with a stop codon located in the middle of the gene, was used as our negative control for the cassette. If translation is stopped partway through the target gene, the helicase activity on the ribosome will not break the hairpin and RFP will not be generated.
From this data, three things can be determined. First, the positive control is producing RFP, meaning the hairpin is being broken by the ribosome. This is seen in Figure 3A and 3C, where the J23102-E0040-TCC is well above the negative controls. Second, the construct containing a stop codon midway through the target gene is not generating RFP, which is seen in the same two figures. This demonstrates that the hairpin stays intact when the target gene does not translate. Third, the codon-optimized limonene synthase is being translated, and the non-codon optimized is not. The fluorescence produced by the codon-optimized construct is much lower than the positive control, but we cannot definitively explain this.
The second assay used to quantify fluorescence was done by using a 96-well plate reader. Optical densities and fluorescence were taken over a 24 period, and used to determine if our target gene was being translated. The same strains used in the Typhoon were tested using this method. Fluorescence was divided by optical density because all of the strains did not grow at the same rate. This allowed the normalization of fluorescence, and thus the quantification and comparison of it between strains.
Figure 4: Plot of fluorescence over turbidity
Figure 5: Plot of fluorescence over turbidity
These plots supports the conclusions drawn from the Typhoon images. The translational coupling cassette produces RFP. The codon optimized limonene synthase seems to be translating more efficiently than the non-codon optimized version. In fact, the limonene synthase may not be translating at all. However, we still have not been able to detect any amount of limonene in our GC/MS assays. We were able to make amorphadiene, so we know that the mevalonate pathway works correctly up until the GPP synthase gene. We know the limonene synthase (now codon optimized) is being translated. From here, we would require more time and materials to fully troubleshoot why no limonene is being produced. We do have a few hypotheses, based on observations and information about this pathway. First, limonene synthase may be forming an inclusion body in the cell. Second, it is a possibility that GPP is not being produced, and we would require standards (which are quite expensive) and a new protocol to detect for it on the GC/MS. If we had these standards, we would also be able to try to purify limonene synthase using our TCC his-tagged version. This purified enzyme could then be doped with GPP, and potentially create limonene. If this was the case, we would know that GPP is not being produced, or something is happening to the limonene synthase between translation and the active synthesis of limonene. Third, many of the molecules and enzymes in the mevalonate pathway can be toxic to Escherichia coli cells at high levels, including limonene.
How to clone using the Translational Coupling Cassette
In order to use the translational coupling cassette, the target gene must be PCR amplified using BioFusion primers. If the gene is amplified using standard BioBrick primers, an open reading frame shift will occur, a premature stop codon will be formed, and the hairpin may not break. In addition to using BioFusion primers, the stop codon must also be taken out of the target gene being amplified for the same reason. If a stop codon prematurely stops translation, then the hairpin may not break and the response gene (RFP) will not be translated. As an example:
We cloned in the E0040 gene (GFP) into the TCC as the target gene.  The first and last 30 base pairs of the E0840 ribosome binding site plus open reading frame (GFP) are as follows:
atgcgtaaaggagaagaacttttc
catggcatggatgaactatacaaataataa
The forward and reverse primers (respectively) would then have to be:
5’ 3’
CCCCGAATTCGCGGCCGCTTCTAGAatgcgtaaaggagaagaacttttc
5’                                                          3’
CCGCTACTAGTtttgtatagttcatccatgccatg
This PCR fragment would then have to be cut with EcoR1 and Spe1, while the translational coupling cassette should be cut with EcoR1 and Xba1.
Reference:
Mendez-Perez et. al, 2012. A translation-coupling DNA cassette for monitoring protein translation in bacteria. Metabolic engineering. 14:4:298-305
Retrieved from " http://2012.igem.org/Team:Wisconsin-Madison/SDF "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
