"
Page
Discussion
View source
History
teams
Log in
 
Team:Wisconsin-Madison/modeling
From 2012.igem.org
Team
Team Members
Advisors
Sponsors
Projects
Engineering Limonene Production in E.coli
Translational Coupling Cassette (TCC)
Parts
Translational Coupling Cassette
Negative TCC Control
Positive TCC Control
Codon-optomized LIMS1
Modeling
Protocol
Safety
Human practice
Kinetic modeling for the limonene production pathway
Objective: To model the limonene production pathway and find the effect of different enzyme activities on maximum limonene production
Author      : Shashank Shekhar
Advisors : Jennifer Reed, PhD Brian Pfleger, PhD Matthew Copeland, PhD
Introduction
An enzyme kinetic modelling of the Limonene production pathway is done in this study. The work reported uses Michaelis-Menten assumptions to model the enzyme activity and rate of production of substrates
A set of ordinary differential equations were used to represent change in concentration of different substrates in the pathway using the Michaelis-Menten equations and solved using the ode45 solver available in MATLAB. The given differential equations is the rate of production of limonene, V9M and K9 are the VMax and Km for the Limonene synthase enzyme.
A steady state analysis was targeted and concentration of all enzymes was kept at 10-6mM. The initial concentration of all substrates was set to 0mM.The equations solved at (T = 1013 seconds equivalent to 27x108hours) did not achieve steady state. The substrates acetoacetyl-CoA, GPP and Limonene did not achieve a steady concentration. The concentration of mevalonate phosphate (Mev-P) showed stable oscillations. It was concluded that a steady state analysis cannot be used.
The aim of the project is to find the optimum range of enzyme concentrations for all enzymes in the limonene synthesis pathway used in our engineered organisms after maximizing Limonene production in a in-silico model of the same pathway. The control parameters for the model simulation are only the enzyme concentrations. This input parameter (Ei, Enzyme concentration) was in real-time optimized using a direct evolution algorithm. The range of enzyme concentrations in which maximum limonene (with a tolerance of 1%) in time T was produced in the in-silico model is the result.
Enzyme Kinetic Data:
The enzyme kinetics data was collected through research in various journals and the online database available on Brenda-enzyme .
Km values in the original organism were available for all enzymes except phosphomevalonate Kinase (ERG8), in this case the protein sequence for ERG8 was blasted (NCBI) and the closest homologous sequence from a different organism human protein ERG8 was used in the model3. Similarly for Kcat values where data was not available fitting approximations from a different source was made.
The mevalonate pathway begins with acetyl-CoA, an abundant cellular metabolite. Studies on recombinant poly(3-hydroxybutyrate) (PHB) production demonstrated that acetyl-CoA is available in E. coli to produce up to194.1 g/L of PHB from the native source of E.Coli. (Choi et al., 1998; Choi and Lee,1999). Hence, the input substrate acetyl – CoA and other key substrates in the model (ATP, NADPH) were assumed at a constant level,A-CoA4 = 0.96 mM ATP = 0.12 mM NADPH4 = 0.61 mM. These substrates are part of pathways in the cell metabolism. It is essential that the cell keep them at a steady state concentration. Mevalonate is not known to be metabolize inside the cell and same is true for subsequent intermediate substrates in the pathway. Hence, any regulation in the organism has also been excluded in the model. The minor substrates such as CoA is not considered in the rate equation since it is recycled in the pathway and it is an important co-factor maintained in a steady concentration in the cell. The reported Km values of the genes reported in Table 1 were used. The tHMGR is substituted with a more stable enzyme HMG2. The kinetic data of other enzymes is from in-vitro assays reported in other publications.
Model specifications:
The maximum enzyme concentration was limited to 10mM, the limit was guessed using data available on protein concentration in E.Coli cytosol reported in5. The 10mM maximum enzyme concentration value is corroborated with data available on high copy number of oriCs in available plasmids and transcription rates [Dennis and Bremer 1996], so that the cell can achieve the necessary enzymes concentration in the growth phase.
Enzyme concentration = Transcription rate x Copy number of plasmid x Time in growth phase
The time of T = 100Hrs was used to solve the set of ordinary differential equations to capture sufficient time of cell in stationary phase. The optimization model shown in figure 3, takes given Time, initial concentration of substrates and enzyme kinetic model. The algorithm sets the concentration of all enzymes at initially 1nM. Through a directed evolutionary approach, the concentration of the enzymes in sequence beginning from E1, is modified by a multiplication factor and the new Limonene produced by the model solved at time (T) is calculated by the algorithm.
The best performing enzyme concentration of E1 is fed back to the model. In the subsequent turn the following enzyme E2 is multiplied from a set of factors and sent as input. Similarly, the best performing enzyme concentration is fed back to the model for the next enzymes (E3)‘s run.      The algorithm is run several times until the maximum limonene produced reaches a stable concentration with the enzyme concentrations optimized at after each run.
Results:
The concentration of limonene produced , using the kinetic model solved at T=100 Hours (Figure below) at different concentration of each enzyme, keeping the concentration of other enzymes at their best optimum, shows that HMGS, GPPS and LIMS are the rate limiting enzymes . The production is also sensitive to the concentration of IDI.  A comparison of rate of change in slope of the curves of HMGR, GPPS and LIMS showed that the limonene production was more sensitive to HMGS and LIMS compared to GPPS, at high concentration near 10mM.
Criticism:
• Substrates in the pathway such as limonene and HMG-CoA have been shown to be toxic1 to cell growth. Hence the over-expression of HMGS for large production will have toxic effect on cell growth. In which case, HMGS may not play the role of the rate limiting enzyme.
• Many enzymatic reactions are reversible in this pathway, however in this model all reactions have been considered to be irreversible. This fallback is probably responsible for the sensitivity of the IDI enzyme reported, since IPP -> DMAPP maintains an equilibrium balance using only a forward reaction.
Reference:
Piterra et. al, 2007. Balancing a heterologous mevalonate pathway for improved isoprenoid production in Escherichia coli. Metabolic Engineering. 2:193-207
Dunlop et. al, 2011. Engineering microbial biofuel tolerance and export using efflux pumps. Molecular Systems Biology. 7:487:487
Primak et. al, 2011. Characterization of a Feedback-Resistant Mevalonate Kinase from the Archaeon Methanosarcina mazei. Applied Environmental Microbiology. 77:7772-7778
Bennett et. al, 2009. Absolute metabolite concentrations and implied enzyme active site occupancy in Escherichia coli. Nat Chem Bio. 5:593-599
Ishihama et. al, 2009. Protein abundance profiling of the Escherichia coli cytosol. BMC Genomics. 9:102
Martin et. al, 2003. Engineering a mevalonate pathway in Escherichia coli for production of terpenoids. Nature Biotechnology. 21:7:796-802
Retrieved from " http://2012.igem.org/Team:Wisconsin-Madison/modeling "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
