<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NTU-Singapore_Basic_Part skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NTU-Singapore/Basic Part</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Template:Nav</TITLE></P><DIV class="bloc b-parallax bgc-white l-bloc " id="bloc-0"><DIV class="container bloc-sm"><NAV class="navbar row mini-nav"><DIV class="navbar-header"><BUTTON id="nav-toggle" type="button" class="ui-navbar-toggle navbar-toggle" data-toggle="collapse" data-target=".navbar-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse navbar-1 special-dropdown-nav"><UL class="site-navigation nav navbar-nav"><LI><A href="https://2018.igem.org/Team:NTU-Singapore" style="padding-top: 17px;color:black;">Home</A></LI><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle ltc-black" data-toggle="dropdown" aria-expanded="false" style="color:black;">Team</A></DIV></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Team">Our Team</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Attributions">Attributions</A></LI></UL></DIV><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle ltc-black" data-toggle="dropdown" aria-expanded="false" style="color:black;">Project</A></DIV></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/DNA">DNA Editor</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/RNA">RNA REPAIR</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Nanopore"> Nanopore Reader</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Demonstrate">Demonstrate</A></LI></NAV></DIV><LI><A href="https://2018.igem.org/Team:NTU-Singapore/InterLab" style="padding-top: 17px; color:black;">InterLab</A></LI><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle ltc-black" data-toggle="dropdown" aria-expanded="false" style="color:black;">Parts</A></DIV></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Basic_Part">New Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Improve">Improved Part</A></LI></DIV><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle ltc-black" data-toggle="dropdown" aria-expanded="false" style="color:black;">Human Practices</A></DIV></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Human_Practices">Overview</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Public_Engagement">Engagement &amp; Outreach</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/IHP">Integrated Human Practice</A></LI></DIV><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle ltc-black" data-toggle="dropdown" aria-expanded="false" style="color:black;">Awards</A></DIV></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Medals" class="ltc-black">Medals Criteria</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Best_HP" class="ltc-black">Best Human Practice</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Measurement" class="ltc-black">Best Measurement</A></LI></DIV><LI><A href="https://igem.org/2018_Judging_Form?id=2818" target="_blank" style="padding-top: 17px; color:black;">Judging Form</A></LI></DIV></DIV></DIV><DIV class="page-container" style="margin-top:0.6em;"><DIV class="bloc bgc-white bg-T--NTU-Singapore--ImprovedPart-Section-3 d-bloc" id="bloc-91"><DIV class="container bloc-lg"><DIV class="row"><DIV class="col-sm-12"><H1 class="tc-white  mg-md text-center"><STRONG>Validated New Parts</STRONG></H1></DIV></DIV></DIV></DIV><DIV class="bloc bgc-white page-title b-parallax none tc-black" id="bloc-92"><DIV class="container bloc-md"><DIV class="row"><DIV class="col-sm-11"><H2 class="mg-md  tc-black" style="margin-top: 0px;"> Our Submission
				</H2><P class=" text-left">
					This year we submitted two basic parts that are central to our REPAIR project. In the REPAIR project, we aimed to characterize and develop constructs that perform target-specific A-to-I base modification on RNA, and these two constructs are the RNA base editors, namely dCas13b-ADAR2<SPAN class="small-letter">DD </SPAN> (<A href="http://parts.igem.org/Part:BBa_K2818002" target="_blank">BBa_K2818002</A>) and dCas13d-ADAR2<SPAN class="small-letter">DD </SPAN> (<A href="http://parts.igem.org/Part:BBa_K2818001" target="_blank">BBa_K2818001</A>).
				</P><P class=" text-left" style="padding-top: 0.5em;">
					These two constructs have two essential domains. One part is the RNA-targeting scaffold from the nuclease-dead variant of the Type VI RNA-targeting CRISPR-associated protein 13, which directs the construct to its target site in the presence of a single RNA guide, while the other is hyperactive variant of adenosine deaminase acting on RNA (ADAR2<SPAN class="small-letter">DD</SPAN>) that catalyzes the hydrolytic deamination in the conversion from adenosine to inosine.</P><P class=" text-left" style="padding-top: 0.5em;">
					They are designed to target and edit specific adenosine bases on mRNA and their activities were proven and characterized in both of our luciferase reporter assay and the editing of endogenous mRNA sequences.</P></DIV></DIV></DIV></DIV><DIV class="bloc bgc-white tc-black" id="bloc-pos-2"><DIV class="container bloc-sm"><DIV class="row"><DIV class="col-sm-12"><H2 class="mg-md  tc-black" style="margin-top:0px; padding-top:0em; padding-left:1.4em;"> Luciferase Reporter Assay
				</H2><P class=" text-left" style="padding-top: 1em; margin-left:5em!important;">
					As described in our REPAIR project, both the dCas13b-ADAR2<SPAN class="small-letter">DD</SPAN> and dCas13d-ADAR2<SPAN class="small-letter">DD</SPAN> have demonstrated A-to-I base modification activities on exogeneous mRNAs in the reporter assay under appropriate RNA guides. With a guanosine in the codon of Rluc luciferase, luminescence can only be observed if the constructs successfully perform the base modification. The graph below shows the results for luminescence intensity after transfection of the necessary constructs. On all five editing sites, all constructs showed some, if not significant level of base editing activities, proving the functionality of our constructs.</P><H5 class="mg-md text-center" style="padding-top: 0.5em;"><STRONG>Figure 1. Editing rate of different RNA editors at different target positions on Rluc (n = 2)</STRONG></H5><P class=" text-left" style="padding-top: 1em; margin-left:5em!important;">
					Besides just proving that our constructs’ functionality, we further characterized important design parameters for optimal RNA base editing using dCas13d-ADAR2<SPAN class="small-letter">DD</SPAN>, like spacer length and the location of coverage of the target sequence. Below shows the different luminescence levels when the dCas13d construct is guided with guides of different parameters. <A href="https://2018.igem.org/Team:NTU-Singapore/RNA" target="_blank">More detailed analysis can be found on this page.</A></P><H5 class="mg-md text-center" style="padding-top: 0.5em;"><STRONG>Figure 2. Editing rate of different RNA editors with different spacer lengths and different guide mismatch distances. (n = 2)</STRONG><P class="text-center" style="padding-top:0.5em;">where dash line shows the non-targeting control</P></H5><H2 class="mg-md  tc-black" style="padding-top: 1em; padding-left:1.4em!important;"> Endogenous RNA Editing
				</H2><P class=" text-left" style="padding-top: 1em; margin-left:5em!important;">
					After the proof-of-concept with the successful A-to-I editing in our luciferase assay, we further demonstrated the functionality of our constructs when targeting endogenous mRNA transcripts of the PPIB and KRAS genes. Sanger sequencing showed that a significant portion of the targeted adenosine were called as guanosine for both the dCas13b-ADAR2<SPAN class="small-letter">DD</SPAN> and the dCas13d-ADAR2<SPAN class="small-letter">DD</SPAN>, which is the functional equivalence of inosine in base pairing. Figure below illustrates such an observation, suggesting that our constructs indeed perform as expected in mammalian cell lines.
				</P><H5 class="mg-md text-center" style="padding-top: 0.5em;"><STRONG>Figure 5. Editing rate of different RNA editors with different spacer lengths and different guide mismatch distances on mRNA transcripts of the endogenous PPIB and KRAS gene. (n = 2)</STRONG></H5></DIV></DIV></DIV></DIV><DIV class="bloc bgc-white page-title b-parallax none tc-black" id="bloc-94"><DIV class="container bloc-md"><DIV class="row"><DIV class="col-sm-11"><H2 class="mg-md  tc-black"> Best Basic Part
				</H2><P class=" text-left" style="padding-top: 1em;">
					In closing, not only did we validated the functionality of the parts we provided, but also have we characterized important parameters for distribution and the usage of our parts in future applications. More importantly, the fusion of ADAR2<SPAN class="small-letter">DD</SPAN> with dCas13d is an innovative and important construction, for its promised high editing efficiency and specificity at its impressively small size. It is by far the smallest Class 2 CRISPR-associated protein being profiled and one of the few proteins successfully packaged into rAAV without any rational truncation. When fused with ADAR2, a significant portion of the 32,000 known pathogenic mutations in humans can be targeted by this fusion protein and potentially yields beneficial tools for research or therapeutic uses. For its potential for exploration and well characterization, we nominate our dCas13d-ADAR2<SPAN class="small-letter">DD</SPAN> (<A href="http://parts.igem.org/Part:BBa_K2818001" target="_blank">BBa_K2818001</A>) to be the Best Basic Part.
				</P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>