<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NTU-Singapore_Improve skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NTU-Singapore/Improve</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Template:Nav</TITLE></P><DIV class="bloc b-parallax bgc-white l-bloc " id="bloc-0"><DIV class="container bloc-sm"><NAV class="navbar row mini-nav"><DIV class="navbar-header"><BUTTON id="nav-toggle" type="button" class="ui-navbar-toggle navbar-toggle" data-toggle="collapse" data-target=".navbar-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse navbar-1 special-dropdown-nav"><UL class="site-navigation nav navbar-nav"><LI><A href="https://2018.igem.org/Team:NTU-Singapore" style="padding-top: 17px;color:black;">Home</A></LI><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle ltc-black" data-toggle="dropdown" aria-expanded="false" style="color:black;">Team</A></DIV></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Team">Our Team</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Attributions">Attributions</A></LI></UL></DIV><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle ltc-black" data-toggle="dropdown" aria-expanded="false" style="color:black;">Project</A></DIV></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/DNA">DNA Editor</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/RNA">RNA REPAIR</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Nanopore"> Nanopore Reader</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Demonstrate">Demonstrate</A></LI></NAV></DIV><LI><A href="https://2018.igem.org/Team:NTU-Singapore/InterLab" style="padding-top: 17px; color:black;">InterLab</A></LI><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle ltc-black" data-toggle="dropdown" aria-expanded="false" style="color:black;">Parts</A></DIV></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Basic_Part">New Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Improve">Improved Part</A></LI></DIV><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle ltc-black" data-toggle="dropdown" aria-expanded="false" style="color:black;">Human Practices</A></DIV></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Human_Practices">Overview</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Public_Engagement">Engagement &amp; Outreach</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/IHP">Integrated Human Practice</A></LI></DIV><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle ltc-black" data-toggle="dropdown" aria-expanded="false" style="color:black;">Awards</A></DIV></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Medals" class="ltc-black">Medals Criteria</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Best_HP" class="ltc-black">Best Human Practice</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Measurement" class="ltc-black">Best Measurement</A></LI></DIV><LI><A href="https://igem.org/2018_Judging_Form?id=2818" target="_blank" style="padding-top: 17px; color:black;">Judging Form</A></LI></DIV></DIV></DIV><DIV class="page-container" style="margin-top:0.6em;"><DIV class="bloc bgc-white bg-T--NTU-Singapore--ImprovedPart-Banner d-bloc" id="bloc-54"><DIV class="container bloc-lg"><DIV class="row"><DIV class="col-sm-12"><H1 class="tc-white  mg-md text-center"><STRONG>Improved Part</STRONG></H1></DIV></DIV></DIV></DIV><DIV class="bloc bgc-white page-title b-parallax none tc-black d-bloc" id="bloc-55"><DIV class="container bloc-md"><DIV class="row"><DIV class="col-sm-11"><H2 class="mg-md  tc-black" style="margin-top: 0px;"> Nuclease-Dead Cas9 Truncation
				</H2><P class=" text-left">
					Previous studies have shown that a nuclease null version of the Cas9 protein (dCas9) is still capable of binding to sgRNA and subsequently be targeted to a genetic locus without cutting it. This dCas9 has been shown to effectively block transcription in bacteria, achieving gene knockdown. In mammals, gene knockdown requires fusion of repression proteins such as KRAB. Alternatively, the fusion of transcription activating proteins such as VPR, followed by targeting to promoters can achieve upregulation of downstream genes (Chavez et al., 2015). Such gene regulation can allow for treatment of diseases with abnormal gene expressions, like haploinsufficiency or cancer. </P><P class=" text-left" style="padding-top:0.5em;">
					However, one major barrier to clinical adoption is the requirement of in vivo delivery. Efficient in vivo delivery of Cas9 can be achieved with recombinant viruses, where the Cas9 gene is inserted into mammalian cells, then expressed to produce the protein. Recombinant adeno-associated virus (rAAV) is especially desirable, as it is minimally immunogenic, and does not integrate into the host genome. However, the rAAV has a packing limit of approximately 4.7kB, making it impossible to fit the Cas9 gene, gRNA expressing cassette as well as features such as a strong mammalian promoter. At around 4.2kB, the SpCas9 protein is the main culprit of the lack of space. Hence, in our previous projects, we designed the following assay to report gene activation and screen for the truncated dCas9 protein with significant binding efficiency.</P><H2 class="mg-md " style="padding-top:1em;"> Experimental Design
				</H2><P class=" text-left" style="padding-top:1em;">
					In our experiment, wild-type dCas9 and its truncated variants were fused with the tripartie VP64-p65-Rta (VPR) transcriptional activator, as described by Chavez et. al. (2015). Truncations and fusions were performed using Gibson assembly, which allows for scarless cloning required by fusion proteins construction. Guide RNAs for dCas9 were also cloned into expression vectors.</P><P class=" text-left" style="padding-top:0.5em;">
					The following three plasmids were then transfected into HEK293FT cells and allowed for expression for 48 hours.
				</P><P class=" text-left" style="padding-top: 1em; padding-left: 1em;">
					1. The dCas9+VPR plasmid contains dCas9/tCas9-VPR under CMV immearly enhancer and promoter for constitutive expression. Negative control only expresses VPR, without the dCas9. </P><P class=" text-left" style="padding-top: 1em; padding-left: 1em;">
					2. The sgRNA plasmid contains U6 promoter expressing spacer and gRNA. Constitutive mCherry fluorescence protein by EF1a promoter serves for gating of successfully transfected cells (Addgene <U><A class="ltc-black" href="https://www.addgene.org/65777/" target="_blank" style="color:darkblue; padding-right:0px;">#65777</A></U>).</P><P class=" text-left" style="padding-top: 1em; padding-left: 1em;">
					3. Reporter plasmid contains a minimal TREtight promoter preceded by 2 replicated protospacer to be targeted by gRNA for the dCas/tCas9-VPR to bind to. Downstream ZSGreen fluorescent protein gene serves to quantitatively report gene expression level. Levels of gene activation were then measured by flow cytometry and quantitative PCR.<U><A class="ltc-black" href="https://static.igem.org/mediawiki/2018/c/c1/T--NTU-Singapore--NTU-Protocol-Improvement.pdf" target="_blank" style="color:darkblue;">A complete protocol can be found here.</A></U></P><H4 class="mg-md text-center" class=" text-left" style="padding-top: 1em; font-size:0.8em;"><STRONG>Figure 1. Experimental design to measure level of gene activation.</STRONG></H4></DIV></DIV></DIV></DIV><DIV class="bloc bgc-white bg-T--NTU-Singapore--ImprovedPart-Section-3 d-bloc" id="bloc-56"><DIV class="container bloc-lg"><DIV class="row"><DIV class="col-sm-12"><H1 class="mg-md tc-white vanishIn animated"><STRONG style="font-family:Montserrat; font-weight:bold;">Results for DNA Binding Proteins</STRONG></H1></DIV></DIV></DIV></DIV><DIV class="bloc bgc-white page-title b-parallax none tc-black d-bloc" id="bloc-57"><DIV class="container bloc-md"><DIV class="row"><DIV class="col-sm-11"><P style="padding-top:1em;">
					We have previously shown that the dCas9 can be truncated in various domains and subdomains with a corresponding decrease in functionality (iGEM Team: NTU-SINGAPORE, 2016). We also showed that the decrease in functionality is due to drop in target DNA binding affinity, and can be partially recovered with site-specific DNA mutations to improve its affinity to DNA backbone (iGEM Team: NTU_SINGAPORE, 2017). In our previous work, we showed that DNA proximal amino acids in our tCas9 protein could be mutated to cationic arginine or lysine to improve DNA binding affinity. A key roadblock with this approach is that there is an upper limit in the number of amino acids identified to be close to the DNA backbone and hence the degree by which we can improve our tCas9. Thus, the fusion of DNA binding peptides to augment DNA binding affinity could potentially bypass this issue.</P><H4 class="mg-md text-center" style="padding-top:1em;font-size:0.8em;"><STRONG> Figure 2. Truncation of WT-dCas9 in Rec2, HNH and RuvCIII-2 subdomains result in ∆3ple.</STRONG><SPAN class="text-center" style="font-size: 0.8em;line-height:150%; padding-top:0.5em;">Improved ∆3ple 5_6_10 has 3 DNA proximal mutations that improved DNA binding affinity.</SPAN></H4><P style="padding-top:1.5em;">
					This year, we hypothesized that <STRONG>DNA binding affinity</STRONG> of our enhanced truncated-Cas9 (tCas9) can be further improved upon <STRONG>by fusions of DNA binding proteins and peptides.</STRONG> The specific Cas9 variant being improved upon is ∆RuvCIII-2 ∆HNH ∆REC2 Sp-dCas9 (<U><A class="ltc-black" href="http://parts.igem.org/Part:BBa_K2316000" target="_blank" style="color:darkblue;">BBa_K2316000</A></U>), henceforth referred to as ∆3ple. ∆3ple has truncations on the ∆HNH, ∆REC2 domains and the ∆RuvCIII-2 subdomain, as previously described (iGEM Team:NTU_SINGAPORE, 2017). </P><P style="padding-top:0.5em;">
					Archaeal DNA packaging protein sso7d is a non-specific double-stranded DNA (dsDNA) binding protein. It has been shown that sso7d fused to Taq polymerase improves the rate of extension due to less probability of complete dissociation from DNA (Wang et al., 2004). Thus, a similar approach could improve tCas9 DNA dwell time, increasing fusion protein activity. </P><P style="padding-top:0.5em;">
					A total of 22 identified proteins are commercially synthesized for human codon optimization, then fused to the N-terminus (nte) of ∆3ple via a short GGS linker (Figure 3). Not all fusions led to improved gene activation, with many having unexpected deleterious effect. </P><H4 class="mg-md text-center" style="padding-top:1em;font-size:0.8em;"><STRONG>Figure 3. Reporter gene activation with DNA binding proteins (DBDs) fused to nte of ∆3ple.</STRONG></H4><P style="padding-top:2em;">
					Next, we considered the possibility of better performance when the DBDs are fused to other loci. The set of 3ple_DBD with the best performing DBD B are fused to the C-terminus or other loci that are close to the DNA backbone. In the crystal structure visualization, two of the domains truncations in ∆3ple (∆HNH and ∆Rec2) were close to the DNA backbone (Figure 4). Thus, fusions were performed on these positions, with GGGS linkers. </P><H4 class="mg-md text-center" style="padding-top:1.5em; font-size:0.8em;"><STRONG>Figure 4. Original position of domains HNH (in violet) and Rec2 (in dark grey) in WT-SpCas9.</STRONG></H4><H4 class="mg-md text-center" style="padding-top:1em;font-size:0.8em;"><STRONG>Figure 5. DBD fusion on various positions of ∆3ple. </STRONG><P style="padding-top:0.5em;">Underscore is for type of DBD. Fusion position indicated with hyphen (i.e. Truncation _ site – fused DBD)</P></H4><P style="padding-top:2em;">
					Reporter gene activation consistently suggests that fusion to the HNH position and nte position is ideal, with general deleterious effect in Rec2 and c-terminus position (Figure 4). DBD B remains the best DNA binding peptide that can augment the DNA binding affinity of ∆3ple. </P><P style="padding-top:0.5em;">
					However, one concern with DNA binding proteins is their large size. Despite the over 70% improvement in reporter gene activation results by the DBD fusion, this came at the cost of increasing the size of our truncated dCas9 by an unacceptable margin, undoing the truncation that was performed. </P></DIV></DIV></DIV></DIV><DIV class="bloc bgc-white bg-T--NTU-Singapore--ImprovedPart-Section-1 d-bloc" id="bloc-58"><DIV class="container bloc-lg"><DIV class="row"><DIV class="col-sm-12"><H1 class="mg-md tc-white vanishIn animated"><STRONG style="font-family:Montserrat; font-weight:bold;">Results for DNA Binding Peptides</STRONG></H1></DIV></DIV></DIV></DIV><DIV class="bloc bgc-white page-title b-parallax none tc-black" id="bloc-59"><DIV class="container bloc-md"><DIV class="row"><DIV class="col-sm-11"><P style="padding-top:1em;">
					An alternative approach is to look for even smaller fusions to augment DNA binding affinity. To that end, we searched for short DNA binding peptides from literature. Peptides and protein subdomains identified were generally tested in original literature as free peptides for affinity characterizations (Alam et al., 2000, Sugimoto, 2001). </P><P style="padding-top:0.5em;">
					5 DNA binding peptides (DBP) were identified and fused to the N-terminus of ∆3ple using GGS linker (Figure 6). It was found that the pK and pR short cationic peptides performed well, with pK being the best. Thus, pK DBP fusion at the truncation positions with GGGS linkers is studied (Figure 7). Once again, similar to results from DBD fusion experiment, HNH position fusion appears to be the best. However, when pK is fused to multiple positions such as both N-terminus and HNH position, deleterious effects are observed. </P><H4 class="mg-md text-center" style="padding-top:1.5em;font-size:0.8em;"><STRONG>Figure 6. Reporter activation results for ∆3ple nte fusions of DBPs. </STRONG></H4><H4 class="mg-md text-center" style="padding-top:0em;font-size:0.8em;"><STRONG>Figure 7. pK fusions at truncated position.</STRONG></H4><P style="padding-top:2em;">
					Nest, endogenous gene activation is performed using the ∆3ple with pK fused to various positions. 2 genes were tested for verification and replication purposes. The 3 genes were also selected for varying response to our tCas9 improvement, with MIAT responding best to improvements followed by ASCL1 and TTN. Results generally reflect reporter gene activation (Figure 7). Interestingly, the poor performance of Rec2-pK fusion is apparent here, confirming what was observed in DBD fusion experiments. MIAT results also suggested that multiple pK fusion may still work, although further replicates of data are required. </P><H4 class="mg-md text-center" style="padding-top:1em;font-size:0.8em;"><STRONG>Figure 8. Fusion of pK to the original position of HNH results in best reporter and endogenous gene activation.</STRONG></H4></DIV></DIV></DIV></DIV><DIV class="bloc bgc-white bg-T--NTU-Singapore--ImprovedPart-Section-4 d-bloc" id="bloc-60"><DIV class="container bloc-lg"><DIV class="row"><DIV class="col-sm-12"><H1 class="mg-md tc-white vanishIn animated"><STRONG style="font-family:Montserrat; font-weight:bold; font-size:34px;">Combining Rational Mutations with DNA Binding Peptides</STRONG></H1></DIV></DIV></DIV></DIV><DIV class="bloc bgc-white page-title b-parallax none tc-black" id="bloc-bdp"><DIV class="container bloc-md"><DIV class="row"><DIV class="col-sm-11"><P style="padding-top:1em;">
					Improving upon our previously submitted part ∆RuvCIII-2 ∆HNH ∆REC2 Sp-dCas9 Enhanced (<U><A class="ltc-black" href="http://parts.igem.org/Part:BBa_K2316001" target="_blank" style="color:darkblue;">BBa_K2316001</A></U>), also referred to as ∆3ple 5_6_10, mutation at the position 15 is found to produce the best performing variant. Further compounding of mutations does not appear to improve ∆3ple 5_6_10_15 (Figure 9). This highlights the aforementioned roadblock with this approach. Thus,  identified fusions with DNA binding peptides may potentially solve this issue. </P><H4 class="mg-md text-center" style="padding-top:1em;font-size:0.8em;"><STRONG>Figure 9. Further compounding of mutations upon BBa_K2316001. </STRONG></H4><P style="padding-top:2em;"> 
					Fusion of pK to various positions was performed on ∆3ple 5_6_10_15. Although improvements were not apparent in reporter gene activation, it can be seen in endogenous gene activation, with pK fusion to the HNH position consistently improving gene activation in all 3 endogenous genes tested. Thus, this ∆RuvCIII-2 ∆HNH ∆REC2 4QC_HNHpK Sp-dCas9 is henceforth submitted this year as Part:<U><A class="ltc-black" href="http://parts.igem.org/Part:BBa_K2818000" target="_blank" style="color:darkblue;">BBa_K2818000</A></U>. </P><H4 class="mg-md text-center" style="padding-top:1em;font-size:0.8em;"><STRONG>Figure 10. Reporter gene activation by pK fused to different positions on ∆3ple 5_6_10_15. </STRONG></H4><H4 class="mg-md text-center" style="padding-top:1em;font-size:0.8em;"><STRONG>Figure 11. Combination of both approaches – improved and augmented DNA binding affinity, </STRONG> Resulted in best performing “∆RuvCIII-2 ∆HNH ∆REC2 4QC_HNHpK” Sp-dCas9 variant. 
				</H4></DIV></DIV></DIV></DIV><DIV class="bloc bgc-white page-title b-parallax none tc-black" id="bloc-62"><DIV class="container bloc-sm"><DIV class="row"><DIV class="col-sm-11"><H2 class="mg-md  tc-black"> Conclusion
				</H2><P class=" text-left" style="padding-top:1em;">
					Our project demonstrates additional methods towards rational improvements upon a rationally truncated SpCas9 to recover lost DNA binding affinity. In addition to cationic mutations to rationally identified positions on the Cas9 protein, we demonstrated that it is possible to directly augment the DNA binding affinity of our enhanced tCas9 by site sensitive fusion of DNA binding peptides. This overcomes the upper limit of possible mutations to improve DNA binding affinity. Fusion of DNA binding peptides was preferred over DBDs as it results in a negligible increase in the size of our enhanced tCas9. Additional work is required to build upon this study in order to achieve a therapeutically useful tCas9 small enough for rAAV delivery alongside fusion proteins. </P><H2 class="mg-md " style="padding-top:1em;"> References</H2><P style="padding-top:1em;">
					1. Alam, R., Maeda, M., &amp; Sasaki, S. (2000). DNA-Binding Peptides Searched from the Solid-Phase Combinatorial Library with the Use of the Magnetic Beads Attaching the Target Duplex DNA, 8, 465–473.2. Jinek, M., Chylinski, K., Fonfara, I., Hauer, M., Doudna, J. A., &amp; Charpentier, E. (2012). A Programmable Dual-RNA – Guided, 337(August), 816–822.3. Kosicki, M., Tomberg, K., &amp; Bradley, A. (2018). Repair of double-strand breaks induced by CRISPR–Cas9 leads to large deletions and complex rearrangements. Nature Biotechnology. doi: 10.1038/nbt.41924. Chavez, A., Scheiman, J., Vora, S., Pruitt, B. W., Tuttle, M., P R Iyer, E., … Church, G. M. (2015). Highly efficient Cas9-mediated transcriptional programming. Nat Methods, 12(4), 326–328. 5. Sugimoto N. (2001). DNA Recognition of a 24-mer Peptide Derived from RecA Protein, 416–424.6. Wang, Y. (2004). A novel strategy to engineer DNA polymerases for enhanced processivity and improved performance in vitro. Nucleic Acids Research, 32(3), pp.1197-1207.</P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>