<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NTU-Singapore_RNA skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NTU-Singapore/RNA</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Template:Nav</TITLE></P><DIV class="bloc b-parallax bgc-white l-bloc " id="bloc-0"><DIV class="container bloc-sm"><NAV class="navbar row mini-nav"><DIV class="navbar-header"><BUTTON id="nav-toggle" type="button" class="ui-navbar-toggle navbar-toggle" data-toggle="collapse" data-target=".navbar-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse navbar-1 special-dropdown-nav"><UL class="site-navigation nav navbar-nav"><LI><A href="https://2018.igem.org/Team:NTU-Singapore" style="padding-top: 17px;color:black;">Home</A></LI><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle ltc-black" data-toggle="dropdown" aria-expanded="false" style="color:black;">Team</A></DIV></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Team">Our Team</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Attributions">Attributions</A></LI></UL></DIV><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle ltc-black" data-toggle="dropdown" aria-expanded="false" style="color:black;">Project</A></DIV></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/DNA">DNA Editor</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/RNA">RNA REPAIR</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Nanopore"> Nanopore Reader</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Demonstrate">Demonstrate</A></LI></NAV></DIV><LI><A href="https://2018.igem.org/Team:NTU-Singapore/InterLab" style="padding-top: 17px; color:black;">InterLab</A></LI><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle ltc-black" data-toggle="dropdown" aria-expanded="false" style="color:black;">Parts</A></DIV></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Basic_Part">New Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Improve">Improved Part</A></LI></DIV><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle ltc-black" data-toggle="dropdown" aria-expanded="false" style="color:black;">Human Practices</A></DIV></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Human_Practices">Overview</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Public_Engagement">Engagement &amp; Outreach</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/IHP">Integrated Human Practice</A></LI></DIV><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle ltc-black" data-toggle="dropdown" aria-expanded="false" style="color:black;">Awards</A></DIV></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Medals" class="ltc-black">Medals Criteria</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Best_HP" class="ltc-black">Best Human Practice</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Measurement" class="ltc-black">Best Measurement</A></LI></DIV><LI><A href="https://igem.org/2018_Judging_Form?id=2818" target="_blank" style="padding-top: 17px; color:black;">Judging Form</A></LI></DIV></DIV></DIV><DIV class="page-container" style="margin-top:0.6em;"><DIV class="bloc bgc-white bg-T--NTU-Singapore--Bkgd d-bloc" id="bloc-70"><DIV class="container bloc-lg"><DIV class="row"><DIV class="col-sm-12"><H1 class="tc-white  mg-md text-center"><STRONG>RNA Base-Editor</STRONG></H1></DIV></DIV></DIV></DIV><DIV class="bloc bgc-white page-title b-parallax none tc-black " id="bloc-71"><DIV class="container bloc-md"><DIV class="row"><DIV class="col-sm-11"><H2 class="mg-md  tc-black" style="margin-top: 0px;"> Motivation for RNA Editing
				</H2><P class=" text-left">
					From the results of our human practice, we have seen a great value in RNA editing. First, with the cautious attitude towards gene editing in all three countries we surveyed, many incline to the reversibility of RNA editing, considering it being safer. Also, professional views from doctors reveal that transcriptome analysis and the lack of better RNA modifying protein scaffold contributes to the high cost of RNA editing. All of our discussions with stakeholders indicate that greater effort should be put to develop RNA editing, while in reality, it is lagging behind its DNA editing counterpart. Hence, we are motivated to develop programmable mRNA-editing analogous to our DNA base-editors as well as realizing cheaper transcriptome analysis by developing better RNA sequencing techniques.</P><P class=" text-left" style="padding-top:0.5em;">
					Our DNA base-editors perform two types of base changes. One is the C-to-T editing by the enhanced APOBEC1 protein (BE3) fused with Cas9 protein, while the other is the A-T to G-C base pair change by an evolved TadA*-dCas9 fusion protein. The former already has an obvious solution. The APOBEC1 cytidine deaminase in our DNA base-editor was endogenously used and engineered for cytosine-to-uracil editing on mRNA (Petersen-Mahrt, S. K.,2003). Hence, with an RNA-targeting partner, it should naturally induce an equivalent change on mRNA like a C-to-T base change on DNA. Actually, Team NUS-Singapore-Sci is working on such an RNA-targeting C-to-U editor. Hence, we decided to focus our effort on the latter, inducing an equivalent change on mRNA as if an A-to-G base change occurred on DNA.
				</P></DIV></DIV></DIV></DIV><DIV class="bloc bgc-white bg-T--NTU-Singapore--R-Banner1 d-bloc" id="bloc-72"><DIV class="container bloc-lg"><DIV class="row"><DIV class="col-sm-12"><H1 class="mg-md tc-white vanishIn animated"><STRONG style="padding-left:1.8em; font-family:Montserrat;">Theoretical Background</STRONG></H1></DIV></DIV></DIV></DIV><DIV class="bloc bgc-white page-title b-parallax none tc-black d-bloc" id="bloc-73"><DIV class="container bloc-sm"><DIV class="row"><DIV class="col-sm-11"><H2 class="mg-md " style="padding-top:1em;"> An Answer through Inosine
				</H2><P style="padding-top:1em;">
					While direct A-to-G base change on mRNA is much more difficult, one way to bypass such difficulty is to convert adenosine to inosine, which can later be converted to guanosine or simply allows for pairing with a cytosine. Such an A-to-I editing can be achieved through the adenosine deaminase acting on RNA (ADAR) enzyme family, which mediates endogenous A-to-I modification on mRNA by hydrolytic deamination. </P><P style="padding-top:0.5em;">
					Human has two ADAR orthologues, ADAR1, which targets repetitive regions and ADAR2 which targets non-repetitive coding regions (Tan. et. al., 2017). The latter is a great candidate for targeted A-to-I editing on mRNA. However, as ADAR proteins have preferred motifs on their editing target, a hyperactive mutant of ADAR2 (ADAR2<SPAN class="small-letter">DD</SPAN>), which has its glutamic acid at position 488 replaced by a glutamine (E488Q) (Kuttan. et. al., 2012), is used in our experiment. It has looser target stringency and a higher level of on-target activities.</P><H2 class="mg-md " style="padding-top:1em;"> RNA-Targeting Cas13 Proteins
				</H2><P style="padding-top:1em;">
					After the choice of our catalytic domain, we now need to choose an RNA-targeting partner. Previous literature has attempted to engineer ADAR to directly target with RNA guides (Montiel-González, 2016), and such a hybridization-based recognition does not guarantee high specificity and high editing efficiency. Hence, we decided to use a protein partner to assist the target recognition. The recently profiled and characterized Cas13 protein family is our choice.
				</P><P style="padding-top:0.5em;">
					Cas13 proteins are a family of type IV CRISPR-associated RNA-guided ribonucleases (RNase), which mediate precisely guided mRNA cleavage. These proteins have two higher eukaryotes and prokaryotes nucleotide-binding (HEPN) and RNase domains, and prefer but do not require protospacer flanking sites (PFS) (Cox. et. al., 2017). As they are guided by a single guide RNA to target RNA sequences, their programmable nature allows for targeting specific sites on mRNA and their catalytically inactive versions by mutating conserved catalytic residues in the HEPN domains, dCas13, are then good candidates to fuse with our ADAR proteins to perform site-specific A-to-I editing on mRNA.
				</P><H2 class="mg-md " style="padding-top:1em;"> Our Two Choices
				</H2><P style="padding-top:1em;">
					We identified two families of Cas13 proteins to investigate, namely the Cas13b and the Cas13d family. For Cas13b, Feng et. al. (2017) have demonstrated its consistently robust knockdown of mRNA in mammalian cell and engineered a dCas13b-ADAR2<SPAN class="small-letter">DD</SPAN> fusion protein to achieve effective site-specific A-to-I editing on mRNA, called the REPAIR (RNA Editing for Programmable A to I Replacement) system. We find their work a great inspiration for our project and we decided to characterize this construct further. </P><P style="padding-top:0.5em;margin-bottom:2em!important;">
					With a basis of comparison established with the REPAIR system, we were also curious about the recently identified Cas13d system (Konermann. et. Al, 2018). With the average size of just 930 amino acids, it is the smallest Class 2 CRISPR effector ever being characterized in mammalian cells. Despite its small size, one nuclease-dead variant derived from Ruminococcus flavefaciens XPD3002 (also known as CasRx)  has demonstrated alternative splicing modulation in vivo with high efficiency and specificity. Hence, we believe it would be interesting and meaningful to compare its A-to-I editing activities when Cas13d is fused with the same ADAR2<SPAN class="small-letter">DD</SPAN> domain with that of REPAIR.</P></DIV></DIV></DIV></DIV><DIV class="bloc bgc-white bg-T--NTU-Singapore--ABE-Bkgd d-bloc" id="bloc-74"><DIV class="container bloc-lg"><DIV class="row"><DIV class="col-sm-12"><H1 class="mg-md tc-white vanishIn animated"><STRONG style="padding-left:2em; font-family:Montserrat;">Experimental Design</STRONG></H1></DIV></DIV></DIV></DIV><DIV class="bloc bgc-white page-title b-parallax none tc-black" id="bloc-75"><DIV class="container bloc-sm"><DIV class="row"><DIV class="col-sm-11"><P style="padding-top:2em;">Based on Zhang et. al., 2017, we have developed two methods to characterize the functionality and important parameters in designing its guide RNA in mediating A-to-I editing activities, namely by a luciferase reporter assay and sequencing of mRNA of the endogenous target. In both of our experiments, plasmids coding for dPspCas13b-ADAR2<SPAN class="small-letter">DD</SPAN> and dCas13d-ADAR2<SPAN class="small-letter">DD</SPAN> fusion proteins and different RNA guides to investigate the different targeting parameters were cloned into mammalian expression vectors and transfected into HEK293FT cells. Other important sequences for their functions like the nuclear export signals (NES) were also cloned together with the construct.
				</P><H2 class="mg-md " style="padding-top:1em;"> Luciferase Reporter Assay
				</H2><P style="padding-top:1em;">
					In the luciferase reporter assay, plasmid coding for a modified <I><STRONG>Renilla</STRONG></I> luciferase was constructed, where a guanosine is replaced by an adenosine at the codon of a key residue, resulting in a nonsense mutation. As such, A-to I editing activities on the mRNA transcript by the dCas-ADAR2<SPAN class="small-letter">DD</SPAN> constructs will functionally restore the sequence and restores the luciferase protein back to the wildtype and allow for the quantification of editing activity by the Rluc luminescence. In our experiment, two parameters, namely spacer length and regions of coverage on the target were characterized in mediating A-to-I RNA editing.</P><P style="padding-top:1em;padding-left:1em;">•  Spacer length is the region of guide with homology to the target.
				</P><P style="padding-top:0.5em;padding-left:1em;">•  Regions of target coverage is measured by guide mismatch distance, that is the distance from the 3’-end of the spacer to the base targeting the adenosine to be edited.
				</P><P style="padding-top:1em;">
					In an experiment, the plasmid coding for mRNA target, plasmid coding for the dCas-ADAR and plasmid coding for guide RNA are transfected into HEK293FT cells. After 48 hours of transfection, media with the expressed luciferase is harvested and its luminescence was analysed and compared to that of the sample with non-targeting guide, marked as negative. Difference between the two would tell the A-to-I editing efficiency. </P><H5 class="mg-md text-center" style="padding-top:0.5em;"><STRONG>Figure 1. Experimental design of luciferase assay</STRONG></H5><H2 class="mg-md " style="padding-top:1em;"> Endogenous mRNA Targeting
				</H2><P style="padding-top:1em;">
					With the parameters obtained from the luciferase reporter assay, we further characterized the A-to-I editing activities of the dCas-ADAR2<SPAN class="small-letter">DD</SPAN> constructs on endogenous mRNA. In such an experiment, plasmids coding for dPspCas13b-ADAR2<SPAN class="small-letter">DD</SPAN> and dCas13d-ADAR2<SPAN class="small-letter">DD</SPAN> fusion proteins were transfected into HEK293FT cells, together with different guide RNAs targeting endogenous PPIB and KRAS mRNA transcripts. After 48 hours of transfection, transcriptome of the cells was extracted and the target regions were amplified for Sanger sequencing. Fractions of the adenosine being called as guanosine and therefore being edited can then report for the on-target efficiency of A-to-I editing. Different guides were used to investigate the activities with different spacer lengths and guide mismatch locations.</P><H5 class="mg-md text-center" style="padding-top:0.5em;padding-bottom:4em;"><STRONG>Figure 2. Experimental design in endogenous mRNA targeting</STRONG></H5></DIV></DIV></DIV></DIV><DIV class="bloc bgc-white bg-T--NTU-Singapore--R-Banner2 d-bloc" id="bloc-76"><DIV class="container bloc-lg"><DIV class="row"><DIV class="col-sm-12"><H1 class="mg-md vanishIn animated"><STRONG style="padding-left:2em;font-family:Montserrat; color:white;">Results and Discussions</STRONG></H1></DIV></DIV></DIV></DIV><DIV class="bloc bgc-white page-title b-parallax none tc-black bg-T--NTU-Singapore--APOBEC-Method-1 d-bloc" id="bloc-77"><DIV class="container bloc-md"><DIV class="row"><DIV class="col-sm-11"><H2 class="mg-sm "> Luciferase Reporter Assay
				</H2><P style="padding-top:1em;">
					In the luciferase experiment, we first evaluated the A-to-I editing activities of different RNA editors at different target positions on the Rluc mRNA. The aforementioned nonsense mutation of G to A was performed and tested at five tryptophan residues, at position 60, 104, 121, 153 and 219 respectively. Figure 3 below shows the luminescence levels after the restoration of Rluc sequence with different editors at different positions.</P><H5 class="mg-md text-center"><STRONG style="padding-top:0.5em;">Figure 3. Editing rate of different RNA editors at different target positions (n = 2)</STRONG></H5><P style="padding-top:1em;">
					From the results, we can observe that except for REPAIRv2, other dCas-ADAR2<SPAN class="small-letter">DD</SPAN> constructs showed significant A-to-I editing activities on the target and showed different target preferences from dCas13b to dCas13d. For Cas13d, editing activities on Rluc W153X is particularly significant. Therefore, it is selected as the target position to investigate the effect of guide length and guide mismatch distance on the A-to-I editing activities. </P><DIV class="row"><DIV class="col-sm-6"><P style="padding-top:0.5em;">
							Figure 4 shows the luminescence levels after the restoration of Rluc sequence by Cas13d using guides at different lengths and with different guide-target mismatch distance. The horizontal axis shows the mismatch distance and the number after items in the legend indicates different spacer length. Table 1 summarised all the observations made.</P></DIV><DIV class="col-sm-6"><H5 class="mg-md text-center" style="padding-top:0.5em; font-size:0.7em;"><STRONG>Table 1. Summary of optimal spacer length and guide mismatch distances</STRONG></H5><DIV class="table-wrapper"><DIV class="table-table"><DIV class="table-row header"><DIV class="table-cell" style="text-align:center">
                   Spacer Length (bp)
                </DIV><DIV class="table-cell">
                    Min Mismatch Distance
                </DIV><DIV class="table-cell">
                    Max Mismatch Distance
                </DIV></DIV><DIV class="table-row"><DIV class="table-cell" data-title="Set" style="text-align:center">
                    30
                </DIV><DIV class="table-cell" data-title="Neg">
                    7              </DIV><DIV class="table-cell" data-title="Pos">
                    13
                </DIV></DIV><DIV class="table-row"><DIV class="table-cell" data-title="Set" style="text-align:center">
                    50
                </DIV><DIV class="table-cell" data-title="Neg">
                    13                </DIV><DIV class="table-cell" data-title="Pos">
                    37
                </DIV></DIV><DIV class="table-row"><DIV class="table-cell" data-title="Set" style="text-align:center">
                    70
                </DIV><DIV class="table-cell" data-title="Neg">
                    25              </DIV><DIV class="table-cell" data-title="Pos">
                    43
                </DIV></DIV></DIV></DIV></DIV></DIV><H5 class="mg-md text-center" style="padding-top:1em;"><STRONG>Figure 4. Editing rate of different RNA editors with different spacer lengths and different guide mismatch distances. (n = 2)</STRONG>where dash line shows the results for non-targeting control</H5><P style="padding-top:1em;">
						From the results, we were then able to design homology-based guides with appropriate spacer length and guide mismatch distances and evaluate the performance of A-to-I editing on target mRNA. </P><H2 class="mg-md " style="padding-top:1em;"> Endogenous mRNA Targeting
					</H2><P style="padding-top:1em;">
						In this part of the experiment, we used guides to target two different regions of the PPIB and KRAS mRNA. They were termed as guide 1 and 2 for PPIB and KRAS. Then, some of them will be given a suffix of X.Y, where X indicates the target length and Y indicates the guide mismatch distance. For example, KRAS-1-50.25 is the guide RNA targeting region 1 of KRAS with a spacer length of 50 base-pairs and a mismatch distance of 25 base-pairs. The following results were then obtained from Sanger sequencing. Editing rate is calculated as the area under the guanosine signal in the chromatograph over that of adenosine.</P><H5 class="mg-md text-center" style="padding-top:1em; line-height:100%"><STRONG>Figure 5. Editing rate of different RNA editors with different spacer lengths and different guide mismatch distances  on endogenous PPIB and KRAS mRNA. (n = 2)</STRONG></H5></DIV></DIV></DIV></DIV><DIV class="bloc bgc-white page-title b-parallax none tc-black" id="bloc-78"><DIV class="container bloc-md"><DIV class="row"><DIV class="col-sm-11"><H2 class="mg-md  tc-black" style="margin-top:0px"> Conclusion
				</H2><P class=" text-left" style="padding-top:1em;">
					Here we have demonstrated Type VI Cas13 proteins can mediate efficient A-to-I base editing on mRNA, for both exogeneous and endogenous transcripts. From both experiments we can conclude that while the optimized REPAIR enzyme showed higher A-to-I editing efficiency, unoptimized dCas13d-ADAR2<SPAN class="small-letter">DD</SPAN> constructs exhibited similar A-to-I editing activity level on mRNA on the PPIB loci. This shows great potential for the dCas13d-ADAR2<SPAN class="small-letter">DD</SPAN> construncts as it has significantly smaller size and there is still possibility for protein engineering and optimization.</P><H2 class="mg-md " style="padding-top:1em;"> References</H2><P style="padding-top:1em;">
					1. Harris, R. S., Petersen-Mahrt, S. K., &amp; Neuberger, M. S. (2002). RNA editing enzyme APOBEC1 and some of its homologs can act as DNA mutators. Molecular cell, 10(5), 1247-1253.2. Tan, M. H., Li, Q., Shanmugam, R., Piskol, R., Kohler, J., Young, A. N., ... &amp; Gupte, A. (2017). Dynamic landscape and regulation of RNA editing in mammals. Nature, 550(7675), 249.3. Montiel-González, M. F., Vallecillo-Viejo, I. C., &amp; Rosenthal, J. J. (2016). An efficient system for selectively altering genetic information within mRNAs. Nucleic acids research, 44(21), e157-e157. 4. Kuttan, A., &amp; Bass, B. L. (2012). Mechanistic insights into editing-site specificity of ADARs. Proceedings of the National Academy of Sciences, 109(48), E3295-E3304. 5. Abudayyeh, O. O., Gootenberg, J. S., Essletzbichler, P., Han, S., Joung, J., Belanto, J. J., ... &amp; Lander, E. S. (2017). RNA targeting with CRISPR–Cas13. Nature, 550(7675), 280. 6. Cox, D. B., Gootenberg, J. S., Abudayyeh, O. O., Franklin, B., Kellner, M. J., Joung, J., &amp; Zhang, F. (2017). RNA editing with CRISPR-Cas13. Science, 358(6366), 1019-1027. 7. Konermann, S., Lotfy, P., Brideau, N. J., Oki, J., Shokhirev, M. N., &amp; Hsu, P. D. (2018). Transcriptome engineering with RNA-targeting type VI-D CRISPR effectors. Cell, 173(3), 665-676.</P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>