<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NTU-Singapore_DNA skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NTU-Singapore/DNA</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Template:Nav</TITLE></P><DIV class="bloc b-parallax bgc-white l-bloc " id="bloc-0"><DIV class="container bloc-sm"><NAV class="navbar row mini-nav"><DIV class="navbar-header"><BUTTON id="nav-toggle" type="button" class="ui-navbar-toggle navbar-toggle" data-toggle="collapse" data-target=".navbar-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse navbar-1 special-dropdown-nav"><UL class="site-navigation nav navbar-nav"><LI><A href="https://2018.igem.org/Team:NTU-Singapore" style="padding-top: 17px;color:black;">Home</A></LI><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle ltc-black" data-toggle="dropdown" aria-expanded="false" style="color:black;">Team</A></DIV></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Team">Our Team</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Attributions">Attributions</A></LI></UL></DIV><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle ltc-black" data-toggle="dropdown" aria-expanded="false" style="color:black;">Project</A></DIV></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/DNA">DNA Editor</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/RNA">RNA REPAIR</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Nanopore"> Nanopore Reader</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Demonstrate">Demonstrate</A></LI></NAV></DIV><LI><A href="https://2018.igem.org/Team:NTU-Singapore/InterLab" style="padding-top: 17px; color:black;">InterLab</A></LI><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle ltc-black" data-toggle="dropdown" aria-expanded="false" style="color:black;">Parts</A></DIV></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Basic_Part">New Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Improve">Improved Part</A></LI></DIV><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle ltc-black" data-toggle="dropdown" aria-expanded="false" style="color:black;">Human Practices</A></DIV></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Human_Practices">Overview</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Public_Engagement">Engagement &amp; Outreach</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/IHP">Integrated Human Practice</A></LI></DIV><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle ltc-black" data-toggle="dropdown" aria-expanded="false" style="color:black;">Awards</A></DIV></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Medals" class="ltc-black">Medals Criteria</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Best_HP" class="ltc-black">Best Human Practice</A></LI><LI><A href="https://2018.igem.org/Team:NTU-Singapore/Measurement" class="ltc-black">Best Measurement</A></LI></DIV><LI><A href="https://igem.org/2018_Judging_Form?id=2818" target="_blank" style="padding-top: 17px; color:black;">Judging Form</A></LI></DIV></DIV></DIV><DIV class="page-container"><DIV class="bloc bgc-white bg-T--NTU-Singapore--DNA-Editor d-bloc" id="bloc-35"><DIV class="container bloc-lg"><DIV class="row"><DIV class="col-sm-12"><H1 class="tc-white  mg-md text-center"><STRONG>DNA Base-Editor</STRONG></H1></DIV></DIV></DIV></DIV><DIV class="bloc bgc-white page-title b-parallax none tc-black" id="bloc-36"><DIV class="container bloc-md"><DIV class="row"><DIV class="col-sm-11"><H2 class="mg-md  tc-black" style="margin-top: 0px;"> Type II CRISPR/Cas System
				</H2><P class=" text-left">
					The bacterial CRISPR/Cas system acts as a form of the adaptive immune system against foreign nucleic acids such as viruses and plasmids. The system is capable of non-self DNA identification, and acquires an excerpt of the non-self DNA – also known as spacers, storing it in the Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR) locus of the bacterial genome. Subsequently, a CRISPR-associated RNA (crRNA) with the spacer can be expressed and complexed with the CRISPR-associated nuclease (Cas), cleaving the target DNA matching the spacer. Non-self-identification is achieved by an invariant protospacer adjacent motif (PAM), that is found only on the targeted invading DNA, but not integrated into its CRISPR locus. </P><P class=" text-left" style="padding-top: 0.5em;">
					Various types of the CRISPR/Cas systems have been described, with the Type II system being the best studied and used, owing to its relative simplicity. In Type II system, the final Cas9 nuclease is a single, multi-functional protein, making it easy to express for genetic engineering purposes. As Cas proteins can only target sequences that have an associated PAM sequence, <I><STRONG>Streptococcus pyogenes</STRONG></I><STRONG>Cas9 (SpCas9)</STRONG> is especially attractive with its relaxed PAM requirement of NGG, compared to variants such as SaCas9 (NNGRRT). Further simplifying the process, it has been shown that the spacer containing crRNA can be fused to an invariant trans-crRNA (tracrRNA), resulting in a single guide RNA (gRNA). Thus, expression of the SpCas9 and a modifiable gRNA is sufficient to target cuts on genomes (Jinek et al., 2012).</P><P class=" text-left" style="padding-top: 0.5em;">
					One of the first and most well-researched uses of the programmable Cas-nuclease is genome engineering. A programmable nuclease can perform cuts on specific loci of the mammalian genome triggering DNA repair machinery that can be taken advantaged of to perform desired modification in the genome. This can potentially allow for the treatment of genetic diseases by correcting aberrant mutations. However, one of the main concerns with Cas9 mediated gene editing is potential off-target cutting. This can lead to indels in unintended loci. Severe aberrances such as large deletions and rearrangements have also recently been described (Kosicki et. Al, 2018). Thus, there is mounting interest in non-permanent solutions for the treatment of diseases such as gene regulation and RNA editing. </P><H2 class="mg-md " style="padding-top: 1em; padding-bottom: 0.17em;"> Cas9-Fusion Proteins
				</H2><P class="text-left" style="padding-top: 1em; padding-bottom: 2em;">

One of the most exciting applications of Cas9-fusion proteins is in base editing. Fusion of <U>RNA editing APOBEC1 cytidine deaminase</U>, which can also perform DNA editing, allowed for <STRONG>site programmable C to T editing</STRONG> (Komor et al. 2016). Fusion of an evolved <U> TadA adenine deaminase</U> allowed for <STRONG> A to G editing </STRONG> (Gaudelli et al. 2017). Unlike in HDR directed gene editing, base editors used a nickase cas9 (nCas9) that do not produce a double-strand break (DSB). Thus, it has comparatively higher editing frequency, with much lower undesired mutations such as indels. Base editors may represent a safer method for gene therapy, at least with regards to diseases treatable with single nucleotide changes possible with the current toolset.  

				</P></DIV></DIV></DIV></DIV><DIV class="bloc bgc-white bg-T--NTU-Singapore--APOBEC-Bkgd d-bloc" id="bloc-37"><DIV class="container bloc-lg"><DIV class="row"><DIV class="col-sm-12"><H1 class="mg-md tc-white vanishIn animated"><STRONG style="padding-left:2em;font-family:Montserrat; font-weight:900;">APOBEC C-To-T Editing</STRONG></H1></DIV></DIV></DIV></DIV><DIV class="bloc bgc-white page-title b-parallax none tc-black d-bloc" id="bloc-38"><DIV class="container bloc-md"><DIV class="row"><DIV class="col-sm-11"><H2 class="mg-md " style="padding-top:1em; padding-bottom: 0.17em;"> Design of Experiment
				</H2><P class="text-left" style="padding-top:1em;">

We have demonstrated the utility of our tCas9 in the context of VPR fusion for gene activation. Next, we seek to expand on the potential utility of our tCas9 with other fusion proteins. The APOBEC1 fusion as described by Komor et al. (2016) in 3rd generation BE3 stands at close to 700bp, fused to the N-terminus of a nCas9. A 250bp uracil-DNA glycosylase inhibitor (UGI) is also fused to the C-terminus to improve editing frequency. The resulting BE3 construct gene is over 5kB, greatly exceeding the packing limit of rAAV viral vector ideal for in-vivo delivery. Thus, there is value in getting tCas9 variants to work with base editors.
 
				</P><P class="text-left" style="padding-top:0.5em;">

We first tested a model tCas9 variant, ∆Rec3, with the BE2 fusion. The ∆Rec3 fusion was selected for this preliminary experiment for two reasons. First, the ∆Rec3 truncation has been shown to still maintain considerable DNA binding affinity, with over 50% of WT dCas-VPR gene activation in reporter assays (NTU Singapore iGEM, 2017). Second, the Rec3 domain has been shown to play a key proofreading step to allow for HNH cleavage. This is of concern as the high efficiency of both base editors are dependent on HNH nicking of the non-edited DNA strand, promoting repair to match the edited strand. 

				</P><H2 class="mg-md " style="padding-top:1em; padding-bottom: 0.17em;"> Results and Analysis</H2><P class="text-left" style="padding-top:0.5em; padding-bottom: 2em;">

Results indicate that there is a big drop in base editing efficiency when Rec3 has been truncated (Figure 1). This is likely due to a decrease in DNA binding affinity. 

				</P><H4 class="mg-md text-center" style="padding-top:0.5em; margin-top:0px;"><STRONG>Figure 1. ∆Rec3-BE2 compared against background (Empty) and WT-BE2. </STRONG></H4><P class="text-left" style="padding-top:1em;">
Two possible mechanisms for the drop in base editing efficiency of Rec3-BE is suggested,
				</P><P class="text-left" style="padding-top:0.5em; padding-left:1em;">
1) Impaired HNH nicking for non-target strand repair;
				</P><P class="text-left" style="padding-left: 1em;">
2) Lowered DNA binding affinity as a result of truncation destabilization.
				</P><P class="text-left" style="padding-top:1em;">
To delineate the effects of both mechanisms, 2 further experiments were performed. First Rec3 is tested in the context of BE2 and BE3, where BE3 has a functional HNH domain, and would normally lead to much higher base editing efficiency. Rec3-BE3 exhibit remarkably similar base editing efficiency as Rec3-BE2, suggesting completely impaired HNH nickase activity (Figure 2). 
				</P><H4 class="mg-md text-center" style="padding-top:1em; margin-top:0px;"><STRONG style="line-height:150%;">Figure 2. Base editing efficiencies of Rec3 in BE2 and BE3 platform.</STRONG><SPAN style="font-size:0.8em;">Results shown are only of the fifth and sixth strongly edited nucleotides for EMX1. </SPAN></H4><P class="text-left" style="padding-top:1em;">
Next, we looked at the performance of Rec2 truncation on BE3. We have previously shown that Rec2 truncation leads to similar gene activation performance as Rec3 truncation, at around 50% of WT. As seen in Figure 3, there is still an observable drop in editing efficiency when Rec3-BE3 is compared against Rec2-BE3. This confirms the impact on efficiency when HNH nickase activity is lost. 
				</P><P class="text-left" style="padding-top:0.5em;">
In addition Rec2-BE3 demonstrates considerably lowered gene activation – highlighting that loss of DNA binding affinity is an exceedingly important reason for drop in editing efficiency.
				</P><H4 class="mg-md text-center" style="padding-top:0.5em; margin-top:0px;"><STRONG style="line-height:150%;">Figure 3. Base editing efficiencies of Rec2 and Rec3 in BE3 platform.</STRONG><SPAN style="font-size:0.8em;">Results shown are only of the fifth and sixth strongly edited nucleotides for EMX1. </SPAN></H4><P class="text-left" style="padding-top:1em;">
We also tested other truncated variants of Cas9 protein. You can view the complete results <A class="class1" href="https://static.igem.org/mediawiki/2018/b/b0/T--NTU-Singapore--More-Results.pdf"> here. </A></P><H2 class="mg-md " style="padding-top:1em; padding-bottom: 0.17em;"> Summary of Findings</H2><P class="text-left" style="padding-top:1em;">
Results indicated the importance of maintaining HNH nickase activity for optimal base editing efficiency. The removal of Rec3 led to the loss of HNH nuclease activity in the context of base editors. In addition, Rec3 participates in interaction with target DNA:gRNA heteroduplex, and the loss of which probably further decreased binding affinity of tCas9. 
				</P><P class="text-left" style="padding-top:0.5em;">
Results also suggest that base editing is stricter than gene activation in requiring high affinity to the target DNA. This is not unexpected as base editing would likely require longer dwell time for the deamination reaction to be catalyzed. The improvements achieved with our ∆3ple in this year’s improved part project could potentially solve this issue, although HNH may have to be replaced to maintain nickase function. 
				</P><P class="text-left" style="padding-top:0.5em;">
Further improvements are also desired. To that end, we had also begun setting up a guided evolution platform to select for tCas9 with improved DNA binding. 
				</P></DIV></DIV></DIV></DIV><DIV class="bloc bgc-white bg-T--NTU-Singapore--ABE-Bkgd d-bloc" id="bloc-39"><DIV class="container bloc-lg"><DIV class="row"><DIV class="col-sm-12"><H1 class="mg-md tc-white vanishIn animated"><STRONG style="padding-left:2em; font-family:Montserrat; font-weight:900;">ABE Base Pair Change</STRONG></H1></DIV></DIV></DIV></DIV><DIV class="bloc bgc-white page-title b-parallax none tc-black d-bloc" id="bloc-40"><DIV class="container bloc-md" style="padding-bottom:1em;"><DIV class="row"><DIV class="col-sm-11"><H2 class="mg-md " style="padding-top:1em; padding-bottom: 0.17em;"> Experimental Design
				</H2><P class="text-left" style="padding-top:1em;">
					The adenine base editor (ABE) has been evolved from a transfer RNA adenosine deaminase (TadA) that is capable of converting A•T to G•C base pairs to treat diseases caused by single-base mutations in the genome such as sickle-cell anemia. The ABE can then be fused to a dCas9 to target a desired site to edit the DNA guiding by sgRNA. </P><P class="text-left" style="padding-top:0.5em;">
					However, overexpression of a protein can lead to toxicity issue, especially with the large size of dCas9. In order to have a tight control of gene regulation, we constructed two plasmids where TadA*-dCas9 is expressed by arabinose under pBAD promoter and sgRNA is expressed by IPTG under lac operator. The TadA*-dCas9 plasmid consists of chloramphenicol resistance while the sgRNA plasmid consists of a gentamycin resistance and a kanamycin resistance with a premature stop codon. </P><H4 class="mg-md text-center" style="padding-top:0.5em;"><STRONG>Figure 4. Two constructs to be transformed into E. coli</STRONG></H4><P class="text-left" style="padding-top:1em;">
					In order for the bacteria to survive on Chloramphenicol + Gentamycin + Kanamycin condition, the TadA*-dCas9 needs to restore the premature stop codon of Kanamycin. </P><P class="text-left" style="padding-top:0.5em;">
					We transformed bacteria BW25141 with these two plasmids and used 2 hours of LB recovery under four conditions: no induction, 100mM arabinose, 500μM IPTG, and 100mM arabinose + 500μM IPTG. The samples were then spread on a Chloramphenicol + Gentamycin plate for positive control and a Chloramphenicol + Gentamycin + Kanamycin plate for selection. </P><H2 class="mg-md " style="padding-top:1em; padding-bottom: 0.17em;"> Results and Analysis</H2><P class="text-left" style="padding-top:1em;">
					The results of the transformation are shown in figure below. Colonies grown on chloramphenicol+gentamycin plate show that the host cells contained TadA*-dCas9 and sgRNA plasmids, but colonies only grown on chloramphenicol +  gentamycin + kanamycin plate prove the tightness of regulation of both promoters. </P><H4 class="mg-md text-center" style="padding-top:1em;"><STRONG>Figure 5. Survival of colonies after transformation and induction</STRONG></H4><P class="text-left" style="padding-top:1.5em;">
					As seen from above, similar growth was observed on control plates without the selection pressure of Kanamycin, suggesting a similar transformation efficiency. However, in the presence of Kanamycin, only the experiment with both inducers added showed significant growth, suggesting that both constructs are necessary for bacteria to grow in Kanamycin and proving the editing activities of the TadA*-dCas9 construct. Such a growth can be more rigorously quantified as survival frequency, defined as the ratio of the number of colonies on the Chloramphenicol + Gentamycin + Kanamycin plate to that on the Chloramphenicol + Gentamycin plate. <A class="class1" href="https://2018.igem.org/Team:NTU-Singapore/Collaborations">We also collaborated with NUSGEM this year to validate our TadA*-dCas9 construct.</A></P></DIV></DIV></DIV></DIV><DIV class="bloc bgc-white page-title b-parallax none tc-black" id="bloc-41"><DIV class="container bloc-sm"><DIV class="row"><DIV class="col-sm-11"><H2 class="mg-md  tc-black" style="padding-bottom: 0.17em;"> Conclusion
				</H2><P class=" text-left" style="padding-top:1em;">
					From both experiments, we can conclude that the fusion protein between base editors and Cas9 indeed functioned as expected to perform either single base change or base pair change. However, the editing efficiency of these proteins remains to be further explored and optimized. It is also worth noting here that different approaches can be employed to quantify base editing activities. In the tCas9-BE3 experiment, direct sequencing was used to determine the percentage of edited DNA while in the TadA*-dCas9 experiment, a more intuitive reporting system of measuring colony formation was employed. This shows our creativity in designing our experiment, which we should continue to innovate and develop.</P><H2 class="mg-md" style="padding-top:1em; padding-bottom: 0.17em;"> References</H2><P class="text-left" style="padding-top:1em;">
					1. Chen, J. S., Dagdas, Y. S., Kleinstiver, B. P., Welch, M. M., Sousa, A. A., Harrington, L. B., … Doudna, J. A. (2017). Enhanced proofreading governs CRISPR-Cas9 targeting accuracy. Nature, 550(7676), 407–410. 2. Komor, A., Kim, Y., Packer, M., Zuris, J., &amp; Liu, D. (2016). Programmable editing of a target base in genomic DNA without double-stranded DNA cleavage. Nature, 533(7603), 420-424. doi: 10.1038/nature179463. Gaudelli, N., Komor, A., Rees, H., Packer, M., Badran, A., Bryson, D., &amp; Liu, D. (2017). Programmable base editing of A•T to G•C in genomic DNA without DNA cleavage. Nature, 551(7681), 464-471. doi: 10.1038/nature24644</P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>