Team:NTU-Singapore/Basic Part
Template:Nav
Toggle navigation
Home
Team
Our Team
Collaborations
Attributions
Project
DNA Editor
RNA REPAIR
Nanopore Reader
Demonstrate
InterLab
Parts
New Basic Parts
Improved Part
Human Practices
Overview
Engagement & Outreach
Integrated Human Practice
Awards
Medals Criteria
Best Human Practice
Best Measurement
Judging Form
Validated New Parts
 Our Submission
This year we submitted two basic parts that are central to our REPAIR project. In the REPAIR project, we aimed to characterize and develop constructs that perform target-specific A-to-I base modification on RNA, and these two constructs are the RNA base editors, namely dCas13b-ADAR2DD ( BBa_K2818002 ) and dCas13d-ADAR2DD ( BBa_K2818001 ).
These two constructs have two essential domains. One part is the RNA-targeting scaffold from the nuclease-dead variant of the Type VI RNA-targeting CRISPR-associated protein 13, which directs the construct to its target site in the presence of a single RNA guide, while the other is hyperactive variant of adenosine deaminase acting on RNA (ADAR2DD) that catalyzes the hydrolytic deamination in the conversion from adenosine to inosine.
They are designed to target and edit specific adenosine bases on mRNA and their activities were proven and characterized in both of our luciferase reporter assay and the editing of endogenous mRNA sequences.
 Luciferase Reporter Assay
As described in our REPAIR project, both the dCas13b-ADAR2DD and dCas13d-ADAR2DD have demonstrated A-to-I base modification activities on exogeneous mRNAs in the reporter assay under appropriate RNA guides. With a guanosine in the codon of Rluc luciferase, luminescence can only be observed if the constructs successfully perform the base modification. The graph below shows the results for luminescence intensity after transfection of the necessary constructs. On all five editing sites, all constructs showed some, if not significant level of base editing activities, proving the functionality of our constructs.
Figure 1. Editing rate of different RNA editors at different target positions on Rluc (n = 2)
Besides just proving that our constructs’ functionality, we further characterized important design parameters for optimal RNA base editing using dCas13d-ADAR2DD, like spacer length and the location of coverage of the target sequence. Below shows the different luminescence levels when the dCas13d construct is guided with guides of different parameters.  More detailed analysis can be found on this page.
Figure 2. Editing rate of different RNA editors with different spacer lengths and different guide mismatch distances. (n = 2)
where dash line shows the non-targeting control
 Endogenous RNA Editing
After the proof-of-concept with the successful A-to-I editing in our luciferase assay, we further demonstrated the functionality of our constructs when targeting endogenous mRNA transcripts of the PPIB and KRAS genes. Sanger sequencing showed that a significant portion of the targeted adenosine were called as guanosine for both the dCas13b-ADAR2DD and the dCas13d-ADAR2DD, which is the functional equivalence of inosine in base pairing. Figure below illustrates such an observation, suggesting that our constructs indeed perform as expected in mammalian cell lines.
Figure 5. Editing rate of different RNA editors with different spacer lengths and different guide mismatch distances
on mRNA transcripts of the endogenous PPIB and KRAS gene. (n = 2)
 Best Basic Part
In closing, not only did we validated the functionality of the parts we provided, but also have we characterized important parameters for distribution and the usage of our parts in future applications. More importantly, the fusion of ADAR2DD with dCas13d is an innovative and important construction, for its promised high editing efficiency and specificity at its impressively small size. It is by far the smallest Class 2 CRISPR-associated protein being profiled and one of the few proteins successfully packaged into rAAV without any rational truncation. When fused with ADAR2, a significant portion of the 32,000 known pathogenic mutations in humans can be targeted by this fusion protein and potentially yields beneficial tools for research or therapeutic uses. For its potential for exploration and well characterization, we nominate our dCas13d-ADAR2DD ( BBa_K2818001 ) to be the Best Basic Part.
