Team:NTU-Singapore/Collaborations
Template:Nav
Toggle navigation
Home
Team
Our Team
Collaborations
Attributions
Project
DNA Editor
RNA REPAIR
Nanopore Reader
Demonstrate
InterLab
Parts
New Basic Parts
Improved Part
Human Practices
Overview
Engagement & Outreach
Integrated Human Practice
Awards
Medals Criteria
Best Human Practice
Best Measurement
Judging Form
Collaboration
 Motivation
Difficulties are inevitable in a project, but help from our friends would always bring us to cross the finishing line. We were fortunate to have met many teams and built much friendship through iGEM. We helped each other in different aspects of our projects, and much of our effort bore meaningful fruits in our laboratory work and beyond. Here is the documentation of our precious memory and our expression of gratitude for all of their help.
UI-Indonesia
This year, we were glad that we could collaborate with team from University of Indonesia ( UI_Indonesia ). Our extensive collaboration ranges from wet lab to human practice.
 How the Story Began?
We helped UI_Indonesia to clone their HT constructs as they faced great difficulty. The construct is a fusion of native Escherichia coli Tar chemotaxis receptor and human HB-EGF receptor which in the purpose of diphtheria toxin can be recognized by bacteria. This HT was designed and ordered in a gBlock from Integrated DNA Technologies (IDT).
Initially, UI_Indonesia ordered the HT gBlock as one fragment with a size ~1900 bp. However, they were informed that there was a problem during the construction, so they would need to change it and came up with ordering two separate fragments with one restriction between 2 parts, namely HT1 and HT2.
They planned to combine these two fragments together into a vector by ligation. However, when they tried to amplify HT1 to insert it in into a vector, they found out that the amplified product of HT1 was not specific. The size for HT1 was supposed to be ~1100 bp, but the band was very faint in 1100 bp. For HT2, there was a band at 800 bp, but it showed multiple bands, indicating low specificity. Because of this problem, UI_Indonesia sought help to ligate the two fragments together and put it in a backbone. Therefore, they asked us for help in ligating and cloning this fragment into a vector, so it would be easier to amplify and proceed for their assay.
Figure 1.  PCR Result of HT1 and HT2 from UI_Indonesia indicating non-specificity in amplification
How We Helped?
One of our team members Angelysia Cardilla managed to meet up with UI_Indonesia in Jakarta, Indonesia, to discuss about our projects and get to know more about each team's project. In the session, UI_Indonesia also passed us with the gBlocks that they wanted to clone. 
Figure 2.  Meetup with UI_Indonesia
We cloned the HT1-HT2 fragments into the pcDNA3 backbone. Firstly, we kept all the gBlock fragments in the TOPO clone vector before amplifying them. We found that amplifying from TOPO template could lead to higher yield and efficiency. We designed some Gibson primers to combine the two fragments into the pcDNA3 vector. There were a total of 4 primers that we used to do Gibson cloning, HT1 F, HT1 R, HT2 F, and HT2 R. To check if the cloning was successful, we did colony PCR. We also confirmed the successful cloning of HT into pcDNA3 by sending the plasmid for sequencing. The sequence below shows the conjunction of the vector backbone and the HT construct, and the part that is not highlighted is pcDNA3 which has Ampicillin resistance as selective marker. 
Figure 3. Sequence of HT in pcDNA3
As they had faced difficulties in cloning these constructs and such problems may arise again in preparing for part submission, we proposed to them to help to move HT fragment into BioBricks compatible vectors. We cut both the backbone and the plasmid pcDNA3-HT1-HT2 to get the insert with restriction enzyme EcoRI and PstI. Afterward, we ligated them and transformed into competent cells, cultured them on the plate with CML resistance. The correct insertions were confirmed after colony PCR with VF2 and VR primers which are shown on the figure below. All the colony we picked showed correct band size, indicating successful results. 
Figure 4. Colony PCR of psB1C3-HT 
Anticipating other possible difficulties they might have in preparing for part submission, we further offered to help them to remove prohibited restriction sites on the final plasmid and made the plasmid BioBricks compatible. In the end, we submitted their constructs on their behalf as  BBa_K2607001  to save the unnecessary time and cost in shipping.
How They Helped Us?
UI_Indonesia helped us greatly in our human practice, for both the outreach and the feedback garnering of our social study.  As a part of our study about public’s opinion regarding gene therapy, they helped us to translate our research questionnaire into Bahasa and spread to the society in Indonesia.
This was of great help as there were a great number of respondents from different educational backgrounds, age groups, gender to provide us with an extensive demographic profile, avoiding potential selection bias.  As Singapore and Indonesia are two countries in different phases of development, in terms of the social and religious environment, size of population, rules and regulations etc, with data from Indonesia, we were able to compare how the public opinion might differ for two different countries. 
Figure 5. Sample of data UI_Indonesia collected for us
Our main reason to extend our human practice to other countries is that we hope that ultimately gene therapy can be applied evenly to everyone in the world. Therefore, it is important to understand the public views of different countries with different socio-economic conditions. It is a good chance for us to utilize this collaborating opportunity with UI_Indonesia to conduct an international study regarding gene editing. We are also thankful that UI_Indonesia helped us to analyze the survey question for the study in Indonesia. From UI_Indonesia analysis, we found that most of Indonesians support gene therapy editing for treatment. However, most of the respondents also have many concerns about cost, safety, and ethics. We deeply appreciate their help in sampling and  survey analysis  and their results are valuable inputs to our  human practice .
Figure 6. Outreach by UI_Indonesia
Apart from helping us in our survey study, UI_Indonesia also planned an outreach event for us. We believe that it is important for the public to know more and be aware of this gene-editing therapy. Moreover, since gene editing is not a common issue, there is a lack of knowledge about this in Indonesia. Therefore, as a part of our outreach, UI_Indonesia helped us to educate students from the University of Indonesia so that they could be better prepared for gene editing technologies in the future.
We are pleased that we were able to share about our project to students in Indonesia through the help from UI_Indonesia . We prepared the material that we would like to be delivered in this outreach. The discussion was held in September and delivered in their campus auditorium. This session attracted about 60 students from Faculty of Engineering. We are very thankful that UI_Indonesia helped us to extend our outreach to the international community.
iGEM Meet-up Singapore
After a long history of collaborating with iGEM teams fron National University of Singapore (NUS), we were honored to be invited to the first iGEM meeting ever in Singapore. During the meet-up, we were happy to meet with NUS Singapore-A and NUS_Singapore-Sci.
During the event, we had both casual conversations to know each other as well as in-depth discussions about our projects. We were able to propose and find potential areas for collaboration, in both wet laboratory work and human practice. We believe this meet up has laid out a great foundation for our later meetings and collaborations with them.
Photo Credit to Team NUSGEM
Photo Credit to Team NUSGEM
Photo Credit to Team NUSGEM
Photo Credit to Team NUSGEM
Previous Next
NUSGEM Singapore
 How the Story Began?
After meeting with Team NUSGEM at the Singapore iGEM meet-up, both teams were able to quickly find area for collaboration. Afterwards, we met multiple times in our laboratory and after a few weeks, we finalized our plan for collaboration.
 How We Helped?
Team NUSGEM ( NUS_Singapore-A ) developed a novel biomanufacturing method and produced a flavonoid dye, Luteolin, in Escherichia coli. Three plasmids were constructed which are pBrep-F3'H-pTet-FNS, pBrep-F3’H and pBAD-FNS. pBrep-F3'H-pTet-FNS plasmid consists of F3’H gene under blue light repressible promoter and FNS gene under pTet promoter. F3’H plasmid consists of F3’H gene under a blue light repressible promoter, while FNS plasmid consists of FNS gene under pBAD promoter. Our team had helped them to quantify each of the gene expression level by RT-qPCR in cells containing these plasmids versus empty cells.
Figures below shows the exponential fold change of the gene, after normalizing with the wild-type cells and the house-keeping 16s gene.
Figure 7. Gene expression level of F3’H gene in pBrep-F3’H-pTet-FNS plasmid and pBrep-F3’H plasmid
Figure 8. Gene expression level of F3’H gene in pBrep-F3’H-pTet-FNS plasmid and pBrep-F3’H plasmid
As shown from the above bar charts, F3’H and FNS genes level is confirmed. While this is expected for the F3’H gene, for its blue light repressible promoter acting like a constitutive one in dark conditions, FNS gene under pBAD promoter shows expression even without arabinose induction.
It is worth to notice that the RT-PCR efficiency might be different for different genes due to the sizes of their PCR products and that the formula used above is assuming the PCR efficiency to be 100% that eventually resulted in 2 copies of products.
 How They Helped Us?
We appreciate and thank Team NUS_Singapore-A in helping us by validating the construct in our adenine base editing project. With only growth on Kanamycin when both plasmids were induced, they verified that our system works as proposed and regulation of these two plasmids is indeed tight.  The detailed experimental procedure can be found here.
Figure 9. Characterisation of our TadA*-dCas9 construct by NUSGEM
MacQuarie Austrailia
 Our Collaboration
This year, in our collaboration with Macquarie University Australia, We performed qPCR measurement for two of their constructs to verify plasmid gene expression profile upon IPTG induction. In return, we leveraged upon their protein purification and analysis technique to perform an EMSA shift in hope of observing improved DNA or gRNA binding by our improved tCas9 variants. Two improved variants with site-specific mutations were sent for analysis – Rec2_HNH 5_6_10_15 HNH-pK and 3ple 5_6_10_15 HNH-pK. Following figures show the results they obtained.
Constructs were tested against their unimproved variants (without 5_6_10_15 HNH-pK). Results were depicted as protein gel (bright green) overlaid against nucleic acid gel (black). Results indicate that all tested tCas9 binds to target DNA efficiently. Further repeats should be done with normalized, high amount of purified tCas9 to achieve results similar to lane 5 in figure 2, where some unbound tCas9 is observable.
Figure 10. Rec2_HNH on the lane 1-3, Rec2_HNH 5_6_10_15 HNH-pK on lanes 4-6.
For each sample, samples are loaded as follows: tCas9 only, tCas9+gRNA+target DNA, denatured tCas9+gRNA+target DNA.
Figure 11. 3ple on the lane 1-3, 3ple 5_6_10_15 HNH-pK on lanes 4-6.
For each sample, samples are loaded as follows: tCas9 only, tCas9+gRNA+target DNA, denatured tCas9+gRNA+target DNA.
 Our Takeaways
Through our collaboration with other iGEM teams we indeed felt the sense of community in iGEM. While Singapore is a small country, our shared goals and interests have brought us together to brainstorm ideas and tackles daring problems that may be unimaginable for us, if not for this competition. As such, we feel that our journey in collaboration is indeed invaluable and a memorable one.
