"
Page
Discussion
View source
History
teams
Log in
 
Team:Concordia/Notebook
From 2014.igem.org
iGEM Concordia 2014
Back to iGEM HQ
iGEM Concordia 2014
Clean Green Lipid Machines: Synthetic Biology Tools for Microalgae
Homepage
The Project
Sustainability
Introduction
Microalgae
Toolkit
The Parts
The Notebook
Human Practices
Safety
Attributions
The Team
Team Members
iGEM Team Profile
Navigation
Growth Curve and Cell Count
Antibiotic Spot Test
Gibson Assembly
Transformation & Genomic DNA Extraction
Cryofreeze
Growth Curve and Cell Count
Cell count via Hemocytonomer
Figure1: Diagram of the parts of the hemocytometer
Clean the cover glass mounting support and the coverslip of the hemocytometer with a lens paper and some ethanol.
Note: Coverslips for counting chambers are specially made and are thicker than those for conventional microscopy, since they must be heavy enough to overcome the surface tension of a drop of liquid.
Place the coverslip over the counting surface prior to putting on the cell suspension.
Introduce the suspension into one of the V-shaped wells with a pasteur or other type of pipet.
Note: The area under the coverslip fills by capillary action. Enough liquid should be introduced so that the mirrored surface is just covered.
Place the counting chamber on the microscope stage and bring the counting grid is into focus at low power.
Growth Curve
With the use of a hemocytometer, count the number of cells present in a small sample of your culture
With this number, calculate the cell density (cells/ml) of your culture
For the same culture, take cell count measurements on a regular basis (every 5 hours) for an entire week
Record all the data you collect and the time at which you took the measurement!
Once the week is over, plot cell density versus time in order to obtain a growth curve
You must be able to see the different growth phases the species goes through (log, stationary)
Figure 2: Growth curves of all 5 species of algae (C. vulgaris, C.kessleri, C. ellipsoidea, C. saccharophila, C.reinhardtii UVM1) studied over the summer, with error bars. Done in triplicates.
Back to Top
Antibiotic Spot Test
Type of Plate
ug/ml
Species
UVM1
Vul
Kess
Control
0
+
+
+
Hygromycin
1
-
+
+
2.5
-
+
+
5
-
+
+
10
-
+
-
15
-
+
-
20
-
+
-
25
-
+
--
30
-
+
-
35
-
+
-
40
-
+
-
45
-
+
-
50
-
-
-
60
-
-
-
Phleomycin
0.5
-
+
+
1
-
+
-
Spectinomycin
5
+
+
+
10
-
+
+
15
-
+
+
20
-
+
+
25
-
+
+
Chloramphenicol
200
+
+
+
300
-
+
+
400
-
+
+
Paromomycin
2
-
+
+
4
-
+
+
Figure 3. Antibiotic spot tests on various antibiotics shown here. (a) TAP media containing glacial acetic acid, and no antibiotics. Control. (b) Hygromycin at 10 µg/ml. (c) Hygromycin at 20 µg/ml. (d) Phleomycin at 1.0 µg/ml. (e) Chloramphenical at 200, 300, and 400 µg/ml
Chlamydomonas reinhardtii (UVM1), Chlorella kessleri, and Chlorella vulgaris showed various sensitivities to different antibiotics and concentrations of each. UVM1 was observed as the most sensitive to the antibiotics we tested, by showing no growth on 1 µg/ml hygromycin, 0.5 µg/ml phleomycin, 10 µg/ml, 300 µg/ml chloramphenicol, and 2 µg/ml paromomycin plates. This differed from the two chlorella species, in that the two chlorella species did not grow on either higher  concentration plates, or did not die within our tested range of concentrations.
For spectinomycin, paromomycin, chloramphenicol, and phleomycin, at least one of the two species (C. kessleri and C. vulgaris) grew on our upper range of concentrations. Only Hygromycin, at 50 µg/ml, did we observed no growth on all three microalgae species. Due to these results, we continued with hygromycin for selection experiments to use when working with C. reinhardtii (UVM1), C. kessleri, and C. vulgaris. We can also conclude that when considering Chlorella kessleri, and Chlorella vulgaris as novel genetic engineering chassis, C. kessleri is more sensitive to the antibiotics tested in this experiment, which include hygromycin, spectinomycin, paromomycin, chloramphenicol and phleomycin. It follows to reason that C. kessleri is the more versatile and cheaper microalgae species to work with when compared to C. vulgaris.
Back to Top
Gibson Assembly
The DNA you wish to be assembled and the Gibson Master Mix should be combined with a volumetric ratio of 1:3 in a PCR tube.
Note: we need 300 femtomoles of each part for the reaction
The total volume can be from 20-50µl.
The PCR tube should then be incubated for 1 hour at 50°C
Gibson Master Mix:
50 µl
Taq Ligase (40 u/µl)
100 µl
5x isothermal buffer
2 µl
T5 exonuclease (1u/µl)
6.25 µl
Phusion polymerase
216.75 µl
Nuclease-free water
375 µl
Total Volume
5x Isothermal buffer:
0.75 g
25% PEG-8000
1500 µl
500 mM Tris-HCl pH 7.5
75 µl
50mM MgCl2
150 µl
50mM DTT
30 µl
1 mM dATP
30 µl
1 mM dTTP
30 µl
1 mM dCTP
30 µl
1 mM dGTP
300 µl
5mM NAD
Remainder
Nuclease-free water
3000 µl
Total Volume
Figure 4: Gibson Assembly ® Master Mix." Reagents For the Life Sciences Industry. N.p., n.d. Web. 10 Oct. 2014
Figure 5. Plasmid
Back to Top
Transformation and Genomic DNA Extraction
Electroporation
Vulgaris, Kessleri, UVM1
Calculate OD of sample of interest to determine desired total cell count. Densities may range from: 1 x 10^6 cells - 1 x 10^8 cells (Use gamma radiated centrifugation tubes)
Calculate OD [750nm] of culture and compare with growth curve to determine cells/mL and determine total volume of culture required to get desired total number of cells.
Note: aim for cells in mid log phase of the growth cycle.
Take note of values used in the lab book.
Note: You will require both a -ve and +ve control. Take this into account when preparing cells for centrifugation.
Obtain plasmid of interest
i.e. CrGFP, No linker GFP, etc.
Ensure that the part is Gibson assembled.
Set up restriction digest. All pieces to mixed together in 1.5ml centrifuge tube.
Determine volume required to obtain 5 ug of plasmid DNA. [Calculation example: Part Concentration [µg/µL] x unknown volume [µL] = 5ug of plasmid DNA
Set up requirements for a 30 µL digest:
5 µg Plasmid DNA [Volume as per calculation above]
2 µL SwaI [Last piece to be added to tube]
3 µL NEBuffer (10X)
0.5 µL 100X BSA
TBD dH20 [Top up to 30µL with dH20] **30µL Total**
Incubate each tube at 25 °C for 1-2 hours. This may require incubation in water bath or other climate controlled     apparatus.
Optional: Run gel to confirm cassette is assembled properly. With the except of the ladder [5µL], each well should contain the following:
1.2 µL DNA [~200ng]
3.3 µL dye [Following a 5:1 ratio of total reaction: dye]
15.5 µl dH20
20 µL Total/well
Harvest the cells via centrifugation at 2500 rpm for 10 minutes at room temperature. Discard the supernatant by decanting. Remove the remaining supernatant using a pipette.
Re-suspend the cells in ~80 µL of dH20. Pipette up and down multiple times to ensure homogeneity.
May want to resuspend in osmotic buffer [Tap] for 1hr and then centrifuge again and use a different electroporation buffer, not water.
Add the entire content of the RE digest to the cells + dH20. Pipette up and down once again.
Note: it is highly desirable to have the RE digest complete at the same time as the cells are resuspended in dH20
Note 2: May be advantageous to add 25µg of Salmon sperm DNA at this point to act as carrier DNA. [or 200µg/ml]
Transfer the contents of the centrifugation tube to a 0.2cm electroporation cuvette. Place the cuvettes on ice for 5-10 minutes prior to incubation.
Use Bio-Rad 2 for electroporation with the following parameters:
C.Vulgaris, C.Kessleri
Field Strength: 1800 V/cm
Voltage: 360 V
Ohm: 200Ω
Capacity: 25 µF
Number of Pulses: >8
UVM1
Field Strength: 1500 V/cm
Voltage: 300V
Ohm: ∞
Capacity: 50 µF
Number of Pulses: 4-5
Figure 6. Electroporation FAQ." Cell Transfection and Cell Fusion Products. BTX Harvard Apparatus, n.d. Web. 10 Oct. 2014
Prior to applying voltage, tap the cuvettes to mix contents
Note: The voltage may be adjusted. Take note of the health/number of cells post electroporation. The species may require a decrease in voltage &/or pluses.
Once electroplated, place the cuvettes on ice for an additional 5-10 minutes.
Make 3mL aliquots of Tap + 40 mM Sucrose into 6 well plates. Wash each cuvette with ~1mL of distilled water and evenly divide the contents of the cuvette into 2 separate wells.
Note: It is advisable to have these plates ready prior to performing electroporation.
Label Clearly!!!
Place the 6-well plate in the dark at room temperature and let incubate for 24 hours.
Once incubation is complete, gently agitate, ~100-150 rpm, for 5-10 minutes to allow for homogeneity within the culture.
Pipette contents of sister wells into gamma radiated centrifugation tubes.
Centrifuge at 2500 rpm for 10 minutes at room temperature.
Remove supernatant via decanting.
Re-suspend pellets in 200 µL of distilled water.
Plate 100 µL x2 onto appropriate plates.
positive control = Tap
negative control = Tap + 50µg/ml hygromycin
Experimental = Tap + 50µg/ml hygromycin
**Note: Ensure that the plates are free from condensation**
Place plates agar side down in incubator.
Note: Photoperiod may be adjusted once heterotrophic growth ends.
Check for successful transformations in 8-10 days.
Restreak transformed colonies on sister plates and allow to grow for 3 days.
Inoculate in 3mL Tap + appropriate [Hyg]. Let stand.
Extract Chlorella DNA.
References
Chow, C., & Tung, W. L. (1999). Electrotransformation of Chlorella vulgaris, (December 1997), 778–780.
Hawkins, R. L., & Nakamura, M. (1999). Expression of Human Growth Hormone by the Eukaryotic Alga , Chlorella, 38, 335–341.
Liu, L., Wang, Y., Zhang, Y., Chen, X., Zhang, P., & Ma, S. (2013). Development of a new method for genetic transformation of the green alga Chlorella ellipsoidea. Molecular Biotechnology, 54(2), 211–9. doi:10.1007/s12033-012-9554-3
Macc, C. (n.d.). Transient Expression of the GUS Gene in a Unicellular Marine Green Alga , Chlorella sp .
Algae DNA Extraction CTAB/DTAB Method
Reagents
200 µl
Extraction buffer (1M NaCl, 70 mM Tris, 30 mM, pH = 8.6)
~500 µl
Glass Beads
12 µl
10% DTAB
450 µl
Chloroform
170 µl
0.5% CTAB
100 µl
40 mM NaCl
Pellet and Lyse Cells:
Spin cells down in 1.5 tube at 12500 rpm for 3 minutes
Wash Pellet with 100 uL extraction buffer
Resuspend pellet in 100 uL extraction buffer
Add glass beads and vortex for 1 min to break the cells
Add 12 µl of 10% DTAB
Heat samples in boiling water for 30s
Chloroform Extraction:
Add 150 ul chloroform. Vortex for 5 seconds. Centrifuge at max rpm for 1 minute. Transfer aqueous layer (top) to a fresh tube.
Repeat previous step once or twice more, making sure to not take any chloroform during the last repeat.
Add and mix with 170 uL 0.5% CTAB (40 mM NaCl)
Centrifuge at 14,000 rpm for 2 min to pellet DNA. Decant. Do not lose any pellet at this point
Add 100 ul of 1.2 M NaCl
Ethanol Precipitation:
Add 50 ul of 5M Sodium Acetate or 10 ul of 3.2 M Sodium Acetate pH 5.2
Mix in 250 ul of ethanol 100%
Freeze overnight in -20 or place in -80 for 2-3 hours
Centrifuge in the 4 degree at full speed for 30 mins
Decant very carefully
Add 250 ul of 70% ethanol (stored in -20)
Centrifuge in 4 degree at full speed for 30 mins
Decant very carefully
Air Dry or vacuum
TAQ PCR Screening on Transformant DNA Extracts
Figure 7. TAQ PCRs done on the transformant samples of algae genomic DNA extractions with GFP as the reporter gene in each of these transformants. Subsequent GFP-amplifying primers were used to see if the coding sequence was present in the genomic extract.
On the left,  one of the earliest cassettes built can be seen showing an intense band, and to its immediate right there is  a faint, but still present band of one of the later cassettes using PSAD gene’s promoter/terminator flanking a GFP with a nuclear localization signal. To the far right, there is a GFP control which has all the same reagents (including primers) of the other reactions, differing only in the template (pCrGFP)  - the template we used to amplify our GFP to build our cassettes.
Fluorescence Microscopy Analysis of Microalgae
Figure 8. Fluorescence Microscopy Image of Wild Type Chlamydomonas reinhardtii [UVM1]
Figure 9.  Fluorescence Microscopy Image of Transformed Chlamydomonas reinhardtii [UVM1] with PpsaD - CrGFP-NLS - TpsaD
The images agove were taken using the Nikon Eclipse TiE invertted epifluorescence microscope. The Nikon Eclipse TiE is a fully automated inverted microscope outfitted with a Heliphor LED-based source (405, 480 (GFP), 555, 640 nm) with for fluorescence imaging and to limit phototoxicity and a Photometrics CoolSNAP KINO CCD camera for high resolution live cell imaging. It is equipped with five bright lenses (we used 100x), DIC optics and a motorized stage (including peizo Z control) .  The system is operated using NIS-Elements 4.0 software.
Figure 8 shows wild type UVM1 strain under a fluorescent microscope, while figure 9 shows UVM1 transformants. Our GFP-NLS cassette, containing the strong PSAD promoter, was used for this transformation. The light green band in the center of the cells in figure 8 only appear in our transformant culture, in which previously a GFP insert’s presence was verified by a TAQ PCR screen. Comparing the two figures, the light green band did not appear in our wild type culture. This indicates that GFP has been successfully transformed into our UVM1 strain, and, in addition, the GFP appears to be localized to a specific region of the cell. We suspect that this region is the nucleus, due to the presence of a nuclear localization signal (NLS) coded next to the GFP. However, we cannot conclude that this region is where the nucleus is, and subsequently that the nuclear localization signal works, until a nuclear stain is performed. We plan on experimentally confirming the specific area of this GFP localization using nuclear staining methods. "
References
Fawley, M. W., M. Qin, and Y. Yun. 1999. The Relationship Between Pseudoscourfieldia marina and Pycnococcus provasolii (Prasionphyceae, Chlorophyta): Evidence From 18S rDNA Sequence DATA. J. Phycol. 35, 838-843.
Back to Top
Cryofreezing Cells
Materials Needed:
2 Spill-proof stryofoam box
Metal Rack
white gloves
Isopropanol
Methanol
Liquid Nitrogen
1.8 mL crytubes
Cryocontainer
Liquid Nitrogen Storage & Transportation:
Disinfect the bench by spraying ethanol
Prepare the foam box + metal rack to hold the tubes for N2 freezing
Wear white gloves + rubber to do N2 freeze & retrieve tubes
Open & leave N2 out to evaporate (end of the day, somewhere where it won’t spill)
Conditions:
2 sets of culture: Log & Stationary
§  (Note: stationary phase→ higher lipid profile, cryoprotectant cells
Vary methanol percentages: 6% methanol (3% final conc.) worked best*
Species/Strains: UVM1, Kessleri, Ellipsoidea, Vulgaris, Saccharophila
PROTOCOL:
Grow cells to approximately 1,000,000 cells per ml in TAP (strain 2137) or TAP plus Arg (CC-425). Pellet cells and resuspend in 1/10 volume fresh growth media.
To a 1.8 ml Nunc style cryotube add 250 µl of appropriate growth media containing 2 - 10% (v/v) methanol or DSMO. We find 6% methanol (3% final concentration) works best for strain 2137 and CC-425 (see below).
Add 250 µl(equal volume) of 10X cells to the tube. This gives a final cryoprotectant concentration range of between 1-5% (v/v).
Place the tubes in a Nalgene Cryo 1C Freezing Container (esssentially an isopropanol bath and place the Cryo-container in a minus 80 C freezer. The cryo-container slowly freezes the cells at about 0.9-1.0 C/min. Leave Cryo-container in freezer until isopropanol reaches - 40 C; about 65-72 min.
Remove tubes and immediately freeze in liquid nitrogen. Store at liquid nitrogen temperatures.
Thaw cells by placing in a 35 C water bath for two minutes with gentle shaking.
Transfer cells to 10 ml of appropriate growth medium and grow 6-18 hours before plating.
Results
Figure 10. UVM1 plates at different methanol concentrations
Figure 11. C. kessleri plates at different methanol concentrations
Figuere 12. C. vulgaris at different methanol concentrations
Cryopreservation of Chlamydomonas cells has shown very low cell viability in past experiments [1]. However, cryopreservation is an invaluable tool for any model organism because it allows researchers to inexpensively maintain cell cultures for extended periods of time, without constant maintenance. It is especially appealing to geneticists, who are often interested in unique transformant cell lines, which can sometimes take months of work to create. If there is cryopreservation protocol for chlorella and chlamydomonas species, then working in these organisms becomes more appealing.
In this experiment, we modified a protocol for cryopreservation of Chlamydomonas reinhardtii, which was created by Richard Sayer[2]. Chlorella vulgaris, Chlorella kessleri and Chlamydomonas reinhardtii (UVM1) were frozen with methanol concentrations of 3 %, 6%, 9% in TAP growth media. We compared early to stationary phase cultures, in 25000 and 2500 cells/ml. Growth of all three species was observed and a comparative analysis of growth between species was performed. Chlorella kessleri responded best to the cryopreservation procedure.
At all methanol percentages (3%, 6%, and 9%), there was growth. There appears to be more growth, and also larger colonies on the higher methanol percentage plates. Future experiments could involve testing higher methanol percentages to see if cell viability can be increased even further. The number of colonies were difficult to count due to c. kessleri’s tendency to grow as “smears” when growth is high, and not nicely isolated colonies. However, for UVM1, isolated colonies could be easily counted. UVM1 responded poorly to cryopreservation – we suspect that this is because of the lack of cell wall in this strain. This hypothesis can be tested with an experiment comparing growth between a wildtype C. reinhardtii strain and UVM1 after cryopreservation. Despite poor growth, there were a range of 1-50 isolated small colonies on about half of the plates tested. For C. vulgaris, high growth was observed on one plate of 3% methanol, while the rest could be described as medium to low growth. Only one plate at 9% methanol produced isolated colonies for a total count of 29.
References
[1]Harris, Elizabeth H. (2009). The Chlamydomonas Sourcebook (2nd Edition) Volume 1: Introduction to Chlamydomonas and Laboratory Use. Durham, North Carolina. Elsevier.
[2]Experiment based on protocol posted to bionet.chlamydomonas by Richard Sayre (with modifications)
Back to Top
Synbiota Lab Notebook
To see our complete online lab notebook with Synbiota, click here .
Stay in touch with the iGEM Concordia 2014 team:
Retrieved from " http://2014.igem.org/Team:Concordia/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
