<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_GDSYZX-United_results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:GDSYZX-United/results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="wrapper wrapper-content"><NAV class="navbar navbar-fixed-top navbar-inverse" role="navigation"><DIV class="navbar-collapse collapse" id="navbar"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A aria-expanded="false" role="button" href="https://2016.igem.org/Team:GDSYZX-United">HOME</A></LI><LI class="dropdown"><A aria-expanded="false" role="button" href="#" class="dropdown-toggle" data-toggle="dropdown">TEAM</A></LI><LI><A href="https://2016.igem.org/GDSYZX-United/TEAM/team_members">Team members</A></LI><LI><A href="https://2016.igem.org/Team:GDSYZX-United/Collaborations">Collarboration</A></LI><LI><A href="https://2016.igem.org/Team:GDSYZX-United/Attributions">Attribution</A></LI></UL><LI class="dropdown"><A aria-expanded="false" role="button" href="#" class="dropdown-toggle" data-toggle="dropdown">PROJECT</A></LI><LI><A href="https://2016.igem.org/Team:GDSYZX-United/Description">Project description</A></LI><LI><A href="https://2016.igem.org/GDSYZX-United/PROJECT/extract _dna">Extract DNA in arabidopsis thaliana</A></LI><LI><A href="https://2016.igem.org/GDSYZX-United/PROJECT/enzyme_cleavage">Enzyme cleavage</A></LI><LI><A href="https://2016.igem.org/GDSYZX-United/PROJECT/preparation_arabidopsis_protoplast">Preparation of the Arabidopsis protoplast</A></LI><LI><A href="https://2016.igem.org/Team:GDSYZX-United/Proof">Proof</A></LI><LI><A href="https://2016.igem.org/Team:GDSYZX-United/Demonstrate">Demonstrate</A></LI><LI class="dropdown"><A aria-expanded="false" role="button" href="https://2016.igem.org/Team:GDSYZX-United/results">RESULTS</A></LI><LI class="dropdown"><A aria-expanded="false" role="button" href="#" class="dropdown-toggle" data-toggle="dropdown">PARTS</A></LI><LI><A href="https://2016.igem.org/GDSYZX-United/PARTS/parts">Parts</A></LI><LI class="dropdown"><A aria-expanded="false" role="button" href="#" class="dropdown-toggle" data-toggle="dropdown">NOTEBOOK</A></LI><LI><A href="https://2016.igem.org/GDSYZX-United/NOTEBOOK/day_note">Day notes</A></LI><LI><A href="https://2016.igem.org/GDSYZX-United/NOTEBOOK/procedure_record">Procedure record</A></LI><LI><A href="https://2016.igem.org/GDSYZX-United/NOTEBOOK/protocol">Protocol</A></LI><LI class="dropdown"><A aria-expanded="false" role="button" href="#" class="dropdown-toggle" data-toggle="dropdown">HUMAN PRACTICE</A></LI><LI><A href="https://2016.igem.org/Team:GDSYZX-United/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:GDSYZX-United/HP/Gold">Gold</A></LI><LI><A href="https://2016.igem.org/GDSYZX-United/HUMAN_PRACTICE/album">Album</A></LI><LI class="dropdown"><A aria-expanded="false" role="button" href="#" class="dropdown-toggle" data-toggle="dropdown">SAFETY</A></LI><LI><A href="https://2016.igem.org/GDSYZX-United/SAFETY/lab_safety">Lab safety</A></LI><LI><A href="https://2016.igem.org/GDSYZX-United/SAFETY/safety_of_process">Safety of process</A></LI><LI class="dropdown"><A aria-expanded="false" role="button" href="https://2016.igem.org/Team:GDSYZX-United/Entrepreneurship">ENTREPRENEURSHIP</A></LI></DIV></NAV><DIV class="row animated fadeInRight"><DIV class="col-sm-12"><DIV class="ibox float-e-margins"><DIV class="timeline"><DIV><DIV class="row"><DIV class="text-center"><H1><STRONG>Results</STRONG></H1></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-sm-8 content"><H1>Design of our experiment</H1><H2><STRONG>Hhl1-protein coding region</STRONG></H2><DIV class="text_font" style="text-indent:0em;">Gene hhl1 encodes a 231-amino acids protein HHL1.HHL1 helps protect the PSⅡagainst photodamage and help increase quantum yield.However,HHL1 is only expressed when PSⅡ is already damaged , we speculate that the quantity of HHL1 expressed after PSⅡis damaged may be not enough to repair PSⅡ efficiently.We want to prevent photodamage in PSⅡ，so we add photosensitive promoters to initiate the transcription of gene hhl1 before PSⅡ is photodamaged. At certain light intensity,the promoter initiate the transcription of gene hhl1,so there will be HHL1 in chloroplast before PSⅡ is damaged，and the repair process can be faster.</DIV><H2><STRONG>Selecting the light sensitive promoters</STRONG></H2><DIV class="text_font">Different kinds of Plants have different needs for light, and their resistance against light stress is different.Because of this, the light intensity for photo -sensitive promoters to be activated should be different according to the type of plant. So we use 3 photosensitive promoters in our parts,and then compare the expression level of HHL1 in certain light intensities. After that we select the most-expressed part in certain light intensities. Those corresponding proteins of the three photosensitive promoters we used are:</DIV><DIV class="text_font"><STRONG>pif1</STRONG></DIV><DIV class="text_font">It is a key negative regulator of phytochrome-mediated seed germination and acts by inhibiting chlorophyll biosynthesis, light-mediated suppression of hypocotyl elongation and far-red light-mediated suppression of seed germination, and promoting negative gravitropism in hypocotyls. Light reduces this activity in a phy-dependent manner. The protein preferentially interacts with the Pfr forms of Phytochrome A (PhyA) and Phytochrome B (PhyB), is physically associated with APRR1/TOC1 and is degraded in red (R) and far-red (FR) light through the ubiquitin (ub)-26S proteasome pathway to optimize photomorphogenic development in Arabidopsis. It also negatively regulates GA3 oxidase expression.</DIV><DIV class="text_font"><STRONG>phyB</STRONG></DIV><DIV class="text_font">Red/far-red photoreceptor involved in the regulation of de-etiolation. Exists in two inter-convertible forms: Pr and Pfr (active). Involved in the light-promotion of seed germination and in the shade avoidance response.</DIV><DIV class="text_font"><STRONG>cop1</STRONG></DIV><DIV class="text_font">Represses photomorphogenesis and induces skotomorphogenesis in the dark. Contains a ring finger zinc-binding motif, a coiled-coil domain, and several WD-40 repeats, similar to G-beta proteins. The C-terminus has homology to TAFII80, a subunit of the TFIID component of the RNA polymerase II of Drosophila. Nuclear localization in the dark and cytoplasmic in the light.</DIV><DIV class="text_font">Many genes in Arabidopsis are regulated by light.We want to find out these genes and use their promoters.To find out these special genes,we use a website database (www.kegg.jp)to find signaling pathways regulated by light in Arabidopsis. After knowing which protein is expressed in response to light , we use website NCBI to find out the gene sequence that encodes these proteins. To get the promoter sequence of each gene , we use another website(www.cbs.dtu.dk/services/promoter) to estimate the promoter sequence.</DIV></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-sm-8 content"><H1>Extract the Arabidopsis genomic DNA</H1><DIV class="text_font">Arabidopsis genomic DNA is used as template strand in PCR for all our basic parts.We extracted Arabidopsis genomic DNA using a kit box.</DIV><DIV class="text_font">The electrophoregram of the detection of the Arabidopsis genomic DNA is shown below:</DIV></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-sm-8 content"><H1>Obtain our basic parts</H1><DIV class="text_font">To get five basic parts(pCop1,pPif1,pPhyB,hhl1,pHhl1-hhl1), we have to do five groups of PCR . Before that,we use the software VectorNTI to do the primer design，then we asked a biotech company(BGI,The Beijing Genomics Institude) to do primer synthesis.</DIV><DIV class="text_font">Following prictures are the electrophoregrams for PCR confirmation of these five DNA fragments:</DIV><DIV class="text_font">The size of the PCR fragment of gene hhl1,PHhl1L-hhl1,phyB,cop1,pif1 are supported to be 3.1kb,4.0kb,0.8kb,1.8kb,1.5kb,respectively.</DIV><DIV class="text_font">After assembly our basic parts into a composite part,we use E.coli DH5α to clone the plasmid.</DIV></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-sm-8 content"><H1>Construction of the biobricks</H1><DIV class="text_font">Followings are the parts and we used in our project(all the prictures shown below are the electrophoregrams of the selections and identifications of recombinant plasmids using colony PCR method):</DIV><H2><STRONG>PphyB-hhl1</STRONG></H2><DIV class="text_font">This biobrick consists of a photo -sensitive promoter PphyB, and protein coding sequence hhl1. This biobrick is constructed to produce more protein HHL1 in chloro- plasts to help protect PSⅡ before photodamage takes place.</DIV><DIV class="text_font">We used the forward primer of the part PphyB and the downstream primer of the part hhl1 to select the recombinant plasmids.The size of the recombinant fragment is supported to be 3.9kb.</DIV><H2><STRONG>Ppif1-hhl1</STRONG></H2><DIV class="text_font">This biobrick consists of a photosensitive promoter Ppif1,and protein coding sequence hhl1.This biobrick is constructed to produce more protein HHL1 in chloroplasts to help protect photosystemⅡ before photodamage takes place.</DIV><DIV class="text_font">We used the forward primer of the part Ppif1 and the downstream primer of the part hhl1 to select the recombinant plasmids.The size of the recombinant fragment is supported to be 4.6 kb.</DIV><H2><STRONG>Pcop1-hhl1</STRONG></H2><DIV class="text_font">This biobrick consists of a photosensitive promoter Pcop1,and protein coding sequence hhl1.This biobrick is constructed to produce more protein HHL1 in chloroplasts to help protect photosystemⅡ before photodamage takes place.</DIV><DIV class="text_font">We used the forward primer of the part Pcop1 and the downstream primer of the part hhl1 to select the recombinant plasmids.The size of the recombinant fragment is supported to be 4.9 kb.</DIV><H2><STRONG>PphyB-RFP</STRONG></H2><DIV class="text_font">This biobrick consists of a photosensitive promoter PphyB and an RFP as a reporter.</DIV><DIV class="text_font">We used the forward primer of the part PphyB and the downstream primer of the part RFP to select the recombinant plasmids.The size of the recombinant fragment is supported to be 1.7 kb.</DIV><H2><STRONG>Ppif1-RFP</STRONG></H2><DIV class="text_font">This biobrick consists of a photosensitive promoter Ppif1 and an RFP as a reporter.</DIV><DIV class="text_font">We used the forward primer of the part Ppif1 and the downstream primer of the part RFP to select the recombinant plasmid.The size of the recombinant fragment is supported to be 2.4 kb.</DIV><H2><STRONG>Pcop1-RFP</STRONG></H2><DIV class="text_font">This biobrick consists of a photosensitive promoter Pcop1 and an RFP as a reporter.</DIV><DIV class="text_font">We used the forward primer of the part Pcop1 and the downstream primer of the part RFP to select the recombinant plasmid.The size of the recombinant fragment is supported to be 2.7 kb.</DIV></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-sm-8 content"><H1>Proof of concept</H1><H2><STRONG>Expression level of protein HHL1</STRONG></H2><DIV class="text_font">After transfection, we incubate the protoplasts at 2 groups of light intensities to induce the expression of hhl1.</DIV><DIV class="text_font">We use semi-quantitive RT-PCR to detect the mRNA from gene hhl1 in protoplasts，the concentration of amplified cDNA of each group is measured by fluorescent quantitation. Data of the concentration of amplified products are used to calculate the relative expression level of mRNA.</DIV><TABLE class="table" style="text-align:center"><THEAD><TR><TH colspan="2" style="text-align:center">Growth light</TH><TH colspan="2" style="text-align:center">High light</TH></TR></THEAD><TBODY><TR><TD>REL</TD><TD>SE</TD><TD>REL</TD><TD>SE</TD></TR><TR><TD>Phhl1</TD><TD>1</TD><TD>0.05</TD><TD>1.1</TD><TD>0.07</TD></TR><TR><TD>Pcop1</TD><TD>1</TD><TD>0.03</TD><TD>10.1</TD><TD>1.26</TD></TR><TR><TD>Ppif1</TD><TD>1</TD><TD>0.06</TD><TD>8.9</TD><TD>1.01</TD></TR><TR><TD>PphyB</TD><TD>1</TD><TD>0.09</TD><TD>15.6</TD><TD>1.36</TD></TR></TBODY></TABLE><DIV class="text_font">This graph shows the relative expression level of mRNA of gene hhl1 expressed by 4 different parts in growth light(100lux) and high light(1200lux).</DIV><DIV class="text_font">pCop1,pPif1,pPhyB do not respond to growth light, the expression level of mRNA is similar to that of pHhl1.However, under high light,three photosensitive promoters all respond specifically and the expression level of mRNA is obviously higher than that in pHhl1.(The transcription level of pCop1, pPif1 and pPhyB under highlight are 10.1,8.9 and 15.6 times of the mRNA level under growth light,respectively),suggesting that our parts can help produce more protein HHL1 to protect PSⅡ under high light.</DIV></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-sm-8 content"><H1>Demonstrate</H1><H2><STRONG>The trascription level of HHL1 in transgentic Arabidopsis</STRONG></H2><DIV class="text_font">In the complete transgenic Arabidopsis，the result is very similar. The transcription controlled by pHhl1 is also very low. However, under high light, the transcription  controlled by three photosensitive promoters are also very high, and the highest is phyB, and then is pCop1.</DIV><DIV class="text_font">These suggested that our parts can also function in the complete plants, and the result is similar.</DIV><DIV class="text_font">Fig.The transcription level of HHL1 in transgentic Arabidopsis</DIV><TABLE class="table" style="text-align:center"><THEAD><TR><TH colspan="2" style="text-align:center">Growth light</TH><TH colspan="2" style="text-align:center">High light</TH></TR></THEAD><TBODY><TR><TD>REL</TD><TD>SE</TD><TD>REL</TD><TD>SE</TD></TR><TR><TD>Phhl1</TD><TD>1</TD><TD>0.07</TD><TD>1.1</TD><TD>0.17</TD></TR><TR><TD>Pcop1</TD><TD>1</TD><TD>0.13</TD><TD>15.1</TD><TD>1.86</TD></TR><TR><TD>Ppif1</TD><TD>1</TD><TD>0.26</TD><TD>10.9</TD><TD>0.61</TD></TR><TR><TD>PphyB</TD><TD>1</TD><TD>0.19</TD><TD>20.6</TD><TD>1.96</TD></TR></TBODY></TABLE><H2><STRONG>Photosynthetic efficiency in transgentic Arabidopsis</STRONG></H2><DIV class="text_font">We transformed four vectors with stably expressions into Arabidopsis, then detected the photosynthetic efficiency of transgenic Arabidopsis using FluroCam. </DIV><DIV class="text_font">  1）Under  growth light  condition, the  chloroplast  of the plants that  transformed  in four different   vectors  were  not significantly  damaged, so the  red  fluorescence of the chloroplasts have no obvious difference and maintain a high level.</DIV><DIV class="text_font">  2）However, under high light condition, photosynthetic apparatuses were damaged in different levels although the protein  HHL1 exsist. So the red fluorescence of chloroplast had decreased, but the decrease level is quit different, the lower the fluorescence is, the stronger the damage is. </DIV><DIV class="text_font">From the diagram we know that the fluorescence of plants transformed with three photosensitive promoters are stronger than the promoter pHhl1, and the pPhyB is the strongest.And that plants transformed with promoter pPhyB, its photosensitive efficiency is the best and its photodamage is the lowest. </DIV><DIV class="text_font">Fig.Photosynthetic efficiency in transgentic Arabidopsis</DIV><TABLE class="table" style="text-align:center"><THEAD><TR><TH colspan="2" style="text-align:center">Growth light</TH><TH colspan="2" style="text-align:center">High light</TH></TR></THEAD><TBODY><TR><TD>Fv/Fm</TD><TD>SE</TD><TD>Fv/Fm</TD><TD>SE</TD></TR><TR><TD>Phhl1</TD><TD>0.82</TD><TD>0.02</TD><TD>0.65</TD><TD>0.017</TD></TR><TR><TD>Pcop1</TD><TD>0.825</TD><TD>0.013</TD><TD>0.72</TD><TD>0.0086</TD></TR><TR><TD>Ppif1</TD><TD>0.813</TD><TD>0.006</TD><TD>0.69</TD><TD>0.0161</TD></TR><TR><TD>PphyB</TD><TD>0.819</TD><TD>0.019</TD><TD>0.78</TD><TD>0.0216</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-sm-8 content"><H1>CONCLUSION</H1><DIV class="text_font">From the result above, we can know that, three photosensitive promoters can be activated by highlight to improve the expression of hhl1 and decrease photodamage, and the promoter phyB is the most efficient one. Therefore, in the plants, with our biobricks, the expression of hhl1 under highlight can be improved dramatically to protect PSⅡ and improve the the photosynthetic efficiency.</DIV></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-sm-8 content"><H1>CONCLUSION</H1><DIV class="text_font">1. Establish the mathematical model of how the  hhl1 transcription levels of four biobricks that change with light intensities . </DIV><DIV class="text_font">2. Apply our biobricks to other plants to see if they can be used to decrease photodamage in other plants as well </DIV></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-sm-8 content"><H1>FUTURE APPLICATION</H1><DIV class="text_font">1.Apply our idea in other plants to decrease the photodamage under highlight,especially in tropical and semitropical zone where the plants often suffer from highlight. </DIV><DIV class="text_font">2.To improve the resistance of plants ,applying our parts may improve the output of grain. </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>