<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Minnesota_Templates"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Minnesota/Templates">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Minnesota/Templates&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Minnesota/Templates&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Minnesota/Templates&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Minnesota/Templates" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Minnesota/Templates</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>remove style</TITLE></P><P><TITLE>UMN iGEM 2014</TITLE></P><DIV id="header"><UL id="menu"><LI data-menuanchor="Home" class="active"><A href="#Home">Home</A></LI><LI data-menuanchor="Idea" class="active"><A href="#Idea">Idea</A></LI><LI data-menuanchor="Project" class="active"><A href="#Project">Project</A></LI><LI data-menuanchor="Team" class="active"><A href="#Team">Team</A></LI><LI data-menuanchor="Policies" class="active"><A href="#Policies">Policies &amp; Practices</A></LI><LI data-menuanchor="Safety" class="active"><A href="#Safety">Safety</A></LI><LI data-menuanchor="Attributions" class="active"><A href="#Attributions">Attributions</A></LI><LI data-menuanchor="Medal" class="active"><A href="#goldmedal">Medal Requirements</A></LI><LI data-menuanchor="Sponsors" class="active"><A href="#Sponsors">Sponsors</A></LI></UL></DIV><DIV id="footer"><A href="https://igem.org/Main_Page" style="color:white;"> iGEM Home </A> | <A href="https://igem.org/2014_Judging_Form?id=1187" style="color:white;"> iGEM Judge-Click Here! </A> |  
<A href="http://www1.umn.edu/twincities/index.html" style="color:white;"> University of Minnesota Home </A> |  <A href="https://2014.igem.org/Team:Minnesota/Contact" style="color:white;"> Contact Us! </A></DIV><DIV id="fullpage"><DIV class="section " id="section0"><DIV class="intro"><H6>Minnesota iGEM 2014</H6></DIV></DIV><DIV class="section" id="section2"><DIV class="slide active" id="slide8" data-anchor="slide8"><H4> Project </H4><TABLE style="width:100%"><TBODY><TR><TD><UL><LI><LAB>  </LAB></LI><LABSECTION><LI><A href="https://static.igem.org/mediawiki/2014/a/ad/EncapsuLabNotebook.pdf">Dry Lab Notebook</A></LI></LABSECTION></UL></TD><TD><UL><A href="https://static.igem.org/mediawiki/2014/7/72/WetLabNotebookV2.pdf">Wet Lab Notebook</A></UL></TD></TR></TBODY></TABLE></DIV><DIV class="slide" id="slide1" data-anchor="slide1"><TABLE style="width: 100%; margin: 5px;"><TBODY><TR><H4>Mercury Bioremediation</H4>   0                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  
   0
 </TR><TR><TD><H3>
To design a system for biological remediation of not only mercury ions in contaminated waters, but also the more toxic form, methyl mercury, we’ve selected to use 5 genes of the mercury resistance (mer) operon of which over 10 genes have been identified and characterized in various strains of mercury resistant bacteria in the environment. This construct was assembled from the mer operon in Serratia marscecens in the plasmid pDU1358, and is designed to contain an upstream regulatory gene merR, two transport proteins merP (periplasmic) and merT (transmembrane), a gene encoding mercuric reductase MerA, and finally a gene encoding organomercurial lyase MerB. This system is regulated by a bidirectional promoter so that merR on one side of the operon is constitutively expressed and allows for the repression of the mer operon in the absence of mercury ions, and the downstream activation and transcription of merT, P, A, B when mercury ions are in close proximity. 
MerT and MerP were selected as transporters for their high turnover rates to bring in mercury ions, which are subsequently bound by MerA to catalyze their conversion into volatile mercury eventually captured within a carbon filter in our device and disposed of sustainably. The organic and more toxic form, methylmercury, can diffuse into the cytosol of the bacteria where MerB catalyzes its conversion into mercury ions, which are then bound to MerA and converted into less toxic, volatile elemental mercury in an NADP dependent reaction. The system is very tightly regulated and allows for continuous turnover within our bacterial chassis as the mercury ions are volatalized and then captured externally rather than sequestered within our bacteria which would eventually lead to cell death and the requirement to replace the cells. Due to the NADP requirement of MerA, the cells are required to stay alive throughout this process, accounted for this with novel cell encapsulation technology that allows the cells to remain viable and at the same time not in direct contact..?
Our system was tested in 3 different chassis: E. coli, Pseudomonas, and Shewanella, encapsulated and unencapsulated in the presence of either mercury chloride or methylmercury chloride and showed very promising results!  
</H3></TD><TD><H3>Fig 1 illustrating the relative positions of Mer proteins within the cell, modified mer operon. 
 click the individual genes to read more about our parts, click here to read more about our composite mer operon part</H3></TD><TD>000000000
</TD></TR></TBODY></TABLE></DIV><DIV class="slide" id="slide2" data-anchor="slide2"><H4>Biobrick Assembly</H4><H3>edit legend here
</H3></DIV><DIV class="slide" id="slide3" data-anchor="slide3"><TABLE style="width: 100%; margin: 5px;"><TBODY><TR><H4>Heavy Metal Bioprecipitation</H4></TR><TR><H3> Click the operon to be linked to the phsABC part page! </H3></TR><TR><TD><H3>The phsABC genes from Salmonella enterica serovar Typhimurium LT2 encode thiosulfate reductase, which catalyzes the stoichiometric production of hydrogen sulfide and sulfite from thiosulfate for heavy metal removal by precipitation. Within a separate bacterium from that containing the mer operon construct, This system allows us to extend our heavy metal bioremediation device to be applicable to a wide range of heavy metals in addition to mercury in both ionic and organic form.
The phsABC operon encodes three open reading frames (ORFs), designated phsA, phsB, and phsC. Based on sequence homology to formate dehydrogenase-N, it is predicted that thiosulfate reductase behaves in a similar fashion. The PhsA subunit is predicted to be a peripheral membrane protein active site bis(molybdopterin guanine dinucleotide) molybdenum (MGD) cofactor. PhsC is an integral membrane protein that anchors the other two subunits to the membrane, and contains the site for menaquinol oxidation and two heme cofactors located at opposite sides of the membrane. PhsB is predicted to possess four iron-sulfur centers that transfer electrons between PhsC and PhsA. 
was shown to have the highest catalytic activity in the IPTG- inducible plasmid pSB74. The part was used by the Yale 2010 iGEM Team (Part:BBa_K393000) (inducible by IPTG) to deposit copper sulfide in a specified geometry. We sought to both improve and characterize this part for future utilization in our filtration device by adding a modified lac promoter to allow for constitutive expression rather than IPTG induction within the biological system, and thus make it more applicable in the environment (Figure 5). We also improved the characterization of their part by testing its application for biological precipitation of iron and cadmium in addition to their copper testing to add to the functionality of the part.

</H3></TD><TD>&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;&amp;</TD></TR></TBODY></TABLE></DIV><DIV class="slide" id="slide4" data-anchor="slide4"><H4>Biobrick Assembly</H4><H3>Modified construct containing “phsABC”. The <I> phsABC </I>operon was amplified from the pSB74 plasmid and inserted into pSB1C3 (shipping vector) and the pBBRBB plasmid (for characterization) with the novel addition of a constitutive promoter.
</H3></DIV><DIV class="slide" id="slide5" data-anchor="slide5"><H4>Biosafety</H4><H3>
In response to our survey results showing that there was some concern about the bacteria escaping the device, we have designed two kill switches to address this concern and highlighted their advantages and disadvantages. Click either of the pictures below to view more information on each proposal.
</H3><TABLE style="width: 100%; margin: 5px;"><TBODY><TR><TD><H3> Kill Switch 1 </H3></TD><TD><H3> Kill Switch 2 </H3></TD></TR></TBODY></TABLE></DIV><DIV class="slide" id="slide6" data-anchor="slide6"><TABLE style="width: 100%; margin: 5px;"><TBODY><TR><H4>Mercury Ion Test Results (I)</H4></TR><TR><TD><H3>Fig Zones of Inhibition Test For Mercury Resistance. </H3></TD></TR><TR><TD><H3>
In this assay, Escherichia coli K12 expressing three different constructs were spread on agar plates to compare levels of mercury resistance. Each agar plate contained a filter disk spotted with 10µL of 0.1M HgCl2 in the middle. ++++++
</H3></TD></TR></TBODY></TABLE></DIV><DIV class="slide" id="slide7" data-anchor="slide7"><TABLE style="width: 100%; margin: 5px;"><TBODY><TR><H4>Mercury Ion Test Results (II)</H4></TR><TR><TD><H3>Fig Zones of Inhibition Test For Mercury Resistance. </H3></TD></TR><TR><TD><H3>
Zones of Inhibition Test For Mercury Resistance. In this assay, Escherichia coli K12 expressing three different constructs was spread on agar plates to compare levels of mercury resistance. Each agar plate contained a filter disk spotted with 10µL of 0.1M HgCl2 in the middle.
(A) Agar Plates of K12 with each construct. Left, K12 containing pBBRBB::mer (mer operon); center, K12 containing pBBRBB::gfp (vector control); right, K12 containing pBBRBB::merΔmerA(mer operon with merA deleted).
(B) Diameter of zones of inhibition. The diameter of the zone of inhibition was measured in triplicate. Green corresponds to pBBRBB::gfp, blue to pBBRBB::merRTPAB, and red to pBBRBB::merΔmerA. Individual constructs range from highest resistance in the following order: merRTPAB &gt; vector control &gt;merΔmerA.  Our results show that both the recombinant E. coli and Pseudomonas putida containing the pBBRBB::mer plasmid were able to grow significantly closer to the HgCl2 disc than the negative control (pBBRBB::GFP). The E.coli strain containing the merA deletion plasmid could not survive as close to the heavy metal spotted disc as the negative control. This is likely due to the fact that it still contains the transport proteins MerP and MerT, and therefore transports the toxic mercury ions into the cell without remediating it due to the lack of MerA, causing cell death. On the other hand, the pBBRBB::GFP control was not able to take up any mercury ions.
</H3></TD></TR></TBODY></TABLE></DIV><DIV class="slide" id="slide8" data-anchor="slide8"><H4>Methylmercury Results</H4><TABLE style="width: 100%; margin: 5px;"><TBODY><TR><TD>   0   
</TD><TD>   00000000000000000000000000000000000000000000
</TD></TR><TR><TD>
0000000000
</TD><TD><H3>Fig . </H3></TD><TD><H3>1st MeHg data
In addition to testing 2 biological replicates of &quot;E.coli&quot; and &quot;Pseudomonas&quot; strains carrying the “pBBRBB::mer” plasmid, strains containing the “pBBRBB::gfp” within encapsulation beads were tested as a negative control within encapsulation beads. LB (abiotic) as well as LB (abiotic) containing 0.5g encapsulation beads were also used as negative controls. Unencapsulated bacteria of each strain were also tested to compare the efficiency of those cells in direct contact with MeHg to those within the encapsulation beads.

Experiments were conducted by adding methylmercury chloride to 7 mL of LB at a final concentration of 1 mg/L. The methylmercury levels were analyzed at the start of the experiment and after 36 hours. At each time point the samples were diluted a million-fold before taking measurements with a Tekran model 2700 Automated Methyl Mercury Analyzer using EPA method 1630 without distillation. This is a highly sensitive and ultra-stable cold vapor atomic fluorescence spectrometry (CVAFS) Hg detector. All quality assurance and quality control measures were taken as outlined in EPA method 1630. All MeHg standards (ongoing precision recoveries) were within the acceptable range averaging 96%.

The samples showed growth of both encapsulated and unencapsulated “E. coli” and “Pseudomonas” carrying “pBBRBB::mer” after 36 hours, and complete demethylation of the 1mg MeHg in both the encapsulated and unencapsulated samples. No cell growth was observed in strains containing the “pBBRBB::gfp”negative control, LB (abiotic) or LB (abiotic) containing 0.5g empty encapsulation beads, and methylmercury levels remained relatively unchanged. This preliminary run showed that methylmercury levels could drop to undetectable levels in the presence of our bacteria (both E.coli and Pseudomonas) but not in their absence.



</H3></TD></TR></TBODY></TABLE></DIV><DIV class="slide" id="slide9" data-anchor="slide9"><H4> Methylmercury Results (II) </H4><H3>
2nd MeHg- 
- It was expected that the rate of demethylation would increase exponentially, and thus we took samples at time points that would reflect that rate (Fig __). 
Within 24 hours, methylmercury levels dropped slightly  in the abiotic LB sample, most likely only due to photolysis in the presence of light
           - “E.coli” cells with the GFP control plasmid absorbed (number - abiotic)  mg into the cells before their eventual cell death.
            - abiotic encapsulation beads absorbed as much methylmercury from the surroundings as the E.coli cells with the GFP control plasmid. 
           - E.coli cells with the mer operon remediated all of the methylmercury (1 mg in 1L) in the samples within approximately 5 hours. 
           &quot;Pseudomonas” could not survive or decrease methylmercury levels in the samples after 24 hours. We suspect that the “Pseudomonas” require a longer time for the construct to be activated, since they were able to grow and decrease methyl mercury to below detectable levels twithin 36 hours. 

The &quot;E.coli&quot; cells containing the mer operon were tested in both 1mg/L and 4mg/L of methylmercury chloride. (48 samples)
	MeHg was only demethylated by 1 mg before reaching a stationary level. We suspect that this concentration was too high for cell survival that initial demethylation of MeHg occurred due to the presence of enzymes within the cells after being grown overnight and before their addition to the sample tubes. Thus MeHg levels decreased until the NADP pool within the cell was depleted and MerA was no longer able to remediate mercury ions resulting from demethylation, leading to cell death. 
</H3></DIV><DIV class="slide" id="slide11" data-anchor="slide11"><TABLE style="width: 100%; margin: 5px;"><TBODY><TR><H4>Encapsulation</H4><TD>   0                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               
</TD><TD>   0
 </TD></TR><TR><TD><H3>
The main goal of EncapsuLab was to create a system for the preservation and protection of the bacteria in our system,
 as well as physically separating them from the outside environment. To achieve this, we created a water-porous 
 silica matrix using techniques developed by the Aksan and Wackett labs at the U of M. Furthermore, we developed 
 a device to work as a proof of concept for the use of encapsulated bacteria in a real water-cleaning system. 
In addition to this, we conceptualized a scaling-up of our system for larger water-cleaning problems. Lastly, 
we developed a mathematical model to compare our experimental data in order to better understand the biochemical networks behind our work.
</H3></TD></TR><TR><TD><H2>
Title1
</H2></TD></TR><TR><TD><H4>
Yoloswag
</H4></TD></TR></TBODY></TABLE></DIV><DIV class="slide" id="slide12" data-anchor="slide12"><DIV id="title"><H4>Encapsulation Procedure</H4></DIV><H3>To make a silica matrix surrounding a cell, we needed a supply of silica nanoparticles and a reactive cross-linker capable of covalently bonding reagents into a unified volume. Two solutions were prepared: one was a combination of TM-40 (colloidal silica) and diluting components (ultrapure water and polyethylene glycol). The second solution was tetramethyl orthosilicate (TMOS) cross-linker. In slightly acidic conditions, methyl groups of TMOS are hydrolyzed, producing methanol and the silica cross-linker. Resuspended and washed cells are then added to the relatively innocuous colloidal solution. Upon combining these two mixtures, a rapid polymerization linking colloidal silica together occured. In order to achieve proper bead sphericity, we combined all of these components in an hydrophobic solution and let them set. After some experimentation, we were able to produce beads with a consistent size. To purify the mixture, we phase-separated the silica beads by adding water to solution, which forced them into the aqueous phase. These were then recovered to assess viability.
</H3></DIV><DIV class="slide" id="slide13" data-anchor="slide13"><TABLE style="width: 100%; margin: 5px;"><TBODY><TR><H4>The Biochemical Network</H4></TR><TR><TD><H3>In this project, systems of mathematical equations are used to model the biochemical network of methylmercury conversion in E. coli and to determine the effects of changing methylmercury concentration on the conversion rate. A combination of enzyme kinetics and protein expression equations was in the model. The reactions below shows the four-step reaction network for the conversion of methylmercury to volatile mercury using the mer operon in E. coli. This network involves the transport of methylmercury into the cell, its subsequent conversion into ionic and volatile mercury, and the transport of volatile mercury to the extracellular space. This simplified network follows the proposed mechanism of the mer operon and it captures the primary steps of methylmercury breakdown in E. coli. Furthermore, a system of ordinary differential equations was used to model the biochemical reactions taken place at each step of the conversion process. The cofactors were not included in the equations because at sufficient concentrations, they do not significantly alter the rates.</H3></TD></TR></TBODY></TABLE></DIV><DIV class="slide" id="slide14" data-anchor="slide14"><H4>Mathematical Modeling</H4><H3>The equations used in conversion steps 3 and 4 are extracted from studies done by Maria on ionic mercury uptake by E. coli cells (Maria, 2010). These equations can be used because ionic mercury is converted into elemental mercury which in turn diffuses into the extracellular space; these two steps were described in the Maria paper. It is assumed that methylmercury and ionic mercury have similar orders of magnitude of transport rate into the cell. 
Since step 2 is not explicitly included in the Maria paper, a different approach was used to determine the equation. During our literature search, we did not find any primary publications that specify the methylmercury conversion behavior of both MerA and MerB. Our pBBRBB plasmid is designed in the way that the mer genes are induced in the presence of methylmercury and ionic mercury. Through our literature search, we were able to calculate the steady state concentrations of basal and induced states based on the presence of methlymercury and ionic mercury in the cytosolic space. Therefore, we used the below equation to extrapolate the reaction rates of both the mer proteins. For details, please see Supplemental Materials A.
The above equations show the reaction rates of the mer proteins under induced steady state. To represent the full MerA and MerB expression, we include terms for the basal protein expression, the induction of the Mer proteins, and the degradation of proteins. For details on the calculations, please refer to Supplemental Materials A. We have determined from our calculations that Va and Vb resemble Vmax in the equations. 
</H3></DIV><DIV class="slide" id="slide15" data-anchor="slide15"><H4>Sensitivity Analysis</H4><H3>We performed sensitivity analysis to give us insight into which parameters will have the greatest effect on each species in the reaction equations. We calculated the local derivatives of each species with respect to the parameters to determine how much the parameters affected the result. The concentration of methylmercury in the cytoplasm was found to be most sensitive to the V and K parameters of MerB. Additionally, the concentration of Hg in the extracellular space and cytosol were both most sensitive to the V and K parameters of MerA. We can use this sensitivity analysis as a guide to what parameters are most useful to improve. If we want the device to be more efficient at methylmercury conversion, efforts should be directed towards improving the enzymatic activity of MerA and MerB. 
</H3></DIV><DIV class="slide" id="slide16" data-anchor="slide16"><H4> Device Design </H4><TABLE style="width: 100%; margin: 5px;"><TBODY><TR><TD><H3>
This device incorporates two key elements of mechanical engineering design: simplicity and functionality. Key concepts from fluid mechanics are applied to pump water into and out of the encapsulation as efficiently as possible. The system can be placed in or out of the water, which makes it easy to use in any situation. The sleek aluminum casing offers a durable resistance to the ever-changing surroundings and environment. Overall, the simplicity of the design makes it user-friendly and adaptable. 
</H3></TD></TR><TR><TD><H3>
Our design is very similar to commonly used fixed-bed adsorption and ion exchange columns in that it involves flow of a fluid down through a packed bed that removes a solute. However, an important difference is those types of columns contain chemical resins or adsorbents that can become saturated (i.e. reach capacity) as they adsorb solute. Since we use encapsulated bacteria whose &quot;capacity&quot; is only limited their lifetime, our device has the added benefit that the mass-transfer zone (the part of the column where the solute is removed) doesn't move down the column due to saturation. This also means that we don't need to alternate between adsorption and regeneration cycles.
</H3></TD></TR></TBODY></TABLE></DIV><DIV class="slide" id="slide17" data-anchor="slide17"><H4> Scalability </H4></DIV></DIV><DIV class="section" id="section3a"><DIV id="title"><H4>Meet The Team</H4></DIV></DIV><DIV class="section" id="section4"><DIV class="slide" id="PnPslide1" data-anchor="PnPslide1"><H4>Policies &amp; Practices</H4><H3>Our Policy and Practices approach this year has focused on establishing an effective two-way discourse between our team and the public in order to inform our design, educate the public, and illuminate ethical issues related to both our project and integration of public opinion into project implementation. We focused our discussion on safety, ethics, sustainability, and intellectual property in order to design a project that fits the needs of the public while maintaining technical viability and high potential for commercialization. Our team did extensive educational outreach in addition to educating ourselves on public perception of our work. We used this input to inform our device design such that it could be implemented in a way that addresses the major concerns of both scientists and consumers. We also took steps to protect our intellectual property and explore the patenting and commercialization process of our product. Finally, our team sought to investigate ethical issues brought up by our discourse with the public by discussing both our work and ethics related to the project with a variety of experts and ethicists.

</H3></DIV><DIV class="slide" id="slide2" data-anchor="slide2"><H4>Educational Outreach</H4><H3>Building on past successes, our team has been devoted to volunteering our services to the community in a number of educational venues. The team took our curriculum, first developed in 2013, and improved the structure and delivery of our lesson plans in the hopes of encouraging awareness and education on topics in synthetic biology. Since 2013 our educational outreach group ECORI (Educating Communities On Research Innovation) has taught our <A href="https://static.igem.org/mediawiki/2014/a/ab/Curriculumhandbook.pdf">original, interactive curriculum</A> to over 200 students (K-12) and their teachers. This year we also created a mobile exhibit form of our curriculum along with a layman’s introduction to our project that we displayed on over half a dozen weekends to visitors of all ages at the Science Museum of Minnesota. Our curriculum has also been brought to several other STEM fairs and family fun events in the Twin Cities area including the 3M Science Day Fair for 3M employees and their families, UMN Biodiversity Fair, CSE Family Fun Fair, and the Middle School STEM Fair hosted by the Association of Multicultural Students at UMN. Finally, the team designed a Synthetic Biology Game Show that was presented on stage with 30 participants at the Minnesota State Fair to assess the general public’s knowledge of the subject and teach hundreds of passers-by in a way that was both engaging and interactive. Winners were rewarded with reusable bags, magnets, and gift cards donated by our sponsors. In the spirit of science, our curriculum has been ever evolving to constantly address salient topics and educational materials. The variable versions of our curriculum allow it to be flexible and practical in various settings.

 </H3></DIV><DIV class="slide" id="slide3" data-anchor="slide3"><H4 style="padding: -20px;">Public Perception</H4><H3> Our team sought to inform the majority stakeholders in our community concerning the scope of our project. This year our team chose to have an exhibit catered towards adult residents at the Minnesota State Fair (the largest statewide annual gathering with over 1.8 million visitors each year) to learn how we can best design our technology to meet the needs and concerns of the people whose waters we hope to bioremediate. We delivered a short synopsis of our device, the synthetic biology involved, and safety precautions we have outlined for our project. We then presented visitors with a five question survey using a Likert Scale to gauge public perception of both our device, and the synthetic biology methods used. The survey was a huge success with over 320 participants. With such a diverse attendance, our survey captured a great cross-section of the Minnesota community that would be impacted by the implementation of our device. The results of our survey, illustrated below, informed how and where the public would be most comfortable with implementing our device, and illustrated the need for catered education addressing the public’s major concerns prior to applying our device in the environment. Our model for gauging public perception allowed for a wide, diverse crowd to be accessed. This model can be used upon request.


 </H3></DIV><DIV class="slide" id="slide4" data-anchor="slide4"><H4>Intellectual Property</H4><H3>
Members of our team attended three Intellectual Property Protection and patenting workshops that educated them on the process of commercialization of our invention. This allowed us to develop a comprehensive business plan and perform an economic analysis of the mining, fisheries, and governmental markets that could potentially benefit from the use of our invention. We also worked in conjunction with the Office of Technology Commercialization to explore the patentability of our project, the novelty, the non obviousness, and utility of our product and how to make the best claims to patent our device or license it to Minnepura Technologies, Inc.
</H3><TABLE style="width: 100%; margin: 5px;"><TBODY><TR><H3> Basem and Patrick meet with patent attorneys at the Office for Technology Commercialization </H3><TD><H3> Kill Switch 1 </H3></TD></TR><TR><H3> Kill Switch 2 </H3></TR></TBODY></TABLE></DIV><DIV class="slide" id="slide5" data-anchor="slide5"><H4>Documentary</H4><H3>
In order for our project to reach commercialization and success with the public, we needed to inform ourselves as well as others of the ethics of the use of our invention. We compiled this documentary in order to inform those unfamiliar with the problem of global mercury contamination and to discuss the bioethical questions of synthetic biology as they related to our device implementation. We conducted interviews with specialists from environmental toxicology, biotechnology, and philosophy. Through our collaboration with the 2014 Colombia iGEM Team, we further examined the current mercury contamination in Colombia and around the globe.
            </H3></DIV></DIV><DIV class="section" id="section5"><DIV class="slide" id="safetySlide1"><H4>Safety in the Lab</H4><H3><B>Our Training </B> 
                 DEHS Introduction: Research Safety 
                 
                 DEHS Chemical Safety
                 
                 DEHS Waste Management
            </H3></DIV><DIV class="slide" id="safetySlide2"><H4>Safety in the Lab</H4><H3>2. Our Local Rules and Regulations
                   Who is responsible for biological safety at our institution? 
                   The project was discussed with the Department of Environmental Health and Safety at our university, and a plan was devised for mercury waste disposal based on their input. General biosafety guidelines found at https://www.dehs.umn.edu/bio.htm, http://www.dehs.umn.edu/bio_pracprin.htm and http://www.cdc.gov/biosafety/publications/bmbl5/bmbl.pdf were followed.</H3></DIV><DIV class="slide" id="safetySlide3"><H4>Safety in the Lab</H4><H3>3. Risks of Your Project Now
To mitigate risks to the safety and health of team members, or other people working in the lab:
Gloves are used in any protocol that utilizes Ethidium Bromide, including gel electrophoresis.
Lab coat, gloves, and full face shields are used when cutting gel fragments in proximity of ultraviolet light.

For handling mercury, a lab coat, inner and outer (long cuffed) nitrile gloves, lab goggles and face shields will be used, and used materials will be disposed of by the University of Minnesota Department of Environmental Health and Safety.
There are assigned incubators, hoods, and disposal containers for hazardous materials like mercury chloride.
Design features to minimize risk

kill switch proposal that would not allow the bacteria to survive outside of the encapsulation or even device holding the cells
Air tight?
non pathogenic lab strains

</H3></DIV></DIV><DIV class="section" id="section6"><DIV id="title"><H4>Attributions</H4></DIV></DIV><DIV class="section" id="section7"><H4>Gold Medal Requirements</H4><H3>
UMN iGEM has strived to produce Gold Medal level work through the duration of our project. Here we outline our work that specifically pertains to each of the gold medal requirements. Scroll to the right to see how we met each requirement!</H3></DIV><DIV class="slide" id="goldslide2"><H4>Requirement 1: Improving function or characterization of an existing part</H4><H3>
To compliment the UMN iGEM 2014 bioremediation project, we chose to improve the &quot;phsABC&quot; biological system first added to the registry by the Yale 2010 team (BBa_K393001). The operon produces hydrogen sulfide to allow for the biological precipitation of heavy metals, such as copper used for the Yale team's biological circuit. We sought to both improve and characterize the part for future utilization in our filtration device. To improve &quot;phsABC&quot;, we added a modified lac promoter to allow for constitutive expression rather than IPTG induction within the biological system, and thus make it more applicable in a real world environmental setting where an induction strategy would be costly and inefficient for remediation purposes. We also improved the characterization of their part by testing its application for biological precipitation of Iron and Cadmium in addition to the Yale team's Copper testing to add to the functionality of the part.. [modify if we get to testing]
</H3></DIV><DIV class="slide" id="goldslide3"><H4>Requirement 2: Collaborations with other teams</H4><H3> 
The UMN collaborated with Wisconsin Lutheran to facilitate a week’s worth of synthetic biology curriculum by sending detailed lesson plans developed by our iGEM team, along with strains of microorganisms used to facilitate trainings. In addition, we participated in Columbia iGEM's low-budget lab exercise, who in turn provided UMN iGEM with an interview with a local environmental health specialist who has extensive knowledge of the mercury contamination problem within Columbia. 

</H3></DIV><DIV class="slide" id="goldslide4"><H4>Requirement 3: Describing and evaluating questions beyond the bench</H4><H3>

The primary goal of UMN iGEM 2014 has been to design a project in line with the major safety, ethics, and intellectual property concerns of both scientists and the public. We took the following steps to address each of these issues:<B>Safety</B>: Our project gives detailed attention to the prevention of releasing genetically modified bacteria into the environment via encapsulation and the development of several killswitch proposals. The device also includes an activated carbon filter to both collect mercury and prevent release of the bacteria. During an outreach event at 3M we also discussed this filtration element of our device with one of their filtration experts to further inform our safety measures.<B>Ethics</B>: We have been committed to establishing an open and productive dialogue between UMN iGEM and the public. Our surveying at the State Fair of public opinion on safety and implementation concerns of our device helped to inform our design. We believe there is great ethical importance to involving the community in the design of a device that could potentially be used on their waters. We also developed an interactive game show at the fair to actively engage the public in synthetic biology knowledge. Our dialogue with the public has also included our outreach program at local schools, the Science Museum of Minnesota, and various STEM fairs allowing us to develop a dynamic dialogue with both children and their parents. The program focuses on the understanding of DNA in order to demystify the basic principles behind synthetic biology. It is our hope that a more solid understanding of these principles may help to familiarize children with biotechnological tools and decrease misconceptions about the nature of DNA manipulation. We hope this may help to develop an informed future community of consumers and constituents to facilitate open dialogues between scientists and the public. Finally, we interviewed various experts in bioethics and scientific fields related to our project to help further inform ourselves and give essential background to those unfamiliar with the global issue of mercury contamination.<B>Intellectual Property</B>: We sought to create an integrative and marketable project by developing a business plan along with marketing analysis for our device. We also educated ourselves about the patenting process via on campus seminars. Finally, we applied for a patent to protect the intellectual property of our team in order to help protect us during any future implementation of the developed business plan.</H3></DIV></DIV><DIV class="section" id="section8" data-anchor="Medal"><DIV id="title"><H4>Sponsors</H4></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Minnesota/Templates">http://2014.igem.org/Team:Minnesota/Templates</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Minnesota/Templates" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Minnesota/Templates" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Minnesota/Templates&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Minnesota/Templates&amp;oldid=386914" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>