<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Minnesota"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Minnesota">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Minnesota&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Minnesota&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Minnesota&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Minnesota" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Minnesota</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>remove style</TITLE></P><P><TITLE>UMN iGEM 2014</TITLE></P><DIV id="header"><UL id="menu"><LI data-menuanchor="Home" class="active"><A href="#Home">Home</A></LI><LI data-menuanchor="Idea" class="active"><A href="#Idea">Idea</A></LI><LI data-menuanchor="Project" class="active"><A href="#Project">Project</A></LI><LI data-menuanchor="Team" class="active"><A href="#Team">Team</A></LI><LI data-menuanchor="Policies" class="active"><A href="#Policies">Policies &amp; Practices</A></LI><LI data-menuanchor="Safety" class="active"><A href="#Safety">Safety</A></LI><LI data-menuanchor="Attributions" class="active"><A href="#Attributions">Attributions</A></LI><LI data-menuanchor="Medal" class="active"><A href="#Medal">Medal Requirements</A></LI><LI data-menuanchor="Sponsors" class="active"><A href="#Sponsors">Sponsors</A></LI></UL></DIV><DIV id="footer"><A href="https://igem.org/Main_Page" style="color:white;"> iGEM Home </A> | <A href="https://igem.org/2014_Judging_Form?id=1420" style="color:white;"> iGEM Judge-Click Here! </A> |  
<A href="http://www1.umn.edu/twincities/index.html" style="color:white;"> University of Minnesota Home </A> |  <A href="https://2014.igem.org/Team:Minnesota/Contact" style="color:white;"> Contact Us! </A></DIV><DIV id="fullpage"><DIV class="section " id="section0"><DIV class="intro"><H6>Minnesota iGEM 2014</H6></DIV></DIV><DIV class="section" id="section2"><DIV class="slide active" id="safetySlide3" data-anchor="safetySlide3"><H4> Project </H4><TABLE style="width:100%"><TBODY><TR><TD><UL><LI><LAB>  </LAB></LI><LABSECTION><LI><A href="https://static.igem.org/mediawiki/2014/a/ad/EncapsuLabNotebook.pdf">Dry Lab Notebook</A></LI></LABSECTION></UL></TD><TD><UL><A href="https://static.igem.org/mediawiki/2014/7/72/WetLabNotebookV2.pdf">Wet Lab Notebook</A></UL></TD></TR></TBODY></TABLE></DIV><DIV class="slide" id="slide1" data-anchor="slide1"><TABLE style="width: 100%; margin: 5px;"><TBODY><TR><H4>Mercury Bioremediation</H4>   0                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  
   0
 </TR><TR><TD><H3>
To design a system for biological remediation of not only mercury ions in contaminated waters, but also the more toxic form, methylmercury, we’ve selected to use 5 genes of the mercury resistance (<I>mer</I>) operon of which over 10 genes have been identified and characterized in various strains of mercury resistant bacteria in the environment. This construct was assembled from the mer operon in <I>Serratia marscecens</I> in the plasmid pDU1358, and is designed to contain an upstream regulatory gene <I>merR</I>, two genes encoding for transport proteins <I>merP</I> (periplasmic) and <I>merT</I> (transmembrane), a gene encoding mercuric reductase MerA, and finally a gene encoding organomercurial lyase MerB. This system is regulated by a bidirectional promoter so that ''merR'' on one side of the operon is constitutively expressed and allows for the repression of the mer operon in the absence of mercury ions, and the downstream activation and transcription of ''merT, merP, merA, merB'' when mercury ions are in close proximity. 
MerT and MerP were selected as transporters for their high turnover rates to bring in mercury ions, which are subsequently bound by MerA to catalyze their conversion into volatile mercury eventually captured within a carbon filter in our device and disposed of sustainably. The organic and more toxic form, methylmercury, can diffuse into the cytosol of the bacteria where MerB catalyzes its conversion into mercury ions, which are then bound to MerA and converted into less toxic, volatile elemental mercury in an NADP(H) dependent reaction. The system is very tightly regulated and allows for continuous turnover within our bacterial chassis as the mercury ions are volatalized and then captured externally rather than sequestered within our bacteria which would eventually lead to cell death and the requirement to replace the cells. Due to the NADP(H) requirement of MerA, metabolically active cells are required throughout this process. We accomplished this via novel cell encapsulation technology that keeps cells remain viable and at the same time not in direct contact.
Our system was tested in 3 different chassis: ''E. coli, Pseudomonas'', and ''Shewanella'', encapsulated and unencapsulated in the presence of either mercury chloride or methylmercury chloride and showed very promising results!  
</H3></TD><TD><H3>Figure 1. Relative positions of Mer proteins within the cell (bottom) and the modified ''mer'' operon (top).
 click the individual genes to read more about our parts, 
<A href="http://parts.igem.org/Part:BBa_K1420000 " style="color:white;"> click here to read more about our composite ''mer'' operon part</A></H3></TD></TR></TBODY></TABLE></DIV><DIV class="slide" id="slide2" data-anchor="slide2"><H4>Biobrick Assembly</H4></DIV><DIV class="slide" id="slide3" data-anchor="slide3"><TABLE style="width: 100%; margin: 5px;"><TBODY><TR><H4>Heavy Metal Bioprecipitation</H4></TR><TR><A href="http://parts.igem.org/Part:BBa_K1420006"><H3> Click the operon to be linked to the phsABC part page! </H3></A></TR><TR><TD><H3>The phsABC genes from Salmonella enterica serovar Typhimurium LT2 encode thiosulfate reductase, which catalyzes the stoichiometric production of hydrogen sulfide and sulfite from thiosulfate for heavy metal removal by precipitation. Within a separate bacterium from that containing the mer operon construct. This system allows us to extend our heavy metal bioremediation device to be applicable to a wide range of heavy metals in addition to mercury in both ionic and organic form.
The phsABC operon encodes three open reading frames (ORFs), designated phsA, phsB, and phsC. Based on sequence homology to formate dehydrogenase-N, it is predicted that thiosulfate reductase behaves in a similar fashion. The PhsA subunit is predicted to be a peripheral membrane protein active site bis(molybdopterin guanine dinucleotide) molybdenum (MGD) cofactor. PhsC is an integral membrane protein that anchors the other two subunits to the membrane, and contains the site for menaquinol oxidation and two heme cofactors located at opposite sides of the membrane. PhsB is predicted to possess four iron-sulfur centers that transfer electrons between PhsC and PhsA. 
was shown to have the highest catalytic activity in the IPTG- inducible plasmid pSB74. The part was used by the Yale 2010 iGEM Team (Part:BBa_K393000) (inducible by IPTG) to deposit copper sulfide in a specified geometry. We sought to both improve and characterize this part for future utilization in our filtration device by adding a modified lac promoter to allow for constitutive expression rather than IPTG induction within the biological system, and thus make it more applicable in the environment. We also improved the characterization of their part by testing its application for biological precipitation of iron and cadmium in addition to their copper testing to add to the functionality of the part.

</H3></TD></TR></TBODY></TABLE></DIV><DIV class="slide" id="slide4" data-anchor="slide4"><H4>Biobrick Assembly</H4><H3>Modified construct containing “phsABC”. The <I> phsABC </I>operon was amplified from the pSB74 plasmid and inserted into pSB1C3 (shipping vector) and the pBBRBB plasmid (for characterization) with the novel addition of a constitutive promoter.
</H3></DIV><DIV class="slide" id="slide5" data-anchor="slide5"><H4>Biosafety</H4><H3>
In response to our survey results showing that there was some concern about the bacteria escaping the device, we have designed two kill switches to address this concern and highlighted their advantages and disadvantages. Click either of the pictures below to view more information on each proposal.
</H3><TABLE style="width: 100%; margin: 5px;"><TBODY><TR><TD><A href="https://static.igem.org/mediawiki/2014/b/b0/Sarah%27s_Killswitch_with_Pictures_2_%281%29.pdf"><H3> Kill Switch 1 </H3></A></TD><TD><A href="https://static.igem.org/mediawiki/2014/b/bc/David%27s_Killswitch_with_Picture_2.pdf"><H3> Kill Switch 2 </H3></A></TD></TR></TBODY></TABLE></DIV><DIV class="slide" id="slide6" data-anchor="slide6"><TABLE style="width: 100%; margin: 5px;"><TBODY><TR><H4>Mercury Ion Test Results (I)</H4></TR><TR><TD><H3>Zones of Inhibition Test For Mercury Resistance. </H3></TD></TR><TR><TD><H3>
In this assay, Escherichia coli K12 expressing three different constructs were spread on agar plates to compare levels of mercury resistance. Each agar plate contained a filter disk spotted with 10µL of 0.1M HgCl2 in the middle allowing the mercury ions to diffuse throughout the media. The E. coli strain containing the modified mer operon showed comparable results to the positive control with the original pDU1358, as they both grew very closely to the filter disc. pBBRBB::GFP negative control showed growth significantly further away from the mercury disc due to its inability to detoxify mercury ions.
</H3></TD></TR></TBODY></TABLE></DIV><DIV class="slide" id="slide7" data-anchor="slide7"><TABLE style="width: 100%; margin: 5px;"><TBODY><TR><H4>Mercury Ion Test Results (II)</H4></TR><TR><TD><H3>Zones of Inhibition Test For Mercury Resistance. </H3></TD></TR><TR><TD><H3>
Zones of Inhibition Test For Mercury Resistance. In this assay, Escherichia coli K12 expressing three different constructs was spread on agar plates to compare levels of mercury resistance. Each agar plate contained a filter disk spotted with 10µL of 0.1M HgCl2 in the middle.
(A) Agar Plates of K12 with each construct. Left, K12 containing pBBRBB::mer (mer operon); center, K12 containing pBBRBB::gfp (vector control); right, K12 containing pBBRBB::merΔmerA(mer operon with merA deleted).
(B) Diameter of zones of inhibition. The diameter of the zone of inhibition was measured in triplicate. Green corresponds to pBBRBB::gfp, blue to pBBRBB::merRTPAB, and red to pBBRBB::merΔmerA. Individual constructs range from highest resistance in the following order: merRTPAB &gt; vector control &gt;merΔmerA.  Our results show that both the recombinant E. coli and Pseudomonas putida containing the pBBRBB::mer plasmid were able to grow significantly closer to the HgCl2 disc than the negative control (pBBRBB::GFP). The E.coli strain containing the merA deletion plasmid could not survive as close to the heavy metal spotted disc as the negative control. This is likely due to the fact that it still contains the transport proteins MerP and MerT, and therefore transports the toxic mercury ions into the cell without remediating it due to the lack of MerA, causing cell death. On the other hand, the pBBRBB::GFP control was not able to take up any mercury ions.
</H3></TD></TR></TBODY></TABLE></DIV><DIV class="slide" id="slide8" data-anchor="slide8"><H4>Methylmercury Results</H4><TABLE style="width: 100%; margin: 5px;"><TBODY><TR><TD><H3>
Methylmercury degradation rates were measured for ‘’E.coli K12’’ strains carrying the pBBRBB::’’mer'' plasmid compared to a vector control (pBBRBB::’’gfp’’), abiotic encapsulation beads, and abiotic medium. Negative controls were included to determine the amount of methylmercury absorbed by the beads and abiotic degradation rate of methyl mercury.
Methylmercury chloride was added to all tubes at a final concentration of 1 mg/L. Methylmercury concentrations were measured over a period of 24 hours. At each time point the samples were diluted a million-fold before taking measurements with a Tekran model 2700 Automated Methyl Mercury Analyzer.
’’E.coli K12’’ strains carrying the pBBRBB::’’mer'' plasmid degraded methylmercury to levels below detection of the Tekran Analyzer after approximately 4 hours while the vector control showed only a slight decrease (similar to that for abiotic beads). Low amounts of degradation in abiotic medium are likely due to photolysis of the organomercury. 



</H3></TD></TR></TBODY></TABLE></DIV><DIV class="slide" id="slide9" data-anchor="slide9"><H4> Methylmercury Results (II) </H4><TABLE style="width: 100%; margin: 5px;"><TBODY><TR><TD>   0   
</TD></TR><TR><TD>
0000000000
</TD><TD><H3>


Unencapsulated &quot;E.coli&quot; cells containing the mer operon were tested in both 1mg/L and 4 mg/L starting concentrations of methylmercury chloride to provide degradation rate information for the mathematical modelers. Methylmercury was only degraded in the 1 mg/L cultures and 4 mg/L was toxic to all cells tested. We suspect that this concentration was too high for cell survival and that initial demethylation of methylmercury occurred due to the presence of Mer enzymes from leaky expression in overnight cultures. 


</H3></TD></TR></TBODY></TABLE></DIV><DIV class="slide" id="slide10" data-anchor="slide10"><H4> Heavy Metal Bioprecipitation Results </H4><TABLE style="width: 100%; margin: 5px;"><TBODY><TR><H3>
In order to measure thiosulfate reducing activity of phsABC of NaS2O3 to H2S, the operon was first inserted into the pBBRBB vector with a constitutive Plac promoter and transformed into E. coli K12. As a negative control, pBBRBB::gfp was tested under the same conditions. The pBBRBB::phsABC K12 and pBBRBB::gfp K12 cells were grown in three test tubes each containing heavy metal tryptone medium as well as 3mM NaS2O3. A third set of test tubes were set up with the same contents except without cells as an additional negative control. After 24 hours of incubation, The exact amount of H2S present in each of three different sets of tubes was then measured using a hydrogen sulfide assay designed by J.D. Cline in 1968 to determine hydrogen sulfide concentrations in natural waters. This consisted of adding 1x or 0.5x 30μL of Cline's Reagent (2g Diamine + 3g FeCl3 in 50mL of 50% cool HCl) to 270μL of sample. The results were tested against a known standard curve of various Na2S concentrations. Each sample was allowed 20 minutes for the color to develop before being diluted 1:10 with water for testing. The plate was then read at 670nm with the numerical results displayed. The bar graph shows that the sulfide concentration was considerable higher for the cultures containing pBBRBB:phsABC (380.1 μM ± 13.5) compared to the pBBRBB::gfp negative control (118.9μM ± 1.1). These results are in line with those seen by the Keasling lab. Following sulfide measurements, cadmium chloride was added (200 μM), and cells were allowed to incubate without shaking at 37C overnight. Cells were pelleted to look for a color change indicating precipitation of CdS.  Cell pellets for K12 expressing phsABS were yellow/brown indicating precipitation of CdS while the vector control cells (pBBRBB::gfp) remained white. 
</H3></TR></TBODY></TABLE></DIV><DIV class="slide" id="slidephs" data-anchor="slidephs"><H4> Heavy Metal Bioprecipitation Results (II) </H4><H3>A second set of experiments was also conducted with pBBRBB:phsABC K12, pBBRBB::GFP K12, and an abiotic control grown in heavy metal tryptone medium, 3mM NaS2O3tubes, and 2.5mM Fe(II)Cl2. Since the phsABC gene is responsible reducing thiosulfate, NaS2O3 would be converted to H2S, which will further react with Fe(II)Cl2 to produce FeS, a black precipitate. After a 24 hour incubation period, the cell cultures appeared as displayed. The pBBRBB:phsABC K12 cells were the only ones seen to produce FeS, the black precipitate seen in the figure, confirming the role of the phsABC gene in reducing thiosulfate. To affirm that this reaction is also successful under non-enclosed systems, the same sets of samples were also tested on 0.2% plates containing 3mM NaS2O3tubes and 2.5mM Fe(II)Cl2. The results after 24 hours of incubation were similar to the experiments conducted in test tubes</H3></DIV><DIV class="slide" id="slide11" data-anchor="slide11"><TABLE style="width: 100%; margin: 5px;"><TBODY><TR><H4>Encapsulation</H4><TD>   0                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               
</TD><TD>   0
 </TD></TR><TR><TD><H3>
The main goal of EncapsuLab was to design a system to physically separate our living bacteria from the outside environment as well as to preserve and protect the bacteria inside our system. This was essential that the bacteria survive the process we subjected them to in order to be able to actively remediate mercury over time. To achieve this, we created a water-porous silica matrix using techniques developed by the Aksan and Wackett labs at the U of M. Furthermore, we developed a device to house the encapsulated bacteria for application in real world water-cleaning system. In addition to this, we conceptualized a scaling-up of our system for larger water-cleaning problems. Lastly, we developed a mathematical model to compare our experimental data in order to better understand the biochemical networks behind our work.
</H3></TD></TR><TR><TD><H2>
Series of SEM images of encapsulated bacteria. Furthest zoom: a collection of microbead encapsulations approximately 300µm in diameter. Closest zoom: The inside of a crushed microbead encapsulation showing collections of E.coli. 
</H2></TD></TR><TR><TD><H4>
SEM
</H4></TD></TR></TBODY></TABLE></DIV><DIV class="slide" id="slide12" data-anchor="slide12"><DIV id="title"><H4>Encapsulation Procedure</H4></DIV><H3>To make a silica matrix surrounding a cell, we needed a supply of silica nanoparticles and a reactive cross-linker capable of covalently bonding reagents into a unified volume. Two solutions were prepared: one was a combination of TM-40 (colloidal silica) and diluting components (ultrapure water and polyethylene glycol). The second solution was tetramethyl orthosilicate (TMOS) cross-linker. In slightly acidic conditions, methyl groups of TMOS are hydrolyzed, producing methanol and the silica cross-linker. Resuspended and washed cells are then added to the relatively innocuous colloidal solution. Upon combining these two mixtures, a rapid polymerization linking colloidal silica together occured. In order to achieve proper bead sphericity, we combined all of these components in an hydrophobic solution and let them set. After some experimentation, we were able to produce beads with a consistent size. To purify the mixture, we phase-separated the silica beads by adding water to solution, which forced them into the aqueous phase. These were then recovered to assess viability.
</H3></DIV><DIV class="slide" id="slide13" data-anchor="slide13"><TABLE style="width: 100%; margin: 5px;"><TBODY><TR><H4>The Biochemical Network</H4></TR><TR><TD><H3>In this project, systems of mathematical equations are used to model the biochemical network of methylmercury conversion in E. coli and to determine the effects of changing methylmercury concentration on the conversion rate. A combination of enzyme kinetics and protein expression equations was in the model. The reactions below shows the four-step reaction network for the conversion of methylmercury to volatile mercury using the mer operon in E. coli. This network involves the transport of methylmercury into the cell, its subsequent conversion into ionic and volatile mercury, and the transport of volatile mercury to the extracellular space. This simplified network follows the proposed mechanism of the mer operon and it captures the primary steps of methylmercury breakdown in E. coli. Furthermore, a system of ordinary differential equations was used to model the biochemical reactions taken place at each step of the conversion process. The cofactors were not included in the equations because at sufficient concentrations, they do not significantly alter the rates.</H3></TD></TR></TBODY></TABLE></DIV><DIV class="slide" id="slide19" data-anchor="slide19"><A href="https://static.igem.org/mediawiki/2014/a/ab/Supplementary_Materials.pdf"><H3><B> Supplementary materials here.</B></H3></A></DIV><DIV class="slide" id="slide20" data-anchor="slide20"><H4> Device Design </H4><TABLE style="width: 100%; margin: 5px;"><TBODY><TR><TD><H3>
This device incorporates two key elements of mechanical engineering design: simplicity and functionality. Key concepts from fluid mechanics are applied to pump water into and out of the encapsulation as efficiently as possible. The system can be placed in or out of the water, which makes it easy to use in any situation. The sleek aluminum casing offers a durable resistance to the ever-changing surroundings and environment. Overall, the simplicity of the design makes it user-friendly and adaptable. 
</H3></TD></TR><TR><TD><H3>
Our design is very similar to commonly used fixed-bed adsorption and ion exchange columns in that it involves flow of a fluid down through a packed bed that removes a solute. However, an important difference is those types of columns contain chemical resins or adsorbents that can become saturated (i.e. reach capacity) as they adsorb solute. Since we use encapsulated bacteria whose &quot;capacity&quot; is only limited their lifetime, our device has the added benefit that the mass-transfer zone (the part of the column where the solute is removed) doesn't move down the column due to saturation. This also means that we don't need to alternate between adsorption and regeneration cycles.
</H3></TD></TR></TBODY></TABLE></DIV><DIV class="slide" id="slide21" data-anchor="slide21"><H4> Scalability </H4><H3>Based on the small-scale experiments we conducted in lab, we calculated a few values that will be useful in <B>scaling up</B> our process to a pilot-plant size. Shown above is a simple process flow diagram (PFD) for a pilot scale wastewater treatment process utilizing our encapsulated bacteria. An in-depth scalability analysis is linked below, and the results are quickly summarized on this page. A residence time of 8 hrs is used as a first approximation based on small scale time-point studies of 1 mg/L methylmercury degradation. For a flow rate of 0.1 m3/h, which is within the range used in other pilot-plant studies, a 0.8 m3 packed bed will be needed, with a diameter of 0.6 m and a length of 2.8 m. Based on a SEM characterization of our beads and an approximation for how they would pack in our reactor, the pressure drop across the reactor was calculated to be 5970 Pa•s, equivalent to frictional losses of 5.97 J/kg. Based on these calculated values, it is concluded that our encapsulation technology can be used in a larger scale plant. <A href="https://static.igem.org/mediawiki/2014/c/cf/Scalability_Blurb.pdf " style="color:white;"> Click here  for more detail! </A>
Additionally, a <B>small scale</B> device can be envisioned for household use in contaminated areas. Our system was tested to successfully remediate at least 1mg/L of methylmercury within a 5 hour time period. Water entering these homes will likely have methylmercury concentrations a hundred-a thousand fold lower than 1 mg/L. Based on our time-point degradation studies, a filter for this concentration level would need smaller residence times and consequently a smaller volume. Therefore, a filter using encapsulated bacteria on the scale of domestic water softener filters is possible.


</H3></DIV></DIV><DIV class="section" id="section3a"><DIV id="title"><H4>Meet The Team</H4></DIV></DIV><DIV class="section" id="section4"><DIV class="slide" id="PnPslide1" data-anchor="PnPslide1"><H4>Policies &amp; Practices</H4><H3>Our Policy and Practices approach this year has focused on establishing an effective two-way discourse between our team and the public in order to inform our design, educate the public, and illuminate ethical issues related to both our project and integration of public opinion into project implementation. We focused our discussion on safety, ethics, sustainability, and intellectual property in order to design a project that fits the needs of the public while maintaining technical viability and high potential for commercialization. Our team did extensive educational outreach in addition to educating ourselves on public perception of our work. We used this input to inform our device design such that it could be implemented in a way that addresses the major concerns of both scientists and consumers. We also took steps to protect our intellectual property and explore the patenting and commercialization process of our product. Finally, our team sought to investigate ethical issues brought up by our discourse with the public by discussing both our work and ethics related to the project with a variety of experts and ethicists.

</H3></DIV><DIV class="slide" id="slide2" data-anchor="slide2"><H4>Educational Outreach</H4><H3>Building on past successes, our team has been devoted to volunteering our services to the community in a number of educational venues. The team took our curriculum, first developed in 2013, and improved the structure and delivery of our lesson plans in the hopes of encouraging awareness and education on topics in synthetic biology. Since 2013 our educational outreach group ECORI (Educating Communities On Research Innovation) has taught our <A href="https://static.igem.org/mediawiki/2014/a/ab/Curriculumhandbook.pdf" style="color:white;"> original interactive classroom curriculum </A>to over 200 students (K-12) and their teachers. This year we also created a mobile exhibit form of our curriculum along with a layman’s introduction to our project that we displayed on over half a dozen weekends to visitors of all ages at the Science Museum of Minnesota. Our curriculum has also been brought to several other STEM fairs and family fun events in the Twin Cities area including the 3M Science Day Fair for 3M employees and their families, UMN Biodiversity Fair, CSE Family Fun Fair, and the Middle School STEM Fair hosted by the Association of Multicultural Students at UMN. Finally, the team designed a Synthetic Biology Game Show that was presented on stage with 30 participants at the Minnesota State Fair to assess the general public’s knowledge of the subject and teach hundreds of passers-by in a way that was both engaging and interactive. Winners were rewarded with reusable bags, magnets, and gift cards donated by our sponsors. In the spirit of science, our curriculum has been ever evolving to constantly address salient topics and educational materials. The variable versions of our curriculum allow it to be flexible and practical in various settings.

 </H3></DIV><DIV class="slide" id="slide3" data-anchor="slide3"><H4>Public Perception</H4><H3> Our team sought to inform the majority stakeholders in our community concerning the scope of our project. This year our team chose to have an exhibit catered towards adult residents at the Minnesota State Fair (the largest statewide annual gathering with over 1.8 million visitors each year) to learn how we can best design our technology to meet the needs and concerns of the people whose waters we hope to bioremediate. We delivered a short synopsis of our device, the synthetic biology involved, and safety precautions we have outlined for our project. We then presented visitors with a five question survey using a Likert Scale to gauge public perception of both our device, and the synthetic biology methods used. The survey was a huge success with over 320 participants. With such a diverse attendance, our survey captured a great cross-section of the Minnesota community that would be impacted by the implementation of our device. The results of our survey, illustrated below, informed how and where the public would be most comfortable with implementing our device, and illustrated the need for catered education addressing the public’s major concerns prior to applying our device in the environment. Our model for gauging public perception allowed for a wide, diverse crowd to be accessed. This model can be used upon request.


 </H3></DIV><DIV class="slide" id="slide4" data-anchor="slide4"><H4>Intellectual Property</H4><H3>
Members of our team attended three Intellectual Property Protection and patenting workshops that educated them on the process of commercialization of our invention. This allowed us to develop a comprehensive <A href="https://static.igem.org/mediawiki/2014/e/e4/IGEM_Business_Plan.pdf " style="color:white;"> business plan </A>and perform an economic analysis of the mining, fisheries, and governmental markets that could potentially benefit from the use of our invention. We presented our work and received feedback and advice from employees and scientists at both Cargill and 3M. We also worked in conjunction with the Office of Technology Commercialization to explore the patentability of our project, the novelty, the non obviousness, and utility of our product and how to make the best claims to patent our device or license it to Minnepura Technologies, Inc.
</H3><TABLE style="width: 100%; margin: 5px;"><TBODY><TR><H3> Basem and Patrick meet with patent attorneys at the Office for Technology Commercialization </H3></TR></TBODY></TABLE></DIV><DIV class="slide" id="slide5" data-anchor="slide5"><H4>Documentary</H4><H3>
In order for our project to reach commercialization and success with the public, we needed to inform ourselves as well as others of the ethics of the use of our invention. We compiled this documentary in order to inform those unfamiliar with the problem of global mercury contamination and to discuss the bioethical questions of synthetic biology as they related to our device implementation. We conducted interviews with specialists from environmental toxicology, biotechnology, and philosophy. Through our collaboration with the 2014 Colombia iGEM Team, we further examined the current mercury contamination in Colombia and around the globe.
            </H3></DIV></DIV><DIV class="section" id="section5"><DIV class="slide" id="safetySlide1"><H4>Safety in the Lab</H4><H3><B>Our Training </B> 
                 DEHS Introduction: Research Safety 
                 
                 DEHS Chemical Safety
                 
                 DEHS Waste Management
            </H3></DIV><DIV class="slide" id="safetySlide2"><H4>Safety in the Lab</H4><H3>Our Local Rules and Regulations: 
                   The project was discussed with the Department of Environmental Health and Safety at our university, and a plan was devised for mercury waste disposal based on their input. General biosafety guidelines found at https://www.dehs.umn.edu/bio.htm, http://www.dehs.umn.edu/bio_pracprin.htm and http://www.cdc.gov/biosafety/publications/bmbl5/bmbl.pdf were followed. 
Risks of Our Project: 
In order to mitigate risks to the safety and health of team members, or other people working in the lab, gloves were used in any protocol that utilizes Ethidium Bromide, including gel electrophoresis. 
Lab coats, gloves, and full face shields were used when cutting gel fragments in proximity of ultraviolet light. A lab coat, inner and outer (long cuffed) nitrile gloves, lab goggles and face shields will be used for handling mercury, and used materials were disposed of by the University of Minnesota Department of Environmental Health and Safety. In addition, there are assigned incubators, hoods, and disposal containers specifically for experiments that involved mercury. </H3></DIV><DIV class="slide" id="safetySlide3"><H4>Safety in the Lab</H4><H3>
Design features to Minimize Risk: 
We used non-pathogenic (BSL1) lab strains of bacteria to minimize the risk to humans. Second, our device would be air tight to prevent the bacteria from escaping and a filter to store the mercury that has been biologically remediated. Third, we could use one of the kill switch proposals that were created so that if the bacteria were to escape outside they would swiftly self-destruct.

</H3></DIV></DIV><DIV class="section" id="section6"><DIV id="title"><H4>Attributions</H4></DIV><P><B>Wet Lab:</B></P><P> Mercury Project Design: </P>
Basem, Aunica

Mercury Ion Testing: 
Sarah, Cassandra, Camilo, Srijay, Jennifer, Suzie, Aunica

Methylmercury Testing: 
Nater Lab, Niko, Srijay, Patrick, Suzie, Basem, Aunica

Cadmium, Zinc, Copper project design/ Testing
Basem, Stephen, Aunica, Cassandra

Kill Switch Biosafety Proposal: 
David, Sarah

pDU1358 received from Dr. Anne O. Summers, University of Georgia
pSB74 received through addgene from Keasling Lab
pBBRBB recieved from Dr. Claudia Schmidt-Dannert, University of Minnesota

Composite parts:
mer operon:
Primer design: Basem, Stephen
Parts cloning: Basem, Jennifer, Valeriu

phsABC:
Primer design: Basem, Stephen
Parts cloning: Basem, Valeriu

Single parts:
	merR:
Basem, Stephen, Cassandra
merT:
Basem, Stephen, Sarah, Jennifer
merP:
Basem, Stephen, Camilo, Logan
merA:
Basem, Stephen, Valeriu, Jessica, Cassandra, Sarah
merB:
Basem, Stephen, Logan, David

Chassis Transformations:
Pseudomonas putida: Basem
Shewanella oneidensis: Basem
E. coli K12: Basem, David
Rhodopseudomonas: Basem, Stephen


EncapsuLab:

Encapsulation Protocol Design: Niko, Srijay, Patrick, David
Cell encapsulation: Niko, Patrick, Srijay, David, Basem, Suzie
Cell Viability Testing: Niko, Srijay, Patrick, David
SEM encapsulation imaging: Niko, UofM imaging center
Device design: Roxana, Niko
Mathematical modelling: Di, Zhiyi, Patrick, David

Policies and Practices:

Outreach, presentations, public perception studies

School Curriculum design: Basem, Suzie
Science Museum Curriculum Design: Sarah L, Jessica, Cassandra, Sarah Perdue
Middle School Classroom outreach: Jess, Basem, Cassandra, Jennifer, Suzie
Science Museum outreach: Jess, Jen, David, Sarah, Cassandra, Basem, Srijay, Di, Holly, Logan, Valeriu, Taylor
3M, Cargill company presentations: Suzie, Basem, Cassandra, Stephen, Jess
State Fair outreach:
	tabling &amp; survey: Cassandra Taylor Jess Jen Basem Suzie Niko Stephen Roxana Di Srijay Patrick Sarah Perdue Holly Logan Valeriu Sarah
	Survey statistics: Taylor
	slideshow: Jessica
	giveaways: CBS, Fridley Super Target, Rob Rakow
	survey content: Jessica, Srijay, Taylor, Cassandra
State Fair game show:  Cassandra, Taylor, Sarah Perdue
Multicultural Student Association Collaboration: Jessica
Colombia collaboration: (magnetic stirrer) Stephen

Ethics :
Blog: Basem, Cassandra, Logan, Jen
Documentary: Interview Questions, Content: Jennifer, David,  Sarah Perdue, Colombia iGEM team
Camera, Editing, Production: Connor Gleason

Business Plan:
Justin, Tanner, Basem, Tamara

Economic Analysis: 
Justin, Tanner

Intellectual Property Rights and Patenting
Basem, Office of Technology Commercialization

Wiki development
Design: Basem, Mari, Chris, Aaron, Tanner
Icons: Mari
Figures development: Mari, Basem, Niko
Coding, CSS, javascript: Aaron, Chris, Basem
Content: Stephen, Basem, Srijay, Niko, Patrick, Di, Holly, 
Lab notebook: Sarah Lucas

Poster:
Basem

Team Logo
Niko

Forms, IP, safety:
Basem

Parts Submission form &amp; shipping
Stephen

Public relations and team contact
Basem, Jessica

Grant writing, fundraising
Basem, Jess, David, Cassandra


</DIV><DIV class="section" id="section7"><H4>Gold Medal Requirements</H4><H3>
UMN iGEM has strived to produce Gold Medal level work through the duration of our project. Here we outline our work that specifically pertains to each of the gold medal requirements. <B>Please click the dots below to naviagte through this page.</B></H3></DIV><DIV class="slide" id="goldslide2"><H4>Requirement 1: Improving function or characterization of an existing part</H4><H3>
In order to measure the constitutive phsABC thiosulfate reducing activity of NaS2O3 to H2S, the operon was first inserted into the pBBRBB vector with a constitutive Placpromoter and transformed into E. coli K12. As a negative control, pBBRBB::gfp was tested under the same conditions. The pBBRBB::phsABC K12 and pBBRBB::gfp K12 cells were grown in three test tubes each containing heavy metal tryptone medium as well as 3mM NaS2O3. A third set of test tubes were set up with the same contents except without cells as an additional negative control. After 24 hours of incubation, the exact amount of H2S present in each of three different sets of tubes was then measured using a hydrogen sulfide assay tested against a known standard curve of various Na2S concentrations. The compiled results showed that the sulfide concentration was considerably higher for the cultures containing pBBRBB:phsABC (380.1 μM ± 13.5) compared to the pBBRBB::gfp negative control (118.9μM ± 1.1). A second set of experiments was also conducted with pBBRBB:phsABC K12, pBBRBB::GFP K12, and an abiotic control grown in heavy metal tryptone medium, 3mM NaS2O3tubes, and 2.5mM Fe(II)Cl2. Since the phsABC gene is responsible reducing thiosulfate, NaS2O3 would be converted to H2S, which will further react with Fe(II)Cl2 to produce FeS, a black precipitate. After a 24 hour incubation period, the pBBRBB:phsABC K12 cells were the only ones seen to produce FeS, confirming the role of the phsABC gene in reducing thiosulfate. To affirm that this reaction is also successful under non-enclosed systems, the same sets of samples were also tested on 0.2% plates containing 3mM NaS2O3tubes and 2.5mM Fe(II)Cl2. The results after 24 hours of incubation were similar to the experiments conducted in test tubes. Following sulfide measurements, cadmium chloride was added (200 μM), and cells were allowed to incubate without shaking at 37C overnight. Cells were pelleted to look for a color change indicating precipitation of CdS. Cell pellets for K12 expressing phsABS were yellow/brown indicating precipitation of CdS while the vector control cells (pBBRBB::gfp) remained white.

</H3></DIV><DIV class="slide" id="goldslide3"><H4>Requirement 2: Collaborations with other teams</H4><H3> 
The UMN collaborated with Wisconsin Lutheran to facilitate a week’s worth of synthetic biology curriculum by sending detailed lesson plans developed by our iGEM team, along with strains of microorganisms used to facilitate trainings. In addition, we participated in Columbia iGEM's low-budget lab exercise, who in turn provided UMN iGEM with an interview with a local environmental health specialist who has extensive knowledge of the mercury contamination problem within Columbia. 

</H3></DIV><DIV class="slide" id="goldslide4"><H4>Requirement 3: Describing and evaluating questions beyond the bench</H4><H3>

The primary goal of UMN iGEM 2014 has been to design a project in line with the major safety, ethics, and intellectual property concerns of both scientists and the public. We took the following steps to address each of these issues:<B>Safety</B>: Our project gives detailed attention to the prevention of releasing genetically modified bacteria into the environment via encapsulation and the development of several killswitch proposals. The device also includes an activated carbon filter to both collect mercury and prevent release of the bacteria. During an outreach event at 3M we also discussed this filtration element of our device with one of their filtration experts to further inform our safety measures.<B>Ethics</B>: We have been committed to establishing an open and productive dialogue between UMN iGEM and the public. Our surveying at the State Fair of public opinion on safety and implementation concerns of our device helped to inform our design. We believe there is great ethical importance to involving the community in the design of a device that could potentially be used on their waters. We also developed an interactive game show at the fair to actively engage the public in synthetic biology knowledge. Our dialogue with the public has also included our outreach program at local schools, the Science Museum of Minnesota, and various STEM fairs allowing us to develop a dynamic dialogue with both children and their parents. The program focuses on the understanding of DNA in order to demystify the basic principles behind synthetic biology. It is our hope that a more solid understanding of these principles may help to familiarize children with biotechnological tools and decrease misconceptions about the nature of DNA manipulation. We hope this may help to develop an informed future community of consumers and constituents to facilitate open dialogues between scientists and the public. Finally, we interviewed various experts in bioethics and scientific fields related to our project to help further inform ourselves and give essential background to those unfamiliar with the global issue of mercury contamination.<B>Intellectual Property</B>: We sought to create an integrative and marketable project by developing a business plan along with marketing analysis for our device. We also educated ourselves about the patenting process via on campus seminars. Finally, we applied for a patent to protect the intellectual property of our team in order to help protect us during any future implementation of the developed business plan.</H3></DIV></DIV><DIV class="section" id="section8" data-anchor="Medal"><DIV id="title"><H4>Sponsors</H4></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Minnesota">http://2014.igem.org/Team:Minnesota</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Minnesota" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Minnesota" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Minnesota&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Minnesota&amp;oldid=406352" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>