"
Page
Discussion
View source
History
teams
Log in
 
Team:HokkaidoU Japan/Shuffling Kit
From 2013.igem.org
(Redirected from Team:HokkaidoU Japan/Optimization )
HOKKAIDO U
Top
Maestro E. coli
Overview
Project Description
Background
Achievements
Shuffling Kit
Overview
How To Use
Primer Designer
Examples
Future Work
Promoter
Overview
Methods
Modeling
Results
Conclusion
RBS
Overview
Methods
Results
Conclusion
Extra Materials
Parts
Notebook
Safety
Human Practice
Education
University Festival
Highschool
Survey
University Festival
Highschool
About Us
Acknowledgements
Team
Maestro E. coli
Shuffling Kit
Overview
As we referred at the motivation, the transcription efficiency can vary about 1000 fold by choosing different promoter. Translation efficiency varies about 100 folds by choosing different RBS. So the selection of the expression regulatory region is extremely important for protein you want to express.
However, selecting the strongest regulatory region does not always result in the best production. It is a hard work to select the best combination of a promoter and a RBS by yourself.
Therefore, we made a remarkable kit to select the regulatory region and optimize the expression, which suits the user's demand and can be done in single pot reaction. It will be a new standard of choosing promoters and RBSs!
Our kits are for both promoters and RBSs. Both kits use Golden Gate Assembly (GGA), one-pot DNA shuffling (Engler, 2009). BsaI cloning site used by that method is the key of our kits.
We made a animation movie to introduce our project.
One-pot DNA shuffling method
fig.1 BsaI cloning site.  BsaI recognizes GGTCTC sequence, and cuts other region.
BsaI cloning site has unique characteristics that enabled us to construct our shuffling kits. BsaI restriction enzyme is classified as Type IIs restriction endonuclease. The unique property of this class is that recognition site and the cutting sites are apart. Unlike EcoRI, BsaI recognizes GGTCTC sequence, but cuts the sequence located 7 bases downstream from first base in recognition site (fig.1). Which results in a 5 prime 4 base overhang structure (fig.2).
fig.2 How BsaI works.  BsaI cuts different region form recognition site.
Usual restriction enzyme has a decided overhang sequence, and the sequence is palindromic. So DNA cloning requires more than 1 kind of restriction enzyme. In contrast, BsaI DNA overhangs can be made into any desired sequence. Therefore, 256 different overhangs can be created using a BsaI restriction site. By designing the overhangs, DNA fragments can be assembled together in a defined order, and inserted in a vector in one step.
Shuffling promoters and RBSs by using GGA
Our kits are using this remarkable method called "Golden Gate Assembly one-pot DNA shuffling". Thus, our kits are capable to make several variants of the constructs in one reaction.
Promoter Selector can insert protein sequence into constructs with five different promoters with different strengths (fig.3).
fig.3 Promoter shuffling.  Your CDS will be inserted the downstream of a certain promoter randomly.
RBS Selector can insert four different RBSs with different strengths. Currently it is possible to shuffle up to 3 RBSs in one operon with our kit (fig.4).
fig.4 RBS shuffling.  RBS Selector contains 4RBSs, and you can randomize max 3 CDSs.
How To Use
Follow us on
Twitter
Retrieved from " http://2013.igem.org/Team:HokkaidoU_Japan/Shuffling_Kit "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
