"
Page
Discussion
View source
History
teams
Log in
 
Team:HokkaidoU Japan/RBS/Conclusion
From 2013.igem.org
HOKKAIDO U
Top
Maestro E. coli
Overview
Project Description
Background
Achievements
Shuffling Kit
Overview
How To Use
Primer Designer
Examples
Future Work
Promoter
Overview
Methods
Modeling
Results
Conclusion
RBS
Overview
Methods
Results
Conclusion
Extra Materials
Parts
Notebook
Safety
Human Practice
Education
University Festival
Highschool
Survey
University Festival
Highschool
About Us
Acknowledgements
Team
Maestro E. coli
RBS
Conclusion
We were successful at making four RBS set with four strength levels.   Construct with SD4 showed the strongest β-Galactosidase activity.   Second strongest was the SD8, followed by B0034 and SD6. SD2 had the weakest activity.
Though we synthesized the RBSs based on previous reports, we got unexpected results.   Preciously SD6 was reported as the strongest. However our results indicated SD4.
fig.1 The difference of expression level.  If there is A/U rich enhancer, SD sequence has mighty effect.
fig.2 LacZα expression level.  Difference of SD sequence gives different amount of LacZα expression.
The sequence we synthesized was completely the same.   Both plasmids had low copy number.   One thing we changed was the reporter gene.   In the previous experiments, GFP was used for the reporter.   In contrast, we used LacZα.   The expected strength differed only by changing the coding sequence.
It is well known fact that mRNA makes a secondary structure. The secondary structure of RNA takes an important role in the process of life. Identically, the secondary structure and the folding of mRNA is an important factor in translation efficiency. Kudla et al. (2009) showed that translation efficiency is determined by factors in Coding-Sequence. Overall, the translational efficiency is a correlation between RBS sequence and the Coding-Sequence.
Our results show different levels of translation efficiency.   We should try and repeat our assays with other reporter genes.   The translation efficiency might change by choosing different coding sequences.   The translation efficiency not depended on RBS but also influenced by coding sequence.   This fact complicates designing biological devices.
When regulating the expression it is important to have variability in RBSs strength. Our set RBSs proved to have different efficiencies in both GFP and LacZα assays. From our results and the previous research we can expect that our RBSs will show variability in translational efficiency using any coding site. Therefore, we made a useful set of RBSs for regulating expression.
Grzegorz Kudla, et al. Coding-Sequence Determinants of Gene Expression in Escherichia coli (2009) Science
Results
Follow us on
Twitter
Retrieved from " http://2013.igem.org/Team:HokkaidoU_Japan/RBS/Conclusion "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
