"
Page
Discussion
View source
History
teams
Log in
 
Team:HokkaidoU Japan/RBS
From 2013.igem.org
HOKKAIDO U
Top
Maestro E. coli
Overview
Project Description
Background
Achievements
Shuffling Kit
Overview
How To Use
Primer Designer
Examples
Future Work
Promoter
Overview
Methods
Modeling
Results
Conclusion
RBS
Overview
Methods
Results
Conclusion
Extra Materials
Parts
Notebook
Safety
Human Practice
Education
University Festival
Highschool
Survey
University Festival
Highschool
About Us
Acknowledgements
Team
Maestro E. coli
RBS
Overview
To make RBS selector, we constructed well-selected sets of RBSs.       For parts controlling gene expression such as promoters or RBSs, it is desired that their prospective functions are explainable.       We wanted our parts to have, "transparent structure", "reliable function" and "reproducibility".       Thus, when making our new parts, we decided to change only one region in mRNA.
Ribosome binding site (RBS) is located upstream of initiation codon in mRNA. Translation efficiency depends on RBS sequence.       RBS has Shine-Dalgarno sequence (SD).       SD binds Anti-Shine-Dalgarno sequence (ASD) on ribosomal 30S subunit.       Then initiation codon binds with fMet-tRNA anticodon and the translation will begin.       SD-ASD binding strength is important for translation efficiency.       However, there are results that show RBS binding to 30S subunit even if there is no SD sequence.
fig.1: Ribosome and mRNA. First, S1 protein binds A/U rich sequence. Then, ASD binds SD.
There is another place that binds with the ribosome in mRNA.       Upstream of the SD sequence, there is an A/U rich sequence.       This A/U rich sequence binds with S1 protein, which is one of proteins that makes 30S ribosome [1] .       The sequence has an important role to make the translation initiation complex.       To make it, mRNA first has to bind with 30S ribosome which results to the binding of SD and ASD.       Then A/U rich sequence and S1 protein binds together.       The loose binding with A/U rich sequence and S1 protein, leads binding with SD and ASD (fig.1).       Thus, this A/U rich sequence is called translational "enhancer"!
Vimberg [2] constructed RBSs by changing enhancer sequence and SD length (fig.2).       Although SD length was changed, there wasn't big difference in translation efficiency among RBSs' without enhancer (fig.3).       But big difference appeared in enhancer RBSs when SD length was changed (fig.4).       Strong SDs are stimulated and weak SDs are repressed by A/U rich enhancer.
fig.2: Various enhancer and SD. Vimberg combined 3 enhancers and 10 SDs and measured GFP expression.
fig.3: GFP expression in No enhancer RBSs.
fig.4: GFP expression in A/U rich RBSs.
We decided to constructed 4 new RBSs based on Vimberg.       These RBSs have A/U rich enhancer.       To change the translation efficiencies we varied the length of SD sequence.
B. S. Laursen, H. P. Sørensen, et al. Initiation of Protein Synthesis in Bacteria (2005) Microbiol. Mol. Biol. Rev.
V. Vimberg, A. Tats, et al. Translation initiation region sequence preferences in Escherichia coli (2007) BMC Molecular Biology
Methods
Follow us on
Twitter
Retrieved from " http://2013.igem.org/Team:HokkaidoU_Japan/RBS "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
