"
Page
Discussion
View source
History
teams
Log in
 
Team:HokkaidoU Japan/Shuffling Kit/How To Use
From 2013.igem.org
(Redirected from Team:HokkaidoU Japan/Optimization/How To Use )
HOKKAIDO U
Top
Maestro E. coli
Overview
Project Description
Background
Achievements
Shuffling Kit
Overview
How To Use
Primer Designer
Examples
Future Work
Promoter
Overview
Methods
Modeling
Results
Conclusion
RBS
Overview
Methods
Results
Conclusion
Extra Materials
Parts
Notebook
Safety
Human Practice
Education
University Festival
Highschool
Survey
University Festival
Highschool
About Us
Acknowledgements
Team
Maestro E. coli
Shuffling Kit
How to use
What users should prepare
To use the kit, the protein sequence you chose must have specific prefix/suffix which contains a BsaI site and produce overhang. Therefore, users must design a primer to add it. To reduce time and trouble, we automated the design by creating program "Primer Designer for Maestro"!
Promoter Selector
What our kit contains
Our Promoter Selector consists of 5 different plasmids (table.1). Each has a promoter with a different strength. Downstream of the RBS there is a BsaI site to insert the protein sequence. There is a color expression construct downstream of protein insertion site (fig.1). Each color is paired with different strength promoter.  The pairings are shown on the table below.
Part number
Promoter
Promoter strength
Paired protein
Protein color
BBa_K1084501
BBa_K1084001
Strongest
amilGFP
yellowish green
BBa_K1084502
BBa_K1084002
Stronger
aeBlue
strong blue
BBa_K1084503
BBa_K1084005
Medium
amilCP
Purple
BBa_K1084504
BBa_K1084009
Weaker
mRFP
Pink
BBa_K1084505
BBa_K1084010
Weakest
eforRED
red
table.1 Matching list of promoters and their colors.
fig.1 The matching color for respective promoters.
The color always expresses. LacZα reporter is placed between two BsaI sites (fig.2). The LacZα expressing construct will be replaced by chosen sequence using BsaI.
fig.2 Insertion of LacZα reporter between two BsaI sites.
How it works
fig.3 Digestion by BsaI.
1. Have BsaI site and specific overhang added to your protein sequence (fig.3). PCR with primers designed with Primer Designer for Maestro should do the trick.
2.   Digest and ligate your protein coding sequence and all our Promoter Selector together (fig.4). This is accomplished by "Golden Gate Assembly" reaction. The detailed recipe is shown in Engler (2009) [1] . All the protein coding sequence will be inserted in the plasmid.
fig.4 How to insert CDS into protein coding sequence.
3. You should get the construct shown below after Golden Gate Assembly (fig.5).
fig.5 Constructs you can get.
4. Transform the ligated DNA to E. coli, and spread it on plate. Then, you will get colonies with five colors easily because you can see 5 colors by naked eyes (fig.6). The colors are paired with the promoters, so you will know what promoter you are using without sequencing all the colonies. You can pick up colonies and have an assay.
fig.6 Method after transformation.
RBS Selector
What our kit contains
Our kit contains tandem RBS (fig.7) and acceptor plasmid (fig.8).
fig.7 The sequence of tandem RBS.
In this part, 4 strength levels of RBSs[BBa_K1084101, BBa_ K1084102, BBa_ K1084103, BBa_ K1084104] are connected in tandem. To optimize up to 3 coding sequence expressions in the operon this part has three sets of RBSs with different overhangs and they are connected together.
fig.8 The acceptor part of the RBS and protein coding region. It has a BsaI site for the parts to be assembled.
How to use
1. Have BsaI site and specific overhang added to your protein sequence. Again, Primer Designer for Maestro should help your primers design (fig.9). Also when you want to apply more than one protein coding sites, add BsaI sites and overhang to them too. Be careful  not to choose the same overhangs.
fig.9 Insertion CDS anf digestion by BsaI.
2.   Digest and ligate your protein coding sequences and all RBS Selector together. This reaction is also accomplished by Golden Gate Assembly. DNA fragments will be assembled in the desired order (fig.10).
fig.10 How to use  Golden Gate Assembly.
3. Transform the ligated DNA to E. coli. If you are using three different proteins, you will get 64 different kinds of constructs.
We will submit these standard methods as RFC to BioBrick Foundation.
C. Engler et al. Golden Gate Shuffling: A One-Pot DNA Shuffling Method Based on Type IIs Restriction Enzymes (2009) PLoS ONE
Shuffling Kit Top
Primer Designer
Follow us on
Twitter
Retrieved from " http://2013.igem.org/Team:HokkaidoU_Japan/Shuffling_Kit/How_To_Use "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
