"
Page
Discussion
View source
History
teams
Log in
 
Team:HokkaidoU Japan/Promoter/Conclusion
From 2013.igem.org
HOKKAIDO U
Top
Maestro E. coli
Overview
Project Description
Background
Achievements
Shuffling Kit
Overview
How To Use
Primer Designer
Examples
Future Work
Promoter
Overview
Methods
Modeling
Results
Conclusion
RBS
Overview
Methods
Results
Conclusion
Extra Materials
Parts
Notebook
Safety
Human Practice
Education
University Festival
Highschool
Survey
University Festival
Highschool
About Us
Acknowledgements
Team
Maestro E. coli
Promoter
Conclusion
Consensus promoter strength
This time we made "consensus promoter" by combining consensus sequence information and pLac promoter sequence. Because we thought that this would be the strongest promoter. It worked as promoter biobrick. However the strength was between pLac and pTet promoters. This was verified by mRFP expression as. Generally pLac is strong and pTet is medium promoter. We came up with 3 hypotheses why "consensus promoter" was not the strongest.
The consensus sequence we designed did not have the strongest σ factor affinity.
The binding interaction was too excessive.
There were some other problems we didn’t know.
For second reason, it is known that the strongest binding of RNAP and promoter region inhibit promoter escape. Promoter escape is the stage when RNAP leaves from promoter region.
Problem in promoter randomization
We found a problem in the experiment for promoter randomizing at an early stage. Some colonies which seemed not to express reporter gene had mutations in CDS. It might have occurred when we used PCR for construction of the region from promoter to double terminator. To avoid this problem, we propose two methods for promoter randomizing. When doing PCR for randomization take a shorter sequence. Another solution is de novo synthesis.
Family expansion
We produced promoter family but their variations can be improved even more. Here we propose ideas to expand the variation.
Mutating TG motif
TG motif is placed at upstream of -10 region. This motif helps bind RNAP to promoter region, and can change transcription level. Previous research showed the level decrease to 20% by mutating this motif.
Mutating other regions related to transcription
Spacer region and discriminator region are known to relate to transcription efficiency. There might be other regions which contribute promoter activity. So changing sequence in these regions could show more various strengths.
There is one point to note to make bigger variety. The regions relating to transcription affect transcription level in different ways difficult to control.
Transcription strength
Transcription efficiency is not simply correlated to binding strength. Generally the strong binding leads to more transcription but too strong binding is known to inhibit transcription level. As described above, a too strong binding inhibits promoter escape. Another example is inclusion bodies. Inclusion body sometimes appears when too many proteins are produced and inactivates them. For these reasons, some researches recommend using weak promoter.
Summary
Above all, transcription efficiency does not equal to expression level or enzyme activity. In our experiment, same promoter shows different activity in different conditions. This means that the strongest promoter is not the best promoter for getting the maximum amount of product. And the best promoter which we should use is different for every gene or condition. So we produced a kit which selects suitable promoter automatically. We call the great kit "Promoter Selector"!
Results
Follow us on
Twitter
Retrieved from " http://2013.igem.org/Team:HokkaidoU_Japan/Promoter/Conclusion "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
