"
Page
Discussion
View source
History
teams
Log in
 
Team:HokkaidoU Japan/Promoter
From 2013.igem.org
HOKKAIDO U
Top
Maestro E. coli
Overview
Project Description
Background
Achievements
Shuffling Kit
Overview
How To Use
Primer Designer
Examples
Future Work
Promoter
Overview
Methods
Modeling
Results
Conclusion
RBS
Overview
Methods
Results
Conclusion
Extra Materials
Parts
Notebook
Safety
Human Practice
Education
University Festival
Highschool
Survey
University Festival
Highschool
About Us
Acknowledgements
Team
Maestro E. coli
Promoter
Overview
Proteins are expressed in mainly 2 steps. First mRNA is polymerized using DNA as a template. Then ribosome binds mRNA and translates it into protein.
Promoter is a DNA sequence initiating transcription from DNA to mRNA. If transcriptional efficiency is defined as "promoter strength", stronger promoter has ability to transcribe more mRNA. This should lead in stronger expression of proteins.
We have created several promoters by randomization of -35 sequence followed by selection. In promoters -35 region is responsible for supporting binding of RNA polymerase (RNAP). This interaction results in closed complex which is rate-limiting step. We focused on this rather transparent function to introduce variability in promoter strength.
Overview about Transcription
We explain the importance of promoter sequence, but before that let's look how RNA binds to a promoter with the help of (fig.1).
fig. 1 mRNA transcription starts with promoter engagement, continues to initiation, elongation, and then it comes to termination (omitted in the figure).
First transcription complex must be formed. Transcription complex polymerizes mRNA in 2 steps. Initiation step starts polymerization followed by elongation step. Promoter serves crucial role on engagement and initiation. After closed complex formation DNA double helix pulled apart to form transcription bubble. During this closed complex changes into open complex. This marks the beginning of mRNA polymerization. Transcription bubble exposes deoxyribonucleotides to form new hydrogen bonds with ribonucleotides. In short DNA serves as template to make mRNA.
Transcription factors related to Promtoer
RNA complex consist of 5 core enzymes and a σ factor. σ factor plays crucial role in promoter recognition. It recognizes and binds to promoter region on DNA sequence and helps to assemble the core enzyme and start transcription. σ factor has several analogs, E. coli which is widely used bacteria by iGEMers is using σ70 for house-keeping gene expression at exponential growth. Bacterial promoter can be roughly divided into three regions; -10 region, spacer and -35 region. Bases in promoter are numbered in descending order from transcription start base which is defined as +1.
-10 region
The -10 region is structurally very important because it is initiates promoter melting in RNAP-promoter complex. This is essential to form open complex. Promoter consensus sequence is TATAAT at -12 to -7 position.
Spacer
Spacer is thought to increase flexibility of σ factor binding requirements.
-35 region
-35 region is also important second to -10. It does not energetically contribute to promoter melting. There reports on promoters without -35 region. In those case TG motif at about -16 is thought as alternative. -35 consensus sequence is TTGACA at from -36 to -31.
Promoters function to bind RNAP is a reason it is genetically well preserved. Most frequently conserved residues in the sequence make a "consensus sequence". In 1983, -35 and -10 consensus was showed to be TTGACA and TATAAT respectively (fig 2). Horizontal axis of the figures represents the position upstream of translation ignition point. Letter at the top of the figure signifies more than over 39% occurrence of that letter at that position. Larger occurrence over 54% is represented as upper case letter. Consensus sequence published by Marjan De Mey et al. (2007) shows that -10 and -35 region is highly preserved (fig 3). There other less preserved regions. The tetramer (TRTG) upstream from -10 region is called TG motif. Upstream of -35 region is UP element and downstream of -10 region is discriminator region. These sequences are thought to bind core enzymes. So these sequences are also well conserved. Each sequence is important to control promoter strength.
fig. 2 Consensus sequence shown in review article in 1983 [3].
fig. 3 Consensus sequence prepared in 2007 [4].
So we went and designed "consensus promoter". It should have strongest binding energy to RNAP. By adding mutations to -35 we sought to construct promoters with various binding energies. There are three reasons why we used -35 region.
First, -35 region is just supporting binding with σ factor. It has less vital role compared to -10 region, which energetically contributes to formation of open complex. Having this in mind we changed -35 region to easily change promoter binding strength without severe errors in promoter function.
Second, RNAP and promoter binding orchestrated by σ factor binding. Complex formation is thought to be rate-limited step. We thought that -35 region performs a simpler function. For this reason, mutations at -35 region can be thought as more structurally transparent.
Recently published research reported the making of promoter family by randomizing both -35 and -10 regions, changing spacer length. However it would be too much of the task for us to make some many changes. By changing hexamer sequence of -35 region there are 4096 variation. This number is a lot smaller compared to mutating every promoter position. So we can get result with a smaller library size.
With these 3 reasons we went on to construct our promoter family.
[1] R. a Mooney, I. Artsimovitch, and R. Landick, “Information processing by RNA polymerase: recognition of regulatory signals during RNA chain elongation.,” Journal of bacteriology, vol. 180, no. 13, pp. 3265–75, Jul. 1998.
[2] M. S. B. Paget and J. D. Helmann, “The σ 70 family of sigma factors,” Genome Biology, vol. 4, no. 1, pp. 203.1–203.6, 2003.
[3] D. K. Hawley, W. R. Mcclure, and I. R. L. P. Limited, “Compilation and analysis of Escherichia coli promoter DNA sequences,”
Nucleic Acids Research, vol. 11, pp. 2237–2255, 1983.
[4] M. De Mey, J. Maertens, G. J. Lequeux, W. K. Soetaert, and E. J. Vandamme, “Construction and model-based analysis of a promoter library for E. coli: an indispensable tool for metabolic engineering.,” BMC biotechnology, vol. 7, p. 34, Jan. 2007.
[5]De Mey, M., Maertens, J., Lequeux, G. J., Soetaert, W. K., & Vandamme, E. J. (2007). Construction and model-based analysis of a promoter library for E. coli: an indispensable tool for metabolic engineering. BMC biotechnology, 7, 34. doi:10.1186/1472-6750-7-34
Methods
Follow us on
Twitter
Retrieved from " http://2013.igem.org/Team:HokkaidoU_Japan/Promoter "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
