<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TMMU-China_SyntheticQS1 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TMMU-China/SyntheticQS1</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="agileits_w3layouts_banner_nav"><NAV class="navbar navbar-default"><DIV class="collapse navbar-collapse navbar-right" id="bs-example-navbar-collapse-1"><NAV><UL class="nav navbar-nav"><LI><A href="https://2017.igem.org/Team:TMMU-China" class="hvr-underline-from-center">Home</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:TMMU-China/Description" class="dropdown-toggle hvr-underline-from-center" data-toggle="dropdown">Project</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/SyntheticQS1">Synthetic QS for <I>L. lactis</I></A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Demonstrate">Synthetic QS for <I>B. subtilis</I></A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Contribution">Contribution</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Model">Modeling</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Notebook">Notebook</A></LI></UL><LI><A href="https://2017.igem.org/Team:TMMU-China/Parts" class="hvr-underline-from-center">Parts</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:TMMU-China/HP" class="dropdown-toggle hvr-underline-from-center" data-toggle="dropdown">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/HP/Silver">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/HP/Gold_Integrated">Integrated and Gold</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:TMMU-China/Team" class="dropdown-toggle hvr-underline-from-center" data-toggle="dropdown">Team</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Safety" class="hvr-underline-from-center">Safety</A></LI></NAV></DIV></NAV></DIV><DIV id="myCarousel" class="carousel slide"><A class="carousel-control left" href="#myCarousel" data-slide="prev">‹</A><A class="carousel-control right" href="#myCarousel" data-slide="next">›</A></DIV><DIV class="banner-bottom"><DIV class="container"><DIV class="mid_agile_bannner_top_info"><H1><SPAN>Background</SPAN></H1><DIV class="col-md-12"><H3 style="color: #00a98f;">The Quorum sensing system of <I>Staphylococcus aureus</I></H3><P><I>S. aureus</I> employs diverse array of virulence factors, involving a large number of cell-surface bound proteins (e.g. adhesins, fibrinogen/fibronectinbinding proteins) that are expressed during colonization of the host, and secreted proteins (e.g. haemolysins, proteases, lipases) that are required for acute infections. In <I>S. aureus</I>, expression of virulence factors is tightly controlled by an auto-inducing quorum-sensing system known as the accessory gene regulator (Agr) system (Fig. 1). The agrD gene encodes the signaling propeptide. The agrB gene encodes the peptidase for processing of the signaling propeptide. Together with the protease SpsB, the AgrD will be matured and cyclized by formation of the cyclic thiolactone bond between an internal cysteine and the C-terminus residue. The AgrD mature peptide is the signaling peptide of the Agr system, which is also usually called the auto-inducing peptide (AIP) molecules. The AgrC and AgrA constitute a two-component system. The AgrC protein is the receptor for AIP. After bound by the AIP, the AgrC will dimerize and phosphorylate each other to activate the AgrC protein. The AgrC can phosphorylate the AgrA protein. Phosphorylated AgrA protein will dimerize and bind to the P2 promoter of the agrBDCA operon, thus forming a positive feedback loop. The active AgrA dimer will also bind to the P3 promoter to activate the expression of a non-coding RNA, the RNA Ⅲ. The RNA Ⅲ can influence the translation process of virulence factors and adhesion factors. The deletion of Agr system usually attenuates <I>S. aureus</I> virulence in animal models. </P><P style="text-align: center;font-family:'Open Sans', sans-serif;font-size: 13px;">Fig.1 The Agr system of <I>S. aureus</I></P><P>A polymorphism in the amino acid sequence of the auto-inducing peptide (AIP) and of its corresponding receptor (AgrC) divides <I>S. aureus</I> strains into four major groups, agr-I, agr-II, agr-III, and agr-IV (Fig. 2). There are cross-talks between these four types of Agr system. It is hypothesized that the divergence of Agr system might influence different lineages of <I>S. aureus</I> to occupy different niches.</P><P style="text-align: center;font-family:'Open Sans', sans-serif;font-size: 13px;">Fig. 2 The four types of AIP molecules</P><H3 style="color: #00a98f;">The probiotic <I>Lactococcus lactis</I></H3><P><I>L. lactis</I> is one of the best characterized lactic acid bacteria with detailed knowledge on genetics, metabolism, and biodiversity. <I>L. lactis</I> is of crucial importance for manufacturing dairy products, such as buttermilk and cheese. <I>L. lactis</I> has once been named the official state microbe of Wisconsin, the No. 1 cheese-producing state in the United States. In addition to its traditional use in food fermentations, <I>L. lactis</I> is increasingly being used in modern biotechnological applications, including cancer, diabetes, vaccination, infection protection and animal diseases. <I>L. lactis</I> has also become famous as the first genetically modified organism to be used alive for the treatment of human disease (Recombinant <I>L. lactis</I> secreting IL-10 for the treatment of Crohn disease, ClinicalTrials.gov Identifier: NCT00729872).</P></DIV></DIV></DIV></DIV><DIV class="banner-bottom"><DIV class="container"><DIV class="mid_agile_bannner_top_info"><H1><SPAN>Design</SPAN></H1><DIV class="col-md-12"><P>To engineer the Agr system for <I>L. lactis</I>, we want to develop a synthetic autoinduction system for <I>L. lactis</I>. Synthetic autoinduction systems have proven very effective for producing high yields of heterologous protein in high cell density cultures. Autoinduction systems can often express higher titers of recombinant protein than more traditional inducible systems, with the added advantage that there is no need to monitor cell growth and add exogenous molecules at a specific cell density. These systems can be particularly useful for high-throughput screening of many cultures in parallel where monitoring cell growth can be a significant burden, and also for industrial applications to provide reliable induction and increased productivity. <I>L. lactis</I> is widely used in recombinant protein production. Such an autoinduction system is very useful for <I>L. lactis</I>. We placed the agrBDCA operon under the constitutive promoter P32 that is widely used in <I>L. lactis</I>. To moniter the autoinduction, we placed the reporter gene GFP under the control of promoter P2 (Fig. 3A). All these elements were cloned into the shuttle vector of <I>L. lactis</I>, pMG36ek.</P><P>The Agr system can be split between cells to engineer synthetic cell-cell communication systems. In this design, the Agr system is split into AIP sender part (Fig. 3B) and AIP receiver part (Fig. 3C). In the Sender cell, the agrB and agrD gene is under the control of the P32 promoter. In the Receiver cell, the agrC and agrA gene is under the control of the P32 promoter together with the reporter module, the P2-GFP composite part. All these elements were cloned into the shuttle vector pMG36ek.</P><P style="text-align: center;font-family:'Open Sans', sans-serif;font-size: 13px;">Figure 3. The design of Autoinduction (A), Sender (B), and Receiver (C) composite parts of the Agr system.</P></DIV></DIV></DIV></DIV><DIV class="banner-bottom"><DIV class="container"><DIV class="mid_agile_bannner_top_info"><H1><SPAN>Results</SPAN></H1><DIV class="col-md-12"><H3 style="color: #00a98f;">Selection of the appropriate Agr system from the four Agr types</H3><P>There are four different classes in the Agr systems which are referred to as Agr-I, Agr-II, Agr-III, and Agr-IV. Among the four classes of AIPs, the five-residue thiolactone ring structure is always conserved, while the other ring and tail residues differ. Similarly, the proteins involved in signal biosynthesis (AgrB and AgrD) and surface receptor binding (AgrC) also show variability. The activity of the four Agr classes is varied. To select the appropriate Agr system for engineering, we first conduct the hemolysis experiment with four <I>S. aureus</I> strains containing different Agr classes.Production of exoproteins such as hemolysis in <I>S. aureus</I> is controlled by the agr system. Different Agr types are usually associated with different hemolytic activity. To establish the link between them, we placed four <I>S. aureus</I> strains (Table 1)from Agr-I to Agr-IV on rabbit blood agar plate (Fig. 4). The hemolytic activity can be clearly observed from the rabbit blood agar plate. The strain of Agr-I and Agr-IV types show visible and very transparent hemolysis zone around the bacteria, indicating the hemolytic activity of strain of Agr-I and Agr-IV are very strong. On the other hand, no hemolysis zone of Agr-II is observed. The hemolysis zone of Agr-III is very small. The larger the hemolysis zone, the higher activity of the Agr system. AIP-I and AIP-IV differ by only one amino acid, they are grouped together and can function interchangeably. From this result, we are going to use the Agr-I system to do the following experiments.</P><P style="text-align: center;font-family:'Open Sans', sans-serif;font-size: 13px;">Table 1 The strains used and their cognate Agr groups</P><DIV class="row clearfix"><DIV class="col-md-12 column" align="center"><TABLE align="center" class="tg"><TBODY><TR><TD class="biaoge">Agr groups</TD><TD class="biaoge">Strain Name</TD></TR><TR><TD class="biaoge">Agr-I</TD><TD class="biaoge">Newman</TD></TR><TR><TD class="biaoge">Agr-II</TD><TD class="biaoge">N315</TD></TR><TR><TD class="biaoge">Agr-III</TD><TD class="biaoge">MW2</TD></TR><TR><TD class="biaoge">Agr-IV</TD><TD class="biaoge">XQ</TD></TR></TBODY></TABLE></DIV></DIV><P style="text-align: center;font-family:'Open Sans', sans-serif;font-size: 13px;">Fig. 4 Four <I>S. aureus</I> strains from Agr-I to Agr-IV on rabbit blood agar plate.</P><P>Interestingly, different AIP signals cross-inhibit the activity of the others (Table 2). For example, group I <I>S. aureus</I> quorum sensing can be activated by AIP-I but is inhibited by the AIPs produced by group II or III <I>S. aureus</I> trains. From the engineering perspective, the AIP-II or AIP- III sender cell can be used to inhibit the Agr-I Receiver cell once it is activated. Since AIP-II displays large variation from AIP-I, we are going to develop an AIP-II Sender cell. The AIP-II Sender cell can also be used to inhibit the virulence of Agr group I <I>S. aureus</I>.</P><P style="text-align: center;font-family:'Open Sans', sans-serif;font-size: 13px;">Table 2 Interaction of different AIP signals to other Agr systems.</P><H3 style="color: #00a98f;">Characterization of the P2-GFP composite part in <I>S. aureus</I></H3><P>In addition to the hemolysis zone phenotype, we want to establish another phenotype to reveal the activity of the Agr system. In the Agr locus, two promoters P2 and P3 were found to the direct target of the active AgrA transcription factor. Place the GFP gene under the promoter of P2 or P3 can be used as the reporter of the Agr system. The 2007 Cambridge Team constructed the P2-GFP composite part (BBa_I746105). They intended to test the part in <I>E. coli</I>; however they failed. To characterize whether this P2-GFP part can be function in the Gram-positive strain, we test this composite part directly in <I>S. aureus</I>. The P2-GFP composite fragment was cut by restriction endonuclease from the BBa_I746105 part, then the fragment was inserted at the same restriction site of the shuttle vector pLI50 (Fig. 5A) by ligation, the result plasmid named pLI50-P2-GFP (Fig. 5B). The constructed pLI50-P2-GFP was then verified by restriction endonuclease digestion (Fig. 6) and sequencing.</P><P style="text-align: center;font-family:'Open Sans', sans-serif;font-size: 13px;">Fig. 5 Map of pLI50 (A) and pLI50-P2-GFP (B).</P><P style="text-align: center;font-family:'Open Sans', sans-serif;font-size: 13px;">Fig. 6 Confirmation of the pLI50-P2-GFP by restriction endonuclease digestion.</P><P>After that the pLI50-P2-GFP was transformed into the <I>S. aureus</I> strain RN4220. Strong green fluorescence was observed from RN4220::pLI50-P2-GFP strain colonies (Fig. 7B), while not any fluorescence was observed from RN4220::pLI50 strain on plate (Fig. 7A). This data suggest that the P2-GFP composite can be functional in <I>S. aureus</I> when the Agr system is present. We want further to know whether this part can be used to reveal the varied activity of the four Agr types, we transformed the pLI50-P2-GFP purified from RN4220 into Newman(Agr type I), and N315 (Agr type II). We found that the Newman::pLI50-P2-GFP strain emitted strong green fluorescence; however, barely detectable green fluorescence can be observed of the N315::pLI50-P2-GFP strain (data not shown). The expression of P2-GFP in Newman and N315 is consistent with their hemolysis activity on the blood agar plate. These data collectively confirmed that the P2-GFP composite part can be used to track the activity of the Agr system in <I>S. aureus</I>.</P><P style="text-align: center;font-family:'Open Sans', sans-serif;font-size: 13px;">Fig. 7 White light (A) or fluorescence (B) of RN4220::pLI50-P2-GFP strain on plate.</P><H3 style="color: #00a98f;">Measurement of fluorescence curve along time</H3><P>To explore the dynamics of the autoinduction of the synthetic AIP system, we track the green fluorescence intensity of the RN4220::pLI50-P2-GFP strain along time using the Microplate Reader. As shown in the fig. 8, we found that the expression of GFP increased quickly and steadily, and finally reached a plateau. This result is consistent with the autoinduction prediction of the composite part. The same fluorescence changes along time have been observed in Newman::pLI50-P2-GFP (data no shown).</P><P style="text-align: center;font-family:'Open Sans', sans-serif;font-size: 13px;">Fig. 8 Fluorescence curve along time</P><H3 style="color: #00a98f;">Construction of the recombinant plasmids used in <I>L. lactis</I></H3><P>To construct the autoinduction, AIP sender, and AIP receiver system in <I>L. lactis</I>, corresponding genes will be cloned in shuttle plasmid pMG36e (Fig. 9A), which has erythromycin resistance only. Erythromycin resistance has been reported to be a useful selectable marker in <I>E. coli</I> or <I>L. lactis</I>. However, in our hands, erythromycin selection was usually problematic in <I>E. coli</I>, giving rise to high background levels of untransformed cells. Since selection for kanamycin resistance is much more efficient in <I>E. coli</I>, kanamycin resistance gene with promoter from pEASY was cloned in pMG36e EcoRI site by seamless clone. The resulted plasmid, pMG36ek, contains kanamycin- and erythromycin-resistance (Fig. 9B). The constructed pMG36ek was then verified by restriction endonuclease digestion (Fig. 10) and sequencing.</P><P style="text-align: center;font-family:'Open Sans', sans-serif;font-size: 13px;">Fig. 9 pMG36e (A) and pMG36ek (B) Map</P><P style="text-align: center;font-family:'Open Sans', sans-serif;font-size: 13px;">Fig. 10 Confirmation of the pMG36ek and pMG36ek-P2-GFP by restriction endonuclease digestion analysis</P><P>The agrB-agrD fragments were amplified from genome of four types of <I>S. aureus</I>, and then were cloned at the downstream of P32 promotor in pMG36ek by seamless clone. The resulted plasmids were named pMG36ek-BD-I (from Newman, Agr-I), pMG36ek-BD-II (from N315, Agr-II), and pMG36ek-BD-IV (from XQ, Agr-IV) (Fig. 11). Unfortunately, we can not get the pMG36ek-BD-III (from MW2, Agr-III) plasmid. Maybe BD-III from MW2 is toxic to <I>E. coli</I>. Then, the pMG36ek-BD-I/II/IV were transformed into the <I>L. lactis</I> strain NZ9000. Then, the AIP-producing senders, NZ9000::pMG36ek-BD-I, NZ9000::pMG36ek-BD-II, and NZ9000::pMG36ek-BD-IV, were constructed.</P><P style="text-align: center;font-family:'Open Sans', sans-serif;font-size: 13px;">Fig. 11 Confirmation of the pMG36ek-BD-I/II/IV by restriction endonuclease digestion analysis</P><P>We originally planned to construct the AIP receiver or autoinduction systems by cloning agrC-agrA, or argBDCA fragment at the downstream of P32 in pMG36ek. To test whether the AIP receiver or autoinduction system work or not, gfp gene with promoter P2 (P2-GFP) from BBa_I746105 was cloned at the EcoRI site of pMG36ek, which produced pMG36ek-P2-GFP (Fig. 11). If the agrC-agrA fragment was cloned at the downstream of promoter P32 in pMG36ek-P2-GFP, the expressed protein AgrC and AgrA will receive the signal of corresponding AIP, and induce gfp gene at the downstream of P2 promoter. Unfortunately again, we can not get the plasmid, maybe the agrC-agrA/argBDCA fragment was toxic to <I>E. coli</I> and <I>L. lactis</I> also. We also used various of promoter, such as P2 or PnisZ (nisin inducible promoter in NZ9000), to instead promoter P32 , but we still not get the corresponding vectors. Then, the <I>S. aureus</I> Newman (Agr-I) carrying pLI50-P2-GFP plasmid were used as AIP receiver to test the function of AIP sender (NZ9000::pMG36ek-BD-I/II/IV).</P><P style="text-align: center;font-family:'Open Sans', sans-serif;font-size: 13px;">Fig. 12 pMG36ek-P2-GFP Map</P><H3 style="color: #00a98f;">Test the AIP-producing Sender/receiver synthetic communication system cell with <I>S. aureus</I></H3><P>As the receiver plasmid (including fragments agrC-agrA, or argBDCA) can not be constructed, <I>S. aureus</I> containing argBDCA operon was used as receiver. As shown in table 2, different AIP signals cross-inhibit the activity of the others. For example, AIP-I will activate group I Agr system and inhibited group II or III Agr system, and the most significant phenomenon change of inhibition is the change of hemolysis zone on rabbit blood agar plate. Therefore, <I>S. aureus</I> strains Newman (Agr-I) was co-cultured with NZ9000 or NZ9000::pMG36ek-BD-I/II/IV on rabbit blood agar plate. We can see the hemolysis zone of each co-culture colony has no significant change at size or transparent (Fig. 13). This phenomenon maybe caused by low concentration of AIP signals secreted by <I>L. lactis</I>. Then the NZ9000 or NZ9000::pMG36ek-BD-I/II/IV were cultured 24 hr, and the supernatant were filtered by 0.22 μm filter, and concentrated at ten times. The treated supernatants were co-cultured with strain Newman on rabbit blood agar plate. We can see the hemolysis zone of each co-culture colony has no significant change too (Fig. 14). This test indicated that, the sender (NZ9000::pMG36ek-BD-I/II/IV) did not produce functional AIP signals.</P><P style="text-align: center;font-family:'Open Sans', sans-serif;font-size: 13px;">Fig. 13 <I>S. aureus</I> strains Newman (Agr-I) co-cultured with NZ9000 (A), NZ9000::pMG36ek-BD-I (B), NZ9000::pMG36ek-BD-II (C), and NZ9000::pMG36ek-BD-IV (D) on rabbit blood agar plate.</P><P style="text-align: center;font-family:'Open Sans', sans-serif;font-size: 13px;">Fig. 14 <I>S. aureus</I> strains Newman (Agr-I) co-cultured with the supernatant of NZ9000 (A), NZ9000::pMG36ek-BD-I (B), NZ9000::pMG36ek-BD-II (C), and NZ9000::pMG36ek-BD-IV (D) on rabbit blood agar plate.</P></DIV></DIV></DIV></DIV><DIV class="agileinfo_copyright"><DIV class="container"><DIV class="row clearfix"><DIV class="col-md-4 column"><P style="font-family:'Open Sans', sans-serif">Contact us
                   Email: <A href="Mailto:igem@tmmu.edu.cn" style="color:#000">igem@tmmu.edu.cn</A>
                   Address:Third Military Medical University, No.30 Gaotanyan Street Shapingba District, Chongqing, P.R.China 400038
                </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>