<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TMMU-China_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TMMU-China/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="agileits_w3layouts_banner_nav"><NAV class="navbar navbar-default"><DIV class="collapse navbar-collapse navbar-right" id="bs-example-navbar-collapse-1"><NAV><UL class="nav navbar-nav"><LI><A href="https://2017.igem.org/Team:TMMU-China" class="hvr-underline-from-center">Home</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:TMMU-China/Description" class="dropdown-toggle hvr-underline-from-center" data-toggle="dropdown">Project</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/SyntheticQS1">Synthetic QS for <I>L. lactis</I></A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Demonstrate">Synthetic QS for <I>B. subtilis</I></A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Contribution">Contribution</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Model">Modeling</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Notebook">Notebook</A></LI></UL><LI><A href="https://2017.igem.org/Team:TMMU-China/Parts" class="hvr-underline-from-center">Parts</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:TMMU-China/HP" class="dropdown-toggle hvr-underline-from-center" data-toggle="dropdown">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/HP/Silver">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/HP/Gold_Integrated">Integrated and Gold</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:TMMU-China/Team" class="dropdown-toggle hvr-underline-from-center" data-toggle="dropdown">Team</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Safety" class="hvr-underline-from-center">Safety</A></LI></NAV></DIV></NAV></DIV><DIV id="myCarousel" class="carousel slide"><A class="carousel-control left" href="#myCarousel" data-slide="prev">‹</A><A class="carousel-control right" href="#myCarousel" data-slide="next">›</A></DIV><DIV class="banner-bottom"><DIV class="container"><DIV class="mid_agile_bannner_top_info"><H1><SPAN>Background</SPAN></H1><DIV class="col-md-12"><H3 style="color: #00a98f;">The RRNPP family</H3><P>In addition to be sensed by the receptor histidine kinase of two component system, the peptide can be actively imported back into bacteria by the oligopeptide transporter Opp, at which point they interact with sensors in the cytoplasm. These sensors belong to the RRNPP family of cytoplasmic regulatory receptors. The RRNPP family is named for the prototypical members, Rap, Rgg, NprR, PlcR, and PrgX. The RRNPP members that have been studied are found in bacilli, streptococci, or enterococci. The C-terminal regions of the RRNPP proteins adopt a tetratricopeptide repeat (TPR) domain-like conformation which binds the signaling peptides. The signaling peptides are linear and exhibit mature lengths between 5 and 10 amino acids. With roles in virulence, natural competence, sporulation, biofilm formation, and other activities, RRNPP signaling pathways stand as attractive targets for treatments aimed at manipulating bacterial behaviors. </P><P style="text-align: center;font-family:'Open Sans', sans-serif;font-size: 13px;">Figure 1. Schematic representation of the RRNPP mechanism involving transcriptional regulators, cited from reference [1].</P><H3 style="color: #00a98f;">The AimR-AimP system of <I>Bacillus subtilis </I>bactriophage phi3T</H3><P>The bacteriophage can employ the RRNPP family proteins to make the lysis-lysogeny decision. In a paper published on Nature in 2017, the authors found the <I>B. subtilis</I> bacteriophage phi3T encode the Aim system to make lysis-lysogeny decision. The AimR is a transcription factor, which has a helix-turn-helix domain to bind DNA and a TPR domain to bind the signal peptide. The AimP is the propeptide of the mature signal peptide, sequence of the mature signal peptide is SAIRGA. Binding of AimP to the AimR will disrupt the dimer forms of AimR. After that, the AimR can no longer bind to the promoter of AimX, a potential non coding RNA involved in the process of lysis-lysogeny. A superb characteristic of the Aim system is that the AimR only bind to the promoter of the AimX gene. The ChIP-seq data revealed that there is only one binding site of the AimR in the whole hpi3T genome. This data indicate that the specificity of the Aim system is very high, which is the required characteristic for the synthetic QS system.</P><P style="text-align: center;font-family:'Open Sans', sans-serif;font-size: 13px;">Figure 2. The AimR-AimP system and its role in the phage lysis-lysogeny decision, cited from reference [2]. </P><H3 style="color: #00a98f;">The PlcR-PapR system of <I>B.cereus</I></H3><P><I>B. cereus</I> cause acute diarrheal disease by the production and secretion of a variety of hemolysins, phospholipases, and toxins. The production of virulence factors is controlled by the PlcR-PapR QS system. PapR is 48 amino acids long and contains an N-terminal signal peptide that targets it for secretion. Outside the cell, the PapR pro-AIP is processed by the secreted neutral protease B (NprB) to form the active AIP. The mature PapR oligopeptide sequence is ADVPFEL. The processed PapR AIP is imported back into the cell by the oligopeptide permease system (Opp). The PlcR protein has two domains. The helix-turn-helix domain is involved in the DNA binding activity. The TPR domain is involved in signal peptide binding. Inside the cell, PapR binds to the transcription factor PlcR, and this causes conformational changes in the DNA-binding domain of PlcR, facilitates PlcR oligomerization, DNA binding, and regulation of transcription. The plcA gene is under the control of PlcR, its promoter is usually used to construct reporter system.</P><P style="text-align: center;font-family:'Open Sans', sans-serif;font-size: 13px;">Figure 3. PlcR activation Model, cited from reference [3].</P><H3 style="color: #00a98f;">Introduction of <I>B. subtilis</I></H3><P>As a model organism, <I>B. subtilis</I> is commonly used in laboratory studies directed at discovering the fundamental properties and characteristics of Gram-positive spore-forming bacteria. Due to its excellent fermentation properties, with high product yields (20 to 25 gram per litre) it is used to produce various enzymes, such as amylase and proteases. Other enzymes produced by <I>B. subtilis</I> are widely used as additives in laundry detergents. It is also used to produce hyaluronic acid, which is used in the joint-care sector in healthcare and cosmetics. <I>B. subtilis</I> is the most widely used Gram positive bacteria chassis in synthetic biology. It has its own QS systems. However, to avoid interfering with its own physiology, synthetic communication system is needed for this important chassis.</P><P style="font-family:'Open Sans', sans-serif;"><B>References:</B>[1] Perez-Pascual, D., Monnet, V., and Gardan, R. (2016). Bacterial Cell-Cell Communication in the Host via RRNPP Peptide-Binding Regulators. Front Microbiol 7, 706.
   [2] Erez, Z., Steinberger-Levy, I., Shamir, M., Doron, S., Stokar-Avihail, A., Peleg, Y., Melamed, S., Leavitt, A., Savidor, A., Albeck, S., et al. (2017). Communication between viruses guides lysis-lysogeny decisions. Nature 541, 488-493.
   [3] Grenha, R., Slamti, L., Nicaise, M., Refes, Y., Lereclus, D., and Nessler, S. (2013). Structural basis for the activation mechanism of the PlcR virulence regulator by the quorum-sensing signal peptide PapR. Proc Natl Acad Sci U S A 110, 1047-1052.</P></DIV></DIV></DIV></DIV><DIV class="banner-bottom"><DIV class="container"><DIV class="mid_agile_bannner_top_info"><H1><SPAN>Design</SPAN></H1><DIV class="col-md-12"><P>By combining the expression of AimR and AimP components, we want to develop a synthetic QS system in <I>B. subtilis</I> for target gene autoinhibition (Figure 4A). A synthetic communication pathway between <I>B. subtilis</I> strains by co-culturing AimP-producing “sender” cells with AimR-sensing “receiver” cells to inhibit gene expression will also be constructed (Figure 4B). To develop a synthetic QS system in <I>B. subtilis</I> for target gene autoinduction, we are going to combine the expression of PlcR and PapR components (Figure 4C). Furthermore, we will develop a synthetic communication pathway between <I>B. subtilis</I> strains by co-culturing PapR-producing “sender” cells with PlcR-sensing “receiver” cells to induce gene expression (Figure 4D). These new tools will be vital for controlling gene expression in this industrially important Gram positive host, and may lead to the expanded use of <I>B. subtilis</I> in laboratory and industrial settings. The detailed design of these composite parts were shown in Figure 5.</P><P style="text-align: center;font-family:'Open Sans', sans-serif;font-size: 13px;">Figure 4. The design of (A) autoinhibiton system, (B) AimR-AimP based Sender and Receiver cells, (C) autoinduction system, (D) PlcR-PapR based Sender and Receiver Cells.</P><P style="text-align: center;font-family:'Open Sans', sans-serif;font-size: 13px;">Figure 5. (A) The design of Sender, Receiver and Autoinhibition composite parts of the AimR-AimP system using the promoter of the Aim operon. (B) The design of Sender, Receiver and Autoinduction composite parts of the PlcR-PapR system using the constitutive promoter pVeg.</P></DIV></DIV></DIV></DIV><DIV class="banner-bottom"><DIV class="container"><DIV class="mid_agile_bannner_top_info"><H1><SPAN>Results</SPAN></H1><DIV class="col-md-12"><H3 style="color: #00a98f;">Construction of the recombinant plasmids</H3><P>All the constructs were made using standard molecular cloning methods or seamless cloning methods. Here we take the pDG1730-pAim-AimR-AimP-pAimX-GFP (abbreviated as pDG1730-Aim-GFP) construct as an example. The Aim sequence was synthesized. After that, we cloned the pAim-AimR-AimP-pAimX fragment using PCR. We also cloned the GFP sequence by PCR. The two fragments were purified and assembled with the enzyme digested pDG1730 plasmid using seamless cloning method. The pDG1730 plasmid is used to knock the composite part into the genome of <I>B. subtilis</I>. The amyE gene upstream and downstream sequences were used to target the composite part into the amyE locus. The Spc gene confers resistance to spectinomycin. </P><P style="text-align: center;font-family:'Open Sans', sans-serif;font-size: 13px;">Figure 6. The construct map of pDG1730</P><P style="text-align: center;font-family:'Open Sans', sans-serif;font-size: 13px;">Figure 7. Construction of the pDG1730-Aim-GFP plasmid</P><H3 style="color: #00a98f;">Construction of the composite part knock-in <I>B. subtilis</I> strains</H3><P>We used natural transformation to transform the composite parts on pDG1730 into the <I>B. subtilis</I> 168 strain. The competent cells of <I>B. subtilis</I> 168 strain were induced using nutrient-limited medium. The knock-in strains were selected by 100 mg/mL spectinomycin. The genomic DNA of the knock-in strains were extracted and used as the PCR template. Specific primers were used to conduct PCR to confirm the integration of the composite part.</P><H3 style="color: #00a98f;">Characterization of the pAim-AimR-AimP-pAimX-GFP composite part in <I>B. subtilis</I></H3><P>We also incorporate the pDG1730 vector into the <I>B. subtilis</I> 168 strain. Compared to this reference strain, we found that the pAim-AimR-AimP-pAimX-GFP composite part knocked-in strain can emit green fluorescence. </P><P style="text-align: center;font-family:'Open Sans', sans-serif;font-size: 13px;">Figure 8. Characterization of the pAim-AimR-AimP-pAimX-GFP composite part</P><H3 style="color: #00a98f;">Measurement of fluorescence curve along time</H3><P>To explore the dynamics of the auotoinhibition of the synthetic AimR-AimP system, we track the green fluorescence intensity of the pAim-AimR-AimP-pAimX-GFP knock-in strain along time using the Microplate Reader. As shown in the Figure 9, we found that the expression of GFP increased steadily to a peak, and then the green fluorescence intensity decreased and finally reached a plateau. This result is consistent with the autoinhibition prediction of the pAim-AimR-AimP-pAimX-GFP composite part. </P><P style="text-align: center;font-family:'Open Sans', sans-serif;font-size: 13px;">Figure 9. Fluorescence curve of the pAim-AimR-AimP-pAimX-GFP composite part along time.</P></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row clearfix"><DIV class="col-md-4 column"><P style="font-family:'Open Sans', sans-serif;font-size:12px;">Contact us
                   Email: <A href="Mailto:igem@tmmu.edu.cn" style="color:#000">igem@tmmu.edu.cn</A>
                   Address:Third Military Medical University, No.30 Gaotanyan Street Shapingba District, Chongqing, P.R.China 400038
                </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>