<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TMMU-China_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TMMU-China/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="agileits_w3layouts_banner_nav"><NAV class="navbar navbar-default"><DIV class="collapse navbar-collapse navbar-right" id="bs-example-navbar-collapse-1"><NAV><UL class="nav navbar-nav"><LI><A href="https://2017.igem.org/Team:TMMU-China" class="hvr-underline-from-center">Home</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:TMMU-China/Description" class="dropdown-toggle hvr-underline-from-center" data-toggle="dropdown">Project</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/SyntheticQS1">Synthetic QS for <I>L. lactis</I></A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Demonstrate">Synthetic QS for <I>B. subtilis</I></A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Contribution">Contribution</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Model">Modeling</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Notebook">Notebook</A></LI></UL><LI><A href="https://2017.igem.org/Team:TMMU-China/Parts" class="hvr-underline-from-center">Parts</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:TMMU-China/HP" class="dropdown-toggle hvr-underline-from-center" data-toggle="dropdown">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/HP/Silver">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/HP/Gold_Integrated">Integrated and Gold</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:TMMU-China/Team" class="dropdown-toggle hvr-underline-from-center" data-toggle="dropdown">Team</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Safety" class="hvr-underline-from-center">Safety</A></LI></NAV></DIV></NAV></DIV><DIV id="myCarousel" class="carousel slide"><A class="carousel-control left" href="#myCarousel" data-slide="prev">‹</A><A class="carousel-control right" href="#myCarousel" data-slide="next">›</A></DIV><DIV class="banner-bottom"><DIV class="container"><DIV class="mid_agile_bannner_top_info"><H1><SPAN>Journal</SPAN></H1><DIV class="col-md-12"><P><B>December</B>：We began to recruit team members to establish the 2017’s team,  and team members consulted literature and widely read wikis in previous years to have a general understanding about iGEM.</P><P><B>January</B>：We held quite a few times of brainstorms to discuss the potential project. </P><P><B>March</B>：After a couple of months of learning and training, 14 team members were gathered to establish the TMMU-China team. Team registration was completed. And we set our project .</P><P><B>April</B>：We attended iGEM Southwest Union and had an overall assessment of the safety and feasibility of our project.</P><P><B>June</B>：The <I>B. subtilis</I> and <I>L. lactis</I> strain was constructed and the utility of it was tested.</P><P><B>July</B>：We completed the optimizations of the system, and constructed the plasmid containing the transcription factor AimR and PlcR, signal peptide gene AimP and PapR. Some of the members set out to design and finish the mathematic modeling.</P><P><B>August</B>：We attended the 2017 CCiC(Central China iGEM Consortium) held in Fujian Agriculture and Forestry University, receiving advice from Igem headquarter King L. CHOW and former iGEMer Zhang Haoqian.  And together with SCU-WestChina , we cooperated in helding the exhibition in Chengdu Public Library, publicizing “GMO &amp; Human Health”.</P><P><B>September</B>：We completed the construction of all the 4 Parts.</P><P><B>October-November</B>：We at last perfected our preparation for presentation, poster and wiki. </P></DIV></DIV></DIV></DIV><DIV class="banner-bottom"><DIV class="container"><DIV class="mid_agile_bannner_top_info"><H1><SPAN>Protocols</SPAN></H1><DIV class="col-md-12"><H4 style="color: #00a98f;">1. Bacteria and culture media</H4><P>Bacteria strains used in this study are NZ9000 (<I>L. lactis</I>) and DH5α (<I>E. coli</I>). DH5α was used as the cloning host. <I>E. coli</I> were cultured in Luria-Bertani (LB) medium (agar or broth) at 37 °C. <I>L. lactis</I> was cultured in M17GS (M17 broth supplemented with 0.5 % (wt/vol) glucose, 0.55 % (wt/vol) sucrose) medium at 30 °C (agar or broth). Kanamycin was used at the concentration of 50 μg/mL for <I>E. coli</I>. Erythromycin was used at the concentration of 20 μg/mL for <I>L. lactis</I>. </P><H4 style="color: #00a98f;">2.Preparation for competent <I>B. subtilis</I> cells</H4><P>Streak out the strain to be made competent on an LB or TBAB agar plate as a large patch and incubate overnight at 30 ℃. The following morning scrape off the cell growth off the plate and use to inoculate fresh, pre-warmed, SpC medium to give an OD600 reading of about 0.5. Incubate the culture at about 37℃ with vigorous aeration and take periodic OD readings to assess cell growth. When the rate of cell growth is seen to depart from exponential inoculate 200ml of pre-warmed, SpⅡmedium with 2ml of stationary-phase culture and continue incubation at 37℃ with slower aeration. After 90min incubation, pellet the cells by centrifugation at room temperature. Carefully decant the supernatant into a sterile container and save. Gently resuspend the cell pellet in 18ml of saved supernatant and add 2ml of sterile glycerol; mix gently. Aliquot the competent cells in sterile tubes, freeze rapidly in liquid nitrogen or a dry-ice bath and store at -70℃.</P><H4 style="color: #00a98f;">3. Polymerase Chain Reaction</H4><P>All PCR reactions were done with Fast Taq Mix DNA polymerase from Novoprotein Scientific Inc (Shanghai, China) and Primerstar Max DNA polymerase from Takara Bio Inc. (Dalian, China).</P><P><B>(1). Fast Taq Mix DNA polymerase</B></P><P>For colony PCR, Fast Taq polymerase is used. As the template, the colony of <I>E. coli</I> is picked from a plate and culture the <I>E. coli</I> when the medium become turbid. The component and cycling conditions are listed in the table below.</P><DIV class="row clearfix"><DIV class="col-md-12 column" align="center"><TABLE class="tg"><TBODY><TR><TH class="biaoge" colspan="2">2×Fast Taq Mix(SinoBio)</TH><TH class="biaoge" colspan="3">Cycling conditions</TH></TR><TR><TH class="biaoge">Component</TH><TH class="biaoge">Volume(μL)</TH><TH class="biaoge">Tempreture(℃)</TH><TH class="biaoge">Time</TH><TH class="biaoge">Cycle</TH></TR><TR><TD class="biaoge">Premix Taq</TD><TD class="biaoge">5</TD><TD class="biaoge">94</TD><TD class="biaoge">5 min</TD><TD class="biaoge">1</TD></TR><TR><TD class="biaoge">Bacteria solution</TD><TD class="biaoge">1</TD><TD class="biaoge">94</TD><TD class="biaoge">30 sec</TD><TD class="biaoge" rowspan="3">35</TD></TR><TR><TD class="biaoge">Primer F (20 μM)</TD><TD class="biaoge">0.2</TD><TD class="biaoge">55</TD><TD class="biaoge">30 sec</TD></TR><TR><TD class="biaoge">Primer R (20 μM)</TD><TD class="biaoge">0.2</TD><TD class="biaoge">72</TD><TD class="biaoge">1 kb/min</TD></TR><TR><TD class="biaoge">dd H<SUB>2</SUB>O</TD><TD class="biaoge">3.6</TD><TD class="biaoge">72</TD><TD class="biaoge">5 min</TD><TD class="biaoge">1</TD></TR><TR><TD class="biaoge">Total</TD><TD class="biaoge">10</TD><TD class="biaoge">16</TD><TD class="biaoge">1 min</TD><TD class="biaoge">1</TD></TR></TBODY></TABLE></DIV></DIV><P><B>(2). Primerstar Max DNA polymerase</B></P><P>Primerstar Max DNA polymerase is used when high fidelity is needed. Primerstar Max has been used for amplifying DNA fragments and generating specific mutations in genes with the use of primers. The cycling program is listed in the table below.</P><DIV class="row clearfix"><DIV class="col-md-12 column" align="center"><TABLE class="tg"><TBODY><TR><TH class="biaoge" colspan="2">2×PrimerStar Max(Takara)</TH><TH class="biaoge" colspan="3">Cycling conditions</TH></TR><TR><TH class="biaoge">Component</TH><TH class="biaoge">Volume(μL)</TH><TH class="biaoge">Tempreture(℃)</TH><TH class="biaoge">Time</TH><TH class="biaoge">Cycle</TH></TR><TR><TD class="biaoge">PrimerStar Max</TD><TD class="biaoge">25</TD><TD class="biaoge">98</TD><TD class="biaoge">5 min</TD><TD class="biaoge">1</TD></TR><TR><TD class="biaoge">DNA Template</TD><TD class="biaoge">1</TD><TD class="biaoge">98</TD><TD class="biaoge">10 sec</TD><TD class="biaoge" rowspan="3">35</TD></TR><TR><TD class="biaoge">Primer F (20 μM)</TD><TD class="biaoge">1</TD><TD class="biaoge">52</TD><TD class="biaoge">15 sec</TD></TR><TR><TD class="biaoge">Primer R (20 μM)</TD><TD class="biaoge">1</TD><TD class="biaoge">72</TD><TD class="biaoge">10 sec/min</TD></TR><TR><TD class="biaoge">dd H<SUB>2</SUB>O</TD><TD class="biaoge">22</TD><TD class="biaoge">72</TD><TD class="biaoge">5 min</TD><TD class="biaoge">1</TD></TR><TR><TD class="biaoge">Total</TD><TD class="biaoge">50</TD><TD class="biaoge">16</TD><TD class="biaoge">1 min</TD><TD class="biaoge">1</TD></TR></TBODY></TABLE></DIV></DIV><H4 style="color: #00a98f;">4. Restriction enzyme digestion and T4 DNA ligation</H4><P><B>(1).Restriction enzymes used in this work were FastDigest enzymes (Thermo Scientific). </B></P><P>Combine the following reaction components list at room temperature and the mixtures were incubated with recommended temperatures and time.</P><DIV class="row clearfix"><DIV class="col-md-12 column" align="center"><TABLE class="tg"><TBODY><TR><TH class="biaoge">Component</TH><TH class="biaoge">Volume</TH></TR><TR><TD class="biaoge">FastDigest Enzyme 1</TD><TD class="biaoge">2.5 μL</TD></TR><TR><TD class="biaoge">FastDigest Enzyme 2</TD><TD class="biaoge">2.5 μL</TD></TR><TR><TD class="biaoge">FastDigest Green Buffer</TD><TD class="biaoge">5 μL</TD></TR><TR><TD class="biaoge">Plasmid or DNA fragment</TD><TD class="biaoge">up to 5 μg</TD></TR><TR><TD class="biaoge">dd H<SUB>2</SUB>O</TD><TD class="biaoge">add to 50μL</TD></TR><TR><TD class="biaoge">Total</TD><TD class="biaoge">50 μL</TD></TR></TBODY></TABLE></DIV></DIV><P>The digestion productions were purified by the Wizard SV Gel and PCR Clean-Up Kit (Promega Corporation).</P><P><B>(2).DNA ligation used in this work was DNA Ligation Kit Ver.2.1 from Takara Bio Inc. (Dalian, China).</B></P><P>Mix linearized plasmid vector DNA and a DNA fragment in a total volume of 2.5 μL. The amounts of vector: fragment is 0.03 pmol : 0.03 - 0.3 pmol. Add an equal volume of ligation mixture (2.5 μL) as the DNA solution and mix thoroughly. Incubate at 16 °C for 30 minutes.</P><H4 style="color: #00a98f;">5. Seamless Cloning</H4><P>For cloning our target fragments into vectors, NovoRec seamless cloning KIT from Novoprotein Scientific Inc. was used. Seamless cloning can simultaneously combine one or more PCR products with a linearized vector when the DNA to be joined shares 15 – 25 bp of homology at each end. The required homology can be easily generated by adding complementary sequence to the ends of the PCR primers. No additional treatment of the PCR fragment is required, such as restriction digestion, ligation, phosphorylation, or blunt-end polishing. The efficiency of NovoRec PCR Seamless Cloning is over 95 %.</P><P><B>(1)</B>. The acquisition of the linearized vector by enzyme digestion (&gt; 15 ng/μL). <B>(2)</B>. A small sequence (15-25 bps) overlapped with the end of the cloning site will be added onto the insert through a PCR step. <B>(3)</B>. Recombination reaction.</P><DIV class="row clearfix"><DIV class="col-md-12 column" align="center"><TABLE class="tg"><TBODY><TR><TH class="biaoge">Component</TH><TH class="biaoge">Volume</TH></TR><TR><TD class="biaoge">ddH<SUB>2</SUB>O</TD><TD class="biaoge">up to 20 μL</TD></TR><TR><TD class="biaoge">5 × Buffer</TD><TD class="biaoge">4 μL</TD></TR><TR><TD class="biaoge">Linearized cloning vector</TD><TD class="biaoge">0.03 pmol</TD></TR><TR><TD class="biaoge">PCR products of insertions</TD><TD class="biaoge">0.09 ~ 0.3 pmol</TD></TR><TR><TD class="biaoge">Enzyme</TD><TD class="biaoge">2 μL</TD></TR></TBODY></TABLE></DIV></DIV><P>The recommended amount of vector for recombination reaction is 0.03 pmol. The recommended amount of insertion for recombination reaction is 0.09 ~ 0.3 pmol.</P><P><B>(4)</B>. Incubate the mixture at 37 °C for 20 min and transform it into <I>E. coli</I>.</P><H4 style="color: #00a98f;">6.Preparation of chemically competent <I>E. coli</I> cells</H4><P>Inoculate 2ml LB broth with an aliquot (about 50ul)of the desired <I>E. coli</I> from the -80℃ freezer stock of cells. Incubate for 2h at 37℃. Add the 2ml seed culture to 250ml LB broth and grow at 37℃, shaking (about 200rpm) until OD600 of 0.3-0.4 (about 5 hours).Pre-cool the 50ml polypropylene tube, 80 EP tubes, CaCl2-glycerine (0.1mol/L CaCl2) and CaCl2- MgCl2 (80mmol/L MgCl2, 20mmol/L CaCl2). Set the centrifuge and prepare the ice tray. Transfer the bacteria into the 50ml polypropylene tube. Place it on ice for 10 minutes. Centrifuge at 4℃, 4100rpm for 10 minutes. Discard supernatant, then place the tube upside down to make sure trace liquid medium runs out. Add 30ml of pre-cooled CaCl2- MgCl2 per 50ml of initial liquid medium to resuspend bacteria cell pellet. Centrifuge at 4℃, 4100rpm for 10 minutes. Discard supernatant then place the tube upside down to make sure trace liquid medium runs out. Add 2ml of pre-cooled CaCl2 per 50ml of initial liquid medium to resuspend bacteria cell pellet. Transfer to EP tubes (50ul every tube) and store at -80℃.</P><H4 style="color: #00a98f;">7. Transformation of <I>E. coli</I></H4><P>Example Protocol: Standard heat-shock transformation of chemically competent bacteria.</P><P>Take competent cells out of -80°C and thaw on ice (approximately 20-30min). Take agar plates (containing the appropriate antibiotic) out of 4°C to warm up to room temperature or place in 37°C incubator. Mix 1 to 5μl of DNA (usually 10pg to 100ng) into 20-50μL of competent cells in a microcentrifuge or falcon tube. GENTLY mix by flicking the bottom of the tube with your finger a few times. Note: Transformation efficiencies will be approximately 10-fold lower for ligation of inserts to vectors than for an intact control plasmid. Place the competent cell/DNA mixture on ice for 20-30min. Heat shock each transformation tube by placing the bottom 1/2 to 2/3 of the tube into a 42°C water bath for 30-60 seconds (45sec is usually ideal, but this varies depending on the competent cells you are using). Put the tubes back on ice for 2 min. Add 250-500μl LB or SOC media (without antibiotic) and grow in 37°C shaking incubator for 45min. Note: This outgrowth step allows the bacteria time to generate the antibiotic resistance proteins encoded on the plasmid backbone so that they will be able to grow once plated on the antibiotic containing agar plate. This step is not critical for Ampicillin resistance but is much more important for other antibiotic resistances. Plate some or all of the transformation onto a 10cm LB agar plate containing the appropriate antibiotic. Note: We recommend that you plate 50μL on one plate and the rest on a second plate. This gives the best chance of getting single colonies, while allowing you to recover all transformants. Incubate plates at 37°C overnight.</P><H4 style="color: #00a98f;">8.Transformation of <I>B. subtilis</I></H4><P>Thaw competent cells rapidly by immersing frozen tubes in a 37 ℃ water bath. Immediately, add one volume of SpⅡEGTA to the thawed cells; mix gently. In a sterile test tube add competent cells to the DNA solution and incubate in a roller drum at 37℃. Dilute the transformed cells as appropriate in T Base containing 0.5% glucose and plate immediately onto selective media.</P><H4 style="color: #00a98f;">9. Transformation of <I>Lactococcus lactis</I></H4><P><B>(1) Preparation of competent <I>L. lactis</I> cells</B></P><P>To obtain competent cells, the cultures were grown to an optical density at 600 nm of 0.5 to 0.8 and then diluted 100-fold in SGM17 (M17GS containing 0.5 M sucrose) supplemented with glycine. After growth at 30 °C to an optical density at 600 nm of 0.2 to 0.4, the cells were harvested by centrifugation at 4 °C at 5,000x g. Following two washes in ice-cold 0.5 M sucrose containing 10% glycerol, the cells were suspended in 1/100 culture volume of washing solution and then stored in aliquots at -85 °C until use.</P><P><B>(2) Transformation by electroporation</B></P><P>The cell suspensions were thawed on ice. Portions (40 μL) were mixed with 1 μL of DNA and then transferred to an ice-cooled electroporation cuvette (2-mm electrode gap) and exposed to a single electrical pulse. The pulse was delivered by a Gene-Pulser (Bio-Rad, Calif.) set at 25 μF and normally at 2.0 kV. The cuvette was connected in parallel to a 200-Ω resistor (pulse controller; Bio-Rad), resulting in time constants of 4.5 to 5ms.</P><P><B>(3) Incubation and plate</B></P><P>Immediately following the discharge, the suspensions were mixed with 0.96 ml of ice-cold SGM17MC (SGM17 containing 20 mM MgCl2 and 2 mM CaCl2) and left on ice for about 5 min. Appropriate dilutions were then made in SGM17MC, and the cells were incubated at 30°C for 2h. The plates M17GS contained 20 μg/mL of erythromycin for the selection of erythromycin-resistant transformants. Transformants were enumerated after 2 days of incubation at 30°C.</P><H4 style="color: #00a98f;">10.Amylase activity analysis</H4><P>Apply iodine on the plate with 0.1% starch and spectinomycin. Incubate the engineered bacteria and non-engineered bacteria onto the plate at 37℃. See if there are hydrolysis circles on the plate.</P><H4 style="color: #00a98f;">11.Agarose Gel Electrophoresis</H4><P>Weigh agarose powder and TAE buffer according to a proper portion, and add them to a 100ml conical flask (we usually make 1.5% Agarose Gel). Melt the mixture in a microwave until the solution becomes clear (don’t leave the microwave). Let the solution cool down to about 40-50℃ and add DNA gel stain (usually we use EB), pour the solution into the gel casting tray with appropriate comb. Let the gel cool until it becomes solid. Pull out the comb carefully. Place the gel in the electrophoresis chamber. Add enough TAE Buffer so that there is about 2-3mm of buffer over the gel. Pipette DNA samples mixed with appropriate amount of DNA loading buffer (the dye/GeneFinder is in the loading buffer) into wells on the gel. Run the gel at 135V for about twenty minutes.</P></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row clearfix"><DIV class="col-md-4 column"><P style="font-family:'Open Sans', sans-serif;font-size:12px;">Contact us
                   Email: <A href="Mailto:igem@tmmu.edu.cn" style="color:#000">igem@tmmu.edu.cn</A>
                   Address:Third Military Medical University, No.30 Gaotanyan Street Shapingba District, Chongqing, P.R.China 400038
                </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>