<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TMMU-China_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TMMU-China/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Home</TITLE></P><DIV class="agileits_w3layouts_banner_nav"><NAV class="navbar navbar-default"><DIV class="navbar-header navbar-left"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse navbar-right" id="bs-example-navbar-collapse-1"><NAV><UL class="nav navbar-nav"><LI class="dropdown"><A href="https://2017.igem.org/Team:TMMU-China" class="dropdown-toggle hvr-underline-from-center" data-toggle="dropdown">Home </A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China">Abstract</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Background">Background</A></LI></UL><LI class="dropdown"><A href="https://2017.igem.org/Team:TMMU-China/Team" class="dropdown-toggle hvr-underline-from-center" data-toggle="dropdown">Team</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle hvr-underline-from-center" data-toggle="dropdown">Project</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Contribution">Contribution</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Improve">Improve</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Attributions">Attributions</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:TMMU-China/Parts" class="dropdown-toggle hvr-underline-from-center" data-toggle="dropdown">Parts </A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Part_Collection">Part Collection</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Safety" class="hvr-underline-from-center">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle hvr-underline-from-center" data-toggle="dropdown">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/HP/Silver">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/HP/Gold_Integrated">Integrated and Gold</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Engagement">Public Engagement</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle hvr-underline-from-center" data-toggle="dropdown">AWARDS</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Applied_Design"> Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Hardware">Hardware</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Measurement">Measurement</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Plant">Plant</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Software">Software</A></LI><LI><A href="https://2017.igem.org/Team:TMMU-China/Judging" class="hvr-underline-from-center">JUDGING FORM</A></LI></NAV></DIV></NAV></DIV><DIV id="myCarousel" class="carousel slide"><A class="carousel-control left" href="#myCarousel" data-slide="prev">‹</A><A class="carousel-control right" href="#myCarousel" data-slide="next">›</A></DIV><DIV class="banner-bottom"><DIV class="container"><DIV class="mid_agile_bannner_top_info"><H1><SPAN>Results</SPAN></H1><DIV class="col-md-12"><H3>Construction of the recombinant plasmids</H3><P align="left">The Aim sequence was synthesized. After that, we cloned the AimR-AimP-pAimX fragment using PCR. We also cloned the GFP sequence by PCR. The two fragments were purified and assembled with the enzyme digested pDG1730 plasmid. The pDG1730 plasmid is used to knock the composite part into the genome of <I>B.subtilis</I>. The <I>amyE</I> gene upstream and downstream sequences were used to target the composite part into the amyE locus. The Spc gene confers resistance to spectinomycin. </P><P style="text-align: center;font-family:'Open Sans', sans-serif">Fig The construct map of pDG1730</P><H3>Construction of the composite part knock-in <I>B.subtilis</I> strains</H3><P>We used natural transformation to transform the B.subtilis 168 strain. The competent cells of B.subtilis 168 strain were induced using nutrient-limited medium. The knock-in strains were selected by 100 mg/mL spectinomycin. The genomic DNA of the knock-in strains were extracted and used as the PCR template. Specific primers were used to conduct PCR to confirm the integration of the composite part. Here are some examples of the confirmation results.</P><H3>Expression of the GFP reporter in the knock-in strains</H3><P>We also incorporate the pDG1730 vector into the B.subtilis 168 strain. Compared to this reference strain, we found that the pAim-AimR-AimP-pAimX-GFP composite part can emit green fluorescence. However, to our surprise, the pVeg-AimR-AimP-pAimX-GFP composite part can not emit green fluorescence. The phage genes are usually highly packed. In the promoter region of the AimR genes, some regulatory elements might be important for the expression of composite part, thus the pVeg promoter can not control the expression of AimR genes. The exact mechanisms are not known.</P><P style="text-align: center;font-family:'Open Sans', sans-serif">Fig Characterization of the pAim-AimR-AimP-pAimX-GFP composite part</P><H3>Measurement of fluorescence curve along time</H3><P>To explore the dynamics of the auotoinhibition of the synthetic AimR-AimP system, we track the green fluorescence intensity of the pAim-AimR-AimP-pAimX-GFP knock-in strain along time using the Microplate Reader. As shown in the figure, we found that the expression of GFP increased steadily to a peak, and then the green fluorescence intensity decreased and finally reached a plateau. This result is consistent with the autoinhibition prediction of the pAim-AimR-AimP-pAimX-GFP composite part.</P><P style="text-align: center;font-family:'Open Sans', sans-serif">Fig. Fluorescence curve along time.</P><H3>	Selection of the appropriate Agr system from the four Agr types</H3><P>There are four different classes in the Agr systems which are referred to as Agr-I, Agr-II, Agr-III, and Agr-IV. Among the four classes of AIPs, the five-residue thiolactone ring structure is always conserved, while the other ring and tail residues differ. Similarly, the proteins involved in signal biosynthesis (AgrB and AgrD) and surface receptor binding (AgrC) also show variability. The activity of the four Agr classes is varied. To select the appropriate Agr system for engineering, we first conduct the hymolysis experiment with four <I>S. aureus</I> strains containing different Agr classes.</P><P style="text-align: center;font-family:'Open Sans', sans-serif">Table 1 The strains used and their cognate Agr groups</P><DIV class="row clearfix"><DIV class="col-md-12 column" align="center"><TABLE align="center" class="tg"><TBODY><TR><TD class="biaoge">Agr groups</TD><TD class="biaoge">Strain Name</TD></TR><TR><TD class="biaoge">Agr-I</TD><TD class="biaoge">Newman</TD></TR><TR><TD class="biaoge">Agr-II</TD><TD class="biaoge">N315</TD></TR><TR><TD class="biaoge">Agr-III</TD><TD class="biaoge">MW2</TD></TR><TR><TD class="biaoge">Agr-IV</TD><TD class="biaoge">XQ</TD></TR></TBODY></TABLE></DIV></DIV><P>Production of exoproteins such as hemolysins in <I>S. aureus</I> is controlled by the agr system. Different Agr types are usually associated with different hemolytic activity. To establish the link between them, we placed four <I>S. aureus</I> strains from Agr Ⅰ to Agr Ⅳ on rabbit blood agar plate (Fig. 2). The hemolytic activity can be clearly observed from the rabbit blood agar plate. The strain of Agr Ⅰ and Agr Ⅳ types show visible and very transparent hemolysis zone around the bacteria, indicating the hemolytic activity of strain of Agr Ⅰ and Agr Ⅳ are very strong. On the other hand, no hemolysis zone of Agr Ⅱ is observed. The hemolysis zone of Agr Ⅲ is very small. The larger the hemolysis zone, the higher activity of the Agr system. AIP-I and AIP-IV differ by only one amino acid, they are grouped together and can function interchangeably. From this result, we are going to use the Agr Ⅰsystem to do the following experiments. </P><P>Interestingly, different AIP signals cross-inhibit the activity of the others. For example, group I <I>S. aureus</I> quorum sensing can be activated by AIP-I but is inhibited by the AIPs produced by group II or III <I>S. aureus</I> trains. From the engineering perspective, the AIP-II or AIP- III sender cell can be used to inhibit the Agr I Receiver cell once it is activated. Since AIP-II displays large variation from AIP-I, we are going to develop an AIP-II Sender cell. The AIP-II Sender cell can also be used to inhibit the virulence of Agr group I <I>S. aureus</I>.</P><P style="text-align: center;font-family:'Open Sans', sans-serif">Fig. 2 Four <I>S. aureus</I> strains from Agr I to Agr IV on rabbit blood agar plate.</P><H3>Characterization of the P2-GFP composite part in <I>S. aureus</I></H3><P>In addition to the hemolysis zone phenotype, we want to establish another phenotype to reveal the activity of the Agr system. In the Agr locus, two promoters P2 and P3 were found to the direct target of the active AgrA transcription factor. Place the GFP gene under the promoter of P2 or P3 can be used as the reporter of the Agr system. The 2007 Cambridge Team constructed the P2-GFP composite part. They intended to test the part in E. coli; however they failed to do so. To characterize whether this P2-GFP part can function also in the Gram-positive strain, we test this composite part directly in <I>S. aureus</I>. The P2-GFP composite were cut from the pSB1C3-P2-GFP part, then it was ligated into the shuttle vector pLI50. After that the pLI50-P2-GFP was transformed into the <I>S. aureus</I> strain RN4220. We found that the RN4220::pLI50-P2-GFP strain show strong green fluorescence compared to the RN4220::pLI50 strain on plate (Fig. 3). This data suggest that the P2-GFP composite can be functional when the Agr system is present. We want further to know whether this part can be used to reveal the varied activity of the four Agr types, we transformed the pLI50-P2-GFP construct from RN4220 to Newman(Agr typeⅠ), N315 (Agr type Ⅱ). We found that the Newman::pLI50-P2-GFP strain emitted strong green fluorescence, however, barely detectable green fluorescence can be observed of the N315::pLI50-P2-GFP strain. The expression of P2-GFP in Newman and N315 is consistent with their hemolysis activity on the blood agar plate. These data collectively confirmed that the P2-GFP composite part can be used to track the activity of the Agr system.</P><P style="text-align: center;font-family:'Open Sans', sans-serif">Fig. 3 White light (A) or fluorescence (B) of RN4220::pLI50-P2-GFP strain on plate.</P><H3>Measurement of fluorescence curve along time</H3><P>To explore the dynamics of the autoinduction of the synthetic AIP system, we track the green fluorescence intensity of the RN4220::pLI50-P2-GFP strain along time using the Microplate Reader. As shown in the figure, we found that the expression of GFP increased steadily, and finally reached a plateau (Fig. 4). This result is consistent with the autoinduction prediction of the composite part. The same fluorescence changes along time have been observed in Newman::pLI50-P2-GFP (data no shown).</P><P style="text-align: center;font-family:'Open Sans', sans-serif">Fig. 4 Fluorescence curve along time</P><H3>Construction of the recombinant plasmids used in L. lactis</H3><P>Erythromycin resistance has been reported to be a useful selectable marker in E. coli or L. lactis. However, in our hands, erythromycin selection was usually problematic in E. coli, giving rise to high background levels of untransformed cells. Since selection for kanamycin resistance is much more efficient in E. coli, kanamycin resistance gene with promoter from pEASY was cloned in pMG36e EcoRI site by seamless clone. The resulted plasmid, pMG36ek, contains ampicillin- and erythromycin-resistance (Fig. 5).</P><P style="text-align: center;font-family:'Open Sans', sans-serif">Fig. 5 pMG36ek Map.</P><P>The Agr system was divided into two parts, AIP-producing sender and AIP receiver, which constructed sender/receiver synthetic communication system.</P><P>The agrB-agrD fragment was cloned at the downstream of P32 promotor in pMG36ek by seamless clone. The resulted plasmids were named pMG36ek-BD-I (from Agr-I), pMG36ek-BD-II (from Agr-II), and pMG36ek-BD-IV (from Agr-IV). Unfortunately, we didn’t get the pMG36ek-BD-III (from Agr-III) plasmid, maybe BD from Agr-III is toxic to cell. Then, the pMG36ek-BD was transformed into the L. lactis strain NZ9000. The constructed NZ9000::pMG36ek-BD-I, NZ9000::pMG36ek-BD-II, and NZ9000::pMG36ek-BD-IV were the AIP-producing sender.</P><P>To test whether the AIP receiver work or not, gfp gene with P2 promoter was cloned at the EcoRI site of pMG36ek, which produced pMG36ek-P2-GFP (Fig. 6). If the agrC-agrA fragment was cloned at the downstream of P32 promotor in pMG36ek, the expressed protein AgrC and AgrA will receive the signal of corresponding AIP, and induce gfp gene at the downstream of P2 promoter. Unfortunately, maybe the agrC-agrA fragment was toxic to E. coli and L. lactis, and we cannot get the plasmid. We also used various of promoter, such as P2 or PnisZ (nisin inducible promoter in NZ9000), to change the promoter of agrC-agrA fragment, and we still not get the corresponding vectors. Then, the <I>S. aureus</I> (strains from Agr I to Agr IV) or <I>S. aureus</I> carrying pLI50-P2-GFP plasmid were used as AIP receiver.</P><P style="text-align: center;font-family:'Open Sans', sans-serif">Fig. 6 pMG36ek-P2-GFP Map.</P><H3>Test the AIP-producing Sender/receiver synthetic communication system cell with <I>S. aureus</I></H3><P>To test the sender/receiver synthetic communication system work or not, two tests have been done. Different AIP signals cross-inhibit the activity of the others. For example, AIP-I will activate group I Agr system and inhibited group II or III Agr system, and the most significant phenomenon change of inhibition is the change of hemolysis zone on rabbit blood agar plate. Therefore, <I>S. aureus</I> strains Agr I was co-cultured with NZ9000 or NZ9000::pMG36ek-BD-I/II/IV on rabbit blood agar plate, but the hemolysis zone of each co-culture colony has not significant changes (Fig. 7). The same phenomenon changes have been observed in <I>S. aureus</I> (strains from Agr II to Agr IV) colonies co-cultured with NZ9000::pMG36ek-BD-II and NZ9000::pMG36ek-BD-IV (data no shown). To further assess our sender/receiver synthetic communication system, stationary-phase cultures of <I>S. aureus</I> (strains from Agr I to Agr IV) carrying pLI50-P2-GFP plasmid and NZ9000::pMG36ek-BD-I/II/IV were spotted onto M17GS agar separately, their centers were separated by ~8 mm. If the AIP signal was produced, when sender and receiver cells were spotted directly adjacent to one another, we will observe fluorescence changes from the receiver cells at the edge of the colony closest to the sender cells. But we didn’t observe any significant changes of the receiver cells (<I>S. aureus</I> strains from Agr I to Agr IV, data not shown). These two tests indicated that, the sender (NZ9000::pMG36ek-BD-I/II/IV) did not produce functional AIP signals.</P><P style="text-align: center;font-family:'Open Sans', sans-serif">Fig. 7 Four S.aureus (strains from Agr I to Agr IV) co-cultured with NZ9000 (A) or NZ9000::pMG36ek-BD-I (B) co-culture on rabbit blood agar plate.</P></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row clearfix"><DIV class="col-md-4 column"><P style="font-family:'Open Sans', sans-serif;font-size:12px;">Contact us
                   Email: <A href="Mailto:igem@tmmu.edu.cn" style="color:#000">igem@tmmu.edu.cn</A>
                   Address:Third Military Medical University, No.30 Gaotanyan Street Shapingba District, Chongqing, P.R.China 400038
                </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>