Team:Glasgow/Measurement
Glasgow iGEM 2016
Project
Interlab
β-carotene Synthesis
Proof-of-Concept
Chassis
Characterising amilCP
Measurement
Shuttle Vector
Transcriptomics
Self-Inactivating Mechanism
Engineering
Design
Entrepreneurship
Prototype
Practices
Silver
Gold
GMO Legislation
Outreach
Collaborations
Integrated Human Practices
Notebook
Protocols
Weekly Journal
People
Team Members
Attributions
Measurement
Contents
1 Overview
2 The Promoters
3 Protocol
3.1 Assembly
3.2 Transformation and Culturing
3.3 Measurement
4 Results
5 Discussion
5.1 p25
5.2 J23100 Fluorescence in E5501
5.3 Parts submitted to the registry
6 References
Overview
As part of our project, we aimed to turn Streptococcus thermophilus into a chassis organism that could potentially be used by future iGEM teams in food and nutrition projects. Part of this undertaking was the identification and characterization of S. thermophilus promoters for use in S. thermophilus constructs. We used fluorescence to quantify the strength of three S. thermophilus promoters and compared them to six different Anderson family promoters with known relative strengths, in E. coli. We wanted to check the strength of these promoters in E. coli to see whether it is viable to make a single BioBrick construct for use in both E. coli and S. thermophilus.
The Promoters
We tested and characterized the strengths of three previously identified lactic acid bacterial promoters:
pHLBA: A constitutive promoter responsible for the expression of histone-like proteins in Lactobacillus bulgaricus and other lactic acid bacteria. [1]
p32: A constitutive promoter used in expression vectors in S. thermophilus. [2]
p25: A constitutive promoter identified in a genome survey for S. thermophilus promoters. [2]
The following Anderson family promoters were used as a known comparison, relative strength in parentheses: J23100 (100%), J23101 (70%), J23106 (47%), J23105 (24%), J23117 (6%), J23113 (1%)
Protocol
Assembly
We created two reporter constructs for each promoter:
Medium RBS GFP: Promoter + E5501 (B0032 + E0040)
Strong RBS GFP: Promoter + I13500 (B0034 + E0040)
The three S. thermophilus promoters were synthesized as oligos by IDT and ligated into E5501 and I13500 cut with EcoRI and XbaI.  E5501 and I13500 were cut with XbaI and PstI and ligated into existing Anderson promoter plasmids cut with SpeI and PstI. This resulted in a total of 18 constructs.
Transformation and Culturing
E5501, I13500 and all 18 constructs were transformed into E. coli TOP10 cells made competent with calcium chloride according to a standard transformation protocol . Transformants were plated out on chloramphenicol agar and cultured overnight at 37⁰C. Three colonies from each plate were subsequently restreaked to rule out false positives and provide a fresh supply of colonies for the measurement step. Three colonies from each restreak plate were picked and inoculated in 3ml of L-broth and chloramphenicol overnight at 37⁰C and 225rpm in a shaking incubator. Three non-transformed TOP10 colonies were also picked and cultured to serve as non-fluorescent controls alongside the promoter-less E5501 and I13500 controls. Therefore, each construct was being measured using three biological replicates. E5501 and I13500 constructs were cultured and measured on separate days to reduce the volume of samples.
Measurement
Following removal from the incubator, 100µl from each overnight was added to a single well in a 96-well plate. Sampling was repeated three times for each biological replicate, yielding a total of 9 samples for each promoter-reporter construct. Each promoter’s samples occupied 9 wells in a single row. Fluorescence and OD600 measurements were performed using a Bio-Tek Synergy HT plate reader using GFP excitation and emission spectra with a gain of 40 and a read height of 10mm. Following measurement, data was exported to excel and standardized for an OD600 of 1.
Results
Measurements using the strong RBS variants revealed the expected order of promoter strength for Anderson promoters and showed that the S. thermophilus promoters were relatively strong:
Figure 1: Mean fluorescence values for I13500 under the control of various promoters. Orange represents S. thermophilus promoters, blue represents Anderson family promoters. Error bars represent standard deviation for each set of samples.
pHLBA was the strongest S. thermophilus promoter, exhibiting 93% the fluorescence value of J23100, followed by p32 which exhibited 81% J23100 fluorescence. P25 was the weakest promoter at 30.8% the fluorescence value of J23100. However, it was still moderately strong.
While the order of Anderson promoter strengths fits in with prior measurements, the precise values were different than those previously reported:
J23101 exhibited fluorescence 70% as strong as J23100, consistent with prior measurements.
J23106 exhibited fluorescence 24% as strong as J23100, compared to the reported 47%.
J23105 exhibited fluorescence 14% as strong as J23100, compared to the reported 24%.
J23117 exhibited fluorescence 1% as strong as J23100, compared to the reported 6%.
J23113 exhibited fluorescence 0.4% as strong as J23100, compared to the reported 1%. J23113 fluorescence was lower than that of the I13500 promoter-less control and indistinguishable from the TOP10 untransformed control.
Fluorescence measurements for constructs using E5501 (GFP with B0032, a medium promoter) showed slightly different results:
Figure 2: Mean fluorescence values for E5501 under the control of various promoters. Red represents S. thermophilus promoters, blue represents Anderson family promoters. Error bars represent standard deviation for each set of samples. Promoters are ordered according to their strength in Figure 1.
P25 showed a much higher strength than expected: 227% that of J23100.
p32+E5501 fluorescence was not significantly different from that of J23100 (failed p=0.05 T test) in contrast to p32+I13500. pHLBA+E5501 exhibited 83% the fluorescence of J23100+E5501, in contrast to 93% in I13500.
The Anderson promoters also showed slightly different strengths relative to each other than in the I13500 experiment:
J23101 exhibited fluorescence 75% as strong as J23100, consistent with prior measurements.
J23106 exhibited fluorescence 31% as strong as J23100, compared to the reported 47% and 24% in I13500.
J23105 exhibited fluorescence 15% as strong as J23100, compared to the reported 24%, and 14% in I13500.
J23117 exhibited fluorescence 5% as strong as J23100, compared to the reported 6% and 1% in I13500.
J23113 exhibited fluorescence 2% as strong as J23100, compared to the reported 1% and 0.4% in I13500.  However, J23113 fluorescence was still indistinguishable from the TOP10 untransformed control.
Discussion
Several discrepancies between the two experiment were apparent. Although we did not have the time to repeat the experiments, we attempted to explain them using the available data.
p25
When under the control of p25, E5501 exhibited fluorescence comparable to I13500 (~2300 arbitrary units). This value was much higher than for E5501 under the control of all other promoters. We first believed that there had been a mistake during assembly and I13500 had been used instead.  This unusually high level of fluorescence was attributed to the fact that the plasmid had dimerised. The dimerisation was verified with a gel electrophoresis:
Figure 3: Gel picture showing sizes of uncut and cut p25-E5501 alongside sizes of uncut and cut phlbA-E5501 as comparison. It can be seen that the uncut p25-E5501 is roughly double the size of phlbA-E5501, suggesting that the plasmid containing p25-E5501 has dimerised. The cut lanes for both of these constructs show equal band sizes.
Sequencing revealed that the p25+E5501 plasmid had dimerized in all three biological replicates. While this may the cause of the unexpected fluorescence readings, we did not have the time to repeat testing.
J23100 Fluorescence in E5501
Both p32 and pHLBA exhibited higher promoter strength relative to J23100 in E5501 compared to I13500. The other Anderson promoters showed the same increase in strength relative to J23100. However, this may mean there were problems with J23100+E5501 expression.
J23113+E5501 stronger than expected (2% instead of 1% or 0.04% in the I13500 experiment) but also no different than the untransformed control, it is likely that J23100+E5501 itself was not being expressed as strongly as possible. This may account for the apparently equal fluorescence between J23100 and p32. Unfortunately, time constraints prevented us from repeating the experiment to see if the result was down to experimental error.
Parts submitted to the registry
We created six new GFP-based BioBrick constructs to submit to the registry- these are both I13500 and E5501 ligated to each of the native S. thermophilus promoters (phlbA, p25, and p32)
BBa_K2151006 : phlbA ligated to I13500 (GFP with strong RBS)
BBa_K2151007 : p25 ligated to I13500 (GFP with strong RBS)
BBa_K2151008 : p32 ligated to I13500 (GFP with strong RBS)
BBa_K2151003 : phlbA ligated to E5501 (GFP with medium RBS)
BBa_K2151004 : p25 ligated to E5501 (GFP with medium RBS)
BBa_K2151005 : p32 ligated to E5501 (GFP with medium RBS)
References
↑ Chouayekh, H., Serror, P., Boudebbouze, S. and Maguin, E. (2009). Highly efficient production of the staphylococcal nuclease reporter in Lactobacillus bulgaricus governed by the promoter of the hlbA gene. FEMS Microbiology Letters, 293(2), pp.232-239.
↑ 2.0 2.1 Slos, P., Bourquin, J., Lemoine, Y. and Mercenier, A. (1990). Isolation and Characterization of Chromosomal Promoters of Streptococcus salivarius subsp. thermophilus. APPLIED AND ENVIRONMENTAL MICROBIOLOGY, 57(5), pp.1333-1339.
