Team:Glasgow/Description
Glasgow iGEM 2016
Project
Interlab
β-carotene Synthesis
Proof-of-Concept
Chassis
Characterising amilCP
Measurement
Shuttle Vector
Transcriptomics
Self-Inactivating Mechanism
Engineering
Design
Entrepreneurship
Prototype
Practices
Silver
Gold
GMO Legislation
Outreach
Collaborations
Integrated Human Practices
Notebook
Protocols
Weekly Journal
People
Team Members
Attributions
Characterising amilCP in S. thermophilus
Contents
1 Building on the work of iGEM Uppsala
2 Experimental Design
3 Observations and Inferences
4 Parts submitted to the registry
Building on the work of iGEM Uppsala
During our project, we aimed to characterise a previous BioBrick part in the lactic acid bacteria we were working with, Streptococcus thermophilus. As we aimed to improve the understanding of this organism as part of the project, we decided to focus on reporter constructs in order to test whether these were viable in S. thermophilus. We looked into the work of the Uppsala teams between 2011 and 2013, who BioBricked various chromoproteins originating from sea anenomes and corals and expressed them in E. coli. We selected a single chromoprotein for our experiments: this was the blue chromoprotein amilCP. When transformed into E. coli, amilCP produces a vibrant dark blue pigment within 24 hours of incubation. We wanted to test whether amilCP was a useful reporter construct in our lactic acid bacteria.
Experimental Design
We ligated the BioBrick BBa_K592025 (strong RBS B0034 and blue chromoprotein amilCP ) to a native S. thermophilus promoter, P25. We then inserted the BioBrick construct into pSB1C3 and transformed it into TOP10 E. coli in order to check that the gene functioned as expected. Once this was confirmed, the construct was then inserted into the shuttle vector pMG36ET for transforming both TOP10 E. coli and S. thermophilus. This plasmid also contained an erythromycin resistance cassette in order to select for transformant colonies.
Figure 1: K592025 BioBrick inserted in both plasmids we worked with: pMG36ET on the left, and pSB1C3 on the right
Observations and Inferences
As previously described on the main page for BBa_K592025, a vibrant blue colour was produced by transformant E. coli colonies with the BioBrick construct in pSB1C3 within 24 hours (Figure 2).
Figure 2: K592025 BioBrick ligated to p25 in pSB1C3 and transformed into E. coli. A vibrant blue colour can be seen in transformant colonies.
Due to the low copy number of pMG36ET, the blue colour of amilCP was less vibrant in E. coli transformed with the construct in pMG36ET than in bacteria transformed with pSB1C3. The BioBrick could still be used as a reporter in E. coli with this plasmid, but it was not as obvious a colour change as in the higher copy number plasmid, pSB1C3. When transforming the BioBrick construct into S. thermophilus, the resultant transformants (selected for with erythromycin) did not produce any blue chromoprotein even after several days. We checked that amilCP was present in this bacteria by isolating the plasmid DNA through a miniprep, and then running it on an agarose gel to verify that the size was correct. The higher copy number native S. thermophilus plasmids obscured the results of this gel and we were unable to tell whether pMG36ET was present in the bacteria. In order to check this, we then transformed TOP10 E. coli with the miniprep, and the transformant cells were a pale blue colour. From this it can be inferred that S. thermophilus did have the pMG36ET vector containing the P25-K592025 construct.
Figure 3: Pale blue TOP10 E. coli cells which have been transformed with an S. thermophilus miniprep containing the shuttle vector plasmid with amilCP. The bottom row of cells are control cells producing no pigment.
Despite the presence of amilCP in the transformed S. thermophilus, no pigment was produced even after several days. We have come up with four hypotheses as to why this could have occurred:
The ribosome binding site B0034 was not suitable for S. thermophilus- therefore, translation of the gene was not able to occur and blue chromoprotein was not synthesised. This is unlikely due to the consensus sequences of ribosome binding sites for both organisms being similar, with the only nucleotide base difference between the two being energetically equivalent (ACCTCCTTT/A).
Each of the BioBrick parts in the construct was suitable for S. thermophilus, but the cellular conditions meant that the expressed amilCP was degraded before any visible pigmentation could develop.
The low copy number of the plasmid meant that although amilCP was expressed, it was not expressed in a large enough amount to be visible to the naked eye.
There was a mutation in the p25-K592025 BioBrick which we did not discover until our submission BioBricks were sent for sequencing. This mutation, though small (1bp deletion in the -35 region) could have affected transcription of amilCP in S.thermophilus but not in E.coli>. This mutation is described on our BBa_K2151010 parts page for this BioBrick.
It initially appears that amilCP is not a suitable reporter for S. thermophilus, however we cannot directly conclude this from our results. We would need to carry out more experiments in order to ensure the other BioBrick parts we used were all suitable for S. thermophilus before committing to the idea that amilCP is not useful in this organism.
Parts submitted to the registry
We ligated Uppsala's BioBrick BBa_K592025 to the three native S.thermophilus promoters we worked with throughout the project and submitted three new BioBricks. These could be utilised by teams working with S. thermophilus in order to expand upon the results we gathered.
BBa_K2151009 : phlbA ligated to BBa_K592025
BBa_K2151010 : p25 ligated to BBa_K592025
BBa_K2151011 : p32 ligated to BBa_K592025
Home
Project
Interlab
β-carotene Synthesis
Proof-of-Concept
Chassis
Characterising amilCP
Measurement
Shuttle Vector
Transcriptomics
Self-Inactivating Mechanism
Engineering
Design
Entrepreneurship
Prototype
Practices
Silver
Gold
GMO Legislation
Outreach
Collaborations
Integrated Human Practices
Notebook
Protocols
Weekly Journal
People
Team
Attributions
Like us on Facebook
Follow us on Twitter
Follow us on Instagram
