<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Glasgow_Measurement skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Glasgow/Measurement</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="title-bar" data-responsive-toggle="main-menu" data-hide-for="large"><DIV class="title-bar-title">Glasgow iGEM 2016</DIV></DIV><DIV class="top-bar" id="main-menu"><DIV><DIV class="top-bar-right"><UL class="vertical large-horizontal menu" id="main-nav" data-responsive-menu="accordion large-dropdown"><LI><A class="headline" href="#">Project</A></LI><LI><A href="https://2016.igem.org/Team:Glasgow/Interlab">Interlab</A></LI><LI><A href="https://2016.igem.org/Team:Glasgow/beta-carotene">β-carotene Synthesis</A></LI><LI><A href="https://2016.igem.org/Team:Glasgow/ProofofConcept">Proof-of-Concept</A></LI></UL><LI><A href="https://2016.igem.org/Team:Glasgow/Chassis">Chassis</A></LI><LI><A href="https://2016.igem.org/Team:Glasgow/Description">Characterising amilCP</A></LI><LI><A href="https://2016.igem.org/Team:Glasgow/Measurement">Measurement</A></LI><LI><A href="https://2016.igem.org/Team:Glasgow/ShuttleVector">Shuttle Vector</A></LI><LI><A href="https://2016.igem.org/Team:Glasgow/Transcriptomics">Transcriptomics</A></LI><LI><A href="https://2016.igem.org/Team:Glasgow/SIM">Self-Inactivating Mechanism</A></LI><LI><A class="headline" href="#">Engineering</A></LI><LI><A href="https://2016.igem.org/Team:Glasgow/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Glasgow/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:Glasgow/Yoghurt-maker">Prototype</A></LI><LI><A class="headline" href="#">Practices</A></LI><LI><A href="https://2016.igem.org/Team:Glasgow/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:Glasgow/HP/Gold">Gold</A></LI><LI><A href="https://2016.igem.org/Team:Glasgow/GMOLegislation">GMO Legislation</A></LI><LI><A href="https://2016.igem.org/Team:Glasgow/Engagement">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Glasgow/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Glasgow/Integrated_Practices">Integrated Human Practices</A></LI><LI><A class="headline" href="https://2016.igem.org/Team:Glasgow/Protocols">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Glasgow/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Glasgow/WeeklyJournal">Weekly Journal</A></LI><LI><A class="headline" href="https://2016.igem.org/Team:Glasgow/Team">People</A></LI><LI><A href="https://2016.igem.org/Team:Glasgow/Team">Team Members</A></LI><LI><A href="https://2016.igem.org/Team:Glasgow/Attributions">Attributions</A></LI></DIV></DIV></DIV><DIV class="photo-block" id="measurement"><DIV class="text left"><DIV class="title">
			Measurement
		</DIV></DIV></DIV><DIV class="wiki-block"><DIV id="article"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#The_Promoters"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">The Promoters</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Protocol"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Protocol</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Assembly"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Assembly</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Transformation_and_Culturing"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Transformation and Culturing</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Measurement"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Measurement</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-7"><A href="#Results"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Discussion"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#p25"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">p25</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#J23100_Fluorescence_in_E5501"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">J23100 Fluorescence in E5501</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Parts_submitted_to_the_registry"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">Parts submitted to the registry</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#References"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">References</SPAN></A></LI></DIV><H2><SPAN class="mw-headline" id="Overview">Overview</SPAN></H2><P>As part of our project, we aimed to turn <I>Streptococcus thermophilus</I> into a chassis organism that could potentially be used by future iGEM teams in food and nutrition projects. Part of this undertaking was the identification and characterization of <I>S. thermophilus</I> promoters for use in <I>S. thermophilus</I> constructs. We used fluorescence to quantify the strength of three <I>S. thermophilus</I> promoters and compared them to six different Anderson family promoters with known relative strengths, in <I>E. coli</I>. We wanted to check the strength of these promoters in <I>E. coli</I> to see whether it is viable to make a single BioBrick construct for use in both <I>E. coli</I> and <I>S. thermophilus</I>. 
</P><H2><SPAN class="mw-headline" id="The_Promoters">The Promoters</SPAN></H2><P>We tested and characterized the strengths of three previously identified lactic acid bacterial promoters:
</P><UL><LI>pHLBA: A constitutive promoter responsible for the expression of histone-like proteins in <I>Lactobacillus bulgaricus</I> and other lactic acid bacteria.<SUP id="cite_ref-1" class="reference"><A href="#cite_note-1">[1]</A></SUP></LI><LI>p32: A constitutive promoter used in expression vectors in <I>S. thermophilus</I>.<SUP id="cite_ref-Slos_2-0" class="reference"><A href="#cite_note-Slos-2">[2]</A></SUP></LI><LI>p25: A constitutive promoter identified in a genome survey for <I>S. thermophilus</I> promoters.<SUP id="cite_ref-Slos_2-1" class="reference"><A href="#cite_note-Slos-2">[2]</A></SUP></LI></UL><P>The following Anderson family promoters were used as a known comparison, relative strength in parentheses:
 <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_J23100">J23100</A> (100%), <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_J23101">J23101</A> (70%),  <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_J23106">J23106</A> (47%), <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_J23105">J23105</A> (24%), <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_J23117">J23117</A> (6%), <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_J23113">J23113</A> (1%)</P><H2><SPAN class="mw-headline" id="Protocol">Protocol</SPAN></H2><H3><SPAN class="mw-headline" id="Assembly">Assembly</SPAN></H3><P>We created two reporter constructs for each promoter:
</P><UL><LI>Medium RBS GFP: Promoter + <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_E5501">E5501</A> (B0032 + E0040)</LI><LI>Strong RBS GFP: Promoter + <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_I13500">I13500</A> (B0034 + E0040)</LI></UL><P>The three <I>S. thermophilus</I> promoters were synthesized as oligos by IDT and ligated into E5501 and I13500 cut with EcoRI and XbaI. 
E5501 and I13500 were cut with XbaI and PstI and ligated into existing Anderson promoter plasmids cut with SpeI and PstI.
This resulted in a total of 18 constructs. 
</P><H3><SPAN class="mw-headline" id="Transformation_and_Culturing">Transformation and Culturing</SPAN></H3><P>E5501, I13500 and all 18 constructs were transformed into <I>E. coli</I> TOP10 cells made competent with calcium chloride according to a standard <A rel="nofollow" class="external text" href="https://2016.igem.org/Team:Glasgow/Protocols">transformation protocol</A>. Transformants were plated out on chloramphenicol agar and cultured overnight at 37⁰C. Three colonies from each plate were subsequently restreaked to rule out false positives and provide a fresh supply of colonies for the measurement step.
Three colonies from each restreak plate were picked and inoculated in 3ml of L-broth and chloramphenicol overnight at 37⁰C and 225rpm in a shaking incubator. Three non-transformed TOP10 colonies were also picked and cultured to serve as non-fluorescent controls alongside the promoter-less E5501 and I13500 controls. Therefore, each construct was being measured using three biological replicates.
E5501 and I13500 constructs were cultured and measured on separate days to reduce the volume of samples.
</P><H3><SPAN class="mw-headline" id="Measurement">Measurement</SPAN></H3><P>Following removal from the incubator, 100µl from each overnight was added to a single well in a 96-well plate. Sampling was repeated three times for each biological replicate, yielding a total of 9 samples for each promoter-reporter construct. Each promoter’s samples occupied 9 wells in a single row.
Fluorescence and OD600 measurements were performed using a Bio-Tek Synergy HT plate reader using GFP excitation and emission spectra with a gain of 40 and a read height of 10mm.
Following measurement, data was exported to excel and standardized for an OD600 of 1.
</P><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><P>Measurements using the strong RBS variants revealed the expected order of promoter strength for Anderson promoters and showed that the <I>S. thermophilus</I> promoters were relatively strong:
</P><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 1:</B> Mean fluorescence values for I13500 under the control of various promoters. Orange represents <I>S. thermophilus</I> promoters, blue represents Anderson family promoters. Error bars represent standard deviation for each set of samples.
		</DIV></DIV><P>pHLBA was the strongest S. thermophilus promoter, exhibiting 93% the fluorescence value of J23100, followed by p32 which exhibited 81% J23100 fluorescence. P25 was the weakest promoter at 30.8% the fluorescence value of J23100. However, it was still moderately strong. 
</P><P>While the order of Anderson promoter strengths fits in with prior measurements, the precise values were different than those previously reported:
</P><UL><LI>J23101 exhibited fluorescence 70% as strong as J23100, consistent with prior measurements.</LI><LI>J23106 exhibited fluorescence 24% as strong as J23100, compared to the reported 47%. </LI><LI>J23105 exhibited fluorescence 14% as strong as J23100, compared to the reported 24%.</LI><LI>J23117 exhibited fluorescence 1% as strong as J23100, compared to the reported 6%.</LI><LI>J23113 exhibited fluorescence 0.4% as strong as J23100, compared to the reported 1%. J23113 fluorescence was lower than that of the I13500 promoter-less control and indistinguishable from the TOP10 untransformed control.</LI></UL><P>
Fluorescence measurements for constructs using E5501 (GFP with B0032, a medium promoter) showed slightly different results:
</P><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 2:</B> Mean fluorescence values for E5501 under the control of various promoters. Red represents S. thermophilus promoters, blue represents Anderson family promoters. Error bars represent standard deviation for each set of samples. Promoters are ordered according to their strength in Figure 1.
		</DIV></DIV><P>P25 showed a much higher strength than expected: 227% that of J23100. 
</P><P>p32+E5501 fluorescence was not significantly different from that of J23100 (failed p=0.05 T test) in contrast to p32+I13500. pHLBA+E5501 exhibited 83% the fluorescence of J23100+E5501, in contrast to 93% in I13500.
</P><P>The Anderson promoters also showed slightly different strengths relative to each other than in the I13500 experiment:
</P><UL><LI>J23101 exhibited fluorescence 75% as strong as J23100, consistent with prior measurements.</LI><LI>J23106 exhibited fluorescence 31% as strong as J23100, compared to the reported 47% and 24% in I13500. </LI><LI>J23105 exhibited fluorescence 15% as strong as J23100, compared to the reported 24%, and 14% in I13500. </LI><LI>J23117 exhibited fluorescence 5% as strong as J23100, compared to the reported 6% and 1% in I13500. </LI><LI>J23113 exhibited fluorescence 2% as strong as J23100, compared to the reported 1% and 0.4% in I13500.  However, J23113 fluorescence was still indistinguishable from the TOP10 untransformed control. </LI></UL><H2><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H2><P>Several discrepancies between the two experiment were apparent. Although we did not have the time to repeat the experiments, we attempted to explain them using the available data.
</P><H3><SPAN class="mw-headline" id="p25">p25</SPAN></H3><P>When under the control of p25, E5501 exhibited fluorescence comparable to I13500 (~2300 arbitrary units). This value was much higher than for E5501 under the control of all other promoters. We first believed that there had been a mistake during assembly and I13500 had been used instead. 
This unusually high level of fluorescence was attributed to the fact that the plasmid had dimerised. The dimerisation was verified with a gel electrophoresis:
</P><DIV class="article-image-container"><DIV class="article-image-caption "><B> Figure 3:</B> Gel picture showing sizes of uncut and cut p25-E5501 alongside sizes of uncut and cut phlbA-E5501 as comparison. It can be seen that the uncut p25-E5501 is roughly double the size of phlbA-E5501, suggesting that the plasmid containing p25-E5501 has dimerised. The cut lanes for both of these constructs show equal band sizes.
		</DIV></DIV><P>Sequencing revealed that the p25+E5501 plasmid had dimerized in all three biological replicates. While this may the cause of the unexpected fluorescence readings, we did not have the time to repeat testing.
</P><H3><SPAN class="mw-headline" id="J23100_Fluorescence_in_E5501">J23100 Fluorescence in E5501</SPAN></H3><P>Both p32 and pHLBA exhibited higher promoter strength relative to J23100 in E5501 compared to I13500. The other Anderson promoters showed the same increase in strength relative to J23100. However, this may mean there were problems with J23100+E5501 expression.
</P><P>J23113+E5501 stronger than expected (2% instead of 1% or 0.04% in the I13500 experiment) but also no different than the untransformed control, it is likely that J23100+E5501 itself was not being expressed as strongly as possible. This may account for the apparently equal fluorescence between J23100 and p32. Unfortunately, time constraints prevented us from repeating the experiment to see if the result was down to experimental error.
</P><H3><SPAN class="mw-headline" id="Parts_submitted_to_the_registry">Parts submitted to the registry</SPAN></H3><P>We created six new GFP-based BioBrick constructs to submit to the registry- these are both I13500 and E5501 ligated to each of the native <I>S. thermophilus</I> promoters (phlbA, p25, and p32)
</P><UL><LI><B><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2151006">BBa_K2151006</A></B>: phlbA ligated to I13500 (GFP with strong RBS)
</LI><LI><B><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2151007">BBa_K2151007</A></B>: p25 ligated to I13500 (GFP with strong RBS)
</LI><LI><B><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2151008">BBa_K2151008</A></B>: p32 ligated to I13500 (GFP with strong RBS)
</LI><LI><B><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2151003">BBa_K2151003</A></B>: phlbA ligated to E5501 (GFP with medium RBS)
</LI><LI><B><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2151004">BBa_K2151004</A></B>: p25 ligated to E5501 (GFP with medium RBS)
</LI><LI><B><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2151005">BBa_K2151005</A></B>: p32 ligated to E5501 (GFP with medium RBS)
</LI></UL><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><SMALL><OL class="references"><LI id="cite_note-1"><SPAN class="mw-cite-backlink"><A href="#cite_ref-1">↑</A></SPAN><SPAN class="reference-text">Chouayekh, H., Serror, P., Boudebbouze, S. and Maguin, E. (2009). Highly efficient production of the staphylococcal nuclease reporter in <I>Lactobacillus bulgaricus</I> governed by the promoter of the hlbA gene. FEMS Microbiology Letters, 293(2), pp.232-239.</SPAN></LI><LI id="cite_note-Slos-2"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-Slos_2-0">2.0</A></SUP><SUP><A href="#cite_ref-Slos_2-1">2.1</A></SUP></SPAN><SPAN class="reference-text">Slos, P., Bourquin, J., Lemoine, Y. and Mercenier, A. (1990). Isolation and Characterization of Chromosomal Promoters of <I>Streptococcus salivarius subsp. thermophilus</I>. APPLIED AND ENVIRONMENTAL MICROBIOLOGY, 57(5), pp.1333-1339.</SPAN></LI></OL></SMALL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>