"
Page
Discussion
View source
History
teams
Log in
 
Team:Lethbridge/projectreferences
From 2012.igem.org
2012 iGEM - University of Lethbridge
The Project
Results
Notebook
Parts
Human Practices
Safety
Attributions
Our Team
References
1. Hryhor, D. W. Emerging Solutions for Heavy Oil Production from Carbonates. (TAMM Oil and Gas Corp., 2008).
2. Bott, R. D. Canada's Oil Sands. (Canadian Centre for Energy Information, 2011).
3. Buijse, M., de Boer, P., Breukel, B. & Burgos, G. Organic acids in carbonate addizing. Spe Production & Facilities 19, 128-134 (2004).
4. World Energy Outlook 2010 - Executive Summary. (International Energy Agency, 2010).
5. MicroPro-Gmbh. MEOR case study - oil field “Doebern” (Germany),
(2011).
6. Sen, R. Biotechnology in petroleum recovery: The microbial EOR. Progress in Energy and Combustion Science 34, 714-724, doi:10.1016/j.pecs.2008.05.001 (2008).
7. Niederholtmeyer, H., Wolfstaedter, B. T., Savage, D. F., Silver, P. A. & Way, J. C. Engineering Cyanobacteria To Synthesize and Export Hydrophilic Products. Applied and Environmental Microbiology 76, 3462-3466, doi:10.1128/aem.00202-10 (2010).
8. Iancu, C. V. et al. Organization, Structure, and Assembly of α-Carboxysomes Determined by Electron Cryotomography of Intact Cells. Journal of Molecular Biology 396, 105-117, doi:10.1016/j.jmb.2009.11.019 (2010).
9. Bonacci, W. et al. Modularity of a Carbon-Fixing Protein Organelle. Proceedings of the National Academy of Sciences of the United States of America 109, 478-483, doi:10.1073/pnas.1108557109 (2012).
10. Price, G. D. Inorganic carbon transporters of the cyanobacterial CO2 concentrating mechanism. Photosynthesis Research 109, 47-57, doi:10.1007/s11120-010-9608-y (2011).
11. Bar-Even, A., Noor, E., Lewis, N. E. & Milo, R. Design and Analysis of Synthetic Carbon Fixation Pathways. Proceedings of the National Academy of Sciences 107, 8889-8894, doi:10.1073/pnas.0907176107 (2010).
12. Davies, S. & Kelkar, S. Carbonate Stimulation. Middle East & Asia Reservoir Review 8, 52-63 (2007).
13. Ferry, J. G. in Methods in Enzymology: Methods in Methane Metabolism, Pt A Vol. 494 Methods in Enzymology (eds A. C. Rosenzweig & S. W. Ragsdale)  219-231 (Elsevier Academic Press Inc, 2011).
14. Latimer, M. T. & Ferry, J. G. Cloning, Sequence-Analysis, and Hyperexpression of the Genes Encoding Phosphotransacetylase and Acetate Kinase from Methanosarcina thermophila. J. Bacteriol. 175, 6822-6829 (1993).
15. Nakano, S., Fukaya, M. & Horinouchi, S. Putative ABC Transporter Responsible for Acetic Acid Resistance in Acetobacter aceti. Applied and Environmental Microbiology 72, 497-505, doi:10.1128/aem.72.1.497-505.2006 (2006).
16. Wang, Q. et al. Engineering Bacteria for Production of Rhamnolipid as an Agent for Enhanced Oil Recovery. Biotechnology and Bioengineering 98, 842-853, doi:10.1002/bit.21462 (2007).
17. Patel, R. M. & Desai, A. J. Biosurfactant production by Pseudomonas aeruginosa GS3 from molasses. Letters in Applied Microbiology 25, 91-94, doi:10.1046/j.1472-765X.1997.00172.x (1997).
18. iGEM2011. Results Summary,
(2011).
19. Delebecque, C. J., Lindner, A. B., Silver, P. A. & Aldaye, F. A. Organization of Intracellular Reactions with Rationally Designed RNA Assemblies. Science 333, 470-474, doi:10.1126/science.1206938 (2011).
20. Ohno, H. et al. Synthetic RNA-protein complex shaped like an equilateral triangle. Nature Nanotechnology 6, 115-119, doi:10.1038/nnano.2010.268 (2011).
21. Zhu, C. & Ye, Q. [Selection of acetate-tolerant mutants from Escherichia coli DH5alpha and the metabolic properties of mutant DA19]. Wei Sheng Wu Xue Bao 43, 460-465 (2003).
22. Ochsner, U. A., Fiechter, A. & Reiser, J. Isolation, Characterization, and Expression in Escherichia coli of the Pseudomonas aeruginosa rhlAB Genes Encoding a Rhamnosyl Transferase Involved in Rhamnolipid Biosurfactant Synthesis. Journal of Biological Chemistry 269, 19787-19795 (1994).
23. iGEM2011. BioBricks,
(2011).
24. Ohlendorf, R., Vidavski, R. R., Eldar, A., Moffat, K. & Möglich, A. From Dusk till Dawn: One-Plasmid Systems for Light-Regulated Gene Expression. Journal of Molecular Biology 416, 534-542, doi:10.1016/j.jmb.2012.01.001 (2012).
25. Flynn, J. M., Neher, S. B., Kim, Y.-I., Sauer, R. T. & Baker, T. A. Proteomic Discovery of Cellular Substrates of the ClpXP Protease Reveals Five Classes of ClpX-Recognition Signals. Molecular Cell 11, 671-683, doi:10.1016/s1097-2765(03)00060-1 (2003).
Retrieved from " http://2012.igem.org/Team:Lethbridge/projectreferences "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
