<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Valencia_UPV_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Valencia UPV/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="wrapper"><DIV id="header" class="navbar-toggleable-md sticky clearfix"><HEADER id="topNav"><DIV class="container"><DIV class="navbar-collapse collapse float-right nav-main-collapse submenu-color"><NAV class="nav-main"><UL id="topMain" class="nav nav-pills nav-main nav-hover-animate nav-animate-fadeIn"><LI class="dropdown"><A class="dropdown-toggle" href="#">
                           TEAM
                           </A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Team">
                                 Team
                                 </A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Attributions">
                                 Attributions
                                 </A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" href="#">
                           PROJECT
                           </A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Description">
                                 Description
                                 </A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Results">
                                 Results
                                 </A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Applied_Design">
                                 Applied Design
                                 </A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Collaborations">
                                 Collaborations
                                 </A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Achievement">
                                 Achievements
                                 </A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Safety">
                                 Safety
                                 </A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#">
                           WET LAB
                           </A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Design">
                                 Biological Design
                                 </A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Experiments">
                                 Experiments
                                 </A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Protocols">
                                 Protocols
                                 </A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Notebook">
                                 Notebook
                                 </A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Contribution">
                                 Contribution
                                 </A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Improve">
                                 Improve
                                 </A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Plant">
                                 PlantSynBio
                                 </A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Parts">
                                 Parts
                                 </A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#">
                           DRY LAB
                           </A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Model">
                                 Modeling
                                 </A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Software">
                                 Software
                                 </A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Hardware">
                                 Hardware
                                 </A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/GraphicDesign">
                                 Graphic Design
                                 </A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#">
                           HUMAN PRACTICES
                           </A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/HP/Silver">                            Outreach
                                 </A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/HP/Gold_Integrated">
                                 Integrated and gold
                                 </A></LI></NAV></DIV></DIV></HEADER></DIV><SECTION class="section-xs" style="border-bottom: none;"><DIV class="container"><DIV class="text-center"><P class="font-lato fw-300 lead mt-0" style="padding-top: 20px !important;margin-bottom: 20px !important;text-align: left !important;">Home • Wet Lab •</P><DIV class="col-md-12"><H3 class="jopetitles">RESULTS</H3><DIV><BUTTON type="button" class="btn btn-primary" style="margin: 5px 5px 25px 5px !important; background-color: rgba(0,0,0,0.3) !important; border: none !important;"><A href="https://static.igem.org/mediawiki/2017/f/f1/Valencia_UPV_Results.pdf" class="btn-dwnl" target="_blank"><SPAN style="margin-right: 2px;">PDF</SPAN></A></BUTTON></DIV></DIV></DIV></DIV></SECTION><SECTION class="section-xs"><P> </P><SECTION class="section-xs"><DIV class="container"><DIV class="row"><DIV class="col-md-8"><DIV class="price-clean" style="border: 1px solid #005569;"><DIV class="row"><DIV class="col-md-12"><DIV class="jopeprize"><DIV id="section1" class="container"><DIV class="heading-color"><H2><SPAN>ChatterPlant’s results
                             </SPAN></H2></DIV><P style="color:black">Aiming to establish a bidirectional communication with plants, two different channels for interchanging information with them were developed (these channels are addressed in modules 1 and 2). Furthermore, seeking the goal of obtaining a future sustainable agriculture, a hardware device was designed and built (module 3) and a software tool that facilitates Plant SynBio knowledge was developed (module 4).</P><UL class="jopeli"><LI><STRONG>Module 1</STRONG>: An optogenetic toggle switch circuit capable of tuning the expression of a desired protein depending on the wavelength of the light stimuli was developed.
                             </LI><LI>Red activation of the optogenetic toggle switch was characterized.</LI><LI>Root-aerial part transport by viral autoreplicative vectors was demonstrated.</LI></UL><LI><STRONG>Module 2</STRONG>: A Color Code System triggered by external inducible stimulus together with stress-signals was achieved in order to flag any change in their status accelerating corrective measures.
                             </LI><LI>PhiC31 was characterized.</LI><LI>Metabolic load vs unproductive complex theory hypothesis were assessed.</LI><LI>Recombination Directionality Factor (RDF) action was demonstrated.</LI><LI>RDF was characterized.</LI><LI>Stress-inducible promoter feasibility was demonstrated.</LI><LI>Color code protein expression through a viral vector strategy was achieved.</LI><LI>Chromoproteins expression in <EM>Nicotiana benthamiana</EM> was tested.</LI><LI><STRONG>Module 3: </STRONG>A hardware device (ChatterBox) was engineered in order to provide a SynBio-based solution to improve agriculture sustainability.</LI><LI><STRONG>Module 4: </STRONG>PlantLabCo, an open-access platform aiming to unify Plant SynBio researchers work, was developed.</LI></DIV></DIV></DIV></DIV></DIV><DIV id="section1" class="container"><DIV class="heading-color"><H2><SPAN>MODULE 1: Human – Plant communication channel
                           </SPAN></H2></DIV><P>ChatterPlant is a SynBio-based project that works as an interface between plants and humans to increase control over plant physiology. The human-plant communication channel comprises a root-specific modular optogenetic circuit that enables low-cost control on plants´ endogenous gene expression, therefore linking desired physiological outputs (e.g flowering) to precise, cheap environmental signals (light pulses), rather than expensive-to-create artificial environmental conditions.</P><P>Therefore, we demonstrated the performance of the light-inducible system based on red/far-red light in order to control the expression on any desired protein (Müller, Naumann, Weber &amp; Zurbriggen, 2015).</P><DIV class="heading-color jopesubapartado"><H4><SPAN>CHARACTERIZATION OF RED/FAR RED GENETIC SWITCH</SPAN></H4></DIV><P>In order to test the red-light switch, a GoldenBraid multipartite assembly reaction was performed to create a genetic construct composed by the transcriptional factor, PIF6, fused to a DNA-binding domain (E) and the photoreceptor (PhyB) fused to the activator domain (VP64) together with a nuclear location sequence (NLS). When PhyB (which carry a VP64 activator domain fused to it) is irradiated with 660nm light, its conformation changes and it interacts with PIF6 at the promoter site sparking off the signalling pathway (Fig. 1). A luciferase gene regulated by a minCMV promoter containing an Etr8 operator site was used as a reporter gene in order to characterize the behaviour of this optogenetic circuit.</P><P>Following the <A href="https://2017.igem.org/Team:Valencia_UPV/Experiments#optogenetic">experimental design</A>, our data shows differential expression rates between the ON state (red light) and the OFF state (dark conditions). As Figure 1C and 1D shows, plants irradiated with red light showed an increasing luciferase activity after 23 hours from red-light activation, plants kept in dark conditions exhibited lower expression rates (corresponding with the basal expression) and plants irradiated with white light showed a varying behaviour response. Constitutive expression controls under strong (35s promoter) and weak (pNOS promoter) promoters were treated in white light and dark conditions.</P><P>Note that in the graphs below same error bars have negative values. This is due to the assumption of symetrical standard error when Matlab is computing error bars. In this case, since there is a small sample size (n) and the standard deviation (s) has values close to the mean, the standard error (s/ sqrt(n)) is nearly the standard deviation, and consequently it is also close to the mean value. The positive value of the standard error is calculated, and because of the mentioned simetry assumption, the same absolute value is assumed for the negative standard error. Since Matlab does not have any non-negativity restriction imposed on this estimation, it is possible that the estimated standard error is higher than the mean value of the samples, obtaining negative results when the standard error is substracted to the mean.
</P><P><STRONG>Figure 1. Graphic design of human-plant circuit.</STRONG> a) The transcriptional factor, PIF6, is fused to a DNA-binding domain (E) and PhyB is fused to the activator domain (VP64) and a nuclear location sequence (NLS). Both genetic expressions are controlled by plant strong promoters. b) <EM>When irradiated with 660nm light, PhyB changes its conformation and this complex is recruited to PIF6 at the promoter site. The polymerase III will recognize the activation domain and the transcription will begin. Only upon absorption of a far-red photon (740nm) the interaction between PhyB and PIF6 is terminated, resulting in a shut-off of gene expression. c) Line graph representation of luciferase inducible expression pattern under different light conditions (dark, red and white light). d) Bars graphic representing the luciferase expression in all conditions (white light, red light and dark conditions) after 23 hours from red-light activation. “Strong promoter” corresponds to the luciferase expression obtained with a constitutive strong promoter (35s promoter) under different conditions, while “weak promoter” corresponds to the luciferase expression obtained with a constitutive weak promoter (pNOS promoter) under different conditions.</EM></P><P><STRONG><EM>Figure 2. Charts representing the induction dynamics obtained from optogenetic toggle switch activation experiment after 23h. </EM></STRONG><EM>a) Plot of the dynamics obtained after analyzing luciferase expression along time for different treatments, including the light-inducible construct (Etr8), and a weak constitutive promoter and a strong constitutive promoter under red-light, white-light and dark conditions. b) Plot of the dynamics obtained after analyzing luciferase expression along time for different treatments, including the light-inducible construct (Etr8) under red-light conditions, and a weak constitutive promoter and a strong constitutive promoter under white-light and dark conditions. c) Plot of the dynamics obtained after analyzing luciferase expression along time for different treatments, including the light-inducible construct (Etr8) under dark conditions, and a weak constitutive promoter and a strong constitutive promoter under white-light and dark conditions. d) Plot of the dynamics obtained after analyzing luciferase expression along time for different treatments, including the light-inducible construct (Etr8) under white-light conditions, and a weak constitutive promoter and a strong constitutive promoter under white-light and dark conditions.</EM></P><P><STRONG><EM>Conclusion</EM></STRONG><EM>: </EM>Red-light activation of the optogenetic toggle switch was demonstrated. As shown in Figures 1 and 2, a significantly difference can be observed between plants irradiated with red light and in dark conditions. As can be seen in Figure 2, plants irradiated with white light show a peculiar luciferase activation due to white light contains the red and far-red wavelength.</P><DIV class="heading-color jopesubapartado"><H4><SPAN>VIRAL VECTOR TRANSPORT</SPAN></H4></DIV><P>Up to this point, we have demonstrated the ability to control plant endogenous’ expression profile using a modular and standard red/far-red switch in order to reach all users’ necessities. However, since this circuit will be located on the roots propagating the signal from there to the rest of the plant, and many proteins are not able to move itself to other parts of the plant, we resolved to use a viral vector transport strategy.</P><P>The feasibility of Potato Virus X (PVX) as a viral vector with systemic movement was tested using the following <A href="https://2017.igem.org/Team:Valencia_UPV/Experiments#chromoPVX">experimental design</A>. Thus, <EM>Agrobacterium tumefaciens</EM> culture carrying a genetic construct with a DsRED (red fluorescent protein) transcriptional unit (TUs) was <A href="https://2017.igem.org/Team:Valencia_UPV/Protocols#infiltration">agroinfiltrated </A> in <EM>N. benthamiana</EM> roots, leaves and lower part of the stem. After two weeks, in all treatments, the fluorescent protein could be observed in meristems, buds, new and old leaves and even flowers.</P><P><EM><STRONG>Figure 3</STRONG>. a) Fluorescent images produced by the infection of the stem with an Agrobacterium tumefaciens carrying a PVX with DsRED. b) and c) Fluorescence signal images showing were the infection of the buds with A. tumefaciens carrying a PVX with DsRED. d) Pictures were taken from the leaf of the plant after the infection with PVX with DsRED by A. tumefaciens.</EM></P><P><STRONG>Conclusion:</STRONG> Therefore, as shown in Figure 2, PVX arises as a suitable approach for transporting proteins from roots to the aerial part of the plant due to its speed and yield. It has not escape to our attention the possible reversibility of viral infections as some of the evidence collected may suggest. This would allow plant to return its original state after a certain period of time (i.e. resetting the system) (Fig.4).</P><P><EM><STRONG>Figure 4.</STRONG> Fluorescent pattern shows an abundant protein concentration at the base of the leaf whereas in the leaf tip, there seems to be any fluorescent signal.</EM></P><P>However, viral vector strategy underlies some drawbacks. Thus, usage of mRNAs as long-distance signalling molecules postulates as a good alternative to develop in future steps, due to its ability to deliver a signal in its non-functional form. Additional advantage is provided by the long-distance movement of mRNA over protein trafficking, since translation of the transcript occurs specifically at the target site. Moreover, an mRNA molecule can be translated to numerous copies of a protein being plainly more efficient than traffic several copies of them (Spiegelman, Golan &amp; Wolf, 2013).</P></DIV><DIV id="section1" class="container"><DIV class="heading-color"><H2><SPAN>MODULE 2: Plant – Human communication channel
                           </SPAN></H2></DIV><P>As far as communication is concerned, a bidirectional channel must be established. In our work, we demonstrate that not only delivering orders into plants is feasible but also that plants are able to notify us of any change in their status.</P><P>A leaf’s colour change triggered by a viral vector approach apprises us whether plants are affected by any stress through our Colour Code System. Thus, we resolved to control biotic and abiotic stresses using a modular and orthogonal genetic AND gate in order to decrease leakage possibilities.</P><DIV class="heading-color jopesubapartado"><H4><SPAN>CHARACTERIZATION OF PhiC31 RECOMBINASE ACTION</SPAN></H4></DIV><P>PhiC31 is a serine recombinase with the ability of performing site-specific recombination between two attachment sites attP and attB (Keravala et al., 2006). We demonstrate that PhiC31 allows an accurate regulation of luciferase expression when it performs the inversion between its attachment sites. Following this <A href="https://2017.igem.org/Team:Valencia_UPV/Experiments#Phic31promot">experimental design</A> we were able to characterize recombination performance under weak and strong plant-specific promoters (Pnos and p35s respectively).</P><P><A href="https://2017.igem.org/Team:Valencia_UPV/Improve">Reporter assembly construct</A> is composed by a 35s promoter sequence and Mtb terminator flanked by two opposing PhiC31 site-specific recombination sites (Figure 5).</P><P><EM><STRONG>Figure 5.</STRONG> Graphic representation of Plant-Human genetic circuit with the construct comprised by a promoter and a terminator in opposite directions flanked by ΦC31 attachment sites. a) In presence of PhiC31 integrase, recombination occurs allowing the expression of a translation of a nonsense sequence or stuffer fragment . In this case, the circuit is in OFF state b) Graphic representation of the toggle switch in ON state only when PhiC31 and gp3 are expressed. Only when the promoter is activated under stress conditions, corresponding color protein will be expressed.</EM></P><P>These results showed a counterintuitive correlation between the recombinase and luciferase. The higher the expression of recombinase is, the lower the probability that inversion occurs is. That is, the more concentration of recombinase in plant cells, the fewer inversion efficiency is.</P><P><EM><STRONG>Figure 6.</STRONG> a) Graphic representation of the construct comprised by a promoter and a terminator in opposite directions flanked by PhiC31 attachment sites (attB and attP). It represents the negative control of our experiment. Only when PhiC31 inversion occurs, luciferase protein will be expressed. b) Genetic construct that allows constitutive expression of phiC31 in the plant under the control of a strong promoter c) Graphic representation of the construct comprised by a promoter and a terminator in opposite directions flanked by PhiC31 recombined attachment sites (attR and attL). In normal basis, the promoter is inverted, allowing the expression of luciferase protein. It represents the positive control of our experiment. d) Genetic construct that allows constitutive expression of phiC31 in the plant under the control of a weak constitutive promoter. e) Transcriptional unit for the expression of the Firefly Luciferase under the control of a strong constitutive promoter. f) Transcriptional unit for the expression of the Firefly luciferase under the control of a weak promoter. g) Bars chart representing luciferase expression levels before and after induction with PhiC31.</EM></P><P>After analysing the data obtained from luciferase assay, it can be observed at Figure 4:</P><P>1) A significantly difference between the OFF and the ON state expression levels.</P><P>2) The recombinase expression under a weak promoter (pNOS promoter) shows the same luciferase expression that the constitutive positive control (genetic construct which is expressing constitutively luciferase protein under the strong promoter).</P><P>3) However, the recombinase expression under the strong promoter shows the same expression level of the constitutive weak control (genetic construct based on the constitutive expression of luciferase under a pNOS weak promoter)</P><P>The likelihood of producing the inversion event should be directly proportional to the luciferase protein concentration inside plant cell. Consequently, more quantity of phiC31 recombinase entails more probability of activating reporter gene expression. However, this experiment shows that once recombinase expression exceeds a certain threshold, the effect changes so the likelihood of up-regulating the reporter gene expression decreases.</P><P>In order to provide robust evidences to the hypothesis explained above, a subsequent <A href="https://2017.igem.org/Team:Valencia_UPV/Experiments#Phic31Per">experiment </A>was performed. In this case, PhiC31 behaviour was analysed while being regulated by a weak promoter (pNOS) together with a optical density (<EM>A.tumefaciens</EM> culture concentration) was 10-fold higher than the used in aforementioned experiment (Figure 7).</P><P><EM><STRONG>Figure 7.</STRONG> a) Graphic representation of the construct comprised by a promoter and a terminator in opposite directions flanked by PhiC31 attachment sites (attB and attP). It represents the negative control of our experiment. Only when PhiC31 inversion occurs, luciferase protein will be expressed. b) Genetic construct that allows constitutive expression of PhiC31 in the plant under a strong promoter c) Graphic representation of the construct comprised by a promoter and a terminator in opposite directions flanked by PhiC31 recombined attachment sites (attR and attL). In normal basis, the promoter is inverted, allowing the expression of luciferase protein. It represents the positive control of our experiment. d) Plot representing PhiC31 behavior when infiltrated at OD 0,1.</EM></P><P><STRONG>Conclusion: </STRONG>It was demonstrated the proper PhiC31 functioning and determined its behavior under different protein concentrations and promoter strengths. Although the expected hypothesis may be that the more recombinase concentration the more inversion events,the obtained results suggest the existence of a threshold above which that assumption is not true. Therefore, a low recombinase expression is needed in order to maximize recombinase action.</P><P>Therefore, the following question raised: <STRONG>What is the reason for this counter-intuitive phenomenon? </STRONG>We proposed two hypotheses which may explain this:</P><OL class="jopeli"><LI>The high expression of recombinase may demand too much cell resources, causing a metabolic overload. Thus, luciferase expression decreases due to resource competition.</LI><LI>A PhiC31 tetramer binds to the attachment sites and it triggers the recombinase action. However, when PhiC31 is overexpressed, unproductive complexes are formed in the site-specific recombination sites obstructing the inversion event.</LI></OL><DIV class="heading-color jopesubapartado"><H4><SPAN>METABOLIC LOAD VS UNPRODUCTIVE COMPLEX HYPOTHESIS</SPAN></H4></DIV><P>An experimental approach was performed in order to demonstrate if the metabolic load hypothesis is correct. Following the <A href="https://2017.igem.org/Team:Valencia_UPV/Experiments#metabolic">experimental design</A> it was proved that there is no relation between a higher expression of PhiC31 with a lower luciferase expression in the plant cell (Figure 8).</P><P><EM><STRONG>Figure 8.</STRONG> a) Graphic representation of the construct comprised by a promoter and a terminator in opposite directions flanked by PhiC31 recombined attachment sites (OD* 0,02). b) Genetic construct that allows constitutive expression of phiC31 under a weak promoter. It was tested at several concentrations (OD* 0,01 and OD* 0,02). c) Construct that allows constitutive expression of luciferase under a strong promoter (35s Promoter). d) Graphic representation of luciferase expression levels after 48 hours post infiltration (blue) and 96 hours post infiltration (red). *Final Optical Density of Agrobacterium tumefaciens culture.</EM></P><P>As Figure 8 shows, the expression of luciferase under the strong promoter (35s promoter) decreases along with time because of the high metabolic load in the cell plant. However, there is no differential luciferase expression between both different recombinase expression concentrations when the inducible reporter was already activated before adding PhiC31. Suggesting that decrease of luciferase expression observed in the aforementioned experiment was not caused by metabolic load but by the formation of PhiC31 unproductive complexes.</P><P><STRONG>Conclusion</STRONG>: We demonstrate that metabolic load cannot explain the counter-intuitive hypothesis previously reported. That means PhiC31 unproductive complex formation hypothesis may be the correct one.</P><DIV class="heading-color jopesubapartado"><H4><SPAN>RECOMBINASE DIRECTIONALITY FACTOR (RDF) PERFORMANCE</SPAN></H4></DIV><P>In order to increase the genetic control over the expression of any color inducing protein through a viral vector strategy, a modular and orthogonal AND gate was designed. In normal basis, plant would be constitutively expressing PhiC31 recombinase and the stress-inducible promoter would be in OFF state (i.e. in opposite direction to the desired CDS) . In this case, PhiC31 attachment sites have been already recombined (attR and attL), being PhiC31 uncapable of inversing them back by itself. To do so a helper protein is required, so called recombinase directionality factor (<A href="https://2017.igem.org/Team:Valencia_UPV/Design">RDF</A>). With this approach recombinase switch becomes more robust, stable and less leaky.</P><P><EM><STRONG>Figure 9. Dynamic performance between RDF and PhiC31.</STRONG> a) Graphic representation of the register assembly comprised by a promoter and a terminator with the recombined sites. It represents the negative control in the graphic. b) Genetic construct that allows a constitutively expression of PhiC31 under a <STRONG>weak </STRONG>promoter (OD* <STRONG>0,01</STRONG>) c) Graphic representation of the genetic construct comprised by a promoter and a terminator with the attachment sites. It represents the positive control in the graphic. d) Graphic representation of the gp3 genetic construct. Its expression is under a <STRONG>strong </STRONG>promoter and was tested in both treatments (OD* <STRONG>0,1</STRONG> and OD* <STRONG>0,2</STRONG>). *Final Optical Density of Agrobacterium tumefaciens culture. e) Plot of the luciferase expression levels obtained over time. f) Bars chart representing the difference of luciferase expression levels obtained before and after RDF induction.</EM></P><P>Recombination directionality factor’s (RDF or gp3) ability to reverse the integrase actuation direction allowing the inducible stress-promoter to acquire the active state was demonstrated in this experiment. Following the <A href="https://2017.igem.org/Team:Valencia_UPV/Experiments#GP3">experimental design</A> we proved the effectiveness of the RDF in the recombination system.</P><P>As can be seen in Figure 9, the expression of gp3 together with PhiC31 trigger luciferase expression obtaining expression levels similars to a weak promoter (Pnos promoter). Interestingly, the highest level of luciferase expression (expression levels similar to a weak promoter — pNOS) was obtained with a low RDF expression (infiltrated at OD 0.1). Whereas, when the RDF was relatively high (infiltrated at OD 0.1, that is 2-fold in comparison with the other experimental treatment), the activation was lower than a weak promoter (pNOS) expression levels.</P><P><STRONG>Conclusion: </STRONG>The ability of the recombinase directionality factor (RDF) to reverse the integrase action direction was demonstrated. Thus, postulating as the best way to control the direction of the stress-inducible promoter to express a color inducing protein.</P><DIV class="heading-color jopesubapartado"><H4><SPAN>RDF CHARACTERIZATION</SPAN></H4></DIV><P>RDF:phiC31 ratios define re-inversion efficiency. Bearing that in mind, we resolved to demonstrate that the optimal ratio between RDF and PhiC31 is 20:1 using a weak promoter for the recombinase expression and a strong one for the RDF. However, several experiments were performanced in order to demonstrate this hypothesis.</P><P><STRONG><EM>Figure 10. Dynamic performance between RDF and PhiC31.</EM></STRONG><EM> a) Graphic representation of the register assembly comprised by a promoter and a terminator with the recombined sites. It represents the negative control in the graphic. b) Genetic construct that allows a constitutively expression of PhiC31 under a </EM><STRONG><EM>strong </EM></STRONG><EM>promoter (OD* </EM><STRONG><EM>0,1</EM></STRONG><EM>) c) Graphic representation of the genetic construct comprised by a promoter and a terminator with the attachment sites. It represents the positive control in the graphic. d) Graphic representation of the gp3 genetic construct. Its expression is under a </EM><STRONG><EM>strong </EM></STRONG><EM>promoter and was tested in both treatments (OD* </EM><STRONG><EM>0,05</EM></STRONG><EM> and OD* </EM><STRONG><EM>0,15</EM></STRONG><EM>). e) Line graph representing the data obtained of the level of luciferase expression over time for RDF module infiltrated at OD 0.05 and 0.15. f) Line graph representing the data obtained of the level of luciferase expression over time for RDF module infiltrated at OD 0.2 and 0.35. *Final Optical Density of Agrobacterium tumefaciens culture.</EM></P><P>As can be seen in Figure 10 e) and 10 f), any of the RDF’s ODs assessed were able to trigger luciferase expression (i.e. re-inverting the promoter).</P><P><STRONG>Conclusion: </STRONG>These results help to corroborate the unproductive complex hypothesis. When PhiC31 expression is under a strong promoter regulation, the formation of this complexes impedes the site-specific recombination event.</P><DIV class="heading-color jopesubapartado"><H4><SPAN>STRESS- INDUCIBLE PROMOTER</SPAN></H4></DIV><P>ChatterPlant proposes a modular and orthogonal <A href="https://2017.igem.org/Team:Valencia_UPV/Design">AND gate</A> as regulatory system for the color inducing protein. In this circuit leaf’s color changes are triggered by the designed genetic toggle switch comprised by an invertible stress-inducible promoter. This promoter up-regulates the expression of tobacco pathogenesis-related protein 1a (<A href="https://2017.igem.org/Team:Valencia_UPV/Parts#basicPartsPH">Pr-1a</A>) in presence of salicylic acid (pathogenesis signaling phytohormone).</P><P>Pr-1a promoter feasibility inside our circuit for flagging pathogen presence was assesed following the <A href="https://2017.igem.org/Team:Valencia_UPV/Experiments#stress">experimental design</A>.</P><P><STRONG><EM>Figure 11.</EM></STRONG><EM> a) Transcriptional unit for the expression of the Firefly luciferase under the control of a stress promoter. b) Transcriptional unit for the expression of the Firefly luciferase under the control of the stress promoter when Salicylic acid is being synthesized c) Transcriptional unit for the expression of the Firefly luciferase under the control of a weak promoter. d) Graphic representation of ratios luciferase/renilla of all treatments explained previously.</EM></P><P>As can be seen in Figure 11, a significative differential expression were obtained between the salicylic acid treatment and the mock one (just water). That means, those plants infiltrated with the inducible construct and treated with salicylic acid presented higher levels of luciferase expression over those treated only with water.</P><P><STRONG>Conclusion</STRONG>: Results show the suitability of using Pr1-a promoter for up-regulating expression levels. Therefore, Pr1-a and potentially other types of inducible promoters, postulate as candidates to develop a stress-inducible color code system.</P><DIV class="heading-color jopesubapartado"><H4><SPAN>COLOR CODE PROTEIN EXPRESSION BY VIRAL VECTOR DELIVERY</SPAN></H4></DIV><P>Viral vector auto-replicative capacity and cell-to-cell movement convert this strategy into one of the best strategies for high protein expression and yield production. Thus, we developed our proof of concept for this approach using the Tobacco etch virus (TEV) as a vehicle for the transport of the color inducing protein Rosea 1 (Ros1). This protein, Rosea 1 is an Antirrhinum majus transcription factor and it has been reported as an ideal visual reporter gene. Following the<A href="https://2017.igem.org/Team:Valencia_UPV/Design"> experimental design</A>, different N.benthamiana plants were infiltrated with A.tumefaciens culture carrying a TEV vector with Ros1 insert.</P><P><EM><STRONG>Figure 12</STRONG>. Images obtained two weeks after infiltrating A.tumefaciens carrying a PVX vector with a Ros1 insert inside it.</EM></P><P><STRONG>Conclusion: </STRONG>As shown in Figure 12, the suitability of plant viral vectors as color inducing system vehicles was demonstrated. Furthermore, the viral vector’s systemic movement was proved since the agroinfiltration was performed in old leaves while color change was observed not in the old leaves but in the younger ones.</P><DIV class="heading-color jopesubapartado"><H4><SPAN>CHROMOPROTEINS</SPAN></H4></DIV><P>Chromoproteins suppose a simple and effective way of inducing different color changes in a desired organism. Its implementation in our circuit, would confer it with versatility, facilitating the creation of a color code system. However, to date, a color change in plant organisms has not been reported.</P><P>In order to expand our Color Code System, we propose using chromoproteins since their absorbance spectrum is in the visible one and non-special equipment is necessary. Amil CP, AmajLime and eforRed (BioBricks supplied to the Registry by Uppsala iGEM team between 2011 and 2013) were codon-optimized for Nicotiana benthamiana in order to make its expression easier.</P><P><STRONG>Conclusion:</STRONG> Unfortunately, no color change was observed after infiltrating A.tumefaciens culture with the different chromoproteins (AmilCP, AmajLime and eforRed) either with their E.coli codon optimized sequence or with their plant codon optimized. Future experiments could be carried out after adding a chloroplast location sequence to these proteins. The transport of the chromoproteins to the chloroplast may help to their correct folding since its redox environment is more similar to the prokaryotic cytosol.</P></DIV><DIV id="section1" class="container"><DIV class="heading-color"><H2><SPAN>MODULE 3: CHATTERBOX
                           </SPAN></H2></DIV><P>Our aim with ChatterPlant was to increase sustainability of food production with a Synbio approach. To reach this final end different systems were tested and finally implemented in the device known as ChatterBox.</P><P>The first one is <STRONG>temperature regulation</STRONG>, key to control plant’s metabolism and homeostasis. With our refrigeration system we were able to stabilize 20 ºC in the growing chamber, an ideal temperature for growth.</P><P><EM><STRONG>Figure 13.</STRONG> Graphic representation of the temperature control carried out by the ChatterBox system.</EM></P><P>Also, <STRONG>humidity </STRONG>is a very important factor to keep in mind, a low humidity level can induce stress, so we designed and tested a low cost DIY humidifier capable of proving a fine mist.</P><P><EM><STRONG>Figure 14</STRONG>. Picture of a ChatterBox prototype used in one of our first light-induction expression experiments.</EM></P><P>Next step, light. With the white LED strips direct sunlight was substituted. Entering the SynBio field, with the “<STRONG>Optogenetic lamps</STRONG>” and their specific wavelength the conformation of a protein was changed in the plant’s genome.</P><P>Finally, <STRONG>color changes</STRONG> in the leaves caused by an infection were detected with the camera and recognition software as shown in Figure 15</P><P><EM><STRONG>Figure 15.</STRONG> Demonstration of the results of the visual recognition software (right) after processing a real plant picture with induction of leave’s color change (left).</EM></P><P> </P></DIV><DIV id="section1" class="container"><DIV class="heading-color"><H2><SPAN>MODULE 4: PLANTLABCO
                           </SPAN></H2></DIV><P><EM><STRONG>Figure 16</STRONG>. Frontpage of PlantLabCo online platform.</EM></P><P>The objective of PlantLabCo was to provide a solution to the main problems that Plant SynBio researches are facing up to know. In order to achieve this goal we studied the different tools that were provided in other fields of study to solve similar problems. After the proper research was taken we arrived to the point in which PlantLabCo is today. The four main approaches that we took were:</P><UL class="jopeli"><LI>Creating a forum where Plant SynBio researches can actively discuss and interact freely, discuss and explore every aspect of this field.</LI><LI>Provide a web repository for experiments to be upload and consulted following a standardized system of information and metadata.</LI><LI>Presenting a modeling software tool capable of recreating a vast amount of circuits to ease and help with the mathematical modeling as it is a key component on Plant SynBio</LI><LI>An informative magazine that works as a medium to spread and educate the general public about this field</LI></UL></DIV></DIV></DIV></DIV></SECTION><FOOTER id="footer"><DIV class="container" style="padding: 0px !important; margin: 0px !important; width: 100%;"><DIV class="row" style="margin: 10px;"><DIV class="col-md-8 jopefooter"><P style="margin: 0px !important; text-align: center !important;"><STRONG class="jopenegrita">Valencia UPV iGEM</STRONG> Team</P></DIV></DIV></DIV></FOOTER></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>