<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Valencia_UPV_Plant skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Valencia UPV/Plant</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="wrapper"><DIV id="header" class="navbar-toggleable-md sticky clearfix"><HEADER id="topNav"><DIV class="container"><DIV class="navbar-collapse collapse float-right nav-main-collapse submenu-color"><NAV class="nav-main"><UL id="topMain" class="nav nav-pills nav-main nav-hover-animate nav-animate-fadeIn"><LI class="dropdown"><A class="dropdown-toggle" href="#">
										TEAM
									</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Team">
												Team
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Attributions">
												Attributions
											</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" href="#">
										PROJECT
									</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Description">
												Description
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Applied_Design">		Applied Design
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Collaborations">
												Collaborations
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Achievement">
												Achievements
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Safety">
												Safety
											</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#">
										WET LAB
									</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Design">
												Biological Design
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Experiments">
												Experiments
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Protocols">
												Protocols
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Results">
												Results
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Notebook">
												Notebook
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Contribution">
												Contribution
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Plant">
												PlantSynBio
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Parts">
												Parts
											</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#">
										DRY LAB
									</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Model">
												Modeling
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Software">
												Software
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Hardware">
												Hardware
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/GraphicDesign">
												Graphic Design
											</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#">
										HUMAN PRACTICES
									</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/HP/Silver">
												Outreach
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/HP/Gold_Integrated">
												Integrated and gold
											</A></LI></NAV></DIV></DIV></HEADER></DIV><SECTION class="section-xs" style="border-bottom: none;"><DIV class="container"><DIV class="text-center"><P class="font-lato fw-300 lead mt-0" style="padding-top: 20px !important;margin-bottom: 20px !important;text-align: left !important;">Home • Wet Lab •</P><DIV class="col-md-12"><H3 class="jopetitles">PLANTSYNBIO</H3><DIV><BUTTON type="button" class="btn btn-primary" style="margin: 5px 5px 25px 5px !important; background-color: rgba(0,0,0,0.3) !important; border: none !important;"><A href="https://static.igem.org/mediawiki/2017/7/7d/PDFPlantSynBioValenciaUPViGEM.pdf" class="btn-dwnl" target="_blank"><SPAN style="margin-right: 2px;">PDF</SPAN></A></BUTTON></DIV></DIV></DIV></DIV></SECTION><SECTION class="section-xs"><DIV class="container"><DIV class="row"><DIV class="col-md-3"><DIV id="index" class="side-nav"><DIV class="side-nav-head"><H4 style="color:#005368; padding-left: 20px !important;">INDEX</H4></DIV><UL class="list-group list-unstyled"><LI class="list-group-item active"><A href="#section1">Abstract</A></LI><LI class="list-group-item"><A href="#section2">Introduction</A></LI><LI class="list-group-item"><A href="#section3">Plant Chassis</A></LI><LI class="list-group-item"><A href="#section4">Gene circuits design</A></LI><LI class="list-group-item"><A href="#section5">How does ref/far-red light swith work?</A></LI><LI class="list-group-item"><A href="#section6">PhytoBricks</A></LI></UL></DIV></DIV><DIV class="col-md-9"><DIV id="section1" class="container"><DIV class="heading-color"><H2><SPAN>ABSTRACT</SPAN></H2></DIV><P class="jopesangria">One of the main ambitions of Plant SynBio is to ensure a future sustainable agriculture by increasing food production and promoting seasonless proximity products. To achieve this goal, having <STRONG class="jopenegrita">complete control over plant development</STRONG> is decisive. ChatterPlant helps in this commitment by developing an integrated orthogonal interface between plants and humans.</P><P class="jopesangria">Bearing that in mind, two genetic circuits were engineered. The first one contains an <STRONG class="jopenegrita">optogenetic switch</STRONG> allowing us to intervene in the plant development by regulating its genetic profile. The second one consists of a color code sensor circuit that flags the presence of specific plant stresses. This color code circuit contains different <STRONG class="jopenegrita">AND gates</STRONG> triggered by recombinase actuation together with different plant-specific inducible promoters. Furthermore, viral vectors were implemented in both circuits to amplify output signal. Finally, all Phytobricks built in this project can be reused in novel plant genetic circuits, thus contributing to <STRONG class="jopenegrita">Plant SynBio progress</STRONG>.</P></DIV><DIV id="section2" class="container"><DIV class="heading-color"><H2><SPAN>INTRODUCTION</SPAN></H2></DIV><P class="jopesangria">Plant Synthetic Biology is a novel interdisciplinary field that offers the pathway to design and alter natural systems in order to achieve plants’ predictable behaviours. This novel field aims to play an essential role in <STRONG class="jopenegrita">future agriculture</STRONG> by enhancing food production or increasing food nutritional quality.</P><P class="jopesangria">Therefore, one of their greater ambitions is producing plants with customized functionality through the design of genetic devices. For that purpose, two concepts should be considered. First, the engineering principles (Modularity, abstraction, standardization and orthogonality) must be applied to facilitate the <STRONG class="jopenegrita">design of new biological devices</STRONG> or Phytobricks. Second, an exhaustive characterization is necessary in order to predict the behaviour of plant genetic devices.</P><P class="jopesangria">ChatterPlant follows SynBio principles to design, develop and implement plant genetic devices, which help in the commitment of creating a brand-new more sustainable agriculture. Furthermore, these devices have been generated following SynBio principles by using standard and <STRONG class="jopenegrita">modular approaches</STRONG>, thus generating genetic tools not only for ChatterPlant but for any future Plant SynBio project.</P><DIV id="section3" class="heading-color"><H4><SPAN>PLANT CHASSIS</SPAN></H4></DIV><P class="jopesangria">Bacterial systems were the first chassis chosen due to its simplicity. However, complex proteins (i.e. glycoproteins) need many post translational modification in order to provide the native structure. Therefore, yeast and mammalian cells were studied to spread SynBio possibilities.</P><P class="jopesangria">Finally, plants arose as a new and interesting chassis in SynBio progress. They are one of the main primary sources of biomass as they can provide basic nutrients and valuable secondary metabolites. Furthermore, plants suppose an advantageous platform for recombinant protein production which can be very useful for laboratories and researchers.</P><P class="jopesangria">Plants’ interactome is far more complicated due to its complex metabolism, difficulting the prediction of circuits behaviour. However, Plant SynBio offers multiple purposes such as making molecular pharming easier and more efficient, reprogramming metabolisms or dealing with plant stress conditions.  </P><P class="jopesangria">ChatterPlant proposes a new sustainable agriculture for any desired plant species. However, <STRONG class="jopenegrita">N. benthamiana</STRONG> was chosen in our proof of concept due to its fast growth and easy genetic manipulation. In this way, agroinfiltration procedure provides a straight-forward and relatively fast method to test genetic circuits by transiently introducing heterologous genes inside plant cells. </P><DIV id="section4" class="heading-color"><H4><SPAN>GENE CIRCUITS DESIGN</SPAN></H4></DIV><P class="jopesangria">This year, our team proposes ChatterPlant, a SynBio-based solution that works as human-plant interface to allow this bidirectional communication. An optogenetic circuit and a genetic AND gate were designed to control both development and behaviour making crops more accessible and sustainable</P><P class="jopesangria">The communication channel between humans and plants is comprised by a designed optogenetic circuit which is triggered by the red/far-red light inducible switch.</P><DIV id="section5" class="heading-color"><H4><SPAN>HOW DOES RED/FAR-RED SWITCH WORK?</SPAN></H4></DIV><P class="jopesangria"><I class="jopecursiva">Figure 1: Graphic design of human-plant circuit. a)  The transcriptional factor, PIF6, is fused to a DNA-binding domain (E) and PhyB is fused to the activator domain (VP64) and a nuclear location sequence (NLS). Both genetic expressions are controlled by plant strong promoters. b) When irradiated with 660nm light, PhyB changes its conformation and this complex is recruited to PIF6 at the promoter site. The polymerase III will recognize the activation domain and the transcription will begin. Only upon absorption of a far-red photon (740nm) the interaction between PhyB and PIF6 is terminated, resulting in a shut-off of gene expression.</I></P><P class="jopesangria">Furthermore, the <STRONG class="jopenegrita">Phytochrome B </STRONG>(PhyB) is fused to an activator domain (VP64) and a nuclear location sequence (NLS). Thus, when plant is irradiated with 660nm light, PhyB’s conformation is modified, allowing its interaction with PIF6 and transporting VP64 activator domain to the proximal upstream region of the desired CDS. Therefore, transcription is up-regulated by promoting the polymerase III complex recruitment through VP64 activity. PhyB-PIF6 interaction will end when PhyB’s conformation returns to its inactivated state after absorbing a far-red photon (740nm), thus resulting in a shut-off of gene expression (Fig.1).</P><P class="jopesangria">However, this approach has some concerns that need to be addressed. First, since white light interferes with circuit signaling in aerial parts of the plant and plant growth and development would be altered under light deprivation conditions, we postulate plant root cells as ideal hosts for our optogenetic circuit.</P><P class="jopesangria">Thus, the circuit could be regulated by irradiating <STRONG class="jopenegrita">plant’s roots</STRONG> with red or far-red stimuli. Several root-specific promoters have been reported for genetic applications. In order to delimit the activity of this system only to roots, root-specific promoters must be used for regulating the constitutive expression of both E-PIF6 and PhyB-VP64 fusion proteins genes. One example is NtREL1 whose expression pattern is strictly root-specific. Second, in order to make this optogenetic circuit modular to cover almost any necessity users may have, an <STRONG class="jopenegrita">autoreplicative vector</STRONG> is implemented as final interchangeable element. This vector, derived from potato virus X (PVX) vector, confers to the optogenetic circuit with the ability of propagating the expression of any desired protein from the roots to the rest of the plant. </P><P class="jopesangria">Moreover, we do not only intend to give orders to plants but also to acquire relevant information from them. Therefore, we generated a second communication channel through a Color Code System that allows plants to send us information about their current status (e.g. salt stress, pathogen presence, lack of some nutrient …). We resolved to detect these biotic and abiotic stresses using a modular and orthogonal genetic AND gate. Thus, color changes on plants are triggered when two different signals coincide. One of this signals corresponds to the presence of a determined stress, while the other is triggered when user apply an external chemical stimuli (i.e. dexamethasone).</P><P class="jopesangria"><I class="jopecursiva">Figure 2. Graphic representation of the toggle switch in ON state only when PhiC31 and gp3 are expressed. Only when the promoter is activated under stress conditions (e.g. Production of salicylic acid or Jasmonic acid), corresponding color protein will be expressed.</I></P><P class="jopesangria"> This circuit is fundamented on the utilization of an <STRONG class="jopenegrita">invertible and stress-inducible promoter</STRONG>.  First, the constitutive expression of PhiC31 recombinase keeps the inducible promoter in OFF state (i.e. in opposite direction to the CDS). However, when a Recombination Directionality Factor (RDF) protein expression is up-regulated by a controlled external stimulus, the stress-inducible promoter will be inverted. Once inversion led by RDF-phiC31 complex occurs, promoter acquires the proper orientation to regulate the transcription of a CDS coding a determined color protein (Fig.2). In this situation, if the plant is experiencing some kind of stress the respective stress-inducible promoter will be activated and the expression of a determined color protein up-regulated. Finally, the user will see a change in plant coloration and will allow him to apply corrective measures.</P><DIV id="section6" class="heading-color"><H4><SPAN>PHYTOBRICKS</SPAN></H4></DIV><P class="jopesangria">In order to build the aforementioned genetic circuits several parts and genetic devices were generated. However, the usefulness of these parts is not limited to the context of our project since all of them have been generated following <STRONG class="jopenegrita">Phytobricks standard</STRONG>. Thus, the plethora of different Phytobricks we provided the registry with can be now used in any other Plant SynBio project. Bearing that in mind, we have provided iGEM’s registry with four different types of parts. </P><P class="jopesangria">First of all, the basic parts required to implement PhiC31 recombinase in plant genetic circuits have been submitted. Thus, enabling the possibility of developing more elaborated circuit networks which allow plant to carry out more complex tasks. The implementation of these parts could even confer memory to a determined circuit.</P><P class="jopesangria">Second, the required parts for implementing an optogenetic toggle switch in any plant genetic circuit have also been submitted to the Registry. Thus, enabling the construction of plant genetic circuits which can be easily and accurately controlled by light stimuli.</P><P class="jopesangria">Third, different stress-inducible promoters have also been submitted to the Registry, which expand plant genetic circuits possibilities by increasing the number of available signal inputs that can actuate over them (i.e. pathogen presence).</P><P class="jopesangria">Finally, we have submitted to the Registry new plant-codon optimized chromoproteins derived from Uppsala 2012 Team’s chromoproteins. Thus, we do not only increase the number of reporter genes available in plants but also have provided the Registry with reporter genes which can be easily detectable with the naked eye.</P><P class="jopesangria">With all the plethora of different Phytobricks we provided the Registry with, we have helped in contributing Plant SyBio progress. We have contributed to enable the engineering of more versatile and complex plant genetic circuits architectures which will be able to perform more complex tasks and even processes we cannot imagine now. We have contributed with these standard, modular and reusable parts to <STRONG class="jopenegrita">broaden Plant SynBio possibilities</STRONG> and to make any Plant SynBio user’s desires or ideas feasible.</P></DIV></DIV></DIV></DIV></SECTION><FOOTER id="footer"><DIV class="container" style="padding: 0px !important; margin: 0px !important; width: 100%;"><DIV class="row" style="margin: 10px;"><DIV class="col-md-8 jopefooter"><P style="margin: 0px !important; text-align: center !important;"><STRONG class="jopenegrita">Valencia UPV iGEM</STRONG> Team</P></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>