Team:Valencia UPV/Protocols
TEAM
Team
Attributions
PROJECT
Description
Applied Design
Collaborations
Achievements
Safety
WET LAB
Biological Design
Experiments
Protocols
Results
Notebook
Contribution
PlantSynBio
Parts
DRY LAB
Modeling
Software
Hardware
Graphic Design
HUMAN PRACTICES
Outreach
Integrated and gold
Home • Wet Lab •
PROTOCOLS
PDF
INDEX
Introduction
Goldenbraid Ligation Reaction
Bacterial Transformation
E.Coli Top10 Culture Conditions
Agrobacterium Tumefaciens GV3101 Culture Conditions
E.coli Plasmid DNA Isolation
Agrobacterium Tumefaciens Plasmid DNA Isolation
Miniprep Digestion
PCR Amplification
Agarose Gel Electrophoresis
Gel Band Purification
Agrobacterium Infiltration
Luciferase Assay
N. Benthamiana Genomic DNA Extraction
Goldenbraid Assembly Protocol
Gibson Assembly Protocol
INTRODUCTION
ChatterPlant team would like to share the protocols that we have been using over the summer. This collection of protocols has been developed for the purpose of endow scientific community with well characterized standards. Here are described the exact methods we use to proceed with the experiments and generate our results. Thus they are shared in the interest of reproducibility.
GOLDENBRAID LIGATION REACTION
Restriction-ligation reactions were set up as described elsewhere (Sarrion-Perdigones et al., 2011) using BsaI, BsmBI, BtgZI, or BbsI as restriction enzymes and T4 Ligase. Reactions were set up in 25- or 50-cycle digestion/ligation reactions (2 min at 37°C, 5 min at 16°C), depending on assembly complexity. The assembly of DNA fragments is an efficient technique where multiple inserts could be assembled into a vector backbone. The net result is the ordered and seamless assembly of DNA fragments in only one reaction.
Reagent
Volume (μL)
Concentration
DNA fragments
1 of each one
75 ng/ μL
Plasmid (pUPD2 - α1 - α2)
1
75 ng/ μL
BSA
1.2
10 X
Ligase Buffer
1.2
10 X
BsmbI
1
-
T4 ligase
1
-
H2O milli-Q
Raise until final volume (12 μL)
-
BACTERIAL TRANSFORMATION
Plasmids obtained from ligation reactions were introduced inside Electrocompetent Escherichia coli DH5α cells through electroporation. Electrocompetent Agrobacterium tumefaciens is used once constructs have been validated and ready to be expressed in Nicotiana benthamiana.
The electroporation cuvette needs to be in ice 10 minutes before inserting the cells. An aliquot of electrocompetent cells is taken out of the -80ºC freezer, and must be put immediately into ice.
1-2 µl of ligation product is used and added carefully to them.
60 µl of the mix are taken and put into an electroporation cuvette making sure that there are no bubbles.
The cuvette is dried and put in the electroporator, making sure that any spark has appeared. In that case, the process has not worked.
1500V electric field was applied to E. coli cells whereas 1440V is applied in Agrobacterium tumefaciens cells.
Transformed cells are resuspended in 300 μL of LB and put into an Eppendorf letting them grow in the shaker for 1 hours at 37ºC.
E.COLI TOP10 CULTURE CONDITIONS
This strain was grown in Luria-Bertani broth (LB) at 37ºC for 16h under continuous stirring conditions (200rpm). Antibiotic varies depending on resistance gene carried by each vector: ampicillin (pUPD vector), chloramphenicol (pUPD2 vector), kanamycin (alpha vector) or spectinomycin (omega vector). As far as plate culture is concerned, LB agar supplemented with the corresponding antibiotic was used. Isopropyl β-D-1-thiogalatopyranoside (IPTG) 0,5mM and 5-bromo-4-chloro-3-indolyl-D-galactopryanoside (X-Gal) 40µg/mL were also added to LB agar to allow color selection of colonies after 16 h.
AGROBACTERIUM TUMEFACIENS GV3101 CULTURE CONDITIONS
This strain was grown in Luria-Bertani broth (LB) at 28ºC for 2 days under continuous stirring conditions (200rpm). LB was supplemented with the corresponding antibiotic for each plasmid and with rifampicin (50µg/mL). As far as plate culture is concerned, LB agar supplemented with the corresponding antibiotics was used to select colonies after 2 days at 28ºC.
E.COLI PLASMID DNA ISOLATION
Plasmid DNA isolation was carried out using commercial kits and following manufacture’s protocol. E.Z.N.A. Plasmid Mini Kit I (Omega Bio-Tek, Norcross, USA) was utilized in order to extract plasmid DNA from E. coli cells. The protocol is the following:
Grow 1-5 mL culture overnight in a 10-20 mL culture tube.
Centrifuge at 10.000 x g for 1minute at room temperature. Decant or aspirate and discard the culture media.
Add 250 µl Solution I mixed with RNase A. Vortex or pipet up and down to mix thoroughly. Transfer suspension into a new 1.5mL microcentrifuge tube.
Add 250 µl Solutions II. Invert and gently rotate the tube several times to obtain a clear lysate. A 2-3 minutes incubation may be necessary. Avoid vigorous mixing and do not exceed a 5 minutes incubation.
Add 350 µl Solution III. Immediately invert several times until a flocculent white precipitate forms. Centrifuge at maximum speed (>13.000xg) for 10 minutes. A compact white pellet will form. Promptly proceed to the next step.
Insert a HiBind DNA Mini Column into a 2 mL Collection tube.
Transfer the cleared supernatant from Step 6 CAREFULLY aspirating it into the HiBind DNA Mini Column. Centrifuge at maximum speed for 1 minute. Discard the filtrate and reuse the collection tube.
Add 500 µl HBC Buffer diluted with isopropanol
Centrifuge at maximum speed for 1 minute. Discard the filtrate and reuse collection tube.
Add 700 µl DNA Wash Buffer diluted with ethanol. Centrifuge at maximum speed for 1 minute. Discard the filtrate and reuse the collection tube.
Centrifuge the empty HiBind DNA Mini Column at maximum speed for 2 minutes to dry the column. This step is critical for removal of trace ethanol that may interfere with downstream applications.
Transfer the HiBind DNA Mini Column into a nuclease-free 1.5 mL microcentrifuge tube.
Add 50 µl Elution Buffer or sterile deionized water directly to the center of the column membrane. Let sit at room temperature for 1 minute. Centrifuge at maximum speed for 1 minute.
Store eluted DNA at -20ºC.
AGROBACTERIUM TUMEFACIENS PLASMID DNA ISOLATION
QIAprep Spin Miniprep Kit (Qiagen, Hlden, Germnay) was used to extract plasmid DNA from A. tumefaciens with several alterations.
Pellet twice bacterial culture by centrifugation at >8000 rpm for 3 min at rt.
Resuspend pelleted bacterial cells in 250 µl Buffer P1 and 20 µl of lysozyme.
Add 250 µl Buffer P2 and mix thoroughly by inverting the tube 4-6 times until the solution turns blue.
Add 350 µl Buffer N3 and mix immediately and thoroughly by inverting the tube 4-6 times. If using LyseBlue reagent, the solution will turn colorless.
Centrifuge for 10 min at 13000rpm.
Apply the supernatant to the QIAprep spin column by decanting. Centrifuge for 30-60s and discard the flow-through the solution through the QIAprep spin column.
Wash the QIAprep spin column by adding 500 µl Buffer PB. Centrifuge for 30-60 s and discard the flow-through.
Wash the QIAprep spin column by adding 750 µl Buffer PE. Centrifuge for 30-60 s and discard the flow-through or apply vacuum to the manifold to draw the solution through the QIAprep spin column and switch off the vacuum source. Transfer the QIAprep spin column to the collection tube.
Centrifuge for 1 min to remove residual wash buffer.
Place the QIAprep column in a clean 1.5 ml microcentrifuge tube. To elute DNA, add 50 µl Buffer EB (10 mM Tris Cl, pH 8.5) or water to the center of the QIAprep spin column, let it stand for 1 min, and centrifuge for 1 min.
MINIPREP DIGESTION
One or more restriction enzymes are used to digest the DNA resulting in either non-directional or directional insertion into the compatible plasmid. In order to carry out the digestion to check if desired fragment is inside the plasmid it is necessary perform digestion reactions following manufacturer’s protocol (Thermo Scientific, Rockford, USA):
1X Reaction buffer, 0,5 enzyme unities and 200-500 ng of plasmid DNA.
PCR AMPLIFICATION
Different PCR reactions were performed with GB domestication. Benchling online tool was used to carry out PCR in silico simulations. Primers annealing temperature was determined for each PCR reaction through New England Biolabs online calculator tool (http://tmcalculator.neb.com). For domestication and cloning, Phusion® DNA polymerase (New England Biolabs, Ipswich, USA) was utilized in a reaction with conditions shown in Table 2.
Reactants
Reaction Conditions
Template DNA (10ng)
Initial denaturation
98ºC
5min
1 cycle
dNTPs mix (200µM)
Denaturation
98ºC
5-10s
35 cycles
Forward primer (0.5µM)
Annealing
45-72ºC
10-30s
 
Reverse primer (0.5µM)
Extension
72ºC
15-30s/kb
 
Phusion® DNA polymerase (1 unit)
Final extension
72ºC
10min
1 cycle
1x Phusion® HF Buffer
Hold
4ºC
∞
1 cycle
AGAROSE GEL ELECTROPHORESIS
Agarose gel electrophoresis were carried out to check restriction assays and PCRs results. Gels were composed by a variable concentration of agarose depending on each analysis necessities, TAE 1X (40mM Tris-acetatE, 1mM EDTA) and Ethidium Bromide. “GeneRuler DNA Ladder 1kb and 100bp” were used as size markers (Thermo Scientific, Rockford, USA). “Orange Loading Dye” (Thermo Scientific, Rockford, USA) was added to samples before loading the gel. Electrophoresis were carried out at different constant current intensities depending on the indications of manufacturer of electrophoresis cuvette. Once electrophoresis finished, gels were visualized with a transilluminator (Syngene, Cambridge, UK) and pictures of the resulting electrophoresis bands were taken with GeneSnap software.
GEL BAND PURIFICATION
Excise the DNA band from the agarose gel with a clean, sharp scalpel. Minimize the size of the gel slice by removing excess agarose. Use a 1.5 ml microfuge tube for processing up to 250 mg agarose.
Weigh the gel slice in a colorless tube. Add 3 volumes of Buffer QX1 to 1 volume of gel for DNA fragments 100 bp – 4 kb; otherwise, follow the table below.
Resuspend QIAEX II by vortexing for 30 s. Add QIAEX II to the sample.
Incubate at 50°C for 10 min to solubilize the agarose and bind the DNA. Mix by vortexing* every 2 min to keep QIAEX II in suspension. Check that the color of the mixture is yellow.
Centrifuge the sample for 30 s and carefully remove supernatant with a pipet.
Wash the pellet with 500 µl of Buffer QX1. Resuspend the pellet by vortexing. Centrifuge the sample for 30 s and remove all traces of supernatant with a pipet. This wash step removes residual agarose contaminants.
Wash the pellet twice with 500 µl of Buffer PE. Resuspend the pellet by vortexing. Centrifuge the sample for 30 s and carefully remove all traces of supernatant with a pipet. These washing steps remove residual salt contaminants.
Air-dry the pellet for 10–15 min or until the pellet becomes white and then resuspend the pellet with H20 by vortexing.
AGROBACTERIUM INFILTRATION
To carry out transient expression experiments, the method described by Orzaez et al. was followed (Orzaez et al. 2009). First of all, Agrobacterium tumefaciens GV3101 cultures incubated at 28ºC overnight were centrifuged at 3000rpm for 15 minutes. Cultures were pelleted and resuspended in 5mL of agroinfiltration solution (MES 10mM, pH 5.6, MgCl2 10mM and acetosyringone 200mM diluted with dimethyilsulfoxide (DMSO)).
Once pellet was resuspended, cultures were protected from light and incubated at room temperature for 2h under stirring conditions. Afterwards, cultures Optical Density at 600nm (OD600) was measured with a spectrophotometer, and readjusted to the desired OD by diluting with agroinfiltration solution. If coinfiltration with different genetic constructions is needed, A.tumefaciens culture with the corresponding plasmids were equally mixed.
Agroinfiltration mix was inoculated in 4 weeks old N. benthamiana plants using needleless syringes through abaxial surface of the three youngest leaves of each plant. Plant growing conditions were 16 light hours at 24ºC and 8 dark hours photoperiod at 20ºC. Finally, leaves samples were taken 5 days post infiltration (d.p.i.).
LUCIFERASE ASSAY
This procedure is done with the Promega; kit (Dual-Luciferase Reporter Assay System). Using Agrobacterium tumefaciens as a vehicle to insert the desired devise, it is necessary insert it into a leaf of N.benthamiana by a direct injection. This method is known as Agroinfiltration. The next step is letting infiltrated leafs for two or three days depending on how the experiment is programmed.
After two days post infiltration, users can get leaf disks from N.benthamiana using a hole punch. It is recommended to take the maximum agroinfiltrated area avoiding plant nerves. Leaf discs are put in a specific plate depending on the light condition requirements. Different samples are taken during the next two days after discs were made and immediately they are put in liquid nitrogen and stored at -80ºC. The steps to follow are:
The Passive lysis buffer 1x is prepared. Each disk of leaf needs 200µl. The passive lysis buffer is stored at 5X so it must be diluted with distilled water. Place it on the ice besides the LUCII substrate and the STOP solution.
Cut off two little leaf disks of approximately 0.8 cm and put it into an Eppendorf tube. Immediately, freeze it with liquid nitrogen to avoid the deterioration of vegetal material. Grind the freeze sample using the metabolomics robot. Put the samples on ice.
Add 150µl of passive lysis buffer 1x to each Eppendorf tube.
Vortex gently and centrifuge 13200rpm during 15 minutes at 4ºC. While switch on the luminometer.
Dilute 2:3 the extracts on a new Eppendorf tube. Add 36µl of Passive lysis buffer 1x and 24µl of sample.
Take an optimal plate to use in the luminometer. Fill luminometer wells with 40 µl of LUCII which is stored at -20ºC.
10 µl of sample is added in each well. Wait 10min. During this time turn on and configure the luminometer.
Measure luciferase activity.
Prepare 40 µl/well of Dual Glo 1x (STOP solution + substrate). The substrate is at 50x concentration and stored at –20ºC.
When the first luciferase measure is done, it is necessary to add 40 µl of Dual Glo into each well. Let it rest during 10 min.
Measure the Renilla activity.
Take the obtained information and analyze it.
Things to keep in mind for the next experiment: The luminometer (machine to measure the luminescence) has to be ready before adding the reagents to the samples because it needs 10min to be ready. Set the timer (10min) with the first sample of luciferase and add the reactant to the other samples as quick as possible.
N. BENTHAMIANA GENOMIC DNA EXTRACTION
Samples for genomic DNA extraction were collected from 5 days post infiltrated leaves. For genomic DNA extraction, 50 mg of tissue powder coming from a pool of three leaves were mixed with 500µL of DNA extraction buffer (200mM TrisHCl-pH 7.5, 250mM NaCl, 25mM EDTA, 0.5% SDS). Plant extract was mixed gently and it was spun at 14000 x g for 3 minutes). The supernatant was transferred to a new tube and an equal volume of isopropanol was added for DNA precipitation. The supernatant was carefully removed after centrifugation (5min at 14000 x g) and the DNA was washed twice with 70% ethanol. The pellet was dried for half an hour and it was dissolved with 100µL of elution buffer (10Mm TrisHCl-pH8, 1mM EDTA).
GOLDENBRAID ASSEMBLY PROTOCOL
Domestication strategy described by Sarrion-Perdigones and co-workers (Sarrion-Perdigones et al. 2013) was followed to create all the GBparts used in this study. For level 0 parts 40ng of DNA were cloned into pUPD2 (Universal Parts Domesticator Plasmid 2) with a BsmBI restriction-ligation reaction. GoldenGate-like multipartite BsaI restriction-ligation reactions were performed to obtain fully functional transcriptional units (TU) from basic domesticated genetic parts (level 0 parts).
GIBSON ASSEMBLY PROTOCOL
NEB recommends a total of 0.02–0.5 pmols of DNA fragments when 1 or 2 fragments are being assembled into a vector and 0.2–1.0 pmols of DNA fragments when 4–6 fragments are being assembled. Efficiency of assembly decreases as the number or length of fragments increases. To calculate the number of pmols of each fragment for optimal assembly, based on fragment length and weight, we recommend using NEB's online tool, NEBioCalculator. The mass of each fragment was measured using the NanoDrop instrument. The steps to follow:
Set up the following reaction on ice:
Fragments Used for Assembly
 
2-3 Fragment Assembly
4-6 Fragment Assembly
Positive Control
Total Amount of Fragments
0.02-0.5 pmols*    X μl
0.2-1 pmols*    X μl
10 μl
Gibson Assembly Master Mix (2X)
10 μl
10 μl
10 μl
Deionized H2O
10-X μl
10-X μl
0
Total Volume
20 μl
20 μl
20 μl
* Optimized cloning efficiency is 50–100 ng of vectors with 2–3 fold of excess inserts. Use 5 times more of inserts if size is less than 200 bps. Total volume of unpurified PCR fragments in Gibson Assembly reaction should not exceed 20%.
Incubate samples in a thermocycler at 50°C for 15 minutes when 2 or 3 fragments are being assembled or 60 minutes when 4-6 fragments are being assembled. Following incubation, store samples on ice or at –20°C for subsequent transformation.
Transform NEB 5-alpha Competent E. coli cells (provided with the kit) with 2 μl of the assembly reaction, following the transformation protocol.
Valencia UPV iGEM Team
