<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Valencia_UPV_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Valencia UPV/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="wrapper"><DIV id="header" class="navbar-toggleable-md sticky clearfix"><HEADER id="topNav"><DIV class="container"><DIV class="navbar-collapse collapse float-right nav-main-collapse submenu-color"><NAV class="nav-main"><UL id="topMain" class="nav nav-pills nav-main nav-hover-animate nav-animate-fadeIn"><LI class="dropdown"><A class="dropdown-toggle" href="#">
										TEAM
									</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Team">
												Team
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Attributions">
												Attributions
											</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" href="#">
										PROJECT
									</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Description">
												Description
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Results">
												Results
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Protocols">
												Protocols
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Applied_Design">
												Applied Design
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Collaborations">
												Collaborations
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Achievement">
												Achievements
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Safety">
												Safety
											</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#">
										WET LAB
									</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Design">
												Biological Design
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Experiments">
												Experiments
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Notebook">
												Notebook
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Contribution">
												Contribution
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Plant">
												PlantSynBio
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Parts">
												Parts
											</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#">
										DRY LAB
									</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Model">
												Modeling
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Software">
												Software
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/Hardware">
												Hardware
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/GraphicDesign">
												Graphic Design
											</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#">
										HUMAN PRACTICES
									</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/HP/Silver">
												Outreach
											</A></LI><LI><A href="https://2017.igem.org/Team:Valencia_UPV/HP/Gold_Integrated">
												Integrated and gold
											</A></LI></NAV></DIV></DIV></HEADER></DIV><SECTION class="section-xs" style="border-bottom: none;"><DIV class="container"><DIV class="text-center"><P class="font-lato fw-300 lead mt-0" style="padding-top: 20px !important;margin-bottom: 20px !important;text-align: left !important;">Home • Wet Lab •</P><DIV class="col-md-12"><H3 class="jopetitles">BIOLOGICAL DESIGN</H3><DIV><BUTTON type="button" class="btn btn-primary" style="margin: 5px 5px 25px 5px !important; background-color: rgba(0,0,0,0.3) !important; border: none !important;"><A href="https://static.igem.org/mediawiki/2017/1/1c/PDFBiologicalDesignValenciaUPV.pdf" class="btn-dwnl" target="_blank"><SPAN style="margin-right: 2px;">PDF</SPAN></A></BUTTON></DIV></DIV></DIV></DIV></SECTION><SECTION class="section-xs"><DIV class="container"><DIV class="row"><DIV class="col-md-3"><DIV id="index" class="side-nav"><DIV class="side-nav-head"><H4 style="color:#005368; padding-left: 20px !important;">INDEX</H4></DIV><UL class="list-group list-unstyled"><LI class="list-group-item"><A class="dropdown-toggle" href="#section1">Human-Plant</A></LI><LI><A href="#optcirc">Optogenetic Switch</A></LI><LI><A href="#light">Light Application Site</A></LI><LI><A href="#transport">Transport From Roots</A></LI><LI><A href="#why">Why an ON-OFF System?</A></LI></UL><LI class="list-group-item"><A class="dropdown-toggle" href="#section1">How can we communicate with plants?</A></LI><LI><A href="#toggle">Toggle Switch</A></LI><LI><A href="#gp3">GP3 Delivery</A></LI><LI><A href="#and">AND Gate</A></LI><LI><A href="#chromoproteins">Chromoproteins</A></LI><LI class="list-group-item"><A href="#section3">Bibliography</A></LI></DIV></DIV><DIV class="col-md-9"><DIV id="section1" class="container"><DIV class="heading-color"><H2><SPAN>HUMAN-PLANT</SPAN></H2></DIV><P class="jopesangria">As far as communication is concerned, a bidirectional channel must be established. In our project, we evidence that is possible to deliver stimuli light orders into plants. An optogenetic circuit was designed to establish a reliable communication channel between humans and plants. We can control both plant development and behavior with the aim of helping to achieve a future sustainable agriculture</P><P class="jopesangria">One of the main challenges in plant platforms is using light to precisely control cellular behavior. Recently, it started to be addressed and ChatterPlant aims to contribute to this knowledge. In order to control in time and space protein expression levels accurately, light irradiation can be used due to its easy manipulation and precision. </P><P class="jopesangria">Therefore, we resolved designing a light-inducible system based on red/far-red light in order to control the genetic expression on any desired protein (Müller, Naumann, Weber &amp; Zurbriggen, 2015). Therefore, we will be able to regulate plants at genetic level accurately, contributing to the development of a more efficient and sustainable agriculture in our world. </P><DIV id="optcirc" class="heading-color jopesubapartado"><H4><SPAN>OPTOGENETIC SWITCH</SPAN></H4></DIV><P class="jopesangria">Control of gene expression is carried out using a red/far-red light-switch (Müller et al., 2014), allowing us to activate the genetic circuit with red light (660 nm) and turn it off by subsequent illumination with far-red light (740 nm). The molecular response of this optogenetic switch is based on the interaction between phytochrome-interacting factor 6 (PIF6) and phytochrome B (PhyB).</P><P class="jopesangria">The photoreceptor (PhyB) was isolated from Arabidopsis thaliana and it can integrate light signals, as well as it participates in the control of floral induction and germination. Despite this, this protein was found to be useful as a tool for Plant Synthetic Biology. As a photosensor protein, is able to change its conformation to the active form when receiving red photons (660 nm). Once in an activated state, it will be able to interact specifically with the transcription factor PIF6 (Khanna, 2004). Bounded complex allows expression of the desired coding sequence under control of operator site Etr8 (an E-responsive operator motif) and minimal promoter (minCMV).</P><P class="jopesangria">In order to allow any protein expression, PIF6 is fused to a DNA-binding domain (E), which binds to the operator site (Etr8) in the construction we intend to express. Equally, PhyB is linked to a nuclear location sequence (NLS) and an activator domain (VP64), that has the ability to act as a strong transcriptional activator of a gene. This ON state can up-regulate the transcription of the next element of the circuit sparking off the signaling pathway. Likewise, absorption of a far-red photon (740 nm) converts PhyB into the inactive form, leading to dissociation from PIF6 and turning the circuit in an OFF state. (Ni, Tepperman &amp; Quail, 1999). Once genetic pathway is inactivated, gene transcription returns to its basal expression. Plants can produce wanted transcripts in proper time, avoiding continuously even unnecessary expression (i.e. decreasing cell-burden effects).</P><P class="jopesangria"><I class="jopecursiva">Figure 1: Graphic design of human-plant circuit. a)  The transcriptional factor, PIF6, is fused to a DNA-binding domain (E) and PhyB is fused to the activator domain (VP64) and a nuclear location sequence (NLS). Both genetic expressions are controlled by plant strong promoters. b) When irradiated with 660nm light, PhyB changes its conformation and this complex is recruited to PIF6 at the promoter site. The polymerase III will recognize the activation domain and the transcription will begin. Only upon absorption of a far-red photon the interaction between PhyB and PIF6 is terminated, resulting in a shut-off of gene expression.</I></P><P class="jopesangria">We proposed using Flowering Locus T (FT) because of its ability of inducing flowering process when it is activated. This protein is constitutively expressed in plant cotyledons and leaves, and it involves a remarkable and conservable role in plants. In response to inductive long days Flowering Locus T protein (i.e. phosphatidylethanolamine-binding protein) acts in the shoot apex to induce target meristem identity genes such as APETALA1 (AP1) (Notaguchi, Daimon, Abe &amp; Araki, 2009) and initiates floral morphogenesis. Bearing that in mind, we propose to regulate FT transcriptional rate in order to help improving the current agriculture.</P><P class="jopesangria">However, a modular and standard system was designed in order to reach all users’ necessities. Circuit’s modularity allows to select easily the light-regulated cellular process by changing the output module. Therefore, we are able to induce almost any response on plant (e.g. flowering induction, climate conditions protection, organoleptic features enhancement). Essentially, we could easily personalize any desired element to provide the maximum possible control over the plant. </P><DIV id="light" class="heading-color jopesubapartado"><H4><SPAN>LIGHT APPLICATION SITE</SPAN></H4></DIV><P class="jopesangria">Since white light presents all visible wavelengths, controlling system activation/inactivation becomes almost impossible. We resolved this issue by plant root-specific promoter so optogenetic circuit can be activated when roots (which do not need light and are usually in dark conditions) are illuminated by red/far red light. This strategy guarantees almost a total root-specific regulated transcription, preventing interferences in circuit signaling and making our system more accurate, safe and efficient. </P><DIV id="transport" class="heading-color jopesubapartado"><H4><SPAN>TRANSPORT FROM ROOTS</SPAN></H4></DIV><P class="jopesangria">Since our genetic circuit will be activated in roots, proteins of the desired regulated element will be only synthesized there. Unfortunately, many proteins are not capable to transport themselves to other parts of the plant so, a systemic movement strategy along the plant becomes necessary to fulfil is metabolic role.</P><P class="jopesangria">One of the main strategy that is still being developed is the use of mRNAs as long-distance signaling molecules because of its ability to deliver a signal in its non-functional form (Spiegelman, Golan &amp; Wolf, 2013). Therefore, the translation of the desired element occurs specifically at the target site. Nevertheless, long-distance movement of transcripts precise much more control and to date, the complete understanding of the molecular machine underlying the mRNA trafficking has not clearly been reported to provide a definitive proof for this notion. For instance, it has been shown that protein CmPP16 from Cucurbita maxima possess properties similar to those of viral movement proteins. However it might not behave similar in other species (Xoconostle-Cázares, 1999). Equally with AtABCG14, critical for cytokinin translocation (Ko et al., 2014) </P><P class="jopesangria">Despite that, there are recent evidence of different molecules that can act as potential components of transport long-distance mRNAs. One is the presence of translocatable RNA-binding proteins (RBPs) in the phloem (Pallas &amp; Gómez, 2013) while another possibility is cell-penetrating peptides (CPPs) that transport hydrophilic macromolecules into cells. Here, application of CPPs for macromolecule delivery has been successfully demonstrated for plant cells even though further characterization is needed to determine which structural features influence its function (Chugh, Eudes &amp; Shim, 2010).</P><P class="jopesangria">Considering that mRNA transport approach is not the most suitable approach yet, we resolved to use plant viral vectors to amplify and transport any desired molecule. This strategy involves two main advantages in our system. First, the protein production is faster, and its yield is higher due to its ability to auto replicate itself. The second one is related to viral vector’s systemic movement ability. Thus, any desired protein can be transported from roots to the aerial part of the plant in an efficient way. These reasons encouraged us to work with viral vector strategy although we know that our main future challenge is understanding the mechanisms of mRNA import and transport in order to enhance and promote ChatterPlant’s possibilities.       </P><DIV id="why" class="heading-color jopesubapartado"><H4><SPAN>WHY AN ON-OFF SYSTEM?</SPAN></H4></DIV><P class="jopesangria">Constitutive expression of transgenes in plants often leads to pleiotropic undesirable phenotypes, limiting Plant SynBio applications in real world (Yi et al, 2010). Different transgenic plants show abnormal development (Romero et al, 1997), growth retardation or severe reduction in seed production (Liu et al, 1998) during their development. Accurate control of transgene expression allows to optimize development and behavior of transgenic plants without giving up improved features of the organism.</P><P class="jopesangria">An ON-OFF optogenetic circuit offers us the possibility to activate and deactivate transgene expression depending on plant status, avoiding interferences with plant development and decreasing metabolic charge in cells.</P><DIV class="heading-color"><H4 class="jopesubapartado2"><SPAN>BIOFORTIFICATION: AN OPTOGENETIC APPROACH</SPAN></H4></DIV><P class="jopesangria">Maintaining essential mineral micronutrients and vitamins levels of staple food crops allows to ensure access to affordable food for people with nutritional deficiencies. Biofortification born as a whole of techniques used to improve nutritional content of staple food crops through modern biotechnology, agronomic practices or conventional plant breeding. </P><P class="jopesangria">As an example, phytoene synthase (PSY) is involved in carotenoids biosynthesis. (Rodríguez-Villalón, Gas and Rodríguez-Concepción, 2009). Carotenoids are isoprenoids essential for plant life and human health, working as antioxidant molecules. Epidemiological studies correlates lack of carotenoid consumption with different diseases related with oxidative stresses and chronic disorders. Induced expression of PSY-encoding genes in transgenic plants results in increased carotenoid levels. However, constitutive expression of PSY-gene causes dwarfism in transgenic tomatoes, interfering in both behavior and development of the plant.</P></DIV><DIV id="section2" class="container"><DIV class="heading-color"><H2><SPAN>HOW CAN WE COMMUNICATE WITH PLANTS?</SPAN></H2></DIV><P class="jopesangria">In our work, we demonstrate that not only delivering orders into plants is feasible but also that plants are able to notify us of any change in their status. A leaf’s colour change apprises us whether plants are affected by any stress through our Colour Code System. Thus, we resolved to control biotic and abiotic stresses using a modular and orthogonal genetic AND gate.</P><DIV id="toggle" class="heading-color jopesubapartado"><H4><SPAN>TOGGLE SWITCH</SPAN></H4></DIV><P class="jopesangria">Genetic switches allow biological systems to process information of a determined stimulus in order to produce a desired biological response. To date, different synthetic and biostable genetic toggle switch designs with the ability to flip between stable states have been reported. As a cellular memory unit, the continued presence of the inductor is not required in order to keep the toggle in one state or another (Gardner, Cantor &amp; Collins, 2000).</P><P class="jopesangria">Here, we report a genetic toggle switch as an ideal system to allow an easy spatiotemporal manipulation of gene expression that is tuneable, reversible and repeatable. ΦC31 is a serine recombinase with the ability of performing a site-specific recombination between two attachment sites attP and attB (Keravala et al., 2006). After recognition process, site-specific recombinases cut off DNA strands (within recognition sequence) and perform a chain exchange with a subsequent religation to complete recombination event without addition or removal of nucleotides (Figure 2).</P><P class="jopesangria">For that reason, once ΦC31 mediates recombination between attP and attB, the recombinant sites (attL and attR) are produced. This reaction proceeds exclusively in a unidirectional manner because PhiC31 integrase is unable to synapse attL and attR sites per se. However, a recombination directionality factor (RDF) was identify recently to reverse the recombinase action and now, the activation or inactivation of any desired pathway could be triggered by the actuation of PhiC31 recombinase together with its RDF (Brophy and Voigt, 2014).</P><P class="jopesangria"><I class="jopecursiva">Figure 2. Graphic genetic design of PhiC31 actuation in plant-human genetic circuit. A) Register assembly construct composed by 35s promoter inverted sequence and Mtb terminator flanked by two opposing PhiC31 site-specific recombination. B) Dimers of ΦC31 bind to the attachment sites and each subunit binds to one half site. When recombinase is bound to specific pairs of sites (e.g. attP and attB sites), an integrase tetramer is assembled and a synaptic complex between both sites is constituted. C) Double-stranded DNA breaks are produced by the activated tetramer and recombinant product is produced.</I></P><P class="jopesangria">ChatterPlant proposes a modular and orthogonal AND gate in order to increase the genetic control of expressing any color protein. Leaf’s colour changes are triggered by the designed genetic toggle switch comprised by a stress-inducible promoter and a terminator in opposite directions flanked by ΦC31 attachment sites (attB and attP).</P><P class="jopesangria">In normal basis, ΦC31 recombinase is being expressed in the plant and the stress-inducible promoter is always inverted. In this case, no coding sequence is expressed, and the circuit is inactivated (Figure 3A).</P><P class="jopesangria">The RDF from ΦC31 - gp3 - was used to reverse the directionality of integrase actuation allowing the inducible promoter to return to the original state (Khaleel, Younger, McEwan, Varghese &amp; Smith, 2011). The use of gp3 customized in plants allows us to activate the biological system in order to produce a colour protein corresponding to a specific stress. Once RDF expression stops, the toggle switch return to the OFF state (Figure 3B).</P><P class="jopesangria"><I class="jopecursiva">Figure 3. Graphic representation of Plant-Human genetic circuit with the construct comprised by a promoter and a terminator in opposite directions flanked by ΦC31 attachment sites. A) In presence of PhiC31 integrase, recombination occurs allowing the expression of a translation of a nonsense sequence or stuffer fragment . In this case, the circuit is in OFF state B) Graphic representation of the toggle switch in ON state only when PhiC31 and gp3 are expressed. Only when the promoter is activated under stress conditions (e.g. Production of salicylic acid or Jasmonic acid), corresponding color protein will be expressed.</I></P><DIV id="gp3" class="heading-color jopesubapartado"><H4><SPAN>GP3 DELIVERY</SPAN></H4></DIV><P class="jopesangria">An optimal inducible system should provide elevated levels of continuous induction product together with no gene expression in absence of inducer. It has been reported several chemical-dependent gene activation systems such as steroids or copper. We proposed a spatial controlled gene expression using the dexamethasone inducible system because of the sensibility, efficiency and tightly regulation that can offer.</P><H4 class="jopesubapartado2"><SPAN>HOW DOES IT WORK?</SPAN></H4><P class="jopesangria">alcR is a positive regulatory gene from filamentous fungus Aspergillus nidulans which encodes a transcription factor that binds to alcA promoter. Therefore, alcR protein, under the influence of metabolized ethanol (acetaldehyde), can initiate the transcription process of gp3 by binding to specific sites on the alcA Promoter. alcA promoter is widely used to overexpress proteins since is one of the strongest inducible promoter in A.nidulans (Figure 4).</P><P class="jopesangria">The pOpLac/LhGR system comprises a chimeric promoter that consists of lac operators cloned up-stream of a minimal 35s promoter and a transcription factor LhGR which is a phusion between the rat glucocorticoid receptor (GR) to the chimeric construct assembled by a high-affinity DNA-binding mutant of lac repressor (LacIBD) and the transcription-activation-domain-II of GAL4 from Saccharomyces cerevisiae.</P><P class="jopesangria"><I class="jopecursiva">Figure 4. Graphic representation of the dexamethasone inducible system. a) Constitutive expression of the chimeric complex that consists of glucocorticoid-dependent transcription factor fused with the transcription activation domain II (Gal4AD) and the ligand binding domain (LacIBD). In the absence of the steroid ligand, the transcription factor is trapped in an inactive complex via interaction between the GR LBD. b) Once dexamethasone is metabolized, the binding of dexamethasone to the LBD mediates dissociation of the fusion protein.  </I></P><P class="jopesangria">Dexamethasone inducible system exhibits a tight regulation that responds to low concentrations of the steroids. Samalova and co-workers reported that induction occurs within one hour of dexamethasone application and 1nM is sufficient for half maximal induction (Samalova, Brzobohaty &amp; Moore, 2005). Since it has been reported that plants of Solanaceae family can be induced by this chemical inductor, a steroid injector should be incorporated in the hardware device. </P><P class="jopesangria">Combining control over climatic conditions and Synthetic Biology, ChatterPlant intends to increase the total control of any crop in each moment through distinct color protein expressions. Nevertheless, consumers wish to find fruits and vegetables with the best possible organoleptic properties. In this context, our project is based on sentinel plants. With the aim of controlling those conditions that humans can resolve (e.g. nutritional deficiency), we propose a diagonal system of sentinel plants in order to cover the maximum possible area (Figure 5). Sentinels plants will be able to warn us in order to modify some condition (eg. Nutritional solution improvement).</P><P class="jopesangria"><I class="jopecursiva">Figure 5. Graphic representation of a possible sentinel plants distribution.</I></P><P class="jopesangria">However, pathogen infections are the major cause of diseases in greenhouse crops . Being aware of its repercussions, our team proposes that all plants located in the ChatterBox incorporate the pathogen detection system to increase detection control. Bearing that in mind, we resolved to use a steroid injector in order to trigger color code genetic circuit in order to accelerate the needed corrective measures. </P><DIV id="and" class="heading-color jopesubapartado"><H4><SPAN>AND GATE</SPAN></H4></DIV><P class="jopesangria">Tightly regulated promoters are essential to construct more reliable and predictable genetic circuits for Synthetic Biology. Weakly expression of any interested gene is an undesired effect which is usually caused by promoter leakiness. We resolved to design an AND gate because it is a digital logic gate that use logical conjunction -two both inputs are required in order to produce a specific output-. In our work, only when stress signals appear, and RDF is expressed by external stimuli, colour protein is synthesized, and we will be able to detect it.</P><P class="jopesangria">Furthermore, the recombinase logic gate (without gp3) operates as a memory circuit and the input promoters do not have to be turned on at the same time for the output to respond. For instance, for the AND gate the inputs can be ON at different times and never simultaneously and the output will still be ON (Fernandez-Rodriguez, Yang, Gorochowski, Gordon &amp; Voigt, 2015). To surmount this issue, Endy and co-workers suggested that rewritable logic could be designed by continuously expressing PhiC31 recombinase to hold one state and then an external input is connected to the expression of gp3 in order to produce color protein. </P><P class="jopesangria">Spatial and/or developmental controlled gene expression of the color protein can be achieved using a recombinase logic AND gate.</P><DIV class="heading-color jopesubapartado"><H4><SPAN>VIRAL VECTOR STRATEGY</SPAN></H4></DIV><P class="jopesangria">Gene viral delivery arises as a novel tool due to its high protein expression and yield production. In the last years, it has been reported several virus-derived systems since they can provide us with multiples advantages, for instance, its auto-replicative capacity and its movement from cell-to-cell (through plasmodesmata) to the vascular tissue. Thus, we decided to develop our proof of concept from this approach using auto replicative vectors as a vehicle for the colour protein expression.</P><P class="jopesangria">In order to approach this, we used a viral vector derived from Tobacco etch virus (TEV) due to the amount of extra genetic material that they can accommodate and the ability to express a large viral polyprotein by viral-encoded proteases (Majer et al. 2017). The Antirrhinum majus transcription factor, Rosea1 was used as a visual reporter gene. Viral vector strategy enables us to detect adequately color halo effect.</P><P class="jopesangria">Majer and co-workers (Majer et al. 2017) described several proteins from carotenoid pathway that display an alternative output colours giving us the possibility to design our Colour Code System. Furthermore, anthocyanin pigments are quantitative visual marker due to the correlation with the viral load making these proteins suitable for our project.</P><P class="jopesangria">However, viral vector strategy is not the most convenient one for this project because of its infection process and the superinfection exclusion phenomenon. Once viral vector is expressed, the production of a colour protein is permanent. Moreover, once the viral vector infects the plant, superinfection exclusion does not allow a secondary infection (Julve et al., 2013). For this reason, we will use them only as a vehicle for expressing the colour protein as a proof of concept.</P><DIV id="chromoproteins" class="heading-color jopesubapartado"><H4><SPAN>CHROMOPROTEINS</SPAN></H4></DIV><P class="jopesangria">A repository of ten chromoproteins have been characterized, codon-optimized and standardized in E.coli by Uppsala iGEM team between 2011 and 2013. Chromoproteins can be used as bioreporters since they exhibit a striking color when they are expressed. Thus, several teams have implemented them in their projects such as Wageningen UR in filamentous fungi or NRP-UEA-Norwich 2014 and Cambridge-JIC 2014 using plant chassis. However, they cannot see a strong color expression in plants.</P><P class="jopesangria">In order to expand our Color Code System, we propose using chromoproteins since their absorbance spectrum is in the visible one and non-special equipment is necessary. Amil CP, AmajLime and eforRed were codon-optimized for Nicotiana benthamiana because of other teams have tried to demonstrate that they work. In addition, we intend to endow iGEM teams with renewed functional reported proteins with the aim of finding new purposes for them in plant chassis. </P></DIV><DIV id="section3" class="container"><DIV class="heading-color"><H2><SPAN>BIBLIOGRAPHY</SPAN></H2></DIV><P class="jopesangria">Brophy, J., &amp; Voigt, C. (2014). Principles of genetic circuit design. Nature Methods, 11(5), 508-520. http://dx.doi.org/10.1038/nmeth.2926</P><P class="jopesangria">Chugh, A., Eudes, F., &amp; Shim, Y. (2010). Cell-penetrating peptides: Nanocarrier for macromolecule delivery in living cells. IUBMB Life, 62(3), 183-193. http://dx.doi.org/10.1002/iub.297</P><P class="jopesangria">Fernandez-Rodriguez, J., Yang, L., Gorochowski, T., Gordon, D., &amp; Voigt, C. (2015). Memory and Combinatorial Logic Based on DNA Inversions: Dynamics and Evolutionary Stability. ACS Synthetic Biology, 4(12), 1361-1372. http://dx.doi.org/10.1021/acssynbio.5b00170</P><P class="jopesangria">Julve, J., Gandía, A., Fernández-del-Carmen, A., Sarrion-Perdigones, A., Castelijns, B., Granell, A., &amp; Orzaez, D. (2013). A coat-independent superinfection exclusion rapidly imposed in Nicotiana benthamiana cells by tobacco mosaic virus is not prevented by depletion of the movement protein. Plant Molecular Biology, 81(6), 553-564. http://dx.doi.org/10.1007/s11103-013-0028-1</P><P class="jopesangria">Keravala, A., Groth, A., Jarrahian, S., Thyagarajan, B., Hoyt, J., Kirby, P., &amp; Calos, M. (2006). A diversity of serine phage integrases mediate site-specific recombination in mammalian cells. Molecular Genetics And Genomics, 276(2), 135-146. http://dx.doi.org/10.1007/s00438-006-0129-5</P><P class="jopesangria">Khanna, R. (2004). A Novel Molecular Recognition Motif Necessary for Targeting Photoactivated Phytochrome Signaling to Specific Basic Helix-Loop-Helix Transcription Factors. THE PLANT CELL ONLINE, 16(11), 3033-3044. http://dx.doi.org/10.1105/tpc.104.025643</P><P class="jopesangria">Khaleel, T., Younger, E., McEwan, A., Varghese, A., &amp; Smith, M. (2011). A phage protein that binds φC31 integrase to switch its directionality. Molecular Microbiology, 80(6), 1450-1463. http://dx.doi.org/10.1111/j.1365-2958.2011.07696.x</P><P class="jopesangria">Ko, D., Kang, J., Kiba, T., Park, J., Kojima, M., &amp; Do, J. et al. (2014). Arabidopsis ABCG14 is essential for the root-to-shoot translocation of cytokinin. Proceedings Of The National Academy Of Sciences, 111(19), 7150-7155. http://dx.doi.org/10.1073/pnas.1321519111</P><P class="jopesangria">Majer, E., Llorente, B., Rodríguez-Concepción, M., &amp; Daròs, J. (2017). Rewiring carotenoid biosynthesis in plants using a viral vector. Scientific Reports, 7, 41645. http://dx.doi.org/10.1038/srep41645</P><P class="jopesangria">Müller, K., Naumann, S., Weber, W., &amp; Zurbriggen, M. (2015). Optogenetics for gene expression in mammalian cells. Biological Chemistry, 396(2). http://dx.doi.org/10.1515/hsz-2014-0199</P><P class="jopesangria">Ni, M., Tepperman, J., &amp; Quail, P. (1999). Binding of phytochrome B to its nuclear signalling partner PIF3 is reversibly induced by light. Nature, 400(6746), 781-784. http://dx.doi.org/10.1038/23500</P><P class="jopesangria">Notaguchi, M., Daimon, Y., Abe, M., &amp; Araki, T. (2009). Graft-transmissible action of Arabidopsis FLOWERING LOCUS T protein to promote flowering. Plant Signaling &amp; Behavior, 4(2), 123-125. http://dx.doi.org/10.4161/psb.4.2.7558</P><P class="jopesangria">Liu Q, Kasuga M, Sakuma Y, Abe H, Miura S, Yamaguchi-Shinozaki K, Shinozaki K (1998) Two transcription factors, DREB1 and DREB2, with an EREBP/AP2 DNA binding domain separate two cellular signal transduction pathways in drought- and low-temperature-responsive gene expression, respectively, in Arabidopsis.Plant Cell 10:1391–1406 http://dx.doi.org/10.2307/3870648</P><P class="jopesangria">Pallas, V., &amp; Gómez, G. (2013). Phloem RNA-binding proteins as potential components of the long-distance RNA transport system. Frontiers In Plant Science, 4. http://dx.doi.org/10.3389/fpls.2013.00130</P><P class="jopesangria">Rodríguez-Villalón, A., Gas, E. and Rodríguez-Concepción, M. (2009). Phytoene synthase activity controls the biosynthesis of carotenoids and the supply of their metabolic precursors in dark-grown Arabidopsis seedlings. The Plant Journal, 60(3), pp.424-435. http://dx.doi.org/10.1111/j.1365-313x.2009.03966.x</P><P class="jopesangria">Romero C, Bellés JM, Vayá JL, Serrano R, Culiáñez-Macià FA (1997) Expression of the yeast trehalose-6-phosphate synthase gene in transgenic tobacco plants: pleiotropic phenotypes include drought tolerance. Planta 201:293–297 http://dx.doi.org/10.1007/s004250050069</P><P class="jopesangria">Samalova, M., Brzobohaty, B., &amp; Moore, I. (2005). pOp6/LhGR: a stringently regulated and highly responsive dexamethasone-inducible gene expression system for tobacco. The Plant Journal, 41(6), 919-935. http://dx.doi.org/10.1111/j.1365-313x.2005.02341.x</P><P class="jopesangria">Spiegelman, Z., Golan, G., &amp; Wolf, S. (2013). Don’t kill the messenger: Long-distance trafficking of mRNA molecules. Plant Science, 213, 1-8. http://dx.doi.org/10.1016/j.plantsci.2013.08.011</P><P class="jopesangria">Xoconostle-Cázares, B. (1999). Plant Paralog to Viral Movement Protein That Potentiates Transport of mRNA into the Phloem. Science, 283(5398), 94-98. http://dx.doi.org/10.1126/science.283.5398.94</P><P class="jopesangria">Yi, N., Kim, Y., Jeong, M., Oh, S., Jeong, J., Park, S., Jung, H., Choi, Y. and Kim, J. (2010). Functional analysis of six drought-inducible promoters in transgenic rice plants throughout all stages of plant growth. Planta, 232(3), pp.743-754. http://dx.doi.org/10.1007/s00425-010-1212-z</P></DIV></DIV></DIV></DIV></SECTION><FOOTER id="footer"><DIV class="container" style="padding: 0px !important; margin: 0px !important; width: 100%;"><DIV class="row" style="margin: 10px;"><DIV class="col-md-8 jopefooter"><P style="margin: 0px !important; text-align: center !important;"><STRONG class="jopenegrita">Valencia UPV iGEM</STRONG> Team</P></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>