Team:Washington/gRNA Expression
Washington iGEM
Main Project
Background
Our project was inspired by the work of the Koffas lab¹. Researchers developed a  repressible promoter system using sgRNA sequences coupled with the CRISPR/dCas9 mechanism. They incorporated these promoters into the biosynthetic violacein pathway and were able to successfully throttle carbon flux in E. coli, effectively turning promoters into orthogonal on/off switches commanding the direction of metabolic production in bacteria. We realized the boundless implications of this method and thus decided to incorporate this construct into our yeast system.
In order to realize our goal of creating an autonomous yeast culture management platform, we are employing a violacein pathway to visualize metabolic processes in yeast. The pathway consists of five genes: VioA, VioB, and VioE (which are constitutively expressed), VioC and VioD. A single promoter determines expression of VioA, VioB, and VioE, so in our diagrams, the three genes are abbreviated to VioABE. The genes are controlled by inducible promoters. The Dueber Lab performed HPLC on yeast that expressed this pathway to reveal that primarily predeoxyvioacein, deoxyviolacein, proviolacein, and violacein were produced in mass quantities⁴. Culture pigment expression changes according to the following gene activation combinations:
VioABE → prodeoxyviolacein
VioABE + VioC → deoxyviolacein
VioABE + VioD → proviolacein
VioABE + VioC + VioD → violacein
Figure: Violacein Pathway
Methods
The regulation of our system has several steps:
sgRNAs that target either VioC or VioD are transcribed depending on how they are expressed. In our design this is based on Cup1 and modGal1 inducible promoters.
The sgRNAs form a complex with dCas9.
The large complex will bind to either VioC or VioD and prevent the transcription of these genes into mRNA.
While VioABE are always transcribed and translated to form prodeoxyviolcein, VioC and VioD expression varies based upon the presence of the sgRNA/dCas9 complexes.
The alternate forms of violacein vary in color. The observed color of a liquid culture of the transgenic yeast changes based on the composition of the forms of violacein produced. Achieving changes in color is possible by changing the expression of VioC and VioD.
The sgRNAs were uniquely designed in yeast to both accurately target VioC and VioD and to contain ribozymes to keep the genes within the nucleus. More specifically, ribozymes were placed upstream and downstream of the sgRNA sequences to “cut” off the 5’ cap and Poly-A tail attached during mRNA processing that marks RNA to leave the nucleus.²
Additionally, we designed the sgRNA plasmid with inducible promoters, modGal1 and CUP1. ModGal1 is coupled with a zinc finger transcription factor that is induced by the presence of beta-estradiol. We included this transcription factor to be constitutively expressed in the same plasmid. CUP1 is a promoter whose transcription factor is induced by the presence of copper to express the VioD sgRNA. Its transcription factor ACE is naturally expressed in the yeast genome.
The sgRNA plasmid was designed to be constructed through Gibson assembly. The insert with the sgRNAs would be ordered, and then assembled with a linearized pRS425 as a vector. This vector contains ampicillin resistance for E. coli selection and leucine for yeast selection.
Figure: Digest
Problems arose when we first attempted to order the designed insert. Our modified GAL1 promoter with its zinc finger binding site had too many sequence repeats. The team then attempted to use a restriction enzyme (BaeI) to linearize the plasmid. This linearized plasmid would be assembled with a second geneblock. The plasmid carrying the modified GAL1 promoter was never able to be linearized. As a result, we had to switch this promoter for ZAP1. Additionally, the designed insert was too many base pairs to order in one piece at 5kb. This was solved by ordering two inserts with homology and first attaching them together through Gibson assembly. Another Gibson assembly was performed to complete the plasmid, and finally, the plasmid was transformed into DH5-alpha E. coli.
Figure: Gibson Assembly Part 1
Figure: Gibson Assembly Part 2
We integrated the Violacein pathway,pWCD1133, and pdCas9/Mxi1, pMOD4-CYC1-dCas9-mxi1, into the yeast genome at the URA3 and TRP1 loci respectively.  The inducible sgRNAs responsible for guiding the dCas9/Mxi1 complex to its target site were transformed in a non-integrating plasmid.  Collectively, this three-plasmid system is intended to create a strain capable of producing all four colors to be analyzed by the Chromastat.
Figure: Transformation
Results
E. coli
The first half of our project was to design an E. coli inducible small guide RNA (sgRNA) system similar to that found in the paper, Rapid generation of CRISPR/dCas9-regulated, orthogonally repressible hybrid T7-lac promoters for modular, tuneable control of metabolic pathway fluxes in Escherichia coli. The purpose was to get the system up and running to provide initial color data for calibration. However, we were unsuccessful in our attempt.
First attempt of transforming gibson assembled VioABE, sgRNA and dCas9 plasmid. From left to right 3A2 sgRNA insert and 4A6 sgRNA insert. Pink colonies indicative of mCherry present in plasmid.
We first attempted to insert the VioABE insert into a pSB1K3 backbone, and we were successful. Then, we amplified more of the backbone pSB1K3-VioABE and sgRNA insert (3A2 and 4A6) plasmids. We were able to Gibson assemble these two together components and dCas9 together. From initial attempts of growing up these plasmids in DH5-alpha E. coli competent cells, we were unsuccessful because our colonies were pink. This was indicative of the mCherry construct being present, a region of the plasmid we were trying to remove.
We eventually had white colonies, as shown below:
Transformed gibson assembled VioABE, sgRNA and dCas9 plasmid. From left to right 3A2 sgRNA insert and 4A6 sgRNA insert.
After this success we wanted to test if we were able to integrate the dCas9, backbone, and sgRNA into one plasmid. We ran a digest using HindIII restriction and enzyme and ran a diagnostic gel.
Gel to determine if plasmids were integrated. From left to right, NEB 2Log Ladder, VioABE+dCas9+3A2 (A) [0.1x], VioABE+dCas9+3A2 (B) [0.1x], VioABE+dCas9+3A2+4A6 (A) #2, VioABE+dCas9+4A6 (A) [0.1x], VioABE+dCas9+4A6 (B) [0.1x], remaining lanes were 10 uL of DI water and 2 uL of 6x Loading dye.
We then decided to do a two step transformation of BL21(DE3) competent E. coli cells. The first transformation was of just the VioABE into the component cells. Then, the sgRNA and pdCas9 plasmids were transformed once the cells were made competent again. They were successfully transformed on a plate but we had difficulty growing them in a liquid culture that would have them express colors in the presence of IPTG.
After talking to Brady Cress and referencing his publication, we were able to design color optimization experiments. We determined that our dCas9 component was constitutively repressed and needed aTc to induce activity. Our first objective was to get an optimized protocol for getting purple with the violacein, pdCas9, sgRNA plasmids (3A2 and 4A6). We performed an overnight outgrowth of our liquid cultures at 37 °C and induced them with IPTG at varying times. We also moved them to a shaker at a lower temperature of 20 °C. After letting them grow overnight again, we removed the tubes and found that they grew but did not change color.
After IPTG induction and an additional overnight shaking, A: 3A2 sgRNA system liquid culture B:4A6 sgRNA system liquid culture.
We left these tubes in the shaker for two days and they turned purple.
After an additional two days of shaking, A: 3A2 sgRNA system liquid culture. B:4A6 sgRNA system liquid culture.
Unfortunately, due to time constraints and supply ordering difficulties, we were unable to continue this part of the project.
Yeast:
We encountered many problems in our yeast inducible sgRNA system. The two major roadblocks were that nuclear exportation occurred when the Pol II promoters modified transcripts, and we also could not reliably perform PCR over large regions containing many homologies.
For us to tackle these issues we went on to do more research. We looked through literature with themes similar to our project and made inquiries with graduate students and advisors. Through our search we were able to come to these conclusions. Self-cleaving ribozymes, such as Hammerhead and HDV, have been demonstrated to be able to decrease nuclear exportation of sgRNA². This particular design component is called ribozyme-guide RNA ribozyme or RGR. We were also able to find that with the inclusion of unique terminators, PCR amplification can be performed on homology heavy regions³. From our very own university, the UW DAWGMA team was successful in initiating transcription with an induced CUP1 promoter using copper solutions. Additionally, after meeting with an advisor, we were give permission to use a plasmid containing a modified Gal1 promoter and a fused transcription factor. The latter was successful in activating the promoter. The transcription factor included three distinct sections fused together; they include a zinc finger, human estrogen receptor, and Vp16.
In our first iteration, we initially planned to order two RGRs on a geneblock for the Bae1 digested plasmid. However, once the geneblock and the enzyme were ordered, we realized that Bae1 was not able to cut the plasmid. We came to this conclusion when we observed that after numerous attempts to grow a glycerol stock of the E. coli that had our plasmid of interest, it did not propagate. The E. coli also appeared to not be resistant to Ampicillin, our selection marker.
Given these results, we decided to redesign our system to include either a ZAP1 and CUP1 or ICL1 and CUP1 promoter pair. These would be ordered on geneblocks and then assembled to create cassettes with homology to the pRS425 vector. We used Gibson and NEBuilder assembly to assemble the plasmid. We also attempted to insert a well known primer and PST1 cut sites to the cassette in tangent with the assemblies as a backup plan. This allowed for the amplification of the assembled cassette and the additional restriction enzyme cut site allowed for T4 ligation.
We attempted to transform the Gibson and NEBuilder assemblies into DH5-alpha E. coli and colonies grew.
PCR colonies for assembly products. From left to right: pRS425-ICL1 Gibson assembly, pRS425-ZAP1 Gibson assembly, pRS425-ICL1 NEBuilder assembly, and pRS425-ZAP1 NEBuilder assembly.
We took colonies to do colony PCR and plated duplicates on new plates. However, colony PCR as well as sequencing verifications failed, indicating that these were false colonies. This was likely caused by the non linearized vector plasmid, pRS425. We have repeated our assemblies and are currently transforming them again.
Citations
1. Gander MW, Vrana JD, Voje WE, Carothers JM, Klavins E. Digital logic circuits in yeast with CRISPR-dCas9 NOR gates. Nat Commun. 2017;8:15459. doi:10.1038/ncomms15459.
2. Cress BF, Jones JA, Kim DC, et al. Rapid generation of CRISPR/dCas9-regulated, orthogonally repressible hybrid T7-lac promoters for modular, tuneable control of metabolic pathway fluxes in Escherichia coli. Nucleic Acids Res. 2016;44(9):4472-4485. doi:10.1093/nar/gkw231.
3. Curran KA, Karim AS, Gupta A, Alper HS. Use of expression-enhancing terminators in Saccharomyces cerevisiae to increase mRNA half-life and improve gene expression control for metabolic engineering applications. Metab Eng. 2013;19:88-97. doi:10.1016/j.ymben.2013.07.001.
4. Lee ME, Aswani A, Han AS, Tomlin CJ, Dueber JE. Expression-level optimization of a multi-enzyme pathway in the absence of a high-throughput assay. Nucleic Acids Res. 2013;41(22):10668-10678. doi:10.1093/nar/gkt809.
