<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Washington skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Washington</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Washington iGEM</TITLE></P><DIV class="jumbotron jumbotron-fluid mainBodypic vertical-center" id="backgroundImage" style="margin-bottom: 0px; min-width: 200px"><DIV class="container-fluid mainBody"><H1 class="display-1 fadeOnLoad" id="projectTitle" style="font-size: 12vh; margin: 0px; padding: 0px">Viva la Violacein:</H1><H2 class="display-4 fadeOnLoad" style="font-size: 7vh; margin: 0px; padding: 0px">a Real-Time Metabolics Tracker</H2><BUTTON type="button" class="fadeOnLoad hvr-sweep-to-right abelFont"><A id="linkToDescription" href="#overview">Overview</A></BUTTON></DIV></DIV><DIV class="jump" id="overview"><DIV class="container-fluid"><DIV class="row"><DIV class="container-fluid"><H4 class="display-4 centered">Project Overview<DIV class="container centered"><P>Synthetic biology can be used to create new, cost-effective, metabolites resulting from metabolic pathways. However, managing cultures containing these metabolic pathways is difficult and time-consuming. Constantly measuring and adjusting culture conditions in order to produce a desired metabolite in a specific quantity is both tedious and labor intensive. Furthermore, modern assays that accomplish this, such as high precision liquid chromatography, can also be prohibitively expensive.</P><P>Our project aims to reduce the amount of time and effort needed to maintain cultures through real-time, automated analysis of metabolic products in an adjustable turbidostat. This includes the creation of an affordable image analysis system that reads visual data to measure the current state of a culture and then provide feedback to release inducers to alter the expression of the metabolic pathway.</P><P>Our project utilizes the violacein pathway as a pigment-based proxy to predict the production of other metabolic pathways. By regulating gene expression within the Violacein gene set with two different inducible promoters, we are able to yield up to four different color outputs. These outputs are measured by an open-sourced Raspberry Pi setup, which captures visual data, calculates feedback based on the culture’s RGB value, and then directs the gradual release of inducer chemicals to maintain or change the culture’s color over time. Therefore, this process allows us to better understand the relationship between gene expression and actual metabolite production rate.</P><P>Currently, yeast strains capable of coexpressing both a violacein and a non-visible pathway are being cloned. In addition to this, a machine comprised of 3D-printed syringe pumps, a turbidostat, and image analysis software is in development. By combining biological, software, and hardware systems, we expect our unique design to be able to generate previously unavailable visual data in certain biosynthesis processes, such as those involved in antibiotic production or fermentation.</P><P style="text-align: center">Click on a circle to see more information!</P></DIV><DIV class="container-fluid" style="min-width:10px; padding: 20px"><DIV class="row justify-content-md-center"><DIV class="col-md-auto" style="text-align: center"><P style="text-align: center"><B>Wet Lab</B></P><DIV class="container" style="max-width: 250px">Our wetlab team was responsible for the molecular biology side of our project. They designed and implemented a system for regulation of the violacein pathway in yeast. They also created BioBricks and completed the InterLab measurements.</DIV></DIV><DIV class="col-md-auto" style="text-align: center"><P style="text-align: center"><B>Dry Lab</B></P><DIV class="container" style="max-width: 250px">Our dry lab team built a low-cost modular device with a closed loop control system implemented by a Raspberry Pi to manipulate and analyze the products of cell cultures in real time.</DIV></DIV><DIV class="col-md-auto" style="text-align: center"><P style="text-align: center"><B>Human Practices</B></P><DIV class="container" style="max-width: 250px">This year, we had the largest team in the history of iGEM at the University of Washington, with a consistent 34 undergraduates working on various aspects of the project. For the first time, this allowed us to seriously incorporate hardware and software development as well as wetlab, and allowed us to do more engagement and human practices work than in previous years.</DIV></DIV><DIV class="col-md-auto" style="text-align: center"><P style="text-align: center"><B>Team</B></P><DIV class="container" style="max-width: 250px">Meet the team!</DIV></DIV></DIV></DIV></H4></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>