<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Paris_Saclay_Modeling_Fusion_Proteine"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Paris_Saclay/Modeling/Fusion_Proteine">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Paris_Saclay/Modeling/Fusion_Proteine&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Saclay/Modeling/Fusion_Proteine&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Saclay/Modeling/Fusion_Proteine&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Paris_Saclay/Modeling/Fusion_Proteine" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Paris Saclay/Modeling/Fusion Proteine</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="menups" class="menups"><UL><LI><A href="https://2014.igem.org/Team:Paris_Saclay">Home</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Team">Team</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Team">Presentation</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=Paris_Saclay">Official Profile</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Team/Sponsor">Sponsors</A></LI></UL><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Project/Inspirations">Inspirations</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Project/Fastlemon">Fastlemon</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Project/Odor-free_ecoli">Remove the bad smell of E.coli</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Project/Lemon_Scent">Lemon Scent</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Project/Salicylate_Inducible_System">Lemon Ripening</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Project/Lemon_Shaping">Lemon Shaping</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Project/Boston_Installation">Boston Installation</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Modeling">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Modeling/oxygen_diffusion">Oxygen Diffusion</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Modeling/bacterial_Growth">Bacterial Growth</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Modeling/Fusion_Proteine">Fusion Protein</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Modeling/odor">Odor</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Labwork">Labwork</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Ethics">Ethics</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Ethics/About_Life_Art_Science">Philosophical and Historical Aspects</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Ethics/Sociological_Cultural_Definition_Living_Being">Sociological and Cultural Aspects</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Ethics/Scientific Definition of Living-Being">Scientific Aspects</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Ethics/Interviews">Expert's Opinions</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Ethics/Round_Table">French iGEMers Discussion</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Ethics/Survey">International iGEMers Point-of-View</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Ethics/Reflection_about_Artificial_Food">Reflection about Artificial Food</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Outreach">Outreach</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Project/Workshop">French Meetup</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Outreach/Events/Curiositas">Curiositas</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Outreach/Events/Science_Festival">Science Festival</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Outreach/Collaborations">Collaborations</A></LI></DIV><DIV id="pageContent"><DIV id="submenups" class="submenups"><UL><LI>Modeling</LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Modeling">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Modeling/oxygen_diffusion">Oxygen Diffusion</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Modeling/bacterial_Growth">Bacterial Growth</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Modeling/Fusion_Proteine">Fusion Protein</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Saclay/Modeling/odor">Odor</A></LI></UL></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Fusion_Protein"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Fusion Protein</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#The_first_part_.E2.80.93_Yellow_chromoprotein:"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">The first part – Yellow chromoprotein:</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#The_second_part_.E2.80.93_blue_chromoprotein:"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">The second part – blue chromoprotein:</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#The_last_part_.E2.80.93_Global_modelisation:"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">The last part – Global modelisation:</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#References"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Fusion_Protein">Fusion Protein</SPAN></H1><P>Here we will discuss the structural modeling of the fusion protein that we had planned in order to obtain a <A href="https://2014.igem.org/Team:Paris_Saclay/Project/Salicylate_Inducible_System" class="external text" rel="nofollow">green chromoprotein</A>. We used the bioinformatics tools developed by the Swiss Institute of Bioinformatics (Swiss-Model <B>[1]</B>) and by the Xu group of Chicago University (RaptorX <B>[2]</B>).
</P><P>Swiss-Model has given us three highly probable models. Two models for the first part of our Fusion Protein and another for second part of our protein. We chose the first model for the yellow chromoprotein (1st part) and the second model for the blue chromoprotein (2nde part).
</P><H2><SPAN class="mw-headline" id="The_first_part_.E2.80.93_Yellow_chromoprotein:">The first part – Yellow chromoprotein:</SPAN></H2><P>The yellow chromoprotein used to create this fusion protein is nammed the fwYellow, a yellow chromoprotein (<A href="http://parts.igem.org/Part:BBa_K1033910" class="external text" rel="nofollow">BBa_K1033910</A> developed by iGEM13_Uppsala).
Homology modeling via Swiss-Model for the first chromoprotein gives us a nearly exact match with amFP486 which is a cyan fluorescent protein from <I>Anemonia majano</I> (the same organism as the Uppsala's biobrick). Only 3 amino-acids change between the sequence of the two chromoproteins. These amino-acids are in the core of the protein. We can therefore hypothesize that at least three positions can be modified to change the emission spectrum of the chromoprotein.
</P><P>Note that this chromoprotein is a homotetramer. What might be able to raise problems is if homotetramerisation is essential to the functionality of the protein. The literature <B>[3]</B> tells us that it does not appear to be the case: fusion proteins have already been created with chromoproteins, which were homo-multimeric and the protein fusions were completely functional.
</P><P>We predict that in our project, an homotetrameric state is not important to the functionality of our fusion protein. It is believed that the light emission will be less.
</P><H2><SPAN class="mw-headline" id="The_second_part_.E2.80.93_blue_chromoprotein:">The second part – blue chromoprotein:</SPAN></H2><P>We can do the same analysis for the blue chromoprotein.
For the second part of the fusion protein we use to the aeBlue which is a blue chromoprotein (<A href="http://parts.igem.org/Part:BBa_K1033929" class="external text" rel="nofollow">Bba_K1033929</A> developed by iGEM13_Uppsala).
Swiss-Model gives us a model for the second part: a near perfect match between our protein sequence and chromoprotein (far-red fluorescent protein AQ143). There is a 95.59% similarity between the two sequences, the major differences are present at the C-terminus and N-terminus. As for the Yellow chromoprotein, three amino acids of the core protein are not the same.
</P><P>Note that this chromoprotein is also a homotetramer.
</P><P>Modeling with Swiss-Model does not provide information on the overall structure of our chromoprotein. Nonetheless, it gives us a very good approach to the structure of the two chromoproteins which make up our fusion protein.
</P><H2><SPAN class="mw-headline" id="The_last_part_.E2.80.93_Global_modelisation:">The last part – Global modelisation:</SPAN></H2><P>We could make a much more complex modelisation for the overall structure of the fusion protein, we think it is unnecessary. Therefore, to have a global vision of our protein we will use the RaptorX model.
</P><P>Once this new model was obtained, we performed structural alignments and we discovered the structure of the two chromoproteins, so it seems that the linker between the two proteins is not sufficient to induce stress on the structures of our fusion protein.
</P><P>Obviously many biases have been introduced during this modeling and the conception of the fusion protein. This modelisation is closer to a structural analysis that a complex protein modeling.
The assumptions used in the design of this chromoprotein (fusion protein) does not seem to be so far from reality. We found similar cases in scientific publications, which are perfectly functional.
We suggest several hypothesis to explain the failure of our experience. The main hypothesis is an error in the design of the nucleotide sequence that we requested to synthesize
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P><B>[1]</B><A href="http://swissmodel.expasy.org/" class="external text" rel="nofollow">Swiss-Model server</A></P><P><B>[2]</B> Protein Structure and Function Prediction, <A href="http://raptorx.uchicago.edu/" class="external text" rel="nofollow">RaptorX server</A>, Chicago university
</P><P><B>[3]</B><A href="http://nar.oxfordjournals.org/content/33/5/e49.full.pdf+html" class="external text" rel="nofollow">A dual-fluorescence reporter system for high-throughput clone characterization and selection by cell sorting</A> - Juno Choe, Haiwei H. Guo and Ger van den Engh, <I>Nucleic Acids Research</I>, 2005, Vol. 33, No. 5 
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Paris_Saclay/Modeling/Fusion_Proteine">http://2014.igem.org/Team:Paris_Saclay/Modeling/Fusion_Proteine</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Paris_Saclay/Modeling/Fusion_Proteine" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Paris_Saclay/Modeling/Fusion_Proteine" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Paris_Saclay/Modeling/Fusion_Proteine&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Paris_Saclay/Modeling/Fusion_Proteine&amp;oldid=259729" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>