"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Illinois-Tools/Resources
From 2009.igem.org
Home
The Team
The Project
Demo
Modeling
Safety
Resources
Notebook
As a team who relied mostly on technology, we have a unique list of resources that we either used or investigated.  The resources that we looked at can be divided into three categories: Software, Journal Articles, and Books
Software
This category includes all programs, libraries, modules, websites, and anything else involving computer technology.
Programming Language
For our web-based program, we used Python 2.5/2.6 programming language in conjunction with the web framework, Django 1.0.2, and MySQL, a relational database management system (RDBMS).
Modules/Libraries
Biopython
Networkx
SciPy
SoapPy (for KEGG API access)
Pythongraph
Pygraphviz (to work with graphviz)
Pyexcelerator
Pyopengl
Matplotlib
Websites
Kyoto Encyclopedia of Genes and Genomes (KEGG; http://www.kegg.jp ; our main database that we obtained info from)
Network Analysis Tools (NeAT; http://rsat.ulb.ac.be/rsat/index_neat.html )
Programs/Other
Cytoscape ( http://www.cytoscape.org/ )
Eclipse
Graphviz
Matlab
COBRA toolbox (for Matlab)
Journal Articles
Becker, Scott; et al.Quantitative prediction of cellular metabolism with constraint-based models: the COBRA Toolbox.Department of Bioengineering, University of California San Diego.Nature Protocols Vol.2 No.3. March 2007.
Briefly describes constraint based modeling and give the principles behind it
Describes the functions of the COBRA toolbox and it explains the output of the program
Price, Nathan; et al. Genome-Scale Models of Microbial Cells: Evaluating the Consequences of Constraints.Department of Bioengineering, University of California San Diego. Nature Reviews. Vol.2. November 2004.
Describes the constraints one must take into account when creating genome scale models
Explains each constraint in detail
Chih-Hung Chou; et al. FMM: a web server for metabolic pathway reconstruction and comparative analysis. Institute of Molecular Medicine and Bioengineering, Institute of Bioinformatics and Systems Biology and Department of Biological Science and Technology, National Chiao Tung University, Hsin-Chu 300, Taiwan, Republic of China. Nucleic Acids Research, 2009, Vol. 37. April 2009.
Papin, J.A.; et al. Metabolic pathways in the post-genome era. Department of Bioengineering, University of California, San Diego, La Jolla 92093-0412, USA. Trends Biochem Sci. 2003 May;28(5):250-8.
Hintze A, Adami C, 2008 Evolution of Complex Modular Biological Networks. PLoS Comput Biol 4(2): e23. doi:10.1371/journal.pcbi.0040023
Books
Lutz, Mark. Learning Python; 2nd ed. Sebastopol, CA: O'Reilly, 2004. Print.
Python cookbook; 2nd ed. Beijing: O'Reilly, 2005. Print.
Minieka, Edward. Optimization algorithms for networks and graphs. New York: M. Dekker, 1978. Print.
Retrieved from " http://2009.igem.org/Team:Illinois-Tools/Resources "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
