"
Page
Discussion
View source
History
teams
Log in
 
Team:UESTC-China/pad Design
From 2014.igem.org
UESTC-China
Home
Team
Members
Attribution
Notebook
Project
Overview
Design
Result
Future work
Modeling
Overview
HPS/PHI
FALDH/FDH
AtAHA2
Parts
BioBrick
Protocol
Human Practice
Knowledge quiz
Lecture
Communication
Questionnaire
Safety
Safety
In this project, our objective is to further increase plant formaldehyde uptake and metabolism ability using synthetic biology methods. We find some genes encoding key enzymes related to formaldehyde metabolic pathways from microorganism. They are 3-hexulose-6-phosphate(HPS), 6-phospho-3-hexuloisomerase(PHI), formaldehyde dehydrogenase(FALDH) and formate-dehydrogenase(FDH). These genes are inserted into plants and will work to promote formaldehyde metabolism. For security reasons, we also add gene AdCP into our project because of its capability to lead to pollen abortion. At the same time, chloroplast transformation is taken into consideration to decrease the probability of gene flow.
Stomatal opening
Stomata are microscopic pores surrounded by two guard cells and play an important role in the uptake of CO2 for photosynthesis. Recent researches revealed that light-induced stomatal opening is mediated by at least three key components: blue light receptor phototropin, plasma membrane H+-ATPase, and plasma membrane inward-rectifying K+ channels. However, Yin Wang, et al (2014) showed that only increasing the amount of H+-ATPase in guard cells had a significant effect on light-induced stomatal opening (Fig. 1). Transgenic Arabidopsis plants by overexpressing H+-ATPase in guard cells exhibited enhanced photosynthesis activity and plant growth. Therefore,in order to strengthen the ability of absorbing formaldehyde, we overexpressed H+-ATPase(At AHA2) in transgenic tobacco guard cells,resulting in a significant effect on light-induced stomatal opening.
Fig.1 Typical stomata in the epidermis illuminated with light for 30 min(Yin Wang,et al.2014).
Biosafty
In order to promise biology safety, we use male sterility systems which can be used as a biological safety containment to prevent horizontal transgene flow. Pawan Shukla et al (2014) has used a plant pathogen-induced gene, cysteine protease for inducing male sterility. This gene was identified in the wild peanut, Arachisdiogoi differentially expressed when it was challenged with the late leaf spot pathogen, Phaeoisariopsispersonata.Arachisdiogoi cysteine protease (AdCP) was expressed under the strong tapetum-specific promoter (TA29) and tobacco transformants were generated. Morphological and histological analysis of AdCP transgenic plants showed ablated tapetum and complete pollen abortion.
Fig. 2Pollen germination of untransformed control plant and sterile transgenic plantsin vitro. Pollen grains were germinated on sucrose-boric acid medium and over 500 pollen grains were observed. a. Untansformed control plant pollen, b. Sterile pollen.Scale bar 25 μm (Pawan Shukla et al 2014).
Vectors
To compare the ability of different gene products in the metabolism of formaldehyde, we constructed 11 gene expression vectors(Table 1 and Fig. 3), including two backbone vectors, six mono-gene expression vectors and three multi-gene expression vectors.
Fig.3 The procedure we constructed our vectors.
piGEM001: P35S+P2A+T2A+F2A+nptII+T-HSP+T-35S+T-nos
piGEM002: piGEM001+AtAHA2+PGC1+PTA29+AdCP
piGEM003: piGEM002+TCP02-HPS-PHI
piGEM004: piGEM002+TCP03-AtFDH
piGEM005: piGEM002+TCP01-BbFALDH
piGEM006: piGEM002+HPS-PHI
piGEM007: piGEM002+AtFDH
piGEM008: piGEM002+BbFALDH
piGEM009: piGEM002+HPS-PHI+AtFDH+BbFALDH
piGEM010: piGEM002+TCP02-HPS-PHI+TCP03-AtFDH+FALDH
piGEM011: piGEM002+TCP02-HPS-PHI+TCP03-AtFDH+TCP01-BbFALDH
piGEM012: piGEM001+TCP02-HPS-PHI+TCP03-AtFDH+TCP01-BbFALDH
2A peptide Sequence
2A peptide sequences were found in Picornaviruses to mediate ‘cleavage’ between two proteins. We use 2A peptide linked multi-cistronicvectors to express multiple proteins from a single open reading frame (ORF) effectively.
Fig. 4 Schematic of 2A peptide
The 18-22 amino acids 2A self-cleaving oligo peptides can be used for co-expression of multiple, discrete proteins from a single ORF. Based on highly inefficient peptide bond formation between glycine and proline residues within the 2A peptide, placement of 2A peptide sequence as a linker region between tandem cDNA’s allows the stoichiometric translation of multiple unfused protein products. These sequences were first discovered in the foot-and-mouth disease virus (FMDV). And since then many 2A-like sequences have been identified in other viruses and some parasites. To minimize the risk of homologous recombination, it is important to use different 2A peptide sequences if more than two genes are being linked. The 2A peptide system has thus far worked successfully in all eukaryotic systems tested, from mammalian cells, yeast, and plants.In our project,we use F2A(from foot-and-mouth disease virus), P2A(from porcine teschovirus-1) and T2A (from Thoseaasigna virus) to achieve our goal.
Retrieved from " http://2014.igem.org/Team:UESTC-China/pad_Design "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
