<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UESTC-China_result"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UESTC-China/result">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UESTC-China/result&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UESTC-China/result&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UESTC-China/result&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UESTC-China/result" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UESTC-China/result</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>UESTC-China</TITLE></P><DIV id="total-wrap"><DIV id="header"><DIV id="top-menu"><DIV class="top-menu-each"><A href="https://2014.igem.org/Team:UESTC-China"><P>Home</P></A></DIV><DIV class="top-menu-each"><P>Team</P><DIV class="top-menu-sub"><UL><LI><A href="https://2014.igem.org/Team:UESTC-China/Team">Members</A></LI><LI><A href="https://2014.igem.org/Team:UESTC-China/Attribution">Attribution</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=UESTC-China">Team Profile</A></LI><LI><A href="https://2014.igem.org/Team:UESTC-China/Notebook">Notebook</A></LI></UL></DIV></DIV><DIV class="top-menu-each"><P>Project</P><DIV class="top-menu-sub"><UL><LI><A href="https://2014.igem.org/Team:UESTC-China/Project">Overview</A></LI><LI><A href="https://2014.igem.org/Team:UESTC-China/Design">Design</A></LI><LI><A href="https://2014.igem.org/Team:UESTC-China/result">Result</A></LI><LI><A href="https://2014.igem.org/Team:UESTC-China/Futurework">Future work</A></LI><LI><A href="https://2014.igem.org/Team:UESTC-China/Judingform">Judging Criteria</A></LI></UL></DIV></DIV><DIV class="top-menu-each"><P>Modeling</P><DIV class="top-menu-sub"><UL><LI><A href="https://2014.igem.org/Team:UESTC-China/Modeling">Overview</A></LI><LI><A href="https://2014.igem.org/Team:UESTC-China/Modeling1">HPS/PHI</A></LI><LI><A href="https://2014.igem.org/Team:UESTC-China/Modeling2">FALDH/FDH</A></LI><LI><A href="https://2014.igem.org/Team:UESTC-China/Modeling3">AtAHA2</A></LI></UL></DIV></DIV><DIV class="top-menu-each"><P>Parts</P><DIV class="top-menu-sub"><UL><LI><A href="https://2014.igem.org/Team:UESTC-China/BioBrick">BioBrick</A></LI><LI><A href="https://2014.igem.org/Team:UESTC-China/Protocol">Protocol</A></LI></UL></DIV></DIV><DIV class="top-menu-each"><P>Human Practice</P><DIV class="top-menu-sub"><UL><LI><A href="https://2014.igem.org/Team:UESTC-China/Questionnaire.html">Questionnaire</A></LI><LI><A href="https://2014.igem.org/Team:UESTC-China/Knowledgequiz">Knowledge quiz</A></LI><LI><A href="https://2014.igem.org/Team:UESTC-China/Lecture.html">Lecture</A></LI><LI><A href="https://2014.igem.org/Team:UESTC-China/Communication.html">Communication</A></LI><LI><A href="https://2014.igem.org/Team:UESTC-China/Art">Art</A></LI></UL></DIV></DIV><DIV class="top-menu-each"><P>Safety</P><DIV class="top-menu-sub"><UL><LI><A href="https://2014.igem.org/Team:UESTC-China/Safety">Safety</A></LI></UL></DIV></DIV><DIV id="middle-photo"><DIV class="middle-photo-each"><DIV id="SensorEditingArea" class="SensorEditingAreaClass"><H1 style="color:#1b1b1b; position:relative; left:0px; padding:15 5px; font-size:40px; font-family: calibri, arial, helvetica, sans-serif; font-weight: bold;font-style: Italic; text-align:center; width:1140px;">Result</H1><H1 class="SectionTitles" style="width:1100px; ">Vectors construction</H1><P style="color:#1b1b1b;">We have successfully constructed 2 backbones, piGEM001 and piGEM002. And we have verified them using digestion (Fig.1) and sequencing.</P><DIV align="center"><DIV><P style="position:relative; left:0px; padding:15 5px; font-size:20px; font-family: calibri, arial, helvetica, sans-serif; font-style: calibri; text-align:justify; width:1100px; color:#1b1b1b;"><STRONG>Fig.1</STRONG> Verification of backbones using digestion
				A: Digestion the plasmid piGEM001 with <I>Hpa</I>I and <I>Spe</I>I. 
				
				M: DNA marker;
				
				1: plasmid piGEM001 and its digestion product 
				
				B: Digestion the plasmid piGEM002 with <I>Hpa</I>I and <I>Dra</I>III. 
				
				M: DNA marker;
				
				1: plasmid piGEM002 and its digestion production				
				</P></DIV></DIV><DIV><P style="color:#1b1b1b;">Then we have successfully constructed 6 monogene expression vectors, from vector piGEM003 to vector piGEM008. And we have verified all of them using digestion and sequcencing. Here we only show the result of vector piGEM005 (Fig.2). </P><DIV align="center"><P style="position:relative; left:0px; padding:15 5px; font-size:20px; font-family: calibri, arial, helvetica, sans-serif; font-style: calibri; text-align:justify; width:1100px; color:#1b1b1b;"><STRONG>Fig.2</STRONG>
				Digestion the plasmid piGEM005 with <I>EcoR</I>I and <I>Sac</I>I. 
				M: DNA marker;
				1: piGEM005 plasmid and <I>TCP02-HPS-PHI</I> fragment 
				</P></DIV></DIV><P style="color:#1b1b1b;">In order to enhance the ability of tobacco to metabolize formaldehyde, we have successfully constructed 3 multigenge expression vectors, from vector piGEM009 to vector piGEM011. We have verified all of them using digestion (Fig.3) and sequcencing.</P><DIV align="center"><P style="position:relative; left:0px; padding:15 5px; font-size:20px; font-family: calibri, arial, helvetica, sans-serif; font-style: calibri; text-align:justify; width:1100px; color:#1b1b1b;"><STRONG>Fig.3</STRONG> Verification of multi-gene expression vectors using digestion
				A: Digestion the plasmid piGEM009 with <I>Xba</I>I and <I>Sal</I>I. 
				M: DNA marker;
				1: piGEM009 plasmid and it's digestion product 
				B: 
				Digestion the plasmid piGEM010 with <I>Hind</I>III and <I>Sac</I>I. 
				M: DNA marker;
				1: piGEM010 plasmid and it's digestion product 
				C: 
				Digestion of the plasmid piGEM011 with <I>EcoR</I>I and <I>Sac</I>I. 
				M: DNA marker;
				6: plasmid piGEM011 and it's digestion production				
				</P></DIV><H1 class="SectionTitles" style="width:1100px; ">Tobacco transformation</H1><P style="color:#1b1b1b;">Tobacco was transformed essentially using the leaf disk co-cultivation protocol of Horsch. This protocol includes infection and co-cultivation (Fig.4A), selection (Fig.4B), regeneration and rooting (Fig.4C). We have successfully transformed each vector into babacco and got positive transgenic plantlets (Fig.5). Table 1 is the statistical result of quantity of each transgenic line. And we have got PCR positive plantlets of every transgenic line.</P><DIV align="center"><P style="position:relative; left:0px; padding:15 5px; font-size:20px; font-family: calibri, arial, helvetica, sans-serif; font-style: calibri; text-align:justify; width:1100px; color:#1b1b1b;"><STRONG>Fig.4</STRONG>
				Transform piGEM003, piGEM004, piGEM005, piGEM006, piGEM007, piGEM008, piGEM009, piGEM010 and piGEM011 into tobacco. Co-cultured for 48 hours (A); Selection for one month (B); Rooting for one month (C).
				</P></DIV><DIV align="center"><P style="position:relative; left:0px; padding:15 5px; font-size:20px; font-family: calibri, arial, helvetica, sans-serif; font-style: calibri; text-align:justify; width:500px; color:#1b1b1b;"><STRONG>Fig.5</STRONG>
			Positive tobacco plantlets of each transgenic line.
				</P></DIV><DIV align="center"><P style="position:relative; left:0px; padding:15 5px; font-size:20px; font-family: calibri, arial, helvetica, sans-serif; font-style: calibri; text-align:justify; width:800px; color:#1b1b1b;"><STRONG>Table.1</STRONG>
				Statistical result of quantity of each transgenic line.
				</P></DIV><H1 class="SectionTitles" style="width:1100px; ">Expression of four key enzymes in tobacco</H1><P style="color:#1b1b1b;">We extracted DNA from tobacco plants. And then we used specific primers to amplify the target gene to verify the kan-resistant plants (Fig.6). Next we extracted RNA from tobacco leaves which are PCR positive. We used  RT-PCR to detect whether target gene was expressed (Fig.7).</P><DIV align="center"><P style="position:relative; left:0px; padding:15 5px; font-size:20px; font-family: calibri, arial, helvetica, sans-serif; font-style: calibri; text-align:justify; width:1100px; color:#1b1b1b;"><STRONG>Fig.6</STRONG>
				 PCR identification of kan-resistant tobacco plants (piGEM010 transgenic line).
				 M: DNA marker;
				WT: widetype control;
				P: plasmid;
				1-8: 6 individual lines 
				</P></DIV><DIV align="center"><P style="position:relative; left:40px; padding:15 5px; font-size:20px; font-family: calibri, arial, helvetica, sans-serif; font-style: calibri; text-align:justify; width:600px; color:#1b1b1b;"><STRONG>Fig.7</STRONG> RT-PCR verification of positive transgenic tobacco plants.
				</P></DIV><H1 class="SectionTitles" style="width:1100px; ">Enhanced formaldehyde tolerance </H1><P style="color:#1b1b1b;">The transgenic and wildtype plants, which had been grown separately in sealed boxes, were exposed to formaldehyde evaporated from a micro tube (0.5ml) containing formaldehyde solution (37%, 10μl) (Fig.8). Two weeks later we observed the phenotype of transgeneic  and widetype plants (Fig.9). We found that the transgenic plant is stronger than wildtype after formaldehyde exposure. This indicates that production of <I>HPS/PHI</I>, <I>FALDH</I> and <I>FDH</I> enhanced formaldehyde tolerance of transgenic plant.</P><DIV align="center"><P style="position:relative; left:0px; padding:15 5px; font-size:20px; font-family: calibri, arial, helvetica, sans-serif; font-style: calibri; text-align:justify; width:1100px; color:#1b1b1b;"><STRONG>Fig.8</STRONG>
				The transgenic (A) and wildtype plants (B), which had been grown separately in sealed boxes, were exposed to formaldehyde evaporated from a micro tube (0.5ml) containing formaldehyde solution (37%, 10μl)
				</P></DIV><DIV align="center"><P style="position:relative; left:0px; padding:15 5px; font-size:20px; font-family: calibri, arial, helvetica, sans-serif; font-style: calibri; text-align:justify; width:1100px; color:#1b1b1b;"><STRONG>Fig.9</STRONG>
				Phenotype testing of transgenic plants and wildtype. A: Before exposure to HCHO. B: Exposure to HCHO for one week. The transgenic plant is stronger than wildtype after formaldehyde exposure. 10μl 37% HCHO, two weeks.
				</P></DIV><H1 class="SectionTitles" style="width:1100px; ">Enhanced formaldehyde absorbance</H1><P style="color:#1b1b1b;">We detected the concentration of gaseous formaldehyde evaporated from a micro tube (0.5ml) containing formaldehyde solution (37%, 10μl) and made a curve (Fig.10) about relationship between formaldehyde concentration and time. And we saw a linear relationship between formaldehyde concentration and time before formaldehyde is saturated. For quantity result, we used a formaldehyde detector to detect the concentration of  gaseous formaldehyde (Fig.11). The transgenic and wildtype plants, which had been grown separately in sealed boxes, were exposed to formaldehyde evaporated from a micro tube (0.5ml) containing formaldehyde solution (37%, 10μl) for about 3 weeks. Three weeks later, the covers of the plant boxes were removed and quickly replaced with covers equipped with formaldehyde dose-monitoring tubes in order to determine roughly the gaseous formaldehyde levels remaining in the boxes. We have not got the precise data results now, and this work is to be continued.</P><DIV align="center"><P style="position:relative; left:50px; padding:15 5px; font-size:20px; font-family: calibri, arial, helvetica, sans-serif; font-style: calibri; text-align:justify; width:650px; color:#1b1b1b;"><STRONG>Fig.10</STRONG>
				The relationship between formaldehyde concentration and time
				</P></DIV><DIV align="center"><P style="position:relative; left:50px; padding:15 5px; font-size:20px; font-family: calibri, arial, helvetica, sans-serif; font-style: calibri; text-align:justify; width:500px; color:#1b1b1b;"><STRONG>Fig.11</STRONG>
				Folmaldehyde concentration detection.
				</P></DIV><H1 class="SectionTitles" style="width:1100px; ">Transit peptides affect formaldehyde degrading efficiency </H1><P style="color:#1b1b1b;"><I>HPS</I>, <I>PHI</I>, and <I>FDH</I> are located in chloroplast, while <I>FALDH</I> plays a role in cytoplasm. So we used transit peptides to locate the productions of these genes. We hope to know the effects of transit peptide on degrading formaldehyde. So we exposed transgenic tobaccos with and without transit peptides to formaldehyde (37%, 10μl). However, there are no obvious phenotype difference compairing different transgenic lines because time limited.</P><H1 class="SectionTitles" style="width:1100px; ">Different genes affect formaldehyde degrading efficiency </H1><P style="color:#1b1b1b;">To test which enzymes play the most important role in pathway of metabolizing formaldehyde, we constructed mono-gene expression vectors to express each enzyme individually. We also constructed multi-gene expression vectors to test whether the ability of metabolizing formaldehyde of transgenic tobacco enhanced. Then these transgenic tobaccos were exposed to formaldehyde (37%, 10μl). However, we have not got obvious phenotype difference among transgenic lines, because there is not enough time</P><H1 class="SectionTitles" style="width:1100px; ">Tapetal expression of <I>AdCP</I> results in male sterility in high expression plants</H1><P style="color:#1b1b1b;">Considering the problem of environment and safety, we use male sterility system which prevents the horizontal transgene flow. Morphological and histological analysis of <I>AdCP</I> transgenic plants showed ablated tapetum and complete pollen abortion. However, there is not enough time to wait for transgenic tobacco flowering.</P></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:UESTC-China/result">http://2014.igem.org/Team:UESTC-China/result</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UESTC-China/result" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UESTC-China/result" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UESTC-China/result&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UESTC-China/result&amp;oldid=389092" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>