"
Page
Discussion
View source
History
teams
Log in
 
Team:UESTC-China/Protocol
From 2014.igem.org
UESTC-China
Home
Team
Members
Attribution
Team Profile
Notebook
Project
Overview
Design
Result
Future work
Judging Criteria
Modeling
Overview
HPS/PHI
FALDH/FDH
AtAHA2
Parts
BioBrick
Protocol
Human Practice
Questionnaire
Knowledge quiz
Lecture
Communication
Art
Safety
Safety
Protocol
Fragments Ligation
Component
20µl Reaction
Vector DNA
50ng
Insert DNA
the moles rate of insert DNA to vector DNA is 5：1
10X T4 DNA ligase buffer
2µl
T4 DNA ligase
1µl
ddH2O
to 20µl
Incubate at 25℃ for 12 hours
Gibson Assembly
Component
20µl Reaction
1.33X Master Mix
15µl
Insert DNA
the moles rate of insert DNA to vector DNA is 5：1
Vector DNA
50ng
ddH2O
To 20µl
Incubate at 50℃ for 1 hour
Fragment Digestion
Component
50µl Reaction
10x FastDigest buffer
5µl
Restriction enzyme 1
1µl
Restriction enzyme 2
1µl
DNA
1-2µg
ddH2O
to 50µl
Incubate at 37℃ for 2 hours
PCR Amplication Protocol for KOD-Plus-Neo
Component
50µl Reaction
ddH2O
33µl
10X buffer for KOD-Plus-Neo
5µl
MgSO4
3µl
10mM dNTPs
5µg
10µM forward primer
1µl
10µM reverse primer
1µl
Template
1µl
KOD enzyme
1µl
Temperature
Time
Cycle
Step1
94℃
5min
X1 cycle
Step2
94℃
30s
X35 cycles
56℃
30s
68℃
30s/kb
Step3
68℃
5min
X1 cycles
10℃
10min
Fusion PCR Amplication
Each template will be diluted by the number of moles into 2-10ng/µl.
Component
50µl Reaction
ddH2O
32µl
10X buffer for KOD-Plus-Neo
5µl
MgSO4
3µl
10mM dNTPs
5µl
10µM forward primer
1µl
10µM reverse primer
1µl
Template one
1µl
Template two
1µl
KOD enzyme
1µl
Temperature
Time
Cycle
Step1
94℃
5min
X1 cycle
Step2
94℃
30s
X35 cycles
56℃
30s
68℃
30s/kb
Step3
68℃
5min
X1 cycles
10℃
10min
Colony PCR
Component
25µl Reaction
ddH2O
15.8µl
10X Taq buffer
2.5µl
2.mM dNTPs
0.5µl
10µM forward primer
0.5µl
10µM reverse primer
0.5µl
Taq enzyme
0.2µl
Water with colony
5µl
Temperature
Time
Cycle
Step1
95℃
5min
X1 cycles
Step2
94℃
30s
X35 cycles
56℃
30s
72℃
1min/kb
Step3
72℃
10min
X1 cycles
10℃
10min
E. coli Transformation
1）Streak E.coli cells (DH5α) on an LB plate, (BL21 (DE3) LysS cells on LB plate + 25 mg/ml chloramphenicol);
2） Allow cells to grow at 37℃ overnight;
3）Place one colony in 10 ml LB media (+ antibiotic selection if necessary), grow overnight at 37℃;
4） Take 2 ml LB media and save for blank, transfer 5 ml overnight DH5α culture into 500 ml LB media in 3 L flask;
5） Allow cell to grow at 37℃ (250 rpm), until OD600= 0.4 (~2-3 hours);
6） Transfer cells to 2 centrifuge bottles (250 ml), and place cells on ice for 20 min;
7） Centrifuge cells in at 4℃ for 10 min at 3,000 g and subsequent resuspension may be done in the same bottle. Cells must remain cold for the rest of the procedure: Transport tubes on ice and resuspend on ice in the cold room;
8） Pour off media and resuspend cells in 30 ml of cold 0.1 M CaCl2. Transfer the suspended cells into 50 ml polypropylene tubes, and incubate on ice for 30 min;
9） Centrifuge cells at 40℃ for 10 min at 3,000 g;
10） Pour supernatant and resuspend cells (by pipetting) in 8 ml cold 0.1M CaCl2 containing 15% glycerol Transfer 140 ml into (1.5 ml).Eppendorf tubes placed on ice. Freeze the cells in liquid nitrogen. Cells stored at -80℃ can be used for transformation for up to ~6 months;
11） Add 10 to 40 ng (10 to 25 ml volume) of DNA to 250 ml of competent cells in step;
12） Incubate the mixture on ice for 30 minutes;
13） Transfer the reaction to a 42℃ water for 1min;
14） Add 0.9 ml of LB culture to each tube and incubate at 37℃ for 1 hour in a roller drum (250 rpm) to allow cells to recover and express the antibiotic resistance marker;
15） Incubate on ice for 2 minutes;
16） Spread the appropriate quantity of cells (50 to 100 ml) on selective media. Store the remaining cells at 4℃.
(A) E. coli cells from the control tube without DNA in step 12 above are plated on selective medium and nonselective medium. The first plating ensures that the selective medium is working properly since no growth should be observed. The second plating provides the number of viable cells in the absence of selective medium.
(B) E. coli cells being tested for competency are plated on LB agar containing ampicillin (50 mg/ml final concentration) to ensure that the transformation efficiency has not decreased over time due to storage.
17） Incubate all plates overnight at 37℃.
18） Count the number of colonies.
A.tume Transformation
1）Have 0.5~1 g plasmid DNA into 100  L competent cells, on ice for 30 min；
2）Frozen in liquid nitrogen for 5 min, grow at 37 C for 5 min, on ice for 2 min；
3）Pour in 1000 LYEB, 200 rpm, grow at 28℃ for 2~3 h；
4）6000rpm，2min. Suspend collected bacteria with 100µl YEB and evenly coat that at a YEB medium. (125 mg/L Sm or 50 mg/L rif, and 50 mg/L Kan included);
5）Grow upside down at 28℃, for 48h;
6）Pick positive clones and grow them in LB medium with antibiotic at 28℃ for 48h;
7）Inject to LB medium in flasks by the day of transformation, in the rate of 1:50. Grow to OD600 = 0.5. Ready to infect tobacco leaf discs.
Agrobacterium-mediated Tobacco Transformation
Tobacco was transformed essentially by following the leaf disk co-cultivation protocol of (McCormick et al., 1986). Co-cultivation was initiated by dipping leaf disks in an Agrobacterium suspension, blotting them on sterile tissue paper, and incubating them for 2 d on MS medium (Murashige and Skoog 1962) containing naphthalene acetic acid (NAA 0.1mg/L), 6-Benzylaminopurine (6-BA, 2.0mg/L). Cefotaxime sodium (Cef) was included in the medium (500mg/L) to inhibit Agrobacterium growth. The leaf disks were then transferred onto a medium containing antibiotics for transgenic plant selection(kanamycin, 50 mg/L), and NAA (0.1 mg/L), 6-BA (2.0 mg/L), Cef (500mg/L). And incubate them for 1 month on the medium above. At last, cut off the bud from the callus, put the buds into the mudium containing NAA (0.1mg/L), Cef (500mg/L) and kanamycin (25 mg/L).
Detection Method
1）Molecular Identification
Extract genomic DNA from leaves of tobacco seedlings. Using specific primers to amplify the target gene. Extract leaf mRNA from leaves of grown tobacco which DNA testing was positive. Detected by the method of RT-PCR whether target gene is expressed.
2）Phenotype testing
The experimental group
control group
Qualitative detection
Have 6 positive plants from every transgenic line (about 8 leaves age) , and 6 wild-type seedings with the same growth equally distributed into three 650ml culture bottles. Treat with 10µl 37% formaldehyde for two weeks.
Quantitative detection
Have 4 positive plants from every transgenic line (about 8 leaves age) in a 650ml culture bottle. Treat with 10µl 37% formaldehyde for 3 weeks. Using a formaldehyde detector (Gastec Passive Dositube, 91D, Ayase, Kanagawa, Japan) was set in the hole to detect gaseous formaldehyde. Three and a half hours later, the measurement was stopped and the results were photographed (Chen et al., 2010).
Put 4 wild -type seedlings with the same growth of seedlings in experimental group into 650ml culture bottle，with same processing as the case of the experimental group.
References
McCormick, S., J. Niedermeyer, J. Fry, A. Barnason, R. Horsch and R. Fraley (1986). "Leaf disc transformation of cultivated tomato (L. esculentum) using Agrobacterium tumefaciens."Plant Cell Rep 5(2): 81-84.
Chen, L. M., H. Yurimoto, K. Z. Li, I. Orita, M. Akita, N. Kato, Y. Sakai and K. Izui (2010). "Assimilation of formaldehyde in transgenic plants due to the introduction of the bacterial ribulose monophosphate pathway genes." Biosci Biotechnol Biochem 74(3): 627-635.
Retrieved from " http://2014.igem.org/Team:UESTC-China/Protocol "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
