<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Queens_Results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Queens/Results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Queens/Results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Queens/Results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Queens/Results&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Queens/Results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Queens/Results</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><TABLE style="background-color:#922334; position:relative; overflow:auto; left:200px; top:-215px; width:750px"><TBODY><TR><TD align="left"><P style="font-size:175%;font-family:corbel;color:#172C4E;font-weight:bold"><I>Results</I></P></TD></TR><TR><TD align="left"><P style="font-size=120%;font-family:palatino linotype;color:#ECB528">
The following summarizes the results of the various aspects of this year's QGEM Project. 
</P></TD></TR><TR><TD align="left"><A href="#Part1"><P style="font-size=150%;font-family:Corbel;color:#172C4E;font-weight:bold">
Part One: SAA
</P></A><A href="#Part2"><P style="font-size=150%;font-family:Corbel;color:#172C4E;font-weight:bold">
Part Two: Binding Construct
</P></A><A href="#Part3"><P style="font-size=150%;font-family:Corbel;color:#172C4E;font-weight:bold">
Part Three: Heme and HO-1
</P></A><A href="#Part4"><P style="font-size=150%;font-family:Corbel;color:#172C4E;font-weight:bold">
Part Four: Future Directions
</P></A><A href="#Part5"><P style="font-size=150%;font-family:Corbel;color:#172C4E;font-weight:bold">
Safety Considerations
</P></A></TD></TR><TR><TD align="left"><A name="Part1"><P style="font-size:150%;font-family:corbel;color:#172C4E;font-weight:bold">Part One: SAA</P></A><P style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528">
Serum amyloid A (SAA) featured prominently on our list of possible effectors to release at the site of atherosclerotic plaques due to research done at Queen’s University by Tam et al. (2005). This research showed that treatment of SAA caused macrophages to reverse and prevent esterification of cholesterol, thereby allowing it to be exported out of plaques by high-density lipoprotein (HDL). In order to get the SAA to be picked up by the macrophages we needed to produce large quantities of the molecule, and then secrete it extracellularly. In order to secrete the protein we decided on using the twin-arginine translocase (TAT) system, which can transport fully folded proteins outside the cell (Sargent et al., 2006) and is found in the majority of prokaryotes. We then fused the TAT signal sequence to the front of our SAA sequence in order to produce a construct that should be able produce and secrete the protein, allowing it to be taken up by macrophages.
</P><P style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528"><I>Fig. 1</I> SAA expression and secretion by <I>E. coli</I> cells. Cultures of <I>E. coli</I> cells containing either the SAA construct or control plasmid were spun down at 0, 4, and 12 hours after seeding. The culture medium and whole cell lysates were analyzed by SDS-PAGE and Western blot analysis with a polyclonal antibody recognizing whole SAA protein.   
</P><P style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528">
Our <I>E. coli</I> cells transformed with SAA construct did not appear to express or secrete SAA into the growth medium even after growing for 12 hours in optimum condition. This might be due to defective ligation of the Ptet-RBS fragment to the SAA-encoding gene. We plan to sequence the construct in the future. 
</P><P style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528">
References:

Benditt, E.P., Hoffman, J.S. &amp; Eriksen, N. (1982). SAA, an apoprotein of HDL: its structure and function. <I> New York Academy of Sciences. </I> 82:183-189.

Ancsin, J.B. &amp; Kisilevsky, R. (1999). The heparin/heparin sulfate-binding site on apo-serum amyloid A. <I>Journal of Biological Chemistry.</I> Vol. 274, 11:7172-7181.
</P></TD></TR><TR><TD align="left"><A name="Part2"><P style="font-size:150%;font-family:corbel;color:#172C4E;font-weight:bold">Part Two: Binding Construct</P></A><P style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528">
In our original design of the binding construct, constitutive promoter BBa_J23119 was used to control the expression of the chimeric protein receptor consisted of the Lpp-OmpA fusion, TEV protease cut sites and ITGA4 fragment. We ordered our construct to be synthesized by Mr. GENE in early June. However, Mr. GENE notified us in August that our construct appeared to be unstable in and toxic to <I>E. coli</I> cells. The reason might be that high expression level of Lpp-OmpA might interfere with bacterial physiology, causing severe growth inhibition and reduced viability. (Daugherty et al, 1999) Mr. Gene was able to give us a sequence-confirmed PCR product of the synthesized construct. Thus, we decided to replace P¬const with P¬tet (BBa_R0040) by PCR. Below is a schematic diagram of the construction process.   
</P><P style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528"><I>Fig. 2</I> Flowchart of PCR stitching for constructing the binding construct. 
</P><P style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528">
Due to time constraint, we were only able to confirm the stitching of Ptet-RBS fragment to Lpp-OmpA-Linker-TEVx2-Linker fragment (PCR round 2) using Agarose gel electrophoresis. The third round of PCR stitching did not yield expected band. Construction of the binding construct is ongoing.   
</P><P style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528"><I>Fig. 3</I><I>In silico</I> model of the binding construct. 
</P><P style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528">
References:

Daugherty, P.S., Olsen, M.J., Iverson, B.L. &amp; Georgiou, G. (1999). Development of an optimized expression system for the screening of antibody libraries displayed on the <I>Escherichia coli</I> surface. <I>Protein Engineering.</I> Vol. 12, 7:613-621.

Yang, Z., Lui, Q., Wang, Q. &amp; Zhang, Y. (2008). Novel bacterial surface display systems based on outer membrane anchoring elements from the marine bacterium <I>Vibrio anguillarum</I>. <I> American Society for Microbiology.</I> Vol. 74, 14: 4359-4365.
</P></TD></TR><TR><TD align="left"><A name="Part3"><P style="font-size:150%;font-family:corbel;color:#172C4E;font-weight:bold">Part Three: Heme and HO-1</P></A><P style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528">
Before  presenting our results some useful parts have been outlined briefly.

BioBrick and Construct Legend:

BioBrick I716390: Plasmid containing a composite of Heme A, B C &amp; D with T7-promoter and RBS. Amp Resistance.

BioBrick K098010: HO and phycocaynobilin:ferredoxin oxidoreductase. KAN Resistance.

BioBrick I15009: HO-1 from <I>Synechocystis</I>. Amp Resistance.

NLM 350: Rough <I> E. coli</I> strain containing T7-polymerase. Queen's University Research
</P><P style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528">
In order to determine whether or not the BioBrick I716390 submitted by the Berkeley iGEM team in 2007 was producing heme, NLM350. Overnight cultures containing 100 μg/mL ampicillin were set up and inoculated with NLM350 containing I716390. The culture was analyzed daily by wavelength scan in a spectrometer. We ran the wavelength scan twice, once with a blank of LB and once with a blank of LB culture containing NLM350 without the BioBrick. Both of the scan results were identical, which indicates that the peaks present are due to the BioBrick plasmid and not resulting from compounds produced by NLM350. The results from day three with a blank of LB are shown in Figure 4.
</P><P style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528"><I>Fig. 4</I> Results of a wavelength scan between 350 and 550nm of the NLM350 + I716390 culture three days after inoculation. Strongest peaks at 412nm. 
</P><P style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528">
In order to check for the presence of the heme/HO-1 complex, NLM350 + I716390 cells were made electrocompetent and two sets of electroporations were performed. The BioBricks K098010 and I15008 were taken up by NLM350 + I716390 and the resulting colonies were used to inoculate cultures containing 40 μg/mL kanamycin and 100 μg/mL ampicillin, respectively. Triplicate testing of NLM350 + 176390 + KO98010 as well as NLM350 + 176390 + I15008 was conducted over five days. Both tests produced similar strong peaks for heme/HO-1 complexes, ranging from 407-409 nm. Again, we ran the scans twice with blanks of LB and LB inoculated with NLM350 and found that results were identical. The results from day five with blanks of LB are shown in Figures 5 and 6.
</P><P style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528"><I>Fig. 5</I> Wavelength scan between 350 and 550 nm of the NLM350 + I716390 + K098010 culture 5 days after inoculation. Strong peak of 407.5 nm. 
</P><P style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528"><I>Fig. 6</I> Wavelength scan between 350 and 550 nm of the NLM350 + I716390 + I15008 culture 5 days after inoculation. Strong peak of 407.5 nm. 
</P><P style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528">
As can be seen from our results above, the peaks in the cultures containing heme and HO-1 producing BioBricks become clearer and occur at a lower absorbance, around 408nm, than the peaks from cultures containing only heme-producing BioBricks. Cultures containing only the heme-producing BioBrick has the strongest peaks at 412nm, which corresponds to the peak found by Moffet et al. in 2003 <SUP>1</SUP>. Since HO-1 does not have an absorbance peak of its own <SUP>2,3,4</SUP>, the presence of HO-1 can only be inferred by identifying the heme/HO-1 complexes via spectroscopy. According to several authors <SUP>2,4</SUP>, Soret bands for the oxy-bound form of the heme/HO-1 complex occur at 410nm. The peaks seen in Figures 5 and 6 are slightly blue-shifted compared to the literature values, which may be the result of different biochemical subunits present in the heme protein<SUP>5</SUP>. However, the difference between the spectra of Figures 4, 5, and 6 indicates that HO-1 is being produced and is binding with heme to form a complex which peaks at a different wavelength than heme alone.
</P><P style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528">
It was expected to see a peak for biliverdin as well, which would have occurred between 440 – 460nm according to the literature<SUP>6,7</SUP>, though we did not. This might have been due to the fact that the cultures containing heme and HO-1 producing BioBricks had some exposure to light. Researchers have noted in their methods that they had avoided light when handling biliverdin <SUP>8,9</SUP>. If biliverdin is photosensitive, this might be why these peaks did not appear in our wavelength scans.
</P><P style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528">
In order to continue on this project track, the genetic effector system would have to be streamlined and optimized. More thorough kinetics testing for the production of heme as well as its metabolism by HO-1 would be required to maximize effector quality and concentration. Though many of the products of heme metabolism were likely produced, bilirubin was almost certainly not. 
</P><P style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528">
As mentioned previously<SUP>2</SUP>, there is a lack of biliverdin reductases available for cloning into bacterial systems, especially mammalian. Recently the entire genome of <I>Synechocystis sp.</I> PP6803 was sequenced, along with a cyanobacterial biliverdin reductase homologue. This enzyme is used in the production of light-harvesting pigments called phycobilins which is a product in the breakdown of heme<SUP>10</SUP>. With more research into extraction of this gene from blue-green algae, a very useful biliverdin reductase BioBrick could be contributed the registry, allowing full degradation of heme to bilirubin. This would enable the microbe as well as the system to which it is contributing, to reap the strong antioxidant properties of this molecule.
</P><P style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528">
References:

1. Moffet, D.A., Foley J. &amp; Hecht M.H. (2003). Midpoint reduction potentials and heme binding stoichiometries of de novo proteins from designed combinatorial libraries. <I>Biophysical Chemistry</I>. 105:231-239.

2. Frankenberg-Dinkel, N. (2004). Forum review: bacterial Heme Oxygenases. Antioxidants and Redox Signalling. 6:825 - 834.

3. Huber W. &amp; Backes W. (2008). Quantitation of heme oxygenase 1: Heme titration increases yield of purified protein. <I>Analytical Biochemistry</I>. 373:167-169.

4. Migita C.T., Zhang X. &amp; Yoshida T. (2003). Expression and characterization of cyanobacterium heme oxygenase, a key enzyme in the phycobilin synthesis. <I>European Journal of Biochemistry</I>. 270:687 – 698.

5. Wang N., Zhao X. &amp; Lu Y. (2005). Role of Heme Types in Heme-Copper Oxidases:  Effects of Replacing a Heme b with a Heme to Mimic in an Engineered Heme-Copper Center in Myoglobin. <I>Journal of American Chemical Society</I>. 127:16541 – 16547.

6. Yoshida T. &amp; Kikuchi G. (1979). Purification and Properties of Heme Oxygenase from Rat Liver Microsomes. The <I>Journal of Biological Chemistry</I>. 254:4487 – 4491.

7. Ding Z.K. &amp; Xu Y.Q. (2002). Purification and characterization of biliverdin IXalpha from Atlantic salmon (Salmo salar) bile. <I>Biochemistry</I>. 67:927 – 932. 

8. Ollinger R., Yamashita K., Bilban M., Erat A., Kogler P., Thomas M., Csizmadia E.,  Usheva A., Margreiter R. &amp; Bach F. H. (2007). Bilirubin and Biliverdin Treatment of Atherosclerotic Diseases. <I>Cell Cycle</I>. 6:39 – 43. 

9. Ishikawa K., Navab M., Leitinger N., Fogelman A.M. &amp; Lusis A. J. (1997). Induction of heme oxygenase-1 inhibits the monocyte transmigration induced by mildly oxidized LDL. <I>Journal of Clinical Investigation</I>. 100:1209-1216.

10. Cornejo J., Willows R. D. &amp; Beale S. I. (1998). Phytobilin biosynthesis: cloning and expression of a gene encoding ferredoxin-dependent heme oxygenase from Synechocystis sp. PCC 6803. <I>The Plant Journal</I>. 15:99 – 107.
</P></TD></TR><TR><TD align="left"><A name="Part4"><P style="font-size:150%;font-family:corbel;color:#172C4E;font-weight:bold">Part Four: Future Directions</P></A><P style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528">
Due to the delay of the synthesis of our binding construct, we were unable to test the presentation of the ITGA-4 chain of the VLA-4 antigen on <I>E. coli</I> outer membrane and the binding of the <I>E. coli</I> to endothelial cells expressing VCAM-1. In the future, we are planning to pursue this project further in the following areas.
</P><P style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528"><U>Construction</U></P><PRE style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528">
1. Complete round 3 of PCR stitching and insert the binding construct into BioBrick backbone.

2. Sequence the binding construct and the SAA construct.
</PRE><P style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528"><U>Endothelial Adhesion Assay</U></P><PRE style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528">
1. Transform the binding construct and constitutive GFP expression construct into E. coli cells. 

2. Plate C166 murine endothelial cells on a 6-well plate. Incubate overnight. 

3. Prepare serial dilutions of the binding E. coli and control cells (containing plasmid backbone). 

4. Incubate C166 cells with E. coli for two hours and then wash wells 3X with PBS. 

5. Quantify GFP emission using fluorimeter.  

6. Alternative test: ELISA binding assay with recombinant VCAM-1 protein. 
</PRE><P style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528"><U>Atrial Natriuretic Peptide</U></P><PRE style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528">
1. Transform <I>E. coli</I> cells with ANP construct (P<SUB>tet</SUB>-RBS-Lpp-OmpA-Linker-TEVx2-Linker-ANP)
    and constitutive GFP expression construct. 

2. Use the same protocol as endothelial adhesion assay to test the activation of membrane-bound 
    guanylate cyclase (GCA) on endothelial cells by ANP. 

3. Harvest the endothelial cells and prepare whole cell lysates. 

4. Run SDS-PAGE and Western blotting analysis with antibodies recognizing whole and 
    phosphorylated Vasodilator Stimulated Phosphoprotein (VASP). VASP is phosphorylated 
    by cGMP dependent kinase, which is activated by ANP. 
</PRE><P style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528"><U>Inducible Effector System</U>
In order to make our <I>E. coli</I> cells releasing the effectors at the site of plaque, we plan to put the SAA and HO-1 gene under the control of the P<SUB>lux</SUB> promoter, which will be activated by a threshold concentration of AHL at the site of plaque.
</P><P style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528"><I>Fig. 7</I> Planned inducible effector construct.
</P><P style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528">
We plan to transform <I>E. coli</I> cells with the effector construct and the binding construct, and conduct the endothelial adhesion assay. We will then analyze the growth medium of the endothelial culture by SDS-PAGE and Western blot analysis to determine the concentration of AHL, SAA, and biliverdin in the medium. We will also determine the expression level of TEV protease, which is located downstream of a double terminator (BBa_B0015) and RBS (BBa_B0034) on the effector construct. Expression level of TEV protease directly contributes to the efficiency of detachment from the plaque and termination by the kill switch.
</P></TD></TR><TR><TD align="left"><A name="Part5"><P style="font-size:150%;font-family:corbel;color:#172C4E;font-weight:bold">Safety Considerations</P></A><PRE style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528">
We currently have no reason to believe that there are any safety concerns with our project.
The iGEM safety questionnaire is below. 

1. Would any of your project ideas raise safety issues in terms of:
       * researcher safety,
       * public safety, or
       * environmental safety? 

    At the current stage the project poses no safety concerns of any kind.

2. Is there a local biosafety group, committee, or review board at your institution?

    In short, yes there is an ethics and safety board that we must pass new ideas through.

3. What does your local biosafety group think about your project?

    The project is considered within the approved guidelines.

4. Do any of the new BioBrick parts that you made this year raise any safety issues?
       * If yes, did you document these issues in the Registry? 

    The only issue is the toxicity of BBa_K214001 to host bacteria if over-expressed.
    This issue is mentioned in the registry.
</PRE></TD></TR><TR><TD align="left"><PRE style="border-style:none;background-color:#922334;font-size:120%;font-family:palatino linotype;color:#ECB528">
Please note that Results are also available in PDF Format. 
Please click on the document you wish to view.
</PRE></TD></TR><TR><TD align="left"><A href="https://static.igem.org/mediawiki/2009/f/fc/QueensResultsJamesBogdanBryantHarry.pdf"><P style="font-size:150%;font-family:corbel;color:#ECB528;font-weight:bold">Laboratory One: Harry, Bogdan, James, Bryant</P></A><A href="https://static.igem.org/mediawiki/2009/2/26/QueensResultsKateMike.pdf"><P style="font-size:150%;font-family:corbel;color:#ECB528;font-weight:bold">Laboratory Two: Kate, Mike</P></A></TD></TR></TBODY></TABLE><FOOTER><FONT color="#172C4E"><CENTER>
Last Updated: October 21, 2009 by Fr<SUB>3</SUB>P</CENTER></FONT></FOOTER><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Queens/Results">http://2009.igem.org/Team:Queens/Results</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Queens/Results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Queens/Results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Queens/Results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Queens/Results&amp;oldid=143835" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>