<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_HKU-Hong_Kong_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:HKU-Hong_Kong/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:HKU-Hong_Kong/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:HKU-Hong_Kong/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:HKU-Hong_Kong/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:HKU-Hong_Kong/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:HKU-Hong Kong/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="main1"><DIV id="main2"><DIV id="middle"><DIV class="column-in"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Introduction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Overall_project"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Overall project</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Ways_to_achieve_this"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Ways to achieve this</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-4"><A href="#Project_Details"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Project Details</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Killing_Mechanism"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Killing Mechanism</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Changes_in_our_plans"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Changes in our plans</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Difficulties"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Difficulties</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#The_changes"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">The changes</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#The_plasmid_design"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">The plasmid design</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Scenarios"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Scenarios</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Possible_flaws"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Possible flaws</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Results"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Possible_further_research"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Possible further research</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Introduction"><B>Introduction</B></SPAN></H2><P>After the British Petroleum oil spill that occurred off the coast of the Gulf of Mexico earlier this year, it seemed only natural that our team members take an environmental outlook when it came to brainstorming possible research areas. While researching bacteria capable of oil digestion we found that some varieties are in actual fact pathogenic and so if actually used to clean up oil in the ocean, they may result in harm to the environment or wildlife after completion of their function (degrading oil). Armed with this knowledge, we naturally started to envision a project where we would not just create a solution to the problem of oil spills but also engineer a mechanism that could have potential use in any application of genetic engineering. Apart from bacteria in the degradation of oil we believe that our “bio-safety net” must be implemented into bacteria used in many other applications such as for the killing of cancer cells.
</P><H2><SPAN class="mw-headline" id="Overall_project"><B>Overall project</B></SPAN></H2><H3><SPAN class="mw-headline" id="Ways_to_achieve_this"> Ways to achieve this </SPAN></H3><UL><LI>Insertion of killing genes
</LI><LI>Regulated by specific promoters
</LI></UL><UL><LI>Promoters respond to changes in the environment
</LI></UL><P>Hence bacteria can and can only function well and survive in a specific environment
</P><H2><SPAN class="mw-headline" id="Project_Details"><B>Project Details</B></SPAN></H2><H3><SPAN class="mw-headline" id="Killing_Mechanism"> Killing Mechanism </SPAN></H3><P><B>Enterobacteriophage T4 Lysis Device</B></P><UL><LI>We found that The Berkeley's 2008 iGEM team has created a biobrick [Part:BBa_K112808, with no promotor] that could possibly be used as a means to attain cell-lysis.
</LI></UL><UL><LI>Using genes from enterobacteriophage T4 (endolysin-lysozyme, holin and antiholin). 
</LI></UL><DL><DT>T4 Holin 
</DT><DD> inner membrane-bound proteins, creating pores in the membrane and thus permeabilizing the membrane.
</DD></DL><DL><DT>T4 Antiholin 
</DT><DD> binding to and inactivating holins, producing inactive holin-antiholin dimers.
</DD></DL><DL><DT>Lysozyme</DT><DD> T4 endolysin
</DD><DD> entering the periplasm and degrading peptidoglycan, resulting in cell lysis. 
</DD></DL><P>The above picture illustrates the mechanism of the cell-lysis
</P><UL><LI>As the device comes with no promotor, a variety of promoters could be installed to control cell-lysis. 
</LI></UL><UL><LI>It should be pointed out that antiholin has its own constitutive promoter to prevent formation of holin multimers from basal expression. It is of importance as when the device is turned off, relatively higher expression of antiholin is needed and necessary for better stability of the device.
</LI></UL><UL><LI>below is another gene we found it possible to achieve the cell-lysis
</LI></UL><P><B>Gene <I>E</I> from Bacteriophage phiX174</B></P><P>The figure shows the location of gene <I>E</I> in the phiX174 genome
(YOUNG, MICROBIOLOGIcAL REVIEWS, Sept. 1992,Vol. 56, No. 3)
</P><UL><LI>Gene <I>E</I> is the killing gene in the bacteriophage phiX174. Expression of gene <I>E</I> lead to lysis of the cell.
</LI></UL><UL><LI>Lysis works without  elaborating a lytic enzyme, lysozyme
</LI></UL><UL><LI>Small, phase-dark spherical bulges or blebs were developed in host cells 
</LI></UL><UL><LI>These blebs are able to grow as large as the summed volumes of the residual rod-shaped portions of the cell.
</LI></UL><UL><LI>Spherical bodies become freed and emptied to become broken ghosts
</LI></UL><UL><LI>They become less refractile and leaving visible membranous debris
</LI></UL><UL><LI>Some blebs burst and other phage are released
</LI></UL><UL><LI>Small lesions which are also called transmembrane tunnels start to appear
</LI></UL><UL><LI>Lesions cause destruction of inner and outer membrane
</LI></UL><UL><LI>Small holes at specific and distinct location in the cell formed
</LI></UL><UL><LI>Sudden collapse of membrane integrity can be caused
</LI></UL><UL><LI>The membrane potential and osmotic gradient across envelop collapsed
</LI></UL><H2><SPAN class="mw-headline" id="Changes_in_our_plans"><B>Changes in our plans</B></SPAN></H2><P>As a team we decided to make changes to our original goal of creating a bio-safety net to be implemented into an oil degrading bacteria, due to difficulties we encountered and the time constraints on our project.
</P><H3><SPAN class="mw-headline" id="Difficulties">Difficulties</SPAN></H3><P>Oil is a mixture of chemicals composed mainly of hydrocarbons such as alkanes, carboxylic acids and various other aromatic compounds. As a direct result of this crude oil as a whole component cannot be relied upon to induce any promoter, since the promoters available to us may only be induced by certain chemical compounds (not a mixture) and because it is very difficult to target a specific component in crude oil. After assessing the feasibility of using a true oil degrading mechanism, we have concluded that it would be far too difficult to implement such a system into an E. Coli within the time period available to us.
</P><H3><SPAN class="mw-headline" id="The_changes">The changes</SPAN></H3><P>After identification of the difficulties, we decided to change our plan and instead of focusing on oil, we will use lactose to imitate ‘oil’. This lactose analogue can also be substituted with different subtracts with appropriate modification on the biobricks in initiating the mechanism. Lactose is used as the analogue owing to various reasons:
</P><UL><LI>Lactose operon is widely studied
</LI><LI>No reporter needed
</LI><LI>Lac-Z assay
</LI><LI>Optimal difficulty and achievable for 2 months’ time
</LI></UL><P>So, now our plan has changed from making an oil digesting bacterium to a lactose digesting bacterium. After the digestion is done, the bacteria will be programmed to kill themselves.
</P><H2><SPAN class="mw-headline" id="The_plasmid_design"><B>The plasmid design</B></SPAN></H2><DL><DT><I>PlacBAD</I></DT><DD> It is the promoter of the lac operon inducible by lactose or arabinose. Without the substrate, the promoter is suppressed by the repressor protein. Under this condition, the gene is considered as turned off. No expression of genes that are upstream the suppressed promoter can be observed. No proteins that are encoded by these genes are produced.
</DD><DT><I>cI</I></DT><DD> It is the coding sequence of cI protein. The cI protein acts as a repressor protein and bind to the promoter of the suicide gene. The suicide mechanism is then turned off without expression of gene <I>E</I> and production of E protein.
</DD><DT><I>Lac I</I></DT><DD> Encode for the inhibitor of the lac operon
</DD><DT><I>Lac Z</I></DT><DD> Encode for Beta-galactosidase which breakdowns lactose to glucose &amp; galactose and convert lactose to L-lactose which can activate the lac operon
</DD><DT><I>Lac A</I></DT><DD> Encode for lactose acetylase which transport lactose from medium to <I>E.Coli</I></DD><DT><I>Lac Y</I></DT><DD> Encode for lactose permease which regulates concentration lactose inside the cell
</DD><DT><I>pL</I></DT><DD> a promoter that can be inhibited by cI
</DD></DL><P><B>Reasons for choosing the lac operon <I>(since Lactose is now an analog for “crude oil molecules”)</I></B></P><OL><LI>   Found naturally in <I>E.coli</I></LI><LI>   Well-studied
</LI><LI>   Easy to understand how the mechanisms of the operon work
</LI><LI>   Present in the chromosomes of <I>E.coli</I>, so that recombination of a new promoter can be easily performed
</LI></OL><H3><SPAN class="mw-headline" id="Scenarios">Scenarios </SPAN></H3><UL><LI>In the presence of lactose, the lac operon and <I>cI</I> which were originally suppressed by a repressor are now turned on. Transcription, translation and post-modifications are carried out in order to express the gene. cI protein and the β-galactosidase <I>(hydrolytic enzyme catalyzing the hydrolysis of lactose)</I> are produced. The lactose starts to get digested. The cI protein represses the promoter of suicide gene <I>E</I>, <I>pL</I>, which in turn suppresses the expression of the suicide gene.
</LI></UL><UL><LI>The lactose in the cell culture is continuously digested. After all the lactose has been digested, the promoters for the <I>cI</I> gene and the lac operon are then suppressed. No β-galactozidase and the cI protein are produced. In the absence of cI protein, the suicide gene is turned on. The expression of gene <I>E</I><I>(production of E protein)</I> leads to the formation of holes in the cell membrane which causes cell death. The mechanism for which can be referred to in the previous section. 
</LI></UL><UL><LI>During the culture of the bacteria, arabinose is added to the LB agar <I>(the growth medium for the bacteria)</I>. The promoter for the cI and lac operon can also be activated by arabinose. Therefore, cI protein is produced continuously to suppress the expression of the suicide gene. This can prevent bacterial cell death under undesirable conditions.
</LI></UL><H3><SPAN class="mw-headline" id="Possible_flaws">Possible flaws</SPAN></H3><OL><LI>  Leaky expression: It might lead to the production of cl proteins which inhibit the  expression of the suicide gene. This means that bacteria might still survive after they have finished their function <I>(of digesting all the lactose)</I>.
</LI><LI>  The suicide genes do not work:  Even when there is no inhibition to the suicide gene, the killing mechanism might still not work due to a variety of reasons such as incorrect modifications to proteins, improper folding of the proteins or undesired mutations in the gene.
</LI><LI>  The efficacy of the killing mechanism <I>(i.e. time needed for cell-lysis)</I> If the efficacy  is too low the bacteria will not die as quickly as we want them to.
</LI></OL><H2><SPAN class="mw-headline" id="Results"> Results </SPAN></H2><P>The figure above shows the gel photo of the suicide gene <I>E</I>  
　　　　　　　　The gel photo of the plasmid <I>pET28a</I></P><P>The table shows the optical densities of the bacterial culture with the bacteria contain transformed plasmids
</P><P>The graph shows the changes in optical density versus time
</P><P>Optical density measures the concentration of bacteria in suspension, by measuring the amount of light that they collectively absorb. The blue and green curves represent samples taken from bacteria growth of cells where activity of the killing genes was suppressed <I>(i.e. with the presence of arabinose)</I>, because of this inhibition the unexpressed genes had no control on cell division and growth resulting in the relatively rapid increases to the curve. The curves that appear more flat are those where the expression of the killing genes were induced <I>(by the presence of IPTG)</I> meaning that cell lysis occurred, resulting in cell death. These curves show very low optical densities which indicate almost nonexistent bacterial growth in those samples, meaning the killing genes successfully work.
</P><H2><SPAN class="mw-headline" id="Possible_further_research"> Possible further research </SPAN></H2><OL><LI>  A key area for future research would be to engineer the plasmid in such a way that promoters could be more readily interchangeable, so that the killing genes can be induced by molecules relevant to their purposes <I>(e.g. heavy metals or rganic materials)</I>. 
</LI><LI>  With interchangeable promoters we would also need to make chosen bacteria code for certain proteins that are specific to intended functions <I>(e.g. cancer digesting proteins from bacteria)</I>. 
</LI><LI>  By making it easier for the promoters and coded proteins to be selected individually, the “bio-safety net” would become a universal safety mechanism that could be incorporated into engineered bacteria performing any function, and would be regulated by a user-defined substance.
</LI></OL></DIV></DIV><DIV class="cleaner"> </DIV><DIV id="footer"><P>By the HKU-Hong Kong iGEM team 2010</P><P>Sponsors:</P><P>Best viewed with firefox/google chrome</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:HKU-Hong_Kong/Project">http://2010.igem.org/Team:HKU-Hong_Kong/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:HKU-Hong_Kong/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:HKU-Hong_Kong/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:HKU-Hong_Kong/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:HKU-Hong_Kong/Project&amp;oldid=181661" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>