<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Arizona_State_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Arizona_State/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Arizona_State/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Arizona_State/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Arizona_State/Notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Arizona_State/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Arizona State/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><H2><SPAN class="mw-headline" id="June_07">June 07</SPAN></H2><H3><SPAN class="mw-headline" id="Strep">Strep</SPAN></H3><UL><LI> Transformation (LSE)
</LI><LI> Transformed DNA:
</LI><LI> lacZ (well 4:12G, I732019)
</LI><LI> p + lacO (well 1:6G, R0011)
</LI></UL><LI> Cells: neb10beta (donated)
</LI><LI> Protocol from: <A href="http://www.neb.com/nebecomm/products/productc3019.asp" class="external free" rel="nofollow">http://www.neb.com/nebecomm/products/productc3019.asp</A></LI><LI> Controls: puc19, no DNA (8 plates)
</LI><H2><SPAN class="mw-headline" id="June_08">June 08</SPAN></H2><H3><SPAN class="mw-headline" id="Strep_2">Strep</SPAN></H3><UL><LI> Transformation results
</LI><LI> puc19: growth
</LI></UL><UL><LI> negative control: no growth
</LI></UL><UL><LI> lacZ, lacO: possible small colonies
</LI></UL><UL><LI> liquid culture in amp media (100 ug / ml):
</LI><LI> no growth of lacZ, lacO
</LI><LI> growth of puc19
</LI></UL><H2><SPAN class="mw-headline" id="June_12">June 12</SPAN></H2><H3><SPAN class="mw-headline" id="General">General</SPAN></H3><UL><LI> DH5a Competent Cell Prep
</LI><LI> Streak plated cells on LB no amp plate, let grow overnight
</LI></UL><H2><SPAN class="mw-headline" id="June_13">June 13</SPAN></H2><H3><SPAN class="mw-headline" id="Strep_3">Strep</SPAN></H3><UL><LI> Transformation (LSE)
</LI><LI> Transformed DNA:
</LI><LI> lacZ (well 4:12G, I732019)
</LI><LI> p + lacO (well 1:6G, R0011)
</LI></UL><LI> Cells: DH5 alpha (donated)
</LI><LI> Protocol from: <A href="http://openwetware.org/wiki/Haynes:Assembly101" class="external free" rel="nofollow">http://openwetware.org/wiki/Haynes:Assembly101</A> (30 minute transformation)
</LI><LI> Controls: puc19, no DNA (8 plates)
</LI><H3><SPAN class="mw-headline" id="General_2">General</SPAN></H3><UL><LI> DH5a Chemically Competent cell prep
</LI><LI> Grew 2 seed colonies from streak plate in LB no amp
</LI><LI> Grew controls to test for contamination
</LI><LI> Both Seed colonies grew, no contamination present
</LI></UL><H2><SPAN class="mw-headline" id="June_14">June 14</SPAN></H2><H3><SPAN class="mw-headline" id="General_3">General</SPAN></H3><UL><LI> Competent cell prep
</LI><LI> Prepared CaCl2 buffer solution and CaCl2 glycerol buffer solution
</LI><LI> Grew seed colony in 400mL LB no amp
</LI></UL><H2><SPAN class="mw-headline" id="June_15">June 15</SPAN></H2><H3><SPAN class="mw-headline" id="General_4">General</SPAN></H3><UL><LI> Competent cell prep
</LI><LI> Centrifuged falcon test tubes containing liquid colonies
</LI><LI> Resuspended in CaCl2 buffer solution and incubated for 15 mins
</LI><LI> Centrifuged and resuspended in CaCl2 glycerol buffer solution
</LI><LI> Chilled overnight
</LI></UL><H2><SPAN class="mw-headline" id="June_16">June 16</SPAN></H2><H3><SPAN class="mw-headline" id="General_5">General</SPAN></H3><UL><LI> Competent cell prep
</LI><LI> Aliquotted 200uL into test tubes
</LI><LI> Stored in -80C
</LI></UL><H2><SPAN class="mw-headline" id="June_17">June 17</SPAN></H2><H3><SPAN class="mw-headline" id="General_6">General</SPAN></H3><UL><LI> Streak plated prepared competent cells on LB no amp plate
</LI><LI> Colonies observed
</LI><LI> No picture
</LI></UL><H2><SPAN class="mw-headline" id="June_18">June 18</SPAN></H2><H3><SPAN class="mw-headline" id="Strep_4">Strep</SPAN></H3><UL><LI> Digest: Lac Z (X and P) and p + lacO (S and P)
</LI><LI> Protocol: <A href="http://openwetware.org/wiki/Haynes:Assembly101" class="external free" rel="nofollow">http://openwetware.org/wiki/Haynes:Assembly101</A></LI><LI> Gel extraction - Sigma Aldrich Gel Kit
</LI></UL><H2><SPAN class="mw-headline" id="June_19">June 19</SPAN></H2><H3><SPAN class="mw-headline" id="Strep_5">Strep</SPAN></H3><UL><LI> Ligation
</LI><LI> P+lacO+Lac Z (samples still need to be tested)
</LI><LI> Control (No DNA)
</LI></UL><H3><SPAN class="mw-headline" id="General_7">General</SPAN></H3><UL><LI> Transformation (LSE)
</LI><LI> Transformed DNA:
</LI><LI> T7 promoter BBa_I712074
</LI><LI> Constitutive promoter BBa_J23102
</LI></UL><LI> Cells: DH5 alpha (donated)
</LI><LI> Protocol from: <A href="http://tools.invitrogen.com/content/sfs/manuals/subcloningefficiencydh5alpha_man.pdf" class="external free" rel="nofollow">http://tools.invitrogen.com/content/sfs/manuals/subcloningefficiencydh5alpha_man.pdf</A> (30 minute transformation)
</LI><LI> Controls: puc19, no DNA
</LI><LI> Plated 2 copies of each (100 ul, 250 ul) on LB amp plates.
</LI><LI>Made 50 LB Amp plates.
</LI><H2><SPAN class="mw-headline" id="June_20">June 20</SPAN></H2><H3><SPAN class="mw-headline" id="Strep_6">Strep</SPAN></H3><UL><LI> PCR
</LI><LI> Lac Z
</LI><LI> Primers: Standard BBa F and R primers (<A href="http://partsregistry.org/Primers/Catalog" class="external free" rel="nofollow">http://partsregistry.org/Primers/Catalog</A>)
</LI><LI> 1000 fold dilution DNA used
</LI><LI> Protocol 
</LI><LI> PCR water - 74.4 uL
</LI><LI> 1000 fold dilution DNA - 6 uL
</LI><LI> dNTP (10mM) - 2.4 uL
</LI><LI> F and R primers (10 uM) - 6 uL each
</LI><LI> 5x HF buffer - 24 uL
</LI><LI> Phusion DNA polymerase - 1.2 uL
</LI></UL><LI> Digest
</LI><LI> P+Lac O
</LI><LI> S and P
</LI><LI> Treated with Antartic Phosphatase (avoids religation) 
</LI><LI> Plated negative control on LB Amp plate
</LI><LI> Liquid cultures of T7 promoter and constitutive promoter
</LI><H3><SPAN class="mw-headline" id="General_8">General</SPAN></H3><UL><LI> Transformation (LSE)
</LI><LI> Transformed DNA:
</LI><LI> RBS (well 1:1H BBa_B0030)
</LI><LI> TetR GFP (well 2:8A Part:BBa_I13522)
</LI></UL><LI> Cells: DH5 alpha (donated)
</LI><LI> Protocol from: <A href="http://tools.invitrogen.com/content/sfs/manuals/subcloningefficiencydh5alpha_man.pdf" class="external free" rel="nofollow">http://tools.invitrogen.com/content/sfs/manuals/subcloningefficiencydh5alpha_man.pdf</A> (30 minute transformation)
</LI><LI> Controls: puc19, no DNA
</LI><H2><SPAN class="mw-headline" id="June_21">June 21</SPAN></H2><H3><SPAN class="mw-headline" id="General_9">General</SPAN></H3><UL><LI> Made Liquid Cultures of E.coli transformed with RBS B0030
</LI><LI> Made Liquid Cultures of E.coli transformed with TetR GFP
</LI><LI> miniprepped and nanodropped T7 promoter BBa_I712074 and Constitutive promoter BBa_J23102 liquid cultures
</LI><LI> liquid cultures:
</LI><LI> RBS1
</LI><LI> RBS2 (duplicate_
</LI><LI> GFP1
</LI><LI> puc19
</LI><LI> negative controls
</LI><LI> 5 ml LB amp
</LI><LI> overnight cultures
</LI></UL><LI> replated GFP1 &amp; 2 (duplicates)
</LI><LI> Nanodropped plasmid DNA samples
</LI><LI> Constitutive promoter 1: 2.554 ng/uL
</LI><LI> Constitutive promoter 2: 2.345 ng/uL
</LI><LI> T7 promoter 1: 3.369 ng/uL
</LI><LI> T7 promoter 2: 3.049 ng/uL
</LI><H2><SPAN class="mw-headline" id="June_22">June 22</SPAN></H2><H3><SPAN class="mw-headline" id="Strep_7">Strep</SPAN></H3><UL><LI> PCR clean up (Sigma Aldrich Kit) Lac Z samples
</LI><LI> Digest Lac Z with X and P
</LI><LI> Nano Drop the PCR clean samples:
</LI><LI> Lac O: 2.554 ug/uL
</LI><LI> Lac Z: 3.369 ug/uL
</LI></UL><LI> Gel
</LI><LI> PCR
</LI><LI> Lac Z (25ul rxn)
</LI><LI> Primers: Standard BBa F and R primers (<A href="http://partsregistry.org/Primers/Catalog" class="external free" rel="nofollow">http://partsregistry.org/Primers/Catalog</A>) 
</LI><LI> Varied Concentration DNA samples 
</LI><LI> Syzygy Mean Green MM
</LI><LI> Lac Z samples - 10 fold D, 100 fold D, 1000 fold D and Regular DNA.
</LI><UL><LI> PCR cleaned samples
</LI></UL><H3><SPAN class="mw-headline" id="General_10">General</SPAN></H3><UL><LI> Miniprepped liquid cultures: RBS (well 1:1H BBa_B0030) and TetR GFP (well 2:8A Part:BBa_I13522)
</LI><LI> Picked colonies:
</LI><LI> 1 colony from double terminator (dt1) plate
</LI><LI> 1 colony from t7 polymerase (pol1) plate
</LI><LI> 1 colony from puc19 plate (positive control)
</LI><LI> 1 colony from dh5a plate (negative control)
</LI></UL><LI> started liquid cultures of each colony (5 mL LB amp each)
</LI><H2><SPAN class="mw-headline" id="June_26">June 26</SPAN></H2><H3><SPAN class="mw-headline" id="General_11">General</SPAN></H3><UL><LI> Transformation:
</LI><LI> Transformed DNA:
</LI><LI> double terminator (B0017, 2:6K)
</LI><LI> T7 RNA polymerase (I715038, 2:15C)
</LI><LI> puc19, negative control
</LI></UL><LI> Protocol from: <A href="http://tools.invitrogen.com/content/sfs/manuals/subcloningefficiencydh5alpha_man.pdf" class="external free" rel="nofollow">http://tools.invitrogen.com/content/sfs/manuals/subcloningefficiencydh5alpha_man.pdf</A> (30 minute transformation)
</LI><LI> Cells: dh5a
</LI><H2><SPAN class="mw-headline" id="June_27">June 27</SPAN></H2><H3><SPAN class="mw-headline" id="General_12">General</SPAN></H3><UL><LI> 6-26 transformation results:
</LI><LI> Controls correct
</LI><LI> 2x terminator: ~19 colonies
</LI><LI> RNA pol: 1 colony
</LI></UL><LI> Liquid cultures including controls
</LI><H2><SPAN class="mw-headline" id="June_28">June 28</SPAN></H2><H3><SPAN class="mw-headline" id="General_13">General</SPAN></H3><UL><LI> Miniprepped double terminator (B0017, 2:6K) and T7 RNA polymerase (I715038, 2:15C) liquid cultures
</LI></UL><H2><SPAN class="mw-headline" id="July_2">July 2</SPAN></H2><H3><SPAN class="mw-headline" id="General_14">General</SPAN></H3><UL><LI> Cleaned up liquid waste
</LI><LI> Made SOB media
</LI></UL><H3><SPAN class="mw-headline" id="Magainin">Magainin</SPAN></H3><UL><LI> Finalized oligos for magainin construct
</LI></UL><H2><SPAN class="mw-headline" id="July_3">July 3</SPAN></H2><H3><SPAN class="mw-headline" id="General_15">General</SPAN></H3><UL><LI> Third attempt of putting the bricks together.  And samples were plated on X-gal plates to show that the construct worked.
</LI></UL><UL><LI> Autoclaved SOB media
</LI><LI> Added glucose to make SOC media
</LI><LI> Nanodropped double terminator (B0017, 2:6K) [DT1: 24.5, DT2: 29.6] and T7 RNA polymerase (I715038, 2:15C) [P1: 64.6, P2: 55.3] liquid cultures
</LI></UL><H2><SPAN class="mw-headline" id="July_12">July 12</SPAN></H2><H3><SPAN class="mw-headline" id="Magainin_2">Magainin</SPAN></H3><UL><LI> Searched for split beta gal fragment. Alpha fragment (BBa_I732006) was found on the distribution plates and transformed (NEB 10B).
</LI></UL><H2><SPAN class="mw-headline" id="July_14">July 14</SPAN></H2><H3><SPAN class="mw-headline" id="Magainin_3">Magainin</SPAN></H3><UL><LI> Designed primers to remove the stop codon on the alpha fragment. Also attained two flexible linkers from Dr. Haynes and transformed them.
</LI></UL><H2><SPAN class="mw-headline" id="July_15">July 15</SPAN></H2><H3><SPAN class="mw-headline" id="Magainin_4">Magainin</SPAN></H3><UL><LI> Ran a miniprep of Bgal alpha 1&amp;2, T7 and PSV
</LI><LI> Procedure from GenElute HP PLasmid Miniprep Kit (Sigma-Aldrich)
</LI></UL><H2><SPAN class="mw-headline" id="July_16">July 16</SPAN></H2><H3><SPAN class="mw-headline" id="Strep_8">Strep</SPAN></H3><UL><LI> Transformation of PLflex (part: BBa_J176040) and PLflex4 (part: BBa_J176130) linkers. 
</LI></UL><UL><LI> Incubate at 37C for 8 hours for DH 5alpha turbo cells
</LI><LI> Inoculated 5 mL LB/AMP media in 15 mL culture tubes and added the colony. Incubated in 37C overnight.
</LI></UL><H2><SPAN class="mw-headline" id="July_17">July 17</SPAN></H2><H3><SPAN class="mw-headline" id="Strep_9">Strep</SPAN></H3><UL><LI> Miniprep of pLFlex DH5alpha and pLFlex 4 DH5alpha
</LI><LI> Procedure comes from Zymo Research's Zyppy Kit. 
</LI><LI> Ran the spectrophotometer to determine the concentration for the miniprep DNA for the linker pLFlex and linker pLFlex 4
</LI><LI> Concentrations were: 
</LI><LI> pLFlex: 73.792 ng/uL
</LI><LI> pLFlex 4: 38.587 ng/uL
</LI></UL><LI> Absorbance ratios were:
</LI><LI> pLFlex: 1.864
</LI><LI> pLFlex 4: 1.851
</LI><LI> Ran the miniprep for pLFlex and pLFlex 4 again.
</LI><LI> Same procedure
</LI><LI> Ran the spectrophotometer again
</LI><LI> Concentrations were: 
</LI><LI> pLFlex: 48.095 ng/uL
</LI><LI> pLFlex 4: 18.214 ng/uL
</LI><LI> Absorbance ratios were:
</LI><LI> pLFlex: 1.79
</LI><LI> pLFlex 4: 1.85
</LI><LI> Transformation was successful but 8-9 hour growth period was not sufficient for DH5 alpha.
</LI><H2><SPAN class="mw-headline" id="July_23">July 23</SPAN></H2><H3><SPAN class="mw-headline" id="Strep_10">Strep</SPAN></H3><UL><LI> Plated BBa_K283010 (Streptavidin) on LB amp plate from the agar stab provided from the iGEM head-quarters.
</LI><LI> Incubated the dish overnight at 37 C.
</LI></UL><UL><LI> Plate 1: pipet tip, colonies are in wells, neither distinct nor segregated
</LI><LI> Plate 2: inoculating loop, numerous non-distinct colonies on the plate
</LI></UL><H2><SPAN class="mw-headline" id="July_24">July 24</SPAN></H2><H3><SPAN class="mw-headline" id="Topo">Topo</SPAN></H3><UL><LI> Plasmids arrived courtesy of University of Pennsylvania School of Medicine
</LI><LI> pET29a vectors containing coding sequence for Topoisomerase mutants CSCS and CSCS D168A described here
</LI></UL><H2><SPAN class="mw-headline" id="July_25">July 25</SPAN></H2><H3><SPAN class="mw-headline" id="Strep_11">Strep</SPAN></H3><UL><LI> Ran a miniprep of streptavidin BBa_K283010
</LI><LI> Protocol from Zymo Research's Zyppy Plasmid Miniprep Kit
</LI></UL><H3><SPAN class="mw-headline" id="Topo_2">Topo</SPAN></H3><UL><LI> Tranformed CSCS topo 0 plasmid and CSCS D168A topo into DH5a Turbo cells (with neg control and Puc19 neg control)
</LI><LI> Plated on Kanamycin plates
</LI></UL><H2><SPAN class="mw-headline" id="July_26">July 26</SPAN></H2><H3><SPAN class="mw-headline" id="Topo_3">Topo</SPAN></H3><UL><LI> Picked colonies colonies from Topo 0 and Topo D168A and grew liquid cultures in Kan media
</LI></UL><H2><SPAN class="mw-headline" id="July_27">July 27</SPAN></H2><H3><SPAN class="mw-headline" id="Topo_4">Topo</SPAN></H3><UL><LI> Miniprepped liquid colonies and nanodropped.
</LI><LI> Plasmid concentrations
</LI><LI> Topo O: 63.7 ng/uL 
</LI><LI> Topo D168A1: 125.4 ng/uL</LI><LI> Topo D168A2: 116.6 ng/uL
</LI></UL><H2><SPAN class="mw-headline" id="July_30">July 30</SPAN></H2><H3><SPAN class="mw-headline" id="General_16">General</SPAN></H3><UL><LI> Prepared Kan Media and Kan Plates
</LI></UL><H2><SPAN class="mw-headline" id="July_31">July 31</SPAN></H2><H3><SPAN class="mw-headline" id="Topo_5">Topo</SPAN></H3><UL><LI> PCR amplified polylinker sequence of Topo plasmid with Promega GoTaq protocol
</LI><LI> Used pET29a upstream forward primer and T7 terminator reverse primer
</LI></UL><H2><SPAN class="mw-headline" id="August_3">August 3</SPAN></H2><H3><SPAN class="mw-headline" id="Topo_6">Topo</SPAN></H3><UL><LI> Submitted pET29a Topoisomerase plasmid to Biodesign for sequencing
</LI><LI> Resuspended GFPT1 and GFPT2 oligos with molecular grade (nuclease-free) H2O.
</LI><LI> Final Concentration 100uM
</LI><LI> (gfpt1 top1, gfpt2 top1, gfpt1 top2, gftp2 top2, gfpt1 bot1, gfpt2 bot1, gfpt1 bot2, gfpt2 bot2)
</LI><LI> (3uL of each oligo + 2uL 10x annealing buffer, 6uL molecular grade H2O. 20uL Reactions)
</LI></UL><LI> Heated for 5 minutes at 100C. Let cool to room temperature on the heating block, stored at -20C.
</LI><LI> Digested BBa_I13522 with XbaI and PstI.
</LI><LI> Attempted ligating annealed oligos into a digested plasmid from Ryan (realized it was cut with E and P).
</LI><H2><SPAN class="mw-headline" id="August_6">August 6</SPAN></H2><H3><SPAN class="mw-headline" id="Topo_7">Topo</SPAN></H3><UL><LI> Annealed oligos for GFPT1 and GFPT2 (target probes)
</LI><LI> Ligated oligos with digested GFP plasmid (BBa_I13522)
</LI><LI> Transformed into competent DH5alpha
</LI><LI> Added SOC and incubated at 37C for 15 minutes.
</LI><LI> Plated on amp treated plates.
</LI></UL><H2><SPAN class="mw-headline" id="August_7">August 7</SPAN></H2><H3><SPAN class="mw-headline" id="Topo_8">Topo</SPAN></H3><UL><LI> Only one colony on each plate (both were white)
</LI><LI> Picked colonies and started 5mL LB amp cultures of each, stored at 37C
</LI><LI> Stored plates in 37C
</LI></UL><H2><SPAN class="mw-headline" id="August_8">August 8</SPAN></H2><H3><SPAN class="mw-headline" id="Topo_9">Topo</SPAN></H3><UL><LI> Picked the colonies again and started new liquid cultures (5mL LB amp).
</LI><LI> Discarded cultures for 8/7/12
</LI></UL><H2><SPAN class="mw-headline" id="August_9">August 9</SPAN></H2><H3><SPAN class="mw-headline" id="Topo_10">Topo</SPAN></H3><UL><LI> Miniprepped 3mL of each 8/8/12 culture and nanodropped:
</LI><LI> gfpt1 - 155 ng/uL
</LI><LI> gfpt2 - 114 ng/uL
</LI></UL><LI> Digested gfpt1 and gfpt2 with X and P
</LI><LI> Ran on a 1% agarose gel with the digested GFP plasmid and hyperladder V
</LI><UL><LI> Didn't work
</LI><LI> Made glyercol stocks with aliquot of the remaining liquid cultures
</LI></UL><H2><SPAN class="mw-headline" id="August_10">August 10</SPAN></H2><H3><SPAN class="mw-headline" id="Magainin_5">Magainin</SPAN></H3><UL><LI> PCR of Alpha-4, 1-omega, omega, and alpha fragments using corrected primers
</LI></UL><H2><SPAN class="mw-headline" id="August_13">August 13</SPAN></H2><H3><SPAN class="mw-headline" id="Magainin_6">Magainin</SPAN></H3><UL><LI> Ran gel of split beta gal fragments. Confirmed 3 out of the 4 fragments except for the alpha-4 fragment. </LI></UL><H3><SPAN class="mw-headline" id="Topo_11">Topo</SPAN></H3><UL><LI> Prepared sequencing samples
</LI><LI> Sample w/ Primer: 
</LI><LI> GFPT1 w/ VF2 GFPT1 w/ VR GFPT2 w/ VF2 GFPT2 w/ VR
</LI><LI> 200ng of DNA + 16 pmol of primer
</LI></UL><LI> Annealed oligos again GFPT1/2
</LI><LI> Repeated ligation of oligos with digested GFP plasmid (BBa_I13522)
</LI><LI> Followed Haynes assembly protocol instead of standard DH5alpha protocol. (<A href="http://openwetware.org/wiki/Haynes:Assembly101" class="external free" rel="nofollow">http://openwetware.org/wiki/Haynes:Assembly101</A>)
</LI><LI> Transformed ligations into competent DH5alpha
</LI><LI> Plated on amp treated plates
</LI><H2><SPAN class="mw-headline" id="August_14">August 14</SPAN></H2><H3><SPAN class="mw-headline" id="Topo_12">Topo</SPAN></H3><UL><LI> Took pictures of plates
</LI><LI> Green-white screened plates
</LI><LI> Picked 4 white colonies from each of gfpt1/2 plates
</LI><LI> Made 5mL LB amp cultures of each colony
</LI><LI> Delivered GFPT1/2 dna samples to biodesign for sequencing (samples from 8/13/12)
</LI></UL><H3><SPAN class="mw-headline" id="Magainin_7">Magainin</SPAN></H3><UL><LI> Assembled magainin insert via Overlapping oligo assembly
</LI><LI> Digested pUC 19 plasmid with EcoRI and PstI
</LI><LI> Transformed Magainin insert into digested pUC 19 plasmid. Failed. Probably too much X-gal on plate.
</LI></UL><P>Ran gel for the beta gal alpha-4 fragment. Failed. Fragment not in the correct size-band. </P><H2><SPAN class="mw-headline" id="August_15">August 15</SPAN></H2><H3><SPAN class="mw-headline" id="Topo_13">Topo</SPAN></H3><UL><LI><A href="http://129.219.2.10/home2/dnalims/fragment/32/21/pfasta/topo0_T7Term.seq" class="external text" rel="nofollow">Topo 0</A>, <A href="http://129.219.2.10/home2/dnalims/fragment/32/21/pfasta/D168Atopo1_T7Term.seq" class="external text" rel="nofollow">D168A Topo1</A>, and <A href="http://129.219.2.10/home2/dnalims/fragment/32/21/pfasta/D168Atopo2_T7Term.seq" class="external text" rel="nofollow">D168A Topo2</A> sequence results
</LI><LI> Miniprepped 3mL of each liquid culture of GFPT1/2
</LI><LI> Prepared glycerol stocks using 100uL of each liquid culture
</LI><LI> Nanodropped samples:
</LI><LI> GFPT1-1 - 172.6 ng/uL 
</LI><LI> GFPT1-2 - 203.7 ng/uL
</LI><LI> GFPT1-3 - 197.4 ng/uL
</LI><LI> GFPT1-4 - 178.9 ng/uL
</LI><LI> GFPT2-1 - 107.3 ng/uL
</LI><LI> GFPT2-2 - 131.2 ng/uL
</LI><LI> GFPT2-3 - 145.5 ng/uL 
</LI><LI> GFPT2-4 - 172.0 ng/uL
</LI></UL><H2><SPAN class="mw-headline" id="August_16">August 16</SPAN></H2><H3><SPAN class="mw-headline" id="Magainin_8">Magainin</SPAN></H3><UL><LI> Replated magainin insert + plasmid into the grid. Failed. All blue colonies meaning that no insert.</LI></UL><P>Tried gel for all gel-isolated fragments. Failed. Did not get a band in the 2000 bp region. Only got things below 200 bp.</P><H2><SPAN class="mw-headline" id="August_17">August 17</SPAN></H2><H3><SPAN class="mw-headline" id="Topo_14">Topo</SPAN></H3><UL><LI> GFPT1 sequence confirmed
</LI><LI> Prepared aliquots of GFPT2 minipreps from 8/15/12 for sequencing
</LI><LI> Delivered GFPT2 samples to biodesign for sequencing
</LI></UL><H2><SPAN class="mw-headline" id="August_18">August 18</SPAN></H2><H3><SPAN class="mw-headline" id="Strep_12">Strep</SPAN></H3><UL><LI> PCRâed Streptavidin (biobrick) and omega fragment of beta (PSV plasmid) and confirmed size on a gel. 
</LI></UL><P>Streptavidin was digested with E and S. Also, both flexible linkers attained from Dr. Haynes were digested with E and X and dephosphorlated. 
</P><H2><SPAN class="mw-headline" id="August_19">August 19</SPAN></H2><H3><SPAN class="mw-headline" id="Strep_13">Strep</SPAN></H3><UL><LI> Restricted PLFlex and PLFlex 4 with ecori and xbai.
</LI><LI> Restricted Bgala with Xbai and psti
</LI><LI> Used Hayne's Lab Protocol
</LI><LI> GFPT2 sequences confirmed
</LI></UL><H2><SPAN class="mw-headline" id="August_22">August 22</SPAN></H2><H3><SPAN class="mw-headline" id="Strep_14">Strep</SPAN></H3><UL><LI> Transformed the two constructs.
</LI></UL><UL><LI> The two constructs were confirmed on gel and were ligated together to attain Streptavidin + pLFlex and Streptavidin+pLFlex4.
</LI></UL><H2><SPAN class="mw-headline" id="August_27">August 27</SPAN></H2><H3><SPAN class="mw-headline" id="Topo_15">Topo</SPAN></H3><UL><LI> Revived GFPT1 (from 8/9/12) and GFPT2 (2-2 from 8/15/12) cultures from glycerol scrapes
</LI><LI> Made 4mL cultures in LB Amp
</LI></UL><H2><SPAN class="mw-headline" id="August_29">August 29</SPAN></H2><H3><SPAN class="mw-headline" id="Topo_16">Topo</SPAN></H3><UL><LI> Discarded GFPT1/2 cultures from 8/27/12
</LI><LI> Revived GFPT1 (from 8/9/12) and GFPT2 (2-2 from 8/15/12) cultures from glycerol scrapes
</LI><LI> Made 4mL cultures in LB Amp
</LI><LI> Digested GFPT1/2 with X and S
</LI><LI> Ran a 1% agarose gel with GFPT1/2 digestions
</LI></UL><UL><LI> Cut out inserts and GFPT2 backbone and stored in 4C for gel extraction and tandum repeat assembly experiments
</LI></UL><H2><SPAN class="mw-headline" id="August_30">August 30</SPAN></H2><H3><SPAN class="mw-headline" id="Topo_17">Topo</SPAN></H3><UL><LI> Prepared extra glyercol stocks of GFPT1/2 cultures from 8/29/12
</LI><LI> Miniprepped 3mL of each culture, stored at -20C
</LI></UL><H2><SPAN class="mw-headline" id="September_2">September 2</SPAN></H2><H3><SPAN class="mw-headline" id="Strep_15">Strep</SPAN></H3><UL><LI> Digested strep+pLFlex4(L4) with S and P and pLflex(L) with E and X and inserted alpha and omega fragments of beta gal.
</LI><LI> Transformed these constructs
</LI></UL><H2><SPAN class="mw-headline" id="September_10">September 10</SPAN></H2><H3><SPAN class="mw-headline" id="Strep_16">Strep</SPAN></H3><UL><LI> Assembled:
</LI><LI> Strep+L4+Î± which was a success
</LI><LI> Strep+L4+Ï which was a success
</LI><LI> Strep+L which was a success
</LI></UL><LI>All digested samples were confirmed on a 1% agarose gel
</LI><H2><SPAN class="mw-headline" id="September_18">September 18</SPAN></H2><H3><SPAN class="mw-headline" id="Strep_17">Strep</SPAN></H3><UL><LI> Assembled:
</LI><LI> Prsf Duest+Strep+L4+Î± which failed
</LI><LI> Prsf Duet+Strep+L4+Ï which failed
</LI><LI> Strep+L+Î± which was a success
</LI><LI> Strep+L4+Ï which was a success
</LI></UL><LI> All digested samples were confirmed on a 1% agarose gel
</LI><LI> No picture of the gel.
</LI><H2><SPAN class="mw-headline" id="September_19">September 19</SPAN></H2><H3><SPAN class="mw-headline" id="Strep_18">Strep</SPAN></H3><UL><LI> Assembled
</LI><LI> Prsf Duest+Strep+L4+Î± which failed
</LI><LI> Prsf Duest+Strep+L4+Ï which failed
</LI><LI> Prsf Duest+Strep+L+Î± which failed
</LI><LI> Prsf Duest+Strep+L+Ï which failed
</LI></UL><LI> All digested samples were confirmed on a 1% agarose gel
</LI><H3><SPAN class="mw-headline" id="Topo_18">Topo</SPAN></H3><UL><LI> Set up VF2/VR endpoint PCR for double transform minipreps
</LI><LI> 1-1, 1-1I, 1-2, 1-2I, 1-3, 1-3I, 2-1, 2-1I, 2-2, 2-2I, 2-3, 2-3I, GFPT1 (positive controls), GFPT2 (positive controls)
</LI></UL><LI> Annealing temp set to 55C for 25 cycles
</LI><LI> Resuspended GFPT1 probe and GFPT2 probe oligos in molecular grade H2O (Final concentration: 100uM), stored at -20C
</LI><H2><SPAN class="mw-headline" id="September_21">September 21</SPAN></H2><H3><SPAN class="mw-headline" id="Strep_19">Strep</SPAN></H3><UL><LI> Miniprepped all the streps samples and digested them.
</LI></UL><H2><SPAN class="mw-headline" id="September_22">September 22</SPAN></H2><H3><SPAN class="mw-headline" id="Strep_20">Strep</SPAN></H3><UL><LI> Assembles with new Prsf samples
</LI><LI> Prsf Duest+Strep+L4+Î± which failed
</LI><LI> Prsf Duest+Strep+L4+Ï which failed
</LI><LI> Prsf Duest+Strep+L+Î± which failed
</LI><LI> Prsf Duest+Strep+L+Ï which failed
</LI></UL><LI> Failed because outgrowth step want provided.
</LI><LI> All digested samples were confirmed on a 1% agarose gel
</LI><H2><SPAN class="mw-headline" id="September_25">September 25</SPAN></H2><H3><SPAN class="mw-headline" id="Topo_19">Topo</SPAN></H3><UL><LI> Resuspended pSB1A2 FWD and pSB1A2 REV (amp resistance primers) oligos in molecular grade H2O (Final concentration: 100uM), stored at -20C
</LI><LI> Prepared 1.6uM dilutions (500uL)
</LI><LI> Did endpoint PCR using pSB1A2 primer pair on
</LI><LI> Topo, Topo IPTG, Topo D168A, Topo D168A IPTG, 1-1, 1-1I, 2-1, 2-1I, GFPT1, GFPT2
</LI></UL><LI> Did endpoint PCR using VF2/VR primer pair on
</LI><LI> Topo, Topo IPTG, Topo D168A, Topo D168A IPTG
</LI><LI> Annealing temp 55C for 25 cycles, stored products at -20C
</LI><LI> Made 3mL liquid cultures of the shipping vector (pSB1C3 with RFP insert) in DH5alpha in chloramphenicol resistant LB
</LI><LI> Made 10mL liquid cultures of:
</LI><LI> Topo in kanamycin
</LI><LI> topo D168A in kanamycin
</LI><LI> topo + GFPT1 in kanamycin + ampicillin
</LI><LI> topo + GFPT2 in kanamycin + ampicillin
</LI><LI> stored @ 37C
</LI><H2><SPAN class="mw-headline" id="September_26">September 26</SPAN></H2><H3><SPAN class="mw-headline" id="Strep_21">Strep</SPAN></H3><UL><LI> Assembled
</LI><LI> Prsf Duest+Strep+L4+Î± which was a Success
</LI><LI> Prsf Duest+Strep+L4+Ï which was a Success
</LI><LI> Prsf Duest+Strep+L+Î± which was a Success
</LI><LI> Prsf Duest+Strep+L+Ï which was a Success
</LI></UL><H3><SPAN class="mw-headline" id="Topo_20">Topo</SPAN></H3><UL><LI> Picked two new colonies for topo + GFPT2 and grew 10mL cultures of amp+kan LB broth for each
</LI><LI> Ran a gel with Hyperladder I, GFPT1 pSB1A2, GFPT2 pSB1A2, Topo pSB1A2, Topo D168A
</LI></UL><UL><LI> Ran a gel with PCR samples from 9/19 and 9/25 Hyperladder I, 1-1, 1-1I, 2-1, 2-1I, 2-1I(VF), 2-1(VF), 1-1I(VF), 1-1(VF), Hyperladder I
</LI></UL><UL><LI> Samples in wells 2-5 used the pSB1A2 primer pair. Samples in wells 6-9 used VF2/VR primer pair.
</LI><LI> Revived GFPT1/2 from glycerol stocks (from 8/9 and 8/15) in 5mL LB amp each.
</LI><LI> Nanodropped miniprepped DNA samples:
</LI><LI> pSB1C3 I - 253.75 ng/uL
</LI><LI> pSB1C3 II - 258.38 ng/uL
</LI><LI> Topo - 24.84 ng/uL
</LI><LI> Topo I - 17.33 ng/uL
</LI><LI> Topo D168A - 7.24 ng/uL
</LI><LI> Topo D168A I - 15.62 ng/uL
</LI><LI> 1-1 - 16.49 ng/uL
</LI><LI> 1-1I - 16.46 ng/uL
</LI><LI> 2-1 - 142.05 ng/uL
</LI><LI> 2-1I - 16.98 ng/uL
</LI></UL><LI> Picked new colonies from Topo, Topo D168A, Topo + G1, Topo + G2 and made 1mL colonies in their respective medias
</LI><LI> Stored at -37C
</LI><H2><SPAN class="mw-headline" id="September_27">September 27</SPAN></H2><H3><SPAN class="mw-headline" id="Strep_22">Strep</SPAN></H3><UL><LI> Testing R0011 promoter. Digested R0011 with S and P generated the following assembly:
</LI><LI> R0011+Strep+L4+Î± which failed
</LI><LI> R0011+Strep+L4+Ï which failed
</LI><LI> R0011+Strep+L+Î± which failed
</LI><LI> R0011+Strep+L+Ï which was a Success
</LI></UL><H3><SPAN class="mw-headline" id="Topo_21">Topo</SPAN></H3><UL><LI> Prepared serial dilutions of GFPT2 plasmid 1:10, 1:100, 1:1000, 1:10000
</LI><LI> Prepared primer mixes for pSB and VF/VR primers
</LI><LI> Prepared realtime PCR plate, ran RT-qPCR with annealing temp 57
</LI><LI> Miniprepped GFPT1/2 cultures from 9/26
</LI><LI> Nanodropped Miniprepped DNA:
</LI><LI> GFPT1 - 276.12 ng/uL
</LI><LI> GFPT2 - 219.84 ng/uL
</LI></UL><LI> Used 100uL of each 1mL culture from 9/26 to seed 10 mL cultures in their respective media
</LI><LI> Added 10uL of 1M IPTG to each culture ~4 hours after seeding
</LI><LI> Removed cells from 37C ~4 hours after IPTG inducing
</LI><LI> Pelleted and lysed following the bugbuster protocol (<A href="http://openwetware.org/wiki/User:Behzad_Damadzadeh/Notebook/PcTF_Subcloning_in_E-coli/2012/05/22" class="external free" rel="nofollow">http://openwetware.org/wiki/User:Behzad_Damadzadeh/Notebook/PcTF_Subcloning_in_E-coli/2012/05/22</A>) (used lysonase for Topo and Topo D168A only)
</LI><LI> HIS purified proteins using the Zymo HIS purification kit
</LI><LI> Digested pSB1C3 plasmid with X and P
</LI><LI> Sent protein samples (HIS purified samples from 9/19, 20uL) and ssDNA control (GFPT1/2 probe oligos 20uL @ 10uM) for mass spec
</LI><H2><SPAN class="mw-headline" id="September_28">September 28</SPAN></H2><H3><SPAN class="mw-headline" id="Topo_22">Topo</SPAN></H3><UL><LI> Tested another promoter (P2547) for the construct and repeated the assembly from 9/27.
</LI><LI> Resuspended 8 new oligos, final volume 100 uM each
</LI><LI> Annealed pet29 top/bot oligos
</LI><LI> Redid the RT-PCR, 3x primer concentration, added 1:1 plasmid concentration
</LI><LI> Made 1.6uM aliquots of primer stocks 1-7 (including topo add X primer previously ordered)
</LI><LI> Diluted aliquot of Topo D168A plasmid 4:10
</LI><LI> Diluted PSV plasmid 2:20
</LI><LI> Performed Endpoint PCR on:
</LI><LI> Topo D168A using primers 1,2 and primers 1,3
</LI></UL><LI> PSV using primers 4-5 and primers 6-7
</LI><LI> 4 samples with low primer concentration, 4 samples with twice as much primer (labelled 'H')
</LI><LI> Miniprepped Topo1 2, Topo1 3, Topo1 4 (biobricked Topo D168A without T7, multiple colonies from ligation into shipping vector)
</LI><LI> Nanodropped:
</LI><LI> Topo1 2 - 63.63 ng/uL
</LI><LI> Topo1 3 - 75.07 ng/uL
</LI><LI> Topo1 4 - 184.93 ng/uL
</LI><LI> 1mL chloramphenicol cultures of GFPT1/GFPT2 in shipping vector prepared
</LI><LI> Digested Topo1 2, Topo1 3, Topo1 4 with E and P
</LI><LI> Ran digested Topo plasmids on 1% agarose gel with Hyperladder I
</LI><UL><LI> Revived cultures of 2xGFPT1, 2xGFPT2, Topo, and Topo D168A (4 mL cultures each in their respective medias)
</LI><LI> T5 exonuclease treated miniprepped plasmids (1-1, 1-1I)
</LI><LI> A1 1-1I + t5
</LI><LI> B1 1-1 + t5
</LI><LI> A2 1-1I untreated
</LI><LI> B2 1-1 untreated
</LI></UL><H2><SPAN class="mw-headline" id="September_29">September 29</SPAN></H2><H3><SPAN class="mw-headline" id="Strep_23">Strep</SPAN></H3><UL><LI> Assembly was successful as liquid cultures were made. It was realized that due to the lack of beta gal negative strains the omega fragment couldnât be tested and R0011 wouldn't code for the protein due to the lack of an RBS. 
</LI><LI> J61100 and J61101 constitutive promoters were extracted from the distribution plates and an additional promoter, pLux-Lac+RBS, were used to prove that the probe functions as desired.
</LI><LI> Following assemblies were set up:
</LI><LI> Plated on 50ul xgal LB amp Plates
</LI><LI> J61100+ Î±
</LI><LI> J61100+ strep+L+Î± (1a)
</LI><LI> J61100+ strep+L+Î± (2c)
</LI><LI> J61100+ strep +L +Î±(2b)
</LI><LI> J61101+ Î±
</LI><LI> J61101+strep+L+Î±(1a)
</LI><LI> J61101+strep+L+Î± (2c)
</LI><LI> J61101+strep+L+ Î± (2b)
</LI><LI> plux-lac1+rbs+Î±
</LI><LI> plux-lac1+rbs+strep+L+Î±(1a)
</LI><LI> plux-lac1+rbs+strep+L+Î±(2c)
</LI><LI> plux-lac1+rbs+strep+L+Î±(2b)
</LI><LI> plux-lac2+rbs+Î±
</LI><LI> plux-lac2+rbs+strep+L+Î± (1a)
</LI><LI> plux-lac2+rbs+strep+L+Î± (2c)
</LI><LI> plux-lac2+rbs+strep+L+Î± (2b)
</LI></UL><LI> Following were plated on 50ul X-Gal and 25ul IPTG LB AMP plates (no growth expected)
</LI><LI> plux-lac1+rbs+Î±      
</LI><LI> plux-lac2+rbs+strep+L+Î± (1a)
</LI><LI> plux-lac2+rbs+strep+L+Î± (2c)  
</LI><LI> plux-lac2+rbs+strep+L+Î± (2b)
</LI><LI> Plate pictures:
</LI><H3><SPAN class="mw-headline" id="Topo_23">Topo</SPAN></H3><UL><LI> Miniprepped GFPT1, GFPT2, Topo, and Topo D168A (Topo cultures separated into 3mL and 1mL minipreps)
</LI><LI> Nanodropped miniprepped DNA:
</LI><LI> GFPT1 I - 275.29 ng/uL
</LI><LI> GFPT1 II - 101.08 ng/uL
</LI><LI> GFPT2 I - 222.11 ng/uL
</LI><LI> GFPT2 II - 230.52 ng/uL
</LI><LI> Topo I - 117.97 ng/uL
</LI><LI> Topo II - 70.28 ng/uL
</LI><LI> Topo D168A I - 69.47 ng/uL
</LI><LI> Topo D168A II - 51.87 ng/uL
</LI></UL><LI> HIS purified crude lysates from 9/27/12 (Topo, Topo D168A, Topo + GFPT1, Topo + GFPT2, all IPTG induced)
</LI><LI> Digested Topo I plasmid (pet29a) with E and X
</LI><LI> Ran a gel with Hyperladder I, 1-2 1-3 4-5 and 6-7 PCR products, pet29a digestion, and pSB1C3 digestion
</LI><UL><LI> Confirmed that PCR made amplicons
</LI></UL><UL><LI> Excised bands for digested pet29a and pSB1C3 plasmids
</LI><LI> Gel extracted 4 gel fragments (2 wells per sample: digested pSB1C3 plasmid, digested pet29a plasmid)
</LI><LI> Nanodropped gel extractions:
</LI><LI> Digested pSB1C3 - 25.54 ng/uL
</LI><LI> Digested pet29a - 22.95 ng/uL
</LI></UL><LI> Set up a bradford assay of topo, topo D168A, topo + G1, topo + G2 (5uL protein, 10uL protein, 20uL protein + 200uL reagent)
</LI><LI> Used ~100uL aliquot of BL21 competent glycerol stock to seed 10mL of LB medium (no antibiotic), stored at 37C
</LI><LI> Ran 1% agarose gel with samples: A1, B1, A2, B2 and Hyperladder I
</LI><UL><LI> Did bug buster protocol to lyse BL21 control culture (used lysonase)
</LI><LI> Ligated digested pet29a with pet29 top/bot annealed oligos
</LI><LI> Transformed ligation using invitrogen DH5alpha transformation protocol
</LI><LI> Prepared LB kanamycin plates
</LI><LI> plated transformed ligations on prewarmed kanamycin plates
</LI></UL><H2><SPAN class="mw-headline" id="September_30">September 30</SPAN></H2><H3><SPAN class="mw-headline" id="Topo_24">Topo</SPAN></H3><UL><LI> All assemblies were successful and went as planned. Blue colonies were picked to generate liquid cultures and streak plates to have a better visual result. 
</LI><LI> Pet29a plates did not grow
</LI><LI> Kinase treated pet29a oligos
</LI><LI> Annealed kinase treated pet29a oligos
</LI><LI> Ligated digested pet29a (from gel extraction) with kinase treated oligos
</LI><LI> Transformed ligations into DH5alpha, used topo plasmid as a positive control
</LI><LI> Plated transformations on kanamycin plates and stored overnight at 37C
</LI><LI> HIS purified BL21 control crude lysate
</LI><LI> Set up a bradford assay with:
</LI><LI> uninduced &amp; induced topo protein extractions from 9/19
</LI><LI> uninduced &amp; induced topo D168A protein extractions from 9/19
</LI><LI> BL21 control lysate
</LI><LI> Topo, Topo D168A, Topo + G1, Topo + G2 from 9/27
</LI></UL><UL><LI> All samples prepared (10uL protein, 20uL protein + 200uL reagent)
</LI><LI> Miniprepped GFPT1 1,2,3 and GFPT2 (17,18,26) (~600uL of each) (1mL liquid cultures made from colonies on the chloramphenicol plates of GFPT1 and GFPT2 ligated into the shipping vector)
</LI><LI> Nanodropped:
</LI><LI> GFPT1-1 - 38.5 ng/uL
</LI><LI> GFPT1-2 - 77.6 ng/uL
</LI><LI> GFPT1-3 - 74.8 ng/uL
</LI><LI> GFPT2-17 - 61.6 ng/uL
</LI><LI> GFPT2-18 - 65.9 ng/uL
</LI><LI> GFPT2-26 - 64.2 ng/uL
</LI></UL><LI> Ran a 1% agarose gel with 1-1, 1-2, 1-3, 2-17, 2-18, 2-26 plasmid miniprep samples and Hyperladder I
</LI><UL><LI> Prepared a 1:2 dilution of GFPT2 plasmid from 9/27 miniprep
</LI><LI> Treated 5uL of diluted plasmid with 5uL of water, BL21 protein, topo protein, topo D168A protein
</LI><LI> Incubated 30 minutes at 37C
</LI><LI> Ran a 1% agarose gel with protein treated target plasmid samples and Hyperladder I
</LI></UL><UL><LI> Digested GFPT2 plasmid with X (let run at 37C for 30 minutes)
</LI><LI> Used DNA clean up kit on digested GFPT2
</LI><LI> Nanodropped digested GFPT2:
</LI><LI> GFPT2(X) - 39.85 ng/uL
</LI></UL><LI> Prepared DNA seq samples using VF2 and VR
</LI><LI> Sample# - PrimerPair - DNA sample (sample 1, GFPT2 uncut + VF2; sample 2, GFPT2 uncut + VR)
</LI><LI> 1/2 - FWD/REV - uncut GFPT2 plasmid
</LI><LI> 3/4 - FWD/REV - cut GFPT2 plamid
</LI><LI> 5/6 - FWD/REV - 2:1 uncut:cut GFPT2 plasmid mixture
</LI><LI> 7/8 - FWD/REV - 1:1 uncut:cut GFPT2 plasmid mixture
</LI><LI> 9/10 - FWD/REV - 1:2 uncut:cut GFPT2 plasmid mixture
</LI><LI> 11/12 - FWD/REV - 1-2 miniprep sample from 9/19 double transformations
</LI><LI> 13/14 - FWD/REV - 1-2I
</LI><LI> 15/16 - FWD/REV - 1-3
</LI><LI> 17/18 - FWD/REV - 1-3I
</LI><LI> 19/20 - FWD/REV - 2-1
</LI><LI> 21/22 - FWD/REV - 2-1I
</LI><LI> 23/24 - FWD/REV - 2-2
</LI><LI> 25/26 - FWD/REV - 2-2I
</LI><LI> 27/28 - FWD/REV - 2-3
</LI><LI> 29/30 - FWD/REV - 2-3I
</LI><H2><SPAN class="mw-headline" id="October_1">October 1</SPAN></H2><H3><SPAN class="mw-headline" id="Strep_24">Strep</SPAN></H3><UL><LI> Strep samples were minipreped and digested. The samples were inserted into the shipping vector.
</LI><LI> Minipreps of:
</LI><LI> J61011 + S + L + ALPHA 2B
</LI><LI> J61100 + S + L + ALPHA 2B
</LI><LI> J61100 + S + L + ALPHA 2C
</LI><LI> PLUX2 + RBS + ALPHA 1A
</LI><LI> PLUX2 + RBS + ALPHA 1A (2)
</LI><LI> PLUX + S + L + ALPHA 2C
</LI><LI> J61101 + S + L + ALPHA 1A
</LI><LI> J61101 + S + L + ALPHA 1A (2)
</LI><LI> PLUX2 + RBS + S + L + ALPHA 2B
</LI><LI> PLUX2 + RBS + S + L + ALPHA 2B (2)
</LI><LI> J61101 + S + L + ALPHA 2C
</LI><LI> PLUX + S + L + ALPHA 2B
</LI></UL><LI> Ran Twice when finding concentrations:
</LI><LI> J61101 + S + L + ALPHA 1A 
</LI><LI> J61101 + S + L + ALPHA 2B
</LI><LI> Restricted all the above and: 
</LI><LI> Strep + Linker + OMEGA
</LI><LI> Strep + Linker-4 + OMEGA
</LI><LI> with:
</LI><LI> X+P
</LI><LI> Gel confirmed
</LI><UL><LI> Ligated into pSB1C3 shipping vector and transformed into BL21(DE3) cells.
</LI><LI> Prepared all 14 above samples for sequencing.
</LI></UL><H2><SPAN class="mw-headline" id="October_2">October 2</SPAN></H2><H3><SPAN class="mw-headline" id="Strep_25">Strep</SPAN></H3><UL><LI> Transformation failed and DNA was turned in for sequencing.
</LI></UL><H3><SPAN class="mw-headline" id="Topo_25">Topo</SPAN></H3><UL><LI> Prepared PCR tubes with 10uL Topo1 4 (25ng/uL topo d168a in the shipping vector), 10uL GFPT2-26 (25ng/uL GFPT2 in the shipping vector), and 10uL GFPT1-3 (25ng/uL GFPT1 in the shipping vector)
</LI><LI> Labelled the tubes K891234, K891999, K891000 respectively and shipped overnight to iGEM Headquarters
</LI><LI> Topo D168A treated DNA samples, incubated for 10 minutes at 37C:
</LI><LI> Omega fragment PCR amplicon
</LI><LI> Topo coding sequence PCR
</LI><LI> GFPT1 VF2/VR PCR amplicon
</LI></UL><LI> Ran a 1% agarose gel containing untreated omega PCR, untreated topo PCR, untreated GFPT1 PCR, treated omega PCR,  treated topo PCR, treated GFPT1 PCR, and Hyperladder I
</LI><H2><SPAN class="mw-headline" id="October_3">October 3</SPAN></H2><H3><SPAN class="mw-headline" id="Topo_26">Topo</SPAN></H3><UL><LI> Protein treated GFPT2 plasmid, incubated for 10 minutes at 37C:
</LI><LI> BL21 HIS-purified Control Lysate
</LI><LI> Topo
</LI><LI> Topo D168A
</LI></UL><LI> Ran a 1% agarose gel containing GFPT1, GFPT1 + BL21 protein, GFPT1 + Topo, GFPT1 + Topo D168A, Topo D168A (no DNA), and Hyperladder I
</LI><H2><SPAN class="mw-headline" id="October_18">October 18</SPAN></H2><H3><SPAN class="mw-headline" id="Magainin_9">Magainin</SPAN></H3><UL><LI> Overlapping oligo assembly of Magainin + Linker + His-tag
</LI><LI> Ligated assembly into GFP vector BBa_I13522 for green-white screen
</LI><LI> Transformed into DH5a cells and incubated overnight with negative control
</LI></UL><H2><SPAN class="mw-headline" id="October_19">October 19</SPAN></H2><H3><SPAN class="mw-headline" id="Magainin_10">Magainin</SPAN></H3><UL><LI> Picked a single white colony from overnight plate and made liquid culture.
</LI></UL><H3><SPAN class="mw-headline" id="Topo_27">Topo</SPAN></H3><UL><LI> Biobricked template: pet29a adapter F and R - cut with X&amp;E, ligate into open pet vector
</LI><LI> PSV template: 1 - omega R + SP and 1-Omega F+TSE
</LI><LI> Topo in pet29a template: Topo R omega His + TSE and T7 forward adding X
</LI><LI> Topo in pet29a template: Topo R alpha His+S and T7 forward adding X
</LI><LI> PSV template: alpha-4 F+X and alpha-4 R+SP
</LI><LI> PSV template: new R alpha SC+SP and 6:F beta-gal-alpha X
</LI><LI> 5:R Beta-gal-omega SP &amp; 4F: beta-gal-omega TSE --&gt; PSV template
</LI><LI> Cut PCR products &amp; ligate
</LI><LI> 3 SpeI + 4Xbai, 5Xbai     ligate --&gt; cut X&amp;P, drop into appropriate vector
</LI><LI> 2 TSE + 1 TSE, 6 TSE     ligate --&gt; cut X&amp;P, drop into appropriate vector
</LI></UL><LI>Made liquid cultures off of plates: 
</LI><LI> T7 (1:15N)
</LI><LI> PSV
</LI><LI> magainin (in psb1A2)
</LI><H2><SPAN class="mw-headline" id="October_20">October 20</SPAN></H2><H3><SPAN class="mw-headline" id="Magainin_11">Magainin</SPAN></H3><UL><LI> Miniprepped Magainin + Linker + His-tag +BBa_I13522 (274 ng/uL)
</LI><LI> Cut with S+P and ligated to alpha-4 fragment cut with X+P.
</LI><LI> Cut with S+P and ligated to omega fragment cut with X+P. 
</LI><LI> Transformed both ligations into DH5a cells with negative control
</LI></UL><H3><SPAN class="mw-headline" id="Topo_28">Topo</SPAN></H3><UL><LI> Miniprepped: T7, PSV, magainin
</LI><LI> gel visualization of PRC products from 10/19/12 (all of them worked)
</LI></UL><UL><LI>PCR clean-up of PCR products
</LI><LI> Digestion &amp; Ligation of PCR Products (pet 29 adapter, E&amp;X)
</LI><LI>Digestion of TSE fragments (15 uL PCR product, 3uL NEB4 buffer, 1 uL TSE, 11 uL H2O)
</LI><LI>Digestion with S&amp;X
</LI><LI>ligated and ran on a gel
</LI><LI>restarted PCRs (Topo+Alpha, Alpha, Alpha-4, Omega)
</LI><LI> 1 hour anneal times for omega topo alpha
</LI><LI> 2 hour anneal times for omega topo alpha-4
</LI><LI> 27 ccles 55 anneal temp
</LI><LI> mixed matching PCR products
</LI><LI> PCR cleaned (kit) PCR products; eluted in 30 uL
</LI><LI> Ligated EX digest pet adapter to EX open isolated pet plasmid backbone
</LI><LI> Transformed into DH5alpha
</LI><LI> outgrowth in SOC
</LI><LI> plated on kanamycin plate; stored @ 37C
</LI><LI> Digested 4 PCR products: Beta-gal fragments; Topo w/ S
</LI><LI> Digested T7 plasmid S&amp;P
</LI><LI> PCR cleaned T7, and 3 ligates eluted in 15 uL
</LI><LI> Digested all 3 ligations with X &amp; P for 30 min @ 37C
</LI><LI> Ran on a gel hyperladder 1
</LI></UL><UL><LI> Excised 1100, 3000, 2000 bp bands
</LI><LI> Gel extracted eluted w/ 8 uL
</LI><LI> Ligated T7: let run @ room temp for 30 min; transformed into DH5alpha, seeded into amp plates; outgrowth in SOC
</LI></UL><H2><SPAN class="mw-headline" id="October_21">October 21</SPAN></H2><H3><SPAN class="mw-headline" id="Magainin_12">Magainin</SPAN></H3><UL><LI> Picked single colonies of both Magainin + Linker + His-tag + alpha-4 and Magainin + Linker + His-tag + omega and made liquid cultures.
</LI></UL><H3><SPAN class="mw-headline" id="Topo_29">Topo</SPAN></H3><UL><LI> Colony PCR on 6 colonies from each topo fusion plate
</LI><LI> Made master mix, prepared 10 uL PCR reactions for each colony
</LI><LI> Picked half of each colony and swirled tip in PCR mix
</LI><LI> Used beta-gal primers for each respective part: alpha, alpha-4, omega (25 cycles annealing temp)
</LI><LI> Digest ~5uL (100ng/uL) T7 promoter at 37C for 45 min
</LI><LI> PCR cleaned digestion
</LI><LI> Transformed into DH5alpha (followed Haynes 101 procedure)
</LI></UL><P>Ran PCR product on gel w/ hyperladderI
</P><UL><LI> No bands anywhere (thick primers, smear in all wells, thicker in first 7 samples)
</LI><LI> made liquid cultures of
</LI><LI> Amp: alpha, omega
</LI><LI> Kanamycin: pet+adapter
</LI><LI> stored alpha, omega, &amp; alpha-4 in 4C
</LI></UL><H2><SPAN class="mw-headline" id="October_22">October 22</SPAN></H2><H3><SPAN class="mw-headline" id="Topo_30">Topo</SPAN></H3><UL><LI> Miniprepped alpha (1) and omega (1) and (2)
</LI><LI> Restricted 4uL each of topo alhpa and topo omega with 1 uL EcoRI
</LI><LI> Ran on gel
</LI></UL><UL><LI> Ran 5 uL of unrestricted plasmids on a gel topo alpha, omega
</LI><LI> Colony PCR'd alpha-4 and T7 with VF2 and VR
</LI><LI> Ligated T7 promoter from 10/21/12 to topo + alpha-4
</LI><LI> Transformed into DH5alpha
</LI><LI> Outgrowth in 50C for an hour at 37C
</LI><LI> Plated on amp plate
</LI></UL><H2><SPAN class="mw-headline" id="October_23">October 23</SPAN></H2><H3><SPAN class="mw-headline" id="Water_Sample_Testing">Water Sample Testing</SPAN></H3><UL><LI> Figure 1: Left to right, water sample from Artisan Courtyard fountain (ASU campus), water sample from Tempe Town Lake and Nuclease free water. The following picture was taken 10 minutes after X-gal, PSV cell lysate (contains full LacZ) and DTT were added to the solution. The following test samples were under buffering conditions. All three samples changed colors but the Tempe town lake water sample reflected a color change 2 minutes after the addition of the reagents.  
</LI></UL><UL><LI> Figure 2: Left to right, Nuclease free water sample, Tempe Town Lake water sample and Artisan Courtyard fountain (ASU campus) water sample. The following picture was taken 10 minutes after X-gal, PUC 19 cell lysate (contains split LacZ fragements) and DTT were added to the solution. The following test samples were under buffering conditions. Faint color change was observed in all three samples.
</LI></UL><UL><LI> Figure 3: Control Samples. Left to right, water sample from Artisan Courtyard fountain (ASU campus), water sample from Tempe Town Lake and Nuclease free water. The following picture was taken 10 minutes after X-gal, NEB10 B cell lysate (contains omega fragment of LacZ) and DTT were added to the solution. The following test samples were under buffering conditions. No color change was observed.
</LI></UL><UL><LI> Figure 4: Left to right, water sample from Artisan Courtyard fountain (ASU campus), Nuclease free water and water sample from Tempe Town Lake. The following picture was taken 10 minutes after X-gal, commercial beta gal enzyme and DTT were added to the solution. The following test samples were under buffering conditions. No color change was observed.
</LI></UL><UL><LI> Figure 5: Left to right, water sample from Tempe Town Lake, Nuclease free water and water sample from Artisan Courtyard fountain (ASU campus). The following picture was taken 10 minutes after X-gal and commercial beta gal enzyme were added to the solution. The following test samples were not under buffering conditions. No color change was observed.
</LI></UL><UL><LI> Figure 6: Left to right, water sample from Tempe Town Lake, Nuclease free water and water sample from Artisan Courtyard fountain (ASU campus). 
</LI></UL><UL><LI> Figure 7: Figure 1, observed after 10 minutes (Left to right, water sample from Tempe Town Lake, water sample from Artisan Courtyard fountain (ASU campus) and Nuclease free water). Samples are attaining a darker blue shade.
</LI></UL><UL><LI> Figure 8: Figure 2, observed after 10 minutes. Samples still only have faint blue color.  
</LI></UL><UL><LI> Figure 9: Figure 3, observed after 10 minutes. No color change.
</LI></UL><UL><LI> Figure 10: Figure 4, observed after 10 minutes. No color change.
</LI></UL><UL><LI> Figure 11: Figure 5, observed after 10 minutes. No color change. 
</LI></UL><UL><LI> Figure 12: Figure 7, observed after 10 minutes. All the samples turned blue.
</LI></UL><UL><LI> Figure 13: Figure 8, observed after 10 minutes. Water samples from the Lake and Nuclease free water had a deeper blue color compared to the sample from Artesian Courtyard fountain.
</LI></UL><UL><LI> Figure 14: Figure 9, observed after 10 minutes. No color chance as expected. 
</LI></UL><UL><LI> Figure 15: Figure 10, observed after 10 minutes. No color change.
</LI></UL><UL><LI> Figure 16: Figure 11, observed after 10 minutes. No color change.
</LI></UL><H3><SPAN class="mw-headline" id="Topo_31">Topo</SPAN></H3><UL><LI>Ran colony PCR of 8 colonies from each of the three transformed plates from 10/23 (alpha, alpha 4,omega) 
</LI><LI>Used VF2 and VR biobrick sequencing primers
</LI><LI>55C annealing temperature
</LI><LI>3.5 min elongation time
</LI></UL><UL><LI>Visualized the PCR product on a gel.
</LI></UL><H2><SPAN class="mw-headline" id="October_24">October 24</SPAN></H2><H3><SPAN class="mw-headline" id="Topo_32">Topo</SPAN></H3><UL><LI>PCR amplification of:
</LI><LI>Topo adding XbaI site in front, SpeI site behind
</LI><LI>Alpha 4 adding XbaI site in front, SNP biobrick suffix behind
</LI><LI>Alpha adding XbaI site in front, SNP biobrick suffix behind
</LI><LI>Omega adding XbaI site in front, SNP biobrick suffix behind
</LI></UL><UL><LI>PCR mutation of:
</LI><LI> Topoisomerase coding sequence (Bba_K891234) to remove EcoRI site at 896th nucleotide
</LI><LI> Omega coding sequence to remove internal EcoRI site.
</LI></UL><UL><LI>Made liquid cultures of Topo+GFPT1 and Topo+GFPT2
</LI></UL><H2><SPAN class="mw-headline" id="October_25">October 25</SPAN></H2><H3><SPAN class="mw-headline" id="Topo_33">Topo</SPAN></H3><UL><LI>SOEing PCR for mutation of Topo coding sequence and Omega coding sequence
</LI><LI>25uL Syzygy PCR master mix
</LI><LI>12.5uL PCR front mutational half 
</LI><LI>12.5uL PCR reverse mutational half
</LI></UL><H2><SPAN class="mw-headline" id="October_26">October 26</SPAN></H2><H3><SPAN class="mw-headline" id="Topo_34">Topo</SPAN></H3><UL><LI> gel visualization of the PCR products from 10/25
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Arizona_State/Notebook">http://2012.igem.org/Team:Arizona_State/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Arizona_State/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Arizona_State/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Arizona_State/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Arizona_State/Notebook&amp;oldid=297963" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>