<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UChile-OpenBio_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UChile-OpenBio/Description</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>TODO supply a title</TITLE><HEADER><NAV id="menu"><UL><LI><A href="https://2015.igem.org/Team:UChile-OpenBio">HOME</A></LI><LI><A href="https://2015.igem.org/Team:UChile-OpenBio/Description">PROJECT</A></LI><LI><A href="https://2015.igem.org/Team:UChile-OpenBio/Description">PROJECT DESCRIPTION</A></LI><LI><A href="https://2015.igem.org/Team:UChile-OpenBio/Experiments">EXPERIMENT AND PROTOCOLS</A></LI><LI><A href="https://2015.igem.org/Team:UChile-OpenBio/Results">RESULTS</A></LI></UL><LI><A href="https://2015.igem.org/Team:UChile-OpenBio/Notebook">NOTEBOOK</A></LI><LI><A href="https://2015.igem.org/Team:UChile-OpenBio/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2015.igem.org/Team:UChile-OpenBio/Collaborations">COLLABORATION</A></LI><LI><A href="https://2015.igem.org/Team:UChile-OpenBio/Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:UChile-OpenBio/Safety">SAFETY</A></LI><LI><A href="https://2015.igem.org/Team:UChile-OpenBio/Modeling">MODELING</A></LI><LI><A href="https://2015.igem.org/Team:UChile-OpenBio/Team">TEAM</A></LI></NAV></HEADER></P><DIV id="contentContainer"><SECTION><SPAN class="titulo_seccion">Project</SPAN></SECTION><SECTION id="background"><SPAN class="titulo_seccion">Overview: Background</SPAN><DIV class="division"><DIV class="half"><P>For many years our society has wanted to have a comfortable life. It has invented and produced a lot of things that allow this comfort [1]. For example, in a typical day we can observe the use of plastic glasses for coffee, plastic bottles for water, disposable cutlery for lunching, plastic bags for supermarket and so others. </P><P>Each year, about 300 million tons of plastics are manufactured and between 5 to 13 million tons of it ends up in the ocean [3], which are responsible for the death of 1,5 million of marine animals alla around the world [4].</P></DIV><DIV class="half last"><P> In Chile about 25 thousand tons of wastes are thrown into the ocean, where it can be brought back to the coast, sunk or accumulated near the Easter Island [5], but that is not all! Recently, a report published in the Proceedings of the National Academy of Sciences of the United States revealed an emerging and worrying problem: all this plastic thrown to the ocean has starting to fragment in micro-particles which can act as sponges for waterborne contaminants such as dioxins and are swallowed by marine species accumulating in their bodies [3].</P></DIV></DIV></SECTION><ARTICLE><DIV class="division"><DIV class="half"><P>Fossil plastic contamination is not a new issue and several ways to reduce it have been explored. For example, in Chile, the government has generated a proposal of law to forbid using supermarket plastic bags made of polyethylene, polypropylene and other artificial polymers which are non-biodegradable, which was accepted in the Patagonian territory last year [7]. On the other hand, recycling seems to be a great action, but is not a really viable solution, knowing that only up to the 30% of plastic produced is actually reused [8]. </P></DIV><DIV class="half last"><P>A more sustainable initiative is to produce, from renewable resources, biodegradable plastics, due to their short degradation time, for example it can be up to two years in the case of PolyLactic Acid (PLA) which physical properties are very similar to the classic plastic ones [9]. Nevertheless, the current synthesis, essentially driven by chemical reactions, is quite expensive since the process requires complex experimental conditions, for instance the absence of any trace of water, rising production costs [2]. Besides, we estimated that today the cost production of biodegradable plastics is about 12-times higher than fossil plastics cost production [10-11] and according to Yale iGEM Team (2013) one gram of pure PLA costs around US$90. Moreover it is mostly manufactured from corn, a principal human food source [12], and it is necessary near 2,7 kilograms of corn to make 1 kilogram of PLA, requiring the use of many chemicals which are environmentally unfriendly [13]. </P></DIV></DIV></ARTICLE><ARTICLE><DIV class="division"><DIV class="half"><P>Although several scientific studies already began to produce PLA using genetically modified bacteria [17], the main difficulty resides in finding a way to export the bioplastic chains outside the cell [18]. For example, large scale production of PHB, a type of PHA, is not wide-spread mainly due to the extraction of PHB is a difficult and expensive challenge. For this reason, some studies have achieved secretion of PHB in E. coli using a synthetic biological engineering approach to try to reduce downstream processing costs [19]. </P></DIV><DIV class="half last"><P>Considering all these elements, the team UChile-OpenBio wants to reach, in the long term, the implementation of a secretory biological production of PLA from a renewable resource, the brown macroalgae (kelp), which is located on the Chilean coasts. Our challenge, consists in making the biodegradable plastic production cheaper and develops an integrative way to synthesize them, using E. coli.</P></DIV></DIV></ARTICLE><SECTION id="main_goal"><ARTICLE><DIV class="division"><H1 align="center">Main Goal</H1><P aling="center">For the iGEM competition, the team aims to engineer a biological system, enabling it to degrade glucose in order to produce and export into the medium a biodegradable plastic called PLA.</P><TABLE><TBODY><TR><TD><H1 align="center">Goal 1: Lactadora</H1></TD><TD><P>   </P></TD><TD><H1 align="center">Goal 2: PLAdora</H1></TD><TD><P>   </P></TD><TD><H1 align="center">Goal 3: Arabinita</H1></TD></TR><TR><TD><P>Designing and implementing a self-regulated lactate production system which will allow to control the lactate production by pH-sensing: the higher lactate concentration, the lower the pH, which induces a negative control in the first population of E.coli, stopping the production of lactate and by the way, of PLA</P></TD><TD><P>   </P></TD><TD><P>Designing and implementing a PLA production and exportation system which will allow blue bacteria to send the biological PLA outside the cells, into the medium. This way, the purification of the bioplastic would be easier.</P></TD><TD><P>   </P></TD><TD><P>Designing and implementng a safety system, which will consists in making arabinose-dependent the cell survival. If the medium contains arabinose, bacteria will grow up, but if bacteria escape from their medium, the cells will produce a toxin which will kill them. This way, we will ensure the safety of the persons working in the laboratory and of the environment.</P></TD></TR></TBODY></TABLE></DIV></ARTICLE><ARTICLE><DIV class="division"><H1>References</H1><TABLE><TBODY><TR><TD>[1]</TD><TD>El Banco Mundial, 2014. Una bolsa de plástico para asfixiar el mar. [online] <HTTP:> [consulted: 14-07-2015]</HTTP:></TD></TR><TR><TD>[2]</TD><TD>Garlotta, 2002. A Literature Review of PolyLactic Acid. Journal of Polymers and the Environment, Vol. 9, No. 2.</TD></TR><TR><TD>[3] </TD><TD> Alla Katsnelson. News Feature: Microplastics present pollution puzzle. Proceedings of the National Academy of Sciences (2015) vol. 112 no. 18, p5547-5549</TD></TR><TR><TD>[4] </TD><TD> El Tiempo, 2014. Plásticos matan al año 1,5 millones de animales marinos. [online] <HTTP:> [consulted: 14-07-2015]</HTTP:></TD></TR><TR><TD>[5] </TD><TD>La Tercera, 2015. Cristina Espinoza. Hasta 25 mil toneladas de plástico anuales se arrojan al mar desde Chile. [online] <HTTP:> [consulted: 14-07-2015] </HTTP:></TD></TR><TR><TD>[6] </TD><TD> 5 Gyres. [online] <HTTP:> [consulted: 13-09-2015]</HTTP:></TD></TR><TR><TD>[7] </TD><TD> Chilean Senate, 2014. [online] <HTTP:> [consulted: 14-07-2015]</HTTP:></TD></TR><TR><TD>[8] </TD><TD> PlasticsEurope. Plásticos - Situación en 2011. Análisis de la producción, la demanda y la recuperación de plásticos en Europa en 2010. [online] <HTTP:> [consulted: 15-07-2015]</HTTP:></TD></TR><TR><TD>[9] </TD><TD> Serna et al. Ácido Poliláctico (PLA): Propiedades y Aplicaciones. Ingeniería y Competitividad (2003), Vol.5, 16-26.</TD></TR><TR><TD>[10] </TD><TD>The Economist. The price of making a plastic bottle. 2014 <HTTP:> [consulted: 13-09-2015] </HTTP:></TD></TR><TR><TD>[11] </TD><TD> Facts on Pet. Husky's guide to PET Bottles. http://www.factsonpet.com/Articles/Facts%20on%20PET%20Flyer_June18%20PRINT.pdf [consulted: 13-09-2015]</TD></TR><TR><TD>[12] </TD><TD> The Field Position. The Importance of Corn.2012. &lt; http://www.thefieldposition.com/2012/06/the-importance-of-corn/&gt; [consulted: 13-09-2015]</TD></TR><TR><TD>[13] </TD><TD> Yale iGEM Team 2013. <HTTPS:> [consulted: 13-09-2015]</HTTPS:></TD></TR><TR><TD>[14] </TD><TD> Jung et al. Metabolic Engineering of Escherichia coli for the Production of Polylactic Acid and Its Copolymers. (2010) Biotechnology and Bioengineering, Vol. 105, No. 1</TD></TR><TR><TD>[15] </TD><TD> Yang TH et al. Biosynthesis of polylactic acid and its copolymers using evolved propionate CoA transferase and PHA synthase. (2010) Biotechnol Bioeng 105:150–160</TD></TR><TR><TD>[16] </TD><TD> Park SJ et al. Mutants of PHA synthase from Pseudomonas sp.6-19 and method for preparing lactate homopolymer or copolymer using the same. (2008b) WO/2008/062999</TD></TR><TR><TD>[17] </TD><TD> Mahishi et al. Poly(3-hydroxybutyrate) (PHB) synthesis by recombinant Escherichia coli harbouring Streptomyces aureofaciens PHB biosynthesis genes: Effect of various carbon and nitrogen sources. Microbiol. Res. (2003) 158, 19–27</TD></TR><TR><TD>[18] </TD><TD>Jacquel N, Lo C-W, Wei Y-H, Wu H-S, Wang SS: Isolation and purification of bacterial poly(3-hydroxyalkanoates). Biochem Eng J 2008, 39:15–27 </TD></TR><TR><TD>[19] </TD><TD>Asif Rahman. Secretion of polyhydroxybutyrate in Escherichia coli using a synthetic biological engineering approach. (2013) Journal of Biological Engineering 2013, 7:24 </TD></TR><TR><TD>[20] </TD><TD> Maria Enquist-Newman. Efficient ethanol production from brown macroalgae sugars by a synthetic yeast platform. (2014) Nature. Vol 505. p239-243</TD></TR><TR><TD> [21]</TD><TD> Wargacki et al. An Engineered Microbial Platform for Direct Biofuel Production from Brown Macroalgae. (2012). Science. Vol 335 p308-313</TD></TR><TR><TD>[22] </TD><TD> Hannover, 2001. The Hannover Principles. Design Guidelines for EXPO 2000. The Word’s Fair&gt;</TD></TR><TR><TD>[23] </TD><TD> Dartmouth Undergradruate Journal of Science. Biodegradable Plastic: Its Promises and Consequences. 2013. [online] <HTTP:> [consulted: 12-09-2015]</HTTP:></TD></TR><TR><TD>[24] </TD><TD> M. Gumel, M. Annuar , Y. Chisti. Recent Advances in the Production, Recovery and Applications of Polyhydroxyalkanoates. (2012) Journal of Polymers &amp; the Environment., Vol. 21. p580</TD></TR><TR><TD>[25] </TD><TD> Y. Kohara, K. Akiyama, K. Isono. The physical map of the whole E. coli chromosome: Application of a new strategy for rapid analysis and sorting of a large genomic library. (1987) Cell, Vol 50 (3), 495:508</TD></TR><TR><TD>[26] </TD><TD> 8.Jong et al. Production of recombinant proteins by high cell density culture of Escherichia coli. Chemical Engineering Science (2006). Vol. 61, Issue 3, 876–885.</TD></TR><TR><TD>[27] </TD><TD> AQUA. Analizan potencialidad del cultivo de algas pardas en la Región de Coquimbo. 2007. [online]  [consulted: 12-09-2015]</TD></TR><TR><TD>[28] </TD><TD> Subsecretaría de Pesca y Acuicultura. Propuesta Plan de Manejo de la Pesquería de Algas Pardas Región de Arica y Parinacota. [online] <HTTP:> [consulted: 12-09-2015]</HTTP:></TD></TR><TR><TD>[29]</TD><TD> Ministerio De Economía, Fomento Y Turismo. Informe Tecnico (R. Pesq.) N°74 - 2010 Acoge Medidas Administrativas Para El Recurso Algas Pardas En Zonas De Libre Acceso De Las Regiones De Los Ríos Y Los Lagos Recomendadas Mediante Informe Técnico Dzp N° 005/2010</TD></TR></TBODY></TABLE></DIV></ARTICLE></SECTION><SECTION id="lactadora"><SPAN class="titulo_seccion">Project: Description</SPAN><ARTICLE><H1 align="center">Lactadora: Lactate production and pH regulation system</H1><P align="center">Develop a system that can produces lactate and regulates its concentration. Also, a homoserine lactone molecule would be produce at the same time to activate the second system (PLA production and exportation) by quorum sensing.</P><H2 align="center">What parts did we use? And why?</H2><DIV class="division"><DIV class="half"><B>prcfB (extract of BBa_K609001)</B>: The rcfB promoter is the most recently reported gene which is highly induced by acidity. When the external pH value declines to 5.5, rcfB promoter will be highly upregulated . 
The Harbin Institute of Technology iGEM Team designed in 2011 a part consisting in a promoter rcfB joined to a repressor lacR. We extracted only the DNA sequence corresponding to promoter rcfB to later request to Integrated DNA Technologies  the DNA chemical synthesis using its offer of 20 kb of gBlocks® Gene Fragments to registered teams.
<B>pTet (BBa_R0040)</B>: This is a widely studied and characterized promoter frequently used by iGEM Teams. It is constitutively ON and repressed by TetR protein.

To all of our constructions we use the Ribosome Binding Site based on Elowitz repressilator <B>BBa_B0034</B>. It is considers with efficiency 1.0, it means has the best translation velocity (what’s translation?), so is used to define efficiency of other RBS
</DIV><DIV class="half last"><B>TetR (BBa_C0040)</B>: This part is the coding region for the TetR protein which binds to the pTet promoter to repress it. Also, TetR is inhibited by the addition of tetracycline or its analog aTc.
<B>LuxI (BBa_C0061)</B>: This sequence is a part of the LuxI/LuxR quorum sensing system from V. fischeri. LuxI is an enzyme that converts S-adenosylmethionine into a small molecule called an acyl-homoserine lactone (HSL); LuxI specifically produce 3-oxohexanoyl-homoserine lactone (3OC6HSL). This signal molecule can diffuse across cell membranes and is stable in growth media at a range of pH waiting for binding to a LuxR protein to induce a pLux promoter (explained below).

To all of our constructions we use the Double Terminator <B>BBa_B0015</B>: This part controls the end of transcription. This is the most commonly used terminator and It seems to be reliable. Consist in a double terminator containing BBa_B0010 and BBa_B0012 terminators.
</DIV></DIV><H2 align="center">What modules will we assembly?</H2><DIV class="division"><DIV class="half"><H3 align="center">Module 1</H3></DIV><DIV class="half last"><H3 align="center">Module 2</H3></DIV></DIV></ARTICLE><ARTICLE><DIV class="division"><DIV class="half"><H1 aling="center">How does our Lactadora system work?</H1><P>Our first system <SPAN class="destacado">Lactadora</SPAN> consists of <EM>E.coli</EM> lab strain with a synthetic gene circuit presented at the rigth. 
We construct the ppH-TetR module to synthetize the TetR protein which is able to repress the pTetR promoter. This synthesis is up-regulated by a pH promoter which is induced when pH is lower than 5.5, otherwise (pH&gt;5.5) there is no TetR protein production. In this last case, the second module (pTetR-DLDH-LuxI) will express the DLDH enzyme to synthetize lactate from pyruvate produced from a glucose molecule and diffuse to medium.</P><P>Also, will be expressed the LuxI protein to generate homoserine lactone (HSL), a quorum sensing molecule that can diffuse outside the cell. While concentration of lactate outside the cell increases, medium will turn it an acidic environment; the higher lactate concentration, the lower pH. </P><H3>When pH&gt;5.5, lactate production is ON.</H3></DIV><DIV class="half last"><H3>When pH&lt;5.5, lactate production is OFF.</H3><P>This way, when pH reaches a 5.5 value or lower, ppH-TetR module will be activated and TetR protein will repress the second module and synthesis of lactate will stop until pH increases again. </P></DIV></DIV></ARTICLE></SECTION><SECTION id="PLA"><ARTICLE><H1 align="center">PLAdora: PLA production and exportation system</H1><P align="center">Develop a system that can polymerize lactic acid to polylactic acid (PLA) and it can be able to export PLA outside the bacteria.
</P><H2 align="center">What parts did we use? And why?</H2><DIV class="division"><DIV class="half"><B>pC (BBa_J23119)</B>: This is a constitutive promoter which is part of the constitutive promoter family serie J23100 through J23119; this last is the &quot;consensus&quot; promoter sequence and the strongest member of the family.
<B>pLux (BBa_R0062)</B>: This is an up-regulated promoter by the action of the LuxR protein in concert with AHL (3OC6HSL). Two molecules of LuxR protein form a complex with two molecules of the signalling compound AHL, increasing the rate of transcription. This promoter is a part of the LuxI/LuxR quorum sensing system from <EM>V. fischeri</EM>.

To all of our constructions we use the Ribosome Binding Site based on Elowitz repressilator <B>BBa_B0034</B>. It is considers with efficiency 1.0, it means has the best translation velocity (what’s translation?), so is used to define efficiency of other RBS
</DIV><DIV class="half last"><B>LuxR generator (BBa_I0462)</B>: LuxR is a protein that can bind AHL present in the media, forming the complex that can activate transcription of PLux promoter [http://parts.igem.org/Lux]. LuxR generator is consists of RBS (BBa_B0034), LuxR (BBa_C0062) and Terminator (BBa_B0015) parts and is it the module able to synthetize LuxR protein when a promote is assembled at beginning
<B>phasin (BBa_K300002)</B>:    <B>signal peptide HIyA (BBa_K208006)</B>: This part is the HlyA signal peptide which is used to target proteins for secretion via the Type I secretion pathway of gram-negative bacteria. Type I secretion is a single-step translocation of protein across both inner and outer membranes (Binet, 1997). Accordingly (York, 2001), PHA recovery may be possible by tagging the phasin protein with HIyA for translocation, so we think it would be possible recovery PLA by the same method because of it similar properties to PHA  

To all of our constructions we use the Double Terminator <B>BBa_B0015</B>: This part controls the end of transcription. This is the most commonly used terminator and It seems to be reliable. Consist in a double terminator containing BBa_B0010 and BBa_B0012 terminators.
</DIV></DIV><H2 align="center">What modules will we assembly?</H2><DIV class="division"><DIV class="one_third"><H3 align="center">Module 1</H3></DIV><DIV class="one_third"><H3 align="center">Module 2</H3></DIV><DIV class="one_third last"><H3 align="center">Module 3</H3></DIV></DIV></ARTICLE><ARTICLE><H1>How does our PLAdora system work?</H1><ARTICLE><P>When HSL molecules diffuse from bacteria 1 to bacteria 2, will bind LuxR protein (completing quorum sensing) and will induced plux promoter so pCoAT and phaC enzyme and phasin-HIyA protein will be expressed at the same time. This system depend of the first system, because if pH is lower than 5.5 (e.g. due to a low consumption of lactate from medium by the second system), there is no production either of lactate and HSL in the first system so there is no induction of plux promoter by LuxR-HSL complex, thereby second system would be OFF.</P><H2 align="center">Represantations of interaction of the different modules of the second system</H2></ARTICLE></ARTICLE><SECTION id="safety"><ARTICLE><H1 align="center">Arabinita: Safety System</H1><P align="center">Develop a system that can guarantee human and environmental safety by destroying cells which escape from the controlled culture media.</P><H2 align="center">What parts did we use? And why?</H2><DIV class="division"><DIV class="half"><B>PBAD (K206000)</B>: pBAD is an E.coli promoter that is induced by L-arabinose. Based on the Registry, in the absence of arabinose, the repressor protein AraC (<B>BBa_I13458</B>) binds to the AraI1 operator site of pBAD and the upstream operator site AraO2, blocking transcription [1]. In the presence of arabinose, AraC binds to it and changes its conformation such that it interacts with the AraI1 and AraI2 operator sites, permitting transcription.
<B>PTetR</B>: How it was explained before, we did use the <B>BBa_R0040</B> part.

To all of our constructions we use the Ribosome Binding Site based on Elowitz repressilator <B>BBa_B0034</B>. It is considers with efficiency 1.0, it means has the best translation velocity (what’s translation?), so is used to define efficiency of other RBS
</DIV><DIV class="half last"><B>TetR</B>: How it was explained before, we did use the <B>BBa_C0040</B> part.
<B>Lysis gene (BBa_K117000)</B>: This lysis gene encodes for the lysis protein in colicin-producing strains of bacteria. Once activated, it causes the host cell to lyze.

To all of our constructions we use the Double Terminator <B>BBa_B0015</B>: This part controls the end of transcription. This is the most commonly used terminator and It seems to be reliable. Consist in a double terminator containing BBa_B0010 and BBa_B0012 terminators.
</DIV></DIV><H2 align="center">What modules will we assembly?</H2><DIV class="division"><DIV class="half"><H3 align="center">Module 1</H3></DIV><DIV class="half last"><H3 align="center">Module 2</H3></DIV></DIV></ARTICLE><ARTICLE><DIV class="division"><DIV class="half"><H1>How does our safety system work?</H1><P>Essentially, we create a kind of auxotrophic cells which are arabinose-dependent; that means cells need arabinose in the culture medium to survive and grow up. The specific gene circuit is shown in Figure 4. In presence of arabinose pBAD promoter are induced so TetR protein is synthetize to repress pTetR promoter of the PTetR-Lysis gene module.</P></DIV><DIV class="half last"><P>This allows us to control genetically modified bacteria in the lab, because if bacteria escape from their medium; means there is no presence of arabinose anymore (Figure 5), pTetR promoter will no longer be repressed so cells will produce a lysis toxin which will kill them by destroying its cell membrane.
</P></DIV></DIV></ARTICLE></SECTION><FOOTER><H1>Our Sponsors</H1></FOOTER></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>