Team:UChile-OpenBio/Results
TODO supply a title
HOME
PROJECT
PROJECT DESCRIPTION
EXPERIMENT AND PROTOCOLS
RESULTS
NOTEBOOK
ATTRIBUTIONS
COLLABORATION
HUMAN PRACTICES
SAFETY
MODELING
TEAM
Results
We performed the first  attempt of gibson assembly using the LDH_1 (618 bp) and  LDH_2 (636 bp) calcutaling the pmol needed for proper ratio of each one, using the equation of New England Biolabs protocol.  pmol = 50 ng/ uL x 1000 / 618 bp x 650  = 50.000 / 401.700 = 0.124 pmol/uL pmol = 50 ng/uL x 1000 /635 bp x 650 = 50.000 / 412.750 = 0.121 pmol/uL  The figure 1  indicates that the reaction did work but at very low concentration of DNA. We must purify the band and then reamplify using primer suffix an prefix PxSx fwd and rev.
Figure 1. Agarose electrophoresis 1% with 4 uL Gelred DNA staining.
The digestion and ligation of  the 5 biobricks SegA, p_lux, p_rfcB, p_BAD and p_C.  The plasmid backbone and the insert are digested using EcoRI and PstI at 37째C x 30 min, and then ligation is performed overnight at 16째C. Next day chemocompetents cells are a transformed using 2 uL of ligation mix and cultivated agar plates with Ampicillin 100 ug/mL, Chloramphenicol 25 ug/mL and Kanamycin 50 ug/mL correspondingly to prfcB and pBAD in pSB1A3; pC and pLux in pSB1K3 and SegA in pSB1C3. Figure 2 shows one of the transformations in plate using 50 and 100 uL of cell culture. There are no transformation observed. So then  we used a large amount of 5 uL of ligation mix and repeated transformation. Figure 3 shows the efficiente transformatoin of prfcB, the only transformation that did work (other data no shown). The other transformation for SegA, pLux, prfcB, pBAD and pC did not have might not have sufficient concentration of inserts. So the, the ligation must be repeated using higher quantiities of insert.
Figure 2. Transformation with prfcB using 50 and 100 uL of transformed cells.
Figure 3.  Transformation II (5 uL of ligation mix) with prfcB using 50 and 100 uL of transformed cells.
Gibson Assembly p-CoA-T1 and T2. The construction of p-CoA-T2 from C. necator is assembled by using the Gibson assembly of gBlocks. Figure 4 shows the gibson assembly reactions of pCoAT1_1 and pCoAT1_2 (in lane 1), pCoAT1_3 and pCoAT1_4 (in lane 2) and pCoAT2_3 and pCoAT2_4 (lane T2).  The results indicate no assembly of all samples. We inquired the information of the quantities specified in each gBlock tube from the manufacturer. Unfortunately we found that the concentration is lower than the specified and could be the reason why the modules does not worth. This conclude that is imperative to re-amplify gBlocks to perform higher concentrations of isothermal reactions.
Figure 4. Gibson assembly de pCoA-T1 y T2.
We must verify the  make the diagnosis of transformed colonies identified in the plates by a Go Taq Protocol colony PCR.
Figure 5. Plates of transformation of prfcB an LuxR. The colonies are selected in randomdly.
Figure 6. Lane 1-10: colonies prfcB. Lane 10-20: colonies LuxR.
It is not possible to interpret clearly the size of the bands because the marker is not displayed well. It is thought to correspond to the size of 494 bp consistent with amplification of VR and VF2 prfcB (1 to 10). However, amplification of LuxR (11-20) should reach 1382 bp, which is not observed in the gel; all the bands seem to have the same size, without a reasoned explanation. To corroborate size LuxR a "PCR vector" with VF2 and VR (and Px and Sx) will be held from miniprep stored at 20 째 C, in order to compare the results. However, liquid culture is performed 1-5 colonies.
Gibson assembly TetR: A ligation is prepared by Gibson Assembly for the generation of a product for amplification. Two aliquots of 5 uL Gibson assembly are taken, and added 2.1 and 2.9 of TetR_2 TetR_1 these, for a total of 20 uL to be loaded on a gel to be purified by GeneJet Gel Extraction kit. In figure 5 is observed the lane of assembled fragment of TetR (857 bp).
Figure 7.DNA electrophoresis. M: marker 1kb plus; TetR PCR amplification.
The digestion with PstI enzyme is repeated, which has not worked, in order to standardize a protocol. Digestion is performed using a change with the testing protocol to NEB2 and NEB3 buffer, to observe what Buffer is more suitable. In figure 8 the enzyme is cutting properly so it is necessary to continue to finish the amplifications assemblies. Possibly ligations are the limiting , therefore we must obtain a larger amount of amplification. It is proposed to digestion and then purification of two tubes in order to obtain greater volume. Apparently the enzyme PstI fragment does defer interest and the problem is elsewhere posiblemente.
Protocol:
4 IU DNAp
2 uL buffer
2 uL ER PstI
12 sterile distilled water
Total 20 uL
3 Hrs digestion.
Figure 7.DNA electrophoresis. M: marker 1kb plus; TetR PCR amplification.6.- Digestion test of PstI at high concentration.
Figure 8. M1: marker 1kb plus Ladder; 1: undigested plasmid pBluescript; 2: PstI digested plasmid in buffer Neb2; 3: plasmid PstI digested in NEB buffer 3.
pET22 digestion with EcoRI and PstI and Re-amplification gBlocks p_CoA-T2.-T2: We want to test the efficiency of digestion enzymes EcoRI and PstI in different buffers. The digestions were performed with large volumes of ER (2 uL c / u). Combinations of two volumes of ER are performed: 2 uL of PstI and EcoRI, EcoRI and 1 uL 0.5 uL of PstI in NEB 3 buffer and NEB2. The amplification was performed using gBblocks Phusion PCR cocktail of 50 uL and loaded 5 ul on the gel, only for the purpose of displaying the expected size and which have not interference. And then direct the purified PCR product. The annealing temperature for all was 59.5 째 C, with a time of not more than 30 sec extension at 30 cycles.  Reactions (excessive amounts of ER)
4 IU DNAp
2 uL Buffer (10X) NEB 2 or 3
2 uL ER EcoRI
2 uL ER PstI
10 uL water
Figure 9. Lanes 1-4: Are amplification of another experiment; Lane 5: Digestion with EcoRI and PstI pET22b (2 uL c / u) in NEB 2. Expected size (4157 bp and 1336 bp); Lane 6: pET22b not digested; Lane 7: Digestion with EcoRI and PstI pET22b (2 uL c / u) in NEB3. Expected size (4157 bp and 1336 bp).
Colony PCR from ligation pSB1A3.b: Ligation was performed and now the PCR colony of the ligations of the vectors pSB1A3 is performed. Recircularization iGEM the linearized vector produces the loss of an 8 bp, so the ligation product is called pSB1A3.b. Figure 10 shows a correct recircularization of the vector now called pSB1A3.b with a 300 bp amplification corresponding to the space between primer VF2 and VR.
Figure 10. Colony PCR from recircularization of vector variant pSB1A3.b. a fragment of 300 bp aprox is observed indicating the correct recircularization.
Colony PCR in transformants with gblocks as insert in pSB1A3.b: A plating of the pellet of the previously transformed colonies is done to find out if it is possible to place more colonies transformed.Colony PCR is performed with 20 uL Taq go (link to protocol with VR and VF2 primers protocol). Lane 8  is positive for SegA, lanes 11, 13, 14 and 17 is positive for pLux, 19 is positive for prfcB and 23-27 are positive for pC. We clones succesfully the parts for the security system. Further assay are necessary for DNAp purification and sequencing services before constructing by standard biobrick assembly.
Figure 11: Colony PCR from insert in vector pSB1A3.b. Lanes 1-8: SegA (391 bp). Lanes 9-18: pLux (220 bp). Lanes 19-20: prcfB (220 bp). Lanes 20-23: pBad (220 bp). Lanes 24-27: pC (220 bp)
Our Sponsors
