Team:UChile-OpenBio/Modeling
TODO supply a title
HOME
PROJECT
PROJECT DESCRIPTION
EXPERIMENT AND PROTOCOLS
RESULTS
NOTEBOOK
ATTRIBUTIONS
COLLABORATION
HUMAN PRACTICES
SAFETY
MODELING
TEAM
Bacterial system  modeling
Overview
In the fight through plastic contamination, we thought that an important issue to is to understand the phenomenon of communication and transport of the involved molecules in the PLA biosynthesis. The phenomenon that need to be modeled are cell growth, membrane osmolarity, gene expression, protein translation, and the difussion of the molecules related to the regulatory network.  For this reason, the first step of the modeling was to evaluate the whole system, including pH regulation and cellular communication between both kinds of bacteria. As a second step of the study, the idea was to model a system with a deleted pH regulation and to look what happened. As another study, we thought modeling a system without cellular communication between “Lactadora” bacteria and “PLAdora” bacteria, and to analyze the situation (see Figure 1).
Figure 1: A: Our project, two bacteria growing together, the orange producing lactate, green producing PLA; B: Two bacteria growing together, the orange producing lactate without regulation and the green producing PLA; C Two bacteria growing together, the orange producing lactate and the green producing PLA, without communication between them.
On one hand, the aim of our modeling was to evaluate how the regulatory network could respond in a two stage cell system, to get a better understanding of how the relaxation times of the reactions will respond to achieve the proper flux for PLA  production. On the other hand, we considered an important learning the process to model our system itself, and it would be a reference for other futures UChile-iGEM teams.
To construct the model, we searched for all kinetics involved in the genetics circuits, and then constructed the model utilizing CellDesigner and COPASI.
To simplify the understanding of the process of modeling, we separated this section in 4 principal areas:
- “Lactadora” bacteria” which includes all assumptions, kinetics and mass balances related to this bacteria
- “PLAdora” bacteria” that contains all assumptions, kinetics and mass balances related to this second bacteria
- “The whole system” that shows the assumptions and mass balances involving cellular growing, communication between both bacteria, and the final production of PLA. It shows also our discussions and conclusions.
- “Nomenclature”, to explain all abbreviations we utilized in the modeling.
We achieved to elaborate the programs in CellDesigner and COPASI, containing all the reactions involved our system. The second step, consisting in adjusting parameters to generate a correct simulation is proposed for another iGEM team interested to continue or improve our project.
Also, we have an advice for other new iGEM teams: modeling is an important and complex area of the project, so work on that as early as you can.
“Lactadora” bacteria
This first superhero bacteria is responsible to produce lactic acid, the monomer of PLA. It contains the modules that are shown in Figure 2. Another important activity of this bacteria is to control pH values in the medium, and regulate the acid lactic production. This is made by a sensitive pH promoter, explicated in this section .
Figure 2: Modules inside "Lactadora" bacteria.
For modeling, we took some assumptions to make the model easier to construct, and to make the difficult labor of looking for the constants kinetics less hard. The assumptions were:
- For the rate of degradation of proteins we assumed a first order reaction.
- For mRNA degradation rate also we assumed a first order reaction.
- We combined two different studies for the transport of protons: one mentioned in the lactate transport and kinetic study, and another presented in the study of quorum sensing.
- To determine the rate of translation of all mRNA, we investigated the speed of ribosomal translation per amino acid and then we searched of the number of amino acid per enzyme produced. Combining both, we obtained the rate of translation.
- The pH regulation was found for Lactobacillus bulgarius, and we assumed a similar behavior of the model.
- The rate of transcription was determined using the quotient between RNA polymerase speed per nucleotide, and the number of nucleotides per each gene.
- We used a study of genetic regulation for synthetic tetracycline as an assumption to determine the velocity of the same regulation for TetR produced biologically.
- We assumed that transport of malate and lactate (as organic acids from glycolysis pathway) are behaved similar between each other, considering that both metabolites are responsible for maintaining cell membrane potential.
Taking this on consideration, the equations we used in the modeling, for this first bacteria were:
d
[TetR]
&CenterDot;
V
c1
d
t
=
+
V
c1
&CenterDot;
0.036244
&CenterDot;
[mTetR]
-
V
c1
&CenterDot;
2.86e+006
&CenterDot;
[TetR]
&CenterDot;
["Gen 1"]
-
0.000511
&CenterDot;
["complex 2"]
-
0.00016
&CenterDot;
[TetR]
d
["Gen 1"]
&CenterDot;
V
c1
d
t
=
-
V
c1
&CenterDot;
2.86e+006
&CenterDot;
[TetR]
&CenterDot;
["Gen 1"]
-
0.000511
&CenterDot;
["complex 2"]
-
V
c1
&CenterDot;
0.04444
&CenterDot;
["Gen 1"]
-
V
c1
&CenterDot;
0.0682171
&CenterDot;
["Gen 1"]
d
["complex 1"]
&CenterDot;
V
c1
d
t
=
+
V
c1
&CenterDot;
0.00027778
+
0.0055556
&CenterDot;
1
-
tanh
[TetRg]
&CenterDot;
-
ln
["H+{c1}"]
-
5.5
&CenterDot;
2.4e-009
-
["complex 1"]
&CenterDot;
0.01155
+
0.0002222
&CenterDot;
[Lactate{default}]
[Lactate{default}]
+
0.00982
&CenterDot;
1
-
["Lactic acid"]
0.448
-
V
c1
&CenterDot;
0.0642336
&CenterDot;
["complex 1"]
d
["complex 2"]
&CenterDot;
V
c1
d
t
=
+
V
c1
&CenterDot;
2.86e+006
&CenterDot;
[TetR]
&CenterDot;
["Gen 1"]
-
0.000511
&CenterDot;
["complex 2"]
d
["D-LDH"]
&CenterDot;
V
c1
d
t
=
-
V
c1
&CenterDot;
0.00016
&CenterDot;
["D-LDH"]
+
V
c1
&CenterDot;
0.0251515
&CenterDot;
["mD-LDH"]
d
["NAD+"]
&CenterDot;
V
c1
d
t
=
+
V
c1
&CenterDot;
0.00142167
&CenterDot;
["D-LDH"]
&CenterDot;
1
0.0015
&CenterDot;
0.1
&CenterDot;
[Pyruvate]
&CenterDot;
[NADH]
-
[Lactate{c1}]
&CenterDot;
["NAD+"]
21.1207
1
+
[Pyruvate]
0.0015
+
[Lactate{c1}]
0.1
&CenterDot;
1
+
[NADH]
8e-005
+
["NAD+"]
0.0024
d
[Pyruvate]
&CenterDot;
V
c1
d
t
=
V
c1
&CenterDot;
6e-6
&CenterDot;
0.00550965
-
[Pyruvate]
d
[NADH]
&CenterDot;
V
c1
d
t
=
V
c1
&CenterDot;
6e-6
&CenterDot;
0.00550965
-
[Pyruvate]
d
[LuxI]
&CenterDot;
V
c1
d
t
=
+
V
c1
&CenterDot;
0.0386047
&CenterDot;
[mLuxI]
-
V
c1
&CenterDot;
0.00016
&CenterDot;
[LuxI]
d
[Lactate{c1}]
&CenterDot;
V
c1
d
t
=
+
V
c1
&CenterDot;
0.00142167
&CenterDot;
["D-LDH"]
&CenterDot;
1
0.0015
&CenterDot;
0.1
&CenterDot;
[Pyruvate]
&CenterDot;
[NADH]
-
[Lactate{c1}]
&CenterDot;
["NAD+"]
21.1207
1
+
[Pyruvate]
0.0015
+
[Lactate{c1}]
0.1
&CenterDot;
1
+
[NADH]
8e-005
+
["NAD+"]
0.0024
-
1.25e-008
&CenterDot;
[Lactate{c1}]
0.015
-
4.16667e-009
&CenterDot;
[Lactate{default}]
0.00054
1
+
[Lactate{c1}]
0.015
+
[Lactate{default}]
0.00054
d
["H+{c1}"]
&CenterDot;
V
c1
d
t
=
+
4.1667e-009
&CenterDot;
["H+{default}"]
0.00054
-
1.25e-008
&CenterDot;
["H+{c1}"]
0.015
1
+
["H+{default}"]
0.00054
+
["H+{c1}"]
0.015
-
V
c1
&CenterDot;
0.00027778
+
0.0055556
&CenterDot;
1
-
tanh
[TetRg]
&CenterDot;
-
ln
["H+{c1}"]
-
5.5
&CenterDot;
2.4e-009
-
["complex 1"]
&CenterDot;
0.01155
+
0.0002222
&CenterDot;
[Lactate{default}]
[Lactate{default}]
+
0.00982
&CenterDot;
1
-
["Lactic acid"]
0.448
d
[TetRg]
&CenterDot;
V
c1
d
t
=
-
V
c1
&CenterDot;
0.00027778
+
0.0055556
&CenterDot;
1
-
tanh
[TetRg]
&CenterDot;
-
ln
["H+{c1}"]
-
5.5
&CenterDot;
2.4e-009
-
["complex 1"]
&CenterDot;
0.01155
+
0.0002222
&CenterDot;
[Lactate{default}]
[Lactate{default}]
+
0.00982
&CenterDot;
1
-
["Lactic acid"]
0.448
d
[mTetR]
&CenterDot;
V
c1
d
t
=
-
V
c1
&CenterDot;
0.036244
&CenterDot;
[mTetR]
+
V
c1
&CenterDot;
0.0642336
&CenterDot;
["complex 1"]
-
V
c1
&CenterDot;
4e-007
&CenterDot;
[mTetR]
d
[dmLuxI]
&CenterDot;
V
c1
d
t
=
+
V
c1
&CenterDot;
4e-007
&CenterDot;
[mLuxI]
d
[dLuxI]
&CenterDot;
V
c1
d
t
=
+
V
c1
&CenterDot;
0.00016
&CenterDot;
[LuxI]
d
[dmTetR]
&CenterDot;
V
c1
d
t
=
+
V
c1
&CenterDot;
4e-007
&CenterDot;
[mTetR]
d
["dmD-LDH"]
&CenterDot;
V
c1
d
t
=
+
V
c1
&CenterDot;
4e-007
&CenterDot;
["mD-LDH"]
d
[mLuxI]
&CenterDot;
V
c1
d
t
=
-
V
c1
&CenterDot;
0.0386047
&CenterDot;
[mLuxI]
-
V
c1
&CenterDot;
4e-007
&CenterDot;
[mLuxI]
+
V
c1
&CenterDot;
0.0682171
&CenterDot;
["Gen 1"]
d
["dD-LDH"]
&CenterDot;
V
c1
d
t
=
+
V
c1
&CenterDot;
0.00016
&CenterDot;
["D-LDH"]
d
["mD-LDH"]
&CenterDot;
V
c1
d
t
=
-
V
c1
&CenterDot;
0.0251515
&CenterDot;
["mD-LDH"]
+
V
c1
&CenterDot;
0.04444
&CenterDot;
["Gen 1"]
-
V
c1
&CenterDot;
4e-007
&CenterDot;
["mD-LDH"]
d
[dTetR]
&CenterDot;
V
default
d
t
=
+
0.00016
&CenterDot;
[TetR]
Finally, values of constants per reaction that occurs inside “Lactadora” bacteria are in Table 1.
Table 1: Kinetic and values of constants reaction for “Lactadora” bacteria.
Name
Reaction
Equations
Comments
References
d1
TetR -> dTetR
Mass action (irreversible)
k1(d1)= 0.00016
[1]
d1,2
mTetR -> dmTetR
Mass action (irreversible)
k1(d1,2)=4e-7
[2]
d2
D-LDH -> dD-LDH
Mass action (irreversible)
k1(d2)= 0.00016
[1]
d2,2
mD-LDH -> dmD-LDH
Mass action (irreversible)
k1(d2,2)= 0.00016
[2]
d2,3
mLuxI -> dmLuxI
Mass action (irreversible)
k1(d2,3)=4e-7
[2]
d3
LuxI -> dLuxI
Mass action (irreversible)
k1(d3)= 0.00016
[1]
E4
H+{default} = H+{c1}
Reversible Michaelis -Menten
Kms(E4)=0.00054,
Kmp(E4)=0.015,
Vf(E4)=4.1667e-9,
Vr(E4)=1.25e-8
[3], [8]
K1
mTetR -> TetR
Mass action (irreversible)
k1(K1)=0.0362440
[4]
K1,1
H+{c1} + TetRg = "complex 1";  Lactate{default} "Lactic acid"
Rate Law for K1,1
Vo(K1,1)=0.00027778,
V1(K1,1)=0.0055556,
GR(K1,1)=2.4e-9,
dm(K1,1)=0.01155,
umax(K1,1)=0.0002222,
Ks(K1,1)=0.00982,
Kla(K1,1)=0.448
[5]
K1,2
complex 1 -> mTetR
Mass action (irreversible)
k1(K1,2)=0.0642336
[4]
K2
mD-LDH -> D-LDH
Mass action (irreversible)
k1(K2)=0.0251515
[4]
K2,1
TetR + "Gen 1" = "complex 2"
Mass action (reversible)
k1(K2,1)=2.86e6,
k2(K2,1)=0.000511
[6]
K2,2
Gen 1 -> mD-LDH
Mass action (irreversible)
k1(K2,2)=0.04444
[4]
K2,3
Gen 1 -> mLuxI
Mass action (irreversible)
k1(K2,3)=0.0682171
[4]
K3
mLuxI -> LuxI
Mass action (irreversible)
k1(K3)=0.0386047
[4]
K4
Pyruvate + NADH = Lactate{c1} + NAD+;  D-LDH
Rate Law for K4
V(K4)=0.00142167,
Kmb(K4)=0.0015,
Kmd(K4)=0.1,
Keq(K4)=21.1207,
Kma(K4)=8e-5,
Kmc(K4)=0.0024
[7]
S1
Lactate{c1} = Lactate{default}
Reversible Michaelis -Menten
Kms(S1)=0.00054, Kmp(S1)=0.015, Vf(S1)=4.1667e-9, Vr(S1)=1.25e-8
[8]
“PLAdora” bacteria
This second superhero bacteria is responsible to produce PLA, and export it to the medium. It contains the modules that are shown in Figure 3, and the reactions involved in the genetics circuits inside “PLAdora” bacteria are:
Figure 3: Modules inside "PLAdora" bacteria.
For modeling, we took some assumptions to make the model easier to construct, and to make the difficult labor of looking for the constants kinetics less hard. The assumptions were:
- For the rate of degradation of proteins we assumed a first order reaction.
- For mRNA degradation rate also we assumed a first order reaction.
- We assumed that transport of malate and lactate (as organic acids from glycolysis pathway) are behaved similar between each other, considering that both metabolites are responsible for maintaining cell membrane potential.
- We assumed that the concentration of the metabolites 5'-methylthioadenosine and [acyl-carrier protein] are not for determining the reaction rate, so their kinetic constants are assumed high.
- To determine the rate of translation of all mRNA, we investigated the speed of ribosomal translation per amino acid and then we searched of the number of amino acid per enzyme produced. Combining both, we obtained the rate of translation.
- The rate of transcription was determined using the quotient between RNA polymerase speed per nucleotide, and the number of nucleotides per each gene.
- Using studies about PHB we assumed a similar rate of transportation for the complex PLA-Phasin-Hly.
Taking this into account, the equations we used in the modeling, for this second bacteria were:
d
[HSL{c2}]
&CenterDot;
V
c2
d
t
=
+
0.127892
&CenterDot;
[HSL{default}]
-
[HSL{c2}]
+
0.0438
&CenterDot;
8e-007
-
1.67e-007
&CenterDot;
[HSL{default}]
-
V
c2
&CenterDot;
0.5
3600e-9
&CenterDot;
[LuxR]
+
2.7e-007
4
&CenterDot;
[HSL{c2}]
2
-
[LuxR]
+
2.7e-007
4
&CenterDot;
[HSL{c2}]
2
-
[LuxR]
2
0.5
d
[PLA]
&CenterDot;
V
c2
d
t
=
+
V
c2
&CenterDot;
1.19e-005
&CenterDot;
[phaC]
&CenterDot;
["Lactyl-CoA"]
-
[PLA]
1
["Lactyl-CoA"]
+
0.000119
&CenterDot;
1
+
[PLA]
8.5e-005
-
V
c2
&CenterDot;
4.73843e-009
&CenterDot;
["Phasin-Hly"]
&CenterDot;
[PLA]
-
4.7843e-010
&CenterDot;
["PLA-Phasin-Hly{c2}"]
d
["PLA-Phasin-Hly{c2}"]
&CenterDot;
V
c2
d
t
=
+
V
c2
&CenterDot;
4.73843e-009
&CenterDot;
["Phasin-Hly"]
&CenterDot;
[PLA]
-
4.7843e-010
&CenterDot;
["PLA-Phasin-Hly{c2}"]
-
6.944e-011
&CenterDot;
["PLA-Phasin-Hly{c2}"]
d
[Acetate]
&CenterDot;
V
c2
d
t
=
+
V
c2
&CenterDot;
0.38
&CenterDot;
["P-CoA-T"]
&CenterDot;
1.36667e-007
&CenterDot;
[Lactate{c2}]
&CenterDot;
["acetyl-CoA"]
0.0006
-
4.16667e-008
&CenterDot;
["Lactyl-CoA"]
&CenterDot;
[Acetate]
0.0003
1
+
[Lactate{c2}]
&CenterDot;
["acetyl-CoA"]
0.0006
+
["Lactyl-CoA"]
&CenterDot;
[Acetate]
0.0003
d
[Lactate{c2}]
&CenterDot;
V
c2
d
t
=
+
4.1667e-009
&CenterDot;
[Lactate{default}]
0.00054
-
1.25e-008
&CenterDot;
[Lactate{c2}]
0.015
1
+
[Lactate{default}]
0.00054
+
[Lactate{c2}]
0.015
-
V
c2
&CenterDot;
0.38
&CenterDot;
["P-CoA-T"]
&CenterDot;
1.36667e-007
&CenterDot;
[Lactate{c2}]
&CenterDot;
["acetyl-CoA"]
0.0006
-
4.16667e-008
&CenterDot;
["Lactyl-CoA"]
&CenterDot;
[Acetate]
0.0003
1
+
[Lactate{c2}]
&CenterDot;
["acetyl-CoA"]
0.0006
+
["Lactyl-CoA"]
&CenterDot;
[Acetate]
0.0003
d
["Lactyl-CoA"]
&CenterDot;
V
c2
d
t
=
+
V
c2
&CenterDot;
0.38
&CenterDot;
["P-CoA-T"]
&CenterDot;
1.36667e-007
&CenterDot;
[Lactate{c2}]
&CenterDot;
["acetyl-CoA"]
0.0006
-
4.16667e-008
&CenterDot;
["Lactyl-CoA"]
&CenterDot;
[Acetate]
0.0003
1
+
[Lactate{c2}]
&CenterDot;
["acetyl-CoA"]
0.0006
+
["Lactyl-CoA"]
&CenterDot;
[Acetate]
0.0003
-
V
c2
&CenterDot;
1.19e-005
&CenterDot;
[phaC]
&CenterDot;
["Lactyl-CoA"]
-
[PLA]
1
["Lactyl-CoA"]
+
0.000119
&CenterDot;
1
+
[PLA]
8.5e-005
d
["acetyl-CoA"]
&CenterDot;
V
c2
d
t
=
-
V
c2
&CenterDot;
0.38
&CenterDot;
["P-CoA-T"]
&CenterDot;
1.36667e-007
&CenterDot;
[Lactate{c2}]
&CenterDot;
["acetyl-CoA"]
0.0006
-
4.16667e-008
&CenterDot;
["Lactyl-CoA"]
&CenterDot;
[Acetate]
0.0003
1
+
[Lactate{c2}]
&CenterDot;
["acetyl-CoA"]
0.0006
+
["Lactyl-CoA"]
&CenterDot;
[Acetate]
0.0003
d
[dphaC]
&CenterDot;
V
c2
d
t
=
+
V
c2
&CenterDot;
0.00016
&CenterDot;
[phaC]
d
[phaC]
&CenterDot;
V
c2
d
t
=
+
V
c2
&CenterDot;
0.014795
&CenterDot;
[mphaC]
-
V
c2
&CenterDot;
0.00016
&CenterDot;
[phaC]
d
["Phasin-Hly"]
&CenterDot;
V
c2
d
t
=
+
V
c2
&CenterDot;
0.0326772
&CenterDot;
["mPhasin-Hly"]
-
V
c2
&CenterDot;
0.00016
&CenterDot;
["Phasin-Hly"]
-
V
c2
&CenterDot;
4.73843e-009
&CenterDot;
["Phasin-Hly"]
&CenterDot;
[PLA]
-
4.7843e-010
&CenterDot;
["PLA-Phasin-Hly{c2}"]
d
["dPhasin-Hly"]
&CenterDot;
V
c2
d
t
=
+
V
c2
&CenterDot;
0.00016
&CenterDot;
["Phasin-Hly"]
d
["dmPhasin-Hly"]
&CenterDot;
V
c2
d
t
=
+
V
c2
&CenterDot;
4e-007
&CenterDot;
["mPhasin-Hly"]
d
["mPhasin-Hly"]
&CenterDot;
V
c2
d
t
=
-
V
c2
&CenterDot;
0.0326772
&CenterDot;
["mPhasin-Hly"]
+
V
c2
&CenterDot;
0.0577428
&CenterDot;
["complex 4"]
-
V
c2
&CenterDot;
4e-007
&CenterDot;
["mPhasin-Hly"]
d
[LuxR]
&CenterDot;
V
c2
d
t
=
-
V
c2
&CenterDot;
0.5
3600e-9
&CenterDot;
[LuxR]
+
2.7e-007
4
&CenterDot;
[HSL{c2}]
2
-
[LuxR]
+
2.7e-007
4
&CenterDot;
[HSL{c2}]
2
-
[LuxR]
2
0.5
+
V
c2
&CenterDot;
0.0266026
&CenterDot;
[mLuxR]
-
V
c2
&CenterDot;
0.00016
&CenterDot;
[LuxR]
d
[dLuxR]
&CenterDot;
V
c2
d
t
=
+
V
c2
&CenterDot;
0.00016
&CenterDot;
[LuxR]
d
[dmLuxR]
&CenterDot;
V
c2
d
t
=
+
V
c2
&CenterDot;
4e-007
&CenterDot;
[mLuxR]
d
[mLuxR]
&CenterDot;
V
c2
d
t
=
-
V
c2
&CenterDot;
0.0266026
&CenterDot;
[mLuxR]
+
V
c2
&CenterDot;
0.0470085
&CenterDot;
[LuxRg]
-
V
c2
&CenterDot;
4e-007
&CenterDot;
[mLuxR]
d
["P-CoA-T"]
&CenterDot;
V
c2
d
t
=
+
V
c2
&CenterDot;
0.0152855
&CenterDot;
["mP-CoA-T"]
-
V
c2
&CenterDot;
0.00016
&CenterDot;
["P-CoA-T"]
d
["dP-CoA-T"]
&CenterDot;
V
c2
d
t
=
+
V
c2
&CenterDot;
0.00016
&CenterDot;
["P-CoA-T"]
d
["dmP-CoA-T"]
&CenterDot;
V
c2
d
t
=
+
V
c2
&CenterDot;
4e-007
&CenterDot;
["mP-CoA-T"]
d
["mP-CoA-T"]
&CenterDot;
V
c2
d
t
=
-
V
c2
&CenterDot;
0.0152855
&CenterDot;
["mP-CoA-T"]
+
V
c2
&CenterDot;
0.0270104
&CenterDot;
["complex 3"]
-
V
c2
&CenterDot;
4e-007
&CenterDot;
["mP-CoA-T"]
d
[dmphaC]
&CenterDot;
V
c2
d
t
=
+
V
c2
&CenterDot;
4e-007
&CenterDot;
[mphaC]
d
[mphaC]
&CenterDot;
V
c2
d
t
=
-
V
c2
&CenterDot;
0.014795
&CenterDot;
[mphaC]
+
V
c2
&CenterDot;
0.0261438
&CenterDot;
["complex 3"]
-
V
c2
&CenterDot;
4e-007
&CenterDot;
[mphaC]
d
["Phasin-Hlyg"]
&CenterDot;
V
c2
d
t
=
-
V
c2
&CenterDot;
0.04
+
0.05
&CenterDot;
[HSL/LuxR]
1
+
0.04
+
0.05
&CenterDot;
[HSL/LuxR]
+
0.011
&CenterDot;
["Phasin-Hlyg"]
+
0.05
&CenterDot;
0.011
&CenterDot;
[HSL/LuxR]
&CenterDot;
["Phasin-Hlyg"]
d
[LuxRg]
&CenterDot;
V
c2
d
t
=
-
V
c2
&CenterDot;
0.0470085
&CenterDot;
[LuxRg]
d
[HSL/LuxR]
&CenterDot;
V
c2
d
t
=
+
V
c2
&CenterDot;
0.5
3600e-9
&CenterDot;
[LuxR]
+
2.7e-007
4
&CenterDot;
[HSL{c2}]
2
-
[LuxR]
+
2.7e-007
4
&CenterDot;
[HSL{c2}]
2
-
[LuxR]
2
0.5
-
V
c2
&CenterDot;
0.04
+
0.05
&CenterDot;
[HSL/LuxR]
1
+
0.04
+
0.05
&CenterDot;
[HSL/LuxR]
+
0.011
&CenterDot;
["Gen 2"]
+
0.05
&CenterDot;
0.011
&CenterDot;
[HSL/LuxR]
&CenterDot;
["Gen 2"]
-
V
c2
&CenterDot;
0.04
+
0.05
&CenterDot;
[HSL/LuxR]
1
+
0.04
+
0.05
&CenterDot;
[HSL/LuxR]
+
0.011
&CenterDot;
["Phasin-Hlyg"]
+
0.05
&CenterDot;
0.011
&CenterDot;
[HSL/LuxR]
&CenterDot;
["Phasin-Hlyg"]
d
["complex 4"]
&CenterDot;
V
c2
d
t
=
+
V
c2
&CenterDot;
0.04
+
0.05
&CenterDot;
[HSL/LuxR]
1
+
0.04
+
0.05
&CenterDot;
[HSL/LuxR]
+
0.011
&CenterDot;
["Phasin-Hlyg"]
+
0.05
&CenterDot;
0.011
&CenterDot;
[HSL/LuxR]
&CenterDot;
["Phasin-Hlyg"]
-
V
c2
&CenterDot;
0.0577428
&CenterDot;
["complex 4"]
d
["complex 3"]
&CenterDot;
V
c2
d
t
=
+
V
c2
&CenterDot;
0.04
+
0.05
&CenterDot;
[HSL/LuxR]
1
+
0.04
+
0.05
&CenterDot;
[HSL/LuxR]
+
0.011
&CenterDot;
["Gen 2"]
+
0.05
&CenterDot;
0.011
&CenterDot;
[HSL/LuxR]
&CenterDot;
["Gen 2"]
-
V
c2
&CenterDot;
0.0270104
&CenterDot;
["complex 3"]
-
V
c2
&CenterDot;
0.0261438
&CenterDot;
["complex 3"]
d
["Gen 2"]
&CenterDot;
V
c2
d
t
=
-
V
c2
&CenterDot;
0.04
+
0.05
&CenterDot;
[HSL/LuxR]
1
+
0.04
+
0.05
&CenterDot;
[HSL/LuxR]
+
0.011
&CenterDot;
["Gen 2"]
+
0.05
&CenterDot;
0.011
&CenterDot;
[HSL/LuxR]
&CenterDot;
["Gen 2"]
Finally, values of constants per reaction that occurs inside “PLAdora” bacteria are in Table 2.
Table 2: Kinetic and values of constants reaction for “PLAdora” bacteria.
Name
Reaction
Equations
Comments
References
d10
phaC -> dphaC
Mass action (irreversible)
k1(d10)= 0.00016
[1]
d11
Phasin-Hly -> dPhasin-Hly
Mass action (irreversible)
k1(d10)= 0.00016
[1]
d11,2
mPhasin-Hly -> dmPhasin-Hly
Mass action (irreversible)
k1(d11,2)=4e-7
[2]
d8
LuxR -> dLuxR
Mass action (irreversible)
k1(d8)= 0.00016
[1]
d8,1
mLuxR -> dmLuxR
Mass action (irreversible)
k1(d8,1)=4e-7
[2]
d9
P-CoA-T -> dP-CoA-T
Mass action (irreversible)
k1(d9)= 0.00016
[1]
d9,2
mP-CoA-T -> dmP-CoA-T
Mass action (irreversible)
k1(d9,2)=4e-7
[2]
d9,3
mphaC -> dmphaC
Mass action (irreversible)
k1(d9,3)= 0.00016
[2]
E1
Lactate{default} = Lactate{c2}
Reversible Michaelis -Menten
Kms(E1)=0.00054,
Kmp(E1)=0.015,
Vf(E1)=4.1667e-9,
Vr(E1)=1.25e-8
[8]
E2
HSL{default} -> HSL{c2};  HSL{c2}
Rate Law for E2
D(E2)= 0.127892,
k1(E2)=0.0438,
fl(E2)=8e-7,
k2(E2)=1.67e-7
[3]
G2
5'-methylthioadenosine -> S-adenosyl-L-methionine
Mass action (irreversible)
k1(G2)=0.1
[-]
G3
[acyl-carrier protein] -> "3-oxohexanoyl-[acyl-carrier protein]"
Mass action (irreversible)
k1(G3)=0.1
[-]
K10
mphaC -> phaC
Mass action (irreversible)
k1(K10)=0.014795
[4]
K11
mPhasin-Hly -> Phasin-Hly
Mass action (irreversible)
k11(K11)=0.036244
[4]
K11,1
HSL/LuxR + Phasin-Hlyg = "complex 4"
Rate Law for K11,1
c0(K11,1)=0.04,
c1(K11,1)=0.05,
c2(K11,1)=0.011
[9]
K11,2
complex 4 -> mPhasin-Hly
Mass action (irreversible)
k1(K11,2)=0.0577428
[4]
K12
Phasin-Hly + PLA = PLA-Phasin-Hly{c2}
Mass action (irreversible)
k1(K12)=4.73843e-9,
k2(K12)=4.7843e-10
[-]
K13
S-adenosyl-L-methionine + "3-oxohexanoyl-[acyl-carrier protein]" = HSL{c1} + 5'-methylthioadenosine + "[acyl-carrier protein]";  LuxI
Rate Law for K13
Vf(K13)=0.2666,
Keq(K13)=1,
Ki3(K13)=6e-6,
Km1(K13)=2.3e-5,
Km2(K13)=1.5e-5,
Vr(K13)=0.03333,
Km4(K13)=4.4e-6,
Ki1(K13)=2.4e-6,
Km3(K13)=5e-6,
Ki2(K13)=2.5e-6
[10]
K5
LuxR + HSL{c2} = HSL/LuxR
Rate Law for K5
K(K5)=2.7e-7
[11]
K6
Lactate{c2} + acetyl-CoA = Lactyl-CoA + Acetate;  P-CoA-T
Rate Law for K6
por(K6)=0.38,
Vf(K6)=1.36667e-7,
Kms(K6)=0.0006,
Vr(K6)=4.16667e-8,
Kmp(K6)=0.0003
[12]
K7
Lactyl-CoA = PLA;  phaC
Rate Law for K7
Vf (K7)=1.19e-5,
Keq(K7)=1,
Kms(K7)=0.000119,
Kmp(K7)=8.5e-5
[13], [14]
K8
mLuxR -> LuxR
Mass action (irreversible)
k1(K8)=0.0266026
[4]
K8,1
LuxRg -> mLuxR
Mass action (irreversible)
k1(K8,1)=0.0470085
[4]
K9
mP-CoA-T -> P-CoA-T
Mass action (irreversible)
k1(K9)=0.0152855
[4]
K9,1
HSL/LuxR + "Gen 2" = "complex 3"
Rate Law for K11,1
c0(K9,1)=0.04,
c1(K9,1)=0.05,
c2(K9,1)=0.011
[15]
K9,2
complex 3 -> mP-CoA-T
Mass action (irreversible)
k1(K9,2)=0.0270104
[4]
K9,3
complex 3 -> mphaC
Mass action (irreversible)
k1(K9,3)=0.0261438
[4]
S2
HSL{c1} -> HSL{default};  HSL{default}
Rate Law for S2
D(S2)= 0.127892,
k1(S2)=0.0438,
fl(S2)=8e-7,
k2(S2)=1.67e-7
[3]
S3
PLA-Phasin-Hly{c2} -> PLA-Phasin-Hly{default}
Mass action (irreversible)
k1(S3)=6.944e-11
[16], [17]
The whole system looks like Figure 4, where lactate performs two roles: regulate its own production, and be captured by “PLAdora” bacteria, to produce PLA. HSL plays a similar part, be produced by the first bacteria, and regulating the polymerization and exportation of PLA.
FIGURE 4. Diagram for the whole bacteria system. Download PLA Model CellDeigner
The equations related cell growing and reactions outside bacteria are:
d
V
c1
d
t
=
V
c1
+
0.8
&CenterDot;
5
7
&CenterDot;
3600
&CenterDot;
[Lactate{default}]
[Lactate{default}]
+
9.8e-3
&CenterDot;
1
-
"[Lactic acid]"
0.448
d
V
c2
d
t
=
V
c2
+
0.8
&CenterDot;
5
7
&CenterDot;
3600
&CenterDot;
[Lactate{default}]
[Lactate{default}]
+
9.8e-3
&CenterDot;
1
-
"[Lactic acid]"
0.448
d
["H+{default}"]
&CenterDot;
V
default
d
t
=
-
4.1667e-009
&CenterDot;
["H+{default}"]
0.00054
-
1.25e-008
&CenterDot;
["H+{c1}"]
0.015
1
+
["H+{default}"]
0.00054
+
["H+{c1}"]
0.015
-
V
default
&CenterDot;
0.000138
&CenterDot;
["H+{default}"]
&CenterDot;
[Lactate{default}]
-
1
&CenterDot;
["Lactic acid"]
d
[Lactate{default}]
&CenterDot;
V
default
d
t
=
-
V
default
&CenterDot;
0.000138
&CenterDot;
["H+{default}"]
&CenterDot;
[Lactate{default}]
-
1
&CenterDot;
["Lactic acid"]
+
1.25e-008
&CenterDot;
[Lactate{c1}]
0.015
-
4.16667e-009
&CenterDot;
[Lactate{default}]
0.00054
1
+
[Lactate{c1}]
0.015
+
[Lactate{default}]
0.00054
-
4.1667e-009
&CenterDot;
[Lactate{default}]
0.00054
-
1.25e-008
&CenterDot;
[Lactate{c2}]
0.015
1
+
[Lactate{default}]
0.00054
+
[Lactate{c2}]
0.015
d
["Lactic acid"]
&CenterDot;
V
default
d
t
=
+
V
default
&CenterDot;
0.000138
&CenterDot;
["H+{default}"]
&CenterDot;
[Lactate{default}]
-
1
&CenterDot;
["Lactic acid"]
d
[HSL{default}]
&CenterDot;
V
default
d
t
=
-
0.127892
&CenterDot;
[HSL{default}]
-
[HSL{c2}]
+
0.0438
&CenterDot;
8e-007
-
1.67e-007
&CenterDot;
[HSL{default}]
+
0.127892
&CenterDot;
[HSL{c1}]
-
[HSL{default}]
+
0.0438
&CenterDot;
8e-007
-
1.67e-007
&CenterDot;
[HSL{c1}]
d
["PLA-Phasin-Hly{default}"]
&CenterDot;
V
default
d
t
=
+
6.944e-011
&CenterDot;
["PLA-Phasin-Hly{c2}"]
The next stage of the process was to adjust parameters, to obtain the right simulation of the whole system. The results we obtained didn't show the tendencies we were expected, and finding the solution is a work proposed for a future interested iGEM team. We uploaded both programs (CellDesigner and COPASI).
Although we couldn’t obtain the final simulation, the process of modeling taught us this is a complex area of the project, and if the bacterial system is complex, of course the modeling would be.
Another thing we want to comment is our experience utilizing TinkerCell. We started using this program to construct and simulate our project, and we took the time to learn about how it works, but with the course of the days we had many problems with it. First it was problems related to the cellular network construction itself, and then it was problems to open the program! For that reason then we moved to CellDesigner and COPASI to model the system, so we need to utilized extra time to learn about how other programs work.
We think that we didn't predict correctly the necessary times per each area of the project, because this is the first time we work in something similar. But now, we have the experience to tell to other new teams that this work is so hard, and it's importante to start working on it as early as they can.
Nomenclature
The nomenclature we used, both here and in the program, is shown in Table 4.
Table 4: Nomenclature used.
mX
Messenger RNA protein X
dmX
product of messenger RNA degradation of protein X
X{c1}
metabolite contained in the cell 1
X{c2}
metabolite contained in the cell 2
X{default}
metabolite content in the culture medium
dX
degradation product of protein X
complex 1
complex formed by the TetR gene and H +
complex 2
complex formed by the TetR protein and gene 1
complex 3
complex formed by the HSL/LuxR complex and gene 2
complex 4
complex formed by the HSL/LuxR and Phasin-Hly gene
HSL
homoserine lactone
P-CoA-T
Propionate CoA transferase
Xg
gene metabolite X
Gen 1
Gene containing the genes D-LDH and LuxI
Gen 2
Gene containing the genes Lux and phaC
References
[1] N. N. Starkova, E. P. Koroleva, L. D. Rumsh, L. M. Ginodman, and T. V. Rotanova, “Mutations in the proteolytic domain of Escherichia coli protease Lon impair the ATPase activity of the enzyme,” FEBS Lett., vol. 422, no. 2, pp. 218–220, 1998.
[2] V. J. Cannistraro and D. Kennell, Escherichia coli ribonuclease II, vol. 342, no. 1964. Elsevier Masson SAS, 2001.
[3] P. Nilsson, a Olofsson, M. Fagerlind, T. Fagerström, S. Rice, S. Kjelleberg, and P. Steinberg, “Kinetics of the AHL regulatory system in a model biofilm system: how many bacteria constitute a ‘quorum’?,” J. Mol. Biol., vol. 309, no. 3, pp. 631–640, 2001.
[4] S. Proshkin, A. R. Rahmouni, A. Mironov, and E. Nudler, “Cooperation between translating ribosomes and RNA polymerase in transcription elongation.,” Science, vol. 328, no. 5977, pp. 504–8, Apr. 2010.
[5]  K. Liu, X. Zeng, L. Qiao, X. Li, Y. Yang, C. Dai, A. Hou, and D. Xu, “The sensitivity and significance analysis of parameters in the model of pH regulation on lactic acid production by Lactobacillus bulgaricus.,” BMC Bioinformatics, vol. 15 Suppl 1, no. Suppl 13, p. S5, 2014.
[6] V. Sotiropoulos and Y. N. Kaznessis, “Synthetic tetracycline-inducible regulatory networks: computer-aided design of dynamic phenotypes.,” BMC Syst. Biol., vol. 1, p. 7, 2007.
[7] M. H. N. Hoefnagel, M. J. C. Starrenburg, D. E. Martens, J. Hugenholtz, M. Kleerebezem, I. I. Van Swam, R. Bongers, H. V Westerhoff, and J. L. Snoep, “Metabolic engineering of lactic acid bacteria, the combined approach: kinetic modelling, metabolic control and experimental analysis,” Microbiology-Sgm, vol. 148, pp. 1003–1013, 2002.
[8] E. B. Olsen, J. B. Russell, and T. Henick-Kling, “Electrogenic L-malate transport by Lactobacillus plantarum: A basis for energy derivation from malolactic fermentation,” J. Bacteriol., vol. 173, no. 19, pp. 6199–6206, 1991.
[9] J. J. Tabor, H. M. Salis, Z. B. Simpson, A. a. Chevalier, A. Levskaya, E. M. Marcotte, C. a. Voigt, and A. D. Ellington, “A Synthetic Genetic Edge Detection Program,” Cell, vol. 137, no. 7, pp. 1272–1281, 2009.
[10] B. L. Hanzelka, M. R. Parsek, D. L. Val, P. V. Dunlap, J. E. Cronan, and E. P. Greenberg, “Acylhomoserine lactone synthase activity of the Vibrio fischeri AinS protein,” J. Bacteriol., vol. 181, no. 18, pp. 5766–5770, 1999.
[11] P. D. Pérez, J. T. Weiss, and S. J. Hagen, “Noise and crosstalk in two quorum-sensing inputs of Vibrio fischeri,” BMC Syst. Biol., vol. 5, no. 1, p. 153, 2011.
[12] E. Volodina, M. Schürmann, N. Lindenkamp, and A. Steinbüchel, “Characterization of propionate CoA-transferase from Ralstonia eutropha H16.,” Appl. Microbiol. Biotechnol., vol. 98, no. 8, pp. 3579–89, 2014.
[13] Y. M. Normi, T. Hiraishi, S. Taguchi, H. Abe, K. Sudesh, N. Najimudin, and Y. Doi, “Characterization and properties of G4X mutants of Ralstonia eutropha PHA synthase for poly(3-hydroxybutyrate) biosynthesis in Escherichia coli,” Macromol. Biosci., vol. 5, no. 3, pp. 197–206, 2005.
[14] J. J. Song, S. Zhang, R. W. Lenz, and S. Goodwin, “In vitro polymerization and copolymerization of 3-hydroxypropionyl-CoA with the PHB synthase from Ralstonia eutropha.,” Biomacromolecules, vol. 1, no. 3, pp. 433–439, 2000.
[15] J. J. Tabor, H. M. Salis, Z. B. Simpson, A. a. Chevalier, A. Levskaya, E. M. Marcotte, C. a. Voigt, and A. D. Ellington, “A Synthetic Genetic Edge Detection Program,” Cell, vol. 137, no. 7, pp. 1272–1281, 2009.
[16] A. Rahman, E. Linton, A. D. Hatch, R. C. Sims, and C. D. Miller, “Secretion of polyhydroxybutyrate in Escherichia coli using a synthetic biological engineering approach.,” J. Biol. Eng., vol. 7, no. 1, p. 24, 2013.
[17] C. Sam, S. Nicol, T. S. J. Simini, and T. S. Cc, “CAMBIOS EN EL PESO MOLECULAR, BIODEGRADACIÓN Y CRISTALINIDAD DE POLIHIDROXIALCANOATOS SOMETIDOS A IRRADIACIÓN GAMMA,” no. 1, pp. 1789–1794, 2007.
[18] UAI, “Tablas de Constantes.” [Online]. Available: http://www.uia.mx/campus/publicaciones/quimanal/pdf/tablasconstantes.pdf.
Bioreactor modeling
What is a bioreactor?
Is an equipment which keeps a controlled active biological environment and is used to carry out a transformation due to the performance of a biocatalyst (1). In this project, these biocatalysts are bacteria.
Why is important a bioreactor?
“The heart of a typical bioprocess is the reactor or fermenter. Flanked by unit operations which carry out physical changes for medium preparation and recovery of products, the reactor is where the major chemical and biochemical transformations occur. In many bioprocesses, characteristics of the reaction determine to a large extent the economic feasibility of the project.”[2]
How works a bioreactor?
A bioreactor is an equipment in which some substrates, from the culture, are transformed to a product through the performance of bacteria. A bioreactor gives all necessary conditions for the culture, like mixing, temperature regulation, oxygen supply, substrates ports, sampling ports, pH control, etc. Then, within bioreactor is carrying out a bio-reaction, because  this equipment gives operation conditions for a complete reaction. There are three ways to carry out the reaction after charging the bioreactor with cell y substrate: giving a continuous feed (substrate), giving a semi-continuous feed and without feed, they are called a continuous, fed-batch and batch bioreactor respectively [3].
In this project, a continuous bioreactor was used. Thus, a continuous feed rate is equal to an outflow rate, maintaining a constant volume within bioreactor. Once PLA is produced, it is got from the outflow and then will be purified to use it to make any PLA product. After then we could consider a cell recirculation system for recovering PLA and cells responsible for the excretion to be utilized again in the reactor.
Which are the goals of a bioreactor?
These are the goals of a bioreactor[1]
Keep cells distributed as uniformly as possible.
Keep temperature constant and homogeneous
Minimize nutrient concentration gradients
Preventing sedimentation and flocculation
Allow gas diffusion
Aseptic environment culture
Keep safe and reproducible sampling ports
Foam control
Maximize performance and production
Reduce production costs
Optimize volume
Reduce the reaction time
What is the advantage of this project’s bioreactor design?
Due to it uses genetically modified bacteria to transform glucose to PLA, which is released to the culture, it is not necessary to kill these bacteria to get PLA, because in this project, PLA is purified and characterized after its production within bacteria. That’s why the most important advantage of this bioreactor, over chemical bioreactor, is an environmental friendly reactor, because it does not pollute a the air, water neither land.
Besides, bacteria do not need to grow in an environment with high temperatures(between 28-30ºC [3])  or pressures, then the bioreactor does not use a great amount of energy (electrical energy or heat, comparing with the temperatures of chemical reactor that are about 130ºC [4]), being a low cost reactor.
In conclusion, this bioreactor is a sustainable equipment in terms of energy demand for a biological system and nutrient availability.
Equations related to bioreactor
Before writing the equations, it’s important to understand what is happening inside the bioreactor. That’s why a mass balance for the system is showed [2]:
{
M
a
s
s
 
a
c
c
u
m
u
l
a
t
e
d
 
w
i
t
h
i
n
 
s
y
s
t
e
m
}
=
{
M
a
s
s
 
i
n
 
t
h
r
o
u
g
h
 
t
h
e
 
s
y
s
t
e
m
 
b
o
u
n
d
a
r
i
e
s
}
−
{
M
a
s
s
 
o
u
t
 
t
h
r
o
u
g
h
 
t
h
e
 
s
y
s
t
e
m
 
b
o
u
n
d
a
r
i
e
s
}
+
{
M
a
s
s
 
g
e
n
e
r
a
t
e
d
 
w
i
t
h
i
n
 
s
y
s
t
e
m
}
−
{
M
a
s
s
 
c
o
n
s
u
m
e
d
 
w
i
t
h
i
n
 
s
y
s
t
e
m
}
An example of a bioreactor, is a CSTR (continuous stirred-tank reactor) where the feed rate is equal to outflow, maintaining a constant culture volume. In figure 1 is showed a diagram of a CSTR.
Figure 1. Diagram of a CSTR. Where
 
F
i
 
=
 
F
e
e
d
 
r
a
t
e
;
 
C
A
i
=
 
F
e
e
d
 
c
o
n
c
e
n
t
r
a
t
i
o
n
 
o
f
t
h
e
 
c
o
m
p
o
n
e
n
t
 
A
;
 
p
i
=
 
f
e
e
d
 
d
e
n
s
i
t
y
;
 
F
o
 
=
 
o
u
t
f
l
o
w
 
c
o
n
c
e
n
t
r
a
t
i
o
n
;
 
C
A
o
=
o
u
t
f
l
o
w
 
c
o
n
c
e
n
t
r
a
t
i
o
n
;
 
p
o
=
 
o
u
t
f
l
o
w
 
d
e
n
s
i
t
y
Then, considering the mass balance, we can write the following equations [5]:
Global mass balance
d
(
ρ
o
*
V
)
d
t
=
F
i
*
ρ
i
−
F
o
*
ρ
o
(1)
Where:
ρ
o
 
=
 
c
u
l
t
u
r
e
 
d
e
n
s
i
t
y
 
(
k
g
/
m
3
 
)
V
 
=
 
c
u
l
t
u
r
e
 
v
o
l
u
m
e
 
(
m
3
)
F
i
 
=
 
f
e
e
d
 
r
a
t
e
 
(
m
3
 
/
 
h
)
ρ
o
 
=
 
f
e
e
d
 
d
e
n
s
i
t
y
 
(
k
g
/
m
3
 
)
F
o
 
=
 
o
u
t
f
l
o
w
 
(
m
3
 
/
 
h
)
ρ
o
 
=
 
o
u
t
f
l
o
w
 
d
e
n
s
i
t
y
 
(
k
g
/
m
3
 
)
Supposed:
1.   Constant density all the time
 
(
ρ
o
=
ρ
i
 
)
2.   Constant volume all the time
3.   Steady state
In conclusion, we have:
F
i
=
F
o
=
F
(2)
For next equations, it’s supposed the following scheme showed in the figure 2:
Figure 2. Scheme of the bioreactor. The substrate is glucose and the product is PLA. Within bioreactor are the two genetically modified bacteria.
Cell balance
F
*
X
i
−
F
*
X
o
+
 
µ
*
X
o
*
V
−
α
*
X
o
*
V
=
d
(
X
o
*
V
)
d
t
=
X
o
*
d
V
d
t
+
V
*
d
X
o
d
t
(3)
Where:
X
i
,
o
 
=
 
c
e
l
l
 
c
o
n
c
e
n
t
r
a
t
i
o
n
 
(
k
g
/
m
^
3
)
 
(
f
e
e
d
 
r
a
t
e
 
a
n
d
 
o
u
t
f
l
o
w
)
µ
 
=
 
s
p
e
c
i
f
i
c
 
g
r
o
w
t
h
 
r
a
t
e
 
o
f
 
t
h
e
 
b
a
c
t
e
r
i
a
 
(
1
/
h
)
α
=
 
d
e
a
t
h
 
r
a
t
e
 
o
f
 
t
h
e
 
b
a
c
t
e
r
i
a
 
(
1
/
h
)
Supposed:
1.   Constant volume all the time
 
(
d
V
d
t
 
=
 
0
)
2.   Steady state
 
(
 
d
x
d
t
=
0
)
3.   There are not cells in the feed
 
(
x
o
)
4.   Death rate(α) is less than the specific growth rate(µ)
Then we have:
−
F
*
X
o
+
µ
*
X
o
*
V
=
0
 
 
 
F
*
X
=
µ
*
X
 
*
V
(4)
F
V
=
µ
(5)
And we know that the dilution is
D
 
=
F
/
V
 
 
 
D
=
µ
Substrate balance
F
*
G
i
−
F
*
G
o
−
µ
*
X
o
*
V
Y
X
G
−
m
S
*
X
o
*
V
−
q
P
*
X
o
*
V
Y
P
G
 
=
d
(
G
o
*
V
)
d
t
=
G
o
*
d
(
V
)
d
t
+
V
*
d
(
G
o
)
d
t
(6)
Where:
G
i
,
o
 
 
=
 
g
l
u
c
o
s
e
 
c
o
n
c
e
n
t
r
a
t
i
o
n
 
(
f
e
e
d
 
r
a
t
e
 
a
n
d
 
o
u
t
f
l
o
w
)
(
k
g
/
m
3
)
µ
=
 
s
p
e
c
i
f
i
c
 
g
r
o
w
t
h
 
r
a
t
e
 
(
1
/
h
)
X
o
 
 
=
 
c
e
l
l
 
c
o
n
c
e
n
t
r
a
t
i
o
n
 
(
 
k
g
/
m
3
)
Y
X
G
 
 
 
=
 
c
o
n
v
e
r
s
i
o
n
 
o
f
 
c
e
l
l
 
r
e
f
e
r
r
e
d
 
t
o
 
c
o
n
s
u
m
e
d
 
g
l
u
c
o
s
e
 
(
 
k
g
 
c
e
l
l
 
/
 
k
g
 
g
l
u
c
o
s
e
 
)
m
S
 
 
=
 
m
a
i
n
t
e
n
a
n
c
e
 
c
o
e
f
f
i
c
i
e
n
t
(
1
/
h
)
 
q
P
=
 
s
p
e
c
i
f
i
c
 
p
r
o
d
u
c
t
 
f
o
r
m
a
t
i
o
n
 
r
a
t
e
 
(
k
g
 
p
r
o
d
u
c
t
/
 
k
g
 
c
e
l
l
 
/
 
h
)
 
Y
P
G
=
 
c
o
n
v
e
r
s
i
o
n
 
o
f
 
p
r
o
d
u
c
t
 
r
e
f
e
r
r
e
d
 
t
o
 
c
o
n
s
u
m
e
d
 
g
l
u
c
o
s
e
 
(
 
k
g
 
p
r
o
d
u
c
t
/
 
k
g
 
g
l
u
c
o
s
e
 
)
Supposed:
1.
m
S
*
 
X
o
 
<
<
 
µ
*
X
o
2. Steady state
(
 
d
(
V
)
d
t
=
0
 
 
y
 
 
d
(
G
o
)
d
t
=
0
 
)
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
Then we have:
F
*
G
i
−
F
*
G
o
−
µ
*
X
o
*
V
Y
X
G
−
q
P
*
X
o
*
V
Y
P
G
 
=
0
(7)
Dividing by:
F
V
*
(
G
i
−
G
o
)
−
X
o
 
*
 
(
 
 
µ
Y
X
G
−
q
P
Y
P
G
 
 
)
 
=
0
(8)
Using equation 5:
X
o
=
µ
*
(
G
i
−
G
o
)
 
(
 
µ
Y
X
G
−
q
P
Y
P
G
)
(9)
Product balance
F
*
P
L
A
i
−
F
*
P
L
A
o
+
q
P
L
A
*
X
o
*
V
=
d
(
P
L
A
*
V
)
d
t
(10)
Where:
P
L
A
i
,
o
 
=
 
P
L
A
 
c
o
n
c
e
n
t
r
a
t
i
o
n
(
k
g
/
m
^
3
)
 
(
f
e
e
d
 
r
a
t
e
 
a
n
d
 
o
u
t
f
l
o
w
)
q
P
L
A
=
 
s
p
e
c
i
f
i
c
 
P
L
A
 
f
o
r
m
a
t
i
o
n
 
r
a
t
e
 
(
k
g
 
p
r
o
d
u
c
t
/
 
k
g
 
c
e
l
l
 
/
 
h
)
 
Supposed:
1.  There is not PLA in feed rate
(
P
L
A
i
 
=
0
)
2.  Steady state
(
 
d
P
L
A
d
t
 
=
0
 
)
3.
q
P
L
A
known(experimental data)
Then we have:
q
P
L
A
*
X
o
*
V
=
 
F
*
P
L
A
o
(11)
For instance, if we want to produce a
F
*
P
L
A
 
(
K
g
/
h
)
Known, we will be able to calculate the culture volume using the equation 11:
V
=
 
F
*
P
L
A
o
q
P
L
A
*
X
o
Is this PLA as good as the standard PLA?
After the production of PLA, we have to purify and characterize our polymer to know the most important characteristics of it. So, we’ll use the following protocol to purify and characterize PLA.
Protocol to purify PLA
Recovery of secreted PLA. This protocol is carry out according to a similar protocol used to recover PHB [6]:
1.   At 24 and 48 hours, from the beginning of the culture, add CaCl2 till get a concentration of 0,01M within system. Then mix them by inverting the tube several times.
2.   Let tube rest for 10 minutes at room temperature and centrifuge it at 54g for 5 minutes.
3.   Remove the supernatant and transfer it to a fresh tube, then freeze dry the pellet.
4.   Centrifuge the supernatant (in the fresh tube) at 3452g for 10 minutes and freeze dry the pellet.
In conclusion, PLA and CaCl2 are in the pellet from the first centrifugation and the pellet from the second centrifugation contains bacterial mass and non-secreted PLA.
After having the purified PLA, it is necessary characterize it. Thus, the group used the next protocol.
Protocol to characterize the purified PLA
To see functional groups it will be required an IR Transmission (because the necessary equipment to do this characterization is in FCFM) and to evaluate thermal parameters will be required a DSC analysis. The protocol to follow is divided in three steps: the first step is necessary to dry PLA to ensure the elimination of moisture traces and to avoid any undesirable hydrolysis reaction [5], the second step is making a PLA film and the final step is characterize.
Removing moisture
Dry PLA in an oven heating it at 98°C for 3 hours.
Making a PLA based film
PLA based film with thicknesses between 20 and 60 µm was obtained by extrusion with the adequate filming die. Screw speed at 100 rpm was used to optimize the material final properties, while the temperature profile was set up at 180-190-200 °C in the three different extruder heating zones to ensure the complete processing of all systems. The total processing time was established at 6 min. Neat PLA was mixed for 6 min[5].
IR
Infrared spectroscopy analysis was performed at room temperature in transmission and reflection modes by using a spectrometer (from FCFM) at a wavenumber range 4000-400 cm-1[7].
Thermal analysis (DSC)
Tests were performed from 25 °C to 200 °C at 10 °C min-1 under nitrogen flow rate, with two heating and one cooling scans. The first heating scan was used to erase all thermal history. All measurements were performed in triplicate and data were reported as the mean value ± standard deviation[7].
References
[1]Améstica, Luis. 2013. BIOREACTORES 2013.[Slides]
[2] Pauline M. Doran. 1995. Bioprocess Engineering Principles. San Diego, U.S.A. Academic Press. 257p.
[3] M. Gumel, M. Annuar , Y. Chisti. Recent Advances in the Production, Recovery and Applications of Polyhydroxyalkanoates. [online]                       http://download.springer.com/static/pdf/755/art%253A10.1007%252Fs10924-012-0527-1.pdf?originUrl=http%3A%2F%2Flink.springer.com%2Farticle%2F10.1007%2Fs10924-012-0527-1&token2=exp=1442075234~acl=%2Fstatic%2Fpdf%2F755%2Fart%25253A10.1007%25252Fs10924-012-0527-1.pdf%3ForiginUrl%3Dhttp%253A%252F%252Flink.springer.com%252Farticle%252F10.1007%252Fs10924-012-0527-1*~hmac=82054d4c4e0398948ab429d62a1fc0d87abe11f525673e4e785bddc44698801a [consulted: 12-09-2015]
[4] Garlotta, 2001. A Literature Review of Poly(Lactic Acid). Journal of Polymers and the Environment, Vol. 9, No. 2.
[5] Lienqueo, María Elena. 2015. Diseño de Bio-reactores, Cultivos continuos, Fermentación e Igeniería Metabólica.
[6] Asif Rahman, Elisabeth Linton, Alex D Hatch, Ronald C Sims and Charles D Miller: Secretion of polyhydroxybutyrate in Escherichia coli using a synthetic biological engineering approach.  Journal of Biological Engineering 2013,7:24.
[7] Ilaria Armentano, Elena Fortunati, Nuria Burgos, Franco Dominici, Francesca Luzi, Stefano Fiori, Alfonso Jiménez, Kicheol Yoon, Jisoo Ahn, Sangmi Kang, José M. Kenny, Bio-based PLA_PHB plasticized blend films: Processing and structural characterization, LWT - Food Science and Technology, Volume 64, Issue 2, December 2015, Pages 980-988, ISSN 0023-6438, http://dx.doi.org/10.1016/j.lwt.2015.06.032.
Our Sponsors
