Team:UChile-OpenBio/Notebook
TODO supply a title
HOME
PROJECT
PROJECT DESCRIPTION
EXPERIMENT AND PROTOCOLS
RESULTS
NOTEBOOK
ATTRIBUTIONS
COLLABORATION
HUMAN PRACTICES
SAFETY
MODELING
TEAM
Notebook
Previous activities: The beginning
We formed as an iGEM group in 2014, when we started to have recurrent meetings for creating new ideas of possible projects of synthetic biology, including some projects in the Centre of Bioengineering and Biotechnology. In our first meeting we made several brainstorming considering two main objectives: to generate new ideas of projects and assess the feasibility in terms of experimental design.
The generated ideas were related to solve problem of routine laboratory methodologies, sustentabilities issues, and other problems of molecular biology. After several brainstorming activities we gather several projects that needed to be processed and assessed its feasibility. One of the meetings of the recently formed iGEM group, in that time called just “OpenBio”, we met the Fab-LAB, a group that were interested in a new form of producing a raw material used for 3D printing, Polylactic Acid (PLA). Until that we saw different ways of how to produce polymers of different kind in a biological system, comprehending the metabolic pathways known and searching for new genetic sources. Then, the project of PLA production in a biological system using synthetic biology tools was formed.
Previous activities: Preparation of the design and complementary activities
After the formation of the PLA project was elected by unanimity, we started to work in the design of the project and in the molecular biology needed to achieve a biological system controlled by a regulatory network that accomplished the principles of synthetic biology. Then, the first ideas of a regulatory network were created using the microorganism E. coli as a host. PLA is actually produced by chemical synthesis using an old system called O’Ring polymerization, so then a biological system should have comparable yields and low cost of purification process. Then the first conception was to enable a biological system that could handle a large production of recombinant proteins derived of a synthetic pathway, and that could be achieved using a controlled system based in the kinetic growth of the microorganism. Thus, a quorum sensing based expression system could handle the metabolic burden involved in a new pathway, and the production of PLA could be sustainable in time. The second concept is to improve the extraction processes of PLA after fermentations. Lately, biological production of PLA consider harvesting cell pellet and then destroy cell membrane to extract intracellular PLA. The significant costs associated to this process could be improved implementing a PLA export system that could be similar to a PHA phasins export systems reported. So the goal of the project is to design all of these genetic circuits properly regulated by the cell growth and the other physiological conditions.
Week 1: The design
The design of the project is divided in three stages. First the design of the genetic circuits from different biological sources. We first define the synthesis of PLA and lactic acid with the currently conventional gene codifying enzymes and mutated ones. The main enzymes are poly-hydroxyalcanoate synthase (PhaC) PLA polymerization and propionyl-CoA-Transferase (P-Co-A-T) for lactyl-CoA substrate production. We decided to synthesize and assembly the whole metabolic pathway of PLA production regulated by quorum sensing and PLA exportation using gBlocks assembly from IDT technologies.
Week 2: Design of gblock for ordering synthesis
The initial design of the gBlocks differ significantly from the final one. First we considered the design of the enzyme including the respective promoters, but then we hypothesized how could we discriminate between the behavior of promoter and their regulations and the enzyme activities of the metabolic pathway?. So then, we designed the sequences only with the ORF and suffix and prefix for the standard enzyme restriction assembly. After that, we have trouble ordering because our sequences have low GC% and could be able to be synthesized only changing bases using silence mutations of the conceived genes. The gBlocks of the partial part of ORF were designed considering the overhang for gibson assembly, so we did not order primers for reamplification.
Week 3: First experiments
The Gibson assembly g blocks has arrived!, So the first experiments will be performed.  The order was read carefully and the incubation of gBlocks was done by the manufacturer instructions. All the gBlocks consist in the parts required for the genetic circuit and the some of them must be assembled by Gibson assembly, especially the gene codifying enzymes of the metabolic pathway.  Unfortunately we had problems with gBlocks, because apparently the concentration was much lower than the specified. This issue produce a delay in our program of activities, when we decided to re-amplify them designing and ordering new primers.
We tried to calculate the concentration of all the gBlocks to proceed the assembly, unfortunately we would not be able to complete the process, because of low concentration of templates. The first Gibson assembly of Templates was LDH gblocks, with low concentration we fairly could able to generate the desired product (1237 bp).  Other assemblies like LuxI failed in the assembly attempt.
Week 4: Continuation...
The LDH module and the other gBlocks  that do not require assembly would be able to obtain and reamplify (pLux, prfcB, pBAD, pC, SegA), using the corresponding primers PxSx fwd and rev.  Then these gBlocks are quantified for digestion and ligation in the corresponding vectors. We test the integrity and linearity of the vector pSB1A3, pSB1C3 and pSB1K3 in an electrophoresis gel (view protocol available). The digestion Protocol is performed for all the amplified gBlocks.
Week 5:  The problem of digestions
Until we wait the primers for the amplification of all the unsuccessful gblocks assemblies, we perform several digestion for ligate gblocks into vector. We had several problem with enzymes that did not work properly for several causes. We first use the digestion protocol stablished by Standard Biobrick assembly protocol but we had problem of insufficient DNA. Then we increased the concentration of DNA and performed with different NEB buffers for EcoRI and PstI enzymes. This process take us the entire week.
Week 6: The problem of digestion completed
We successfully completed the digestions of gBlocks increasing the concentration of DNA up to 1 ug in a 50 uL reaction, then we doubled the reaction volume to obtain more digestion product and then purified by geneJET gel extraction kit. After the purification process the final DNA concentration is reduced, but still the digestion were possible.
Week 7: We received the primers for intermediate gBlocks  amplificaction!
We finally received the primers for intermediate amplification of gBlocks, this oligonucleotides are crucial for assembly of modules that are assembled using more than 2 blocks. These are PhaC, pCoAT1 and T2 and we started PCR pre-amplification process of all the gblocks.  We first assess the assembly of 2   gBlocks, and then the partial fragments to build the whole module.  We successfully assembly the first 2 gblocks of each module, but in the second stage there are some incorrect band indicating prescence of secondary structures in the isothermal reaction
Week 8: Preparation of chemocompetent cells and electrocompetent cells
We performed the procotols of chemocompetent cells and electrocompetent cells on this week, for preparation of biological material of E. coli DH5-alpha  for cloning purposes and preparation of vectors for biobrick assembly with ER
Week 9: Problems of ligation
We had several attempts of ligation with unsuccessful results. Unfortunately we depleted all of the pSB1A3 vectors, so we performed a recue recircularization of vector and then transformed DH5-alpha chemocompetent cells. This will perpetuate the vector for further cloning purposes.
Week 10: Standardization of digestion and ligation cells
We have changed the enzyme PstI that were making partial cuts in sequences and the ligation were more efficient. We successfully cloned the gblocks SegA, pLux, prfcB, pLux and pC. Now are moving forward to digest and ligate PhaC and pCoAT1 and T2. After all these ligation are completed we must order sequence services for all the correctly assembled modules. Then, the second stage of our project start, expression of modules in a expression E. coli BL21 system
Our Sponsors
