Team:IONIS-PARIS/labwork/labbook
Toggle navigation
PROJECT
DESCRIPTION
DEFINITIONS
BRAINSTORMING PROCESS
MODELING
DSBOX + AmilCP
RNA FOLDING SIMULATION
LABORATORY WORK
PARTS
PROTOCOLS
RESULTS & CHARACTERISATIONS
WORKING AT LA PAILLASSE
LABORATORY NOTEBOOK
APPLIED DESIGN
OVERVIEW
FOLIAR APPLICATION
CHASSIS SELECTION
PROTECTIVE COMPOUNDS
RISK AVOIDANCE: BIOSAFETY STRATEGIES
RISK ASSESSMENT: TOXICITY & ECOTOXICITY STUDIES
CASE STUDY: FROSTBAN
HUMAN PRACTICES
OVERVIEW
PUBLIC OUTREACH
SOCIETY LAW & GMOs
PERSPECTIVES
ENTREPRENEURSHIP
OVERVIEW
MARKET STUDY
FUNDRAISING & COMMUNICATION
SPONSORS
PERSPECTIVES
TEAM
ATTRIBUTIONS
COLLABORATIONS
MEMBERS
Back to top
Laboratory Notebook
Note before you start reading
Instead of using the iGEM parts’ nomenclature (BBa_…) to name the sequences, in the following Laboratory Notebook section, we are using the following abbreviations:
sequence 1: BBa_K228005
sequence 2: BBa_K228006
sequence 3: BBa_K228007
sequence 4: BBa_K228008
sequence 5: BBa_K228009
sequence 6: BBa_K228010
sequence 7: BBa_K228011
sequence 8: BBa_K228012
sequence 9: BBa_K228013
Read more about our Parts .
Introduction
Here you will find the complete report of all our experiments since the 24/08. We decided to not post the first two months of our experiments because they were mostly optimisation protocols. We reached the lab at the end of June without any experience in synthetic biology and few in molecular biology, acquired from diverse internships and courses. This world was new to us so a lot of troubles awaited us. We had many troubles with PCR and ligations because of diverse reasons that we understood as the days were going on.
However, a lot of positive things are to be retained from this preliminary work. We learnt how to organise ourselves properly, how to optimise protocols according to our lab (and with the precious help of our advisors), and we learnt also to persevere.
Most of all, science is built on failure, defined and molded by it. What is most important thing for us is that although we had many difficulties, we continued no matter what. In the end we managed to make our sequence work, and that was very rewarding. We are aware some of you could learn from the experience we got at La Paillasse, therefore don’t hesitate to contact us by mail so we can share our acquired knowledge.
You can now consult our experiments, the 08/24 began with a digestion of the BBA_J04450 part in order to use its backbone pSB1C3 for the ligation of our sequences acquired by PCR.
Table of content
Week 1
08/24 - Digestion of pSB1C3 - mRFP
08/24 - PCR of sequence 2 (BBa_K2282006) with the TATA primers
08/24 - Digestion of seq 2 (BBa_K2282006) + pSB1C3 control and Ligation
08/25 - Transformation seq 2 (BBa_K2282006)
Week 2
08/28 - PCR on sequence 7 (BBa_K2282011); digestion pSB1A3-mRFP; PCR clean up; Digestion sequence 7; Ligations OverNight
08/29 - Transformation with sequence 7
08/30 - Transformation of AmilCP (BBa_K592009) from iGEM kit and sequence 2 (BBa_K2282006)
08/31 - PCR colony on sequence 7 (BBa_K2282011)
09/01 - PCR sequence 7 (BBa_K2282011) & Protegrin-1 (BBa_K628000)
Week 3
09/04 - PCR on sequences 8 (BBa_K2282012) and 9 (BBa_K2282013)
09/04 - Digestion of pSB1C3-Seq2 (BBa_K2282006)
09/05 - Gel migration of sequences 8 and 9 from 09/04 PCR; digestion; ligation
09/06 - Bacterial transformation for sequences 8 and 9
09/08 - Liquid culture of transformed colonies
Week 4
09/11 - PCR colony sequence 8 (BBa_K2282012) and 9 (BBa_K2282013)
09/11 - PCR on sequences 8 and 9
09/11 - PCR on sequence 1 (BBa_K2282005)
09/11 - Digestion pSB1C3-seq2
09/11 - Digestion of pSB1C3-mRFP
09/12 - Migration of PCR and digestions products
09/12 - PCR sequences 8 (BBa_K2282012) & 9 (BBa_K2282013)
09/13 - Digestion 1 (BBa_K2282005) & 9 (BBa_K2282013) + pSB1C3-seq2 & mRFP
09/13 - Ligations of sequences 1, 8, 9 in pSB1C3 backbone
09/14 - Transformations with sequences 1, 8, 9
Week 5
09/18 - PCR colony from 09/14 transformations
09/18 - Digestion pSB1C3-mRFP
09/20 - PCR seq 1, 7, 8, 9 + gel migration + PCR clean up + digestion + ligation
09/21 - Transformations with sequences 1, 7, and 8
09/21 - PCR sequence 9 (BBa_K2282013)
09/23 - PCR colony from 21/09 transformations + culture of pSB1C3-seq 2 transformed bacteria
Week 6
09/25 - Miniprep of pSB1C3-seq 9, pSB1C3-seq 2 and pSB1A3-mRFP
09/26 - Miniprep on sequences 9 (09/08), 8 (09/08) and 1 (09/25)
09/27 - Digestions of plasmides and gel migration
09/29 - Transformation with japanese plasmids (pHeat & pCold) from team Tokyo Tech
09/29 - Sequencing by GATC and results
Week 7
10/02 - Miniprep for sequences 1 / 2 / 7 / 8 / 9 and cultures for sequences 1 / 2 / 7 / 8 / 9
10/03 - PCR colony on japanese sequences
10/03 - OD measure for sequences 1 and 2, and protein extraction
10/04 - Protein extraction: AmilCP from sequences 1 and 2
Week 8
10/09 - Characterisation Protocole 0: Testing the kinetic of evolution of E.coli in Liquid LB at 15°C and 37°C
10/09 - Characterisation Protocole 1: Protein extraction
10/09 - Miniprep for sequences 1 / 2 / 7 / 8 / 9 and cultures for sequences 1 / 2 / 7 / 8 / 9
10/10 - Characterisation Protocol 1: Protein extraction for mRFP
10/10 - Characterisation Protocole 2: DSBox effect on AmilCP Properties - Determination of the absorption peak
10/10 - Migration PCR colony japanese plasmids
10/10 - 10/11 - Characterisation Protocole 0': Measure of the kinetic apparition of AmilCP
10/11 - PCR on sequence 3 and Protegrin-1
10/11 - Digestion protegrin-1 + Ligation
10/12 - Transformation Protegrin + produits minipreps 1, 2, 7, 8, 9 + GFP constitutive (iGEM distribution kit)
10/12 - Cold shock characterisation
10/12 - Minipreps sequence pCold and pHeat
10/13 - Liquid culture of transformed bacteria
10/13 - PCR colony on Protegrin
10/13 - Cold shock characterisation
10/14 - Cold shock characterisation
Week 9
10/16 - Miniprep of sequences 1/2/7/8/9/Protegrin/Constitutive GFP of 14/10 cultures
10/16 - Cold shock characterisation Duplicat 1 - Day 1
10/16 - Competent cells by CaCl2 - Day 1
10/17 - Migration of minipreps from 10/16
10/17 - Cold shock characterisation Duplicat 1 Day 2
10/17 - Competent cells by CaCl2 - Day 2
10/17 - Heat response characterisation - Pre culture
10/18 - Digestion recombinant plasmids for gel migration
10/18 - Cold shock characterisation Duplicat 1 - Day 3
10/18 - Competent cells by CaCl2 - Day 3 and transformation test with NEB kit
10/18 - Heat shock characterisation - Day 1
10/19 - Bacterial Transformation with pSB1C3-seq2
10/19 - Digestion and ligation of the miniprep seq 7-pSB1A3 and GFP-pSB1K3 with pSB1C3
10/19 - Heat shock characterisation - Day 2
10/20 - Bacterial transformation (seq 7, GFP, seq 8, seq 9 and M9 & N6 plate 1 2017)
10/21 - OD Measurements and kinetics with iGEM Bettencourt
10/21 - PCR colony pSB1C3-seq7 + pSB1C3-GFP + mRFP protein generator
10/21 - Miniprep and glycerol stock of seq2 (19/10 culture)
10/21 - 10/21 - Transformation of part BBa_K608351 (N6 plate 1 2017) and part BBa_K516032 (M9 plate 1 2017) - second try
Week 10
10/23 - PCR colony (second try) on pSB1C3-seq7
10/23 - GATC sequencing
10/23 - Cold shock characterisation 3 - Test at 12°C - Day 0
10/24 - Cold shock characterisation 3 - Test at 12°C - Day 1
10/25 - Cold shock characterisation 3 - Test at 12°C - Day 2
10/25 - Cold shock characterisation 4 - Test at 12°C - Day 0
10/25 - Preparation of iGEM plate for registry
10/25 - New PCR Seq2
10/26 - Cold shock characterisation 4 - Test at 12°C - Day 1
10/26 - Sequence 2: PCR clean-up, digestion and ligation in pSB1C3 backbone
10/27 - Cold shock characterisation 4 - Test at 20°C - Day 2
10/27 - Transformation 2
Week 11
10/31 - Cold-shock characterization at 37°C & 27°C
Week 1
08/24 - Digestion of pSB1C3 - mRFP
Objective:
To digest the plasmid pSB1C3-mRFP with the two restriction enzymes EcoRI and PstI in order to separate the backbone and the insert.
Material:
Plasmid pSB1C3-mRFP (psB1C3-BBa_J04450 from the IGEM Distribution Kit)
EcoRI-HF (NEB: 20000 U/mL)
PstI (NEB: 20000 U/mL)
10X NEB buffer 2.1
Ultrapure water
1,5mL Eppendorf tubes
Micropipettes + tips
Waterbath
Heat Block
Protocol:
The pSB1C3-mRFP plasmid was digested according to the following instructions:
Reagents
Quantity
Final concentration
EcoRI-HF
1µL
40U/µL
PstI
1µL
40U/µL
pSB1C3-mRFP
10µL
10ng/µL
10X NEB buffer 2.1
5µL
1X
Ultrapure water
33µL
Digestion was carried out 2 hours at 37°C
Heat inactivation was proceeded at 80°C for 20 minutes.
For the migration (cf 08/24 - PCR of sequence 2 with the TATA primers), we used 2/4/8µl of the digestion product to quantify the Backbone concentration efficiently.
08/24 - PCR of sequence 2 (BBa_K2282006) with the TATA primers
Objective:
We received our TATA primers from IDT. Primers TATA Suffix and TATA prefix were first resuspended in 250µl of TE buffer each.
Both tubes were vortexed, heated at 50°C for 20 minutes and vortexed again.
A white precipitate was still observable. According to IDT, it could be silica or other by-products. Still we need to be careful.
Part 1: PCR
Material:
Forward primer (100µM) synthesized by IDT
Reverse primer (100µM) synthesized by IDT
DNA template (BBa_K2282006, synthesized by IDT)
Q5® High-Fidelity 2X Master Mix (NEB)
DNase/RNase free water
PCR tubes
Micropipettes + tips
Thermocycler
Protocol:
Pre-dilutions préparation:
Primers 100µM: 1 µL primer + 9 dH2O --> 10µM final
Tubes were prepared according to the following indications:
Tube
Annealing T°C
Primer Prefix (10µM)
Primer Suffix (10µM)
DNA template
Q5 Mix
dH2O DNase/RNase free
Total Vol. (µL)
C+
60°C
2,5µL
2,5µL
1µL pSB1C3-mRFP (miniprep)
12.5 µL
6,5µL
25
2
60°C
5µL
5µL
2 µL seq 2 (IDT)
25 µL
13µL
30
The tubes were put in the thermocycler with the following program:
PCR Cycle
Repetition
Step
Time
Temperature
0
Initial denaturation
30 sec
98°C
30
Denaturation
10 sec
98°C
Annealing
30 sec
60°C
DNA polymerization
30 sec (25-30sec/kb)
72°C
0
DNA polymerization
2 min
72°C
END
∞
4°C
For the positive control plasmid, DNA polymerisation was set at 72°C for 1min 30 sec because of its length (3 kb vs. 1 kb for seq2). PCR tubes were placed in different thermocyclers. → This was a mistake. We should have let 30sec for the positive control also, as the amplified fragment (mRFP) is of 1kb.
Part 2: Gel migration of seq 2 (BBa_K2282006) PCR + pSB1C3-mRFP digested
Material:
Agarose
Distilled water
Gel cast and combs
Electrophoresis cuve
Microwave
Micropipettes and tips
2-Log DNA Ladder (0.1-10.0 kb) (NEB)
Gel loading dye purple 6X (NEB)
Protocol:
Mixes preparation:
Sample
Product (µL)
6X Loading Dye (µL)
Total (µL)
seq 2 (from PCR)
7
1.4
8.4
C+ (positive control) (from PCR)
15
3
18
Digested pSB1C3-mRFP
15
3
18
Plan of deposit:
Ladder (5µL)
C+ - 15µL DNA (18µL of the mix)
seq 2 - 2µL DNA (2,4µL of the mix)
seq 2 - 4µL DNA (4,8µL of the mix)
pSB1C3-mRFP - 2µL (2,4µL of the mix)
pSB1C3-mRFP - 4µL (4,8µL of the mix)
pSB1C3-mRFP - 8µL (9,6µL of the mix)
Migration was set at 110V for 1 hour.
The gel was put inside a Gel Red bath for 1h30 and revealed under UV light (new bath: 50µL  GelRed into 166mL dH2O).
Results:
Interpretation:
Bands were visible and their respective thickness were as expected. Except for the positive control, which did not work (weak smear).
Concentration estimations:
Seq 2 (BBa_K2282006): 50ng in 4µL
→ 475ng in 38µL
PCR cleanup* (-25%) → 356,25ng in 30µL
→ 11,875 ng/µL
pSB1C3 backbone: 70ng in 8µL
→ 306ng in 35µL
→ 8,75 ng/µL
*PCR cleanup was performed using the Qiagen kit from 2016.
Twice 15µL of DNase-free water were used for the final elution (30µL total).
08/24 - Digestion of seq 2 (BBa_K2282006) + pSB1C3 control and Ligation
Objective:
To digest the sequence 2 for subsequent ligation into pSB1C3-mRFP backbone, and make a ligation control by digesting pSB1C3-mRFP with EcoR1-HF endonuclease.
Part 1: Digestion
Material:
Plasmid pSB1C3-mRFP (BBa_J04450 from IGEM Kit)
PstI (NEB: 20000U/mL)
10X NEB Buffer 2.1
Ultrapure water
1,5mL Eppendorf tubes
Micropipettes + tips
Waterbath
Heat Block
Protocol:
We added the components of the mixes in the following order:
Reagents
Quantity
EcoRI-HF
1µL
pSB1C3-mRFP (from miniprep)
10µL
10X NEB buffer 2.1
5µL
Ultrapure water
34µL
Total volume
50µL
Reagents
Quantity
Purified seq 2 (11,875 ng/µL)
30µL
EcoRI-HF
1µL
PstI
1µL
10X NEB buffer 2.1
5µL (1X)
Ultrapure water
13µL
Total volume
50µL
Final seq 2 (BBa_K2282006) concentration: 7,125 ng/µL
We incubated for 2 hours at 37°C
Heat inactivation at 80°C for 20min
Part 2: Ligation
Material:
Digested insert (BBa_K2282006)
Digested backbone (pSB1C3 from IGEM kit)
10X T4 DNA Ligase Buffer (NEB)
T4 DNA ligase (400000U/mL)
Ultrapure water
Micropipettes and tips
Eppendorf tubes
Protocol:
We added the components of the mix in the following order:
Sample
01:01
01:03
01:05
1:3 *2
Ligase control 1
Ligase control 2
insert (7,125 ng/µL)
1,6µL
4,9µL
8,2µL
9,8µL
-
-
backbone pSB1C3 (8,75 ng/µL)
2,9µL
2,9µL
2,9µL
5,8µL
-
-
backbone control
-
-
-
-
5µL
5µL
10X T4 DNA Ligase Buffer
2µL
2µL
2µL
2µL
2µL
2µL
T4 DNA Ligase
2 µL
2 µL
2 µL
2 µL
2 µL
-
dH2O
11,5µL
8,2µL
4,9µL
0,4µL
11µL
13µL
Total volume
20µL
20µL
20µL
20µL
20µL
20µL
Incubation overnight at Room Temperature.
08/25 - Transformation seq 2 (BBa_K2282006)
Objective: To transform competent cells with the sequence 2 ligated into pSB1C3 backbone in order to amplify the recombinant plasmid.
Material:
Ligated plasmid (pSB1C3 + BBa_K2282006)
Competent cells (DH5-alpha)
LB medium liquid and solid
SOC medium
Chloramphenicol
Ice
Eppendorf tubes
Waterbath
Micropipettes and tips
Petri diches
Glass beads or inoculating loop
Incubator
Centrifuge
Protocol:
Clean your working area by wiping down with 70% ethanol.
Thaw competent cells on ice. Label one 1.5 mL microcentrifuge tubes for each transformation and then pre-chill by placing the tubes on ice.
Tubes
DNA
Transformation
Plate
Competent cells
(1) 1:1
20µL
Yes
LB + Chloramphenicol
50µL
(2) 1:3
20µL
Yes
LB + Chloramphenicol
50µL
(3) 1:5
20µL
Yes
LB + Chloramphenicol
50µL
(4) 1:3 * 2
20µL
Yes
LB + Chloramphenicol
50µL
(5) Control Ligase
20µL
Yes
LB + Chloramphenicol
50µL
(6) Control -
20µL
Yes
LB + Chloramphenicol
50µL
(7) Control Transformation (miniprep)
10µL
Yes
LB + Chloramphenicol
50µL
(8) LB medium
-
No
LB
50µL
Pipet 50 µL of competent cells into each tube. Flick the tube gently with your finger to mix. Incubate on ice for 30 minutes. Put it in the fridge, the temperatures in the room are too high. Pre-heat waterbath now to 42°C. Otherwise, hot water and an accurate thermometer works, too!
Heat-shock the cells by placing into the waterbath for 30 seconds (no longer than 1 min). Be careful to keep the lids of the tubes above the water level, and keep the ice close by.
Immediately transfer the tubes back to ice, and incubate on ice for 5 minutes. IN THE FRIDGE AS WELL
Add qs 950 µL of SOC media WITHOUT CHLORAMPHENICOL per tube, and incubate at 37°C for 1 hour shaking at 200-300rpm.
Prepare the agar plates during this time: label them, and add sterile glass beads if using beads to spread the mixture. Put the agar plates at 37°C.
Tubes were centrifuged at 3500 rpm for 4minutes and threw away most of the culture media. Bacteria were resuspended into the remaining culture media and the total volume was spread in the corresponding petri dishes.
Incubate at 37°C overnight or approximately 16 hours (we let 2 days). Position the plates with the agar side at the top, and the lid at the bottom.
Results:
Interpretation:
Results were as expected for the controls as for the ligation tests.
The different backbone/insert ratios for the ligation suggest that a ratio of 1:3 or 1:5 works better than 1:1 or 1:3 with double quantities.
Some backbones have ligated themselves spontaneously without ligase, but fewer colonies are observable for this condition showing the efficiency of ligase.
The presence of red colonies in the test conditions was predictable as we did not purify the backbone digestion products before performing the ligation. They correspond to the non-digested pSB1C3-mRFP backbones transformed into the bacteria, giving a red coloration. We will have to pick the isolated blue colonies in order to purify the pSB1C3-seq2 plasmid through culture and miniprep.
Week 2
08/28 - PCR on sequence 7 (BBa_K2282011); digestion pSB1A3-mRFP; PCR clean up; Digestion sequence 7; Ligations OverNight
Objective: To amplify sequence 7 in order to ligate it in pSB1A3 backbone, and then to transform DH5α.
Part 1: PCR
Material:
Forward primer (100µM) synthesized by IDT
Reverse primer (100µM) synthesized by IDT
DNA template (sequence 7 BBa_K2282011, synthesized by IDT)
Q5® High-Fidelity 2X Master Mix (NEB)
DNase/RNase free water
PCR tubes
Micropipettes and tips
Thermocycler
Protocol:
Pre-dilutions preparation:
Primers 100µM: 1 µL primer + 9 dH2O → 10µM final
Tubes
Primer Prefix (10µM)
Primer Suffix (10µM)
DNA template
Q5 Mix
dH2O DNase/RNase free
Total Volume (µL)
7
5µL
5µL
2 µL seq 7
25 µL
13 µL
50
The tubes were put in the thermocycler with the following program:
PCR Cycle
Repetition
Step
Time
Temperature
0
Initial denaturation
30 sec
98°C
30
Denaturation
10 sec
98°C
Annealing
30 sec
60°C
DNA polymerization
30 sec (25-30sec/kb)
72°C
0
DNA polymerization
2 min
72°C
END
∞
4°C
Part 2: Digestion of pSB1A3-mRFP
Objective: The goal here is to simply digest our plasmid PSB1A3-mRFP.
Material:
Plasmid pSB1A3-mRFP (from IGEM Kit)
EcoRI-HF (NEB: 20000U/mL)
PstI (NEB: 20000U/mL)
10X NEB Buffer 2.1
Ultrapure water
1,5mL Eppendorf tubes
Micropipettes + tips
Waterbath
Heat Block
Protocol:
We prepared the mix according to the following instructions:
Reagents
Quantity
Final concentration
EcoRI-HF
1 µL
40U/µL
PstI
1 µL
40U/µL
pSB1A3-mRFP
10 µL
10ng/µL
10X NEB buffer 2.1
5 µl
1X
Ultrapure water
33 µL
Digestion was carried out 2 hours at 37°C
Heat inactivation was proceeded at 80°C for 20 minutes.
For the migration, we are going to use 2/4/8µL of the digestion product to quantify the Backbone concentration efficiently.
Part 3: Gel migration
Objective: To perform the migration of seq 7 fragment obtained from PCR and the digested pSB1A3-mRFP.
Material:
Agarose
Distilled water
Distilled water
TAE buffer 50X
Gel cast and combs
Electrophoresis cuve
Microwave
Micropipettes and tips
2-Log DNA Ladder (0.1-10.0 kb) (NEB)
Gel loading dye purple 6X (NEB)
Protocol:
We made an agarose gel. (cf our Protocol section)
We want to migrate the seq 7 (BBa_K2282011) from PCR and the pSB1A3-mRFP digested.
Mixes preparation:
Sample
Product (µL)
6X Loading dye (µL)
Total (µL)
seq 7 (from PCR)
7
1.4
8.4
Digested pSB1A3-mRFP
15
3
18
Plan of deposit:
1: Ladder (5µL)
2: seq 7 - 2µL DNA (2,4µL of the mix)
3: seq 7 - 4µL DNA (4,8µL of the mix)
4: pSB1A3-mRFP - 2µL (2,4µL of the mix) (this well has not been correctly filled up)
5: pSB1A3-mRFP - 4µL (4,8µL of the mix)
6: pSB1A3-mRFP - 8µL (9,6µL of the mix)
7: Ladder (5µL)
Migration 1h30 at 110V.
Revelation in GelRed bath during 1h30.
Results:
Interpretation:
Estimation of quantities:
Seq 7: 50ng in 4µL
→ 537.5ng in 43µL
PCR cleanup* (-25%) → 403ng in 50µL
→ 8.06 ng/µL
pSB1A3 backbone: 70ng in 8µL
→ 306ng in 35µL
→ 8,75 ng/µL
*cf Qiagen Kit protocol
Part 4: Digestion of sequence 7 (BBa_K2282011)
Objective: To digest the sequence 7 for subsequent ligation into pSB1A3-mRFP backbone, and make a ligation control by digesting pSB1A3-mRFP with PstI endonuclease.
Material:
Sequence 7 (BBa_K2282011, synthesized by IDT)
EcoRI-HF (NEB: 20000U/mL)
PstI (NEB: 20000U/mL)
10X NEB Buffer 2.1
Ultrapure water
1,5mL Eppendorf tubes
Micropipettes + tips
Waterbath
Heat Block
Protocol:
We prepare the mixes according to the following instructions:
Reagents
Quantity
Final concentration
EcoRI-HF
1µL
80U/µL
PstI
1µL
80U/µL
purified sequence 7
30µL
8.06 ng/µL
10X NEB buffer 2.1
5µL
1X
Ultrapure water
13µL
Reagents
Quantity
Final concentration
PstI
1µL
40U/µL
pSB1A3-mRFP
10µL
10 ng/µL
10X NEB buffer 2.1
5µL
1X
Ultrapure water
34µL
Digestion was carried out 2 hours at 37°C
Heat inactivation was proceeded at 80°C for 20 minutes.
Part 5: Ligation
Material:
Digested insert (seq 7 BBa_K2282011)
Digested backbone (pSB1A3)
10X T4 DNA Ligase Buffer (NEB)
T4 DNA ligase (400000U/mL)
Ultrapure water
Eppendorf tubes
Protocol:
We prepared the mixes according to the following instructions:
insert (4,836 ng/µL)
8µL
13.3µL
-
-
backbone pSB1A3 (8,75 ng/µL)
2.9µL
2.9µL
-
-
backbone control digested
-
-
5µL
5µL
10X T4 DNA Ligase Buffer
2µL
2µL
2µL
2µL
T4 DNA Ligase
1µl
1µL
1µL
-
Ultrapure water
6.1µL
0.8µL
12µL
13µL
Total volume
20µL
20µL
20µL
20µL
Ligation overnight in a room at about 16°C.
08/29 - Transformation with sequence 7
Objective: To transform competent cells with the sequence 7 ligated into pSB1A3 backbone in order to amplify the recombinant plasmid.
Material:
Ligated plasmid
Competent cells (DH5-alpha)
LB medium liquid and solid
SOC medium
Ampicillin
Ice
Eppendorf tubes
Micropipettes and tips
Waterbath
Petri dishes
Glass beads
Incubator
Centrifuge
Protocol:
Clean your working area by wiping down with 70% ethanol.
Thaw competent cells on ice. Label one 1.5 mL microcentrifuge tubes for each transformation and then pre-chill by placing the tubes on ice.
Tubes
DNA
Transformation
Plate
Competent cells
1:3
20µL
Yes
LB + Ampicillin
50µL
1:5
20µL
Yes
LB + Ampicillin
50µL
Control ligase +
20µL
Yes
LB + Ampicillin
50µL
Control ligase -
20µL
Yes
LB + Ampicillin
50µL
Control Transformation (miniprep pSB1A3)
10µL
Yes
LB + Ampicillin
50µL
LB medium
-
No
LB
50µL
Pipet DNA in each tube and then 50 µL of competent cells into each tube. Flick the tube gently with your finger to mix.
Incubate on ice for 30 minutes. Put it in the fridge, the temperatures in the room are too high.
Pre-heat waterbath now to 42°C. Otherwise, hot water and an accurate thermometer works, too!
Heat-shock the cells by placing into the waterbath for 30 seconds (no longer than 1 min). Be careful to keep the lids of the tubes above the water level, and keep the ice close by.
Immediately transfer the tubes back to ice, and incubate on ice for 5 minutes. (IN THE FRIDGE AS WELL)
Add qs 950 µL of SOC media WITHOUT CHLORAMPHENICOL per tube, and incubate at 37°C for 1 hour shaking at 200-300rpm.
Prepare the agar plates during this time: label them, and add sterile glass beads if using beads to spread the mixture. Put the agar plates at 37°C.
Tubes were centrifuged at 3500rpm for 4minutes and threw away most of the culture media.
Bacteria were resuspended into the remaining culture media and the total volume was spread in the corresponding petri dishes.
Incubate at 37°C overnight or approximately 16 hours. Position the plates with the agar side at the top, and the lid at the bottom.
Results:
Interpretation:
Controls worked. However there is not a lot of colonies in the dish. Thus we can imagine that there is no lot of bacteria transformed with the seq 7.
08/30 - Transformation of AmilCP (BBa_K592009) from iGEM kit and sequence 2 (BBa_K2282006)
Objective: To transform competent cells with the iGEM plasmid pSB1C3-AmilCP (BBa_K592009) and sequence 2 (BBa_K2282006) ligated into pSB1C3 on the 24/08 (miniprep performed on 30/08) in order to compare the expression of AmilCP at high and low temperatures.
Note: The constitutive promoters and the RBS we used were different between the two plasmids. Therefore, comparing their respective protein quantity or kinetics of appearance would be irrelevant. This assay aims at ensuring that the presence of the DS box into the AmilCP sequence does not alter its coloration.
Material:
Ligated plasmid (sequence 2-pSB1C3)
IGEM plasmid pSB1C3-amilCP (BBa_K592009)
Competent cells (DH5-alpha)
LB medium liquid and solid
SOC medium
Ampicillin
Ice
Eppendorf tubes
Micropipettes and tips
Waterbath
Petri dishes
Glass beads
Incubator
Centrifuge
Protocol:
Clean your working area by wiping down with 70% ethanol.
Thaw competent cells on ice. Label one 1.5 mL microcentrifuge tubes for each transformation and then pre-chill by placing the tubes on ice.
Tubes
DNA
Transformation
Plate
Competent cells
SOC medium
pSB1C3-amilCP (from iGEM kit)*
3µL
Yes
LB + Chloramphenicol
50µL
947µL
Seq 2 (from the 30/08 miniprep)
10µL
Yes
LB + Chloramphenicol
50µL
940µL
*The plasmid was taken from the DNA distribution kit plate given by the iGEM (plate 6 well 1N). The DNA was resuspended into 10µL of dH20 to reach a final concentration of approximately 200-300 pg/µL.
Pipet DNA in each tube and then 50 µL of competent cells into each tube. Flick the tube gently with your finger to mix.
Incubate on ice for 30 minutes. Put it in the fridge, the temperatures in the room are too high. Pre-heat waterbath now to 42°C. Otherwise, hot water and an accurate thermometer works, too!
Heat-shock the cells by placing into the waterbath for 30 seconds (no longer than 1 min). Be careful to keep the lids of the tubes above the water level, and keep the ice close by.
Immediately transfer the tubes back to ice, and incubate on ice for 5 minutes. IN THE FRIDGE AS WELL
Add qs 950 µL of SOC media WITHOUT CHLORAMPHENICOL per tube, and incubate at 37°C for 1 hour shaking at 200-300rpm.
Prepare the agar plates during this time: label them, and add sterile glass beads if using beads to spread the mixture. Put the agar plates at 37°C.
Tubes were centrifuged at 3500rpm for 4minutes and 850µL of supernatant was discarded. Bacteria were resuspended into the remaining culture media and bacteria were spread in the corresponding petri dishes as follows:
Incubate at 37°C overnight or approximately 16 hours. Position the plates with the agar side at the top, and the lid at the bottom.
Results:
Results (pictures were taken after 5 days):
Interpretation:
Blue colonies were observable with the recombinant pSB1C3-seq2 plasmid (containing amilCP + DSbox). This shows that the miniprep performed the same day has worked properly, and that this isolated plasmid can be used in further experiments.
Colonies transformed with the plasmid from iGEM distribution kit plates (containing amilCP without DS box) did not grow very well: only one colony was observable in one of the two dishes. This can be explained by the fact that only 600pg (in average) of DNA was used for the transformation, whereas the first plasmid came from a miniprep that is known to give a better yield.
However, one colony was picked up from each condition on the friday and was cultured in liquid medium over the week-end. This will serve as a comparison experiment to assess if any difference is observable in the blue coloration (absorbance at 588nm). Given the fact that the two plasmids do not carry the same promoter and RBS sites, and therefore induce a slightly differential expression level of the protein, we can only use the iGEM plasmid to compare the color characteristics.
08/31 - PCR colony on sequence 7 (BBa_K2282011)
Objective: To check if our sequence 7 (BBa_K2282011) is present in our colonies.
Material:
10X Standard taq reaction Buffer (NEB)
10µM dNTPS (NEB)
Forward primer 100µM (synthesized by IDT)
Reverse primer 100µM (synthesized by IDT)
Taq DNA polymerase (5000U/mL)
DNAse/ RNAse free water
PCR tubes
Thermocycler
Micropipettes and tips
Protocol:
Part 1: PCR colony
Mix preparation:
We have 11 tubes of 25µL to test. We prepared a pre-mix for a final volume of 300µL.
Assemble all reaction components on ice and quickly transfer to a thermocycler preheated to the denaturation temperature (95°C).
Component
Volume
Final concentration
10X Standard Taq Reaction Buffer
30 µL
1X
10mM dNTPs
6 µL
200µM (1/50)
100µM Prefix Primer
3 µL
1µM
100µM Suffix Primer
3 µL
1µM
Taq DNA Polymerase
3 µL
2,5 units/50µL PCR
Nuclease-free water
255 µL
qs 300 µL
Add the colonies from the plates:
1:5
1:5
1:5
1:5
1:3
1:3
1:3
1:3
pSB1A3-mRFP
white colony from ligation of seq 2, 1:3 (25/08)
non transformed bacteria (from antibiotic-free dish, 25/08)
At the same time we introduce a part of each colony (from 1:5 and 1:3) in 1mL liquid LB medium containing ampicillin (100µg for 1mL). We put the tubes in the incubator at 37°C, 200rpm shaking.
Notes: Gently mix the reaction. Collect all liquid to the bottom of the tube by a quick spin if necessary.
Put the PCR tubes in the thermocycler and run the following program:
PCR Cycle
Repetition
Step
Time
Temperature
0
Initial denaturation
5 min
95°C
30
Denaturation
30 sec
95°C
Annealing
1 min
60°C
DNA polymerization
1 min (25-30sec/kb)
68°C
0
DNA polymerization
5 min
68°C
END
∞
4°C
Part 2: Gel migration
Agarose
Distilled water
Gel cast and combs
Electrophoresis cuve
Microwave
Micropipettes and tips
2-Log DNA Ladder (0.1-10.0 kb) (NEB)
Gel loading dye purple 6X (NEB)
Protocol:
Make an agarose gel (cf our "Protocol" section)
Mixes preparation: 1µL 6X Loading Dye + 5µL sample from PCR
Plan of deposit:
Ladder (5µL)
1:5_1 (6µL)
1:5_2 (6µL)
1:5_3 (6µL)
1:5_4 (6µL)
1:3_5 (6µL)
1:3_6 (6µL)
1:3_7 (6µL)
1:3_8 (6µL)
pSB1A3-mRFP (6µL)
white colony (6µL)
non transformed (6µL)
Ladder (5µL)
Migration during 1h30 at 110V.
GelRed bath overnight.
Results:
There is nothing on the gel.
Interpretation:
We make several suppositions in order to explain our results;
- The concentrations is maybe too low to be visible
- Sequence 7 has maybe not been amplified during the PCR (PCR failed)
- Ligation didn't work efficiently for sequence 7
- Transformation didn't work efficiently for sequence 7
- PCR colony didn't work
Also one of the Ladder is missing, maybe it is due to a mistake during the deposit.
HOWEVER we can distinguish a little light for the pSB1A3-mRFP. It could confirm that PCR colony worked.
09/01 - PCR sequence 7 (BBa_K2282011) & Protegrin-1 (BBa_K628000)
Objective: Amplify our cold-shock sequence 7 to further ligate it into pSB1C3.
Material:
Forward primer (100µM) synthesized by IDT
Reverse primer (100µM) synthesized by IDT
DNA template (sequence 7: BBa_K2282011 from IDT and protegrin-1: BBa_K628000 from iGEM kit
Q5® High-Fidelity 2X Master Mix (NEB)
10X Standard taq reaction Buffer (NEB)
10µM dNTPS (NEB)
Taq DNA polymerase (5000U/mL)
DNase/RNase free water
PCR tubes
Micropipettes and tips
Thermocycler
Agarose
Distilled water
TAE buffer 50X
Gel cast and combs
Electrophoresis cuve
Microwave
2-Log DNA Ladder (0.1-10.0 kb) (NEB)
Gel loading dye purple 6X (NEB)
Protocol:
Tube Prot-1
Pre-dilutions preparation:
Primers 100µM: 1 µL primer + 9 µL dH2O --> 10µM final
Tubes
Primer Prefix (10µM)
Primer Suffix (10µM)
DNA template
Q5 Mix
dH2O DNase/RNase free
Total Volume
Prot-1
5 µL
5 µL
2 µL
25 µL
13 µL
50 µL
Tube Sequence 7
Assemble all reaction components on ice and quickly transfer to a thermocycler preheated to the denaturation temperature (95°C).
Pre-dilutions (1/10):
Sample
dH2O DNase/RNase free
Final concentration
Taq Polymerase
1 µL
9 µL
500 U/mL
dNTPs
1 µL
9 µL
1 mM
We then prepare the mix according to the following instructions:
Component
Volume
Final concentration
10X Standard Taq Reaction Buffer
2.5 µL
1X
1mM dNTPs
5 µL
200µM (1/50)
10µM Prefix Primer
2.5 µL
1µM
10µM Suffix Primer
2.5 µL
1µM
Taq DNA Polymerase 500U/mL
2.5 µL
1,25 units/25µL PCR
DNA template (10ng/µL)
1 µL
Nuclease-free water
10 µL
qs 25 µL
Notes: Gently mix the reaction. Collect all liquid to the bottom of the tube by a quick spin if necessary.
The tubes were put in the thermocycler with the following program:
Tube Prot-1
PCR Cycle
Repetition
Step
Time
Temperature
0
Initial denaturation
30 sec
98°C
30
Denaturation
10 sec
98°C
Annealing
30 sec
60°C
DNA polymerization
20 sec (25-30sec/kb)
72°C
0
DNA polymerization
2 min
72°C
END
∞
4°C
Tube Sequence 7
PCR Cycle
Repetition
Step
Time
Temperature
0
Initial denaturation
5 min
95°C
30
Denaturation
30 sec
95°C
Annealing
1 min
60°C
DNA polymerization
1 min (25-30sec/kb)
68°C
0
DNA polymerization
5 min
68°C
END
∞
4°C
Migration
Sample preparation:
for PCR colony samples of 08/31: 20µl sample + 4µl 6X Loading dye
for PCR of 09/01: 5µl sample + 1µl 6X Loading dye
Deposit plan:
Ladder (5µL)
1:5_1 (24µL)
1:5_2 (24µL)
1:5_3 (24µL)
1:5_4 (24µL)
1:3_5 (24µL)
1:3_6 (24µL)
1:3_7 (24µL)
1:3_8 (24µL)
pSB1A3-mRFP (24µL)
white colony (24µL)
non transformed (24µL)
Prot-1 (24µL)
seq 7 (24µL)
Ladder (5µL)
Gel migration was performed during 1h30 at 110V.
GelRed bath during 1h30.
Results:
Interpretation:
Protegrin-1 is maybe out of the gel (because the sequence is too small, 60bp).
Seq 7 amplified by Taq did not work (while it seems to have been efficient on some colonies and pSB1A3-mRFP).
There is nothing for not transformed colonie and the white colonie from seq 2 dish, as expected.
Our problem is to differentiate mRFP and seq 7 which have a very similar length.
Week 3
09/04 - PCR on sequences 8 (BBa_K2282012) and 9 (BBa_K2282013)
Objective: To amplify our heat-shock sequences 8 and 9 in order to further ligate them into our recombinant pSB1C3-Seq2 plasmid (containing AmilCP and the DS box).
Material:
Forward primer (100µM) synthesized by IDT
Reverse primer (100µM) synthesized by IDT
DNA template (BBa_K2282012 and BBa_K2282013)
Q5® High-Fidelity 2X Master Mix (NEB)
DNase/RNase free water
PCR tubes
Micropipettes and tips
Thermocycler
Protocol:
Assemble all reaction components on ice and quickly transfer to a thermocycler preheated to the denaturation temperature (95°C).
Pre-dilutions (1/10):
Samples
dH2O DNase/RNase free
Final concentration
Taq Polymerase
1.5 µL
13.5 µL
500 U/mL
Primers (100µM)
1.5 µL
13.5 µL
10 µM
We prepared the mix for 25µl PCR mix volume (5 tubes: 2 x seq8, 2 x seq9, 1 x control pSB1C3-mRFP) according to the following instructions:
Components
Volume
Final concentration
10X Standard Taq Reaction Buffer
2.5 µL
1X
1mM dNTPs
0.5 µL
200µM (1/50)
10µM Prefix Primer
2.5 µL
1µM
10µM Suffix Primer
2.5 µL
1µM
Taq DNA Polymerase 500U/mL
2.5 µL
1,25 units/25µL PCR
DNA template (10ng/µL)
1 µL
Nuclease-free water
13.5 µL
qs 25 µL
The PCR mixes are divided in two tubes of 25µl for each sample.
The tubes were put in the thermocycler with the following program:
Seq 8 and control pSB1C3-mRFP:
PCR Cycle
Repetition
Step
Time
Temperature
0
Initial denaturation
30 sec
95°C
18
Denaturation
30 sec
95°C
Annealing
1 min
60°C
DNA polymerization
1 min (30sec/kb)
68°C
0
DNA polymerization
5 min
68°C
END
∞
4°C
Seq 9:
PCR Cycle
Repetition
Step
Time
Temperature
0
Initial denaturation
30 sec
95°C
18
Denaturation
30 sec
95°C
Annealing
1 min
60°C
DNA polymerization
2 min (30sec/kb)
68°C
0
DNA polymerization
5 min
68°C
END
∞
4°C
We decided to perform 18 cycles instead of 30 to reduce the risk of error.
09/04 - Digestion of pSB1C3-Seq2 (BBa_K2282006)
Objective: The goal here is to simply digest our plasmid pSB1C3-Seq 2  from a miniprep of the 08/28.
Material:
Plasmid pSB1C3-seq2 (BBa_K2282006)
EcoRI-HF (NEB: 20000U/mL)
PstI (NEB: 20000U/mL)
10X NEB Buffer 2.1
Ultrapure water
1,5mL Eppendorf tubes
Micropipettes + tips
Waterbath
Heat Block
Protocol:
We prepare the mix for the digestion of pSB1C3-Seq 2 according to the following instructions:
Reagents
Quantity
Final concentration
EcoRI-HF
1 µL
40U/µL
PstI
1 µL
40U/µL
pSB1C3-Seq 2
10 µL
~10ng/µL
10X NEB buffer 2.1
5 µl
1X
Ultrapure water
33 µL
We proceeded 2h at 37°C and heat inactivated for 20 min at 80°C.
09/05 - Gel migration of sequences 8 and 9 from 09/04 PCR; digestion; ligation
Part 1: Gel Migration
Objective: To run a gel with the PCR samples (seq 8, 9, ctrl) from the PCR performed with Taq polymerase on the previous day.
Material:
Agarose
Distilled water
TAE buffer 50X
Gel cast and combs
Electrophoresis cuve
Microwave
Micropipettes and tips
2-Log DNA Ladder (0.1-10.0 kb) (NEB)
Gel loading dye purple 6X (NEB)
Protocol:
Gel preparation: please see the Protocol section
Sample preparation: We put together the two tubes of sequence 8 in one single tube (Vf = 50µL). Same for sequence 9. For the sample preparation we ue 5µL of sample + 1µL of 6X Loading Dye.
Deposit plan:
Ladder (5µL)
Seq 8 (6µL)
Seq 9 (6µL)
Ctrl (pSB1C3-mRFP) (6µL)
Gel migration was performed during 1h30 at 110V.
GelRed bath during 1h30.
Results:
Interpretation:
Results are as we expected: Seq 8 at 1000 bp, Seq 9 at 2000 bp and Control at 1000 bp.
Part 2. PCR clean-up
Objective: To clean our PCR products.
Material:
QIAquick PCR purification kit (Quiagen)
PCR products
Eppendorf tubes
Deionised water
Micropipettes and tips
Centrifuge
Protocol: (see the Protocol section)
Elution was performed with deionised water: 15µL twice, for a total volume of 30µL.
Estimations of DNA concentration thanks to the gel:
Seq8: For 5µL → 30ng so for 45µL → 270ng so after PCR clean up (25% lost) → 202,5ng (6.75 ng/µL)
Seq9: For 5µL → 45ng so for 45µL → 405ng so after PCR clean up (25% lost) → 303,75ng (10.13 ng/µL)
Part 3. Digestion
Objective: to digest our sequences in order to perform the ligation.
Material:
Amplified and purified sequences (8 = BBa_K2282012 and 9 = BBa_K2282013)
EcoRI-HF (NEB: 20000U/mL)
PstI (NEB: 20000U/mL)
10X NEB Buffer 2.1
Ultrapure water
1,5mL Eppendorf tubes
Micropipettes + tips
Waterbath
Heat Block
Protocol:
We prepared the mixes according to the following instructions:
Reagents
Quantity
Final concentration
EcoRI-HF
1 µL
40U/µL
PstI
1 µL
40U/µL
sequence (8 or 9)
30 µL
6,7ng/µL (seq8) ; 10,1ng/µL (seq9)
10X NEB buffer 2.1
5 µl
1X
Ultrapure water
13 µL
We incubated 2h at 37°C.
Heat inactivation was performed at 80°C for 20min.
Part 4. Ligation
Objective: to ligate our sequences within the backbone.
Material:
Digested insert (Sequence 8 BBa_K2282012 and sequence 9 BBa_K2282013)
Digested backbone (pSB1C3-seq2)
10X T4 DNA Ligase Buffer (NEB)
T4 DNA ligase (400000U/mL)
Ultrapure water
Eppendorf tubes
Micropipettes and tips
Protocol:
Here are the estimated quantities of DNA we have (estimation from the gel):
Seq 8: 6,7ng/µL
Seq 9: 10,1ng/µL
Backbone: 16ng/µL
We prepare the ligation mixes according to the following instructions:
seq 8 ratio 1:3
seq 8 ratio 1:5
seq 9 ratio 1:3
seq 9 ratio 1:5
insert
6,2µL
10.4µL
7.15µL
11.9µL
backbone pSB1C3-Seq 2 (16ng/µL)
1.5µL
1.5µL
1.5µL
1.5µL
10X T4 DNA Ligase Buffer
2µL
2µL
2µL
2µL
T4 DNA Ligase
1µL
1µL
1µL
1µL
dH2O
9.3µL
5.1µL
8.35µL
3.6µL
Total volume
20µL
20µL
20µL
20µL
Ligation overnight in a room at about 16°C.
09/06 - Bacterial transformation for sequences 8 and 9
Objective: To introduce our recombinant plasmids pSB1C3-seq 8 and pSB1C3-seq 9 in E. coli and amplify them.
Material:
Ligated plasmid (pSB1C3-seq 8 and pSB1C3-seq9)
Competent cells (DH5-alpha)
LB medium liquid and solid
SOC medium
Chloramphenicol
Ice
Eppendorf tubes
Micropipettes and tips
Waterbath
Petri dishes
Glass beads
Incubator
Centrifuge
Protocol:
Clean your working area by wiping down with 70% ethanol.
Thaw competent cells on ice. Label one 1.5 mL microcentrifuge tubes for each transformation and then pre-chill by placing the tubes on ice.
Pipet DNA in each tube and then 50 µL of competent cells into each tube. Flick the tube gently with your finger to mix.
Incubate on ice for 30 minutes. Put it in the fridge, the temperatures in the room are too high.
Pre-heat waterbath now to 42°C. Otherwise, hot water and an accurate thermometer works, too!
Heat-shock the cells by placing into the waterbath for 45 seconds (no longer than 1 min). Be careful to keep the lids of the tubes above the water level, and keep the ice close by
Immediately transfer the tubes back to ice, and incubate on ice for 5 minutes. IN THE FRIDGE AS WELL
Add qs 950 µL of SOC media WITHOUT CHLORAMPHENICOL per tube, and incubate at 37°C for 1 hour and 20 min shaking at 200-300rpm (250rpm).
Prepare the agar plates during this time: label them, and add sterile glass beads if using beads to spread the mixture. Put the agar plates at 37°C.
Tubes were centrifuged at 3500rpm for 4minutes and threw away most of the culture media. Bacteria were resuspended into the remaining culture media and the total volume was spread in the corresponding petri dishes.
Incubate at 30°C and 37°C overnight or approximately 16 hours. Position the plates with the agar side at the top, and the lid at the bottom.
Results:
Pictures were taken 9 days after transformation.
.
Interpretation:
We can observe that we obtained red and white colonies. The red ones might have our sequences of interest (they may contain mRFP part).
09/08 - Liquid culture of transformed colonies
Objective: The goal here was to make liquid cultures of transformed colonies of the 06/09 to test out temperatures effects on sequences 8 and 9.
Material:
LB medium with chloramphenicol
Transformed colonies (on solid LB medium)
Sterile loops
Incubator with shaker (25°C and 37°C)
Sterile falcon 50mL
Protocol:
8 falcon tubes of 15ml of LB medium with chloramphenicol were prepared:
2 pSB1C3-mRFP at 25°C and 37°C
2 pSB1C3-seq 8 at 25°C and 37°C
2 pSB1C3-seq 9 at 25°C and 37°C
1 pSB1C3-seq 1 at 37 °C
1 pSB1C3-seq 2 at 37 °C
Both incubator were set at 250 rpm, however the multitron (used for 25°C condition) is a multiplate shaker.
Results:
Nothing grew in the pSB1C3-mRFP tubes, highlighting the fact that the colonies chosen were probably dead (the petri dish was outdated).
Both seq 8 and 9 displayed red color at 37°C, which is a logical result.
None of the seq 8 and 9 displayed colors at 25°C, however the shaking might have disturbed and considerably stressed the bacteria, hence nothing can really be interpreted from the results.
Both seq 1 and 2 colonies grew but no color was observed.
Protocol (continuation):
Aliquots of 1ml were made with every tube at 25°C and the seq 1 and seq 2 tubes at 37°C, and centrifuged at 10000 rpm for 1 minute.
Results:
Blue was observed for the seq 1 and seq 2 tube.
Red was observed for the seq 9 tube at 25 °C → Failure of the system?
Red was observed for the seq 8 tube at 25°C  → Normal results
No pellet was observed for the pSB1C3-mRFP, showing clearly that no bacteria developed
This will force us to incubate in petri dishes rather than liquid temperatures. This is rather grim in a way that our system doesn't work. What is curious is that this system has been characterised in the iGEM and scientific literature, hence the problem might come from our sequences.
Week 4
09/11 - PCR colony sequence 8 (BBa_K2282012) and 9 (BBa_K2282013)
Objective: To check if we have the correct inserts in our colonies from transformations with sequence 8 and 9 on 09/06.
Material:
10X Standard taq reaction Buffer (NEB)
10µM dNTPS (NEB)
Transformed colonies (on solid LB medium)
Forward primer 100µM (synthesized by IDT)
Reverse primer 100µM (synthesized by IDT)
Taq DNA polymerase (5000U/mL)
DNAse/ RNAse free water
PCR tubes
Thermocycler
micropipettes and tips
Protocol:
Prepare the PCR mix in each PCR tube (the enzyme is added at the end) stay in ice.
We choose colonies in the dishes:
For Sequence 8: test on 3 white colonies and 3 coloured colonies
For Sequence 9: test on 3 white colonies and 3 coloured colonies
Control: colony containing pSB1C3-amilCP (blue colony)
Total = 13 PCR tubes of 25µL
Each chosen colony has been annotated on the Petri dishes so we will be able to use them again for liquid culture later. They are now stocked in the fridge, protected by parafilm.
Cell membrane destruction:
In each PCR tube: put 8µL of dH20. Pick a colony (with a toothpick or a micropipette tip), add a stab of it into the PCR tube.
Incubate the tubes at 95°C for 5min.
Prepare the PCR mix according to the following informations:
Component
Volume
Final concentration
10X Standard Taq Reaction Buffer
7µL
1X
10mM dNTPs
3,5µL
200µM (1/50)
100µM Prefix Primer
3,5µL
1µM
100µM Suffix Primer
3,5µL
1µM
Taq DNA Polymerase
3,5µL
2,5 units/50µL PCR
Nuclease-free water
172,25µL
qs 350µL
Calculation of H2O: 8 x 13 = 104µL (the quantity of water already added separately in the tubes) ; 276,25 - 104 = 172,25µL
Put 17µL of mix in each tube (25 - 8 = 17).
Cycles for the thermocycler:
Sequence 8:
PCR Cycle
Repetition
Step
Time
Temperature
0
Initial denaturation
30 sec
95°C
18
Denaturation
30 sec
95°C
Annealing
1 min
60°C
DNA polymerization
1 min (30 sec/kb)
68°C
0
DNA polymerization
5 min
68°C
FINISH
∞
4°C
Sequence 9:
PCR Cycle
Repetition
Step
Time
Temperature
0
Initial denaturation
30 sec
95°C
18
Denaturation
30 sec
95°C
Annealing
1 min
60°C
DNA polymerization
2 min (30 sec/kb)
68°C
0
DNA polymerization
5 min
68°C
FINISH
∞
4°C
We make an agarose gel (cf our Protocols).
Deposit plan:
Ladder
Sequence 8 - Colony 1 (white) 37°C
Sequence 8 - Colony 2 (white) 37°C
Sequence 8 - Colony 3 (white) 37°C
Sequence 8 - Colony 4 (Red) 37°C
Sequence 8 - Colony 5 (Red) 37°C
Sequence 8 - Colony 6 (Red) 37°C
Sequence 9 - Colony 1 (white)
Sequence 9 - Colony 2 (white) 37°C
Sequence 9 - Colony 3 (white)
Sequence 9 - Colony 4 (Red) 30°C
Sequence 9 - Colony 5 (Red) 30°C
Sequence 9 - Colony 6 (Red) 37°C
Control (Blue)
Ladder
Gel migration was performed during 1h30 at 110V.
The gel was incubated in a GelRed bath during 1h30.
Waited results:
Sequence 8, we are waiting for a band at 1202bp.
Sequence 9: 2076bp.
Control (amilCP): 669bp.
Results:
Interpretation:
37°C - Sequence 8 (white colonies): unexpected bands at 2000pb
→ sequence 9? amilCP not expressed? mRFP (wrong backbone?)
37°C - Sequence 8 (red colonies): expected bands at 1000pb
→ should be sequence 8
Sequence 9 (white colonies): We obtained two bands with the expected size (2000 pb)
30 & 37°C - Sequence 9 (red colonies): unexpected bands at 1000pb
→ Results are not viable and we can not make interpretations. We need to perform this experiment again.
09/11 - PCR on sequences 8 and 9
Objective: To amplify sequences 8 and 9.
Material:
DNA template (seq 8 BBa_K2282012 and seq 9 BBa_K2282013)
10X Standard taq reaction Buffer (NEB)
10µM dNTPS (NEB)
Forward primer 100µM (synthesized by IDT)
Reverse primer 100µM (synthesized by IDT)
Taq DNA polymerase (5000U/mL)
DNAse/ RNAse free water
PCR tubes
Thermocycler
micropipettes and tips
Protocol:
Assemble all reaction components on ice and quickly transfer to a thermocycler preheated to the denaturation temperature.
Pre-dilutions (1/10):
Sample
dH2O DNase/RNase free
Final concentration
Taq Polymerase
1.5 µL
13.5 µL
500 U/mL
Primers (100µM)
1.5 µL
13.5 µL
10 µM
We prepared 2 tubes of 25µL, one for the sequence 8 and one for the sequence 9, according to the following instructions:
Component
Volume
Final concentration
10X Standard Taq Reaction Buffer
2,5µL
1X
1mM dNTPs
0,5µL
200µM (1/50)
10µM Prefix Primer
2,5µL
1µM
10µM Suffix Primer
2,5µL
1µM
Taq DNA Polymerase 500U/mL
,5µL
1,25 units/25µL PCR
DNA template (10ng/µL)
1µL
Nuclease-free water
13,5µL
qs 25µL
PCR program on the thermocycler:
Sequence 8 and control pSB1C3-mRFP:
PCR Cycle
Repetition
Step
Time
Temperature
0
Initial denaturation
30 sec
95°C
18
Denaturation
30 sec
95°C
Annealing
1 min
60°C
DNA polymerization
1 min (30 sec/kb)
68°C
0
DNA polymerization
5 min
68°C
FINISH
∞
4°C
Sequence 9:
PCR Cycle
Repetition
Step
Time
Temperature
0
Initial denaturation
30 sec
95°C
18
Denaturation
30 sec
95°C
Annealing
1 min
60°C
DNA polymerization
2 min (30 sec/kb)
68°C
0
DNA polymerization
5 min
68°C
FINISH
∞
4°C
09/11 - PCR on sequence 1 (BBa_K2282005)
Objective: To amplify our sequence 1 (BBa_K2282005) to further ligate it into pSB1C3-mRFP.
Material:
DNA template (seq 1 BBa_K2282005)
10X Standard taq reaction Buffer (NEB)
10µM dNTPS (NEB)
Forward primer 100µM (synthesized by IDT)
Reverse primer 100µM (synthesized by IDT)
Taq DNA polymerase (5000U/mL)
DNAse/ RNAse free water
PCR tubes
Thermocycler
micropipettes and tips
Protocol:
Sequence 1
Assemble all reaction components on ice and quickly transfer to a thermocycler preheated to the denaturation temperature (95°C).
Pre-dilutions (1/10):
Sample
dH2O DNase/RNase free
Final concentration
Taq Polymerase
1 µL
9 µL
500 U/mL
Primers (100µM)
1 µL
9 µL
10 µM
We prepared a 50µL PCR mix volume according to the following instructions:
Component
Volume
Final concentration
10X Standard Taq Reaction Buffer
5µL
1X
10mM dNTPs
1µL
200µM (1/50)
100µM Prefix Primer
5µL
1µM
100µM Suffix Primer
5µL
1µM
Taq DNA Polymerase 500U/mL
5µL
2,5 units/50µL PCR
DNA template (10ng/µL)
2µL
Nuclease-free water
27µL
qs 50µL
The PCR mix was divided in two tubes of 25µL each.
PCR program on the thermocycler:
PCR Cycle
Repetition
Step
Time
Temperature
0
Initial denaturation
30 sec
95°C
18
Denaturation
30 sec
95°C
Annealing
1 min
60°C
DNA polymerization
1 min (30 sec/kb)
68°C
0
DNA polymerization
5 min
68°C
FINISH
∞
4°C
The two tubes of sequence 1 will be mixed together
09/11 - Digestion pSB1C3-seq2
Objective: To digest a plasmid pSB1C3-Seq 2 (amilCP + DS box) taken from a miniprep of 08/30.
Material:
Plasmid pSB1C3-seq2
EcoRI-HF (NEB: 20000U/mL)
PstI (NEB: 20000U/mL)
10X NEB Buffer 2.1
Ultrapure water
1,5mL Eppendorf tubes
Micropipettes + tips
Waterbath
Heat Block
Protocol:
We prepare the mix for the digestion of pSB1C3-Seq 2 according to the following instructions:
Reagents
Quantity
Final concentration
EcoRI-HF
1µL
40U/µL
PstI
1µL
40U/µL
pSB1C3-Seq 2
10µL
~10ng/µL
10X NEB buffer 2.1
5µL
1X
Ultrapure water
33µL
We proceeded 2h at 37°C and heat inactivated for 20 min at 80°C.
For the migration, we are going to use 2/4/8µL of the digestion product to quantify the backbone concentration efficiently.
09/11 - Digestion of pSB1C3-mRFP
Objective: To digest our plasmid pSB1C3-mRFP from a miniprep of the 08/14.
Material:
Plasmid pSB1C3-mRFP (BBa_J04450 from IGEM Kit)
EcoRI-HF (NEB: 20000U/mL)
PstI (NEB: 20000U/mL)
10X NEB Buffer 2.1
Ultrapure water
1,5mL Eppendorf tubes
Micropipettes + tips
Waterbath
Heat Block
Protocol:
We prepared the mixes according to the following instructions:
Reagents
Quantity
Final concentration
EcoRI-HF
1µL
40U/µL
PstI
1µL
40U/µL
pSB1C3-mRFP
10µL
10ng/µL
10X NEB buffer 2.1
5µL
1X
Ultrapure water
33µL
We incubated 2h at 37°C.
Heat inactivation was performed at 80°C for 20 min.
For the migration, we are going to use 2, 4, 8µL of the digestion product to quantify the backbone concentration efficiently.
09/12 - Migration of PCR and digestions products
Objective: To perform to gel migration of the products of PCR for sequences 1 (BBa_K2282005), 8 (BBa_K2282012) and 9 (BBa_K2282013), and the products of digestions of pSB1C3-mRFP and pSB1C3-seq2.
Material:
Agarose
Distilled water
TAE buffer 50X
Gel cast and combs
Electrophoresis cuve
Microwave
Micropipettes and tips
2-Log DNA Ladder (0.1-10.0 kb) (NEB)
Gel loading dye purple 6X (NEB)
Protocol:
We made an agarose gel (cf our Protocols section)
We prepared the mixes:
For sequences 1, 8, 9: 7µL PCR product + 1,4µL 6X loading dye → 8,4µL total
For digestion products: 15µL digestion product + 3µL 6X loading dye → 18 µL total
Plan of deposit:
Ladder (5µL)
seq 1 - 2µL DNA (2,4µL of the mix)
seq 1 - 4µL DNA (4,8µL of the mix)
pSB1C3-mRFP - 2µL DNA (2,4µL of the mix)
pSB1C3-mRFP - 4µL DNA (4,8µL of the mix)
pSB1C3-mRFP - 8µL DNA (9,6µL of the mix)
seq 8 - 2µL DNA (2,4µL of the mix)
seq 8 - 4µL DNA (4,8µL of the mix)
seq 9 - 2µL DNA (2,4µL of the mix)
seq 9 - 4µL DNA (4,8µL of the mix)
pSB1C3-seq2 - 2µL DNA (2,4µL of the mix)
pSB1C3-seq2 - 4µL DNA (4,8µL of the mix)
pSB1C3-seq2 - 8µL DNA (9,6µL of the mix)
Ladder (5µL)
Gel migration was performed during 1h30 at 110V.
The gel was incubated in a GelRed bath during 1h30.
Results:
Interpretation:
We observe the waited bands for sequences 1 and 9, and also for the digested plasmids.
However we do not see anything for sequence 8.
We estimated DNA quantity from the gel observation:
Sample
Quantity in the gel
Quantity in the tube
Quantity after PCR clean up (-25%)
Sequence 1
22,5ng in 4µL
242,09ng in 43µL
181,57ng in 50µL → 3,63 ng/µL
Sequence 9
61ng in 4µL
274,5ng in 18µL
205,88ng in 50µL → 4,12 ng/µL
pSB1C3-mRFP digested
28,5ng in 8µL
3,56 ng/µL
pSB1C3-seq 2 digested
50ng in 8µL
6,25 ng/µL
09/12 - PCR sequences 8 (BBa_K2282012) & 9 (BBa_K2282013)
Part 1: PCR
Objective: To amplify sequences 8 and 9.
Material:
DNA template (seq 8 BBa_K2282012 and seq 9 BBa_K2282013)
10X Standard taq reaction Buffer (NEB)
10µM dNTPS (NEB)
Forward primer 100µM (synthesized by IDT)
Reverse primer 100µM (synthesized by IDT)
Taq DNA polymerase (5000U/mL)
DNAse/ RNAse free water
PCR tubes
Thermocycler
micropipettes and tips
Protocol:
Assemble all reaction components on ice and quickly transfer to a thermocycler preheated to the denaturation temperature (95°C).
Pre-dilutions (1/10):
Sample
dH2O DNase/RNase free
Final concentration
Taq Polymerase
1 µL
9 µL
500 U/mL
Primers (100µM)
1 µL
9 µL
10 µM
For a 50µL PCR mix volume:
Component
Volume
Final concentration
10X Standard Taq Reaction Buffer
5µL
1X
1mM dNTPs
1µL
200µM (1/50)
10µM Prefix Primer
5µL
1µM
10µM Suffix Primer
5µL
1µM
Taq DNA Polymerase 500U/mL
5µL
2,5 units/50µL PCR
DNA template (10ng/µL)
2µL
Nuclease-free water
27µL
qs 50µL
PCR mix is divided in two tubes of 25µL each.
PCR thermocycler program:
PCR Cycle
Repetition
Step
Time
Temperature
0
Initial denaturation
30 sec
95°C
18
Denaturation
30 sec
95°C
Annealing
1 min
60°C
DNA polymerization
seq 8: 1  min (30 sec/kb)
seq 9: 2 min
68°C
0
DNA polymerization
5 min
68°C
FINISH
∞
4°C
The two 25µL tubes of sequence 8 and 9 will be mixed together.
Part 2: Gel Migration
Protocol:
Please refer to our Protocols for the agarose gel preparation.
We prepared the mixes as follow:
7µL PCR product + 1,4µL 6X loading dye → 8,4µL total
Deposit plan:
Ladder - 5µl
seq 8 - 2µL DNA (2,4µL of the mix)
seq 8 - 4µL DNA (4,8µL of the mix)
seq 9 - 2µL DNA (2,4µL of the mix)
seq 9 - 4µL DNA (4,8µL of the mix)
Gel migration was performed during 1h30 at 110V.
The gel was incubated in a GelRed bath during 1h30.
Results:
There was nothing on gel, except the Ladder.
Interpretation:
We will need to get back to the PCR of the sequences 9 and 1 made the day before to clean them up and pursue ligations.
09/13 - Digestion 1 (BBa_K2282005) & 9 (BBa_K2282013) + pSB1C3-seq2 & mRFP
Objective: Sequences 1 and 9 were purified by following the PCR clean-up Protocol. We now want to digest them in order to insert them into pSB1C3 backbone.
Material:
Plasmid pSB1C3-mRFP, pSB1C3-seq2 (BBa_K2282006) and seq 1 (BBa_K2282005) and 9 (BBa_K2282013)
EcoRI-HF (NEB: 20000U/mL)
PstI (NEB: 20000U/mL)
10X NEB Buffer 2.1
Ultrapure water
1,5mL Eppendorf tubes
Micropipettes + tips
Waterbath
Heat Block
Protocol:
We prepared the mixes for sequences 1 and 9 according to the following instructions:
Reagents
Quantity
Final concentration
EcoRI-HF
1µL
40U/µL
PstI
1µL
40U/µL
Purified sequence
30µL
10ng/µL
10X NEB buffer 2.1
5µL
1X
Ultrapure water
13µL
We prepared the mix for pSB1C3-mRFP according to the following instructions:
NOTE: SpeI WAS MISTAKEN FOR PstI, HENCE THE DIGESTION WAS STILL CARRIED ON BUT ANOTHER DIGESTION OF THE PSB1C3-SEQ2 BACKBONE WAS NEEDED, WITH ECORI AND SPEI THIS TIME.
Reagents
Quantity
Final concentration
EcoRI-HF
1µL
40U/µL
SpeI
1µL
40U/µL
pSB1C3-seq2
10µL
10ng/µL
10X NEB buffer 2.1
5µL
1X
Ultrapure water
33µL
We also performed a control for the ligation:
Reagents
Quantity
Final concentration
EcoRI-HF
1µL
40U/µL
pSB1C3-mRFP
10µL
10ng/µL
10X NEB buffer 2.1
5µL
1X
Ultrapure water
34µL
Digestion was carried out 2 hours at 37°C
Heat inactivation was proceeded at 80°C for 20 minutes.
09/13 - Ligations of sequences 1, 8, 9 in pSB1C3 backbone
Objective: To ligate the sequences in the pSB1C3 backbone.
Material:
Digested insert (Sequences 1, 8 and 9)
Digested backbone (pSB1C3)
10X T4 DNA Ligase Buffer (NEB)
T4 DNA ligase (400000U/mL)
Ultrapure water
Eppendorf tubes
Micropipettes and tips
Protocol:
Here is the concentration for each DNA component we have:
Seq 1: 3,63 ng/µL
Seq 9: 4,12 ng/µL
Backbone from pSB1C3-mRFP: 3,56 ng/µL
Backbone from pSB1C3-seq2: 6,25 ng/µL
Seq 8 (from 09/05): 6,7ng/µL
Seq 9 (from 09/05): 10,1ng/µL
We performed ligations with a 1:5 ratio.
seq 1
seq 9
seq 8 (09/05)
seq 9 (09/05)
Control ligase +
Control ligase -
insert
11,85 µL
15,3 µL
10,8µL
12,4µL
-
-
backbone
pSB1C3-mRFP 5,25µL
pSB1C3-seq2 2µL
pSB1C3-seq2 4µL
pSB1C3-seq2 4µL
pSB1C3-mRFP (EcoRI) 5µL
pSB1C3-mRFP (EcoRI) 5µL
10X T4 DNA Ligase Buffer
2µL
2µL
2µL
2µL
2µL
2µL
T4 DNA Ligase
1µL
1µL
1µL
1µL
1µL
0
dH2O
0
0
2,2µL
0
12µL
13µL
Total volume
20µL
20µL
20µL
20µL
20µL
20µL
We performed the ligations Overnight in a 16°C room.
09/14 - Transformations with sequences 1, 8, 9
Objective: To insert the ligated sequences into E.coli.
Material:
Ligated plasmids
Competent cells (DH5-alpha)
LB medium liquid and solid
SOC medium
Chloramphenicol
Ice
Eppendorf tubes
Micropipettes and tips
Waterbath
Petri dishes
Glass beads
Incubator
Centrifuge
Protocol:
Clean your working area by wiping down with 70% ethanol.
Thaw competent cells on ice. Label one 1.5 mL microcentrifuge tubes for each transformation and then pre-chill by placing the tubes on ice.
Tubes
DNA
Plate
Competent cells
SOC medium
1
20µL
LB + Chloramphenicol
50µL
930µL
8 old
20µL
LB + Chloramphenicol
50µL
930µL
9 old
20µL
LB + Chloramphenicol
50µL
930µL
9
20µL
LB + Chloramphenicol
50µL
930µL
L +
20µL
LB + Chloramphenicol
50µL
930µL
L -
20µL
LB + Chloramphenicol
50µL
930µL
control mRFP
0µL
LB + Chloramphenicol
50µL
90µL
control seq2
0µL
LB + Chloramphenicol
50µL
940µL
tube 1: Sequence 1 in pSB1C3-mRFP tube 8 (09/05): Sequence 8 digested (from 09/05) in pSB1C3-seq2 (double digested) (25°C & 37°C) tube 9 (09/05): Sequence 9 digested (from 09/05) in pSB1C3-seq2 (double digested) (25°C & 37°C) tube 9: Sequence 9 digested (09/13) in pSB1C3-seq2 (double digested) (25°C & 37°C) tube L+: pSB1C3-mRFP (digested with EcoRI) + ligase tube L-: pSB1C3-mRFP (digested with EcoRI) tube control seq2: pSB1C3-seq 2 not digested tube control mRFP: pSB1C3-mRFP not digested (25°C & 37°C)
Pipet DNA in each tube and then 50 µL of competent cells into each tube. Flick the tube gently with your finger to mix.
Incubate on ice for 30 minutes. Put it in the fridge, the temperatures in the room are too high.
Pre-heat water bath now to 42°C. Otherwise, hot water and an accurate thermometer works, too!
Heat-shock the cells by placing into the water bath for 30 seconds (no longer than 1 min). Be careful to keep the lids of the tubes above the water level, and keep the ice close by.
Immediately transfer the tubes back to ice, and incubate on ice for 5 minutes. IN THE FRIDGE AS WELL
Add qs 950 µL of SOC media WITHOUT CHLORAMPHENICOL per tube, and incubate at 37°C for 1 hour shaking at 200-300rpm.
Prepare the agar plates during this time: label them, and add sterile glass beads if using beads to spread the mixture. Put the agar plates at 37°C.
Tubes were centrifuged at 3500rpm for 4minutes and 850µL of supernatant was discarded. Bacteria were resuspended into the remaining culture media and bacteria were spread in the corresponding petri dishes.
Results:
A - Controls
B - Transformations with our sequences
NOTE: There is a mistake on the plates, we wrote 30°C instead of 25°C.
Interpretation:
Our controls worked, even if there is maybe too much colonies for the dish without ligase (B-4). However, we obtained only one red colony for sequence 9 at 37°C, and we suppose that blue colonies are the pSB1C3-seq2 plasmid (B-2). We obtained very few colonies at 25°C, all of them are white (B-3 and B-4).
For sequence 8 (from 09/05), we had also only few white colonies at 25°C (B-5). We had also only 2 red colonies the plate (B-6) at 37°C which are supposed to correspond to the pSB1C3-seq 8 plasmid.
For sequence 1, we observed 2 red colonies (B-7), which are supposed to be the pSB1C3-mRFP plasmid.
We will perform PCR colony in order to confirm the presence of the plasmids.
Week 5
09/18 - PCR colony from 09/14 transformations
Objective: To verify if the desired inserts are found in the colonies we obtained on the 09/14 transformations.
Part 1: PCR colony
Material:
10X Standard taq reaction Buffer (NEB)
10µM dNTPS (NEB)
Forward primer 100µM (synthesized by IDT)
Reverse primer 100µM (synthesized by IDT)
Taq DNA polymerase (5000U/mL)
DNAse/ RNAse free water
PCR tubes
Thermocycler
micropipettes and tips
Protocol:
Prepare the PCR mix in each PCR tube (the enzyme is added at the end). Manipulate in ice.
We take colonies from the dishes:
Sequence 8, 25°C: 3 white colonies (called 8-1 to 8-3)
Sequence 8, 37°C: 2 red colonies (called 8-4 and 8-5)
Sequence 9, 25°C, OLD: 3 white colonies (called 9-1, 9-2, 9-3)
Sequence 9, 25°C, NEW: 3 white colonies (called 9-4 to 9-6)
Sequence 9, 37, OLD: 1 red colony (called 9-7)
Control: pSB1C3-mRFP
Total = 13 colonies
Each chosen colony has been annotated on the petri dishes so we will be able to use them again for liquid culture later.     They are now stocked in the fridge, protected by parafilm.
Cell membrane destruction:
In each PCR tube: put 8µL of ddH2O. Pick a colony (with a toothpick or a micropipette tip), add a stab of it into the PCR tube.
Incubate the tubes at 95°C for 5 min.
Prepare the PCR mix:
Component
Volume
Final concentration
10X Standard Taq Reaction Buffer
35µL
1X
10mM dNTPs
7µL
200µM (1/50)
100µM Prefix Primer
3,5µL
1µM
100µM Suffix Primer
3,5µL
1µM
Taq DNA Polymerase
3,5µL
2,5 units/50µL PCR
Nuclease-free water
172,25µL
qs 350µL
Put 17µL of mix in each tube (25-8=17µL)
PCR thermocycler program:
PCR Cycle
Repetition
Step
Time
Temperature
0
Initial denaturation
30 sec
95°C
18
Denaturation
30 sec
95°C
Annealing
1 min
60°C
DNA polymerization
seq 8: 1  min (30 sec/kb)
seq 9: 2 min
68°C
0
DNA polymerization
5 min
68°C
FINISH
∞
4°C
Part 2: Gel migration
Material:
Agarose
Distilled water
TAE buffer 50X
Gel cast and combs
Electrophoresis cuve
Microwave
Micropipettes and tips
2-Log DNA Ladder (0.1-10.0 kb) (NEB)
Gel loading dye purple 6X (NEB)
Protocol:
Deposit plan:
Ladder
8-1 (Sequence 8, 25°C)
8-2 (Sequence 8, 25°C)
8-3 (Sequence 8, 25°C)
8-4 (Sequence 8, 37°C)
8-5 (Sequence 8, 37°C)
9-1 (sequence 9, 25°C, from 09/05)
9-2 (sequence 9, 25°C, from 09/05)
9-3 (sequence 9, 25°C, from 09/05)
9-4( sequence 9, 25°C)
9-5 (sequence 9, 25°C)
9-6 (sequence 9, 25°C)
9-7 (sequence 9, 37°C, from 09/05)
Control
Ladder
Gel migration was performed during 1h30 at 110V.
The gel was incubated in a GelRed bath during 1h30.
Results:
Except the Ladder, we do not see any band in the gel.
Interpretation:
We may have made a mistake while performing the PCR colony.
09/18 - Digestion pSB1C3-mRFP
Objective: The goal here is to simply digest our plasmid pSB1C3-mRFP from a miniprep of the 08/02.
Material:
Plasmid pSB1C3-mRFP (BBa_J04450 from IGEM Kit)
EcoRI-HF (NEB: 20000U/mL)
PstI (NEB: 20000U/mL)
10X NEB Buffer 2.1
Ultrapure water
1,5mL Eppendorf tubes
Micropipettes + tips
Waterbath
Heat Block
Protocol:
We prepared the mixes according to the following instructions:
Reagents
Quantity
Final concentration
EcoRI-HF
1µL
40U/µL
PstI
1µL
40U/µL
pSB1C3-mRFP
10µL
10ng/µL
10X NEB buffer 2.1
5µL
41X
Ultrapure water
33µL
Digestion was carried out 2 hours at 37°C
Heat inactivation was proceeded at 80°C for 20 minutes.
For the agarose gel migration, we are going to use 2, 4 and 8µL of the digestion product to quantify the DNA concentration efficiently.
09/20 - PCR seq 1, 7, 8, 9 + gel migration + PCR clean up + digestion + ligation
Part 1: PCR
Objective: To amplify our sequences, then digest and ligate them into our backbone in order to transform bacteria the day after.
Material:
Forward primer (100µM) synthesized by IDT
Reverse primer (100µM) synthesized by IDT
DNA template (seq 1=BBa_K2282005, seq 7=BBa_K2282011, seq 8=BBa_K2282012, seq9=BBa_K2282013)
10X Standard taq reaction Buffer (NEB)
10µM dNTPS (NEB)
Taq DNA polymerase (5000U/mL)
DNase/RNase free water
PCR tubes
Micropipettes and tips
Thermocycler
Protocol:
Instead of one 50µL reaction mix we prepared two 25µL reaction mix. The PCR is then performed with only 18 cycles to reduce     the risk of errors (as Taq polymerase is not very accurate). Finally the two PCR products are mixed together before the clean up.
Assemble all reaction components on ice and quickly transfer to a thermocycler preheated to the denaturation     temperature (95°C).
Pre-dilutions (1/10):
Sample
dH2O DNase/RNase free
Final concentration
Taq Polymerase
2,2µL
19,8µL
500U/mL
Primers (100µM)
2,2µL
19,8µL
10µM
For a 25µL PCR mix volume:
Component
Volume
Final concentration
10X Standard Taq Reaction Buffer
2,5µL
1X
1mM dNTPs
0,5µL
200µM (1/50)
10µM Prefix Primer
2,5µL
1µM
10µM Prefix Primer
2,5µL
1µM
10µM Suffix Primer
2,5µL
1µM
Taq DNA Polymerase 500U/mL
2,5µL
2,5 units/50µL PCR
DNA template (10ng/µL)
1µL
Nuclease-free water
13,5µL
qsp 25 µL
PCR thermocycler program:
PCR Cycle
Repetition
Step
Time
Temperature
0
Initial denaturation
30 sec
95°C
18
Denaturation
30 sec
95°C
Annealing
1 min
60°C
DNA polymerization
seq 1, 7, 8: 1  min (30 sec/kb)      seq 9: 2 min
68°C
0
DNA polymerization
5 min
68°C
FINISH
∞
4°C
Part 2: Gel Migration
Material:
Agarose
Distilled water
TAE buffer 50X
Gel cast and combs
Electrophoresis cuve
Microwave
Micropipettes and tips
2-Log DNA Ladder (0.1-10.0 kb) (NEB)
Gel loading dye purple 6X (NEB)
Protocol:
We are going to use 2 and 4µl of the PCR products to quantify the sequences concentrations efficiently, and 4, 8 µL of the digested backbone.
Mixes for deposit:
7µL PCR product + 1,4µL 6X loading dye → 8,4µL total
13µL of digested backbones + 2,6µL of 6X loading buffer → 15,6µL total
Deposit plan:
Ladder - 5µL
seq 1 - 2µL DNA: 2,4µl of the mix
seq 1 - 4µL DNA: 4,8µL of the mix
seq 7 - 2µL DNA: 2,4µl of the mix
seq 7 - 4µL DNA: 4,8µL of the mix
seq 8 - 2µL DNA: 2,4µl of the mix
seq 8 - 4µL DNA: 4,8µL of the mix
seq 9 - 2µL DNA: 2,4µl of the mix
seq 9 - 4µL DNA: 4,8µL of the mix
pSB1C3-mRFP digested - 4µL DNA: 4,8µL of the mix
pSB1C3-mRFP digested - 8µL DNA: 9,6µL of the mix
pSB1A3-mRFP digested - 4µL DNA: 4,8µL of the mix
pSB1A3-mRFP digested - 8µL DNA: 9,6µL of the mix
pSB1C3-seq 2 digested - 4µL DNA: 4,8µL of the mix
pSB1C3-seq 2 digested - 8µL DNA: 9,6µL of the mix
Gel migration was performed during 1h30 at 110V.
The gel was incubated in a GelRed bath during 1h30.
Results:
Interpretation:
The results are as we expected except for the sequence 9 (to do again).
Quantity estimations:
Ladder - 5µL
Sequence 1: 241,9ng in 43µL
Sequence 7: 107,5ng in 43µL
Sequence 8: 215ng in 43µL
Backbones: 693,8 in 37µL → 18,8 ng/µL final
Part 3: Digestion
Objective: We performed a PCR clean-up for the sequences 1, 7, 8. We now want to digest them prior to perform a ligation.
Material:
Plasmid pSB1C3-mRFP (BBa_J04450), Inserts: seq 1=BBa_K2282005, seq
7=BBa_K2282011, seq 8=BBa_K2282012, seq9=BBa_K2282013
EcoRI-HF (NEB: 20000U/mL)
PstI (NEB: 20000U/mL)
Ultrapure water
1,5mL Eppendorf tubes
Micropipettes + tips
Waterbath
Heat Block
Material:
We prepared the mixes according to the following instructions:
For sequences 1, 7 and 8:
Reagents
Quantity
Final concentration
EcoRI-HF
1 µL
40U/µL
PstI
1 µL
40U/µL
Purified sequence
30 µL
10 ng/µL
10X NEB buffer 2.1
5 µL
1X
Ultrapure water
13 µL
For pSB1C3-mRFP (control ligase):
Reagents
Quantity
Final concentration
EcoRI-HF
1 µL
40U/µL
backbone
10 µL
10 ng/µL
10X NEB buffer 2.1
5 µL
1X
Ultrapure water
34 µL
For pSB1A3-mRFP (miniprep from the 08/14), and pSB1C3-seq2 (miniprep from the 08/30):
Reagents
Quantity
Final concentration
EcoRI-HF
1 µL
40U/µL
PstI
1 µL
40U/µL
backbone
10 µL
10 ng/µL
10X NEB buffer 2.1
5 µL
1X
Ultrapure water
33 µL
Digestion was carried out 2 hours at 37°C
Heat inactivation was proceeded at 80°C for 20 minutes.
Final concentrations:
Sequence 1: 3,6 ng/µL
Sequence 7: 1,61 ng/µL
Sequence 8: 3,2 ng/µL
Part 4: Ligation
Material:
Digested inserts (see previous digestion)
Digested backbone (see previous digestion)
10X T4 DNA Ligase Buffer (NEB)
T4 DNA ligase (400000U/mL)
Ultrapure water
Eppendorf tubes
Micropipettes and tips
Protocol:
seq 1 1:3
seq 1 1:5
seq 7 1:3
seq 7 1:5
seq 8 1:3
seq 8 1:5
Control ligase +
Control ligase -
insert
9,6 µL
15,9 µL
17,3 µL
16 µL
13,6 µL
17,2 µL
-
-
backbone
pSB1C3-mRFP (double digested, 25ng) 1,3µL
pSB1C3-mRFP (double digested, 25ng) 1,3µL
pSB1A3-mRFP (double digested, 18ng) 1µL
pSB1A3-mRFP (double digested, 10ng) 1µL
pSB1C3-seq2 (double digested, 25ng) 1,3µL
pSB1C3-seq2 (double digested, 19ng) 1µL
pSB1C3-mRFP (EcoRI) 5µL
pSB1C3-mRFP (EcoRI) 5µL
10X T4 DNA Ligase Buffer
2µL
2µL
2µL
2µL
2µL
2µL
2µL
2µL
T4 DNA Ligase
1 µL
1 µL
1 µL
1 µL
1 µL
1 µL
1 µL
0
dH2O
6,1 µL
0
0
0
2,1 µL
0
12 µL
13 µL
Total volume
20 µL
20,2 µL
21,3 µL
19,5 µL
20 µL
21,2 µL
20 µL
20 µL
Ligation was performed overnight in a 16°C room.
09/21 - Transformations with sequences 1, 7, and 8
Objective: To transform E. coli with the sequences 1, 7, and 8.
Material:
Ligated plasmids (see previous ligation)
Competent cells (DH5-alpha)
LB medium liquid and solid
T4 DNA ligase (400000U/mL)
SOC medium
Chloramphenicol
Eppendorf tubes
Micropipettes and tips
Waterbath
Petri dishes
Glass beads
Incubator
Centrifuge
Protocol:
Clean your working area by wiping down with 70% ethanol.
Thaw competent cells on ice. Label one 1.5 mL microcentrifuge tubes for each transformation and then pre-chill by placing the tubes on ice.
Tubes
DNA
Plate
Competent cells
SOC medium
1 (1:3)
20 µL
LB + Chloramphenicol
50µL
930µL
1 (1:5)
20 µL
LB + Chloramphenicol
50µL
930µL
7 (1:3)
20 µL
LB + Ampicilline
50µL
930µL
7 (1:5)
20 µL
LB + Ampicilline
50µL
930µL
8 (1:3)
20 µL
LB + Chloramphenicol
50µL
930µL
8 (1:5)
20 µL
LB + Chloramphenicol
50µL
930µL
Ligase +
20 µL
LB + Chloramphenicol
50µL
930µL
Ligase -
20 µL
LB + Chloramphenicol
50µL
930µL
pSB1A3-mRFP
10 µL
LB + Ampicilline
50µL
990µL
pSB1C3-seq2
10 µL
LB + Chloramphenicol
50µL
990µL
Pipet DNA in each tube and then 50 µL of competent cells into each tube. Flick the tube gently with your finger to mix.
Incubate on ice for 30 minutes. Put it in the fridge, the temperatures in the room are too high.
Pre-heat waterbath now to 42°C. Otherwise, hot water and an accurate thermometer works, too!
Heat-shock the cells by placing into the waterbath for 30 seconds (no longer than 1 min). Be careful to keep the lids of the tubes above the water level, and keep the ice close by.
Immediately transfer the tubes back to ice, and incubate on ice for 5 minutes. IN THE FRIDGE AS WELL!
Add qs 950 µL of SOC media WITHOUT CHLORAMPHENICOL per tube, and incubate at 37°C for 1 hour shaking at 200-300rpm.
Prepare the agar plates during this time: label them, and add sterile glass beads if using beads to spread the mixture. Put the agar plates at 37°C.
Tubes were centrifuged at 3500rpm for 4minutes and 850µL of supernatant was discarded. Bacteria were resuspended into the remaining culture media and bacteria were spread in the corresponding petri dishes.
Results:
Interpretation:
All the transformations worked with coherent results. We can indeed see the huge bacterial concentration in miniprep     transformed bacteria (as the DNA is very concentrated) and logic results for the control (more colonies with ligase-ligated     plasmid than with naturally ligated plasmid only). We are going to use the color reporters to select the appropriate colonies     when necessary. Note that the molar ratio didn’t seem to impact the transformation results.
09/21 - PCR sequence 9 (BBa_K2282013)
Objective: As we seems to have had trouble to amplify sequence 9 before, we want to succeed this time! We will use 2 different type of DNA polymerase in order to compare their efficiency.
Material:
Forward primer (100µM) synthesized by IDT
Reverse primer (100µM) synthesized by IDT
DNA template(BBa_K2282013)
10X Standard taq reaction Buffer (NEB)
10µM dNTPS (NEB)
Taq DNA polymerase (5000U/mL)
DNase/RNase free water
PCR tubes
Micropipettes and tips
Thermocycler
Protocol:
Taq polymerase: We prepared two 25µL reaction mixes. The PCR is then performed with only 18 cycles to reduce the risk of errors (as Taq polymerase is not very accurate compared to Q5). Finally the two PCR products are mixed together before the clean up.
Q5 polymerase: We prepared one 25µL reaction mix with the remaining Q5 polymerase we had in the fridge.
Pre-dilutions (1/10):
Sample
dH2O DNase/RNase free
Final concentration
Taq Polymerase
2.2 µL
19.8 µL
500 U/mL
Primers (100µM)
2.2 µL
19.8 µL
10 µM
Assemble all reaction components on ice and quickly transfer to a thermocycler preheated to the denaturation temperature (95°C).
Composition of the PCR mix (25µL) for Taq Polymerase:
Components
Volume
Concentration
10X Standard Taq Reaction Buffer
2.5 µL
1X
10mM dNTPs
0.5 µL
200µM (1/50)
10µM Prefix Primer
2.5 µL
1µM
10µM Suffix Primer
2.5 µL
1µM
Taq DNA Polymerase 500U/mL
2.5 µL
2,5 units/50µL PCR
DNA template (10ng/µL)
2 µL
Nuclease-free water
12,5 µL
qsp 25µL
Composition of the PCR mix (25µL) for Q5 polymerase:
Components
Volume
Concentration
10µM Prefix Primer
2.5 µL
1µM
10µM Suffix Primer
2.5 µL
1µM
Q5 polymerase 2X
12.5 µL
1X
DNA template (10ng/µL)
2 µL
Nuclease-free water
5,5 µL
qsp 25µL
PCR thermocycler programs:
-> For Taq polymerase reaction:
PCR Cycle
Repetition
Step
Time
Temperature
0
Initial denaturation
30 sec
95°C
18
Denaturation
30 sec
95°C
Annealing
1 min
60°C
DNA polymerization
2  min (30 sec/kb)
68°C
0
DNA polymerization
5 min
68°C
FINISH
∞
4°C
-> For Q5 polymerase reaction:
PCR Cycle
Repetition
Step
Time
Temperature
0
Initial denaturation
30 sec
98°C
30
Denaturation
10 sec
98°C
Annealing
30 sec
60°C
DNA polymerization
1  min
72°C
0
DNA polymerization
2 min
72°C
FINISH
∞
4°C
09/23 - PCR colony from 21/09 transformations + culture of pSB1C3-seq 2 transformed bacteria
Part 1: PCR colony
Objective: To check if we have the expected insert in our colonies
Material:
10X Standard taq reaction Buffer (NEB)
10µM dNTPS (NEB)
Forward primer 100µM (synthesized by IDT)
Reverse primer 100µM (synthesized by IDT)
Taq DNA polymerase (5000U/mL)
DNase/RNase free water
PCR tubes
Micropipettes and tips
Thermocycler
Protocol:
Prepare the PCR mix in each PCR tube (the enzyme is added at the end) stay in ice.
From:
Sequence 1: 3 blue colonies (named 1-1 to 1-3)
Sequence 7: 3 red colonies (7-1 to 7-3) + 3 white colonies (7-4 to 7-6)
Sequence 8: 2 blue colonies (8-1, 8-2), 2 white colonies (8-3, 8-4)
Total = 13
Each chosen colony has been annotated on the Petri dishes so we will be able to use them again for liquid culture later. They are now stocked in the fridge, protected by parafilm.
Cell membrane destruction:
In each PCR tube: put 8µL of dH2O. Pick a colony (with a toothpick or a micropipette tip), add a stab of it into the PCR tube.
Incubate the tubes at 95°C for 5 min.
Prepare the PCR mix:
Components
Volume
Final concentration
10X Standard Taq Reaction Buffer
35 µL
1X
10mM dNTPs
7 µL
200µM (1/50)
10µM Prefix Primer
3.5 µL
1µM
10µM Suffix Primer
3.5 µL
1µM
Taq DNA Polymerase
3.5 µL
2,5 units/50µL PCR
Nuclease-free water
172,25 µL
qs 350µL
PCR thermocycler program:
PCR Cycle
Repetition
Step
Time
Temperature
0
Initial denaturation
30 sec
95°C
30
Denaturation
30 sec
95°C
Annealing
1 min
60°C
DNA polymerization
2  min (30 sec/kb)
68°C
0
DNA polymerization
5 min
68°C
FINISH
∞
4°C
Part 2: Agarose gel migration
Please refer to our Protocol to make an agarose gel in our Protocol section.
Preparation of the samples to load in the gel:
5µL of the PCR mix + 1µL of 6X Loading Dye
Deposit plan:
Ladder (5µL)
1-1 (blue)
1-2 (blue)
1-3 (blue)
7-1 (red)
7-2 (red)
7-3 (red)
7-4 (white)
7-5 (white)
7-6 (white)
8-1 (blue)*
8-2 (blue)*
8-3 (white)
8-4 (white)
Ladder (5µL)
Gel migration was performed during 1h30 at 110V.
The gel was incubated in a GelRed bath during 1h30.
Results:
Interpretation:
Sequence 1: We have the correct insert for blue colonies
Sequence 7: We have the correct insert with white colonies
*Sequence 8: We made a mistake, we took blue colonies instead of red colonies. The correct insert is supposed to be in the red colonies.
Part 3: Culture (on 09/25)
2 blue colonies with seq 1 (LB + Chloramphenicol)
2 white colonies with seq 7 (LB + Ampicillin)
2 red colonies with seq 8 (LB + Chloramphenicol) incubation at 37°C.
Also, we put a transformed colony with pSB1C3-seq 2 in culture (in LB+chloramphenicol), at 37°C.
Week 6
09/25 - Miniprep of pSB1C3-seq 9, pSB1C3-seq 2 and pSB1A3-mRFP
Objective: We want to isolate the plasmid constructions containing sequences 2 and 9.
Part 1: Miniprep
Material:
Monarch® Plasmid Miniprep Kit (NEB)
Micropipettes + tips
Eppendorf tubes
Centrifuge
Ice
Protocol:
Details of the tubes:
Plasmid
Date of transformation
Conditions
pSB1C3-Seq 9
09/21
LB medium + Chloramphenicol - Not red when taken off the incubator but became red during the manipulation
pSB1C3-Seq 2
09/23
LB medium + Chloramphenicol
pSB1A3-mRFP
09/23
LB medium + Ampicillin
The Monarch protocol and Antoine's indications were used:
All centrifugation steps should be carried out at 16,000 x g (~13,000 RPM).
If precipitate has formed in Lysis Buffer (B2), incubate at 30–37°C, inverting periodically to dissolve.
Store Plasmid Neutralization Buffer (B3) at 4°C after opening.
Note:unlike other commercial kits, all wash steps are required.
1) Pellet 1–5 ml (3 mL is fine) bacterial culture (3ml for this experiment) by centrifugation for 30 seconds (we prefered to centrifuge 1min because of the high cell concentration). Discard supernatant.
2) Resuspend pellet in 200 µl Plasmid Resuspension Buffer (B1)  (pink). Vortex or pipet to ensure cells are completely resuspended. There should be no visible clumps.
3) Lyse cells by adding 200 µl Plasmid Lysis Buffer (B2)  (blue/green). Invert tube immediately and gently 5–6 times until color changes to dark pink and the solution is clear and viscous. Do not vortex! Incubate for one minute at room temperature.
Note: Care should be taken not to handle the sample roughly and risk shearing chromosomal DNA, which will co-purify as a contaminant. Avoid incubating longer than one minute to prevent irreversible plasmid denaturation.
4) Neutralize the lysate by adding 400 µl of Plasmid Neutralization Buffer (B3)  (yellow). Gently invert tube until color is uniformly yellow and a precipitate forms. Do not vortex! Incubate for 2 minutes.
Note: Be careful not to shear chromosomal DNA by vortexing or vigorous shaking. Firmly inverting the tube promotes good mixing, important for full neutralization.
5) Clarify the lysate by spinning for 2–5 minutes at 16,000 x g.
Note: Spin time should not be less than 2 minutes. Careful handling of the tube will ensure no debris is transferred and the 2 minute recommended spin can be successfully employed to save valuable time. For culture volumes > 1 ml, we recommend a 5 minute spin to ensure efficient RNA removal by RNase A. Also, longer spin times will result in a more compact pellet that lower the risk of clogging the column.
To save time, spin for two minutes only.
6) Carefully transfer supernatant to the spin column and centrifuge for 1 minute. Discard flow-through.
To save time, spin for 30 seconds, instead of 1 minute.
If using a vacuum manifold* instead of centrifugation, insert the column into a manifold and switch the vacuum on. Allow the solution to pass through the column, then switch the vacuum source off.
7) Re-insert column in the collection tube and add 200 µl of Plasmid Wash Buffer 1. Plasmid Wash Buffer 1 removes RNA, protein and endotoxin. (Add a 5 minute incubation step before centrifugation if the DNA will be used in transfection.) Centrifuge for 1 minute. Discarding the flow-through is optional.
Note: The collection tube is designed to hold 800 µl of flow-through fluid and still allow the tip of the column to be safely above the top of the liquid. Empty the tube whenever necessary to ensure the column tip and flow-though do not make contact.
To save time, spin for 30 seconds, instead of 1 minute.
If using a vacuum manifold, add 200 µl of Plasmid Wash Buffer 1 and switch the vacuum on. Allow the solution to pass through the column, then switch the vacuum source off.
* Make sure to follow the manifold manufacturer's instructions to set-up the manifold and connect it properly to a vacuum source.
8) Add 400 µl of Plasmid Wash Buffer 2 and centrifuge for 1 minute.
When using a manifold add 400 µl of Plasmid Wash Buffer 2 and switch the vacuum on. Allow the solution to pass through the column, then switch the vacuum source off.
9) Transfer column to a clean 1.5 ml microfuge tube. Use care to ensure that the tip of the column has not come into contact with the flow-through. If there is any doubt, re-spin the column for 1 minute before inserting it into the clean microfuge tube.
If using a vacuum manifold: Since vacuum set-ups can vary, a 1 minute centrifugation is recommended prior to elution to ensure that no traces of salt and ethanol are carried over to the next step.
10) (repeat this step two times to get more plasmids, total volume=60ul) Add ≥ 30 µl DNA Elution Buffer to the center of the matrix. Wait for 1 minute, then spin for 1 minute to elute DNA.
Note: Nuclease-free water (pH 7–8.5) can also be used to elute the DNA. Delivery of the Monarch DNA Elution Buffer should be made directly to the center of the column to ensure the matrix is completely covered for maximal efficiency of elution. Additionally, yield may slightly increase if a larger volume of DNA Elution Buffer is used, but the DNA will be less concentrated as a result of dilution. For larger plasmids (≥ 10 kb), heating the DNA Elution Buffer to 50°C prior to eluting and extending the incubation time after buffer addition to 5 minutes can improve yield.
Part 2: Gel Migration
Material and Protocol:
Please refer to our Protocol section for the agarose gel fabrication.
We prepared the samples to load in the gel:
Minipreps: 5µL sample + 1µL 6X loading dye
PCR samples of 09/21: 7µL PCR product + 1,4µL 6X loading dye --> 8,4µL total
Quantity of DNA to load:
2µl: 2,4µl of the mix
4µl: 4,8µl of the mix
Deposit plan:
Ladder (5µL)
pSB1A3-mRFP (miniprep) 5µL
Seq9 (miniprep) 5µL
pSB1C3-Seq2 (miniprep) 5µL
PCR Q5 2,4µL
PCR Q5 4,8 µL
PCR Taq 2,4µL
PCR Taq 4,8µL
Gel migration was performed during 1h30 at 110V.
The gel was incubated in a GelRed bath during 1h30.
Results:
Interpretation:
Minipreps: There are problems with all the sizes we observed. They are unexpected. Maybe we should send the samples to be sequenced by GATC to be sure.
Sequence 9: There is still nothing. We definitely have a problem to amplify this particular sequence.
09/26 - Miniprep on sequences 9 (09/08), 8 (09/08) and 1 (09/25)
Objective: We want to isolate the plasmid constructions containing sequences 1, 8 and 9.
Material:
Monarch® Plasmid Miniprep Kit (NEB)
Micropipettes + tips
Eppendorf tubes
Centrifuge
Ice
Protocol:
Volume of LB medium in sequence 7 tube (09/25) was too low (5mL), we added 10mL of LB medium containing ampicillin.
Details of the tubes:
pSB1C3-seq 9 08/09 LB medium + Chloramphenicol - red when taken off the incubator )
pSB1C3-seq 8 08/09 LB medium + Chloramphenicol
pSB1C3-seq 1 25/09 LB medium + Chloramphenicol
NB: Pay attention to the elution volume. In order to prepare sample for GATC, we eluted in 60µL nuclease-free water and not 30µL. Indeed, we had approximately 120ng/5µL DNA on electrophoresis gel and GATC asks for 20µL with 30-100ng/µL, therefore our DNA is too much concentrated.
The Monarch protocol and Antoine's indications were used as usual (please refer to our “Protocols” section)
09/27 - Digestions of plasmides and gel migration
Objective:We want to digest the following plasmids with EcoRI-HF and PstI, and then verify by gel migration the efficiency of the digestion step:
pSB1A3-mRFP (23/09)
pSB1C3-seq2 (?)
pSB1C3-seq9 (21/09)
pSB1C3-seq8 (from miniprep of 08/09)
pSB1C3-seq9 (from miniprep of 08/09)
pSB1C3-seq1 (from miniprep of 25/09 culture)
pSB1C3-seq7 (from miniprep of 25/09 culture)
pSB1C3-seq8 (from miniprep of 25/09 culture)
Part 1: Digestion
Material:
Plasmids (Cf above)
EcoRI-HF (NEB: 20000U/mL)
PstI (NEB: 20000U/mL)
10X NEB Buffer 2.1
Ultrapure water
1,5mL Eppendorf tubes
Micropipettes + tips
Waterbath
Heat Block
Protocol:
We prepared the mixes according to the following instructions:
Reagents
Quantity
Final concentration
EcoRI-HF
1 µL
80U/µL
PstI
1 µL
80U/µL
Backbone
5 µL
10ng/µL
10X NEB buffer 2.1
2,5 µL
1X
Ultrapure water
15,5 µL
td>
Digestion was carried out 2 hours at 37°C
Heat inactivation was proceeded at 80°C for 20 minutes.
Part 2: Gel Migration
Material and Protocol:
We made an agarose gel according to the protocol (please refer to our "Protocols" section).
We prepared the samples for the migration: 5µL of digestion product + 1µL 6X Loading dye
Deposit plan:
pSB1A3-mRFP (23/09)
pSB1C3-seq2 (?)
pSB1C3-seq9 (21/09)
pSB1C3-seq8 (from miniprep of 08/09)
pSB1C3-seq9 (from miniprep of 08/09)
pSB1C3-seq1 (from miniprep of 25/09 culture)
pSB1C3-seq7 (from miniprep of 25/09 culture)
pSB1C3-seq8 (from miniprep of 25/09 culture)
Gel migration was performed during 1h30 at 110V.
The gel was incubated in a GelRed bath during 1h30.
Results:
Interpretation:
pSB1A3-mRFP: we are waiting for a band at 2 155 bp (backbone) + a band at 706bp (mRFP). We can't see very well the mRFP band.
pSB1C3-seq 2: we are waiting for a band at 2 070 bp (backbone) + a band at 965 bp (seq 2). It seems to be ok !
pSB1C3-seq 9: we are waiting for a band at 2 070 bp (backbone) + a band at 2076bp (seq 9). We have only a band at 1200bp for the seq 9 for both wells 3 and 5. This length represents only the half of the sequence.
pSB1C3-seq8: we are waiting for a band at 2 070 bp (backbone) + a band at 1202bp (seq 8). It seems to be ok !
pSB1C3-seq1: we are waiting for a band at 2 070 bp (backbone) + a band at 950bp (seq 1). It seems to be ok !
pSB1C3-seq7: we are waiting for a band at 2 070 bp (backbone) + a band at 1110 bp (seq 7). It seems to be ok !
09/29 - Transformation with japanese plasmids (pHeat & pCold) from team Tokyo Tech
Objective: We received the plasmids from the team Tokyo Tech (pCOLD: BBa_K1949000). We need to transform E.coli with them.
Material:
Ligated plasmid
Competent cells (DH5-alpha)
LB medium liquid and solid
SOC medium
Chloramphenicol
Ice
Eppendorf tubes
Micropipettes and tips
Waterbath
Petri dishes
Glass beads
Incubator
Centrifuge
Protocol:
The same protocol as usual was used. Please refers to our “Protocols” section.
Tubes
DNA
Plate
ompetent cells
SOC medium
pCold
10 µL
LB + Chloramphenicol
50µL
940µL
pHeat
10 µL
LB + Chloramphenicol
50µL
940µL
pSB1C3-mRFP
10 µL
LB + Chloramphenicol
50µL
940µL
Results:
Results are observed after 3 days at 37°C.
Interpretation:
Transformation worked for all the sequences, we tried to streak to obtain isolated colonies and it worked. pCOLD seems to be fluorescent but this is very hard to tell since the plate itself produces a blue light. pCOLD should not be fluorescent at 37°C anyway so we are going to assess if it is the case in further characterisations.
09/29 - Sequencing by GATC and results
We sent some of our sequences and our primers to GATC in order to obtain their sequence.
miniprep pSB1C3-seq 1
pSB1C3-seq2
pSB1C3-seq7
miniprep pSB1C3-seq8 (from culture 08/09)
miniprep pSB1C3-seq8 (from culture 25/09)
miniprep pSB1C3-seq9 (from culture 08/09)
pSB1C3-seq9 (from 21/09)
Primer preparation: 2µL primer + 18µL dH2O
Sample preparation: 10µL sample + 10µL dH2O (concentration is estimated between 30 and 50ng/µL)
Results:
1. a) Sequencing of miniprep pSB1C3- seq 1 with primer prefix:
The green part means that we have 100% of identity. For this sequence, we have:
VF2: not covered
Prefix: not covered
the promoter isn't completely covered but it is normal because it is the beginning.
RBS: covered
AmilCP: covered
double terminator: covered
suffix: quite covered (normal: it is the end)
VR: quite covered (normal: it is the end)
This sequencing is good, we have the correct sequence.
1. b)  Sequencing of miniprep pSB1C3- seq 1 with primer suffix:
VF2: not covered (normal, it is the end)
Prefix: not covered (normal, it is the end)
promoter: not covered
RBS: not covered
AmilCP: not completely covered
double terminator: quite covered but some mispairing
suffix: quite covered but some mispairing
VR: quite covered but some mispairing
This sequencing is not a very good quality. But we have good results on the previous one so it is OK.
2. a) sequencing of pSB1C3-seq2 with primer prefix:
NO result; either a problem of primer or a problem of sequencing.
2. b) sequencing of pSB1C3-seq2 with primer suffix:
VF2: not covered
Prefix: not covered
promoter: not covered
RBS: not covered
AmilCP: not completely covered (only the end)
double terminator: not completely covered (only the beginning)
suffix: not covered
VR: not covered
We don't have the sequence of AmilCP/DSB. This is a problem of sequencing or primer. We cannot take these results into account.
3. a) sequencing of pSB1C3-seq7 with primer prefix:
VF2: not covered (normal it is the beginning)
Prefix: not covered (normal it is the beginning)
UPelmt/CspA prom: not completely covered, only the end.
5'UTR: OK
AmilCP/DSB: OK
double terminator: not completely covered
suffix: not covered
VR: not covered
This sequencing result is quite good as we can see the main parts: Upelmt/CSPAprom (not entirely), 5'UTR, AmilCP/DSB.
3. b) sequencing of pSB1C3-seq7 with primer suffix:
Not a good sequencing. We have only the double terminator. But we can use the previous one which is good.
4. a) sequencing of miniprep pSB1C3-seq8 (culture 08/09) with primer prefix:
pL promoter: not completely covered, there are some mismatches
eRFP: OK
This sequencing result is acceptable but the pL promoter is not completely sequenced.
4. b) sequencing of miniprep pSB1C3-seq8 (culture 08/09) with primer suffix:
Not very relevant: we have only the double terminator. But we can use the previous result.
5) a. sequencing miniprep pSB1C3-seq8 (culture 25/09) with primer prefix:
NO RESULT
5) b. sequencing miniprep pSB1C3-seq8 (culture 25/09) with primer suffix:
NO RESULT
6) a. sequencing miniprep pSB1C3-seq9 (culture 08/09) with primer prefix:
VF2: OK
Prefix: OK
promoter: OK
RBS: quite OK
CI857: quite OK
Terminator: not completely covered.
eColi operon: not covered
eRFP: not covered
RBS: not covered
pL: not covered
We don't have the entire sequence. (remember the gel: we had only the half of the sequence).
6) b. sequencing miniprep pSB1C3-seq9 (culture 08/09) with primer suffix:
VF2: not covered
Prefix: not covered
promoter: not covered
RBS: not covered
CI857: not covered
Terminator: not covered.
eColi operon: OK
eRFP: not covered
RBS: not covered
pL: not covered
Here we have the end of the sequence. On the previous one, we have the beginning. It seems that despite what we saw on the gel yesterday, we have the entire sequence in our plasmid.
7. a) sequencing of pSB1C3-seq9 (21/09) with primer prefix:
VF2: OK
Prefix: OK
promoter: OK
RBS: OK
CI857: Ok but some mismatches
Terminator: OK
eColi operon: not covered
eRFP: not covered
RBS: not covered
pL: not covered
7. b) sequencing of pSB1C3-seq9 (21/09) with primer suffix:
VF2: not covered
Prefix: not covered
promoter: not covered
RBS: not covered
CI857: not covered
Terminator: only the end is covered
eColi operon: OK
eRFP: OK
RBS: OK
pL: OK but not completely covered
Week 7
10/02 - Miniprep for sequences 1 / 2 / 7 / 8 / 9 and cultures for sequences 1 / 2 / 7 / 8 / 9
Objective: To make stocks of our sequenced plasmids and prepare cultures for further characterisation.
Part 1: Minipreps
Material:
Monarch® Plasmid Miniprep Kit (NEB)
Micropipettes + tips
Eppendorf tubes
Centrifuge
Ice
Protocol:
The Monarch protocol and Antoine's indications were used. The same protocol as usual was used. Please refers to our “Protocols” section.
Part 2: Cultures of DH5alpha transformed with sequences 1, 2, 7, 8 and 9
Material:
LB medium
Chloramphenicol
50mL falcon tubes
Micropipettes + tips
Incubator
Protocol:
We prepared for each culture 100µL of pre-cultures in 10ml of LB medium with antibiotics.
Remark:
seq8 (1) and seq9(1) → come from culture prepared the 08/09
seq8 (2) and seq(2) → come from culture prepared the 25/09 and 21/09 respectively
10/03 - PCR colony on japanese sequences
Objective: To check if the bacteria have the correct insert sizes.
Part 1: PCR colony
Material:
10X Standard taq reaction Buffer (NEB)
10µM dNTPS (NEB)
Forward primer 100µM (synthesized by IDT)
Reverse primer 100µM (synthesized by IDT)
Taq DNA polymerase (5000U/mL)
DNAse/ RNAse free water
PCR tubes
Thermocycler
micropipettes and tips
Protocol:
Prepare the PCR mix in each PCR tube (the enzyme is added at the end) stay in ice.
Japanese pHeat: 3 colonies (white) (1694bp)
Japanese pCold: 3 colonies (white) (1033bp)
Each chosen colony has been annotated on the petri dishes so we will be able to use them again for liquid culture later. They are now stocked in the fridge, protected by parafilm.
Cell membrane destruction:
In each PCR tube: put 8µL of ddH20. Pick a colony (with a toothpick or a micropipette tip), add a stab of it into the PCR tube. Incubate the tubes at 95°C for 5min.
Prepare the PCR mix:
Component
Volume
Final concentration
10X Standard Taq Reaction Buffer
17,5 µL
1X
10mM dNTPs
3,8 µL
200µM (1/50)
10µM Prefix Primer
1,75 µL
1µM
10µM Suffix Primer
1,75 µL
1µM
Taq DNA Polymerase
1,75 µL
2,5 units/50µL PCR
Nuclease-free water
148,75µL
qsp 175µL
Put 17µL of mix in each tube (25-8=17)
PCR thermocycler program:
PCR Cycle
Repetition
Step
Time
Temperature
0
Initial denaturation
30 sec
95°C
30
Denaturation
30 sec
95°C
Annealing
1 min
60°C
DNA polymerization
1  min (30 sec/kb)
68°C
0
DNA polymerization
5 min
68°C
FINISH
∞
4°C
Part 2: Gel migration:
Material and Protocol:
We made an agarose gel according to the protocol (please refer to our "Protocols" section).
We prepared the samples to  load in the gel:
10µL of the PCR mix + 2µL of 6X loading dye
Deposit plan:
Ladder (5µL)
pCold 1
pCold 2
pCold 3
pHeat 1
pHeat 2
pHeat 3
PCR 9 (old)
Ladder (5µL)
Gel migration was performed during 1h30 at 110V.
The gel was incubated in a GelRed bath during 1h30.
Results:
The gel was unreadable (blurred ladder and GelRed everywhere).
10/03 - OD measure for sequences 1 and 2, and protein extraction
Objective: To quantify a protein expression.
Part 1: OD600nm of bacterial culture (seq 1&2)
Material:
LB medium
Chloramphenicol
Micropipettes + tips
Spectrophotometer
Cuvettes
Protocol:
Blanc = LB + chloramphenicol
OD600nm DH5alpha-pSB1C3-seq1 = 0,719
OD600nm DH5alpha-pSB1C3-seq2 = 0,692
Part 2: Protein extraction
Material:
LB medium
Chloramphenicol
50mL Falcon tubes
Eppendorf tubes
Centrifuge
Spectrophotometer
Cuvettes
Vortex
Freezer
Waterbath
Protocol:
Extraction of total proteins from:
WT
DH5alpha-pSB1C3-seq1
DH5alpha-pSB1C3-seq2
We have to adapt the protocol because the bacterial culture's OD has been measured into LB medium+chloramphenicol. So if we resuspend the proteins into water, we won't be able to compare the OD. So we need to resuspend the proteins into LB medium + chloramphenicol.
Grow the cells and induce protein expression
Take as 1mL sample from the cell culture at OD600 of 1 as experiments you would like to carry out. If the OD of the culture is not 1 you have to correct the sample volume to obtain an equivalent size pellet.
Spin down the cells for 5min at 6000 rpm in a microfuge.
For each pellet, remove supernatant and add 150µL of LB medium + chloramphenicol.
Vortex to resuspend the cells.
Freeze quickly at -20°C and leave for 3 min.
Thaw immediately at 42°C 3mn. Vortex vigorously to mix well.
Repeat the two previous steps for three more times (4 freeze-thaw-vortex cycles in all).
Spin down the cells for 5min at 6000 rpm in a microfuge. Proteins are contained in the supernatant.
Results:
Interpretation:
The protein extraction did not work (the color is still in the pellet).
We will try again the same protocol but with ultrapure water instead of LB, because maybe water is absolutely needed for cell lysis.
10/04 - Protein extraction: AmilCP from sequences 1 and 2
Objective: We want to extract the AmilCP from colonies transformed with AmilCP (seq 1) and with AmilCP + DS box (seq 2).
Material:
Bacterial cultures
Ultrapure water
Eppendorf tubes
Centrifuge
Spectrophotometer
Cuvettes
Vortex
Freezer
Waterbath
Protocol:
Put 1mL of culture into eppendorf tubes
Centrifuge at 6000rpm for 5min
Discard supernatant, resuspend in ultrapure water
Centrifuge again at 6000rpm for 5min, discard supernatant
Freeze-thaw cycles: 5min -20°C; 5min 80°C (4 times)
Resuspend in ultrapure water
Centrifuge at 6000rpm for 5min
Results:
We failed at extracting the proteins.
Week 8
10/09 - Characterisation Protocole 0: Testing the kinetic of evolution of E.coli in Liquid LB at 15°C and 37°C
Objective: To perform a growth curve of E.coli DH5alpha at 37°C and at 15°C to compare the kinetic, as we want to anticipate the kinetic of growth of an E.coli population at 15°C and compare it to the one at 37°C.
Material:
Liquid LB without antibiotic
DH5 alpha E.Coli WT
Sterile Erlenmeyers
Shaker-incubator 37°C 250 rpm and Multitron 15°C 250 rpm
Centrifuge
Spectrophotometer
Cuvettes
Protocol:
Put 50 ml of LB in each erlenmeyers
Incubate 1 ml of E.Coli DH5 alpha WT in the two different erlenmeyers.
Immediately take off 1ml of culture IN ICE and calculate OD600 for t=0
Put one erlenmeyer at 37°C 250 rpm and the other one at 15°C and repeat every 1h30 for each erlenmeyers and note OD600.
Carry on until you reach stationary phase
Results:
Figure 1 - OD measured at 600nm at 15°C and 37°C at different times.
Figure 2 - Evolution of the OD600nm measured according to the time
Interpretation:
At 37°C, exponential phase is reached after 3h and stationary phase after 6h of incubation.
No growth at 15°C.
10/09 - Characterisation Protocole 1: Protein extraction
Objective: To extract the total intracellular proteins from E.coli to measure the absorbance and deduce the proteins (amilCP, mRFP) concentrations.
Material:
SDS 1%
PBS 10X w/o Ca2+ and Mg2+ → PBS 2014 (dubious)
ICE COLD Acetone
Bacteria culture you wish to work on --> culture seq8 (the blue one)
Centrifuge
Eppendorf tubes
Micropipette + tips
Protocol:
Stock: SDS (20%) stock solution
Preparation of a 1% SDS solution: put 10ml of 20% SDS in a 250mL bottle and add 190ml of deionised water.
Preparation of a 1X PBS solution: put 100ml of 10X PBS solution and complete to 1L with deionised water.
1) Cells from 3 ml of culture were harvested by centrifugation (13000 x g 1 min), washed twice with phosphate-buffered saline without Mg2+ and Ca2+ (3), and recentrifuged.
2) The cells were then suspended in 1,5 ml of ice-cold acetone (analytical grade), allowed to stand on ice for 5 min, and collected by centrifugation (7,000 x g).
Experiments demonstrated that acetone treatment from 5 to 30 min or the use of more acetone did not change the efficiency of protein extraction.
3) The proteins were then extracted by incubating with 150ul ml of 1% sodium dodecyl sulfate (SDS) for 2 min.
The acetone pretreatment was found necessary since the direct utilization of detergents did not extract cellular proteins from gram-positive bacteria.
4) To perform an absorption spectrum we completed the volume with 1,5ml of water.
Source: Bhaduri et al., “Simple and Rapid Method for Disruption of Bacteria for Protein Studies”, APPLIED AND ENVIRONMENTAL MICROBIOLOGY, Oct. 1983, p. 941-943
Results:
Interpretation:
Partial results, not all the proteins were extracted but it allows to realise an absorption spectrum. We can see the absorption peak of amilCP and another peak which could be mRFP. (the culture was purple → mix of blue and red).
10/09 - Miniprep for sequences 1 / 2 / 7 / 8 / 9 and cultures for sequences 1 / 2 / 7 / 8 / 9
Objective: To make stocks of our sequenced plasmids and prepare cultures for further characterisation.
Part 1: Miniprep
Material:
Monarch® Plasmid Miniprep Kit (NEB)
Micropipettes + tips
Eppendorf tubes
Centrifuge
Ice
Protocol:
The Monarch protocol and Antoine's indications were used. The same protocol as usual was used. Please refers to our “Protocols” section.
Part 2: Cultures of DH5alpha transformed with seq 1,2,7,8,9
Put 100µL of precultures in 10ml of LB medium with antibiotics.
10/10 - Characterisation Protocol 1: Protein extraction for mRFP
Objective: To extract the total intracellular mRFP from E.coli in order to measure the absorbance and deduce the proteins concentration.
Material:
SDS 1%
PBS 10X w/o Ca2+ and Mg2+ → PBS 2014 (dubious)
ICE COLD Acetone
Bacteria culture you wish to work on → culture seq 8 (the blue one)
Protocol:
Stock: SDS (20%) stock solution
Preparation of a 1% SDS solution: put 10ml of 20% SDS in a 250mL bottle and add 190ml of deionised water.
Preparation of a 1X PBS solution: put 100ml of 10X PBS solution and complete to 1L with deionised water.
1) Cells from 3 ml of culture were harvested by centrifugation (13000 rpm 1 min), washed twice with 1,5 ml of phosphate-buffered saline without Mg2+ and Ca2+ (3) (homogenize with pipeting and vortex), and recentrifuged (13000 rpm, 1min).
2) The cells were then suspended in 1,5 ml of ice-cold acetone (analytical grade), allowed to stand in fridge for 5 min, and collected by centrifugation (13000 rpm 1min).
Tips: Break the pellet with your P1000 tips right after you put the acetone.
Experiments demonstrated that acetone treatment from 5 to 30 min or the use of more acetone did not change the efficiency of protein extraction.
3) The proteins were then extracted by incubating with 150ul ml of 1% sodium dodecyl sulfate (SDS) for 2 min. Homogenize by pipeting and vortexing. Centrifuge 1min at 13000rpm to get the supernatant.
The acetone pretreatment was found necessary since the direct utilization of detergents did not extract cellular proteins from gram-positive bacteria.
4) Put the supernatant in an eppendorf tube and add 1,35ml of water (Vtotal=1,5ml). Vortex. Perform the absorption spectrum. For the blank: 150ul of SDS and 1350µL of water (homogenize).
Source: Bhaduri et al., “Simple and Rapid Method for Disruption of Bacteria for Protein Studies”, APPLIED AND ENVIRONMENTAL MICROBIOLOGY, Oct. 1983, p. 941-943
Results:
The culture used for extraction was seq 9 (1) (we couldn't see the red colour directly but after centrifugation the pellet was slightly pink). A little peak was observed at 584 nm.
10/10 - Characterisation Protocole 2: DSBox effect on AmilCP Properties - Determination of the absorption peak
Objective: To assess the effect of the DSBox alone on amilCP coloration properties (absorbance spectrum).
Material:
O/N culture of Seq 1 at 15 or 37°C
O/N culture of Seq 2 at 15 or 37°C
O/N culture of WT at 15 or 37°C
SDS 1%
PBS w/o Ca2+ and Mg2+
ICE COLD Acetone
Protocol:
1) IN ICE Take 4,5 ml of each culture and use 1,5ml to measure the OD600.
2) IN ICE Note and adjust the OD600 to obtain the same cell concentration in each tubes
3) Once the concentration is adjusted, take 3ml of each adjusted culture and proceed the protein extraction protocol.
4) With the 1,5ml of protein extract of each culture and the blank, make an absorption spectrum and note the results.
You should at the end obtain the complete spectrum of AmilCP both native and modified at both 15 and 37°C
→ Determine the maximal absorbance and the corresponding concentration
NOTE: The protocol failed and the proteins were not extracted. We had to re-do it and include the acetone step again with what was remaining of the samples (600 µl).
This will explain the low absorbance values we obtain in the results. The extraction worked with the acetone.
Results:
After doing the blank with the extracted proteins of the Wild-Type culture, we obtained:
The absorption spectrum of the native AmilCP from the sequence 1:
The absorption spectrum of the modified AmilCP with the DSBox from the sequence 2:
Interpretation:
The peaks are very similar but present some variations, although the native AmilCP spectrum is analysable on a different scale (ymax = 0.002) that the one of the modified AmilCP (y = 0,09) and the resolution of the other smaller peaks is hence different and they are more easily distinguishable. This proves however that the maximum of absorption from the native and the modified AmilCP is the same, at 588 nm. We will carry on the same experiment soon too see if we have similar results.
10/10 - Migration PCR colony japanese plasmids
Objective: To migrate the PCR colony results from the 10/03.
Protocol:
We made an agarose gel according to the protocol (please refer to our "Protocols" section).
Preparation of the samples:
for pCold and pHeat sequences: 10µL PCR + 2µL loading dye
for sequence 9 (old) from PCR: 5µL + 1µL loading dye
Deposit plan:
Ladder (5µL)
pCold 1 (12µL)
pCold 2 (12µL)
pCold 3 (12µL)
pHeat 1 (12µL)
pHeat 2 (12µL)
pHeat 3 (12µL)
PCR 9 (old) (6µL)
Ladder (5µL)
Gel migration was performed during 1h30 at 110V.
The gel was incubated in a GelRed bath during 1h30.
Results:
Interpretation:
pCOLD (1033bp): 1200 bp
pHEAT (1694bp):  1 band at 1700bp (light) and 1 at 400bp
10/10 - 10/11 - Characterisation Protocole 0': Measure of the kinetic apparition of AmilCP
Objective: To measure the kinetic of AmilCP.
Protocol and Results:
We decided to carry on a kinetic of protein expression during 24hours to measure the influence of the DSBox on AmilCP expression by comparing the seq1 and 2, but the manipulation was unsuccessful and protein extraction was very compromised, hence we decided to proceed the Cold-shock characterisation with colorimetric measures only, waiting for our results through the iGEM Bettencourt collaboration and Nicolas Cornille's help.
10/11 - PCR on sequence 3 and Protegrin-1
Objective: Amplification of Protegrin-1 (BBa_K628006) (418 bp) and sequence 3 (BBa_K228007) (1091 bp).
Part 1: PCR
Material:
10X Standard taq reaction Buffer (NEB)
10µM dNTPS (NEB)
Transformed colonies (on solid LB medium)
Forward primer 100µM (synthesized by IDT)
Reverse primer 100µM (synthesized by IDT)
Taq DNA polymerase (5000U/mL)
DNAse/ RNAse free water
PCR tubes
Thermocycler
Micropipettes and tips
Protocol:
We divided our 50µL mix into two 25ul mixes. The PCR is then performed with only 18 cycles to reduce the risk of errors (Taq polymerase not very accurate). Finally the two PCR products are mixed together before the clean up.
Pre-dilution Primers 100µM: 1,2 µL primer + 10,8 µL dH2O → 10µM final, volume total final
Pre-dilution Taq Polymerase 5000U/mL: 1,2µL Taq + 10,8µL dH2O → 500U/mL final, volume total final
For a 50µL PCR mix volume:
Component
Volume
Final concentration
10X Standard Taq Reaction Buffer
5 µL
1X(1/10)
10mM dNTPs
1 µL
200µM (1/50)
10µM Prefix Primer
5 µL
1µM
10µM Suffix Primer
5 µL
1µM
Taq DNA Polymerase 500U/mL
5 µL
2,5 units/50 µL PCR
DNA template (10ng/µL)
2 µL
Nuclease-free water
27 µL
qsp 50ul
PCR program on the thermocycler:
PCR Cycle
Repetition
Step
Time
Temperature
0
Initial denaturation
30 sec
95°C
18
Denaturation
30 sec
95°C
Annealing
1 min
60°C
DNA polymerization
seq 3: 1 min
protegrin-1: 30 sec(30 sec/kb)
68°C
0
DNA polymerization
5 min
68°C
FINISH
∞
4°C
Part 2: Gel migration
We made an agarose gel according to the protocol (please refer to our "Protocols" section). We are going to use 2/4µl of the PCR products to quantify the sequences concentrations efficiently.
Mixes for deposit: 7µL PCR product + 1,4µL 6X loading dye → 8,4µL total
2µl DNA: 2,4µl of the mix
4µl DNA: 4,8µl of the mix
Deposit plan:
Ladder
protegrin-1 (2µL)
protegrin-1 (4µL)
sequence 3 (2µL)
sequence 3 (4µL)
Ladder
Gel migration was performed during 1h30 at 110V.
The gel was incubated in a GelRed bath during 1h30.
Results:
Interpretation:
We obtained two bands for the protegrin-1 at the expected size (between 400 and 600 bp), but no bands for the sequence 3, we suppose PCR has failed for this one.
10/11 - Digestion protegrin-1 + Ligation
Objective: To digest the sequence protegrin-1, and then ligate it into pSB1C3 backbone.
Part 1: Digestion
Material:
Sequence protegrin-1
Plasmid pSB1C3-mRFP (BBa_J04450 from IGEM Kit)
EcoRI-HF (NEB: 20000U/mL)
PstI (NEB: 20000U/mL)
10X NEB Buffer 2.1
Ultrapure water
1,5mL Eppendorf tubes
Micropipettes + tips
Waterbath
Heat Block
Protocol:
We prepared the mixes according to the following instructions:
Reagents
Quantity
Final concentration
EcoRI-HF (20000U/mL)
1 µL
80U/µL
PstI (20000U/mL)
1 µL
80U/µL
Purified sequence
30 µL
10ng/µL
10X NEB buffer 2.1
5 µL
1X
Ultrapure water
13 µL
Digestion was carried out 2 hours at 37°C
Heat inactivation was proceeded at 80°C for 20 minutes.
Part 2: Ligation
Material:
Digested insert (protegrin-1 and backbone pSB1C3)
Digested backbone (pSB1C3)
10X T4 DNA Ligase Buffer (NEB)
T4 DNA ligase (400000U/mL)
Ultrapure water
Eppendorf tubes
Micropipettes and tips
Protocol:
We prepared the mix for the ligation according to the following instructions (ratio 1:5):
insert (protegrin)
15,9µL
backbone pSB1C3
pSB1C3-mRFP (double digested, 25ng) 1,3µL
10X T4 DNA Ligase Buffer
2µL
T4 DNA Ligase
1µL
dH2O
6,1µL
Total volume
20µL
The sample was incubated overnight in the fridge.
10/12 - Transformation Protegrin + produits minipreps 1, 2, 7, 8, 9 + GFP constitutive (iGEM distribution kit)
Objective:
To amplify the ligation product of protegrin intro pSB1C3-mRFP, and select the right colonies to do minipreps and send to GATC
To cultivate the miniprep products of plasmids 1, 2, 7, 8 & 9 to assess prepare fresh cultures
To amplify the constitutive GFP plasmid to serve as a control for the pCOLD japanese test (plate 4 well 18A)
Material:
Ligated plasmids
Competent cells (DH5-alpha)
LB medium liquid and solid
SOC medium
Chloramphenicol
Ice
Eppendorf tubes
Micropipettes + tips
Waterbath
Petri dishes
Glass beads
Incubator
Centrifuge
Protocol:
Clean your working area by wiping down with 70% ethanol.
Thaw competent cells on ice. Label one 1.5 mL microcentrifuge tubes for each transformation and then pre-chill by placing the tubes on ice.
Tubes
DNA
Plate
Competent cells
SOC medium
Protegrin
20 µL
LB + Chloramphenicol
50 µL
930 µL
1
10 µL
LB + Chloramphenicol
50 µL
940 µL
2
10 µL
LB + Chloramphenicol
50 µL
940 µL
7
10 µL
LB + Ampicillin
50 µL
940 µL
8
10 µL
LB + Chloramphenicol
50 µL
940 µL
9
10 µL
LB + Chloramphenicol
50 µL
940 µL
GFP
3 µL
LB + Kanamycin
50 µL
947 µL
Ligase +
20 µL
LB + Chloramphenicol
50 µL
930 µL
Ligase -
20 µL
LB + Chloramphenicol
50 µL
930 µL
Pipet DNA in each tube and then 50 µL of competent cells into each tube. Flick the tube gently with your finger to mix.
Incubate on ice for 30 minutes. Put it in the fridge, the temperatures in the room are too high.
Pre-heat waterbath now to 42°C. Otherwise, hot water and an accurate thermometer works, too!
Heat-shock the cells by placing into the waterbath for 30 seconds (no longer than 1 min). Be careful to keep the lids of the tubes above the water level, and keep the ice close by.
Immediately transfer the tubes back to ice, and incubate on ice for 5 minutes. IN THE FRIDGE AS WELL
Add qs 940 µL of SOC media WITHOUT CHLORAMPHENICOL per tube, and incubate at 37°C for 1 hour 30min shaking at 200-300rpm.
Prepare the agar plates during this time: label them, and add sterile glass beads if using beads to spread the mixture. Put the agar plates at 37°C.
Tubes were centrifuged at 3500rpm for 4minutes and 850µL of supernatant was discarded.
Bacteria were resuspended into the remaining culture media and bacteria were spread in the corresponding petri dishes.
Important notes:
The petri dishes we wanted to use were dried and we noticed that during the SOC incubation. We hence made new dishes quickly but the cultures had to stay in the SOC about 2h longer than usual. This is supposingly not impacting the cultures.
We however incubated 100µl of each culture (without centrifugation) on the dried plate to ensure security.
Results:
Interpretation:
Results of the transformation with GFP constitutive under UV light after 2 days of incubation at 37°C, in comparison with a transformation with sequence 7 (negative control).
The results are not significant.
As for the dishes made with long incubated bacteria in SOC medium, the dishes were crowded and uniform.
10/12 - Cold shock characterisation
Objective: 100µl of each transformed culture has been incubated in corresponding LB+ antibiotics medium. The objective is to measure the growth of the cold shock through optical density (OD) at 600 nm and then when it reaches 0.5 at 600 nm, extract the proteins and measure OD at 588 nm.
Protocol:
For all measures, 1,5 ml of liquid culture were taken from cultivated medium.
Results:
OD600 8h30 PM: OD 600 Sequence 7: 0.013 (We added 1ml of fresh seq 7 that was incubating since the 10/11 to "boost" the growth and avoid any time shift. The bacteria came from the same original glycerol stock. The addition was not sufficient and we added 1 more mL that we however pelleted into 200µl before).
OD 600 9h40 PM: OD 600 Sequence 7: 0,139 -The boost seems to have worked but not enough. OD 600 Sequence pCold: 0,522 → 9h53 PM: one cuve sent to be observed under fluorescence, then we divided the volume in two separate erlenmeyer with one at 15°C and the other at 37°C.
OD600 11h10 PM: OD 600 Sequence 7: 0.285 (0,325 after adding 1,0ml of sequence 7 pelleted into 200µl from 11/10 coming from the same glycerol stock)
Sequences
OD 600 17H20
OD600 19H
OD 600 20H30
OD600 21H40
OD600 23H10
Sequence 1
0.004
0.055
0.166
0.307
0,486
Sequence 2
0.008
0.041
0.155
0.298
0,479
Sequence 7
0.007
0.013
0.013
0.139
0,285
pCold
0.023
0.167
0.370
0.522
After the OD600 measurements, 40mL was left in each erlenmeyers, we splitted the cultures in two. For each cultures we obtained 20mL liquid medium with our bacteria, 20mL was put at 15°C and the other was kept at 37°C.
Incubation sequences 1 and 2 at 15°C at 23H45 (divided in 2 with one half at 37°C and another at 15°C).
OD600 and incubation at 00h15 AM: Sequence 7: 0,464 ==> divided in 2 with one half at 37°C and another at 15°C .
Time of incubation split:
pCOLD: 9h53 PM Sequence 1: 11h45 PM Sequence 2: 11h45 PM Sequence 7: 00h15 AM
10/12 - Minipreps sequence pCold and pHeat
Objective: To make stocks
Material:
Monarch® Plasmid Miniprep Kit (NEB)
Micropipettes + tips
Eppendorf tubes
Centrifuge
Ice
Protocol:
The same protocol as usual was used.
NB: Pay attention to the elution volume. We eluted in 60µL nuclease-free water and not 30µL.
10/13 - Liquid culture of transformed bacteria
Protocol:
Liquid cultures of the transformation obtained from the first batch of plate (on semi-dried petri dishes) were made with 10ml of liquid LB supplemented with appropriate antibiotics:
The colonies were selected based on their colors. The GFP transformed colonies emitted no color but were transformed with pure plasmid from the iGEM plates so we were confident no other colonies than the transformed ones would be on the plate.
Seq 1/2/8/9 - Chloramphenicol
Seq 7 - Ampicillin
GFP Constitutive - Kanamycin
Results:
The ligase control shown appropriate results.
The culture liquid of the Protegrin transformed bacteria was not made since we had to wait for the results of a PCR Colony to proceed.
UPDATE:
Half the cultures had grown after 24h, we assumed the RPM were not optimal and the culture might have been badly poured into the medium at first, another batch will be done on the 10/14.
10/13 - PCR colony on Protegrin
Objective: Select the colonies transformed with the plasmid pSB1C3-protegrin-1 to further purify it and send it to GATC.
Part 1: PCR colony
Material:
10X Standard taq reaction Buffer (NEB)
10µM dNTPS (NEB)
Forward primer 100µM (synthesized by IDT)
Reverse primer 100µM (synthesized by IDT)
Taq DNA polymerase (5000U/mL)
DNAse/ RNAse free water
PCR tubes
Thermocycler
micropipettes and tips
Protocol:
We picked:
for protegrin-1: 3 colonies (white) (418bp)
for the control: 1 pSB1C3-mRFP (1069bp)
Pre-dilutions (1/10):
Primers: 1µL + 9µL dH2O → 10µM final
Taq Polymerase 5000U/mL: 1µL + 9µL dH2O → 500U/mL final
Each chosen colony has been annotated on the petri dishes so we will be able to use them again for liquid culture later. They are now stocked in the fridge, protected by parafilm.
Cell membrane destruction:
In each PCR tube: put 8µL of ddH20. Pick a colony (with a toothpick or a micropipette tip), add a stab of it into the PCR tube.
Incubate the tubes at 95°C for 5min.
Tube mRFP: put 2µL DNA template
Prepare the PCR mix:
Component
Volume
Final concentration
10X Standard Taq Reaction Buffer
8µL
1X
10mM dNTPs
1,6µL
200µM (1/50)
10µM Preffix Primer
8µL
1µM
10µM Suffix Primer
8µL
1µM
Taq DNA Polymerase
8µL
2,5 units/50µL PCR
Nuclease-free water
46,4µL
qsp 80µL
Put 17µL of mix in each colony tube (25-8=17)
Put 23µL in the mRFP tube (25-2=23)
PCR thermocycler program:
PCR Cycle
Repetition
Step
Time
Temperature
0
Initial denaturation
30 sec
95°C
30
Denaturation
30 sec
95°C
Annealing
1 min
60°C
DNA polymerization
protegrin-1: 30 sec
mRFP: 1 min (30 sec/kb)
68°C
0
DNA polymerization
5 min
68°C
FINISH
∞
4°C
Part 2: Gel migration
We made an agarose gel according to the protocol (please refer to our "Protocols" section).
Preparation of the samples for the migration: 10µL sample + 2µL 6X loading buffer
Deposit plan:
Ladder (5µL)
Protegrin-1 1
Protegrin-1 2
Protegrin-1 3
Ladder (5µL)
Gel migration was performed during 1h30 at 110V.
The gel was incubated in a GelRed bath during 1h30.
Results:
Interpretation:
The colony 2 seems better (the band is more visible).
10/13 - Cold shock characterisation
Objective: To measure the growth of the cold shock through Optical Density (OD) at 600 nm and then when it reaches 0.5 at 600 nm, extract the proteins and measure OD at 588 nm.
Protocol:
For all measures, 1,5 ml of liquid culture were taken from cultivated medium.
Beginning of extraction: 5pm → Total of approximately 18h of incubation at 15°C and 37°C for all cultures.
Visual results: we pelleted 1.5ml of each culture
Wild-type DH5alpha (ancient culture in liquid LB at 37°C)
Results:
Interpretation:
A: Wild-type DH5 alpha (ancient culture in liquid LB at 37°C)
B: Sequence 7 at 15°C (Left) and 37°C (Right): We can clearly see AmilCP expression at 15°C, so this is good for us ! Note the greyish aspect for the 37°C pellet, implying a potential low expression of the protein (but still very low as compared to the 15°C pellet).
C: Another sneak peak at our sequence 7  expected results.
D: Seq 2 at 15°C (left) and at 37°C (right): Here the difference is not exceptional but the right pellet coming from the culture at 37°C is greyer than the left one, showing "better" AmilCP expression and coherent results.
The poor expression is supposedly due to the long time storage in LB of our Seq 2 transformed colonies, we hence will re-do the protocol with fresh transformed bacteria and analyse.
E: Seq 1 at 15°C (left) and 37°C (right): exact same results as for the sequence 2, we still need confirmation with fresh bacteria.
F: pCOLD: The left pellet seems a little fluorescent as compared to the right one. Efficiency of the response will be further seen on the 10/14 and with the Bettencourt collaboration.
Protein extraction results:
Seq 1 - 15°C: DO = 0.012
Seq 2 - 15°C: DO = 0.025
Seq 7 - 15°C: DO = 0.007
Seq 1 - 37°C: DO = 0.017
Seq 2 - 37°C: DO = 0.011
Seq 7 - 37°C: DO = 0.014
10/14 - Cold shock characterisation
Objective:We end this first characterisation by a final visual measure on our culture. We decided to skip the protein extraction protocol since it provided very few results on the 10/13 and was not reliable.
Protocol:
Hour of end: 5h11 pm
Total: 37-44h
Results:
OD600 Seq 1 37°C: 0,690
OD600 Seq 1 15°C: 0,688
OD600 Seq 2 37°C: 0,685
OD600 Seq 2 15°C: 0,674
OD600 Seq 7 37°C: 0,715
OD600 Seq 7 15°C: 0,615
OD600 Seq pCOLD 37°C: 0,698
OD600 Seq pCOLD 15°C: 0,674
Interpretation:
OD600 are almost equal between all samples, showing that the maximal growth plateau has been reached. These similar values allow a relevant comparison of cells coloration, as pellets contain approximately the same number of bacteria.
Once again, this experiment proves the efficiency of the cold-shock plasmid (pSB1C3-seq7): no coloration was observed at 37°C after 37h, whereas a strong blue coloration appeared during the same time at 15°C.
In comparison, the sequences 1 and 2 coding for constitutive amilCP expression remained white at 15°C and gave a slight blue coloration at 37°C. At 15°C, the absence of coloration can be explained by the cold temperature slowing down the metabolic process. At 37°C, we should observe a stronger coloration. However, amilCP takes approximately 30 hours to be fully functional. After 40h, it is possible that the amilCP expression was just at its beginning. We should therefore reproduce the experiment after a longer incubation. Indeed, some liquid culture put at 37°C for several days displayed a strong blue coloration.
Week 9
10/16 - Miniprep of sequences 1/2/7/8/9/Protegrin/Constitutive GFP of 14/10 cultures
Objective: To stock our plasmids to send them to the iGEM. We are going to migrate a sample of each miniprep to make sure they are present. Some of the protegrin miniprep will also be sent to GATC to be sequenced.
Material and Protocol:
The same protocol as usual was used for the miniprep of each plasmid.
10/16 - Cold shock characterisation Duplicat 1 - Day 1
Objective: To obtain the same results as last week a second time in order to confirm those results.
Protocol:
This time no pCOLD culture was made. We assessed it would be more efficient to quantify pCOLD effect with the use of microplate reader for the GFP.
We prepared then:
One culture of 1ml of seq 1 from the 10/14 liquid culture in 50ml LB medium + chloramphenicol
One culture of 100µl of seq 2 from a glycerol stock  in 50ml LB medium + chloramphenicol. We took the glycerol stock because of unwished events with the 10/14 culture. We later took the 10/13 culture to enhance the growth of the former, because it was very slow. Both cultures came from the same origin and were transformed by the same plasmid.
One culture of 1ml of seq 7 from the 10/14 liquid culture in 50ml LB medium + ampicillin
OD600 at t=0h Seq1: 0,062 Seq2: 0,010 Seq7: 0,038
OD600 at t=1h30 Seq1: 0,057 Seq2: 0,010, we added 1ml of 13/10 and reached 0,031 Seq7: 0,092
OD600 at t=3h Seq1: 0,077 Seq2: 0,029 Seq7: 0,322
From this we assessed we would measure the next OD600 of seq 7 in one hour and leave the other incubate more.
OD600 at t=4h Seq7: 0, 463
Alright ! We splitted the culture in two erlenmeyer of 20ml, one was put at 15°C and the other at 37°C for 24h
Time of incubation: 16h20
OD600 at t=5h Seq1: 0,195 Seq2: 0,103
OD600 at t=6h30 Seq1: 0,357 Seq2: 0,257
From this we assessed we would measure the next OD600 of seq 1 in one hour and leave the seq2 incubate more
OD600 at t=7h30 Seq1: 0,473
We splitted the culture in two and incubated one erlenmeyer at 37°C and the other at 15°C
Time of incubation: 19h10
OD600 at t=8h15 Seq2: 0.488
Time of incubation: 20h05
10/16 - Competent cells by CaCl2 - Day 1
Objective: To make competent cells.
Protocol:
We incubated wild-type DH5 alpha from the IGEM 2016 stock into 15 ml of liquid LB medium without antibiotics.
10/17 - Migration of minipreps from 10/16
Objective:We decided to migrate 5µl of each miniprep from the 10/16 to ensure everything was here before sending anything to GATC (protegrin) or the IGEM registry. This migration will also permit us to quantify the pSB1A3-Seq7 and pSB1K3-GFP before following digestions and ligations into pSB1C3. We also migrated a pSB1C3-mRFP control.
Material:
Agarose
Distilled water
TAE buffer 50X
Gel cast and combs
Electrophoresis cuve
Microwave
Micropipettes and tips
2-Log DNA Ladder (0.1-10.0 kb) (NEB)
Gel loading dye purple 6X (NEB)
Protocol:
Sample preparation: 5µl of each sample + 1µl of DNA 6X Loading dye.
We deposited 5µl of each sample in the gel and 5µl of Ladder.
We made an agarose gel according to the protocol (please refer to our "Protocols" section).
Deposit plan:
Ladder
pSB1C3-Seq1
pSB1C3-Seq2
pSB1A3-Seq7
pSB1C3-Seq8
pSB1C3-Seq9
pSB1K3-GFP
pSB1C3-Protegrin
pSB1C3-mRFP
Ladder
Gel migration was performed during 1h30 at 110V.
The gel was incubated in a GelRed bath during 1h30.
Usual average size:
pSB1C3-Seq1: 3000bp
pSB1C3-Seq2: 3000bp
pSB1A3-Seq7: 3000bp
pSB1C3-Seq8: 3000bp
pSB1C3-Seq9: 4000bp
pSB1K3-GFP: 2989 + 919 = about 3800bp
pSB1C3-Protegrin: 2500bp
pSB1C3-mRFP: 300bp
Results:
Interpretation:
If we refer to the control, the seq 8/9/7/Protegrin in their respective plasmids all migrated as predicted. However Seq 1 and 2 seems to be very high (4000bp). The GFP seems a little low too. This migration pattern could be explained by the fact that plasmid migrate sometimes not as planned and that there are different plasmid forms in each well. Maybe the heavier plasmid form (Nicked) predominates totally for seq 1 and 2 and the lighter (linear) predominates for the GFP.
There are important smears for sequences 1/2/7/9.
Importantly enough if we look at the ladder on the right, the size seems more adequate and the unexpected bands could be entirely explained by plasmid forms.
10/17 - Cold shock characterisation Duplicat 1 Day 2
Protocol:
At 10 AM, we took the seq 7 at 15°C and 37°C and pelleted 1,5ml after measuring the OD600:
OD600 seq 7 at t = 18h: 37°C: 0,685 15°C: 0,576
We then we took the seq1 and seq2 out at both 37 and 15°C, 0D600 of 1,5ml of each was measured and then was pelleted:
OD600 seq 1 at t = 18h: 37°C: 0,707 15°C: 0,587
OD600 seq 2 at t = 18h: 37°C: 0,694 15°C: 0,618
Results:
Interpretation:
For sequence 7, again we can clearly see AmilCP expressed for the incubated transformed bacteria at 15°C (blue). We can distinguish a greyish pellet at 37°C, indicating AmilCP expression but in low quantity.
10/17 - Competent cells by CaCl2 - Day 2
Day 2
1. Inoculate 100 mL of LB with 1 mL of the starter culture in a 500 mL Erlenmeyer
2. Place some 50 mL falcon tubes and 1.5 mL eppendorf tubes on ice and put them in the fridge at 4°C. Turn on the centrifuge and cool the rotor to 4°C.
3. Incubate at 37°C with shaking until the cells reach early exponential phase, it takes 1- 3h until the culture becomes slightly turbid (OD600~0.4). Do not let the culture grow more than OD600 0.4.
We incubated 1 ml of wild-type DH5 alpha from yesterday's culture into 100 ml of liquid LB medium without antibiotics and we measure  the OD600 every 1h30.
Time
OD600
0
0,036
1h30
0,073
3h
0,457
4. Aliquot into 50 mL pre chilled tubes and centrifuge 15 min at 4000 rpm in the big centrifuge at 4°C. Decelerate without brake. Discard supernatant.
KEEP THE CELLS IN ICE ALL TIMES AND DO NOT VORTEX.
5. Resuspend GENTLY in 25 mL cold MgCl2 100 mM. Centrifuge cells 15 min at 4000 rpm in the big centrifuge at 4°C. Decelerate without brake. Discard supernatant.
6. Resuspend GENTLY each pellet in 10 mL ice-cold CaCl2 100 mM solution. Keep resuspended cells on ice and place the box in the fridge overnight at 4.
10/17 - Heat response characterisation - Pre culture
Protocol:
Pre-culture of seq 8 and 9 were made in 15ml falcons (3ml of LB + chloramphenicol AMl of culture from 10/14). Incubation was made at 37°C O/N. This is to prevent any long wait caused by any unfreezing of glycerol stocks that could lead to long incubation time to reach OD600 = 0.5.
10/18 - Digestion recombinant plasmids for gel migration
Objective: To digest our recombinant plasmids in order to analyze them on an electrophoresis gel.
Part 1: Digestion
Material:
DNA to digest
EcoRI-HF (NEB: 20000U/mL)
PstI (NEB: 20000U/mL)
10X NEB Buffer 2.1
Ultrapure water
1,5mL Eppendorf tubes
Micropipettes + tips
Waterbath
Heat Block
Protocol:
We perform this digestion on plasmids containing: Seq 1, 2, 7, 8, 9, protegrin and GFP
Preparation of the mix:
5µL of miniprep product
1µL EcoRI HF (20000U/mL)
2,5µL 2.1 NEB Buffer 10X
16,5µL H2O
Digestion was carried out 2 hours at 37°C
Heat inactivation was proceeded at 80°C for 20 minutes.
Part 2: Gel migration
We made an agarose gel according to the protocol (please refer to our "Protocols" section)
Preparation of the samples for deposit: 5µL of digestion product + 1µL 6X loading dye
Deposit plan:
Ladder
pSB1C3 seq 1
pSB1C3 seq 2
pSB1A3 seq 7
pSB1C3 seq 8
pSB1C3 seq 9
pSB1K3 GFP
pSB1C3 protegrin
Ladder
Gel migration was performed during 1h30 at 110V.
The gel was incubated in a GelRed bath during 1h30.
Results:
Interpretation:
After the migration, as compared to the previous gel, all the bands seem to be at the right height. The GFP is higher because of the weight of pSB1K3 (2204bp). The protegrin is located correctly (around 2500bp). The only strange result comes from the seq 2 which presents two bands. That does not make sense and an explanation could be that our plasmid is mutated and that this mutation induced the formation of an EcoRI site, leading to the opening of the plasmid. We are going to use another miniprep of seq 2 that dates from the 08/30 to maybe try to get the unmutated sequence and pursue our experiments.
10/18 - Cold shock characterisation Duplicat 1 - Day 3
At 10am, we took the seq 7 at 15°C and 37°C and pelleted 1,5ml after measuring the OD600:
OD600 seq 7 at t= 42h:
37°C: 0,685
15°C: 0,656
We can clearly see AmilCP expressed for the incubated seq7-transformed bacteria at 15°C (blue). Even after 42h of incubation at 37°C, seq7-transformed bacteria do not express AmilCP.
We then we took the seq 1 and seq2 out at both 37 and 15°C, 0D600 of 1,5ml of each was measured and then was pelleted:
OD600 seq 1 at t = 42h: 37°C: 0,707 15°C: 0,587
OD600 seq 2 at t = 42h: 37°C: 0,694 15°C: 0,618
10/18 - Competent cells by CaCl2 - Day 3 and transformation test with NEB kit
Day 3
1.  Make sure the centrifuge is cold and harvest the cells by centrifuging 15 min @4000 rpm in the big centrifuge at 4°C. Decelerate without brake. Discard supernatant and GENTLY resuspend each pellet in 10 mL ice-cold 100 mM CaCl2  + 15% glycerol solution.
2.  Centrifuge cells 15 min at 2500 rpm 4°C. Discard supernatant and GENTLY resuspend each pellet in 2 mL ice-cold 100 mM CaCl2  + 15% glycerol solution.
3.  Make aliquots with 100 uL each. Freeze immediately at -80°C.
Transformation with cell test kit from NEB:
Pipet 1µl of DNA from the 100µg/µl tube of the NEB Kit in a tube and then 50 µL of competent cells into each tube. Flick the tube gently with your finger to mix.
We also did a blank with 50µl of competent cells and 1µl of water.
Incubate on ice for 30 minutes. Put it in the fridge, the temperatures in the room are too high.
Pre-heat waterbath now to 42°C. Otherwise, hot water and an accurate thermometer works, too!
Heat-shock the cells by placing into the waterbath for 30 seconds (no longer than 1 min). Be careful to keep the lids of the tubes above the water level, and keep the ice close by.
Immediately transfer the tubes back to ice, and incubate on ice for 5 minutes. IN THE FRIDGE AS WELL
Add qs 940 µL of SOC media WITHOUT CHLORAMPHENICOL per tube, and incubate at 37°C for 1 hour 30min shaking at 200-300rpm.
Prepare the agar plates during this time: label them, and add sterile glass beads if using beads to spread the mixture. Put the agar plates at 37°C.
Tubes were centrifuged at 3500rpm for 4minutes and 850µL of supernatant was discarded. Bacteria were resuspended into the remaining culture media and bacteria were spread in the corresponding petri dishes.
One petri dish with chloramphenicol and one petri dish without antibiotics was hence incubated.
Important notes:
The petri dishes we wanted to use were dried and we noticed that during the SOC incubation. We hence made new dishes quickly but the cultures had to stay in the SOC about 2h longer than usual. This is supposingly not impacting the cultures.
We however incubated 100µl of each culture (without centrifugation) on the dried plate to ensure security.
10/18 - Heat shock characterisation - Day 1
Objective: To measure mRFP expression by seq9-transformed bacteria depending on the temperature at 27°C and 37°C.
Protocol:
We prepared:
One culture of 2ml of seq 8 from the 10/17 liquid pre-culture in 100ml LB medium + chloramphenicol
One culture of 2ml of seq 9 from the 10/17 liquid pre-culture  in 100ml LB medium + chloramphenicol.
One culture of 2ml of WT DH5-alpha from the 10/14 liquid culture in 50ml LB medium + ampicilin
We start the incubation at T=0 (14h15) and every 1h30 we measured the OD600 until 0D600=0,4.
OD600 at t=0h Seq8: 0,058 Seq9: 0,040 WT: 0,062
OD600 at t=1h30 Seq8: 0,075 Seq9: 0,047
WT: 0,113
OD600 at t=3h Seq8: 0,190 Seq9: 0,054 WT: 0,344
We stopped the experiment at this point because of several technical problems. (Multitron doesn't accept erlens > 500ml and seq 9 wasn't growing correctly). We decided to launch a second time the experiment the 10/19 and we let the bacteria O/N at 37°C 250RPM.
10/19 - Bacterial Transformation with pSB1C3-seq2
We start a transformation using competent cells from 10/18 and miniprep of seq2-PSB1C3 from 30/08.
Transformation of competent cells with pSB1C3-Seq2:
Pipet 5µl of DNA from the then 50 µL of competent cells into each tube. Flick the tube gently with your finger to mix.
We also did a blank with 50µl of competent cells and 1µl of water.
Incubate on ice for 30 minutes. Put it in the fridge, the temperatures in the room are too high.
Pre-heat waterbath now to 42°C. Otherwise, hot water and an accurate thermometer works, too!
Heat-shock the cells by placing into the waterbath for 30 seconds (no longer than 1 min). Be careful to keep the lids of the tubes above the water level, and keep the ice close by.
Immediately transfer the tubes back to ice, and incubate on ice for 5 minutes. IN THE FRIDGE AS WELL
Add qs 940 µL of SOC media WITHOUT CHLORAMPHENICOL per tube, and incubate at 37°C for 1 hour 30min shaking at 200-300rpm.
Prepare the agar plates during this time: label them, and add sterile glass beads if using beads to spread the mixture. Put the agar plates at 37°C.
Tubes were centrifuged at 3500rpm for 4minutes and 850µL of supernatant was discarded.
Bacteria were resuspended into the remaining culture media and bacteria were spread in the corresponding petri dishes.
One petri dish with chloramphenicol was incubated.
10/19 - Digestion and ligation of the miniprep seq 7-pSB1A3 and GFP-pSB1K3 with pSB1C3
Objective: To digest seq 7-pSB1A3 and GFP-pSB1A3 miniprep (10/16/17) and ligate it with pSB1C3-mRFP digested.
Part 1: Digestion
Material:
DNA to digest
EcoRI-HF (NEB: 20000U/mL)
PstI (NEB: 20000U/mL)
10X NEB Buffer 2.1
Ultrapure water
1,5mL Eppendorf tubes
Micropipettes + tips
Waterbath
Heat Block
Protocol:
We took the following samples:
Seq 7-pSB1A3 miniprep
Sep GFP-pSB1K3 miniprep
pSB1C3 miniprep
We prepared the mixes according to the following instructions:
Reagent
Quantity
Final concentration
EcoRI-HF (20000U/mL)
1 µL
80U/µL
PstI (20000U/mL)
1 µL
80U/µL
plasmid
5 µL
10ng/µL
10X NEB buffer 2.1
2,5 µL
1X
Ultrapure water
15,5 µL
Digestion was carried out 3 hours at 37°C
Heat inactivation was proceeded at 80°C for 20 seconds.
Part 2: Ligations
Material:
Digested inserts (see previous digestion)
Digested backbone (see previous digestion)
10X T4 DNA Ligase Buffer (NEB)
T4 DNA ligase (400000U/mL)
Ultrapure water
Eppendorf tubes
Micropipettes + tips
Protocol:
Ligate Seq 7-pSB1A3 digested with pSB1C3 digested
Ligate GFP-pSB1K3 digested with pSB1C3 digested
We prepared the mixes according to the following instructions:
Seq 7 1:5
Seq 7 1:5
insert
15,9 µL
15,9 µL
backbone
pSB1C3 (double digested, approx 25ng) 1,3µL
pSB1A3 (double digested, approx 25ng) 1,3 µL
10X T4 DNA Ligase Buffer
2 µL
2 µL
T4 DNA Ligase
1 µL
1 µL
dH2O
0 µL
0 µL
Total volume
20,2 µL
20,2 µL
The ligation was performed overnight at 16°C.
10/19 - Heat shock characterisation - Day 2
Objective: To measure the OD600 from the bacteria of 10/18.
Protocol:
WT: 0,695 Seq8: 0,709 Seq9: 0,709
We decided to dilute these culture to 1/10.
OD600 t=0: WT: 0,282 Seq8: 0,300 Seq9: 0,282
We then incubated bacteria at 37°C and we measured the OD600 every 1h30 (starting time: 11h50)
t=1h30 (13h20) WT: 0,527 Seq8: 0,514 Seq9:0,489
We splitted the cultures in two as in the cold-response protocol, 45ml of each at 27°C and 45ml of each at 37°C for approximately 24h. Hour of incubation: 14h10
10/20 - Bacterial transformation (seq 7, GFP, seq 8, seq 9 and M9 & N6 plate 1 2017)
Objective: To perform bacterial transformations of sequence 7 in pSB1C3, GFP in pSB1C3, sequence 9 in pSB1C3, in order to make them grow at 25ºC and sequence 8 as a control. We also took the brick (BBa_K608351) Prom-CI regressor -Term -PR prom - N6 plate 1 2017 and the brick (BBa_K516032)RBS-mRFP-Term - M9 plate 1. We will later do the folding and create a big brick.
Material:
Ligated plasmids (see previous ligation)
Competent cells (DH5-alpha)
LB medium liquid and solid
SOC medium
Chloramphenicol
Ice
Eppendorf tubes
Micropipettes and tips
Waterbath
Petri dishes
Glass beads
Incubator
Centrifuge
Protocol:
We follow the bacterial transformation protocol:
Total: 6 petri dishes.
Seq 7 in pSB1C3 → All the ligation product with 50 µl of competent cells.
GFP in pSB1C3 → All the ligation product with 50 µl of competent cells.
Seq 9 miniprep → 5µl with 50µl  of competent cells.
Seq 8 miniprep →5µl with 50µl  of competent cells.
M9 PLATE 1 2017→ 2µl with 50µl  of competent cells.
N6 PLATE 1 2017→ 2µl with 50µl  of competent cells.
10/21 - OD Measurements and kinetics with iGEM Bettencourt
Objective: We prepared on the 10/20 8 petri dishes for this experiment (proteins expressed are indicated) at the IGEM Bettencourt team’s lab (we did the transformation with 10/16 minipreps):
Seq 8 pSB1C3 (containing mRFP)
Seq 9 pSB1C3 (containing mRFP)
Seq 1 pSB1C3 (containing AmilCP)
Seq 2 pSB1C3 (containing AmilCP)
pHeat Tokyo 2016 pSB1C3 (containing GFP)
pCold Tokyo 2016 pSB1C3 (containing GFP)
Seq 7 pSB1A3  (containing AmilCP))
Constitutive GFP pSB1C3 (containing GFP)
Protocol:
We arrived at the iGEM Bettencourt HQ and they generously provided us 2 96 well microplates, LB medium with chloramphenicol/ampicillin/kanamycin, cones and pipets and permitted us to work in sterile conditions.
Part 1. Medium preparation
Prepare 15ml of LB medium with chloramphenicol
Prepare 3ml of LB medium with ampicillin
Prepare 3ml of LB medium with kanamycin
Part 2. Prepare the deposit plan (two plates)
Be careful to work in sterile conditions. Put 300 µl of LB + corresponding antibiotics in each well, then take a cone tip, prelevate colonies in each petri dishes and put them in the corresponding wells.
Part 3. Incubation in microplate readers (Model TECAN 2000)
We used two microplates readers of the same model for both plates.
One plate was incubated at 37°C, the other at 27°C.
OD Measured:
OD 600nm (cells)
OD 700nm (cells)
OD 588nm  (AmilCP)
FLUORESCENCE mRFP:  Excitation at OD584 nm and emitting at OD607 nm
FLUORESCENCE GFP:  Excitation at OD501 nm and emitting at OD511 nm
The readers were put to the following shaking conditions for 48 hours (2x24h cycles):
Amplitude: 3,5mm
Frequency: 198,4
Duration: 550
10/21 - PCR colony pSB1C3-seq7 + pSB1C3-GFP + mRFP protein generator
Objective: To select the colonies transformed with the plasmid pSB1C3-protegrin to further purify it and send it to GATC for sequencing.
Part 1: PCR colony
Material:
Forward primer (100µM) synthesized by IDT
Reverse primer (100µM) synthesized by IDT
Colonies to pick
10X Standard taq reaction Buffer (NEB)
10µM dNTPS (NEB)
Taq DNA polymerase (5000U/mL)
DNase/RNase free water
PCR tubes
Micropipettes and tips
Thermocycler
Protocol:
Colonies we picked up:
3 colonies seq7 (1070bp without VF2 and VR)
3 colonies GFP (962bp with prefix and suffix binding domains)
1 colonie M9 (905bp with prefix and suffix binding domains)
1 control pSB1C3-mRFP (1112bp with prefix and suffix binding domains)
Each chosen colony has been annotated on the petri dishes so we will be able to use them again for liquid culture later. They are now stocked in the fridge, protected by parafilm.
Cell membrane destruction:
In each PCR tube: put 8µL of ddH2O. Pick a colony (with a toothpick or a micropipette tip), add a stab of it into the PCR tube.
Incubate the tubes at 95°C for 5min.
Tube mRFP: put 2µL DNA template
Prepare the PCR mix:
Component
Volume
Final concentration
10X Standard Taq Reaction Buffer
15µL
1X
10mM dNTPs
3µL
200µM (1/50)
10µM Preffix Primer
1,5µL
1µM
10µM Suffix Primer
1,5µL
1µM
Taq DNA Polymerase
1,5µL
2,5 units/50µL PCR
Nuclease-free water
127,5µL
qsp 150µL
Put 17µL of mix in each colony tube (25-8=17)
Put 23µL in the mRFP tube (25-2=23)
PCR thermocycler program:
PCR Cycle
Repetition
Step
Time
Temperature
0
Initial denaturation
30 sec
95°C
30
Denaturation
30 sec
95°C
Annealing
1 min
60°C
DNA polymerization
1 min (30 sec/kb)
68°C
0
DNA polymerization
5 min
68°C
FINISH
∞
4°C
Part 2: Gel migration
Protocol:
We made an agarose gel according to the protocol (please refer to our "Protocols" section).
Preparation of the samples: 10µL sample + 2µL 6X loading buffer
Deposit plan:
Ladder (5µL)
mRFP protein generator
GFP 1
GFP 2
GFP 3
Seq7 1
Seq7 2
Seq7 3
Ladder 5µL
mRFP control
Results:
Interpretation:
We observed a band at 900bp for GFP, and a band between 1000 and 900bp for mRFP generator. These two colonies were correctly transformed.
We therefore miss the second Ladder, some bands for sequence 7 and for GFP. We do not distinguish clearly the mFRP control.
10/21 -  Miniprep and glycerol stock of seq2 (19/10 culture)
Objective: To ensure the stock of transformed bacteria with the sequenced plasmids.
Protocol:
We incubated 500µL of each culture with 500µL of 50% sterile glycerol solution in cryotubes at 80°C.
The same protocol as usual were used for Miniprep.
10/21 - Transformation of part BBa_K608351 (N6 plate 1 2017) and part BBa_K516032 (M9 plate 1 2017) - second try
Objective: To perform bacterial transformations of M9 and N6 to make them grow at 25ºC and seq 8 as a control.
N6 plate 1 2017: BBa_K608351 → Prom-CI repressor -Term -PR prom  
M9 plate 1: BBa_K516032 → RBS-mRFP-Term
Protocol:
We followed the bacterial transformation protocol:
We tried to put more of insert (5µL instead of 2µL):
M9 PLATE 1 2017--> 5µl with 50µl of competent cells
N6 PLATE 1 2017--> 5µl with 50µl of competent cells
Week 10
10/23 - PCR colony (second try) on pSB1C3-seq7
Objective: Try to isolate white colonies containing pSB1C3-seq7
Material:
10X Standard taq reaction Buffer (NEB)
10µM dNTPS (NEB)
Forward primer 100µM (synthesized by IDT)
Reverse primer 100µM (synthesized by IDT)
Taq DNA polymerase (5000U/mL)
DNAse/ RNAse free water
PCR tubes
Thermocycler
Micropipettes and tips
Protocol:
Part 1: PCR colony
3 colonies transformed with seq7 (1070bp without VF2 and VR)
Each chosen colony has been annotated on the petri dishes so we will be able to use them again for liquid culture later. They are now stocked in the fridge, protected by parafilm.
Cell membrane destruction:
In each PCR tube: put 8µL of ddH20. Pick a colony (with a toothpick or a micropipette tip), add a stab of it into the PCR tube.
Incubate the tubes at 95°C for 5min.
Primers pre-dilution: 1µL + 9µL dH2O
Taq pre-dilution: 1µL + 9µL dH2O
Tube mRFP: put 2µL DNA template
Prepare the PCR mix:
Component
Volume
Final concentration
10X Standard Taq Reaction Buffer
8µL
1X
10mM dNTPs
1,6µL
200µM (1/50)
10µM Preffix Primer
8µL
1µM
10µM Suffix Primer
8µL
1µM
Taq DNA Polymerase
8µL
2,5 units/50µL PCR
Nuclease-free water
46,4µL
qsp 80µL
Put 17µL of mix in each colony tube (25-8=17)
Put 23µL in the mRFP tube (25-2=23)
Cycles:
Put the PCR tubes in the thermocycler and run the following program:
PCR Cycle
Repetition
Step
Time
Temperature
0
Initial denaturation
30 sec
95°C
30
Denaturation
30 sec
95°C
Annealing
1 min
60°C
DNA polymerization
1 min
68°C
0
DNA polymerization
5 min
68°C
FINISH
∞
4°C
Part 2: Gel migration
We made an agarose gel according to the protocol (please refer to our "Protocols" section).
We will migrate the PCR colony product, and also the plasmids pSB1C3-seq2, pSB1C3-Protegrin, pSB1C3-GFP, and pSB1C3-pCold.
Preparation of samples for the migration: 10µL sample + 2µL 6X loading buffer, 10µL were loaded in each well.
Deposit plan:
Ladder
seq7 (4)
seq7 (5)
seq7 (6)
mRFP control
pSB1C3-pCold (10/12 miniprep)
pSB1C3-Seq2 (10/21 miniprep)
pSB1C3-GFP (10/23 miniprep)
pSB1C3-protegrin (10/16 miniprep) (diluted in 30µL: we added 15µL of dH2O)
Ladder
Gel migration was performed during 1h30 at 110V.
The gel was incubated in a GelRed bath during 1h30.
Results:
Interpretation:
We cultivated the colony 5 in liquid culture.
pCold: 1033+2070=3103bp -> middle band
seq2: 965+2070=3035 -> unexpected
GFP: 962+2070=3032 -> middle band
Prot: 418+2070=2488 -> middle band
10/23 - GATC sequencing
Objective: To prepare the tubes for sequencing by GATC.
Protocol:
Samples:
pSB1C3-Sequence 2 (10/21)
pSB1C3-pCold (10/12)
pSB1C3-protegrin (10/24)
DNA: 4µL miniprep + 16µL H2O
Primers (called TATApref and TATAsuf): 2µL primer + 18µL H2O
Results: (on 10/25)
pCOLD BBa_K1949000 (313bp):
1. a) Sequencing with Forward primer (TATAprefix)
The sequencing worked well (there is just a mutation on the 38th base).
1. b) Sequencing with Reverse primer (TATAsufix)
With the reverse primer: very bad results! Not usable.
Seq 2 BBa_K2282006:
2. a) Sequencing with Forward primer (TATAprefix)
Still nothing
The weird thing is that we have results on the electrophoresis gels but not by sequencing. So we know that we have something in our tubes.
Is it a problem with the primers? Maybe, but we can amplify it by PCR so they work.
2. b) Sequencing with Reverse primer (TATAsufix)
Still nothing
Prot-1 (353bp) BBa_K628006:
3. a) Sequencing with Forward primer (TATAprefix)
The sequencing worked well even if there are some "dirty" nucleotides and some mutations.
3. b) Sequencing with Reverse primer (TATAsufix)
Nothing.
It seems that we have a problem for sequencing with the reverse primer (TATAsufix). We observed difficulties also on the previous sequencing (29/09).
10/23 - Cold shock characterisation 3 - Test at 12°C - Day 0
Objective: We are going to try another temperature, colder this time, to see if the mechanism still works below 15°C as it is the temperature recommended by TAKARA and mentioned in bibliography.
Protocol:
Day 0:
We incubated 100µl of glycerol stock from 10/16 of the transformed cultures with seq 1/2/7 in 5ml of LB with corresponding antibiotics.
We also did a pre-culture of WT DHalpha from iGEM IONIS 2016 stocked at -80°C to have a control
This will ensure a better growth of our bacteria for the OD600 measurement on the 10/24, as we waited around 9h last time.
10/24 - Cold shock characterisation 3 - Test at 12°C - Day 1
Protocol:
For this day we took the pre-culture made the 10/23.
We set the Multitron at 12°C to let it cool down.
We prepared four erlenmeyers:
Wild-type (LB without antibiotics)
Seq 1 (LB chloramphenicol)
Seq 2 (LB chloramphenicol)
Seq 7 (LB Ampicillin)
We put 50 ml of LB with corresponding antibiotics in each erlenmeyers, and incubated 4ml of each pre-culture (10/23) in their respective erlenmeyers.
We then measured the OD600 at = 0h: Seq 7: 0,110 WT: 0,099 Seq 2: 0,087 Seq 1: O,070
We put then the erlenmeyers in the 37°C incubator at 200 rpm.
OD600 at = 1h30: Seq 7: 0,312 WT: 0,331 Seq 2: 0,166 Seq 1: 0,232
At this point we assessed to remake a measure of seq 7 and WT cultures and let both seq 1 and 2 incubate a little longer
OD600 at = 2h15: Seq 7: 0,436 WT: 0,488
We are fine ! We splitted the cultures in two erlenmeyers (2x20ml approximately), put one at 12°C and the other at 37°C.
Time of incubation: 12h35
OD600 at = 2h15: Seq 1: 0,408 Seq 2: 0,558
The surprisingly quick growth of the seq 2 culture nearly caught us by surprise but we managed to start the incubation during the right phase for both. We splitted the cultures as well in two erlenmeyers of approximately 20ml.
Time of incubation: 13h30
We then left our cultures until the next day for 20-24h.
10/25 - Cold shock characterisation 3 - Test at 12°C - Day 2
After 20h of incubation at their respective temperatures, we took all the erlenmeyers and did OD600 measurements:
OD600 at t=20h after incubation at 37°c and 15°C:
Seq 1 at 12°C:0,471
Seq 1 at 37°C: 0,703
Seq 2 at 12°C: 0,580
Seq 2 at 37°C: 0,688
Seq 7 at 12°C: 0,489
Seq 7 at 37°C: 0,687
WT at 12°C: 0,503
WT at 37°C: 0,685
The OD600 are all consistent and show well that no or few E.coli growth occurs at 12°C, proving the interest of the pre-incubation until the OD600 of 0,5 is reached.
As for the expression of AmilCP:
A. Seq 1 at 12°C and 37°C after 20h: The control worked well, AmilCP is expressed in both conditions, we can clearly see that the pellet from the 37°C culture is larger and displays more color that the one at 12°C, showing better growth and expression of the protein.
B. Seq 2 at 12°C and 37°C after 20h: This results highlights even more that, even though the DSBox does not modify the characteristic of our protein, it might alter its expression overall at both temperatures. Indeed, if we compare the two pellets with the ones from the seq 1, we can clearly see that they are both displaying lighter colors at both temperatures, showing lesser AmilCP expression. This adds up to the other experiment and help us to show the effect of the DSBox in the protein sequence. Note that this results could be compromised by the fact that our seq 2 might have a mutation or unwished modifications , therefore this results is to be taken with care.
C. Seq 7 at 12°C and 37°C after 20h: As with the 15°C incubation, it worked again. We can clearly see no or very few expression of AmilCP (the pellet remains a little gray) at 37°C as compared with 15°C, showing that our construction works too at 12°C. Comparing the results with the control shows well that the system works. The expression of AmilCP at 12°c is however lower (less color) with the seq 7 than with the seq 1. This could be due to the DSBox. What could be interesting is to test another sequence similar to the seq 7 but without the DSBox.
D. WT DHalpha at 12°C and 37°C after 20h: Nothing special to notice here. This negative control shows, with the pellet size and OD600 measurements, that E.Coli doesn't grow or grow very slowly at 12°C as compared with 37°C. We wanted to add this control as compared with the previous tests because we thought it was necessary.
Overall this experiment shows again that our system works and permitted us to characterise the seq 7 and seq 2 a little more. We want now to test the system at 20°C. By time constraints we won't do a duplicate this time for now but we might have the time before the wiki freeze or the Jamboree. We are also not going to do a measurement at 40h for the same reasons.
10/25 - Cold shock characterisation 4 - Test at 12°C - Day 0
For this experiment, we took 1ml of each 37°C culture from the Cold-shock characterisation 3 at 12°C (10/23-10/25) and diluted them in 3ml of LB with corresponding antibiotics. This serves the same purpose as the Day 0 from the Cold-shock characterisation 3 and will permit us to launch quickly the experiment on the 10/26 for the OD600 0,5 measurement.
We also washed all the erlenmeyers and autoclaved them.
10/25 - Preparation of iGEM plate for registry
We followed the guidelines in the following link.
We measured the concentration of the DNA we want to send with the help of our advisor Nicolas Cornille who used his Nanodrop.
We prepared the samples for him by sending 3µl of each sequence in PCR tubes.
Here are his measurements:
BBa_K2282005 (seq 1) = 47ng/µl
BBa_K2282006 (seq 2) = 51ng/µl
BBa_K2282011 (seq 7) = 33ng/µl
BBa_K2282012 (seq 8) = 322ng/µl
BBa_K2282005 (seq 9) = 198 ng/µl
BBa_I20260 (GFP) = 53ng/µl
BBa_K628006 (Protegrin-1) = 20ng/µl
BBa_K1949000 (pCOLD) = 57ng/ µl
The advised concentration by the iGEM HQ is 25ng/µl, hence we put in our wells:
1A: 5,4µl seq 1
1B: 5µl seq 2
1C: 7,6µl seq 7
1D: 0,8 µl seq 8 (for this one we diluted 2µl into 18µl and took 7,8µl from the obtained solution)
1E: 1,3µl seq 9
1F: 4,8µl GFP
1G: 12,5µl Protegrin-1
1H: 57 ng/µl pCOLD Tokyo 2016
We left the plate to dry under the BSC and followed the protocol.
10/25 - New PCR Seq2
Objective: To amplify sequence 2 once again as we suspect the appearance of mutations (925bp expected).
Part 1: PCR
Material:
10X Standard taq reaction Buffer (NEB)
10µM dNTPS (NEB)
Forward primer 100µM (synthesized by IDT)
Reverse primer 100µM (synthesized by IDT)
Taq DNA polymerase (5000U/mL)
DNAse/ RNAse free water
PCR tubes
Thermocycler
Micropipettes and tips
Protocol:
We divided our 50µL mix into two 25ul mixes. The PCR is then performed with only 18 cycles to reduce the risk of errors (Taq polymerase not very accurate). Finally the two PCR products are mixed together before the clean up.
Pre-dilution Primers 100µM: 1 µL primer + 9 µL dH2O → 10µM final, volume total final (10µL → we need 5µL)
Pre-dilution Taq Polymerase 5000U/mL: 1µL Taq + 9µL dH2O → 500U/mL final, volume total final (10µL → we need 5µL)
For a 50µL PCR mix volume:
Component
Volume
Final concentration
10X Standard Taq Reaction Buffer
5µL
1X(1/10)
10mM dNTPs
1µL
200µM (1/50)
10µM Preffix Primer
5µL
1µM
10µM Suffix Primer
5µL
1µM
Taq DNA Polymerase
5µL
2,5 units/50µL PCR
DNA template (10ng/µL)
2µL
Nuclease-free water
27µL
qsp 50µL
PCR thermocycler program:
PCR Cycle
Repetition
Step
Time
Temperature
0
Initial denaturation
30 sec
95°C
30
Denaturation
30 sec
95°C
Annealing
1 min
60°C
DNA polymerization
1 min(30 sec/kb)
68°C
0
DNA polymerization
5 min
68°C
FINISH
∞
4°C
Part 2: Gel Migration
We are going to use 2/4µL of the PCR products to quantify the sequences concentrations efficiently. We made an agarose gel according to the protocol (please refer to our "Protocols" section).
Mixes for deposit: 7µL PCR product + 1,4µL 6X loading dye → 8,4µL total
2µl DNA: 2,4µl of the mix
4µl DNA: 4,8µl of the mix
Deposit plan:
Ladder (5µL)
2µL DNA
4µL DNA
Gel migration was performed during 1h30 at 110V.
The gel was incubated in a GelRed bath during 1h30.
Results:
Interpretation:
Thanks to the gel, we estimated quantities:
40ng in 4µL
430ng in 43µL
After a PCR cleanup → 322,5 in 30µL
The concentration is then 10,75 ng/µL
10/26 - Cold shock characterisation 4 - Test at 12°C - Day 1
We took off the pre-cultures and incubated 1ml of each in erlenmeyers containing 50ml of corresponding antibiotics.
OD600 was measured at t = 0h: Seq 1: 0,048 Seq 2: 0,04 Seq 7: 0,047 WT: 0,048
OD600 at t = 3h30 (we waited this much because we judged making an OD measurement at t = 1h30 with such low starting value was useless). Seq 1: 0,160 Seq 2: 0,248 Seq 7: 0,375 WT: 0,492
The WT grew apparently really fast so we had to split the culture in two erlenmeyers (2 x 20ml) and put one at 20°C and the other at 37°C.
OD600 at t = 4h15: we measured only the seq 7 OD600 because the seq 1 and seq 2 had low OD600 value at t = 3h30. Seq 7: 0,490
For the sequence 7, we splitted the culture into two erlenmeyers (2x20ml) and incubated one at 20°C and the other at 37°C.
OD600 at t = 5h: we measured only the seq 2 OD600 for the same reason as before. The seq 1 OD600 value was judged too low to make any measurement at this time relevant. Seq 2: 0,457
For the sequence 2, we splitted the culture into two erlenmeyers (2x20ml) and incubated one at 20°C and the other at 37°C. Note that the OD600 value is a little lower than the others.
The most important point in this experiment is the fact that E.coli is in its exponential growth phase, hence the OD600 of 0,457 was validated.
OD600 at t = 6h: Seq 1: 0,355
OD600 at t = 6h40: Seq 1: 0,399
OD600 at t = 7h10: Seq 1: 0,437
For the sequence 1, we splitted the culture into two erlenmeyers (2x20ml) and incubated one at 20°C and the other at 37°C. Note that the OD600 value is a little lower than the others.
The most important point in this experiment is the fact that E.Coli is in its exponential growth phase, hence the OD600 of 0,437 was validated.
The culture were then left for 20-24h at their respective temperatures for the colorimetric measurement the following day.
10/26 - sequence 2: PCR clean-up, digestion and ligation in pSB1C3 backbone
Objective: To perform the PCR clean-up of the seq 2, for further uses (miniprep - sequencing GATC).
Protocol:
Part 1: Miniprep
Follow the QUIAGEN protocole, and we resuspended in 30µL for final volume.
Part 2: Digestion
We prepared the mixes according to the following instructions:
Reagents
Quantity
Final concentration
EcoRI-HF (20000U/mL)
1µL
80U/µL
PstI (20000U/mL)
1µL
80U/µL
DNA
5µL
10ng/µL
10X NEB buffer 2.1
2,5µL
1X
Ultrapure water
15,5µL
Digestion was carried out 2 hours at 37°C
Heat inactivation was proceeded at 80°C for 20 minutes.
Part 3: Ligation
We ligate sequence 2 with the pSB1C3 backbone; prepare the mixe of ligation as follow:
1:5 ratio
insert (seq 2)
15,9 µL
backbone (pSB1C3)
pSB1C3 (double digested, approximately 25ng) 1,3µL
10X T4 DNA Ligase Buffer
2 µL
T4 DNA Ligase
1 µL
dH2O
0 µL
Total volume
20,2 µL
Incubate overnight at 16°C.
10/27 - Cold shock characterisation 4 - Test at 20°C - Day 2
Protocol and Results:
OD600 at 20h:
Seq 1 - 20°C: 0.666
Seq 1 - 37°C: 0.705
Seq 2 - 20°C: 0.676
Seq 2 - 37°C: 0.689
Seq 7 - 20°C: 0.679
Seq 7 - 37°C: 0.685
Seq WT - 20°C: 0.686
Seq WT - 37°C: 0.708
All the OD are approximately at the same level, which shows that our bacteria are all at the same step of the growth.
A. We do not have any color here, as it is the WT sample; This is our control.
B. At 20°C, we have a  very slightly lighter color that at 37°C. This is quite normal, as the seq 1 is constitutive (so always expressed). There are less produced proteins when lower temperature.
C. At 20°C, we have a lighter color that at 37°C. This is quite normal, as the seq 2 is constitutive (so always expressed). There are less produced proteins when lower temperature.
D. Here, we can see that at 20°C, we have a blue color, although there is no color in the 37°C tube. This is what we expected for the 37°C tube: there is no cold response at 37°C.
10/27 - Transformation 2
Objective: To transform E.coli with the ligation product pSB1C3-seq2 from 10/26.
Protocol:
We used the competent cells made on the 10/18.
Clean your working area by wiping down with 70% ethanol.
Thaw competent cells on ice. Label one 1.5 mL microcentrifuge tubes for each transformation and then pre-chill by placing the tubes on ice.
Tubes
DNA
Plate
Competent cells
SOC medium
2
20 µL
LB + Chloramphenicol
50 µL
930 µL
Pipet DNA in each tube and then 50 µL of competent cells into each tube. Flick the tube gently with your finger to mix.
Incubate on ice for 30 minutes. Put it in the fridge, the temperatures in the room are too high.
Pre-heat waterbath now to 42°C. Otherwise, hot water and an accurate thermometer works, too!
Heat-shock the cells by placing into the waterbath for 30 seconds (no longer than 1 min). Be careful to keep the lids of the tubes above the water level, and keep the ice close by.
Immediately transfer the tubes back to ice, and incubate on ice for 5 minutes. IN THE FRIDGE AS WELL
Add qs 930 µL of SOC media WITHOUT CHLORAMPHENICOL per tube, and incubate at 37°C for 1 hour 30min shaking at 200-300rpm.
Prepare the agar plates during this time: label them, and add sterile glass beads if using beads to spread the mixture. Put the agar plates at 37°C.
Tubes were centrifuged at 3500rpm for 4minutes and 850µL of supernatant was discarded. Bacteria were resuspended into the remaining culture media and bacteria were spread in the corresponding petri dishes.
At the same time, we incubated cultures from the 10/14, frozen cells containing the pSB1C3-seq2 plasmid, on agar plates.
Week 11
10/31 - Cold-shock characterization at 37°C & 27°C
Objective: Assessing AmilCP expression at 37°C and 27°C of WT (DH5alpha -80°C), seq1 (Transformation from 27/10), seq2 (Transformation from 27/10) and seq7 (-80°C).
Protocol and Results:
Pre-cultures started on 10/30 morning at 37°C in 50mL of LB liquid medium added with chloramphenicol (seq 1 & 2) or ampicillin (seq 7, 300µL)
OD600 at t = 0
Seq 7: 0,018
WT: 0,015
Seq 1: 0,098
Seq 2: 0,015
OD600 at t = 3h
Seq 7: 0,060
WT: 0,049
Seq 1: 0,108
Seq 2: 0,034
OD600 at t = 6h
Seq 7: 0,408 -> put 20mL at 27°C
WT: 0,034
Seq 1: 0,172
Seq 2: 0,074
OD600 at t = 8h
WT: 0,173 (we then added supplementary bacteria)
Seq 1: 0,302
Seq 2: 0,278
OD600 at t = 9h
WT: 0,288
Seq 1: 0,457 -> put 20mL at 27°C
Seq 2: 0,389
OD600 at t = 9h30
WT: 0,391
Seq 2: 0,478 -> put 20mL at 27°C
OD600 at t = 10h
WT: 0,422 -> put 20mL at 27°C
OD600 at t = +20h
WT 37°C: 0,693
WT 27°C: 0,680
Seq 1 37°C: 0,706
Seq 1 27°C: 0,686
Seq 2 37°C: 0,694
Seq 2 27°C: 0,680
Seq 7 37°C: 0,693
Seq 7 27°C: 0,681
Igem ionis
Is an association created by Sup’Biotech student in 2015. Since this first participation, two teams (2015 and 2016) won the gold medal and several nominations: « Best presentation », « Best applied design », and « Best environmental project ». The strength of the IGEM IONIS comes from its multidisciplinarity and its complementarity.
This year we are 20 members from different schools:18 students from Sup’Biotech1 student from e-artsup1 student from Epita Read more …
Follow us
keep IN TOUCH
Location: 66 Rue Guy Môquet94800 Villejuif, France
Email: igem.ionis@gmail.com
