<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_IONIS-PARIS_Modeling_rnafolding skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:IONIS-PARIS/Modeling/rnafolding</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#menu-container" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="menu-container"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle menu-item" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">PROJECT</A></LI><LI><A href="https://2017.igem.org/Team:IONIS-PARIS/project/description">DESCRIPTION</A></LI><LI><A href="https://2017.igem.org/Team:IONIS-PARIS/project/definition">DEFINITIONS</A></LI><LI><A href="https://2017.igem.org/Team:IONIS-PARIS/project/brainstorming_process">BRAINSTORMING PROCESS</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle menu-item" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">MODELING</A></LI><LI><A href="https://2017.igem.org/Team:IONIS-PARIS/Modeling/dsbox-amilcp">DSBOX + AmilCP</A></LI><LI><A href="https://2017.igem.org/Team:IONIS-PARIS/Modeling/rnafolding">RNA FOLDING SIMULATION</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle menu-item" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">LABORATORY WORK</A></LI><LI><A href="https://2017.igem.org/Team:IONIS-PARIS/labwork/parts">PARTS</A></LI><LI><A href="https://2017.igem.org/Team:IONIS-PARIS/labwork/protocols">PROTOCOLS</A></LI><LI><A href="https://2017.igem.org/Team:IONIS-PARIS/labwork/results">RESULTS &amp; CHARACTERISATIONS</A></LI><LI><A href="https://2017.igem.org/Team:IONIS-PARIS/labwork/lapaillasse">WORKING AT LA PAILLASSE</A></LI><LI><A href="https://2017.igem.org/Team:IONIS-PARIS/labwork/labbook">LABORATORY NOTEBOOK</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle menu-item" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">APPLIED DESIGN</A></LI><LI><A href="https://2017.igem.org/Team:IONIS-PARIS/applied-design/overview">OVERVIEW</A></LI><LI><A href="https://2017.igem.org/Team:IONIS-PARIS/applied-design/foliar-application">FOLIAR APPLICATION</A></LI><LI><A href="https://2017.igem.org/Team:IONIS-PARIS/applied-design/chassis-selection">CHASSIS SELECTION</A></LI><LI><A href="https://2017.igem.org/Team:IONIS-PARIS/applied-design/protective_compound">PROTECTIVE COMPOUNDS</A></LI><LI><A href="https://2017.igem.org/Team:IONIS-PARIS/applied-design/risk-avoidance">RISK AVOIDANCE: BIOSAFETY STRATEGIES</A></LI><LI><A href="https://2017.igem.org/Team:IONIS-PARIS/applied-design/risk-assessment">RISK ASSESSMENT: TOXICITY &amp; ECOTOXICITY STUDIES</A></LI><LI><A href="https://2017.igem.org/Team:IONIS-PARIS/applied-design/frostban">CASE STUDY: FROSTBAN</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle menu-item" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false" data-target="#human">HUMAN PRACTICES</A></LI><LI><A style="padding: 35px 35px" href="https://2017.igem.org/Team:IONIS-PARIS/hp/overview">OVERVIEW</A></LI><LI><A style="padding: 35px 35px" href="https://2017.igem.org/Team:IONIS-PARIS/hp/public-outreach">PUBLIC OUTREACH</A></LI><LI><A style="padding: 35px 35px" href="https://2017.igem.org/Team:IONIS-PARIS/hp/society-law-gmos">SOCIETY LAW &amp; GMOs</A></LI><LI><A style="padding: 35px 35px" href="https://2017.igem.org/Team:IONIS-PARIS/hp/perspectives">PERSPECTIVES</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle menu-item" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">ENTREPRENEURSHIP</A></LI><LI><A href="https://2017.igem.org/Team:IONIS-PARIS/Entrepreneurship/overview">OVERVIEW</A></LI><LI><A href="https://2017.igem.org/Team:IONIS-PARIS/Entrepreneurship/market_study">MARKET STUDY</A></LI><LI><A href="https://2017.igem.org/Team:IONIS-PARIS/Entrepreneurship/fundraising">FUNDRAISING &amp; COMMUNICATION</A></LI><LI><A href="https://2017.igem.org/Team:IONIS-PARIS/Entrepreneurship/sponsors">SPONSORS</A></LI><LI><A href="https://2017.igem.org/Team:IONIS-PARIS/Entrepreneurship/perspectives">PERSPECTIVES</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle menu-item" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">TEAM</A></LI><LI><A href="https://2017.igem.org/Team:IONIS-PARIS/Team/attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2017.igem.org/Team:IONIS-PARIS/Team/collaboration">COLLABORATIONS</A></LI><LI><A href="https://2017.igem.org/Team:IONIS-PARIS/Team/team">MEMBERS</A></LI></DIV></DIV><FIGURE class="retour" style="z-index: 99"><FIGCAPTION id="retour" class="retour"><B>Back to top</B></FIGCAPTION></FIGURE><H1 class="centered">
    Structure-function characterization of the cspA 5'UTR through molecular modeling
    </H1><DIV class="igem-container"><DIV class="row"><DIV class="col-md-12"><DIV class="colored-container"><P><B>Overview</B>:</P><LI>We used a computanional approach to study the folding and the conformation of our RNA to understand and bring an explanation of its mechanism.</LI><LI>We identified the more stable and realistic structures based on computational analysis</LI><LI>In complement to the folding simulation, we used RNA-protein docking to study the interaction with a ribosomal protein.</LI></DIV><H2>Introduction</H2><P>In this project we developed a genetic sequence that allows bacteria to produce amilCP when it undergoes cold shock. To achieve this, we chose to use the cspA promoter that is activated during cold shock in several bacteria such as wild type (WT) Escherichia coli. Also, based on the literature we choose to include the UP element of the cspA promoter, the cspA 5’UTR, and the DownStream Box (DSBox) in our final construction. These temperature sensing sequence motifs are upstream of our reporter protein (amilCP), with the exception of the DSBox (see Fig.1). Our bibliographic review indicates that these components are required to see the conditional expression of amilCP only at low temperature<SUP>1,2</SUP> (under 15°C, approximately).</P><FIGURE><FIGCAPTION class="centered marged">Figure 1: Schema of sequence 7 construction.</FIGCAPTION></FIGURE><P>We found in the bibliography that the current main hypothesis that could explain the expression of the cold shock protein family is the structure of the mRNA. At 37°C their mRNA is quickly degraded. As the temperature decreases the mRNA assumes a different structure. This leads to the stabilization of the mRNA structure at low temperatures.
          

          There were no experimental structures solved for the cspA 5’UTR, nor for any close sequence homologs. Therefore, we used a computational method, SimRNA<SUP>3</SUP>, in order to generate 3D models of this region of the cspA mRNA.
          

          Our results suggest that several short distinct sequence motifs classically identified as important for ribosome binding actually participate together in the formation of a secondary structure motif that is recognized in a generally non base-pair specific manner by ribosomal accessory proteins in the formation of the pre-initiation complex at low temperatures.
          </P><H2>Materials and Methods</H2><H3 style="text-decoration: underline;">Software</H3><P>SimRNA models RNA with a coarse grained 3D model subjected to a statistical potential derived from a set of experimentally determined RNA structures, and  uses a Monte Carlo method for conformational sampling.
            

            For our first simulations we decided to try the isotemperature method. It was supposed to show the folding of RNA at a fixed temperature. Unfortunately, initial results were quite variable in their final structures. The simulations begin with RNA in a planar circle conformation. At each iteration of the Monte Carlo procedure, it calculates the lowest energy conformation for the RNA. The problem is that the simulation can get stuck in a local energy minimum, and as the simulation runs it becomes unable to escape it. Thus, we can’t see if there is another local minimum deeper than the one where our simulation was stuck. This is why we decided to switch to the Replica Exchange Monte Carlo (REMC) protocol for our simulations.
            

            Replica-Exchange Monte Carlo is a technique for enhancing the conformational sampling of systems by Monte Carlo methods. It requires the creation of several replica simulations of the same system, with each replica running at a different simulated temperature. Conformations that are close in energy are exchanged periodically between each replica, allowing the detection and exploration of the principal local minima on the system’s potential energy surface. This corresponds to the discovery of new conformations separated from initial conditions by energy barriers.
            </P><H3 style="text-decoration: underline;">Simulation parameters</H3><P>In our simulations, the cspA 5’UTR sequence comes from the registered part <A href="http://partsregistry.org/Part:BBa_K328001" style="color:#1155CC">BBa_K328001:</A></P><P class="wrap marged">ACGGUUUGACGUACAGACCAUUAAAGCAGUGUAGUAAGGCAAGUCCCUUCAAGAGUUAUCGUUGAUACCCCUCGUAGUGCACAUUCUUUAACGCUUCAAAAUCUGUAAAGCACGCCAUAUCGCCGAAAGGCACACUUAAUUAUUAAAGGUAAUACACUAUGAGUAUGACUGGUAUCGUAAAUGACU</P><P>The REMC
                  simulations were made of eight replicas, running for 16 000 000 steps in total.
                  One frame was recorded every 320 steps, which correspond to a total of 50 000 frames
                  per replica. Each frame is a PDB file of one particular step of the folding
                  process.</P><P>The initial
                    temperature of the annealing process was set to 1.35 and the final temperature
                    to 0.9, (per the published SimRNA benchmark settings), recalling that these
                    values are dimensionless (they are the Lennard Jones reduced temperature unit).</P><P>Once the simulation
                      is done, SimRNA generates one trajectory file per replica. To obtain the
                      representative structures of the simulation, we have done a clustering analysis.
                      SimRNA has a built in clustering routine. We choose to take the top 1% of the
                      lowest energy structures and separated them with a threshold of 18.7 Ångström
                      (per the published clustering guideline of allowing 0.1 Ångström/nucleotide).</P><H3 style="text-decoration: underline;">Hardware</H3><P>To run our
                        simulation we used a personal gaming computer running Linux 16.04 LTS.</P><P>Hardware
                          Specifications:</P><P style="margin-left:36.0pt;line-height:normal"><U>CPU</U>: Intel Core i7 6700K @ 4.00GHz</P><P style="margin-left:36.0pt;line-height:normal"><U>RAM</U>: 16,0 Go Dual-Channel DDR4</P><P style="margin-left:36.0pt;line-height:normal"><U>Motherboard</U>: ASUSTeK COMPUTER INC. Z170 PRO GAMING (LGA1151)</P><P style="margin-left:36.0pt;line-height:normal"><U>Graphics</U>: NVIDIA GeForce GTX 1070 - 8Go</P><CENTER><P><B>We know that it could be hard to launch yourself into this kind of work. To help you with that, we wrote a handbook to introduce you to the RNA simulation. We hope it will help all the teams to get involve with the computational approach.</B></P><A class="cacahuete" role="button" target="_blank" href="https://static.igem.org/mediawiki/2017/4/4b/Ionis-Paris-2017-Modelling-handbook.pdf">Download Modeling Handbook</A></CENTER><H2>Results</H2><P>We chose to take
                            the top 1% of the lowest energy structures and separated them with a threshold
                            of 18.7 Ångström (per the published clustering guideline of allowing 0.1
                            Ångström/nucleotide).</P><FIGURE><FIGCAPTION class="centered marged">Figure 2: Percentage of structures in each cluster.</FIGCAPTION></FIGURE><P>For our simulation
                              we choose to work with the top 6 clusters (out of 25 identified in total) as
                              they represent more than 90% of the clustered structures, as displayed in the
                              clustering log file</P><P class="centered marged"><B><U>3D representative structures viewer</U></B>:</P><P>With the REMC
                                simulation, clustering process, and the use of GROMACS<SUP>5</SUP> to analyze
                                the trajectory file we obtained the 3D model of each lowest energy structure of
                                each cluster and also the energy value.</P><FIGURE><FIGCAPTION class="centered marged">Figure 3: representative structures of
                                    each cluster with the DS box in yellow.</FIGCAPTION></FIGURE><TABLE class="centered" border="1" cellspacing="0" cellpadding="0" width="392" style="border-collapse:
                                  collapse;border:none"><TBODY><TR><TD style="border:solid black 1.0pt;padding:5.0pt 5.0pt 5.0pt 5.0pt"><P class="MsoNormal" align="center" style="text-align:center;line-height:normal">Cluster number</P></TD><TD valign="top" style="border:solid black 1.0pt;border-left:none;padding:5.0pt 5.0pt 5.0pt 5.0pt"><P class="MsoNormal" align="center" style="text-align:center;line-height:normal">Energy of the lowest energy
                                        structures of each cluster in SimRna energy units</P></TD></TR><TR><TD valign="top" style="border:solid black 1.0pt;border-top:none;padding:5.0pt 5.0pt 5.0pt 5.0pt"><P class="MsoNormal" align="center" style="text-align:center;line-height:normal">1</P></TD><TD valign="top" style="border-top:none;border-left:none;border-bottom:solid black 1.0pt;
                                      border-right:solid black 1.0pt;padding:5.0pt 5.0pt 5.0pt 5.0pt"><P class="MsoNormal" align="center" style="text-align:center;line-height:normal">-2043</P></TD></TR><TR><TD valign="top" style="border:solid black 1.0pt;border-top:none;padding:5.0pt 5.0pt 5.0pt 5.0pt"><P class="MsoNormal" align="center" style="text-align:center;line-height:normal">2</P></TD><TD valign="top" style="border-top:none;border-left:none;border-bottom:solid black 1.0pt;
                                    border-right:solid black 1.0pt;padding:5.0pt 5.0pt 5.0pt 5.0pt"><P class="MsoNormal" align="center" style="text-align:center;line-height:normal">-2073</P></TD></TR><TR><TD valign="top" style="border:solid black 1.0pt;border-top:none;padding:5.0pt 5.0pt 5.0pt 5.0pt"><P class="MsoNormal" align="center" style="text-align:center;line-height:normal">3</P></TD><TD valign="top" style="border-top:none;border-left:none;border-bottom:solid black 1.0pt;
                                  border-right:solid black 1.0pt;padding:5.0pt 5.0pt 5.0pt 5.0pt"><P class="MsoNormal" align="center" style="text-align:center;line-height:normal">-2048</P></TD></TR><TR><TD valign="top" style="border:solid black 1.0pt;border-top:none;padding:5.0pt 5.0pt 5.0pt 5.0pt"><P class="MsoNormal" align="center" style="text-align:center;line-height:normal">4</P></TD><TD valign="top" style="border-top:none;border-left:none;border-bottom:solid black 1.0pt;
                                border-right:solid black 1.0pt;padding:5.0pt 5.0pt 5.0pt 5.0pt"><P class="MsoNormal" align="center" style="text-align:center;line-height:normal">-2046</P></TD></TR><TR><TD valign="top" style="border:solid black 1.0pt;border-top:none;padding:5.0pt 5.0pt 5.0pt 5.0pt"><P class="MsoNormal" align="center" style="text-align:center;line-height:normal">5</P></TD><TD valign="top" style="border-top:none;border-left:none;border-bottom:solid black 1.0pt;
                              border-right:solid black 1.0pt;padding:5.0pt 5.0pt 5.0pt 5.0pt"><P class="MsoNormal" align="center" style="text-align:center;line-height:normal">-2034</P></TD></TR><TR style="height:24.0pt"><TD valign="top" style="border:solid black 1.0pt;border-top:none;padding:5.0pt 5.0pt 5.0pt 5.0pt;
                            height:24.0pt"><P class="MsoNormal" align="center" style="text-align:center;line-height:normal">6</P></TD><TD valign="top" style="border-top:none;border-left:none;border-bottom:solid black 1.0pt;
                          border-right:solid black 1.0pt;padding:5.0pt 5.0pt 5.0pt 5.0pt;height:24.0pt"><P class="MsoNormal" align="center" style="text-align:center;line-height:normal">-2032</P></TD></TR></TBODY></TABLE><FIGCAPTION class="centered marged">Table 1: Energy value of lowest
                      energy frame of each cluster in SimRNA energy units</FIGCAPTION><FIGURE><FIGCAPTION class="centered marged">Figure 4: REMC cloud of the cspA 5’UTR simulation. 40 000 data points were plot with
                        GNUplot.
                      </FIGCAPTION></FIGURE><P>The REMC Cloud is a graph that allow usto sort RNA structures
                      generated during the simulation according to their energy and their
                      Root-Mean-Square-Deviation (RMSD) from a selected reference structure. This the
                      average distance between the homologous  atoms of different conformations of a
                      molecule.
                    </P><P>
                      This  graph allow us to see the different steps of the
                      simulation with 3 dot groups. Group I corresponds
                      to the first RNA structures generated, as the simulation begin with a circular
                      RNA it take a little bit of iterations to find lower energy structures. Group II corresponds to intermediate
                      structures, we can see that at this point RMSD of the structures have also
                      decreased meaning that the RNA is becoming more folded. Finally, Group III corresponds to final structures,
                      folded into structures of low energy.
                    </P><P>
                      Regarding the energy values of each cluster’s
                      representative structure, they are all in Group III, so this group represents 6
                      different RNA structures. We can’t characterize the clusters by global energy
                      or RMSD. To go further in the cluster characterisation we have to study the
                      conformations and nucleic acid interactions in each structure.
                    </P><FIGURE><FIGCAPTION class="centered marged">Figure 5: Contact map of each cluster.</FIGCAPTION></FIGURE><P>The intramolecular contact map, generated with RNAmap2D<SUP>6</SUP>
                      allow us to see which nucleic acids are interacting between each others. It
                      also allow us to rapidly compare different conformations between several RNA
                      structures. Here, it reveals that the top 6 clusters actually describe 3
                      possible main folding topologies for the sequence. </P><P><U>Possible Similar Clusters</U>:</P><P style="margin-left:36.0pt;line-height:normal"><U>Set 1</U>: clusters 1,3,4,6 (topologically
                        similar to each other, despite some differences in 3D)</P><P style="margin-left:36.0pt;line-height:normal"><U>Set 2</U>: cluster 2</P><P style="margin-left:36.0pt;line-height:normal"><U>Set 3</U>: cluster 5</P><P> We aim to compare the clusters of the same set and
                          between sets, in order to see the similarities and differences. We use a “model
                          vs model contact map” to achieve that goal.</P><FIGURE><FIGCAPTION class="centered marged">Figure 6: Contact map Model vs Model.
                              On each figure, one cluster is at the left bottom side, and the other at the right
                              top side. The common nucleic acid interactions are colored in blue and unique
                              nucleic acid interactions in red.</FIGCAPTION></FIGURE><P>We saw that the cluster 1, 3, 4, and 6 have some close
                              similarities, the fact that they are in different cluster can be explained by
                              rigid body movements of subdomains and not topologically different folding. The
                              clusters 2 and 5 are rather unique, compared to each other and to cluster 1.</P><P>So we can consider that there are 3 main topologies, and
                                the secondary structure study will be based on those results.</P><P>The study of the article “The cspA mRNA Is a Thermosensor
                                  that Modulates Translation of the Cold-Shock Protein CspA.”<SUP>4</SUP>
                                  indicates that ribosomal protein S1 could be the key to cspA translation.
                                  Ribosomal protein S1 is crucial for ribosome initiation on many mRNAs,
                                  particularly for those with highly structured 5′ regions, and those with
                                  no or weak Shine-Dalgarno sequences.  To compare the three differents secondary
                                  structures from our sets with the wild-type cspA secondary structure from the
                                  article (mapped by RNA footprinting), we have highlighted in green the putative
                                  S1 binding domain (residue 134 to 147), in blue the initiation codon (residue
                                  161 to 163), in red the Shine-Dalgarno sequence (residue 148 to 151) and in
                                  yellow the DSbox sequence as general information.</P><FIGURE><FIGCAPTION class="centered marged">Figure 7: Comparison between Cluster 1 and the article’s figure</FIGCAPTION></FIGURE><P>The first similarity between the two structures is the
                                    big semi-loop that begin with the first nucleic acid and end with the end of
                                    the mRNA sequence where the reference sequences are located. On both
                                    structures, the S1 binding site and the Shine-Dalgarno sequences are located on
                                    a hairpin. Finally, under the semi-loop, the two structures present a long
                                    branch.
                                  </P><FIGURE><FIGCAPTION class="centered marged">Figure 8: Comparison between Cluster 2 and the article’s figure</FIGCAPTION></FIGURE><P>The cluster 2 presents no real similarities to the
                                    article’s figure. Indeed, we can’t see this big semi-loop and the DS box is not
                                    located on a hairpin as it is on the article’s figure.</P><FIGURE><FIGCAPTION class="centered marged">Figure 9: Comparison between Cluster 5 and the article’s figure</FIGCAPTION></FIGURE><P>When we first look at the general structure of the two
                                      structures we can see a striking resemblance: there is the semi-loop with three
                                      hairpins and a long branch at the bottom. But when we look at the reference
                                      sequences locations on the cluster 5, it doesn’t correspond to the reference
                                      sequences locations on the article’s figure.</P><P>The secondary structure analysis shows that the wild-type
                                        cspA has a secondary structure rather similar to our cspA secondary structure
                                        from cluster 1. Therefore, the lowest energy structure from the first cluster
                                        could be a good docking template for the S1 binding protein. This docking is
                                        presented in the next section.</P><H3>cspA 5’UTR - S1 domain 3 docking</H3><P>Based on our previous cspA 5’UTR results, we wished to
                                          study the docking procedure of the csPA 5’UTR on its natural ligand. To do so,
                                          we extensively researched the possible ligand on the ribosome<SUP>7</SUP>.</P><P>Our research lead us to consider the ribosomal protein
                                            subunit 1 (S1), and more specifically its domain 3 (d3).</P><P>Through modeling, we used several modeling servers, step
                                              by step, to obtain a trustworthy docking structure. We used the phyre2<SUP>8</SUP>
                                              server to have a homology model (based on template selection, followed by
                                              MAFFT-L-INSI<SUP>9</SUP> (to check the template-target alignment)and I-Tasser<SUP>10</SUP>
                                              (to generate the final model, based on previous information). All parameters
                                              inputted were defaults, except for manually indicating MAFFT the L-INSI
                                              algorythm.</P><FIGURE><FIGCAPTION class="centered marged">Figure 10: S1 modeled protein structure  with (left) and without (right) Sticks</FIGCAPTION></FIGURE><P>On this structure, we observed a specific motif with
                                                conserved residues, that we will call “valley” from now onwards. This valley
                                                seemed to be an accessible site, being located on the exterior side of S1_d3,
                                                while having two strands of highly conserved residues bordering it. As such, we
                                                considered it a very likely docking site for our cspA 5’UTR structure.</P><P>To determine whether this was due solely to the modeling
                                                  process, we compared it to a reference sequence found in the PROTEIN DATABASE
                                                  (PDB). This sequence, referred as the code 4v9r on the PDB, consisted of S1_d3
                                                  bound to single stranded RNA (ssRNA)<SUP>12</SUP>. We applied the same steps
                                                  used for our S1_d3, resulting in the following model.</P><FIGURE><FIGCAPTION class="centered marged">Figure 11: S1 reference protein structure  with (left) and without (right) Sticks</FIGCAPTION></FIGURE><P>On this structure, we also observe the same valley motif
                                                    with highly conserved residues, reinforcing our previous hypothesis.</P><P>From the previous models, we decided to study more
                                                      precisely the conservation of residues to determine the best docking site possible
                                                      and test our valley hypothesis. This was done using the Consurf web server for
                                                      both S1-d3 and 4v9r<SUP>13</SUP></P><FIGURE><FIGCAPTION class="centered marged">Figure 12:  S1 protein residue conservation: S1_d3 (left) and 4v9r (right)</FIGCAPTION></FIGURE><P>As we suspected, Consurf reaffirmed the existence of the valley presenting many highly conserved residues (darker coloration). This confirmed our previous hypothesis concerning the docking site. </P><P>Now that we had the ligand, the receptor and the likely
                                                        RNA binding site, we had all we needed to launch an actual docking simulation.
                                                        To do so, we used the PatchDock server<SUP>13</SUP>, setting the receptor as S1_d3, the
                                                        ligand as our 5’UTR (SimRNA cluster 1 result) and using default settings (4
                                                        REMC, Default complex type).</P><P>Contrary to what we initially expected, we did not get on
                                                          clear docking position (the ssRNA part), but several groups of docking
                                                          positions, as shown below.</P><FIGURE><FIGCAPTION class="centered marged">Figure 13 &amp; 14: S1 + cspA 5'UTR docking snapshots/results.</FIGCAPTION><FIGCAPTION class="centered marged">“Many of the resulting poses show the the
                                                              conserved binding site interacting with the RNA. We are still analyzing these
                                                              results.”</FIGCAPTION></FIGURE><P>Although we obtained several positions, we still notice
                                                              that many of them correspond to the conserved binding sites interacting with
                                                              the RNA. We are still analyzing these results to determine the most likely
                                                              docking structure.</P><H2>Conclusion</H2><P>The SimRNA approach offers a good way to obtain a pool of
                                                                structures with Replica Exchange Monte Carlo. The clustering allowed the
                                                                selection and the discrimination of structures by energy value, and RMSD, and
                                                                fold topology. We have isolated 3 structures sets from 6 clusters which
                                                                represent more than 90% of the isolated structures from the clustering process,
                                                                thanks to the contact maps analysis. Now we work on one defined structure, and
                                                                the analysis on the docking protein/RNA is in progress.</P><P><B><U>References</U></B>:</P><OL style="margin-top:0cm" start="1" type="1"><LI>Yang, L., Zhou, D., Liu, X., Han, H., Zhan, L., Guo, Z., Zhang, L., Qin, C., Wong, H. and Yang, R. (2009).
                                                                    Cold-induced gene expression profiles of Vibrio parahaemolyticus: a
                                                                    time-course analysis. <I>FEMS Microbiology Letters</I>, 291(1), pp.50-58.</LI><LI>Xia, B., Ke, H., Jiang, W. and Inouye, M. (2001). The Cold Box Stem-loop Proximal to the 5′-End of
                                                                      theEscherichia coli cspAGene Stabilizes Its mRNA at Low Temperature. <I>Journal
                                                                        of Biological Chemistry</I>, 277(8), pp.6005-6011.</LI><LI>Michal J. Boniecki, Grzegorz Lach, Wayne K. Dawson, Konrad Tomala, Pawel Lukasz,
                                                                          Tomasz Soltysinski, Kristian M. Rother, Janusz M. Bujnicki; SimRNA: a
                                                                          coarse-grained method for RNA folding simulations and 3D structure
                                                                          prediction, <I>Nucleic Acids Research</I>, Volume 44, Issue 7, 20 April
                                                                          2016, Pages e63, <A style="color:#1155CC" href="https://doi.org/10.1093/nar/gkv1479">doi.org/10.1093/nar/gkv1479</A></LI><LI>Giuliodori, Anna &amp; Di Pietro, Fabio &amp; Marzi, Stefano &amp; Masquida, Benoît &amp; Wagner, Rolf &amp;
                                                                            Romby, Pascale &amp; Gualerzi, Claudio &amp; L Pon, Cynthia. (2010). The
                                                                            cspA mRNA Is a Thermosensor that Modulates Translation of the Cold-Shock
                                                                            Protein CspA. Molecular cell. 37. 21-33. 10.1016/j.molcel.2009.11.033</LI><LI>GROningen MAchine for Chemical Simulations, <A style="color:#1155CC" href="http://www.gromacs.org/">gromacs.org</A></LI><LI>Pietal, M. J., Szostak, N.,
                                                                              Rother, K. M., &amp; Bujnicki, J. M. (2012). RNAmap2D–calculation,
                                                                              visualization and analysis of contact and distance maps for RNA and
                                                                              protein-RNA complex structures. BMC bioinformatics, 13(1), 333.</LI><LI>Duval, M., Korepanov, A., Fuchsbauer, O., Fechter, P., Haller, A., Fabbretti, A., … Marzi, S. (2013).
                                                                                Escherichia coli Ribosomal Protein S1 Unfolds Structured mRNAs Onto the
                                                                                Ribosome for Active Translation Initiation. PLoS Biology, 11(12),
                                                                                e1001731, <A style="color:#1155CC" href="http://doi.org/10.1371/journal.pbio.1001731">doi.org/10.1371/journal.pbio.1001731</A></LI><LI>Protein Homology/analogy Recognition Engine V 2.0, <A style="color:#1155CC" href="http://www.sbg.bio.ic.ac.uk/~phyre2/html/page.cgi?id=index">Phyre 2</A></LI><LI>Multiple alignment program for amino acid or nucleotide sequences, <A style="color:#1155CC" href="https://mafft.cbrc.jp/alignment/server/">mafft.cbrc.jp/alignment/server/</A></LI><LI class="MsoNormalCxSpMiddle" style="line-height:normal">Iterative Threading ASSEmbly
                                                                                  Refinement, <A style="color:#1155CC" href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/">I-TASSER/</A></LI><LI><A style="color:#1155CC" href="http://www.rcsb.org/pdb/explore.do?structureId=4v9r">Protein 4v9r</A></LI><LI><A style="color:#1155CC" href="http://consurf.tau.ac.il/2016/">ConSurf Server</A></LI><LI>PATCHDOCK  Molecular Docking Algorithm Based on Shape Complementarity Principles, <A style="color:#1155CC" href="http://bioinfo3d.cs.tau.ac.il/PatchDock/index.html">PATCHDOCK</A></LI></OL></DIV></DIV></DIV></DIV><DIV style="background-color:#F59D36;"><DIV class="container-fluid"><DIV class="row" style="border-top:1px solid #F7F2E9;"><DIV class="col-md-7"><H1 style="font-size: 20px">Igem ionis</H1><P style="font-size: 15px">
			  Is an association created by Sup’Biotech student in 2015. Since this first participation, two teams (2015 and 2016) won the gold medal and several nominations: « Best presentation », « Best applied design », and « Best environmental project ». The strength of the IGEM IONIS comes from its multidisciplinarity and its complementarity.
			</P><P>
			  This year we are 20 members from different schools:18 students from Sup’Biotech1 student from e-artsup1 student from Epita<B><A href="https://2017.igem.org/Team:IONIS-PARIS/Team/team">Read more …</A></B></P></DIV><DIV class="col-md-2"><H1 style="font-size: 20px">Follow us</H1></DIV><DIV class="col-md-2"><H1 style="font-size: 20px">keep IN TOUCH</H1><P><B>Location:</B> 66 Rue Guy Môquet94800 Villejuif, France
			</P><P><B>Email:</B> igem.ionis@gmail.com
			</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>