Team:IONIS-PARIS/Model
Toggle navigation
PROJECT
DESCRIPTION
DEFINITIONS
BRAINSTORMING PROCESS
MODELING
DSBOX + AmilCP
RNA FOLDING SIMULATION
LABORATORY WORK
PARTS
PROTOCOLS
RESULTS & CHARACTERISATIONS
WORKING AT LA PAILLASSE
LABORATORY NOTEBOOK
APPLIED DESIGN
OVERVIEW
FOLIAR APPLICATION
CHASSIS SELECTION
PROTECTIVE COMPOUNDS
RISK AVOIDANCE: BIOSAFETY STRATEGIES
RISK ASSESSMENT: TOXICITY & ECOTOXICITY STUDIES
CASE STUDY: FROSTBAN
HUMAN PRACTICES
OVERVIEW
PUBLIC OUTREACH
SOCIETY LAW & GMOs
PERSPECTIVES
ENTREPRENEURSHIP
OVERVIEW
MARKET STUDY
FUNDRAISING & COMMUNICATION
SPONSORS
PERSPECTIVES
TEAM
ATTRIBUTIONS
COLLABORATIONS
MEMBERS
Back to top
MODELING AWARD CONTENT
Placement of the Downstream box for amilCP
In this project we want to produce a bacterium expressing amilCP only at low temperature. The majority of the 5’ regulatory sequence, the cspA 5’UTR, is upstream of the start codon. However, a small but important sequence motif, called the DS box, is present after the start codon, and is translated as part of the target protein. Therefore, it is important to consider the structure and function of the target protein, particularly the N-terminal region, in order to correctly place the DS box insertion so that it does not alter the structure or function. However, there is no experimental structure for amilCP. Therefore, we use a computational approach, homology modeling, to create a model of amilCP and identify a safe insertion site in the sequence that would not alter the structure. Read the simulation results
RNA folding
We chose to use the cspA promoter that is activated during cold shock in several bacteria such as wild type (WT) Escherichia coli. Also, based on the literature we choose to include the UP element of the cspA promoter, the cspA 5’UTR, and the DownStream Box (DSBox) in our final construction. These temperature sensing sequence motifs are upstream of our reporter protein (amilCP), with the exception of the DSBox. Our bibliographic review indicates that these components are required to see the conditional expression of amilCP only at low temperature1,2 (under 15°C, approximately).
We found in the bibliography that the current main hypothesis that could explain the expression of the cold shock protein family is the structure of the mRNA. At 37°C their mRNA is quickly degraded. As the temperature decreases the mRNA assumes a different structure. This leads to the stabilization of the mRNA structure at low temperatures.  We used a computational method, SimRNA3, in order to generate 3D models of this region of the cspA mRNA.  Our results suggest that several short distinct sequence motifs classically identified as important for ribosome binding actually participate together in the formation of a secondary structure motif that is recognized in a generally non base-pair specific manner by ribosomal accessory proteins in the formation of the pre-initiation complex at low temperatures. Read the simulation results
