Team:IONIS-PARIS/Modeling/dsbox-amilcp
Toggle navigation
PROJECT
DESCRIPTION
DEFINITIONS
BRAINSTORMING PROCESS
MODELING
DSBOX + AmilCP
RNA FOLDING SIMULATION
LABORATORY WORK
PARTS
PROTOCOLS
RESULTS & CHARACTERISATIONS
WORKING AT LA PAILLASSE
LABORATORY NOTEBOOK
APPLIED DESIGN
OVERVIEW
FOLIAR APPLICATION
CHASSIS SELECTION
PROTECTIVE COMPOUNDS
RISK AVOIDANCE: BIOSAFETY STRATEGIES
RISK ASSESSMENT: TOXICITY & ECOTOXICITY STUDIES
CASE STUDY: FROSTBAN
HUMAN PRACTICES
OVERVIEW
PUBLIC OUTREACH
SOCIETY LAW & GMOs
PERSPECTIVES
ENTREPRENEURSHIP
OVERVIEW
MARKET STUDY
FUNDRAISING & COMMUNICATION
SPONSORS
PERSPECTIVES
TEAM
ATTRIBUTIONS
COLLABORATIONS
MEMBERS
Back to top
Placement of the Downstream box for amilCP
Overview:
We generated a novel 3D model of wild-type amilCP as it has never been done before.
We analysed the possible location to introduce our DS box.
We modeled our modified amilCP with the addition of the DS box.
We studied the possible modifications generated by the DS box to predict that the chromofunction will stay intact.
Introduction
In this project we want to produce a bacterium expressing amilCP only at low temperature.
The majority of the 5’ regulatory sequence, the cspA 5’UTR, is upstream of the start codon.
However, a small but important sequence motif, called the DS box, is present after the start             codon, and is translated as part of the target protein. Therefore, it is important to             consider the structure and function of the target protein, particularly the N-terminal region,             in order to correctly place the DS box insertion so that it does not alter the structure or function.             However, there is no experimental structure for amilCP. Therefore, we use a computational approach,             homology modeling, to create a model of amilCP and identify a safe insertion site in the sequence             that would not alter the structure.
Our version of the DS box RNA sequence is 15 nucleotides long (as opposed to the 14 nucleotide             version reported in YAMANAKA1); we added an extra nucleotide at the 3’ end of this motif             so as to not introduce a frame shift error in amilCP.
Materials and Methods
We used the Phyre2 homology modeling server2 to find templates, generate alignments, and create 3D models. Phyre2 use the hidden Markov model. This model is trustworthy up to 15% identity between the protein to model and the chosen template.
Modeling parameters
We submitted the amilCP sequence ( Part:BBa_K592009 )           to the Phyre2 server for all steps of homology modeling, from template search to model generation.
MSVIAKQMTYKVYMSGTVNGHYFEVEGDGKGKPYEGEQTVKLTVTKGGPLPFAWDILSPQCQYGSIPFTKYPEDIPDYVKQSFPEGYTWERIMNFEDGAVCTVSNDSSIQGNCFIYHVKFSGLNFPPNGPVMQKKTQGWEPNTERLFARDGMLLGNNFMALKLEGGGHYLCEFKTTYKAKKPVKMPGYHYVDRKLDVTNHNKDYTSVEQCEISIARKPVVA
Results
Homology modelling of wild-type AmilCP by Phyre2
Phyre2 choose as the best template PDB entry 1mou         ( See Protein Data Bank )         which is a blue chromoprotein from the GFP family found in Montipora efflorescens.
We have made available all the           complete Phyre 2 results: Download here
Phyre2’s best template match was PDB entry 1mou (a coral pocilloporin pigment), calculated to have a sequence           identity of 95% to amilCP. This implies that the structure of amilCP will be largely identical.
Figure 1: Best template according to Phyre2 for the amilCP
Phyre2’s resulting model is indeed a GFP-like beta barrel with modified chromophore residue in the middle.
Figure 2: Summary of the amilCP homology modeling simulation given by Phyre2
Note: Please wait a few seconds after clicking !
Model evaluation
The very high sequence ID gave us high confidence in the resulting model, even without further optimization         or comparative analysis. We were specifically interested in the position of the N-terminus relative to the         structure of the rest of the protein, as this would be the general insertion site of the DS Box sequence.         In order to evaluate the model quality and obtain a precise geometrical defintion of secondary structure at         the residue level we analyzed the resulting model’s Ramachandran plot.
Analysis of the DS box’s placement within amilCP protein
All the following visualisations have been performed using the BioVia Discovery Studio Visualizer developed         by Dassault Systemes.
Here we highlight the position of the N-terminus on the 3D structure. AmilCP has a barrel Beta sheet         conformation. Based on the Ramachandran plot, we see that the fourth amino acid (isoleucine) is involved in         the beta sheet. To avoid the disturbing the protein structure, we must place the DS box before this fourth         amino acid.
Figure 3: Ramachandran Plot of the amilCP Wilde-Type.
Solvent accessibility
Next, we calculated the solvent accessibility of the protein structure, paying particular attention to the             N-terminus. This allowed us to assess it’s interaction with the rest of the protein structure. The blue             color means that a residue is solvent accessible; green indicates that it is buried in the protein structure.             The amilCP model is represented in ribbon on the left and as a space filling model on the right.
Figure 4: Solvent Accessibility of the amilCP.
From the Ramachandran plot we know that this region is not locked into a beta sheet conformation, and from the         solvent accessible surface calculation it appears that it makes little contact with the rest of the protein         structure. After consideration we decided to place the DS box after the first two residues.
Placement of the DS box
Once the DS box insertion site was chosen, Phyre2 was used to create a 3D model of amilCP with the DS box sequence         added after the second residue.         AmilCp + DS box sequence (DS box in red):
MSMTGIVVIAKQMTYKVYMSGTVNGHYFEVEGDGKGKPYEGEQTVKLTVTKGGPLPFAWDILSPQCQYGFTKYPEDIPDYVKQSFPEGYTWERIMNFEDGAVCTVSNDSSIQGNCFIYHVKFSGLNFPPNGPVMQKKTQGWEPNTERLFARDGMLLGNNFMALKLEGGGHYLCEFKTTYKAKKPVKMPGYHYVDRKLDVTNHNKDYTSVEQCEISIARKPVVA
Phyre 2 returned this 3D model. The DS box is highlighted in yellow. The beginning of the protein strand, is clearly not involved in the beta sheets barrel.
Figure 5: Location of the DS Box (in yellow) on the amilCP
Note: Please wait a few seconds after clicking !
We have made available all the complete Phyre 2 results: Download here
To confirm the viability of this model, a residue interaction analysis was done with the “Bioviva Discovery Studio           Visualizer” software, developed by DASSAULT SYSTEMES.
Figure 6: Amino Acids contact with the DS Box.
The residues having a possible interaction with the DS box sequence are highlighted in yellow on the 3D model and           in black on the sequence.
This shows only two residues in the protein seems to have a probability to interact with the DS box which is less           than the wild type beginning of the protein.
Conclusion
Based on the very high (95%) sequence identity between target and template, we have confidence in Phyre2’s homology model. We extended the DS box sequence from 14 to 15 nucleotides in order to avoid a frame shift error and then placed this insert into the coding sequence at a position which is predicted to not interefere with amilCP’s chromogenic structure. This will be verified experimentally by activity assays of our rationally designed construct.
References
Yamanaka K, Mitta M, Inouye M. Mutation Analysis of the 5′ Untranslated Region of the Cold Shock cspA mRNA of Escherichia coli. Journal of Bacteriology. 1999;181(20):6284-6291.
Kelley LA et al. The Phyre2 web portal for protein modeling, prediction and analysis. Nature Protocols 10, 845-858 (2015)
Igem ionis
Is an association created by Sup’Biotech student in 2015. Since this first participation, two teams (2015 and 2016) won the gold medal and several nominations: « Best presentation », « Best applied design », and « Best environmental project ». The strength of the IGEM IONIS comes from its multidisciplinarity and its complementarity.
This year we are 20 members from different schools:18 students from Sup’Biotech1 student from e-artsup1 student from Epita Read more …
Follow us
keep IN TOUCH
Location: 66 Rue Guy Môquet94800 Villejuif, France
Email: igem.ionis@gmail.com
