<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TPHS_San_Diego_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TPHS San Diego/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="navbar"><DIV id="navbar-fill"><DIV class="dropdown nav-animation"><BUTTON class="dropbtn" class="" id="title" style="color: white"><A href="https://2018.igem.org/Team:TPHS_San_Diego"> Home </A></BUTTON></DIV><DIV class="dropdown"><BUTTON class="dropbtn"> Project </BUTTON><DIV class="dropdown-content" style="left:0;"><A style="font-family: 'Montserrat', sans-serif;" href="https://2018.igem.org/Team:TPHS_San_Diego/Description">Description</A><A style="font-family: 'Montserrat', sans-serif;" href="https://2018.igem.org/Team:TPHS_San_Diego/Model">Modeling</A><A style="font-family: 'Montserrat', sans-serif;" href="https://2018.igem.org/Team:TPHS_San_Diego/Parts">Parts</A><A style="font-family: 'Montserrat', sans-serif;" href="https://2018.igem.org/Team:TPHS_San_Diego/Results">Results</A><A style="font-family: 'Montserrat', sans-serif;" href="https://2018.igem.org/Team:TPHS_San_Diego/Safety">Safety</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn"> Notebook </BUTTON><DIV class="dropdown-content"><A style="font-family: 'Montserrat', sans-serif;" href="https://2018.igem.org/Team:TPHS_San_Diego/Notebook">Lab Notebook</A><A style="font-family: 'Montserrat', sans-serif;" href="https://2018.igem.org/Team:TPHS_San_Diego/Experiments">Experiments</A><A style="font-family: 'Montserrat', sans-serif;" href="https://2018.igem.org/Team:TPHS_San_Diego/Improve">Part Improvements</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn"> Human Practices </BUTTON><DIV class="dropdown-content"><A style="font-family: 'Montserrat', sans-serif;" href="https://2018.igem.org/Team:TPHS_San_Diego/Human_Practices">Integrated Human Practices</A><A style="font-family: 'Montserrat', sans-serif;" href="https://2018.igem.org/Team:TPHS_San_Diego/Collaborations">Collaborations</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn"> Team </BUTTON><DIV class="dropdown-content"><A style="font-family: 'Montserrat', sans-serif;" href="https://2018.igem.org/Team:TPHS_San_Diego/Team">Team</A><A style="font-family: 'Montserrat', sans-serif;" href="https://2018.igem.org/Team:TPHS_San_Diego/Attributions">Attributions</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn"> Judging </BUTTON><DIV class="dropdown-content"><A style="font-family: 'Montserrat', sans-serif;" href="https://2018.igem.org/Team:TPHS_San_Diego/Medal_Criteria">Medal Criteria</A></DIV></DIV></DIV></DIV><P><TITLE>TPHS IGEM Wiki</TITLE></P><DIV class="content_wrapper">
                Lab Notebook
            </DIV><DIV class="body_wrapper"><SECTION><CENTER><A href="https://2018.igem.org/Team:TPHS_San_Diego/Experiments">Click Here to Refer to Experiments Page</A></CENTER><H3 class="day">
Day 1
</H3><P class="des">
Miniprepped bacteria with pBAD-D4 (plasmid in which we will be inserting the chitinase genes, tags, etc.) to isolate the pBAD backbone

Final DNA concentration: 123.7 ng/μL</P><H3 class="day">
Day 2
</H3><P class="des">
Restriction digest using BamHI and EcoRI to check for bacterial transformation and make sure that the plasmid is the expected length
</P><DIV class="img_group"><P class="des">
Wells: 1. DNA Ladder. 2. Only EcoRI 3. Only BamHI. 4. No enzyme. 5. Both enzymes</P></DIV><H3 class="day">
Day 3
</H3><P class="des">
Made KPi Buffer to be used in the chitinase assay
</P><OL><LI>Prepare 800 mL of dH2O in a suitable container</LI><LI>Add 2.405 g of K2HPO4 to the solution</LI><LI>Add 11.73 g of KH2PO4 to the solution</LI><LI>Add distilled water until volume is 1 L</LI></OL><H3 class="day">
Day 4
</H3><P class="des">
Started cloning of pBAD-GST-ChiA-FLAG construct.

(GST and FLAG are protein tags which are used to purify and detect chitinase, respectively)

Completed the restriction digest portion. Gel purification, ligation, and plating will be finished next time.
</P><H3 class="day">
Day 5
</H3><P class="des">
Ran gel of restriction digest for pBAD only and did Gel Purification of restriction digest for GST/gBlock. We want to preserve the amount of DNA gBlock that we have and will lose less sample via PCR purification. 
Final concentrations: 
pBAD: 22 ng/μL
gBlock: 10.6 ng/μL
Did DNA ligation and plated BL21 competent cells cloned with GST-ChiA full construct </P><H3 class="day">
Day 6
</H3><P class="des">
Selected 10 colonies from Vector+Insert plate and put in 4 mL of LB+Ampicillin (LB is a nutrient-rich medium for bacterial growth, Ampicillin assists selection of transformed bacteria).
Incubated in 37 ºC shaker for 24 hours
Did restriction digest and gel on pBAD-D4 vector using MluI and HindIII to make sure the enzymes are cutting properly. (If DNA length matches expected length, then enzymes are working properly)
Depending on how the colonies turn out after they are sequenced, we may need to repeat the processes because there are not many colonies. We will likely use primers to enhance the gBlock/insert DNA and then try again. 
</P><DIV class="img_group"><P>
Vector+Insert colony
</P></DIV><H3 class="day">
Day 7
</H3><P class="des">
Miniprepped 10 bacteria colonies that we selected from Wednesday. Used NanoDrop machine to find DNA concentrations of all 10 samples. 
</P><DIV class="table_wrapper"><TABLE><TBODY><TR><TD>Sample 1: 97.1 ng/μL</TD><TD>Sample 2: 39.6 ng/μL</TD><TD> Sample 3: 47.3 ng/μL</TD></TR><TR><TD>Sample 4: 47.0 ng/μL</TD><TD>Sample 5: 41.8 ng/μL</TD><TD>Sample 6: 57.8 ng/μL</TD></TR><TR><TD>Sample 7: 33.6 ng/μL</TD><TD>Sample 8: 51.7 ng/μL</TD><TD>Sample 9: 41.5 ng/μL</TD></TR><TR><TD>Sample 10: 23.0 ng/μL</TD><TD>Sample pBAD-D4: 123.7 ng/μL</TD></TR></TBODY></TABLE></DIV><P class="des">We chose a specific restriction enzyme to cut the original pBAD-D4 plasmid and our constructed plasmid so that they can be distinguished based on the length of their base pairs and the number of cuts that are made. We chose EcoRI-HF in order to check that the plasmid being used are the correct ones.

250 ng per restriction digest reaction. Add 2 μL of buffer, 1 μL of enzyme, and fill to 20 μL with water. Incubate at 37 ºC for 30 mins-1hr.</P><DIV class="table_wrapper"><TABLE><CAPTION><H3>Table of Samples</H3></CAPTION><TBODY><TR><TH>Sample</TH><TH>Amount of DNA</TH><TH>Amount of Cut Smart Buffer</TH><TH>Amount of Water</TH><TH>Amount of Enzyme (EcoRI)</TH></TR><TR><TD>pBAD-D4</TD><TD>2.02 μL</TD><TD>2 μL</TD><TD>14.98 μL</TD><TD>1 μL</TD></TR><TR><TD>Sample 1:</TD><TD>2.57 μL</TD><TD>2 μL</TD><TD>14.43 μL</TD><TD>1 μL</TD></TR><TR><TD>Sample 2:</TD><TD>6.31 μL</TD><TD>2 μL</TD><TD>10.69 μL</TD><TD>1 μL</TD></TR><TR><TD>Sample 3:</TD><TD>5.29 μL</TD><TD>2 μL</TD><TD>11.71 μL</TD><TD>1 μL</TD></TR><TR><TD>Sample 4:</TD><TD>5.32 μL</TD><TD>2 μL</TD><TD>11.68 μL</TD><TD>1 μL</TD></TR><TR><TD>Sample 5:</TD><TD>5.98 μL</TD><TD>2 μL</TD><TD>11.02 μL</TD><TD>1 μL</TD></TR><TR><TD>Sample 6:</TD><TD>4.33 μL</TD><TD>2 μL</TD><TD>12.7 μL</TD><TD>1 μL</TD></TR><TR><TD>Sample 7:</TD><TD>7.44 μL</TD><TD>2 μL</TD><TD>9.56 μL</TD><TD>1 μL</TD></TR><TR><TD>Sample 8:</TD><TD>4.84 μL</TD><TD>2 μL</TD><TD>12.16 μL</TD><TD>1 μL</TD></TR><TR><TD>Sample 9:</TD><TD>6.02 μL</TD><TD>2 μL</TD><TD>10.98 μL</TD><TD>1 μL</TD></TR><TR><TD>Sample 10:</TD><TD>10.87 μL</TD><TD>2 μL</TD><TD>6.13 μL</TD><TD>1 μL</TD></TR></TBODY></TABLE></DIV><P class="des">Made a 1.0% agarose gel with 150 mL 1x TAE buffer and 15 μL of Ethidium Bromide in a medium sized gel frame. Loaded 10 μL of DNA Ladder and put 4 μL of 6x loading dye into each sample. Ran the gel at 175 V for 45 mins-1 hr. </P><DIV class="img_group"><P>
An image of the gel simulated on a computerprogram called SnapGene.
</P></DIV><DIV class="img_group"><P>
An image of the actual gel.
</P></DIV><P class="des">Wells 3-12 were extracted from bacterial colonies that all have the same DNA inserted. However, when we run a gel or a restriction digest in which all the samples were cut with ECORI, the results are not the same. We hypothesize that this is probably due to the enzyme not cutting the DNA for some samples. Further testing is needed to determine the issue.</P><P class="des">Resolved: The issue was likely caused by the incompatibility of the enzymes in the buffer and poor primer design. When performing PCR, it is important to add restriction sites to the primers so that the product would include the restriction site. This was not done originally. Also, the enzymes used to perform the restriction digest, HindIII-HF and MluI, were not compatible in the same buffer. HF(High Fidelity) indicates that the enzyme has reduced star activity, or a reduced tendency to lose specificity and can be used in a wider range of buffers.</P><H3 class="day">
Day 8
</H3><P class="des">
Did restriction digest on pBAD-D4 vector using HindIII-HF and MluI-HF. Let incubate overnight.</P><H3 class="day">
Day 9
</H3><P class="des">
Did PCR on GST-ChiA with MluI forward primers and HindIII reverse primers. (These primers attach restriction sites onto the insert along with a buffer region in case of exonuclease action)<PRE> 	     PCR cycle: 66ºC  10 seconds (phusion)        72ºC 1.5 mins
             20 cycles</PRE>
Did PCR clean up using Qiagen kit (eluted with 50 μL H2O)
Performed restriction digest on PCR product
<PRE>  	     50 μL DNA+6 μL CutSmart Buffer+2 μL HindIII-HF+ 2 μL MluI-HF
	     Incubate at 37ºC ~3-4hrs</PRE>
Made 1% gel and ran pBAD digest and GST-ChiA digest (from above) 
<PRE> 	     GST-ChiA: 60 μL = 12 μL 6x loading dye (36 μL into each well)
	     pBAD: 50.4 μL = 10 μL 6x loading dye (30 μL into each well)</PRE>
Used QiaGen gel purification kit to isolate pBAD
<PRE>             1.  pBAD extract from 3,772 bp
             GST-ChiA extract from ~2.7 kb</PRE></P><H3 class="day">
Day 10
</H3><P class="des">
Made 5 g of colloidal chitin. Redid PCR reaction for GST-ChiA. Ran the gel.
</P><P class="des">
Used Qubits to find DNA concentrations.<PRE> 	     pBAD	  23.0 ng/μL
	     GST-ChiA	  39.2 ng/μL</PRE>
Ligated the two pieces.Transformed the bacteria (Repeated protocol from earlier, except added primers before).Plated 50 μL of cells on Amp-LB agar plate.Incubated at 37C O/N</P><H3 class="day">
Day 11
</H3><P class="des">
Transformation Attempt #2
</P><DIV class="table_wrapper"><TABLE><TBODY><TR><TD>Results with Chitinase+Chitin (incubation period of 2.5 hrs)</TD></TR></TBODY></TABLE></DIV><P class="des">
We made colloids of chitin, which we used in a Chitinase Assay to test the viability for the experiment. For the assay, the Chitinase concentration was diluted in a series, while the amount of chitin colloid was kept constant.This is how the assay works:
</P><OL><LI>When chitinase digests chitin, chito-oligosaccharides are generated as a product.</LI><LI>This is digested using ChitO, generating hydrogen peroxide.</LI><LI>HRP uses the hydrogen peroxide to convert AAP and DCHBS into a pink compound, which allows us to visually detect the presence of chitinase.</LI><LI>To quantify the amount of chitinase present, a spectrophotometer will be used to detect the absorbance: higher means more concentrated. (absorbance = extinction coefficient * cuvette length * concentration) </LI></OL><H3 class="day">
Day 12
</H3><P class="des"> 
Mini-prepped colonies that we picked yesterday. Ran a gel and sent the purified DNA for sequencing. Selected 9 colonies:
</P><DIV class="table_wrapper"><TABLE><TBODY><TR><TH>Sample</TH><TH>DNA</TH><TH>Buffer</TH><TH>Enzyme</TH><TH>Water</TH></TR><TR><TD>General</TD><TD>250 ng</TD><TD>2 μL</TD><TD>1 μL</TD><TD>To 20 μL</TD></TR><TR><TD>1</TD><TD>132.1 ng/μL</TD><TD>2 μL</TD><TD>1 μL</TD><TD>15.11 μL</TD></TR><TR><TD>2</TD><TD>95.8 ng/μL</TD><TD>2 μL</TD><TD>1 μL</TD><TD>14.39 μL</TD></TR><TR><TD>3</TD><TD>70.4 ng/μL</TD><TD>2 μL</TD><TD>1 μL</TD><TD>13.45 μL</TD></TR><TR><TD>4</TD><TD>111.4 ng/μL</TD><TD>2 μL</TD><TD>1 μL</TD><TD>14.76 μL</TD></TR><TR><TD>5</TD><TD>76.9 ng/μL</TD><TD>2 μL</TD><TD>1 μL</TD><TD>13.75 μL</TD></TR><TR><TD>6</TD><TD>105.5 ng/μL</TD><TD>2 μL</TD><TD>1 μL</TD><TD>14.63 μL</TD></TR><TR><TD>7</TD><TD>83.6 ng/μL</TD><TD>2 μL</TD><TD>1 μL</TD><TD>14.01 μL</TD></TR><TR><TD>8</TD><TD>120.4 ng/μL</TD><TD>2 μL</TD><TD>1 μL</TD><TD>14.92 μL</TD></TR><TR><TD>8</TD><TD>132.9 ng/μL</TD><TD>2 μL</TD><TD>1 μL</TD><TD>15.12 μL</TD></TR></TBODY></TABLE></DIV><P class="des">Ran gel using EcoRI. The expected results are the same as above. The first well is 1kb DNA ladder, wells 2-10 are GST-ChiA, and the last well is pBAD-D4 vector cut with EcoRI as a control.</P><DIV class="img_group"><P class="des">
This gel is just to check that the plasmid was transformed correctly as we know the restriction sites for the cloned construct. Based on this gel, 7/9 of the clones successfully received the plasmid and we can use all of those clones (except clone 3 and 7) in experimentation.
</P></DIV><DIV class="table_wrapper"><TABLE><CAPTION><H3>Chitinase Assay</H3></CAPTION><TBODY><TR><TD>Assay (take.1)</TD><TD>Assay (supernatant only)</TD><TD>Results (take.1)</TD><TD>Results (supernatant only)
</TD></TR><TR><TD>t</TD></TR></TBODY></TABLE></DIV><P class="des">Based on these results, the assay protocol that we chose is sensitive enough. We performed this assay with commercially purchased chitinase in order to test the sensitivity of the assay protocol that we chose. Based on these results it seems that our protocol could be usable, however, we would need to filter the colloidal chitin somehow so that there are fewer chunks floating around in the solution. Considering that we now have a potent strain of GST-ChiA, we can soon use this DNA with the assay to identify chitinase activity.</P><H3 class="day">
Day 13
</H3><P class="des">
The colloidal chitinase particles used in the previous assay test were too large. Redid the assay test with smaller particles. We modified the protocol based on cross-references from several other colloidal chitin protocols and decided to increase the volume and chitin to HCl ratio to 1:10. We also increased the incubation period for the 12M HCl and Chitin to 2.5 hrs instead of the original 1 hr. We covered the beaker with foil as it stirred but the vapor from the acid degraded the foil and caused flakes to fall into the foil. We will need to remake it tomorrow.
Also transformed BL21 Cells with complete plasmid and plate 50 μL to LB-AMP plates. Incubate at 37C overnight. Will select colonies and put in shaker overnight.</P><H3 class="day">
Day 14
</H3><P class="des">
Redid colloidal chitin, selected colonies and put in 4 mL LB-AMP and left in shaking incubator (37C O/N). Prepared 10 aliquots: 4 tubes with 2 mL of LB per for clone 1 and 9 to be used for arabinose induction and WB. 2 tubes of 4 mL of LB for miniprep.</P><H3 class="day">
Day 15
</H3><P class="des">
Performed Arabinose Induction:
</P><OL><LI>Took 100 uL of cell culture (from yesterday) and put in 10 mL of LB-AMP. Put in shaking incubator for 2 hrs</LI><LI>Perform serial dilution of Arabinose
<PRE> 	     a. 20% to 2% to 0.2% and 0% (just H2O)</PRE></LI><LI>Add 100 uL of each dilution to tube samples </LI><LI>Leave in shaking incubator for 4 hrs</LI><LI>Take 1 mL of each sample, spin at max speed for 1 minute, and aspirate supernatant</LI></OL><P class="des">
Prepared samples for western blot (SDS page): Gel electrophoresis for proteins
</P><OL><LI>Centrifuge samples and aspirate supernatant</LI><LI>Resuspend pellet in 100 uL SDS page buffer.</LI><LI>Boil 5 min and centrifuge</LI><LI>Load 12 uL of protein ladder into the first well and 10 uL of each sample and a negative control into the other wells.</LI><LI>Record position of samples.</LI><LI>Adjust to desired voltage and run for 1 hr</LI></OL><P class="des">
In this experiment, we are checking for the expression of chitinase. After the proteins are separated by molecular weight, we will transfer them onto a different membrane that will be stained with two antibodies: one to bind to chitinase, the other to help us visualize its presence by binding to the first antibody. The second antibody has an HRP (horseradish peroxidase) tag which will produce light when reacted with luminol and hydrogen peroxide. We will capture this light on a film and develop the film to get a better image. This is possible because the FLAG tag, which is included in our DNA construct, attaches itself onto chitinase and is recognized by the first antibody (called anti-FLAG) with high specificity. The second antibody recognizes the first antibody and “brings” the HRP. 
We left the membrane in blocking buffer at 4C O/N<PRE> 	     Clone 1: Seems okay will need to sequence further to double check
             Clone 9: There is a frameshift early in the GST sequence so the protein will not be expressed properly. </PRE></P><H3 class="day">
Day 16
</H3><P class="des">
Finishing Western Blot:
<PRE>        Primary Antibody: Mouse αFLAG M2
             4 mL Blocking Buffer (milk) + 1.25 λ of αFLAG (1:4000)
             (Blocking: a nonspecific antibody such as those found in milk will bind randomly, “blocking” the primary antibody and secondary antibody                from binding to a wrong target, increasing specificity, decreasing the background noise in the film).
             Add to membrane (after draining milk) and leave on shaking @ RT for 1 hr
             Wash with TBSTE buffer 2-3 times
             Secondary Antibody: Goat derived mouse antibody- with HRP
             10 mL blocking buffer + 1 uL of GMHRP and incubate on shaker for 1 hr
             Wash with TBSTE buffer 3 times (five minutes on shaker each time)
             Drip off excess and add 1 mL of buffer and luminol
             Prepare cassette and drip off excess before putting in membrane and take the picture. 
             Expose for 1 second, 5 seconds, 10 seconds, and 30 seconds.</PRE></P><P class="des">
Results:
</P><DIV class="img_group"><P class="des">
(photo is 1 second exposure and no red light) Well 1-4 is for clone 1 with serial dilution of arabinose. Well 5-8 is the same for clone 9. Expected size of protein= 84.99 Dka
1 second was the optimal exposure time since it had the least amount of background disturbance.</P></DIV><P class="des">
Conclusion We did the serial dilution of arabinose in order to determine the concentration that we would need to induce protein expression. Based on these results, 0.2% is too little but we can get induction with 2% arabinose. Well 1 had 20% arabinose, which also induced expression, but it is better to use the least concentration that is still viable. After sequencing, we discovered that clone 9 had a frameshift mutation. Thus, there is not much protein expression to be expected in this Western Blot, as seen in wells 5-8. Once we purify the protein, we can further decide if 2% arabinose is too much or too little concentration (if the purification process does not go well). Otherwise, we can continue with purification and the experiment. 
We also redid the chitinase assay with the colloidal chitin that we remade to avoid undissolved particles that could affect the spectrophotometer reading.</P><H3 class="day">
Day 17
</H3><P class="des">
Previously made chitin agar plates (with ampicillin and arabinose) plated GST-ChiA-FLAG construct on one plate and pBAD-D4 on another plate as a control
</P><DIV class="img_group"><P class="des">
left:GST-ChiA. right:pBAD</P></DIV><P class="des">
As shown above, there are distinct halos around the GST-ChiA colonies showing chitin degradation. We will later repeat this process using a recipe that includes calcofluor white which is a dye that will enhance the image and make it easier for us to see the chitin degradation. We also stored two samples of GST-ChiA in 20 mL of LB-Amp media overnight.</P><H3 class="day">
Day 18
</H3><P class="des">
Started protocol for GST Protein purification. 
Transferred bacterial cultures to larger aliquots of LB, induced protein expression with arabinose, and spun down the cells and saving the pellet. Poured LB and Chitin agar plates, as well as plated transform bacteria with GST-ChiA-FLAG and GST control on 5%, 10% LB and Chitin
Plates must incubate overnight at 37C. Miniprep samples of GST-ChiA and GST. </P><H3 class="day">
Day 19
</H3><P class="des">
The protein purification that was supposed to be spun down for 30 minutes and then stored at -80C, was left in the centrifuge at 4C. We still continued with the purification protocol and prepared another round of bacterial cultures to repeat the process if necessary.
PCR Tubes
<PRE> 	     Clone 7
             Clone 7+ MnCl2
             Clone 10
             Clone 10+ MnCl2</PRE></P><P class="des">
PCR Reaction mix:
28 uL H2O
5 uL 10x Buffer
5 uL 10 mM dNTP
5 uL of 10 mM Primer mix (either clone 7 or clone 10)
5 uL of DNA Template (200 pg/uL)*
1 uL 25 mM MnCl2** (for non MnCl2 samples just add 1 more uL of H2O)
1 uL of Taq</P><P class="des">
*Template Stock= 145.8 ng/uL
<PRE> 	     +  1uL stock and 99 uL H2O (1:100 dilution)Clone 7
                New Concentration= 1.458 ng/uL
             +  10 uL of diluted stock and 62.9 uL H2O
                Final Concentration= 200 pg/uL</PRE>
**MnCl2 Stock= 1M
<PRE> 	     +  25 uL MnCl2 and 975 uL H2O
                Final concentration= 25 mM</PRE>
PCR conditions
94C 5’
94C 10”
65C 30”
72C 3’ (repeat 94, 65, and 72-3 for 20 cycles)
72C 6’
4C</P><H3 class="day">
Day 20
</H3><P class="des">
Ran Gel for PCR Reaction- there were no bands- so redid PCR reaction. Reaction mix is the same as last time. PCR conditions were changed.
Dilutions:
*Template Stock= 123.7 ng/uL
<PRE> 	     +  1uL stock and 99 uL H2O (1:100 dilution)
                New concentration= 1.237 ng/uL
             +  10 uL of diluted stock and 61.9 uL H2O
                Final Concentration= 200 pg/uL</PRE>
**MnCl2 Stock= 1M
<PRE> 	     +  25 uL MnCl2 and 975 uL H2O
                Final concentration= 25 mM</PRE>
PCR Conditions (take 2)
94C 5’
94C 10”
52C 30”
72C 3’(repeat 94, 52, and 72-3 for 30 cycles)
72C 6’
4C
Plated cells from Thursday. Need to be at room temperature for 2-3 days to fully grow.</P><DIV class="table_wrapper"><TABLE><TBODY><TR><TD>Plated without calcofluor white (left is ChiA, right is GST only control)</TD><TD>5% Chitin/LB with calcofluor white (left is ChiA, right is GST only control)</TD><TD>10% chitin/LB with calcofluor white (left is ChiA, right is GST only control)</TD></TR></TBODY></TABLE></DIV><P class="des">Conclusion
The plates that did not have calcofluor white showed more chitin degradation and it was easier to see the degradation, this could be because the dye decreases enzymatic activity or the makes the bacteria less efficient. In any case, it will be better to use the plates without calcofluor white dye for chitinase plate assay.<PRE> 	  Resuspended beads and washed 200 uL with 1 mL of TKET
                  Put 200 uL and 1 mL of TKET into Eppendorf tube
                  Do a quick spin
                  Put the tube on a magnet rack
                  Remove supernatant
                  Repeat above for second wash
             Spun lysate down at 13,000 rpm for 15 mins save 50 uL of supernatant (total lysate)
             Continued to follow Protein Purification protocol
             Did Bradford assay as a quick check and there were no proteins. Did bacterial culture again. Will do protein purification again tomorrow.</PRE></P><H3 class="day">
Day 21
</H3><P class="des">
Did Protein Purification with the second round of colonies that we prepared. Froze all of the total lysate sample (process will be continued next time).
Ran a gel for PCR reaction from Monday. No bands were visible.</P><H3 class="day">
Day 22
</H3><P class="des">
Redid PCR reaction. Bands were visible. The results are shown below.</P><DIV class="img_group"><P class="des">
DNA Ladder. Well 1: C7. Well 2: C7 with MnCl2. Well 3: C10. Well 4: C10 with MnCl2
Finished Protein Purification. Bradford assay needs to be done to confirm that the protein was purified (See Results page).</P></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>