<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TPHS_San_Diego_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TPHS San Diego/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="navbar"><DIV id="navbar-fill"><DIV class="dropdown nav-animation"><BUTTON class="dropbtn" class="" id="title" style="color: white"><A href="https://2018.igem.org/Team:TPHS_San_Diego"> Home </A></BUTTON></DIV><DIV class="dropdown"><BUTTON class="dropbtn"> Project </BUTTON><DIV class="dropdown-content" style="left:0;"><A style="font-family: 'Montserrat', sans-serif;" href="https://2018.igem.org/Team:TPHS_San_Diego/Description">Description</A><A style="font-family: 'Montserrat', sans-serif;" href="https://2018.igem.org/Team:TPHS_San_Diego/Model">Modeling</A><A style="font-family: 'Montserrat', sans-serif;" href="https://2018.igem.org/Team:TPHS_San_Diego/Parts">Parts</A><A style="font-family: 'Montserrat', sans-serif;" href="https://2018.igem.org/Team:TPHS_San_Diego/Results">Results</A><A style="font-family: 'Montserrat', sans-serif;" href="https://2018.igem.org/Team:TPHS_San_Diego/Safety">Safety</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn"> Notebook </BUTTON><DIV class="dropdown-content"><A style="font-family: 'Montserrat', sans-serif;" href="https://2018.igem.org/Team:TPHS_San_Diego/Notebook">Lab Notebook</A><A style="font-family: 'Montserrat', sans-serif;" href="https://2018.igem.org/Team:TPHS_San_Diego/Experiments">Experiments</A><A style="font-family: 'Montserrat', sans-serif;" href="https://2018.igem.org/Team:TPHS_San_Diego/Improve">Part Improvements</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn"> Human Practices </BUTTON><DIV class="dropdown-content"><A style="font-family: 'Montserrat', sans-serif;" href="https://2018.igem.org/Team:TPHS_San_Diego/Human_Practices">Integrated Human Practices</A><A style="font-family: 'Montserrat', sans-serif;" href="https://2018.igem.org/Team:TPHS_San_Diego/Collaborations">Collaborations</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn"> Team </BUTTON><DIV class="dropdown-content"><A style="font-family: 'Montserrat', sans-serif;" href="https://2018.igem.org/Team:TPHS_San_Diego/Team">Team</A><A style="font-family: 'Montserrat', sans-serif;" href="https://2018.igem.org/Team:TPHS_San_Diego/Attributions">Attributions</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn"> Judging </BUTTON><DIV class="dropdown-content"><A style="font-family: 'Montserrat', sans-serif;" href="https://2018.igem.org/Team:TPHS_San_Diego/Medal_Criteria">Medal Criteria</A></DIV></DIV></DIV></DIV><P><TITLE>TPHS IGEM Wiki</TITLE></P><DIV class="content_wrapper">
                Experiments
            </DIV><DIV class="body_wrapper"><NAV><DIV id="nav_content_wrapper"><UL><LI><A href="#assays_clone">Assays and Plasmid Cloning</A></LI><LI><A href="#error_prone_pcr">Error Prone PCR</A></LI><LI><A href="#western_blot">Western Blot</A></LI><LI><A href="#purification">Protein Purification</A></LI></UL></DIV></NAV><SECTION><H1 id="assays_clone">
                    Assays and Plasmid Cloning Protocol
                </H1><P>
                    The Potassium Phosphate buffer, or KPi buffer,  was prepared with 800 mL of distilled water, then 14.135g of K2HPO4 was mixed in, then diluted to 1 liter volume with more distilled water.
</P><P>
In order to prepare colloidal chitin, 20 grams of chitin was measure then added to a 1.5 liter beaker, which was stirred by a magnetic stir bar.150 milliliters of 37% hydrochloric acid was slowly mixed while stirring, left alone to settle for an hour. After which, 850 milliliters of distilled water was added then transferred to be centrifuged at 4,000 rpm for ten minutes. After vortexing to disperse chitin pellets, 400 milliliters of distilled water was added. Repeat centrifuging and vortexing for a total of three times. For the last wash, 100ml instead of 400, then transfer to 250mL centrifuge bottle and spin again with discarding supernatant. This then becomes a 20% solution by adding water, and sterilized via sterilizing.
</P><P>
	For the chitin assay on plate, 4g colloidal chitin, 0.70g K2HPO4, 0.30g KHPO4, .01g MGSO4-5H2O, 0.01gFeSO4-7H2O, 0.0097g FeCl3-6H2O, 0.001g ZnSO4. 0.000844g ZnCl2, and 0.001g MnCl2 were dissolved into 800mL of water, and NaOH was added until it was a pH of 8.0 was reached. 20g of agar was dissolved before autoclaving. 
</P><P>
	To begin the chitinase assay, the colloidal chitin was washed with the KPi buffer, and the CC concentration was 6.0 mg/mL (0.6%; 2xCC). A serial dilution was then performed for chitinase with KPi buffer in a 96-well flat bottom plate, with 50uL per well. After incubation at 30ºC for two hours, the plate
was centrifuged briefly at 4ºC. Then  20μL AAP (1mM), 20μL DCHBS (10mM), 4 μL HRP (200 U/mL), and 56 μL KPi buffer (pH 6.0) was added with a final volume of 200uL, wich was incubated 15 at room temperature.
</P><H1 id="error_prone_pcr">Error Prone PCR Protocol</H1><P>
                    This method of enzyme evolution induces random errors into the genome of GST-ChiA resulting in thousands of different mutations of the ChiA strain transformed into competent B-cells. Then the cells are plated and the colonies with the highest chitin degradation are selected, mutated again, and replated. This method of inducing random mutations and selecting the most efficient expression of enzyme simulates natural selection and essentially evolves the enzyme over the course of several generations. This method, error prone PCR (EP-PCR), is a cheaper alternative to synthesizing DNA sequence mutations, especially for shorter segments of DNA. The sequence of interest is amplified with primers and random mutations are induced using MnCl2. The protocol used in this experiment is based off <A style="padding: 0; margin: 0;" href="https://drive.google.com/file/d/16qbyJAi3pg9WQXS-bcgoTqAkit_DJTPU/view">Wilson et al (2001)</A>, who wrote the protocol based on the EP-PCR written by Cadwell and Joyce in 1994.
                </P><PRE><B>1. Make 2x Buffer+dNTP (1mL)-- one with MnCl2 one without.</B>				

10x Buffer (contain final 1.5mM MgCl2)		        200uL		→	1x
2M MgCl2						3.5uL		→	5mM
1M MnCl2						2uL		→	0.5mM
100mM dATP						4uL		→	0.4mM
100mM dGTP						4uL		→	0.4mM
100mM dCTP						20uL		→	2.0mM
100mM dTTP						20uL		→	2.0mM

Final 1x buffer also contain: 10mM TrisHCl pH8.3; 50mM KCl

<B>2. Setup PCR reaction</B>

Template 1ng		  __uL
Primer mix (10uM)	  10uL		
2x buffer+dNTP	          25uL
Taq			  1uL
H2O		       to 50uL

Total will have 4 conditions: 2 primer mixes; +/- MnCl2
PCR Condition
94C	2’
94C	15”	\
52C	30”	x17
68C	3’	/
68C	6’
12C	**
</PRE><H1 id="western_blot">Western Blot Protocol</H1><P>
                    The first step in western blot is to dilute the SDS PAGE Sample Buffer. Inside this buffer is SDS, which is responsible for creating a uniform charge to mass ratio among proteins as the SDS has a negative charge and binds consistently in 1.4g SDS per 1g protein ratio. This masks the proteins charge and creates a uniform charge to mass ratio, allowing the proteins to be separated solely by its mass. The 2-ME is used to disrupt the disulfide bridges so that the SDS can effectively bind to the proteins.
                </P><P>
                    The pellet (a concentrated blob of bacteria) is placed into the sample buffer and is heated at 95 degrees celsius, lysing the bacteria, which releases its contents, including proteins. SDS is a detergent and helps with this process as it inserts itself between the cell membrane.
                </P><DIV class="img_group"><P>
                        A molecule of SDS detergent.
                    </P></DIV><P>
                    After heating the sample is placed on ice for 5 minutes as the heat can disrupt attractions between the SDS and the proteins. While this is happening, the SDS-PAGE machine will be set up, which includes a tray and a gel suspended in running buffer. After loading, in separate wells, the samples, a negative control, and a protein ladder, which is a standard to base the protein sizes off of, a current is sent through the machine for 2 hours, causing the negatively charged proteins to flow from the anode to cathode. The further the protein travels, the smaller the protein.
                </P><P>
                    After the electrophoresis is finished, the proteins must be transferred onto a membrane so that it can be “blotted” and analyzed. If the membrane is made of PVDF, soak the membrane in methanol as they are both nonpolar. This increases the sensitivity of the membrane by raising its binding capacity. If the membrane is made of nitrocellulose, just wet the membrane with TBST. The proteins are transferred in TBST buffer using electrophoresis.
                </P><P>
                    In the blotting step, a blocking antibody is used to cover the remaining surface on the membrane to reduce background noise (this shows up as random dots where the target protein should not be present). Noise occurs due to the affinity proteins have for the membrane. Although the blotting antibody may have high affinity towards a certain protein, Chitinase-FlagTag in this case, it also has affinity for the membrane, causing it to have off target binding. The third antibody binds to the blotting antibody, marking an area that should not be marked. In short, the blotting step is necessary because it reduces the likelihood of off target binding so that membrane analysis is clear: both the blotting and secondary antibodies are administered to the membrane with a blocking antibody for this purpose.
                </P><P>
                    After some preparatory steps, substrates are added to the membrane which luminesce when reacting with the secondary antibody. The membrane is then exposed to a film in a dark room, like taking a picture. This snapshot of the membrane can be analyzed for protein expression.
                </P><P>
                    After washing away any unbound probes, the western blot can be used to to detect the probes labeled to the protein. Through fluorescent light, the probe can be detected by a photo sensor which can capture the digital image.
                </P><H1 id="purification">Protein Purification</H1><P>
In order to purify protein, there must be bacteria to produce the desired protein, Chitinase. To do so, bacteria is usually inoculated on Luria Broth (LB) and Ampicillin (Amp). LB contains many nutrients for bacterial growth while ampicillin ensures that the bacteria that grow on the plate are transformed as the transformed bacteria contain an Ampicillin resistance gene. Since the ChiA gene is located inside an Ara operon, arabinose is needed in order for the bacteria to produce Chitinase. The bacteria must be incubated on a shaker so that it has produced sufficient protein that can be collected for western blot. Shaking the bacteria ensures that it has access to nutrients and facilitates oxygen exchange. After two hours, the bacteria samples are spun down in a centrifuge and stored at -80C until needed. Freezing the bacteria samples keeps them from growing so that they do not die. 
</P><P>
To extract the protein, a lysis buffer is used to rupture the bacteria cell, the buffer includes protease inhibitors, which block the activity of proteins that may degrade the released proteins, including Chitinase. Other chemicals such as Triton X-100 are added to solubilize proteins. The sample is spun down in a centrifuge to extract only the protein: cellular debris and un-lysed bacteria are left behind. 
</P><P>
The lysate is then used to resuspend GSH beads (mixed with the beads), magnetic beads covered in GSH, which is a substrate for GST, a protein that is grafted onto ChiA when it is translated. This purifies the ChiA as it is the only protein that should have the GST tag, causing it to bind to the GSH beads. After the beads have been resuspended, the beads are washed using a buffer to remove other proteins that may be attached to the beads. When the beads have been washed, a final elution buffer is used to wash off the ChiA, ending purification. A Bradford assay may be used to check for protein concentration: it utilizes Coomassie blue, a dye that changes color when exposed to certain proteins.
</P></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>