<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TPHS_San_Diego_Lab_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TPHS San Diego/Lab Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>TPHS IGEM Wiki</TITLE></P><DIV class="navbar"><A href="#">Home</A><DIV class="dropdown"><BUTTON class="dropbtn">Project
                    </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:TPHS_San_Diego/Project">Description</A><A href="https://2018.igem.org/Team:TPHS_San_Diego/Design">Design</A><A href="https://2018.igem.org/Team:TPHS_San_Diego/Modeling">Modeling</A><A href="https://2018.igem.org/Team:TPHS_San_Diego/Parts">Parts</A><A href="https://2018.igem.org/Team:TPHS_San_Diego/Results">Results</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Notebook
                    </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:TPHS_San_Diego/Lab_Notebook">Lab Notebook</A><A href="https://2018.igem.org/Team:TPHS_San_Diego/Experiments">Experiments</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Human Practices
                    </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:TPHS_San_Diego/Human Practices">Human Practices</A><A href="https://2018.igem.org/Team:TPHS_San_Diego/Collaborations">Collaborations</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Team
                    </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:TPHS_San_Diego/Team">Team</A><A href="https://2018.igem.org/Team:TPHS_San_Diego/Attributions">Attributions</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Judging
                    </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:TPHS_San_Diego/Media Criteria">Medal Criteria</A></DIV></DIV></DIV><DIV class="content_wrapper">
                Lab Notebook
            </DIV><DIV class="body_wrapper"><NAV><DIV id="nav_content_wrapper"><UL><LI><A href="#heading1">link 1</A></LI><LI><A href="#">link 2</A></LI><LI><A href="#">link 3</A></LI><LI><A href="#">sublist link 4</A></LI><LI><A href="#">sublist link 5</A></LI><LI><A href="#">sublist link 6</A></LI></UL></DIV></NAV><SECTION><H1 id="heading1">
                    Day 1
                </H1><P>
Miniprep bacteria with pBAD-D4 (name of the plasmid in which we will be inserting the Chitinase genes, tags, etc.) to isolate the pBAD backbone
Final DNA concentration: 123.7 ng/μL
                </P><H1 id="heading1">
                    Day 2
                </H1><P>
Restriction digest using BamHI and EcoRI to check for bacterial transformation to check to make sure that the plasmid is the expected length (will also send samples for sequencing)
Wells: 1. DNA Ladder. 2. Just EcoRI 3. Just BamHI. 4. No enzyme. 5. Both enzymes

                </P><H1 id="heading1">
                    Day 3
                </H1><P>
                    Made KPi Buffer… (used in Chitinase Assay)
          </P><OL><LI>Prepare 800 mL of dH2O in a suitable container.</LI><LI>Add 2.405 g of K2HPO4 to the solution.</LI><LI>Add 11.73 g of KH2PO4 to the solution.</LI><LI>Add distilled water until volume is 1 L.</LI></OL><H1 id="heading1">
                    Day 4
                </H1><P>
Started Cloning of pBAD-GST-ChiA-FLAG construct. 

(Function of GST and FLAG: these are protein tags (onto chitinase) to purify and detect chitinase respectively)

Did the restriction digest portion, will do gel purification, ligation, and plating tomorrow
For protocol click here

                </P><H1 id="heading1">
                    Day 5
                </H1><P>
Ran gel of restriction digest of pBAD only and did Gel Purification (very straightforward after PCR, you want only the copied DNA) of restriction digest of GST/gBlock as we want to preserve the amount of DNA gBlock that we have and will lose less sample via PCR purification. 
Final concentrations: 
pBAD: 22 ng/μL
gBlock: 10.6 ng/μL
Did DNA ligation and plated BL21 competent cells cloned with GST-ChiA full construct (complete protocol is linked in yesterday’s log)

                </P><H1 id="heading1">
                    Day 6
                </H1><P>
Selected 10 colonies from Vector+Insert plate and put in 4 mL of LB+Ampicillin (LB is nutrients for bacterial growth, Ampicillin assists selection of transformed bacteria) media.
Incubate in 37 ºC shaker for 24 hrs.
Also, did restriction digest and gel on pBAD-D4 vector using MluI and HindIII to check and make sure the enzymes are cutting properly. (If DNA length match expected length, then enzymes are working properly)
	There is a chance we will have to do ligation and stuff again because there aren’t that many colonies. We will most likely use primers to enhance the gBlock/insert DNA and then try again. Depending on how these colonies turn out after we sequence them.
Vector+Insert colony:

                </P><H1 id="heading1">
                    Day 7
                </H1><P>
Did Miniprep of the 10 bacteria colonies that we selected from Wednesday. Used NanoDrop machine to find DNA concentrations of all 10 samples. 
                </P><DIV class="grid-container"><DIV class="grid-item">Sample 1: 97.1 ng/μL</DIV><DIV class="grid-item">Sample 2: 39.6 ng/μL</DIV><DIV class="grid-item">Sample 3: 47.3 ng/μL</DIV><DIV class="grid-item">Sample 4: 47.0 ng/μL</DIV><DIV class="grid-item">Sample 5: 41.8 ng/μL</DIV><DIV class="grid-item">Sample 6: 57.8 ng/μL</DIV><DIV class="grid-item">Sample 7: 33.6 ng/μL</DIV><DIV class="grid-item">Sample 8: 51.7 ng/μL</DIV><DIV class="grid-item">Sample 9: 41.5 ng/μL</DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>