Team:York/ProjectBreakdown
Home
Project
Description
Project Breakdown
Modeling
Parts
Parts
Basic Parts
Composite Parts
Part Collection
Part Guide
Human Practices
Outreach
Survey
Collaborations
Business Plan
Notebooks
Team Notebook
Protocols
About Us
Team Bios
Attributions
Acknowledgements
Sponsors
Project Breakdown
Click to view:
Molecular Biology
Phosphate Assay
Growth Assay
(Click Images to Enlarge)
Molecular Biology
Inorganic phosphate ions enter the periplasm through the PhoE phosphorin.
Once in the Periplasm phosphate can enter the cytoplasm in one of two ways – non-specific (through Pit) or specific transport. For the latter, the substrate binding protein (PstS) of the Phosphate-specific ABC transporter (Pst) binds the phosphate ion.
It then brings the ion in close proximity to the transporter’s transmembrane domains – the permeases PstC and PstA. These two proteins provide the translocation pathway and the phosphate’s uptake is energized by a homodimer of PstB (an ATP-ase protein that hydrolyses ATP).
Phosphate ions can then be used for cellular metabolism which includes ATP formation from ADP and phosphate by the F-ATPase protein. ATP is the substrate for the Polyphosphate kinase enzyme (PPK1) which reversibly transfers the γ-phosphate of ATP onto a chain of polyphosphate (PolyP). Amongst many roles polyphosphate acts as energy and phosphate storage molecule.
Polyphosphate can be used to generate ATP from ADP and in the case of phosphate shortage, the cell releases orthophosphate residues using the Exopolyphosphatase (PPX) enzyme which cleaves a residue at a time from PolyP’s
Gene Sources:
In our project we decided to sample multiple organisms.  The most significant source of genes has been the uncultured species of Candidatus Accumulibacter phosphatis. The known Candidatus Accumulibacter phosphatis strains are considered Polyphosphate-Accumulating Organisms (PAOs). They were first discovered in Activated Sludge performing Enhanced Biological Phosphorus Removal in wastewater treatment facilities. Those organisms are capable of accumulating more than two times higher percentage of their dry weight in phosphorus than their non-PAO counterparts. Unfortunately, Accumulibacter has so far proven unculturable which makes it difficult to examine its genetics and biochemistry. However, metagenomic sequences from enriched cultures have been obtained in recent years and this has enabled the production of phylogenetic analysis and mostly-complete genomic sequence assemblies. We decided to tap into the PAOs potential and systematically test Accumulibacter phosphate-uptake and storage related enzymes. Rather than attempt to clone the genes from Activated sludge, we designed chemically synthesized DNA based on the metagenomic sequence samples available. Below is an image of a flow-cytometry enriched Accumulibacter culture.
Miyauchi, Ryuki, et al. "Diversity of nitrite reductase genes in “Candidatus Accumulibacter phosphatis”-dominated cultures enriched by flow-cytometric sorting." Applied and environmental microbiology 73.16 (2007): 5331-5337.
Another less known organism that we sampled was Sinorhizobium meliloti 2011 strain. The phosphate-specific transporter of S. meliloti has been described as a high-affinity and high velocity transporter(2) (as opposed to 'E.coli's high affinity but low velocity transporter(3). S. meliloti is a root nodule symbiont of legumes. Below is an image of S. meliloti bacteroids in a nodule.
1. Arnold, M. F. F., et al. "Partial complementation of Sinorhizobium meliloti bacA mutant phenotypes by the Mycobacterium tuberculosis BacA protein." Journal of bacteriology 195.2 (2013): 389-398.  2. Yuan, Ze-Chun, Rahat Zaheer, and Turlough M. Finan. "Regulation and properties of PstSCAB, a high-affinity, high-velocity phosphate transport system of Sinorhizobium meliloti." Journal of bacteriology 188.3 (2006): 1089-1102.  3. Rao, N. N., and A. Torriani. "Molecular aspects of phosphate transport in Escherichia coli." Molecular microbiology 4.7 (1990): 1083-1090.
We also extensively sampled the molecular biology workhorse Escherichia coli. Genes that were on our cloning list were the Polyphosphate kinase 1, exopolyphosphatase, Phosphate-specific transporter and PhoE porin (we were unable to clone the latter two genes).
We also had little succes cloning Pseudomonas aeruginosa PAO1's outer membrane porins involved in phosphate and polyphosphate uptake (OprP and OprO).
Phosphate Assay - Measuring orthophosphate and polyphosphate levels
Phosphate assay
The aim of the phosphate assay was to create a reliable method of determining the concentration of phosphate both inside of the cell and within the cell growth media. As a starting point, we used an ABCAM colorimetric assay kit (ab65622) containing a phosphate reagent dye composed of malachite green and ammonium molybdate. A chromogenic complex is formed between the dye and the phosphate ion (PO43-), producing an intense absorption at around 650nm. The kit also contains a 10mM phosphate standard solution, which can be diluted to create a series of standards (protocol can be found on our wiki under phosphate assay protocols.
Our first phosphate assay using the ABCAM kit was run using the wild-type BW25113 Escherichia coli as well as the ∆ppk and ∆ppx knockout strains from the KEIO collection. This assay involved experimenting with the quantity of cells under test and the volume of sample used in each well. From our results we found that the most reliable results were gained using 108 cells.  We also found that phosphate within the cells can be detected and varied between different strains (Figure 1).
Figure 1: A graph to show results from phosphate assay using ABCAM kit comparing two strains from the KEIO collection ∆ppk and ∆ppx and the parent strain BW25113. Error bars show the standard error of the mean from concentrations calculated from technical repeats. Results were measured using a BMG FLUOstar Omega plate reader at OD650 and concentrations calculated from known phosphate concentration standards.
From these results, we proceeded to run a second assay using two strains of ∆ppx (47A7,48A7) and two strains of ∆ppk (5D10, 6D10), along with the parent strain BW25113. Following sonication of the cell samples, we added 50uL of 1 moldm-3 formic acid to hydrolyse the chains of polyphosphate to orthophosphate. This step was added because we suspected that the ABCAM kit detected phosphate ions but not polyphosphate as the assay was only developed for the detection of orthophosphate, as correspondence with ABCAM, stated that the assay had not been used before to detect polyphosphate. Therefore volumes of 10uL, 15uL, 20uL and 100uL of sample were added to each well to determine best dilution for the ABCAM assay. A BMG FLUOstar Omega plate reader was used to measure the absorbance at 650nm and the phosphate concentration in each well was determined using a series of standards with known phosphate concentration.
Results from the previous experiment found that the phosphate concentrations recorded were far lower when the acid hydrolysis step was incorporated (results not shown). This is contrary to our hypothesis that the concentration of phosphate would increase as the polyphosphate in the cells was hydrolysed. This result may have been due to the acid addition and thus resulting in a decrease in pH, affecting the complex formed between the phosphate ions and dye. This result therefore did not confirm our theory that the kit does not detect polyphosphate and therefore, in order to effectively test the ability of the kit to detect polyphosphate; we planned to use glassmilk solution as a filter in our samples to separate the polyphosphate from the rest of the lysate.
Preparing glassmilk solution involved using a fume hood located in the chemistry teaching labs and heating concentrated nitric acid almost to boiling; a 50% water to glassmilk slurry was made and stored at room temperature. As well as using glassmilk to determine cellular polyphosphate concentration, we planned to use NMR to create a spectrum showing two separate peaks for orthophosphate and polyphosphate, of which could have been used to calculate concentration. We used impure DNAse which contains RNAases to digest the DNA and RNA found within our samples. This was because the phosphate backbone in DNA produces a peak in 31P NMR which overlaps with the orthophosphate peak, making the spectrum difficult to analyse. However, from the results shown below in figure 2, the acid hydrolysed the DNA resulting in no overlapping peaks.
Figure 2: An NMR spectrum showing analysis of BW25113 cell lysate with and without DNAse. No polyphosphate peak can be observed, however a clear 31P peak can be seen, indicating the presence of orthophosphate. The results were gained following 18000 runs on a Bruker NMR spectrometer.
Our next assay focussed on altering the polyphosphate assay by finding a way to hydrolyse the polyphosphate chains within our modified cells, without affecting the complex formed between the malachite green, ammonium molybdate and inorganic phosphate. Biological repeats of a BW25113 colony was used, each set of three repeats contained a different volume of 1M formic acid, ranging between 0uL and 100uL. The samples were not neutralised before plating.  The results can be seen in figure 3 below. This assay showed concentrations similar to those calculated in the previous assay; however an observation was that as the volume of acid added increases, apart from peak at lower formic acid amounts, the concentration of phosphate recorded decreases. This therefore indicates that the chromogenic complex formed between the phosphate reagent dye and the orthophosphate is affected by the formic acid by a decrease in pH.
Figure 3: Polyphosphate assay following hydrolysis of polyphosphate chains using varying volumes of 1M formic acid. Error bars are calculated using standard error of the mean cellular phosphate concentrations of technical repeats. Results were measured using a BMG FLUOstar Omega plate reader at OD650 and concentrations calculated from known phosphate concentration standards.
A potential solution to a pH effect was to add 1M NaOH to neutralise the acid hydrolysed solution before adding the phosphate reagent dye. However, a large volume of NaOH was needed to neutralise the solution which created large dilution factors. Figure 4 shows a weak trend between increased formic acid volume and decreased phosphate concentration.  From the results shown in Figure 4, we can see that there is large variation in the phosphate concentrations; we also found that addition of small volumes of acid, resulted in higher phosphate concentrations being detected.
Figure 4: Polyphosphate assay using varying volumes of formic acid and neutralising using 1M NaOH. Error bars are calculated using standard error of the mean. Results were measured using a BMG FLUOstar Omega plate reader at OD650 and concentrations calculated from known phosphate concentration standards.
Following from this, we ran an assay on three colonies of BW25113 using our pre-prepared glassmilk solution. In this assay, we used glassmilk to remove the polyphosphate from the cell lysate. We then tested samples that had been subjected to the glassmilk as well as samples that had not. Our hypothesis was that if the assay did not detect polyphosphate, we would have seen a significant decrease in the levels of phosphate detected when the glassmilk was used. However, if the assay did detect polyphosphate, the phosphate levels would have been roughly the same. From the results presented in the graph below (Figure 5), we found no difference between phosphate concentrations when using or not using glassmilk.
Figure 5: Phosphate assay using glassmilk. Error bars calculated using standard error of the mean concentrations of phosphate. Results were measured using a BMG FLUOstar Omega plate reader at OD650 and concentrations calculated from known phosphate concentration standards
We then modified the acid hydrolysis step of our polyphosphate assay by increasing the concentration of NaOH from 1M to 6M to decrease the extent of dilution during neutralisation.  For this assay, we used the strains BW25113, ∆ppk and ∆ppx of the KEIO collection. From the results shown in the graph below (Figure 6), there is no difference between the cellular phosphate concentrations of ∆ppk , ∆ppx and BW25113 when using different volumes of acid. However, this succeeded in reducing the amount of dilution the samples were subjected to.
Figure 6: Results from phosphate assay using various volumes of 1M formic acid, neutralised with 6M NaOH before plating. Error bars are calculated using standard error of mean. Results were measured using a BMG FLUOstar Omega plate reader at OD650 and concentrations calculated from known phosphate concentration standards.
We then decided to join forces with the growth assay team to analyse the phosphate concentration in the growth media of ∆pstA, ∆pstC, ∆pstB, ∆pstS, ∆phoE, ∆ppk, ∆ppx and BW25113 using our phosphate assay kit. This was with the aim of monitoring the change in phosphate concentration in the media following cell growth. LB was tested however an issue arose where the sample was not diluted sufficiently and a precipitant formed (Figure 7). This therefore altered the OD to produce an unreliable reading.
Figure 7: Image of plate showing precipitant due to high phosphate concentrations indicated by arrows.
Figure 8: Results from phosophate concentration within media after 12 hours. Error bars calculated using standard error of the mean. Results were measured using a BMG FLUOstar Omega plate reader at OD650 and concentrations calculated from known phosphate concentration standards.
From the results shown above in figure 8, we compared the amount of phosphate in the media of each sample.  We found there was no difference between the phosphate concentrations in the growth media of all cell types under test. This might be due to the fact that cells were allowed to reach stationary phase induced by phosphate limitation.
Following from this, we prepared an additional phosphate assay using BW25113, ∆ppk and ∆ppx using the formic acid hydrolysis at 0uL, 1uL and 5uL of formic acid and neutralising using 6M NaOH. Here we brought all sample volumes to an arbitrary value (200uL) to minimise the effect of varying dilution factors. Comparision of 0ul and 1ul of formic acid against 5ul of formic acid showed a significant difference(LSD, p < 0.001). Although the difference between the phosphate concentrations recorded when 0uL and 1uL of formic acid were added were not found to be significant (LSD, p > 0.05).
Figure 9:Results from polyphosphate assay. Error bars calculated using standard error of mean cellular phosphate concentrations of technical repeats. Results were measured using a BMG FLUOstar Omega plate reader at OD650 and concentrations calculated from known phosphate concentration standards. *** indicates p < 0.001. NS indicates no significant difference. Results were measured using a BMG FLUOstar Omega plate reader at OD650 and concentrations calculated from known phosphate concentration standards
From these results we deduced that there was a significant difference between the volume of acid added and the cellular phosphate concentration. We also drew the conclusion that using 5uL may have hydrolysed a larger volume of polyphosphate than lower volumes of acid and is therefore the most reliable volume to use for further assays.
As part of our collaborations with other iGEM teams from around the world, we performed a phosphate assay on water samples that had kindly been sent to our team. To carry this out we used 200uL per well of each sample and used the dye from the ABCAM kit to analyse the phosphate concentration in each sample.
After receiving our constructs, we needed to test the amount of phosphate that they could uptake. To be able to characterise our ppks properly, we transformed our constructs in to  ∆ppk from KEIO collection. In order to do this we decided to test the amount of phosphate up taken by a known amount of cells (figure 10). For this we took the media concentration and grew our cells for six hours, after this we took samples of the media and the cell lysate and analysed their phosphate content. Figure 10 shows the amount of phosphate up taken from the media standardised using cell count (1.0 OD600 - 8x108 cells). However we were unable to conduct the same experiment on other constructs due to the MOPS media not being detected by the phosphate assay. Therefore we present the amount of phosphate present in the media (Figure 13). We also present phosphate amounts within cell lysate using the protocol stated on our wiki in our protocol section (Figure 12 , Figure 14)
Within our results, we tested all ppk variant constructs. Some of our constructs are shown to be effective in accumulating phosphate. Figure 10 and figure 11 show  all three ppks from Candidatus Accumulibacter have shown to be significantly more efficient at collecting phosphate than wild type BW25113. However, some of our constructs were not as efficient at taking up phosphate (Figure 12, Figure 13).
Figure 10. Graph showing the phosphate concentration within  the cell lysate of  10^8, cells for 5 different ppk variants after 6 hours; ; Δppk Ac SK12 - Candidatus Accumulibacter phosphatis SK-12 strain ppk variant, Δppk Ac UW1- Candidatus Accumulibacter phosphatis UW-1 strain ppk variant, Δppk Ac BA91- Candidatus Accumulibacter phosphatis BA-91 strain ppk variant, Δ ppk is Escherichia coli with ppk knockout using kanamycin KEIO collection. BW25113 is the Escherichia coli strain used for the experiment. One way ANOVA was used and overall there was a statistically significant difference between groups and post-hoc LSD was used; NS signifies no statistical significance, *** signifies p<0.001. (ANOVA: F=48.503; d.f. = 14, p<0.001) Results were measured using a BMG FLUOstar Omega plate reader at OD650 and concentrations calculated from known phosphate concentration standards.
Figure 11. Graph showing uptake of phosphate after 6 hours from the media for 5 different ppk variants standardized against cell amount (1 OD600); Δppk Ac SK12 - Candidatus Accumulibacter phosphatis SK-12 strain ppk variant, Δppk Ac UW1- Candidatus Accumulibacter phosphatis UW-1 strain ppk variant, Δppk Ac BA91- Candidatus Accumulibacter phosphatis BA91 strain ppk variant, Δ ppk is Escherichia coli with ppk knockout using kanamycin KEIO collection. BW25113 is Escherichia coli strain used for experiment. One way ANOVA was used and the result showed overall there was statistically significant different. Post-hoc LSD was used; NS signifies no statistical significance, * signifies p<0.05. (ANOVA: F=3.810; d.f. = 14, p<0.05) Results were measured using a BMG FLUOstar Omega plate reader at OD650 and concentrations calculated from known phosphate concentration standards.
Figure 12. Graph showing the phosphate concentration of cell lysate of 10^8, cells for 5 different ppk variants and 1 ppx variant after 6 hours. ; Δppk Ecppk- native Escherichia coli ppk variant, Ecppx- native Escherichia coli ppx variant, Δ ppk is Escherichia coli with ppk knockout using kanamycin KEIO collection, Δppk Koppk - Kingella oralis ppk variant, pKDM12 Escherichia coli ppk variant from Keasling collection. BW25113 is Escherichia coli strain used for experiment. No statistical difference was shown. Results were measured using a BMG FLUOstar Omega plate reader at OD650 and concentrations calculated from known phosphate concentration standards.
Figure 13. Graph showing the  phosphate concentration within the media for 5 different ppk variants and 1 ppx variant after 6 hours . ; Δppk Ecppk- native Escherichia coli ppk variant, Ecppx- native Escherichia coli ppx variant, Δ ppk is Escherichia coli with ppk knockout using kanamycin KEIO collection, Δppk Koppk - Kingella oralis ppk variant, PKDM12 Escherichia coli ppk variant from Keasling collection. BW25113 is Escherichia coli strain used for experiment. One way ANOVA was used and overall there was no statistical difference between strains and post-hoc LSD was used and there was statistical difference between Δppx and BW25113. * signifies there is p< 0.05.(ANOVA: F=1.616; d.f. = 17, p<0.229). Results were measured using a BMG FLUOstar Omega plate reader at OD650 and concentrations calculated from known phosphate concentration standards.
Although we determined our polyphosphate assay unreliable, we tested our constructs’ lysates with 5uL of 1M formic acid. We found that this addition revealed a source of phosphate which is different from 0 uL of formic acid added. Although this may show polyphosphate, this unidentified source could be due to nucleic acids being hydrolysed by the acid. Here we report that with formic acid, we are able to detect other sources of phosphate, but we are unable to say with confidence the origin of these sources without further research.
Figure 14: Graph showing ppk variants’ cell lysate treated with 0uL and 5uL of formic acid. Results were measured using a BMG FLUOstar Omega plate reader at OD650 and concentrations calculated from known phosphate concentration standards.
References
Baba T1, Ara T, Hasegawa M, Takai Y, Okumura Y, Baba M, Datsenko KA, Tomita M, Wanner BL, Mori H (2006) Construction of Escherichia coli K-12 in-frame, single-gene knockout mutants: the Keio collection. Mol Syst Biol .2006.0008. (Feb).p1-11
Visualization of polyphosphate granules in E.coli nucleoids using fluorescent microscopy
Introduction
A widely accepted method for in situ polyphosphate (polyP) detection is fluorescence microscopy using a fluorochrome, 4’, 6 – diamino-2-phenylindole (DAPI) at high concentrations ranging from 3 to 50 µg/ ml1. Interaction of polyP with DAPI shifts the emission maxima of the dye from 456 nm emitting blue fluorescence to 526 nm emitting bright yellow- green fluorescence2,3,4.
Fluorescence microscopy imaging (63x)
Micrographs of immunostained cell coagulates and tissue sections were captured with Nikon  Eclipse  T300  inverted wide  field  microscope  equipped  with  a  plan  apochromat  63x  oil  immersion  objective  (NA 1.4) (Nikon, Tokyo, Japan). The images were acquired using Volocity 6.0 software (PerkinElmer, Inc) on the same day using the same exposure time, which allowed a direct comparison between slides.
Confocal microscopy imaging (100x)
Confocal  imaging  was  performed  using  Zeiss  LSM  710  inverted  confocal  microscope equipped  with  a  plan  apochromat  100x  oil  immersion  objective  (NA  1.4).  The excitation filter was set at 405 nm and emission filter at 526 nm. Image acquisition was performed using Carl Zeiss imaging software ZEN (black edition 2012).
Troubleshooting
The pH of the washing buffer should be 7.0 as the DAPI stain does not react with the polyphosphate granules even at the slight variation of pH. It was also observed that the application of the dye to the specimen at concentrations greater than 25 µg/ ml resulted in only blue light emission making the detection of polyP impossible.
Anticipated results
DAPI-stained polyphosphate granules present in the E.coli nucleoids show bright yellow- green fluorescence at 526 nm upon observation under confocal microscope (Figure 1) whereas the cells devoid of the granules emit a light blue fluorescence at 456 nm.
Figure 1: Detection of polyPgranules in E.coli nucleoids stained with DAPI using fluorescent microscopy. Green signal indicates the presence of polyphosphate granules in the cells of:
A)Chlorella sp. (example from Mukherjee and Ray, 2015); 63x
B)Wild type strain BW25113; 100x
C)Δppk; 100x
D)Δppx; 100x
References
Ryazanova, L. et al. The early stage of polyphosphate accumulation in Saccharomyces cerevisiae: comparative study by extraction and DAPI staining. Adv. Biosci Biotech. 2, 293-297 (2011).
Tijssen, J.P. et al. Localization of polyphosphates in Saccharomyces fragilis, as revealed by 4’,6 – diamidino-2-phenylindole fluorescence. Biochim Biophys Acta. 721, 394-398 (1982).
DAPI Staining Protocol. Laboratory for Experimental Botany, University of Ljubljana, Slovenia. http:// botanika.biologija.org/exp/protocols/DAPI-staining-protocol.pdf (accessed September 18, 2015).
Gunther, S. et al. Dynamics of polyphosphate-accumulating bacteria in wastewater treatment plant microbial communities detected via DAPI (4’,6’ – diamidino-2-phenylindole) and tetracycline labeling. Appl. Environ. Microb.75, 2111-2121 (2009).
Mukherjee C. and Ray Krishna. An improved DAPI staining procedure for visualization of polyphosphate granules in cyanobacterial and microlagal cells. Nature Protocol Exchange (accessed September 18, 2015)
Growth Assay - Measuring Phil's growth
To be able to tell whether Phil was going to work in environments with variable phosphate concentrations, we devised an assay to measure his growth rates. Using the KEIO collection we were able to determine the effect of each gene we were tampering with. Knocking out phosphate transporters exhibits a growth phenotype, and based on modelling results we expected a change in the growth as phosphate metabolism is interfered with.
Growth assays were based on optical density measurements of cell cultures at 650nm over a set period of time, either 7, 24 or 48 hours. The growth rates and curves of multiple phosphate transporter knockout strains have been compared and analysed to find both a chassis for characterisation of genes, as well as to experimentally test for a decrease in cell growth.
Initially we decide to test each of our KEIO collection knockouts, these consisted of the interference of several genes:
ΔpstA - Phosphate ABC transporter permease
ΔpstB - Phosphate ABC transporter, ATPase
ΔpstC - Phosphate ABC transporter permease
ΔpstS - Phosphate ABC transporter periplasmic binding component
ΔphoE - Outer membrane phosphoporin protein E
Δppk - Polyphosphate kinase
Δppx - Exopolyphosphatase
Each gene was tested in triplicate in MOPS media (see fine details ) containing a variety of phosphate concentrations in order to determine how each phosphate transporter reacts. Before carrying out growth assays with these particular mutants we used colony PCR to verify the knockouts. This showed that the mutations were correct in the bacteria.
Figure 1: Picture showing Colony PCR verification of KEIO collection
Name
Sequence
Estimated Tm °C
Size bp
PhoE for
CCGGCAATATTCATTAAAACTGATACGTC
58
29
Phoe rev
ATTCGCGCGTTAATTAAAATCAGGAAT
58
27
ppk for
CCGTGAATAAAACGGAGTAAAAGTGG
58
26
ppk rev
AGGGTTATTCAGGTTGTTCGAGTGA
59
25
ppx for
AATCACTCGAACAACCTGAATAACCC
58
26
ppx rev
AGTATTAAGCGGCGATTTCTGGTGT
60
25
pstA for
AGCAATATCAACCGTGTTTATTCTTCGC
59
28
pstA rev
CTAAGAATGAGGGGGCACGCTAATG
60
25
pstB for
TGAATCAACCGTAACGACCGGTGAT
61
25
pstB rev
GCACGATGAGGAAAAGATTGCAATG
59
25
pstC for
CCATTAGCGTGCCCCCTCATT
60
21
pstC rev
AAACGCGTTTAACTGAAGAGTAACTTATG
57
29
pstS for
TTTATTAGTACAGCGGCTTACCGCT
59
25
pstS rev
ATGAATCCTCCCAGGAGACATTATG
57
25
Table 1. Primers used for PCR verification
Our first growth experiment was to test two knockouts, Δppx and Δppk , against the parent strain BW25113. We did this for 48 hours at concentrations of 1.32mM, 0.1mM and  0 mM phosphate. Overall no real difference was observed in Δppx and Δppk compared to the Parent.
Figure 2: Graphs showing a 48 hour growth assay comparing BW25113, ΔPPK and ΔPPX. Cells were grown in MOPS media at different phosphate concentrations of 1.32mM (Figure 1A)  0.1mM (Figure 1B) and 0mM ( Figure 1C). Overall no significant difference observed. Error bars show standard error of the mean.  Assay was started at 0.01 OD650 and OD650 was measured using a BMG FLUOstar Omega plate reader.
Our next experiment was to consider the difference in growth of knockouts ΔpstA and ΔpstC compared to that of the parent strain. This assay was performed for only 24h hours as we saw that stationary phase was reached at 12 hours and so it was unnecessary to continue for a longer time period. Also we decided to increase the number of different phosphate concentrations used. We tested the strains at 1.32mM, 0.1mM, 0.05mM, 0.01mM and 0mM phosphate concentration. A significant difference in growth phenotype was observed with ΔpstC as well as ΔpstA showing a reduction in growth at 0.05mM. This corresponds as these knockouts might be candidates to characterise our transport constructs which after transformation in these knockouts should restore the wild type phenotype .
Figure 3: Graphs showing a 24 hour growth assay comparing BW25113, ΔpstA and Δ pstC. Cells were grown in MOPS media with different phosphate concentrations of 1.32mM (Figure 2A) , 1.0mM (Figure 2B), 0.1mM (Figure 2C), 0.05mM (Figure 2D), 0.01mM (Figure 2E)  and 0mM (Figure 2F). Error bars show standard error of the mean. Assay was started at 0.01 OD650 and OD650 was measured using a BMG FLUOstar Omega plate reader.
Furthering our tests on the KEIO collection knockouts we performed a growth assay testing ΔpstB and ΔpstS against the parent strain. This assay used the same phosphate concentrations as our previous one and again lasted 24 hours. Overall no real difference was shown.
Figure 4: Graphs showing a 24 hour growth assay comparing BW25113, ΔpstB and Δ pstS. Cells were grown in MOPS media with different phosphate concentrations of 1.32mM (Figure 3A) , 1.0mM (Figure 3B), 0.1mM (Figure 3C), 0.05mM (Figure 3D), 0.01mM (Figure 3E)  and 0mM (Figure 3F). Error bars show  standard error of the mean . Assay was started at 0.01 OD650 and OD650 was measured using a BMG FLUOstar Omega plate reader.
The last of the KEIO collection knockouts to test was ΔphoE. This was tested against parent strain BW25113 for 24 hours in phosphate concentrations of 1mM, 0.1mM, 0.05mM and 0mM. A slight difference between growth phenotypes was observed, however since PhoE is a phosphoporin which is expressed under stress a knockout does correspond to a reduction in growth.
Figure 5: Graphs showing a 24 hour growth assay comparing BW25113 with ΔphoE. Cells were grown in MOPS media with different phosphate concentrations of 1mM (Figure 4A) , 0.1mM (Figure 4B), 0.05mM (Figure 4C) and 0mM (Figure 4D). Standard Error Bars. Assay was started at 0.01 OD650 and OD650 was measured using a BMG FLUOstar Omega plate reader.
From the growth assays using the KEIO collection mutants we decided that the best chassis for characterisation of genes would be ΔpstC, since this is the one that showed the greatest change in growth phenotype from the parent strain. Unfortunatly due to unforseen circumstance we were unable to use the plate reader for the remainder of our experminents. Therefore using this data that we collected we applied a manual version of our growth assay (See Protocols) . Due to the limitations of time we have we were unable to use MOPS as the growth was too slow to see a difference. Therefore we switched to LB.
We then used this growth assay to focus on characterising our transporters, however at the time only Sinorhizobium meliloti pstSCAB could be characterised. Here we tested this construct in triplicate in LB media. The growth assay was performed over 7 hours to capture the growth rate in its exponential phase. Our construct had been put under a promoter induced by IPTG so we decided to test the genes both with and without this due to the stress induced by IPTG. When IPTG is added to the strains there is a significant difference observed between the growth rates (ANOVA, Greenhouse-Geisser: F=12.006; d.f.=1.784; P=0.002). Post hoc LSD test showed a significant difference between the parent strain BW25113 and ΔpstC or ΔpstC+smp however there is no significant difference between ΔpstC and ΔpstC+smpst alone. Similarly when there is no IPTG added to the cells there is a significant difference observed between the growth rate of the strains (ANOVA, Greenhouse-Geisser: F=9.506; d.f.= 1.405 ; P=0.008). Again a post hoc LSD test showed significant difference between BW25113 against ΔpstC or ΔpstC+smpst. When comparing ΔpstC+smpst with IPTG and ΔpstC+smpst without IPTG there is a significant difference in the growth rates being observed (ANOVA Greenhouse-Geisser: F=15.167; d.f.= 1 ; P=0.006).
Figure 6: Graphs showing 7 hour growth assays comparing BW25113, ΔpstC and ΔpstC+smpst. Cell were tested LB without IPTG (Figure 6A) and with IPTG (Figure 6B). Error bars show standard error of the mean. Assay was started at 0.1 OD650
Figure 7. A graph showing a comparison of the growth rates observed in ΔpstC+smpstC with and without IPTG. Assay was started at 0.1 OD650.
Fine Details
What is MOPS media?
Shortened for "3-(N-morpholino) propanesulfonic acid", it has an almost completely neutral pH and can be used as a replacement media for LB, as it doesn't have phosphate in it. This allowed us to measure growth at variable concentrations of phosphate by adding the desired amounts of phosphate.
What is the KEIO [1] collection?
This is biggest collection of knockout strains of Escherichia coli using parent strain BW25113. The knockout strains were created through single-gene deletions of all nonessential genes.
References
Baba T1, Ara T, Hasegawa M, Takai Y, Okumura Y, Baba M, Datsenko KA, Tomita M, Wanner BL, Mori H (2006) Construction of Escherichia coli K-12 in-frame, single-gene knockout mutants: the Keio collection. Mol Syst Biol .2006.0008. (Feb).p1-11
