<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_York_Modeling skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:York/Modeling</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navmenu"><DIV id="navmenubox"><UL id="nav" class="col-md-8"><LI><A href="https://2015.igem.org/Team:York">Home</A></LI><LI><A href="https://2015.igem.org/Team:York/Description"> Project</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:York/Description">Description</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:York/ProjectBreakdown">Project Breakdown</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:York/Modeling">Modeling</A></LI></UL><LI><A href="https://2015.igem.org/Team:York/Parts"> Parts</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:York/Parts">Parts</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:York/Basic_Part">Basic Parts</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:York/Composite_Part">Composite Parts</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:York/Part_Collection">Part Collection</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:York/Data_Page">Part Guide</A></LI><LI><A href="https://2015.igem.org/Team:York/Practices"> Human Practices</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:York/Practices">Outreach</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:York/Survey">Survey</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:York/Collaborations">Collaborations</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:York/Entrepreneurship">Business Plan</A></LI><LI><A href="https://2015.igem.org/Team:York/Notebook"> Notebooks</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:York/Notebook">Team Notebook</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:York/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:York/Team"> About Us</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:York/Team">Team Bios</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:York/Attributions">Attributions</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:York/Acknowledgements">Acknowledgements</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:York/Sponsors">Sponsors</A></LI></DIV></DIV><DIV class="col-md-10 layer"><H1>Modeling</H1><P style="font-size:14px"><I>(Click Images to Enlarge)</I></P><P>To guide the design of our final organism the team has employed the use of Flux Balance Analysis (FBA) modeling. FBA calculates estimates of the rates at which metabolites in an organism will pass through various reaction pathways. As phosphate is used in nearly every part of the cells metabolome, we felt it was important to determine how the accumulation of large amounts might affect the overall function of our organism. Using the <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1911197/">iAF1260 model</A><A href="#ref1"><SUP>[1]</SUP></A> of <I>E. coli</I> as a basis, we first attempted to look at the reactions controlled by the genes we had found as candidates for manipulation during the team's dry lab period. These include, Polyphosphate Kinase (PPK), Exopolyphosphatase (PPX), and the Phosphate Specific Transporters (Pst). One of the immediate problems we found is that phosphate is used by many different parts of the cell, and as such altering any one part has knock on effects on many others. Attempting to alter phosphate pathways requires a fine hand.</P><CENTER><P style="font-size:14px">Figure 1. Metabolic pathway map for <I>E. Coli</I> generated from KEGG<A href="#ref2"><SUP>[2]</SUP></A>. Each of the red nodes represents a metabolic reaction in which phosphate is a major component. <A href="http://www.genome.jp/kegg-bin/show_pathway?scale=0.35&amp;query=phosphate&amp;map=eco01100&amp;scale=0.35&amp;auto_image=&amp;show_description=hide&amp;multi_query=&amp;show_module_list=">View Original</A></P></CENTER><P>The FBA computer model looks for the most efficient, if not the most realistic, pathways to use, meaning it was difficult to force any amount of flux through the pathways we had created to accumulate phosphate. Results from multiple different attempts to achieve theoretical uptake showed that to be able to accumulate phosphate, growth needed to be compromised. By forcing the model to compromise a fraction of it's growth potential we were able to show much higher storage rates. These rates increase in a linear way as further restrictions on growth are put in place, showing direct links between growth and our organisms ability to accumulate phosphate.</P><CENTER><P style="font-size:14px">Figure 2. Graph showing correlation between theoretical growth restriction and accumulation of polyphosphate.</P></CENTER><P>The results of our model allowed us to better direct the project towards the genes that would give us our desired result, and provide a good starting point for optimising the levels of gene expression. The outcomes of the model also point to potential further developments that are currently out of reach due to the iGEM timeframe, but would greatly develop our final product. Some of these ideas are explored in our <A href="https://2015.igem.org/Team:York/Entrepreneurship">business plan.</A></P><P>References</P><UL><LI id="ref1">[1]   Feist AM, Henry CS, Reed JL, et al. A genome-scale metabolic reconstruction for <I>Escherichia coli</I> K-12 MG1655 that accounts for 1260 ORFs and thermodynamic information. <I>Molecular Systems Biology</I>. 2007;3:121. doi:<A href="http://msb.embopress.org/content/3/1/121">10.1038/msb4100155.</A></LI><LI id="ref2">[2]   Kanehisa M, Goto S (2000). &quot;KEGG: Kyoto Encyclopedia of Genes and Genomes&quot;. <I>Nucleic Acids Res</I><B>28</B> (1): 27â€“30. doi:<A href="https://dx.doi.org/10.1093%2Fnar%2F28.1.27">10.1093/nar/28.1.27</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>