<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BGU_Israel_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BGU Israel/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>OriginALS</TITLE></P><TITLE>OriginALS</TITLE><DIV class="topnav" id="myTopnav"><A href="https://2018.igem.org/Team:BGU_Israel" id="brand-img"> </A><DIV id="topnav-right"><A href="https://2018.igem.org/Team:BGU_Israel">Home</A><DIV class="dropdown"><BUTTON class="dropbtn">Project
      </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:BGU_Israel/Description	">Description</A><A href="https://2018.igem.org/Team:BGU_Israel/Design">Design</A><A href="https://2018.igem.org/Team:BGU_Israel/Experiments">Experiments</A><A href="https://2018.igem.org/Team:BGU_Israel/Results">Results</A><A href="https://2018.igem.org/Team:BGU_Israel/Demonstrate">Demonstrate</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Lab
      </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:BGU_Israel/Notebook">Notebook</A><A href="https://2018.igem.org/Team:BGU_Israel/Protocols">Protocols</A><A href="https://2018.igem.org/Team:BGU_Israel/InterLab">InterLab</A><A href="https://2018.igem.org/Team:BGU_Israel/Safety">Safety</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Team
      </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:BGU_Israel/Team">Members</A><A href="https://2018.igem.org/Team:BGU_Israel/Collaborations">Collaborations</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Parts
      </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:BGU_Israel/Parts">Overview</A><A href="https://2018.igem.org/Team:BGU_Israel/Basic_Part">Basic Parts</A><A href="https://2018.igem.org/Team:BGU_Israel/Part_Collection">Collection</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Human Practices
      </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:BGU_Israel/Human_Practices">Integrated Human Practices</A><A href="https://2018.igem.org/Team:BGU_Israel/ALSConference">ALS Conference</A><A href="https://2018.igem.org/Team:BGU_Israel/Public_Engagement">Public Engagement</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Acknowledgements
      </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:BGU_Israel/Attributions">Attributions</A><A href="https://2018.igem.org/Team:BGU_Israel/Sponsors">Sponsors</A></DIV></DIV><A href="javascript:void(0);" style="font-size:15px; " class="icon" onclick="myFunction()">☰</A></DIV></DIV><HEADER style="background-image:url('https://static.igem.org/mediawiki/2018/e/ea/T--BGU_Israel--blue_subhead.png')!important;" class="sub-page-header text-center"><DIV class="container my-auto"><H1 class="animated fadeInUp">Design</H1></DIV></HEADER><SECTION><DIV class="bg-dark"><DIV class="container text-center"><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><H4 style="color:white!important;text-align:center!important;">
Our team spent over six months designing our project to create a concept which would be applicable not only for the iGEM competition, but also would be viable for professional applications. 
</H4></DIV></DIV></DIV></DIV><DIV class="bg-white"><DIV class="container text-center"><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><H1>The plan</H1><P class="justified">
Our hypothesis states that removing reactive astrocytes from the central nervous system (CNS) will slow down the rate of neuron degeneration under reactive gliosis conditions. The proposed therapeutic approach has two main objectives: remove existing reactive astrocytes while simultaneously preventing the creation of new reactive astrocytes. We wish to achieve these objectives by inducing apoptosis in reactive astrocytes and inhibiting the secretion of pro-inflammatory cytokines by microglia.

We decided to inhibit cytokine production through inhibition of the nuclear factor kappa-light-chain-enhancer of activated B cells (NFκB) pathway<SUP>1</SUP>.
We chose to implement this inhibition using the CRISPR-Cas9 system, as this revolutionary tool allows targeted gene editing and the flexibility to test targets simply by adding different gRNAs.
</P><VIDEO width="100%" height="100%" controls=""><P class="justified">
The second system aims to remove reactive astrocytes from the CNS. We decided to achieve this by activating apoptosis, as programmed cell death will prevent further damage to the CNS. 
We chose to induce apoptosis by expressing Caspase-3, whose activation plays a central role in apoptosis. Here we deliberated between activating an endogenic Caspase 3 promoter or inserting an exogenous Caspase 3 under a promoter of our choice.

This version reaches its active conformation without activation through cleavage while the endogenic Caspase3 must undergoes proteolysis in order to activate apoptosis, thus we therefore considered it less controllable and reliable<SUP>2</SUP>. Additionally, using reversed-caspase-3 (rev-caspase-3) gives us complete control over the characteristics of the promoter and affects the effector stage of apoptosis.
The rev-caspase-3 expression will be induced by deactivated Cas9, this time CRISPR system will be used to induce gene expression rather than for gene splicing.
</P></VIDEO></DIV></DIV></DIV></DIV><DIV class="bg-dark"><DIV class="container text-center"><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><H4 style="color:white!important;text-align:center!important;">
A critical aspect of our entire system is specificity, in order to prevent damage to healthy astrocytes and to improve the treatment efficiency. We decided to increase specificity by constructing a three level hierarchy: specific delivery to our target cells, a two-plasmid system, reactive astrocytes specific promoters.
</H4></DIV></DIV></DIV></DIV><DIV class="bg-white"><DIV class="container text-center "><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><H1>It's All About Specificity</H1><P class="justified" style="margin:20px;">
In order to select specific promoters, we turned to the literature and examined RNA sequencing and microarray datasets. We found that the <EM>F4/80</EM> is expressed specifically in macrophage cells. With specific delivery to the CNS, this promoter would target the microglia, which are the macrophage representatives in the CNS. Reactive astrocytes change their expression profile significantly when compared to resting astrocytes<SUP>3</SUP>.  Based on this, we selected pSteap4 and pTimp1, as these genes are highly expressed in reactive astrocytes and expressed in negligible amounts in resting astrocytes and other brain cells such as neurons and microglia<SUP>4-7</SUP>.

The 2015 BGU-iGEM team, Boomerang, has developed a system to specifically target cancer cells. In this system, they split the CRISPRcas9 and its gRNA onto two separate plasmids, meaning that only if both these plasmids enter the target cell and only if both these plasmids are expressed will the system be activated. We decided to build on this previous ingenious strategy and implement a two-plasmid system in our project as well. This technique will increase the probability of the expression of our constructs in our target cells, and decrease chances of off-target activity, thus increasing the safety of any potential therapy derived from this project.

Most prominently, we wished to include a specific delivery system which could be viable and acceptable for administration to patients. After much deliberation we chose to use Adeno-associated virus types 6 and 9, which directly target microglia and astrocytes, respectively. For more info on AAV6 and AAV9 specific delivery and why they are viable for a human medication in the future, visit our <A style="color:#0186ac!important;" data-toggle="tooltip" title="Click to visit" href="https://2018.igem.org/Team:BGU_Israel/Safety">safety page</A>. At this point we had chosen all the components necessary for our two-dimensional therapeutic approach.
</P></DIV></DIV></DIV></DIV><DIV class="bg-grey"><DIV class="container text-center "><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><H1>The Model</H1><P class="justified" style="margin:20px;">
Before commencing with the technical design of our plasmids, we had to decide on a suitable model to work with. This model needed to allow us to stimulate ALS disease conditions that will be also practical based on our available facilities and time constraints. When considering our long-term goals, we examined the notions of both zebra fish and mouse models. We chose to base our design on the mouse models.

Next, we considered primary mouse cells versos cell lines. Although primary cells model ALS conditions more accurately than cell-lines, it is very laborious and, requires expertise that we do not have and it is very time consuming and delicate. Furthermore, although our project focuses on ALS, the therapy we propose is very general as it targets downstream processes common to all cases of ALS (rather than a specific model such as SOD1) and in fact applicable to any condition which exhibits reactive gliosis.

Therefore, murine CNS cell-lines were our choice of models, as they are less time consuming and can be used to model more general reactive gliosis conditions. 
</P></DIV></DIV></DIV></DIV><DIV class="bg-white"><DIV class="container text-center"><DIV class="row"><H1>Microglia Pathway Plasmids</H1><DIV class="col-sm-6 col-sm-offset-1"><P class="justified">The microglia destined plasmids were designed based on a pX601-AAV-CMV::NLS-SaCas9-NLS-3xHA-bGHpA;U6::BsaI-sgRNA as a backbone<SUP>8</SUP>. This backbone expresses <EM>Staphylococcus aureus</EM> CRISPRcass9 (<EM>Sa</EM>Cas9) conjugated under the cytomegalovirus (CMV) promoter, a U6 promoter with a location for gRNA insertion and the SaCas9 scaffold, inverted terminal repeats (ITR) for the AAV 2/6 hybrid, bacterial and mammalian origins of replication (Ori), and Ampicillin resistance<SUP>8</SUP>.

We chose to work with the SaCas9 as it is smaller compared to streptococcus pyogenes Cas9 (SpCas9), making it easier to pack in the target viral vectors.</P></DIV></DIV><DIV class="row"><DIV class="col-sm-10 col-sm-offset-1"><P>We designed 5 guide RNAs (gRNAs) complementary to exons 2, 4, 5 of the IKBKB gene with the purpose of causing a deleterious mutation in the IKBKB gene. We inserted the gRNA sequences under the U6 promoter (a strong RNA polymerase III promoter).
To confer specificity to our system, we cloned the murine pF4/80 promoter and inserted it instead of the CMV promoter.
Next, we added an enhanced green fluorescent protein (EGFP) sequence under a T2A self-cleaving peptide, to allow the EGFP expression in conjugation with the SaCas9, therefore providing a selection method for successful insertion in our cells.
The name of the plasmid is pX601 F4/80 EGFP gRNA (AAV-dCas9 plasmid)</P></DIV></DIV></DIV></DIV></SECTION></DIV><DIV class="bg-grey"><DIV class="container text-center"><DIV class="row"><H1>Astrocyte Pathway Plasmids</H1><DIV class="col-sm-5"><P class="justified">For the astrocyte pathway, we designed two constructs which can be delivered sequentially or simultaneously. 
The backbone of the first plasmid includes the deactivated SaCas9 fused to VPR transcription factors (VP64, p65 and RTA) under pCMV, and AAV2/9 ITR’s. 
First, we replaced the CMV promoter with our chosen promoters pTimp1\pSteap4 and added T2A EGFP. We created two versions of this plasmid (for each promoter), one with VPR which has been shown to be stronger than VP64 alone and the other with VP64 alone, as this transcription factor is sufficient for induction and we were concerned that increased plasmid size would decrease transfection efficiency.</P></DIV></DIV><DIV class="row"><DIV class="col-sm-5  col-sm-offset-1"><P class="justified">The second plasmid of the astrocyte pathway, named pSynt, was designed by us from scratch and synthesized by Integrated DNA Technologies Inc. This construct expresses the sgRNA under pSteap4, complimentary to a synthetic non active promoter, pMLPm.  pMLPm is conjugated to rev-caspase-3 and T2A mCherry, a red fluorescent protein. 
The pMLPm promoter includes a triple repeat sequence, this radically reduces the chances of off target induction, increasing the safety of this construct.
The gRNA sequence is found between two ribozymes, Hammerhead and Hepatitis delta virus (HDV), which cleave the transcribed RNA molecule right at the point of their connection to the gRNA<SUP>9</SUP>. This design allows for gRNAs to be transcribed using RNA polymerase II promoter and then processed into a functioning gRNA. 
</P></DIV></DIV></DIV></DIV></DIV><DIV class="bg-dark"><DIV class="container text-center"><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><H4 style="color:white!important;text-align:center!important;">
We believe that these constructs can be used to implement the two-dimensional ALS therapy we have envisioned!
</H4></DIV></DIV></DIV></DIV><DIV class="bg-white"><DIV class="container text-center "><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><H1>References:</H1><OL><LI>Tak, Paul P., and Gary S. Firestein. &quot;NF-κB: a key role in inflammatory diseases.&quot; <EM>The Journal of clinical investigation</EM>1 (2001): 7-11.‏</LI><LI>Srinivasula, Srinivasa M., et al. &quot;Generation of constitutively active recombinant caspases-3 and-6 by rearrangement of their subunits.&quot; <EM>Journal of Biological Chemistry</EM>17 (1998): 10107-10111.‏</LI><LI>Trakhtenberg, Ephraim F., et al. &quot;Cell types differ in global coordination of splicing and proportion of highly expressed genes.&quot; <EM>Scientific Reports</EM>6 (2016): 32249.‏</LI><LI>Zamanian, Jennifer L., et al. &quot;Genomic analysis of reactive astrogliosis.&quot; <EM>Journal of neuroscience</EM>18 (2012): 6391-6410.‏</LI><LI>Zhang, Ye, et al. &quot;An RNA-sequencing transcriptome and splicing database of glia, neurons, and vascular cells of the cerebral cortex.&quot; <EM>Journal of Neuroscience</EM>36 (2014): 11929-11947.‏</LI><LI>Tokuda, Eiichi, Eriko Okawa, and Shin‐ichi Ono. &quot;Dysregulation of intracellular copper trafficking pathway in a mouse model of mutant copper/zinc superoxide dismutase‐linked familial amyotrophic lateral sclerosis.&quot; <EM>Journal of neurochemistry</EM>1 (2009): 181-191.‏</LI><LI>Lorenzl, S., et al. &quot;Tissue inhibitors of matrix metalloproteinases are elevated in cerebrospinal fluid of neurodegenerative diseases.&quot; <EM>Journal of the neurological sciences</EM>1-2 (2003): 71-76.‏</LI><LI>Ran, F. Ann, et al. &quot;In vivo genome editing using Staphylococcus aureus Cas9.&quot; <EM>Nature</EM>7546 (2015): 186.‏</LI><LI>Gao, Yangbin, and Yunde Zhao. &quot;Self‐processing of ribozyme‐flanked RNAs into guide RNAs in vitro and in vivo for CRISPR‐mediated genome editing.&quot; <EM>Journal of integrative plant biology</EM>4 (2014): 343-349.‏</LI></OL></DIV></DIV></DIV></DIV><DIV id="myModal" class="modal"><SPAN class="close" onclick="modal.style.display = 'none'">×</SPAN></DIV><TITLE>OriginALS</TITLE><FOOTER style="margin-bottom:-10px!important;margin-top:-10px!important;"><DIV class="container my-auto"><DIV class="row"><DIV class="col-md-4"><H3 style="color:white!important;">About Us</H3><P style="color:white!important;text-align: justify!important;">The BGU-iGEM team “OriginALS” hopes to develop an innovative therapeutic approach to prolong the life expectancy of ALS patients, using Synthetic Biology. We are dedicated to promoting ALS awareness and research in Israel through public engagement and educational activities.</P></DIV><DIV class="col-md-4"><UL class="footer-list"><H3>Keep in Touch</H3><LI class="footer-list-item"> @iGEMBGU</LI><LI class="footer-list-item">@igem_2018_bgu</LI><LI class="footer-list-item"> @originalsbgu</LI><LI class="footer-list-item"> originalsbgu@gmail.com</LI></UL></DIV><DIV class="col-md-4"><UL class="footer-list"><H3>Address</H3><LI class="footer-list-item">Ben-Gurion University of the Negev
Ben Gurion 1, Beer Sheva 8410501, Israel</LI></UL></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></BODY></HTML>