<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BGU_Israel_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BGU Israel/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>OriginALS</TITLE></P><TITLE>OriginALS</TITLE><DIV class="topnav" id="myTopnav"><A href="https://2018.igem.org/Team:BGU_Israel" id="brand-img"> </A><DIV id="topnav-right"><A href="https://2018.igem.org/Team:BGU_Israel">Home</A><DIV class="dropdown"><BUTTON class="dropbtn">Project
      </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:BGU_Israel/Description	">Description</A><A href="https://2018.igem.org/Team:BGU_Israel/Design">Design</A><A href="https://2018.igem.org/Team:BGU_Israel/Experiments">Experiments</A><A href="https://2018.igem.org/Team:BGU_Israel/Results">Results</A><A href="https://2018.igem.org/Team:BGU_Israel/Demonstrate">Demonstrate</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Lab
      </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:BGU_Israel/Notebook">Notebook</A><A href="https://2018.igem.org/Team:BGU_Israel/Protocols">Protocols</A><A href="https://2018.igem.org/Team:BGU_Israel/InterLab">InterLab</A><A href="https://2018.igem.org/Team:BGU_Israel/Safety">Safety</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Team
      </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:BGU_Israel/Team">Members</A><A href="https://2018.igem.org/Team:BGU_Israel/Collaborations">Collaborations</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Parts
      </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:BGU_Israel/Parts">Overview</A><A href="https://2018.igem.org/Team:BGU_Israel/Basic_Part">Basic Parts</A><A href="https://2018.igem.org/Team:BGU_Israel/Part_Collection">Collection</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Human Practices
      </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:BGU_Israel/Human_Practices">Integrated Human Practices</A><A href="https://2018.igem.org/Team:BGU_Israel/ALSConference">ALS Conference</A><A href="https://2018.igem.org/Team:BGU_Israel/Public_Engagement">Public Engagement</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Acknowledgements
      </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:BGU_Israel/Attributions">Attributions</A><A href="https://2018.igem.org/Team:BGU_Israel/Sponsors">Sponsors</A></DIV></DIV><A href="javascript:void(0);" style="font-size:15px; " class="icon" onclick="myFunction()">☰</A></DIV></DIV><HEADER style="background-image:url('https://static.igem.org/mediawiki/2018/e/ea/T--BGU_Israel--blue_subhead.png')!important;" class="sub-page-header text-center"><DIV class="container my-auto"><H1 class="animated fadeInUp">Results</H1></DIV></HEADER><DIV class="bg-grey"><DIV class="container"><DIV class="row"><DIV class="col-sm-10 col-sm-offset-1"><H4 style="text-align:center!important;">
Here are our greatest achievements!
Each experiment that we consider a pillar of our project and  essential to its success is marked with a  icon.</H4></DIV></DIV><DIV class="row"><DIV class="col-md-4"><UL class="list-group nav-card"><LI class="list-group-item"><A href="#Technical">Technical</A></LI><LI class="list-group-item"><A style="margin:0 0 0 31px!important;" href="#CalciumPhosphate">Calcium Phosphate Transfection</A></LI><LI class="list-group-item"><A style="margin:0 0 0 31px!important;" href="#Electroporation">Electroporation Transfection</A></LI><LI class="list-group-item"><A style="margin:0 0 0 31px!important;" href="#Reagent">Transfection By Transfection-Reagent</A></LI><LI class="list-group-item"><A href="#" style="color:white!important;">C</A></LI><LI class="list-group-item"><A href="#" style="color:white!important;">C</A></LI></UL></DIV><DIV class="col-md-4"><UL class="list-group nav-card"><LI class="list-group-item item-dark"><A href="#Microglia">Microglia Pathway</A></LI><LI class="list-group-item"><A style="margin:0 0 0 31px!important;" href="#Infection">BV2 infection by Lentivirus</A></LI><LI class="list-group-item"><A href="#qPCR">Cytokines Inhibition Assay</A></LI><LI class="list-group-item"><A href="#" style="color:white!important;">C</A></LI><LI class="list-group-item"><A href="#" style="color:white!important;">C</A></LI><LI class="list-group-item"><A href="#" style="color:white!important;">C</A></LI></UL></DIV><DIV class="col-md-4"><UL class="list-group nav-card"><LI class="list-group-item item-dark"><A href="#Astrocyte">Astrocyte Pathway</A></LI><LI class="list-group-item"><A href="#AstroActive">Astrocyte Activation</A></LI><LI class="list-group-item"><A href="#Promoter">Promoter Assay</A></LI><LI class="list-group-item"><A style="margin:0 0 0 31px!important;" href="#Infection2">C8D30 infection by Lentivirus</A></LI><LI class="list-group-item"><A href="#Caspase3_HEK">pMLPm Activation</A></LI><LI class="list-group-item"><A href="#Caspase3_Astro">Apoptosis Induction</A></LI></UL></DIV></DIV></DIV></DIV><DIV id="Technical" class="bg-dark text-center sub-nav"><DIV class="container"><H1>Technical Experiments</H1><DIV class="row animated fadeInRight"><A href="#CalciumPhosphate" class="btn btn-info" role="button">Calcium Phosphate  Transfection</A><A href="#Electroporation" class="btn btn-info" role="button">Electroporation Transfection</A><A href="#Reagent" class="btn btn-info" role="button">Transfection By Transfection-Reagent</A></DIV></DIV></DIV><DIV id="CalciumPhosphate" class="bg-white"><DIV class="container text-center"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="row"><H1>Calcium Phosphate Transfection</H1><P><STRONG><U>Title</U>: </STRONG>
    Transfection to microglia and astrocytes via “calcium phosphate” method.
</P><P><STRONG><U>Conducted by</U>: </STRONG>
    Sagi Angel
</P><P><STRONG><U>Date</U>: </STRONG>
    3-7.6.18</P><P><STRONG><U>Aim</U>: </STRONG>
    In this experiment we have tried to use the calcium phosphate protocol in
    order to transfect microglia astrocytes and HEK as control in GFP gene. The
    use of HEK cells as control, is due to its ability to be transfected
    relatively easily by the various techniques, including calcium phosphate
    method.</P><P><STRONG><U>Importance</U></STRONG>: This experiment was carried out in parallel with experiments using
    different methods of transfection (different reagents and electroporation)
    in order to find an efficient way of inserting our plasmids into astrocytes
    and microglia for the continuation of the project.</P><P><STRONG><U>Design</U>:</STRONG></P><P> </P><P>1</P><P>2</P><P>3</P><P>Option A</P><P>HEK</P><P>Astrocytes</P><P>Microglia</P><P>Option B</P><P>HEK</P><P>Astrocytes</P><P>Microglia</P><P><STRONG><U>Expectations</U></STRONG><STRONG>:</STRONG></P><P>The expected results were the illumination of GFP under a fluorescent microscope. It could be assumed that if the experiment was conducted well, the HEK cells would be able to insert the GFP plasmid easily and the illumination would be high, and one could hope that some of the experimental cells, astrocytes and microglia, would also show fluorescence. If there was a problem during the experiment or in the quantities of the materials, it would not be possible to see fluorescence in the HEK cells or we would see a high cell mortality rate.</P><P><U><STRONG>Results</STRONG></U><STRONG>:</STRONG></P><OL><LI>All of the cell types survived the transfection process –no significant death rate was noticed</LI><LI>In the HEK cells used as control indeed there was an expression of the GFP gene under the fluorescence microscope the expression wasn’t high</LI><LI>No GFP was detected in the microglia and astrocytes cells in both option of transfection</LI></OL><P>Figure 1 - Cells after transfection with calcium phosphate.
On top cells underregular light and on the bottom the same picture under fluorescence light.
From left to right: HEK, BV2, C8D30.</P><P><STRONG><U>Discussion</U></STRONG><STRONG>:</STRONG></P><P>The purpose of the experiment was to introduce a fluorescent gene into microglia and astrocytes using the HEK cells as control. Because fluorescence illumination is obtained in the control cells, it is assumed that the transduction process worked properly, but the specific cells of the microglia and astrocytes do not respond well to the process in a manner similar to the control cells. It should be noted that the percentage of infection in the control cells was not particularly high, which may indicate a process that is not done optimally or on problems in concentrations of DNA.</P><P><STRONG><U>Conclusion:</U></STRONG></P><P>The process of transfusion using calcium chloride is simple and inexpensive, but does not allow high penetration rates in our target cells - astrocytes and microglia.</P><P>While there was fluorescence illumination in the control cells, it was lower than expected, which means that conditions may be improved to increase efficiency, and this may also could have effect on the target cells.</P><P>Since there are many materials that make up the buffer, it is possible that a change in quantity or lack of accurate preparation may affect the quality of the results.</P><P>It should be concluded that this method, especially in a short period of time that does not allow many adjustments and repetitions, is not suitable for the continuation of work in the project and has indeed been abandoned in favor of the use of ready-made reagents that showed higher efficiency in control cells and astrocytes and microglia.</P><A href="https://2018.igem.org/Team:BGU_Israel/Experiments#CalciumPhosphate" class="btn btn-info" role="button">See Experiment Background</A></DIV></DIV></DIV></DIV><DIV id="Electroporation" class="bg-grey"><DIV class="container text-center"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="row"><H1>Electroporation Transfection</H1><P><B><U>Title</U>: </B>
    Knockout of IKK-β gene in microglia cell-line.
</P><P><U><B>Conducted by</B></U><B>: </B>Einan Farhi and Mor Pasi
</P><P><B><U>Date</U>: </B>15.7.18
</P><P><B><U>Experiment goal and significance</U>:</B>
    The objective is to create a stable cell-line of the BV2 cells with a
    knockout mutation of the Inhibitor of Nuclear Factor kappa-B Kinase subunit
    beta (IKBKB) gene. With which, we aim to demonstrate how targeting the
    Nuclear Factor kappa B (NFκB) pathway will diminish the amount of
    inflammation promoting cytokines produced by the immune representative
    cells of the brain.
</P><P><U><B>Design</B></U>:
</P><P>
    Each transfection mixture was pipetted evenly into 6 wells of a 24
    well-plate. DNA quantities that were used are as following: 2.5, 5, 9 μg of
    DNA. As a positive control, BV-2 cells were transfected with 5 μg pAc-GFP
    and as a negative control cells were electroporated with no plasmid and
    also seeded without electroporation. Here is a schematic diagram of the
    wells mentioned:
</P><DIV align="center"><TABLE border="1" cellspacing="1" cellpadding="0" width="0"><TBODY><TR><TD width="521" colspan="6"><P>
BV2 electroporation with px601-f4/80-g2; 2.5x10<SUP>6</SUP> cells per cuvette; A-030 program
                    </P></TD></TR><TR><TD width="87"><P>
                        2.5 μg DNA
                    </P></TD><TD width="87"><P>
                        2.5 μg DNA
                    </P></TD><TD width="87"><P>
                        2.5 μg DNA
                    </P></TD><TD width="87"><P>
                        2.5 μg DNA
                    </P></TD><TD width="87"><P>
                        2.5 μg DNA
                    </P></TD><TD width="87"><P>
                        2.5 μg DNA
                    </P></TD></TR><TR><TD width="87"><P>
                        5 μg DNA
                    </P></TD><TD width="87"><P>
                        5 μg DNA
                    </P></TD><TD width="87"><P>
                        5 μg DNA
                    </P></TD><TD width="87"><P>
                        5 μg DNA
                    </P></TD><TD width="87"><P>
                        5 μg DNA
                    </P></TD><TD width="87"><P>
                        5 μg DNA
                    </P></TD></TR><TR><TD width="87"><P>
                        9 μg DNA
                    </P></TD><TD width="87"><P>
                        9 μg DNA
                    </P></TD><TD width="87"><P>
                        9 μg DNA
                    </P></TD><TD width="87"><P>
                        9 μg DNA
                    </P></TD><TD width="87"><P>
                        9 μg DNA
                    </P></TD><TD width="87"><P>
                        9 μg DNA
                    </P></TD></TR><TR><TD width="87"><P>
                        No DNA
                    </P></TD><TD width="87"><P>
                        No DNA
                    </P></TD><TD width="87"><P>
                        No DNA
                    </P></TD><TD width="87"><P>
                        No DNA
                    </P></TD><TD width="87"><P>
                        No DNA
                    </P></TD><TD width="87"><P>
                        No DNA
                    </P></TD></TR><TR><TD width="87"><P>
                        No transfection
                    </P></TD><TD width="87"><P>
                        No transfection
                    </P></TD><TD width="87"><P>
                        No transfection
                    </P></TD><TD width="87"><P>
                        No transfection
                 </P></TD><TD width="87"><P>
                        No transfection
                    </P></TD><TD width="87"><P>
                        No transfection
                    </P></TD></TR></TBODY></TABLE></DIV><P><STRONG><U>Expectations</U></STRONG><STRONG>:</STRONG></P><P>We expected to observe a GFP signal in all the wells that were successfully transfected with the px601-f4/80-g2 plasmid, while we did not expect to observe any GFP signal from the wells in the “No DNA” or “No transfection” groups.</P><P><U><STRONG>Results</STRONG></U><STRONG>:</STRONG></P><P>In the following 3 days, the culture was monitored using a fluorescent microscope.</P><P>Unfortunately, no GFP positive cells were observed under the microscope lens. As the positive control showed a successful transfection in the technical sense, we concluded that there is a problem in transfecting this particular cell-line with such a large construct. This is plausible because the transfection efficiency was low even with a small construct such as pAc-GFP  with only ~5% transfection efficiency (Figure 2).</P><P><U>Figure 2</U>: BV-2 cells, transfected with 5 μg pAc-GFP, captured under the microscope with a green fluorescent filter.</P><P><STRONG><U>Discussion</U></STRONG><STRONG>:</STRONG></P><P>As we didn’t manage to observe the GFP in the cells that were transfected with our vector, there was no reason to continue to the next step of the experiment. However, in our discussion, we arrived to an alternative explanation of No GFP result. We suggested that maybe the cells were successfully transfected and had the plasmids in them but maybe the F4/80 promoter was too weakly expressed so that a GFP was below our detection limit.</P><P><STRONG><U>Conclusion</U></STRONG><STRONG>:</STRONG></P><P>The experiment with our plasmid px601-f4/80-g2 (and the rest of our plasmids with the same goal but with different guide sequences) failed. We concluded that continuing with the attempted method and plasmid would be time consuming and we would have to check for many different reasons as of why did the transfection failed. Therefore we made a change in our course of action and decided to try to accomplish the inhibition of the NFκB pathway with a different approach using Lentiviral vectors.</P><A href="https://2018.igem.org/Team:BGU_Israel/Experiments#Electroporation" class="btn btn-info" role="button">See Experiment Background</A></DIV></DIV></DIV></DIV><DIV id="Reagent" class="bg-white"><DIV class="container text-center"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="row"><H1>Transfection by transfection-reagent</H1><P><STRONG><U>Title</U>:</STRONG> Transfection of the C8-D30 astrocytes and BV-2 microglia cell lines with PUC-GFP vector.
</P><P><B><U>Conducted by</U>:</B>Liat Tsoran and Ori Tulchinsky
</P><P><STRONG><U>Date</U>:</STRONG></P><P><STRONG><U>Aim</U>:</STRONG> In this experiment we have tried to use jetPEI-Macrophage and jetPRIME
    transfection reagent in order to transfect microglia cells (BV2) and
    astrocyte cells (C8-D30), respectively, while using HEK-cell as a positive
    control. The use of HEK cell-line as a positive control, is due to its
    transfectability by the various techniques. </P><P><STRONG><U>Importance</U>:</STRONG> This experiment was carried out in parallel with experiments using
    different methods of transfection (different reagents, calcium phosphate
    and electrophoresis) in order to find an efficient way of inserting our
plasmids into microglia for the continuation of the project.    </P><STRONG><U>Design</U>:</STRONG><P><U>Experiment 1 – Microglia (BV2) transfection:</U></P><TABLE><TBODY><TR><TD width="125"><P> </P></TD><TD width="125"><P>1</P></TD><TD width="125"><P>2</P></TD><TD width="125"><P>3</P></TD></TR><TR><TD width="125"><P>A</P></TD><TD width="125"><P>Microglia</P></TD><TD width="125"><P>Microglia</P></TD><TD width="125"><P>Microglia</P></TD></TR><TR><TD width="125"><P>B</P></TD><TD width="125"><P>HEK</P></TD><TD width="125"><P>HEK</P></TD><TD width="125"><P>HEK</P></TD></TR></TBODY></TABLE><P><U>Experiment 2 – Astrocyte (C8-D30) transfection:</U></P><TABLE><TBODY><TR><TD width="125"><P> </P></TD><TD width="125"><P>1</P></TD><TD width="125"><P>2</P></TD><TD width="125"><P>3</P></TD></TR><TR><TD width="125"><P>A</P></TD><TD width="125"><P>Astrocytes</P></TD><TD width="125"><P>Astrocytes</P></TD><TD width="125"><P>Astrocytes</P></TD></TR><TR><TD width="125"><P>B</P></TD><TD width="125"><P>HEK</P></TD><TD width="125"><P>HEK</P></TD><TD width="125"><P>HEK</P></TD></TR></TBODY></TABLE><P><STRONG><U>Expectations</U>:</STRONG></P><P>The expected results were to detect GFP fluorescence under a fluorescent microscope. It could be assumed that if the experiment was conducted well, the HEK cells would be able to harbor the GFP containing plasmid and fluorescence would be high. One could have hoped that some of the cell-lines of our experimetns would also show fluorescence. If there was a problem during the experiment or in the quantities of the materials, it would not be possible to see fluorescence in the HEK cells or we would see a high cell mortality rate.</P><P><STRONG><U>Results</U>:</STRONG></P><OL><LI>All the cell types survived the transfection process –no significant death rate was noticed.</LI><LI>In the HEK cells used as control indeed there was an expression of the GFP.</LI><LI>No GFP was detected in the microglia nor the astrocyte cells in all groups of transfection.</LI></OL><P><STRONG><U>Discussion</U>:</STRONG></P><P>The purpose of the experiment was to introduce a fluorescent gene into microglia and astrocytes using the HEK cells as control. Because fluorescence illumination is obtained in the control cells, it is assumed that the transfection process worked properly, although that microglia and astrocytes did not respond well to the process in a manner similar to the control cells.</P><P><STRONG><U>Conclusion:</U></STRONG></P><P>The process of transfection using jetPEI-Macrophage transfection reagent for microglia cells and jetPRIME for astrocyte cells was unsuccessful. For this reason, we switched to other methods - prior to electrophoresis and when this method was not successful we transferred all our vectors to Lantivirus vectors and we used the Lentivirus infection method.</P><A href="https://2018.igem.org/Team:BGU_Israel/Experiments#Reagent" class="btn btn-info" role="button">See Experiment Background</A></DIV></DIV></DIV></DIV><DIV id="Microglia" class="bg-dark sub-nav"><DIV class="container text-center"><H1>Microglia Experiments</H1><DIV class="row animated fadeInRight"><A href="#Infection" class="btn btn-info" role="button"> BV2 infection</A><A href="#qPCR" class="btn btn-info" role="button">Cytokines Inhibition</A></DIV></DIV></DIV><DIV id="Infection" class="bg-grey"><DIV class="container text-center"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="row"><H1>BV2 Infection using Lentivirus</H1><P><U>Title</U>: Lentivirus infection of BV2 microglia cells.</P><P><U> Conducted by</U>: Einan Farhi, Ori Tulchinsky, Liat Tsoran and Mor Pasi</P><P><U>Date</U>:  30.8.18, 15.9.18</P><P><U>Aim:</U> The objective is to create a stable line of BV2 microglia cells with a genomic insertion of the ShIKKb vector or a stable line with the IkBM vector.</P><P><U>Importance</U>: The insertion of the IκBαM or ShIKK vectors will cause knockdown to the IKKB regulation factor in the NFκB pathway. We aim to demonstrate how targeting the Nuclear Factor kappa B (NFκB) pathway will diminish the amount of inflammation promoting cytokines produced by the immune representative cells of the brain, microglia.</P><P><U>Design</U>:</P><P>Experiment 1: BV2 spinfection with 400 µl IκBαM viral medium.</P><TABLE width="0"><TBODY><TR><TD width="36"> </TD><TD width="83"><P>1</P></TD><TD width="84"><P>2</P></TD><TD width="84"><P>3</P></TD><TD width="84"><P>4</P></TD></TR><TR><TD width="36"><P>a</P></TD><TD width="83"><P>No treatment</P></TD><TD width="84"><P>No treatment</P></TD><TD width="84"><P>No treatment</P></TD><TD width="84"><P>No treatment</P></TD></TR><TR><TD width="36"><P>b</P></TD><TD width="83"><P>IkBM</P></TD><TD width="84"><P>IkBM</P></TD><TD width="84"><P>IkBM</P></TD><TD width="84"><P>IkBM</P></TD></TR><TR><TD width="36"><P>c</P></TD><TD width="83"><P>pGreen-puro</P></TD><TD width="84"><P>pGreen-puro</P></TD><TD width="84"><P>pGreen-puro</P></TD><TD width="84"><P>pGreen-puro</P></TD></TR><TR><TD width="36"><P>d</P></TD><TD width="83"><P>No treatment</P></TD><TD width="84"><P>No treatment</P></TD><TD width="84"><P>No treatment</P></TD><TD width="84"><P>No treatment</P></TD></TR></TBODY></TABLE><P>* Red wells were treated with Puromycin and Blue wells weren't</P><P>HEK293T cell line cultured in 10 cm plates was co-transfected with the IκBαM vector and 3<SUP>rd</SUP> generation Lentivirus helper plasmids. Positive infection control viruses were produced in co-transfection with pGreen-puro vector and 2<SUP>nd</SUP> generation Lentivirus helper plasmids. 48 hours later, viral medium was extracted, filtered and used to infect.</P><P>The BV-2 cell line was infected with the virus immediately after extraction in 24 well-plates using the spinfection method. As positive control for the infection process, we infected cells with pGreen-puro virus with which we previously demonstrated a visible expression of GFP and Puromycin resistance. As negative controls, cells were centrifuged with regular medium (no viral vector).</P><P><U>Results:</U></P><P>Wells which had cells that survived the process in them were transferred to bigger wells to grow and then RNA was extracted for qPCR analysis. We didn't manage to distinguish GFP expression in any wells that were infected with the IκBαM vector but as it was known that the expression is very weak, we went through with the following analysis. BV2 cells that were infected with pGreen-puro were GFP positive with high efficiency of infection so we concluded that the virus production and and infection were done properly in the technical sense.</P><P>Experiment 2: BV2 spinfection with 400 µl Shikkb viral medium</P><P><U>Design</U>:</P><TABLE width="0"><TBODY><TR><TD width="34"> </TD><TD width="104"><P>1</P></TD><TD width="113"><P>2</P></TD><TD width="113"><P>3</P></TD><TD width="123"><P>4</P></TD></TR><TR><TD width="34"><P>a</P></TD><TD width="104"><P>Shikkb</P><P>Not Concentrated</P></TD><TD width="113"><P>Shikkb</P><P>Not Concentrated</P></TD><TD width="113"><P>Shikkb</P><P>Not Concentrated</P></TD><TD width="123"><P>Shikkb</P><P>Not Concentrated</P></TD></TR><TR><TD width="34"><P>b</P></TD><TD width="104"><P>Shikkb</P><P>Concentrated</P></TD><TD width="113"><P>Shikkb</P><P>Concentrated</P></TD><TD width="113"><P>Shikkb</P><P>Concentrated</P></TD><TD width="123"><P>Shikkb</P><P>Concentrated</P></TD></TR><TR><TD width="34"><P>c</P></TD><TD width="104"><P>pGreen-puro</P></TD><TD width="113"><P>pGreen-puro</P></TD><TD width="113"><P>pGreen-puro</P></TD><TD width="123"><P>pGreen-puro</P></TD></TR><TR><TD width="34"><P>d</P></TD><TD width="104"><P>No treatment</P></TD><TD width="113"><P>No treatment</P></TD><TD width="113"><P>No treatment</P></TD><TD width="123"><P>No treatment</P></TD></TR></TBODY></TABLE><P> </P><P>HEK293T cell line cultured in 10 cm plates was co-transfected with the IκBαM vector and 3<SUP>rd</SUP> generation Lentivirus helper plasmids. Positive infection control viruses were produced in co-transfection with pGreen-puro vector and 2<SUP>nd</SUP> generation Lentivirus helper plasmids. 48 hours later, viral medium was extracted, filtered and used to infect. Virus was concentrated using 10% Sucrose solution.</P><P>The BV-2 cell line was infected with the virus immediately after extraction in 24 well-plates using the spinfection method. As positive control for the infection process, we infected cells with pGreen-puro virus with which we previously demonstrated a visible expression of GFP and Puromycin resistance. As negative controls, cells were centrifuged with regular medium (no viral vector).</P><P><U>Results</U>:</P><P>Three days after infection, selection began with 2 µg/ml of Puromycin, changing medium every other day. Wells which had cells that survived the process in them were transferred to bigger wells to grow and then RNA was extracted for qPCR analysis. BV2 cells that were infected with pGreen-puro were GFP positive with high efficiency of infection so we concluded that the virus production and infection were done properly in the technical sense.</P><P><U>Figure 3</U>: BV2 cells infected with control virus, pGreen-Puro under the fluorescent microscope.</P><P><U>Conclusion:</U></P><P>Infection succeed according to positive control (infection of pGreen puro).</P><P>We were able to infect BV-2 cell line with Shikkb.</P><P><U>References:</U></P><OL><LI>ian Liang Wu, Tatsuya Abe, Ryo Inoue, Minoru Fujiki &amp; Hidenori Kobayashi (2004) IκBαM suppresses angiogenesis and tumorigenesis promoted by a constitutively active mutant EGFR in human glioma cells, Neurological Research, 26:7, 785-791</LI><LI>Mori N and Prager D. (1996). Blood, 87, 3410 ± 3417.</LI><LI>Shakhov AN, Collart MA, Vassalli P, Nedospasov SA and Jongeneel CV. (1990). J. Exp. Med., 171, 35 ± 47.</LI><LI>Wang, X. , Li, H. , Xu, K. , Zhu, H. , Peng, Y. , Liang, A. , Li, C. , Huang, D. and Ye, W. (2016), SIRT1 expression is refractory to hypoxia and inflammatory cytokines in nucleus pulposus cells: Novel regulation by HIF‐1α and NF‐κB signaling. Cell Biol Int, 40: 716-726.</LI></OL><P> </P><P> </P><A href="https://2018.igem.org/Team:BGU_Israel/Experiments#Infection" class="btn btn-info" role="button">See Experiment Background</A></DIV></DIV></DIV></DIV><DIV id="qPCR" class="bg-white"><DIV class="container text-center"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="row"><H1>Cytokines Inhibition Assay</H1><P><U>Title</U>: Validation of IKKB knockdown through measurement of cytokine TNFa and IL1a expression in BV2 cells.</P><P><U>Conducted by</U>: Avital Bailen, Daniel Deitch, Mor Sela</P><P><U>Date</U>:  20.9.18-11.10.18</P><P><U>Aim:</U> Quantify the expression of IL1a and TNFa in BV2 microglia cell line infected with the shIKK vector using Lentivirus, using qPCR for cytokines mRNA quantification and ELISA for cytokines quantification.</P><P><U>Importance</U>: We predict that reducing the cytokines, IL1α and TNFα, will prevent the creation of new reactive astrocytes in the brain <SUP>1</SUP>. We predict following the microglia IKKb knockdown will reduce the secretion of the mentioned cytokines and new reactive astrocytes will not be created, preventing further damage to motor neurons.</P><P><U>Design</U>:</P><P>Experiment 1: Quantify the mRNA levels of IL1a and TNFa through qPCR using BV2 cell line infected with the shIKK vector using Lentivirus.</P><TABLE><TBODY><TR><TD colspan="2" rowspan="3" width="66">      </TD><TD colspan="5" width="180"><P>2hr LPS</P></TD><TD colspan="5" width="198"><P>No LPS</P></TD><TD width="60"> </TD></TR><TR><TD colspan="3" width="96"><P>+shIKK</P></TD><TD colspan="2" width="84"><P>WT</P></TD><TD colspan="3" width="108"><P>+shIKK</P></TD><TD colspan="2" width="90"><P>WT</P></TD><TD width="60"><P>No cDNA</P></TD></TR><TR><TD width="30"><P>1</P></TD><TD width="30"><P>2</P></TD><TD width="36"><P>3</P></TD><TD width="42"><P>4</P></TD><TD width="42"><P>5</P></TD><TD width="36"><P>6</P></TD><TD width="36"><P>7</P></TD><TD width="36"><P>8</P></TD><TD width="42"><P>9</P></TD><TD width="48"><P>10</P></TD><TD width="60"><P>11</P></TD></TR><TR><TD rowspan="2" width="66"><P>IL1</P></TD><TD width="24"><P>A</P></TD><TD width="30"> </TD><TD width="30"> </TD><TD width="36"> </TD><TD width="42"> </TD><TD width="42"> </TD><TD width="36"> </TD><TD width="36"> </TD><TD width="36"> </TD><TD width="42"> </TD><TD width="48"> </TD><TD width="60"> </TD></TR><TR><TD width="24"><P>B</P></TD><TD width="30"> </TD><TD width="30"> </TD><TD width="36"> </TD><TD width="42"> </TD><TD width="42"> </TD><TD width="36"> </TD><TD width="36"> </TD><TD width="36"> </TD><TD width="42"> </TD><TD width="48"> </TD><TD width="60"> </TD></TR><TR><TD rowspan="2" width="66"><P>TNF K</P></TD><TD width="24"><P>C</P></TD><TD width="30"> </TD><TD width="30"> </TD><TD width="36"> </TD><TD width="42"> </TD><TD width="42"> </TD><TD width="36"> </TD><TD width="36"> </TD><TD width="36"> </TD><TD width="42"> </TD><TD width="48"> </TD><TD width="60"> </TD></TR><TR><TD width="24"><P>D</P></TD><TD width="30"> </TD><TD width="30"> </TD><TD width="36"> </TD><TD width="42"> </TD><TD width="42"> </TD><TD width="36"> </TD><TD width="36"> </TD><TD width="36"> </TD><TD width="42"> </TD><TD width="48"> </TD><TD width="60"> </TD></TR><TR><TD rowspan="2" width="66"><P>B-Actin</P></TD><TD width="24"><P>E</P></TD><TD width="30"> </TD><TD width="30"> </TD><TD width="36"> </TD><TD width="42"> </TD><TD width="42"> </TD><TD width="36"> </TD><TD width="36"> </TD><TD width="36"> </TD><TD width="42"> </TD><TD width="48"> </TD><TD width="60"> </TD></TR><TR><TD width="24"><P>F</P></TD><TD width="30"> </TD><TD width="30"> </TD><TD width="36"> </TD><TD width="42"> </TD><TD width="42"> </TD><TD width="36"> </TD><TD width="36"> </TD><TD width="36"> </TD><TD width="42"> </TD><TD width="48"> </TD><TD width="60"> </TD></TR></TBODY></TABLE><P> </P><P><U>Expectations</U>:</P><P> </P><P>We expect to see lower mRNA expression of IL1a and TNFa in the +shIKK +LPS samples (columns 1,2,3) when compared to the WT +LPS samples (columns 4,5). This would indicate that the infection was successful and that IKKb was indeed knocked down, leading to a decrease in cytokine secretion. We also expect to see a lower expression of cytokines in the -LPS wells (columns 6-10) when compared to the same treatments with LPS (columns 1-5). The no cDNA column should show no result, as there is no cDNA present to amplify. This column acts as a negative control in order to identify a contamination, if one occurs. Βeta-actin acts as an housekeeping gene, therefore we expect to see similar expression levels in all samples 1-10.</P><P> </P><P><U>Results</U>:</P><P> </P><TABLE width="0"><TBODY><TR><TD width="59"> </TD><TD colspan="5" width="270"><P>2hr LPS</P></TD><TD colspan="5" width="246"><P>No LPS</P></TD><TD width="54"> </TD></TR><TR><TD width="59"> </TD><TD colspan="3" width="162"><P>+shIKK</P></TD><TD colspan="2" width="108"><P>No treatment</P></TD><TD colspan="3" width="144"><P>+shIKK</P></TD><TD colspan="2" width="102"><P>No treatment</P></TD><TD width="54"><P>No cDNA</P></TD></TR><TR><TD width="59"> </TD><TD width="54"><P>1</P></TD><TD width="54"><P>2</P></TD><TD width="54"><P>3</P></TD><TD width="54"><P>4</P></TD><TD width="54"><P>5</P></TD><TD width="48"><P>6</P></TD><TD width="48"><P>7</P></TD><TD width="48"><P>8</P></TD><TD width="48"><P>9</P></TD><TD width="54"><P>10</P></TD><TD width="54"><P>11</P></TD></TR><TR><TD width="59"><P>IL1a</P></TD><TD width="54"><P>4.70</P></TD><TD width="54"><P>3.33</P></TD><TD width="54"><P>5.36</P></TD><TD width="54"><P>2.89</P></TD><TD width="54"><P>2.58</P></TD><TD width="48"><P>0.61</P></TD><TD width="48"><P>1.19</P></TD><TD width="48"><P>0.64</P></TD><TD width="48"><P>0.00</P></TD><TD width="54"><P>NA</P></TD><TD width="54"><P>NA</P></TD></TR><TR><TD width="59"><P>TNF a</P></TD><TD width="54"><P>22.83</P></TD><TD width="54"><P>12.80</P></TD><TD width="54"><P>15.70</P></TD><TD width="54"><P>34.53</P></TD><TD width="54"><P>43.90</P></TD><TD width="48"><P>4.27</P></TD><TD width="48"><P>3.87</P></TD><TD width="48"><P>6.33</P></TD><TD width="48"><P>1.00</P></TD><TD width="54"><P>0.50</P></TD><TD width="54"><P>NA</P></TD></TR></TBODY></TABLE><P> </P><P><U>Discussion</U>:</P><P>The IL1a qPCR results were not reliable. We noticed a high disparity in the given values and Cq curves without the exponential shape. As this experiment was the last attempt, out of several, to quantify mRNA levels of IL1a after using two different primers, we decided to exclude this cytokine from further experiments and continue exclusively with TNFa.</P><P>The TNFa results agreed with our expectations. There is a clear reduction of TNFa between the +LPS +shIKK samples when compared to the +LPS WT samples. This indicates that the shIKK vector successfully induced knockdown of the IKKβ gene. These values were later used with other repetitions to create the graphs shown below.</P><P>We also see lower TNFa expression in each No LPS sample when compared to the same sample with LPS. This indicates that our microglia activation process is indeed successful.</P><P>Experiment Weaknesses:</P><UL><LI>We are missing a negative control of BV2 microglia which underwent the infection process without the plasmid. Without this sample our conclusions are less concrete as we are not able to quantify the effect of the infection process on cytokine secretion.</LI><LI>Not enough biological repetitions for all samples.</LI></UL><P>Experiment Strengths:</P><UL><LI>There was no contamination in this experiment and the technical and biological repetitions indicated no technical errors.</LI></UL><P> </P><P><U>Experiment 2:</U> Quantify the expression of TNFa through qPCR using BV2 cell line infected with the shIKK vector using Lentivirus. With the addition of a negative control of BV2 microglia which underwent the infection process without the plasmid.</P><P><U>Design</U>:</P><P> </P><TABLE><TBODY><TR><TD colspan="2" rowspan="3" width="80"> </TD><TD colspan="9" width="208"><P>    2hr LPS</P></TD><TD colspan="9" width="268"><P>                       No LPS</P></TD><TD width="72"><P> </P></TD></TR><TR><TD colspan="3" width="70"><P>+shIKK</P></TD><TD colspan="3" width="70"><P>-shIKK</P></TD><TD colspan="3"><P>WT</P></TD><TD colspan="3"><P>+shIKK</P></TD><TD colspan="3"><P>-shIKK</P></TD><TD colspan="3"><P>WT</P></TD><TD><P>no cDNA</P></TD></TR><TR><TD><P>1</P></TD><TD><P>2</P></TD><TD width="24"><P>3</P></TD><TD width="24"><P>4</P></TD><TD><P>5</P></TD><TD><P>6</P></TD><TD><P>7</P></TD><TD><P>8</P></TD><TD><P>9</P></TD><TD><P>10</P></TD><TD><P>11</P></TD><TD><P>12</P></TD><TD><P>13</P></TD><TD><P>14</P></TD><TD><P>15</P></TD><TD><P>16</P></TD><TD><P>17</P></TD><TD><P>18</P></TD><TD><P>19</P></TD></TR><TR><TD rowspan="2"><P>TNF</P></TD><TD><P>A</P></TD><TD> </TD><TD> </TD><TD width="24"> </TD><TD width="24"> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD><P>B</P></TD><TD> </TD><TD> </TD><TD width="24"> </TD><TD width="24"> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD rowspan="2"><P>B-Actin</P></TD><TD><P>C</P></TD><TD> </TD><TD> </TD><TD width="24"> </TD><TD width="24"> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD><P>D</P></TD><TD> </TD><TD> </TD><TD width="24"> </TD><TD width="24"> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR></TBODY></TABLE><P> </P><P><U>Expectations</U>:</P><P>We expect to see a low expression of TNFa in the +shIKK +LPS samples (columns 1,2,3) when compared to the -shIKK +LPS (columns  4,5,6). This would indicate that the infection was successful and that IKKb was indeed knocked down, leading to a decrease in cytokine secretion. Additionally, we expect to see lower values of TNFa in the +LPS WT samples (columns 7, 8, 9) when compared to the +LPS -shIKK samples (columns 4,5,6). This would confirm that the spinfection (transfection reagent) process stresses the cells thus inducing increased cytokine secretion.</P><P>We also expect to see lower expression of cytokines in the -LPS wells (columns 10-18) when compared to the same treatments with LPS (columns 1-9).</P><P>The no cDNA column should show no result, as there is no cDNA present to amplify. This column represents a negative control in order to identify a contamination, if one occurs. Beta-actin acts as an housekeeping gene, therefore we expect to see similar expression levels in all samples 1-10.</P><P><U>Results</U>:</P><P>Figure 4: Quantitative qPCR of TNFα mRNA expression in knocked-down IKK (shIKK) microglia BV2 cells.   LPS activated BV2 microglia with and without IKK knockdown (shIKK) showed reduction in TNFα expression in microglia cells treated with shIKK compared to wild-type control cells (WT).  </P><P>The +shIKK +LPS samples expressed less TNFa than the -shIKK +LPS samples. Each sample with LPS expressed more TNFa than the same sample without LPS.</P><P><U>Discussion</U>:</P><P>We were unable to include the -shIKK samples in our conclusions, as the values from these samples were erratic due to technical errors. Therefore, the results of this experiment are inconclusive. Time allowing, this experiment must be repeated.</P><P>Experiment 4: Quantify the expression of TNFa protein levels through ELISA, using supernatant of BV2 cell line infected with the shIKK vector using Lentivirus.</P><P><U>Design</U>:</P><TABLE><TBODY><TR><TD colspan="2" rowspan="3" width="75"> </TD><TD colspan="6" width="129"><P>LPS</P></TD><TD colspan="6" width="146"><P>No LPS</P></TD></TR><TR><TD colspan="3" width="68"><P>+shIKK</P></TD><TD colspan="3" width="61"><P>-shIKK</P></TD><TD colspan="3" width="61"><P>+shIKK</P></TD><TD colspan="3" width="85"><P>-shIKK</P></TD></TR><TR><TD width="23"><P>1</P></TD><TD width="23"><P>2</P></TD><TD width="23"><P>3</P></TD><TD width="20"><P>4</P></TD><TD width="20"><P>5</P></TD><TD width="20"><P>6</P></TD><TD width="20"><P>7</P></TD><TD width="20"><P>8</P></TD><TD width="20"><P>9</P></TD><TD width="28"><P>10</P></TD><TD width="28"><P>11</P></TD><TD width="28"><P>12</P></TD></TR><TR><TD rowspan="3" width="47"><P>TNF</P></TD><TD width="28"><P>A</P></TD><TD width="23"> </TD><TD width="23"> </TD><TD width="23"> </TD><TD width="20"> </TD><TD width="20"> </TD><TD width="20"> </TD><TD width="20"> </TD><TD width="20"> </TD><TD width="20"> </TD><TD width="28"> </TD><TD width="28"> </TD><TD width="28"> </TD></TR><TR><TD width="28"><P>B</P></TD><TD width="23"> </TD><TD width="23"> </TD><TD width="23"> </TD><TD width="20"> </TD><TD width="20"> </TD><TD width="20"> </TD><TD width="20"> </TD><TD width="20"> </TD><TD width="20"> </TD><TD width="28"> </TD><TD width="28"> </TD><TD width="28"> </TD></TR><TR><TD width="28"><P>C</P></TD><TD width="23"><P> </P></TD><TD width="23"><P> </P></TD><TD width="23"><P> </P></TD><TD width="20"><P> </P></TD><TD width="20"><P> </P></TD><TD width="20"><P> </P></TD><TD width="20"><P> </P></TD><TD width="20"><P> </P></TD><TD width="20"><P> </P></TD><TD width="28"><P> </P></TD><TD width="28"><P> </P></TD><TD width="28"><P> </P></TD></TR></TBODY></TABLE><P> </P><P><U>Expectations</U>:</P><P>We expect to see low protein levels of TNFa in the +shIKK +LPS samples (columns 1,2,3) when compared to the -shIKK +LPS (columns 4,5,6). This would indicate that the infection was successful and that IKKb was indeed knocked down, leading to a decrease in cytokine secretion. We also expect to see a lower expression of cytokines in the -LPS wells (columns 7-12) when compared to the same treatments with LPS (columns 1-6), since non-activated microglia should secrete a negligible number of cytokines.</P><P><U>Results</U>:</P><P>Figures 5-6 - ELISA for TNFα expression in knockdown IKK (shIKK) microglia BV2 cells. LPS activated BV2 microglia with and without IKKb knockdown (shIKK) showed reduction in TNFα expression in microglia cells treated with shIKK compared to -shIKK (spinfection, transfection reagent).</P><P>The +shIKK +LPS samples expressed lower TNFa protein than the -shIKK +LPS samples. Additionally,  the -LPS samples expressed TNFa protein in levels lower than the standard curve and therefore are not presented in this graph. Each sample with LPS expressed more TNFa than the same sample without LPS.</P><P>Discussion:</P><P>All of the results agreed with our expectations. There is a clear reduction of TNFa between the +LPS +shIKK samples when compared to the -shIKK +LPS samples. This indicates that the shIKK vector successfully induced knockdown to the IKKβ gene. In addition, the fact that the TNFa protein levels in the absence of induction by LPS were below the standard curve, indicates that our microglia activation process is indeed successful.</P><P>Experimental Weaknesses:</P><UL><LI>Must preform more repetitions to statistically prove our results.</LI></UL><P>Experimental Strengths:</P><UL><LI>Very reliable results since we are measuring protein levels.</LI><LI>ELISA is a very sensitive assay</LI></UL><P> </P><P>References:</P><OL><LI>Liddelow, Shane A., et al. “Neurotoxic reactive astrocytes are induced by activated microglia.”<EM>Nature7638</EM> (2017): 481.</LI></OL><A href="https://2018.igem.org/Team:BGU_Israel/Experiments#qPCR" class="btn btn-info" role="button">See Experiment Background</A></DIV></DIV></DIV></DIV><DIV id="Astrocyte" class="bg-dark sub-nav"><DIV class="container text-center"><H1>Astrocyte Experiments</H1><DIV class="row animated fadeInRight"><A href="#AstroActive" class="btn btn-info" role="button"> Astrocyte Activation</A><A href="#Promoter" class="btn btn-info" role="button">Promoter Assay</A><A href="#Infection2" class="btn btn-info" role="button"> C8D30 Infection</A></DIV><DIV class="row animated fadeInLeft"><A href="#Caspase3_HEK" class="btn btn-info" role="button">pMLPm Activation</A><A href="#Caspase3_Astro" class="btn btn-info" role="button">Apoptosis Induction</A></DIV></DIV></DIV><DIV id="AstroActive" class="bg-grey"><DIV class="container text-center"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="row"><H1>Astrocyte Activation</H1><P><STRONG><U>Title</U>: </STRONG>Activation of astrocyte cell line C8-D30 using lipopolysaccharide (LPS) and pro-inflammatory cytokines.<U><STRONG>Conducted by</STRONG></U><STRONG>: </STRONG>Mor Sela</P><P><STRONG><U>Date</U>: </STRONG>29.7.18-2.8.18</P><P><STRONG><U>Aim:</U></STRONG>Activation of astrocytes was performed to confirm that our C8D30 astrocyte cell line can accurately model resting and reactive astrocytes for our experimental design.</P><P><STRONG><U>Importance</U></STRONG>:Our project is based on the assumption that reactive astrocytes are a main factor in ALS and therefore our product is designed to specifically disarm them. Without a “reactive astrocyte” experimental group, we can not test the efficiency and specificity of our product in reactive astrocytes when compared to other cells in the system.</P><P><STRONG>Design:</STRONG></P><P>Experiment 1 – Measurement of C3 in activated astrocytes using ELISA</P><TABLE style="width: 642px;"><TBODY><TR><TD style="width: 122px;"><P>C8D30 who grow with ACM + LPS (from microglia plate)</P></TD><TD style="width: 122px;"><P>C8D30 who grow with MCM + LPS (from microglia plate) + ACM</P></TD><TD style="width: 122px;"><P>C8D30 who grow with MCM (from microglia plate) without LPS +  ACM <STRONG>Negative control</STRONG></P></TD><TD style="width: 144px;"><P>MCM without Microglia cells + LPS (from microglia plate) +  </P><P>ACM + C8D30</P><P><STRONG>Negative control</STRONG></P></TD><TD style="width: 100px;"><P> </P></TD></TR><TR><TD style="width: 122px;"><P>V</P></TD><TD style="width: 122px;"><P>V</P></TD><TD style="width: 122px;"><P>V</P></TD><TD style="width: 144px;"><P>V</P></TD><TD style="width: 100px;"><P>Biological repetition 1</P></TD></TR><TR><TD style="width: 122px;"><P>V</P></TD><TD style="width: 122px;"><P>V</P></TD><TD style="width: 122px;"><P>X</P></TD><TD style="width: 144px;"><P>      X</P></TD><TD style="width: 100px;"><P>Biological repetition 2</P></TD></TR><TR><TD style="width: 122px;"><P>X</P></TD><TD style="width: 122px;"><P>V</P></TD><TD style="width: 122px;"><P>      X</P></TD><TD style="width: 144px;"><P>      X</P></TD><TD style="width: 100px;"><P>Biological repetition 3</P></TD></TR><TR><TD style="width: 122px;"><P>X</P></TD><TD style="width: 122px;"><P>V</P></TD><TD style="width: 122px;"><P>      X</P></TD><TD style="width: 144px;"><P>      X</P></TD><TD style="width: 100px;"><P>Biological repetition 4</P></TD></TR></TBODY></TABLE><P><STRONG><U>Expectations</U>:</STRONG></P><UL><LI>In negative control 1 the astrocytes were treated with microglia growth medium which did not contain microglia. Then the astrocytes were grown with this medium and with astrocyte growth medium. We expected a negligible C3 concentration in negative control 1.</LI><LI>In negative control 2 the astrocytes were treated with medium from microglia cell culture (without LPS). Then the astrocytes were grown with this medium and with astrocyte growth medium. We expected a negligible C3 concentration in negative control 2.</LI><LI>In sample 1 the astrocytes are treated with medium from microglia cell culture with LPS. Then the astrocytes are grown with this medium and astrocyte medium. We expect a high C3 concentration in sample 1.</LI><LI>In sample 2 the astrocytes are treated with medium from microglia cell culture grown in astrocyte medium with LPS. Then the astrocytes are grown with this medium and more astrocyte medium. We expect a high C3 concentration in sample 2, possibly a different concentration than seen in sample 1 as the microglia are not grown in medium specific to them.</LI></UL><P><STRONG><U>Results</U></STRONG><STRONG>:</STRONG></P><P>The calibration curve is close to linearity with an R<SUP>2</SUP> value &gt;0.99, meaning that the extrapolated results, such as the unknown protein concentration, are statistically reliable.</P><P>The results show a high concentration of C3 in samples 1 and 2. There is no significant difference in C3 concentration between these two samples.</P><P>Conversely, a low C3 concentration were found in negative control 1 and 2.  There is a small increase in C3 in sample 2 over sample 1.</P><P><U>Figure 5</U>: MCM from BV2 cells induces secretion of TNF-α, C1q , and IL1-α that cause to upregulation of C3 in reactive astrocytes. To confirm this, C3 protein levels were quantified in supernatants using ELISA.</P><P><STRONG><U>Discussion</U></STRONG><STRONG>:</STRONG></P><P>This is the first attempt at activating astrocytes and was performed with LPS added to microglia medium. The purpose of this experiment was initial examination of the reactivity level of C8D30 astrocyte cell line.</P><P>The results agree with our expectations. The C3 concentration is lower in the negative controls as opposed to the astrocytes which were activated. These results indicate that the astrocytes were indeed activated and that we can create two experimental groups – resting and reactive astrocytes.</P><P>We must note that the negative controls contained a low concentration of C3, possibly because astrocytes secrete a certain amount of C3 when in a resting state. The difference in C3 concentration displayed in the negative controls does not concur with our expectations. We expected there to be no difference LPS (rather than cytokines) is not supposed to activate astrocytes, and therefore the C3 concentrations should be low and similar in both negative controls. However, this can be explained by the lack of biological repetitions. As only one biological repetition was performed for the negative controls, we can not be sure there was indeed a significant difference in concentration between them.</P><P>Experiment Weaknesses:</P><UL><LI>We are missing a negative control of astrocytes grown in their medium with no treatment. Without this sample we can not make a reliable comparison of C3 between activated and resting astrocytes.</LI><LI>Not enough and unequal biological repetitions for all samples.</LI><LI>Missing additional comparative parameters such as astrocytes activated using commercial cytokines, incubation for different periods of time, etc…</LI><LI>The ELISA kit is very expensive; therefore, we were limited in the repetitions we could perform for this experiment.</LI></UL><P>Experiment Strengths:</P><UL><LI>ELISA is a very precise, sensitive method which produces quantitative results increasing the reliability of these results.</LI></UL><P><STRONG><U>Experiment 2 – Measurement of C3 in activated astrocytes using Western Blot</U></STRONG></P><P><STRONG><U>Design</U></STRONG><STRONG>:</STRONG></P><TABLE width="0"><TBODY><TR><TD width="75"><P><STRONG>8</STRONG></P></TD><TD width="75"><P><STRONG>7</STRONG></P></TD><TD width="73"><P><STRONG>6</STRONG></P></TD><TD width="73"><P><STRONG>5</STRONG></P></TD><TD width="73"><P><STRONG>4</STRONG></P></TD><TD width="73"><P><STRONG>3</STRONG></P></TD><TD width="73"><P><STRONG>2</STRONG></P></TD><TD width="73"><P><STRONG>1</STRONG></P></TD><TD width="79"><P><STRONG>Sample #</STRONG></P></TD></TR><TR><TD width="75"><P>ACM + 3 cytokines + C8D30 after 48hr.</P><P><STRONG> </STRONG></P></TD><TD width="75"><P>ACM + 3 cytokines + C8D30 after 24 hr.</P><P><STRONG> </STRONG></P></TD><TD width="73"><P>ACM + C8D30</P><P> </P><P><STRONG>Negative control</STRONG></P><P> </P></TD><TD width="73"><P>C8D30 who grow with ACM + LPS (from microglia plate) after 48 hr.</P></TD><TD width="73"><P>C8D30 who grow with ACM + LPS (from microglia plate) after 24 hr.</P></TD><TD width="73"><P>C8D30 who grow with MCM + LPS (from microglia plate) after 48 hr.</P></TD><TD width="73"><P>C8D30 who grow with MCM + LPS (from microglia plate) after 24 hr.</P></TD><TD width="73"><P>MCM without LPS + C8D30</P><P> </P><P><STRONG>Negative control </STRONG></P></TD><TD width="79"><P> </P></TD></TR><TR><TD width="75"><P>V</P></TD><TD width="75"><P>V</P></TD><TD width="73"><P>V</P></TD><TD width="73"><P>V</P></TD><TD width="73"><P>V</P></TD><TD width="73"><P>V</P></TD><TD width="73"><P>V</P></TD><TD width="73"><P>V</P></TD><TD width="79"><P>Biological repetition 1</P></TD></TR><TR><TD width="75"><P>V</P></TD><TD width="75"><P>V</P></TD><TD width="73"><P>X</P></TD><TD width="73"><P>V</P></TD><TD width="73"><P>V</P></TD><TD width="73"><P>V</P></TD><TD width="73"><P>V</P></TD><TD width="73"><P>      X</P></TD><TD width="79"><P>Biological repetition 2</P></TD></TR></TBODY></TABLE><P> <STRONG><U>Expectations</U></STRONG><STRONG>:</STRONG></P><UL><LI>In sample 1 the astrocytes are treated with microglia medium which did not contain microglia or LPS. We expect a very faint band in the gel was these should be resting astrocytes which secrete a small amount of C3.</LI><LI>In samples 2 and 3 the astrocytes are treated with medium from microglia cell culture with LPS for 24 and 48 hours, respectively. We expect very bold, thick bands with sample 3 slightly stronger than sample 2.</LI><LI>In samples 4 and 5 the astrocytes are treated with medium from microglia cell culture grown in astrocyte medium with LPS for 24 and 48 hours, respectively. We expect very bold, thick bands as well. The band may be weaker than samples 2 and 3, since the microglia are not grown in their native medium and therefore may not survive as well leading to lower cytokine secretion. Alternately, the band may be stronger than samples 2 and 3 and the astrocytes grow in only their native medium and therefore will survive better and secrete more C3.</LI><LI>In sample 6 the astrocytes are grown in their native medium with out any treatment. Therefore, we expect a vary faint band which will correspond to the amount of C3 secreted by resting astrocytes.</LI><LI>In samples 7 and 8 the astrocytes are treated directly with TNFa, IL1a and C1q cytokines for 24 and 48 hours, respectively. We expect very bold, thick bands in these samples.</LI></UL><P> <STRONG><U>Results</U></STRONG><STRONG>:</STRONG></P><P>The bands for all of the samples appear at 120kD, which is the molecular weight of the alpha subunit of the C3 protein (the long chain)<SUP>1</SUP>. In some of the lanes (7-15) ,there also appear faint bands at the 45kD and 13kD height. These sizes correspond to fragments of C3 which are produced after digestion by β-mercaptoethanol (which is used during western blot analysis – adding a &quot;Sample Buffer&quot;) .</P><P>The bands in lanes 2,3,4,5,6,7,8,14,15 present very similar color and width.</P><P>The two biological repetitions of sample 5 (lanes 9 and 10) do not match, rather they appear to have different color and width.</P><P>Lanes 11 (sample 6), 12, and 13 (sample 7) appear bolder and thicker than the rest of the bands in the gel.</P><P><U>Figure 7</U>: Effects of LPS and three cytokines (IL1a , C1q and TNFa) on activation of C8D30 astrocyte by demonstrating C3 protein levels of reactive astrocytes . Cell supernatant were immunoblotted with C3 antibody after 24 hr. and 48 hr. of activation.</P><P><STRONG><U>Discussion</U></STRONG><STRONG>:</STRONG></P><P>The purpose of this experiment was to examine the level of reactivity in C8D30 astrocyte cell line. Some of the results obtained did not comply with our expectations. We expected bands 2 and 11, which correspond to the negative controls (samples 1 and 6), to appear significantly less bold than the other bands. Instead, the negative control results appear like the experimental samples. Furthermore, band 11 (astrocytes which did not receive any treatment) appears very similar to bands 12 and 13 (astrocytes activated by cytokines).</P><P>Bands 12 and 13 (astrocytes activated by cytokines) do appear as expected, as the bands are very thick and bold when compared to the negative control in band 2 (sample 1).</P><P>The difference in the band strength in lanes 2 and 11 (11 is stronger) indicates that it is preferable to grow astrocytes exclusively in their native medium, rather than in medium intended for microglia.</P><P>The different activation methods do not significantly increase the reactivity of the astrocytes. Possibly, adding the three cytokines increases the secretion of C3 protein in the astrocytes and therefore a stronger band appears in this sample (bands 12 and 13). However, the difference in band width is not significant enough to make concrete conclusions.</P><P>We repeated this experiment several times (5) with similar analysis (western blot) with similar results, therefore we conclude that these results are reliable and not results of technical or human error.</P><P>From these results we must conclude that cell line C8D30 is in a state of reactiveness, without additional induction of activation. Although this does not match the expectations which we based on Liddelow, 2017 these results are not so surprising, as the article is based on work with primary cells while we are working with cell lines.</P><P>In conclusion, we can proceed with experiments on our C8D30 astrocyte cell line as a reactive astrocyte model without performing activation treatments. The problem with these results is that we do not have a resting astrocyte model to work with.</P><P> Experiment Weaknesses:</P><UL><LI>We used astrocyte cell line rather than primary cells which were used in article from which we took our protocol.</LI><LI>LPS was added to microglia cell line to induce cytokine secretion, rather than LPS injection to mouse models as in the article.</LI><LI>Western blot analysis is less precise and sensitive than ELISA. Additionally, the results are qualitative rather than quantitative.</LI></UL><P>Experiment Strengths:</P><UL><LI>The experiment we discussed is representative of four other similar experiments. Therefore, we base our conclusions on many repetitions which indicate that the C8D30 cell line is in a reactive state.</LI></UL><P>Future Plans:</P><UL><LI>GFAP is also considered a marker of A1 reactive astrocytes<SUP>3</SUP>. Therefore, we recommend using both C3 and GFAP as markers to strengthen the reliability of experimental results.</LI><LI>We recommend using ELISA rather than western blot analysis to measure reactivity.</LI><LI>It is recommended to perform immunohistochemistry with markers for reactivity in the specific cell line used.</LI></UL><P> <STRONG><U>Experiment 3 – Identification of reactive astrocytes using immuno-staining</U></STRONG><STRONG>:</STRONG></P><P>The previous methods showed opposing results. The ELISA results indicated that LPS activates C8D30 astrocytes, while the western blot analysis results indicated that LPS does not significantly increase the cell line reactivity. Therefore, we decided to perform a third experiment to confirm whether or not the C8D30 cell line is in a reactive state.</P><P>We organized a collaboration with Dr. Dinorah Friedmann-Morvinski from Tel-Aviv university. The DFM lab specializes in combines cell biology, molecular biology, biochemistry, immunology, cancer research, as well as advanced genome-wide techniques and microscopy. The DFM lab performs immune-staining to confirm reactive states in glioblastoma and therefore were well equipped to assist us with this matter, as our lab does not possess the necessary equipment. The DFM lab performed immune-staining on a sample of our C8D30 astrocyte cell line.</P><P><STRONG><U>Results</U></STRONG><STRONG>:</STRONG></P><P><U>Figure 8</U>: C8D30 cells stained with DAPI (blue), Nestin (green) and GFAP (Red)
Conducted by: Dinorah Friedmann-Morvinski's lab</P><P>GFAP and Nestin, both reactivity markers, were found and appear in the same location as the cells stained with DAPI.</P><P><STRONG><U>Discussion</U></STRONG><STRONG>:</STRONG></P><P>The immune-staining results confirm the results we obtained in through western blog analysis. The C8D30 cell line is always in a reactive state. Apparently, the differences in C3 concentration we found when using ELISA were negligible and would have been rejected with enough repetitions.</P><P><STRONG><U>Conclusion</U></STRONG><STRONG>:</STRONG></P><P>The C8D30 astrocyte cell line is in a reactive state without any additional treatment. In order to support this conclusion, we recommend using ELISA with many biological repetitions and biomarkers.</P><P><STRONG><U>References</U></STRONG><STRONG>:</STRONG></P><OL><LI>Third Component of Complement (C3): Structural Properties in Relation to Functions (topology of functional sites/physiologic C3 fragments/C3-membrane interaction/immune adherence/enzyme subunit) VIKTOR A. BOKISCH, MANFRED P. DIERICH, AND HANS J. MULLER-EBERHARD Department of Molecular Immunology, Scripps Clinic and Research Foundation, La Jolla, California 92037 Contributed by Hans J. Muller-Eberhard, March 10, 1975.</LI><LI>Liddelow, S.A., Guttenplan, K.A., Clarke, L.E., Bennett, F.C., Bohlen, C.J., Schirmer, L., Bennett, M.L., M€unch, A.E., Chung, W.S., Peterson, T.C., et al.(2017). Neurotoxic reactive astrocytes are induced by activated microglia. Nature541, 481–487.</LI></OL><OL start="3"><LI>&quot;Reactive Astrocytes: Production, Function, and Therapeutic Potential&quot;Shane A. Liddelow1,* and Ben A. Barres1,Department of Neurobiology, Stanford University School of Medicine, Stanford, CA 94305, USA,Correspondence: liddelow@stanford.edu (S.A.L.), barres@stanford.edu (B.A.B.),J Neurosci Methods.;1995 May;58(1-2):181-92.</LI></OL><A href="https://2018.igem.org/Team:BGU_Israel/Experiments#Promoter" class="btn btn-info" role="button">See Experiment Background</A></DIV></DIV></DIV></DIV><DIV id="Promoter" class="bg-white"><DIV class="container text-center"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="row"><H1>Promoter Assay</H1><P><STRONG><U>Title</U>: </STRONG>Promoter assay for Timp1 and Steap4 promoters in reactive astrocytes.</P><P><STRONG><U>Conducted by</U>: </STRONG>Nitzan Keidar and Mor Sela</P><P><STRONG><U>Date</U>: </STRONG>24.9.18-28.9.18</P><P><STRONG><U>Aim:</U></STRONG> Our goal in this experiment is to assess the strength and the specificity of the promoters Timp1 and Steap4 by quantifying the amount of luminescence produced by the Luciferase enzyme cloned downstream of these promoters, under our experimental conditions.</P><P><STRONG><U>Importance: </U></STRONG>Our project is based on the assumption that reactive astrocytes can be targeted based on specific genetic markers (e.g Timp1 and Steap4). Non-specific expression can lead to off target activity such as healthy resting astrocytes, microglia or other neighboring brain cells.</P><B>Experiment 1: Promoter assay pTIMP1 and pSTEAP4 in reactive astrocyte cell line (C8D30).</B><P><STRONG><U>Design</U></STRONG><STRONG>:</STRONG></P><TABLE width="0"><TBODY><TR style="height: 48px;"><TD style="height: 48px;" width="121"><P>1. pGL3 + Timp1 &amp; Renilla</P></TD><TD style="height: 48px;" width="121"><P>2. pGL3 + Timp1 &amp; Renilla</P></TD><TD style="height: 48px;" width="121"><P>3. pGL3 + Timp1 &amp; Renilla</P></TD><TD style="height: 48px;" width="98"><P>4. Puc GFP</P></TD><TD style="height: 48px;" width="98"><P>5. Puc GFP</P></TD><TD style="height: 48px;" width="98"><P>6.</P></TD></TR><TR style="height: 48px;"><TD style="height: 48px;" width="121"><P>7. pGL3 + Steap4 &amp; Renilla</P></TD><TD style="height: 48px;" width="121"><P>8. pGL3 + Steap4 &amp; Renilla</P></TD><TD style="height: 48px;" width="121"><P>9. pGL3 + Steap4 &amp; Renilla</P></TD><TD style="height: 48px;" width="98"><P>10. Enhancer E7 + Renilla</P></TD><TD style="height: 48px;" width="98"><P>11. Enhancer E7 + Renilla</P></TD><TD style="height: 48px;" width="98"><P>12. Enhancer E7 + Renilla</P></TD></TR><TR style="height: 48px;"><TD style="height: 48px;" width="121"><P>13. pGL3 no promoter &amp; Renilla</P></TD><TD style="height: 48px;" width="121"><P>14. pGL3 no promoter &amp; Renilla</P></TD><TD style="height: 48px;" width="121"><P>15. pGL3 no promoter &amp; Renilla</P></TD><TD style="height: 48px;" width="98"><P>16. Enhancer E9 + Renilla</P></TD><TD style="height: 48px;" width="98"><P>17. Enhancer E9 + Renilla</P></TD><TD style="height: 48px;" width="98"><P>18. Enhancer E9 + Renilla</P></TD></TR><TR style="height: 35px;"><TD style="height: 35px;" width="121"><P>19. No transfection</P></TD><TD style="height: 35px;" width="121"><P>20. No transfection</P></TD><TD style="height: 35px;" width="121"><P>21. No transfection</P></TD><TD style="height: 35px;" width="98"><P>22.</P></TD><TD style="height: 35px;" width="98"><P>23.</P></TD><TD style="height: 35px;" width="98"><P>24.</P></TD></TR></TBODY></TABLE><P><STRONG><U>Expectations</U></STRONG><STRONG>:</STRONG></P><P>The luminometer values from the Renilla plasmid provide an indication for the efficiency of transfection of the experimental plasmid. The Firefly luciferase values indicate the strength of the tested promoter.</P><P>The promoters of TIMP1(pTIMP1) and STEAP4 (pSTEAP4) were amplified and cloned into the pGL3 plasmid that contains the luciferase reporter gene (Luc). As the prmotoer is cloned proximal to the Luc, it allows to test the activity on the cloned promoter by the amount of luciferase expression. </P><P>In this experiment, wells 1,2, and 3 contain reactive astrocytes co-transfected with pGL3+pTIMP1 and Renilla plasmids. According to the literature, we expect to see high levels of luminescence from the firefly luciferase<SUP>3-6</SUP>.</P><P> Wells 6, 7, and 8 contain reactive astrocytes co-transfected with pGL3+pSTEAP4 and Renilla plasmids. According to the literature, we expect to see high levels of luminescence from the firefly luciferase. This would indicate that the TIMP1 and STEAP4 promoters are expressed strongly in reactive astrocytes (C8D30 cell line)<SUP>3-6</SUP>.</P><P>Wells 12, 13, and 14 contain reactive astrocytes co-transfected with pGL3 without a promoter (empty vector) and Renilla. These wells act as a negative control for this experiment. We expect to see negligible levels of luminescence from the firefly luciferase when compared to the luminescence in wells 1-3 and 6-8, as this pGL3 plasmid does not contain a promoter. The luminometer values of these wells serve as the reference for the background\noise of the experiment and will be used for calculating the relative activity for the tested promoters. </P><P>Wells 4 and 5 act as a positive control to indicate a successful transfection, as they are transfected with Puc-GFP, a vector that can express green florescence protein (GFP). We expect see GFP expression when examining these wells in a fluorescence microscope, which would indicate a successful transfection.</P><P>Wells 9, 10, and 11 contain reactive astrocytes co-transfected with Enhancer E7 and Renilla. Wells 15, 16, and 17 contain astrocytes co-transfected with Enhancer E9 and Renilla. Enhancer E7 is an active enhancer in HEK293 that was cloned into a pGL4.23 that contains Luc and Enhancer E9 is a negative  enhancer in HEK293 cells. These wells act as a negative control for reactive astrocyte cells as the enhancers are specific for HEK293 cells, yet we have transfected them into astrocytes. Therefore, we expect low values of firefly luciferase luminescence.</P><P>Wells 18, 19, and 20 contain reactive astrocyte cells which did not undergo transfection. These wells act as a luminescence baseline for each transfected cells and to validate that our experiment is not contaminated with plasmids. We expect to see low levels of luminescence both for Renilla and firefly luciferase in these wells.</P><P>In order to increase the transfection efficiency, we preformed this experiment twice with and without trypsin. The use of trypsin should enlarge the cell surface area, therefore improving the transfection efficiency. Therefore, we used both options in order to test the promoter activity. </P><P><U><STRONG>Results</STRONG></U><STRONG>:</STRONG></P><P>We found that the pTIMP1 and pSTEAP4 values are very high when compared to the negative controls of the empty vector. The values obtained from the E7 and E9 enhancers are also very low. We received reliable results both for the experiment with and without the trypsin.</P><P><U>Figure 9</U>: <STRONG>The activity of</STRONG><STRONG>TIMP</STRONG>1<STRONG> and STEAP4 promoters in </STRONG><STRONG>reactive astrocyte cell line (C8D30)</STRONG>. Luciferase reporter assay demonstrating transcriptional activation mediated by promoters of TIMP1 and STEAP4 in C8D30 cells. The results show cells that got no treatment before transfection and cells were treated with trypsin before the transfection. The <EM>TIMP1 promoter</EM> showed 27.5-fold increased activity without trypsin and 17.07- fold increased activity with trypsin, as compared to the control (empty vector). The <EM>STEAP4 promoter</EM> showed 8.14-fold increased activity without trypsin and 9.67- fold increased activity with trypsin, as compared to the control. Enhancers 7 that serves as a negative control for Luciferase activity in C8D30 showed 4.74-fold increased activity without trypsin and 1.29- fold increased activity with trypsin, as compared to the control. Enhancers 9 that serves as a negative control for Luciferase activity in C8D30 showed decreased activity, as compared to the control. Relative luciferase expression results are presented after normalization to Renilla activity and represent the means ± standard deviation of three independent experiments.</P><P><U>Discussion</U>:</P><P>As expected, the pTIMP1 and pSTEAP4 values are very high when compared to the negative controls of empty vector, which suggest that our two promoter are active in our reactive astrocyte C8D30 cell line. We also found that the promoter are active both with and without trypsin, which support our conclusion.  Additionally, the values obtained from the E7 and E9 enhancers are very low, as expected, since these enhancers are specific for HEK293 cells and not reactive astrocytes.</P><P>As we preformed this experiment on reactive astrocytes, we would be interested to test these promoters on normal resting astrocytes and after they become reactive. Therefore, our next step would be to perform this experiment on primary astrocyte cells. This way we could compare the promoter expression in resting v. reactive astrocytes and strengthen the assessment that STEAP4 and TIMP1 are specific to reactive astrocytes over resting astrocytes. We plan to preformed this experiment on mouse primary astrocytes but the plasmids that we generated are without the AAV9 elements that required for infection such plasmids into primary astrocytes, therefore, we could not do this experiment.</P><P><U>Conclusion:</U></P><P>According to the literature and this experiment, STEAP4 and TIMP1 promoters are indeed expressed strongly in our reactive astrocytes. Therefore, while working with the cell line we can rely on these promoters to express our constructs specifically in reactive astrocytes. However, further experiments with a resting astrocyte sample must be performed in order to further confirm that these two promoters are active only in reactive astrocytes, but not in resting one.</P><P><STRONG>Experiment 2: promoter assay of pF4/80 in C8D30 astrocytes</STRONG></P><P>As our approach is based on cell type specific activity of the promoter, we aim to test whether the pF4/80 promoter that is active promoter in microglia, will be active in astrocytes. We expected that this promoter will not be active in reactive astrocytes and only in microglia cells</P><P><U>Design</U>:</P><TABLE width="0"><TBODY><TR><TD colspan="3" width="347"><P style="text-align: center;">No trypsin</P></TD><TD colspan="3" width="347"><P style="text-align: center;">Trypsin</P></TD></TR><TR><TD width="116"><P>1. pGL3 + pF4/80 &amp; Renilla</P></TD><TD width="116"><P>2. pGL3 + pF4/80 &amp; Renilla</P></TD><TD width="116"><P>3. pGL3 + pF4/80 &amp; Renilla</P></TD><TD width="116"><P>4. pGL3 + pF4/80 &amp; Renilla</P></TD><TD width="116"><P>5. pGL3 + pF4/80 &amp; Renilla</P></TD><TD width="116"><P>6. pGL3 + pF4/80 &amp; Renilla</P></TD></TR><TR><TD width="116"><P>7.  Empty pGL3 &amp; Renilla</P></TD><TD width="116"><P>8.  Empty pGL3 &amp; Renilla</P></TD><TD width="116"><P>9.  Empty pGL3 &amp; Renilla</P></TD><TD width="116"><P>10. Empty pGL3 &amp; Renilla</P></TD><TD width="116"><P>11.  Empty pGL3  &amp; Renilla</P></TD><TD width="116"><P>12.  Empty pGL3 &amp; Renilla</P></TD></TR><TR><TD width="116"><P>13. No transfection</P></TD><TD width="116"><P>14. No transfection</P></TD><TD width="116"><P>15. No transfection</P></TD><TD width="116"><P>16. No transfection</P></TD><TD width="116"><P>17. No transfection</P></TD><TD width="116"><P>18. No transfection</P></TD></TR></TBODY></TABLE><P><U>Expectations</U>:</P><P>The promoter of F4/80(pF4/80) was amplified and cloned into the pGL3 plasmid that contains the luciferase reporter gene (Luc). As the promoter is cloned proximal to the Luc, it allows to test the activity on the cloned promoter by the amount of luciferase expression. </P><P>In this experiment, wells 1,2, 3, 4, 5 and 6 contain reactive astrocytes co-transfected with pGL3+pF4/80 and Renilla plasmids. According to the literature, we expect to see insignificant levels of luminescence from the firefly luciferase, if any This result would indicate that the F4/80 promoter is not expressed in reactive astrocytes (C8D30 cell line)<SUP>1</SUP>.</P><P>Wells 7, 8, 9, 10, 11 and 12 contain reactive astrocytes co-transfected with pGL3 without a promoter (empty vector) and Renilla. These wells serve as a negative controls for this experiment. We expect to see negligible levels of luminescence from the firefly luciferase when compared to the luminescence in wells 1-6, as this pGL3 plasmid does not contain a promoter. The luminometer values of these wells serve as the reference for the background\noise of the experiment and will be used for calculating the relative activity for the tested promoters. </P><P>Wells 13, 14, 15, 16, 17 and 18 contain reactive astrocyte cells which did not undergo transfection. These wells act as a luminescence baseline for each transfected cells and to validate that our experiment is not contaminated with plasmids. We expect to see low levels of luminescence both for Renilla and firefly luciferase in these wells.</P><P>In order to increase the transfection efficiency, we preformed this experiment twice with and without trypsin. The use of trypsin should enlarge the cell surface area, therefore improving the transfection efficiency. Therefore, we used both options in order to test the promoter activity. </P><P><U>Results</U>:</P><P>We found that pF4/80  can drive luciferase expression in about 3 fold compare to the empty vector control samples, which indicate that this promoter could drive expression of gene/s.</P><P><U>Figure 10</U>: <STRONG>The activity of</STRONG><STRONG>F4/80 promoter in </STRONG><STRONG>reactive astrocyte cell line (C8D30)</STRONG>. Luciferase reporter assay demonstrating transcriptional activation mediated by promoters of F4/80 in C8D30 cells. The results show cells that got no treatment before transfection and cells were treated with trypsin before the transfection. The <EM>F4/80 promoter</EM> showed 3.05-fold increased activity without trypsin and 2.56- fold increased activity with trypsin, as compared to the control (empty vector). Relative luciferase expression results are presented after normalization to Renilla activity and represent the means ± standard deviation of three independent experiments.</P><P><U>Discussion</U>:</P><P>As we found that the pF4/80 promoter is active in our reactive astrocytes cell line, it suggests that this promoter can be active both in reactive astrocytes and microglia cells.  Another explanation for the activity of this promoter could be that used murine astrocyte cell-line, when we based our experiment on the literature, where this promoter was tested on primary cells, suggesting that it might not be active in primary cells. In summary, we need to take under consideration that this promoter could be active when we use this promoter in our model system.</P><P><STRONG>Experiment 3: Promoter Assay pTIMP1 and pSTEAP4 and pF4/80 in HEK293 cells.</STRONG></P><P>In order to test the cell specific activity of the promoters, we plan to use HEK293 to perform the promoter assay and examine their activity. We used HEK293 cells because they are from human embryonic kidney and therefore are not relate to neuronal cells or microglia. In addition, these cells are easy to transfect which suggest that this assay will be easy to perform.</P><P><U>Design</U>:</P><TABLE width="0"><TBODY><TR><TD width="104"><P>1. pGL3 + pTIMP1 &amp; Renilla</P></TD><TD width="104"><P>2. pGL3 + pTIMP1 &amp; Renilla</P></TD><TD width="104"><P>3. pGL3 + pTIMP1 &amp; Renilla</P></TD><TD width="84"><P>4. pGL3 + pF4/80 &amp; Renilla</P></TD><TD width="84"><P>5. pGL3 + pF4/80 &amp; Renilla</P></TD><TD width="84"><P>6. pGL3 + pF4/80 &amp; Renilla</P></TD></TR><TR><TD width="104"><P>7. pGL3 + pSTEAP4 &amp; Renilla</P></TD><TD width="104"><P>8. pGL3 + pSTEAP4 &amp; Renilla</P></TD><TD width="104"><P>9. pGL3 + pSTEAP4 &amp; Renilla</P></TD><TD width="84"><P>10. Enhancer E7 + Renilla</P></TD><TD width="84"><P>11. Enhancer E7 + Renilla</P></TD><TD width="84"><P>12. Enhancer E7 + Renilla</P></TD></TR><TR><TD width="104"><P>13. Empty pGL3 &amp; Renilla</P></TD><TD width="104"><P>14. Empty pGL3 &amp; Renilla</P></TD><TD width="104"><P>15. Empty pGL3 &amp; Renilla</P></TD><TD width="84"> </TD><TD width="84"> </TD><TD width="84"> </TD></TR><TR><TD width="104"><P>16. No transfection</P></TD><TD width="104"><P>17. No transfection</P></TD><TD width="104"><P>18. No transfection</P></TD><TD width="84"><P>19. Puc GFP</P></TD><TD width="84"><P>20. Puc GFP</P></TD><TD width="84"> </TD></TR></TBODY></TABLE><P><U>Expectations</U>:</P><P>The promoters of TIMP1(pTIMP1), STEAP4 (pSTEAP4), F4/80 (pF4/80) were amplified and cloned into the pGL3 plasmid that contains the luciferase reporter gene (Luc). As the promoter is cloned proximal to the Luc, it allows to test the activity on the cloned promoter by the amount of luciferase expression.</P><P>In this experiment, wells 1,2 and 3 contain HEK293 cells co-transfected with pGL3+pTIMP1 and Renilla plasmids. According to the literature, we expect to see very low levels of luminescence from the firefly luciferase<SUP>4</SUP>.</P><P>Wells 4, 5, and 6 contain HEK293 cells co-transfected with pGL3+pF4/80 and Renilla plasmids. We did not have background information regarding the expression of pF4/80 in HEK293 cells but we know this gene is specific for macrophage cells, therefore we did not anticipate for pF4/80 expression in these cells.</P><P>Wells 7, 8, and 9 contain HEK293 cells co-transfected with pGL3+pSTEAP4 and Renilla plasmids. According to the literature, we didn’t expect to see any levels of luminescence from the firefly luciferase <SUP>1</SUP>.</P><P>Wells 10, 11, and 12 contain HEK293 cells co-transfected with Enhancer E7 and Renilla plasmids<SUP>4</SUP>. Enhancer E7 is an active enhancer in HEK293 that was cloned into a pGL4.23 that contains Luc. These wells act as a positive control for HEK293 cells as the enhancers are specific for HEK293 cells, Therefore, we expect a suffiient values of firefly luciferase luminescence.</P><P>Wells 13, 14, and 15 contain HEK293 cells co-transfected with pGL3 without a promoter (empty vector) and Renilla. These wells act as a negative control for this experiment. We expect to see negligible levels of luminescence from the firefly luciferase, as this pGL3 plasmid does not contain a promoter. The luminometer values of these wells serve as the reference for the background\noise of the experiment and will be used for calculating the relative activity for the tested promoters. </P><P>Wells 16, 17, and 18 contain HEK293 cells which did not undergo transfection. These wells act as a luminescence baseline for each transfected cells and to validate that our experiment is not contaminated with plasmids. We expect to see low levels of luminescence both for Renilla and firefly luciferase in these wells.</P><P>Wells 19 and 20 act as a positive control to indicate a successful transfection, as they are transfected with Puc-GFP, a vector that can express green florescence protein (GFP). We expected to see GFP expression when examining these wells in a fluorescence microscope, which would indicate a successful transfection..</P><P><U>Results</U>:</P><P>We found that the values of pTIMP1 and pF4/80 are about 2.5-4 fold when compared to the empty vector (negative control). These values are lower than enhancer 7 that is positive in HEK293 but are still show promoter activity. However, the value of pSTEAP4  is about 24 fold when compared to the  empty vector, which suggests that pSTEAP4 is a strong  promoter in HEK293 cells.</P><P><U>Figure 11</U>: <STRONG>The activity of</STRONG><STRONG>TIMP1, STEAP4 or F4/80 promoters in HEK293</STRONG>. Luciferase reporter assay demonstrating transcriptional activation mediated by promoters of TIMP1, STEAP4 and F4/80 in HEK293 cells. The <EM>TIMP1 promoter</EM> showed 2.4-fold increased activity, as compared to the control (empty vector). The <EM>STEAP4 promoter</EM> showed 24-fold increased activity, which is 10 time more that pTIMP1, as compared to the control, and the <EM>F4/80 promoter</EM> showed 4-fold increased activity, as compared to the control. Enhancer 7 that serves as a positive control for Luciferase activity in HEK293 showed 5 fold increased activity. Relative luciferase expression results are presented after normalization to Renilla activity and represent the means ± standard deviation of three independent experiments.</P><P><U>Discussion</U>:</P><P>We found that these promoters are active in HEK293, but they have different activity. While TIMP1 and F4/80 showed lower activity than the positive control (enhancer 7), STEAP4 promoter showed high activity that is 10 fold from TIMP1 suggesting that this STEAP4 promoter is a strong promoter in HEK293 that can drive high expression of gene/s in this cells as well as in reactive astrocytes (previous experiment). Therefore, we can use this promoter to test our model system in HEK293 as these promoters are active and will be able to activate the required transcription in our model.</P><P> </P><P><U>References:</U></P><OL><LI>Trakhtenberg, Ephraim F., et al. &quot;Cell types differ in global coordination of splicing and proportion of highly expressed genes.&quot; <EM>Scientific Reports</EM>6 (2016): 32249.</LI><LI>Austyn, Jonathan M., and Siamon Gordon. &quot;F4/80, a monoclonal antibody directed specifically against the mouse macrophage.&quot; European journal of immunology10 (1981): 805-815.‏</LI><LI>Zamanian, Jennifer L., et al. &quot;Genomic analysis of reactive astrogliosis.&quot; <EM>Journal of neuroscience</EM>18 (2012): 6391-6410.</LI><LI>Zhang, Ye, et al. &quot;An RNA-sequencing transcriptome and splicing database of glia, neurons, and vascular cells of the cerebral cortex.&quot; <EM>Journal of Neuroscience</EM>36 (2014): 11929-11947.</LI><LI>Tokuda, Eiichi, Eriko Okawa, and Shin‐ichi Ono. &quot;Dysregulation of intracellular copper trafficking pathway in a mouse model of mutant copper/zinc superoxide dismutase‐linked familial amyotrophic lateral sclerosis.&quot; <EM>Journal of neurochemistry</EM>1 (2009): 181-191.</LI><LI>Lorenzl, S., et al. &quot;Tissue inhibitors of matrix metalloproteinases are elevated in cerebrospinal fluid of neurodegenerative diseases.&quot; <EM>Journal of the neurological sciences</EM>1-2 (2003): 71-76.‏</LI></OL><A href="https://2018.igem.org/Team:BGU_Israel/Experiments#Promoter" class="btn btn-info" role="button">See Experiment Background</A></DIV></DIV></DIV></DIV><DIV id="Infection2" class="bg-grey"><DIV class="container text-center"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="row"><H1>C8D30 infection by Lentivirus</H1><P><U>Title</U>: Lentivirus infection of C8D30 Astrocyte and HEK293 cells.</P><P><U>Conducted by</U>: Einan Farhi, Ori Tulchinsky, Liat Tsoran and Mor Pasi</P><P><U>Date</U>: 30.8.18, 15.9.18</P><P><U>Aim:</U> The objective is to create a stable line of the C8D30 astrocytes with a genomic insertion of the CMV-VP64 vector and another with the Timp1-VP64 vector.</P><P><U>Importance</U>: The CMV-VP64 and TIMP1-VP64 vectors are the first component in the two component system with which we aim to target and eliminate reactive astrocytes.</P><TABLE><TBODY><TR><TD width="208"><P>Experiments</P></TD><TD width="208"><P>Protocols</P></TD><TD width="208"><P>Notebook</P></TD></TR><TR><TD width="208"><P>Production of viral vector</P></TD><TD width="208"><P> </P></TD><TD width="208"><P> </P></TD></TR><TR><TD width="208"><P>Spin-infection</P></TD><TD width="208"><P> </P></TD><TD width="208"><P> </P></TD></TR></TBODY></TABLE><P> </P><P><U>Design</U>:</P><P>C8D30 spinfection with 500 µl CMV-VP64 and Timp1-VP64 viral medium.</P><TABLE width="0"><TBODY><TR><TD width="36"><P> </P></TD><TD width="93"><P>1</P></TD><TD width="93"><P>2</P></TD><TD width="93"><P>3</P></TD><TD width="93"><P>4</P></TD></TR><TR><TD width="36"><P>a</P></TD><TD width="93"><P>CMV-VP64</P></TD><TD width="93"><P>CMV-VP64</P></TD><TD width="93"><P>CMV-VP64</P></TD><TD width="93"><P>CMV-VP64</P></TD></TR><TR><TD width="36"><P>b</P></TD><TD width="93"><P>Timp1-VP64</P></TD><TD width="93"><P>Timp1-VP64</P></TD><TD width="93"><P>Timp1-VP64</P></TD><TD width="93"><P>Timp1-VP64</P></TD></TR><TR><TD width="36"><P>c</P></TD><TD width="93"><P>pGreen-puro</P></TD><TD width="93"><P>pGreen-puro</P></TD><TD width="93"><P>pGreen-puro</P></TD><TD width="93"><P>pGreen-puro</P></TD></TR><TR><TD width="36"><P>d</P></TD><TD width="93"><P>No treatment</P></TD><TD width="93"><P>No treatment</P></TD><TD width="93"><P>No treatment</P></TD><TD width="93"><P>No treatment</P></TD></TR></TBODY></TABLE><P> </P><P>HEK293T spinfection with 400 µl CMV-VP64 and Timp1-VP64 viral medium</P><TABLE width="0"><TBODY><TR><TD width="36"> </TD><TD width="93"><P>1</P></TD><TD width="93"><P>2</P></TD><TD width="93"><P>3</P></TD><TD width="93"><P>4</P></TD></TR><TR><TD width="36"><P>a</P></TD><TD width="93"><P>CMV-VP64</P></TD><TD width="93"><P>CMV-VP64</P></TD><TD width="93"><P>CMV-VP64</P></TD><TD width="93"><P>CMV-VP64</P></TD></TR><TR><TD width="36"><P>b</P></TD><TD width="93"><P>Timp1-VP64</P></TD><TD width="93"><P>Timp1-VP64</P></TD><TD width="93"><P>Timp1-VP64</P></TD><TD width="93"><P>Timp1-VP64</P></TD></TR><TR><TD width="36"><P>c</P></TD><TD width="93"><P>pGreen-puro</P></TD><TD width="93"><P>pGreen-puro</P></TD><TD width="93"><P>pGreen-puro</P></TD><TD width="93"><P>pGreen-puro</P></TD></TR><TR><TD width="36"><P>d</P></TD><TD width="93"><P>No treatment</P></TD><TD width="93"><P>No treatment</P></TD><TD width="93"><P>No treatment</P></TD><TD width="93"><P>No treatment</P></TD></TR></TBODY></TABLE><P> </P><P>HEK293T cell line cultured in 10 cm plates was co-transfected with the CMV-VP64 or Timp1-VP64 vector and 2<SUP>nd</SUP> generation Lentivirus helper plasmids. Positive infection control viruses were produced in co-transfection with pGreen-puro vector and 2<SUP>nd</SUP> generation Lentivirus helper plasmids. 48 hours later, viral medium was extracted, filtered and used to infect cells.</P><P>The C3D30 cell line was infected with the virus immediately after extraction in 24 well-plates using the spinfection method. As positive control for the infection process, we infected cells with pGreen-puro virus with which we previously demonstrated a visible expression of GFP and Puromycin resistance. As negative controls, cells were changed with fresh regular medium (no viral vector). Also, another control was to infect with all vectors an easy to handle cell line HEK293T.</P><P><U>Results:</U></P><P>Six days after infection, selection began with 2 µg/ml of Puromycin , changing medium every other day. GFP was seen in HEK293T infected with pGreen-puro vector with low efficiency suggesting that the virus production was not fully successful. No GFP was noticed in C8D30 wells that were infected with the same virus.  Eight days after the selection began, all the control cells had died, meaning all surviving cells in the infection wells, had been successfully infected.</P><P><U>Conclusion:</U></P><P>The original intention for this experiment was to create a stable line of C8D30 astrocytes that express deactivated cas9 (with the necessary accessories) under the pTIMP1 promoter. Then this stable line would be infected, using AAV2 with the pSynt plasmid.</P><P>The infection, described in this text, was successful in C8D30 astrocytes and HEK293, however, the recuperation period is lengthy. Due to time constraints, we were unable to use the infected cells in our experiments. Instead, we preformed the experiments using co-transfection into our target cells.</P><P><U>References:</U></P><OL><LI>Zamanian, Jennifer L., et al. &quot;Genomic analysis of reactive astrogliosis.&quot; Journal of neuroscience18 (2012): 6391-6410.‏</LI><LI>Zhang, Ye, et al. &quot;An RNA-sequencing transcriptome and splicing database of glia, neurons, and vascular cells of the cerebral cortex.&quot; Journal of Neuroscience36 (2014): 11929-11947.‏</LI><LI>Tokuda, Eiichi, Eriko Okawa, and Shin‐ichi Ono. &quot;Dysregulation of intracellular copper trafficking pathway in a mouse model of mutant copper/zinc superoxide dismutase‐linked familial amyotrophic lateral sclerosis.&quot; Journal of neurochemistry1 (2009): 181-191.‏</LI><LI>Lorenzl, S., et al. &quot;Tissue inhibitors of matrix metalloproteinases are elevated in cerebrospinal fluid of neurodegenerative diseases.&quot; Journal of the neurological sciences1-2 (2003): 71-76.‏</LI><LI>Maeder ML, Linder SJ, Cascio VM, Fu Y, Ho QH, Joung JK. CRISPR RNA-guided activation of endogenous human genes. Nature methods. 2013;10(10):977-979. doi:10.1038/nmeth.2598.</LI></OL><A href="https://2018.igem.org/Team:BGU_Israel/Experiments#Infection2" class="btn btn-info" role="button">See Experiment Background</A></DIV></DIV></DIV></DIV><DIV id="Caspase3_HEK" class="bg-white"><DIV class="container text-center"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="row"><H1>pMLPm Activation</H1><P><U>Title</U>: Validation of pMLPm activation with dCas9-VP64-gRNA and induction of apoptotic signal in HEK293 cells<U> Conducted by</U>: Liat Tsoran, Mor Pasi and Ori Tulchinsky</P><P><U>Date</U>: 30/9/18-14/10/18</P><P><U>Aim: </U>In this experiment we wanted to validate our synthetic engineered system that should induce apoptotic death by the activation of the synthetic minimal adenovirus major late promoter (pMLPm) via dCas9-VP64-gRNA complex.</P><P><U>Importance</U>: In our project, one of our goals is to engineera dCas9-VP64-gRNA system that could activate the pMLPm promoter to induce apoptotic death by the expression of exogenous reverse caspas3. Since we are using staphylococcus aureus dCas9-VP64-gRNA, we made changes in the pMLPm Protospacer Adjacent Motif (PAM), hence we had to verify pMLPm regulating activity trough the expression of the mCherry fluorophore. We would also like to verify the programmed apoptosis of the exogenous reverse caspas3 that is used in our project.</P><P><U>Theoretical background</U>:</P><P>The synthetic promoter minimal adenovirus major late promoter (pMLPm) contains three repetitions of the &quot;a1&quot; sequence, which is complementary to the gRNA sequence, enabling dCas9-VP64-gRNA complex to target the pMLPm synthetic promoter<SUP>1</SUP>. When the transcription factor VP64 (an engineered tetramer of the herpes simplex  virus VP16 transcriptional activator domain)  is fused to dCas9 enzyme it can be guided to a specific location in the genome, in this manner we can exploit it to promote downstream translation<SUP>2</SUP>.</P><P>The dCas9-VP64-gRNA complex will target the pMLPm synthetic promoter and promote the expression of exogenous reverse caspase3, that in contrast to the endogenous caspas3 will be activated after transcription due to its autocatalytic processing, meaning this enzyme will trigger an apoptotic signal that will lead to apoptotic death  (caspase3 is responsible for chromatin condensation and DNA fragmentation)<SUP>3</SUP>.</P><P>mCherry reporter protein is also expressed by the pMLPm promoter, it is fused to the exogenous reverse caspase3 with Thoseaasigna virus 2A (T2A) peptide that is cleaved during translation<SUP>4</SUP>.</P><P>In order to activate pMLPm promoter and to induce apoptotic death we used 2 plasmids:</P><OL><LI><STRONG><U>Lenti-viral plasmid: CMV dCas9 VP64</U></STRONG>– in this plasmid the CMV promoter regulates the expression of dCas9 enzyme which is fused to the transcription factor VP64.</LI><LI><STRONG><U>pSynt CMV</U></STRONG>– in this plasmid the CMV promoter regulates the expressession of the gRNA while the pMLPm promoter regulates the downstream translation of exogenous reverse caspase3 which is fused to the mCherry fluorophore.</LI></OL><P>We expected that: cells that were co-transfected with both plasmids will show mCherry expression and apoptotic death will be triggered. In order to detect leakage in pMLPm promoter we used a control of cells that were transfected only with pSynt CMV plasmid.</P><P><U>Design</U>:</P><OL><LI><STRONG> validation of pMLPm activation with dCas9-VP64-gRNA-</STRONG></LI><P><U>Plate A</U>:FACS</P><TABLE width="678"><TBODY><TR><TD width="242"><P>sample number</P></TD><TD width="149"><P>1</P></TD><TD width="144"><P>2</P></TD><TD width="144"><P>3</P></TD></TR><TR><TD width="242"><P>Transfection plasmids</P></TD><TD width="149"><P>Lenti: CMV dCas9 VP64 + pSynt CMV</P></TD><TD width="144"><P>pSynt CMV</P></TD><TD width="144"><P>Empty plasmid</P></TD></TR><TR><TD width="242"><P>wells</P></TD><TD width="149"><P>A1+B1</P></TD><TD width="144"><P>A2+B2</P></TD><TD width="144"><P>A3+B3</P></TD></TR></TBODY></TABLE><P><U> </U></P><P><U>Plate B: </U>confocal microscope</P><TABLE width="678"><TBODY><TR><TD width="242"><P>sample number</P></TD><TD width="149"><P>1</P></TD><TD width="144"><P>2</P></TD><TD width="144"><P>3</P></TD></TR><TR><TD width="242"><P>Transfection plasmids</P></TD><TD width="149"><P>Lenti: CMV dCas9 VP64 + pSynt CMV</P></TD><TD width="144"><P>pSynt CMV</P></TD><TD width="144"><P>Empty plasmid</P></TD></TR><TR><TD width="242"><P>wells</P></TD><TD width="149"><P>A1</P></TD><TD width="144"><P>A2</P></TD><TD width="144"><P>A3</P></TD></TR></TBODY></TABLE></OL><OL start="2"><LI><STRONG>induction of apoptosis</STRONG></LI></OL><P><U>Plate A:</U>     </P><TABLE width="652"><TBODY><TR><TD width="295"><P>sample number</P></TD><TD width="181"><P>1</P></TD><TD width="176"><P>2</P></TD></TR><TR><TD width="295"><P>sample name</P></TD><TD width="181"><P>CMV dCas + CMV pSynt</P></TD><TD width="176"><P>pSynt CMV</P></TD></TR><TR><TD width="295"><P>Transfection plasmids</P></TD><TD width="181"><P>Lenti: CMV dCas9 VP64 + pSynt CMV</P></TD><TD width="176"><P>pSynt CMV</P></TD></TR><TR><TD width="295"><P>wells</P></TD><TD width="181"><P>A1-3</P></TD><TD width="176"><P>B1-3</P></TD></TR><TR><TD width="295"><P>APC annexin V</P></TD><TD width="181"><P>+</P></TD><TD width="176"><P>+</P></TD></TR><TR><TD width="295"><P>Sytox green</P></TD><TD width="181"><P>+</P></TD><TD width="176"><P>+</P></TD></TR></TBODY></TABLE><P><U>Plate B</U>:       </P><TABLE width="652"><TBODY><TR><TD width="295"><P>sample number</P></TD><TD width="181"><P>3</P></TD><TD width="176"><P>4</P></TD></TR><TR><TD width="295"><P>sample name</P></TD><TD width="181"><P>mCherry control (for FACS calibration)</P></TD><TD width="176"><P>annexin control (for FACS calibration)</P></TD></TR><TR><TD width="295"><P>Transfection plasmids</P></TD><TD width="181"><P>Lenti: CMV dCas9 VP64 + pSynt CMV</P></TD><TD width="176"><P>Lenti: CMV dCas9 VP64 + pSynt CMV</P></TD></TR><TR><TD width="295"><P>wells</P></TD><TD width="181"><P>A1-3</P></TD><TD width="176"><P>B1-3</P></TD></TR><TR><TD width="295"><P>APC annexin V</P></TD><TD width="181"><P>-</P></TD><TD width="176"><P>+</P></TD></TR><TR><TD width="295"><P>Sytox green</P></TD><TD width="181"><P>-</P></TD><TD width="176"><P>-</P></TD></TR></TBODY></TABLE><P><U> </U></P><P><U>Plate C: </U>      </P><TABLE width="652"><TBODY><TR><TD width="295"><P>sample number</P></TD><TD width="181"><P>5</P></TD><TD width="176"><P>6</P></TD></TR><TR><TD width="295"><P>sample name</P></TD><TD width="181"><P>necrosis sytox control (for FACS calibration)</P></TD><TD width="176"><P>necrosis</P></TD></TR><TR><TD width="295"><P>Transfection plasmids</P></TD><TD width="181"><P>-</P></TD><TD width="176"><P>-</P></TD></TR><TR><TD width="295"><P>wells</P></TD><TD width="181"><P>A1-3</P></TD><TD width="176"><P>B1-3</P></TD></TR><TR><TD width="295"><P>APC annexin V</P></TD><TD width="181"><P>-</P></TD><TD width="176"><P>+</P></TD></TR><TR><TD width="295"><P>Sytox green</P></TD><TD width="181"><P>+</P></TD><TD width="176"><P>+</P></TD></TR></TBODY></TABLE><P><U>Plate D: </U>      </P><TABLE width="476"><TBODY><TR><TD width="295"><P>sample number</P></TD><TD width="181"><P>7</P></TD></TR><TR><TD width="295"><P>sample name</P></TD><TD width="181"><P>without apoptosis induction</P></TD></TR><TR><TD width="295"><P>Transfection plasmids</P></TD><TD width="181"><P>-</P></TD></TR><TR><TD width="295"><P>wells</P></TD><TD width="181"><P>A1-3</P></TD></TR><TR><TD width="295"><P>APC annexin V</P></TD><TD width="181"><P>+</P></TD></TR><TR><TD width="295"><P>Sytox green</P></TD><TD width="181"><P>+</P></TD></TR></TBODY></TABLE><P><U>Expectations:</U></P><OL><LI><STRONG> Validation of pMLPm activation with dCas9-VP64-gRNA:</STRONG></LI><TABLE width="678"><TBODY><TR><TD width="242"><P>sample number</P></TD><TD width="149"><P>1</P></TD><TD width="144"><P>2</P></TD><TD width="144"><P>3</P></TD></TR><TR><TD width="242"><P>Transfection plasmids</P></TD><TD width="149"><P>Lenti CMV dCAs9 VP64 + pSynt CMV</P></TD><TD width="144"><P>pSynt CMV</P></TD><TD width="144"><P>Empty plasmid</P></TD></TR><TR><TD width="242"><P>mCherry expression</P></TD><TD width="149"><P>+</P></TD><TD width="144"><P>-</P></TD><TD width="144"><P>-</P></TD></TR></TBODY></TABLE></OL><OL start="2"><LI><STRONG>Induction of apoptotic signal:</STRONG></LI></OL><TABLE><TBODY><TR><TD width="84"><P>sample number</P></TD><TD width="51"><P>1</P></TD><TD width="51"><P>2</P></TD><TD width="89"><P>3</P></TD><TD width="89"><P>4</P></TD><TD width="68"><P>5</P></TD><TD width="68"><P>6</P></TD><TD width="77"><P>7</P></TD></TR><TR><TD width="84"><P>sample name</P></TD><TD width="51"><P>CMV dcas+ CMV pSynt</P></TD><TD width="51"><P>PSynt CMV</P></TD><TD width="89"><P>mCherry control (for FACS calibration)</P></TD><TD width="89"><P>annexin control (for FACS calibration)</P></TD><TD width="68"><P>necrosis sytox control (for FACS calibration)</P></TD><TD width="68"><P>necrosis</P></TD><TD width="77"><P>without apoptosis induction</P></TD></TR><TR><TD width="84"><P>Dead</P></TD><TD width="51"><P>+</P></TD><TD width="51"><P>-</P></TD><TD width="89"><P>+</P></TD><TD width="89"><P>+</P></TD><TD width="68"><P>+</P></TD><TD width="68"><P>+</P></TD><TD width="77"><P>-</P></TD></TR><TR><TD width="84"><P>Alive</P></TD><TD width="51"><P>-</P></TD><TD width="51"><P>+</P></TD><TD width="89"><P>-</P></TD><TD width="89"><P>-</P></TD><TD width="68"><P>-</P></TD><TD width="68"><P>-</P></TD><TD width="77"><P>+</P></TD></TR><TR><TD width="84"><P>Necrosis</P></TD><TD width="51"><P>-</P></TD><TD width="51"><P>-</P></TD><TD width="89"><P>-</P></TD><TD width="89"><P>-</P></TD><TD width="68"><P>+</P></TD><TD width="68"><P>+</P></TD><TD width="77"><P>-</P></TD></TR><TR><TD width="84"><P>Apoptosis</P></TD><TD width="51"><P>+</P></TD><TD width="51"><P>-</P></TD><TD width="89"><P>+</P></TD><TD width="89"><P>+</P></TD><TD width="68"><P>-</P></TD><TD width="68"><P>-</P></TD><TD width="77"><P>-</P></TD></TR><TR><TD width="84"><P>mCherry expression</P></TD><TD width="51"><P>+</P></TD><TD width="51"><P>-</P></TD><TD width="89"><P>+</P></TD><TD width="89"><P>+</P></TD><TD width="68"><P>-</P></TD><TD width="68"><P>-</P></TD><TD width="77"><P>-</P></TD></TR></TBODY></TABLE><P><U>Results</U>:</P><P><U>Figure 12</U>: mCherry in FACS ARIA III of HEK293 cells (excitation: 561nm emission: 610\20nm). A – cells without transfection, 78.8% of sample are viable cells. B – 48 hours post co-transfection with lent-virus:CMV-VP64 and pSynt-CMV, 88.0% of sample are viable cells.</P><P><U>Figure 13</U>: Hystogramm analysis of viable HEK293 cells. Number of cells with mCherry expression. Blue - cells without transfection. Red – 48 hours post co-transfection with lenti-virus: CMV-VP64 and pSynt-CMV, 23.1% of viable cells expressed mCherry.</P><P><U>Figure 14</U>: Images of mCherry in HEK293 cells 24 hours post co-transfection with lenti:CMV-VP64 and pSynt-CMV. A – bright field. B – Fluorescent microscope.</P><P><U>Discussion</U>:</P><OL><LI>mCherry expression in HEK293 cells was validated:</LI><UL><LI>In FACS ARIA we were able to see that although we had low transfection efficiency, 23.1% of cells expressed mCherry.</LI><LI>In fluorescent and confocal microscope we were able to detect mCherry fluorophore in several cells.</LI><LI>Although, ew do not have a proof of apoptosis, we can see some background red-flourescene in our images, cells that undergone apoptosis will dispese their expressed m-Cerry in the media, thus the fluorescence will not be as focused as seen with localized living cells using confocal microscopy, rather there will be a background of red-flourescene, hence we may cautiously say that some of the cells that have expressed m-Cerry, have undergone apoptosis as well. This notion will require further validation.</LI></UL></OL><OL start="2"><LI>When we used APC Annexin V/Dead cell apoptosis kit we run into several issus:</LI><UL><LI>The process of transfection and APC Annexin V/Dead cell apoptosis kit procedure could influence test results, since in both of these procedures cells may die.</LI><LI>Co transfection efficiency has a major rule in the success of our apoptotic induction, in this experiment the co-transfection efficiency was low but we didn't had the time to repeat it.</LI><LI>APC Annexin V/Dead cell apoptosis kit was calibrated to Jurkat cells. Since we are working with a different cell line further calibration is required.</LI></UL></OL><P><U>Conclusion:</U></P><P>In this experiment we saw that pMLPm can be activated in HEK293 cells by dCas9-VP64-gRNA complex, since we were able to validate mCherry expression in several methods.</P><P>In HEK293 cells we did it when both gRNA and dCas9-VP64 are expressed by CMV promoter, our next step is to regulate pMLPm promoter activity in C8D30 cell line when dCas9-VP64-gRNA comlex is expressed by A1 reactive astrocyte specific promoters (pTimp1 and pSteap4).</P><P>In order to prove that the exogenous reverse caspase3 that was chosen in our design can trigger apoptotic death when it is regulated by pMLPm promoter, calibration on APC Annexin V/Dead cell apoptosis kit is nesseccery.</P><P><U>References:</U></P><OL><LI>Farzadfard, F., Perli, S. D., &amp; Lu, T. K. (2013). Tunable and multifunctional eukaryotic transcription factors based on CRISPR/Cas. <EM>ACS synthetic biology</EM>, <EM>2</EM>(10), 604-613.‏</LI><LI>La Russa, M. F., &amp; Qi, L. S. (2015). The new state of the art: CRISPR for gene activation and repression. <EM>Molecular and cellular biology</EM>, MCB-00512.‏</LI><LI>Srinivasula, S. M., Ahmad, M., MacFarlane, M., Luo, Z., Huang, Z., Fernandes-Alnemri, T., &amp; Alnemri, E. S. (1998). Generation of constitutively active recombinant caspases-3 and-6 by rearrangement of their subunits. <EM>Journal of Biological Chemistry</EM>, <EM>273</EM>(17), 10107-10111.‏</LI><LI>Kim, J. H., Lee, S. R., Li, L. H., Park, H. J., Park, J. H., Lee, K. Y., ... &amp; Choi, S. Y. (2011). High cleavage efficiency of a 2A peptide derived from porcine teschovirus-1 in human cell lines, zebrafish and mice. <EM>PloS one</EM>, <EM>6</EM>(4), e18556.‏</LI></OL><A href="https://2018.igem.org/Team:BGU_Israel/Experiments#Caspase3_HEK" class="btn btn-info" role="button">See Experiment Background</A></DIV></DIV></DIV></DIV><DIV id="Caspase3_Astro" class="bg-grey"><DIV class="container text-center"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="row"><H1>Apoptosis Induction</H1><P><U>Title</U>: Validation of pMLPm activation with dCas9-VP64-gRNA and induction of apoptosis in C8-D30 astrocyte cells<U> Conducted by</U>: Liat Tsoran, Mor Pasi and Ori Tulchinsky</P><P><U>Date</U>:<STRONG> </STRONG>30/9/18-14/10/18</P><P><U>Aim: </U>In this experiment we wanted to test the activity and efficiency of our synthetic engineered system that should induce apoptotic death by the activation of the synthetic minimal adenovirus major late promoter (pMLPm) via dCAs9-VP64-gRNA complex that is expressed by A1 reactive astrocyte specific promoters.</P><P><U>Importance</U>: In the astrocyte pathway we aimed to eliminate only A1 reactive astrocyte by triggering apoptotic death, while maintaining the resting cells in the central bervous system (CNS) intact. In this experiment we wanted to test the effectiveness of our enginnered system, in which, dCas9-VP64-gRNA complex that is expressed under the regulation of A1 reactive astrocyte specific markers, can activate the pMLPm promoter to regulate the expression of exogenous reverse caspase3 that will trigger apoptosis.</P><P><U>Theoretical background</U>:</P><P>pSTEAP4 (Six Transmembrane Epithelial Antigen of Prostate 4 promoter)  and pTIMP1 (Tissue inhibitor of metalloproteinases-1 promoter) are regulating genes expression which are highly expressed in A1 reactive astrocyte cells when compared to other cells in the central nervous system<SUP>1,2</SUP></P><P>In our therapeutic approach, we wanted to use these markers in order to express dCas9-VP64-gRNA complex that will activate a synthetic promoter to induce apoptosis.</P><P>The synthetic minimal adenovirus major late promoter (pMLPm) contains three repetitions of the &quot;a1&quot; sequence, which is complementary to the gRNA sequence, enabling dCas9-VP64-gRNA complex to target the pMLPm synthetic promoter<SUP>1</SUP>. When the transcription factor VP64 (an engineered tetramer of the herpes simplex virus VP16 transcriptional activator domain) is fused to dCas9 enzyme it can be guided to a specific location in the genome, in this manner we can exploit it to promote downstream translation<SUP>2</SUP>.</P><P>The dCas9-VP64-gRNA complex will target the pMLPm synthetic promoter and promote the expression of exogenous reverse caspase3, that in contrast to the endogenous caspase3 will be activated after transcription due to its autocatalytic processing, meaning that this enzyme will trigger an apoptotic signal that will lead to apoptotic death  (caspase3 is responsible for chromatin condensation and DNA fragmentation)<SUP>3</SUP>.</P><P>mCherry reporter protein is also expressed by the pMLPm promoter, it is fused to the exogenous reverse caspase3 with Thoseaasigna virus 2A (T2A) peptide that is cleaved during translation<SUP>4</SUP>.</P><P>In order to activate pMLPm promoter and to induce apoptotic death we used 2 plasmids:</P><OL><LI>Lenti: pTIMP1 dCas9 VP64 – in this plasmid TIMP1 promoter express dCas9 enzyme which is fused to the transcription factor VP64.</LI><LI>pSynt – in this plasmid pSTEAP4 promoter express gRNA. And pMLPm promoter regulate the downstream translation of reverse caspase3 which is fused to mCherry fluorophore.</LI></OL><P>A1 reactive astrocyte cells that were co-transfection with both plasmids will show mCherry expression and will apoptotic death will be triggered. In order to detect leakage in pMLPm promoter we used a control of cells that were transfected only with pSynt plasmid.</P><P><U>Design</U>:</P><OL><LI><STRONG> validation of pMLPm activation with dCas9-VP64-gRNA:</STRONG></LI><P><U>Plate A:</U> FACS     </P><TABLE width="678"><TBODY><TR><TD width="242"><P>sample number</P></TD><TD width="149"><P>1</P></TD><TD width="144"><P>2</P></TD><TD width="144"><P>3</P></TD></TR><TR><TD width="242"><P>Transfection plasmids</P></TD><TD width="149"><P>Lenti: pTIMP1 dCas9 VP64 + pSynt CMV</P></TD><TD width="144"><P>pSynt</P></TD><TD width="144"><P>Empty plasmid</P></TD></TR><TR><TD width="242"><P>wells</P></TD><TD width="149"><P>A1+B1</P></TD><TD width="144"><P>A2+B2</P></TD><TD width="144"><P>A3+B3</P></TD></TR></TBODY></TABLE><P><U> </U></P><P><U>Plate B: </U>confocal microscope     </P><TABLE width="678"><TBODY><TR><TD width="242"><P>sample number</P></TD><TD width="149"><P>1</P></TD><TD width="144"><P>2</P></TD><TD width="144"><P>3</P></TD></TR><TR><TD width="242"><P>Transfection plasmids</P></TD><TD width="149"><P>Lenti: pTIMP1 dCas9 VP64 + pSynt CMV</P></TD><TD width="144"><P>pSynt</P></TD><TD width="144"><P>Empty plasmid</P></TD></TR><TR><TD width="242"><P>wells</P></TD><TD width="149"><P>A1</P></TD><TD width="144"><P>A2</P></TD><TD width="144"><P>A3</P></TD></TR></TBODY></TABLE><P><STRONG> </STRONG></P></OL><OL start="2"><LI><STRONG> induction of apoptotic signal-</STRONG></LI><P><U>Plate A: </U>      </P><TABLE width="652"><TBODY><TR><TD width="295"><P>sample number</P></TD><TD width="181"><P>1</P></TD><TD width="176"><P>2</P></TD></TR><TR><TD width="295"><P>sample name</P></TD><TD width="181"><P>pTIMP1 dCas+ pSynt</P></TD><TD width="176"><P>pSynt</P></TD></TR><TR><TD width="295"><P>Transfection plasmids</P></TD><TD width="181"><P>Lenti: pTIMP1 dCas9 VP64 + pSynt</P></TD><TD width="176"><P>pSynt</P></TD></TR><TR><TD width="295"><P>wells</P></TD><TD width="181"><P>A1-3</P></TD><TD width="176"><P>B1-3</P></TD></TR><TR><TD width="295"><P>APC annexin V</P></TD><TD width="181"><P>+</P></TD><TD width="176"><P>+</P></TD></TR><TR><TD width="295"><P>Sytox green</P></TD><TD width="181"><P>+</P></TD><TD width="176"><P>+</P></TD></TR></TBODY></TABLE><P> </P><P><U>Plate B:</U> </P><TABLE width="652"><TBODY><TR><TD width="295"><P>sample number</P></TD><TD width="181"><P>3</P></TD><TD width="176"><P>4</P></TD></TR><TR><TD width="295"><P>sample name</P></TD><TD width="181"><P>mCherry control (for FACS calibration)</P></TD><TD width="176"><P>annexin control (for FACS calibration)</P></TD></TR><TR><TD width="295"><P>Transfection plasmids</P></TD><TD width="181"><P>Lenti: pTIMP1 dCas9 VP64 + pSynt</P></TD><TD width="176"><P>Lenti: pTIMP1 dCas9 VP64 + pSynt</P></TD></TR><TR><TD width="295"><P>wells</P></TD><TD width="181"><P>A1-3</P></TD><TD width="176"><P>B1-3</P></TD></TR><TR><TD width="295"><P>APC annexin V</P></TD><TD width="181"><P>-</P></TD><TD width="176"><P>+</P></TD></TR><TR><TD width="295"><P>Sytox green</P></TD><TD width="181"><P>-</P></TD><TD width="176"><P>-</P></TD></TR></TBODY></TABLE><P><U>Plate C</U>:       </P><TABLE width="652"><TBODY><TR><TD width="295"><P>sample number</P></TD><TD width="181"><P>5</P></TD><TD width="176"><P>6</P></TD></TR><TR><TD width="295"><P>sample name</P></TD><TD width="181"><P>necrosis sytox control (for FACS calibration)</P></TD><TD width="176"><P>necrosis</P></TD></TR><TR><TD width="295"><P>Transfection plasmids</P></TD><TD width="181"><P>-</P></TD><TD width="176"><P>-</P></TD></TR><TR><TD width="295"><P>wells</P></TD><TD width="181"><P>A1-3</P></TD><TD width="176"><P>B1-3</P></TD></TR><TR><TD width="295"><P>APC annexin V</P></TD><TD width="181"><P>-</P></TD><TD width="176"><P>+</P></TD></TR><TR><TD width="295"><P>Sytox green</P></TD><TD width="181"><P>+</P></TD><TD width="176"><P>+</P></TD></TR></TBODY></TABLE><P><U>Plate D</U>:       </P><TABLE width="476"><TBODY><TR><TD width="295"><P>sample number</P></TD><TD width="181"><P>7</P></TD></TR><TR><TD width="295"><P>sample name</P></TD><TD width="181"><P>without apoptosis induction</P></TD></TR><TR><TD width="295"><P>Transfection plasmids</P></TD><TD width="181"><P>-</P></TD></TR><TR><TD width="295"><P>wells</P></TD><TD width="181"><P>A1-3</P></TD></TR><TR><TD width="295"><P>APC annexin V</P></TD><TD width="181"><P>+</P></TD></TR><TR><TD width="295"><P>Sytox green</P></TD><TD width="181"><P>+</P></TD></TR></TBODY></TABLE><P> </P></OL><P><SPAN style="text-decoration: underline;">Expectations:</SPAN></P><OL><LI><STRONG> validation of pMLPm activation with dCas9-VP64-gRNA:</STRONG></LI><TABLE width="678"><TBODY><TR><TD width="242"><P>sample number</P></TD><TD width="149"><P>1</P></TD><TD width="144"><P>2</P></TD><TD width="144"><P>3</P></TD></TR><TR><TD width="242"><P>Transfection plasmids</P></TD><TD width="149"><P>Lenti Timp1 dCAs9 VP64 + pSynt</P></TD><TD width="144"><P>pSynt</P></TD><TD width="144"><P>Empty plasmid</P></TD></TR><TR><TD width="242"><P>mCherry expression</P></TD><TD width="149"><P>+</P></TD><TD width="144"><P>-</P></TD><TD width="144"><P>-</P></TD></TR></TBODY></TABLE></OL><OL start="2"><LI><STRONG> induction of apoptotic signal:</STRONG></LI><TABLE><TBODY><TR><TD width="84"><P>sample number</P></TD><TD width="51"><P>1</P></TD><TD width="51"><P>2</P></TD><TD width="89"><P>3</P></TD><TD width="89"><P>4</P></TD><TD width="68"><P>5</P></TD><TD width="68"><P>6</P></TD><TD width="77"><P>7</P></TD></TR><TR><TD width="84"><P>sample name</P></TD><TD width="51"><P>Timp1 dcas+ pSynt</P></TD><TD width="51"><P>PSynt</P></TD><TD width="89"><P>mCherry control (for FACS calibration)</P></TD><TD width="89"><P>annexin control (for FACS calibration)</P></TD><TD width="68"><P>necrosis sytox control (for FACS calibration)</P></TD><TD width="68"><P>necrosis</P></TD><TD width="77"><P>without apoptosis induction</P></TD></TR><TR><TD width="84"><P>Dead</P></TD><TD width="51"><P>+</P></TD><TD width="51"><P>-</P></TD><TD width="89"><P>+</P></TD><TD width="89"><P>+</P></TD><TD width="68"><P>+</P></TD><TD width="68"><P>+</P></TD><TD width="77"><P>-</P></TD></TR><TR><TD width="84"><P>Alive</P></TD><TD width="51"><P>-</P></TD><TD width="51"><P>+</P></TD><TD width="89"><P>-</P></TD><TD width="89"><P>-</P></TD><TD width="68"><P>-</P></TD><TD width="68"><P>-</P></TD><TD width="77"><P>+</P></TD></TR><TR><TD width="84"><P>Necrosis</P></TD><TD width="51"><P>-</P></TD><TD width="51"><P>-</P></TD><TD width="89"><P>-</P></TD><TD width="89"><P>-</P></TD><TD width="68"><P>+</P></TD><TD width="68"><P>+</P></TD><TD width="77"><P>-</P></TD></TR><TR><TD width="84"><P>Apoptosis</P></TD><TD width="51"><P>+</P></TD><TD width="51"><P>-</P></TD><TD width="89"><P>+</P></TD><TD width="89"><P>+</P></TD><TD width="68"><P>-</P></TD><TD width="68"><P>-</P></TD><TD width="77"><P>-</P></TD></TR><TR><TD width="84"><P>mCherry expression</P></TD><TD width="51"><P>+</P></TD><TD width="51"><P>-</P></TD><TD width="89"><P>+</P></TD><TD width="89"><P>+</P></TD><TD width="68"><P>-</P></TD><TD width="68"><P>-</P></TD><TD width="77"><P>-</P></TD></TR></TBODY></TABLE></OL><P><U>Results</U>:</P><P><U>Figure 15</U>: Images of mCherry in C8-D30 cells 24 hours post co-transfection with lenti:pTimp1-VP64 and pSynt. A – Fluorescent microscope. B – Bright field.</P><P><U>Discussion</U>:</P><OL><LI>mCherry expression in C8-D30 reactive astrocyte cells was visible. Demonstrating the activity of our dcas9 editing approach and verifying expression of rev-caspase3. In fluorescent microscope we were able to detect mCherry fluorophore in several cells.</LI><LI>By demonstrating mCherry expression we were able to demonstrate both the specificity of our chosen promoter as well as the ability of our system to drive the expression of caspase3 (through the mCherry marker) we have designed into our plasmid.</LI><LI>When we used APC Annexin V/Dead cell apoptosis kit we run into several issues:</LI><UL><LI>The process of transfection and APC Annexin V/Dead cell apoptosis kit procedure could influence test results, since in both of these procedures cells may die.</LI><LI>Co-transfection efficiency has a major role in the success of our apoptotic induction, in this experiment the co-transfection efficiency was low but we didn't have time to repeat it.</LI><LI>APC Annexin V/Dead cell apoptosis kit was calibrated in Jurkat cells. Since we are working with a different cell line further calibration is required.</LI></UL></OL><P><U>Conclusion:</U></P><P>In this experiment we saw that pMLPm can be activated in C8-D30 cells by dCas9-VP64-gRNA complex, since we were able to validate mCherry expression by several methods.</P><P>This means that pMLPm promoter activity can be regulated in reactive astrocytes (C8D30) cell line when dCas9-VP64-gRNA comlex is expressed by A1 reactive astrocyte specific promoters (pTimp1 and pSteap4).</P><P>In order to prove that the exogenous reverse caspase3 that was chosen in our design can trigger apoptotic death when it is regulated by pMLPm promoter, calibration on APC Annexin V/Dead cell apoptosis kit is nesseccery.</P><P>Since co-transfection efficiency is very low, the next step is to infect the cells with lenti:pTimp1-dCAs9-VP64, and create a stable cell-line, and only only then to insert the pSynt plasmid (either by transfection or by AAV2/9 infection).</P><P><U>References:</U></P><OL><LI>Liddelow SA, Guttenplan KA, Clarke LE, et al. Neurotoxic reactive astrocytes are induced by activated microglia. <EM>Nature</EM>. 2017;541(7638):481-487. doi:10.1038/nature21029.</LI><LI>Zamanian J, Xu L, Foo L, et al. Genomic Analysis of Reactive Astrogliosis. <EM>The Journal of neuroscience : the official journal of the Society for Neuroscience</EM>. 2012;32(18):6391-6410. doi:10.1523/JNEUROSCI.6221-11.2012.</LI><LI>Farzadfard, F., Perli, S. D., &amp; Lu, T. K. (2013). Tunable and multifunctional eukaryotic transcription factors based on CRISPR/Cas. <EM>ACS synthetic biology</EM>, <EM>2</EM>(10), 604-613.‏</LI><LI>La Russa, M. F., &amp; Qi, L. S. (2015). The new state of the art: CRISPR for gene activation and repression. <EM>Molecular and cellular biology</EM>, MCB-00512.‏</LI><LI>Srinivasula, S. M., Ahmad, M., MacFarlane, M., Luo, Z., Huang, Z., Fernandes-Alnemri, T., &amp; Alnemri, E. S. (1998). Generation of constitutively active recombinant caspases-3 and-6 by rearrangement of their subunits. <EM>Journal of Biological Chemistry</EM>, <EM>273</EM>(17), 10107-10111.‏</LI><LI>Kim, J. H., Lee, S. R., Li, L. H., Park, H. J., Park, J. H., Lee, K. Y., ... &amp; Choi, S. Y. (2011). High cleavage efficiency of a 2A peptide derived from porcine teschovirus-1 in human cell lines, zebrafish and mice. <EM>PloS one</EM>, <EM>6</EM>(4), e18556.‏</LI></OL><A href="https://2018.igem.org/Team:BGU_Israel/Experiments#Caspase3_Astro" class="btn btn-info" role="button">See Experiment Background</A></DIV></DIV></DIV></DIV><DIV id="myModal" class="modal"><SPAN class="close" onclick="modal.style.display = 'none'">×</SPAN></DIV><TITLE>OriginALS</TITLE><FOOTER style="margin-bottom:-10px!important;margin-top:-10px!important;"><DIV class="container my-auto"><DIV class="row"><DIV class="col-md-4"><H3 style="color:white!important;">About Us</H3><P style="color:white!important;text-align: justify!important;">The BGU-iGEM team “OriginALS” hopes to develop an innovative therapeutic approach to prolong the life expectancy of ALS patients, using Synthetic Biology. We are dedicated to promoting ALS awareness and research in Israel through public engagement and educational activities.</P></DIV><DIV class="col-md-4"><UL class="footer-list"><H3>Keep in Touch</H3><LI class="footer-list-item"> @iGEMBGU</LI><LI class="footer-list-item">@igem_2018_bgu</LI><LI class="footer-list-item"> @originalsbgu</LI><LI class="footer-list-item"> originalsbgu@gmail.com</LI></UL></DIV><DIV class="col-md-4"><UL class="footer-list"><H3>Address</H3><LI class="footer-list-item">Ben-Gurion University of the Negev
Ben Gurion 1, Beer Sheva 8410501, Israel</LI></UL></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>