<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BGU_Israel_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BGU Israel/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>OriginALS</TITLE></P><TITLE>OriginALS</TITLE><DIV class="topnav" id="myTopnav"><A href="https://2018.igem.org/Team:BGU_Israel" id="brand-img"> </A><DIV id="topnav-right"><A href="https://2018.igem.org/Team:BGU_Israel">Home</A><DIV class="dropdown"><BUTTON class="dropbtn">Project
      </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:BGU_Israel/Description	">Description</A><A href="https://2018.igem.org/Team:BGU_Israel/Design">Design</A><A href="https://2018.igem.org/Team:BGU_Israel/Experiments">Experiments</A><A href="https://2018.igem.org/Team:BGU_Israel/Results">Results</A><A href="https://2018.igem.org/Team:BGU_Israel/Demonstrate">Demonstrate</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Lab
      </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:BGU_Israel/Notebook">Notebook</A><A href="https://2018.igem.org/Team:BGU_Israel/Protocols">Protocols</A><A href="https://2018.igem.org/Team:BGU_Israel/InterLab">InterLab</A><A href="https://2018.igem.org/Team:BGU_Israel/Safety">Safety</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Team
      </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:BGU_Israel/Team">Members</A><A href="https://2018.igem.org/Team:BGU_Israel/Collaborations">Collaborations</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Parts
      </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:BGU_Israel/Parts">Overview</A><A href="https://2018.igem.org/Team:BGU_Israel/Basic_Part">Basic Parts</A><A href="https://2018.igem.org/Team:BGU_Israel/Part_Collection">Collection</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Human Practices
      </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:BGU_Israel/Human_Practices">Integrated Human Practices</A><A href="https://2018.igem.org/Team:BGU_Israel/ALSConference">ALS Conference</A><A href="https://2018.igem.org/Team:BGU_Israel/Public_Engagement">Public Engagement</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Acknowledgements
      </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:BGU_Israel/Attributions">Attributions</A><A href="https://2018.igem.org/Team:BGU_Israel/Sponsors">Sponsors</A></DIV></DIV><A href="javascript:void(0);" style="font-size:15px; " class="icon" onclick="myFunction()">☰</A></DIV></DIV><HEADER style="background-image:url('https://static.igem.org/mediawiki/2018/e/ea/T--BGU_Israel--blue_subhead.png')!important;" class="sub-page-header text-center"><DIV class="container my-auto"><H1 class="animated fadeInUp">Experiments</H1></DIV></HEADER><DIV class="bg-grey"><DIV class="container"><DIV class="row"><DIV class="col-sm-10 col-sm-offset-1"><H4 style="text-align:center!important;">
Here are our greatest achievements!
Each experiment that we consider a pillar of our project and  essential to its success is marked with a  icon.</H4></DIV></DIV><DIV class="row"><DIV class="col-md-4"><UL class="list-group nav-card"><LI class="list-group-item"><A href="#Technical">Technical</A></LI><LI class="list-group-item"><A style="margin:0 0 0 31px!important;" href="#CalciumPhosphate">Calcium Phosphate Transfection</A></LI><LI class="list-group-item"><A style="margin:0 0 0 31px!important;" href="#Electroporation">Electroporation Transfection</A></LI><LI class="list-group-item"><A style="margin:0 0 0 31px!important;" href="#Reagent">Transfection By Transfection-Reagent</A></LI><LI class="list-group-item"><A href="#" style="color:white!important;">C</A></LI><LI class="list-group-item"><A href="#" style="color:white!important;">C</A></LI></UL></DIV><DIV class="col-md-4"><UL class="list-group nav-card"><LI class="list-group-item item-dark"><A href="#Microglia">Microglia Pathway</A></LI><LI class="list-group-item"><A style="margin:0 0 0 31px!important;" href="#Infection">BV2 infection by Lentivirus</A></LI><LI class="list-group-item"><A href="#qPCR">Cytokines Inhibition Assay</A></LI><LI class="list-group-item"><A href="#" style="color:white!important;">C</A></LI><LI class="list-group-item"><A href="#" style="color:white!important;">C</A></LI><LI class="list-group-item"><A href="#" style="color:white!important;">C</A></LI></UL></DIV><DIV class="col-md-4"><UL class="list-group nav-card"><LI class="list-group-item item-dark"><A href="#Astrocyte">Astrocyte Pathway</A></LI><LI class="list-group-item"><A href="#AstroActive">Astrocyte Activation</A></LI><LI class="list-group-item"><A href="#Promoter">Promoter Assay</A></LI><LI class="list-group-item"><A style="margin:0 0 0 31px!important;" href="#Infection2">C8D30 infection by Lentivirus</A></LI><LI class="list-group-item"><A href="#Caspase3_HEK">pMLPm Activation</A></LI><LI class="list-group-item"><A href="#Caspase3_Astro">Apoptosis Induction</A></LI></UL></DIV></DIV></DIV></DIV><DIV id="Technical" class="bg-dark text-center sub-nav"><DIV class="container"><H1>Technical Experiments</H1><DIV class="row animated fadeInRight"><A href="#CalciumPhosphate" class="btn btn-info" role="button">Calcium Phosphate  Transfection</A><A href="#Electroporation" class="btn btn-info" role="button">Electroporation Transfection</A><A href="#Reagent" class="btn btn-info" role="button">Transfection By Transfection-Reagent</A></DIV></DIV></DIV><DIV id="CalciumPhosphate" class="bg-white"><DIV class="container text-center"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="row"><H1>Calcium Phosphate Transfection</H1><P><STRONG><U>Title</U>: </STRONG>
    Transfection to microglia and astrocytes via “calcium phosphate” method.
</P><P><STRONG><U>Conducted by</U>: </STRONG>
    Sagi Angel
</P><P><STRONG><U>Date</U>: </STRONG>
    3-7.6.18</P><P><STRONG><U>Aim</U>: </STRONG>
    In this experiment we have tried to use the calcium phosphate protocol in
    order to transfect microglia astrocytes and HEK as control in GFP gene. The
    use of HEK cells as control, is due to its ability to be transfected
    relatively easily by the various techniques, including calcium phosphate
    method.</P><P><STRONG><U>Importance</U></STRONG>: This experiment was carried out in parallel with experiments using
    different methods of transfection (different reagents and electroporation)
    in order to find an efficient way of inserting our plasmids into astrocytes
    and microglia for the continuation of the project.</P><TABLE><TBODY><TR><TD><P>
                    Experiments
                </P></TD><TD><P>
                    Protocols
                </P></TD><TD><P>
                    Notebook
                </P></TD></TR><TR><TD><P>
                    Calcium Phosphate
                    </P></TD><TD><A target="_blank" style="color:#0186ac!important;" href="https://static.igem.org/mediawiki/2018/a/a8/T--BGU_Israel--protocols_exp_calcium_phospate_transfection.pdf">Link</A></TD><TD><A target="_blank" style="color:#0186ac!important;" href="https://static.igem.org/mediawiki/2018/3/34/T--BGU_Israel--notebook_exp_calcium_phosphate.pdf">Link</A></TD></TR></TBODY></TABLE><P><STRONG><U>Theoretical background</U></STRONG>:
Transfection of DNA into cells via calcium phosphate is a simple,
    efficient and inexpensive method is to transfect eukaryotic cells via
    calcium phosphate co-precipitation with DNA (Graham and van der Eb, 1973).
    The insoluble calcium phosphate precipitate with the attached DNA adheres
    to the cell surface and is brought into the cells by endocytosis. Calcium
    phosphate transfection has been optimized and widely used with many
    adherent and non-adherent cell lines (Jordan et al., 1996). Calcium
    phosphate transfection can result in transient expression of the delivered
    DNA in the target cell, or establishment of stable cell lines.
</P><P><STRONG><U>Procedure</U></STRONG><STRONG>:</STRONG></P><P>The ingredients prepared according to the protocol with the GFP gene
    plasmid:
</P><OL><LI>23 ul of PUC GFP DNA ,187 ul OF 1M CaCl2, DDW up to 750ul + 750 HEBSX2</LI><LI>All ingredients were made and then filtered in 0.22 filter for sterile solution</LI><LI>The ingredients were mixed for 30 minutes in a 1.5 ml Eppendorf for the 6 wells plate (250ul of complete reagent for each well)</LI><LI>In the 6 wells there were 2 options for the transformation:
<P>
    a) Medium removed, 250ul reagent added. after 30 min new medium(2.5ml)
    added
</P><P>
    b) 250ul reagent added + old medium(2.5ml)
</P></LI><LI>After 8 hours all old medium removed and added new 2.5ml of relevantmedium</LI></OL><P><STRONG><U>Design</U>:</STRONG></P><P> </P><P>1</P><P>2</P><P>3</P><P>Option A</P><P>HEK</P><P>Astrocytes</P><P>Microglia</P><P>Option B</P><P>HEK</P><P>Astrocytes</P><P>Microglia</P><P><B><U>References:</U></B></P><OL><LI>Chen, Y., Lu, B., Yang, Q., Fearns, C., Yates, J. R., 3rd and Lee, J. D.
    (2009). Combined integrin phosphoproteomic analyses and small interfering
    RNA--based functional screening identify key regulators for cancer cell
    adhesion and migration. Cancer Res 69(8): 3713-3720.</LI><LI>Graham, F. L. and van der Eb, A. J. (1973). A new technique for the
    assay of infectivity of human adenovirus 5 DNA. Virology 52(2): 456-67.</LI><LI>Jordan, M., Schallhorn, A. and Wurm, F. M. (1996). Transfecting
    mammalian cells: optimization of critical parameters affecting
    calcium-phosphate precipitate formation. Nucleic Acids Res 24(4): 596-601.</LI></OL><A href="https://2018.igem.org/Team:BGU_Israel/Results#CalciumPhosphate" class="btn btn-info" role="button">See Experiment Result</A></DIV></DIV></DIV></DIV><DIV id="Electroporation" class="bg-grey"><DIV class="container text-center"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="row"><H1>Electroporation Transfection</H1><P><B><U>Title</U>: </B>
    Knockout of IKK-β gene in microglia cell-line.
</P><P><U><B>Conducted by</B></U><B>: </B>Einan Farhi and Mor Pasy
</P><P><B><U>Date</U>: </B>15.7.18
</P><P><B><U>Experiment goal and significance</U>:</B>
    The objective is to create a stable cell-line of the BV2 cells with a
    knockout mutation of the Inhibitor of Nuclear Factor kappa-B Kinase subunit
    beta (IKBKB) gene. With which, we aim to demonstrate how targeting the
    Nuclear Factor kappa B (NFκB) pathway will diminish the amount of
    inflammation promoting cytokines produced by the immune representative
    cells of the brain.
</P><TABLE><TBODY><TR><TD><P>
                    Experiments
                </P></TD><TD><P>
                    Protocols
                </P></TD><TD><P>
                    Notebook
                </P></TD></TR><TR><TD><P>
                    Electroporation
                </P></TD><TD><A target="_blank" style="color:#0186ac!important;" href="https://static.igem.org/mediawiki/2018/c/c1/T--BGU_Israel--protocols_exp_electroporation.pdf">Link</A></TD><TD><A target="_blank" style="color:#0186ac!important;" href="https://static.igem.org/mediawiki/2018/7/79/T--BGU_Israel--notebook_week20.pdf">Link</A></TD></TR></TBODY></TABLE><P><U><B>Theoretical background</B></U><B>:</B>
In the experiment we used a px601 commercial vector designed to express a    <I>Staphylococcus aureus</I> (SaCas9) conjugated with a Green Fluorescent
    Protein (GFP). Mor P. cloned the F4/80 promoter into the vector upstream of
    the Cas9-GFP construct instead of the original Cytomegalovirus (CMV)
    promoter. This promoter is considered to be expressed highly in microglia
    versus the other types of cells of the brain<SUP>1</SUP>. Into the guide
    RNA sequence of the vector was cloned a targeting sequence complementary to
    sequences that reside in various exons of the IKBKB gene. Generally, the
    CRISPR/Cas9 system is used to deliver a sequence-wise accurate double
    strand break which should dramatically increase the chances of a knockout
    mutation in the targeted gene<SUP>2</SUP>. The knockout of IKK-β should, in
    theory, decrease the amount of Inhibitor of kappa-B (IκB) that is sent to
    degradation and thus maintain a persistent inhibition of NFκB. A stronger
    inhibition of the NFκB complex might produce a weaker expression of its
    target genes, among them: Interleukin 1 subunit α (Il1α)<SUP> 3</SUP> and
    Tumor Necrosis Factor subunit α (TNFα)<SUP>4</SUP>.
</P><P><B><U>Procedure</U>:</B></P><OL><LI>Cloning of plasmid.</LI><LI>Electroporation with plasmid.</LI><LI>Validation of transfection success according to expression of GFP.</LI><LI>Validation of resulted mutation using the T7E1 assay.</LI><LI>Checking for a diminished expression of IKK-β using Western Blot
    analysis.</LI><LI>Cytokine assay to determine if an inhibition of the cytokine production
    was achieved&gt;</LI></OL><P>
    [Picture of the experimental procedure will be added]
</P><P><U><B>Design</B></U>
    :
</P><P>
    Each transfection mixture was pipetted evenly into 6 wells of a 24
    well-plate. DNA quantities that were used are as following: 2.5, 5, 9 μg of
    DNA. As a positive control, BV-2 cells were transfected with 5 μg pAc-GFP
    and as a negative control cells were electroporated with no plasmid and
    also seeded without electroporation. Here is a schematic diagram of the
    wells mentioned:
</P><DIV align="center"><TABLE border="1" cellspacing="1" cellpadding="0" width="0"><TBODY><TR><TD width="521" colspan="6"><P>
BV2 electroporation with px601-f4/80-g2; 2.5x10<SUP>6</SUP> cells per cuvette; A-030 program
                    </P></TD></TR><TR><TD width="87"><P>
                        2.5 μg DNA
                    </P></TD><TD width="87"><P>
                        2.5 μg DNA
                    </P></TD><TD width="87"><P>
                        2.5 μg DNA
                    </P></TD><TD width="87"><P>
                        2.5 μg DNA
                    </P></TD><TD width="87"><P>
                        2.5 μg DNA
                    </P></TD><TD width="87"><P>
                        2.5 μg DNA
                    </P></TD></TR><TR><TD width="87"><P>
                        5 μg DNA
                    </P></TD><TD width="87"><P>
                        5 μg DNA
                    </P></TD><TD width="87"><P>
                        5 μg DNA
                    </P></TD><TD width="87"><P>
                        5 μg DNA
                    </P></TD><TD width="87"><P>
                        5 μg DNA
                    </P></TD><TD width="87"><P>
                        5 μg DNA
                    </P></TD></TR><TR><TD width="87"><P>
                        9 μg DNA
                    </P></TD><TD width="87"><P>
                        9 μg DNA
                    </P></TD><TD width="87"><P>
                        9 μg DNA
                    </P></TD><TD width="87"><P>
                        9 μg DNA
                    </P></TD><TD width="87"><P>
                        9 μg DNA
                    </P></TD><TD width="87"><P>
                        9 μg DNA
                    </P></TD></TR><TR><TD width="87"><P>
                        No DNA
                    </P></TD><TD width="87"><P>
                        No DNA
                    </P></TD><TD width="87"><P>
                        No DNA
                    </P></TD><TD width="87"><P>
                        No DNA
                    </P></TD><TD width="87"><P>
                        No DNA
                    </P></TD><TD width="87"><P>
                        No DNA
                    </P></TD></TR><TR><TD width="87"><P>
                        No transfection
                    </P></TD><TD width="87"><P>
                        No transfection
                    </P></TD><TD width="87"><P>
                        No transfection
                    </P></TD><TD width="87"><P>
                        No transfection
                    </P></TD><TD width="87"><P>
                        No transfection
                    </P></TD><TD width="87"><P>
                        No transfection
                    </P></TD></TR></TBODY></TABLE></DIV><P><B><U>References</U>:</B></P><OL><LI>Helen L. Fitzsimons, Matthew J. During, CHAPTER 1 - Design and
    Optimization of Expression Cassettes Including Promoter Choice and
    Regulatory Elements, Gene Therapy of the Central Nervous System, Academic
    Press, 2006, Pages 3-16.</LI><LI>Genome engineering using the CRISPR-Cas9 system. 2013. Ran FA, Hsu PD,
    Wright J, Agarwala V, Scott DA, Zhang F. Nat Protoc. 8(11):2281-308.</LI><LI>Mori N and Prager D. (1996). Blood, 87, 3410 ± 3417.</LI><LI>Shakhov AN, Collart MA, Vassalli P, Nedospasov SA and Jongeneel CV.
    (1990). J. Exp. Med., 171, 35 ± 47.</LI></OL><A href="https://2018.igem.org/Team:BGU_Israel/Results#Electroporation" class="btn btn-info" role="button">See Experiment Result</A></DIV></DIV></DIV></DIV><DIV id="Reagent" class="bg-white"><DIV class="container text-center"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="row"><H1>Transfection by transfection-reagent</H1><P><STRONG><U>Title</U>:</STRONG> Transfection of the C8-D30 astrocytes and BV-2 microglia cell lines with PUC-GFP vector.
</P><P><B><U>Conducted by</U>:</B>Liat Tsoran and Ori Tulchinsky
</P><P><STRONG><U>Date</U>:</STRONG></P><P><STRONG><U>Aim</U>:</STRONG> In this experiment we have tried to use jetPEI-Macrophage and jetPRIME
    transfection reagent in order to transfect microglia cells (BV2) and
    astrocyte cells (C8-D30), respectively, while using HEK-cell as a positive
    control. The use of HEK cell-line as a positive control, is due to its
    transfectability by the various techniques. </P><P><STRONG><U>Importance</U>:</STRONG> This experiment was carried out in parallel with experiments using
    different methods of transfection (different reagents, calcium phosphate
    and electrophoresis) in order to find an efficient way of inserting our
plasmids into microglia for the continuation of the project.    </P><TABLE><TBODY><TR><TD><P>
                    Experiments
                </P></TD><TD><P>
                    Protocols
                </P></TD><TD><P>
                    Notebook
                </P></TD></TR><TR><TD><P>
                    Transfection of the BV-2 cell-line using jetPEI
                    transfection reagent
                </P></TD><TD><A target="_blank" style="color:#0186ac!important;" href="https://static.igem.org/mediawiki/2018/e/e6/T--BGU_Israel--protocols_exp_jetPEI.pdf">Link</A></TD><TD><A target="_blank" style="color:#0186ac!important;" href="https://static.igem.org/mediawiki/2018/3/34/T--BGU_Israel--notebook_week13.pdf">Link</A></TD></TR><TR><TD><P>
                    Transfection of the C8D30 cell-line using jetPRIME
                    transfection reagent
                </P></TD><TD><A target="_blank" style="color:#0186ac!important;" href="https://static.igem.org/mediawiki/2018/8/86/T--BGU_Israel--protocols_exp_jetPRIME.pdf">Link</A></TD></TR></TBODY></TABLE><P><STRONG><U>Theoretical background</U>:</STRONG>
jetPEI®-Macrophage allows DNA transfection of macrophages and
    macrophage-like cells. It contains a mannose-conjugated linear
    polyethylenimine that enhances binding to cells expressing mannose
    receptors, such as macrophages. jetPEI®-Macrophage is able to condense DNA
    into compact particles similarly to jetPEI®
</P><P>
    jetPRIME® is a novel powerful transfection reagent based on a polymer
    formulation manufactured at Polyplus-transfection®. jetPRIME® ensures
    effective and reproducible DNA and siRNA transfection into mammalian cells.
    jetPRIME® is extremely efficient on a wide variety of cell lines. This
    powerful reagent only requires low amounts of nucleic acid per
    transfection, hence resulting in very low cytotoxicity.
</P><P><STRONG><U>Procedure</U>:</STRONG></P><OL><LI>Day 1 - Splitting of C8-D30 cells to 6-well plate for experiment</LI><LI>Day 2 - transfection</LI><LI>Day 3 - illumination of GFP under a fluorescent microscope</LI></OL><P><STRONG><U>Design</U>:</STRONG></P><P><U>Experiment 1 – Microglia (BV2) transfection:</U></P><TABLE><TBODY><TR><TD width="125"><P> </P></TD><TD width="125"><P>1</P></TD><TD width="125"><P>2</P></TD><TD width="125"><P>3</P></TD></TR><TR><TD width="125"><P>A</P></TD><TD width="125"><P>Microglia</P></TD><TD width="125"><P>Microglia</P></TD><TD width="125"><P>Microglia</P></TD></TR><TR><TD width="125"><P>B</P></TD><TD width="125"><P>HEK</P></TD><TD width="125"><P>HEK</P></TD><TD width="125"><P>HEK</P></TD></TR></TBODY></TABLE><P><U>Experiment 2 – Astrocyte (C8-D30) transfection:</U></P><TABLE><TBODY><TR><TD width="125"><P> </P></TD><TD width="125"><P>1</P></TD><TD width="125"><P>2</P></TD><TD width="125"><P>3</P></TD></TR><TR><TD width="125"><P>A</P></TD><TD width="125"><P>Astrocytes</P></TD><TD width="125"><P>Astrocytes</P></TD><TD width="125"><P>Astrocytes</P></TD></TR><TR><TD width="125"><P>B</P></TD><TD width="125"><P>HEK</P></TD><TD width="125"><P>HEK</P></TD><TD width="125"><P>HEK</P></TD></TR></TBODY></TABLE><A href="https://2018.igem.org/Team:BGU_Israel/Results#Reagent" class="btn btn-info" role="button">See Experiment Result</A></DIV></DIV></DIV></DIV><DIV id="Microglia" class="bg-dark sub-nav"><DIV class="container text-center"><H1>Microglia Experiments</H1><DIV class="row animated fadeInRight"><A href="#Infection" class="btn btn-info" role="button"> BV2 infection</A><A href="#qPCR" class="btn btn-info" role="button">Cytokines Inhibition</A></DIV></DIV></DIV><DIV id="Infection" class="bg-grey"><DIV class="container text-center"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="row"><H1>BV2 infection by Lentivirus</H1><P><U>Title</U>: Lentivirus infection of BV2 microglia cells.<U> Conducted by</U>: Einan Farhi, Ori Tulchinsky, Liat Tsoran and Mor Pasi</P><P><U>Date</U>:<STRONG> </STRONG>30.8.18, 15.9.18</P><P><U>Aim:</U> The objective is to create a stable line of BV2 microglia cells with a genomic insertion of the shIKKb vector or a stable line with the IkBM vector.</P><P><U>Importance</U>: The insertion of the IκBαM or ShIKK vectors will cause knockdown to the IKKB regulation factor in the NFκB pathway. We aim to demonstrate how targeting the Nuclear Factor kappa B (NFκB) pathway will diminish the amount of inflammation promoting cytokines produced by the immune representative cells of the brain, microglia.</P><P><U>Procedure:</U></P><OL><LI>Production of viral vector using co-transfection of HEK293T cell line with vector and accompanying plasmids.</LI><LI>Infection of cells by spinfection method</LI><LI>Validation of infection success according to expression of GFP.</LI><LI>Selection for infected cells using puromycin. *</LI></OL><P>*Only for ShIKKb vector, as IkBM does not contain puromycin resistance gene.</P><P><U>Design</U>:</P><P>Experiment 1: BV2 spinfection with 400 µl IκBαM viral medium.</P><TABLE width="0"><TBODY><TR><TD width="36"> </TD><TD width="83"><P>1</P></TD><TD width="84"><P>2</P></TD><TD width="84"><P>3</P></TD><TD width="84"><P>4</P></TD></TR><TR><TD width="36"><P>a</P></TD><TD width="83"><P>No treatment</P></TD><TD width="84"><P>No treatment</P></TD><TD width="84"><P>No treatment</P></TD><TD width="84"><P>No treatment</P></TD></TR><TR><TD width="36"><P>b</P></TD><TD width="83"><P>IkBM</P></TD><TD width="84"><P>IkBM</P></TD><TD width="84"><P>IkBM</P></TD><TD width="84"><P>IkBM</P></TD></TR><TR><TD width="36"><P>c</P></TD><TD width="83"><P>pGreen-puro</P></TD><TD width="84"><P>pGreen-puro</P></TD><TD width="84"><P>pGreen-puro</P></TD><TD width="84"><P>pGreen-puro</P></TD></TR><TR><TD width="36"><P>d</P></TD><TD width="83"><P>No treatment</P></TD><TD width="84"><P>No treatment</P></TD><TD width="84"><P>No treatment</P></TD><TD width="84"><P>No treatment</P></TD></TR></TBODY></TABLE><P>Experiment 2: BV2 spinfection with 400 µl Shikkb viral medium</P><TABLE width="0"><TBODY><TR><TD width="34"> </TD><TD width="104"><P>1</P></TD><TD width="113"><P>2</P></TD><TD width="113"><P>3</P></TD><TD width="123"><P>4</P></TD></TR><TR><TD width="34"><P>a</P></TD><TD width="104"><P>Shikkb</P><P>Not Concentrated</P></TD><TD width="113"><P>Shikkb</P><P>Not Concentrated</P></TD><TD width="113"><P>Shikkb</P><P>Not Concentrated</P></TD><TD width="123"><P>Shikkb</P><P>Not Concentrated</P></TD></TR><TR><TD width="34"><P>b</P></TD><TD width="104"><P>Shikkb</P><P>Concentrated</P></TD><TD width="113"><P>Shikkb</P><P>Concentrated</P></TD><TD width="113"><P>Shikkb</P><P>Concentrated</P></TD><TD width="123"><P>Shikkb</P><P>Concentrated</P></TD></TR><TR><TD width="34"><P>c</P></TD><TD width="104"><P>pGreen-puro</P></TD><TD width="113"><P>pGreen-puro</P></TD><TD width="113"><P>pGreen-puro</P></TD><TD width="123"><P>pGreen-puro</P></TD></TR><TR><TD width="34"><P>d</P></TD><TD width="104"><P>No treatment</P></TD><TD width="113"><P>No treatment</P></TD><TD width="113"><P>No treatment</P></TD><TD width="123"><P>No treatment</P></TD></TR></TBODY></TABLE><P><U>References:</U></P><OL><LI>ian Liang Wu, Tatsuya Abe, Ryo Inoue, Minoru Fujiki &amp; Hidenori Kobayashi (2004) IκBαM suppresses angiogenesis and tumorigenesis promoted by a constitutively active mutant EGFR in human glioma cells, Neurological Research, 26:7, 785-791</LI><LI>Mori N and Prager D. (1996). Blood, 87, 3410 ± 3417.</LI><LI>Shakhov AN, Collart MA, Vassalli P, Nedospasov SA and Jongeneel CV. (1990). J. Exp. Med., 171, 35 ± 47.</LI><LI>Wang, X. , Li, H. , Xu, K. , Zhu, H. , Peng, Y. , Liang, A. , Li, C. , Huang, D. and Ye, W. (2016), SIRT1 expression is refractory to hypoxia and inflammatory cytokines in nucleus pulposus cells: Novel regulation by HIF‐1α and NF‐κB signaling. Cell Biol Int, 40: 716-726.</LI></OL><A href="https://2018.igem.org/Team:BGU_Israel/Results#Infection" class="btn btn-info" role="button">See Experiment Result</A></DIV></DIV></DIV></DIV><DIV id="qPCR" class="bg-white"><DIV class="container text-center"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="row"><H1>Cytokines Inhibition Assay</H1><P><U>Title</U>: Validation of IKKB knockdown through measurement of cytokine TNFa and IL1a expression in BV2 cells.</P><P><U>Conducted by</U>: Avital Bailen, Daniel Deitch and Mor Sela</P><P><U>Date</U>:  20.9.18-11.10.18</P><P><U>Aim:</U> Quantify the expression of IL1a and TNFa in BV2 microglia cell-line infected with the shIKK vector using Lentivirus, using qPCR and ELISA.</P><P><U>Importance</U>: We predict that reducing the cytokines, IL1α and TNFα, will prevent the creation of new reactive astrocytes in the brain <SUP>1</SUP>. We predict following the microglia IKKb knockdown will reduce the secretion of the mentioned cytokines and new reactive astrocytes will not be created, preventing further damage to motor neurons.</P><TABLE><TBODY><TR><TD width="208"><P>Experiments</P></TD><TD width="208"><P>Protocols</P></TD><TD width="208"><P>Notebook</P></TD></TR><TR><TD width="208"><P>qPCR</P></TD><TD width="208"><A target="_blank" style="color:#0186ac!important;" href="https://static.igem.org/mediawiki/2018/2/22/T--BGU_Israel--protocols_exp_qpcr.pdf">Link</A></TD><TD width="208"><A target="_blank" style="color:#0186ac!important;" href="https://static.igem.org/mediawiki/2018/d/da/T--BGU_Israel--notebook_exp_qpcr.pdf">Link</A></TD></TR></TBODY></TABLE><P> </P><P><U>Theoretical background</U>:</P><P>The synthesis of IL1α and TNFα cytokines is mediated by the NFkB transcription factor. NFkB activation in microglia causes motor neuron death <EM>in vitro</EM>, as well as <EM>in vivo</EM>. Heterozygous inhibition of NFkB in microglia substantially delayed disease progression in ALS mice model <SUP>2</SUP>. In the experiment we used a commercial shIKKb viral vector. This vector expresses a short hairpin RNA targeting the IKKβ mRNA. An RNA interference with the expression of IKK-β should inhibit the activation of the NFκB pathway which would produce a weaker expression of its target genes<SUP>3</SUP>, among them: Interleukin 1 subunit α (Il1α)<SUP>4</SUP> and Tumor Necrosis Factor subunit α (TNFα)<SUP>5</SUP>.</P><P><U>Procedure:</U></P><OL><LI>Infect BV2 microglia cells with shIKKb plasmid.</LI><LI>Perform selection of transfected cells with Puromycin.</LI><LI>Grow BV2 cells (wild type and infected) in 6-well plates to 80% confluence.</LI><LI>Activate the cells with LPS for 2 hours to induce microglia activation and cytokine secretion.</LI><LI>Extract RNA and create cDNA.</LI><LI>Preform ELISA with the supernatant.</LI><LI>Preform qPCR with cells.</LI></OL><P> <U>Design</U>:</P><TABLE width="0"><TBODY><TR><TD colspan="2" rowspan="3" width="80"> </TD><TD colspan="10" width="220"><P>2hr LPS</P></TD><TD colspan="12" width="323"><P>No LPS</P></TD></TR><TR><TD colspan="3" width="75"><P>+shIKK</P></TD><TD colspan="3" width="68"><P>-shIKK</P></TD><TD colspan="4" width="78"><P>WT</P></TD><TD colspan="3" width="83"><P>+shIKK</P></TD><TD colspan="4" width="89"><P>-shIKK</P></TD><TD colspan="4" width="98"><P>WT</P></TD><TD width="54"><P>no cDNA</P></TD></TR><TR><TD width="27"><P><STRONG>1</STRONG></P></TD><TD width="24"><P><STRONG>2</STRONG></P></TD><TD width="24"><P><STRONG>3</STRONG></P></TD><TD width="24"><P><STRONG>4</STRONG></P></TD><TD width="24"><P><STRONG>5</STRONG></P></TD><TD width="20"><P><STRONG>6</STRONG></P></TD><TD width="24"><P><STRONG>7</STRONG></P></TD><TD width="24"><P><STRONG>8</STRONG></P></TD><TD width="28"><P><STRONG>9</STRONG></P></TD><TD colspan="2" width="30"><P><STRONG>10</STRONG></P></TD><TD width="30"><P><STRONG>11</STRONG></P></TD><TD colspan="2" width="30"><P><STRONG>12</STRONG></P></TD><TD width="30"><P><STRONG>13</STRONG></P></TD><TD width="30"><P><STRONG>14</STRONG></P></TD><TD colspan="2" width="30"><P><STRONG>15</STRONG></P></TD><TD width="31"><P><STRONG>16</STRONG></P></TD><TD width="31"><P><STRONG>17</STRONG></P></TD><TD width="33"><P><STRONG>18</STRONG></P></TD><TD width="54"><P><STRONG>19</STRONG></P></TD></TR><TR><TD rowspan="2" width="56"><P><STRONG> </STRONG></P><P><STRONG>IL1a</STRONG></P></TD><TD width="24"><P><STRONG>A</STRONG></P></TD><TD width="27"> </TD><TD width="24"> </TD><TD width="24"> </TD><TD width="24"> </TD><TD width="24"> </TD><TD width="20"> </TD><TD width="24"> </TD><TD width="24"> </TD><TD width="28"> </TD><TD colspan="2" width="30"> </TD><TD width="30"> </TD><TD colspan="2" width="30"> </TD><TD width="30"> </TD><TD width="30"> </TD><TD colspan="2" width="30"> </TD><TD width="31"> </TD><TD width="31"> </TD><TD width="33"> </TD><TD width="54"> </TD></TR><TR><TD width="24"><P><STRONG>B</STRONG></P></TD><TD width="27"> </TD><TD width="24"> </TD><TD width="24"> </TD><TD width="24"> </TD><TD width="24"> </TD><TD width="20"> </TD><TD width="24"> </TD><TD width="24"> </TD><TD width="28"> </TD><TD colspan="2" width="30"> </TD><TD width="30"> </TD><TD colspan="2" width="30"> </TD><TD width="30"> </TD><TD width="30"> </TD><TD colspan="2" width="30"> </TD><TD width="31"> </TD><TD width="31"> </TD><TD width="33"> </TD><TD width="54"> </TD></TR><TR><TD rowspan="2" width="56"><P><STRONG>TNFa</STRONG></P></TD><TD width="24"><P><STRONG>C</STRONG></P></TD><TD width="27"><P> </P></TD><TD width="24"><P> </P></TD><TD width="24"><P> </P></TD><TD width="24"><P> </P></TD><TD width="24"><P> </P></TD><TD width="20"><P> </P></TD><TD width="24"><P> </P></TD><TD width="24"><P> </P></TD><TD width="28"><P> </P></TD><TD colspan="2" width="30"><P> </P></TD><TD width="30"><P> </P></TD><TD colspan="2" width="30"><P> </P></TD><TD width="30"><P> </P></TD><TD width="30"><P> </P></TD><TD colspan="2" width="30"><P> </P></TD><TD width="31"><P> </P></TD><TD width="31"><P> </P></TD><TD width="33"><P> </P></TD><TD width="54"><P> </P></TD></TR><TR><TD width="24"><P><STRONG>D</STRONG></P></TD><TD width="27"><P> </P></TD><TD width="24"><P> </P></TD><TD width="24"><P> </P></TD><TD width="24"><P> </P></TD><TD width="24"><P> </P></TD><TD width="20"><P> </P></TD><TD width="24"><P> </P></TD><TD width="24"><P> </P></TD><TD width="28"><P> </P></TD><TD colspan="2" width="30"><P> </P></TD><TD width="30"><P> </P></TD><TD colspan="2" width="30"><P> </P></TD><TD width="30"><P> </P></TD><TD width="30"><P> </P></TD><TD colspan="2" width="30"><P> </P></TD><TD width="31"><P> </P></TD><TD width="31"><P> </P></TD><TD width="33"><P> </P></TD><TD width="54"><P> </P></TD></TR><TR><TD rowspan="2" width="56"><P><STRONG>B-Actin</STRONG></P></TD><TD width="24"><P><STRONG>E</STRONG></P></TD><TD width="27"> </TD><TD width="24"> </TD><TD width="24"> </TD><TD width="24"> </TD><TD width="24"> </TD><TD width="20"> </TD><TD width="24"> </TD><TD width="24"> </TD><TD width="28"> </TD><TD colspan="2" width="30"> </TD><TD width="30"> </TD><TD colspan="2" width="30"> </TD><TD width="30"> </TD><TD width="30"> </TD><TD colspan="2" width="30"> </TD><TD width="31"> </TD><TD width="31"> </TD><TD width="33"> </TD><TD width="54"> </TD></TR><TR><TD width="24"><P><STRONG>F</STRONG></P></TD><TD width="27"> </TD><TD width="24"> </TD><TD width="24"> </TD><TD width="24"> </TD><TD width="24"> </TD><TD width="20"> </TD><TD width="24"> </TD><TD width="24"> </TD><TD width="28"> </TD><TD colspan="2" width="30"> </TD><TD width="30"> </TD><TD colspan="2" width="30"> </TD><TD width="30"> </TD><TD width="30"> </TD><TD colspan="2" width="30"> </TD><TD width="31"> </TD><TD width="31"> </TD><TD width="33"> </TD><TD width="54"> </TD></TR></TBODY></TABLE><P> </P><P>+shIKK - BV2 microglia cells infected with shIKK</P><P>-shIKK / spinfection -BV2 microglia cells which underwent the infection process without a plasmid</P><P>WT – WT BV2 microglia</P><P><U>Strengths and weaknesses:</U></P><P>Strengths:</P><UL><LI>Provide quantitative results.</LI><LI>Easily reproducible procedure.</LI><LI>ELISA and qPCR are very sensitive</LI></UL><P> </P><P>Weaknesses:</P><UL><LI>As these assays are very sensitive, the experiments require a certain amount of technical ability, therefore it may take several experiments before achieving reliable results.</LI></UL><P>References:</P><OL><LI>Liddelow, Shane A., et al. “Neurotoxic reactive astrocytes are induced by activated microglia.”<EM>Nature7638</EM> (2017): 481.</LI><LI>Tak, Paul P., and Gary S. Firestein. “NF-kB: a key role in inflammatory diseases.” The Journal of Clinical Investigation1 (2001): 7-11.</LI><LI>C Wang, X. , Li, H. , Xu, K. , Zhu, H. , Peng, Y. , Liang, A. , Li, C. , Huang, D. and Ye, W. (2016), SIRT1 expression is refractory to hypoxia and inflammatory cytokines in nucleus pulposus cells: Novel regulation by HIF‐1α and NF‐κB signaling. <EM>Cell Bio Int</EM> 40: 716-726.</LI><LI>D Mori N and Prager D. (1996). Blood, 87, 3410 ± 3417.</LI><LI>E Shakhov AN, Collart MA, Vassalli P, Nedospasov SA and Jongeneel CV. (1990). J. Exp. Med., 171, 35 ± 47.</LI><LI>F Das, Amitabh, et al. “Transcriptome Sequencing Reveals That LPS-Triggered Transcriptional Responses in Established Microglia BV2 Cell Lines Are Poorly Representative of Primary Microglia.” <EM>Journal of Neuroinflammation</EM>, vol. 13, no. 1, 2016, doi:10.1186/s12974-016-0644-1.</LI></OL><A href="https://2018.igem.org/Team:BGU_Israel/Results#qPCR" class="btn btn-info" role="button">See Experiment Result</A></DIV></DIV></DIV></DIV><DIV id="Astrocyte" class="bg-dark sub-nav"><DIV class="container text-center"><H1>Astrocyte Experiments</H1><DIV class="row animated fadeInRight"><A href="#AstroActive" class="btn btn-info" role="button"> Astrocyte Activation</A><A href="#Promoter" class="btn btn-info" role="button">Promoter Assay</A><A href="#Infection2" class="btn btn-info" role="button"> C8D30 Infection</A></DIV><DIV class="row animated fadeInLeft"><A href="#Caspase3_HEK" class="btn btn-info" role="button">pMLPm Activation</A><A href="#Caspase3_Astro" class="btn btn-info" role="button">Apoptosis Induction</A></DIV></DIV></DIV><DIV id="AstroActive" class="bg-grey"><DIV class="container text-center"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="row"><H1>Astrocyte Activation</H1><P><STRONG><U>Title</U>: </STRONG>Activation of astrocyte cell line C8-D30 using lipopolysaccharide (LPS) and pro-inflammatory cytokines.<U><STRONG>Conducted by</STRONG></U><STRONG>: </STRONG>Mor Sela</P><P><STRONG><U>Date</U>: </STRONG>29.7.18-2.8.18</P><P><STRONG><U>Aim:</U></STRONG>Activation of astrocytes was performed to confirm that our C8D30 astrocyte cell line can accurately model resting and reactive astrocytes for our experimental design.</P><P><STRONG><U>Importance</U></STRONG>:Our project is based on the assumption that reactive astrocytes are a main factor in ALS and therefore our product is designed to specifically disarm them. Without a “reactive astrocyte” experimental group, we can not test the efficiency and specificity of our product in reactive astrocytes when compared to other cells in the system.</P><TABLE><TBODY><TR><TD width="208"><P>Experiments</P></TD><TD width="208"><P>Protocols</P></TD><TD width="208"><P>Notebook</P></TD></TR><TR><TD width="208"><P>ELISA</P></TD><TD width="208"><A target="_blank" style="color:#0186ac!important;" href="https://static.igem.org/mediawiki/2018/3/3e/T--BGU_Israel--protocols_exp_elisa_c3.pdf">Link</A></TD><TD width="208"><A target="_blank" style="color:#0186ac!important;" href="https://static.igem.org/mediawiki/2018/2/2d/T--BGU_Israel--notebook_week16.pdf">Link</A></TD></TR><TR><TD width="208"><P>Western Blot</P></TD><TD width="208"><A target="_blank" style="color:#0186ac!important;" href="https://static.igem.org/mediawiki/2018/0/08/T--BGU_Israel--protocols_exp_western_blot.pdf">Link</A></TD><TD width="208"><A target="_blank" style="color:#0186ac!important;" href="https://static.igem.org/mediawiki/2018/f/f1/T--BGU_Israel--notebook_week24.pdf">Link</A></TD></TR></TBODY></TABLE><P> </P><P><STRONG><U>Theoretical background:</U></STRONG> Reactive astrocytes are found in several forms, including forms called A1 and A2. Gene expression analysis in reactive astrocytes has shown that reactive astrocytes type A1 express many genes that are detrimental to synapses (such as complement cascade genes)<SUP>1</SUP>. Studies show that A1 reactive astrocytes are produced as a result of an NFkB protein signal. A process termed nuclear factor kappa light- chain enhancer of activated B cells<SUP>2</SUP>.  Reactive astrocytes type A2 over-express neurotropic factors, which promote synapse repair. Meaning A1 reactive astrocytes are harmful while A2 reactive astrocytes are helpful to the central nervous system (CNS) and brain<SUP>1</SUP>.</P><P>Recently, researchers have developed a model of exclusively A1 reactive astrocytes in cell culture which allows for targeted research. In this model, researchers quickly extract astrocytes from brain tissue which has not yet been damaged. Then these astrocytes are grown in tissue culture with the relevant medium. Finally, a cocktail of cytokines, taken from the medium of activated microglia, is added to the astrocyte culture. At this point, the astrocytes exhibit an A1 reactive astrocytes phenotype<SUP>1</SUP>.</P><P>Microglia can be activated <EM>in vivo</EM> by inducing chronic CNS damage or by injection of lipopolysaccharides (LPS). Activated microglia secrete three cytokines which induce A1 reactive astrocytes: interleukin 1 alpha (IL1a), tumor necrosis factor alpha (TNFa), and complement component 1q (C1q). Adding LPS or these three cytokines <EM>in vitro</EM> produces A1 reactive astrocytes with a genetic profile very similar to A1 reactive astrocytes <EM>in vivo</EM>.<SUP>1</SUP></P><P>Experiments on this model have shown that A1 reactive astrocytes lose almost all functions displayed by resting astrocytes. A1 reactive astrocytes have very low ability to create synapses, can not perform phagocytosis, and do not induce neuronal rehabilitation or growth.</P><P>Single cell data has shown that the complement component C3 is a preferred reactive astrocytes marker to glial fibrillary acidic protein (GFAP). C3 is specific to A1 reactive astrocytes, over A2 reactive astrocytes and resting astrocytes<SUP>3</SUP>. Meaning A1 reactive astrocytes secrete many classical complement cascade components which accelerate synapse degeneration and other toxic substances which cause damage to neurons and oligodentrocytes<SUP>,4,5,6</SUP>.</P><P> </P><P><STRONG>Procedure:</STRONG></P><P> </P><OL><LI>Activation of Microglia (BV2 cell line) by adding LPS to their medium inducing the secretion of cytokines<SUP>7,8</SUP></LI><LI>After 24 hours – transfer Microglia medium (cytokines +) to the “resting” Astrocyte (C8D30) wells <STRONG>(Activation step)<SUP>1</SUP></STRONG></LI><LI>Adding commercial cytokines (IL-1a, TNF, C1q) to different “resting” Astrocyte wells <STRONG>(Activation step).</STRONG><STRONG><SUP>1</SUP></STRONG></LI><LI>Validation of astrocyte reactivity - using ELISA and Western Blot to measure the expression of C3 protein in the samples<SUP>9</SUP></LI></OL><P>[procedure img]</P><P><STRONG>Design:</STRONG></P><P>Experiment 1 – Measurement of C3 in activated astrocytes using ELISA</P><TABLE width="0"><TBODY><TR><TD width="122"><P>C8D30 who grow with ACM + LPS (from microglia plate)</P></TD><TD width="122"><P>C8D30 who grow with MCM + LPS (from microglia plate) + ACM</P></TD><TD width="122"><P>C8D30 who grow with MCM (from microglia plate) without LPS +  ACM <STRONG>Negative control</STRONG></P></TD><TD width="122"><P>MCM without Microglia cells + LPS (from microglia plate) +  </P><P>ACM + C8D30</P><P><STRONG>Negative control</STRONG></P></TD><TD width="122"><P> </P></TD></TR><TR><TD width="122"><P>V</P></TD><TD width="122"><P>V</P></TD><TD width="122"><P>V</P></TD><TD width="122"><P>V</P></TD><TD width="122"><P>Biological repetition 1</P></TD></TR><TR><TD width="122"><P>V</P></TD><TD width="122"><P>V</P></TD><TD width="122"><P>X</P></TD><TD width="122"><P>      X</P></TD><TD width="122"><P>Biological repetition 2</P></TD></TR><TR><TD width="122"><P>X</P></TD><TD width="122"><P>V</P></TD><TD width="122"><P>      X</P></TD><TD width="122"><P>      X</P></TD><TD width="122"><P>Biological repetition 3</P></TD></TR><TR><TD width="122"><P>X</P></TD><TD width="122"><P>V</P></TD><TD width="122"><P>      X</P></TD><TD width="122"><P>      X</P></TD><TD width="122"><P>Biological repetition 4</P></TD></TR></TBODY></TABLE><P> </P><P>Experiment 2 – Measurement of C3 in activated astrocytes using Western Blot</P><TABLE width="0"><TBODY><TR><TD width="74"><P>ACM + 3 cytokines + C8D30 after 48hr.</P></TD><TD width="74"><P>ACM + 3 cytokines + C8D30 after 24 hr.</P></TD><TD width="75"><P>ACM + C8D30</P><P> </P><P><STRONG>Negative control</STRONG></P></TD><TD width="76"><P>C8D30 who grow with ACM + LPS (from microglia plate) after 48 hr.</P></TD><TD width="76"><P>C8D30 who grow with ACM + LPS (from microglia plate) after 24 hr.</P></TD><TD width="76"><P>C8D30 who grow with MCM + LPS (from microglia plate) after 48 hr.</P></TD><TD width="76"><P>C8D30 who grow with MCM + LPS (from microglia plate) after 24 hr.</P></TD><TD width="75"><P>MCM without LPS + C8D30</P><P><STRONG>Negative control</STRONG></P></TD><TD width="78"><P> </P></TD></TR><TR><TD width="74"><P>V</P></TD><TD width="74"><P>V</P></TD><TD width="75"><P>V</P></TD><TD width="76"><P>V</P></TD><TD width="76"><P>V</P></TD><TD width="76"><P>V</P></TD><TD width="76"><P>V</P></TD><TD width="75"><P>V</P></TD><TD width="78"><P>Biological repetition 1</P></TD></TR><TR><TD width="74"><P>V</P></TD><TD width="74"><P>V</P></TD><TD width="75"><P>X</P></TD><TD width="76"><P>V</P></TD><TD width="76"><P>V</P></TD><TD width="76"><P>V</P></TD><TD width="76"><P>V</P></TD><TD width="75"><P>      X</P></TD><TD width="78"><P>Biological repetition 2</P></TD></TR></TBODY></TABLE><P> </P><P> </P><P><STRONG><U>Strengths</U></STRONG><STRONG><U>and weaknesses:</U></STRONG></P><P>Strengths:</P><UL><LI>Adding LPS to microglia medium mimics the <EM>in vivo</EM> conditions and process better than adding commercial cytokines directly to astrocyte medium</LI><LI>This activation process is fast and simple.</LI><LI>Western Blot Analysis:</LI></UL><OL><LI>Sensitivity – detect protein at very low concentrations (0.1 ng protein per sample).</LI><LI>Specificity- Gel electrophoresis sorts each protein sample according to size, shape, and charge. The observed bands give an indication as to the size of the protein or polypeptide. Additionally, as the detection is based on antibody binding, the process can locate a specific protein in a sample of over 300,000 different proteins.</LI></OL><UL><LI>ELISA (Enzyme-Linked Immunosorbent Assay)<SUP>10</SUP></LI></UL><OL><LI>Relatively cheap reagents with a long shelf life.</LI><LI>Sensitivity and specificity higher than western blot analysis.</LI><LI>No radiation, as opposed to exposure during binding the antibody to the protein or disposing of chemicals from western blot analysis.</LI><LI>Faster and easier procedure than western blot analysis.</LI><LI>The results are quantitative rather than qualitative (as in western blot analysis).</LI><LI>Applicable for a wide range of proteins.</LI></OL><P> </P><P>Weaknesses:</P><UL><LI>In the article Liddelow, 2017<SUP>1</SUP> the activation is induced by injecting mouse models with LPS rather than adding LPS to cell culture. Therefore, the strength of the activation may be lower in our experiment. Additionally, we do not measure the concentration of the cytokines produced after the microglia are incubated with LPS overnight. Meaning that the astrocyte activation is induced with an unknown concentration of cytokines, as well as other unknown factors found in the medium.</LI><LI>Activation of astrocytes in this way does not fully mimic the process <EM>in vivo</EM>. Although there is evidence that these three cytokines are enough to induce reactive astrocytes, it is possible that other factors are involved <EM>in vivo</EM> which may affect the phenotype.</LI><LI>In Liddelow, 2017<SUP>1</SUP> primary astrocytes are activated, while we are working with C8D30 cell lines. Therefore, the protocol may not correspond exactly.</LI></UL><P><STRONG><U>References:</U></STRONG></P><OL><LI>Liddelow, S.A., Guttenplan, K.A., Clarke, L.E., Bennett, F.C., Bohlen, C.J., Schirmer, L., Bennett, M.L., M€unch, A.E., Chung, W.S., Peterson, T.C., et al.(2017). Neurotoxic reactive astrocytes are induced activated microglia. Nature541, 481–487.</LI><LI>Lian, H., Yang, L., Cole, A., Sun, L., Chiang, A.C.-A., Fowler, S.W., Shim, D.J., Rodriguez-Rivera, J., Taglialatela, G., Jankowsky, J.L., et al. (2015). NFkB activated astroglial release of complement C3 compromises neuronal morphology and function associated with Alzheimer’s disease. Neuron 85, 101–115.</LI><LI>&quot;Reactive Astrocytes: Production, Function, and Therapeutic Potential&quot; Shane A. Liddelow1,* and Ben A. Barres1,* 1Department of Neurobiology, Stanford University School of Medicine, Stanford, CA 94305, USA *Correspondence: liddelow@stanford.edu (S.A.L.), barres@stanford.edu (B.A.B.)</LI><LI>Stevens, B., Allen, N.J., Vazquez, L.E., Howell, G.R., Christopherson, K.S., Nouri, N., Micheva, K.D., Mehalow, A.K., Huberman, A.D., Stafford, B., et al. (2007). The classical complement cascade mediates CNS synapse elimination. Cell 131, 1164–1178.</LI><LI>Hong, S., Beja-Glasser, V.F., Nfonoyim, B.M., Frouin, A., Li, S., Ramakrishnan, S., Merry, K.M., Shi, Q., Rosenthal, A., Barres, B.A., et al. (2016). Complement and microglia mediate early synapse loss in Alzheimer mouse models. Science 352, 712–716.</LI><LI>Sekar, A., Bialas, A.R., de Rivera, H., Davis, A., Hammond, T.R., Kamitaki, N., Tooley, K., Presumey, J., Baum, M., Van Doren, V., et al. (2016). Schizophrenia risk from complex variation of complement component 4. Nature 530,177–183.</LI><LI>&quot;Activation of BV2 microglia by lipopolysaccharide triggers an inflammatory reaction in PC12 cell apoptosis through a toll-like receptor 4-dependent pathway&quot; Xiao-jing Dai, Na Li, Le Yu, Zi-yang Chen, Rong Hua, Xia Qin, and Yong-Mei Zhang</LI><LI>&quot;Development of an Insert Co-culture System of Two Cellular Types in the Absence of Cell-Cell Contact.&quot; Renaud J1, Martinoli MG2.</LI><LI>Lindblom, Rickard PF, et al. &quot;Unbiased expression mapping identifies a link between the complement and cholinergic systems in the rat central nervous system.&quot; The Journal of Immunology 192.3 (2014): 1138 1153.‏</LI><LI>&quot;Advantages, Disadvantages and Modifications of Conventional ELISA&quot; Samira Hosseini, Patricia Vázquez Villegas,Marco Rito-Palomares,Sergio O. Martinez-Chapa 31 December 2017</LI></OL><A href="https://2018.igem.org/Team:BGU_Israel/Results#AstroActive" class="btn btn-info" role="button">See Experiment Result</A></DIV></DIV></DIV></DIV><DIV id="Promoter" class="bg-white"><DIV class="container text-center"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="row"><H1>Promoter Assay</H1><P><STRONG><U>Title</U>: </STRONG>Promoter assay for Timp1 and Steap4 promoters in reactive astrocytes.</P><P><STRONG><U>Conducted by</U>: </STRONG>Nitzan Keidar and Mor Sela</P><P><STRONG><U>Date</U>: </STRONG>24.9.18-28.9.18</P><P><STRONG><U>Aim:</U></STRONG> Our goal in this experiment is to assess the strength and the specificity of the promoters Timp1 and Steap4 by quantifying the amount of luminescence produced by the Luciferase enzyme cloned downstream of these promoters, under our experimental conditions.</P><P><STRONG><U>Importance: </U></STRONG>Our project is based on the assumption that reactive astrocytes can be targeted based on specific genetic markers (e.g Timp1 and Steap4). Non-specific expression can lead to off target activity such as healthy resting astrocytes, microglia or other neighboring brain cells.</P><TABLE><TBODY><TR><TD width="208"><P>Experiments</P></TD><TD width="208"><P>Protocols</P></TD><TD width="208"><P>Notebook</P></TD></TR><TR><TD width="208"><P>Promoter assay in reactive astrocytes</P></TD><TD width="208"><A target="_blank" style="color:#0186ac!important;" href="https://static.igem.org/mediawiki/2018/b/b4/T--BGU_Israel--protocols_exp_promoterassay.pdf">Link</A></TD><TD width="208"><A target="_blank" style="color:#0186ac!important;" href="https://static.igem.org/mediawiki/2018/5/51/T--BGU_Israel--notebook_exp_promoter_assay.pdf">Link</A></TD></TR></TBODY></TABLE><P> </P><P><STRONG><U>Theoretical background</U></STRONG>:</P><P>“Reactive astrocytes&quot; change their gene expression profile relative to quiescent astrocytes. Two such distinguishing genetic markers are Steap4 and Timp1 genes, expressed exclusively in reactive astrocytes<SUP>1-4</SUP>. Genetic reporter systems are widely used to study eukaryotic gene expression and cellular physiology.</P><P>Our promoter assay kit is a &quot;Dual-Luciferase® Reporter Assay System&quot; of Promega. The term “dual reporter” refers to the simultaneous expression and measurement of two individual reporter enzymes within a single system.</P><P>Typically, the “experimental” reporter is correlated with the effect of specific experimental conditions, while the activity of the co-transfected “control” reporter provides an internal control that serves as the baseline response. Normalizing the activity of the experimental reporter to the activity of the internal control minimizes experimental variability caused by differences in cell viability or transfection efficiency.</P><P>Thus, dual-reporter assays often allow more reliable interpretation of the experimental data by reducing external influences.</P><P>We used pGL3 series of firefly and Renilla luciferase vectors for the DLR™ Assay Systems. Our vectors are: [Add picture of experiment plasmids]</P><P> </P><P><STRONG><U>Procedure</U></STRONG><STRONG>:</STRONG></P><OL><LI>Co-transfect cells with plasmids pGL3+Timp1/ PGL3+Steap4 and Renilla+T7</LI><LI>Allow translation of Luciferese enzyme (48 hours).</LI><LI>Cell lysis to release Luc enzymes</LI></OL><P>In luminometer:</P><OL><LI>Provide enzymes with substrate and co-factors to produce light.</LI><LI>Measure light emission against controls. Renilla Luc correspond to efficiency of transfection, Firefly Luc correspond to strength of promoter.</LI></OL><P>[picture of experimental procedure]</P><P> </P><P> </P><P><STRONG><U>Design</U></STRONG><STRONG>:</STRONG></P><TABLE width="0"><TBODY><TR style="height: 48px;"><TD style="height: 48px;" width="121"><P>1. pGL3 + Timp1 &amp; Renilla</P></TD><TD style="height: 48px;" width="121"><P>2. pGL3 + Timp1 &amp; Renilla</P></TD><TD style="height: 48px;" width="121"><P>3. pGL3 + Timp1 &amp; Renilla</P></TD><TD style="height: 48px;" width="98"><P>4. Puc GFP</P></TD><TD style="height: 48px;" width="98"><P>5. Puc GFP</P></TD><TD style="height: 48px;" width="98"><P>6.</P></TD></TR><TR style="height: 48px;"><TD style="height: 48px;" width="121"><P>7. pGL3 + Steap4 &amp; Renilla</P></TD><TD style="height: 48px;" width="121"><P>8. pGL3 + Steap4 &amp; Renilla</P></TD><TD style="height: 48px;" width="121"><P>9. pGL3 + Steap4 &amp; Renilla</P></TD><TD style="height: 48px;" width="98"><P>10. Enhancer E7 + Renilla</P></TD><TD style="height: 48px;" width="98"><P>11. Enhancer E7 + Renilla</P></TD><TD style="height: 48px;" width="98"><P>12. Enhancer E7 + Renilla</P></TD></TR><TR style="height: 48px;"><TD style="height: 48px;" width="121"><P>13. pGL3 no promoter &amp; Renilla</P></TD><TD style="height: 48px;" width="121"><P>14. pGL3 no promoter &amp; Renilla</P></TD><TD style="height: 48px;" width="121"><P>15. pGL3 no promoter &amp; Renilla</P></TD><TD style="height: 48px;" width="98"><P>16. Enhancer E9 + Renilla</P></TD><TD style="height: 48px;" width="98"><P>17. Enhancer E9 + Renilla</P></TD><TD style="height: 48px;" width="98"><P>18. Enhancer E9 + Renilla</P></TD></TR><TR style="height: 35px;"><TD style="height: 35px;" width="121"><P>19. No transfection</P></TD><TD style="height: 35px;" width="121"><P>20. No transfection</P></TD><TD style="height: 35px;" width="121"><P>21. No transfection</P></TD><TD style="height: 35px;" width="98"><P>22.</P></TD><TD style="height: 35px;" width="98"><P>23.</P></TD><TD style="height: 35px;" width="98"><P>24.</P></TD></TR></TBODY></TABLE><P><STRONG><U> </U></STRONG></P><P><STRONG><U>References:</U></STRONG></P><OL><LI>Zamanian, Jennifer L., et al. &quot;Genomic analysis of reactive astrogliosis.&quot; <EM>Journal of neuroscience</EM>18 (2012): 6391-6410.‏</LI><LI>Zhang, Ye, et al. &quot;An RNA-sequencing transcriptome and splicing database of glia, neurons, and vascular cells of the cerebral cortex.&quot; <EM>Journal of Neuroscience</EM>36 (2014): 11929-11947.‏</LI><LI>Tokuda, Eiichi, Eriko Okawa, and Shin‐ichi Ono. &quot;Dysregulation of intracellular copper trafficking pathway in a mouse model of mutant copper/zinc superoxide dismutase‐linked familial amyotrophic lateral sclerosis.&quot; <EM>Journal of neurochemistry</EM>1 (2009): 181-191.‏</LI><LI>Lorenzl, S., et al. &quot;Tissue inhibitors of matrix metalloproteinases are elevated in cerebrospinal fluid of neurodegenerative diseases.&quot; <EM>Journal of the neurological sciences</EM>1-2 (2003): 71-76.‏</LI></OL><A href="https://2018.igem.org/Team:BGU_Israel/Results#Promoter" class="btn btn-info" role="button">See Experiment Result</A></DIV></DIV></DIV></DIV><DIV id="Infection2" class="bg-grey"><DIV class="container text-center"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="row"><H1>C8D30 infection by Lentivirus</H1><P><U>Title</U>:<STRONG> </STRONG>Lentivirus infection of C8D30 Astrocyte and HEK293 cells.<U> Conducted by</U>: Einan Farhi, Ori Tulchinsky, Liat Tsoran and Mor Pasi</P><P><U>Date</U>: <STRONG> </STRONG>30.8.18, 15.9.18</P><P><U>Aim:</U> The objective is to create a stable line of the C8D30 astrocytes with a genomic insertion of the CMV-dCas9-VP64 vector and another with the Timp1-dCas9-VP64 vector.</P><P><U>Importance</U>: The CMV-dCas9-VP64 and TIMP1-dCas9-VP64 vectors are the first component in the two-component system with which we aim to target and eliminate reactive astrocytes.</P><P> </P><P>Theoretical background:</P><P>C8D30 cell line is an astroglioma cell line with which we aim to model reactive astrocytes of the brain. Reactive astrocytes are different from resting astrocytes in their gene expression profile. Our designed vectors express the first part of our two-component system under a promoter of a gene known to be highly expressed in reactive astrocytes, Timp1 <SUP>1-4</SUP>. As a backup, we have the same component designed to be expressed under a constitutive strong promoter, CMV. The construct to be expressed according to the activity of said promoters is an enzymatically inactive Cas9 (dCas9) conjugated to a transcription promoting factor, VP64<SUP>5</SUP>. This recombinant protein is designed to activate specific endogenous genes in a guide RNA dependent manner.  The vector also expresses a Puromycin resistance gene.</P><P>Procedure:</P><OL><LI>Production of viral vector using co-transfection of HEK293T cell line with vector and accompanying plasmids.</LI><LI>Infection of cells by spinfection method</LI><LI>Validation of infection success according to expression of GFP.</LI><LI>Selection for infected cells using puromycin. *</LI></OL><P>*Only for shIKKb vector, as IkBM does not contain puromycin resistance gene.</P><P><U>Design</U>:</P><P>C8D30 spinfection with 500 µl CMV-dCas9-VP64 and Timp1-dCas9-VP64 viral medium.</P><P> </P><TABLE width="0"><TBODY><TR><TD width="36"><P> </P></TD><TD width="93"><P>1</P></TD><TD width="93"><P>2</P></TD><TD width="93"><P>3</P></TD><TD width="93"><P>4</P></TD></TR><TR><TD width="36"><P>a</P></TD><TD width="93"><P>CMV-dCas9-VP64</P></TD><TD width="93"><P>CMV-dCas9-VP64</P></TD><TD width="93"><P>CMV-dCas9-VP64</P></TD><TD width="93"><P>CMV-dCas9-VP64</P></TD></TR><TR><TD width="36"><P>b</P></TD><TD width="93"><P>Timp1-dCas9-VP64</P></TD><TD width="93"><P>Timp1-dCas9-VP64</P></TD><TD width="93"><P>Timp1-dCas9-VP64</P></TD><TD width="93"><P>Timp1-dCas9-VP64</P></TD></TR><TR><TD width="36"><P>c</P></TD><TD width="93"><P>pGreen-puro</P></TD><TD width="93"><P>pGreen-puro</P></TD><TD width="93"><P>pGreen-puro</P></TD><TD width="93"><P>pGreen-puro</P></TD></TR><TR><TD width="36"><P>d</P></TD><TD width="93"><P>No treatment</P></TD><TD width="93"><P>No treatment</P></TD><TD width="93"><P>No treatment</P></TD><TD width="93"><P>No treatment</P></TD></TR></TBODY></TABLE><P>HEK293T spinfection with 400 µl CMV-VP64 and Timp1-VP64 viral medium</P><TABLE width="0"><TBODY><TR><TD width="36"> </TD><TD width="93"><P>1</P></TD><TD width="93"><P>2</P></TD><TD width="93"><P>3</P></TD><TD width="93"><P>4</P></TD></TR><TR><TD width="36"><P>a</P></TD><TD width="93"><P>CMV-dCas9-VP64</P></TD><TD width="93"><P>CMV-dCas9-VP64</P></TD><TD width="93"><P>CMV-dCas9-VP64</P></TD><TD width="93"><P>CMV-dCas9-VP64</P></TD></TR><TR><TD width="36"><P>b</P></TD><TD width="93"><P>Timp1-dCas9-VP64</P></TD><TD width="93"><P>Timp1-dCas9-VP64</P></TD><TD width="93"><P>Timp1-dCas9-VP64</P></TD><TD width="93"><P>Timp1-dCas9-VP64</P></TD></TR><TR><TD width="36"><P>c</P></TD><TD width="93"><P>pGreen-puro</P></TD><TD width="93"><P>pGreen-puro</P></TD><TD width="93"><P>pGreen-puro</P></TD><TD width="93"><P>pGreen-puro</P></TD></TR><TR><TD width="36"><P>d</P></TD><TD width="93"><P>No treatment</P></TD><TD width="93"><P>No treatment</P></TD><TD width="93"><P>No treatment</P></TD><TD width="93"><P>No treatment</P></TD></TR></TBODY></TABLE><P><U>References</U>:</P><OL><LI>Zamanian, Jennifer L., et al. &quot;Genomic analysis of reactive astrogliosis.&quot; Journal of neuroscience18 (2012): 6391-6410.‏</LI><LI>Zhang, Ye, et al. &quot;An RNA-sequencing transcriptome and splicing database of glia, neurons, and vascular cells of the cerebral cortex.&quot; Journal of Neuroscience36 (2014): 11929-11947.‏</LI><LI>Tokuda, Eiichi, Eriko Okawa, and Shin‐ichi Ono. &quot;Dysregulation of intracellular copper trafficking pathway in a mouse model of mutant copper/zinc superoxide dismutase‐linked familial amyotrophic lateral sclerosis.&quot; Journal of neurochemistry1 (2009): 181-191.‏</LI><LI>Lorenzl, S., et al. &quot;Tissue inhibitors of matrix metalloproteinases are elevated in cerebrospinal fluid of neurodegenerative diseases.&quot; Journal of the neurological sciences1-2 (2003): 71-76.‏</LI><LI>Maeder ML, Linder SJ, Cascio VM, Fu Y, Ho QH, Joung JK. CRISPR RNA-guided activation of endogenous human genes. Nature methods. 2013;10(10):977-979. doi:10.1038/nmeth.2598.</LI></OL><P> </P><A href="https://2018.igem.org/Team:BGU_Israel/Results#Infection2" class="btn btn-info" role="button">See Experiment Result</A></DIV></DIV></DIV></DIV><DIV id="Caspase3_HEK" class="bg-white"><DIV class="container text-center"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="row"><H1>pMLPm Activation</H1><P><U>Title</U>: Validation of pMLPm activation with dCas9-VP64-gRNA and induction of apoptotic signal in HEK293 cells<U>Conducted by</U>: Liat Tsoran, Mor Pasi and Ori Tulchinsky</P><P><U>Date</U>:<STRONG>  </STRONG>30/9/18-14/10/18</P><P><U>Aim:</U>In this experiment we wanted to validate our synthetic engineered system that should induce apoptotic death by the activation of the synthetic minimal adenovirus major late promoter (pMLPm) via dCas9-VP64-gRNA complex. This experiment is a control experiment aimed at demonstrating the specificity of our system. We have hypothesized that if our system is indeed specfici enough, we will not be able to induce apoptosis in HEK293 cells, as these are human kidney cells and not murine reactive astrocytes cell line, the real target of our system.  </P><P><U>Importance</U>: In our project, one of our goals was to engineer a dCas9-VP64-gRNA system that could activate the pMLPm promoter to induce apoptotic death by the expression of exogenous reverse caspas3. Since we are using staphylococcus aureus dCas9-VP64-gRNA, we made changes in the pMLPm Protospacer Adjacent Motif (PAM), and now we must verify pMLPm regulating activity trough the expression of an mCherry fluorophore. We will also verify the apoptotic triggering abilities of the exogenous reverse caspas3 that is used in our project.</P><P><U>Theoretical background</U>:</P><P>The synthetic minimal adenovirus major late promoter (pMLPm) contains three repetitions of the &quot;a1&quot; sequence, which are complementary to the gRNA sequence, enabling dCas9-VP64-gRNA complex to target the pMLPm synthetic promoter<SUP>1</SUP>.When the transcription factor VP64 (an engineered tetramer of the herpes simplex VP16 transcriptional activator domain) is fused to dCas9 enzyme, it can be guided to a specific location in the genome, in this manner we can exploit it to promote downstream translation<SUP>2</SUP>.</P><P>The dCas9-VP64-gRNA complex will target the pMLPm synthetic promoter and promote the expression of exogenous reverse caspase3, that in contrast to the endogenous caspas3 will be activated after transcription due to its autocatalytic processing, meaning that this enzyme will trigger an apoptotic signal that will lead to apoptotic death  (caspase3 is responsible for chromatin condensation and DNA fragmentation)<SUP>3</SUP>.</P><P>The mCherry reporter protein is also expressed by the pMLPm promoter in our system, it is fused to the exogenous reverse caspase3 with Thoseaasigna virus 2A (T2A) peptide that is cleaved during translation<SUP>4</SUP>.</P><P>In order to activate pMLPm promoter and to induce apoptotic death we used 2 plasmids:</P><OL><LI><U>A Lenti viral plasmid: CMV dCas9 VP64</U>– in this plasmid CMV promoter express dCas9 enzyme which is fused to the transcription factor VP64.</LI><LI><U>pSynt CMV</U>– in this plasmid CMV promoter express the gRNA. And pMLPm promoter regulate the downstream translation of exogenous reverse caspase3 which is fused to mCherry fluorophore.</LI></OL><P>Cells that were co-transfection with both plasmids will show mCherry expression and apoptotic death will be triggered. In order to detect leakage in pMLPm promoter we used a control of cells that were transfected only with pSynt CMV plasmid.</P><P><U>Procedure</U>:</P><P><U>Day 1</U>-  Splitting of HEK293 cells to 6 wells plates</P><P><U>Day 2</U>- transfection with TurboFect™ Transfection Reagent</P><P><U>Day 3</U>- We verified mCherry expression in fluorescent microscope (24 hours after transfection).</P><P><U>Day 4</U>:</P><OL><LI><STRONG>validation of pMLPm activation with dCas9-VP64-gRNA-</STRONG></LI><UL><LI>We used ARIA FACS in order to document mCherry expression (48 hours after transfection).</LI><LI>We used confocal microscope in order to document mCherry expression (48 hours after transfection).</LI></UL></OL><OL start="2"><LI><STRONG>induction of apoptotic signal -</STRONG> We used APC Annexin V/Dead Cell Apoptosis Kit with APC annexin V and SYTOX® Green for Flow Cytometry</LI></OL><P><U>Design:</U></P><OL><LI><STRONG> validation of pMLPm activation with dCas9-VP64-gRNA-</STRONG></LI><P><U>Plate A: </U>FACS</P><TABLE width="0"><TBODY><TR><TD width="242"><P>sample number</P></TD><TD width="149"><P>1</P></TD><TD width="144"><P>2</P></TD><TD width="144"><P>3</P></TD></TR><TR><TD width="242"><P>Transfection plasmids</P></TD><TD width="149"><P>Lenti: CMV dCas9 VP64 + pSynt CMV</P></TD><TD width="144"><P>pSynt CMV</P></TD><TD width="144"><P>Empty plasmid</P></TD></TR><TR><TD width="242"><P>wells</P></TD><TD width="149"><P>A1+B1</P></TD><TD width="144"><P>A2+B2</P></TD><TD width="144"><P>A3+B3</P></TD></TR></TBODY></TABLE><P><U> </U></P><P><U>Plate B</U>: confocal microscope</P><P><U> </U>     </P><TABLE width="0"><TBODY><TR><TD width="242"><P>sample number</P></TD><TD width="149"><P>1</P></TD><TD width="144"><P>2</P></TD><TD width="144"><P>3</P></TD></TR><TR><TD width="242"><P>Transfection plasmids</P></TD><TD width="149"><P>Lenti: CMV dCas9 VP64 + pSynt CMV</P></TD><TD width="144"><P>pSynt CMV</P></TD><TD width="144"><P>Empty plasmid</P></TD></TR><TR><TD width="242"><P>wells</P></TD><TD width="149"><P>A1</P></TD><TD width="144"><P>A2</P></TD><TD width="144"><P>A3</P></TD></TR></TBODY></TABLE><P><STRONG> </STRONG></P></OL><OL start="2"><LI><STRONG> induction of apoptotic signal:</STRONG></LI><P><U>Plate A</U>:  </P><TABLE width="0"><TBODY><TR><TD width="295"><P>sample number</P></TD><TD width="181"><P>1</P></TD><TD width="176"><P>2</P></TD></TR><TR><TD width="295"><P>sample name</P></TD><TD width="181"><P>CMV dCas + CMV pSynt</P></TD><TD width="176"><P>pSynt CMV</P></TD></TR><TR><TD width="295"><P>Transfection plasmids</P></TD><TD width="181"><P>Lenti: CMV dCas9 VP64 + pSynt CMV</P></TD><TD width="176"><P>pSynt CMV</P></TD></TR><TR><TD width="295"><P>wells</P></TD><TD width="181"><P>A1-3</P></TD><TD width="176"><P>B1-3</P></TD></TR><TR><TD width="295"><P>APC annexin V</P></TD><TD width="181"><P>+</P></TD><TD width="176"><P>+</P></TD></TR><TR><TD width="295"><P>Sytox green</P></TD><TD width="181"><P>+</P></TD><TD width="176"><P>+</P></TD></TR></TBODY></TABLE><P><U>Plate B</U>:  </P><TABLE width="0"><TBODY><TR><TD width="295"><P>sample number</P></TD><TD width="181"><P>3</P></TD><TD width="176"><P>4</P></TD></TR><TR><TD width="295"><P>sample name</P></TD><TD width="181"><P>mCherry control (for FACS calibration)</P></TD><TD width="176"><P>annexin control (for FACS calibration)</P></TD></TR><TR><TD width="295"><P>Transfection plasmids</P></TD><TD width="181"><P>Lenti: CMV dCas9 VP64 + pSynt CMV</P></TD><TD width="176"><P>Lenti: CMV dCas9 VP64 + pSynt CMV</P></TD></TR><TR><TD width="295"><P>wells</P></TD><TD width="181"><P>A1-3</P></TD><TD width="176"><P>B1-3</P></TD></TR><TR><TD width="295"><P>APC annexin V</P></TD><TD width="181"><P>-</P></TD><TD width="176"><P>+</P></TD></TR><TR><TD width="295"><P>Sytox green</P></TD><TD width="181"><P>-</P></TD><TD width="176"><P>-</P></TD></TR></TBODY></TABLE><P><U>Plate C: </U>  </P><TABLE width="0"><TBODY><TR><TD width="295"><P>sample number</P></TD><TD width="181"><P>5</P></TD><TD width="176"><P>6</P></TD></TR><TR><TD width="295"><P>sample name</P></TD><TD width="181"><P>necrosis sytox control (for FACS calibration)</P></TD><TD width="176"><P>necrosis</P></TD></TR><TR><TD width="295"><P>Transfection plasmids</P></TD><TD width="181"><P>-</P></TD><TD width="176"><P>-</P></TD></TR><TR><TD width="295"><P>wells</P></TD><TD width="181"><P>A1-3</P></TD><TD width="176"><P>B1-3</P></TD></TR><TR><TD width="295"><P>APC annexin V</P></TD><TD width="181"><P>-</P></TD><TD width="176"><P>+</P></TD></TR><TR><TD width="295"><P>Sytox green</P></TD><TD width="181"><P>+</P></TD><TD width="176"><P>+</P></TD></TR></TBODY></TABLE><P><U>Plate D</U>:       </P><TABLE width="0"><TBODY><TR><TD width="295"><P>sample number</P></TD><TD width="181"><P>7</P></TD></TR><TR><TD width="295"><P>sample name</P></TD><TD width="181"><P>without apoptosis induction</P></TD></TR><TR><TD width="295"><P>Transfection plasmids</P></TD><TD width="181"><P>-</P></TD></TR><TR><TD width="295"><P>wells</P></TD><TD width="181"><P>A1-3</P></TD></TR><TR><TD width="295"><P>APC annexin V</P></TD><TD width="181"><P>+</P></TD></TR><TR><TD width="295"><P>Sytox green</P></TD><TD width="181"><P>+</P></TD></TR></TBODY></TABLE><P> </P></OL><P><U>References:</U></P><OL><LI>Farzadfard, F., Perli, S. D., &amp; Lu, T. K. (2013). Tunable and multifunctional eukaryotic transcription factors based on CRISPR/Cas. <EM>ACS synthetic biology</EM>, <EM>2</EM>(10), 604-613.‏</LI><LI>La Russa, M. F., &amp; Qi, L. S. (2015). The new state of the art: CRISPR for gene activation and repression. <EM>Molecular and cellular biology</EM>, MCB-00512.‏</LI><LI>Srinivasula, S. M., Ahmad, M., MacFarlane, M., Luo, Z., Huang, Z., Fernandes-Alnemri, T., &amp; Alnemri, E. S. (1998). Generation of constitutively active recombinant caspases-3 and-6 by rearrangement of their subunits. <EM>Journal of Biological Chemistry</EM>, <EM>273</EM>(17), 10107-10111.‏</LI><LI>Kim, J. H., Lee, S. R., Li, L. H., Park, H. J., Park, J. H., Lee, K. Y., ... &amp; Choi, S. Y. (2011). High cleavage efficiency of a 2A peptide derived from porcine teschovirus-1 in human cell lines, zebrafish and mice. <EM>PloS one</EM>, <EM>6</EM>(4), e18556.‏</LI></OL><P> </P><A href="https://2018.igem.org/Team:BGU_Israel/Results#Caspase3_HEK" class="btn btn-info" role="button">See Experiment Result</A></DIV></DIV></DIV></DIV><DIV id="Caspase3_Astro" class="bg-grey"><DIV class="container text-center"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="row"><H1>Apoptosis Induction</H1><P><U>Title</U>: Validation of pMLPm activation with dCas9-VP64-gRNA and induction of apoptotic signal in C8-D30 astrocyte cells.<U>Conducted by</U>: Liat Tsoran, Mor Pasi and Ori Tulchinsky</P><P><U>Date</U>: 30/9/18-14/10/18</P><P><U>Aim: </U>In this experiment we wanted to validate that our synthetic engineered system could induce apoptotic death by the activation of the synthetic minimal adenovirus major late promoter (pMLPm) via dCAs9-VP64-gRNA complex that is expressed by A1 reactive astrocyte specific promoters.</P><P><U>Importance</U>: In the astrocyte-cells pathway we aimed to eliminate only A1 reactive astrocytes by triggering apoptotic death, while maintaining the resting cells in the central nervous system (CNS) intact.</P><P>In this experiment we wanted to validate our enginnered system, in which, dCas9-VP64-gRNA complex that is expressed by A1 reactive astrocyte specific markers, can activate the pMLPm promoter to regulate the expression of exogenous reverse caspase3 that will trigger an apoptotic signal.</P><P><U>Theoretical background</U>:</P><P>pSTEAP4 (Six Transmembrane Epithelial Antigen of Prostate 4 promoter)  and pTIMP1 (Tissue inhibitor of metalloproteinases-1 promoter) are regulating the exoression of genes that are highly expressed in A1 reactive astrocyte cells when compared to other cells in the CNS<SUP>1-2</SUP>.</P><P>In our therapeutic approach, we wanted to use these markers in order to express dCas9-VP64-gRNA complex that will activate a synthetic promoter to induce an apoptotic signal.</P><P>The synthetic minimal adenovirus major late promoter (pMLPm) contains three repetitions of the &quot;a1&quot; sequence, which are complementary to the gRNA sequence, enabling dCas9-VP64-gRNA complex to target the pMLPm synthetic promoter<SUP>1</SUP>.When the transcription factor VP64 (an engineered tetramer of the herpes simplex VP16 transcriptional activator domain) is fused to dCas9 enzyme, it can be guided to a specific location in the genome, in this manner we can exploit it to promote downstream translation<SUB>2.</SUB></P><P>The dCas9-VP64-gRNA complex will target the pMLPm synthetic promoter and promote the expression of exogenous reverse caspase3, that in contrast to the endogenous caspas3 will be activated after transcription due to its autocatalytic processing, meaning that this enzyme will trigger an apoptotic signal that will lead to apoptotic death  (caspase3 is responsible for chromatin condensation and DNA fragmentation)<SUP>3</SUP>.</P><P>The mCherry reporter protein is also expressed by the pMLPm promoter in our system, it is fused to the exogenous reverse caspase3 with Thoseaasigna virus 2A (T2A) peptide that is cleaved during translation<SUP>4</SUP>.</P><P>In order to activate pMLPm promoter and to induce apoptotic death we used 2 plasmids:</P><OL><LI>A Lenti viral plasmid: pTIMP1 dCas9 VP64 – in this plasmid TIMP1 promoter expresses dCas9 enzyme which is fused to the transcription factor VP64.</LI><LI>pSynt – in this plasmid pSTEAP4 promoter expresses gRNA and pMLPm promoter regulates the downstream translation of reverse caspase3 which is fused to mCherry fluorophore.</LI></OL><P>A1 reactive astrocyte cells that were co-transfected with both plasmids will drove apoptosis simultaneously with expression of mCherry. In order to test the specificity of pMLPm promoter activation, we used a control of cells that were transfected only with the pSynt plasmid.</P><P><U>Procedure</U>: </P><P><U>Day 1</U>-  Splitting of C8-D30-cells to 6 wells plates</P><P><U>Day 2</U>- transfection with TurboFect™ Transfection Reagent</P><P><U>Day 3</U>- We verified mCherry expression with fluorescent microscope (24 hours after transfection).</P><P><U>Day 4</U>:</P><OL><LI><STRONG>validation of pMLPm activation with dCas9-VP64-gRNA-</STRONG></LI><UL><LI>We used ARIA FACS in order to document mCherry expression (48 hours after transfection).</LI><LI>We used confocal microscope in order to document mCherry expression (48 hours after transfection).</LI></UL></OL><OL start="2"><LI><STRONG>induction of apoptotic signal -</STRONG> We used APC Annexin V/Dead Cell Apoptosis Kit with APC annexin V and SYTOX® Green for Flow Cytometry</LI></OL><P><U>Design</U>:</P><OL><LI><STRONG> validation of pMLPm activation with dCas9-VP64-gRNA-</STRONG></LI><P><U>Plate A</U>: FACS</P><P>     </P><TABLE width="0"><TBODY><TR><TD width="242"><P>sample number</P></TD><TD width="149"><P>1</P></TD><TD width="144"><P>2</P></TD><TD width="144"><P>3</P></TD></TR><TR><TD width="242"><P>Transfection plasmids</P></TD><TD width="149"><P>Lenti: pTIMP1 dCas9 VP64 + pSynt CMV</P></TD><TD width="144"><P>pSynt</P></TD><TD width="144"><P>Empty plasmid</P></TD></TR><TR><TD width="242"><P>wells</P></TD><TD width="149"><P>A1+B1</P></TD><TD width="144"><P>A2+B2</P></TD><TD width="144"><P>A3+B3</P></TD></TR></TBODY></TABLE><P> </P><P><U>Plate B: </U>confocal microscope  </P><TABLE width="0"><TBODY><TR><TD width="242"><P>sample number</P></TD><TD width="149"><P>1</P></TD><TD width="144"><P>2</P></TD><TD width="144"><P>3</P></TD></TR><TR><TD width="242"><P>Transfection plasmids</P></TD><TD width="149"><P>Lenti: pTIMP1 dCas9 VP64 + pSynt CMV</P></TD><TD width="144"><P>pSynt</P></TD><TD width="144"><P>Empty plasmid</P></TD></TR><TR><TD width="242"><P>wells</P></TD><TD width="149"><P>A1</P></TD><TD width="144"><P>A2</P></TD><TD width="144"><P>A3</P></TD></TR></TBODY></TABLE><P><STRONG> </STRONG></P><OL start="2"><LI><STRONG> induction of an apoptotic signal-</STRONG></LI></OL><P><U>Plate A: </U>      </P><TABLE width="0"><TBODY><TR><TD width="295"><P>sample number</P></TD><TD width="181"><P>1</P></TD><TD width="176"><P>2</P></TD></TR><TR><TD width="295"><P>sample name</P></TD><TD width="181"><P>pTIMP1 dCas+ pSynt</P></TD><TD width="176"><P>pSynt</P></TD></TR><TR><TD width="295"><P>Transfection plasmids</P></TD><TD width="181"><P>Lenti: pTIMP1 dCas9 VP64 + pSynt</P></TD><TD width="176"><P>pSynt</P></TD></TR><TR><TD width="295"><P>wells</P></TD><TD width="181"><P>A1-3</P></TD><TD width="176"><P>B1-3</P></TD></TR><TR><TD width="295"><P>APC annexin V</P></TD><TD width="181"><P>+</P></TD><TD width="176"><P>+</P></TD></TR><TR><TD width="295"><P>Sytox green</P></TD><TD width="181"><P>+</P></TD><TD width="176"><P>+</P></TD></TR></TBODY></TABLE><P><U>Plate B</U>:       </P><TABLE width="0"><TBODY><TR><TD width="295"><P>sample number</P></TD><TD width="181"><P>3</P></TD><TD width="176"><P>4</P></TD></TR><TR><TD width="295"><P>sample name</P></TD><TD width="181"><P>mCherry control (for FACS calibration)</P></TD><TD width="176"><P>annexin control (for FACS calibration)</P></TD></TR><TR><TD width="295"><P>Transfection plasmids</P></TD><TD width="181"><P>Lenti: pTIMP1 dCas9 VP64 + pSynt</P></TD><TD width="176"><P>Lenti: pTIMP1 dCas9 VP64 + pSynt</P></TD></TR><TR><TD width="295"><P>wells</P></TD><TD width="181"><P>A1-3</P></TD><TD width="176"><P>B1-3</P></TD></TR><TR><TD width="295"><P>APC annexin V</P></TD><TD width="181"><P>-</P></TD><TD width="176"><P>+</P></TD></TR><TR><TD width="295"><P>Sytox green</P></TD><TD width="181"><P>-</P></TD><TD width="176"><P>-</P></TD></TR></TBODY></TABLE><P><U>Plate C</U>:       </P><TABLE width="0"><TBODY><TR><TD width="295"><P>sample number</P></TD><TD width="181"><P>5</P></TD><TD width="176"><P>6</P></TD></TR><TR><TD width="295"><P>sample name</P></TD><TD width="181"><P>necrosis sytox control (for FACS calibration)</P></TD><TD width="176"><P>necrosis</P></TD></TR><TR><TD width="295"><P>Transfection plasmids</P></TD><TD width="181"><P>-</P></TD><TD width="176"><P>-</P></TD></TR><TR><TD width="295"><P>wells</P></TD><TD width="181"><P>A1-3</P></TD><TD width="176"><P>B1-3</P></TD></TR><TR><TD width="295"><P>APC annexin V</P></TD><TD width="181"><P>-</P></TD><TD width="176"><P>+</P></TD></TR><TR><TD width="295"><P>Sytox green</P></TD><TD width="181"><P>+</P></TD><TD width="176"><P>+</P></TD></TR></TBODY></TABLE><P><U>Plate D</U>:</P><TABLE width="0"><TBODY><TR><TD width="295"><P>sample number</P></TD><TD width="181"><P>7</P></TD></TR><TR><TD width="295"><P>sample name</P></TD><TD width="181"><P>without apoptosis induction</P></TD></TR><TR><TD width="295"><P>Transfection plasmids</P></TD><TD width="181"><P>-</P></TD></TR><TR><TD width="295"><P>wells</P></TD><TD width="181"><P>A1-3</P></TD></TR><TR><TD width="295"><P>APC annexin V</P></TD><TD width="181"><P>+</P></TD></TR><TR><TD width="295"><P>Sytox green</P></TD><TD width="181"><P>+</P></TD></TR></TBODY></TABLE></OL><P><U>References:</U></P><OL><LI>Liddelow SA, Guttenplan KA, Clarke LE, et al. Neurotoxic reactive astrocytes are induced by activated microglia. <EM>Nature</EM>. 2017;541(7638):481-487. doi:10.1038/nature21029.</LI><LI>Zamanian J, Xu L, Foo L, et al. Genomic Analysis of Reactive Astrogliosis. <EM>The Journal of neuroscience : the official journal of the Society for Neuroscience</EM>. 2012;32(18):6391-6410. doi:10.1523/JNEUROSCI.6221-11.2012.</LI><LI>Farzadfard, F., Perli, S. D., &amp; Lu, T. K. (2013). Tunable and multifunctional eukaryotic transcription factors based on CRISPR/Cas. <EM>ACS synthetic biology</EM>, <EM>2</EM>(10), 604-613.‏</LI><LI>La Russa, M. F., &amp; Qi, L. S. (2015). The new state of the art: CRISPR for gene activation and repression. <EM>Molecular and cellular biology</EM>, MCB-00512.‏</LI><LI>Srinivasula, S. M., Ahmad, M., MacFarlane, M., Luo, Z., Huang, Z., Fernandes-Alnemri, T., &amp; Alnemri, E. S. (1998). Generation of constitutively active recombinant caspases-3 and-6 by rearrangement of their subunits. <EM>Journal of Biological Chemistry</EM>, <EM>273</EM>(17), 10107-10111.‏</LI><LI>Kim, J. H., Lee, S. R., Li, L. H., Park, H. J., Park, J. H., Lee, K. Y., ... &amp; Choi, S. Y. (2011). High cleavage efficiency of a 2A peptide derived from porcine teschovirus-1 in human cell lines, zebrafish and mice. <EM>PloS one</EM>, <EM>6</EM>(4), e18556.‏</LI></OL><A href="https://2018.igem.org/Team:BGU_Israel/Results#Caspase3_Astro" class="btn btn-info" role="button">See Experiment Result</A></DIV></DIV></DIV></DIV><DIV id="myModal" class="modal"><SPAN class="close" onclick="modal.style.display = 'none'">×</SPAN></DIV><TITLE>OriginALS</TITLE><FOOTER style="margin-bottom:-10px!important;margin-top:-10px!important;"><DIV class="container my-auto"><DIV class="row"><DIV class="col-md-4"><H3 style="color:white!important;">About Us</H3><P style="color:white!important;text-align: justify!important;">The BGU-iGEM team “OriginALS” hopes to develop an innovative therapeutic approach to prolong the life expectancy of ALS patients, using Synthetic Biology. We are dedicated to promoting ALS awareness and research in Israel through public engagement and educational activities.</P></DIV><DIV class="col-md-4"><UL class="footer-list"><H3>Keep in Touch</H3><LI class="footer-list-item"> @iGEMBGU</LI><LI class="footer-list-item">@igem_2018_bgu</LI><LI class="footer-list-item"> @originalsbgu</LI><LI class="footer-list-item"> originalsbgu@gmail.com</LI></UL></DIV><DIV class="col-md-4"><UL class="footer-list"><H3>Address</H3><LI class="footer-list-item">Ben-Gurion University of the Negev
Ben Gurion 1, Beer Sheva 8410501, Israel</LI></UL></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>