<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Sorbonne_U_Paris_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Sorbonne U Paris/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="collapse navbar-collapse " id="navbarTogglerDemo02"><UL class=" navbar-nav "><LI class="nav-item dropdown" id="mobile-homepage"><A class="nav-link" href="https://2018.igem.org/Team:Sorbonne_U_Paris" style="color:white;"> Home page</A></LI><LI class="nav-item dropdown  "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white">  Project
        </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Description">Description</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Design">Design</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Model">Model</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Demonstrate">Demonstrate</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Improve">Improve</A></DIV></LI><LI class="nav-item dropdown  "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Biology</A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Experiments">Experiments</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Results">Results</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Parts">Parts</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/InterLab">Interlab</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Safety">Safety</A></DIV></LI><LI class="nav-item dropdown  "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Human practices
        </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Human_Practices">Human Practices</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Integrated_Human_Practices">Integrated HP</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Public_Engagement">Education &amp; Engagement</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Meet_up">Meet up</A></DIV></LI><LI class="nav-item dropdown "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Notebook
        </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Notebook">Notebook</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Protocols">Protocols</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Brainstorming">Brainstorming</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Team </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Team">Members</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Collaborations">Collaborations</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Attributions">Attributions</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Sponsors">Sponsors</A></DIV></LI><LI class="nav-item dropdown  "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Awards</A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://igem.org/2018_Judging_Form?team=Sorbonne_U_Paris">Judging form</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Medals">Medals</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Giant_Jamboree">Giant Jamboree</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Sitemap" style="color:white;"> Site map</A></LI></UL></DIV><DIV class="global"><DIV class="container"><ARTICLE class="col-md-10"><H1>Experiments</H1></ARTICLE></DIV><DIV class="container"><DIV id="index1" class="side-nav index fixed-top"><UL class="list-group list-unstyled"><LI style="font-size: 25px; font-weight:bold;text-align:center">Index</LI><LI class="list-group-item"><A class="pointer" href="#1">I-Synthetic retrotransposon</A></LI><LI class="list-group-item"><A class="pointer" href="#2">II-Retrotransposon functionality(MoClo) </A></LI><LI class="list-group-item"><A class="pointer" href="#3">III- Trehalose pathway</A></LI><LI class="list-group-item"><A class="pointer" href="#4">IV-Protocols</A></LI><ARTICLE class="col-md-10"><H2 class="title-h2" id="1">Construction of the synthetic retrotransposon </H2><H4>Cloning of the parts from the Chlamydomonas MoClo Kit </H4><P>Some of the parts (intron, paromomycin, PpSAD, 5’UTRpSAD, PpNIT promoter, 5’UTRpNIT, 3’UTR) we used to construct our synthetic retrotransposon were taken from the MoClo Kit. The way we designed our system is different from a classical transcription unit (TU) assembly. Almost every part we use is in a “different position”. That means the Level 0 (L0) parts stored in the kit were PCR-amplified (and sequenced) with primers that allow the modification of the Fusion sites. Then the PCR products were cloned by Golden Gate cloning in new L0 backbones resistant to spectinomycine: pICH41258, pICH41264 or pAGM9121. In some cases we could use acceptor plasmids with fusion sites that already match our requirements (pICHX).  Most of the time we had to use a universal plasmid from the original MoClo kit (pAGM9121). This acceptor plasmid works as the Phytobrick universal acceptor plasmid (BBa_P10500).  </P><H4>Cloning of the parts with genes synthesis </H4><P>We could not get genomic DNA from <I>Volvox carterii</I> to amplify the specific sequences of the <I>Osser</I> retrotransposon. <I>Osser</I> is quite big: 4.8 kb and contains more than seven illegal restrictions sites for the GoldenGate assembly. We chose to synthetize all the retrotransposons parts by using the IDT 20 kb program. We used the same PCR cloning strategy to add the restrictions sites and fusion sites in 5’ and 3’ of the synthetized sequences. </P><P>5’LTR and 3’LTR were quite easy as they are quite short: 197-bp and no restriction sites were in those two sequences. Even if the LTRs are the same, we made two synthesises because we truncated 5’LTR and we did not want to make a PCR from 3’LTR to avoid resequencing
 </P><P>The Gag/Pol design for synthesis was quite challenging! First we had to remove all the illegal restriction sites (<I>BpiI</I> and <I>BsaI</I>). Then, this part is huge: 4.3 kb. In fact, it was planned to order it in three g-block fragments and assemble them in pL0 through scarless GoldenGate cloning using the same type IIS restriction enzyme (<I>BpiI</I>). However, IDT’s algorithms did not validate this strategy because of various repeat sequence and high GC clusters</P><P>Finally, we managed to find a design with 6 different g-block with sizes between 90-bp and 1.3 kb. All of it was supposed to be assembled in a one-pot reaction in the universal acceptor plasmid.  </P><P><B>Unfortunately, this was the biggest mistake we did this year.</B> The complexity and the number of fragments were too high to get this cloning done. Because of that we could not construct the whole synthetic retrotransposon. We should have ordered them in a plasmid with quality control even if it would have been very expensive.  </P><P>We are working now on cloning every fragment in its own L-1 plasmid and then make a final L0 assembly using donor plasmids instead of linear fragments. To do so, we need to order primers for the 6 fragments to add the L-1 restriction site, and sequence everything. However, we will not have enough time to do this before the Giant Jamboree.  </P><H2 class="title-h2" id="2">Characterisation of the retrotransposons' functionality</H2><H4>Paromomycine assay </H4><P>Because microalgae do not bear plasmids, the Chlamydomonas MoClo kit is designed to integrate TUs inside the genome. To do so, once the construction is done, a cassette containing the TUs and the resistance gene for hygromycin is linearized by digestion with a restriction enzyme.  The linear DNA is transformed in <I>C. reinhardtii D66</I> by electroporation and plated on TAP agar with hygromycine to select mutants after 5 days of incubation at 25 °C.  </P><P>To assay the functionality of our synthetic retrotransposon, it was planned to induce our system for 6 days by changing the nitrogen source: ammonium 7.5 mM (NH<SUB>4</SUB>)Cl for non-inducing condition or nitrate 4 mM KNO<SUB>3</SUB> for inducing the PNIT promoter as describe in Crozet et al 2018.  Then the algae will be plated on TAP agar with hygromycin and paromoycine. By counting the number of resistant colonies we can deduce if our system is functional.  </P><P>Further information about the design of our part are available on Design page </P><DIV class="row text-center text-xs-center text-sm-enter text-md-center  text-lg-center  "><DIV class="col-xs-4 col-sm-4 col-md-12 col-lg-12"><A class="btn-sm" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Design" role="button" style=" color:white; border-radius: 10px; background-color: #22D66C; text-decoration:none; "> Design </A></DIV></DIV><H4>Screening for the biggest in vivo library.  </H4><P>Our model allows to calculate the library size with two simple parameters. Therefore, need to measure the number of cells while inducing the system. At the end, we need to count the number of transposons per clones.  </P><P>Further information about the modeling of our project are available on Modeling page. </P><DIV class="row text-center text-xs-center text-sm-enter text-md-center  text-lg-center  "><DIV class="col-xs-4 col-sm-4 col-md-12 col-lg-12"><A class="btn-sm" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Model" role="button" style=" color:white; border-radius: 10px; background-color: #22D66C; text-decoration:none; "> Model </A></DIV></DIV><P>The OD<SUB>600</SUB> can be easily measured in a 96-well plate reader using microplate. For example, every validated construction correctly inserted can be put in culture in a well until OD<SUB>600</SUB> =1. After centrifugation of the microplate, when suspend the cells with the inducing media. OD<SUB>600</SUB> will me measure during the whole experiment.  </P><P>At the end of the induction, genomic DNA of all the cells inside the wells can be extracted by automation with the magnetic deck provides by Opentron. This will allow to do a qPCR normalized with the OD<SUB>600</SUB></P><H2 class="title-h2" id="3">Construction of the trehalose pathway </H2><P>The two CDS OTSA and OTSB genes used to produce trehalose were codon-optimized for <I>C. reinhardtti</I> which has a high GC codon usage bias.  Moreover, we removed all the MoClo illegal restriction sites. We ordered those two sequence from DS BIOSCIENCE lDT because IDT’s algorithm could not synthetize the two 1 kb genes even if it was split in smaller fragment. OTSA and OTSB were split in three fragments with a scarless GolgenGate strategy and ordered as plasmid with quality controls.  Then following the MoClo kit procedure they were cloned in L0 acceptor plasmids.  </P><H2 class="title-h2" id="4">Protocols </H2><P>You can find all the protocols we used during our experiments on the Protocols page</P><DIV class="row text-center text-xs-center text-sm-enter text-md-center  text-lg-center  "><DIV class="col-xs-4 col-sm-4 col-md-12 col-lg-12"><A class="btn-sm" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Protocols" role="button" style=" color:white; border-radius: 10px; background-color: #22D66C; text-decoration:none;"> Protocols </A></DIV></DIV></ARTICLE><DIV class="container fixed-bottom backtotop" style="padding-bottom: 10px; padding-right:0px; position: fixed; left: 120px;"><DIV class="row text-right text-xs-right text-sm-right text-md-right  text-lg-right  "><DIV class="col-xs-4 col-sm-4 col-md-12 col-lg-12"><A class="y btn-sm" href="#backtotop" role="button" style=" color:white; border-radius: 10px; background-color:#22D66C; text-decoration:none; ">  Back to top</A></DIV></DIV></DIV><SECTION id="footer"><DIV class="container"><DIV class="row text-center text-xs-center text-sm-center text-md-right"><DIV class="col-xs-12 col-sm-3 col-md-3 col-lg-6"><H5 class="text-left" style="border: none;padding: 0px">iGEM Sorbonne U Paris</H5><P class="text-left" style="color:white; font-size:16px;"> 4 place Jussieu  75005 Paris France</P></DIV></DIV></DIV><DIV class="container"><P class="text-center" style="color: white; padding:20px 0 20px 0">Made in Paris with love © iGEM Sorbonne U Paris 2018</P></DIV></SECTION></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>