Team:Sorbonne U Paris/Experiments
Home page
Project
Description Design Model Demonstrate Improve
Biology
Experiments Results Parts Interlab Safety
Human practices
Human Practices Integrated HP Education & Engagement Meet up
Notebook
Notebook Protocols Brainstorming
Team
Members Collaborations Attributions Sponsors
Awards
Judging form Medals Giant Jamboree
Site map
Experiments
Index
I-Synthetic retrotransposon
II-Retrotransposon functionality(MoClo)
III- Trehalose pathway
IV-Protocols
Construction of the synthetic retrotransposon
Cloning of the parts from the Chlamydomonas MoClo Kit
Some of the parts (intron, paromomycin, PpSAD, 5’UTRpSAD, PpNIT promoter, 5’UTRpNIT, 3’UTR) we used to construct our synthetic retrotransposon were taken from the MoClo Kit. The way we designed our system is different from a classical transcription unit (TU) assembly. Almost every part we use is in a “different position”. That means the Level 0 (L0) parts stored in the kit were PCR-amplified (and sequenced) with primers that allow the modification of the Fusion sites. Then the PCR products were cloned by Golden Gate cloning in new L0 backbones resistant to spectinomycine: pICH41258, pICH41264 or pAGM9121. In some cases we could use acceptor plasmids with fusion sites that already match our requirements (pICHX).  Most of the time we had to use a universal plasmid from the original MoClo kit (pAGM9121). This acceptor plasmid works as the Phytobrick universal acceptor plasmid (BBa_P10500).
Cloning of the parts with genes synthesis
We could not get genomic DNA from Volvox carterii to amplify the specific sequences of the Osser retrotransposon. Osser is quite big: 4.8 kb and contains more than seven illegal restrictions sites for the GoldenGate assembly. We chose to synthetize all the retrotransposons parts by using the IDT 20 kb program. We used the same PCR cloning strategy to add the restrictions sites and fusion sites in 5’ and 3’ of the synthetized sequences.
5’LTR and 3’LTR were quite easy as they are quite short: 197-bp and no restriction sites were in those two sequences. Even if the LTRs are the same, we made two synthesises because we truncated 5’LTR and we did not want to make a PCR from 3’LTR to avoid resequencing
The Gag/Pol design for synthesis was quite challenging! First we had to remove all the illegal restriction sites (BpiI and BsaI). Then, this part is huge: 4.3 kb. In fact, it was planned to order it in three g-block fragments and assemble them in pL0 through scarless GoldenGate cloning using the same type IIS restriction enzyme (BpiI). However, IDT’s algorithms did not validate this strategy because of various repeat sequence and high GC clusters
Finally, we managed to find a design with 6 different g-block with sizes between 90-bp and 1.3 kb. All of it was supposed to be assembled in a one-pot reaction in the universal acceptor plasmid.
Unfortunately, this was the biggest mistake we did this year. The complexity and the number of fragments were too high to get this cloning done. Because of that we could not construct the whole synthetic retrotransposon. We should have ordered them in a plasmid with quality control even if it would have been very expensive.
We are working now on cloning every fragment in its own L-1 plasmid and then make a final L0 assembly using donor plasmids instead of linear fragments. To do so, we need to order primers for the 6 fragments to add the L-1 restriction site, and sequence everything. However, we will not have enough time to do this before the Giant Jamboree.
Characterisation of the retrotransposons' functionality
Paromomycine assay
Because microalgae do not bear plasmids, the Chlamydomonas MoClo kit is designed to integrate TUs inside the genome. To do so, once the construction is done, a cassette containing the TUs and the resistance gene for hygromycin is linearized by digestion with a restriction enzyme.  The linear DNA is transformed in C. reinhardtii D66 by electroporation and plated on TAP agar with hygromycine to select mutants after 5 days of incubation at 25 °C.
To assay the functionality of our synthetic retrotransposon, it was planned to induce our system for 6 days by changing the nitrogen source: ammonium 7.5 mM (NH4)Cl for non-inducing condition or nitrate 4 mM KNO3 for inducing the PNIT promoter as describe in Crozet et al 2018.  Then the algae will be plated on TAP agar with hygromycin and paromoycine. By counting the number of resistant colonies we can deduce if our system is functional.
Further information about the design of our part are available on Design page
Design
Screening for the biggest in vivo library.
Our model allows to calculate the library size with two simple parameters. Therefore, need to measure the number of cells while inducing the system. At the end, we need to count the number of transposons per clones.
Further information about the modeling of our project are available on Modeling page.
Model
The OD600 can be easily measured in a 96-well plate reader using microplate. For example, every validated construction correctly inserted can be put in culture in a well until OD600 =1. After centrifugation of the microplate, when suspend the cells with the inducing media. OD600 will me measure during the whole experiment.
At the end of the induction, genomic DNA of all the cells inside the wells can be extracted by automation with the magnetic deck provides by Opentron. This will allow to do a qPCR normalized with the OD600
Construction of the trehalose pathway
The two CDS OTSA and OTSB genes used to produce trehalose were codon-optimized for C. reinhardtti which has a high GC codon usage bias.  Moreover, we removed all the MoClo illegal restriction sites. We ordered those two sequence from DS BIOSCIENCE lDT because IDT’s algorithm could not synthetize the two 1 kb genes even if it was split in smaller fragment. OTSA and OTSB were split in three fragments with a scarless GolgenGate strategy and ordered as plasmid with quality controls.  Then following the MoClo kit procedure they were cloned in L0 acceptor plasmids.
Protocols
You can find all the protocols we used during our experiments on the Protocols page
Protocols
Back to top
iGEM Sorbonne U Paris
4 place Jussieu
75005 Paris France
Made in Paris with love © iGEM Sorbonne U Paris 2018
