<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Sorbonne_U_Paris_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Sorbonne U Paris/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="collapse navbar-collapse " id="navbarTogglerDemo02"><UL class=" navbar-nav "><LI class="nav-item dropdown" id="mobile-homepage"><A class="nav-link" href="https://2018.igem.org/Team:Sorbonne_U_Paris" style="color:white;"> Home page</A></LI><LI class="nav-item dropdown  "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white">  Project
        </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Description">Description</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Design">Design</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Model">Model</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Demonstrate">Demonstrate</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Improve">Improve</A></DIV></LI><LI class="nav-item dropdown  "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Biology</A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Experiments">Experiments</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Results">Results</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Parts">Parts</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/InterLab">Interlab</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Safety">Safety</A></DIV></LI><LI class="nav-item dropdown  "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Human practices
        </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Human_Practices">Human Practices</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Integrated_Human_Practices">Integrated HP</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Public_Engagement">Education &amp; Engagement</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Meet_up">Meet up</A></DIV></LI><LI class="nav-item dropdown "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Notebook
        </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Notebook">Notebook</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Protocols">Protocols</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Brainstorming">Brainstorming</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Team </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Team">Members</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Collaborations">Collaborations</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Attributions">Attributions</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Sponsors">Sponsors</A></DIV></LI><LI class="nav-item dropdown  "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Awards</A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://igem.org/2018_Judging_Form?team=Sorbonne_U_Paris">Judging form</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Medals">Medals</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Giant_Jamboree">Giant Jamboree</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Sitemap" style="color:white;"> Site map</A></LI></UL></DIV><DIV class="global"><DIV class="container"><ARTICLE class="col-md-10"><H1>Project description</H1></ARTICLE></DIV><DIV class="container"><ARTICLE class="col-md-10"><P>Synthetic biology aims at understanding by building. To do so, various tools and methodologies were developed and allow design of metabolic pathways on demand. In particular, microorganisms can be used as cell factories which paves the way to a world based on a renewable chemistry. The main source of energy of these cells is sugar. However, industrial use of these heterotrophic cells requires massive production of sugar with canes and beets which has dramatic ecological and environmental impacts. To bring a solution to this problem, we want to engineer <I>Chlamydomonas reinhardtti</I>, a green microalgae adapted to live in marine water, to produce glucose in large amounts. This way, sugar production should not compete for arable lands which is increasingly demanded for feeding the world. </P><DIV class="container d-flex justify-content-center" style="padding:20px"><FIGURE style=" width: 80%;"><FIGCAPTION style="padding: 10px;"><B>Through our SugaRevolution we want to produce sustainable sugar without harming our
soil and competing with food crop!</B>We want to engineer a strain of the green microalgae <I>Chlamydomonas reinhardtii</I>,adapted to live in sea water, and able to produce sugar in large amounts. Microalgae‘s photosynthesis uses sunlight and atmospheric CO2 to accumulate sugar in its stored form: starch. By using a photobioreactor for the containment of the algae in the oceans, sugar production would not compete with arable lands. </FIGCAPTION></FIGURE></DIV><P>Using microalgae as a photosynthetic cellular chassis is a promising alternative to heterotrophic microorganisms such as bacterium or yeast (the current favorite chassis). Unfortunately, industrial use of microalgae to produce low value compounds such as oil or sugar is barely profitable as demonstrate by the OMEGA project (Johnathan Trent, NASA). Yields must be increase by a 10 factor. </P><P>In order to spread the use of <I>C.reinhardtii</I> as an industrial green cell factory, genetic engineering may be an answer.  To do so, more genetic tools to engineer it are still required. To do so, we will enrich the recently developed Modular Cloning (MoClo) toolkit for C. reinhardtti (Pierre Crozet, Stephane Lemaire and their collaborators, publication submitted) with a synthetic retrotransposon to generate in vivo continuous directed evolution. It will be the first time that such genetic tool is applied to non-baring plasmids cells such as plants or microalgae. This approach allows to either generate new proteins with tailor-made functional properties or to optimize biological systems. </P><P>To adapt this system, a transposable element which relies on a reverse-transcription event will be used. During the transposition, the target sequence (CARGO) inserted within the transposable element undergoes an important mutagenesis linked to the error rate of the reverse transcriptase. This strategy allows us to quickly generate a large library of sequences in vivo speeding up screening for phenotype. Indeed, in classical directed evolution, the library is generated in vitro and then transformed into the organism. As transforming alga could be a limiting factor in the design/build/test cycles, the proposed methodology should greatly increase the speed of this cycles and thus make <I>C. reinhardtii</I> a better chassis. </P><DIV class="container d-flex justify-content-center" style="padding:20px"><FIGURE style=" width: 80%;"><FIGCAPTION style="padding: 10px;"><B>In-Vivo Directed Evolution mediated by a synthetic LTR retrotransposon.</B>We want to engineer a strain of the green microalgae LTR retrotransposons are transposable elements that duplicate within their host genome by inserting new copies of themselves after a transcription and a retro-transcription event. They
encode a minimum of two genes, gag and pol. The latter produces the Reverse Transcriptase (RT) which has a high error rate leading to random mutations in the resulting copy of the transcript. This newly synthetized cDNA will be re-inserted, and only the cargo will have a promoter guaranteeing that the transposon will not be expressed again.</FIGCAPTION></FIGURE></DIV><P>Further information about the design of our part are available on Design page </P><DIV class="row text-center text-xs-center text-sm-enter text-md-center  text-lg-center  "><DIV class="col-xs-4 col-sm-4 col-md-12 col-lg-12"><A class="btn-sm" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Design" role="button" style=" color:white; border-radius: 10px; background-color: #22D66C; text-decoration:none; "> Design </A></DIV></DIV><P>Synthetic biology works better when rational design is improved by empiric optimization. To generate the biggest <I>in vivo</I> library for a given sequence we need to screen the activity of different retrotransposon constructions with the goal to select the best one. Unfortunately, experimentally assessing he size of the library can be tedious and expensive as it requires isolating and sequencing hundreds of colonies to count how many time one specific mutant is on the library. By modelling the size of the library from two “easy to get” parameters (directly from a 96-wells microplate) we can calculate the libraries size for every variation of the synthetic retrotransposons. </P><DIV class="container d-flex justify-content-center" style="padding:20px"><FIGURE style=" width: 70%;"><FIGCAPTION style="padding: 10px;"><B>Global scheme of our future strategy</B></FIGCAPTION></FIGURE></DIV><P>Our iGEM project lays the foundation stone of a high-throughput fully automated and integrated system to construct synthetic retrotransposons and refine them with automated screen base on the modelling of mutants’ library sizes from two simple experiments. We will use the open source liquid-handling robot Opentron OT-2 to develop a ready-to-use solution to make directed evolution in <I>C. reinhardtii</I>. </P><P>Further information about the modeling of our project are available on Modeling page. </P><DIV class="row text-center text-xs-center text-sm-enter text-md-center  text-lg-center  "><DIV class="col-xs-4 col-sm-4 col-md-12 col-lg-12"><A class="btn-sm" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Model" role="button" style=" color:white; border-radius: 10px; background-color: #22D66C; text-decoration:none; "> Model </A></DIV></DIV><P>The proof-of-concept of this designed retrotransposon will be made on a gene conferring resistance to paromomycine, and then applied to the trehalose production (disaccharide of glucose). We identified in the literature a fluorescent biosensor for trehalose that will allow us to efficiently screen best producing clones </P><DIV class="container" id="references"><H5 class="text-center">References </H5><UL class="list-unstyled text-left"><LI> [1] Crook, N. et al. In vivo continuous evolution of genes and pathways in yeast. Nat. Commun. 7, 13051 (2016).</LI><LI> [2] Weber, E., Engler, C., Gruetzner, R., Werner, S. &amp; Marillonnet, S. A Modular Cloning System for Standardized Assembly of Multigene Constructs. PLOS ONE 6, e16765 (2011)</LI><LI>  [3] A One Pot, One Step, Precision Cloning Method with High Throughput Capability</LI></UL></DIV></ARTICLE></DIV></DIV><DIV class="container fixed-bottom backtotop" style="padding-bottom: 10px; padding-right:0px; position: fixed; left: 120px;"><DIV class="row text-right text-xs-right text-sm-right text-md-right  text-lg-right  "><DIV class="col-xs-4 col-sm-4 col-md-12 col-lg-12"><A class="y btn-sm" href="#backtotop" role="button" style=" color:white; border-radius: 10px; background-color:#22D66C; text-decoration:none; ">  Back to top</A></DIV></DIV></DIV><SECTION id="footer"><DIV class="container"><DIV class="row text-center text-xs-center text-sm-center text-md-right"><DIV class="col-xs-12 col-sm-3 col-md-3 col-lg-6"><H5 class="text-left" style="border: none;padding: 0px">iGEM Sorbonne U Paris</H5><P class="text-left" style="color:white; font-size:16px;"> 4 place Jussieu  75005 Paris France</P></DIV></DIV></DIV><DIV class="container"><P class="text-center" style="color: white; padding:20px 0 20px 0">Made in Paris with love © iGEM Sorbonne U Paris 2018</P></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>