<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Sorbonne_U_Paris_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Sorbonne U Paris/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="collapse navbar-collapse " id="navbarTogglerDemo02"><UL class=" navbar-nav "><LI class="nav-item dropdown" id="mobile-homepage"><A class="nav-link" href="https://2018.igem.org/Team:Sorbonne_U_Paris" style="color:white;"> Home page</A></LI><LI class="nav-item dropdown  "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white">  Project
        </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Description">Description</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Design">Design</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Model">Model</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Demonstrate">Demonstrate</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Improve">Improve</A></DIV></LI><LI class="nav-item dropdown  "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Biology</A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Experiments">Experiments</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Results">Results</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Parts">Parts</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/InterLab">Interlab</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Safety">Safety</A></DIV></LI><LI class="nav-item dropdown  "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Human practices
        </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Human_Practices">Human Practices</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Integrated_Human_Practices">Integrated HP</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Public_Engagement">Education &amp; Engagement</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Meet_up">Meet up</A></DIV></LI><LI class="nav-item dropdown "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Notebook
        </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Notebook">Notebook</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Protocols">Protocols</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Brainstorming">Brainstorming</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Team </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Team">Members</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Collaborations">Collaborations</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Attributions">Attributions</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Sponsors">Sponsors</A></DIV></LI><LI class="nav-item dropdown  "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Awards</A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://igem.org/2018_Judging_Form?team=Sorbonne_U_Paris">Judging form</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Medals">Medals</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Giant_Jamboree">Giant Jamboree</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Sitemap" style="color:white;"> Site map</A></LI></UL></DIV><DIV class="global"><DIV class="container"><DIV id="index1" class="side-nav index fixed-top"><UL class="list-group list-unstyled"><LI style="font-size: 25px; font-weight:bold;text-align:center">Index</LI><LI class="list-group-item"><A class="pointer" href="#1">I-Problematic</A></LI><LI class="list-group-item"><A class="pointer" href="#2">II-Why do we need a model?</A></LI><LI class="list-group-item"><A class="pointer" href="#3">III- First strategy</A></LI><LI class="list-group-item"><A class="pointer" href="#4">IV-Robust and easy model</A></LI><LI class="list-group-item"><A class="pointer" href="#5">V-Test of the model</A></LI><DIV class="container"><ARTICLE class="col-md-10"><H1>Modelling</H1><P class="introduction-text">Living organisms and biological processes are usually complex to understand and their behavior hard to predict, therefore mathematical models help in simplifying the phenomena in question. In fact, these models, through their logical and scientific simplifications, help us predict the behaviours and through this, they help to define the experiments or even more, the project’s finalities. 
Synthetic biology is based on a design test learn virtuous circle. We designed a genetic tool to enable in vivo directed evolution in microalgae, more particularly in <I>Chlamydomonas reinhardtii</I>. Our aim is to create a mutant library of the targeted gene, large enough to permit the directed evolution. We designed a genetic tool fully modular. The whole system is split into basic parts stored in Level 0 plasmids of the MoClo kit. This allows to change every element one by one or by combinatorial assembly. Our system is made to be improved through systematic assembly of different regulatory elements or retrotransposon parts. Selecting the construction by experimentally testing the synthetic transposons for every change is a very tedious and exhausting work. Moreover, getting a real estimation of the library’s size for every variation of the system is quite challenging. In fact, we would need an in silico model to find the most important and predictable bottleneck.</P></ARTICLE></DIV><ARTICLE class="col-md-10"><H2 class="title-h2" id="2"> Why do we need a model and influence of the model on the whole project </H2><H4>Initial thoughts </H4><P>Initially, when we started working on ourin vivo directed evolution, one of the first questions was : “How do we know when to stop the induction of the retrotransposon to have the biggest library for a given sequence length in the shortest time with the smallest culture volume ?”. The next question was : what are the more important parameters that will have an influence on the library size? ”What are the parameters affecting the most the size of the library ? </P><P>Afterwards, we realized that the model can do much more than answer these questions. Modeling has a huge impact on our project. If we manage to have a reliable model, we can create a web tool helpful for in vivo directed evolution-users to choose the best different “MoClo parts” to assemble synthetic retrotransposons that would fit their applications. </P><P>The first step will be to make a first run with the CARGO and a standard Reverse-Transcriptase (RT). This run will allow users to test the activity of their construction and collect simple data that will help them to refine their construction. It will help scientists to find an answer to those questions (not exhaustive): 
<UL><LI> Should I use a stronger promoter to induce the transposons?</LI><LI>Should I use a different Reverse-Transcriptase? </LI><LI>Should I use different LTRs/integrases?  </LI><LI>Is the CARGO sequence too big? </LI><LI>Can I use more than one gene as CARGO?
 </LI></UL></P><P>The models need to use simple data that can easily be collected during the first retrotransposons experiments like the number of cells alive during RT induction, the number of RT inserted inside the genomic DNA, the transcriptomic level of the mRNA … </P><H4>How modeling impacts our project </H4><P>At the beginning, we were thinking of modelisation as a tool to get insight on how can we improve our retrotransposon. Our first modeling strategy needed experimental parameters than are tedious to obtain with complicated experiments and a lot of sequencing data.</P><P>We came up with a new strategy. Our second model which is more simple and robust : we could automate all the project because modeling was the perfect tool to screen the library size made by the retrotransposons directly from a 96-well microplate.  </P><P>When we understood the usefulness of a model that could answer the previous questions, we came up with a new idea that will combine MoClo assembly, automation and modeling analysis. </P><DIV class="container d-flex justify-content-center" style="padding:20px"><FIGURE style=" width: 70%;"><FIGCAPTION style="padding: 10px;"><B>Figure 1 </B>Global scheme of our future strategy </FIGCAPTION></FIGURE></DIV><P>MoClo assembly allows the quick and easy generation of a small library composed of a synthetic retrotransposon with differents parts. These assemblies can easily be generated by automation with the help of an Opentron OT-2 (protocols available for MoClo cloning). Once the synthetic transposons constructs are complete, they will be transformed in <I>Chlamydomonas</I>. By automated or manual colony picking, every clone will be cultured in a 96-well plate. </P><P>The next step was to calibrate the model. This will be done by automation with induction of the retrotransposons in 96-well microplates. Simple parameters will be monitored: 
- <B>Growth curve</B> of <I>Chlamydomonas</I> while being induced. This will be used to calculate the half-life of the strain while induced. In other words, it measures the aggressivity of the retrotransposons (mobile elements can insert in essential genes etc…). <B>Number of retrotransposons inserted at the end of the induction.</B> For our first test, we will consider that the number of inserted retrotransposons is linear correlated with time of induction.  </P><P>Once these parameters are collected, they will be used by our model to determine the best synthetic retrotransposon construction for a given CARGO. It will also estimate the best induction time and optimal parameters (if they are tested in calibration experiments, for example induction temperature has a huge impact in our experiments).
 </P><P>This modeling study will provide preliminary results, before the generation of an in vivo library at scale (hundred milliliters of cultures or even liters) with optimized parameters. </P><P>Opentron OT-2 can be use for every step of this process. Most of the protocols are already online and it is easy to develop with Python the ones that will be specific to our iGEM project. </P><H2 class="title-h2" id="3">First strategy</H2><P>Our first approach was a mechanistic and dynamic model based on differential equations. The goal was to describe and integrate the individual parameters one by one at the molecular level (gene, RNAs, proteins etc...). We wanted to model what will happen in a single cell and then multiply by the number of cells implied to get the size of the library(<B>Figure 2 </B>). </P><DIV class="container d-flex justify-content-center" style="padding:20px"><FIGURE style=" width: 100%;"><FIGCAPTION style="padding: 10px;"><B>Figure 2 </B>Mechanistic and dynamic model with differential equations of our first approche</FIGCAPTION></FIGURE></DIV><P>Here is an overview of the different parameters we wanted to integrate: </P><P>While the RT was still in construction, we could not use real parameters to calibrate our model. We started looking into the literature to find some data to run the model for a first trial. So, we wanted to perform a mathematical modeling of our system in order to predict the different factors involved.</P><P>While this model can be very efficient, a lot of experiments are needed to calibrate it properly. For example:  </P><UL><LI><B>Fp</B>: The promoter of the retrotransposon is pNIT and based on previous reports, we estimated its force of transcription which helped us in calculating an approximate frequency of transcription of the construct. </LI><LI><B>Futr:</B> The number of transcribed RNAs is closely monitored in eukaryotes as the RNAs are usually degraded after exiting the nucleus. However, there are different conditions that lead to their degradations before they get translated or reverse- transcribed. In fact, several events can lead to the early degradation of the RNAs: the splicing of the introns can induce the total degradation of the RNA, mutated RNAs are eliminated as part of the quality control process and finally the “constitutive” degradation of RNA occurs regularly in an eukaryotic cell (Houseley and Tollervey, 2009). qPCR can be used to determine the quantity of mRNA over the time.</LI><LI><B>Frt</B>: Reverse Transcriptase (RT) that has a high error rate leading to random mutations in the resulting copy of the transcript.  </LI><LI><B>Fint</B>: Number of retrotransposons integrated after induction. This depends of the aggressivity of the retrotransposons and should vary within the various types of retrotransposons. qPCR can be used to count the number of copies of the retrotransposon.  </LI><LI><B>b(-cDNA)</B>:  After being spliced, the resulting RNA is treated by the Reverse Transcriptase that is encoded by the POL gene to generate a complementary DNA (cDNA). It has been reported that the cDNA in human and mouse cells are usually digested by a protein called TREX1 (Kawane et al., 2014). Since we are working with Chlamydomonas reinhardtii which is an eukaryotic cell, we assumed that the cDNA produced in our system would also be digested. In fact, this process would prevent its integration into the genome and therefore it stops the process of directed evolution. This step is a turning point in our system. </LI><LI><B>Fdeath:</B> cDNA is supposed to integrate into the genome with the help of integrase which is encoded by the pol gene of the retrotransposon. According to previous studies in plants, an LTR-retrotransposon found in plants, named tobacco Tnt1 elements known for its high transposition, integrates mostly into or next to gene-coding sequences and that these integrations are usually not detrimental for the host (Le et al., 2007). But there are still chances that the integration will lead to the death of the cell. This parameter can be monitored with the growth curve of the cells while being induced. </LI></UL><P>After some literature research, we do not pursued with this model as too much work was needed, with the risk of getting results that we can not easily confront with the real events. In fact, even if the parameters we wanted to calibrate could be obtained experimentally, it needed too much work and optimization that would not fit into the tight schedule of the iGEM competition. Plus, the differents experiments to get the data needed are not easy to adapt to automation. Moreover, every variable and parameters can be the source of variability making the strategy too complex and not reliable for our needs. </P><H2 class="title-h2" id="4">A more robust and easy model to calibrate </H2><P>We decided to switch strategy for the model and developed a simplest one that way more robust with an analytical solution. Only two experimental parameters are needed: i) a growth curve while induction of RT and ii) the number of integrated RT. This model will allow us to determine the best induction time to generate the largest library. 
 </P><UL><LI><B>Growth curve and half-life of yeast cells</B> : 96 h. We make the hypothesis that the half-life of microalgae will be much longer than that of yeast.  We choose to describe the population of microalgae with an exponential model survival. We think the process of survival after the integration is without memory.  That means that if a cell has already 3 integrated retrotransposons, the chance of kill it after a fourth integration has the same probability if the cell has zero retrotransposons or 10. </LI><LI><B>Number of integrated retrotransposons per cell </B>: We used 1 transposon per cell (0.02 for yeast). The microalgae’s genome has much more transposable elements than yeast (up to 30%). For example in Volvox carteri there are 80 copies of Osser in the genome. In yeast, there are only 5 copy of Ty16. For the linear function that describe the number of retrotransposons over time. We made this postulate because it simply the model for the first run. This function can be change and refine if needed. We will need more insight and experimental data to find a function that describe perfectly the reality.</LI><LI><B>CARGO sequence length</B> : We use a cargo of 1000 base pairs. 
 </LI></UL><DIV class="container-fluid main-container"><P>Let <SPAN class="math inline">\(\frac{1}{\lambda}\)</SPAN> be the average survival time of a cell, we suggest an exponential model of survival: <SPAN class="math display">\[x(t) = e^{-t\lambda}\]</SPAN> To obtain <SPAN class="math inline">\(\lambda\)</SPAN> from <SPAN class="math inline">\(t_{\frac{1}{2}}\)</SPAN> the half-life: <SPAN class="math display">\[\frac{1}{2} = e^{-t_{\frac{1}{2}}\lambda}\]</SPAN><SPAN class="math display">\[-\log{2} = -t_{\frac{1}{2}}\lambda\]</SPAN><SPAN class="math display">\[\boxed{\lambda = \frac{\log{2}}{t_{\frac{1}{2}}}}\]</SPAN> So for <SPAN class="math inline">\(t_{\frac{1}{2}}=24\mathrm{h}\)</SPAN>:</P><P>Let us suppose the number <SPAN class="math inline">\(y\)</SPAN> of transposons per cell is a linear function of time, and that there is for example the number of transposons <SPAN class="math inline">\(y\left(t_{\frac{1}{2}}\right) = y_{t_{\frac{1}{2}}}\)</SPAN> then: <SPAN class="math display">\[y(t) = t\frac{y_{t_{\frac{1}{2}}}}{t_{\frac{1}{2}}}\]</SPAN> So we have for <SPAN class="math inline">\(y_{t_{\frac{1}{2}}} = 1\)</SPAN>:</P><P>Finally we propose <SPAN class="math inline">\(z(t)\)</SPAN> the number of transposons in the culture as being the product of the number of cells by the number of transposons per cell:: <SPAN class="math display">\[z(t) = x(t)y(t)\]</SPAN> And therefore:</P><P>Once the number of transposons is obtained, the question of the number of unique alleles generated L comes. Consider the Coupon collector’s problem: a collector looking to have all the coupons of a particular series however at the time of purchase the coupon’s number is unknown; the question is how much does take to have the complete collection? One can show that the solution is a sum of random variables according to a geometrical law (cf.Wikipedia). In the Coupon collector’s problem the draw probabilities are uniform, but in reality mutations are not at all uniform, we will figure out how to overcome this difficulty.</P><P>Let us proceed by steps:</P><OL style="list-style-type: decimal"><LI><P>Let us suppose that each nucleotide in the DNA sequence of interest of length <SPAN class="math inline">\(N\)</SPAN> has a uniform probability <SPAN class="math inline">\(p\)</SPAN> to mutate into one of the three other possible nucleotides. In this case for a given transposon, the number of mutated bases <SPAN class="math inline">\(K\)</SPAN> is a random variable following a binomial law : <SPAN class="math inline">\(K \sim B(N, p)\)</SPAN> and so of expectation <SPAN class="math inline">\(\mathbb{E}(K) = Np\)</SPAN></P></LI><LI><P>Let us separate all of the possible alleles in <SPAN class="math inline">\(N+1\)</SPAN> subsets <SPAN class="math inline">\(A_k\quad (k \in0..N)\)</SPAN> each containing the <SPAN class="math inline">\(a_k\)</SPAN> alleles with the same number <SPAN class="math inline">\(k\)</SPAN> of mutated bases. Let us determine the value of the <SPAN class="math inline">\(a_k = \#A_k\)</SPAN> (cardinal of <SPAN class="math inline">\(a_k\)</SPAN> meaning the number of elements in <SPAN class="math inline">\(A_k\)</SPAN>). For any element of <SPAN class="math inline">\(A_k\)</SPAN> we select <SPAN class="math inline">\(k\)</SPAN> modified nucleotides among <SPAN class="math inline">\(N\)</SPAN>, each of which can have one of 3 new values possible. So we have : <SPAN class="math display">\[a_k = {N \choose k}3^k \quad \left(= \frac{N!3^k}{(N-k)!k!}\right)\]</SPAN></P></LI><LI><P>Each mutation of a nucleotide having a uniform probability, then within one <SPAN class="math inline">\(A_k\)</SPAN> all alleles are equiprobable. We get back to the Coupon collector’s problem if one considers only a subset. However the solution to the Coupon collector’s problem gives us the number of necessary transposons <SPAN class="math inline">\(T_{L_k}\)</SPAN> on average to obtain a certain number of unique alleles <SPAN class="math inline">\(L_k\)</SPAN>, and not the inverse that interests us, that is the number of alleles <SPAN class="math inline">\(L\)</SPAN> depending on the number of transposons <SPAN class="math inline">\(T\)</SPAN>.</P></LI></OL><P><SPAN class="math display">\[\mathbb{E}(T_{L_k}) = a_k\sum_{j=0}^{L_k-1}\frac{1}{a_k-j}\]</SPAN></P><OL start="4" style="list-style-type: decimal"><LI><P>To obtain the solution of the inverse problem, the simplest way is to proceed by framing: if <SPAN class="math inline">\(\mathbb{E}(T_{L_k}) \le T \le \mathbb{E}(T_{L_k+1})\)</SPAN> then <SPAN class="math inline">\(L_k \le L \le L_k+1\)</SPAN></P></LI><LI><P>We know how many integrated transposons we have, from the first point we know their distribution by number of mutations, and from the second and third points we can find the number of unique alleles per number of mutations, so we have enough information to solve the whole problem, meaning to determine the number of unique alleles that we can expect.</P></LI></OL><P>To illustrate, let us consider at the optimal time: <SPAN class="math display">\[\begin{cases}
N = 1000\\
p = \frac{1}{1500}\\
\mathrm{cells} = 8,000,000,000
\end{cases}\]</SPAN></P><PRE><CODE>##     K       L.k
## 1   0         1
## 2   1      3000
## 3   2   4495500
## 4   3 106240523
## 5   4  17877883
## 6   5   2375773
## 7   6    262830
## 8   7     24897
## 9   8      2061
## 10  9       151
## 11 10        10</CODE></PRE><PRE><CODE>## [1] &quot;Total : 131282791&quot;</CODE></PRE></DIV><H2 class="title-h2" id="5">Test of the model with data published by Crook and al. (2016) on Saccharomyces cerevisae </H2><P>Since we could not use data provided by our system in Chlamydomonas, we decided to use the data of the original paper of Crook et al to confront our model with the reality<SUP>5</SUP>.  </P><P>Then we used parameters determined from the original paper of Crook et al 2016 to confront our model with experimental results<SUP>5</SUP>:  
<UL><LI><B>Growth curve and half-life of yeast cells </B> : 48 h. We match this value so the maximum inducing time determined by our model is roughly the same as the inducing time used in the paper (3 days) </LI><LI><B>Number of integrated retrotransposons per cell </B>: We used 0.02 transposons per cell.  </LI><LI><B>CARGO sequence length</B> : We use a cargo of 1000 base pairs.  </LI></UL></P><DIV class="container-fluid main-container"><P><SPAN class="math display">\[\begin{cases}
N = 1000\\
p = \frac{1}{1500}\\
\mathrm{cells} = 8,000,000,000
\end{cases}\]</SPAN></P><PRE><CODE>##     K       L.k
## 1   0         1
## 2   1      3000
## 3   2   4495500
## 4   3 209965272
## 5   4  35755671
## 6   5   4751546
## 7   6    525660
## 8   7     49795
## 9   8      4123
## 10  9       303
## 11 10        20
## 12 11         1</CODE></PRE><PRE><CODE>## [1] &quot;Total : 255551054&quot;</CODE></PRE></DIV><P> Our model finds a library size of : 2.5  million mutants for 8 billion cells. When we translate for 10 billion cells, 3.1 million of mutants are generated.  Crook et al 2016, estimated 2.7 millions with the use of the PEDEL software. In order to use this software, they isolated and sequenced hundreds of colonies to count how many times 3 RT mutants (one with 1 mutation, 1 with 2 mutations and 1 with 3 mutations) can be isolated. This is an expensive and tremendous work. 
</P><P>Some work is still needed to fully validate and refine our model with differents conditions. Moreover, we will need to test the whole system once we have finished with the construction of our first synthetic retrotransposon. Nevertheless, those preliminary results are promising because they are matching real experiments from Crook et al work on S. cerevisiae and retrotransposon Ty1.  </P><P>To conclude, our model could be used to characterize the activity of synthetic retrotransposons in a high-through pipeline. We can automate the construction of different retrotransposons because we design it to be fully modular with the use of a MoClo kit. Then by using qPCR experiments and OD measurements in 96-well microplates we can screen for the best construction. This can only be done using the GoldenGate cloning system along with our model.  </P><DIV class="container" id="references"><H5 class="text-center">References </H5><UL class="list-unstyled text-left"><LI> [1] Bengtsson, M., Hemberg, M., Rorsman, P. &amp; Ståhlberg, A. Quantification of mRNA in single cells and modelling of RT-qPCR induced noise. BMC Mol. Biol.9,63 (2008).
</LI><LI> [2] Houseley, J. &amp; Tollervey, D. The Many Pathways of RNA Degradation. Cell136,763–776 (2009).</LI><LI>  [3] Kawane, K., Motani, K. &amp; Nagata, S. DNA Degradation and Its Defects. Cold Spring Harb. Perspect. Biol.6,a016394 (2014).</LI><LI>  [4] Distribution dynamics of the Tnt1 retrotransposon in tobacco | SpringerLink. Available at: https://link.springer.com/article/10.1007%2Fs00438-007-0281-6. (Accessed: 1st October 2018)</LI><LI>  [5] Crook, N., Abatemarco, J., Sun, J., Wagner, J.M., Schmitz, A., and Alper, H.S. (2016). In vivo continuous evolution of genes and pathways in yeast. Nat. Commun. </LI><LI>  [6]Curcio, M.J., Lutz, S., Lesage, P., and Diderot, P. (2015). The Ty1 LTR-retrotransposon of budding yeast, Saccharomyces cerevisiae. Microbiol Spectr 3, 1–35

</LI></UL></DIV></ARTICLE><DIV class="container fixed-bottom backtotop" style="padding-bottom: 10px; padding-right:0px; position: fixed; left: 120px;"><DIV class="row text-right text-xs-right text-sm-right text-md-right  text-lg-right  "><DIV class="col-xs-4 col-sm-4 col-md-12 col-lg-12"><A class="y btn-sm" href="#backtotop" role="button" style=" color:white; border-radius: 10px; background-color:#22D66C; text-decoration:none; ">  Back to top</A></DIV></DIV></DIV><SECTION id="footer"><DIV class="container"><DIV class="row text-center text-xs-center text-sm-center text-md-right"><DIV class="col-xs-12 col-sm-3 col-md-3 col-lg-6"><H5 class="text-left" style="border: none;padding: 0px">iGEM Sorbonne U Paris</H5><P class="text-left" style="color:white; font-size:16px;"> 4 place Jussieu  75005 Paris France</P></DIV></DIV></DIV><DIV class="container"><P class="text-center" style="color: white; padding:20px 0 20px 0">Made in Paris with love © iGEM Sorbonne U Paris 2018</P></DIV></SECTION></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>