Team:Sorbonne U Paris/Results
Home page
Project
Description Design Model Demonstrate Improve
Biology
Experiments Results Parts Interlab Safety
Human practices
Human Practices Integrated HP Education & Engagement Meet up
Notebook
Notebook Protocols Brainstorming
Team
Members Collaborations Attributions Sponsors
Awards
Judging form Medals Giant Jamboree
Site map
Results
Here we are presenting the progression of our lab work, the things we managed to archive and the ones that we could not record, the problems we faced and our new perspectives.
Our progression in cloning
As of october 16 2018, by using the Moclo tool kit for Chlamydomonas Reinhardtii developed by Pierre Crozet and Stephane Lemaire, we were able to clone most of the Level 0 parts for our retrotransposon tool except for the Gag/Pol part. We believe we will be able to complete the Gag/Pol cloning, to create our Level 1 plasmids as well as the level M in the near future. For each Level 0 plasmid we cloned, we systematically checked the sequence, by sending samples to Eurofin Genomics.
Figure 1: Overview of our progress in the cloning experiments In blue: parts that we have managed to clone or are available. In purple: parts we have yet to clone or we have difficulties cloning. The name of the plasmid backbones are referring to the Crozet et al. MoClo tool kit. The available parts are also from that kit. Spec is a short word for Spectinomycin and Amp for Ampicillin.
Details on our currently available parts
1) Rubisco intron ( RBCS2i1): Bba_K2703000  - Backbone: piCH41258
This plasmid contains the intron for the rubisco. Based on the retrotransposon tool designed by Crook et al., an RT-PCR experiment will help us determine whether or not this intron was spliced in our microalgae and therefore if an event of retro transposition took place. We designed and cloned this part ourselves, then checked its sequence.
2) Paromycin CDS:  Bba_K2703008 - Backbone: pICH41264
This plasmid contains the coding region of the Paromomycin, a resistance gene we use in our construction, and that we are placing upstream from the intron so that it would be transcribed whether or not the intron is spliced.  We designed and cloned this part ourselves, then checked its sequence.
3) pSAD promoter: Bba_K2703002 - Backbone: pAGM9121
This plasmid contains the promotor of pSAD gene and is necessary for the transcription of pSAD. We designed and cloned this part ourselves, then checked its sequence.
4) 3’LTR : Bba_K2703003 - Backbone: pAGM9121
This part, as well as the 5’LTR is necessary to insure a single copy, site specific reintegration of the transposable element. We designed and cloned this part ourselves, then checked its sequence.
5) 5’LTR  : Bba_K2703004- Backbone: pAGM9121
This part, as well as the 5’LTR is necessary to insure a single copy, site specific reintegration of the transposable element. We designed and cloned this part ourselves, then checked its sequence.
6) pNIT promoter-5UTR- Backbone: pICH41233
This plasmid contains an inducible promoter pNIT, and a 5’UTR region. This part was given to us by Crozet et al., and is available on their MoClo tool kit.
7)TPSAD (Terminator+3’UTR)- Backbone:pICH41276
This plasmid contains an inducible promoter pNIT, and a 5’UTR region. This part was given to us by Crozet et al., and is available on their MoClo tool kit.
8) OtsA - Backbone:pAGM1276
This plasmid contains the OtsA sequence, which when fused with OtsB, will be responsible our trehalose production. We designed and cloned this part ourselves, then checked its sequence.
9) Trehalose biosensor from Addgene #79754 - Backbone: pAGM1276
This plasmid will be transfected into C.reinardtii. When this biosensor will be expressed, it will enable us to identify the clones producing the most important quantities of trehalose. This will be determined by the reading of a fluorescence signal emitted by the biosensor when it binds trehalose.
Figure 2: Representation of a trehalose compound (4)
Details on the parts we have yet to create:
1)Gag/Pol- Backbone: pAGM9121
This plasmid contains the Gag/Pol part encoding for all the necessary factors for the transposition (RNA binding protein for gag, reverse transcriptase, integrase and RNAseH for pol). We designed and cloned this part ourselves, but the cloning experiments did not succeed due to the problems we faced.
2) OtsB-  Backbone: pAGM1299
This plasmid contains the OtsB sequence, which when fused with a 2A peptide OtsA will be responsible for trehalose production. We designed and cloned this part ourselves, then checked its sequence.
Parts characterization
We characterized the paromomycin resistance genes. To see more details about this, please refer to the characterization rubric.
Experiments
We assess the functionality of the Trehalose biosensor (TBMP):
Before starting the cloning of this gene we wanted to make sure it was working as described. This synthetic biosensor was already under the control of the IPTG inducible promoter Ptrc on a kanamycin resistance backbone. We transformed this plasmid in E.coli DH5 alpha and induced the promoter without or with three different concentrations of trehalose. We used the minimal media M63 without glucose. Trehalose is imported inside the cell to be metabolized by E.coli. We collected GFP-fluorescence data with a microplate reader at a rate of one measure every 10 minutes for 12 hours.
Figure 3: Plate plan for the characterization experiments of the Trehalose biosensorThe trehalose concentration changes every 3 columns:  1-3: 0µM of trehalose, 4-6: 10µM, 7-9: 1Mm, 10-12: 10mM. Each well contains 200µL of media.  For each row, the strain is different. A: Untransformed DH5α, B: DH5-alpha ttransformed with a pl1, C: DH5α transformed with the biosensor (Addgene plasmid), Kanamycin resistant strain (KR) without IPTG, D: DH5α transformed with the biosensor (Addgene plasmid) (KR) + IPTG, E: Replicate of D.
Figure 4: A: Ratio fluo/OD of strains cultured with 10 uM of trehalose B: Relative standard variation of the ratio fluo/OD of strains cultured with 10 uM of trehalose (expressed in %).A/ Fluorescence values of E.coli DH5alpha normalized by the OD 600 , data were collected every 10 minutes over 12 hours in triplicate. Yellow is the Addgene #79754 plasmid with IPTG (1 M), and in gray the same strain without IPTG. There is a leaky expression because we did not use a tightly regulated system. In blue is the negative control, it is the strain without plasmid. In orange and cyan E.coli DH5 alpha transformed with a MoClo plasmid that should contain the TBMP CDS. After further verification of the plasmid was not correct. B/ Relative standard deviation of the normalized fluorescence values (expressed in %).
The biosensor TBMP appears to work as described in the paper of Nadler DC et al., with trehalose at 10 uM (4).
Here we presented some of our results. And we are still continuing the experiments, so new results will be generated.
References
[1] Crozet, P. et al. Birth of a Photosynthetic Chassis: A MoClo Toolkit Enabling Synthetic Biology in the Microalga Chlamydomonas reinhardtii. ACS Synth. Biol. 7, 2074–2086 (2018).
[2] Crook, N. et al. In vivo continuous evolution of genes and pathways in yeast. Nat. Commun. 7, 13051 (2016).
[3] rehalose Compounds | Synthose. Available at: http://www.lcsci.com/chemicals/trehalose.cfm. (Accessed: 4th October 2018)
[4] . Rapid construction of metabolite biosensors using domain-insertion profiling | Nature Communications. Available at: https://www.nature.com/articles/ncomms12266. (Accessed: 4th October 2018)
Back to top
iGEM Sorbonne U Paris
4 place Jussieu
75005 Paris France
Made in Paris with love © iGEM Sorbonne U Paris 2018
