<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Sorbonne_U_Paris_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Sorbonne U Paris/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="collapse navbar-collapse " id="navbarTogglerDemo02"><UL class=" navbar-nav "><LI class="nav-item dropdown" id="mobile-homepage"><A class="nav-link" href="https://2018.igem.org/Team:Sorbonne_U_Paris" style="color:white;"> Home page</A></LI><LI class="nav-item dropdown  "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white">  Project
        </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Description">Description</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Design">Design</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Model">Model</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Demonstrate">Demonstrate</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Improve">Improve</A></DIV></LI><LI class="nav-item dropdown  "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Biology</A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Experiments">Experiments</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Results">Results</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Parts">Parts</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/InterLab">Interlab</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Safety">Safety</A></DIV></LI><LI class="nav-item dropdown  "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Human practices
        </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Human_Practices">Human Practices</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Integrated_Human_Practices">Integrated HP</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Public_Engagement">Education &amp; Engagement</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Meet_up">Meet up</A></DIV></LI><LI class="nav-item dropdown "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Notebook
        </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Notebook">Notebook</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Protocols">Protocols</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Brainstorming">Brainstorming</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Team </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Team">Members</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Collaborations">Collaborations</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Attributions">Attributions</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Sponsors">Sponsors</A></DIV></LI><LI class="nav-item dropdown  "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Awards</A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://igem.org/2018_Judging_Form?team=Sorbonne_U_Paris">Judging form</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Medals">Medals</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Giant_Jamboree">Giant Jamboree</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Sitemap" style="color:white;"> Site map</A></LI></UL></DIV><DIV class="global"><DIV class="container"><ARTICLE class="col-md-10"><DIV id="index1" class="side-nav index fixed-top"><UL class="list-group list-unstyled"><LI style="font-size: 25px; font-weight:bold;text-align:center">Index</LI><LI class="list-group-item"><A class="pointer" href="#1">I-Chlamydomonas</A></LI><LI class="list-group-item"><A class="pointer" href="#2">II-Modular Cloning (MoClo) </A></LI><LI class="list-group-item"><A class="pointer" href="#3">III- MoClo Synthetic Retrotransposon</A></LI><LI class="list-group-item"><A class="pointer" href="#4">IV-Trehalose and biosensor</A></LI></UL></DIV><H1>Design</H1><DIV class="container paragraph-bottom"><H2 id="1" class="title-h2">I – Chlamydomonas  </H2><P><I>Chlamydomonas reinhardtii</I> is a unicellular green microalga that lives in freshwater. </P><P>Microalgae are photosynthetic organisms that have huge potentials for synthetic biology as they can be used to produce human therapeutic proteins (Rasala and Mayfield 2011), biofuels (Scranton et al. 2015) or sugar in a more sustainable way than common heterotrophic chassis. Indeed, as microalgae are photoautotrophic, they need much less inputs than heterotrophic chassis and can be grown in marine water or photobioreactors, thus they do not compete with arable land. </P><P>Initially our project was to make a proof of concept of an engineered microalgae for sugar production in marine waters. The goal was to fix carbon as carbohydrates to feed large scale fermentation with a minimal impact on the environment. We chose the microalgae <I>Chlamydomonas reinhardtii</I> as chassis even though it is a unicellular green microalga that lives in freshwater, however, Lachapelle and colleagues adapted it through long term selection to marine waters (Lachapelle, Bell, and Colegrave 2015).</P><P> Moreover, the sequencing of the Chlamydomonas genome is complete since 2007 (Merchant et al. 2007) and several molecular tools are available to perform transformation and edition of the genome (Jinkerson and Jonikas 2015). More recently, a Modular Cloning (MoClo) kit has been generated allowing combinatorial assembly and faster building than other cloning methods (Crozet et al. 2018). The combination of these tools should allow efficient engineering of Chlamydomonas and a great potential to the development of our project</P><P>One of the most powerful tools for metabolic engineering is directed evolution. This is usually done by an <I>in vitro</I> library generated by error-prone PCR and cloned by a restriction enzyme into a plasmid that is transformed with high efficiency into competent cells. However, “PCR-mediated directed evolution” cannot be done effectively and easily on slow-replicating and non-plasmid-bearing eukaryotes such as microalgae. To date, no study of directed evolution in vivo in microalgae has been reported.  </P><P c="">Crook et al. developed <I>in vivo</I> directed evolution mediated by retrotransposons (Crook et al., 2016). They synthetically optimized the retrotransposons <I>Ty1</I> to enable in vivo generation of mutant libraries in the yeast S. cerevisiae. When coupled to growth selection, this approach enabled in vivo continuous evolution (ICE). We chose to adapt this strategy to the microalgae <I>Chlamydomonas reinhardtii</I>. We aim to go further and improve this work (that was also the project on the Vienna BOKU team in 2017) by creating a synthetic retrotransposons completely modular and compatible with the MoClo standard.  We hope this work will help spreading in vivo directed evolution in algae among fundamental scientists as well for industrial application.  </P></DIV><DIV class="container paragraph-bottom"><H2 id="2" class="title-h2">II – Modular Cloning (MoClo)</H2><P> As we want the retrotransposon to be fully modular so we used a standardized DNA strategy called Modular Cloning (referred to here after as MoClo). Both BioBricks and MoClo allow the assembly of basic parts such as promoters, CDS or terminators however the MoClo relies on the Golgen Gate cloning strategy (Weber et al. 2011).  </P><H4>A- Type IIS restriction enzymes</H4><P> Restriction enzymes are endonucleases which catalyze the hydrolysis of a double stranded DNA resulting in a double strand break at a specific sequence.  </P><P>The classical restriction enzymes (e.g. EcoRI) cut inside their recognition site. They release a part of this recognition site and have to be ligated with a DNA sequence cut with a compatible restriction enzyme. After the ligation, the restriction site is formed again. (<B>Scheme 1</B>)   </P><P>The type IIS restriction enzyme (<I>BpiI, BsaI</I>) cut downstream of their recognition sites and generate 4 bp  5’-overhangs (Engler et al. 2009) that are not specific to the restriction enzyme (<B>Scheme 1</B>).  </P><DIV class="container d-flex justify-content-center" style="padding:20px"><FIGURE style=" width: 100%;"><FIGCAPTION style="padding: 10px;"><B>Scheme 1: </B>Type II restriction enzyme </FIGCAPTION></FIGURE></DIV><H4>B- One-pot, one-step reaction (Engler et al. 2009)</H4><P>The golden gate cloning allows a directional assembly of multiple DNA fragments into a one-pot, one-step reaction. Indeed, the restriction enzyme and the ligase are added at the same time as there are no scars of the restriction site after the ligation when type IIS enzymes are used, making the final product stable during the reaction. The two enzymes (for digestion and ligation) do not work at the same temperature (37°C for the restriction enzyme and 16°C for the ligase). To increase the cloning efficiency, several cycles (37°C-16°C) are made.  </P><H4>C- MoClo assembly standard </H4><P> In the MoClo assembly standard (Weber et al. 2011), the smallest units are basic genetic parts such as promoter, CDS, 5’UTR. Each type of the basic parts has a position defined by the fusion sites on both sides (<B>Scheme 2</B>). Also, as all the same basic parts have a unique fusion site they are commutable at will. These basic units are cloned into plasmids with <I>BpiI</I> and are called &quot;Level 0&quot;. A Level 0 acceptor plasmid contains <I>BpiI</I> restriction sites flanking a <I>lacZ</I> cassette, a pUC origin of replication, and the spectinomycin resistance gene. Upon digestion with <I>BpiI</I>, the plasmid releases the <I>lacZ</I> cassette and the part of interest hybridizes to the plasmid through complementarity of these flanking 4 bp overhangs, corresponding to the fusion site. This implies that there is a specific acceptor plasmid for each position within the standard. These molecules are then ligated and the final products are used to transform E. coli (<I>LacZdelta</I>). The selection of the transformants carrying Level 0 plasmids is made by blue/white selection (pick white) on a LB medium with X-gal and spectinomycin.  </P><P>All Level 0 plasmids also contain a <I>BsaI</I> recognition site that allows a second golden gate reaction to create an entire transcription unit (Level 1) from several Level 0 plasmids (Weber et al. 2011). This is the specificity of the overhangs which define the order of the parts into a transcriptional unit. And we can screen for the Level 1 plasmid because they have the ampicillin resistance gene (and not spectinomycin as the Level 0).  </P><P>A full transcription unit can be made by only choosing the basic units. It also allows to test a module library in the same transcription unit and thus screen the best module for a specific application (e.g. different promoter for a coding sequence). It is very useful to optimize complex pathways in metabolic engineering.  </P><P>For <I>Chlamydomonas reinhardtii,</I> a specific MoClo toolkit has just been reported (Crozet et al. 2018), and we used their standard for the construction of the retrotransposon, as well as some parts of this kit </P></DIV><H2 id="3" class="title-h2">III – MoClo Synthetic Retrotransposon</H2><H4>A- Retrotransposon Generality </H4><P>Retrotransposons are Mobile Elements of the genome. They are classified in two main categories: class I and class II mobile elements. The latter have an RNA intermediate in opposition to class I that have a DNA intermediate. The retrotransposon uses a “copy and paste” mechanism where the DNA is first transcribed into RNA, then reverse transcribed into DNA and finally inserted at the target site (Piégu et al., 2015). </P><DIV class="container d-flex justify-content-center" style="padding:20px"><FIGURE style=" width: 60%;"><FIGCAPTION style="padding: 10px;"><B>Scheme 3: </B> Scheme of transposition </FIGCAPTION></FIGURE></DIV><H4>B – The choice of the Retrotransposon </H4><P>There are few LTR retrotransposons described in green algae. There are <I>REM1</I> (related to <I>Ty3/gypsy</I> retrotransposon) and <I>TOC1</I> retrotransposons in <I>Chlamydomonas reinhardtii</I>, and <I>OSSER</I> (related to <I>Ty1/copia</I> retrotransposon) in <I>Volvox carteri</I> (Sormacheva and Blinov 2011). As <I>TOC1</I> belongs to the DIRS-I family and transposes via a circular intermediate (Kim, Kustu, and Inwood 2006), we chose to focus on <I>REM1</I> and <I>OSSER</I> retrotransposons that are classical RNA intermediates retrotransposons. The main difference between these two is the order between the reverse transcriptase (RT) and the integrase within the POL gene (Sormacheva and Blinov 2011). <I>REM1</I> is a functional retrotransposon in <I>Chlamydomonas reinhardtii</I> but it has some structural features not described yet. Indeed, the typical RNAse H and integrase have not been identified, and a third ORF is in the reverse orientation between the 3’LTR and the ORF1 of GAG. Moreover, the copy number present in the genome vary depending on the strains which suggests different regulation mechanisms (Pérez-Alegre, Dubus, and Fernández 2005). 

</P><DIV class="container d-flex justify-content-center" style="padding:20px"><FIGURE style=" width: 100%;"><FIGCAPTION style="padding: 10px;"><B>Scheme 4: </B> Scheme of retrotransposon </FIGCAPTION></FIGURE></DIV><I>OSSER</I> retrotransposon (<B>Figure 4</B>) is similar to other copia retrotransposons already known, with a central domain coding for GAG and POL flanked by a direct repeat of 197 bp. The only specificity of <I>OSSER</I> is a one bp insertion in the middle of the RT, which is a translational slippage (Lindauer et al. 1993). We haven’t found data about its transposition rate.  <P>These data led us to choose <I>OSSER</I> as the retrotransposon in our system, knowing the risks that it might not be functional. Hoever, we did not choose <I>Volvox</I> as it is a very close relative to Chlamydomonas and orthogonality of DNA parts between species is expected. </P><H4>C – Design of our System  </H4><P>The purpose of our system is the induction of mutations via the error rate of the reverse transcriptase into a gene (named hereafter CARGO) inserted in the redesigned retrotransposon. We were inspired by the article of Crook and colleagues which describes this strategy to perform <I>in vivo</I> evolution in yeast (Crook et al. 2016). </P><P>We designed the synthetic retrotransposon to be fully modular. This was quite challenging in terms of MoClo design as we had to build a library of the different parts  composing the synthetic retrotransposon: pNIT1, 5’UTR of NIT1, 5‘LTR, GAG-POL, pPSAD, 5’UTR of PSAD, paromomycine resistance CDS (AphVIII), the 1st intron of CrRBCS2,  3’UTR of PSAD, and 3’LTR. Instead of using Level 1 plasmids to assemble transcription unit, we split the synthetic retrotransposons in two level 1 plasmid pL1-1F and pL1-2R. The inserts of these two plasmids can then be linked to a recipient plasmid forming a final construct (level M) giving the full synthetic retrotransposon.  </P><P>Our synthetic <I>Osser</I> retrotransposon consists of different main parts and features (<B>Scheme 5</B>):  
- The 5’ LTR: The main difficulty was to identify the wild type transcription regulators inside the 5’LTR. The purpose was to remove it and put Osser under the control of an inducible promoter pNIT. There are three regions: U3 with the TATA box, R with the transcription initiation and U5 (Curcio et al., 2015) . We manually annotated the LTR prior to design. The first feature we used to delimit U3 from R is the Polyadenylation site (PAS). PAS are both in 5’LTR and 3’LTR because these two sequences are strictly identical. Interestingly, the <I>Osser</I> PAS from <I>Volvox carterii</I> UGUAA is the same in <I>Chlamydomonas reinhardtii</I>. We truncated U3 which 22 bases upstream from the PAS motif. We linked the inducible promoter pNIT1 with the fusion site B2 (CCAT).</P><DIV class="container d-flex justify-content-center" style="padding:20px"><FIGURE style=" width: 100%;"><FIGCAPTION style="padding: 10px;"><B>Scheme 5: </B> Different main parts and features of retrotransposon. The scheme is interactive, by clicking on the gray circle you will have more details about this zone. </FIGCAPTION></FIGURE></DIV>


-The GAG-POL part contains all the activities necessary for the retrotransposition process(RNA binding protein for gag, reverse transcriptase, integrase and RNAseH for pol). We did not change anything in this sequence. One of the project’s perspectives will be to add all the GAGPOL elements as separated parts in the MoClo standard to increase modularity.
- Based on Crook et al's design we inserted in reverse a whole transcription unit between the end of GAGPOL and before the 3’LTR. This Cargo contains a resistance gene under the control of a constitutive promoter (pPSAD) and an intron (RBCS2i1) in the reverse orientation compared to the cargo. The orientation of the intron inserted at the beginning of the sequence of the resistance gene will avoid the expression of the cargo without retrotransposition. Indeed, only after RNA splicing will the cargo have an ORF that can be translated. We chose to use the B3 fusion site which contains the ATG codon initiator. In order for the paromomycine resistance gene (or any other CDS) to be in frame with this ATG, once the intron is spliced we added 2 nucleotides after the B4 site. Moreover, it is important to note that the PSAD 3’ UTR and terminator part should not have a bidirectional terminator activity. 
-To allow selection of transformants of the transposon, another resistance gene will be coupled to these TUs, the hygromycin resistance module. This module expresses under the control of the pAR strong and constitutive promoter and the 3’UTR/terminator of RbcS2 (Schroda et al Plant Journal, 2002), the AphVII CDS coding for an aminoglycoside phosphotransferase (Berthold et al, Protsist, 2012).
-We inverted the 3’LTR in pL1-R so it will be in the 5’-3’ direction in the final construction. <DIV class="container d-flex justify-content-center" style="padding:20px"><FIGURE style=" width: 100%;"><FIGCAPTION style="padding: 10px;"><B>Scheme 6: </B>Strategy of Moclo </FIGCAPTION></FIGURE></DIV><H4>D – Proof of Concept</H4><P>Once the construction will be obtained, we want to test its activity with our reporter system which relies on the paromomycine resistance gene (controlled by the pPSAD constitutive promoter) with the intron in reverse that disrupts the transcription frame. The main steps are: 1/ Transformation of WT <I>Chlamydomonas reinhardtti</I>, 2/ selection of hygromycine resistant clones, 3/ culture in TAP with ammonium until early log phase (106 cells/mL); 4/ Induction of the pNIT promoter by changing the nitrogen source (TAP with nitrate); plating the microalgae on TAP + paromomycine medium. The number of resistant colonies will depend of the transposition rate.  </P><P><B>Highlight of the reporter system to test the synthetic transposon activity:</B></P><P>1 – The pSAD promoter is in the reverse orientation of that of the intron. When the resistance gene is transcribed, the protein cannot be produced because the intron cannot be spliced. 
2 – If the pNIT1 promoter controlling GAG-POL expression is active (nitrate as nitrogen source), GAG-POL and the cargo are transcribed as a single mRNA allowing the intron to be spliced. However, the resistance gene is still untranslated because it is not in the correct orientation. 
3 – If the transposition works and the resistance gene (which has undergone splicing and reverse transcription) is reinserted in the genome, the pPSAD promoter will allow constitutive expression of the paromomycine resistance, now without intron and thus giving a fully functional protein. </P><P>To check if the transposition is working well, we designed a cargo with the intron upstream from the paromomycin resistance CDS. Transposon-activated (through pNIT1) lines selected from the first screen based on hygromycin resistance will be grown in a medium supplemented with paromomycin. Thereby, the only colonies that should be able to grow would be the ones having undergone the transposition. We plan to sequence the locus of insertion and verify the integrity of the transgene.  </P><P>Once the proof of concept will be done on the retrotransposon, we want to engineer a simple metabolic pathway composed of two enzymes producing trehalose (otsA and otsB, see IV – trehalose and biosensor) instead of the paromomycin resistance in the cargo. This way, in vivo directed evolution can be performed on these enzymes.  </P><H2 id="4" class="title-h2">IV – Trehalose and its Biosensor</H2><P>To produce trehalose in Chlamydomas, we chose to use the otsA and otsB genes from E. coli. Indeed, they have already been described to be functional in land plants such as Arabidopsis thaliana (Schluepmann et al. 2003; Wingler et al. 2012). We designed them to be in a unique transcription unit, expressed as a cistron-like thanks to the 2A peptide from the foot-and-mouth-disease virus (FMDV) linking the 2 coding sequences. This way, they will be under the control of a unique promoter and co-expressed in the same amount (Wang et al. 2015). Upstream and downstream in the ORF, we added tags to easily detect the production of each protein.  </P><P>To be able to detect the production of trehalose and thus the functionality of this metabolic pathway, a trehalose biosensor was also adapted to C. reinhardtii. This biosensor is a modified GFP coupled with a protein that can bind specifically to the trehalose. As the protein binds to it, the GFP conformation is modified allowing it to be functional (Nadler et al. 2016). </P><DIV class="container" id="references"><H5 class="text-center">References </H5><UL class="list-unstyled text-left"><LI> [1] Crook, N., Abatemarco, J., Sun, J., Wagner, J.M., Schmitz, A., and Alper, H.S. (2016). In vivo continuous evolution of genes and pathways in yeast. Nat. Commun. 7..
</LI><LI> [2] Crozet, P., Navarro, F.J., Willmund, F., Mehrshahi, P., Bakowski, K., Lauersen, K.J., Pérez-Pérez, M.-E., Auroy, P., Gorchs Rovira, A., Sauret-Gueto, S., et al. (2018). Birth of a Photosynthetic Chassis: A MoClo Toolkit Enabling Synthetic Biology in the Microalga Chlamydomonas reinhardtii. ACS Synth. Biol.</LI><LI>  [3] Curcio, M.J., Lutz, S., Lesage, P., and Diderot, P. (2015). The Ty1 LTR-retrotransposon of budding yeast, Saccharomyces cerevisiae. Microbiol Spectr 3, 1–35.</LI><LI>  [4] Engler, C., Gruetzner, R., Kandzia, R., and Marillonnet, S. (2009). Golden gate shuffling: A one-pot DNA shuffling method based on type ils restriction enzymes. PLoS ONE 4.</LI><LI>  [5] Jinkerson, R.E., and Jonikas, M.C. (2015). Molecular techniques to interrogate and edit the Chlamydomonas nuclear genome. Plant J. 82, 393–412. </LI><LI>  [6]Kim, K.S., Kustu, S., and Inwood, W. (2006). Natural history of transposition in the green alga Chlamydomonas reinhardtii: Use of the AMT4 locus as an experimental system. Genetics 173, 2005–2019</LI><LI>  [7]Lachapelle, J., Bell, G., and Colegrave, N. (2015). Experimental adaptation to marine conditions by a freshwater alga: EXPERIMENTAL ADAPTATION TO MARINE CONDITIONS. Evolution 69, 2662–2675.</LI><LI>  [8]Lindauer, A., Fraser, D., Brüderlein, M., and Schmitt, R. (1993). Reverse transcriptase families and a copia-like retrotransposon, Osser, in the green alga Volvox carteri. FEBS Lett. 319, 261–266.</LI><LI>  [9]Merchant, S.S., Prochnik, S.E., Vallon, O., Harris, E.H., Karpowicz, S.J., Witman, G.B., Terry, A., Salamov, A., Fritz-Laylin, L.K., Maréchal-Drouard, L., et al. (2007). The Chlamydomonas genome reveals the evolution of key animal and plant functions. Science 318, 245–250.</LI><LI>  [10]Nadler, D.C., Morgan, S.A., Flamholz, A., Kortright, K.E., and Savage, D.F. (2016). Rapid construction of metabolite biosensors using domain-insertion profiling. Nat. Commun. 7, 1–11..</LI><LI>  [11]Pérez-Alegre, M., Dubus, A., and Fernández, E. (2005). REM1, a New Type of Long Terminal Repeat Retrotransposon in Chlamydomonas reinhardtii. Mol. Cell. Biol. 25, 10628–10638.</LI><LI>  [11]Piégu, B., Bire, S., Arensburger, P., and Bigot, Y. (2015). A survey of transposable element classification systems – A call for a fundamental update to meet the challenge of their diversity and complexity. Mol. Phylogenet. Evol. 86, 90–109.</LI><LI>  [12]Rasala, B.A., and Mayfield, S.P. (2011). The microalga Chlamydomonas reinhardtii as a platform for the production of human protein therapeutics. Bioeng. Bugs 2, 50–54.</LI><LI>  [13]Schluepmann, H., Pellny, T., van Dijken, A., Smeekens, S., and Paul, M. (2003). Trehalose 6-phosphate is indispensable for carbohydrate utilization and growth in Arabidopsis thaliana. Proc. Natl. Acad. Sci. 100, 6849–6854.</LI><LI>  [14]Scranton, M.A., Ostrand, J.T., Fields, F.J., and Mayfield, S.P. (2015). Chlamydomonas as a model for biofuels and bio-products production. Plant J. 82, 523–531.</LI><LI>  [15]Sormacheva, I.D., and Blinov, A.G. (2011). LTR retrotransposons in plants. Russ. J. Genet. Appl. Res. 1, 540–564</LI><LI>  [16]Wang, Y., Wang, F., Wang, R., Zhao, P., and Xia, Q. (2015). 2A self-cleaving peptide-based multi-gene expression system in the silkworm Bombyx mori. Sci. Rep. 5.</LI><LI>  [17]Weber, E., Engler, C., Gruetzner, R., Werner, S., and Marillonnet, S. (2011). A Modular Cloning System for Standardized Assembly of Multigene Constructs. PLOS ONE 6, e16765.</LI><LI>  [18]Wingler, A., Delatte, T.L., O’Hara, L.E., Primavesi, L.F., Jhurreea, D., Paul, M.J., and Schluepmann, H. (2012). Trehalose 6-Phosphate Is Required for the Onset of Leaf Senescence Associated with High Carbon Availability. PLANT Physiol. 158, 1241–1251.</LI></UL></DIV></ARTICLE></DIV></DIV><DIV class="container fixed-bottom backtotop" style="padding-bottom: 10px; padding-right:0px; position: fixed; left: 120px;"><DIV class="row text-right text-xs-right text-sm-right text-md-right  text-lg-right  "><DIV class="col-xs-4 col-sm-4 col-md-12 col-lg-12"><A class="y btn-sm" href="#backtotop" role="button" style=" color:white; border-radius: 10px; background-color:#22D66C; text-decoration:none; ">  Back to top</A></DIV></DIV></DIV><SECTION id="footer"><DIV class="container"><DIV class="row text-center text-xs-center text-sm-center text-md-right"><DIV class="col-xs-12 col-sm-3 col-md-3 col-lg-6"><H5 class="text-left" style="border: none;padding: 0px">iGEM Sorbonne U Paris</H5><P class="text-left" style="color:white; font-size:16px;"> 4 place Jussieu  75005 Paris France</P></DIV></DIV></DIV><DIV class="container"><P class="text-center" style="color: white; padding:20px 0 20px 0">Made in Paris with love © iGEM Sorbonne U Paris 2018</P></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>