<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Sorbonne_U_Paris_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Sorbonne U Paris/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="collapse navbar-collapse " id="navbarTogglerDemo02"><UL class=" navbar-nav "><LI class="nav-item dropdown" id="mobile-homepage"><A class="nav-link" href="https://2018.igem.org/Team:Sorbonne_U_Paris" style="color:white;"> Home page</A></LI><LI class="nav-item dropdown  "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white">  Project
        </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Description">Description</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Design">Design</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Model">Model</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Demonstrate">Demonstrate</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Improve">Improve</A></DIV></LI><LI class="nav-item dropdown  "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Biology</A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Experiments">Experiments</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Results">Results</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Parts">Parts</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/InterLab">Interlab</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Safety">Safety</A></DIV></LI><LI class="nav-item dropdown  "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Human practices
        </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Human_Practices">Human Practices</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Integrated_Human_Practices">Integrated HP</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Public_Engagement">Education &amp; Engagement</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Meet_up">Meet up</A></DIV></LI><LI class="nav-item dropdown "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Notebook
       </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Notebook">Notebook</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Protocols">Protocols</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Brainstorming">Brainstorming</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Team </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Team">Members</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Collaborations">Collaborations</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Attributions">Attributions</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Sponsors">Sponsors</A></DIV></LI><LI class="nav-item dropdown  "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Awards</A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://igem.org/2018_Judging_Form?team=Sorbonne_U_Paris">Judging form</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Medals">Medals</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Giant_Jamboree">Giant Jamboree</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Sitemap" style="color:white;"> Site map</A></LI></UL></DIV><DIV class="global"><DIV class="container"><ARTICLE class="col-md-11"><H1>Laboratory Notebook</H1><P class="introduction-text">A laboratory notebook is our real time record of our expriments. It porvides a detailled record of exactly what we do in the laboratory to obtain our experimental results. It documents our hypotheses, experiments and initial analysis or interpretation of our experiments. Moreover, we also mentionned the name of exprimenters and the place of our experiments. The protocols that we follow  are available on protocols page: <A class="btn-sm" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Protocols" role="button" style=" color:white; border-radius: 10px; background-color: #22D66C; text-decoration:none;"> Protocols </A></P></ARTICLE></DIV><H2 class="text-center" id="h2-calendar"> Calendars 2018</H2><DIV class="container"><DIV class="row text-center text-xs-center text-sm-center text-md-center align-items-center justify-content-center"><DIV class="col-xs-5 col-sm-5 col-md-5 col-lg-2"><TABLE><THEAD><TR><TH colspan="7">June</TH></TR></THEAD><TBODY><TR><TD><B>M</B></TD><TD><B>T</B></TD><TD><B>W</B></TD><TD><B>T</B></TD><TD><B>F</B></TD><TD><B>S</B></TD><TD><B>S</B></TD></TR><TR><TD>1</TD><TD>2</TD><TD>3</TD></TR><TR><TD>4</TD><TD>5</TD><TD>6</TD><TD>7</TD><TD>8</TD><TD>9</TD><TD>10</TD></TR><TR><TD>11</TD><TD class="date"><A href="#jun12" class="pointer" title="Tuesday 12th June">12</A></TD><TD class="date"><A href="#jun13" class="pointer" title="Wednesday 13th June">13</A></TD><TD class="date"><A href="#jun14" class="pointer" title="Thurday 14th June">14</A></TD><TD>15</TD><TD>16</TD><TD>17</TD></TR><TR><TD>18</TD><TD>19</TD><TD class="date"><A href="#jun20" class="pointer" title="Wednesday 20th June">20</A></TD><TD>21</TD><TD>22</TD><TD>23</TD><TD>24</TD></TR><TR><TD>25</TD><TD>26</TD><TD class="date"><A href="#jun27" class="pointer" title="Wednesday 27th June">27</A></TD><TD>28</TD><TD class="date"><A href="#jun29" class="pointer" title="Friday 29th June">29</A></TD><TD>30</TD><TD id="j1">1</TD></TR></TBODY></TABLE></DIV><DIV class="col-xs-5 col-sm-5 col-md-5 col-lg-2"><TABLE><THEAD><TR><TH colspan="7">July</TH></TR></THEAD><TBODY><TR><TD><B>M</B></TD><TD><B>T</B></TD><TD><B>W</B></TD><TD><B>T</B></TD><TD><B>F</B></TD><TD><B>S</B></TD><TD><B>S</B></TD></TR><TR><TD class="date"><A href="#jul2" class="pointer" title="Monday 2nd July">2</A></TD><TD>3</TD><TD>4</TD><TD class="date"><A href="#jul5" class="pointer" title="Thurday 5th July">5</A></TD><TD>6</TD><TD>7</TD><TD>8</TD></TR><TR><TD class="date"><A href="#jul9" class="pointer" title="Monday 9th July">9</A></TD><TD class="date"><A href="#jul10" class="pointer" title="Tuesday 10th July">10</A></TD><TD>11</TD><TD>12</TD><TD class="date"><A href="#jul13" class="pointer" title="Friday 13th Jult">13</A></TD><TD>14</TD><TD>15</TD></TR><TR><TD class="date"><A href="#jul16" class="pointer" title="Monday 16th July">16</A></TD><TD class="date"><A href="#jul16" class="pointer" title="Tuesday 17th July">17</A></TD><TD>18</TD><TD class="date"><A href="#jul19" class="pointer" title="Thurday 19th July">19</A></TD><TD>20</TD><TD>21</TD><TD>22</TD></TR><TR><TD class="date"><A href="#jul23" class="pointer" title="Monday 23th July">23</A></TD><TD class="date"><A href="#jul24" class="pointer" title="Tuesday 24th July">24</A></TD><TD class="date"><A href="#jul25" class="pointer" title="Wedneday 25th July">25</A></TD><TD class="date"><A href="#jul26" class="pointer" title="Thurday 26th July"> 26</A></TD><TD class="date"><A href="#jul27" class="pointer" title="Friday 27th July">27</A></TD><TD>28</TD><TD>29</TD></TR><TR><TD>30</TD><TD class="date"><A href="#jul31" class="pointer" title="Tuesday 31th July">31</A></TD></TR></TBODY></TABLE></DIV><DIV class="col-xs-5 col-sm-5 col-md-5 col-lg-2"><TABLE><THEAD><TR><TH colspan="7">August</TH></TR></THEAD><TBODY><TR><TD><B>M</B></TD><TD><B>T</B></TD><TD><B>W</B></TD><TD><B>T</B></TD><TD><B>F</B></TD><TD><B>S</B></TD><TD><B>S</B></TD></TR><TR><TD class="date"><A href="#a1" class="pointer" title="Wednesday 1st  August">1</A></TD><TD class="date"><A href="#a2" class="pointer" title="Thurday 2nd August">2</A></TD><TD class="date"><A href="#a3" class="pointer" title="Friday 3th August">3</A></TD><TD>4</TD><TD>5</TD></TR><TR><TD class="date"><A href="#a6" class="pointer" title="Monday 6th August">6</A></TD><TD class="date"><A href="#a7" class="pointer" title="Tuesday 7th August">7</A></TD><TD>8</TD><TD>9</TD><TD>10</TD><TD>11</TD><TD>12</TD></TR><TR><TD>13</TD><TD>14</TD><TD>15</TD><TD>16</TD><TD>17</TD><TD>18</TD><TD>19</TD></TR><TR><TD>20</TD><TD class="date"><A href="#a21" class="pointer" title="Tuesday 21th August">21</A></TD><TD class="date"><A href="#a22" class="pointer" title="Tuesday 22th August">22</A></TD><TD>23</TD><TD>24</TD><TD>25</TD><TD>26</TD></TR><TR><TD>27</TD><TD class="date"><A href="#a28" class="pointer" title="Tuesday 28th August">28</A></TD><TD class="date"><A href="#a29" class="pointer" title="Tuesday 29th August">29</A></TD><TD class="date"><A href="#a30" class="pointer" title="Tuesday 30th August">30</A></TD><TD>31</TD></TR></TBODY></TABLE></DIV><DIV class="col-xs-5 col-sm-5 col-md-5 col-lg-2"><TABLE><THEAD><TR><TH colspan="7">September</TH></TR></THEAD><TBODY><TR><TD><B>M</B></TD><TD><B>T</B></TD><TD><B>W</B></TD><TD><B>T</B></TD><TD><B>F</B></TD><TD><B>S</B></TD><TD><B>S</B></TD></TR><TR><TD>1</TD><TD>2</TD></TR><TR><TD class="date"><A href="#s3" class="pointer" title="Monday 3th September">3</A></TD><TD class="date"><A href="#s4" class="pointer" title="Tuesday 4th September">4</A></TD><TD class="date"><A href="#s5" class="pointer" title="Wednesday 5th September">5</A></TD><TD class="date"><A href="#s6" class="pointer" title="Thurday 6th September">6</A></TD><TD class="date"><A href="#s7" class="pointer" title="Fridat 7th September">7</A></TD><TD>8</TD><TD>9</TD></TR><TR><TD class="date"><A href="#s10" class="pointer" title="Monday 10th September">10</A></TD><TD class="date"><A href="#s11" class="pointer" title="Tuesday 11th September">11</A></TD><TD class="date"><A href="#s12" class="pointer" title="Wednesday 12th September">12</A></TD><TD class="date"><A href="#s13" class="pointer" title="Monday 13th September">13</A></TD><TD>14</TD><TD>15</TD><TD>16</TD></TR><TR><TD class="date"><A href="#s17" class="pointer" title="Monday 17th September">17</A></TD><TD>18</TD><TD>19</TD><TD>20</TD><TD>21</TD><TD>22</TD><TD>23</TD></TR><TR><TD>24</TD><TD>25</TD><TD>26</TD><TD>27</TD><TD>28</TD><TD>29</TD><TD>30</TD></TR></TBODY></TABLE></DIV><DIV class="col-xs-5 col-sm-5 col-md-5 col-lg-2"><TABLE><THEAD><TR><TH colspan="7">October</TH></TR></THEAD><TBODY><TR><TD><B>M</B></TD><TD><B>T</B></TD><TD><B>W</B></TD><TD><B>T</B></TD><TD><B>F</B></TD><TD><B>S</B></TD><TD><B>S</B></TD></TR><TR><TD>1</TD><TD>2</TD><TD>3</TD><TD>4</TD><TD>5</TD><TD>6</TD><TD>7</TD></TR><TR><TD>8</TD><TD>9</TD><TD>10</TD><TD>11</TD><TD>12</TD><TD>13</TD><TD>14</TD></TR><TR><TD>15</TD><TD>16</TD><TD>17</TD><TD>18</TD><TD>19</TD><TD>20</TD><TD>21</TD></TR><TR><TD>22</TD><TD>23</TD><TD>24</TD><TD>25</TD><TD>26</TD><TD>27</TD><TD>28</TD></TR><TR><TD>29</TD><TD>30</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="d-flex justify-content-center"><H2 class="text-center" style="background:#e6e6e6;  width: 300px; border-radius: 10px;border-bottom: none;"> June</H2></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-left text-sm-left text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="jun12"> Tuesday 12th June 2018</H2><H4 text-xs-left="" text-sm-left="">PCR of rubisco intron, paromomycin CDS and pSAD promoter </H4><H6>Institut de biologie physico-chimique (IBPC) </H6><DIV class="d-flex justify-content-xs-start justify-content-sm-start  justify-content-md-start justify-content-lg-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie
</DIV><DIV class="chip">
 Jimmy
</DIV></DIV></DIV><DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class=" text-center tablenotebook"><THEAD><TR><TH class="th-notebook">Quantity in µL</TH><TH class="th-notebook"> Mix*8</TH></TR></THEAD><TBODY><TR><TD class="td-notebook">Plasmid DNA</TD><TD class="td-notebook">1</TD></TR><TR><TD class="td-notebook">Phusion enzyme</TD><TD class="td-notebook">0.25</TD><TD class="td-notebook">2
   </TD></TR><TR><TD class="td-notebook">HF buffer (5X)</TD><TD class="td-notebook">5</TD><TD class="td-notebook">40</TD></TR><TR><TD class="td-notebook">dNTPs (10µM)</TD><TD class="td-notebook">0.5</TD><TD class="td-notebook">4</TD></TR><TR><TD class="td-notebook">Fw Primer (10µM)</TD><TD class="td-notebook">1.25</TD></TR><TR><TD class="td-notebook"> Rv Primer (10µM) </TD><TD class="td-notebook">1.25</TD></TR><TR><TD class="td-notebook">H20</TD><TD class="td-notebook">15.75</TD><TD class="td-notebook">126</TD></TR></TBODY></TABLE></DIV></DIV><P clas="text-notebook"> 21.5µL of mix by tube + 1µL of DNA + 2*1.25µL of primer.</P><P>pSAD promoter : primers Ig3/Ig4, template p0-07 : Expected length : 807 pb </P><P>Paromomycin CDS : primers Ig11/Ig12, template p0-12: Expected length : 1652 pb </P><P>Rubisco intron : primers Ig5/Ig6, template p0-02: Expected length : 177 pb </P><OL><LI>negative control : puc 19 + Ig5/Ig6  </LI><LI>Rubisco intron : p0-02 + Ig5/Ig6 </LI><LI> negative control : puc 19 + Ig3/Ig4 </LI><LI> pSAD promoter : p0-07 + Ig3/Ig4</LI><LI>negative control : puc 19 + Ig11/Ig12 </LI><LI> paromomycin CDS : p0-12 + Ig11/Ig12</LI><LI>negative control : H20 + Ig5/Ig6 </LI><LI> negative control : H20 + Ig11/Ig12</LI></OL><P><SPAN style="font-weight:bold;">Cycle </SPAN> : 98°C, 35*(98°C - 1min, 50°C - 2min, 72°C - 3min), 72°C - 3min, 12°C - hold </P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-left text-sm-left text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="jun13"> Wednesday 13th June</H2><H4> Migration and purification of PCR product  “paromomycin”, “pSAD”, “rubisco intron”  </H4><H6>Institut de biologie physico-chimique (IBPC) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
Aurélie
</DIV><DIV class="chip">
 Soukeyna
</DIV></DIV></DIV><P class="text-notebook">+ 5µL of loading buffer 6X in PCR product 
15 µL/well, 1% agarose gel 
Migration : 100V, 20 min 
</P><DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class=" tablenotebook text-center"><THEAD><TR><TH class="th-notebook">Position</TH><TH class="th-notebook">1</TH><TH class="th-notebook">2</TH><TH class="th-notebook">3</TH><TH class="th-notebook">4</TH><TH class="th-notebook">5</TH><TH class="th-notebook">6</TH><TH class="th-notebook">7</TH><TH class="th-notebook">8</TH><TH class="th-notebook">9</TH></TR></THEAD><TBODY><TR><TD class="td-notebook">Content</TD><TD class="td-notebook">Ladder</TD><TD class="td-notebook">puc19 Ig5/Ig6 rubisco</TD><TD class="td-notebook">p0-2 Ig5/Ig6   rubisco</TD><TD class="td-notebook">puc19 Ig3/Ig4  pSAD</TD><TD class="td-notebook">p0-7 Ig3/Ig4 pSAD</TD><TD class="td-notebook">puc19 Ig11/Ig12paromycine R</TD><TD class="td-notebook">p0-12 Ig11/Ig12 paromycine R</TD><TD class="td-notebook">H2O Ig5/Ig6</TD><TD class="td-notebook">H2O Ig11/Ig12</TD></TR><TR><TD class="td-notebook">Expected lenght (pb)</TD><TD class="td-notebook">0</TD><TD class="td-notebook">177</TD><TD class="td-notebook">0</TD><TD class="td-notebook">807</TD><TD class="td-notebook">0</TD><TD class="td-notebook">1 652</TD><TD class="td-notebook">0</TD><TD class="td-notebook">0</TD></TR></TBODY></TABLE></DIV></DIV><H5>Gel analysis </H5><A class="pointer" title="Click on the image for a larger view " data-toggle="modal" data-target=".Gel-180613"><DIV class="container-figurecaption"><DIV class="content-figurecaption"><P><SPAN style="font-weight: bold;">Figure 1 : </SPAN>Gel electrophoresis of PCR products  rubisco (2&amp;3) intron,pSAD (4&amp;5), paromomycin(6&amp;7)   </P></DIV></DIV></A>Rubisco and pSAD : Ok for rubisco intron and pSAD, visible bands 
Paromycin: the main band does not have the expected length, the band at 1.6 kb is still purifed
Cutting of the bands corresponding to the expected bands (photobenchling)<H5> DNA purification </H5>
            Kit used: Nucleospin Gel 2 PCR Clean up- gel extraction (NT1= 800µl)
            Elution in 15µl , mesure of the DNA extracts concentration  using a nanodrop<DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class="tablenotebook text-center"><THEAD><TR><TH class="th-notebook">Fragment</TH><TH class="th-notebook">Notation</TH><TH class="th-notebook">Measured concentration (ng/µL)</TH></TR></THEAD><TBODY><TR><TD class="td-notebook">Rubisco intron</TD><TD class="td-notebook">i0- IGEM1</TD><TD class="td-notebook">26,9</TD></TR><TR><TD class="td-notebook">pSAD</TD><TD class="td-notebook"> i0-IGEM3</TD><TD class="td-notebook">19,3</TD></TR><TR><TD class="td-notebook">paromycin</TD><TD class="td-notebook">i0-ApHVIII</TD><TD class="td-notebook">7,8</TD></TR></TBODY></TABLE></DIV></DIV><P><SPAN style="font-weight: bold;">Note</SPAN> : We should re-do the PCR of “ paromycine” </P><H5> Launch of  2 bacterial cultures  overnight (4ml with spectinomycine)</H5><P> - p0universal = pAGM9121 
- p0B3 pICH41258
</P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="jun14"> Wednesday 14th June</H2><H4> Plasmid purification &amp; Reaction of ligation/digestion  </H4><H6>Institut de biologie physico-chimique (IBPC) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
Victor
</DIV><DIV class="chip">
 Aurélie
</DIV></DIV><DIV class="chip-icone"><DIV class="chip">
Soukeyna
</DIV></DIV></DIV><P class="text-notebook"><SPAN style="font-weight:bold">Aim</SPAN> : Extraction of plasmides from <A href="#jun13" style="text-decoration: none;">yesterday’s cultures </A> and reactions of digestion/ligation p0universal/ pSAD and p0-B3/intron rubisco.</P><H5>Plasmid purification </H5><P>
(p0- B3 = p0B3- pICH41258)</P><P class="text-notebook">  Purification of p0universel and p0- B3 plasmides  (kit used: Nucleopsin Plasmid) : Elution volume= 40µL</P><P class="text-notebook">Measure of DNA extracts concentration using Nanodrop (2 extracts per plasmid)
- p0-universel= 71,5 ng/µl ,  64,9ng/µl 
- p0-B3= 92,8ng/µl, 90,7ng/µL
</P><H5> Reaction of ligation/digestion </H5><P class="text-notebook"><U>Mix DNA</U></P><DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class="tablenotebook text-center"><THEAD><TR><TH class="th-notebook">Tube number</TH><TH class="th-notebook">2</TH><TH class="th-notebook">6</TH><TH class="th-notebook">1</TH><TH class="th-notebook">5</TH></TR></THEAD><TBODY><TR><TD class="td-notebook">µl plasmid</TD><TD class="td-notebook">0,68</TD><TD class="td-notebook">0,68</TD><TD class="td-notebook">0,97</TD><TD class="td-notebook">0,97</TD></TR><TR><TD class="td-notebook">µl insert </TD><TD class="td-notebook"> 2,733</TD><TD class="td-notebook">1,98</TD></TR><TR><TD class="td-notebook">µl H2O</TD><TD class="td-notebook">10,56</TD><TD class="td-notebook">13,32</TD><TD class="td-notebook">11,06</TD><TD class="td-notebook">13,07</TD></TR><TR><TD class="td-notebook">pSAD</TD><TD class="td-notebook">pSAD</TD><TD class="td-notebook">Rubisco</TD><TD class="td-notebook">Rubisco</TD></TR></TBODY></TABLE></DIV></DIV><P class="text-notebook"><SPAN style="font-weight:bold;">2</SPAN>: plasmide uni+ pSAD+ enzymes <SPAN style="font-weight:bold;">6</SPAN>: plasmide uni + enzymes <SPAN style="font-weight:bold;">1</SPAN>: plasmides B3+ introns enzymes <SPAN style="font-weight:bold;">5</SPAN>: plasmides B3+ enzymes
</P><P class="text-notebook"><U>Mid enzyme</U> (6µl/tube) number of ligations : 4.5 </P><DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class="tablenotebook text-center"><THEAD><TR><TH class="th-notebook">Reaction mix</TH><TH class="th-notebook">µL</TH><TH class="th-notebook">master mix</TH></TR></THEAD><TBODY><TR><TD class="td-notebook"> BS/HF (µl)</TD><TD class="td-notebook">1</TD><TD class="td-notebook">4,5</TD></TR><TR><TD class="td-notebook">T4 ligase(µl)</TD><TD class="td-notebook"> 1</TD><TD class="td-notebook">4,5</TD></TR><TR><TD class="td-notebook">Buffer NEB CS (µl)</TD><TD class="td-notebook">2</TD><TD class="td-notebook">9</TD></TR><TR><TD class="td-notebook"> ATP 10mM(µl)</TD><TD class="td-notebook">2</TD><TD class="td-notebook">9</TD></TR></TBODY></TABLE></DIV></DIV><P>Cycle: 
- 37°C- 10min * 3 
- 16°C- 10min *3 
- 37°C- 10min 
- 65°C- 20min 
- 16°C- ∞ 
</P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="jun20"> Wednesday 20th June</H2><H4> PCR of the “ paromycine” fragment   Purification of “p0- IGEM1” and “p0- IGEM3” plasmides  </H4><H6> Muséum national d'histoire naturelle (MNHN) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie
</DIV><DIV class="chip">
 Ursula
</DIV></DIV></DIV><H5> PCR reactions</H5><DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">Product</TH><TH class="th-notebook">Quantity (µL) </TH><TH class="th-notebook"> Quantity (µL)</TH></TR></THEAD><TBODY><TR><TD class="td-notebook">Plasmide de p0-12</TD><TD class="td-notebook">1</TD><TD class="td-notebook">↗</TD></TR><TR><TD class="td-notebook">Enzyme Phusion</TD><TD class="td-notebook">0.25</TD><TD class="td-notebook">2
   </TD></TR><TR><TD class="td-notebook">HF buffer (5X)</TD><TD class="td-notebook">5</TD><TD class="td-notebook">40</TD></TR><TR><TD class="td-notebook">dNTPs (10mM)</TD><TD class="td-notebook">0.5</TD><TD class="td-notebook">4</TD></TR><TR><TD class="td-notebook">Primer F Ig11 (10µM)</TD><TD class="td-notebook">1.25</TD><TD class="td-notebook">10 </TD></TR><TR><TD class="td-notebook"> Primer F Ig12 (10µM) </TD><TD class="td-notebook">1.25</TD><TD class="td-notebook">10</TD></TR><TR><TD class="td-notebook">DNA</TD><TD class="td-notebook">0 </TD><TD class="td-notebook">1</TD></TR><TR><TD class="td-notebook">H20</TD><TD class="td-notebook">15.75</TD><TD class="td-notebook">126</TD></TR><TR><TD class="td-notebook">Total quantity</TD><TD class="td-notebook">25 </TD><TD class="td-notebook">25</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">DNA (template) </TH><TH class="th-notebook"> Hybridation temperature</TH></TR></THEAD><TBODY><TR><TD class="td-notebook">Tube 1</TD><TD class="td-notebook">p⦽-12</TD><TD class="td-notebook">50°C</TD></TR><TR><TD class="td-notebook">Tube 2</TD><TD class="td-notebook">p⦽-12</TD><TD class="td-notebook">52°C</TD></TR><TR><TD class="td-notebook">Tube 3</TD><TD class="td-notebook">p⦽-12</TD><TD class="td-notebook">53°C</TD></TR><TR><TD class="td-notebook">Tube 4</TD><TD class="td-notebook">p⦽-12</TD><TD class="td-notebook">54°C</TD></TR><TR><TD class="td-notebook">Tube 5</TD><TD class="td-notebook">p⦽-12</TD><TD class="td-notebook">55°C </TD></TR><TR><TD class="td-notebook"> Tube 6 </TD><TD class="td-notebook">p⦽-12</TD><TD class="td-notebook">157°C</TD></TR><TR><TD class="td-notebook">Controle E </TD><TD class="td-notebook">p⦽-19 </TD><TD class="td-notebook">50°C</TD></TR></TBODY></TABLE></DIV></DIV><P class="text-notebook">Cycle: 
98°C 
98°C     - 1min    *30
50-57°C- 2 min   *30 
72°C     - 3 min   *30
72°C     - 3 min 
12°C     - ∞
</P><P class="text-notebook">- Addition of  5µl loading buffer 6* (5*µL in 25µl)
- Deposit of 30µl of extract on 1% agarose gel
- Migration at 90V for 40 min</P><P class="text-notebook"> Missing: PCR photo 
- PCR ok with paromycine at 800pb different from the expected 1657 pb expected before 
- Insert Paro stocked at -20°C </P><H5> Purification of “p⦽-IGEM1” and “p⦽-IGEM3” plasmids </H5><P class="text-notebook">Kit used: Gene JET Plasmid-MiniprepKit- Thermo Scientific 
Centrifugation 13000 RPM 
Digestion to check the size of the inserts (using BsaI) </P><DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">Plasmid(µl)</TH><TH class="th-notebook">5 </TH><TH class="th-notebook">5</TH></TR></THEAD><TBODY><TR><TD class="td-notebook">Cut Smart Buffer (10X) (µl)</TD><TD class="td-notebook">11</TD><TD class="td-notebook">9C</TD></TR><TR><TD class="td-notebook">Enzyme Bsa  (µl)</TD><TD class="td-notebook">0.5</TD><TD class="td-notebook">4.5</TD></TR><TR><TD class="td-notebook">Water (µl)</TD><TD class="td-notebook">3.5</TD><TD class="td-notebook">31.5</TD></TR></TBODY></TABLE></DIV></DIV><P class="text-notebook"> Addition of 2µl of loading buffer 6* (2µl in 10µl)
Deposit of 12µl on 1% agarose gel 
Migration at 90V, for 40min </P><A class="pointer" title="Click on the image for a larger view " data-toggle="modal" data-target=".Gel-180620"><DIV class="container-figurecaption"><DIV class="content-figurecaption"><P><SPAN style="font-weight: bold;">Figure : </SPAN>Gel electrophoresis  : Purification of “p⦽-IGEM1” and “p⦽-IGEM3” plasmids </P></DIV></DIV></A><P ckass="text-notebook">We keep : 1a= rubisco intron= p⦽-IGEM1(1b too is conserved)
                 2a= pSAD- p⦽-IGEM3
Stock in glycerol￫ -30°C (DH5𝛂)mycologue <SPAN style="font-weight:bold;">Note</SPAN>: Transplanting of the microalguae strain that was previously blocked by FedEX
</P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="jun27"> Wednesday 27th June</H2><H4>Bacterial culture </H4><H6> Muséum national d'histoire naturelle (MNHN) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie
</DIV><DIV class="chip">
 Ursula
</DIV></DIV></DIV><P class="text-notebook">Launch of 2* 3.5 mL of bacterial culture → +spectromycine  of pICH4 1264 which contains the destination plasmid of the “paromicycn fragment)
</P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="jun29"> Wednesday 29th June</H2><H4> Purification of pICH41264 plasmid using the Nucleospin Plasmid K  Plasmid concentration measures using nanodrop  </H4><H6> Muséum national d'histoire naturelle (MNHN) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie
</DIV><DIV class="chip">
 Ursula
</DIV></DIV></DIV><DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class="text-center tablenotebook"><TBODY><TR><TD class="td-notebook">Culture 1</TD><TD class="td-notebook">26,7ng/µl</TD><TD class="td-notebook">1.92</TD><TD class="td-notebook">1.73</TD></TR><TR><TD class="td-notebook">Culture  2 </TD><TD class="td-notebook">25,7ng/µl</TD><TD class="td-notebook">1.92</TD><TD class="td-notebook">2.02</TD></TR></TBODY></TABLE></DIV></DIV><P class="text-notebook">The pICH41264 extracts are stocked at -20°C</P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="d-flex justify-content-center"><H2 class="text-center" style="background: #e6e6e6; width: 300px; border-radius: 10px; border-bottom: none;"> july</H2></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-left text-sm-left text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="jul2"> Monday 2 July</H2><H4 text-xs-left="" text-sm-left=""> Transformation of 𝝱-10 bacterias with  p⦽ pICH4 1264 + insert paroR Production of p⦽ plasmids for 3’LTR and 5’LTR parts. </H4><H6>Institut de biologie physico-chimique (IBPC) </H6><DIV class="d-flex justify-content-xs-start justify-content-sm-start  justify-content-md-start justify-content-lg-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie
</DIV><DIV class="chip">
 Ursula
</DIV><DIV class="chip">
 Dounia
</DIV></DIV></DIV><H5> Transformation of 𝝱-10 bacterias with  p⦽ pICH4 1264 + insert paroR</H5><P class="text-notebook">-𝝱-10 bacteria : 30µl 
Ligation product (paro): 4µl (insert or negative control)
30min_ 4°C 
55 sec_ 42°C 
2 min_ 4°C 
+700µL of LB medium 
1h_ 37°C 
Staggering of 350 on LB+spec+Xgal dish 
37°C O/N </P><H5> Production of p⦽ plasmids for 3’LTR and 5’LTR parts. </H5><P clas="text-notebook">Resuspension of IDT1 3’LTR in 31,03µL 
Resuspension of IDT2 5’LTR in 27,56µL 
[C]= 100 fmol/µL </P>

Sub cloning of 2 white colonies of the day before. These colonies are supposed to contain the pICH4 1264 plasmid and the “PARO” insert. </DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="jul5"> Thurday 5 July</H2><H4> Transplanting of single clones from each subcloning of the 3rd of  july  Transformation of 𝝱-10 bacterias with  p⦽ pUniversal, containing either LTR5’ or LTR3’ as insert  </H4><H6>Institut de biologie physico-chimique (IBPC) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie
</DIV><DIV class="chip">
 Dounia
</DIV></DIV></DIV><P class="text-notebook">𝝱-10  Bacteria _ 30µL 
Ligation product_ 4µL (insert or control) 
      - 30 min_4°c 
     - 55 sec_ 42°C 
     - 2 min  _ 4°C 
  + 700µl of LB 
 Spreading of 250µl on a dish containing LB + SPEC+ Xgal at 37°C
</P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="jul9"> Thurday 9 July</H2><H4> Lauch of miniprep Verifcation of Paro1 and Paro 2 clones  Assembly of GAG-Pol   </H4><H6>Institut de biologie physico-chimique (IBPC) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie
</DIV><DIV class="chip">
Ursula
</DIV></DIV></DIV><H5> Lauch of miniprep of 4 clones of each LTD in media spec+ Xgal </H5><DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class="text-center tablenotebook"><TBODY><TR><TD class="td-notebook">LTR3’1</TD><TD class="td-notebook">LTR5’1</TD></TR><TR><TD class="td-notebook">LTR3’2 </TD><TD class="td-notebook">2LTR5’2</TD></TR><TR><TD class="td-notebook">LTR3’3 </TD><TD class="td-notebook">LTR5’3</TD></TR><TR><TD class="td-notebook">LTR3’4 </TD><TD class="td-notebook">LTR5’4</TD></TR></TBODY></TABLE></DIV></DIV><H5>Verifcation of Paro1 and Paro 2 clones  </H5><P class="text-notebook">Purification of plasmids (methode for digestion of verification)*? , using Nucleospin Plasmid Kit</P><H5> Assembly of GAG-Pol ⇨ 6 parts</H5><P class="text-notebook">1) Dilution at 10fmol/µl of each fragment received </P><DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">Fragment received</TH><TH class="th-notebook">volume of H2O mQ to add (µl) </TH></TR></THEAD><TBODY><TR><TD class="td-notebook">IDT3_L0_B1</TD><TD class="td-notebook">1578/10 ⇒ 157,8</TD></TR><TR><TD class="td-notebook">IDT4_L0_B2 </TD><TD class="td-notebook">3216/10⇒321,6</TD></TR><TR><TD class="td-notebook">IDT6_L0_B3 </TD><TD class="td-notebook">900/10 ⇒ 90</TD></TR><TR><TD class="td-notebook">IDT4_L0_A3</TD><TD class="td-notebook">1098/10⇒109,8</TD></TR><TR><TD class="td-notebook">IDT4_L0_A1 </TD><TD class="td-notebook">2078/10⇒207,8</TD></TR><TR><TD class="td-notebook">IDT4_L0_A2</TD><TD class="td-notebook">1171/10⇒ 117,1</TD></TR></TBODY></TABLE></DIV></DIV><P class="text-notebook">2) Reaction of digestion/ligation </P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="jul10"> Thurday 10th July</H2><H4> Transformation of 𝝱10 with digestion/ligation products  of GAG POL  (<A href="#jul9">experiences of 07/09/18</A>) </H4><H6>Institut de biologie physico-chimique (IBPC) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie</DIV><DIV class="chip">
Ursula
</DIV></DIV></DIV><P class="text-notebook">- 30µl of  𝝱+ 4µl of ligation product 
- 30min at 4°C 
- 55 sec at 42°C 
- 2 min at 4°C 
- + 700µL of LB 
Spreading of 250µL on 2  petri dishes  Xgal+ Spec : control/ Gagpol
Incubation at 30°C over night</P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="jul13"> Friday 13th July</H2><H4> Launch of minipreps for 6 clones of GAG-Pol </H4><H6>Institut de biologie physico-chimique (IBPC) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Ursula
</DIV><DIV class="chip">
 Asmaa
</DIV></DIV></DIV><H5> Launch of minipreps for 6 clones of GAG-Pol in Xgal+spec media GAG_Pol 1, GAG_Pol 2, GAG_Pol 3, GAG_Pol 4 , GAG_Pol 5, GAG_Pol 6</H5><P class="text-notebook">Purification of PAGM9121 plasmide using Nucleopsin Plasmid Kit</P><H5>Measure of DNA concentration at Nanodrop  </H5><DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">Fragment</TH><TH class="th-notebook">Concentration (ng/µl) </TH></TR></THEAD><TBODY><TR><TD class="td-notebook">LTR3’1</TD><TD class="td-notebook">42,2</TD><TD class="td-notebook">1.97</TD><TD class="td-notebook">1.85</TD></TR><TR><TD class="td-notebook">LTR3’2</TD><TD class="td-notebook">38,4</TD><TD class="td-notebook">1.98</TD><TD class="td-notebook">1.86</TD></TR><TR><TD class="td-notebook">LTR3’3 </TD><TD class="td-notebook">37,7</TD><TD class="td-notebook">1.95</TD><TD class="td-notebook">1.64</TD></TR><TR><TD class="td-notebook">LTR3’4</TD><TD class="td-notebook">94,1</TD><TD class="td-notebook">1.91</TD><TD class="td-notebook">2.06</TD></TR><TR><TD class="td-notebook">LTR5’1 </TD><TD class="td-notebook">48,4</TD><TD class="td-notebook">1.95</TD><TD class="td-notebook">1.86</TD></TR><TR><TD class="td-notebook">LTR5’3</TD><TD class="td-notebook">120</TD><TD class="td-notebook">1.89</TD><TD class="td-notebook">1.94</TD></TR><TR><TD class="td-notebook">Paromycin 1</TD><TD class="td-notebook">55,8</TD><TD class="td-notebook">1.91</TD><TD class="td-notebook">1.89</TD></TR><TR><TD class="td-notebook">Paromycin 2</TD><TD class="td-notebook">49,1</TD><TD class="td-notebook">1.91</TD><TD class="td-notebook">1.87</TD></TR><TR><TD class="td-notebook">GAG_Pol 1</TD><TD class="td-notebook">78,7</TD><TD class="td-notebook">1.89</TD><TD class="td-notebook">1.92</TD></TR><TR><TD class="td-notebook">GAG_Pol 2</TD><TD class="td-notebook">25,3</TD><TD class="td-notebook">1.97</TD><TD class="td-notebook">2.85</TD></TR><TR><TD class="td-notebook">GAG_Pol 3</TD><TD class="td-notebook">49,0</TD><TD class="td-notebook">1.94</TD><TD class="td-notebook">2.62</TD></TR><TR><TD class="td-notebook">GAG_Pol 4</TD><TD class="td-notebook">17,7</TD><TD class="td-notebook">2.00</TD><TD class="td-notebook">3.90</TD></TR><TR><TD class="td-notebook">GAG_Pol 5</TD><TD class="td-notebook">44,0</TD><TD class="td-notebook">1.97</TD><TD class="td-notebook">2.68</TD></TR><TR><TD class="td-notebook">GAG_Pol 6</TD><TD class="td-notebook">21,2</TD><TD class="td-notebook">2.01</TD><TD class="td-notebook">3.65</TD></TR></TBODY></TABLE></DIV></DIV><H5>Digestion to check the inserts size ( using BsaI) </H5><DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">Component</TH><TH class="th-notebook"> 10 µl reaction </TH></TR></THEAD><TBODY><TR><TD class="td-notebook">DNA</TD><TD class="td-notebook">200 ng (except for GAG pol 4 &amp; 6 : DNA=150ng)</TD></TR><TR><TD class="td-notebook">10X CutSmart Buffer</TD><TD class="td-notebook">1µl</TD></TR><TR><TD class="td-notebook">BsaI </TD><TD class="td-notebook"> 0.2µ</TD></TR><TR><TD class="td-notebook">Nuclease-free Water</TD><TD class="td-notebook">	to 50 µl</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="jul16"> Monday 16th July</H2><H4>  Migration of digestion reaction products </H4><H6>Institut de biologie physico-chimique (IBPC) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie
</DIV><DIV class="chip">
 Anaïs
</DIV></DIV></DIV><H5> Migration of digestion reaction products from the day before </H5><A class="pointer" title="Click on the image for a larger view " data-toggle="modal" data-target=".Migration-180716"><DIV class="container-figurecaption"><DIV class="content-figurecaption"><P><SPAN style="font-weight: bold;">Figure : </SPAN>Gel electrophoresis of digestion reaction products to check the inserts size  </P></DIV></DIV></A><P class="text-notebook">Agarose gel 1% 
2µL loading buffer 6X per 10µl sample + BET 
Migration 100V, 20 min 
</P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="jul17"> Tuesday 17th July</H2><H4> Arrangement in the box iGEM 2018  Preparation of all the inserts in order to send them to sequencing </H4><H6>Institut de biologie physico-chimique (IBPC) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
Aurélie
</DIV><DIV class="chip">
 Anaïs
</DIV></DIV></DIV><H5> Arrangement in the box iGEM 2018  </H5><H5>Preparation of all the inserts in order to send them to sequencing </H5><P class="text-notebook">[C] Nanodrop 
PSAD 2a: 107.9 ng/µL 
1.97 
3.81 
PSAD 2d: 150.6 ng/µl 
1.94 
3.12 
Intron Rb= 103 ng/µL 
1.99 
3.82 

[AMG 9121]: 71.5 ng/µl 
Samples sent to sequencing </P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="jul19"> Thursday 19th July</H2><H4> Analysis of sequencing results </H4><H6>Institut de biologie physico-chimique (IBPC) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie
</DIV><DIV class="chip">
 Anaïs
</DIV></DIV></DIV><DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">Fragment</TH><TH class="th-notebook">Comments </TH></TR></THEAD><TBODY><TR><TD class="td-notebook">Paromycine</TD><TD class="td-notebook">ok</TD></TR><TR><TD class="td-notebook">Gagpol (GG40 and GG41)  Rubisco intron 
pSAD
</TD><TD class="td-notebook">Absurd results 
Re-do the digestion reaction whith different  BsaI enzymes
</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="jul23"> Monday 23th July</H2><H4>Transformation to redo the stock of each plasmid</H4><H6>Institut de biologie physico-chimique (IBPC) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie
</DIV><DIV class="chip">
 Anaïs
</DIV></DIV></DIV><P class="text-notebook"> - 10µL of  𝝱10 bacterias 
- Gag Pol 5 : 1.5 µL 
            Gag Pol 3 : 0.5 µL
            Igem 2, 4 and 6: 2µL
            Igem 3, 2 : 2.5 µL
            ↳ plasmids to transfect 
- 30 min at 4°C 
- 55 sec at 42°C
- 2 min at 4°C 
+ 700 µl of LB 
 Spreading of 250µL on plates containing LB+ spec
</P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="jul24"> Thurday 24th July</H2><H4> Transplanting of a clamydomonas clone  Plasmid extraction from 𝝱 bacterias transformed  Concentration measurements </H4><H6>Institut de biologie physico-chimique (IBPC) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie
</DIV><DIV class="chip">
 Dounia
</DIV></DIV></DIV><H5>1. Transplanting of a clamydomonas clone in 3ml of LB+ Spec </H5><H5>2. Plasmid extraction from 𝝱 bacterias transformed <A href="#jul23">the day before</A> ( directly from dish).</H5><P class="text-notebook">Kit used = Nucleopsin Plasmid 
Elution volum= 40µL </P><H5>3. Concentration measurements using Nanodrop </H5><DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">Fragment</TH><TH class="th-notebook">Concentration(ng/µl) </TH></TR></THEAD><TBODY><TR><TD class="td-notebook">LTR3’2</TD><TD class="td-notebook">241.3</TD><TD class="td-notebook">1.90</TD><TD class="td-notebook">2.17</TD></TR><TR><TD class="td-notebook">ParoII</TD><TD class="td-notebook">196.0</TD><TD class="td-notebook">1.90</TD><TD class="td-notebook">2.09</TD></TR><TR><TD class="td-notebook">LTR5’</TD><TD class="td-notebook">166.2</TD><TD class="td-notebook">1.89</TD><TD class="td-notebook">2.15</TD></TR><TR><TD class="td-notebook">ParoI</TD><TD class="td-notebook">160.9</TD><TD class="td-notebook">1.91</TD><TD class="td-notebook">2.11</TD></TR><TR><TD class="td-notebook">LTR3’4</TD><TD class="td-notebook">232.8</TD><TD class="td-notebook">1.89</TD><TD class="td-notebook">2.21</TD></TR><TR><TD class="td-notebook">Gag5</TD><TD class="td-notebook">393.6</TD><TD class="td-notebook">1.91</TD><TD class="td-notebook">2.16</TD></TR><TR><TD class="td-notebook">Rubisco</TD><TD class="td-notebook">31.3</TD><TD class="td-notebook">2.04</TD><TD class="td-notebook">2.09</TD></TR><TR><TD class="td-notebook">pSAD</TD><TD class="td-notebook">133.8</TD><TD class="td-notebook">1.90</TD><TD class="td-notebook">2.12</TD></TR><TR><TD class="td-notebook">Gag3</TD><TD class="td-notebook">370.6</TD><TD class="td-notebook">1.91</TD><TD class="td-notebook">2.11</TD></TR></TBODY></TABLE></DIV></DIV><H5>4. Replanting of Chlamydomonas  </H5></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="jul25"> Wednesday 25th July</H2><H4> Plasmid extraction from bacteria transformed   Concentration measurement  Digestion </H4><H6>Institut de biologie physico-chimique (IBPC) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie
</DIV><DIV class="chip">
 Anais
</DIV></DIV></DIV><H5>Plasmid extraction from <A href="#jul24">bacteria transformed</A></H5><P class="text-notebook"> Put on liquid media the 24/07  kit used : Nucleopsin plasmid  
</P><H5>Concentration measurement using Nanodrop  </H5><DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">Fragment</TH><TH class="th-notebook">Concentration(ng/µl) </TH></TR></THEAD><TBODY><TR><TD class="td-notebook">Paro II</TD><TD class="td-notebook">110.4</TD><TD class="td-notebook">1.89</TD><TD class="td-notebook">2.21</TD></TR><TR><TD class="td-notebook">PSAD</TD><TD class="td-notebook">62.1</TD><TD class="td-notebook">1.89</TD><TD class="td-notebook">2.21</TD></TR><TR><TD class="td-notebook">GagPol 5</TD><TD class="td-notebook">231.6</TD><TD class="td-notebook">1.89</TD><TD class="td-notebook">2.21</TD></TR><TR><TD class="td-notebook">ParoI</TD><TD class="td-notebook">96.4</TD><TD class="td-notebook">1.89</TD><TD class="td-notebook">2.18</TD></TR><TR><TD class="td-notebook">LTR5’</TD><TD class="td-notebook">65.4</TD><TD class="td-notebook">1.90</TD><TD class="td-notebook">2.17</TD></TR><TR><TD class="td-notebook">LTR3’2</TD><TD class="td-notebook">102.3</TD><TD class="td-notebook">1.91</TD><TD class="td-notebook">2.26</TD></TR><TR><TD class="td-notebook">LTR3’4</TD><TD class="td-notebook">111.0</TD><TD class="td-notebook">1.88</TD><TD class="td-notebook">2.29</TD></TR><TR><TD class="td-notebook">Gag Pol3</TD><TD class="td-notebook">189.8</TD><TD class="td-notebook">1.90</TD><TD class="td-notebook">2.21</TD></TR><TR><TD class="td-notebook">Intron Rubisco</TD><TD class="td-notebook">73.9</TD><TD class="td-notebook">1.89</TD><TD class="td-notebook">2.19</TD></TR></TBODY></TABLE></DIV></DIV><H5>Digestion</H5><DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">Intron </TH><TH class="th-notebook">PSAD</TH><TH class="th-notebook">LTR5’1</TH><TH class="th-notebook">LTR3’4 </TH><TH class="th-notebook"> LTR3’2</TH><TH class="th-notebook">Uni </TH><TH class="th-notebook">GagPol3 </TH><TH class="th-notebook">GagPol5 </TH></TR></THEAD><TBODY><TR><TD class="td-notebook">Enzyme</TD><TD class="td-notebook">0.1</TD><TD class="td-notebook">0.1</TD><TD class="td-notebook">0.1</TD><TD class="td-notebook">0.1</TD><TD class="td-notebook">0.1</TD><TD class="td-notebook">0.1</TD><TD class="td-notebook">0.2</TD><TD class="td-notebook">0.2</TD></TR><TR><TD class="td-notebook">Buffer</TD><TD class="td-notebook">1</TD><TD class="td-notebook">1</TD><TD class="td-notebook">1</TD><TD class="td-notebook">1</TD><TD class="td-notebook">1</TD><TD class="td-notebook">1</TD><TD class="td-notebook">1</TD><TD class="td-notebook">1</TD></TR><TR><TD class="td-notebook">ADN</TD><TD class="td-notebook">2</TD><TD class="td-notebook">2</TD><TD class="td-notebook">4</TD><TD class="td-notebook">2</TD><TD class="td-notebook">5</TD><TD class="td-notebook">3</TD><TD class="td-notebook">3</TD><TD class="td-notebook">3</TD></TR><TR><TD class="td-notebook">H2O</TD><TD class="td-notebook">6.8</TD><TD class="td-notebook">6.8</TD><TD class="td-notebook">4.9</TD><TD class="td-notebook">6.8</TD><TD class="td-notebook">3.9</TD><TD class="td-notebook">5.9</TD><TD class="td-notebook">5.9</TD><TD class="td-notebook">5.9</TD></TR><TR><TD class="td-notebook">BstEII  SaeI</TD><TD class="td-notebook">BstEI  StuI</TD><TD class="td-notebook">BstEII  BamHI</TD><TD class="td-notebook">BstEII  BamHI</TD><TD class="td-notebook">BstEII  BamHI</TD><TD class="td-notebook">BstEII  BamHI</TD><TD class="td-notebook">BstEII</TD><TD class="td-notebook">BstEII</TD></TR></TBODY></TABLE></DIV></DIV><P class="text-notebook">Digestion at 37°C/ 1H , conservation at -20°C overnight</P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="jul26"> Thurday 26th July</H2><H4> Migration of digestion samples  Digestion  Migration on gel electrophoresis </H4><H6> Muséum national d'histoire naturelle (MNHN) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie
</DIV><DIV class="chip">
 Anais
</DIV></DIV></DIV><H5>Migration of digestion samples from the day before 
 </H5><P class="text-notebook">1% agarose gel, deposit of 10µ+2µL of loading buffer 6X, ladder “2log”  Gel plan</P><DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">Rb intron</TH><TH class="th-notebook">pSAD </TH><TH class="th-notebook">LTR3’2</TH><TH class="th-notebook">LTR3’4</TH><TH class="th-notebook">LTR5’4 </TH><TH class="th-notebook"> 9121</TH><TH class="th-notebook">Gag3 </TH><TH class="th-notebook">Gag5 </TH></TR></THEAD><TBODY><TR><TD class="td-notebook">1 2 3</TD><TD class="td-notebook">1 2 3</TD><TD class="td-notebook">1 2 3</TD><TD class="td-notebook">1 2 3</TD><TD class="td-notebook">1 2 3</TD><TD class="td-notebook">1</TD><TD class="td-notebook">1 3</TD><TD class="td-notebook">1 3</TD></TR><TR><TD class="td-notebook">3 bands  1714 pb  620 pb  66 pb</TD><TD class="td-notebook">2 bands  1662 pb  1182 pb </TD><TD class="td-notebook" colspan="2">1 band 2000-3000pb  (2270 pb)</TD><TD class="td-notebook">1 band  2270 pb</TD><TD class="td-notebook">2 band  11831pb 828 pb</TD><TD class="td-notebook" colspan="2">2 bands  4242 pb  2260 pb</TD></TR></TBODY></TABLE></DIV></DIV><A class="pointer" title="Click on the image for a larger view " data-toggle="modal" data-target=".Migration-180726"><DIV class="container-figurecaption"><DIV class="content-figurecaption"><P><SPAN style="font-weight: bold;">Figure : </SPAN>Gel electrophoresis of digestion reaction products to check the inserts size  </P></DIV></DIV></A><P class="text-notebook">There seems to be no inserts</P><H5>Digestion </H5><P class="text-notebook">- NDE I, 1h, 37°C 
- Tampon 10X : 1µl 
- ADN : 3µL 
- Nde I : 0.2 µL 
- H2O : 5.8 µ </P><H5> Migration on gel</H5><DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">Rb Intron</TH><TH class="th-notebook">pSAD </TH><TH class="th-notebook">LTR3’2</TH><TH class="th-notebook">LTR3’4</TH><TH class="th-notebook">LTR5’1 </TH><TH class="th-notebook"> GagP3</TH><TH class="th-notebook">Gag P5 </TH><TH class="th-notebook">Paro I </TH><TH class="th-notebook">Paro II </TH></TR></THEAD><TBODY><TR><TD class="td-notebook">1 2 </TD><TD class="td-notebook">1 2 </TD><TD class="td-notebook">1 2 </TD><TD class="td-notebook">1 2 </TD><TD class="td-notebook">1 2 </TD><TD class="td-notebook">1</TD><TD class="td-notebook">1 </TD><TD class="td-notebook">1 2</TD><TD class="td-notebook">1 2</TD></TR></TBODY></TABLE></DIV></DIV><P class="text-notebook">Non informative, forgotten time of incubation </P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="jul27"> Friday 27th July</H2><H4> Enzymatic digestion reaction  </H4><H6> Muséum national d'histoire naturelle (MNHN) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
Aurélie
</DIV></DIV></DIV><H5>Enzymatic digestion reaction ( fast digest enzyme)</H5><DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">1 reaction </TH><TH class="th-notebook">mix *20</TH></TR></THEAD><TBODY><TR><TD class="td-notebook">Bcl Enzyme  </TD><TD class="td-notebook">0.2 µL </TD><TD class="td-notebook">4 µL </TD></TR><TR><TD class="td-notebook">Buffer FD (10X) </TD><TD class="td-notebook">1  µL </TD><TD class="td-notebook">20µL </TD></TR><TR><TD class="td-notebook">DNA  </TD><TD class="td-notebook">3  µL </TD><TD class="td-notebook">↗ </TD></TR><TR><TD class="td-notebook">H2O </TD><TD class="td-notebook">5.8 µL  </TD><TD class="td-notebook">116 µL </TD></TR></TBODY></TABLE></DIV></DIV><P class="text-notebook">↪7µl mix/ per tube 
 + 3µL  DNA 
+ 0.2 µl XmaII in pSAD  

Incubation 2H at 37°C
Migration in 0.1 % agarose gel, ladder “2log”, deposit of 10µL of reaction sample + 2µl de loading buffer 6X</P>
&gt; </DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="jul31"> Tuesday 31th July</H2><H4> DNA sent to sequencage  Digestion reaction using BSaI </H4><H6> Muséum national d'histoire naturelle (MNHN) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
Aurélie
</DIV></DIV></DIV><H5>1. DNA sent to sequencage </H5><P class="text-notebook">pSAD  AYL0014041 ← GG40  042 ← GG41 </P><H5>Digestion reaction using BSaI </H5><DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">1 reaction </TH><TH class="th-notebook">mix *12</TH></TR></THEAD><TBODY><TR><TD class="td-notebook">Bcl Enzyme  </TD><TD class="td-notebook">0.2 µL </TD><TD class="td-notebook">24 µL </TD></TR><TR><TD class="td-notebook">Buffer Cutsmart (10X) </TD><TD class="td-notebook">1  µL </TD><TD class="td-notebook">12µL </TD></TR><TR><TD class="td-notebook">DNA  </TD><TD class="td-notebook">4  µL </TD><TD class="td-notebook">↗ </TD></TR><TR><TD class="td-notebook">H2O mq</TD><TD class="td-notebook">4.8 µL  </TD><TD class="td-notebook">57.6µL </TD></TR></TBODY></TABLE></DIV></DIV><P class="text-notebook">Incubation 3h 37°C 
Migration on 1% agarose gel of 10µl sample + 2µL loading buffer  
Ladder 2Log  
90V, 20 minutes </P><P class="text-notebook"> OK for gag pol 3’ and 5’ LTR, Rb+ paro Sequencing   
pSAD not ok → replanting of colonies </P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="d-flex justify-content-center"><H2 class="text-center" style="background: #e6e6e6; width: 300px; border-radius: 10px; border-bottom: none;"> August</H2></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-left text-sm-left text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="a1"> Wednesday 1st August</H2><H4 text-xs-left="" text-sm-left=""> Sending samples to sequencing  Cloning of pSAD</H4><H6> Muséum national d'histoire naturelle (MNHN) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie
</DIV></DIV></DIV><H5>1. Sending samples to sequencing  </H5><P class="text-notebook">- GAG 3 (10µL DNA+ 10µL H2O) 
- GAG 5 (10µL DNA+ 10µL H2O) 
- 3’ LTR (3’4) (10µL DNA+ 5 µL H2O) 
- 5’ LTR (10µL DNA+ 5 µL H2O) 
- Rb (10µL DNA+ 5 µL H2O) 
- GG40 (50µL) and GG41 (10µL) 
-Ig18 (20µL), Ig19, Ig20 (10µL) 
Stock from 07/25/18
</P><H5>Cloning of pSAD from 07/23/2018 experiment, in solid LB+ spec </H5></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="a2"> Thurday 2nd August</H2><H4> Replanting of PSAD and Rb intron</H4><H6> Muséum national d'histoire naturelle (MNHN) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie
</DIV></DIV></DIV><P class="text-notebook">Replanting of PSAD and Rb intron in liquid media+ spec .</P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="a3"> Friday 3th August</H2><H4> Miniprep of PSAD and Rb</H4><H6> Muséum national d'histoire naturelle (MNHN) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie
</DIV></DIV></DIV><P class="text-notebook">Miniprep of PSAD and Rb (5 clones of each) 
Kit used: Nucleopsin Plasmid 
</P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="a6"> Monday 6th August</H2><H4> Digestion of the 5 new clones of pSAD and Rb  </H4><H6> Muséum national d'histoire naturelle (MNHN) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie
</DIV></DIV></DIV><DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">1 reaction </TH><TH class="th-notebook">mix *12</TH></TR></THEAD><TBODY><TR><TD class="td-notebook">BSAI Enzyme </TD><TD class="td-notebook">0.2 µL </TD><TD class="td-notebook">2.4 µL</TD></TR><TR><TD class="td-notebook">Buffer Cut Smart (10X) </TD><TD class="td-notebook">1  µL </TD><TD class="td-notebook">12µL </TD></TR><TR><TD class="td-notebook">DNA  </TD><TD class="td-notebook">4  µL </TD><TD class="td-notebook">↗ </TD></TR><TR><TD class="td-notebook">H2O mq</TD><TD class="td-notebook">4.8 µL  </TD><TD class="td-notebook">57.6µL </TD></TR></TBODY></TABLE></DIV></DIV><P class="text-notebook">6µl of mix per tube + 4µl of DNA 
Digestion 2H at 37°C+ 2µL of loading buffer 6X, 4µL of 2log ladder 
Migration 90V, 40min </P><A class="pointer" title="Click on the image for a larger view " data-toggle="modal" data-target=".Migration-180806"><DIV class="container-figurecaption"><DIV class="content-figurecaption"><P><SPAN style="font-weight: bold;">Figure : </SPAN>Gel electrophoresis of digestion reaction products to check the inserts size : 5 new clones of pSAD and Rb </P></DIV></DIV></A></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="a7"> Tuesday 7th August</H2><H4> Digestion/ ligation reaction of pSAD 2 Rb intron  Transformation  Preparation X-GAL </H4><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie
</DIV></DIV></DIV><H5>1. Digestion/ ligation reaction of pSAD 2 Rb intron </H5><P class="text-notebook"><A href="#jun14">see 06/14/2018 protocol</A></P><H5>2. Transformation  </H5><P class="text-notebook">-90µl of DH5α +Ligation product +10µl CMR 
           30 min 4°C 
           90 sec at 42°C 
           2 min on ice 
700µL LB 
            1h at 37°C
                  spreading on 2 dishes </P><H5> 3. Preparation X-GAL </H5><P class="text-notebook">40mg ⇨ 2mL DMSO</P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="a21"> Tuesday 21th August</H2><H4> New PCR of “pSAD” and “rubisco” intron  </H4><H6>Institut de biologie physico-chimique (IBPC) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie
</DIV><DIV class="chip">
 Ursula
</DIV></DIV></DIV><P class="text-notebook">Reaction Mix</P><DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">Solutions</TH><TH class="th-notebook">Volume for 1 tube (uL) </TH><TH class="th-notebook">Volume for 7 tubes (uL)</TH></TR></THEAD><TBODY><TR><TD class="td-notebook">Plasmid DNA</TD><TD class="td-notebook">1</TD><TD class="td-notebook">-</TD></TR><TR><TD class="td-notebook">Phusion Enzyme</TD><TD class="td-notebook">0.25 </TD><TD class="td-notebook">1.75 </TD></TR><TR><TD class="td-notebook">HF Buffer (5X) </TD><TD class="td-notebook">5 </TD><TD class="td-notebook">35 </TD></TR><TR><TD class="td-notebook">dNTP</TD><TD class="td-notebook">0.5  </TD><TD class="td-notebook">3.5</TD></TR><TR><TD class="td-notebook">Forward Primer</TD><TD class="td-notebook">1.25  </TD><TD class="td-notebook">- </TD></TR><TR><TD class="td-notebook">Reverse Primer</TD><TD class="td-notebook">1.25  </TD><TD class="td-notebook">- </TD></TR><TR><TD class="td-notebook">H2O</TD><TD class="td-notebook">15.75 µL  </TD><TD class="td-notebook">110.25</TD></TR></TBODY></TABLE></DIV></DIV><P class="text-notebook">- 21.5 uL of Mix/Tube 
- 1 uL of DNA 
- 2 X 1.25 uL Primer 

pSAD = Ig3/Ig4 primers + template p0.07 → 807 bp 
Rb intron = Ig5/Ig6 primers + template 0.07 → 177 bp 

1- p007 + Ig3/Ig4 
2- pUC19 + Ig3/Ig4 
3- H2O + Ig3/Ig4 
4- p0-02 + Ig5/Ig6 
5- pUC19 + Ig5/Ig6 
6- H2O + Ig5/Ig6 </P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="a22"> Tuesday 22th August</H2><H4> Migration of the PCR products and the fragments of GAG Pol  Purification of the product the gel of the 2 PCRs:  Cloning of pSAD </H4><H6>Institut de biologie physico-chimique (IBPC) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie
</DIV><DIV class="chip">
 Ursula
</DIV></DIV></DIV><H5>1. Migration of the PCR products and the fragments of GAG Pol sent by IDT </H5><P class="text-notebook">Products of the PCR à OK 
IDT4 L0A1 + IDT4 L0A2 + IDT4 L0A3 + IDT3 L0B1 à OK 
 
No bands for IDT4 L0B2                	Were the samples mixed during the loading?  
Plusieurs bands for IDT6 L0B3 </P><H5>2.Purification of the product the gel of the 2 PCRs </H5><P class="text-notebook">Elution in 20 uL 
Used kit: Nucleospin Gel and PCR clean up 
Concentration measurements using the Nanodrop: 
- PSAD= 39.8 ng/uL
- Rb= 55.1 ng/uL</P><H5>3. Cloning of pSAD </H5><H5>Intron Rubisco </H5><H5> Ligation </H5><H5> Cycle</H5></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="a28"> Tuesday 28th August</H2><H4> Plasmidic extraction of bacteria transformed  Measure of the concentration   Digestion of the plasmid by BstEII and SalI  Preparation of DNA samples for sequencing  </H4><H6>Institut de biologie physico-chimique (IBPC) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie
</DIV><DIV class="chip">
Soukeyna
</DIV></DIV></DIV><H5> Plasmidic extraction of bacteria transformed by plasmid containing “Rubisco intron”</H5><P class="text-notebook">Used kit : NucleoSpin Plasmid (Macherey-Nagel) </P><H5> Measure of the concentration at Nanodrop </H5><P class="text-notebook">Rb 1 : 109.9 ng/µl 
Rb 2 :  96.8 ng/µl 
Rb 3 :  77.2 ng/µl
Rb 4 :  88.7 ng/µl</P><H5> Measure of the concentration at Nanodrop </H5><P class="text-notebook">Digestion of the plasmid by BstEII and SalI 
Goal : Check if the plasmid effectively contain the insert “Rubisco intron” 
Expected result : 
</P><DIV class="d-flex justify-content-center"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">Lader (2log)</TH><TH class="th-notebook">Rb1+ </TH><TH class="th-notebook">Rb2+</TH><TH class="th-notebook">Rb3+</TH><TH class="th-notebook">Rb4*</TH></TR></THEAD></TABLE></DIV><A class="pointer" title="Click on the image for a larger view " data-toggle="modal" data-target=".Migration-180828"><DIV class="container-figurecaption"><DIV class="content-figurecaption"><P><SPAN style="font-weight: bold;">Figure : </SPAN>Gel electrophoresis of digestion reaction products to check the inserts size </P></DIV></DIV></A><H5>Preparation of DNA samples for sequencing  </H5><P class="text-notebook"> add 2 µl of primers at 10 µL for each tube  </P><H5>Concentration measurement on Nanodrop of recieved parts for trehalose   </H5><P class="text-notebook"> Provider : DC biosciences  </P><DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">Part</TH><TH class="th-notebook">DNA concentration (ng/µl) </TH></TR></THEAD><TBODY><TR><TD class="td-notebook">pUC-SP VS7 otsA_B21</TD><TD class="td-notebook">165.3 </TD></TR><TR><TD class="td-notebook">pUC-SP VS8 otsA_B22</TD><TD class="td-notebook">103.9 </TD></TR><TR><TD class="td-notebook">pUC-SP VS9 otsA_B23 </TD><TD class="td-notebook">107.6 </TD></TR><TR><TD class="td-notebook">pUC-SP VS10 otsB_B41</TD><TD class="td-notebook">242.7 </TD></TR><TR><TD class="td-notebook">pUC-SP VS11 otsB_B42</TD><TD class="td-notebook">81.2  </TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="a29"> Tuesday 29th August</H2><H4>Sample preparation to send “rubisco intro” at sequencing </H4><H6>Institut de biologie physico-chimique (IBPC) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie
</DIV><DIV class="chip">
 Ursula
</DIV></DIV></DIV><P class="text-notebook">Add 2µl of primers at 10µM</P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="a30"> Tuesday 30th August</H2><H4> Digestion/ligation reaction of otsA and otsB   Transformation of B10 bacteria with ligation product  PlCH47811 plasmid extraction  Replanting of 𝝱10 clones transformed  Checking of DNA content of IDT4 sample for Gag Po </H4><H6>Institut de biologie physico-chimique (IBPC) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie
</DIV><DIV class="chip">
Soukeyna
</DIV></DIV></DIV><H5>1. Digestion/ligation reaction of otsA and otsB  </H5><H5>2. Transformation of B10 bacteria with ligation product </H5><P class="text-notebook">- 25µl of B10 bacteria + 4µl of ligation product (4 different transformations) 
- 30 min at 4°C 
- 55 sec at 42°C 
- 2 min at 4°C 
+700µl of LB medium  
1 h at 37°C  
Spreading of 250 µl in plates containing LB+spectinomomycin+X-gal </P><H5>3. PlCH47811 plasmid extraction from overnight culture </H5><P class="text-notebook">kit used: Machinery Nagel easypure kit plasmid DNA extraction</P><H5>4. PlCH47811 plasmid  DNA concentration measurement on Nanodrop </H5><P class="text-notebook">a :  52.8 ng/µl    1.91    1.98 
b  : 39.8 ng/µl    1.90    1.84
</P><H5>5. Replanting of 𝝱10 clones transformed the day before on LB media + spectinomycine </H5><P class="text-notebook">2 clones of OTSA : OTSA1  OTSA2 (one culture per clone)
2 clones of OTSB : OTSB1  OTSB2 (one culture per clone)
1 clone of  pGAM 1276
1 clone of  pAGM1299

the 6 cultures previously described are placed on a will for over week/end culture

</P><H5>6. Checking of DNA content of IDT4 sample for Gag Pol</H5>
The tube is vortexed upside down then centrifuged (to check if the DNA is not stuck on the tube’s lid) 
DNA concentration measurement on nanodrop: 
(blank : H2O)
1.5ng/µL
1.6ng/µL
1.5ng/µl</DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="d-flex justify-content-center"><H2 class="text-center" style="background: #e6e6e6; width: 300px; border-radius: 10px; border-bottom: none;"> September</H2></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="s3"> Monday 3th September</H2><H4>Suspension of primers IDT-4L0-B2-F and IDT-4L0-B2-R  Miniprep of otsA, otsB and pAGM1276 and pAGM1299 Concentration measurement  Level 0 ligation protocol for GAGPOL parts and Level 1 ligation protocol for the CARGO retrotransposon</H4><H6>Institut de biologie physico-chimique (IBPC) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie
</DIV></DIV></DIV><H5>1. Suspension of primers IDT-4L0-B2-F and IDT-4L0-B2-R received today according to Eurofins instructions</H5><P class="text-notebook">Then, hybridation during 15 minutes (there are in fact, the missing DNA fragment of GAGPOL)
.</P><H5>2. Miniprep on the over week-end culture of otsA, otsB and pAGM1276 and pAGM1299</H5><P class="text-notebook">Used kit : NucleoSpin Plasmid (Macherey-Nagel)</P><H5>3. Concentration measurement at Nanodrop</H5><P class="text-notebook">- pAGM1276 : 56ng/µl 
- pAGM1299 : 76.6ng/µl 
- otsA 1 : 78.4ng/µl 
- otsA 2 : 75.6ng/µl 
- otsB 1 : 73.1ng/µL
- otsB2 : 78.2ng/µl </P><H5>4. Level 0 ligation protocol for GAGPOL parts : <A href="jul9">(see 09/07/2018)</A></H5><H5>5. Level 1 ligation protocol for the CARGO retrotransposon </H5></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="s4"> Tuesday 4th September</H2><H4>Digestion of otsA and otsB ligation reaction  Transformation of B10 bacteria with cloning reaction  </H4><H6>Institut de biologie physico-chimique (IBPC) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie
</DIV><DIV class="chip">
Soukeyna
</DIV></DIV></DIV><H5>1. Digestion of otsA and otsB ligation reaction of <A href="#a3">03/08/2018</A></H5><P class="text-notebook">Digestion reaction of otsA : 

- bstEII-HF : 0.2µl  
- CutSmart buffer (10X) : 1 µl 
- DNA plasmid : 3µl  
- H20: 5.8µl </P><P class="text-notebook">Digestion reaction of otsB : 

- bstEII-HF : 0.1µl 
- NotI-HF : 0.1µl 
- CutSmart buffer (10X) : 1µl 
- DNA plasmid : 3µl  
- H20 : 5.8µl 
Electrophoresis on 0.1% agarose, with 2 log ladder and 12µl of reaction
</P><A class="pointer" title="Click on the image for a larger view " data-toggle="modal" data-target=".Migration-180904"><DIV class="container-figurecaption"><DIV class="content-figurecaption"><P><SPAN style="font-weight: bold;">Figure : </SPAN>Gel electrophoresis of digestion reaction products to check the inserts size </P></DIV></DIV></A><P class="text-notebook">The band on the gel are correct but there is a lot od DNA on the top of the gel and the secnd band is very weak. Morover, the sequence we ordered for otsB is wrong, so we need to correct it by PCR. So we will ordered primers and we’ll send to sequence after the PCR. For otsA, we’ll make an other transformation to select only one pure clone
</P><H5>2. Transformation of B10 bacteria with cloning reaction of <A href="#s3">03/09 </A>: p1-iGEM1 and p0-iGEM5 </H5><P class="text-notebook">25µl of B10 bacteria + 4µl of ligation product (4 different transformations) 
30 min at 4°C 
55 sec at 42°C 
2 min at 4°C 
+700µl of LB medium 
1 h at 37°C 

Spreading of 250 µl in plates containing LB+spectinomomycin+X-gal for p0-iGEM5 
Spreading of 250 µl in plates containing LB+ampicilin+X-gal for p1-iGEM2</P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="s5"> Wednesday 5th September</H2><H4>Transformation of B10 bacteria with cloning reaction p1-iGEM1 and p0-iGEM5 Replanting of white colonies from p0-iGEM-GAGPO  Re-do of L1 ligation  of p1-iGEM2 (Cargo) </H4><H6>Institut de biologie physico-chimique (IBPC) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
Soukeyna
</DIV></DIV></DIV><H5>1. Transformation of B10 bacteria with cloning reaction of <A href="#s3">03/09 </A>: p1-iGEM1 and p0-iGEM5 </H5><P class="text-notebook">Observation of plates from the day before 
- p1-iGEM2 (Cargo) : only blue colonies for + and - controle 
- p0-iGEM5- GagPol: only blue colonies for the positive (+) experiment , and both white and blue colonies for the negative (-)  control
</P><H5>2. Replanting of white colonies from p0-iGEM-GAGPOL (-)on liquid media </H5><P class="text-notebook">Composition of the media :Amp+ Spec+ Xgal 
4 colonies were replanted 
</P><H5>Re-do of L1 ligation  of p1-iGEM2 (Cargo) <A href="#s3"> (same experiment as the 09/03/2019)</A></H5></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="s6"> Tuesday 6th September</H2><H4> PDNA extraction of 4 differents clones p0-iGEM5  Concentration measurement  Verification digestion of p0-iGEM5 plasmid  Transformation of bacteria with the ligation produc </H4><H6>Institut de biologie physico-chimique (IBPC) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie
</DIV><DIV class="chip">
 Soukeyna
</DIV></DIV></DIV><H5>1. DNA extraction of 4 differents clones p0-iGEM5  </H5><P class="text-notebook">Kit utilisé : NucleoSpin Plasmid (Macherey-Nagel)</P><H5>2. Concentration measurement at Nanodrop  </H5><P class="text-notebook">p0-iGEM5-1 : 97.8ng/µl 
p0-iGEM5-2 : 89.5ng/µl 
p0-iGEM5-3 : 123.4ng/µl 
p0-iGEM5-4 : 67.6ng/µL </P><H5>3.Verification digestion of p0-iGEM5 plasmid </H5><DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">Clone 1 and 3</TH><TH class="th-notebook">Clone 2 and 4</TH></TR></THEAD><TBODY><TR><TD class="td-notebook">bstEII enzyme</TD><TD class="td-notebook">0.2µl </TD><TD class="td-notebook">0.2µl </TD></TR><TR><TD class="td-notebook">Plasmid DNA</TD><TD class="td-notebook">2µl </TD><TD class="td-notebook">3µ </TD></TR><TR><TD class="td-notebook">NEB cutsmart buffer (10X) </TD><TD class="td-notebook">1µl </TD><TD class="td-notebook">1µ </TD></TR><TR><TD class="td-notebook">H20</TD><TD class="td-notebook">6.8µl </TD><TD class="td-notebook">5.8µl </TD></TR></TBODY></TABLE></DIV></DIV><A class="pointer" title="Click on the image for a larger view " data-toggle="modal" data-target=".Migration-180906"><DIV class="container-figurecaption"><DIV class="content-figurecaption"><P><SPAN style="font-weight: bold;">Figure : </SPAN>Gel electrophoresis of digestion reaction products to check the inserts size </P></DIV></DIV></A><H5>4. Transformation of bacteria with the ligation product of the 05/09 </H5><P class="text-notebook">25µl of bacteria + 3µl of ligation product 
30min at 4°C 
55sec at 42°C 
2min at 4°C 
+700µl of LB medium on each tube 
1h at 37°C 
Spreading of 200µl of each transformation on LB + Xgal + amp + IPTG plate

Oovernight at 37°C

</P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="s7"> Friday 7th September</H2><H4> Ligation reaction of p0-iGEM5 (GAGPOL)  otsB PCR </H4><H6>Institut de biologie physico-chimique (IBPC) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie
</DIV></DIV></DIV><H5>1. Ligation reaction of p0-iGEM5 (GAGPOL)</H5><H5>2. otsB PCR </H5><P class="text-notebook">- p0-iGEM8 (otsB) : 1µl at 2ng/µl 
- Enzyme Phusion : 0.25µl  
- HF Buffer (5X) : 5 µl 
- dNTP : 0.5µl  
- Primer OTSB-v2-F : 1.25 µl 
- Primer OTSB-v2-R : 1.25µl 
-H20 : 15.75 µl </P><DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook" colspan="3">*30</TH></TR></THEAD><TBODY><TR><TD class="td-notebook">98°C</TD><TD class="td-notebook">98°C </TD><TD class="td-notebook">50°C </TD><TD class="td-notebook">72°C</TD><TD class="td-notebook">72°C </TD><TD class="td-notebook">12°C</TD></TR><TR><TD class="td-notebook">1min</TD><TD class="td-notebook">1 min </TD><TD class="td-notebook">2 min </TD><TD class="td-notebook">3 min</TD><TD class="td-notebook">3 min </TD><TD class="td-notebook">Hold </TD></TR></TBODY></TABLE></DIV></DIV><H5>3. 6 colonies <A href="s6">from yesterday</A> transformation p1igem2 in LB medium + amp </H5></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="s10"> Monday 10th September</H2><H4>DNA extraction of the 6 bacteria culture  Concentrations measurement  Digestion reaction to check p1-iGEM2 cloning reaction  Transformation of B10 bacteria  </H4><H6>Institut de biologie physico-chimique (IBPC) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Aurélie
</DIV></DIV></DIV><H5>1. DNA extraction of the 6 bacteria culture prepare on <A href="#s7">7/09</A></H5><P class="text-notebook">Used kit : NucleoSpin Plasmid (Macherey-Nagel)</P><H5>2. Concentrations measurement at Nanodrop </H5><P class="text-notebook">#1 : 101.5ng/µl 
#2 : 90.0ng/µl 
#3 : 97.7ng/µL 
#4 : 75.1ng/µl 
#5 : 61.5ng/µl 
#6 : 115.6ng/µl </P><H5>3. Digestion reaction to check p1-iGEM2 cloning reaction </H5><DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">NcoI</TH><TH class="th-notebook">0.1µl</TH><TH class="th-notebook">6.5µl</TH></TR></THEAD><TBODY><TR><TD class="td-notebook">NdeI</TD><TD class="td-notebook">0.1µl </TD><TD class="td-notebook">6.5µL </TD></TR><TR><TD class="td-notebook">NotI</TD><TD class="td-notebook">0.1µl</TD><TD class="td-notebook">6.5µl </TD></TR><TR><TD class="td-notebook">Cutsmart buffer 10X</TD><TD class="td-notebook">1µl</TD><TD class="td-notebook">6.5µl</TD></TR><TR><TD class="td-notebook">ADN</TD><TD class="td-notebook">2.5µl</TD></TR><TR><TD class="td-notebook">H2O</TD><TD class="td-notebook">6.2µl</TD><TD class="td-notebook">40.3µl </TD></TR></TBODY></TABLE></DIV></DIV><P class="text-notebook">Used kit : NucleoSpin Plasmid (Macherey-Nagel)</P><H5>Concentrations measurement at Nanodrop </H5><P class="text-notebook">7.5µl of mix by tube + 2.5µl of DNA 
On 1% agarose gel, with otsB PCR from <A href="#s7">7/09</A> : expected result : &gt;800bp
</P><H5> 4. Transformation of B10 bacteria with ligation product from <A href="#s7">07/09</A></H5><P class="text-notebook">B10 bacteria + 3.5µl of ligation product 
- 30 min at 4°C 
- 55sec at 42°C 
- 2min at 4°C 
- +700µl of LB medium 
- 1h at 37°C 
- Spreading of 200µl on plate with LB+spec+Xgal  </P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="s11"> Tuesday 11th September</H2><H4>Digestion of p1-IGEM2 with bbsI  Transformation of bacteria with Gagpol </H4><H6> Muséum national d'histoire naturelle (MNHN) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
Sara and Parisa
</DIV></DIV></DIV><H5>1. Digestion of p1-IGEM2 with bbsI</H5><DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">Components</TH><TH class="th-notebook">1 reaction</TH><TH class="th-notebook">Mix (*6)</TH></TR></THEAD><TBODY><TR><TD class="td-notebook">bbsI</TD><TD class="td-notebook">0.5µL </TD><TD class="td-notebook">3 µL</TD></TR><TR><TD class="td-notebook">Cutsmart buffer (10X)</TD><TD class="td-notebook">1 µL</TD><TD class="td-notebook">6 µL </TD></TR><TR><TD class="td-notebook">DNA</TD><TD class="td-notebook">2.5 µL</TD><TD class="td-notebook">6.5µl </TD></TR><TR><TD class="td-notebook">H2O</TD><TD class="td-notebook">6 µL</TD><TD class="td-notebook">36µL</TD></TR></TBODY></TABLE></DIV></DIV><H5>2. Transformation of bacteria with Gagpol </H5><P class="text-notebook">2 white  colonies were observed on the plate containing bacterias transformed with GAGPOL 
- Replanting on liquid media 
- Replanting on plate (not isolable) </P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="s12"> Wednesday 12th September</H2><H4>Migration of the digestion products from the day before (pI-IGEM2)
  Plasmid extraction from the GAG clone culture  Purification of DNA contained in the migration gel for OtsB PCR products  Digestion reaction for Gag POL  Level 0 Otsb (digestion/ligation reaction)  </H4><H6> Muséum national d'histoire naturelle (MNHN) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Sara and Parisa
</DIV><DIV class="chip">
Victor
</DIV></DIV></DIV><H5>1. Migration of the digestion products from the day before (pI-IGEM2)</H5><P class="text-notebook"> (0.1% agarose gel, 2-log ladder, 100V)
We were expecting a 4.5kb band for the plasmid and a 2.6kb band for the insert. 
We do have the expected band for the plasmid but, we observe a band at 0.6kb for the insert (Lac Z maybe?)</P><H5>2. Plasmid extraction from the GAG clone culture launched the day before </H5><P class="text-notebook">Kit used Nucleospin Plasmid </P><H5>3. Purification of DNA contained in the migration gel for OtsB PCR products <A href="#s7">( from 09/07/2018)</A></H5><P class="text-notebook">Kit used : Nucleospin gel to PCR clean up </P><H5>4. DNA concentration measurement on Nanodrop </H5><P class="text-notebook">Insert OtsB= 4ng/µL 
GAG palamsid = 48.1 ng/µl
 </P><H5>5. DNA concentration measurement on Nanodrop </H5><P class="text-notebook">Insert OtsB= 4ng/µL 
GAG palamsid = 48.1 ng/µl
 </P><H5>6. Level 0 Otsb (digestion/ligation reaction) </H5><P class="text-notebook">→ purification of PCR band and Launch of miniprep 
(To do next level 1 pI-Igem2)</P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="s13"> Tuesday 13th September</H2><H4>Transformation of 𝛃10 bacterias with OtsB ligation products  DNA plamsid extraction of pO-IGEM5  Digestion reaction  </H4><H6> Muséum national d'histoire naturelle (MNHN) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Sara
</DIV><DIV class="chip">
Victor
</DIV></DIV></DIV><H5>1. Transformation of 𝛃10 bacterias with OtsB ligation products obtained the day before </H5><P class="text-notebook"> 𝛃10+ 4µL of  OtsB ligation reaction products 
- 30min at 4°C 
- 55 sec at 42°C 
- 2 min at 4°C 
- Addition of 70µL of LB 
1 Hour at 37°C 
Spreading of 200µL on LB+Xgal + Spec plate 
37°C O/N
</P><H5>2. DNA plamsid extraction of pO-IGEM5 launch on liquid media yesterday
 </H5><P class="text-notebook">Kit used Nucleospin Plasmid </P><H5>3. Nanodrop measurement for p0-IGEM5 </H5><P class="text-notebook">26.4 ng/µL </P><H5>4.Digestion reaction </H5><P class="text-notebook">bbsI =0.5µL 
buffer= 2µL
DNA =10µL 
H2O= 6.5µL 
Expected result: 4,4kb + (?)</P><H5>5. Gel migration of digestion reaction products </H5><P class="text-notebook">ast well = digestion reaction product of GagPOL from the day before 
→ Does not correspond to the expected lenght </P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row text-center text-xs-center text-sm-center text-md-left  text-lg-left "><DIV class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><DIV class="note"><H2 class="text-center" id="s17"> Monday 17th September</H2><H4>Ligation  Migration of  the Gag Pol sample  Preculture of OTSB (7 white colonies)
  </H4><H6> Muséum national d'histoire naturelle (MNHN) </H6><DIV class="d-flex justify-content-end"><DIV class="chip-icone"><DIV class="chip">
 Sara
</DIV><DIV class="chip">
Victor
</DIV></DIV></DIV><H5>1. Ligation </H5><P class="text-notebook">3 ligation tubes are prepared , T-, G, pre-mix of E, T- 
- Tube 1 : T-  1.2µl of blackbone +12.8 µL of H2O (14µL in total) 
- Tube 2 : G  2 µL of the 6 sub-parts for GagPol)+.0.8µl of H2O +1.2µL of backbone ( 14µl in total)
Tube 3: Master mix of enzymes 
</P><DIV class="d-flex justify-content-center"><DIV class="table-responsive-sm"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">Components</TH><TH class="th-notebook">1 reaction</TH></TR></THEAD><TBODY><TR><TD class="td-notebook">ATP</TD><TD class="td-notebook">4 µL </TD></TR><TR><TD class="td-notebook">Cutsmart buffer (10X)</TD><TD class="td-notebook">4 µL</TD></TR><TR><TD class="td-notebook">T4 ligase</TD><TD class="td-notebook">2 µL</TD></TR><TR><TD class="td-notebook">bbstI</TD><TD class="td-notebook">2 µL</TD></TR></TBODY></TABLE></DIV></DIV><P class="text-notebook"> 12µL of mix in total dispatched in tubes 1 and 2 (6µL each)</P><H5>2. 1% agarose gel preparation, in order to migrate the Gag Pol sample  </H5><P class="text-notebook">- 0.5g of agarose 
- 50ml of TEA 1X 
- Cyber green 

Gel plan:  well 1 : 3µl of DNA ladder, well 2 and 3 10µl of Gagpol
 </P><H5>Preculture of OTSB (7 white colonies) </H5><P class="text-notebook">For each : 
- 4ml of LB media 
- 2µl of spectinomycin (100mg/ml) (in order to obtain 10µg/ml)  
-  1 white clone  
Incubation at 37°C overnight

 </P><H5>4. To do next</H5><P class="text-notebook">
- Otsb miniprep + digestion 
- Gag digestion 
- Gag transfo)
</P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="d-flex justify-content-center"><H2 class="text-center" style="background: #e6e6e6; width: 300px; border-radius: 10px; border-bottom: none;"> October</H2></DIV></DIV></DIV><DIV class="container fixed-bottom backtotop" style="padding-bottom: 10px; padding-right:0px; position: fixed; left: 120px;"><DIV class="row text-right text-xs-right text-sm-right text-md-right  text-lg-right  "><DIV class="col-xs-4 col-sm-4 col-md-12 col-lg-12"><A class="y btn-sm" href="#backtotop" role="button" style=" color:white; border-radius: 10px; background-color:#22D66C; text-decoration:none; ">  Back to top</A></DIV></DIV></DIV><SECTION id="footer"><DIV class="container"><DIV class="row text-center text-xs-center text-sm-center text-md-right"><DIV class="col-xs-12 col-sm-3 col-md-3 col-lg-6"><H5 class="text-left" style="border: none;padding: 0px">iGEM Sorbonne U Paris</H5><P class="text-left" style="color:white; font-size:16px;"> 4 place Jussieu  75005 Paris France</P></DIV></DIV></DIV><DIV class="container"><P class="text-center" style="color: white; padding:20px 0 20px 0">Made in Paris with love © iGEM Sorbonne U Paris 2018</P></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>