<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Sorbonne_U_Paris_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Sorbonne U Paris/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="collapse navbar-collapse " id="navbarTogglerDemo02"><UL class=" navbar-nav "><LI class="nav-item dropdown" id="mobile-homepage"><A class="nav-link" href="https://2018.igem.org/Team:Sorbonne_U_Paris" style="color:white;"> Home page</A></LI><LI class="nav-item dropdown  "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white">  Project
        </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Description">Description</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Design">Design</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Model">Model</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Demonstrate">Demonstrate</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Improve">Improve</A></DIV></LI><LI class="nav-item dropdown  "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Biology</A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Experiments">Experiments</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Results">Results</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Parts">Parts</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/InterLab">Interlab</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Safety">Safety</A></DIV></LI><LI class="nav-item dropdown  "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Human practices
        </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Human_Practices">Human Practices</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Integrated_Human_Practices">Integrated HP</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Public_Engagement">Education &amp; Engagement</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Meet_up">Meet up</A></DIV></LI><LI class="nav-item dropdown "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Notebook
        </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Notebook">Notebook</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Protocols">Protocols</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Brainstorming">Brainstorming</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Team </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Team">Members</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Collaborations">Collaborations</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Attributions">Attributions</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Sponsors">Sponsors</A></DIV></LI><LI class="nav-item dropdown  "><A class="nav-link" href="#" id="navbarDropdownMenuLink" style="color:white"> Awards</A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item pointer" href="https://igem.org/2018_Judging_Form?team=Sorbonne_U_Paris">Judging form</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Medals">Medals</A><A class="dropdown-item pointer" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Giant_Jamboree">Giant Jamboree</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link" href="https://2018.igem.org/Team:Sorbonne_U_Paris/Sitemap" style="color:white;"> Site map</A></LI></UL></DIV><DIV class="global" id="backtotop"><DIV class="container"><ARTICLE class="col-md-10"><H1>Protocols</H1><P class="introduction-text">On this page, you can find the protocols we used during de iGEM project. Feel free to have a look. If you have any questions, you can contact us at <A class="link-inside-page" href="" mailto:igemsorbonneuniv2018=""> igemsorbonneuniv2018@gmail.com</A></P></ARTICLE></DIV><DIV class="container"><ARTICLE class="col-md-10"><DIV id="inside-description"><A class="pointer" data-toggle="collapse" href="#collapseExample1" role="button" aria-expanded="false" aria-controls="collapseExample"><H1 class="titleHP">Polymerisation Chain Reaction (PCR) </H1><P class="inside-text"><SPAN style="font-weight: bold; color:black; font-size: 18px;">Goal: </SPAN>  In order to amplify a copy or a few copies of a specific fragment of deoxyribonucleic acid (DNA) to generate
thousands to millions of copies of DNA fragments of interest.</P></A></DIV><DIV class="collapse" id="collapseExample1"><DIV><P><B>Materials</B>:
- Phusion DNA polymerase 
- High Fidelity buffer (5X)
- Free dNTPs mix (10µM)
- Forward Primer (10µM)
- Reverse Primer (10µM)
- Sterile water 
- Plasmid DNA
- PCR tubes (0.2 mL) for the PCR machine</P><P><B>Methods</B>:
1. Choose the appropriate primers, for example use a software such as DNA strider 
2. Prepare a pre-mix (without the primers and plasmid DNA) based on the reaction volume and the number of tubes  </P><DIV class="d-flex justify-content-center"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">Components</TH><TH class="th-notebook">Volume for 1 reaction (in µL)
(total volume = 25 µL)
</TH></TR></THEAD><TBODY><TR><TD class="td-notebook">DNA template at 2 ng/µL</TD><TD class="td-notebook">1 </TD></TR><TR><TD class="td-notebook">Phusion Enzyme*</TD><TD class="td-notebook">0.25</TD></TR><TR><TD class="td-notebook">HF Buffer (5X)*</TD><TD class="td-notebook">5</TD></TR><TR><TD class="td-notebook">dNTP*</TD><TD class="td-notebook">0.5</TD></TR><TR><TD class="td-notebook">Forward Primer</TD><TD class="td-notebook">1.25</TD></TR><TR><TD class="td-notebook">Reverse Primer</TD><TD class="td-notebook">1.25</TD></TR><TR><TD class="td-notebook">H20 up to 25 µL*</TD><TD class="td-notebook">15.75</TD></TR></TBODY></TABLE></DIV><P class="text-center">*in the master mix</P><P>3. Distribute 21.5 µL of master Mix per Tube 
4. Add 1 µL of DNA and 1.25 µL of each Primer 
5. Mix gently the reaction. If necessary, collect the liquid to the bottom of the PCR tube by spinning briefly 
6. Put the PCR tubes to the thermocycler and parameter the cycles (take into account the melting temperature Tm for each 
primer) <B>Annealing temperature (Ta) calculation:</B>	
Ta  = Tm – 5°C with Tm  = 2°C  x  (A+T) + 4°C  x (G+C) for primers less than 30 bases long.<B>Note:</B> this approximate formula tends to overestimate the Tm value!	
 </P><DIV class="d-flex justify-content-center"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">Step</TH><TH class="th-notebook">Temperature (°C)</TH><TH class="th-notebook">Time (minutes)</TH></TR></THEAD><TBODY><TR><TD class="td-notebook">98</TD><TD class="td-notebook">1</TD></TR><TR><TD class="td-notebook" rowspan="3">35</TD><TD class="td-notebook">98</TD><TD class="td-notebook">1</TD></TR><TR><TD class="td-notebook">50</TD><TD class="td-notebook">2</TD></TR><TR><TD class="td-notebook">72</TD><TD class="td-notebook">3</TD></TR><TR><TD class="td-notebook">72</TD><TD class="td-notebook">3</TD></TR><TR><TD class="td-notebook">12</TD><TD class="td-notebook">Hold</TD></TR></TBODY></TABLE></DIV><P class="text-center"><B>Table: </B>Parameters of the thermocycler for our exepriments </P></DIV></DIV></ARTICLE></DIV><DIV class="container"><ARTICLE class="col-md-10"><DIV id="inside-description"><A class="pointer" data-toggle="collapse" href="#collapseExample2" role="button" aria-expanded="false" aria-controls="collapseExample2"><H1 class="titleHP">DNA gel electrophoresis</H1><P class="inside-text"><SPAN style="font-weight: bold; color:black; font-size: 18px;">Goal:</SPAN> After amplification by PCR, the DNA fragments are separated according to their size
on an agarose gel. The size of the DNA fragments is evaluated in comparison with a size
marker.</P></A></DIV><DIV class="collapse" id="collapseExample2"><DIV><P><B>Materials</B>:
- Agarose Powder 
- TAE buffer 10X (Tris-acetate 40 mM, EDTA 1 mM, pH 7.8)
- DNA ladder (NEB 1 log) 
- Loading buffer 6X (NEB) 
- Gel Tank 
- Comb 
- Balance 
- Microwave oven 
- Ethidium bromide 10mg/ml (EtBr) </P><P><B>Methods</B>:

</P><H6> Preparation of agarose gel </H6><DIV class="d-flex justify-content-center"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">% Gel</TH><TH class="th-notebook">Linear DNA size (Kb)</TH></TR></THEAD><TBODY><TR><TD class="td-notebook">0.5</TD><TD class="td-notebook">30 to 1.0</TD></TR><TR><TD class="td-notebook">0.7</TD><TD class="td-notebook">12 to 0.8</TD></TR><TR><TD class="td-notebook">1.0</TD><TD class="td-notebook">10 to 0.5</TD></TR><TR><TD class="td-notebook">1.2</TD><TD class="td-notebook">7 to 0.4</TD></TR><TR><TD class="td-notebook">1.5</TD><TD class="td-notebook">3 to 0.2</TD></TR><TR><TD class="td-notebook">2.0</TD><TD class="td-notebook">1 to 0.1</TD></TR></TBODY></TABLE></DIV><P class="text-center">Electrophoresis gel resolution </P><P> Agarose gel x % (w/v) in 1X TAE buffer. 	
Chose	the agarose concentration you need using the above table 

According to the gel’s size, prepare the appropriate volume of solution (for exemple 50 mL for a small sized agarose gel)  
1. Prepare 1% w/v solution of agarose powder in 1X TAE buffer (e.g. 1 g agarose powder in 100 mL TAE buffer) 
2. Heat the solution in a microwave to dissolve agarose until the solution is limpid (usually 30-45 sec at max power, avoid boiling) 	 
3. Cool the solution (not for too long or it will solidify). Avoid bubbles and vigorous stirring. You can allow hot agarose to cool in a water bath set at 50–55°C for 10 min. Swirl the flask occasionally to cool evenly. 	
4. Add Ethidium bromide EtBr solution(0.5mg/ml) to the cooled gel and pour into the gel tray  
5. Cast the solution on an electrophoresis gel mould insert the comb and let the gel solidify at room temperature </P><H6> Caution with hot agarose</H6><P>Wear protective gloves when handling extremely hot agarose solution and heat the solution in several short intervals—do not let the solution boil for long periods as it may boil out of the flask or cause a loss to water vapor </P><H6>Gel migration </H6><P>1. Remove the comb 
2. Immerse the gel with 1X migration buffer (TAE). Cover the gel with ~2–3 mm of buffer for best results. 
3. Load 3 µL of the DNA ladder on a well 
4. Load 15 µL DNA samples with the corresponding amount of DNA loading dye (DNA loading dye 1X) 
Add 5 µL of 6X loading buffer to the 25 µL of each sample and load in the  expected wells (respect chosen order)
5. Run the gel for 20 minutes at 100V </P><H6> Caution with Ethidium Bromide</H6><P>Ethidium Bromide is a sensitive fluorescent dye used to detect nucleic acids in agarose gels. It is a mutagen and probable carcinogen. It is toxic. Always wear gloves when working with ethidium bromide. Wipe the area with a damp cloth after the work with ethidium bromide. Also, while wearing gloves after handling ethidium bromide, be careful not to touch and thereby contaminate other surfaces. Be sure to wash hands with soap and warm water after working with ethidium bromide. </P><H6>Gel revelation:</H6><P>At the end of the electrophoresis migration, collect the Gel and take a picture of it using a gel reader, adjust the brightness if necessary. </P></DIV></DIV></ARTICLE></DIV><DIV class="container"><ARTICLE class="col-md-10"><DIV id="inside-description"><A class="pointer" data-toggle="collapse" href="#collapseExample3" role="button" aria-expanded="false" aria-controls="collapseExample3"><H1 class="titleHP">DNA Gel extraction </H1><P class="inside-text"><SPAN style="font-weight: bold; color:black; font-size: 18px;">Goal: </SPAN> NucleoSpin® Gel and PCR Clean-up allows DNA fragments to be purified from enzymatic reactions, such as PCR, as well as from agarose gels.</P></A></DIV><DIV class="collapse" id="collapseExample3"><DIV><P>The sample is mixed with Binding Buffer NTI and in case of a cut-out gel band, it is heated to dissolve the agarose. In the presence of chaotropic salt, the DNA is bound to the silica membrane of a NucleoSpin® Gel and PCR Clean-up Column. Contaminants are removed by simple washing steps with ethanolic Wash Buffer NT3. Finally, the pure DNA is eluted under low salt conditions with slightly alkaline Elution Buffer NE (5 mM Tris-HCl, pH 8.5). </P><P><B>Material:</B>
- NucleoSpin® Gel and PCR Clean-up kit (Macherey-Nagel) </P><P><B>Methods:</B>
1. Spot on the gel the expected bands 
2. Cut with scalpel on the gel the most precisely possible to extract the desired bands 
3. Use the Gel DNA extraction kit to extract the DNA samples from the gel 

DNA Gel extraction was performed according to the NucleoSpin® Gel and PCR Clean-up kit manufacturer’s instructions.
 </P></DIV></DIV></ARTICLE></DIV><DIV class="container"><ARTICLE class="col-md-10"><DIV id="inside-description"><A class="pointer" data-toggle="collapse" href="#collapseExample4" role="button" aria-expanded="false" aria-controls="collapseExample4"><H1 class="titleHP">MoClo Assembly Conditions</H1><P class="inside-text">  It is a standardized DNA strategy called Modular cloning ( MoClo). MoClo allows the assembly of basic parts such as promoters, CDSs or terminators and the MoClo relies on the Golgen Gate cloning strategy</P></A></DIV><DIV class="collapse" id="collapseExample4"><DIV><P><B>Materials:</B>
- BbsI-HF (NEB)  
- T4 ligase (NEB) 
- Cutsmart buffer (NEB) 
- ATP (10 mM solubilized in 0.1 M Tris-HCl, pH 7.9). 
 </P><P><B>Methods: </B>
To make our MoClo Assembly protocols, we used an excel table provided on the supplementary material from: <I> “Birth of a Photosynthetic Chassis: A MoClo Toolkit Enabling Synthetic Biology in the Microalga Chlamydomonas reinhardtii. Crozet P, et al. ACS Synth Biol. 2018” </I>

For each reactions this excel table enabled us to calculate the appropriate volumes of water, DNA, cutsmart buffer, enzymes and additives (for example ATP), based on the DNA concentrations we had. </P><DIV class="d-flex justify-content-center"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">Reaction mix</TH><TH class="th-notebook">ul</TH><TH class="th-notebook">Master mix</TH></TR></THEAD><TBODY><TR><TD class="td-notebook">BbsI-HF</TD><TD class="td-notebook">1</TD><TD class="td-notebook">2</TD></TR><TR><TD class="td-notebook">T4 ligase</TD><TD class="td-notebook">1</TD><TD class="td-notebook">2</TD></TR><TR><TD class="td-notebook">Buffer NEB cutsmart</TD><TD class="td-notebook">2</TD><TD class="td-notebook">4</TD></TR><TR><TD class="td-notebook">ATP (10 mM)</TD><TD class="td-notebook">2</TD><TD class="td-notebook">4</TD></TR><TR><TD class="td-notebook">Total volume</TD><TD class="td-notebook">20</TD><TD class="td-notebook">6 uL/tube</TD></TR></TBODY></TABLE></DIV><P>The parameters for the PCR thermocyclers following our digestion/ligation experiment were also based on the data from the article cited above, and whether we were performing a level 0 or level 1 cloning.
 </P><DIV class="d-flex justify-content-center"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">Step</TH><TH class="th-notebook">Temperature (°C)</TH><TH class="th-notebook">Time (minutes)</TH></TR></THEAD><TBODY><TR><TD class="td-notebook">3 cycles</TD><TD class="td-notebook">37</TD><TD class="td-notebook">10</TD></TR><TR><TD class="td-notebook">16</TD><TD class="td-notebook">10</TD></TR><TR><TD class="td-notebook">37</TD><TD class="td-notebook">10</TD></TR><TR><TD class="td-notebook">65</TD><TD class="td-notebook">20</TD></TR><TR><TD class="td-notebook">Hold</TD><TD class="td-notebook">16</TD><TD class="td-notebook">Indefinitely</TD></TR></TBODY></TABLE></DIV></DIV></DIV></ARTICLE></DIV><DIV class="container"><ARTICLE class="col-md-10"><DIV id="inside-description"><A class="pointer" data-toggle="collapse" href="#collapseExample5" role="button" aria-expanded="false" aria-controls="collapseExample5"><H1 class="titleHP">Quality Control by Restriction</H1><P class="inside-text"><SPAN style="font-weight: bold; color:black; font-size: 18px;">Goal: </SPAN>To check the size of the inserts </P></A></DIV><DIV class="collapse" id="collapseExample5"><DIV><P><B>Materials:</B>
- Cut smart  Buffer (10X) 
- Enzyme BsaI-HF®v2 (NEB) 
- Enzyme NdeI(NEB) 
- DNA 
- Sterile water <B>Methods:</B>
1. Set up the reaction as follows </P><DIV class="d-flex justify-content-center"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">Component</TH><TH class="th-notebook">Volume for 1 reaction (in µL)
(total volume = 10 µL)
</TH></TR></THEAD><TBODY><TR><TD class="td-notebook">DNA</TD><TD class="td-notebook">3</TD></TR><TR><TD class="td-notebook">Cutsmart Buffer 10X</TD><TD class="td-notebook">1 µL (1X)</TD></TR><TR><TD class="td-notebook">BsaI or NdeI </TD><TD class="td-notebook">0.2 µL (or 2 units)</TD></TR><TR><TD class="td-notebook">Nuclease-free Water</TD><TD class="td-notebook">Up to 10 µL</TD></TR></TBODY></TABLE></DIV><P>Incubate at 37°C for 1 hou </P></DIV></DIV></ARTICLE></DIV><DIV class="container"><ARTICLE class="col-md-10"><DIV id="inside-description"><A class="pointer" data-toggle="collapse" href="#collapseExample6" role="button" aria-expanded="false" aria-controls="collapseExample6"><H1 class="titleHP">Enzymatic digestion reaction (fast digest enzyme) </H1></A></DIV><DIV class="collapse" id="collapseExample6"><DIV><P><B>Materials:</B>
- FastDigest Buffer (FD) 10X (Thermofisher Scientific) 
- BcII Enzyme (NEB) <B>Methods:</B> 
1. Set up the reaction as follows </P><DIV class="d-flex justify-content-center"><TABLE class="text-center tablenotebook"><THEAD><TR><TH class="th-notebook">Component</TH><TH class="th-notebook">Volume for 1 reaction (in µL)
(total volume = 10 µL)
</TH></TR></THEAD><TBODY><TR><TD class="td-notebook">DNA</TD><TD class="td-notebook">3</TD></TR><TR><TD class="td-notebook">FastDigest Buffer (10X)</TD><TD class="td-notebook">1 µL (1X)</TD></TR><TR><TD class="td-notebook">BclI Enzyme </TD><TD class="td-notebook">0.2 µL (or 2 units)</TD></TR><TR><TD class="td-notebook">Nuclease-free Water</TD><TD class="td-notebook">Up to 10 µL</TD></TR></TBODY></TABLE></DIV><P> 2.	Incubate at 37°C for 1 hour</P></DIV></DIV></ARTICLE></DIV><DIV class="container"><ARTICLE class="col-md-10"><DIV id="inside-description"><A class="pointer" data-toggle="collapse" href="#collapseExample7" role="button" aria-expanded="false" aria-controls="collapseExample7"><H1 class="titleHP">Transformation of Escherichia coli 10-bêta and DH5-alpha</H1><P class="inside-text"><SPAN style="font-weight: bold; color:black; font-size: 18px;">Goal: </SPAN>  the integration of a plasmid with a specific inserts into bacteria</P></A></DIV><DIV class="collapse" id="collapseExample7"><DIV><P>Choose the appropriate bacterial strain based on your knowledge and your lab’s material. The strain of bacteria used for our experiments were chemically competent Escherichia coli DH5-alpha and 10-bêta kept at -80°C. 

E. coli bacteria are transformed by the heat shock technique: chemically competent cells are incubated at 4°C with DNA for 20 min, then heat shocked at 42°C for 1 min and put back on ice for 2 min. Then the bacteria are cultivated in liquid LB medium supplemented with glucose for 1 hour at 37°C in order to reconstitute their weakened wall and to express the antibiotic resistance gene carried by the plasmids. The bacteria are then spread on the LB medium supplemented with Xgal and adapted antibiotic in order to select the bacteria that have been transformed by the plasmid. </P><P><B>Materials:</B>
- E. coli 10-bêta (New England Biolabs® 10-beta Competent E. coli) or E. coli DH5-alpha 
- ligation products (assay or negative control) 
- LB medium 
- Ice bucket filled with ice 
- LB+ Xgal + spectinomycine solid medium 
- Shaking incubator with modular temperature </P><P><B>Methods</B> :
1. Take competent cells out of -80°C and thaw on ice (approximately 10 min). 
2. Mix 4 µL of ligation products with 30 µL of E. coli 10-bêta bacteria (or 90 µL of E. coli DH5-alpha).
3. Incubate the competent cell/DNA mixture on ice (4°C) for 30 minutes. 
4. Heat shock each transformation tube by placing the tube at 42°C for 55 seconds (E. coli 10-bêta bacteria) or 90 seconds (E. coli DH5-alpha).
5. Put the tubes back on ice (4°C) for 2 min. 
6. Add then 700 µL of LB medium with glucose (without antibiotic) and leave to incubate 1 h at 37°C. Shake vigorously (250 rpm) or rotate 
7. Spread 200 µL on plate containing LB + Xgal +spectinomycin 
Incubate plates at 37°C overnight. <B>NB:</B> Between the 2 and 4 steps, do not mix. 
 </P></DIV></DIV></ARTICLE></DIV><DIV class="container"><ARTICLE class="col-md-10"><DIV id="inside-description"><A class="pointer" data-toggle="collapse" href="#collapseExample8" role="button" aria-expanded="false" aria-controls="collapseExample8"><H1 class="titleHP">Transplanting a colony into a liquid medium</H1><P class="inside-text"><SPAN style="font-weight: bold; color:black; font-size: 18px;">Goal: </SPAN>Transfer microorganisms from agar to liquid medium for growth</P></A></DIV><DIV class="collapse" id="collapseExample8"><DIV><P><B>Material:</B>
-	LB-X-Gal-Spec liquid media 
-	LB agar plate with bacteria <B>Methods</B> 
1. Collect a clone of interest using a tip 
2.= Drop the tip in a liquid selective media LB- X-Gal-Spec liquid media <B>NB</B>: Label the tube with the plasmid name and the date. It is also a good idea to add the antibiotic resistance and your initials.
 </P></DIV></DIV></ARTICLE></DIV><DIV class="container"><ARTICLE class="col-md-10"><DIV id="inside-description"><A class="pointer" data-toggle="collapse" href="#collapseExample9" role="button" aria-expanded="false" aria-controls="collapseExample9"><H1 class="titleHP">Plasmid extraction (Mini-prep)</H1><P class="inside-text"><SPAN style="font-weight: bold; color:black; font-size: 18px;">Goal:</SPAN>The mini-preparation consists in recovering the plasmid present in the bacterium</P></A></DIV><DIV class="collapse" id="collapseExample9"><DIV><P><B>Material:</B>
For plasmid extractions we performed minipreps using: 
- NucleoSpin® Plasmid EasyPure (Macherey-Nagel), 
- GeneJET Plasmid Miniprep Kit (Thermofisher Scientific) 
- Nucleospin Plasmid Kit (Macherey-Nagel) <B>Methods:</B> 
Minipreps were carried out according to the manufacturer’s instructions. 

After an overnight liquid culture, the plasmid DNA is extracted, based on the alkaline lysis technique of the bacterial cells. The DNA is purified by means of a silica membrane which selectively retains DNA and selectively ionic strength. 
 </P></DIV></DIV></ARTICLE></DIV><DIV class="container"><ARTICLE class="col-md-10"><DIV id="inside-description"><A class="pointer" data-toggle="collapse" href="#collapseExample10" role="button" aria-expanded="false" aria-controls="collapseExample10"><H1 class="titleHP">DNA concentration measurement </H1><P class="inside-text"><SPAN style="font-weight: bold; color:black; font-size: 18px;">Goal: </SPAN> The concentration and purity of the extracted DNA are measured with the NanoDrop 2000 spectrophotometer (Thermo Fisher Scientific)</P></A></DIV><DIV class="collapse" id="collapseExample10"><DIV><P><B>Material:</B>
- Nanodrop 2000 
- lint-free laboratory wipe 
- Tip 2 μL 
- Pipette 2 μL 
- Samples <B>Methods:</B> 
The blank was made:
- using highly pure MilliQ water for ordered DNA sample
- using elution buffer from minipreps kits, for DNA previously extracted from minipreps 

DNA concentration measurement was performed according to the NanoDrop 2000 spectrophotometer protocol.

1. Pipet 1 – 2 μL of sample directly onto the measurement pedestal 
2. When the measurement is complete, raise the sampling arm and wipe the sample from both the upper and lower pedestals using a dry, lint-free laboratory wipe. <B>Note:</B>  Between measurements: Wipe the sample from both the upper and lower pedestals with a clean, dry, lint-free lab wipe.
 </P></DIV></DIV></ARTICLE></DIV><DIV class="container"><ARTICLE class="col-md-10"><DIV id="inside-description"><A class="pointer" data-toggle="collapse" href="#collapseExample11" role="button" aria-expanded="false" aria-controls="collapseExample11"><H1 class="titleHP">Escherichia coli growing culture</H1></A></DIV><DIV class="collapse" id="collapseExample11"><DIV><P><B>Material:</B>
- LB Broth 
- Agar powder 
- spectinomycin (50 μg/mL) 
- X-gal (40 μg/mL) </P></DIV></DIV></ARTICLE></DIV><DIV class="container"><ARTICLE class="col-md-10"><DIV id="inside-description"><A class="pointer" data-toggle="collapse" href="#collapseExample12" role="button" aria-expanded="false" aria-controls="collapseExample12"><H1 class="titleHP">Chlamydomonas reiinhardtii culture</H1></A></DIV><DIV class="collapse" id="collapseExample12"><DIV><P><B>Materials:</B>
- Tris-Acetate-Phosphate (TAP) medium 65 
- Agar (1.6 % m/V), 

Bold’s minimal media supplemented with 36 gL-1 NaCl
</P></DIV></DIV></ARTICLE></DIV><DIV class="container"><ARTICLE class="col-md-10"><DIV id="inside-description"><A class="pointer" data-toggle="collapse" href="#collapseExample13" role="button" aria-expanded="false" aria-controls="collapseExample13"><H1 class="titleHP">Preparation of DNA samples for sequencing</H1></A></DIV><DIV class="collapse" id="collapseExample13"><DIV><P>Preparation of DNA samples for sequencing  were carried out according to the manufacturer’s instructions : add 2µL of primer at 10µM and then the DNA and the water</P></DIV></DIV></ARTICLE></DIV><DIV><DIV class="container fixed-bottom backtotop" style="padding-bottom: 10px; padding-right:0px; position: fixed; left: 120px;"><DIV class="row text-right text-xs-right text-sm-right text-md-right  text-lg-right  "><DIV class="col-xs-4 col-sm-4 col-md-12 col-lg-12"><A class="y btn-sm" href="#backtotop" role="button" style=" color:white; border-radius: 10px; background-color:#22D66C; text-decoration:none; ">  Back to top</A></DIV></DIV></DIV><SECTION id="footer"><DIV class="container"><DIV class="row text-center text-xs-center text-sm-center text-md-right"><DIV class="col-xs-12 col-sm-3 col-md-3 col-lg-6"><H5 class="text-left" style="border: none;padding: 0px">iGEM Sorbonne U Paris</H5><P class="text-left" style="color:white; font-size:16px;"> 4 place Jussieu  75005 Paris France</P></DIV></DIV></DIV><DIV class="container"><P class="text-center" style="color: white; padding:20px 0 20px 0">Made in Paris with love © iGEM Sorbonne U Paris 2018</P></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>