<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Trieste_project_mainres"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Trieste/project/mainres">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Trieste/project/mainres&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Trieste/project/mainres&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Trieste/project/mainres&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Trieste/project/mainres" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Trieste/project/mainres</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="head"><TITLE>Team Trieste iGEM 2012</TITLE></DIV><DIV id="body"><DIV id="container"><DIV id="header"><DIV id="payoff">The Jolly JoCare<DIV>a safe probiotic platform for protein expression</DIV></DIV><UL id="navigation"><LI><A href="https://2012.igem.org/Team:Trieste">HOME</A></LI><LI><A href="https://2012.igem.org/Team:Trieste/team">TEAM</A></LI><LI><A href="https://2012.igem.org/Team:Trieste/team">Team</A></LI><LI><A href="https://2012.igem.org/Team:Trieste/City&amp;Lab">City&amp;Lab</A></LI><LI><A href="https://2012.igem.org/Team:Trieste/Team Gal">Team Gallery</A></LI></UL><LI><A href="https://2012.igem.org/Team:Trieste/project">PROJECT</A></LI><LI><A href="https://2012.igem.org/Team:Trieste/project">Abstract</A></LI><LI><A href="https://2012.igem.org/Team:Trieste/project/overview">Project Overview</A></LI><LI><A href="https://2012.igem.org/Team:Trieste/project/applications">Applications</A></LI><LI><A href="https://2012.igem.org/Team:Trieste/project/modeling">Modeling</A></LI><LI><A href="https://2012.igem.org/Team:Trieste/project/mainres">Main Results</A></LI><LI><A href="https://2012.igem.org/Team:Trieste/data">DATA</A></LI><LI><A href="https://2012.igem.org/Team:Trieste/parts">PARTS</A></LI><LI><A href="https://2012.igem.org/Team:Trieste/notebook">DIARY</A></LI><LI><A href="https://2012.igem.org/Team:Trieste/notebook">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:Trieste/protocols">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:Trieste/safety">SAFETY</A></LI><LI><A href="https://2012.igem.org/Team:Trieste/safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:Trieste/survey">Safety Survey</A></LI><LI><A href="http://www.icgeb.org/~bsafesrv/" target="_blank">ICGEB Safety Unit</A></LI><LI><A href="https://2012.igem.org/Team:Trieste/outreach">OUTREACH</A></LI><LI><A href="https://2012.igem.org/Team:Trieste/outreach">Human Practice</A></LI><LI><A href="https://2012.igem.org/Team:Trieste/outreach/press">Press</A></LI><LI><A href="https://2012.igem.org/Team:Trieste/extra">EXTRA</A></LI><LI><A href="https://2012.igem.org/Team:Trieste/extra">Attributions</A></LI><LI><A href="https://2012.igem.org/Team:Trieste/extra/ach">Achievements</A></LI><LI><A href="https://2012.igem.org/Team:Trieste/extra/aks">Acknowledgement</A></LI></DIV><DIV id="content"><H1 id="h1_lf" class="main_tit"><DIV>Main Results</DIV></H1><H1 id="h1_rt" class="main_tit"><DIV>More</DIV></H1><DIV id="box_main"><DIV class="box_contenuti"><H2> Results of Cumate Switch Regulation System</H2>
We cloned the CymR-SV5 tag transcriptional unit under the constitutive promoter J23100 (BBa_K875003).

The production of the protein CymR  was verified  through  the  Western Blot analysis (here below), using an anti-SV5 antibody.   Here  we can see one band of  almost 25.7 KDa, which corresponds  to the molecular weight of the protein CymR-SV5 tag.

The J23100-CymR-B0015 was then cloned upstream the T5 promoter-cumate operator (BBa_K875001) + GFP (I13504) .

In order  to verify the functionality of the system we test it using different concentrations of p-cumate. The cumate binds the CymR preventing its repression thus allowing the GFP expression.
<STRONG>In liquid assay:</STRONG>
We inoculated a 20ml culture. After overnight growth, we diluted the culture to O.D. <SUB>600</SUB> = 0.2. Then we aliquoted 200 μl in 8 replicates in a microtiter plate at different concentrations of cumate. The reading was performed in a monochromator at 485-510nm. 

Both graphs show the relation between p-cumate concentration and the GFP fluorescence intensity at different times.  In the first one, the modulation range is explored and it has been relevated that the maximum yield is reached at 30uM. In the second one the fluorescence of a culture is followed in the time and it has been noticed an interesting difference between the induced and not induced cultures. The data were normalized over the background given by the LB media.
<DIV class="notebook_section"><H2 class="notebook_title">Post European-Jamboree</H2><STRONG>CymR genomic integration</STRONG>
We have confirmed the good functioning of the cumate switch when all the components were present on the same plasmid. The next step, according  to our project, was the  integration  of the constitutively expressed CymR in the Nissle's chromosome. To do this we cloned the CymR ( BBa_K875003 ) into the  pUC18R6KT miniTn7-Gm  developed by iGEM11_ UPO-Sevilla (BioBrick K510012) and we transformed it  in Nissle cells together with the transposase coding plasmid  PTNS2 (gently provided by iGEM11_ UPO-Sevilla). 
To check the success of the recombination event, we performed a colony PCR  by using  the primers: glmFw (located at the 3’ of the glmS gene were the recombination occurs) and the Tn7Rev primer (located on the Tn7R end). Only in the case of integration we should  get  a 165 bp fragment. <P class="didascalia">Colony PCR in 2% agarose gel. Fifthteen colonies were checked by PCR and three of them were positives.</P></DIV><STRONG>In plate assay:</STRONG> 

We streaked the culture on LB plates containing different cumate concentrations.

We proved that the combination of CymR and T5 Cumate Operator  creates a strong, very stringent system for protein expression with a rapid reactivity.

The goal will be to validate the activity of this system as gene guard. The limiting point could be infact the poor expression of CymR that will be put in the bacterial genome in just two copies. Future work will confirm if two copies are sufficient for repressing the expression of the T5CumateOperator-toxin carried by the plasmid. 
<DIV class="notebook_section"><H2 class="notebook_title">Post European-Jamboree</H2><STRONG>Cuminaldehyde activation of cumate switch</STRONG>
Up to now all our tests were based on p-cumate according  to the literature <SPAN class="small">(Novel, Versatile, and Tightly Regulated Expression System for <I>Escherichia coli</I> Strains . Young J. Choi, Lyne Morel, Teffanie Le François Denis Bourque, Lucie Bourget, Denis Groleau, Bernard Massie and Carlos B. Míguez.  Applied and Environmental Microbiology, Aug. 2010, p. 5058–5066 Vol.76 doi:10.1128/AEM.00413-10).</SPAN>
But since it is known that the most abundant component in the cumin spice is the cuminaldehyde,  we  later decided to test also its activity in the regulation of the cumate switch. We confirmed that also the cuminaldehyde was able to activate the cumate switch, although in higher concentrations.<P class="didascalia">Cuminaldehyde test. We streaked the <I>E.coli</I> containing the plasmid with J23100-CymR-B0015+T5CumateOperator-I13504(GFP) on LB agar plates at different concentration of cuminaldehyde.</P><STRONG>In vivo experiment</STRONG>
We are planning to test our probiotic in vivo on BALB/c mice. Our aim is to verify : 

- if our probiotic is able to attach on the intestinal epithelium and create a stable niche;
- if our system is induced by cumate in vivo.
We are going to feed the mice every day  with 10<SUP>9</SUP> cells of our probiotic, expressing the GFP under the control of cumate switch, mixed with milk and PBS; the control mice will be fed only with milk and PBS. Some of the mice receiving the bacteria will be also fed with the iducer p-cumate in order to verify the functionality of the cumate switch in vivo.
We hope to present our results in Boston.</DIV><H2> Suicide System</H2>
Our first idea was to use the LL 37, the only cathelicidin-derived antimicrobial peptide found in humans. It was known that the LL 37 kills the bacteria, so we decided to use it as the toxin in our system.
As you can see in the pictures above this approach was unsuccessfully, so we thought that maybe LL 37 can not kill the bacteria if expressed inside them. So, we decided to use the LL 37 in another way, combining it with the T4holin from the Registry(BBa_K112000). The T4holin acts on the inner membrane, creating pores through which the LL 37 could reach the outer membrane and cause lysis.
This approach should be successful, but the T4Holin has the restriction sites (RFC 21) not compatible with the standard ones (RFC 10), so we had to transform the biobricks we needed into BglBricks according to the RFC 21. 
The rational of this construct is represented below:
Meanwhile, we took an optional way as a suicide system, using the <STRONG>Tse2</STRONG>, a toxin from the registry (BBa_K314200).
We cloned it downstream the T5Lac Operator Promoter and we tested it inducing and non-inducing the cells with IPTG.
The rational of this construct is represented below:
The M15 cells were transformed with our construct. Two positive colonies (colony n.11 and colony n.47), checked by PCR, were plated on +/-IPTG plates. The colony n.47 lives in the plate with -IPTG and dies in the +IPTG plate as expected.<DIV class="notebook_section"><H2 class="notebook_title">Post European-Jamboree</H2><STRONG>Suicide System</STRONG>
The Tse2 toxin was cloned, together with the constitutively expressed CymR, into the pMP220 low copy plasmid and upstream this construct we cloned the T5CumateOperator Promoter.
The rational of this construct is represented below:


The DH5-alpha bacterial cells were transformed with this construct and the positive colonies, checked by PCR, were plated with and without cumate. Their growth was also tested in liquid culture at different cumate concentrations (no cumate, 50uM, 100uM, 200uM), as you can see in graph below.
<P class="didascalia">After an over night liquid culture (LB broth) the O.D.<SUB>600</SUB> absorbance was recorded and the culture was diluted to O.D.<SUB>600</SUB>=0,1 and splitted and cultured in different cumate concentration conditions. The culture were induced with different cumate concentrations ( 50 µM, 100 µM, 200 µM ) and not induced. 

The O.D.600 was then recorded every hour.
</P></DIV><H2> Antibody</H2>
We produced an engeneered antibody (in different formats: SIP and scFv) using two different expression systems. In order to obtain a secreted version of the antibody, we cloned it under PelB leader sequence. Then, we fused the antibody in frame with LPP-OmaA fragment in order to display it extracellularly. All constructs were cloned downstream T5 Lac Operator.

We tested the expression of our antibodies by Western blotting.  

a) <STRONG>T5LacO-LPP-OmpA-scFv</STRONG><P>
The construct was tested in <I>E.coli</I> W3110 strain which was previously trasformed with p-REP 4 encoding for the Lac Repressor. The recombinant bacterial culture was induced at O.D.= 0.4 (2x108 bacterial cells/ml) with IPTG (1mM) at 37°C in shaker. 2ml of bacterial culture were centrifuged and the pellet was resuspended in 200μl of lysis buffer. The samples were then sonicated and boiled for 5 min. 10μl of lysates of induced, non-induced and non trasformed bacterial cultures were resolved on SDS-PAGE. The expression of fusion protein LPP-OmpA-scFv 54.6-His was tested by Western blotting with anti-6HIS antibodies
</P><P>
Expression of scFv 54.6 cloned in fusion with the LPP-OmpA leader sequence . Western blots of lysates of<I> E.coli</I> W3110 bacterial strain expressing the recombinant protein scFv 54.6 (45,19KDa), induced or non-induced with IPTG. The blot was reacted with the monoclonal F24-796 anti-6XHIS antibody.
</P><P>
OmpA-scFv fragment was cloned under Constitutive operator (BBa_J23100) and then tested in <I>E.coli</I> strain DH5alpha. The recombinant bacterial culture was inoculated over night at 37°C in shaker. 2ml of bacterial culture were centrifuged and the pellet was resuspended in 200μl of lysis buffer. The samples were then sonicated and boiled for 5 min. 10μl of lysates of induced, non-induced and non trasformed bacterial cultures were resolved on SDS-PAGE. The expression of fusion protein LPP-OmpA-scFv 54.6-His was tested by Western blotting with anti-6HIS antibodies. 
</P><P>
Expression of scFv 54.6 cloned in fusion with the LPP-OmpA leader sequence . Western blots of lysates(C= with Lpp-OmpA-scFv 54.6; WC= without LPP-OmpA-scFv 54.6) of <I>E.coli</I> DH5alpha bacterial strain expressing the recombinant protein scFv 54.6 (45,19KDa) . The blot was reacted with the monoclonal F24-796 anti-6XHIS antibody.
</P><P>
The localization of the scFv 54.6 fused with LPP-OmpA into the outer bacterial membrane was proved by performing an immunoflorescence assay on the bacterial strain DH5alpha expressing this construct.
5x108 recombinant bacterial cells were incubated with 5μl of Mouse Anti-6HIS (0,5μg/μl) for 3hs at 4°C followed by an incubation   with TRITC conjugated anti-mouse IgG .
</P><DIV class="notebook_section"><H2 class="notebook_title">Post European-Jamboree</H2><P class="didascalia"><I>E.coli</I> DeRed Express 63x (Positive Control)</P><P class="didascalia">[Left] DH5alpha strain expressing LPP-OmpA-scFv 54.6 stained with Mouse Anti-6HIS and Anti-mouse IgG TRITC conjugated 63x. [Right] DH5alpha strain expressing LPP-OmpA-scFv 54.6 stained with DAPI 63x. </P><P class="didascalia">DH5alpha strain expressing LPP-OmpA-scFv 54.6 stained with Mouse Anti-6HIS and Anti-mouse IgG TRITC conjugated 63x.</P><P class="didascalia">[Left] Wild-type DH5alpha strain stained with Mouse Anti-6HIS and Anti-mouse IgG TRITC conjugated 10x (Negative Control).
[Right] Wild-type DH5alpha strain stained with DAPI 10x (Negative Control).</P><P>
Immunoflorescence assay proves that the LPP-OmpA-scFv 54.6 is localized into the outer bacterial membrane and that the C-terminal containing the scFv is displayed extracellularly.
</P></DIV>
b) <STRONG>T5LacO-LPP-OmpA-SIP</STRONG><P>
The construct was tested in <I>E.coli</I> W3110 strain which was previously trasformed with p-REP 4 encoding for the Lac Repressor. The recombinant bacterial culture was induced at O.D.= 0.4 (2x108 bacterial cells/ml) with IPTG (1mM) at 37°C in shaker. 2ml of bacterial culture were centrifuged and the pellet was resuspended in 200μl of lysis buffer. The samples were then sonicated and boiled for 5 min. 10μl of lysates of induced, non-induced and non trasformed bacterial cultures were resolved on SDS-PAGE. The expression of fusion protein LPP-OmpA-scFv 54.6-His was tested by Western blotting with anti-6HIS antibodies. 
</P><P>
Expression of SIP 54.6 cloned in fusion with the LPP-OmpA leader sequence. Western blots of lysates of <I>E.coli</I> W3110 bacterial strain expressing the recombinant protein SIP 54.6 (60KDa), induced or non-induced with IPTG. The blot was reacted with the Affinity purified antibody goat anti-Human IgA(α).
</P>
c) <STRONG>T5LacO-PelB-scFv</STRONG><P>
The construct was tested in <I>E.coli</I> W3110 strain which was previously transformed with p-REP 4 encoding for the Lac Repressor. The recombinant bacterial colonies were induced at O.D.= 0.4 (2x108 bacterial cells/ml) with IPTG (1mM) at 37°C in shaker. 2ml of bacterial colture were centrifuged and the pellet was resuspended in 200μl of lysis buffer. The samples were then sonicated and boiled for 5 min. 10μl of lysates of induced, non-induced and non-trasformed bacterial cultures were resolved on SDS-PAGE. The expression of fusion protein scFv 54.6-His was tested by Western blotting with anti-6HIS antibodies 
</P><P>
Expression of scFv 54.6 cloned in fusion with the pelB leader sequence. Western blots of lysates (C= with pelB-scFv 54.6; WC= without pelB-scFv 54.6) of E.coli HB2151 bacterial strain expressing the recombinant protein scFv 54.6 (29,25KDa). The blot was reacted with the monoclonal F24-796 anti-6XHIS antibody.
</P>
d) <STRONG>T5LacO-PelB-SIP</STRONG><P>
The construct was tested in <I>E.coli </I>W3110 strain which was previously trasformed with p-REP 4 encoding for the T5 Lac Repressor. The recombinant bacterial colonies were induced at O.D.= 0.4 (2x108 bacterial cells/ml) with IPTG (1mM) at 37°C in shaker. 2ml of bacterial colture were centrifuged and the pellet was resuspended in 200μl of lysis buffer. The samples were then sonicated and boiled for 5 min. 10μl of lysates of induced, non-induced and non trasformed bacterial cultures were resolved on SDS-PAGE. The expression of protein SIP 54.6-His was tested by Western blotting with anti-6HIS antibodies.
</P><P>
Expression of SIP 54.6 cloned in fusion with the pelB leader sequence. Western blots of lysates (SN=supernatant; P=pellet) of <I>E.coli</I> W3110 bacterial strain expressing the recombinant protein SIP 54.6, induced or non-induced with IPTG. The blot was reacted with the monoclonal F24-796 anti-6XHIS antibody.
</P></DIV></DIV><DIV id="box_right"><UL id="sub_menu"><LI><A href="https://2012.igem.org/Team:Trieste/project">Abstract</A></LI><LI><A href="https://2012.igem.org/Team:Trieste/project/overview">Project Overview</A></LI><LI><A href="https://2012.igem.org/Team:Trieste/project/applications">Applications</A></LI><LI><A href="https://2012.igem.org/Team:Trieste/project/modeling">Modeling</A></LI><LI class="select"><A href="https://2012.igem.org/Team:Trieste/project/mainres">Main Results</A></LI></UL><DIV class="box_contacts"><H2>Contact us</H2><P>For other information, write to:</P><A href="mailto:igem2012@gmail.com" class="btn">igem2012@gmail.com</A><DIV class="social">
                    Follow us also:
			</DIV></DIV></DIV></DIV><DIV id="footer"><DIV id="box_footer">iGEM 2012 - For info: <A href="mailto:igem2012@gmail.com">igem2012@gmail.com</A></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Trieste/project/mainres">http://2012.igem.org/Team:Trieste/project/mainres</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Trieste/project/mainres" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Trieste/project/mainres" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Trieste/project/mainres&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Trieste/project/mainres&amp;oldid=288312" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>