<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Greece_Project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Greece/Project</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV style="display: none;" id="main-wraper"><NAV class="menu"><UL class="clearfix" id="list"><SPAN id="just_to_align"><LI style="width:" class="li_indiv"><A target="_self" href="https://2017.igem.org/Team:Greece/Project">PROJECT</A></LI><LI style="width:" class="li_indiv"><A target="_self" href="https://2017.igem.org/Team:Greece/Model">MODELING</A></LI><LI style="width:" class="li_indiv"><A target="_self" href="https://2017.igem.org/Team:Greece/human_practices">HUMAN PRACTICES</A></LI><LI style="width:" class="li_indiv"><A target="_self" href="https://2017.igem.org/Team:Greece/Achievements">ACHIEVEMENTS</A></LI><LI style="width:" class="li_indiv"><A target="_self" href="https://2017.igem.org/Team:Greece/Team">TEAM</A></LI></SPAN></UL></NAV></DIV><SVG style="display:none; position:absolute; z-index:1000;" id="svg_graphics_box" class="responsive_box" height="580" width="99%" viewbox="0 0 800 700"><A href="#" onmouseover="onHoverChanges( $(this).children('circle:first').attr('id') )" onmouseleave="onHoverHide($(this).children('circle:first').attr('id'))"><TEXT class="animate_text" id="t1" x="0" y="335" text-anchor="middle" fill="#000000"><TSPAN x="-230" dy="1.5em" onclick="displayMe('c_t1')" id="f_circle_text">DESCRIPTION</TSPAN></TEXT></A><A href="#" onmouseover="onHoverChanges( $(this).children('circle:first').attr('id') )" onmouseleave="onHoverHide($(this).children('circle:first').attr('id'))"><TEXT class="animate_text" id="t2" x="0" y="335" text-anchor="middle" fill="#000000"><TSPAN x="193" dy="1.5em" onclick="displayMe('c_t2')" id="s_circle_text">RESULTS</TSPAN></TEXT></A><A href="#" onmouseover="onHoverChanges( $(this).children('circle:first').attr('id') )" onmouseleave="onHoverHide($(this).children('circle:first').attr('id'))"><TEXT class="animate_text" id="t3" x="0" y="335" text-anchor="middle" fill="#000000"><TSPAN x="610" dy="1.5em" onclick="displayMe('c_t3')" id="t_circle_text">NOTEBOOK</TSPAN></TEXT></A><A href="#" onmouseover="onHoverChanges( $(this).children('circle:first').attr('id') )" onmouseleave="onHoverHide($(this).children('circle:first').attr('id'))"><TEXT class="animate_text" id="t4" x="0" y="335" text-anchor="middle" fill="#000000"><TSPAN x="1020" dy="1.5em" onclick="displayMe('c_t4')" id="fo_circle_text">DESIGN</TSPAN></TEXT></A></SVG><VIDEO id="video" autoplay="autoplay" muted="muted" preload="auto" loop="loop"><DIV id="display_box"><DIV id="ct1"><DIV id="Description" width="100%" height="100%" class="display_frame hide"><DIV class="alert info"><SPAN class="closebtn">×</SPAN>  
  Click on the toggle button to switch between the two description modes.
</DIV></DIV></DIV><DIV id="Results" width="100%" height="100%" class="display_frame hide"><DIV id="article_box"><ARTICLE id="static_content"><SECTION><ARTICLE><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>A therapeutic system needs to be characterized by two crucial features; safety and effectiveness. By giving both features extreme attention, we managed to develop a novel modus operandi, where both the <STRONG>delivery system</STRONG> efficiently targets only cancer cells and the therapeutic <STRONG>RNAi-based logic circuit</STRONG> only affects the specific cancer cell types. </P></DIV></SECTION></ARTICLE><ARTICLE><DIV style="text-align:center"><HEADER><STRONG class="sub_headers">Delivery System</STRONG></HEADER></DIV><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>When it comes to colorectal cancer, what better delivery system can one use than the bacteria, that already naturally dwell in the colon? E.coli transformed with a low-copy plasmid containing the part BBa_K1850011 adhere only to colorectal cancer cells and not onto healthy cells, according to iGEM Harvard 2015.</P><P>But, in order to improve the system Harvard worked on, we recreated it without the need to control the entire fim operon expression or burden the bacteria with an extra 8kb-plasmid. Finally, going one step further again, we tested the delivery system in a co-culture of bacteria and colorectal cancer cells and succeeded in optimizing the conditions needed for the system to work. </P></DIV></SECTION></ARTICLE><ARTICLE><DIV style="text-align: center"><HEADER><STRONG class="sub_headers">Agglutination Assay</STRONG></HEADER></DIV><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>In order to test our colorectal cancer-targeting delivery system, we tested two different strains of E.coli; fimE KO (JW4276-1) and fimH KO (JW4283-3), which were obtained from the Keio collection. The fimE KO is overproducing the type-1 pili, thus we expect them to show extensive adhesion to mannose residues on glycoproteins found on human epithelial cells. Type-1 pili adhesion relies on the fimH lectin which binds to mannosylated glycoproteins, therefore a fimH KO strain is expected to be unable to adhere to epithelial cells. In order to elicit bacterial binding to cancer cells, we transformed them with a plasmid containing the rhamnose inducible fimH 49 KO +RPMrel gene (BBa_K1850011). We then tested our bacteria for their ability to bind to yeast <I>S. cerevisiae,</I> which according to van Asbeck et al.  shows a high affinity to mannose binding lectins. The results demonstrated that, when mixed together with yeast, the fimE KO strains appeared to form clumps which precipitated to the bottom of the eppendorf tubes, whilst fimH KO and the fimH KO with the BBa_K1850011 construct did not show any clumps. </P><P>Picture 1. Results seen after 1 hour of mixing 500 ul of yeast and E.coli, both grown for 24 hours to saturation. Some minor precipitation has formed in all tubes due to bacterial cells sitting down due to lack of turbulence. * indicates induction with 0.05% rhamnose. </P><P>Since the pSB1C3 backbone, in which we received the part BBa_K1850011, is a high copy number plasmid, we wanted to avoid over-expression and the possibility of inclusion body formations by transferring the part to a low copy backbone (pSB1T3) obtained from the part BBa_J04450. This way, our transformed strain possessed both Kanamycin (due to the Keio mutation) and Tetracycline (from pSB1T3) resistance. These bacterial strains were grown in LB Medium with both kanamycin and tetracycline antibiotics and the saturation OD was half  (OD600  1.5) compared to the one of fimH KO and fimE KO strains (OD600 3). In order to have comparative results, we diluted the culture in half (OD600 1.5) and the results remained the same.</P></DIV></SECTION></ARTICLE><ARTICLE><DIV style="text-align: center"><HEADER><STRONG class="sub_headers">Confirmation of fimH 49 KO+ RPMrel expression in the fimH KO strain</STRONG></HEADER></DIV><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>To validate the expression of the BBa_K1850011 construct, we grew three fimH KO cultures; two were transformed with the RPMrel containing construct, one of which was induced with rhamnose and the last one was transformed with an RFP insert in the same vector as a negative control. Samples were collected at specific time points from each culture, sonicated,   centrifuged and finally we separated the supernatant that contains the correctly folded proteins from the pellet. We did a Bradford assay to determine the protein concentration of each supernatant. Afterwards, we run on an SDS-Page 25 ug of the total protein from each sample. The image below shows the SDS-Page, stained with Coomasie G250.</P><P>According to the literature, we expect the fimH protein to appear at 30 kDa. Because the pRha has been reported to be leaky, a low intensity band is most likely to appear even in the uninduced samples. We also want to prove that increasing the duration of Rhamnose induction will result in more fimH KO – RPMrel production. To do so, we performed an anti-His Western Blot taking advantage of the his-tag located in the C-terminus of our construct. The fimH KO - RFP sample was used as a negative control. The image below shows the results after ECL staining.</P><P>Samples taken at 4 timepoints show increasing expression of fimH in the induced culture. As expected, the uninduced culture appears to have a low intensity band at all timepoints due to the leakiness of the promoter. There is no band on the negative control sample. </P><P>Our next goal was to examine whether fimH KO – RPMrel precipitates after induction with Rhamnose. The formation of inclusion bodies containing unfolded protein was observed through a second western blot in which we noticed a very intense band at 30 kDa in the pellet as shown in the picture bellow. </P></DIV></SECTION></ARTICLE><ARTICLE><DIV style="text-align: center"><HEADER><STRONG class="sub_headers">Co-culture</STRONG></HEADER></DIV><SECTION class="sub_sections"><DIV style="text-align: justify;"><P> For the first time, instead of using indirect means such as dot blotting (iGEM Harvard 2015) or immunocytochemistry between cells and peptides (Kelly et al., 2003), we proved that the cancer-targeting adhesion system works, using direct means of co-culturing E.coli and Caco-2 cells (human epithelial colorectal adenocarcinoma cells). The RPMrel oligopeptide has been shown by Kelly et al. to bind to 5 specific cancer cell types (HT29, CaCo-2, RKO, SW480 and DLD-1), thus we worked with one of them; Caco-2. For this adhesion test we used the same strains as the agglutination assay. We, again, expected fimE to play the role of the positive control as Caco-2 cells do not lack the natural mannosylated glycoproteins found in epithelial cells. Also, we did not expect the fimH KO mutants to attach to the cells at all, whilst the fimH KO strain with the BBa_K1850011 construct in a low copy backbone was expected to bind. The co-culture protocol was a modified version of Tatsuno et al. and after several trials with various MOI, cell confluences and co-culture incubation times, we managed to standardize the procedure, where both fimE and fimH KOs perfectly served their roles as positive and negative controls respectively, whilst the fimH KOs with the fimH 49 KO+RPMrel clearly showed adhesion to the cancer cells.</P><P>The fimE KO strains show visible rings around the Caco-2 cells. In the plates were fimH KO bacteria were inserted, there were almost no bacteria found in the well even after the first washing steps of the immunocytochemistry. The fimH KO strains with the fimH 49KO+RPMrel gene, without rhamnose induction, showed little but clear attachment to the cells, but if induced with rhamnose for 3 hours, clumps of bacteria formed around the Caco-2 cells, which despite all the washing steps did not detach off the cells! Several pictures were taken from all around the 6-wells and all showed similar results to the ones shown.</P><P>Taking into account the results from the Agglutination assay, where the fimH KO with the BBa_K1850011 construct on a low copy backbone did not agglutinate to mannose (therefore at healthy epithelial cells), while at the same time our co-culture showed clear adhesion to Caco-2 cells. We are now confident that our targeted delivery system is completely functional!</P></DIV></SECTION></ARTICLE></SECTION></ARTICLE><ARTICLE><HEADER><STRONG class="sub_headers" style="font-size: 55px;">RNAi-based logic circuit</STRONG></HEADER><SECTION><ARTICLE><DIV style="text-align: center"><HEADER><STRONG class="sub_headers">Transfections</STRONG></HEADER></DIV><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>Lipofectamine 3000 (ThermoFischer Scientific) was used in experiments with the Caco-2, HEK-293 and A549 cell lines. Approximately 4x10<SUP>4</SUP> Caco-2 cells or 1x10<SUP>5</SUP> HEK-293 and A549 cells in 1 ml of high-glucose DMEM complete medium were seeded into each well of a 24-well and incubated for 24 hours. Transfections were performed according to the instructions of the manufacturer, using the ratios described in [<A href="’#ref6_results’">6</A>] for the different plasmids. Doxycycline was added to a final concentration of 1 ug/ml. After a 16-hour incubation, media containing the lipid transfection complexes were replaced with fresh DMEM media and incubated for 2 days before being analyzed for fluorescence.</P></DIV></SECTION><DIV style="text-align: center"><HEADER><STRONG class="sub_headers">FACS Analysis</STRONG></HEADER></DIV><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>A BD FACSCalibur analyzer was used for the flow cytometry experiments of the transfected cells. The cells were prepared by trypsinizing each well with 0,25% trypsin-EDTA, collecting the cells, centrifuging them, removing trypsin and resuspending the pellet in fresh PBS.</P><P>For each sample, we measured the percentage of DsRed expressing cells and their geometric mean relative fluorescence intensity compared to untransfected controls. We used a GFP expressing plasmid as a transfection control and measured the percentage of GFP expressing cells as well.</P><P>To account for difference in transfection efficiencies between the three cell lines we performed the following transformation, calculating the Cellular Fluorescence Intensity for each sample:<MATH display="block"><MROW><MI>C</MI><MI>F</MI><MI>I</MI><MO stretchy="false">(</MO><MI>s</MI><MI>a</MI><MI>m</MI><MI>p</MI><MI>l</MI><MI>e</MI><MO stretchy="false">)</MO><MO>=</MO><MI>D</MI><MI>s</MI><MI>Re</MI><MI>d</MI><MI>F</MI><MI>I</MI><MFRAC><MROW><MI>%</MI><MI>D</MI><MI>s</MI><MI>Re</MI><MI>d</MI><MO>+</MO></MROW><MROW><MI>%</MI><MI>G</MI><MI>F</MI><MI>P</MI><MO>+</MO></MROW></MFRAC></MROW></MATH>

						To account for differential promoter activity and leakage between our cell lines, so as to be able to compare our results and gauge the performance of our circuit across cell lines we performed an additional normalization step by dividing the CFI of each sample with the CFI of a control plasmid in that cell line, namely pCMV-DsRed-sfGFP-SV40, obtaining the Normalized Fluorescence Intensity, the measure we use for our circuit output.</P></DIV></SECTION><DIV style="text-align:center"><HEADER><STRONG class="sub_headers">Functionality of the Inversion Module</STRONG></HEADER></DIV><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>The double-inversion module necessary for our circuit function is actuated through a Tet-On system which translates the repression on rtTA caused by the “high markers” ,i.e. the upregulated miRNAs, to a reduced post-transcriptional repression on the output plasmid caused by the synthetic miRNA FF4, which is coded downstream of pTRE and therefore increased output. So, we begun by testing whether our Tet-On system along with our synthetic miRNA FF4 were functional in the absence of miRNA control on rtTA, therefore expecting higher circuit output in the Doxycycline uninduced cells.</P><P>As expected, we observe a 4-fold increase in circuit output in the uninduced cells, allowing us to proceed with our experiments.</P></DIV></SECTION><DIV style="text-align:center"><HEADER><STRONG class="sub_headers">Classification performance of various circuit topologies</STRONG></HEADER></DIV><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>We proceeded by closely examining the classification performance of three circuit topologies; A, B and C characterized by extensive overlap, between Caco-2, our cell line of interest, and HEK-293 and A549. </P></DIV></SECTION></ARTICLE><ARTICLE><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>In addition, we investigated the effect of altered ratios between the components of the inversion module and demonstrated that the system functions best on 1:1 ratio compared to the 2:1 used most often in Tet-On systems.</P></DIV></SECTION></ARTICLE></SECTION></ARTICLE><ARTICLE><SECTION><ARTICLE><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>As we can see our RNAi-based logic circuits are capable of actuating our pre-programmed response (fluorescence in this case) in a cell-type specific manner! Another striking result is that our model seems to be remarkably successful at predicting the classification performance of a given circuit!</P><P>Disclaimer: Due to the strict timetable, not all the experiments have been performed in biological triplicates.</P></DIV></SECTION></ARTICLE><DIV id="references"><ARTICLE><DIV style="text-align: center"><HEADER><STRONG class="sub_headers">References</STRONG></HEADER></DIV><SECTION style="text-align:justify" class="sub_sections">
					[<SPAN id="ref1_results">1</SPAN>] Kline, K. A., Fälker, S., Dahlberg, S., Normark, S., &amp; Henriques-Normark, B. (2009). Bacterial adhesins in host-microbe interactions. <I>Cell host &amp; microbe</I>, 5(6), 580-592.
					[<SPAN id="ref2_results">2</SPAN>] Van Asbeck, E. C., Hoepelman, A. I., Scharringa, J., Herpers, B. L., &amp; Verhoef, J. (2008). Mannose binding lectin plays a crucial role in innate immunity against yeast by enhanced complement activation and enhanced uptake of polymorphonuclear cells. <I>BMC microbiology, 8</I>
					[<SPAN id="ref3_results">3</SPAN>] Kelly, K. A., &amp; Jones, D. A. (2003). Isolation of a colon tumor specific binding peptide using phage display selection. Neoplasia, 5(5), 437-444.
					[<SPAN id="ref4_results">4</SPAN>] Sussman, M. (Ed.). (1997). Escherichia coli: mechanisms of virulence. Cambridge University Press.
					[<SPAN id="ref5_results">5</SPAN>] Tatsuno, I., Nagano, K., Taguchi, K., Rong, L., Mori, H., &amp; Sasakawa, C. (2003). Increased adherence to Caco-2 cells caused by disruption of the yhiE and yhiF genes in enterohemorrhagic Escherichia coli O157: H7. <I>Infection and immunity, 71</I>(5), 2598-2606.
					[<SPAN id="ref6_results">6</SPAN>] Xie, Z., Wroblewska, L., Prochazka, L., Weiss, R., &amp; Benenson, Y. (2011). Multi-input RNAi-based logic circuit for identification of specific cancer cells. <I>Science, 333</I>(6047), 1307-1311.</SECTION></ARTICLE></DIV></SECTION></ARTICLE></DIV></DIV><DIV id="ProjectDesign" width="100%" height="100%" class="display_frame hide"><DIV id="article_box"><ARTICLE id="static_content"><HEADER><STRONG class="sub_headers" style="font-size: 55px;">PROJECT DESIGN</STRONG></HEADER><SECTION><ARTICLE><DIV style="text-align:center"><HEADER><STRONG class="sub_headers">Overview</STRONG></HEADER></DIV><SECTION class="sub_sections"><DIV style="text-align: justify;"><P><I>“Design is the distinguishing activity of synthetic biology”</I></P><P>This paraphrased quote, originally coined by Herbert A. Simon to describe engineering, perfectly outlines the cornerstone of synthetic biology and the iGEM competition: the rational design of biological circuits by modular parts. Fueled by pure excitement for the promising and novel field of synthetic biology, we embarked on a multi-faceted journey into the world of iGEM and designed a bimodal project. The principal pillar of our efforts has been pANDORRA; a programmable AND OR RNAi Assembly platform engineered to optimize logic circuit design and implementation. We applied our modular assembly platform to build a multi-input RNAi based logic circuit to specifically target colorectal cancer cells. Adjuvantly, we concocted a bactofection system performing cell-specific adhesion and bacterial density dependent invasion and plasmid transference.</P></DIV></SECTION></ARTICLE><ARTICLE><DIV style="text-align: center"><HEADER><STRONG class="sub_headers">pANDORRA pipelinea re-invention of the engineering cycle</STRONG></HEADER></DIV><SECTION class="sub_sections"><DIV style="text-align: justify;"><SPAN><B>RNAi-based logic circuits</B></SPAN><P>We aimed to develop a fully-predictable regulatory program, exploiting the distributed cellular availability of specific molecular input to differentiate various cell types by the production of a protein output. Following the engineering cycle, as described in [<A href="#ref1_pd">1</A>], our first step was “Specification”. Our end goal at the beginning, was quite singular: create a molecular logic circuit, a biocomputer, that can trigger cell death or produce fluorescence when a certain expression profile is found in a cell. Before delving deeper into the inner working of our logic circuit design, let’s review two fundamental notions concerning the computing of such circuits:</P><UL><LI style="list-style: none;"><STRONG>1. The nature of the biological switches.</STRONG> Switches are the physical entities that implement a universal set of logic gates, thus enabling computation. A plethora of biomolecules can be utilized upon which to build switches. Between gene-based, RNA-based, protein-based etc. biological switches, we chose trans-acting RNA switches and specifically miRNAs. Thanks to their ability to regulate a large fraction of the human transcriptome and natural implementation NOR logic [<A href="#ref2_pd">2</A>] when multiple ones regulate the same gene, miRNAs have been extensively studied in mammalian systems. Moreover, they are excellent internal inputs since miRNAs are found to play crucial roles in the disease spectrum [<A href="#ref3_pd">3</A>].</LI><LI style="list-style: none"><STRONG>2. The rudimentary circuit abstraction.</STRONG> In order for a miRNA-based cell profiling to function, in accordance with seminal papers of the field [<A href="#ref2_pd">2</A>, <A href="#ref4_pd">4-7</A>], a number of miRNA markers are selected and the circuit computes an AND gate  with these markers in order to perform a classification task. Since miRNAs are molecules exerting solely inhibitory effects on expression, a repressor is required to repress the output, “linking” the high miRNA-markers that inhibit (directly or indirectly) the production of the repressor and the low miRNA-markers that typically target the output gene. As a result, we needed to select the nature of the repressor, with options including a transcriptional one such as LacI, a post-transcriptional one like a synthetic miRNA or both, as well as the in-depth topology, by determining the layers of the circuit (two or more). More elaborate architectures can be employed by utilizing this basic architecture.</LI></UL><P>In conclusion, we set our “classification” task as follows:</P></DIV></SECTION></ARTICLE><ARTICLE><SECTION class="sub_sections"><DIV style="text-align: justify;"><SPAN><STRONG>Produce fluorescence or induce apoptosis when a specific miRNA expression profile* is found in colorectal cancer cells (Caco-2).</STRONG></SPAN><P>*The miRNA expression profile should be predetermined in order to discriminate Caco-2 cells from healthy cells.</P></DIV></SECTION></ARTICLE><ARTICLE><SECTION class="sub_sections"><DIV style="text-align: justify;"><SPAN><STRONG>Circuit topology optimization &amp; miRNA Boolean expression selection</STRONG></SPAN><P>Εxperimental classiﬁers have been designed by trial-and-error, by tweaking the parameters of the network in order to identify the optimal architecture and Boolean expression, or in a semi-manual fashion, via ranking and manual selection of differentially expressed miRNAs retrieved from databases produced by large scale studies. [<A href="#ref8_pd">8</A>] There are several constraints that dictated these approaches, for example the inadequacy of basic building blocks to better assemble and characterize various mammalian classifiers and the lack of powerful tools to automate logic circuit design based on miRNA molecular switches. Although daunting as a task, we set off to address both of these issues by:</P><P>-Creating pANDORRA (programmable AND OR RNAi Assembly) in order to produce a large number of mammalian parts, which can be used for a bottom-up construction of any conceivable logic circuit based on universal logic gates.</P><P>-Increasing the functionality and directionality of our assembly process by following a step-by-step cloning workflow and using standardized primers or overhangs after the integration of extensive technical feedback received by Stamatis Damalas. Click <A href="’https://2017.igem.org/Team:Greece/HP/Gold_Integrated’">here</A> to check it out.</P><P>-Employing a computational framework to facilitate the selection of circuit inputs (miRNAs), form the logic expression and simulate optimal circuit-performance in different topologies. Check out <A href="https://2017.igem.org/Team:Greece/RNAi_Classifier_Design">our model</A>.
						</P><P>As in mature engineering clades, models simplify the real work and facilitate design in ideal conditions. Other models then evaluate the proposed designs more thoroughly and either send the designers back to the drawing board or to the test bench; that is the essence of our design approach: a progressive dance where modelling and design come ever closer together.</P></DIV></SECTION></ARTICLE></SECTION></ARTICLE><ARTICLE><HEADER><STRONG class="sub_headers" style="font-size: 55px;">A. THE BASIC COMPONENTS OF THE CIRCUIT</STRONG></HEADER><SECTION><ARTICLE><DIV style="text-align:left"><HEADER><STRONG class="sub_headers">Wet Lab Input</STRONG></HEADER></DIV><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>The sequences of the plasmids used for the construction of the cell-type classifier between HeLa cells and healthy cells described in [<A href="#ref4_pd">4</A>] were kindly provided by Prof. Xie. We've used these plasmids as a starting point to “detach” three different mammalian promoters, four different protein coding regions and two polyA signals-terminators, codon-optimize them in order to make them BioBrick compatible and order them for de novo gene synthesis. Analytically:</P><UL><LI style="list-style: none"><STRONG>1. Promoters</STRONG></LI><LI style="list-style: none">-CMV (strong constitutive promoter from the human cytomegalovirus)</LI><LI style="list-style: none">-TRE (inducible tetracycline response element promoter)</LI><LI style="list-style: none">-CAGop (strong hybrid CAG promoter followed by an intron with two LacO sites [<A href="#ref2_pd">2</A>]</LI></UL><LI style="list-style: none"><STRONG>2. Protein Coding Regions</STRONG></LI><LI style="list-style: none">-rtTA (reverse tetracycline-controlled transactivator by fusing rTetR with VP16, utilized in Tet-On systems) [<A href="#ref10_pd">10</A>]</LI><LI style="list-style: none">-LacI (DNA-binding transcription factor that binds to LacO sites) </LI><LI style="list-style: none"><STRONG>- Transcriptional repressor </STRONG></LI><LI style="list-style: none">-DsRed (Discosoma sp. red fluorescent protein)</LI><LI style="list-style: none">-Apoptin </LI><LI style="list-style: none"><STRONG>3.  polyA signals &amp; terminators </STRONG> (include the motif AAUAAA which promotes both polyadenylation and termination)
								</LI><LI style="list-style: none">-SV40 polyA</LI><LI style="list-style: none">-rbGlob polyA*</LI><P>*Next to the rbGlob polyA (upstream) there is a sequence coding for a synthetic miRNA, targeting a region of firefly luciferase [<A href="#ref11_pd">11</A>]. It is a <STRONG>post-transcriptional repressor</STRONG>, named FF4.</P><P>As a result, using these <A href="https://2017.igem.org/Team:Greece/Basic_Part">Basic Parts</A>, a large number of circuit topologies can be envisioned.</P></DIV></SECTION></ARTICLE><ARTICLE><DIV style="text-align: left"><HEADER><STRONG class="sub_headers">Dry Lab Input</STRONG></HEADER></DIV><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>There is an ongoing debate regarding the choice of using transcriptional repressors (LacI) that act as a roadblock to RNA Polymerase or posttranscriptional repressors (FF4 and other synthetic intronic miRNAs) that act by a completely different mechanism by mRNA degradation. Experimental evidence [<A href="#ref4_pd">4</A>] support the combination of transcriptional and post-trascriptional repression as a boost in circuit performance. However, we wanted to proceed to a more detailed analysis regarding the selection of an appropriate repressor. You can check the results of this analysis <A href="https://2017.igem.org/Team:Greece/RNAi%20Classifier%20Design">here</A>.</P></DIV></SECTION></ARTICLE><ARTICLE><DIV style="text-align: left"><HEADER><STRONG class="sub_headers">Integrated Design Output</STRONG></HEADER></DIV><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>Thanks to this combined analysis, we have designed the Biobricks to use in the experiments/simulations and made a strong case for the case-by-case use of a transcriptional or post-trascriptional repressor according to the miRNA dataset.</P></DIV></SECTION></ARTICLE></SECTION></ARTICLE><ARTICLE><HEADER><STRONG class="sub_headers" style="font-size: 55px;">B. MODULAR CIRCUIT ASSEMBLY &amp; OPTIMIZATION ALGORITHM</STRONG></HEADER><SECTION><ARTICLE><DIV style="text-align:left"><HEADER><STRONG class="sub_headers">Wet Lab Input</STRONG></HEADER></DIV><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>We envisage pANDORRA as a cloning toolkit with standard parts (designed in the previous step) that can be easily applicable to the construction of a wide range of transcriptional/post-transcriptional synthetic circuits. In this endeavor, we focused on two aspects, interoperability and modularity, by <A href="https://2017.igem.org/Team:Greece/HP/Gold_Integrated">integrating extensive technical feedback from experts in the field.</A></P><P>According to Benenson et al., 2012 [<A href="#ref9_pd">9</A>], an effective molecular switch or gate is characterized by:</P><UL><LI style="list-style: disc">The existence of a robust digital regime (that is, input levels that produce either a very low or a very high (saturated) output).</LI><LI style="list-style: disc">Gate scalability, which is the capacity to receive an increasing number of inputs without dramatic design alterations.</LI><LI style="list-style: disc">Composability, which is the capacity to operate together with other gates in parallel and/or in cascades in a predictable manner.</LI></UL><P>In order to add the aforementioned features to our toolkit and to aid in the cloning process, we created various Composite Parts, that can be used to create multi-layered classifiers. At first glance, the constructs have the following general structure:</P><P>Promoter + Protein Coding Region + <A href="http://parts.igem.org/Part:BBa_K515105">BBa_K515105</A> + polyA signal &amp; terminator</P><P>Analytically, <A href="http://parts.igem.org/Part:BBa_K515105">BBa_K515105</A> consists of superfolder GFP (sfGFP), a very brightly fluorescent protein under the control of the bacterial constitutive promoter J23100 and is used as a reporter to simplify the validation process during cloning. </P><P>BBa_K515105 is flanked by two recognition sites for BbsI, a type IIS restriction enzyme and two annealing sites for a universal M13 forward &amp; reverse primer. As type IIS restriction enzymes recognize asymmetric DNA sequences and cleave outside of their recognition sequence, they are central to our approach for fusing miRNA target sequences into the 3’-untranslated region, as described in the next section. Examples of these constructs include:</P><P>Notably, the output module can include either a coding region for a fluorescent protein, DsRed or a toxin, Apoptin (<A href="http://parts.igem.org/Part:BBa_K1061001">BBa_K1061001</A>), a selective cancer cell killer derived from the Chicken Anemia Virus (CAV) known to cause p53-independent apoptosis in more than 70 human cancer cell lines while leaving normal cells unharmed [<A href="#ref12_pd">12</A>]. The use of Apoptin instead of hbax [<A href="#ref4_pd">4</A>] was a change incorporated for increased cytotoxicity and an additional safety fail-safe after discussing with <A href="https://2017.igem.org/Team:Greece/HP/Gold_Integrated">Prof. JD Keasling and interpreting OSIRIS results.</A></P><P>The aforementioned Biobricks can be used to fuse any desired complementary miRNA binding site (target tandem repeats) into the 3’-untranslated region, between the BbsI restriction sites hardcoded into the coding and the terminator sequences to control the expression of each module by specific miRNAs. This process has the following requirements:</P><UL><LI style="list-style:none">(1) Composite Parts in the form of Promoter + Protein Coding Region + <A href="http://parts.igem.org/Part:BBa_K515105">BBa_K515105</A>  + polyA signal &amp; terminator, BioBrick compatible format</LI><LI style="list-style:none">(2) A number of repeated miRNA binding sites in BioBrick compatible format</LI><LI style="list-style:none">(3) A set of standarized Extended Primers which are part of the MetaBrick Platform [<A href="#ref13_pd">13</A>] that incorporate BsaI restriction sites to the Prefix-Suffix.</LI></UL><P>If the aforementioned requirements are fulfilled, then the following method is followed:</P><UL><LI style="list-style:none">(1) PCR amplification of the miRNA binding sites BioBricks</LI><LI style="list-style:none">(2) Digestion of the  Promoter + Protein Coding Region + <A href="http://parts.igem.org/Part:BBa_K515105">BBa_K515105</A> + polyA signal &amp; terminator BioBricks with BbsI</LI><LI style="list-style:none">(3) Digestion of the PCR amplified miRNA binding sites BioBricks with BsaI</LI><LI style="list-style:none">(4) The digested products have compatible sticky ends. As a result, one example of the final constructs is:</LI></UL></DIV></SECTION></ARTICLE><ARTICLE><DIV style="text-align: left"><HEADER><STRONG class="sub_headers">Dry Lab Input</STRONG></HEADER></DIV><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>The multiple architectures that emerge from the pANDORRA toolkit can be evaluated by using the <A href="https://2017.igem.org/Team:Greece/RNAi%20Classifier%20Design">RNAi classifier design model</A> in order to recognize the optimal one for every different classification task, for example for classifying Caco-2 cells. Moreover, in our model, the number of repeats for miRNA binding sites is evaluated. Another point that gets elucidated is the position of the TFF4 binding site. In our models, it is adjacent to the polyA terminator, as Haefliger et al., 2016 [<A href="#ref14_pd">14</A>] showed that when the TFF4 is upstream of other binding sites, the downstream binding sites for other miRNAs are not affected by their miRNA mimics and thus optimal knockdown efficiency isn’t observed. You can check the results of this process <A href="https://2017.igem.org/Team:Greece/RNAi%20Classifier%20Design">here</A>.</P></DIV></SECTION></ARTICLE><ARTICLE><DIV style="text-align: left"><HEADER><STRONG class="sub_headers">Integrated Design Output</STRONG></HEADER></DIV><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>By this back-and-forth approach, the optimal classifier can be computationally predicted and assembled from compartmentalized modules. The architectures that emerged through this exhaustive analysis have been characterized in 3 cell lines, Caco-2, HEK-293, A549. You can check the results of this process <A href="https://2017.igem.org/Team:Greece/Results">here</A>.</P></DIV></SECTION></ARTICLE><ARTICLE><DIV style="text-align: center"><HEADER><STRONG class="sub_headers">Cancer-targeting and invasion module</STRONG></HEADER></DIV><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>Type I pilli, surface rod-shaped organelles of 7nm in diameter and 1μm in length, are the best studied system of bacterial adhesion [<A href="#ref15_pd">15</A>].They are heteropolymers of four proteins with FimA being the main structural protein  of the pilli, which polymerizes approximately 1000 times to form a right-handed helix that constitutes the main axis of the structure and includes smaller concentrations of FimG, FimF and FimH [<A href="#ref16_pd">16-18</A>]. FimH is the functional component of the structure as it alone confers the ability to bind to a-D-mannose of various eukaryotic cells and is located at the tip and the shafts of the pilus, whereas FimF and FimG seem to be responsible for docking  FimH to FimA [<A href="#ref19_pd">19-20</A>]. To achieve selective adhesion to colorectal cancer cells using type I pilli we need to disrupt their natural ability to bind to a-D-mannose and introduce a mechanism to facilitate adhesion to CRC cells by a mannose-independent mechanism. A mutation in the 49th amino acid of FimH has been demonstrated to completely abolish mannose binding [<A href="#ref21_pd">21</A>]. In addition, a small peptide called RPMrel has been identified through phage display assays due to its ability to bind to five different colorectal cancer cell lines as well as cancerous tissues obtained by biopsies and not to other kinds of cancer [<A href="#ref22_pd">22</A>].  Taken together these two modifications perform both the functions specified for CRC selective binding and have been successfully used by previous iGEM teams, iGEM Harvard 2015 and iGEM Ankara 2016 to that end. We employed the same part <A href="’http://parts.igem.org/Part:BBa_K1850011’">BBa_K1850011</A> that was submitted by iGEM Harvard 2015, in a fimH KO strain. Having achieved selective adhesion to colorectal cancer cells we move on to the second half of our device, internalization and transference of genetic material. Strains of the bacteria E. coli can be modified so that they will express two key proteins : invasin and listeriolysin O [<A href="#ref23_pd">23-24</A>]. Invasin gives the bacteria, the ability to enter epithelial and other non phagocytic cells [<A href="#ref25_pd">25-26</A>]. Listeriolysin O, on the other hand, has to do with what happens to the bacteria after they enter the target-cell. This particular protein allows the bacteria to free themselves of the vesicle that was used for their phagocytosis, without damaging the plasmatic/cell membrane of the target-cell. This happens due to the low pH of the vesicle (~5.9-6) that is also the optimal pH range for the protein listeriolysin [<A href="#ref27_pd">27-28</A>]. The modified strains of <I>E. coli</I>, through expressing these proteins are able to not only enter non phagocytic target-cells that express b1-integrins but also to transfer  their load to them, through escaping the phagolysosome [<A href="#ref23_pd">23</A>]. Finally, we aim to put invasin and listeriolysin O under quorum control, through the use of the <I>lux</I> genetic circuit of <I>Vibrio ﬁscheri</I>, as this operon has been utilized to achieve cell-density dependent invasion. During our communication with the safety committee, we were extremely glad to hear that we submitted a thorough and analytical Check In form and the Committee Members advised us to focus on the safe implementation of our classifier module and then consider a transfer method. As a result, we switched our focus and put great effort to characterize the components of our classifier to demonstrate the while integrating feedback from the scientific community about the health risks of our conceptual anticancer agent, formulating what we term, a <STRONG>5-STAR security system</STRONG>, which represents the culmination of our proposed modifications:</P><STRONG>
						Level 1:
						Bacteria preferentially colonize cancer tissue compared to the adjacent healthy
						Level 2:
						Bacteria are genetically modified to express a mutated fimH, carrying a colorectal cancer-binding peptide and a point-mutation abolishing its natural substrate binding affinity.
						Level 3:
						Bacteria express the invasion mechanism under the control of a lux operon, in order to achieve cell-density dependent invasion
						Level 4:
						The classifier circuit, transferred by the bacteria for the final bactofection goal, is meticulously designed in order to elicit a actuation (e.g. apoptosis) selectively in cancer cells
						Level 5:
						The therapeutic protein output of our classifier is Apoptin, which causes p53-independent apoptosis in more than 70 human cancer cell lines while leaving normal cells unharmed [<A href="#ref15_pd">12</A>]</STRONG></DIV></SECTION></ARTICLE><DIV id="references"><ARTICLE><DIV style="text-align: left"><HEADER><STRONG class="sub_headers">References</STRONG></HEADER></DIV><SECTION style="text-align:justify" class="sub_sections">
					[<SPAN id="ref1_pd">1</SPAN>] Kitney, R., Calvert, J., Challis, R., Cooper, J., Elfick, A., Freemont, P. S., ... &amp; Paterson, L. (2009). Synthetic Biology: scope, applications and implications. London: The Royal Academy of Engineering.
					[<SPAN id="ref2_pd">2</SPAN>] Rinaudo, K., Bleris, L., Maddamsetti, R., Subramanian, S., Weiss, R., &amp; Benenson, Y. (2007). A universal RNAi-based logic evaluator that operates in mammalian cells. Nature biotechnology, 25(7), 795-801.
					[<SPAN id="ref3_pd">3</SPAN>] Soifer, H. S., Rossi, J. J., &amp; Sætrom, P. (2007). MicroRNAs in disease and potential therapeutic applications. Molecular therapy, 15(12), 2070-2079.
					[<SPAN id="ref4_pd">4</SPAN>] Xie, Z., Wroblewska, L., Prochazka, L., Weiss, R., &amp; Benenson, Y. (2011). Multi-input RNAi-based logic circuit for identification of specific cancer cells. Science, 333(6047), 1307-1311.
					[<SPAN id="ref5_pd">5</SPAN>] Miki, K., Endo, K., Takahashi, S., Funakoshi, S., Takei, I., Katayama, S., ... &amp; Okubo, C. (2015). Efficient detection and purification of cell populations using synthetic microRNA switches. Cell Stem Cell, 16(6), 699-711.
					[<SPAN id="ref6_pd">6</SPAN>] Li, Y., Jiang, Y., Chen, H., Liao, W., Li, Z., Weiss, R., &amp; Xie, Z. (2015). Modular construction of mammalian gene circuits using TALE transcriptional repressors. Nature chemical biology, 11(3), 207-213.
					[<SPAN id="ref7_pd">7</SPAN>] Sayeg, M.K., Weinberg, B.H., Cha, S.S., Goodloe, M., Wong, W.W., and Han, X. (2015). Rationally designed microRNA-based genetic classiﬁers target speciﬁc neurons in the brain. ACS Synth. Biol. 4, 788–795.
					[<SPAN id="ref8_pd">8</SPAN>] Landgraf, P., Rusu, M., Sheridan, R., Sewer, A., Iovino, N., Aravin, A., ... &amp; Lin, C. (2007). A mammalian microRNA expression atlas based on small RNA library sequencing. Cell, 129(7), 1401-1414.				[<SPAN id="ref9_pd">9</SPAN>] Benenson, Y. (2012). Biomolecular computing systems: principles, progress and potential. Nature Reviews Genetics, 13(7), 455-468.
					[<SPAN id="">10</SPAN>] Gossen, M., Freundlieb, S., Bender, G., Muller, G., Hillen, W., &amp; Bujard, H. (1995). Transcriptional activation by tetracyclines in mammalian cells. Science-AAAS-Weekly Paper Edition, 268(5218), 1766-1768.
					[<SPAN id="ref11_pd">11</SPAN>] Leisner, M., Bleris, L., Lohmueller, J., Xie, Z., &amp; Benenson, Y. (2010). Rationally designed logic integration of regulatory signals in mammalian cells. Nature nanotechnology, 5(9), 666-670.
					[<SPAN id="ref12_pd">12</SPAN>] Backendorf, C., Visser, A. E., De Boer, A. G., Zimmerman, R., Visser, M., Voskamp, P., ... &amp; Noteborn, M. (2008). Apoptin: therapeutic potential of an early sensor of carcinogenic transformation. Annu. Rev. Pharmacol. Toxicol., 48, 143-169.
					[<SPAN id="ref13_pd">13</SPAN>] Damalas, S. (2017, June 9). The MetaBrick platform for DNA manipulation and standardization. Bridging Synthetic Biology standards for optimized interoperability. 
					[<SPAN id="ref14_pd">14</SPAN>] Haefliger, B., Prochazka, L., Angelici, B., &amp; Benenson, Y. (2016). Precision multidimensional assay for high-throughput microRNA drug discovery. Nature communications, 7.
					[<SPAN id="ref15_pd">15</SPAN>] Stentebjerg-Olesen, B., Chakraborty, T., &amp; Klemm, P. (1999). Type 1 Fimbriation and Phase Switching in a Natural Escherichia coli fimB Null Strain, Nissle 1917. Journal of bacteriology, 181(24), 7470-7478.
					[<SPAN id="ref16_pd">16</SPAN>] Brinton, C. C. (1965). The structure, function, synthesis and genetic control of bacterial pili and a molecular model for DNA and RNA transport in gram negative bacteria. Transactions of the New York Academy of Sciences, 27(8 Series II), 1003-1054. 
					[<SPAN id="ref17_pd">17</SPAN>] Klemm, P., &amp; Christiansen, G. (1987). Three fim genes required for the regulation of length and mediation of adhesion of Escherichia coli type 1 fimbriae. Molecular and General Genetics MGG, 208(3), 439-445. 
					[<SPAN id="ref18_pd">18</SPAN>] Krogfelt, K. A., &amp; Klemm, P. (1988). Investigation of minor components of Escherichia coli type 1 fimbriae: protein chemical and immunological aspects. Microbial pathogenesis, 4(3), 231-238. 
					[<SPAN id="ref19_pd">19</SPAN>] Jones, C. H., Pinkner, J. S., Roth, R., Heuser, J., Nicholes, A. V., Abraham, S. N., &amp; Hultgren, S. J. (1995). FimH adhesin of type 1 pili is assembled into a fibrillar tip structure in the Enterobacteriaceae. Proceedings of the National Academy of Sciences, 92(6), 2081-2085.
					[<SPAN id="ref20_pd">20</SPAN>] Krogfelt, K. A., Bergmans, H., &amp; Klemm, P. (1990). Direct evidence that the FimH protein is the mannose-specific adhesin of Escherichia coli type 1 fimbriae. Infection and immunity, 58(6), 1995-1998.
					[<SPAN id="ref21_pd">21</SPAN>] Schembri, M. A., Sokurenko, E. V., &amp; Klemm, P. (2000). Functional flexibility of the FimH adhesin: insights from a random mutant library. Infection and immunity, 68(5), 2638-2646.
					[<SPAN id="ref22_pd">22</SPAN>] Kelly, K. A., &amp; Jones, D. A. (2003). Isolation of a colon tumor specific binding peptide using phage display selection. Neoplasia, 5(5), 437-444.
					[<SPAN id="ref23_pd">23</SPAN>] Fajac, I., Grosse, S., Collombet, J. M., Thevenot, G., Goussard, S., Danel, C., &amp; Grillot-Courvalin, C. (2004). Recombinant Escherichia coli as a gene delivery vector into airway epithelial cells. Journal of controlled release, 97(2), 371-381.
					[<SPAN id="ref24_pd">24</SPAN>] Grillot-Courvalin, C., Goussard, S., Huetz, F., Ojcius, D. M., &amp; Courvalin, P. (1998). Functional gene transfer from intracellular bacteria to mammalian cells. Nature biotechnology, 16(9), 862-866.
					[<SPAN id="ref25_pd">25</SPAN>] Isberg, R. R., Voorhis, D. L., &amp; Falkow, S. (1987). Identification of invasin: a protein that allows enteric bacteria to penetrate cultured mammalian cells. Cell, 50(5), 769-778. 
					[<SPAN id="ref26_pd">26</SPAN>] Pepe, J. C., &amp; Miller, V. L. (1993). Yersinia enterocolitica invasin: a primary role in the initiation of infection. Proceedings of the National Academy of Sciences, 90(14), 6473-6477. 
					[<SPAN id="ref27_pd">27</SPAN>] McKAY, D. B., &amp; Lu, C. Y. (1991). Listeriolysin as a virulence factor in Listeria monocytogenes infection of neonatal mice and murine decidual tissue. Infection and immunity, 59(11), 4286-4290. 
					[<SPAN id="ref28_pd">28</SPAN>] Dramsi, S., &amp; Cossart, P. (2002). Listeriolysin O. The Journal of cell biology, 156(6), 943-946.</SECTION></ARTICLE></DIV></SECTION></ARTICLE></DIV></DIV></DIV></VIDEO></DIV></DIV></DIV></DIV></DIV></BODY></HTML>