Team:Greece/Achievements
PROJECT
MODELING
HUMAN PRACTICES
ACHIEVEMENTS
TEAM
MEDAL
CRITERIA
COLLABORATIONS
PARTS
BASIC
COMPOSITE
COLLECTIONS
INTERLAB
Bronze
Silver
Gold
Compete: We successfully registered in March, had a great and educational summer and are looking forward to present our work at the Giant Jamboree and meet the other teams.
Deliverables: Completed all of competition’s deliverables.
We created our wiki and documented our entire project.
We plan to present our poster and our project in the Giant Jamboree.
We completed all of the safety forms as well as the judging form.
We created and documented Registry pages for the Parts we made.
We submitted DNA samples of our Parts.
Attributions: We presented our project’s attributions in order to give credit to everyone who offered their support, be it financial assistance, supplying lab materials or invaluable advice.
Contribution: We participated in this year’s Interlab study. You can read all about our findings here .
Validated Part: We validated the following parts
BBa_K2364021
BBa_K2364025
BBa_K2364027
Collaborations: Throughout our iGEM journey, we collaborated with 6 teams and documented our give and take relationship on our designated Collaborations page . In addition, we engaged every iGEM team to participate in our OSIRIS protocol.
Human Practices: We conducted an excessive risk assessment analysis of our project, received biosafety training, reached out to the international scientific community to receive feedback and reflected on our project’s potential impact on cancer treatment through a short comic strip.
All information regarding the aforementioned efforts is documented here .
Integrated Human Practices: We created a modular protocol (OSIRIS) that can be applied on any project, aiming to get quantitative feedback from experts from all over the world and analyzed the results to identify and tackle our design’s shortcomings. Multiple points from experts were finally integrated into our project.
You can read our documentation regarding this long and fruitful process here .
Our OSIRIS protocol can be found here .
Model our project: The entirety of our classifier circuit design, ranging from miRNA selection to circuit topology, was based on insights we gained from our modeling.
Furthermore, we designed our experimental protocols for bacterial invasion and  transfection using data obtained from our models. You can read all about our models and how we integrated them to our design .
Demonstrate your project works: In our proof of concept experiments, we transfected Caco-2, the cell line that our logic circuit classifies, along with HEK-293 and A549 as controls with our RNAi-based classifier and demonstrated that our circuit can indeed perform identification and subsequent actuation of the pre-programmed response. In addition, we successfully utilized our engineered bacteria to achieve selective adhesion to a colorectal cancer cell line. You can read more about our results here .
×
INTERACTIVITY CONTROLS
Move the graph by clicking and dragging in a blank area
Hover on the nodes to see the team's name
Click on the rectangular nodes for more information on the collaboration project
Click and drag on the nodes to make our jelly fish wiggle! ☺
POSTCARD COLLABORATION
iGEM Team Dusseldorf-Cologne
In 2016, iGEM team of Duesseldorf launched a campaign initiative to promote synthetic biology to local communities through self-designed postcards. This year, team Duesseldorf-Cologne, expanding on the idea, reached out to all European iGEM teams to collaborate, design and exchange synthetic biology inspired postcards to further engage the public.
Overall, 28 teams participated in the postcard collaboration. All teams sent and received imaginative postcards, that teased their respective projects.
Our team’s design showcases all aspects of our project, from the various dry lab modeling tasks to the OSIRIS protocol developed to integrate human practices to our work. The design reconciles the minimal aesthetic with the hectic nature of the iGEM journey and encircles -pun intended- all these aspects with our logo, our team’s emblem.
We shared the fruits of this collaboration with the public during the TeamFair 5.0 event that took place last October in our university campus. You can read more about our public engagement initiatives here .
HUMAN PRACTICES COLLABORATIONS
ham
COLUMBIA-NYC
We forwarded their survey concerning bacteria-mediated cancer therapies throughout different institutes and hospitals, employing simultaneously our Muses initiative. We received answers from both patients and field specialists and sent them the survey results.
IONIS-PARIS
We’re very glad to have come across team IONIS. Our collaboration began with a mutual interest in our projects. Despite cracking completely different nuts, we love wine and they’d like to fight cancer; our spears in line and our backs to the wall, because we could find no way to provide scientific assistance to each other, we decided to aim at the Human Practices. iGEM IONIS wanted to reach out with their project to local communities around the world, both to inform them about synthetic biology and SofterShock, and to gather opinions on the effects their project could have on the world. We took their message to Greece. We engaged the local agricultural community in Krini Trikalon, where viticulture is a major economic activity, and discussed the pros and cons of such a system with farmers, answering iGEM IONIS’ survey. We did the same with a winemaker from our city, Thessaloniki. On their part, IONIS provided us with a missing link on our human practices campaign OSIRIS: opinions from regulators. They presented our work to their own contacts in French bioethics regulatory agencies, at the same time forwarding our questions. They did the same with 3 synthetic biology experts. Thanks, IONIS!
INSA-UPS-TOULOUSE
The team INSA-UPS France created a survey regarding the disease of Cholera, which we forwarded to our university’s student community during a Team Fair Day (presentation of students’ teams). We managed to collect enough answers in order to analyse them statistically and send them back to Toulouse. In return, the Toulouse team supported us with our OSIRIS initiative by forwarding our email to their PIs and Instructors according to the OSIRIS Protocol.
UPV-VALENCIA
Our team’s collaboration with the team Valencia_UPV was very fruitful. They were working on a transgenic situation world study and we answered a few questions regarding the current situation of transgenic consumption in Greece in order to support them with their study and for our collaboration to be bidirectional. From their part, our collaborators participated in our Human Practices Campaign - OSIRIS Round 2 Quantitative survey. The sent the questionnaire to an advisor and an instructor.
EPIPHANY-NYC
Our OSIRIS Quantitative Survey was a stepping stone and a guidance for the EpiphanyNYC team as they adopted our OSIRIS Protocol and used it in their own Human Practices Campaign by modifying its content.
OSIRIS iGEM Teams We kindly thank the following iGEM Teams that participated in our Integrated Human Practices effort that implements OSIRIS Protocol. Specifically, they forwarded our OSIRIS Round 2 surveys. UNAMBG, Lund, Westminster UK, SSTi-SZGD, IISER, EpiphanyNYC, Valencia_UPV, Heidelberg, UiOSLO, Northwestern University, Harvard, INSA-UPS, Toulouse, AFCM-EGYPT, Pittsburgh, NTHU, Taiwan, SDU-Denmark, Aix-Marseille, UCSC, TecCEM, Tec-Chihuahua, WPI, University of Toronto, Paris-Saclay, UNIFI, Michigan, Grenoble Alpes, Groningen, UPMC, Dalhousie, Calgary, Technion, Stuttgart, XJTLU, TCFSH_Taiwan, TNCR_Korea
QUORUM SENSING
iGEM Team ColumpiaNYC
After our contact with Columpia NYC, we asked them to conduct experiments to data feed our quorum sensing model. They took OD600 measurements over 9 and 12 hrs with 10 minutes intervals from inoculations of bacteria transformed with their quorum sensing plasmid.
Basic Parts
Name
Description
Length
BBa_K2364000
pCMV
589 bp
BBa_K2364001
pTRE
315 bp
BBa_K2364002
rtTA (pANDORRA compatible)
779 bp
BBa_K2364003
DsRed (pANDORRA compatible)
710 bp
BBa_K2364004
Apoptin (pANDORRA compatible)
398 bp
BBa_K2364005
Lacl (pANDORRA compatible)
1151 bp
BBa_K2364006
SV40 polyA (pANDORRA compatible)
244 bp
BBa_K2364007
FF4 - rbGlob polyA (pANDORRA compatible)
976 bp
BBa_K2364042
miRNA-21-5p Target Tandem Repeats (x4) (T21)
88 bp
BBa_K2364043
miRNA-372-3p Target Tandem Repeats (x4) (T372)
92 bp
BBa_K2364044
miRNA-373-3p Target Tandem Repeats (x4) (T373)
92 bp
BBa_K2364045
miRNA-143-3p Target Tandem Repeats (x4) (T143)
84 bp
BBa_K2364046
FF4 Target Tandem Repeats (x3) (TFF4)
66 bp
Composite Parts
Name
Description
Length
BBa_K2364008
J23100 promoter - sfGFP - SV40 polyA (pANDORRA compatible)
1039 bp
BBa_K2364009
J23100 promoter - sfGFP - FF4 - rbGlob polyA (pANDORRA compatible)
1771 bp
BBa_K2364010
rtTA - J23100 promoter - sfGFP - SV40 polyA (pANDORRA compatible)
1826 bp
BBa_K2364011
DsRed - J23100 promoter - sfGFP - SV40 polyA (pANDORRA compatible)
1757 bp
BBa_K2364012
Apoptin - J23100 promoter - sfGFP - SV40 polyA (pANDORRA compatible)
1445 bp
BBa_K2364013
LacI - J23100 promoter - sfGFP - SV40 polyA (pANDORRA compatible)
2198 bp
BBa_K2364014
rtTA - T21 - SV40 polyA (pANDORRA compatible)
1106 bp
BBa_K2364015
rtTA - T372 - T373 - SV40 polyA (pANDORRA compatible)
1229 bp
BBa_K2364016
DsRed - T21 - SV40 polyA (pANDORRA compatible)
1037 bp
BBa_K2364017
DsRed - T372 - T373 - SV40 polyA (pANDORRA compatible)
1160 bp
BBa_K2364018
DsRed - T145 - T143 - TFF4 - SV40 polyA (pANDORRA compatible)
1245 bp
BBa_K2364019
Apoptin - T145 - T143 - TFF4 - SV40 polyA (pANDORRA compatible)
933 bp
BBa_K2364020
pCMV - rtTA - J23100 promoter - sfGFP - SV40 polyA (pANDORRA compatible)
2421 bp
BBa_K2364021
pCMV - DsRed - J23100 promoter - sfGFP - SV40 polyA (pANDORRA compatible)
2352 bp
BBa_K2364022
pCMV - Apoptin - J23100 promoter - sfGFP - SV40 polyA (pANDORRA compatible)
2040 bp
BBa_K2364023
pCMV - rtTA - T21 - SV40 polyA (pANDORRA compatible)
1701 bp
BBa_K2364024
pCMV - rtTA - T372 - T373 - SV40 polyA (pANDORRA compatible)
1824 bp
BBa_K2364025
pCMV - DsRed - T21 - SV40 polyA (pANDORRA compatible)
1632 bp
BBa_K2364026
pCMV - DsRed - T372 - T373 - SV40 polyA (pANDORRA compatible)
1755 bp
BBa_K2364027
pCMV - DsRed - T145 - T143 - TFF4 - SV40 polyA (pANDORRA compatible)
1840 bp
BBa_K2364028
pCMV - Apoptin - T145 - T143 - TFF4 - SV40 polyA (pANDORRA compatible)
1528 bp
BBa_K2364029
pTRE - FF4 - rbGlob polyA (pANDORRA compatible)
1299 bp
BBa_K2364030
pCMV - LacI - J23100 promoter - sfGFP - SV40 polyA (pANDORRA compatible)
2793 bp
BBa_K2364031
pTRE - LacI - J23100 promoter - sfGFP - SV40 polyA (pANDORRA compatible)
2519 bp
BBa_K2364032
pCMV - sfGFP (pANDORRA compatible)
1341 bp
BBa_K2364033
pTRE - sfGFP (pANDORRA compatible)
1067 bp
BBa_K2364034
pCMV - sfGFP - SV40 polyA (pANDORRA compatible)
1593 bp
BBa_K2364035
pCMV - sfGFP - FF4 - rbGlob polyA (pANDORRA compatible)
2325 bp
BBa_K2364036
pTRE - sfGFP - SV40 polyA (pANDORRA compatible)
1319 bp
BBa_K2364039
pCMV - LacI (pANDORRA compatible)
1746 bp
BBa_K2364040
pTRE - LacI (pANDORRA compatible)
1472 bp
BBa_K2364041
LacI - T21 - SV40 polyA (pANDORRA compatible)
1478 bp
BBa_K2364047
pLuxR - RBS - invasin
2907 bp
BBa_K2364048
pLuxL-RBS-LuxR-pLuxR-RBS-LuxL-pLuxR-RBS-invasin
4879 bp
Collection Parts
Name
Description
Length
BBa_K2364008
J23100 promoter - sfGFP - SV40 polyA (pANDORRA compatible)
1039 bp
BBa_K2364009
J23100 promoter - sfGFP - FF4 - rbGlob polyA (pANDORRA compatible)
1771 bp
BBa_K2364010
rtTA - J23100 promoter - sfGFP - SV40 polyA (pANDORRA compatible)
1826 bp
BBa_K2364011
DsRed - J23100 promoter - sfGFP - SV40 polyA (pANDORRA compatible)
1757 bp
BBa_K2364012
Apoptin - J23100 promoter - sfGFP - SV40 polyA (pANDORRA compatible)
1445 bp
BBa_K2364013
LacI - J23100 promoter - sfGFP - SV40 polyA (pANDORRA compatible)
2198 bp
BBa_K2364014
rtTA - T21 - SV40 polyA (pANDORRA compatible)
1106 bp
BBa_K2364015
rtTA - T372 - T373 - SV40 polyA (pANDORRA compatible)
1229 bp
BBa_K2364016
DsRed - T21 - SV40 polyA (pANDORRA compatible)
1037 bp
BBa_K2364017
DsRed - T372 - T373 - SV40 polyA (pANDORRA compatible)
1160 bp
BBa_K2364018
DsRed - T145 - T143 - TFF4 - SV40 polyA (pANDORRA compatible)
1245 bp
BBa_K2364019
Apoptin - T145 - T143 - TFF4 - SV40 polyA (pANDORRA compatible)
933 bp
BBa_K2364020
pCMV - rtTA - J23100 promoter - sfGFP - SV40 polyA (pANDORRA compatible)
2421 bp
BBa_K2364021
pCMV - DsRed - J23100 promoter - sfGFP - SV40 polyA (pANDORRA compatible)
2352 bp
BBa_K2364022
pCMV - Apoptin - J23100 promoter - sfGFP - SV40 polyA (pANDORRA compatible)
2040 bp
BBa_K2364023
pCMV - rtTA - T21 - SV40 polyA (pANDORRA compatible)
1701 bp
BBa_K2364024
pCMV - rtTA - T372 - T373 - SV40 polyA (pANDORRA compatible)
1824 bp
BBa_K2364025
pCMV - DsRed - T21 - SV40 polyA (pANDORRA compatible)
1632 bp
BBa_K2364026
pCMV - DsRed - T372 - T373 - SV40 polyA (pANDORRA compatible)
1755 bp
BBa_K2364027
pCMV - DsRed - T145 - T143 - TFF4 - SV40 polyA (pANDORRA compatible)
1840 bp
BBa_K2364028
pCMV - Apoptin - T145 - T143 - TFF4 - SV40 polyA (pANDORRA compatible)
1528 bp
BBa_K2364029
pTRE - FF4 - rbGlob polyA (pANDORRA compatible)
1299 bp
BBa_K2364030
pCMV - LacI - J23100 promoter - sfGFP - SV40 polyA (pANDORRA compatible)
2793 bp
BBa_K2364031
pTRE - LacI - J23100 promoter - sfGFP - SV40 polyA (pANDORRA compatible)
2519 bp
BBa_K2364032
pCMV - sfGFP (pANDORRA compatible)
1341 bp
BBa_K2364033
pTRE - sfGFP (pANDORRA compatible)
1067 bp
BBa_K2364034
pCMV - sfGFP - SV40 polyA (pANDORRA compatible)
1593 bp
BBa_K2364035
pCMV - sfGFP - FF4 - rbGlob polyA (pANDORRA compatible)
2325 bp
BBa_K2364036
pTRE - sfGFP - SV40 polyA (pANDORRA compatible)
1319 bp
BBa_K2364039
pCMV - LacI (pANDORRA compatible)
1746 bp
BBa_K2364040
pTRE - LacI (pANDORRA compatible)
1472 bp
BBa_K2364041
LacI - T21 - SV40 polyA (pANDORRA compatible)
1478 bp
InterLab
InterLab
Reproducibility is the ability to recompute data, by following the exact same protocol and analysis methods of an experiment. The replicability or repeatability of a study is the chance that an independent experiment targeting the same scientific question will produce consistent results. Consistent findings from independent investigators and laboratories are the primary means by which scientific evidence accumulates for or against a hypothesis (Leek and Peng 2015). To maintain the integrity of scientific research and the public's trust in science, the scientific community must ensure reproducibility and replicability by engaging in wide-range interlaboratory studies. One such study is the International InterLab measurement study organized by Jacob Beal et al.
The InterLab study has been a part of the iGEM competition for four years, with 2017 being the fourth year, where teams participating in the competition can join in the effort to improve the measurement tools available to both the iGEM community and the synthetic biology community as a whole. The aim of this study is to develop a robust measurement procedure for the quantification of green fluorescent protein (GFP) expression and thereby provide researchers with a detailed protocol and data analysis form which yields absolute and interlaboratory - comparable units for GFP measurement.
This year, the fourth International InterLab Measurement Study aims to establish a standard GFP measurement protocol based on engineering principles that any lab with a plate reader can use. The procedure involves the measurement of GFP expressed by E.coli DH5α which has been transformed with plasmids expressing GFP, each of which has a different promoter or ribosome binding site. The measurement involves the use of a plate reader but, optionally, a flow cytometry measurement could also be performed for more data to support the standardizing procedure. Our team was more than excited to provide flow cytometry data but we couldn't get the appropriate calibration beads on time.
Materials and methods
All plasmids used have the standard pSB1C3 backbone which has the high copy number pMB1 Origin of Replication resulting in 100-300 copies in each cell. The plasmids also have the Chloramphenicol acetyltransferase gene which detoxifies the antibiotic chloramphenicol providing resistance and therefore a selection ability of transformed bacteria when grown in LB-agar plates with it. The negative control only has an extra TetR promoter while the all other constructs have a promoter site, a ribosome binding site, GFP and two terminators built in them. The GFP gene is the same in all constructs (E0040) but the promoters and the ribosome bindings sites vary. The promoters were recovered from a library screened by Chris Anderson and each of them has a different strength. Three different ribosome binding sites were used; the first being a weak one based on Ron Weiss thesis (BBa_B0032), a strong one based on Elowitz repressilator (BBa_B0034) and the bicistronic design element Number 2 (BCD2) (BBa_J364100) designed by Mutalik et al (Mutalik et al. 2013).
Plate reader measurements
The first step is to obtain a ratiometric conversion factor to transform absorbance data into a standard OD600 measurement through the use of LUDOX-HS40 (provided by iGEM) as a single point reference. This ratiometric conversion of 600nm absorbance into OD600 normalizes variances in data obtained from instruments with different sensitivities. The absorbance at 600nm is measured in both ddH2O and LUDOX and the corrected absorbance is calculated by subtracting the water reading. The Reference OD600 defined as that measured by the reference spectrophotometer (0.0425; value provided by iGEM) is then divided by the corrected absorbance and the result is the correction factor. By multiplying the absorbance at 600nm measured by each instrument, with the correction factor, all readings are now aligned to the reference spectrophotometer readings. Our plate reader was PerkinElmer EnSpire Multimode Plate Reader 2300 and measurements were made in the standard modes of the plate reader.
After the OD600 measurements are calibrated, we needed to create a standard curve for fluorescent measurements, again so that any differences in the instrument sensitivities, do not affect the results. By taking reads from a series of serial dilutions of a fluorescein sodium salt solution in the plate reader, the standard curve, with which GFP measurements can be normalized among different instruments, was created.
After these calibrations were made, the 8 devices had to be tested. All test devices were transformed into K12 DH5a E.coli by all iGEM teams participating, so that there are no variations in the expression systems between different strains. After plating onto agar plates with chloramphenicol as a resistance marker, two colonies were picked and grown overnight into LB Medium with chloramphenicol. Then they were diluted to an OD600 of 0.02 (conversion of Absorbance 600nm to OD was made by the correction from the LUDOX measurements) and allowed to grow for up to another 6 hours while taking time points every two hours. Samples were obtained every two hours and they were all measured together in the end by our plate reader. All of our results have been submitted to iGEM for further processing and comparison with data provided by the rest of teams.
We hope that our measurements will provide helpful data in this international study and confidently, along with the assistance of all other participating teams, create a standardized protocol for measuring absolute units of GFP expression for all researchers to use. We would like to encourage all teams in the upcoming years to participate in these studies, as together we can make large-scale experimental measurements seem like just another every-day experiment in the lab.
References
Leek, J. T., and R. D. Peng. 2015. "Opinion: Reproducible research can still be wrong: Adopting a prevention approach." Proceedings of the National Academy of Sciences of the United States of America 112 (6):1645-1646. doi: 10.1073/pnas.1421412111. Mutalik, V. K., J. C. Guimaraes, G. Cambray, C. Lam, M. J. Christoffersen, Q. A. Mai, A. B. Tran, M. Paull, J. D. Keasling, A. P. Arkin, and D. Endy. 2013. "Precise and reliable gene expression via standard transcription and translation initiation elements."  Nature Methods 10 (4):354-+. doi: 10.1038/nmeth.2404.
