<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Greece_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Greece/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV style="display: none;" id="main-wraper"><NAV class="menu"><UL class="clearfix" id="list"><SPAN id="just_to_align"><LI style="width:" class="li_indiv"><A target="_self" href="https://2017.igem.org/Team:Greece/Project">PROJECT</A></LI><LI style="width:" class="li_indiv"><A target="_self" href="https://2017.igem.org/Team:Greece/Model">MODELING</A></LI><LI style="width:" class="li_indiv"><A target="_self" href="https://2017.igem.org/Team:Greece/human_practices">HUMAN PRACTICES</A></LI><LI style="width:" class="li_indiv"><A target="_self" href="https://2017.igem.org/Team:Greece/Achievements">ACHIEVEMENTS</A></LI><LI style="width:" class="li_indiv"><A target="_self" href="https://2017.igem.org/Team:Greece/Team">TEAM</A></LI></SPAN></UL></NAV></DIV><SVG style="display:none; position:absolute; z-index:1000;" id="svg_graphics_box" class="responsive_box" height="580" width="99%" viewbox="0 0 800 700"><A href="#" onmouseover="onHoverChanges( $(this).children('circle:first').attr('id') )" onmouseleave="onHoverHide($(this).children('circle:first').attr('id'))"><TEXT class="animate_text" id="t1" x="0" y="335" text-anchor="middle" fill="#000000"><TSPAN x="-227" dy="1em" onclick="displayMe('c_t1')" id="f_circle_textA">QUORUM</TSPAN><TSPAN x="-230" dy="1.5em" onclick="displayMe('c_t1')" id="f_circle_textB">SENSING</TSPAN></TEXT></A><A href="#" onmouseover="onHoverChanges( $(this).children('circle:first').attr('id') )" onmouseleave="onHoverHide($(this).children('circle:first').attr('id'))"><TEXT class="animate_text" x="0" y="335" text-anchor="middle" fill="#000000"><TSPAN x="195" dy="0em" onclick="displayMe('c_t2')" id="s_circle_textA">RNAi</TSPAN><TSPAN x="195" dy="1.5em" onclick="displayMe('c_t2')" id="s_circle_textB">CLASSIFIER</TSPAN><TSPAN x="193" dy="1.5em" onclick="displayMe('c_t2')" id="s_circle_textC">DESIGN</TSPAN></TEXT></A><A href="#" onmouseover="onHoverChanges( $(this).children('circle:first').attr('id') )" onmouseleave="onHoverHide($(this).children('circle:first').attr('id'))"><TEXT class="animate_text" id="t3" x="0" y="335" text-anchor="middle" fill="#000000"><TSPAN x="612" dy="0em" onclick="displayMe('c_t3')" id="t_circle_textA">PROTEIN</TSPAN><TSPAN x="610" dy="1.5em" onclick="displayMe('c_t3')" id="t_circle_textB">STRUCTURE</TSPAN><TSPAN x="610" dy="1.5em" onclick="displayMe('c_t3')" id="t_circle_textC">PREDICTION</TSPAN></TEXT></A><A href="#" onmouseover="onHoverChanges( $(this).children('circle:first').attr('id') )" onmouseleave="onHoverHide($(this).children('circle:first').attr('id'))"><TEXT class="animate_text" id="t4" x="0" y="335" text-anchor="middle" fill="#000000"><TSPAN x="1025" dy="-0.5em" onclick="displayMe('c_t4')" id="fo_circle_textA">COLON</TSPAN><TSPAN x="1023" dy="1.5em" onclick="displayMe('c_t4')" id="fo_circle_textB">CANCER</TSPAN><TSPAN x="1025" dy="1.5em" onclick="displayMe('c_t4')" id="fo_circle_textC">FLUID</TSPAN><TSPAN x="1023" dy="1.5em" onclick="displayMe('c_t4')" id="fo_circle_textD">DYNAMICS</TSPAN></TEXT></A></SVG><VIDEO id="video" autoplay="autoplay" muted="muted" preload="auto" loop="loop"><DIV id="display_box"><DIV id="ct1"><DIV id="Quorum_Sensing" width="100%" height="100%" class="display_frame hide"><DIV id="article_box"><ARTICLE id="static_content"><HEADER><STRONG class="sub_headers" style="font-size: 55px;">Quorum​ ​Sensing</STRONG></HEADER><SECTION><ARTICLE><HEADER><STRONG class="sub_headers">Motivation</STRONG></HEADER><SECTION class="sub_sections"><DIV style="text-align: justify;">
					Can we predict if and when our engineered bacteria will invade their surrounding tissue in order to transfer the classifier plasmids?  

Tumor invasion is mediated by a quorum sensing mechanism controlling the production of invasin and listeriolysin, the machinery that actuates it. It is a critical module of the integrated system. This basic safety- and efficacy -minded question motivates a closer examination of the system we call “quorum sensing”. It has been thoroughly examined in the literature [<A href="#ref1">1</A>] and by previous iGEM teams (ETH 2014, KU Leuven 2015, etc.), but with the assumption of spatial equilibrium -- for example, bacteria growing in a well-stirred bioreactor. But what happens with bacteria growing on top of a surface, such as the colon epithelium? Are the existing models adequate to describe this phenomenon emerging at the population level -- or does the distinctly different diffusive environment lead to unexpected phenomena?
To answer these questions, we’ve produced a modular matlab code that can serve future iGEM teams concerned with how diffusion can interact with other biological mechanisms.
				</DIV></SECTION></ARTICLE><ARTICLE><HEADER><STRONG class="sub_headers">Overview</STRONG></HEADER><SECTION class="sub_sections"><DIV style="text-align: justify;">
					Quorum sensing (QS) is the phenomenon of coordinated behaviour triggered by high bacterial population density. The Lux operon, responsible for quorum sensing, is a leaky switch. When it’s turned off, LuxR and LuxI are produced very slowly. LuxI produces a small signalling molecule in the homoserine lactone family (simply called “AHL” hereafter). LuxR is activated by it and in turn induces the Lux operon, in a positive feedback loop. The entire system thus functions as a latch-on switch; once the concentration of AHL enters a critical area, it increases rapidly and reaches a much higher equilibrium.

The name “Quorum sensing” is most often used to describe the Lux operon’s regulatory system - but does the operon itself always function as implied by that name? Does it turn on only when the bacterial population density is high? To answer this question, let’s first imagine a thought experiment. In the shady forest underbrush, on top of a wax-covered leaf there is a minuscule water droplet. Measuring no more that 0.1 nL in size, it has carried with it a few nutrients and a few bacterial cells happen to grow inside it, let’s say 10. Even though the bacterial density is not very high, the limited space could “fill up” with the AHL produced by only a handful of bacteria. In that case, the Lux operon wouldn’t be triggered by the bacterial density per se, but by the restrictive diffusive environment -- thus giving merit to such a term as “Diffusive sensing” <SUP>[<A href="#ref3">3</A>,<A href="#ref4">4</A>,<A href="#ref5">5</A>]</SUP>. These alternate hypotheses provide extra reasons to explore the system through simulation.

Our modelling starts with a closer look on previous work. We reimplement the deterministic model described in [<A href="#ref1">1</A>] and play with different parameters and with adding extensions such as diffusive losses and growth models [<A href="#ref2">2</A>] to get a better grasp of the phenomenon. This model describes continuous liquid cultures, where the bacteria are kept in a constant exponential growth phase by continuous dilution of the culture medium. Arguably these conditions, amenable though they may be to experimental validation, do not reflect the natural conditions in the colon nor the conditions our experiment entails. The most significant deviations arise from two facts of the natural environment we want to simulate: the microbial community originates from a small initial population colonizing a new environment (thus requiring a custom <B>growth model</B>) and instead of developing in the full volume of a liquid medium, it colonizes the solid, permeable surface of the colon epithelium (thus requiring the simulation of <B>diffusion</B>).
This model therefore comprises 3 parts:<UL class="sub_list_qs"><LI style="list-style:none">1. A​ ​network​ ​of​ ​chemical​ ​reactions​ ​in​ ​fixed-volume​,​​ ​well-stirred​ ​conditions​ ​that​ ​model​ ​the​ ​production​ ​and​ ​consumption​ ​of​ ​AHL​ ​as​ ​well​ ​as​ ​the​ ​systems regulatory​ ​elements. </LI><LI style="list-style:none">2. A​ ​custom​ ​growth​ ​model​ ​that​ ​evolves​ ​an​ ​initial​ ​inoculation​ ​to​ ​the​ ​environment's​ ​assumed​ ​carrying​ ​capacity. </LI><LI style="list-style:none">3. A​ ​diffusive​ ​model​ ​for​ ​the​ ​evolution​ ​of​ ​AHL​ ​spatial​ ​distribution.</LI></UL>
The “spatial equilibrium model” is built of the first 2 parts, while the “diffusive model” incorporates the last. Previous models of quorum sensing that we have studied only supply the first part. 
				</DIV></SECTION></ARTICLE><ARTICLE><HEADER><STRONG class="sub_headers">Availability</STRONG></HEADER><SECTION class="sub_sections">The​ ​accompanying​ ​code​ ​is​ ​available​ ​on ​GitHub​: <A href="https://github.com/igem-greece-2017" style="color: #106860">https://github.com/igem-greece-2017</A></SECTION></ARTICLE></SECTION></ARTICLE><BUTTON id="Section1_collapsable" class="accordion" style="font-size: 20px; letter-spacing: 1.5px;">Spatial equilibrium model</BUTTON><DIV id="panelQS1" class="panel" style="text-align: justify; padding: 20px 0px 0px 0px;"><ARTICLE id="article_content"><SECTION id="section_1" style="margin: 0px;" class="sections_qs"><ARTICLE id="sub_article_1"><HEADER id="sub_header_1" class="sub_headeren"><STRONG class="sub_headers">Chemical reaction model description</STRONG></HEADER><DIV style="position: relative; left: 50%; margin-right: 50%; transform: translate(-50%, 0%); text-align: center"><TABLE><TBODY><TR class="caption_sector" style="text-align: center"><TD style="font-size: 14.5px"><I>Figure 1: Quorum sensing network. The arrows imply chemical reactions.</I></TD></TR></TBODY></TABLE></DIV><SECTION id="sub_section_1" class="sub_sections"><P style="line-height: 1.5em">This model takes the form of a network of chemical reactions that simulate intracellular processes at the population level. Although the same processes seem anything but deterministic upon a closer look at the single cell, the individual variations can be assumed to be independent and identically distributed for each cell and their averaging eliminates the variability at the population level. Consequently, these chemical reactions are simulated as <STRONG>ordinary differential equations</STRONG>.</P><P style="line-height: 1.5em">The centerpiece is AHL. AHL is the small, freely-diffusing molecule that mediates cell-to-cell communication: when AHL levels are low, the quorum sensing switch is turned off; when they’re high, the switch is turned on. The rest of the interactions concern the protein LuxI, which produces AHL, and LuxR, which binds it and then, activated, goes on to induce the “DNA”. The “DNA” species refers to plasmids carrying the Lux regulatory system. Its induction by (LuxR.AHL)<SUB>2</SUB> marks the off-&gt;on transition; when most of the DNA is in the (uninduced) “DNA’ form, the switch is off; when most of it is in the “DNA.(LuxR.AHL)<SUB>2</SUB>” form, the switch is on.</P></SECTION><DIV style="position: relative; left: 50%; margin-right: 50%; transform: translate(-50%, 0%); text-align: center"><TABLE><TBODY><TR style="text-align:center"><TD style="font-size: 14.5px">Figure 2: Dilution mechanism that keeps the growth rate and the cell density constant. These are the conditions modelled in [<A href="#ref1">1</A>](Adapted from [<A href="#ref1">1</A>]).</TD></TR></TBODY></TABLE></DIV><SECTION id="sub_section_2" class="sub_sections"><P style="line-height: 1.5em">Supplementary to the core QS functionality, DNA undergoes duplication and AHL undergoes diffusion between 2 spaces: the “internal” (intracellular) and the “external”. To understand the latter process, consider all the bacterial cytoplasms conjoined in a single volume separated by a membrane from the outside.</P><P style="line-height: 1.5em">This model has originally been built to simulate an entire bacterial population living in constant exponential growth conditions in a constant volume with continuous dilution.. We consider these conditions not to influence the cell’s internal mechanisms significantly though and expand their approach to other living conditions, only adjusting the DNA duplication and AHL diffusion processes.</P><P style="line-height: 1.5em">We built this first model to better understand the constraints and basic properties of the physical system, before going into greater depth in the following sections. A caveat to our extrapolation is that cellular metabolism could be significantly altered when cells enter a stationary growth phase, impacting the core QS functionality. We keep this in mind, but have found no way to account for it.</P><P style="line-height: 1.5em">In the following section, we present exploratory simulations of the system's behaviour.</P></SECTION></ARTICLE></SECTION><SECTION id="section_3" class="sections_qs"><ARTICLE id="sub_article_3"><HEADER id="sub_header_3" class="sub_headeren" style="margin: 12px 0px -20px 0px"><STRONG class="sub_headers">Results (no custom growth model)</STRONG></HEADER><DIV style="text-align: center"><TABLE><TBODY><TR class="caption_sector" style="text-align: center"><TD style="font-size: 14.5px">Figure 3: Evolution of the QS dynamics when the dilution protocol described in [<A href="#ref1">1</A>] is implemented. Dividing bacteria constantly dilute their cytoplasm, severely slowing down QS. Culture volume: 0.2nL</TD></TR></TBODY></TABLE></DIV><DIV style="text-align: center"><TABLE><TBODY><TR class="caption_sector" style="text-align: center"><TD style="font-size: 14.5px">Figure 4: Same culture volume (0.2nL), but without the dilution protocol and with the bacteria in the stationary phase</TD></TR></TBODY></TABLE></DIV><SECTION id="sub_section_3" class="sub_sections"><P style="line-height: 1.5em">First, we simulate the system exactly as specified in the source material (<A href="#figure3">figure 3</A>), with the cells in a constant exponential growth phase and their density ($\Rightarrow$ their number) maintained. This implies constant dilution, which affects all chemical species apart from the DNA, which exactly compensates with replication. Because there are only 100 bacteria with a total cytoplasmic volume of 1.7e-4 nL in a 0.2 nL culture, AHL increases very slowly and QS toggling doesn’t happen within 25 hours.</P><P style="line-height: 1.5em">Without growing, and therefore without diluting to keep the bacteria at a constant density, the quorum sensing transition is triggered at 15 hours (<A href="#figure4">figure 4</A>) (<STRONG>t=0</STRONG> refers to the time when the bacteria have adapted to their environment and begin producing AHL). This is evident by the beginning of the sharp drop in uninduced “DNA”, as well as by a wrinkle on the AHL graph. This wrinkle is telltale: as the DNA is induced, the production rate of LuxI &amp; LuxR is increased. LuxI increases more slowly than LuxR however, resulting in a transient drop in AHL, as more of it is captured by LuxR. A little later, LuxI catches up and AHL levels increase faster.</P></SECTION><DIV style="text-align: center"><TABLE><TBODY><TR class="caption_sector"><TD colspan="2" style="font-size: 14.5px">Figure 5: Same conditions as in <A href="#figure4">figure 4</A>, changing only the culture volume. Left: 0.1nL Right: 0.4nL</TD></TR></TBODY></TABLE></DIV><SECTION id="sub_section_4" class="sub_sections"><P style="line-height: 1.5em">Keeping all the conditions the same as in <A href="#figure4">figure 4</A> and only tweaking the culture volume, its effect on QS becomes evident (<A href="#figure5">figure 5</A>). When the total volume is reduced by half (0.1nL), QS occurs at 10 hours. When it is doubled (0.4nL), QS occurs at 23 hours (but the transition is more gradual). The QS triggering time in these conditions depends linearly on the total volume.</P></SECTION></ARTICLE></SECTION><SECTION id="section_4" class="sections_qs"><ARTICLE id="sub_article_4"><HEADER id="sub_header_4" class="sub_headeren"><STRONG class="sub_headers">Growth model description</STRONG></HEADER><SECTION id="sub_section_5" class="sub_sections"><P style="line-height: 1.5em">How does a bacterial population colonize a solid surface? Are the dynamics similar to the liquid media?</P><P style="line-height: 1.5em">According to [<A href="#ref2">2</A>], the growth dynamics are indeed very similar, the greatest difference being a more gradual transition between the exponential and stationary phases. We use the growth model III from [<A href="#ref2">2</A>]:</P><MATH xmlns="http://www.w3.org/1998/Math/MathML" display="block" style="font-size: 20x"><MFRAC><MROW><MI>d</MI><MI>N</MI></MROW><MROW><MI>d</MI><MI>t</MI></MROW></MFRAC><MO>=</MO><MI>r</MI><MI>N</MI><MROW><MO>(</MO><MN>1</MN><MO>−</MO><MSUP><MROW><MO>(</MO><MFRAC><MI>N</MI><MSUB><MI>N</MI><MROW class="MJX-TeXAtom-ORD"><MI>m</MI><MI>a</MI><MI>x</MI></MROW></MSUB></MFRAC><MO>)</MO></MROW><MI>m</MI></MSUP><MO>)</MO></MROW><MROW><MO>(</MO><MN>1</MN><MO>−</MO><MSUP><MROW><MO>(</MO><MFRAC><MSUB><MI>N</MI><MROW class="MJX-TeXAtom-ORD"><MI>m</MI><MI>i</MI><MI>n</MI></MROW></MSUB><MI>N</MI></MFRAC><MO>)</MO></MROW><MI>n</MI></MSUP><MO>)</MO></MROW></MATH></SECTION><DIV style="text-align: center"><TABLE><TBODY><TR class="caption_sector"><TD style="font-size: 14.5px">Figure 6: Population growth on an agar surface. The population grows exponentially from a small inoculum to the environment’s carrying capacity after a short lag period. This is the full growth model, but in the simulations we disregard the lag phase. Its duration can’t be modelled precisely and, more importantly, we don’t expect the bacteria to actively express the Lux operon at that phase.</TD></TR></TBODY></TABLE></DIV><SECTION id="sub_section_6" class="sub_sections"><P style="line-height: 1.5em">The model can be seen as a transition function between 2 population levels. The steepness of the transition, $r$, depends primarily on temperature and to a smaller extent to nutritional levels; $m$ and $n$ are mostly fixed and $N_0$ is a parameter without a clear significance which only affects the duration of the lag phase. Here we ignore the lag phase, so $N_0=0$. In our simulations, we use the best estimates of these parameters for T=30oC and an environment with relatively few resources (agar with 1/25 the usual nutrient levels) based on [<A href="#ref2">2</A>]: r=1.5 [1/h], m=0.52, n=3.5.</P><P style="line-height: 1.5em">The new equation is concatenated to the system that expresses the chemical reactions and supplies a variable dilution loss, dependent on the variable growth rate. The simulations with the growth model keep a constant culture volume, like the previous ones, but allow the bacterial density to increase, without any external dilution of the entire culture. However, the bacterial equations experience the same dilution term, which is a result of the cytoplasm constantly expanding during the growth phase. As the population transits into the stationary phase, the dilution, following the growth rate, slows down as well, allowing the concentrations to increase freely.</P></SECTION></ARTICLE></SECTION><SECTION id="section_5" class="sections_qs"><ARTICLE id="sub_article_5"><HEADER id="sub_header_5" class="sub_headeren" style="margin: 15px 0px -20px 0px"><STRONG class="sub_headers">Results (with growth model)</STRONG></HEADER><DIV style="text-align: center"><TABLE><TBODY><TR class="caption_sector"><TD style="font-size: 14.5px">Figure 7: QS curves with growth for increasing culture volume. The final time is larger for the last plots. Culture volume left to right: 4μL,16μL,64μL,512μL,1024μL</TD></TR></TBODY></TABLE></DIV><DIV style="text-align: center"><TABLE><TBODY><TR class="caption_sector"><TD style="font-size: 14.5px">Figure 8: QS curves with a reduced growth rate. Culture volume: 4μL. Compared to <A href="#figure7">figure 7</A>, QS is indeed triggered later (since here there are fewer bacteria at equal times), but at an earlier growth phase, before the transition to the stationary phase.</TD></TR></TBODY></TABLE></DIV><DIV style="text-align: center"><TABLE><TBODY><TR class="caption_sector"><TD style="font-size: 14.5px">Figure 9: QS curves at an extreme colony volume: 5000μL. This volume is about the same as the agar in a small petri dish, which will become a useful reference for the diffusive model.</TD></TR></TBODY></TABLE></DIV><SECTION id="sub_section_7" class="sub_sections"><P style="line-height: 1.5em">The growth model impacts the QS system greatly. As is evident in figure <A href="#figure7">7</A> vs <A href="#figure8">8</A>, while the growth rate is high (r=1.5) quorum sensing is difficult to achieve. This corroborates the result in <A href="#figure3">figure 3</A>. As the volumes increase and the growth curve remains the same, more AHL has to be produced to achieve the same concentration, which takes more time. At an extreme volume of 5mL, in <A href="#figure9">figure 9</A> QS still happens, but much later, at 40 hours. This volume is significant, because it is the volume of a small petri dish, which we would like to simulate with the diffusive model to compare the results.</P></SECTION></ARTICLE></SECTION></ARTICLE></DIV><BUTTON onclick="sizeWindowQS2()" id="Section2_collapsable" class="accordion" style="font-size: 20px; letter-spacing: 1.5px;">Diffusive model</BUTTON><DIV id="panelQS2" class="panel" style="text-align: justify; padding: 20px 0px 0px 0px;"><ARTICLE><SECTION id="section_6" style="margin:0px" class="sections_qs"><ARTICLE id="sub_article_10"><SECTION id="sub_section_10" class="sub_sections"><P style="line-height: 1.5em">We’ve modelled a bacterial population in a well-stirred liquid culture so far. Without the “growth model”, we either model a stationary population or one that grows at a steady rate, its density maintained constant by compensating dilution. With the growth model, an initial inoculation grows to the environment’s carrying capacity, modelling a bacterial colonization of a new environment.</P><P style="line-height: 1.5em">Bacteria growing on a surface are packed very closely together, but the AHL they produce is free to leave their immediate surroundings and diffuse into the surrounding area. Diffusion is a well-described physical phenomenon and this model aims to couple the diffusive process with the chemical reactions of AHL inside millions of independent bacterial cells that are geometrically defined. There doesn’t seem to be any case of electrochemical l gradients affecting AHL diffusion, therefore our model is concerned only with its concentration.</P><P style="line-height: 1.5em">The primary goal was to simulate bacteria growing on the surface of an agar plate, as these conditions are easy to recreate experimentally and thus provide verification to our model and afterwards expand it to growth on a cell line monolayer we use in our experiments as a colorectal cancer model. Our collaboration with iGEM Columbia was meant to enable these experiments, but unfortunately material shortages only allowed us to experiment in liquid cultures.</P></SECTION></ARTICLE><SECTION id="section_7" style="margin:0px" class="sections_qs"><ARTICLE id="sub_article_11"><HEADER id="sub_header_11" class="sub_headeren"><STRONG class="sub_headers">Model Description</STRONG></HEADER><SECTION id="sub_section_11" class="sub_sections"><P style="line-height: 1.5em">We start with Fick’s laws of diffusion. In the simplest case of isotropic media without mass transport phenomena or external potentials, the driving force of diffusion is the concentration gradient and the diffusion coefficient is a constant real number. Thus, the general diffusion equation takes the form of the simpler heat equation:</P><MATH xmlns="http://www.w3.org/1998/Math/MathML" display="block" style="font-size: 20px"><MFRAC><MROW><MI mathvariant="normal">∂</MI><MROW class="MJX-TeXAtom-ORD"><MROW class="MJX-TeXAtom-ORD"><MO class="MJX-tex-mathit" mathvariant="italic" stretchy="false">[</MO><MI class="MJX-tex-mathit" mathvariant="italic">A</MI><MI class="MJX-tex-mathit" mathvariant="italic">H</MI><MI class="MJX-tex-mathit" mathvariant="italic">L</MI><MO class="MJX-tex-mathit" mathvariant="italic" stretchy="false">]</MO></MROW></MROW></MROW><MROW><MI mathvariant="normal">∂</MI><MROW class="MJX-TeXAtom-ORD"><MI>t</MI></MROW></MROW></MFRAC><MO>=</MO><MI>D</MI><MSUP><MI mathvariant="normal">∇</MI><MN>2</MN></MSUP><MROW class="MJX-TeXAtom-ORD"><MO class="MJX-tex-mathit" mathvariant="italic" stretchy="false">[</MO><MI class="MJX-tex-mathit" mathvariant="italic">A</MI><MI class="MJX-tex-mathit" mathvariant="italic">H</MI><MI class="MJX-tex-mathit" mathvariant="italic">L</MI><MO class="MJX-tex-mathit" mathvariant="italic" stretchy="false">]</MO></MROW></MATH><P style="line-height: 1.5em">It is a parabolic partial differential equation (PDE) in space and time. To specify a solvable problem based on such an equation, many more ingredients are needed: </P><UL><LI style="list-style-type: disc">a geometry</LI><LI style="list-style-type: disc">initial conditions</LI><LI style="list-style-type: disc">boundary conditions</LI></UL><P style="line-height: 1.5em">To actually solve it, we furthermore need to select a solution algorithm, which requires its own ingredients.</P><P style="line-height: 1.5em">If we allowed the diffusion coefficient to vary in space, we’d have a more general form of diffusion. Adding a production term $q$ to the right side, it becomes:</P><MATH xmlns="http://www.w3.org/1998/Math/MathML" display="block" style="font-size: 20px"><MFRAC><MROW><MI mathvariant="normal">∂</MI><MROW class="MJX-TeXAtom-ORD"><MROW class="MJX-TeXAtom-ORD"><MO class="MJX-tex-mathit" mathvariant="italic" stretchy="false">[</MO><MI class="MJX-tex-mathit" mathvariant="italic">A</MI><MI class="MJX-tex-mathit" mathvariant="italic">H</MI><MI class="MJX-tex-mathit" mathvariant="italic">L</MI><MO class="MJX-tex-mathit" mathvariant="italic" stretchy="false">]</MO></MROW></MROW></MROW><MROW><MI mathvariant="normal">∂</MI><MROW class="MJX-TeXAtom-ORD"><MI>t</MI></MROW></MROW></MFRAC><MO>=</MO><MI mathvariant="normal">∇</MI><MO>⋅</MO><MROW><MO>(</MO><MI>D</MI><MI mathvariant="normal">∇</MI><MROW class="MJX-TeXAtom-ORD"><MO class="MJX-tex-mathit" mathvariant="italic" stretchy="false">[</MO><MI class="MJX-tex-mathit" mathvariant="italic">A</MI><MI class="MJX-tex-mathit" mathvariant="italic">H</MI><MI class="MJX-tex-mathit" mathvariant="italic">L</MI><MO class="MJX-tex-mathit" mathvariant="italic" stretchy="false">]</MO></MROW><MO>)</MO></MROW><MO>+</MO><MI>q</MI></MATH></SECTION></ARTICLE></SECTION><SECTION id="section_8" style="margin:0px" class="sections_qs"><ARTICLE id="sub_article_12"><HEADER id="sub_header_12" class="sub_headeren"><STRONG class="sub_headers">Geometry</STRONG></HEADER><SECTION id="sub_section_1" class="sub_sections"><P style="line-height: 1.5em">The first design decision is to express the problem’s geometry. At a first glance at the task at hand, to model bacteria growing on an agar plate, one might assume a top-down 2D perspective, with the AHL diffusing across the surface away from the bacteria. The diffusion of AHL is inherently a 3D phenomenon though, and this perspective couldn’t easily incorporate the effects of diffusion along the height of the agar gel. In the end, we decided to model the entire 3D volume of an agar plate, with the bacteria at the top of the agar. The agar forms a short cylinder (a disk), surrounded by plastic on 3 sides and air on top. The cylinder is in the order of millimetres in height and centimetres across. An E. Coli cell is about 1μm -- a huge difference in scale! This difference makes the problem quite difficult to solve in practice.</P><P style="line-height: 1.5em">An important simplification at this point is to assume axial symmetry around the axis of the cylinder, thus making the problem tractable. We express the PDE in cylindrical coordinates;  after eliminating the angular coefficients of the derivatives, we are left with:</P>
				$$\rho \frac{\partial{\mathit{[AHL]}}}{\partial{t}} = D \left(\frac{\partial}{\partial{\rho}}\left(\rho \frac{\partial{\mathit{[AHL]}}}{\partial{\rho}}\right) + \frac{\partial}{\partial{z}}\left(\rho \frac{\partial{\mathit{[AHL]}}}{\partial{z}}\right)\right) + \rho q$$
				<P style="line-height: 1.5em">If we now transform $\rho \mapsto x$ &amp; $z \mapsto y$, thus having:</P>
				$$x \frac{\partial{\mathit{[AHL]}}}{\partial{t}} = \nabla \cdot \left(xD \nabla \mathit{[AHL]} \right) + x q$$
				<P style="line-height: 1.5em">This is identical to the diffusion equation above, where the time coefficient is $x$, the diffusion coefficient $xD$ and the production coefficient $xq$. Therefore, we’ll solve this problem on a 2D vertical cross-section of the cylinder, whose solutions are the same as the initial equation on the full 3D cylinder. The final geometry is shown in <A href="#figure10">figure 10</A>. The bacteria are the small red rectangles shown in the zoomed-in image.</P></SECTION><DIV style="text-align: center"><TABLE><TBODY><TR class="caption_sector"><TD width="50%" style="font-size: 14.5px">Figure 10 left: The geometry on which the diffusion PDE is solved. It represents an axisymmetric 3D cylindrical geometry: an agar plate. The left side is the cylinder’s axis, the right side is the rim. The top is the cylinder’s surface. On the top near the axis there are some bacteria. Since this perspective is a cross-section of the agar plate, the bacteria actually occupy a small disk on the surface of the agar near the axis (every shape in this geometry should be rotated around the axis to imagine its 3D representation).</TD><TD width="50%" style="font-size: 14.5px">Figure 10 right: Each red rectangle represents an E. Coli cell. The bacteria are organized in orderly rings and layers with no spaces between them (maximum density). In this case, there are 600 rings of bacteria and 4 layers**. The blue lines are the mesh, the solver’s spatial discretization. Observe how the mesh around the bacteria is very orderly, but also rather coarse (compared to the feature size). The loss in accuracy in this area is intentional: our model inherently can’t resolve concentration differences inside each bacterium’s cytoplasm, therefore a finer mesh would not provide extra information, only modelling artifacts -- and much more computation time!
<SUB>
** Due to constraints with the mesh generation, this isn’t exactly the case. The reality is more complicated, but it simulates bacteria packed closely together. Notice that each red rectangle has a blue rectangle next to it. Only 1 in 3 red rectangles actually interacts with the AHL, the rest is inert geometry. Thus, 1 cell covers the space of 6 rectangles on the same layer, plus 6 more on the layer below. The cell’s AHL output is multiplied by the number of bacteria it replaces, thus in fact concentrating the production of this entire region on 1 cell. This should be a slight source of error though, because the diffusion coefficient is large.
</SUB></TD></TR></TBODY></TABLE></DIV></ARTICLE></SECTION><SECTION id="section_9" style="margin:0px" class="sections_qs"><ARTICLE id="sub_article_13"><HEADER id="sub_header_13" class="sub_headeren"><STRONG class="sub_headers">Boundary and initial conditions</STRONG></HEADER><SECTION id="sub_section_13" class="sub_sections"><P style="line-height: 1.5em">Boundary conditions specify what happens to the concentration (Dirichlet BC), or to its gradient (Neumann BC), at the geometric boundaries. There’s a lot of those: the edges of the agar, as well as the edges of the rectangles that represent a ring of bacteria. A Neumann BC takes the form $\nabla \mathit{[AHL]} \cdot \hat{n} = q$, where $q$ is the flux through the boundary. $\hat{n}$ is the unit vector normal to the boundary.</P><P style="line-height: 1.5em">The edges of an agar plate are all reflecting boundaries, because AHL can’t diffuse through them: plastic walls at the sides and bottom, air at the top. Thus, at the boundary $q=0$: no AHL goes through.</P><P style="line-height: 1.5em">A boundary condition on the edge of the bacteria could simulate the semipermeable cell membrane, but unfortunately the current version of Matlab doesn’t accommodate conditions on internal boundaries. To evaluate the significance of this restriction we’ve run a test simulation (<A href="#figure11">figure 11</A>).</P><P style="line-height: 1.5em">The initial conditions are described by the [AHL] at each point in the geometry at $t=0$. Here, they are 0.</P></SECTION></ARTICLE></SECTION><SECTION id="section_10" style="margin:0px" class="sections_qs"><ARTICLE id="sub_article_14"><HEADER id="sub_header_14" class="sub_headeren"><STRONG class="sub_headers">Solution algorithm</STRONG></HEADER><SECTION id="sub_section_1" class="sub_sections"><P style="line-height: 1.5em">We solve the PDE with the <I>finite element method</I> (FEM). This method discretizes the continuous space into small elements by overlaying a mesh (<A href="#figure10">figure 10</A>) and transforming the continuous geometry into a graph. Each node is a dependent variable. The time-dependent PDE is transformed in this manner into a large system of ODEs, with 1 equation for each node (because the boundary conditions are Neumann).</P></SECTION></ARTICLE></SECTION><SECTION id="section_11" style="margin:0px" class="sections_qs"><ARTICLE id="sub_article_15"><HEADER id="sub_header_15" class="sub_headeren"><STRONG class="sub_headers">Coupling AHL diffusion and localized chemical interactions</STRONG></HEADER><SECTION id="sub_section_15" class="sub_sections"><P style="line-height: 1.5em">By and large the most interesting piece of this puzzle is how to couple the spatially-oblivious chemical reaction network described above with the diffusion. [AHL] has become a spatial field. Each ring/layer of bacteria (see footnote in <A href="#figure10">figure 10</A>) is an autonomous agent that interacts with the locally available AHL. A few things are evident: AHL concentrations at points on cytoplasms depend both on diffusion and on chemical interactions, and many more dependent variables are required, to store the concentration of every species at every bacterium.</P><P style="line-height: 1.5em">A bacterium can be thought of as a dynamical system with all the other chemical species as its state and local AHL as its input signal. The dynamical system can be seen as a function of the input and the previous state. Multiple dynamical systems can also be seen as a single function, because the function takes a point in space as input and knows which individual system to feed the input to. Since it can be seen as a function, it can readily be plugged into the equation above as the production coefficient - problem solved!</P><P style="line-height: 1.5em">… solved, but for the ODE solver which fails miserably at such a convoluted, nested system of equations! The alternative that has been successfully implemented is to augment the system of node equations. To the equations produced by the spatial discretization a new set of equations for every bacterium involved is added.</P><P style="line-height: 1.5em">The bacteria expect a single value for the concentration of AHL in the cytoplasm, but as can be seen in <A href="#figure10">figure 10</A> to each bacterium correspond 6 mesh nodes. For simplicity, the value of [AHL] that the bacteria see is the mean of those nodes.</P><P style="line-height: 1.5em">The d[AHL] (change in [AHL]) produced from the bacterial equations also has to be distributed correctly on the mesh nodes. What we want to simulate when distributing the bacterial output is a constant source on the entire cytoplasm. There is a certain complexity in how the finite element method formulates the node equations and the nodes are not all equivalent, so we can’t simply split the d[AHL] into many parts and give one to each node. Instead, we rely on the FEM algorithm to solve the spatial distribution problem for us by assigning to the bacterial geometries a constant production coefficient of 1, then hijack it by multiplying the resulting node increment coefficients by the d[AHL] produced by the bacteria.</P><P style="line-height: 1.5em">Much is assumed or reverse-engineered in order to arrive at this coupling mechanism. To verify that the model is still on track, we run test simulations with a single bacterium and a small surrounding space. The results should be similar to (but not exactly the same with) the non-diffusive model. Indeed, there is close agreement (<A href="#figure11">figure 11</A>).</P></SECTION><DIV style="text-align: center"><TABLE><TBODY><TR id="figure11"><TD colspan="3">Figure 11:</TD></TR><TR class="caption_sector"><TD style="font-size: 14.5px">A: The spatial equilibrium model for 1 bacterium in a 2.12E-4 nL volume.</TD><TD style="font-size: 14.5px">B: Same conditions, but simulated with the diffusive model. A diffusive barrier simulates the cell wall. Good agreement with the equilibrium model.</TD><TD style="font-size: 14.5px">C: Diffusive model without the cell wall. Again, quite similar to the case with the wall, but much simpler to scale up. This bacterial model is used in the larger geometries.</TD></TR></TBODY></TABLE></DIV></ARTICLE></SECTION><SECTION id="section_12" style="margin:0px" class="sections_qs"><ARTICLE id="sub_article_16"><HEADER id="sub_header_16" class="sub_headeren"><STRONG class="sub_headers">Implementing Growth</STRONG></HEADER><SECTION id="sub_section_16" class="sub_sections"><P style="line-height: 1.5em">Growth requires adding new bacteria to the geometry, but the finite element method doesn’t accommodate such changes. Consequently, the solution has to be stopped and restarted every time a new bacterium is added. This would be computationally prohibitive.</P><P style="line-height: 1.5em">Instead, the complete growth curve is precalculated and then <STRONG>quantized</STRONG> adaptively to levels corresponding to adding many rings of bacteria at the same time, possibly adding millions of new bacteria at each growth step (<A href="#figure12">figure 12</A>). The final number of bacteria generally depends on the nutrients provided by the growth medium and, for an agar plate with few added nutrients, is expected to be around $10^{8.9}$ bacteria [<A href="#ref2">2</A>]. To further mimic the way bacterial colonies grow, once there are enough bacteria the older cells in the center die.</P><P style="line-height: 1.5em">Bacterial growth constantly dilutes the cytoplasm - a process which heavily affects QS (<A href="#figure3">figure 3</A>). Here, the growth model is implemented in large discrete steps. In each step, the ratio of existing to new bacteria is calculated and a dilution performed on the existing ones, in order to keep the quantity of every chemical species constant during the growth step, apart from the DNA, which cancels its dilution with replication. This discrete dilution event creates minor artifacts on the simulations that manifest as discontinuities on the graphs.</P></SECTION><DIV style="text-align: center"><TABLE><TBODY><TR class="caption_sector"><TD style="font-size: 14.5px">Figure 12 left: The growth curve (blue) of a bacterial colony on a tiny agar disk (3.4mm across) without the lag phase. The simulation follows the quantized version of the growth curve (red).</TD><TD style="font-size: 14.5px">Figure 12 right: Quantized growth curve for a small agar disk 34mm across. Notice the much higher final population. It simulates growth on agar with few added nutrients as described in [<A href="#ref2">2</A>]. Mean relative quantization error: 4.9%</TD></TR></TBODY></TABLE></DIV></ARTICLE></SECTION><SECTION id="section_12" style="margin:0px" class="sections_qs"><ARTICLE id="sub_article_17"><HEADER id="sub_header_17" class="sub_headeren"><STRONG class="sub_headers">Results</STRONG></HEADER><SECTION id="sub_section_17" class="sub_sections"><P style="line-height: 1.5em">The basic question that we want to answer at this point is if and when will the bacterial colony exhibit QS behavior on an agar plate - faster or later than in a liquid culture with the same number of bacteria? We present 2 simulations that differ in scale. The first is a tiny agar plate measuring 3.4mm across and .551mm in depth, for a volume of 5μL (<A href="#figure13">figure 13</A>). The second is much larger in scope and computational effort: agar in a small petri dish 34mm in diameter and 5.51mm in depth. The larger scope allows direct experimental evaluation of the model, but to make it more computationally friendly the growth happens in a more spread-out fashion than normal and the colony appears to “walk” across the agar.</P></SECTION><DIV style="text-align: center"><TABLE><TBODY><TR><TD>Progression of [AHL] dynamics on the tiny agar plate of simulation 1. Left: spatial distribution of [AHL] on the agar cylinder as it evolves in time. Same geometry as on [figure 13a][here] (cylinder axis on the left, rim on the right, surface on top). During the first 6sec (video time), the dynamics are dominated by the colony’s outwards growth. This expansion phase prevents AHL from reaching critical concentrations and delays QS. After the colony growth slows down, the spatial maximum in [AHL] stops moving and the QS transition quickly follows.
Right top: total amount of AHL in the agar plate. Right bottom: maximum [AHL] from the left graph at each time step.</TD></TR></TBODY></TABLE></DIV><SECTION id="sub_section_8" class="sub_sections"><P style="line-height: 1.5em">The tiny disk is inoculated with 7e4 bacteria, which gradually grow to a final size of 8.12e6 bacteria over 11 growth steps (<A href="#qs_video_1">video</A>). Quorum sensing is indeed triggered early under these conditions, at 11 hours (<A href="#figure14">figure 14</A>).</P><P style="line-height: 1.5em">The small agar plate (which, despite its name, is the large simulation!) is inoculated with 8.2e4 bacteria that grow over the course of 15 hours to become 5.1e8. They achieve quorum sensing at 20 hours, after filling a 5mL agar disk with 1.6pmol AHL (see <A href="#qs_video_2">video</A>). This is a huge amount in comparison to the previous simulation.</P></SECTION><SECTION id="section_13" style="margin:0px" class="sections_qs"><ARTICLE id="sub_article_18"><HEADER id="sub_header_18" class="sub_headeren"><STRONG class="sub_headers">Conclusions</STRONG></HEADER><SECTION id="sub_section_18" class="sub_sections"><P style="line-height: 1.5em">A comparison between the two simulations at 5mL volumes is telling. The growth curve is the same in both by design: same inoculations, same growth rates and same final populations. In the surface growth case however, as AHL diffuses slowly from the bacteria to the rest of the agar plate, it has the chance to achieve higher local concentrations. That’s why quorum sensing is achieved at 20 hours, versus 41 for the spatial equilibrium model.</P><P style="line-height: 1.5em">This observation has significant repercussions on the applicability of our project. Bacteria that manage to colonize a solid surface seem to have a much higher probability of achieving quorum sensing under conditions of competitive growth, where their total population size will be limited by resource availability, such as in the colon. Of even greater practical importance, it shows that transfections to monolayers of Caco-2 cells in the lab, which we use as a cancer model, by quorum sensing bacteria have a reasonable chance of success.</P><P style="line-height: 1.5em">The simulations were designed in this way to allow for easy experimental validation. Unfortunately, material shortages prevented the realization of the suggested experiments and the experimental validation of our model is deferred.</P></SECTION></ARTICLE></SECTION><DIV style="text-align: center"><TABLE><TBODY><TR><TD style="font-size: 14.5px">Figure 13 left: The geometry of the first simulation on the tiny agar disk with the bacterial colony fully grown. On the left is the disk’s axis and on the right is the rim. The bacteria sit on the disk’s surface.</TD><TD style="font-size: 14.5px">Figure 13 right: Geometry for the second simulation. Notice the much larger agar disk. The large scale difference makes the bacteria invisible at this zooms. Their position can be inferred by the fine mesh around them.</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD style="font-size: 14.5px">Figure 14: Quorum sensing is achieved by an E. Coli bacterium, part of a colony growing on a tiny agar plate at 11 hours. This bacterium was created at 3 hours and lived until the end. Because of its central position, it is the first bacterium in the colony to transition into quorum sensing.</TD></TR></TBODY></TABLE></DIV><SECTION class="sub_sections"><DIV style="text-align: center"><TABLE><TBODY><TR><TD style="font-size: 14.5px">Figure 16: Close look at a bacterium from simulation 2. It is created at 9 hours and the levels of [AHL] around it are rising rapidly, because the colony has just expanded to its area. At 20 hours [DNA.(LuxR.AHL)2] begins to rise rapidly, signifying the quorum sensing transition. It is one of the first bacteria in the colony to transition into QS, thanks to its central position.</TD></TR></TBODY></TABLE></DIV></SECTION><DIV style="text-align: center"><TABLE><TBODY><TR><TD>Progression of [AHL] dynamics on the small agar plate of simulation 2. Left: spatial distribution of [AHL] on the agar cylinder as it evolves in time. Same geometry as on [figure 13b][here] (cylinder axis on the left, rim on the right, surface on top). During the first 10sec (video time), the dynamics are dominated by the colony’s outwards growth. This expansion phase prevents AHL from reaching critical concentrations and delays QS. After the colony growth slows down, the spatial maximum in [AHL] stops moving and grows slowly. In this case, there are many more bacteria occupying a much larger area than in simulation 1. At 14sec the quorum sensing transition begins, but the visualization appears quite different than in simulation 1. One reason for this is that the larger area the bacteria occupy produces a phase difference between them, with those that are further away from the [AHL] maximum taking more time to reach QS. The transition is therefore more gradual. After 16sec, most bacteria have transitioned and the rapid increase in AHL production is evident by the sharpening spatial distribution.
Right top: total amount of AHL in the agar plate. Right bottom: maximum [AHL] from the left graph at each time step.
</TD></TR></TBODY></TABLE></DIV></ARTICLE></SECTION></SECTION></ARTICLE></DIV><DIV id="references"><ARTICLE><HEADER><STRONG class="sub_headers" class="sub_headeren">References</STRONG></HEADER><SECTION style="text-align:justify" class="sub_sections">
			[<SPAN id="ref1">1</SPAN>] Weber, M. &amp; Buceta, J. (2013). Dynamics of the quorum sensing switch: stochastic and non-stationary effects. <I>BMC Systems Biology 2013, 7:6</I>
			[<SPAN id="ref2">2</SPAN>] Fujikawa, H. &amp; Morozumi, S. (2005). Modeling Surface Growth of Escherichia coli on Agar Plates. <I>APPLIED AND ENVIRONMENTAL MICROBIOLOGY, Dec. 2005, 7920–7926</I>
			[<SPAN id="ref3">3</SPAN>] Trovato, A., Seno, F, Zanardo, M., Alberghini, S., Tondello, A. &amp; Squartini A. (2014). Quorum vs. diffusion sensing: a quantitative analysis of the relevance of absorbing or reflecting boundaries. <I>FEMS Microbiology Letters, Volume 352, Issue 2, 1 March 2014, Pages 198–203</I>
			[<SPAN id="ref4">4</SPAN>] Rosemary, J. R. Is quorum sensing a side effect of diffusion sensing? <I>Trends in Microbiology, Volume 10, Issue 8, Pages 365-370</I>
			[<SPAN id="ref5">5</SPAN>] West, S. A., Winzer, K., Gardner, A., Diggle, S. P. Quorum sensing and the confusion about diffusion. <I>Trends in Microbiology, Volume 20, Issue 12, 2012, Pages 586-594</I>. <A href="https://doi.org/10.1016/j.tim.2012.09.004">https://doi.org/10.1016/j.tim.2012.09.004</A>.</SECTION></ARTICLE></DIV></DIV></DIV></DIV><DIV id="RNAi_Classifier_Design" width="100%" height="100%" class="display_frame hide"><DIV id="article_box"><ARTICLE id="static_content"><HEADER><STRONG class="sub_headers" style="font-size: 55px;">pANDORRA Design Algorithm</STRONG></HEADER><DIV style="text-align: right"><SPAN style="text-align: right; font-size:16px">“If you know yourself and you know your enemy, you shouldn’t fear the result of a thousand battles”-Sun Tzu</SPAN></DIV><SECTION><ARTICLE><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>If knowledge is power, then modeling is the key to a successful engineering endeavour. Our modeling of the biophysical characteristics of the classifier circuits that follows in this section is largely based upon the exciting work of research teams led by Professors Z. Xie and Y. Benenson [<A href="#ref1a">1</A>, <A href="#ref2a">2</A>]. The latter’s newest paper [<A href="#ref2a">2</A>] faces the same questions: it analyzes the modeling process and optimizes the classifier’s logical expression. Standing on their shoulders, we expand upon their work by analyzing architectural variations of the classifier circuits, which add an additional optimization axis.</P></DIV></SECTION></ARTICLE></SECTION><SECTION><ARTICLE><HEADER><STRONG class="sub_headers">Architecture of a cell-type classifier</STRONG></HEADER><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>pANDORRA is based on a family of layered genetic circuits that aim to biophysically implement logical expressions involving miRNA molecules in 2 basic roles: upregulated or downregulated (on the target cells in respect to the control). The family is characterized by a basic structure emanating from these two roles. The target gene (the system’s output) is controlled by a promoter and RNA interference (RNAi) from the downregulated miRNAs and forms the lower layer of the system. The upper layer comprises target sites for the upregulated miRNAs and an inhibitor for the lower layer.</P><P>This basic architecture can be significantly improved with a double-inversion module for the the upper layer [<A href="#ref1a">1</A>]. It takes the form of a middle layer with a promoter that is activated by the upper layer’s product and in turn produces the inhibitor for the lower layer. This extra part significantly improves the system’s efficiency.</P><P>But how do we define the system’s efficiency and how do we optimize its architecture, out of the many options left open by the previous definitions? And after finding the optimal architecture, how do we choose the optimal miRNA targets to separate particular cell groups? These are the basic questions we set out to answer by modeling the pANDORRA system.</P></DIV></SECTION></ARTICLE></SECTION><SECTION><ARTICLE><HEADER><STRONG class="sub_headers">Modeling classifiers</STRONG></HEADER><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>The main task of a classifier system, given a group of target and control cells, is to produce large amounts of output when inside a target cell and little output when inside a control cell. Ideally, it should function as a logical function: maximum output expression in all target cells and zero in all controls. We will therefore judge its efficiency by how close it is to the ideal. We define the fold change of a concrete classifier instance as the average output margin between the two groups.</P><P>The circuit’s layered design naturally guides the modeling effort in two paths: estimating each layer’s output and the effects this output has on the next layer. Basic elements of the model therefore would be relationships between the concentrations of the involved regulatory species and the outputs of the regulated elements.</P><P>Take the RNAi mechanism: let P<SUB>max</SUB> be the unregulated (maximum) output concentration and C* the miRNA concentration that results in a 50% reduction of the output. Then the relationship between the output [P] and the input [miRNA] is:</P><SPAN class="equation"><MATH xmlns="http://www.w3.org/1998/Math/MathML" display="block" style="font-size: 20px;"><MO stretchy="false">[</MO><MI>P</MI><MO stretchy="false">]</MO><MO>=</MO><MSUB><MI>P</MI><MROW class="MJX-TeXAtom-ORD"><MI>m</MI><MI>a</MI><MI>x</MI></MROW></MSUB><MFRAC><MN>1</MN><MROW><MN>1</MN><MO>+</MO><MFRAC><MROW class="MJX-TeXAtom-ORD"><MO class="MJX-tex-mathit" mathvariant="italic" stretchy="false">[</MO><MI class="MJX-tex-mathit" mathvariant="italic">m</MI><MI class="MJX-tex-mathit" mathvariant="italic">i</MI><MI class="MJX-tex-mathit" mathvariant="italic">R</MI><MI class="MJX-tex-mathit" mathvariant="italic">N</MI><MI class="MJX-tex-mathit" mathvariant="italic">A</MI><MO class="MJX-tex-mathit" mathvariant="italic" stretchy="false">]</MO></MROW><MROW><MI>C</MI><MO>∗</MO></MROW></MFRAC></MROW></MFRAC></MATH></SPAN><P>Next question: if P is regulated by 2 miRNAs together, how do they interact? Following the same logic as in [<A href="#ref2a">2</A>], we assume that the behave additively. If each miRNA has a C<SUB>i</SUB>* concentration of half repression and n is the number of miRNAs, the cumulative regulation is:</P><SPAN class="equation"><MATH xmlns="http://www.w3.org/1998/Math/MathML" display="block" style="font-size: 20px"><MO stretchy="false">[</MO><MI>P</MI><MO stretchy="false">]</MO><MO>=</MO><MSUB><MI>P</MI><MROW class="MJX-TeXAtom-ORD"><MI>m</MI><MI>a</MI><MI>x</MI></MROW></MSUB><MFRAC><MN>1</MN><MROW><MN>1</MN><MO>+</MO><MUNDEROVER><MO>∑</MO><MROW class="MJX-TeXAtom-ORD"><MI>i</MI><MO>=</MO><MN>1</MN></MROW><MROW class="MJX-TeXAtom-ORD"><MI>n</MI></MROW></MUNDEROVER><MROW class="MJX-TeXAtom-ORD"><MFRAC><MROW class="MJX-TeXAtom-ORD"><MO class="MJX-tex-mathit" mathvariant="italic" stretchy="false">[</MO><MI class="MJX-tex-mathit" mathvariant="italic">m</MI><MI class="MJX-tex-mathit" mathvariant="italic">i</MI><MI class="MJX-tex-mathit" mathvariant="italic">R</MI><MI class="MJX-tex-mathit" mathvariant="italic">N</MI><MSUB><MI class="MJX-tex-mathit" mathvariant="italic">A</MI><MI class="MJX-tex-mathit" mathvariant="italic">i</MI></MSUB><MO class="MJX-tex-mathit" mathvariant="italic" stretchy="false">]</MO></MROW><MSUBSUP><MI>C</MI><MI>i</MI><MO>∗</MO></MSUBSUP></MFRAC></MROW></MROW></MFRAC></MATH></SPAN><P>If there are multiple target sites for a particular miRNA we can assume that their effect on interference will also be additive, so if there are r<SUB>i</SUB> target repeats for miRNA<SUB>i</SUB>, the output will be:</P><SPAN class="equation"><MATH xmlns="http://www.w3.org/1998/Math/MathML" display="block" style="font-size:20px"><MO stretchy="false">[</MO><MI>P</MI><MO stretchy="false">]</MO><MO>=</MO><MSUB><MI>P</MI><MROW class="MJX-TeXAtom-ORD"><MI>m</MI><MI>a</MI><MI>x</MI></MROW></MSUB><MFRAC><MN>1</MN><MROW><MN>1</MN><MO>+</MO><MUNDEROVER><MO>∑</MO><MROW class="MJX-TeXAtom-ORD"><MI>i</MI><MO>=</MO><MN>1</MN></MROW><MROW class="MJX-TeXAtom-ORD"><MI>n</MI></MROW></MUNDEROVER><MROW class="MJX-TeXAtom-ORD"><MSUB><MI>r</MI><MI>i</MI></MSUB><MFRAC><MROW class="MJX-TeXAtom-ORD"><MO class="MJX-tex-mathit" mathvariant="italic" stretchy="false">[</MO><MI class="MJX-tex-mathit" mathvariant="italic">m</MI><MI class="MJX-tex-mathit" mathvariant="italic">i</MI><MI class="MJX-tex-mathit" mathvariant="italic">R</MI><MI class="MJX-tex-mathit" mathvariant="italic">N</MI><MSUB><MI class="MJX-tex-mathit" mathvariant="italic">A</MI><MI class="MJX-tex-mathit" mathvariant="italic">i</MI></MSUB><MO class="MJX-tex-mathit" mathvariant="italic" stretchy="false">]</MO></MROW><MSUBSUP><MI>C</MI><MI>i</MI><MO>∗</MO></MSUBSUP></MFRAC></MROW></MROW></MFRAC></MATH></SPAN><P>To simplify the following expressions, we can define the interference on a scale of 0 to 1 based on the above:</P><SPAN class="equation"><MATH xmlns="http://www.w3.org/1998/Math/MathML" display="block" style="font-size: 20px"><MROW class="MJX-TeXAtom-ORD"><MI class="MJX-tex-mathit" mathvariant="italic">R</MI><MI class="MJX-tex-mathit" mathvariant="italic">N</MI><MI class="MJX-tex-mathit" mathvariant="italic">A</MI><MI class="MJX-tex-mathit" mathvariant="italic">i</MI></MROW><MO>=</MO><MFRAC><MN>1</MN><MROW><MN>1</MN><MO>+</MO><MUNDEROVER><MO>∑</MO><MROW class="MJX-TeXAtom-ORD"><MI>i</MI><MO>=</MO><MN>1</MN></MROW><MROW class="MJX-TeXAtom-ORD"><MI>n</MI></MROW></MUNDEROVER><MROW class="MJX-TeXAtom-ORD"><MSUB><MI>r</MI><MI>i</MI></MSUB><MFRAC><MROW class="MJX-TeXAtom-ORD"><MO class="MJX-tex-mathit" mathvariant="italic" stretchy="false">[</MO><MI class="MJX-tex-mathit" mathvariant="italic">m</MI><MI class="MJX-tex-mathit" mathvariant="italic">i</MI><MI class="MJX-tex-mathit" mathvariant="italic">R</MI><MI class="MJX-tex-mathit" mathvariant="italic">N</MI><MSUB><MI class="MJX-tex-mathit" mathvariant="italic">A</MI><MI class="MJX-tex-mathit" mathvariant="italic">i</MI></MSUB><MO class="MJX-tex-mathit" mathvariant="italic" stretchy="false">]</MO></MROW><MSUBSUP><MI>C</MI><MI>i</MI><MO>∗</MO></MSUBSUP></MFRAC></MROW></MROW></MFRAC></MATH></SPAN><P>These observations can provide the output of the classifier’s upper layer. A valid design is not limited to 1 OR gate, therefore the total output (let’s assume the output is rtTA) is the sum of each gate’s output:</P><SPAN class="equation"><MATH xmlns="http://www.w3.org/1998/Math/MathML" display="block" style="font-size: 20px"><MO stretchy="false">[</MO><MI>r</MI><MI>t</MI><MI>T</MI><MI>A</MI><MO stretchy="false">]</MO><MO>=</MO><MI>r</MI><MI>t</MI><MI>T</MI><MSUB><MI>A</MI><MROW class="MJX-TeXAtom-ORD"><MI>m</MI><MI>a</MI><MI>x</MI></MROW></MSUB><MUNDEROVER><MO>∑</MO><MROW class="MJX-TeXAtom-ORD"><MI>g</MI><MO>=</MO><MN>1</MN></MROW><MROW class="MJX-TeXAtom-ORD"><MI>m</MI></MROW></MUNDEROVER><MROW class="MJX-TeXAtom-ORD"><MSUB><MROW class="MJX-TeXAtom-ORD"><MI class="MJX-tex-mathit" mathvariant="italic">R</MI><MI class="MJX-tex-mathit" mathvariant="italic">N</MI><MI class="MJX-tex-mathit" mathvariant="italic">A</MI><MI class="MJX-tex-mathit" mathvariant="italic">i</MI></MROW><MI>g</MI></MSUB></MROW></MATH></SPAN><P>rtTA is an inducer of the pTRE promoter and its induction can be modelled on a scale of 0 to 1 with the use of the reaction’s dissociation constant K<SUB>d</SUB> as follows:</P><SPAN class="equation"><MATH xmlns="http://www.w3.org/1998/Math/MathML" display="block" style="font-size 22px"><MI>A</MI><MO>=</MO><MFRAC><MROW class="MJX-TeXAtom-ORD"><MI class="MJX-tex-mathit" mathvariant="italic">r</MI><MI class="MJX-tex-mathit" mathvariant="italic">t</MI><MI class="MJX-tex-mathit" mathvariant="italic">T</MI><MI class="MJX-tex-mathit" mathvariant="italic">A</MI></MROW><MROW><MSUB><MI>K</MI><MI>d</MI></MSUB><MROW class="MJX-TeXAtom-ORD"><MI class="MJX-tex-mathit" mathvariant="italic">r</MI><MI class="MJX-tex-mathit" mathvariant="italic">t</MI><MI class="MJX-tex-mathit" mathvariant="italic">T</MI><MI class="MJX-tex-mathit" mathvariant="italic">A</MI></MROW></MROW></MFRAC></MATH></SPAN><P>The second layer can be regulated in 2 ways: by rtTA through its promoter and by RNA interference, like the other layers. The first regulation is transcriptional, whereas the second post-transcriptional. Let’s assume the outputs of the second layer can be the protein LacI and the miRNA FF4. LacI will be affected by both regulatory mechanisms; in our particular implementation though FF4 will only be affected by the transcriptional regulation due to the FF4 being produced. Their levels would be then:</P><SPAN class="equation"><MATH xmlns="http://www.w3.org/1998/Math/MathML" display="block" style="font-size: 20px"><MROW class="MJX-TeXAtom-ORD"><MO class="MJX-tex-mathit" mathvariant="italic" stretchy="false">[</MO><MI class="MJX-tex-mathit" mathvariant="italic">L</MI><MI class="MJX-tex-mathit" mathvariant="italic">a</MI><MI class="MJX-tex-mathit" mathvariant="italic">c</MI><MI class="MJX-tex-mathit" mathvariant="italic">I</MI><MO class="MJX-tex-mathit" mathvariant="italic" stretchy="false">]</MO></MROW><MO>=</MO><MSUB><MROW class="MJX-TeXAtom-ORD"><MO class="MJX-tex-mathit" mathvariant="italic" stretchy="false">[</MO><MI class="MJX-tex-mathit" mathvariant="italic">L</MI><MI class="MJX-tex-mathit" mathvariant="italic">a</MI><MI class="MJX-tex-mathit" mathvariant="italic">c</MI><MI class="MJX-tex-mathit" mathvariant="italic">I</MI><MO class="MJX-tex-mathit" mathvariant="italic" stretchy="false">]</MO></MROW><MROW class="MJX-TeXAtom-ORD"><MI>m</MI><MI>a</MI><MI>x</MI></MROW></MSUB><MI>A</MI><MO stretchy="false">(</MO><MROW class="MJX-TeXAtom-ORD"><MI class="MJX-tex-mathit" mathvariant="italic">r</MI><MI class="MJX-tex-mathit" mathvariant="italic">t</MI><MI class="MJX-tex-mathit" mathvariant="italic">T</MI><MI class="MJX-tex-mathit" mathvariant="italic">A</MI></MROW><MO stretchy="false">)</MO><MROW class="MJX-TeXAtom-ORD"><MI class="MJX-tex-mathit" mathvariant="italic">R</MI><MI class="MJX-tex-mathit" mathvariant="italic">N</MI><MI class="MJX-tex-mathit" mathvariant="italic">A</MI><MI class="MJX-tex-mathit" mathvariant="italic">i</MI></MROW><MO stretchy="false">(</MO><MTEXT>up_miR</MTEXT><MO stretchy="false">)</MO></MATH></SPAN><SPAN class="equation"><MATH xmlns="http://www.w3.org/1998/Math/MathML" display="block" style="font-size: 20px"><MROW class="MJX-TeXAtom-ORD"><MO class="MJX-tex-mathit" mathvariant="italic" stretchy="false">[</MO><MI class="MJX-tex-mathit" mathvariant="italic">L</MI><MI class="MJX-tex-mathit" mathvariant="italic">a</MI><MI class="MJX-tex-mathit" mathvariant="italic">c</MI><MI class="MJX-tex-mathit" mathvariant="italic">I</MI><MO class="MJX-tex-mathit" mathvariant="italic" stretchy="false">]</MO></MROW><MO>=</MO><MSUB><MROW class="MJX-TeXAtom-ORD"><MO class="MJX-tex-mathit" mathvariant="italic" stretchy="false">[</MO><MI class="MJX-tex-mathit" mathvariant="italic">L</MI><MI class="MJX-tex-mathit" mathvariant="italic">a</MI><MI class="MJX-tex-mathit" mathvariant="italic">c</MI><MI class="MJX-tex-mathit" mathvariant="italic">I</MI><MO class="MJX-tex-mathit" mathvariant="italic" stretchy="false">]</MO></MROW><MROW class="MJX-TeXAtom-ORD"><MI>m</MI><MI>a</MI><MI>x</MI></MROW></MSUB><MI>A</MI><MO stretchy="false">(</MO><MROW class="MJX-TeXAtom-ORD"><MI class="MJX-tex-mathit" mathvariant="italic">r</MI><MI class="MJX-tex-mathit" mathvariant="italic">t</MI><MI class="MJX-tex-mathit" mathvariant="italic">T</MI><MI class="MJX-tex-mathit" mathvariant="italic">A</MI></MROW><MO stretchy="false">)</MO></MATH></SPAN><P>The final layer’s output applies the exact same logic:</P><SPAN class="equation"><MATH xmlns="http://www.w3.org/1998/Math/MathML" display="block" style="font-size: 20px"><MROW class="MJX-TeXAtom-ORD"><MO class="MJX-tex-mathit" mathvariant="italic" stretchy="false">[</MO><MI class="MJX-tex-mathit" mathvariant="italic">O</MI><MI class="MJX-tex-mathit" mathvariant="italic">u</MI><MI class="MJX-tex-mathit" mathvariant="italic">t</MI><MI class="MJX-tex-mathit" mathvariant="italic">p</MI><MI class="MJX-tex-mathit" mathvariant="italic">u</MI><MI class="MJX-tex-mathit" mathvariant="italic">t</MI><MO class="MJX-tex-mathit" mathvariant="italic" stretchy="false">]</MO></MROW><MO>=</MO><MSUB><MROW class="MJX-TeXAtom-ORD"><MO class="MJX-tex-mathit" mathvariant="italic" stretchy="false">[</MO><MI class="MJX-tex-mathit" mathvariant="italic">O</MI><MI class="MJX-tex-mathit" mathvariant="italic">u</MI><MI class="MJX-tex-mathit" mathvariant="italic">t</MI><MI class="MJX-tex-mathit" mathvariant="italic">p</MI><MI class="MJX-tex-mathit" mathvariant="italic">u</MI><MI class="MJX-tex-mathit" mathvariant="italic">t</MI><MO class="MJX-tex-mathit" mathvariant="italic" stretchy="false">]</MO></MROW><MROW class="MJX-TeXAtom-ORD"><MI>m</MI><MI>a</MI><MI>x</MI></MROW></MSUB><MROW class="MJX-TeXAtom-ORD"><MI class="MJX-tex-mathit" mathvariant="italic">R</MI><MI class="MJX-tex-mathit" mathvariant="italic">N</MI><MI class="MJX-tex-mathit" mathvariant="italic">A</MI><MI class="MJX-tex-mathit" mathvariant="italic">i</MI></MROW><MO stretchy="false">(</MO><MTEXT>FF4,down_miR</MTEXT><MO stretchy="false">)</MO><MI>A</MI><MO stretchy="false">(</MO><MROW class="MJX-TeXAtom-ORD"><MI class="MJX-tex-mathit" mathvariant="italic">L</MI><MI class="MJX-tex-mathit" mathvariant="italic">a</MI><MI class="MJX-tex-mathit" mathvariant="italic">c</MI><MI class="MJX-tex-mathit" mathvariant="italic">I</MI></MROW><MO stretchy="false">)</MO></MATH></SPAN><P>This biophysical model expresses an input-output relationship between the miR inputs and the classifier output. Given an miRNA expression dataset, we can calculate the expected output for each sample and evaluate the performance of a concrete classifier.</P></DIV></SECTION></ARTICLE></SECTION><SECTION><HEADER><STRONG class="sub_headers" style="font-size: 55px;">Classifier design process</STRONG></HEADER><ARTICLE><HEADER><STRONG class="sub_headers">Circuit optimization algorithm</STRONG></HEADER><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>pANDORRA’s goal is to discriminate 2 groups of cells. If there is an miRNA expression dataset for these cells, we need to find the optimal circuit inputs that maximize output in cancer cells and minimize it in control cells. Thanks to the evaluation function derived above, if we have a way to generate candidate circuits, we can employ a search method to optimize the logical circuit.</P><P>The search space is huge and looking at every possible combination of circuit inputs is impractical; the search is guided instead by a genetic algorithm. The search begins at the simplest circuits: those with a single up- or downregulated miRNA. At each search step, the pool of candidate circuits is evaluated and only the top best-performing are kept. These are then used as a seed to propose new circuits by randomly recombining them together or adding to them single miRNAs. This process is repeated until the best-performing circuit isn’t dethroned for enough steps [<A href="#ref2a">2</A>]. The parallels between this process and natural selection and genetic recombination are obvious, justifying the name “genetic algorithm”.</P><P>Optimizing the logical circuit therefore requires selecting an architecture first (to have a concrete evaluation function). However, if there are many candidate architectures it can’t generally be known a priori which one will perform best on a particular dataset. Different architectures can be sensitive to different levels of miRNA input - imagine using a particularly strong repressor in the middle layer: low concentrations would be sufficient to completely knock out the output. The only way this classifier would produce output is if the levels of the upregulated miRNAs are exceedingly high on an absolute scale. This architecture wouldn’t work well to recognize cancer cells whose upregulated miRNAs are expressed in lower levels. Therefore, deciding on an architecture is also an optimization problem that should be solved for a specific dataset.</P><P>Contrary to the huge search space in circuit optimization, the biologically realizable architectures are few and each can be examined closely. To optimize across the architectural axis then, one need only find the best classifier circuit given the current architecture’s evaluation function. In the following section, we examine the various architectures in more detail.</P></DIV></SECTION></ARTICLE></SECTION><SECTION><ARTICLE><HEADER><STRONG class="sub_headers">Classifier Architectures</STRONG></HEADER><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>The first proposed architecture is straightforward: it accommodates upregulated miR on the top layer, downregulated miR directly on the output layer and couples the upregulated ones with 2 parallel mechanisms in the middle layer: transcriptional repression of the output layer by LacI and post-transcriptional inhibition by FF4.</P><P>The second architecture removes the parallel inhibition mechanism by FF4 and loosens the coupling of low levels of upregulated miR to the output. At the same time, by adding upregulated miR targets at the middle layer, it increases the coupling of high levels of upregulated miR to the output.</P><P>By removing upregulated miR targets on the upper layer, the third architecture allows greater LacI leakage when the upregulated miR are high.</P><P>The fourth architecture is very similar to the third, only simpler to implement.</P><P>The fifth architecture again combines the complementary couplings provided by LacI and FF4.</P><P>By reestablishing the upper layer, we get architecture six.</P><P>And by removing LacI we get architecture seven.</P></DIV></SECTION></ARTICLE></SECTION><SECTION><ARTICLE><HEADER><STRONG class="sub_headers">Transcriptional vs Post-transcriptional Repression</STRONG></HEADER><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>The basic variable element among these architectures is the method of double inversion. The two middle layer coupling mechanisms we explore, LacI and FF4, are used individually or combined. To assess the significance of this decision, we explore the input-output relationship for the corresponding architectures by scanning a pair of input upregulated miRNAs connected together in the same OR gate along the biologically plausible range they can cover.</P><P>There is stark difference in their behaviour. LacI maps low counts of both the miRNAs to relatively low output levels (OUT<SUB>max</SUB> = 10000) and high counts of either on to high output levels, thus implementing the OR gate. Its transition occurs very rapidly at relatively low miRNA counts, while its on/off ratio is 20. FF4 on the other hand is even more effective at mapping low levels of both miRNA to low output levels and high levels of both to high output, but maps high levels of only one of them to middle output, therefore implementing an arithmetic AND gate more than a logical OR gate. Its transition is much more gradual than LacI’s, leaving a large zone of uncertain output, while its overall on/off ratio is 15. It also appears that the FF4’s presence masks the effect of LacI.</P><P>AND gates are implemented more successfully, at least for upregulated miRNAs. Although the overall on/off ratio remains the same, the output is more consistent with a logical expression when one miRNA is high and the other is low.</P><P>Overall this analysis shows that LacI is more appropriate to implement logic gates. However, it is not a silver bullet. Its transition occurs at a point that might be inappropriate for some datasets, leaving even the miRNAs that are “low” on the positive side of the output. </P></DIV></SECTION></ARTICLE></SECTION><SECTION><ARTICLE><HEADER><STRONG class="sub_headers">miRNA Target Tandem Repeats</STRONG></HEADER><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>An interesting possibility is the adaptation of the number of target sites where each miRNA can attach to cause interference. It has been shown by [3] that there is little interaction between multiple miRNA and RISC complexes attaching next to each other, allowing us to model the overall dynamics additively. This assumption is challenged when there are low copy numbers of the participating molecules, but let’s follow it as a working hypothesis. Additive RNAi dynamics implies that, given an interference strength x by some miRNA for 1 repeat of the target site, if there are multiple repeats the interference becomes n*x.</P><P>To examine the effect of target site repeats, we’ll use an example dataset and circuit on the previous architectures. </P><P>Our Caco-2 and healthy dataset was obtained from the work by Cummins et al., 2006 [<A href="#ref4a">4</A>]. The miRNA expression data regarding the HEK-293 and A549 cell lines were retrieved from the mammalian microRNA expression atlas [<A href="#ref5a">5</A>]</P></DIV></SECTION><SECTION class="sub_sections"><HEADER><STRONG class="sub_headers">Architecture &amp; Circuit Optimization</STRONG></HEADER><DIV style="text-align: justify;"><P>We use the methods and insights derived so far to implement classifiers for our lab experiments. The experimental goal is to prove the ability of an optimized classifier to target cells from the Caco-2 cell line versus a healthy tissue control. Due to practical considerations, the controls on the actual experiments were the HEK-293 and A549 cell lines.</P><P>Running the circuit optimization algorithm on a dataset comprising Caco-2 and healthy samples [<A href="#ref4a">4</A>] for the various architectures suggests the following circuits for each:</P><DIV style="text-align: center"><TABLE><TBODY><TR><TD>
								- where “cMargin” is the logarithm of the fold change
							</TD></TR></TBODY></TABLE></DIV><P>Apparently the best architectures are indeed the ones that use LacI in the middle layer and don’t use FF4 (which overrules LacI’s influence).</P><P>The LacI architectures are optimal with a very small number of inputs. The others have more inputs, but due to the way the optimization works, many of those inputs might be of only marginal significance. It is then in our best interest from a designer's perspective to attempt to simplify the proposed circuits - a process called “pruning”. Taking cues from LacI, we attempt to prune the other architectures to the same small circuit, which is a subset of all the larger circuits.</P><P><I>Experiments are expensive and models are useful lies</I>; acknowledging this statement, it is also useful to put the results of our model in a broader perspective provided by the literature and increase our chances of a successful experiment. miR-21 was included due to exceptionally high expression level in most cance cell lines but not in the majority of healty and immortalized tissues [<A href="#ref2a">2</A>]. While examining miR-373, we notice it shares the same seed sequence with miR-372. The optimization algorithm takes this similarity into account at a preprocessing step, adding the expressions of miR-372 and miR-373 together and reporting only miR-372.</P><P>Putting all these elements together leads us to propose the following circuit for the wet lab experiments:</P><UL><LI style="list-style: disc">Architecture: <STRONG>2 (LacI + miR targets at middle layer)</STRONG></LI><LI style="list-style: disc">Circuit:<SPAN style="text-align: center"><STRONG>hsa-mir-372 OR hsa-mir-373
							AND
							hsa-mir-21
							AND
							~hsa-mir-143
							AND
							~hsa-mir-145</STRONG></SPAN></LI></UL></DIV></SECTION></ARTICLE></SECTION><SECTION><ARTICLE><HEADER><STRONG class="sub_headers">Re-evaluation of the proposed circuit</STRONG></HEADER><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>Although we’ve determined architecture 2 to be optimal, we reevaluate the proposed circuit across all the architectures. The resulting logarithm of the fold change is:</P><TABLE style="position: relative; left: 50%; margin-right: -50%; transform: translate(-50%, 0%);"><TBODY><TR><TD>arch1</TD><TD>arch2</TD><TD>arch3</TD><TD>arch4</TD><TD>arch5</TD><TD>arch6</TD><TD>arch7</TD></TR><TR><TD>1.45</TD><TD>2.69</TD><TD>2.64</TD><TD>2.65</TD><TD>0.90</TD><TD>1.67</TD><TD>0.77</TD></TR></TBODY></TABLE><P>Architecture 2 retains the same high fold change with the proposed circuit.</P></DIV></SECTION></ARTICLE></SECTION><SECTION><ARTICLE><HEADER><STRONG class="sub_headers">Implementation problems and fallback</STRONG></HEADER><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>We designed experiments to implement this final classifier circuit in the lab. The promoter bearing LacO operator sites, CAGop, was ordered to be synthesized, but unfortunately synthesis proved very difficult due to the high GC content and the construct arrived at the 31st of October. Necessity forced us to fall back to an alternate architecture for which we already had the Parts: architecture 7 (FF4). Thanks to the way we had structured the circuit elements, we also had some leeway in choosing which miRNA targets to actually include: miR-21, miR-372 OR miR-372, or both. We reevaluate these 3 options for architecture 7 and also compute the efficiency of the circuits on datasets with HEK-293 and A549 as controls, apart from healthy tissue samples.</P><P>Expected fold change logarithm for healthy samples:</P><TABLE style="position: relative; left: 50%; margin-right: -50%; transform: translate(-50%, 0%);"><TBODY><TR><TD>mir21 &amp; ~mir143 &amp; ~mir145</TD><TD>0.835</TD></TR><TR><TD>mir372|mir373 &amp; ~mir143 &amp; ~mir145</TD><TD>0.881</TD></TR><TR><TD>mir21 &amp; mir372|mir373 &amp; ~mir143 &amp; ~mir145</TD><TD>0.766</TD></TR></TBODY></TABLE><P>Expected fold change logarithm for the HEK-293 cell line:</P><TABLE style="position: relative; left: 50%; margin-right: -50%; transform: translate(-50%, 0%);"><TBODY><TR><TD>mir21 &amp; ~mir143 &amp; ~mir145</TD><TD>1.572</TD></TR><TR><TD>mir372|mir373 &amp; ~mir143 &amp; ~mir145</TD><TD>1.038</TD></TR><TR><TD>mir21 &amp; mir372|mir373 &amp; ~mir143 &amp; ~mir145</TD><TD>1.054</TD></TR></TBODY></TABLE><P>Expected fold change logarithm for the A549 cell line:</P><TABLE style="position: relative; left: 50%; margin-right: -50%; transform: translate(-50%, 0%);"><TBODY><TR><TD>mir21 &amp; ~mir143 &amp; ~mir145</TD><TD>0.620</TD></TR><TR><TD>mir372|mir373 &amp; ~mir143 &amp; ~mir145</TD><TD>1.138</TD></TR><TR><TD>mir21 &amp; mir372|mir373 &amp; ~mir143 &amp; ~mir145</TD><TD>1.068</TD></TR></TBODY></TABLE></DIV></SECTION></ARTICLE></SECTION><DIV id="references"><ARTICLE><DIV style="text-align: left"><HEADER><STRONG class="sub_headers">References</STRONG></HEADER></DIV><SECTION style="text-align:justify" class="sub_sections"><P id="ref1a">[1] Xie, Z., Wroblewska, L., Prochazka, L., Weiss, R., &amp; Benenson, Y. (2011). Multi-input RNAi-based logic circuit for identification of specific cancer cells. <I>Science, 333</I>(6047), 1307-1311.</P><P id="ref2a">[2] Mohammadi, P., Beerenwinkel, N., &amp; Benenson, Y. (2017). Automated Design of Synthetic Cell Classifier Circuits Using a Two-Step Optimization Strategy. <I>Cell Systems, 4</I>(2), 207-218.</P><P id="ref3a">[3] Schmitz, U., Lai, X., Winter, F., Wolkenhauer, O., Vera, J., &amp; Gupta, S. K. (2014). Cooperative gene regulation by microRNA pairs and their identification using a computational workflow. Nucleic acids research, 42(12), 7539-7552.</P><P id="ref4a">[4] Cummins, J. M., He, Y., Leary, R. J., Pagliarini, R., Diaz, L. A., Sjoblom, T., ... &amp; Raymond, C. K. (2006). The colorectal microRNAome. <I>Proceedings of the National Academy of Sciences of the United States of America, 103</I>(10), 3687-3692.</P><P id="ref5a">[5] Landgraf, P., Rusu, M., Sheridan, R., Sewer, A., Iovino, N., Aravin, A., ... &amp; Lin, C. (2007). A mammalian microRNA expression atlas based on small RNA library sequencing. <I>Cell, 129</I>(7), 1401-1414.</P></SECTION></ARTICLE></DIV></ARTICLE></DIV></DIV><DIV id="Protein_Structure_Prediction" width="100%" height="100%" class="display_frame hide"><DIV id="article_box"><ARTICLE id="general_content"><HEADER><STRONG class="sub_headers" style="font-size: 55px;">Protein Structure Prediction</STRONG></HEADER><BUTTON id="Section1_collapsable" class="accordion" style="font-size: 20px; letter-spacing: 1.5px;">Why do we need to predict the tertiary structure of proteins? - MOTIVATION</BUTTON><DIV id="panelPSP1" class="panel" style="text-align: justify; padding: 20px 0px 0px 0px;"><ARTICLE id="article_content"><SECTION id="section_1" style="margin: 0px;" class="sections"><P>Proteins consist of amino acid chains that fold in 3-dimensional space in ways we do not yet completely comprehend. Each protein chain spans from a handful of amino acids to more than a thousand residues, rendering the problem of determining the relationship between primary and tertiary structure immensely complex . </P><P>In our project, we employed a mutated fimH modifiedby adding the RPMrel peptide in order to achieve selective adhesion of our E. coli strains on cancer cells. In particular, we the fimH sequence was altered by substituting 1 (Proline to Glutamine) and inserting 28 new residues (RPMrel, SpyTag, HisTag). We would like to explore how these changes affect the 3-dimensional structure of the fimH protein. </P><P>To this end, we embarked on a journey to create an artificial neural network model that, given a protein's primary structure, can yield sufficient information to reconstruct the full 3-dimensional structure.</P><P>The idea is that our model can provide insight as to how the tertiary structure changes after the modifications, in that we can compare the native structure of the protein with the structure of the protein's modified sequence.</P><P>Moreover, such a model would be invaluable as it could predict structures de novo for proteins for which we do not yet have structural information. Such a protein is Apoptin, the toxin we used as our classifier's output in order to induce selective cell death.</P><P>Therefore, our motivation is 2-fold:</P><UL><LI style="list-style:none"><P>to study how a protein's structure changes after altering its primary sequence profile (in our case FimH)</P></LI><LI style="list-style:none"><P>to predict the de novo tertiary protein structures based solely on primary structure information (in our case Apoptin)</P></LI></UL><P>What we propose is an end-to-end pipeline that solely requires the amino acid sequence of the protein (primary structure) as input, predicts its corresponding secondary structure &amp; contact map and finally recreates and visualizes the full 3-dimensional tertiary structure.</P></SECTION></ARTICLE></DIV><BUTTON id="Section2_collapsable" class="accordion" style="font-size: 20px; letter-spacing: 1.5px;">From Amino acids to 0s and 1s - ENCODING PROTEINS</BUTTON><DIV id="panelPSP2" class="panel" style="text-align: justify; padding: 20px 0px 0px 0px;"><ARTICLE id="article_content"><SECTION id="section_2" style="margin: 0px;" class="sections"><P>In order to unravel the protein sequence-to-structure mystery one inevitably needs to summon the power of neural networks due to the aforementioned complexity and data volume. To fully exploit the generalization power of neural networks, we need to design a concise yet unabridged representation for our data, the protein sequences and their corresponding tertiary structures.</P><P>To make use of the primary and secondary structure sequence information we map the one-letter Amino acid codes to integers in range 1-20 and we use 0 to signify unknown encountered residues. We have now a vocabulary of 21 words that we project to a high-dimensional vector through the use of word embeddings [<A href="#ref1_psp" class="reference">1</A>], a technique widely used in natural language processing. Similarly, regarding the secondary structure, we have a vocabulary of 8 words that is mapped to high-dimensional vectors through embedding.</P><P>A common representation of a protein's tertiary structure is the contact map. This N x N matrix (N the number of residues in the chain) encompasses all the required information to reconstruct the 3-dimensional structure. Every (x,y) point on the map is interpreted as a contact probability between the residues x and y. </P><DIV style="text-align: center"><TABLE style="position: relative; left: 50%; margin-right: -50%; transform: translate(-50%, 0%);"><TBODY><TR><TD>The fimH contact map with distance cutoff 9 Å recreated from chain A of PDB id 2VCO</TD></TR></TBODY></TABLE></DIV><P>We can see that the majority of contacts span around the diagonal and that is only natural as it means that residues which are at a closer distance have a higher probability of forming a contact than distant ones. In general, secondary structures form patterns on the contact map, e.g. helices are adjacent to the diagonal, and provide a foundation for the prediction of the most difficult long-range contacts. That being said, we first need to obtain information about the secondary structure and subsequently use that to assist in contact prediction.</P><P>Since, we have access to ground truth for every protein sequence in our dataset we can employ supervised learning approaches. Secondary structure prediction is a classification problem of 8 classes, where we need to predict the secondary structure class of every residue, while providing context information for residues of the same sequence. Contact map prediction can be interpreted as a regression problem, where one needs to predict the contact probabilities for every pair of residues in the chain. </P></SECTION></ARTICLE></DIV><BUTTON id="Section3_collapsable" class="accordion" style="font-size: 20px; letter-spacing: 1.5px;">Gaining local structural insight - SECONDARY STRUCTURE PREDICTION MODEL</BUTTON><DIV id="panelPSP3" class="panel" style="text-align: justify; padding: 20px 0px 0px 0px;"><ARTICLE id="article_content"><SECTION id="section_3" style="margin: 0px;" class="sections"><P>The first step, given the protein's primary structure is to make an educated guess about its secondary structure. In 2017, we don't even need to do that. We can have machine learning models do that for us.</P><P>Although secondary structure provides valuable information for the local structural entities along the chain, it is not enough on its own to produce the contact map. Therefore, we can interpret this step, in terms of machine learning terminology, as an internal representation of the input that is passed on deeper in the network to assist contact prediction. However, since secondary structure has a semantic meaning of its own in the eyes of biologists, we treat the two models separately as it is standard practice in the literature.</P><P>Our Secondary Structure Predictor consists of a wide bidirectional recurrent layer, followed by a number of fully connected layers and the output layer. The model trains on primary sequence samples and predicts the secondary structure class for every Amino acid.</P><P>In a nutshell, the sequence passes through the recurrent layer and the model “remembers” what has already happened in the sequence -in terms of secondary structures- to provide context for every subsequent residue that it sees. The bigger the size of the recurrent layer the greater the memory capacity of the network and, in turn, its predictive power. </P><P>The additional layers that follow progressively reduce the representation dimension until we are left with a 1-hot vector (1 x 8) that indicates the most prevalent class choice. In the end, the full predicted sequence corresponds to all secondary structure segments.</P><DIV style="text-align: center"><TABLE style="position: relative; left: 50%; margin-right: -50%; transform: translate(-50%, 0%);"><TBODY><TR><TD>Top Row: primary structure sequence of one-letter Amino acid codes</TD></TR><TR><TD>Bottom Row: secondary structure sequence of one-letter class of the 8 classes</TD></TR></TBODY></TABLE></DIV></SECTION></ARTICLE></DIV><BUTTON id="Section4_collapsable" class="accordion" style="font-size: 20px; letter-spacing: 1.5px;">Inferring the contacts - CONTACT MAP PREDICTION MODEL</BUTTON><DIV id="panelPSP4" class="panel" style="text-align: justify; padding: 20px 0px 0px 0px;"><ARTICLE id="article_content"><SECTION id="section_4" style="margin: 0px;" class="sections"><P>Our end goal is to predict contacts between residues in a pairwise fashion. For a sequence of N Amino acids we want to derive a N x N map, where each position shows the probability of the two residues being in contact.</P><P>At first, we need to build a N x N map, hereafter referred to as “tensor”, that will serve as the input layer to our network and will be updated after each epoch as the network learns to solve the regression task. The tensor captures the semantic relationship between residues and constantly changes as training progresses.</P><P>In the previous section, we mentioned how we use embeddings to map our single-letter Amino acid codes to high-dimensional vectors. That is, for a primary structure sequence of N residues, we end up with a N x m matrix, where m is the size of the embedding. </P><P>In order to exploit the additional information we have for every Amino acid about its secondary structure classification, we employ, once again, word embeddings of size k that we stack to our previous m-dimensional vector. Now, every Amino acid is encoded as a concatenation of the two embeddings with a resulting vector of size m + k.</P><P>The secondary structure information may be incorporated directly as a result of our Secondary Structure Predictor model or - for proteins with known structures- can be provided directly as additional input. However, the latter can only be of use for model evaluation as, in practice, we are more interested to predict structures de novo.</P><P>The next step is to create a representation for every pair of Amino acids in order to create the full N x N tensor. For a pair (x,y) of Amino acids we concatenate their respective embeddings thus creating a new vector of size 2 * (m + k). As an alternative, we can perform an element-wise operation (e.g. multiplication) between the two distinct residue embeddings and maintain the resulting pair embedding dimension to m + k. </P><P>Now that we have constructed our tensor, we add several 2-dimensional convolutional layers to scan every Amino acid pair on our map for contacts. When training is complete, the resulting map is compared to the native contact map by specifying a probability threshold that distinguishes contacts from non-contacts.</P></SECTION></ARTICLE></DIV><BUTTON id="Section5_collapsable" class="accordion" style="font-size: 20px; letter-spacing: 1.5px;">Recreating proteins in three dimensions - TERTIARY STRUCTURE RECONSTRUCTION</BUTTON><DIV id="panelPSP5" class="panel" style="text-align: justify; padding: 20px 0px 0px 0px;"><ARTICLE id="article_content"><SECTION id="section_5" style="margin: 0px;" class="sections"><P>The final step of the process is to retrieve the 3d structure from the predicted contact map. To achieve that, we feed the contact map to <A href="http://bioinformatics.cs.unibo.it/FT-COMAR/">FT-COMAR</A> ], a tool that is able to recreate the tertiary structure by reading a contact map with values 1 (contact), 0 (no contact), -1 (uncertain) [<A href="#ref2_psp" class="reference">2</A>]. In our implementation, a contact is declared uncertain if the predicted probability is between 0.3 and 0.6. The resulting 3-dimensional information is written to a file that we can load to <A href="https://www.ncbi.nlm.nih.gov/Structure/icn3d/full.html">iCin3D</A> to visualize and interact with the reconstructed structure.</P></SECTION></ARTICLE></DIV><BUTTON id="Section6_collapsable" class="accordion" style="font-size: 20px; letter-spacing: 1.5px;">What have we learned? - PRELIMINARY RESULTS</BUTTON><DIV id="panelPSP6" class="panel" style="text-align: justify; padding: 20px 0px 0px 0px;"><ARTICLE id="article_content"><SECTION id="section_6" style="margin: 0px;" class="sections"><P>Due to limited access to computational resources, we were not able to train models of adequate performance to assess the changes in FimH tertiary structure or visualize the estimated structure of Apoptin in a reliable way. Although the models' performance were monotonously improving we could not reach convergence before the wiki freeze, as we were restricted by shallow architectures and small mini-batch sizes. </P><P>For example, for the Secondary Structure Predictor we obtained a 27% Q8 accuracy using a relatively shallow 2-layer architecture. Q8 accuracy measures the percent of residues for which 8-state secondary structure is correctly predicted. For the Contact Map Predictor, we set a probability threshold of 0.45 to determine whether two residues are in contact and the model correctly predicted 43% of all contacts.</P><P>We will continue to work towards improving the model performances as well as attempt different approaches to solve the contact map.</P><P>However, for the sake of completeness, we employed a publicly available tool that offers similar functionality but different approach. RaptorX-Contact-Predict is a web server tool that predicts the contact map and provides direct visualization of the resulting tertiary structure through JSmol.</P><DIV style="text-align: center"><TABLE><TBODY><TR><TD>Contact Map comparison of fimH wild type and our modified fimH. Black dots signify common contacts that exist in both structures, green dots the unique contacts in wild type fimH and the magenta dots the unique contacts that exist in our modified fimH sequence. The locations where the RPMrel and tags were inserted are evident in the contact map in places where there exist only magenta dots along the diagonal.</TD></TR></TBODY></TABLE></DIV><TABLE><TBODY><TR><TD>The top left structure corresponds to wild type fimH, the bottom left to our modified fimH. To the right, we see the two structures superimposed, with wild type fimH shown in grey.</TD></TR></TBODY></TABLE></SECTION></ARTICLE></DIV><BUTTON id="Section7_collapsable" class="accordion" style="font-size: 20px; letter-spacing: 1.5px;">A Deep Dive into Specifics - IMPLEMENTATION DETAILS</BUTTON><DIV id="panelPSP7" class="panel" style="text-align: justify; padding: 20px 0px 0px 0px;"><ARTICLE id="article_content"><SECTION id="section_7" style="margin: 0px;" class="sections"><HEADER><STRONG class="sub_headers">Data</STRONG></HEADER><P>Our dataset is a set of 10932 proteins from the PDB database, that were selected based on certain criteria using the <A href="http://dunbrack.fccc.edu/Guoli/pisces_download.php">PISCES</A> server. Specifically, the percentage identity cutoff is 60%, the resolution cutoff is 1.8 angstroms, and the R-factor cutoff is 0.25. </P><P>We observed that 9754 out of 10932 proteins have some missing residues, for which there is no structural information available. In all such cases, the missing residues were discarded and the sequence truncated. 21 proteins were excluded from the dataset as there was no available secondary structure information whatsoever. </P><P>We split the dataset of 10911 remaining proteins, into 8728 for training and 2183 for validation. In addition, we tested our Secondary Structure Predictor to the benchmark dataset CB513 in order to compare our approach with existing methods.</P></SECTION><SECTION id="section_7a" style="margin: 0px;" class="sections"><HEADER><STRONG class="sub_headers">Labels</STRONG></HEADER><P>For the Secondary Structure Predictor the required label for every Amino acid in the sequence is the corresponding secondary structure information. Every Amino acid belongs to one of the 8 classes [<A href="#ref3_psp" class="reference">3</A>]. Hence, every protein sequence is paired with its label, a sequence of equal length consisting of the class id for every Amino acid.</P><P>To derive secondary structure labels for our dataset, we employed STRIDE, an algorithm designed for the assignment of protein secondary structure elements given the atomic coordinates of the protein, as defined by X-ray crystallography, protein NMR, or another protein structure determination method [<A href="#ref4_psp" class="reference">4</A>].</P><P>For the Contact Map Predictor the required label is the native contact map. To obtain the contact map we need the structural information provided in the PDB file of every protein sample, that is the atomic coordinates of every Amino acid. The contact map is constructed by computing the Cb distance for every pair of Amino acids in the chain and ,subsequently, if that distance satisfies a specified threshold [<A href="#ref5_psp" class="reference">5</A>] the pair is considered to be in contact. </P><P>To extract the native contact map for every PDB file, we implemented a group of helper functions that allow for the generation of contact maps of different distance cutoffs and distance types, namely Ca, Cb and Ca + Cb. </P></SECTION><SECTION><HEADER><STRONG class="sub_headers">Model Architecture</STRONG></HEADER><P>The Secondary Structure Predictor model consists of an embedding layer that maps the one hot Amino acid representations to 100-dimensional vectors, a bidirectional LSTM layer of 180 cells and a fully connected layer of 140 neurons with linear activation function. Finally, we have the output softmax layer of 8 neurons that points to the most probable class choice. The training loss is calculated according to the formula of categorical cross entropy.</P><P>The Contact Map Predictor model has a tensor input that is feeded to 6 2D-convolutional layers. After each convolutional layer we enforce batch normalization to prevent exploding gradients. The output layer that computes the probability of contact has a linear activation function.</P></SECTION><SECTION><HEADER><STRONG class="sub_headers">Frameworks</STRONG></HEADER><P>The models were developed in Python using the Theano and Keras frameworks. </P></SECTION></ARTICLE></DIV><DIV id="references"><ARTICLE><HEADER><STRONG class="sub_headers">References</STRONG></HEADER><SECTION style="text-align:justify" class="sub_sections">
					[<SPAN id="ref1_psp">1</SPAN>] Mikolov, T., Sutskever, I., Chen, K., Corrado, G. S., &amp; Dean, J. (2013). Distributed representations of words and phrases and their compositionality. In Advances in neural information processing systems (pp. 3111-3119).
					[<SPAN id="ref2_psp">2</SPAN>] Vassura, M., Margara, L., Di Lena, P., Medri, F., Fariselli, P., &amp; Casadio, R. (2008). FT-COMAR: fault tolerant three-dimensional structure reconstruction from protein contact maps. Bioinformatics, 24(10), 1313-1315.
					[<SPAN id="ref3_psp">3</SPAN>] Kabsch, W., &amp; Sander, C. (1983). Dictionary of protein secondary structure: pattern recognition of hydrogen‐bonded and geometrical features. Biopolymers, 22(12), 2577-2637.
					[<SPAN id="ref4_psp">4</SPAN>] Frishman, D., &amp; Argos, P. (1995). Knowledge‐based protein secondary structure assignment. Proteins: Structure, Function, and Bioinformatics, 23(4), 566-579.
					[<SPAN id="ref5_psp">5</SPAN>] Duarte, J. M., Sathyapriya, R., Stehr, H., Filippis, I., &amp; Lappe, M. (2010). Optimal contact definition for reconstruction of contact maps. BMC bioinformatics, 11(1), 283.</SECTION></ARTICLE></DIV></ARTICLE></DIV></DIV><DIV id="Colon_Cancer_Fluid_Dynamics" width="100%" height="100%" class="display_frame hide"><DIV id="article_box"><ARTICLE id="static_content"><HEADER><STRONG class="sub_headers" style="font-size: 55px;">CFD – Large Intestine (Colon) Model</STRONG></HEADER><SECTION><BUTTON id="Section_a_collapsable" class="accordion" style="font-size: 20px; letter-spacing: 1.5px;">Overview – Goals</BUTTON><DIV id="panelCCFDa" class="panel" style="text-align: justify; padding: 20px 0px 0px 0px;"><ARTICLE><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>Let's imagine that the classifier plasmids are ready to be used in vivo, thus, in clinical trials.</P><UL><LI>How can we predict the path, which the genetically engineered bacteria will follow inside the large intestine?</LI><LI>How can we be certain that our bacteria will eventually reach the area around the cancerous tissue?</LI><LI>Will our bacteria attach to the carcinoma or will the Colon Chyme wash them out? </LI></UL><P>The purpose of our Colon Fluid Dynamics model is to give an answer to those questions and to further explore the unknown and chaotic world of the human large intestine.</P></DIV></SECTION></ARTICLE></DIV><BUTTON id="Section_b_collapsable" class="accordion" style="font-size: 20px; letter-spacing: 1.5px;">The Large Intestine</BUTTON><DIV id="panelCCFDb" class="panel" style="text-align: justify; padding: 20px 0px 0px 0px;"><ARTICLE><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>In our project, the RNAi-based apoptotic circuit is to be eventually distributed via the gastrointestinal system towards the tumor inside the large intestine. Thus, it is essential that we examined carefully the properties of the colon's environment. </P><P>The large intestine is the last section of the gastrointestinal tract that is responsible for absorbing water and nutrients and concurrently converting digested food into feces. (Even though the length of the colon is considerably smaller than the small intestine's, the thickness of the walls as well as its diameter are bigger.) The length of the colon approaches the mean value of 1.5m and 6-7cm in diameter in most humans.</P><P>The Chyme (intestinal fluid containing the digested food) is transported from the small intestine towards the large intestine via the ileocecal sphincter. Chyme “settles” in the cecum, after its entrance into the large bowel, where it is mixed with bacteria which naturally reside in it constituting the gut's flora and contribute to much of the large intestine's functionality. The chyme is transferred consequently from one haustra to another by slow peristaltic waves. Those waves can be either mass movements, which are very slow and widespread movements happening a few times throughout the day and associated with food consumption by means of the gastrocolic and duodenocolic reflex or segmentation movements that mostly serve to chop and mix the intestinal content.</P><P>Peristalsis is a radially symmetrical contraction and relaxation of muscles that propagates as a wave down the tube (in our case the intestinal tube), in an anterograde direction. In the case of the human intestinal tract, smooth muscle tissue contracts in sequence to produce a peristaltic wave, which propels fecal aggregates along the tract. This is the main kind of movement that will be taken into account in our model. More technical details on the physical model of peristalsis are presented in the Human Colon Fluid Dynamics section of our model.</P><P>The large intestine can be considered a mechanical propulsion system implying that one should determine its elastic and viscous properties before proceeding with the model. </P><P>Some primitive tensile properties of the human large intestine are the maximal stress and destructive strain, valued at 0.9 MPa and 180% respectively<SUP>[<A href="#ref1_ccfd">1</A>]</SUP>. In addition, to gauge the elastic properties of the intestinal wall one needs to determine the elastic modulus (Young's modulus) as well as the Poisson Ratio.</P><P>Young's modulus (E) is a numerical constant that describes the elastic properties of an elastic material undergoing tension or compression unidirectionally and functions as ameasure of the material's capacity to withstand alterations in its shape. The mathematical formula describing this physical entity is the following:</P><SPAN class="equation">\[E = \frac{{stress}}{{strain}} = \frac{{F{L_0}}}{{A({L_n} - {L_0})}}\]</SPAN><P>The physical units of E in the metric system are Newtons per square meter (N/m<SUP>2</SUP>).</P><P>Poisson's Ratio (σ) is the ratio of transverse contraction strain to longitudinal extension strain in the direction of the stretching force applied on the material's surface. The large intestinal tissue is commonly considered as soft tissue and the mathematical formula describing its Poisson's ratio is the following:</P><SPAN class="equation">\[\sigma  =  - \frac{{d{\varepsilon _{trans}}}}{{d{\varepsilon _{axial}}}}\]</SPAN><P>Those values exist in the literature; as a mean value across all locations of the colon (distal, medial and proximal) as well as in every possible orientation (circumferental and longtitudinal) for the Elastic Modulus equal to 5.18 MPa and as a commonly accepted value of 0.5 for incompressible materials for soft tissue, such as the colon.[<A href="#ref2">2</A>,<A href="#ref3">3</A>]</P></DIV></SECTION></ARTICLE></DIV><BUTTON id="Section_c_collapsable" class="accordion" style="font-size: 20px; letter-spacing: 1.5px;">Human Colon Fluid Dynamics</BUTTON><DIV id="panelCCFDc" class="panel" style="text-align: justify; padding: 20px 0px 0px 0px;"><ARTICLE><HEADER><STRONG class="sub_headers">Human Colon Fluid Dynamics </STRONG></HEADER><SECTION class="sub_sections"><DIV style="text-align: justify;"><SPAN>Our genetically engineered bacteria are supposed to navigate towards the cancerous tissue inside the large intestine via the gastrointestinal tract's chyme. As mentioned before, chyme is a fluid consisting of partly digested food flowing through the digestive tract. Thus, in order to determine the bacterial flow characteristics, a fluid dynamics analysis is mandatory. </SPAN><ARTICLE><DIV style="text-align: left"><HEADER><STRONG class="sub_headers">GEOMETRY</STRONG></HEADER></DIV><P>Colon cancer constitutes one of the most commonly diagnosed cancers worldwide, thus one could easily consider the importance of determining its morphology. More often than not, colorectal cancer emerges as a malignant transition of a polyp, therefore one could assume that they share their macroscopic geometry, particularly in the early stages.</P><P>In our Colon Cancer CFD model, we assumed that the polyp, which evolved to cancer, is found in the proximal ascending colon of the large intestine as it is one of the most common colon regions where carcinomas are found (together with the Sigmoid Colon). Thus, the geometry of our model represents the caecum and a major part of the ascending colon. The CAD file for the geometry was designed with the “Autodesk – Inventor Professional” software.</P><TABLE><TBODY><TR><TD><P>It contains 2D identical structures which were then revolved into 3D objects and unified as an assembly so that they form a 3D solid structure.
	The round surface on the left of the solid represents the fluid inlet, thus it could be considered as the final part of the caecum </P></TD></TR></TBODY></TABLE><P>region close to the ileocecal sphincter. After the inlet, one could see the characteristic “curved cylindrical” shapes which represent the intestinal haustra. On the upper right side of the image is the fluid outlet which could be considered as the section close to the hepatic flexure which then continues to the transverse colon. </P><P>After the construction of the outer intestinal geometry, the geometry of the cancerous polyp was added to the structure. By adding a 2D sketch on a working plane parallel to one of the haustra and by constructing a mushroom-like shaped solid, we were able to design the tumor inside the colon.</P><TABLE><TBODY><TR><TD><P>The tumor is the blue object shown on the diagram. The solid which we created is very similar to the anatomical morphology of a human polyp. </P></TD></TR></TBODY></TABLE></ARTICLE></DIV></SECTION></ARTICLE><ARTICLE><DIV style="text-align: left"><HEADER><STRONG class="sub_headers">FLUID DYNAMICS GOVERNING EQUATIONS</STRONG></HEADER></DIV><SECTION class="sub_sections"><DIV style="text-align: justify"><P>In this section we examine the physical laws behind the flow of the gastrointestinal fluids.</P><P>The basic differential equations that govern the flow of fluids are the Navier – Stokes equations and the continuity equation. The mathematical form of those equations is as follows:</P><SPAN class="equation">\[\rho \left( {\frac{{\partial {\bf{V}}}}{{\partial t}} + {\bf{V}} \cdot \nabla {\bf{V}}} \right) =  - \nabla p + \rho {\bf{g}} + \nabla  \cdot \left( {\mu (\nabla {\bf{V}} + {{(\nabla {\bf{V}})}^T})} \right) - \frac{2}{3}\mu (\nabla  \cdot {\bf{V}}){\bf{I}})\]</SPAN><SPAN class="equation">\[\rho \nabla  \cdot {\bf{V}} = 0\]</SPAN><P>where <STRONG>ρ</STRONG> (rho) stands for fluid density, <STRONG>V</STRONG> is the velocity vector of the fluid which produces a velocity field and <STRONG>μ</STRONG> (mu) stands for dynamic viscosity of the fluid. Let's analyze further each term in the equations:</P><UL><LI style="list-style: none"><SPAN class="equation" style="font-size: 25px;">$\rho \left( {\frac{{\partial {\bf{V}}}}{{\partial t}} + {\bf{V}} \cdot \nabla {\bf{V}}} \right)$</SPAN>: This term represents all the inertial forces.</LI><LI style="list-style: none"><SPAN class="equation" style="font-size: 25px;">$ - \nabla p$</SPAN>: This term all the pressure forces.</LI><LI style="list-style: none"><SPAN class="equation" style="font-size:25px;">$\nabla  \cdot \left( {\mu \left( {\nabla {\bf{V}} + {{(\nabla {\bf{V}})}^T}} \right) - \frac{2}{3}\mu (\nabla  \cdot {\bf{V}}){\bf{I}}} \right)$</SPAN>: This term all the viscous forces.</LI><LI style="list-style: none"><SPAN class="equation" style="font-size:25px;">$\rho {\bf{g}}$</SPAN>: This term represents the external gravitational force of weight.</LI></UL><P>All these terms combined together, comprise the well-known Navier-Stokes which represent the conservation of momentum. The continuity equation for incompressible fluids represents the conservation of mass in the system. Another way of writing the Navier – Stokes equations for incompressible fluids is the following one:</P><SPAN class="equation">\[\rho \left( {\frac{{\partial {\bf{V}}}}{{\partial t}} + {\bf{V}} \cdot \nabla {\bf{V}}} \right) =  - \nabla p + \rho {\bf{g}} + \mu {\nabla ^2}{\bf{V}}\]</SPAN><P>These equations generally hold for incompressible Newtonian fluids. In the case of fecal matter, we consider the fluid to be Newtonian as it is composed mainly of liquid water (close to 90%) while inside the caecum and ascending colon and because the relationship between the viscous stresses and the local strain rate is linear.</P><P>Once we know the velocity field <STRONG>V</STRONG> as well as the pressure <STRONG>p</STRONG> in every spatial point of our geometry, we can determine the stress tensor, which in mathematical terms stands as follows:</P><SPAN class="equation">\[{\bf{\sigma }} =  - p{\bf{I}} + \eta [\nabla {\bf{V}} + {(\nabla {\bf{V}})^T}]\]</SPAN><P>From the stress tensor one could determine the values of two other important fluid mechanical quantities, the hydrodynamic force and the hydrodynamic torque, which are expressed mathematically as follows:</P><UL><LI>Hydrodynamic Force: <SPAN class="equation">\(F(t) = \iint\limits_S({{\bf{\sigma }} \cdot {\bf{n}}}) dS\)</SPAN></LI><LI>Hydrodynamic Torque: <SPAN class="equation" style="font-size: 25px;">$$L(t) = \iint {{\bf{x}} \times ({\bf{\sigma }} \cdot {\bf{n}})dS}$$</SPAN></LI></UL><P>These two integrals are calculated on the surface S and x stands for the positions on that surface.</P><P>We should refer to a common dimensionless group in fluid dynamics that we used in our model. The ratio of inertial forces to viscous forces, Reynold's number (Re), is one of the most important dimensionless groups in fluid dynamics. The mathematical representation of Re is the following one:</P><SPAN class="equation">\[{\mathop{\rm Re}\nolimits}  = \frac{{\rho Vl}}{\mu }\]</SPAN><P>where <STRONG>ρ</STRONG> is the fluid density, <STRONG>V</STRONG> the fluid velocity, <STRONG>l</STRONG> the characteristic length (characteristic area in 3D objects) and μ the dynamic viscosity. The Reynold's number could be used as an indicator in order to determine whether a specific kind of fluid motion is laminar or turbulent of even transitional. In our case, the Reynold's number has a very low value equal to 0.00003 and thus, the viscous effects dominate the fluid. The reason why Reynold's number takes such a low value is that if we zoomed in the fluid we could see bacteria swimming at the same velocity as the fluid velocity field. Assuming that the size of a bacterium is about 10<SUP>-6</SUP>m, their swimming velocity is 30x10<SUP>-6</SUP> m/s and the density of the fluid in which the bacteria swim is 1000 kg/m<SUP>3</SUP>  and the dynamic viscosity is around 0.001 Pa.s, the Reynold's number turns out to be equal to 1x10<SUP>-5</SUP>.</P><BUTTON class="accordion" style="font-size: 30px; letter-spacing: 1.5px">Life at low Reynolds numbers</BUTTON><DIV class="panel" id="panelCCFDd" style="font-size: 25px; text-align: justify; color: orange; margin: 10px;"><I><P>Life at low Reynold's numbers: In environments such as a fluid with a very low Reynold's number, the particles which “live” inside the fluid are considered “swimmers” as they deform their surface to sustain their movement. In our case, the E. coli bacteria, which constitute an important part of the gut's flora, have helical flagella to help them with their locomotion. </P><P>If a bacterium suddenly stops deforming its body, it will become a “victim” of the inertial forces of the fluid and thus, it will decelerate. At low Reynold's numbers, the drag force acting on the bacterium takes the form of viscous forces, <SPAN class="equation" style="font-size: 25px">$f_{drag} \sim \eta UL$</SPAN> , and the bacterium can reach a distance of about 0.1 nm before stopping its movement. In contrast, at high Reynold numbers the distance is longer and thus, Re at low values can be interpreted as a non-dimensional coasting distance.</P><P>One could describe the locomotion of a bacterium in low Re as a function of the swimming gait <STRONG>${{\bf{u}}_s}(t)$</STRONG>(velocity field on the body surface). At every instant, it can be assumed that the body is solid with unknown velocity <STRONG>${\bf{U}}(t)$</STRONG> and rotation rate <STRONG>${\bf{\Omega }}(t)$</STRONG>. Thus, the instantaneous velocity on the body surface is <SPAN class="equation">${\bf{u}} = {\bf{U}} + {\bf{\Omega }} \times {\bf{x}} + {{\bf{u}}_s}$</SPAN>. To calculate in every time step the values of <STRONG>${\bf{U}}(t)$ </STRONG> and <STRONG>${\bf{\Omega }}(t)$</STRONG>, one needs to determine both the velocity and stress fields in the problem in order to utilize the following integral:</P><SPAN class="equation">$${\bf{\hat F}} \cdot U + {\bf{\hat L}} \cdot \Omega  =  - \iint_{S(t)} {{u_s} \cdot {\bf{\hat \sigma }} \cdot ndS}$$</SPAN><P>Assuming that the swimmer is a bacterium with filament, what makes its movement in low Re possible is mainly the existence of drag forces perpendicular to the motion of the filament. The propulsive force generated along the filament (length L and deformation amplitude y(x,t)) is given by:</P><SPAN class="equation">$${{\bf{F}}_{prop}} \approx \left( {{\xi _ \bot } - {\xi _{||}}} \right)\int_0^L {\left( {\frac{{\partial y}}{{\partial t}}\frac{{\partial y}}{{\partial x}}} \right)dx{e_x}} $$</SPAN><P>This equation is a consequence of Purcell's Scallop Theorem and is derived for Re -&gt; 0<SUP>[<A href="#ref4_ccfd">4</A>]</SUP>. </P></I></DIV></DIV></SECTION></ARTICLE><ARTICLE><DIV style="text-align: left"><HEADER><STRONG class="sub_headers">MESHING &amp; NUMERICAL SIMULATION</STRONG></HEADER></DIV><SECTION class="sub_sections"><DIV style="text-align: justify"><P>The software for the model's numerical simulations is Comsol Multiphysics (v. 5.0) thus, the numerical method utilized is Finite Element Method (FEM).</P><P>The number of physical problems that can be solved analytically is very small compared to all the real world physical systems that can be solved only by approximations. One way to find a solution to a complex physical problem such as fluid flow in the human large intestine is to utilize discretization methods that lead to numerical solutions. An example of space discretization method is the Finite Element Method which considers an analytical function as a linear combination of basis functions. </P><P>Comsol Multiphysics uses this method in order to compute the numerical solutions of f<SUB>ap</SUB> in every single node of the space discretization. However, in order for such calculations to be possible one needs to discretize the problem geometry into nodes and elements. The grid that is being produced by the software user is called a “Mesh” and the shape of its elements can vary. In our model, the tetrahedron mesh was utilized, which is the 3D representation of the 2D triangle mesh. Although the meshing was implemented by the software itself by using “physics-based” mesh, the discretization method was tetrahedral as it is easy and quick to create but it also creates an unstructured grid which in our case is helpful as there are domain regions with large difference in the required scales. For instance, in the proximity of the tumor we needed a finer meshing as the numerical solutions would converge to more accurate values that way.</P><TABLE><TBODY><TR><TD><P>The mesh generated by Comsol Multiphysics is a physics-controlled mesh with finer elements. The tetrahedral space discretization is visible on the mesh plot. We present here two mesh plots; the first shows the tetrahedral meshing of the intestinal geometry and the second one depicts the interior region where the tumor meshing is visible.</P></TD></TR></TBODY></TABLE></DIV></SECTION></ARTICLE><ARTICLE><DIV style="text-align: left"><HEADER><STRONG class="sub_headers">Initial Parameters &amp; Boundary Conditions</STRONG></HEADER></DIV><SECTION class="sub_sections"><DIV style="text-align: justify"><P>In CFD models one needs to determine the initial values of the parameters as well as the boundary conditions of the problem in order to get an accurate result from the solver.</P><P>The normal velocity of Chyme inside the small intestine is below 1 cm/s and inside the large intestine it's even slower. Thus, we assumed that a logical value for the inlet normal velocity is 0.5 cm/s. The outlet boundary condition is pressure and its value is 101.16 kPa, close to 1 atm <SUP>[<A href="#ref5_ccfd">5</A>]</SUP>.</P><P>The intestinal “tube” in our geometry is considered to be a no-slip wall in our study which implies that the velocity of the fluid particles close to the wall will tend to zero. The tumor region is a no-slip interior wall. </P></DIV></SECTION></ARTICLE></DIV><BUTTON id="Section_k_collapsable" class="accordion" style="font-size: 20px; letter-spacing: 1.5px;">Results</BUTTON><DIV id="panelCCFDe" class="panel" style="text-align: justify; padding: 20px 0px 0px 0px;"><ARTICLE><SECTION class="sub_sections"><DIV style="text-align: justify;"><P>The velocity field of the fluid flow inside the large intestine is quite complex, when peristalsis occurs, as the intestinal walls contract in a periodical manner in order to forward the colon contents towards the rectum where they are being secreted. However, that kind of study is beyond the scope of our model. In our model the inlet velocity is a consequence of the contraction of the caecum but we assume that the time scale of the peristalsis is bigger than the fluid movement's; thus, no peristaltic waves occur throughout our model and the fluid is driven by its inlet velocity and a pressure outlet which is close to 1 atm.</P><P>Also, viscosity is a rather important factor when it comes to fluid flow. The intestinal fluid, due to nutrients and water absorption, doesn't possess a constant viscosity value; although the site of absorption and phase change of Chyme is the Transverse Colon and not the Caecum and Ascending Colon, the digested food determines to a high degree the success of nutrients and water absorption. Thus, we considered essential to examine the fluid flow under different dynamic viscosity values.</P></DIV></SECTION></ARTICLE><ARTICLE><DIV style="text-align: left"><HEADER><STRONG class="sub_headers">Velocity Field – Dynamic Viscosity μ=0.6913 mPa∙s</STRONG></HEADER></DIV><SECTION class="sub_sections"><P>The dynamic viscosity in this section is the value of Water Dynamic Viscosity at T = 37<SUP>o</SUP>C. The following diagram shows the velocity field distribution inside the colon geometry.</P></SECTION></ARTICLE><ARTICLE><DIV style="text-align: left"><HEADER><STRONG class="sub_headers">Velocity Field – Dynamic Viscosity μ=0.01 Pa∙s</STRONG></HEADER></DIV><SECTION class="sub_sections"><DIV style="text-align: justify"><P>The following diagram shows the velocity field distribution inside the colon geometry.</P>
						\[\mathop {\lim }\limits_{x \to \infty } x\]
						</DIV></SECTION></ARTICLE><ARTICLE><DIV style="text-align: left"><HEADER><STRONG class="sub_headers">Velocity Field – Dynamic Viscosity μ=1 Pa∙s</STRONG></HEADER></DIV><SECTION class="sub_sections"><DIV style="text-align: justify"><P>The dynamic viscosity in this section is the value of Water Dynamic Viscosity at T = 37<SUP>o</SUP>C. The following diagram shows the velocity field distribution inside the colon geometry.</P></DIV></SECTION></ARTICLE><ARTICLE><DIV style="text-align: left"><HEADER><STRONG class="sub_headers">VELOCITY FIELD MAGNITUDE – TUMOR REGION </STRONG></HEADER></DIV><SECTION class="sub_sections"><DIV style="text-align: justify"><P>In these diagrams, the magnitude of the fluid's velocity field is given with respect to the arc length of the tumor. In the following diagram, we depict the boundary arc chosen.</P><P>One could observe that between the velocity magnitude maxima appear a few local minima which we decided to call “Velocity Field Valley Traps (VFVT)” (or Valleys of the Muses). In these regions, the velocity around them has a greater value, thus, the bacteria “swimmers” that will reach these regions will most likely stay there and attach to the tumor rather than being washed out by the fluid. We could consider those regions as local stagnation regions of the fluid. Although the local maxima for each dynamic viscosity value are found at about the same location on the arc boundary of the tumor, the local minima or VFVTs don't follow the same rule. Having said that, the VFVTs for viscosity values 0.01 Pa.s and 1 Pa.s are located on the same regions which implies that for viscosity changes in that range the regions of the tumor on which the bacteria would attach doesn't change. Those regions are:</P><P>However, for viscosity value 0.6913 mPa.s the VFVTs seem to have a slight displacement from the other VFVT locations. More specifically, the first and third VFVTs “moved” towards the top arc length which implies that for low viscosities the VFVTs appear closer to more curved regions of the tumor geometry. </P></DIV></SECTION></ARTICLE><ARTICLE><DIV style="text-align: left"><HEADER><STRONG class="sub_headers">DRAG FORCES ON THE TUMOR REGION</STRONG></HEADER></DIV><SECTION class="sub_sections"><DIV style="text-align: justify"><P>In our study of the fluid flow near the tumor, it is essential to determine the drag forces on the tumor if we want to predict whether our 'bactofectors' will manage to stay attached to the tumor. Thus, due to the small cohesive forces between the bacteria and the tumor surface, it is safe enough to assume that the total stress forces on the tumor due to the fluid will apply to the bacteria as well.</P><P>We computed the total drag forces on the tumor arc boundary (same as the velocity magnitude) by integrating the total stresses over the entire arc length. </P><P>What seems interesting in these diagrams is the fact that for different values of fluid viscosity the total stress on the boundary remains the same. Why does this happen?</P><P>Well, let's remind ourselves that by integrating the total stresses on the arc boundary, we assume both the pressure and viscous forces. Thus, we should distinguish the two cases and examine which case is affected by the varying viscosity values.</P><P>If we could make a prediction, it would be logical to assume that viscosity affects the viscous forces which interest us.</P></DIV></SECTION></ARTICLE><ARTICLE><DIV style="text-align: left"><HEADER><STRONG class="sub_headers">VISCOUS FORCES ON THE TUMOR REGION</STRONG></HEADER></DIV></ARTICLE><ARTICLE><DIV style="text-align: left"><HEADER><STRONG class="sub_headers">Shear Rate and probability of Tumor Surface Attachment</STRONG></HEADER></DIV><SECTION class="sub_sections"><DIV style="text-align: justify"><P>Bacterial locomotion, in fluid flows with low Reynolds numbers, behave in a bizarre manner. To be more specific, the Navier – Stokes equation for an incompressible fluid takes the following form:</P><SPAN class="equation">\[\nabla p - {\bf{f}} = \mu {\nabla ^2}{\bf{u}}\]</SPAN><P>In case there is an imbalance between the driving forces and the dissipation, the fluid reacts to this imbalance by altering the magnitude and direction of u in order to reach a state where there is a balance between the viscous and other forces. The imposed boundary conditions on the system determine the state of the system on the same time step as no boundary “memory” appears. Thus, the fluid depends on each time step on the boundary conditions and that implies the existence of time-reversibility [<A href="#ref13_ccfd">13</A>].</P><P>According to literature [<A href="#ref6_ccfd">6</A>], surface attachment of bacteria is enhanced not in high-shear regions as one could expect but in low-shear regions. More specifically, the factor which determines the degree of bacterial attachment on a surface is the shear-induced trapping which in low-shear regions is enhanced. Thus, for small shear rates (0 – 10 s-1) one could expect sufficient bacterial surface attachment. However, in these low-shear regions chemotaxis may be compromised. Fortunately, in our Quorum Sensing Model, our bacterial communication through the AHL molecule does not affect bacterial locomotion thus chemotaxis in our case is not an issue.</P><P>It is interesting to notice that the shear stress values on the tumor boundaries are within the low-shear region limits and thus, we expect a considerable number of our bacteria to attach to the tumor surface.</P></DIV></SECTION></ARTICLE></DIV></SECTION><DIV id="references"><ARTICLE><DIV style="text-align: left"><HEADER><STRONG class="sub_headers">References</STRONG></HEADER></DIV><SECTION style="text-align:justify" class="sub_sections">
					[<SPAN id="ref1_ccfd">1</SPAN>] Egorov, Viacheslav &amp; Schastlivtsev, Ilya &amp; Prut, E &amp; Baranov, Andrey &amp; Turusov, Robert. (2002). Mechanical properties of the human gastrointestinal tract. Journal of biomechanics. 35. 1417-25. 10.1016/S0021-9290(02)00084-2. 
					[<SPAN id="ref2_ccfd">2</SPAN>] Christensen, Ben &amp; Oberg, Kevin &amp; C Wolchok, Jeffrey. (2015). Tensile Properties of the Rectal and Sigmoid Colon: A Comparative Analysis of Human and Porcine Tissue. SpringerPlus. 4. . 10.1186/s40064-015-0922-x. Hoeg, Slatkin, Burdick and Grundfest [Proc. ICRA200]
					[<SPAN id="ref3_ccfd">3</SPAN>] Lauga, Eric &amp; Powers, Thomas. (2008). The hydrodynamics of swimming microorganisms. Report Progr Phys. 72. . 10.1088/0034-4885/72/9/096601.
					[<SPAN id="ref4_ccfd">4</SPAN>] Ji-Hong Chen, Yuanjie Yu, Zixian Yang, Wen-Zhen Yu (2017, February 20). Intraluminal pressure patterns in the human colon assessed by high-resolution manometry. Nature Scientific Reports 2017
					[<SPAN id="ref5_ccfd">5</SPAN>] D. Heg, H &amp; B. Slatkin, A &amp; W. Burdick, J &amp; Warren, Dr &amp; Grundfest, S. (1999). Biomechanical Modeling of the Small Intestine as Required for the Design and Operation of a Robotic Endoscope. Proceedings - IEEE International Conference on Robotics and Automation. 2. . 10.1109/ROBOT.2000.844825.
					[<SPAN id="ref6_ccfd">6</SPAN>] Rusconi, Roberto &amp; Guasto, Jeffrey &amp; Stocker, Roman. (2014). Bacterial transport suppressed by fluid shear. Nature Physics. 10. 212-217. 10.1038/nphys2883.
					[<SPAN id="ref7_ccfd">7</SPAN>] Yamada, H. (1970). Strength of Biological Material.
					[<SPAN id="ref8_ccfd">8</SPAN>] Happel, John &amp; Brenner, R. (1965). Low Reynolds Number Hydrodynamics: With Special Applications to Particulate Media. Prentice-Hall International Series in the Physical and Chemical Engineering Sciences.
					[<SPAN id="ref9_ccfd">9</SPAN>] T. Chwang, Allen. (1975). Hydromechanics of low-Reynolds-number flow. II: Singularity method for Stokes flows. Journal of Fluid Mechanics. 67. 787 - 815. 10.1017/S0022112075000614.
					[<SPAN id="ref10_ccfd">10</SPAN>] T.  Chwang , Allen &amp; Yao-Tsu  Wu , T. (1974). Hydromechanics of low-Reynolds-number flow. Part 1. Rotation of axisymmetric prolate bodies. Journal of Fluid Mechanics. 63. 607 - 622. 10.1017/S0022112074001819.
					[<SPAN id="ref11_ccfd">11</SPAN>] Cohen, Netta &amp; Boyle, Jordan. (2010). Swimming at low Reynolds number: A beginners guide to undulatory locomotion. Contemporary Physics - CONTEMP PHYS. 51. 103-123. 10.1080/00107510903268381.
					[<SPAN id="ref12_ccfd">12</SPAN>] Taktikos, Johannes. (2013). Modeling the random walk and chemotaxis of bacteria: Aspects of biofilm formation. . 10.14279/depositonce-3477.
					[<SPAN id="ref13_ccfd">13</SPAN>] Ouldridge, Tom. Low Reynolds number hydrodynamics: Stoke’s flow. University College
					[<SPAN id="ref14_ccfd">14</SPAN>] Moore, Keith L. Clinically Oriented Anatomy. Wolters Kluwer. Lippincot Williams &amp; Wilkins</SECTION></ARTICLE></DIV></ARTICLE></DIV></DIV></DIV></VIDEO></DIV></DIV></DIV></DIV></DIV></BODY></HTML>