Team:Hong Kong HKUST/Recombination
HKUST iGEM Team 2017
Team Hong Kong - HKUST
Project
Project Overview
Sensing Module
Time Control Module
Recombination Module
Possible Future Application
Modelling
Single-cell
Model-Sensing
Model-Time Control
Model-Recombination
Population
References
Human Practices
Human Practices (Silver)
Integrated Human Practices
Education & Public Engagement
Medal Requirements
Collaborations
Interlab
About Us
Attributions
Our Team
Logbook & Protocols
Safety
Giant Jamboree Photo Gallery
Recombination Module
Overview / Rationale
The final project in our module which activates after the delay produced by the Time Control module is the Recombination module. The purpose of this module is to create recombinase proteins which will cleave the plasmid at specific sites and separate the origin of replication from the gene of interest. The recombinase we chose to use in our project was Cre recombinase.
Cre recombinase does not destroy the entire plasmid and allows the cell to live after the gene sequence is removed. This could be advantageous in eukaryotic organisms where the death of healthy cells is not favorable and only the unwanted gene portion needs to be removed.
Cre recombinase is simple to use when two loxP sites are flanked around the sequence to be excised without the addition of other guided DNA/mRNA. It has been experimented with in mammalian organisms and extensively in mice. Similar recombinases such as Dre can be further added to splice other inserted gene sequences. The Dre recombinase recognizes its own specific sites so there will not be cross recombination between the two recombinases.
While Cre recombinase itself can only recombine about 55% of the plasmids to be in the recombined state due to the fact that it also catalyzes the reverse reaction back to the original state, our model shows that under when the population size is in equilibrium, the amount of the gene of interest is diluted after generations of cell replication and cell death.
Diagram 1. Cre recombinase targets lox mutant sites which will then recombine gene A, while Dre recombinase targets rox site which will recombine gene B
The function of the Cre-lox & Dre-rox is to knock out the gene of interest from the plasmid. Cre is a site-specific         DNA recombinase derived from the P1 bacteriophage while Dre derives from P1-like transducing phage D6 isolated from         Salmonella oranienburg. (Anastassiadis et al., 2009) (Nagy, 2000) Both of them are tyrosine family of site-specific         recombinases.
Our Design
We designed recombinase expressing genes located just downstream of our time delay module under the regulation of pPhlF.
Diagram 2. Design of recombination system for application
The application of Cre and Dre is one of our main focus in our project. Our design is to insert the lox and rox sites         into the vector plasmid that flanks the origin of replication. When there is an expression of Cre and Dre protein,         Cre and Dre will cause recombination of their corresponding target site. The Cre-lox and Dre-rox systems then induce         deletion of the origin of replication, resulting in two occasions. One is the plasmid without the origin of replication.         The other one is the plasmid with the origin of replication. The plasmid without the origin of replication cannot         carry out DNA replication. In each generation, the population will contain only half of the plasmid with the gene         of interest. After many generations, the amount of plasmid with the gene of interest will be diluted and eventually         none of the cells will contain the plasmid with the gene of interest.
The second occasion, the population will contain the plasmids with the origin of replication only (because of deletion).         However, as the cell will use its resource to replicate the plasmids with the origin of replication. The cell containing         the plasmids with the origin of replication are less competitive than the wild-type. The population containing the         plasmid flanked with origin of replication will be eventually wiped out due to selective pressure. As a result, the         population would revert to its original state before the introduction of the plasmid.
Nevertheless, lox and rox sites are not necessarily flanked around the origin of replication.
Diagram 3. Plasmid containing 3 modules construct: Sensing, Time control and recombination modules, with origin of replication and chloramphenicol antibiotic resistance gene
Maximizing the efficiency of our knock out switch
Our team emphasizes on the safety of construct design. That is why maximizing the efficiency of our knock out switch         is of paramount importance in our project. We have designed several measures to increase the efficiency of our knock         out switch. First is the implementation of two recombination deletion systems: Cre-lox and Dre-rox systems. Due to         the property of non-crossover recombination between Cre-rox or Dre-loxP, if one of our recombination systems fails,         other recombination system can still induce the deletion of the origin of replication, thus termination of the expression         of gene of interest and making our knockout switch doubly safe. Second is the use of mutant loxP sites, lox 66 and         lox 71. After the recombination between lox 71 and lox 66 sites, the characteristic property of these target site         prevents recombination back to the original state, increasing the efficiency of deletion.
Mechanism of Cre/lox system induced deletion
In the case of Cre protein, it recognizes 34 base pair sequences known as loxP sites. There are also mutants of the         recognition site, for example lox 66 and lox 71 which have distinctive activities from loxP site. The lox sequence         consists of asymmetric 8 bp sequences of spacer region (also called linker), which is flanked by 13 bp palindromic         repeats. Because of the asymmetrical properties of linkers, it provides a control of directionality of recombination         by Cre protein. When two lox sites have compatible linkers with identical orientations, Cre-mediated recombination         causes a deletion of the sequence in between the two lox sites.
Mechanism of Dre/rox system induced deletion
Dre protein is a Cre-like site-specific recombinase (SSR) which had diverged sufficiently to recognize its own recombination         site that is distinct from lox site named rox. The base pair of rox site is 32 (lox site has 34 bp). The rox site         is structurally similar to lox site. It also has two palindromic repeats but are 14 bp (lox site has 13 bp repeats)         and are separated by only 4bp linker sequence. Multiple research papers mention that there is no crossover recombination         between Cre-rox or Dre-loxP. (Brian L, 2004)
References
Anastassiadis, K., Fu, J., Patsch, C., Hu, S., Weidlich, S., & Duerschke, K. et al. (2009). Dre recombinase, like Cre, is a highly efficient site-specific recombinase in E. coli, mammalian cells and mice. Disease Models & Mechanisms, 2(9-10), 508-515. http://dx.doi.org/10.1242/dmm.003087
Nagy, A. (2000). Cre recombinase: The universal reagent for genome tailoring. Genesis, 26(2), 99-109. http://dx.doi.org/10.1002/(sici)1526-968x(200002)26:2<99::aid-gene1>3.0.co;2-b
Patent US7915037 - Dre recombinase and recombinase systems employing Dre recombinase. (2017). Google Books. Retrieved 26 October 2017, from https://www.google.com/patents/US7915037
Follow Us Now !!!
SiteMap
© iGEM 2017 Team HKUST. All rights reserved.
made with ♥ @ HKUST
