<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Hong_Kong_HKU_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Hong Kong HKU/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right" style="margin-top: 15px;"><LI class="dropdown"><A class="dropdown-toggle active" href="https://2017.igem.org/Team:Hong_Kong_HKU/Description">Project </A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKU/Description">Motivation</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKU/projectoverview">Project Overview</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKU/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKU/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKU/InterLab">InterLab</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle active" href="https://2017.igem.org/Team:Hong_Kong_HKU/Notebook">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKU/Notebook">Lab book</A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2017.igem.org/Team:Hong_Kong_HKU/HP/Silver">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKU/HP/Silver">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKU/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKU/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKU/HP/Gold_Integrated"> Integration</A></LI><LI><A href="https://2017.igem.org/wiki/index.php?title=Team:Hong_Kong_HKU/safety"> Safety</A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2017.igem.org/Team:Hong_Kong_HKU/Team">Team </A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKU/Attributions">Attributions</A></LI></DIV></DIV><DIV><H3><CENTER>What is Huntington's disease?</CENTER></H3><H3><P>Huntington’s disease (HD) is an inherited neurodegenerative disorder that results in the death of brain cells. Genetic mutations, namely the trinucleotide repeats in the Huntingtin gene (HTT) located on Chromosome 4, lead to the development of HD [1]  .

Though the disease is incurable, early diagnosis can help to better relieve symptoms by allowing treatments to start sooner. In the early stages of HD, only subtle changes in personality, cognitive and physical abilities can be identified.
</P></H3><H3><CENTER>Early Diagnosis of Huntington's disease</CENTER></H3><P>
	Diagnosis of HD (in cases where the parents do not have HD) is usually carried out only after symptoms are identified and the patient approaches the medical professionals. As the early symptoms are generally not severe enough to be recognized as HD symptoms on their own, the treatment is usually delayed.

Although commercially available RT-PCR arrays for Huntington’s disease gene targets do exist, they tend to be time-consuming and require a level of lab expertise[2] (Figure 2)

According to the interviews conducted with the medical workers using various diagnostics tools, their primary concerns were the accuracy of the test and the ease of use. We hope to allow prompt treatment with our non-invasive and highly accessible diagnostic method.

</P></DIV><DIV><H3><CENTER>DNA nanostructures and miRNAs as biomarkers</CENTER></H3><H5><P>Compared to its antibody based counterparts, DNA based diagnostic device are more stable and do not require a cold-chain, the maintenance of proteins at cool
		temperature to avoid degradation[3]. DNA has emerged as a promising material that allows researchers to construct novel designs as its structure could be predicted easily and accurately[4]. Examples of DNA nanostructures
		include nano-tweezers to detect norovirus and a DNA ‘Nano-Claw’ (Figure 4) to detect membrane markers of cancer cells [5,6]
</P></H5><H4>Figure: Nano-Claw</H4><DIV><P>
		DNA Boolean logic gates have been constructed to produce signals in the
		presence of multiple targets, such as OR-gate and AND-gate DNA tetrahedra that generate fluorescence resonance energy transfer (FRET) signal when multiple inputs hybridize with the probe[7].

	</P><H4 style="float:right">Figure: Protein hybridization process.</H4></DIV><DIV><P>
		As for the targets to be detected, different microRNAs (miRNAs) have been identified to be associated with cancers. For example, miR-15b-5p, miR-338-5p, and miR-764 found in plasma are potential biomarkers for detecting hepatocellular carcinoma cancer (HCC), a common type of liver cancer[8]. It has already been reported that it is promising to use these biomarkers - miRNAs to detect cancers[9].

	</P><H4>Figure: Plasma levels of miR-15b-5p, miR-338-5p, and miR-764</H4><DIV><P>For Huntington’s disease, Hsa-miR-34b can be used as a biomarker. This miRNA is stable in the plasma, allowing diagnosis to be carried out just by extracting small amounts of the patient’s blood. The miRNA is also elevated in pre-manifest Huntington’s Disease, which allows HD to be detected at the earliest stage possible, before the surfacing of symptoms[10].
</P><FOOTER><P><B>References:</B></P><OL><LI>Fo, W. (2007). Huntington’s Disease. The Lancet, 369(9557), 218-228</LI><LI>https://www.qiagen.com/us/shop/pcr/primer-sets/rt2-profiler-pcr-arrays/?catno=PAHS-123Z#resources.</LI><LI>Thiviyanathan, Varatharasa, and David G. Gorenstein. &quot;Aptamers and the next generation of diagnostic reagents.&quot; PROTEOMICS-Clinical Applications 6.11-12 (2012): 563-573.</LI><LI>Chen, Y. J., Groves, B., Muscat, R. A., &amp; Seelig, G. (2015). DNA nanotechnology from the test tube to the cell. Nature nanotechnology, 10(9), 748-760.</LI><LI>Nakatsuka, K., Shigeto, H., Kuroda, A., &amp; Funabashi, H. (2015). A split G-quadruplex-based DNA nano-tweezers structure as a signal-transducing molecule for the homogeneous detection of specific nucleic acids. Biosensors and Bioelectronics, 74, 222-226. </LI><LI>You, M., Peng, L., Shao, N., Zhang, L., Qiu, L., Cui, C., &amp; Tan, W. (2014). DNA “nano-claw”: logic-based autonomous cancer targeting and therapy. Journal of the American Chemical Society, 136(4), 1256-1259.</LI><LI>Pei, H., Liang, L., Yao, G., Li, J., Huang, Q., &amp; Fan, C. (2012). Reconfigurable Three‐Dimensional DNA Nanostructures for the Construction of Intracellular Logic Sensors. Angewandte Chemie, 124(36), 9154-9158. </LI><LI>Chen, Y., Chen, J., Liu, Y., Li, S., &amp; Huang, P. (2015). Plasma miR-15b-5p, miR-338-5p, and miR-764 as Biomarkers for Hepatocellular Carcinoma. Medical science monitor: international medical journal of experimental and clinical research, 21, 1864. </LI><LI>Montani, F., &amp; Bianchi, F. (2016). Circulating Cancer Biomarkers: The Macro-revolution of the Micro-RNA. EBioMedicine, 5, 4-6.</LI><LI>Kantcheva, R.B. (2013). Identification and analysis of genetic modifiers of mutant huntingtin toxicity in Saccharomyces cerevisiae. Diss. University of Leicester.</LI></OL></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>