<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Hong_Kong_HKUST_HP_Gold_Integrated skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Hong Kong HKUST/HP/Gold Integrated</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>HKUST iGEM Team 2017</TITLE></P><DIV id="page-wrapper"><HEADER id="header" style="margin-top:30px; color:	rgba(255, 255, 255, 0.9);"><H1 id="logo" style="font-family:  font-family: 'Coming Soon', cursive; font-weight: 100; font-size: 15pt;"><A href="https://2017.igem.org/Team:Hong_Kong_HKUST" style="color:white; text-decoration: none; font-size:16px; ">Team Hong Kong - HKUST</A></H1><NAV id="nav"><UL><LI><A>Project</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Overview">Project Overview</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Sensing">Sensing Module</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/TimeControl">Time Control Module</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Recombination">Recombination Module</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/FutureApplication">Possible Future Application</A></LI></UL><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Model">Modelling</A></LI><LI><A>Single-cell</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Model/Sensing">Model-Sensing</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Model/TimeControl">Model-Time Control</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Model/Recombination">Model-Recombination</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Model/Population">Population</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Model/References">References</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/HP">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/HP/Silver">Human Practices (Silver)</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Engagement">Education &amp; Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Medal_Requirements">Medal Requirements</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/InterLab">Interlab</A></LI><LI><A>About Us</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Team">Our Team</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Logbook_and_Protocol">Logbook &amp; Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Giant_Jamboree_Photo_Gallery">Giant Jamboree Photo Gallery</A></LI></NAV></HEADER><DIV id="main" style="padding-left: 8%; padding-right:8%; "><HEADER class="major"><H2>Integrated Human Practices</H2></HEADER></DIV><DIV id="main" style="padding-left: 4px; padding-right:4px; "><SECTION class="steps steps-area"><DIV><UL class="timeline"><LI class="timeline-box wow fadeInLeft"><P class="timeline-title">MOTIVATION</P><P>
                            As part of our project brain-storming process, we had been looking through some of the previous projects for inspiration. However, when we start looking into how teams tackle the problem of their project safety, we realized that most teams uses kill switch as their genetic containment strategies. However, there are no practical implementation and standardization of the strategy.

We wanted to create a universal safety switch that teams can easily implement it into their project in the future. We were inspired by the application of recombinases in eukaryotes where only a specific gene inside the cell will be knocked out without harming the cell itself. As a proof-of-concept, we therefore uses the Cre-lox system to illustrate how recombinase can excise the gene of interest in the DH10b strain of E.coli bacteria that are easy to grow and safe to use in the laboratory. 


                        </P><P>
                            Having received feedbacks from people joining our organized activities in Hong Kong Science Park, where the majority is parents,
                            it surprisingly shows that our samples view GMOs to be more safe if the genetically modified
                            gene within an organism has been knocked out before the consumption. We, therefore, directed
                            our focus on Genetically Modified Organisms (GMOs) with the addition of time delay controlling
                            the expression of gene of interest for safety reason.
                        </P><SPAN>1</SPAN></LI><LI class="timeline-box wow fadeInRight"><P class="timeline-title">TOWARDS OUR APPLICATION</P><P>
                            Feedbacks from the participants in our workshops surprisingly shows that the public view GMOs, specially food to be more
                            safe if the genetically modified gene within an organism has been knocked out before consumption.
                            We, therefore, directed our focus on Genetically Modified Organisms (GMOs) with the addition
                            of time delay controlling the expression of gene of interest to satisfy the public’s concern.
                            Time control mechanism can allow the genetically modified crops to grow under extreme conditions
                            to increase crop yield while having this foreign gene removed before reaching the consumers so
                            that these crops will be considered safer in the public’s point of view.
                        </P><P>
                            The introduction of GMOs, however, varies between countries in terms of regulation and stringency. In order to explore the
                            situation of GMOs and their products in Hong Kong, we had contacted and interviewed with Dr.
                            Terence Lau, a biosafety expert from the Hong Kong government. Dr. Lau shared with us about the
                            current regulation and control of GMOs. To our surprise, there is yet no measure in preventing
                            or mitigating the risks of GMOs being accidentally released in Hong Kong. The government only
                            emphasizes on the control of genetically modified organism in some products such as vaccine and
                            GM crops where the Centre for food safety had provided guidelines on voluntary labelling of Genetically
                            Modified (GM) Food. In addition, industries will take risk to benefit ratios into account before
                            considering GMOs as their profit-based business. Throughout this interview, the issue raised
                            has inspired us to pursue deeper into the application of Genetic Containment Strategy.
                        </P><SPAN>2</SPAN></LI><LI class="timeline-box  wow fadeInLeft"><P class="timeline-title">WHAT IS GENETIC CONTAINMENT STRATEGY? HOW IS IT DIFFERENT FROM BIO-CONTAINMENT?</P><P>
                            Genetic (Biological) Containment Strategy is a measure to safeguard recombinant DNA in microorganism such that it can only
                            be used in lab experiment but not outside. If accidental release happens, growth of modified
                            microbes and rDNA replication should be immediately inhibited.
                        </P><P>
                            On the other hand, biocontainment is a prevention of risk of genetically engineered microorganism (GEM) release by using
                            physical containment.
                        </P><P> Hong Kong laboratories mainly relies on biocontainment. But due to the demand for scaling up GEM,
                            biocontainment may not be sufficient to contain large scale of GEM. Thus, we propose the use
                            of genetic containment strategy as genetic safeguards can potentially prevent the escape of microbes
                            from proliferating. <SUP>1</SUP></P><SPAN>3</SPAN></LI><LI class="timeline-box wow fadeInRight"><P class="timeline-title">THE SENSOR'S POSITIVE FEEDBACK LOOP</P><P>
                            Dr. Lau not only shared with us some knowledge regarding biosafety. He also raised questions concerning how we can achieve
                            the purpose of knocking out in large population of cells.
                        </P><P>
                            We decided on using feedback loop to amplify signals such that sufficient receiver cells can be knocked out at time bound.
                            The propagation of AHL signals can be simulated by modeling the rate of diffusion from the senders
                            to the receivers and the rate of AHL being amplified again to affect peripheral cells. The positive feedback loop we found involves in the use of luxR cassette in E.coli. We characterized and found that there's basal expression level under no induction. We, therefore, addressed the issues of promoter leakiness and basal level expression through the use of antisense RNA based on our modelling analysis.
                            <A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Model/Sensing"><U> Click to see our Modelling Page.</U></A></P><SPAN>4</SPAN></LI><LI class="timeline-box wow fadeInLeft"><P class="timeline-title">Feedbacks from conference (2017 iGEM Pan-Pearl conference organized by the CUHK 2018 team)</P><P>
                            Q1: Is there a chance where random homologous recombination occurs between your recombinant plasmid and bacterial genome
                            inside the cell?
                        </P><P>
                            Q2: Is there any enzymes that can digest your engineered plasmids instead of using these recombinases?
                        </P><P>
                            Q3: How can you eliminate the plasmid after it’s recombined?
                        </P><P>
                            Q4: How would you use your construct in particular scenarios. For example, how will your construct help if you accidentally
                            spill the GM bacteria into environment?
                        </P><SPAN>5</SPAN></LI><LI class="timeline-box wow fadeInRight"><P class="timeline-title">ADDRESSING ISSUES</P><P>
                            In addition to these feedback questions, we looked more into details of our constructs where improvement can further be made
                            to reduce the safety issues. We conducted an interview with Prof. Matthew Bennett from Rice University
                            in Houston, who is an expert in organism’s genetic system and circuits.
                        </P><P>
                            Q1: It was found that even though <I>luxR</I> sequences do not appear in <I>E. coli</I> in nature,
                            there is a homologous gene found in <I>E. coli</I> genome called <I>sdiA</I>. The <I>luxR</I>                            gene, a family of <I>sdiA</I> gene, shares homologous sequence such that random recombination
                            between the plasmid and genome may occur many times, although not frequent. We address this problem
                            by finding a chassis that has the <I>sdiA</I> gene being knocked out. See <A href="https://cgsc2.biology.yale.edu/Strain.php?ID=107863">strain JW1901-5 <SUP>2</SUP></A></P><P>
                            Q2: Prof. Matthew Bennett commented that endonucleases may be more efficient in targeting multiple restriction sites and
                            we have explored some more nucleases that could do the job of DNA degradation. One efficient
                            method to target DNA degradation is the use of CRISPR-Cas3 device, where CRISPR nuclease specifically
                            degrades the targeted DNA, leaving the non-targeted DNA unaffected <SUP>3</SUP>                            There is also a nuclease that can be transiently transcribed in vivo such as Yeast homothallic
                            switching endonuclease (HO endo) which recognizes HO recognition sites and cleaves DNA to become
                            linearized <SUP>4</SUP>.
                        </P><P>
                            Q3: As our project describes a method that uses recombinase to separate between the gene of interest and the plasmid containing
                            the origin of replication and marker gene, we use modelling to simulate a scenario to measure
                            the dilution of plasmids and how long the recombined plasmid can be degraded over time.
                        </P><P>
                            Nevertheless, after the interview and through further research, we conjectured that a more effective way to digest recombinant
                            plasmid is through the use of endonuclease. Apart from considering endonucleases such as a DNAi
                            device called CRISPR-Cas3 as a choice for further improvement in our project, our team has continued
                            to find potential application that recombinase can be compatible with (See Application page).
                            We also designed plasmid simulation where CRISPR-Cas3 replaces Cre-recombinase<SUP>8</SUP>,
                            but we were not able to incorporate into our project due to the time limitation.
                        </P><P>The system works by expressing Cas3 and CasABCDE elements that will complex together and are guided
                            by CRISPR RNAs (crRNAs) with its spacer which will recognize PAM sequence correspnding with their
                            complementary protospacer. The DNAi device will then cleave double stranded DNA, which will be
                            degraded by exonuclease activity. The figure below shows the construct design (left and bottom)
                            and how Cas3, CasABCDE and crRNAs target their sites (right)</P><P>We investigated the pros and cons of using 3 different knockout methods such as Cas9, Cas3 and recombinase,
                            by listing out in the following tables</P><DIV style="overflow-x:auto;"><TABLE class="tg" style="undefined;table-layout: fixed; width: 100%"><TBODY><TR><TH class="tg-wh4d" colspan="2">Cas9</TH><TH class="tg-wh4d">Cas3-CasABCDE</TH><TH class="tg-wh4d">Cre recombinase</TH></TR><TR><TD class="tg-2jlv">Characteristic</TD><TD class="tg-gvwu">Genome editing at specific interval</TD><TD class="tg-i6eq">Remove the gene beyond the point of repair like DNA shredder</TD><TD class="tg-i6eq">Genome editing at specific interval; Knockout genes in tissue-specific and time-controlled
                                        fashion (1)</TD></TR><TR><TD class="tg-9d65">Efficiency</TD><TD class="tg-pxng">Depends on chassis:Up to 45% in iPSC cells (2)</TD><TD class="tg-3we0">Dependent on number of plasmids and spacers (3)</TD><TD class="tg-3we0">Variable in mouse strains (4)</TD></TR><TR><TD class="tg-xy75">Advantages</TD><TD class="tg-i6eq">- Target different genome/plasmid with different crRNA design- Works in all organisms</TD><TD class="tg-i6eq">- Host cells kept intact- Can trigger bacterial self-destruction- Future application
                                        on antibiotic resistance such as MRSA</TD><TD class="tg-i6eq">-Host cells kept intact- No cross recombination with sites that are not loxP (5)-
                                        Good temporal or tissue-specific control of genetic modification- Knockout efficiency
                                        is independent of target DNA length (6)- Off-target not shown in this type of
                                        chicken (7)</TD></TR><TR><TD class="tg-o4ll">Limitations</TD><TD class="tg-3we0">- Different Cas9 homologs require different PAM sequences- Mosaicism in mice (8)-
                                        Large fragment deletion is achieved at low efficiency (less than 10%) (9)</TD><TD class="tg-3we0">- Not a site-specific genome editing- defective CRISPR- Organism may tolerate
                                        PAM sequences (PAM sequences should be properly designed)- More targetable spacer:
                                        Use more flexible PAM sequence- Reduce off-target effects: Use more stringent
                                        PAM sequence (10)- Off-target effects- Not yet experimented in mammal cells
                                        due to large size- Can only target bacterial cells because cas3 is too big to
                                        enter mammalian cells (11)</TD><TD class="tg-3we0" rowspan="2">- Mutations in target sequence- Knockout efficiency is dependent of target DNA length
                                        (12)- Recombine sites that are a family of loxP e.g. LoxP, mLoxP- Pseudo-loxP
                                        sites may present in mammalian genome, leading to aberrant activity (13). Must be
                                        careful on selecting strain</TD></TR><TR><TD class="tg-xy75">Common limitations among CRISPR</TD><TD class="tg-i6eq" colspan="2">- Mutated target sequence (14)- defective CRISPR- Organism may tolerate PAM sequences
                                        (PAM sequences should be properly designed)- More targetable spacer: Use more
                                        flexible PAM sequence- Reduce off-target effects: Use more stringent PAM sequence
                                        (11)- Off-target effects</TD></TR></TBODY></TABLE></DIV><P style="word-break: break-all; font-size:10px">
                            References:
                            (1) Loonstra, A., Vooijs, M., Beverloo, H. B., Allak, B. A., van Drunen, E., Kanaar, R.,
                            Berns, A., &amp; Jonkers, J. (2001). Growth inhibition and DNA damage induced by Cre recombinase
                            in mammalian cells. Proceedings of the National Academy of Sciences, 98(16), 9209–9214. doi:
                            10.1073/pnas.161269798 (2) Liang, X., Potter, J., Kumar, S., Ravinder, N., &amp; Chesnut, J. D. (2017).
                            Enhanced CRISPR/Cas9-mediated precise genome editing by improved design and delivery of gRNA,
                            Cas9 nuclease, and donor DNA. Journal of Biotechnology, 241, 136-146. doi: https://doi.org/10.1016/j.jbiotec.2016.11.011                            (3) Caliando, B. J., &amp; Voigt, C. A. (2015). Targeted DNA degradation using a CRISPR device stably
                            carried in the host genome. Nature Communications, 6, 6989. doi:10.1038/ncomms7989 (4) Garcia-Arocena,
                            D. (2013). Cre-lox Myths Busted. The Jackson Laboratory. Retrieved from https://www.jax.org/news-and-insights/jax-blog/2013/september/cre-lox-myths-busted                            (5) Anastassiadis, K., Fu, J., Patsch, C., Hu, S., Weidlich, S., Duerschke, K., Buchholz, F.,
                            Edenhofer, F., &amp; Stewart, A. F. (2009). Dre recombinase, like Cre, is a highly efficient site-specific
                            recombinase in E. coli, mammalian cells and mice. Disease Models &amp; Mechanisms, 2(9-10), 508-15.
                            doi: 10.1242/dmm.003087 (6) Ullrich, S., &amp; Schüler, D. (2010). Cre-lox-Based Method for Generation
                            of Large Deletions within the Genomic Magnetosome Island of Magnetospirillum gryphiswaldense.
                            Applied and Environmental Microbiology, 76(8), 2439-2444. doi: 10.1128/AEM.02805-09 (7) Leighton,
                            P. A., Pedersen, D., Ching, K., Collarini, E. J., Izquierdo, S., Jacob, R., &amp; van de Lavoir,
                            M.-C. (2016). Generation of chickens expressing Cre recombinase. Transgenic Research, 25(5),
                            609–616. doi: 10.1007/s11248-016-9952-6 (8) Yeadon, J. (2014). Pros and cons of ZNFs, TALENs,
                            and CRISPR/Cas. Pros and cons of ZNFs, TALENs, and CRISPR/Cas. The Jackson Laboratory. Retrieved
                            from https://www.jax.org/news-and-insights/jax-blog/2014/march/pros-and-cons-of-znfs-talens-and-crispr-cas                            (9) Song, Y., Lai, L., &amp; Li, Z. (2017). Large-scale genomic deletions mediated by CRISPR/Cas9
                            system. Oncotarget, 8(4), 5647. doi: 10.18632/oncotarget.14543
                             (10) Larson, M. H., Gilbert, L. A., Wang, X., Lim, W. A., Weissman, J. S., &amp; Qi, L. S. (2013).
                            CRISPR interference (CRISPRi) for sequence-specific control of gene expression. Nature Protocols,
                            8(11), 2180-2196. doi: 10.1038/nprot.2013.132
                             (11) Buhr, S. (2016). Move over Cas9, CRISPR-Cas3 might hold the key to solving the antibiotics
                            crisis. TechCrunch. Retrieved from https://techcrunch.com/2016/12/21/move-over-cas9-crispr-cas3-might-hold-the-key-to-solving-the-antibiotics-crisis/
                             (12) Coppoolse, E. R., de Vroomen, M. J., van Gennip, F., Hersmus, B. J. M., &amp; van Haaren,
                            M. J. J. (2005). Size Does Matter: Cre-mediated Somatic Deletion Efficiency Depends on the Distance
                            Between the Target lox-Sites. Plant Molecular Biology, 58(5), 687-698. doi:10.1007/s11103-005-7705-7
               (13) Loonstra, A., Vooijs, M., Beverloo, H. B., Allak, B. A., van Drunen, E., Kanaar, R.,
                            Berns, A., &amp; Jonkers, J. (2001). Growth inhibition and DNA damage induced by Cre recombinase
                            in mammalian cells. Proceedings of the National Academy of Sciences, 98(16), 9209–9214. doi:
                            10.1073/pnas.161269798
                             (14) Bikard, D., Euler, C., Jiang, W., Nussenzweig, P. M., Goldberg, G. W., Duportet, X.,
                            Fischetti, V.A., &amp; Marraffini, L. A. (2014). Development of sequence-specific antimicrobials
                            based on programmable CRISPR-Cas nucleases. Nature Biotechnology, 32(11), 1146-1150. doi: 10.1038/nbt.3043
                             (15) Larson, M. H., Gilbert, L. A., Wang, X., Lim, W. A., Weissman, J. S., &amp; Qi, L. S. (2013).
                            CRISPR interference (CRISPRi) for sequence-specific control of gene expression. Nature Protocols,
                            8(11), 2180-2196. doi: 10.1038/nprot.2013.132

                        </P><P> Q4: The desirable design for eliminating bacterial spill is to immediately knockout the construct
                            containing GM gene without the delay of recombinase or endonuclease expression. Thus, one question
                            raised would be ‘When and in which scenario should time delay be used?’ if time delay is not
                            used in the accidental release of GM bacteria.</P><P>Time delay can be useful in laboratory experiments where GM bacteria should not last for so long
                            time. This does not require modification of bacteria having genetic code dependent on some nutrients
                            to survive nor require the activation of toxin killing GM organism in the culture. Thus, safer
                            extraction of proteins (both secreted and non-secreted) can be achieved with no contamination
                            of toxic compounds. Additionally, we would not need to worry about any changes in the organism’s
                            genomic content.. </P><SPAN>6</SPAN></LI><LI class="timeline-box wow fadeInRight"><SPAN>♡</SPAN></LI><LI class="timeline-box wow fadeInRight"><SPAN>❀</SPAN></LI><LI class="timeline-box wow fadeInLeft"><P class="timeline-title">FUTURE SUGGESTIONS</P><P><U>Suggest for an alternative way of resolving the leakiness of promoters,pluxR</U></P><P> We suggest that expressing our construct in gram-positive bacteria may avoid leaky expression
                                due to an auto-activation of pluxR. Gram-positive bacterium has SAM molecules that are conserved
                                in both gram-positive and gram-negative species. However, gram-positive bacteria do not use
                                the same communication pathway. It employs two component systems called oligopeptide auto-inducers,
                                instead of AHL molecules (<I>luxI</I> from <I>Vibrio fischeri</I> produces AHL with a distinct
                                side chain that can’t be found in Gram positive bacteria)<SUP>6<SUP>. 
                                </SUP></SUP></P><P>Thus, we may try to express <I>pluxR</I> promoter along with <I>luxI</I> synthases within the gram-positive bacteria such as Bacillus subtilis and test whether there is a reduction in promoter leakiness in such chassis.
                                </P><P><U>Suggestion for more specific recombination</U></P><P>
                                   While Cre-loxP system allows
                                    the study of spatial and temporal gene expression, CRISPR/Cas9 can target specific site
                                    to mediate gene editing. According paper published in 2017,  CRISPR/Cas9 was shown to be able to work with loxP for the first time. This has an application on site-specific genome editing in human cells where using loxP mutant that contains sgRNA could allow a selective gene excision. <SUP>7</SUP></P><P>
                                   Nevertheless, future improvements need to be made. Fayu Yang and his research team suggested that loxP66 and loxP71 mutants (which we also employed in this iGEM project) may be more suitable for CRISPR/Cas9-loxP design because they contain PAM sequences which sgRNA can recognize. 
                                </P><P>Thus, this system may enhance effectiveness in knockout performance inside bacterial cells if two or more external genes are to be inserted and knocked out at the same time. Knocking out long sequences of target gene such as sensing and time-delay module may be done more easily especially when Cre recombinase fails to recognize lox site. Due to time limits, we could not explore more in details so we consider this method as our future suggestions and possible improvement for our project</P><SPAN>7</SPAN></LI><LI class="timeline-box wow fadeInLeft"><P class="timeline-title">REFERENCES</P></LI><LI style="word-break: break-all; font-size:12px"><A href="http://dx.doi.org/10.1038/nchembio.2004">Karmella A Haynes (2016) Synthetic biology: Building genetic containment. Nature Chemical Biology, 12, 55–56. doi:10.1038/nchembio.2004</A></LI><LI style="word-break: break-all; font-size:12px"><A href="https://cgsc2.biology.yale.edu/Strain.php?ID=107863">E. Coli Genetic Resources at Yale CGSC, The Coli Genetic Stock Center  (n.d.) Retrieved from: https://cgsc2.biology.yale.edu/Strain.php?ID=107863</A></LI><LI style="word-break: break-all; font-size:12px"><A href="https://www.ncbi.nlm.nih.gov/pubmed/25988366">Calianda BJ (2015) Targeted DNA degradation using a CRISPR device stably carried in the host genome. Nature Communications 6, 6989. doi :10.1038/ncomms7989 </A></LI><LI style="word-break: break-all; font-size:12px"><A href="https://www.ncbi.nlm.nih.gov/pubmed/10075888">Chien-Ping Liang (1999) Targeted Linearization of  DNA in vivo. Elsevier, 17(2) 95-103, Retrieved from: https://doi.org/10.1006/meth.1998.0721</A></LI><LI style="word-break: break-all; font-size:12px"><A href="https://www.ncbi.nlm.nih.gov/pubmed/10075888">Chien-Ping Liang (1999) Targeted Linearization of  DNA in vivo. Elsevier, 17(2) 95-103, Retrieved from: https://doi.org/10.1006/meth.1998.0721</A></LI><LI style="word-break: break-all; font-size:12px"><A href="https://www.ncbi.nlm.nih.gov/pubmed/16497924">Andrew Camilli (2009) Bacterial Small-Molecule Signaling Pathways. Science, 311(5764), 1113-1116. Doi: 10.1126/science.1121357</A></LI><LI style="word-break: break-all; font-size:12px"><A href="https://www.ncbi.nlm.nih.gov/pubmed/16672621">Brooke A. McDaniel  (2006) Identification of a Mutation in the Bacillus subtilis S-Adenosylmethionine Synthetase Gene That Results in Derepression of S-Box Gene Expression. J. Bacteriol, 188(10), 3674-3681, Doi:  10.1128/JB.188.10.3674-3681.2006</A></LI><LI style="word-break: break-all; font-size:12px"><A href="http://www.sciencedirect.com/science/article/pii/S2162253117301695">Fayu Yang (2017) CRISPR/Cas9-loxP-Mediated Gene Editing as a Novel Site-Specific Genetic Manipulation Tool. Mol Ther Nucleic Acids 7, 378-386. Doi: 10.1016/j.omtn.2017.04.018</A></LI><LI style="word-break: break-all; font-size:12px"><A href="https://www.nature.com/articles/ncomms7989">Brian J. Caliando &amp; Christopher A. Voigt (2015) Targeted DNA degradation using a CRISPR device stably carried in the host genome. Nature Communications 6. Doi:10.1038/ncomms7989</A></LI><SPAN>Ref.</SPAN></UL></DIV></SECTION></DIV></DIV></DIV><DIV><FOOTER style="opacity:0.7" id="footer"><UL class="icons"><LI><DIV style="display:flex; flex-direction:row; justify-content:center"><A href="https://www.facebook.com/hkustigem2017domilox/" style="font-size:15px; color:blue"> Follow Us Now !!!</A></DIV></LI><LI><DIV style="display:flex; flex-direction:row; justify-content:center"><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/SiteMap" style="font-size:15px; color:red">SiteMap</A></DIV></LI></UL><UL class="copyright"><LI><B>© iGEM 2017 Team HKUST. All rights reserved.</B></LI><LI>made with <SPAN style="color: red">♥</SPAN> @ HKUST</LI></UL></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>