<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Hong_Kong_HKUST_Sensing skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Hong Kong HKUST/Sensing</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>HKUST iGEM Team 2017</TITLE></P><DIV id="page-wrapper"><HEADER id="header" style="margin-top:30px; color:	rgba(255, 255, 255, 0.9);"><H1 id="logo" style="font-family:  font-family: 'Coming Soon', cursive; font-weight: 100; font-size: 15pt;"><A href="https://2017.igem.org/Team:Hong_Kong_HKUST" style="color:white; text-decoration: none; font-size:16px; ">Team Hong Kong - HKUST</A></H1><NAV id="nav"><UL><LI><A>Project</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Overview">Project Overview</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Sensing">Sensing Module</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/TimeControl">Time Control Module</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Recombination">Recombination Module</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/FutureApplication">Possible Future Application</A></LI></UL><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Model">Modelling</A></LI><LI><A>Single-cell</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Model/Sensing">Model-Sensing</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Model/TimeControl">Model-Time Control</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Model/Recombination">Model-Recombination</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Model/Population">Population</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Model/References">References</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/HP">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/HP/Silver">Human Practices (Silver)</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Engagement">Education &amp; Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Medal_Requirements">Medal Requirements</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/InterLab">Interlab</A></LI><LI><A>About Us</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Team">Our Team</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Logbook_and_Protocol">Logbook &amp; Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Giant_Jamboree_Photo_Gallery">Giant Jamboree Photo Gallery</A></LI></NAV></HEADER><DIV id="main" style="padding-left: 8%; padding-right:8%"><DIV><HEADER class="major"><H2>Sensing Module</H2></HEADER><SECTION id="content"><CENTER><CENTER><H3>Overview / Rationale</H3></CENTER></CENTER><P>In order to control the expression in a population of cells, it is essential that the signal can propagate from one
        cell to another. It is therefore, important to consider the diffusion of signals. Signals should be able to pass
        through cell membrane and cell wall, from cytoplasm to the outside environment, and vice versa. Adding a positive
        feedback loop can enhance the transmission of the signal through the population
      </P><P>To ensure the knockout process occurs in every single cell, we select the diffusible signals that a bacterium can produce
        quickly. This inducer is the 3O6CHSL, or AHL molecule.</P><CENTER><CENTER><H3>Sensing Module Construct</H3></CENTER></CENTER><CENTER><A class="image fit" style="color:white; width:100%; max-width:1000px;">Diagram 1. Sensing Module Construct</A></CENTER><CENTER><H3>Positive feedback loop mechanism</H3></CENTER><CENTER><A class="image fit" style="color:white; width:100%; max-width:1000px;">Diagram 2. Positive feedback loop mechanism</A></CENTER><CENTER><H3>Experimental Construct</H3></CENTER><CENTER><A class="image fit" style="color:white; width:100%; max-width:1000px;">Diagram 3. Experimental Construct</A></CENTER><CENTER><H3>Our Design</H3></CENTER><P>The function of the sensing module is to detect the deliberately released stimulus and initiate the process of “knockout”.
        Considering that the released stimulus can become diluted in an environment, a positive feedback loop is included
        to amplify the signal. Thus, 3OC6HSL, which is a member of acyl-homoserine lactone (AHL) family, was chosen as the
        inducer. 3OC6HSL, originally from <I>V. fischeri</I>, is a lipid molecule that can diffuse through bacterial cell
        membrane, facilitating cell-to-cell communication.</P><P>The sensing module begins with the TetR repressible promoter or <I>pTetR</I> (BBa_R0040), which can be treated as constitutive
        promoter of LuxR protein (BBa_C0062) under no repressor TetR. This segment of the biosensor works to produce an abundance
        of LuxR. Once the 3OC6HSL is added to the cell environment, LuxR forms a complex with 3OC6HSL and then activates
        downstream promoter, <I>pLuxR</I> (BBa_R0062).</P><P>After the activation of <I>pLuxR</I> promoter, LuxI protein (BBa_C0061) then expresses. LuxI is an autoinducer synthetase
        which catalyzes 3OC6HSL from S-adenosyl-L-methionine (SAM) in cell. The entire part from pTetR to LuxI generates
        a positive feedback loop, because it increases the concentration of 3OC6HSL/LuxR complex and hence induces <I>pLuxR</I>        more strongly. The 3OC6HSL molecule can also diffuse back to the extracellular environment and affect the nearby
        cells.
      </P><P>With its positive feedback loop, the sensing module is able to convert from signal receiver into signal emitter whenever
        it receives 3OC6HSL molecule. Thus, it is expected to increase the efficiency of activation in a targeted environment.
        </P><H3>Potential Issue</H3><P>The issue that previous iGEM team encountered (See experience in BBa_F2620) is the leakiness of <I>pLuxR</I> promoter
            (BBa_R0062). It is a situation when there is little expression of gene even without initial activation of AHL.
            This time we improved by adding sequences of antisense RNA Binding regions and antisense RNA in which their interaction
            can inhibit the translation of excess mRNA. </P><P>First, the antisense RNA Binding Region (ABR) is placed right before our targeted Ribosomal Binding Site (RBS),
            which is upstream to the LuxI (BBa_C0061) and GFP (BBa_E0040). Second, the antisense RNA is placed downstream
            the positive feedback loop and its reporter together with a medium strength promoter, <I>pLuxL</I> (BBa_R0063).

            </P><P>The antisense RNA we use has two important characteristics that will help reduce leakiness. Firstly, it comprises
              of sequence that is complementary to the sequence of mRNA of ABR. The affiliation of antisense RNA to the complementary
              ABR prevents ribosome from binding to the mRNA of the targeted RBS. Secondly, it has a Hfq binding site, which
              is suspected to recruit RNase to degrade the targeted RNA chain. (Wagner, 2009)(Hoynes-O’Connor &amp; Moon, 2016).
              In consequence, translation of LuxI:GFP mRNA will be reduced, and so will the leakiness of <I>pLuxR</I>. </P><P> To see the theoretically expected behavior of the construct with and without asRNA, please refer to our <A style="color:white; word-wrap: break-word;" href="https://2017.igem.org/Team:Hong_Kong_HKUST/Model/Sensing"> model </A></P><CENTER><H3>Experiments &amp; Results</H3></CENTER><CENTER><H4><SPAN style="text-decoration: underline;">Introduction &amp; Background</SPAN></H4></CENTER><P>To investigate whether our antisense RNAs can reduce basal level expression, we ligated GFP (BBa_E0240) in each
              construct, and measured the fluorescence/OD600 unit produced: pSB1C3-BBa_T9002 (Annotated as w/o PFB): Plasmid
              without positive feedback loop</P><CENTER><A class="image fit" style="color:white; width:100%; max-width:1000px;">Diagram 4. pSB1C3-BBa_T9002</A></CENTER><P>pSB1C3-BBa_F2620-C0261-E0240 (Annotated as w/ PFB): Plasmid with positive feedback loop due to insertion of LuxI
              gene (BBa_C0261)</P><CENTER><A class="image fit" style="color:white; width:100%; max-width:1000px;">Diagram 5. pSB1C3-BBa_F2620-C0261-E0240</A></CENTER><P>pSB1C3-BBa_K2240000 (Annotated as as PFB + asRNA1): Plasmid resulted from insertion of antisense type I and antisense
              binding region complementary to antisense RNA type I into the positive feedback loop system (pSB1C3-BBa_F2620-ABR1-C0261-ABR1-E0240-R0063-Anti1).
              This part has GFP expresses in the construct.</P><P>pSB1C3-BBa_K2240003 (Annotated as as PFB + asRNA2): Construct with Anti II; Plasmid resulted from insertion of
              antisense type II and antisense binding region complementary to antisense RNA type II into the positive feedback
              loop system (pSB1C3-BBa_F2620-ABR2-C0261-ABR2-E0240-R0063-Anti2). This part has GFP expresses in the construct.</P><CENTER><A class="image fit" style="width:100%; max-width:1000px;color:white;">
             Diagram 6. pSB1C3-BBa_K40000 or pSB1C3-BBa_K40003</A></CENTER><CENTER><H4><SPAN style="text-decoration: underline;">Antisense RNA reduces basal expression level</SPAN></H4></CENTER><P>We first characterized the pSB1C3-BBa_T9002 (w/ PFB) to find suitable time for induction, which we found that
              fluorescence/OD600 can reach over than 600,000 GFP/OD600 after more than 3 hours. Thus, we used control variables
              of 3 hours and [AHL] = 1.00E-05 M as an optimum level of induction.</P><CENTER><A class="image fit" style="color:white; width:100%; max-width:600px;"> Diagram 7. Characterization of pSB1C3-BBa_T9002 </A></CENTER><CENTER><H4><SPAN style="text-decoration: underline;">Results</SPAN></H4></CENTER><TABLE style="font-size:16px;"><THEAD><TR><TH scope="col">AHL[M]</TH><TH scope="col">All constructs comparison</TH><TH scope="col">Antisense RNA comparison</TH></TR></THEAD><TBODY><TR><TD data-label="AHL[M]"> 0 M
                    </TD><TD data-label="All constructs comparison"><CENTER><DIV class="center" style="width:auto; margin-left:auto; margin-right:auto;">Fig. 1 Error bar presents SD from 6 biological replicates. </DIV></CENTER></TD><TD data-label="Antisense RNA comparison"><CENTER><DIV class="center" style="width:auto; margin-left:auto; margin-right:auto;">Fig. 2 Error bar presents SD from 6 biological replicates.</DIV></CENTER></TD></TR><TR><TD data-label="AHL[M]"> 1.00E-05 M
                    </TD><TD data-label="All constructs comparison"><CENTER><DIV class="center" style="width:auto; margin-left:auto; margin-right:auto;">Fig. 3 Error bar presents SD from 6 biological replicates. </DIV></CENTER></TD><TD data-label="Antisense RNA comparison"><CENTER><DIV class="center" style="width:auto; margin-left:auto; margin-right:auto;">Fig. 4 Error bar presents SD from 6 biological replicates.</DIV></CENTER></TD></TR></TBODY></TABLE><CENTER><H4><SPAN style="text-decoration: underline;">Discussions</SPAN></H4></CENTER><CENTER><H4>Antisense RNAs can reduce basal expression level</H4></CENTER><P>The efficiency of basal level reduction for antisense RNA type I and II are compared prior to AHL induction ([AHL]=0M)
              and after the induction ([AHL] = 1.00E-05 M) Unpaired t-test analysis suggests that the differences between
              w/PFB and PFB+ asRNA2 and the difference between w/PFB and PFB+asRNA2 could be considered as highly statistically
              significant.
            </P><CENTER><H4>Antisense RNA type II reduces basal level more significantly</H4></CENTER><P>To visualize the difference between antisense type I and type II better, Fig. 2 and 4 were illustrated. Using
              unpaired t-test calculation, it suggests that there are significant differences between antisense RNA type
              I and type II both before and after induction with 1.00E-05 M AHL, where Fig. 2 and 4. imply that construct
              with antisense RNA type II can reduce basal expression level more significantly. This may be due to higher
              GC content of its complementary sites (55% for asRNA type II comparing to 47.4% for asRNA type I) according
              to our first hypothesis.
            </P><CENTER><H4>Positive Feedback Loop works after AHL induction</H4></CENTER><P>There are also notable statistical differences when comparing the changes before and after induction using two-tailed
              paired t-test analysis for both PFB+asRNA1 (p
              &lt;0.001) and PFB+asRNA2 (p&lt;0.01). The average of fluorescence/OD600 for PFB+asRNA2 is 4,199.18 while the average after its
                induction is 6,121.58, showing an increase for around 2,000 GFP/OD600 (45.8% growth) after 3 hours. Whereas,
                PFB+asRNA1 also experiences an increase by 4,765 GFP/OD600 (65.7% growth) under the same controlled time (Fig. 5),
                suggesting that antisense RNA regulates translation inhibition more tightly. This supports our second hypothesis
                that there should be an increase in expression after induction by AHL.</P><TABLE style="font-size:16px;"><THEAD><TR><TH scope="col">Antisense type I (PFB+asRNA1)</TH><TH scope="col">Antisense type II (PFB+asRNA2)</TH></TR></THEAD><TBODY><TR><TD data-label="Antisense type I (PFB+asRNA1)"><CENTER><DIV class="center" style="width:auto; margin-left:auto; margin-right:auto;">Fig. 5 Error bar presents SD from 6 biological replicates.</DIV></CENTER></TD><TD data-label="Antisense type II (PFB+asRNA2)"><CENTER><DIV class="center" style="width:auto; margin-left:auto; margin-right:auto;">Fig. 6 Error bar presents SD from 6 biological replicates.</DIV></CENTER></TD></TR></TBODY></TABLE><CENTER><H4>Conclusions</H4></CENTER><P>Our antisense RNAs construct can verify the following hypotheses:</P><UL><LI><I>There is a significant decrease in basal expression level when antisense RNA type I and II are used. </I></LI><LI><I>Antisense RNA type II reduces basal level expression more efficiently.</I></LI><LI><I>Positive feedback loop remains function after activation by 1.00E-05 M of AHL after 3 hours.</I></LI></UL><SPN style="height:25px;"><CENTER><H4>Suggestions</H4></CENTER><UL><LI><I>Antisense RNA sequences can be further fine-tuned to investigate higher efficiency of basal level reduction. Insertion of a stabilizer element would increase antisense RNA’s half-life (Engdahl, 2001).</I></LI><LI><I>Higher concentration of AHL can be tested with PFB+asRNA1 and PFB+asRNA2 to investigate their effects on expression level in positive feedback loop.</I></LI><LI><I> The effect of AHL positive feedback loop can be tested in population dynamics </I></LI></UL><CENTER><H4>Additional comments/Notes</H4></CENTER><P>W/ PFB sample increases the expression by only 4,316.5 gfp/OD600 when induced by 1.00E-05 M AHL after 3
                    hours, at the same time which fluorescence of w/o PFB increases by 10 folds. </P><P>Fig. 1 and 3 show that w/o PFB produces the highest fluorescence/OD600, while W/ PFB is significantly lower.
                    This has been experienced by brown iGEM team 2007, where their hypotheses have been proven false due
                    to an inverse relationship of GFP expression between the control (no feedback loop) and the one with
                    LuxI (Positive feedback loop). Given that the concentration of SAM is constant, the unpredictable lower level of fluorescence may be due to high concentration of AHL and low concentration of luxI that cause the shift in equilibrium to the reactant (SAM) rather than to the product side. However, in our course of experiment, we were interested in the efficiency of two types
                    of antisense RNA in reducing basal expression level, so we took W/ PFB construct for the comparison.
                  </P><P>We have considered tuning <I>pLuxR</I> promoter instead of adding ABR and antisense RNA. However, we predicted
                    that there will be a tradeoff. If the promoter is tuned to express low level of LuxI, that will minimize
                    AHL production. Thus, the leakiness will be reduced. However, that means the impact of AHL diffusing
                    across cell membrane will be impeded given that AHL concentration inside the cell affects the diffusion
                    rate across cell membrane.</P><P>The adjustment of <I>pLuxR</I> promoter should satisfy the criteria that:</P><UL><LI><I>It will not constitutively express until LuxR:AHL complex induces</I></LI><LI><I>It can produce high LuxI product which catalyzes high concentration of AHL once induced.</I></LI></UL><P>Apparently, mutation of promoter to lower the leakiness cannot compensate the second criterion. Thus, the
                    second suggestion for improvement is to use Gram-positive bacteria as a chassis. </P><CENTER><H4>CUHK's characterization</H4></CENTER><P> We sent our parts to CUHK to characterize and validate our sensing module’s constructs for the effect of
                    antisense RNA on the basal expression level. However, their results are not consistent with our results.
                    Because there was no increase in fluorescence/OD600 for our positive control I, we omit their results
                    in our data processing and interpretation. </P><OL><CENTER><H3> References</H3></CENTER><LI id="ref1"><A style="color:white; word-wrap: break-word;" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2776824/#R3">
Andrew Camilli (2006) Bacterial Small-Molecule Signaling Pathways. Science. 311(5764), 1113–1116. doi:  10.1126/science.1121357</A></LI><LI id="ref2"><A style="color:white; word-wrap: break-word;" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1482843/">Brooke A. McDaniel (2006) Identification of a Mutation in the Bacillus subtilis S-Adenosylmethionine Synthetase Gene That Results in Derepression of S-Box Gene Expression.J Bacteriol.188(10), 3674–3681. doi:  10.1128/JB.188.10.3674-3681.2006</A></LI><LI id="ref3"><A style="color:white; word-wrap: break-word;" href="https://www.ncbi.nlm.nih.gov/pubmed/11258619">Engdahl HM (2001), Introduction of an RNA stability element at the 5'-end of an antisense RNA cassette increases the inhibition of target RNA translation. Antisense Nucleic Acid Drug Dev 11(1), 29-40. Doi: 10.1089/108729001750072100</A></LI><LI id="ref4"><A style="color:white; word-wrap: break-word;" href="http://europepmc.org/abstract/med/17905197">Daniel J. Sayut (2007) Noise and kinetics of LuxR Positive feedback loops, Biochemical and biophysical Research Communities, 363(3), 667-673, doi: https://doi.org/10.1016/j.bbrc.2007.09.057</A></LI><LI id="ref5"><A style="color:white; word-wrap: break-word;" href="http://pubs.acs.org/doi/abs/10.1021/acssynbio.6b00036">Hoynes-O’Connor, A., &amp; Moon, T. (2016). Development of Design Rules for Reliable Antisense RNA Behavior in E. coli. ACS Synthetic Biology, 5(12), 1441-1454. http://dx.doi.org/10.1021/acssynbio.6b00036</A></LI><LI id="ref6"><A style="color:white; word-wrap: break-word;" href="http://www.diva-portal.org/smash/record.jsf?pid=diva2%3A387484&amp;dswid=6111">Wagner, E. (2009). Kill the messenger: bacterial antisense RNA promotes mRNA decay. Nature Structural &amp; Molecular Biology, 16(8), 804-806. http://dx.doi.org/10.1038/nsmb0809-804</A></LI></OL></SPN></SECTION></DIV></DIV><DIV><FOOTER style="opacity:0.7" id="footer"><UL class="icons"><LI><DIV style="display:flex; flex-direction:row; justify-content:center"><A href="https://www.facebook.com/hkustigem2017domilox/" style="font-size:15px; color:blue"> Follow Us Now !!!</A></DIV></LI><LI><DIV style="display:flex; flex-direction:row; justify-content:center"><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/SiteMap" style="font-size:15px; color:red">SiteMap</A></DIV></LI></UL><UL class="copyright"><LI><B>© iGEM 2017 Team HKUST. All rights reserved.</B></LI><LI>made with <SPAN style="color: red">♥</SPAN> @ HKUST</LI></UL></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>