<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aix-Marseille_Composite_Part skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aix-Marseille/Composite Part</SPAN></H1></DIV></DIV></DIV><DIV id="amu-content"><NAV class="navbar navbar-fixed-top"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle">
Toggle navigation ▤
</BUTTON><A class="navbar-brand" href="/Team:Aix-Marseille">iGEM Aix-Marseille Université</A></DIV><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav"><LI class="menu_item" id="menu-team"><A href="#menu-team">Team</A></LI><LI><A href="/Team:Aix-Marseille/Team"> Team   </A></LI><LI><A href="/Team:Aix-Marseille/Collaborations">★  Collaborations </A></LI></UL><LI class="menu_item" id="menu-project"><A href="#menu-project">Project</A></LI><LI><A href="/Team:Aix-Marseille/Description"> ★  Description </A></LI><LI><A href="/Team:Aix-Marseille/Design"> ★ Design </A></LI><LI><A href="/Team:Aix-Marseille/Experiments"> Experiments </A></LI><LI><A href="/Team:Aix-Marseille/Results"> Results </A></LI><LI><A href="/Team:Aix-Marseille/Notebook"> Notebook </A></LI><LI class="menu_item" id="menu-parts"><A href="#menu-parts"> Parts</A></LI><LI><A href="/Team:Aix-Marseille/Parts">Parts </A></LI><LI><A href="/Team:Aix-Marseille/Basic_Part"> ★ Basic Parts </A></LI><LI><A href="/Team:Aix-Marseille/Composite_Part"> ★ Composite Parts </A></LI><LI class="menu_item"><A href="/Team:Aix-Marseille/Safety"> Safety </A></LI><LI class="menu_item"><A href="/Team:Aix-Marseille/Medals"> Medals</A></LI><LI class="menu_item"><A href="/Team:Aix-Marseille/Attributions">★  Attributions </A></LI><LI class="menu_item" id="menu-hp"><A href="#menu-hp">Human Practices</A></LI><LI><A href="/Team:Aix-Marseille/Human_Practices"> Human Practices </A></LI><LI><A href="/Team:Aix-Marseille/Integrated_Practices"> ★ Integrated Practices </A></LI><LI><A href="/Team:Aix-Marseille/Engagement">★ Engagement </A></LI><LI class="menu_item"><A href="/Team:Aix-Marseille/Model"> Model </A></LI><DIV class="social"><UL><LI><A href="https://www.facebook.com/iGEM.AMU">
                            facebook
                        </A></LI><LI><A href="https://twitter.com/iGEMteamAMU">
                            twitter
                        </A></LI></UL></DIV></DIV></NAV><DIV class="content_wrapper"><DIV class="tocp"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Mobilisation_by_a_Siderophore"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Mobilisation by a Siderophore</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#BBa_K1951004_:_DesA_producer"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">BBa_K1951004 : DesA producer</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#Part_Composition:"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Part Composition:</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Lysine_decarboxylase_DesA_:"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">Lysine decarboxylase DesA :</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-5"><A href="#BBa_K1951011_:_Desferrioxamine_B_pathway_producer_in_Streptomyces_coelicolor"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">BBa_K1951011 : Desferrioxamine B pathway producer in <I> Streptomyces coelicolor </I></SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#A_precious_metal_binder"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">A precious metal binder</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#A_medical_treatment"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">A medical treatment</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Pathway_demonstration"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">Pathway demonstration</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Biosorption_parts"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Biosorption parts</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#BBa_K1951008_:_FliC_E._coli_producer"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">BBa_K1951008 : FliC <I>E. coli</I> producer</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Part_Composition:_2"><SPAN class="tocnumber">2.1.1</SPAN><SPAN class="toctext">Part Composition:</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Flagellin_C_from_Escherichia_coli"><SPAN class="tocnumber">2.1.2</SPAN><SPAN class="toctext">Flagellin C from <I>Escherichia coli</I></SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Flagellin_C_swimming_test"><SPAN class="tocnumber">2.1.3</SPAN><SPAN class="toctext">Flagellin C swimming test</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Part_Assembly:"><SPAN class="tocnumber">2.1.4</SPAN><SPAN class="toctext">Part Assembly:</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#BBa_K1941009_:_FliC_Desulfovibrio_vulgaris_producer"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">BBa_K1941009 : FliC <I><I>Desulfovibrio vulgaris</I></I> producer</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#General"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">General</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#Metal_Biosorption_Capacity"><SPAN class="tocnumber">2.2.2</SPAN><SPAN class="toctext">Metal Biosorption Capacity</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#Immune_response_capacity"><SPAN class="tocnumber">2.2.3</SPAN><SPAN class="toctext">Immune response capacity</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#BBa_K1941010_:_CsgA_Escherichia_coli_producer"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">BBa_K1941010 : CsgA <I><I>Escherichia coli</I></I> producer</SPAN></A></LI></DIV></DIV><DIV class="page_content"><H1>Composite part</H1><H2><SPAN class="mw-headline" id="Mobilisation_by_a_Siderophore">Mobilisation by a Siderophore</SPAN></H2><H3><SPAN class="mw-headline" id="BBa_K1951004_:_DesA_producer"><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1951004">BBa_K1951004</A> : DesA producer</SPAN></H3><H4><SPAN class="mw-headline" id="Part_Composition:">Part Composition:</SPAN></H4><P>This part is composed of 3 subparts :
</P><UL><LI><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_I0500">I0500</A> : pARA/araC inducible promoter</LI><LI><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_B0034">B0034</A> : Ribosome Binding Site</LI><LI><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1951000">K1951000</A> : Lysine decarboxylase</LI></UL><H4><SPAN class="mw-headline" id="Lysine_decarboxylase_DesA_:">Lysine decarboxylase DesA :</SPAN></H4><P>This DNA sequence codes a <A rel="nofollow" class="external text" href="http://metacyc.org/gene?orgid=META&amp;id=SCO2782">Lysine decarboxylase</A> (<I>Streptomyces coelicolor</I>) which is an enzyme from the lyase family that converts lysine into cadaverine. The enzyme releases the carbonyl group of the lysin amino acid. Cadaverine (or 1,5-diaminopentane) is a primary diamine which renders the medium alkaline. The lysine decarboxylase is an enzyme whose synthesis is promoted by anaerobiosis and an acidic pH. 
This enzyme is also the first step in the production of desferrioxame B which is a siderophore.
</P><P>We succeed to produce DesA under an Arabinose indcution.
</P><P>We registered the original sequence of this subpart in the iGEM registry of standard parts (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1951000">BBa_K1951000</A>). We optimized our sequence for <I>E. coli</I> and ordered the synthesis by addition of an inducible promoter.
</P><P>As you can see in <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1951004">the registry</A> all our expectations on this BioBrick were validated and it worked fine.
</P><H3><SPAN class="mw-headline" id="BBa_K1951011_:_Desferrioxamine_B_pathway_producer_in_Streptomyces_coelicolor"><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1951011">BBa_K1951011</A> : Desferrioxamine B pathway producer in <I> Streptomyces coelicolor </I></SPAN></H3><DIV class="emph"><P>The purpose of this biobrick is to produce desferrioxamine B siderophore by a controled manner, for controlled potential toxicity. This is also to reconstruct the full pathway of this siderophore production. 
</P><P>This biobrick was made from 6 parts:
</P><P>Available in the registry : 
</P><UL><LI><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_I0500">BBa_I0500</A></LI><LI><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A></LI></UL><P>Designed by our team : 
</P><UL><LI><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1951000">BBa_K1951000</A></LI><LI><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1951001">BBa_K1951001</A></LI><LI><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1951002">BBa_K1951002</A></LI><LI><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1951003">BBa_K1951003</A></LI></UL></DIV><P>
Siderophores are small, high-affinity iron chelating compounds secreted by microorganisms such as bacteria, fungi and grasses. Siderophores are amongst the strongest soluble Fe3+ binding agents known.
</P><H4><SPAN class="mw-headline" id="A_precious_metal_binder">A precious metal binder</SPAN></H4><P>Previous work showed that siderophores are able to catch other metals by default. Especifically, many articles showed a high affinity of Desferrioxamine B (produce by <I>Streptomyces coelicolor</I>) for tetravalent metal ions like platinum. These results encouraged us to make a biobrick coding the sequence corresponding to the 4 enzymes involved on the metabolic pathway of Desferrioxamine B. <I>E. coli</I> was used to produce this biobrick. Produce a gram positive bacteria pathway in a gram negative bacteria is restrictive <SUP id="cite_ref-1" class="reference"><A href="#cite_note-1">[1]</A></SUP> , considering the risk of toxicity for themselves. To counter this potential issue, we regulate transcription using the control of an inductible promotor (pBAD/araC).
</P><H4><SPAN class="mw-headline" id="A_medical_treatment">A medical treatment</SPAN></H4><P>Desferrioxamine B is a really well referenced siderophore. 
</P><P>Deferoxamine acts by binding free iron in the bloodstream and enhancing its elimination in the urine. By removing excess iron, the agent reduces the damage done to various organs and tissues, such as the liver. Also, it speeds healing of nerve damage (and minimizes the extent of recent nerve trauma). 
</P><UL><LI> It is currently used of on the World Health Organization's List of Essential Medicines, the most important medications needed in a basic health system. But it is used in the treatment of many others diseases as well. </LI></UL><UL><LI> Desferrioxamine B may modulate expression and release of inflammatory mediators by specific cell types.</LI></UL><H4><SPAN class="mw-headline" id="Pathway_demonstration">Pathway demonstration</SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Test of our biobrick proteins production using a SDS PAGE and coomassie blue staining. - : no induction ; + : induction with 0.02% arabinose at Abs(600nm)=0.4</DIV></DIV></DIV><P>We investigated if the DesA, DesB, DesC and DesD proteins were well produced by our biobrick <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1951011">BBa_K1951011</A> using SDS PAGE. 
</P><P>To do this we performed SDS PAGE <A rel="nofollow" class="external text" href="https://2016.igem.org/Team:Aix-Marseille/Experiments/Protocols#Protocol_.2312_:_SDS_page_and_coomassie_blue">(protocol)</A> and stained with coomassie blue using cells containing this biobrick. From an over night starter, cells were diluted and grown from Abs(600nm)=0.2 to Abs(600nm)=1. 
Then 1UOD of cells (1.67ml at 0.6OD) was collected and centrifuged at 5000g for 5min. 
After removal of the supernatant, the cell pellet was resuspended in 50µL SDS-PAGE sample buffer. 
We heated the mix at 95°C during 15min.
The sample was loaded onto a polyacrylamide gel and migrated during 50min at 180V. 
Staining was done using coomassie blue.
</P><P>We compared the production of proteins in different background : 
</P><P>- <I>E. coli</I>  TG1 strain with pSB1C3 containing the RFP coding sequence (negative control)
</P><P>- <I>E. coli</I>  TG1 strain with <I>des</I> operon (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1951011">BBa_K1951011</A>) before and after induction. (You can observe the production of the 4 proteins on the figure on the right; left : before induction, right : after induction)
</P><P>Results showed the production of the 4 proteins DesA, DesB, DesC and DesD, all involved in the desferrioxamine B biosynthesis pathway. We can notice that a leak of the promoter pBAD, indeed we note the production of Des proteins without arabinose induction.
</P><H2><SPAN class="mw-headline" id="Biosorption_parts">Biosorption parts</SPAN></H2><H3><SPAN class="mw-headline" id="BBa_K1951008_:_FliC_E._coli_producer"><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1951008">BBa_K1951008</A> : FliC <I>E. coli</I> producer</SPAN></H3><DIV class="emph"><P>We also improved an existing BioBrick, Glasgow 2014 team's BioBrick (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1463604">BBa K1463604</A>), making our brick (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1951008">BBa K1951008</A>). This BioBrick is an improvement of the orignial BioBrick in three ways:
</P><UL><LI> The codon usage has been optimized to ensure a strong expression in E.coli, </LI><LI> The RBS has no mutation so the flagellin is well expressed,</LI><LI> Our swimming test show that the swimming phenotype of the BioBick is improved thanks to our changes.</LI></UL><P>All the functional tests and experiments, show this part is functional, are listed <A rel="nofollow" class="external text" href="https://2016.igem.org/Team:Aix-Marseille/Composite_Part#BBa_K1951008_:_FliC_E._coli_producer">here</A>.
</P></DIV><H4><SPAN class="mw-headline" id="Part_Composition:_2">Part Composition:</SPAN></H4><P>This part is a composite part composed of 2 Biobricks :
</P><UL><LI><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1951005">BBa_K1951005</A> : Flagellin C</LI><LI><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K880005">BBa_K880005</A> : Strong promoter, strong RBS combination</LI></UL><H4><SPAN class="mw-headline" id="Flagellin_C_from_Escherichia_coli">Flagellin C from <I>Escherichia coli</I></SPAN></H4><P>Flagellin C (FliC) protein from <I>Escherichia coli</I> strain is the main protein constitutive of the flagelar filament and is involved to promote bacterial swimming. This sequence is conserved in many bacterial strains. It has been demonstrated that flagellin has the ability to adsorb precious metal such as platinum or gold.
We made a FliC mutant by transduction using phage P1 in a <I>E. Coli</I> W3110 strain. Then we have complemented the <I>fliC</I> mutant W3110 with <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1951008">BBa_K1951008</A> and performed a swimming test for every background. The result has shown that swimming was recovered into the complemented <I>fliC</I> mutant W3110
</P><H4><SPAN class="mw-headline" id="Flagellin_C_swimming_test">Flagellin C swimming test</SPAN></H4><P>We constructed a fliC deletion mutant that is unable to swim, from the wild-type strain W3110. The ability to swim was restored to this mutant by our biobrick, as can be seen in the illustration here. This demonstrated that the protein can be correctly inserted into flagella and functions.
</P><H4><SPAN class="mw-headline" id="Part_Assembly:">Part Assembly:</SPAN></H4><P>The subparts were assembled using standard BioBrick Assembly.
</P><H3><SPAN class="mw-headline" id="BBa_K1941009_:_FliC_Desulfovibrio_vulgaris_producer"><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1951009">BBa_K1941009</A> : FliC <I><I>Desulfovibrio vulgaris</I></I> producer</SPAN></H3><H4><SPAN class="mw-headline" id="General">General</SPAN></H4><P>Flagellin (FliC) protein from <I>Desulfovibrio vulgaris</I> strain is the main protein constitutive of the flagellum filament and is involved in bacterial swimming. This protein is conserved in many bacterial strains as the capacity of swimming given by the flagellum confers a great selective advantage.
</P><H4><SPAN class="mw-headline" id="Metal_Biosorption_Capacity">Metal Biosorption Capacity</SPAN></H4><P>It has been demonstrated that Flagellin has the ability to adsorb precious metal on its surface such as platinum, palladium gold <SUP id="cite_ref-2" class="reference"><A href="#cite_note-2">[2]</A></SUP><SUP id="cite_ref-3" class="reference"><A href="#cite_note-3">[3]</A></SUP> and this was important for our project <A rel="nofollow" class="external text" href="https://2016.igem.org/Team:Aix-Marseille/Design#Biosorption_and_reduction_using_flagellin_and_peptides">Highway to platinum</A></P><H4><SPAN class="mw-headline" id="Immune_response_capacity">Immune response capacity</SPAN></H4><P>The propensity of the immune response to flagellin may be explained by two facts:
</P><UL><LI> Flagellin is an extremely abundant protein in flagellated bacteria.</LI><LI> There exists a specific innate immune receptor that recognizes flagellin, Toll-like receptor 5 (TLR5). <SUP id="cite_ref-4" class="reference"><A href="#cite_note-4">[4]</A></SUP></LI></UL><H3><SPAN class="mw-headline" id="BBa_K1941010_:_CsgA_Escherichia_coli_producer"><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1941010">BBa_K1941010</A> : CsgA <I><I>Escherichia coli</I></I> producer</SPAN></H3><P>CsgA is the major structural subunit of the curli fimbriae.  Curli fibers are involved in adhesion to surfaces, cell aggregation, and biofilm formation<SUP id="cite_ref-Curli_5-0" class="reference"><A href="#cite_note-Curli-5">[5]</A></SUP>. Curli also mediate host cell adhesion and invasion, and they are potent inducers of the host inflammatory response. The biobrick contains a strong promotor.
</P><OL class="references"><LI id="cite_note-1"><SPAN class="mw-cite-backlink"><A href="#cite_ref-1">↑</A></SPAN><SPAN class="reference-text"> Wandersman &amp; Delepaire <A rel="nofollow" class="external free" href="https://www.ncbi.nlm.nih.gov/pubmed/15487950">https://www.ncbi.nlm.nih.gov/pubmed/15487950</A></SPAN></LI><LI id="cite_note-2"><SPAN class="mw-cite-backlink"><A href="#cite_ref-2">↑</A></SPAN><SPAN class="reference-text">Deplanche &amp; al., 2007 <A rel="nofollow" class="external free" href="http://onlinelibrary.wiley.com/doi/10.1002/bit.21688/abstract">http://onlinelibrary.wiley.com/doi/10.1002/bit.21688/abstract</A>. </SPAN></LI><LI id="cite_note-3"><SPAN class="mw-cite-backlink"><A href="#cite_ref-3">↑</A></SPAN><SPAN class="reference-text">Capeness &amp; al. 2015, <A rel="nofollow" class="external free" href="http://eprints.nottingham.ac.uk/27979/1/Michael%20Capeness%20-%20Thesis%20-%20PDF.pdf">http://eprints.nottingham.ac.uk/27979/1/Michael%20Capeness%20-%20Thesis%20-%20PDF.pdf</A></SPAN></LI><LI id="cite_note-4"><SPAN class="mw-cite-backlink"><A href="#cite_ref-4">↑</A></SPAN><SPAN class="reference-text"> Kathrani A. &amp; al, 2012 <A rel="nofollow" class="external free" href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0030117">http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0030117</A>) </SPAN></LI><LI id="cite_note-Curli-5"><SPAN class="mw-cite-backlink"><A href="#cite_ref-Curli_5-0">↑</A></SPAN><SPAN class="reference-text"><A rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2838481/">Michelle M. Barnhart and Matthew R. Chapman 2010, Annual Review of Microbiology</A></SPAN></LI></OL></DIV></DIV><FOOTER><A href="http://igem.cnrs-mrs.fr">iGEM Aix-Marseille Université</A> - All content on this wiki is available under the Creative Commons Attribution 4.0 license (or any later version).
</FOOTER></DIV></BODY></HTML>