<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aix-Marseille_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aix-Marseille/Results</SPAN></H1></DIV></DIV></DIV><DIV id="amu-content"><NAV class="navbar navbar-fixed-top"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle">
Toggle navigation ▤
</BUTTON><A class="navbar-brand" href="/Team:Aix-Marseille">iGEM Aix-Marseille Université</A></DIV><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav"><LI class="menu_item" id="menu-team"><A href="#menu-team">Team</A></LI><LI><A href="/Team:Aix-Marseille/Team"> Team   </A></LI><LI><A href="/Team:Aix-Marseille/Collaborations">★  Collaborations </A></LI></UL><LI class="menu_item" id="menu-project"><A href="#menu-project">Project</A></LI><LI><A href="/Team:Aix-Marseille/Description"> ★  Description </A></LI><LI><A href="/Team:Aix-Marseille/Design"> ★ Design </A></LI><LI><A href="/Team:Aix-Marseille/Experiments"> Experiments </A></LI><LI><A href="/Team:Aix-Marseille/Results"> Results </A></LI><LI><A href="/Team:Aix-Marseille/Notebook"> Notebook </A></LI><LI class="menu_item" id="menu-parts"><A href="#menu-parts"> Parts</A></LI><LI><A href="/Team:Aix-Marseille/Parts">Parts </A></LI><LI><A href="/Team:Aix-Marseille/Basic_Part"> ★ Basic Parts </A></LI><LI><A href="/Team:Aix-Marseille/Composite_Part"> ★ Composite Parts </A></LI><LI class="menu_item"><A href="/Team:Aix-Marseille/Safety"> Safety </A></LI><LI class="menu_item"><A href="/Team:Aix-Marseille/Medals"> Medals</A></LI><LI class="menu_item"><A href="/Team:Aix-Marseille/Attributions">★  Attributions </A></LI><LI class="menu_item" id="menu-hp"><A href="#menu-hp">Human Practices</A></LI><LI><A href="/Team:Aix-Marseille/Human_Practices"> Human Practices </A></LI><LI><A href="/Team:Aix-Marseille/Integrated_Practices"> ★ Integrated Practices </A></LI><LI><A href="/Team:Aix-Marseille/Engagement">★ Engagement </A></LI><LI class="menu_item"><A href="/Team:Aix-Marseille/Model"> Model </A></LI><DIV class="social"><UL><LI><A href="https://www.facebook.com/iGEM.AMU">
                            facebook
                        </A></LI><LI><A href="https://twitter.com/iGEMteamAMU">
                            twitter
                        </A></LI></UL></DIV></DIV></NAV><DIV class="content_wrapper"><DIV class="tocp"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Mobilisation_results"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Mobilisation results</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Protein_production"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Protein production</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Proof_of_fonctionnality"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Proof of fonctionnality</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#In_the_future"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">In the future</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-5"><A href="#Biosorption_result"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Biosorption result</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Protein_production_2"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Protein production</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Proof_of_swimming_recovery"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Proof of swimming recovery</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Electron_microscopy_of_the_flagella"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Electron microscopy of the flagella</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#In_the_futur"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">In the futur</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Improvement_of_FliC_E._coli_BBa_K1463604"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Improvement of FliC <I>E. coli</I> BBa_K1463604</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Futur_plan_for_our_project"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Futur plan for our project</SPAN></A></LI></DIV></DIV><DIV class="page_content"><H1> Results</H1><P>Our lab experience enabled us to create the final Biobricks of our project which are a siderophore Desferrioxamine B producer and a flagellin producer.
</P><H2><SPAN class="mw-headline" id="Mobilisation_results">Mobilisation results</SPAN></H2><H3><SPAN class="mw-headline" id="Protein_production">Protein production</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Test of our biobrick proteins production using a SDS PAGE and coomassie blue staining. - : no induction ; + : induction with 0.02% arabinose at Abs(600nm)=0.4</DIV></DIV></DIV><P>We investigated if the DesA, DesB, DesC and DesD proteins were well produced by our biobrick <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1951011">BBa_K1951011</A> using SDS PAGE. 
</P><P>To do this we performed SDS PAGE <A rel="nofollow" class="external text" href="https://2016.igem.org/Team:Aix-Marseille/Experiments/Protocols#Protocol_.2312_:_SDS_page_and_coomassie_blue">(protocol)</A> and stained with coomassie blue using cells containing this biobrick. From an over night starter, cells were diluted and grown from Abs(600nm)=0.2 to Abs(600nm)=1. 
Then 1UOD of cells (1.67ml at 0.6OD) was collected and centrifuged at 5000g for 5min. 
After removal of the supernatant, the cell pellet was resuspended in 50µL SDS-PAGE sample buffer. 
We heated the mix at 95°C during 15min.
The sample was loaded onto a polyacrylamide gel and migrated during 50min at 180V. 
Staining was done using coomassie blue.
</P><P>We compared the production of proteins in different background : 
</P><P>- <I>E. coli</I>  TG1 strain with pSB1C3 containing the RFP coding sequence (negative control)
</P><P>- <I>E. coli</I>  TG1 strain with <I>des</I> operon (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1951011">BBa_K1951011</A>) before and after induction. (You can observe the production of the 4 proteins on the figure on the right; left : before induction, right : after induction)
</P><P>Results showed the production of the 4 proteins DesA, DesB, DesC and DesD, all involved in the desferrioxamine B biosynthesis pathway.
</P><H3><SPAN class="mw-headline" id="Proof_of_fonctionnality">Proof of fonctionnality</SPAN></H3><P>We investigated if DesA (Lysine decarboxylase) was fonctionnal by measurement of cadaverine using HPLC with C18 column and we proved that our biobrick allows the production of DesA which enables to produce the cadaverine and makes it fonctionnal.  
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Investigation of the cadaverine production by the lysine decarboxylase DesA. The cadaverin production has been detected by HPLC using C18 column after induction of the <I>des</I> genes. Different backgrounds were analysed : wild type <I>Escherichia coli</I> TG1 strain (yellow column), <I>cadA</I> mutant from Keio bank (blue column), complemented <I> cadA </I> mutant from Keio bank with p<I>desA</I> (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1951004">BBa_K1951004</A>) (orange column), complemented <I>cadA</I> mutant from Keio bank with <I>des</I> operon (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1951011">BBa_K1951011</A>) (grey column).</DIV></DIV></DIV><P>Results showed cadaverine detection in the wild type meaning the original strain well produces the cadaverine.
</P><P>In <I>cadA</I> mutant from Keio bank, cadaverine was also produced in a least quantity showing that an other pathway is responsible for the production of cadaverine (existence of an other constitutive lysine decarboxylase in the <I> E. coli </I> genome. 
</P><P>In the <I>cadA</I> mutant complemented by p<I>desA</I> (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1951004">BBa_K1951004</A>), the amount of cadaverine was recovered and even beyond the wild type production.
</P><P>Moreover, in the <I>cadA</I> mutant complemented by the des operon (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1951011">BBa_K1951011</A>), the cadaverine level produced was even over the wild type and complemented Bba_K1951004 production. We explain this result because of the higher stability of this big composite part.
</P><P>To conclude, we made a big composite part able to produce every proteins involved in the biosynthetic pathway of the desferrioxamine B.
</P><H3><SPAN class="mw-headline" id="In_the_future">In the future</SPAN></H3><UL><LI> Test the whole pathway</LI></UL><P>	- siderophore production (HPLC and absorption spectroscopy)
	- Improve the siderophore production if needed (cofactors, protein stability, metabolic engineering)
</P><UL><LI> Binding affinity for platinum by our siderophore</LI></UL><P>	 - ICPAES (ion coupled plasma atomic emission spectroscopy)
</P><UL><LI> Importation of the siderophore in <I>Streptomyces coelicolor</I></LI></UL><H2><SPAN class="mw-headline" id="Biosorption_result">Biosorption result</SPAN></H2><H3><SPAN class="mw-headline" id="Protein_production_2">Protein production</SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Test of proteins production with biobrick <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1951008">BBa_k1951008</A> using a SDS PAGE and coomassie blue stain</DIV></DIV></DIV><P>We investigated if the FliC protein was well produced by our biobrick (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1951008">BBa_k1951008</A>). 
</P><P>To do this we performed SDS PAGE and stained with coomassie blue using cells containing this biobrick.
</P><P>From an over night starter, cells were diluted and grown from Abs(600nm)=0.2 to Abs(600nm)=0.6.
</P><P>Then 1UOD of cells (1.67ml at 0.6OD) was collected and centrifuged at 5000g for 5min. 
</P><P>After removal of the supernatant, the cell pellet was resuspended in 50µL SDS-PAGE sample buffer. 
</P><P>The sample was loaded onto a polyacrylamide gel and migrated during 50min at 180V. 
</P><P>Staining was done using coomassie blue. 
</P><P>The FliC is at mass 51,3kDa, and can be seen in the gel photograph.
</P><H3><SPAN class="mw-headline" id="Proof_of_swimming_recovery">Proof of swimming recovery</SPAN></H3><P>We have made a biobrick <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1951008">BBa_k1951008</A> ables to produce Flagellin (FliC protein of the flagellum). In the aim to test the flagellin integrity, we made a <I>fliC</I> mutant in a <I>E. coli</I> W3110 strain by transduction using phage P1 (<A rel="nofollow" class="external text" href="https://2016.igem.org/Team:Aix-Marseille/Experiments/Protocols#Step2_:_Transduction_P1">protocol</A>). 
</P><P>In the figure the deletion mutant (lower left sector) shows no swimming motility as expected and a small white colony. In contrast the wild-type colony (lower right) has a diffuse halo due to swimming cells around the central white colony. 
Finally the complemented strain, the deletion mutant complemented with our biobrick, (top panel) shows two colonies with intense halos surrounding them. This illustrated clearly that our biobrick can restore motility and is functional.
The intensity of the halo suggests that a greater proportion of the cells are mobile or swimming is in someway better than the wild-type.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:772px;"><DIV class="thumbcaption">We investigated if swimming motility was recovered by a knockout <I>fliC</I> strain.  To test complementation with our biobrick (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1951008">BBa_K1951008</A>), strains were stabbed into soft (0.3%) LB agar plates, and incubated at 37°C for 4 hours. Three strains are shown:  <I> Escherichia coli</I> W3110 wild-type strain, which has a good swimming capacity (lower right); a fliC deletion mutant of W3110 (lower left); and the fliC mutant complemented with <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1951008">BBa_K1951008</A> (top).</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Electron_microscopy_of_the_flagella">Electron microscopy of the flagella</SPAN></H3><P>This check the flagella assembly and the integration of the flagellin protein expressed from our biobrick BBa_K1951008 we have observed bacteria with an electron microscope. 
</P><P>
The image shows mutiple polar flagella in an <I>E. coli fliC</I> deletion mutant containing our biobrick. We saw that these cells had more flagella than wild-type (W3110) cells. The experiments suggest the functionality of our biobrick even if this experiment need a proper negative control i.e. a deletion mutant with no flagella in order to prove effect of our biobrick.
</P><H3><SPAN class="mw-headline" id="In_the_futur">In the futur</SPAN></H3><UL><LI> Adsorption of platinum by our flagella</LI></UL><P>	 - election microscopy observation 
</P><UL><LI> Finalise the insertion of platinum specific peptide</LI></UL><P>	- directed mutagenesis introducing a Bbs1 restriction site
</P><P>	- insertion of coding sequence of peptide by annealing
</P><UL><LI> Metal recovery by calcination or proteolysis</LI></UL><UL><LI> Extend to other precious metals (other peptides : gold, silver, palladium)</LI></UL><H3><SPAN class="mw-headline" id="Improvement_of_FliC_E._coli_BBa_K1463604">Improvement of FliC <I>E. coli</I><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1463604">BBa_K1463604</A></SPAN></H3><P>This biobrick is an improvement on the biobrick designed by the Glasgow 2014 team. 
<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1463604">BBa_K1463604</A></P><P>The improvement of this part is multiple.
</P><UL><LI> First there is no mutation in the promotor or RBS of our part so the flagellin is well expressed and functional. Unfortunately when the Glasgow team trie to make this part they picked up a mutation in the promotor.</LI><LI> Second the sequence that we have used it a synthetic gene with codon optimisation, designed specifically for high level expression in <I>E. coli</I>. Thus as an expression part is improved over the initial design.</LI><LI> Third in the functional swimming assay, we see evidence for improved swimming (denser halo), in cells expressing our biobrick, a phenotype not observed by the Glasgow team in 2014.</LI></UL><H2><SPAN class="mw-headline" id="Futur_plan_for_our_project">Futur plan for our project</SPAN></H2><P>Now that every biobrick have been created and are available, make proof of concept could allow to envisage investigation about possibilities of an industrial application. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:772px;"><DIV class="thumbcaption">Maquette of our process in the futur</DIV></DIV></DIV></DIV><P>The biologic production of high and specific adsorber could reduce environnemental impact of the metal production and its cost while renew essential precious metals like platinum, for a sustainable production. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Igem Aix Marseille 2016 in a sustainable circle</DIV></DIV></DIV></DIV></DIV></DIV><FOOTER><A href="http://igem.cnrs-mrs.fr">iGEM Aix-Marseille Université</A> - All content on this wiki is available under the Creative Commons Attribution 4.0 license (or any later version).
</FOOTER></DIV></BODY></HTML>