<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aix-Marseille_test-Collaborations skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aix-Marseille/test-Collaborations</SPAN></H1></DIV></DIV></DIV><DIV id="amu-content"><NAV class="navbar navbar-fixed-top"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle">
Toggle navigation ▤
</BUTTON><A class="navbar-brand" href="/Team:Aix-Marseille">iGEM Aix-Marseille Université</A></DIV><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav"><LI class="menu_item" id="menu-team"><A href="#menu-team">Team</A></LI><LI><A href="/Team:Aix-Marseille/Team"> Team   </A></LI><LI><A href="/Team:Aix-Marseille/Collaborations">★  Collaborations </A></LI></UL><LI class="menu_item" id="menu-project"><A href="#menu-project">Project</A></LI><LI><A href="/Team:Aix-Marseille/Description"> ★  Description </A></LI><LI><A href="/Team:Aix-Marseille/Design"> ★ Design </A></LI><LI><A href="/Team:Aix-Marseille/Experiments"> Experiments </A></LI><LI><A href="/Team:Aix-Marseille/Results"> Results </A></LI><LI><A href="/Team:Aix-Marseille/Notebook"> Notebook </A></LI><LI class="menu_item" id="menu-parts"><A href="#menu-parts"> Parts</A></LI><LI><A href="/Team:Aix-Marseille/Parts">Parts </A></LI><LI><A href="/Team:Aix-Marseille/Basic_Part"> ★ Basic Parts </A></LI><LI><A href="/Team:Aix-Marseille/Composite_Part"> ★ Composite Parts </A></LI><LI class="menu_item"><A href="/Team:Aix-Marseille/Safety"> Safety </A></LI><LI class="menu_item"><A href="/Team:Aix-Marseille/Medals"> Medals</A></LI><LI class="menu_item"><A href="/Team:Aix-Marseille/Attributions">★  Attributions </A></LI><LI class="menu_item" id="menu-hp"><A href="#menu-hp">Human Practices</A></LI><LI><A href="/Team:Aix-Marseille/Human_Practices"> Human Practices </A></LI><LI><A href="/Team:Aix-Marseille/Integrated_Practices"> ★ Integrated Practices </A></LI><LI><A href="/Team:Aix-Marseille/Engagement">★ Engagement </A></LI><LI class="menu_item"><A href="/Team:Aix-Marseille/Model"> Model </A></LI><DIV class="social"><UL><LI><A href="https://www.facebook.com/iGEM.AMU">
                            facebook
                        </A></LI><LI><A href="https://twitter.com/iGEMteamAMU">
                            twitter
                        </A></LI></UL></DIV></DIV></NAV><DIV class="content_wrapper"><DIV class="tocp"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Development_of_the_model"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Development of the model</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Objectives"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Objectives</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Proposition:_Adding_contention"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Proposition: Adding contention</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Integration"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Integration</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Results"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Results</SPAN></A></LI></UL></DIV></DIV><DIV class="page_content"><H1>Collaborations</H1><H1><SPAN class="mw-headline" id="Development_of_the_model">Development of the model</SPAN></H1><P>In 1967 Fredrickson <I>et al.</I>  studied mathematically development of a bacterial population, under the assumptions of a large population of independant bacteria in a well mixed solution of constant volume. The large population ensures that for the population the expectation value is a good estimate of the average. The bacteria being independant ensures that the behaviour of each individual depends only on its internal state &lt;math&gt;\mathbf{z}&lt;/math&gt; and the conditions &lt;math&gt;\mathbf{c}&lt;/math&gt; which are the same for all individuals. The volume is well mixed so the conditions &lt;math&gt;\mathbf{c}&lt;/math&gt; which are the same everywhere. The volume is constant so that the population caracteristics can be evaulated by integration over the volume. In their development &lt;math&gt;\mathbf{z}&lt;/math&gt; and &lt;math&gt;\mathbf{c}&lt;/math&gt; are considered to be arbitrary vector quantities.
</P><P>From this starting point they develop a pair of <I>master equations of change</I> to describe the evolution of the population: &lt;math&gt;\begin{gathered}
</P><PRE>\frac{\partial}{\partial t} W_{\mathbf{Z}}(\mathbf{z},t) 
+ \nabla_{\mathbf{Z}} \cdotp [(\mathbf{\beta} \cdotp \bar{\mathbf{R}}(\mathbf{z,c}) W_{\mathbf{Z}}(\mathbf{z},t)] \\
= 2 \int \sigma (\mathbf{z',c}) p(\mathbf{z,z',c}) W_{\mathbf{Z}}(\mathbf{z'},t)\mathrm{d}v'
- (D+\sigma (\mathbf{z,c})) W_{\mathbf{Z}}(\mathbf{z},t)\end{gathered}&lt;/math&gt;&lt;math&gt;\frac{d\mathbf{c}}{dt} 
= D(\mathbf{c_f} - \mathbf{c} ) + \mathbf{\gamma} 
\cdotp \int  \bar{\mathbf{R}}(\mathbf{z,c}) W_{\mathbf{Z}}(\mathbf{z},t)\mathrm{d}v&lt;/math&gt; In these equations the various symbols are as follows:</PRE><TABLE><TBODY><TR><TD>&lt;math&gt;\mathbf{z}                          &lt;/math&gt;</TD><TD> Vector for internal state of a bacteria.
</TD></TR><TR><TD>&lt;math&gt;\mathbf{c}                          &lt;/math&gt;</TD><TD> Time dependant vector for conditions.
</TD></TR><TR><TD>&lt;math&gt;W_{\mathbf{Z}}(\mathbf{z},t)        &lt;/math&gt;</TD><TD> Distribution of bacteria in &lt;math&gt;\mathbf{z}&lt;/math&gt; space at time &lt;math&gt;t&lt;/math&gt;.
</TD></TR><TR><TD>&lt;math&gt;\bar{\mathbf{R}}(\mathbf{z,c})      &lt;/math&gt;</TD><TD> The expected value or the reaction rate vector function of &lt;math&gt;\mathbf{z}&lt;/math&gt; and &lt;math&gt;\mathbf{c}&lt;/math&gt;.
</TD></TR><TR><TD>&lt;math&gt;\sigma (\mathbf{z',c})              &lt;/math&gt;</TD><TD> Rate of fision for bacteria as a scalar function of &lt;math&gt; \mathbf{z,c} &lt;/math&gt;.
</TD></TR><TR><TD>&lt;math&gt;p(\mathbf{z,z',c})                  &lt;/math&gt;</TD><TD> Partitioning probability of generating a child in state &lt;math&gt;\mathbf{z}&lt;/math&gt; from a parent in state &lt;math&gt;\mathbf{z'}&lt;/math&gt; given the conditions &lt;math&gt;\mathbf{c}&lt;/math&gt;.
</TD></TR><TR><TD>&lt;math&gt;\nabla_{\mathbf{Z}}\cdot \mathbf{V} &lt;/math&gt;</TD><TD>&lt;math&gt;\sum \frac{\partial}{\partial z_i}\mathbf{V}_i &lt;/math&gt;</TD></TR><TR><TD>&lt;math&gt;\textrm{d}v'                        &lt;/math&gt;</TD><TD> Integral over state space &lt;math&gt;v'&lt;/math&gt; .
</TD></TR><TR><TD>&lt;math&gt;D                                   &lt;/math&gt;</TD><TD> Dilution rate of the culture (for fermenters).
</TD></TR><TR><TD>&lt;math&gt;\beta                               &lt;/math&gt;</TD><TD> Stochiometric matrix for cellular substances.
</TD></TR><TR><TD>&lt;math&gt;\gamma                              &lt;/math&gt;</TD><TD> Stochiometric matrix for extra-cellular substances.
</TD></TR></TBODY></TABLE><P>
With these relations:</P><TABLE><TBODY><TR><TD>&lt;math&gt;\bar{\dot{\mathbf{V}}}(\mathbf{z,c})&lt;/math&gt;</TD><TD>&lt;math&gt;= \mathbf{\beta} \cdotp \bar{\mathbf{R}}(\mathbf{z,c}) &lt;/math&gt;</TD><TD> The expected internal state change rate vector.
</TD></TR><TR><TD>&lt;math&gt; -\mathbf{\gamma} \cdotp \bar{\mathbf{R}}(\mathbf{z,c}) &lt;/math&gt;</TD><TD> The expected consumation of substances in the environment by a cell in state &lt;math&gt;\mathbf{z}&lt;/math&gt; .
</TD></TR></TBODY></TABLE><P>
Thus for a particular problem in hand it is necessary to chose &lt;math&gt;\mathbf{z}&lt;/math&gt; and &lt;math&gt;\mathbf{c}&lt;/math&gt; that represent the state of cells and the media. Then the matrices and functions &lt;math&gt;\beta&lt;/math&gt;, &lt;math&gt;\gamma&lt;/math&gt;, &lt;math&gt;\bar{\mathbf{R}}(\mathbf{z,c})&lt;/math&gt;, &lt;math&gt;\sigma (\mathbf{z',c})&lt;/math&gt; and &lt;math&gt;p(\mathbf{z,z',c}) &lt;/math&gt; need to be defined for the problem considered. Finally the inital conditions &lt;math&gt;W_{\mathbf{Z}}(\mathbf{z},t)&lt;/math&gt; and &lt;math&gt;\mathbf{c}_0 &lt;/math&gt; and growth conditions &lt;math&gt;D&lt;/math&gt; and &lt;math&gt;\mathbf{c}_f &lt;/math&gt; need to be fixed.
</P><P>For the problem in hand, plasmid maintenance during growth with 2 different plasmids, and attempting to find a simple solution to the problem we propose a 3 variable internal state vector:&lt;math&gt;\mathbf{z} =  \begin{bmatrix} z_0 \\ z_1 \\ z_2 \end{bmatrix} = 
\begin{bmatrix}\textrm{Cell maturity} \\ \textrm{count of plasmid 1} \\ \textrm{count of plasmid 2} \end{bmatrix}&lt;/math&gt;
In this internal state vector: &lt;math&gt;z_0&lt;/math&gt; is a mesure of the growth of the bacteria, encompassing such things as size, number of chromosomes and mass; &lt;math&gt;z_1&lt;/math&gt; and &lt;math&gt;z_2&lt;/math&gt; represent the number of copies of each plasmid. For the external conditions we propose simply the substrate concentration &lt;math&gt;S&lt;/math&gt;. The maturity parameter has a minimum value of 1 and must increase to 2 before division can occur.
</P><P>For the rates of change of the internal state vector then we propose for the bacterial maturity to extend the development presented in Shene <I>et al.</I>  to include 2 plasmids and incorporate the cell maturity as a state variable. This gives: &lt;math&gt;\dot{z}_0 (\mathbf{z},S) = \mu = \mu _{max} \frac{S}{K_S+S} \frac{K_{z_1}}{K_{z_1}+z_1^{m_1}} \frac{K_{z_2}}{K_{z_2}+z_2^{m_2}}&lt;/math&gt; Here &lt;math&gt; \mu _{max}&lt;/math&gt; is the maximum growth rate &lt;math&gt;hr^{-1}&lt;/math&gt;: &lt;math&gt;\mu (\mathbf{z,S})&lt;/math&gt; the growth rate ; &lt;math&gt;K_S&lt;/math&gt; is the Monod constant in &lt;math&gt;g/l&lt;/math&gt; for the substrate; &lt;math&gt;K_{z_1}&lt;/math&gt; is the inhibition constant for plamid number 1 in (plasmids per cell)&lt;math&gt;^{m_1}&lt;/math&gt;, and &lt;math&gt;m_1&lt;/math&gt; the Hill coefficient for the cooperativity of inhibition. &lt;math&gt;K_{z_2}&lt;/math&gt; and &lt;math&gt;m_2&lt;/math&gt; represent the same parameters for plasmid 2.
</P><P>For plasmid replication rate we propose, again following Shene <I>et al.</I> , the empirical relationship : &lt;math&gt;\dot{z}_1 (\mathbf{z},S) = 
</P><PRE> \begin{cases}
  k_1 \frac{\mu (\mathbf{z},S)}{K_1 + \mu (\mathbf{z},S)} ( z_{1_{max}} - z_1 ) &amp; \text{if } z_1 \geq 1.0 \\
  0 &amp; \text{if } 0.0 \leq z_1 &lt; 1.0 \\
 \end{cases}&lt;/math&gt; This relation, and equivalent one for plasmid number 2 &lt;math&gt;\dot{z}_2 (\mathbf{z,S})&lt;/math&gt; is designed to satisfy the boundary conditions of no reproduction if there is less than 1 plasmid in the cell, and a maximum copy number of &lt;math&gt;z_{1_{max}}&lt;/math&gt;. This introduces the parameters &lt;math&gt;k_1&lt;/math&gt; and &lt;math&gt;K_1&lt;/math&gt; which are respectively the plasmid replication rate (in &lt;math&gt;hr^{-1}&lt;/math&gt;) and the inhibition constant (also in &lt;math&gt;hr^{-1}&lt;/math&gt;). The inhibition constant reduces plasmid replication rate at slower growth rates.
</PRE><P>Notice that here we have directly defined &lt;math&gt;\bar{\dot{\mathbf{V}}}(\mathbf{z,c})&lt;/math&gt; rather than &lt;math&gt;\beta &lt;/math&gt; and &lt;math&gt;\bar{\mathbf{R}}(\mathbf{z,c})&lt;/math&gt;. For the growth yield we propose : &lt;math&gt;\gamma \cdotp \bar{\mathbf{R}}(\mathbf{z,c}) = \alpha \mu(\mathbf{z},S)&lt;/math&gt; Where alpha is the growth yield in &lt;math&gt;g/l/cell&lt;/math&gt;. The remaining functions and parameters in equations 1 and 2 are the division rate &lt;math&gt;\sigma (\mathbf{z,c})&lt;/math&gt; and the partitioning function &lt;math&gt;p(\mathbf{z,z',c}) &lt;/math&gt;. There is less consensus in the litterature for an at least empirically appropriate form for these equations. To remain simple we propose: &lt;math&gt;\sigma (\mathbf{z,c}) = \sigma \times H[2.0] =
</P><PRE>\begin{cases}
 0      &amp; \text{if } z_0 &lt; 2.0 \\
 \sigma &amp; \text{if } z_0 \geq 2.0
\end{cases}&lt;/math&gt; Here we assume that there is a fixed rate of division &lt;math&gt;\sigma &lt;/math&gt; once cells are big enough to divide (&lt;math&gt;H[]&lt;/math&gt; is the Heaviside function). &lt;math&gt;p(\mathbf{z,z',c}) = p(z_0,z'_0) \times p(z_1,z'_1) \times p(z_2,z'_2)&lt;/math&gt;&lt;math&gt;p(z_0,z'_0) = \delta_{z_0,\frac{z'_0}{2}} = \begin{cases}
               1 &amp; \text{if } z_0 = z'_0/2.0 \\
               0 &amp; \text{if } z_0 \ne z'_0/2.0.
              \end{cases}&lt;/math&gt;&lt;math&gt;p(z_1,z'_1) = 0.5^{z'_1} \begin{pmatrix} z_1 \\ z'_1 \\ \end{pmatrix} = 0.5^{z'_1} \frac{z'_1 !}{(z'_1-z_1)! z_1 !}&lt;/math&gt; In these equations we assume that the partitioning of the three internal state variables are independant. That cells divide exactly in half, that is the maturity parameter is exactly halved when the cells divide (&lt;math&gt;\delta&lt;/math&gt; is a Kronecker delta function). That the two plasmids segregate independantly and as individual plasmids according to a binomial distribution. These assumptions are probably the most suspect in the model.
</PRE><P>This initial version of the model has no contention, that is &lt;math&gt;z_1&lt;/math&gt; and &lt;math&gt;z_2&lt;/math&gt; do not influence the growth rate &lt;math&gt;\mu &lt;/math&gt;. In order to develop the model for the system envisaged this needs to be introduced.
</P><P><SPAN>cccp<SPAN>0.55</SPAN></SPAN> Symbol &amp; Value &amp; Units &amp; Comments&lt;math&gt;S_f&lt;/math&gt;&amp;10.0&amp;&lt;math&gt;g/l&lt;/math&gt;&amp; Substrate concentration in feed.&lt;math&gt;S_0&lt;/math&gt;&amp;10.0&amp;&lt;math&gt;g/l&lt;/math&gt;&amp; Initial substrate concentration in fermenter.&lt;math&gt;D&lt;/math&gt;&amp;0.5&amp;&lt;math&gt;hr^{-1} &lt;/math&gt;&amp; Dilution rate&lt;math&gt;\alpha &lt;/math&gt;&amp;0.5&amp;&lt;math&gt;g/l/cell&lt;/math&gt;&amp; Growth yield on substrate.&lt;math&gt;\mu _{max} &lt;/math&gt;&amp;2.0&amp;&lt;math&gt;hr^{-1}&lt;/math&gt;&amp; Maximum growth rate on substrate.&lt;math&gt;K_S&lt;/math&gt;&amp;0.25&amp;&lt;math&gt;g/l&lt;/math&gt;&amp; Monod constant for substrate.&lt;math&gt;K_{z_1}&lt;/math&gt;&amp;100.0&amp;&lt;math&gt;(cell^{-m_1})&lt;/math&gt;&amp; Plasmid 1 growth inhibition constant&lt;math&gt;m_1&lt;/math&gt;&amp;1&amp;&amp; Plasmid 1 Hill coefficient&lt;math&gt;k_1&lt;/math&gt;&amp;20&amp;&lt;math&gt;hr^{-1}&lt;/math&gt;&amp; Plasmid 1 replication rate.&lt;math&gt;K_1&lt;/math&gt;&amp;0.1&amp;&lt;math&gt;hr^{-1}&lt;/math&gt;&amp; Plasmid 1 replication inhibition constant.&lt;math&gt;z_{1_{max}} &lt;/math&gt;&amp;10&amp;&amp; Plasmid 1 maximum copy number.&lt;math&gt;K_{z_2}&lt;/math&gt;&amp;100.0&amp;&lt;math&gt;(cell^{-m_1})&lt;/math&gt;&amp; Plasmid 2 growth inhibition constant&lt;math&gt;m_2&lt;/math&gt;&amp;1&amp;&amp; Plasmid 2 Hill coefficient&lt;math&gt;k_2&lt;/math&gt;&amp;20&amp;&lt;math&gt;hr^{-1}&lt;/math&gt;&amp; Plasmid 2 replication rate.&lt;math&gt;K_2&lt;/math&gt;&amp;0.1&amp;&lt;math&gt;hr^{-1}&lt;/math&gt;&amp; Plasmid 2 replication inhibition constant.&lt;math&gt;z_{2_{max}} &lt;/math&gt;&amp;10&amp;&amp; Plasmid 2 maximum copy number.&lt;math&gt;\sigma&lt;/math&gt;&amp;5.0&amp;&lt;math&gt;hr^{-1}&lt;/math&gt;&amp; Rate constant for cell division once cells are big.&lt;math&gt;k_{a1} &lt;/math&gt;&amp;2.0&amp;&amp; Toxin 1 efficiency parameter&lt;math&gt;k_{a2} &lt;/math&gt;&amp;0.5&amp;&amp; Toxin 2 efficiency parameter&lt;math&gt;k_b&lt;/math&gt;&amp;2.0&amp;&amp; Ratio of anti-toxin to toxin production rates</P><P>
Substituting into the equations 1 and 2 we obtain: &lt;math&gt;\begin{gathered}
</P><PRE> \frac{\partial}{\partial t} W_{\mathbf{Z}}(\mathbf{z},t) 
+ [\nabla_{\mathbf{Z}} \cdotp \bar{\dot{\mathbf{Z}}}(\mathbf{z},S)] W_{\mathbf{Z}}(\mathbf{z},t)
+ \sum_i \bar{\dot{z_i}} \times \frac{\partial}{\partial z_i} W_{\mathbf{Z}}(\mathbf{z},t) \\
= 2 \sigma \int_{z'_0&gt;2.0} \delta_{z_0,\frac{z'_0}{2}} \times 0.5^{z'_1+z'_2} \times \begin{pmatrix} z_1 \\ z'_1 \\ \end{pmatrix} 
\times \begin{pmatrix} z_2 \\ z'_2 \\ \end{pmatrix} \times
W_{\mathbf{Z}}(\mathbf{z'},t)\mathrm{d}v' \\
- (D+\sigma H[2.0] W_{\mathbf{Z}}(\mathbf{z},t))\end{gathered}&lt;/math&gt;&lt;math&gt;\frac{d\mathbf{c}}{dt} 
= D(c_f - c ) - \alpha \int  \mu (\mathbf{z},S) W_{\mathbf{Z}}(\mathbf{z},t)\mathrm{d}v&lt;/math&gt;</PRE><H1><SPAN class="mw-headline" id="Objectives">Objectives</SPAN></H1><P>The aim of studying the behaviour of this model is to investigate how growth conditions &lt;math&gt;D, S_f&lt;/math&gt; and time modulate the development of the population in state space &lt;math&gt;W_{\mathbf{Z}}(\mathbf{z},t)&lt;/math&gt;. In particular we are interested in finding how the number of bacteria without plasmids &lt;math&gt;W_{\mathbf{Z}}([z_0,0,0],t)&lt;/math&gt; in the culture progresses, and how this depends on the various parameters in the model.
</P><H1><SPAN class="mw-headline" id="Proposition:_Adding_contention">Proposition: Adding contention</SPAN></H1><P>We need to introduce a modification to equation 3 (the state dependant growth rate) in which the equilibrium between &lt;math&gt;z_1&lt;/math&gt; and &lt;math&gt;z_2&lt;/math&gt; features. This modification is to take into account that each toxin should be inhibited by anti-toxin for maximal growth. A possibility is that we consider that for each toxin anti-toxin pair:
</P><UL><LI> the genes produce toxin at a constant rate dependant on the number of copies &lt;math&gt;k_1\times z_1&lt;/math&gt;,</LI><LI> the anti-toxin genes produce anti-toxin at a rate dependant on the number of copies &lt;math&gt;k_2\times z_2&lt;/math&gt;,</LI><LI> anti-toxin instantly and irreversible kills the toxin in a stochiometric manner.</LI><LI> undestroyed toxin disappears at a constant rate, by dilution and other pathways &lt;math&gt;k_3&lt;/math&gt;,</LI><LI> the concentration of toxin is at a dynamic steady state, i.e. the rates of production and disappearance are equal.</LI><LI> growth in inhibited in an exponential manner by free toxin with a characteristic &lt;math&gt;IC_{50}&lt;/math&gt;.</LI></UL><P>This model of toxin anti-toxin interaction takes into account the bacteriostatic nature of most such toxins, and the presence of measurable &lt;math&gt;IC_{50}&lt;/math&gt; values. Clearly a more realsitic model would need to take into account cell volume, protein synthesis rates etc. Nevertheless, this simple model gives:&lt;math&gt;\begin{aligned}
</P><PRE>\mu &amp;= \mu_{0} \times e^{-\frac{[Toxin]}{IC_{50}}} \\
[Toxin] &amp;= max(0,\frac{k_1z_1-k_2z_2}{k_3}) \\
\mu &amp;= \mu_{0} \times min(1.0,e^{-k_a(z_1-k_bz_2)}) \\
k_a &amp;= \frac{k_1}{k_3 \times IC_{50}} \\
k_b &amp;= \frac{k_2}{k_1}
\end{aligned}&lt;/math&gt; Incorporating 2 toxin anti toxin pairs, if we assume that the production rate ratio is the same for both, &lt;math&gt;k_b&lt;/math&gt;, is independant of the system we have: &lt;math&gt;\mu = \mu_{0} \times min(1,e^{-k_a(z_1-k_bz_2)})) \times min(1,e^{-k_a(z_2-k_bz_1)}))&lt;/math&gt; where &lt;math&gt;\mu_0&lt;/math&gt; is given by equation 3. For each toxin the efficiency parameter is a measure of ratio of toxin accumulation in cells with one gene copy and without anti-toxin to the &lt;math&gt;IC_{50}&lt;/math&gt;.
</PRE><H1><SPAN class="mw-headline" id="Integration">Integration</SPAN></H1><P>The equations are hard to integrate and with partial derivatives did not seem easy to tackle with standard packages. We therefor wrote a simple custom integrator that calculates the growth trajectory of about &lt;math&gt;10^6&lt;/math&gt; bacteria in a fermenter at constant dilution rate. This essentially implements a Monte-carlo Markov-chain integrator.
</P><P>The first version of this program is written in Python, and is shown and described here.
</P><P>The first lines of the program load various modules that are used, these are: “math”, required for the &lt;math&gt;exp()&lt;/math&gt; function; “copy”, for duplicating a list of individuals with the &lt;math&gt;deepcopy()&lt;/math&gt; function, and the numerical package “numpy”, for the random number generator.
</P><P>A simple function “michaelis” is then defined. As several of the model equations contain elements in the form of the Michaelis Menten equation, for example equations 3 and 4, this makes the code easier to read.
</P><P>The program then initializes the various model parameters. These include the parameters in table [tbl:parameters], and a few other parameters specific to the integrator. These extra parameters are: &lt;math&gt;V&lt;/math&gt;, the observation volume in ml; &lt;math&gt;V_{tot}&lt;/math&gt;, the total fermenter volume (taken as 100 ml); &lt;math&gt;total&lt;/math&gt;, the total number of bacteria in the integration volume &lt;math&gt;V&lt;/math&gt;; &lt;math&gt;t_{max}&lt;/math&gt;, the total time to run the integrator for, in hours; &lt;math&gt;dt&lt;/math&gt;, the time step, in hours. In addition the dilution parameter, &lt;math&gt;D&lt;/math&gt;, is changed from a simple first order rate constant to be a flow rate in ml/hr. This can be divided by &lt;math&gt;V_{tot}&lt;/math&gt; to obtain the first order rate constant. In this section the time counter and the random number generator are both initialized.
</P><P>An empty list of bacteria, “population” is created. Various integration parameters are zeroed. The next lines create the initial population of cells, assigning a random growth state between 1 and 2 to each cell, and giving the maximum number of plasmids. It was decided to give each cell the maximum number of plasmids, this was done to avoid the rapid creation of a significant population of plasmid free cells, due to the division of cells with few plasmids early in the simulation, when this quantity was randomized.
</P><P>In the next lines of the program we enter the integration loop which runs until the timer expires. The first operation is output the current state of the population, line 92. Then the various counters that keep track of the population status are re-initialized ready for the next time-step. At line 103 the program enters the loop to update each bacterium in the population. This loop runs backwards (from the end to the begining of the list), so we can delete the current bacterium if necessary. we then collect a random number to determine the fate of the current bacterium. If this number is less than the probability of a bacterium being lost from the simulated volume &lt;math&gt;V&lt;/math&gt; in a timestep by washout then the current bacterium is destroyed. The random value is reduced by the probability of washout so it can be used for further testing.
</P><P>If the bacterium survives the washout then it grows. Here the values of &lt;math&gt;\dot z_0&lt;/math&gt;, &lt;math&gt;\dot z_1&lt;/math&gt; and &lt;math&gt;\dot z_2&lt;/math&gt; are calculated using equations 3, 4 and 17. Then the state vector of the current bacterium is updated. In line 135 the total growth of bacteria during the time-step is summed, this is used to calculate the substrate use later, line 147.
</P><P>After the cells, that avoided being washed out have grown, those that are large enough to divide (&lt;math&gt;z_0 &gt; 2.0&lt;/math&gt;) are given a chance to divide, depending on the random control parameter “cntrl”. If the cell is selected for division a new bacterium is created and allocated about half of the resources. The other half of the resources remain in the original bacterium. The allocation of the plasmids (&lt;math&gt;z_1&lt;/math&gt; and &lt;math&gt;z_2&lt;/math&gt;) follow a binomial distribution, equation 9. The division of the remaining cell contents, was adjusted to follow a normal distribution, with mean 0.5 the parents content and standard deviation 0.05. This was implemented rather than the simple division proposed in equation 8 and 10 to avoid persistent synchrony and oscillations in cell number during slow growth.
</P><P>Having updated all the cells, including additions and subtractions due to cell division and washout, the new total number of cells is obtained and the substrate concentration updated according to equation 11.
</P><P>In the last lines of the program the total cell number is controlled to avoid it becoming either too large or too small, this is achieved my modifying the simulated volume.
</P><P>This Python program handles the integration as the growth of a population in fermenter like conditions, essentially following the equations set out above.
</P><H1><SPAN class="mw-headline" id="Results">Results</SPAN></H1><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><A href="/File:Contour_mu.eps" title="File:Contour mu.eps">File:Contour mu.eps</A><DIV class="thumbcaption">caption Contour plot of growth rate dependence on plasmid number, as predicted by equation 17.</DIV></DIV></DIV><P>Equation 17 describes how growth rate depends on the number of each type of plasmid, and 5 of the parameters in table [tbl:parameters] namely &lt;math&gt;K_{z1}&lt;/math&gt;, &lt;math&gt;K_{z2}&lt;/math&gt;, &lt;math&gt;k_{a1}&lt;/math&gt;, &lt;math&gt;k_{a2}&lt;/math&gt;, and &lt;math&gt;k_{b}&lt;/math&gt;. The graph of this function growth rate (&lt;math&gt;\mu &lt;/math&gt; or &lt;math&gt;\dot Z_0&lt;/math&gt;), as a function of the number of each type of plasmid (&lt;math&gt;Z_1&lt;/math&gt; and &lt;math&gt;Z_2&lt;/math&gt;), is shown in figure [fig:contour] as a contour plot. This shows a central flat surface sloping away from the no plasmid situation, that is cut away close to the axes where the amount of anti-toxin is no longer enough to prevent toxicity. In the parameter space the slope of the central surface is determined by &lt;math&gt;K_{z1}&lt;/math&gt; and &lt;math&gt;K_{z2}&lt;/math&gt;, the more expensive plasmid maintenance is the faster the fall off. The width of the central wedge is determined by the parameter &lt;math&gt;k_b&lt;/math&gt; larger values give a wider wedge, increased anti-toxin production reduces toxicity due to plasmid imbalance. The fall off in growth rate outside this central region depends on the toxicity parameters &lt;math&gt;k_{a1}&lt;/math&gt; and &lt;math&gt;k_{a2}&lt;/math&gt;.
</P><P>We can this note that the best growth rate with plasmids is with one of each, and no plasmids is always best. This is because in our model &lt;math&gt;K_{z1}&lt;/math&gt; and &lt;math&gt;K_{z2}&lt;/math&gt; are positive. As mutations, which are not included in the model, will tend to increase growth rate they will tend to increase &lt;math&gt;k_b&lt;/math&gt; and decrease the other parameters. To minimize the risk of plasmid loss the parameter &lt;math&gt;k_b&lt;/math&gt; would seem particularly critical. If the wedge is too narrow all plasmids become toxic, if it is too wide growth with few of one plasmid becomes possible because of insufficient toxicity.
</P><P>As the model for plasmid segregation is a binomial distribution, with probability of 1/2 for each daughter cell, the probability of generating a daughter with no plasmids in a division is &lt;math&gt;2 \times \frac{1}{2^n}&lt;/math&gt;, where n is the total number of plasmids to segregate. The prefactor of 2 is as each division gives 2 daughter cells. Thus with 2 plasmids with a copy number of 50 the probability of generating a plasmid free daughter is &lt;math&gt;2^{-99}&lt;/math&gt; or about &lt;math&gt;10^{-30}&lt;/math&gt;. As a 1.0 &lt;math&gt;OD_{600}&lt;/math&gt; culture corresponds to about &lt;math&gt;10^8&lt;/math&gt; cells/ml in 100ml of culture growing at constant volume at OD=1.0 and a doubling rate of 2 &lt;math&gt;hr^{-1}&lt;/math&gt; the probability of this arriving is about 1 in &lt;math&gt;10^{20}&lt;/math&gt; per division, or once in &lt;math&gt;5 \times 10^{19}&lt;/math&gt; hours.
</P><P>However the rate is likely to be increased as not all cells will have 100 plasmids when they divide, and the probability is very sensitive to this. With only 20 of each plasmid/cell the frequency shoots up, under the same growth conditions, to 1% of the times the &lt;math&gt;10^{10}&lt;/math&gt; cells divide, ie on average once in a hundred generations or every 2 days.
</P><P>Therefor critical to the maintenance of the plasmids is the number of plasmid copies in dividing cells. This needs to be evaluated by integration.
</P><P><SPAN>0.8</SPAN><SPAN>CCCC</SPAN> D(ml/hr)&amp;S(g/l)&amp;Cell density(&lt;math&gt;OD_{600}&lt;/math&gt;)&amp;Average number of plasmids
5 &amp; 0.0030 &amp; 2.05 &amp; 19.87
10 &amp; 0.0063 &amp; 2.02 &amp; 19.79
20 &amp; 0.0138 &amp; 1.99 &amp; 19.60
50 &amp; 0.0473 &amp; 1.89 &amp; 18.99
100 &amp;&amp;&amp;
200 &amp;&amp;&amp;
250 &amp; 10.00 &amp; 0.00 &amp; n.a.</P><P>
Simulations were run using the model to see how the cell density and substrate concentration adjusted to the dilution rate. As the number of plasmids in cells is an important parameter this was also calculated. These results, shown in table [tbl:density], demonstrate how the equilibrium cell density decreases as dilution rate increases, while the equilibrium substrate concentration increases. The average number of plasmids per cell also decreases as growth rate increases. In these simulations, as the maximal number of each type of plasmid was 10, see table [tbl:parameters], plasmid loss was a frequent occurence. As can be seen once the flow rate gets too great the above about 250 ml/hr the bacteria are washed out and a stable culture is not formed.
</P><P>TODO: next experiments number of plasmids at division for 100 and 200 with max = 20 (graph) p(n) vs n for 2 conditions.
</P>
TODO: then calculate prob child with 0 as &lt;math&gt;\int P(n) 0.5^(n-1) dn&lt;/math&gt; for a dividing bacterium.</DIV></DIV></DIV></BODY></HTML>