<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aix-Marseille_Experiments_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aix-Marseille/Experiments/Protocols</SPAN></H1></DIV></DIV></DIV><DIV id="amu-content"><NAV class="navbar navbar-fixed-top"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle">
Toggle navigation ▤
</BUTTON><A class="navbar-brand" href="/Team:Aix-Marseille">iGEM Aix-Marseille Université</A></DIV><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav"><LI class="menu_item" id="menu-team"><A href="#menu-team">Team</A></LI><LI><A href="/Team:Aix-Marseille/Team"> Team   </A></LI><LI><A href="/Team:Aix-Marseille/Collaborations">★  Collaborations </A></LI></UL><LI class="menu_item" id="menu-project"><A href="#menu-project">Project</A></LI><LI><A href="/Team:Aix-Marseille/Description"> ★  Description </A></LI><LI><A href="/Team:Aix-Marseille/Design"> ★ Design </A></LI><LI><A href="/Team:Aix-Marseille/Experiments"> Experiments </A></LI><LI><A href="/Team:Aix-Marseille/Results"> Results </A></LI><LI><A href="/Team:Aix-Marseille/Notebook"> Notebook </A></LI><LI class="menu_item" id="menu-parts"><A href="#menu-parts"> Parts</A></LI><LI><A href="/Team:Aix-Marseille/Parts">Parts </A></LI><LI><A href="/Team:Aix-Marseille/Basic_Part"> ★ Basic Parts </A></LI><LI><A href="/Team:Aix-Marseille/Composite_Part"> ★ Composite Parts </A></LI><LI class="menu_item"><A href="/Team:Aix-Marseille/Safety"> Safety </A></LI><LI class="menu_item"><A href="/Team:Aix-Marseille/Medals"> Medals</A></LI><LI class="menu_item"><A href="/Team:Aix-Marseille/Attributions">★  Attributions </A></LI><LI class="menu_item" id="menu-hp"><A href="#menu-hp">Human Practices</A></LI><LI><A href="/Team:Aix-Marseille/Human_Practices"> Human Practices </A></LI><LI><A href="/Team:Aix-Marseille/Integrated_Practices"> ★ Integrated Practices </A></LI><LI><A href="/Team:Aix-Marseille/Engagement">★ Engagement </A></LI><LI class="menu_item"><A href="/Team:Aix-Marseille/Model"> Model </A></LI><DIV class="social"><UL><LI><A href="https://www.facebook.com/iGEM.AMU">
                            facebook
                        </A></LI><LI><A href="https://twitter.com/iGEMteamAMU">
                            twitter
                        </A></LI></UL></DIV></DIV></NAV><DIV class="content_wrapper"><DIV class="tocp"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Protocol_.230_:_iGEM_General_Protocol"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Protocol #0 : iGEM General Protocol</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Protocol_.231_:_Preparation_of_competent_bacteria_cells"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Protocol #1 : Preparation of competent bacteria cells</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Protocol_.231bis_:_Preparation_of_Tbf1_and_Tbf2_buffers"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Protocol #1bis : Preparation of Tbf1 and Tbf2 buffers</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Preparation_of_50_mL_of_Tbf1_Buffer"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Preparation of 50 mL of Tbf1 Buffer</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Preparation_of_8_mL_of_Tbf2_Buffer"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Preparation of 8 mL of Tbf2 Buffer</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-6"><A href="#Protocol_.232_:_Transformations"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Protocol #2 : Transformations</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Plasmids_transformation"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Plasmids transformation</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Ligation_transformation"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Ligation transformation</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Protocol_.233_:_PCR_and_electrophoresis"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Protocol #3 : PCR and electrophoresis</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#PCR"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">PCR</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Electrophoresis"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Electrophoresis</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Protocol_.234_:_Plasmid_DNA_purification"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Protocol #4 : Plasmid DNA purification</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Protocol_.235_:_Cloning_protocol_for_IDT_sequences"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Protocol #5 : Cloning protocol for IDT sequences</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Resuspension"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Resuspension</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Digestion_E.2FP"><SPAN class="tocnumber">7.2</SPAN><SPAN class="toctext">Digestion E/P</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Ligation"><SPAN class="tocnumber">7.3</SPAN><SPAN class="toctext">Ligation</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Transformation"><SPAN class="tocnumber">7.4</SPAN><SPAN class="toctext">Transformation</SPAN></A></LI><LI class="toclevel-1 tocsection-18"><A href="#Protocol_.236_:_Digestion_and_ligation_for_verification_and_BioBrick_Assembly"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Protocol #6 : Digestion and ligation for verification and BioBrick Assembly</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#Digestion_.28verification.29_protocol"><SPAN class="tocnumber">8.1</SPAN><SPAN class="toctext">Digestion (verification) protocol</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Digestion_protocol_BioBrick_Assembly_Kit"><SPAN class="tocnumber">8.2</SPAN><SPAN class="toctext">Digestion protocol BioBrick Assembly Kit</SPAN></A></LI><LI class="toclevel-3 tocsection-21"><A href="#Upstream_part_:"><SPAN class="tocnumber">8.2.1</SPAN><SPAN class="toctext">Upstream part :</SPAN></A></LI><LI class="toclevel-3 tocsection-22"><A href="#Downstream_part.2C_optional.2C_for_3A_strategy"><SPAN class="tocnumber">8.2.2</SPAN><SPAN class="toctext">Downstream part, optional, for 3A strategy</SPAN></A></LI><LI class="toclevel-3 tocsection-23"><A href="#Destination_plasmid_:"><SPAN class="tocnumber">8.2.3</SPAN><SPAN class="toctext">Destination plasmid :</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#Ligation_protocol_BioBrick_Assembly"><SPAN class="tocnumber">8.3</SPAN><SPAN class="toctext">Ligation protocol BioBrick Assembly</SPAN></A></LI><LI class="toclevel-1 tocsection-25"><A href="#Protocol_.237_:_PCR_clean-up_and_gel_extraction"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Protocol #7 : PCR clean-up and gel extraction</SPAN></A></LI><LI class="toclevel-1 tocsection-26"><A href="#Protocol_.238_:_Generalised_transduction_using_phage_P1"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Protocol #8 : Generalised transduction using phage P1</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#Step1_:_P1_lysat_on_the_given_strain"><SPAN class="tocnumber">10.1</SPAN><SPAN class="toctext">Step1 : P1 lysat on the given strain</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#Step2_:_Transduction_P1"><SPAN class="tocnumber">10.2</SPAN><SPAN class="toctext">Step2 : Transduction P1</SPAN></A></LI><LI class="toclevel-2 tocsection-29"><A href="#Step3_:_Clones_isolation"><SPAN class="tocnumber">10.3</SPAN><SPAN class="toctext">Step3 : Clones isolation</SPAN></A></LI><LI class="toclevel-2 tocsection-30"><A href="#Step_4_:_PCR_screen_and_transplantation"><SPAN class="tocnumber">10.4</SPAN><SPAN class="toctext">Step 4 : PCR screen and transplantation</SPAN></A></LI><LI class="toclevel-1 tocsection-31"><A href="#Protocol_.239_:_Cadaverin_HPLC_analysis"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">Protocol #9 : Cadaverin HPLC analysis</SPAN></A></LI><LI class="toclevel-2 tocsection-32"><A href="#Reaction_of_lysine_and_cadaverine_produced_by_biotransformation_with_whole_cells"><SPAN class="tocnumber">11.1</SPAN><SPAN class="toctext">Reaction of lysine and cadaverine produced by biotransformation with whole cells</SPAN></A></LI><LI class="toclevel-2 tocsection-33"><A href="#Derivatization_reaction"><SPAN class="tocnumber">11.2</SPAN><SPAN class="toctext">Derivatization reaction</SPAN></A></LI><LI class="toclevel-2 tocsection-34"><A href="#HPLC_analysis"><SPAN class="tocnumber">11.3</SPAN><SPAN class="toctext">HPLC analysis</SPAN></A></LI><LI class="toclevel-1 tocsection-35"><A href="#Protocol_.2310_:_SLIC_.28sequence-_and_ligation-independent_cloning.29"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Protocol #10 : SLIC (sequence- and ligation-independent cloning)</SPAN></A></LI><LI class="toclevel-2 tocsection-36"><A href="#Procedures"><SPAN class="tocnumber">12.1</SPAN><SPAN class="toctext">Procedures</SPAN></A></LI><LI class="toclevel-1 tocsection-37"><A href="#Protocol_.2311_:_Swimming_test"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">Protocol #11 : Swimming test</SPAN></A></LI><LI class="toclevel-1 tocsection-38"><A href="#Protocol_.2312_:_SDS_page_and_coomassie_blue"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">Protocol #12 : SDS page and coomassie blue</SPAN></A></LI><LI class="toclevel-2 tocsection-39"><A href="#Culture"><SPAN class="tocnumber">14.1</SPAN><SPAN class="toctext">Culture</SPAN></A></LI><LI class="toclevel-2 tocsection-40"><A href="#Staining_with_Coomassie_Blue_R250"><SPAN class="tocnumber">14.2</SPAN><SPAN class="toctext">Staining with Coomassie Blue R250</SPAN></A></LI><LI class="toclevel-2 tocsection-41"><A href="#Staining_with_Colloidal_Coomassie_Blue_Staining_Kit_.28Invitrogen_LC6025.29"><SPAN class="tocnumber">14.3</SPAN><SPAN class="toctext">Staining with Colloidal Coomassie Blue Staining Kit (Invitrogen LC6025)</SPAN></A></LI><LI class="toclevel-1 tocsection-42"><A href="#SLIC_Oligo_table"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">SLIC Oligo table</SPAN></A></LI></DIV></DIV><DIV class="page_content"><H1>Protocols</H1><H2><SPAN class="mw-headline" id="Protocol_.230_:_iGEM_General_Protocol">Protocol #0 : iGEM General Protocol</SPAN></H2><OL><LI>Obtaining Biobricks
</LI><LI>EITHER on IDT → 
</LI><LI>Order on IDT when the sequences are not too long (chemical synthesis).</LI><LI>The received biobricks are lyophilized, they must be resuspended in water (cf protocole).</LI><LI>OR by PCR → 
</LI><LI>When sequences are too long and costly to order on IDT, or when a sequence must be added to an existing biobrick : 
</LI><LI>Order primers</LI><LI>synthesize sequence by PCR (cf protocole Q5)</LI><LI>add the suffixes and prefixes during the synthesis « E, X » and « S, P » (E=EcoRI, X=XbaI, S=SpeI, P=PstI) and homologue sequences to plasmids</LI><LI>Do a SLIC (cf protocole) to insert the wanted sequence in the wanted plasmid</LI><LI>the Biobrick is obtained</LI><LI>Transform the Bacteria with the Biobrick 
</LI><LI>E/S Digestion of the Biobrick from the original plasmid and reinsertion in the E/S pre-restricted wanted plasmid.</LI><LI>Transform the pre-cultivated competent cells, via a thermal shock (DH5-α, TGI, BL21).</LI><LI>Add  an LB-agar in a petri dish and cultivate the bacteria in rich medium added with antibiotics of interest.</LI></OL><LI>Test the Transformation 
</LI><LI>Run the obtained bacteria culture DNA in a PCR and  an electrophoresis gel (qualitative assessment)</LI><LI>Purify plasmids (by miniprep Kit) from the previously obtained positive colonies.</LI><LI>Do an E/P digestion verification to see if the inserted biobricks have the correct size (On only 100ng of plasmids) </LI><LI>Assemble two biobricks 
</LI><LI>“Digestion protocol BioBrick Assembly Kit” et “Ligation protocol BioBrick Assembly”</LI><OL><LI>Do an E/S digestion on the first biobrick and an X/P on the second</LI><LI>Ligate the two biobricks then insert them into the plasmid</LI></OL><LI>Transforming Bacteria with a new BioBrick
</LI><LI>E/P Restriction of the biobrick and ligation into a E/P restricted plasmid</LI><LI>Transform the pre-cultivated competent cells, via a thermal shock (DH5-α, TGI ou BL21).</LI><LI>Pour an LB gel in a petri dish and cultivate the bacteria in an antibiotic rich medium</LI><LI>Test the Transformation
</LI><LI>Run the obtained bacteria culture DNA in a PCR and pour an electrophoresis gel (qualitative assessment)</LI><LI>Purify plasmids (by miniprep Kit) from the previously obtained positive colonies.</LI><LI>Do an E/P digestion verification to see if the inserted biobricks have the correct size (On only 100ng of plasmids) </LI><LI>Send to sequencing</LI><LI>Repeat steps 4, 5, 6 until obtaining the wanted sequence</LI><H2><SPAN class="mw-headline" id="Protocol_.231_:_Preparation_of_competent_bacteria_cells">Protocol #1 : Preparation of competent bacteria cells</SPAN></H2><P><I>Note : Everything should be done in sterile conditions and on ice (4°C).</I><I>For 100ml of competent cells</I></P><OL><LI>Grow a culture of bacteria in LB medium until it reaches OD600 = 0.5.
</LI><LI>While it grows, prepare Tbf1 and Tbf2 buffers.</LI><LI>Centrifuge cells for 10 minutes at 3500g at 4°C</LI><LI>Resuspend the pellet slowly in 50mL of Tfb1 buffer.</LI><LI>Centrifuge 5 min at 3500g at 4°C.</LI><LI>Resuspend the pellet in 8 mL of Tbf2 buffer.</LI><LI>Incubate 15 min in ice.</LI><LI>Aliquot 200µL of cell suspension in sterile eppendorf tubes and conserve it at -80°C.</LI></OL><H2><SPAN class="mw-headline" id="Protocol_.231bis_:_Preparation_of_Tbf1_and_Tbf2_buffers">Protocol #1bis : Preparation of Tbf1 and Tbf2 buffers</SPAN></H2><P>For 100 mL of culture:
</P><H3><SPAN class="mw-headline" id="Preparation_of_50_mL_of_Tbf1_Buffer">Preparation of 50 mL of Tbf1 Buffer</SPAN></H3><TABLE class="wikitable"><TBODY><TR><TD>KAc 1M
</TD><TD>1.5 mL
</TD></TR><TR><TD>MnCl2 0.5M
</TD><TD>5 mL
</TD></TR><TR><TD>KCl 1 M
</TD><TD>5 mL
</TD></TR><TR><TD>CaCl2 0.1M
</TD><TD>5 mL
</TD></TR><TR><TD>Gly 80%
</TD><TD>0.93 mL
</TD></TR><TR><TD>H2O
</TD><TD>32.56 mL
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Preparation_of_8_mL_of_Tbf2_Buffer">Preparation of 8 mL of Tbf2 Buffer</SPAN></H3><TABLE class="wikitable"><TBODY><TR><TD>NaMOPS 0.2M
</TD><TD>400 µL
</TD></TR><TR><TD>CaCl2 0.1M
</TD><TD>6 mL
</TD></TR><TR><TD>KCl 1 M
</TD><TD>8 mL
</TD></TR><TR><TD>Gly 80%
</TD><TD>1.5 mL
</TD></TR><TR><TD>KCl 1M
</TD><TD>80 µL
</TD></TR><TR><TD>H2O
</TD><TD>500 µL
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Protocol_.232_:_Transformations">Protocol #2 : Transformations</SPAN></H2><H3><SPAN class="mw-headline" id="Plasmids_transformation">Plasmids transformation</SPAN></H3><OL><LI>Add 20 ng of plasmid to 100 μL of competent cells thawed in ice</LI><LI>Incubate 30-45 min in ice</LI><LI>Thermal shock : put tubes in the Thermomixer at 42°C for 2 min</LI><LI>Incubate 5 min in ice</LI><LI>Add 900 μL {of LB</LI><LI>Incubate 1 hour at 37°C with agitation</LI><LI>Spread 100 μL on LB limp (with antibiotic)</LI></OL><H3><SPAN class="mw-headline" id="Ligation_transformation">Ligation transformation</SPAN></H3><OL><LI>Add 20 ng of ligation product to 100 μL of competent cells thawed in ice</LI><LI>Incubate 30-45 min in ice</LI><LI>Thermal shock : put tubes in the Thermomixer at 42°C during 2 min</LI><LI>Incubate 5 min in ice</LI><LI>Add 900 μL of LB</LI><LI>Incubate 1 hour at 37°C with agitation</LI><LI>Centrifuge 5 min at 5000 rpm</LI><LI>Eliminate 850 μL of medium</LI><LI>Spread 150 μL on LB limp (with antibiotic)</LI></OL><P>To make negative control, follow the same procedure but without adding plasmids and spreading 300 μL
</P><H2><SPAN class="mw-headline" id="Protocol_.233_:_PCR_and_electrophoresis">Protocol #3 : PCR and electrophoresis</SPAN></H2><H3><SPAN class="mw-headline" id="PCR">PCR</SPAN></H3><TABLE class="wikitable"><TBODY><TR><TD> 25µL reaction
</TD><TD> 50µL reaction
</TD><TD> 100µL reaction
</TD><TD> Finalconcentration
</TD></TR><TR><TD> EconoTaq PLUS2X Master Mix
</TD><TD> 12.5µL
</TD><TD> 25µL
</TD><TD> 50µL
</TD><TD> 1X
</TD></TR><TR><TD>Forward Primer(100 pmol/μL)
</TD><TD> 0.5µL
</TD><TD> 0.5µL
</TD><TD> 1µL
</TD><TD> 1 pmol/μL (1 μM)
</TD></TR><TR><TD>Reverse Primer(100 pmol/μL)
</TD><TD> 0.5µL
</TD><TD> 0.5µL
</TD><TD> 1µL
</TD><TD> 1 pmol/μL (1 μM)
</TD></TR><TR><TD>DNA Template*
</TD><TD> 0.5µL
</TD><TD> 1µL
</TD><TD> 1µL
</TD><TD> ?
</TD></TR><TR><TD>Waternuclease-free
</TD><TD> [ ] μL qsp 25μL
</TD><TD> [ ] μL qsp 50µL
</TD><TD> [ ] μL qsp 100μL
</TD></TR></TBODY></TABLE><UL><LI> *One colony in 50 μL of Water. 15-20 min at 80°C. 5 min at 96°C.</LI></UL><P>Cycling steps
</P><TABLE class="wikitable"><TBODY><TR><TD> Cycling step
</TD><TD> Temperature
</TD><TD> Time
</TD><TD> Number of cycles
</TD></TR><TR><TD> Initial denaturation
</TD><TD> 98°C
</TD><TD> 3 min
</TD><TD> 1
</TD></TR><TR><TD> Denaturation
</TD><TD> 95°C
</TD><TD> 10 sec
</TD><TD rowspan="3">X27
</TD></TR><TR><TD> Annealing
</TD><TD> 50-65°C
</TD><TD> 40 sec
</TD></TR><TR><TD> Extension
</TD><TD> 72°C
</TD><TD> 1 min/kb
</TD></TR><TR><TD> Final extension
</TD><TD> 72°C
</TD><TD> 1 min/kb
</TD><TD> 1
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Electrophoresis">Electrophoresis</SPAN></H3><P>Preparation of 0.8% agarose gel
</P><UL><LI>Put 0.8g of agarose powder into 100mL of TAE 1X.</LI><LI>Melt in microwave and sitr occasionnally.</LI><LI>Let cool until you can take it into hands.</LI><LI>Flow the support with the gel.</LI><LI>Wait until it's solid.</LI></UL><P>Sample preparation and migration
</P><UL><LI>Put the gel in the migration tank filled with 0.5X TAE.</LI><LI>Mix the samples with 1X marker***.</LI><LI>Migrate 30 min at 100V.</LI></UL><P>Revelation
</P><UL><LI>Put the gel under UV.</LI></UL><H2><SPAN class="mw-headline" id="Protocol_.234_:_Plasmid_DNA_purification">Protocol #4 : Plasmid DNA purification</SPAN></H2><P>Protocol used for a Macherey-Nagel purification kit, for more details see <A rel="nofollow" class="external text" href="http://www.mn-net.com/Portals/8/attachments/Redakteure_Bio/Protocols/Plasmid%20DNA%20Purification/UM_pDNA_NS.pdf">here</A>.
For better results use gloves and if so do not use gloves.
</P><TABLE class="wikitable"><TBODY><TR><TD><B>Step</B></TD><TD><B>Actions</B></TD></TR><TR><TD>Cultivate and harvest cells
</TD><TD>Centrifuge at 11,000g for 30 sec
</TD></TR><TR><TD>Cell lysis
</TD><TD>Add 250µL of buffer A1  Add 250µL of buffer A2  Incubate at room temperature up to 5 min  300µL buffer A3
</TD></TR><TR><TD>Clarification of the lysate
</TD><TD>Centrifuge at 11,000g for 5 min
</TD></TR><TR><TD>Bind DNA
</TD><TD>Place a purification column in a collection tube (2mL) and  carefully decant supernatant from the previous step, to a maximum of 750µL  Then centrifuge at 11,000g for 1 min and discard the flow-through.  Put the column back in the collection tube  Repeat this step to load the rest of the supernatant if need be
</TD></TR><TR><TD>Wash silica membrane
</TD><TD>Add 600µl of buffer A4  Centrifuge at 11,000g for 1 min
</TD></TR><TR><TD>Dry silica membrane
</TD><TD>Centrifuge at 11,000g for 2 min
</TD></TR><TR><TD>Elute DNA
</TD><TD>Place the columns in an 1.5mL eppendorf and add 50 µL of buffer AE and incubate at room temperature for 1 min  Centrifuge at 11,000g for 1 min
</TD></TR></TBODY></TABLE><P>To measure your DNA concentration, use a Biospec Nano apparel or Nanodrop apparel. 
Make the blank with  1µL of pure water and use 1µl of sample. 
Make sure to clear the lens with water and with a special paper  in order to no line the lens.
</P><H2><SPAN class="mw-headline" id="Protocol_.235_:_Cloning_protocol_for_IDT_sequences">Protocol #5 : Cloning protocol for IDT sequences</SPAN></H2><H3><SPAN class="mw-headline" id="Resuspension">Resuspension</SPAN></H3><P>Resuspend gBlocks Gene fragment to a final concentration of 10 ng/µL in TE or AE (elution buffer).
The tube contain 1000 ng of gBlocks Gene fragment so you have to add 100µL of AE
Mix well, a vortex shall be used
</P><H3><SPAN class="mw-headline" id="Digestion_E.2FP">Digestion E/P</SPAN></H3><P>1. Digest 100 ng in 50µL
</P><TABLE class="wikitable"><TBODY><TR><TD>component
</TD><TD>50 µL of reaction
</TD></TR><TR><TD>DNA
</TD><TD>10 µL
</TD></TR><TR><TD>Buffer 2 (10X)
</TD><TD>5 µL
</TD></TR><TR><TD>BSA (100X)
</TD><TD>0.5 µL
</TD></TR><TR><TD>H2O
</TD><TD>33 µL
</TD></TR><TR><TD>EcoRI
</TD><TD>1 µL
</TD></TR><TR><TD>PstI
</TD><TD>1 µL
</TD></TR></TBODY></TABLE><P>2. Incubate for 45 min at 37°C
3. Incubate 20 min at 80°C (inactivation of restriction enzyme)
</P><H3><SPAN class="mw-headline" id="Ligation">Ligation</SPAN></H3><P>1. Ligate 50 ng vector with a 3X molar excess of gBlocks Gene fragments in fresh T4 DNA ligase buffer diluted to 1X and 400 U T4 DNA ligase for a total volume of 20 µL
</P><TABLE class="wikitable"><TBODY><TR><TD>component
</TD><TD>20 µL of reaction
</TD></TR><TR><TD>PSBIC3 (or other vector) at 12.5 ng/µL
</TD><TD>2 µL
</TD></TR><TR><TD>insert digested (at 37.5 ng/µL)
</TD><TD>12 µL
</TD></TR><TR><TD>T4 Buffer
</TD><TD>2 µL
</TD></TR><TR><TD>T4 ligase 400 U
</TD><TD>1 µL
</TD></TR><TR><TD>H2O
</TD><TD>3 µL
</TD></TR></TBODY></TABLE><P>2. Incubate 2h at room temperature
</P><H3><SPAN class="mw-headline" id="Transformation">Transformation</SPAN></H3><OL><LI>Add 10 µL of ligation’s product to 100 µL of competent cells thawed in ice (TGI cells)</LI><LI>Incubate 45 min in ice</LI><LI>Thermal shock: put tubes in the thermomixer at 42°C during 2 min</LI><LI>Incubate 5 min in ice</LI><LI>Add 900 µL of LB</LI><LI>Incubate 1 hour at 37°C with agitation</LI><LI>Centrifuge 5 min at 5000 rpm</LI><LI>Eliminate 850 µL of supernatant</LI><LI>Suspend the pellet in the 150µL of remaining medium</LI><LI>Spread 150 µL on LB limp (with antibiotic)</LI></OL><P>To make negative control, follow the same procedure but without add plasmids and spread 300 µL
</P><H2><SPAN class="mw-headline" id="Protocol_.236_:_Digestion_and_ligation_for_verification_and_BioBrick_Assembly">Protocol #6 : Digestion and ligation for verification and BioBrick Assembly</SPAN></H2><H3><SPAN class="mw-headline" id="Digestion_.28verification.29_protocol">Digestion (verification) protocol</SPAN></H3><TABLE class="wikitable"><TBODY><TR><TD>DNA
</TD><TD>Between 50 and 100 ng
</TD></TR><TR><TD>EcoRI-HF
</TD><TD>0.2 µL
</TD></TR><TR><TD>PstI
</TD><TD>0.2 µL
</TD></TR><TR><TD>10X NEBuffer 2
</TD><TD>2 µL
</TD></TR><TR><TD>H2O
</TD><TD>QS 20 µL
</TD></TR></TBODY></TABLE><P>Incubate all digest reactions at 37°C for 1 hour and then add 3 µL of SES 4X and migrate 30 min at 150V on a 1% agarose gel.
</P><H3><SPAN class="mw-headline" id="Digestion_protocol_BioBrick_Assembly_Kit">Digestion protocol BioBrick Assembly Kit</SPAN></H3><H4><SPAN class="mw-headline" id="Upstream_part_:">Upstream part :</SPAN></H4><TABLE class="wikitable"><TBODY><TR><TD>Upstream part plasmid
</TD><TD>500 ng
</TD></TR><TR><TD>EcoRI-HF
</TD><TD>1 µL
</TD></TR><TR><TD>SpeI
</TD><TD>1 µL
</TD></TR><TR><TD>10X NEBuffer 2
</TD><TD>5 µL
</TD></TR><TR><TD>H2O
</TD><TD>To 50 µL
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Downstream_part.2C_optional.2C_for_3A_strategy">Downstream part, optional, for 3A strategy</SPAN></H4><TABLE class="wikitable"><TBODY><TR><TD>Downstream part plasmid
</TD><TD>500 ng
</TD></TR><TR><TD>XbaI
</TD><TD>1 µL
</TD></TR><TR><TD>PstI
</TD><TD>1 µL
</TD></TR><TR><TD>10X NEBuffer 2
</TD><TD>5 µL
</TD></TR><TR><TD>H2O
</TD><TD>To 50 µL
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Destination_plasmid_:">Destination plasmid :</SPAN></H4><TABLE class="wikitable"><TBODY><TR><TD>Destination plasmid
</TD><TD>500 ng
</TD></TR><TR><TD>EcoRI-HF
</TD><TD>1 µL
</TD></TR><TR><TD>PstI
</TD><TD>1 µL
</TD></TR><TR><TD>DpnI
</TD><TD>1 µL
</TD></TR><TR><TD>10X NEBuffer 2
</TD><TD>5 µL
</TD></TR><TR><TD>100X BSA
</TD><TD>0.5 µL
</TD></TR><TR><TD>H2O
</TD><TD>To 50 µL
</TD></TR></TBODY></TABLE><P>Incubate the three restriction digest reactions at 37°C for 10 minutes and then heat inactivate at 80°C for 20 minutes.
</P><H3><SPAN class="mw-headline" id="Ligation_protocol_BioBrick_Assembly">Ligation protocol BioBrick Assembly</SPAN></H3><TABLE class="wikitable"><TBODY><TR><TD> Upstream part digestion
</TD><TD> 2 µL
</TD></TR><TR><TD> Downstream part digestion
</TD><TD> 2 µL
</TD></TR><TR><TD> Destination plasmid digestion
</TD><TD> 2 µL
</TD></TR><TR><TD> 10X T4 DNA ligase buffer
</TD><TD> 2 µL
</TD></TR><TR><TD> T4 DNA ligase
</TD><TD> 1 µL
</TD></TR><TR><TD> H2O
</TD><TD> 11 µL
</TD></TR></TBODY></TABLE><P>Incubate at RT for 1 hour.
</P><H2><SPAN class="mw-headline" id="Protocol_.237_:_PCR_clean-up_and_gel_extraction">Protocol #7 : PCR clean-up and gel extraction</SPAN></H2><TABLE class="wikitable"><TBODY><TR><TD> PCR clean-up
</TD><TD> Gel extraction
</TD></TR><TR><TD>PCR clean-up,  DNA clean-up,or single stranded DNA clean-up: Adjust binding conditionGel extraction: Excise DNA fragment / solubilize gel slice
</TD><TD>200 μL NTI/ 100 μL PCR
</TD><TD>200 μL NTI/ 100 mg gel(melt at 50°C for 5–10 min)
</TD></TR><TR><TD> Bind DNA
</TD><TD colspan="2">Spin at 11,000g during 30s in a clean-up column
</TD></TR><TR><TD> Wash silica membrane
</TD><TD colspan="2">Add 700 μL NT3 and spin at 11,000g during 30s <U>Recommended</U>: 2nd wash again with 700μL NT3 spin 11,000g 30s
</TD></TR><TR><TD> Dry silica membrane
</TD><TD colspan="2">Spin at 11,000g during 1 min
</TD></TR><TR><TD> Elute DNA
</TD><TD colspan="2">Add 15–30μL NE, let at RT 1min and spin 11,000g during 1min
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Protocol_.238_:_Generalised_transduction_using_phage_P1">Protocol #8 : Generalised transduction using phage P1</SPAN></H2><H3><SPAN class="mw-headline" id="Step1_:_P1_lysat_on_the_given_strain">Step1 : P1 lysat on the given strain</SPAN></H3><P>From a overnight starter, (3mL in 2YT medium + appropriated antibiotiques) from the strand holding the transduct mutation, make a 1/100 dilution in 3mL 2YT medium and add 150µL of Cacl2 0.1M ( without antibiotique corresponding of the transduct allele).
Incubate at 37°C (under agitation) until obtening a DO600=0.5-1 (more 0.5)
Add 150µL of a P1 lysat wt
Incubate at 37°C without agitation (water bath) during 20min
Incubate at 37°C under agitation during 3 or 4h (even overnight)
Divide the culture in 2 eppendorfs of 2mL (=2*1.5mL)
Add 150µL chloroform then vortex
Centrifuge 10min at 4°C and 6000rpm
Transfer 1mL supernatant into 2 news eppendorf of 2mL 
Add 10µL chloroform (under a fumehood) in each eppendorf, vortex and keep at 4°C
(option: to increase the titration level, you can do a second cycle)
</P><H3><SPAN class="mw-headline" id="Step2_:_Transduction_P1">Step2 : Transduction P1</SPAN></H3><P>The eve, prepare a culture of the recipient strain in 3mL 2YT medium (+eventual appropriated antibiotique)
The D-day, start a culture from the recipient strain (1/50 dilution) in 3mL 2YT + 150µL of CaCl2 0.1M
( prepare the LB-citrate 5mM requiered and LBagar- appropriated AB + citrate 2mM petri dishes)
Incubate until reaching a DO600 = 1 or + (2.10^6 bact/mL)
Prepare 3 tubes eppendorf of 2mL (1 control with 500µL cells, 1 with 10µL P1 + 500µL cells, 1 with 100µL P1 + 500µL cells)
Infect during 20min at 37°C (heat wuthout agitation)
Add 1mL steril LB-citrate 5mM and heat at 37°C during 50min under agitation (700-750rpm)
Centrifuge 5min at 5000rpm (RT)
Remove supernatant sterilely with a P1000 and resuspend in 1mL steril LB-citrate 5mM
Spread on LBagar- appropried AB + 2mM citrate
Incubate ON at 37°C 
</P><H3><SPAN class="mw-headline" id="Step3_:_Clones_isolation">Step3 : Clones isolation</SPAN></H3><P>Isolate clones on petri dishes (approprieted AB + citrate 2mM)
Incubate at 37°C
</P><H3><SPAN class="mw-headline" id="Step_4_:_PCR_screen_and_transplantation">Step 4 : PCR screen and transplantation</SPAN></H3><P>Test clones by PCR
Transplant the good clones on LB-AB appropriate petri dishes
</P><H2><SPAN class="mw-headline" id="Protocol_.239_:_Cadaverin_HPLC_analysis">Protocol #9 : Cadaverin HPLC analysis</SPAN></H2><H3><SPAN class="mw-headline" id="Reaction_of_lysine_and_cadaverine_produced_by_biotransformation_with_whole_cells">Reaction of lysine and cadaverine produced by biotransformation with whole cells</SPAN></H3><P>The assays were performed in a total volume of 500µL, containing 500mM sodium acetate buffer (ph = 6.0), 10mM L-lysine, 0.1 mM pyridoxal-5-phosphate, and 20µL whole cells, at 37°C in a water bath. The reaction was stopped after 24h by the addition of 10µL ethanol. The reaction mixtures were centrifuged and the 30µL of supernatant was applied to derivatization to determine the amounts of residual lysine and product, i.e cadaverine. 
</P><H3><SPAN class="mw-headline" id="Derivatization_reaction">Derivatization reaction</SPAN></H3><P>Diamine deritatives were obtained by the reaction of 180µL of borate buffer 0.05M (pH9), 60µL of 100% methanol, 47µL diethyl ethoxymethylenemalonate without any pretreatment. The samples were heated at 70)C for 2h to allow complete degradation of excess DEEMM and derivatization.
</P><H3><SPAN class="mw-headline" id="HPLC_analysis">HPLC analysis</SPAN></H3><P>After derivatization with diethyl ethoxymethylmalonate, analyses were performed on a high performance liquid chromatograph (HPLC Agilent technologies, 1260 Infinity) consisting of a binary pump, an inline degasser, an autosampler and a column thermostat. Chromatographic separation was carried out by reverse-phasechromatography on a C18 column maintained at 35°C. Mobile phase A was composed of 100% acetonitrile, and B was made up of 25mM aqueous sodium acetate buffer (pH = 4.8). The flow rate of 1mL/min was used, with the following gradient program : 0-2min, (20-25% A; 2-32min, 25-60%A; 32-40min, 60-20%A. Detection was carried out at 284nm.
</P><H2><SPAN class="mw-headline" id="Protocol_.2310_:_SLIC_.28sequence-_and_ligation-independent_cloning.29">Protocol #10 : SLIC (sequence- and ligation-independent cloning)</SPAN></H2><H3><SPAN class="mw-headline" id="Procedures">Procedures</SPAN></H3><P>1. Digest vector with restriction enzyme during 3h and purify the linearized vector with a commercial PCR purification kit. Elute DNA with elution buffer or 10mM TrisCl, pH 8.0-8.5. Do nnot elute the DNA with water or TE. Measure the concentration of the vector.  
</P><P>2. Amplify your gene of interest by PCR using primers with 15 mer (or inferior) homology extension to the linearized vector end. We usually use 15bp homology for single fragment cloning, and 20bp for multiple fragment cloning. Purify the linearized vector with a commercial PCR purification kit. Elute the DNA with water or TE. Measure the concentration of the insert(s). 
</P><P>3. Mix the linearized vector and insert at a molar ratio of 1.2 ina 1.5mL tube. An examble is shown as follows. (Vector to insert molar ratio of 1/1 for single fragment cloning, 1:2:2 for multiple fragments cloning. AN example of 3 fragments cloning is shown below with vector: insert 1: insert 2 molar ratio as shown is 1:2:2 as shown in Fig. 4A).
</P><TABLE class="wikitable"><TBODY><TR><TD> Stock concentration
</TD><TD> Volume added
</TD><TD> Final concentration
</TD></TR><TR><TD> Linearized vector
</TD><TD> 100ng/µL
</TD><TD> 1µL
</TD><TD> 10ng/µL
</TD></TR><TR><TD> Insert 1
</TD><TD> 40ng/µL
</TD><TD> 1µL
</TD><TD> 4ng/µL
</TD></TR><TR><TD> Insert 2
</TD><TD> 40ng/µL
</TD><TD> 1µL
</TD><TD> 4ng/µL
</TD></TR><TR><TD> 10X BSA
</TD><TD> 1µL
</TD><TD> 1X
</TD></TR><TR><TD> 10X NEB Buffer 2
</TD><TD> 1µL
</TD><TD> 1X
</TD></TR><TR><TD> H2O
</TD><TD> Up to 10µL
</TD></TR></TBODY></TABLE><P>4. Add 0.2µL of T4 DNA polymerase (3U/µL, NEB) to the mixture and incubate 2min at room temperature
</P><H2><SPAN class="mw-headline" id="Protocol_.2311_:_Swimming_test">Protocol #11 : Swimming test</SPAN></H2><P>Soft geloses have been made by adding 0.3% agar in LB medium. Once dried, each bacteria strain has been spotted using a tooth pic on the petri dishes and incubated for 3h in a 37°C room.
</P><H2><SPAN class="mw-headline" id="Protocol_.2312_:_SDS_page_and_coomassie_blue">Protocol #12 : SDS page and coomassie blue</SPAN></H2><H3><SPAN class="mw-headline" id="Culture">Culture</SPAN></H3><P>From an over night starter, cells were diluted and grown from Abs(600nm)=0.2 to Abs(600nm)=1. Then 1UOD of cells (1.67ml at 0.6OD) was collected and centrifuged at 5000g for 5min. After removal of the supernatant, the cell pellet was resuspended in 50µL SDS-PAGE sample buffer. We heated the mix at 95°C during 15min 
</P><H3><SPAN class="mw-headline" id="Staining_with_Coomassie_Blue_R250">Staining with Coomassie Blue R250</SPAN></H3><UL><LI> Stain the gel with 0.1% (or less) Coomassie Blue R250 in 10% acetic acid, 50% methanol, and 40% H2O for the minimum time (typically less than one hour) necessary to visualize the bands of interest. </LI></UL><UL><LI> The gel should be exposed to 10% acetic acid, 50% methanol for a total (stain plus destain) period of at least 3 hours (with shaking and at least three solvent changes) to ensure adequate removal of SDS.</LI></UL><UL><LI> Destain the gel by soaking for at least 2 hours in 10% acetic acid, 50% methanol, and 40% H2O with at least two changes of this solvent. If the gel still has a Coomassie Blue background then continue destaining until the background is nearly clear.</LI></UL><H3><SPAN class="mw-headline" id="Staining_with_Colloidal_Coomassie_Blue_Staining_Kit_.28Invitrogen_LC6025.29">Staining with Colloidal Coomassie Blue Staining Kit (Invitrogen LC6025)</SPAN></H3><P>• After electrophoresis, fix gel in 40% methanol/ 50%water/ 10% acetic acid for approximately ½ hr.  • Expose the gel in staining solution overnight and destain the gel by changing water frequently. 
</P><TABLE class="wikitable"><TBODY><TR><TD> Staining solution
</TD><TD> 18 x 16cm per gel
</TD></TR><TR><TD> Total volumes (ml)
</TD><TD> 200
</TD></TR><TR><TD> water
</TD><TD> 110
</TD></TR><TR><TD> MeOH
</TD><TD> 10
</TD></TR><TR><TD> Stainer A
</TD><TD> 40
</TD></TR><TR><TD> Stainer B
</TD><TD> 10
</TD></TR></TBODY></TABLE><P>Note: To get the highest sensitivity mix water, MeOH and Stainer A together and expose the gel in this solution for 10 min, then add the appropriate volume of Stainer B.
</P><H2><SPAN class="mw-headline" id="SLIC_Oligo_table">SLIC Oligo table</SPAN></H2><TABLE class="wikitable"><TBODY><TR><TD> DesA slic forward
</TD><TD> cgctaaggatgatttctgGAATTCGCGGCCGCTTCTAGATGCGCTCGCACTTGCTT
</TD></TR><TR><TD> desA slic reverse
</TD><TD> ttgcccttttttgccggaCTGCAGCGGCCGCTACTAGTATTATTAGGAGGCACGGTC
</TD></TR><TR><TD> desB slic forward
</TD><TD> cgctaaggatgatttctgGAATTCGCGGCCGCTTCTAGATGGGTATTGGTCTTGGG
</TD></TR><TR><TD> desB slic reverse
</TD><TD> ttgcccttttttgccggaCTGCAGCGGCCGCTACTAGTATTATTACACTGCAAATTC
</TD></TR><TR><TD> desC slic forward
</TD><TD> cgctaaggatgatttctgGAATTCGCGGCCGCTTCTAGATGTCTCGCCTTTCCACG
</TD></TR><TR><TD> desC slic reverse
</TD><TD> ttgcccttttttgccggaCTGCAGCGGCCGCTACTAGTATTATTAGGCTGAAACCGC
</TD></TR><TR><TD> desD slic forward
</TD><TD> cgctaaggatgatttctgGAATTCGCGGCCGCTTCTAGATGAGTTTAGCTGATGCA
</TD></TR><TR><TD> des D slic reverse
</TD><TD> ttgcccttttttgccggaCTGCAGCGGCCGCTACTAGTATTATTAACGCCCGGCTAA
</TD></TR><TR><TD> FliC E. coli slic forward
</TD><TD> cgctaaggatgatttctgGAATTCGCGGCCGCTTCTAGATGGCACAAGTCATTAAT
</TD></TR><TR><TD> FliC E.coli slic reverse
</TD><TD> ttgcccttttttgccggaCTGCAGCGGCCGCTACTAGTATTATTAACCCTGGAGCAG
</TD></TR><TR><TD> FliC desulfo slic forward
</TD><TD> cgctaaggatgatttctgGAATTCGCGGCCGCTTCTAGATGTCACTGGTTATCAAT
</TD></TR><TR><TD> FliC desulfo slic reverse
</TD><TD> ttgcccttttttgccggaCTGCAGCGGCCGCTACTAGTATTATTAGCCGCCGAGAAG
</TD></TR><TR><TD> CsgA E. Coli slic forward
</TD><TD> tttGAATTCGCGGCCGCTTCTAGatgaaacttttaaaagtagcagcaattg
</TD></TR><TR><TD> CsgA E. Coli slic reverse
</TD><TD> aaaCTGCAGCGGCCGCTACTAGTAttattagtactgatgagcggtcgc
</TD></TR></TBODY></TABLE></DIV></DIV><FOOTER><A href="http://igem.cnrs-mrs.fr">iGEM Aix-Marseille Université</A> - All content on this wiki is available under the Creative Commons Attribution 4.0 license (or any later version).
</FOOTER></DIV></BODY></HTML>