Team:Aix-Marseille/Experiments/Protocols
Toggle navigation ▤
iGEM Aix-Marseille Université
Team
Team
★  Collaborations
Project
★  Description
★ Design
Experiments
Results
Notebook
Parts
Parts
★ Basic Parts
★ Composite Parts
Safety
Medals
★  Attributions
Human Practices
Human Practices
★ Integrated Practices
★ Engagement
Model
facebook
twitter
Contents
1 Protocol #0 : iGEM General Protocol
2 Protocol #1 : Preparation of competent bacteria cells
3 Protocol #1bis : Preparation of Tbf1 and Tbf2 buffers
3.1 Preparation of 50 mL of Tbf1 Buffer
3.2 Preparation of 8 mL of Tbf2 Buffer
4 Protocol #2 : Transformations
4.1 Plasmids transformation
4.2 Ligation transformation
5 Protocol #3 : PCR and electrophoresis
5.1 PCR
5.2 Electrophoresis
6 Protocol #4 : Plasmid DNA purification
7 Protocol #5 : Cloning protocol for IDT sequences
7.1 Resuspension
7.2 Digestion E/P
7.3 Ligation
7.4 Transformation
8 Protocol #6 : Digestion and ligation for verification and BioBrick Assembly
8.1 Digestion (verification) protocol
8.2 Digestion protocol BioBrick Assembly Kit
8.2.1 Upstream part :
8.2.2 Downstream part, optional, for 3A strategy
8.2.3 Destination plasmid :
8.3 Ligation protocol BioBrick Assembly
9 Protocol #7 : PCR clean-up and gel extraction
10 Protocol #8 : Generalised transduction using phage P1
10.1 Step1 : P1 lysat on the given strain
10.2 Step2 : Transduction P1
10.3 Step3 : Clones isolation
10.4 Step 4 : PCR screen and transplantation
11 Protocol #9 : Cadaverin HPLC analysis
11.1 Reaction of lysine and cadaverine produced by biotransformation with whole cells
11.2 Derivatization reaction
11.3 HPLC analysis
12 Protocol #10 : SLIC (sequence- and ligation-independent cloning)
12.1 Procedures
13 Protocol #11 : Swimming test
14 Protocol #12 : SDS page and coomassie blue
14.1 Culture
14.2 Staining with Coomassie Blue R250
14.3 Staining with Colloidal Coomassie Blue Staining Kit (Invitrogen LC6025)
15 SLIC Oligo table
Protocols
Protocol #0 : iGEM General Protocol
Obtaining Biobricks
EITHER on IDT →
Order on IDT when the sequences are not too long (chemical synthesis).
The received biobricks are lyophilized, they must be resuspended in water (cf protocole).
OR by PCR →
When sequences are too long and costly to order on IDT, or when a sequence must be added to an existing biobrick :
Order primers
synthesize sequence by PCR (cf protocole Q5)
add the suffixes and prefixes during the synthesis « E, X » and « S, P » (E=EcoRI, X=XbaI, S=SpeI, P=PstI) and homologue sequences to plasmids
Do a SLIC (cf protocole) to insert the wanted sequence in the wanted plasmid
the Biobrick is obtained
Transform the Bacteria with the Biobrick
E/S Digestion of the Biobrick from the original plasmid and reinsertion in the E/S pre-restricted wanted plasmid.
Transform the pre-cultivated competent cells, via a thermal shock (DH5-α, TGI, BL21).
Add  an LB-agar in a petri dish and cultivate the bacteria in rich medium added with antibiotics of interest.
Test the Transformation
Run the obtained bacteria culture DNA in a PCR and  an electrophoresis gel (qualitative assessment)
Purify plasmids (by miniprep Kit) from the previously obtained positive colonies.
Do an E/P digestion verification to see if the inserted biobricks have the correct size (On only 100ng of plasmids)
Assemble two biobricks
“Digestion protocol BioBrick Assembly Kit” et “Ligation protocol BioBrick Assembly”
Do an E/S digestion on the first biobrick and an X/P on the second
Ligate the two biobricks then insert them into the plasmid
Transforming Bacteria with a new BioBrick
E/P Restriction of the biobrick and ligation into a E/P restricted plasmid
Transform the pre-cultivated competent cells, via a thermal shock (DH5-α, TGI ou BL21).
Pour an LB gel in a petri dish and cultivate the bacteria in an antibiotic rich medium
Test the Transformation
Run the obtained bacteria culture DNA in a PCR and pour an electrophoresis gel (qualitative assessment)
Purify plasmids (by miniprep Kit) from the previously obtained positive colonies.
Do an E/P digestion verification to see if the inserted biobricks have the correct size (On only 100ng of plasmids)
Send to sequencing
Repeat steps 4, 5, 6 until obtaining the wanted sequence
Protocol #1 : Preparation of competent bacteria cells
Note : Everything should be done in sterile conditions and on ice (4°C). For 100ml of competent cells
Grow a culture of bacteria in LB medium until it reaches OD600 = 0.5.
While it grows, prepare Tbf1 and Tbf2 buffers.
Centrifuge cells for 10 minutes at 3500g at 4°C
Resuspend the pellet slowly in 50mL of Tfb1 buffer.
Centrifuge 5 min at 3500g at 4°C.
Resuspend the pellet in 8 mL of Tbf2 buffer.
Incubate 15 min in ice.
Aliquot 200µL of cell suspension in sterile eppendorf tubes and conserve it at -80°C.
Protocol #1bis : Preparation of Tbf1 and Tbf2 buffers
For 100 mL of culture:
Preparation of 50 mL of Tbf1 Buffer
KAc 1M
1.5 mL
MnCl2 0.5M
5 mL
KCl 1 M
5 mL
CaCl2 0.1M
5 mL
Gly 80%
0.93 mL
H2O
32.56 mL
Preparation of 8 mL of Tbf2 Buffer
NaMOPS 0.2M
400 µL
CaCl2 0.1M
6 mL
KCl 1 M
8 mL
Gly 80%
1.5 mL
KCl 1M
80 µL
H2O
500 µL
Protocol #2 : Transformations
Plasmids transformation
Add 20 ng of plasmid to 100 μL of competent cells thawed in ice
Incubate 30-45 min in ice
Thermal shock : put tubes in the Thermomixer at 42°C for 2 min
Incubate 5 min in ice
Add 900 μL {of LB
Incubate 1 hour at 37°C with agitation
Spread 100 μL on LB limp (with antibiotic)
Ligation transformation
Add 20 ng of ligation product to 100 μL of competent cells thawed in ice
Incubate 30-45 min in ice
Thermal shock : put tubes in the Thermomixer at 42°C during 2 min
Incubate 5 min in ice
Add 900 μL of LB
Incubate 1 hour at 37°C with agitation
Centrifuge 5 min at 5000 rpm
Eliminate 850 μL of medium
Spread 150 μL on LB limp (with antibiotic)
To make negative control, follow the same procedure but without adding plasmids and spreading 300 μL
Protocol #3 : PCR and electrophoresis
PCR
25µL reaction
50µL reaction
100µL reaction
Final
concentration
EconoTaq PLUS
2X Master Mix
12.5µL
25µL
50µL
1X
Forward Primer
(100 pmol/μL)
0.5µL
0.5µL
1µL
1 pmol/μL (1 μM)
Reverse Primer
(100 pmol/μL)
0.5µL
0.5µL
1µL
1 pmol/μL (1 μM)
DNA Template*
0.5µL
1µL
1µL
 ?
Water
nuclease-free
[ ] μL qsp 25μL
[ ] μL qsp 50µL
[ ] μL qsp 100μL
*One colony in 50 μL of Water. 15-20 min at 80°C. 5 min at 96°C.
Cycling steps
Cycling step
Temperature
Time
Number of cycles
Initial denaturation
98°C
3 min
1
Denaturation
95°C
10 sec
X27
Annealing
50-65°C
40 sec
Extension
72°C
1 min/kb
Final extension
72°C
1 min/kb
1
Electrophoresis
Preparation of 0.8% agarose gel
Put 0.8g of agarose powder into 100mL of TAE 1X.
Melt in microwave and sitr occasionnally.
Let cool until you can take it into hands.
Flow the support with the gel.
Wait until it's solid.
Sample preparation and migration
Put the gel in the migration tank filled with 0.5X TAE.
Mix the samples with 1X marker***.
Migrate 30 min at 100V.
Revelation
Put the gel under UV.
Protocol #4 : Plasmid DNA purification
Protocol used for a Macherey-Nagel purification kit, for more details see here . For better results use gloves and if so do not use gloves.
Step
Actions
Cultivate and harvest cells
Centrifuge at 11,000g for 30 sec
Cell lysis
Add 250µL of buffer A1
Add 250µL of buffer A2
Incubate at room temperature up to 5 min
300µL buffer A3
Clarification of the lysate
Centrifuge at 11,000g for 5 min
Bind DNA
Place a purification column in a collection tube (2mL) and  carefully decant supernatant from the previous step, to a maximum of 750µL
Then centrifuge at 11,000g for 1 min and discard the flow-through.
Put the column back in the collection tube
Repeat this step to load the rest of the supernatant if need be
Wash silica membrane
Add 600µl of buffer A4
Centrifuge at 11,000g for 1 min
Dry silica membrane
Centrifuge at 11,000g for 2 min
Elute DNA
Place the columns in an 1.5mL eppendorf and add 50 µL of buffer AE and incubate at room temperature for 1 min
Centrifuge at 11,000g for 1 min
To measure your DNA concentration, use a Biospec Nano apparel or Nanodrop apparel.  Make the blank with  1µL of pure water and use 1µl of sample.  Make sure to clear the lens with water and with a special paper  in order to no line the lens.
Protocol #5 : Cloning protocol for IDT sequences
Resuspension
Resuspend gBlocks Gene fragment to a final concentration of 10 ng/µL in TE or AE (elution buffer). The tube contain 1000 ng of gBlocks Gene fragment so you have to add 100µL of AE Mix well, a vortex shall be used
Digestion E/P
1. Digest 100 ng in 50µL
component
50 µL of reaction
DNA
10 µL
Buffer 2 (10X)
5 µL
BSA (100X)
0.5 µL
H2O
33 µL
EcoRI
1 µL
PstI
1 µL
2. Incubate for 45 min at 37°C 3. Incubate 20 min at 80°C (inactivation of restriction enzyme)
Ligation
1. Ligate 50 ng vector with a 3X molar excess of gBlocks Gene fragments in fresh T4 DNA ligase buffer diluted to 1X and 400 U T4 DNA ligase for a total volume of 20 µL
component
20 µL of reaction
PSBIC3 (or other vector) at 12.5 ng/µL
2 µL
insert digested (at 37.5 ng/µL)
12 µL
T4 Buffer
2 µL
T4 ligase 400 U
1 µL
H2O
3 µL
2. Incubate 2h at room temperature
Transformation
Add 10 µL of ligation’s product to 100 µL of competent cells thawed in ice (TGI cells)
Incubate 45 min in ice
Thermal shock: put tubes in the thermomixer at 42°C during 2 min
Incubate 5 min in ice
Add 900 µL of LB
Incubate 1 hour at 37°C with agitation
Centrifuge 5 min at 5000 rpm
Eliminate 850 µL of supernatant
Suspend the pellet in the 150µL of remaining medium
Spread 150 µL on LB limp (with antibiotic)
To make negative control, follow the same procedure but without add plasmids and spread 300 µL
Protocol #6 : Digestion and ligation for verification and BioBrick Assembly
Digestion (verification) protocol
DNA
Between 50 and 100 ng
EcoRI-HF
0.2 µL
PstI
0.2 µL
10X NEBuffer 2
2 µL
H2O
QS 20 µL
Incubate all digest reactions at 37°C for 1 hour and then add 3 µL of SES 4X and migrate 30 min at 150V on a 1% agarose gel.
Digestion protocol BioBrick Assembly Kit
Upstream part :
Upstream part plasmid
500 ng
EcoRI-HF
1 µL
SpeI
1 µL
10X NEBuffer 2
5 µL
H2O
To 50 µL
Downstream part, optional, for 3A strategy
Downstream part plasmid
500 ng
XbaI
1 µL
PstI
1 µL
10X NEBuffer 2
5 µL
H2O
To 50 µL
Destination plasmid :
Destination plasmid
500 ng
EcoRI-HF
1 µL
PstI
1 µL
DpnI
1 µL
10X NEBuffer 2
5 µL
100X BSA
0.5 µL
H2O
To 50 µL
Incubate the three restriction digest reactions at 37°C for 10 minutes and then heat inactivate at 80°C for 20 minutes.
Ligation protocol BioBrick Assembly
Upstream part digestion
2 µL
Downstream part digestion
2 µL
Destination plasmid digestion
2 µL
10X T4 DNA ligase buffer
2 µL
T4 DNA ligase
1 µL
H2O
11 µL
Incubate at RT for 1 hour.
Protocol #7 : PCR clean-up and gel extraction
PCR clean-up
Gel extraction
PCR clean-up,  DNA clean-up,
or single stranded DNA clean-up: Adjust binding condition
Gel extraction: Excise DNA fragment / solubilize gel slice
200 μL NTI/ 100 μL PCR
200 μL NTI/ 100 mg gel
(melt at 50°C for 5–10 min)
Bind DNA
Spin at 11,000g during 30s in a clean-up column
Wash silica membrane
Add 700 μL NT3 and spin at 11,000g during 30s Recommended: 2nd wash again with 700μL NT3 spin 11,000g 30s
Dry silica membrane
Spin at 11,000g during 1 min
Elute DNA
Add 15–30μL NE, let at RT 1min and spin 11,000g during 1min
Protocol #8 : Generalised transduction using phage P1
Step1 : P1 lysat on the given strain
From a overnight starter, (3mL in 2YT medium + appropriated antibiotiques) from the strand holding the transduct mutation, make a 1/100 dilution in 3mL 2YT medium and add 150µL of Cacl2 0.1M ( without antibiotique corresponding of the transduct allele). Incubate at 37°C (under agitation) until obtening a DO600=0.5-1 (more 0.5) Add 150µL of a P1 lysat wt Incubate at 37°C without agitation (water bath) during 20min Incubate at 37°C under agitation during 3 or 4h (even overnight) Divide the culture in 2 eppendorfs of 2mL (=2*1.5mL) Add 150µL chloroform then vortex Centrifuge 10min at 4°C and 6000rpm Transfer 1mL supernatant into 2 news eppendorf of 2mL  Add 10µL chloroform (under a fumehood) in each eppendorf, vortex and keep at 4°C (option: to increase the titration level, you can do a second cycle)
Step2 : Transduction P1
The eve, prepare a culture of the recipient strain in 3mL 2YT medium (+eventual appropriated antibiotique) The D-day, start a culture from the recipient strain (1/50 dilution) in 3mL 2YT + 150µL of CaCl2 0.1M ( prepare the LB-citrate 5mM requiered and LBagar- appropriated AB + citrate 2mM petri dishes) Incubate until reaching a DO600 = 1 or + (2.10^6 bact/mL) Prepare 3 tubes eppendorf of 2mL (1 control with 500µL cells, 1 with 10µL P1 + 500µL cells, 1 with 100µL P1 + 500µL cells) Infect during 20min at 37°C (heat wuthout agitation) Add 1mL steril LB-citrate 5mM and heat at 37°C during 50min under agitation (700-750rpm) Centrifuge 5min at 5000rpm (RT) Remove supernatant sterilely with a P1000 and resuspend in 1mL steril LB-citrate 5mM Spread on LBagar- appropried AB + 2mM citrate Incubate ON at 37°C
Step3 : Clones isolation
Isolate clones on petri dishes (approprieted AB + citrate 2mM) Incubate at 37°C
Step 4 : PCR screen and transplantation
Test clones by PCR Transplant the good clones on LB-AB appropriate petri dishes
Protocol #9 : Cadaverin HPLC analysis
Reaction of lysine and cadaverine produced by biotransformation with whole cells
The assays were performed in a total volume of 500µL, containing 500mM sodium acetate buffer (ph = 6.0), 10mM L-lysine, 0.1 mM pyridoxal-5-phosphate, and 20µL whole cells, at 37°C in a water bath. The reaction was stopped after 24h by the addition of 10µL ethanol. The reaction mixtures were centrifuged and the 30µL of supernatant was applied to derivatization to determine the amounts of residual lysine and product, i.e cadaverine.
Derivatization reaction
Diamine deritatives were obtained by the reaction of 180µL of borate buffer 0.05M (pH9), 60µL of 100% methanol, 47µL diethyl ethoxymethylenemalonate without any pretreatment. The samples were heated at 70)C for 2h to allow complete degradation of excess DEEMM and derivatization.
HPLC analysis
After derivatization with diethyl ethoxymethylmalonate, analyses were performed on a high performance liquid chromatograph (HPLC Agilent technologies, 1260 Infinity) consisting of a binary pump, an inline degasser, an autosampler and a column thermostat. Chromatographic separation was carried out by reverse-phasechromatography on a C18 column maintained at 35°C. Mobile phase A was composed of 100% acetonitrile, and B was made up of 25mM aqueous sodium acetate buffer (pH = 4.8). The flow rate of 1mL/min was used, with the following gradient program : 0-2min, (20-25% A; 2-32min, 25-60%A; 32-40min, 60-20%A. Detection was carried out at 284nm.
Protocol #10 : SLIC (sequence- and ligation-independent cloning)
Procedures
1. Digest vector with restriction enzyme during 3h and purify the linearized vector with a commercial PCR purification kit. Elute DNA with elution buffer or 10mM TrisCl, pH 8.0-8.5. Do nnot elute the DNA with water or TE. Measure the concentration of the vector.
2. Amplify your gene of interest by PCR using primers with 15 mer (or inferior) homology extension to the linearized vector end. We usually use 15bp homology for single fragment cloning, and 20bp for multiple fragment cloning. Purify the linearized vector with a commercial PCR purification kit. Elute the DNA with water or TE. Measure the concentration of the insert(s).
3. Mix the linearized vector and insert at a molar ratio of 1.2 ina 1.5mL tube. An examble is shown as follows. (Vector to insert molar ratio of 1/1 for single fragment cloning, 1:2:2 for multiple fragments cloning. AN example of 3 fragments cloning is shown below with vector: insert 1: insert 2 molar ratio as shown is 1:2:2 as shown in Fig. 4A).
Stock concentration
Volume added
Final concentration
Linearized vector
100ng/µL
1µL
10ng/µL
Insert 1
40ng/µL
1µL
4ng/µL
Insert 2
40ng/µL
1µL
4ng/µL
10X BSA
1µL
1X
10X NEB Buffer 2
1µL
1X
H2O
Up to 10µL
4. Add 0.2µL of T4 DNA polymerase (3U/µL, NEB) to the mixture and incubate 2min at room temperature
Protocol #11 : Swimming test
Soft geloses have been made by adding 0.3% agar in LB medium. Once dried, each bacteria strain has been spotted using a tooth pic on the petri dishes and incubated for 3h in a 37°C room.
Protocol #12 : SDS page and coomassie blue
Culture
From an over night starter, cells were diluted and grown from Abs(600nm)=0.2 to Abs(600nm)=1. Then 1UOD of cells (1.67ml at 0.6OD) was collected and centrifuged at 5000g for 5min. After removal of the supernatant, the cell pellet was resuspended in 50µL SDS-PAGE sample buffer. We heated the mix at 95°C during 15min
Staining with Coomassie Blue R250
Stain the gel with 0.1% (or less) Coomassie Blue R250 in 10% acetic acid, 50% methanol, and 40% H2O for the minimum time (typically less than one hour) necessary to visualize the bands of interest.
The gel should be exposed to 10% acetic acid, 50% methanol for a total (stain plus destain) period of at least 3 hours (with shaking and at least three solvent changes) to ensure adequate removal of SDS.
Destain the gel by soaking for at least 2 hours in 10% acetic acid, 50% methanol, and 40% H2O with at least two changes of this solvent. If the gel still has a Coomassie Blue background then continue destaining until the background is nearly clear.
Staining with Colloidal Coomassie Blue Staining Kit (Invitrogen LC6025)
• After electrophoresis, fix gel in 40% methanol/ 50%water/ 10% acetic acid for approximately ½ hr.  • Expose the gel in staining solution overnight and destain the gel by changing water frequently.
Staining solution
18 x 16cm per gel
Total volumes (ml)
200
water
110
MeOH
10
Stainer A
40
Stainer B
10
Note: To get the highest sensitivity mix water, MeOH and Stainer A together and expose the gel in this solution for 10 min, then add the appropriate volume of Stainer B.
SLIC Oligo table
DesA slic forward
cgctaaggatgatttctgGAATTCGCGGCCGCTTCTAGATGCGCTCGCACTTGCTT
desA slic reverse
ttgcccttttttgccggaCTGCAGCGGCCGCTACTAGTATTATTAGGAGGCACGGTC
desB slic forward
cgctaaggatgatttctgGAATTCGCGGCCGCTTCTAGATGGGTATTGGTCTTGGG
desB slic reverse
ttgcccttttttgccggaCTGCAGCGGCCGCTACTAGTATTATTACACTGCAAATTC
desC slic forward
cgctaaggatgatttctgGAATTCGCGGCCGCTTCTAGATGTCTCGCCTTTCCACG
desC slic reverse
ttgcccttttttgccggaCTGCAGCGGCCGCTACTAGTATTATTAGGCTGAAACCGC
desD slic forward
cgctaaggatgatttctgGAATTCGCGGCCGCTTCTAGATGAGTTTAGCTGATGCA
des D slic reverse
ttgcccttttttgccggaCTGCAGCGGCCGCTACTAGTATTATTAACGCCCGGCTAA
FliC E. coli slic forward
cgctaaggatgatttctgGAATTCGCGGCCGCTTCTAGATGGCACAAGTCATTAAT
FliC E.coli slic reverse
ttgcccttttttgccggaCTGCAGCGGCCGCTACTAGTATTATTAACCCTGGAGCAG
FliC desulfo slic forward
cgctaaggatgatttctgGAATTCGCGGCCGCTTCTAGATGTCACTGGTTATCAAT
FliC desulfo slic reverse
ttgcccttttttgccggaCTGCAGCGGCCGCTACTAGTATTATTAGCCGCCGAGAAG
CsgA E. Coli slic forward
tttGAATTCGCGGCCGCTTCTAGatgaaacttttaaaagtagcagcaattg
CsgA E. Coli slic reverse
aaaCTGCAGCGGCCGCTACTAGTAttattagtactgatgagcggtcgc
iGEM Aix-Marseille Université - All content on this wiki is available under the Creative Commons Attribution 4.0 license (or any later version).
