Team:NCKU Tainan/Model
Toggle navigation
MathJax AsciiMath Test Page
Model
Abstract
In order to combine the sensing device with synthetic biotechnology, first of all, we us Simbiology software to built a theoretical model for our promoter PyeaR and describing the whole mechanism and reaction.
Then we found out the concentration of each substance varied with time by Matlab, and approximate method was applied for choosing a proper model fitting with GFP fluorescence variation curve.
To improve our sensing detection, we built a statistical model by nonlinear regression and calibration. Furthermore, we created analysis method for sensing data by our empirical model constructed with data from more than 150 experiments;           this model is able to divide nitrate concentration into 5 sections.
Motivation of Improving PyeaR Model
After taking a look to the result of previous teams, we found out that it’s not enough if we only describe the sensing reaction of PyeaR with NsrR; so to realize the mechanism, the paper, Activation of yeaR-yoaG Operon Transcription by the Nitrate-Responsive Regulator NarL Is Independent of Oxygen-Responsive Regulator Fnr in Escherichia coli K-12,           was referred. From figure 1. in the paper, no matter with or without O2, the influence of NsrR and NarL to NO3- had not much difference.
Instead, from Figure 2. the influence of NsrR and NarL to PyeaR promoter was significant, which was the reason for us to consider the effects of both NsrR and NarL while building a more complete model.
Figure 1.
Figure 2.
PyeaR Mechanism
There are Nap and NirK enzymes that can catalyze NO3- to NO2- and NO2- to NO separately in our E. coli system. After paper searching, we found that the promoter's reaction           was controlled by two gene representing two binding sites, one of which was NarL, and the other was NsrR. NarL is able to sense both nitrate and nitrite, promoting PyeaR to produce GFP further. On the other hand, NsrR has the ability           to block RNA polymerase from binding with PyeaR promoter, but nitric oxide will release NsrR from the binding site, therefore allow transcription process to start.
Equations of Our Sensing Pathway Model
NO3- will be consumed in two ways, one of which is turning into NO2- by Nap enzyme, and the other one is binding with NarL thus directly induce the transcription of GFP gene
The rate of [NO3-] can be expressed by:
NO2- can be produced by Nap enzyme, being consumed by NarL and NirK enzyme, or induce the production of mRNA of GFP (mGFP).
The rate of [NO2-] can be expressed by:
NO can be produced by NirK enzyme.
The rate of [NO] can be expressed by:
There are 3 major sources of mGFP production: one is from NO3-, another is from NO2-, and the other is from NO.
Also, mGFP will be degraded naturally, which is also included in our model.
The rate of [mGFP] can be expressed by:
How can we derive this term
For NsrR
Two NO and NsrR combine to form a complex before they are functional. Therefore, transcription factors cooperatively regulate expression of genes.
Consider some free inducer NO* that bind to two repressors NsrR, giving rise to a complex NsrR2NO:
2 [NsrR] + [NO]
↔
k
o
n
/
k
o
f
f
[NsrR 2NO]
a) rate of complex formation:
kon[NO][NsrR]2
b) rate of dissociation:
koff[NsrR2NO]
c) The rate of complex formation at steady state:
d
N
s
r
R
2
N
O
d
t
=
=k on[NO][NsrR]2-koff[NsrR2NO]=0
For conservation equation, describing the total concentration can be obtained:
[NsrR2NO] + [NO*] = [NOTotal]
NsrR dimerization with respect to unbound NsrR can be represented by the following equation by considering the degree of cooperativity of NO binding into account:
Therefore,
Finally, mGFP varied with time can be described into: (for only influenced by NO/NsrR effect)
More information can be looked for in the following references.
Finally, mRNA of GFP will be translated into GFP.
The rate of [GFP] can be expressed by:
d
[
G
F
P
]
d
t
=
k
t
r
a
n
s
l
a
t
i
o
n
×
m
G
F
P
-
r
G
F
P
×
G
F
P
------(5)
Parameter Table
Description
Value
Unit(SI)
[NO3-] (100ppm)
Nitrate initial value
1.6x10-6
mol/m3
Km (Nap)
the NO3- at which the reaction rate is at half-maximum
8x10-3
mol/m3
Vmax(Nap)
Maximum velocity of Nap
4.7x10-1
mol/s x m3
Km(NirK)
the NO2- at which the reaction rate is at half-maximum
2.5x10-1
mol/m3
Vmax(NirK)
Maximum velocity of NirK
5x10-3
mol/s x m3
[PyeaR]
Concentration of PyeaR
10-10
mol/m3
ktranscription
Rate of mGFP synthesis
1.8x10-5
1/s
kfNo3
Related constant of NO3- and NarL
3x10-4
1/s
kfNo2
Related constant of NO2- and NarL
6x10-5
1/s
r
m
G
F
P
mGFP degradation rate
5x10-5
1/s
r
G
F
P
GFP degradation rate
2.5x10-6
1/s
ktranslation
Rate of GFP synthesis
4x10-4
1/s
kd(NsrR)
Dissociation constant of NsrR
3.5x10-6
m3/mol
kNO
Dissociation constant of NO
1.4x10-4
mol/m3
[NsrR]
Concentration of NsrR
10-6
mol/m3
Simulation
Simbiology of Matlab is used to simulate the model:
Get a Function to Describe GFP varied with time
By taking the parameters into the (1)~(5) equations, the [GFP] can be described by:
GFP(t)= (2.9x10-8)e-2.5x10-6t - (3.08x10-8)e-4.485x10-4t + (6.06x10-10)e-2x10-2t - (1.48x10-9)e-1.74x10-2t
(This equation is for initial concentration of nitrate 100ppm)
The Fitting Results
As to know each term how to influence GFP, we divide GFP(t) into 4 parts:
A(t)=(2.9x10-8)e-2.5x10-6t
B(t)=(3.08x10-8)e-4.485x10-4t
C(t)=(6.06x10-10)e-2x10-2t
D(t)=(1.48x10-9)e-1.74x10-2t
Obviously, we easily know the influence within 2 hours of terms A and B are more essential than that of terms C and D. Consequently, we modify equation to be simpler, which neglact C and D terms, to fit our data getting from experiments with our device           and with powder.
By using general model Exp2:
f
x
=
d
×
e
g
x
+
h
×
e
j
x
Coefficients (with 95% confidence bounds):
Coefficients Table
value
min
max
d
2.8x103
2.8x103
2.9x103
g
2x10-5
1.8x10-5
2.2x10-5
h
-3.9x102
-4.3x102
-3.6x102
j
-5.6x10-4
-6.4x10-5
-4.5x10-5
Goodness of fit:
SSE: 5.624e+04
R-square: 0.9953
Adjusted R-square: 0.9952
RMSE: 15.41
GFP
t
=
2846
×
e
0.00002
t
-
390.5
×
e
-
-0.000558
t
(for 100 ppm)
Statistical Model
We randomly set 15 ppm of concentration of nitrate as the separating level of clean and polluted water. Hence, we dichotomized the concentration of nitrate into binary data (the outcomes of Bernoulli trials). Then, in order to fit a general linear model           we use the concentration of nitrate, time as explain variables, and the electrical signals collected from our device as response variables.
Here is the regression equation:
Yij
Electrical signals collected from i th time series data with jth second.
Tij
Time of ith time series data with jth second.
Xi = 1
When the concentration of nitrate above 15 ppm.
Xi = 0
When the concentration of nitrate below 15 ppm.
εij
Random error term of ith time series data with jth second.
After that, we use calibration to forecast the concentration of nitrate within the time series data with our model by minimizing the sum of square of time series data from 1st second to 1800th second. The training sensitivity           and specificity are shown in the table below.
Empirical Model
In order to build an empirical model for our sensing boat, we had done more than 150 experiments to set up our database.
We chose 5 intervals:
0 – 4 ppm
4 – 10 ppm
10 – 20 ppm
20 – 60 ppm
Over 60 ppm
And our sensing device only need to detect the Optical signal on 5 min, 10 min, 15 min, 20 min. By using this method, we can easily, quickly and precisely distinguish the concentration into this 5 intervals.
Conclusion
Not only to build a more complete model, according to species and application in the reality, we can also set different intervals by empirical method and any separating level by statistical method. With this advance, we are able to get the results within           20 to 30 minutes, which is quicker and more precise as well.
References
[1] Lin, H. Y., Bledsoe, P. J., & Stewart, V. (2007) "Activation of yeaR-yoaG operon transcription by the nitrate-responsive regulator NarL is independent of oxygen-responsive regulator Fnr in Escherichia coli K-12." Journal of bacteriology, 189(21),           7539-7548.
[2] Csicsery, N. & O’Laughlin, R. (2013) "A Mathematical Model of a Synthetically Constructed Genetic Toggle Switch." BENG 221 – Mathematical Methods in Bioengineering
[3] Nohno, T., Noji, S., Taniguchi, S., & Saito, T. (1989) "The narX and narL genes encoding the nitrate-sensing regulators of Escherichia coli are homologous to a family of prokaryotic two-component regulatory genes." Nucleic acids research, 17(8),           2947-2957.
[4] Partridge, J. D., Bodenmiller, D. M., Humphrys, M. S., & Spiro, S. (2009) "NsrR targets in the Escherichia coli genome: new insights into DNA sequence requirements for binding and a role for NsrR in the regulation of motility." Molecular microbiology, 73(4),           680-694.
[5] Peshawar iGEM team (2016)
[6] NYMU-Taipei IGEM team (2012)
[7] HKUST-Rice IGEM team (2015)
Abstract
Motivation of Improving PyeaR Model
PyeaR Mechanism
Equations of Our Sensing Pathway Model
Simulation
Get a Function to Describe GFP varied with time
The Fitting Results
Statistical Model
Empirical Model
Conclusion
References
