<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NCKU_Tainan_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NCKU Tainan/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="container-fluid"><DIV class="row"><DIV class="col"><DIV id="category" class="vertical-container"><H1 class="wet">Design</H1></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row"><DIV id="paragraph" class="paragraph col-md-8 col-md-offset-1 col-xs-offset-1 col-xs-10"><DIV class="title"><H1 id="sensing" class="os-animation" data-os-animation="fadeIn" data-os-animation-delay="0s">Sensing</H1></DIV><P>
        Nitrate and nitrite concentration is an important index in aqua quality, so we decided to monitor the nitrate and nitrite concentration in the water. We used the biobrick K381001 from BCCS-Bristol 2010 iGEM team to do the nitrate and nitrite sensing.
        The biobrick K381001 is composed of a nitrate-sensing promoter P<SUB>yeaR</SUB>, a strong RBS B0030 and a reporter gene GFP. The promoter P<SUB>yeaR</SUB> isn’t repressed under aerobic conditions, but it’s inhibited by protein NsrR, which
        is native to most <I><I><I>E. coli</I></I></I>. When nitrate or nitric oxide is bound to NsrR, the inhibition is halted. On the other hand, P<SUB>yeaR</SUB> is mainly activated by phospho-NarL, and NarL is phosphorylated by nitrate or nitrite (Lin <I>et al</I> ,2007). Accordingly, we can
        use K381001 as a sensor to detect nitrate, whose concentration can be estimated by quantified the fluorescence level.
      </P><P class="margin_bottom">
        Besides, in order to enhance the nitrate-sensing accuracy, we also planned to modify the biobrick K381001. By adding an additional <I>nsr</I>R binding sequence to K381001, the fluorescence basal level can be decreased since the additional <I>nsr</I>R
        binding sequence can reinforce the repression to P<SUB>yeaR</SUB> under no nitrate condition. As a result, the interference to nitrate-detection can be decreased as much as possible, and sensing of nitrate can be more accurate.
      </P><DIV class="title"><H1 id="regulation">Regulation</H1></DIV><P>
        In aquaculture industry, sickness or massive death of aquatic products may be attributed to virus, temperature and so on. However, increase of nitrate is also a serious problem. High concentration of nitrate may cause not only sickness of aquatic products
        but also eutrophication which leads to low dissolved oxygen in water body, and then destroys the water ecosystem. In addition, the discharge of nitrate-rich waste water can have negative influence on water ecosystem. To sum up, high amount of
        nitrate can lead to aquaculturing, environmental and ecological problem.
      </P><P>
        Due to the problem above, we aim at reducing the nitrate ion in the aquaculture water body. Moreover, we expect metabolite could be some valuable products. After literature searching, we didn’t choose the denitrification process which involves lots of
        enzymes and toxic intermediate, the nitrate and nitrite respiration<SUP> [1]</SUP> and nitrogen assimilation pathway in <I>E. coli</I><SUP> [2]</SUP> fit our goal instead.
      </P><P class="margin_bottom">
        According to the literature on nitrate and nitrite respiration, we decided to take advantage of nitrate reductase (NaR) and nitrite reductase (NiR) to convert nitrate to ammonia. Surprisingly, both pathways can be found in <I>E. coli</I>; hence,
        we take <I>E. coli</I> as the primary host. In nitrogen assimilation pathway, ammonium can be converted to glutamate by glutamate dehydrogenase (GDH), and further to glutamine by glutamine synthetase (GS). In these pathways, intermediates of the
        metabolic pathway are not toxic to the host, <I>E. coli</I>. Moreover, nitrate ion can be successfully transformed into glutamine, an important and valuable amino acid.
      </P><H2 id="nitrate" class="os-animation" data-os-animation="fadeIn" data-os-animation-delay="0s">Nitrate Assimilation Process</H2><H3>From nitrate to nitrite</H3><P>
        In <I>E. coli</I>, there are two ways to catalyze the nitrate to ammonium. One is in the periplasm and the other is in the cytoplasm. Based on our project, we want to capture the nitrate in the waters and change it to other product inside the
        cytoplasm. As a result, we use the nitrate and nitrite reductase to convert the nitrate to ammonia. At first, we want to use the <I>nar</I> cluster in <I>Pseudonomas</I>, because the activity is much more than the <I>E. coli</I>. We divided the
        <I>nar</I> cluster into three parts and took it for IDT synthesis. However, due to too high GC content of one-part sequence, it failed to synthesize. Therefore, we are looking for another solution, in which original <I>nar</I> cluster existed
        in the wild type <I>E. coli</I> MG1655 had been used in our project.
      </P><DIV class="row imagerow"><DIV class="col-md-12"><P>
            Figure 1. The arrangement of <I>nar</I> cluster in the genome
          </P></DIV></DIV><H3>From nitrite to ammonia</H3><P>
        Furthermore, we take advantage of the gene <I>nir</I>BD which encodes the sequence of nitrite reductase to convert nitrite into ammonia. Both two gene are NADH-dependent. The reaction formula shows below:
      </P><P style="text-align:center;color:#3cbac8">
        nitrite + 3 NADH + 5 H<SUP>+</SUP> → ammonium + 3 NAD<SUP>+</SUP> + 2 H<SUB>2</SUB>O
      </P><P>
        According to the data from Brenda database, the specific activity of nitrite reductase is just 0.052 µmol/min/mg in <I>E. coli</I>. We want to amplify the amount of nitrite reductase in the cytoplasm. As a result, we designed the primers as to get
        the <I>nir</I> cluster gene from genome. The primers contain the restriction sites of <I>Hind</I>III at the 5’ terminal and <I>Spe</I>I at the 3’ terminal. We digested the gene with the two restriction enzymes and then ligased into the pSB1C3
        vector. Using the strong promoter P<SUB>LacI</SUB> and high copy number ori pUC rather than pMB1. After constructing the plasmid, we transform it into <I>E. coli</I> DH5α for construction. Further, transform the plasmid into <I>E. coli</I>        MG1655 for protein expression.
      </P><DIV class="row imagerow"><DIV class="col-md-12"><P>
            Figure 2. The genetic circuit of BBa_K2275007
          </P></DIV></DIV><H3>Do our biobrick work?</H3><P class="margin_bottom">
        We used the method of griess reaction to test if our biobrick can successfully work. When the broth contained nitrite, the color will turn pink and have absorbance at wavelength 545 nm
      </P><H2 class="os-animation" data-os-animation="fadeIn" data-os-animation-delay="0s">Ammonium Assimilation</H2><H3>From ammonium to glutamate</H3><P>
        In order to convert ammonium to glutamate, we go through the GDH process and with the gene <I>gudB</I> which encodes the sequence of glutamate dehydrogenase. The gene comes from <I>Bacillus subtilis</I> and the activity is almost twice higher
        than the gene <I>gdhA</I> which has the same effect in <I>E. coli.</I> The reaction formula is shown below:
      </P><P style="text-align:center;color:#3cbac8">
        ammonium + 2-oxoglutarate + NADPH + H<SUP>+</SUP> ↔ L-glutamate + NADP<SUP>+</SUP> + H<SUB>2</SUB>O
      </P><P>
        Hence, we designed primers which contained the restriction sites of BamHIat 5’ terminal and <I>Pst</I>I at 3’ terminal to get the gene from genome. We digested the gene with the two restriction enzymes and then ligased into the pSB1C3 vector
        which is the same as before. After constructing the plasmid, we transformed it into <I>E. coli</I> DH5α for construction. Further, we transformed the plasmid into <I>E. coli</I> MG1655 for protein expression.
      </P><DIV class="row imagerow"><DIV class="col-md-12"><P>
            Figure 3. The genetic circuit of BBa_K2275008
          </P></DIV></DIV><H3>From glutamate to glutamine</H3><P>
        Next, to convert the glutamate to glutamine, we used the GS pathway. Using the gene <I>glnA </I>which encodes the sequence of glutamine synthetase is needed. The gene comes from Pseudomonas putida. The primers we use to amplify <I>glnA </I>gene
        contain restriction sites of <I>Hind</I>III and <I>Pst</I>I. We digested the gene with the two restriction enzymes and then ligased into the pSB1C3 vector which is the same as before. After constructing the plasmid, we transformed it into <I>E. coli</I>        DH5α for construction. Afterward, transformed the plasmid into <I>E. coli</I> MG1655 for protein expression.
      </P><DIV class="row imagerow"><DIV class="col-md-12"><P>
            Figure 4. The genetic circuit of BBa_K2275009
          </P></DIV></DIV><H3>From ammonium to glutamine</H3><P>
        We also constructed the <I>gudB</I> and <I>glnA </I>into the same backbone. Re-design the primers of <I>gudB</I> to reverse the 5’ and 3’ direction compare to previous one. The primers contained restriction sites of <I>Xba</I>I and <I>Pst</I>I.
        Afterward, digested the rev-<I>gudB</I> with <I>Xba</I>I and <I>Pst</I>I and BBa_K2275009 with <I>Spe</I>I and <I>Pst</I>I. Ligased and transformed into DH5α for construction. Further, transforming into <I>E. coli</I> MG1655 for protein
        expression.
      </P><DIV class="row imagerow"><DIV class="col-md-12"><P>
            Figure 5. The genetic circuit of BBa_K2275010
          </P></DIV></DIV><H2 class="os-animation" data-os-animation="fadeIn" data-os-animation-delay="0s">Do our biobrick function?</H2><P class="margin_bottom">
        For the purpose of testing Biobricks function, we use two kit to detect the concentration of glutamate and glutamine; one is Glutamate Colorimetric Assay Kit for analyzing concentration of glutamate, and the other is Glutamine Colorimetric Assay Kit for
        analyzing concentration of glutamine.
      </P><DIV id="reference" class="os-animation" data-os-animation="fadeIn" data-os-animation-delay="0s"><DIV class="title"><H1>Reference</H1></DIV><H3>Sensing</H3><P>
          Lin, H. Y., Bledsoe, P. J., &amp; Stewart, V. (2007). &quot;Activation of <I>yeaR-yoaG</I> operon transcription by the nitrate-responsive regulator NarL is independent of oxygen-responsive regulator Fnr in <I>Escherichia coli</I> K-12.&quot; <I>Journal of bacteriology</I>,
          189(21), 7539-7548.
        </P><H3>Regulation</H3><P>
          [1] Steen, A., Ütkür, F. Ö., Borrero-de Acuña, J. M., Bunk, B., Roselius, L., Bühler, B., Jahn, D. &amp; Schobert, M. (2013). &quot;Construction and characterization of nitrate and nitrite respiring Pseudomonas putida KT2440 strains for anoxic biotechnical applications.&quot; <I>Journal of biotechnology, 163</I>(2), 155-165.
        </P><P>
          [2] van Heeswijk, W. C., Westerhoff, H. V., &amp; Boogerd, F. C. (2013). &quot;Nitrogen assimilation in Escherichia coli: putting molecular data into a systems perspective.&quot; <I>Microbiology and Molecular Biology Reviews, 77</I>(4), 628-695.
        </P><P>
          [3] Lundberg, J. O., Weitzberg, E., Cole, J. A., &amp; Benjamin, N. (2004). &quot;Nitrate, bacteria and human health.&quot; <I>Nature Reviews Microbiology, 2</I>(7), 593-602.
        </P></DIV></DIV><DIV id="sidemenu" class="col-md-2"><DIV class="list-group"><A onclick="scrollto('#sensing')" class="list-group-item">Sensing</A><A onclick="scrollto('#regulation')" class="list-group-item">Regulation</A><A onclick="scrollto('#reference')" class="list-group-item">Reference</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>