<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NCKU_Tainan_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NCKU Tainan/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><TITLE>MathJax AsciiMath Test Page</TITLE><DIV class="container-fluid"><DIV class="row"><DIV class="col"><DIV id="category" class="vertical-container"><H1 class="wet">Model</H1></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row"><DIV id="paragraph" class="paragraph col-md-8 col-md-offset-1 col-xs-offset-1 col-xs-10"><H2 id="abstract">
        Abstract
      </H2><P>
          In order to combine the sensing device with synthetic biotechnology, first of all, we us Simbiology software to built a theoretical model for our promoter P<SUB>yeaR</SUB> and describing the whole mechanism and reaction.
           Then we found out the concentration of each substance varied with time by Matlab, and approximate method was applied for choosing a proper model fitting with GFP fluorescence variation curve.
           To improve our sensing detection, we built a statistical model by nonlinear regression and calibration. Furthermore, we created analysis method for sensing data by our empirical model constructed with data from more than 150 experiments;
          this model is able to divide nitrate concentration into 5 sections.
        </P><H2 id="motivation">
        Motivation of Improving P<SUB>yeaR</SUB> Model
      </H2><P>
          After taking a look to the result of previous teams, we found out that it’s not enough if we only describe the sensing reaction of P<SUB>yeaR</SUB> with NsrR; so to realize the mechanism, the paper, <I>Activation of yeaR-yoaG Operon Transcription by the Nitrate-Responsive Regulator NarL Is Independent of Oxygen-Responsive Regulator Fnr in Escherichia coli K-12</I>,
          was referred. From figure 1. in the paper, no matter with or without O2, the influence of NsrR and NarL to NO<SUB>3</SUB><SUP>-</SUP> had not much difference.
           Instead, from Figure 2. the influence of NsrR and NarL to P<SUB>yeaR</SUB> promoter was significant, which was the reason for us to consider the effects of both NsrR and NarL while building a more complete model.
        </P><DIV class="row imagerow"><DIV class="col-12"><P>
              Figure 1.
            </P><P>
              Figure 2.
            </P></DIV></DIV><H2 id="pyear">
        P<SUB>yeaR</SUB> Mechanism
      </H2><P>
          There are Nap and NirK enzymes that can catalyze NO<SUB>3</SUB><SUP>-</SUP> to NO<SUB>2</SUB><SUP>-</SUP> and NO<SUB>2</SUB><SUP>-</SUP> to NO separately in our <I>E. coli</I> system. After paper searching, we found that the promoter's reaction
          was controlled by two gene representing two binding sites, one of which was NarL, and the other was NsrR. NarL is able to sense both nitrate and nitrite, promoting P<SUB>yeaR</SUB> to produce GFP further. On the other hand, NsrR has the ability
          to block RNA polymerase from binding with P<SUB>yeaR</SUB> promoter, but nitric oxide will release NsrR from the binding site, therefore allow transcription process to start.
        </P><H2 id="equation">
        Equations of Our Sensing Pathway Model
      </H2><P>
          NO<SUB>3</SUB><SUP>-</SUP> will be consumed in two ways, one of which is turning into NO<SUB>2</SUB><SUP>-</SUP> by Nap enzyme, and the other one is binding with NarL thus directly induce the transcription of GFP gene
           The rate of [NO<SUB>3</SUB><SUP>-</SUP>] can be expressed by:
        </P><P>
          NO<SUB>2</SUB><SUP>-</SUP> can be produced by Nap enzyme, being consumed by NarL and NirK enzyme, or induce the production of mRNA of GFP (mGFP).
           The rate of [NO<SUB>2</SUB><SUP>-</SUP>] can be expressed by:
        </P><P>
          NO can be produced by NirK enzyme.
           The rate of [NO] can be expressed by:
        </P><P>
          There are 3 major sources of mGFP production: one is from NO<SUB>3</SUB><SUP>-</SUP>, another is from NO<SUB>2</SUB><SUP>-</SUP>, and the other is from NO.
           Also, mGFP will be degraded naturally, which is also included in our model.
        </P><P>
          The rate of [mGFP] can be expressed by:
        </P><DIV id="p1-1" class="wrap-title" onclick="show('#p1-1','#p1-2')"><H2 style="color:#f19181 ;">
          How can we derive this term
      </H2></DIV><DIV id="p1-2" class="wrap-content"><H2>      For NsrR
        </H2><P>
            Two NO and NsrR combine to form a complex before they are functional. Therefore, transcription factors cooperatively regulate expression of genes.
             Consider some free inducer NO* that bind to two repressors NsrR, giving rise to a complex NsrR<SUB>2</SUB>NO:
          </P><P>
            2 [NsrR] + [NO]
            <MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math"><MOVER accent="false"><MO>↔</MO><MROW><MI>k</MI><MI>o</MI><MI>n</MI><MO>/</MO><MI>k</MI><MI>o</MI><MI>f</MI><MI>f</MI></MROW></MOVER></MATH>            [NsrR
            <SUB>2</SUB>NO]
          </P><P>
            a) rate of complex formation:
          </P><P style="text-align: center">
            k<SUB>on</SUB>[NO][NsrR]<SUB>2</SUB></P><P>
            b) rate of dissociation:
          </P><P style="text-align: center">
            k<SUB>off</SUB>[NsrR<SUB>2</SUB>NO]
          </P><P>
            c) The rate of complex formation at steady state:
          </P><P style="text-align: center;"><MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math"><MFRAC><MROW><MI mathvariant="normal">d</MI><MFENCED open="[" close="]" separators="|"><MROW><MSUB><MROW><MI mathvariant="normal">N</MI><MI mathvariant="normal">s</MI><MI mathvariant="normal">r</MI><MI mathvariant="normal">R</MI></MROW><MROW><MN>2</MN></MROW></MSUB><MI mathvariant="normal">N</MI><MI mathvariant="normal">O</MI></MROW></MFENCED></MROW><MROW><MI mathvariant="normal">d</MI><MI mathvariant="normal">t</MI></MROW></MFRAC><MO>=</MO></MATH>            =k
            <SUB>on</SUB>[NO][NsrR]<SUB>2</SUB>-k<SUB>off</SUB>[NsrR<SUB>2</SUB>NO]=0
          </P><P>
            For conservation equation, describing the total concentration can be obtained:
          </P><P style="text-align: center;">
            [NsrR<SUB>2</SUB>NO] + [NO*] = [NO<SUB>Total</SUB>]
          </P><P>
            NsrR dimerization with respect to unbound NsrR can be represented by the following equation by considering the degree of cooperativity of NO binding into account:
          </P><P>
            Therefore,
          </P><P>
            Finally, mGFP varied with time can be described into: (for only influenced by NO/NsrR effect)
          </P><P>
            More information can be looked for in the following references.
          </P></DIV><P>
          Finally, mRNA of GFP will be translated into GFP.
           The rate of [GFP] can be expressed by:
        </P><P style="text-align: center"><MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math"><MFRAC><MROW><MI>d</MI><MO>[</MO><MI>G</MI><MI>F</MI><MI>P</MI><MO>]</MO></MROW><MROW><MI>d</MI><MI>t</MI></MROW></MFRAC><MO>=</MO><MI>k</MI><MI>t</MI><MI>r</MI><MI>a</MI><MI>n</MI><MI>s</MI><MI>l</MI><MI>a</MI><MI>t</MI><MI>i</MI><MI>o</MI><MI>n</MI><MO>×</MO><MFENCED open="[" close="]" separators="|"><MROW><MI>m</MI><MI>G</MI><MI>F</MI><MI>P</MI></MROW></MFENCED><MO>-</MO><MSUB><MROW><MI>r</MI></MROW><MROW><MI>G</MI><MI>F</MI><MI>P</MI></MROW></MSUB><MO>×</MO><MFENCED open="[" close="]" separators="|"><MROW><MI>G</MI><MI>F</MI><MI>P</MI></MROW></MFENCED></MATH>------(5)
        </P><H3>
        Parameter Table
      </H3><DIV class="table-responsive"><TABLE class="table margin_bottom" style="padding-left: 30px;"><THEAD><TR><TH>
                Description
              </TH><TH>
                Value
              </TH><TH>
                Unit(SI)
              </TH></TR></THEAD><TBODY><TR><TD>
                [NO<SUB>3</SUB><SUP>-</SUP>] (100ppm)
              </TD><TD>
                Nitrate initial value
              </TD><TD>
                1.6x10<SUP>-6</SUP></TD><TD>
                mol/m<SUP>3</SUP></TD></TR><TR><TD>
                Km <SUB>(Nap)</SUB></TD><TD>
                the NO<SUB>3</SUB><SUP>-</SUP> at which the reaction rate is at half-maximum
              </TD><TD>
                8x10<SUP>-3</SUP></TD><TD>
                mol/m<SUP>3</SUP></TD></TR><TR><TD>
                Vmax<SUB>(Nap)</SUB></TD><TD>
                Maximum velocity of Nap
              </TD><TD>
                4.7x10<SUP>-1</SUP></TD><TD>
                mol/s x m<SUP>3</SUP></TD></TR><TR><TD>
                Km<SUB>(NirK)</SUB></TD><TD>
                the NO<SUB>2</SUB><SUP>-</SUP> at which the reaction rate is at half-maximum
              </TD><TD>
                2.5x10<SUP>-1</SUP></TD><TD>
                mol/m<SUP>3</SUP></TD></TR><TR><TD>
                Vmax<SUB>(NirK)</SUB></TD><TD>
                Maximum velocity of NirK
              </TD><TD>
                5x10<SUP>-3</SUP></TD><TD>
                mol/s x m<SUP>3</SUP></TD></TR><TR><TD>
                [P<SUB>yeaR</SUB>]
              </TD><TD>
                Concentration of P<SUB>yeaR</SUB></TD><TD>
                10<SUP>-10</SUP></TD><TD>
                mol/m<SUP>3</SUP></TD></TR><TR><TD>
                k<SUB>transcription</SUB></TD><TD>
                Rate of mGFP synthesis
              </TD><TD>
                1.8x10<SUP>-5</SUP></TD><TD>
                1/s
              </TD></TR><TR><TD>
                k<SUB>fNo3</SUB></TD><TD>
                Related constant of NO<SUB>3</SUB><SUP>-</SUP> and NarL
              </TD><TD>
                3x10<SUP>-4</SUP></TD><TD>
                1/s
              </TD></TR><TR><TD>
                k<SUB>fNo2</SUB></TD><TD>
                Related constant of NO<SUB>2</SUB><SUP>-</SUP> and NarL
              </TD><TD>
                6x10<SUP>-5</SUP></TD><TD>
                1/s
              </TD></TR><TR><TD><MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math"><MSUB><MROW><MI>r</MI></MROW><MROW><MI>m</MI><MI>G</MI><MI>F</MI><MI>P</MI></MROW></MSUB></MATH></TD><TD>
                mGFP degradation rate
              </TD><TD>
                5x10<SUP>-5</SUP></TD><TD>
                1/s
              </TD></TR><TR><TD><MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math"><MSUB><MROW><MI>r</MI></MROW><MROW><MI>G</MI><MI>F</MI><MI>P</MI></MROW></MSUB></MATH></TD><TD>
                GFP degradation rate
              </TD><TD>
                2.5x10<SUP>-6</SUP></TD><TD>
                1/s
              </TD></TR><TR><TD>
                k<SUB>translation</SUB></TD><TD>
                Rate of GFP synthesis
              </TD><TD>
                4x10<SUP>-4</SUP></TD><TD>
                1/s
              </TD></TR><TR><TD>
                k<SUB>d(NsrR)</SUB></TD><TD>
                Dissociation constant of NsrR
              </TD><TD>
                3.5x10<SUP>-6</SUP></TD><TD>
                m<SUP>3</SUP>/mol
              </TD></TR><TR><TD>
                k<SUB>NO</SUB></TD><TD>
                Dissociation constant of NO
              </TD><TD>
                1.4x10<SUP>-4</SUP></TD><TD>
                mol/m<SUP>3</SUP></TD></TR><TR><TD>
                [NsrR]
              </TD><TD>
                Concentration of NsrR
              </TD><TD>
                10<SUP>-6</SUP></TD><TD>
                mol/m<SUP>3</SUP></TD></TR></TBODY></TABLE></DIV><H2 id="simulation">
        Simulation
      </H2><P>
          Simbiology of Matlab is used to simulate the model:
        </P><H2 id="GFP">
        Get a Function to Describe GFP varied with time
      </H2><P>
          By taking the parameters into the (1)~(5) equations, the [GFP] can be described by:
        </P><P style="font-size: 18px;text-align:center;">
          GFP(t)=
          <SPAN style="color:#00b3ea;">(2.9x10<SUP>-8</SUP>)e<SUP>-2.5x10<SUP>-6t</SUP></SUP></SPAN> -
          <SPAN style="color:#f9cb8f;">(3.08x10<SUP>-8</SUP>)e<SUP>-4.485x10<SUP>-4t</SUP></SUP></SPAN> +
          <SPAN style="color:#f19181;">(6.06x10<SUP>-10</SUP>)e<SUP>-2x10<SUP>-2t</SUP></SUP></SPAN> -
          <SPAN style="color:#3cbac8;">(1.48x10<SUP>-9</SUP>)e<SUP>-1.74x10<SUP>-2t</SUP></SUP></SPAN></P><P class="margin_bottom" style="text-align: center;">
          (This equation is for initial concentration of nitrate 100ppm)
        </P><H2 id="fitting">
        The Fitting Results
      </H2><P>
          As to know each term how to influence GFP, we divide GFP(t) into 4 parts:
        </P><OL id="ollist"><LI><SPAN style="color:#00b3ea;">A(t)=(2.9x10<SUP>-8</SUP>)e<SUP>-2.5x10<SUP>-6t</SUP></SUP></SPAN></LI><LI><SPAN style="color:#f9cb8f;">B(t)=(3.08x10<SUP>-8</SUP>)e<SUP>-4.485x10<SUP>-4t</SUP></SUP></SPAN></LI><LI><SPAN style="color:#f19181;">C(t)=(6.06x10<SUP>-10</SUP>)e<SUP>-2x10<SUP>-2t</SUP></SUP></SPAN></LI><LI><SPAN style="color:#3cbac8;">D(t)=(1.48x10<SUP>-9</SUP>)e<SUP>-1.74x10<SUP>-2t</SUP></SUP></SPAN></LI></OL><P>
          Obviously, we easily know the influence within 2 hours of terms A and B are more essential than that of terms C and D. Consequently, we modify equation to be simpler, which neglact C and D terms, to fit our data getting from experiments with our device
          and with powder.
          By using general model Exp2:
        </P><P style="text-align: center"><MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math"><MI mathvariant="normal">f</MI><MFENCED separators="|"><MROW><MI mathvariant="normal">x</MI></MROW></MFENCED><MO>=</MO><MI mathvariant="normal">d</MI><MO>×</MO><MSUP><MROW><MI mathvariant="normal">e</MI></MROW><MROW><MI mathvariant="normal">g</MI><MI mathvariant="normal">x</MI></MROW></MSUP><MO>+</MO><MI>h</MI><MO>×</MO><MSUP><MROW><MI>e</MI></MROW><MROW><MI>j</MI><MI>x</MI></MROW></MSUP></MATH></P><H3>
        Coefficients (with 95% confidence bounds):
      </H3><P style="text-align: center;">
          Coefficients Table
        </P><DIV class="table-responsive"><TABLE class="table"><THEAD><TR><TH>
                  value
                </TH><TH>
                  min
                </TH><TH>
                  max
                </TH></TR></THEAD><TBODY><TR><TD>
                  d
                </TD><TD>
                  2.8x10<SUP>3</SUP></TD><TD>
                  2.8x10<SUP>3</SUP></TD><TD>
                  2.9x10<SUP>3</SUP></TD></TR><TR><TD>
                  g
                </TD><TD>
                  2x10<SUP>-5</SUP></TD><TD>
                  1.8x10<SUP>-5</SUP></TD><TD>
                  2.2x10<SUP>-5</SUP></TD></TR><TR><TD>
                  h
                </TD><TD>
                  -3.9x10<SUP>2</SUP></TD><TD>
                  -4.3x10<SUP>2</SUP></TD><TD>
                  -3.6x10<SUP>2</SUP></TD></TR><TR><TD> j
                </TD><TD>
                  -5.6x10<SUP>-4</SUP></TD><TD>
                  -6.4x10<SUP>-5</SUP></TD><TD>
                  -4.5x10<SUP>-5</SUP></TD></TR></TBODY></TABLE></DIV><P style="text-align: center">
          Goodness of fit:
           SSE: 5.624e+04
           R-square: 0.9953
           Adjusted R-square: 0.9952
           RMSE: 15.41
        </P><P style="text-align: center">
          GFP<MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math"><MFENCED separators="|"><MROW><MI mathvariant="bold">t</MI></MROW></MFENCED><MO>=</MO><MN> 2846</MN><MO>×</MO><MSUP><MROW><MI mathvariant="bold">e</MI></MROW><MROW><MN>0.00002</MN><MI mathvariant="bold">t</MI></MROW></MSUP><MO>-</MO><MN>390.5</MN><MO>×</MO><MSUP><MROW><MI mathvariant="bold-italic">e</MI></MROW><MROW><MO>-</MO><MN>-0.000558</MN><MI mathvariant="bold-italic">t</MI></MROW></MSUP></MATH>          (for 100 ppm)
        </P><H2 id="statistical">
        Statistical Model
      </H2><P>
          We randomly set 15 ppm of concentration of nitrate as the separating level of clean and polluted water. Hence, we dichotomized the concentration of nitrate into binary data (the outcomes of Bernoulli trials). Then, in order to fit a general linear model
          we use the concentration of nitrate, time as explain variables, and the electrical signals collected from our device as response variables.
          Here is the regression equation:
        </P><TABLE class="table center-block" style="margin-bottom: 100px;padding-left: 30px"><TBODY><TR><TD>
                Y<SUB>ij</SUB></TD><TD>
                Electrical signals collected from i <SUP>th</SUP> time series data with j<SUP>th</SUP> second.
              </TD></TR><TR><TD>
                T<SUB>ij</SUB></TD><TD>
                Time of i<SUP>th</SUP> time series data with j<SUP>th</SUP> second.
              </TD></TR><TR><TD>
                X<SUB>i</SUB> = 1
              </TD><TD>
                When the concentration of nitrate above 15 ppm.
              </TD></TR><TR><TD>
                X<SUB>i</SUB> = 0
              </TD><TD>
                When the concentration of nitrate below 15 ppm.
              </TD></TR><TR><TD>ε<SUB>ij</SUB></TD><TD>
                Random error term of i<SUB>th</SUB> time series data with j<SUB>th</SUB> second.
              </TD></TR></TBODY></TABLE><P>
          After that, we use calibration to forecast the concentration of nitrate within the time series data with our model by minimizing the sum of square of time series data from 1<SUP>st</SUP> second to 1800<SUP>th</SUP> second. The training sensitivity
          and specificity are shown in the table below.
        </P><H2 id="empirical">
        Empirical Model
      </H2><P>
          In order to build an empirical model for our sensing boat, we had done more than 150 experiments to set up our database.
 We chose 5 intervals:
        </P><P style="text-align: center;">
          0 – 4 ppm
           4 – 10 ppm
           10 – 20 ppm
           20 – 60 ppm
           Over 60 ppm
          </P><P>
          And our sensing device only need to detect the Optical signal on 5 min, 10 min, 15 min, 20 min. By using this method, we can easily, quickly and precisely distinguish the concentration into this 5 intervals.
        </P><H2 id="conclusion">
        Conclusion
      </H2><P class="margin_bottom">
          Not only to build a more complete model, according to species and application in the reality, we can also set different intervals by empirical method and any separating level by statistical method. With this advance, we are able to get the results within
          20 to 30 minutes, which is quicker and more precise as well.
        </P><H2 id="references">
        References
      </H2><P style="margin-bottom: 25px;">
          [1] Lin, H. Y., Bledsoe, P. J., &amp; Stewart, V. (2007) &quot;Activation of yeaR-yoaG operon transcription by the nitrate-responsive regulator NarL is independent of oxygen-responsive regulator Fnr in Escherichia coli K-12.&quot; <I>Journal of bacteriology, 189</I>(21),
          7539-7548.
        </P><P style="margin-bottom: 25px;">
          [2] Csicsery, N. &amp; O’Laughlin, R. (2013) &quot;A Mathematical Model of a Synthetically Constructed Genetic Toggle Switch.&quot; <I>BENG 221 – Mathematical Methods in Bioengineering</I></P><P style="margin-bottom: 25px;">
          [3] Nohno, T., Noji, S., Taniguchi, S., &amp; Saito, T. (1989) &quot;The narX and narL genes encoding the nitrate-sensing regulators of Escherichia coli are homologous to a family of prokaryotic two-component regulatory genes.&quot; <I>Nucleic acids research, 17</I>(8),
          2947-2957.
        </P><P style="margin-bottom: 25px;">
          [4] Partridge, J. D., Bodenmiller, D. M., Humphrys, M. S., &amp; Spiro, S. (2009) &quot;NsrR targets in the Escherichia coli genome: new insights into DNA sequence requirements for binding and a role for NsrR in the regulation of motility.&quot; <I>Molecular microbiology, 73</I>(4),
          680-694.
        </P><P style="margin-bottom: 25px;"><A href="https://2016.igem.org/Team:Peshawar/Model">[5] Peshawar iGEM team (2016)</A></P><P style="margin-bottom: 25px;"><A href="https://2012.igem.org/Team:NYMU-Taipei/ymim2">[6] NYMU-Taipei IGEM team (2012)</A></P><P style="margin-bottom: 25px;"><A href="https://2015.igem.org/Team:HKUST-Rice/Nitrate_Sensor_PyeaR">[7] HKUST-Rice IGEM team (2015)</A></P></DIV><DIV id="sidemenu" class="col-md-2"><DIV class="list-group"><A onclick="scrollto('#abstract')" class="list-group-item">Abstract</A><A onclick="scrollto('#motivation')" class="list-group-item">Motivation of Improving P<SUB>yeaR</SUB> Model</A><A onclick="scrollto('#pyear')" class="list-group-item">P<SUB>yeaR</SUB> Mechanism</A><A onclick="scrollto('#equation')" class="list-group-item">Equations of Our Sensing Pathway Model</A><A onclick="scrollto('#simulation')" class="list-group-item">Simulation</A><A onclick="scrollto('#GFP')" class="list-group-item">Get a Function to Describe GFP varied with time</A><A onclick="scrollto('#fitting')" class="list-group-item">The Fitting Results</A><A onclick="scrollto('#statistical')" class="list-group-item">Statistical Model</A><A onclick="scrollto('#empirical')" class="list-group-item">Empirical Model</A><A onclick="scrollto('#conclusion')" class="list-group-item">Conclusion</A><A onclick="scrollto('#references')" class="list-group-item">References</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>