<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_BIOSINT_Mexico_Sensor"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:BIOSINT_Mexico/Sensor">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:BIOSINT_Mexico/Sensor&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:BIOSINT_Mexico/Sensor&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:BIOSINT_Mexico/Sensor&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:BIOSINT_Mexico/Sensor" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:BIOSINT Mexico/Sensor</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="navmenubkg"><DIV id="navmenubox"><UL id="nav"><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico" title="Project Overview">HOME</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Project">PROJECT</A></LI><LI style="margin-left:14.5px"><A href="https://2014.igem.org/Team:BIOSINT_Mexico/overview" title="Project Overview">OVERVIEW</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Chassis">CHASSIS</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Bioaccumulation" title="Bioaccumulation">BIOACCUMULATION</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Sensor">SENSOR</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Switch" title="Red Light Switch">LIGHT SWITCH</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Assembly" title="Assembly">ASSEMBLY</A></LI></UL></DIV><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/safety">SAFETY</A></LI><LI style="margin-left:141px;"><A href="https://igem.org/Safety/Safety_Form?team_id=1471" title="Safety form">SAFETY FORM</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Garden" title="Garden">GARDEN</A></LI></DIV><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Policyandpractices" title="Policy&amp;Practices">PRACTICES</A></LI><LI style="margin-left: 300px;"><A href="https://2014.igem.org/Team:BIOSINT_Mexico/art" title="Art">ART</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/symposium" title="Symposium">SYMPOSIUM</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/collaboration" title="Collaboration">COLLABORATION</A></LI></DIV><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/team" title="TEAM">TEAM</A></LI><LI style="margin-left:300px;"><A href="https://2014.igem.org/Team:BIOSINT_Mexico/members" title="Members">MEMBERS</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Sponsors" title="Sponsors">SPONSORS</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Gallery" title="Gallery">GALLERY</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Atributions" title="Atributions">ATRIBUTIONS</A></LI><LI><A href="https://igem.org/Team.cgi?id=1471" title="Official page">OFFICIAL PAGE</A></LI></DIV><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Achievements" title="Achievements">ACHIEVEMENTS</A></LI><LI style="margin-left:300px"><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Parts">PARTS</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Results" title="Results">RESULTS</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Modeling" title="Modeling">MODELING</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Judging" title="Judging">JUDGING</A></LI></DIV><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/notebook" title="Notebook">NOTEBOOK</A></LI><LI style="margin-left:649.5px"><A href="https://2014.igem.org/Team:BIOSINT_Mexico/protocols" title="Protocols">PROTOCOLS</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/lab records" title="Lab records">LAB RECORDS</A></LI><H1>Modular Sensor </H1><H2>Description</H2><P>
There has been an increasing necessity to implement new, efficient and inexpensive techniques for the identification of biological and chemical agents that contaminate the environment, one of the most developed strategies for solving this trouble is the use of biosensors. A reporter device must be an easily detectable mechanism for sensing a specific substance of interest, this reporter needs to have a monitoring and a resettable capacity. 
</P><P>In this project the target pollutant that will induce gene expression of the biological sensor is the mercury, this is based on the loss of green pigmentation in <I>Arabidopsis thaliana</I>. A substantially faster loss of chlorophyll is needed if is used as a reporter system for a plant sentinel. Chlorophyll loss in plants is normally a slow process that occurs during the complex mechanism of senescence. The half-life of chlorophyll has been estimated to be 2–5 days for relatively mature and fully greened leaves (Stobart and Hendry, 1984), visual perception of chlorophyll loss in leaves can take longer.
</P><P>The process for whitening the plant requires following the method explained below  and takes around 48 hours in order to see a complete change in color. In order to accomplish this, it is necessary to use specific enzymes which degrades the chlorophyll that is already in the plant and also a doubled stranded RNA which inhibits the production of an important molecule involved in the production of chlorophyll (Medford, J. et al , 2006).
</P><H3><SPAN class="mw-headline" id="Biosynthesis_and_Breakdown">Biosynthesis and Breakdown</SPAN></H3><P>The first phase of the chlorophyll biosynthesis starts with the glutamic acid, after nine chemical steps this amino acid produces a four ring structure, called protoporphyrin  IX. A molecule of magnesium is added to the ring structure by the magnesium chelatase, through two more steps this is converted in monovinyl protochlorophyllide and is reduce to chlorophyllide a. by the enzyme protochlorophyllide oxidoreductase (POR). The chlorophyllide a. is transformed in darker green chlorophyll by the chlorophyll synthetase enzyme, this add a 20 carbon phytol tail. 
</P><P>The chlorophyll pathway degradation is an important catabolic process for the senescence of the leaf. This breakdown pathway starts with the chlorophyllase enzyme, which removes the  hydrophobic twenty carbon phytol tail from the chlorophyll. As the synthetic pathway, the chlorophyll turn into the light green molecule called chlorophyllide. This molecule is converted to pheophorbide a. by the magnesium dechelatase enzyme at removing the magnesium and the red chlorophyll catabolite (RCC) is formed aside pheophorbide  a oxygenase. Then the RCC reductase produces fluorescent chlorophyll catabolite (FCC). The FCC goes through different steps and its converted into nonfluorescent chlorophyll catabolites.
</P><H3><SPAN class="mw-headline" id="De-greening">De-greening</SPAN></H3><P>Since the chlorophyll degradation is a slow process, a synthetic system that removes and degrades chlorophyll in a fast and efficient way is needed. For this reason we used a de-greening circuit, that allows the regulation of the chlorophyll breakdown  and the response to a specific input. Our system detects heavy metals,  and in this case we used methylmercury, which is transcriptionally linked to the de-greening system. 
</P><P>One of the solutions for accelerating  the breakdown  is  an inducible gene silencing system using interference RNA. The silencing gene used was phytoene desaturase( PDS). This  enzymes produced by PDS  blocks carotenoids synthesis, culminating in a photobleaching phenotype  given by the photo-oxidation of chlorophylls. 
</P><P>If a characteristic white phenotype is going to be used to identify methylmercury presence, it must be much faster than previous RNAi circuits. Medford, J et al (2006) proposed the combination of the following regulatory circuits for de-greening:
</P><P><B>a) Stop synthesis circuit:</B> induction of diRNA to reduce protochlorophyllide oxidoreductase (POR) or diRNA to lessen GENOMES UNCOUPLED 4 (GUN4). For the project we only use GUN4, this is a single copy gene that regulates chlorophyll biosynthesis by activating magnesium chelatase, a key enzyme complex that produces magnesium protoporphyrin IX, basic structure for chlorophyll. With a double-stranded interfering RNA  construct designed for each one of them and placed under the control of certain promoter,<I> A. thaliana</I> demonstrates chlorophyll loss. (Antunes et. al 2006).  
</P><P><B>b) Initiate breakdown circuit:</B> induction of chlorophyllase (CHLASE) and red chlorophyll catabolite reductase (RCCR) or CHLASE and pheophorbide a oxygenase (PAO).Chlorophyll breakdown involves a series of enzymatic steps. Key processes are the hydrophobic tail removal by CHLASE, red chlorophyll catabolite reductase (RCCR) and porphyrin ring cleavage by PAO.
</P><P>By combining both processes the de-greening is easily recognized within 24-48 hrs of induction. Also we added two more miRNA for the de-greening circuit: one of the miRNA it's for the RBS and the other it's for the IRES, both of them are in the system of the chlorophyll breakdown.
</P><H3><SPAN class="mw-headline" id="Inducible_promoter_by_Hg.28II.29">Inducible promoter by Hg(II)</SPAN></H3><P>For the activation of the degreening device the use of an inducible promoter is needed. Since the project is about bioremediation of methylmercury, we applied the double promoter of the mercury resistance operon of the transposon Tn21. These are two overlapping divergent oriented promoters, the first one regulates the MerR protein and  it’s called PR, the second it’s called PTPCAD and it regulates the gene transcription. The set of these promoters it’s called MerOP . 
</P><DL><DD><BLOCKQUOTE><FONT size="3"><DIV class="thumb tright"><DIV class="thumbinner" style="width:268px;"><DIV class="thumbcaption"><B>Figure 2</B> Structure of MerR<I>.</I></DIV></DIV></DIV></FONT></BLOCKQUOTE></DD></DL><P>The substance that activates and regulates merOP promoter is the Hg(II). The PTPCAD expression is repressed in the absence of Hg (II) by the merR gene, this merR product is used as a regulator. Meanwhile the expression of  PR doesn’t change in the presence or absence of this molecule. MerR always represses its own expression  so  it doesn’t matter of there is presence of Hg(II). 
</P><P>
MerR functions as a homodimer and it’s always bound to the MerOP. This has an extremely affinity for Hg(II) due to its unique ability to coordinate that molecule in a trigonal planar configuration. MerR has three cysteine residues (Cys118, Cys126, Cys82’) that comprise the metal binding sites, which are located at each end of coiled coil.
</P><P><B>The action of MerR at MerOP region</B></P><P>
MerR make a bound with the DNA and RNA polymerase, is recruited but not able to form an open complex. There will be a conformational change induced by Hg(II), this will bind the ternary complex and will initiate the transcription of the genes. 
</P><P>
In the absence of Hg(II) and MerR, the RNA polymerase will transcribe from the merR promoter, and this will increase the concentration of MerR in the cell. When MerR finally binds to the MerOP region, the transcription of the merR promoter is going to be repressed and consequently the DNA will bent and unfold at the MerOP sequence. The RNA polymerase will attach to the mer promoter, forming a complex between the DNA, MerR and RNA polymerase. 
</P><P>In the absence of Hg(II) and  presence of MerR,  this protein is going to bound the DNA in the repressor conformation, maintaining the repression of the promoter. The Hg(II) will bind to one of the two binding sites on the MerR, causing a conformational change to put MerR in the activation conformation. This will cause the DNA distortion at the centre of the operator, unwinding of DNA and straightening of the helix. The reorientation of the sequences so caused, allows them to interact with the RNA polymerase and initiate the transcription. 
</P><DL><DD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Figure 3</B> Action of MerR<I>.</I></DIV></DIV></DIV></DIV></DD></DL><H2>Modeling</H2><P><B>Equations for de-greening regulatorsystem</B></P><P>An inducible two way promoter is connected downstream to a de-greening system and upstream to a regulator from the mer operon. This regulator, called merR, lead the expression of the PmerOT promoter. 
In normal conditions, PmerOT promoter is induced by mercury in the cell and transcribes downstream. However, when HG is absent, PmerOT transcribes upstream and express the protein merR, which also works as an activator-repressor for PmerOT.
PmerOT repressed the expression upstream in a negative feedback. Reaction coul be expressed as:
</P><P>Where, merR is in charge of inhibiting the transcription rate.
The negative feedback reaction can be expressed as: 
</P><P>Being f (merR), the predictive function for the transcription rate. From the Hill’s function, it can be inferred for a repressed promoter that:
</P><P>Then, the function for merR messenger is given by:
</P><P>Thus, the function behavior will be cyclic, autoregulating itself.
</P><P><B>Equations for the downstream de-greening system</B>Downstream, PmerOT promoter will be activated by metallic mercury. Thus, the activity for the promoter downstream is given by the Hill’s function: 
</P><P>Where DPC, represents the De-greening protein complex. Starting from this, we can infer the concentration of the construct by the equations:
</P><P>Chalase:
</P><P>RCCR: 
</P><P>PAO:
</P><P>And for GUN4:
</P><H2>Results</H2><P>Due to the fact that an extended phase of de-greening can cause severe damage to the plant cells, is necessary to stop its action. In order to accomplish this we designed a system of miARN only for the de-greening system.
</P><P>The miARN where designed using a free online software called WMD3 – Web microARN Designer, this program is especially designed for creating miARN for plants, it includes a huge library that contains a great quantity of information for eukaryote organisms. 
</P><P>To stop the mechanism of de-greening it was necessary three specific miARN, 1 for the first RBS1 of the sequence, 1 for IRES (internal ribosome entry site) which is placed between the genes involved in the process as shown in the construct and 1 for GUN4.
</P><P>The miARN for RBS1: GAATTCGCGGCCGCTTCTAGAGTAGGGCGGCGGTGGTACCTCTACTAGTAGCGGCCGCTGCAG.
</P><P>The miARN for IRES: GAATTCGCGGCCGCTTCTAGAGTAAACAGAAGTGCTTGTTCGGTACTAGTAGCGGCCGCTGCAG.
</P><P>The miARN for GUN4: GAATTCGCGGCCGCTTCTAGAGTAACGTCGAATATGGTCGCAGTACTAGTAGCGGCCGCTGCAG.
</P><H2>References</H2><P>Brown, N. L., Stoyanov, J. V., Kidd, S. P., &amp; Hobman, J. L. (2003). The MerR family of transcriptional regulators. FEMS microbiology reviews, 27(2‐3), 145-163.
Park, S. J., Wireman, J., &amp; Summers, A. O. (1992). Genetic analysis of the Tn21 mer operator-promoter. Journal of bacteriology, 174(7), 2160-2171.
Antunes, M. S., Ha, S. B., Tewari‐Singh, N., Morey, K. J., Trofka, A. M., Kugrens, P., ... &amp; Medford, J. I. (2006). A synthetic de‐greening gene circuit provides a reporting system that is remotely detectable and has a re‐set capacity. Plant biotechnology journal, 4(6), 605-622.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:BIOSINT_Mexico/Sensor">http://2014.igem.org/Team:BIOSINT_Mexico/Sensor</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:BIOSINT_Mexico/Sensor" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:BIOSINT_Mexico/Sensor" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:BIOSINT_Mexico/Sensor&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:BIOSINT_Mexico/Sensor&amp;oldid=401756" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>