<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_BIOSINT_Mexico_Assembly"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:BIOSINT_Mexico/Assembly">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:BIOSINT_Mexico/Assembly&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:BIOSINT_Mexico/Assembly&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:BIOSINT_Mexico/Assembly&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:BIOSINT_Mexico/Assembly" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:BIOSINT Mexico/Assembly</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="navmenubkg"><DIV id="navmenubox"><UL id="nav"><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico" title="Project Overview">HOME</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Project">PROJECT</A></LI><LI style="margin-left:14.5px"><A href="https://2014.igem.org/Team:BIOSINT_Mexico/overview" title="Project Overview">OVERVIEW</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Chassis">CHASSIS</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Bioaccumulation" title="Bioaccumulation">BIOACCUMULATION</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Sensor">SENSOR</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Switch" title="Red Light Switch">LIGHT SWITCH</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Assembly" title="Assembly">ASSEMBLY</A></LI></UL></DIV><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/safety">SAFETY</A></LI><LI style="margin-left:141px;"><A href="https://igem.org/Safety/Safety_Form?team_id=1471" title="Safety form">SAFETY FORM</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Garden" title="Garden">GARDEN</A></LI></DIV><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Policyandpractices" title="Policy&amp;Practices">PRACTICES</A></LI><LI style="margin-left: 300px;"><A href="https://2014.igem.org/Team:BIOSINT_Mexico/art" title="Art">ART</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/symposium" title="Symposium">SYMPOSIUM</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/collaboration" title="Collaboration">COLLABORATION</A></LI></DIV><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/team" title="TEAM">TEAM</A></LI><LI style="margin-left:300px;"><A href="https://2014.igem.org/Team:BIOSINT_Mexico/members" title="Members">MEMBERS</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Sponsors" title="Sponsors">SPONSORS</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Gallery" title="Gallery">GALLERY</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Atributions" title="Atributions">ATRIBUTIONS</A></LI><LI><A href="https://igem.org/Team.cgi?id=1471" title="Official page">OFFICIAL PAGE</A></LI></DIV><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Achievements" title="Achievements">ACHIEVEMENTS</A></LI><LI style="margin-left:300px"><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Parts">PARTS</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Results" title="Results">RESULTS</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Modeling" title="Modeling">MODELING</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Judging" title="Judging">JUDGING</A></LI></DIV><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/notebook" title="Notebook">NOTEBOOK</A></LI><LI style="margin-left:649.5px"><A href="https://2014.igem.org/Team:BIOSINT_Mexico/protocols" title="Protocols">PROTOCOLS</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/lab records" title="Lab records">LAB RECORDS</A></LI><H1> ASSEMBLY </H1><P>
The assembly of our work is explained in this section. We briefly explain the interactions of our individual gene circuits and then we tie the 3 gene circuits of  the Mercury Bioaccumulation , Degreening and the Reset - Red light switch mechanisms. 
</P><H2> Mercury Bioaccumulation</H2><P>
The genetic circuit that makes the mercury bioaccumulation process consists of several gene constructs. 
</P><H3><SPAN class="mw-headline" id="MerE_and_Mer_B_assembly_construct.">MerE and Mer B assembly construct.</SPAN></H3><P>The production of MerE and MerB are fundamental and maybe one of the most important proteins in the project that need to be synthesized in order to achieve   the detection, transportation and reduction of mercury into a less toxic compound. 
</P><P>For this reason we have designed Gene construct NO. 1 in order to produce these important elements of our genetic circuit. 
</P><BLOCKQUOTE><FONT size="3"><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:632px;"><DIV class="thumbcaption"><B>Figure 1</B> MerE and MerB construct.</DIV></DIV></DIV></DIV></FONT></BLOCKQUOTE><P>Since our chassis is Arabidopsis thaliana the use of a eukaryotic promoter was needed in order to start transcription. We used a commonly used viral promoter for plants, known as the Cauliflower mosaic virus (CamV), as a constitutional promoter that has shown high protein expression in plants. 
We then added a ribosomal binding site (rbs) in order to express our merE gene. The expression of merE will lead to the production of a merE transposon protein that is important for transporting Ch3Hg+ inside the cell, an essential part of our detection system. To stop the transcription of merE a terminator was added. 
</P><P>The production of another important part of the mer Operon was designed and constructed in the same plasmid as the merE gene. 
</P><P>Since the CamV promoter previously initiated transcription an rbs  was added upstream in order to start the transcription of  the merB gene. Once the merB gene was synthesized protein synthesis was stoped. 
</P><P>(The function of Mer E and MerB are explained with further detail in the Mercury Bioaccumulation section of our wiki.)
</P><P>Once Mer E is translated into a transposon protein the fun begins and we can move to our other gene construct, famously known as Gene construct NO.2   or our reporter construct.
</P><H3>Reporter Construct </H3><P>
In the past, a broad range of Biosensors have been made throughout the igem competition and bacteria that produce color pigmentations have been one of the most outstanding biosensor ideas so far, because they are a visible way to detect the presence of a target compound. (Check out The Cambridge 2009 iGEM team for more info!)
It is just an easier way and a more effective method to use colors visible to the human eye. 
Saying this the identification of mercury without sophisticated equipment is important if a good biosensor is to be created. 
</P><P>Nevertheless the production of a new pigment by a plant seems like a more complex idea, because a lot of factors affect the color of a plant, and altering its normal biosynthesis pathways carried out in this process could affect negatively the plant.
</P><P>However the inhibition and production of the natural pigments found in plants seem like a better option. Since chlorophyll is the most common pigment in plants we will use this pigment as our reporter signal.  
</P><P>We have designed construct 2 in order to inhibit the production of chlorophyll in the plant.  The inhibition of chlorophyll will make our plant turn white, making a visible reporter signal indicating the presence of Mercury. 
</P><P>Now that the role of chlorophyll has been explained its time to tie it up with our pals merE and merB previously explained. 
</P><H2>Degreening process </H2><BLOCKQUOTE><FONT size="3"><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:632px;"><DIV class="thumbcaption"><B>Figure 2</B> Degreening construct.</DIV></DIV></DIV></DIV></FONT></BLOCKQUOTE><P>In this construct we can see the presence of the merE and the merB gene of the mercury resistant operon that were previously synthesized.  
This merE protein has, as it was mentioned before, the function to import the methylmercury inside the cell. Once methylmercury is inside, the merB protein will cleavage the carbon and mercury bond forming Hg2 molecules, which are a less toxic compound for the plant. 
</P><P>The degreening system in this plasmid starts with the PmerOT inducible promoter. 
The special characteristic of this promoter is that it is a tightly overlapped, divergently oriented promoter. In other words this promoters transcribes in opposite directions, both upstream and downstream. (A detailed explanation of this promoter is explained in the Sensor section of the wiki).
</P><P>In the downstream transcription the PmertOT codifies the MerR gene and is auto regulated and auto induced through a negative feedback loop. 
</P><P>How ever the upstream transcription of this promoter is not carried out normally.
In order to start the upstream transcription of this promoter the presence of Hg2 is needed. The merR protein won’t induce the upstream capacity of the Pmert promoter by itself unless a Hg2/merR complex is made. 
Once the Hg2 /merR complex is made the Pmert promoter is induced to transcribe in a 3’ to 5’ direction hence starting the transcription of the rest of the genes in the circuit in the upstream direction.
</P><P>If the Hg2/merR complex is not made, the rest of the genes will not be transcribed and the MerR will just autoregulate itself. How ever if this happens it means that there is no mercury present in the cell and the degreening process will never occur. 
</P><P>Nonetheless if Hg2 is present, the regulatory circuits and protein synthesis necessary for the degreening will occur. 
</P><P>The genes for the chalase, RCCR, PAO and the single interference microRna were placed in front of internal ribosome entry site, abbreviated IRES, in order to achieve a greater process of protein synthesis. 
</P><P>These 4 genes have the following  primary functions: 
</P><P>Chalase:  Takes part in the inhibition of the chlorophyll
</P><P>RCCR: Reduces red chlorophyll 
</P><P>PAO:  Degrades chlorophyll by doing a cleavage in the porphyrin ring
</P><P>MicroRna:  GUN 4 is a gene that works as a chlorophyll regulator. In our project we inhibit Gun 4 with  interference microRNA that works to inhibit the synthesis of Gun 4
</P><P>So far the mechanisms for the detection, transportation, reduction and the degreening process have been explained. 
Until this part of the project the system is designed to act as a biosensor that reduces methyl mercury into a less toxic compound.
How ever the degreening process that occurs in order to report the presence of methyl mercury compromises the wellbeing of the plant because chlorophyll is inhibited. Since chlorophyll is vital for photosynthesis the absence of this compound would kill the plant. 
</P><P>In order to prevent our plant from dying we designed an on and off switch for the degreening mechanism. Once the degreening process has occurred this switch will help us to stop the production of chlorophyll inhibitory molecules, reactivating chlorophyll production in the plant. 
(for a detailed explanation of the degreening process check our Biosensor section  in our wiki) 
</P><H2><SPAN class="mw-headline" id="Reset_-_Red_light_switch">Reset - Red light switch</SPAN></H2><BLOCKQUOTE><FONT size="3"><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:632px;"><DIV class="thumbcaption"><B>Figure 3</B> Red light switch construct.</DIV></DIV></DIV></DIV></FONT></BLOCKQUOTE><P>
In order to stop the degreening process we incorporate a red light-responsive gene expression system using far red (740 nm) light wave. This far red light sensor works as a switch that inactivates the expression of the de-greening system genes, and is based on the interaction between the Phytochrome B and the phytochrome-interacting factor 6 (PIF6) from A. thaliana. 
</P><P>The production of Phytochrome and PIF6 was designed in 2 constructs. 
PIF6 construct (PIF6-TetR BD-NLS) and Phytochrome-B construct (PhyB-VP16 AD-AGS-NLS-PolyA). 
</P><P>The PIF6 construct (PIF6-TetR BD-NLS) was designed by fusing  the DNA-Binding Domain of the TetR,  with a Nuclear Localization Sequence and a polyadenylation tail . The activation domain was used in order to induce the system expression and was attached to the DNA binding domain of TetR. Everything was under the control of the CaMV promoter. 
</P><P>The Phytochrome-B construct was designed by  fusing an eucaryotic transactivation domain VP16 from Herpes simplex virus, a 3 amino acid protein linker and the NLS and PolyA, everything under the control of the CaMV promoter 
</P><P>These 2 proteins are photosensitive and their interaction and activation is mediated through light waves.  In the presence of an specific light wave length  Phytochrome B and PIF6 bind together  making a Phytochrome-PIF6 complex. This complex in its active form triggers the transcription of two small interference RNAs which are complementary with two RBSs presents in the de-greening construct, blocking the production of Degreening molecules, thus shouting off permanently this system.
(For a further explanation check our Reset - Red light switch)
</P><P>By stoping the degreening process we can asure that the plant will not die due to chloropyll deficiency. 
</P><H2>Final Assembly </H2><P>Now that we know how our 3 systems work individually we can make a final construct integrating all of our systems. 
Our final construct would look like this 
</P><P>Once the final construct was stablished and the parts assembly was done, the final vectors to accomplish plant cells transfection were the next ones.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:BIOSINT_Mexico/Assembly">http://2014.igem.org/Team:BIOSINT_Mexico/Assembly</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:BIOSINT_Mexico/Assembly" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:BIOSINT_Mexico/Assembly" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:BIOSINT_Mexico/Assembly&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:BIOSINT_Mexico/Assembly&amp;oldid=399649" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>