"
Page
Discussion
View source
History
teams
Log in
 
Team:BIOSINT Mexico/protocols
From 2014.igem.org
HOME
PROJECT
OVERVIEW
CHASSIS
BIOACCUMULATION
SENSOR
LIGHT SWITCH
ASSEMBLY
SAFETY
SAFETY FORM
GARDEN
PRACTICES
ART
SYMPOSIUM
COLLABORATION
TEAM
MEMBERS
SPONSORS
GALLERY
ATRIBUTIONS
OFFICIAL PAGE
ACHIEVEMENTS
PARTS
RESULTS
MODELING
JUDGING
NOTEBOOK
PROTOCOLS
LAB RECORDS
Protocols
Preparation of Terrific Broth and LB Agar
Preparation of MS media for Arabidopsis thaliana
Preparation of Antibiotic stocks
CaCl2 Competent Cells
Heat Shock Transformation of E. coli
Transformation by Electroporation
Plasmid purification by Miniprep
Plasmid purification by Midiprep
Electrophoresis Agarose Gel
Restriction Digest
Parts Ligation
Gel DNA Purification
Arabidopsis transformation: Floral Deep
References
Preparation of Terrific Broth and LB Agar
Weight 35 grams of LB Agar (Sigma-Aldrich) for each liter of media.
Weight 42.5 grams of Terrific Broth (Sigma-Aldrich) for each liter of media.
Dilute in Erlenmeyer flasks with the adequate amount of distilled water.
Heat in constant agitation.
Place the flasks in the autoclave with autoclave tape.
Set te autoclave until it reaches 120 °C and 15 PSI.
Store both the Broth and Agar at 4 °C.
Preparation of MS media for Arabidopsis thaliana
For 500 ml of MS (Murashige-Skoog) media measure:
5 g of sucrose
2.2 g of basal MS salt (Sigma-Aldrich)
4 g of agar
5 ml of Vitamin stock mix (Gamblor stock solution)
500 ml of distilled water
Mix all the components, except the agar, in constant agitation.
Adjust the pH to 5.7 ± 0.1.
Add the agar and mix gently and heat until it reaches 65 – 70 °C. (Do not boil or autoclave)
Pour the agar in plates, let them cool and store at 4 °C.
Preparation of Antibiotic stocks
Cloramphenicol Stock:
Weight 10 mg of lyophilized antibiotic for each ml of stock solution.
Dissolve in ethanol and mix gently (It is possible to use Vortex)
Kanamicin Stock:
Weight 50 mg of lyophilized antibiotic for each ml of stock solution.
Dissolve in distilled water and mix gently (It is possible to use Vortex).
Ampicilin Stock:
Weight 100 mg of liophylized antibiotic for each ml of stock solution.
Dissolve in distilled water and mix gently (It is possible to use Vortex).
For each 2 ml of media use 1 ul of stock antibiotic.
Therefore the working concentration for each antibiotic (for use in and Agrobacterium tumefaciens and E. coli) will be:
Cloramphenicol: 5ul/ml
Kanamicin: 25 ul/ml
Ampicilin: 50 ul/ml
CaCl2 Competent Cells
Streak out frozen glycerol stock of bacterial cells (Top10, DH5α, etc.) onto an LB plate (no antibiotics since these cells do not have a plasmid in them). Work sterile. Grow plate overnight at 37°C.
Make sure to autoclave 1 L LB (or your preferred media), 1 L of 100 mM CaCl2, 1 L of 100 mM MgCl2, 100 mL of 85 mM CaCl2, 15% glycerol v/v, 4 centrifuge bottles and caps, lots of microfuge tubes
Chill overnight at 4°C 100 mM CaCl2, 100 mM MgCl2, 85 mM CaCl2, 15% glycerol v/v
Prepare starter culture of cells
Select a single colony of E. coli from fresh LB plate and inoculate a 10 mL starter culture of LB (or your preferred media – no antibiotics). Grow culture at 37°C in shaker overnight.
For the next day, inoculate 1 L of LB media with 10 mL starter culture and grow in 37°C shaker.
Measure the OD600 every hour, then every 15-20 minutes when the OD getsabove 0.2.
When the OD600 reaches 0.35-0.4, immediately put the cells on ice. Chill the culture for 20-30 minutes, swirling occasionally to ensure even cooling. Place centrifuge bottles on ice at this time.
(Spin #1) Split the 1 L culture into four parts by pouring about 250 mL into ice cold centrifuge bottles. Harvest the cells by centrifugation at 3000g for 15 minutes at 4°C.
Decant the supernatant and gently resuspend each pellet in about 100 mL of ice cold MgCl2. Combine all suspensions into one centrifuge bottle. Make sure to prepare a blank bottle as a balance.
(Spin #2) Harvest the cells by centrifugation at 2000g in the refrigerated centrifuge (~3000 rpm) for 15 minutes at 4°C.
Decant the supernatant and resuspend the pellet in about 200 mL of ice cold CaCl2. Keep this suspension on ice for at least 20 minutes. Start putting 1.5 mL microfuge tubes on ice if not already chilled.
(Spin #3) Harvest the cells by centrifugation at 2000g (~3000 rpm) for 15 minutes at 4°C. At this step, rinse a 50 mL conical tube with ddH2O and chill on ice.
Decant the supernatant and resuspend the pellet in ~50 mL of ice cold 85 mM CaCl2, 15% glycerol. Transfer the suspension to the 50 mL conical tube.
(Spin #4) Harvest the cells by centrifugation at 1000g (~2100) for 15 minutes at 4°C.
Decant the supernatant and resuspend the pellet in 2 mL of ice cold 85 mM CaCl2, 15% glycerol. The final OD600 of the suspended cells should be ~200-250.
Aliquot 50 μL into sterile 1.5 mL microfuge tubes and snap freeze with liquid nitrogen. Store frozen cells in the -80°C freezer.
Heat Shock Transformation of E. coli
Note: Never vortex competent cells. Mix cells by gentle shaking.
Thaw competent cells on ice. These can be prepared using the CaCl2 protocol.
Place 20 ul of cells in a pre-chilled Eppendorf tube.
For an Intact Vector: Add 0.5 ul or less to the chilled cells
For a Ligation Product: Add 2-3 ul to the chilled cells.
Mix gently by flicking the tube.
Chill on ice for 10 minutes. (Optional)
Heat shock at 42 °C for 50 seconds.
Incubate on ice for 2 minutes.
Add 200 ul LB, Terrific or SOC medium and recover the cells by shaking at 37 °C.
The recovery time varies with the antibiotic selection.
Ampicillin: 15-30 minutes
Kanamycin or Spectinomycin: 30-60 minutes
Chloramphenicol: 60-120 minutes
Plate out the cells on selective LB. Use glass beads to spread the cells. The volume of cells plated depends on what is being transformed.
For an Intact Vector: High transformation efficiencies are expected. Plating out 10 ul of recovered cells should produce many colonies.
For a Ligation Product: Lower transformation efficiencies are expected. Therefore you can plate the entire 200 ul volume of recovered cells.
Incubate at 37 °C. Transformants should appear within 8 – 16 hrs.
Transformation by Electroporation
Preparation of Electrocompetent Cells
Competent cells should never be vortexted, as this will cause them to lyse and release salts into the media. Resuspend cells by pipeting up and down with a large pasteur pipet. Once they are chilled, cells should be continuously cold.
The night before the transformation, start an overnight culture of cells. 5 ml LB Amp.
The day of the transformation, dilute the cells 100X. 100 ml LB Amp.
Grow at 30°C for about 90 minutes.
Harvest the cells. When the cells reach an OD600 of between 0.6 and 0.8.
Split the culture into 2x 50 ml falcon tubes, on ice.
Centrifuge at 4 °C for 10 min at 4000 rpm.
Wash and combine the cells.
Remove the supernatant.
Resuspend the cells in 2x 25 ml of ice cold water.
Combine the volumes in a single 50 ml falcon tube.
Wash the cells 2 more times.
Centrifuge at 4 °C for 10 min at 4000 rpm.
Resuspend in 50 ml of ice cold water.
Repeat.
Wash and concentrate the cells for electroporation.
Centrifuge at 4 °C for 10 min at 4000 rpm.
Resuspend in 1-2 ml of ice cold water.
We will use 200 ul of washed cells per transformation.
Dialysis of PCR or Digestion Products
DNA for electroporation must be free of salts to avoid arcing.
Float a filter in a Petri dish filled with water.
Millipore membrane filter 0.025 uM.
Pipet one drop of PCR product onto the filter.
200 ng is needed per transformation.
20 - 100 ul fits well on one filter.
Collect the drop after 30 - 45 minutes.
The volume will change, but the DNA is not lost.
Plasmid purification by Miniprep
Centrifuge an eppendorf tube of 1.5ml with L. Plantarum culture at 12000 rpm in the mini-spin for 1 minute, two times.
Discard the supernatant and resuspend the tube’s content with more L. Plantarum culture in MRS broth.
Centrifuge the eppendorf tube for 15min at 6000 rpm.
Discard the supernatant.
Resuspend the pellet in 250ml Resuspension buffer.
Add 250ml of Lysis Buffer.
Gently mix the tube carefully inverting it 5 times carefully.
Add and mix softly 350ml of Precipitation Buffer, inverting the tube.
Centrifuge it at 12000 rpm for 10 minutes.
Transfer the supernatant into a spin column inside a washtube.
Centrifuge it at 12000 rpm for a minute.
Discard the supernatant and add 500 ml of Wash Buffer with ethanol (w10) to the column.
Incubate it for one minute at room temperature.
Centrifuge the column at 12000 rpm for 1 minute.
Discard the liquid from the washtube and place the column inside the tube.
Add 700ml of Wash Buffer W9 with ethanol to the column.
Centrifuge the column with the washtube at 12000 rpm for 1 minute.
Discard the liquid from the washtube.
Centrifuge the column with the washtube at 12000 rpm for 1 minute.
Discard the liquid from the washtube.
Place the column inside an eppendorf tube of 1.5ml.
Add 75ml of preheated TE Buffer at the center of the column.
(Warm the TE Buffer previously in water bath at 65⁰c-70⁰c for 3 minutes).
Incubate the column for 1 minute at room temperature.
The column was centrifuged at 12000 rpm for 2 minutes.
(The eppendorf tube contains the purified plasmid).
Plasmid purification by Midiprep
Harvest the cells by centrifugation for 10 min at 12000 rpm in the mini-spin. Discard the supernatant.
Resuspend the pelleted cells in 2 mL of Resuspension Solution auditioned with RNase solution. The bacterial pellet should be resuspended by vortexing or pipetting up and down until no cell clumps remain.
Add 2 mL of Lysis Solution and mix gently by inverting the tube 4-6 times until the solution becomes viscous and slightly clear. Incubate for 3 min at room temperature.
Add 0.5 mL of the Endotoxin Binding Reagent. Mix immediately by inverting the tube 5-8 times.
Incubate for 5 min at room temperature. Note. After the addition of the Neutralization Solution and Endotoxin Binding Reagent it is important to mix gently, but thoroughly, to avoid localized precipitation of bacterial cell debris. The neutralized bacterial lysate should appear cloudy and contain white precipitate.
Add 3 mL of 96% ethanol. Mix immediately by inverting the tube 5-6 times.
Centrifuge for 10 min at 4,000-5,000 rpm to pellet cell debris and chromosomal DNA.
Transfer the supernatant to a 15 mL tube (not provided) by decanting or pipetting. Avoid disturbing or transferring the white precipitate.
Add 3 mL of 96% ethanol. Mix immediately by inverting the tube 5-6 times.
Transfer part of the sample (~ 5.5 mL) to the supplied column pre-assembled with a collection tube (15 mL). Be careful not to overfill the column. Centrifuge for 3 min 10000 rpm in the mini-spin. Discard the flow-through and place the column back into the same collection tube.
Repeat the last step to process any remaining lysate through the purification column.
Add 4 mL of Wash Solution I (diluted with isopropanol) to the purification column. Centrifuge for 2 min. at 4000 rpm in a swinging bucket rotor. Discard the flow-through and place the column back into the same collection tube.
Add 4 mL of Wash Solution II (diluted with ethanol) to the purification 6 column. Centrifuge for 2 min. at 5000 rpm. Discard the flow-through and place the column back into the same collection tube.
Repeat the column wash with Wash Solution II
Centrifuge for 5 min at 3,000 × g in a swinging bucket rotor to remove residual wash solution. Discard the collection tube containing the flow-through.
Transfer the column into a fresh 15 mL collection tube (provided). Add 0.35 mL of the Elution Buffer to the centre of the purification column membrane. Incubate for 2 min at room temperature and centrifuge for 5 min at 5000 rpm to elute plasmid DNA.
Discard the purification column. Use the purified plasmid DNA in downstream applications or store DNA at -20°C.
Electrophoresis Agarose Gel
For each 100 ml of electrophoresis gel, weight 1 g of agarose.
Dissolve in the adequate amount of 1x TAE Buffer.
Heat in constant agitation. It could be done in microwave by heating in short intervals and agitating manually.
When completely dissolved let the flask cool a little then and 1.2 l of ethidium bromide 5 % p/p for each 50 ml of gel.
Agitate until it’s completely dissolved. If not used immediately store at 4 °C.
Pour the agarose into a gel tray with the suitable well comb in place (pour slowly to avoid bubbles which will disrupt the gel).
Place newly poured gel at room temperature for 20-30 minutes, until the gel has completely solidified.
Once solidified, remove the comb and place the gel into the electrophoresis unit (gel box).
Fill the gel box with 1x TAE buffer.
Load GeneRuler 1Kb Plus DNA Ladder 0.1 ug/ml weight one lane of the gel.
Carefully load your samples into the additional wells of the gel, carefully mixed with 6X DNA Loading Dye.
Run the gel at 50-150V until the dye line is approximately 75-80% of the way down the gel (20-45 min).
Use an UV light transilluminator to observe the DNA fragments on the gel.
Restriction Digest
Keep all enzymes and buffers used on ice.
Thaw NEB Buffer 2 and BSA in room temperature water. Mix by shaking the tubes, and flick/spin them to collect the liquid at the bottom of the tube.
Add 250ng of DNA to the appropriately labelled tube. Add distilled water to the tubes for a total volume of 16ul in each tube.
Pipet 2.5ul of NEB Buffer 2 to each tube.
Pipet 0.5ul of BSA to each tube.
In the Part A tube: Add 0.5ul of EcoRI, and 0.5ul of SpeI.
In the Part B tube: Add 0.5ul of XbaI, and 0.5ul of PstI.
In the pSB1C3 tube: Add 0.5ul of EcoRI, 0.5ul of PstI, and 0.5ul of Dpn1.
The total volume in each tube should be approximately 20ul. Mix well by pipetting slowly up and down. Spin the samples briefly to collect all of the mixture to the bottom of the tube.
Incubate the restriction digests at 37°C for 30 minutes, then 80°C for 20 minutes. We use a thermal cycler with a heated lid.
Use ~2ul of the digest (25ng of DNA) for ligations.
Part A
Part B
linearized plasmid backbone
DNA
250ng
250ng
250ng (10ul @ 25ng/ul)
dH2O
adjust to 16ul
adjust to 16ul
6ul
NEB Buffer 2
2.5ul
2.5ul
2.5ul
BSA
0.5ul
0.5ul
0.5ul
Enzyme 1
0.5ul EcoRI
0.5ul Xbal
0.5ul EcoRI
Enzyme 2 Cell:1
0.5ul SpeI
0.5ul PstI
0.5ul Pst1
Enzyme 3
0.5ul Dpnl
Parts Ligation
Sticky-end ligation.
Prepare the following reaction mixture:
Linear Vector DNA
20 - 100 ng
Insert DNA
1:1 to 5:1 molar ratio over vector
10X T4 DNA Ligase Buffer
2 µl
T4 DNA Ligase
1 u
Nuclease-free water
To 20 µl
Total volume
20 µl
Incubate 10 min at 22°C.
Use up to 5 µl of the mixture for transformation of 50 µl of chemically competent cells or 1-2 µl per 50 µl
The electrotransformation efficiency may be improved by heat inactivation of T4 DNA ligase at 65°C for 10 min or at 70°C for 5 min
The overall number of transformants may be increased by extending the reaction time to 1 hour.
If more than 2 u of T4 DNA ligase is used in 20 μl reaction mixture, it is necessary to purify DNA (by spin column or chloroform extraction) before electrotransformation.
Gel Purification
Excise the area of the gel containing your desired DNA fragment using a clean, sharp razor blade. Minimize the amount of agarose surrounding the DNA fragment.
Weigh the gel slice containing the DNA fragmen, then place the gel into a 1.5- or 5.0-mL microcentrifuge tube.
Note: The maximum amount of starting material (gel) is ≤400 mg per tube. If your gel slice exceeds 400 mg, cut the gel into smaller slices so that no one piece exceeds 400 mg. Place each additional gel slice created into separate microcentrifuge tubes. For ≤2% agarose gels:
Place up to 400 mg of the excised gel containing the DNA fragment (previous page) into a 1.5-mL polypropylene microcentrifuge tube.
Add 3 volumes Gel Solubilization Buffer for every 1 volume of gel (e.g., add 1.2 mL Gel Solubilization Buffer for a 400 mg gel piece).
Place the tube(s) containing your gel slice and Gel Solubilization Buffer into a 50°C water bath or heat block.
Incubate the gel slice and Gel Solubilization Buffer at 50°C for at least 10 minutes. Invert the tube by hand every 3 minutes to mix and ensure gel dissolution.
Note: High concentration gels (>2% agarose) or large gel slices may take longer than 10 minutes to dissolve.
After the gel slice appears dissolved, incubate the tube for an additional 5 minutes.
Optional: For optimal DNA yields, add 1 gel volume isopropanol to the dissolved gel slice. Mix well.
Pipet the dissolved gel piece containing the DNA fragment of interest (steps 4–5, page 6) onto the center of a Quick Gel Extraction Column inside a Wash Tube.
Note: Do not load >400 mg dissolved agarose per Quick Gel Extraction Column.
Centrifuge at >12,000 × g for 1 minute.
Discard the flow-through and replace the Quick Gel Extraction Column into the Wash Tube.
Add 500 μL Wash Buffer (W1), containing ethanol to the Quick Gel Extraction Column.
Centrifuge at >12,000 × g for 1 minute.
Discard the flow-through and replace the column into the Wash Tube.
Centrifuge the column again at maximum speed for 1–2 minutes to remove any residual Wash Buffer and ethanol. Discard the Wash Tube and place the Quick Gel Extraction Column into a Recovery Tube.
Add 50 μL Elution Buffer (E5) to the center of the Quick Gel Extraction Column.
Incubate the column for 1 minute at room temperature.
Centrifuge the Column at >12,000 × g for 1 minute. The Recovery Tube contains the purified DNA. Discard the Quick Gel Extraction Column.
Store the purified DNA, or proceed to your downstream application of choice.
Floral Deep
According to the temperature, 22-23°C ir the optimum condition, lower temperatures can be accepted but higher are impossible. The consequences with higher temperatures may result with the reduce number of the leaves, flowers and seeds while in lower temperatures growth is slow.
Other condition is the water required, the optimal humidity is the mild (50% to 60%). Lower humidity cause drying soil and higher humidity cause plant sterility.
For transformation we used the floral dip method described in this protocol: (Sushanta, 2013).
Preparation of Agrobacterium strain containing the gene. Inoculate in to a 5 ml LB medium, incubate at 28°C for 2 days.
Inoculation of culture in a 500 ml LB medium containing antibiotics
Centrifugate at 4000rpm for 10 min. at room temperature.
Resuspended in 1 volume of 5% sucrose solution. 0.02% silwet L-77 is use in the mixing solution andAgrobacterium cell suspension transfer to a 500 ml beaker. Agrobacterium suspension quantity up to 400-500 ml required for transformation of at least six pots of A. thaliana.
Arabidopsis thaliana plants are invert and dip into the Agrobacterium suspension for 10 seconds with gentle agitation.
The plants are removed from the solution and wash.
Plants are wrapping with plastic cover, are laydown on their sides for 16-24 hours to maintain high humidity.
On the next day, remove the cover and keep it in a growth chamber.
Plant dry seeds collect using a sieve mesh.
References
Ho, C. H., Lin, S. H., Hu, H. C., & Tsay, Y. F. (2009). CHL1 functions as a nitrate sensor in plants. Cell, 138(6), 1184-1194.
Yang, J., Ordiz, M. I., Semenyuk, E. G., Kelly, B., & Beachy, R. N. (2012). A safe and effective plant gene switch system for tissue-specific induction of gene expression in Arabidopsis thaliana and Brassica juncea. Transgenic research, 21(4), 879-883.
Müller, K., Siegel, D., Jahnke, F. R., Gerrer, K., Wend, S., Decker, E. L., ... & Zurbriggen, M. D. (2014). A red light-controlled synthetic gene expression switch for plant systems. Molecular BioSystems, 10(7), 1679-1688.
Antunes, M. S., Ha, S. B., Tewari‐Singh, N., Morey, K. J., Trofka, A. M., Kugrens, P., ... & Medford, J. I. (2006). A synthetic de‐greening gene circuit provides a reporting system that is remotely detectable and has a re‐set capacity. Plant biotechnology journal, 4(6), 605-622.
Guo, H. S., Fei, J. F., Xie, Q., & Chua, N. H. (2003). A chemical‐regulated inducible RNAi system in plants. The Plant Journal, 34(3), 383-392.
Chen, Z., Wang, J., Ye, M. X., Li, H., Ji, L. X., Li, Y., ... & An, X. M. (2013). A novel moderate constitutive promoter derived from poplar (Populus tomentosa Carrière). International journal of molecular sciences, 14(3), 6187-6204.
Boyle, P. M., Burrill, D. R., Inniss, M. C., Agapakis, C. M., Deardon, A., DeWerd, J. G., ... & Silver, P. A. (2012). A BioBrick compatible strategy for genetic modification of plants. Journal of biological engineering, 6(1), 1-8.
Nucifora, G. I. U. S. E. P. P. I. N. A., Chu, L., Silver, S. I. M. O. N., & Misra, T. K. (1989). Mercury operon regulation by the merR gene of the organomercurial resistance system of plasmid pDU1358. Journal of bacteriology, 171(8), 4241-4247.
Rugh, C. L., Wilde, H. D., Stack, N. M., Thompson, D. M., Summers, A. O., & Meagher, R. B. (1996). Mercuric ion reduction and resistance in transgenic Arabidopsis thaliana plants expressing a modified bacterial merA gene.Proceedings of the National Academy of Sciences, 93(8), 3182-3187.
Bizily, S. P., Rugh, C. L., Summers, A. O., & Meagher, R. B. (1999). Phytoremediation of methylmercury pollution: merB expression in Arabidopsis thaliana confers resistance to organomercurials. Proceedings of the National Academy of Sciences, 96(12), 6808-6813.
Sone, Y., Nakamura, R., Pan-Hou, H., Itoh, T., & Kiyono, M. (2013). Role of MerC, MerE, MerF, MerT, and/or MerP in resistance to mercurials and the transport of mercurials in Escherichia coli. Biological and Pharmaceutical Bulletin, 36(11), 1835-1841.
Sone, Y., Nakamura, R., Pan-Hou, H., Sato, M. H., Itoh, T., & Kiyono, M. (2013). Increase methylmercury accumulation in Arabidopsis thaliana expressing bacterial broad-spectrum mercury transporter MerE. AMB Express,3(1), 52.
Kleiner, O., Kircher, S., Harter, K., & Batschauer, A. (1999). Nuclear localization of the Arabidopsis blue light receptor cryptochrome 2. The Plant Journal, 19(3), 289-296.
Mathews, S. (2006). Phytochrome‐mediated development in land plants: red light sensing evolves to meet the challenges of changing light environments.Molecular Ecology, 15(12), 3483-3503.
Sone, Y., Pan-Hou, H., Nakamura, R., Sakabe, K., & Kiyono, M. (2010). Roles played by MerE and MerT in the transport of inorganic and organic mercury compounds in Gram-negative bacteria. Journal of Health Science, 56(1), 123-127.
Kwok, R. (2010). Five hard truths for synthetic biology. Nature, 463(7279), 288-290.
Auburn, A. L. (2000). Heavy Metal Soil Contamination. Soil Quality–Urban Technical Note, (3).
Highered.mheducation.com, (2014). Genetic Portrait Chapters A-E. [online] Available at: http://highered.mheducation.com/sites/007352526x/student_view0/genetic_portrait_chapters_a-e.html [Accessed 26 Sep. 2014]. *Boyle et al.: A BrioBrick compatible strategy for genetic modification of plants. Journal of Biological Engineering 2012 6:8.
Sushanta, K. (2013). Floral dip: A simple and efficient agrobacterium mediated transformation method is used in a model plant Arabidopsis Thaliana. Cibtech Journal of Bio-Protocols, 14-20.
Arabidopsis Biological Resource Center, (2014). Handling Arabidopsis plants and seeds. [online] Available at: https://abrc.osu.edu/sites/abrc.osu.edu/files/abrc_handling_seed_2013.pdf [Accessed 27 Sep. 2014].
Analysis of the genome sequence of the flowering plant Arabidopsis thaliana. (n.d.). Proquest. Retrieved October 12, 2014, from http://0-search.proquest.com.millenium.itesm.mx/docview/204469095/43FC9248199F47A8PQ/11?accountid=11643
Smith, K. (2009). USFDA/CFSAN.
Bizily S., et al. (1999). Phytoremediation of methylmercury pollution: MerB expression in Arabidopsis Thaliana confers resistance to organomercurials. Proc. Natl. Acad. Vol. 96, pp. 6806-6813, June1999.
Sone Y., et al. (2010). Roles played by MerE and MerT  in the transport of inorganic and organic mercury compounds in Gram-negative Bacteria. Journal of Health Science, 56(1) 123-127 2010.
Sone Y., et al. (2013). Increase methylmercury accumulation in Arabidopsis thaliana expressing bacterial broad-spectrum mercury transporter MerE. AMB Express 2013 3:52.
Barkay, T., Miller, S. and Summers, A. (2003). Bacterial mercury resistance from atoms to ecosystems. FEMS microbiology reviews, 27(2-3), pp.355--384.
Chaney, R. L., Malik, M., Li, Y. M., Brown, S. L., Brewer, E. P., Angle, J. S., & Baker, A. J. (1997). Phytoremediation of soil metals. Current opinion in Biotechnology, 8(3), 279-284.
Dekker, J., Vidal, M., & Walhout, A. M. (Eds.). (2014). Handbook of Systems Biology: Concepts and Insights.
Alon, U. (2006). An introduction to systems biology: design principles of biological circuits. CRC press.
Abramson, G. (2010). La matemática de los sistemas biológicos.
López, M. (1999). Un modelo matemático para el estudio de la incidencia del SIDA en la propagación de la tuberculosis (Doctoral dissertation, Tesis, Universidad Nacional de Colombia).
Klipp, E., Herwig, R., Kowald, A., Wierling, C., & Lehrach, H. (2008). Systems biology in practice: concepts, implementation and application. John Wiley & Sons.
Retrieved from " http://2014.igem.org/Team:BIOSINT_Mexico/protocols "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
