<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_BIOSINT_Mexico_Switch"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:BIOSINT_Mexico/Switch">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:BIOSINT_Mexico/Switch&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:BIOSINT_Mexico/Switch&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:BIOSINT_Mexico/Switch&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:BIOSINT_Mexico/Switch" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:BIOSINT Mexico/Switch</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="navmenubkg"><DIV id="navmenubox"><UL id="nav"><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico" title="Project Overview">HOME</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Project">PROJECT</A></LI><LI style="margin-left:14.5px"><A href="https://2014.igem.org/Team:BIOSINT_Mexico/overview" title="Project Overview">OVERVIEW</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Chassis">CHASSIS</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Bioaccumulation" title="Bioaccumulation">BIOACCUMULATION</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Sensor">SENSOR</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Switch" title="Red Light Switch">LIGHT SWITCH</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Assembly" title="Assembly">ASSEMBLY</A></LI></UL></DIV><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/safety">SAFETY</A></LI><LI style="margin-left:141px;"><A href="https://igem.org/Safety/Safety_Form?team_id=1471" title="Safety form">SAFETY FORM</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Garden" title="Garden">GARDEN</A></LI></DIV><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Policyandpractices" title="Policy&amp;Practices">PRACTICES</A></LI><LI style="margin-left: 300px;"><A href="https://2014.igem.org/Team:BIOSINT_Mexico/art" title="Art">ART</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/symposium" title="Symposium">SYMPOSIUM</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/collaboration" title="Collaboration">COLLABORATION</A></LI></DIV><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/team" title="TEAM">TEAM</A></LI><LI style="margin-left:300px;"><A href="https://2014.igem.org/Team:BIOSINT_Mexico/members" title="Members">MEMBERS</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Sponsors" title="Sponsors">SPONSORS</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Gallery" title="Gallery">GALLERY</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Atributions" title="Atributions">ATRIBUTIONS</A></LI><LI><A href="https://igem.org/Team.cgi?id=1471" title="Official page">OFFICIAL PAGE</A></LI></DIV><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Achievements" title="Achievements">ACHIEVEMENTS</A></LI><LI style="margin-left:300px"><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Parts">PARTS</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Results" title="Results">RESULTS</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Modeling" title="Modeling">MODELING</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/Judging" title="Judging">JUDGING</A></LI></DIV><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/notebook" title="Notebook">NOTEBOOK</A></LI><LI style="margin-left:649.5px"><A href="https://2014.igem.org/Team:BIOSINT_Mexico/protocols" title="Protocols">PROTOCOLS</A></LI><LI><A href="https://2014.igem.org/Team:BIOSINT_Mexico/lab records" title="Lab records">LAB RECORDS</A></LI><H1>Reset - Red light switch</H1><H2>Description</H2><P>We incorporate a red light-responsive gene expression system using far red (740 nm) light wave. This far red light sensor works as a switch that inactivates the expression of the de-greening system genes, is based on the interaction between the Phytochrome B and the phytochrome-interacting factor 6 (PIF6) from A. thaliana (Müller,K.  et al, 2013).
</P><P>In the literature, the switch used two different light waves, deep red (660 nm) for the activation, and far red (740 nm) for the immediate and permanent deactivation of the de-greening system. However, for this project, we designed a system responsive only for the far red light and which only function is the inactivation of the degreening of the plant.
</P><P>The first three hundreds nucleotides of PIF6 (BBa_K1150005) are fused to the DNA-Binding Domain of the TetR (BBa_K909007) protein and attaches to its operator site upstream a minimal promoter (PCaMV 35S BBa_K788000), also, its fused with a Nuclear Localization Sequence (NLS: BBa_K1150010) and a polyadenylation tail (PolyA: BBa_K1150012). The activation domain was used in order to induce the system expression, as proposed by Müller,K.  et al, attached to the DNA binding domain of TetR.
</P><P>The first 650 amino acids (1950 bp) of the Phytochrome B (BBa_K1150004) protein are fused to an eucaryotic transactivation domain (VP16: BBa_K105001) from Herpes simplex virus, a 3 amino acid protein linker (AGS: Linker BBa_K1150013) and the NLS and PolyA, everything attached to the same minimal promoter (PCaMV).
</P><P>In presence of far red light, the phytochrome and PIF6 complex will transform to its active form, and subsequently, it will activate the production of the small interference RNAs (RBS1 and IRES), that binds to the mRNA of the de-greening system at the RBSs positions, thus, stopping the binding of the ribosomes and the traduction of the messenger to protein.  Also, on dark times this system can be turned OFF for several hours (Müller,K.  et al, 2013)
</P><P>The components of the PhyB transcription factor (PhyB-VP16-NLS and tetR-PIF6) were under the control of a minimal promoter, specifically, the Cauliflower mosaic virus 35S promoter (PCaMV 35S) to optimize the red light inducible gene expression. (Müller,K.  et al, 2013)  This promoter was selected because it is widely used on plant genetic engineering, because its strength  and constitutive nature.
</P><H3><SPAN class="mw-headline" id="Light_response">Light response</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Layout of an experimental set-up for local red light-controlled gene expression in whole plants. To prevent activation of the red light-responsive gene switch in light-grown plants, supplementary 740 nm illumination is applied that constantly returns the system to the OFF state (müller et al, 2012).</DIV></DIV></DIV><P>Proteins responsive to light called photoreceptors are present in plant cells and its main function is to provide information about the environment, about circadian, seasonal and positional information. This information is crucial in plant development, given that it controls many processes of the organism such as germination, seedling development, sleep movements and many others (Mathew, 2006), and also, activate or regulate some cellular pathways, like chloroplast movement, cytoplasmic motility or endoreduplication 
</P><P>Phytochromes are some kind of photoreceptors sensitive to different shades of red light, with a range of reception between 620 and 740 approximately, including deep red and far red light (∼660 nm and ∼740 nm wavelength). Because of its nature, and the difference with other plant pigments, that only absorb wavelengths lower than 700 nm, these molecules are specific for detecting small changes in environment, like detecting the the length of the day or the relative position to other plants. (Mathew, 2006)
</P><P>PIF6 construct (PIF6-TetR BD-NLS) and Phytochrome-B construct (PhyB-VP16 AD-AGS-NLS-PolyA) are both attached to the same promoter, so the expression of both constructs is equal and the concentration of both proteins is the same. In the cytoplasm, these proteins are found floating free in the cytosol when the environmental conditions are normal. 
</P><P>However, absorption of photons of deep red light (660 nm) makes the proteins form a complex, thus, going through its active form, and switching the system to its ON state. Alike, if cells are exposed to far red light (740 nm wavelength) the system could be permanently turning to the off state (Mueller et al, 2013)
</P><P>Phytochrome-PIF6 complex in its active form triggers the transcription of two small interference RNAs which are complementaries to two RBSs presents in the de-greening construct, thus, shouting off permanently this system.
</P><H2>Modeling</H2><P><B>Equations for Phytochrome-PIF6 complex</B>In order to produce proteins, in the cell, two main reactions happen. First, DNA is transcripted to mRNA and then it is translated to a peptide sequence, it could be represented in the reaction by:
</P><P>All these reactions are mediated by two specific enzymes, which are RNA polymerase (transcription) and ribosome (translation). These reactions have been well described in literature and can be modeled to a Ordinary Differential Equation system.
The rate of production of the PhyB construct is given by the rate of transcription of the protein from the mRNA molecule and the degradation of the messenger. Therefore:
</P><P>Where γ is the translation rate of the cell, and α is the degradation of the degradation rate of the protein. 
Also, the concentration of the messenger molecule is given by the equation
</P><P>Where a represents the activity of the promoter, that is related to the capacity of being activated or suppressed by a transcription factor; β is the maximal production rate of the CaMV promoter (which is attached upstream to PhyB) and is a linear function dependent on the time of the reaction; as in the last equation, α is the degradation rate.
Also, since PCaMV is a constitutive promoter, the value of its activity is equal to 1. So the equation is reduced to  
</P><P>As PIF6 construct is connected to the same promoter, the equations of its expression are deduced by the same way, so:
</P><P>In our system, both expressed gene constructs are part of another reaction in presence of deep red light (660 nm lightwave). When photoreceptor protein (PhyB construct) detects deep red light, it fuses to its interaction factor (PIF6 construct) and form a protein 
For the complex creation, follow the following reaction in presence of 660 nm light.
</P><P>Where Ku is the constant of the reaction, and is on concentration units. And K-u is the constant for the inverse reaction (separation of the complex), and the active form of the complex can be abbreviated as CPPA. Then, we can infer the concentration of active complex by the equation.
</P><P>Also, in presence of far red light, the complex passes to its inactive form, represented as CPPI, by the following reaction:
</P><P>Where KD is the constant of the reaction and K-D equals zero, because deactivated complex can’t return to its active form.
</P><P>Then, if we despise the degradation rate of the inactivation of the protein complex, we can simplify the calculi of concentration of active photo protein by replacing 6 in 5.
</P><P><B>Equations for the repressor system</B>The active protein complex that is formed by phyb construct and pif6 construct becomes a transcription factor that activates the production of two small interference RNAs. One is specific for the RBS1 sequence and the other for the IRES sequence. 
Since the complex has a binding domain that joins to the promoter inducible PCMV, the rate of production of the siRNAs is given by the strength of the promoter and its activity (which is proportional to the activation by CPPA).
Since the activator has to bind to the DNA molecule in the promoter site, we use Hill’s function for macromolecules binding.
From: 
</P><P>Can be deduced for promoters with activators:
</P><P>To find the complete demonstrations for the Hill’s equation for inducible promoters, you can visit Aberdeen 2009 wiki.
Then, the activity for our promoter can be expressed as:
</P><P>Where βPCMV is the maximum expression achieved by our promoter. Kd is the constant is the equilibrium constant for the attachement of the inducer and the promoter, and n is the Hill’s coefficient for the cooperative ligation Induce-promoter.
Since the affinity of the tow molecules is high, the Hill’s coefficient is n&gt;1, then, aPCMV, is also greater than one. Thus, the equations that models the expression of siRNAs are:
</P><P>And given that both sequences are connected to the same promoter, the activity is the same for siRBS1 and siIRES. 
</P><H2>Results</H2><P>We fused the promoter PCMV (BBa_K747096) to a YFP reporter (BBa_E0030). 
</P><P>In order to measure its expression, we culture transformed E.coli in agar plates and measured the fluorescense intensity every 45 minutes. After that, we obtained the following data. [Fig.1]
</P><DL><DD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption"><B>Figure 1</B> YFP intensity every 45 minutes.</DIV></DIV></DIV></DD></DL><P>We made three repetitions, so in order to model the equation, we obtained the mean measure, therefore: 
</P><P>We analysed the data and using the Wolfram Mathematica software, we obtained the following equation that fits the data.
If we plot this equation, we obtain:
</P><P>Where we can see that the intensity of YFP grows directly proportional to the time and concentration of the molecule. 
</P><BLOCKQUOTE><FONT size="3"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:578px;"><DIV class="thumbcaption"><B>Figure 1</B>intensity of YFP grows directly proportional to the time and concentration of the molecule.   <I>.</I></DIV></DIV></DIV></FONT></BLOCKQUOTE><H2>References</H2><P>Müller, K., Siegel, D., Jahnke, F. R., Gerrer, K., Wend, S., Decker, E. L., ... &amp; Zurbriggen, M. D. (2014). A red light-controlled synthetic gene expression switch for plant systems. Molecular BioSystems, 10(7), 1679-1688.

</P><P>Mathews, S. (2006). Phytochrome‐mediated development in land plants: red light sensing evolves to meet the challenges of changing light environments.Molecular Ecology, 15(12), 3483-3503.

</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:BIOSINT_Mexico/Switch">http://2014.igem.org/Team:BIOSINT_Mexico/Switch</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:BIOSINT_Mexico/Switch" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:BIOSINT_Mexico/Switch" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:BIOSINT_Mexico/Switch&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:BIOSINT_Mexico/Switch&amp;oldid=400101" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>