<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Macquarie_Australia_chloroSummary skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Macquarie Australia/chloroSummary</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="TopBar"><UL id="TopLvlNav"><LI><A href="https://2016.igem.org/Team:Macquarie_Australia">Home</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Description">Project</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/interlab">Interlab study</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/protocols">Protocols/Testing</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Safety">Safety</A></LI></UL><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Summary">Summary Table</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Part_Collection">Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Theoretical">Theoretical</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Experimental">Experimental</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Demonstrate">Demonstration</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Integrated_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/HP/Gold">Integration</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/HP/Silver">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Students">Team</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Students">Students</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Advisers">Advisors</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Singapore ">NTU Singapore</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Paris">Paris Bettencourt</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/South Africa">Other</A></LI></DIV><HEADER><DIV class="ex1" align="justify"><FONT face="Arial" font="" size="5" color="#808080" font="" style="line-height:0.4"><CENTER>Aim</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>Improve <I>chlH</I> biobrick from 2015 Macquarie iGEM team by adding the lac promoter to allow expression in <I>E. coli</I>.</LI><LI>Assemble lac promoter with <I>ycf54</I> and <I>chlM</I> in one plasmid.</LI><LI>Assemble genes responsible for the conversion of protoporphyrin IX (PPIX) to Mg-PPIX in one plasmid (Mg-chelatase plasmid).</LI><LI>Assemble genes responsible for the conversion of Mg-PPIX to chlorophyll a in one plasmid (Mg-PPIX to chlorophyll a plasmid).</LI><LI>Assemble Mg-chelatase plasmid with Mg-PPIX to chlorophyll a plasmid to form a final plasmid of all 12 genes required to convert PPIX to chlorophyll a (chlorophyll a plasmid).</LI><LI>Protein expresses the Mg-chelatase plasmid and Mg-PPIX to chlorophyll a plasmid.</LI><LI>Characterise and analyse the function of the Mg-chelatase plasmid and Mg-PPIX to chlorophyll a plasmid.</LI></FONT></UL><FONT face="Arial" font="" size="5" color="#808080" font="" style="line-height:0.4"><CENTER>Experimental Design</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>Digest and ligate Biobricks together via 3A Assembly</LI><LI>IPTG induce plasmids for protein expression</LI><LI>Run cell lysate on SDS PAGE and use mass spectrometry to analyse gel bands</LI><LI>Transform and IPTG induce Mg-plasmid into <I>ΔHemH</I> mutants to analyse Mg-PPIX production via fluorescence emission and excitation</LI></FONT></UL><FONT face="Arial" font="" size="5" color="#808080" font="" style="line-height:0.4"><CENTER>Summarised results</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>Lac promoter was successfully cloned in front of <I>chlH</I> biobrick </LI><LI>Lac promoter, <I>ycf54</I> and <I>chlM</I> was successfully assembled in the correct order</LI><LI>Mg-chelatase plasmid was successfully assembled in the following order <I>pLac-ChlI1-ChlD-GUN4-ChlI2-CTH1-pLac-ChlH</I>. PCR and plasmid double digest confirmed the presence and size of the genes. Also confirmed by DNA sequencing</LI><LI>Successfully assembled Mg-PPIX to chlorophyll a plasmid. Plasmid double digest confirmed the correct size of the plasmid. </LI><LI>Unable to assemble the 12 gene plasmid due to time constraints</LI><LI>Protein induction revealed intense bands of correct sizes correlating with each gene in the Mg-chelatase plasmid. Mass spectrometry results also confirmed the presence of <I>chlH, chlI1</I> and <I>chlI2</I>. </LI><LI>Fluorescence excitation and emission spectra revealed that when the Mg-chelatase plasmid is added to <I>ΔHemH</I> mutants, Mg-PPIX production increases proving that the plasmid is functional. </LI></FONT></UL><DIV class="ex2" align="justify"><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><B>Fig 1.</B>The above gel image shows a ladder in lane 1, the <I>chlH</I> gene by itself in lane 2 and the <I>plac-ChlH</I> sequence in lanes 3-10. Lane 3 shows a slightly smaller band then the others and was excluded from sequencing results. Lanes 4-10 were confirmed as correct through sequencing. The lac promoter has a 200 base pair sequence which was added to the start of the gene and taken from iGem part BBa_R0010.</FONT></DIV><DIV class="ex2" align="justify"><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><B>Fig 2.</B>Gel electrophoresis (1% agarose) provides evidence of successful assembly of Mg-chelatase plasmid (10.9 kbp), <I>pLac-ycf54-chlM</I> (1.5 kbp) plasmid and Mg-PPIX to Chlorophyll a plasmid (6.3 kbp). Each plasmid was assembled via 3A assembly and double digested to reveal the biobrick backbone (2000 bp) and the correct insert size.</FONT></DIV><DIV class="ex2" align="justify"><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><B>Fig 3.</B>PCR confirmation of the <I>pLac-ycf54-chlM</I> plasmid. Forward and reverse <I>chlM</I> primers were used to confirm the presence of the <I>chlM</I> gene. The presence of <I>yCF54</I> gene was also confirmed using the <I>pLac</I> forward primer with the reverse <I>yCF54</I> primer. This also confirmed the presence of the lac promoter followed by <I>yCF54</I> (700 bp in total).</FONT></DIV><DIV class="ex2" align="justify"><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><B>Fig 4.</B>PCR screening of the Mg-chelatase operon produced products for all 6 expected genes aswell as the correct sizes.</FONT></DIV><DIV class="ex2" align="justify"><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><B>Fig 5.</B>Protein expression of the Mg-chelatase plasmid induced with IPTG. Lane 2 is the uninduced culture. Highly expressed band at approximately 144 kDa represents the ChlH protein. The MW of other proteins of interest include ChlI1 (40 kDa), ChlD (63 kDa), Gun4 (24 kDa), ChlI2 (40 kDa) and CTH1 (43 kDa).</FONT></DIV></DIV></HEADER><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><B>Fig 6.</B>All bands the size of the proteins of interest expressed from the induced magnesium chelatase plasmid were extracted from the SDS PAGE. MALDI TOF analysis of these proteins revealed that ChlH and ChlI2 were expressed successfully according to GPM.</FONT><DIV class="ex2" align="justify"><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><B>Fig 7.</B>Fluorescence excitation and emission spectra demonstrating the production of Mg-PPIX upon the addition of Mg-chelatase plasmid to ΔhemeH mutants. Mg-PPIX has an emission and excitation spectra of 418nm and 590nm respectively.</FONT></DIV><FIGURE><CENTER><FIGCAPTION><CENTER><B>Fig 8.</B>Emission and excitation spectra of the in vitro assay of Mg-chelatase from the Mg-P plasmid. Zero time control is the initial emission/excitation spectra when Mg-chelatase is added to 10mM PPIX, 10mM ATP and 15mM MgCl<SUB>2</SUB>. After incubating the assay under constant reagents as the control at room temperature, Mg-PPIX production increases. When additional 50mM MgCl<SUB>2</SUB> is added to the assay in comparison to the control, Mg-PPIX is production increases further. However, upon addition of 50mM MgCl<SUB>2</SUB> including 20nM of ChlID complex, the production of Mg-PPIX remains the same. Since adding additional ChlID enzyme to the last assay does not change Mg-PPIX production, the ChlID complex is not limiting.</CENTER></FIGCAPTION></CENTER></FIGURE></DIV><FONT face="Arial" font="" size="5" color="#808080"><CENTER>Collaboration with NTU Singapore</CENTER></FONT><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040">NTU Singapore helped us Team Macquarie by completing a RT-PCR to test if our Mg-chelatase plasmid is being expressed under IPTG induction.</FONT><DIV class="ex1" align="center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 9.</B> Gene expression rates of biosynthetic pathway genes relative to ampicillin in the Mg-chelatase plasmid: induced with varying concentrations of IPTG. This shows that the Mg-chelatase plasmid is best induced with 2mM IPTG.</CENTER></FONT></FIGCAPTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>