<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Macquarie_Australia_NoteNPS skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Macquarie Australia/NoteNPS</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="TopBar"><UL id="TopLvlNav"><LI><A href="https://2016.igem.org/Team:Macquarie_Australia">Home</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Description">Project</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/interlab">Interlab study</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/protocols">Protocols/Testing</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Safety">Safety</A></LI></UL><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Summary">Summary Table</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Part_Collection">Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Theoretical">Theoretical</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Experimental">Experimental</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Demonstrate">Demonstration</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Integrated_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/HP/Gold">Integration</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/HP/Silver">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Students">Team</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Students">Students</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Advisers">Advisors</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Singapore ">NTU Singapore</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Paris">Paris Bettencourt</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/South Africa">Other</A></LI></DIV><DIV class="column full_size"><HEADER><DIV class="ex1" align="justify"><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Week 1</SPAN></CENTER></FONT></H1><FONT face="Arial" font="" size="5" color="#808080" font="" style="line-height:0.4"><CENTER>(1st - 7th August)</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>An overview of the Photosystem II (PSII) project was given. What had been completed and what needed to be done was discussed.</LI><LI>Planning to build biobricks with Gblocks <I>psbD</I> and <I>psbT-B</I> and how assemble operons that are apart of PSII.</LI></FONT></UL><DIV class="ex2" align="justify"><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Week 2</SPAN></CENTER></FONT></H1><FONT face="Arial" font="" size="5" color="#808080" font="" style="line-height:0.4"><CENTER>(8th - 14th August)</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>Built biobricks with Gblocks <I>psbD</I> and <I>psbT-B</I>. The parts and plasmid backbones with antibiotic resistance were digested. The parts were ligated into the backbones to build biobricks. These plasmids were transformed into <I> E. coli </I>.</LI><LI>3A assembly was used to make composite parts <I>MZH+WK</I> (for operon 4) and <I>O+P</I> (for operon 5). These were also transformed into <I> E.coli </I> (both commercial and competent cells).
</LI><LI>A gel was run to check the digestions:
<DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 1.</B>Samples digested as a part of 3A assembly were run on an agarose gel against a DNA ladder to check whether the samples were digested correctly. From these digest results it was expected that for each sample there would be a band slightly above 2000 representative of the backbone that the Gblock was in and a band corresponding to the size of the Gblock. The digest of <I>WK</I> (614bp), <I>MZH</I> ( 541bp) and one sample of <I>O</I> (908bp) provided expected results. Results for <I>P</I> (727bp) did not match what was expected.</CENTER></FONT></FIGCAPTION></DIV></LI></FONT><LI>Digestion of <I>O</I> and <I>P</I> didn’t work very well – may be due to a higher concentration of DNA which can inhibit the restriction enzymes. <I>MZH</I> and <I>WK</I> digested well.
</LI><LI>The transformed <I>E. coli</I> was plated onto LB agar plates with the corresponding antibiotic to the antibiotic resistance in the backbone of each backbone.
</LI><LI>Colonies grew on plates for both the new biobricks and composite parts that were made.</LI></UL></DIV><DIV class="ex3" align="justify"><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Week 3</SPAN></CENTER></FONT></H1><FONT face="Arial" font="" size="5" color="#808080" font="" style="line-height:0.4"><CENTER>(15th - 21st August)</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>The transformant colonies containing <I>MZH+WK, O+P</I> and <I>D</I> were cultured, the plasmids extracted, digested and checked on a gel.
</LI><LI>3A assembly was used to make composite parts <I>C+A</I> (for operon 1). The digestion of <I>C</I> and <I>A</I> was checked on a gel. This was transformed into <I>E. coli </I> (both commercial and competent cells) and plated onto LB agar plates with the corresponding antibiotic to the antibiotic resistance in the backbone. Transformation appeared successful as colonies grew using commercial cells. No colonies grew using the competent cells indicating a low transformation efficiency for the competent cells.
<DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 2.</B>Samples digested as a part of 3A assembly were run on an agarose gel against a DNA ladder to check whether the samples were digested correctly. From these digest results it was expected that for each sample there would be a band slightly above 2000 representative of the backbone that the Gblock was in and a band corresponding to the size of the Gblock. The digest of <I>WK</I> (614bp), <I>MZH</I> ( 541bp) and one sample of <I>O</I> (908bp) provided expected results. Results for <I>P</I> (727bp) did not match what was expected.</CENTER></FONT></FIGCAPTION></DIV></LI></FONT><LI>Lane 5 &amp; 6 - <I>C+A</I> were digested with the intention of ligating, lane 8 &amp; 9 - <I>MZH+WK</I> were ligated together to form a new biobrick (lane 8 sample kept), lane 10 &amp; 11 -  <I>O+P</I> were ligated together (lane 10 sample kept) and lane 12 &amp; 13 - <I>D</I> was successfully ligated into an Amp backbone.
</LI><LI>A reattempt to put the part <I>TB</I> into a Gblock into a plasmid was made. Transformation, culturing, plasmid extraction and digestion was conducted so that it could be checked on a gel in week 4.</LI></UL></DIV><DIV class="ex4" align="justify"><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Week 4</SPAN></CENTER></FONT></H1><FONT face="Arial" font="" size="5" color="#808080" font="" style="line-height:0.4"><CENTER>(22nd - 28th August)</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>The transformant colonies from week 3 were cultured, the plasmids extracted, digested and checked on a gel. 
</LI><LI>Results of gel: <I>C+A</I> had not been successfully ligated, may have <I>TB-Cut Test</I> and <I>TB-IDT</I> successfully in plasmids (need to be sequenced) and <I>MZH+WK</I> in one plasmid. 
</LI><LI>Samples of <I>ELJ, TB-CUT-test, MZH-WK</I> were sent to be sequenced. 
</LI><LI>3A assembly was used to make composite part <I>OP+QR</I> (for operon 5) and reattempt the ligation of C+A (for operon 1). The digestions were checked on a gel. 
<DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 3.</B>Samples digested as part of a 3A assembly were run on an agarose gel against a DNA ladder to check whether the samples were digested correctly. The digest of <I>OP</I> (1635bp), <I>QR</I> (1092bp), <I>C</I> (1456bp) and <I>A</I> (1257bp) provided expected results with a bands corresponding to both the backbone and to the parts. .</CENTER></FONT></FIGCAPTION></DIV></LI></FONT><LI>The ligated parts were transformed into <I>E. coli</I> (both commercial and competent cells) and plated onto LB agar plates with the corresponding antibiotic to the antibiotic resistance in the backbone. 
</LI><LI>The results were promising with colonies growing on the plates. A digest, however, was still required check the results.</LI></UL></DIV><DIV class="ex5" align="justify"><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Week 5</SPAN></CENTER></FONT></H1><FONT face="Arial" font="" size="5" color="#808080" font="" style="line-height:0.4"><CENTER>(29th - 4th September)</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>Sequencing results sent off in week 4 were received and were analysed using a sequence alignment tool.
</LI><LI><I>ELJ</I> both forward and reverse sequences were fine as expected as they were joined last year. The sequence of <I>MZH+WK</I> did not provide conclusive results, with most of the sequence not matching to the corresponding template. Analysis of <I>TB-Cut</I> created confusion with regards to which template we should actually be using to complete the sequence analysis. The analysis of <I>TB-cut</I> still needed to be confirmed.</LI><LI>While attempting to analyse the sequencing results, 3A assembly was used to ligate <I>ELJ+TB</I> (operon 2 and 3). This was transformed into <I>E. coli</I> (both commercial and competent cells) and plated onto LB agar plates with the corresponding antibiotic to the antibiotic resistance in the backbone.</LI><LI>OP+QR transformant colonies from were cultured, the plasmids extracted, digested and checked on a gel.</LI><LI>OP+QR and more MZH+WK samples we had been sent off for sequencing. <I>C+A</I>, again, did not successfully ligate.</LI><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 4.</B>Samples for ELJ(756bp) and TB (3027bp) were digested as a part of 3A assembly and run on this gel to check whether the digest worked correctly. Results were as expected. Ligation products of <I>OP+QR</I> (2727bp), <I>C+A</I> (2713bp) were tested using digest check to see if products were of expected size corresponding to the backbone and composite part. Results were as expected for the ligation of <I>OP+QR</I>. The results for <I>C+A</I> suggest that the ligation of these parts did not work.</CENTER></FONT></FIGCAPTION></DIV></FONT><DIV class="ex6" align="justify"><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Week 6</SPAN></CENTER></FONT></H1><FONT face="Arial" font="" size="5" color="#808080" font="" style="line-height:0.4"><CENTER>(5th - 11th September)</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>Sequencing results for <I>OP+QR</I> and <I>MZH+WK</I> were received. Initially the <I>MZH+WK</I> did not look good upon analysis and the <I>OPQR</I> results were yet to be analysed.</LI><LI>3A assembly was used to, again, reattempt the ligation of <I>C+A</I> (for operon 1) and kept in the freezer ready to be transformed in week 7.</LI><LI>ELJ was sequenced confirmed and the sequenced confirmed sample was used in attempt to express this operon as a protein.</LI><LI>Samples of <I>D, TB-cut test</I> and more samples of <I>MZH-WK</I> were sent off for sequencing.</LI></FONT></UL></DIV><DIV class="ex7" align="justify"><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Week 7</SPAN></CENTER></FONT></H1><FONT face="Arial" font="" size="5" color="#808080" font="" style="line-height:0.4"><CENTER>(12th - 18th September)</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>3A assembly was used to make composite parts <I>ELJ+TB</I> (operon 2 and 3), <I>C+A</I> (for operon 1) and <I>MZHWK+OPQR</I> (operons 4 and 5). The digests were run on a gel. This was transformed into <I> E.coli </I> (both commercial and competent cells) and plated onto LB plates with the corresponding antibiotic to the antibiotic resistance in the backbone.</LI><LI>Promising results were seen when colonies grew on plates for each of the transformations. Transformant colonies from were cultured, the plasmids extracted, digested, checked on a gel and sent to be sequenced. Sequences were analysed and confirmed before being chosen to move forward with.</LI><LI>Sequence confirmed operons were transformed into DH10beta cells for protein expression during the following week.</LI></FONT></UL></DIV><DIV class="ex8" align="justify"><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Mid-semester Break</SPAN></CENTER></FONT></H1><FONT face="Arial" font="" size="5" color="#808080" font="" style="line-height:0.4"><CENTER>(19th September - 2nd October)</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>Received sequence confirmed results for <I>MZHWK+OPQR</I> and <I>ELJ+TB</I>. The results for <I>CA+D</I> were inconclusive; samples kept for reference. </LI><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 5.</B>Image results of a 1% gel with PSII PCR products designed to test the primers that were ordered several weeks ago. From these results many problems seem to occur, by measuring the size of the PCR products against the gel several primers do not appear to be binding to their template enough to for the correct PCR product for example <I>M</I>, <I>H</I>, <I>Z</I> and <I>T</I> do not produce the right sized PCR product and these need to be either retested or started again. For the rest, these will need to be retested.</CENTER></FONT></FIGCAPTION></DIV></FONT><LI>Primers designed for the individual genes were tested, multiple times, using PCR. There were good amplifications for a number of them, but the theoretical product sizes do not match up. The product sizes needed to be checked again.</LI><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 6.</B>Results of the operon <I>MZHWK’s</I> protein expression under IPTG induction. When compared to the protein standard a strong over induced band occurs between 25Kb and 37Kb. This is the correct size of the <I>MZHWK</I> protein complex when completely assembled. Although these samples underwent SDS and boiling for over 20mins and therefore should show a fraction of smaller bands. It can be clearly seen here that IPTG is not an effective way to overexpress <I>MZHWK</I> as the control in lane 3 has a similar sized band from due to background induction.</CENTER></FONT></FIGCAPTION></DIV><LI>Now that we have most of our operons together and separate, we decided to try and expressed them individually via IPTG induction (can be found under our protocols tab) as our operons all have lac promoters by design.</LI><LI>We began protein work and this involved expressing the operons <I>MZHWK, OPQR, ELJ</I> and <I>TB</I> separately. From the results only <I>MZHWK</I> was conclusively overexpressed from the normally expressed proteins. These <I>MZHWK</I> bands were then cut out for Mass Spec (MS) analysis.</LI><LI>Due to this, alternate methods of expression must be used like Auto induction.</LI><LI>We continued to digest the cut out bands with trypsin, left for overnight digestion and these then went to our Australian Proteome Analysis Facility (APAF) for MS.</LI><LI><I>MZHWK</I> protein expression yields an interesting looking band that is roughly at the combined size of the 5 proteins. This &quot;complex&quot; does not break down with additional SDS, and is very mass spec unfriendly; too hydrophobic throughout, lack of good peptides. Next attempt, is to show that it enriches in a &quot;membrane&quot; fraction as comparison, and if so, show chlorophyll binding. Note: no good mass spec results could be obtained from this protein complex.</LI><LI>Another attempt to ligate <I>CA+D</I> in a Kan backbone was made and this resulted in several cultures being produced. These were then digest checked and 3 samples were chosen <I>DCA</I><SUB>10</SUB>, <I>DCA</I><SUB>11</SUB> and <I>DCA</I><SUB>12</SUB>.</LI><LI>Ligation of <B>MOET</B> (<I>MZHWK-OPQR-ELT-TB</I>) was attempted, digest checked (GEL) – the results showed that the MOET bands form a E/P digest were smaller than should be (5,500bp smaller) and weren’t sent for sequencing for obvious reasons.
<DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 7.</B> Similar to the results shown in in figure 6, MZHWK can be seen as an over-expressed band. Although multiple methods for preparation were used here to try and isolate where the protein is being expressed although as shown here it can be found in the lysate and in the pellet.</CENTER></FONT></FIGCAPTION></DIV></LI><LI>From figure 7 is clear further work needs to be done when preparing this <I>MZKWK</I> protein as is found almost everywhere so far.</LI></UL></DIV><DIV class="ex9" align="justify"><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Week 8</SPAN></CENTER></FONT></H1><FONT face="Arial" font="" size="5" color="#808080" font="" style="line-height:0.4"><CENTER>(3rd - 9th October)</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>Attempted to bind chlorophyll to <I>MZHWK</I>, but it could not work. It is believed the incubation time allowed for the chlorophyll to bind was not long enough.</LI><LI>The results of the <I>DCA</I> sequences showed that <I>DCA</I><SUB>11</SUB> was very close and could possibly be correct. While the sequence analysis of <I>DCA</I><SUB>12</SUB> showed a perfect sequence alignment and these were then re-transformed to product more plasmid as stocks were running low.</LI><LI>More colonies of MOET were also selected for culturing to attempt to perform more screening. Although no colonies grew as they had been dead for too long. In the future another ligation attempt for MOET needs to be made, unfortunately time has run out to attempt again.</LI><LI>Additionally, all the operons were digested with E/P to do a final check on their size and all ligations products and operons were digested with NcoI (each gene should have a Nco1 site upstream of the sequence).</LI><DIV class="ex10" align="justify"><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Week 9</SPAN></CENTER></FONT></H1><FONT face="Arial" font="" size="5" color="#808080" font="" style="line-height:0.4"><CENTER>(10th - 16th October)</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040">
Additionally, several backbone swaps we done for part submissions:
<LI><I>D</I><SUB>3</SUB> already in Cam.</LI><LI><I>CA</I><SUB>1</SUB> in Kan (was swapped to Cam)</LI><LI><I>TB</I><SUB>1</SUB> in Cam</LI><LI><I>ELJ</I><SUB>1</SUB> in Cam</LI><LI><I>MZHWK</I><SUB>2-2</SUB> in Kan (was swapped to Cam)</LI><LI><I>OPQR</I><SUB>1</SUB> in Amp (was swapped to Cam)</LI><LI><I>MZHWK-OPQR</I><SUB>1</SUB> in Cam</LI><LI><I>ELJ-TB</I><SUB>1</SUB> in Kan (was swapped to Cam)</LI><LI>The swaps resulted in good colonies, although some background colonies occurred and these were avoided when selecting colonies for culture (Cam resistance plates and LB were used).</LI></FONT></UL></DIV><DIV class="ex11" align="justify"><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Week 10</SPAN></CENTER></FONT></H1><FONT face="Arial" font="" size="5" color="#808080" font="" style="line-height:0.4"><CENTER>(16th - 19th October)</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>Final gels were made to characterise <I>MZHWK</I>, this was done by incubating several versions of <I>MZHWK</I> with pure chlorophyll on a gel to visualise if chlorophyll binds to the protein complex. </LI><LI>Inclusion bodies etc. Plasmid mini preps were performed for <I>CA, MZHWK, OPQR</I> and <I>ELJ-TB</I> and a digest check was performed to create a final gel.</LI><LI>To finish we performed basic E/P digests of all submitting parts to create a pretty gel for our wiki.</LI></FONT></UL><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 8.</B> Shown here is a EcoR1 and Pst1 digest of our MZWH-OPQR composite part compared against a 1Kb ladder. The results show that samples 1, 3 and 4 are of the right size and sample 1 will be used for submission.</CENTER></FONT></FIGCAPTION></DIV></DIV></FONT><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 9.</B> Shown here are E/P digest of all submitting parts with our first gel attempt. All band sizes are promising although a cleaner gel needs to be done which also includes <I>MZHWK-OPQR</I> digests as well.</CENTER></FONT></FIGCAPTION></DIV><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><B>Note: All protocols or methods mentioned can be found under the tab “Project” and the “Protocols”.</B></FONT></UL></DIV></UL></DIV></DIV></HEADER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>