Team:Macquarie Australia/NoteNPS
Home
Project
Description
Results
Interlab study
Protocols/Testing
Notebook
Safety
Parts
Summary Table
Best Composite Part
Part Collection
Proof of Concept
Model
Theoretical
Experimental
Demonstration
Human Practices
Integration
Outreach
Team
Students
Advisors
Sponsors
Achievements
Collaborations
NTU Singapore
Paris Bettencourt
Other
Week 1
(1st - 7th August)
An overview of the Photosystem II (PSII) project was given. What had been completed and what needed to be done was discussed.
Planning to build biobricks with Gblocks psbD and psbT-B and how assemble operons that are apart of PSII.
Week 2
(8th - 14th August)
Built biobricks with Gblocks psbD and psbT-B. The parts and plasmid backbones with antibiotic resistance were digested. The parts were ligated into the backbones to build biobricks. These plasmids were transformed into E. coli .
3A assembly was used to make composite parts MZH+WK (for operon 4) and O+P (for operon 5). These were also transformed into E.coli (both commercial and competent cells).
A gel was run to check the digestions:
Fig 1.Samples digested as a part of 3A assembly were run on an agarose gel against a DNA ladder to check whether the samples were digested correctly. From these digest results it was expected that for each sample there would be a band slightly above 2000 representative of the backbone that the Gblock was in and a band corresponding to the size of the Gblock. The digest of WK (614bp), MZH ( 541bp) and one sample of O (908bp) provided expected results. Results for P (727bp) did not match what was expected.
Digestion of O and P didn’t work very well – may be due to a higher concentration of DNA which can inhibit the restriction enzymes. MZH and WK digested well.
The transformed E. coli was plated onto LB agar plates with the corresponding antibiotic to the antibiotic resistance in the backbone of each backbone.
Colonies grew on plates for both the new biobricks and composite parts that were made.
Week 3
(15th - 21st August)
The transformant colonies containing MZH+WK, O+P and D were cultured, the plasmids extracted, digested and checked on a gel.
3A assembly was used to make composite parts C+A (for operon 1). The digestion of C and A was checked on a gel. This was transformed into E. coli (both commercial and competent cells) and plated onto LB agar plates with the corresponding antibiotic to the antibiotic resistance in the backbone. Transformation appeared successful as colonies grew using commercial cells. No colonies grew using the competent cells indicating a low transformation efficiency for the competent cells.
Fig 2.Samples digested as a part of 3A assembly were run on an agarose gel against a DNA ladder to check whether the samples were digested correctly. From these digest results it was expected that for each sample there would be a band slightly above 2000 representative of the backbone that the Gblock was in and a band corresponding to the size of the Gblock. The digest of WK (614bp), MZH ( 541bp) and one sample of O (908bp) provided expected results. Results for P (727bp) did not match what was expected.
Lane 5 & 6 - C+A were digested with the intention of ligating, lane 8 & 9 - MZH+WK were ligated together to form a new biobrick (lane 8 sample kept), lane 10 & 11 - O+P were ligated together (lane 10 sample kept) and lane 12 & 13 - D was successfully ligated into an Amp backbone.
A reattempt to put the part TB into a Gblock into a plasmid was made. Transformation, culturing, plasmid extraction and digestion was conducted so that it could be checked on a gel in week 4.
Week 4
(22nd - 28th August)
The transformant colonies from week 3 were cultured, the plasmids extracted, digested and checked on a gel.
Results of gel: C+A had not been successfully ligated, may have TB-Cut Test and TB-IDT successfully in plasmids (need to be sequenced) and MZH+WK in one plasmid.
Samples of ELJ, TB-CUT-test, MZH-WK were sent to be sequenced.
3A assembly was used to make composite part OP+QR (for operon 5) and reattempt the ligation of C+A (for operon 1). The digestions were checked on a gel.
Fig 3.Samples digested as part of a 3A assembly were run on an agarose gel against a DNA ladder to check whether the samples were digested correctly. The digest of OP (1635bp), QR (1092bp), C (1456bp) and A (1257bp) provided expected results with a bands corresponding to both the backbone and to the parts. .
The ligated parts were transformed into E. coli (both commercial and competent cells) and plated onto LB agar plates with the corresponding antibiotic to the antibiotic resistance in the backbone.
The results were promising with colonies growing on the plates. A digest, however, was still required check the results.
Week 5
(29th - 4th September)
Sequencing results sent off in week 4 were received and were analysed using a sequence alignment tool.
ELJ both forward and reverse sequences were fine as expected as they were joined last year. The sequence of MZH+WK did not provide conclusive results, with most of the sequence not matching to the corresponding template. Analysis of TB-Cut created confusion with regards to which template we should actually be using to complete the sequence analysis. The analysis of TB-cut still needed to be confirmed.
While attempting to analyse the sequencing results, 3A assembly was used to ligate ELJ+TB (operon 2 and 3). This was transformed into E. coli (both commercial and competent cells) and plated onto LB agar plates with the corresponding antibiotic to the antibiotic resistance in the backbone.
OP+QR transformant colonies from were cultured, the plasmids extracted, digested and checked on a gel.
OP+QR and more MZH+WK samples we had been sent off for sequencing. C+A, again, did not successfully ligate.
Fig 4.Samples for ELJ(756bp) and TB (3027bp) were digested as a part of 3A assembly and run on this gel to check whether the digest worked correctly. Results were as expected. Ligation products of OP+QR (2727bp), C+A (2713bp) were tested using digest check to see if products were of expected size corresponding to the backbone and composite part. Results were as expected for the ligation of OP+QR. The results for C+A suggest that the ligation of these parts did not work.
Week 6
(5th - 11th September)
Sequencing results for OP+QR and MZH+WK were received. Initially the MZH+WK did not look good upon analysis and the OPQR results were yet to be analysed.
3A assembly was used to, again, reattempt the ligation of C+A (for operon 1) and kept in the freezer ready to be transformed in week 7.
ELJ was sequenced confirmed and the sequenced confirmed sample was used in attempt to express this operon as a protein.
Samples of D, TB-cut test and more samples of MZH-WK were sent off for sequencing.
Week 7
(12th - 18th September)
3A assembly was used to make composite parts ELJ+TB (operon 2 and 3), C+A (for operon 1) and MZHWK+OPQR (operons 4 and 5). The digests were run on a gel. This was transformed into E.coli (both commercial and competent cells) and plated onto LB plates with the corresponding antibiotic to the antibiotic resistance in the backbone.
Promising results were seen when colonies grew on plates for each of the transformations. Transformant colonies from were cultured, the plasmids extracted, digested, checked on a gel and sent to be sequenced. Sequences were analysed and confirmed before being chosen to move forward with.
Sequence confirmed operons were transformed into DH10beta cells for protein expression during the following week.
Mid-semester Break
(19th September - 2nd October)
Received sequence confirmed results for MZHWK+OPQR and ELJ+TB. The results for CA+D were inconclusive; samples kept for reference.
Fig 5.Image results of a 1% gel with PSII PCR products designed to test the primers that were ordered several weeks ago. From these results many problems seem to occur, by measuring the size of the PCR products against the gel several primers do not appear to be binding to their template enough to for the correct PCR product for example M, H, Z and T do not produce the right sized PCR product and these need to be either retested or started again. For the rest, these will need to be retested.
Primers designed for the individual genes were tested, multiple times, using PCR. There were good amplifications for a number of them, but the theoretical product sizes do not match up. The product sizes needed to be checked again.
Fig 6.Results of the operon MZHWK’s protein expression under IPTG induction. When compared to the protein standard a strong over induced band occurs between 25Kb and 37Kb. This is the correct size of the MZHWK protein complex when completely assembled. Although these samples underwent SDS and boiling for over 20mins and therefore should show a fraction of smaller bands. It can be clearly seen here that IPTG is not an effective way to overexpress MZHWK as the control in lane 3 has a similar sized band from due to background induction.
Now that we have most of our operons together and separate, we decided to try and expressed them individually via IPTG induction (can be found under our protocols tab) as our operons all have lac promoters by design.
We began protein work and this involved expressing the operons MZHWK, OPQR, ELJ and TB separately. From the results only MZHWK was conclusively overexpressed from the normally expressed proteins. These MZHWK bands were then cut out for Mass Spec (MS) analysis.
Due to this, alternate methods of expression must be used like Auto induction.
We continued to digest the cut out bands with trypsin, left for overnight digestion and these then went to our Australian Proteome Analysis Facility (APAF) for MS.
MZHWK protein expression yields an interesting looking band that is roughly at the combined size of the 5 proteins. This "complex" does not break down with additional SDS, and is very mass spec unfriendly; too hydrophobic throughout, lack of good peptides. Next attempt, is to show that it enriches in a "membrane" fraction as comparison, and if so, show chlorophyll binding. Note: no good mass spec results could be obtained from this protein complex.
Another attempt to ligate CA+D in a Kan backbone was made and this resulted in several cultures being produced. These were then digest checked and 3 samples were chosen DCA10, DCA11 and DCA12.
Ligation of MOET (MZHWK-OPQR-ELT-TB) was attempted, digest checked (GEL) – the results showed that the MOET bands form a E/P digest were smaller than should be (5,500bp smaller) and weren’t sent for sequencing for obvious reasons.
Fig 7. Similar to the results shown in in figure 6, MZHWK can be seen as an over-expressed band. Although multiple methods for preparation were used here to try and isolate where the protein is being expressed although as shown here it can be found in the lysate and in the pellet.
From figure 7 is clear further work needs to be done when preparing this MZKWK protein as is found almost everywhere so far.
Week 8
(3rd - 9th October)
Attempted to bind chlorophyll to MZHWK, but it could not work. It is believed the incubation time allowed for the chlorophyll to bind was not long enough.
The results of the DCA sequences showed that DCA11 was very close and could possibly be correct. While the sequence analysis of DCA12 showed a perfect sequence alignment and these were then re-transformed to product more plasmid as stocks were running low.
More colonies of MOET were also selected for culturing to attempt to perform more screening. Although no colonies grew as they had been dead for too long. In the future another ligation attempt for MOET needs to be made, unfortunately time has run out to attempt again.
Additionally, all the operons were digested with E/P to do a final check on their size and all ligations products and operons were digested with NcoI (each gene should have a Nco1 site upstream of the sequence).
Week 9
(10th - 16th October)
Additionally, several backbone swaps we done for part submissions:
D3 already in Cam.
CA1 in Kan (was swapped to Cam)
TB1 in Cam
ELJ1 in Cam
MZHWK2-2 in Kan (was swapped to Cam)
OPQR1 in Amp (was swapped to Cam)
MZHWK-OPQR1 in Cam
ELJ-TB1 in Kan (was swapped to Cam)
The swaps resulted in good colonies, although some background colonies occurred and these were avoided when selecting colonies for culture (Cam resistance plates and LB were used).
Week 10
(16th - 19th October)
Final gels were made to characterise MZHWK, this was done by incubating several versions of MZHWK with pure chlorophyll on a gel to visualise if chlorophyll binds to the protein complex.
Inclusion bodies etc. Plasmid mini preps were performed for CA, MZHWK, OPQR and ELJ-TB and a digest check was performed to create a final gel.
To finish we performed basic E/P digests of all submitting parts to create a pretty gel for our wiki.
Fig 8. Shown here is a EcoR1 and Pst1 digest of our MZWH-OPQR composite part compared against a 1Kb ladder. The results show that samples 1, 3 and 4 are of the right size and sample 1 will be used for submission.
Fig 9. Shown here are E/P digest of all submitting parts with our first gel attempt. All band sizes are promising although a cleaner gel needs to be done which also includes MZHWK-OPQR digests as well.
Note: All protocols or methods mentioned can be found under the tab “Project” and the “Protocols”.
