<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Macquarie_Australia_NoteNPH skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Macquarie Australia/NoteNPH</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="TopBar"><UL id="TopLvlNav"><LI><A href="https://2016.igem.org/Team:Macquarie_Australia">Home</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Description">Project</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/interlab">Interlab study</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/protocols">Protocols/Testing</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Safety">Safety</A></LI></UL><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Summary">Summary Table</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Part_Collection">Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Theoretical">Theoretical</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Experimental">Experimental</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Demonstrate">Demonstration</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Integrated_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/HP/Gold">Integration</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/HP/Silver">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Students">Team</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Students">Students</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Advisers">Advisors</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Singapore ">NTU Singapore</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Paris">Paris Bettencourt</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/South Africa">Other</A></LI></DIV><DIV class="column full_size"><HEADER><DIV class="ex1" align="justify"><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Week 1</SPAN></CENTER></FONT></H1><FONT face="Arial" font="" size="5" color="#808080"><CENTER>(1st - 7th August)</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>Our aim was to design and assemble sequences of the G-blocks for 5 genes coding only for proteins involved in hydrogen production to produce functioning hydrogenase enzymes in <I>E.coli</I>, thus allowing the production of hydrogen by reducing H<SUP>+</SUP> ions.</LI><LI>Background research on the G-blocks, Bio bricks and codon optimization restriction enzymes was conducted. BlastX was used to search each gene and alter any mismatches, ensuring that the identity of the genes are completely to the protein we require. Due to gene 3 being too big (maximum it can be is 500bp) it was made into two parts. A mismatch was found on gene 3 part 1 (75 bases from the end) and this was corrected by changing the proline was changed to alanine.</LI><LI>After the clarity of the genes was confirmed, a restriction mapping site, NEBcutter, was used to find the frequency of the restriction cut sites for E (EcoRI), X (XbaI), S (SmaI) and P (PstI) on the G-blocks. It was confirmed that the presence of 1 cut site for each restriction enzymes was present in the correct order required for the biobricks. </LI><LI>Sent the sequences of the genes to IDT for the G-blocks. </LI></FONT></UL></DIV><DIV class="ex2" align="justify"><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Week 2</SPAN></CENTER></FONT></H1><FONT face="Arial" font="" size="5" color="#808080" font="" style="line-height:0.4"><CENTER>(8th - 14th August)</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI> The G-blocks arrived and they came as Hyd EF part 1, Hyd EF part 2, Hyd1, HydG and ferredoxin. Each sample contained 1000ng of DNA.</LI><LI>The G-blocks were resuspended in 100μl of nuclease free water.</LI><FONT face="Arial" font="" size="4" color="#000000"><I>For D1, G and Ferredoxin:</I></FONT><LI>Restriction enzyme digest was performed; the G block were digested with EcoRI HF and PstI HF and Cut-smart buffer. Plasmid backbone with CAM resistance was also digested with the same enzymes.</LI><LI> The insert and backbone were then ligated using T4 DNA ligase and these parts were used to transform competent <I>E. coli</I> cells.</LI><LI> The cells were then plated on CAM LB agar plates for 20μl and 200μl and were left to grow overnight at 37°C.</LI><FONT face="Arial" font="" size="4" color="#000000"><I>For EF Part 1 and 2:</I></FONT><LI> This G-block was brought in as 2 parts because it was too large to synthesize in one part. Both parts had to be inserted into the same backbone. </LI><LI> The 3A assembly method was conducted for this; got as far as the ligation step. The parts were stored for later.</LI><LI> Initially came in to finish performing 3A assembly for the genes HYDEF part 1 and HYDEF part 2. However, when we checked the plates from the previous week, no colonies were present were on the plates.</LI><LI> Because of this we re-performed restriction enzyme digest, however we increased the amount of backbone as we realized we previously had equivalent backbone to insert.</LI></FONT></UL></DIV><DIV class="ex3" align="justify"><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Week 3</SPAN></CENTER></FONT></H1><FONT face="Arial" font="" size="5" color="#808080" font="" style="line-height:0.4"><CENTER>(15th - 21st August)</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>Using the ligated products containing EF part 1 and 2 from the previous week, transformation was performed and the cells were plated.</LI><LI>Growth was only observed in the plates for Fe and D1 (each plate had one colony). </LI><LI>Colony PCR was carried out in order to check if the cells had taken up the correct ligated backbone and insert. This was done by resuspending the colonies in LB CAM stock - half of the liquid was used for PCR and the other half more stock was added and incubated at 37°C to allow the cells to grow.</LI><LI>Restriction digests (using EcoRI) was performed on our ligated parts and a gel was ran to check if the biobricks had been successfully inserted into the plasmid backbone. </LI><LI>Colony PCR products (Fe and D1) and ligated parts digested EcoRI (G, D1, EF and Fe) were run on a gel. From the gel we observed that only Fe had a band at about the right size. We suspected that our ligation step may have been the issue, and realized that our first ligation mix did not have enough insert in the mix. </LI><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 1.</B>The gel contains E/P double digested plasmids of Hyd1 and 2 unique Fe gene transformants that are E/P double digested. </CENTER></FONT></FIGCAPTION></FONT><LI> After some calculations to find the most suitable insert: backbone ratio, new ligation mixes were made containing 16μl of insert (compared to the 4μl used in the previous reactions).</LI><DIV class="ex4" align="justify"><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Week 4</SPAN></CENTER></FONT></H1><FONT face="Arial" font="" size="5" color="#808080" font="" style="line-height:0.4"><CENTER>(22nd - 28th August)</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>Transformations of the new ligation mixes was carried out, however, this time RFP was used as a positive control and no growth was observed in any of the plates. </LI><LI>Transformation was performed again however, this time using a different RFP gene. After plating, the remaining solutions were inoculated in LB CAM and left to grow. </LI><LI>The following day, many red colonies grew on the control plate, and 1 or 2 colonies were observed for Fe, G and D1. For the inoculated cultures growth was only observed in D1.</LI><FONT face="Arial" font="" size="4" color="#000000"><I>Colony PCR was conducted for Fe, G and D1; the colonies were resuspended in:</I></FONT><LI>    10ul master mix</LI><LI>    1ul primer forward</LI><LI>    1ul primer reverse</LI><LI>    8ul H<SUB>2</SUB>O</LI><FONT face="Arial" font="" size="4" color="#000000"><I>Plasmid prep was conducted for D1, Fe1 and Fe2. From the plasmid prep, double digest of D1 and Fe with EcoRI and PstI was performed:</I></FONT><LI>    1ul buffer</LI><LI>    1ul DNA</LI><LI>    0.5ul EcoRI</LI><LI>    0.5ul Pst I</LI><LI>    7ul H<SUB>2</SUB>O</LI><LI>Products from the colony PCR and the double digest were run on a gel:</LI><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 2.</B>The gel contains E/P double digested plasmids of Hyd1 and 2 unique Fe gene transformants that are E/P double digested.</CENTER></FONT></FIGCAPTION></FONT><LI>Fe and D1 were sent for sequencing</LI><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Week 5</SPAN></CENTER></FONT></H1><FONT face="Arial" font="" size="5" color="#808080" font="" style="line-height:0.4"><CENTER>(29th - 4th September)</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>Sequences for Fe and D1 came back and they were confirmed to be correct.</LI><LI> Plasmids extracted from the plasmid preps for Fe and D1 were transformed into DH10β cells. This was done in preparation for protein expression.</LI><LI>Glycerol stocks for Fe and D1 were used for streak plates on LB CAM and left overnight at 37°C.</LI><LI>Updated the Gene Registry with a brief background and functions of our genes.</LI><LI>3A assembly was carried out for EF1 and EF2. Digest for G was done with EcoRI and Pst, followed by ligation and the ligated parts were stored for later use. </LI></FONT></UL><DIV class="ex6" align="justify"><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Week 6</SPAN></CENTER></FONT></H1><FONT face="Arial" font="" size="5" color="#808080" font="" style="line-height:0.4"><CENTER>(5th - 11th September)</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>Transformation for G and EF took place.</LI><LI>No colonies observed in G. One colony was observed for EF; the colony was placed in liquid culture.</LI><LI>Transformations were repeated, however no growth was observed.</LI><LI>Transformation of EF and G was conducted through electroporation, however, no growth was observed.</LI><LI>Plasmid prep of EF in liquid culture was performed.</LI><LI>3 separate transformations for EF1, EF2 and G was conducted and plated onto KAN. </LI><LI>The DNA concentration from EF plasmid prep was 62 ng/μl. The EF plasmid was digested and ran on a gel. However, the gel band was not the correct size.</LI><LI>Colonies were observed in EF1 and EF2 and they were inoculated in KAN LB broth.</LI></FONT></UL></DIV><DIV class="ex7" align="justify"><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Week 7</SPAN></CENTER></FONT></H1><FONT face="Arial" font="" size="5" color="#808080" font="" style="line-height:0.4"><CENTER>(12th – 18th September)</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>Plasmid prep was carried out for EF1 and EF2 (2 colonies were in EF2)</LI><LI>Nanodrop results (to check the concentration of DNA):</LI><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 3.</B>Nanodrop results showed that the DNA concentration of ef 1 is 91.9 ng/ul, ef 2a is 116.3 ng/ul and ef 2b is 134.2 ng/ul.</CENTER></FONT></FIGCAPTION></FONT><FONT face="Arial" font="" size="4" color="#000000"><I>Digestion of G and the backbone (CAM) was conducted:</I></FONT><LI>1ul DNA </LI><LI>2ul buffer</LI><LI>0.5ul EcorI</LI><LI>0.5ul PstI</LI><LI>Added water till 20ul was reached </LI><FONT face="Arial" font="" size="4" color="#000000"><I>Ligation was performed; for G both CAM and KAN backbone was used:</I></FONT><LI>2ul backbone</LI><LI>4ul insert</LI><LI>1ul T4 buffer</LI><LI>1ul T4 DNA ligase</LI><LI>2ul H<SUB>2</SUB>0</LI><LI>Transformation of G took place; plated on both CAM and KAN.</LI><LI>3A assemble was used to combine EF1 and EF2 a and EF1 and EF2 b.</LI><LI>No growth was observed on CAM or KAN for G, however, there was growth on the control.</LI><FONT face="Arial" font="" size="4" color="#000000"><I>Double digests was performed on the following:</I></FONT><LI>EF1 plasmid prep</LI><LI>EF2 plasmid prep (a and b)</LI><LI>G KAN ligation </LI><LI>G CAM ligation </LI><LI>EF a</LI><LI>EF b</LI><LI>Gel electrophoresis was performed:</LI><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 4.</B>The Gel consisting of E/P double digested DNA from EF part1 and 2 EF part2 transformants. </CENTER></FONT></FIGCAPTION><LI>EF 1, EF2 a and EF2 b were sent to sequencing:</LI><TABLE><TBODY><TR><TH colspan="2"><B>Forward</B></TH><TH colspan="4"><B>Reverse</B></TH></TR><TR><TD><B>EF1</B></TD><TD>1996ZAD098</TD><TD><B>EF1</B></TD><TD>1996ZAD099</TD></TR><TR><TD><B>EF2</B></TD><TD>187AZAB011</TD><TD><B>EF2 a</B></TD><TD>187AZAB003</TD></TR><TR><TD><B>EF2 b</B></TD><TD>1996ZAD096</TD><TD><B>EF2 b</B></TD><TD> 1996ZAD097 </TD></TR></TBODY></TABLE><LI>Digestion of G and RFP CAM backbone was performed:</LI><TABLE><TBODY><TR><TH colspan="1"><B>E5</B></TH><TH colspan="2"><B>XP</B></TH></TR><TR><TD>1uL DNA           </TD><TD>1uL DNA</TD></TR><TR><TD>2uL Buffer       </TD><TD>2uL Buffer</TD></TR><TR><TD>0.5uL EcoRI       </TD><TD>0.5uL XbaI</TD></TR><TR><TD>0.5uL SpeI       </TD><TD>0.5uL PstI</TD></TR><TR><TD>6uL of H<SUB>2</SUB>O</TD><TD>6uL of H<SUB>2</SUB>O</TD></TR></TBODY></TABLE><FONT face="Arial" font="" size="4" color="#000000"><I>Ligation was performed:</I></FONT><LI>1ul backbone</LI><LI>7ul insert</LI><LI>1ul T4 buffer</LI><LI>1ul T4 DNA ligase </LI><LI>Transformation for both ES and XP was carried out and the remaining solution was placed in LB CAM.</LI><LI>HydG screening with E/P:</LI></UL><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 5.</B>The gel contains X/P and E/S double digestions of DNA from different colonies of HydG transformants. E/S and X/P digests were used instead of EP due to previous failure in screening.</CENTER></FONT></FIGCAPTION><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Week 8</SPAN></CENTER></FONT></H1><FONT face="Arial" font="" size="5" color="#808080" font="" style="line-height:0.4"><CENTER>(3rd – 9th October)</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>Plasmid prep was performed for G. Hyd G screening with EP was then performed with gel electrophoresis. </LI><LI>Updated the parts registry.</LI><LI>Performed 3A assembly on the EF gene.</LI></FONT></UL></DIV><DIV class="ex9" align="justify"><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Week 9</SPAN></CENTER></FONT></H1><FONT face="Arial" font="" size="5" color="#808080" font="" style="line-height:0.4"><CENTER>(10th – 16th October)</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>Repeated Digestion on digestion of the EF gene.</LI><LI>Plasmid prep of the ef gene:</LI><CENTER><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 6.</B>Nanodrop for ef showing DNA concentration of 60.0 ng/ul</CENTER></FONT></FIGCAPTION></CENTER></FONT><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 7.</B>Screening for <I>hydG</I> with E/P digest. Transformation was not successful because no band was visible at 1900bp (theoretical size of <I>hydG</I></CENTER></FONT></FIGCAPTION><OL><LI>Gel showed that the band in lane 7 was not the correct size</LI><LI>Transformation of EF took place on a CAM plate.</LI><LI>Made 4 liquid cultures of the EF gene using the CAM media.</LI><LI>Performed plasmid prep of the 4 liquid cultures.</LI><LI>Figure 8. shows the double digested plasmids, EF New H had the correct bands.</LI></OL><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 8.</B>The gel consists of DNA extracted from 4 EF transformants, the EF plasmid was made through 3A assembly of the EF part 1 and part 2.</CENTER></FONT></FIGCAPTION></DIV></UL></DIV></UL></DIV></UL></DIV></HEADER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>