<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Macquarie_Australia_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Macquarie Australia/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="TopBar"><UL id="TopLvlNav"><LI><A href="https://2016.igem.org/Team:Macquarie_Australia">Home</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Description">Project</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/interlab">Interlab study</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/protocols">Protocols/Testing</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Safety">Safety</A></LI></UL><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Summary">Summary Table</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Part_Collection">Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Theoretical">Theoretical</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Experimental">Experimental</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Demonstrate">Demonstration</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Integrated_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/HP/Gold">Integration</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/HP/Silver">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Students">Team</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Students">Students</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Advisers">Advisors</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Singapore ">NTU Singapore</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Paris">Paris Bettencourt</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/South Africa">Other</A></LI></DIV><DIV class="ex1" align="justify"><H1 id="top"><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Our Project</SPAN></CENTER></FONT></H1><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040">The hydrogen industry produces over <B>50 million metric tons of hydrogen per year</B>, sourcing over <B>95% from fossil fuels </B>[1].With the growing demand in refinery hydroprocessing, the global market for hydrogen is projected to reach over 700 billion metric tons by 2018 [1]. The overall aim of our project is to generate a <B>clean, renewable source of hydrogen</B> by expressing genes of the photosynthetic pathway within <I>E. coli</I>. Our first goal is to assemble the <B>Chlorophyll biosynthesis pathway</B> in <I>E. coli</I> in order to metabolically engineer chlorophyll in this organism. Up to date, no researcher has been successful in producing chlorophyll in non-photosynthetic organisms before [2]. Once produced, the chlorophyll will bind to <B>Photosystem II</B>, which is another pathway that we seek to construct within <I>E. coli</I>, to generate hydrogen ions, oxygen and electrons from the oxidation of water. The hydrogen ions and electrons generated from Photosystem II can then be converted to hydrogen gas using a <B>Hydrogenase</B>. Our modelling and human practice approaches will allow us to assess the viability of our hydrogen production on an industrial scale.</FONT></DIV><DIV class="ex1" align="justify"><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Motivation</SPAN></CENTER></FONT></H1><FONT face="Arial" font="" size="5" color="#808080" font="" style="line-height:0.4"><CENTER>Climate Change</CENTER></FONT><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040">In the last few decades, clean renewable energy has emerged as the frontrunner for research into alternatives to fossil fuels. Whilst running out of fossil fuels is not an immediate concern, as we continue to discover and produce fossil fuel reserves, the pollutions effects inherent, which cause <B>climate change</B>. At this rate, we will surpass an atmospheric [CO<SUB>2</SUB>] of 700 ppmv well before 2100. This would be an irreversible global catastrophe and severe climate volatility would ensue [3,4]. The Paris Agreement on climate change has come into effect on the 5<SUP>th</SUP> of October, 2016. This plan outlines certain emission targets that countries promise to conform to. Huge steps have been made towards developing a clean, reliable energy source with hydrogen being the primary candidate. Hydrogen combusts to produce water and energy without producing harmful emissions. To generate hydrogen within <I>E. coli</I> we are building on the work of previous Macquarie iGEM teams. 
<FONT face="Arial" font="" size="5" color="#808080"><CENTER>Rural Vulnerability</CENTER></FONT><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040">More than one-quarter of the world’s population live in remote areas that lack access to grid electricity. As a result of this they rely heavily on local diesel generating systems to meet their electricity demands. This reliance is paralleled with a variety of challenges including; time delays, higher prices, supply problems and increased vulnerability to outages. 

Team Macquarie’s conceptual design, <A class="regularHyperlink" href="https://2016.igem.org/Team:Macquarie_Australia/Design#Design"><B>[The Living Battery]</B></A> will provide individuals living in remote areas of Australia with a portable, renewable and low cost energy source. The H<SUB>2</SUB> gas produced by “The Living Battery” can be used in fuel cells to power generators to meet the daily electricity demands of the community.</FONT><FONT face="Arial" font="" size="5" color="#808080"><CENTER>Our Contribution</CENTER></FONT>
Currently, the high price of photovoltaic cells, and the low efficiency of solar to hydrogen systems are delaying the transition from fossil fuels to a clean energy source [5]. The H<SUB>2</SUB> bio-chamber we are developing at Macquarie may hold the answers to this problem. Our hydrogen-producing, conceptual design could be used within rural communities; supplementing or even replacing traditional fossil fuels. An important part of this project is determining if our system will be embraced by our target market. A few of our team members ventured to outlying communities to talk to business owners and farmers. We discussed our technology and the concerns they had with adopting our technology. Based off their feedback we revised our <A class="regularHyperlink" href="https://2016.igem.org/Team:Macquarie_Australia/Design#Design"><B>[The Living Battery]</B></A> prototype so that it incorporated kill switches for safety and a sealed culture tank with an import valve to reduce the risk of tank contamination.</FONT><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Project Design</SPAN></CENTER></FONT></H1><FONT face="Arial" font="" size="5" color="#808080"><CENTER>Chlorophyll Biosynthesis Pathway</CENTER></FONT><FONT face="Arial" font="" size="3" color="#404040"><CENTER><B>Figure. 1</B> Chlorophyll Biosynthesis Pathway</CENTER></FONT><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040">Chlorophyll plays an essential role in the photosynthetic pathway as it has the ability to absorb and transfer energy from light [6]. Chlorophyll molecules are embedded in thylakoid membranes within chloroplasts. Chlorophyll absorbs energy from incoming photons and uses resonance energy transfer to excite the p680 reaction centre of Photosystem II.

Chlorophyll a synthesis (Fig. 1) can be significantly increased in <I>E. coli</I> by eliminating the heme production pathway.  We have achieved this by knocking out the <I>hemH</I> gene (encodes for Ferrochelatase), using CRISPR/Cas9 approach. Ferrochelatase utilises Protoporphyrin IX to make heme in the heme synthesis pathway (Fig. 2). Therefore, by making this enzyme dysfunctional, we can redirect all cellular Protoporphyrin IX to the Chlorophyll synthesis pathway [7]. We have designed and synthesised 12 gBlocks, which we hypothesise are necessary for chlorophyll biosynthesis. Using the 3A assembly method, we ligated these genes to form two large operons; operon 1 is 10.6 kbp and has been characterised as functional and operon 2 is 6.2 kbp but has not yet been characterised. We hope that the ligation of these two operons will allow us to characterise a unified chlorophyll biosynthesis part. Once chlorophyll a is synthesised within the <I>E. coli</I>, Photosystem II will have the energy required to oxidise water - producing hydrogen ions and electrons.  Researchers are yet to successfully synthesise chlorophyll a in a non-photosynthetic organism. There may be unknown post-translational mechanisms that are essential for tetrapyrrole synthesis under variable environmental conditions [8].</FONT><FONT face="Arial" font="" size="3" color="#404040"><CENTER><B>Figure. 2</B> Protoporphyrin IX Pathway</CENTER></FONT><FONT face="Arial" font="" size="5" color="#808080"><CENTER>Photosystem II Synthesis</CENTER></FONT><FONT face="Arial" font="" size="3" color="#404040"><CENTER><B>Figure. 3</B> Photosystem II </CENTER></FONT><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040">Photosystem II (Fig. 3) is one of the main protein complexes involved in the photosynthesis pathway. The complex is comprised of photons, chlorophyll-binding proteins that work with a pair of chlorophyll molecules (known as the P680 reaction centre), pheophytin molecules, plastaquinones, water, and oxygen. Photosystem II is a multimeric protein complex that binds a pair of chlorophyll molecules at its core, which work together to supply electrons to the electron transport system. This is done by splitting water molecules via absorbed photons in the Photosystem II reactions centre [9].

Developed from <I>Chlamydomonas reinhardtii</I>, the Photosystem II complex used has 17 genes. These 17 genes, involved with the Photosystem II construction, are grouped into 5 operons. In operon 1 there are 3 genes: <I>psbD</I>, <I>psbC</I>, and <I>psbA</I>. <I>psbA</I> encodes the major subunit D1, located in the reaction centre that forms a heterodimer with the D2 subunit encoded by <I>psbD</I> [10]. Attached to D1-D2 heterodimer is CP43, encoded by <I>psbC</I>, which is a subunit of the proximal antennae in Photosystem II [11].

Operon 2 consists of 3 genes <I>psbE</I>, <I>psbL</I>, and <I>psbJ</I>. <I>psbE</I> encodes for a cytochrome b559 subunit alpha [12]. <I>psbL</I> encodes the protein PsbL involved in the reaction centre and psbJ encodes for a core complex subunit [13]. Operon 3 consists of 2 genes <I>psbT</I> and <I>psbB</I>. <I>psbT</I> encodes for a small chloroplast hydrophobic polypeptide associated with the core complex [14], while <I>psbB</I> encodes for one of the subunits comprised in protein P680 of Photosystem II [15].

Operon 4 consists of 5 genes <I>psbM</I>, <I>psbZ</I>, <I>psbH</I>, <I>psbW</I>, and <I>psbK</I>. <I>psbM</I> and <I>psbH</I> encode for core complex subunit proteins [16] and <I>psbZ</I>, which is a stabilising protein for the Photosystem II/Light Harvesting Complex II [16]. <I>psbW</I> encodes for a transmembrane protein involved in Photosystem II dimer-stabilisation and photo-protection [17]. The final gene <I>psbK</I> encodes for a low molecular weight subunit of Photosystem II involved in stabilising the whole complex [18]. 

Operon 5 consists of 4 genes <I>psbO</I>, <I>psbP</I>, <I>psbQ</I>, and <I>psbR</I>. <I>psbO</I> encodes for a protein that acts to stabilise the cluster of four Mn<SUP>2+</SUP> that forms the catalytic centre of the oxygen evolving complex (OEC) [19]. <I>psbP</I>, encodes a protein that optimizes the availability of Ca<SUP>2+</SUP> and Cl<SUP>-</SUP> cofactors in the OEC in Photosystem II to maintain the active Manganese cluster [20]. <I>psbQ</I>, is involved with oxygen-evolving and the enhancer protein 3 [21]. <I>psbR</I> encodes for a protein essential for the stable assembly of <I>psbP</I>, that is a component of the OEC of photosystem II [22].
</FONT><FONT face="Arial" font="" size="5" color="#808080"><CENTER>Hydrogenase Synthesis</CENTER></FONT><FONT face="Arial" font="" size="3" color="#404040"><CENTER><B>Figure. 4</B> Hydrogen Production </CENTER></FONT><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040">In 1939, Hans Gaffron discovered that unicellular green algae were able to produce hydrogen [23]. This was because of Hydrogenase activity under very low oxygen conditions [24]. Later he attempted to decouple the Hydrogenase activity from the oxidation of water by expressing Hydrogenase in a non-photosynthetic organism, but was unsuccessful [25]. Within this project we are attempting to produce [Fe] Hydrogenase, ferredoxin, and maturation enzymes. In conjunction they reduce hydrogen ions and electrons to produce hydrogen gas. As this is a new addition to our project we have synthesised all four genes (<I>hyd1</I>, <I>hydEF</I>, <I>hydG</I>, and Ferredoxin), as gBlocks and successfully ligated all 4 gBlocks into Biobricks. We are yet to confirm the expression of each gene in <I>E. coli</I> for characterisation, which if successful will allow hydrogen production (Fig. 4).
<FONT face="Arial" font="" size="5" color="#808080"><CENTER>CRISPR/Cas9</CENTER></FONT><FONT face="Arial" font="" size="3" color="#404040"><CENTER><B>Figure. 5</B> HemH knock-out using CRISPR/Cas9 </CENTER></FONT><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040">CRISPR/Cas9 was discovered as a component in the immune system of <I>Streptococcus pyogenes</I>. Since its discovery CRISPR/Cas9 has been an effective tool for genomic manipulation within many organisms [26]. The ability of CRISPR to manipulate <I>E. coli</I> genes has been demonstrated previously, with a near 100% editing success rate for insertions and deletions for double and single stranded DNA [27]. 

We have incorporated this technique into our project to increase the efficiency and effectiveness of the Chlorophyll and Photosystem II pathways. By knocking out the <I>hemH</I> gene that produces Ferrachelatase in heme biosynthesis pathway [16] using CRISPR/Cas9 approach, more Protoporphyrin IX (PPIX) can be accumulated [7], providing more substrate for our Chlorophyll synthesis pathway. As a result, all cellular PPIX could be redirected for chlorophyll synthesis and can be used for the Photosystem II within <I>E. coli</I>.

We have also collaborated with the Singapore NTU iGEM team by testing the activity of their mammalian CRISPR/Cas9 in a bacterial system. Their Cas9 plasmids, including one wild type and two mutants, has been His-tag purified and coupled with our sgRNA in an vitro assay, in an attempt to characterise their parts.
<H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt"><SMALL>References</SMALL></SPAN></CENTER></FONT></H1><FONT size="2" color="#404040"><OL><FONT style="line-height:1.5" font="" face="corbel" font="" size="2" color="#404040"><LI>Sathre, R., et al., <I>Life-cycle net energy assessment of large-scale hydrogen production via photoelectrochemical water splitting</I>. Energy &amp; Environmental Science, 2014. 7(10): p. 3264-3278.</LI><LI>Brzezowski, P., et al., <I>Mg chelatase in chlorophyll synthesis and retrograde signaling in Chlamydomonas reinhardtii: CHLI2 cannot substitute for CHLI1</I>. Journal of Experimental Botany, 2016.</LI><LI>Goddéris, Y., et al., <I>Rates of consumption of atmospheric CO 2 through the weathering of loess during the next 100 yr of climate change</I>. Biogeosciences, 2013. 10(1): p. 135-148.</LI><LI>Sanborn Scott, D., Fossil Sources: <I>“Running Out” is Not the Problem.</I> International Journal of Hydrogen Energy, 2005. 30(1): p. 1-7.</LI><LI>Hosseini, S.E. and M.A. Wahid, <I>Hydrogen production from renewable and sustainable energy resources: Promising green energy carrier for clean development.</I> Renewable and Sustainable Energy Reviews, 2016. 57: p. 850-866.</LI><LI>Eckhardt, U., B. Grimm, and S. Hörtensteiner, <I>Recent advances in chlorophyll biosynthesis and breakdown in higher plants</I>. Plant molecular biology, 2004. 56(1): p. 1-14.</LI><LI>Jacobs, J.M. and N.J. Jacobs, <I>Oxidation of protoporphyrinogen to protoporphyrin, a step in chlorophyll and haem biosynthesis. Purification and partial characterization of the enzyme from barley organelles.</I> Biochemical journal, 1987. 244(1): p. 219-224.</LI><LI>Czarnecki, O. and B. Grimm, <I>Post-translational control of tetrapyrrole biosynthesis in plants, algae, and cyanobacteria.</I> Journal of Experimental Botany, 2012.</LI><LI>McEvoy, J.P. and G.W. Brudvig, <I>Water-splitting chemistry of photosystem II.</I> Chemical reviews, 2006. 106(11): p. 4455-4483.</LI><LI>Marder, J.B., et al., <I>Identification of psbA and psbD gene products, D1 and D2, as reaction centre proteins of photosystem 2.</I>  Plant Molecular Biology, 1987. 9(4): p. 325-333.</LI><LI>Bricker, T.M. and L.K. Frankel, <I>The structure and function of CP47 and CP43 in Photosystem II. </I> Photosynthesis Research, 2002. 72(2): p. 131.</LI><LI>Mor, T.S., et al., <I>An unusual organization of the genes encoding cytochrome b559 in Chlamydomonas reinhardtii: psbE and psbF genes are separately transcribed from different regions of the plastid chromosome.</I> Mol Gen Genet, 1995. 246(5): p. 600-4.</LI><LI>Nowaczyk, M.M., et al., <I>Deletion of psbJ leads to accumulation of Psb27–Psb28 photosystem II complexes in Thermosynechococcus elongatus </I>. Biochimica et Biophysica Acta (BBA)-Bioenergetics, 2012. 1817(8): p. 1339-1345.</LI><LI>Ohnishi, N. and Y. Takahashi, <I>PsbT polypeptide is required for efficient repair of photodamaged photosystem II reaction center. </I> Journal of Biological Chemistry, 2001. 276(36): p. 33798-33804.</LI><LI>Cameron, K.M. and M. Carmen Molina, <I>Photosystem II gene sequences of psbB and psbC clarify the phylogenetic position of Vanilla (Vanilloideae, Orchidaceae). </I> Cladistics, 2006. 22(3): p. 239-248.</LI><LI>Ferreira, K.N., et al., <I>Architecture of the photosynthetic oxygen-evolving center.</I> Science, 2004. 303(5665): p. 1831-1838.</LI><LI>Woolhead, C.A., et al., <I>Conformation of a purified “spontaneously” inserting thylakoid membrane protein precursor in aqueous solvent and detergent micelles.</I> Journal of Biological Chemistry, 2001. 276(18): p. 14607-14613.</LI><LI>Takahashi, Y., et al., <I>Directed disruption of the Chlamydomonas chloroplast psbK gene destabilizes the photosystem II reaction center complex.</I> Plant molecular biology, 1994. 24(5): p. 779-788.</LI><LI>Murata, N. and M. Miyao, <I>Extrinsic membrane proteins in the photosynthetic oxygen-evolving complex</I>. Trends in Biochemical Sciences, 1985. 10(3): p. 122-124.</LI><LI>Roose, J.L., K.M. Wegener, and H.B. Pakrasi, <I>The extrinsic proteins of photosystem II.</I> Photosynthesis Research, 2007. 92(3): p. 369-387.</LI><LI>Balsera, M., et al., <I>The 1.49 Å resolution crystal structure of PsbQ from photosystem II of Spinacia oleracea reveals a PPII structure in the N-terminal region. </I> Journal of molecular biology, 2005. 350(5): p. 1051-1060.</LI><LI>Suorsa, M., et al., <I>PsbR, a missing link in the assembly of the oxygen-evolving complex of plant photosystem II </I> Journal of Biological Chemistry, 2006. 281(1): p. 145-150.</LI><LI>Gaffron, H., <I>Carbon Dioxide Reduction with Molecular Hydrogen in Green Algae.</I> American Journal of Botany, 1940. 27(5): p. 273-283.</LI><LI>Lakatos, G., et al., <I>Bacterial symbionts enhance photo-fermentative hydrogen evolution of Chlamydomonas algae.</I> Green Chemistry, 2014. 16(11): p. 4716-4727.</LI><LI>Stuart, T.S. and H. Gaffron, <I>The mechanism of hydrogen photoproduction by several algae : I. The effect of inhibitors of photophosphorylation</I>. Planta, 1972. 106(2): p. 91-100.</LI><LI>Zhang, F., Y. Wen, and X. Guo, <I>CRISPR/Cas9 for genome editing: progress, implications and challenges</I>. Hum Mol Genet, 2014. 23(R1): p. R40-6.</LI><LI>Li, Y., et al., <I>Metabolic engineering of Escherichia coli using CRISPR-Cas9 meditated genome editing</I>. Metab Eng, 2015. 31: p. 13-21.</LI><LI>Ferreira, G.C., et al., <I>Structure and function of ferrochelatase</I>. J Bioenerg Biomembr, 1995. 27(2): p. 221-9.</LI></FONT></OL></FONT></FONT></FONT></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>