<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Macquarie_Australia_CRISPRSummary skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Macquarie Australia/CRISPRSummary</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="TopBar"><UL id="TopLvlNav"><LI><A href="https://2016.igem.org/Team:Macquarie_Australia">Home</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Description">Project</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/interlab">Interlab study</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/protocols">Protocols/Testing</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Safety">Safety</A></LI></UL><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Summary">Summary Table</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Part_Collection">Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Theoretical">Theoretical</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Experimental">Experimental</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Demonstrate">Demonstration</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Integrated_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/HP/Gold">Integration</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/HP/Silver">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Students">Team</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Students">Students</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Advisers">Advisors</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Singapore ">NTU Singapore</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Paris">Paris Bettencourt</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/South Africa">Other</A></LI></DIV><DIV class="ex1" align="justify"><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"> The aim of our work with Clustered Regularly Interspaced Short Palindromic Repeats (CRIPSR) was to metabolically engineer the increase accumulation of our chlorophyll synthesis precursor, protoporphyrin-IX (PPIX). This was achieved by disrupting the  <I> hemH </I> gene, there by decreasing the production of hemH, preventing the diversion of the PPIX towards heme, increasing the accumulation of PPIX. The results of this work can be seen below. Mutant colonies were detected by an increase of PPIX seen using fluorescence spectroscopy. Amongst the mutants generated, we selected the highest accumulating mutant for modelling experiments and device verification of our Mg-Chelatase Plasmid. </FONT></DIV><DIV class="ex1" align="justify"><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Summary of Results</SPAN></CENTER></FONT></H1><OL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>A functional Cas9 RNP complex was demonstrated by specifically cleaving the <I>hemH</I> gene PCR product as expected.</LI><LI>The Cas9 RNP complex was functional <I>in vivo</I> as demonstrated by its effect on <I> E. coli</I> by killing them in a concentration dependent manner, which is expected if excess double stranded breaks are made in a <I>RecA</I> minus strain.</LI><LI>The Cas9 mutagenesis produced a <I>hemH</I> mutant of E.coli which accumulates protoporphyrin IX.</LI></FONT></OL></DIV><DIV class="ex1" align="justify"><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Results</SPAN></CENTER></FONT></H1><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Metabolically Engineering with CRIPSR Results</SPAN></CENTER></FONT></H1><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040">We have designed primers to amplify the <I>hemH</I> gene using <I>E.coli</I> genomic DNA as a template. The results can be seen in the amplification products of Figure 1(a). Figure 1(b) indicates that the Cas<FONT face="arial">9</FONT> RNP complex works <I> in vitro </I> 
and cleaves the PCR product into the expected sizes as designed by the target guide RNA. The expected sizes of the cleaved products 
are approximately 850/150bp. The incomplete cleavage of the PCR product is likely to be due to a insufficient amounts of the 
Cas<FONT face="arial">9</FONT> guide RNA complex.</FONT></DIV><DIV class="ex2" align="justify"><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><B>Fig 1(a).</B> Ferrochelatase PCR products using our designed primers used to amplify the <I>hemH</I> gene (990bp).
<B>Fig 1(b).</B><I>In vitro</I> assay indicating the function of Cas9. Expected sizes can be seen according to the designed target guide RNA.</FONT></DIV><DIV class="ex1" align="justify"><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040">The cell viability curve below outlines the survival of cells after electroporation of the Cas<FONT face="arial">9</FONT> RNP 
complex <I> in vivo</I>. Higher concentration of Cas<FONT face="arial">9</FONT> RNP complex results in lower cell viability correlating with an 
increase of double stranded breaks. This indicates that the Cas<FONT face="arial">9</FONT> RNP complex is efficiently introduced in to the bacteria cells without a loss of 
function. These results were also observed by Dutra <I> et al </I> [1].</FONT></DIV><DIV class="ex2" align="justify"><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><B>Fig 2.</B> Cell viability curve of CFU/mL against increasing Cas9 concentration. The curve indicates that increasing Cas9 concentration inversely effects cell viability.</FONT></DIV><DIV class="ex1" align="justify"><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040">Individual colonies from the serial dilution plates were patch plated from M9-glucose to M9-glycerol plates which is expected to cause increased production of heme to allow oxidative metabolism of the glycerol. These patched colonies were screened for the accumulation of PPIX by fluorescence spectroscopy 
(Ex 404nm/ Em 600-700nm and Em 630/Ex 350-500nm). The emission peak at 630nm and excitation peak at 404nm indicates the presence of PP-IX. The excitation spectra from this screening is shown below.</FONT></DIV><DIV class="ex2" align="justify"><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><B>Fig 3. </B>Fluorescence spectroscopy results demonstrating the accumulation of PPIX at 404nm excitation. (A) Mutants are indicated by peaks at 620nm suggesting PPIX accumulation. (B) Control assay grown on glycerol, showing no changes in peak emission.</FONT></DIV><DIV class="ex1" align="justify"><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040">We found 12 colonies which were slightly coloured on the cells from Cas<FONT face="arial">9</FONT> RNP complex colonies patch plated from M9-glucose to M9-glycerol plates. These were screened for fluorescence and showed PPIX accumulation. The highest accumulator, mutant #1, which was the most coloured was selected for further experiments.

Mutant #1 was streaked out to single colonies and the single colonies were screen for PPIX accumulation. The fluorescence emission spectrum (Ex 404nm) of two of these colonies is shown below, indicating significant accumulation of protoporphyrin IX within these cells. This mutant was then made competent and used in further <A class="regularHyperlink" href="https://2016.igem.org/Team:Macquarie_Australia/Model"><B>experiments</B></A>.</FONT></DIV></DIV><DIV class="ex2" align="justify"><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><B>Fig 4.</B> Mutant #1 was selected and further screened. Accumulation of PPIX was confirmed using further fluorescence spectroscopy assays. </FONT></DIV></DIV></DIV></DIV></DIV></BODY></HTML>