<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Macquarie_Australia_CRISPR skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Macquarie Australia/CRISPR</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="TopBar"><UL id="TopLvlNav"><LI><A href="https://2016.igem.org/Team:Macquarie_Australia">Home</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Description">Project</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/interlab">Interlab study</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/protocols">Protocols/Testing</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Safety">Safety</A></LI></UL><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Summary">Summary Table</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Part_Collection">Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Theoretical">Theoretical</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Experimental">Experimental</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Demonstrate">Demonstration</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Integrated_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/HP/Gold">Integration</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/HP/Silver">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Students">Team</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Students">Students</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Advisers">Advisors</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Singapore ">NTU Singapore</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/Paris">Paris Bettencourt</A></LI><LI><A href="https://2016.igem.org/Team:Macquarie_Australia/South Africa">Other</A></LI></DIV><HEADER><DIV class="ex1" align="justify"><DIV class="ex2" align="justify"><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Week 6</SPAN></CENTER></FONT></H1><FONT face="Arial" font="" size="5" color="#808080" font="" style=""><CENTER>(5th - 11th September)</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>We then performed a colony PCR to amplify the ferrochelatase gene to confirm its presence within BL21 and DH5<I>a</I><I>E. coli</I> cells.</LI><LI>We further optimised the PCR to obtain clean PCR products for our <I>in vitro</I> test (fig 2).</LI></FONT></UL><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 1.</B> Shows the ferrochelatase PCR products (990bp) shown by the bold bands and by-products shown in lower concentrations below</CENTER></FONT></FIGCAPTION></DIV><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 2.</B> Shown here are the PCR optimisation for ferrochelatase gene conformation from different <I>E.coli</I> strains.</CENTER></FONT></FIGCAPTION></DIV><DIV class="ex3" align="justify"><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Week 7</SPAN></CENTER></FONT></H1><FONT face="Arial" font="" size="5" color="#808080" font="" style=""><CENTER>(12th - 18th September)</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>Following the PCR amplification another gel cast to clean up the products and allow or better <I>in vitro</I> testing.</LI><LI>Using our amplified PCR products and transformed cells we performed an <I>in vitro</I> test of the commercial Cas9 by digesting it for 30 minutes at 37<SUP>o</SUP>C. This showed that the Cas9 cuts our PCR product at the right location and that our <I>in vitro</I> testing is appropriate as well.</LI><LI>The <I>in vitro</I> testing was performed the vitro test under similar condition to optimise the Protocol.
Using pre temps, then re amplified and cut just the top 1000bp band for purity and then for Cas9 <I>in vitro</I> test.</LI></FONT></UL><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 3.</B> Shown here are results of a colony PCR using the ferrochelatase as a primer, which allowed for our vitro testing of the Cas9 proteins as ferrochelatase is cut by our Cas9 at 990bp. This gel was to clean up our PCR amplification products for future <I>in vitro</I> testing.</CENTER></FONT></FIGCAPTION></DIV><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 4.</B> Shown here are results of a colony PCR using the ferrochelatase as a primer, which allowed for our vitro testing of the Cas9 proteins as ferrochelatase is cut by our Cas9 at 990bp. This gel was to clean up our PCR amplification products for future <I>in vitro</I> testing.</CENTER></FONT></FIGCAPTION></DIV><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 5.</B> Shown here is confirmation that our Cas9 cuts our PCR products, which we previously generated. Labelled DH5<I>a</I> and Bl21 shown without Cas9 results in only one band as expected demonstrating that the <I>hemeH</I> gene here is not cut. Then as shown in the lanes labelled BL21+Cas9 and DH5<I>a</I>+Cas9 two bands can be seen demonstrating that the Cas9 is cutting the <I>hemeH</I> gene into two pieces from PCR products of both <I>E.coli</I> strains.</CENTER></FONT></FIGCAPTION></DIV><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 6.</B> Shown here are an attempt to amplify the ferrochelatase PCR products to give help with <I>in vitro</I> testing. Although because many smaller amplification products occurred they could not be used.
</CENTER></FONT></FIGCAPTION></DIV><DIV class="ex4" align="justify"><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Mid-semester Break</SPAN></CENTER></FONT></H1><FONT face="Arial" font="" size="5" color="#808080" font="" style=""><CENTER>(19th September - 2nd October)</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>Colonies from DH5<I>a</I>, BL21 and NEB10B underwent two dilutions; 1 colony in 100 µL and 1 colony in 1000 µL.</LI><LI>Dilutions were boiled for 5 mins (100 µL samples) or 10 mins (1000 µL samples). Samples then pipetted up and down 2-4 times before centrifuging for 10 mins at 13, 400g.</LI>
A 1:1 ratio of PCR MasterMix was made containing:
<LI>20 µL of [10x] KOD DNA polymerase buffer</LI><LI>20 µL of [10x] dNTPs.</LI><LI>20 µL of [25mM] MgSO4.</LI><LI>4 µL forward primer 1/10 dilution.</LI><LI>4 µL reverse primer 1/10 dilution.</LI><LI>4 µL KOD DNA polymerase.</LI><LI>28 µL Milli-Q water.</LI>
Final solution=100 µL of 2x MasterMix Ferrecheletase PCR reaction:
<LI>5 µL of template supernatant for each sample with 5 µL of MasterMix.</LI><LI>Negative control= 5 µL MasterMix with 5 µL Milli-Q water.</LI><LI>Positive control= 5 µL MasterMix with 5 µL of DH5<I>a</I> (Successful previous ferrecheletase PCR mix).</LI><LI>Reaction: 30 cycles starting at 5 mins at 95°C, 30s at 95°C, 15s at 55°C, 45s at 70°C and final extension of 5 mins at 72°C.</LI><LI>A 1% agarose (w/v) was prepared in [1x] Tris base, acetic acid and EDTA (TAE) buffer with 2 µL gel red. 2 µL of each sample was mixed with 1 µL purple loading dye and 4 µL TAE buffer, except positive control which had 1 µL of sample, 1 µL cas9, 1 µL purple loading dye and 4 µL TAE buffer. 1kb ladder was loaded. A positive and negative control were loaded for PCR reaction, and another positive and negative control were loaded for Cas9. 
Gel was run for 60 mins at 90V.</LI><LI>Lanes containing BL21 (1/100), DH5<I>a</I> (1/100) and DH5<I>a</I> (1/1000) showed band weights of ~800bp.
These PCR reaction tubes were then incubated for 15 mins at 37°C with the Cas9 and re run on the gel for 60min at 110V. Image of gel showed that both DH5<I>a</I> diluted strains were successfully cut at ~800bp and ~200bp showing that the Cas9 works for the ferrecheletase PCR product.</LI>
Following samples plated onto glucose and glycerol M9 plates :
<LI>3µM 104 : Glucose (966), Glycerol (888).</LI><LI>17µM 104 : Glucose (40), Glycerol (38).</LI><LI>3µM 106 : Glucose (5), Glycerol (18).</LI><LI>17µM 104 : Glucose (0), Glycerol (0).</LI><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 7.</B> Cell viability curve of CFU/mL against increasing Cas9 concentration. The curve indicates that increasing Cas9 concentration inversely effects cell viability.</CENTER></FONT></FIGCAPTION></DIV></FONT><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 8.</B> Fluorescence spectroscopy results demonstrating the accumulation of PPIX at 404nm excitation. (A) Mutants are indicated by peaks at 620nm suggesting PPIX accumulation. (B) Control assay grown on glycerol, showing no changes in peak emission.</CENTER></FONT></FIGCAPTION></DIV><LI>From these results samples of 7, 12 and 17 were sent off for sequencing.</LI><LI>For 7, 12 and 17 we obtained wild type sequences for ferrochelatase, although no sequence results were sent off for number 1.</LI><LI>Sample 1 was re-amplified a number of times and no ferrochelatase product was detected? This indicates possible detection and is consistent with the accumulation of PPIX in this strain.</LI></UL><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 9.</B> Shown here for the samples, 18, 23 and 33 show an increase of PPIX accumulation within the cells. This allowed for selection of the respective cells.</CENTER></FONT></FIGCAPTION></DIV><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 10.</B> Shows good example of PPIX accumulation in the last spectrum. This allowed for cell selection for further work.</CENTER></FONT></FIGCAPTION></DIV><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 11.</B> Shows no accumulation of PPIX within cells transformed with 3uM of RNP.</CENTER></FONT></FIGCAPTION></DIV><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 12.</B> Shows no accumulation of PPIX within cells transformed with 17uM of RNP.</CENTER></FONT></FIGCAPTION></DIV><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 13.</B> Confirm PPIX accumulation in our selected mutant (mutant #1) for <I>hemH</I> knock-out.</CENTER></FONT></FIGCAPTION></DIV><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 14.</B> Shown here all the peaks shown an accumulation of PPIX. Especially the first peak, which shows a strong accumulation of PPIX from our mutant colony #1.</CENTER></FONT></FIGCAPTION></DIV><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 15.</B> Shown here all the peaks shown an accumulation of PPIX. Especially the first peak, which shows a strong accumulation of PPIX from our mutant colony #1.</CENTER></FONT></FIGCAPTION></DIV><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 16.</B> Shown here are 5’ ferrecheletase and 3’ ferrecheletase PCR products produced for our <I>in vitro</I> testing of the Cas9. Which, we will use are 5’ and 3’ flanking regions to be used in Cas9 insertional mutagenesis. The bands are at the expected sizes of 120bp and 450bp are shown here.</CENTER></FONT></FIGCAPTION></DIV><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 17.</B> Shown here is different concentration of BL21 (Lanes 1-3) and NHEJ (Lane 4). Sample 1 was re-amplified with no ferrochelatase detected. This can be attributed to the repair mechanisms removal of primers after breaking the double strand.</CENTER></FONT></FIGCAPTION></DIV><B>NTU-Singapore</B><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>During this our midsemester break we finally obtained the Cas9 plasmid from NTU-Singapore for our collaboration named Wt (wild type), 459 (mutant Cas9) and 462 (mutant Cas9). We then transformed these plasmids into DH5<I>a</I><I>E. coli</I> cells. Following a standard transformation, we then cultured colonies.</LI><LI>After culturing the OD was checked, the 100ul of each type Wt, 459 and 662 was put in 2000ml of LB media and left at 20C to incubate for 24 hours.</LI><LI>Following this the cells were pelleted and the lysate loaded onto a IMAC column.</LI><LI>After protein purification the elution was tested in Bradford reagent for protein concentration, unfortunately no proteins were obtained. We concluded that the elution solution was not strong enough 100M concentration and we performed this again with a 500M concentration, which resulted in a slight positive Bradford elution.</LI><LI>A protein gel was run to test the purified proteins but no proteins at the Cas9 size could be seen, at this point we realised that we were not using a cell line with a T7 promoters and this would result in our cas9 protein not been expressed.</LI><LI>To correct our errors, we re-transformed the NTU-Singapore cas9 into cells, which have T7 promoters.</LI><LI>We then performed the protein culturing steps again and isolated then eluted the say way as mentioned previously. The results from this Bradford were also positive and the 3 sample Wt, 459 and 662 were run on a protein gel with the results indicating that we successfully purified out the cas9 protein from Singapore.</LI><LI>When then performed the same <I>in vitro</I> testing outline earlier to test its functionality, which resulting in a successful visualisation as the Cas9 cut the PCR product into two portions (1000bp and 200bp).</LI><LI>Further testing was done to try and optimise this system.</LI></FONT></UL><DIV class="ex5" align="justify"><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Week 8</SPAN></CENTER></FONT></H1><FONT face="Arial" font="" size="5" color="#808080" font="" style=""><CENTER>(3rd - 9th October)</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>1 colony of mutant heme #1 was diluted in 100 µL and 1000 µL Milli-Q water and 1 µL of DH5<I>a</I> pure PCR product in 100 µL and 1000 µL Milli-Q water.</LI><LI>Dilutions were boiled for 5 mins (100 µL samples) or 10 mins (1000 µL samples). Samples then pipetted up and down 2-4 times before centrifuging for 10 mins at 13, 400g.</LI><LI>1:1 PCR MasterMix was used.</LI><LI>5 µL of template supernatant for each mutant heme #1 sample with 5 µL of MasterMix.</LI><LI>Negative control= 5 µL MasterMix with 5 µL Milli-Q water.</LI><LI>Positive control= Both DH5<I>a</I> 5µL MasterMix with 5 µL of DH5<I>a</I> (Successful previous ferrecheletase PCR mix but therefore did not need to re-undergo PCR amplification).</LI><LI>Reaction: 30 cycles starting at 5 mins at 95°C, 30s at 95°C, 15s at 55°C, 45s at 70°C and final extension of 5 mins at 72°C.</LI><LI>None of samples were incubated with Cas9. Just wanted to obtain ~800bp weight bad on agarose gel for mutant heme #1.</LI><LI>1% (w/v) agarose gel was prepared in [1x] TAE buffer. Loaded samples contained 4 µL TAE buffer, 1 µL purple loading dye and 1 µL sample. Gel run for 60 mins at 100V. 1kb ladder loaded.</LI><LI>No desired weight bands were detected.</LI><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 18.</B> Shown here are results of the colony PCR for ferrochelatase. Here the forward primer was not included, this attempt to see if the mutant appears on the PCR, but due to faulty controls this cannot be used.</CENTER></FONT></FIGCAPTION></DIV></FONT>
This week we also started a modelling “feeding study”: 
This involved making 3 concentrations of ALA for each liquid culture: 0mM, 2mM, 5mM. Had 4 tubes for each concentration for each culture for extraction on days 1, 3, 5 and 7. 3*3*4 = 36 tubes.<LI>IPTG was added to induce the Mg-chelatase plasmid (as it has a lac operon).</LI><LI>AMP was added as the cells were grown with ampicillin resistance.</LI><LI>All tubes were incubated on the shaker.</LI><LI>6th October 2016, Generated liquid cultures of <I>hemH</I> mutant cells, hemH + Mg-chelatase plasmid and wildtype <I>E. coli</I> of DH5<I>a</I> cells using 10mL of LB broth with a line of cells from plates and mixed until opaque. Incubated on the shaker until the optical densities were approximately 0.8.</LI><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 19.</B> This protein gel shows the crude products of the Cas9 proteins from NTU-Singapore. This gel shows the overexpressed bands of the Cas9 protein at 155kD we compared to the protein standards.</CENTER></FONT></FIGCAPTION></DIV><LI>7th October 2016, Removed the day one cultures and froze them.</LI><LI>9th October 2016, Removed the day three cultures and froze them.</LI></UL><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 20.</B> WT and 459 + 662 Cas9 Mutant runs.</CENTER></FONT></FIGCAPTION></DIV><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 21.</B> Repeat gel run of WT bands.</CENTER></FONT></FIGCAPTION></DIV><DIV class="ex6" align="justify"><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Week 9</SPAN></CENTER></FONT></H1><FONT face="Arial" font="" size="5" color="#808080" font="" style=""><CENTER>(10th - 16th October)</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>11th October 2016:</LI><LI>Removed the day five cultures and froze them.</LI><LI>13th October 2016:</LI><LI>Removed the day seven cultures. Thawed out each culture and took photos of the tubes for visual analysis. Added 50l sample and 450l methanol for protein extraction. Measured spectrofluorescence at excitation levels for 420nm for MgPPIX and 404nm for PPIX. Measured 550nm-700nm spectrum. Generated graphs of the 620nm values (coproporphyrin III) and 630nm values (protoporphyrin IX).</LI><LI>Generated IPTG samples with <I>hemH</I> mutant + Mg-chelatase plasmid as the table above states. Incubated the samples.</LI><LI>14th October:</LI><LI>Removed the day one cultures and froze them.</LI><LI>16th October:</LI><LI>Removed the day three cultures and froze them.</LI></FONT></UL><DIV class="ex7" align="justify"><H1><FONT face="Arial"><CENTER><SPAN style="font-weight:normal; font-size: 23pt">Week 10</SPAN></CENTER></FONT></H1><FONT face="Arial" font="" size="5" color="#808080" font="" style=""><CENTER>(16th - 19th October)</CENTER></FONT><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>17th October, Removed the day five cultures. Ran the same analyses with spectrofluorescence as indicated above and added in more graphs of the PPIX and CPIII values obtained.</LI><LI>We discarded the supernatant and resuspended the pellets of the day 5 0mM, 2mM and 5mM Mg-chelatase plasmid + mutant + IPTG cells in 1ml of SOC and 15μl of 1M MgCl2 and incubated for an hour. We re-recorded the fluorescence of the three samples and generated another graph as described above.</LI><LI>This week we also finally managed to isolate the Cas9 from Singapore and then to purify it. Once this was achieved we performed our <I>in vitro</I> testing protocol to test its function. The result of the <I>in vitro</I> test can be seen in the gel below.</LI></FONT></UL><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 22.</B> SDS-PAGE gel shows cas9 plasmids for Singapore NTU have been expressed and purified in our system. The first 3 samples are crude lysate, and the last 3 samples are His-tag purified proteins. Cas9 can be seen at approximately 160kDa. </CENTER></FONT></FIGCAPTION></DIV><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 23.</B> Shown here are the <I>in vitro</I> confirmations of the functional CRISPR/Cas9 from NTU-Singapore. The three Cas9 proteins studied in this experiment: wild type (Wt), 459 mutant, and 462 mutant, were seen to cut our Ferrochelatase PCR product target amplified in earlier weeks. This digestion was confirmed against the negative control without Cas9, and the commercial Cas9, as the Ferrochelatase was seen to be accurately cleaved into a 1000bp fragment and a 200bp fragment. Due to our efforts we have proven their mutant Cas9 is functional within <I>E. coli</I> , which helps to achieve their main goal of characterising their mutant Cas9 within bacterial systems.</CENTER></FONT></FIGCAPTION></DIV><DIV class="align: center"><FIGCAPTION><FONT face="Corbel" font="" style="line-height:1.5" font="" size="2" color="#404040"><CENTER><B>Fig 24.</B> Shown here is the result to test the effect of Cas9 on <I>E. coli</I>. It can seen here by the graph that the more cas9 there is within our cells the more toxic it becomes and the less they survive.</CENTER></FONT></FIGCAPTION></DIV><UL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI>Cas9 electorporation procedure: Cas9-RNP complex was mixed with the electrocompetent cells, and electroporated as described in the protocols page. After recovering for 2 hours in SOC media, the cells were serially diluted (10-2, 10-4, 10-6) and plated out on M9-glucose and M9-glycerol plates for incubation at 37<SUP>o</SUP>C. The viable colony count was conducted, and colonies from the M9-glucose plates were patched plated onto another set of M9-glucose and M9-glycerol plates. Accumulation of PPIX was then measured for each colony to detect for possible <I>hemH</I> mutants.</LI></FONT></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></HEADER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>