Team:Macquarie Australia/NoteNBchloro
Home
Project
Description
Results
Interlab study
Protocols/Testing
Notebook
Safety
Parts
Summary Table
Best Composite Part
Part Collection
Proof of Concept
Model
Theoretical
Experimental
Demonstration
Human Practices
Integration
Outreach
Team
Students
Advisors
Sponsors
Achievements
Collaborations
NTU Singapore
Paris Bettencourt
Other
NOTE:
Super Operon refers to Magnesium chelatase plasmid [BBa_K1998000] .
Mega Operon refers to Magnesium protoporphyrin IX to Chlorophyll a [BBa_K1998013]
Week 1
(1st - 7th August)
Firstly, the gene ChlH was digested with XbaI and PstI and ligated with the gene pLac which was digested with SpeI and PstI.  This resulted in the newly formed bio-brick pLac-ChlH. Addition of the lac promoter allowed us to control the expression of the ChlH gene in our composite parts through the introduction of IPTG in the growth media.
Following this, the pre-existing bio-brick pLac-ChlI-ChlD-GUN4 was digested with XbaI and PstI and ligated with ChlI2 which was digested with SpeI and PstI.
The resulting operon was then transformed into DH5 strains of E.coli . The plasmids were isolated from the transformant colonies, and PCR and gel electrophoresis was performed to verify the presence of each gene on the operon.
Fig 1. A lac promoter (pLac – 200bp) was incorporated into the front of CHLH (400bp), and a digest of the two genes was run on a gel. Lane 3 shows a shorter gene sequence and was excluded. Lanes 4-10 show the correct gene size for the pLac+ChlH gene and were used for sequencing.
Fig 2. The bio-brick pLac-YCF54-CHLM was digested and amplified through PCR using the lac forward and YCF54 reverse primers. The results were run on a gel, and the bands are indicative of a correct construction of this bio-brick.
Fig 3. Super operon was digested and each biosynthetic pathway gene was amplified through PCR. The bands on the gel correspond to each gene (CHLH-CHLI1-CHLD-GUN4-CHLI1-CTH1) which was successfully incorporated into the super operon.
Fig 4. Plasmid digest of the super operon and the plasmid backbone was run on a gel and the observed sizes corresponded to the expected sizes (Super Operon – 10,600 bp, plasmid backbone – 2000).
Week 2
(8th - 14th August)
The operon containing the genes pLac-ChlI-ChlD-GUN4-ChlI2-CTH1-pLac-ChlH (Super Operon) was re-transformed into DH5a strains of E.coli and cultured on ampicillin media overnight at 37oC.
This was done in order to increase the amount of available operon needed for further experimentation.
Following this, digestions and ligations of the genes YCF54-ChlM-POR-ChlP-DVR1-ChIG and an ampicillin resistant backbone (Mega Operon) was performed in preparation for transformation into DH5a E.coli cells in the following week.
Fig 5. The bio-bricks pLac-YCF54-CHLM (1500 bp) and DVR1-CHLD(4000bp) was digested a run on a gel. The bands seen on the gel correspond with expected sizes.
Week 3
(15th - 21st August)
The Super Operon containing transformant cells did not grow on the media due to experimental errors, fortunately, quantities of the Super Operon previously prepared were found, and used for the remainder of the experimental series.
Complete ligation mix was double digested and electrophoresed to confirm the presence of the Mega Operon according its expected size 6.2kb.
The results of the electrophoresis showed consistent bands around the 5-6kb region, indicating a successful ligation of the genes compiling the Mega Operon.
Following this, the Mega operon was transformed into DH5a and plated onto LB ampicillin media, and incubated overnight at 37oC.
The Mega Operon transformants showed growth on the ampicillin media, in preparation for plasmid mini prep and digestion in the following week.
Week 4
(22nd - 28th August)
18 colonies were then isolated from the plates, and plasmid was mini-prepped, digested and underwent gel electrophoresis to confirm the presence of the Mega Operon isolated from the transformant cells.
The Super Operon retrieved in the previous week was also run on the gel in order to assess the quality of the operon and determine whether it was appropriate to use for further experiments.
The gel indicated a successful ligation of the Mega Operon genes, as the size of the bands observed on the gel (roughly 6kb) corresponded with the expected size of the Mega operon (6.2kb). To validate the results of the Mega Operon, its genes were re-ligated and run on a gel in the following week.
The Super Operon bands corresponded with its expected size on the gel (10.5 kb) making it appropriate to use in further experiments.
Fig 6. Mini-prep of the Mega Operon and Super Operon was performed and the results were run on a gel.  The gel shows 6 bands at roughly 10,600bp for the super operon and 16 bands at roughly 8000bp for the mega operon.
The Super Operon and Mega Operon were prepared for ligation. First, the Super Operon was digested with the restriction endonucleases, SpeI and PstI and then treated with alkaline phosphatase immediately after digestion to avoid relegation.  The Mega Operon was digested with the enzymes Xbal and PstI.  These two digests were then ligated together directly.  Some of the Mega and Super Operons were also ligated together using the 3A assembly method on a KAN backbone using the enzymes EcoR1 and Spel.
Week 5
(29th - 4th September)
Following the ligation of the Mega and Super Operon together into an Ultra Operon, PCR and gel electrophoresis was performed to assess the success of the ligation. The gel results indicated a negative result, due to the large size of the operon and multiple regions of primer annealing.
The products of the ligation were also transformed into three different strains of E. coli ; DH10B, BL21 and DH5a.  The transformants were cultured for 37oC.
Following the incubation, DH5a strains grew on the media, suggesting a successful ligation and transformation of the Ultra Operon.
Week 6
(5th - 11th September)
From the DH5a colonies possessing the Ultra Operon, the plasmids were isolated and prepped for PCR to confirm the presence of the Ultra Operon.  A total of 75 plasmid samples underwent PCR and were prepared for analysis via gel electrophoresis in the following week.
Week 7
(12th - 18th September)
To identify successful ligation of the mega operon and super operon, the 75 PCR products that were prepared in the previous week were visualised using gel electrophoresis.  Along with these PCR products KAN and CAM backbones were also digested and run on the gel to ensure correct size, confirming backbone functionality.
No bands were visible across all 75 PCR samples indicating unsuccessful ligation of the super operon and mega operon.  KAN and CAM backbone samples showed bands at 2kb region, agreeing with the expected size, indicating that these backbones are still appropriate to use in future.
Fig 7. Part 1 of PCR of the ligation between Mega Operon + Super Operon (Ultra Operon) shows no evident bands across all 75 samples.
Fig 8. Part 2 of PCR of the ligation between Mega Operon + Super Operon (Ultra Operon) shows no evident bands across all 75 samples.
Mid-semester Break
(19th - 2nd October)
Functionality of the super operon was identified using a spectrofluorometer. There was successful chelation of magnesium into protoporphorin IX.
When incubated for an hour with Protophorphorin IX, ATP and MgCl2 there is an increase in emission intensity, peaking at 591 nm. This indicates the presence of a functional magnesium cheletase complex consisting of the 5 subunits ChlH, GUN4, CHLD, CHLi1, CHli2, Suggesting the presence and function of the appropriate genes on the super operon. There is an evident peak on the blank sample, indicating the endogenous protoporphorin.
It was found that our original mega operon was unreal. The mega operon only contained YCF54-CHLM and POR-CHLP-DvrI-ChlG genes, missing the ChlG gene and the SpeI restriction site.  Therefore, a new mega operon was made and sent for sequencing in order to confirm the presence of all the required genes.
Fig 9. Raw data of spectrofluorometer functionality test.  Increasing emission intensity indicates the presence of a functional magnesium chelatase complex, thus genes from the super operon are functional.
Fig 10. SDS PAGE of the IPTG induced magnesium chelatase showed a highly expressed band at 144 KDa indicating the presence of CHLH. Other bands were seen on the gel corresponding with  expected protein sizes; CHLI1 (40KDa), CHLD (63KDa), GUN4 (24KDa), CHLI2 (40KDa) and CTH1 (43KDa).
Fig 11. Bands cut from the SDS PAGE of magnesium chaltase plasmid were analysed using MALDI TOF Mass Spec, indicating the presence of the protein CHLH.
Fig 12. Bands cut from the SDS PAGE of magnesium chaltase plasmid were analysed using MALDI TOF Mass Spec, indicating the presence of the protein CHLI2.
Fig 13. Digest of Super Operon with Xbal and Spe1 showed two bands, indicating that the super operon contains all functional restriction sites.  The same digest was performed onto the Mega Operon, only showing one band, suggesting that the Xbal site was missing from the operon thus the Mega operon is not real.
Fig 14. The super operon was ligated with (YCF54+CHLM).  The digest of this bio-brick when run on a gel showed incorrect sizes of the bands, indicating an unsuccessful ligation between these two parts.
Week 8
(3rd - 9th October)
Sequencing results showed poor results, signalling preparation or primer issues. This lead to the attempted construction of the super operon + YCF54 + ChlM. Following this PCR screening of Mega opera + Super Operon + YCF54 +ChlM.
Mega operon showed positive results, S+M PCR products failed. Mega and Super operon were then sent for sequencing analysis. Following this genes were digested inserting them in Cam backbones for iGEM submission.
Fig 15. PCR was performed on the Super Operon containing YCF54+CHLM, showing incorrect sized bands, reinforcing that the ligation of these parts was unsuccessful.
Fig 16. An NCO1 digest of the Mega Operon and Super Operon was run on a gel.  The gel showed bands, when summed, corresponded to the expected sizes for each operon (Mega Operon 8000bp and Super Operon 10,600bp).
