Team:Macquarie Australia/chloroSummary
Home
Project
Description
Results
Interlab study
Protocols/Testing
Notebook
Safety
Parts
Summary Table
Best Composite Part
Part Collection
Proof of Concept
Model
Theoretical
Experimental
Demonstration
Human Practices
Integration
Outreach
Team
Students
Advisors
Sponsors
Achievements
Collaborations
NTU Singapore
Paris Bettencourt
Other
Aim
Improve chlH biobrick from 2015 Macquarie iGEM team by adding the lac promoter to allow expression in E. coli.
Assemble lac promoter with ycf54 and chlM in one plasmid.
Assemble genes responsible for the conversion of protoporphyrin IX (PPIX) to Mg-PPIX in one plasmid (Mg-chelatase plasmid).
Assemble genes responsible for the conversion of Mg-PPIX to chlorophyll a in one plasmid (Mg-PPIX to chlorophyll a plasmid).
Assemble Mg-chelatase plasmid with Mg-PPIX to chlorophyll a plasmid to form a final plasmid of all 12 genes required to convert PPIX to chlorophyll a (chlorophyll a plasmid).
Protein expresses the Mg-chelatase plasmid and Mg-PPIX to chlorophyll a plasmid.
Characterise and analyse the function of the Mg-chelatase plasmid and Mg-PPIX to chlorophyll a plasmid.
Experimental Design
Digest and ligate Biobricks together via 3A Assembly
IPTG induce plasmids for protein expression
Run cell lysate on SDS PAGE and use mass spectrometry to analyse gel bands
Transform and IPTG induce Mg-plasmid into ΔHemH mutants to analyse Mg-PPIX production via fluorescence emission and excitation
Summarised results
Lac promoter was successfully cloned in front of chlH biobrick
Lac promoter, ycf54 and chlM was successfully assembled in the correct order
Mg-chelatase plasmid was successfully assembled in the following order pLac-ChlI1-ChlD-GUN4-ChlI2-CTH1-pLac-ChlH. PCR and plasmid double digest confirmed the presence and size of the genes. Also confirmed by DNA sequencing
Successfully assembled Mg-PPIX to chlorophyll a plasmid. Plasmid double digest confirmed the correct size of the plasmid.
Unable to assemble the 12 gene plasmid due to time constraints
Protein induction revealed intense bands of correct sizes correlating with each gene in the Mg-chelatase plasmid. Mass spectrometry results also confirmed the presence of chlH, chlI1 and chlI2.
Fluorescence excitation and emission spectra revealed that when the Mg-chelatase plasmid is added to ΔHemH mutants, Mg-PPIX production increases proving that the plasmid is functional.
Fig 1.The above gel image shows a ladder in lane 1, the chlH gene by itself in lane 2 and the plac-ChlH sequence in lanes 3-10. Lane 3 shows a slightly smaller band then the others and was excluded from sequencing results. Lanes 4-10 were confirmed as correct through sequencing. The lac promoter has a 200 base pair sequence which was added to the start of the gene and taken from iGem part BBa_R0010.
Fig 2.Gel electrophoresis (1% agarose) provides evidence of successful assembly of Mg-chelatase plasmid (10.9 kbp), pLac-ycf54-chlM (1.5 kbp) plasmid and Mg-PPIX to Chlorophyll a plasmid (6.3 kbp). Each plasmid was assembled via 3A assembly and double digested to reveal the biobrick backbone (2000 bp) and the correct insert size.
Fig 3.PCR confirmation of the pLac-ycf54-chlM plasmid. Forward and reverse chlM primers were used to confirm the presence of the chlM gene. The presence of yCF54 gene was also confirmed using the pLac forward primer with the reverse yCF54 primer. This also confirmed the presence of the lac promoter followed by yCF54 (700 bp in total).
Fig 4.PCR screening of the Mg-chelatase operon produced products for all 6 expected genes aswell as the correct sizes.
Fig 5.Protein expression of the Mg-chelatase plasmid induced with IPTG. Lane 2 is the uninduced culture. Highly expressed band at approximately 144 kDa represents the ChlH protein. The MW of other proteins of interest include ChlI1 (40 kDa), ChlD (63 kDa), Gun4 (24 kDa), ChlI2 (40 kDa) and CTH1 (43 kDa).
Fig 6.All bands the size of the proteins of interest expressed from the induced magnesium chelatase plasmid were extracted from the SDS PAGE. MALDI TOF analysis of these proteins revealed that ChlH and ChlI2 were expressed successfully according to GPM.
Fig 7.Fluorescence excitation and emission spectra demonstrating the production of Mg-PPIX upon the addition of Mg-chelatase plasmid to ΔhemeH mutants. Mg-PPIX has an emission and excitation spectra of 418nm and 590nm respectively.
Fig 8.Emission and excitation spectra of the in vitro assay of Mg-chelatase from the Mg-P plasmid. Zero time control is the initial emission/excitation spectra when Mg-chelatase is added to 10mM PPIX, 10mM ATP and 15mM MgCl2. After incubating the assay under constant reagents as the control at room temperature, Mg-PPIX production increases. When additional 50mM MgCl2 is added to the assay in comparison to the control, Mg-PPIX is production increases further. However, upon addition of 50mM MgCl2 including 20nM of ChlID complex, the production of Mg-PPIX remains the same. Since adding additional ChlID enzyme to the last assay does not change Mg-PPIX production, the ChlID complex is not limiting.
Collaboration with NTU Singapore
NTU Singapore helped us Team Macquarie by completing a RT-PCR to test if our Mg-chelatase plasmid is being expressed under IPTG induction.
Fig 9. Gene expression rates of biosynthetic pathway genes relative to ampicillin in the Mg-chelatase plasmid: induced with varying concentrations of IPTG. This shows that the Mg-chelatase plasmid is best induced with 2mM IPTG.
