<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Lethbridge_HS_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Lethbridge HS/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><NAV class="navbar fixed-top navbar-expand-lg navbar-light bg-light;" style="background-color: #FFFFFF;font-family: 'Montserrat', sans-serif;margin-top: 9px; opacity:0.93;"><A class="navbar-brand" style="font-size:20px;" href="https://2018.igem.org/Team:Lethbridge_HS">Lethbridge HS</A></NAV></P><DIV class="collapse navbar-collapse" id="navbarNavDropdown"><UL class="navbar-nav ml-auto"><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" style=" font-size: 15px;">
      PROJECT
    </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink;"><A class="dropdown-item" href="https://2018.igem.org/Team:Lethbridge_HS/Description">Description</A><A class="dropdown-item" href="https://2018.igem.org/Team:Lethbridge_HS/Design">Design</A><A class="dropdown-item" href="https://2018.igem.org/Team:Lethbridge_HS/Model">Model</A><A class="dropdown-item" href="https://2018.igem.org/Team:Lethbridge_HS/Results">Results</A><A class="dropdown-item" href="https://2018.igem.org/Team:Lethbridge_HS/Demonstrate">Demonstrate</A><A class="dropdown-item" href="https://2018.igem.org/Team:Lethbridge_HS/Applied_Design">Applied Design</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" style=" font-size: 15px;">
    PARTS
  </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink;"><A class="dropdown-item" href="https://2018.igem.org/Team:Lethbridge_HS/Parts">Parts Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:Lethbridge_HS/Basic_Part">Basic Parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:Lethbridge_HS/Composite_Part">Composite Parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:Lethbridge_HS/Improve">Improve</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" style=" font-size: 15px;">
       LAB
     </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink;"><A class="dropdown-item" href="https://2018.igem.org/Team:Lethbridge_HS/Experiments">Experiments</A><A class="dropdown-item" href="https://2018.igem.org/Team:Lethbridge_HS/Notebook">Notebook</A><A class="dropdown-item" href="https://2018.igem.org/Team:Lethbridge_HS/InterLab">Interlab</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" style=" font-size: 15px;">
       HUMAN PRACTICES
     </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink;"><A class="dropdown-item" href="https://2018.igem.org/Team:Lethbridge_HS/Human_Practices">Human Practices</A><A class="dropdown-item" href="https://2018.igem.org/Team:Lethbridge_HS/Public_Engagement">Engagement</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" style=" font-size: 15px;">
        TEAM
      </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink;"><A class="dropdown-item" href="https://2018.igem.org/Team:Lethbridge_HS/Team">Members</A><A class="dropdown-item" href="https://2018.igem.org/Team:Lethbridge_HS/Collaborations">Collaboration</A><A class="dropdown-item" href="https://2018.igem.org/Team:Lethbridge_HS/Attributions">Attributions</A><A class="dropdown-item" href="https://2018.igem.org/Team:Lethbridge_HS/Attributions#sponsors">Sponsors</A></DIV></LI><LI class="nav-item"><A class="nav-link" style=" font-size:15px;" href="https://2018.igem.org/Team:Lethbridge_HS/Safety">SAFETY</A></LI></UL></DIV><DIV id="banner" class=" jumbotron text-center container-fluid "><DIV class="container-fluid"><DIV style="font-family: 'Montserrat' , serif;font-size: 10vw;text-align:center;margin-top:50px;"><CENTER>DESIGN</CENTER></DIV></DIV></DIV><DIV class="container-fluid;" style="margin-left:10%; margin-right:10%;"><H1 style="font-size: 3vw; font-family:Montserrat;" class="w100">OVERVIEW</H1><P style="font-size: 18px; font-family: 'Open Sans'">
Our system is composed of four main biological components: bacteriophage, host bacteria, elastin-like-polymers (ELP), and a metal binding protein. </P><P style="font-size: 18px; font-family: 'Open Sans'"> Together, the four components will work to sequester and remove metal ions from a solution. 
</P><H1 style="font-size: 3vw; font-family:Montserrat;" class="w100">BACTERIAL HOST</H1><P style="font-size: 18px; font-family: 'Open Sans'"> 
When considering a potential bacterial host that would be compatible with our system, the following criteria must be met.</P><P style="font-size: 18px; font-family: 'Open Sans'"> I) The host can be parasitized by a bacteria-phage amenable to engineering.

II) The host can survive in high metal concentrations.

III) The host can resist temperature fluctuations and aggregation. </P><P style="font-size: 18px; font-family: 'Open Sans'"> Chosen according to the criteria, we have decided to explore the host bacteria: <I>Escherichia coli</I> as our proof of concept as it is a well-studied model organism with defined methods of genetic manipulation and is easily accessible in our lab.</P><P style="font-size: 18px; font-family: 'Open Sans'">In real-life applications of our system, we will be using methanotrophic bacteria as they are one of the few natural organisms that can survive in the detrimental environment of tailings ponds. By using methanotrophic bacteria it also ensures a cost-efficient and sustainable system as the bacteria already naturally exists in the tailings pond. The additional benefit of working with <I>E. coli</I> is that results obtained in the system will inform <I>in silico</I> modelling, allowing us to design other phage systems for different organisms.</P><P style="font-size: 18px; font-family: 'Open Sans'">In summary, the choice of host defines numerous properties of the system in terms of energy input, phage production, and efficiency of removing the metal ions from the oil and mining tailings ponds. Our initial efforts with <I>E. coli</I> will inform our design of other host-phage systems, and alter the way in which our device works to capture and sequester ions in mining and oil tailings ponds.</P><FIGURE><FIGCAPTION style="font-size: 16px; font-family: 'Open Sans'"><B>Figure 1.</B> Bacterial host-phage pairs with <I>E. coli</I> being used as a proof of concept host with the T4 bacteriophage. For the M4 phage, the compatible bacterial host will be a methanotrophic bacteria to be used in tailings pond application.</FIGCAPTION></FIGURE><H1 style="font-size: 3vw; font-family:Montserrat;" class="w100"><B>BACTERIOPHAGE CASPID PROTEIN</B></H1><P style="font-size: 18px; font-family: 'Open Sans'">The T4 Bacteriophage capsid is an icosahedron with proteins Gp23, 24, and 31 contributing to its assembly. The highly antigenic and small outer capsid proteins serve as accessory proteins on the outer shell. Capsid proteins from this bacteriophage will be used as a proof of concept for further usage in the finalized construct.</P><FIGURE><FIGCAPTION style="font-size: 16px; font-family: 'Open Sans'"><B>Figure 2:</B> Proteins and their respective functions in capsid assembly as well as the HOC/SOC accessory proteins used for fusion with the construct.</FIGCAPTION></FIGURE><H1 style="font-size: 3vw; font-family:Montserrat;" class="w100"><B>PHAGE SELECTION</B></H1><P style="font-size: 18px; font-family: 'Open Sans'">In future design application, we have looked at phages that have a host range corresponding to our potential hosts: <I>E. coli</I> as the proof of concept and the methanotrophic bacteria as the real-life bacteria application.</P><P style="font-size: 18px; font-family: 'Open Sans'">Through the utilization of phage display technology, we will attempt to express ELPs fused to metal binding proteins on the surfaces of the T4 phage capsid for <I>E. coli</I> and in later experimentation, the M4 phage for the methanotrophic bacteria.</P><P style="font-size: 18px; font-family: 'Open Sans'">The T4 bacteriophage belongs to the phage family <I>Myoviridae</I> and interacts with <I>E. coli</I>[1]. The capsid of this phage has been successfully modified to display peptides and protein domains on the surface[1]. In past experiments, biologically active, full-length foreign proteins were displayed by fusion to SOC (small outer capsid protein) and HOC (highly antigenic outer capsid protein) accessory protein genes found on the T4 capsid surface. In fact, a bipartite T4 SOC–HOC protein display system allowed two different proteins to be displayed on one T4 particle simultaneously[2]. The high abundance of these proteins (960 SOC and 160 HOC molecules per phage capsid) increases the efficiency with which potential metal-binding proteins may function, once attached to this phage[2].</P><P style="font-size: 18px; font-family: 'Open Sans'">The M4 bacteriophage will be used in the real-life application of our system in the mining and oil tailing ponds as it is a natural occurring phage that can survive in the environment of the tailings pond.</P><P style="font-size: 18px; font-family: 'Open Sans'">As a proof of principle, we will be using the capsid protein taken from the T4 phage: gp23, 24 and 31. The proteins gp23 and gp24 are coat proteins making up the phage capsid, and gp31 acts as a chaperone in the gp23 folding process[3]. These proteins will produce an empty viral capsid to test the parts individually with the bacteria. This will allow for further experimentation with the full bacteriophage construct including the other parts.</P><H1 style="font-size: 3vw; font-family:Montserrat;" class="w100"><B>ELASTIN-LIKE POLYMERS</B></H1><P style="font-size: 18px; font-family: 'Open Sans'">The purpose of elastin-like polymers (ELPs) is to remove the metal ion-laden phage from the solution[4]. At a specific temperature, ELPs will aggregate and precipitate out of solution. This transition temperature (T<SUB>t</SUB>) is influenced by factors such as the heavy metal concentration, concentration of ELP, the length of the ELP chain, and the variable amino acids in each monomer. We will be altering the length and the concentration of ELP to achieve our goal T<SUB>t</SUB>. We plan for the ELP to have a T<SUB>t</SUB> between 45<SUP>ο</SUP>C to 50<SUP>o</SUP>C as this will allow the ELP to remain in solution while maintaining the optimal efficiency of our chosen bacteria[4]. The length and the T<SUB>t</SUB> have an inverse relationship; when we increase the length of the ELP, the transition temperature decreases. Using this information we intend to keep the guest residue the same while altering the length to achieve the goal T<SUB>t</SUB>.</P><P style="font-size: 18px; font-family: 'Open Sans'">To influence the characteristics of our ELP, we can also experiment with the variable amino acid in each monomer. All ELPs contain the amino acid sequence Val-Pro-Gly-X-Gly, with X representing the variable amino acid. For our project, we want to use a hydrophobic amino acid, specifically valine, glycine, or alanine. By using a hydrophobic amino acid, we can destabilize the ELP and it will more readily drop out of solution. There are several residues that are found to work well with the precipitation of ELP; valine is found to be the most effective for protein aggregation. We will add standardized linker sequences to the N- and C-termini of the ELP to allow for easy fusion to other functional proteins. In our case, we will be fusing the capsid protein of the bacteriophage and the metal-binding peptide to either end of the ELP.</P><FIGURE><FIGCAPTION style="font-size: 16px; font-family: 'Open Sans'"><B>Figure 3.</B> Schematics of different variants of elastin-like polymers[4]. The T<SUB>t</SUB> for both ELPs is 50<SUP>o</SUP>C but the ELP that uses valine as the guest residue in the 40 pentapeptides is much shorter and therefore more efficient in precipitation as it uses fewer resources.  </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION style="font-size: 16px; font-family: 'Open Sans'"><B>Figure 4:</B> Schematic of Elastin-like polymer extension design. Parts 3 and 4 as represented on the diagram are able to be repeated by multiplying by a factor of n, and increasing the length.</FIGCAPTION></FIGURE><H1 style="font-size: 3vw; font-family:Montserrat;" class="w100">METAL BINDING PROTEIN</H1><P style="font-size: 18px; font-family: 'Open Sans'">The use of the CutA protein is employed as a proof of concept because it is found in <I>E. coli</I> and binds to copper. This protein ensures that ion homeostasis is preserved within the cell, whereby the excess of copper ions is removed by this protein[5]. There will also be a hexahistidine tag for purification purposes. The CutA protein has three binding sites allowing for three copper ions to be bound per protein.</P><P style="font-size: 18px; font-family: 'Open Sans'">Our primary proof of concept proteins to be used are CutA and Cup1. Both are copper binding proteins to be used in the final construct attached to the ELP N- or C-terminal in order to effectively bind to ions. For protein purification, a hexahistidine tag will be used in the constructs for either protein.</P><P style="font-size: 18px; font-family: 'Open Sans'">In further applications of our project, there are many different proteins that can be used in our final system for the tailings ponds that are able to bind various metal ions. By using the CutA as well as Cup1 proteins as a proof of concept, this shows that our system is able to work properly with a metal binding protein.</P><H1 style="font-size: 3vw; font-family:Montserrat;" class="w100">CONSTRUCT</H1><FIGURE><FIGCAPTION style="font-size: 16px; font-family: 'Open Sans'"><B>Figure 5:</B> Simple schematic displaying the fusion construct of the Capsid proteins, Elastin-like polymer, and metal binding protein parts. </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION style="font-size: 16px; font-family: 'Open Sans'"><B>Figure 6:</B> Container model for metal ion extraction. Employing usage of the full construct, the ELP will allow for the phage and metal binding protein fusion to aggregate. The metal ions can then be removed from the solution and be re-purposed. </FIGCAPTION></FIGURE><H1 style="font-size: 3vw; font-family:Montserrat;" class="w100">REFERENCES</H1><P style="font-size: 18px; font-family: 'Open Sans'">1. Yap, M.L. and Rossmann, M.G. (2014). Structure and function of bacteriophage T4. Future Microbiol Volume 9, 1319-1327.</P><P style="font-size: 18px; font-family: 'Open Sans'">2. Ren, Z.J. and Black, L.W. (1998). Phage T4 SOC and HOC display of biologically active, full-length proteins on the viral capsid. Gene Volume 15, 439-444</P><P style="font-size: 18px; font-family: 'Open Sans'">3. Bakkes P.J, Faber B.W, Heerikhuizen H.V. and Van der Vies S.M (2005). The T4-encoded cochaperonin, gp31, has unique properties that explain its requirement for the folding of the T4 major capsid protein. Proceedings of the National Academy of Sciences Volume 23, 8144-8149</P><P style="font-size: 18px; font-family: 'Open Sans'">4. Meyer D. and Chilkoti A. (2004) Quantification of the Effects of Chain Length and Concentration on the Thermal Behaviour of Elastin-like Polypeptides. Biomacromolecules Volume 5, 846-851</P><P style="font-size: 18px; font-family: 'Open Sans'">5. Rensing C. and Grass G. (2003) <I>Escherichia. Coli</I> mechanisms of copper homeostasis in a changing environment. FEMS Microbiology Reviews, Volume 27, 197-213</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>