<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Lethbridge_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Lethbridge/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Project</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Description">Description</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Design">Design</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Model">Model</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Demonstrate">Demonstrate</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Results">Results</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Collaborations">Collaborations</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="https://2018.igem.org/Team:Lethbridge/Parts">Parts</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Parts">Overview</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Composite_Part">Composite</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Improve">Improved Parts</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Basic_Part">Basic</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Human Practices</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Human_Practices">Human Practices</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Software">Software</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Public_Engagement">Public Engagement</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Entrepreneurship">Entrepreneurship</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Applied_Design">Product Design</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Lab</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Experiments">Experiments</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Notebook">Notebook</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Safety">Safety</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/InterLab">InterLab</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="https://2018.igem.org/Team:Lethbridge/Team">Team</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Team">Team Members</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Attributions">Attributions</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Attributions#Sponsors">Sponsors</A></LI></DIV></DIV><DIV class="content"><H1>Arc PNCs as a Transfection Agent</H1><DIV class="oneText-Wrapper"><DIV class="oneText-Text"><P class="f12">mRNA delivery is a promising avenue for the application of novel gene therapies. The expression of therapeutic proteins from exogenous DNA requires transport of these constructs to the nucleus for transcription, complicating the route to expression. Additionally, current methods of DNA delivery, such as lentiviral vectors, come with the risk of random genome integration which can induce dangerous mutations in transduced cells.
Delivering therapeutic proteins encoded in mRNA is not without drawbacks. Cell cultures treated with untreated mRNA exhibit an inflammatory response when compared to cells treated with encapsulated mRNA (Uchida <I>et al.,</I> 2013). We think V.I.N.C.En.T. could help with this! 
In order to show the capacity for delivery of mRNA of one of our most interesting PNCs, we attempted to deliver an mRNA encoding a green fluorescent protein called Clover to cultures of HT22 neuronal cells using our minimal Arc Gag particles. This protein was designed by selecting the region of the <I>Arc</I> coding sequence with the highest homology to the HIV-1 Gag protein. It has been theorized that the stem loops in the RNA sequence encoding the N-lobe of the HIV-1 Gag protein play a role in regulating the uptake of RNA into the Gag complex, so we also investigated the effect of adding this sequence to the 3’ untranslated region of the Clover mRNA.  
</P><P class="f12">Experimental Protocol</P><P class="f12">Overexpressed the ArcMin protein in E. coli</P><P class="f12">Purified minimal Arc Gag, suspended in PBS (147µg/ml or 147 ng/µl)</P><P class="f12">Added Arc Gag N-lobe homology sequence to 3’ UTR of mRNA encoding Clover fluorescent protein</P><P class="f12">Produced mRNAs encoding Clover with and without the 3’UTR via in-vitro transcription (100 ng/µl in water)</P><P class="f12">Mix 50 µl ArcMin solution (7350 ng) with 10 µl RNA (1000 ng)</P><P class="f12">2 groups of encapsulation approaches:
Mixed protein and RNA, heated to 55℃ for 10 mins,
Vortexed Arc protein, added RNA and mixed by flicking tube</P><P class="f12">Added dropwise to each test chamber</P><P class="f12">For control tests, either 50ul of protein or 10µl of either mRNA added alone</P><P class="f12">Cells were live imaged on an inverted confocal fluorescence microscope using a 10X objective 3 hours after treatment. The slides were illuminated with a 488 nm laser and images were collected through a filter set used for detecting EGFP emission.</P><P class="f12">After live imaging, the cells were fixed on the chambered slides with 4% PFA for 20 minutes and coverslipped with Vectashield containing DAPI to image nuclei. The fixed cells were imaged on a confocal microscope.</P><H2>Results</H2><H2><P class="f12">Neither imaging sessions were able to detect green fluorescence beyond background fluorescence. </P></H2><H2>Improvements</H2><P class="f12">Longer incubation time: While it has been observed that treatment of cell cultures with naked mRNA encoding fluorescent proteins results in detectable fluorescent as early as 2 hours post treatment (Leonhardt et. al., 2014), it is possible that encapsulation of mRNA in a protein nanocompartment would result in delayed translation. We will attempt this test again with longer incubation times to see if this is the case. </P><P class="f12">More mRNA: the cells were only ~60% confluent, and each well was treated with only 1 µg of mRNA. As we did not see Clover expression, even in the naked mRNA It is possible that the cells did not receive enough mRNA to produce detectable amounts of Clover protein. We will increase the amount of mRNA added</P><P class="f12">Modified encapsulation: it is possible that the methods used here (vortexing ArcMin protein and heat treating ArcMin protein and Clover mRNA) may not have disrupted the ArcMin PNC, preventing the Clover mRNA from entering the PNC. It is also possible that the ArcMin proteins may not have reformed PNCs after these approaches to disruption. Going forward, we will express the ArcMin protein and translate Clover mRNA in the same cell. This way, the mRNA is available for encapsulation as the ArcMin PNCs are being formed.</P><P class="f12">mRNA capping: Mammalian mRNAs must be poly-adenylated on the 3’ end and have a 5’ cap in order for translation to be initiated. In-vitro transcription with T7 polymerase can produce poly-adenylated. </P></DIV></DIV><H1>P22 PNCs as a Biological Control Agent</H1><DIV class="oneText-Wrapper"><DIV class="oneText-Text"><P class="f12">The spread of invasive <I>Dreissena</I> mussel species, especially Zebra (<I>D. polymorpha</I>) and Quagga (<I>D. bugensis</I>) mussels, across aquatic environments in Canada represents serious ecological and economic hazard. Originating in Russia and the Ukraine, these mussel species spread rapidly through lakes and rivers to which they are introduced. They are responsible for injuries to swimmers, mass die-offs of native mollusk species, and hundreds of millions of dollars per year in damage to industries dependent on the Great Lakes (Rosaen et al. 2012). While <I>Dreissena</I> species have not yet invaded Alberta, the impact of a significant infestation would be felt across a wide range of industries. One of the most promising new methods of controlling the spread of invasive mussels is Zequanox®, a cell lysate of <I>Pseudomonas fluorescens</I>. This treatment is believed to be effective due to FitD, a protein found in <I>P. fluorescens</I> that exhibits molluscicidal activity specifically against <I>Dreissena</I> species while sparing most indigenous mollusk species.To demonstrate that PNCs enable a wide range of applications, including species-specific control of invasive organisms that pose an economic or environmental threat, we designed a PNC based system for delivering the protein toxin, FitD, to mussels. We chose to base our design on the bacteriophage P22 coat protein, as this PNC has well defined methods for encapsulating protein cargo and decorating the surface with functional peptides. See our <A href="https://2018.igem.org/Team:Lethbridge/Applied_Design">Product Design</A> page for more information</P></DIV></DIV><H1>P22 PNCs as a Gene Therapy</H1><DIV class="oneText-Wrapper"><DIV class="oneText-Text"><P class="f12">One of the primary technical barriers to the application of CRISPR/Cas9-based genome editing is the delivery of DNA encoding all components of the system. Non-integrating viral vectors, like AAVs, are limited in the size of DNA constructs packaged and often have capacities below that required to encode all necessary regulatory sequences, proteins, and RNAs for CRISPR/Cas9 to function. Larger capacity viral vectors, like lentiviruses, insert their DNA cargo randomly in the genome and may produce dangerous mutations in transduced cells.</P><P class="f12">PNCs loaded with a Cas9-guide RNA complex represent a promising alternative to viral vectors for delivery of genome editing technologies for gene therapy and research applications. This need for simple and effective delivery of CRISPR/Cas9 systems was highlighted this year by the University of Calgary iGEM team. They focused on developing a novel approach to genome editing that combined the flexibility of the CRISPR/Cas9 system with the reliability of recombinases such as Cre and Flp. Check out their project page <A href="”https://2018.igem.org/Team:Calgary”">here.</A></P><P class="f12">To show that PNCs can be used to deliver RNA-protein complexes, such as Cas9 protein loaded with guide RNA, we turned to the P22 coat protein once again. The Weidenheft lab at Montana State University has fused the Cas9 protein with the P22 scaffolding protein to encapsulate Cas9 when this fusion protein is combined with P22 coat protein. By co-expressing a desired guide RNA along with the Cas9-scaffold protein fusion, isolating the fusion protein, and combining with purified samples of the RNP with P22 coat protein, they managed to encapsulate a fully functional CRISPR/Cas9 system in the P22 PNC and went on to demonstrate that this system can cut dsDNA (Qazi et. al., 2016). </P></DIV></DIV><H1>References</H1><DIV class="oneText-Wrapper"><DIV class="oneText-Text"><UL id="left"><LI class="f10">Rosaen A. L., et al. The Costs of Aquatic Invasive Species to Great Lakes States. Anderson Economic Group LLC. Nature Conservancy. 2012</LI><LI class="f10">Uchida, S., Itaka, K., Uchida, H., Hayakawa, K., Ogata, T., Ishii, T., Fukushima, S., Osada, K., Kataoka, K. (2013) In Vivo Messenger RNA Introduction into the Central Nervous System Using Polyplex Nanomicelle. <I>PLOS ONE, 8,</I> e56220.</LI><LI class="f10">Leonhardt, C., Schwake, G., Stogbauer, T. R., Rappl, S., Juhr, J. T., Ligon, T. S., Radler, J. O. (2014) Single-cell mRNA transfection studies: Delivery, kinetics and statistics by numbers. Nanomedicine: Nanotechnology, Biology and Medicine 10(4): 679-688</LI><LI class="f10">Qazi, S., Miettinen, H., Wilkinson, R., McCoy, K., Douglas, T., and Wiedenheft, B. (2016)Programmed self-assembly of an active P22-Cas9 Nanocarrier System. Molecular Pharmaceutics. 13, 1191-1196</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>