Team:Lethbridge/Model
Project
Description
Design
Model
Demonstrate
Results
Collaborations
Parts
Overview
Composite
Improved Parts
Basic
Human Practices
Human Practices
Software
Public Engagement
Entrepreneurship
Product Design
Lab
Experiments
Notebook
Safety
InterLab
Team
Team Members
Attributions
Sponsors
While VLPs will be useful components of VINCEnT, we wanted to focus on development of a novel non-immunogenic PNC with RNA packaging capabilities. Such a tool could enable simpler transfection of mammalian cell lines for fellow iGEMers and other researchers.
To do this, we identified the Rattus norvegicus Arc protein as a candidate for modelling a minimal packaging protein. Arc is an activity-regulated cytoskeletal-associated protein that has recently been recognized as a repurposed Ty3/Gypsy retrotransposon. A bi-lobar domain within Arc has significant homology to Gag proteins, which are the major capsid proteins of many viruses including Human Immunodeficiency Virus type 1 (HIV-1), Rous-Sarcoma Virus (RSV), and Bovine Leukemia Virus (BLV). In response to synaptic activity in neurons, Arc proteins self-assemble via this Gag domain (similar to the related viral particles) to encapsulate Arc mRNA and shuttle it to neighbouring cells (Pastuzyn et al., 2018; Ashley et al., 2018).
To ensure we would not retain any native Arc functionality that might impact cellular activity, we designed a “minimal” Arc Gag protein based on homology with other known Gag domains. We used template-based structural predictions to model this minimal Arc Gag and its predicted assembly into higher-order structures.
Sequence Design
Using the EMBL-EBI Clustal Omega tool, we aligned the protein sequences of HIV-1 (GenBank BAF32552.1), BLV (GenBank BAA00543.1), and RSV (PDB 5A9E) Gag proteins to R. norvegicus Arc (NCBI Ref NP_062234.1). The most closely conserved sequences are shown below:
Based on HIV-1 homology to Arc, Zhang et al. (2015) similarly predicted a conserved Gag domain to span R. norvegicus Arc amino acids 207-278 (Gag N-lobe) and 278-370 (Gag C-lobe).
We then compared this conserved bi-lobar Gag region with the predicted RNA binding region for Gag gene products to ensure RNA packaging functionality would be maintained (Clever, Sassetti, & Parslow, 1995). A portion of the 4 stem loop RNA secondary structure in HIV-1 strongly aligns with a sequence located within the Arc Gag N-lobe:
Structural Model of Minimal Arc
Using I-TASSER (Roy, Kucukural, & Zhang, 2010; Yang et al., 2015; Zhang, 2008), we then predicted and modeled the secondary and tertiary structure of this minimal Arc Gag protein (where a higher score indicates a more confident prediction of secondary structure):
Figure 1: Minimal Arc Gag (rainbow ribbon indicating red N-terminus to blue C-terminus). Model produced with I-TASSER.
Figure 2: Minimal Arc Gag (rainbow ribbon indicating red N-terminus to blue C-terminus) overlaid with HIV-1 Gag (purple backbone; PDB ID: 3DIK). Model produced with I-TASSER using the TM-align structural alignment algorithm.
Higher-Order Assembly
This Arc Gag subunit was predicted to form a hexamer from C-terminal subunit interactions and confirmed using GalaxyHomomer (a GalaxyWEB server for prediction of homomeric protein structures; Ko et al., 2012; Shin et al., 2014). HIV-1 and RSV Gag proteins similarly interact at the C-terminus to form hexamers for assembly of higher-order spherical multi-subunit structures (de Marco et al., 2010). Thus, we were confident that the minimal Arc Gag would similarly be capable of forming a spherical or pseudo-icosahedral nanocompartment.
Figure 3: Based on structural homology with RSV (PDB ID: 3TIR), minimal Arc Gag proteins are predicted to form a hexameric subunit (each monomer indicated by a different colour). Model produced with GalaxyWEB using template-based oligomer modeling.
References
Ashley, J., Cordy, B., Lucia, D., Fradkin, L. G., Budnik, V., & Thomson, T. (2018). Retrovirus-like Gag protein Arc1 binds RNA and traffics across synaptic boutons. Cell, 172, 262-274.
Clever, J., Sassetti, C., & Parslow, T. G. (1995). RNA secondary structure and binding sites for gag gene products in the 5' packaging signal of human immunodeficiency virus type 1. J Virol, 69, 2101-2109.
de Marco, A., Davey, N. E., Ulbrich, P., Phillips, J. M., Lux, V., Riches, J. D., Fuzik, T., Ruml, T., Kräusslich, H.-G., Vogt, V. M., & Briggs, J. A. G. (2010). Conserved and variable features of Gag structure and arrangement in immature retrovirus particles. J Virol, 84, 11729-11736.
Ko, J., Park, H., Heo, L., & Seok, C. (2012). GalaxyWEB server for protein structure prediction and refinement. Nucleic Acids Res., 40, W294-W297.
Pastuzyn, E. D., Day, C. E., Kearns, R. B., Kyrke-Smith, M., Taibi, A. V., McCormick, J., Yoder, N., Belnap, D. M., Erlendsson, S., Morado, D. R., Briggs, J. A. G., Feschotte, C., & Shepherd, J. D. (2018). The neuronal gene Arc encodes a repurposed retrotransposon Gag protein that mediates intercellular RNA transfer. Cell, 172, 275-288.
Roy, A., Kucukural, A., Zhang, Y. (2010). I-TASSER: a unified platform for automated protein structure and function prediction. Nature Protocols, 5, 725-738.
Shin, W.-H., Lee, G. R., Heo, L., Lee, H., & Seok, C. (2014). Prediction of protein structure and interaction by GALAXY protein modeling programs. Bio Design, 2, 1-11.
Yang, J., Yan, R., Roy, A., Xu, D., Poisson, J., Zhang, Y. (2015). The I-TASSER Suite: Protein structure and function prediction. Nature Methods, 12, 7-8.
Zhang, W., Wu, J., Ward, M. D., Yang, S., Chuang, Y.-A., Xiao, M., Li, R., Leahy, D. J., & Worley, P. F. (2015). Structural basis of Arc binding to synaptic proteins: implications for cognitive disease. Neuron, 86, 490-500.
Zhang, Y. (2008). I-TASSER server for protein 3D structure prediction. BMC Bioinformatics, 9, 40.
