<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Lethbridge_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Lethbridge/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Project</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Description">Description</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Design">Design</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Model">Model</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Demonstrate">Demonstrate</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Results">Results</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Collaborations">Collaborations</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="https://2018.igem.org/Team:Lethbridge/Parts">Parts</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Parts">Overview</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Composite_Part">Composite</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Improve">Improved Parts</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Basic_Part">Basic</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Human Practices</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Human_Practices">Human Practices</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Software">Software</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Public_Engagement">Public Engagement</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Entrepreneurship">Entrepreneurship</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Applied_Design">Product Design</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Lab</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Experiments">Experiments</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Notebook">Notebook</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Safety">Safety</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/InterLab">InterLab</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="https://2018.igem.org/Team:Lethbridge/Team">Team</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Team">Team Members</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Attributions">Attributions</A></LI><LI><A class="color-wh" href="https://2018.igem.org/Team:Lethbridge/Attributions#Sponsors">Sponsors</A></LI></DIV></DIV><DIV class="content"><H1>Protein Nanocompartments</H1><DIV class="oneText-Wrapper"><H3>Overview</H3><DIV class="oneText-Text"><P class="f12">Our goal is to enable a wide range of bioengineering and biomedical applications by making a collection of protein nanocompartments (PNCs) that are modular, specific, standardized, and safe. With that in mind, we designed a number of PNCs for encapsulation of various cargo types, as well as application-specific surface modifiers and cargo-loading approaches.</P><P class="f12">The surface modifiers typically act as a targeting mechanism, which directs the PNC to desired cell types or other targets. However, some surface modifiers may change other PNC attributes, such as tendency for aggregation, which makes each PNC specific for each application. Ultimately, VINCEnT is intended to be a diverse tool for research, drug and vaccine development, gene therapy delivery, and many other applications.</P><P class="f12">Proper encapsulation of small molecule, nucleic acid, or protein cargos relies on stable and predictable protein-protein and protein-chemical interactions between the PNC and the intended cargo. Of particular relevance to nucleic acid encapsulation, PNCS derived from viruses have naturally evolved to have specific interactions with the viral nucleic acid genome. Though we are NOT using viruses, we are still able to exploit the natural strategies that these systems use for encapsulation.</P><P class="f12">In total, we have designed nine new parts:</P><OL class="f12" id="left"><LI>P22 coat protein (PNC)</LI><LI>P22 scaffolding protein fused to Cas9 (CRISPR-associated cargo)</LI><LI>P22 scaffolding protein fused to sfGFP (reporter cargo)</LI><LI>P22 decoration protein, DecS134C (surface modifier for P22 PNC aggregation)</LI><LI>MS2 coat protein (PNC)</LI><LI>Polyarginine cell penetrating peptide (surface modifier) </LI><LI>GFP protein with an anionically charged peptide (cargo modified for MS2)</LI><LI>Arc (full-length) protein (PNC for RNA cargo, specifically)</LI><LI>Minimal Arc Gag (PNC for RNA cargo, specifically)</LI></OL></DIV></DIV><DIV class="oneText-Wrapper"><DIV class="oneText-Text"><P class="f12">Figure 1. Schematic of our design. One single coat protein will be modified with a user-specified surface modification and a modification to the desired cargo to ensure it can be encapsulated. These will then assemble to form a full capsid or protein nanocompartment.</P></DIV></DIV><DIV class="oneText-Wrapper"><H3>Basic Construct Design</H3><DIV class="oneText-Text"><P class="f12">All designed parts contain the same T7 promoter (BBa_I712074), medium strength ribosome binding site (RBS) (BBa_J61100), the coding sequence with special modifications (which will be explained further on), and double terminator (BBa_B0014).</P></DIV></DIV><DIV class="oneText-Wrapper"><H3>P22</H3><DIV class="oneText-Text"><P class="f12">The coat protein from the <I>Salmonella typhimuriam</I> P22 bacteriophage produces one of the largest known capsids. It assembles from 420 copies of the coat protein and 100-300 copies of a scaffolding protein (SP) to form a PNC with a diameter of approximately 60nm (O’Neil <I>et al.,</I> 2011; Parent <I>et al.,</I> 2010). P22 is being preferentially used for our toolkit because of its large, hollow interior space that can accommodate larger cargos. Although this capsid protein has the ability to self-assemble, efficiency and stability of assembly is enhanced with the addition of the P22 SP (O’Neil <I>et al.,</I> 2012; 2011). For our purposes, because the SP interacts with the P22 coat protein on the C-terminus, the SP has been truncated on the N-terminus and can be fused to protein cargos or other modifiers (O’Neil <I>et al.,</I> 2011).  For VINCEnT, the P22 SP has been fused to our three cargos: Cas9, a super-folded GFP (sfGFP), and FitD to ensure encapsulation.
The first cargo, Cas9, is the DNA endonuclease specific to the CRISPR system. This RNA guided protein targets and cuts specific DNA sites for no- homologous end-joining or homology-directed repair. As a part of our collaboration with the University of Calgary iGEM team, we have encapsulated SP-Cas9 and their specific guide RNA within the P22 capsid to enhance delivery of their CRISPR system to eukaryotic cells. This construct was a generous gift from the Wiedenheft lab at the University of Montana (Qazi <I>et al.,</I> 2016) and we are still in the process of BioBricking this part for the VINCEnT toolkit (all results were obtained with the gifted plasmids). This part also includes a hexamer Histidine tag for purification. 
Our second cargo is the scaffolding protein attached to FitD, a molluscicidal toxin specific to Zebra and Quagga mussels. Zebra mussels are an invasive species in North America and have become an increasing threat to Canadian waterways. Therefore, as a method of preventing the spread of invasive mussel populations, we have designed a method to deploy SP-FitD encapsulated in P22 PNCs. For this application, we have also designed a P22 surface modifier, a mutated P22 decoration (Dec) dimer. This DecS134C dimer has been previously shown to cause aggregation of PNCs (Uchida <I>et al.,</I> 2015). The size of the PNC aggregates corresponds with the size of phytoplankton, the typical food for invasive mussel species (Molloy <I>et al.,</I> 2013; Sprung &amp; Rose, 1988; Baldwin <I>et al.,</I> 2002). For more detail on the Zebra mussel application, please see our <A href="https://2018.igem.org/Team:Lethbridge/Applied_Design">Product Design</A> and <A href="https://2018.igem.org/Team:Lethbridge/Human_Practices">Human Practices</A> pages.
Finally, for additional proof of concept experiments, we have also designed a reporter cargo where the truncated P22 SP was fused to a super-folded GFP. 
</P></DIV></DIV><DIV class="oneText-Wrapper"><H3>MS2</H3><DIV class="oneText-Text"><P class="f12">MS2 bacteriophage coat proteins self-assemble to form a 27nm PNC (Rohovie <I>et al.,</I> 2017). To improve encapsulation of protein cargo, we fused our cargo to an anionic tag to improve protein interactions on the interior surface of the PNC. We also added a modification to the surface of the MS2 capsid, a polyarginine peptide, which can target and penetrate cells non-specifically.
The natural MS2 bacteriophage has an RNA genome. Within this genome there is an RNA oligonucleotide that, when in proximity to the capsid protein, causes the self-assembly of the capsid, effectively encasing the RNA genome inside. This mechanism relies primarily on electrostatic interactions as the RNA is highly anionic (Glasgow <I>et al.</I>, 2012). Thus we have put a highly anionic peptide on the end of the GFP protein to facilitate encapsulation at a higher efficiency. This also avoids the need for attaching DNA/RNA to proteins via “click chemistry”. In this case, GFP also acts as a reporter for our validation experiments. 
We have also modified the exterior surface of the MS2 capsid with a polyarginine tag. Polyarginine peptides have been shown to penetrate and move through the membrane of eukaryotic cells and has also been previously used to deliver macromolecules through the membrane (Schmidt <I>et al.,</I> 2010). Although this peptide is non-specific, it is a great proof of concept modification for cell-penetrating small molecule delivery.
Furthermore, the polyarginine tag enhances the modularity and accessibility of this part as they are also effective for purifying proteins (Fuchs and Raines, 2005). Thus, the MS2 capsid can be purified in one step using this tag. The polyarginine tag linker to the coding sequence has also been designed with a thrombin cut site for users who would like to exclude the cell-penetrating peptide. 
 </P></DIV></DIV><DIV class="oneText-Wrapper"><H3>Arc (Full-Length and &quot;Minimal&quot;)</H3><DIV class="oneText-Text"><P class="f12">While virus-like particles are useful components of VINCEnT, we also wanted to develop a non-immunogenic PNC with RNA packaging capabilities. Such a tool could potentially enable simpler transfection of mammalian cell lines for fellow iGEMers and other researchers.Arc is an activity-regulated cytoskeletal-associated protein that has recently been recognized as a repurposed Ty3/Gypsy retrotransposon. A bi-lobar domain within Arc has significant homology to Gag proteins, which are the major capsid proteins of many viruses including Human Immunodeficiency Virus type 1 (HIV-1) and Rous-Sarcoma Virus (RSV). In response to synaptic activity in neurons, Arc proteins self-assemble via this Gag domain (similar to the related viral particles) to encapsulate Arc mRNA and shuttle it to neighbouring cells (Pastuzyn <I>et al.,</I> 2018; Ashley <I>et al.,</I> 2018).For RNA encapsulation, we take advantage of the self-mRNA encapsulation strategy employed by Arc proteins <I>in vivo</I>. Based on sequence homology with HIV-1 Gag, the predicted mRNA encapsulation sequence is located within the N-lobe of the Arc protein and so fusion of cargo RNA to the Arc mRNA (or this short predicted consensus sequence) should enable encapsulation (Clever <I>et al.,</I> 1995). However, Arc PNCs also readily encapsulate nearby non-specific RNA molecules lacking this consensus sequence in vitro (Pastuzyn <I>et al.,</I> 2018). To test RNA encapsulation efficiency, Clover and mRuby RNA with or without the predicted Arc encapsulation sequence was incubated with Arc proteins in vitro before application to various cell cultures.To ensure we would not retain any native Arc functionality that might impact cellular activity in culture, we also designed a “minimal” Arc Gag protein based on homology with other known Gag domains, including HIV-1 and RSV. We used template-based structural predictions to <A href="https://2018.igem.org/Team:Lethbridge/Model">model</A> this minimal Arc Gag and its predicted assembly into higher-order structures. We expected this minimal Arc PNC to perform similarly to the full-length Arc PNC.</P></DIV></DIV><H1>Purification Strategies</H1><DIV class="oneText-Wrapper"><DIV class="oneText-Text"><P class="f12">We have employed several methods of purification. Our purification methods for non-tagged PNCs are based on those found in the literature (Qazi <I>et al.,</I> 2016;  Pastuzyn <I>et al.,</I> 2018). Other parts use common methods such as arginine and nickel affinity chromatography. 
The Arc proteins were initially purified using a sucrose gradient followed by a cesium chloride (CsCl) cushion. Both consist of two different percentages of either sucrose or CsCl, which isolates the protein nanocompartments in a specific region of the “cushion”. The sucrose cushion enables isolation of the “crude fraction” which contains the PNCs at a relatively impure stage. The second cesium chloride cushion further purifies the PNCs, which can then be used for experiments. We have also employed size exclusion chromatography after the use of the sucrose cushion instead of the cesium chloride gradient. 
The crude P22 PNCs were isolated and purified in a similar way, except that the sucrose concentration was uniform instead of two separate concentrations. We have no not further purified the protein further but the next step is size exclusion chromatography.
The MS2 PNC includes a removable polyarginine tag on the N-Terminus. Although this was originally designed to enhance cell penetration, it also enables purification (Fuchs and Raines, 2005). Thus, the polyarginine system is also a one-step purification method. 
Other parts (including anionic GFP, SP-Cas9, and DecS134C) have hexamer HIstidine tags on the N-terminus, allowing for nickel affinity chromatography followed by size exclusion chromatography. 
</P></DIV></DIV><H1>Software Tool</H1><DIV class="oneText-Wrapper"><DIV class="oneText-Text"><P class="f12">
      In addition to our molecular wetlab work, we have converted our initial literature search into standardized protein nanocompartment datasheets to help other users choose a PNC that suits their needs. To help make this recommendation, we have designed a software tool to process user input and direct towards a datasheet PDF. The tool requires the sequence of your molecular cargo, the cell type you wish to target, and the approxiamte internal radius of the PNC. Moving forward, we plan to integrate IGSC compliant screening to ensure that the PNC recommendation is not used to distribute sequences of select agents, toxins, or other sequences of concern.
    </P></DIV></DIV><H1>Future Directions</H1><DIV class="oneText-Wrapper"><H3>Full Parts Characterization </H3><DIV class="oneText-Text"><P class="f12">This iGEM season, we were only able to clone one part (the novel minimal Arc Gag construct) and begin characterizing it. We intend to finish characterizing our toolkit, which includes collecting additional data from transmission electron microscopy imaging, analytical ultracentrifugation, mass spectroscopy, and application experiments (including eukaryotic cell transfection, CRISPR system delivery, and protein delivery). </P></DIV></DIV><DIV class="oneText-Wrapper"><H3>Complete Submission to the Parts Registry </H3><DIV class="oneText-Text"><P class="f12">We would like for our system to be accessible to other iGEM teams and researchers who may want to use these parts for their own projects. Therefore, we hope to eventually submit all nine of our designed parts to the registry (and more!). </P></DIV></DIV><DIV class="oneText-Wrapper"><H3>Modifications to our Purification Methods</H3><DIV class="oneText-Text"><P class="f12">As most of our PNC purification methods include high speed centrifugation and specialized equipment, we intend to add His tags (or other purification tags) to all of our PNCs to make purifications easier and enable higher yield.</P></DIV></DIV><DIV class="oneText-Wrapper"><H3>Expression in Different Systems</H3><DIV class="oneText-Text"><P class="f12">This year, because we only expressed parts in <I>E. coli</I>, our system was limited as to what types of cell-penetrating peptides could be tested due to lack of specificity and because the peptides often killed the host during expression. In the future, we would like to characterize PNCs with antimicrobial surface peptides as a possible novel antibiotic using yeast models.</P></DIV></DIV><H1>References</H1><DIV class="oneText-Wrapper"><DIV class="oneText-Text"><UL id="left"><LI class="f10">Ashley, J., Cordy, B., Lucia, D., Fradkin, L. G., Budnik, V., &amp; Thomson, T. (2018). Retrovirus-like Gag protein Arc1 binds RNA and traffics across synaptic boutons. <I>Cell, 172,</I> 262-274.</LI><LI class="f10">Baldwin, B., Mayer, M., Dayton, J., Pau, N., Mendila, J., Sullivan, M., Moore, A., &amp; Mills, E. (2002) Comparative growth and feeding in zebra and quagga mussels (Dreissena Polymorpha and Dreissena bugensis): implication for North American Lakes. <I>Canadian Journal of Aquatic Science, 59,</I> 680-694.</LI><LI class="f10">Fuchs, S., and Raines, R (2005) Polyarginine as a multifunctional fusion tag. <I>Protein Science, 14,</I> 1538-1544.</LI><LI class="f10">Clever, J., Sassetti, C., &amp; Parslow, T. G. (1995) RNA secondary structure and binding sites for gag gene products in the 5' packaging signal of human immunodeficiency virus type 1. <I>J Virol, 69,</I> 2101-2109.</LI><LI class="f10">Glasgow, J., Capehart, S., Francis, M., &amp; Tullman-Ercek, D. (2012) Osmolyte-mediated encapsulation of proteins inside MS2 viral capsids. <I>ACS Nano, 6,</I> 8658-8664.</LI><LI class="f10">Mazzone, H. (1998) CRC Handbook of Viruses: Mass-Molecular Weight Value and Related Properties. Boca Raton (FL): CRC Press LLC.</LI><LI class="f10">Molloy, D., Mayer, D., Giamberini, L., &amp; Gaylo, M. (2013) Mode of action of Pseudomonas fluorescens strain CL145A, a lethal control agent of Dreissend mussels (Bivalvaia: Dreissenidae). <I>Journal of Invertebrate Pathology, 113,</I> 115-121.</LI><LI class="f10">O’Neil, A., Prevelidge, P., Basu, G., &amp; Douglas, T. (2012) Coconfinement of fluorescent proteins: spatially enforced communication of GFP and mCherry encapsulated within the P22 capsid. <I>Biomacromolecules, 13,</I> 3902-3907.</LI><LI class="f10">O’Neil, A., Reichhardt, C., Johnson, B., Prevelige, P., &amp; Douglas, T. (2011) Genetically programmed in vivo packaging of protein cargo and its controlled release from bacteriophage P22. <I>Angewandte Chemie International Edition, 50,</I> 7425-7428.</LI><LI class="f10">Parent, K. N., Khayat, R., Tu, L. H., Suhanovsky, M. M., Cortines, J. R., Teschke, C. M., Johnson, J. E., &amp; Baker, T. S. (2010) P22 coat protein structures reveal a novel mechanism for capsid maturation: stability without auxiliary proteins or chemical cross-links. <I>Structure, 18,</I> 390-401.</LI><LI class="f10">Pastuzyn, E., Da, C., Kearns, R., Kyrke-Smith, M., Taibi, A., McCormick, J., Yoder, N., Belnap, D., Erlendsson, S., Morado, D., Briggs, J., Feschotte, C., &amp; Shepherd, D. (2018) The neuronal gene Arc encodes a repurposed retrotransposon gag protein that mediates intercellular RNA transfer. <I>Cell. 172,</I> 275-288.</LI><LI class="f10">Qazi, S., Miettinen, H., Wilkinson, R., McCoy, K., Douglas, T., &amp; Wiedenheft, B. (2016) Programmed self-assembly of an active P22-Cas9 nanocarrier system. <I>Molecular Pharmaceutics, 13,</I> 1191-1196.</LI><LI class="f10">Rohovie, M.J., Nagasawa, M., Swartz, J. R. (2017). Virus-like particles: next-generation nanoparticles for targeted therapeutic delivery. <I>Bioengineering &amp; Translational Medicine, 2,</I> 43-57.</LI><LI class="f10">Schmidt, N., Mishra, A., Lai, G., &amp; Wong, G. (2010) Arginine-rich cell-penetrating peptides. <I>FEBS Letters, 584,</I> 1806-1813.</LI><LI class="f10">Sprung, M., &amp; Rose, U. (1988) Influence of food size and food quantity on the feeding of the mussel Driessena Polymorpha. <I>Oncologia, 77,</I> 526-532.</LI><LI class="f10">Uchida, M., LaFrance, B., Broomwell, C., Prevelige Jr., P., &amp; Douglas, T. (2015) Higher order assembly of virus-like particles (VLPs) mediated by multi-valent protein linkers. <I>Small, 13,</I> 1562-1570.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>