Team:TP CC SanDiego/Description
PROJECT
T P - C C
HOME PROJECT NOTEBOOK OUTREACH TEAM ATTRIBUTIONS COLLABORATIONS
Back to Top PROJECT PROTOCOLS SAFETY RESULTS
PROJECT
Overview
Fungal infections have detrimental impacts in agriculture by decreasing crop yields. Particularly, Fusarium wilt, a fungal disease caused by Fusarium oxysporum, affects a wide variety of foodstuffs. Current treatments for Fusarium wilt involve the cultivation of resistant crops; however, ever-evolving fungal pathogens can circumvent said resistances and induce blight within the resistant crop strains. As a result, an inherent component of all fungal cell walls, chitin, was targeted to combat fungal infections in plants.  A diversely found glucose-derivative, chitin provides rigidity in all fungal species and also acts as a similar constituent of arthropod and insect exoskeletons and harder external tissues in various organisms. Our proposed mechanism for the antifungal treatment was to degrade chitin through the use of the enzyme chitinase. Various chitinase isozymes act upon the varying chitin structures in different organisms, so chitinases LbCHI31 and LbCHI32 were selected as the chitinases of choice for Fusarium oxysporum.
Procedural Outline
The goal of this project was to engineer a bacterial strain capable of secreting viable chitinase to attack the cellular stability and structures of Fusarium oxysporum, thereby inducing structural collapse of the fungal cell wall. A plasmid containing genes encoding for LbCHI31 production and secretion was used to transform bacteria using standard heatshock and recovery protocols.
Plasmid Design
The pBR322 plasmid was used as a base construct for the plasmid design. When designing the plasmid, the gene sequence for chitinase LbCHI31 was incorporated with a GFP tag and a linker into the plasmid. Since the GFP gene was linked to the LbCHI31, the presence of green fluorescence under UV exposure would indicate successful secretion of the GFP-LbCHI31 complex (i.e., since GFP is secreted, the linked the LbCHI31 product is consequently secreted). A signal peptide following the GFP tag was added and linked to the chitinase in order to signal the E. coli to begin type II secretion. The type II secretion system is generally utilized to secrete degradative enzymes such as chitinase. Additionally, the pBR322 contains inherent ampicillin resistance; as such, by using ampicillin as a selective agent, transformed e. coli was easily isolated and cultured.   In order to break down the chitin in the cell wall of fungus, our team has chosen to utilize E. coli and have them secrete a chitinase that will break down the chitin. Chitinases essentially break down glycosidic bonds in chitin, suggesting that by getting E. coli to secrete chitinases, we can successfully combat fungal infections. The particular chitinase we chose was LbCHI31, an enzyme that is known to be able to degrade the chitin in Fusarium oxysporum. Through transformation, we will introduce a plasmid with the gene that encodes for LbCHI31 into E. coli.
Abstract
Fungi producing harmful mycotoxins flourish on a variety of widely consumed crops, notably bananas, tomatoes, potatoes, and grains. Such fungal infections significantly reduce sustainability and food production in developing countries, where mycotoxin exposure from lack of advanced food storage are responsible for severe economic losses and 40% of diseases. As such, our team developed regulatable plasmids encoding secretable chitinases that hydrolyse the glycosidic bonds of chitin, a key structural polysaccharide in fungal cellular walls. By fusing LbCHI31 and LbCHI32 chitinase genes with a signal sequence from the alkaline phosphatase phoA gene, we successfully generated an Escherichia coli line that secretes chitinase specific to Fusarium oxysporum, a major pathogenic fungi. LbCHI31 and LbCHI32 expression and extracellular secretion were further quantified through characterization and analysis. This project will provide easily accessible, cost-effective methods for producing effective chitinases to combat fungal infections, thereby increasing crop yield, stabilizing financial growth, and reducing famine globally.
PROTOCOLS
TRANSFORMATION
Thaw competent cells on ice ( Dh5-alpha)
Add 2.5 uL diluted plasmid to e.Coli
Incubate in ice for 15 mins.
Heat shock at 42deg C for 45sec
Ice for 2 mins to complete heat shock
Add 900uL of LB
Let sit for ~40 mins at 37deg C
Centrifuge for 2 min. to collect bacteria
Vacuum so that ~100 ul is left
Pipet to resuspend bacteria
Plate 50 - 100 uL on LB+Amp plate
Let it grow overnight.
MAKING PLATES
LB+Ampicillin Agar Plates
Place 10g of LB, 6g of Agar, and 400mL of dH2O in an autoclave beaker
Autoclave for 1 hour
Remove flask and let cool for 20 min
Add 400 uL of ampicillin; swirl gently to avoid making bubbles
Pour into plates
Let plates sit for 20 min to solidify
Turn upside down, label, and store in fridge
LB+Spectinomycin Agar Plates
Place 5g of LB, 3g of Agar, and 200mL of dH2O in an autoclave beaker
Autoclave for 1 hour
Remove flask and let cool for 20 min
Add 200 uL of 50 mg/mL spectinomycin; swirl gently to avoid making bubbles
Pour into plates
Let plates sit for 20 min to solidify
Turn upside down, label, and store in fridge
PCR/COLONY PCR
Primers prep
Dilute primers (Forward primer was originally 22 nmol, Reverse was originally 27.1 nmol) to achieve 100 µM concentration (25 nmol/250 µL).
Suspend primers in 250 µL MilliQ H2O for dilution by centrifuging primer+H2O mix for 1 min. at Lvl 4. Keep each primer in separate tubes (i.e. keep forward primers separate from reverse).
Take 1 µL of the 100 µM primer mix and suspend in 9 µL MilliQ H2O to achieve 10 µL of 10 µM primers.
DNA Template prep
Dilute original pUCIDT plasmid 0.1 µg/µL. For this batch prep, 4 µg of plasmid + 40 µL of MilliQ H2O were prepared together.
PCR Setup
Place the following into PCR tubes:
1 µL of Forward Primer (10µM)
1 µL of Reverse Primer (10µM)
1 µL of DNA Template (0.1 µg/µM)
1 µL of Phusion polymerase + 10 µL of 5x HF Buffer + 1 µL of dNTPs (25 mM)
NOTE: For this lab, Part D’s components were all in a single mix stored in the -20C fridge.
35 µL of MilliQ H2O
Cycle in PCR:
98 C: 30 sec
98 C: 10 sec
55 C: 15 sec
72 C: 2 min (1700bp region, 1min/kb)
Repeat Steps 2A-2D 29 more times
72 C: 5 min
Hold at 4 C overnight (~ 15 hr)
PCR PURIFICATION
Add 5 volumes of Buffer PB to 1 volume of PCR sample and mix.
To bind DNA, apply sample to QIAquick column and centrifuge for 60 s.
Discard flow-through. Place QIAquick column back into same tube.
To wash, add 0.75 mL Buffer PE to the QIAquick column and centrifuge for 60s.
Discard flow-through. Place QIAquick column back into same tube.
Centrifuge for an additional 1 min. Place QIAquick column in clean 1.5 mL tube.
To elute DNA, add 50 uL Buffer EB to center of QIAquick membrane and centrifuge the column for 1 min.
SPECTROPHOTOMETRY
Obtain a quartz cuvette.
Clean the cuvette by rinsing with MilliQ H2O and drying with vacuum.
Add 100 uL of water to cuvette using pipette. Wipe black walls of cuvette with kimwipe to help with spectrophotometer reading.
Enter on spectrophotometer: nucleic acid, dsDNA concentration. Blank with water sample.
Clean cuvette.
Make 1:100 dilution of DNA sample and pipette into cuvette. Wipe black walls of cuvette, and measure. Type in the dilution as 100 and record concentration(ng/uL) and absorbance.
Clean cuvette.
Repeat steps 6 and 7 for any other DNA samples you may have.
Put cuvette back into box.
NANODROP
Wipe down Nanodrop machine with a kimwipe wet with MilliQ H2O. Clean the part that measures concentration.
Blank with 2 uL MilliQ H2O. Wipe down machine with kimwipe.
Pipet 2 uL of sample onto machine. Measure. Record concentration(ng/uL) and absorbance ratio(260/280). Wipe down machine with kimwipe.
Repeat step 3 for any other DNA samples you may have.
GIBSON ASSEMBLY
2-3 Fragment Assembly (0.02-0.5 pmols):
1uL vector, .43 uL insert, 10 uL Master Mix, 8.57 uL H2O
Incubate samples in thermocycler at 50 deg. C for 60 min.
Store in -20C
MAKING SDS PAGE GELS
For 10% Separating Gel: (10.11mL) - Enough for 2 gels
ddH2O 4.1mL
30% Acrylamide 3.3mL
1.5M Tris pH 8.8 2.5mL
10% SDS 0.1mL = 100uL
10% APS 100uL
TEMED 10uL
NOTE: TEMED is toxic so only add in fume hood. Add TEMED and APS LAST! APS is necessary for the gel to solidify so only add right before adding solution to plate.
For 4% Stacking Gel: (4.088mL) - Enough for 2 gels
ddH2O 2.93mL
30% Acrylamide 0.53mL
1.0M Tris pH 6.8 0.5mL
10% SDS 0.04mL
10% APS 80uL
TEMED 8uL
NOTE: Again, TEMED is toxic so only add in fume hood. Add TEMED and APS LAST! APS is necessary for the gel to solidify so only add right before adding solution to plate.
Combine all solutions into a 50 mL tube, making sure to add TEMED and APS last
Pipette 10% Separating Gel to about 3/4 of the way up from the casting frames
Immediately add Isopropanol to get rid of bubbles
Let polymerize for about 15 min
Remove Isopropanol with Kim Wipes
Add TEMED and APS to 4% Stacking Gel
Pipette to the brim of the casting frames
Place combs in, making sure to avoid bubbles
Let sit for about 15 minutes
Storing gels for SDS:
Wrap gels, with the comb still in, in paper towels
Moisten the paper towels so as to keep the gels hydrated
Wrap gels in Saran wrap, label, store in 4 deg. C
Setting up Casting Frame and Casting Stand
Set up the tall plate and short plate in the casting frame, making sure that the tall plate is in the back
Lock plates in place and clip to casting stand
Pipette water between plates to make sure that there are no leaks
Remove water from the plates if there are no leaks; if there are leaks, reposition the casting plates and repeat step 3
OVERNIGHT CULTURE FOR SDS PAGE GEL
Make LB+Amp (or LB+Spectinomycin) solution (4 mL LB & 4 uL Amp (or Spectinomycin) for each tube)
Pick one “good” colony and place in a shaker-compatible tube; a good colony is a single isolated colony and try to avoid satellite bacteria
Repeat steps 1 & 2 for remaining samples
Add Dh5-Alpha (or BL21) from glycerol stock to LB+Amp solution for positive control
Place in 37 deg.C shaker
SDS PAGE DNA SAMPLE PREP
Pipette 2 mL of saturated LB+Amp+Sample into cuvettes
Find absorption of LB+Amp+Sample by blanking the machine with LB and then inserting your LB+Amp+Sample cuvette
Repeat steps 1-2 for the next 5 samples
Pipet 2 mL of control into cuvette and find absorption
Record absorption of samples
Pipette 250 uL of Sample 1 into an Eppendorf tube
Repeat step 6 for remaining samples and control
Centrifuge at 10,000 rpm for 1 minute
Vacuum out supernatant with pipette
Remove samples 4-6 due to the failure of pellet formation
Pipette 16 uL of buffer (to break down/loosen up the bonds in our cells) to Samples 1-3
Pipette 4 uL of SDS running dye to Samples 1-3
Place Samples 1-3 (in Eppendorf tubes) in water bath for 10-15 minutes
SDS PAGE GEL BOX SETUP
Lock gel in place on one side and dummy slide on other side of gel box
IMPORTANT: Load gel with short side first!! Loading tall side first could damage the gel and causes the gel box to not run properly
Remove comb carefully so as to not puncture/deform the wells
Overflow inner chamber with 1x SDS Buffer
Fill outer chamber with 1x SDS Buffer until the bottom of the inner box is slightly submerged
PROTEIN PURIFICATION FOR SDS PAGE GEL
Put 40uL of each sample in eppendorf tubes
Place on dry heat bath for ~5 minutes so cells release proteins
Centrifuge for 1 minute at ~10K RPM
You should find pellets in all of your samples at this point
Use a pipette set at slightly over 40uL to pipette out the supernatant (avoid touching the pellet)
Add about 8uL of 5x SDS buffer dye to each sample. (approx ⅕ of sample that was centrifuged)
SDS PAGE GEL RUN
Clean out each well of the gel with 1x SDS buffer with a pipette set to 18 uL (use well-compatible tips)
Load 7 uL of ladder into second well
Load 18 uL of samples 1-3 into wells 2-4, respectively
Create a “master mix” of buffer and SDS running dye (40 uL of buffer and 10 uL of SDS running dye)
Load 18 uL of buffer+SDS running dye into wells 5-7
Cover gel box with cover - match red to red and black to black
Hook the gel box up to the volt generator
Set the electrical current to 110 V (Milliamps should be between 20-50; Watts should be around 2 (~5 max)) for about 30 minutes or until the loaded samples surpass the separating gels
Increase the voltage to 200V for ~15 min (check after 10 min. in case samples already reached the bottom of the gel)
STAINING AND DESTAINING SDS PAGE GEL
Put gel into a gel box and cover it with Coomassie Blue
Put box on shaker in darkroom for ~20 min.
Use thumb to hold gel gently against box as you pour out Coomassie Blue into used bottle
Pour Destaining Buffer in gel box
Crumple up Kimwipe and put in gel box too (helps absorb stain)
Put box on shaker in darkroom for ~10 min.
Use thumb to hold gel gently against box as you pour out Destaining Buffer into waste storage bottle under the table
Repeat steps 4,5, and 7 two more times
*OPTIONAL: put gel box on light to see how bands are developing
Put deionized water(or MilliQ H2O) in gel box (helps gel grow again because it shrinks from de-staining)
Put box on shaker for 5-10 min.
SDS PAGE GEL ANALYSIS
Take out a sheet protector
Wet fingers and gently take out gel and put in sheet protector
Make sure there are no bubbles
Put gel in scanner
On the computer: open Adobe Photoshop CS2 -> import from Canon scanner -> Preview -> re-size frame to focus on gel ->Scan
Save picture to iGEM folder (under Users) and e-mail a copy to yourself too.
Throw away gel and dab sheet protector dry
TEST WITH CHITIN
Aliquot 5 uL into each of 10 Eppendorf tubes
Dilute tube 1 to 10:1, tube 2 to 9:1, tube 3 to 8:1, and so on and so forth
Combine 5 mL of minimal media with 5 uL of 10:1 diluted chitin
Repeat step 3 for remaining tubes
Centrifuge overnight saturated cultures for 1 min on 10,000 rpm
Vacuum out supernatant and resuspend pellet in 10 uL of minimal media
Pipette 5 uL of pellet solution into each Falcon tube containing chitin and minimal media
SAFETY
1. Our project would not cause any problem in terms of researcher safety, public safety, and environmental safety.
We are Biosafety Level 1 (BSL1) and we use E. Coli Epi300 strains and BL21 strains, commonly used industrial E. coli strains.
The final construct and the transformed bacteria have two major characteristics - resilience to tetracycline and the ability to break down mycotoxins. The resistance to tetracycline may create a potential biohazard due to its inability to be killed with this specific antibiotic, however other types of antibiotics readily eliminate this bacteria. The ability to break down mycotoxins will benefit and not harm society.
The team has received biosafety training; all of the lab team members received independent online training created by UCSD or JCVI. It covered all of the possible issues that could arise while in the lab, ranging from using the fume hood to regulatory affairs issues.
Members of the lab team received additional official biosafety and laboratory safety training from the University of California, San Diego and J. Craig Venter Institute, and is approved by the respective Environmental Health & Safety departments. Possible hazards of microorganisms being released into the wild, methods of disaster prevention, laboratory techniques and safety precautions, and first aid measurements were all covered.
2. Although it is unlikely, if anything detrimental happens, the modified E. Coli...
cannot cause any harm since the strain and final constructs we use are not pathogenic. Even if some sort of infection occurs, it would be minor and can be treated with a number of different antibiotics that we did not engineer resistance for in the cells. Other than possible infection, there is little risk to the environment since, as an invasive species, the cells have no special adaptations other than being resistant to tetracycline. To prevent a breach or to clean up a spill, simply treat the area with bleach or an alcohol solution. There are no security risks from malicious misuse; the bacteria again is innocuous and would not do much harm. To prevent such mistakes and containment breaches, we are sure to clean our lab stations as well as to be cautious with our mixtures and cultures.
3. Two BioBrick parts were submitted by our team this year (BBa_K2090000 and BBa_K2090001).
4. Some of our ideas about how to deal with safety issues that could be useful for future iGEM competitions include…
In order to prevent accidental spread of possible pathogenic or environmentally harmful strains of organisms, the organism could be engineered to become an auxotrophic organism; the bacteria would be largely unable to spread beyond the petri dish. This, however, might be disabled by evolutionary mechanisms, i.e. bacteria becoming able to resynthesize whatever amino acid they were engineered not to synthesize. Engineering an auxotrophic form of E. Coli might also be useful in reducing use of antibiotic resistance genes. The gene coding for an amino acid could be substituted for the antibiotic resistance gene, with all petri dishes containing essential amino acids, except one. This would ensure growth of just transformed bacteria. To prevent spread of the newly engineered parts, it might be ideal to integrate them into the bacterial chromosome, as to prevent spreading parts through bacterial conjugation. For conducting the tests for our experiments, e.g. gel electrophoresis, there is little to report. The only major thing that should be improved on that has some safety risk is the handling of the hot agarose solution in gel electrophoresis. Instead of the hot-mitts that gives one a somewhat crab-like grip, it might be a better idea to provide hot mitts that enable a 5 fingered grip. Most of our other lab procedures are quite safe; for instance, we do not use UV light to determine banding for gel electrophoresis. Instead we use regular lighting due to our visualizing solution.
UCLA iGEM Meetup
On September 23 we met with UCLA’s iGEM team via Skype. During this meeting, we discussed details of our project’s goals, progress, and impact. In particular, we shared what new updates we’ve made to our experiment from last year, discussing what changes worked better. This included updates to our secretion system, and updates to our method of analyzing chitinase gene expression via FLAG-tag. UCLA iGEM gave us advice on how to expand our chitinase project, especially regarding ways to create a chitin medium that would enable us to test whether the chitinase was working. We also shared ideas for human practices, discussing public outreach strategies that have worked and failed for us in the past. We also provided key suggestions to help UCLA iGEM with cell-cell adhesion as well as growth inhibition.
RESULTS
Lab Work
New design for chitinase secretion system: pBAD promoter, upstream phoA secretion signal peptide, FLAG-tag for more effective protein purification and extraction methods
Successful detection of LBCHI31 and LBCHI32 protein expression
Human Practices
Sparked synthetic biology interest in middle school kids
Discussed potential for the real-world setting to implement project in the near future
Potential Implications
Agricultural - increase crop yield, especially in crops susceptible to fungal diseases
Economic/commercial - cut revenue loss in crop sales
Health - reduce malnutrition and risk of cancer
Increased food supply
Mycotoxins accumulate in poorly stored food crops
HT2 is the 3rd leading cause of cancer worldwide
Improve quality of life in developing countries
Future Directions
Future Tests
Further secretion analysis: concentrating media & testing for presence of chitinase proteins
Viability Tests: efficacy of the produced LbCHI31 & LbCHI32 against chitin and eventually Fusarium oxysporum
Direct Cell Application
Population control for regulated secretion
Fungal-bacterial interactions
Our Approach in Practice
METHOD 1: Direct Cell Application
Introduce bacteria with chitinase secretion genes to plant environment
CONSIDERATIONS:
Control mechanism
Bacterial response
METHOD 2: Secretion Extraction
Extract chitinase secretions from sample
CONSIDERATIONS:
Chitinase extraction
Viability loss from extraction methods
