<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TP_CC_SanDiego_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TP CC SanDiego/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="whole-wrapper"><DIV id="header-text"><H6>PROJECT</H6></DIV><DIV id="nav-wrapper"><DIV id="nav"><SPAN style="color: #f15258; border: 2px solid #f15258; padding: 5px 10px 5px 10px;">T P - C C</SPAN><DIV id="nav-right"><A href="https://2016.igem.org/Team:TP_CC_SanDiego">
		        HOME</A><A href="https://2016.igem.org/Team:TP_CC_SanDiego/Description">
		        PROJECT</A><A href="https://2016.igem.org/Team:TP_CC_SanDiego/Notebook">
		        NOTEBOOK</A><A href="https://2016.igem.org/Team:TP_CC_SanDiego/Practices">
		        OUTREACH</A><A href="https://2016.igem.org/Team:TP_CC_SanDiego/Team">
		        TEAM</A><A href="https://2016.igem.org/Team:TP_CC_SanDiego/Attributions">
			    ATTRIBUTIONS</A><A href="https://2016.igem.org/Team:TP_CC_SanDiego/Collaborations">
			    COLLABORATIONS</A></DIV></DIV></DIV><DIV id="content-wrapper"><DIV id="sidebar"><DIV id="sidebar-contents"><A href="#top">Back to Top</A><A href="#project">PROJECT</A><A href="#protocols">PROTOCOLS</A><A href="#safety">SAFETY</A><A href="#results">RESULTS</A></DIV></DIV><DIV id="text-wrapper"><H5>PROJECT</H5><H2>Overview</H2><P>

Fungal infections have detrimental impacts in agriculture by decreasing crop yields. Particularly, Fusarium wilt, a fungal disease caused by Fusarium oxysporum, affects a wide variety of foodstuffs. Current treatments for Fusarium wilt involve the cultivation of resistant crops; however, ever-evolving fungal pathogens can circumvent said resistances and induce blight within the resistant crop strains. As a result, an inherent component of all fungal cell walls, chitin, was targeted to combat fungal infections in plants.

A diversely found glucose-derivative, chitin provides rigidity in all fungal species and also acts as a similar constituent of arthropod and insect exoskeletons and harder external tissues in various organisms. Our proposed mechanism for the antifungal treatment was to degrade chitin through the use of the enzyme chitinase. Various chitinase isozymes act upon the varying chitin structures in different organisms, so chitinases LbCHI31 and LbCHI32 were selected as the chitinases of choice for Fusarium oxysporum.
  </P><H2>Procedural Outline</H2><P>

The goal of this project was to engineer a bacterial strain capable of secreting viable chitinase to attack the cellular stability and structures of Fusarium oxysporum, thereby inducing structural collapse of the fungal cell wall. A plasmid containing genes encoding for LbCHI31 production and secretion was used to transform bacteria using standard heatshock and recovery protocols.

  </P><H2>Plasmid Design</H2><P>

The pBR322 plasmid was used as a base construct for the plasmid design. When designing the plasmid, the gene sequence for chitinase LbCHI31 was incorporated with a GFP tag and a linker into the plasmid. Since the GFP gene was linked to the LbCHI31, the presence of green fluorescence under UV exposure would indicate successful secretion of the GFP-LbCHI31 complex (i.e., since GFP is secreted, the linked the LbCHI31 product is consequently secreted). A signal peptide following the GFP tag was added and linked to the chitinase in order to signal the E. coli to begin type II secretion. The type II secretion system is generally utilized to secrete degradative enzymes such as chitinase. Additionally, the pBR322 contains inherent ampicillin resistance; as such, by using ampicillin as a selective agent, transformed e. coli was easily isolated and cultured.


In order to break down the chitin in the cell wall of fungus, our team has chosen to utilize E. coli and have them secrete a chitinase that will break down the chitin. Chitinases essentially break down glycosidic bonds in chitin, suggesting that by getting E. coli to secrete chitinases, we can successfully combat fungal infections. The particular chitinase we chose was LbCHI31, an enzyme that is known to be able to degrade the chitin in Fusarium oxysporum. Through transformation, we will introduce a plasmid with the gene that encodes for LbCHI31 into E. coli.

  </P><H2>Abstract</H2><P>

Fungi producing harmful mycotoxins flourish on a variety of widely consumed crops, notably bananas, tomatoes, potatoes, and grains. Such fungal infections significantly reduce sustainability and food production in developing countries, where mycotoxin exposure from lack of advanced food storage are responsible for severe economic losses and 40% of diseases. As such, our team developed regulatable plasmids encoding secretable chitinases that hydrolyse the glycosidic bonds of chitin, a key structural polysaccharide in fungal cellular walls. By fusing LbCHI31 and LbCHI32 chitinase genes with a signal sequence from the alkaline phosphatase phoA gene, we successfully generated an Escherichia coli line that secretes chitinase specific to Fusarium oxysporum, a major pathogenic fungi. LbCHI31 and LbCHI32 expression and extracellular secretion were further quantified through characterization and analysis. This project will provide easily accessible, cost-effective methods for producing effective chitinases to combat fungal infections, thereby increasing crop yield, stabilizing financial growth, and reducing famine globally. 

  </P><H5>PROTOCOLS</H5><P>TRANSFORMATION</P><OL><LI><P>Thaw competent cells on ice ( Dh5-alpha)</P></LI><LI><P>Add 2.5 uL diluted plasmid to e.Coli</P></LI><LI><P>Incubate in ice for 15 mins.</P></LI><LI><P>Heat shock at 42deg C for 45sec</P></LI><LI><P>Ice for 2 mins to complete heat shock</P></LI><LI><P>Add 900uL of LB</P></LI><LI><P>Let sit for ~40 mins at 37deg C</P></LI><LI><P>Centrifuge for 2 min. to collect bacteria</P></LI><LI><P>Vacuum so that ~100 ul is left</P></LI><LI><P>Pipet to resuspend bacteria</P></LI><LI><P>Plate 50 - 100 uL on LB+Amp plate</P></LI><LI><P>Let it grow overnight.</P></LI></OL><P>MAKING PLATES</P><P>LB+Ampicillin Agar Plates</P><OL><LI><P>Place 10g of LB, 6g of Agar, and 400mL of dH2O in an autoclave beaker</P></LI><LI><P>Autoclave for 1 hour</P></LI><LI><P>Remove flask and let cool for 20 min</P></LI><LI><P>Add 400 uL of ampicillin; swirl gently to avoid making bubbles</P></LI><LI><P>Pour into plates</P></LI><LI><P>Let plates sit for 20 min to solidify</P></LI><LI><P>Turn upside down, label, and store in fridge</P></LI></OL><P>LB+Spectinomycin Agar Plates</P><OL><LI><P>Place 5g of LB, 3g of Agar, and 200mL of dH2O in an autoclave beaker</P></LI><LI><P>Autoclave for 1 hour</P></LI><LI><P>Remove flask and let cool for 20 min</P></LI><LI><P>Add 200 uL of 50 mg/mL spectinomycin; swirl gently to avoid making bubbles</P></LI><LI><P>Pour into plates</P></LI><LI><P>Let plates sit for 20 min to solidify</P></LI><LI><P>Turn upside down, label, and store in fridge</P></LI></OL><P>PCR/COLONY PCR</P><P>Primers prep</P><OL><LI><P>Dilute primers (Forward primer was originally 22 nmol, Reverse was originally 27.1 nmol) to achieve 100 µM concentration (25 nmol/250 µL).</P></LI><LI><P>Suspend primers in 250 µL MilliQ H2O for dilution by centrifuging primer+H2O mix for 1 min. at Lvl 4. Keep each primer in separate tubes (i.e. keep forward primers separate from reverse).</P></LI><LI><P>Take 1 µL of the 100 µM primer mix and suspend in 9 µL MilliQ H2O to achieve 10 µL of 10 µM primers.</P></LI></OL><P>DNA Template prep</P><OL><LI><P>Dilute original pUCIDT plasmid 0.1 µg/µL. For this batch prep, 4 µg of plasmid + 40 µL of MilliQ H2O were prepared together.</P></LI></OL><P>PCR Setup</P><OL><LI><P>Place the following into PCR tubes:</P></LI><OL><LI><P>1 µL of Forward Primer (10µM)</P></LI><LI><P>1 µL of Reverse Primer (10µM)</P></LI><LI><P>1 µL of DNA Template (0.1 µg/µM)</P></LI><LI><P>1 µL of Phusion polymerase + 10 µL of 5x HF Buffer + 1 µL of dNTPs (25 mM)</P></LI><OL><LI><P>NOTE: For this lab, Part D’s components were all in a single mix stored in the -20C fridge.</P></LI></OL><LI><P>35 µL of MilliQ H2O</P></LI></OL><LI><P>Cycle in PCR:</P></LI><OL><LI><P>98 C: 30 sec</P></LI><LI><P>98 C: 10 sec</P></LI><LI><P>55 C: 15 sec</P></LI><LI><P>72 C: 2 min (1700bp region, 1min/kb)</P></LI><LI><P>Repeat Steps 2A-2D 29 more times</P></LI><LI><P>72 C: 5 min</P></LI><LI><P>Hold at 4 C overnight (~ 15 hr)</P></LI></OL></OL><P>PCR PURIFICATION</P><OL><LI><P>Add 5 volumes of Buffer PB to 1 volume of PCR sample and mix.</P></LI><LI><P>To bind DNA, apply sample to QIAquick column and centrifuge for 60 s.</P></LI><LI><P>Discard flow-through. Place QIAquick column back into same tube.</P></LI><LI><P>To wash, add 0.75 mL Buffer PE to the QIAquick column and centrifuge for 60s.</P></LI><LI><P>Discard flow-through. Place QIAquick column back into same tube.</P></LI><LI><P>Centrifuge for an additional 1 min. Place QIAquick column in clean 1.5 mL tube.</P></LI><LI><P>To elute DNA, add 50 uL Buffer EB to center of QIAquick membrane and centrifuge the column for 1 min.</P></LI></OL><P>SPECTROPHOTOMETRY</P><OL><LI><P>Obtain a quartz cuvette. </P></LI><LI><P>Clean the cuvette by rinsing with MilliQ H2O and drying with vacuum.</P></LI><LI><P>Add 100 uL of water to cuvette using pipette. Wipe black walls of cuvette with kimwipe to help with spectrophotometer reading.</P></LI><LI><P>Enter on spectrophotometer: nucleic acid, dsDNA concentration. Blank with water sample.</P></LI><LI><P>Clean cuvette.</P></LI><LI><P>Make 1:100 dilution of DNA sample and pipette into cuvette. Wipe black walls of cuvette, and measure. Type in the dilution as 100 and record concentration(ng/uL) and absorbance.</P></LI><LI><P>Clean cuvette.</P></LI><LI><P>Repeat steps 6 and 7 for any other DNA samples you may have.</P></LI><LI><P>Put cuvette back into box.</P></LI></OL><P>NANODROP</P><OL><LI><P>Wipe down Nanodrop machine with a kimwipe wet with MilliQ H2O. Clean the part that measures concentration.</P></LI><LI><P>Blank with 2 uL MilliQ H2O. Wipe down machine with kimwipe.</P></LI><LI><P>Pipet 2 uL of sample onto machine. Measure. Record concentration(ng/uL) and absorbance ratio(260/280). Wipe down machine with kimwipe.</P></LI><LI><P>Repeat step 3 for any other DNA samples you may have.</P></LI></OL><P>GIBSON ASSEMBLY</P><OL><LI><P>2-3 Fragment Assembly (0.02-0.5 pmols):</P></LI></OL><P>1uL vector, .43 uL insert, 10 uL Master Mix, 8.57 uL H2O</P><OL start="2"><LI><P>Incubate samples in thermocycler at 50 deg. C for 60 min.</P></LI><LI><P>Store in -20C</P></LI></OL><P>MAKING SDS PAGE GELS</P><P>For 10% Separating Gel: (10.11mL) - Enough for 2 gels</P><P>ddH2O 4.1mL</P><P>30% Acrylamide 3.3mL</P><P>1.5M Tris pH 8.8 2.5mL</P><P>10% SDS 0.1mL = 100uL</P><P>10% APS 100uL</P><P>TEMED 10uL</P><P>NOTE: TEMED is toxic so only add in fume hood. Add TEMED and APS LAST! APS is necessary for the gel to solidify so only add right before adding solution to plate.</P><P>For 4% Stacking Gel: (4.088mL) - Enough for 2 gels</P><P>ddH2O 2.93mL</P><P>30% Acrylamide 0.53mL</P><P>1.0M Tris pH 6.8 0.5mL</P><P>10% SDS 0.04mL</P><P>10% APS 80uL</P><P>TEMED 8uL</P><P>NOTE: Again, TEMED is toxic so only add in fume hood. Add TEMED and APS LAST! APS is necessary for the gel to solidify so only add right before adding solution to plate.</P><OL><LI><P>Combine all solutions into a 50 mL tube, making sure to add TEMED and APS last</P></LI><LI><P>Pipette 10% Separating Gel to about 3/4 of the way up from the casting frames</P></LI><LI><P>Immediately add Isopropanol to get rid of bubbles</P></LI><LI><P>Let polymerize for about 15 min</P></LI><LI><P>Remove Isopropanol with Kim Wipes</P></LI><LI><P>Add TEMED and APS to 4% Stacking Gel</P></LI><LI><P>Pipette to the brim of the casting frames</P></LI><LI><P>Place combs in, making sure to avoid bubbles</P></LI><LI><P>Let sit for about 15 minutes</P></LI></OL><P>Storing gels for SDS:</P><OL><LI><P>Wrap gels, with the comb still in, in paper towels</P></LI><LI><P>Moisten the paper towels so as to keep the gels hydrated</P></LI><LI><P>Wrap gels in Saran wrap, label, store in 4 deg. C</P></LI></OL><P>Setting up Casting Frame and Casting Stand</P><OL><LI><P>Set up the tall plate and short plate in the casting frame, making sure that the tall plate is in the back</P></LI><LI><P>Lock plates in place and clip to casting stand</P></LI><LI><P>Pipette water between plates to make sure that there are no leaks</P></LI><LI><P>Remove water from the plates if there are no leaks; if there are leaks, reposition the casting plates and repeat step 3</P></LI></OL><P>OVERNIGHT CULTURE FOR SDS PAGE GEL</P><OL><LI><P>Make LB+Amp (or LB+Spectinomycin) solution (4 mL LB &amp; 4 uL Amp (or Spectinomycin) for each tube)</P></LI><LI><P>Pick one “good” colony and place in a shaker-compatible tube; a good colony is a single isolated colony and try to avoid satellite bacteria</P></LI><LI><P>Repeat steps 1 &amp; 2 for remaining samples</P></LI><LI><P>Add Dh5-Alpha (or BL21) from glycerol stock to LB+Amp solution for positive control</P></LI><LI><P>Place in 37 deg.C shaker</P></LI></OL><P>SDS PAGE DNA SAMPLE PREP</P><OL><LI><P>Pipette 2 mL of saturated LB+Amp+Sample into cuvettes</P></LI><LI><P>Find absorption of LB+Amp+Sample by blanking the machine with LB and then inserting your LB+Amp+Sample cuvette</P></LI><LI><P>Repeat steps 1-2 for the next 5 samples</P></LI><LI><P>Pipet 2 mL of control into cuvette and find absorption</P></LI><LI><P>Record absorption of samples</P></LI><LI><P>Pipette 250 uL of Sample 1 into an Eppendorf tube</P></LI><LI><P>Repeat step 6 for remaining samples and control</P></LI><LI><P>Centrifuge at 10,000 rpm for 1 minute</P></LI><LI><P>Vacuum out supernatant with pipette</P></LI><LI><P>Remove samples 4-6 due to the failure of pellet formation</P></LI><LI><P>Pipette 16 uL of buffer (to break down/loosen up the bonds in our cells) to Samples 1-3</P></LI><LI><P>Pipette 4 uL of SDS running dye to Samples 1-3</P></LI><LI><P>Place Samples 1-3 (in Eppendorf tubes) in water bath for 10-15 minutes</P></LI></OL><P>SDS PAGE GEL BOX SETUP</P><OL><LI><P>Lock gel in place on one side and dummy slide on other side of gel box</P></LI><OL><LI><P>IMPORTANT: Load gel with short side first!! Loading tall side first could damage the gel and causes the gel box to not run properly</P></LI></OL><LI><P>Remove comb carefully so as to not puncture/deform the wells</P></LI><LI><P>Overflow inner chamber with 1x SDS Buffer</P></LI><LI><P>Fill outer chamber with 1x SDS Buffer until the bottom of the inner box is slightly submerged</P></LI></OL><P>PROTEIN PURIFICATION FOR SDS PAGE GEL</P><OL><LI><P>Put 40uL of each sample in eppendorf tubes</P></LI><LI><P>Place on dry heat bath for ~5 minutes so cells release proteins </P></LI><LI><P>Centrifuge for 1 minute at ~10K RPM</P></LI><LI><P>You should find pellets in all of your samples at this point</P></LI><LI><P>Use a pipette set at slightly over 40uL to pipette out the supernatant (avoid touching the pellet)</P></LI><LI><P>Add about 8uL of 5x SDS buffer dye to each sample. (approx ⅕ of sample that was centrifuged)</P></LI></OL><P>SDS PAGE GEL RUN</P><OL><LI><P>Clean out each well of the gel with 1x SDS buffer with a pipette set to 18 uL (use well-compatible tips)</P></LI><LI><P>Load 7 uL of ladder into second well</P></LI><LI><P>Load 18 uL of samples 1-3 into wells 2-4, respectively</P></LI><LI><P>Create a “master mix” of buffer and SDS running dye (40 uL of buffer and 10 uL of SDS running dye)</P></LI><LI><P>Load 18 uL of buffer+SDS running dye into wells 5-7</P></LI><LI><P>Cover gel box with cover - match red to red and black to black</P></LI><LI><P>Hook the gel box up to the volt generator</P></LI><LI><P>Set the electrical current to 110 V (Milliamps should be between 20-50; Watts should be around 2 (~5 max)) for about 30 minutes or until the loaded samples surpass the separating gels</P></LI><LI><P>Increase the voltage to 200V for ~15 min (check after 10 min. in case samples already reached the bottom of the gel) </P></LI></OL><P>STAINING AND DESTAINING SDS PAGE GEL</P><OL><LI><P>Put gel into a gel box and cover it with Coomassie Blue</P></LI><LI><P>Put box on shaker in darkroom for ~20 min.</P></LI><LI><P>Use thumb to hold gel gently against box as you pour out Coomassie Blue into used bottle</P></LI><LI><P>Pour Destaining Buffer in gel box</P></LI><LI><P>Crumple up Kimwipe and put in gel box too (helps absorb stain)</P></LI><LI><P>Put box on shaker in darkroom for ~10 min.</P></LI><LI><P>Use thumb to hold gel gently against box as you pour out Destaining Buffer into waste storage bottle under the table</P></LI><LI><P>Repeat steps 4,5, and 7 two more times </P></LI></OL><P>*OPTIONAL: put gel box on light to see how bands are developing</P><OL start="9"><LI><P>Put deionized water(or MilliQ H2O) in gel box (helps gel grow again because it shrinks from de-staining)</P></LI><LI><P>Put box on shaker for 5-10 min.</P></LI></OL><P>SDS PAGE GEL ANALYSIS</P><OL><LI><P>Take out a sheet protector </P></LI><LI><P>Wet fingers and gently take out gel and put in sheet protector</P></LI><LI><P>Make sure there are no bubbles</P></LI><LI><P>Put gel in scanner </P></LI><LI><P>On the computer: open Adobe Photoshop CS2 -&gt; import from Canon scanner -&gt; Preview -&gt; re-size frame to focus on gel -&gt;Scan </P></LI><LI><P>Save picture to iGEM folder (under Users) and e-mail a copy to yourself too.</P></LI><LI><P>Throw away gel and dab sheet protector dry</P></LI></OL><P>TEST WITH CHITIN</P><OL><LI><P>Aliquot 5 uL into each of 10 Eppendorf tubes</P></LI><LI><P>Dilute tube 1 to 10:1, tube 2 to 9:1, tube 3 to 8:1, and so on and so forth</P></LI><LI><P>Combine 5 mL of minimal media with 5 uL of 10:1 diluted chitin</P></LI><LI><P>Repeat step 3 for remaining tubes</P></LI><LI><P>Centrifuge overnight saturated cultures for 1 min on 10,000 rpm</P></LI><LI><P>Vacuum out supernatant and resuspend pellet in 10 uL of minimal media</P></LI><LI><P>Pipette 5 uL of pellet solution into each Falcon tube containing chitin and minimal media</P></LI></OL><H5>SAFETY</H5><H4 dir="ltr">1. Our project would not cause any problem in terms of researcher safety, public safety, and environmental safety.
</H4><P>We are Biosafety Level 1 (BSL1) and we use E. Coli Epi300 strains and BL21 strains, commonly used industrial E. coli strains.</P><P>The final construct and the transformed bacteria have two major characteristics - resilience to tetracycline and the ability to break down mycotoxins. The resistance to tetracycline may create a potential biohazard due to its inability to be killed with this specific antibiotic, however other types of antibiotics readily eliminate this bacteria. The ability to break down mycotoxins will benefit and not harm society.</P><P>The team has received biosafety training; all of the lab team members received independent online training created by UCSD or JCVI. It covered all of the possible issues that could arise while in the lab, ranging from using the fume hood to regulatory affairs issues.</P><P>Members of the lab team received additional official biosafety and laboratory safety training from the University of California, San Diego and J. Craig Venter Institute, and is approved by the respective Environmental Health &amp; Safety departments. Possible hazards of microorganisms being released into the wild, methods of disaster prevention, laboratory techniques and safety precautions, and first aid measurements were all covered.</P><H4 dir="ltr">2. Although it is unlikely, if anything detrimental happens, the modified E. Coli...</H4><P>cannot cause any harm since the strain and final constructs we use are not pathogenic. Even if some sort of infection occurs, it would be minor and can be treated with a number of different antibiotics that we did not engineer resistance for in the cells. Other than possible infection, there is little risk to the environment since, as an invasive species, the cells have no special adaptations other than being resistant to tetracycline. To prevent a breach or to clean up a spill, simply treat the area with bleach or an alcohol solution. There are no security risks from malicious misuse; the bacteria again is innocuous and would not do much harm. To prevent such mistakes and containment breaches, we are sure to clean our lab stations as well as to be cautious with our mixtures and cultures.</P><H4 dir="ltr">3. Two BioBrick parts were submitted by our team this year (BBa_K2090000 and BBa_K2090001).</H4><H4 dir="ltr">4. Some of our ideas about how to deal with safety issues that could be useful for future iGEM competitions include…</H4><P>In order to prevent accidental spread of possible pathogenic or environmentally harmful strains of organisms, the organism could be engineered to become an auxotrophic organism; the bacteria would be largely unable to spread beyond the petri dish. This, however, might be disabled by evolutionary mechanisms, i.e. bacteria becoming able to resynthesize whatever amino acid they were engineered not to synthesize. Engineering an auxotrophic form of E. Coli might also be useful in reducing use of antibiotic resistance genes. The gene coding for an amino acid could be substituted for the antibiotic resistance gene, with all petri dishes containing essential amino acids, except one. This would ensure growth of just transformed bacteria. To prevent spread of the newly engineered parts, it might be ideal to integrate them into the bacterial chromosome, as to prevent spreading parts through bacterial conjugation.
For conducting the tests for our experiments, e.g. gel electrophoresis, there is little to report. The only major thing that should be improved on that has some safety risk is the handling of the hot agarose solution in gel electrophoresis. Instead of the hot-mitts that gives one a somewhat crab-like grip, it might be a better idea to provide hot mitts that enable a 5 fingered grip. Most of our other lab procedures are quite safe; for instance, we do not use UV light to determine banding for gel electrophoresis. Instead we use regular lighting due to our visualizing solution.</P><H4>UCLA iGEM Meetup</H4><P>On September 23 we met with UCLA’s iGEM team via Skype. During this meeting, we discussed details of our project’s goals, progress, and impact. In particular, we shared what new updates we’ve made to our experiment from last year, discussing what changes worked better. This included updates to our secretion system, and updates to our method of analyzing chitinase gene expression via FLAG-tag. UCLA iGEM gave us advice on how to expand our chitinase project, especially regarding ways to create a chitin medium that would enable us to test whether the chitinase was working. We also shared ideas for human practices, discussing public outreach strategies that have worked and failed for us in the past. We also provided key suggestions to help UCLA iGEM with cell-cell adhesion as well as growth inhibition.</P><H5>RESULTS</H5><H4 dir="ltr"><P>Lab Work</P><UL><LI><P>New design for chitinase secretion system: pBAD promoter, upstream phoA secretion signal peptide, FLAG-tag for more effective protein purification and extraction methods</P></LI><LI><P>Successful detection of LBCHI31 and LBCHI32 protein expression</P></LI></UL></H4><H4 dir="ltr"><P>Human Practices</P><UL><LI><P>Sparked synthetic biology interest in middle school kids</P></LI><LI><P>Discussed potential for the real-world setting to implement project in the near future</P></LI></UL></H4><H4 dir="ltr"><P>Potential Implications</P><UL><LI><P>Agricultural - increase crop yield, especially in crops susceptible to fungal diseases</P></LI><LI><P>Economic/commercial - cut revenue loss in crop sales</P></LI><LI><P>Health - reduce malnutrition and risk of cancer</P></LI><LI>Increased food supply</LI><LI>Mycotoxins accumulate in poorly stored food crops</LI><LI>HT2 is the 3rd leading cause of cancer worldwide</LI></UL><LI><P>Improve quality of life in developing countries</P></LI></H4><H4 dir="ltr"><P>Future Directions</P><UL><LI><P>Future Tests</P></LI><LI>Further secretion analysis: concentrating media &amp; testing for presence of chitinase proteins
</LI><LI>Viability Tests: efficacy of the produced LbCHI31 &amp; LbCHI32 against chitin and eventually Fusarium oxysporum
</LI></UL><LI><P>Direct Cell Application</P></LI><LI>Population control for regulated secretion</LI><LI>Fungal-bacterial interactions</LI></H4><H4 dir="ltr"><P>Our Approach in Practice</P><UL><LI><P>METHOD 1: Direct Cell Application</P></LI><UL id="innerinner"><LI>Introduce bacteria with chitinase secretion genes to plant environment</LI><LI>CONSIDERATIONS:</LI><UL id="innerinner"><LI>Control mechanism</LI><LI>Bacterial response</LI></UL></UL><LI><P>METHOD 2: Secretion Extraction</P></LI><UL id="innerinner"><LI>Extract chitinase secretions from sample</LI><LI>CONSIDERATIONS:</LI><UL id="innerinner"><LI>Chitinase extraction</LI><LI>Viability loss from extraction methods</LI></UL></UL></UL></H4></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>