<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Linkoping_Sweden_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Linkoping Sweden/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="topnav" id="myTopnav"><DIV class="mobile-menu"><CENTER><DIV class="topnav-bottom-content"><DIV id="mobile-only-text"><P>LiU iGEM 2017</P></DIV><DIV class="dropdown"><A href="https://2017.igem.org/Team:Linkoping_Sweden">Home</A></DIV><DIV class="dropdown"><SPAN>Team</SPAN><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Linkoping_Sweden/Team">Team</A><A href="https://2017.igem.org/Team:Linkoping_Sweden/Collaborations">Collaborations</A></DIV></DIV><DIV class="dropdown"><SPAN> Background </SPAN><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Linkoping_Sweden/Description"> Project Description</A><A href="https://2017.igem.org/Team:Linkoping_Sweden/Safety">Safety</A><A href="https://2017.igem.org/Team:Linkoping_Sweden/AlzheimersDisease"> Alzheimer's Disease </A></DIV></DIV><DIV class="dropdown"><SPAN> Project </SPAN><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Linkoping_Sweden/Design">Design</A><A href="https://2017.igem.org/Team:Linkoping_Sweden/Experiments">Experiments</A><A href="https://2017.igem.org/Team:Linkoping_Sweden/Notebook">Notebook</A><A href="https://2017.igem.org/Team:Linkoping_Sweden/Contribution">Contribution</A><A href="https://2017.igem.org/Team:Linkoping_Sweden/Model">Model</A><A href="https://2017.igem.org/Team:Linkoping_Sweden/Results">Results</A><A href="https://2017.igem.org/Team:Linkoping_Sweden/Attributions">Attributions</A></DIV></DIV><DIV class="dropdown"><SPAN>Parts</SPAN><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Linkoping_Sweden/Parts">Parts</A><A href="https://2017.igem.org/Team:Linkoping_Sweden/UnfinishedParts">Planned Parts</A></DIV></DIV><DIV class="dropdown"><SPAN>Human Practices</SPAN><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Linkoping_Sweden/Human_Practices">Overview</A><A href="https://2017.igem.org/Team:Linkoping_Sweden/HP/Silver">Silver HP</A><A href="https://2017.igem.org/Team:Linkoping_Sweden/HP/Gold_Integrated">Integrated and Gold</A><A href="https://2017.igem.org/Team:Linkoping_Sweden/Engagement">Public Engagement</A></DIV></DIV><DIV class="dropdown"><A href="https://2017.igem.org/Team:Linkoping_Sweden/Contact">Contact us</A></DIV><A href="javascript:void(0);" style="font-size:2em;" class="icon" onclick="menuButton()">☰</A></DIV></CENTER></DIV><DIV id="flex-wrapper"><DIV class="flex-container"><ARTICLE class="flex-text-100"><H1> Design </H1><H3> Construction of Biobricks </H3><P>
				When we constructed our biobricks we used sequences from iGEM´s registry,  our supervisors and Benchling’s search function, that accesses several databases. When all the sequences had been gathered, we put them together with Benchling. After the final touch was made, the biobricks were sent to IDT who transformed our constructed sequences into DNA.
			</P></ARTICLE><DIV class="flex-sidebyside"><ARTICLE class="flex-text-70"><P>
				Our plasmids were designed to include different options for overexpression of chaperones. This was done by connecting each chaperone to an individual promoter which enables overexpression both in individual chaperones, but also overexpression of different combinations of chaperones. Our method of assembly was NEB Hi-Fi DNA assembly. To enable this we included the starting and ending sequence of psb1C3 in our two flanking chaperones and created synthetic overlaps between theese and the middle chaperone. All chaperone sequences was also equipped with a terminating sequence to prevent unwanted downstream protein expression. 
			We also designed a second plasmid containing the different fusion proteins. pSB_1A3 was chosen as the vector to enable co-transformation of the two plasmids by selecting with both ampicillin and chloramphenicol. The fusion protein consisted of two parts, a aggregation prone part (either amyloid beta 42 or Tau0N4R) and a fluorescent part  (mNeongreen or eGFP). Amyloid-beta 42 was chosen since aggregations of it is thought to be the cause of Alzheimer's disease (1). Although the isoform Tau0N4R is not fully confirmed to be a specific cause of Alzheimer's disease Tau aggregation is still seen in the disease. The R segment is thought to be the part that aggregates. Therefore Tau0N4R was chosen over its counterpart Tau0N3R (2). When choosing the fluorescent part two factors were taken into account. On one hand we wanted to work with a well characterised protein and on the other hand we wanted a fluorescent protein that did not prevent the expression of our aggregation prone proteins, therefore the two fluorescent proteins we chose was eGFP and mNeongreen. eGFP is a well used and characterized protein and mNeongreen is about 1,5 - 3 times as bright as eGFP and has proven to be a good fluorescent tag in fusion proteins (3). All the aggregation prone fusion proteins carried a histidine tag to enable IMAC purification (4). A TEV site was placed after the histidine tag to enable a histidine tag free end product after purification. We also designed a GS-linker between the two components of the fusion proteins to make them more motile (5). One difference between the fusion proteins containing Tau0N4R and Amyloid-beta-42 is that Amyloid-beta-42 is placed C-terminally and Tau0N4R is placed N-terminally.
			</P></ARTICLE></DIV><ARTICLE class="flex-text-100"><P>
				Press<A href="https://2017.igem.org/Team:Linkoping_Sweden/Parts"> here </A>for a closer look at our parts.
			</P><P>
				To verify a successfully assembled biobrick we used several different methods both for DNA analysis and protein analysis.  
			</P></ARTICLE><ARTICLE class="flex-text-100"><H3> Gel </H3><P>
				Gel electrophoresis with a 1.3% agarose gel was used to control the size of the attempted assemblies of the fusion proteins and chaperone systems. This gave a hint on whether the assembly products were correctly assembled and if all parts in the assembly reaction had ligated or not. Because we already knew the size of our designed constructs we could easily see when the assembled products had the right number of nucleotides. If not all the parts in the assembly had ligated this would be shown as lighter bands on the gel. 
			</P></ARTICLE><ARTICLE class="flex-text-100"><H3> Sequencing </H3><P>
				To verify our assemblies, one of our PIs helped us send our DNA to GATC Biotech in Germany to get a full sequencing. 
			</P></ARTICLE><ARTICLE class="flex-text-100"><H3> Fluorescence </H3><P>
				There are several different kinds of molecules called fluorophores that are prone to fluoresce, which is the outcome of a three-step process. The first step involves that the fluorophore absorb energy in the form of photons from a lightsource. Under the second step the fluorophore changes its’ conformation and some energy goes to molecules in its surroundings. During this time the fluorophores will lose some of the excitation energy. In the last step a photon is released from the fluorophore and it goes back to its ground state. Due to the energy lost in the second step the photon will have a lower energy and a bigger wavelength. Therefore a different color than the absorption light will be detected (6).
				In our project we decided to use fluorescence as one method of detection of our fusion proteins as it could have given us the ability to quickly and easily determine the protein concentration. In future science this could be used to analyse Amyloid Beta proteins where some have the fluorescent tag and some does not. In this way scientists will get new insights of how Amyloid Beta aggregates.
			</P></ARTICLE><ARTICLE class="flex-text-100"><H3> Protein analysis </H3><P>
				We analyzed the fusion protein mNeongreen-Amyloid-Beta with a plate reader. mNeonGreen have its excitation peak at 506 nm and its emission peak at 517 nm (3). The plate reader enables quantification of the fusion protein if one has a standard with measurements of solutions with known concentrations. We did not lyse the cells before the plate reading as we wanted to simultaneously measure the optical density (OD) of each well. The OD measurement would’ve made the amount of protein relative to the amount of bacteria in each well.
				To further analyze our protein we have used a variant of western blot called a dot-blot. Western blot is a method where the proteins are denatured with SDS (Sodium dodecyl sulfate ) and run through a polyacrylamide gel electrophoresis. The proteins are then transferred to a membrane and  primary and secondary antibodies are applied. We used a primary antibody called 6E10 from mouse that is specific to amyloid beta. We used a secondary antibody from rabbit that binds to the primary mouse antibody. The secondary antibody is conjugated with alkaline phosphatase and addition of CDP-star chemiluminescent substrate allows detection by chemiluminescent signals [8, 9].
			</P></ARTICLE><ARTICLE class="flex-text-100"><H3> Positive/negative controls   </H3><P>
				Positive and negative controls were used in the methods to ensure reliable results and to simplify troubleshooting in case of unsuccessful results. This was especially important when we ran a gel, made a culture of bacteria that had gone through a heat shock or electroporation and when taking values on the micro plate reader.
			</P><H3>Overview of project layout</H3></ARTICLE><ARTICLE class="flex-text-100"><H3> Induction in different temperatures </H3><P>
				We wanted to find out which temperature would give rise to the most optimal expression of our fusion proteins with the different chaperone combinations. The plan was to try different temperatures and then use the data to fit our mathematical model so that it could predict the optimal expression temperature.
			</P></ARTICLE><ARTICLE class="flex-text-100"><H3> Induction with different concentrations of inducing agent </H3><P>
				Finding the right amount of inducing agents for each promoter we believed to be an important factor to investigate for finding the optimal protein expression. By testing bacteria with different concentrations of the inducing agents an approximative optimal concentration could be attained.
			</P></ARTICLE><ARTICLE class="flex-text-100"><H3> Induction with varied time  </H3><P>
				The third parameter of induction we wanted to study was time. We wanted to see which time would give rise to the most optimal expression for our proteins and thereby the most fluorescence. The plan was to increase the amount of induction time in relation to lower temperatures because of the lower growth rate. 
			</P></ARTICLE><ARTICLE class="flex-text-100"><H3> Finding the best combination of chaperones </H3><P>
				Our plan for the chaperone plasmid was, as already mentioned, to find the best combination of the chaperones in the best possible environment for maximum fluorescence. If this had succeeded, our plan was to amplify just that part of the superduper plasmid in a PCR and then create a new biobrick of that sequence. 
			</P></ARTICLE><ARTICLE class="flex-text-100"><H3> The model </H3><P>
				The purpose of our model was to enable faster information about what temperature, amount of inducing agent and time that would be the best for protein expression by doing experiments that could calibrate the model which in turn could inform us of promising new untested conditions. This model could also prevent us from focusing too much on local optimums and ensure our progress towards the global optimum.  
			</P></ARTICLE><ARTICLE class="flex-text-100"><H3> Host organism </H3><P>
				In this project we used 3 different kinds of E.coli strains BL21(DE3), XL1-Blue  and NEB-5alfa cells. The reason is that each strain is optimized for different uses. NEB-5alfa cells were used for heat shock transformation after HiFi -assembly since they are more competent than any other cells we used. XL1-Blue was used for amplification of DNA in preparation of cloning. BL21(DE3) was used for protein expression since thay lac the proteases lon and ompT (7).
			</P></ARTICLE><ARTICLE class="flex-text-100 text-break"><H3> Sources </H3><P class="text-small">
				1. Findeis MA. The role of amyloid β peptide 42 in Alzheimer’s disease. Vol. 116, Pharmacology and Therapeutics. 2007. p. 266–86.
				2. Fitzpatrick AWP, Falcon B, He S, Murzin AG, Murshudov G, Garringer HJ, et al. Cryo-EM structures of tau filaments from Alzheimer’s disease. Nature [Internet]. 2017;547(7662):185–90. Available from: http://www.nature.com/doifinder/10.1038/nature23002
				3. Shaner NC, Lambert GG, Chammas A, Ni Y, Cranfill PJ, Baird MA, et al. A bright monomeric green fluorescent protein derived from Branchiostoma lanceolatum. Nat Methods [Internet]. 2013;10(5):407–9. Available from: http://www.nature.com/doifinder/10.1038/nmeth.2413
				4. Bornhorst BJA, Falke JJ. Reprint of: Purification of Proteins Using Polyhistidine Affinity Tags. Protein Expression and Purification.
				5. Chen X, Zaro JL, Shen WC. Fusion protein linkers: Property, design and functionality. Vol. 65, Advanced Drug Delivery Reviews. 2013. p. 1357–69.
				6.	ThermoFisher Scientific. Fluorescnece Fundamentals [Internet]. ThermoFischer Scientific [Cited 1 Nov 2017] Available from: https://www.thermofisher.com/se/en/home/references/molecular-probes-the-handbook/introduction-to-fluorescence-techniques.html
                7. Jeong H, Kim HJ, Lee SJ. Complete Genome Sequence of Escherichia coli Strain BL21. Genome Announc [Internet]. 2015;3(2):e00134-15. Available from: http://genomea.asm.org/lookup/doi/10.1128/genomeA.00134-15
				8. BioLegend. Anti-beta-amyloid, 1-16 antibody. [Internet]. Available from:	https://www.biolegend.com/en-us/products/anti-beta-amyloid--1-16-antibody-10998 
				9. BioRad Immun-Star AP Chemiluminescent Protein Detection Systems [Internet]. Bio Rad Laboratories, Inc [Cited 1 Nov 2017] Available from: http://www.bio-rad.com/webroot/web/pdf/lsr/literature/4006074.pdf 
			</P></ARTICLE></DIV><FOOTER class="footer"><DIV id="foot_text"><P style="text-align: center;">LiU iGEM is proudly sponsored by</P></DIV><DIV class="bottom-footer"><DIV class="foot-1">
    LiU iGEM
    
    Kårallen, Linköpings Universitet
    
    581 83 Linköping
    <A href="https://2017.igem.org/Team:Linkoping_Sweden/Contact" style="color: white;">Contact us</A></DIV><DIV class="foot-2"><A href="https://liu.se/en/about-liu" target="_blank" style="color: white;">About LiU</A><A href="https://liu.se/en/education" target="_blank" style="color: white;">Study in Linköping</A><A href="https://liu.se/en/research" target="_blank" style="color: white;">Research</A><A href="https://liu.se/en" target="_blank" style="color: white;">www.liu.se</A></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>