<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_HUST-Innovators"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:HUST-Innovators">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:HUST-Innovators&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:HUST-Innovators&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:HUST-Innovators&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:HUST-Innovators" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:HUST-Innovators</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TABLE width="70%" align="center"><TBODY id="bar"><TR><TD align="center" colspan="3"><DIV id="nav"><UL><LI><A id="Home" onclick="change('detail_home')" style="color:#000000">Home </A></LI><LI id="Background" onmouseover="list(1)" onmouseout="clean(1)"><A href="#" style="color:#000000"> Background </A></LI><LI id="Project" onmouseover="list(0)" onmouseout="clean(0)"><A href="#" style="color:#000000"> Project </A></LI><LI id="model_bar" onmouseover="list(2)" onmouseout="clean(2)"><A href="#" style="color:#000000"> Modeling </A></LI><LI id="achievements" onmouseover="list(3)" onmouseout="clean(3)"><A href="#" style="color:#000000"> Achievements </A></LI><LI id="team" onmouseover="list(4)" onmouseout="clean(4)"><A href="#" style="color:#000000"> Team</A></LI><LI id="humanpractice" onmouseover="list(5)" onmouseout="clean(5)"><A href="#" style="color:#000000"> HumanPractice
</A></LI></UL><DIV id="second" style="width: 50px;position: absolute;font-size: 18px;filter: alpha(opacity=88);-moz-opacity:0.88;opacity:0.88; "><UL id="1" style="left: 104px; top: 50px; display: none;" onmouseover="list(1)" onmouseout="mouseOut(1)"><LI style="width: 100px;"><A onclick="change('SeqevolutionforSB')">Seq.evolution for Synthe.Bio.</A></LI><LI style="width: 100px;" onmouseover="list(6)" onmouseout="clean(6)"><A href="#">3rd Generation Sequencing</A></LI></UL><UL id="0" style="left: 230px; top: 50px; display: none;" onmouseover="list(0)" onmouseout="mouseOut(0)"><LI style="width: 80px;"><A onclick="change('OurProj')">Our Project</A></LI></UL><UL id="2" style="left: 325px; top: 50px; display: none;" onmouseover="list(2)" onmouseout="mouseOut(2)"><LI><A onclick="change('NewModel')">New Model</A></LI><LI><A onclick="change('DHAProduc')">DHA Production</A></LI></UL><UL id="3" style="left: 448px; top: 50px; display: none;" onmouseover="list(3)" onmouseout="mouseOut(3)"><LI style="width: 110px;"><A onclick="change('Software')">Software</A></LI><LI style="width: 110px;"><A onclick="change('PastSub')">Parts Submitted</A></LI></UL><UL id="4" style="left: 594px; top: 50px; display: none;" onmouseover="list(4)" onmouseout="mouseOut(4)"><LI style="width: 50px;"><A onclick="change('Intro')">Intro.</A></LI></UL><UL id="5" style="left: 680px; top: 50px; display: none;" onmouseover="list(5)" onmouseout="mouseOut(5)"><LI style="width: 120px;"><A onclick="change('WhatWeDid')">What we did</A></LI></UL><DIV id="third"><UL id="6" style="left: 225px; top: 130px; display: none;" onmouseover="list(6)" onmouseout="mouseOut(6)"><LI onmouseover="list(1)" onmouseout="mouseOut(1)"><A onclick="change('3rdWhat')">What is SMRT</A></LI><LI onmouseover="list(1)" onmouseout="mouseOut(1)"><A onclick="change('3rdWhy')">Why use SMRT</A></LI></UL></DIV></DIV></DIV></TD></TR></TBODY></TABLE></P><H2> Home </H2><H style="font-size:30px; line-height:50px; top:300px;" align="right">While the start of Synthetic Biology is the sequenced data from databases, if there are some new species whose genetic information is unknown, how can we continue the synthesis? Therefore, we decided to expand the definition and add modules  of Synthetic Biology...<A style="font-weight:bold; font-style:oblique;" href="#" onclick="change('SeqevolutionforSB')">more</A></H><H2> Evolution of Synthetic Biology  </H2><H3>Overture（IGS and Bioinformatics）</H3><SPAN>
In 1977 Sanger and colleagues introduced the &quot;dideoxy&quot; chain-termination method for sequencing DNA molecules, also known as the &quot;Sanger method&quot;.This was a major breakthrough and allowed long stretches of DNA to be rapidly and accurately sequenced. It earned him his second Nobel prize in Chemistry in 1980, which he shared with Walter Gilbert and Paul Berg.The new method was used by Sanger and colleagues to sequence human mitochondrial DNA (16,569 base pairs) and bacteriophage λ (48,502 base pairs).The dideoxy method was eventually used to sequence the entire human genome.
</SPAN><SPAN>
As the accumulation of sequence data, databases formed, which announced the birth of the bioinformatics.
</SPAN><SPAN>
In short, the bioinformatics is the harbinger of the Synthetic-Biology’s emergence.
</SPAN><H3>Sonata（NGS and Synthetic Biology）</H3><SPAN>
In short, the First GS has some fatal drawbacks such as high cost and low throughput. This is why it was replaced by NGS, such as 454, Solexa, Hiseq, etc.  The NGS has distinct effects on cost-reduction and time-saving with high accuracy. For instance, it took only one week to finish the human genome project by NGS, while the time using by First GS is 3 years.
</SPAN><SPAN>
As more and more species were sequenced, the size of genomic data is enlarged. As is known, the sequence of species is the major prerequisite of the SB. So the conclusion is drawn that the emergence of the NGS indicated the naissance of SB.
</SPAN><H3>cadenza (III GS and ???)</H3><SPAN style="font-weight:bolder;">
That is why we start our project this year.
</SPAN><H4>Docosahexaenoic Acid</H4><SPAN>
Docosahexaenoic acid (DHA) is an omega-3 fatty acid that is a primary structural component of the human brain, cerebral cortex, skin, sperm, testicles and retina. It can be synthesized from alpha-linolenic acid or obtained directly from maternal milk or fish oil. DHA's structure is a carboxylic acid with a 22-carbon chain and six  cis double bonds; with the first double bond located at the third carbon from the omega end. Its trivial name is cervonic acid, its systematic name is all-cis-docosa-4,7,10,13,16,19-hexa-enoic acid, and its shorthand name is 22:6(n-3) in the nomenclature of fatty acids.
</SPAN><SPAN>
Cold-water oceanic fish oils are rich in DHA. Most of the DHA in fish and multi-cellular organisms with access to cold-water oceanic foods originates from photosynthetic and heterotrophic microalgae, and becomes increasingly concentrated in organisms the further they are up the food chain. DHA is also commercially manufactured from microalgae: Crypthecodinium cohnii and another of the genus Schizochytrium. DHA manufactured using microalgae is vegetarian. 
</SPAN><H4> Thraustochytrids</H4><SPAN>
Thraustochytrids is rich in oceanic environment. And they can produce large amount of DHA. However our strain T-roseum ATCC28210 was still not sequenced yet. It means our project was blocked if we used the traditional Synthetic Biology Methods. To achieve our goal, we need to do something before the Traditional Process.
</SPAN><H2> Tips  </H2><SPAN>This wiki will be your team’s first interaction with the rest of the world, so here are a few tips to help you get started: </SPAN><UL><LI>State your accomplishments! Tell people what you have achieved from the start. </LI><LI>Be clear about what you are doing and what you plan to do.</LI><LI>You have a global audience! Consider the different backgrounds that your users come from.</LI><LI>Make sure information is easy to find; nothing should be more than 3 clicks away.  </LI><LI>Avoid using very small fonts and low contrast colors; information should be easy to read.  </LI><LI>Start documenting your project as early as possible; don’t leave anything to the last minute before the Wiki Freeze. For a complete list of deadlines visit the <A href="">iGEM 2013 calendar</A></LI><LI>Have lots of fun! </LI></UL><H2> What is SMRT </H2><SPAN><A style="font-weight:bolder;">Single molecule real time sequencing</A> (also known as SMRT) is a parallelized single molecule DNA sequencing by synthesis technology developed by Pacific Biosciences. Single molecule real time sequencing utilizes the zero-mode waveguide (ZMW), developed in the laboratories of Harold G. Craighead and Watt W. Webb[1] at Cornell University. A single DNA polymerase enzyme is affixed at the bottom of a ZMW with a single molecule of DNA as a template. The ZMW is a structure that creates an illuminated observation volume that is small enough to observe only a single nucleotide of DNA (also known as a base) being incorporated by DNA polymerase. Each of the four DNA bases is attached to one of four different fluorescent dyes. When a nucleotide is incorporated by the DNA polymerase, the fluorescent tag is cleaved off and diffuses out of the observation area of the ZMW where its fluorescence is no longer observable. A detector detects the fluorescent signal of the nucleotide incorporation, and the base call is made according to the corresponding fluorescence of the dye.
</SPAN><SPAN>
The DNA sequencing is done on a chip that contains many ZMWs. Inside each ZMW, a single active DNA polymerase with a single molecule of single stranded DNA template is immobilized to the bottom through which light can penetrate and create a visualization chamber that allows monitoring of the activity of the DNA polymerase at a single molecule level. The signal from a phospho-linked nucleotide incorporated by the DNA polymerase is detected as the DNA synthesis proceeds which results in the DNA sequencing in real time.
</SPAN><H3>Phospholinked nucleotide</H3><SPAN>
For each of the nucleotide bases, there are four corresponding fluorescent dye molecules that enable the detector to identify the base being incorporated by the DNA polymerase as it performs the DNA synthesis. The fluorescent dye molecule is attached to the phosphate chain of the nucleotide. When the nucleotide is incorporated by the DNA polymerase, the fluorescent dye is cleaved off with the phosphate chain as a part of a natural DNA synthesis process during which a phosphodiester bond is created to elongate the DNA chain. The cleaved fluorescent dye molecule then diffuses out of the detection volume so that the fluorescent signal is no longer detected.
</SPAN><H3>Zero-mode waveguide</H3><SPAN>
The zero-mode waveguide (ZMW) is a nanophotonic confinement structure that consists of a circular hole in an aluminum cladding film deposited on a clear silica substrate.[3] The ZMW holes are ~70 nm in diameter and ~100 nm in depth. Due to the behavior of light when it travels through a small aperture, the optical field decays exponentially inside the chamber.[4] The observation volume within an illuminated ZMW is ~20 zeptoliters (20 X 10−21 liters). Within this volume, the activity of DNA polymerase incorporating a single nucleotide can be readily detected.
</SPAN><H2> Why uses SMRT(The merits compared to the NGS) </H2><H3>Relative Terminologies</H3><H4>Contigs</H4><SPAN>
A sequence contig is a contiguous, overlapping sequence read resulting from the reassembly of the small DNA fragments generated by bottom-up sequencing strategies. This meaning of contig is consistent with the original definition by Rodger Staden (1979). The bottom-up DNA sequencing strategy involves shearing genomic DNA into many small fragments (&quot;bottom&quot;), sequencing these fragments, reassembling them back into contigs and eventually the entire genome (&quot;up&quot;). Because current technology allows for the direct sequencing of only relatively short DNA fragments (300–1000 nucleotides), genomic DNA must be fragmented into small pieces prior to sequencing. In bottom-up sequencing projects, amplified DNA is sheared randomly into fragments appropriately sized for sequencing. The subsequent sequence reads, which are the data that contains the sequence of each fragment, are assembled into contigs, which are finally connected by sequencing the gaps between them resulting in a sequenced genome. The ability to assemble contigs depends on the overlap of reads. Because shearing is random and performed on multiple copies of DNA, each portion of the genome should be represented multiple times in different fragment frames. In other words, the sequences of the fragments (and thus the reads) should overlap. After sequencing, the overlapping reads are assembled into contigs by assembly software.
</SPAN><H4>Scaffold</H4><SPAN>
Scaffolds consist of overlapping contigs separated by gaps of known length. The new constraints placed on the orientation of the contigs allows for the placement of highly repeated sequences in the genome. If one end read has a repetitive sequence, as long as its mate pair is located within a contig, its placement is known. The remaining gaps between the contigs in the scaffolds can then be sequenced by a variety of methods, including PCR amplification followed by sequencing (for smaller gaps) and BAC cloning methods followed by sequencing for larger gaps.
</SPAN><H3>Advancement from NGS</H3><H4>Extremely Long Reads</H4><SPAN>
The NGS sequencing Technology requires prohibitive storage space and operation time for the sake of relatively short digital crumbs( In fact, the average length of NGS reads is far longer than the Sanger, the first GS reads). Clearly, the shortage of NGS is their assembly, like puzzle games. The shorter the reads are, the more barriers we will meet in the concatenation process. Unfortunately, this problem is fundamental data property, even if we have more proper modeling and more efficient algorithms. Until now we have found no chance to compensate this demerit.
</SPAN><SPAN>
Comparatively speaking, the SMRT sequencing operating on PacBio RS II 
</SPAN><SPAN>
1.<A style="font-weight:bold; font-style:oblique;">Incredibly long read length</A>: the average read length is between 5，000 to 8，000 bases per read( the average length of NGS read ranges from 150 to 400 bp). The maximum read length can reach 20，000 bp.

For instance, the USDA had intended to sequence the microbes in goats. Compared to the least 18 contigs produced by the NGS, which means the project is uncompleted, the SMRT can get the ultimate contig—a successful sequencing.
</SPAN><SPAN>
2.<A style="font-weight:bold; font-style:oblique;">High accuracy</A>：In short, the accuracy of sequencing can reach 99.999%. 
</SPAN><SPAN>
3.<A style="font-weight:bold; font-style:oblique;">GC Problem</A>：For the NGS, areas with high GC concentration mean low coverage. Only insufficient info can be dug out for assembly. This is why many gaps between DNA sequences emerge in the result.

In SMRT, the coverage do not fluctuate as the GC content varies. So the Problem can be avoided.
</SPAN><H2> Our Project </H2><H3>Sequencing</H3><SPAN>First, we decided to sequence the unknown Thraustochytrium roseum. As we read the thesis on Velvet, SSpace, GRASS and so on, we finally made a determination to use the struture of the GRASS to do the scaffolding process after finishing getting contigs. As the data is sequenced from the 3rd Generation Sequencing Method, we had changed some lines of the code in order to fix the characteristics of the data. 
GRASS has some progresses: Sort, Bundle, Extract, RemoveAmbiguous, Erode   IsolateContigsThink. Those are not needed while we are using the SMRT sequencing data. So we deleted code about them. 
Think about the NGS Assembly, nearly all the softwares use pair-end reads to link the contigs through the scaffolding process. Knowing that the reads from the Pacbio machines are extremely,after self-overlapping, we let the Pacbio sequence replace the contigs in GRASS to do the scaffolding.
Here is a part of result:
</SPAN><SPAN>
Then, with the help of Nextomics, we finished the annotation and found that the scaffold shown above is the required one.
</SPAN><H3>DHA engineering yeast</H3><H4>Abstract</H4><H style="font-size:18px;">Unsaturated fatty acids</H><SPAN>Unsaturated fatty acids have one or more double bonds between carbon atoms. (Pairs of carbon atoms connected by double bonds can be saturated by adding hydrogen atoms to them, converting the double bonds to single bonds. Therefore, the double bonds are called unsaturated.)
The two carbon atoms in the chain that are bound next to either side of the double bond can occur in a cis or trans configuration.
</SPAN><H style="font-size:18px;">Traditional origin</H><H5 style="font-weight:bold; font-style:oblique;">A.Deep sea fish oil</H5><SPAN>
This is the major source of PUFAs. The disadvantages include complex purification processes and limited resources.
</SPAN><H5 style="font-weight:bold; font-style:oblique;">B.Marine microalgae</H5><SPAN>
This is the original source of the PUFAs. The disadvantages are low productivity and unstability output.
</SPAN><H style="font-size:18px;">The synthesis path of PUFAs</H><SPAN>
In our project, we use EPA and ARA as substrates to produce DHA and DPA.
</SPAN><H4>Technical route</H4><SPAN>
Thraustochytrids and Isochrysis galbana as materials is used for producing DHA and DPA in yeast in order to provide research materials and scientific data.

Results:
</SPAN><H2> New Modle </H2><H3>The INNOVATION of the Synthetic-Biology’s Modules</H3><SPAN>Now the Biobricks become the transition of 2 submodules. </SPAN><H3>How to find the biobricks</H3><SPAN style="font-style:oblique;">Traditional Method:</SPAN><SPAN>Its origin is the Database. However, like our strain T-roseum ATCC28210 which was not sequenced yet, we could not get the biobricks.</SPAN><SPAN style="font-style:oblique;">Novel Definition:</SPAN><SPAN>After adding a new part of modules, we successfully finished our project.</SPAN><H2> DHA Production </H2>definitionunitn<A style="font-size: 8px;">1</A>Methanol concentrationMn<A style="font-size: 8px;">ARA</A>ARA concentrationMn<A style="font-size: 8px;">EPA</A>EPA concentrationMr<A style="font-size: 8px;">1</A>Concentration of transcribed mRNAs of PAOX1Mr<A style="font-size: 8px;">2</A>Concentration of transcribed mRNAs of PAOX1Mr<A style="font-size: 8px;">3</A>Concentration of transcribed mRNAs of PAOX5'Mm<A style="font-size: 8px;">1</A>Max transcription rates of PAOX1nM/Sm<A style="font-size: 8px;">2</A>Max transcription rates of PAOX5nM/Sm<A style="font-size: 8px;">3</A>Max transcription rates of PAOX5'nM/Sδ<A style="font-size: 8px;">1</A>Degradation rate of EPA1/Hδ<A style="font-size:8px;">2</A>Degradation rate of DHA1/Hδ<A style="font-size:8px;">3</A>Degradation rate of DPA1/HkTranslation rateM/SγDegradation rate of mRNAM/S<H2> Software  </H2><SPAN>
The software is decided for those who is not familiar to the Bioinformatics and Computer Science. We gave it a simple interface and simple buttons. </SPAN><SPAN>
What guests need to do is to print the location of the required files. Then a final sequencing result will emerge. The software works on Unix system only.
Downloading: <A href="https://static.igem.org/mediawiki/2014/8/8a/Build-untitled6-Desktop-Debug.zip">click here</A>PS: Perl and Comprehensive Perl Archive Network are needed. 
</SPAN><H2> Part Submitted </H2><H2> Team Introduction </H2><SPAN>The iGEM HUST-INNOVATORS 2014 is an energetic family composed of 10 undergraduate students   and 2 advisors. Our members come from a variety of departments, majoring in non-bio-concerned subjects like Energy &amp; Power Engineering, Computer Science. And other 5 of us come from biology department.The detailed work distribution is as follows:</SPAN><SPAN>Leaders: He Yu, Zhang Yue
Advisors: Chen Gang, Wang Depeng ,Gong Yangming
Team Members: Tang Yu, Dong Xiaolei, Wang Yiqiao, Hou Yuhan 
</SPAN><H3> Attribution </H3><SPAN><A style="font-weight:bold">Sequencing Procedure Group:</A>Proposal: Zhang Yue
  Program designer: Tangyu, Dong Xiaolei
Documentation: Zhang Yue, Hou Yuhan
</SPAN><SPAN><A style="font-weight:bold">Experiments Group:</A>Proposal: He Yu
  Program designer: Liang Qihua, Zhang Zihe, Li Xiaotong, Yang Kairan
</SPAN><SPAN><A style="font-weight:bold">Modeling: </A>Main Designer: Hou Yuhan
   Modeling :Hou Yuhan, Wang Yiqiao
</SPAN><SPAN><A style="font-weight:bold">Wiki:</A>Main designer: Mo Bufei
    Content: Tang Yu, Dong Xiaolei, Wang Yiqiao, Hou Yuhan,
Liang Qihua, Zhang Zihe, Li Xiaotong, Yang Kairan
</SPAN><H3> Acknowledgment </H3><SPAN>
We are truly grateful to the following people and organizations for their kind support with funding, materials, facilities, and professional advice:
</SPAN><SPAN>
Prof. Gong Yangming, the researcher of the OILCROP REASEARCH INSTITUTE, for providing the strain of the Thraustochytrids and the experiments platform for us.
</SPAN><SPAN> 
Mr. Wang Depeng, the head of Nextomics, for helping us to change some part of code of some assembly tools and sequence the Thraustochytrids we need to use.
</SPAN><SPAN>
Mr. Liang Fan, a proposal of Nextomics, for helping us construct our tools on computers.
</SPAN><SPAN>
Mr. Liu Zhenhua, a proposal of Nextomics, for teaching us knowledge about sequencing history and tools. 
</SPAN><SPAN>
Prof. Chen Gang, the Chef of the School of Energy and Power Engineering, HUST, for instructing our work and supporting our team fund and transportation fee.
</SPAN><SPAN>
Mr. Mo Bufei, for helping us construct our wiki before wiki freeze. 
Team WHU, for helping us construct the biobrick. 
</SPAN><H2> Special Thanks </H2><FONT size="5px">Sincerely,we express our gratefulness to all the members in Team WHU-China. As our team is not from the College of Life Science 
and Technology in HUST University, we found some troubles while finishing our experiments. When we could not find anyone for help 
immediately, however, the Team WHU-China gave us a hand. They provided us a lab and gave us instructions during experiments, and 
finally helped us construct our biobrick（BBa_K1551000) successfully.</FONT><H2> What we did </H2><H3>Safety</H3><H4>Would any of your project ideas raise safety issues in terms of: researcher safety, public safety, or environmental safety?</H4><SPAN>-All researchers have been trained with lab safety protocols and conducted the experiment strictly following the safety protocols. No one was injured or infected while working on our project. The microorganisms used in our project are T-roseum ATCC28210, which are not toxic or pathogenic. No lab waste was released to the environment before sterilization,too.</SPAN><H4>Do any of the new BioBrick parts (or devices) that you made this year raise any safety issues?</H4><SPAN>-No. None of the BioBrick parts we made pose any safety problems we can foresee. None of the parts we made produce toxic or pathogenic protein, and all parts are found in common bacterial species.</SPAN><H4>Is there a local biosafety group, committee, or review board at your institution?</H4><SPAN>- The OILCROP RESEARCH INSTITUTE, CHINESE ACADEMY OF AGRICULTURAL SCIENCES makes sure that every project is evaluated of its bio-safety issues before initiation. The head of Institute and our advisors approved that our project would not raise safety problems, etc.</SPAN><H4>Do you have any other ideas how to deal with safety issues that could be useful for future iGEM competitions? How could parts, devices and systems be made even safer through biosafety engineering?</H4><SPAN>- iGEM could make every team sign a safety form when applying, in order for all teams to follow a same set of safety rules.</SPAN><H3>Total Innovation for Synthetic Biology</H3><FONT size="5px">In the initiation of our Project, we found our target part was not sequenced yet. As a result, the Project got swamped. So we chose to cooperate with Nextomics. In the first place, we introduced Synthetic Biology to the staff of Nextomics. Consequently, we added some certain part of bioinformatics to our modules, expanding the definition of Synthetic Biology. In conclusion, not only did we achieved our Project, but also gave Synthetic Biology a privilege to expand its biobricks collection from species directly, not just from the databases. 
</FONT><H4>Cooperation with Nextomics</H4><SPAN>While doing our project, we had an cooperation with Nextomics. The company is a good at dealing with the SMRT Data, which helped us construct the biobricks.</SPAN><H5>“What are the benefits using SMRT Data while the construction of biobricks?”</H5><H5>“Faster, cheaper and able to be used by individuals.”</H5><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:HUST-Innovators">http://2014.igem.org/Team:HUST-Innovators</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:HUST-Innovators" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:HUST-Innovators" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:HUST-Innovators&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:HUST-Innovators&amp;oldid=377420" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>