<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ZJU-China_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ZJU-China/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="header-w3layouts"><NAV class="navbar navbar-default navbar-fixed-top"><DIV class="collapse navbar-collapse navbar-ex1-collapse"><UL class="nav navbar-nav navbar-right cl-effect-15"><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">Overview</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Overview">Project Description</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Achievements">Achievements</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/ImproveParts">Improve Parts</A></LI></UL><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">Project</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Project/tp">Trichoderma Proof</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Project/voc">VOC sensors</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Project/st">Signal Transduction</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Project/Downstream">Downstream</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Project/conclusion">Conclusion</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Notebook">Notebook</A></LI><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">Modelling</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Model">VOC analysis</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Model/Coculture">Coculture</A></LI><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">Parts</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Parts">All Parts</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Parts/Basic">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Parts/Composite">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Parts/Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Hardware">Hardware</A></LI><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">Safety</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Safety">Environment</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Safety/Lab">Laboratory</A></LI><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">HP</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Human_Practices">Summary</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/HP/Gold_Integrated">Gold</A></LI><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">Team</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Team">Teammates</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Attributions">Attribution</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Collaborations">Collaboration</A></LI></DIV></NAV></DIV></DIV><DIV style="width: 100%" class="container zjuContent"><DIV class="col-md-9" role="main"><DIV class="bs-docs-section"><H1 id="design" class="page-header ArticleHead GreenAH">Design</H1><H2 id="introduction" class="H2Head">Introduction</H2><P class="PP">When our <EM>T.atroviride</EM> is activated by the signals which have been described above, they will produce the corresponding effects to save our little plants. Those can be zwittermycin A, chitinase, serine protease and anything you need to protect your lovely plants. Zwittermycin A is an antibiotic which can inhibite the growth of <EM>P. nicotianae</EM>. Chitinase is a lytic enzyme that breaks down fungal cell walls. Serine protases plays an important role in hydrolyzing the eggshell of root-knot nematodes. With these fungal growth inhibitors our engineered <EM>T.atroviride</EM> will be able to better protect our plants.</P><P class="capture">Fig.1 Genetic circuit of the downstream</P><H3 id="za" class="H3Head">Zwittermicin A</H3><P class="PP">Zwittermicin A is an antibiotic that  has the potential to suppress plant disease due to its broad spectrum activity against certain gram positive and gram negative prokaryotic micro-organisms. Since <EM>T.atroviride</EM> does not produce Zwittermycin A by itself, a gene cluster obtained from  Bacillus cereus UW85 was introduced into <EM>T.atroviride</EM>. The genes responsible for the production of zwittermicin A are located on a 16 kb cluster containing nine orfs, from orf1 to orf9, and a self resistant gene zmaR, a gene that encodes an acylation enzyme that deactivate zwittermicin A.[1]</P><P class="capture">Fig.2A Gene organization of the Zwittermicin A biosynthetic cluster</P><P class="capture">Fig.2B Units used in Zwittermicin A Production.[2]</P><H3 id="sp" class="H3Head">Serine protease</H3><P class="PP">Root-knot nematodes (Meloidogyne spp.), which are one of the most destructive nematodes, cause the loss of crop about 10%, serious as high as 75%. [3] In the present, the egg-parasitic fungus P.lilacinum is the main biocontrol material of root-knot nematodes.P.lilacinum. secretes protease and chitinase to hydrolyze the nematode eggshell, so that the root knot nematodes cannot grow normally.[4] However, because P.lilacinum can live in the cornea, the usage of P.lilacinum is still dangerous. Therefore, in this part, the purpose of our project is to give our harmless <EM>T.atroviride</EM> the ability to kill the root-knot nematodes by overexpressing serine protease which plays an important role in hydrolyzing the eggshell of root-knot nematodes.</P><H3 id="ch" class="H3Head">Chitinase</H3><P class="PP">Chitinase is a hydrolytic enzyme that breaks down hydrolytic bonds in chitin.  As chitin is a component of the cell walls of fungi and exoskeletal elements of some animals (including worms and arthropods), chitinase has been shown to be useful in biological control against fungi.[5] Therefore, in order to inhibit fungl growth, our <EM>T.atroviride</EM> can produce chitinase when the plants are infested by fungi and the signal conversion systems work well.</P><H3 id="anythingelse" class="H3Head">Anything else</H3><P class="PP">Your lovely plants will face many challenges in the complex and dangerous soil condition, so that, the little plants must be protected by the strong <EM>T.atroviride</EM> which can product corresponding effects to inhibite the fungi. The following table can help you choose the right inhibitor to help your plants.</P><H2 id="result" class="H2Head">Result</H2><P class="PP">Because of lacking of time, we only did the experience of serine protease. We structured two plasimads: one could work in the yeast and the other could work in <EM>T.atroviride</EM>.</P><H3 id="yeast" class="H3Head">Yeast</H3><P class="PP">The gene which can express serine protease in yeast was synthesized by Genscript. Before synthesizing this gene, we did codon optimization based on the codon preference of yeast and added a flag-tag to the N-terminal of the serine protease.After extracting the whole proteins of the yeast which transferred plasmid successfully, we performed western-blot and checked the serine protein was expressed in the yeast. (Result is as follow) The band was very shallow, in other words, the concentration of the serine protease was very low.</P><P class="capture">Fig3Â· Western-blot result</P><P class="capture">The band with red circle was the band of the serine protease and the marker was a protein marker of aidlab./p&gt;
                  </P><P class="PP">In order to test whether the serine protease could work normally in the yeast, we performed the enzyme activity detection using BAEE solution. We did two sets of experiments: one added PMSF, which was a inhibitor of serine protease, and the other did not. And then, reading the OD253 of these two solutions.(You can know more details about the detection from the protocol) Obviously, the OD253 of the former one is higher than the later one and the values of OD253 increased with time in a period of time ; therefore, we made the conclusion that the yeast produced the serine protase successfully and effectively.</P><P class="capture">Fig.4 The values of OD253 increased with time</P><P class="capture">Fig.5 The values of OD253 increased with time and this solution did not add the PMSF. </P><P class="PP">These were the OD253 of the solutions after a period of reaction.The red one was the solution that did not add the PMSF; the blue one was the solution that added. Obviously, the OD253 of the former one is higher than the later one, so that, we could say that the yeast produced the serine protase successfully and effectively.</P><H3 id="ta" class="H3Head"><EM>T.atroviride</EM></H3><P class="PP">By contrast to the yeast, the serine protease that worked in the <EM>T.atroviride</EM> was cloned from the genome of P.lilacinum for they have high homology. Because of lacking of time to extract protein from <EM>T.atroviride</EM>, we putted the EGFP gene behind the serine protease gene, so that, we could test the serine protease by detecting the fluorescent. From the picture below, mycelium was fluorescent and the <EM>T.atroviride</EM> expressed the protease successfully. In the future, we will extract the serine protease and detect the activity of it.</P><P class="capture">Fig.6 Fluorescence of mycelium</P><H2 id="ref" class="H2Head">Reference</H2><P class="ref">[1] Stohl E A, Milner J L, Handelsman J. Zwittermicin A biosynthetic cluster[J]. Gene, 1999, 237(2):403-411.</P><P class="ref">[2] Zwittermicin A, https://en.wikipedia.org/wiki/Zwittermicin_A  ,7 June 2016(24/10/2017)</P><P class="ref">[3] Wang J P, Wang J X, Liu F, et al. Enhancing the virulence of Paecilomyces lilacinus against Meloidogyne incognita eggs by overexpression of a serine protease. Biotechnology Letters, 32, 1159-1166[J]. Biotechnology Letters, 2010, 32(8):1159-1166.</P><P class="ref">[4] Brand D, Roussos S, Pandey A, et al. Development of a bionematicide with Paecilomyces lilacinus to control Meloidogyne incognita.[J]. Applied Biochemistry &amp; Biotechnology, 2004, 118(1-3):81-88.</P><P class="ref">[5] SÃ¡mi L, Pusztahelyi T, Emri T, et al. Autolysis and aging of Penicillium chrysogenum cultures under carbon starvation: Chitinase production and antifungal effect of allosamidin.[J]. Journal of General &amp; Applied Microbiology, 2001, 47(4):201.</P></DIV><DIV class="col-md-3 disappear-on-top" role="complementary"><NAV style="position: fixed; top: 100px ; left:50px; " class="bs-docs-sidebar hidden-print hidden-xs hidden-sm"><UL class="nav bs-docs-sidenav"><LI><A href="#introduction">Introduction</A></LI><LI><A href="#za">Zwittermicin A</A></LI><LI><A href="#sp">Serine protease</A></LI><LI><A href="#ch">Chitinase</A></LI><LI><A href="#anythingelse">Anything else</A></LI></UL><LI><A href="#result">Result</A></LI><LI><A href="#yeast">Yeast</A></LI><LI><A href="#ta">T.atroviride</A></LI><LI><A href="#ref">Reference</A></LI></NAV></DIV></DIV></DIV><DIV class="footer" style="background-color: #000000"><DIV class="copyright container text-center"><P class="copyright">Copyright Â© 2017 ZJU-IGEM | <A href="http://www.zju.edu.cn/">Zhejiang University</A> |</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>