<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ZJU-China_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ZJU-China/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="header-w3layouts"><NAV class="navbar navbar-default navbar-fixed-top"><DIV class="collapse navbar-collapse navbar-ex1-collapse"><UL class="nav navbar-nav navbar-right cl-effect-15"><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">Overview</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Overview">Description</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Achievements">Achievements</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Improve">Improve Parts</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/InterLab">InterLab</A></LI></UL><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">Project</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Project/tp">Trichoderma Proof</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Project/voc">VOC sensors</A></LI><LI><A style="font-size: 0.7em!important;" href="https://2017.igem.org/Team:ZJU-China/Project/st">Chemical Signal Transduction</A></LI><LI><A style="font-size: 0.7em!important;" href="https://2017.igem.org/Team:ZJU-China/Project/mt">Medium Wave Transduction</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Project/Downstream">Downstream</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Project/conclusion">Conclusions</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Protocols">Protocols</A></LI><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">Model</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Model">VOC analysis</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Model/Coculture">Coculture</A></LI><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">Parts</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Parts">All Parts</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Part_Collection">Part Collection</A></LI><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">Hardware</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Hardware">Overview</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Hardware/Device">Device</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Hardware/Improvements">Improvements</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Hardware/MediumWave">Medium Wave</A></LI><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">Safety</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Safety">Environment</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Safety/Lab">Laboratory</A></LI><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">HP</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Human_Practices">Summary</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/HP/Gold_Integrated">Gold Integrated</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Engagement">Engagement</A></LI><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">Team</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Team">Teammates</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Attributions">Attribution</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Collaborations">Collaboration</A></LI></DIV></NAV></DIV></DIV><DIV style="width: 100%" class="container zjuContent"><DIV class="col-md-9" role="main"><DIV class="bs-docs-section" style="list-style: none !important;"><H1 class="page-header ArticleHead GreenAH">Protocol</H1><H1 id="GeneralEcoliprotocol" class="YellowAH ArticleHead">General/E.coli protocol</H1><UL><LI><DIV class="clickMore" id="protocolPart1"><H2 class="H2Head">General cloning workflow</H2><H3 class="H3Head">Plasmid Cloning</H3><P class="PP">PCR was widly used to make a copy of a piece of DNA and at the same time add restriction sites to the ends of that piece of DNA so that it can be easily cloned into a plasmid of interest.</P><P class="PP">Designing primers for PCR based cloning. Before running PCR, we needed to design appropriate primers based on the method that we chose to clone a piece of DNA into a plasmid. For example, in our experiment,  we used EcoRI and XabI to ligate our target gene into the recipient plasmid. Therefore, PCR primers for molecular cloning consisted of not hybridization sequence, but EcoRI and XabI restriction site.</P><P class="PP">Run PCR and test. In order to amplify insert DNA, we used a high fidelity polymerase to minimize mutations. After that, running Agarose Gel Electrophoresis to check whether the DNA size was correct.</P><P class="PP">Isolate the insert and vector by gel purification or PCR purified.</P><P class="PP">Ligate the insert into vector. Insert and vector backbone DNA was digested with the appropriate restriction enzymes. In fact, we had try many other ways to ligate the insert into vector, for example, In-Fusion Cloning.</P><H3 class="H3Head">Transformation</H3><P class="PP">Transformed with 5-10μl ligation reaction into competent cells using the general heat shock.</P><H3 class="H3Head">Screening positive clones</H3><P class="PP">If there were a high number of colonies on plate, colony PCR was used and only the positive clones were mini-prepped.  On the contrary, we picked single colonies for mini-prepping.</P><P class="PP">After purifying the DNA, we conducted a diagnostic restriction digest or PCR  and products analysed using agarose gel electrophoresis to confirm if correct construct was present.</P><H3 class="H3Head">Verify Plasmid by Sequencing</H3><P class="PP">Positive clones were sent for sequencing of the insert using appropriate primers.</P></DIV><SPAN class="key CuteButton SmallCB GreenCB" onclick="fn(this,1)">More</SPAN></LI><LI><DIV class="clickMore" id="protocolPart2"><H2 class="H2Head">Restriction Digestion</H2><H3 class="H3Head">Materials</H3><P class="PP">Thermo Scientific FastDigest enzyme</P><P class="PP">10X FastDigest</P><P class="PP">DNA</P><P class="PP">Purified water, nuclease-free</P><H3 class="H3Head">Methods</H3><TABLE class="table"><TBODY><TR><TD>Plasmid DNA(Double digestion)</TD></TR><TR><TD>Sterile water</TD><TD>to 20 ul</TD></TR><TR><TD>10X FatDigest</TD><TD>2 ul</TD></TR><TR><TD>DNA</TD><TD>2ul(up to 1 ug)</TD></TR><TR><TD>FastDigest enzyme</TD><TD> 1ul+1ul</TD></TR></TBODY></TABLE><P class="PP">Set up the reaction following the instruction in the table above.</P><P class="PP">Incubate digestion reaction at 37°C, 20 min. </P><P class="PP">Perform heat deactivation at 65°C for 15 minutes, if not running on a gel at the end of incubation.</P></DIV><SPAN class="key CuteButton SmallCB GreenCB" onclick="fn(this,2)">More</SPAN></LI><LI><DIV class="clickMore" id="protocolPart3"><H2 class="H2Head">Ligation</H2><H3 class="H3Head">Materials</H3><P class="PP">TaKaRa DNA Ligation Kit Ver.2.1</P><P class="PP">DNA solution</P><H3 class="H3Head">Methods</H3><P class="PP">Mix linearized plasmid vector DNA and a DNA fragment in a total volume of 5 - 10 μl.</P><P class="PP">Add an equal volume of Solution I (5 - 10 μl) as the DNA solution and mix thoroughly.</P><P class="PP">Incubate at 16℃ for 30 minutes</P><H3 class="H3Head">More details</H3><P class="PP">DNA Ligation Kit Ver. 2.1</P></DIV><SPAN class="key CuteButton SmallCB GreenCB" onclick="fn(this,3)">More</SPAN></LI><LI><DIV class="clickMore" id="protocolPart4"><H2 class="H2Head">PCR Amplification</H2><H3 class="H3Head">Materials:</H3><P class="PP">dNTP mix</P><P class="PP">Upstream primer</P><P class="PP">Downstream primer</P><P class="PP">ddH2O</P><P class="PP">Phanta Max Super-Fidelity DNA Polymerase</P><P class="PP">DNA template</P><P class="PP">2X Phanta Max buffer</P><H2 class="H2Head">Methods:</H2><TABLE id="partTable" class="table tableNice"><TBODY><TR><TD>components</TD><TD>50ul reaction volume</TD></TR><TR><TD>dNTP mix</TD><TD>1 ul</TD></TR><TR><TD>Upstream primer</TD><TD>2 ul</TD></TR><TR><TD>Downstream primer</TD><TD>2ul</TD></TR><TR><TD>2X Phanta Max buffer</TD><TD>25ul</TD></TR><TR><TD>ddH2O</TD><TD>To 50ul</TD></TR><TR><TD>Phanta Max Super-Fidelity DNA Polymerase</TD><TD>1ul</TD></TR><TR><TD>DNA template</TD><TD>variable</TD></TR></TBODY></TABLE><P class="PP">Set up the reaction following the instruction in the table above.</P><TABLE class="table tableNice"><TBODY><TR><TH>Segment</TH><TH>Number of cycles</TH><TH>Temperature(℃)</TH><TH>Duration</TH></TR><TR><TH>Initial denaturation</TH><TD>1</TD><TD>95</TD><TD>3min(30 sec)</TD></TR><TR><TH>PCR</TH><TD>95</TD><TD>15 sec</TD></TR><TR><TD>25-35 cycles</TD><TD>55-65</TD><TD>15 sec</TD></TR><TR><TD>72</TD><TD>30-60 sec/kb</TD></TR><TR><TH>Final extension</TH><TD>1</TD><TD>72</TD><TD>5 min</TD></TR><TR><TH>Hold</TH><TD>1</TD><TD>4</TD><TD>∞</TD></TR></TBODY></TABLE></DIV><SPAN class="key CuteButton SmallCB GreenCB" onclick="fn(this,4)">More</SPAN></LI><LI><DIV class="clickMore" id="protocolPart5"><H2 class="H2Head">PCR Purification, Gel Extraction &amp; Miniprep</H2><H3 class="H3Head">PCR purification was performed according to the Axygen® AxyPrep™ PCR Clean-Up Kit.</H3><P class="PP"><STRONG>Materials:</STRONG></P><P class="PP">Buffer PCR-A</P><P class="PP">2 ml Microfuge tube</P><P class="PP">Buffer W2</P><P class="PP">1.5 ml Microfuge tube</P><P class="PP">Eluent</P><P class="PP"><STRONG>Methods:</STRONG></P><P class="PP">Add a 3×reaction volume of Buffer PCR-A to the sample. If the required volume of Buffer PCR-A is less than 100 μl, add 100 μl of Buffer PCR-A. Vortex briefly to mix the contents.</P><P class="PP">Place a  PCR column into a 2 ml Microfuge tube (provided). Pipette the reaction from Step1 into the PCR column. Centrifuge at 12,000×g for 1 minute.</P><P class="PP">Discard the filtrate from the 2 ml Microfuge tube.  Return the PCR column to the 2 ml  Microfuge tube. Pipette 700 μl of Buffer W2 into the column and centrifuge at 12,000×g for 1 minute.</P><P class="PP">Discard the filtrate. Return the PCR column to the 2 ml Microfuge tube. Pipette 400 μl  of Buffer W2 into the column and centrifuge at 12,000×g for 1 minute.</P><P class="PP">Transfer the PCR column into a clean 1.5 ml Microfuge tube (provided). To  elute the  DNA,  add 25-30μl of Eluent (pre-warmed at 65°C) to the center of the membrane. Let it stand for 1 minute at room temperature. Centrifuge at 12,000×g for 1 minute.</P><H3 class="H3Head">Gel extraction was performed according to the Axygen® AxyPrep™ DNA Gel Extraction Kit.</H3><P class="PP"><STRONG>Materials:</STRONG></P><P class="PP">1.5 ml microfuge tube</P><P class="PP">Buffer DE-A</P><P class="PP">Buffer DE-B</P><P class="PP">DNA fragment</P><P class="PP">2 ml microfuge tube</P><P class="PP">Buffer W1</P><P class="PP">Buffer W2</P><P class="PP">Eluent</P><P class="PP"><STRONG>Methods:</STRONG></P><P class="PP">Excise the agarose gel slice containing the DNA fragment of interest with a clean, sharp scalpel under ultraviolet illumination. Briefly place the excised gel slice on absorbent toweling to remove residual buffer. Transfer the gel slice to a piece or plastic wrap or a weighing boat. Mince the gel into small pieces and weigh. In this application, the weight of gel is regarded as equivalent to the volume. For example, 100 mg of gel is equivalent to a 100 μl volume. Transfer the gel slice into a 1.5 ml microfuge tube.</P><P class="PP">Add a 3x sample volume of Buffer DE-A.</P><P class="PP">Resuspend the gel in Buffer DE-A by vortexing. Heat at 75°C until the gel is completely dissolved (typically, 6-8 minutes). Heat at 40°C if low-melt agarose gel is used. Intermittent vortexing (every 2-3 minutes) will accelerate gel solubilization.</P><P class="PP">Add 0.5x Buffer DE-A volume of Buffer DE-B, mix. If the DNA fragment is less than 400 bp, supplement further with a 1x sample volume of isopropanol.</P><P class="PP">Place a Miniprep column into a 2 ml microfuge tube. Transfer the solubilized agarose from Step 4 into the column. Centrifuge at 12,000xg for 1 minute.</P><P class="PP">Discard the filtrate from the 2 ml microfuge tube. Return the Miniprep column to the 2 ml microfuge tube and add 500 μl of Buffer W1. Centrifuge at 12,000xg for 30 seconds.</P><P class="PP">Discard the filtrate from the 2 ml microfuge tube. Return the Miniprep column to the 2 ml microfuge tube and add 700 μl of Buffer W2. Centrifuge at 12,000xg for 30 seconds.</P><P class="PP">Discard the filtrate from the 2 ml microfuge tube. Place the Miniprep column back into the 2 ml microfuge tube. Add a second 700 μl aliquot of Buffer W2 and centrifuge at 12,000xg for 1 minute.</P><P class="PP">Discard the filtrate from the 2 ml microfuge tube. Place the Miniprep column back into the 2 ml microfuge tube. Centrifuge at 12,000xg for 1 minute.</P><P class="PP">Transfer the Miniprep column into a clean 1.5 ml microfuge tube (provided). To elute the DNA, add 25-30 μl of Eluent or deionized water to the center of the membrane. Let it stand for 1 minute at room temperature. Centrifuge at 12,000xg for 1 minute.</P><H3 class="H3Head">Minipreps were carried out according to the  Axygen® Plasmid Miniprep Kit.</H3><P class="PP"><STRONG>Materials:</STRONG></P><P class="PP">Buffer S1</P><P class="PP">Buffer S2</P><P class="PP">Buffer S3</P><P class="PP">2 ml Microfuge tube</P><P class="PP">Miniprep column</P><P class="PP">Buffer W2</P><P class="PP">Buffer W1</P><P class="PP">Eluent</P><P class="PP"><STRONG>Methods:</STRONG></P><P class="PP">Collect 1-4 ml of overnight LB culture. Centrifuge at 12,000×g for 1 minute to pellet the bacteria.Decant or pipette off as much of the supernatant as practical.</P><P class="PP">Resuspend the bacterial pellet in 250 μl of Buffer S1 by vortexing. Please be sure that the bacteria are completely resuspended before proceeding.</P><P class="PP">Add 250 μl of Buffer S2, and mix by gently inverting the tube for 4-6×. Do not vortex.</P><P class="PP">Add 350 µl of Buffer S3, and mix by gently inverting 6-8×. Centrifuge at 12,000×g for 10 minutes to clarify the lysate. Do not vortex.</P><P class="PP">Place a Miniprep column into an uncapped 2 ml Microfuge tube. Transfer the clarified supernatant from Step 4 into the Miniprep column. Transfer the Miniprep column and 2 ml Microfuge tube to microcentrifuge and spin at 12,000×g for 1 minute.</P><P class="PP">Pipette 500 µl of Buffer W1 into each Miniprep column. Centrifuge at 12,000×g for 1 minute.</P><P class="PP">Pipette 700 µl of Buffer W2 into each Miniprep column. Centrifuge at 12,000×g for 1 minute.</P><P class="PP">Discard the filtrate from the 2 ml Microfuge tube. Place the Miniprep column back into the 2 ml Microfuge tube. Add 700 μl of Buffer W2 to the Miniprep column and centrifuge at 12,000×g for 1 minute.</P><P class="PP">Discard filtrate from the 2 ml Microfuge tube. Place the Miniprep column back into the 2 ml Microfuge tube. Centrifuge at 12,000×g for 1 minute.</P><P class="PP">Transfer the Miniprep column into a clean 1.5 ml Microfuge tube (provided). To elute the purified plasmid DNA, add 60~80 µl of Eluent (or deionized water) to the center of the membrane. Let it stand for 1 min at room temperature. Centrifuge at 12,000×g for 1 minute.</P></DIV><SPAN class="key CuteButton SmallCB GreenCB" onclick="fn(this,5)">More</SPAN></LI><LI><DIV class="clickMore" id="protocolPart6"><H2 class="H2Head">General Heat-Shock Transformation</H2><P class="PP">Add DNA to 50 ul of cells on ice</P><P class="PP">Incubate on ice 30 min</P><P class="PP">Heat shock 42degC 90 s</P><P class="PP">Place samples back on ice for 5 minutes</P><P class="PP">Add 1 ml of LB broth</P><P class="PP">Incubate at 180 rpm, 37°C for 60 minutes, shaking</P><P class="PP">Plate out cells on LB agar with appropriate antibiotic, maximum 200 ul</P><P class="PP">Incubate at 37°C overnight.</P><H2 class="H2Head">Growing Overnight Cultures</H2><H3 class="H3Head">Materials:</H3><P class="PP">5ml of LB broth/5ml SOC medium</P><P class="PP">5μl of specific antibiotic</P><P class="PP">12ml of culture tube</P><P class="PP">tips
                        </P><H3 class="H3Head">Methods:</H3><P class="PP">Overnight cultures were prepared under sterile conditions</P><P class="PP">Add 5 ml of liquid LB media/SOC into 12 ml culture tubes</P><P class="PP">Add 5 μl of appropriate antibiotic into the broth</P><P class="PP">Using the tip, pick a single colony and inoculate the cultures by putting the tip into the LB broth/SOC</P><P class="PP">Seal the tubes and incubate overnight at 37°C shaking at 180 rpm</P></DIV><SPAN class="key CuteButton SmallCB GreenCB" onclick="fn(this,6)">More</SPAN></LI><LI><DIV class="clickMore" id="protocolPart7"><H2 class="H2Head">Colony PCR</H2><H3 class="H3Head">Materials:</H3><P class="PP">Sterile water</P><P class="PP">E.coli colony</P><P class="PP">2 ul of forward primer</P><P class="PP">2 ul of reverse primer</P><P class="PP">2 × Taq Master Mix</P><H3 class="H3Head">Methods:</H3><P class="PP">Add a single colony of E.coli to 10 ul of soc.</P><TABLE class="table tableNice"><TBODY><TR><TD>ddH2O</TD><TD>7 ul</TD></TR><TR><TD>2 × Taq Master Mix</TD><TD>10 ul</TD></TR><TR><TH>Primer 1</TH><TD>1 ul</TD></TR><TR><TH>Primer 2</TH><TD>1 ul</TD></TR><TR><TH>1 ul</TH></TR></TBODY></TABLE><P class="PP">Set up the reaction following the instruction in the table above.</P><TABLE class="table tableNice"><TBODY><TR><TH>Segment</TH><TH>Number of cycles</TH><TH>Temperature(℃)</TH><TH>Duration</TH></TR><TR><TH>Initial denaturation</TH><TD>1</TD><TD>94</TD><TD>5min</TD></TR><TR><TH>PCR</TH><TD>94</TD><TD>30 sec</TD></TR><TR><TD>35 cycles</TD><TD>55</TD><TD>30 sec</TD></TR><TR><TD>72</TD><TD>60 sec/kb</TD></TR><TR><TH>Final extension</TH><TD>1</TD><TD>72</TD><TD>7 min</TD></TR><TR><TH>Hold</TH><TD>1</TD><TD>4</TD><TD>∞</TD></TR></TBODY></TABLE></DIV><SPAN class="key CuteButton SmallCB GreenCB" onclick="fn(this,7)">More</SPAN></LI><LI><DIV class="clickMore" id="protocolPart8"><H2 class="H2Head">LB Broth &amp; LB Ager</H2><H3 class="H3Head">LB Broth</H3><P class="PP"><STRONG>Materials:</STRONG></P><P class="PP">10g tryptone</P><P class="PP">5g yeast extract</P><P class="PP">10g NaCl</P><P class="PP">1 litre of purified water</P><P class="PP"><STRONG>Methods:</STRONG></P><P class="PP">Add 10g tryptone, 5g yeast extract and 10g NaCl to 1 litre purified water</P><P class="PP">Autoclave(120℃, 20min)
                          </P><H3 class="H3Head">LB Ager</H3><P class="PP"><STRONG>Materials:</STRONG></P><P class="PP">1 litre of LB broth</P><P class="PP">15-20g ager</P><P class="PP"><STRONG>Methods:</STRONG></P><P class="PP">Add 15-20g ager to 1 litre purified water</P><P class="PP">Autoclave(120℃, 20min)</P></DIV><SPAN class="key CuteButton SmallCB GreenCB" onclick="fn(this,8)">More</SPAN></LI><LI><DIV class="clickMore" id="protocolPart9"><H2 class="H2Head">Making Agar Plates</H2><H3 class="H3Head">Materials:</H3><P class="PP">LB ager</P><P class="PP">Specific antibiotics</P><P class="PP">plates</P><H3 class="H3Head">Methods:</H3><P class="PP">Once agar is molten, mix carefully. Add specific antibiotics and any other desired additives.</P><P class="PP">Pour plates carefully.</P><P class="PP">Allow to set for approximately 10 min.</P><P class="PP">Invert plates and allow to dry for 20-30 min.</P><P class="PP">Store plates at 4°C</P></DIV><SPAN class="key CuteButton SmallCB GreenCB" onclick="fn(this,9)">More</SPAN></LI><LI><DIV class="clickMore" id="protocolPart10"><H2 class="H2Head">SOC Medium</H2><H3 class="H3Head">Materials:</H3><P class="PP">5g yeast extract</P><P class="PP">20g tryptone</P><P class="PP">0.584g NaCl</P><P class="PP">0.186g KCl</P><P class="PP">2.4g MgSO4</P><P class="PP">4g glucose</P><P class="PP">1Litre of purified water</P><H3 class="H3Head">Methods:</H3><P class="PP">Add 5g yeast extract, 20g tryptone, 0.584g NaCl, 0.186g KCl, 2.4g MgSO4 and 4g glucose to 1 liture of purified water</P><P class="PP">Autoclave(110℃, 30min)</P></DIV><SPAN class="key CuteButton SmallCB GreenCB" onclick="fn(this,10)">More</SPAN></LI><LI><DIV class="clickMore" id="protocolPart11"><H2 class="H2Head">1% Agarose Gel</H2><H3 class="H3Head">Materials:</H3><P class="PP">0.25g agarose</P><P class="PP">25 ml of 1 X TAE buffer</P><P class="PP">2.5 uL of Gelred</P><H3 class="H3Head">Methods:</H3><P class="PP">Mix Agarose and 1x TAE buffer</P><P class="PP">Heat up until Agarose is dissolved</P><P class="PP">Add Gelred</P><P class="PP">Pour into gel tray and let cool</P></DIV><SPAN class="key CuteButton SmallCB GreenCB" onclick="fn(this,11)">More</SPAN></LI><LI><DIV class="clickMore" id="protocolPart12"><H2 class="H2Head">Agarose Gel Electrophoresis</H2><H3 class="H3Head">Materials:</H3><P class="PP">1% Agarose gel</P><P class="PP">DNA Ladder</P><P class="PP">10x DNA loading buffer</P><P class="PP">Electrophoresis cuvette</P><H3 class="H3Head">Methods:</H3><P class="PP">Set gel tray into cuvette, filled with 1x TAE buffer</P><P class="PP">Inoculate samples, previously dyed with 10x loading buffer.</P><P class="PP">Additionally, provided a DNA ladder for further reference of DNA sizes</P><P class="PP">Run gel at 90-120V for 20-40min</P><P class="PP">Image under UV light</P></DIV><SPAN class="key CuteButton SmallCB GreenCB" onclick="fn(this,12)">More</SPAN></LI><LI><DIV class="clickMore" id="protocolPart13"><H2 class="H2Head">1 X TAE buffer</H2><P class="PP">1x solution contains 40nM Tris, 20mM acetic acid, 1mM EDTA</P><H2 class="H2Head">60% Glycerol Preparation</H2><P class="PP">Add 60ml glycerol to 20ml sterile water</P><P class="PP">Autoclave(120 degrees, 20min)</P><H2 class="H2Head">Glycerol Stock Preparation</H2><P class="PP">Cultures plated on LB Agar + specific antibiotic and grown at 37°C overnight.</P><P class="PP">A 5ml LB culture in LB+ specific antibiotic inoculated from a single, freshly growing colony.</P><P class="PP">Cultivate for 16h at 37°C, with constant shaking</P><P class="PP">1 ml of this culture inoculated into sterile vial</P><P class="PP">Add 1ml of 60% glycerol</P><P class="PP">Mix.</P><P class="PP">Freeze them at -80 degrees</P></DIV><SPAN class="key CuteButton SmallCB GreenCB" onclick="fn(this,13)">More</SPAN></LI><LI><DIV class="clickMore" id="protocolPart14"><H2 class="H2Head">fluorescence testing</H2><P class="PP"><STRONG>Materials:</STRONG></P><P class="PP">1 X YPAD</P><P class="PP">ddH2O</P><P class="PP">96-well plate</P><P class="PP">Plate reader</P><P class="PP"><STRONG>Methods:</STRONG></P><P class="PP">Culture the yeast in 25ml 1xYPAD medium，220rpm，for 24h</P><P class="PP">Harvest the culture and add 1ml into a 1.5ml EP tube</P><P class="PP">Centrifuge for 2 min, 6000 rpm</P><P class="PP">Collect the yeast and wash it with ddH2O twice</P><P class="PP">Suspend the yeast in 1ml ddH2O，100𝜇l per well, add into the 96-well plate</P><P class="PP">Detect the fluorescence(mRFP） with plate reader, excitation length 585nm，emission length 605nm</P></DIV><SPAN class="key CuteButton SmallCB GreenCB" onclick="fn(this,14)">More</SPAN></LI><LI><DIV class="clickMore" id="protocolPart15"><H2 class="H2Head">Extraction of DAPG from E.coli and Detection of DAPG by HPLC</H2><P class="PP"><STRONG>Materials:</STRONG></P><P class="PP">DAPG</P><P class="PP">Methanol</P><P class="PP">ethyl acetate</P><P class="PP">LB broth</P><P class="PP">1 mol / L HCl</P><P class="PP"><STRONG>Methods:</STRONG></P><P class="PP">Preparation of standard samples. Take 0.02 g of DAPG and dissolve it with 5 ml of methanol as a stock solution. Diluting the stock solution 50 times with methanol, and taking 1 mL diluent(at a concentration of 8 * 10-5 g / ml), which was used as a measuring standard sample, into a tube. Two standard samples (labeled as STD1.STD2) were prepared for the determination of retention time. Diluting the stock solution 20 times with methanol, and taking 1 mL diluent(at a concentration of 8 * 10-5 g / ml), which was used as a standard add sample, into a tube.</P><P class="PP">Pretreatment. Using the tip, pick a single colony and inoculate the cultures by putting the tip into the LB broth. Incubate at 180 rpm, 37°C for overnigt, shaking. Centrifuge at 5000 rpm for 10 minutes.</P><P class="PP">Extraction. The supernatant was acidified with 1 mol / L HCl (pH 2.0). Extracted with 1.5 volumes of ethyl acetate.</P><P class="PP">Detection of DAPG by HPLC.After removing the solvent by reduced pressure distillation, we used methanol to dissolve DAPG, which was residue in the flask. Pipette it into tube for HPLC.</P></DIV><SPAN class="key CuteButton SmallCB GreenCB" onclick="fn(this,15)">More</SPAN></LI><H1 id="Trichodermaatrovirideprotocol" class="page-header ArticleHead YellowAH">Trichoderma atroviride protocol</H1><LI><DIV class="clickMore" id="protocolPart16"><H2 class="H2Head">Potato-dextrose agar media and culturing</H2><H3 class="H3Head">Materials:</H3><P class="PP">1 Litre purified water</P><P class="PP">200g potatoes(sliced washed unpeeled)</P><P class="PP">20g glucose</P><P class="PP">15-20g ager</P><H3 class="H3Head">Methods:</H3><P class="PP">Potato infusion can be made by boiling 200 grams of sliced potatoes in ~ 1 litre purified water for 30 minutes and then decanting or straining the broth through cheesecloth. 20 grams dextrose and 20 grams agar powder is then added.</P><P class="PP">Autoclave(110℃, 30min)</P><P class="PP">Streak/inoculate T.atroviride onto plates.</P><H2 class="H2Head">Induction Broth &amp; Induction Ager</H2><H3 class="H3Head">Materials and Methods:</H3><TABLE class="table tableNice"><TBODY><TR><TH>Compound</TH><TH>Liquid medium</TH><TH>Solid medium</TH></TR><TR><TD>1.25 K-Phosphate-buffer pH 4.8</TD><TD>8ml</TD><TD>8ml</TD></TR><TR><TD>1 M/L MES(pH5.5)</TD><TD>40ml</TD><TD>40ml</TD></TR><TR><TD>MN-buffer
                                              (30g/l MgSO4·7H2O, 15g/l NaCl)
                                          </TD><TD>20ml</TD><TD>20ml</TD></TR><TR><TD>1% CaCl·2H2O(w/v)</TD><TD>1ml</TD><TD>1ml</TD></TR><TR><TD>0.01%FeSO4(w/v)</TD><TD>10ml</TD><TD>10ml</TD></TR><TR><TD>20% NH4NO3(w/v)</TD><TD>2.5ml</TD><TD>2.5ml</TD></TR><TR><TD>50% glycerol(v/v)</TD><TD>10ml</TD><TD>10ml</TD></TR><TR><TD>20% glucose(w/v)</TD><TD>10ml</TD><TD>5ml</TD></TR><TR><TD>Ager</TD><TD>15g</TD></TR></TBODY></TABLE><P class="PP">Autoclave(110℃, 30min)</P><P class="PP">Spore elements(100 mg/l ZnSO4·7H2O, 100mg/l CuSO4·5H2O,100 mg/l H3BO3, 100 mg/l Na2MoO4;2H2O), filter sterilization.</P><P class="PP">Add 5000ul spore, 1000ul Kanamycin and 1000ul AS into it.</P></DIV><SPAN class="key CuteButton SmallCB GreenCB" onclick="fn(this,16)">More</SPAN></LI><LI><DIV class="clickMore" id="protocolPart17"><H2 class="H2Head">Agrobacterium Tumefaciens-mediated Transformation (ATMT) of Trichoderma Atroviride</H2><H3 class="H3Head">Materials:</H3><P class="PP">Kanamycin</P><P class="PP">LB agar</P><P class="PP">LB media/SOC</P><P class="PP">Agrobacterium</P><P class="PP">Plasmids</P><P class="PP">Liquid nitrogen</P><P class="PP">200 µg/ml hygromycin B</P><P class="PP">SS (Streptomycin sulfate)</P><P class="PP">Glufosinate</P><P class="PP">Nitrocellulose membrane</P><P class="PP">Induction medium</P><P class="PP">AIM</P><P class="PP">PDA plates</P><H3 class="H3Head">Methods:</H3><P class="PP">For plasmid constructions, E.coli was used and grown on LB agar supplemented with kanamycin (50 µg ml−1).</P><P class="PP">Mini-prepping.</P><P class="PP">Add 5 ul plasmids to 50μl competent cells(agrobacterium) on ice. Mixed.</P><P class="PP">Place the tube in a mortar and pour liquid nitrogen into it. When liquid nitrogen is completely gasified, putting the tube in 37degree, 5min.</P><P class="PP">Add 1 ml LB broth. Incubate at 28°C, 90 rpm for 1.5-2 hours, shaking.</P><P class="PP">Centrifuge at 7500 rpm for 1 minute.</P><P class="PP">Plate out cells on LB agar(with kanamycin).</P><P class="PP">Incubate at 28°C, 1-2 days.</P><P class="PP">Using colony PCR to screen positive clones. Picking single positive colones to 5 ml liquid LB media/SOC and adding 5 μl of appropriate antibiotic into the broth.Incubate at 28°C, 90 rpm for overnight, shaking.</P><P class="PP">Take 400 ul of the culture to 4ml AIM and the mix.</P><P class="PP">Incubate at 28°C, 200 rpm for 5 hours, shaking.</P><P class="PP">Take 100ul of the mixture, and mixe it with 100 µl of a conidial suspension (106, 107, or 108 ml−1) from T. atroviride.</P><P class="PP">The nitrocellulose membrane was cut into strips and placed on the induction medium.</P><P class="PP">The mixture was plated onto sterile NC film placed on induction medium and then incubated at 22 °C, in the dark.</P><P class="PP">After growth on induction medium 2 days, the strips were transferred to selection medium (PDA plates supplemented with 200 µg/ml hygromycin B (HygB), 1ul/ml SS and 1 ul/ml Glufosinate)</P><P class="PP">Colonies appearing after incubation at 28 °C were transferred to PDA with 200 µg HygB ml−1 and afterwards purified to mitotic stability by two rounds of single-spore isolation.</P></DIV><SPAN class="key CuteButton SmallCB GreenCB" onclick="fn(this,17)">More</SPAN></LI><LI><DIV class="clickMore" id="protocolPart18"><H2 class="H2Head">gDNA Library preparation for genome sequencing</H2><H3 class="H3Head">materials：</H3><P class="PP">Quartz sand</P><P class="PP">Trichoderma atroviride</P><P class="PP">ddH2O</P><P class="PP">CTAB</P><P class="PP">CIA</P><P class="PP">Isopropyl alcohol</P><H3 class="H3Head">methods：</H3><P class="PP">Add quartz sand and 500 ul ddH20 to a conidial suspension from T. atroviride.</P><P class="PP">Grind with a grinder for 2 minutes(65 HZ)</P><P class="PP">Add 500 ul CTAB. Incubate at 65 ℃, 30 minutes and mix it every 10 minutes.</P><P class="PP">Add 500 ul CIA. Shaking on the shaker for 15-20 minutes.</P><P class="PP">Centrifuge at 10000 rpm for 5-10 minutes to pellet it. Transfer the clarified supernatant into the 2 ml column.</P><P class="PP">Add 750 ul isopropyl alcohol to the supernatant. Stand still for 5 minutes.</P><P class="PP">Centrifuge at 10,000 rpm, 10 minutes. Discard the supernatant. And then</P><P class="PP">Upside down the tube on absorbent paper, about 1 hours.</P><P class="PP">Add 50 ul ddH20 into the tube for dissolving. Stored at 4cdegree.</P></DIV><SPAN class="key CuteButton SmallCB GreenCB" onclick="fn(this,18)">More</SPAN></LI><H1 id="Phytophthoraprotocol" class="ArticleHead YellowAH">Phytophthora protocol</H1><LI><DIV class="clickMore" id="protocolPart19"><H2 class="H2Head">Phytophthora culture medium</H2><H3 class="H3Head">Materials:</H3><P class="PP">PDA media</P><P class="PP">Ampicillin</P><P class="PP">Rifampicin</P><P class="PP">ager</P><H3 class="H3Head">Methods:</H3><P class="PP">Add Rifampicin(20ng/µ), ampicillin(200ng/µ) and ager(15-20g/L)to PDA madia</P><H2 class="H2Head">Confrontation between Trichoderma and Phytophthora</H2><H3 class="H3Head">Materials:</H3><P class="PP">PDA ager media</P><H3 class="H3Head">Methods:</H3><P class="PP">Trichoderma and Phytophthora infestans were respectively connected to the edge of the PDA.</P><P class="PP"> Incubated at 25 ° C, and the growth of the colony was recorded by continuous observation.</P></DIV><SPAN class="key CuteButton SmallCB GreenCB" onclick="fn(this,19)">More</SPAN></LI><H1 id="Yeastprotocol" class="ArticleHead YellowAH">Yeast protocol</H1><LI><DIV class="clickMore" id="protocolPart20"><H2 class="H2Head">PEG3350</H2><H3 class="H3Head">Materials:</H3><P class="PP">50 g PEG3350</P><P class="PP">ddH2O</P><H3 class="H3Head">Methods:</H3><P class="PP">50g polyethylene glycol 3350, adding double distilled water to 100mL.</P><P class="PP">121 ℃ sterilization 15min</P><P class="PP">Store in 4 degree</P><H2 class="H2Head">Lithium acetate</H2><H3 class="H3Head">Materials:</H3><P class="PP">5.1g Lithium acetate</P><P class="PP">ddH2O</P><P class="PP">Acetic acid</P><H3 class="H3Head">Methods:</H3><P class="PP">5.1g Lithium acetate, adding ddH2O to 50ml</P><P class="PP">Use phytic acid and sodium hydroxide to adjust pH to 7.5</P><P class="PP">121 ℃ sterilization 15min</P><P class="PP">Store in 4 degree</P><H2 class="H2Head">1 X YPAD</H2><H3 class="H3Head">Materials:</H3><P class="PP">20g glucose</P><P class="PP">20g tryptone</P><P class="PP">10g yeast extract</P><P class="PP">80mg Adenine Sulfate</P><P class="PP">1Litre purified water</P><H3 class="H3Head">Methods:</H3><P class="PP">Add 20g glucose, 20g tryptone, 10g yeast extract and 80mg Adenine Sulfat to 1liture purified water</P><P class="PP">Autoclave(110℃, 30min)</P><H2 class="H2Head">2 X YPAD</H2><H3 class="H3Head">Materials:</H3><P class="PP">40g glucose</P><P class="PP">40g tryptone</P><P class="PP">20g yeast extract</P><P class="PP">80mg Adenine Sulfate</P><P class="PP">1Litre purified water</P><H3 class="H3Head">Methods:</H3><P class="PP">Add 40g glucose, 40g tryptone, 20g yeast extract and 80mg Adenine Sulfate to 1liture purified water</P><P class="PP">Autoclave(110℃, 30min)</P><H2 class="H2Head">YPD Ager</H2><H3 class="H3Head">Materials:</H3><P class="PP">20g glucose</P><P class="PP">20g tryptone</P><P class="PP">10g yeast extract</P><P class="PP">15g ager</P><P class="PP">1Litre purified water</P><H3 class="H3Head">Methods:</H3><P class="PP">Add 20g glucose, 20g tryptone, 10g yeast extract and 15g ager to 1liture purified water</P><P class="PP">Autoclave(110℃, 30min)</P><H2 class="H2Head">YNB</H2><H3 class="H3Head">Materials:</H3><P class="PP">6.7g YNB</P><P class="PP">20g glucose</P><P class="PP">0.6mg Leucine</P><P class="PP">0.2mg Histidine</P><P class="PP">0.2mg Methionine</P><H3 class="H3Head">Methods:</H3><P class="PP">Add 6.7g YNB, 20g glucose, 0.6mg Leucine, 0.2mg Histidine and 0.2mg Methionine to 1 liture ddH2O</P><P class="PP">Autoclave(110℃, 30min)</P><P class="PP">Save at room temperature</P></DIV><SPAN class="key CuteButton SmallCB GreenCB" onclick="fn(this,20)">More</SPAN></LI><LI><DIV class="clickMore" id="protocolPart21"><H2 class="H2Head">Transformation of yeast</H2><H3 class="H3Head">Materials:</H3><P class="PP">1 X YPAD</P><P class="PP">2 X YPAD</P><P class="PP">ddH2O</P><P class="PP">PEG3350</P><P class="PP">1 mol/l LiAc</P><P class="PP">Single chain salmon sperm DNA</P><P class="PP">YPD</P><P class="PP">YNB</P><P class="PP">yeast plasmid DNA</P><H3 class="H3Head">Methods:</H3><P class="PP">For plasmid constructions, E.coli was used and grown on LB agar supplemented with appropriate antibiotic</P><P class="PP">Mini-prepping.</P><P class="PP">Add a single colony of yeast(BY4741) to 25 ml 1 X YPAD.Incubate at 30°C, 200 rpm for overnigt, shaking.</P><P class="PP">Take 1ml of the culture medium to 25ml 2 X YPAD. Incubate at 30°C, 200 rpm for 3-4 hours, shaking.</P><P class="PP">Collect 1 ml of the culture. Centrifuge at 600×g for 2 minute to pellet the yeast.</P><P class="PP">Wash the yeast pellet with cold ddH2O twice.</P><P class="PP">Add 240 ul PEG3350, 36 ul of 1 mol/l LiAc, 50ul single chain salmon sperm DNA and 25ul of yeast plasmid DNA in turn into tube and then mix.</P><P class="PP">Incubate at 30°C for 3 hours.</P><P class="PP">Heat shock 42degC 40 min.</P><P class="PP">Centrifuge at 12000×g for 1 minute to pellet the yeast.</P><P class="PP">Resuspend the yeast pellet in 1 ml of YPD broth. Incubate at 30°C, 120 rpm for 2-3 hours.</P><P class="PP">Plate out 100 ul of cells on YNB agar with appropriate antibiotic. Incubate at 37°C 2-3 days.</P></DIV><SPAN class="key CuteButton SmallCB GreenCB" onclick="fn(this,21)">More</SPAN></LI><LI><DIV class="clickMore" id="protocolPart22"><H2 class="H2Head">Yeast Total Protein Extraction</H2><H3 class="H3Head">Materials:</H3><P class="PP">Isotonic solution</P><P class="PP">snail enzyme</P><P class="PP">Mercaptoethanol</P><P class="PP">PMSF</P><H3 class="H3Head">Methods:</H3><P class="PP">Add a single colony of yeast(BY4741) to 25 ml 1 X YPAD.Incubate at 30°C, 200 rpm until OD600=1.0, shaking.</P><P class="PP">Centrifuge at 5000×g for 1 minute to pellet the yeast. Weigh the wet weight of the cells.</P><P class="PP">Add 500 μl of isotonic solution, 5ul snail enzyme and 1 ul mercaptoethanol to 50 mg wet weight. Resuspend the yeast pellet by vortexing.</P><P class="PP">Incubate at 30°C for 1 hours. Centrifuge at 2500×g for 1 minute. Discard the supernatant.</P><P class="PP">Resuspend the yeast pellet in 500 μl of isotonic solution by vortexing and wash the yeast pellet with ddH2O once.</P><P class="PP">Resuspend the yeast pellet in 500 ul of low penetration fluid (before using it, add 5 ul PMSF to 500 ul low penetration fluid).</P><P class="PP">Put it into -20 ℃ refrigerator frezzing 30 min and then dissolve at room temperature.</P><P class="PP">Do it again.</P><P class="PP">Centrifuge at 12000 rpm for 5 minute and keep in the supernation.</P></DIV><SPAN class="key CuteButton SmallCB GreenCB" onclick="fn(this,22)">More</SPAN></LI><LI><DIV class="clickMore" id="protocolPart23"><H2 class="H2Head">Calcium ion detection</H2><H3 class="H3Head">Materials:</H3><P class="PP">5μM Fluo-4-acetoxymethyl ester solution (solute in sterile water)</P><P class="PP">1 x PBS (pH 7.4)</P><P class="PP">5μL 20%(w/v) Pluronic F-127 solution (solute in DMSO)</P><P class="PP">Sterile water</P><P class="PP">1/2 SD/Gal-Raf/-Ura growth medium</P><P class="PP">Loops</P><P class="PP">Fluorescence plate reader</P><P class="PP">96-well plate</P><H3 class="H3Head">Methods:</H3><P class="PP">Culture yeast in 1/2 SD/Gal-Raf/-Ura growth medium at 30℃ for about 6 hours to reach a density of about 1×107-1.5×107cells per mL.</P><P class="PP">Harvest the cells and wash with PBS(pH7.4) three times.</P><P class="PP">Resuspend the cells with 2mL 5μM Fluo-4-acetoxymethyl ester solution, add 5μL 20%(w/v) Pluronic F-127 solution, shake gently to mix it up and then incubate in dark at 30℃ (note that all following steps should be done in dark).</P><P class="PP">Wash the cells with PBS three times.</P><P class="PP">Resuspend cells in sterile water to reach a density of about 1×107-1.5×107cells per mL.</P><P class="PP">Add 200μL cell suspensions in each well of the 96-well plates</P><P class="PP">Measure fluorescence with plate reader, the excitation wavelength is 488nm and the emission wavelength is between 512-520nm.</P><P class="PP">Measure OD600 with fluorescence plate reader.</P><P class="PP">Divide the fluorescence intensity with OD600 to get a relative fluorescence intensity</P></DIV><SPAN class="key CuteButton SmallCB GreenCB" onclick="fn(this,23)">More</SPAN></LI><LI><DIV class="clickMore" id="protocolPart24"><H2 class="H2Head">Western-blot and check flag-tag</H2><H3 class="H3Head">Sample preparation for protein gel.</H3><P class="PP">Yeast total protein extraction. Load 15-20 ul of supematant per lane on a protein gel. Load 10 ul of prestained protein ladder. Run protein gel at 200V until dye front reaches the bottom of the gel.</P><H3 class="H3Head">Western Blot：</H3><P class="PP">Run gel as usual. Take gel out of electrophoresis apparatus. Cut into segments as required; Part of gel can be stained directly in Coomassie brilliant blue R-250. Part to be used for electroblotting is put into tap water on shaker, after first having marked it unambiguously to identify top/bottom, left and right etc.</P><P class="PP">Leave in water on shaker for 5 minutes. This step can be substituted by washing the gel in electro-transfer buffer (see below) for 5 minutes.</P><P class="PP">We use a semidry blotter, which we have found to be quicker, more economical and easier than fully submerged blotting methods. We cut Whatman 3M filter papers to the size of our gels, and place three of these onto the semi dry blotter. These are then wet with transfer buffer (we routinely use 3.03 g Tris base, 14.4 g Glycine, 10% Methanol per liter). The gel is put onto the filters and a prewetted nitrocellulose filter is put ontop of the gel. Alternately put a PVDF membrane on top; if you are using PVDF remember it is essential to prewet the PVDF in 100% methanol. Great care should be taken to ensure that no air bubbles are anywhere in this stack of membranes. Then three more wetted Whatman 3M filters should be placed ontop of the pile, again taking great care not to have any bubbles in pile. Put the top onto the apparatus and screw it down. Proteins in transfer buffer are negative in charge mostly due to residual SDS and they therefore move from -ve to +ve pole. So the +ve electrode is above the nitrocellulose and the -ve side is below the gel.</P><P class="PP">Run for 30 minutes to 1 hour at ~100mA. The most reliable way of doing this is to use a powerful power supply 200-500mA and put it on constant voltage, with a setting of 5 to 10 Volts. Low molecular weight proteins (20kDa or less) will transer in 30 minutes at 5 Volts, while higher molecular weight (150kDa or more) transfer in 60 minutes at 10 Volts.</P><P class="PP">After running disassemble the apparatus and remove nitrocellulose filter. Stain this for 5 minutes on shaker in Ponceau reagent. Destain with regular SDS-PAGE gel destain solution.</P><P class="PP">After Ponceau staining put the nitrocellulose filter into blocking solution, such as 1% bovine serum albumin (BSA) or 1% Carnation non fat milk (NFM), for 20 minutes to 1 hr at RT or 37°C. Since the NFM works just as well as BSA but is much cheaper, there is really no good reason to use BSA. Ponceau staining will fade to become completely invisible. Carry on with antibody incubations etc.</P><H3 class="H3Head">Antibody Incubations</H3><P class="PP">Put in antibody solutions. Volume should be enough to cover blot and allow it to float freely when you agitate. In initial experiments, antibody concentration should generally be about 1:100 - 1:1,000 for ascites, CL350 tissue culture supernatant or antiserum, undiluted to 1:10 for monoclonal supernatant, and about 1-10µg/ml for a pure IgG. If dilution brings antibody concentration to less than 50 µgs/ml, add some BSA or NFM to act as carrier protein. Incubate for at least 1 hour with shaking (can be room temperature or at 37°C, can also do overnight at 4°C).</P><P class="PP">Wash membranes in TBS (10mM Tris, 154mM NaCl, pH=7.5 plus 0.1% Tween 20) for 3 times at least five minutes each time with extensive agitation.</P><P class="PP">Incubate in second antibody (peroxidase-conjugate, phosphatase conjugate or radioactive). Add BSA or NFM carrier as before if necessary. Incubate for at least one hour at room temperature or 37°C can also do overnight at 4°C with shaking as before.</P><P class="PP">Wash membranes in TBS for 3 times at least five minutes each time with extensive agitation.</P><H3 class="H3Head">Alkaline Phosphatase Blot System</H3><P class="PP">Incubate in alkaline phosphatase conjugated antibody against the primary antibody. Typical concentration is 1:1,000 in TBS. Add a small amount of BSA or NFM to act as carrier. Incubate for 1 hour at room temperature (or 37°C) with shaking.</P><P class="PP">Wash in TBS three times 5 minutes each.
                              </P><P class="PP">Put into developer. Buffer is 100mM Tris/HCl, 100mM NaCl, 5mM MgCl2 pH=9.5. To 10ml of this add 33µl BCIP-T and 33µl of NBT. Can store these solutions at -20°C. Can buy this solution made up already from Sigma. Reaction product is purple, and appears in a few minutes; can incubate for up to an hour if the signal is weak. Watch development of reaction and stop with water. Some of background disappears on drying.</P><H3 class="H3Head">Horse Radish Peroxidase Staining</H3><P class="PP">After washing of blots in TBS or PBS add reaction mixture. This is; 20 mls 0.1M Tris/HCl pH=7.2. 200 µl NiCl (80 mg/ml), 6 µl 30% hydrogen peroxide, 1ml of 5mgs/ml diaminobenzidine. Alternate protocol; Make 20 mls ammonium acetate buffer (50mM, pH=5.0). Add 1 ml of 10mg/ml Diaminobenzidine, 40µl 30% hydrogen peroxide. Brown reaction product is seen in 1-10 minutes, not quite so nice as above method.</P><H3 class="H3Head">Chemiluminescence Staining</H3><P class="PP">Chemiluminescence has an advantage of perhaps an order of magnitude greater sensitivity than the dye based methods above. In addition, several films may be exposed from a single blot, giving an advantage in interpretation of weak and strong signals on the same membrane. However it requires a darkroom to perform and is more expensive in reagents. Reagents are generally bought in a kit, and we recommend simply following the kit instructions.</P></DIV><SPAN class="key CuteButton SmallCB GreenCB" onclick="fn(this,24)">More</SPAN></LI><LI><DIV class="clickMore" id="protocolPart25"><H2 class="H2Head">Enzyme activity detection</H2><H3 class="H3Head">materials：</H3><P class="PP">1.0 mmol / L BAEE solution</P><P class="PP">enzyme extract</P><P class="PP">PMSF</P><H3 class="H3Head">method：</H3><P class="PP">Prepare three 1 cm cuvettes with lid and add 2.8 mL of 1.0 mmol / L BAEE solution, which has been preheated at 25 ° C, to the three cuvettes.</P><P class="PP">Add 0.2 mL of 10 mmol / L HCl, 0.2 mL of the enzyme extract, the mixture of 0.2ml enzyme extract and inhibitor(PMSF) to cuvettes respectively. The blank which adds HCl is used to set zero at a wavelength of 253 nm.</P><P class="PP">Immediately cover the lid quickly and mix. Read every half minutes and read a total of 3 ~ 4min.</P><P class="PP">Set the coordinate axis with the time (t) as the abscissa, the light absorption (A253nm) as the ordinate.</P></DIV><SPAN class="key CuteButton SmallCB GreenCB" onclick="fn(this,25)">More</SPAN></LI></UL></DIV><DIV class="col-md-3 disappear-on-top" role="complementary"><NAV style="position: fixed; top: 100px ; left:50px;" class="bs-docs-sidebar hidden-print hidden-xs hidden-sm"><UL class="nav bs-docs-sidenav shorterli"><LI><A href="#GeneralEcoliprotocol">General/E.coli protocol</A></LI><LI><A href="#Trichodermaatrovirideprotocol">Trichoderma atroviride protocol</A></LI><LI><A href="#Phytophthoraprotocol">Phytophthora protocol</A></LI><LI><A href="#Yeastprotocol">Yeast protocol</A></LI></UL></NAV></DIV></DIV></DIV><DIV class="footer" style="background-color: #000000"><DIV class="copyright container text-center"><P class="copyright">Copyright © 2017 ZJU-IGEM | <A href="http://www.zju.edu.cn/">Zhejiang University</A> |</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>