<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ZJU-China_Project_st skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ZJU-China/Project/st</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="header-w3layouts"><NAV class="navbar navbar-default navbar-fixed-top"><DIV class="collapse navbar-collapse navbar-ex1-collapse"><UL class="nav navbar-nav navbar-right cl-effect-15"><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">Overview</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Overview">Description</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Achievements">Achievements</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Improve">Improve Parts</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/InterLab">InterLab</A></LI></UL><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">Project</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Project/tp">Trichoderma Proof</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Project/voc">VOC sensors</A></LI><LI><A style="font-size: 0.7em!important;" href="https://2017.igem.org/Team:ZJU-China/Project/st">Chemical Signal Transduction</A></LI><LI><A style="font-size: 0.7em!important;" href="https://2017.igem.org/Team:ZJU-China/Project/mt">Medium Wave Transduction</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Project/Downstream">Downstream</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Project/conclusion">Conclusions</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Protocols">Protocols</A></LI><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">Model</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Model">VOC analysis</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Model/Coculture">Coculture</A></LI><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">Parts</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Parts">All Parts</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Part_Collection">Part Collection</A></LI><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">Hardware</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Hardware">Overview</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Hardware/Device">Device</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Hardware/Improvements">Improvements</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Hardware/MediumWave">Medium Wave</A></LI><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">Safety</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Safety">Environment</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Safety/Lab">Laboratory</A></LI><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">HP</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Human_Practices">Summary</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/HP/Gold_Integrated">Gold Integrated</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Engagement">Engagement</A></LI><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">Team</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Team">Teammates</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Attributions">Attribution</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Collaborations">Collaboration</A></LI></DIV></NAV></DIV></DIV><P>&quot;</P><DIV style="width: 100%" class="container zjuContent"><DIV class="col-md-9" role="main"><H1 id="ct" class="page-header ArticleHead GreenAH">Chemical Transduction</H1><H2 class="H2Head" id="overview">Overview</H2><P class="PP">This year, we aim to construct a diseases-monitoring system for automatic agriculture in which hardware and organisms can act as a whole. Hence it requires us to make our device and our engineered organisms connected. We choose the DAPG-Off/On System to serve as the link between our device and engineered organisms for it is supersensitive. When our device detected that the plant is infected with pathogens and is in poor condition by analyzing the changes in volatile organic compounds (VOC) released by plants, it will tell our engineered <EM>T. atroviride</EM> through the DAPG-Off/On system, and <EM>T. atroviride</EM> will respond to the invasion of pathogens. Luckily, after further literature reading and brainstorming, we find this system will bring us more surprise.</P><H2 class="H2Head" id="introduction">Introduction——A New Type of Transcriptional Switch</H2><P class="PP">2,4-DAPG (DAPG), full name 2,4-diacetylphloroglucinol, is a natural phenol found in specific strains of Gram-negative bacterium Pseudomonas fluorescens<SUP>[1]</SUP>. Recently, it is demonstrated that 2,4-DAPG can serve as signal molecule to construct a transcriptional switch that is similar to Tet on/off system.</P><P class="PP capture">Fig. 1 2,4-diacetylphloroglucinol</P><P class="PP">According to a research conducted to study the biosynthesis pathway of DAPG, researchers found that there is a negative correlation between phlF gene expression level and the DAPG synthetic amount. Further study shows that phlF gene codes for a repressor protein which can interact with an inverted repeated sequence, phlO, located upstream of the gene related to biosynthesis of DAPG. PhlF protein bound its operator sequence (phlO) in the absence of 2,4-diacetylphloroglucinol (DAPG) but dissociated from phlO in the presence of DAPG<SUP>[2]</SUP>.</P><P class="PP">Therefore, we can use DAPG, phlO and phlF to construct a new type of transcriptional switch. As Fig.2 shown, while the concentration of 2,4-DAPG is below the threshold, the phlF protein binds to the phlO domain upstream the gene of interest(GOI), which serves as a high order complex and block the recruitment of RNA polymerase<SUP>[3]</SUP>, leading to the inhibition of GoI expression. DAPG remove the phlF binding to the phlO domain, resulting in the expression of GoI, at the time of its concentration exceeds specific value. What’s more, it is demonstrated that the transcriptional regulation is rapid<SUP>[4]</SUP>, so this set of system has strong potential to be applied to several signal transduction pathways thanks to its supersensitive feature.</P><P class="PP capture">Fig. 2 Characteristics of the DAPG transcriptional switch.</P><H2 class="H2Head" id="design">Design</H2><H3 class="H3Head">Interact with the device</H3><P class="PP">This year we aim to integrate both hardware and engineered organisms to provide a new blue print of automatic agricultural. We take advantage of DAPG-Off/On System to make our device interact with our engineered <EM>T. atroviride</EM>.</P><P class="PP">When our device detected that the plant is infected with pathogens and is in poor condition by analyzing the changes in volatile organic compounds (VOC) given off by plants, it will release a few DAPG molecules as signal to communicate with <EM>T. atroviride</EM>. The molecules diffused into our engineered <EM>T. atroviride</EM> nearby to remove the phlF protein  binding to the phlO domain so as to turn on the expression of corresponding gene that is responsible for reporting signals or inhibition of pathogens. What’ s worth to mention is that the DAPG our device releasing can be synthesized by the engineered <EM>T. atroviride</EM>, which to some extent reduces the cost. Last but not least, previous studies comfirmed that DAPG almost has no harmful impact on human health under the effective concentration, and can be degraded easily in nature. <SUP>[5]</SUP></P><H3 class="H3Head">Signal Amplification</H3><P class="PP">On account of the physical and chemical conditions in edatope being complex and varied, the signal intensity declines rapidly with the distance increasing, which restricts the scope to respond of our device. Despite the fact that we can extend response range by using more device in real world, it do increase the cost. In order to enhance the feasibility and practicability of our system in the real agricultural district , we use the DAPG-Off/On system to amplify the signal generated by our device.</P><P class="PP">We insert the gene related to the biosynthesis of DAPG downstream the promoter or operon corresponding to responding to signals in part of our engineered <EM>T. atroviride</EM>. When this part of engineered <EM>T. atroviride</EM> sense the signal, they will synthesis and released more DAPG molecules to induce more engineered <EM>T. atroviride</EM> to respond to the invasion of pathogens. In this way, we will amplify the signal and expand the scope to repond of our device.</P><H3 class="H3Head">Enhance Plant's Resistance to Disease</H3><P class="PP">Plants possess a range of active defense apparatuses that can be actively expressed in response to biotic stresses. Induced resistance is a state of enhanced defensive capacity developed by a plant when appropriately stimulated. Systemic acquired resistance (SAR) and induced systemic resistance (ISR) are two forms of induced resistance wherein plant defenses are preconditioned by prior infection, some chemical substances or treatment that results in resistance against subsequent challenge by pathogens. <SUP>[6]</SUP></P><P class="PP">DAPG is exactly one of induction factors of ISR<SUP>[7]</SUP>. The plants' autoimmune mechanism can be activated under the presence of the DAPG, which will lead the plants to their prime state. Thus, the plants will have stronger resistance to the following diseases and invasion.</P><H2 class="H2Head" id="result">Result</H2><H3 id="biosynthesis" class="H3Head">Biosynthesis of 2,4-Diacetylphloroglucinol (2,4-DAPG)</H3><P class="PP capture">Fig.3 2,4-DAPG synthesis pathway</P><P class="PP">As has mentioned before, DAPG is initially found in specific strain of Pseudomonas fluorescens. And it has been demonstrated that a cluster named phl responsible for the biosynthesis of DAPG, which contains eight genes, from phlA to phlH. Further studies shows that phlABCD are the DAPG synthesis genes<SUP>[8][9][10]</SUP>.</P><P class="PP">We successfully cloned the phlABCD via colony PCR from Pseudomonas fluorescens 2P24. We designed three steps to prove that DAPG can be synthesised in <EM>T. atroviride</EM>, with the aim of confirming DAPG can be biosynthesized heterologously, in eukaryotic cells and in <EM>T. atroviride</EM> respectively.</P><H3 id="first" class="H3Head">The first step: Biosynthesis the DAPG in <EM>E.coli</EM></H3><P class="PP">We inserted four genes, phlABCD, into plasmid pSB1CE, which has a chloramphenicol resistance, a T7 promoter and RBS. Then we transformed the plasmid containing the phlABCD gene cluster into BL21 <EM>E.coli</EM> cells. Successful transformants were verified via colony PCR.</P><P class="PP">For detection of DAPG, We utilized the High-performance liquid chromatography (HPLC) method. Given a lot of background noise of the samples, we improve the chromatographic condition got from literature. Luckily, we finally detected the DAPG from <EM>E.coli</EM> cells after induction with IPTG.</P><P class="PP">In this way, we confirmed that phlABCD can be expressed heterologously.</P><P class="PP capture">Fig.4 Results of DAPG detecting by HPLC</P><H3 id="second" class="H3Head">The second step: Biosynthesis the DAPG in <EM>S. cerevisiae</EM></H3><P class="PP">With the M2S integration technique<SUP>[11]</SUP>, whose mechanism is similar to that of <A class="cite" target="_blank" href="https://2017.igem.org/Team:ZJU-China/Project/tp">Trichoderma Homologous Recombination Kit</A> we providing, two dual transcription units, homologous arm-URA3 marker-L1-tRPS2-phlA-TEF2-PGK1-phlB-tTDH1-L2(<A class="cite" target="_blank" href="http://parts.igem.org/Part:BBa_K2207016">BBa_2207016</A>) and L2-tCCW12-phlC-TEF2-PGK1-phlD-tRPL9A-L3-homologous arm(<A class="cite" target="_blank" href="http://parts.igem.org/Part:BBa_K2207017">BBa_ 2207017</A>), were constructed as illustrated in Fig.5. We cotransformed this two plasmids into S. cerevisiae so that this two fragments can be integrated into the S. cerevisiae genome.However, we have no time to detect whether DAPG is synthesized due to the limitation of time.</P><P class="PP capture">Fig.5  Structure of two dual transcription units</P><H3 id="third" class="H3Head">The third step: Biosynthesis the DAPG in <EM>T. atroviride</EM></H3><P class="PP">Considering the fact we have no time to prove whether DAPG can be biosynthesized in eukaryotic cells due to the limitation of time, so we choose to prove that phlABCD can be expressed in T. atirivide first.</P><P class="PP">We used the F2A sequenece(gtgaaacagactttgaattttgaccttctcaagttggcgggaga cgtggagtccaaccctggacct)<SUP>[12]</SUP>to link the phlA with phlC, and then combined them to eGFP to construct a fusion protein. The same is true for phlB and phlD. F2A sequence codes for F2A peptide which allows multiple independent genes to be transcribed as a single mRNA. Upon translation, the 2A peptide sequence causes a &quot;ribosomal skip&quot; generating two (or more) independent gene products. When the 2A peptide is translated, the &quot;skip&quot; occurs between its two C-terminal amino acids (glycine and proline), resulting in the addition of extra amino acids on the C terminus of the upstream protein and a single proline addition to the N terminus of the downstream protein<SUP>[13]</SUP>.</P><P class="PP capture">Fig.6 Mechanism of 2A sequence</P><P class="PP">In this way we can identify whether the proteins are expressed by detecting the fluorescent.</P><P class="PP">We successfully inserted those two plasmids into <EM>T. atroviride</EM> respectively, and used fluorescence microscope to detect the hyphae of our engineered <EM>T. atroviride</EM>. To our joy, we successfully saw the green fluorescence as Fig.7 shows, which means phlABCD all expressed in our engineered <EM>T. atroviride</EM>.</P><P class="PP capture">Fig. 7 Result of fluorescence detecting. (a)T. atroviride that is transformed in phlAC gene. (b)T. atroviride that is transformed in phlBD gene</P><H2 class="H2Head" id="tr">Transcriptional Regulation</H2><P class="PP">PhlF protein is a transcription factor, so it need to enter the nucleus to work. We added nuclear localization signal(NLS) sequence from SV40(CCAAAGAAGAAGAGAAAGGTA), which is commonly used in S. cerevisiae, to upstream of Pseudomonas phlF according to the literature. Then we cotransformed the plasmids containing phlF and phlO respectively to S. cerevisiae to check if it can work properly. For the convenience of western-blot detection, we add Flag tag sequence downstream of phlF.</P><P class="PP capture"> Fig.8 Yeast expression system</P><P class="PP">We successfully constructed the pathway as Fig.8 shows and inserted it to the vector pYES2.1. And we proved by gel electrophoresis (Fig.9). Then we transformed the plasmid to S. cerevisiae, and after cultivating several time, extracted protein from them to do western blot to detect whether phlF protein is synthesized successfully. To our joy, we find it is as Fig.9 shows.</P><P class="PP capture">Fig.9(a) Gel electrophoresis for phlF on pyes2.1 plasmid.S1:The fragments from prGPD to CYC1 Terminator.S2:The fragments of phlF. (b) Western blotting for phlF on pyes2.1 plasmid.
        </P><P class="PP">We improved the NLS sequence (CCCAAGAAGAAGAGGAAGGTG) afterwards to make phlF protein work well in filamentous fungi according to literature(Fig.11). Considering the fact that extracting the protein from <EM>T. atroviride</EM> is so complex, so we decided to fuse GFP to phlF protein so that we can detect our protein by detecting the fluorescent. Due to the limitation of time, the <EM>T. atroviride</EM> inserted gene of fusion protein hasn’t grown up, so we don't know whether phlF is expressed as expected.</P><P class="PP capture">Fig.10 Trichoderma expression system. T.reesei NLS means this NLS had been optimized to be appropriately used in Trichoderma reesei.</P><P class="PP">Phlo is the operator which can bind to repressor phlF. When phlF is binding to pho sequence, it will interrupt the binding of RNA polymerase and result in no expression outcome. However, when DAPG cross into the cell, this kind of signal molecular will occupy the DNA binding domain of phlF and prevent it from binding to the operator. Then the downstream starts expressing.</P><P class="PP">To construct such a promoter, two phlo segments are inbedded in the ADH1 promoter. It’s really kind of Aleksandra. Wudzinska and Jef D. Boeke who provided this promoter for us. We chose mRFP as the report gene to test this promoter.</P><P class="PP">From the Fig.11, you can clearly tell that our BY4741 transferred with phlo-mRFP does turn red (to be honest, none of us knows why the red fluorescent protein will turn to be orange in our strain and we are really curious about this question). When phlF binds to phlO, expression of gene downstream will be inhibited. </P><P class="PP capture">Fig.11(a) BY4741 transferred with plasmid backbone.   (b) BY4741 transferred with pho-mRFP</P><H2 class="H2Head" id="future">Future Work</H2><P class="PP">Research has shown that Trichoderma can recognize the pathogenic fungi by a special mechanism and utilize the hyphae to twine around them, which is called mycoparasitism. We plan to take advantage of this feature in the future. We insert the gene cluster which is responsible for the synthesis of 2,4-Diacetylphloroglucinol(2,4-DAPG) downstream the corresponding promoters. Thus, we can transduce and amplify the signal resulting from the invasion of pathogen by producing 2,4-DAPG, which strengthens Trichoderma’s ability to forecast and inhibit infection of tobacco. In this way, we can achieve the fact that <EM>T. atroviride</EM> can also interact with device, which means that they work more like a whole.</P><P class="PP">What's more, we also plan to find more VOC that can satisfying our needs more, and can be sensed by our device more sensitively and specially.</P><DIV style="text-align: center"><A class="CuteButton YellowCB" href="https://2017.igem.org/Team:ZJU-China/Project/mt">See Microwave Transduction...</A></DIV><H2 class="H2Head" id="ref">Reference</H2><P class="PP ref" id="mwt">[1] https://en.wikipedia.org/wiki/DAPG</P><P class="PP ref">[2] Yang F, Cao Y. Biosynthesis of phloroglucinol compounds in microorganisms—review[J]. Applied Microbiology and Biotechnology, 2012, 93(2):487-495.</P><P class="PP ref">[3] Abbas A, Morrissey J P, Marquez P C, et al. Characterization of interactions between the transcriptional repressor PhlF and its binding site at the phlA promoter in Pseudomonas fluorescens F113.[J]. Journal of Bacteriology, 2002, 184(11):3008.</P><P class="PP ref">[4] Ikushima S, Boeke J D. New Orthogonal Transcriptional Switches Derived from Tet Repressor Homologues for Saccharomyces cerevisiae Regulated by 2, 4-Diacetylphloroglucinol and Other Ligands[J]. CS Synthetic Biology, 2016.</P><P class="PP ref">[5] Gao Y, Chi FJ, eta. Effects of 2,4-diacetylphloroglucinol on two plant diseases[J]. Shandong Science, 2006, 19(4):36-39.</P><P class="PP ref">[6] Madamanchi N R, Kuć J. Induced Systemic Resistance in Plants[M]. The Fungal Spore and Disease Initiation in Plants and Animals. Springer US, 1991:347-362.</P><P class="PP ref">[7] Rezzonico F, Zala M, Keel C, et al. Is the ability of biocontrol fluorescent pseudomonads to produce the antifungal metabolite 2,4‐diacetylphloroglucinol really synonymous with higher plant protection?[J]. New Phytologist, 2007, 173(4):861-872.</P><P class="PP ref">[8] Yang F, Cao Y. Biosynthesis of phloroglucinol compounds in microorganisms—review[J]. Applied Microbiology and Biotechnology, 2012, 93(2):487-495.</P><P class="PP ref">[9] Bangera M G, Thomashow L S. Characterization of a genomic locus required for synthesis of the antibiotic 2,4-diacetylphloroglucinol by the biological control agent Pseudomonas fluorescens Q2-87[J]. Molecular plant-microbe interactions : MPMI, 1996, 9(2):83.</P><P class="PP ref">[10] Bangera M G, Thomashow L S. Identification and Characterization of a Gene Cluster for Synthesis, of the Polyketide Antibiotic 2,4-Diacetylphloroglucinol, from Pseudomonas fluorescens Q2-87[J]. Journal of Bacteriology, 1999, 181(10):3155.</P><P class="PP ref">[11] Li S, Ding W, Zhang X, et al. Development of a modularized two-step (M2S) chromosome integration technique for integration of multiple transcription units inSaccharomyces cerevisiae:[J]. J Immunoassay Immunochem, 2016, 9(1):232-243.</P><P class="PP ref">[12] Jin H K, Lee S R, Li L H, et al. High Cleavage Efficiency of a 2A Peptide Derived from Porcine Teschovirus-1 in Human Cell Lines, Zebrafish and Mice[J]. Plos One, 2011, 6(4):e18556.</P><P class="PP ref">[13] Subramanian V, Schuster L A, Moore K T, et al. A versatile 2A peptide-based bicistronic protein expressing platform for the industrial cellulase producing fungus, Trichoderma reesei[J]. Biotechnology for Biofuels, 2017, 10(1):34.</P></DIV><DIV class="col-md-3 disappear-on-top" role="complementary"><NAV style="position: fixed; top: 100px ; left:50px;" class="bs-docs-sidebar hidden-print hidden-xs hidden-sm"><UL class="nav bs-docs-sidenav shorterli"><LI><A class="" href="#ct">Chemical Transduction</A></LI><LI><A href="#overview">Overview</A></LI><LI><A href="#introduction">Introduction</A></LI><LI><A href="#result">Result</A></LI><LI><A href="#biosynthesis">Biosynthesis</A></LI><LI><A href="#first">The First Step</A></LI><LI><A href="#second">The Second Step</A></LI><LI><A href="#third">The Third Step</A></LI></UL><LI><A href="#tr">Transcriptional Regulation</A></LI><LI><A href="#future">Future Work</A></LI><LI><A href="#ref">Reference</A></LI></NAV></DIV></DIV></DIV><DIV class="footer" style="background-color: #000000"><DIV class="copyright container text-center"><P class="copyright">Copyright © 2017 ZJU-IGEM | <A href="http://www.zju.edu.cn/">Zhejiang University</A> |
        </P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>