<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ZJU-China_Basic_Part skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ZJU-China/Basic Part</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="header-w3layouts"><NAV class="navbar navbar-default navbar-fixed-top"><DIV class="collapse navbar-collapse navbar-ex1-collapse"><UL class="nav navbar-nav navbar-right cl-effect-15"><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">Overview</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Overview">Description</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Achievements">Achievements</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Improve">Improve Parts</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/InterLab">InterLab</A></LI></UL><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">Project</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Project/tp">Trichoderma Proof</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Project/voc">VOC sensors</A></LI><LI><A style="font-size: 0.7em!important;" href="https://2017.igem.org/Team:ZJU-China/Project/st">Chemical Signal Transduction</A></LI><LI><A style="font-size: 0.7em!important;" href="https://2017.igem.org/Team:ZJU-China/Project/mt">Medium Wave Transduction</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Project/Downstream">Downstream</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Project/conclusion">Conclusions</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Protocols">Protocols</A></LI><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">Model</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Model">VOC analysis</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Model/Coculture">Coculture</A></LI><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">Parts</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Parts">All Parts</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Part_Collection">Part Collection</A></LI><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">Hardware</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Hardware">Overview</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Hardware/Device">Device</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Hardware/Improvements">Improvements</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Hardware/MediumWave">Medium Wave</A></LI><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">Safety</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Safety">Environment</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Safety/Lab">Laboratory</A></LI><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">HP</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Human_Practices">Summary</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/HP/Gold_Integrated">Gold Integrated</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Engagement">Engagement</A></LI><LI class="m_nav_item dropdown"><A href="#" class="dropdown-toggle link" data-toggle="dropdown">Team</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Team">Teammates</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Attributions">Attribution</A></LI><LI><A href="https://2017.igem.org/Team:ZJU-China/Collaborations">Collaboration</A></LI></DIV></NAV></DIV></DIV><P>&quot;</P><DIV style="width: 100%" class="container zjuContent"><DIV class="col-md-9" role="main"><H1 id="bbp" class="page-header ArticleHead GreenAH">Best Basic Part</H1><P class="PP">For this year’s iGEM competition, our team has chosen to present Calcineurin-dependent response element (CDRE) (BBa_K2207021)and Mature serine protein from <EM>Paecilomyces lilacinus</EM> (BBa_K2207023) for the award of the basic part.</P><H2 class="H2Head" id="cdre">CDRE</H2><P class="PP">CDRE，calcineurin-dependent response element，is a segment of DNA sequence which can be regulated by specific transcription factor. This TF is activated by calcineurin in the present of calcium ion. To create a promoter can be up-regulated by calcium influx，we replaced the upstream activating sequence(UAS) of CYC1 promoter with 4 CDREs. This promoter is designed for Saccharomyces cerevisiae and it depends on S.cerevisiae's endogenous calcineurin and calmodulin<SUP>[1][2]</SUP>. We choose mRFP as the report gene. We cultured the yeast in calcium-inducing medium and uninduced medium which contains a relatively low concentration of calcium ion (1xYPD medium).</P><P class="PP">Obviously, the transgenic yeast cultured in the calcium-inducing medium(200mM Ca<SUP>2+</SUP>) is turning red while the control shows no significant changes. Then we detected the fluorescence of this two cultures. To eliminate the influence of the concentration of the yeast, we calculated the value of Fluorescent Intensity/OD600 to evaluate these two groups.</P><P class="PP">The value indicated that the fluorescence intensity of the calcium-induced group has improved about 147%. It’s not quite a huge change and the uninduced group has already shown a great intensity of fluorescence. Then we detected the intracellular calcium ion concentration with Fluo 4-AM whose fluorescent intensity can represent the relative concentration of calcium ion.</P><P class="capture">Fig.1 (a) Relative fluorescent intensity of two groups. (b) Relative calcium content in two groups</P><P class="PP"> As the result shows, 8% change of the intracellular ion concentration can contribute to 147% higher expression level of the downstream gene. It seems that our CDRE promoter is much more sensitive than we expected.</P><H2 class="H2Head" id="msp">Mature serine protein</H2><P class="PP">Root-knot nematodes (Meloidogyne spp.), which are one of the most destructive nematodes, cause the loss of crop about 10%, serious as high as 75%. Like the insect cuticlehe, nematode eggshell consists mainly of proteins and chitins. The egg-parasitic fungus <EM>P.lilacinum</EM> secretes protease and chitinase to hydrolyze the nematode eggshell, so that the root knot nematodes cannot grow normally<SUP>[3]</SUP>. Among them, serine protases plays an important role in hydrolyzing the eggshell of root-knot nematodes.</P><P class="PP">The gene which can express serine protease in yeast was synthesized by Genscript. Before synthesizing this gene, we did codon optimization based on the codon preference of yeast and added a flag-tag to the N-terminal of the serine protease.After extracting the whole proteins of the yeast which transferred plasmid successfully, we performed western-blot and checked the serine protein was expressed in the yeast. (Result is as follow) The band was very shallow, in other words, the concentration of the serine protease was very low.</P><P class="capture">Fig.2 Western-blot result</P><P class="PP">The band with red circle was the band of the serine protease and the marker was a protein marker of aidlab.</P><P class="PP">In order to test whether the serine protease could work normally in the yeast, we performed the enzyme activity detection using BAEE solution. We did two sets of experiments: one added PMSF, which was a inhibitor of serine protease, and the other did not. And then, reading the OD253 of these two solutions.(You can know more details about the detection from the protocol) Obviously, the OD253 of the former one is higher than the later one and the values of OD253 increased with time in a period of time ; therefore, we made the conclusion that the yeast produced the serine protase successfully and effectively.</P><P class="capture">Fig.3 The values of OD253 increased with time</P><P class="PP">The values of OD253 increased with time and this solution did not add the PMSF.</P><P class="capture">Fig.4 A box-plot of the OD253</P><P class="PP">These were the OD253 of the solutions after a period of reaction.The red one was the solution that did not add the PMSF; the blue one was the solution that added. Obviously, the OD253 of the former one is higher than the later one, so that, we could say that the yeast produced the serine protase successfully and effectively.</P><P class="PP">This year we submitted several well-working basic parts to the BioBrick Registry.</P><P class="PP" style="text-align: center !important;"><STRONG>Table of the Basic Parts we submitted to the BioBrick Registry</STRONG></P><TABLE class="table"><TBODY><TR><TH class="yellowTable">Name</TH><TH class="yellowTable">Type</TH><TH class="yellowTable">Description</TH><TH class="yellowTable">Designer</TH><TH class="yellowTable">Length(bp)</TH><TH class="yellowTable">Submitted</TH></TR><TR><TH class="grayTable">BBa_K2207003</TH><TD>Composite Parts</TD><TD>Trichoderma HR System I SOD-Tubulin dobule promoter</TD><TD>Yihe Zhang</TD><TD>3155</TD><TD>✔</TD></TR><TR><TH class="grayTable">BBa_K2207004</TH><TD>Composite Parts</TD><TD>Trichoderma HR System II Ech42-H3 double promoter</TD><TD>Yihe Zhang</TD><TD>4845</TD><TD>✔</TD></TR><TR><TH class="grayTable">BBa_K2207005</TH><TD>Composite Parts</TD><TD>Trichoderma HR System III L1-ADH1-RP27-L2</TD><TD>Yihe Zhang</TD><TD>903</TD><TD>✔</TD></TR><TR><TH class="grayTable">BBa_K2207006</TH><TD>Composite Parts</TD><TD>Trichoderma HR System IV L3-ADH1-RP27-L4</TD><TD>Yihe Zhang</TD><TD>903</TD><TD>✔</TD></TR><TR><TH class="grayTable">BBa_K2207007</TH><TD>Composite Parts</TD><TD>Trichoderma HR System V Homologous Binding SiteA</TD><TD>Yihe Zhang</TD><TD>1676</TD><TD>✔</TD></TR><TR><TH class="grayTable">BBa_K2207008</TH><TD>Composite Parts</TD><TD>Trichoderma HR System VI Homologous Binding SiteB</TD><TD>Yihe Zhang</TD><TD>1650</TD><TD>✔</TD></TR><TR><TH class="grayTable">BBa_K2207009</TH><TD>Coding Sequences</TD><TD>PhlF transcription factor</TD><TD>Qianjin Jiang</TD><TD>627</TD><TD>✔</TD></TR><TR><TH class="grayTable">BBa_K2207010</TH><TD>Other</TD><TD>Phl Operator</TD><TD>Yuxing Chen</TD><TD>825</TD><TD>✔</TD></TR><TR><TH class="grayTable">BBa_K2207021</TH><TD>Promoters</TD><TD>Calcineurin-dependent response element (CDRE)</TD><TD>Shisheng Li</TD><TD>474</TD><TD>✔</TD></TR><TR><TH class="grayTable">BBa_K2207023</TH><TD>Coding Sequences</TD><TD>Mature serine protein from Paecilomyces lilacinus</TD><TD>Zifan Xie</TD><TD>882</TD><TD>✔</TD></TR><TR><TH class="grayTable">BBa_K2207024</TH><TD>Promoters</TD><TD>T7-mutant1</TD><TD>Yuxing Chen</TD><TD>42</TD><TD>✔</TD></TR><TR><TH class="grayTable">BBa_K2207025</TH><TD>Promoters</TD><TD>T7-mutant2</TD><TD>Yuxing Chen</TD><TD>42</TD><TD>✔</TD></TR><TR><TH class="grayTable">BBa_K2207026</TH><TD>Promoters</TD><TD>T7-mutant3</TD><TD>Yuxing Chen</TD><TD>42</TD><TD>✔</TD></TR><TR><TH class="grayTable">BBa_K2207027</TH><TD>Promoters</TD><TD>T7-mutant4</TD><TD>Yuxing Chen</TD><TD>42</TD><TD>✔</TD></TR><TR><TH class="grayTable">BBa_K2207028</TH><TD>Promoters</TD><TD>T7-mutant5</TD><TD>Yuxing Chen</TD><TD>42</TD><TD>✔</TD></TR><TR><TH class="grayTable">BBa_K2207029</TH><TD>Promoters</TD><TD>T7-mutant6</TD><TD>Yuxing Chen</TD><TD>42</TD><TD>✔</TD></TR><TR><TH class="grayTable">BBa_K2207030</TH><TD>Promoters</TD><TD>T7-mutant7</TD><TD>Yuxing Chen</TD><TD>42</TD><TD>✔</TD></TR><TR><TH class="grayTable">BBa_K2207031</TH><TD>Coding Sequences</TD><TD>PhlE efflux pump</TD><TD>Junming Qian</TD><TD>1272</TD><TD>✔</TD></TR></TBODY></TABLE><P class="capture">Table.OD600 reference point</P><H2 class="H2Head" id="ref">Reference</H2><P class="PP ref">[1] Cyert M S. Calcineurin signaling in Saccharomyces cerevisiae: how yeast go crazy in response to stress[J]. Biochemical and biophysical research communications, 2003, 311(4): 1143-1150.</P><P class="PP ref">[2] Cyert M S. Genetic analysis of calmodulin and its targets in Saccharomyces cerevisiae[J]. Annual review of genetics, 2001, 35(1): 647-672.</P><P class="PP ref">[3] Brand D, Roussos S, Pandey A, et al. Development of a bionematicide with Paecilomyces lilacinus to control Meloidogyne incognita.[J]. Applied Biochemistry &amp; Biotechnology, 2004, 118(1-3):81-88.</P></DIV><DIV class="col-md-3 disappear-on-top" role="complementary"><NAV style="position: fixed; top: 100px ; left:50px;" class="bs-docs-sidebar hidden-print hidden-xs hidden-sm"><UL class="nav bs-docs-sidenav shorterli"><LI><A href="#cdre">CDRE</A></LI><LI><A href="#msp">Mature serine protein</A></LI><LI><A href="#ref">Reference</A></LI></UL></NAV></DIV></DIV></DIV><DIV class="footer" style="background-color: #000000"><DIV class="copyright container text-center"><P class="copyright">Copyright © 2017 ZJU-IGEM | <A href="http://www.zju.edu.cn/">Zhejiang University</A> |</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>