<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Macquarie_Australia_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Macquarie Australia/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV class="igem_2017_content_wrapper"><HEADER style="height: 80px"><DIV id="TopBar"><UL id="TopLvlNav"><LI><A href="https://2017.igem.org/Team:Macquarie_Australia">Home</A></LI><LI><A href="https://2017.igem.org/Team:Macquarie_Australia/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Macquarie_Australia/Collaborations"> Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Macquarie_Australia/Attributions">Attributions</A></LI></UL><LI><A> Project </A></LI><LI><A href="https://2017.igem.org/Team:Macquarie_Australia/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Macquarie_Australia/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Macquarie_Australia/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Macquarie_Australia/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Macquarie_Australia/Improve">Improve</A></LI><LI><A href="https://2017.igem.org/Team:Macquarie_Australia/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:Macquarie_Australia/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Macquarie_Australia/Experiment">Experiment</A></LI><LI><A href="https://2017.igem.org/Team:Macquarie_Australia/InterLab">Interlab</A></LI><LI><A>Parts</A></LI><LI><A href="https://2017.igem.org/Team:Macquarie_Australia/Parts">Parts Summary </A></LI><LI><A href="https://2017.igem.org/Team:Macquarie_Australia/Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Macquarie_Australia/Validation">Validation </A></LI><LI><A>Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Macquarie_Australia/HP/Silver"> Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Macquarie_Australia/HP/Gold_Integrated"> Integrated  Human Practices </A></LI><LI><A>Awards</A></LI><LI><A href="https://2017.igem.org/Team:Macquarie_Australia/Applied_Design">Applied Design </A></LI><LI><A href="https://2017.igem.org/Team:Macquarie_Australia/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2017.igem.org/Team:Macquarie_Australia/Model">Model</A></LI><LI><A href="https://igem.org/2017_Judging_Form?id=2300"> Judging Form </A></LI></DIV></HEADER><H2> Glossary </H2><P><UL><LI><B> FNR </B>- Ferredoxin NADP+ reductase.</LI><LI><B>Fer</B>- Ferredoxin and Ferredoxin NADP+ reductase (FNR).</LI><LI><B>Hyd</B>- <I>hyd 1</I> coding for the enzymatic part of the Hydrogenase molecular machine.</LI><LI><B>Fer/Hyd</B>- Fer and Hyd as described above in a new biobrick construct.</LI><LI><B>HydEF</B>- coding two of the three maturation enzymes.</LI><LI><B>HydEFG</B>- <I>hyd EF</I> and <I>hyd G</I>, coding all three maturation enzymes in a new biobrick construct.</LI><LI><B>Fer/Hyd/EFG, Omega Ω or HGPCC </B>- plasmid with all the necessary genes coding the total Hydrogenase molecular machine. Ferredoxin, Ferredoxin NADP+ reductase, Hyd 1, Hyd EF and Hyd G all in a new biobrick construct. This plasmid gained the official bio-brick name – <B> Hydrogen Gas Producing Gene Cluster (HGPGC)</B>.</LI></UL></P><H2> Week 1 (3/7/17-7/7/17)</H2><DIV class="panel"><H4> Dry Lab </H4><UL style="list-style-type:circle"><LI> The first week involved learning all about synthetic biology and what lies ahead in terms of the competition. </LI><LI>Dr Louise Brown and explained all things iGEM including the biobrick system and 3A assembly. </LI><LI> An overview of the chlorophyll biosynthesis pathway, photosystem II and hydrogenase activity presented by Professor Robert Willows since our project would focus on one component of this bigger picture. </LI><UL><LI> Our project being to use biosynthetic techniques to create the hydrogen producing hydrogenase.</LI></UL><LI> We started planning for creation of  complete hydrogenase plasmid by combining the Ferredoxin/Ferredoxin reductase biobrick with the Hyd1 biobrick. </LI><UL><LI> From this we expect some preliminary production of hydrogen however, with the addition of maturation enzymes this production would be maximised. </LI></UL><LI> Discussion on construction of the maturation plasmid in which HydEF biobrick would be combined with HydG biobrick then the two larger constructs would be ligated together to create the complete plasmid coding for the total Hydrogenase molecular machine, fondly named Omega Ω.  </LI><LI>  We created a “3-day plan” from 3A assembly through transformation and plating up cells to liquid media overnight cultures, miniprep kit extraction of plasmid and nanodrop concentration measurement, and finally electrophoresis gel screening of our newly made construct. </LI><LI> Discussion on human outreach was started. </LI><UL><LI> Began talking to the SDU iGEM team on collaboration. </LI><LI> Discussed the creation of an iGEM Macquarie game. </LI><LI> Started planning for attending synthetic biology conference and ACUR (Adelaide). </LI></UL></UL></DIV><BUTTON class="accordion"><H2> Week 2 (10/7/17-14/7/17)</H2></BUTTON><DIV class="panel"><H4> Dry Lab </H4><UL><LI> India suggested the idea of a creating a game as her brother could help us out with that. </LI><LI> We got in contact with Associate Professor Joanne Jamie, an academic at Macquarie University, to gain more information about National Science Week and the university’s Orientation Day for our outreach possibilities. </LI><UL><LI> She invited us to help out in the Chifley School Program (19th September) which we agreed would be better than the other two. We have a 1 hour time slot in which we can conduct wet and dry lab activities with gifted and talented high school science students. </LI></UL><LI> The team was approached by Nebraska to fill out their survey. </LI></UL><TOP><H4> Wet Lab  </H4><UL><LI> Checked lab supplies to ensure we had what we needed. </LI><LI> Became familiar with protocols we will be using. </LI><LI> Poured LB plates made up with each of the three antibiotics we will be using- Amp, Kan and CAM. </LI><LI> Conducted three transformations of DH5α cells with DNA plasmids of Fer, Hyd1 and HydEF. </LI><UL><LI> These were grown successfully on plates. </LI></UL><LI> Dived into our first ligation attempt for Fer/Hyd following our “3-day plan” </LI><UL><LI> Results = no growth on the plates.</LI><LI> This lead us to testing all our restriction enzymes on familiar plasmids, which gave us a chance to pour our own electrophoresis gel, load and run it. </LI><LI> It was great to have Mike Gibbs, Dominic Logel, Thi Huynh and Ed Moh supervising and sharing their expertise.</LI></UL><BUTTON class="accordion"><H2> Week 3 (17/7/17-21/7/17) </H2></BUTTON><DIV class="panel"><H4> Dry Lab </H4><UL><LI> Nebraska got back to us asking for collaboration: discussion of Australian policies in regards to synthetic biology. </LI><UL><LI> In return we hope to get the survey results that we filled out to run our own questionnaire and compare Australian/International response. </LI></UL><LI> Had a Skype call with SDU to discuss possible collaboration. </LI><LI> India and her brother got back to the rest of the team with game ideas and responded to our feedback. </LI><LI> University of Sydney contacted us to get in touch with the rest of the Australasian teams to organise a meetup. </LI><UL><LI> We replied back saying we were interested. </LI></UL><LI> ACUR Abstract submitted. </LI></UL><H4> Wet Lab  </H4><UL><LI> We reviewed the results from the restriction enzyme testing experiment conducted last week. </LI><LI> We interpreted electrophoresis gel results </LI><UL><LI><I>Xba</I>I did not perform well </LI><LI><I>Spe</I>I appeared to be contaminated </LI><LI>We discovered that although 3.1 buffer is ideal for <I>Pst</I> when doing double digest it is ideal to use Cutsmart or 2.1. </LI><LI> New restriction enzymes are expected this week. </LI></UL><LI> Conducted PCR of three backbones so we would have more stock available (CAM, Kan, Amp). </LI><LI> Mike re did our digest test of restriction enzymes. Ran electrophoresis gel of PCR’s and digests. Q5 didn’t work at all and KOD did not work for Kan. </LI><UL><LI>Mike’s digests worked out.</LI><LI>The changes he used included Smart cut buffer used across all enzymes and digest extended to 90min. </LI><LI>The team re did the digest and re did Kan PCR again. Ran gel again. Digests look great but still no success with Kan PCR. Ed has suggested we try again with a lower temp for annealing as it’s possible our anneal temp was too close to the primer melt temp. He has suggested that the CAM and Amp pcr’s were not very successful either considering the main bands were weak and primer dimer bands strong.</LI></UL><LI>We grew overnight subculture of three colonies from each of three plates- Fer, Hyd1, HydEF.
The next day we extracted the plasmid using miniprep (Qiagen) followed by digestion using <I>Eco</I>RI as well.
</LI></UL></DIV><BUTTON class="accordion"><H2> Week 4 (24/7/17-28/7/17)</H2></BUTTON><DIV class="panel"><H4> Dry Lab </H4><UL><LI> Unofficial team  meeting took place as some team members still absent. </LI><UL><LI> To Do List was discussed.  </LI></UL><LI>There was a meet-up with Ari, Steve, Indi, Louise and Catherine (Pop Up Incubator) where we consult in a game they are designing. </LI><LI> Discussion around potential collaboration opportunities with CSIRO. </LI><LI> Game avatar design finalised and comments shared amongst the team. </LI><LI>Abstract edited for SynBio by India and given to Louise to hand in for registration. </LI><LI> University of Sydney iGEM team approached to form Australasian Meeting for iGEM-they paid, we don't want to. Need to find out if University of Melbourne and Auckland (other invitees) have paid-if not allows us leverage. </LI></UL><TOP><H4> Wet Lab </H4><UL><LI> We were ready to re-attempt the three day plan which starts with 3A assembly ligation of the lac-ferredoxin- FNP with the lac-hydrogenase. </LI><UL><LI> As these were both in CAM backbones, they were digested then ligated into Amp backbone. </LI><LI> We used new incubation settings for ligation which ramped up the temperature based on advice from Rob. </LI><LI> Transformed cells were grown on plates overnight. </LI><UL><LI> We were excited to achieve one colony of our Fer/Hyd plasmid cells however with very low numbers even on the puC19 positive control plates the competency of the cells was in question.</LI></UL><UL><LI> At the same time we ligated the newly arrived biobricks- GUN4, HydG and Double terminator into CAM backbones. </LI></UL><LI> These were also transformed into cells and successfully grown overnight. </LI><LI> Subcultures of these as well as the one Fer/Hyd colony were grown in liquid media ready for miniprep. </LI></UL><LI> PCR (both KOD and Q5) was attempted again on all three backbones and the gel showed disappointing results (Figure 1). </LI><UL><LI>Having discussed this with Rob it was decided that our templates were a poor choice (iGEM linear backbones). </LI><LI> Our next attempt to amplify the backbones- CAM, Amp and Kan utilised the RFP/antibiotic resistant backbone plasmids from previous iGEM members. </LI></UL><LI>While running PCR products on gels we had the opportunity to tweak our NEB 1kb ladder. </LI><UL><LI> The ladder used was a 1:10 dilution of the original now with loading dye added to the mix so we could confidently just drop 2uL in a well and see a great result every time. </LI></UL><LI> In an attempt to test the competency of the cells we’d been using we challenged Thi’s cells against Rob’s cells (all DH5α). </LI><UL><LI> We transformed each with our Fer/Hyd ligation and with puC19 +ve control. </LI><LI> As we plated up the cells someone noticed that our heat block temp was not set correctly so effectively our transformants had never been heat shocked. </LI><UL><LI> Given these probably won’t work we re-did the transformation. </LI></UL><LI> The next day we found those that weren’t heat shocked did grow very few cells for the puC19 +ve control and none for Fer/Hyd. </LI><LI> Rob’s cells showed more colonies. 3 colonies for Fer/Hyd. </LI></UL><LI> We had also grow one more colony on a plate from spin down of the previously transformed Fer/Hyd cells which had be held in reserve in the fridge.</LI><LI> Sent off for sequencing were- lacHydG, Double terminator, GUN4 and our Fer/Hyd construct.</LI><LI> That was the last week of what was fondly known as bootcamp. Now that the vacation period is over we will be challenged with university classes and other challenges to our lab time. </LI></UL><CENTER><P align="justify" style="width:500px;word-wrap:break-word"><B><I>Figure 1.</I></B> Agarose gel (1%) electrophoresis. Lanes 1 and 10 contain a Neb 1kb marker. Lanes 2, and 6 were negative controls. Lane 4 shows no bands and indicates failed PCR of AMP in KOD. Lanes 7-9 show no bands and indicate unsuccessful PCR of respective antibiotics in Quick cells. Lanes 3 and 5 show successful band of CAM and AMP respectively at ~1600 bp. Lanes 11-14 show bands of ~3000 bp psbMZHWK (PS2) plasmid as expected cut with <I>Eco</I>-RI (E), <I>Xba</I>I (X), <I>Spe</I>I (S) and <I>Pst</I>I (P). Lanes 15-18 show successful cut of PS2 plasmid at ~1000 bp and ~2000 bp, indicating all restriction enzymes are functional in cutsmart buffer. </P></CENTER></TOP></DIV><BUTTON class="accordion"><H2> Week 5 (31/7/17-4/7/17)</H2></BUTTON><DIV class="panel"><H4> Dry Lab </H4><UL><LI> Some wiki layout ideas were drawn up by Winonah.</LI><LI> Our promotional sponsorship brochure was drafted and so was a powerpoint presentation, which was far less professional and more suited to a personal viewing.</LI><LI> A logo competition was started (set up by Ari) to choose which logo would best represent our team.</LI><UL><LI> The designs are still being created to be voted on at a later date.</LI></UL><LI>  Wiki team profiles were drafted and needed to become more succinct as they seemed bulky on the wiki. </LI><LI> Project title ideas were considered. </LI><LI> Our abstract needed changing to include some dry lab activities. </LI><LI>  GoGreen collaboration was introduced as a possible collaboration.
</LI></UL><TOP><H4> Wet Lab </H4><UL><LI> Sequencing results came back and unfortunately our Fer/Hyd plasmid was lacking <I>hyd 1</I>. </LI><UL><LI> This was unexpected. </LI></UL><LI> We miniprep extracted Fer/Hyd plasmid from the three new colonies which grew from Rob’s cells, as well as the one colony we achieved from previous cells spun down. </LI><UL><LI> Once run on gel, two from Rob’s cells (A and B) looked promising so these have been sent off for sequencing (Figure 2).</LI></UL><LI> A new ligation was made with HydEFG and Thi’s batch of cells transformed with this new construct. </LI><UL><LI> Unfortunately, nothing grew. </LI><LI> With 5µL of HydEFG plasmid still available we decided we should reduce our transformations to use only 2µL. </LI><LI> This would leave us enough  to digest and run on a gel next week. </LI><LI> The plan was to make up new competent cells as it appeared the cells we are using were not as competent as expected. </LI><LI> Keen to keep going we proceeded with commercially available competent cells of our next HydEFG transformation attempt.</LI></UL><LI>Running a gel showed that the latest attempt to PCR amplify the three backbones- Amp, CAM, Kan, was not successful (Figure 3). </LI><UL><LI> Another attempt has been made to achieve this using different templates.</LI><LI> We plan to run these on a gel next week. </LI></UL></UL><CENTER><P align="justify" style="width:428px;word-wrap:break-word"><B><I>Figure 2.</I></B> Agarose gel (1%) electrophoresis of single and double digests using <I>Eco</I>-RI (E) and <I>Pst</I>I (P) in fer/hyd DNA in transformed colony samples A, B, C and D. Lane 1 contains a 1kb ladder. Samples A (lanes 3-4), B (lanes 4-5) and C (lanes 6-7) are from the same transformed plate. Samples A and B show expected band weights for the single digests (~5400bp) and double digests (~3400 bp and 2000 bp) respectively, and were submitted for sequencing confirmation. Band weights in sample C do not correspond with expected band weights and were unsuccessful.  Sample D was spun down prior to loading and no band weights were detected. This gel validates the fer/hyd1 Biobrick to the designed constructs in samples A and B.
</P></CENTER><CENTER><P align="justify" style="width:500px;word-wrap:break-word"><B><I>Figure 3.</I></B> Agarose gel (1%) electrophoresis. Lane 1 contains a 10kb marker. Lanes 2-5 show CHLP with <I>Eco</I>-RI (E), <I>Xba</I>I (X), <I>Spe</I>I (S) and <I>Pst</I>I (P) respectively. Lanes 6-9 show restriction enzymes cutting plasmid at~1300bp and all are functional. Lanes 10-11 show unsuccessful PCR of Kan resistance.
</P></CENTER></TOP></DIV><BUTTON class="accordion"><H2> Week 6 (7/8/17-11/8/17) </H2></BUTTON><DIV class="panel"><H4> Dry Lab </H4><UL><LI> Game design was discussed in regards to background animation and sounds. The main character was finalised.</LI><LI> Our promotional brochure was completed and the powerpoint presentation was abandoned in favour of it. </LI><LI> The team decided to go ahead with the GoGreen collaboration. </LI><LI> Sponsorship was finalised. Emails went out to new potential sponsors and contact was made with previous sponsors. Other fundraisers discussed included an Indiegogo page (linked to Adrianna) and potentially Grill’d “Local Matters.” (linked to Ari). Paul Jaschke advised of Twist Bioscience sponsorship opportunity, so Ali started working on an application for it (closed 25th August). </LI><LI> Logo design contest closed and decisions progressed towards a winner (closed Monday 14th August) . </LI><LI> Official announcement that India will be presenting at ACUR Adelaide. Funding paperwork was underway and due on the 16th August. Evidence of flights/accommodation was due on 21st August so India and Ali worked on this. Ali sent an email to the Office of the PVC with questions. </LI><LI> Received an update from USyd about their project and forwarded email to Thi who thinks we may be able to collaborate (we have a lot of E.coli strain that they are using). Emailed back with our project. </LI><LI> Steven has uploaded his hypothesis and survey questions for review. </LI><LI>
On Tuesday Ari, Steven, and India had a meeting with the Dr Egg team to consult on the scientific themes incorporated into their puzzle games and brainstormed ideas. It was discussed that one of us will be at a table with a game developer and 4-5 kids for 30 minutes at a time to explain the science background of each theme and have the kids develop game ideas. It was also discussed that we would do a 10-15 minute presentation at the beginning of the event to give a basic introduction to synthetic biology and our specific iGEM project. The event will be on the 18th of August.  
</LI></UL><TOP><H4> Wet Lab </H4> 
Our plan was as follows:
<OL><LI> Make up new competent cells. </LI><LI> Run a gel of PCR products. </LI><LI>Transform cells using only 2µl HydEFG and  new competent cells. Plate up these transormants.</LI><LI>Make overnight liquid cultures of successful HydEFG cultures.</LI><LI>Miniprep HydEFG from the above.</LI><LI>Digest and run of gel the resultant extracted plasmid to check if worth sending for sequencing.</LI><LI>If result from FER/Hyd A and B sequencing is unsuccessful then a new ligation of Fer/Hyd should be attempted. Look at what might need tweaking. </LI></OL><UL><LI> Instead of making up new competent cells, we used commercial competent cells which proved successful as we were able to grow HydEFG transformants. </LI><UL><LI> Plasmids from these were extracted via miniprep. </LI><LI> They were digested and  run on a gel. </LI><LI> The gel results were not as expected. Because of this, new colonies should be screened. </LI></UL><LI> The sequencing came back for Fer/Hyd A and B. Both were as confirmed. </LI><LI> The glycerol stocks were then used to prep overnight cultures which were induced and tested for their ability to produce hydrogen. </LI><LI> The Clark electrode was used to measure the H<SUB>2</SUB> gas production and recordings went off the scale. This was great news. We are looking into the best ways to present this data. </LI><LI> Unfortunately we still have not had any success with PCR amplification of backbones.</LI><LI>HydEFG were sent to be sequenced and other cultures were grown as well as their glycerol stock. </LI></UL></TOP></DIV><BUTTON class="accordion"><H2> Week 7 (14/8/17-18/8/17) </H2></BUTTON><DIV class="panel"><H4> Dry Lab </H4><UL><LI> Organisation of ACUR conference. </LI><UL><LI> ACUR funding submitted, India and Ali are preparing slideshow presentation. </LI></UL><LI> Indiegogo online fundraiser set up by Adrianna. </LI><LI> Ali, Adrianna and Winonah attended a meeting with a potential key sponsor (Bioplatforms). They discussed all aspects of the project from human practices through to the wet lab. </LI><UL><LI> Currently waiting to hear back from the company. </LI></UL><LI> Dr Egg Digital: Team members Ari, Steven and India collaborated with a team of game developers and artists from Dr Egg Digital as science consultants for a game testing and puzzle planning event. </LI><UL><LI> Led by Dr Catherine Fargher, the concept of the videogame is based from her successful theatre show Dr Egg and the Man with No Ear and chapter book publication.</LI><LI> A class of 8-12 year olds from the Queenwood School Catalyst Program were audience to an age appropriate lecture led by the Macquarie iGEM team on the basics of synthetic biology and its applications, and the possible future prospects of our project in renewable energy. </LI><UL><LI> The lecture spurred the creativity of the children who then brainstormed game puzzles and character scripts for the Dr Egg Digital game that is being designed to incorporate the national school science curriculum for grades 3-8. </LI><LI> The resulting ideas from this collaboration will be incorporated into this educational videogame that is set to be released in February 2018. </LI></UL><LI> The event was part of Australia’s National Science Week 2017, with our project aptly relating to this year’s theme being ‘Future Earth’. </LI><LI> Impressed by how this event engaged their students, Queenwood School shared information of our iGEM team and the community outreach we have participated in on their social media channels and their school publications. </LI><UL><LI> “Thanks for your AWESOME work yesterday! Lindy the main teacher from Queenwood said she has never seen them so engaged.” – Dr Catherine Fargher (Dr Egg Digital)</LI><LI> “Dear Dr Walsh, thank you for letting me come to the Dr Egg excursion. It was really fun, it was possibly one of the best days of my life!!!! I loved inventing the game about Vivi.
P.S. glowing Cats.
P.P.S cat DNA with glowing jellyfish DNA with narwhal horn DNA = cat-a-corn.&quot; – Siana (Queenwood School student)
<A href="http://www.dreggdigital.com/"> Dr Egg Digital Website </A></LI></UL></UL><LI> Open Day: Team member India co-presented a lecture with the head of department for Chemical and Biomolecular Sciences, Professor Alison Rodger, at Macquarie University’s Open Day. </LI><UL><LI> The lecture introduced the basic concepts behind synthetic biology, what to study to enter a career in this field, and introduced the iGEM competition and our project to prospective new students. </LI><LI> A similar talk was completed by Ali, to incoming BSc (Human Bio) and BMedScs students, on what iGEM is and Macquarie’s history in the competition. </LI></UL></UL><TOP><H4> Wet Lab </H4><UL><LI> This week had a major focus on completing the Interlab study (Adrianna, Winonah, Jocelyn, Ali, Adrianna, Indi, Ed and Thi were involved).</LI><UL><LI> Transformations and overnight cultures were completed in time for the big day of data collection on Thursday which included the Interlab fluorescence measurement of plates. </LI><LI> We started early on the interlab task, first we made fresh cell cultures from the cells that had been incubating over night. Then we started with the absorbance measurements. We then made our time 0 samples, put the new cell cultures in the incubating shaker and left for 2 hours. </LI><UL><LI> During the 2 hour period the serial dilution plate was set up.</LI></UL><LI> Towards the end of interlab day, Team Member numbers were dwindling so Ed and Thi assisted with some pipetting to ensure we would be able to obtain spectrometer readings that day. </LI></UL><LI> At the same time, a backbone swap to CAM for Fer/Hyd took place followed by transformation and plating up. </LI><LI> Hydrogen production data collection by Clark electrode continued. </LI><LI> HydEFG was digested (single and double digests performed on 7 separate colonies) and screened on gel. Meanwhile we await sequencing result. </LI><UL><LI> As the gel was running, a team member messaged the group saying that  unfortunately the protocol used was not an updated version… </LI><LI> After visualising the completed gel, it appeared the digests were only successful for 3/7 samples? </LI></UL><LI> No more backbone PCR amplifications took place this week.</LI></UL><CENTER><P align="justify" style="width:500px;word-wrap:break-word"><B><I>Figure 4.</I></B> Agarose gel (1%) electrophoresis. Lanes 7 and 14 show 1kb marker. Lanes 1, 3, 5, show single digest of fer plasmid using Eco-RI (E) at ~2700 bp. Lanes 2, 4 and 6 show double digest of fer using Eco-RI and PstI (E+P) with bands at ~1700 bp and ~2000 bp. Lanes 8, 10 and 12 show hyd1 single digest with E (~3700bp). Lanes 9, 11 and 13 show double digest of <I>hyd1</I> using E+P (~1700 bp and ~2000 bp).
</P></CENTER></TOP></DIV><BUTTON class="accordion"><H2> Week 8 (21/8/17-25/8/17) </H2></BUTTON><DIV class="panel"><H4> Dry Lab </H4><UL><LI> We decided on a group name; H<SUB>2</SUB>ydroGEM. </LI><LI> We reviewed our budget with supervisor Dr Louise Brown sharing some budget number crunching. We are awaiting a final list of lab sponsors from Adrianna by the end of next week. </LI><LI> The Synthetic Biology Australasia (SBA) conference will allow 6 people (chosen) to attend and Bioplatforms may allow two more (awaiting confirmation). Due to SBA funding, alongside Adriana’s new sponsor, we only lost a total of $ 1K AUD.  We aim to stream our presentation at the conference live on Facebook. </LI><LI> The team members traveling to Boston for the jamboree will be finalised by the end of this month. We have not yet heard back from Bioplatforms (Andrew Gilbert) in regards to the sponsorship opportunity. </LI><LI> Australasian Conference of Undergraduate Research (ACUR) grant was approved. Flights and accommodation have been booked and finalised. Ali and India are preparing the iGEM presentation. We plan on live streaming our presentation of Facebook as well. </LI><LI> Great team member photos have been taken by Emma for the wiki, and will be daily released on Facebook and Twitter. </LI><LI> Survey was finalised by Steven, Ali, Jocelyn, Ari, and launched by Taylor. It was shared on the Macquarie University Facebook group, as of now have 35 submissions. Ali will post to Macquarie University Facebook page at 10 am Monday to boost submissions. Pending survey results, a prototype will be designed. </LI><LI> Interlab forms were submitted and spreadsheets emailed. So, Interlab wet lab work is now done and dusted. </LI><LI> Taylor has started work on the mathematical modelling. </LI><LI> Emma has started liaising with team Manchester on a collaboration, and is in charge of business plan. 
We have not heard back from team Melbourne on a possible collaboration. </LI><LI> Our Chifley event team is being finalised, and our Working With Children Checks are in the process of approval. India is in charge of this event.</LI></UL><TOP><H4> Wet Lab </H4><UL><LI> Since HydEFG are maturation genes, it was expected we may have some hydrogenase activity in the absence of this part hence we began testing the Fer/Hyd cells. </LI><UL><LI> The first attempt at the SDS-PAGE gel of induced Fer/Hyd cells, checking for the hydrogenase enzyme was unsuccessful. This will need to be attempted again later. </LI></UL><LI>Fer/Hyd on CAM backbone (from our backbone swap) was successfully screened by running digests on a gel so we sent that one off for sequencing (Figure 5).</LI><LI> Omega Ω which is our fondly named complete hydrogenase encoding plasmid with Fer/Hyd/HydEFG was not only assembled this week but cells were transformed, cultures grown, miniprepped plasmids digested and screened on gel showing positive potential (Figure 6). </LI><UL><LI> This is a major milestone. </LI><LI>Being a large construct of 8777 bp primers were designed and ordered in readiness or sequencing.</LI></UL><LI> A further attempt has been made to induce our transformed <I>E. coli</I> to translate our plasmids and measure hydrogen production.  </LI><UL><LI>There was an error in the media mix so things didn’t work this time. </LI></UL></UL><CENTER><P align="justify" style="width:400px;word-wrap:break-word"><B><I>Figure 5.</I></B> Agarose gel (1%) electrophoresis of Biobrick Fer/Hyd after having the backbone swapped to CAM resistance using digests using Eco-RI (E) and PstI (P). Single (E) and double (E+P) digests were performed. The bands of the single digests correspond to the expected size (~5300 bp) as well as the bands of the double digests (~3500 bp and ~2000 bp). These bands correspond with the expected weights of <I>fer/hyd1</I> in CAM.
</P></CENTER><CENTER><P align="justify" style="width:600px;word-wrap:break-word"><B><I>Figure 6.</I></B> Agarose gel (1%) electrophoresis of transformed Hydrogen Gas Producing Gene Cluster plasmid with single (S-EcoRI-HF) and double (D-EcoRI-HF and PstI) digests. Lanes 2-9 were performed on the 23/8/17 of 4 sample colonies of Quick cells. Lanes 3-5 and 7-9 (samples B, C, D) display expected band weights of ~10,700bp for single digests and ~8700 bp with ~2000 bp for double digests. Sample A of Quick cells in lanes 2, 6, 13 and 14 did not possess necessary band weights and were discarded. Sample A of commercial cells in lanes 11 and 12 correspond with expected single and double digest band weights. Samples B and C show expected band weights for all single and double digests in both Quick and commercial cells (lanes 15-22). Sample D in commercial cells (lanes 23 and 24) did not possess the expected band weights and were discarded. Sample D in quick cells (lanes 25 and 26) showed the expected band weights for single and double digests. This gel validates the design construct of the HGPGC plasmid.
</P></CENTER></TOP></DIV><BUTTON class="accordion"><H2> Week 9 (28/8/17-1/9/17) </H2></BUTTON><DIV class="panel"><H4> Dry Lab </H4><UL><LI> Our Chifley event group was finalised and discussion was focussed around what activities to include.</LI><LI> Our business plan was set in motion, and our survey was launched on the Survey Monkey website.</LI><LI> India applied and was accepted to present a 30 second pitch at TEDxYouth Sydney. </LI><LI> The Manchester collaboration has been completed - we are awaiting communication from Manchester to confirm how we plan on putting this on our wikis. </LI><LI> We decided it would add an extra dimension to have a video on our wiki. However, this will only be produced if we have enough time.</LI><LI> A thorough literature review was completed.</LI><LI> The ACUR presentation slides were started. </LI></UL><TOP><H4> Wet Lab </H4><UL><LI>This week we again induced our Omega Ω <I>E. coli</I> and tested hydrogen production using the Clark electrode. One of the challenges with our Clark electrode is the analogue format of the readout. </LI><UL><LI> Thankfully we have found someone with expertise in the area, Dr Ian Jamie, who will assist us in setting up connections to our system to record data in a digital format. </LI><LI> We will continue with our analogue readings in parallel to assist in estimating parameters for the digital data collection.</LI></UL><LI> The twelve primers we designed to sequence our 8777 bp Omega Ω arrived this week so sequencing was set up and sent off. </LI><LI> HydEFG was sequence confirmed, so we did a backbone swap ligation to CAM in readiness to submit the part to iGEM.</LI><UL><LI> Following this, there was a cell transformation with the newly created HydEFG CAM, followed by inoculation of overnight cultures for a miniprep. The miniprepped plasmids were then digest screened on an electrophoresis gel and behaved just as expected (Figure 7). </LI></UL><LI> It appears all our constructs are completed and our focus will now move to the hydrogen production testing of our induced cells as well as demonstrating the presence of the hydrogenase enzyme using SDS page gel. </LI><LI> Meanwhile in the lab we tried out some cells with fluorescent plasmids in readiness for a fun activity with Chifley students. The Chifley event is fast approaching. Unfortunately, the cells were no longer viable but there is still time for us to transform new cells with fluorescent biobricks. </LI></UL><CENTER><P align="justify" style="width:500px;word-wrap:break-word"><B><I>Figure 7.</I></B> Agarose gel (1%) electrophoresis of single (E) and double (E+P) digests on colony samples A, B and C. All three samples display expected band weights of ~7500bp for single digests and ~5500bp with ~2000bp double digests.  This gel indicates successful ligation of HydG and HydEF biobricks and validates the HydEFG biobrick.</P></CENTER></TOP></DIV><BUTTON class="accordion"><H2> Week 10 (4/9/17-8/9/17)</H2></BUTTON><DIV class="panel"><H4> Dry Lab </H4><UL><LI> Taylor and Ali went to an ‘Ideas to Concept’ workshop designed by and held at the Macquarie Incubator. The workshop was aimed at introducing ideas into an entrepreneurial or commercialised environment and provided tips on where to focus our attention with the prototype (for example, on customer value, strategic objectives-costs, revenue, metric success measurements, understanding our competitive positioning).  Some new contacts were also gained who could help should we seek further assistance with our business plan or prototype. </LI><LI> Our business plan was further discussed and it was decided to give our prototype a more national and local emphasis, as opposed to international. </LI><LI> ACUR and SBA slides were worked on. </LI><UL><LI> At SBA, we found that all the Australasian iGEM teams in attendance will be invited to an official meet up organised by Abigail from iGEM HQ. </LI></UL><LI> T-shirt designs were drafted. </LI><LI> Various teams approached us for a collaboration such as Singapore NTU, Amazon, Franconia, and USyd. </LI></UL><TOP><H4> Wet Lab </H4><UL><LI> This week the focus switched to Dry lab activities and planning what next for Wet lab while Dr Ian Jamie continued to work out program so we can go digital with our Clark electrode readings.</LI></UL></TOP></DIV><BUTTON class="accordion"><H2> Week 11 (22/9/17-15/9/17)</H2></BUTTON><DIV class="panel"><H4> Dry Lab </H4><UL><LI> SBA and ACUR slides were finalised. </LI><LI> The background for the modelling was completed and presented to the team.</LI><LI> Potential prototype design ideas were researched.</LI></UL><TOP><H4> Wet Lab </H4><UL><LI>This week Omega Ω was sequence confirmed. YAY!</LI><LI> The Clark electrode is now set up to successfully collect data digitally.</LI><LI> We successfully grew fluorescent <I>E. coli</I> in readiness for our Chifley student outreach day. We took these from plate to liquid culture. We also prepared the many LB chloramphenicol plates they require and autoclaved the cotton swabs to be used as paint brushes. </LI><LI> Much debate and discussion went into planning our upcoming experimentation designed to confirm our Omega Ω part. </LI><UL><LI> Once a plan was in place we got to work preparing quantities of media required. M9 medias and LB medias and prepared some cultures. </LI></UL></UL></TOP></DIV><BUTTON class="accordion"><H2> Week 12 (18/9/17-22/9/17)</H2></BUTTON><DIV class="panel"><H4> Dry Lab </H4><UL><LI> SBA conference was attended by the team, and the iGEM project was showcased by Taylor and Ali. A social event was arranged by Abigail and USyd, MElbourne and Auckland were all represented along with Macquarie at Luna Park. </LI><LI> Prototype research, business plan, interviews were all continued.  </LI></UL><TOP><H4> Wet Lab </H4><UL><LI> This week there were a lot of activities outside of the lab. We did aliquot our fluorescent <I>E. coli</I> with fresh media and incubated them overnight in readiness for the artistic talents of the Chifley students. </LI><UL><LI>We plated up a couple of test plates so show the students what their efforts might look like.</LI></UL><LI> The next day was the Chifley outreach day and lots of fun for all. </LI><UL><LI> Our lab was transformed into workshops and the students created fabulous art. Naturally our own iGEM team couldn’t resist trying their hand at it. </LI><LI> After overnight incubation we photographed all the plates and shared them on our facebook page.</LI></UL><LI> With all the team busy attending the Synthetic Biology Australasia conference lab work was put on hold.
</LI></UL></TOP></DIV><BUTTON class="accordion"><H2> Week 13 (25/9/17-29/9/17)</H2></BUTTON><DIV class="panel"><H4> Dry Lab </H4><UL><LI>After crunching the numbers it looked like all 9 members of our team who were willing and able could be flown and accommodated in Boston for the giant jamboree but without solid quotes there was a degree of uncertainty.</LI><UL><LI> After some serious accommodation research, while simultaneously negotiating with the travel agent for the best possible flights we managed to bring it all together. Two lovely apartments, and thanks to a late breaking Qantas sale, nine comfortable and reliable return flights to Boston. </LI><LI> The organisation aka chaos aka administrative nightmare of all this took a week and almost broke us but now it’s real. We are on our way to Jamboree!</LI></UL><LI> India and Ali showcased the iGEM research at ACUR in The University of Adelaide, receiving questions mostly on the prototype and ethics of having GMO in the home. </LI></UL><TOP><H4> Wet Lab </H4><UL><LI> This week we began execution of the new experimental plan to collect replicate measurements of our hydrogen production using the Clark electrode. </LI><UL><LI> After our preliminary experiments, we have our protocol optimised. </LI><LI> The aim is to test Omega Ω, comparing both induced and uninduced, Fer/Hyd and non-transformed DH5α<I>E. coli</I>.</LI></UL><LI> We also set up a saturated culture with an inverted measuring cylinder in water to do a crude but visual demonstration of the hydrogen production from our cells.
SDS Page gel was done for induced and uninduced Fer (Figure 8).</LI></UL><CENTER><P align="justify" style="width:500px;word-wrap:break-word"><B><I>Figure 8.</I></B> SDS-PAGE gel of induced and purified Fer fractions.
</P></CENTER></TOP></DIV><BUTTON class="accordion"><H2> Week 14 (2/10/17-6/10/17) </H2></BUTTON><DIV class="panel"><H4> Dry Lab </H4><UL><LI> All team members have started their wiki submissions for Adrianna and Winonah.</LI><LI> India started working on the human outreach activities submissions, and Emma worked on the business plan throughout the entirety of the session. </LI><LI> Ali and Emma also drafted a doc for the gold criteria. </LI><LI> All human outreach activities other than Women in Science have ceased. </LI></UL><TOP><H4> Wet Lab </H4><P><UL><LI>  This week parts were registered online so we are ready to prepare them for shipment next week.</LI><LI> The apparatus set-up to demonstrate the hydrogen production didn’t work first round as gas was leaking however with tubing switched out, and a few other adjustments to ensure no leakage, we collected a decent sample of hydrogen gas. Enough for a pop test. </LI><LI> This week we cut two bands from the Fer SDS page gel in readiness for mass spectrometry. </LI><UL><LI> Since this gel wasn’t very photographic we thought it was worthwhile to replicate. </LI></UL><LI> We continued to collect data from our Clark electrode and ran into an unexpected anomaly. </LI><UL><LI> Our non- transformed DH5α<I>E. coli</I> are producing much more hydrogen than expected. So much so we are unsure that the culture can be trusted. </LI><LI> We mini-prepped to see if our cells contained plasmid and made a rooky mistake of using wash without ethanol which gave us haywire poor quality nanodrops. </LI><LI> Third time we got it right and oddly there was a little plasmid present. Even though growing up new mature cultures is a setback it is the only solution at this point so we plated up some cultures in readiness for Monday.</LI></UL><LI> This week we also began collaboration work for Singapore NTU by initially transforming BL21 cells with their plasmids then growing these up in liquid culture. </LI></UL></P></TOP></DIV><BUTTON class="accordion"><H2> Week 15 (9/10/17-13/10/17)</H2></BUTTON><DIV class="panel"><H4> Dry Lab </H4><UL><LI> Adrianna and Winonah killing the wiki game. Issues on resolution need to be addressed. </LI><LI> A new banner design was created and ordered in readiness for Boston. </LI><LI>T-shirts arrived, more will be ordered along with hoodies (BOOYAH!)</LI><LI> All members successfully are completing their assigned tasks to upload to the wiki.</LI><LI> Women in Science event was cancelled due to more pressing deadlines. </LI></UL><TOP><H4> Wet Lab </H4><UL><LI> We set up three more inverted measuring cylinder set ups, full of water and linked to cultures via a tube, to measure total volume of gas produced by three cultures: non-transformed DH5α, Omega induced and Fer/Hyd induced. The race is on to see which makes the most gas. </LI><UL><LI> The question remains how much of that gas is hydrogen? Knowing some of that gas will be carbon dioxide, a sample of gas was drawn from each measuring cylinder and injected into sealed test tubes containing barium hydroxide which creates a solid precipitate when it reacts with CO<SUB>2</SUB>. By weighing this precipitate, we hope to determine just how much of this gas is carbon dioxide. We also weighed a sample of gas, comparing it to the weight of the equivalent volume of air, and hydrogen, to estimate the percentage of hydrogen in our gas.</LI></UL><LI> Maldi TOF mass spectrometry (MS) was used to assess. Fer.</LI><LI> We continued to collect Clark electrode data. With fresh cultures, and no cross contamination, we now had good analog data accumulating. </LI><UL><LI> We had to abandon the digital setup which limited us to one Clark electrode. With limited time we needed to collect our data across all four cultures in synchrony. </LI></UL><LI> Another exciting milestone: we prepared and shipped off our four biobricks- Omega Ω, Fer/Hyd, HydEFG and HydG. Omega Ω is given the official bio-brick name – Hydrogen Gas Producing Gene Cluster (HGPGC).</LI><LI> Continuing our collaboration work for NTU Singapore we grew up the cells in liquid culture, sonically lysed them then isolated proteins using a histidine column. We did a PCR clean up of a ferrochelatase gene in readiness to test their dCAS9 mutant.</LI></UL></TOP></DIV><BUTTON class="accordion"><H2> Week 16 (16/10/17-20/10/17)</H2></BUTTON><DIV class="panel"><H4> Dry Lab </H4><UL><LI> Our primary focus has been judging forms and updating our wiki before the deadline. So far so good! Go TEAM! </LI><LI>A meeting was set with Michael Ramp from the Macquarie Incubator who helped in creating a 3D model for our prototype.</LI></UL><TOP><H4> Wet Lab </H4><UL><LI> This week marked the end of the Clark electrode era.  The last of the data was collected and it was time to crunch the numbers and show that data graphically. </LI><UL><LI>Adrianna did an amazing job turning all those analog scrolls into easily understood graphs.</LI></UL><LI> Furthering work for the NTU Singapore collaboration, more large scale cultures were made with ZYM-5052 media. This week we did an electrophoretic mobility shift assay (EMSA) testing the commercial Cas9 ready for comparison next week with an EMSA of NTU’s Cas9.</LI><LI>We did further replicates of the gas measuring apparatus as well as weighing gas and reacting samples with barium hydroxide.</LI><LI>Our MS results for Fer were ready for interpretation. </LI></UL></TOP></DIV><BUTTON class="accordion"><H2> Week 17 (23/10/17-27/10/17)</H2></BUTTON><DIV class="panel"><H4> Dry Lab </H4><UL><LI> Judging forms were completed at our student meeting! Go us!</LI><LI> All our wiki entries were drafted and double checked and corrections and edits are being processed. </LI><LI>Second team member shirts were ordered. </LI><LI> In the meeting with Michael Rampe, we were offered the opportunity for our prototype to be featured in the Pedestal website, and we sent them updated logo files, the design branding that we wanted for the prototype to ensure we can get as much as we can done prior to wiki freeze. </LI><LI>Last minute changes were made to our collaboration, project attribution and applied design page to reflect the updates given to us by Pedestal.</LI></UL><TOP><H4> Wet Lab </H4><UL><LI>Being the last week for any lab work we are wrapping things up.</LI><LI> Four attempts were made at producing the most photogenic electrophoresis gel to present our digested parts to the world with pride.</LI><UL><LI> It is one thing to run a gel for a screening, it is an art to create one which is both accurate and appealing.</LI></UL><LI> We set up one more set of cultures in the gas collecting apparatus. Continued to collect data and took a sample of gas. </LI><UL><LI> Again, we compared the weight of the gas with that of air and hydrogen. </LI><LI> We also did another barium hydroxide reaction to determine how much CO<SUB>2</SUB> was present. </LI><LI> Displacement was measured for each set up then it was time to analyse.</LI></UL><LI> Finalising the collaboration work for NTU Singapore, we did an electrophoretic mobility shift assay testing their dCAS9 mutants.</LI><LI> After weeks of waiting we finally received a delivery of cytochrome c so there was a mad scramble to do an assay to demonstrate the activity of Fer.</LI><LI> Hanging up the lab coat marked a brief moment of reflection then someone reminded us about the looming wiki deadline and everyone dived head first into their computers.  </LI><LI> The End. </LI></UL><CENTER><P align="justify" style="width:500px;word-wrap:break-word"><B><I>Figure 9.</I></B> Purification of FNR on Q column fractions for samples A, B and C. The intense yellow colouring corresponds with a greater degree of protein purification. Of the 12 fractions taken from each sample, the ones pictured (A10-12, B1, B6-12 and C1) contained purified protein and were used for spectrometry. 
</P><CENTER><P align="justify" style="width:500px;word-wrap:break-word"><B><I>Figure 10.</I></B> Spectrophotometer at 550 nm of purified FNR fraction. The peaks (blue) correspond to the reduction of cytochrome C by the oxidisation of NADPH to NADP<SUP>+</SUP> by FNR. This validates the characterisation of the <I>fer</I> biobrick.
</P></CENTER></CENTER></TOP></DIV><BUTTON class="accordion"><H2> Week 18 (30/10/17-3/10/17) </H2></BUTTON><DIV class="panel"><H4> Dry Lab </H4><UL><LI> Wiki freeze was on Thursday so all team members were working quickly to finalise any changes. </LI><LI> Auditions took place to select speakers to present our project at the Jamboree. </LI></UL></DIV><CENTER><TABLE style="width:1290px; background-color:#50b738;"><TBODY><TR><TD width="322px"><CENTER><H4><STRONG> GOLD SPONSORS </STRONG></H4></CENTER></TD><TD width="322px"><CENTER><H4><STRONG> BRONZE SPONSORS </STRONG></H4></CENTER></TD><TD width="322px"><CENTER><H4><STRONG>LOCATION</STRONG></H4></CENTER><CENTER> Faculty of Science and Engineering, 
Macquarie University  
Balaclava Road, North Ryde, NSW, 2109, Australia  
E7B 350 </CENTER><CENTER><H4><STRONG>CONTACT US</STRONG></H4></CENTER><P style="text-align:center">
                    Email: <A href="mailto:macquarie.australia@gmail.com" style="color:white" target="_blank">macquarie.australia@gmail.com</A></P></TD><TD width="200px"><CENTER><H4><STRONG>FOLLOW US ON</STRONG></H4></CENTER></TD></TR></TBODY></TABLE></CENTER></UL></TOP></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>