<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_OUC-China_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:OUC-China/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:OUC-China/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:OUC-China/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:OUC-China/Notebook&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:OUC-China/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:OUC-China/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><HEADER><DIV class="navbar navbar-inverse"><DIV class="navbar-collapse collapse navbar-inverse-collapse"><UL class="nav navbar-nav"><LI><A href="https://2014.igem.org/Team:OUC-China">HOME</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">TEAM </A></LI><LI><A href="https://2014.igem.org/Team:OUC-China/Team">Team members</A></LI><LI><A href="https://2014.igem.org/Team:OUC-China/Team_Instructor">Instructor</A></LI><LI><A href="https://2014.igem.org/Team:OUC-China/Team_Acknowledgment">Acknowledgment</A></LI><LI><A href="https://2014.igem.org/Team:OUC-China/Team_Lab">Lab</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=OUC-China">Official team profile</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">PROJECT </A></LI><LI><A href="https://2014.igem.org/Team:OUC-China/Project">Overview</A></LI><LI><A href="https://2014.igem.org/Team:OUC-China/Project_Background">Background</A></LI><LI><A href="https://2014.igem.org/Team:OUC-China/Project_Design">Design</A></LI><LI><A href="https://2014.igem.org/Team:OUC-China/Project_Result">Result</A></LI><LI><A href="https://2014.igem.org/Team:OUC-China/Project_Future">Future</A></LI><LI><A href="https://2014.igem.org/Team:OUC-China/Project_Modeling">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:OUC-China/Project_Policy_Practise">Policy&amp;Practice</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">JUDGING </A></LI><LI><A href="https://2014.igem.org/Team:OUC-China/Judging">Biobrick</A></LI><LI><A href="https://2014.igem.org/Team:OUC-China/Judging_Achievment">Achievement</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">SAFETY </A></LI><LI><A href="https://2014.igem.org/Team:OUC-China/Safety">Biosafety</A></LI><LI><A href="https://2014.igem.org/Team:OUC-China/Safety_Lab_safety">Labsafety</A></LI><LI class="dropdown active"><A href="#" class="dropdown-toggle" data-toggle="dropdown">NOTEBOOK </A></LI><LI class="active"><A href="https://2014.igem.org/Team:OUC-China/Notebook">Lab note</A></LI><LI><A href="https://2014.igem.org/Team:OUC-China/Notebook_Modeling_Note">Modeling note</A></LI><LI><A href="https://2014.igem.org/Team:OUC-China/Notebook_Protocols">Protocols</A></LI></DIV></DIV></HEADER><DIV id="mainContent" style="width:1024px;background-color:#d2e9fd;padding:10px;margin:0 auto"><H1 class="text-primary">Lab Note</H1><P><SPAN class="lead">WEEK 1</SPAN></P><P>We acquired foundational knowledge as preparation for our project.Under the guidance of teachers and fellow students, we studied molecular biology and synthetic biology.</P><P><SPAN class="lead">WEEK 2</SPAN></P><P>Then we learned molecular experiments, included bacterination, plasmid miniprep,	agarose electrophoresis, enzyme digestion, gel extraction, ligation, transformation, bacterial coating, PCR and so on.</P><P><SPAN class="lead">WEEK 3</SPAN></P><P>We acquired plasmid RP4 fromother lab,professor QIU Zhigang provided us HB101 strain which contains plasmid RP4. After getting HB101 strain, we tested the plasmid in several ways. In the end, the test results show the plasmid is exactly what we want. Firstly, through resistance screening experiment to determine if the types of antibiotics genes carried by the plasmid are consistent with literature. Secondly, getting the vital sequence from the plasmid by means of PCR to confirm whether the plasmid is RP4 or not.Thirdly, by doing conjugation experiment to test if the plasmid could conjugate with other strains.</P><P>We built a plasmid which used pet32a as backbone and the gene sequence of TAT-H4, and then we transformed it into BL21 (DE3).</P><P>Experiment:</P><P>a)	Transform：transform our plasmid into BL21(DE3) and culture it on LB solid medium. </P><P>b)	Plasmid miniprep: plasmid extraction from BL21 (DE3).</P><P>c)	Agarose electrophoresis: detect length of the extracted plasmid, it’s about 6000bp so the it confirms the correctness of the result.</P><P>d)	Culture: HB101 strain wasscreened by ampicillin and kanamycinrespectivelyin LB broth. And both bottles of </P><P>broth were turbid.</P><P>e)	plasmid miniprep: plasmid extraction from HB101.</P><P>f)	agaroseelectrophoresis: detect length of the extracted plasmid, and the result is right.</P><P>g)	PCR: after repeated experiment, we get the vital sequence, traF and trbIfrom plasmid RP4.</P><P>h)	Agaroseelectrophoresis: detect the plasmid PCR product, and it is consistent with the expected result.</P><P> i)	Conjugation: The donor strain was HB101 with the plasmid RP4. Then, we usednalidixic acidas screening condition and use JM109 as recipient cell, but failed.So we used streptomycin as screening condition and use Top10 as recipient cell.As expected, the result was right.</P><P><SPAN class="lead">WEEK 4</SPAN></P><P>We designed PCR primers of oriTRP4 sequence with restriction endonuclease cleavage sitesEcoRI, XbaI, SpeI, PstI.After getting the fragment, we madeDNA sequencing for oriT region of RP4. In addition, the sequencing result of oriT region turned out right.</P><P>We cultured the BL21(DE) which containing the pet32a plasmid at 37℃.We constantly measured the OD values of the broth, and when the values of the OD is within 0.2-0.6,we added the broth with IPTG(Its concentration is 1Mm) for 12h.After that, we make the bacterial lysis and release all protein. By using SDS we found that the bacteria after induction produce another kind of protein (33KD) compared with the contrast group, this is in line with our expectations.

</P><P>Experiment:
</P><P>a)	Induction：We constantly measured the OD values of the broth, and when the values of the OD is within 0.2-0.6,we induced the broth with IPTG(Its concentration is 1Mm) for 12h.
</P><P>b)	SDS：We make the bacterial lysis and extracted all protein. By using SDS we found that the bacteria after induction produced another kind of protein(33KD) compared with the control group, this is in line with our expectations.
</P><P>c)	PCR: get the oriT region sequence from plasmid RP4.</P><P>d)	Agarose electrophoresis: detect the PCR product, and the length is 386bp.</P><P>  e)	product purification: purify the PCR product.
</P><P>f)	Ligation: ligate oriTRP4 with T-vector.
</P><P>g)	Transformation: transform the ligation product into JM109.
</P><P>h)	Blue –white selection.
</P><P>i)	Pick bacteria and culture for four hours.
</P><P>j)	PCR and electrophoresis examing: there is no fragment, so we failed.
</P><P>k)	Adjust the PCR system: after analysising the experiment, we add DMSO in original system. And the target fragment was 386bp.
</P><P>l)	Sequencing.

</P><P><SPAN class="lead">WEEK 5</SPAN></P><P>We ligated oriTRP4 with reporter gene BBa_J04450.At the same time, we linked oriTRwith reporter gene BBa_J04450 in order to test whether BBa_J01003 could work or not.	

</P><P>Experiment:
</P><P>a)	Enzymedigestion: Enzyme digestion of oriTRP4 for 25μL system, use EcoRI and SpeI; Enzyme digestion of pSB1K3 (2013 Kit Plate 2, 6B) for 25μL system, use EcoRI and XbaI. Enzyme digestion ofpSB1C3 (2013 Kit Plate 3, 22O)for25μL system, use SpeI andPstI; Enzyme digestion of BBa_J04450-pSB1K3 (2013 Kit Plate 2, 6B) for 25μL system, use XbaI andPstI.
</P><P>b)	Agarose electrophoresis: production of oriTRP4 and pSB1K3, gel is good; BBa_J01003-pSB1C3 and BBa_J04450, gel is good.
</P><P>c)	Gel extraction of DNA.
</P><P>d)	Ligation: ligate oriTRP4 with pSB1K3; ligate BBa_J04450 with BBa_J01003-pSB1C3
</P><P>e)	Transformation: transform the ligation products into Top10 respectively.
</P><P>f)	Bacterial coating.
</P><P>g)	Pick the red colonies and cultured.
</P><P>h)	PCR and electrophoresis examing: to detect the bacteria we want (oriTRP4-BBa_J04450-pSB1K3); to detect the bacteria we want (BBa_J01003-BBa_J04450-pSB1C3).</P><P>i)	Sequencing.
</P><P>j)	Conservation: conserve the picked strains with oriTRP4-BBa_J04450-pSB1K3 plasmid.
</P><P>k)	Plasmid miniprep: plasmid extraction of BBa_J01003-BBa_J04450-pSB1C3, which the sequencing result is right.
</P><P>l)	Agarose electrophoresis: plasmid BBa_J01003-BBa_J04450-pSB1C3, the part BBa_K1439002 is constructed.
</P><P>m)	Conservation: conserve the picked strains with BBa_J01003-BBa_J04450-pSB1C3 plasmid

</P><P><SPAN class="lead">WEEK 6</SPAN></P><P>Plasmid backbone pSB1K3 was replaced by backbone pSB1C3 due to the requirement of part submission.Constructed the parts BBa_K1439001 and BBa_K1439000.

</P><P>Continuing to amplificate engineering bacteria and using affinity chromatography to purify our protein ,then we use SDS to find that only our protein was in the eluent.

</P><P>Experiment:
</P><P>a)	Affinity chromatography：Amplification of our engineering bacteria continued and we used affinity chromatography to purify our protein, then we used SDS to find that only our protein was in the eluent.
</P><P>b)	Enzyme digestion: Enzyme digestion oforiTRP4-BBa_J04450-pSB1K3 plasmid, use EcoRI and PstI. Enzyme digestion of backbone pSB1C3 provided by 2014 IGEM, use EcoRI and PstI. Enzyme digestion of oriTRP4-pMD19-T vector, use EcoRI and PstI.
</P><P>c)	Agarose electrophoresis: detect the production, and gel is good.</P><P>d)	Gel extraction of DNA, get the part oriTRP4,oriTRP4-BBa_J04450 and pSB1C3
</P><P>e)	Ligation：ligateoriTRP4-BBa_J04450 withpSB1C3, ligateoriTRP4 with pSB1C3.
</P><P>f)	Transformation: transform the ligation products into Top10.
</P><P>g)	Bacterial coating.
</P><P>h)	Pick the colonies and cultured.
</P><P>i)	Plasmid miniprep: plasmids extraction of oriTRP4-BBa_J04450-pSB1C3 and oriTRP4- pSB1C3of which the sequencing resultsare right.
</P><P>j)	Agarose electrophoresis: plasmid oriTRP4-BBa_J04450-pSB1C3 and plasmid oriTRP4- pSB1C3, so the part BBa_K1439001 andBBa_K1439000is constructed.

</P><P><SPAN class="lead">WEEK 7</SPAN></P><P>We constructed double plasmids system to confirm if mini plasmid could conjugate with the help of original plasmid RP4. We used two mini plasmid which carries oriTR-J04450 and oriTRP4-J04450 severally.

</P><P>Experiment：
</P><P>a)	Bacterination: use the HB101 strain 
</P><P>b)	Make competent cells: use the HB101 strain, provided by Prof. QIU Zhigang, which is sensitive tostreptomycin.
</P><P>c)	Bacterination: HB101 with plasmid RP4, Top10 with mini plasmid (oriTRP4-BBa_J04450), and Top10 with mini plasmid (oriTR-BBa_J04450). Culture for 14 hours.
</P><P>d)	Plasmid miniprep: plasmids extraction from HB101 and Top10respectively, plasmid RP4, plasmid oriTR-BBa_J04450 and plasmid oriTRP4-BBa_J04450.
</P><P>e)	Agarose electrophoresis: detect the production, and gel is good.
</P><P>f)	Transformation: transform the plasmid oriTR-BBa_J04450and plasmid oriTRP4-BBa_J04450 into competent cells HB101 (no streptomycin resistance) respectively.
</P><P>g)	Bacterial coating
</P><P>h)	Pick red colonies and culture.
</P><P>i)	Conjugation: HB101 with oriTR-BBa_J04450 and HB101 with RP4; HB101 with oriTRP4-BBa_J04450 and HB101 with RP4.
</P><P>j)	Pick the red colonies and culture, add ampicillin and chloramphenicol.
</P><P>k)	Plasmid miniprep: plasmids extraction of double plasmids system. RP4 and oriTRP4-BBa_J04450, RP4 and oriTR-BBa_J04450.
</P><P>l)	Agarose electrophoresis: detect if the double plasmids systems were constructed successfully, gel is good.
</P><P>m)	Culture Top10 with no plasmid.
</P><P>n)	Conjugation: HB101 withdouble plasmids system(RP4 andoriTRP4-BBa_J04450) and Top10; HB101 withdouble plasmids system(RP4 andoriTR-BBa_J04450) and Top10;</P><P>o)	Pick the red colonies and culture, the strain is Top10 only due tostreptomycin and chloramphenicol.
</P><P>p)	Plasmid miniprep: plasmids extraction of mini plasmid oriTRP4-BBa_J04450 and oriTR-BBa_J04450 respectively. 
</P><P>q)	Agarose electrophoresis: detect the mini plasmids.
Gel is good.

</P><P><SPAN class="lead">WEEK 8</SPAN></P><P> We confirm that mini plasmid can’t conjugate by itself.

</P><P>We built a composite plasmid which used pcDNA3.1(+) backbone and have the gene sequence of EGFP, then we transformed it into Top10 for amplification.

</P><P>a)	Transform：transform the plasmid into Top10 and coating it on LB solid medium.
</P><P>b)	Plasmid miniprep: plasmid extraction from Top10(pcDNA3.1(+)-EGFP).</P><P>c)	Agarose electrophoresis: detect length of the extracted plasmid, and the result is right.
</P><P>d)	Conjugation: HB101 with plasmid (oriTRP4-BBa_J04450) only and Top10; HB101 with plasmid (oriTR-BBa_J04450) only and Top10.
</P><P>e)	Pick the red colonies and culture, the strain is Top10 only due tostreptomycin and chloramphenicol.
</P><P>f)	Plasmid miniprep:plasmids extraction from Top10.
</P><P>g)	Agarose electrophoresis: detect that there is no plasmid.
</P><P>Gel is good.

</P><P><SPAN class="lead">WEEK 9</SPAN></P><P> We attempted many times to make HB101 conjugate with <EM>Vibrio harveyi</EM> according to paper.

</P><P>We mixed the protein TAT:H4 with the plasmid pcDNA3.1(+), and the mass ratios of DNA and protein were 0,1:1, 1:2, 1:4, 1:8, 1:16 in 2ul glucose solution. We performed gel retardation experiments, a delay in the electrophoretic mobility of sample was observed and the level of which is coincident with increased concentration of TAT: H4.

</P><P>Experiment：
</P><P>a)	Plasmid and protein binding experiments: Use 5% glucose solution to dissolve protein and mixed it with plasmid. Incubation for 1h at 37℃.We mixed up the protein TAT:H4 with the plasmid pcDNA3.1(+),and the mass ratios of DNA and protein were 0,1:1, 1:2, 1:4, 1:8, 1:16 in 2ul glucose solution. We performed gel retardation experiments, a delay in the electrophoretic mobility of sample was observed and the level of which is coincident with increased concentration of TAT: H4. 
</P><P>b)	Bacterination: HB101 with double plasmids system, <EM>Vibrio harveyi</EM>.
</P><P>c)	Conjugation: between HB101 and <EM>Vibrio harveyi</EM>.</P><P>d)	Pick the right colonies and culture.
</P><P>e)	Plasmid miniprep: plasmids extraction from <EM>Vibrio harveyi</EM>.
</P><P>f)	Agarose electrophoresis: detect the plasmid.Gel is good.


</P><P><SPAN class="lead">WEEK 10</SPAN></P><P> We tried to inactivate the oriT region of RP4, and consulted our instructor the problem, then made evaluation for several approaches. Finally, according to our laboratory facility, we chose the way of homologous recombination. 

</P><P>We used DNaseI to deal with the protein-DNA complex, and the mass ratios of DNA and protein were 0,1:1, 1:2, 1:4, 1:8, 1:16 in 2ul glucose solution , compared to the control, we could find that the protect effects is synchronous with the increased concentration of TAT:H4.

</P><P>We put pcDNA3.1(+)-EGFP which bound with TAT:H4 into the cell-culture medium. We put 500ng complex into 500ul cell-culture medium. After 24h ,the green fluorescence appeared in the cells when the section of living material was observed by fluorescent microscope with blue light.


</P><P>Experiment：
</P><P>a)	Designed primers for homologous recombination.
</P><P>b)	After getting the primers, we gained fragment of homologous recombination from PCR.</P><P>c)	We prepared electro-cells (E.coli BL21) for electroporation.
</P><P>d)	By the means of electroporation, plasmid RP4 and DNA fragment of homologous recombination were transformed into prepared electro-cells BL21, and we cultured the cells. But we failed the experiment. 
We plan to continue the experiment.
</P><P>e)	Plasmid protection：We used DNaseI to deal with the protein-DNA complex, and the mass ratios of DNA and protein were 0,1:1, 1:2, 1:4, 1:8, 1:16 in 2ul glucose solution , compared to the control, we could find that the protect effects is coincident with the increased concentration of TAT:H4.
</P><P>f)	Cell transfection：We put pcDNA3.1(+)-EGFP which bound with TAT:H4 into the cell-culture medium. We put 500ng complex into 500ul cell culture medium. After 24h, the green fluorescence appeared in the cells when the section of life material was observed by fluorescent microscope with blue light.

</P><P><SPAN class="lead">WEEK 11</SPAN></P><P> Let our protein bind with pcDNA3.1-EGFP and then inject it to zebrafish’s muscle tissue .We inject 100ng plasmid and protein complexes to each fish in experimental group and to the control group we only inject 100ng plasmid per fish. Then we carried out RT-PCR experiment after 24h, and it can be found that both of the fish in experimental group and control group had the transcription of mRNA but the experimental group’s transfection efficiency is higher than the control group. We do some quantitative analysis experiments to confirm this fact.

</P><P>We added plasmid and protein complexes into Cell culture then observed after 24 hours. green fluorescence can be observed in the dark environment

</P><P>Experiment：

</P><P>a)	Plasmid and protein binding experiments: Use 5% glucose solution to dissolve protein and mixed it with plasmid. Incubation for 1h at 37℃.

</P><P>b)	Cell transfection：We add plasmid and protein complexes into cell culture then observed it after 24 hours. We can see green fluorescence in the dark environment

</P><P><SPAN class="lead">WEEK 12</SPAN></P><P> We tested the self lysis devices parts which are induced by aTc or ara.

</P><P>Experiment：
</P><P>a)	We got the parts BBa_K112808 and BBa_K206000 from kit plates and transformed the genes into Top10.
</P><P>b)	We picked these bacteria and shaked bacteria to 14 hours.
</P><P>c)	Plasmid miniprep：plasmids extraction from top10.</P><P>d)	Two PCR reactions were set up to clone out genes(T4holin、endolysin) from BBa_k112808.
</P><P>e)	And then we did the experiment of enzyme digestion and gel extraction. 
</P><P>f)	The production PCR purification was combined with a promoter and overnight ligation.
</P><P>g)	We inserted the genes into Top10. What’s more, we picked the colonies and culture them.
</P><P>h)	We tested it and observed the experiment phenomenon.
</P><P>i)	Ligation products of promoter and lysis genes into pSB1C3 were transformed into competent TOP10 cells.
</P><P>j)	We got the Tet operon. We inoculated 10ul of long term storage glycerol stock in 100 ml of LB and incubated it at 37℃, 160 rpm for about 16 hours. 
</P><P>k)	Plasmid miniprep：plasmids extraction from top10.  
</P><P>l)	Two PCR reactions were set up to clone out genes(T4holin、endolysin) from BBa_k112808.
</P><P>m)	And then we did the experiment of enzyme digestion and gel extraction. 
</P><P>n)	The production PCR purification was combined with a promoter and overnight ligation.
</P><P>o)	We tested the two parts which have three genes from enterobacteria phage T4: lysozyme, holin, and antiholin. One of the parts is induced by aTc, the other is induced by ara.
</P><P>p)	We tested another two parts which have two genes from enterobacteria phage T4: lysozyme, holin.One of the </P><P>parts is induced by aTc, the other is induced by ara.

    </P></DIV><DIV class="footer"><P>contact us: <A href="mailto:oucigem@163.com">oucigem@163.com</A></P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:OUC-China/Notebook">http://2014.igem.org/Team:OUC-China/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:OUC-China/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:OUC-China/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:OUC-China/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:OUC-China/Notebook&amp;oldid=390166" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>