Team:Elan Vital Korea/Notebook
Team
Members
Attributions
Project
Project Overview
Plasmids Design
Background Information
Parts
Result
Wetlab
Protocol
Notebook
Safety
Human Practice
Practices
Collaborations
Video
iGEM Resource
Medal Requirement
WETLAB
-Notebook-
NOTEBOOK
4/28/15
We received the DNA distribution kit from iGEM, and in order to amplify our amount of DNA, we transformed the DNA in competent cells, using the following LB growth medium.
1. Solid Medium
Trypton
2g
Sodium chloride
2g
Yeast
1g
Bacto­Agar
3g
*D.W 200ml standard
2. Liquid Medium
Trypton
2g
Sodium chloride
2g
Yeast
1g
*D.W 200ml standard
Because  we  needed  to  add  antibiotics,  we  created  a  stock  solution,  with  a  concentration  of 34mg/ml, diluted by 1000 times. When the growth medium cooled, we put in antibiotics.
Next, we marked the plasmids that we wanted to use from the DNA distribution kit.
Then  we  added  10ul  of  distilled  water  to  each  plasmid,  and  stirred  well,  which  caused  the plasmid to turn red. We left the mixture to lay for 5 minutes.
These were the plasmids we had selected:
BBa J23100
17D plate 4 (ampicillin)
BBa B0030
4G plate 4
BBa C0062
4L plate 2
BBa B0024
1D plate 4
BBa R0062
6H plate 2
BBa E0040
13L plate 4 (ampicillin)
BBa B0015
3F plate 3
BBa C0060
4H plate 2
BBa I732006
3B plate 3
Next, we prepared the transformation of the plasmids into competent cells. Since the cells had been stored at ­70°C, we had to leave them out previous to the experiment to let them thaw, along with the S.O.C medium.
First, we added 3ul of DNA to the cells, and shook the mixture (pipetting didn’t do the job). After leaving the mixture on ice for 30 minutes, we heat­shocked it at 42°C, for 30 seconds. Then, we left it on ice again for two minutes.
After that, we added 250ul of a pre­warmed S.O.C medium, and left the mixture in a shaking incubator at 37°C and 225rpm for an hour. We then spread the mixture onto an LB plate, and allowed it to incubate overnight, for 16 hours.
4/29/15
The  previous  results  were  divided,  as  B0030,  C0062,  R0062,  I732006,  B0015  formed observable  colonies,  while  J23100,  B0024,  C0060,  E0040  did  not.  Thus,  we  decided  to  re-spread these four onto another LB plate.
Successful Results:
4/30/15
We re­transformed J23100, B0024, C0060, and E0040, and spread them onto a growth medium with ampicillin.
5/1/15
As  of  May  1>st  C0060,  E0040  colony  were  additionally  verified,  while  B0024,  J23100  still continued  to  not  show  any  sign  of  colony  growth.  We  decided  to  run  a  miniprep  with  the successful colonies B0030, C0062, R0062, I732006, B0015, C0060, and E0040, and thus left them in a 5ml liquid growth medium each.
Successful Results:
5/2/15
We ran the miniprep, which was unsuccessful as the resulting amount of DNA was too small to work with.
5/4/15
Two weeks after receiving the DNA distribution kit from iGEM, we got the plasmids we needed to work with two weeks later, in bacterial cells.
5/6/15
We  used  the  successful  colonies  to  run  our  experiments.  We  transformed  B0024  into  a competent cell and spread it onto an ampicillin growth medium. We streaked bacteria containing J23100 and I732073. We also prepared C0060 and B0030 for a miniprep, and put them each in a 5ml liquid growth medium.
5/7/15
We  looked  at  the  results  of  the  procedures  of  the  day  before,  and  while  B0024  had  no observable  colony  growth,  J23100  and  I732073  did. Then,  we ran  a  miniprep  on  C0060  and B0030, which we had prepared for 18 hours the day before.
5/10/15
The  team  made  the  decision  to  change  the  plasmids  we  were  working  with,  opting  for  the J37032  plasmid,  a  LuxR­responsive  GFP,  in  order  to  save  time.  We  also  changed  our terminator to the K823017 plasmid. B0024 was supposed to be our bidirectional terminator, but because it couldn’t produce enough DNA, we had to find another plasmid to replace it.
Here is the complete list of plasmids we are using:
constitutive promoter
J23100
ampicillin
RBS
B0030
chloramphenicol
LuxR
C0062
ampicillin
terminator
K823017
chloramphenicol
LuxR­responsive GFP
J37032
chloramphenicol
terminator
B0015
chloramphenicol
AiiA
C0060
chloramphenicol
Plasmid Construction Scheme
1. LuxR
C0062 (vector) :  EcoRI - XbaI
B0030, J23100 (fragment) : EcoRI - SpeI
2. GFP
J37032
3. LuxR - GFP
K823017 (fragment) :  EcoRI - SpeI
promoter RBS­LuxR (vector) : EcoRI - XbaI
J37032 (fragment) : EcoRI - SpeI
4. Test plasmid
C0060 (vector) : EcoRI - XbaII
B0030, J23100 (fragment) : EcoRI - SpeI
5/11/15
We decided to make 1% concentration agarose gel, using 0.7 grams of Seakem LE Agarose, 0.3  grams  of  Nusieye  GTG  agarose,  and  100  mililliters  of  1xTBE  buffer. After putting all the mentioned  ingredients  into  a  flask,  we  put  wrap  over  the  entrance,  and  microwaved  for  two minutes, melting it entirely. Then we solidified the agarose by pouring it into a mold. Out of the DNA kit, we also transformed K823017 and J37032 into competent cells.
5/11/15
We  verified  the  transformations  from  the  day  before,  and  both  the K823017  and  the  J37032 colony were successful.
5/13/15~5/19/15
In  order  to  increase  the  amount  of  plasmids  of  B0015,  B0030,  J37032,  K823017,  C0060, C0062, I732073, and J23100, we did minipreps and midipreps, which left us with more viable DNA to work with.
5/20/15
To  verify  whether  the  plasmids  we’ve  created  were relevant  to  our  experiment,  we  cut  them using a restriction enzyme.
These were the plasmids we used: B0015, B0030, J37032, K823017, C0060, C0062, I732073, J23100
The results (depending on the enzyme used, highlighted in bold):.
DNA of each
1uL
XbaI/PstI
2uL
Buffer
2uL
Distilled Water
15uL
Total
20uL
This was the order of the gel electrophoresis markers:
(2.19kb, 2.2kb, 2.08kb, 3.0kb, 2.88kb, 2.85kb, 5.1kb, 2.1kb)
B0015, K823017, B0030, J37032, C0060, C0062, I732073, J23100 / XbaI
B0015, K823017, B0030, J37032, C0060, C0062, I732073, J23100 / PstI
C0062 and J23100 had larger sizes than were expected.
5/29/15
Due to our electrophoresis results being incorrect, in order to verify the existence of fragments we had to do a double digestion procedure, cutting B0015, K823017, B0030, J37032, C0060, I732073, J23100 with EcoRI and XbaI.
DNA of each
2uL
EcoRI
1uL
PstI
1uL
Buffer
2uL
Distilled Water
14uL
Total
20uL
Because of C0062 being far too big in the electrophoresis results, we redid the miniprep.
Since we got two types of C0062 from the iGEM kit, one that grows in an ampicillin medium and one  that  grows  in  a  chloramphenicol  medium.  We  decided  to  do  a  miniprep,  then  run  a restriction  enzyme  procedure  to  see  which  one  would  be  better  for  our  experiment. We  then prepared the C0062s for a miniprep.
6/1/15
We ran a miniprep on C0062.
C0062 #1, #2, #3  (ampicillin)
C0062 #1, #2, #3, #4, #5 ( chloramphenicol )
DNA of each
2uL
EcoRI
1uL
PstI
1uL
Buffer
2uL
Distilled Water
14uL
Total
20uL
Lane 1 : DNA ladder marker    Lane 2 : J23100    Lane 3 : C0060    Lane 4 : B0030
Lane 5 : B0015                     Lane 6 : I732073   Lane 7 : J37032   Lane 8 : K823017
All plasmids were cut with EcoRI and PstI, then underwent gel electrophoresis. Looking at the DNA  map of  J23100, which had 2,105bp, we noticed  that when we  cut with EcoRI  and PstI, there were two fragments, of 2kbp and 850bp.
Lane 1,2,3 : ① ② ③    Lane 4,5,6,7,8 :④ ⑤ ⑥ ⑦ ⑧    Lane 9 : DNA ladder marker
*If we cut C0062 with EcoRI and PstI, we get two fragments as well, of 2,030 bp and 822bp.
The plasmids labeled ① ② ③ have ampicillin resistance vectors.
The plasmids labeled ④ ⑤ ⑥ ⑦ ⑧ have chloramphenicol resistance vectors. Plasmid ⑧ is a C0062.
While the DNA map states that the sizes of plasmids 1, 2, 3 are accurate, because we know that they ampicillin resistance genes in the vector, there are issues with our experiment scheme.
To be able to verify the above results, we ran sequencing with B0030, C0062, and J23100.
These were the sequencing primers:
VF2
TGC CAC CTG ACG TCT AAG AA
VR
ATT ACC GCC TTT GAG TGA GC
6/5/15 ~ 6/7/15
We sent J23100, B0030, C0062, C0060, B0015, K823017, J37032, and J61100 plasmid to a sequencing company to see these exact DNA sequence of our samples.
Out of these, J23100, B0030, and B0015 had different sequence results than we had expected when comparing with samples on the Registry.
So we decided to change these parts.
constitutive promoter
J23100 -> J61100
ampicillin
RBS
B0030 -> J61100
LuxR
C0062
ampicillin
terminator
K823017 -> K823017
chloramphenicol
LuxR­responsive GFP
J37032
chloramphenicol
AiiA
C0060
chloramphenicol
BBa J61100    plate 4 16C
BBa C0062     plate 2 4L
BBa K823017 plate 1 3D
BBa J37032    plate 3 8O
BBa C0060     plate 2 4H
Plasmid Construction Scheme
1. LuxR
J61100                    (vector)     : SpeI - PstI
C0062                     (fragment) : XbaI - PstI
C0060                     (vector)     : EcoRI - PstI
J6110 + C0062        (fragment) : EcoRI - PstI
2. GFP
K823017                 (vector)     : SpeI - PstI
J37032                    (fragment) : XbaI - PstI
3. LuxR-GFP
promoter RBS-LuxR (vector)     : SpeI - PstI
K823017 + J37032   (fragment) : XbaI - PstI
4. Test plasmid
J61100                    (vector)     : SpeI - PstI
C0060                     (fragment) : XbaI - PstI
C0060                     (vector)     : EcoRI - PstI
J6110 + C0060        (fragment) : EcoRI - PstI
6/7/15
We transformed J61100, and spread it on the ampicillin medium.
6/8/15
We picked 3 colonies from J61100 and inoculated for the miniprep.
6/11/15
We inoculated J61100 for the midiprep.
6/12/15
We ran a midiprep on the J61100.
6/15/15
Enzyme digestion
We digested J61100 with SpeI and PstI (used as a vector), while we digested C0062 with XbaI and PstI(used as a fragment), and digested C0060 (used as a fragment) with a XbaI and PstI. We then ran a gel electrophoresis and then extracted the gel for the plasmid fragment.
Ligation
We ligated J61100 and C0062 while we ligated J61100 and C0060.
Transformation
We then ran a transformation of these onto some competent cells.
6/16/15
We picked 6 colonies of each of J61100+C0062 and J61100+C0060, and inoculated them on the liquid LB. We then incubated the cell culture overnight for 16 hours.
6/17/15
We conducted a mini prep on J61100+C0062 and J61100+C0060.
Enzyme digestion
We digested J37032 with XbaI and PstI (used as a fragment), and K823017 with SpeI and PstI (used as a vector). We then ran a gel electrophoresis and then extracted the gel for the plasmid fragment.
Ligation
We ligated J37032 and K823017.
Transformation
We then ran a transformation of these onto some competent cells.
6/18/15
We picked up 6 colonies of J37032+K823017 and inoculated them on the liquid LB. We then incubated the cell culture overnight for 16 hours.
6/19/15
We conducted a mini prep on J37032+K823017
6/22/15
We chose the plasmid which has plasmid that we want, and then we cut those plasmids with EcoRI and PstI. After that, we sent them to the sequencing company.
6/25/15
We  learned  that  J61100  had  ampicillin  expression  so  we  changed  the  vector  to  have chloramphenicol resistance.
6/29/15
Enzyme digestion
We cut J61100+C0062 with SpeI and PstI (used as the vector), while we cut K823017+J37032 with XbaI and PstI (used as the fragment). We then ran a gel electrophoresis and then extracted the gel for the plasmid fragment.
Ligation
We ligated J61100+C0062 and K823017+J37032, creating the reporter plasmid.
Transformation
We then ran a transformation of these onto some competent cells.
6/30/15
We  picked  up  6  colonies  of  the reporter  cell  and  inoculated  them  on  the  liquid  LB.  We  then incubated the cell culture overnight for 16 hours.
7/1/15
We conducted a mini prep on the reporter plasmid.
7/2/15
We chose the plasmid which has plasmid that we want, and then we cut those plasmids with EcoRI and PstI. After that, we sent them to the sequencing company.
Also, to make our new LacZ biobrick, we decided to use R0062’s LuxpR fragment.
But when we cut R0062 with EcoRI and SpeI, we got a very small fragment, of around 70 base pairs, making it difficult to extract viable DNA from the gel.
7/3/15
To extract LuxpR from R0062, we decided to run a PCR instead.
We made the following Primers:
R62_F
TGA TTT CTG GAA TTC GCG GC
R62_R
CAG CGG CCG CTA CTA GTA
7/5/15
We templated R0062 and ran the PCR. The results and details are as below.
PCR
DNA Template
1uL
95°C for 2min
95°C for 20sec
57°C for 20sec
72°C for 20sec<
35 cycles
72°C for 2min
Forward Primer
1uL
Reverse Primer
1uL
2X Buffer
10uL
Distilled Water
7uL
Total
20uL
Gel Electrophoresis:
*Lane 1 : DNA Ladder marker. Lane 2 and 3 : PCR product
After the PCR, we cut the DNA with EcoRI과 SpeI.
Using a gel extraction kit, we managed to obtain the needed fragment from R0062.
We cut J61100 with EcoRI XbaI, then used it as a vector. Then we ligated this with the fragment from R0062, and transformed them.
7/6/15
We picked up 6 colonies from the LB plate they were on, and ran a mini prep.
  *Lane 1 : DNA Ladder marker Lane 2~7 : J61100 + R0062 / EcoRI-PstI
    Lane 8 : J61100/EcoRI-XbaI Lane 9~14 : J61100 + R0062 / XbaI
We also ran a midiprep on Number 5, on lane 6. This was because Number 5, if cut with EcoRI and PstI, will yield around a 110-base pair fragment. As is shown in the picture, it also was the clearest on the gel. We also sent it to a sequencing company to be sequenced.
7/8/15
We cut R0062 and J61100 with PstI and SpeI, then used it as a vector.
For our fragment, we cut I732006 with XbaI and PstI.
We ligated and transformed both, and noted that our vector had ampicillin resistance and not chloramphenicol.
7/9/15
We picked up 4 colonies from our plate and incubated them in a liquid LB for around 16 to 18 hours.
7/10/15
We ran a miniprep on R0062+J61100+I732006. Then we cut them with XbaI and PstI.
Gel Electrophoresis:
*Lane 1: DNA Ladder Marker
  Lane 2~4: R0062+J61100+I732006 / XbaI-PstI
  Lane 5: I732006 / XbaI-PstI
We got all our fragments from Lanes 2 to 4. We sent these to be sequenced as well.
7/13/15
We cut R0062+J61100+I732006 with XbaI and PstI for our fragment.
We cut K823017 with PstI and SpeI for our vector. We then ligated and transformed.
7/14/15
We picked up the colony and incubated in a liquid LB for around 16 to 18 hours.
7/15/15
We ran a miniprep and enzyme digestion following that. Then we sent the resulting sample to be sequenced.
7/17/15
Lane 1 : DNA Ladder marker
Lane 2 : J61100 / EcoRI
Lane 3 : R0062+J61100  / EcoRI
Lane 4 : R0062+J61100+I732006 / EcoRI
Lane 5 : K823017+ R0062+J61100+I732006 / EcoRI
Lane 6 : R0062+J61100  / EcoRI-PstI
Lane 7 : R0062+J61100+I732006 / EcoRI-PstI
Lane 8 : K823017+ R0062+J61100+I732006 / EcoRI-PstI
We cut J61100+C0062 with PstI and SpeI for our vector.
We then cut K823017+R0062+J61100+I732006 with XbaI and PstI for our fragment. We then ligated and transformed both.
7/20/15
We picked up the colony and incubated in a liquid LB for around 16 to 18 hours.
7/21/15
We ran a miniprep and enzyme digestion, then sent the result to be sequenced.
7/23/15
We ran a midiprep on the DNA we got.
