<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Elan_Vital_Korea_NCNotebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Elan Vital Korea/NCNotebook</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="list-note-01"><H1 class="day">4/28/15</H1><P class="content-txt">We received the DNA distribution kit from iGEM, and in order to amplify our amount of DNA, we transformed the DNA in competent cells, using the following LB growth medium.</P><H2 class="sub-title">1. Solid Medium</H2><DIV class="tbl"><TABLE cellpadding="0" cellspacing="0"><TBODY><TR><TH>Trypton</TH><TD class="back-y">2g</TD></TR><TR><TH>Sodium chloride</TH><TD class="back-y">2g</TD></TR><TR><TH>Yeast</TH><TD class="back-y">1g</TD></TR><TR><TH class="bdr-bt-none">Bacto­Agar</TH><TD class="bdr-bt-none back-y">3g</TD></TR></TBODY></TABLE></DIV><H3 class="cmt">*D.W 200ml standard</H3><H2 class="sub-title">2. Liquid Medium</H2><DIV class="tbl"><TABLE cellpadding="0" cellspacing="0"><TBODY><TR><TH>Trypton</TH><TD class="back-y">2g</TD></TR><TR><TH>Sodium chloride</TH><TD class="back-y">2g</TD></TR><TR><TH class="bdr-bt-none">Yeast</TH><TD class="bdr-bt-none back-y">1g</TD></TR></TBODY></TABLE></DIV><H3 class="cmt">*D.W 200ml standard</H3><P class="content-txt">Because  we  needed  to  add  antibiotics,  we  created  a  stock  solution,  with  a  concentration  of 34mg/ml, diluted by 1000 times. When the growth medium cooled, we put in antibiotics.
Next, we marked the plasmids that we wanted to use from the DNA distribution kit.
Then  we  added  10ul  of  distilled  water  to  each  plasmid,  and  stirred  well,  which  caused  the plasmid to turn red. We left the mixture to lay for 5 minutes.
These were the plasmids we had selected:</P><DIV class="tbl"><TABLE cellpadding="0" cellspacing="0"><TBODY><TR><TH>BBa J23100</TH><TD class="left">17D plate 4 (ampicillin)</TD></TR><TR><TH>BBa B0030</TH><TD class="left">4G plate 4</TD></TR><TR><TH>BBa C0062</TH><TD class="left">4L plate 2</TD></TR><TR><TH>BBa B0024</TH><TD class="left">1D plate 4</TD></TR><TR><TH>BBa R0062</TH><TD class="left">6H plate 2</TD></TR><TR><TH>BBa E0040</TH><TD class="left">13L plate 4 (ampicillin)</TD></TR><TR><TH>BBa B0015</TH><TD class="left">3F plate 3</TD></TR><TR><TH>BBa C0060</TH><TD class="left">4H plate 2</TD></TR><TR><TH class="bdr-bt-none">BBa I732006</TH><TD class="bdr-bt-none left">3B plate 3</TD></TR></TBODY></TABLE></DIV><P class="content-txt">Next, we prepared the transformation of the plasmids into competent cells. Since the cells had been stored at ­70°C, we had to leave them out previous to the experiment to let them thaw, along with the S.O.C medium.
First, we added 3ul of DNA to the cells, and shook the mixture (pipetting didn’t do the job). After leaving the mixture on ice for 30 minutes, we heat­shocked it at 42°C, for 30 seconds. Then, we left it on ice again for two minutes.
After that, we added 250ul of a pre­warmed S.O.C medium, and left the mixture in a shaking incubator at 37°C and 225rpm for an hour. We then spread the mixture onto an LB plate, and allowed it to incubate overnight, for 16 hours.</P><H1 class="day m40">4/29/15</H1><P class="content-txt">The  previous  results  were  divided,  as  B0030,  C0062,  R0062,  I732006,  B0015  formed observable  colonies,  while  J23100,  B0024,  C0060,  E0040  did  not.  Thus,  we  decided  to  re-spread these four onto another LB plate.Successful Results:</P><H1 class="day m40">4/30/15</H1><P class="content-txt">We re­transformed J23100, B0024, C0060, and E0040, and spread them onto a growth medium with ampicillin.</P><H1 class="day m40">5/1/15</H1><P class="content-txt">As  of  May  1&gt;st  C0060,  E0040  colony  were  additionally  verified,  while  B0024,  J23100  still continued  to  not  show  any  sign  of  colony  growth.  We  decided  to  run  a  miniprep  with  the successful colonies B0030, C0062, R0062, I732006, B0015, C0060, and E0040, and thus left them in a 5ml liquid growth medium each.Successful Results:</P><H1 class="day m40">5/2/15</H1><P class="content-txt">We ran the miniprep, which was unsuccessful as the resulting amount of DNA was too small to work with.</P><H1 class="day m40">5/4/15</H1><P class="content-txt">Two weeks after receiving the DNA distribution kit from iGEM, we got the plasmids we needed to work with two weeks later, in bacterial cells.</P><H1 class="day m40">5/6/15</H1><P class="content-txt">We  used  the  successful  colonies  to  run  our  experiments.  We  transformed  B0024  into  a competent cell and spread it onto an ampicillin growth medium. We streaked bacteria containing J23100 and I732073. We also prepared C0060 and B0030 for a miniprep, and put them each in a 5ml liquid growth medium.</P><H1 class="day m40">5/7/15</H1><P class="content-txt">We  looked  at  the  results  of  the  procedures  of  the  day  before,  and  while  B0024  had  no observable  colony  growth,  J23100  and  I732073  did. Then,  we ran  a  miniprep  on  C0060  and B0030, which we had prepared for 18 hours the day before.</P><H1 class="day m40">5/10/15</H1><P class="content-txt">The  team  made  the  decision  to  change  the  plasmids  we  were  working  with,  opting  for  the J37032  plasmid,  a  LuxR­responsive  GFP,  in  order  to  save  time.  We  also  changed  our terminator to the K823017 plasmid. B0024 was supposed to be our bidirectional terminator, but because it couldn’t produce enough DNA, we had to find another plasmid to replace it.Here is the complete list of plasmids we are using:</P><DIV class="tbl"><TABLE cellpadding="0" cellspacing="0"><TBODY><TR><TH>constitutive promoter</TH><TD class="left bdr-right">J23100</TD><TD class="left">ampicillin</TD></TR><TR><TH>RBS</TH><TD class="left bdr-right">B0030</TD><TD class="left">chloramphenicol</TD></TR><TR><TH>LuxR</TH><TD class="left bdr-right">C0062</TD><TD class="left">ampicillin</TD></TR><TR><TH>terminator</TH><TD class="left bdr-right">K823017</TD><TD class="left">chloramphenicol</TD></TR><TR><TH>LuxR­responsive GFP</TH><TD class="left bdr-right">J37032</TD><TD class="left">chloramphenicol</TD></TR><TR><TH>terminator</TH><TD class="left bdr-right">B0015</TD><TD class="left">chloramphenicol</TD></TR><TR><TH class="bdr-bt-none">AiiA</TH><TD class="left bdr-right bdr-bt-none">C0060</TD><TD class="left bdr-bt-none">chloramphenicol</TD></TR></TBODY></TABLE></DIV><H2 class="sub-title">1. LuxR</H2><P class="content-txt m-10">C0062 (vector) :  EcoRI ­ XbaIB0030, J23100 (fragment) : EcoRI ­ SpeI</P><H2 class="sub-title">2. GFP</H2><P class="content-txt m-10">J37032</P><H2 class="sub-title">3. LuxR­GFP</H2><P class="content-txt m-10">K823017 (fragment) :  EcoRI ­ SpeIpromoter RBS­LuxR (vector) : EcoRI ­ XbaIJ37032 (fragment) : EcoRI ­ SpeI</P><H2 class="sub-title">4. Test</H2><P class="content-txt m-10">C0060 (vector) : EcoRI ­ XbaIIB0030, J23100 (fragment) : EcoRI ­ SpeI</P><H1 class="day m40">5/11/15</H1><P class="content-txt">We decided to make 1% concentration agarose gel, using 0.7 grams of Seakem LE Agarose, 0.3  grams  of  Nusieye  GTG  agarose,  and  100  mililliters  of  1xTBE  buffer. After putting all the mentioned  ingredients  into  a  flask,  we  put  wrap  over  the  entrance,  and  microwaved  for  two minutes, melting it entirely. Then we solidified the agarose by pouring it into a mold. Out of the DNA kit, we also transformed K823017 and J37032 into competent cells.</P><H1 class="day m40">5/11/15</H1><P class="content-txt">We  verified  the  transformations  from  the  day  before,  and  both  the K823017  and  the  J37032 colony were successful.</P><H1 class="day m40">5/13/15~5/19/15</H1><P class="content-txt">In  order  to  increase  the  amount  of  plasmids  of  B0015,  B0030,  J37032,  K823017,  C0060, C0062, I732073, and J23100, we did minipreps and midipreps, which left us with more viable DNA to work with.</P><H1 class="day m40">5/20/15</H1><P class="content-txt">To  verify  whether  the  plasmids  we’ve  created  were relevant  to  our  experiment,  we  cut  them using a restriction enzyme.These were the plasmids we used: B0015, B0030, J37032, K823017, C0060, C0062, I732073, J23100The results (depending on the enzyme used, highlighted in bold):.</P><DIV class="tbl"><TABLE cellpadding="0" cellspacing="0"><TBODY><TR><TH>DNA of each</TH><TD class="back-y">1uL</TD></TR><TR><TH>XbaI/PstI</TH><TD class="back-y">2uL</TD></TR><TR><TH>Buffer</TH><TD class="back-y">2uL</TD></TR><TR><TH>Distilled Water</TH><TD class="back-y">15uL</TD></TR><TR><TH class="bdr-bt-none">Total</TH><TD class="bdr-bt-none back-y">20uL</TD></TR></TBODY></TABLE></DIV><H2 class="sub-title align-c">XbaI (8) ↰↱ PstI (8)</H2><P class="content-txt">This was the order of the gel electrophoresis markers:
(2.19kb, 2.2kb, 2.08kb, 3.0kb, 2.88kb, 2.85kb, 5.1kb, 2.1kb)
B0015, K823017, B0030, J37032, C0060, C0062, I732073, J23100 / XbaI
B0015, K823017, B0030, J37032, C0060, C0062, I732073, J23100 / PstI
C0062 and J23100 had larger sizes than were expected.</P><H1 class="day m40">5/29/15</H1><P class="content-txt">Due to our electrophoresis results being incorrect, in order to verify the existence of fragments we had to do a double digestion procedure, cutting B0015, K823017, B0030, J37032, C0060, I732073, J23100 with EcoRI and XbaI.</P><DIV class="tbl"><TABLE cellpadding="0" cellspacing="0"><TBODY><TR><TH>DNA of each</TH><TD class="back-y">2uL</TD></TR><TR><TH>EcoRI</TH><TD class="back-y">1uL</TD></TR><TR><TH>PstI</TH><TD class="back-y">1uL</TD></TR><TR><TH>Buffer</TH><TD class="back-y">2uL</TD></TR><TR><TH>Distilled Water</TH><TD class="back-y">14uL</TD></TR><TR><TH class="bdr-bt-none">Total</TH><TD class="bdr-bt-none back-y">20uL</TD></TR></TBODY></TABLE></DIV><P class="content-txt">Because of C0062 being far too big in the electrophoresis results, we redid the miniprep.
        Since we got two types of C0062 from the iGEM kit, one that grows in an ampicillin medium and one  that  grows  in  a  chloramphenicol  medium.  We  decided  to  do  a  miniprep,  then  run  a restriction  enzyme  procedure  to  see  which  one  would  be  better  for  our  experiment. We  then prepared the C0062s for a miniprep.</P><H1 class="day m40">6/1/15</H1><P class="content-txt">We ran a miniprep on C0062.C0062 #1, #2, #3  (ampicillin)C0062 #1, #2, #3, #4, #5 ( chloramphenicol )</P><DIV class="tbl"><TABLE cellpadding="0" cellspacing="0"><TBODY><TR><TH>DNA of each</TH><TD class="back-y">2uL</TD></TR><TR><TH>EcoRI</TH><TD class="back-y">1uL</TD></TR><TR><TH>PstI</TH><TD class="back-y">1uL</TD></TR><TR><TH>Buffer</TH><TD class="back-y">2uL</TD></TR><TR><TH>Distilled Water</TH><TD class="back-y">14uL</TD></TR><TR><TH class="bdr-bt-none">Total</TH><TD class="bdr-bt-none back-y">20uL</TD></TR></TBODY></TABLE></DIV><P class="content-txt">All plasmids were cut with EcoRI and PstI, then underwent gel electrophoresis. Looking at the DNA  map of  J23100, which had 2,105bp, we noticed  that when we  cut with EcoRI  and PstI, there were two fragments, of 2kbp and 850bp.</P><P class="content-txt txt-i">*If we cut C0062 with EcoRI and PstI, we get two fragments as well, of 2,030 bp and 822bp.The plasmids labeled ① ② ③ have ampicillin resistance vectors.The plasmids labeled ④ ⑤ ⑥ ⑦ ⑧ have chloramphenicol resistance vectors. Plasmid ⑧ is a C0062.</P><P class="content-txt">While the DNA map states that the sizes of plasmids 1, 2, 3 are accurate, because we know that they ampicillin resistance genes in the vector, there are issues with our experiment scheme.To be able to verify the above results, we ran sequencing with B0030, C0062, and J23100.These were the sequencing primers:</P><DIV class="tbl"><TABLE cellpadding="0" cellspacing="0"><TBODY><TR><TH>VF2</TH><TD class="left">TGC CAC CTG ACG TCT AAG AA</TD></TR><TR><TH class="bdr-bt-none">VR</TH><TD class="bdr-bt-none left">ATT ACC GCC TTT GAG TGA GC</TD></TR></TBODY></TABLE></DIV><H1 class="day m40">6/5/15 ~ 6/7/15</H1><P class="content-txt">We sent J23100, B0030, C0062, C0060, B0015, K823017, J37032, and J61100 plasmid to a sequencing company to see thee exact DNA sequence of our samples.Out of these, J23100, B0030, and B0015 had different sequence results than we had expected when comparing with samples on the Registry.So we decided to change these parts.</P><DIV class="tbl"><TABLE cellpadding="0" cellspacing="0"><TBODY><TR><TH>constitutive promoter</TH><TD class="left bdr-right"><FONT class="txt-g">J23100</FONT> -&gt; J61100</TD><TD class="left">ampicillin</TD></TR><TR><TH>RBS</TH><TD class="left bdr-right"><FONT class="txt-g">B0030</FONT> -&gt; J61100</TD></TR><TR><TH>LuxR</TH><TD class="left bdr-right">C0062</TD><TD class="left">ampicillin</TD></TR><TR><TH>terminator</TH><TD class="left bdr-right"><FONT class="txt-g">K823017</FONT> -&gt; K823017</TD><TD class="left">chloramphenicol</TD></TR><TR><TH>LuxR­responsive GFP</TH><TD class="left bdr-right">J37032</TD><TD class="left">chloramphenicol</TD></TR><TR><TH class="bdr-bt-none">AiiA</TH><TD class="left bdr-right bdr-bt-none">C0060</TD><TD class="left bdr-bt-none">chloramphenicol</TD></TR></TBODY></TABLE></DIV><P class="content-txt">BBa J61100    plate 4 16CBBa C0062     plate 2 4LBBa K823017 plate 1 3DBBa J37032    plate 3 8OBBa C0060     plate 2 4H
        </P><H1 class="day m40">6/7/15</H1><P class="content-txt">We transformed J61100, and spread it on the ampicillin medium.</P><H1 class="day m40">6/8/15</H1><P class="content-txt">We picked 3 colonies from J61100 and inoculated for the miniprep.</P><H1 class="day m40">6/11/15</H1><P class="content-txt">We inoculated J61100 for the midiprep.</P><H1 class="day m40">6/12/15</H1><P class="content-txt">We ran a midiprep on the J61100.</P><H1 class="day m40">6/15/15</H1><H2 class="sub-title">Enzyme digestion</H2><P class="content-txt">We digested J61100 with SpeI and PstI (used as a vector), while we digested C0062 with XbaI and PstI(used as a fragment), and digested C0060 (used as a fragment) with a XbaI and PstI. We then ran a gel electrophoresis and then extracted the gel for the plasmid fragment.</P><H2 class="sub-title">Ligation</H2><P class="content-txt">We ligated J61100 and C0062 while we ligated J61100 and C0060.</P><H2 class="sub-title">Transformation</H2><P class="content-txt">We then ran a transformation of these onto some competent cells.</P><H1 class="day m40">6/16/15</H1><P class="content-txt">We picked 6 colonies of each of J61100+C0062 and J61100+C0060, and inoculated them on the liquid LB. We then incubated the cell culture overnight for 16 hours.</P><H1 class="day m40">6/17/15</H1><P class="content-txt">We conducted a mini prep on J61100+C0062 and J61100+C0060.</P><H2 class="sub-title">Enzyme digestion</H2><P class="content-txt">We digested J37032 with XbaI and PstI (used as a fragment), and K823017 with SpeI and PstI (used as a vector). We then ran a gel electrophoresis and then extracted the gel for the plasmid fragment.</P><H2 class="sub-title">Ligation</H2><P class="content-txt">We ligated J37032 and K823017.</P><H2 class="sub-title">Transformation</H2><P class="content-txt">We then ran a transformation of these onto some competent cells.</P><H1 class="day m40">6/18/15</H1><P class="content-txt">We picked up 6 colonies of J37032+K823017 and inoculated them on the liquid LB. We then incubated the cell culture overnight for 16 hours.</P><H1 class="day m40">6/19/15</H1><P class="content-txt">We conducted a mini prep on J37032+K823017</P><H1 class="day m40">6/22/15</H1><P class="content-txt">We chose the plasmid which has plasmid that we want, and then we cut those plasmids with EcoRI and PstI. After that, we sent them to the sequencing company.</P><H1 class="day m40">6/25/15</H1><P class="content-txt">We  learned  that  J61100  had  ampicillin  expression  so  we  changed  the  vector  to  have chloramphenicol resistance.</P><H1 class="day m40">6/29/15</H1><H2 class="sub-title">Enzyme digestion</H2><P class="content-txt">We cut J61100+C0062 with SpeI and PstI (used as the vector), while we cut K823017+J37032 with XbaI and PstI (used as the fragment). We then ran a gel electrophoresis and then extracted the gel for the plasmid fragment.</P><H2 class="sub-title">Ligation</H2><P class="content-txt">We ligated J61100+C0062 and K823017+J37032, creating the reporter cell.</P><H2 class="sub-title">Transformation</H2><P class="content-txt">We then ran a transformation of these onto some competent cells.</P><H1 class="day m40">6/30/15</H1><P class="content-txt">We  picked  up  6  colonies  of  the reporter  cell  and  inoculated  them  on  the  liquid  LB.  We  then incubated the cell culture overnight for 16 hours.</P><H1 class="day m40">7/1/15</H1><P class="content-txt">We conducted a mini prep on the reporter cell.</P><H1 class="day m40">7/2/15</H1><P class="content-txt">We chose the plasmid which has plasmid that we want, and then we cut those plasmids with EcoRI and PstI. After that, we sent them to the sequencing company.</P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>