<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Elan_Vital_Korea_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Elan Vital Korea/Description</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="default"><UL><LI><A href="https://2015.igem.org/Team:Elan_Vital_Korea">HOME</A></LI><LI><A href="https://2015.igem.org/Team:Elan_Vital_Korea/Team">TEAM</A></LI><LI><A href="https://2015.igem.org/Team:Elan_Vital_Korea/Description">PROJECT</A></LI><LI><A href="https://2015.igem.org/Team:Elan_Vital_Korea/Notebook">WETLAB</A></LI><LI><A href="https://2015.igem.org/Team:Elan_Vital_Korea/Practices">HUMAN PRACTICES</A></LI></UL></DIV><DIV class="subnav" style="padding-left:70px;"><UL><LI style="color:#5ab9cc;
"><A href="https://2015.igem.org/Team:Elan_Vital_Korea/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Elan_Vital_Korea/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Elan_Vital_Korea/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Elan_Vital_Korea/Design">Design</A></LI></UL></DIV><DIV class="sub-top"><P class="sub-top-title mar-c">Description</P></DIV><DIV class="list-note-01"><H1 class="sub-title align-c">DESCRIPTION</H1><P class="content-txt m20">Bacteria that acquire resistance to antibiotics pose a serious health problem globally. Following our team’s project model for last year, Elan Vital Korea’s new project is also related to MRSA. This year, however, we instead focused on the early detection of MRSA infections using quorum sensing. Below, we have briefly described the health threats caused by MRSA, and have explained the method of quorum sensing. We also detail how we have designed and implemented our experiments, and what results we have obtained. Finally, we briefly outlined the implication of our results and our future plans. 
        </P><H1 class="sub-title align-c m80">Threats of Antibiotics-Resistant Bacteria</H1><P class="content-txt m20">
Infections by antibiotic-resistant bacteria are serious health threats worldwide, including countries such as Korea and the United States of America. It is a serious threat primarily because, as its name suggests, bacteria have evolutionarily developed a resistance to antibiotics. It simply means that our current drugs don’t work. Furthermore, the spread of antibiotic-resistant bacteria makes it more difficult to control or contain the spread of infectious diseases, because it undermines the effectiveness of treatments. It also substantially increases the cost of healthcare, the burdens placed on society by prolonging treatment periods and the likelihood of death. The WHO declared that the spread of antibiotic-resistant bacteria “threatens the achievements of modern medicine” (Antimicrobial Resistance: Global Report on Surveillance 2014, WHO, 2014). Antimicrobial resistance already causes 700,000 deaths every year, which is expected to increase to 10 million annually by 2050 (An international legal framework to address antimicrobial resistance, WHO, 2015)
        </P><H1 class="sub-title align-c m80">Existing Methods Used for Detection</H1><P class="content-txt m20">The CDC’s efforts at outsmarting the antibiotic resistance focuses on 4 core actions: detect, respond, prevent and discover. The project is called the AR Initiative (Detect and Protect Against Antibiotic Resistance Initiative), and it is is an integral part of the CDC’s strategy to target investment aimed at AR. Among the AR initiative, detection is the first step that impacts the whole controlling process. Detecting antibiotic resistance quickly and effectively is crucial for determining treatment methods for different patients as well as for quarantines to prevent it from becoming epidemic. Currently, several methods are used for the detection of the antibiotic resistance. The most common and traditional method is using growth inhibition assays performed in broth or by agar disc diffusion. For clinically critical bacteria, diagnostic laboratories perform phenotypic-based analyses using standardized susceptibility testing methods, usually in accordance with the guidelines published by the Clinical and Laboratory Standards Institute.


Using the culture-based approach, it can take 1—2 days to produce results for fast-growing bacteria such as Escherichia coli or Salmonella, but several weeks for slow-growing bacteria such as Mycobacterium tuberculosis. Moreover, culturing only works for a small fraction of microbes; although most pathogens can be cultured relatively easily thanks to years of accumulated experimental experiences, the vast majority of microbes cannot grow outside their host environment, including pathogens such as Chlamydia or Trypanosomes.


Using newer molecular detection techniques for antibiotic resistance such as quantitative PCR (qPCR) or microarrays, we can determine the presence of specific resistance genes within hours, and obtain improved diagnosis results. However, these culture-independent approaches target well-known and well-studied pathogens or resistance-causing genes only, and cannot be easily used for broader spectrum screening.



The CDC dramatically innovated the detection process by adopting the Advanced Molecular Detection (AMD) technique, which combines the latest pathogen identification technologies with bioinformatics and advanced epidemiology to more effectively understand, prevent and control infectious diseases. Using those technologies, it is possible to rapidly look for a microbe's match among thousands of reference samples in a microbe library. If no match is found, the whole genomic sequence of the microbe's DNA code can be taken, then quickly analyzed using disease detective works and bioinformatics to answer critical disease-response questions. However, this new method, despite being an interesting area of study, is not to be completed until 2020, still requires incubation, and is very expensive.
 </P><H1 class="sub-title align-c m80">Our Hypothesis: Possibility of Using QuorumSensing for Early Detection</H1><P class="content-txt m20 mb20">Our team, Elan Vital Korea, addressed the problem of rapidly detecting antibiotic-resistant bacteria. We were interested in a rapid and efficient method of antibiotic resistance detection, and we believed that such a method could be engineered using quorum sensing. Our hypothesis was that we would be able to use quorum sensing – a method bacteria use to communicate with each other – to make the cells quickly report the existence of antibiotic-resistant bacteria.
By quorum sensing, bacteria can perform many cooperative functions, such as biofilm formation, antibiotic production, motility, swarming, virulence, and much more. While most quorum sensing takes place between bacteria of the same species, there are cases of interspecies quorum sensing. Auto-inducers affect the gene expression of the bacteria once they reach a certain concentration threshold. Bacteria using quorum sensing usually produce small amounts of auto-inducers, so that the concentration of auto-inducers are affected by the concentration of the bacteria. In other words, quorum sensing, in essence, regulates gene expression in response to cell density. Using quorum sensing, bacteria are able to act in unison, as if they were a single organism.
Quorum sensing is widely used by various bacteria for various functions, so each uses a slightly different auto-inducer so the signals are not mixed up. In general, gram-negative bacteria use a class of molecules called N-acyl homoserine lactones, or AHL, while gram-positive bacteria use short processed polypeptides.
For example, the picture below represents the quorum sensing mechanism in the bacteria vibrio fisheri. 
Vibrio fisheri is a bacteria that produces bioluminescence, and is famous for revealing quorum sensing for the first time. Vibrio fisheri uses quorum sensing to produce light in high cell density, and researchers first discovered quorum sensing from examining it.
Quorum sensing in vibrio fishri involves LuxI and LuxR as well as AHL. LuxI is the protein that produces AHL, and LuxR forms a complex with AHL to affect the regulation of genes. In this case, it produces luciferase, which produces bioluminescence. Furthermore, the process also boosts the production of LuxI, which creates a positive feedback loop. This AHL-LuxR quorum sensing mechanism is one of the most well known gram-negative quorum sensing pathways, and it can be engineered to affect almost any coding sequence we'd like to.</P><P class="content-txt m20">For the project, we have developed a reporter cell that expresses GFP in the presence of the QS signaling molecule acyl homoserine lactone (AHL). Our test cells (which act as a simulation of antibiotic-resistant bacteria) express lactonase, which breaks down AHL. In our experimental system, test cells should signify their presence by breaking down AHL and preventing GFP expression in reporter cells.</P><H1 class="sub-title align-c m80">Experiment: Process and Results</H1><P class="content-txt m20 mb20">There are many ways of utilizing quorum sensing for medicinal use, and one of the most intuitive and most well-known methods is quorum quenching.
Quorum quenching takes advantage of the fact that quorum sensing also plays a role in expressing virulence, and interferes with the quorum sensing that produces virulence. There are many ways of utilizing quorum sensing for medicinal use, and one of the most intuitive and most well-known methods is quorum quenching.
Quorum quenching takes advantage of the fact that quorum sensing also plays a role in expressing virulence, and interferes with the quorum sensing that produces virulence. However, for our project this year, we decided to focus on engineering a detection method for antibiotic resistance. For the project, we created a test plasmid and a reporter plasmid. We then transformed competent E. coli with the plasmids to produce a test cell and a reporter cell. As shown in the picture below, the test cell produces lactonase, which breaks down AHL, a common auto-inducer in gram-negative bacteria. And the reporter cell produces GFP (or luciferase) which creates a visible difference that we can detect. Both plasmids were engineered using the BioBrick DNA recombination process. With such a set up, it will be possible to detect the presence of the test cell, or lactonase.
For the confirmation of our hypothesis, we conducted some experiments. Ideally, mixing AHL with the test cell will break down the AHL. And, adding the reporter after that will not result in any fluorescence. But, if we do the same process with the control bacteria instead of the test cell, there will be fluorescence. As theorized, the control experiments produced fluorescence, but the experiments with the test cell produced no fluorescence.</P><H1 class="sub-title align-c m80">Expected Benefits</H1><P class="content-txt m20">Thanks to bacteria’s ability to make quick and profound changes in gene transcription, quorum sensing can be used to detect a low amount of signaling molecules and report their presence quickly. With further research and thorough engineering applications, it may be possible to detect other antibiotic-resistant bacteria that are unknown until now.
If proven as valid and effective through sufficient tests, this technique could be disseminated to hospitals and clinics to test the presence of antibiotic-resistant bacteria. We hope that this technique, once if properly adjusted for functional advancement, can detect antibiotic-resistant bacteria in a relatively short time with only a small amount of sample secured from the patient. This would provide an advantage over the traditional detection methods, culture-based approaches which require one or several days of incubation period.
Because the chemicals involved in species-specific quorum sensing are very specific, it might be possible to dramatically resolve the problem of overnight incubation. Because an initial sample from a patient is usually contaminated and has only a small concentration of the wanted bacteria, it is often impossible to detect any antibiotic-resistance without purification and amplification through overnight incubation. But because species-specific quorum sensing involves biochemical that are highly specific, and the quorum sensing chemicals are not affected as much by the contamination, the method utilizing quorum sensing might be applied with relatively less purification processes. Also, because some quorum sensing mechanisms have built in positive feedback, with the right engineering, the mechanism could work with only a little amplification process.
More innovative detection methods such as quantitative PCR(qPCR) or microarrays, and advanced molecular detection (AMD) are based on accumulated previous data and, thus, render very accurate results, but they require complicated procedures and heavy equipment. On the other hand, this quorum sensing-based detection method will provide benefits to patients with handy procedure and quicker detection results. We believe quicker and easy detection of antibiotic-resistant bacteria will lead to better containment of such dangerous bacterial strains.</P><H1 class="sub-title align-c m80">Reference</H1><P class="content-txt m20">Antibiotic Resistance Threat in the United States 2013, US Department of Health and Human Services, Center for Disease Control and Prevention About Quorum Sensing
Annual Review of Microbiology, Volume 55:pp 165-199 (volume publication date, October 2001) Melissa B.Miller and Bonnie L. 
Bassler Department of Molecular Biology, Princeton University, Princeton, New Jersey 
Bacterial Quorum Sensing: Its Role in Virulence and Possibilities for Its Control Steven T. Rutherfold and Bonnie L.Bassler. 
Cold Spring Harb Perspect Med. 2012.2, Cold Spring Harbor Laboratory Press
Quorum Sensing: Bacteria Talk Sense Costi D. Sifri, Oxford Journals, Volume 47, Issue 8 Pp 1070-1076, 2015
Infectious Diseases Society of America
Bacterial Quorum Sensing in Pathogenic Relationships Teresa R. de Kievit, Barbara H.Iglewski, Infection and Immunity, 
Volume 68, September 2000, 2000 American Society for Microbiology</P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>