Team:NEU-China/PartOR1A1
Toggle navigation
Home
Project
Background
Description
Design
Experiment & Results
Demostrate
Notebook
Progress
Protocol
Parts
Our Parts
Biobricks For Medal
Basic Parts
Safety
InterLab
Collaborations
Human Practices
Human Practices
Silver
Gold
Lab
Team
Attributions
Lab Life
Part OR1A1 Document
Part: BBa_K2288000
Designed by: Ni Zong Group: iGEM17_NEU-China (2017-10-09)
Rho-OR1A1(HEK293)
Encoding the fragments of the olfactory receptor OR1A1 with Rho-tag in its N-terminal in HEK293 cells. Once the specific                                         odor, β-citronellol, binds to OR1A1 on the cell membrane, it leads to a series of                                         signaling rally and consequently generates a flux of cAMP which could be detected                                         by a cAMP-activated reporter gene system. The Rho tag was added to its N-terminal,                                         which could improve the olfactory receptor to express on the surface of HEK293.
The whole OR1A1 signaling exists primarily in the olfactory epithelia cells which                                         are not easily cultured and amplified to generate the ex vivo biosensor, nor does                                         a simplified prokaryote can reconstitute the whole OR1A1 pathway. Therefore we chose                                         an easily cultured eukaryotic HEK293FT cells instead to reconstitute the olfactory                                         signaling cascade for the biosensor with a CRE-luciferase construct as the reporter.
Function
Receptor
RFC standard
RFC 10
Backbone
pSB1C3
Organism
Human
Source
Human HEK293FT
Submitted by
NEU-China 2017
Usage and Biology
Protein data table for BioBrick BBa_ automatically created by the BioBrick-AutoAnnotator version 1.0
Nucleotide sequence in RFC 10: (underlined part encodes the protein)
  ATGACCGAG ... ATCTCCTCGTAA
  ORF from nucleotide position 1 to 960 (excluding stop-codon)
Amino acid sequence: (RFC 25 scars in shown in bold, other sequence features                                     underlined; both given below)
1 
101 
201 
301 
MTETSQVAPAGGRENNQSSTLEFILLGVTGQQEQEDFFYILFLFIYPITLIGNLLIVLAICSDVRLHNPMYFLLANLSLVDIFFSSVTIPKMLANHLLGS
KSISFGGCLTQMYFMIALGNTDSYILAAMAYDRAVAISRPLHYTTIMSPRSCIWLIAGSWVIGNANALPHTLLTASLSFCGNQEVANFYCDITPLLKLSC
SDIHFHVKMMYLGVGIFSVPLLCIIVSYIRVFSTVFQVPSTKGVLKAFSTCGSHLTVVSLYYGTVMGTYFRPLTNYSLKDAVITVMYTAVTPMLNSFIYS
LRNRDMKAALRKLFNKRISS*
Sequence features: (with their position in the amino acid sequence, see the list of supported features )
None of the supported features appeared in the sequence
Amino acid composition:
Ala (A)
21 (6.6%)
Arg (R)
11 (3.4%)
Asn (N)
15 (4.7%)
Asp (D)
9 (2.8%)
Cys (C)
8 (2.5%)
Gln (Q)
8 (2.5%)
Glu (E)
6 (1.9%)
Gly (G)
18 (5.6%)
His (H)
7 (2.2%)
Ile (I)
26 (8.1%)
Leu (L)
41 (12.8%)
Lys (K)
10 (3.1%)
Met (M)
13 (4.1%)
Phe (F)
20 (6.3%)
Pro (P)
12 (3.8%)
Ser (S)
31 (9.7%)
Thr (T)
23 (7.2%)
Trp (W)
2 (0.6%)
Tyr (Y)
16 (5.0%)
Val (V)
23 (7.2%)
Amino acid counting
Total number:
320
Positively charged (Arg+Lys):
21 (6.6%)
Negatively charged (Asp+Glu):
15 (4.7%)
Aromatic (Phe+His+Try+Tyr):
45 (14.1%)
Biochemical parameters
Atomic composition:
C 1630H 2548N 402O 444S 21
Molecular mass [Da]:
35553.8
Theoretical pI:
8.85
Extinction coefficient at 280 nm [M -1 cm -1]:
34840 / 35340 (all Cys red/ox)
Plot for hydrophobicity, charge, predicted secondary structure, solvent accessability,                                         transmembrane helices and disulfid bridges 
Codon usage
Organism:
E. coli
B. subtilis
S. cerevisiae
A. thaliana
P. patens
Mammals
Codon quality ( CAI ):
good (0.69)
good (0.70)
good (0.61)
good (0.70)
excellent (0.81)
good (0.75)
Alignments (obtained from PredictProtein.org )
   There were no alignments for this protein in the data base. The BLAST                                     search was initialized and should be ready in a few hours. Predictions (obtained from PredictProtein.org )    There were no predictions for this protein in the data base. The prediction                                     was initialized and should be ready in a few hours. The BioBrick-AutoAnnotator was created by TU-Munich 2013 iGEM team. For more information please see the documentation .
If you have any questions, comments or suggestions, please leave us a comment .
How to use
If you want to use it, you should digest it with the enzyme PstI and XbaI to get the                                         fragments. Then you need to add the restriction sites on both sides of the sequence                                         by PCR. A pair of reference primers below is provided.
Forward: cgtaagcttatgaccgagacatctcaggtggcccctgccggcggcagggaaaataac
Reverse: tatggatccttacgaggagattctcttgttg
Most importantly, you need to clone the parts into the commonly used eukaryotic                                         expression vector pcDNA3.1+. As shown, you also need to digest the vector with HindIII                                         and BamHI endonuclease in advance.
Sequencing results of OR1A1
Proof of expression
Sensitivity Testing
Test the sensitivity and specificity of this basic odor biosensors with their cognate                                         odors β-citronellol by qPCR.
Cloning into lenti sgRNA(MS2)-puro backbone vector for CRISPRa（gRNA）
Referencing to some papers, we knew that there are three genes which can help the olfactory                                         receptors anchored in the membrane. So we aimed to activate these three key accessory                                         protein to enhance the express of odor receptors.
Part
Plasmid Construction
Transformation to Host Strain
GNAL
Lenti MS2
E.coli.DH5α
RIC8B
Lenti MS2
E.coli.DH5α
RTP1
Lenti MS2
E.coli.DH5α
Virus infection experiment
First, we transfected HEK293 with lenti dCAS9-VP64-Blast, lenti MS2-P65-HSF1-Hygro and                                         lenti sgRNA to produce virus.
Then, we infected HEK293 with the virus of dCAS9-VP64-Blast and MS2-P65-HSF1-Hygro                                         to construct stable cell lines. Based on these cell lines, we infected different                                         sgRNAs of GNAL, RIC8B and RTP1 respectively.
This are the results of real-time PCR.
These three graphs show that our sgRNAs has successfully activated the three genes                                         that mentioned before.
As for the cAMP-activated reporter gene system, we choose Forskolin to induce expression                                         of the reporter gene, and expression of the reporter protein luciferase was confirmed                                         by treatment with 10 millimole（mM）forskolin.
After a 24 h incubation, cracked the cell and added the substrate reacting with luciferase,                                         then detected it by fluorescent microplate reader. Here is the data we got. Cells                                         expressing luciferase treated only with 0.1%DMSO was used as a negative control.                                         We can see that the luminescence data gets significantly improved, which proves that                                         the reporter gene we choose is available.
Here comes the most essential experiment. We infected the stable cell lines with                                         all virus to construct a super cell line, which has a high expression of the three                                         genes. Then we transfected OR1A1 and OR1D2 respectively in the super cell, and stimulated                                         with β-citronellol and bourgeonal, which can recognized by OR1A1 and OR1D2 respectively. At last, we used Firefly Luciferase Assay Kit to test the cyclic AMP (cAMP) signal                                         pathway.
This is the luminescence result of OR1A1.
OR1A1 biosensor evaluation with its cognate odor molecule
Here comes the most essential experiment. We infected the stable cell lines with all                                         virus to construct a super cell line, which has a high expression of the three genes.                                         Then we transfected OR1A1 and OR1D2 respectively in the super cell, and stimulated                                         with β-citronellol and bourgeonal, which can recognized by OR1A1 and OR1D2 respectively.                                         At last, we used Firefly Luciferase Assay Kit to test the cyclic AMP (cAMP) signal                                         pathway.
This is the luminescence result of OR1A1.
Conclusion：By using the Luciferase Assay Kit,we can see that the luminescence                                         of super cell transfected with OR1A1 is significantly higher than that in the control                                         group——the normal 293FT and odorless stimulated control group, which confirming the                                         feasibility of the biosensor we built.
Copyright © 2017 NEU-CHINA Team. All Rights Reserved.
