Team:NEU-China/PartOR1D2
Toggle navigation
Home
Project
Background
Description
Design
Experiment & Results
Demostrate
Notebook
Progress
Protocol
Parts
Our Parts
Biobricks For Medal
Basic Parts
Safety
InterLab
Collaborations
Human Practices
Human Practices
Silver
Gold
Lab
Team
Attributions
Lab Life
Part OR1D2 Document
Part:BBa_K2288001
Designed by: Jiawen Yang Group: iGEM17_NEU-China (2017-10-10)
Rho-OR1D2(HEK293)
Encoding the fragments of the olfactory receptor OR1D2 with Rho-tag in its N-terminal in HEK293 cells. Once the specific                                         odor, bourgeonal, binds to OR1D2 on the cell membrane, it leads to a series of signaling                                         rally and consequently generates a flux of cAMP which could be detected by a cAMP-activated                                         reporter gene system. The Rho tag was added to its N-terminal, which could improve                                         the olfactory receptor to express on the surface of HEK293.
The whole OR1D2 signaling exists primarily in the olfactory epithelia cells which                                         are not easily cultured and amplified to generate the ex vivo biosensor, nor does                                         a simplified prokaryote can reconstitute the whole OR1D2 pathway. Therefore we chose                                         an easily cultured eukaryotic HEK293FT cells instead to reconstitute the olfactory                                         signaling cascade for the biosensor with a CRE-luciferase construct as the reporter.
Function
Receptor
RFC standard
RFC 10
Backbone
pSB1C3
Organism
Human
Source
Human HEK293FT
Submitted by
NEU-China 2017
Usage and Biology
Protein data table for BioBrick BBa_K2288001 automatically created by the BioBrick-AutoAnnotator version 1.0
Nucleotide sequence in RFC 10: (underlined part encodes the protein)
  ATGACCGAG ... AGGCTGACATAA
  ORF from nucleotide position 1 to 969 (excluding stop-codon)
Amino acid sequence: (RFC 25 scars in shown in bold, other sequence features                                     underlined; both given below)
1 
101 
201 
301 
MTETSQVAPAGGDGGNQSEGSEFLLLGMSESPEQQQILFWMFLSMYLVTVVGNVLIILAISSDSRLHTPVYFFLANLSFTDLFFVTNTIPKMLVNLQSHN
KAISYAGCLTQLYFLVSLVALDNLILAVMAYDRYVAICCPLHYTTAMSPKLCILLLSLCWVLSVLYGLIHTLLMTRVTFCGSRKIHYIFCEMYVLLRMAC
SNIQINHTVLIATGCFIFLIPFGFVIISYVLIIRAILRIPSVSKKYKAFSTCASHLGAVSLFYGTLCMVYLKPLHTYSVKDSVATVMYAVVTPMMNPFIY
SLRNKDMHGALGRLLDKHFKRLT*
Sequence features: (with their position in the amino acid sequence, see the list of supported features )
None of the supported features appeared in the sequence
Amino acid composition:
Ala (A)
20 (6.2%)
Arg (R)
10 (3.1%)
Asn (N)
11 (3.4%)
Asp (D)
8 (2.5%)
Cys (C)
11 (3.4%)
Gln (Q)
8 (2.5%)
Glu (E)
6 (1.9%)
Gly (G)
16 (5.0%)
His (H)
10 (3.1%)
Ile (I)
24 (7.4%)
Leu (L)
48 (14.9%)
Lys (K)
12 (3.7%)
Met (M)
15 (4.6%)
Phe (F)
19 (5.9%)
Pro (P)
11 (3.4%)
Ser (S)
27 (8.4%)
Thr (T)
21 (6.5%)
Trp (W)
2 (0.6%)
Tyr (Y)
17 (5.3%)
Val (V)
27 (8.4%)
Amino acid counting
Total number:
323
Positively charged (Arg+Lys):
22 (6.8%)
Negatively charged (Asp+Glu):
14 (4.3%)
Aromatic (Phe+His+Try+Tyr):
48 (14.9%)
Biochemical parameters
Atomic composition:
C 1671H 2624N 406O 436S 26
Molecular mass [Da]:
36211.2
Theoretical pI:
8.90
Extinction coefficient at 280 nm [M -1 cm -1]:
36330 / 37018 (all Cys red/ox)
Plot for hydrophobicity, charge, predicted secondary structure, solvent accessability,                                         transmembrane helices and disulfid bridges 
Codon usage
Organism:
E. coli
B. subtilis
S. cerevisiae
A. thaliana
P. patens
Mammals
Codon quality ( CAI ):
good (0.68)
good (0.69)
acceptable (0.57)
good (0.67)
excellent (0.81)
excellent (0.80)
Alignments (obtained from PredictProtein.org )
   There were no alignments for this protein in the data base. The BLAST                                     search was initialized and should be ready in a few hours. Predictions (obtained from PredictProtein.org )    There were no predictions for this protein in the data base. The prediction                                     was initialized and should be ready in a few hours. The BioBrick-AutoAnnotator was created by TU-Munich 2013 iGEM team. For more information please see the documentation .
If you have any questions, comments or suggestions, please leave us a comment .
How to use
If you want to use it, you should digest it with the enzyme PstI and XbaI to get the                                         fragments. Then you need to add the restriction sites on both sides of the sequence                                         by PCR. A pair of reference primers below is provided.
Forward: cgtaagcttatgaccgagacatctcaggtggcccctgccggcggcgatggaggcaac
Reverse: tatctcgagttatgtcagcctcttaaagtgtttatctagg
Most importantly, you need to clone the parts into the commonly used eukaryotic                                         expression vector pcDNA3.1+. As shown, you also need to digest the vector with HindIII                                         and XhoI endonuclease in advance.
Sequencing results of OR1D2
Proof of expression
Sensitivity Testing
Test the sensitivity and specificity of this basic odor biosensors with their cognate                                         odors bourgeonal by qPCR.
Copyright © 2017 NEU-CHINA Team. All Rights Reserved.
