<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NEU-China_PartOR1D2 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NEU-China/PartOR1D2</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar navbar-inverse" role="banner"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2017.igem.org/Team:NEU-China">Home</A></LI><LI class="dropdown"><A href="#">Project
                                </A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/Background">Background</A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/Experiments">Experiment &amp; Results</A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/Demonstrate">Demostrate</A></LI></UL><LI class="dropdown"><A href="#">Notebook
                                </A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/Progress">Progress</A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/Protocol">Protocol</A></LI><LI class="dropdown"><A href="#">Parts
                                </A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/Parts">Our Parts</A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/BiobricksForMedal">Biobricks For Medal</A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/InterLab">InterLab </A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/Collaborations">Collaborations </A></LI><LI class="dropdown"><A href="#">Human Practices
                                </A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/Human_Practices">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/HP/Gold_Integrated">Gold</A></LI><LI class="dropdown"><A href="#">Lab
                                </A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/LabLife">Lab Life</A></LI></DIV></DIV></DIV><SECTION id="page-breadcrumb"><DIV class="vertical-center sun"><DIV class="container"><DIV class="row"><DIV class="action"><DIV class="col-sm-12"><H1 class="title">Part OR1D2 Document</H1></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION id="blog-details" class="padding-top" style="padding-top: 0px;"><DIV class="container"><DIV class="row"><DIV class="col-md-12 col-sm-12" style="width: auto;"><DIV class="row"><DIV class="col-md-12 col-sm-12"><DIV class="single-blog blog-details two-column"><DIV class="post-content overflow"><H2>
                                        Part:BBa_K2288001


                                    </H2><H3>

                                        Designed by: Jiawen Yang Group: iGEM17_NEU-China (2017-10-10)



                                    </H3><H2>
                                        Rho-OR1D2(HEK293)



                                    </H2><H3>
                                        Encoding the fragments of the olfactory receptor OR1D2 with Rho-tag in its N-terminal in HEK293 cells. Once the specific
                                        odor, bourgeonal, binds to OR1D2 on the cell membrane, it leads to a series of signaling
                                        rally and consequently generates a flux of cAMP which could be detected by a cAMP-activated
                                        reporter gene system. The Rho tag was added to its N-terminal, which could improve
                                        the olfactory receptor to express on the surface of HEK293.
       The whole OR1D2 signaling exists primarily in the olfactory epithelia cells which
                                        are not easily cultured and amplified to generate the ex vivo biosensor, nor does
                                        a simplified prokaryote can reconstitute the whole OR1D2 pathway. Therefore we chose
                                        an easily cultured eukaryotic HEK293FT cells instead to reconstitute the olfactory
                                        signaling cascade for the biosensor with a CRE-luciferase construct as the reporter.



                                    </H3><TABLE class="table table-striped text-center table-hover table-responsive" style="font-size: 14px"><THEAD><TR><TH style="text-align: center">
                                                    Function
                                                </TH><TH style="text-align: center">
                                                    Receptor
                                                </TH></TR></THEAD><TBODY><TR><TD style="text-align: center">RFC standard</TD><TD>RFC 10</TD></TR><TR><TD style="text-align: center">Backbone</TD><TD>pSB1C3</TD></TR><TR><TD style="text-align: center">Organism</TD><TD>Human</TD></TR><TR><TD style="text-align: center">Source</TD><TD>Human HEK293FT</TD></TR><TR><TD style="text-align: center">Submitted by</TD><TD>NEU-China 2017</TD></TR></TBODY></TABLE><H2>
                                        Usage and Biology

                                    </H2></DIV></DIV></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION id="blog-details" class="padding-top" style="padding-top: 0px;"><DIV class="container"><DIV class="row"><DIV class="col-md-12 col-sm-12" style="width: auto;"><DIV id="AutoAnnotator_container_1508918687956"><TABLE id="AutoAnnotator"><TBODY><TR><TH id="AutoAnnotatorHeader" colspan="2">Protein data table for BioBrick
                                    <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2288001">BBa_K2288001

                                    </A> automatically created by the
                                    <A href="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator">BioBrick-AutoAnnotator</A> version 1.0</TH></TR><TR><TD class="AutoAnnotator1col" colspan="2"><STRONG>Nucleotide sequence</STRONG> in
                                    <STRONG>RFC 10</STRONG>: (underlined part encodes the protein)
                                    <SPAN class="AutoAnnotatorSequence"> <U>ATGACCGAG ... AGGCTGACA</U>TAA</SPAN> <STRONG>ORF</STRONG> from nucleotide position 1 to 969 (excluding stop-codon)</TD></TR><TR><TD class="AutoAnnotator1col" colspan="2"><STRONG>Amino acid sequence:</STRONG> (RFC 25 scars in shown in bold, other sequence features
                                    underlined; both given below)
                                    </TD></TR><TR><TD class="AutoAnnotatorSeqNum">1 101 201 301 </TD><TD class="AutoAnnotatorSeqSeq">MTETSQVAPAGGDGGNQSEGSEFLLLGMSESPEQQQILFWMFLSMYLVTVVGNVLIILAISSDSRLHTPVYFFLANLSFTDLFFVTNTIPKMLVNLQSHN
                                                    KAISYAGCLTQLYFLVSLVALDNLILAVMAYDRYVAICCPLHYTTAMSPKLCILLLSLCWVLSVLYGLIHTLLMTRVTFCGSRKIHYIFCEMYVLLRMAC
                                                    SNIQINHTVLIATGCFIFLIPFGFVIISYVLIIRAILRIPSVSKKYKAFSTCASHLGAVSLFYGTLCMVYLKPLHTYSVKDSVATVMYAVVTPMMNPFIY
                                                    SLRNKDMHGALGRLLDKHFKRLT*</TD></TR></TBODY></TABLE><STRONG>Sequence features:</STRONG> (with their position in the amino acid sequence, see the
                                    <A href="https://2013.igem.org/Team:TU-Munich/Results/Software/FeatureList">list of supported features</A>)
                                    <TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorSeqFeat2b">None of the supported features appeared in the sequence</TD></TR></TBODY></TABLE><STRONG>Amino acid composition:</STRONG><TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">Ala (A)</TD><TD class="AutoAnnotatorInnerAmino">20 (6.2%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Arg (R)</TD><TD class="AutoAnnotatorInnerAmino">10 (3.1%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Asn (N)</TD><TD class="AutoAnnotatorInnerAmino">11 (3.4%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Asp (D)</TD><TD class="AutoAnnotatorInnerAmino">8 (2.5%)</TD></TR></TBODY></TABLE><TABLE class="AutoAnnotatorWithBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">Cys (C)</TD><TD class="AutoAnnotatorInnerAmino">11 (3.4%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Gln (Q)</TD><TD class="AutoAnnotatorInnerAmino">8 (2.5%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Glu (E)</TD><TD class="AutoAnnotatorInnerAmino">6 (1.9%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Gly (G)</TD><TD class="AutoAnnotatorInnerAmino">16 (5.0%)</TD></TR></TBODY></TABLE><TABLE class="AutoAnnotatorWithBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">His (H)</TD><TD class="AutoAnnotatorInnerAmino">10 (3.1%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Ile (I)</TD><TD class="AutoAnnotatorInnerAmino">24 (7.4%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Leu (L)</TD><TD class="AutoAnnotatorInnerAmino">48 (14.9%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Lys (K)</TD><TD class="AutoAnnotatorInnerAmino">12 (3.7%)</TD></TR></TBODY></TABLE><TABLE class="AutoAnnotatorWithBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">Met (M)</TD><TD class="AutoAnnotatorInnerAmino">15 (4.6%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Phe (F)</TD><TD class="AutoAnnotatorInnerAmino">19 (5.9%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Pro (P)</TD><TD class="AutoAnnotatorInnerAmino">11 (3.4%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Ser (S)</TD><TD class="AutoAnnotatorInnerAmino">27 (8.4%)</TD></TR></TBODY></TABLE><TABLE class="AutoAnnotatorWithBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">Thr (T)</TD><TD class="AutoAnnotatorInnerAmino">21 (6.5%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Trp (W)</TD><TD class="AutoAnnotatorInnerAmino">2 (0.6%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Tyr (Y)</TD><TD class="AutoAnnotatorInnerAmino">17 (5.3%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Val (V)</TD><TD class="AutoAnnotatorInnerAmino">27 (8.4%)</TD></TR></TBODY></TABLE><STRONG>Amino acid counting</STRONG><TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorAminoCountingInner2">Total number:</TD><TD class="AutoAnnotatorAminoCountingInner3">323</TD></TR><TR><TD class="AutoAnnotatorAminoCountingInner2">Positively charged (Arg+Lys):</TD><TD class="AutoAnnotatorAminoCountingInner3">22 (6.8%)</TD></TR><TR><TD class="AutoAnnotatorAminoCountingInner2">Negatively charged (Asp+Glu):</TD><TD class="AutoAnnotatorAminoCountingInner3">14 (4.3%)</TD></TR><TR><TD class="AutoAnnotatorAminoCountingInner2">Aromatic (Phe+His+Try+Tyr):</TD><TD class="AutoAnnotatorAminoCountingInner3">48 (14.9%)</TD></TR></TBODY></TABLE><STRONG>Biochemical parameters</STRONG><TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorBiochemParInner2">Atomic composition:</TD><TD class="AutoAnnotatorBiochemParInner3">C
                                                <SUB>1671</SUB>H
                                                <SUB>2624</SUB>N
                                                <SUB>406</SUB>O
                                                <SUB>436</SUB>S
                                                <SUB>26</SUB></TD></TR><TR><TD class="AutoAnnotatorBiochemParInner2">Molecular mass [Da]:</TD><TD class="AutoAnnotatorBiochemParInner3">36211.2</TD></TR><TR><TD class="AutoAnnotatorBiochemParInner2">Theoretical pI:</TD><TD class="AutoAnnotatorBiochemParInner3">8.90</TD></TR><TR><TD class="AutoAnnotatorBiochemParInner2">Extinction coefficient at 280 nm [M
                                                <SUP>-1</SUP> cm
                                                <SUP>-1</SUP>]:</TD><TD class="AutoAnnotatorBiochemParInner3">36330 / 37018 (all Cys red/ox)</TD></TR></TBODY></TABLE><STRONG>Plot for hydrophobicity, charge, predicted secondary structure, solvent accessability,
                                        transmembrane helices and disulfid bridges</STRONG> <STRONG>Codon usage</STRONG><TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorCodonUsage2">Organism:</TD><TD class="AutoAnnotatorCodonUsage3"><I>E. coli</I></TD><TD class="AutoAnnotatorCodonUsage3"><I>B. subtilis</I></TD><TD class="AutoAnnotatorCodonUsage3"><I>S. cerevisiae</I></TD><TD class="AutoAnnotatorCodonUsage3"><I>A. thaliana</I></TD><TD class="AutoAnnotatorCodonUsage3"><I>P. patens</I></TD><TD class="AutoAnnotatorCodonUsage3">Mammals</TD></TR><TR><TD class="AutoAnnotatorCodonUsage2">Codon quality (
                                                <A href="http://en.wikipedia.org/wiki/Codon_Adaptation_Index">CAI</A>):</TD><TD class="AutoAnnotatorCodonUsage3">good (0.68)</TD><TD class="AutoAnnotatorCodonUsage3">good (0.69)</TD><TD class="AutoAnnotatorCodonUsage3">acceptable (0.57)</TD><TD class="AutoAnnotatorCodonUsage3">good (0.67)</TD><TD class="AutoAnnotatorCodonUsage3">excellent (0.81)</TD><TD class="AutoAnnotatorCodonUsage3">excellent (0.80)</TD></TR></TBODY></TABLE><STRONG>Alignments</STRONG> (obtained from
                                    <A href="http://predictprotein.org">PredictProtein.org</A>)
                                       There were no alignments for this protein in the data base. The BLAST
                                    search was initialized and should be ready in a few hours.<STRONG>Predictions</STRONG> (obtained from
                         <A href="http://predictprotein.org">PredictProtein.org</A>)   There were no predictions for this protein in the data base. The prediction
                                    was initialized and should be ready in a few hours. The BioBrick-AutoAnnotator was created by
                                        <A href="https://2013.igem.org/Team:TU-Munich">TU-Munich 2013</A> iGEM team. For more information please see the
                                        <A href="https://2013.igem.org/Team:TU-Munich/Results/Software">documentation</A>.
                                        If you have any questions, comments or suggestions, please leave us a
                                        <A href="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator">comment</A>.</DIV></DIV></DIV></DIV></SECTION><SECTION id="blog-details" class="padding-top" style="padding-top: 0px;"><DIV class="container"><DIV class="row"><DIV class="col-md-12 col-sm-12" style="width: auto;"><DIV class="row"><DIV class="col-md-12 col-sm-12"><DIV class="single-blog blog-details two-column"><DIV class="post-content overflow"><H2>
                                        How to use


                                    </H2><H3>If you want to use it, you should digest it with the enzyme PstI and XbaI to get the
                                        fragments. Then you need to add the restriction sites on both sides of the sequence
                                        by PCR. A pair of reference primers below is provided.
                                         Forward: cgtaagcttatgaccgagacatctcaggtggcccctgccggcggcgatggaggcaac
                                         Reverse: tatctcgagttatgtcagcctcttaaagtgtttatctagg
                                         Most importantly, you need to clone the parts into the commonly used eukaryotic
                                        expression vector pcDNA3.1+. As shown, you also need to digest the vector with HindIII
                                        and XhoI endonuclease in advance.


                                    </H3><DIV align="center"><P style="text-align:center">Sequencing results of OR1D2 </P></DIV><H2>Proof of expression
                                         Sensitivity Testing

                                    </H2><H3>Test the sensitivity and specificity of this basic odor biosensors with their cognate
                                        odors bourgeonal by qPCR.


                                    </H3></DIV></DIV></DIV></DIV></DIV></DIV></DIV></SECTION><FOOTER id="footer"><DIV class="container"><DIV class="row"><DIV class="col-sm-12"><DIV class="copyright-text text-center"><P>Copyright © 2017 NEU-CHINA Team. All Rights Reserved.</P></DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>