<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NEU-China_PartOR1A1 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NEU-China/PartOR1A1</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar navbar-inverse" role="banner"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2017.igem.org/Team:NEU-China">Home</A></LI><LI class="dropdown"><A href="#">Project
                                </A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/Background">Background</A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/Experiments">Experiment &amp; Results</A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/Demonstrate">Demostrate</A></LI></UL><LI class="dropdown"><A href="#">Notebook
                                </A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/Progress">Progress</A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/Protocol">Protocol</A></LI><LI class="dropdown"><A href="#">Parts
                                </A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/Parts">Our Parts</A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/BiobricksForMedal">Biobricks For Medal</A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/InterLab">InterLab </A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/Collaborations">Collaborations </A></LI><LI class="dropdown"><A href="#">Human Practices
                                </A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/Human_Practices">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/HP/Gold_Integrated">Gold</A></LI><LI class="dropdown"><A href="#">Lab
                                </A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:NEU-China/LabLife">Lab Life</A></LI></DIV></DIV></DIV><SECTION id="page-breadcrumb"><DIV class="vertical-center sun"><DIV class="container"><DIV class="row"><DIV class="action"><DIV class="col-sm-12"><H1 class="title">Part OR1A1 Document</H1></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION id="blog-details" class="padding-top" style="padding-top: 0px;"><DIV class="container"><DIV class="row"><DIV class="col-md-9 col-sm-7" style="width: auto;"><DIV class="row"><DIV class="col-md-12 col-sm-12"><DIV class="single-blog blog-details two-column"><DIV class="post-content overflow"><H2>
                                        Part: BBa_K2288000

                                    </H2><H3>

                                        Designed by: Ni Zong Group: iGEM17_NEU-China (2017-10-09)


                                    </H3><H2>
                                        Rho-OR1A1(HEK293)


                                    </H2><H3>
                                        Encoding the fragments of the olfactory receptor OR1A1 with Rho-tag in its N-terminal in HEK293 cells. Once the specific
                                        odor, β-citronellol, binds to OR1A1 on the cell membrane, it leads to a series of
                                        signaling rally and consequently generates a flux of cAMP which could be detected
                                        by a cAMP-activated reporter gene system. The Rho tag was added to its N-terminal,
                                        which could improve the olfactory receptor to express on the surface of HEK293.
              The whole OR1A1 signaling exists primarily in the olfactory epithelia cells which
                                        are not easily cultured and amplified to generate the ex vivo biosensor, nor does
                                        a simplified prokaryote can reconstitute the whole OR1A1 pathway. Therefore we chose
                                        an easily cultured eukaryotic HEK293FT cells instead to reconstitute the olfactory
                                        signaling cascade for the biosensor with a CRE-luciferase construct as the reporter.


                                    </H3><TABLE class="table table-striped text-center table-hover table-responsive" style="font-size: 14px"><THEAD><TR><TH style="text-align: center">
                                                    Function
                                                </TH><TH style="text-align: center">
                                                    Receptor
                                                </TH></TR></THEAD><TBODY><TR><TD style="text-align: center">RFC standard</TD><TD>RFC 10</TD></TR><TR><TD style="text-align: center">Backbone</TD><TD>pSB1C3</TD></TR><TR><TD style="text-align: center">Organism</TD><TD>Human</TD></TR><TR><TD style="text-align: center">Source</TD><TD>Human HEK293FT</TD></TR><TR><TD style="text-align: center">Submitted by</TD><TD>NEU-China 2017</TD></TR></TBODY></TABLE><H2>
                                        Usage and Biology

                                    </H2></DIV></DIV></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION id="blog-details" class="padding-top" style="padding-top: 0px;"><DIV class="container"><DIV class="row"><DIV class="col-md-12 col-sm-12" style="width: auto;"><DIV id="AutoAnnotator_container_1508919693721"><TABLE id="AutoAnnotator"><TBODY><TR><TH id="AutoAnnotatorHeader" colspan="2">Protein data table for BioBrick
                                    <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_&lt;!------------------------Enter BioBrick number here------------------------&gt;">BBa_
                                        </A> automatically created by the
                                    <A href="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator">BioBrick-AutoAnnotator</A> version 1.0</TH></TR><TR><TD class="AutoAnnotator1col" colspan="2"><STRONG>Nucleotide sequence</STRONG> in
                                    <STRONG>RFC 10</STRONG>: (underlined part encodes the protein)
                                    <SPAN class="AutoAnnotatorSequence"> <U>ATGACCGAG ... ATCTCCTCG</U>TAA</SPAN> <STRONG>ORF</STRONG> from nucleotide position 1 to 960 (excluding stop-codon)</TD></TR><TR><TD class="AutoAnnotator1col" colspan="2"><STRONG>Amino acid sequence:</STRONG> (RFC 25 scars in shown in bold, other sequence features
                                    underlined; both given below)
                                    </TD></TR><TR><TD class="AutoAnnotatorSeqNum">1 101 201 301 </TD><TD class="AutoAnnotatorSeqSeq">MTETSQVAPAGGRENNQSSTLEFILLGVTGQQEQEDFFYILFLFIYPITLIGNLLIVLAICSDVRLHNPMYFLLANLSLVDIFFSSVTIPKMLANHLLGS
                                                    KSISFGGCLTQMYFMIALGNTDSYILAAMAYDRAVAISRPLHYTTIMSPRSCIWLIAGSWVIGNANALPHTLLTASLSFCGNQEVANFYCDITPLLKLSC
                                                    SDIHFHVKMMYLGVGIFSVPLLCIIVSYIRVFSTVFQVPSTKGVLKAFSTCGSHLTVVSLYYGTVMGTYFRPLTNYSLKDAVITVMYTAVTPMLNSFIYS
                                                    LRNRDMKAALRKLFNKRISS*</TD></TR></TBODY></TABLE><STRONG>Sequence features:</STRONG> (with their position in the amino acid sequence, see the
                                    <A href="https://2013.igem.org/Team:TU-Munich/Results/Software/FeatureList">list of supported features</A>)
                                    <TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorSeqFeat2b">None of the supported features appeared in the sequence</TD></TR></TBODY></TABLE><STRONG>Amino acid composition:</STRONG><TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">Ala (A)</TD><TD class="AutoAnnotatorInnerAmino">21 (6.6%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Arg (R)</TD><TD class="AutoAnnotatorInnerAmino">11 (3.4%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Asn (N)</TD><TD class="AutoAnnotatorInnerAmino">15 (4.7%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Asp (D)</TD><TD class="AutoAnnotatorInnerAmino">9 (2.8%)</TD></TR></TBODY></TABLE><TABLE class="AutoAnnotatorWithBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">Cys (C)</TD><TD class="AutoAnnotatorInnerAmino">8 (2.5%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Gln (Q)</TD><TD class="AutoAnnotatorInnerAmino">8 (2.5%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Glu (E)</TD><TD class="AutoAnnotatorInnerAmino">6 (1.9%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Gly (G)</TD><TD class="AutoAnnotatorInnerAmino">18 (5.6%)</TD></TR></TBODY></TABLE><TABLE class="AutoAnnotatorWithBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">His (H)</TD><TD class="AutoAnnotatorInnerAmino">7 (2.2%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Ile (I)</TD><TD class="AutoAnnotatorInnerAmino">26 (8.1%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Leu (L)</TD><TD class="AutoAnnotatorInnerAmino">41 (12.8%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Lys (K)</TD><TD class="AutoAnnotatorInnerAmino">10 (3.1%)</TD></TR></TBODY></TABLE><TABLE class="AutoAnnotatorWithBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">Met (M)</TD><TD class="AutoAnnotatorInnerAmino">13 (4.1%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Phe (F)</TD><TD class="AutoAnnotatorInnerAmino">20 (6.3%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Pro (P)</TD><TD class="AutoAnnotatorInnerAmino">12 (3.8%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Ser (S)</TD><TD class="AutoAnnotatorInnerAmino">31 (9.7%)</TD></TR></TBODY></TABLE><TABLE class="AutoAnnotatorWithBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">Thr (T)</TD><TD class="AutoAnnotatorInnerAmino">23 (7.2%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Trp (W)</TD><TD class="AutoAnnotatorInnerAmino">2 (0.6%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Tyr (Y)</TD><TD class="AutoAnnotatorInnerAmino">16 (5.0%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Val (V)</TD><TD class="AutoAnnotatorInnerAmino">23 (7.2%)</TD></TR></TBODY></TABLE><STRONG>Amino acid counting</STRONG><TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorAminoCountingInner2">Total number:</TD><TD class="AutoAnnotatorAminoCountingInner3">320</TD></TR><TR><TD class="AutoAnnotatorAminoCountingInner2">Positively charged (Arg+Lys):</TD><TD class="AutoAnnotatorAminoCountingInner3">21 (6.6%)</TD></TR><TR><TD class="AutoAnnotatorAminoCountingInner2">Negatively charged (Asp+Glu):</TD><TD class="AutoAnnotatorAminoCountingInner3">15 (4.7%)</TD></TR><TR><TD class="AutoAnnotatorAminoCountingInner2">Aromatic (Phe+His+Try+Tyr):</TD><TD class="AutoAnnotatorAminoCountingInner3">45 (14.1%)</TD></TR></TBODY></TABLE><STRONG>Biochemical parameters</STRONG><TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorBiochemParInner2">Atomic composition:</TD><TD class="AutoAnnotatorBiochemParInner3">C
  <SUB>1630</SUB>H
                                                <SUB>2548</SUB>N
                                                <SUB>402</SUB>O
                                                <SUB>444</SUB>S
                                                <SUB>21</SUB></TD></TR><TR><TD class="AutoAnnotatorBiochemParInner2">Molecular mass [Da]:</TD><TD class="AutoAnnotatorBiochemParInner3">35553.8</TD></TR><TR><TD class="AutoAnnotatorBiochemParInner2">Theoretical pI:</TD><TD class="AutoAnnotatorBiochemParInner3">8.85</TD></TR><TR><TD class="AutoAnnotatorBiochemParInner2">Extinction coefficient at 280 nm [M
                                                <SUP>-1</SUP> cm
                                                <SUP>-1</SUP>]:</TD><TD class="AutoAnnotatorBiochemParInner3">34840 / 35340 (all Cys red/ox)</TD></TR></TBODY></TABLE><STRONG>Plot for hydrophobicity, charge, predicted secondary structure, solvent accessability,
                                        transmembrane helices and disulfid bridges</STRONG> <STRONG>Codon usage</STRONG><TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorCodonUsage2">Organism:</TD><TD class="AutoAnnotatorCodonUsage3"><I>E. coli</I></TD><TD class="AutoAnnotatorCodonUsage3"><I>B. subtilis</I></TD><TD class="AutoAnnotatorCodonUsage3"><I>S. cerevisiae</I></TD><TD class="AutoAnnotatorCodonUsage3"><I>A. thaliana</I></TD><TD class="AutoAnnotatorCodonUsage3"><I>P. patens</I></TD><TD class="AutoAnnotatorCodonUsage3">Mammals</TD></TR><TR><TD class="AutoAnnotatorCodonUsage2">Codon quality (
                                                <A href="http://en.wikipedia.org/wiki/Codon_Adaptation_Index">CAI</A>):</TD><TD class="AutoAnnotatorCodonUsage3">good (0.69)</TD><TD class="AutoAnnotatorCodonUsage3">good (0.70)</TD><TD class="AutoAnnotatorCodonUsage3">good (0.61)</TD><TD class="AutoAnnotatorCodonUsage3">good (0.70)</TD><TD class="AutoAnnotatorCodonUsage3">excellent (0.81)</TD><TD class="AutoAnnotatorCodonUsage3">good (0.75)</TD></TR></TBODY></TABLE><STRONG>Alignments</STRONG> (obtained from
                                    <A href="http://predictprotein.org">PredictProtein.org</A>)
                                       There were no alignments for this protein in the data base. The BLAST
                                    search was initialized and should be ready in a few hours.<STRONG>Predictions</STRONG> (obtained from
                                    <A href="http://predictprotein.org">PredictProtein.org</A>)   There were no predictions for this protein in the data base. The prediction
                                    was initialized and should be ready in a few hours. The BioBrick-AutoAnnotator was created by
                                        <A href="https://2013.igem.org/Team:TU-Munich">TU-Munich 2013</A> iGEM team. For more information please see the
                                        <A href="https://2013.igem.org/Team:TU-Munich/Results/Software">documentation</A>.
                                        If you have any questions, comments or suggestions, please leave us a
                                        <A href="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator">comment</A>.</DIV></DIV></DIV></DIV></SECTION><SECTION id="blog-details" class="padding-top" style="padding-top: 0px;"><DIV class="container"><DIV class="row"><DIV class="col-md-12 col-sm-12" style="width: auto;"><DIV class="row"><DIV class="col-md-12 col-sm-12"><DIV class="single-blog blog-details two-column"><DIV class="post-content overflow"><H2>
                                        How to use


                                    </H2><H3>If you want to use it, you should digest it with the enzyme PstI and XbaI to get the
                                        fragments. Then you need to add the restriction sites on both sides of the sequence
                                        by PCR. A pair of reference primers below is provided.
                                         Forward: cgtaagcttatgaccgagacatctcaggtggcccctgccggcggcagggaaaataac
                                         Reverse: tatggatccttacgaggagattctcttgttg
                                         Most importantly, you need to clone the parts into the commonly used eukaryotic
                                        expression vector pcDNA3.1+. As shown, you also need to digest the vector with HindIII
                                        and BamHI endonuclease in advance.

                                    </H3><DIV align="center"><P style="text-align:center">Sequencing results of OR1A1 </P></DIV><H2>Proof of expression
                                         Sensitivity Testing

                                    </H2><H3>Test the sensitivity and specificity of this basic odor biosensors with their cognate
                                        odors β-citronellol by qPCR.

                                    </H3><H2>Cloning into lenti sgRNA(MS2)-puro backbone vector for CRISPRa（gRNA）

                                    </H2><H3>Referencing to some papers, we knew that there are three genes which can help the olfactory
                                        receptors anchored in the membrane. So we aimed to activate these three key accessory
                                        protein to enhance the express of odor receptors.


                                    </H3><TABLE class="table table-striped text-center table-hover table-responsive" style="font-size: 14px"><THEAD><TR><TH style="text-align: center">
                                                    Part
                                                </TH><TH style="text-align: center">
                                                    Plasmid Construction
                                                </TH><TH style="text-align: center">
                                                    Transformation to Host Strain
                                                </TH></TR></THEAD><TBODY><TR><TD>GNAL</TD><TD>Lenti MS2</TD><TD>
                                                    E.coli.DH5α
                                                </TD></TR><TR><TD>RIC8B</TD><TD>Lenti MS2</TD><TD>
                                                    E.coli.DH5α
                                                </TD></TR><TR><TD>RTP1</TD><TD>Lenti MS2</TD><TD>
                                                    E.coli.DH5α
                                                </TD></TR></TBODY></TABLE><H2>Virus infection experiment

                                    </H2><H3>First, we transfected HEK293 with lenti dCAS9-VP64-Blast, lenti MS2-P65-HSF1-Hygro and
                                        lenti sgRNA to produce virus.
                                         Then, we infected HEK293 with the virus of dCAS9-VP64-Blast and MS2-P65-HSF1-Hygro
                                        to construct stable cell lines. Based on these cell lines, we infected different
                                        sgRNAs of GNAL, RIC8B and RTP1 respectively.
                                         This are the results of real-time PCR.


                                    </H3><H3> These three graphs show that our sgRNAs has successfully activated the three genes
                                        that mentioned before.
                                         As for the cAMP-activated reporter gene system, we choose Forskolin to induce expression
                                        of the reporter gene, and expression of the reporter protein luciferase was confirmed
                                        by treatment with 10 millimole（mM）forskolin.

                                    </H3><H3>After a 24 h incubation, cracked the cell and added the substrate reacting with luciferase,
                                        then detected it by fluorescent microplate reader. Here is the data we got. Cells
                                        expressing luciferase treated only with 0.1%DMSO was used as a negative control.
                                        We can see that the luminescence data gets significantly improved, which proves that
                                        the reporter gene we choose is available.
                                         Here comes the most essential experiment. We infected the stable cell lines with
                                        all virus to construct a super cell line, which has a high expression of the three
                                        genes. Then we transfected OR1A1 and OR1D2 respectively in the super cell, and stimulated
                                        with β-citronellol and bourgeonal, which can recognized by OR1A1 and OR1D2 respectively.
                                        At last, we used Firefly Luciferase Assay Kit to test the cyclic AMP (cAMP) signal
                                        pathway.
                                         This is the luminescence result of OR1A1.

                                    </H3><H2>OR1A1 biosensor evaluation with its cognate odor molecule
                                    </H2><H3>Here comes the most essential experiment. We infected the stable cell lines with all
                                        virus to construct a super cell line, which has a high expression of the three genes.
                                        Then we transfected OR1A1 and OR1D2 respectively in the super cell, and stimulated
                                        with β-citronellol and bourgeonal, which can recognized by OR1A1 and OR1D2 respectively.
                                        At last, we used Firefly Luciferase Assay Kit to test the cyclic AMP (cAMP) signal
                                        pathway.

                                         This is the luminescence result of OR1A1.
                                    </H3><H3><STRONG>Conclusion：</STRONG>By using the Luciferase Assay Kit,we can see that the luminescence
                                        of super cell transfected with OR1A1 is significantly higher than that in the control
                                        group——the normal 293FT and odorless stimulated control group, which confirming the
                                        feasibility of the biosensor we built.
                                    </H3></DIV></DIV></DIV></DIV></DIV></DIV></DIV></SECTION><FOOTER id="footer"><DIV class="container"><DIV class="row"><DIV class="col-sm-12"><DIV class="copyright-text text-center"><P>Copyright © 2017 NEU-CHINA Team. All Rights Reserved.</P></DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>