Team:NEU-China/Progress
Toggle navigation
Home
Project
Background
Description
Design
Experiment & Results
Demostrate
Model
Notebook
Progress
Protocol
Parts
Our Parts
Biobricks For Medal
Basic Parts
Safety
InterLab
Collaborations
Human Practices
Human Practices
Silver
Gold
Lab
Team
Attributions
Lab Life
Progress
1. Lentiviral vector digestion, oligo annealing and cloning into digested vector
We respectively designed two sgRNAs of RTP1, RIC8B, GNAL. Respectively cloned them into digested vector and transformed into                                             DH5α bacteria.
2. We extracted human genome from HEK293 cell.
· We used the primers we designed to PCR amplify the olfactory receptor gene.
· As the brightness of the band is too weak, we have tried to found the best annealing temperature. And we have tested 56.4℃, 57.6℃, 58℃, 60℃, finding the fittest annealing temperature at 60℃.
· We ran the secondary PCR by using the special primer which can add N-terminal rhodopsin tag to the olfactory receptors                                             to instead previous prime. Here we chose annealing temperature at 60℃ and got clearer                                             bind of OR1A1(the same as OR1D2).
· We digested the OR1A1, OR1D2 and pcDNA3.1 fragments by Hind III and Xho I, and then had them gel purification respectively.
· After restriction digest of the PCR product and the expression vector pcDNA3.1. Ligation of the digested products and then                                             transformed the ligation product into DH5alpha bacteria cells. Patch and miniprep                                             to obtain our expression plasmid that produces olfactory receptors.
· In order to test whether the expression vector has carried the gene we want, we                                             used not only single(NdeI) enzyme digestion but also double enzyme (Hind III, Xho                                             I) digestion test showed the correct band of OR1A1(1Kb).
· As for OR1D2 the same procedure was carried out: PCR with specific primers we booked, double enzyme digestion, phenol purification,                                             ligation, transformation, colony PCR, path, and restriction digest testing at last.                                             the following is our digestion test results, sequencing results show completely consistent                                             with the expected return sequence.
Abstract: Unexpected mutation might occur when you amplify a specific gene through PCR method. What’s more, the efficiency of restricted digestion and ligation would also effect the result of constructing a functional plasmid. We have improved our protocol according to the experiments we performed before: PCR annealing temperature was reset at 60 degrees Celsius up and down; After removal from the PCR instrument, the PCR product was rapidly cooled; Enzyme digestion for the night to ensure its efficiency; ligation and transformation were both performed on the ice with low temperature and so on. In conclusion, we have constructed an expression vector for olfactory receptors. The corresponding improvements were made about the the mutation occurred at experiments.
3. gRNA designed and booked the specific primers.
Cloned the gRNA fragment into lentivirus expression vector. Transfected all of the gRNAs (included RTP1, RIC8B, GNAL,), lenti dCas9-VP64-Blast and lenti MS2-P65-HSF1-Hygro into HEK293 cell. Test the transfection efficiency by GFP control.
sgRNA with different target loci have been tested by real-time PCR. The target sites can be determined by directing the gRNA consisting of 20 bp length against the desired sequence of interest. R1 with target sites at different distances to the promotor regions proved successfully as potential activation sites (see Table 2 and Figure 3).
4. We screened out the cells carried the crispra elements by blasticidin and Hygromycin, and by using puromycin to screen                                             out the cells which have been transfected by lenti MS2-sgRNA.
· We tested the expression of OR1A1, OR1D2, RTP1, RIC8B and GNAL by real-time PCR, and attached the data in table3-7.
Then we found that the expression of all the gRNAs we designed had a certain degree of promotion, so we infected the HEK293 cell with all the lenti MS2-gRNA generating our iSmeller. And text the the expression of OR1A1 in ismeller by qPCR, and showed the data in table 8.
5. Luminescence signal testing.
Here we choose luciferase as our reporter gene. First we used the forskolin to stimulate the cAMP pathway, and found a significant                                             enhance of luminescence, and showed the data in table 9.
· And then we used different odor to test the specificity of OR1A1, successfully proved that β-citronellol is the specific                                             odor of OR1A1. Table 10 shows the luminescence intensity of different odors tested                                             by given to OR1A1 biosensor.
And then we respectively detected the luminescence intensity in HEK293 cell, which transfected by OR1A1 and iSmeller, and showed the data in table 11.
6. As for the parts construction, we respectively designed the primers for OR1A1 and OR1D2 as following.
· OR1A1:
Forward: cgtaagcttatgaccgagacatctcaggtggcccctgccggcggcagggaaaataac
Reverse: tatggatccttacgaggagattctcttgttg
· OR1D2:
Forward: cgtaagcttatgaccgagacatctcaggtggcccctgccggcggcgatggaggcaac
Reverse: tatctcgagttatgtcagcctcttaaagtgtttatctagg
· As the gene segment of OR1A1 has the cutting site of EcoR I, we first designed                                             the primer which would mutate GAA to GAG generating an synonymous mutation. And then                                             same procedure was carried out: PCR with specific primers we booked, double enzyme                                             digestion, phenol purification, ligation, transformation, colony PCR, path ,and restriction                                             digest testing at last. the following is our gel extraction results which shows completely                                             consistent with the expected return sequence.
7. Here we helped Lan Zhou University's igemers to construct the hpRNA producing vector ecrL, which sped up the construction                                             progress of their vectors.
· Zhejiang University igemers gave us to the culture dishes with Trichoderma (one transfected with the plasmid, a plate for                                             the wild type), then we picked hyphae to sterile water and measured its fluorescence                                             values recording in table 12.
We analyze:
1. The differences between the two groups confirmed that the modified Trichoderma                                             did successfully express EGFP.
2. The numerical fluctuation in the same group is                                             due to the difference in the amount of hyphae we picked.
• We helped Nankai University to verify the function of the lyase gene. We synthesize the lyase gene whose sequence from Xanthomonas oryzae bacteriophage Xp10 and transformed it into Enterobacter sp. FY-07. After we added the inducer (arabinose), the optical density of the bacteria decreased significantly as shown in table12.
Copyright © 2017 NEU-CHINA Team. All Rights Reserved.
