"
Page
Discussion
View source
History
teams
Log in
 
Team:Hong Kong-CUHK/BIOBRICKS CONSTRUCTION
From 2012.igem.org
 
 
Check out our FACEBOOK page!
HOME
TEAM
Team Members
Instructors and Advisors
PROJECT
Overview
Background
Modeling
Results
Applications
Judging Form
BIOBRICKS
Parts
Construction Notes
Characterizations
HUMAN PRACTICE
Diary
Survey Analysis
High School
University Scholars
Potential iGEMers
Conclusion & Future Plans
Museum Proposal
Q&A
Guide on High School Division
SAFETY
Safety Questions
New Safety Approach
DOCUMENTATION
Protocols
Notebook
May
June
July
August
September
ACKNOWLEDGEMENT
Support
Sponsors
 
CONSTRUCTION OF PARTS AND DEVICE
Method of construction
Our team made use of a fast and convenient assembly method developed recently [1] to construct all of our biobricks in an effective way without the use of restriction enzymes and ligase - the direct transformation of prolonged overlap extension PCR products.
Prolonged Overlap Extension PCR (POE-PCR)
Amplification of genes
Linear fragment DNA of the insert(s) and vector were amplified from corresponding templates by using specially designed primers which can add overlapping regions (40 bps per linear DNA) onto the DNA fragments.
Prolonged overlap extension PCR
Equal molar of insert(s) and vector DNA were added into a PCR reaction mix. The POE-PCR was conducted as follows: denaturation at 98°C for 30 s; 25 cycles of denaturation at 98°C for 10 s, annealing at 60°C for 10 s, and extension at 72°C for 2.5 min.
Direct Transformation
Five microliter of the prolonged overlap extension PCR products was used to transform competent cells directly.
Constructs
List of primers
Primer#   primer sequence
TGAAAGAGGAGAAATACTAGAAGCTTATGGTGGGACTTACGACCCT
CGCCGACGCGCCGTTCGACGCGGATCCGTCGGCGACCGCAGGCGTGT
GGATCCGCGTCGAACGGCGCGTCGGCGATGTCGCTGAACGTATCACG
TGCGCCAGTCGGTGCGGACAACCGTCGGTGATGTGCGCAA
CTACACTAGCACTATCAGCGTTAAAATGTTTCCCAGTTCT
AGAACTGGGAAACATTTTAACGCTGATAGTGCTAGTGTAG
AGGGTCGTAAGTCCCACCATAAGCTTCTAGTATTTCTCCTCTTTCA
TCGCGGACATGAGTGACGGTTGTCCGCACCGACTGGCGCA
TGCGCCAGTCGGTGCGGACAACCGTCACTCATGTCCGCGA
CTACACTAGCACTATCAGCGTCAAAATGTTTCCCAGTTTG
CAAACTGGGAAACATTTTGACGCTGATAGTGCTAGTGTAG
TGAAAGAGGAGAAATACTAGAAGCTTATGGACGCCGTCGCAACCGC
TGCGCCAGTCGCTTCGTGGCACCGTCACTCATGTCCGCGA
ATTCGCGGCCGCTTCTAGAGTCCCTTGCATTTACATTTTG
ATCTAGTATTTCTCCTCTTTAGTCCATTCTCCCCAAAAAT
CTAAAGAGGAGAAATACTAGATGGCTTCCTCCGAAGACGT
CAAAATGTAAATGCAAGGGACTCTAGAAGCGGCCGCGAAT
GGAAAGAGGAGAAATACTAGATGGCCACCACCGTACAACT
CTAATGATGATGATGATGATGCCCTTCTTTTGTCATGCCCT
CATCATCATCATCATCATTAGTACTAGTAGCGGCCGCTGCA
ATCTAGTATTTCTCCTCTTTCCGGACCGCAGGCTGGCTAG
Negative Phototactic Construct for Blue Light Detection
BBa_K786001
Primers 1 and 2 were used to amplify sensory rhodopsin II coding sequence from the genome of Natronomonas Pharaonis DSM 2160. Restriction sites of HindIII and BamHI were added.
Restriction sites of HindIII and BamHI were added before and after the SRII gene respectively in order to:
Enable further integration of other peptides such as His-tag, or construct a larger fusion protein (HindIII for N-terminus ligation while BamHI for C-terminus).
Enable us to switch the sensory rhodopsin portion of the fusion protein. A series of mutant sensory rhodopsins were identified which cover a large variation of absorption spectrum [2]. These two restriction sites allow further switching of the sensing unit, so the light sensing system can be tuned for sensing different kinds of light source.
Primers 3, 8 were used to amplify the coding sequence of HtrII from the genome of Natronomonas Pharaonis DSM 2160. A linker (GSASNGASA) that was proven not affecting the SR system [3] was added to joint SRII and HtrII.
Primers 4, 5 were used to amplify the coding sequence of Tsr from E. coli K-12genome.
Primers 6, 7 were used to amplify the promoter J23100 and J61002 backbone from biobrick BBa_J23100.
All of the parts amplified were added in PCR mix with equal molar to perform overlapping PCR and the PCR product was used for direct transformation.
The insert was later on switched to pSB1C3 backbone by using EcoRI and PstI restriction enzymes and T4 ligase.
The SpeI site after the promoter was kept so that the constitutive promoter can be switched to strictly controlled promoters such as Ptet (tetracycline-inducible promoter) and PBAD (arabinose-inducible promoter).
 
 
 
Positive Phototactic Construct for Blue Light Detection
BBa_K786002
Primers 1 and 2 were used to amplify sensory rhodopsin II (SRII) coding sequence from the genomic DNA of Natronomonas Pharaonis DSM 2160. Restriction sites of HindIII and BamHI were added.
Restriction sites of HindIII and BamHI were added before and after the SRII gene respectively in order to:
Enable further integration of other peptides such as His-tag, or construct a larger fusion protein (HindIII for N-terminus ligation while BamHI for C-terminus).
Enable us to switch the sensory rhodopsin portion of the fusion protein. A series of mutant sensory rhodopsins were identified which cover a large variation of absorption spectrum [2]. These two restriction sites allow further switching of the sensing unit, so the light sensing system can be tuned for sensing different kinds of light source.
Primers 3, 8 were used to amplify the coding sequence of HtrII from the genome of Natronomonas Pharaonis DSM 2160. A linker (GSASNGASA) that was proven not affecting the SR system [3] was added to joint SRII and HtrII.
Primers 9, 10 were used to amplify the coding sequence of Tar from E. coli K-12 genomic DNA.
Primers 7, 11 were used to amplify the promoter J23100 and J61002 backbone from biobrick BBa_J23100.
All of the parts amplified were added into a single PCR mix with equal molar to perform overlapping PCR. The PCR product was used for direct transformation.
The insert was later on switched to pSB1C3 backbone by using EcoRI and PstI restriction enzymes and T4 ligase.
The SpeI site after the promoter was kept so that the constitutive promoter can be switched to strictly controlled promoters such as Ptet (tetracycline-inducible promoter) and PBAD (arabinose-inducible promoter).
 
 
Phototactic Construct for Orange Light Detection
BBa_K786003
Primers 11, 12 were used to amplify sensory rhodopsin I (SRI) coding sequence from the genome of Halobacterium salinarum. Restriction sites of HindIII and BamHI were added.
Restriction sites of HindIII and BamHI were added before and after the SRI gene respectively in order to:
Enable further integration of other peptides such as His-tag, or construct a larger fusion protein (HindIII for N-terminus ligation while BamHI for C-terminus).
Enable us to switch the sensory rhodopsin portion of the fusion protein. A series of mutant sensory rhodopsins were identified which cover a large variation of absorption spectrum [2]. These two restriction sites allow further switching of the sensing unit, so the light sensing system can be tuned for sensing different kinds of light source.
Primers 3 and 8 were used to amplify the coding sequence of HtrI from the genome of Halobacterium salinarum. A linker (GSASNGASA) that was proven not affecting the SR system [3] was added to joint SRI and HtrI.
Primers 13, 10 were used to amplify the coding sequence of Tar from E. coli K-12genome.
Primers 11, 7 were used to amplify the promoter J23100 and J61002 backbone from biobrick BBa_J23100.
All of the parts amplified were added in PCR mix with equal molar to perform overlapping PCR and the PCR product was used for direct transformation.
The insert was later on switched to pSB1C3 backbone by using EcoRI and PstI restriction enzymes and T4 ligase.
The SpeI site after the promoter was kept so that the constitutive promoter can be switched to strictly controlled promoters such as Ptet (tetracycline-inducible promoter) and PBAD (arabinose-inducible promoter).
 
 
The Gene Expression System
BBa_K786010
Primers 14 and 15 were used to amplify the promoter R0083 from the biobrick BBa_R0083. Primers 16 and 17 were used to amplify the back bone pSB1C3 and the RFP reporter gene by using biobrick BBa_K606030. Strong RBS BBa_B0034 was added in the construct during the primer design.
 
 
Red Light Sensing Construct
BBa_K786004
Primers 18 and 19 were used to amplify the coding sequence of Cph8 from the biobrick BBa_I15010. Primers 20 and 21 were used to amplify the promoter J23100 and the backbone pSB1C3 from the Biobrick BBa_K515102. Poly his-tag and strong RBS BBa_B0034 were added in the construct during the primer design.
 
 
 
 
[1] You C, Zhang XZ, Zhang YH (2012). Simple cloning via direct transformation of PCR product (DNA Multimer) to Escherichia coli and Bacillus subtilis. Appl Environ Microbiol. 78: 1593-1595.
[2] Sudo Y, Yuasa Y, Shibata J, Suzuki D, Homma M (2011). Spectral tuning in sensory rhodopsin I from Salinibacter ruber. J Biol Chem. 286: 11328-11336.
[3] Jung KH, Spudich EN, Trivedi VD, Spudich JL (2001). An archaeal photosignal-transducing module mediates phototaxis in Escherichia coli. J Bacteriol. 183: 6365-6371      .
Home   |    Team    |    Project   |    Biobricks    |    Human Practice    |   Safety    |    Documentation   |    Acknowledgement
 
Address: Rm. 184, Science Centre, CUHK
Email: kingchan@cuhk.edu.hk  Tel: (852)-39434420  Fax: (852)-26037246
©Copyright CUHK iGEM Team 2012, All Rights Reserved.
Copyright © 2012 Apycom jQuery Menus
Retrieved from " http://2012.igem.org/Team:Hong_Kong-CUHK/BIOBRICKS_CONSTRUCTION "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
