"
Page
Discussion
View source
History
teams
Log in
 
Team:Hong Kong-CUHK/SAFETY APPROACH
From 2012.igem.org
 
 
Check out our FACEBOOK page!
HOME
TEAM
Team Members
Instructors and Advisors
PROJECT
Overview
Background
Modeling
Results
Applications
Judging Form
BIOBRICKS
Parts
Construction Notes
Characterizations
HUMAN PRACTICE
Diary
Survey Analysis
High School
University Scholars
Potential iGEMers
Conclusion & Future Plans
Museum Proposal
Q&A
Guide on High School Division
SAFETY
Safety Questions
New Safety Approach
DOCUMENTATION
Protocols
Notebook
May
June
July
August
September
ACKNOWLEDGEMENT
Support
Sponsors
 
NEW SAFETY APPROACH!
Multidrug resistance on microorganism has already changed from nightmare to reality. Horizontal gene transfer is one of the ways for bacteria to obtain exogenous genetic material. The previous proposal on biosafety of synthetic biology by other iGEM teams have led to achievements on different approach, such as incorporating active inhibition of horizontal gene transfer (Imperial 2011), using antibiotic-free selection to reduce potential multidrug resistance on antibiotic, and labeling spec sheet information on engineered bacteria for identity (CUHK 2010).
It is well-known that DNA molecule can be long lasting for thousands years. Therefore, even the engineered bacteria was killed after experiments, the artificial genetic material such as synthetic DNA can still present in the environment. Other bacteria can uptake these genetic information and possibly incorporate into their genome. The active uptake mechanism of DNA from the environment to bacteria can be known as “natural genetic transformation”. Thus, this intelligent design can bypass the previous proposed biosafety measures.
Herein, our proposed system allows a selective cleavage of the target gene, such as antibiotic gene and biobrick gene, by activating an exonuclease to recognize the specific region and undergo digestion. This approach allows regulation of the engineered microorganism down from DNA level.
 
I. Mechanism: CRISPR/Cas systems
II. Biobrick Design
III. Expected result
IV. Impact
 
I. Mechanism: CRISPR/Cas systems
CRISPR/Cas systems are the adaptive immunity systems that are present in many archaea and bacteria to protect themselves from invading genetic materials such as viral DNA. There are three types of CRISPR/Cas system. Here we only exploit type I-E from Escherichia coli (E. coli) to focus on expression and interference stage. This system presents potential in developing into a novel tool in order to reach higher safety standard of using engineered microorganism machine.
 
Foreign double strand DNA (dsDNA) will be targeted by the spacers from repeat-spacer region (RSR) and destroyed by one of the CRISPR/Cas components, Cas3. Through engineering the spacer sequence, theoretically we can target any dsDNA of sequence complementary to the spacer(s) [1].
By re-designing the targeting sequence, we can cleave the transformed plasmid in certain bacteria to reduce environmental hazard, or even manage to create a controlled suicide system to reduce leakage of genetic information from the bacteria. Thus our system provides an alternative to control safety level of engineered microorganism machine.
Back To Top
II. Biobrick design
Our biobrick is designed to be an integration of two parts, the protein part and the R-S-R part. Sequences of proteins Cas3 and cascade complex subunits are located in the upstream of the RSR system.
The number of the repeat and spacer are not fixed and the distance from the promoter does not affect the functionality.
The protein part in the biobrick is not essential for the system to work, as the type I-E CRISPR/Cas system is an endogenous system in E. coli. Since this part is designed to improve the sufficiency of system functionality, we choose to use Pveg, a strong constitutive promoter. This choice also makes it possible to utilize this system in other microorganisms where CRISPR system does not exist naturally, such as Bacillus subtilis.
To improve the efficiency for construction, we split them into two biobricks containing Cas3 and the R-S-R part, and synthesize them separately Therefore, when testing the system in E. coli, transformation of the R-S-R biobrick alone is enough for characterization.
(a) ‘Repeat’ design
A palindrome can be observed by studying some samples of repeat sequences of CRISPR system of E. coli. The secondary structure of pre-crRNA shows that the base pairing helps to form a loop, which may contribute to the Cas3 and Cascade in expression and interference stages of type-I CRISPR system.
From related research papers [2, 3], we obtained several repeat sequences of E. coli strain K-12. Although repeat sequences from different sources may not be exactly identical, similar characteristics can be found.
We also tried other tools for repeat design. An online CRISPR database CRISPRdb (http://crispr.u-psud.fr/) as recommended by 2011 USC iGEM team provides repeat consensus of different strains [4].  
(b) Spacer Design
PAM (Proto-spacer Adjacent Motif) is located adjacent to the proto-spacer, upstream for a sense strand or downstream for an antisense strand. This 3-bp motif is essential for the proto-spacer recognition by Cascade complex. In E. coli, Cascade tolerates at least four PAMs, CTT, CAT, CTC and CCT in an antisense strand [5].
Seed sequence, as indicated in the graph, is another essential for spacer design. Mutants on seed sequence may lead to “escape”. Thus for these 7 base pairs, the crRNA must complement to the target DNA. In our design, we have the whole spacer to be completely complementary to the protospacer, so does the crRNA. Therefore, the problem of "escape" due to the mismatch in seed sequence is not crucial.
The designed length of spacer in our CRISPR system should be 32bp.  The spacer is specially desigened to target on the genomic DNA of a typical strain of E. coli, K12 MG1655. It is a non-coding DNA sequence since it is important to confirm that the factor causing the death of E. coli is due to the cleavage of its genomic DNA but not the malfunction of the gene being targeted, so a non-coding DNA sequence is preferred than a coding DNA sequence. The selection of the non-coding DNA sequence of E. coli K12 MG1655 is random, and the 32 bp is taken from 121595-121626 (GenBank: U00096.2 )
Back To Top
III. Expected Result
Firstly, we need to test if the proposed system present in the E. coli. We Used primers to amplify the Cas3 and cascade proteins from E. coli K12 genome DNA. Expected band size of cas3 2.7kbp and cascade 4.4kbp are shown in the gel photo, and sequencing has be done, which proves the existence of these proteins in its genome, thus we can utilize this CRISPR/Cas3 system as a tool for our new safety approach. In the future we can make them into biobrick format and apply into microorganism other than E. coli.
After the Asia Jamboree, we continued with the molecular cloning job and have successfully cloned the original coding sequence of cas3 and cascade into two biobricks, BBa_K786032 and BBa_K786033 . Here is the electrophotoresis result after they are digested by EcoRI and SpeI.
To evaluate whether the RSR system transformed into E. coli can degrade genomic DNA and eventually lead to cell death, we measured OD600 of cell cultures every 0.5 h and plotted the growth curves of transformed E. coli (indicated as solid lines) with or without 0.4 mM IPTG activation. Paired t-test showed that there is no significant difference in OD600 throughput the whole experiment with or without RSR activation by IPTG.
The result agrees with the percentage of survival (indicated as dashed lines) derived from live/dead fluorescent staining assay (Invitrogen) measuring bacterial apoptotic-like rate, which showed no difference in survival percentage with or without RSR activation. It was expected that the survival percentage drops significantly after IPTG induction. Further optimization of design of the RSR region and experimental conditions is required for more specific targeting.
Back To Top
 
IV. Impact
A. Cleavage of selected biobrick
Our proposed system can be utilized as a new safety approach for synthetic biology, especially for iGEMers. As we know, one of the most concerned safety issues in engineering cells is that the leakage of bacteria with transformed plasmid that may cause environmental pollution. As it was proved that CRISPR system can target on plasmid DNA sequence, our system may also function as a tool for biobrick cleavage. Specifically, this tool can be used for targeting and cleaving the antibiotic resistance genes present on the biobricks or the standard site of the biobrick such as prefix, suffix, in order to prevent horizontal gene transfer caused by DNA fragments released from apoptotic pathways. Our system has the potential to be applied as a tool to provide a higher level of the safety control of engineered microorganism machine starting from DNA level.
B, Self-targeting of Engineered Bacteria
Our biobrick has a potential application of cleaving the genome sequence of certain bacteria as selected, which can function as a novel approach to deal with the new safety issue in synthetic biology because by re-designing the segment of sequence for targeting, it can be used to target particular bacterial genome.
With synthetic biology, we can easily apply the system from one organism to another after characterizations. Therefore, by understanding the gene(s) required and the whole mechanism of this biosafety system, it can be engineered into other microorganisms, including the ones lacking native CRISPR system such as Bacillus subtilis. The modification can be done by substituting the promoter, copying the essential machineries from E. coli CRISPR system, and re-designing ‘spacer’ for certain target(s). Theoretically our biobrick can also function in B. subtilis as we choose to express the genes under the Pveg promoter (BBa_K143053), which is a constitutive promoter that works in both E. coli and B. subtilis. Through engineering the E. coli CRISPR/Cas3 system into certain bacteria as well as combining it with different sensors such as light and chemical ones, we may achieve at making a well-regulated system, which can serve as a flexible yet powerful tool for bacteria self-targeting.
Back To Top
 
                                                                                              
[1] Makarova KS, Haft DH, Barrangou R, et al. (2011). Evolution and classification of the CRISPR-Cas systems. Nat Rev Microbiol. 9: 467-477.
[2] Brouns SJ, Jore MM, Lundgren M, et al. (2008). Small CRISPR RNAs guide antiviral defense in prokaryotes. Science. 321: 960-964.
[3] Yosef I, Goren MG, Qimron U (2012). Proteins and DNA elements essential for the CRISPR adaptation process in Escherichia coli. Nucleic Acids Res. 40: 5569-5576.
[4] Grissa I, Vergnaud G, Pourcel C (2007). The CRISPRdb database and tools to display CRISPRs and to generate dictionaries of spacers and repeats. BMC Bioinformatics. 8: 172.
[5] Semenova E, Jore MM, Datsenko KA, et al. (2011). Interference by clustered regularly interspaced short palindromic repeat (CRISPR) RNA is governed by a seed sequence. Proc Natl Acad Sci U S A. 108: 10098-10103.
[6] Westra ER, van Erp PB, Kunne T, et al. (2012). CRISPR immunity relies on the consecutive binding and degradation of negatively supercoiled invader DNA by Cascade and Cas3. Mol Cell. 46: 595-605.
 
 
 
Home   |    Team    |    Project   |    Biobricks    |    Human Practice    |   Safety    |    Documentation   |    Acknowledgement
 
Address: Rm. 184, Science Centre, CUHK
Email: kingchan@cuhk.edu.hk  Tel: (852)-39434420  Fax: (852)-26037246
©Copyright CUHK iGEM Team 2012, All Rights Reserved.
Copyright © 2012 Apycom jQuery Menus
Retrieved from " http://2012.igem.org/Team:Hong_Kong-CUHK/SAFETY_APPROACH "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
