"
Page
Discussion
View source
History
teams
Log in
 
Team:Hong Kong-CUHK/4.3
From 2012.igem.org
 
 
Check out our FACEBOOK page!
HOME
TEAM
Team Members
Instructors and Advisors
PROJECT
Overview
Background
Modeling
Results
Applications
Judging Form
BIOBRICKS
Parts
Construction Notes
Characterizations
HUMAN PRACTICE
Diary
Survey Analysis
High School
University Scholars
Potential iGEMers
Conclusion & Future Plans
Museum Proposal
Q&A
Guide on High School Division
SAFETY
Safety Questions
New Safety Approach
DOCUMENTATION
Protocols
Notebook
May
June
July
August
September
ACKNOWLEDGEMENT
Support
Sponsors
 
CH4.3 Stage 3 – Make your BioBrick
Here are the descriptions of one of the traditional way to make a BioBrick. General materials are listed to be prepared before the lab starts, and some points should be noted in the process.
This session can be used as your check list before you are familiar with the actual protocol from different kits, but please take note that variation always exists and understanding the protocol before start is helpful as well as necessary for the desired result.
Do not be upset with negative result. Protocols vary among the different designs of the BioBrick, and asides from the one described above, there are other ways to make a BioBrick.
 
Part 1: BioBrick Construction
Step 1: Polymerase Chain Reaction (PCR)
A DNA amplification process to make the target sequence into linear DNA copies
Material: dNTPs, water, DNA templates, forward and reverse primers, heat stable DNA polymerase and its corresponding buffer
Note:
 DNA templates should be added at last, and thermocycle should start immediately after adding all the materials
Step 2: DAN Digestion by Restriction Enzyme (Restriction Cut)
Restriction cut can be used and repeated for different purposes: create a sticky end of the DNA, make blunt end of DNA, or make linear plasmid into circular form.
Material: water , DNA template, restriction enzyme, 100X BSA buffer, 10X NEWB buffer, and 37。C incubator or dry bath
Note:
 Water is added first
 Buffer is added before enzyme
 Ensure the pipette tip touches the surface of solution
 Glycerol can be added to prevent further undesirable DNA digestion
Step 3: DNA Purification
Removing excess nucleotides, enzymes and ions in the solution with desired 100 bp - 10 kb DNA
Material: use PCR Purification Kit
Note:
 Elution efficiency is pH dependent, and the optimal pH for maximum efficiency is in between pH 7 and 8.5
 DNA should be stored at 20 ° C to prevent degradation
 EDTA buffer may inhibit subsequent enzyme reactions
 Recovery can be maximized by warm elution buffer
Source: Loo J. 2012. iGEM Hong_Kong-CUHK 2012 Wet-lab Training Manual.
The Chinese University of Hong Kong School of Life Science.Version 2.2.
Step 4: DNA Purification from Gel
After checking the band size through gel electrophoresis, enzymes, ions, nucleotides, needs to be removed from solution for the target DNA (70 bp - 10 kb)
Material: use gel clean kit, distilled water, and 60。C dry bath
Note:
 Both DNA purifications give high possibility of correct ligation later. But gel purification allows the selection of target DNA sequence from the other DNA fragments while PCR purification only lowers the contamination.
Step 5: T4 DNA Ligation
Linking the sticky ends or blunt end of DNA segments after digestion to make linear or circular plasmid
Materials: T4 DNA ligase, 10X T4 DNA buffer, ddH2O, restriction enzyme cut, 16。C dry bath or - 4。C fridge
Note:
 Reagent should be placed on ice
 Reaction mixture should be prepare in a fixed series of steps
 Insert to mass of vector should be added with one of the following ratio: 1:1; 1:3; 3:1; Or you can use the Bio-math calculator like this one:
https://www.promega.com/techserv/tools/biomath/calc06.htm to find out the amount to be added. Moreover the total amount should be > 100 ng to prevent self-ligation or inhibition of ligation.
 Reaction takes place over night at - 4 or 16 。C or for 2 hours at 22 to 25。 C
Step 6: Bacterial Transformation
Opening pores on the membrane of cell by heat shock or chemical to allow circular DNA plasmid to enter
Material: aseptic technique, cell, LB plates, plasmid DNA, LB media or SOC media, incubator
Notes:
 Timing for heat shock or use of chemical use on competent cell should be accurately measured with timer to prevent killing the cell
 Spreader should be cool after flaming to spread bacteria onto the plate
Source: Loo J. 2012. iGEM Hong_Kong-CUHK 2012 Wet-lab Training Manual.
The Chinese University of Hong Kong School of Life Science.Version 2.2.
Step 7: Biobrick Amplification in Plasmid Form
Amplifying the desire DNA plasmid by growing the colonies from the spread plates with antibiotic to select the expect bacteria which contains the backbone making the bacteria to be resistant to a particular antibiotic.
Material: autoclaved tooth pick, LB media, antibiotic, snap cap, and 37。C shaker
Notes:
 Require aseptic technique for picking clone
 Mark the clone you picked
 Storage of the bacteria cell with desired plasmid: 800 μl of cell + 200 μl of 80% sterile glycerol, and they should be stored in -80。C freezer.
Step 8: Plasmid DNA Extraction
Material: miniprep plasmid DNA extraction kit
Notes:
 Be careful when transferring supernatants from centrifuge tubes into the spin column to avoid contamination of genomic DNA
 To increase the concentration of plasmid, increase the incubation time while the Elution buffer stains in a 60 。C  dry bath
Source: Loo J. 2012. iGEM Hong_Kong-CUHK 2012 Wet-lab Training Manual.
The Chinese University of Hong Kong School of Life Science.Version 2.2.
Part 2: Characterization of BioBrick
Step 1: DNA Size Determination
Through comparing DNA ladder as the reference with your DNA or plasmid, you can know whether you have the desired DNA or plasmid. This step is normally done to ensure the quality of DNA after PCR, restriction enzyme cut, ligation, and extraction.
Material: Agarose gel, TAE buffer, gel imager, computer, DNA ladder, 6X or 10X DNA loading dye, DNA staining dye.
Notes:
 The higher the voltage, the lesser the time to run a gel electrophoresis
 Never run a gel with > 200 V because the heat generated from the machine may melt the gel and leads to electric leakage and inaccurate result of the DNA bands
 120 to 180 V is the acceptable range
 100ng of DNA is needed to be loaded and appeared in the image of the gel
Step 2: Preparation of DNA for Nucleotide Sequence Determination
Material: Backward and forward primers sequence the DNA and result is analysis by sequencing machine and shown in a computer with the software. This is the genotypic method to identify the bacteria, and in this case, to see whether it matches your BioBrick design.
Notes:
 This can be sent and done by some biotechnology companies around your area.
Step 3: Determination of Fluorescent Signal
Fluorescent plate reader is used to measure and calculate the fluorescent signal expressed from fluorescence-fusion protein Material: 96-well plate, standard fluorescence, and micro-plate reader
Notes:
 488 nm for excitement wavelength and 509 nm emission wavelngth
 The standard for one 96-well plate cannot be used to another plate.
Source: Loo J. 2012. iGEM Hong_Kong-CUHK 2012 Wet-lab Training Manual.
The Chinese University of Hong Kong School of Life Science.Version 2.2.
BACK TO CONTENT
Home   |    Team    |    Project   |    Biobricks    |    Human Practice    |   Safety    |    Documentation   |    Acknowledgement
 
Address: Rm. 184, Science Centre, CUHK
Email: kingchan@cuhk.edu.hk  Tel: (852)-39434420  Fax: (852)-26037246
©Copyright CUHK iGEM Team 2012, All Rights Reserved.
Copyright © 2012 Apycom jQuery Menus
Retrieved from " http://2012.igem.org/Team:Hong_Kong-CUHK/4.3 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
