"
Page
Discussion
View source
History
teams
Log in
 
Team:Hong Kong-CUHK/DOC PROTOCOLS
From 2012.igem.org
 
 
Check out our FACEBOOK page!
HOME
TEAM
Team Members
Instructors and Advisors
PROJECT
Overview
Background
Modeling
Results
Applications
Judging Form
BIOBRICKS
Parts
Construction Notes
Characterizations
HUMAN PRACTICE
Diary
Survey Analysis
High School
University Scholars
Potential iGEMers
Conclusion & Future Plans
Museum Proposal
Q&A
Guide on High School Division
SAFETY
Safety Questions
New Safety Approach
DOCUMENTATION
Protocols
Notebook
May
June
July
August
September
ACKNOWLEDGEMENT
Support
Sponsors
 
PROTOCOLS
1. Cloning
1.1 Gene Amplification with PCR
1.2 PCR Purification
1.3 Overlapping PCR
1.4 Agarose Gel Electrophoresis
1.5 Preparation of Competent Cells
1.6 Bacterial Transformation
1.7 Inoculation
1.8 Plasmid DNA Extraction (Miniprep)
1.9 Double Digestion
1.10 Gel Extraction
1.11 Ligation
2. Functional test
2.1 Cell Movement Test
2.2 Sensory Rhodopsin-induced Gene Expression Test
 
1.1 Gene Amplification with PCR
1. Put the reaction tubes on ice.
2. Add the following components into each reaction tube:
5x Phusion HF Buffer* (1X final concentration is recommended)
10 mM dNTPs (200 μM final concentration is recommended)
Primer solution (0.5 μM final concentration of each is recommended)
DNA Template
Phusion DNA Polymerase (0.02 U/μl final concentration is recommended)
3. Mix well and run the following program:
 
2-step PCR Cycling Program
Initial denaturation:
98°C   30 seconds
25–35 cycles
98°C   5–10 seconds
72°C   15–30 seconds/kb
Final extension:
72°C   5–10 minutes
Hold:
4°C
Back To Top
 
1.2 PCR Purification*
Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix.
If pH indicator I has been added to Buffer PB, check that the color of the mixture is yellow. If the color of the mixture is orange or violet, add 10 µl of 3 M sodium acetate (pH 5.0), and mix. The color of the mixture will turn to yellow.
Place a QIAquick spin column in a provided 2 ml collection tube.
To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60s.
Discard flow-through. Place the QIAquick column back into the same tube.
To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through and place the QIAquick column back in the same tube. Centrifuge the column for an additional 1 min.
Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 50 μl Buffer EB (10 mM Tris-Cl, pH 8.5) or water (pH 7.0–8.5) to the center of the QIAquick membrane and centrifuge the column for 1 min. Alternatively, for increased DNA concentration, add 30 μl elution buffer to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge.
If the purified DNA is to be analyzed on a gel, add 1 volume of Loading Dye to 5 volumes of purified DNA. Mix the solution by pipetting up and down before loading the gel.
IMPORTANT: Ensure that the elution buffer is dispensed directly onto the QIAquick membrane for complete elution of bound DNA. The average eluate volume is 48 µl from 50 µl elution buffer volume, and 28 µl from 30 µl elution buffer.
*Protocol adopted from QIAquick PCR purification kit protocol
Back To Top
 
1.3 Overlapping PCR
1. Put the reaction tubes on ice.
2. Add the following components into each reaction tube:
5x Phusion HF Buffer* (1X final concentration is recommended)
10 mM dNTPs (200 μM final concentration is recommended)
Primer solution (0.5 μM final concentration of each is recommended)
DNA Template
Phusion DNA Polymerase (0.02 U/μl final concentration is recommended)
3. Mix well and run the following program:
2-step PCR Cycling Program
Initial denaturation:
98°C   30 seconds
25–35 cycles:
98°C   10 seconds (denaturation)
60°C  10 seconds (annealing)
72°C   15–30 seconds/kb (extension)
Hold:
4°C
Back To Top
 
1.4 Agarose Gel Electrophoresis
A. Cast gel
1. Dissolve 0.55 g agarose into 55 ml 0.5X TBE buffer.
2.  Microwave (high power, 800W) for 1 min.
3.  Cool it down using running water for 1 min.
4.  Add 1 μl GelRed
5.  Pour the solution to tightened tank with gates and gel comb and allow it to solidify.
6.  Transfer the gel to gel tank once it becomes solid.
B. Run gel
1. Orient the gel with wells facing the black negative electrode. Check if the gel is covered by TBE buffer in the tank. If not, add TBE buffer to cover it to about 1mm.
2.  Mix loading dye and the insert/plasmid before adding to the wells. For example, if the DNA we have got is 45μl, and the loading dye we have got is 10X, then add 5μl of loading dye to the samples. Mixture should be in blue.
3.  To run the gel, add all samples to the wells of gel. Then add 1kb DNA ladder to a separate well. 1μl should be enough for detection under UV.
4.  Connect the electrodes to the power supply with correct colour. Set the power supply to 120V. Check if there are bubbles on the negative electrodes.
5.  Allow it to run for about 40 - 60 min. To avoid running the band off the gel, the yellow band (position of the smallest fragments) should stay on the gel.
Back To Top
 
1.5 Preparation of Competent Cells
A. Preparation of detergent-free glassware and media
1. Autoclave glassware filled with 3/4 dd-H2O to remove most residual detergent.
2.  Autoclave media and buffers in detergent-free glassware
B. Preparation of the competent cells
Reagents:
Glycerol stock
LB plate
MgCl2-CaCl2 solution
MgCl2‧6H2O                     3.25g
CaCl2‧2H2O                      0.6g
Add H2O to 200 ml
100mM CaCl2
CaCl2‧2H2O                      2.95g
Add H2O to 200 ml
80% glycerol
Liquid nitrogen
Procedure:
Day 1
1. Flame the metal inoculating loop until it is red got and then cools it down.
2. Scrape off a portion from the top of the frozen glycerol stock [DO NOT THAW].
3. Streak it onto the LB plate.
4. Put the stock back to -80 oC immediately.
5. Leave the plates for 5 min and place them upside down in the 37oC incubator for 16-20 h.
Day 2
6. Pick a single colony into 5 ml of LB medium.
7. Inoculate the culture overnight at 37oC with shaking at 250 rpm.
Day 3
8. Inoculate 100 ml LB medium with 1 ml of saturated overnight culture.
9. Shake at 37oC until OD600 = 0.4 (usually 2-3 h).
10. Place in an ice bath for 10 min.
[After this point, the cells must be placed on ice!]
11. Pre-cool solution, centrifuge, pipette tips, falcon, and eppendorf.
12. Transfer the culture into two pre-chilled 50ml falcon.
13. Centrifuge at 2700 x g for 10 min at 4oC
14. Remove the medium, and resuspend the cell pellet with 1.6 ml ice-cold 100 mM CaCl2 by swirling on ice gently.
15. Incubate on ice for 30 min.
16. Centrifuge at 2700 x g for 10 minutes at 4oC.
17. Remove the medium, and resuspend the cell pellet with 1.6 ml ice-cold 100 mM CaCl2 by swirling on ice gently.
18. Incubate on ice for 20 min.
19. Pool all cells into one tube and add 0.5 ml ice-cold 80% glycerol and swirl to mix.
20. Freeze 100 μl aliquots in liquid nitrogen.
21. Store in -80oC.
Back To Top
 
1.6 Bacterial Transformation
1. Thaw competent cell on ice.
2. Add 50 - 100 ng DNA to competent cell culture.
3. Put in ice for 30 min.
4. Heat shock at 42oC for 1 min.
5. Put in ice for 2 min.
6. Add 1 ml SOC medium.
7. Incubate at 37oC for 90 min with shaking (~ 250 rpm).
8. Spread plate (with suitable antibiotics)
9. Spin down the remaining cells and discard large amount supernatant (1 ml).
10. Resuspend the cell pellet and spread plate.
11. Incubate at 37oC overnight (preferably ~16 – 20 h).
Back To Top
 
1.7 Inoculation
1. Pick colonies and culture in 2 - 3 ml LB broth with antibiotics.
2.    Incubate at 37oC for 12 - 16 h with shaking at 250 rpm.
Back To Top
 
1.8 Plasmid DNA Preparation (Miniprep)
1. Resuspend pelleted bacterial cells in 250 µl Buffer P1 and transfer to a microcentrifuge tube.
2. Add 250 µl Buffer P2 and mix thoroughly by inverting the tube 4 – 6 times and incubate 2 min at room temperature. Do not vortex!
3. Add 350 µl Buffer N3 and mix immediately and thoroughly by inverting the tube 4 – 6 times.
4. Centrifuge for 10 min at 13,000 rpm in a table-top microcentrifuge. A compact white pellet will form.
5. Apply the supernatants from step 4 to the QIAprep spin column
6. Centrifuge for 30–60 s. Discard the flow-through.
7. Recommended: Wash the QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30–60 s. Discard the flow-through.
8. Wash QIAprep spin column by adding 0.75 ml Buffer PE and centrifuging for 30 – 60 s.
9. Discard the flow-through, and centrifuge for an additional 1 min to remove residual wash buffer.
10. Place the QIAprep column in a clean 1.5 ml microcentrifuge tube.
11. Add 50 µl Buffer EB (10 mM Tris·Cl, pH 8.5) or water to the center of each QIAprep spin column, let stand for 1 min, and centrifuge for 1 min to elute the DNA.
Back To Top
 
1.9 Double Digestion
1. Mix the components as follows to prepare a 50 μl reaction mixture:
37.6 μl    Insert/Vector# + ddH2O*
5 μl         10X NEB Buffer3
5 μl         10X BSA
1.2 μl      Enzyme 1
1.2 μl      Enzyme 2
# At least 200ng DNA should be added
* Water is added first and the template the last
2.  Incubate the reaction mixture at 37oC for 2 h.
Buffer Chart
http://www.neb.com/nebecomm/tech_reference/restriction_enzymes/buffer_activity_restriction_enzymes.asp
Back To Top
 
1.10 Gel Extraction*
Excise the DNA fragment from the agarose gel with a clean, sharp scalpel.
Weigh the gel slice in a 2 ml centrifuge tube. Add 3 volumes of Buffer QG to 1 volume of gel (100 mg ~ 100 μl).
Incubate at 50°C until the gel slice has completely dissolved (around 10 min). Mix by vortexing the tube every 2 min during the incubation.
After the gel slice has dissolved completely, check that the color of the mixture is yellow. If the color of the mixture is orange or violet, add 5-10 μl of 3 M sodium acetate (pH 5.0), and mix until it turns to yellow.
Add 1 gel volume of isopropanol to the sample and mix (for DNA fragments < 500 bp and > 4 kb).
Place a QIAquick spin column in a provided 2 ml collection tube.
To bind DNA, apply the sample to the QIAquick column, and centrifuge for 30 s. (The maximum volume of the column reservoir is 800 μl. For sample volumes of more than 800 μl, simply load and spin again).
Discard flow-through and place QIAquick column in the same collection tube. Collection tubes are re-used to reduce plastic waste.
Add 0.5 ml of Buffer QG to QIAquick column and centrifuge for 30 s to remove any traces of agarose.
To wash, add 0.75 ml of Buffer PE to QIAquick column, stand for 2 min and centrifuge for 30 s.
Discard the flow-through and centrifuge the QIAquick column for an additional 30 s and air-dry for 2 min (This step can ensure all ethanol is removed and the column is NOT over-dry)!
Place QIAquick column into a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 35 μl of 50oC ddH2O to the center of the QIAquick membrane, wait for 2 min and centrifuge for 2 min at maximum speed.
*Adopted from CUHK iGEM 2010 protocol.
Back To Top
 
1.11 Ligation
3. Mix the components as follows to prepare a 20 μl reaction mixture:
2 μl         10X ligation buffer
1 μl          T4 DNA ligase
14 μl       Vector
3 μl          Insert
4. Incubate the reactions at 16oC overnight, or 22oC for 1 h.
Back To Top
 
2.1 Cell movement test
Pre-culture E. coli for 1 h in LB medium containing 2 μM all-trans retinal.
Transfer 6 μl of pre-cultured solution onto plate with 0.5% agar and 2 μM all-trans retinal
Incubate the plate for 24 h under specific wavelength of unidirectional light.
Measure the cell movement.
Back To Top
 
2.2 Sensory Rhodopsin-induced Gene Expression Test
Culture E. coli in LB medium with and without light for 24 hrs.
Measure the OD measurement of the corresponding reporter gene.
Back To Top
 
 
Home   |    Team    |    Project   |    Biobricks    |    Human Practice    |   Safety    |    Documentation   |    Acknowledgement
 
Address: Rm. 184, Science Centre, CUHK
Email: kingchan@cuhk.edu.hk  Tel: (852)-39434420  Fax: (852)-26037246
©Copyright CUHK iGEM Team 2012, All Rights Reserved.
Copyright © 2012 Apycom jQuery Menus
Retrieved from " http://2012.igem.org/Team:Hong_Kong-CUHK/DOC_PROTOCOLS "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
