<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Debrecen_Hungary_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Debrecen_Hungary/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Debrecen_Hungary/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Debrecen_Hungary/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Debrecen_Hungary/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Debrecen_Hungary/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Debrecen Hungary/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV style="width: 100%; text-align: center; margin:auto;"><UL id="nav"><LI><A href="https://2011.igem.org/Team:Debrecen_Hungary"><SPAN>Home</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Debrecen_Hungary/Team"><SPAN>Team</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Debrecen_Hungary/Team#About_the_team">Meet the team</A></LI><LI><A href="https://2011.igem.org/Team:Debrecen_Hungary/Team#Gallery">Gallery</A></LI><LI><A href="https://2011.igem.org/Team:Debrecen_Hungary/Team#Debrecen">About Debrecen</A></LI><LI><A href="https://2011.igem.org/Team:Debrecen_Hungary/Team#University_of_Debrecen">University of Debrecen</A></LI><LI><A href="https://2011.igem.org/Team:Debrecen_Hungary/Team#The_Lab">The Lab</A></LI></UL><LI><A href="https://2011.igem.org/Team:Debrecen_Hungary/Project"><SPAN>Project</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Debrecen_Hungary/Project#Abstract">Abstract</A></LI><LI><A href="https://2011.igem.org/Team:Debrecen_Hungary/Project#Project_description">Description</A></LI><LI><A href="https://2011.igem.org/Team:Debrecen_Hungary/Project#Basics">Basics</A></LI><LI><A href="https://2011.igem.org/Team:Debrecen_Hungary/Project#The_Experiments">Experiments</A></LI><LI><A href="https://2011.igem.org/Team:Debrecen_Hungary/Project#Results">Results</A></LI><LI><A href="https://2011.igem.org/Team:Debrecen_Hungary/Project#Contributions_to_the_project">Contributions</A></LI><LI><A href="https://igem.org/Team.cgi?year=2011&amp;team_name=Debrecen_Hungary"><SPAN>Official Team Profile</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Debrecen_Hungary/Parts"><SPAN>Submitted Parts</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Debrecen_Hungary/Protocols"><SPAN>Protocols</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Debrecen_Hungary/Protocols#Team.27s_notebook">Notebook</A></LI><LI><A href="https://2011.igem.org/Team:Debrecen_Hungary/Protocols#Protocols_of_Bacterial_work_and_cloning">Bacterial/Cloning</A></LI><LI><A href="https://2011.igem.org/Team:Debrecen_Hungary/Protocols#Protocols_from_tissue_culture_room">Cell culture</A></LI><LI><A href="https://2011.igem.org/Team:Debrecen_Hungary/Protocols#Measurement_of_Luciferase_reporter">Luciferase</A></LI><LI><A href="https://2011.igem.org/Team:Debrecen_Hungary/Protocols#Apoptosis">Apoptosis</A></LI><LI><A href="https://2011.igem.org/Team:Debrecen_Hungary/Sponsors"><SPAN>Sponsors</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Debrecen_Hungary/More"><SPAN>More</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Debrecen_Hungary/More#Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Debrecen_Hungary/More#Testing_of_arsenic_biosensor">Arsenic</A></LI><LI><A href="https://2011.igem.org/Team:Debrecen_Hungary/More#Hosting_High_School_Students">Hosting</A></LI></DIV><H2><SPAN class="mw-headline" id="Abstract"><B>Abstract</B></SPAN></H2><P>Nuclear hormone receptors (NHRs) are ligand activated transcription factors. They are able to regulate the expression of their target genes by direct DNA-binding, in a ligand-dependent manner. NHRs bear high homology to each other and are modular into distinct regions: N-terminal regulatory region, DNA-binding domain, a Hinge region, Ligand binding domain (LBD) and a C-terminal region. Some Nematode NHRs can be activated by extracted oil contamination of the soil so they can use as possible oil sensors in the future. Zinc finger motifs are the tools of NHRs to bind DNA and regulate gene expression directly. These tiny elements can be tested as direct gene regulators. Controlled gene induction can also lead to programmed cell death which is a less harmful tool in order to quit non-functionable cells.
</P><H2><SPAN class="mw-headline" id="Our_motivation"><B>Our motivation</B></SPAN></H2><P><B>Do we have enough information about our world?</B></P><P><B>Do we have the best tools to characterize the very complex world of lipids?</B></P><P>We deal with lipids everyday: our food, our fuel our body is full of various types of lipids. We decided to create the building blocks of a Genetically Engienered Machine that is able to identify specific lipids, to complete genetic programs if those lipids are detected and to be able of self-destruction, once their job was completed.
</P><P>So, let's see the details!
</P><H2><SPAN class="mw-headline" id="Project_description"><B>Project description</B></SPAN></H2><P>Lipids constitute a broad group of naturally occurring molecules which include fats, waxes, sterols, fat-soluble vitamins (such as vitamins A, D, E and K), monoglycerides, diglycerides, phospholipids, and others. The main biological functions of lipids include energy storage, as structural components of cell membranes, and as important signaling molecules.
</P><P>Although there are certain types of lipids their specific investigation is very difficult. Sophisticated chromatographies, mass spectrometry, nuclear magnetic resonance (NMR) spectroscopy, fluorescence spectroscopy or dual polarisation interferometry are used in certain lipidomics centers. Another possibility is to investigate their specific regulatory effects. The receivers are nuclear hormone receptors, which is a conserved family of proteins responsible for sensing lipids; they may be viewed as lipid activated transcription factors.
</P><P>Ligand binding domains respond only to predefined lipids and selectively activate predetermined gene expression. The tool for regulating directly the gene expression of the DNA binding domains of nuclear hormone receptors are tiny zinc finger motifs. These elements bind straight to the DNA and can act as direct gene expression regulators so they can operate as specific effectors.
</P><P>A machine has to be switched off after using it. In our case the power button means that the genetically modified cells have to be removed without any harm caused in the system. A perfect mechanism for that is inducing Programmed Cell Death in a controlled way.
</P><P>The parts designed by Team: Debrecen-Hungary 2010 allows the ultra sensitive detection of xenobiotic activators in the environment (e.g. oil derived traces in water and soil), the assessment of lipid soluble vitamins, steroids, xenobiotics etc. in food and the potential of inducing specific genetic programs in transfected cells or model organisms with a possibility to destroy these modified cells by an independent induced apoptosis effector system.
</P><P>Our team decided to work with the parts modelled by last year's team in order to further characterize and biuld new composite parts and expression vectors from them which led to compose a lipid driven genetically engineered machine which either regulated by mineral oil or lipids form different kind of food. 
</P><H2><SPAN class="mw-headline" id="Basics"><B>Basics</B></SPAN></H2><H3><SPAN class="mw-headline" id="Synthetic_Biology"> Synthetic Biology </SPAN></H3><P>Synthetic biology is a relatively new area of biological research; similar to many other new scientific fields it has many definitions. The best way to express the meaning of synthetic biology is to understand the desired end-result: engineering of complex biological systems. These systems are best thought of as analogues of everyday machinery: cogwheels, levers, timers, button and buzzers (in this case a clock), only in the case of biological systems (molecular ones) cells, DNA, proteins, lipids, sugars and RNA are the “parts” of the system. 
</P><P>Similar to mechanical engineering (or every other engineering branch) there is a need for standards (consensus way of doing things), abstraction (simple and unified way of thinking about the parts of a system), and modularity (how these parts interact to become the device, or several devices into a system). Thus a good definition for synthetic biology could be engineering of molecular (for the time being) biological systems according to preset standard parts. The international genetically engineered machines and associated parts registry are, to date, one of the largest registries for standard parts in use for synthetic biology. Free information can be found in the registry regarding parts, devices and modules all inputted by various teams worldwide. 
</P><P>Eukaryotic synthetic biology is still in its infancy. The large kingdom of metazoans includes all multicellular eukaryotes such as mammalians, arthropods and nematodes. No standard chassis (framework) exists for the animal kingdom which makes them far less popular then the famous E.coli. The protein modules derived from metazoans (like Drosophila or C. elegans) are functional in yeasts also. Very few iGEM teams (or even labs outside iGEM) have chosen to toggle the animal chassis (two notable examples are team Heidelberg 2009 and team Slovenia 2006). The amount of available compatible parts is limited, which severely restricts the options of creating complex biological devices. Nearly no imagination is required for designing tools, since their analogues already exist in the bacterial chassis. The possible use of such systems is unlimited. Field’s such as of environment, medicine, energy and research all gain to profit from the development of animal synthetic biology. 
</P><P>Systems requiring gene expression input in eukaryotic synthetic biology systems require a way to standardize gene expression, a complicated task. The way from gene to protein contains many steps of possible error: transcription factor binding, promoter strength, recruitment of auxiliary proteins, nuclear RNA synthesis and many more steps finally leading to translation, folding, cleaving and delivery (but hey, you have to start somewhere). 
</P><P>Our team was interested at designing eukaryotic synthetic biology tools related to PoPs. PoPs (polymerase per second), the flying Dutchman of synthetic biology, is a number which represents the rate (base pair per second) at which RNA polymerase crosses past a given DNA position. Currently, no in vivo technique for measuring PoPS directly exists; it can be estimated indirectly by measuring other parameters (eg protein expression or enzyme activity). Nevertheless it is still a useful abstraction for thinking about transcription-based logic devices and it allows the engineer to define devices. Our aim, was not only to infer PoPs but to devise a way to titrate it remotely. 
</P><H3><SPAN class="mw-headline" id="Nuclear_Hormone_Receptors"> Nuclear Hormone Receptors </SPAN></H3><P>Nuclear receptors are ligand activated transcription factors. As such, they are able to regulate the expression of their target genes by direct DNA-binding, in a ligand-dependent manner. They play a central role in endocrine signaling, regulation of embryonic and adult development and differentiation. Many nuclear receptors, are among the primary targets of drug discovery because of their diverse biological actions. Nuclear receptors bear high homology to each other and are modular into distinct domains: N-terminal regulatory domain, DNA-binding domain, a Hinge region, Ligand binding domain (LBD) and a C-terminal domain. 
</P><P>Ligand binding domain (LBD) is a well conserved domain amongst various nuclear receptors whose structure usually referred to as an alpha helical sandwich fold. The LBD shows some diversity among nuclear receptors as it is a site for receptor-specific events. It possesses transactivation ability and contains a ligand-binding pocket as well as the main interaction surfaces for other proteins. 
</P><P>The DNA-binding domain (DBD) contains two Zinc-finger motifs and is linked to the LBD by a highly flexible hinge region. This segment of the nuclear receptors holds the ability to recognize and bind to preferred or specific DNA-motifs, the response elements. 
</P><P>A partially unexpected and amazing feature of the nuclear receptors is that the above mentioned domains can be swapped. It was Vincent Giguere and his colleagues, who first devised a technique called cotransfection assay and developed it to the domain-swap technique.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Basic structure of Nuclear hormone Receptors.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Zinc_Fingers"><B>Zinc Fingers</B></SPAN></H3><P>Zinc fingers are small protein structural motifs binding one or more zinc ions to help stabilize their folds. Zinc finger proteins often bind to DNA and RNA because their shape allows close interaction of the domain with the nucleotides of DNA and RNA, and they can also bind to proteins and small molecules. Their nucleotide binding properties allow them to function in regulating gene expression and in virus assembly.
</P><P>Zinc fingers coordinate zinc ions with a combination of cysteine and histidine residues. They can be classified by the type and order of these zinc coordinating residues (e.g., Cys2His2, Cys4, and Cys6). A more systematic method classifies them into different &quot;fold groups&quot; based on the overall shape of the protein backbone in the folded domain. The most common &quot;fold groups&quot; of zinc fingers are the Cys2His2-like (the &quot;classic zinc finger&quot;), treble clef, and zinc ribbon. The DNA binding domain of nuclear hormone receptors is a highly conserved domain containing two zinc fingers that binds to specific sequences of DNA called hormone response elements (HRE).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Cartoon representation of the protein Zif268 (blue) containing three zinc fingers in complex with DNA (orange).</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Caenorhabditis_elegans"> Caenorhabditis elegans </SPAN></H3><P>Caenorhabditis elegans is a free-living, transparent nematode (roundworm), about 1 mm in length, which lives in temperate soil environments C. elegans is intensively studied as a model organism in biology for a variety of reasons. The developmental fate of every single somatic cell (959 in the adult hermaphrodite; 1031 in the adult male) has been mapped out. The C.elegans genome harbors 284 nuclear receptors (a striking figure), which have been shown to control traits such as sex determination, larva development, life span, neuronal growth and identity and much more. As far as nuclear receptors go, they are a gold mine.
</P><H3><SPAN class="mw-headline" id="Apoptosis"> Apoptosis </SPAN></H3><P>Apoptosis, the process of programmed cell death (PCD), is an elaborate cellular homeostasis mechanism that ensures correct development and function of multicellular organisms. Biochemical events lead to characteristic changes (morphology) and the death of cells. 
</P><P>It has a crucial role in maintaining tissue homeostasis as provides the appropriate rate of cellular turnover. Potentially dangerous, poorly differentiated or excess cells are removed by apoptosis without local inflammation from leakage of cell contents. Dying by apoptosis in many cases has primary importance: lens cells, which lost their nucleus, need the vision, the corneocytes without nucleus and organelles provide mechanical protection on the outer surface of the skin. 
</P><P>Apoptosis might occur in pathological conditions, mainly in secondary, as a part of the defensive mechanism, helping to eliminate the viral infected or malignant cells. The abnormality of the central molecules and genes of apoptosis may play a role in the development of certain diseases: degenerative disease, autoimmune disease, tumor formation and malformations. 
</P><P>Many pathways and signals lead to apoptosis, but there is only one mechanism that actually causes the death of a cell. After a cell receives stimulus, it undergoes organized degradation of cellular organelles by activated proteolytic caspases. A cell undergoing apoptosis shows characteristic morphology.
</P><H2><SPAN class="mw-headline" id="The_Experiments"><B>The Experiments</B></SPAN></H2><H3><SPAN class="mw-headline" id="Cotransfection_Assay.2C_Two_Hybrid_Assay"> Cotransfection Assay, Two Hybrid Assay </SPAN></H3><P>Cotransfection assay requires two types of plasmids to be cotransfected into a cell; an expression vector, coding a functional NR and a reporter-plasmid, harboring an inducible promoter that regulates a reporter-gene the expression level of which can easily be detected and quantified. By this method the NR-activity and ligand-potency can be studied. As a result of the unique domain-structure of NRs, the domain-swap technique was developed, using chimeric proteins; the DBD of a known receptor is fused to any other receptor. This made ligand screening possible. 
</P><P>The two-hybrid assay is a technique in molecular biology which can be used to investigate protein-protein interactions. The interaction of a DNA-binding bait-protein and the prey-protein with a ligand binding and activating ability is shown by the activated reporter. 
</P><P>Multiple applications of these methods have appeared. It is possible to investigate: 
</P><P>- the response elements by changing the sequence that is recognized by the DBD 
</P><P>- potential dimer partners and co-regulators by creating specific bait-prey sets 
</P><P>- potential ligands and activating factors 
</P><P><B>Our main concept was to introduce the iGEM-method into this well established area of research. We showed that these systems (they were published before) can be reproduced with the use of our pre-designed Parts. We could also present that it is possible to take the next step: to investigate new samples as well, i.e. to test the effects of different biological samples on NR-action by our Kits.</B></P><P>This image shows the basic concept of the two-hybrid system: the DNA-binding bait-protein (Gal-fused, in our case), and the activator-domain fused to the prey-protein (mostly VP-fusion, in our case) and the reporter gene with the specific binding site. Yeast Gal4 is a common DBD used for this techniques purpose. Commonly used reporter genes include the product of the LacZ gene (Beta galactosidase) and Luciferase.
</P><P>For a detailed protocol on Luciferase assay please see Protocols <A href="https://2011.igem.org/Team:Debrecen_Hungary/Protocols" class="external autonumber" rel="nofollow">[1]</A>.
</P><H3><SPAN class="mw-headline" id="Luciferase"> Luciferase </SPAN></H3><P>Luciferase is an enzyme class able to produce bioluminescence by oxidizing the substrate luciferin. &quot;Firefly luciferase&quot; as a laboratory reagent usually refers to P. pyralis luciferase. Its emission can be measured photometrically and hence used to deduce the protein enzyme concentration through standardized methods.
</P><H3><SPAN class="mw-headline" id="Cos-1_cells"> Cos-1 cells </SPAN></H3><P>Cos-1 cells (acronym for CV-1 simian origin, SV-40 viral positive) is cell line derived from the African green monkey kidney cells. It is also often used to transfect cells in tissue culture conditions to produce recombinant proteins for molecular biology, biochemistry, and cell biology experiments. Two forms of COS cell lines commonly used are COS-1 and COS-7. The cell line was obtained by immortalizing the original CV-1 cells with SV-40 virus genome. This allows the production of large T antigen but has a defect in genomic replication.
</P><H2><SPAN class="mw-headline" id="Results"><B>Results</B></SPAN></H2><H3><SPAN class="mw-headline" id="Cloning"> Cloning </SPAN></H3><P>Firstly, we had to generate expression vectors that can be used in mammalian expression system. The cloning work of Team Debrecen-Hungary 2011 resulted 3 composite parts and one fully synthetic biology comaptible expression vector from pSB1A3. All of them have been tested by restriction enzyme analysis and by sequencing. Furthermore we have tested two Zn-fingers and fusion vectors by luciferase assay.
</P><H4><SPAN class="mw-headline" id="NHR64-GAL4_fusion"><FONT color="green">NHR64-GAL4 fusion</FONT></SPAN></H4><P>NHR64 LBD of C. elegans has been inserted after Gal4 in pSB1C3, resulting an insert with the lenght of 1008 bp. After that NHR64-GAL4 was transferred into pCDNA3.1 to express it in certain cell lines.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Gel picture of Gal4-NHR64 fusion part in pSB1C3 and in pCDNA3.1 expression vector.</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="CD38-EcR_LBD-pSB1C3.2F_ADRB-EcR_LBD-pSB1C3_fusions"><FONT color="green">CD38-EcR LBD-pSB1C3/ ADRB-EcR LBD-pSB1C3 fusions</FONT></SPAN></H4><P>The CD38/ADRB (Modelled by Team: Debrecen-Hungary 2010) are Zinc-finger proteins that can bind to the DNA, and CD38/ADRB (530bp) were fused with EcR-LBD. The EcR (768 bp) is a nuclear receptor that is the ortolog of mammalian RXR and contains ligand binding domain. After the fusion, the insert length was 1298bp. Then CD38/ADRB-EcR LBD were transferred into pCDNA3.1.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">pSB1C3-ADRB-ECR-LBD and pSB1C3-CD38-ECR-LBD (circular and NotI digested).</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="BAX_expressing_vector_from_pSB1A3"><FONT color="green">BAX expressing vector from pSB1A3</FONT></SPAN></H4><P>Three basic parts from pSB1C3 have been ligated into pSB1A3 consecutively by using chloramphenicol and ampicillin assembly in order to create an expression vector from the ampicillin resistance carrying backbone. The first step was to insert 321 bp long TRE-CMV (Tetracyclin Response Element- Minimal Cytomegalovirus promoter) into pSB1A3 which has the length of 2157 bp. Then BAX (573 bp) were inserted after the TRE- CMV, before PolyA tail (191 bp) was ligated in the end. As the negative control of the expression vector built up by Team: Debrecen-Hungary 2010 from standard biobrick parts TRE- CMV and PolyA tail were ligated together into pSB1C3 and pSB1A3.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:700px;"><DIV class="thumbcaption">TRE-Bax-PolyA in pSB1A3 results in a new expression vector from standard Biobrick parts.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Lipid_Sensors:_Nuclear_Hormone_Receptors"> Lipid Sensors: Nuclear Hormone Receptors </SPAN></H3><P>To test our system, we transfected the Gal-PPARg plasmid into COS-1 and 293T cell lines and applied an agonist treatment (Rosiglitazone,BRL). The expressed Gal-PPARg protein was able to bind to the UAS-sequence of the co-transfected luciferase reporter (MH100-TK-luciferase) and activate it in a dose dependent manner. This agonist is a higly potent activator of the PPARg, which explains the high level of luciferase activity.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Luciferase activity after Gal-PPARg transfection and Rosiglitazone treatment.</DIV></DIV></DIV></DIV><P>As the set up system was ready to test PPARg-ligands, we next tested oil-extracts (extracted from oil sand samples). Suprisingly, the treatment with the oil-extract caused dose dependent activation of PPARg. The level of activation was clearly lower than the one we could detect with the specific agonist, but the dose dependence gave us a hint that the oil extract can activate the PPARg nuclear receptor.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Luciferase activity after Gal-PPARg transfection and Rosiglitazone treatment.</DIV></DIV></DIV></DIV><P>Our idea was to apply the nuclear receptors as parts of a system built up from sensors and effectors. As lipids are relatively hard to detect and investigate and some classes of the nuclear receptors are specialized to lipid sensing in living organisms, we gained a search for potential oil sensors. This is how we came accross with the C. elegans worm and its nuclear receptors (NHR). Due to its living habitat the nuclear receptors of this worm seemed to be good candidates for new oil-sensors. We created plasmids (using iGEM-standards) by the fusion of a Gal UAS-binding domain and a NHR (Gal-NHR8, Gal-NHR64). No agonist ligands are known for these receptors, so far.
</P><P>
Next, we transfected the Gal-fused worm nuclear receptors (NHR8, NHR64) into mammalian cell lines along with the other parts of the luciferase reporter system, treated the cells with the oil extract and measured the luciferase activity. Latter was the result of and activated luciferase gene, that was activated by the Gal-NHRs bound to the UAS, which became active as a result of the oil-treatment. Constructs baring NHR8 or NHR64 showed dose dependent activation upon the addition of the oil extracts to the cells. These NHRs acted as potential oil-sensors on which even a mammalian-cell based detection system can be built.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Activation of Gal-NHR8 and Gal-NHR64 to oil sand extractions.</DIV></DIV></DIV></DIV><P>We were curious, wether other types of lipids could be detected by our system, so next we tested lipids not from the soil, but from food. Lipids were extracted from different types of cheese. We used above mentioned system to carry out similar experiments, but this time with cheese extracts. We could detect PPARg activity in each used cell line after the treatment with the Bresse bleu extract. The activity was dose dependent. This is not just the sign of a molecule of lidid nature in the extract, but a molecule that acts as a PPARg agonist. Even more suprisingly, this extract could activate the Gal-NHR8 system as well. It is important to mention though, that it was not necessarily the same compound of the cheese extract that showed these activities on different receptors.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Activation of PPARg to Bresse bleu cheese extract in different cell lines.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Activation of NHR8 to Blesse bleu cheese extract.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Effector:_Gene_expression_regulated_by_zinc_fingers"> Effector: Gene expression regulated by zinc fingers </SPAN></H3><P>One important feature of most of the nuclear receptors is that they are built up of well conserved protein motifs, domains. Still these motifs are the key to their specific action. The specificity of the ligand binding domain (LBD) was just shown earlier by testing different LBDs for sensing activating agents in oil and cheese extracts. The other domain that is responsible for selectivity is the DNA binding domain (DBD). In our previous experiments the DBD was replaced with a Gal-domain, that was necessary for the lucferase system when the LBD was being tested.
</P><P>To show that even „imaginary” NRs can be built by the fusion of a potent LBD and a well designed DBD, we started to design a NR-specific DBD that contains a Zn-finger motif that is able to recognize a consensus retinoic acid receptor (RAR) response element (the region of an RAR target gene that the RAR is able to recognize and bind to).  The construct is a newly designed Zn-finger motif. The design was based on the promoter region of CD38 gene (an RAR target gene). The Zn-finger sequence was then inserted into the pcDNA vector (ZN-FINGER). The DNA binding capacity of this protein was tested on a DR5-luciferase reporter (that carries a consensus RAR-binding sequence) by a competition experiment.
</P><P>The DR5-sequence of the reporter could be recognized by the endogenous RAR of our cell lines, as it is shown by the base activity. If our ZN-FINGER was able to bind to the same sequence, a competition should occur between the endogenous RARs and our ZN-FINGERS. As the basal activity was reduced with increasing the amount of the transfected ZN-FINGER plasmids, we concluded that the designed protein can bind to the DR5. To show the effect on an enhanced level, we activated the system (by AM580 – RAR agonist ligand). By this, even larger the activity grew without the ZN-FINGERS. But again, by adding the ZN-FINGERS to the system, the level of activity was reduced.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Zinc finger protein competes with endogenous RAR <I>without</I> RAR agonist ligand. The numbers show the ratio between the ZN-FINGERS and the buffer plasmids: 1.ZN-FINGERS:buffer plasmids-1:8; 2.ZN-FINGERS:buffer plasmids-1:2; 3.Only ZN-FINGERS.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Zinc finger protein competes with endogenous RAR <I>in the presence of</I> RAR agonist ligand. The numbers show the ratio between the ZN-FINGERS and the buffer plasmids: 1.ZN-FINGERS:buffer plasmids-1:8; 2.ZN-FINGERS:buffer plasmids-1:2; 3.Only ZN-FINGERS.</DIV></DIV></DIV></DIV><P>
By these experiment we have shown that based on the NR-system a great variety of sensor-effector system can be built. The different LBDs give the arsenal of sensing. From dietary components to different elements of the soil. The other side is the effector side that –via the different DBDs- can also be extremely variable. These sides are potentially switchable; different LBDs with different DBDs. Even at interspecific level.
</P><H3><SPAN class="mw-headline" id="Shut_down:_Apoptosis_inducing_.28mechanism_to_switch_off_cells_in_a_controlable_way.29"> Shut down: Apoptosis inducing (mechanism to switch off cells in a controlable way) </SPAN></H3><P>Fully Synthetic Biology compatible Bax proapoptotic protein expressing vector (BBa_K626000) was assessed from pSB1A3 which was transfected into COS-1, 293T and THP-1 cells. The functionality of the expression vector was proved by immunoblotting by using mouse primary antibody against human Bax (Santa-Cruz Biotechnology) where high protein expression could be detected 24-48 hours after transfection compared to the control cells.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:641px;"><DIV class="thumbcaption">Detection of the protein level of Bax by Western blot. The level of Bax is the highest 24 hours after transfection.</DIV></DIV></DIV></DIV><P>Bax (Bcl-2-associated X) protein is a pro-apoptotic member of the Bcl-2 protein family. It contains three BH (Bcl-2 homology) domains, can form homo- and heterodimers and plays essential role in the formation of MOMP (Mitochondrial outer membrane permeabilization) leading to apoptosis through cytochrome c release which is the most frequent type of PCD (Programmed cell death).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">The action of Bax protein in programmed cell death.</DIV></DIV></DIV></DIV><P>The Bax overexpression led to apoptosis in the transfected cells which was proved by DAPI (Sigma-Aldrich) staining. The nuclei of the apoptotic cells show fragmentation and bleb formation which are well-known markers of apoptosis. They also showed Annexin V positivity detected by flow cytometry.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Picture of DAPI staining: The nuclear fragmentation was clearly visible by Hoechst staining, which is a known marker of apoptosis. The arrows show apoptotic cells.</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Contributions_to_the_project"><B>Contributions to the project</B></SPAN></H2><P>All experiments were performed by our registered team members, and we all take responsibility for the contents of our wiki page and the parts uploaded by us to the Registry of Standard Biological Parts. For detailed contributions see the list below.
</P><H3><SPAN class="mw-headline" id="Contributions_member_by_member:"><B>Contributions member by member:</B></SPAN></H3><P><B>Students</B> (in alphabetical order)
</P><P><I>Tímea Beregi</I>: cloning, transfections, wiki writing.
</P><P><I>Éva Bernadett Bényei</I>: Our young high school student took part in cloning, bacterial and tissue culture works, and helped us to survive evenings in Lab by her stories and games during incubation times.  
</P><P><I>Beáta Boros-Oláh</I>: She participated in all lab works including cloning, bacterial works, tissue culture works, Luciferase/cotransfection assays, and also worked on uploading matter to the wiki. Making photos of labwork.
</P><P><I>Lilla Ozgyin</I>: Cloning and tissue culture works, transfections, luciferase assays and writing matter to the Registry and  coordinating the wiki mission. Protocol writing, arsenic biosensor testing, extraction of contaminated environmental samples.
</P><P><I>Petra Sepsi</I>: Our enthusiastic team member took part in every cloning, bacterial and tissue culture works, luciferase assays and she uploaded materials to the wiki. Making photos on labwork.
</P><P><I>Ildikó Tándor</I>: She helped us in every possible way, including cloning, bacterial and cell culture works, luciferase assays and wiki writing. Uploading materials to the Registry.
</P><P><B>Advisors</B> (in alphabetical order)
</P><P><I>Péter Brázda</I>: He supervised planning and evaluation of luciferase assays, participated in wet-lab education and the wiki-works. Mentored our high school student to get acquainted with the concept of synthetic biology. He also gave us useful advices on the implementation of processes in which the students had some difficulties.
</P><P><I>Gergely Nagy</I>: Gergely was a real advisor by giving us a dozen of tips on labwork, and thought us the scientific way of thinking. He gave us help in the implementation of results and directed us towards critical way of thinking with his fancy imagination as well.
</P><P><I>Katalin Sándor</I>: She was the mentor of our new students in the first few weeks, helping them to get to be in practice with basic laboratory methods, and she passed her experiences by giving presentations to us.
</P><P><B>Instructors</B> (in alphabetical order)
</P><P><I>Bálint L. Bálint M.D. Ph.D.</I>: He was our main coordinator by initiating the project, he made the team recruitment, fund raising, some of the PR and laboratory setup. He supervised the first lab experiments and planned some of the experiments later on, ordered parts and reagents, organized finances. He supervised wiki and partsregistry work as well. He dealed with fundraising.
</P><P><I>Bence Dániel</I>: He was a supervisor of the whole labwork, he gave us new directions when the lab work stucked by unexpected reasons. He shared his huge experience of nuclear receptors, trained our new students and helped us with the cotransfection assays. He also worked on the wiki project and uploaded materials to the registry. He performed the sequencing of our new parts.
</P><P><I>Endre Károly Kristóf M.D.</I>: Endre helped us in the apoptosis project (as he is experienced in examining apoptosis) and in the training of new students during the summer. He also gave help in writing an article about the team, writing the abstract, uploading files to the Registry and our team wiki. Participating in fund raising, in silico design, video and media project. Uploading materials of project description to the wiki and informations about the composite and some basic parts to the partsregistry.
</P><P><I>Gábor Zahuczky Ph.D.</I>: He supported the initiation of the project, fund raising, organization of finances and administration. Participated in the design of parts and wet lab education.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Debrecen_Hungary/Project">http://2011.igem.org/Team:Debrecen_Hungary/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Debrecen_Hungary/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Debrecen_Hungary/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Debrecen_Hungary/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Debrecen_Hungary/Project&amp;oldid=161580" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>