"
Page
Discussion
View source
History
teams
Log in
 
Team:Debrecen Hungary/Protocols
From 2011.igem.org
Home
Team
Meet the team
Gallery
About Debrecen
University of Debrecen
The Lab
Project
Abstract
Description
Basics
Experiments
Results
Contributions
Official Team Profile
Submitted Parts
Protocols
Notebook
Bacterial/Cloning
Cell culture
Luciferase
Apoptosis
Sponsors
More
Safety
Arsenic
Hosting
Team's notebook
All members of the team who has worked in the wet lab this year carried his/her own detailed notebook every day. Our Student team members had to learn how to write useful notes and handle the notes when that was necessary. One of our instructors, Bálint László Bálint shared his own rules on Lab notebook writing to teach us precisity and accuracy. I must say that his guide was very profitable to all of us.
The notebooks were classical paper-based so as to use when and where we needed, although we are considering to write online notebooks next year.
If you have questions about how we proceeded this summer, choose and get in touch with the appropriate team member from the following list:
Bacterial work: Beáta Boros-Oláh
Cloning: Ildikó Tándor
Cell culture work: Petra Sepsi
Luciferase/cotransfection assay: Bence Dániel
Apoptosis project: Endre Károly Kristóf
Arsenic project: Lilla Ozgyin
We are ready to answer any questions come to your mind.
Team's notebook protocols
The iGEM experience is not merely a project or a conference, but it was the way that most of our students got acquainted with the world of biological research laboratory. Pipettes, solutions, gels, electrodes, dishes and other scary machinery quickly filled our lives. From day one we saw the vast importance of teaching our students to keep a proper laboratory journal. We optimized and adjusted the protocols we used during the summer, and we would like to share them to the public. Last year we have made short films of our protocols and also shared the protocols in a text form on Open Wet Ware. With this section you can see into our daily work. For detailed protocols, see below.
Video protocols were created by last year's team members. Some of them are different from this year's, but can be useful for those who have less experience in the methodology of molecular biology. For the short films see debrecenigem2010's channel: [1] .
Protocols of Bacterial work and cloning
Making of LB medium (Lysogeny Broth, also referred as Luria-Bertani Medium)
- Scientific Background
Giuseppe Bertani published the original formula of LB, which is used for the cultivation of E. coli. It provides the main essential nutrition, containing peptides. For transformation, we use LB agar but for small and big cultures we use the original liquid LB.
- Overview
The presparation is to dissolve LB powder in distilled (MilliQ) water, and autoclave it.
- Materials
For 1 L LB Broth
1 L dH20
20g LB powder
a 1 l bottle
measuring cylinder
Laboratory scale
- Procedure
1) Take a jar of LB powder
2) Measure 20 g of LB powder in a scale ( it depends on the amount of LB you need, take 10 g if you’d like to make 500ml of a LB medium )
3) Put the powder into a 1 liter flask or bottle, (again, depends on how much you need)
4) Add the distilled water into the bottle till it reaches 1 liter
5) Shake the bottle carefully
6) Close it and stick a tape from one side of the wall vertically over the cap to the other side of the wall (which means not sterile) which will be black striped after autoclave sterilization
7) Sterilize the bottle (autoclave)
- References
Sambrook, J., E.F. Fritsch, and T. Maniatis. 1989. Molecular Cloning: A Laboratory Manual. 2nd ed.,1.25-1.28. Cold Spring Harbor Laboratory Press, Cold Spring harbor, NY, USA.
Bacterial transformation by heat shock
- Scientific Background
Transformation is the process of introducing foreign DNA (e.g plasmids, BAC) into a bacterium. Bacterial cells into which foreign DNA can be transformed are called competent. Some bacteria are naturally competent (e.g B. subtilis), whereas others such as E. coli are not naturally competent. Non-competent cells can be made competent and then transformed via one of two main approaches; chemical transformation and electroporation (our lab tends to use a protocol based on treatment with calcium-chloride-pipes-glicerol solution). We tried several protocols last year, but we found that this is the best yielding among them. This protocol may be particularly useful if you are finding that your transformations are not working or yiedling few colonies.
- Overview
Put the DNA to be introducend and your competent cells together, change incubation temperature dramatically some times then spread the cells on LB agar plates and incubate at 37 degrees overnight.
- Materials
Competent cells (we always use DH5α strain - [2] )
DNA to be introduced
Crushed ice, 1,5 ml Eppendorf tubes
42°C water bath
37°C incubator
Petri dishes with LB agar and appropriate antibiotic
- Procedure
1. Start thawing the competent cells on crushed ice (we find this cells in the -70°C fridge)
2. Put 50μl competent cells and 100-250ng DNA into a 1,5 ml tube kept on ice
3. Incubate the cells for 20 minutes on ice
4. Heat shock at 42°C for 90 seconds in water bath (do not shake!)
5. Incubate for 5 minutes on ice again
6. Spread the transformed bacteria on the Petri dishes with LB agar and the appropriate antibiotic(s) with the part name, plasmid backbone and antibiotic resistance
7. Incubate the plate at 37°C for 14 hours
- References
Sambrook, J., E.F. Fritsch, and T. Maniatis. 1989. Molecular Cloning: A Laboratory Manual. 2nd ed.,1.25-1.28. Cold Spring Harbor Laboratory Press, Cold Spring harbor, NY, USA.
MiniPrep
- Scientific Background
This protocol is designed for purification of up to 20 μg of high-copy plasmid DNA from 1–5 ml overnight cultures of E. coli in LB (Lysogeny Broth, Luria-Bertani) medium.
- Overview
This year we used GeneJet Plasmid Miniprep Kit (Thermo Scientific), and followed the provided protocol with some minor changes.
- Materials
Kit containts
table-top centrifuge at >12000 x g(10 000-14 000 rpm)
centrifuge for 50 ml tubes with relative high performance
- Notes
All purification steps should be carried out at room temperature. All centrifugations should be carried out in a table-top microcentrifuge at >12000 x g(10 000-14 000 rpm, depending on the rotor type). Use 1-5 ml of E. coli culture in LB media for purification of high-copy plasmids. For low-copy plasmids use up to 10 ml of culture.
- Procedure
0. [small culture] Next day after transformation of the bacteria pick up a colony with a pipette tip from the agar plate, wash into 1-5 ml LB medium (with the proper antibiotic) and put the tube into a shaker to 37°C for 12 hours. After incubation, centrifuge the tube in a adequate centrifuge at 10000xg for 10 minutes. Pour off the used medium.
1. Resuspend the pelleted cells in 250 μl of the Resuspension Solution. Transfer the cell suspension to a microcentrifuge tube. The bacteria should be resuspended completely by vortexing or pipetting up and down until no cell clumps remain. Note. Ensure RNase A has been added to the Resuspension Solution.
2. Add 250 μl of the Lysis Solution and mix thoroughly by inverting the tube 4-6 times until the solution becomes viscous and slightly clear. Note. Do not vortex to avoid shearing of chromosomal DNA. Do not incubate for more than 5 min to avoid denaturation of supercoiled plasmid DNA.
3. Add 350 μl of the Neutralization Solution and mix immediately and thoroughly by inverting the tube 4-6 times. Note. It is important to mix thoroughly and gently after the addition of the Neutralization Solution to avoid localized precipitation of bacterial cell debris. The neutralized bacterial lysate should become cloudy.
4. Centrifuge for 5 min to pellet cell debris and chromosomal DNA. 5 Transfer the supernatant to the supplied GeneJET™ spin column by decanting or pipetting. Avoid disturbing or transferring the white precipitate. Step Procedure
6. Centrifuge for 1 min. Discard the flow-through and place the column back into the same collection tube. Note. Do not add bleach to the flow-through, see p.7 for Safety Information.
7. Add 500 μl of the Wash Solution (diluted with ethanol prior to first use as described on p.3) to the GeneJET™ spin column. Centrifuge for 30-60 seconds and discard the flow-through. Place the column back into the same collection tube.
8. Repeat the wash procedure (step 7) using 500 μl of the Wash Solution.
9. Discard the flow-through and centrifuge for an additional 1 min to remove residual Wash Solution. This step is essential to avoid residual ethanol in plasmid preps.
10. Transfer the GeneJET™ spin column into a fresh 1.5 ml microcentrifuge tube (not included). Add 50 μl of the Elution Buffer to the center of GeneJET™ spin column membrane to elute the plasmid DNA. Take care not to contact the membrane with the pipette tip. Incubate for 2 min at room temperature and centrifuge for 2 min. Note. An additional elution step (optional) with Elution Buffer or water will recover residual DNA from the membrane and increase the overall yield by 10-20%. For elution of plasmids or cosmids >20 kb, prewarm Elution Buffer to 70°C before applying to silica membrane.
11. Discard the column and store the purified plasmid DNA at -20°C.
References
Birnboim, H.C., and Doly, J. (1979) A rapid alkaline lysis procedure for screening recombinant plasmid DNA. Nucleic Acids Res. 7, 1513–1522.
MidiPrep
- Scientific Background
This protocol is designed for the preparation of High Copy Number plasmids. We used Genopure Plasmid Midi Kit (Roche).
- Materials and notes
Sample: 50 ml E. coli culture, transformed with a high copy number plasmid. Harvest cultures at a density between 2.0 and 6.0 A600 units per ml bacterial culture.
Media: The isolation method is optimized for cultures grown in LB media; other rich media may require increased volumes of Suspension-, Lysis- and Neutralization Buffer, and an additional wash step
Plasmid size: The isolation procedure is suitable for all plasmid sizes; lysates of larger constructs (up to 100 kb) should be cleared by filtration to avoid shearing
Suspension Buffer/RNase A: To dissolve the lyophilized enzyme in Suspension Buffer, pipet 1 ml of Suspension Buffer (bottle 1, black cap) into the glass vial containing the lyophilized RNase (bottle 2, black cap). Reinsert the rubber stopper and invert the vial until all lyophilizate (including any that might stick to the rubber stopper) is dissolved. Transfer the dissolved enzyme back to the Suspension Buffer bottle (bottle 1). This is enough working solution for 60 Midi preps (isolation of up to 100 μg plasmid DNA/preparation)
If preparing aliquots of the working solution, remember that the final concentration of RNase A in the working solution must be 100 g/ml. Reconstituted buffer is stable for 6 months is stored properly (+2 to +8°C)
Neutralization Buffer: Before starting, cool it down to 4°C
Elution Buffer: Before starting, warm up the buffer to 50°C
Ethanol: Use 70% ethanol. Before starting, cool it down to 4°C
- Procedure
1. Centrifuge bacterial cells from 30 ml culture grown in LB medium by centrifuging for 10 min at 5000 rpm,4°C. Discard the supernatant. Carefully resuspend the pellet in 4 ml Suspension Buffer + RNase and mix well.
2. Add 4 ml Lysis Buffer to the suspension and mix gently by inverting the tube 6 times. Incubate 2-3 min at room temperature. To avoid shearing genomic DNA, do not vortex the suspension in Lysis Buffer. To prevent release of chromosomal DNA from the cell debris, do not incubate for more than 5 minutes.
3. Add 4 ml chilled Neutralization Buffer to the suspension. Immediately mix the suspension gently by inverting the tube 6 times until a homogenous suspension is formed. Incubate the tube 5 min on ice The solution becomes cloudy and a flocculent precipitate will form.
4. Clear the lysate by filtration. Put a folded filter into a funnel that has been inserted into a 50 ml plastic tube. Moisten the filter with a few drops of Equilibrium Buffer. Load the lysate onto the wet, folded filter and collect the flowthrough. The SDS precipitates with cellular debris when Neutralisation Buffer is added; this white precipitate should not be loaded onto the column. If the solution obtained after step 14 is not clear, remove the remaining precipitate by passing the solution over a folded filter.
5. Mount the sealing ring to the column to fix the column in the collection tube. Insert one column into one collection tube. Equilibrate the column with 2,5 ml Equilibration Buffer. Allow the column to empty by gravity flow. Discard the flowthrough.
6. Load the cleared lysate from step 4 onto the equilibrate column. Allow the column to empty by gravity flow. Discard the flowthrough.
7. Wash the column with 5 ml Wash Buffer. Allow the column to empty by gravity flow. Discard the flowthrough.
8. Repeat step 7 Discard flowthrough and collection tube.
9. Re-insert the column into a new collection tube Elute the plasmid with 5 ml prewarmed Elution Buffer (50°C) Allow the column to empty by gravity flow The collection flowthrough contains the plasmid Elute the plasmid again with the flowthrough Allow the column to empty by gravity flow Plasmid concentration is higher, than after only one elution.
10. Precipitate the eluted plasmid DNA with 3,6 ml isopropanol Total volume ~6,8 ml Divide the eluted plasmid into five 1,5 ml eppendorf tubes and one tube which contains 73 μl. Centrifuge immediately 30 min at 15000×g (rcf), +4°C. Carefully discard the supernatant.
11. Divide the 6 tubes into two groups. Wash the plasmid DNA with 1,5 ml chilled (+4°C) 70% ethanol in the first tube and afterwards wash the other two with the same ethanol. Wash the other group (3 tubes) the same way.Centrifuge 10 min at 15000×g (rcf), +4°C. Carefully remove ethanol from the tube with pipet tip.Air-dry the plasmid DNA pellet for 10 min.
12. Carefully redissolve the plasmid DNA pellet in 100-200 μl TE buffer.
13. Measure the concentration of DNA with NanoDrop.
Restriction digestion
- Scientific Background
BioBrick standard biological parts are flanked by well characterized upstream and downstream sequences which are technically not considered part of the BioBrick part (aka prefix and suffix). These up/down stream segments contain restriction sites for specific restriction enzymes, which allows for the simple creation of larger BioBrick parts by chaining together smaller ones in any desired order.
In the process of chaining biobrick parts together, the restriction sites between the two parts are removed, allowing the use of those restriction enzymes without breaking the new, larger BioBrick apart. To facilitate this assembly process, the BioBrick part itself should not contain any of these restriction sites.
One such type of assemblies is the “three antibiotic” assembly standard. This assembly begins with a restriction step.
- Overview
The following protocol contains detailed instructions on the restriction digestion step of the “three antibiotic” standard assembly. It starts with a medium amount of the two parts to be assembled and a medium quantity of the backbone that the parts will be assembled into. The result is a small amount of the insert part ready to be ligated into a linearized backbone.
Note: This protocol uses NEB High Fidelity restriction enzymes and buffers.
Note: One unit of restriction enzyme cuts 1ug of DNA in 1 hour between optimal conditions.
- Materials
DNA sample
appropriate restriction enzyme(s)/NEB
Nuclease Free Water
Buffers/NEB
occasionally BSA
PCR tubes
crushed ice
waterbath or thermocycler
- Procedure
1. Prepare the following mixes in 3 different PCR strips, work on ice.
Note: the restriction of Part A, Part B and the backbone at the same reaction is not mandatory and may be split over several reactions
basic components:
DNA 3-5 ug
Restriction enzyme(s): 0,5 - (0,5)ul
NEBuffer (1-4): 1 ul
(BSA: 0,1 ul, if one of the enzymes recquires it)
Nuclease free water to 10 or 20 ul.
2. Resuspend with a pipette all the components to assure proper mixing
3. Incubate at 37°C for 30 minutes
4. Run products on gel electrophoresis. (Nice bands on gel are made by 100-300ng of DNA).
5. Use PCR Purification Kit to get rid of salts and unnecessary reagents (this step is needed before every ligation)
- Notes
The volume of restriction enzymes used must not exceed 10% of the reaction volume (enzymes are dissolved in glycerol which damages the reaction).
The optimal total volume is 10 ul.
Before starting the reaction calculation it is vital to examine the enzymes information page in order to obtain:
Optimal working buffer -May cause a change in the procedure stated above
Optimal working temprature -May cause a change in the procedure stated above
- References
Sean C. Sleight, Bryan A. Bartley, Jane A. Lieviant, and Herbert M. Sauro "In-Fusion BioBrick assembly and re-engineering" Nucleic Acids Res. 2010 May; 38(8): 2624–2636.
DNA Ligation
- Scientific Background
The term recombinant DNA encapsulates the concept of recombining fragments of DNA from different sources into a new, and hopefully useful DNA molecule. Joining linear DNA fragments together with covalent bonds is called ligation. More specifically, DNA ligation involves creating a phosphodiester bond between the 3' hydroxyl of one nucleotide and the 5' phosphate of another.
- Owerview
In a ligation process we put the DNA fragments to be ligated together, add T4 ligase enzyme with other supplements, and then incubate at 22,5°C.
-Materials
PCR tubesized
linearized DNA samples with appropriate ends (sticky or blunt but able to join)
ligase buffer (with ATP)
Nuclease Free Water
waterbath
crushed ice
-Procedure
1. Calculate the amuount of vector and insert by this equation:
vector mass is constant: 10 ng
insert mass(ng): 6x vector mass(ng)x insert bp/vector bp.
2. Put the appropriate amount of DNA to a tubeon ice, 1 ul Ligase Buffer (supplemented with ATP), 0,5 ul T4 ligase and fill the reaction volume to 10 ul.
3. Vortex, centrifuge shortly and incubate the mixure for 30 mins at 22,5°C (room temp is also efficient).
4. Transform the mixture into competent cells to get the right clone with your DNA of interest.
NOTE: Heat inactivation will cause decrease in the number of colonies after transformation, because Ligase Buffer contains PEG which is damaged by heat and somehow don't let the DNA to get in the cell.
Gel electrophoresis
- Scientific Background
Gel electrophoresis is a technique used for the separation of deoxyribonucleic acid (DNA), ribonucleic acid (RNA), or protein molecules using an electric field applied to a gel matrix. DNA Gel electrophoresis is usually performed for analytical purposes, often after amplification of DNA via PCR The results can be analyzed quantitatively by visualizing the gel with UV light and a gel imaging device. The image is recorded with a computer operated camera, and the intensity of the band or spot of interest is measured and compared against standard or markers loaded on the same gel. The measurement and analysis are mostly done with specialized software.
- Overview
After the preparation of the gel, we put DNA samples into the preformed wells with Loading Dye, than run by electric field.
-Materials
agarose powder
sterilized bottle
1x TAE solution
microwave oven
GelRed nucleic acid stain (non-teratogen, non-carcinogen, non-mutagen)
laboratory scale
6y loading Dye (dextran/glycerol and bromothymol blue stain)
1 kb DNA ladder
Gel electrophoresis apparatus
- Procedure
1. measure 1 g agarose (for 1% gel agarose).
2. put it into a sterilized bottle.
3. masure 100 ml 1x TAE.
4. put the bottle into the microwave (must not close the bottle totally), heat it still it will be fully clear.
5. Take the bottle out from the microwave with the plastic gripping.
6. Cold down the bottle in water bath until you can touch it.
7. Add 100 µl GelRed into the bottle (because 1000x must attenuate).
8. Shake gently the bottle still the red color disappears.
9. Put the liquid from the bottle into the gel tray (if there are any bubbles in it you can punch them with a tip)
10. After it became solid turn the gel rack to 90°.
11. Put 1x TAE into the gel tray still the level of line on the wall
12. Take out the comb
13. Put a ladder (3-6 µl) into the first hole
14. Into a new tube mix the loading dye(1 µl) and your sample(5 µl)
- References
Theriogenology. 2010 Sep 10 ,Huang HW, Su YF, Yao CT, Hung YC, Chen CC, Cheng CC, Li SS, Chang HW. "High-throughput gender identification of three Columbidae species using melting curve analysis"
PCR cleanup of the restricted DNA
- Scientific background
Some kits are available commercially to purify DNA or RNA from reactions like PCR, restriction digestion, dye labelling etc. Some are based on gel purification, to clean up DNA cut from agarose gel or on the well establisher spin column technique for liquid samples. Our team used this kit for the purification of mixes after restriction digestion to get rid of salts and enzymes of the solution (the rest went on a gel).
- Overview
The method used by out team were based on the well established spin column approach including binding, washing and elution steps for purification.
- Materials
kit contents
restriction mix
table-top centrifuge
- Procedure
For detailed steps see: [ [3] ]
Protocols from tissue culture room
Preparation of DMEM medium and PEI solution
- Scientific Background
I. The Medium
Eukaryotic cells, derived from multicellular animal eukaryotes, can be maintained in culturing media. Aside from temperature and gas mixture, the most commonly varied factor in eucaryotic culture systems is the growth medium. Recipes for growth media can vary in pH, glucose concentration, growth factors, and the presence of other nutrients. The type of the medium used depends on the type of the cell line.
II. PEI solution
PEI (10 mM) is a cationic polymer used for PEI-mediated transfection.
- Overview
I. The Medium
For culturing cells and for ligand treatment we use DMEM (Dulbecco's Modified Eagle's Medium) completed with 10% FBS, 1% Penicillin-streptomycin and 2% L-Glutamine.
For transfections we use DMEM medium without FBS, but other constitutives are included.
The preparation procedure takes 40-50 minutes including the FBS melting time (stored in 50 ml aliquots at -20°C).
II. Preparing PEI solution
The reconstitution consists of 3 steps including dissolving PEI, adjusting pH and sterilization by filter sterilizer.
- Materials
I. For the Medium:
500 ml Basic DMEM ( Dulbecco’s modified Eagle Medium ) ordered from Sigma (D5671)
50 ml FBS (Foetal Bovine Serum)
10 ml 200 mM L-Glutamine ordered from Sigma(G7513)
5 ml 100x Penicillin-Streptomycin ordered from Sigma(P4333)
Serological pipettes, Pipettor
Sterile laminar flow box, kimwipes, 70% ethanol sprinkle bottle
II. Preparing PEI solution:
Pipette, pipet tip, glass, analitical scale
MQ water (double distilled, sterilized)
MW=25.000 Pei (ALDRICH 408727)
15, 50 ml tubes
pH measuring electrodes
HCl-solution
Sterile filter – with 0,2 um diameter pores
- Procedure
I. Preparing Medium:
1. Prepare the laminar flow box: turn on the ventillation, wait for 15 minutes, clean the bottom and the glass of the hood with 70% alcohol, and wipe it with kimwipes.
2. Put the Basic DMEM solution, the FBS, Penicillin Streptomycin and the L-Glutamine solutions into the 37°C waterbath and wait for 30 minutes (The FBS is melting slowly)
3. Take out the melted solutions from the waterbath, wipe them, squeeze them down with 70% ethanol and load them into the hood as well as the serological pipettes and the Pipettor (squeezed down)
4. Put these amounts into the basic DMEM by using a pipettor and serological pipettes:
A. For 10% FBS medium:
Basic DMEM: 500 mL
FBS: 50 mL
L-Glutamine: 10 mL
Penicillin-Streptomycin: 5 mL
B. For Serum free medium:
Basic DMEM: 500 mL
FBS: -
L-Glutamine: 10 mL
Penicillin-Streptomycin: 5 mL
5. Invert the bottle, mark it with your name, actual date and with the constituents.
6. Put the bottle to 4°C, clean up after yourself.
II. Preparing PEI solution:
1. Dissolve 4,5 mg pure PEI in 8 ml MQ water, mix well (maybe it takes one day for proper dissolvation)
2. Neutralize the solution with HCl. The final pH should be between pH 6,5-7,5
3. Adjust the volume to 10 ml
4. Filter sterilize through 0,2 um pores
5. Store the solution at 4°C.
- References
Yves Durocher, Sylvie Perret, and Amine Kamen,"High-level and high-throughput recombinant protein production by transient transfection of suspension-growing human 293-EBNA1 cells". Nucleic Acids Res. 2002 January 15; 30(2)
Cell subculturing (also referred as "passaging")
- Scientific Background
Cell passaging or splitting is a technique that enables an individual to keep cells alive and growing under cultured conditions for extended periods of time. Cells should be passed when they are 90%-100% confluent. You have to do the cell passage on every second to fourth day (i.e. on every Monday, Wednesday and Friday).
After reaching the confluency, the cells do not get enough nutrients and do not have enough place where they can extend. The colour of the medium switches from reddish-pink to orange or yellow which shows acidic metabolic products.
- Notes
While working in the Cell culture lab, always follow the rules of the laboratory. You should wear a lab coat, use gloves and keep the sterile box as clean as possible.
It is advisable not to take your mobile phone into the cell culture lab.
Never bring bacterial samples into the cell culture lab!
Carefully separate dangerous waste from communal waste.
- Materials
For COS1-cells, in 10cm Petri dishes:
I. Cell passaging
Ethanol squirt bottle
paper towels
5 mL and 10 mL sterile pipets
confluent cells in Petri dishes
Medium (DMEM) with 10% (50ml) serum and antibiotics
Trypsin-EDTA
1% PBS (Phosphate Buffered Saline)
Pasteur Pipettes+ vacuum for aspirating the used medium
15 mL centrifuge tube
Petri dishes
Automatic pipettors
gloves
37°C water-bath
37° thermostat
laminar box
tube holders
- Procedure
I. Cell Passaging for adherent culture (also called Splitting, subculturing):
1. Warm media, trypsin-EDTA and PBS in 37°C waterbath
2. Check cells in 10 cm Petri dish under Phase-contrast microscope to confirm that the cells are 90%-100% confluent
3. Clean hood with 70% alcohol
4. Sterilize all materials, bottles, etc. which are loaded into the hood. Spray hands with ethanol. Sterile pipets may be placed in the hood directly
5. Spray hands with ethanol. Remove Petri dishes from the incubator and quickly place them in the hood. (Do not spray flasks with ethanol)
6. Attach a Pasteur pipette to vacuum. Turn on vacuum system by opening vacuum valve in hood
7. Using the empty liquid media covering cells. Be careful to not touch the pipet to anything outside of the Petri dis
8. Add 2-3 mL of PBS to Petri dish. Lightly swirl PBS on base of Petri dish. Aspirate PBS from dishes
9. Add 2 mL trypsin-EDTA to Petri dish. Lightly swish trypsin
10. Place flask in incubator until detached (2-3 mins, epends on the cell-type)
11. Remove cells from incubator. Tap side of Petri dish on hard surface or your hand. Repeat several times. Visually check to ensure lumps of cells are dispersed
12. Check cells under microscope to confirm that cells are detached from the surface
13. Add 5 mL of media to dilute trypsin. Media contains antitrypsin. (Note: The liquid suspension now contains the cells.) Carefully re-suspend cells by using pipettor and pipettes
14. Aliquot appropriate volume of cell suspension into freshly prepared Petri dishes with media (The total volume in a Petri dish is 10 mL; 2mL Trypsin-EDTA, 5 ml DMEM for trypsin dilution, 3 further mL of DMEM)
15. Replace media and cells to mix. Place Petri dishes in incubator
16. Turn off aspiration
17. Dispose of liquid and solid biohazards wastes properly
18.Clean hood with ethanol. Spray ethanol liberally over surfaces and wipe clean with kimwipe
Transfection with FuGENE6
This protocol is written by our team members for nuclear receptor transfections, but can be used for any kind of transfections as well.
- Scientific Background
FuGENE 6 transfection reagent is a multi-component lipid reagent that forms a complex with the DNA, then transports it into animal cells. FuGENE 6 transfection is used as standard method in many different laboratories due to its simple methodology, low cytotoxicity, and ability to provide high transfection efficiency even in the presence of serum. Overview
The transfection is divided into two main steps: 1. plating COS1 cells into glass cell culture vessel 2. the transfection itself
The aim is to get cells of 50-80% confluency in the chosen vessel for the day of transfection.
- Required materials
- COS1 cells in a T75 flask (regularly passaged, proliferating well - best in the log-growth phase) - Trypsin/EDTA solution - 1 x PBS solution - 10% FBS containing DMEM medium - LAB-TEK glass chamber slide with 8 chambers - 15 ml centrifuge tube, tube holders - Bürker chamber, pipettor, serological pipettes, pipettes, pipet tips, Pasteur pipettes - sterile laminar air flow box, 37°C incubator, 37°C waterbath - 70% ethanol, kimwipes
- Procedure
I. Plating
1. Prepare the sterile box: Open the sterile laminar box (Hood), turn on the ventillator and wait for 15 minutes to reach the optimal level of sterility (0,45 um filter). PUT ON YOUR GLOVES, wipe the box with 70% alcohol.
2. Prewarm the DMEM medium, Trypsin/EDTA and the 1x PBS in the 37°C waterbath (10-15 minutes)
3. Take the DMEM, Trypsin-EDTA and PBS and squirt the tubes and bottles with alcohol beforeyou put them in the sterile box. Put the pipettor and the tube holder into the box (after you sprayed them down with 70% ethanol), and load the serlogical pipettes directly,without spraying down into the box.
4. Spray hands with ethanol. Remove the flask from the incubator and quickly place in hood. Fire-sterilize the neck of the flask. (Do not spray flasks with ethanol).
5. Attach a Pasteur pipette to vacuum,turn on vacuum system by opening vacuum valve in hood. You should fire-sterilize the end of the pipette,after this step do not touch anything outside the flask. Aspirate the used medium from the cells by touching the bottom-side corner of the flask with the Pasteur-pipette.
6. Washing step:Add 2-3 mL of 1x PBS to flask by using pipettor and a serological pipette (Release the PBS onto the side of the flask, do not push the solution out directly onto the cells because they can come up easily). Lightly swirl PBS on base of the flask. Aspirate PBS from flasks by using a Pasteur pipette and vacuum.
7. Add 2 mL trypsin-EDTA to Flask. You can release the solution directly onto the cells, from now it does not matter if they come up. Lightly swish trypsin.
8. Place flask in 37°C incubator until detached (3-5 minutes for COS1 cells, depending on the temperature of the Trypsin- opt. temp: 37°C)
9. Remove cells from incubator. Tap side of the flask on hard surface of your hand. Repeat several times. Visually check to ensure lumps of cells are dispersed.
10. Check cells under phase-contrast microscope to confirm that cells are detached from the surface.
11. Put the falsk back to the sterile box, add 4 ml of 10% FBS containing DMEM medium to dilute trypsin (you can change the dilution level depending on the cell number, in order to be able to count the cells easier). Medium contains antitrypsin.(Note: The liquid suspension now contains the cells.)
12. Carefully resuspend cells by using pipettor and serological pipettes. You can repeat this step until you get individual floating cells (microscope check needed).Put the cells into a 50 ml tube, for easier handling.
13. Prepare the Bürker chamber and do a cell counting:
- Cell counting in a Bürker chamber:
1) At first, clean the chamber with alcohol and water, put the thin glass slide onto the thicker slide, compress them together by using the metal screw.
2) Resuspend the cell suspension (for even distribution) in which you want to do the cell counting.
3) Take 10 ul from the cell suspension (using sterile tip),
inject into the chamber(marked with arrow) by touching the border of the two glasses with the pipet tip.
4) Place the chamber under the phase-contrast microscope and try to find the lines.
- count inside 3 big squares
- then we take the average of the 3 big squares, and it will give us the number of the cells in 0,1 ul we take it 10 in the factor of 4 times and it will give us the cell number in 1 ml cell suspension.
NOTE: Don't forget that the cell suspenson in the chamber is now not sterile, don't put it back into the cell solution. Clean the chamber with alcohol and water.
1. To reach the appropriate confluency the day after plating, we put 50.000 cells into each well. For an 8 well chamber, if we calculate with 12 wells (because the volume loss), we put into a 15 ml centrifuge tube: - 50.000 x 12 = 600.000 cells [ in milliliter: counted cell number in 1 ml / 600.000 cells ] - we fill the cell suspension up to 8 x 300 ul = 2, 5 ml with 10% FBS DMEM (total volume of the wells are 300 ul)
2. With a 1 ml pipette put 300 ul from this suspension into each well. Sometimes invert the cell suspension containing tube (the cells decent). After you finished the plate, swirl it circularly.
3. Incubate the cells for 1 day at 37°C, 5% CO2.
II. FuGENE 6 transfection:
The goal is to introduce foreign plasmid DNA into the plated COS1 cells 24 hours after the plating.
Materials required: - Plated cells in a LAB-TEK 8 well glass chamber slide ( with 50-80% confluency) - FBS/antibiotics/other additives Free DMEM medium - Sterile FuGENE 6 transfection reagent in a tightly capped glass vial - Plasmids on ice, in known concentrations: Beta-Gal (normalizer plasmid), Luciferase (tracer), Nuclear receptor, VDR- as negative control, the plasmids are solved in sterile TE-buffer - 1,5 ml sterile Eppendorf tube, pipettes and tips - Sterile laminar air flow box, 37°C incubator, 37°C waterbath - 70% ethanol squirt bottle, kimwipes
Steps:
1. Prepare the sterile box: Open the sterile laminar box (Hood), turn on the ventillator and wait for 15 minutes to reach the optimal level of sterility (0,45 um filter). PUT ON YOUR GLOVES, spray down the base of the box with 70% ethanol, and wipe down with kimwipes.
2. Spray down your hands, the DMEM medium, the glass Reagent vial and the pipettes and tips with 70% ethanol. Put these and the plasmids and the eppendorf tube into the sterile box.
3. Transfection mix reconstitution:
3.1 Invert the room temp. FuGEGE 6 Transfection Reagent glass vial 2-3 times to distribute the components. 3.2 Dilute the FuGENE reagent with Serume/antibiotics/other additives free DMEM medium – the order and manner of   addition is critical: Label a 1,5 ml eppendorf tube. Pipet 75 ul FBS/antibiotics/other additives Free DMEM into the eppendorf tube. Pipet 6 ul FuGENE 6 Transfection Reagent directly into the medium, without allowing contact between the plastic    wall and the undiluted reagent. 3.3 Vortex the mix for 1 second. 3.4 Incubate the mix for 5 minutes at room temperature. 3.5 Add 500 ng from each of the three plasmids (receptor, Luciferase, beta – Gal.) into the diluted FuGENE 6   transfection reagent. 3.6 Vortex the Transfection Reagent:Plasmid mixture for 1 second. 3.7 Incubate the mixture for 20 minutes at room temperature.
4. Remove glass chamber slide with plated cells from the incubator, place in the sterile box without spraying down with ethanol.
5. Note: We don’t need to remove the culturing medium (10% FBS containing DMEM) from the cells, it does not have any effect on the transfection efficiency. Add 9 ul Transfection mix in a dropwise manner to each well. Swirl the chamber slide to ensure distribution over the entire surface.
6. Put on the cap of the slide chamber. Return the cells to the 37°C incubator until the assay for gene expression is to be performed. Note: it is not necessary to remove and replace the transfection mixture-containing medium with fresh medium until the assay, only if you used FBS Free medium during the whole experiment (to avoid the cell starvation).
7. Clean up after yourself, place the FuGENE reagent to +2 - +8 °C and be sure if the cap is tightly turned on the Reagent.
NOTES:
- store the reagent at +2 - +8, with the lid very tightly closed, in the original glass vial. - Do not allow the reagent to contact plastic walls (pipet directly into serum free medium) to keep the maximal biological activity. - Do not use siliconized pipet tips and tubes. - To prepare transfection complexes for larger experiments or parallel experiments, proportionally increase the quantity according to the total surface area of the cell culture vessel being used. (ul FuGENE Reagent: ug DNA = 4:1, the used vessel in this case has a 79,21 cm2 surface area)
- References
1. Horbinski C, Stachowiak MK, Higgins D, Finnegan SG. Polyethyleneimine-mediated transfection of cultured postmitotic neurons from rat sympathetic ganglia and adult human retina. .BMC Neurosci. 2001;2:2.
2. Pollard H, Remy JS, Loussouarn G, Demolombe S, Behr JP, Escande D: Polyethylenimine but not cationic lipids promotes transgene delivery to the nucleus in mammalian cells.J Biol Chem 1998, 273:7507-7511
Ligand treatment after transfection
- Scientific Background
Ligand treatment is a procedure when we add the appropriate ligand (specific ligand, oil sample dairy product - in short the ligand of interest) to the nuclear receptor (NR, transcription factor). The NR got into the cells through a previous transfection step. After treatment the ligand-binded NRs will dimerize and bind to the DNA at specific Nuclear receptor Response Elements and this will promote the gene expression of the downstream gene. In further examinations we detect the expression level of the target gene by Luciferase assay.
- Overview
The target cells in our case are COS1 cells. Performing the protocol from the beginning to the end takes no more than 1 hour, and the following steps are included:
1. Preparation of the ligand solutions
2. Removal of the transfection medium
The goal is to get rid of the transfection medium from the COS1 cells 5-7 hours after transfection, because:
- PEI can damage the cells
- FBS Free medium used for transfection can cause starvation
3. Cell refeeding with Ligand-containing medium
Our aim is to refeed the cells with 10% FBS containing DMEM and also give appropriate amounts of Ligand in one step. The 10% FBS in the medium will provide the necessary proteins and other molecules for proliferation and for life functions.  The ligand will activate the nuclear receptor and will promote gene expression.
Preparation of the ligand solutions
- Materials
Nuclear receptor-transfected cells in a 48-well plate
10% FBS containing DMEM medium
Ligand solution in known concentration
50 ml, 15 ml tubes
Pasteur pipettes, pipettor, serological pipettes, pipettes, pipet tips
Sterile laminar air flow box with flame and vacuum system, 37°C waterbath, 37°C incubator, 70% ethanol squirt bottles
- Procedure
- in the case of 1-type receptor transfection (GAL4PPARg):
We use different concentrations of ligand, so as to make a dose-response curve in the subsequent measuring process:
10 μM: 200 * (x/10) μL Ligand up to 200μL with 10% DMEM /per well/
2 μM: 200 * (x/2) μL Ligand up to 200μL with 10%FBS DMEM /per well/
0,4 μM: 200 * (x/0,4) μL Ligand up to 200μL with 10%FBS DMEM /per well/
0,08 μM: 200 * (x/0,08) μL Ligand up to 200μL with 10%FBS DMEM /per well/
0,016 μM: 200 * (x/0,016) μL Ligand up to 200μL with 10%FBS DMEM /per well/
0 μM(negative control): 1 μL DMSO-ethanol(1:1) up to 200μL with 10%FBS DMEM /per well/
if x = original ligand concentartion[mM]
we use 48 well plates
DMSO-ethanol is the dissolvent of the ligand
Prepare the Ligand dilutions:
- We use 6 wells for 1 concentration / plate
- the total concentration of one well is 200 μL
- For this reason, from 1 concentration we need 6x200=1200 μL
- NOTE: these dilutions are examples, and can vary depending on the system itself.
We are doing serial dilutions (5-fold-dilutions) in 15 ml tubes:
1st dilution (10μM): I calculate with 5000 ul, put 5000*(x/10) μL Ligand solution and fill it up to 5000 μL with 10% FBS containing DMEM. Mix it with pipetting.
2nd dilution (2μM): 1 ml from the 1st dilution + 4 ml DMEM 10%
3rd dilution (0,4μM): 1 ml from the 2nd dilution + 4 ml DMEM 10%
4th dilution (0,08μM): 1 ml from the 3rd dilution + 4 ml DMEM 10%
5th dilution (0,016μM): 1 ml from the 4th dilution + 4 ml DMEM 10%
6th solution (0 μM): 6 μL-ethanol 1:1 solution + 4,994 ml DMEM 10%
(you can use serological pipettes and pipettor, or micropipettes with sterile pipette tips.
these solutions will be enough for three 48-well plates, if you have less than 3 plates you can store the tubes at +4°C and use them next time.)
- in the case of a 2-type receptor transfection:
The receptor to be activated is Vitamin D Receptor (VDR) and the Ligand is Vitamin D. We use 10e-8 μM Vitamin D. We treat 24 wells only, the other 24 will be negative controls.
-Prepare the Vitamin D solution in a 15 ml tube:
Fill 2000 / (original cc. of the ligand solution [mM]/10e-8) μL ligand solution up to 2000 μL with 10% FBS containing DMEM. Use micropipettes and sterile pipet tips. Removal of the transfection medium
- Materials
- Nuclear receptor-transfected cells in a 48-well plate
- Pasteur pipettes
- Sterile box with flame and vacuum system Procedure
- Steps:
1. Prepare the hood : put on your gloves, turn on the ventillation (it needs 15 minutes to filter the air in the box), clean the inner space of the hood with 70% ethanol. Spray hands with ethanol.
2. Place the plate into the hood.Turn on the vacuum system, insert the Pasteur pipette into the vacuum tube. Fire-sterilize the Pasteur pipette.
3. Aspire the used medium from all of the wells, pay attention so as not to aspire the cells (touch only the bottom-side of the wells).
4. Put back the cap of the plate, we don’t want the cells to go dry. Cell refeeding with Ligand-containing medium
- Materials
- Ligand solutions prepared in point 1.
- 48 well plate without medium (point 2.)
- 10% FBS containing medium
- Repeating pipet and tips
- 37°C incubator, 37°C waterbath, Sterile laminar air flow box, 70% ethanol squirt bottles Procedure
- Steps:
1. Warm up the Ligand solutions for 1- receptor transfected cells and the Ligand solution + 10% FBS DMEM for 2-receptor transfected cells
2. You use the same hood which you prapared: put everything what is needed into the hood after spraying down with 70% alcohol.
3. Take down the cap of the plate. Vortex the Ligand solutions, after that put 200-200 ul from these solutions into the adequate wells quickly (in other case the cells will go dry). Release the solutions onto the wall of the wells
After 1-receptor transfection - by using a 200 μl or a 1 ml pipet with sterile tips
10 mM: A1-A3 and E1-E3
2 mM: A4-A6 and E4-E6
0,4 mM: B1-B3 and F1-F3
0,08 mM: B4-B6 and F4-F6
0,016 mM: C1-C3 and G1-G3
0 mM: C4-C6 and G4-G6
After 2-receptor transfection – by using a repeating pipet
Ligand treated /well A1-D6/ : 200 ul 10e-8 mM Ligand- DMEM solution
Non-treated /well E1-H6/ : 200 ul "0 μM" solution
4. Put the „treated” COS1 cells into the 37°C incubator for 2 days, after 2 days comes the Luciferase assay to make sure that the transfection worked and to measure the Nuclear Receptor’s response to the ligand.
5. Clean the surface
Measurement of Luciferase reporter
- Scientific Background
Luciferase assay / cotransfection assay is a laboratory technique used by our team to measure the expression of the reporter firefly luciferase enzyme in order to study the effect of nuclear receptors/zinc fingers in different conditions. Luciferase substrate is transformed and emits light that can be measured (directly proportional to the expression of the enzyme and hence to the upregulation by Nuclear receptors/zinc fingers. The luciferase assay system is an extremely sensitive and rapid method. Linear results are seen over at least eight orders of magnitude of enzyme concentration. Generally, 100-fold greater sensitivity can be achieved over the chloramphenicol acetyltransferase (CAT) assay.
- Overview
After transfection with nuclear receptors and/or zinc fingers and ligand treatment we measure the expression of the firefly luciferase reporter (MH-100) which has a response element to a specific receptor upstream the protein coding sequence.
- Materials
Transfected and ligand treated cells is plates 2 days after ligand treatment
Luciferase plates (96 wells, transparent and non transparent plates)
Deep freezer of -70 degrees
luciferase substrate (home made, see below)
multi-channel pipettes
X-gal
Wallace VICTOR plate reader
- Procedure
I. Sample preparation
1. 2 days after ligand treatment the used medium on the cells has to be discarded (Pasteur pipettes and vacuum). 2. Wash the cells with PBS (150 ul/well), discard with pasteur pipette and vacuum. 3. Wash again with 150 ul 1xPBS. 4. Add 200 ul lysis buffer to the washed cells 5. Put the plate on a plate shaker for 2 hours (chemical lysis) 6. Place the plate to -70°C for 30 minutes (physical lysis) 7. Divide the lysed cells into 2 plates, these will be the plates for beta-Gal measurement (for normalisation) and for luciferase activity measurement.
– Measurement of β-GAL:
1. Put 80 ul lysate from each well to the transparent β-GAL measurement plate. 2. Put 100 ul Luciferase substrate to the cells. 3. Wait for 3-5 minutes and the wells will become visibly yellow. 4. Measure the intensity with VICTOR plate reader
– Measurement of Luciferase:
1. Put 20ul lysate from each well into the luciferase measurement plate 2. Put the plate into the VICTOR plate reader 3. Place the luciferase reagent into the VICTOR plate reader (dark vial). The instrument will add the substrate automatically before plate reding.  4. Measure the intensity of light emission.
Analize the data.
Prepare substrate solutions:
• β-GAL substrate:
– 10ml β-GAL buffer – 20mg ONPG (light sensitive) – 35μl mercaptoethanol
NOTE: We always have to use a BLANK plate which contains lysis buffer triplicates.
• Normalisated luciferase assay measurement:
NLa=((Luc act-Luc. BASE)*10)/((β-GAL-β-GAL BASE)/TIME)
NLa= Normalised Luciferase Assay Luc act= luciferase activity Luc BASE= luciferase BASE
II. Measurement vith VICTOR Plate Reader
- Scientific Background
The Wallac 1420 VICTOR2 is a multilabel, multitask plate reader designed to support the future demands of industrial and academic laboratories for multiple assay technologies on a single platform. An extended version of the successful Wallac VICTOR multilabel reader, the VICTOR2 allows immediate access to more than 10 counting modes, covering all of the main nonradioactive counting technologies.
VICTOR2 accepts all types of microtitration plates with between 1 and 864 (1536 fluorescence and time-resolved fluorescence) wells, as well as Petri dishes, slides, filters and Terasaki plates. All models include scanning, shaking and kinetics modules. To support individual applications in cell or molecular biology, binding studies, environmental and food testing, toxicology and drug screening, VICTOR2 can also be supplied with dispenser module, temperature control, bottom reading, high density reading and various other options.
- Procedure
1. Pepare the following approximately half an hour before you start the B-gal and the Luciferase measurements
Switch on the machine VICTOR
Turn on the computer
Log in with your username
Open the program WALLAC 1420 WORKSTATION
2. Open main controll panel
Open the TOOLS tab and choose EXPLORER, after it you choose the BRAZDA* map
Open the corresponding file to measure the B-gal absorbance
choose the number of wells
save settings
3. go back to the main controll panel and choose the file programm within the PROTOCOL tab
4. Put the plate into the machine in right direction
5. Start the measurement
you may check live the measured absorbance values, if you click to the LIVE DISPLAY tab
6. To measure Luciferase activity, follow steps 1-4 with the corresponding file. Before step 4. add the Luciferase substrate
7. Use the icon which is a part of the main control panel for Dispenser maintenance
8. Appoint fill with the volume 500μL
9. Start measurement You may check live the measured Luciferase values, if you click to the LIVE DISPLAY tab
10. After the measurements empty the machine from the substrate
11. To wash the machine with water have to appoint flush (and the volume 500μL after it click ok) and then empty again
11. Turn off the VICTOR and then the computer.
Apoptosis
DAPI staining
- Scientific Background
DAPI or 4',6-diamidino-2-phenylindole is a fluorescent stain that binds strongly to A-T rich regions in DNA. It is used extensively in fluorescence microscopy. DAPI can pass through an intact cell membrane therefore it can be used to stain both live and fixed cells, though it passes through the membrane less efficiently in live cells and therefore the effectiveness of the stain is lower.
When bound to double-stranded DNA DAPI has an absorption maximum at a wavelength of 358 nm (ultraviolet) and its emission maximum is at 461 nm (blue). Therefore for fluorescence microscopy DAPI is excited with ultraviolet light and is detected through a blue/cyan filter. The emission peak is fairly broad DAPI will also bind to RNA, though it is not as strongly fluorescent. Its emission shifts to around 500 nm when bound to RNA.
- Overwiew
In order to vizualize the nuclear morphology of living and dying cells the nucleus of the cells were stained with DAPI which intercalates to the double stranded DNA. Nuclear fragmentation can be seen in the apoptotic cells.
- Procedure
1. Washing cells which are attached to the surface of the plate with PBS for 5 minutes at room temperature.
2. Fixating cells with 4% paraformaldehyde for 10 minutes on ice.
3. Washing cells which are attached to the surface of the plate with PBS for 5 minutes at room temperature. Fixed cells can be stored at 4 C.
4. Stain cells with DAPI in a concentration of 1 ng/ml for 10 minutes at room temperature in the dark.
5. Washing cells which are attached to the surface of the plate with PBS for 5 minutes at room temperature.
6. Vizualizing the cells by fluorescent microscopy.
Retrieved from " http://2011.igem.org/Team:Debrecen_Hungary/Protocols "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
