<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Slovenia_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Slovenia/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Slovenia/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Slovenia/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Slovenia/Notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Slovenia/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Slovenia/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="mainmenu"><DIV id="cssmenu"><UL><LI><A href="https://2012.igem.org/Team:Slovenia"><SPAN>Home</SPAN></A></LI><LI class="has-sub "><A href="https://2012.igem.org/Team:Slovenia/Idea"><SPAN>Idea</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Slovenia/IdeaBioTherapy"><SPAN>Biological therapy</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Slovenia/IdeaChallenge"><SPAN>Challenge</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Slovenia/Idea"><SPAN>Solution</SPAN></A></LI></UL><LI class="has-sub "><A href="https://2012.igem.org/Team:Slovenia/TheSwitch"><SPAN>The switch</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Slovenia/TheSwitch"><SPAN>Overview</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Slovenia/TheSwitchDesignedTALregulators"><SPAN>Designed TAL regulators</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Slovenia/TheSwitchMutualRepressorSwitch"><SPAN>Mutual repressor switch</SPAN></A></LI><LI><TABLE onclick="window.location = 'https://2012.igem.org/Team:Slovenia/TheSwitchPositiveFeedbackLoopSwitch';" class="newtable"><TBODY><TR class="newtable"><TD class="newtable"><SPAN>Positive feedback loop switch</SPAN></TD></TR></TBODY></TABLE></LI><LI><TABLE onclick="window.location = 'https://2012.igem.org/Team:Slovenia/TheSwitchControls';" class="newtable"><TBODY><TR class="newtable"><TD class="newtable"><SPAN>Controls</SPAN></TD></TR></TBODY></TABLE></LI><LI class="has-sub "><A href="https://2012.igem.org/Team:Slovenia/SafetyMechanisms"><SPAN>Safety mechanisms</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Slovenia/SafetyMechanisms"><SPAN>Overview</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Slovenia/SafetyMechanismsEscapeTag"><SPAN>Escape tag</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Slovenia/SafetyMechanismsTermination"><SPAN>Termination</SPAN></A></LI><LI><TABLE onclick="window.location = 'https://2012.igem.org/Team:Slovenia/SafetyMechanismsMicrocapsuleDegradation';" class="newtable"><TBODY><TR class="newtable"><TD class="newtable"><SPAN>Microcapsule degradation</SPAN></TD></TR></TBODY></TABLE></LI><LI class="has-sub "><A href="https://2012.igem.org/Team:Slovenia/Implementation"><SPAN>Implementation</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Slovenia/Implementation"><SPAN>Overview</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Slovenia/ImplementationHepatitisC"><SPAN>Hepatitis C</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Slovenia/ImplementationIschaemicHeartDisease"><SPAN>Ischaemic heart disease</SPAN></A></LI><LI><TABLE onclick="window.location = 'https://2012.igem.org/Team:Slovenia/ImplementationImpact';" class="newtable"><TBODY><TR class="newtable"><TD class="newtable"><SPAN>Impact</SPAN></TD></TR></TBODY></TABLE></LI><LI class="has-sub "><A href="https://2012.igem.org/Team:Slovenia/Modeling"><SPAN>Modeling</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Slovenia/Modeling"><SPAN>Overview</SPAN></A></LI><LI><TABLE onclick="window.location = 'https://2012.igem.org/Team:Slovenia/ModelingPK';" class="newtable"><TBODY><TR class="newtable"><TD class="newtable"><SPAN>Pharmacokinetics</SPAN></TD></TR></TBODY></TABLE></LI><LI><A href="https://2012.igem.org/Team:Slovenia/ModelingMethods"><SPAN>Modeling methods</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Slovenia/ModelingMutualRepressorSwitch"><SPAN>Mutual repressor switch</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Slovenia/ModelingPositiveFeedbackLoopSwitch"><SPAN>Positive feedback loop switch</SPAN></A></LI><LI><TABLE onclick="window.location = 'https://2012.igem.org/Team:Slovenia/ModelingQuantitativeModel';" class="newtable"><TBODY><TR class="newtable"><TD class="newtable"><SPAN>Experimental model</SPAN></TD></TR></TBODY></TABLE></LI><LI><TABLE onclick="window.location = 'https://2012.igem.org/Team:Slovenia/ModelingInteractiveSimulations';" class="newtable"><TBODY><TR class="newtable"><TD class="newtable"><SPAN>Interactive simulations</SPAN></TD></TR></TBODY></TABLE></LI><LI><A href="https://2012.igem.org/Team:Slovenia/Parts"><SPAN>Parts</SPAN></A></LI><LI class="has-sub "><A href="https://2012.igem.org/Team:Slovenia/Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Slovenia/Notebook"><SPAN>Experimental methods</SPAN></A></LI><LI><TABLE onclick="window.location = 'https://2012.igem.org/Team:Slovenia/NotebookLablog';" class="newtable"><TBODY><TR class="newtable"><TD class="newtable"><SPAN>Lablog</SPAN></TD></TR></TBODY></TABLE></LI><LI><A href="https://2012.igem.org/Team:Slovenia/NotebookLabSafety"><SPAN>Lab safety</SPAN></A></LI><LI class="has-sub "><A href="https://2012.igem.org/Team:Slovenia/Society"><SPAN>Society</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Slovenia/Society"><SPAN>Human practice</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Slovenia/SocietyScientists"><SPAN>Scientists</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Slovenia/SocietyMedicalDoctors"><SPAN>Physicians</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Slovenia/SocietyEthics"><SPAN>Ethics, safety and regulations</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Slovenia/SocietyPatients"><SPAN>Patients</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Slovenia/SocietyMedia"><SPAN>Journalists and general public</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Slovenia/SocietyOutreach"><SPAN>Outreach</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Slovenia/SocietyQuestionnaire"><SPAN>Questionnaire</SPAN></A></LI><LI class="has-sub "><A href="https://2012.igem.org/Team:Slovenia/Team"><SPAN>Team</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Slovenia/Team"><SPAN>Team members</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Slovenia/TeamAttributions"><SPAN>Attributions</SPAN></A></LI><LI><TABLE onclick="window.location = 'https://2012.igem.org/Team:Slovenia/TeamCollaborations';" class="newtable"><TBODY><TR class="newtable"><TD class="newtable"><SPAN>Collaborations</SPAN></TD></TR></TBODY></TABLE></LI><LI><A href="https://2012.igem.org/Team:Slovenia/TeamGallery"><SPAN>Gallery</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Slovenia/TeamSponsors"><SPAN>Sponsors</SPAN></A></LI></DIV></DIV><DIV id="main"><H1>Experimental methods</H1><P><UL style="margin-left:15px;"><LI><A href="#cloning">Cloning</A></LI><LI><A href="#cellcultures">Cell cultures</A></LI><LI><A href="#inductionsystems">Induction systems</A></LI><LI><A href="#effectors">Effectors</A></LI><LI><A href="#microscopy">Microscopy</A></LI><LI><A href="#flowcytometry">Flow cytometry</A></LI><LI><A href="#microencapsulation">Microencapsulation</A></LI><LI><A href="#proteindetection">Protein detection</A></LI></UL></P><P><TABLE class="invisible" style="width:80%;"><TBODY class="invisible"><TR class="invisible"><TD class="invisible"><B>Figure 1.</B> Schematic presentation of methods used for cloning and culturing eukaryotic cells.
</TD></TR></TBODY></TABLE></P><TABLE class="invisible" style="width:80%;"><TBODY><TR class="invisible"><TD class="invisible"><B>Figure 2.</B> Schematic presentation of methods used for characterizing the switch, safety mechanisms, microencapsulation and effectors.</TD></TR></TBODY></TABLE><H2><A name="cloning"> Cloning </A></H2><H3>Plasmid DNA isolation</H3><P><B>MINI PREPs for analysis and sequencing</B></P><OL><LI>A single colony was picked from a LB-agar plate or glycerol stock and inoculated in 10 mL of LB-medium with the appropriate antibiotic for selection (100 mg/L ampicillin, 50 mg/L kanamycin, 35 mg/L chloramphenicol). </LI><LI>Bacteria were grown over night at 37 °C with agitation. </LI><LI>Plasmid DNA was isolated from 6-10 mL of over-night culture with GeneJET plasmid miniprep kit according to the manufacturer's protocol. </LI><LI>Amounts ranging from 6-10 µg of plasmid DNA were obtained. </LI><LI>The purity and concentration of the isolated DNA was analysed using NanoDrop. </LI></OL><H3>Fragment DNA isolation from agarose gel</H3><P><B>AGAROSE ELECTROPHORESIS</B></P><OL><LI>A mixture of different sized DNA fragments was  separated on an agarose gel (from 0.7 to 2% agarose in 1x TAE buffer and 0.1 µg/ml ethidium bromide) at a constant voltage of 100 V. </LI><LI>UV light (λ = 254 nm) was used to visualize DNA with intercalated ethidium bromide </LI></OL><P><B>FRAGMENT ISOLATION from agarose gel </B></P><OL><LI>The band with the desired DNA fragment was excised from the gel, using a clean scalpel. </LI><LI>DNA was isolated from the gel slice with GeneJet Gel Extraction Kit according to the manufacturer’s protocol. </LI><LI>Purity and amount of DNA was determined using NanoDrop. </LI></OL><H3>Restriction digest</H3><OL><LI>To digest the desired DNA restriction reactions were prepared as follows: </LI><UL style="margin-left:15px;"><B>for analysis of cloned DNA</B><LI>2µl of the appropriate restriction buffer (10X)</LI><LI>0.5 µL restriction enzyme</LI><LI>Bring volume to 20 µL with nuclease-free water.</LI></UL><I>or</I><UL style="margin-left:15px;"><B>for isolation of specific DNA</B><LI>2µl of the appropriate restriction buffer (10X)</LI><LI>up to 2 µL restriction enzyme </LI><LI>Bring volume to 50 µL with nuclease-free water.</LI></UL><LI>The sample was incubated at optimal temperature for the restriction enzymes.</LI><LI>Analysis of fragmented DNA was done by gel electrophoreses. </LI><LI>Desired DNA fragment was excised and purified using suitable DNA purification kit. </LI></OL><H3>PCR reaction</H3><P>AccuPrime and Phusion DNA polymerase were used for DNA amplification. Colony PCR was performed with Taq DNA polymerase. </P><OL><LI>The master mix for reactions with Phusion  DNA polymerase contained:
</LI><LI>DNA (1-10 ng)</LI><LI>both primers (0,4 pmol/µl )</LI><LI>1x Phusion HF buffer</LI><LI>0,2 µM dNTPs</LI><LI>Phusion polymerase (0,02 U/ µl) and </LI><LI>MQ up to final volume of 25 µl</LI><LI>
The master mix for reactions with AccuPrime DNA polymerase contained:
</LI><LI>DNA (10 ng),</LI><LI>both primers (0,4 pmol/µl ),</LI><LI>1xRnx mix,</LI><LI>enzyme (0,05 U/ µl) and</LI><LI>MQ up to final volume of 50 µl.</LI><LI>
The master mix for reactions with Taq DNA polymerase contained:
</LI><LI>both primers (0,4 pmol/µl),</LI><LI>1x Taq PCR buffer II,</LI><LI>0,2 µM dNTPs,</LI><LI>5mM MgSO4,</LI><LI>enzyme (0,125 U/ µl) and</LI><LI>MQ up to total volume of 20 µl.</LI><LI>Then the bacterial colony was added to the reaction mix.</LI><LI>All temperature profiles were optimized according to manufacturer’s protocol,the melting temperature of primers, and the length of the desired PCR products. Reactions were performed in the Applied Biosystems Veriti 96 well thermal cycler. </LI></OL><P><B>PCR product purification</B>. Desired PCR products were purified by GeneJet Gel Extraction Kit according to the manufacturer's protocol. </P><P><B>DNA concentration. </B> An aliquot of the isolated DNA was analyzed using NanoDrop. </P><H3>Gibson assembly </H3><P>Gibson assembly master mix was prepared according to the protocol published in Gibson et al., 2009. </P><OL><LI>50 ng of each PCR product was added to the Gibson assembly master mix and incubated at 50 °C 1h. </LI><LI>After incubation, the entire master mix volume was transformed into competent bacterial cells. </LI></OL><H3>Ligation</H3><P>T4 ligase ligates the 5' phosphate and the 3'-hydroxyl groups of DNA. </P><OL><LI>Vector and insert concentrations were estimated and insert and vector fragments joined in a molar ratio of 3:1 (100-150ng Vector DNA). </LI><LI>A ligation mixture was prepared: </LI>
          1X ligase buffer (10X)

          1 µL T4 ligase (3 U/µL)

          Bring volume to 10 or 20 µL with nuclease-free water.
<I>or</I><LI>Blunt-end ligation reactions were incubated at 17 °C for 4 to 18 hours. </LI><LI>After incubation part of the ligation mixture was used for the transformation of bacterial cells (see: transformation of bacteria). </LI></OL><H3> Culturing bacteria</H3><P>For plasmid DNA propagation two bacterial strains were used: <B>DH5alpha</B> [<I>fhuA2Δ(argF-lacZ)U169 phoA glnV44 Φ80 Δ(lacZ)M15 gyrA96 recA1 relA1 endA1 thi-1 hsdR17</I>] and <B>TOP10</B> [<I>mcrA, Δ(mrr-hsdRMS-mcrBC), Phi80lacZ(del)M15, ΔlacX74, deoR, recA1, araD139, Δ(ara-leu)7697, galU, galK, rpsL(SmR), endA1,nupG</I>]. </P><P><B> Growth media for bacteria</B></P><P><B> Luria Broth (LB) </B>: 10 g/L tryptone,    5 g/L yeast extract,    10 g/L NaCl,    media is supplemented with suitable antibiotics depending on the selection marker on the transfected plasmid: ampicilin 100 mg/L or kanamycin 50 mg/L.</P><P><B> LB agar plates</B>: LB with 1.5% agar, media is supplemented with suitable antibiotics depending on the selection marker on the transfected plasmid.</P><H3>Transformation of bacteria</H3><P>  E. coli DH5alpha and TOP10 competent cells were used for the propagation of plasmid DNA. </P><OL><LI>100 µL of competent cells were thawed on ice. </LI><LI>50 – 400 ng DNA solution was added to competent bacterial cells (depending on the concentration of the DNA solution). </LI><LI>A mixture of cells and DNA solution was incubated on ice for 30-60 minutes. </LI><LI>The mixture was heat-shocked for 3 minutes at 42 °C. </LI><LI>Cooled for 3 minutes on ice. </LI><LI> 500 µL  of preheated antibiotic free LB-medium was added and incubated for one hour at 37 °C with agitation for the purpose of inducing antibiotic resistance. </LI><LI>The selection for plasmid containing and therefore antibiotic resistant bacteria was conducted by plating them on antibiotic containing LB-agar plates. </LI></OL><H3>Glycerol stock for long term storage of bacteria</H3><OL><LI>1 mL of an overnight culture was added to 150 µL of 80% glycerol into a cryo-tube. </LI><LI>Mixed and incubated at room temperature for 30 minutes. </LI><LI>Afterwards the glycerol stock was stored at -80 °C. </LI></OL><H2><A name="cellcultures"> Cell cultures </A></H2><H3>Eucaryotic cell lines and cultivation</H3><P><B>HEK293</B> is a human cell line derived from kidney cells and grows in a monolayer culture. Cells were grown in DMEM medium supplemented with 10% FBS. </P><P><B>HEK293T</B> cell line is derived from HEK293 cells. HEK293T cells express the SV40 large T-antigen that enables episomal replication of plasmids containing the SV40 origin of replication in transfected cells. Cells were grown in DMEM medium supplemented with 10% FBS. </P><P><B>NK-92</B> is an interleukin-2 (IL-2) dependent natural killer cell line derived from peripheral blood mononuclear cells from patient with non-Hodgkin's lymphoma. The cell line is cytotoxic to a wide range of malignant cells. Cells were grown in RPMI medium supplemented with 20% FBS and 100 U/ml IL-2.</P><H3>Subculturing monolayer cell cultures</H3><OL><LI>Remove and discard culture medium from a T-75 flask containing a monolayer of HEK293 or HEK293T cells. </LI><LI>Rinse the T-75 flask with 10 ml of PBS buffer to remove all traces of growth medium (DMEM + 10% FBS) which otherwise inhibits trypsin function. Remove and discard the PBS buffer. </LI><LI>Add 2-3 ml of trypsine solution and gently tilt the flask to ensure the trypsine solution covers all the cells. Incubate the cells in trypsin for 0,5 - 3 minutes. </LI><LI>When the cells start to detach from the surface, add 7 ml of growth medium to the trypsin solution. Resuspend all remaining cells from the bottom of the T-75 flask by pipetting. </LI><LI>Transfer the cell suspension to a 15 ml centrifuge tube. </LI><LI>Centrifuge the cell suspension for 5 minutes at 1200 rpm. </LI><LI>Remove the trypsin-containing medium from the centrifuge tube. </LI><LI>Resuspend the cell pellet in fresh medium. </LI><LI>Take as much cells as you need and add fresh medium to a total volume of 10 ml. </LI><LI>Return the cells in a T-75 flask to the incubator (37 °C, 5 % CO2). </LI></OL><H3>Cell plating</H3><OL><LI>Count cells. </LI><LI>Calculate the desired number of cells per well. Dilute cells in DMEM with 10% FBS. </LI><LI>Transfer the cells into an appropriate plate and place in a cell culture incubator. </LI></OL><P><B>Media and buffers</B><B>DMEM</B> supplemented with:    1 % L-Glutamine (GlutaMax),    10 % FBS,    Optionally: 1% Pen/Strep.
<B>RPMI</B> supplemented with:    1 % L-Glutamine (GlutaMax),    20 % FBS.</P><H3>Transfection</H3>
TABLE: Transfection mixtures for different culture format

<TABLE class="normal" style="font-size:90%; width:90%; text-align:center;"><THEAD class="normal"><TR class="normal"><TH>Culture format</TH><TH>jetPEI reagent per µg of DNA (µL)</TH><TH>Typical amount of DNA (ng)</TH><TH>Volume of 150 mM NaCl solution for DNA and jetPEI (µL)</TH><TH>Total transfection mixture volume (µL)</TH></TR></THEAD><TBODY class="normal"><TR class="normal"><TD class="normal">96-well</TD><TD class="normal">2</TD><TD class="normal">200</TD><TD class="normal">10</TD><TD class="normal">20</TD></TR><TR class="normal"><TD class="normal">24-well and 8-well microscope chamber</TD><TD class="normal">2</TD><TD class="normal">500</TD><TD class="normal">50</TD><TD class="normal">100</TD></TR><TR class="normal"><TD class="normal">12-well</TD><TD class="normal">2</TD><TD class="normal">1000</TD><TD class="normal">50</TD><TD class="normal">100</TD></TR><TR class="normal"><TD class="normal">6-well</TD><TD class="normal">2</TD><TD class="normal">2000</TD><TD class="normal">100</TD><TD class="normal">200</TD></TR><TR class="normal"><TD class="normal">10 cm</TD><TD class="normal">2</TD><TD class="normal">15000</TD><TD class="normal">250</TD><TD class="normal">500</TD></TR></TBODY></TABLE><OL><LI>Dilute plasmid DNA to desired concentration in 150 mM NaCl, vortex gently and spin down briefly.  </LI><LI>Dilute jetPEI (PolyPlus) in 150mM NaCl, vortex gently and spin down briefly. </LI><LI>Add the jetPEI solution to the DNA solution. </LI><LI>Vortex the solution immediately and spin down briefly. </LI><LI>Incubate for 15 to 30 minutes at room temperature. </LI><LI>Add the jetPEI/DNA mix to the cells in and gently swirl the plate. </LI><LI>Return the plate to a cell culture incubator. </LI></OL><H2><A name="inductionsystems"> Induction systems </A></H2><H3>Induction systems</H3><P>To control our switch we needed a way to affect it from the outside. For this purpose we chose several inducible transcription systems where the controlled gene is expressed when a small molecule inducer (such as tetracycline) is present and is not expressed when the inducer is absent. We chose specific systems which do not cross react and whose inducers are orally bioavailable and safe for human use (Clackson, 2000). We decided for the systems based on tetracycline, pristinamycin, erythromycin and rapamycin analogs, as inducers. We then adapted these systems by cloning TAL regulators under their control to make them compatible with our genetic circuits. </P><H3>Tetracycline, erythromycin and pristinamycin systems</H3><P>The tetracycline, erythromycin and pristinamycin system all function in a similar manner. They are composed of a DNA binding protein (such as TetR) fused to a KRAB domain which reversibly binds a specific DNA sequence (TRE for example) and silences transcription from nearby promoters. The addition of an inducer causes the DNA binding domain to dissociate from the DNA and allows transcription to start. (Deuschle et al., 1995; Kramer et al., 2004) </P><CENTER><P><B> Figure 1: Induced expression of TAL</B>.</P></CENTER><P>TABLE</P><TABLE class="normal" style="font-size:90%; width:90%; text-align:center;"><THEAD class="normal"><TR class="normal"><TH> System </TH><TH> Tetracycline </TH><TH> Erythromycin</TH><TH> Pristinamycin </TH></TR></THEAD><TBODY class="normal"><TR class="normal"><TD class="normal"> Regulating protein </TD><TD class="normal"> TetR:KRAB </TD><TD class="normal"> E:KRAB </TD><TD class="normal"> PIP:KRAB </TD></TR><TR class="normal"><TD class="normal"> DNA sequence </TD><TD class="normal"> TRE </TD><TD class="normal"> ETR </TD><TD class="normal"> PIR </TD></TR><TR class="normal"><TD class="normal"> Inducer </TD><TD class="normal"> Tetracycline or doxycycline </TD><TD class="normal"> Erythromycin </TD><TD class="normal"> Pristinamycin </TD></TR><TR class="normal"><TD class="normal"> Constructs </TD><TD class="normal"> pCMV_TRE_TAL-A:KRAB </TD><TD class="normal"> pCMV_ETR_TAL-A:KRAB </TD><TD class="normal"> pCMV_PIR_TAL-B:KRAB </TD></TR><TR class="normal"><TD class="normal"> pCMV_TRE_TAL-B:VP16</TD><TD class="normal"> pCMV_ETR_TAL-B:VP16</TD><TD class="normal"> pCMV_PIR_TAL-A:VP16</TD></TR><TR class="normal"><TD class="normal"> TRE_CMV_TAL-A:KRAB</TD><TD class="normal"> pCMV_ETR_TAL-B:KRAB</TD><TD class="normal"> pSV40_PIR_TAL-B:KRAB</TD></TR><TR class="normal"><TD class="normal"> TRE_CMV_PEST-CL1:TAL-A:KRAB</TD><TD class="normal"> pCMV_ETR_TAL-A:VP16</TD><TD class="normal"> PIR_CMV_TAL-B:KRAB </TD></TR><TR class="normal"><TD class="normal">TRE_CMV_TAL-B:VP16 </TD><TD class="normal">pSV40_ETR_TAL-B:KRAB  </TD><TD class="normal">PIR_CMV_PEST-CL1:TAL-B:KRAB</TD></TR><TR class="normal"><TD class="normal">TRE_CMV_PEST-CL1:TAL-B:VP16 </TD><TD class="normal">ETR_CMV_TAL-A:KRAB  </TD><TD class="normal"> TRE_CMV_TAL-A:VP16 </TD></TR><TR class="normal"><TD class="normal"> ETR_CMV_PEST-CL1:TAL-A:KRAB </TD><TD class="normal"> TRE_CMV_PEST-CL1:TAL-A:VP16</TD></TR><TR class="normal"><TD class="normal"> ETR_CMV_TAL-B:VP16 </TD></TR><TR class="normal"><TD class="normal">  ETR_CMV_PEST-CL1:TAL-B:VP16</TD></TR></TBODY></TABLE><H3>Rapamycin system</H3><P>In the rapamycin system the gene of interest is under the control of a minimal promoter. The gene's transcription rate is regulated by two proteins that consist of a drug binding domain and either a DNA binding domain or an activation domain. When rapamycin is added both drug binding domains bind to it, consequently joining the activation domain with the DNA binding domain, resulting in a functional transcription factor, which activates the gene of interest. Instead of rapamycin a rapamycin analogue (rapalogue), which is a 1000-fold less imunosupressive than rapamycin, but activates the inducible system like rapamycin, is usually used as the inducer. (Pollock et al., 2002) </P><TABLE class="normal" style="font-size:90%; width:90%; text-align:center;"><THEAD class="normal"><TR class="normal"><TH> Regulating vector </TH><TH> HetAct </TH></TR></THEAD><TBODY class="normal"><TR class="normal"><TD class="normal"> DNA sequence </TD><TD class="normal"> ZFHD </TD></TR><TR class="normal"><TD class="normal"> Constructs </TD><TD class="normal"> ZFHD_pMIN_TAL-A:KRAB </TD></TR><TR class="normal"><TD class="normal"> ZFHD_pMIN_TAL-B:VP16</TD></TR></TBODY></TABLE><H3>Induction of cells</H3><OL><LI>Transfect HEK293 or HEK293T cells with plasmids using JetPei transfection reagent (Polyplus transfection), following the manufacturers protocol (see cell culturing for details). </LI><LI>Two hours post transfection change media and stimulate the cells by adding dilutions of appropriate inductors to the medium in a 1:10 (v:v). </LI></OL><TABLE class="normal" style="font-size:90%; width:90%; text-align:center;"><THEAD class="normal"><TR class="normal"><TH>Inductor</TH><TH>Stock solution (solvent)</TH><TH>Dilution (solvent)</TH><TH>Concentration in cell medium (% solvent)</TH></TR></THEAD><TBODY class="normal"><TR class="normal"><TD class="normal">Rapalogue (AP21967)</TD><TD class="normal">1 mM (100% ethanol)</TD><TD class="normal">10 µM (1% ethanol)</TD><TD class="normal">1 µM (0,1% ethanol)</TD></TR><TR class="normal"><TD class="normal">Doxycyclin</TD><TD class="normal">1 g/L (MQ)</TD><TD class="normal">10 mg/L (MQ)</TD><TD class="normal">1 mg/L (MQ)</TD></TR><TR class="normal"><TD class="normal">Pristinamycin</TD><TD class="normal">50 g/L (100% DMSO)</TD><TD class="normal">20 mg/L (1% DMSO)</TD><TD class="normal">2 mg/L (0,1% DMSO)</TD></TR><TR class="normal"><TD class="normal">Erythromycin</TD><TD class="normal">50 g/L (100% ethanol)</TD><TD class="normal">20 mg/L (1% ethanol)</TD><TD class="normal">2 mg/L (0,1% ethanol)</TD></TR></TBODY></TABLE><H2><A name="effectors"> Effectors </A></H2><H3>Biological assay-anakinra </H3><OL><LI>HEK293T cells, seeded in 6-well plate, were transfected with anakinra downstream of constitutive promoter. </LI><LI>Transfected cells were incubated for 48 h. </LI><LI>To detect anakinra's effect on NF-κB signalling pathway, other HEK293T cells were transfected with plasmid coding for Renilla luciferase and plasmid reporter with NF-κB-inducible firefly luciferase expression. HEK293T cells express IL-1R, so additional transfection with a receptor gene was notneeded. </LI><LI>After 24 h, the medium was removed from cells transfected with reporter plasmids and 90 μL of anakinra-producing cells' supernatant was added to these wells. </LI><LI>After 24 h of stimulation, cells were lysed and NF-κB activation was assessed using dual luciferase assay. </LI></OL><H3> Biological assay-IFN-alpha </H3><OL><LI>HEK293T cells transfected with eithera  plasmid encoding IFN-alpha under the control of a constitutive promoter or an empty vector, and HEK293T cells transfected with the reporter vector were co-cultivated . </LI><LI>Additionally, we performed a co-transfection experiment, where HEK293T cells were transfected with both the reporter and the IFN-alpha encoding plasmids. </LI><LI>Day after transcfection cells were cultivated into 96-well plate at density 5x104 cells per well.</LI><LI>After 24 hours of incubation, dual luciferase reporter assay was performed. </LI></OL><H3>ELISA for IFN-alpha</H3><OL><LI>HEK293T cells where plated on 6 well plates and transfected with a plasmid coding for human IFN-alpha under the control of a constitutive promoter or a control plasmid (pcDNA3). </LI><LI>Supernatants were collected after 16h and serial dilutions were measured for IFN-alpha levels by Human IFN-alpha Instant Elisa (eBioscience). </LI></OL><H3>Plate reader-fluorescence(The Switch) </H3><OL><LI>HEK293T cells were seeded in black 96-well plates and transfected with plasmids encoding the switch. Plasmids and amounts used for transfection are listed in Figure legends. </LI><LI>Two hours after transfection, media was changed and cells were stimulated with inducers. Inducers and their concentration are described in Figure legends. </LI><LI>Media supplemented with inducer or without inducer were changed after two  or three days of cultivation. </LI><LI>After a maximum of 6 days after transfection,  cells were lysed with 25 µL of 1x Passive lysis buffer (Promega ) per well. </LI><LI>Fluorescence was measured using an automated plate reader. </LI></OL><H3>Plate reader-luminescence  (The Switch) </H3><OL><LI>HEK293T cells were seeded in white 96-well plates and transfected with plasmids encoding the switch. Plasmids and amounts used for transfection are listed in Figure legends. </LI><LI>Two hours after transfection, media was changed and cells were stimulated with inducers. Inducers and their concentration are described in Figure legends. </LI><LI>Media supplemented with inducer or without inducer were changed after two to three days of cultivation. </LI><LI>After a maximum of 6 days after transfection, cells were lysed with 25 µL of 1x Passive lysis buffer (Promega). </LI><LI>Luminescence of expressed reporter firefly luciferase was measured with Orion (Berthold Technologies) using Luciferase buffer with luciferin as a substrate. For normalization Renilla luciferase activity was used. The Renilla luciferase was measured using Renilla buffer supplemented with coelenterazine. </LI></OL><H3>Plate reader-absorbance  (The Switch) </H3><OL><LI>HEK293T cells were seeded in white 24-well plates and transfected with plasmids encoding the positive feedback loop switch and 10x[TALB + TALC] operator_CMV promoter_fLuciferase reporter plasmid and 10x[TALA + TALC]operator_CMV promoter_SEAP plasmid. Plasmids and amounts used for transfection are listed in Figure legends.</LI><LI>Media supplemented with inducer or without inducer were changed after two days of cultivation.</LI><LI>Two and seven days after transfection the growth medium was collected and SEAP QUANTIBlue substrate was added. After 15 minutes incubation at 37°C  the absorbance was measured at 630 nm. </LI></OL><H2><A name="microscopy"> Microscopy </A></H2><P>For spatial and temporal imaging of samples a Leica TCS SP5 laser scanning microscope mounted on a Leica DMI 6000 CS inverted microscope (Leica Microsystems, Germany) with a 10× and 20× dry objective and an HCX plan apo 63× oil (NA 1.4) oil immersion objective was used. For image analysis we used ImageJ (Image Processing and Analysis in Java) software (http://rsbweb.nih.gov/ij/) measuring the mean grey values of each cell containing the promoter of interest. </P><H3>Microscopy-cell viability with Hoechst and SytoxGreen514 (Safety mechanisms) </H3><P><B>Hoechst</B> dye is a membrane permeable dye and stains all cells in a culture. On the other hand a <B>SytoxGreen514</B> dye is a membrane impermeable dye staining only dead cells. Both dyes, blue fluorescent Hoechst and green fluorescent SytoxGreen514, bind to nucleic acids causing emission of fluorescent light. </P><OL><LI>HEK293 cells were seeded in an 8-well microscope chamber and transfected with 200 ng CMV-mGMK_TK30 (pPCMV_mGMK:TK30). </LI><LI>Ganciclovir (GCV) in concentrations 0, 10 and 100 μg/mL was added to the cell cultures. </LI><LI>After 5 days of cultivation, a Hoechst dye (0.4 μg/mL) and a SytoxGreen514 dye (1 μM) were used to stain cells and discriminate between live and dead cells. </LI><LI>Cells were incubated for approximately 10 minutes in the dark at 37 °C before imaging. </LI><LI>A 405-nm diode laser was used to excite Hoechst and a 514-nm line of 25 mW multi ion argon laser was used to excite SytoxGreen514. Successive images excited at 405 nm and 514 nm were captured. Fluorescence emission was detected at 450-500 nm and 520-560 nm for Hoechst and SytoxGreen respectively. </LI></OL><H3>Microscopy-cell growth (Safety mechanisms) </H3><OL><LI>HEK293 cells were seeded in an 8-well microscope chamber and transfected with 100 ng mGMK:TK30 (pPCMV_mGMK:TK30) and/or 20 ng GFP (pPCMV-GFP) (for transfection control). </LI><LI>Ganciclovir (GCV) in concentrations 0, 10 and 100 μg/mL was added to the cell cultures. </LI><LI>After 5 days of cultivation, a cell cultures were imaged. </LI><LI>A 514-nm line of 25 mW multi ion argon laser was used to excite GFP reporter protein. Fluorescence emission was detected at 520-560 nm for GFP. Bright field images were used to visualize the number of cells. </LI></OL><H3>Microscopy-cell count (Safety mechanisms) </H3><OL><LI>HEK293 cells were seeded in 12-well plates and transfected with different amounts of mGMK:TK30 (pPCMV_mGMK:TK30) as indicated in Figure legend. </LI><LI>Cell cultures were treated with ganciclovir (GCV) in concentrations as indicated in the Figure legend. </LI><LI>After incubation the cells were resuspended by pipetting. </LI><LI>Cells suspensions were then mixed with trypan blue. </LI><LI>Viable cell number was determined by counting the cells under a light microscope using a Bürker-Türk counting chamber. </LI></OL><H3>Microscopy-detection of reporter proteins (The Switch) </H3><P>Fluorescent proteins were used as reporters in &quot;The switch experiments&quot;. The fluorescent proteins used were blue (tagBFP), yellow (mCitrine), orange (mCherry) and red (mNeptun) fluorescent proteins. mCherry was used as transfection control while the others were used as reporters of &quot;the switch&quot;.</P><OL><LI>HEK293T cells were seeded in an 8-well microscope chamber or 12-well plate and transfected with plasmids encoding the switch. Plasmids and amounts used for transfection are listed in Figure legends. </LI><LI>Two hours after transfection, media was changed and cells were stimulated with inducers. Inducers and their concentration are described in Figure legends. </LI><LI>Media supplemented with inducer or without inducer were changed after two or three days of cultivation. </LI><LI>Images of cells expressing reporters were taken two days after transfection and then each day for 5 days. </LI><LI>A 405-nm diode laser was used to excite tagBFP, a 514-nm line of 25 mW multi-ion argon laser was used for mCitirne, a 543-nm HeNe laser was used for mCherry and a 633-nm HeNe laser was used to excite mNeptune. Successive images excited at 405, 514, 543 and 633 nm were captured. All intensities of laser and photomultipliers were kept unchanged during one set of experiments to enable comparison of images. Fluorescence emission was detected at 450-500 nm, 520-560 nm, 560-600 nm and 640-700 nm for tagBFP, mCitrine, mCherry and mNeptune, SytoxGreen respectively. </LI></OL><H3>Microscopy-alginate degradation (Microencapsulation) </H3><P>To observe the degradation of alginate beads, 2000 kDa FITC-dexstran (Sigma) was added to 200 µL of culture medium containing alginate beads with immobilized HEK 293T cells. Because FITC-dexstran cannot penetrate the alginate beads, we can easily observe bead degradation upon addition of alginate lyase from Sphingobacterium multivorum (Sigma).</P><OL><LI>Alginate beads suspended in culture medium were seeded in an 8-well microscope chamber (200 µL). </LI><LI>20 µL of 1 mg/mL FITC-dextran were added into well. </LI><LI>After the dye was evenly distributed throughout the suspension, 8 µL of Sphingobacterium multivorum alginate lyase were added. </LI><LI>The microscope was set to capture images every 20 seconds. </LI><LI>Screenshots were collected for at least 15 minutes. </LI><LI>A 488-nm line of 25 mW multi-ion argon laser was used for FITC. Fluorescence emission was detected at 520-560 nm. At the same time a bright field image was taken. </LI></OL><H3>Microscopy-secreted alginate lyase enzymatic activity (Microencapsulation)</H3><OL><LI>HEK293T cells were seeded 1×10<SUP>6</SUP> on 10 cm cell culture dish and grown in DMEM medium supplemented with 10 % FBS.</LI><LI>After reaching 50 – 70 % confluency, cells were transfected with 15 μg of DNA per culture dish with jetPEI transfection reagent (Polyplus Transfection).</LI><LI>Protein production lasted for 72 hours.</LI><LI>Cell supernatants were collected and concentrated 50-times using Sartorius Vivaspin 6 concentrators (5 kDa MWCO PES).</LI><LI>Alginate beads were produced with Büchi BIOTECH Encapsulator (see Microencapsulation: Encapsulation procedure 1.-6.).</LI><LI>Beads were incubated with concentrated supernatants for 72 hours in an 8-well microscope chamber.</LI><LI>20 µL of 1 mg/mL FITC-dextran were added into wells.</LI><LI>A 488-nm line of 25 mW multi-ion argon laser was used for  FITC detection. Fluorescence emission was detected at 520-560 nm.</LI><LI>Beads' diameters were assessed using Leica LAS Image Analysis software.</LI></OL><H3>Microscopy-encapsulated cell viability (Microencapsulation) </H3><P>To observe encapsulated cells' viability, HEK 293T cells were stained with Hoechst and 7-aminoactinomycin D (7-AAD) viability stains. Hoechst stains both live and dead cells, while 7-AAD stains dead cells only.</P><OL><LI>Encapsulated cells were grown in DMEM culture medium supplemented with 10% FBS. </LI><LI>200 µL of the microcapsule suspension was collected and alginate-PLL capsules were seeded into an 8-well microscope chamber. </LI><LI>5 µL of 7-AAD and 1 µL of Hoechst stain were added to one well. </LI><LI>Encapsulated cells were protected from direct light and stained for 30 minutes at 37 °C. </LI><LI>A 405-nm diode laser was used to excite Hoechst and a 543-nm line of HeNe laser was used to excite 7-AAD. </LI><LI>Fluorescence emission was detected at 450-500 nm and 600-700 nm for Hoechst and 7-AAD respectively. </LI></OL><H2><A name="flowcytometry"> Flow cytometry </A></H2><P>Flow cytometry is a laser based technology employed in cell counting and biomarker detection. It allows simultaneous multiparametric analysis of the physical as well as biochemical and biological characteristics of particles. We used a CyFlow Space (Partec) flow cytometer equipped with three lasers (405, 488 and 633 nm). The CyFlow detects forward scatter and side scatter signals and up to 6 colors of fluorescence.</P><H3>Flow cytometry - the annexin assay (Safety mechanisms) </H3><P>To determine the percentage of cells undergoing apoptosis as a result of herpes simplex virus thymidine kinase (HSV-TK) (pCMV-mGMK_TK30) transfection and ganciclovir treatment we labeled cells with Annexin V conjugated with phycoerythrin (PE). Annexin V is a Ca2+ dependent phospholipid-binding protein that has a high affinity for the phospholipid phosphatidylserine and therefore binds to apoptotic cells with  phosphatidylserine exposed on their surface.</P><OL><LI>HEK293 cells were seeded in 12-well plates. </LI><LI>Cells were transfected with pCMV-mGMK_TK30 and treated with ganciclovir. Concentrations of ganciclovir and plasmids are indicated in Figure legends. </LI><LI>After incubation the cells were washed with PBS buffer and resuspended by pipetting. </LI><LI>Cells were pelleted with centrifugation at 1200 rpm. </LI><LI>The cell pellet was washed in 1x Annexin Binding Buffer (10 mM HEPES, 140 mM NaCl, and 2.5 mM CaCl2, pH 7.4). </LI><LI>The pellet was then resuspended in 1x Annexin Binding Buffer and PE-Annexin V (5 µl per 100 μl cell suspension) was added. </LI><LI>Samples were incubated for 20 minutes in the dark at room temperature and then immediately analyzed with a flow cytometer. </LI><LI>Along with site and forward scatter, the signal in the FL2 channel  (540-580 nm) was also recorded. </LI></OL><H3>Flow cytometry - the propidium iodide assay (Safety mechanisms) </H3><P>To determine the percentage of dead cells due to cytotoxic activity of natural killer cells against HEK293T cells expressing MICA protein, cells were stained with propidium iodide dye, which intercalates into DNA and stains only dead cells, because it is a membrane impermeant fluorescent molecule.</P><OL><LI>HEK293T cells seeded in 12-well plates were transfected with plasmids expressing MICA (pPCMV-MICA_pcDNA3).</LI><LI>Two days after transfection with CMV-MICA_pcDNA3, cells were resuspended in PBS at final concentration 1x10<SUP>6</SUP> cells/mL, stained with CFSE (0,6μM) and incubated for 10 minutes at 37°C. Staining was quenched by the addition of 5 volumes of ice-cold culture media (RPMI+ 20% FBS) to the cells. After 5 minutes incubation on ice, cells were pelleted by centrifugation and then washed by resuspending the pellet in fresh media (RPMI+ 20% FBS) a further two times for a total of three washes. CFSE was used to discriminate between HEK293T and NK-92 cells or between NK target cells K562, which were used as a positive control, and NK-92 cells.</LI><LI>HEK293T cells or K562 cells were mixed with NK-92 cells in different ratios (1:1, 1:5, 1:10) and incubated for 4 hours at 37 °C in culture medium consisting of RPMI, 20% FBS and hIL-2 (100 U/ml).</LI><LI>After incubation of HEK293T or K562 cells  with NK-92 cells, cells were treated with propidium iodide.</LI><LI>Along with site and forward scatter the signal in the FL1 channel (530-580 nm) was also recorded.</LI></OL><H3>Flow cytometry - detection of reporter proteins (The Switch) </H3><P>Reporters such as fluorescent proteins were used to detect the expression of effectors in &quot;The switch experiments&quot;. As reporters we used blue (tagBFP) and yellow (mCitrine) fluorescent proteins.</P><OL><LI>HEK293T cells were seeded in a 12 or 24-well plate and transfected with plasmids encoding the switch. </LI><LI>Two hours after transfection, media was changed and cells were stimulated with inducers. </LI><LI>Medium with inducer or without the inducer was changed after two days of cultivation. </LI><LI>Cells were collected at different time points (2 days after induction and then 3 days after the second media change). </LI><LI>Cells were washed and resuspended in PBS buffer. </LI><LI>A 405 nm diode laser was used to excite tagBFP and a 488-nm diode laser was used for mCitirne. </LI><LI>Along with site and forward scatter signals in the FL1 (540-580 nm) channel (mCitrine) and the FL5 (450-480 nm) channel (tagBFP) were also recorded. </LI></OL><H2><A name="microencapsulation"> Microencapsulation </A></H2><H3>Cell preparation for encapsulation</H3><OL><LI>HEK 293T cells were seeded 5x10⁵ per 10 cm cell culture dish (3 per encapsulation) and grown in DMEM medium supplemented with 10 % FBS. </LI><LI>After reaching 50 – 70 % confluency, cells were transfected with 15 μg of DNA per culture dish with jetPEI transfection reagent (Polyplus Transfection). </LI><LI>The medium was removed the next day.. Transfected cells were detached using 3 mL of trypsin solution and centrifuged after the addition of fresh medium to inactivate the trypsin. </LI><LI>Supernatant was removed and cells were resuspended in 15 mL DMEM with 10% FBS. </LI><LI>Cells were counted using Countess automated cell counter (Invitrogen). </LI><LI>HEK 293T cells were again centrifuged and supernatant was removed. </LI><LI>Cells were resuspended in 2 mL of pre-warmed MOPS buffer. </LI><LI>10 mL of pre-warmed alginate solution (1,5%) was added to cell suspension. </LI></OL><H3>Encapsulation</H3><OL><LI>The encapsulator was equipped with a 200-µm nozzle. </LI><LI>The reactor vessel was filled with 225 mL 100 mM CaCl2. </LI><LI>Cell-alginate mixture was transferred to a 20 mL syringe with a Luer lock. </LI><LI>The syringe was connected to the bead producing unit (BPU). </LI><LI>Microcapsules were produced at a flow rate of 12-14 units, vibration frequency 1030-1100 Hz and voltage for bead dispersion 900-1300 V. </LI><LI>Polymerization lasted for 10 minutes. </LI><LI>The polymerization solution was drained and 75 mL of 0,05% poly-L-lysine (PLL) solution was added. </LI><LI>Beads were incubated in PLL solution for 10 minutes. </LI><LI>The PLL solution was removed and beads were washed twice (for 1 and for 5 minutes) with 150 mL of MOPS buffer. </LI><LI>100 mL of 0,03% alginate was added and beads were incubated for 10 minutes. </LI><LI>Alginate solution was drained and beads were washed once with 150 mL of MOPS buffer for 1 minute.</LI><LI>150 mL of depolymerization solution was added for 10 minutes. </LI><LI>Depolymerization solution was removed and capsules were resuspended in 150 mL MOPS and collected in a bead collection flask. </LI><LI>MOPS was removed and microcapsules were transferred to T-75 with 10 mL DMEM, 10% FBS media supplemented with penicillin and streptomycin. </LI></OL><P><B>Buffers and solutions</B><B>10 mM MOPS buffer</B> (pH = 7,2)
<B>MOPS buffer with NaCl</B> (pH = 7,2):    10 mM MOPS,    0,85% NaCl
<B>Polymerisation solution</B> (pH = 7,2):    10 mM MOPS,    100 mM CaCl2
<B>Depolymerisation solution</B> (pH = 7,2):    10 mM MOPS,    50 mM Na3-citrate,    0,45% NaCl
<B>0,05% poly-L-lysine</B> (15-30 kDa) in MOPS buffer (pH = 7,3) 
<B>0,03% alginate</B> in MOPS buffer (pH = 7,2)
<B>1,5% alginate</B> (low viscosity) in MOPS buffer (pH = 7,2)</P><H2><A name="proteindetection"> Protein detection </A></H2><H3>SDS-PAGE</H3><OL><LI>Samples were loaded on a 12% acrylamide gel and ran at a constant voltage (200 V) for 1 h. </LI><LI>Proteins were then blotted on a nitrocellulose membrane at a constant current (350 mA) for 1 h. </LI><LI>The membrane was washed with MQ and PBS and blocked for 1,5 h by incubation in I-Block blocking reagent at room temperature. </LI></OL><H3>Immunodetection </H3><OL><LI>All proteins to be detected had a Myc tag at the C-terminus. </LI><LI>The membrane was incubated with primary antibodies (rabbit anti-Myc diluted 1:500) overnight at 4 °C and 150 rpm. Membrane was washed. </LI><LI>Washing in wash buffer three times for 5 minutes each.</LI><LI>Membrane was incubated with secondary antibodies (anti-rabbit secondary antibodies, conjugated with HRP, diluted 1:3000) for 45 minutes at room temperature and 150 rpm. </LI><LI>HRP activity was detected by addition of SuperSignal West Femto or Pico Substrate (Thermo Scientific). Images were captured with Syngene G:Box chemiluminescent imaging system. </LI></OL><P><B>Buffers and solutions </B></P><P><B>Wash buffer</B>:    1x PBS,    0,01% (v/v) Tween 20</P><H2 style="color:grey;">References</H2><P style="color:grey;">
Clackson, T. (2000) Regulated gene expression systems. <I> Gene ther.</I><B>7</B>, 120–125.
Deuschle, U., Meyer, W.K. and Thiesen, R. (1995) Tetracycline-reversible silencing of eukaryotic promoters. <I>Mol. Cell. Biol. </I><B>15</B>, 1907–1914.
Kramer, B.P., Fischer, C. and Fussenegger, M. (2004) BioLogic gates enable logical transcription control in mammalian cells. <I> Biotech. Bioeng.</I><B>87</B>, 478–484.
Pollock, R. and Clackson, T. (2002) Dimerizer-regulated gene expression. <I> Curr. Opin. Biotech. </I><B>13</B>, 459–467.
Gibson, D.G., Young, L., Chuang, R., Venter J.C., Hutchison III, C. A. and Smith, H.O. (2009) Enzymatic assembly of DNA molecules up to several hundred kilobases. <I>Nature methods.</I><B>6</B>, 343–345.
</P><B>
Next: <A href="https://2012.igem.org/Team:Slovenia/NotebookLablog">Lablog &gt;&gt;</A></B></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Slovenia/Notebook">http://2012.igem.org/Team:Slovenia/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Slovenia/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Slovenia/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Slovenia/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Slovenia/Notebook&amp;oldid=291259" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>