"
Page
Discussion
View source
History
teams
Log in
 
Team:Slovenia/TheSwitch
From 2012.igem.org
Home
Idea
Biological therapy
Challenge
Solution
The switch
Overview
Designed TAL regulators
Mutual repressor switch
Positive feedback loop switch
Controls
Safety mechanisms
Overview
Escape tag
Termination
Microcapsule degradation
Implementation
Overview
Hepatitis C
Ischaemic heart disease
Impact
Modeling
Overview
Pharmacokinetics
Modeling methods
Mutual repressor switch
Positive feedback loop switch
Experimental model
Interactive simulations
Parts
Notebook
Experimental methods
Lablog
Lab safety
Society
Human practice
Scientists
Physicians
Ethics, safety and regulations
Patients
Journalists and general public
Outreach
Questionnaire
Team
Team members
Attributions
Collaborations
Gallery
Sponsors
Overview
Introduction of complex regulatory circuits in mammalian cells requires the availability of a large number of designable orthogonal switches. We provide this new type of bistable toggle switch based on designable TAL regulators as a powerful tool to the synthetic biology community.
We designed and tested several TAL repressors, activators and their reporter plasmids with their corresponding DNA-binding sites and promoters. The TAL repressors exhibited over 90% repression and TAL activators enhanced expression over 1500 fold.
We tested five induction systems for mammalian cells and adapted three of them for driving TAL regulators in genetic switches.
We designed and modeled a mutual repressor toggle switch based on two TAL repressors repressing each other.
Experimental results demonstrated low stability of the basic mutual repressor switch, based on the classical toggle switch topology. This was confirmed by modeling analysis, which indicated high sensitivity to leaky gene expression and a requirement for high cooperativity of repressors in order to exhibit bistable behavior.
We designed an improved toggle switch that included additional positive feedback loops based on orthogonal TAL repressors and activators competing for the same operator.
Modeling demonstrated higher robustness in comparison to the mutual repressor switch and bistability even without cooperative binding to DNA, since the positive feedback loop introduces the nonlinearity.
Experimental results of the improved switch demonstrated two stable states at induction with corresponding inducers and a clear bimodal cell population distribution. The switch remained in a stable state after removal of inducers, which confirmed the epigenetic bistability of our system. We included numerous control experiments demonstrating that the resultsare indeed due to the functional switch.
(NEW) Addition of the second inducer after the system has already adopted one state resulted in a successful switch from one state into the other, confirmed by three different experimental techniques .
Figure 1. Schematic representation of a positive feedback loop switch.
Our project aims to produce different biopharmaceuticals by engineered cells. With tight but versatile regulation and safety in mind, we set out to contribute some fundamental advances to synthetic biology.
We considered two options of regulating production of therapeutic proteins in mammalian cells:
The first option could be a prosthetic network where the cells are engineered to sense endogenous signals such as for example blood glucose levels. This signal would control the production of a therapeutic protein (such as insulin), which in turn would affect the level of the endogenous signal (i.e. glucose) and be thus regulated by a kind of a feedback loop. This type of system can replace the function of a defective tissue or organ, hence the name prosthetic. Although a very attractive option, such a system would need to be tailor-made for each disease in particular.
For the second option we regarded a cellular genetic network that produces the therapeutic protein of choice, but could be controlled by an external signal, such as small molecular drugs, light or metabolites, which can be administered orally or topically. Instead of requiring a continuous presence of an activating or repressing signal, the system should rather function as a bistable or multistable switch, requiring only a signal pulse to change into any selected state and remain stable in this state. This type of toggle switch has already been implemented in mammalian cells based on prokaryotic transcription factors fused to the eukaryotic transactivator (e.g. VP16, VP64) or transrepressor (e.g. KRAB) domains (Karlsson et al., 2012).
We consulted several physicians concerning different potential therapeutic applications of thistype of delivery system. Physicians by and large preferred the ability of the device to be controlled by the therapist to the autonomous switching by sensing internal physiological signals. An additional advantage of a controlled switch is that a very similar device core could be used for many different therapeutic applications.As examples of medical problems from the expertise of consulted physicians we considered wound healing, hepatitis C infection and cardiovascular ischemia .For hepatitis C and ischaemic heart disease we constructed two sets of therapeuticeffectors and tested some of these effectors’ biologic activity.
Synthetic biology has already demonstrated the ability of a genetic device to assume different stable states. However, while prokaryotic regulators appear to be an efficient tool for device construction, their number is limited and currently does not allow the scalability required for construction of complex logical functions. On the other hand, modular DNA-binding protein domains such as zinc finger proteins or Transcription-activator like (TAL) domains can be designed to bind to almost any DNA sequence, providing high orthogonality.
Therefore the challenge was to prepare an epigenetic switch based on designable DNA binding domains, such as zinc finger proteins or TAL effectors that can be designed to recognize selected exogenous DNA sequences. This type of orthogonal switch would allow the simultaneous introduction of several bistable or multistabletoggle switches into mammalian cells for advanced level of control in different applications. To our knowledge, a designed modular DNA binding protein-based toggle switch has not been implemented experimentally so far, neither in mammalian nor in bacterial cells. This endeavor has to be supported by mathematical simulationto evaluate which type of genetic switch topology has the best performance, is more robust and which parameters are most critical for its performance.
Epigenetic toggle switches havepreviously been constructed based onnatural bacterial transcription factors (Gardner et al., 2000). An epigenetic toggle switch is a genetic network, designed to memorize its state and maintain expression of a gene of choice even after the inducer has been removed (Cherry & Adler, 2000). Only when a second signal (inducer) is provided to the cells, the system will switch to a different state, shutting off expression of the first protein and/or initiating production of a second one. This type of toggle switch allows the system to assume two discrete states. For our therapeutic purposes this could mean switching the production of a single protein on and off or selecting between the production of two different protein effectors. Therefore a combination of several switches would permit selection of advanced therapeutic regimens.
If we want the switches to function independently of each other andisolated from other cellular processes, they need to be based on the orthologous DNA-binding domains. We therefore decided to construct the genetic core of our device from TAL effectors, whose code of DNA recognition has been deciphered recently (Boch et al., 2009). TAL effectors have a modular structure, with each module recognizing a single base pair. TAL effectors, fused to theFokI nuclease, were successfullyadopted as highly specific and easily customizable genome editing tools (Christian et al, 2010; Bogdanove et al.; 2011, Li et al., 2011). For synthetic biologists, TAL effectors represent perfect candidates for engineering artificial transcription factors due to their high specificity, modularity, ability to target virtually any sequence and last, but not least, they do not exhibit toxicity when expressed in cells, except in case they target an essential endogenous gene. Indeed, in addition to endonucleases, transcription factors employing TALs as DNA binding domains have also been reported recently: transcription activators were constructed by fusing TALs to the mammalian activation domain VP16 or VP64 (Zhang et al., 2011) and, when our project already started, transcription repressors were reported, based on fusion of TALs with theKRAB heterochromatin silencing domain (Garg et al., 2012; Cong et al., 2012).
References
Boch, J., Scholze, H., Schornack, S., Landgraf, A., Hahn, S., Kay, S., Lahaye, T., Nickstadt, A., and Bonas, U. (2009) Breaking the Code of DNA Binding Specificity of TAL-Type III Effectors. Science. 326, 1509-1512.
Bogdanove, A. J., and Voytas, D. F. (2011) TAL Effectors: Customizable Proteins for DNA Targeting. Science. 333, 1843-1846.
Cherry, J. L. and Adler, F. R. (2000) How to make a Biological Switch. J. Theor. Biol. 203, 117-133.
Christian, M., Cermak, T., Doyle, E. L., Schmidt, C., Zhang, F., hummel, A., Bognadove, A. J., and Voytas, D. F. (2010) Targeting DNA Double-Strand Breaks with TAL Effector Nucleases. Genetics 186, 757-761.
Cong, L., Zhou, R., Kuo, Y.C., Cunniff, M., Zhang, F. (2012) Comprehensive interrogation of natural TALE DNA-binding modules and transcriptional repressor domains. Nat Commun.3, 968.
Gardner, T. S., Cantor, C. R., and Collins, J. J. (2000) Construction of a genetic toggle switch in Escherichia coli. Nature. 403, 339-342.
Garg, A., Lohmueller, J. J., Silver, P. A. and Armel, T.Z. (2012) Engineering synthetic TAL effectors with orthogonal target sites. Nucleic Acids Res. 40, 7584-95.
Karlsson, M., Weber, W., and Fussenegger, M. (2012) Design and construction of synthetic gene networks in mammalian cells. Methods Mol. Biol. 813, 359-376.
Li, T., Huang, S., Jiang, W. Z., Wright, D., Spalding, M. H., Weeks, D., P., and Yang, B. (2011) TAL nucleases (TALNs): hybrid proteins composed of TAL effectors and FokI DNA-cleavage domain. Nucleic Acids Res. 39, 359-372.
Zhang, F., Cong, L., Lodato, S., Kosuri, S., Church, G. M., and Arlotta, P. (2011) Efficient construction of sequence-specific TAL effectors for modulating mammalian transcription. Nat. Biotechnol. 29, 149-153.
Next: Designed TAL regulators >>
Retrieved from " http://2012.igem.org/Team:Slovenia/TheSwitch "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
