"
Page
Discussion
View source
History
teams
Log in
 
Team:Slovenia/TheSwitchDesignedTALregulators
From 2012.igem.org
Home
Idea
Biological therapy
Challenge
Solution
The switch
Overview
Designed TAL regulators
Mutual repressor switch
Positive feedback loop switch
Controls
Safety mechanisms
Overview
Escape tag
Termination
Microcapsule degradation
Implementation
Overview
Hepatitis C
Ischaemic heart disease
Impact
Modeling
Overview
Pharmacokinetics
Modeling methods
Mutual repressor switch
Positive feedback loop switch
Experimental model
Interactive simulations
Parts
Notebook
Experimental methods
Lablog
Lab safety
Society
Human practice
Scientists
Physicians
Ethics, safety and regulations
Patients
Journalists and general public
Outreach
Questionnaire
Team
Team members
Attributions
Collaborations
Gallery
Sponsors
TAL-based transcriptional regulators
We created several TAL repressors by fusions of the KRAB repression domain to different positions relative tothe TAL DNA-binding domain and reporter plasmids with their respective binding sites (operators).
We created several TAL activator constructs by C-terminal fusion of the VP16 domain with TAL DNA-binding domains and reporter plasmids with their respective operators.
We improved and characterized the NicTAL DNA-binding domain (deposited by the iGEM2010 team Slovenia) by adding the missing subdomain of the protein and created a designed repressor and activator.
TAL:KRAB fusions exhibited over 90% repression of reporter gene expression regardless of the position of the KRAB domain.
Minimal promoters used for construction of reporter plasmids showed no or minimal leakiness and were activated over 1500-fold by TAL:VP16 fusions.
(NEW) Our experimental results on designed TAL regulators provided parameters for the quantitative deterministic modeling of bistable switches.
Designed TAL transcriptional regulators
For the past two decades, engineered zinc finger proteins have been extensively used for targeting specific DNA sequences. However, in spite of the many years of technological development, engineered zinc finger proteins are not able to target every desired DNA sequence due to the impact of neighboring fingers on the recognition of base pairs. Recently DNA-binding proteins with a simpler DNA recognition code were discovered. Transcription activator like (TAL) effectors are bacterial plant pathogen transcription factors that bind to DNA by recognizing a specific DNA sequence in which each base pair binds to a single tandem repeat in the TAL DNA-binding domain (Figure 1A). A tandem TAL repeat contains 33 to 35 amino acids, where the 12th and the 13th amino acid, called a “repeat variable diresidue” (RVD), are responsible for specific interactions with the corresponding base pair (Scholze and Boch, 2011). As evident from the crystal structure of TAL effectors (Mak et al., 2012; Deng et al., 2012; Figure 1B), all TAL repeats have almost identical conformations, differing only in the RVDs. This modularity of TAL effector binding domains therefore makes them a perfect tool to target specific DNA sequences.
Figure 1.The structure of TAL effectors. (A) Schematic representation of TAL effector structure and its DNA-binding domain (red), containing multiple 34 aminoacid tandem repeats with RVDs at the 12th and 13th residue (Scholze and Boch, 2011). (B) 3D structure of a TAL effector.
Figure 2.Models of 3D structures of TAL DNA-binding domains fused with the KRAB repression domain (A) and VP16 activation domain (B).
TAL effectors have mostly been used as a tool for plant or mammalian genome editing. The basic idea is the same as with zinc finger nucleases, with TALs replacing zinc fingers as the specific DNA-binding domain (Miller et al., 2010). Several groups (Miller et al., 2011; Zhang et al, 2011; Garg et al., 2012; Cong et al., 2012) have also designed TAL effectors for specific gene activation, by fusing them with either the Herpes simplex virus VP16 activation domain or its tetrameric derivative VP64 (Figure 2). After we already initiated the iGEM 2012 project, TAL repressors were reported by Garg et al., and Cong et al.,who created TAL effectors fused with the KRAB or SID transcriptional repression domain.
Results
Figure 3. Schematic representation of tested plasmids. (A) TAL repressors; fusions of TAL DNA-binding domains with the KRAB repression domain. (B) TAL activator; fusion of TAL DNA-binding domain with the VP16 activation domain. Expression of TAL effectors is under the control of constitutive CMV promoter. (C) Reporter plasmids used to test efficiency of TAL regulators. TAL DNA-binding sites are placed upstream of either a CMV promoter (repression) or a minimal promoter (activation), driving the expression of reporter genes (firefly luciferase or mCitrine).
The TAL transcriptional activators and repressors were basic tools in our iGEM project. We designed and characterized three functional TAL regulators ( TALA , TALB and TALD ) by fusing TAL DNA-binding domains (Sander et al., 2011) with the VP16 activation domain (Figure 3B) or a KRAB repression domain (Figure 3A), as shown on Figure 2. To assess the activity of designed TAL regulators, we also designed reporter plasmids , which contain several repeats of TAL binding sites upstream of either a CMV promoter (repression) or a minimal promoter (activation) (Figure 3C).
In addition to the synthesis of new TAL effector-based parts and their characterization, our team also improved a part which was deposited in the Registry by the Slovenian iGEM2010 team. They synthesized a TAL effector, named NicTAL , which did not work as expected in mammalian cells. We discovered that a subdomain next to the DNA-binding domain was missing, because the requirements for the functional TAL binding domains have not been known two years ago. We linked the missing subdomain to the DNA-binding domain of NicTAL from the Registry. Additionally we prepared chimeric proteins of the NicTAL-DNA binding domain and KRAB or VP16 domains, generating another repressor and activator pair and demonstrated the newly acquired functionality of the NicTAL-based regulators.
Designed repressors
We designed and tested three different variants of TAL DNA-binding domain fusions with the KRAB repression domain. KRAB was placed either on both termini or on the N- or C-terminus of the TAL DNA-binding domain (Figure 3A). All tested constructs offour different TAL domains exhibited over 90% repression of the reporter plasmid (with the exception of KRAB:TALD ). We expected to observe a difference in repression due to potential clustering of KRAB-binding proteins, but no significant variation between constructs was noticeable. Our conclusion is that the position of the effector (regulator) domain on either the N- or C-terminus or both does not influence the binding and repression ability of the designed TAL repressors. All further experiments were performed with TAL:KRAB fusions.
Figure 4. Schematic representation of repression experiments. (A) In the absence of a TAL repressor, the reporter gene is constitutively expressed. (B) When a TAL repressor is expressed, it binds to its respective DNA-binding site upstream of the CMV promoter and represses transcription of the reporter gene through KRAB domain-mediated transcriptional silencing.
Figure 5. TAL repressors potently inhibit expression of reporter genes. HEK293T cells were cotransfected with TAL repressors under the control of a CMV promoter (50 ng), and with a firefly luciferase reporter plasmid (Figure 3C) containing 10 DNA-binding sites for the designated TAL repressor upstream the CMV promoter (10 ng). Along with the tested constructs we transfected cells with 5 ng of Renilla luciferase under the HSV-TK promoter as transfection control. Luciferase activity was measured 3 days post-transfection. All experiments were executed in 3 biological replicates and repeated more than 3 times with similar results.
Results show that the previously non-functional NicTAL10:KRAB fusion (NicTAL DNA-binding domain constructed by the iGEM2010 team Slovenia) acquired functionality by the N-terminal addition of a subdomain to the TAL DNA-binding domain (Figure 5). An excellent repression ability of NicTAL12:KRAB (the improved version by the 2012 team) was observed. The NicTAL12 repressor was further characterized by testing the effect of a different number of binding sites upstream of the PCMV promoter on the inhibition of reporter expression. Results presented in Figure 6 show that the maximal effect of the bound TAL regulator plateaus at 7 or more copies of binding sites per operator. In all further experiments we used plasmids with 10 copies of TAL DNA-binding sites.
Figure 6. Number of DNA-binding sites specific for NicTAL12:KRAB repressor dictates the efficiency of inhibition of reporter expression. HEK293T cells were cotransfected with NicTAL repressors under CMV promoter (50 ng), and firefly luciferase reporter plasmids (Figure 3C) with different number of NicTAL binding sites upstream of the CMV promoter (100 ng). Luciferase activity was measured 3 days post-transfection. The experiment was executed in threebiological replicates and repeated three times with similar results.
Designed activators
Figure 7. Schematic representation of activation experiments. (A) In the absence of a TAL activator, there is no expression of the reporter gene. (B) When TAL activator is present, it binds to its DNA-binding site upstream of the minimal promoter and activates transcription of the reporter gene.
Due to the mechanism of action of the VP16 domain and our previous results with TAL:KRAB repressors (Figure 5, position of the effector domain does not influence repression); we decided to test only the C-terminal variant of the TAL:VP16 fusion (Figure 3A). Both tested TAL activators exhibited over 1500-fold activation of the mCitrine reporter at reporter to activator ratios 1:2. In addition, we confirmed that the minimal promoter used to drive the expression of the reporter gene shows no (or minimal) leakiness - this trait makes this promoter an excellent element for genetic systems, where tight transcriptional regulation is needed.
Figure 8. TAL activators strongly activate the reporter gene expression. The number of DNA-binding sites specific for NicTAL12:KRAB repressor dictates the efficiency of inhibition of reporter expression. HEK293T cells were cotransfected with TAL activator constructs under the CMV promoter (different quantities, for ratios see the x axis), and mCitrine reporter plasmids (Figure 3B) containing 10 copies of binding sites for the designated TAL activator upstream of a minimal promoter (50 ng). Along with the tested constructs we transfected cells with 20 ng of mCherry fluorescent protein under the CMV promoter as transfection control. Fluorescence was measured three days post-transfection. All experiments were executed in 3 biological replicates and repeated three times with similar results.
(NEW) TAL regulators are functional and nontoxic for multicellular animals
In collaboration with iGEM team Evry we tested the ability of TAL regulators to function in cells of amphibians. We selected a reporter plasmid with mCitrine under the operator for TAL VP16 activator in the presence and absence of the TAL activator. Only cells of animals transfected with both plasmids exhibited fluorescence, which was absent in cells transfected only with reporter. It is exciting to see fluorescent tadpoles that were transfected with functional TAL activator and a reporter, which demonstrates that TAL activator is not deleterious for the development of the animal.
Figure 9. TAL-based transcriptional activator is active in a multicellular organism. Reporter plasmid (10x[TALA] pMIN mCitrine, BBa_K782029 ) was injected into Xenopus laevis embryo (2.3 nl of plasmid at concentration 100 ng/µl) in the presence or absence of a plasmid with TAL activator (pCMV-TALAVP16, BBa_K782065 ). Fluorescence of mCitrin and rhodamine, which was used with plasmid for injection control was observed 1 and 3 days after introduction of plasmids under the microscope as described https://2012.igem.org/Team:Evry/Protocols .
Our joint results obtained in collaboration with Evry team (thank you froggies) thus demonstrate that TAL-based logic can be used also in the whole animal, which could be used as an excellent model to study complex synthetic regulatory devices in the multicellular environment.
References
Deng, D., Yan, C., Pan, X., Mahfouz, M., Wang, J., Zhu J. K., Shi, Y., and Yan, N.  (2012) Structural basis for sequence-specific recognition of DNA by TAL effectors. Science 335, 720-723.
Garg, A., Lohmueller, J. J., Silver, P. A. and Armel, T.Z. (2012) Engineering synthetic TAL effectors with orthogonal target sites. Nucleic Acids Res. 40, 7584-95.
Cong, L., Zhou, R., Kuo, Y.C., Cunniff, M., Zhang, F.(2012) Comprehensive interrogation of natural TALE DNA-binding modules and transcriptional repressor domains. Nat Commun.3, 968.
Mak, A. N., Bradley, P., Cernadas, R. A., Bogdanove, A. J., and Stoddard, B. L. ( 2012) The crystal structure of TAL effector PthXo1 bound to its DNA target. Science 335, 716-719.
Miller, J. C , Tan, S., Qiao, G., Barlow, K. A., Wang, J., Xia, D. F., Meng, X., Paschon, D. E., Leung, E., Hinkley, S. J., Dulay, G. P., Hua, K. L., Ankoudinova, I., Cost, G. J., Urnov, F. D., Zhang, H. S., Holmes, M. C., Zhang, L., Gregory, P. D., and Rebar, E. J. (2011) A TALE nuclease architecture for efficient genome editing. Nat. Biotechnol. 29, 143-148.
Sander, J. D., Cade, L., Khayter, C., Reyon, D., Peterson, R. T., Joung, J. K., and Yeh, J.-R. J. (2011) Targeted gene disruption in somatic zebrafish cells using engineered TALENs. Nat. Biotechnol. 29, 697–698.
Scholze, H., and Boch, J. (2011) TAL effectors are remote controls for gene activation. Curr. Opin. Microbiol. 14, 47-53.
Zhang, F., Cong, L., Lodato, S., Kosuri, S., Church, G. M., and Arlotta, P. (2011) Efficient construction of sequence-specific TAL effectors for modulating mammalian transcription. Nat. Biotechnol. 29, 149-153.
Next: Mutual repressor switch >>
Retrieved from " http://2012.igem.org/Team:Slovenia/TheSwitchDesignedTALregulators "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
