<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UAB-Barcelona_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:UAB-Barcelona/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UAB-Barcelona/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UAB-Barcelona/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UAB-Barcelona/Modeling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UAB-Barcelona/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UAB-Barcelona/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P><TITLE>Documento sin título</TITLE></P><DIV id="info"><UL class="tabs"><LI class="courier"><A href="https://2009.igem.org/Team:UAB-Barcelona"><EM>Home</EM></A></LI><LI><A href="https://2009.igem.org/Team:UAB-Barcelona/Team"><EM>The Team</EM></A></LI><LI><A href="https://2009.igem.org/Team:UAB-Barcelona/Project"><EM>The Project I</EM></A></LI><LI><A href="https://2009.igem.org/Team:UAB-Barcelona/Project2"><EM>The Project II</EM></A></LI><LI><A href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2009&amp;group=UAB-Barcelona"><EM>Registered parts</EM></A></LI><LI><A href="https://2009.igem.org/Team:UAB-Barcelona/Modeling"><EM>Model I</EM></A></LI><LI><A href="https://2009.igem.org/Team:UAB-Barcelona/Modeling2"><EM>Model II</EM></A></LI><LI><A href="https://2009.igem.org/Team:UAB-Barcelona/Notebook"><EM>Notebook</EM></A></LI></UL></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Theoretical_Considerations"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Theoretical Considerations</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#What_is_a_model.3F"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">What is a model?</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Assumptions"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Assumptions</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Model_Assignment_is_not_Unique"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Model Assignment is not Unique</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Stochastic_Processes"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Stochastic Processes</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Steady_States"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Steady States</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Modeling_global_transcription"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Modeling global transcription</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-8"><A href="#Biosensor_of_chloroform_modelization"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Biosensor of chloroform modelization</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#INTRODUCTION"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">INTRODUCTION</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#REACTIONS"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">REACTIONS</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#PARAMETERS"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">PARAMETERS</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Ordinary_Differential_Equations_.28ODE.27s.29"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Ordinary Differential Equations (ODE's)</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#ASSUMPTIONS_2"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">ASSUMPTIONS</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#MODELING_THE_DYNAMICS_OF_DIFFERENT_INITIAL_CONCENTRATIONS_OF_CHLOROFORM"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">MODELING THE DYNAMICS OF DIFFERENT INITIAL CONCENTRATIONS OF CHLOROFORM</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#STOCHASTIC_MODEL"><SPAN class="tocnumber">2.7</SPAN><SPAN class="toctext">STOCHASTIC MODEL</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#INITIAL_STATE_OF_THE_SYSTEM"><SPAN class="tocnumber">2.8</SPAN><SPAN class="toctext">INITIAL STATE OF THE SYSTEM</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#STEADY_STATES_2"><SPAN class="tocnumber">2.9</SPAN><SPAN class="toctext">STEADY STATES</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#SENSITIVITY_ANALYSIS"><SPAN class="tocnumber">2.10</SPAN><SPAN class="toctext">SENSITIVITY ANALYSIS</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#PARAMETER_SCAN"><SPAN class="tocnumber">2.11</SPAN><SPAN class="toctext">PARAMETER SCAN</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#MODELING_REFERENCES"><SPAN class="tocnumber">2.12</SPAN><SPAN class="toctext">MODELING REFERENCES</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Theoretical_Considerations"><B>Theoretical Considerations</B></SPAN></H2><TABLE class="wikitable" border="0" cellpadding="10" cellspacing="1" style="padding: 0; background-color:black; border: 1px solid black; text-align:center; margin:5px -5px 5px -5px"><TBODY><TR><TH width="10%" align="left" valign="top" style="background:#ECC850; color:black"><H3><SPAN class="mw-headline" id="What_is_a_model.3F">What is a model?</SPAN></H3><P>The answer will differ among communities of researchers. In the broadest sense, a model is an abstract representation of objects or processes that explains features of these objects or processes.
Is important to remember that we are not only going to assign a model, we are going to simulate it as well to see if it explains the real biological process.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:458px;"><DIV class="thumbcaption">Steps to succes modelizing a system</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Assumptions">Assumptions</SPAN></H3><P>Probably the most vital role that the modeler plays in modeling is in exercising his scientific judgment as to what assumptions can be validly made. Obviously an extremely rigorous model that includes every phenomenon down to microscopic detail would be so complex that it would take a long time to develop and might be impossible to solve. A modeling compromise between a rigorous description and getting an answer that is good enough is always required. This has been called “optimum sloppiness”. It involves making as many simplifying assumptions as are optimum usually corresponds to a model which is as complex as the available computing facilities will permit.
The assumptions that are made should be carefully considered and listed. They impose limitations on the model that should always be kept in mind when evaluating its predicted results.
</P><H3><SPAN class="mw-headline" id="Model_Assignment_is_not_Unique">Model Assignment is not Unique</SPAN></H3><P>Biological phenomena can be described in mathematical terms. It is important to note that a certain process can be described in more than one way.
</P><P>•	A biological object can be investigated with different experimental methods.
</P><P>•	Each biological process can be described with different (mathematical) models.
</P><P>•	A mathematical formalism may be applied to different biological instances.
</P><P>•	The choice of a mathematical model or an algorithm to describe a biological object depends on the problem, the purpose, and the intention of the investigator.
</P><P>•	Modeling has to reflect essential properties of the system. Different models may highlight different aspects of the same instance.
</P><P>This ambiguity has the advantage that different ways of studying a problem also provide different insights into the systems. An important disadvantage is that the diversity of modeling approaches makes it very difficult to merge established models.
</P><H3><SPAN class="mw-headline" id="Stochastic_Processes">Stochastic Processes</SPAN></H3><P>The use of differential equations for describing biochemical processes makes certain assumptions that are not always justified. One assumption is that variables can attain continuous values. This is obviously a simplification, since the underlying biological objects, molecules, have discrete nature. As long as molecule numbers are sufficiently large this is no problem, but if the involved molecule numbers are only on the order of dozens or hundreds, the discreteness should be taken into account. Another important assumption of differential equations is that they treat the described process as deterministic. Random fluctuations are normally not part of differential equations. Again, this presumption does not hold for very small systems.
A solution to both limitations is to use stochastic simulation approach that explicitly calculates the change of the number of molecules of participating species during the time course of a chemical reaction.
</P><H3><SPAN class="mw-headline" id="Steady_States">Steady States</SPAN></H3><P>The concept of stationary states is important for the modeling of dynamical systems. Stationary states are determined by the fact that the values of all state variables remain constant in time. The asymptotic behavior of dynamic systems is often stationary.
The consideration of steady states is actually an abstraction that is based on a separation of time scales. While fast processes often reach a quasi-steady state after short transition period, the change of the value of the slow variables is often negligible in the time window of consideration. Thus each steady state can be regarded as a quasi-steady state of a system that is embedded in a larger non-stationary environment. Although the concept of stationary states is a mathematical idealization, it is important in kinetic modeling since it points to typical behavioral modes of investigated system and the respective mathematical problems are frequently easier to solve.
</P><H3><SPAN class="mw-headline" id="Modeling_global_transcription">Modeling global transcription</SPAN></H3><P>The activity of every bacterial gene depends on the concentration of free RNA polymerase, [Rf], and on the promoter-specific Michaelis-Menten parameters, Vmax and Km, that determine the RNA polymerase-promoter interaction during transcript initiation [1]. Generally constitutive promoters are distinguished from regulated promoters; for the first, Vmax and Km remain constant under varying conditions of growth, whereas for the latter, Vmax and/or Km vary.
</P><P>
EFFECT OF THE FREE RNA POLYMERASE ON THE RATE OF TRANSCRIPTION INITIATION AT A GIVEN PROMOTER
</P><P>
All mathematical relationships involving free RNA polymerase derived below refer to steady-state conditions in exponentially growing bacterial cultures and are based on the standard Michaelis-Menten equation:
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:121px;"><DIV class="thumbcaption">Transcription kinetics 1</DIV></DIV></DIV></DIV><P>Here V is the rate of transcript initiation at a given promoter, [Rf] is the concentration of free functional RNA polymerase, Vmax is the maximum rate of initiation at saturation of the promoter with RNA polymerase, and Km is the free RNA polymerase concentration that produces the half maximal rate.
</P><P>
EFFECT OF FREE RNA POLYMERASE ON THE RATE OF TRANSCRIPTION FROM A GIVEN PROMOTER
</P><P>
The rate of transcription, ri, originating from a given promoter Pi with an associated length of transcripts, Li, equals the product of the initiation rate V, given by the equation written before, the transcript length:
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:129px;"><DIV class="thumbcaption">Transcription kinetics 2</DIV></DIV></DIV></DIV><P>If V and Vmax are measured as initiations per minute per promoter and the length of transcripts is measured in nucleotides, then ri is obtained as RNA nucleotides polymerized per min during steady-state expression from the promoter Pi. From this it is clear that ri is proportional to [Rf] only if the free RNA polymerase concentration is much smaller than Km.
</P></TH></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Biosensor_of_chloroform_modelization"><B>Biosensor of chloroform modelization</B></SPAN></H2><TABLE class="wikitable" border="0" cellpadding="10" cellspacing="1" style="padding: 0; background-color:black; border: 1px solid black; text-align:justify; margin:5px -5px 5px -5px"><TBODY><TR><TH valign="top" style="background:#ECC850; color:black"><H3><SPAN class="mw-headline" id="INTRODUCTION">INTRODUCTION</SPAN></H3><P>A mathematical model of the detection and degradation of chloroform by a recombinant Escherichia Coli is proposed. The model includes chloroform transport to the cell, transport to the gas phase, transformation and output (GFP) induction. Our model consist of a system of seventeen nonlinear ordinary differential equations describing the temporal evolution of the key variables involved in the regulation of the pathway mentioned before. Computational simulations of our model are carried out for different external concentration of chloroform. 
</P></TH></TR></TBODY></TABLE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:573px;"><DIV class="thumbcaption">Model diagram built in Simbiology (R)</DIV></DIV></DIV></DIV><TABLE class="wikitable" border="0" cellpadding="10" cellspacing="1" style="padding: 0; background-color:black; border: 1px solid black; text-align:justify; margin:5px -5px 5px -5px"><TBODY><TR><TH valign="top" style="background:#ECC850; color:black"><H3><SPAN class="mw-headline" id="REACTIONS">REACTIONS</SPAN></H3><P>To check which reactions take place in our model and the kinetic laws assumed for all of them, please download the following file. Here you will find a table summing up all that information.
</P><P><A href="/wiki/images/8/80/Taula_reacc.pdf" class="internal" title="Taula reacc.pdf">REACTIONS TABLE</A></P></TH></TR></TBODY></TABLE><TABLE class="wikitable" border="0" cellpadding="10" cellspacing="1" style="padding: 0; background-color:black; border: 1px solid black; text-align:justify; margin:5px -5px 5px -5px"><TBODY><TR><TH valign="top" style="background:#ECC850; color:black"><H3><SPAN class="mw-headline" id="PARAMETERS">PARAMETERS</SPAN></H3><P>The values of the coefficients used and the reference where they were taken are detailed in the next file. 
</P><P>In the absence of reliable experimental data, we chose some of our parameters based on expected reaction times and reactions equilibriums. Some others were taken from the bibliography cited at the end of this work. Although our simulations show that qualitative behavior of the solutions is similar for a large range of parameters. Therefore, we present here results obtained for the particular choice of parameters.
</P><P><A href="/wiki/images/b/b9/Parameter_table.pdf" class="internal" title="Parameter table.pdf">PARAMETERS TABLE</A></P></TH></TR></TBODY></TABLE><TABLE class="wikitable" border="0" cellpadding="10" cellspacing="1" style="padding: 0; background-color:black; border: 1px solid black; text-align:justify; margin:5px -5px 5px -5px"><TBODY><TR><TH valign="top" style="background:#ECC850; color:black"><H3><SPAN class="mw-headline" id="Ordinary_Differential_Equations_.28ODE.27s.29"> Ordinary Differential Equations (ODE's)</SPAN></H3><P><A href="/wiki/images/2/24/Mod_eq.pdf" class="internal" title="Mod eq.pdf">MODEL EQUATIONS</A></P></TH></TR></TBODY></TABLE><TABLE class="wikitable" border="0" cellpadding="10" cellspacing="1" style="padding: 0; background-color:black; border: 1px solid black; text-align:justify; margin:5px -5px 5px -5px"><TBODY><TR><TH valign="top" style="background:#ECC850; color:black"><H3><SPAN class="mw-headline" id="ASSUMPTIONS_2"> ASSUMPTIONS </SPAN></H3><P>-	No physical interaction between cells. This statement means that cells are completely surrounded by liquid, which leads to the assumption that all they have the same interfacial area.
</P><P>-	The diffusion though the cell membrane by chloroform is supposed to occur by Simple Diffusion.
</P><P>-	Time delay and stochastic fluctuations were not studied.
</P><P>-	Constant amount of cells during the incubation time in chloroform. 
</P><P>-	In all models done the basal transcription rate has been assumed negligible.
</P><P>-	Concentration of RNA polymerase is constant. We don’t take into account the fact that a fraction of these RNA polymerases are occupied.
</P><P>-	Other possible regulations are neglected. Not modeling the entire regulatory factors network.
</P><P>-	pH dependence is described by the following equation: (1+10^(a-pH)+10^(b*pH-c)). For more information see the model equations.
</P><P>-	There is no external mass transfer resistance. The overall reaction rate is slow enough to not take into account the film that surrounds the cell membrane, as well as the fact that the cell volume is much larger than the volume of the hypothetical film.
</P><P>-	Three dimensional spatial distributions are neglected. 
</P><P>-	The culture was done in a perfectly stirred batch reactor.
</P><P>-	The value of the mass transfer coefficient from bulk to gas is system dependent, other systems will have different values of the same coefficient.
</P></TH></TR></TBODY></TABLE><TABLE class="wikitable" border="0" cellpadding="10" cellspacing="1" style="padding: 0; background-color:black; border: 1px solid black; text-align:justify; margin:5px -5px 5px -5px"><TBODY><TR><TH valign="top" style="background:#ECC850; color:black"><H3><SPAN class="mw-headline" id="MODELING_THE_DYNAMICS_OF_DIFFERENT_INITIAL_CONCENTRATIONS_OF_CHLOROFORM"> MODELING THE DYNAMICS OF DIFFERENT INITIAL CONCENTRATIONS OF CHLOROFORM      </SPAN></H3><P><A href="/wiki/images/1/12/Different_concentrations.pdf" class="internal" title="Different concentrations.pdf">SYSTEM SENSITIVITY TO CHLOROFORM</A></P></TH></TR></TBODY></TABLE><TABLE class="wikitable" border="0" cellpadding="10" cellspacing="1" style="padding: 0; background-color:black; border: 1px solid black; text-align:justify; margin:5px -5px 5px -5px"><TBODY><TR><TH valign="top" style="background:#ECC850; color:black"><H3><SPAN class="mw-headline" id="STOCHASTIC_MODEL"> STOCHASTIC MODEL </SPAN></H3><P>We performed deterministic and stochastic simulations based on Mass Action Kinetics. The stochastic simulations turned out to be computationally very exhaustive but generated no further significant information compared to the deterministic simulations.
</P></TH></TR></TBODY></TABLE><TABLE class="wikitable" border="0" cellpadding="10" cellspacing="1" style="padding: 0; background-color:black; border: 1px solid black; text-align:justify; margin:5px -5px 5px -5px"><TBODY><TR><TH valign="top" style="background:#ECC850; color:black"><H3><SPAN class="mw-headline" id="INITIAL_STATE_OF_THE_SYSTEM"> INITIAL STATE OF THE SYSTEM </SPAN></H3><P>Initially all concentrations are 0 except for the chloroform in the liquid bulk, DNA coding AMO, DNA coding sigma70 factor, [H+], RNA polymerase and promoter. Among these species only the chloroform may change, the other concentrations are supposed to be constant during the simulation.  
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:578px;"><DIV class="thumbcaption">System's initial state table</DIV></DIV></DIV></DIV><P>The common concentrations of DNA and free RNA polymerase in an Escherichia Coli cell were taken from the bibliography [3].
</P></TH></TR></TBODY></TABLE><TABLE class="wikitable" border="0" cellpadding="10" cellspacing="1" style="padding: 0; background-color:black; border: 1px solid black; text-align:justify; margin:5px -5px 5px -5px"><TBODY><TR><TH valign="top" style="background:#ECC850; color:black"><H3><SPAN class="mw-headline" id="STEADY_STATES_2"> STEADY STATES </SPAN></H3><P>Waiting our system for arrive to a steady state doesn’t make any sense, because the output to study is the concentration of GFP in a given time and not in the steady state. Doubtlessly the GFP concentration arrive to a maximum (where the production rate and degradation rate will reach the same value) and afterwards it decrease till the completely disappearance of this protein. 
</P></TH></TR></TBODY></TABLE><TABLE class="wikitable" border="0" cellpadding="10" cellspacing="1" style="padding: 0; background-color:black; border: 1px solid black; text-align:justify; margin:5px -5px 5px -5px"><TBODY><TR><TH valign="top" style="background:#ECC850; color:black"><H3><SPAN class="mw-headline" id="SENSITIVITY_ANALYSIS"> SENSITIVITY ANALYSIS </SPAN></H3><P><A href="/wiki/images/5/55/Sens_analysis.pdf" class="internal" title="Sens analysis.pdf">SENSITIVITY ANALYSIS FILE</A></P></TH></TR></TBODY></TABLE><TABLE class="wikitable" border="0" cellpadding="10" cellspacing="1" style="padding: 0; background-color:black; border: 1px solid black; text-align:justify; margin:5px -5px 5px -5px"><TBODY><TR><TH valign="top" style="background:#ECC850; color:black"><H3><SPAN class="mw-headline" id="PARAMETER_SCAN"> PARAMETER SCAN </SPAN></H3><P><A href="/wiki/images/5/56/Par_scan.pdf" class="internal" title="Par scan.pdf">PARAMETER SCAN FILE</A></P></TH></TR></TBODY></TABLE><TABLE class="wikitable" border="0" cellpadding="10" cellspacing="1" style="padding: 0; background-color:black; border: 1px solid black; text-align:justify; margin:5px -5px 5px -5px"><TBODY><TR><TH valign="top" style="background:#ECC850; color:black"><H3><SPAN class="mw-headline" id="MODELING_REFERENCES"> MODELING REFERENCES </SPAN></H3><P>[1] T. S. Gardner, C. R. Canter, J.J. Collins, Construction of a genetic toggle switch in Escherichia coli, Nature vol.403, 20 January 2000.
</P><P>
[2] A. Pai, L. You, Optimal tuning of bacterial sensing potential, Molecular Systems Biology 5:286.
</P><P>
[3] H. Bremer, P. Dennis, M. Ehrenberg, Free RNA polymerase and modeling global transcription in Escherichia coli, Biochimie 85 (2003) 597-609.
</P><P>
[4] J.C. Anderson, C. A. Voigt, A.P. Arkin, Environmental signal integration by a modular AND gate, Molecular Systems Biology 3:133
</P><P>
[5] J. Chela-Flores, A. M. Liquori, A. Florio, A kinetic thermodynamic phenomenological approach to genetic expression of heat-shock proteins, J. theor. Biol. (1988) 134, 319-325.
</P><P>
[6] D. A. Ratkowsky, J. Olley, T. Ross, Unifying temperature effects on the growth rate of bacteria and the stability of globular proteins, Journal of Theoretical Biology 233 (2005) 351-362.
</P><P>
[7] Z. Szymańska, M. Zylicz, Mathematical modeling of heat shock protein synthesis in response to temperature change, J. Theor. Biol. (2009), doi:10.1016/j.jtbi.2009.03.021.
</P><P>
[8] T. Takeuchi, T. Hinohara, G. Kurosawa, K. Uchida, A temperature-compensated model for circadian rhythms that can be entrained by temperature cycles, Journal of Theoretical Biology 246 (2007) 195-204.
</P><P>
[9] S. J. Van Dien, J.D. Keasling, A dynamic model of the Escherichia coli Phosphate-Starvation Response, J. theor. Biol (1998), 37-49
</P><P>
[10] A.B. Goryachev, D.J. Toh, T. Lee, Systems analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants, Biosystems, Volume 83, Issues 2-3, February-March 2006, Pages 178-187, doi:10.1016/j.biosystems.2005.04.006
</P><P>
[11] iGEM 2008 KULeuven
</P><P>
[12] C. R. Albano, L. Randers-Eichhorn, Q. Chang, W.E. Bentley, G. Rao, Quantitative measurement of green fluorescent protein expression, Biotechnology techniques, Volume 10 No.12 (December 1996) pp. 953-958.
</P><P>
[13] A. Sacchetti, T. El Sewedy, A.F. Nasr, S. Alberti, Efficient GFP mutations profoundly affect mRNA transcription and translation rates, FEBS Letters 492 (2001) 151-155.
</P><P>
[14] H. Zouridis, V. Hatzimanikatis, A Model for Protein Translation: Polysome Self-Organization Leads to a Maximum Protein Synthesis Rates, Biphysical Journal Volume 92 February 2007 717-730.
</P><P>
[15] <A href="http://logkow.cisti.nrc.ca/logkow/display?OID=17" class="external free" rel="nofollow">http://logkow.cisti.nrc.ca/logkow/display?OID=17</A></P></TH></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:UAB-Barcelona/Modeling">http://2009.igem.org/Team:UAB-Barcelona/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UAB-Barcelona/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UAB-Barcelona/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UAB-Barcelona/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UAB-Barcelona/Modeling&amp;oldid=147714" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>