<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CSU_Fort_Collins_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CSU Fort Collins/Notebook</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer" style="margin-top:-8px"><SPAN style="color:#fff; margin-left:25px"><B>COLORADO STATE</B></SPAN><SPAN style="color:#fff; margin-left:26px"><B>UNIVERSITY iGEM</B></SPAN><UL><LI><A href="https://2015.igem.org/Team:CSU_Fort_Collins">HOME</A></LI><LI><A href="https://2015.igem.org/Team:CSU_Fort_Collins/Team">TEAM</A></LI><LI><A href="#">PROJECT
            					</A></LI><LI><A href="https://2015.igem.org/Team:CSU_Fort_Collins/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:CSU_Fort_Collins/Experiments">Experiments &amp; Protocols</A></LI><LI><A href="https://2015.igem.org/Team:CSU_Fort_Collins/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:CSU_Fort_Collins/Design">Design</A></LI></UL><LI><A href="#">PARTS
            					</A></LI><LI><A href="https://2015.igem.org/Team:CSU_Fort_Collins/Parts">Team Parts</A></LI><LI><A href="https://2015.igem.org/Team:CSU_Fort_Collins/Basic_Part">Basic Parts</A></LI><LI><A href="https://2015.igem.org/Team:CSU_Fort_Collins/Composite_Part">Composite Parts</A></LI><LI><A href="https://2015.igem.org/Team:CSU_Fort_Collins/Notebook">NOTEBOOK</A></LI><LI><A href="https://2015.igem.org/Team:CSU_Fort_Collins/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2015.igem.org/Team:CSU_Fort_Collins/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2015.igem.org/Team:CSU_Fort_Collins/Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:CSU_Fort_Collins/Safety">SAFETY</A></LI><LI><A href="https://2015.igem.org/Team:CSU_Fort_Collins/Measurement">INTERLAB STUDY</A></LI><LI><A href="https://2015.igem.org/Team:CSU_Fort_Collins/Entrepreneurship">ENTREPRENEURSHIP</A></LI></DIV><DIV id="contentContainer"><DIV class="contentBox" style="margin-top:40px"><H5>Contents</H5><A href="#Mar">1 March</A><A href="#Apr">2 April</A><A href="#May">3 May</A><A href="#Jun">4 June</A><A href="#Jul">5 July</A><A href="#Aug">6 August</A><A href="#Sep">7 September</A></DIV><H1 align="center">Notebook</H1><DIV class="Months"><H4>March:</H4></DIV><DIV class="Days"><H5>3/12/15:</H5></DIV><DIV class="Info"><P>Made overnight cultures of lac promoter in pSB1C3 and of KillerRed and trans-zeatin in backbones from synthesis.</P></DIV><DIV class="Days"><H5>3/13/15:</H5></DIV><DIV class="Info"><P>Prepared minipreps from overnight cultures of lac promoter, KillerRed, and trans-zeatin plasmids.</P></DIV><DIV class="Days"><H5>3/14/15:</H5></DIV><DIV class="Info"><P>Completed digestion, ligation, and transformation of <I>E. coli</I> with lac:KillerRed and lac:trans-zeatin.</P></DIV><DIV class="Days"><H5>3/17/15:</H5></DIV><DIV class="Info"><P>Checked colonies with colony PCR protocol. Used annealing temperature of 65℃ and extension time of 2 minutes. Ran gel of results. lac:KillerRed colonies 4, 9 were successful. </P><P>From left to right: Ladder 1, KillerRed colonies 1-9, ladder 2, trans-zeatin colonies 10-16, positive control</P></DIV><DIV class="Days"><H5>3/18/15:</H5></DIV><DIV class="Info"><P>Miniprepped lac:KillerRed colonies from overnight cultures using kit instructions. Nanodropped results. Re-tried construction of lac:trans-zeatin.</P></DIV><DIV class="Days"><H5>3/19/15:</H5></DIV><DIV class="Info"><P>Checked colonies of lac:trans-zeatin. Results successful.</P></DIV><DIV class="Months"><H4>April:</H4></DIV><DIV class="Months"><H4>May:</H4></DIV><DIV class="Days"><H5>5/15/15:</H5></DIV><DIV class="Info"><P>Miniprepped overnight cultures of KillerRed in pSB1C3. Attempted to create RBS:KillerRed using SLIM procedure. Made new LB, LB + Amp<SUP>100</SUP> plates, and LB + Kn<SUP>50</SUP> plates.</P></DIV><DIV class="Days"><H5>5/18/15:</H5></DIV><DIV class="Info"><P>Checked May 15th results for RBS:KillerRed on gel; results unsuccessful. Retried procedure using a different thermocycler.</P></DIV><DIV class="Months"><H4>June:</H4></DIV><DIV class="Days"><H5>6/4/15:</H5></DIV><DIV class="Info"><P>Miniprepped RBS and P_lac.</P></DIV><DIV class="Days"><H5>6/5/15:</H5></DIV><DIV class="Info"><P>Used PCR to amplify FadD. Annealed at 48.5 ℃ , 55 ℃ , and 62.5 ℃ . Extension for 2 minutes.</P></DIV><DIV class="Days"><H5>6/6/15:</H5></DIV><DIV class="Info"><P>Used PCR to amplify FadD. Annealed at 48.5 ℃ , 55 ℃ , and 62.5 ℃ . Extension for 2 minutes.</P></DIV><DIV class="Days"><H5>6/9/15:</H5></DIV><DIV class="Info"><P>Ran gel on PCR product from FadD </P><P> Lane 1: 1 kb ladder Lane 2 &amp; 3: FadD </P></DIV><DIV class="Days"><H5>6/11/15:</H5></DIV><DIV class="Info"><P>Digested Biobrick backbone pSB1C3 (cut sites Spe, Pst), FadD (cut sites Xba, Pst), and P_lac (cut sites Spe, Pst).  Ligated FadD+P_lac and FadD+pSB1C3 at 16 ℃  overnight.</P></DIV><DIV class="Days"><H5>6/12/15:</H5></DIV><DIV class="Info"><P>Miniprepped P_trp and Ribosome binding site (RBS) + KR and set up for sequencing. Transformed P_lac + FadD and FadD into pSB1C3</P></DIV><DIV class="Days"><H5>6/14/15:</H5></DIV><DIV class="Info"><P>Made overnight cultures of (RBS+KR)</P></DIV><DIV class="Days"><H5>6/15/15:</H5></DIV><DIV class="Info"><P>Prepare media and overnight cultures for trans-zeatin growth experiment. Ran PCR check on P_lac + FadD and FadD.</P><P>Gel 1: Lanes 1 &amp; 2 are FadD in pSB1c3 		Gel 2: Lanes 6 &amp; 7 are P_lac +FadD</P></DIV><DIV class="Days"><H5>6/16/15:</H5></DIV><DIV class="Info"><P>Followed procedure for trans-zeatin growth experiment up to a final growth time of 1 day.</P></DIV><DIV class="Days"><H5>6/17/15:</H5></DIV><DIV class="Info"><P>Miniprepped the two successful FadD + pSB1C3 colonies and the two successful P_lac + FadD colonies.</P></DIV><DIV class="Days"><H5>6/18/15:</H5></DIV><DIV class="Info"><P>Ran HPLC procedure for analysis of <A href="”https://2015.igem.org/Team:CSU_Fort_Collins/Results#prod”">fermentative products</A>. Miniprepped P_trp again and re-digested and ligated. RBS+KR came back from sequencing correct so this was ligated with P_trp</P><P><B>InterLab:</B> Reconstitute BioBrick parts BBa_J23106: 2015 Kit Plate 1, Well 22A. Size: 35 bp Cm<SUP>R</SUP> and BBa_I13504: 2015 Kit Plate 4, Well 21J. Size: 875 bp Amp<SUP>R</SUP>Transform J23106 and  BBa_I13504.</P></DIV><DIV class="Days"><H5>6/19/15:</H5></DIV><DIV class="Info"><P>Transformed P_trp+RBS+KR</P></DIV><DIV class="Days"><H5>6/22/15:</H5></DIV><DIV class="Info"><P>Taq checked P_trp+RBS+KR </P><P><B>InterLab:</B> Reconstitute BioBrick parts BBa_J23101: 2015 Kit Plate 1, Well 20K. Size: 35 bp. Cm<SUP>R</SUP>Transform J23101 (Two plates: unconcentrated and concentrated) . Performed colony PCR on BBa_J23106 and BBa_I13504 transformed colinies.</P></DIV><DIV class="Days"><H5>6/23/15:</H5></DIV><DIV class="Info"><P>Ran gel on Taq check from June 22nd and got no visible results.
Received sequencing results for FadD in pSB1C3 and P_lac+FadD and overnight cultures were made to glycerol stock these products.</P><P><B>InterLab:</B> Annotate and restreak. Pick and annotate 16 colonies onto new LB+Cm<SUP>25</SUP> plates. Leave at 37°C overnight. Made overnight cultures of BBa_J23106 and BBa_I13504 transformed colinie </P></DIV><DIV class="Days"><H5>6/24/15:</H5></DIV><DIV class="Info"><P>Made glycerol stocks of FadD in pSB1C3 and P_lac+FadD
Attempted to dephosphorylate P_trp backbone but was unsuccessful</P><P><B>InterLab:</B> Colony PCR (16 rxns total). Run PCR products in 2% agarose gel.</P><P>The expected size of J23101 sequenced is 318bp, so all colonies look correct. We will move forward using colonies A and F because their bands are clear. Minipreped BBa_J23106 and BBa_I13504 overnights and sent them off for sequencing. Digested J23106 and I13504.</P></DIV><DIV class="Days"><H5>6/25/15:</H5></DIV><DIV class="Info"><P><B>InterLab:</B> Make overnight cultures of J23101 colonies A and F for Miniprep. Use 2mL of LB + 2uL of Cm (25mg/mL).  Leave at 37°C overnight.  </P></DIV><DIV class="Days"><H5>6/26/15:</H5></DIV><DIV class="Info">&gt;<P><B>InterLab:</B> Miniprep J23101 A and F using Promega kit and sent them off for sequencing.Decided to move ahead because the sizes from the gel looked correct. Digested  1ug of J23101 F. Decided to start more overnights from A and F J23101 colonies in case digestion doesn’t work. Sequencing confirmed for J23106 and I13504, digested J23106 and I13504 was done incorrectly because no PCR  clean up was used.</P></DIV><DIV class="Days"><H5>6/29/15:</H5></DIV><DIV class="Info"><P><B>InterLab:</B> Miniprep J23101 A and F. J23106 and I13504 overnight cultures made for glycerol stocking and Miniprep.</P></DIV><DIV class="Days"><H5>6/30/15:</H5></DIV><DIV class="Info"><P><B>InterLab:</B> Overnight culture of J23101 F for glycerol stock. Minipreped, digested, and overnight ligation performed on J23106 and I13504.</P></DIV><DIV class="Months"><H4>July:</H4></DIV><DIV class="Days"><H5>7/1/15:</H5></DIV><DIV class="Info"><P><B>InterLab:</B>Digest I13504 with XbaI and PstI.Glycerol stock J23101 F. Transformed J23106 and 13504 and grew on Amp<SUP>25</SUP> plates.</P></DIV><DIV class="Days"><H5>7/2/15:</H5></DIV><DIV class="Info"><P><B>InterLab:</B> Ligation: J23101 and I13504. J23106 and 13504 had no growth. Religated J23106 into 13504 </P></DIV><DIV class="Days"><H5>7/3/15:</H5></DIV><DIV class="Info"><P><B>InterLab:</B> Transformation of ligation reaction didn’t work. Transformed  J23106 into 13504  and grew on Amp<SUP>25</SUP> plates. </P></DIV><DIV class="Days"><H5>7/6/15:</H5></DIV><DIV class="Info"><P>Made overnight cultures of P_lac and P_lac+KR in LB+Chloramphenicol (CM)</P><P><B>InterLab:</B> Redo J23101 and I13504 digestions, ligation and transformation. Performed colony PCR on J23106 and I13504 colonies.</P></DIV><DIV class="Days"><H5>7/7/15:</H5></DIV><DIV class="Info"><P><B>InterLab:</B> Ran J23106 and I13504 colonies PCR results on 1% gel.</P><P> colonies from well 3 and 4 showed improper ligation.</P></DIV><DIV class="Days"><H5>7/8/15:</H5></DIV><DIV class="Info"><P>Performed Quick and Dirty experiment and got a lawn of growth so protocol was rethought.</P><P><B>InterLab:</B> Ligation transformation didn’t work. Redid digestions.Performed colony PCR on 14 new colonies from J23106 and I13504 transformation.</P></DIV><DIV class="Days"><H5>7/9/15:</H5></DIV><DIV class="Info"><P><B>InterLab:</B> Redo J23101 and I13504 ligation and transformation.Ran J23106 and I13504 colonies PCR results on 1% gel.</P><P> gels show that transformation was unsuccessful.</P></DIV><DIV class="Days"><H5>7/10/15:</H5></DIV><DIV class="Info"><P><B>InterLab:</B> Transformation plates had lots of colonies.
Annotate and restreak colonies  Pick and annotate onto new LB+Cm<SUP>25</SUP> plates. Leave in incubator at 37°C overnight.Re-digest J23106 and I13504 and  use heat inactivation instead of PCR clean up kit.</P></DIV><DIV class="Days"><H5>7/13/15:</H5></DIV><DIV class="Info"><P>Performed Quick and Dirty experiment (see google drive) and got a lawn of growth so protocol was rethought.</P><P><B>InterLab:</B> Run PCR products in agarose gel.</P><P> Colony #2 looks like the (only) ligated product. J23117+I13504 is about 1.2kb.Ligated J23106 into I13504 and transformed. Grew transformed cells on Amp<SUP>25</SUP> plates. </P></DIV><DIV class="Days"><H5>7/14/15:</H5></DIV><DIV class="Info"><P>Performed Killer Red Experiment (see drive) and KR was expressed.</P>&gt;<P><B>InterLab:</B> Make 2 overnight cultures of J23101+I13504 Colony 2.
2mL LB + 2uL Cm (25mg/mL). Ran colony PCR on 14 J23106 and I13504 transformed colonies.</P></DIV><DIV class="Days"><H5>7/15/15:</H5></DIV><DIV class="Info"><P>Killer Red Experiment did not work as planned.</P><P><B>InterLab:</B> Miniprep both cultures. Send one for sequencing with VF2 and VR. Ran J23106 andI13504 PCR products on 1% gel.</P><P>Successful</P></DIV><DIV class="Days"><H5>7/15/15:</H5></DIV><DIV class="Info"><P><B>InterLab:</B> Made  J23106 and I13504 overnight cultures for sequencing.</P></DIV><DIV class="Days"><H5>7/17/15:</H5></DIV><DIV class="Info"><P>Created media and prepared flasks for trans-zeatin growth experiment. The experiment needed to be repeated because we suspected that 1 day was not long enough for the trans-zeatin to be created in detectable amounts.</P><P><B>InterLab:</B> Minipreped J23106 and I13504 and sent the off for sequencing.</P></DIV><DIV class="Days"><H5>7/19/15:</H5></DIV><DIV class="Info"><P>Created overnight cultures of strains for trans-zeatin growth experiment.</P></DIV><DIV class="Days"><H5>7/20/15:</H5></DIV><DIV class="Info"><P>Day 1 of trans-zeatin growth experiment.</P>.<P><B>InterLab:</B> Although the prefix and J23101sequences are correct, I13504 and the suffix are unclear. The chromatogram signals for these last two regions were also very weak. I think the sequence is correct because Construct 2 group (Dylan, Dakota, Sam and Aidan) sequenced I13504. I will send for sequencing again.</P></DIV><DIV class="Days"><H5>7/21/15:</H5></DIV><DIV class="Info"><P>Day 2 of trans-zeatin growth experiment.</P></DIV><DIV class="Days"><H5>7/22/15:</H5></DIV><DIV class="Info"><P>Day 3 of trans-zeatin growth experiment. Collected final samples of liquid culture. Cells were accidentally thrown away, so the experiment needed to be repeated.</P></DIV><DIV class="Days"><H5>7/28/15:</H5></DIV><DIV class="Info"><P>Started overnight culture of P_lac+KR for Miniprep.</P></DIV><DIV class="Days"><H5>7/29/15:</H5></DIV><DIV class="Info"><P>Anthony Miniprepped P_lac+KR</P></DIV><DIV class="Days"><H5>7/30/15:</H5></DIV><DIV class="Info"><P>Attempted transform, but used wrong competent cells</P></DIV><DIV class="Days"><H5>7/31/15:</H5></DIV><DIV class="Info"><P>Did P_lac+KR transformation with cells with strong LacI promoter</P></DIV><DIV class="Months" id="Aug"><H4>August:</H4></DIV><DIV class="Days"><H5>8/1/15:</H5></DIV><DIV class="Info"><P>Plate was too overgrown to do Taq polymerase check so transformation will be redone.</P></DIV><DIV class="Days"><H5>8/3/15:</H5></DIV><DIV class="Info"><P>Transformation on P_lac+KR</P><P><B>Interlab:</B> Reconstitute J23177: 2015 kit, plate 1, Well 22K, Size: 35 bp, and transformed</P></DIV><DIV class="Days"><H5>8/4/15:</H5></DIV><DIV class="Info"><P>Performed Taq check using Promega “GO Taq” </P><P><B>Interlab:</B> Plates for J23117 transformed successfully and annotated on a LB+Cm plate. </P></DIV><DIV class="Days"><H5>8/5/15:</H5></DIV><DIV class="Info"><P>Ran gel on PCR and results came back inconclusive as even ladder did not appear under UV light.</P></DIV><DIV class="Days"><H5>8/6/15:</H5></DIV><DIV class="Info"><P><B>Interlab:</B> Colony PCR for all 9 colonies.</P></DIV><DIV class="Days"><H5>8/10/15:</H5></DIV><DIV class="Info"><P><B>Interlab:</B> Take out samples from thermocycler. Start overnight culture of colony #6 for miniprep. Run 1.5% agarose gel.</P></DIV><DIV class="Days"><H5>8/11/15:</H5></DIV><DIV class="Info"><P><B>InterLab:</B> Send J23101+I13504 Colony 2 for sequencing again using VR primer.</P><P><B>Interlab:</B> No growth in overnight. Accidentally used Amp instead of Cm.</P></DIV><DIV class="Days"><H5>8/12/15:</H5></DIV><DIV class="Info"><P><B>InterLab:</B> Miniprep J23117. </P></DIV><DIV class="Days"><H5>8/17/15:</H5></DIV><DIV class="Info"><P>Overnight cultures of P_lac and P_lac+FadD were started.</P><P><B>InterLab:</B> If the suffix sequence is correct, I think the I13504 sequence is most likely correct as well. At the mismatched sequences, there was a high background signal. I think this part is good for use in the Interlab measurements. Digestion J23117, ligation: J23117 and I13504, and transformed. Need to make glycerol stock of J23117. Prepare overnight culture. </P></DIV><DIV class="Days"><H5>8/18/15:</H5></DIV><DIV class="Info"><P>ODs were taken on overnight samples of P_lac and P_lac+FadD. Results were inconclusive as P_lac cells grew to a higher OD and much faster than the P_lac+FadD cells.
KR+P_lac was transformed and an overnight culture was started. Foil was used the whole time to prevent cells from being exposed to light.
Trans-Zeatin+FadD was transformed and an overnight culture was started.</P><P><B>InterLab:</B> J23117+I13504 Colony PCR Check</P></DIV><DIV class="Days"><H5>8/19/15:</H5></DIV><DIV class="Info"><P>Made media, autoclaved flasks, and prepared overnight cultures to run trans-zeatin growth experiment. Both plates from August 18th had an acceptable number of colonies and the KR+P_lac colonies were not red.</P></DIV><DIV class="Days"><H5>8/20/15:</H5></DIV><DIV class="Info"><P>Day 1 of trans-zeatin growth experiment.</P></DIV><DIV class="Days"><H5>8/21/15:</H5></DIV><DIV class="Info"><P>Day 2 of trans-zeatin growth experiment.</P><P><B>InterLab:</B>J23117 Digestion and J23117+I13504 Ligation and Transformation.</P><P> Redo J23117+I13504 ligation. Make overnight of J23117 just in case ligation didn’t work because we don’t have any more plasmid stock.</P></DIV><DIV class="Days"><H5>8/22/15:</H5></DIV><DIV class="Info"><P>Day 3 of trans-zeatin growth experiment.</P><P><B>InterLab:</B> Check sequencing results for J23117+I13504-12  Colony PCR, and run 1.5% agarose gel.</P></DIV><DIV class="Days"><H5>8/23/15:</H5></DIV><DIV class="Info"><P><B>InterLab:</B>Make overnights for J23117+I13504 #6, I20270 #20, I20270 #21 l.</P></DIV><DIV class="Days"><H5>8/24/15:</H5></DIV><DIV class="Info"><P><B>InterLab:</B> Miniprep and send for sequencing..</P></DIV><DIV class="Days"><H5>8/25/15:</H5></DIV><DIV class="Info"><P>Ran PCR with F’i^2 cells and primers for the FadL gene using Phusion DNA polymerase. Reaction was ran in duplicate (2 total) </P></DIV><DIV class="Days"><H5>8/26/15:</H5></DIV><DIV class="Info"><P>Gel was run and results were inconclusive (smear for product 1 and mark around 500bp for product 2)
SIx overnight KR cultures were started which contained 5ml LB, 5ul CM, and 2.5ul IPTG. 3 of these were inoculated with colonies of KR+P_lac from 8-8-15. The other 3 were inoculated with P_lac cells from glycerol stocks.</P></DIV><DIV class="Days"><H5>8/27/15:</H5></DIV><DIV class="Info"><P>Ran PCR on FadL. Gel results came back with band of the right size indicating presence of FadL.
Started overnight of Lipase L1 from Bacillus Stearothermophilusfor sequencing and an overnight of both P_lac and P_lac+FadD.</P><P><B>InterLab:</B> Check sequencing results.</P></DIV><DIV class="Months" id="Sep"><H4>September:</H4></DIV><DIV class="Days"><H5>9/7/15:</H5></DIV><DIV class="Info"><P>Made media and prepared bioreactors for trans-zeatin prototype (bioreactor experiment).</P></DIV><DIV class="Days"><H5>9/8/15:</H5></DIV><DIV class="Info"><P>Transformed P_lac+Lipase L1, P_lac+FadL, P_lac+FadD+FadL, and P_lac+KR cultures.</P></DIV><DIV class="Days"><H5>9/9/15:</H5></DIV><DIV class="Info"><P>PCR check done for all four transformations in replicates of 8. Taq DNA pol was used and the “Go Taq” DNA polymerase protocol from Promega was used.</P></DIV><DIV class="Days"><H5>9/10/15:</H5></DIV><DIV class="Info"><P>Made 50 mL overnight cultures for bioreactor experiment.</P></DIV><DIV class="Days"><H5>9/11/15:</H5></DIV><DIV class="Info"><P>Day 1 of bioreactor experiment. Gels were run on products 1-7 of P_lac+Lipase L1, P_lac_FadD+FadL, and P_lac + FadL.</P></DIV><DIV class="Days"><H5>9/12/15:</H5></DIV><DIV class="Info"><P>Day 2 of bioreactor experiment.</P></DIV><DIV class="Days"><H5>9/13/15:</H5></DIV><DIV class="Info"><P>Day 3 of bioreactor experiment.</P></DIV><DIV class="Days"><H5>9/14/15:</H5></DIV><DIV class="Info"><P>Extracted zeatin from bioreactor and previous experimental results using single-phase extraction procedure. Set up for KillerRed experiment. Created overnights for part submissions.</P></DIV><DIV class="Days"><H5>9/15/15:</H5></DIV><DIV class="Info"><P>Ran HPLC to detect zeatin production. Ran KillerRed experiment. Set up for 0.04% frying oil experiment. Miniprepped plasmids for submission to registry.</P></DIV><DIV class="Days"><H5>9/16/15:</H5></DIV><DIV class="Info"><P>Ran 0.04% frying oil experiment. Prepared sample plate for submission.</P></DIV><DIV class="Days"><H5>9/17/15:</H5></DIV><DIV class="Info"><P>Ran 100% and 50% frying oil experiment.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>