<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Tsinghua-D_Project_html"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Tsinghua-D/Project.html">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Tsinghua-D/Project.html&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Tsinghua-D/Project.html&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Tsinghua-D/Project.html&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Tsinghua-D/Project.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Tsinghua-D/Project.html</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><TABLE width="965" border="0" align="left"><TBODY><TR><TD class="main"><BLOCKQUOTE><BLOCKQUOTE><P align="center" class="STYLE5">A Computer-aided  Temperature-response Regulatory RNA Design</P><P align="center"><STRONG>CHEN Huaiqing1, CHEN Zheqin2, FAN Xiao2,  LI Renkuan2, LI Tianyi1, LI Zhangqinang1, PENG  Liying2, SUN Xiaochen2, WANG Xuan2, WANG  Zhipeng2, XIE Hengyi1, YANG Tianfang2, SHI  Binbin2,</STRONG><STRONG>※</STRONG><STRONG> and DING Hongxu2,</STRONG><STRONG>※</STRONG></P><P><STRONG> </STRONG></P><OL><LI><STRONG>School of Life Science, Tsinghua  University</STRONG></LI><LI><STRONG>iGEM Tsinghua-D team, Tsinghua  University</STRONG></LI></OL><P><STRONG>    ※</STRONG><STRONG>. To whom  correspondence should be addressed, SH</STRONG><STRONG>I Binbin, </STRONG><A href="mailto:ltbyshi@gmail.com"><STRONG>ltbyshi@gmail.com</STRONG></A><STRONG>; DING Hongxu, </STRONG><A href="mailto:poulainding@163.com"><STRONG>poulainding@163.com</STRONG></A><STRONG>.</STRONG><STRONG> </STRONG></P><DIV align="center" id="SOM"><P align="left"><STRONG>              This article can be downloaded from <U><A href="http://www.htys.org/extra/igem2012/doc/article.pdf">HERE</A></U> (PDF).</STRONG></P><P align="left"><STRONG>   Supporting onl</STRONG><STRONG>ine materials can be  downloaded from <U><A href="http://www.htys.org/extra/igem2012/doc/som.pdf">HERE</A></U> (PDF).</STRONG></P><P align="left"><STRONG>   A demo for RNAThermo can be found <U><A href="https://2012.igem.org/Team:Tsinghua-D/Demo.html">HERE</A></U>.</STRONG></P><P align="left"><STRONG>   RNAThermo can be downloaded from <U><A href="http://www.htys.org/extra/igem2012/software/RNAThermo.zip">HERE</A></U>.</STRONG></P></DIV><P align="left"> </P><P><STRONG>Abstract</STRONG><STRONG>  </STRONG>The first software that designs  temperature-sensing regulatory RNA – RNAThermo is presented in this  article.  Parameters were set and several  temperature-sensing regulatory RNA sequences were given by the RNAThermo.  The designed RNAs have been verified both as  to on the structural and functional aspects.   RNAThermo’s potential application in the fermentation industry is  discussed.</P><P><EM>Keywords:</EM> RNA Thermometer, Computer, Design</P><P><STRONG>Introduction</STRONG></P><P>In addition to exploration,  explanation and prediction, the ultimate goal of science is creation.  In the field of life science, enthusiasm  towards creation originates the synthetic biology.  During the last decade, numerous artificial  biological networks had been made by rearranging the exited biological macromolecules.  Recently, creation of the macromolecules  inside biological networks emerges as a hotspot.  Because of structural simplicity and manipulation  convenience, RNA becomes an ideal model for conducting such researches.  In this article, the first software that designs  temperature-sensing regulatory RNA - RNAThermo is reported.  Structural and functional verifications of  the designed RNATs were made.  RNAThermo’s  potential application in the fermentation industry is discussed:  the software provides a new method for  achieving controlled expression of products in the fermentation industry.</P><P><STRONG>RNA Thermometer (RNAT)</STRONG></P><P align="left">Residing in the 5’ untranslated  region (5’UTR) of the entire mRNA, the RNA thermometer (RNAT) is a kind of  temperature-sensing sequence.  As the  environmental temperature changes, the RNAT folds into a series of different secondary  structures.  Several of the structures  block ribosomes’ access to the mRNA thus hinder translation (referred to as  unmelted structure).  Other structures cause  ribosomes’ binding to the mRNA and the initiation of translation (referred to  as melted structure).  By shifting from  the two kinds of structures, the RNAT regulates gene expression in the level of  translation <STRONG>(1)</STRONG>.  </P><P><STRONG>The software RNAThermo designs RNATs that meet the  given parameters</STRONG></P><P align="left">Based on biological and physical  principles, adapting computer algorithms, TNAThermo designs RNATs that meet the  given parameters.  What the user tells  the software are the regulation temperature, the structure (both unmelted structure  and melted structure) of the RNAT and the SD sequence position of the RNAT.  RNAThermo gives the sequences of RNATs that fulfill  these requirements.  </P><P><STRONG>The design of RNATs based on biological principles</STRONG></P>
        The principle behind the RNATs’ response to temperature is fundamental: At  low temperatures, the sequence that binds to ribosomes will be trapped in a  hairpin structure.  Increasing  temperature destabilizes the structure such that the trapped sequence becomes  accessible, allowing translation to be initiated.  The following <STRONG>(Figure 1)</STRONG> is the schematic diagram <STRONG>(2)</STRONG>:
        <P align="center" class="STYLE1"> </P></BLOCKQUOTE></BLOCKQUOTE><UL><LI><BLOCKQUOTE><P align="left"><STRONG>Figure 1.  </STRONG>Structural change of RNAT’s according to the environmental  temperature.  The SD stands for  Shine-Dalgarno sequence, which is recognized and bind by ribosome to initiate translation.  The AUG stands for start codon, from where  the translation begins.</P><P>One example for this mechanism is the  regulation of <EM>E.Coli’</EM>s <EM>rpoH</EM> gene <STRONG>(Figure 2)</STRONG>.  Responding to  environmental temperature change, <EM>rpoH</EM> gene regulates the expression of the heat shock protein.  Low temperature (30℃)  induces a bend in the ribosome-binding site (RBS)-associated downstream box  (DB) region, thereby interfering with ribosome binding.  High temperature (42℃)  disrupts the bend and initiates the process of translation <STRONG>(3)</STRONG>.</P><P align="left"><STRONG>Figure 2.</STRONG>  <STRONG>a. </STRONG>Formation of stem III in  the <EM>rpoH</EM> transcript at low  temperatures (30 °C) induces a bend in the ribosome-binding site (RBS)-associated  downstream box (DB) region, thereby interfering with ribosome binding.  <STRONG>b. </STRONG>A  rise in temperature to 42 °C opens stem III and stem I of the <EM>rpoH</EM> mRNA, liberates the AUG start codon  and DB region, facilitates ribosome binding.</P><P align="left">Inspired by such a mechanism, our  group designed a series of RNATs whose SD sequence will have trap-release  structural change according to the environmental temperature.  The following is the schematic diagram of the  RNATs we designed <STRONG>(Figure 3)</STRONG>:</P><P align="center"><STRONG>Figure 3.  </STRONG>Schematic diagram of the RNATs we designed.  The red box indicates the SD sequence.</P><P> </P><P><STRONG>The design of RNATs based on physical principles</STRONG></P><P align="left">To give RNAT sequences that meet  the given parameters, the central problem is to predict RNATs’ secondary  structure at a given temperature.  Two  methods are adapted according to the computer algorithm’s requirement.  </P><P align="left">One principle adapted in  predicting RNA secondary structure is free energy minimization <STRONG>(4)</STRONG>.  A secondary structure with the least free  energy is regarded to be the optimal solution <STRONG>(5)</STRONG>.</P><P align="left">Another principle adapted here is the  partition function method <STRONG>(6)</STRONG>.   Rather than give one definite structure as the free energy minimization method,  the partition function gives the probability of each secondary structure’s appearance.  In the following equation, Q stands for the partition  function and P (structure) stands for the probability of one specific structure’s  appearance.</P><P><STRONG>The design of RNATs adapting computer algorithms</STRONG></P><P>The software RNAThermo for designing an RNA  thermometer is presented.  RNAThermo is  based on the Vienna RNA Package, which is a program for RNA Secondary Structure  Prediction and Comparison <STRONG>(7)</STRONG>. </P><P>Before introducing RNAThermo, this is the  basic RNAT design problem: given regulation temperature, the structure of the  RNAT (both unmelted structure S1 and melted structure S2) and the SD sequence  position of the RNAT, find an RNAT whose SD sequence will be released from the  hairpin structure above the desired temperature.</P>
            A three-stage algorithm is designed to solve the  problem (<STRONG>workflow is present in Figure 4</STRONG>):  A set of sequences that fold into the S1  structure at low temperature is found.  Sequences  that cannot fold into the S2 structure at the high temperature are screened out.  After these two stages, sequences meet the  structural requirement are obtained.  However,  regulation temperatures of the obtained sequences remain unmeasured.  The third stage is designed to screen RNATs to  meet the temperature requirement.
            <P align="center"><STRONG>Figure 4.  </STRONG>Workflow  of designing an RNA thermometer</P><P><STRONG>Verification of the designed RNATs’ secondary  structure</STRONG></P><P align="left">The first step in verification the <EM>in silico </EM>design is testing the designed structure <EM>in vitro</EM>.  In-line probing method is adapted to measure  the RNATs’ structure <STRONG>(10)</STRONG>.   The results are as shown in <STRONG>Figure 5</STRONG>.</P><P align="center"> </P><P align="left"><STRONG>Figure 5</STRONG>  Result of the in-line probing.  The sequence of the RNAT is 5’-GAAUACAUGUUAAUUAUGCCAUCCAGGCAUACAGAAGAAGUUAAU-3’ and the regulation temperature of the RNAT is 39.5℃.  RNAT loaded in lane 1, 2, 3 was incubated at 46℃ for 20h, 26h and 32h.  RNAT loaded in lane 4, 5, 6 was incubated at 42℃ for 20h, 26h and 32h.  RNAT loaded in lane 7, 8, 9 was incubated at 37℃ for 20h, 26h and 32h.  The red boxes mark sections that melt when temperature rises.</P><P align="left">When temperature rises, sections marked by the red boxes melt thus bands appear.  The results show strong evidence that the designed RNATs can fold into desired secondary structure.</P><P align="left"> </P><P align="left"><STRONG>Verification of the designed  RNATs’ temperature-sensing regulatory function</STRONG></P><P align="left">Then,  rectification of the temperature-response regulatory function <EM>in vivo</EM> should be taken in verification of the <EM>in silico </EM>design.  GFP is adapted as reporter gene in measuring  the RNATs’ temperature-response regulatory function.  The results are shown in <STRONG>Figure 6</STRONG>.</P><P align="center"><STRONG>Figure 6</STRONG>  Schematic diagram of ‘RNAT + GFP’ gene.</P><P align="left">E.Coli were cultured in 30℃ until they reached stationary phrase.  Then the E.Coli were divided into two flasks.  For the experimental group, a 45℃ heat shock was exerted to the E.Coli.  For the control group, the temperature remained 30℃.  Photos were taken after a two-hour adjustment.  Two RNAT sequences were tested and the results are shown in <STRONG>Figure 8</STRONG> and <STRONG>Figure 9</STRONG>.</P><P align="left"><STRONG>Figure 8</STRONG>  Result of the verification of the RNAT’s regulatory function of the RNAT.  The sequence is 5’-ACACGGAUCUACUAGCGUGAAUUUAUCACGGGAAGAAGUCGCCGUAA-3’.  <STRONG>a</STRONG>.  RNAT + GFP at 30℃.  <STRONG>b</STRONG>.  RNAT + GFP at 45℃.  <STRONG>c</STRONG>.  RNAT Only at 30℃.  <STRONG>d</STRONG>.  RNATOnly at 30℃.  <STRONG>e</STRONG>.  Histogram shows average intensity of the GFP’s luminance.</P><P align="left"> </P><P align="left"><STRONG>Figure 9.</STRONG>  Result of the verification of the RNAT’s regulatory function of the RNAT.  The sequence is 5’-GAAUACAUGUUAAUUAUGCCAUCCAGGCAUACAGAAGAAGUUAAT-3’.  <STRONG>a</STRONG>.  RNAT + GFP at 30℃.  <STRONG>b</STRONG>.  RNAT + GFP at 45℃.  <STRONG>c</STRONG>.  RNAT Only at 30℃.  <STRONG>d</STRONG>.  RNAT Only at 30℃.  <STRONG>e</STRONG>.  Histogram shows average intensity of the GFP’s luminance.</P><P align="left">The results show strong evidence that the designed RNATs can function as desired.</P><P align="left"> </P><P align="left"><STRONG>Potential Application in  Fermentation Industry</STRONG></P><P align="left">Computer  aided RNAT design provides a new method for achieving controlled expression of  products in fermentation industry.  Engineered  microorganisms sense a temperature signal and initiate the regulation.<STRONG>(Figure 10)</STRONG>.</P><P align="center"><STRONG>Figure 10.</STRONG>  Schematic diagram of ‘RNAT + Signal Peptide + Lysozyme’ gene.</P><P align="left"> </P><P align="left"><STRONG>Reference</STRONG><STRONG>(1). </STRONG>Jens Kortmann and Franz Narberhaus.  Bacterial RNA thermometers: molecular zippers  and switches.  <EM>NATURE REVIEWS MICROBIOLOGY</EM>, VOLUME 10, 265, APRIL 2012 <STRONG>(2). </STRONG>Birgit Klinkert and Franz  Narberhaus.  Microbial thermosensors.  <EM>Cell.  Mol. Life Sci.</EM>  (2009) 66:2661–2676<STRONG>(3).</STRONG> Miyo Terao Morita, Yoshiyuki  Tanaka, Takashi S. Kodama, Yoshimasa Kyogoku,
              Hideki Yanagi and Takashi Yura.  Translational induction of heat shock transcription  factor sigma32: evidence for a built-in RNA thermosensor..  <EM>Genes Dev. </EM>1999 13: 655-665 <STRONG>(4). </STRONG>David H. Mathews.  Revolutions in RNA Secondary Structure  Prediction.  <EM>J. Mol. Biol.</EM> (2006) 359, 526–532<STRONG>(5).</STRONG> David H Mathews and Douglas H  Turner.  Prediction of RNA secondary  structure by free energy minimization.  <EM>Current Opinion in Structural Biology</EM> 2006, 16:270–278<STRONG>(6).</STRONG> J. S. McCASKlLL.  The Equilibrium Partition Function and Base  Pair Binding Probabilities for RNA Secondary Structure.  <EM>Biopolymers</EM>,  Vol. 29,1105-1119 (1990)<STRONG>(7). </STRONG><A href="http://www.tbi.univie.ac.at/~ivo/RNA/">http://www.tbi.univie.ac.at/~ivo/RNA/</A><STRONG>(8).</STRONG><A href="http://www.tbi.univie.ac.at/~ivo/RNA/man/RNAfold.html">http://www.tbi.univie.ac.at/~ivo/RNA/man/RNAfold.html</A><STRONG>(9).</STRONG> L.  Hofacker, W. Fontan.  Fast folding and  comparison of RNA secondary structures. <EM>Monatshefte  fur Chemie </EM>, 125, 167-188.<STRONG>(10).</STRONG> In-Line Probing Analysis of  Riboswitches.Elizabeth E.  Regulski and  Ronald R. Breaker.  <EM>NATURE PROTOCOL  EXCHANGE </EM><A href="http://www.nature.com/protocolexchange/protocols/1889">http://www.nature.com/protocolexchange/protocols/1889</A></P><P align="left"> </P><P align="left"><STRONG>Acknowledgement</STRONG></P><P align="left">Thank  Prof. CHEN Guoqiang, Prof. SUN Zhirong and Prof. DAI Junbiao for their guidance  in the project.  Thank Prof. Tom Kellie  for his revision of the PPT and the report.   Thanks Dr. YIN Ping and Dr. QU Peng for their help in the RNA  experiments.  Thanks FU Xiaozhi and LI  Teng for their help in the molecular biology experiments.</P><P align="left"> </P></BLOCKQUOTE></LI></UL></TD></TR><TR><TD> </TD></TR><TR><TD> </TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Tsinghua-D/Project.html">http://2012.igem.org/Team:Tsinghua-D/Project.html</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Tsinghua-D/Project.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Tsinghua-D/Project.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Tsinghua-D/Project.html&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Tsinghua-D/Project.html&amp;oldid=234475" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>