<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Grenoble-EMSE-LSU_Project_Biology_Cell_Density"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Grenoble-EMSE-LSU/Project/Biology/Cell_Density">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Grenoble-EMSE-LSU/Project/Biology/Cell_Density&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Grenoble-EMSE-LSU/Project/Biology/Cell_Density&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Grenoble-EMSE-LSU/Project/Biology/Cell_Density&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Grenoble-EMSE-LSU/Project/Biology/Cell_Density" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Grenoble-EMSE-LSU/Project/Biology/Cell Density</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3></DIV></DIV><TITLE>Grenoble-EMSE-LSU, iGEM</TITLE></DIV><DIV id="wrapper"><DIV id="nav"><NAV><UL class="topnav"><LI id="home"><A href="/Team:Grenoble-EMSE-LSU">Home</A></LI><LI id="proj"><A href="/Team:Grenoble-EMSE-LSU/Project">Scientific Project</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Project">Overview</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Project/Biology">Biology</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Project/Biology/Cell_Density">Light-Controlled Cell Density</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Project/Biology/KR">Light-Controlled KillerRed Concentration</A></LI></UL><LI><A href="/Team:Grenoble-EMSE-LSU/Project/Modelling">Modelling</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Project/Modelling/Building">Building the Model</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Project/Modelling/Parameters">Finding Parameters</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Project/Modelling/Density">Density control</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Project/Validation">Validation</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Project/Validation">Experiments</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Project/Validation/Future">Future Experiments</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Project/Instrumentation">Instrumentation</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Project/Instrumentation">TalkE'coli</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Project/Instrumentation/Fluo">Main parts of the device</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Project/Attribution">Attributions</A></LI><LI id="doc"><A href="/Team:Grenoble-EMSE-LSU/Documentation/Biobricks">Documentation</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Documentation/Biobricks">Data Page</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Documentation/Protocols">Protocols</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Documentation/Notebook">Notebook</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Documentation/Notebook/May">May</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Documentation/Notebook/June">June</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Documentation/Notebook/July">July</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Documentation/Notebook/August">August</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Documentation/Notebook/September">September</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Documentation/Safety/KR">Safety</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Documentation/Safety/KR">KillerRed</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Documentation/Safety/Datasheet">Safety Datasheets</A></LI><LI id="HP"><A href="/Team:Grenoble-EMSE-LSU/Human_Practice">Human Practice</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Human_Practice/Promotion">iGEM Promotion</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Human_Practice/Lyon">Lyon Jamboree</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Human_Practice/Game">Play with us !</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Human_Practice/Collaboration">Collaborations</A></LI><LI id="team"><A href="/Team:Grenoble-EMSE-LSU/Team">Team</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Team/Students">Students</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Team/Advisors">Advisors and Instructors</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Team/Sponsors">Sponsors</A></LI></NAV></DIV></DIV><DIV id="back_proj"><DIV class="left" id="scroll"><DIV id="Project"><UL><LI class="title"><A class="title" href="/Team:Grenoble-EMSE-LSU/Project">Scientific Project</A></LI><LI class="sub"><A class="sub" href="/Team:Grenoble-EMSE-LSU/Project">Overview</A></LI><LI class="sub"><A class="sub" href="/Team:Grenoble-EMSE-LSU/Project/Biology">Biology</A></LI><LI class="subsub"><A class="subsub" href="/Team:Grenoble-EMSE-LSU/Project/Biology/Cell_Density">Light-Controlled Cell Density</A></LI><LI class="subsub"><A class="subsub" href="/Team:Grenoble-EMSE-LSU/Project/Biology/KR">Light-Controlled KR Concentration</A></LI><LI class="sub"><A class="sub" href="/Team:Grenoble-EMSE-LSU/Project/Modelling">Modelling</A></LI><LI class="subsub"><A class="subsub" href="/Team:Grenoble-EMSE-LSU/Project/Modelling/Building">Building the Model</A></LI><LI class="subsub"><A class="subsub" href="/Team:Grenoble-EMSE-LSU/Project/Modelling/Parameters">Finding Parameters</A></LI><LI class="subsub"><A class="subsub" href="/Team:Grenoble-EMSE-LSU/Project/Modelling/Density">Density Control</A></LI><LI class="sub"><A class="sub" href="/Team:Grenoble-EMSE-LSU/Project/Validation">Validation</A></LI><LI class="subsub"><A class="subsub" href="/Team:Grenoble-EMSE-LSU/Project/Validation">Experiments</A></LI><LI class="subsub"><A class="subsub" href="/Team:Grenoble-EMSE-LSU/Project/Validation/Future">Future Experiments</A></LI><LI class="sub"><A class="sub" href="/Team:Grenoble-EMSE-LSU/Project/Instrumentation">Instrumentation</A></LI><LI class="subsub"><A class="subsub" href="/Team:Grenoble-EMSE-LSU/Project/Instrumentation">TalkE'.coli</A></LI><LI class="subsub"><A class="subsub" href="/Team:Grenoble-EMSE-LSU/Project/Instrumentation/Fluo">Main parts of the device</A></LI><LI class="sub"><A class="sub" href="/Team:Grenoble-EMSE-LSU/Project/Attribution">Attributions</A></LI></UL></DIV></DIV></DIV><DIV id="wrap"><DIV id="scroll_right"><UL id="right"><LI class="contact"><A id="contact" href="mailto:igem.grenoble.fr@gmail.com">Contact us !</A></LI></UL></DIV></DIV><P><TITLE>Grenoble-EMSE-LSU, iGEM</TITLE></P><DIV class="back" id="proj"><UL class="texte"><LI id="titre"><H1>Light-Controlled Cell Density</H1><H2 id="KillerRed">The KillerRed Protein</H2><P>KillerRed is our project's star protein and the key to our bacterial cell density control system. It represents one light-sensitive element that allows the cells to receive signals from the control device.</P><H3>A red fluorescent Killer</H3><P>KillerRed is a red fluorescent protein <A href="#ref_bio_1">[1]</A>, meaning that by illuminating it with wavelengths from a certain portion of the visible spectrum, it re-emits light in another portion with longer (less energetic) wavelengths. Below is the absorption and emission spectra for the KillerRed protein:</P><P id="legend">Figure 1.The KillerRed protein absorption (left peak) and emission (right peak) spectra
                                        Source: <A href="http://www.evrogen.com/products/KillerRed/KillerRed_Detailed_description.shtml">Detailed KillerRed description from Evrogen</A></P><P>From the emission and absorption spectra, we can determine that the protein absorbs in the green portion of the spectrum with a peak at 585 nm and emits in the red portion of the spectrum with a peak at 610 nm, hence the name &quot;KillerRed&quot;.
			                Emitted light from bacteria is proportional to the amount of protein in the cells. This allows for measuring protein concentration in a cell culture.
			                The most interesting function of the protein however is that it produces ROS (Reactive Oxygen Species) when exposed to light <A href="#ref_bio_1">[1]</A>.
			                ROS are highly unstable and react chemically with many substrates including proteins, lipids and DNA. These reactions are oxidative and damage the affected molecules, making ROS toxic to the cell. With sufficient amounts of ROS, a cell's essential components can be damaged beyond repair, and the cell is killed. Thus illuminating KillerRed-expressing cells with light in the green portion of the visible spectrum kills them, a mechanism that we use to control cell density in a culture.</P><H3 id="KRStructure">Structure</H3><P>In order to understand why KillerRed has its unique properties it is necessary to look at its structure. The protein is remarkably similar to other fluorescent proteins like GFP <EM>(Aequorea victoria&gt;</EM> and dsRed <EM>(Discosoma striata)</EM>, featuring a beta-barrel housing a central alpha helix with the fluorescent chromophore at its middle <A href="#ref_bio_1">[2]</A>. Normally the chromophore is protected from the outside environment by the protein shell, but this isn't the case with KillerRed.</P><P id="legend">Figure 2.A comparison of the 3D structures of monomeric dsRed (left) and dimeric KillerRed (right)
                                         Credits to Carpentier P.,  Violot S.,  Blanchoin L.,  Bourgeois D. for the KillerRed structure, and Strongin D.E.,  Bevis B.,  Khuong N.,  Downing M.E.,  Strack R.L.,  Sundaram K.,  Glick B.S.,  Keenan R.J. for the dsRed structure.
                                         Source: RCSB protein database entries <A href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2WIQ">2WIQ</A> and <A href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2VAD">2VAD</A>.</P><P>KillerRed is a 240 amino acid protein with a 3D structure similar to other fluorescent proteins, with an eleven-strand beta-barrel surrounding an alpha-helix containing the chromophore, source of the protein's fluorescence and photoxicity.
			                 KillerRed has a DsRed-type chromophore formed with residues 67Q (glutamine), 68Y (tyrosine), and 69G (glycine), to make QYG. The corresponding coding sequence can be found in the DNA coding sequence CAGTACGGC.
			                 The interesting properties of the protein are directly related to a unique structural difference among fluorescent proteins, consisting in an open channel linking the chromophore to the environment outside the protein. According to litterature, this is the reason KillerRed is able to produce 1000-fold more reactive oxygen species compared to EGFP which is another ROS-producing fluorescent protein <A href="#ref_bio_1">[3]</A>.</P><H3>Origin</H3><P>KillerRed was originally engineered from the anm2CP anthomedusa chromoprotein by individual amino acid mutations in order to obtain fluorescence and phototoxicity <A href="#ref_bio_1">[1]</A>.</P></LI><LI><H2>Construction of pLac-RBS-KillerRed and pLac-RBS-mCherry</H2><P>The KillerRed gene that we obtained initially was in an eukaryotic plasmid. To express KillerRed in <EM>E. coli</EM> and characterize its effects in response to light stimulations, we decided to clone KillerRed into the commercial prokaryotic expression vector pQE30 (Qiagen). This plasmid contains a pLac promoter and a Shine-Dalgarno Ribosome Binding Site (RBS) that allow gene expression in response to the presence of Isopropyl Î²-D-1-thiogalactopyranoside (IPTG). The pLac-RBS-KillerRed sequence in the pQE30 vector has been submitted as <A href="/Team:Grenoble-EMSE-LSU/Documentation/Biobricks">BBa_K1141001</A>. Furthermore, the pLac-RBS-KillerRed sequence was cloned into the pSB1C3 plasmid, to give the biobrick <A href="/Team:Grenoble-EMSE-LSU/Documentation/Biobricks">BBa_K1141002</A>. Both <A href="/Team:Grenoble-EMSE-LSU/Documentation/Biobricks">BBa_K1141001</A> and <A href="/Team:Grenoble-EMSE-LSU/Documentation/Biobricks">BBa_K1141002</A> were sent to the standard registry part (Fig 3.).
                                        The choice of an inducible promoter is linked to the absence of literature about the effects of KillerRed on cells in low light. Since KillerRed could be cytotoxic and prevent bacteria from growing even at low doses of light, we wanted to be able to control its intracellular concentration. A negative control for KillerRed characterization was also required. We decided to use the fluorescent protein mCherry, which displays the same excitation and emission spectra as KillerRed <A href="#ref_bio_1">[4]</A>, and was shown not to be cytotoxic upon light illumination <A href="#ref_bio_1">[5]</A>. pSB1C3::pLac-RBS-mCherry <A href="/Team:Grenoble-EMSE-LSU/Documentation/Biobricks">BBa_K1141000</A> was thus constructed from the existing biobricks BBa_R0010 and BBa_J06702 (Fig 3.).</P><P id="legend">Figure 3.Biobricks <A href="/Team:Grenoble-EMSE-LSU/Documentation/Biobricks">BBa_K1141002</A><EM>(A)</EM> and <A href="/Team:Grenoble-EMSE-LSU/Documentation/Biobricks">BBa_K1141000</A><EM>(B)</EM> used for characterizing KillerRed. <EM>(C)</EM> Picture showing a pellet of KillerRed-expressing bacteria.</P></LI><LI><H2 id="exp_prot">Experimental Protocol for KillerRed Characterization</H2><H3>Choice of the <EM>E. coli</EM> strain</H3><P>We first decided to characterize KillerRed in BW25113 bacteria, a wild-type (WT) strain derived from <EM>E. coli</EM> K12. Cells were successfully transformed with pQE30::KillerRed and were shown to express the protein in response to IPTG induction. However, results of OD610 monitoring showed that BW25113 cells transformed with pQE30::KillerRed grew really slowly (r = 0.08 h<SUP>-1</SUP>) as compared to WT cells (r = 0.77 h<SUP>-1</SUP>). One hypothesis was that repression of the pLac promoter by the endogeneous LacI repressor was not sufficient for preventing the expression of KillerRed, a protein that could have affected cell growth even at low light levels.
                                        We thus decided to switch to M15 cells (Qiagen), a commercial strain in which the LacI repressor is expressed at high levels from plasmid pREP4. M15 cells did express the KillerRed protein in response to IPTG addition and displayed a faster growth rate than the BW25113 cells transformed with pQE30::KillerRed (Fig 4.). For this reason, M15 cells were chosen to characterize KillerRed.</P><P id="legend">Figure 4.Comparison between the growth of pQE30::KillerRed-containing BW25113 and M15 cells (without IPTG and in the dark). Cells were pre cultured ON in LB medium, supplemented with antibiotics. They were further re suspended in M9 medium, supplemented with antibiotics at OD610 = 0.1. OD610 was subsequently monitored in a 96-well plate for 600 min, using the Tristar LB941 microplate reader (Tristar, Bad Wildbad, Germany) available in the lab. Error bars represent the standard errors of 4 independent measurements.</P><H3>Choice of the Culture Conditions</H3><H4>Experimental setup</H4><P>For characterizing the effects of KillerRed on <EM>E. coli</EM> viability in different light conditions, we decided to focus on 3 kinetic variables: KillerRed fluorescence, OD610 and colony forming units.
                                        First of all, KillerRed fluorescence can be used as an indicator of the level of expression of the protein in our cell culture. Then, optical density (OD610) provides real-time information on the biomass of the system. However, it cannot be used to distinguish living and non-living cells. This is the reason why the number of colonies growing on agar plates was considered to be able to quantify live cells with an independent technique.
                                        Since the spectrophotometer available in the lab was not suitable for illuminating cell samples for extended periods of time, we decided to perform kinetics in 100 mL Erlenmeyer flasks, incubated at 37Â°C, 200 rpm. A LED light source, interfaced to a computer via a microcontroller, was placed into the incubator for illuminating cell samples. A customized software enabled us to tightly modulate the intensity of the light emitted by the source.</P><P id="legend">Figure 5.Overview on the experimental set up used for KillerRed characterization.</P><P>During most of the kinetic experiments, 800 ÂµL of medium were pipetted every 30-60 min. OD610 measurements were performed using a GENESYS 6 spectrophotometer (Thermo Scientific, Waltham, MA, USA) whereas fluorescence was measured with a Tristar LB941 microplate reader, equipped with a 540/630 nm filter set for excitation and emission. Bacterial cell plating on agar plates was also performed at each time point, using serial dilutions.</P><H4>Growth medium</H4><P>M9-glucose medium was privileged in our experiments. As a matter of fact, it displays very low auto fluorescence and contains a single carbon source - glucose â hence providing more repeatable results than Luria-Bertani (LB) medium. pRep4 and pQE30::KillerRed are respectively kanamycin and ampicillin-resistant, and these antibiotics were used at 50 Âµg/ÂµL and 200 Âµg/ÂµL.</P><H4 id="iptg_induction">IPTG induction</H4><P>One important point for our project was to reach a high level of KillerRed expression, without slowing down cellular growth. As a matter of fact, to increase or decrease the amount of viable cells in our culture, we needed to make sure that the bacteria expressing KillerRed could grow in the dark and be killed in response to light stimulations. Now, the more KillerRed is present inside bacteria, the more ROS are produced upon illumination and the more likely the cells are to die. But is bacterial growth affected by high intracellular concentrations of KillerRed? Is there an optimal IPTG concentration to reach high levels of KillerRed without disturbing cell division?
                                        To answer these questions, we decided to induce KillerRed expression with different concentrations of IPTG, while monitoring OD610 and fluorescence. M15 cells transformed with pSB1C3::pLac-RBS-mCherry (<A href="/Team:Grenoble-EMSE-LSU/Documentation/Biobricks">BBa_K1141000</A>) were used as a negative control. To evaluate the amount of KillerRed proteins per living cell, we normalized fluorescence by optical density. Results are shown in Fig 6.</P><P id="legend">Figure 6.OD610 <EM>(A and C)</EM> and Fluorescence/OD610 ratios <EM>(B and D)</EM> as a function of time for KillerRed (left panels) and mCherry (right panels)-expressing <EM>E. coli</EM>. The curves obtained with different concentrations of IPTG are represented in different colors as indicated on the legends at the right.</P><P><STRONG>This experiment confirms that the expression level of KillerRed has an effect on cell growth that isn't observed for the control red fluorescent protein, mCherry. This effect is threshold-based, meaning that if we exceed a certain concentration of IPTG and thus a certain protein expression level, then the cells start growing much more slowly. We observe that at an IPTG concentration of 0.05 mM, there is no effect on cell growth compared to the mCherry control, while protein expression at that concentration is the best out of all the curves. This experiment allows us to define the IPTG concentration used thereafter in KillerRed characterization: 0.05 mM.</STRONG></P></LI><LI><H2 id="KRcharac">KillerRed Characterization</H2><P>The design of our <A href="#exp_prot">experimental protocol</A> enabled us to show that M15 cells expressing KillerRed could grow in a culture medium supplemented with 0.05 mM IPTG in the dark. We thus demonstrated that the amount of living cells within our KillerRed-expressing bacterial culture could be increased through natural cell division. But could we use KillerRed to decrease, or even stabilize the number of viable bacteria of a liquid culture? To answer this question, we decided to characterize the effects of the KillerRed protein on cell viability with respect to different parameters: onset time, duration and intensity of illumination and growth phase of the bacteria.</P><H3>Response to a Constant Illumination</H3><P>Our first goal was to determine whether or not KillerRed-expressing bacterial cells could be killed under illumination with white light, at constant intensity.</P><P>Our proof of concept experiment was performed using our experimental protocol. Cells from an ON pre culture were re suspended in two different Erlenmeyer flasks, filled with 25 mL M9 medium, supplemented with 200 Âµg/ÂµL ampicillin, 50 Âµg/ÂµL kanamycin and 0.05 mM IPTG. The two cell samples were further incubated at 37Â°C, 200 rpm, while monitoring OD610 and fluorescence at 610 nm. One cell sample was illuminated at maximal intensity (P = 0.03 ÂµW/cm<SUP>2</SUP>) from time point 180 min until the end of the kinetic experiment (740 min) whereas the second one was kept in the dark. Cells were plated on agar plates at each time point, using serial dilutions. Results are shown in Fig. 7.</P><P id="legend">Figure 7.Results of OD610 <EM>(A)</EM>, fluorescence at 540/630 nm <EM>(B)</EM> and number of viable cells per ÂµL <EM>C</EM> as a function of time for both the dark (blue) and illuminated (red) cultures.Cell plating was performed every 60-80 min during the kinetic experiment, using serial dilutions. Each agar plate was incubated 12-13 h at 37Â°C prior to count colonies. Only the plates displaying between 30 and 300 visible colonies were considered for cell counting.</P><P>Results show that the amount of living cells of the illuminated sample decreases significantly in response to constant light illumination (Fig 7.C). However, cells are killed significantly slower than in the experiments performed by Bulina <EM>et al.</EM>, where 96% of the <EM>E. coli</EM> bacterial cell population was shown to be killed after 10 min of irradiation [6]. This could be due to the important difference between the light doses applied to the culture from one experiment to the other (0.03 ÂµW/cm2 in our case, against 1 W/cm2 for Bulina et. al). </P><P>In our case, the number of living cells only starts decreasing at time point 300 min, that is to say 120 min after the light source is switched on (Fig 7.C). One hypothesis is that the cells viability is only affected after a sufficient accumulation of ROS-induced intracellular damages. This idea is confirmed by the fact that OD610 and fluorescence of the illuminated sample are still increasing between 180 and 300 min (Fig 7.A and B), meaning that illuminated cells are still able to divide and produce the KillerRed protein. However, a decrease in the growth rate of the illuminated bacteria occurs at time point 180 min, which proves that ROS start being produced and react with intracellular components right from the beginning of the illumination. Fluorescence also tends to stabilize from 180 to 300 min, due to degradation of the KillerRed chromophore by light, known as photobleaching.  This phenomenon progressively counterbalances the increase in fluorescence resulting from the production and maturation of KillerRed proteins between 180 and 300 min. From time point 300 min onwards occurs fluorescence decreases concomitantly with a stabilization of OD610: cells are progressively killed and are not able to divide or produce KillerRed anymore. </P><P><STRONG>These results clearly show that KillerRed-expressing bacteria are killed in response to illumination at constant intensity. We also demonstrated that information on cell viability could be obtained from the study of the changes in both OD610 and fluorescence intensity.</STRONG></P><H3>Comparison with mCherry: Cellular Death is ROS-mediated</H3><P>We demonstrated in the previous section that KillerRed-expressing bacteria could be killed upon white light illumination. However, exposure to white light and incubation outside of the normal temperature range were shown to affect bacterial growth <A href="#ref_bio_1">[7]</A>. Therefore, we decided to perform additional kinetics, using mCherry-expressing bacteria as a negative control. Results of these experiments demonstrated that KillerRed is responsible for cell death in response to white light stimulations.</P><H4>Kinetics</H4><P> mCherry and KillerRed-expressing M15 bacteria were inoculated at OD610 = 0.015 in LB medium, supplemented with antibiotics and 0.05 mM IPTG. Cell samples were subsequently incubated at 37Â°C, 200 rpm. Fluorescence (540/630 nm) and OD610 measurements were performed every 20-100 min for 535 min. Erlenmeyers were kept in the dark for the first 180 min, and were then illuminated (P = 0.03 ÂµW/cm<SUP>2</SUP>) for the rest of the experiment.</P><P id="legend">Figure 8.OD610 <EM>(A)</EM> and fluorescence <EM>(B)</EM> as a function of time of mCherry and KillerRed expressing M15 bacteria. Constant light illumination at maximum intensity was applied from 180 min to 535 min. Temperature was measured in each Erlenmeyer during illumination and was shown to stay constant and equal to 37Â°C. The error bars represent the standard errors of 2 independent measurements.</P><P>Both cell strains display similar growth dynamics in the dark, with growth rates of 1.39h<SUP>-1</SUP> and 0.57 h<SUP>-1</SUP> in early (0-120 min) and late (120-180 min) exponential phase, respectively. At t = 255 min occurs a strong decrease in the growth rate of KillerRed-expressing cells as compared to mCherry-expressing cells. This phenomenon, described in the previous section, is due to the killing of bacteria in response to light stimulations. Since the viability of mCherry-expressing cells is not affected, we conclude that KillerRed is responsible for the decrease in the number of living bacteria when illuminating the sample with white light. Cell death is coupled to a decrease in the amount of fluorescing KillerRed proteins. This phenomenon, known as photobleaching, was shown to be a good indicator of the amount of ROS produced by KillerRed upon light illumination <A href="#ref_bio_1">[8]</A>. Free radicals such as H<SUB>2</SUB>O<SUB>2</SUB> are highly reactive, and cause damage to endogenous proteins and DNA, ultimately leading to cell death. <EM>E. coli</EM> defense mechanisms against oxidative stress, including the superoxide dismutase and catalase enzymes <A href="#ref_bio_1">[9]</A>, seem insufficient in preventing significant and irreversible ROS-mediated damages inside bacteria.</P><H3>Cell Growth Recovery after Stopping Illumination</H3><P>We showed that we could either increase or decrease the amount of living cells within our sample, by modulating the amount of light reaching the culture. Indeed, KillerRed-expressing cells were shown to be able to divide in the dark whereas they were killed upon appropriate illumination. But can a culture, initially illuminated, recover and grow again ? In other words: what is the viability status of cells that survive oxidative stress ?
					To answer this question, we decided to perform a kinetic experiment, in which a square light function (120 min, P = 0.03 ÂµW/cm<SUP>2</SUP>) was applied. Cells were inoculated at OD610 = 0.02 in 25 mL LB medium, supplemented with antibiotics and 0.05 mM IPTG. The first measurement was performed 30 min after IPTG induction.</P><H4>Results</H4><P id="legend">Figure 9.OD610 <EM>(A)</EM> and Fluorescence <EM>(B)</EM> responses of a culture exposed to a 120 min constant light illumination (P = 0.03 ÂµW/cm2). The illuminated sample is represented in red, the dark sample in blue. Error bars represent the standard errors of duplicates.</P><P>As mentioned before, photobleaching of KillerRed is a good indicator of the cytotoxicity induced by this protein upon light stimulation. This phenomenon occurs right after the beginning of the illumination (t = 210 min), the moment at which ROS start being produced and accumulate inside bacteria (Fig 9.B). Fluorescence of the illuminated cell sample still increases during illumination, probably because KillerRed is still being produced by <EM>E. coli</EM>. There is thus a progressive accumulation of intracellular damages caused by oxidative stress during light illumination. A duration of 120 min of illumination seems long enough for the cell population to accumulate sufficient ROS damage. Indeed, at this illumination threshold, a significant decrease in the amount of living cells is measurable, ultimately leading to the stabilization of the OD610 between 365 and 510 min (Fig 9.A). The cells that have survived the light-induced oxidative stress divide again after time point 510 min.
					Thus, it seems possible for an illuminated culture to recover a growth phase with comparable dynamics as for the culture that was kept in the dark (Fig 8.A).</P><H3>Influence of Light Intensity</H3><P>We demonstrated that illumination of a culture of KillerRed-expressing bacteria at maximal intensity (corresponding power density : P = 0.03 ÂµW/cm<SUP>2</SUP>) could trigger an important decrease in the number of viable cells. How about being able to stabilize the number of living bacteria around a steady value? We thus decided to see whether or not we could change the rate at which cells were killed, by modulating the intensity of the illumination.
                                        In these experiments we simply put an additional light source inside the incubator in order to illuminate two cultures at once, at 100% and 50% light intensity respectively. The light sources were switched on 195 minutes after inoculation, until the end of the kinetic experiment (600 min). Another sample of KillerRed-expressing M15 bacteria was kept in the dark, as a negative control. Results of OD610 and fluorescence measurements are shown in Fig 10.</P><P id="legend">Figure 10.OD610 <EM>(A)</EM> and fluorescence (630 nm) <EM>(B)</EM> as a function of time for 3 different bacterial cell samples, under different light conditions. The sample kept in the dark is represented in blue, the ones illuminated at 50 and 100% of the maximal intensity (Imax) in red and green, respectively. The light sources were switched on 195 min after inoculation, until the end of the experiment. Illuminated samples displayed similar fluorescence/OD610 ratios at time point 240 min (4945+/-49 RFU and 4465+/-182 RFU for 0.5*Imax and Imax, respectively). Error bars represent standard errors of duplicates.</P><P>Optical density values of the 3 bacterial cell samples start differing 105 min after the light sources are switched on. ROS-mediated intracellular damages start accumulating inside the bacteria after t = 195 min, leading to a significant change in the number of living cells after t = 300 min (Fig 10.A). For all cultures, OD610 increases after time point 300 min, but at different rates that depend on the intensity of illumination.
                                        According to our expectations, the sample in which the biomass increases the most is the one that was kept in the dark, the condition in which no ROS is produced by KillerRed.
                                        When illuminating the culture at half of the maximal intensity value, OD610 increases more slowly. At this light level, ROS production is likely to induce the killing of some of the bacteria of the population, and to slow down the division of the remaining viable cells. This hypothesis is confirmed by the fact that fluorescence never stops increasing during illumination (Fig 10.B), meaning that some of the cells are still alive and able to produce the KillerRed protein.
                                        At maximal intensity, the increase of the biomass is slowest and tends to stabilize after time point 540 min. In that situation, most of the cells are being killed. Since OD610 represents both living and dead cells and dead cells are not able to divide, a plateau is progressively reached. At time point 420 min, the fluorescence curve tends to flatten (Fig 10.B), a phenomenon that reinforces the hypothesis of the killing of a higher number of cells at maximal intensity than at lower light doses. Since the number of living bacteria decreases, less KillerRed is produced, and the rate at which fluorescence increases begins to be limited due to photobleaching.
                                        We conclude that KillerRed-mediated phototoxic effects are light intensity-dependent. However, one question remained: could we find an optimal light intensity value, allowing the stabilization of the amount of living bacteria in a shaken culture?
                                        Thatâs where our modeling comes in: we built a predictive mathematical model to determine the light dose to apply in order to stabilize the number of viable cells.</P><H3>KillerRed purification and protein characterization</H3><P>Since KillerRed protein allows controlling the living bacterial cell density with light, additional characterization experiments were performed to relate the fluorescence intensities measured in the microplate readers and in our device to absolute KillerRed concentrations. The His6-tag associated to the protein allowed to purify it from bacteria and thus to derive a relationship between fluorescence and protein concentration. As a consequence, we were able to estimate the amount of KillerRed protein per bacterial cell, as a function of the FLU/OD600 ratio.</P><H4>Protein purification</H4><P>His6-KillerRed was purified by Ni-NTA affinity chromatography from a 100 mL bacterial culture (OD600nm = 1.89; Fluo = 28872 RFU), using standard techniques. Briefly, the cells were resuspended in 6 mL lysis buffer (50 mM Na-phosphate, pH7, 300 mM NaCl), supplemented with 1 mg/mL lysozyme, sonicated (6 x 10 sec pulse, 20% amplitude). The cell extract was clarified by centrifugation (13000 x g, 20 min, Eppendorf) and applied to a 1 mL Ni-NTA column equilibrated in lysis buffer. The column was washed with 4 mL of lysis buffer, and specifically bound proteins were eluted with 4 mL elution buffer (50 mM Na-phosphate, 500 mM Imidazole, pH7, 300 mM NaCl). Most of the His6-KillerRed protein was recovered in one 1 mL fraction. The protein concentration was determined by UV-visible spectroscopy, using the known value of the extinction coefficient at 585 nm: 45 000 M-1.cm-1 <A href="#ref_bio_1">[10]</A>.</P><P id="legend">Figure 11.Results of the experiments performed on the purified KillerRed protein. <EM>(A)</EM> Absorbance spectrum. <EM>(B)</EM> SDS-Page gel from the protein electrophoresis.</P><H4>Conversion between fluorescence units and protein concentration</H4><P>The fluorescence of the purified protein solution was used to calibrate fluorescence measurements made on the Tristar microplate reader (200 ÂµL volume). We obtained the following relationship: 48000 Â± 5000 RFU (relative fluorescence units) correspond to a 1 ÂµM KillerRed concentration.</P><H4>Number of KillerRed molecules per cell</H4><P>It is also possible to calculate how many KillerRed molecules are expressed per bacteria, as a function of the RFU/OD600 ratio. From LB-agar cell plating, we determined that 1 OD600 corresponds to 3.109 cells/mL. Since a bacterial suspension exhibiting a RFU of 1000 contains KillerRed at a 0.021 Â± 0.002 ÂµM concentration, amounting to 1.3 1013 molecules/mL, a RFU/OD600 ratio of 1000 therefore signifies that each bacterium contains 4200 KillerRed molecules. ROS generated by a KillerRed fluorophore affects proteins in a radius of 1â6 nm <A href="#ref_bio_1">[11]</A>. One can thus estimate that the maximum volume damaged in the cell by the ROS generated by KillerRed is 4200 * (6 nm)3 = 900000 nm3. This volume is very small, amounting to about 0.15% of the total volume of the bacterium (0.6 Âµm3) <A href="#ref_bio_1">[12]</A>. In our experiments, we observed cell killing at a RFU/OD600 ratio of 4000-8000, indicating that about 1% of the cell volume was exposed to high ROS concentrations.</P></LI><LI><H2 id="ref_bio_1">References</H2><P><STRONG>[1]</STRONG> M.E. Bulina <EM>et al.</EM>, A genetically encoded photosensitizer, <EM>Nature Biotechnology</EM>, January 2006.<STRONG>[2]</STRONG> Sergei Pletnev <EM>et al.</EM>, Structural Basis for Phototoxicity of the Genetically Encoded Photosensitizer KillerRed, <EM>The Journal of Biological Chemistry</EM>, November 2009.<STRONG>[3]</STRONG> Russell B. Vegh <EM>et al.</EM>, Reactive oxygen species in photochemistry of the red ï¬uorescent protein ââKiller Redââ, <EM>Chem. Commun.</EM>, 2011.<STRONG>[4]</STRONG> N C Shaner <EM>et al.</EM>, Improved monomeric red, orange and yellow fluorescent proteins derived from <EM>Discosomasp.</EM> red fluorescent protein, <EM>Nature Biotechnology</EM>, 2004.<STRONG>[5]</STRONG> W Waldeck <EM>et al.</EM>, ROS-mediated killing efficiency with visible light of bacteria carrying different red fluorochrome proteins, <EM>Journal of Photochemistry and Photobiology</EM>, 2012.<STRONG>[6]</STRONG> M.E. Bulina <EM>et al.</EM>, A genetically encoded photosensitizer, <EM>Nature Biotechnology</EM>, January 2006.<STRONG>[7]</STRONG> J D Oliver, Recent findings on the viable but nonculturable state in pathogenic bacteria, <EM>FEMS Microbiol. Rev.</EM>, 2010.<STRONG>[8]</STRONG> M E Bulina <EM>et al.</EM>, Chromophore-assisted light inactivation (CALI) using the phototoxic fluorescent protein KillerRed, <EM>Nature Protocol</EM>, 2006.<STRONG>[9]</STRONG> E Cabiscol <EM>et al.</EM>, Oxidative stress in bacteria and protein damage by reactive oxygen species, <EM>International Microbiology</EM>, 2000.<STRONG>[10]</STRONG> http://www.evrogen.com/products/KillerRed/KillerRed_Detailed_description.shtml<STRONG>[11]</STRONG> F Baumgart <EM>et al.</EM>, Light inactivation of water transport and proteinâprotein interactions of aquaporinâKiller Red chimeras, <EM>J Gen Physiol.</EM>, 2012.<STRONG>[12]</STRONG> Kubitschek HE, Cell volume increase in Escherichia coli after shifts to richer media, <EM>J. Bacteriol.</EM>, 1990.</P></LI><LI id="next"><A href="/Team:Grenoble-EMSE-LSU/Project/Biology/KR">Next Page</A></LI></UL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Grenoble-EMSE-LSU/Project/Biology/Cell_Density">http://2013.igem.org/Team:Grenoble-EMSE-LSU/Project/Biology/Cell_Density</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Grenoble-EMSE-LSU/Project/Biology/Cell_Density" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Grenoble-EMSE-LSU/Project/Biology/Cell_Density" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Grenoble-EMSE-LSU/Project/Biology/Cell_Density&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Grenoble-EMSE-LSU/Project/Biology/Cell_Density&amp;oldid=317213" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>