<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Grenoble-EMSE-LSU_Documentation_Notebook_July"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Grenoble-EMSE-LSU/Documentation/Notebook/July">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Grenoble-EMSE-LSU/Documentation/Notebook/July&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Grenoble-EMSE-LSU/Documentation/Notebook/July&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Grenoble-EMSE-LSU/Documentation/Notebook/July&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Grenoble-EMSE-LSU/Documentation/Notebook/July" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Grenoble-EMSE-LSU/Documentation/Notebook/July</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3></DIV></DIV><TITLE>Grenoble-EMSE-LSU, iGEM</TITLE></DIV><DIV id="wrapper"><DIV id="nav"><NAV><UL class="topnav"><LI id="home"><A href="/Team:Grenoble-EMSE-LSU">Home</A></LI><LI id="proj"><A href="/Team:Grenoble-EMSE-LSU/Project">Scientific Project</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Project">Overview</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Project/Biology">Biology</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Project/Biology/Cell_Density">Light-Controlled Cell Density</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Project/Biology/KR">Light-Controlled KillerRed Concentration</A></LI></UL><LI><A href="/Team:Grenoble-EMSE-LSU/Project/Modelling">Modelling</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Project/Modelling/Building">Building the Model</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Project/Modelling/Parameters">Finding Parameters</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Project/Modelling/Density">Density control</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Project/Validation">Validation</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Project/Validation">Experiments</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Project/Validation/Future">Future Experiments</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Project/Instrumentation">Instrumentation</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Project/Instrumentation">TalkE'coli</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Project/Instrumentation/Fluo">Main parts of the device</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Project/Attribution">Attributions</A></LI><LI id="doc"><A href="/Team:Grenoble-EMSE-LSU/Documentation/Biobricks">Documentation</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Documentation/Biobricks">Data Page</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Documentation/Protocols">Protocols</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Documentation/Notebook">Notebook</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Documentation/Notebook/May">May</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Documentation/Notebook/June">June</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Documentation/Notebook/July">July</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Documentation/Notebook/August">August</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Documentation/Notebook/September">September</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Documentation/Safety/KR">Safety</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Documentation/Safety/KR">KillerRed</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Documentation/Safety/Datasheet">Safety Datasheets</A></LI><LI id="HP"><A href="/Team:Grenoble-EMSE-LSU/Human_Practice">Human Practice</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Human_Practice/Promotion">iGEM Promotion</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Human_Practice/Lyon">Lyon Jamboree</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Human_Practice/Game">Play with us !</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Human_Practice/Collaboration">Collaborations</A></LI><LI id="team"><A href="/Team:Grenoble-EMSE-LSU/Team">Team</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Team/Students">Students</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Team/Advisors">Advisors and Instructors</A></LI><LI><A href="/Team:Grenoble-EMSE-LSU/Team/Sponsors">Sponsors</A></LI></NAV></DIV></DIV><DIV id="back_doc"><DIV class="left" id="scroll"><DIV id="Documentation"><UL><LI class="title"><A class="title" href="/Team:Grenoble-EMSE-LSU/Documentation/Biobricks">Documentation</A></LI><LI class="sub"><A class="sub" href="/Team:Grenoble-EMSE-LSU/Documentation/Biobricks">Data Page</A></LI><LI class="sub"><A class="sub" href="/Team:Grenoble-EMSE-LSU/Documentation/Protocols">Protocols</A></LI><LI class="sub"><A class="sub" href="/Team:Grenoble-EMSE-LSU/Documentation/Notebook">Notebook</A></LI><LI class="subsub"><A class="subsub" href="/Team:Grenoble-EMSE-LSU/Documentation/Notebook/May">May</A></LI><LI class="subsub"><A class="subsub" href="/Team:Grenoble-EMSE-LSU/Documentation/Notebook/June">June</A></LI><LI class="subsub"><A class="subsub" href="/Team:Grenoble-EMSE-LSU/Documentation/Notebook/July">July</A></LI><LI class="subsub"><A class="subsub" href="/Team:Grenoble-EMSE-LSU/Documentation/Notebook/August">August</A></LI><LI class="subsub"><A class="subsub" href="/Team:Grenoble-EMSE-LSU/Documentation/Notebook/September">September</A></LI><LI class="sub"><A class="sub" href="/Team:Grenoble-EMSE-LSU/Documentation/Safety/KR">Safety</A></LI><LI class="subsub"><A class="subsub" href="/Team:Grenoble-EMSE-LSU/Documentation/Safety/KR">KillerRed</A></LI><LI class="subsub"><A class="subsub" href="/Team:Grenoble-EMSE-LSU/Documentation/Safety/Datasheet">Safety Datasheets</A></LI></UL></DIV></DIV></DIV><DIV id="wrap"><DIV id="scroll_right"><UL id="right"><LI class="contact"><A id="contact" href="mailto:igem.grenoble.fr@gmail.com">Contact us !</A></LI></UL></DIV></DIV><P><TITLE>Grenoble-EMSE-LSU, iGEM</TITLE></P><DIV class="back" id="doc"><UL class="texte"><LI id="titre"><H1>July</H1><H2>Week 1 (1-5)</H2><H3>Monday</H3><P>- Present our project to the American Summer School (poster) at Maison Minatec in Grenoble. 
- Find a way to draw the spectrum of the wratten filter #55  for Killer Red and green sensor - measure the absorbance of the filter with all the filters available of the luminometer - by tapping the filter to a 96 well plate - We need to find a better and more efficient solution to draw these spectrum.
- Try to preculture cells with pJT122 (Voigt 1) and pPLPCB(S)(Voigt 3) but didn’t grow, so re-transform BW with pJT122, pJT106B(Voigt 2)  and pPLPCB(S).</P><H3>Tuesday</H3><P>- Extraction of KillerRed from the pBabe-KillerRed eukaryotic vector by PCR, using the following primers, enabling the addition of BamHI and KpnI at the 5’ and 3’ ends of the KillerRed gene, respectively. The restriction sites are highlighted in yellow.

Name: KR_left_pQE
TAATTCCGGATCCATGGGTTCAGAGGGCGGC (Tm=67∞C for 31 bases)
Restriction Site:  (BamH1)

Name: KR_right_pQE
CAGGTACCTTAATCCTCGTCGCTACCGATGG (Tm=66∞C for 31 bases)
Restriction Site: (Kpn1)

Protocol available here.

10 µL of PCR products were subsequently analyzed by gel electrophoresis (1.2 % agarose, 30 min, 135 V). The results of the migration, on which a 700 bp DNA band is clearly visible, demonstrate the success of the amplification of KillerRed.

The rest of the PCR product was purified using the QIAprep® PCR Purification Kit (#28104, Qiagen, Venlo, Netherlands) and titrated using the nanodrop spectrophotometer available in the lab (final concentration : 95.8 ng/µL). Results of migration and titration are available in the figure below.

					    </P><P>- Use luster terminals to avoid the use of crocodile clip and have longer cable for the highpower LED.
- Test the system where the LED is put in the incubateur and measure the temperature of the surface of a Petri dish exposed to the light of the LED that is 7cm above the dish - conclusion: no overwarming - this is a very good point! If the cells expressing KillerRed died, it would not be because of the overwarming of illumination but because of the ROS emission!
- Put 3 insulated clones of each Petri dishes that contain the transformed cells in order to get a glycerol stock and to do a mini prep in 1mL.</P><H3>Wednesday</H3><P>- Do a glycerol stock of 1.5mL for each clone that grows overnight. Use 500µL of the 1mL preculture for a bigger culture (10mL) to do a miniprep and the liquid that left is used on a new Petri dish.
- Do a miniprep but since the Voigt’s plasmid are low copy, I use 4 Eppendorf for 1 columns to get more concentrated DNA.
In the end, there are more than 20ng/µL for all the plasmid. For others colonies, there are more than 100ng/µL - it may be contaminated with chromosomal DNA.</P><H3>Thursday</H3><P>- Transform all the cells that already contain one of the Voigt’s plasmid with one or two of the others Voigt’s plasmid.</P><H3>Friday</H3><P>- The transformations work more or less for all of them but there is only one colony for the transformed cells with 3 plasmids
- Put three clones of each Petri dish in 3 mL of LB medium to make them grow overnight with the appropriate antibiotics</P><P><STRONG>pLac-RBS-sspB</STRONG>Culture and miniprep pLac, RBS, sspB and BackBone BioBricks
Problems with RBS (Elowitz), I could not miniprem it, it seems like it was a problem of size.
Miniprep sspB: 64,1ng/µL, Miniprep pLac: 172,3ng/µL, Miniprep backbone: 226,8ng/µL
Test of restriction enzyme on pLac
</P><H2>Week 2 (8-12)</H2><H3>Monday</H3><P>- Draw a first draft of the Printed Circuit Board (PCB) for the photodiode
- Try to draw the absorbance spectrum using the spectrometer of the Wratten filters ( 55 and 65A), manage to do number 55 but the machine freezes when I measure number 65… it is weird!
- Preculture cell containing Voigt 1&amp;3 and Voigt 1&amp;2&amp;3.</P><H3>Tuesday</H3><P>- Try one more time to draw the spectrum of the filter but it seems that for Wratten 65A the scattering light is too low and freeze the machine.
- Draw the absorbance spectra for Simon and Adrien's experience of the BW containing KillerRed after different exposure time.
- Put cells containing Voigt 1&amp;3 on Petri dish with 0.5x of Cm and Spec and 1x of Xgal, then put it in aluminium paper and cut out a square and illuminate it with green light overnight (test of photosensitive promoter)</P><H3>Wednesday</H3><P>- Take pictures of the Petri dish with the green sensor, the blue color is not equally distributed. It seems that Xgal is not evenly spread and that there are not enough cells.
- Finish the 2 PCB for the project and send it to an electronic teacher.
- Draw the absorbance spectra for Simon and Adrien's experience of the M15 containing KillerRed after different exposure time.</P><H3>Thursday</H3><P>- Think of a way to put GFP under the PcpcG2 promotor in pJT122 instead of LacZ- put restriction site (BglI) at the beginning and at the end of KR ? (because BglI cut LacZ twice at the beginning and at the end of the gene)
- Think of the design of our device</P><H3>Friday</H3><P><STRONG>pLac-RBS-sspB</STRONG> We realised that given that pLac was already used it was better to use pBad.Also as RBS is only 15 base pair long it was decided that it will be added to sspb via PCR.
Culture and miniprep pBAD</P><H2>Week 3 (15-19)</H2><H3>Monday</H3><P>- Put Voigt 1&amp;3 in liquid culture with 4x Xgal ON</P><H3>Tuesday</H3><P>- Put the liquid culture of Voigt 1&amp;3 under microscope - notice blue pigment in bacteria but don’t know if it is because of LacZ or because of the irisation of the light</P><H3>Wednesday</H3><P>- Evolution of the MATLAB model : 
integration of the folding time of KillerRed
-todo in the MATLAB model :
integrate the defence of cells against ROS (sub a quantity of ROS)
integrate the recovery time of the cell (% of cells unable to divide ?)

-stuff I could need to know about a cell to go on, write it below folks :
?
?
?
- Mr. Bucci answers and gives his point of view on the electronic circuit, some problem because of the low intensity delivered by the system and the fact that the transistor BDX33C is a darlington transistor - means that it is composed by two transistor - consequently Vbe(BDX33C) = 1.4V =2*0.7V= 2*Vbe(classical transistor). Need to redo the calculation</P><H3>Thursday</H3><P>- Find a solution for the electronic circuit, since the apply voltage is 12V and the nominal power of the lamp is 6W the intensity is 0.5A. Instead of the potentiometer we will use two resistors in parallel to avoid burning them. And to have 0.7V on these resistors we need to add one diode on the other branch
- draw new PCBs according the the chosen solution
- Think of a way to put KR and RFP under red sensitive promoter - restriction/ligation seems to be easier here</P><H3>Friday</H3><P>- Print the 2 PCBs and drill holes into them and weld the different component on them.</P><P>
- Test the PCB that control the intensity of the light with a function generator and the oscilloscope, it works well.
- Notice that the connector on the photodiode PCB is not welded on the right side</P><P><STRONG>pBad-RBS-sspB</STRONG>new culture and miniprep of sspb : 173 ng/µL </P><H2>Week 4 (22-26)</H2><H3>Monday</H3><P>- Weld the connector on the photodiode PCB on the right side
- Design the different primers for the new constructions (KR and mRFP in pJT106b under the red sensor)
- Test the photodiode PCB with the oscilloscope and Arduino, it works like on the workbench =)
- Test the PCB that control the intensity of the light with Arduino, something goes wrong after 3 sec, Arduino freezes and seems to receive order from the computer whereas the computer doesn’t send any order... the oscilloscope seems to indicate that the noise could be the problem || The solution : put a condensator between PWM and GND ?</P><H3>Tuesday</H3><P>- The solution of the condensator works but since it is not draw on the main PCB, new holes must be drilled on the existing wires. It works with a 100nF condensator, there is no more noise and Arduino stops freezing =).
- Bolt Arduino on the  main PCB.
- Put <EM>E. Coli</EM> with KR and luciferase in pre-culture to test if the photodiode measure low bioluminescence and fluorescence.</P><H3>Wednesday</H3><P>- Draw a graph where there are the spectrum of the LED and all the exitation spectra of GFP, RFP, KillerRed and the spectra transfer functions of Green and Red Sensor in order for us to have an idea of the efficiency of the fluorescence and the activity of the colored sensor according to the LED illumination.</P><H3>Thursday</H3><P>- Try to measure fluorescence with our device and the Wratten filters (deep green for excitation and red for emission). We worked in a big box in order to avoid the photodiode to be disturbed by the room light. The LED illuminate the sample from under and the photodiode is put at the left of the sample in order to decrease the effect of the LED on the light detected by it. With any sample the photodiode measure an irrandiance of 20µW/cm² that means that a lot of scattering light from the source is still caught by the detector. Since the fluorescence is low it might not be detected. With a sample containing an overnight culture that doesn’t express any fluorescence protein. The irradiance rise up to 22µW/cm². The medium increase the scattering light. But with an overnight culture that express a red fluorescence protein, the irrandiance is the same. Either we cannot measure fluorescence or the filter are not efficient enough and the setup allows to much light that makes a noisy measure.</P><H3>Friday</H3><P>- Have a <STRONG>big and long meeting with the advisor</STRONG></P><H3>Monday</H3><P>- Try to measure fluorecence by changing filters or adding blue filter (as a low pass filter) but it does not improve the measurement.</P><H3>Tuesday</H3><P>- Receive a cube from a microscope with two excitation filters -interference filters- (green and blue), two colored filter as emission filters (red and yellow) and two dichroic mirrors. There is also a lens with a 30mm focal. Change a little bit the setup with the new components, it looks like this:

We cut a 50mL tube and paint it in black and wrap the lamp with aluminium to avoid the photodiode to be disturbed by the lamp.</P><TABLE align="center" style="border-collapse:collapse;font-size: 14px;"><TBODY><TR style="border:1px solid black;"><TH>Sample</TH><TH>Frequency</TH><TH>Irradiance</TH></TR><TR style="border:1px solid black;"><TD>Empty tube</TD><TD>150.2Hz</TD><TD>300.4nW/cm²</TD></TR><TR style="border:1px solid black;"><TD>Tube with ON cells without fluorescence protein</TD><TD>150.2Hz</TD><TD>300.4nW/cm²</TD></TR><TR style="border:1px solid black;"><TD>Tube with ON cells with fluorescence protein</TD><TD>180.2Hz</TD><TD>360.4nW/cm²</TD></TR></TBODY></TABLE><P>We measure fluorescence but with a low dynamic, need to improve the position and the optics
</P><H3>Wednesday</H3><P>- Reduce the size of the black flacon to get more power and place the photodiode where the image of the filter is created by the lens because it is where there is the most of light.</P><TABLE align="center" style="border-collapse:collapse;font-size: 14px;"><TBODY><TR style="border:1px solid black;"><TH>Sample</TH><TH>Frequency</TH><TH>Irradiance</TH></TR><TR style="border:1px solid black;"><TD>Empty tube</TD><TD>30 +/- 0.2Hz </TD><TD>60 +/- 0.4nW/cm²</TD></TR><TR style="border:1px solid black;"><TD>Tube with ON cells without fluorescence protein</TD><TD>30.9 +/- 0.2Hz</TD><TD>61.8 +/- 0.4nW/cm²</TD></TR><TR style="border:1px solid black;"><TD>Tube with ON cells with fluorescence protein</TD><TD>36.7 +/- 0.2Hz</TD><TD>73.4 +/- 0.4nW/cm²</TD></TR></TBODY></TABLE><P>
The photodiode detect more fluorescence and the dynamic is doubled with this setup.
Next step: Measure with a different level of expression of fluorescence proteins</P></LI><LI id="next"><A href="/Team:Grenoble-EMSE-LSU/Documentation/Notebook/August">Go to August Notebook</A></LI></UL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Grenoble-EMSE-LSU/Documentation/Notebook/July">http://2013.igem.org/Team:Grenoble-EMSE-LSU/Documentation/Notebook/July</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Grenoble-EMSE-LSU/Documentation/Notebook/July" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Grenoble-EMSE-LSU/Documentation/Notebook/July" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Grenoble-EMSE-LSU/Documentation/Notebook/July&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Grenoble-EMSE-LSU/Documentation/Notebook/July&amp;oldid=316583" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>