"
Page
Discussion
View source
History
teams
Log in
 
Team:Duke/Notebook/Lab Notebook
From 2013.igem.org
Home
Team Duke 2013
The Team
Undergrads
Grad Students
Instructors
Genetic Toggle Switch
The Project
Background Information
Experimental Design
Results:
Library of DNA Parts
Results:
Parts Characterization
Biological Parts Submitted
Mathematical Modeling
Cooperativity and Hill Equation
Thermodynamic Model : Introduction
Thermodynamic Model : Cooperative Repression
Kinetic Model :
Bifurcation and Bistability
Human Practices
From Bench to Biotech
Lab Notebook
Overview
Protocols
Strain List
Safety
E.coli and S.cerevisiae
Xanthomonas Bacteria
Attributions
iGEM 2013
Contents
1Overview
1.1Duke iGEM Team 2013
1.1.1Week of 2/21/13 ~ 3/18/13
1.1.2Week of 5/13/13
1.1.3Week of 5/20/13
1.1.4Week of 5/27/13
1.1.5Week of 6/3/13
1.1.6Week of 6/10/13
1.1.7Week of 6/17/13
1.1.8Week of 6/24/13
1.1.9Week of 7/1/13
1.1.10Week of 7/8/13
1.1.11Week of 7/15/13
1.1.12Week of 7/22/13
1.1.13Week of 7/29/13
1.1.14Week of 8/5/13
1.1.15Week of 8/12/13
1.1.16Week of 8/26/13
1.1.17Week of 9/2/13
1.1.18Week of 9/9/13
Overview
Duke iGEM Team 2013
Matt Baron, Matt Farnitano, Cameron Kim, HyunSoo Kim, Ashley Reid, Janan Zhu
Week of 2/21/13 ~ 3/18/13
Thursday February 21- Matt Baron cultured the 10 NN E. Coli cultures
Friday February 22- Matt Baron successfully mini prepped all NN plasmids (except 3 and 7)
Sunday February 24-Matt Farnitano cultured the 10 NG, pfusA, and pfusA30A E. Coli
Monday February 25-Matt Farnitano successfully mini prepped all 12 above plasmids
Tuesday March 5- Cameron Kim cultured HD module plasmids and LR module plasmids
Wednesday March 6 - HyunSoo Kim successfully mini prepped HD plasmids
Thursday March 7- Cameron Kim cultured NN3 and NN7 E. Coli
Friday March 8- Matt Baron successfully mini prepped plasmids NN3 and NN7
Friday March 22- Ashley Reid miniprepped NI1-NI10 (threw out NI10)
Week of 5/13/13
Monday 5/13
Matt Farnitano built plasmid iGEM.1 with Golden Gate reaction from backbone, ACT1 promoter, YEVenus reporter gene, and CYC1 terminator
MF ran PCR reaction to amplify truncated ACT1 promoter
MF assayed PCR product on agarose gel, with negative results. Product was discarded.
Tuesday 5/14
Charlie Cooper ran PCR reaction for truncated ACT1p
CC built plasmid iGEM.2 with Golden Gate reaction using truncated ACT1 promoter
Wednesday 5/15
MF designed oligos for insert into iGEM.1 with binding sites for TALEss and CRISPR
Thursday 5/16
MF miniprepped plasmids iGEM.1-1 through iGEM.1-4 and iGEM.2-1
MF ran analytical digest of above plasmids with Kpn1 and Nsi1
MF assayed digest product on agarose gel: positive results for iGEM.1-2 and iGEM.1-3 in both enzymes, positive results for iGEM.1-4 and iGEM.2-1 in only Kpn1, negative results for iGEM.1-1.
MF plated E. coli cells with CRISPR plasmids from Church lab
MF ran sequencing reaction for iGEM.1-2 and iGEM.1-3, sent results off for sequencing (Order 83)
Friday 5/17
MF reran analytical digest of plasmids iGEM.1-4 and iGEM.2-1 in Nsi1
MF assayed digest results on agarose gel: positive results for both
MF ran sequencing reaction for plasmids iGEM.1-4 and iGEM.2-1, sent results off for sequencing (Order 88)
MF modified and proofread oligo sequences from Wednesday
Week of 5/20/13
Monday 5/20
MF ran analytical digest of plasmids p667, p668, p669 in Bgl1 and Xba1
MF assayed digest results on agarose gel: positive results for all lanes
MF analyzed sequencing data from iGEM.1-2 and iGEM.1-3, confirmed iGEM.1-2 as a good match, discarded iGEM.1-3
MF, MB and HK designed oligo sequences for 1,3,5 binding sites with 16, 20, 24 binding site lengths. (updated on iGEM Oligos file)
Tuesday 5/21
MF and MB ran 1st cycle of pTAL synthesis to design four TALEss: bs16A, bs16B, bs20A, and bs20B
MF and HK treated product of 1st cycle with Plasmid Safe DNAse
MF and MB transformed plasmid into DH5alpha and plated on LB+Sp+XGal+IPTG
MF designed and ordered 14 oligo sequences for Cas9 plasmid and gRNA plasmid assembly
MF received sequencing data from 5/17, sequences were erroneous/bad quality
Wednesday 5/22
MF and MB assembled ligation of pCC305 and pRS303H plasmids
MB and MF performed restriction digest using Asc1 and Sal1 on pCC305 and pRS303H plasmids to make TALEs plasmid
MB and MF ran colony PCR for pTAL colonies bs16a, bs16b, bs20a, and bs20b and inoculated colonies in SOC media
MB and MF ran sequencing reaction for iGEM.2-2
MB, MF, and HK performed ligation of pCC305 and pRS303H to make TALEs plasmid
MF ran bs16a, bs16b, bs20a, and bs20b on gel and corresponding TAL bands not present on gel
MB, MF, and HK re-ran colony PCR for pTAL colonies bs16a, bs16b, bs20a, and bs20b and inoculated colonies in SOC media
Thursday 5/23
MB, MF, and HK ran bs16a, bs16b, bs20a, and bs20b TAL plasmids on gel and bands not present again. TAL reaction needs to be re-run.
MB, MF, HK re-designed the BS16a, BS16b, BS20a, BS20b oligos to same lengths (inserted non-binding sequences at the end for 1 BS and 3 BS oligos to be the same lengths as the 5 BS ones)
MB, MF, HK checked the oligo sequences if there were any other TF’s that would bind to the BS’s.
MB and MF re-ran 1st cycle of pTAL synthesis to design four TALEss to match the four binding sites: BS16a, BS16b, BS20a, and BS20b
MF and JZ transformed bacteria cells with the product from pTAL synthesis and plated on LB+Spec+Xgal
MB and HK transformed bacteria cells with the ligated plasmid (from pCC305 and pRS303H) and plated on LB+Amp
MF designed gBLOCKs (?) instead of oligos for the binding site, and designed two new binding sites BS16c and BS20c.
Friday 5/24
Transformation with ligated plasmid (from pCC305 and pRS303H) was unsuccessful → have to regrow the two plasmids and run PCR (on Monday)
HK and JZ ran 1st cycle of golden gate reaction for bs16c and bs20c TAL
Transformed E.coli and incubated overnight
MB and HK ran colony PCR for bs16a, bs16b, bs20a, and bs20b TAL plasmids
MB and HK ran gel for bs16a, bs16b, bs20a, and bs20b PCR products. Poor PCR. Re-ran reaction on 5/28.
MF ran 1st reaction with oligos for gRNA and Z4EV
Gel results not good (probably because the melting temperature on the oligo data sheet was different from ours & because the reaction duration was miscalculated)
MF re-ran 1st reaction with oligos for gRNA and Z4EV.
Saturday 5/25
CK ran colony PCR for 16cA, 16cB, 20cA, and 20cB. Identified positive clones for 16cA, 16cB and 20cB (not 20cA). Clones incubated overnight.
Week of 5/27/13
Monday 5/27
MB re-ran colony PCR for 16aA, 16aB, 16bA, 16bB, 20aA, 20aB, 20bA, and 20bB
gel results mixed--potential success for 20Ba, 16Ba, 16Ab, 16Bb
MF did PCR clean up for gRNA and Z4EV--lost both
MF ran PCR reaction with oligos to amplify gRNA and Z4EV
MF lost gRNA PCR product during PCR clean up procedure for unknown reasons
MF did analytical digest of 4 pRS303H-v2 plasmids with Pst1 and Nde1
HK ran pR303H-v2 digest on gel → unclear results (correct for only Pst1)
MF did analytical digest of pRS303H-v2 and old pRS303H with EcoR1
results confirm that plasmid pRS303H-v2 is as expected
HK ran TAL1 reaction for 16Aa, 20Ab, 20Bb, 20Ca (need to add Ligase, BsaI, Ligase buffer tomorrow because the lab is out of BsaI)
MF made a stock of LB+Spec to grow successful/potentially successful TAL colonies
HK labelled 20 tubes and filled with LB+spec for the 20 successful/potentially successful TAL colonies → incubated
Successful: Set 1 20Aa(1,2,3), Set 3 20Ba(1,2), Set 3 16Ba(2), Set 4 20Ba(1), Set 4 16Ba(1,2,3)
Potentially: Set 1 16Bb(1,2,3), Set 3 16Ab(1,3), Set 3 16Bb(1), Set 4 16Ab(1,3), Set 4 16Bb(1,3)
Tuesday 5/28
Successful TAL
MF mini-prepped successful and potentially successful TAL colonies
Sent off for sequencing
Unsuccessful TAL (re-do)
HK and MB finished TAL1 reaction for 16Aa, 20Ab, 20Bb, and 20Ca
HK and JZ transformed the cells
MF and MB plated the cells
Z4EVpr and Backbone for TALE
MB performed restriction digest of pCC305 and pRS303H with Asc1 and Sal1 to prepare for ligation of the two plasmids
MF, HK, JZ ran them on gel
Positive result for pCC303H but negative for pCC305 (it was actually pCC317, and we will re-do tomorrow)
gRNA Backbone and Z4EVpr for CRISPR
MF ran PCR for the gRNA backbone and Z4EVpr
MB ran the products on gel
Bad results. Need to re-do PCR
Wednesday 5/29
Z4EVpr and Backbone for TALE
MB performed restriction digest of pCC317 with Asc1 and Sal1
MF cut gel, but unexpected third band still present
JZ performed restriction digest of pCC305 with Asc1 and Sal1
MF cut gel, unexpected third band present
MB performed gel extraction on pCC317 and p303H
MB set up ligation reaction of pCC317 and p303H pieces to incubate overnight
gRNA Backbone and Z4EVpr for CRISPR
MF ran PCR for gRNA backbone and Z4EVpr
MF, MB ran gel--positive results for Z4EV, negative for gRNA
MF, JZ reran PCR for gRNA backbone
MF ran gel for gRNA backbone--positive results
MF performed PCR cleanup procedure, nanodrop showed no DNA afterwards
Unsuccessful TALE (re-do)
Plates from 5/29 had no colonies--potential problem was mistaking T4 ligase buffer for its enzyme
MB and HK ran TAL1 reaction for 16Aa, 20Ab, 20Bb, and 20Ca
MF ran TAL1 reaction for 2x20Cb (negative result from sequencing), 16Aa, 20Ab, 20Bb, 20Ca
JZ ran plasmid-safe DNase reaction for first TAL set
MF ran plasmid-safe DNase reaction for second TAL set
HK transformed both sets into competent E. coli with heat shock
MF plated transformed cells to culture overnight
Sequencing results for TALEs
MF, MB checked results from 5/29 sequencing
16Ab, 16Ba, 16Bb, 16Ca, 16Cb, 20Aa, 20Ba all successful
20Cb had RVDs in wrong order/extra RVDs, needs to be redone
20Cb redone with the other unsuccessful TALEs (plated and incubated overnight)
Thursday May 30
gRNA backbone and Z4EVpr for CRISPR
MF performed PCR for gRNA backbone and Z4EVpr
Positive results on gel
PCR clean-up successful
Performed SLIC reaction with gRNA and insert oligos
Transformed E.coli with SLIC product and incubated overnight
TALE
MF, MB, and HK performed colony PCR for 16Aa, 20Ab, 20Bb, 20Ca, and 20Cb
Gel result showed no TALEs (twice, possibly bad colony PCR)
Will redo starting from colony picking & PCR (then gel, grow in SOC+store cells in -80)
Z4EVpr and Backbone for TALEss
Need to transform ligation and control into E. coli and plate overnight
Transformed E.coli with ligated (iGEM3 and control) and incubated overnight
G-Blocks
MF and MB redesigned G-Blocks to minimize repeats so IDT can make the G-Blocks
Friday May 31
Z4EVpr and Backbone for TALEss
Both control and insert plates did not have any colonies
Have to redo
TALE
Performed colony PCR (JZ, HK for first set, CK for second set)
Gel results for both sets showed that 4 out of the 5 worked
CK cultured the four and will miniprep them on saturday
Have to redo 20Ab Golden Gate 1st reaction
gRNA backbone and Z4EVpr for CRISPR
Plates have colonies and the control did not have any
Growing colonies in shaker overnight
CK will miniprep them on saturday
gRNA Oligos for 16C and 20C came in
diluted to 100uM and stored
Saturday June 1
TALE
CK miniprepped 16Aa, 20Bb, 20Ca, 20Cb plasmids
gRNA backbone for CRISPR
CK miniprepped pNB669-bs16A, pNB669-bs16B, pNB669-bs20A, pNB669-bs20B
Week of 6/3/13
Monday June 3
TALE
MF ran sequencing reaction for 16Aa, 20Bb, 20Ca, 20Cb plasmids with primers PCR8-F and PCR8-R, sent off for sequencing (order 150)
MB ran 1st cycle to build 20Ab TAL
JZ ran plasmid-safe DNase treatment
MB transformed 20Ab into competent E. coli and plated to incubate overnight
gRNA plasmids
MF ran sequencing reaction for 16A, 16B, 20A, 20B gRNA plasmids (pNB669-) with primer pRSup, sent off for sequencing (order 150)
MF annealed gRNA 16C up and down oligos, and 20C up and down oligos
MF performed SLIC cloning to insert 16C and 20C binding sites into linearized pNB669 backbone
Used annealed gRNA oligos as inserts
First set of reactions added oligos immediately after T4 polymerase
Second set of reactions added oligos later, before icing reaction
MF transformed SLIC products into competent E. coli via heat shock
CC performed ligation of TAL landing pad, with negative results
Tuesday June 4
TALE
MF ran colony PCR on 3 colonies each from two sets of 20Ab
MF, MB ran results on gel and showed successful
Need to culture a colony overnight so we can mini-prep and send off for sequencing
JZ, MF ran second cycle (TAL2) on 16A, 16B, 20B, 16C, 20C TALs
MB, JZ transformed and plated on LB+Amp+XGal+IPTG for 16A, 16B, 20B, 16C, and 20C
gRNA plasmids
Only a few to no white colonies on each plate, but will transform to see, also try Gibson
MF annealed 16C and 20C oligos for use in Gibson assembly
MF ran Gibson assembly reactions for 16C and 20C gRNAs with 1uL and 2uL annealed oligos
MB, JZ transformed and plated on LB+Amp for 16C and 20C gRNAs into chemically competent DH5Alphas
Reporter plasmid
MB ran Golden Gate reaction to create two new reporter plasmids with different promoters--ACTi (truncated) and ACTpr. Each has yEVenus reporter, CYC1 transcriptional terminator, and (pRS305?) backbone.
MB, JZ transformed and plated on LB+Amp both plasmids into chemically competent DH5Alphas
Competent cells
MF, MB, JZ, CC made chemically competent E. coli from DH5Alphas
Wednesday June 5
TALE
Plates for 16A, 16B, 16C, 20B, and 20C have lots of colonies. Hard to grab a white colony amid all the blue colonies
MB ran colony PCR for 16A, 16B, 16C, 20B, and 20C
MF mini-prepped DNA from 20Ab
MF ran sequencing reaction for 20Ab and sent off for sequencing
MB ran 2nd cycle reaction for 20A
MB ran gel for colony PCR, only 20B looked promising (1)
MF, JZ ran colony PCR again with new colonies for 16A, 16B, 16C, 20B, and 20C
MB, JZ ran gel for re-run of colony PCR. Positive for only 16B (1 and 2).
MB cultured successful colonies overnight from colony PCR overnight: 16B and 20B
MF, JZ re-ran 2nd cycle TAL assembly for 16A, 16B, 16C, 20A, 20B, and 20C
MB transformed and plated re-run of 2nd cycle assembly for 16A, 16B, 16C, 20A, 20B, and 20C on LB+Amp+XGal+IPTG
gRNA
MF mini-prepped gRNA DNA for 16C and 20C from SLIC cloning
MF ran sequencing reaction for gRNA 16C and 20C from SLIC cloning and sent off for sequencing
Plates for gRNA DNA for 16C and 20C from Gibson cloning have few colonies.
MB cultured colonies overnight for 16C and 20C
Reporter Plasmid
Plates for reporter plasmid with full ACT and ACTi promoter have lots of white colonies
MB cultured colonies overnight for full ACT and truncated ACTi promoter
Binding Sites
MF redesigned oligos for 5x binding sites
Thursday June 6
TALEs
MF ran colony PCR for 16A, 16C, 20A, 20C TALEs, with three colonies each
MB ran gel for colony PCR for 16A, 16C, 20A, and 20C TALEs
Positive results for 20B only (3rd lane)
Colony from 20B cultured overnight
MB mini-prepped 16B (2 copies) and 20B
MB performed sequencing reaction for TALs 16B and 20B and sent off for sequencing using TAL-F1, TAL-R2, 306 Ura3p, and 306MCSup oligos
MF, JZ ran gel again for 16A colony PCR
Positive results for 1 and 3
Colonies from 16A cultured overnight (both attempts)
MF analyzed sequencing results for orders 150 and 165: TALs 16Aa, 20Bb, 20Ca, 20Cb, and 20Ab all tested positive
MF, JZ ran colony PCR again for 16A, 16C, 20C (3 new colonies each)
MB, MF ran gel--successful only for 16A (already had successes)
gRNA
MB mini-prepped Gibson results for 16C and 20C (two copies each)
MF analyzed sequencing results for orders 150 and 165:
gRNAs 16A, 16B look good
gRNA 20A has single point mutation in promoter
gRNA 20B has single point mutation in structural gRNA
gRNA 16C showed up as original p669 plasmid
gRNA 20C showed up as 16C--need to test where labelling went wrong
Reporter Plasmid
MB mini-prepped reporters with full ACT promoter and truncated ACTi promoter
MB performed sequencing reaction for ACTi and ACTpr reporters with yEVenus seq1, yEVenus seq2, 306 Ura3p, and 306MCSup oligos.
Friday June 7
TALEs
MB successfully mini-prepped DNA for 16A and 20A
MF ran sequencing reaction for DNA for 16A and 20A
MF ran colony PCR for 16C and 20C
MB ran gel for colony PCR for 16C and 20C (positive for 16C)
MB cultured positive colony PCR result for 16C in LB+Amp overnight
MF re-ran colony PCR for 20C
MF ran gel for colony PCR rerun: potential success for lanes 1, 2, and 4
gRNA
MF re-ran sequencing reaction for 16C and 20C from SLIC and 16C and 20C from Gibson cloning
MF re-ran SLIC reaction for gRNA 20A and 20B
MF, MB transformed SLIC reaction into competent E. coli, plated overnight
Saturday June 8
TALEs
MF successfully mini-prepped DNA for TALs 16C and 20C (only 16C, 20C-1 had grown culture)--concentrations 327.6 and 651.4 ng/uL
gRNA
MF cultured 2 colonies each from SLIC 20A and 20B plates in LB+amp
Sunday June 9
gRNA
MB successfully mini-prepped DNA from SLIC 20A and 20B
Yeast
MB plated yeast
MB cultured yeast in shaker overnight
Week of 6/10/13
Monday June 10
TALEs
MB, MF ran sequencing reaction for TALs 16C and 20C and sent off for sequencing
gRNA
MB, MF ran sequencing reaction for gRNA 20A and 20B and sent off for sequencing
Reporter
MB ran PCR of both reporter plasmids (iGEM.4 and iGEM.5) at 1/500 concentration (0, 03. 0.6, and 0.9 uL plasmid) to get portion of reporter that will be homologous with yeast genome in order to transform into yeast. Used oligos Knopf-3pLeu2 and Knopf-5pLeu2 with corresponding sequences TCGTCTACCCTATGAACATATTCCA and ACCATGTAACTTTGCATCCGA. Initial melting temp of 98 C for 1 min then 35x repeat of 98 C for 10 sec, 66 C for 20 sec, then 72 C for 2:15 min. Final extension at 72 C for 5 min then infinite hold at 4 C.
MB, MF ran PCR for iGEM.4 and iGEM.5 on gel and showed that PCR failed.
MF re-ran PCR of both reporter plasmids (iGEM.4 and iGEM.5) at 1/300 concentration (0, 0.3, 0.6, 0.9 uL plasmid) to get portion of reporter that will be homologous with yeast genome in order to transform into yeast. Used oligos Knopf-3pLeu2 and Knopf-5pLeu2 with corresponding sequences TCGTCTACCCTATGAACATATTCCA and ACCATGTAACTTTGCATCCGA. Initial melting temp of 98 C for 1 min then 35x repeat of 98 C for 10 sec, 66 C for 20 sec, then 72 C for 2:15 min. Final extension at 72 C for 5 min then infinite hold at 4 C.
MB, MF ran PCR for iGEM.4 and iGEM.5 on gel and showed that PCR failed. Need to look at sequencing data for iGEM.4 and iGEM.5 to determine what issues may exist
Binding Sites
MF ran PCR for 6 oligos that overlap to form 5x binding sites for 16A and 16C. Used 0.25 uL each oligo in one trial and 0.5 uL each in second trial. Initial melting at 98 C (1 min). Melting at 98 C (20 sec), 16A was annealed at 64 C and 16C was annealed at 66C (20 sec), and extension for both at 72 C (15 sec). Repeat melting, annealing, and extension 34 times.
MF, JZ ran PCR results on gel. Three bands at ~200 (desired) and lower, lower bands were brighter. Will try again with new PCR protocol
MB ran PCR as before with 0.5 uL each oligo, gradient of annealing temperatures from 64 C to 72 C and 15 sec extension at 72 C.
MB, MF ran PCR results on gel. Similar banding as previous run for both 16A and 16C. Going to try to insert into iGEM.4 and iGEM.5 to make 5x binding site reporter plasmid.
MF performed a PCR clean-up on 5x binding sites 16A and 16C
Yeast
Due to failed PCR of reporter, we cannot perform the transformation of the reporter into yeast today.
MB disposed of overnight yeast culture in bleach
Tuesday June 11- **We received dCas9 from Pablo today!!!**
Reporter
MB, MF performed restriction digest of iGEM.4 with SpeI in order to allow for insertion of the 5x binding sites 16A and 16C
MF treated linearized reporter with CIP to prevent recircularization
MF performed PCR cleanup on linearized reporter
MB transformed iGEM.4 into DH5alphas and plated on LB+Amp
MB cultured iGEM.4 in LB+Amp and put in 37 degree shaker overnight
dCas9
MB performed restriction digest of 303H plasmid with AscI and BamHI-HF to prepare for insertion of Z4EVpr and dCas9 gene
Gel results:
MF performed PCR of dCas9 gene to prepare for SLIC reaction with Z4EVpr and dCas9
JZ, MF ran on gel--large smear and vector showed up
JZ performed PCR again with 1/200 diluted dCas9 (1 ng/uL)
MB ran on gel and showed successful PCR of dCas9
Binding sites
MF saved 5 uL each of PCR results for bs5x16A and bs5x16C, ran 45 uL each on 1.3% TBE gel for extraction of top band
Top band hardly visible, gel extraction not continued
MF ran Gibson assembly protocol to assemble linearized reporter with bs5x16A and bs5x16C PCR results, 1 uL each vector and insert
MB set up CPEC cloning to assemble 3 uL linearized reporter with bs5x16A and bs5x16C (6 oligos each--0.25 uL 100uM each oligo). Annealled at 65C, 30 sec extension time, 10 repetitions.
MB transformed Gibson assembly of BS16A and BS16C with iGEM.4 into chemically competent DH5alphas and plated on LB+Amp
MB transformed CPEC assembly of BS16A and BS16C with iGEM.4 into chemically competent DH5alphas and plated on LB+Amp
Wednesday June 12
Reporter
MB successfully mini-prepped iGEM.4 from the cells cultured overnight in LB+Amp
MB ran analytical digest with EcoRI and HindIII to confirm iGEM.4 (expecting bands of approx 3 kb and 5 kb)
MB cultured colony overnight from iGEM.4 plate
dCas9
MF ran another PCR for the dCas9 gene in order to have enough PCR product, with 0.6 uL vector per tube, 4 tubes
MF ran on gel--positive, clean results
MF performed PCR cleanup on dCas9 (only most recent PCR)
MF performed SLIC cloning with pRS303H-Z4EV-dCas9
MF, MB transformed SLIC in E. Coli via heat shock and plated (single plate)
TALEs
Sequencing data revealed an incorrect sequence for TALE 20C
MF ran colony PCR for TALE 20C
MB ran on gel-negative results and decided plate was of poor quality
MF ran new second cycle reaction for TALE 20C
Two with 20Cb-1 and two with 20Cb-2
MF, MB transformed second cycle reaction via heat shock and plated (4 plates)
Binding Sites
Plates from CPEC assembly of binding sites in the reporter have only one colony
Plates from Gibson assembly of binding sites in the reporter have a few colonies
MB cultured colonies overnight in LB+Amp to mini-prep tomorrow--2 colonies each of Gibson for 5x16A and 5x16C, one CPEC colony for 5x16C
gRNA
MF performed sequencing reaction for gRNA 20B (1 and 2) and 20C (1b and 2) and sent off for sequencing (Order 208)
Thursday June 13
TALEs
MB ran colony PCR for two colonies from each of the four 20C plates
MB ran gel for colony PCR and there were two successful results, from two separate plates 3 and 4 (which are both with 20Cb-2). Both were cultured overnight in LB+Amp
MF ran another colony PCR with two colonies each from plates 1 and 2
MF ran on gel--negative results
JZ ran restriction digest of TALEs plasmids with Sap1 and Nde1 in Cutsmart buffer for yeast transformation
Binding Sites
MF stored glycerol stocks and mini-prepped DNA from the CPEC assembly of 5x16c and the Gibson assemblies of 5x16A and 5x16C
MF ran an analytical restriction digest of mini-prepped DNA from CPEC assembly of 5x16C and the Gibson assemblies of 5x16A and 5x16C with MfeI and BamHI to confirm insertion of binding sites in iGEM.4
All four Gibsons looked potentially good--hard to tell if binding sites are the correct length; CPEC was not good
MF performed sequencing reaction of Gibson assemblies of 5x16A and 5x16C and sent off for sequencing
Reporter
MF stored a glycerol stock and mini-prepped DNA for iGEM.4
MB ran an analytical restriction digest of mini-prepped iGEM.4 with EcoRI-HF and HindIII-HF to confirm plasmid
JZ ran restriction digest of iGEM.4 with Afe1 and Nhe1-HF in Cutsmart buffer for yeast transformation.
dCas9
MB cultured two colonies separately from the SLIC cloning of dCas9 gene with 303H backbone and Z4EVpr
Friday June 14    dCas9
MB saved glycerol stocks and mini-prepped DNA from the SLIC cloning of 303H, Z4EV, and dCas9
Need to perform analytical restriction digest and sequencing reaction but still need plasmid map from Pablo and the correct oligos for sequencing reaction
TALEs
MB saved glycerol stocks and mini-prepped DNA for TAL 20C (20C-3 and 20C-4)
MB ran sequencing reaction for 20C-3 and 20C-4 using oligos TAL F-1, TAL R-2, 306 MCS-up, and 306 ura3p
MF ran TALs 16A, 16B, 16C, 20A, 20B, and 20C on a gel after they had been linearized with SapI and NdeI in order to prepare them for transformation in the yeast genome (should be three bands: ones at 1.3 kb, 2.3 kb, and 6.5 kb). Digest appeared incomplete.
MF performed restriction digest of TALs 16A, 16B, 16C, 20A, 20B, and 20C-3 using SapI and NdeI in order to prepare them for transformation in the yeast genome (should be three bands: ones at 1.3 kb, 2.3 kb, and 6.5 kb)
MF put yesterday’s restriction digests back in the 37C bath with today’s digests to obtain complete digests
20A and 16A TALEs show unexpected band lengths, other 4 are good and ready for yeast integration.
gRNA
MF performed restriction digest of p669 linearized gRNA backbone with DpnI to cut up any remaining full gRNA plasmid with old binding site insert. We noticed that sequencing data for gRNA 20C kept having the old insert even though we thought we had run a PCR that removed that sequence.
Reporter
MF ran iGEM.4 on a gel after it had been linearized with NheI and AfeI in order to prepare it for transformation in the yeast genome (should be two bands: one at 3.5 kb and the other at 4 kb but looks like one big band covering the 3.5-4 range)
MF performed restriction digest  of iGEM.4 using NheI to confirm size of plasmid
Plasmid size confirmed, so first digest with Nhe1 and Afe1 can be used for yeast integration
Week of 6/17/13
Monday June 17
G-blocks arrived!/Binding Site Insertion in Reporter
MF added 200uL water to each tube to obtain 1 ng/uL concentration
MB performed Gibson assembly for 1x-(16A, 16B, 16C, 20A, 20B, and 20C) and 3x-(16A, 16B, 16C, 20A, 20B, and 20C) with 0.5 uL linearized iGEM.4 (0.015 pmol) and 7.2 uL G-block (0.045 pmol)
MF transformed the Gibson assemblies of 1x-(16A, 16B, 16C, 20A, 20B, and 20C) and 3x-(16A, 16B, 16C, 20A, 20B, and 20C) with linearized iGEM.4
gRNA
MF performed SLIC for gRNA 20C with p669 digested with DpnI (3 different insert concentrations)
MF transformed the 3 different insert concentrations of gRNA 20C
dCas9
MF designed and ordered oligos in order to perform PCR of dCas9 in order to insert into pCC305
TALEs
MF transformed remaining DNA for 16A, 16B, 16C, 20A, and 20B in order to make more and save glycerol stocks
Yeast
MF cultured yeast (DBY12397) overnight in order to being yeast transformations tomorrow with linearized iGEM.4 and TALEs
Tuesday June 19
TALEs
Plates for 16A, 16B, 16C, 20A, and 20B have a lawn of bacteria.
MF ran colony PCR on two smears from each plate
MB ran results on gel and all colonies appear successful
JZ, MF cultured 1 colony per TALE from SOC tubes in order to mini-prep and save glycerol stocks tomorrow
gRNA
Plates for gRNA 20C appear to have few to no colonies. Going to try Gibson cloning today
MF annealed 3 uL each of 20C oligos in order to Gibson clone
MB performed Gibson assembly of gRNA 20C with unprepped vector--3 copies
MF performed Gibson assembly of gRNA 20C with PCR-prepped vector--3 copies, annealled oligo volumes 0.25uL, 0.5uL, 1uL, plus no oligo control
MF transformed 7 Gibson reactions into E. coli via heat shock
MB, JZ plated transformations on LB+amp
JZ, MF cultured 2 colonies from 20C SLIC plate overnight to mini-prep
Binding Sites
Plates have lots of colonies for 1x-(16A, 16B, 16C, 20A, 20B, and 20C) and 3x-(16A, 16B, 16C, 20A, 20B, and 20C) with linearized iGEM.4.
JZ, MB cultured colonies overnight in order to mini-prep DNA and send off for sequencing
MF performed SLIC reaction with un-PCR-ed oligos for 5x16A and 5x16C binding sites in linearized iGEM.4 vector. One tube per binding site plus one control
MF transformed 3 SLIC reaction into E. coli via heat shock
MB, JZ plated transformations on LB+amp
Yeast
MF, MB transformed 4 plasmids (TALEs 16B, 16C, 20B, 20C) into yeast to make strains 21, 22, 24, and 25 plus a control (strain 0).
MF, MB, JZ plated 5 transformations on YPD+G418
Wednesday June 19
gRNA
Don’t appear to be any colonies on the gRNA 20C plates (1-6). Need to reevaluate that binding site to see if possible
MB, MF successfully saved glycerol stocks and mini-prepped DNA from gRNA 20C (1 and 2)
MB, JZ performed sequencing reaction for gRNA 20C-1 and 20C-2 using oligos pRS-up and sent off for sequencing
TALEs
MB, MF successfully saved glycerol stocks and mini-prepped DNA from TALEs 16A, 16B, 16C, 20A, and 20B
MB, MF performed an analytical digest of TALEs 16A, 20A, pCC305, 16Aa, 16Ab, 20Aa, and 20Ab with XbaI and MfeI
MF ran digested TALEs 16A and 20A on gel and they are the correct size pieces
Reporter with Binding Sites
MB, MF successfully saved glycerol stocks and mini-prepped DNA from iGEM.4 with BS 1x-(16A, 16B, 16C, 20A, 20B, and 20C) and 3x-(16A, 16B, 16C, 20A, 20B, and 20C). 2 separate stocks and mini-preps for each BS.
MB, MF performed an analytical digest of both copies of BS 1x-(16A, 16B, 16C, 20A, 20B, and 20C) and 3x-(16A, 16B, 16C, 20A, 20B, and 20C) in iGEM.4 with BamHI and MfeI
MF ran digest reporters with binding sites on a gel and it appears as if all g-block insertions were successful
MB, JZ performed sequencing reaction for BS 1x-(16A, 16B, 16C, 20A, 20B, and 20C) and 3x-(16A, 16B, 16C, 20A, 20B, and 20C) in iGEM.4 using oligos yEVenus-seq 1 and 303-MCSup and sent off for sequencing.
MB cultured two colonies from BS 5x-16A and one from BS 5x-16C overnight in order to mini-prep
Yeast
Plates from transformation with TALEs 16B, 16C, 20B, and 20C are growing for one more day
MB cultured a colony of DBY12397 in YPD overnight in order to transform TALEs 16A and 20A tomorrow
Thursday June 20
TALEs
MF ran restriction digest of TALEs 16A and 20A with SapI and NdeI in order to prepare them for transformation into yeast.
Gel results after 1 hr indicate unknown problem with TALEs 16A and 20A (extra bands)
MF ran digest for 3.5 hrs total to check for incomplete digestion--extra bands still present
MF ran analytical digest of TALEs 16A and 16B in NEBuffers 2 and 3 with Thermo’s AdeI (DraIII)--16A had extra band, 16B looked as expected, both buffers showed equal results
JZ, MB transformed TALEs 16A and 20A in order to colony purify
JZ, MB transformed pCC305 to save permanent stock
Yeast
Unable to transform TALEs 16A and 20A into DBY12397 because of unexpected results of digest with SapI and NdeI (5 bands instead of 3).
Unable to pick transformants from plates for TALEs 16B, 16C, 20B, and 20C. Need to let them grow another day
Reporter with Binding Sites
MB mini-prepped iGEM.4 + BS 5x-16A and two copies of iGEM.4 + BS 5x-16C and saved glycerol stocks
MB performed analytical digest of BX 5x-16A and Bs 5x-16C with BamHI and MfeI and ran on gel. (Bands indicated successful insertion of binding sites)
MB performed sequencing reaction for 5x16A and 5x16C with yEVenus-seq1 and 303MCSup, sent off for sequencing
dCas9
MB performed PCR using new oligos for pCC305 backbone and dCas9 gene.
pCC305 anneal at 69C, extension time 3:30
dCas9 anneal at 69C, extension time 2:00
MF ran on gel--negative results
MF performed PCR again for pCC305 and dCas9 in same reaction with annealing temp 69C and extension time 2:30
Need to run on gel
Friday June 21
dCas9
MB ran PCR for pCC305 and dCas9 on gel
Positive results for dCas9
Negative results for pCC305
MF performed PCR clean up for dCas9
MF ran PCR again for pCC305 with 3.5 min elongation and 68C annealing temp
MF ran on gel: partial results, but not clean and not large enough band
MF ran PCR again for pCC305 in buffer HF (4 tubes) and buffer GC (4 tubes) with varying enzyme volumes 0.25, 0.5, 1, and 2 uL. 4.5 min elongation and 68C annealing temp
TALEs
MB cultured two colonies overnight for each 16A and 20A in order to mini-prep
pCC305
MB cultured a colony of pCC305 overnight in order to mini-prep and save stock
gRNAs
Sequencing results (Order 253) arrived, gRNA 20C still had unaltered binding site in both samples.
Saturday June 22
dCas9
MF ran yesterday’s PCR results on gel, results did not look correct
MF, CC redesigned plan for assembly of dCas9 plasmid
TALEs
MF miniprepped two copies each of 16A and 20A (labelled N1 and N2)
pCC305
MF miniprepped pCC305
Sunday June 23
Yeast
MB cultured DBY12397 overnight in order to perform yeast transformation
Week of 6/24/13
Monday June 24
dCas9
MF performed PCR for Z4EVpr using pMN10 backbone, 305-Z4EV-down oligo and Charlie’s pMN10-Z4EV oligo. Annealing temp 62C and 30 sec extension
MF ran on gel, only primer-dimers
MF performed PCR for Z4EVpr using pMN10 backbone, 305-Z4EV-down and pMN10-Z4EV oligos. Used gradient PCR with annealing 60-66C, 30 sec extension
HK ran on gel, still negative results--pMN10 amplification will not work with oligos, pCC305 amplification of Z4EV should be used instead.
MF performed PCR for Z4EV using pCC305 backbone, 305-Z4EV-down and pMN10-Z4EV oligos, with 68C annealing temp and 30 sec extension.
MF ran on gel--
MF performed preparatory restriction digest of pCC305 with SalI and NheI to isolate the backbone
MF ran on gel, cut out wrong piece and discarded correct piece
MF performed same digest again
Yeast
MF, MB transformed 16B, 16C, 20B, 20C TALEs into yeast to make strains 21, 22, 24, and 25.
MF, HK plated 4 yeast strains on 5-FOA plates
TALEs
MB performed restriction digest of TALEs 16A (2 copies), 16B, 16C, 20A (2 copies), 20B, and 20C with AdeI and NdeI in order to prepare them for insertion into the yeast genome
Digest still showed extra band around 1.5 kb for 16A and 20A TALEs
MF performed analytical digest of pCC305 in AdeI and NdeI in order to compare
HK ran on gel--looked as expected, does not explain strange TALE results
Probably need to redo second cycle for TALEs 16A and 20A
Reporter with Binding Sites
Sequencing data for insertion of G-blocks into iGEM.4 appears show successful insertion of 6 G-blocks plus another two with point mutations in spacer regions. Order from IDT is mislabeled. All ones labeled 16 are actually 20 and vice-a-versa.
Tuesday June 25
TALEs
HK performed 2nd cycle reaction for TALEs 16A and 20A
Plated and transformed
Yeast are growing for other TALEs. Need to grow another night before we pick transformed colonies
Reporter/Binding sites
Need to do yeast transformation on 6-8 correct binding sites plus control iGEM.4
MF, AR performed restriction digest of BS 1x-16A, 1x-16C, 1x-20A, 1x-20B, 1x-20C, 3x-16A, 3x-20A, 3x-20B, and iGEM.4 with BsaBI and KpnI in order to prepare for insertion in the yeast genome. Digest failed because BsaBI was unable to function due to methylation site
No other available enzymes to perform transformation today
HK performed PCR of desired piece of reporter plasmids in order to transform tomorrow
MF, AR ran on gel and all appear successful but two had an extra band that could not be accounted for
MB performed sequencing reaction for second copy of incorrect binding sites and the ones with point mutations in spacers (1x-16A, 1x-16B, 1x-16C, 3x-16B, 3x-16C, and 3x-20C), sent off for sequencing (Order 286)
dCas9
MF, AR re-ran PCR of Z4EVpr with temperature gradient (anneal 65-72C, extension 30sec)
MF ran on gel and PCR was successful for 5 of 8 tubes
MB ran gel extraction for pCC305 backbone to purify large band to use in dCas9 plasmid
MF performed PCR cleanup of Z4EVpr
MB, AR performed Gibson assembly of Z4EVpr, pCC305 backbone, and dCas9 gene
MB, MF transformed and plated 2 copies of Gibson assembly plus no insert control
Yeast
MF, HK, AR made YEPD medium
MF, HK, AR cultured DBY12397 yeast overnight for transformation
Wednesday June 26
Reporter/binding sites
AR, MF ran PCR of iGEM.4 control and with binding sites labelled 1x16A, 1x16C, 1x20A, 1x20B, 1x20C, 3x16A, 3x20A, 3x20B, using oligos Knopf His3 5p and 3p
Anneal at 68C, extension time 2 min
MF ran on gel--extra bands on 3x16A and 3x20A, plus more volume is needed to transform
AR ran PCR again with 3 copies each of 9 plasmids and Knopf His3 oligos, Annealing at 68C and extension time 2 min
MF ran on gel--extra bands now shown in all 27
Sequencing confirmed second copy of iGEM.4 with 1x16B, 3x16B, 3x16C
MF ran same PCR as above for 3 copies each of 1x16B, 2x16B, and 3x16C
MF ran on gel--same extra bands shown on all wells
AR ran PCR with new oligo tubes for iGEM.4 control and with bs1x16A and 3x16A
MF cultured two colonies from plate of iGEM.4-bs3x20C to miniprep tomorrow
TALEs
MB ran colony PCR on 3 copies of TALE 20A and 1 copy of TALE 20B
MF ran on gel--negative results
dCas9
MF cultured one colony from Gibson plate 1 for iGEM.6 to miniprep tomorrow
Thursday, June 27
Reporter/Binding Sites
MF ran gel for PCR products iGEM.4 control, bs1x16A, and bs3x16A using oligos Knopf-His3 5’ and 3’ Annealed at 68C with 2min extension
no extra bands, appears successful
need to perform two more times successfully
MF ran PCR with 3 copies each of 9 binding sites: 1x16B, 3x16B, 3x16C, 1x16C, 1x20A, 1x20B, 1x20C, 3x20A, 3x20B
Ran on gel--extra bands showed up again
Decided to transform anyway
MF, AR performed PCR cleanup on 12 iGEM binding sites (inc. control)
Mixed up and lost 2: 3x16B and 3x16C
MB saved glycerol stocks and mini-prepped DNA for iGEM.4 with bs 3x-20C (2 copies)
MB performed analytical digest of iGEM.4 with bs 3x-20C with BamHI and MfeI
Ran on gel and results were positive
MF ran sequencing reaction for iGEM.4 with bs3x20C (2 copies) using oligos yEVenus-seq1 and 303-MCSup and sent off for sequencing
dCas9
MB saved glycerol stocks and mini-prepped DNA for iGEM.6 (pCC305, Z4EVpr, dCas9)
MB performed analytical digest of iGEM.6 with EcoRV and SapI
Ran on gel and results were negative (Need to rebuild iGEM.6)
MF performed PCR of Z4EV for dCas9 from pCC305 with oligos pMN10-Z4EV and 305-Z4EV-down
TALEs
MF performed colony PCR for TALEs 16A and 20A
Ran on gel and results were negative
MB cultured two colonies from each TALE 16A and 20A overnight in order to mini-prep
Friday June 28    TALEs
MB saved glycerol stocks and mini-prepped two copies of TALEs 16A and 20A
MB performed restriction digest with AdeI and NdeI to prepare for yeast transformation (Look for extra band)
MF ran on gel and TALEs 16A and 20A appear to be contaminated with another plasmid
MF ran the second cycle TALE assembly for TALEs 16A and 20A
MB, MF transformed 16A and 20A
Yeast
MF, AR transformed into yeast iGEM.4 control and with binding sites 1x16A, 3x16A, 1x16B, 1x16C, 1x20A, 3x20A, 1x20B, 3x20B, and 1x20C, to make yeast strains
AR, MF, JZ plated all 10 strains
dCas9
MF ran PCR of Z4EV from pCC305 with 305-Z4EV-down and pMN10-Z4EV, Annealing at 68C and 2min extension (8 tubes)
MB ran on gel and results were positive but we got very low concentrations of DNA. Need to try to tweak PCR protocol to optimize concentration
MF performed Gibson assembly of iGEM.6 (pCC305, Z4EV, dCas9) (40uL volume)
MF, MB Transformed via heat shock one copy plus one control
Reporters/Binding Sites
MB, AR ran PCR of bs 3x-16B and 3x-16C to prepare for yeast transformation with Hist3 primers. Annealing at 68C and 2 minute extension
MB ran on gel and results showed amplification of a DNA fragment that doesn’t match our expected size
gRNA
AR, MF performed SLIC for gRNA 20C with new oligos
MF, MBTransformed via heat shock
Saturday June 29
Yeast
AR started overnight cultures of iGEMY-1, iGEMY-2, iGEMY-3, iGEMY-4, and DBY12397
all failed except for DBY12397
TALES/dCas9/gRNA
MF cultured TALES 16A and 20A (2 copies), gRNA20C (2 copies) and iGEM6 (1 copy)
Sunday June 30
Yeast
MF, AR, JZ, HK started 50 mL YEPD and colony of DBY12397 to re-do transformations
TALES/gRNA/dCas9
MB mini-prepped TALES 16A and 20A (2 copies), gRNA20C (2 copies) and iGEM6 (1 copy)
Week of 7/1/13
Monday July 1    TALEs
MB performed digest of TALEs 16A, 16B, 16C, 20A, 20B, and 20C with AdeI and NdeI to prepare for yeast transformation
MB ran on gel and results are positive
gRNA
AR performed sequencing reaction for gRNA 20C
dCas9
MF performed analytical digest of iGEM.6 with BamHI and KpnI and perform sequencing reaction
MB ran on gel and results are negative
Need to rebuild iGEM.6
Binding sites/Reporter
MB performed digest of all reporters to prepare for yeast transformation with AfeI and KpnI
MB ran on gel and results are likely positive (so much DNA that gel is blurry)
MF performed Golden Gate reaction to assemble iGEM.5 (ACTi promoter, YEVenus, CYC terminator, pCC300 backbone)
Need to transform
Tuesday July 2
Yeast
MF, MB measured OD660 of yeast culture from Sunday night at ~4.8, diluted yeast by 2mL culture into 45mL and placed back in shaker, with goal of OD660~1.04
MF, MB measured OD660~0.275 @11:00am
MF measured OD660~0.52 @12:30pm
MF measured OD660~0.97 @2:15pm
Transformation begun @2:45pm
MF, MB transformed iGEM.4 itself and with 7 binding sites (1x16A, 1x16B, 1x16C, 3x16A, 3x16B, 3x16C, 1x20A), as well as all six TALEs, with one no DNA control
MF plated yeast: TALEs on 5FOA plates, reporters on YEPD plates with ~50uL each hygromycin
Reporter
MB, MF performed PCR for Z4EVpr and dCas9 in order to put into pCC305 to make iGEM.6
MB ran Z4EV PCR on gel and PCR was succesful
PCR clean up gave concentration of 30 ng/uL
MB ran dCas9 on gel and PCR was unsuccessful
MF, AR plated stocks of the 1x and 3x binding site versions of iGEM.4
MB, MF transformed and plated iGEM.5 (2 plates)
MB cultured a colony of iGEM.4 in LB+Amp overnight in order to mini-prep tomorrow
TALEs
MF, AR plated stocks of TALEs 16B, 16C, 20B, and 20C
Wednesday July 3
Yeast
MF, MB transformed yeast with bs 1x-20B, 1x-20C, 3x-20A, 3x-20B, 3x-20C, and a no insert reporter control
Yeast could not be plated--more plates being made with YEPD+hyg
Tubes in cold room--Charlie will plate tomorrow
iGEM.6
MF ran PCR for Z4EVpr using PCR product from yesterday and oligos pMN10-Z4EV and 305-Z4EV-down, annealled at 68, extension time 30sec
MF ran PCR for dCas9 with oligos 305-dCas9 up and down, annealled at 69, extension time 2 min
MF ran both PCRs on gel: Z4EV was negative, dCas9 was positive
MF ran PCR for Z4EV again
MF ran PCR cleanup for dCas9 with 8 tubes, final concentration 414 ng/uL in 20 uL
MF ran PCR cleanup for dCas9 with 8 tubes, final concentration 450 ng/uL in 20 uL
Reporter
MB mini-prepped iGEM.4 from colony picked from stock plate
Thursday July 4
Yeast
CC plated iGEM.4-bs1x20B, 1x20C, 3x20A, 3x20A, 3x20B, 3x20C, and no insert control on Hygromycin
CC cultured DBY12397 overnight for transformation
Friday July 5
Reporter
CC digested all twelve iGEM.4-binding sites plasmids with Kpn1/Afe1 for transformation
MF ran on gel: all twelve iGEM.4-binding sites look good
MF digested iGEM.4 control with Afe1/Nhe1 for transformation
MF ran on gel--did not seem to digest--not transformed today
MF, CK transformed 12 reporters plus no DNA control into yeast, making strains 3,4,6,7,9,10,12,13,15,16,18,19.
MF plated 12 reporter yeast strains on Hygromycin
MF ran sequencing reaction for iGEM.5 (two copies) with oligos yEVenus-seq1, yEVenus-seq-2, 306-Ura3p, 306-MCSup and sent off for sequencing
TALEs
CC digested TALEs 16B, 20B, 16C, and 20C for transformation with DraIII/Nde1
MF ran on gel: all four look good
MF spotted yeast from 7/2 transformation of all 6 TALEs onto G418 plates (strains 20-25)
MF, CK transformed TALEs 16B, 20B, 16C, 20C plus no DNA control into yeast, making strains 21,22,24,25
MF plated TALE strains on 5FOA plates
dCas9
MF ran Gibson assembly for iGEM.6 using gel-extracted pCC305 vector, 3xmolar dCas9 PCR, and 3xmolar Z4EVpr PCR, with no insert control
MF transformed and plated iGEM.6 Gibson and control
Saturday July 6
Gibson assembly plate had no colonies to culture
CC spotted yeast from 7/3-4 transformation of iGEM.4-bs1x16B*, 1x16C*, 3x16A*, 3x16B*, 3x16C* (strains 3,6,7,9,10)
= name corrected
names not corrected on labels yet!
Sunday July 7
MF, CC cultured 4 yeast strains from plate spotted on 7/6 for iGEM.4-bs1x16B*-3, 3x16A*-1, 3x16B*-1, 3x16C*-1 (strains 3,6,7,9,10) in YSCD
MF cultured DBY12397 overnight in 45mL YEPD
Week of 7/8/13
Monday July 8
Yeast
MF digested 6 TALEs in Nde1/DraIII with NEBuffer 2 to transform
MB ran on gel and all 6 look mostly good, but 16A and 20A top bands slightly lower than other TALEs?
CC diluted cultures of yeast strains in YSCD and YSCD-BED to flow later today
MF picked and spotted yeast colonies from 7/5/13 transformation plates that looked promising: GEMY-9, 12, 15, 17, 19, 25. (TALE 20C, iGEM.4-bs1x16B*, 3x16C*, 3x20A*, 3x20B*, 1x20C*)
= name corrected
MB, MF, CC transformed 6 TALEs into DBY12397
CC, MF, MB, AR, HK, JZ, CK ran flow cytometer with:
Calibration
DBY12307 (GEMY-0)
GEMY-7.1
GEMY-7.2
GEMY-7.3
GEMY-3.1
GEMY-6.1
GEMY-6.2
GEMY-9.1
Results: flourescence was only 1-2 fold greater than untransformed strain
Reporters
MB ran PCR with 4 tubes each of iGEM.4-with 12 binding sites (48 tubes total) using oligos yEVenus-seq1 and Spe1-iGEM.5-binding-site in order to ligate binding sites into iGEM.5
MF ran on gel: results all look positive, band around 9kb also present
MB performed PCR clean up and got good concentrations for all binding sites
MB, AR performed restriction digest for ligation of binding site PCR into iGEM.5 with SpeI
gRNA
MB ran sequencing reaction for gRNA 20C (2 copies) with oligo pRSup and sent off for sequencing
dCas9
MF performed Gibson assembly of iGEM.6 (Z4EVpr, dCas9, pCC305)
MB transformed and plated Gibson and control
Tuesday July 9
Yeast
MB cultured tubes from yeast reporters spotted on 7/8/13 to flow tomorrow morning
Reporters
MB performed CIP and performed PCR clean up on iGEM.5
Sequencing data came back bad. Need to re-build iGEM.5
MF performed digest of binding site PCR with DpnI to cut up methylated DNA
MB performed PCR clean up
Need to insert into iGEM.5
dCas9
MF performed PCR for pCC305 with Z4EVpr using 305-mCherry-up and Z4EV-down
MB ran on gel and looked as if almost no DNA was recovered. Need to order new oligos to build #iGEM.6
MF performed PCR for dCas9 with dCas9-up and dCas9-down
MB ran on gel and looked like dCas9 was successfully PCRed but bands very smeary. Not sure why
Wednesday July 10
Yeast
MB, MF, CC ran flow cytometry for GEMY.7-(2-4), GEMY.9-(1-4), GEMY.12-1, GEMY.15-1, GEMY.19-(1-3). YFP production appeared to be at most a 2-fold increase over background. Going to try transforming reporters into different yeast strain
Transformations from Monday appear to be more abortive transformants. DNA is getting into the yeast but not integrated in the genome. Not very big colonies. No transformants on 5FOA plates and very small transformants on G418 plates. A few colonies of G418 control plate which shouldn’t have colonies
Need to culture 100mL yeast to do transformation tomorrow
Reporters
MB successfully mini-prepped three copies each of iGEM.5-1 and iGEM.5-2
MB performed analytical digest of iGEM.5 copies with NsiI
MB ran on gel and confirmed that iGEM.5 assembly was unsuccessful
MF digested both iGEM.2 and iGEM.4 with BstEII-HF and SalI-HF in cutsmart for gel extraction to ligate and make iGEM.5
MB ran on gel and correct size bands were present
MF performed gel extraction
MB performed the ligation of the digests of iGEM.2 and iGEM.4 in order to make iGEM.5
Need to digest reporters for transformation tomorrow (need to get AfeI from Mert)
TALEs
MF, AR digested 6 TALEs with DraIII and NdeI in NEbuffer 2 for yeast transformation tomorrow
Thursday July 11
TALEs
MB ran TALEs digested with DraIII and NdeI on a gel and bands appear to be right size. Digest was slightly incomplete and put back in 37C water bath
MB cultured colonies in LB+Amp for TALEs 16A, 16B, 16C, 20A, 20B, and 20C to mini-prep tomorrow
Yeast
MF transferred colonies from Monday’s transformation to new plates. If they were on G418, they were moved to 5FOA and vice-a-versa.  Plates were left in the incubator to look for continued growth
MB centrifuged overnight yeast culture and there was not enough yeast to do transformation. Resuspended yeast in YEPD and allowed to grow more in shaker.
Yeast started to grow again, OD660~0.37 @ 2:30, OD660~0.75 @ 4:15
MF started a culture of MMY-C yeast in YEPD to transform tomorrow
Reporters
MF performed restriction digest of iGEM.4, iGEM.4 + 1x-binding sites, and iGEM.4 + 3x-binding sites with AfeI and KpnI-HF in CutSmart Buffer in order to prepare for integration in the yeast genome
MB ran on gel and all correct bands there. Reporters ready to be transformed
MB cultured colonies in LB+Amp for iGEM.4, iGEM.4 + 1x-(16A, 16B, 16C, 20A, 20B, and 20C), iGEM.4 + 3x-(16A, 16B, 16C, 20A, 20B, and 20C) to mini-prep tomorrow
MB transformed the ligation and control to form iGEM.5 and plated on LB+Amp
Friday July 12
TALEs
MB, MF mini-prepped TALEs 16A, 16B, 16C, 20A, 20B, and 20C
Reporters
MB, MF mini-prepped iGEM.4, iGEM.4 + 1x-(16A, 16B, 16C, 20A, 20B, 20C), and iGEM.4 + 3x-(16A, 16C, 20A, 20B, 20C)
MB cultured colony in LB+Amp for iGEM.4 + 3x-16B
MB cultured 4 colonies from iGEM.5 plate in LB+Amp
Yeast
MF, MB transformed 6 TALEs and 13 Reporters into MMY-C yeast, plus a TALE control and a reporter control
MB, AR plated all recovered transformants on G418 for TALEs and Hygromycin for iGEM.4 and all combinations of iGEM.4 with binding sites
TALEs spotted 7/11 onto 5FOA plates look promising--need to culture to flow tomorrow
MF cultured CCY232-3 for background fluorescence along with all spotted colonies for flow
TALEs spotted on G418 did not grow as well
Saturday July 13
Reporters
MB mini-prepped iGEM.4 + 3x-16B
MB saved glycerol stocks and mini-prepped iGEM.5 (4 copies)
Yeast
MF, HK diluted all 48 cultures for flow. 15 microliters of culture from well into 3 mL of YSCD. Allow to grow in 30C roller for 6 hours before performing flow.
MF, MB, HK, JZ performed flow on 48 tubes. Need to analyze data, but TALEs 16B, 16C, 20B, 20C seem to have fluorescence significantly above background strain
Sunday July 14
Yeast
MB cultured both DBY12397 and CCY-232-3 for transformations tomorrow
Week of 7/15/13
Monday July 15
Yeast
MF, MB digested 13 iGEM.4 reporters with NheI-HF and AfeI for transformation (enough for two transformations per tube) (cutsmart, 2h @37C)
gel looked strange, transformation aborted
MF cultured 45mL DBY12397 for transformation tomorrow
MF, MB digested 13 iGEM.4 reporters with KpnI-HF and AfeI for transformation tomorrow (cutsmart, 3h @37C)
MB spotted reporters in MMY from 7/12/13 transformation onto YEPD+hyg plates
MB, AR, MF cultured reporters in MMY (spotted today) in 3 mL YSCD for flow tomorrow
Reporters
MB ran analytical digest of iGEM.5 (4 copies) in NsiI (buffer 3.1, 2h @37C)
results look strange, undigested?
MF ran analytical digest of iGEM.5 (4 copies) in EcoRI-HF/SalI-HF (cutsmart, 2h@37C)
Looks good, previous problem probably with digest, not DNA
dCas9
MF ran perparatory digest of pCC305 in Nhe1-HF (cutsmart, 3h @37C)
MF ran PCR of dCas9 with oligos dCas9-N1 and dCas9-N2, with annealing temp 67.5C and extension time 2min
MB, JZ ran PCR on gel, successful bands
AR, JZ ran successful PCR cleanup on dCas9
MB, JZ ran prep digest on gel, looked faint/dilute but correct
MF cut out larger band (backbone)
MF, MB ran gel cleanup, little to no DNA present
Tuesday July 16
Yeast
MF diluted yeast cultures for flow so they would be 0.1 OD by 3 pm and put back in 30C roller
Flow data showed good fluorescence levels for iGEM.4 but when the binding sites (1x or 3x) are present there is almost no fluorescence (2-fold max over background)
MB transformed DBY12397 with all 1x and 3x versions of iGEM.4, iGEM.4, and a no DNA control
MB plated on YPD+Hyg and put in 30C incubator
MB, MF started cultures for TALEs 16B, 16C, 20B, and 20C along with a DBY12397 background to do a time course of beta-estradiol induction tomorrow
MF, AR diluted back the cultures at night into YSCD to be ready to flow in morning (target OD660~0.01, 0.02, 0.03 in morning)
Reporters
MF performed sequencing reaction for iGEM.5 (2 copies)
MB performed digest of iGEM.5 with SpeI-HF in order to prepare for insertion of binding sites
MB performed PCR clean-up of digest
MB performed CIP on iGEM.5 digest clean-up
MB performed ligation of binding sites into SpeI digested iGEM.5
MB transformed and plated on LB+Amp
MB cultured two colonies of pCC305 in LB+Amp in order to mini-prep tomorrow
TALEs
MF performed sequencing reaction for TALEs 16A and 20A to try and determine issue with these TALEs
Wednesday July 17
dCas9
MB mini-prepped two copies of pCC305 backbone
MF performed restriction digest of pCC305 with NheI in order to prepare for insertion of dCas9
MF, MB performed gel extraction of larger band
MB performed Gibson assembly of iGEM.6 with gel extraction product of pCC305 and dCas9 PCR product
MB transformed and plated on LB+Amp
Yeast
MF, MB, AR performed time course flow cytometry for TALEs 16B, 16C, 20B, and 20C along with background DBY12397: times every 30 min, inducing and non-inducing medium, start at target OD660~0.02
Reporters
MB performed PCR of 5x-16A and 5x-16C binding sites. Annealing at 67 and extension at 72 for 20 sec
MF ran on gel and PCR appears successful
Need to run on polyacrylamide gel to try and extract product that contains all 6 oligos
Thursday July 18
Reporters
MB, MF mini-prepped iGEM.5 + 1x-(16A, 16B, 16C, 20A, 20B, and 20C) and iGEM. 5 + 3x-(16A, 16B, 16C, 20A, 20B, and 20C)
MB digested with SpeI to look for insertion of binding sites
MB, MF mini-prepped 404 TEF1 and 404 TDH3 in order to swap the TEF and TDH promoters in for ACT1
MB digested iGEM.5 and all versions of it with 1x and 3x binding sites with AfeI and KpnI-HF for yeast transformation
MB, MF, CK ran 5x-16A and 5x-16C PCR on acrylamide gel
Gel extraction failed due to smearing on gel. Need to re-run PCR
Need to perform Gibson/SLIC into iGEM.4
dCas9
Plate for iGEM.6 has no colonies
MB re-ran Gibson assembly of iGEM.6
MB, JZ transformed and plated on LB+Amp
Yeast
MB cultured DBY12397 overnight to perform transformation tomorrow
Friday July 19
Reporters
MB ran digest of iGEM.5 and all versions of it with 1x and 3x binding sites on a gel and all digest appear successful
MB re-ran PCR of 5x-16A and 5x-16C binding sites
MF performed sequencing reaction for iGEM.5 and all 1x and 3x binding site versions (two copies of each)
Yeast
MB transformed iGEM.5 and all 1x and 3x binding sites versions into DBY12397
HK plated on YPD+Hyg
AR balanced the ODs of DBY12397 and transformants with TALEs 16B, 16C, 20B, and 20C and began time course for induction with beta estradiol
AR, MB, JZ, and MF ran time course for DBY 12397, TALEs 16B, 16C, 20B, and 20C, inducing and non-inducing medium, with times every 30 minutes. Aimed for 24 timepoints over 12 hours but ran out of induced DBY 12397 and stopped at 21st time point.
dCas9
Plate for iGEM.6 has no colonies again. Need to check to see if Gibson Master Mix is problem by using the positive control that comes with Gibson assembly kit
TALEs
MF performed sequencing reaction for TALEs 16A and 20A to try and determine the issue with them
Week of 7/22/13
Monday July 22
Reporters
MB performed PCR of TEF1pr and TDH3pr in order to replace Act1pr in iGEM.4
MB ran on gel and PCR appears successful
MB performed PCR clean up
MB performed restriction digest of TEF1pr and TDH3pr with AscI, DpnI, and XbaI
MB performed PCR clean up
MF performed overnight ligation with 1x-(16A, 16B, 16C, 20A, 20B, and 20C) and 3x-(16A, 16B, 16C, 20A, 20B, and 20C)
MF performed PCR with oligos 3 and 4 for 5x-16A and 5x-16C in order to try and make the 5x binding sites
MB ran on gel and bands appeared very smeary
This method for building 5x binding sites has been abandoned
MB performed restriction digest of iGEM.4 with AscI and SpeI
MF performed gel extraction and clean up of backbone
Yeast
MB cultured DBY12397, iGEM.4, 1x-16B, 3x-16A, 3x-16B, 1x-20B, 1x-20C, and 3x-20A (2 copies of everything but DBY12397) in order to flow tomorrow morning
dCas9
MB, MF performed Gibson assembly of iGEM.6 using 1:1, 3:1, 6:1, and 9:1 insert to backbone ratios. The Gibson assembly positive control was also performed
MB transformed and plated on LB+Amp
Competent Cells
MB inoculated a colony of DH5Alphas in 2 mL of LB overnight to make chemically competent cells
Tuesday July 23
Yeast
MB flowed DBY12397, iGEM.4, 1x-16B, 3x-16A, 3x-16B, 1x-20B, 1x-20C, and 3x-20A (2 copies of everything but DBY12397). The ones with binding sites have nearly identical fluorescence to the background indicating that the binding site insertion is disrupting the expression of YFP
MB picked transformants from iGEM.5, iGEM.5+1x-(16A, 16B, 16C, 20A, 20B, and 20C), and iGEM.5+3x-(16A, 16B, 16C, 20A, 20B, and 20C) and put them on new YPD+Hyg plates
MB made glycerol stocks of GEMY 1A, 4A, 6A, 9A, 12A, 18A, and 19A
dCas9
Plates had no colonies including the positive control plate indicating that there is a problem with the Gibson assembly master mix
Competent Cells
MB made 0.1 M CaCl2 and 0.1 M CaCl2 in 15% glycerol
MB, CC made and aliquoted chemically competent DH5Alphas
Reporters
MF performed PCR clean up of ligation of binding sites to either TEF1 or TDH3 promoters
MF performed ligation of promoter-binding site with pCC305 and left overnight in heat block in cold room
Wednesday July 24
Yeast
MB, MF started cultures for DBY12397, iGEM.5, iGEM.5+1x-(16A, 16B, 16C, 20A, 20B, and 20C), and iGEM.5+3x-(16A, 16B, 16C, 20A, 20B, and 20C) in a 96 well plate to flow tomorrow morning
MF diluted back to aim for an OD of 0.1 by 9 am
Reporters
MB transformed ligations of TEF1/TDH3, binding site, and pCC305 backbone
MF performed sequencing reaction for iGEM.5 and iGEM.5 with all the 1x and 3x binding sites and sent off for sequencing
TALEs
MF performed sequencing reaction for TALEs 16A and 20A to try and determine the issue with these plasmids
Thursday July 25
Yeast
MB, MF flowed DBY12397, iGEM.5, iGEM.5+1x-(16A, 16B, 16C, 20A, 20B, and 20C), and iGEM.5+3x-(16A, 16B, 16C, 20A, 20B, and 20C)
Reporters
MB cultured colonies overnight in LB+Amp from ligation plates of iGEM.7 and iGEM.8 with binding sites in order to mini-prep tomorrow
Friday July 26
Reporters
MB, AR miniprepped iGEM.7 and iGEM.8 with 1x and 3x binding sites.
JZ digested iGEM.7 and iGEM.8 plasmids with Mlu1, will run analytical gel on Monday.
Week of 7/29/13
Monday July 29
Reporters
AR ran gel of all iGEM.7 and iGEM.8
unsuccessful, only one band
dCas9
JZ ran digest of PCC305 with Nhe1
AR ran PCR of Cas9 with Cas9-N1 and Cas9-N2
JZ ran PCR clean up and CIP of Cas9
Tuesday July 30
dCas9
AR, CC performed gel extraction and purification of PCC305
Wednesday July 31
Yeast
AR cultured DBY12397 and TALES 16B, 16C, 20B, 20C in 3mL SCD in order to redo flow tomorrow
dCas9
HK re-ran PCR for Cas9 with Cas9-N1 and Cas9-N2. PCR from yesterday contained no product. Confirmed today’s PCR on a gel.
JZ ran a Gibson assembly reaction with the new Gibson kit, the dCas9 PCR product, and the Nhe1 digested pCC305. Gibson kit competent cells were transformed with backbone only, insert only, backbone+insert, and a positive control and plated.
Reporters
JZ redesigned and ordered oligos to introduce new restriction sites for the binding site batteries and the TEF1 and TDH1 promoters for the second attempt at constructing iGEM.7 and iGEM.8 as well as the BioBrick parts containing the binding sites and the promoters.
Thursday August 1
Yeast
AR,  JZ, and HK performed time course flow cytometry for DBY 12397, TALEs 16B, 16C, 20B, and 20C
dCas9
JZ concluded Gibson failed (only colonies on positive control).  Redid Gibson and ran SLIC assembly. Transformed cells with assembly products and plate them to grow overnight.
Friday August 2
Reporters
AR prepped new oligos for TEF and TDH.
JZ ran a PCR to amplify the TEF1 promoter. Ran the reaction product on a gel, PCR appears to have failed. Need to redo.
JZ streaked a plate of stock p404TDH3 for amplification on Monday.
Yeast
AR analyzed data from time course, looks to be successful so far
TALEs
HK ran first cycle of the Golden Gate reaction for pTAL synthesis of bs16A and bs20A.
dCas9
Gibson and SLIC failed, need to design new strategy for dCas9 assembly.
Week of 8/5/13
Monday August 5
Reporters
JZ ran a PCR cleanup on PCRed TEF1 promoter, nanodrop reports no DNA.
JZ redid the PCR of TEF1.
AR ran on gel, PCR was unsuccessful
TALES
AR ran colony PCR on products of pTAL synthesis of bs16A and bs20A
AR ran PCR products on gel and all were unsuccessful, will redo first cycle of Golden Gate assembly
Tuesday August 6
Yeast
AR cultured DBY12397 and TALES 16B, 16C, 20B, 20C for flow tomorrow morning to determine input/output curve for BED concentrations
AR diluted cultures back to flow at an OD of 0.4 tomorrow morning, and induced a set of TALEs and DBY12397 in each of the following concentrations of DMSO/BED
0 M
0.1 nM
1 nM
10 nM
20 nM
50 nM
100 nM
500 nM
1 uM
Wednesday August 7
Yeast
AR ran flow on TALEs and DBY12397 for all concentrations listed above
Reporters
JZ and CC ran PCR of TEF1pr using p404TEF1 backbone and oligos XbaAscTEF1-5p and Age1TEF1-3p.P PCR product confirmed on gel and cleaned up.
JZ and CC ran PCR of binding site batteries using iGEM.4 (1x16A,3x16A,1x16C,1x20A,1x20B,3x20B,3x20C) template and oligos Age1-iGEM5bind and yEVseq1. Gel results unclear, PCR may have been successful. Product frozen for clean up tomorrow.
Week of 8/12/13
Tuesday August 13
AR ran PCR of
TEF1pr using 404TEF1 backbone and oligos XbaAscTEF1-5p and Age1TEF1-3p
binding site batteries using iGEM.4 (3x16C, 3x20C, 1x16B, 3x20A, 3x16B) template and oligos Age1-iGEM5bind and yEVseq1
TDH3pr using 404TDH3 backbone and oligos XbaAscTDH3-5p and Age1TDH3-5p
AR ran gel of PCR products.  TEF1 and bs batteries were successful, TDH3 failed
AR performed PCR clean up on TEF1 and bs batteries
AR ran analytical digest of TEF1pr with Xba, Age1, and Dpn1 and digest of all binding sites (1x16A,B,C, 3x16A,B,C, 1x20A,B,C, 3x20A,B,C) with Spe1, yEVseq1, and Dpn1
AR streaked LB+Amp plates with iGEM.5 and iGEM.5 + 1x(16A, 16B, 16C, 20A, 20B, 20C) and 3x(16A, 16B, 16C, 20A, 20B, 20C)
CC heat shocked with plasmid pSB1C3 from iGEM plate for BioBrick backbone
Wednesday August 14
AR,CC designed new oligos for 5xbinding sites (16A, 16B, 16C, 20A, 20B, 20C) with Age1-binding sites x3- spacers and Spe1-spacers-binding sites x3
AR cultured iGEM.5 and iGEM.5 + 1x(16A, 16B, 16C, 20A, 20B, 20C) and 3x(16A, 16B, 16C, 20A, 20B, 20C) in LB+Amp to miniprep tomorrow morning
Thursday August 15
AR miniprepped iGEM.5 and iGEM.5 + 1x(16A, 16B, 16C, 20A, 20B, 20C) and 3x(16A, 16B, 16C, 20A, 20B, 20C)
AR ran digest of iGEM.5 and iGEM.5+1x(16A, 16B, 16C, 20A, 20B, 20C) and 3x(16A, 16B, 16C, 20A, 20B, 20C) with Kpn1-Hf and Afe1
Ran digest on gel, all were successful
CC ran digest of pSB1C3
AR cultured DBY12397 and TALEs 16B and 16C for transformation
Friday August 16
AR,CC transformed digested iGEM.5 and iGEM.5 + 1x(16A, 16B, 16C, 20A, 20B, 20C) and 3X(16A, 16B, 16C, 20A, 20B, 20C) into DBY12397
AR,CC  transformed digested iGEM.5 + 1x(16B, 16C) and 3x(16B, 16C) into their respective DBY12397+16B and DBY12397+16C strains
Sunday Aug 18
CC patched colonies from Aug 16th yeast transformation to new YEPD+Hyg plates and returned to 30C. Efficiency low in most cases, zero in a few.
Week of 8/26/13
Monday Aug 26
JZ ran PCR of binding site batteries from 5X20 BS oligos, ran products on gel, which looked successful.
Tuesday Aug 27
MF transformed Z4EV-dCas9 plasmid into E. coli and plated on LB+Amp
Wednesday Aug 28
MB cultured 4 colonies from Z4EV-dCas9 plate in LB+Amp in order to mini-prep tomorrow
Thursday Aug 29
JZ ran PCR of binding site batteries from 5X16 BS oligos.
MB mini-prepped 4 samples of Z4EV-dCas9
MB performed analytical digest of Z4EV-dCas9 plasmid with BamHI and XbaI
MB ran on gel and only one band appeared on gel. Assembly of dCas9 plasmid appears to have failed
Week of 9/2/13
Wednesday September 4
MB performed restriction digest of all binding site PCRs with AgeI and SpeI
MB performed restriction digest of TEF1pr PCR with AgeI and XbaI
MB performed ligation of TEF1pr, binding sites, and pSB1C3 in order to make reporter biobricks
MB transformed into E. coli and plate on LB+Cm
MB plated iGEM.4+3x-20A from frozen stock
Thursday September 5
MB cultured colonies in LB+Cm from all biobrick plates except for the 1x-16B binding site because that plate had no colonies
Friday September 6
MB saved glycerol stocks and mini-prepped the colonies cultured yesterday
MB performed an analytical digest of the biobricks with SpeI-HF and XbaI to determine if inserts are there and to determine orientation
JZ digested TEF1pr with AscI/Age1 and iGEM.4 backbone with AscI/Spe1 for ligation.
Saturday September 7
MF performed biobrick ligation of TEF1pr-bs1x16B into pSB1C3
MF transformed ligation and plated on LB+Cm
MF performed ligation of TEF1pr (cut with AgeI/XbaI) and 12 binding sites (1x and 3x) (cut with AgeI and SpeI) into iGEM.4 backbone (cut with XbaI/SpeI)
MF transformed ligations and plated on LB+Amp
Sunday September 8
MF cultured colonies from 13 ligations, two colonies per plate (26 colonies) to miniprep tomorrow
Week of 9/9/13
Monday September 9
MF miniprepped all 26 cultures from ligations and saved frozen stocks
Wednesday September 11
MB performed an analytical digest of all 12 TEF1pr-Binding Site reporters (2 copies each) with SpeI-HF and AfeI
MB performed an analytical digest of TEF1pr-1x 16B BioBrick (2 copies) with SpeI and XbaI
Thursday September 12
MF cultured DBY12397 in YEPD overnight for transformation
Friday September 13
MF digested iGEM.7+binding site (TEF1 constructs) 12 plasmids for transformation with Afe1 and Kpn1-HF
Digest looks good for all except 1x16B and 3x16C
MF transformed 12 plasmids into DBY12397 with one no DNA control
MF plated on YEPD+Hyg plates
Retrieved from " http://2013.igem.org/Team:Duke/Notebook/Lab_Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
