"
Page
Discussion
View source
History
teams
Log in
 
Team:Duke/Project/Design
From 2013.igem.org
Home
Team Duke 2013
The Team
Undergrads
Grad Students
Instructors
Genetic Toggle Switch
The Project
Background Information
Experimental Design
Results:
Library of DNA Parts
Results:
Parts Characterization
Biological Parts Submitted
Mathematical Modeling
Cooperativity and Hill Equation
Thermodynamic Model : Introduction
Thermodynamic Model : Cooperative Repression
Kinetic Model :
Bifurcation and Bistability
Human Practices
From Bench to Biotech
Lab Notebook
Overview
Protocols
Strain List
Safety
E.coli and S.cerevisiae
Xanthomonas Bacteria
Attributions
iGEM 2013
Experimental Design
Our experimental approach involves the following process:
Design synthetic DNA sequences and corresponding artificial transcription factors (ATFs) to bind those sites. Check constructs against the yeast genome and transcription factor sequences to ensure orthogonality of the system and limit unintended off-target effects.
Construct two plasmid libraries: one containing fluorescent reporter genes driven by constitutive promoters, and one containing artificial transcription factor genes (TALEs, dCas9, and sgRNAs) driven by inducible promoters. Binding sequences to match each ATF were inserted between the promoter and the gene in the reporter constructs.
Integrate plasmids into the yeast genome, in pairs of corresponding reporters and ATFs. Use flow cytometry along with control strains to quantifiably test for repressive activity and cooperativity in ATF interactions.
Select compatible sets of ATF-binding sequence cassettes, using mathematical modeling results to determine the most robust circuit designs. Design, integrate, and test these circuits for balanced cooperative repression and bistability.
Figure 1.3.1. Experimental Design: Cooperative repression using TALEs
In the absence of β-estradiol, the Z4EV zinc finger protein is sequestered in the cytoplasm by HSP90. Once B-estradiol is added, the Z4EV protein dissociates from the HSP90, revealing a nuclear localization signal (NLS) that translocates the protein to the nucleus and activating the Z4EV promoter (McIsaac et al. 2012). Activation of the Z4EV promoter activates transcription of the iTAL-mcherry fusion protein, which in turn binds to artificial DNA sequences in front of the yEVenus reporter gene. Binding of the iTAL to multiple sites between the constitutive promoter and the reporter gene blocks transcription by RNA polymerase and suppresses yEVenus levels. Multiple variations of this construct differ in binding sequence length and content, as well as number of tandem binding sites in order to generate cooperativity (Murphy et al. 2007). iTALs are designed to match specific binding sites.
Figure 1.3.2. Experimental Design: Cooperative repression using CRISPRs
In the same fashion as above, addition of β-estradiol activates transcription of dCas9-mCherry fusion protein, which associates with constitutively expressed, sequence-specific small guide RNAs (sgRNAs) (Qi et al. 2013). The RNA-protein complex then binds to artificial DNA sequences in front of the yEVenus reporter gene. Binding of the complex to multiple sites between the constitutive promoter and the reporter gene blocks transcription by RNA polymerase and suppresses yEVenus levels. Multiple variations of this construct differ in binding sequence length and content, as well as number of tandem binding sites in order to generate cooperativity (Murphy et al. 2007). Each sgRNAs is designed to match a specific enhancer sequence.
Figure 1.3.3. Toggle Switch Design with Two Mutually Repressive Genes
Sets of reporter constructs and their corresponding artificial transcription factors that demonstrate cooperative repression can then be used to create a toggle switch. Parameters from models of transcription factor binding will inform the selection of the pair of sets that is most likely to result in balanced mutual repression while maintaining high cooperativity. Binding sites for the repressors will be inserted into the repressor plasmids so that the expression of one represses the expression of the other in a bistable fashion.The repressors in the original library of parts are all driven by the Z4EV promoter and so a different inducible promoter must be switched in for one repressor plasmids in order for the expression system to be able to toggle. When this system is transformed into yeast, the expected result is an artificial bistable toggle switch system that is orthogonal to the yeast genome.
References
McIsaac R.S, Oakes B.L, Wang X, Dummit K.A, Botstein D, Noyes M.B: Synthetic gene expression perturbation systems with rapid, tunable, single-gene specificity in yeast. Nuc Ac Res 2012, 1-10.
Murphy K.F, Balazsi G, Collins J.J: Combinatorial promoter design for engineering noisy gene expression. Proc. Nat. Acad. Sci. USA 2007, 104(31):12726–12731.
Qi L.S, Larson M.H, Gilbert L.A, Doudna J.A, Weissman J.S, Arkin A.P, Lim W.A: Repurposing CRISPR as an RNA-guided platform for sequence-specific control of gene expression. Cell 2013, 152:1173-1183.
Retrieved from " http://2013.igem.org/Team:Duke/Project/Design "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
