<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Duke_Notebook_Lab_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Duke/Notebook/Lab_Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Duke/Notebook/Lab_Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Duke/Notebook/Lab_Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Duke/Notebook/Lab_Notebook&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Duke/Notebook/Lab_Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Duke/Notebook/Lab Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="wrap"><DIV class="arrowsidemenu"><DIV><A href="https://2013.igem.org/Team:Duke">Home</A></DIV><DIV class="menuheaders"><A href="https://2013.igem.org/Team:Duke/Team">Team Duke 2013</A></DIV><UL class="menucontents"><LI><A href="https://2013.igem.org/Team:Duke/Team#The_Team">The Team</A></LI><LI><A href="https://2013.igem.org/Team:Duke/Team#Undergrads">Undergrads</A></LI><LI><A href="https://2013.igem.org/Team:Duke/Team#Grad_Students">Grad Students</A></LI><LI><A href="https://2013.igem.org/Team:Duke/Team#Instructors">Instructors</A></LI></UL><DIV class="menuheaders"><A>Genetic Toggle Switch</A></DIV><UL class="menucontents"><LI><A href="https://2013.igem.org/Team:Duke/Project/Problem">The Project</A></LI><LI><A href="https://2013.igem.org/Team:Duke/Project/Basics">Background Information</A></LI><LI><A href="https://2013.igem.org/Team:Duke/Project/Design">Experimental Design</A></LI><LI><A href="https://2013.igem.org/Team:Duke/Project/Results/Part1">Results:Library of DNA Parts</A></LI><LI><A href="https://2013.igem.org/Team:Duke/Project/Results/Part2">Results:Parts Characterization</A></LI></UL><DIV><A href="https://2013.igem.org/Team:Duke/Parts">Biological Parts Submitted</A></DIV><DIV class="menuheaders"><A>Mathematical Modeling</A></DIV><UL class="menucontents"><LI><A href="https://2013.igem.org/Team:Duke/Modeling/Cooperativity">Cooperativity and Hill Equation</A></LI><LI><A href="https://2013.igem.org/Team:Duke/Modeling/Thermodynamic_Model_Intro">Thermodynamic Model : Introduction</A></LI><LI><A href="https://2013.igem.org/Team:Duke/Modeling/Thermodynamic_Model_Application">Thermodynamic Model : Cooperative Repression</A></LI><LI><A href="https://2013.igem.org/Team:Duke/Modeling/Kinetic_Model">Kinetic Model : Bifurcation and Bistability</A></LI></UL><DIV class="menuheaders"><A href="https://2013.igem.org/Team:Duke/Human/">Human Practices</A></DIV><UL class="menucontents"><LI><A href="https://2013.igem.org/Team:Duke/Human/FromBenchToBiotech">From Bench to Biotech</A></LI></UL><DIV class="menuheaders"><A>Lab Notebook</A></DIV><UL class="menucontents"><LI><A href="https://2013.igem.org/Team:Duke/Notebook/Lab_Notebook">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Duke/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2013.igem.org/Team:Duke/Notebook/Strain List">Strain List</A></LI></UL><DIV class="menuheaders"><A href="https://2013.igem.org/Team:Duke/Safety">Safety</A></DIV><UL class="menucontents"><LI><A href="https://2013.igem.org/Team:Duke/Safety/Ecoli_and_Scerevisiae"><I>E.coli</I> and <I>S.cerevisiae</I></A></LI><LI><A href="https://2013.igem.org/Team:Duke/Safety/Xanthomonas"><I>Xanthomonas</I> Bacteria</A></LI></UL><DIV><A href="https://2013.igem.org/Team:Duke/Attributions">Attributions</A></DIV><DIV><A href="https://2013.igem.org/">iGEM 2013</A></DIV></DIV></DIV><DIV class="right_box"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Duke_iGEM_Team_2013"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Duke iGEM Team 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#Week_of_2.2F21.2F13_.7E_3.2F18.2F13"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Week of 2/21/13 ~ 3/18/13</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Week_of_5.2F13.2F13"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">Week of 5/13/13</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Week_of_5.2F20.2F13"><SPAN class="tocnumber">1.1.3</SPAN><SPAN class="toctext">Week of 5/20/13</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Week_of_5.2F27.2F13"><SPAN class="tocnumber">1.1.4</SPAN><SPAN class="toctext">Week of 5/27/13</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Week_of_6.2F3.2F13"><SPAN class="tocnumber">1.1.5</SPAN><SPAN class="toctext">Week of 6/3/13</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Week_of_6.2F10.2F13"><SPAN class="tocnumber">1.1.6</SPAN><SPAN class="toctext">Week of 6/10/13</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Week_of_6.2F17.2F13"><SPAN class="tocnumber">1.1.7</SPAN><SPAN class="toctext">Week of 6/17/13</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Week_of_6.2F24.2F13"><SPAN class="tocnumber">1.1.8</SPAN><SPAN class="toctext">Week of 6/24/13</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Week_of_7.2F1.2F13"><SPAN class="tocnumber">1.1.9</SPAN><SPAN class="toctext">Week of 7/1/13</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Week_of_7.2F8.2F13"><SPAN class="tocnumber">1.1.10</SPAN><SPAN class="toctext">Week of 7/8/13</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Week_of_7.2F15.2F13"><SPAN class="tocnumber">1.1.11</SPAN><SPAN class="toctext">Week of 7/15/13</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Week_of_7.2F22.2F13"><SPAN class="tocnumber">1.1.12</SPAN><SPAN class="toctext">Week of 7/22/13</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Week_of_7.2F29.2F13"><SPAN class="tocnumber">1.1.13</SPAN><SPAN class="toctext">Week of 7/29/13</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Week_of_8.2F5.2F13"><SPAN class="tocnumber">1.1.14</SPAN><SPAN class="toctext">Week of 8/5/13</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#Week_of_8.2F12.2F13"><SPAN class="tocnumber">1.1.15</SPAN><SPAN class="toctext">Week of 8/12/13</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#Week_of_8.2F26.2F13"><SPAN class="tocnumber">1.1.16</SPAN><SPAN class="toctext">Week of 8/26/13</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Week_of_9.2F2.2F13"><SPAN class="tocnumber">1.1.17</SPAN><SPAN class="toctext">Week of 9/2/13</SPAN></A></LI><LI class="toclevel-3 tocsection-20"><A href="#Week_of_9.2F9.2F13"><SPAN class="tocnumber">1.1.18</SPAN><SPAN class="toctext">Week of 9/9/13</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Overview"><B>Overview</B></SPAN></H1><H2><SPAN class="mw-headline" id="Duke_iGEM_Team_2013">Duke iGEM Team 2013 </SPAN></H2><P><B>Matt Baron, Matt Farnitano, Cameron Kim, HyunSoo Kim, Ashley Reid, Janan Zhu</B></P><H3><SPAN class="mw-headline" id="Week_of_2.2F21.2F13_.7E_3.2F18.2F13"><B>Week of 2/21/13 ~ 3/18/13</B></SPAN></H3><P>Thursday February 21- Matt Baron cultured the 10 NN E. Coli cultures 
Friday February 22- Matt Baron successfully mini prepped all NN plasmids (except 3 and 7) 
Sunday February 24-Matt Farnitano cultured the 10 NG, pfusA, and pfusA30A E. Coli 
Monday February 25-Matt Farnitano successfully mini prepped all 12 above plasmids 
Tuesday March 5- Cameron Kim cultured HD module plasmids and LR module plasmids 
Wednesday March 6 - HyunSoo Kim successfully mini prepped HD plasmids 
Thursday March 7- Cameron Kim cultured NN3 and NN7 E. Coli 
Friday March 8- Matt Baron successfully mini prepped plasmids NN3 and NN7 
Friday March 22- Ashley Reid miniprepped NI1-NI10 (threw out NI10) </P><H3><SPAN class="mw-headline" id="Week_of_5.2F13.2F13"><B>Week of 5/13/13</B></SPAN></H3><P>	Monday 5/13
</P><UL><LI>Matt Farnitano built plasmid iGEM.1 with Golden Gate reaction from backbone, ACT1 promoter, YEVenus reporter gene, and CYC1 terminator
</LI><LI>MF ran PCR reaction to amplify truncated ACT1 promoter
</LI><LI>MF assayed PCR product on agarose gel, with negative results. Product was discarded.
</LI></UL><P>	Tuesday 5/14
</P><UL><LI>Charlie Cooper ran PCR reaction for truncated ACT1p
</LI><LI>CC built plasmid iGEM.2 with Golden Gate reaction using truncated ACT1 promoter
</LI></UL><P>	Wednesday 5/15
</P><UL><LI>MF designed oligos for insert into iGEM.1 with binding sites for TALEss and CRISPR
</LI></UL><P>	Thursday 5/16
</P><UL><LI>MF miniprepped plasmids iGEM.1-1 through iGEM.1-4 and iGEM.2-1
</LI><LI>MF ran analytical digest of above plasmids with Kpn1 and Nsi1
</LI><LI>MF assayed digest product on agarose gel: positive results for iGEM.1-2 and iGEM.1-3 in both enzymes, positive results for iGEM.1-4 and iGEM.2-1 in only Kpn1, negative results for iGEM.1-1.
</LI><LI>MF plated E. coli cells with CRISPR plasmids from Church lab
</LI><LI>MF ran sequencing reaction for iGEM.1-2 and iGEM.1-3, sent results off for sequencing (Order 83)
</LI></UL><P>	Friday 5/17
</P><UL><LI>MF reran analytical digest of plasmids iGEM.1-4 and iGEM.2-1 in Nsi1
</LI><LI>MF assayed digest results on agarose gel: positive results for both
</LI><LI>MF ran sequencing reaction for plasmids iGEM.1-4 and iGEM.2-1, sent results off for sequencing (Order 88)
</LI><LI>MF modified and proofread oligo sequences from Wednesday
</LI></UL><H3><SPAN class="mw-headline" id="Week_of_5.2F20.2F13"><B>Week of 5/20/13</B></SPAN></H3><P>	Monday 5/20
</P><UL><LI>MF ran analytical digest of plasmids p667, p668, p669 in Bgl1 and Xba1
</LI><LI>MF assayed digest results on agarose gel: positive results for all lanes
</LI><LI>MF analyzed sequencing data from iGEM.1-2 and iGEM.1-3, confirmed iGEM.1-2 as a good match, discarded iGEM.1-3
</LI><LI>MF, MB and HK designed oligo sequences for 1,3,5 binding sites with 16, 20, 24 binding site lengths. (updated on iGEM Oligos file)
</LI></UL><P>	Tuesday 5/21
</P><UL><LI>MF and MB ran 1st cycle of pTAL synthesis to design four TALEss: bs16A, bs16B, bs20A, and bs20B
</LI><LI>MF and HK treated product of 1st cycle with Plasmid Safe DNAse
</LI><LI>MF and MB transformed plasmid into DH5alpha and plated on LB+Sp+XGal+IPTG
</LI><LI>MF designed and ordered 14 oligo sequences for Cas9 plasmid and gRNA plasmid assembly
</LI><LI>MF received sequencing data from 5/17, sequences were erroneous/bad quality
</LI></UL><P>	Wednesday 5/22
</P><UL><LI>MF and MB assembled ligation of pCC305 and pRS303H plasmids
</LI><LI>MB and MF performed restriction digest using Asc1 and Sal1 on pCC305 and pRS303H plasmids to make TALEs plasmid
</LI><LI>MB and MF ran colony PCR for pTAL colonies bs16a, bs16b, bs20a, and bs20b and inoculated colonies in SOC media
</LI><LI>MB and MF ran sequencing reaction for iGEM.2-2
</LI><LI>MB, MF, and HK performed ligation of pCC305 and pRS303H to make TALEs plasmid
</LI><LI>MF ran bs16a, bs16b, bs20a, and bs20b on gel and corresponding TAL bands not present on gel
</LI><LI>MB, MF, and HK re-ran colony PCR for pTAL colonies bs16a, bs16b, bs20a, and bs20b and inoculated colonies in SOC media
</LI></UL><P>	Thursday 5/23
</P><UL><LI>MB, MF, and HK ran bs16a, bs16b, bs20a, and bs20b TAL plasmids on gel and bands not present again. TAL reaction needs to be re-run.
</LI><LI>MB, MF, HK re-designed the BS16a, BS16b, BS20a, BS20b oligos to same lengths (inserted non-binding sequences at the end for 1 BS and 3 BS oligos to be the same lengths as the 5 BS ones)
</LI><LI>MB, MF, HK checked the oligo sequences if there were any other TF’s that would bind to the BS’s. 
</LI><LI>MB and MF re-ran 1st cycle of pTAL synthesis to design four TALEss to match the four binding sites: BS16a, BS16b, BS20a, and BS20b
</LI><LI>MF and JZ transformed bacteria cells with the product from pTAL synthesis and plated on LB+Spec+Xgal 
</LI><LI>MB and HK transformed bacteria cells with the ligated plasmid (from pCC305 and pRS303H) and plated on LB+Amp 
</LI><LI>MF designed gBLOCKs (?) instead of oligos for the binding site, and designed two new binding sites BS16c and BS20c.
</LI></UL><P>	Friday 5/24
</P><UL><LI>Transformation with ligated plasmid (from pCC305 and pRS303H) was unsuccessful → have to regrow the two plasmids and run PCR (on Monday) 
</LI><LI>HK and JZ ran 1st cycle of golden gate reaction for bs16c and bs20c TAL
</LI><LI>Transformed E.coli and incubated overnight
</LI><LI>MB and HK ran colony PCR for bs16a, bs16b, bs20a, and bs20b TAL plasmids
</LI><LI>MB and HK ran gel for bs16a, bs16b, bs20a, and bs20b PCR products. Poor PCR. Re-ran reaction on 5/28.
</LI><LI>MF ran 1st reaction with oligos for gRNA and Z4EV
</LI><LI>Gel results not good (probably because the melting temperature on the oligo data sheet was different from ours &amp; because the reaction duration was miscalculated)
</LI><LI>MF re-ran 1st reaction with oligos for gRNA and Z4EV. 
</LI></UL><P>	Saturday 5/25
</P><UL><LI>CK ran colony PCR for 16cA, 16cB, 20cA, and 20cB. Identified positive clones for 16cA, 16cB and 20cB (not 20cA). Clones incubated overnight. 
</LI></UL><H3><SPAN class="mw-headline" id="Week_of_5.2F27.2F13"><B>Week of 5/27/13</B></SPAN></H3><P>Monday 5/27
</P><UL><LI>MB re-ran colony PCR for 16aA, 16aB, 16bA, 16bB, 20aA, 20aB, 20bA, and 20bB
</LI><LI>gel results mixed--potential success for 20Ba, 16Ba, 16Ab, 16Bb
</LI></UL><UL><LI>MF did PCR clean up for gRNA and Z4EV--lost both
</LI><LI>MF ran PCR reaction with oligos to amplify gRNA and Z4EV
</LI><LI>MF lost gRNA PCR product during PCR clean up procedure for unknown reasons
</LI></UL><UL><LI>MF did analytical digest of 4 pRS303H-v2 plasmids with Pst1 and Nde1
</LI><LI>HK ran pR303H-v2 digest on gel → unclear results (correct for only Pst1)
</LI><LI>MF did analytical digest of pRS303H-v2 and old pRS303H with EcoR1
</LI><LI>results confirm that plasmid pRS303H-v2 is as expected
</LI></UL><UL><LI>HK ran TAL1 reaction for 16Aa, 20Ab, 20Bb, 20Ca (need to add Ligase, BsaI, Ligase buffer tomorrow because the lab is out of BsaI)
</LI><LI>MF made a stock of LB+Spec to grow successful/potentially successful TAL colonies
</LI><LI>HK labelled 20 tubes and filled with LB+spec for the 20 successful/potentially successful TAL colonies → incubated
</LI><LI>Successful: Set 1 20Aa(1,2,3), Set 3 20Ba(1,2), Set 3 16Ba(2), Set 4 20Ba(1), Set 4 16Ba(1,2,3)
</LI><LI>Potentially: Set 1 16Bb(1,2,3), Set 3 16Ab(1,3), Set 3 16Bb(1), Set 4 16Ab(1,3), Set 4 16Bb(1,3)
</LI></UL><P>	Tuesday 5/28
</P><OL><LI>Successful TAL
</LI><LI>MF mini-prepped successful and potentially successful TAL colonies 
</LI><LI>Sent off for sequencing
</LI><LI>Unsuccessful TAL (re-do)
</LI><LI>HK and MB finished TAL1 reaction for 16Aa, 20Ab, 20Bb, and 20Ca
</LI><LI>HK and JZ transformed the cells
</LI><LI>MF and MB plated the cells
</LI><LI> Z4EVpr and Backbone for TALE
</LI><LI>MB performed restriction digest of pCC305 and pRS303H with Asc1 and Sal1 to prepare for ligation of the two plasmids
</LI><LI>MF, HK, JZ ran them on gel 
</LI><LI>Positive result for pCC303H but negative for pCC305 (it was actually pCC317, and we will re-do tomorrow)
</LI><LI>  gRNA Backbone and Z4EVpr for CRISPR
</LI><LI>MF ran PCR for the gRNA backbone and Z4EVpr
</LI><LI>MB ran the products on gel 
</LI><LI>Bad results. Need to re-do PCR 
</LI></OL><P>	Wednesday 5/29
</P><OL><LI>Z4EVpr and Backbone for TALE
</LI><LI>MB performed restriction digest of pCC317 with Asc1 and Sal1
</LI><LI>MF cut gel, but unexpected third band still present
</LI><LI>JZ performed restriction digest of pCC305 with Asc1 and Sal1
</LI><LI>MF cut gel, unexpected third band present
</LI><LI>MB performed gel extraction on pCC317 and p303H
</LI><LI>MB set up ligation reaction of pCC317 and p303H pieces to incubate overnight
</LI><LI>gRNA Backbone and Z4EVpr for CRISPR
</LI><LI>MF ran PCR for gRNA backbone and Z4EVpr
</LI><LI>MF, MB ran gel--positive results for Z4EV, negative for gRNA
</LI><LI>MF, JZ reran PCR for gRNA backbone
</LI><LI>MF ran gel for gRNA backbone--positive results
</LI><LI>MF performed PCR cleanup procedure, nanodrop showed no DNA afterwards
</LI><LI>   Unsuccessful TALE (re-do)
</LI><LI>Plates from 5/29 had no colonies--potential problem was mistaking T4 ligase buffer for its enzyme
</LI><LI>MB and HK ran TAL1 reaction for 16Aa, 20Ab, 20Bb, and 20Ca
</LI><LI>MF ran TAL1 reaction for 2x20Cb (negative result from sequencing), 16Aa, 20Ab, 20Bb, 20Ca
</LI><LI>JZ ran plasmid-safe DNase reaction for first TAL set
</LI><LI>MF ran plasmid-safe DNase reaction for second TAL set
</LI><LI>HK transformed both sets into competent E. coli with heat shock
</LI><LI>MF plated transformed cells to culture overnight
</LI><LI>   Sequencing results for TALEs
</LI><LI>MF, MB checked results from 5/29 sequencing
</LI><LI>16Ab, 16Ba, 16Bb, 16Ca, 16Cb, 20Aa, 20Ba all successful
</LI><LI>20Cb had RVDs in wrong order/extra RVDs, needs to be redone
</LI><LI>20Cb redone with the other unsuccessful TALEs (plated and incubated overnight)
</LI></OL><P>Thursday May 30
</P><OL><LI>   gRNA backbone and Z4EVpr for CRISPR
</LI><LI>MF performed PCR for gRNA backbone and Z4EVpr
</LI><LI>Positive results on gel
</LI><LI>PCR clean-up successful
</LI><LI>Performed SLIC reaction with gRNA and insert oligos
</LI><LI>Transformed E.coli with SLIC product and incubated overnight
</LI><LI> TALE 
</LI><LI>MF, MB, and HK performed colony PCR for 16Aa, 20Ab, 20Bb, 20Ca, and 20Cb
</LI><LI>Gel result showed no TALEs (twice, possibly bad colony PCR)
</LI><LI>Will redo starting from colony picking &amp; PCR (then gel, grow in SOC+store cells in -80)
</LI><LI>Z4EVpr and Backbone for TALEss
</LI><LI>Need to transform ligation and control into E. coli and plate overnight
</LI><LI>Transformed E.coli with ligated (iGEM3 and control) and incubated overnight
</LI><LI>G-Blocks
</LI><LI>MF and MB redesigned G-Blocks to minimize repeats so IDT can make the G-Blocks
</LI></OL><P>
Friday May 31
</P><OL><LI> Z4EVpr and Backbone for TALEss
</LI><LI>Both control and insert plates did not have any colonies
</LI><LI>Have to redo 
</LI><LI> TALE
</LI><LI>Performed colony PCR (JZ, HK for first set, CK for second set)
</LI><LI>Gel results for both sets showed that 4 out of the 5 worked
</LI><LI>CK cultured the four and will miniprep them on saturday
</LI><LI>Have to redo 20Ab Golden Gate 1st reaction
</LI><LI>  gRNA backbone and Z4EVpr for CRISPR
</LI><LI>Plates have colonies and the control did not have any
</LI><LI>Growing colonies in shaker overnight
</LI><LI>CK will miniprep them on saturday
</LI><LI>gRNA Oligos for 16C and 20C came in
</LI><LI>diluted to 100uM and stored 
</LI></OL><P>Saturday June 1
</P><OL><LI>  TALE
</LI><LI>CK miniprepped 16Aa, 20Bb, 20Ca, 20Cb plasmids
</LI><LI> gRNA backbone for CRISPR
</LI><LI>CK miniprepped pNB669-bs16A, pNB669-bs16B, pNB669-bs20A, pNB669-bs20B
</LI></OL><H3><SPAN class="mw-headline" id="Week_of_6.2F3.2F13"><B>Week of 6/3/13</B></SPAN></H3><P>Monday June 3
</P><OL><LI>  TALE
</LI><LI>MF ran sequencing reaction for 16Aa, 20Bb, 20Ca, 20Cb plasmids with primers PCR8-F and PCR8-R, sent off for sequencing (order 150)
</LI><LI>MB ran 1st cycle to build 20Ab TAL
</LI><LI>JZ ran plasmid-safe DNase treatment
</LI><LI>MB transformed 20Ab into competent E. coli and plated to incubate overnight
</LI><LI> gRNA plasmids
</LI><LI>MF ran sequencing reaction for 16A, 16B, 20A, 20B gRNA plasmids (pNB669-) with primer pRSup, sent off for sequencing (order 150)
</LI><LI>MF annealed gRNA 16C up and down oligos, and 20C up and down oligos
</LI><LI>MF performed SLIC cloning to insert 16C and 20C binding sites into linearized pNB669 backbone
</LI><LI>Used annealed gRNA oligos as inserts
</LI><LI>First set of reactions added oligos immediately after T4 polymerase
</LI><LI>Second set of reactions added oligos later, before icing reaction
</LI><LI>MF transformed SLIC products into competent E. coli via heat shock
</LI><LI>CC performed ligation of TAL landing pad, with negative results
</LI></OL><P>	Tuesday June 4
</P><OL><LI>   TALE
</LI><LI>MF ran colony PCR on 3 colonies each from two sets of 20Ab
</LI><LI>MF, MB ran results on gel and showed successful 
</LI><LI>Need to culture a colony overnight so we can mini-prep and send off for sequencing
</LI><LI>JZ, MF ran second cycle (TAL2) on 16A, 16B, 20B, 16C, 20C TALs
</LI><LI>MB, JZ transformed and plated on LB+Amp+XGal+IPTG for 16A, 16B, 20B, 16C, and 20C
</LI><LI> gRNA plasmids
</LI><LI>Only a few to no white colonies on each plate, but will transform to see, also try Gibson
</LI><LI>MF annealed 16C and 20C oligos for use in Gibson assembly
</LI><LI>MF ran Gibson assembly reactions for 16C and 20C gRNAs with 1uL and 2uL annealed oligos
</LI><LI>MB, JZ transformed and plated on LB+Amp for 16C and 20C gRNAs into chemically competent DH5Alphas
</LI><LI> Reporter plasmid
</LI><LI>MB ran Golden Gate reaction to create two new reporter plasmids with different promoters--ACTi (truncated) and ACTpr. Each has yEVenus reporter, CYC1 transcriptional terminator, and (pRS305?) backbone.
</LI><LI>MB, JZ transformed and plated on LB+Amp both plasmids into chemically competent DH5Alphas
</LI><LI>Competent cells
</LI><LI>MF, MB, JZ, CC made chemically competent E. coli from DH5Alphas
</LI></OL><P>	Wednesday June 5
</P><OL><LI>  TALE
</LI><LI>Plates for 16A, 16B, 16C, 20B, and 20C have lots of colonies. Hard to grab a white colony amid all the blue colonies
</LI><LI>MB ran colony PCR for 16A, 16B, 16C, 20B, and 20C
</LI><LI>MF mini-prepped DNA from 20Ab
</LI><LI>MF ran sequencing reaction for 20Ab and sent off for sequencing
</LI><LI>MB ran 2nd cycle reaction for 20A
</LI><LI>MB ran gel for colony PCR, only 20B looked promising (1)
</LI><LI>MF, JZ ran colony PCR again with new colonies for 16A, 16B, 16C, 20B, and 20C
</LI><LI>MB, JZ ran gel for re-run of colony PCR. Positive for only 16B (1 and 2).
</LI><LI>MB cultured successful colonies overnight from colony PCR overnight: 16B and 20B
</LI><LI>MF, JZ re-ran 2nd cycle TAL assembly for 16A, 16B, 16C, 20A, 20B, and 20C
</LI><LI>MB transformed and plated re-run of 2nd cycle assembly for 16A, 16B, 16C, 20A, 20B, and 20C on LB+Amp+XGal+IPTG
</LI><LI>gRNA
</LI><LI>MF mini-prepped gRNA DNA for 16C and 20C from SLIC cloning
</LI><LI>MF ran sequencing reaction for gRNA 16C and 20C from SLIC cloning and sent off for sequencing
</LI><LI>Plates for gRNA DNA for 16C and 20C from Gibson cloning have few colonies.
</LI><LI>MB cultured colonies overnight for 16C and 20C 
</LI><LI> Reporter Plasmid
</LI><LI>Plates for reporter plasmid with full ACT and ACTi promoter have lots of white colonies
</LI><LI>MB cultured colonies overnight for full ACT and truncated ACTi promoter
</LI><LI>Binding Sites
</LI><LI>MF redesigned oligos for 5x binding sites
</LI></OL><P>	Thursday June 6
</P><OL><LI>TALEs
</LI><LI>MF ran colony PCR for 16A, 16C, 20A, 20C TALEs, with three colonies each
</LI><LI>MB ran gel for colony PCR for 16A, 16C, 20A, and 20C TALEs
</LI><LI>Positive results for 20B only (3rd lane)
</LI><LI>Colony from 20B cultured overnight
</LI><LI>MB mini-prepped 16B (2 copies) and 20B
</LI><LI>MB performed sequencing reaction for TALs 16B and 20B and sent off for sequencing using TAL-F1, TAL-R2, 306 Ura3p, and 306MCSup oligos
</LI><LI>MF, JZ ran gel again for 16A colony PCR
</LI><LI>Positive results for 1 and 3
</LI><LI>Colonies from 16A cultured overnight (both attempts)
</LI><LI>MF analyzed sequencing results for orders 150 and 165: TALs 16Aa, 20Bb, 20Ca, 20Cb, and 20Ab all tested positive
</LI><LI>MF, JZ ran colony PCR again for 16A, 16C, 20C (3 new colonies each)
</LI><LI>MB, MF ran gel--successful only for 16A (already had successes)
</LI><LI>  gRNA
</LI><LI>MB mini-prepped Gibson results for 16C and 20C (two copies each)
</LI><LI>MF analyzed sequencing results for orders 150 and 165: 
</LI><LI>gRNAs 16A, 16B look good
</LI><LI>gRNA 20A has single point mutation in promoter
</LI><LI>gRNA 20B has single point mutation in structural gRNA
</LI><LI>gRNA 16C showed up as original p669 plasmid
</LI><LI>gRNA 20C showed up as 16C--need to test where labelling went wrong
</LI><LI>   Reporter Plasmid
</LI><LI>MB mini-prepped reporters with full ACT promoter and truncated ACTi promoter
</LI><LI>MB performed sequencing reaction for ACTi and ACTpr reporters with yEVenus seq1, yEVenus seq2, 306 Ura3p, and 306MCSup oligos. 
</LI></OL><P>	Friday June 7
</P><OL><LI>  TALEs
</LI><LI>MB successfully mini-prepped DNA for 16A and 20A
</LI><LI>MF ran sequencing reaction for DNA for 16A and 20A
</LI><LI>MF ran colony PCR for 16C and 20C
</LI><LI>MB ran gel for colony PCR for 16C and 20C (positive for 16C)
</LI><LI>MB cultured positive colony PCR result for 16C in LB+Amp overnight
</LI><LI>MF re-ran colony PCR for 20C
</LI><LI>MF ran gel for colony PCR rerun: potential success for lanes 1, 2, and 4
</LI><LI>gRNA
</LI><LI>MF re-ran sequencing reaction for 16C and 20C from SLIC and 16C and 20C from Gibson cloning
</LI><LI>MF re-ran SLIC reaction for gRNA 20A and 20B 
</LI><LI>MF, MB transformed SLIC reaction into competent E. coli, plated overnight
</LI></OL><P>	Saturday June 8
</P><OL><LI>   TALEs
</LI><LI>MF successfully mini-prepped DNA for TALs 16C and 20C (only 16C, 20C-1 had grown culture)--concentrations 327.6 and 651.4 ng/uL
</LI><LI>gRNA
</LI><LI>MF cultured 2 colonies each from SLIC 20A and 20B plates in LB+amp
</LI></OL><P>	Sunday June 9
</P><OL><LI>gRNA
</LI><LI>MB successfully mini-prepped DNA from SLIC 20A and 20B
</LI><LI>Yeast
</LI><LI>MB plated yeast
</LI><LI>MB cultured yeast in shaker overnight
</LI></OL><H3><SPAN class="mw-headline" id="Week_of_6.2F10.2F13"><B>Week of 6/10/13</B></SPAN></H3><P>	Monday June 10
</P><OL><LI>TALEs
</LI><LI>MB, MF ran sequencing reaction for TALs 16C and 20C and sent off for sequencing
</LI><LI>gRNA
</LI><LI>MB, MF ran sequencing reaction for gRNA 20A and 20B and sent off for sequencing
</LI><LI>Reporter
</LI><LI>MB ran PCR of both reporter plasmids (iGEM.4 and iGEM.5) at 1/500 concentration (0, 03. 0.6, and 0.9 uL plasmid) to get portion of reporter that will be homologous with yeast genome in order to transform into yeast. Used oligos Knopf-3pLeu2 and Knopf-5pLeu2 with corresponding sequences TCGTCTACCCTATGAACATATTCCA and ACCATGTAACTTTGCATCCGA. Initial melting temp of 98 C for 1 min then 35x repeat of 98 C for 10 sec, 66 C for 20 sec, then 72 C for 2:15 min. Final extension at 72 C for 5 min then infinite hold at 4 C.
</LI><LI>MB, MF ran PCR for iGEM.4 and iGEM.5 on gel and showed that PCR failed. 
</LI><LI>MF re-ran PCR of both reporter plasmids (iGEM.4 and iGEM.5) at 1/300 concentration (0, 0.3, 0.6, 0.9 uL plasmid) to get portion of reporter that will be homologous with yeast genome in order to transform into yeast. Used oligos Knopf-3pLeu2 and Knopf-5pLeu2 with corresponding sequences TCGTCTACCCTATGAACATATTCCA and ACCATGTAACTTTGCATCCGA. Initial melting temp of 98 C for 1 min then 35x repeat of 98 C for 10 sec, 66 C for 20 sec, then 72 C for 2:15 min. Final extension at 72 C for 5 min then infinite hold at 4 C.
</LI><LI>MB, MF ran PCR for iGEM.4 and iGEM.5 on gel and showed that PCR failed. Need to look at sequencing data for iGEM.4 and iGEM.5 to determine what issues may exist
</LI><LI>Binding Sites
</LI><LI>MF ran PCR for 6 oligos that overlap to form 5x binding sites for 16A and 16C. Used 0.25 uL each oligo in one trial and 0.5 uL each in second trial. Initial melting at 98 C (1 min). Melting at 98 C (20 sec), 16A was annealed at 64 C and 16C was annealed at 66C (20 sec), and extension for both at 72 C (15 sec). Repeat melting, annealing, and extension 34 times.
</LI><LI>MF, JZ ran PCR results on gel. Three bands at ~200 (desired) and lower, lower bands were brighter. Will try again with new PCR protocol
</LI><LI>MB ran PCR as before with 0.5 uL each oligo, gradient of annealing temperatures from 64 C to 72 C and 15 sec extension at 72 C.
</LI><LI>MB, MF ran PCR results on gel. Similar banding as previous run for both 16A and 16C. Going to try to insert into iGEM.4 and iGEM.5 to make 5x binding site reporter plasmid.
</LI><LI>MF performed a PCR clean-up on 5x binding sites 16A and 16C
</LI><LI>Yeast
</LI><LI>Due to failed PCR of reporter, we cannot perform the transformation of the reporter into yeast today.
</LI><LI>MB disposed of overnight yeast culture in bleach
</LI></OL><P>
	Tuesday June 11- **We received dCas9 from Pablo today!!!**
</P><OL><LI>Reporter
</LI><LI>MB, MF performed restriction digest of iGEM.4 with SpeI in order to allow for insertion of the 5x binding sites 16A and 16C
</LI><LI>MF treated linearized reporter with CIP to prevent recircularization
</LI><LI>MF performed PCR cleanup on linearized reporter
</LI><LI>MB transformed iGEM.4 into DH5alphas and plated on LB+Amp
</LI><LI>MB cultured iGEM.4 in LB+Amp and put in 37 degree shaker overnight
</LI><LI>dCas9
</LI><LI>MB performed restriction digest of 303H plasmid with AscI and BamHI-HF to prepare for insertion of Z4EVpr and dCas9 gene
</LI><LI>Gel results: 
</LI><LI>MF performed PCR of dCas9 gene to prepare for SLIC reaction with Z4EVpr and dCas9
</LI><LI>JZ, MF ran on gel--large smear and vector showed up
</LI><LI>JZ performed PCR again with 1/200 diluted dCas9 (1 ng/uL)
</LI><LI>MB ran on gel and showed successful PCR of dCas9
</LI><LI>Binding sites
</LI><LI>MF saved 5 uL each of PCR results for bs5x16A and bs5x16C, ran 45 uL each on 1.3% TBE gel for extraction of top band
</LI><LI>Top band hardly visible, gel extraction not continued
</LI><LI>MF ran Gibson assembly protocol to assemble linearized reporter with bs5x16A and bs5x16C PCR results, 1 uL each vector and insert
</LI><LI>MB set up CPEC cloning to assemble 3 uL linearized reporter with bs5x16A and bs5x16C (6 oligos each--0.25 uL 100uM each oligo). Annealled at 65C, 30 sec extension time, 10 repetitions. 
</LI><LI>MB transformed Gibson assembly of BS16A and BS16C with iGEM.4 into chemically competent DH5alphas and plated on LB+Amp
</LI><LI>MB transformed CPEC assembly of BS16A and BS16C with iGEM.4 into chemically competent DH5alphas and plated on LB+Amp
</LI></OL><P>	Wednesday June 12
</P><OL><LI> Reporter
</LI><LI>MB successfully mini-prepped iGEM.4 from the cells cultured overnight in LB+Amp
</LI><LI>MB ran analytical digest with EcoRI and HindIII to confirm iGEM.4 (expecting bands of approx 3 kb and 5 kb)
</LI><LI>MB cultured colony overnight from iGEM.4 plate
</LI><LI>  dCas9
</LI><LI>MF ran another PCR for the dCas9 gene in order to have enough PCR product, with 0.6 uL vector per tube, 4 tubes
</LI><LI>MF ran on gel--positive, clean results
</LI><LI>MF performed PCR cleanup on dCas9 (only most recent PCR)
</LI><LI>MF performed SLIC cloning with pRS303H-Z4EV-dCas9
</LI><LI>MF, MB transformed SLIC in E. Coli via heat shock and plated (single plate)
</LI><LI>  TALEs
</LI><LI>Sequencing data revealed an incorrect sequence for TALE 20C
</LI><LI>MF ran colony PCR for TALE 20C
</LI><LI>MB ran on gel-negative results and decided plate was of poor quality
</LI><LI>MF ran new second cycle reaction for TALE 20C
</LI><LI>Two with 20Cb-1 and two with 20Cb-2
</LI><LI>MF, MB transformed second cycle reaction via heat shock and plated (4 plates)
</LI><LI>  Binding Sites
</LI><LI>Plates from CPEC assembly of binding sites in the reporter have only one colony
</LI><LI>Plates from Gibson assembly of binding sites in the reporter have a few colonies
</LI><LI>MB cultured colonies overnight in LB+Amp to mini-prep tomorrow--2 colonies each of Gibson for 5x16A and 5x16C, one CPEC colony for 5x16C
</LI><LI> gRNA
</LI><LI>MF performed sequencing reaction for gRNA 20B (1 and 2) and 20C (1b and 2) and sent off for sequencing (Order 208)
</LI></OL><P>	Thursday June 13
</P><OL><LI>  TALEs
</LI><LI>MB ran colony PCR for two colonies from each of the four 20C plates
</LI><LI>MB ran gel for colony PCR and there were two successful results, from two separate plates 3 and 4 (which are both with 20Cb-2). Both were cultured overnight in LB+Amp
</LI><LI>MF ran another colony PCR with two colonies each from plates 1 and 2
</LI><LI>MF ran on gel--negative results
</LI><LI>JZ ran restriction digest of TALEs plasmids with Sap1 and Nde1 in Cutsmart buffer for yeast transformation
</LI><LI>  Binding Sites
</LI><LI>MF stored glycerol stocks and mini-prepped DNA from the CPEC assembly of 5x16c and the Gibson assemblies of 5x16A and 5x16C
</LI><LI>MF ran an analytical restriction digest of mini-prepped DNA from CPEC assembly of 5x16C and the Gibson assemblies of 5x16A and 5x16C with MfeI and BamHI to confirm insertion of binding sites in iGEM.4
</LI><LI>All four Gibsons looked potentially good--hard to tell if binding sites are the correct length; CPEC was not good
</LI><LI>MF performed sequencing reaction of Gibson assemblies of 5x16A and 5x16C and sent off for sequencing
</LI><LI>  Reporter
</LI><LI>MF stored a glycerol stock and mini-prepped DNA for iGEM.4
</LI><LI>MB ran an analytical restriction digest of mini-prepped iGEM.4 with EcoRI-HF and HindIII-HF to confirm plasmid
</LI><LI>JZ ran restriction digest of iGEM.4 with Afe1 and Nhe1-HF in Cutsmart buffer for yeast transformation.
</LI><LI>  dCas9
</LI><LI>MB cultured two colonies separately from the SLIC cloning of dCas9 gene with 303H backbone and Z4EVpr
</LI></OL><P>	Friday June 14
	  dCas9
</P><OL><LI>MB saved glycerol stocks and mini-prepped DNA from the SLIC cloning of 303H, Z4EV, and dCas9
</LI><LI>Need to perform analytical restriction digest and sequencing reaction but still need plasmid map from Pablo and the correct oligos for sequencing reaction
</LI><LI>  TALEs
</LI><LI>MB saved glycerol stocks and mini-prepped DNA for TAL 20C (20C-3 and 20C-4)
</LI><LI>MB ran sequencing reaction for 20C-3 and 20C-4 using oligos TAL F-1, TAL R-2, 306 MCS-up, and 306 ura3p
</LI><LI>MF ran TALs 16A, 16B, 16C, 20A, 20B, and 20C on a gel after they had been linearized with SapI and NdeI in order to prepare them for transformation in the yeast genome (should be three bands: ones at 1.3 kb, 2.3 kb, and 6.5 kb). Digest appeared incomplete.
</LI><LI>MF performed restriction digest of TALs 16A, 16B, 16C, 20A, 20B, and 20C-3 using SapI and NdeI in order to prepare them for transformation in the yeast genome (should be three bands: ones at 1.3 kb, 2.3 kb, and 6.5 kb)
</LI><LI>MF put yesterday’s restriction digests back in the 37C bath with today’s digests to obtain complete digests
</LI><LI>20A and 16A TALEs show unexpected band lengths, other 4 are good and ready for yeast integration.
</LI><LI>  gRNA
</LI><LI>MF performed restriction digest of p669 linearized gRNA backbone with DpnI to cut up any remaining full gRNA plasmid with old binding site insert. We noticed that sequencing data for gRNA 20C kept having the old insert even though we thought we had run a PCR that removed that sequence.
</LI><LI>  Reporter
</LI><LI>MF ran iGEM.4 on a gel after it had been linearized with NheI and AfeI in order to prepare it for transformation in the yeast genome (should be two bands: one at 3.5 kb and the other at 4 kb but looks like one big band covering the 3.5-4 range)
</LI><LI>MF performed restriction digest  of iGEM.4 using NheI to confirm size of plasmid
</LI><LI>Plasmid size confirmed, so first digest with Nhe1 and Afe1 can be used for yeast integration
</LI></OL><H3><SPAN class="mw-headline" id="Week_of_6.2F17.2F13"><B>Week of 6/17/13</B></SPAN></H3><P>Monday June 17
</P><OL><LI>  G-blocks arrived!/Binding Site Insertion in Reporter
</LI><LI>MF added 200uL water to each tube to obtain 1 ng/uL concentration
</LI><LI>MB performed Gibson assembly for 1x-(16A, 16B, 16C, 20A, 20B, and 20C) and 3x-(16A, 16B, 16C, 20A, 20B, and 20C) with 0.5 uL linearized iGEM.4 (0.015 pmol) and 7.2 uL G-block (0.045 pmol)
</LI><LI>MF transformed the Gibson assemblies of 1x-(16A, 16B, 16C, 20A, 20B, and 20C) and 3x-(16A, 16B, 16C, 20A, 20B, and 20C) with linearized iGEM.4
</LI><LI> gRNA
</LI><LI>MF performed SLIC for gRNA 20C with p669 digested with DpnI (3 different insert concentrations)
</LI><LI>MF transformed the 3 different insert concentrations of gRNA 20C 
</LI><LI>  dCas9
</LI><LI>MF designed and ordered oligos in order to perform PCR of dCas9 in order to insert into pCC305 
</LI><LI> TALEs
</LI><LI>MF transformed remaining DNA for 16A, 16B, 16C, 20A, and 20B in order to make more and save glycerol stocks
</LI><LI>Yeast
</LI><LI>MF cultured yeast (DBY12397) overnight in order to being yeast transformations tomorrow with linearized iGEM.4 and TALEs 
</LI></OL><P>	Tuesday June 19
</P><OL><LI>TALEs
</LI><LI>Plates for 16A, 16B, 16C, 20A, and 20B have a lawn of bacteria. 
</LI><LI>MF ran colony PCR on two smears from each plate
</LI><LI>MB ran results on gel and all colonies appear successful
</LI><LI>JZ, MF cultured 1 colony per TALE from SOC tubes in order to mini-prep and save glycerol stocks tomorrow
</LI><LI>  gRNA
</LI><LI>Plates for gRNA 20C appear to have few to no colonies. Going to try Gibson cloning today
</LI><LI>MF annealed 3 uL each of 20C oligos in order to Gibson clone
</LI><LI>MB performed Gibson assembly of gRNA 20C with unprepped vector--3 copies
</LI><LI>MF performed Gibson assembly of gRNA 20C with PCR-prepped vector--3 copies, annealled oligo volumes 0.25uL, 0.5uL, 1uL, plus no oligo control
</LI><LI>MF transformed 7 Gibson reactions into E. coli via heat shock
</LI><LI>MB, JZ plated transformations on LB+amp
</LI><LI>JZ, MF cultured 2 colonies from 20C SLIC plate overnight to mini-prep
</LI><LI>  Binding Sites
</LI><LI>Plates have lots of colonies for 1x-(16A, 16B, 16C, 20A, 20B, and 20C) and 3x-(16A, 16B, 16C, 20A, 20B, and 20C) with linearized iGEM.4. 
</LI><LI>JZ, MB cultured colonies overnight in order to mini-prep DNA and send off for sequencing
</LI><LI>MF performed SLIC reaction with un-PCR-ed oligos for 5x16A and 5x16C binding sites in linearized iGEM.4 vector. One tube per binding site plus one control
</LI><LI>MF transformed 3 SLIC reaction into E. coli via heat shock
</LI><LI>MB, JZ plated transformations on LB+amp
</LI><LI>  Yeast
</LI><LI>MF, MB transformed 4 plasmids (TALEs 16B, 16C, 20B, 20C) into yeast to make strains 21, 22, 24, and 25 plus a control (strain 0).
</LI><LI>MF, MB, JZ plated 5 transformations on YPD+G418 
</LI></OL><P>	Wednesday June 19
</P><OL><LI>  gRNA
</LI><LI>Don’t appear to be any colonies on the gRNA 20C plates (1-6). Need to reevaluate that binding site to see if possible
</LI><LI>MB, MF successfully saved glycerol stocks and mini-prepped DNA from gRNA 20C (1 and 2)
</LI><LI>MB, JZ performed sequencing reaction for gRNA 20C-1 and 20C-2 using oligos pRS-up and sent off for sequencing
</LI><LI>  TALEs
</LI><LI>MB, MF successfully saved glycerol stocks and mini-prepped DNA from TALEs 16A, 16B, 16C, 20A, and 20B
</LI><LI>MB, MF performed an analytical digest of TALEs 16A, 20A, pCC305, 16Aa, 16Ab, 20Aa, and 20Ab with XbaI and MfeI
</LI><LI>MF ran digested TALEs 16A and 20A on gel and they are the correct size pieces
</LI><LI>  Reporter with Binding Sites
</LI><LI>MB, MF successfully saved glycerol stocks and mini-prepped DNA from iGEM.4 with BS 1x-(16A, 16B, 16C, 20A, 20B, and 20C) and 3x-(16A, 16B, 16C, 20A, 20B, and 20C). 2 separate stocks and mini-preps for each BS.
</LI><LI>MB, MF performed an analytical digest of both copies of BS 1x-(16A, 16B, 16C, 20A, 20B, and 20C) and 3x-(16A, 16B, 16C, 20A, 20B, and 20C) in iGEM.4 with BamHI and MfeI
</LI><LI>MF ran digest reporters with binding sites on a gel and it appears as if all g-block insertions were successful
</LI><LI>MB, JZ performed sequencing reaction for BS 1x-(16A, 16B, 16C, 20A, 20B, and 20C) and 3x-(16A, 16B, 16C, 20A, 20B, and 20C) in iGEM.4 using oligos yEVenus-seq 1 and 303-MCSup and sent off for sequencing.
</LI><LI>MB cultured two colonies from BS 5x-16A and one from BS 5x-16C overnight in order to mini-prep
</LI><LI>  Yeast
</LI><LI>Plates from transformation with TALEs 16B, 16C, 20B, and 20C are growing for one more day
</LI><LI>MB cultured a colony of DBY12397 in YPD overnight in order to transform TALEs 16A and 20A tomorrow
</LI></OL><P>	Thursday June 20
</P><OL><LI> TALEs
</LI><LI>MF ran restriction digest of TALEs 16A and 20A with SapI and NdeI in order to prepare them for transformation into yeast. 
</LI><LI>Gel results after 1 hr indicate unknown problem with TALEs 16A and 20A (extra bands)
</LI><LI>MF ran digest for 3.5 hrs total to check for incomplete digestion--extra bands still present
</LI><LI>MF ran analytical digest of TALEs 16A and 16B in NEBuffers 2 and 3 with Thermo’s AdeI (DraIII)--16A had extra band, 16B looked as expected, both buffers showed equal results
</LI><LI>JZ, MB transformed TALEs 16A and 20A in order to colony purify
</LI><LI>JZ, MB transformed pCC305 to save permanent stock
</LI><LI>Yeast
</LI><LI>Unable to transform TALEs 16A and 20A into DBY12397 because of unexpected results of digest with SapI and NdeI (5 bands instead of 3).
</LI><LI>Unable to pick transformants from plates for TALEs 16B, 16C, 20B, and 20C. Need to let them grow another day
</LI><LI>Reporter with Binding Sites
</LI><LI>MB mini-prepped iGEM.4 + BS 5x-16A and two copies of iGEM.4 + BS 5x-16C and saved glycerol stocks
</LI><LI>MB performed analytical digest of BX 5x-16A and Bs 5x-16C with BamHI and MfeI and ran on gel. (Bands indicated successful insertion of binding sites)
</LI><LI>MB performed sequencing reaction for 5x16A and 5x16C with yEVenus-seq1 and 303MCSup, sent off for sequencing
</LI><LI>	dCas9
</LI><LI>MB performed PCR using new oligos for pCC305 backbone and dCas9 gene. 
</LI><LI>pCC305 anneal at 69C, extension time 3:30
</LI><LI>dCas9 anneal at 69C, extension time 2:00
</LI><LI>MF ran on gel--negative results
</LI><LI>MF performed PCR again for pCC305 and dCas9 in same reaction with annealing temp 69C and extension time 2:30
</LI><LI>Need to run on gel
</LI></OL><P>	Friday June 21
</P><OL><LI>  dCas9
</LI><LI>MB ran PCR for pCC305 and dCas9 on gel
</LI><LI>Positive results for dCas9
</LI><LI>Negative results for pCC305
</LI><LI>MF performed PCR clean up for dCas9
</LI><LI>MF ran PCR again for pCC305 with 3.5 min elongation and 68C annealing temp
</LI><LI>MF ran on gel: partial results, but not clean and not large enough band
</LI><LI>MF ran PCR again for pCC305 in buffer HF (4 tubes) and buffer GC (4 tubes) with varying enzyme volumes 0.25, 0.5, 1, and 2 uL. 4.5 min elongation and 68C annealing temp
</LI><LI>  TALEs
</LI><LI>MB cultured two colonies overnight for each 16A and 20A in order to mini-prep
</LI><LI> pCC305
</LI><LI>MB cultured a colony of pCC305 overnight in order to mini-prep and save stock
</LI><LI>gRNAs
</LI><LI>Sequencing results (Order 253) arrived, gRNA 20C still had unaltered binding site in both samples. 
</LI></OL><P>	Saturday June 22
</P><OL><LI>  dCas9
</LI><LI>MF ran yesterday’s PCR results on gel, results did not look correct
</LI><LI>MF, CC redesigned plan for assembly of dCas9 plasmid
</LI><LI>  TALEs
</LI><LI>MF miniprepped two copies each of 16A and 20A (labelled N1 and N2)
</LI><LI> pCC305
</LI><LI>MF miniprepped pCC305
</LI></OL><P>	Sunday June 23
</P><OL><LI>  Yeast
</LI><LI>MB cultured DBY12397 overnight in order to perform yeast transformation
</LI></OL><H3><SPAN class="mw-headline" id="Week_of_6.2F24.2F13"><B>Week of 6/24/13</B></SPAN></H3><P>	Monday June 24
</P><OL><LI>  dCas9
</LI><LI>MF performed PCR for Z4EVpr using pMN10 backbone, 305-Z4EV-down oligo and Charlie’s pMN10-Z4EV oligo. Annealing temp 62C and 30 sec extension
</LI><LI>MF ran on gel, only primer-dimers
</LI><LI>MF performed PCR for Z4EVpr using pMN10 backbone, 305-Z4EV-down and pMN10-Z4EV oligos. Used gradient PCR with annealing 60-66C, 30 sec extension
</LI><LI>HK ran on gel, still negative results--pMN10 amplification will not work with oligos, pCC305 amplification of Z4EV should be used instead.
</LI><LI>MF performed PCR for Z4EV using pCC305 backbone, 305-Z4EV-down and pMN10-Z4EV oligos, with 68C annealing temp and 30 sec extension. 
</LI><LI>MF ran on gel--
</LI><LI>MF performed preparatory restriction digest of pCC305 with SalI and NheI to isolate the backbone
</LI><LI>MF ran on gel, cut out wrong piece and discarded correct piece
</LI><LI>MF performed same digest again
</LI><LI>  Yeast
</LI><LI>MF, MB transformed 16B, 16C, 20B, 20C TALEs into yeast to make strains 21, 22, 24, and 25. 
</LI><LI>MF, HK plated 4 yeast strains on 5-FOA plates
</LI><LI>  TALEs
</LI><LI>MB performed restriction digest of TALEs 16A (2 copies), 16B, 16C, 20A (2 copies), 20B, and 20C with AdeI and NdeI in order to prepare them for insertion into the yeast genome
</LI><LI>Digest still showed extra band around 1.5 kb for 16A and 20A TALEs
</LI><LI>MF performed analytical digest of pCC305 in AdeI and NdeI in order to compare 
</LI><LI>HK ran on gel--looked as expected, does not explain strange TALE results
</LI><LI>Probably need to redo second cycle for TALEs 16A and 20A
</LI><LI>  Reporter with Binding Sites
</LI><LI>  Sequencing data for insertion of G-blocks into iGEM.4 appears show successful insertion of 6 G-blocks plus another two with point mutations in spacer regions. Order from IDT is mislabeled. All ones labeled 16 are actually 20 and vice-a-versa.
</LI></OL><P>	Tuesday June 25
</P><OL><LI>TALEs
</LI><LI>HK performed 2nd cycle reaction for TALEs 16A and 20A
</LI><LI>Plated and transformed
</LI><LI>Yeast are growing for other TALEs. Need to grow another night before we pick transformed colonies
</LI><LI> Reporter/Binding sites
</LI><LI>Need to do yeast transformation on 6-8 correct binding sites plus control iGEM.4
</LI><LI>MF, AR performed restriction digest of BS 1x-16A, 1x-16C, 1x-20A, 1x-20B, 1x-20C, 3x-16A, 3x-20A, 3x-20B, and iGEM.4 with BsaBI and KpnI in order to prepare for insertion in the yeast genome. Digest failed because BsaBI was unable to function due to methylation site
</LI><LI>No other available enzymes to perform transformation today
</LI><LI>HK performed PCR of desired piece of reporter plasmids in order to transform tomorrow
</LI><LI>MF, AR ran on gel and all appear successful but two had an extra band that could not be accounted for
</LI><LI>MB performed sequencing reaction for second copy of incorrect binding sites and the ones with point mutations in spacers (1x-16A, 1x-16B, 1x-16C, 3x-16B, 3x-16C, and 3x-20C), sent off for sequencing (Order 286)
</LI><LI>dCas9
</LI><LI>MF, AR re-ran PCR of Z4EVpr with temperature gradient (anneal 65-72C, extension 30sec)
</LI><LI>MF ran on gel and PCR was successful for 5 of 8 tubes
</LI><LI>MB ran gel extraction for pCC305 backbone to purify large band to use in dCas9 plasmid
</LI><LI>MF performed PCR cleanup of Z4EVpr
</LI><LI>MB, AR performed Gibson assembly of Z4EVpr, pCC305 backbone, and dCas9 gene
</LI><LI>MB, MF transformed and plated 2 copies of Gibson assembly plus no insert control
</LI><LI>Yeast
</LI><LI>MF, HK, AR made YEPD medium
</LI><LI>MF, HK, AR cultured DBY12397 yeast overnight for transformation
</LI></OL><P>	Wednesday June 26
</P><OL><LI>  Reporter/binding sites
</LI><LI>AR, MF ran PCR of iGEM.4 control and with binding sites labelled 1x16A, 1x16C, 1x20A, 1x20B, 1x20C, 3x16A, 3x20A, 3x20B, using oligos Knopf His3 5p and 3p
</LI><LI>Anneal at 68C, extension time 2 min
</LI><LI>MF ran on gel--extra bands on 3x16A and 3x20A, plus more volume is needed to transform
</LI><LI>AR ran PCR again with 3 copies each of 9 plasmids and Knopf His3 oligos, Annealing at 68C and extension time 2 min
</LI><LI>MF ran on gel--extra bands now shown in all 27
</LI><LI>Sequencing confirmed second copy of iGEM.4 with 1x16B, 3x16B, 3x16C
</LI><LI>MF ran same PCR as above for 3 copies each of 1x16B, 2x16B, and 3x16C
</LI><LI>MF ran on gel--same extra bands shown on all wells
</LI><LI>AR ran PCR with new oligo tubes for iGEM.4 control and with bs1x16A and 3x16A
</LI><LI>MF cultured two colonies from plate of iGEM.4-bs3x20C to miniprep tomorrow
</LI><LI>  TALEs
</LI><LI>MB ran colony PCR on 3 copies of TALE 20A and 1 copy of TALE 20B
</LI><LI>MF ran on gel--negative results
</LI><LI>dCas9
</LI><LI>MF cultured one colony from Gibson plate 1 for iGEM.6 to miniprep tomorrow
</LI></OL><P>	Thursday, June 27
</P><OL><LI>  Reporter/Binding Sites	
</LI><LI>MF ran gel for PCR products iGEM.4 control, bs1x16A, and bs3x16A using oligos Knopf-His3 5’ and 3’ Annealed at 68C with 2min extension
</LI><LI>no extra bands, appears successful
</LI><LI>need to perform two more times successfully
</LI><LI>MF ran PCR with 3 copies each of 9 binding sites: 1x16B, 3x16B, 3x16C, 1x16C, 1x20A, 1x20B, 1x20C, 3x20A, 3x20B
</LI><LI>Ran on gel--extra bands showed up again
</LI><LI>Decided to transform anyway
</LI><LI>MF, AR performed PCR cleanup on 12 iGEM binding sites (inc. control)
</LI><LI>Mixed up and lost 2: 3x16B and 3x16C
</LI><LI>MB saved glycerol stocks and mini-prepped DNA for iGEM.4 with bs 3x-20C (2 copies)
</LI><LI>MB performed analytical digest of iGEM.4 with bs 3x-20C with BamHI and MfeI
</LI><LI>Ran on gel and results were positive
</LI><LI>MF ran sequencing reaction for iGEM.4 with bs3x20C (2 copies) using oligos yEVenus-seq1 and 303-MCSup and sent off for sequencing
</LI><LI> dCas9
</LI><LI>MB saved glycerol stocks and mini-prepped DNA for iGEM.6 (pCC305, Z4EVpr, dCas9)
</LI><LI>MB performed analytical digest of iGEM.6 with EcoRV and SapI
</LI><LI>Ran on gel and results were negative (Need to rebuild iGEM.6)
</LI><LI>MF performed PCR of Z4EV for dCas9 from pCC305 with oligos pMN10-Z4EV and 305-Z4EV-down
</LI><LI>  TALEs
</LI><LI>MF performed colony PCR for TALEs 16A and 20A
</LI><LI>Ran on gel and results were negative
</LI><LI>MB cultured two colonies from each TALE 16A and 20A overnight in order to mini-prep
</LI></OL><P>	Friday June 28
	  TALEs
</P><OL><LI>MB saved glycerol stocks and mini-prepped two copies of TALEs 16A and 20A
</LI><LI>MB performed restriction digest with AdeI and NdeI to prepare for yeast transformation (Look for extra band)
</LI><LI>MF ran on gel and TALEs 16A and 20A appear to be contaminated with another plasmid
</LI><LI>MF ran the second cycle TALE assembly for TALEs 16A and 20A
</LI><LI>MB, MF transformed 16A and 20A
</LI><LI> Yeast
</LI><LI>MF, AR transformed into yeast iGEM.4 control and with binding sites 1x16A, 3x16A, 1x16B, 1x16C, 1x20A, 3x20A, 1x20B, 3x20B, and 1x20C, to make yeast strains 
</LI><LI>AR, MF, JZ plated all 10 strains 
</LI><LI>  dCas9
</LI><LI>MF ran PCR of Z4EV from pCC305 with 305-Z4EV-down and pMN10-Z4EV, Annealing at 68C and 2min extension (8 tubes)
</LI><LI>MB ran on gel and results were positive but we got very low concentrations of DNA. Need to try to tweak PCR protocol to optimize concentration
</LI><LI>MF performed Gibson assembly of iGEM.6 (pCC305, Z4EV, dCas9) (40uL volume)
</LI><LI>MF, MB Transformed via heat shock one copy plus one control
</LI><LI>  Reporters/Binding Sites
</LI><LI>MB, AR ran PCR of bs 3x-16B and 3x-16C to prepare for yeast transformation with Hist3 primers. Annealing at 68C and 2 minute extension
</LI><LI>MB ran on gel and results showed amplification of a DNA fragment that doesn’t match our expected size
</LI><LI>  gRNA
</LI><LI>AR, MF performed SLIC for gRNA 20C with new oligos
</LI><LI>MF, MBTransformed via heat shock
</LI></OL><P>	Saturday June 29
</P><OL><LI>  Yeast
</LI><LI>AR started overnight cultures of iGEMY-1, iGEMY-2, iGEMY-3, iGEMY-4, and DBY12397
</LI><LI>all failed except for DBY12397
</LI><LI>TALES/dCas9/gRNA
</LI><LI>MF cultured TALES 16A and 20A (2 copies), gRNA20C (2 copies) and iGEM6 (1 copy)
</LI></OL><P>Sunday June 30
</P><OL><LI>  Yeast
</LI><LI>MF, AR, JZ, HK started 50 mL YEPD and colony of DBY12397 to re-do transformations
</LI><LI>TALES/gRNA/dCas9
</LI><LI>MB mini-prepped TALES 16A and 20A (2 copies), gRNA20C (2 copies) and iGEM6 (1 copy)
</LI></OL><H3><SPAN class="mw-headline" id="Week_of_7.2F1.2F13"><B>Week of 7/1/13</B></SPAN></H3><P>	Monday July 1
	  TALEs
</P><OL><LI>MB performed digest of TALEs 16A, 16B, 16C, 20A, 20B, and 20C with AdeI and NdeI to prepare for yeast transformation 
</LI><LI>MB ran on gel and results are positive
</LI><LI>  gRNA
</LI><LI>AR performed sequencing reaction for gRNA 20C
</LI><LI>  dCas9
</LI><LI>MF performed analytical digest of iGEM.6 with BamHI and KpnI and perform sequencing reaction
</LI><LI>MB ran on gel and results are negative
</LI><LI>Need to rebuild iGEM.6
</LI><LI>  Binding sites/Reporter
</LI><LI>MB performed digest of all reporters to prepare for yeast transformation with AfeI and KpnI
</LI><LI>MB ran on gel and results are likely positive (so much DNA that gel is blurry)
</LI><LI>MF performed Golden Gate reaction to assemble iGEM.5 (ACTi promoter, YEVenus, CYC terminator, pCC300 backbone)
</LI><LI>Need to transform
</LI></OL><P>	Tuesday July 2
</P><OL><LI>  Yeast
</LI><LI>MF, MB measured OD660 of yeast culture from Sunday night at ~4.8, diluted yeast by 2mL culture into 45mL and placed back in shaker, with goal of OD660~1.04
</LI><LI>MF, MB measured OD660~0.275 @11:00am
</LI><LI>MF measured OD660~0.52 @12:30pm
</LI><LI>MF measured OD660~0.97 @2:15pm
</LI><LI>Transformation begun @2:45pm
</LI><LI>MF, MB transformed iGEM.4 itself and with 7 binding sites (1x16A, 1x16B, 1x16C, 3x16A, 3x16B, 3x16C, 1x20A), as well as all six TALEs, with one no DNA control
</LI><LI>MF plated yeast: TALEs on 5FOA plates, reporters on YEPD plates with ~50uL each hygromycin
</LI><LI>  Reporter
</LI><LI>MB, MF performed PCR for Z4EVpr and dCas9 in order to put into pCC305 to make iGEM.6
</LI><LI>MB ran Z4EV PCR on gel and PCR was succesful
</LI><LI>	PCR clean up gave concentration of 30 ng/uL
</LI><LI>MB ran dCas9 on gel and PCR was unsuccessful
</LI><LI>MF, AR plated stocks of the 1x and 3x binding site versions of iGEM.4
</LI><LI>MB, MF transformed and plated iGEM.5 (2 plates)
</LI><LI>MB cultured a colony of iGEM.4 in LB+Amp overnight in order to mini-prep tomorrow
</LI><LI>TALEs
</LI><LI>MF, AR plated stocks of TALEs 16B, 16C, 20B, and 20C
</LI></OL><P>	Wednesday July 3
</P><OL><LI>  Yeast
</LI><LI>MF, MB transformed yeast with bs 1x-20B, 1x-20C, 3x-20A, 3x-20B, 3x-20C, and a no insert reporter control
</LI><LI>Yeast could not be plated--more plates being made with YEPD+hyg
</LI><LI>Tubes in cold room--Charlie will plate tomorrow
</LI><LI>  iGEM.6
</LI><LI>MF ran PCR for Z4EVpr using PCR product from yesterday and oligos pMN10-Z4EV and 305-Z4EV-down, annealled at 68, extension time 30sec
</LI><LI>MF ran PCR for dCas9 with oligos 305-dCas9 up and down, annealled at 69, extension time 2 min
</LI><LI>MF ran both PCRs on gel: Z4EV was negative, dCas9 was positive
</LI><LI>MF ran PCR for Z4EV again
</LI><LI>MF ran PCR cleanup for dCas9 with 8 tubes, final concentration 414 ng/uL in 20 uL
</LI><LI>MF ran PCR cleanup for dCas9 with 8 tubes, final concentration 450 ng/uL in 20 uL
</LI><LI>  Reporter
</LI><LI>MB mini-prepped iGEM.4 from colony picked from stock plate
</LI></OL><P>	Thursday July 4
</P><OL><LI>  Yeast
</LI><LI>CC plated iGEM.4-bs1x20B, 1x20C, 3x20A, 3x20A, 3x20B, 3x20C, and no insert control on Hygromycin
</LI><LI>CC cultured DBY12397 overnight for transformation
</LI></OL><P>	Friday July 5
</P><OL><LI>	  Reporter
</LI><LI>CC digested all twelve iGEM.4-binding sites plasmids with Kpn1/Afe1 for transformation
</LI><LI>MF ran on gel: all twelve iGEM.4-binding sites look good
</LI><LI>MF digested iGEM.4 control with Afe1/Nhe1 for transformation
</LI><LI>MF ran on gel--did not seem to digest--not transformed today
</LI><LI>MF, CK transformed 12 reporters plus no DNA control into yeast, making strains 3,4,6,7,9,10,12,13,15,16,18,19. 
</LI><LI>MF plated 12 reporter yeast strains on Hygromycin
</LI><LI>MF ran sequencing reaction for iGEM.5 (two copies) with oligos yEVenus-seq1, yEVenus-seq-2, 306-Ura3p, 306-MCSup and sent off for sequencing
</LI><LI>  TALEs
</LI><LI>CC digested TALEs 16B, 20B, 16C, and 20C for transformation with DraIII/Nde1
</LI><LI>MF ran on gel: all four look good
</LI><LI>MF spotted yeast from 7/2 transformation of all 6 TALEs onto G418 plates (strains 20-25)
</LI><LI>MF, CK transformed TALEs 16B, 20B, 16C, 20C plus no DNA control into yeast, making strains 21,22,24,25
</LI><LI>MF plated TALE strains on 5FOA plates
</LI><LI>  dCas9
</LI><LI>MF ran Gibson assembly for iGEM.6 using gel-extracted pCC305 vector, 3xmolar dCas9 PCR, and 3xmolar Z4EVpr PCR, with no insert control
</LI><LI>MF transformed and plated iGEM.6 Gibson and control
</LI></OL><P>	Saturday July 6
</P><OL><LI>Gibson assembly plate had no colonies to culture
</LI><LI>CC spotted yeast from 7/3-4 transformation of iGEM.4-bs1x16B*, 1x16C*, 3x16A*, 3x16B*, 3x16C* (strains 3,6,7,9,10) 
</LI><LI> = name corrected
</LI><LI>names not corrected on labels yet!
</LI></OL><P>	Sunday July 7
</P><OL><LI>MF, CC cultured 4 yeast strains from plate spotted on 7/6 for iGEM.4-bs1x16B*-3, 3x16A*-1, 3x16B*-1, 3x16C*-1 (strains 3,6,7,9,10) in YSCD
</LI><LI>MF cultured DBY12397 overnight in 45mL YEPD 
</LI></OL><H3><SPAN class="mw-headline" id="Week_of_7.2F8.2F13"><B>Week of 7/8/13</B></SPAN></H3><P>Monday July 8
</P><OL><LI>  Yeast
</LI><LI>MF digested 6 TALEs in Nde1/DraIII with NEBuffer 2 to transform
</LI><LI>MB ran on gel and all 6 look mostly good, but 16A and 20A top bands slightly lower than other TALEs?
</LI><LI>CC diluted cultures of yeast strains in YSCD and YSCD-BED to flow later today
</LI><LI>MF picked and spotted yeast colonies from 7/5/13 transformation plates that looked promising: GEMY-9, 12, 15, 17, 19, 25. (TALE 20C, iGEM.4-bs1x16B*, 3x16C*, 3x20A*, 3x20B*, 1x20C*)
</LI><LI> = name corrected
</LI><LI>MB, MF, CC transformed 6 TALEs into DBY12397
</LI><LI>CC, MF, MB, AR, HK, JZ, CK ran flow cytometer with:
</LI><OL><LI> Calibration
</LI><LI> DBY12307 (GEMY-0)
</LI><LI> GEMY-7.1
</LI><LI> GEMY-7.2
</LI><LI> GEMY-7.3
</LI><LI> GEMY-3.1
</LI><LI> GEMY-6.1
</LI><LI> GEMY-6.2
</LI><LI> GEMY-9.1
</LI></OL></OL><UL><LI>Results: flourescence was only 1-2 fold greater than untransformed strain
</LI></UL><LI>  Reporters
</LI><LI>MB ran PCR with 4 tubes each of iGEM.4-with 12 binding sites (48 tubes total) using oligos yEVenus-seq1 and Spe1-iGEM.5-binding-site in order to ligate binding sites into iGEM.5
</LI><LI>MF ran on gel: results all look positive, band around 9kb also present 
</LI><LI>MB performed PCR clean up and got good concentrations for all binding sites
</LI><LI>MB, AR performed restriction digest for ligation of binding site PCR into iGEM.5 with SpeI
</LI><LI>  gRNA
</LI><LI>MB ran sequencing reaction for gRNA 20C (2 copies) with oligo pRSup and sent off for sequencing
</LI><LI>  dCas9
</LI><LI>MF performed Gibson assembly of iGEM.6 (Z4EVpr, dCas9, pCC305)
</LI><LI>MB transformed and plated Gibson and control
</LI><P>	Tuesday July 9
</P><OL><LI>  Yeast
</LI><LI>MB cultured tubes from yeast reporters spotted on 7/8/13 to flow tomorrow morning
</LI><LI>  Reporters
</LI><LI>MB performed CIP and performed PCR clean up on iGEM.5
</LI><LI>Sequencing data came back bad. Need to re-build iGEM.5
</LI><LI>MF performed digest of binding site PCR with DpnI to cut up methylated DNA
</LI><LI>MB performed PCR clean up
</LI><LI>Need to insert into iGEM.5
</LI><LI>  dCas9
</LI><LI>MF performed PCR for pCC305 with Z4EVpr using 305-mCherry-up and Z4EV-down
</LI><LI>MB ran on gel and looked as if almost no DNA was recovered. Need to order new oligos to build #iGEM.6
</LI><LI>MF performed PCR for dCas9 with dCas9-up and dCas9-down
</LI><LI>MB ran on gel and looked like dCas9 was successfully PCRed but bands very smeary. Not sure why
</LI></OL><P>	Wednesday July 10
</P><OL><LI>  Yeast
</LI><LI>MB, MF, CC ran flow cytometry for GEMY.7-(2-4), GEMY.9-(1-4), GEMY.12-1, GEMY.15-1, GEMY.19-(1-3). YFP production appeared to be at most a 2-fold increase over background. Going to try transforming reporters into different yeast strain
</LI><LI>Transformations from Monday appear to be more abortive transformants. DNA is getting into the yeast but not integrated in the genome. Not very big colonies. No transformants on 5FOA plates and very small transformants on G418 plates. A few colonies of G418 control plate which shouldn’t have colonies
</LI><LI>Need to culture 100mL yeast to do transformation tomorrow
</LI><LI>  Reporters
</LI><LI>MB successfully mini-prepped three copies each of iGEM.5-1 and iGEM.5-2
</LI><LI>MB performed analytical digest of iGEM.5 copies with NsiI
</LI><LI>MB ran on gel and confirmed that iGEM.5 assembly was unsuccessful
</LI><LI>MF digested both iGEM.2 and iGEM.4 with BstEII-HF and SalI-HF in cutsmart for gel extraction to ligate and make iGEM.5
</LI><LI>MB ran on gel and correct size bands were present
</LI><LI>MF performed gel extraction
</LI><LI>MB performed the ligation of the digests of iGEM.2 and iGEM.4 in order to make iGEM.5
</LI><LI>Need to digest reporters for transformation tomorrow (need to get AfeI from Mert)
</LI><LI>  TALEs
</LI><LI>MF, AR digested 6 TALEs with DraIII and NdeI in NEbuffer 2 for yeast transformation tomorrow
</LI></OL><P>	Thursday July 11
</P><OL><LI> TALEs
</LI><LI>MB ran TALEs digested with DraIII and NdeI on a gel and bands appear to be right size. Digest was slightly incomplete and put back in 37C water bath
</LI><LI>MB cultured colonies in LB+Amp for TALEs 16A, 16B, 16C, 20A, 20B, and 20C to mini-prep tomorrow
</LI><LI> Yeast
</LI><LI>MF transferred colonies from Monday’s transformation to new plates. If they were on G418, they were moved to 5FOA and vice-a-versa.  Plates were left in the incubator to look for continued growth
</LI><LI>MB centrifuged overnight yeast culture and there was not enough yeast to do transformation. Resuspended yeast in YEPD and allowed to grow more in shaker.
</LI><LI>Yeast started to grow again, OD660~0.37 @ 2:30, OD660~0.75 @ 4:15
</LI><LI>MF started a culture of MMY-C yeast in YEPD to transform tomorrow
</LI><LI> Reporters
</LI><LI>MF performed restriction digest of iGEM.4, iGEM.4 + 1x-binding sites, and iGEM.4 + 3x-binding sites with AfeI and KpnI-HF in CutSmart Buffer in order to prepare for integration in the yeast genome
</LI><LI>MB ran on gel and all correct bands there. Reporters ready to be transformed
</LI><LI>MB cultured colonies in LB+Amp for iGEM.4, iGEM.4 + 1x-(16A, 16B, 16C, 20A, 20B, and 20C), iGEM.4 + 3x-(16A, 16B, 16C, 20A, 20B, and 20C) to mini-prep tomorrow
</LI><LI>MB transformed the ligation and control to form iGEM.5 and plated on LB+Amp
</LI></OL><P>	Friday July 12
</P><OL><LI>  TALEs
</LI><LI>MB, MF mini-prepped TALEs 16A, 16B, 16C, 20A, 20B, and 20C
</LI><LI>  Reporters
</LI><LI>MB, MF mini-prepped iGEM.4, iGEM.4 + 1x-(16A, 16B, 16C, 20A, 20B, 20C), and iGEM.4 + 3x-(16A, 16C, 20A, 20B, 20C)
</LI><LI>MB cultured colony in LB+Amp for iGEM.4 + 3x-16B
</LI><LI>MB cultured 4 colonies from iGEM.5 plate in LB+Amp
</LI><LI>  Yeast
</LI><LI>MF, MB transformed 6 TALEs and 13 Reporters into MMY-C yeast, plus a TALE control and a reporter control
</LI><LI>MB, AR plated all recovered transformants on G418 for TALEs and Hygromycin for iGEM.4 and all combinations of iGEM.4 with binding sites
</LI><LI>TALEs spotted 7/11 onto 5FOA plates look promising--need to culture to flow tomorrow
</LI><LI>MF cultured CCY232-3 for background fluorescence along with all spotted colonies for flow
</LI><LI>TALEs spotted on G418 did not grow as well
</LI></OL><P>	Saturday July 13
</P><OL><LI>  Reporters
</LI><LI>MB mini-prepped iGEM.4 + 3x-16B
</LI><LI>MB saved glycerol stocks and mini-prepped iGEM.5 (4 copies)
</LI><LI> Yeast
</LI><LI>MF, HK diluted all 48 cultures for flow. 15 microliters of culture from well into 3 mL of YSCD. Allow to grow in 30C roller for 6 hours before performing flow.
</LI><LI>MF, MB, HK, JZ performed flow on 48 tubes. Need to analyze data, but TALEs 16B, 16C, 20B, 20C seem to have fluorescence significantly above background strain
</LI></OL><P>	Sunday July 14
</P><OL><LI>	  Yeast
</LI><LI>MB cultured both DBY12397 and CCY-232-3 for transformations tomorrow
</LI></OL><H3><SPAN class="mw-headline" id="Week_of_7.2F15.2F13"><B>Week of 7/15/13</B></SPAN></H3><P>	Monday July 15
</P><OL><LI>	  Yeast
</LI><LI>MF, MB digested 13 iGEM.4 reporters with NheI-HF and AfeI for transformation (enough for two transformations per tube) (cutsmart, 2h @37C)
</LI><LI>gel looked strange, transformation aborted
</LI><LI>MF cultured 45mL DBY12397 for transformation tomorrow
</LI><LI>MF, MB digested 13 iGEM.4 reporters with KpnI-HF and AfeI for transformation tomorrow (cutsmart, 3h @37C)
</LI><LI>MB spotted reporters in MMY from 7/12/13 transformation onto YEPD+hyg plates
</LI><LI>MB, AR, MF cultured reporters in MMY (spotted today) in 3 mL YSCD for flow tomorrow
</LI><LI>  Reporters
</LI><LI>MB ran analytical digest of iGEM.5 (4 copies) in NsiI (buffer 3.1, 2h @37C)
</LI><LI>results look strange, undigested?
</LI><LI>MF ran analytical digest of iGEM.5 (4 copies) in EcoRI-HF/SalI-HF (cutsmart, 2h@37C)
</LI><LI>Looks good, previous problem probably with digest, not DNA
</LI><LI>  dCas9
</LI><LI>MF ran perparatory digest of pCC305 in Nhe1-HF (cutsmart, 3h @37C)
</LI><LI>MF ran PCR of dCas9 with oligos dCas9-N1 and dCas9-N2, with annealing temp 67.5C and extension time 2min
</LI><LI>MB, JZ ran PCR on gel, successful bands
</LI><LI>AR, JZ ran successful PCR cleanup on dCas9
</LI><LI>MB, JZ ran prep digest on gel, looked faint/dilute but correct
</LI><LI>MF cut out larger band (backbone)
</LI><LI>MF, MB ran gel cleanup, little to no DNA present
</LI></OL><P>	Tuesday July 16
</P><OL><LI>	  Yeast
</LI><LI>MF diluted yeast cultures for flow so they would be 0.1 OD by 3 pm and put back in 30C roller
</LI><LI>Flow data showed good fluorescence levels for iGEM.4 but when the binding sites (1x or 3x) are present there is almost no fluorescence (2-fold max over background) 
</LI><LI>MB transformed DBY12397 with all 1x and 3x versions of iGEM.4, iGEM.4, and a no DNA control
</LI><LI>MB plated on YPD+Hyg and put in 30C incubator
</LI><LI>MB, MF started cultures for TALEs 16B, 16C, 20B, and 20C along with a DBY12397 background to do a time course of beta-estradiol induction tomorrow
</LI><LI>MF, AR diluted back the cultures at night into YSCD to be ready to flow in morning (target OD660~0.01, 0.02, 0.03 in morning)
</LI><LI>  Reporters
</LI><LI>MF performed sequencing reaction for iGEM.5 (2 copies)
</LI><LI>MB performed digest of iGEM.5 with SpeI-HF in order to prepare for insertion of binding sites
</LI><LI>MB performed PCR clean-up of digest
</LI><LI>MB performed CIP on iGEM.5 digest clean-up
</LI><LI>MB performed ligation of binding sites into SpeI digested iGEM.5
</LI><LI>MB transformed and plated on LB+Amp
</LI><LI>MB cultured two colonies of pCC305 in LB+Amp in order to mini-prep tomorrow
</LI><LI>  TALEs
</LI><LI>MF performed sequencing reaction for TALEs 16A and 20A to try and determine issue with these TALEs
</LI></OL><P>	Wednesday July 17
</P><OL><LI>  dCas9
</LI><LI>MB mini-prepped two copies of pCC305 backbone
</LI><LI>MF performed restriction digest of pCC305 with NheI in order to prepare for insertion of dCas9
</LI><LI>MF, MB performed gel extraction of larger band
</LI><LI>MB performed Gibson assembly of iGEM.6 with gel extraction product of pCC305 and dCas9 PCR product
</LI><LI>MB transformed and plated on LB+Amp
</LI><LI>  Yeast
</LI><LI>MF, MB, AR performed time course flow cytometry for TALEs 16B, 16C, 20B, and 20C along with background DBY12397: times every 30 min, inducing and non-inducing medium, start at target OD660~0.02
</LI><LI>  Reporters
</LI><LI>MB performed PCR of 5x-16A and 5x-16C binding sites. Annealing at 67 and extension at 72 for 20 sec
</LI><LI>MF ran on gel and PCR appears successful
</LI><LI>Need to run on polyacrylamide gel to try and extract product that contains all 6 oligos
</LI></OL><P>	Thursday July 18
</P><OL><LI>  Reporters
</LI><LI>MB, MF mini-prepped iGEM.5 + 1x-(16A, 16B, 16C, 20A, 20B, and 20C) and iGEM. 5 + 3x-(16A, 16B, 16C, 20A, 20B, and 20C)
</LI><LI>MB digested with SpeI to look for insertion of binding sites
</LI><LI>MB, MF mini-prepped 404 TEF1 and 404 TDH3 in order to swap the TEF and TDH promoters in for ACT1
</LI><LI>MB digested iGEM.5 and all versions of it with 1x and 3x binding sites with AfeI and KpnI-HF for yeast transformation
</LI><LI>MB, MF, CK ran 5x-16A and 5x-16C PCR on acrylamide gel
</LI><LI>Gel extraction failed due to smearing on gel. Need to re-run PCR
</LI><LI>Need to perform Gibson/SLIC into iGEM.4
</LI><LI>  dCas9
</LI><LI>Plate for iGEM.6 has no colonies
</LI><LI>MB re-ran Gibson assembly of iGEM.6
</LI><LI>MB, JZ transformed and plated on LB+Amp
</LI><LI>Yeast
</LI><LI>MB cultured DBY12397 overnight to perform transformation tomorrow
</LI></OL><P>	Friday July 19
</P><OL><LI>  Reporters
</LI><LI>MB ran digest of iGEM.5 and all versions of it with 1x and 3x binding sites on a gel and all digest appear successful
</LI><LI>MB re-ran PCR of 5x-16A and 5x-16C binding sites
</LI><LI>MF performed sequencing reaction for iGEM.5 and all 1x and 3x binding site versions (two copies of each)
</LI><LI>Yeast
</LI><LI>MB transformed iGEM.5 and all 1x and 3x binding sites versions into DBY12397
</LI><LI>HK plated on YPD+Hyg
</LI><LI>AR balanced the ODs of DBY12397 and transformants with TALEs 16B, 16C, 20B, and 20C and began time course for induction with beta estradiol
</LI><LI>AR, MB, JZ, and MF ran time course for DBY 12397, TALEs 16B, 16C, 20B, and 20C, inducing and non-inducing medium, with times every 30 minutes. Aimed for 24 timepoints over 12 hours but ran out of induced DBY 12397 and stopped at 21st time point.
</LI><LI>  dCas9
</LI><LI>Plate for iGEM.6 has no colonies again. Need to check to see if Gibson Master Mix is problem by using the positive control that comes with Gibson assembly kit
</LI><LI>  TALEs
</LI><LI>MF performed sequencing reaction for TALEs 16A and 20A to try and determine the issue with them
</LI></OL><H3><SPAN class="mw-headline" id="Week_of_7.2F22.2F13"><B>Week of 7/22/13</B></SPAN></H3><P>	Monday July 22
</P><OL><LI>  Reporters
</LI><LI>MB performed PCR of TEF1pr and TDH3pr in order to replace Act1pr in iGEM.4
</LI><LI>MB ran on gel and PCR appears successful
</LI><LI>MB performed PCR clean up
</LI><LI>MB performed restriction digest of TEF1pr and TDH3pr with AscI, DpnI, and XbaI
</LI><LI>MB performed PCR clean up
</LI><LI>MF performed overnight ligation with 1x-(16A, 16B, 16C, 20A, 20B, and 20C) and 3x-(16A, 16B, 16C, 20A, 20B, and 20C)
</LI><LI>MF performed PCR with oligos 3 and 4 for 5x-16A and 5x-16C in order to try and make the 5x binding sites
</LI><LI>MB ran on gel and bands appeared very smeary
</LI><LI>This method for building 5x binding sites has been abandoned
</LI><LI>MB performed restriction digest of iGEM.4 with AscI and SpeI
</LI><LI>MF performed gel extraction and clean up of backbone
</LI><LI>  Yeast
</LI><LI>MB cultured DBY12397, iGEM.4, 1x-16B, 3x-16A, 3x-16B, 1x-20B, 1x-20C, and 3x-20A (2 copies of everything but DBY12397) in order to flow tomorrow morning
</LI><LI>  dCas9
</LI><LI>MB, MF performed Gibson assembly of iGEM.6 using 1:1, 3:1, 6:1, and 9:1 insert to backbone ratios. The Gibson assembly positive control was also performed
</LI><LI>MB transformed and plated on LB+Amp 
</LI><LI>  Competent Cells
</LI><LI>MB inoculated a colony of DH5Alphas in 2 mL of LB overnight to make chemically competent cells
</LI></OL><P>	Tuesday July 23
</P><OL><LI> Yeast
</LI><LI>MB flowed DBY12397, iGEM.4, 1x-16B, 3x-16A, 3x-16B, 1x-20B, 1x-20C, and 3x-20A (2 copies of everything but DBY12397). The ones with binding sites have nearly identical fluorescence to the background indicating that the binding site insertion is disrupting the expression of YFP
</LI><LI>MB picked transformants from iGEM.5, iGEM.5+1x-(16A, 16B, 16C, 20A, 20B, and 20C), and iGEM.5+3x-(16A, 16B, 16C, 20A, 20B, and 20C) and put them on new YPD+Hyg plates
</LI><LI>MB made glycerol stocks of GEMY 1A, 4A, 6A, 9A, 12A, 18A, and 19A
</LI><LI>  dCas9
</LI><LI>Plates had no colonies including the positive control plate indicating that there is a problem with the Gibson assembly master mix
</LI><LI> Competent Cells
</LI><LI>MB made 0.1 M CaCl2 and 0.1 M CaCl2 in 15% glycerol
</LI><LI>MB, CC made and aliquoted chemically competent DH5Alphas
</LI><LI>Reporters
</LI><LI>MF performed PCR clean up of ligation of binding sites to either TEF1 or TDH3 promoters
</LI><LI>MF performed ligation of promoter-binding site with pCC305 and left overnight in heat block in cold room
</LI></OL><P>	Wednesday July 24
</P><OL><LI>  Yeast
</LI><LI>MB, MF started cultures for DBY12397, iGEM.5, iGEM.5+1x-(16A, 16B, 16C, 20A, 20B, and 20C), and iGEM.5+3x-(16A, 16B, 16C, 20A, 20B, and 20C) in a 96 well plate to flow tomorrow morning
</LI><LI>MF diluted back to aim for an OD of 0.1 by 9 am
</LI><LI>Reporters
</LI><LI>MB transformed ligations of TEF1/TDH3, binding site, and pCC305 backbone
</LI><LI>MF performed sequencing reaction for iGEM.5 and iGEM.5 with all the 1x and 3x binding sites and sent off for sequencing 
</LI><LI>  TALEs
</LI><LI>MF performed sequencing reaction for TALEs 16A and 20A to try and determine the issue with these plasmids
</LI></OL><P>	Thursday July 25
</P><OL><LI>  Yeast
</LI><LI>MB, MF flowed DBY12397, iGEM.5, iGEM.5+1x-(16A, 16B, 16C, 20A, 20B, and 20C), and iGEM.5+3x-(16A, 16B, 16C, 20A, 20B, and 20C)
</LI><LI>  Reporters
</LI><LI>MB cultured colonies overnight in LB+Amp from ligation plates of iGEM.7 and iGEM.8 with binding sites in order to mini-prep tomorrow
</LI></OL><P>Friday July 26
</P><OL><LI>  Reporters
</LI><LI>MB, AR miniprepped iGEM.7 and iGEM.8 with 1x and 3x binding sites.
</LI><LI>JZ digested iGEM.7 and iGEM.8 plasmids with Mlu1, will run analytical gel on Monday.
</LI></OL><H3><SPAN class="mw-headline" id="Week_of_7.2F29.2F13"><B>Week of 7/29/13</B></SPAN></H3><P>	Monday July 29
</P><OL><LI>  Reporters
</LI><LI>AR ran gel of all iGEM.7 and iGEM.8
</LI><LI>unsuccessful, only one band
</LI><LI>  dCas9
</LI><LI>JZ ran digest of PCC305 with Nhe1
</LI><LI>AR ran PCR of Cas9 with Cas9-N1 and Cas9-N2
</LI><LI>JZ ran PCR clean up and CIP of Cas9
</LI></OL><P>	Tuesday July 30
</P><OL><LI>dCas9
</LI><LI>AR, CC performed gel extraction and purification of PCC305
</LI></OL><P>	Wednesday July 31
</P><OL><LI>  Yeast
</LI><LI>AR cultured DBY12397 and TALES 16B, 16C, 20B, 20C in 3mL SCD in order to redo flow tomorrow
</LI><LI>  dCas9
</LI><LI>HK re-ran PCR for Cas9 with Cas9-N1 and Cas9-N2. PCR from yesterday contained no product. Confirmed today’s PCR on a gel.
</LI><LI>JZ ran a Gibson assembly reaction with the new Gibson kit, the dCas9 PCR product, and the Nhe1 digested pCC305. Gibson kit competent cells were transformed with backbone only, insert only, backbone+insert, and a positive control and plated.
</LI><LI>  Reporters
</LI><LI>JZ redesigned and ordered oligos to introduce new restriction sites for the binding site batteries and the TEF1 and TDH1 promoters for the second attempt at constructing iGEM.7 and iGEM.8 as well as the BioBrick parts containing the binding sites and the promoters.
</LI></OL><P>	Thursday August 1
</P><OL><LI>  Yeast
</LI><LI>AR,  JZ, and HK performed time course flow cytometry for DBY 12397, TALEs 16B, 16C, 20B, and 20C
</LI><LI> dCas9
</LI><LI>JZ concluded Gibson failed (only colonies on positive control).  Redid Gibson and ran SLIC assembly. Transformed cells with assembly products and plate them to grow overnight.
</LI></OL><P>	Friday August 2
</P><OL><LI>  Reporters
</LI><LI>AR prepped new oligos for TEF and TDH.
</LI><LI>JZ ran a PCR to amplify the TEF1 promoter. Ran the reaction product on a gel, PCR appears to have failed. Need to redo.
</LI><LI>JZ streaked a plate of stock p404TDH3 for amplification on Monday.
</LI><LI> Yeast
</LI><LI>AR analyzed data from time course, looks to be successful so far
</LI><LI>TALEs
</LI><LI>HK ran first cycle of the Golden Gate reaction for pTAL synthesis of bs16A and bs20A.
</LI><LI> dCas9
</LI><LI>Gibson and SLIC failed, need to design new strategy for dCas9 assembly.
</LI></OL><H3><SPAN class="mw-headline" id="Week_of_8.2F5.2F13"><B>Week of 8/5/13</B></SPAN></H3><P>	Monday August 5
</P><OL><LI> Reporters
</LI><LI>JZ ran a PCR cleanup on PCRed TEF1 promoter, nanodrop reports no DNA.
</LI><LI>JZ redid the PCR of TEF1.
</LI><LI>AR ran on gel, PCR was unsuccessful
</LI><LI>TALES
</LI><LI>AR ran colony PCR on products of pTAL synthesis of bs16A and bs20A
</LI><LI>AR ran PCR products on gel and all were unsuccessful, will redo first cycle of Golden Gate assembly
</LI></OL><P>	Tuesday August 6
</P><OL><LI> Yeast
</LI><LI>AR cultured DBY12397 and TALES 16B, 16C, 20B, 20C for flow tomorrow morning to determine input/output curve for BED concentrations
</LI><LI>AR diluted cultures back to flow at an OD of 0.4 tomorrow morning, and induced a set of TALEs and DBY12397 in each of the following concentrations of DMSO/BED
</LI><LI>0 M
</LI><LI>0.1 nM
</LI><LI>1 nM
</LI><LI>10 nM
</LI><LI>20 nM
</LI><LI>50 nM
</LI><LI>100 nM
</LI><LI>500 nM 
</LI><LI>1 uM
</LI></OL><P>	Wednesday August 7
</P><OL><LI>  Yeast
</LI><LI>AR ran flow on TALEs and DBY12397 for all concentrations listed above
</LI><LI>Reporters
</LI><LI>JZ and CC ran PCR of TEF1pr using p404TEF1 backbone and oligos XbaAscTEF1-5p and Age1TEF1-3p.P PCR product confirmed on gel and cleaned up.
</LI><LI>JZ and CC ran PCR of binding site batteries using iGEM.4 (1x16A,3x16A,1x16C,1x20A,1x20B,3x20B,3x20C) template and oligos Age1-iGEM5bind and yEVseq1. Gel results unclear, PCR may have been successful. Product frozen for clean up tomorrow.
</LI></OL><H3><SPAN class="mw-headline" id="Week_of_8.2F12.2F13"><B>Week of 8/12/13</B></SPAN></H3><P>	Tuesday August 13
</P><UL><LI>AR ran PCR of
</LI><LI> TEF1pr using 404TEF1 backbone and oligos XbaAscTEF1-5p and Age1TEF1-3p
</LI><LI>binding site batteries using iGEM.4 (3x16C, 3x20C, 1x16B, 3x20A, 3x16B) template and oligos Age1-iGEM5bind and yEVseq1
</LI><LI>TDH3pr using 404TDH3 backbone and oligos XbaAscTDH3-5p and Age1TDH3-5p
</LI></UL><UL><LI>AR ran gel of PCR products.  TEF1 and bs batteries were successful, TDH3 failed 
</LI><LI>AR performed PCR clean up on TEF1 and bs batteries
</LI><LI>AR ran analytical digest of TEF1pr with Xba, Age1, and Dpn1 and digest of all binding sites (1x16A,B,C, 3x16A,B,C, 1x20A,B,C, 3x20A,B,C) with Spe1, yEVseq1, and Dpn1
</LI><LI>AR streaked LB+Amp plates with iGEM.5 and iGEM.5 + 1x(16A, 16B, 16C, 20A, 20B, 20C) and 3x(16A, 16B, 16C, 20A, 20B, 20C)
</LI><LI>CC heat shocked with plasmid pSB1C3 from iGEM plate for BioBrick backbone
</LI></UL><P>	Wednesday August 14
</P><UL><LI>AR,CC designed new oligos for 5xbinding sites (16A, 16B, 16C, 20A, 20B, 20C) with Age1-binding sites x3- spacers and Spe1-spacers-binding sites x3
</LI><LI>AR cultured iGEM.5 and iGEM.5 + 1x(16A, 16B, 16C, 20A, 20B, 20C) and 3x(16A, 16B, 16C, 20A, 20B, 20C) in LB+Amp to miniprep tomorrow morning
</LI></UL><P>	Thursday August 15
</P><UL><LI> AR miniprepped iGEM.5 and iGEM.5 + 1x(16A, 16B, 16C, 20A, 20B, 20C) and 3x(16A, 16B, 16C, 20A, 20B, 20C)
</LI><LI>AR ran digest of iGEM.5 and iGEM.5+1x(16A, 16B, 16C, 20A, 20B, 20C) and 3x(16A, 16B, 16C, 20A, 20B, 20C) with Kpn1-Hf and Afe1
</LI><LI>Ran digest on gel, all were successful
</LI></UL><LI>CC ran digest of pSB1C3
</LI><LI>AR cultured DBY12397 and TALEs 16B and 16C for transformation
</LI><P>	Friday August 16
</P><UL><LI>AR,CC transformed digested iGEM.5 and iGEM.5 + 1x(16A, 16B, 16C, 20A, 20B, 20C) and 3X(16A, 16B, 16C, 20A, 20B, 20C) into DBY12397 
</LI><LI>AR,CC  transformed digested iGEM.5 + 1x(16B, 16C) and 3x(16B, 16C) into their respective DBY12397+16B and DBY12397+16C strains
</LI></UL><P>	Sunday Aug 18
</P><UL><LI>CC patched colonies from Aug 16th yeast transformation to new YEPD+Hyg plates and returned to 30C. Efficiency low in most cases, zero in a few. 
</LI></UL><H3><SPAN class="mw-headline" id="Week_of_8.2F26.2F13"><B>Week of 8/26/13</B></SPAN></H3><P>	Monday Aug 26
</P><UL><LI>JZ ran PCR of binding site batteries from 5X20 BS oligos, ran products on gel, which looked successful.
</LI></UL><P>Tuesday Aug 27
</P><UL><LI>MF transformed Z4EV-dCas9 plasmid into E. coli and plated on LB+Amp
</LI></UL><P>Wednesday Aug 28
</P><UL><LI>MB cultured 4 colonies from Z4EV-dCas9 plate in LB+Amp in order to mini-prep tomorrow
</LI></UL><P>Thursday Aug 29
</P><UL><LI>JZ ran PCR of binding site batteries from 5X16 BS oligos.
</LI><LI>MB mini-prepped 4 samples of Z4EV-dCas9
</LI><LI>MB performed analytical digest of Z4EV-dCas9 plasmid with BamHI and XbaI
</LI><LI>MB ran on gel and only one band appeared on gel. Assembly of dCas9 plasmid appears to have failed
</LI></UL><H3><SPAN class="mw-headline" id="Week_of_9.2F2.2F13"><B>Week of 9/2/13</B></SPAN></H3><P>	Wednesday September 4
</P><UL><LI>MB performed restriction digest of all binding site PCRs with AgeI and SpeI
</LI><LI>MB performed restriction digest of TEF1pr PCR with AgeI and XbaI
</LI><LI>MB performed ligation of TEF1pr, binding sites, and pSB1C3 in order to make reporter biobricks
</LI><LI>MB transformed into E. coli and plate on LB+Cm
</LI></UL><LI>MB plated iGEM.4+3x-20A from frozen stock
</LI><P>Thursday September 5
</P><UL><LI>MB cultured colonies in LB+Cm from all biobrick plates except for the 1x-16B binding site because that plate had no colonies
</LI></UL><P>Friday September 6
</P><UL><LI>MB saved glycerol stocks and mini-prepped the colonies cultured yesterday
</LI><LI>MB performed an analytical digest of the biobricks with SpeI-HF and XbaI to determine if inserts are there and to determine orientation
</LI><LI>JZ digested TEF1pr with AscI/Age1 and iGEM.4 backbone with AscI/Spe1 for ligation.
</LI></UL><P>Saturday September 7
</P><UL><LI>MF performed biobrick ligation of TEF1pr-bs1x16B into pSB1C3 
</LI><LI>MF transformed ligation and plated on LB+Cm
</LI></UL><LI>MF performed ligation of TEF1pr (cut with AgeI/XbaI) and 12 binding sites (1x and 3x) (cut with AgeI and SpeI) into iGEM.4 backbone (cut with XbaI/SpeI)
</LI><LI>MF transformed ligations and plated on LB+Amp
</LI><P>Sunday September 8
</P><UL><LI>MF cultured colonies from 13 ligations, two colonies per plate (26 colonies) to miniprep tomorrow
</LI></UL><H3><SPAN class="mw-headline" id="Week_of_9.2F9.2F13"><B>Week of 9/9/13</B></SPAN></H3><P>	Monday September 9
</P><UL><LI>MF miniprepped all 26 cultures from ligations and saved frozen stocks
</LI></UL><P>	Wednesday September 11
</P><UL><LI>MB performed an analytical digest of all 12 TEF1pr-Binding Site reporters (2 copies each) with SpeI-HF and AfeI
</LI><LI>MB performed an analytical digest of TEF1pr-1x 16B BioBrick (2 copies) with SpeI and XbaI
</LI></UL><P>	Thursday September 12
</P><UL><LI>MF cultured DBY12397 in YEPD overnight for transformation
</LI></UL><P>	Friday September 13
</P><UL><LI>MF digested iGEM.7+binding site (TEF1 constructs) 12 plasmids for transformation with Afe1 and Kpn1-HF
</LI><LI>Digest looks good for all except 1x16B and 3x16C
</LI></UL><LI>MF transformed 12 plasmids into DBY12397 with one no DNA control
</LI><LI>MF plated on YEPD+Hyg plates
</LI></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Duke/Notebook/Lab_Notebook">http://2013.igem.org/Team:Duke/Notebook/Lab_Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Duke/Notebook/Lab_Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Duke/Notebook/Lab_Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Duke/Notebook/Lab_Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Duke/Notebook/Lab_Notebook&amp;oldid=233475" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>