"
Page
Discussion
View source
History
teams
Log in
 
Team:Duke/Project/Problem
From 2013.igem.org
Home
Team Duke 2013
The Team
Undergrads
Grad Students
Instructors
Genetic Toggle Switch
The Project
Background Information
Experimental Design
Results:
Library of DNA Parts
Results:
Parts Characterization
Biological Parts Submitted
Mathematical Modeling
Cooperativity and Hill Equation
Thermodynamic Model : Introduction
Thermodynamic Model : Cooperative Repression
Kinetic Model :
Bifurcation and Bistability
Human Practices
From Bench to Biotech
Lab Notebook
Overview
Protocols
Strain List
Safety
E.coli and S.cerevisiae
Xanthomonas Bacteria
Attributions
iGEM 2013
Designing Synthetic Gene Networks Using Artificial
Transcription Factors in Yeast
Project Description
Following an initial period of growth after the publication of the first synthetic gene circuits, development in the field has stalled. This is due in part to the limited number of well-characterized parts with desired features. For instance, the function of the repressilator and genetic toggle switch both rely on repressible promoters with high cooperativity – provided in these cases by multimerization of the repressor proteins. The TALE family of transcription factors (TFs) and the CRISPR/Cas9 system show promise in expanding the parts list to bind to near-arbitrary target sequences, but because they bind to DNA as monomers, promoters under their control cannot show cooperativity in their response.
It has been shown theoretically and in vivo that repressors binding as monomers to multiple binding sites can introduce cooperativity in to a system. With this in mind, we are developing an organism-independent approach that leverages programmable TFs to create library of independent and orthogonal repressor-promoter pairs with a range of expression parameters (viz. cooperativity, basal and maximal expression rate, response time) of potentially unlimited size. It is our aim that this approach will enable the field to move toward exploring higher-order dynamics.
Abstract
Synthetic gene circuits have the potential to revolutionize gene therapies and bio-industrial methods by allowing predictable, customized control of gene expression. Bistable switches and oscillators, key building blocks of more complex gene networks, have been constructed using naturally occurring and well-characterized regulatory elements. In order to expand the versatility and variety of these circuits, we designed and constructed gene networks using artificial transcription factors (ATFs). The ATFs are of two classes: inhibitory TAL proteins and a catalytically inactive dCas9 protein with small guide RNA elements, each orthogonal to the yeast genome. Using mathematical modeling, we determined the parameters expected to create bistability and oscillation, using tandem binding site kinetics to achieve cooperativity. Based on these results, we assembled a library of plasmids containing ATFs, binding sites, regulatory elements, and fluorescent reporters.  We then integrated these genes into the genome of Saccharomyces cerevisiae and are currently characterizing them using flow cytometry.
Retrieved from " http://2013.igem.org/Team:Duke/Project/Problem "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
